
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs2.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Inhibition of mRNA nuclear export promotes SARS-CoV-2 pathogenesis - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="322C300B24F3920500004B7A2D840FC3.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="pnas" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC11145185/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11145185/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Proceedings of the National Academy of Sciences of the United States of America" /><meta name="citation_title" content="Inhibition of mRNA nuclear export promotes SARS-CoV-2 pathogenesis" /><meta name="citation_author" content="Menghan Mei" /><meta name="citation_author_institution" content="aDepartment of Biochemistry, Center for Structural Biology, Vanderbilt University School of Medicine, Nashville, TN 37232" /><meta name="citation_author" content="Anastasija Cupic" /><meta name="citation_author_institution" content="bDepartment of Microbiology, Icahn School of Medicine at Mount Sinai, New York, NY 10029" /><meta name="citation_author_institution" content="cGraduate School of Biomedical Sciences, Icahn School of Medicine at Mount Sinai, New York, NY 10029" /><meta name="citation_author" content="Lisa Miorin" /><meta name="citation_author_institution" content="bDepartment of Microbiology, Icahn School of Medicine at Mount Sinai, New York, NY 10029" /><meta name="citation_author_institution" content="dGlobal Health and Emerging Pathogens Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029" /><meta name="citation_author" content="Chengjin Ye" /><meta name="citation_author_institution" content="eTexas Biomedical Research Institute, San Antonio, TX 78227" /><meta name="citation_author" content="Tolga Cagatay" /><meta name="citation_author_institution" content="fDepartment of Cell Biology, University of Texas Southwestern Medical Center, Dallas, TX 75390" /><meta name="citation_author" content="Ke Zhang" /><meta name="citation_author_institution" content="fDepartment of Cell Biology, University of Texas Southwestern Medical Center, Dallas, TX 75390" /><meta name="citation_author_institution" content="gShanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai 200031, China" /><meta name="citation_author" content="Komal Patel" /><meta name="citation_author_institution" content="aDepartment of Biochemistry, Center for Structural Biology, Vanderbilt University School of Medicine, Nashville, TN 37232" /><meta name="citation_author_institution" content="hArpirnaut Program, Vanderbilt University School of Medicine, Nashville, TN 37232" /><meta name="citation_author" content="Natalie Wilson" /><meta name="citation_author_institution" content="aDepartment of Biochemistry, Center for Structural Biology, Vanderbilt University School of Medicine, Nashville, TN 37232" /><meta name="citation_author" content="W. Hayes McDonald" /><meta name="citation_author_institution" content="aDepartment of Biochemistry, Center for Structural Biology, Vanderbilt University School of Medicine, Nashville, TN 37232" /><meta name="citation_author_institution" content="iMass Spectrometry Research Center, Vanderbilt University School of Medicine, Nashville, TN 37232" /><meta name="citation_author" content="Nicholas A. Crossland" /><meta name="citation_author_institution" content="jComparative Pathology Laboratory, National Emerging Infectious Diseases Laboratories, Boston University, Boston, MA 02215" /><meta name="citation_author_institution" content="kDepartment of Pathology and Laboratory Medicine, Boston University School of Medicine, Boston, MA 02118" /><meta name="citation_author" content="Ming Lo" /><meta name="citation_author_institution" content="jComparative Pathology Laboratory, National Emerging Infectious Diseases Laboratories, Boston University, Boston, MA 02215" /><meta name="citation_author" content="Magdalena Rutkowska" /><meta name="citation_author_institution" content="bDepartment of Microbiology, Icahn School of Medicine at Mount Sinai, New York, NY 10029" /><meta name="citation_author_institution" content="cGraduate School of Biomedical Sciences, Icahn School of Medicine at Mount Sinai, New York, NY 10029" /><meta name="citation_author" content="Sadaf Aslam" /><meta name="citation_author_institution" content="bDepartment of Microbiology, Icahn School of Medicine at Mount Sinai, New York, NY 10029" /><meta name="citation_author" content="Ignacio Mena" /><meta name="citation_author_institution" content="bDepartment of Microbiology, Icahn School of Medicine at Mount Sinai, New York, NY 10029" /><meta name="citation_author_institution" content="dGlobal Health and Emerging Pathogens Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029" /><meta name="citation_author" content="Luis Martinez-Sobrido" /><meta name="citation_author_institution" content="eTexas Biomedical Research Institute, San Antonio, TX 78227" /><meta name="citation_author" content="Yi Ren" /><meta name="citation_author_institution" content="aDepartment of Biochemistry, Center for Structural Biology, Vanderbilt University School of Medicine, Nashville, TN 37232" /><meta name="citation_author" content="Adolfo Garc&#x000ed;a-Sastre" /><meta name="citation_author_institution" content="bDepartment of Microbiology, Icahn School of Medicine at Mount Sinai, New York, NY 10029" /><meta name="citation_author_institution" content="dGlobal Health and Emerging Pathogens Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029" /><meta name="citation_author_institution" content="lDepartment of Medicine, Division of Infectious Diseases, Icahn School of Medicine at Mount Sinai, New York, NY 10029" /><meta name="citation_author_institution" content="mDepartment of Pathology, Molecular and Cell-Based Medicine, Icahn School of Medicine at Mount Sinai, New York, NY 10029" /><meta name="citation_author_institution" content="nThe Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029" /><meta name="citation_author" content="Beatriz M. A. Fontoura" /><meta name="citation_author_institution" content="fDepartment of Cell Biology, University of Texas Southwestern Medical Center, Dallas, TX 75390" /><meta name="citation_publication_date" content="2024/05/05" /><meta name="citation_issue" content="22" /><meta name="citation_volume" content="121" /><meta name="citation_doi" content="10.1073/pnas.2314166121" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC11145185/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC11145185/" /><meta name="citation_pmid" content="38768348" /><meta name="DC.Title" content="Inhibition of mRNA nuclear export promotes SARS-CoV-2 pathogenesis" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="National Academy of Sciences" /><meta name="DC.Contributor" content="Menghan Mei" /><meta name="DC.Contributor" content="Anastasija Cupic" /><meta name="DC.Contributor" content="Lisa Miorin" /><meta name="DC.Contributor" content="Chengjin Ye" /><meta name="DC.Contributor" content="Tolga Cagatay" /><meta name="DC.Contributor" content="Ke Zhang" /><meta name="DC.Contributor" content="Komal Patel" /><meta name="DC.Contributor" content="Natalie Wilson" /><meta name="DC.Contributor" content="W. Hayes McDonald" /><meta name="DC.Contributor" content="Nicholas A. Crossland" /><meta name="DC.Contributor" content="Ming Lo" /><meta name="DC.Contributor" content="Magdalena Rutkowska" /><meta name="DC.Contributor" content="Sadaf Aslam" /><meta name="DC.Contributor" content="Ignacio Mena" /><meta name="DC.Contributor" content="Luis Martinez-Sobrido" /><meta name="DC.Contributor" content="Yi Ren" /><meta name="DC.Contributor" content="Adolfo Garc&#x000ed;a-Sastre" /><meta name="DC.Contributor" content="Beatriz M. A. Fontoura" /><meta name="DC.Date" content="2024 May 28" /><meta name="DC.Identifier" content="10.1073/pnas.2314166121" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Inhibition of mRNA nuclear export promotes SARS-CoV-2 pathogenesis" /><meta property="og:type" content="article" /><meta property="og:description" content="This study shows the impact of inhibiting the mRNA nuclear export machinery on SARS-CoV-2 pathogenesis. By defining the mechanism and interaction surface between the SARS-CoV-2 Nsp1 protein and the mRNA export receptor NXF1, which mediates inhibition ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11145185/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11145185/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/2/">Proc Natl Acad Sci U S A</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/463946/">v.121(22); 2024 May 28</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11145185
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11145185/pdf/pnas.202314166.pdf" class="int-view">PDF (11M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11145185/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11145185/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11145185" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11145185%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="lhsvues23Q7tr9m3IpRmfghv9252KYAynDSOxPgB9sTt0HxmrIofubr98nqazXoe">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11145185%2F&amp;text=Inhibition%20of%20mRNA%20nuclear%20export%20promotes%20SARS-CoV-2%20pathogenesis" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11145185%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11145185/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11145185/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38768348/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11145185/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38768348/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11145185/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11145185/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/2/">Proc Natl Acad Sci U S A</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/463946/">v.121(22); 2024 May 28</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11145185
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><a href="//doi.org/10.1073%2Fpnas.2314166121" title="Link to Publisher's site" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"><img src="/corehtml/pmc/pmcgifs/logo-pnas.png" alt="Logo of pnas"><!-- --></img></a></div>
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Proc Natl Acad Sci U S A.</span> 2024 May 28; 121(22): e2314166121. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2024 May 20. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1073%2Fpnas.2314166121" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1073/pnas.2314166121</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC11145185</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38768348">38768348</a></div></div></div><h1 class="content-title">Inhibition of mRNA nuclear export promotes SARS-CoV-2 pathogenesis</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Mei%20M%5BAuthor%5D" class="affpopup" co-rid="_co_idm140263087391424" co-class="co-affbox">Menghan Mei</a>,<sup>#</sup><sup>
a
,</sup><sup>
1
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Cupic%20A%5BAuthor%5D" class="affpopup" co-rid="_co_idm140263096370304" co-class="co-affbox">Anastasija Cupic</a>,<sup>#</sup><sup>
b
,</sup><sup>
c
,</sup><sup>
1
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Miorin%20L%5BAuthor%5D" class="affpopup" co-rid="_co_idm140263096365504" co-class="co-affbox">Lisa Miorin</a>,<sup>
b
,</sup><sup>
d
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ye%20C%5BAuthor%5D" class="affpopup" co-rid="_co_idm140263080096448" co-class="co-affbox">Chengjin Ye</a>,<sup>
e
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Cagatay%20T%5BAuthor%5D" class="affpopup" co-rid="_co_idm140263088241216" co-class="co-affbox">Tolga Cagatay</a>,<sup>
f
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Zhang%20K%5BAuthor%5D" class="affpopup" co-rid="_co_idm140263088238112" co-class="co-affbox">Ke Zhang</a>,<sup>
f
,</sup><sup>
g
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Patel%20K%5BAuthor%5D" class="affpopup" co-rid="_co_idm140263097338224" co-class="co-affbox">Komal Patel</a>,<sup>
a
,</sup><sup>
h
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wilson%20N%5BAuthor%5D" class="affpopup" co-rid="_co_idm140263096811344" co-class="co-affbox">Natalie Wilson</a>,<sup>
a
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=McDonald%20WH%5BAuthor%5D" class="affpopup" co-rid="_co_idm140263096808240" co-class="co-affbox">W. Hayes McDonald</a>,<sup>
a
,</sup><sup>
i
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Crossland%20NA%5BAuthor%5D" class="affpopup" co-rid="_co_idm140263089526912" co-class="co-affbox">Nicholas A. Crossland</a>,<sup>
j
,</sup><sup>
k
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Lo%20M%5BAuthor%5D" class="affpopup" co-rid="_co_idm140263096427056" co-class="co-affbox">Ming Lo</a>,<sup>
j
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Rutkowska%20M%5BAuthor%5D" class="affpopup" co-rid="_co_idm140263096423952" co-class="co-affbox">Magdalena Rutkowska</a>,<sup>
b
,</sup><sup>
c
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Aslam%20S%5BAuthor%5D" class="affpopup" co-rid="_co_idm140263086129152" co-class="co-affbox">Sadaf Aslam</a>,<sup>
b
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Mena%20I%5BAuthor%5D" class="affpopup" co-rid="_co_idm140263086126048" co-class="co-affbox">Ignacio Mena</a>,<sup>
b
,</sup><sup>
d
,</sup><sup>
2
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Martinez-Sobrido%20L%5BAuthor%5D" class="affpopup" co-rid="_co_idm140263071638304" co-class="co-affbox">Luis Martinez-Sobrido</a>,<sup>
e
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ren%20Y%5BAuthor%5D" class="affpopup" co-rid="_co_idm140263096435056" co-class="co-affbox">Yi Ren</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>
a
,</sup><sup>
3
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Garc&#x000ed;a-Sastre%20A%5BAuthor%5D" class="affpopup" co-rid="_co_idm140263085986000" co-class="co-affbox">Adolfo Garc&#x000ed;a-Sastre</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>
b
,</sup><sup>
d
,</sup><sup>
l
,</sup><sup>
m
,</sup><sup>
n
,</sup><sup>
3
</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Fontoura%20BM%5BAuthor%5D" class="affpopup" co-rid="_co_idm140263098142592" co-class="co-affbox">Beatriz M. A. Fontoura</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>
f
,</sup><sup>
3
</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140263087391424"><h3 class="no_margin">Menghan Mei</h3><p><sup>a</sup>Department of Biochemistry, Center for Structural Biology, Vanderbilt University School of Medicine, Nashville, TN
37232</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Mei%20M%5BAuthor%5D">Menghan Mei</a></div></div><div id="_co_idm140263096370304"><h3 class="no_margin">Anastasija Cupic</h3><p><sup>b</sup>Department of Microbiology, Icahn School of Medicine at Mount Sinai, New York, NY
10029</p><p><sup>c</sup>Graduate School of Biomedical Sciences, Icahn School of Medicine at Mount Sinai, New York, NY
10029</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Cupic%20A%5BAuthor%5D">Anastasija Cupic</a></div></div><div id="_co_idm140263096365504"><h3 class="no_margin">Lisa Miorin</h3><p><sup>b</sup>Department of Microbiology, Icahn School of Medicine at Mount Sinai, New York, NY
10029</p><p><sup>d</sup>Global Health and Emerging Pathogens Institute, Icahn School of Medicine at Mount Sinai, New York, NY
10029</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Miorin%20L%5BAuthor%5D">Lisa Miorin</a></div></div><div id="_co_idm140263080096448"><h3 class="no_margin">Chengjin Ye</h3><p><sup>e</sup>Texas Biomedical Research Institute, San Antonio, TX
78227</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ye%20C%5BAuthor%5D">Chengjin Ye</a></div></div><div id="_co_idm140263088241216"><h3 class="no_margin">Tolga Cagatay</h3><p><sup>f</sup>Department of Cell Biology, University of Texas Southwestern Medical Center, Dallas, TX
75390</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Cagatay%20T%5BAuthor%5D">Tolga Cagatay</a></div></div><div id="_co_idm140263088238112"><h3 class="no_margin">Ke Zhang</h3><p><sup>f</sup>Department of Cell Biology, University of Texas Southwestern Medical Center, Dallas, TX
75390</p><p><sup>g</sup>Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai
200031, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Zhang%20K%5BAuthor%5D">Ke Zhang</a></div></div><div id="_co_idm140263097338224"><h3 class="no_margin">Komal Patel</h3><p><sup>a</sup>Department of Biochemistry, Center for Structural Biology, Vanderbilt University School of Medicine, Nashville, TN
37232</p><p><sup>h</sup>Arpirnaut Program, Vanderbilt University School of Medicine, Nashville, TN
37232</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Patel%20K%5BAuthor%5D">Komal Patel</a></div></div><div id="_co_idm140263096811344"><h3 class="no_margin">Natalie Wilson</h3><p><sup>a</sup>Department of Biochemistry, Center for Structural Biology, Vanderbilt University School of Medicine, Nashville, TN
37232</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wilson%20N%5BAuthor%5D">Natalie Wilson</a></div></div><div id="_co_idm140263096808240"><h3 class="no_margin">W. Hayes McDonald</h3><p><sup>a</sup>Department of Biochemistry, Center for Structural Biology, Vanderbilt University School of Medicine, Nashville, TN
37232</p><p><sup>i</sup>Mass Spectrometry Research Center, Vanderbilt University School of Medicine, Nashville, TN
37232</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=McDonald%20WH%5BAuthor%5D">W. Hayes McDonald</a></div></div><div id="_co_idm140263089526912"><h3 class="no_margin">Nicholas A. Crossland</h3><p><sup>j</sup>Comparative Pathology Laboratory, National Emerging Infectious Diseases Laboratories, Boston University, Boston, MA
02215</p><p><sup>k</sup>Department of Pathology and Laboratory Medicine, Boston University School of Medicine, Boston, MA
02118</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Crossland%20NA%5BAuthor%5D">Nicholas A. Crossland</a></div></div><div id="_co_idm140263096427056"><h3 class="no_margin">Ming Lo</h3><p><sup>j</sup>Comparative Pathology Laboratory, National Emerging Infectious Diseases Laboratories, Boston University, Boston, MA
02215</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Lo%20M%5BAuthor%5D">Ming Lo</a></div></div><div id="_co_idm140263096423952"><h3 class="no_margin">Magdalena Rutkowska</h3><p><sup>b</sup>Department of Microbiology, Icahn School of Medicine at Mount Sinai, New York, NY
10029</p><p><sup>c</sup>Graduate School of Biomedical Sciences, Icahn School of Medicine at Mount Sinai, New York, NY
10029</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Rutkowska%20M%5BAuthor%5D">Magdalena Rutkowska</a></div></div><div id="_co_idm140263086129152"><h3 class="no_margin">Sadaf Aslam</h3><p><sup>b</sup>Department of Microbiology, Icahn School of Medicine at Mount Sinai, New York, NY
10029</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Aslam%20S%5BAuthor%5D">Sadaf Aslam</a></div></div><div id="_co_idm140263086126048"><h3 class="no_margin">Ignacio Mena</h3><p><sup>b</sup>Department of Microbiology, Icahn School of Medicine at Mount Sinai, New York, NY
10029</p><p><sup>d</sup>Global Health and Emerging Pathogens Institute, Icahn School of Medicine at Mount Sinai, New York, NY
10029</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Mena%20I%5BAuthor%5D">Ignacio Mena</a></div></div><div id="_co_idm140263071638304"><h3 class="no_margin">Luis Martinez-Sobrido</h3><p><sup>e</sup>Texas Biomedical Research Institute, San Antonio, TX
78227</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Martinez-Sobrido%20L%5BAuthor%5D">Luis Martinez-Sobrido</a></div></div><div id="_co_idm140263096435056"><h3 class="no_margin">Yi Ren</h3><p><sup>a</sup>Department of Biochemistry, Center for Structural Biology, Vanderbilt University School of Medicine, Nashville, TN
37232</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ren%20Y%5BAuthor%5D">Yi Ren</a></div></div><div id="_co_idm140263085986000"><h3 class="no_margin">Adolfo Garc&#x000ed;a-Sastre</h3><p><sup>b</sup>Department of Microbiology, Icahn School of Medicine at Mount Sinai, New York, NY
10029</p><p><sup>d</sup>Global Health and Emerging Pathogens Institute, Icahn School of Medicine at Mount Sinai, New York, NY
10029</p><p><sup>l</sup>Department of Medicine, Division of Infectious Diseases, Icahn School of Medicine at Mount Sinai, New York, NY
10029</p><p><sup>m</sup>Department of Pathology, Molecular and Cell-Based Medicine, Icahn School of Medicine at Mount Sinai, New York, NY
10029</p><p><sup>n</sup>The Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY
10029</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Garc&#x000ed;a-Sastre%20A%5BAuthor%5D">Adolfo Garc&#x000ed;a-Sastre</a></div></div><div id="_co_idm140263098142592"><h3 class="no_margin">Beatriz M. A. Fontoura</h3><p><sup>f</sup>Department of Cell Biology, University of Texas Southwestern Medical Center, Dallas, TX
75390</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Fontoura%20BM%5BAuthor%5D">Beatriz M. A. Fontoura</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140263098527584_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140263098527584_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140263098527584_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140263098527584_ai" style="display:none"><div class="fm-affl" id="aff1"><sup>a</sup>Department of Biochemistry, Center for Structural Biology, Vanderbilt University School of Medicine, Nashville, TN
37232</div><div class="fm-affl" id="aff2"><sup>b</sup>Department of Microbiology, Icahn School of Medicine at Mount Sinai, New York, NY
10029</div><div class="fm-affl" id="aff3"><sup>c</sup>Graduate School of Biomedical Sciences, Icahn School of Medicine at Mount Sinai, New York, NY
10029</div><div class="fm-affl" id="aff4"><sup>d</sup>Global Health and Emerging Pathogens Institute, Icahn School of Medicine at Mount Sinai, New York, NY
10029</div><div class="fm-affl" id="aff5"><sup>e</sup>Texas Biomedical Research Institute, San Antonio, TX
78227</div><div class="fm-affl" id="aff6"><sup>f</sup>Department of Cell Biology, University of Texas Southwestern Medical Center, Dallas, TX
75390</div><div class="fm-affl" id="aff7"><sup>g</sup>Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai
200031, China</div><div class="fm-affl" id="aff8"><sup>h</sup>Arpirnaut Program, Vanderbilt University School of Medicine, Nashville, TN
37232</div><div class="fm-affl" id="aff9"><sup>i</sup>Mass Spectrometry Research Center, Vanderbilt University School of Medicine, Nashville, TN
37232</div><div class="fm-affl" id="aff10"><sup>j</sup>Comparative Pathology Laboratory, National Emerging Infectious Diseases Laboratories, Boston University, Boston, MA
02215</div><div class="fm-affl" id="aff11"><sup>k</sup>Department of Pathology and Laboratory Medicine, Boston University School of Medicine, Boston, MA
02118</div><div class="fm-affl" id="aff12"><sup>l</sup>Department of Medicine, Division of Infectious Diseases, Icahn School of Medicine at Mount Sinai, New York, NY
10029</div><div class="fm-affl" id="aff13"><sup>m</sup>Department of Pathology, Molecular and Cell-Based Medicine, Icahn School of Medicine at Mount Sinai, New York, NY
10029</div><div class="fm-affl" id="aff14"><sup>n</sup>The Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY
10029</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup>Corresponding author.</div><div><sup>#</sup>Contributed equally.</div><div><span class="contrib-email" id="contrib-a.l.b.f.p">Yi Ren: <a href="mailto:dev@null" data-email="ude.tlibrednav@ner.iy" class="oemail">ude.tlibrednav@ner.iy</a>; </span> <span class="contrib-email" id="contrib-a.l.b.f.q">Adolfo Garc&#x000ed;a-Sastre: <a href="mailto:dev@null" data-email="ude.mssm@ertsas-aicrag.ofloda" class="oemail">ude.mssm@ertsas-aicrag.ofloda</a>; </span> <span class="contrib-email" id="contrib-a.l.b.f.r">Beatriz M. A. Fontoura: <a href="mailto:dev@null" data-email="ude.nretsewhtuoSTU@aruotnoF.zirtaeB" class="oemail">ude.nretsewhtuoSTU@aruotnoF.zirtaeB</a></span> </div><div id="cor1"><sup>3</sup>To whom correspondence may be addressed. Email: <a href="mailto:dev@null" data-email="ude.tlibrednav@ner.iy" class="oemail">ude.tlibrednav@ner.iy</a>, <a href="mailto:dev@null" data-email="ude.mssm@ertsas-aicrag.ofloda" class="oemail">ude.mssm@ertsas-aicrag.ofloda</a>, or <a href="mailto:dev@null" data-email="ude.nretsewhtuoSTU@aruotnoF.zirtaeB" class="oemail">ude.nretsewhtuoSTU@aruotnoF.zirtaeB</a>.</div><div id="fn1">Edited by Anita H. Corbett, Emory University, Atlanta, GA; received August 16, 2023; accepted April 9, 2024 by Editorial Board Member Rebecca Heald</div><div id="fn2"><sup>1</sup>M.M. and A.C. contributed equally to this work.</div><div id="fn3"><sup>2</sup>Present address: Department of Immunology and Microbiology, The Scripps Research Institute, La Jolla, CA 92037.</div></div><div class="fm-article-notes hide half_rhythm" id="idm140263098527584_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2023 Aug 16; Accepted 2024 Apr 9.</div></div><div class="permissions half_rhythm hide" id="idm140263098527584_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  &#x000a9; 2024 the Author(s). Published by PNAS.</div><div class="license half_rhythm">This open access article is distributed under <a href="https://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">Creative Commons Attribution License 4.0 (CC BY)</a>.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Appendix 01 (PDF)</div><div><a href="/pmc/articles/PMC11145185/bin/pnas.2314166121.sapp.pdf" data-ga-action="click_feat_suppl">pnas.2314166121.sapp.pdf</a><span style="color:gray"> (16M)</span></div><div class="small guid">GUID: 1A005DD1-4E8E-463F-AFCE-1A779751163D</div></div></dd></dl><dl data-length="62" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first-last">All study data are included in the article and/or <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2314166121#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em></a>.</p></dd></dl></div><div id="abstract-a.l.b.q" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.l.b.qtitle" style="text-transform: uppercase;">Significance</h2><!--article-meta--><div><p class="p p-first-last">This study shows the impact of inhibiting the mRNA nuclear export machinery on SARS-CoV-2 pathogenesis. By defining the mechanism and interaction surface between the SARS-CoV-2 Nsp1 protein and the mRNA export receptor NXF1, which mediates inhibition of cellular mRNA nuclear export, we were able to generate a mutant SARS-CoV-2 that does not inhibit mRNA nuclear export and is attenuated. Our results establish the importance of Nsp1-mediated mRNA export inhibition for SARS-CoV-2 replication and pathogenesis and reveal a surface on Nsp1 protein that can be potentially targeted for antiviral development.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">mRNA export, nuclear transport, SARS-CoV-2, NXF1, Nsp1</span></div></div><div id="abstract-a.l.b.r" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.l.b.rtitle" style="text-transform: uppercase;">Abstract</h2><!--article-meta--><div><p class="p p-first-last">The nonstructural protein 1 (Nsp1) of SARS-CoV-2 (Severe Acute Respiratory Syndrome Coronavirus 2) is a virulence factor that targets multiple cellular pathways to inhibit host gene expression and antiviral response. However, the underlying mechanisms of the various Nsp1-mediated functions and their contributions to SARS-CoV-2 virulence remain unclear. Among the targets of Nsp1 is the mRNA (messenger ribonucleic acid) export receptor NXF1-NXT1, which mediates nuclear export of mRNAs from the nucleus to the cytoplasm. Based on Nsp1 crystal structure, we generated mutants on Nsp1 surfaces and identified an acidic N-terminal patch that is critical for interaction with NXF1-NXT1. Photoactivatable Nsp1 probe reveals the RNA Recognition Motif (RRM) domain of NXF1 as an Nsp1 N-terminal binding site. By mutating the Nsp1 N-terminal acidic patch, we identified a separation-of-function mutant of Nsp1 that retains its translation inhibitory function but substantially loses its interaction with NXF1 and reverts Nsp1-mediated mRNA export inhibition. We then generated a recombinant (r)SARS-CoV-2 mutant on the Nsp1 N-terminal acidic patch and found that this surface is key to promote NXF1 binding and inhibition of host mRNA nuclear export, viral replication, and pathogenicity in vivo. Thus, these findings provide a mechanistic understanding of Nsp1-mediated mRNA export inhibition and establish the importance of this pathway in the virulence of SARS-CoV-2.</p></div></div><div id="body-a.m" class="tsec sec"><h2 class="headless nomenu"></h2><p class="p p-first">Inhibition of host gene expression is an important strategy employed by diverse viruses to facilitate their replication, including SARS-CoV-2 which has caused huge impacts on global health and the economy. This strategy allows viruses to not only seize the cellular resources to produce viral proteins but also suppress host immune response by inhibiting the expression of antiviral factors. The nonstructural protein 1 (Nsp1) of SARS-CoV-2 is a key virulence factor that acts at multiple levels to inhibit host gene expression (<a href="#r1" rid="r1" class=" bibr popnode">1</a><a href="#r2" rid="r2" class=" bibr popnode"></a>&#x02013;<a href="#r3" rid="r3" class=" bibr popnode">3</a>). Nsp1 is a 20 kDa protein encoded at the very 5&#x02032; end of the SARS-CoV-2 RNA genome. It is well characterized that Nsp1 targets the ribosome 40S subunit to plug the mRNA entry channel and thereby inhibit protein translation (<a href="#r2" rid="r2" class=" bibr popnode">2</a>, <a href="#r4" rid="r4" class=" bibr popnode">4</a><a href="#r5" rid="r5" class=" bibr popnode"></a>&#x02013;<a href="#r6" rid="r6" class=" bibr popnode">6</a>). Moreover, Nsp1 has been shown to induce mRNA decay (<a href="#r1" rid="r1" class=" bibr popnode">1</a>) and this effect requires mRNA association with the ribosome (<a href="#r7" rid="r7" class=" bibr popnode">7</a>). We have previously uncovered an unknown Nsp1 function as an inhibitor of host mRNA nuclear export (<a href="#r3" rid="r3" class=" bibr popnode">3</a>). We demonstrated that Nsp1 directly targets the host mRNA export receptor, the NXF1-NXT1 heterodimer, which provides mature mRNAs with access to the nuclear pore complex (NPC) for their nuclear export. As a result, SARS-CoV-2 infection reduces NXF1 docking at the NPC and causes nuclear retention of cellular mRNAs (<a href="#r3" rid="r3" class=" bibr popnode">3</a>). Others have also reported on the mRNA export inhibition by Nsp1 during SARS-CoV-2 infection (<a href="#r1" rid="r1" class=" bibr popnode">1</a>, <a href="#r8" rid="r8" class=" bibr popnode">8</a>).</p><p>mRNA nuclear export is an obligatory step for gene expression in all eukaryotic cells (<a href="#r9" rid="r9" class=" bibr popnode">9</a>, <a href="#r10" rid="r10" class=" bibr popnode">10</a>). We previously reported that influenza A virus NS1 protein also targets the NXF1-NXT1 complex to inhibit host mRNA nuclear export (<a href="#r11" rid="r11" class=" bibr popnode">11</a>, <a href="#r12" rid="r12" class=" bibr popnode">12</a>). In particular, NS1 recognizes the NPC binding domain of NXF1 and mimics how NPC interacts with NXF1-NXT1. Consequently, NS1 prevents binding of NXF1-NXT1 to the NPC. Influenza A virus carrying a mutant NS1 incapable of binding to NXF1-NXT1 is attenuated (<a href="#r11" rid="r11" class=" bibr popnode">11</a>). With respect to SARS-CoV-2, Nsp1 acts at a different step during the NXF1-NXT1-mediated process (<a href="#r3" rid="r3" class=" bibr popnode">3</a>). NXF1-NXT1 itself is capable of binding to RNA in vitro, but its loading onto mRNAs in vivo requires the action of the TRanscription-EXport (TREX) complex (<a href="#r13" rid="r13" class=" bibr popnode">13</a>, <a href="#r14" rid="r14" class=" bibr popnode">14</a>). Nsp1 decreased the association of the TREX complex component Aly/REF to NXF1-NXT1, implying that Nsp1 may prevent the proper configuration of NXF1-NXT1 for the assembly of a productive RNA export complex (<a href="#r3" rid="r3" class=" bibr popnode">3</a>) and/or disrupt both nuclear import and export of NXF1. The molecular mechanism underlying the SARS-CoV-2 Nsp1 mediated mRNA export inhibition, however, remains unknown.</p><p>Considering the growing list of SARS-CoV-2 Nsp1&#x02019;s targets in host cells, including ribosome, NXF1-NXT1, and DNA polymerase &#x003b1;-primase (<a href="#r3" rid="r3" class=" bibr popnode">3</a>, <a href="#r15" rid="r15" class=" bibr popnode">15</a>, <a href="#r16" rid="r16" class=" bibr popnode">16</a>), understanding the underlying molecular interactions is crucial to design separation-of-function Nsp1 mutants to dissect the impacts of Nsp1 on different cellular pathways in SARS-CoV-2 infected cells and on viral pathogenesis. An example is the Nsp1-K164A/H165A mutant that inhibits Nsp1 binding to the ribosome. Cryo-EM structures reveal that K164 and H165 reside on the binding interface between Nsp1 and the 40S ribosomal subunit (<a href="#r2" rid="r2" class=" bibr popnode">2</a>, <a href="#r4" rid="r4" class=" bibr popnode">4</a><a href="#r5" rid="r5" class=" bibr popnode"></a>&#x02013;<a href="#r6" rid="r6" class=" bibr popnode">6</a>). This mutant serves as a valuable tool in the studies of Nsp1 translation shutoff because it specifically disrupts the Nsp1-ribosome interaction. To date, there are no available Nsp1 mutants to perturb the Nsp1 and NXF1-NXT1 interaction for further functional studies.</p><p class="p">Here, we identify the key regions of Nsp1 that mediate NXF1-NXT1 recognition and are responsible for Nsp1-mediated mRNA export inhibition. We found that both the N-terminal (Nsp1-N) and C-terminal (Nsp1-C) domains of Nsp1 are involved in NXF1-NXT1 interaction. We demonstrate that Nsp1-N employs a patch of acidic amino acids (D33, E36, E37, and E41) for binding to NXF1-NXT1. Using a photoactivatable Nsp1 probe, we show that this Nsp1 acidic patch interfaces with a basic patch on the RNA Recognition Motif (RRM) of NXF1. Mutation on the Nsp1 acidic patch compromised its ability to block mRNA nuclear export but did not affect Nsp1&#x02019;s inhibitory function in protein translation, indicating this is a separation-of-function mutation. Furthermore, we generated a recombinant (r)SARS-CoV-2 with mutations in the N-terminal acidic patch of Nsp1 and show that this mutant virus is unable to inhibit mRNA export and is attenuated in cells and mice. In fact, the mutant Nsp1 does not properly interact with NXF1 during infection. Together, these findings point to a key function for Nsp1-NXF1 interaction in promoting viral replication and pathogenesis by inhibiting the mRNA export machinery.</p></div><div id="s1" class="tsec sec"><h2 class="head no_bottom_margin" id="s1title" style="text-transform: uppercase;">Results</h2><div id="s2" class="sec sec-first"><p></p><h3 id="s2title" class="inline">An Acidic Patch on the Nsp1-N Terminal Domain Is Critical for NXF1-NXT1 Interaction. </h3><p class="p p-first">Our previous studies established that SARS-CoV-2 Nsp1 inhibits host mRNA export by targeting the mRNA export receptor NXF1-NXT1 complex (<a href="#r3" rid="r3" class=" bibr popnode">3</a>). Here, we sought to determine the mechanisms of the interaction between Nsp1 and NXF1-NXT1. The Nsp1 protein contains an N-terminal globular domain (Nsp1-N) and a C-terminal domain (Nsp1-C). In previous studies, we showed that Nsp1-Full Length (FL, residues 1 to 180) and Nsp1-N (residues 1 to 129) bind to NXF1-NXT1 (<a href="#r3" rid="r3" class=" bibr popnode">3</a>). The contribution of Nsp1-C to NXF1-NXT1 interaction was not examined. We cloned a GST-tagged Nsp1-C (residues 133-180) and used purified recombinant proteins to test interaction with NXF1-NXT1 by GST pull down. We found that each Nsp1-N or Nsp1-C alone is capable of binding to NXF1-NXT1, but both domains are required for optimal binding (<a href="/pmc/articles/PMC11145185/figure/fig01/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig01" rid-ob="ob-fig01" co-legend-rid="lgnd_fig01"><span>Fig. 1 <em>A</em> and <em>B</em></span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig01" co-legend-rid="lgnd_fig01"><a href="/pmc/articles/PMC11145185/figure/fig01/" target="figure" rid-figpopup="fig01" rid-ob="ob-fig01"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140263096464240"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11145185_pnas.2314166121fig01.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pnas.2314166121fig01.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11145185/bin/pnas.2314166121fig01.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140263096464240"><a target="object" rel="noopener" href="/pmc/articles/PMC11145185/figure/fig01/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig01"><div><a class="figpopup" href="/pmc/articles/PMC11145185/figure/fig01/" target="figure" rid-figpopup="fig01" rid-ob="ob-fig01">Fig. 1.</a></div><!--caption a7--><div class="caption"><p>An acidic patch on the Nsp1 N-terminal domain plays a key role in NXF1-NXT1 binding. (<em>A</em>) Schematic representation of SARS-CoV-2 Nsp1 and human NXF1-NXT1. (<em>B</em>) Both the N-terminal and C-terminal domains of Nsp1 are involved in NXF1-NXT1 binding. In vitro GST pull-down assays were performed with purified GST-tagged Nsp1 variants and NXF1-NXT1. (<em>C</em>) Mutations of an acidic patch on Nsp1-N significantly reduced NXF1-NXT1 binding. (<em>D</em>) Cartoon representation of Nsp1-N (PDB ID 7K7P) highlighting the acidic patch on Nsp1-N.</p></div></div></div><p>To probe the mechanism by which Nsp1-N recognizes NXF1-NXT1, we systematically mutated Nsp1-N based on its crystal structure and examined the binding between Nsp1 mutants and NXF1-NXT1 (<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2314166121#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Table S1</a>). The following factors are considered to choose mutation candidates: 1) residues that are solvent exposed, 2) residues with characterized functions (R99A for ribosome binding; R124E/K125E for ribosome binding and promoting RNA degradation (<a href="#r17" rid="r17" class=" bibr popnode">17</a>); L27D/V28D for DNA polymerase &#x003b1;-primase binding) (<a href="#r16" rid="r16" class=" bibr popnode">16</a>), and 3) residues forming charged (i.e. D33K/E36K/E37K/E41K, E55A/E57A/K58E) or hydrophobic patches (L27D/V28D) on the protein surface.</p><p>We found that a quadruple point mutant Nsp1<sup>D33K/E36K/E37K/E41K</sup> substantially reduced NXF1-NXT1 binding compared to wild-type Nsp1 (<a href="/pmc/articles/PMC11145185/figure/fig01/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig01" rid-ob="ob-fig01" co-legend-rid="lgnd_fig01"><span>Fig. 1<em>C</em></span></a>). Some residual binding remained and may involve Nsp1-C. Nsp1-N folds into a &#x003b2;-barrel capped with an &#x003b1;-helix. D33, E36, E37, and E41 are localized at or near the capping helix forming a highly acidic patch on the Nsp1-N surface (<a href="/pmc/articles/PMC11145185/figure/fig01/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig01" rid-ob="ob-fig01" co-legend-rid="lgnd_fig01"><span>Fig. 1<em>D</em></span></a>). The critical role of this acidic patch is consistent with the salt-sensitive nature of the Nsp1 interaction with NXF1-NXT1. Of note, other surface charge mutants of Nsp1 (Nsp1<sup>E55A/E57A/K58E</sup>, Nsp1<sup>E91K/L92A/E93K/I95A</sup>) do not reduce NXF1-NXT1 binding (<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2314166121#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Table S1</a>).</p><p class="p p-last">Several residues on the N-terminal domain of Nsp1 have been previously characterized. For instance, R124 and K125 are highly conserved among &#x003b2;-CoVs and play key roles in Nsp1&#x02019;s RNA cleavage function (<a href="#r18" rid="r18" class=" bibr popnode">18</a>). Nsp1<sup>R99A</sup> and Nsp1<sup>R124E/K125E</sup> mutants were recently shown to reduce Nsp1 binding to ribosome (<a href="#r17" rid="r17" class=" bibr popnode">17</a>). As shown in <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2314166121#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S1</a>, these mutants do not affect the interaction between Nsp1 and NXF1-NXT1. In addition, the hydrophobic patch mutant Nsp1<sup>L27D/V28D</sup>, which disrupts Nsp1 interaction with DNA polymerase &#x003b1;-primase (<a href="#r16" rid="r16" class=" bibr popnode">16</a>), does not affect Nsp1 and NXF1-NXT1 interaction. Together, our results indicate that the acidic patch is critical for Nsp1 binding to NXF1-NXT1 and is separated from the well-characterized residues involved in other Nsp1-host interactions at Nsp1-N.</p></div><div id="s3" class="sec"><p></p><h3 id="s3title" class="inline">Photoactivatable Nsp1 Probe Identifies NXF1-RRM As an Nsp1 Binding Site. </h3><p class="p p-first">Since Nsp1<sup>D33K/E36K/E37K/E41K</sup> is a potential candidate for separation-of-function mutant for mRNA export, we focused on further characterizing the interaction between Nsp1-N and NXF1-NXT1. We sought to determine the Nsp1-N binding site on NXF1 using a photo-activatable Nsp1 probe (<a href="/pmc/articles/PMC11145185/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2<em>A</em></span></a>). We incorporated a photo-crosslinking amino acid <em>p</em>-benzoyl-L-phenylalanine (<em>p</em>Bpa) to replace His45 on the Nsp1 surface (Nsp1<em><sup>p</sup></em><sup>BPa45</sup>) (<a href="#r19" rid="r19" class=" bibr popnode">19</a>). His45 was chosen because it is near the Nsp1-N acidic patch. There is a large chance that this residue is in close proximity to NXF1 in the Nsp1-NXF1-NXT1 complex to allow crosslinking. To generate Nsp1<em><sup>p</sup></em><sup>BPa45</sup>, Nsp1 plasmid with His45 replaced by an amber codon was introduced together in <em>Escherichia coli</em> with a plasmid carrying an engineered aminoacyl-tRNA synthesis/tRNA pair that incorporates <em>p</em>Bpa into proteins in response to the amber codon. We successfully expressed Nsp1<em><sup>p</sup></em><sup>BPa45</sup> in <em>E. coli</em> and purified the protein.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig02" co-legend-rid="lgnd_fig02"><a href="/pmc/articles/PMC11145185/figure/fig02/" target="figure" rid-figpopup="fig02" rid-ob="ob-fig02"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140263095646384"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11145185_pnas.2314166121fig02.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pnas.2314166121fig02.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11145185/bin/pnas.2314166121fig02.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140263095646384"><a target="object" rel="noopener" href="/pmc/articles/PMC11145185/figure/fig02/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig02"><div><a class="figpopup" href="/pmc/articles/PMC11145185/figure/fig02/" target="figure" rid-figpopup="fig02" rid-ob="ob-fig02">Fig. 2.</a></div><!--caption a7--><div class="caption"><p>Photoactivatable Nsp1 probe identifies the RRM domain of NXF1 as an Nsp1-N binding site. (<em>A</em>) Schematics for incorporation of the photoactivatable nonnatural amino acid <em>p</em>Bpa into Nsp1. (<em>B</em>) UV treatment of Nsp1<em><sup>p</sup></em><sup>BPa45</sup> in the presence of NXF1-NXT1 generated a new band consistent with the size of crosslinked Nsp1<em><sup>p</sup></em><sup>BPa45</sup>-NXF1 complex on SDS-PAGE gel. (<em>C</em>) Tandem mass spectrum for a peptide crosslinked to I190 of the NXF1-RRM domain. (<em>D</em>) Cartoon representation of NXF1-RRM and Nsp1-N. Positively charged residues (blue) are enriched on the surface of NXF1-RRM forming a basic patch. The yellow star denotes the position of the incorporated <em>p</em>Bpa at residue 45. The crosslinking site between Nsp1<em><sup>p</sup></em><sup>BPa45</sup> and NXF1-NXT1 is labeled. (<em>E</em>) Mutations of the basic patch residues on NXF1-RRM reduced Nsp1 interaction. In vitro GST pull-down assays were performed with GST-tagged Nsp1 and NXF1-NXT1 variants.</p></div></div></div><p>For photocrosslinking, Nsp1<em><sup>p</sup></em><sup>BPa45</sup> or NXF1-NXT1 or their mixture was mixed and treated with UV light at 365 nm. A new band corresponding to the size of the crosslinked Nsp1<em><sup>p</sup></em><sup>BPa45</sup>-NXF1 complex was observed in the mixture of Nsp1<em><sup>p</sup></em><sup>BPa45</sup> and NXF1-NXT1 on an SDS-PAGE gel (<a href="/pmc/articles/PMC11145185/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2<em>B</em></span></a>). The SDS-PAGE gel band of the putative Nsp1<em><sup>p</sup></em><sup>BPa45</sup>-NXF1 complex from the UV treated mixture was excised and subjected to trypsin digestion and mass spectrometry analysis. A total of 21 interprotein crosslinks were identified with all mapped to the RRM domain of NXF1 (<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2314166121#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix,</em> Table S2</a>). Of note, 12 crosslinks are mapped to the <sup>187</sup>[ISIIINSSAPPHTILNELKPEQVEQLK]<sup>213</sup> peptide of NXF1-RRM and 7 crosslinks are mapped to the same amino acid I190 within this peptide (<a href="/pmc/articles/PMC11145185/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2<em>C</em></span></a>).</p><p class="p p-last">The RRM domain of NXF1 contains a four-stranded, antiparallel &#x003b2;-sheet and two helices packing against the &#x003b2;-sheet (<a href="/pmc/articles/PMC11145185/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2<em>D</em></span></a>). The dominant crosslinked residue I190 is located at the &#x003b2;-sheet face. An examination of the surface property of NXF1 RRM shows that the &#x003b2;-sheet face is highly enriched in positively charged residues (R128, K129, R158, R185, and R186). These positively charged residues are highly localized, forming a basic patch on the RRM surface. Considering that Nsp1 employs an acidic patch for binding to NXF1-NXT1, our results strongly suggest that Nsp1 acidic patch interfaces with the basic patch on the NXF1-RRM surface. To examine the contribution of the NXF1-RRM basic patch in Nsp1 binding, we generated three NXF1 mutants, R128E/K129E, R185E/R186E, and R158E. We found that all three basic patch mutants significantly reduced Nsp1 binding (<a href="/pmc/articles/PMC11145185/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2<em>E</em></span></a>). Some residual binding may involve Nsp1-C. These results demonstrate the critical role of the NXF1-RRM basic patch in binding to Nsp1, which is strongly supported by the crosslinking studies. Together, our crosslinking and mutagenesis studies on NXF1-RRM have two implications. First, residue 45 of Nsp1 is indeed near the interface between Nsp1 and NXF1-NXT1, which reinforces our identification of the Nsp1 acidic patch as a key NXF1-NXT1 binding site. Second, electrostatic interaction involving the basic patch on the RRM domain of NXF1 and the acidic patch on Nsp1 plays a critical role in their interaction.</p></div><div id="s4" class="sec"><p></p><h3 id="s4title" class="inline">The Nsp1 N-Terminal Acidic Patch Is Key for the Inhibitory Function of Nsp1 on Nuclear Export of mRNAs and Not for Inhibition of Translation. </h3><p class="p p-first">Next, we tested the effect of Nsp1<sup>D33K/E36K/E37K/E41K</sup> on nuclear export of host mRNA. To this end, cells were transfected with wild-type Nsp1 or with the Nsp1 mutant for 16 h, and the intracellular distribution of poly(A) RNA was determined by fluorescence in situ hybridization (FISH) combined with immunofluorescence microscopy to detect the Nsp1 proteins (<a href="/pmc/articles/PMC11145185/figure/fig03/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig03" rid-ob="ob-fig03" co-legend-rid="lgnd_fig03"><span>Fig. 3 <em>A&#x02013;</em><em>G</em></span></a>). We observed a decrease in whole cell (total) intracellular levels of poly(A) RNA upon expression of the wild-type or mutant Nsp1 proteins (<a href="/pmc/articles/PMC11145185/figure/fig03/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig03" rid-ob="ob-fig03" co-legend-rid="lgnd_fig03"><span>Fig. 3 <em>C</em> and <em>F</em></span></a>). Additionally, cells expressing wild-type Nsp1 showed an increase in the absolute or relative nuclear fluorescence intensity of poly(A) RNA, suggesting nuclear retention of poly(A) RNA (<a href="/pmc/articles/PMC11145185/figure/fig03/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig03" rid-ob="ob-fig03" co-legend-rid="lgnd_fig03"><span>Fig. 3 <em>D</em> and <em>G</em></span></a>). In contrast, absolute or relative nuclear fluorescence intensity of poly(A) RNA did not increase in cells transfected with Nsp1<sup>D33K/E36K/E37K/E41K</sup>, indicating no nuclear export inhibition of mRNA (<a href="/pmc/articles/PMC11145185/figure/fig03/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig03" rid-ob="ob-fig03" co-legend-rid="lgnd_fig03"><span>Fig. 3 <em>D</em> and <em>G</em></span></a>). Expression levels of wild-type and mutant Nsp1 proteins were quantified and showed that although Nsp1 mutant is expressed at higher levels than the wild-type protein (<a href="/pmc/articles/PMC11145185/figure/fig03/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig03" rid-ob="ob-fig03" co-legend-rid="lgnd_fig03"><span>Fig. 3<em>H</em></span></a>), it is unable to block mRNA export (<a href="/pmc/articles/PMC11145185/figure/fig03/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig03" rid-ob="ob-fig03" co-legend-rid="lgnd_fig03"><span>Fig. 3<em>I</em></span></a>). These results suggest that mutation of the acidic patch disrupted Nsp1-mediated inhibition of mRNA export.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig03" co-legend-rid="lgnd_fig03"><a href="/pmc/articles/PMC11145185/figure/fig03/" target="figure" rid-figpopup="fig03" rid-ob="ob-fig03"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140263088783904"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11145185_pnas.2314166121fig03.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pnas.2314166121fig03.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11145185/bin/pnas.2314166121fig03.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140263088783904"><a target="object" rel="noopener" href="/pmc/articles/PMC11145185/figure/fig03/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig03"><div><a class="figpopup" href="/pmc/articles/PMC11145185/figure/fig03/" target="figure" rid-figpopup="fig03" rid-ob="ob-fig03">Fig. 3.</a></div><!--caption a7--><div class="caption"><p>Nsp1 N-terminal mutant D33K/E36K/E37K/E41K can no longer inhibit nuclear export of cellular poly(A) RNA. (<em>A</em>) Vero cells were non-transfected or transfected with wild-type<sup>3xFLAG</sup>Nsp1. Nsp1 expression was detected with antibody against the 3xFlag peptide (green) and the intracellular distribution of poly(A) RNA was detected by RNA-FISH using oligo-d(T) probes (red). The same cells were also stained with Hoechst 33258 to visualize nuclei (blue). (Scale bar: 10 &#x000b5;m). Images in the second row correspond to a high magnification of the area surrounded by dotted lines. (Scale bar: 3 &#x000b5;m). (<em>B</em>) Vero cells were non-transfected or transfected with mutant <sup>3xFLAG</sup>Nsp1<sup>D33K/E36K/E37K/E41K</sup>. Cells were processed as in (<em>A</em>). (Scale bar: 10 &#x000b5;m.) Second row images correspond to a high magnification of the area surrounded by dotted lines. (Scale bar: 3 &#x000b5;m). Scatter plot representations of absolute fluorescence intensity (arbitrary fluorescence units, AFU) of poly(A) RNA in whole cell (total) (<em>C</em>), nucleus (<em>D</em>), and cytoplasm (<em>E</em>) are shown. Each dot represents a cell. The dashed lines represent quartiles, and the white line represents median value. (<em>F</em> and <em>G</em>) Relative fluorescence intensity of whole cell (total) cellular poly(A) RNA was quantified in individual cells (<em>F</em>) or nuclear to cytoplasmic ratios (N/C) of absolute poly(A) RNA signal (<em>G</em>). Data represent three independent experiments. Control, <em>n</em> = 54 cells; Nsp1-transfected cells, <em>n</em> = 60 cells; Nsp1<sup>D33K/E36K/E37K/E41K</sup> transfected cells <em>n</em> = 53 cells. Image quantification shows the fluorescence intensity of poly(A) RNA from rendered images. Statistical analysis was performed using one&#x02010;way ANOVA with a Tukey post test and <em>P</em> values are depicted in the figure. (<em>H</em>) Scatter plot representations of single cell absolute 488 fluorescence intensity of Nsp1<sup>WT</sup> or Nsp1<sup>D33K/E36K/E37K/E41K</sup> expressed in Vero cells with an overall median 488 fluorescence intensity of 2.40 &#x000d7; 10<sup>9</sup> and 4.66 &#x000d7; 10<sup>9</sup>, respectively. The black lines represent SEM. Statistical analysis was performed using the two-tailed <em>t</em> test and <em>P</em> value is depicted in the figure. (<em>I</em>) Scatter plot showing two dimensions of single infected cells N/C ratios of poly(A) RNA signal versus Nsp1 protein levels (AFU). Clusters of Nsp1<sup>WT</sup> or Nsp1<sup>D33K/E36K/E37K/E41K</sup> transfected cells are circled by red or blue dashed lines, respectively. A cell (red) expressing low level of Nsp1<sup>WT</sup> (2.22 &#x000d7; 10<sup>9</sup> AFU) and high N/C ratio of poly(A) RNA (N/C = 7.83) is compared to another cell (blue) expressing high levels of Nsp1<sup>D33K/E36K/E37K/E41K</sup> (14.4 &#x000d7; 10<sup>9</sup> AFU) and low N/C ratio of poly(A) RNA (N/C = 1.12).</p></div></div></div><p>We then assessed translation in the absence or presence of wild-type Nsp1 or mutant Nsp1 proteins. Cells were cotransfected with capped firefly luciferase reporter mRNA and plasmids encoding wild-type Nsp1, or Nsp1 mutant in the acidic patch, or Nsp1 C-terminal mutant (K164A/H165A). The latter is known to abolish Nsp1-mediated translation inhibition or mRNA degradation (<a href="#r20" rid="r20" class=" bibr popnode">20</a>). While the Nsp1 C-terminal mutant (K164A/H165A) was unable to inhibit translation, as expected, the Nsp1 mutant on the N-terminal acidic patch inhibited translation at similar levels as the wild-type Nsp1 protein (<a href="/pmc/articles/PMC11145185/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>A</em></span></a>). Notably, in agreement with the release of the mRNA export block, when expressed from a plasmid, the mutant Nsp1 protein is expressed at higher levels than the wild-type protein, indicating that Nsp1 inhibits its own synthesis (<a href="/pmc/articles/PMC11145185/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>A</em></span></a>). To measure translation in the presence of equal amounts of Nsp1, we then performed a cell-free in vitro translation assay with reticulocyte lysate using purified Nsp1<sup>WT</sup> or Nsp1<sup>D33K/E36K/E37K/E41K</sup> proteins. We found that wild-type and mutant Nsp1 proteins inhibit translation of the luciferase reporter mRNA to the same extent in this system (<a href="/pmc/articles/PMC11145185/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>B</em></span></a>). Additionally, we performed the fluorescent noncanonical amino acid tagging (FUNCAT) assay to assess the effect of wild-type or mutant Nsp1 proteins on translation activity in situ. A549 cells were transfected with plasmids encoding Nsp1<sup>WT</sup>, Nsp1<sup>D33K/E36K/E37K/E41K</sup>, or Nsp1<sup>K164A/H165A</sup>. Cells were pulse-labeled with 0.5 mM HPG for 30 min and chased for 30 min followed by Click-iT&#x02122; HPG Alexa Fluor&#x02122; 594 protein synthesis assay. Nsp1 expression was detected with antibody against the 3xFlag peptide. We found that both Nsp1<sup>WT</sup> and Nsp1<sup>D33K/E36K/E37K/E41K</sup> inhibited nascent cellular protein synthesis to a similar extent whereas the mutant defective in ribosome binding, Nsp1<sup>K164A/H165A</sup>, showed substantial translation activity (<a href="/pmc/articles/PMC11145185/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4 <em>C</em> and <em>D</em></span></a>). We have also compared absolute fluorescence intensity of newly synthesized proteins with the absolute fluorescence intensity of cytoplasmic poly(A) RNA in cells expressing Nsp1<sup>WT</sup> and Nsp1<sup>D33K/E36K/E37K/E41K</sup> and found that both robustly inhibited translation even though Nsp1<sup>D33K/E36K/E37K/E41K</sup> had higher levels of cytoplasmic poly(A) RNA (<a href="/pmc/articles/PMC11145185/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4 <em>E</em> and <em>F</em></span></a>). Taken together, these different approaches corroborate the translation inhibitory activity of Nsp1<sup>D33K/E36K/E37K/E41K</sup>. Thus, this acidic patch Nsp1 mutant is 1) unable to properly interact with the NXF1-NXT1 heterodimer, 2) is also unable to inhibit host mRNA export; however, 3) this Nsp1 mutant maintains its translation inhibitory effect. Together, these results show the identification of a separation-of-function mutation on SARS-CoV-2 Nsp1 protein that can distinguish its effects on translation versus its role in mRNA export.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig04" co-legend-rid="lgnd_fig04"><a href="/pmc/articles/PMC11145185/figure/fig04/" target="figure" rid-figpopup="fig04" rid-ob="ob-fig04"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140263090899488"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11145185_pnas.2314166121fig04.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pnas.2314166121fig04.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11145185/bin/pnas.2314166121fig04.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140263090899488"><a target="object" rel="noopener" href="/pmc/articles/PMC11145185/figure/fig04/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig04"><div><a class="figpopup" href="/pmc/articles/PMC11145185/figure/fig04/" target="figure" rid-figpopup="fig04" rid-ob="ob-fig04">Fig. 4.</a></div><!--caption a7--><div class="caption"><p>SARS-CoV-2 Nsp1<sup>D33K/E36K/E37K/E41K</sup> maintains the ability to inhibit translation. (<em>A</em>) HEK293T cells were transfected with pCl-neo vector or the indicated pCl-neo SARS-CoV-2 Nsp1 expressing plasmids. Firefly luciferase (Fluc) mRNA was transfected 24 h after DNA transfection. Cells were lysed 10 h after mRNA transfection and luciferase activity was measured. Normalized Nsp1 wildtype or mutant luciferase values were compared to empty vector using a one-way ANOVA with multiple comparisons using Tukey&#x02019;s correction; <em>P</em> values are shown. Cell lysates from the translation assay above were analyzed by western blot to detect Nsp1 (IB: Flag). Tubulin was used as loading control. (<em>B</em>) Cell-free in vitro translation assay in rabbit reticulocytes was performed with a capped Fluc reporter mRNA in the presence of GST, Nsp1<sup>WT</sup>, or Nsp1<sup>D33K/E36K/E37K/E41K</sup> proteins (1 &#x003bc;M). Control reactions were performed in the absence of Nsp1 (Ctrl) with and without capped Fluc mRNA, as well as in the presence of GST (1 &#x003bc;M). To determine statistical significance, one-way ANOVA with multiple comparisons using Dunnett&#x02019;s correction was performed; p-values are shown. Coomassie-stained SDS-PAGE of GST, Nsp1<sup>WT</sup>, and Nsp1<sup>D33K/E36K/E37K/E41K</sup> proteins at equal molar amount. (<em>C</em>) Nascent protein synthesis was monitored using Click-iT HPG Alexa 594 in A549 cells transfected with plasmids encoding Nsp1<sup>WT</sup>, Nsp1<sup>D33K/E36K/E37K/E41K</sup>, or Nsp1<sup>K164A/H165A</sup>. Cells were pulse-labeled with 0.5 mM HPG for 30 min and chased for 30 min followed by Click-iT&#x02122; HPG Alexa Fluor&#x02122; 594 protein synthesis assay. Nsp1 expression was detected with antibody against the 3xFlag peptide (green) and Hoechst 33258 was used to visualize nuclei (blue). Transfected cells are marked by dotted lines. <em>Inset</em> images in the second column are high-contrast overexposed representation of the corresponding Click-IT HPG Alexa 954 images of Nsp1<sup>WT</sup> or Nsp1<sup>D33K/E36K/E37K/E41K</sup> transfected cells to show the low levels of translation activity in these Nsp1 transfected cells. (Scale bar: 10 &#x003bc;m). (<em>D</em>) Relative fluorescence intensity of nascent cellular protein synthesis. Single-cell values are shown as open-circle. Bar graphs represent mean values &#x000b1; SD. (<em>E</em>) Absolute fluorescence intensity (arbitrary fluorescence units, AFU) of global proteins synthesis. Each dot shows single-cell measurements. Bar graphs represent mean values &#x000b1; SD. (<em>F</em>) Absolute cytoplasmic poly(A) RNA fluorescence intensity shown in <a href="/pmc/articles/PMC11145185/figure/fig03/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig03" rid-ob="ob-fig03" co-legend-rid="lgnd_fig03"><span>Fig. 3<em>F</em></span></a>. Each dot represents single-cell quantification. Bar graphs represent mean values &#x000b1; SD. Data represent three independent experiments. Control, <em>n</em> = 50 cells; Nsp1<sup>WT</sup> transfected cells, <em>n</em> = 50 cells; Nsp1<sup>D33K/E36K/E37K/E41K</sup> transfected cells, <em>n</em> = 50 cells; Nsp1<sup>K164A/H165A</sup> transfected cells <em>n</em> = 39 cells.</p></div></div></div><p>To then assess the importance of the Nsp1 N-terminal acidic patch on its function as inhibitor of host mRNA export during infection, we generated a recombinant (r)SARS-CoV-2 containing the D33K/E36K/E37K/E41K mutation in Nsp1. rSARS-CoV-2 Nsp1<sup>D33K/E36K/E37K/E41K</sup> was rescued using a bacterial artificial chromosome (BAC)-based reverse genetics system as previously described (<a href="#r21" rid="r21" class=" bibr popnode">21</a>). The presence of the Nsp1<sup>D33K/E36K/E37K/E41K</sup> mutation was confirmed by genome sequencing of the viral stock (<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2314166121#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S2</a>). We then mock-infected or infected cells with rSARS-CoV-2 Nsp1<sup>WT</sup> or rSARS-CoV-2 Nsp1<sup>D33K/E36K/E37K/E41K</sup> and subjected the samples to proximity ligation assays (PLAs) to determine Nsp1-NXF1 interaction and their intracellular localization in intact infected cells. In this assay, antibodies that recognize Nsp1 and NXF1 are bound by secondary antibodies linked to oligos that prime a DNA amplification reaction, which is then recognized by fluorescent oligos. The PLA positive signal depicting the interaction between Nsp1 and NXF1 is shown by red dots (<a href="/pmc/articles/PMC11145185/figure/fig05/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig05" rid-ob="ob-fig05" co-legend-rid="lgnd_fig05"><span>Fig. 5<em>A</em></span></a>). In mock infected cells, only a small number of background dots were detected, demonstrating specificity of the reaction in the infected cells (<a href="/pmc/articles/PMC11145185/figure/fig05/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig05" rid-ob="ob-fig05" co-legend-rid="lgnd_fig05"><span>Fig. 5 <em>A</em> and <em>B</em></span></a>). In cells infected with the wild-type virus, quantification of interactions and their intracellular localization shows that the Nsp1-NXF1 interaction occurs mostly in the cytoplasm but a small pool is also found in the nucleus (<a href="/pmc/articles/PMC11145185/figure/fig05/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig05" rid-ob="ob-fig05" co-legend-rid="lgnd_fig05"><span>Fig. 5 <em>A</em> and <em>B</em></span></a>). In contrast, the interaction between Nsp1<sup>D33K/E36K/E37K/E41K</sup> and NXF1 was overall reduced in cells infected with this mutant virus, regardless of Nsp1 mutant protein levels (<a href="/pmc/articles/PMC11145185/figure/fig05/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig05" rid-ob="ob-fig05" co-legend-rid="lgnd_fig05"><span>Fig. 5 <em>C</em> and <em>D</em></span></a>). This decrease in binding corroborates the in vitro binding assays shown in <a href="/pmc/articles/PMC11145185/figure/fig01/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig01" rid-ob="ob-fig01" co-legend-rid="lgnd_fig01"><span>Fig. 1</span></a>. Taken together, these findings indicate that Nsp1 may alter the trafficking of NXF1 into and/or out of the nucleus, which is consistent with Nsp1-mediated inhibition of nuclear export of host mRNAs, as we and others have previously shown (<a href="#r1" rid="r1" class=" bibr popnode">1</a>, <a href="#r3" rid="r3" class=" bibr popnode">3</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig05" co-legend-rid="lgnd_fig05"><a href="/pmc/articles/PMC11145185/figure/fig05/" target="figure" rid-figpopup="fig05" rid-ob="ob-fig05"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140263088912000"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11145185_pnas.2314166121fig05.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pnas.2314166121fig05.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11145185/bin/pnas.2314166121fig05.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140263088912000"><a target="object" rel="noopener" href="/pmc/articles/PMC11145185/figure/fig05/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig05"><div><a class="figpopup" href="/pmc/articles/PMC11145185/figure/fig05/" target="figure" rid-figpopup="fig05" rid-ob="ob-fig05">Fig. 5.</a></div><!--caption a7--><div class="caption"><p>Nsp1-NXF1 interaction is mostly cytoplasmic in rSARS-CoV2<sup>WT</sup> infected cells and is decreased in rSARS-CoV2<sup>D33K/E36K/E37K/E41K</sup> infected cells. (<em>A</em>) A549-ACE2 cells were mock infected or infected with rSARS-CoV2<sup>WT</sup> or rSARS-CoV2<sup>D33K/E36K/E37K/E41K</sup>. After 12 h, cells were subjected to PLA to detect the interaction between Nsp1 and NXF1 proteins in situ. The interaction by PLA is detected by fluorescent probes (red dots; &#x003bb;em = 624 nm, TRITC filter). Immunofluorescence staining of Nsp1 is shown in green. Hoechst staining labels the nuclei (blue). (Scale bar: 3 &#x003bc;m). The panels on the <em>Right</em> are three-dimensional projections of chromatin merged surface with the PLA signals detecting the NXF1&#x02013;Nsp1 complexes. Cell boundaries are marked by green-dotted lines. (Scale bar: 2 &#x003bc;m). (<em>B</em>) Quantification of PLA signals per cell from 34 cells. The dashed lines represent quartiles and the white line represents median value. Data are representative of three independent experiments. Mean cytoplasmic/nuclear PLA dots per cell are mock: 0.4956/0.8169; rSARS-CoV-2<sup>WT</sup>: 41.29/13.78; rSARS-CoV-2<sup>D33K/E36K/E37K/E41K</sup>: 6.655/2.466. Statistical analysis was performed using one&#x02010;way ANOVA with a Tukey post test and <em>P</em> values are depicted in the figure. (<em>C</em>) Scatter plot representations of single cell absolute 488 fluorescence intensity of Nsp1 protein levels in infected cells. The black lines represent SEM. Single infected cells expressing similar levels of Nsp1 protein (Nsp1<sup>WT</sup>= 1.68 &#x000d7; 10<sup>8</sup> and Nsp1<sup>D33K/E36K/E37K/E41K</sup>= 1.57 &#x000d7; 10<sup>8</sup>) are shown in the yellow-filled gray brackets with dashed lines projecting on the right to depict their corresponding PLA images (24 PLA dots for rSARS-CoV-2 Nsp1<sup>WT</sup> infected cell and 1 PLA dot for rSARS-CoV-2 Nsp1<sup>D33K/E36K/E37K/E41K</sup> infected cell). Statistical analysis was performed using the two-tailed <em>t</em> test and <em>P</em> value is depicted in the figure. (<em>D</em>) Scatter plot shows two-dimensional analysis of individual cells&#x02014;cytoplasmic PLA spots versus Nsp1 protein level&#x02014;from rSARS-CoV-2 Nsp1<sup>WT</sup> or rSARS-CoV-2 Nsp1<sup>D33K/E36K/E37K/E41K</sup> infected cells. Cells in the yellow-filled gray brackets are the same cells selected in (<em>C</em>).</p></div></div></div><p class="p">Since Nsp1<sup>D33K/E36K/E37K/E41K</sup> binding to NXF1 is reduced compared to Nsp1 wild-type, we then tested whether these mutations would diminish the inhibitory function of Nsp1 on nuclear export of cellular mRNAs. We performed RNA FISH to detect the intracellular localization of specific cellular mRNAs (ATF3, CXCL3, NFKB1, and NUAK2) previously shown to be retained in the nucleus upon SARS-CoV-2 infection or Nsp1 transfection (<a href="#r1" rid="r1" class=" bibr popnode">1</a>). These mRNAs are present at almost undetectable levels in uninfected cells and once they are transcriptionally induced in the nucleus upon infection, they face the mRNA nuclear export inhibition by Nsp1. While we found that the mRNAs encoding ATF3, CXCL3, and NFKB1 were up-regulated upon rSARS-CoV-2 Nsp1<sup>WT</sup> infection but were retained inside the nucleus, cells infected with rSARS-CoV-2 Nsp1<sup>D33K/E36K/E37K/E41K</sup> showed nuclear export of these messages, indicating reversal of mRNA export inhibition by these mutations (<a href="/pmc/articles/PMC11145185/figure/fig06/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig06" rid-ob="ob-fig06" co-legend-rid="lgnd_fig06"><span>Fig. 6 <em>A&#x02013;</em><em>C</em></span></a> and <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2314166121#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S3</a>). In the case of NUAK2 mRNA, there is some degree of mRNA retention in the nucleus upon infection with the wild-type virus, which is not significantly different from the infection with the Nsp1 mutant virus, as shown by the intracellular distribution of mRNAs at the single-cell level (<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2314166121#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S3 <em>D</em> and <em>K</em>&#x02013;<em>M</em></a>). These results suggest differential mRNA export regulation, which is observed in various cellular conditions as diverse mRNAs interact with specific RNA binding proteins and/or mRNA export complexes (<a href="#r22" rid="r22" class=" bibr popnode">22</a>). Taken together, these findings underscore the importance of the N-terminal acidic patch of Nsp1 and its interaction with NXF1 on inhibition of mRNA nuclear export by SARS-CoV-2.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig06" co-legend-rid="lgnd_fig06"><a href="/pmc/articles/PMC11145185/figure/fig06/" target="figure" rid-figpopup="fig06" rid-ob="ob-fig06"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140263084180976"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11145185_pnas.2314166121fig06.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pnas.2314166121fig06.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11145185/bin/pnas.2314166121fig06.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140263084180976"><a target="object" rel="noopener" href="/pmc/articles/PMC11145185/figure/fig06/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig06"><div><a class="figpopup" href="/pmc/articles/PMC11145185/figure/fig06/" target="figure" rid-figpopup="fig06" rid-ob="ob-fig06">Fig. 6.</a></div><!--caption a7--><div class="caption"><p>SARS-CoV-2 Nsp1<sup>D33K/E36K/E37K/E41K</sup> is unable to inhibit cellular mRNA export. A549-ACE2 cells were infected with rSARS-CoV-2 Nsp1<sup>WT</sup> or rSARS-CoV-2 Nsp1<sup>D33K/E36K/E37K/E41K</sup> at MOI 0.25 for 24 h. Cells were subjected to the ViewRNA Cell Plus Assay to detect specific cellular mRNAs, as depicted in the figure, and immunofluorescence microscopy to detect viral N protein. (<em>A</em>) ATF3 mRNA is detected in red, viral N protein is shown in green, and nuclei are stained in blue with DAPI. The dotted square shows a selected cell that is highlighted in the <em>Middle</em> and <em>Bottom</em> panels. Images and quantification of other mRNAs are shown in <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2314166121#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S3</a>. (<em>B</em>) Scatter plot representations of single cell ATF3 mRNA fluorescence signal in whole cells (total), or in the nucleus, or in the cytoplasm of mock, rSARS-CoV-2 Nsp1<sup>WT</sup>, or rSARS-CoV-2 Nsp1<sup>D33K/E36K/E37K/E41K</sup>-infected cells. Each dot represents a cell. The dashed lines represent quartiles and the white line represents median value. (<em>C</em>) The calculated percent nuclear ATF3 mRNA signal is presented as a scattered plot for each individual cell (mock n= 25 cells; wild-type virus n= 51 cells; mutant virus n= 51 cells). Each dot represents a cell and the median value is depicted in the white line. Statistical analysis was performed using one-way ANOVA with a Tukey post test and <em>P</em> values are depicted in the figure.</p></div></div></div></div><div id="s5" class="sec sec-last"><p></p><h3 id="s5title" class="inline">SARS-CoV-2 Nsp1<sup>D33K/E36K/E37K/E41K</sup> Is Attenuated at the Cellular and Animal Levels. </h3><p class="p p-first">We next compared viral replication between the mutant SARS-CoV-2 Nsp1<sup>D33K/E36K/E37K/E41K</sup> and SARS-CoV-2 Nsp1<sup>WT</sup>. A549-ACE2 cells were infected with wild-type or mutant viruses for 12 h, 24 h, 36 h, and 48 h, and viral replication was determined by plaque assay. Replication of the mutant virus is severely compromised, as shown by its lower viral titers compared to wild-type rSARS-CoV-2 (<a href="/pmc/articles/PMC11145185/figure/fig07/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig07" rid-ob="ob-fig07" co-legend-rid="lgnd_fig07"><span>Fig. 7<em>A</em></span></a>). Cell lysates were then subjected to Western blot analysis to detect the viral nucleocapsid (N) protein and Nsp1. The mutant virus is clearly attenuated, showing low levels of N and Nsp1 proteins in infected cells (<a href="/pmc/articles/PMC11145185/figure/fig07/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig07" rid-ob="ob-fig07" co-legend-rid="lgnd_fig07"><span>Fig. 7<em>B</em></span></a>). This is further corroborated by following infection of both wild-type and mutant viruses by immunofluorescence microscopy in which infected cells were stained with antibodies against the N protein (<a href="/pmc/articles/PMC11145185/figure/fig07/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig07" rid-ob="ob-fig07" co-legend-rid="lgnd_fig07"><span>Fig. 7<em>C</em></span></a>). Again, cells infected with the mutant virus show substantially less N staining than cells infected with wild-type SARS-CoV-2.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig07" co-legend-rid="lgnd_fig07"><a href="/pmc/articles/PMC11145185/figure/fig07/" target="figure" rid-figpopup="fig07" rid-ob="ob-fig07"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140263071544608"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11145185_pnas.2314166121fig07.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pnas.2314166121fig07.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11145185/bin/pnas.2314166121fig07.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140263071544608"><a target="object" rel="noopener" href="/pmc/articles/PMC11145185/figure/fig07/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig07"><div><a class="figpopup" href="/pmc/articles/PMC11145185/figure/fig07/" target="figure" rid-figpopup="fig07" rid-ob="ob-fig07">Fig. 7.</a></div><!--caption a7--><div class="caption"><p>rSARS-CoV-2 Nsp1<sup>D33K/E36K/E37K/E41K</sup> is attenuated in A549-ACE2 cells. (<em>A</em>) Growth curves of rSARS-CoV-2 Nsp1<sup>WT</sup> and rSARS-CoV-2 Nsp1<sup>D33K/E36K/E37K/E41K</sup> in A549-ACE2 cells infected at an MOI of 0.1 for 12, 24, 36, and 48 hpi. Titers were quantified by plaque assay (<em>n</em> = 3). The dashed line indicates the limit of detection for plaque assay (50 PFU/mL). Values were log transformed and compared using multiple unpaired <em>t</em> tests; <em>P</em> values are shown. (<em>B</em>) A549-ACE2 cells were mock-infected or infected at an MOI of 0.1 and samples were collected at 12, 24, 36, and 48 hpi. Cell lysates were analyzed by Western blot to assess SARS-CoV-2 N and Nsp1 expression. GAPDH was used as loading control. (<em>C</em>) A549-ACE2 cells were mock-infected or infected with the indicated viruses at an MOI of 0.1 for 12, 24, 36, and 48 h. Cells were subjected to immunofluorescence microscopy to detect SARS-CoV-2 N protein (green). Nuclei were stained with DAPI (blue). (Scale bar: 300 &#x003bc;m).</p></div></div></div><p class="p">To then evaluate the contribution of Nsp1-mediated mRNA export block to the pathogenesis of SARS-CoV-2, 8-wk-old female transgenic mice expressing human ACE2 under the human keratin 18 promoter (K18-hACE2) were intranasally mock-infected or infected with 1 &#x000d7; 10<sup>4</sup> PFU of either parental rSARS-CoV-2 Nsp1<sup>WT</sup> or rSARS-CoV-2 Nsp1<sup>D33K/E36K/E37K/E41K</sup>. Animals from each group were killed at 4 and 7 days postinfection (dpi) to determine lungs and nasal turbinate titers. In addition, body weight was recorded for 10 dpi to monitor morbidity. As shown in <a href="/pmc/articles/PMC11145185/figure/fig08/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig08" rid-ob="ob-fig08" co-legend-rid="lgnd_fig08"><span>Fig. 8</span></a>, we observed that animals infected with rSARS-CoV-2 NSP1<sup>D33K/E36K/E37K/E41K</sup> exhibited reduced body weight loss compared to those infected with the wild-type virus (<a href="/pmc/articles/PMC11145185/figure/fig08/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig08" rid-ob="ob-fig08" co-legend-rid="lgnd_fig08"><span>Fig. 8 <em>A</em> and <em>B</em></span></a>). Additionally, we observed significantly lower viral load in the lungs and nasal turbinate of rSARS-CoV-2 NSP1<sup>D33K/E36K/E37K/E41K</sup> -infected animals in comparison to those infected with wild-type virus at both 4 and 7 dpi (<a href="/pmc/articles/PMC11145185/figure/fig08/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig08" rid-ob="ob-fig08" co-legend-rid="lgnd_fig08"><span>Fig. 8 <em>C</em> and <em>D</em></span></a>). Notably, all rSARS-CoV-2 NSP1<sup>D33K/E36K/E37K/E41K</sup> -infected animals had undetectable levels of infectious virus in the nasal turbinate at both 4 and 7 dpi (<a href="/pmc/articles/PMC11145185/figure/fig08/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig08" rid-ob="ob-fig08" co-legend-rid="lgnd_fig08"><span>Fig. 8<em>D</em></span></a>). Immunohistochemistry (IHC) also showed reduced SARS-CoV-2 N antigen load in the lungs of K18-hACE2 mice infected with rSARS-CoV-2 Nsp1<sup>D33K/E36K/E37K/E41K</sup> compared to rSARS-CoV-2<sup>WT</sup> (<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2314166121#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S4</a>). All mock-infected mice examined at 4 and 7 dpi were histologically within normal limits with no apparent immune cell infiltrates. K18-hACE2 mice infected with both wild-type and mutant viruses displayed similar lesions but exhibited variable severity ranging from minimal to regionally moderate disease, with the latter being more common in the mice infected with the wild-type virus. Histologic findings consisted of bronchointerstitial and perivascular mononuclear infiltrates and endothelial cell hypertrophy (<a href="/pmc/articles/PMC11145185/figure/fig08/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig08" rid-ob="ob-fig08" co-legend-rid="lgnd_fig08"><span>Fig. 8<em>E</em></span></a>). Inflammatory cell infiltrates in each anatomical compartment (peribronchiolar, perivascular, and interstitium) as well as overall percentage of pulmonary parenchyma affected were scored using an ordinal scoring system. The most consistent differences occurred in the interstitial and perivascular compartments, which show less inflammation and infiltrate in mice infected with the mutant virus than in mice infected with the wild-type virus, while the peribronchiolar mononuclear infiltrates were routinely minimal to mild in K18-hACE2 mice infected with both viruses (<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2314166121#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S5 and Table S3</a>). In sum, these findings underscore the importance of the interaction between Nsp1 and NXF1 in the inhibition of host gene expression via mRNA export inhibition and show its impact in promoting viral replication and pathogenesis.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig08" co-legend-rid="lgnd_fig08"><a href="/pmc/articles/PMC11145185/figure/fig08/" target="figure" rid-figpopup="fig08" rid-ob="ob-fig08"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140263080177232"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11145185_pnas.2314166121fig08.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pnas.2314166121fig08.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11145185/bin/pnas.2314166121fig08.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140263080177232"><a target="object" rel="noopener" href="/pmc/articles/PMC11145185/figure/fig08/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig08"><div><a class="figpopup" href="/pmc/articles/PMC11145185/figure/fig08/" target="figure" rid-figpopup="fig08" rid-ob="ob-fig08">Fig. 8.</a></div><!--caption a7--><div class="caption"><p>rSARS-CoV-2 Nsp1<sup>D33K/E36K/E37K/E41K</sup> infected K18-hACE2 mice show less severe lung histopathology. (<em>A</em>) Schematic of in vivo experimental design where K18-hACE2 mice were mock-infected (n = 7 at 4 dpi and n = 4 at 7 dpi) or infected with 1 &#x000d7; 10<sup>4</sup> PFU of rSARS-CoV-2 Nsp1<sup>WT</sup> (n = 7 at 4 dpi and n = 4 at 7 dpi) or rSARS-CoV-2 Nsp1<sup>D33K/E36K/E37K/E41K</sup> (n = 5 at 4 dpi and n = 4 at 7 dpi). (<em>B</em>) Weight loss curve depicting body weights for mock-infected or infected with rSARS-CoV-2 Nsp1<sup>WT</sup> or rSARS-CoV-2 Nsp1<sup>D33K/E36K/E37K/E41K</sup> for the duration of the experiment as a percentage of initial bodyweight. Weight loss data are shown as mean &#x000b1; SEM. (<em>C</em>) Lung viral titers for mock-infected and infected animals at the indicated dpi. (<em>D</em>) Nasal turbinate viral titers for mock-infected and infected animals at the indicated dpi. Dashed lines in (<em>C</em>) and (<em>D</em>) indicate the limit of detection for plaque assay (50 PFU/mL). Values were log transformed and compared using two-way ANOVA with Tukey&#x02019;s correction; <em>P</em> values are shown. (<em>E</em>) Representative images of lung H&#x00026;E staining for all three groups of animals at the indicated dpi. All images were acquired at 200&#x000d7; total magnification. Mock inoculated lungs were within normal limits, while both infected lung samples displayed variable perivascular (blue hashed ovals), peribronchiolar (black hashed ovals), and interstitial mononuclear infiltrates (orange arrowheads). Endothelial hypertrophy was also observed in both infected cohorts. (Scale bars in histology slides = 100 &#x003bc;m).</p></div></div></div></div></div><div id="s6" class="tsec sec"><h2 class="head no_bottom_margin" id="s6title" style="text-transform: uppercase;">Discussion</h2><p class="p p-first">Nsp1 is a major virulence factor of SARS-CoV-1 (<a href="#r20" rid="r20" class=" bibr popnode">20</a>, <a href="#r23" rid="r23" class=" bibr popnode">23</a>, <a href="#r24" rid="r24" class=" bibr popnode">24</a>) and SARS-CoV-2 (<a href="#r1" rid="r1" class=" bibr popnode">1</a>, <a href="#r3" rid="r3" class=" bibr popnode">3</a>, <a href="#r25" rid="r25" class=" bibr popnode">25</a>, <a href="#r26" rid="r26" class=" bibr popnode">26</a>), functioning as inhibitor of host gene expression through multiple mechanisms. Among the key pathways targeted by SARS-CoV-2 Nsp1 is mRNA nuclear export, as we and others recently discovered (<a href="#r1" rid="r1" class=" bibr popnode">1</a>, <a href="#r3" rid="r3" class=" bibr popnode">3</a>, <a href="#r8" rid="r8" class=" bibr popnode">8</a>). Despite intensive studies, the underlying mechanisms of the various Nsp1-mediated functions and their contributions to SARS-CoV-2 virulence remain unclear. In this study, we show that Nsp1 uses a distinct mechanism to interact with the cellular mRNA export receptor NXF1. We identified a separation-of-function mutant of Nsp1, which we employed to investigate the role of the Nsp1-NXF1 interaction in SARS-CoV-2 virulence using SARS-CoV-2 infected cells and an in vivo model. Our results provide a mechanistic understanding of the Nsp1-mediated mRNA export inhibition and reveal the critical importance of this mechanism in the virulence of SARS-CoV-2.</p><p>Through structure-guided mutagenesis, we identified an acidic patch (D33, E36, E37, and E41) on the surface of the N-terminal domain of Nsp1 that mediates interaction with NXF1. We employed site-specific photo-crosslinking using <em>p</em>Bpa-incorporated Nsp1 at a site near the acidic patch to reveal that Nsp1 binds to the RRM domain of NXF1. Among the various Nsp1-mediated mechanisms, it is established from early work on SARS-CoV-1 and recent studies on SARS-CoV-2 that Nsp1 binds to the ribosome (<a href="#r2" rid="r2" class=" bibr popnode">2</a>, <a href="#r4" rid="r4" class=" bibr popnode">4</a><a href="#r5" rid="r5" class=" bibr popnode"></a>&#x02013;<a href="#r6" rid="r6" class=" bibr popnode">6</a>, <a href="#r15" rid="r15" class=" bibr popnode">15</a>). The Nsp1-ribosome interaction is responsible for the inhibition of host translation (<a href="#r2" rid="r2" class=" bibr popnode">2</a>, <a href="#r4" rid="r4" class=" bibr popnode">4</a><a href="#r5" rid="r5" class=" bibr popnode"></a>&#x02013;<a href="#r6" rid="r6" class=" bibr popnode">6</a>), and Nsp1-mediated mRNA degradation also involves mRNA interaction with the ribosome (<a href="#r7" rid="r7" class=" bibr popnode">7</a>). However, some mRNAs that encode antiviral factors, such as interferon mRNAs, can evade degradation during SARS-CoV-2 infection (<a href="#r27" rid="r27" class=" bibr popnode">27</a>). Nsp1 primarily targets the ribosome with its C-terminal region, which blocks the mRNA entry site on the 40S ribosomal subunit (<a href="#r2" rid="r2" class=" bibr popnode">2</a>, <a href="#r4" rid="r4" class=" bibr popnode">4</a><a href="#r5" rid="r5" class=" bibr popnode"></a>&#x02013;<a href="#r6" rid="r6" class=" bibr popnode">6</a>). More recent studies show that Nsp1-N also contributes to ribosome binding, involving R99, R124, and K125 (<a href="#r17" rid="r17" class=" bibr popnode">17</a>). We show that mutations of these residues do not affect the Nsp1-NXF1 interaction. Recent studies have shown that Nsp1-N terminal domain dynamically interacts with the 40S ribosomal subunit (<a href="#r28" rid="r28" class=" bibr popnode">28</a>). In addition to the extensively characterized high-affinity Nsp1-C terminal domain interaction with the 40S subunit, the Nsp1-N terminal domain from three different &#x003b2;-CoVs (SARS-CoV-2, MERS-CoV, and Bat-Hp-CoV) also contributes to inhibition of translation. Cryo-EM studies using Bat-Hp-CoV Nsp1 as a model system captured the Nsp1-N terminal domain in the context of the 40S&#x02013;Nsp1 complex. Indeed, the acidic patch mediating NXF1 interaction is not in contact with the 40S subunit in the structure (<a href="#r28" rid="r28" class=" bibr popnode">28</a>). In addition, mutations of Nsp1-N motifs (<sub>98</sub>GRSG<sub>101</sub> and <sub>124</sub>RK<sub>125</sub>) mediating 40S interaction do not interfere with NXF1 binding, as shown in our GST pull-down assays (<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2314166121#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S1</a>, Nsp1 R99A and R124E/K125E). This recent work supports our conclusion that Nsp1-N terminal domain interacts with NXF1 and ribosome via distinct modes and that the acidic patch is a site that can distinguish Nsp1&#x02019;s effects on mRNA export versus translation. In fact, we demonstrate that the acidic patch mutant of Nsp1 (D33K/E36K/E37K/E41K) is important for the Nsp1-NXF1 interaction but inhibits mRNA translation at similar levels as the wild-type Nsp1, therefore it does not alter Nsp1&#x02019;s impact on translation. In sum, we observed that the intact N- and C-terminal domains of Nsp1 are both critical for proper interaction of Nsp1 with NXF1-NXT1 and we were able to identify a four amino acid patch in the N terminus of Nsp1 which once mutated disrupts Nsp1 interaction with NXF1-NXT1 without interfering with Nsp1 effect on translation.</p><p>Our previous findings showed that NXF1 interactions with mRNA export factors, such as Aly/REF, and nucleoporins decreased in the presence of Nsp1, suggesting that docking and translocation of NXF1-bound mRNAs through the NPC were compromised (<a href="#r3" rid="r3" class=" bibr popnode">3</a>). However, most of Nsp1 is cytoplasmic, and only a small pool is found in the nucleus (<a href="#r3" rid="r3" class=" bibr popnode">3</a>), so determining the localization of the Nsp1-NXF1 complex is key to further understand Nsp1 activity. Here, we demonstrated that NXF1-Nsp1 interaction occurs mostly in the cytoplasm of infected cells, but a small pool of NXF1 bound to Nsp1 is also detected in the nucleus. Mutation of the Nsp1 in the N-terminal acidic patch diminished the interaction in both compartments, further corroborating the in vitro binding assays. These results indicate that Nsp1 may impair the dynamics of NXF1 recycling between the nucleus and the cytoplasm and/or prevent proper import of NXF1 into the nucleus after translation. It is possible that Nsp1 may alter the interaction of NXF1 with karyopherins/importins in the cytoplasm leading to inhibition of nuclear import of NXF1, and in the nucleus, Nsp1 may affect NXF1-binding to mRNA export factors, such as Aly/REF as we have previously shown (<a href="#r3" rid="r3" class=" bibr popnode">3</a>). These effects would certainly lead to impaired nuclear export of mRNAs.</p><p>The Nsp1 N-terminal acidic patch mutant shows clear reversal of mRNA export inhibition compared to the wild-type Nsp1 protein without affecting translation inhibition. This separate-of-function mutant also allowed us to assess the impact of Nsp1-mediated mRNA export inhibition on viral replication at the cellular and animal levels. Indeed, Nsp1 inhibition of mRNA export is important for proper SARS-CoV-2 protein expression and replication. These findings are in agreement with our previous report in which increased levels of the mRNA export receptor NXF1 reverted the mRNA export inhibition mediated by Nsp1 and decreased SARS-CoV-2 infection (<a href="#r3" rid="r3" class=" bibr popnode">3</a>). The rescue of mRNA export includes mRNAs that encode proteins such as NFKB1 and ATF3, which are transcription factors that induce expression of genes involved in immunity and other cellular processes (<a href="#r29" rid="r29" class=" bibr popnode">29</a><a href="#r30" rid="r30" class=" bibr popnode"></a>&#x02013;<a href="#r31" rid="r31" class=" bibr popnode">31</a>). While both NFKB1 and ATF3 mRNAs are up-regulated during SARS-CoV-2 infection, they are retained in the nucleus. Upon infection with the mutant virus, these mRNAs are efficiently exported to the cytoplasm and at least a certain amount is translated to induce an antiviral response and the observed decrease in viral replication. Under this condition, the translation machinery is inhibited by Nsp1 but not fully blocked as cells are alive. These results further point to the importance of Nsp1-NXF1 interaction in the inhibition of nuclear export of mRNAs with key antiviral activities.</p><p class="p p-last">Next, we assessed the impact of Nsp1-NXF1 interaction in the pathogenesis of SARS-CoV-2 infection. Our results demonstrated that targeting the mRNA export machinery is important for efficient viral replication in the respiratory tract. In fact, compared to mice infected with rSARS-CoV-2 Nsp1<sup>WT</sup>, mice infected with rSARS-CoV-2 Nsp1<sup>D33K/E36K/E37K/E41K</sup> displayed significantly lower viral titers in the lung and undetectable viral titers in the nasal turbinate, suggesting decreased viral replication and faster clearance of infection in the respiratory tract. Histologically, this was reflected by less severe interstitial and perivascular mononuclear infiltration in the lungs of rSARS-CoV-2 NSP1<sup>D33K/E36K/E37K/E41K</sup> -infected animals. Additionally, a SARS-CoV-2 with a deletion in the Nsp1 C-terminal domain (amino acids 155 to 165), which disrupts translation inhibition, has also been shown to be attenuated (<a href="#r8" rid="r8" class=" bibr popnode">8</a>). Thus, the combinatory effects of Nsp1 on mRNA export and translation yield a robust inhibitory function on host gene expression that down-regulates IFN response and promote viral replication and pathogenesis.</p></div><div id="s7" class="tsec sec"><h2 class="head no_bottom_margin" id="s7title" style="text-transform: uppercase;">Materials and Methods</h2><div id="s8" class="sec sec-first"><p></p><h3 id="s8title" class="inline">Cell Culture, Plasmids, and Antibodies. </h3><p class="p p-first-last">Vero cells from the American Type Culture Collection were cultured in DMEM (Sigma-Aldrich) supplemented with 10% (v/v) fetal bovine serum (FBS; Sigma-Aldrich) and 1% antibiotic&#x02013;antimycotic (Life Technologies, Thermo Fisher Scientific) at 37 &#x000b0;C in 5% CO2. Cells were plated at 5 &#x000d7; 10<sup>5</sup> density per well (containing a sterile No. 1.5 glass coverslip) of a 24-well culture plate (Phenix Research Products). Transfections were performed according to the manufacturer&#x02019;s instructions (Lipofectamine 3000, Life Technologies, Thermo Fisher Scientific). For all transient transfections, cells were subjected to RNA FISH-Immunofluorescence 16 h post&#x02010;transfection. Vero E6 cells expressing ACE2 and TMPRSS2 (Vero AT) were obtained from BEI Resources (NR-54970) and maintained in Dulbecco&#x02019;s modified Eagle medium (DMEM) supplemented with 10% (v/v) FBS (VWR), 1% penicillin-streptomycin (Corning), and 10 &#x000b5;g/mL puromycin (ThermoFisher Scientific). HEK293T and A549-ACE2 were maintained in Dulbecco&#x02019;s modified Eagle&#x02019;s medium (Corning) supplemented with 10% (v/v) FBS (Peak Serum) and penicillin/streptomycin (Corning) at 37 &#x000b0;C and 5% CO<sub>2</sub>. All cell lines were regularly screened for <em>Mycoplasma</em> contamination using the Universal <em>Mycoplasma</em> Detection Kit (ATCC, 30-1012K) and tested negative. Plasmids and antibodies are described in <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2314166121#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em></a>.</p></div><div id="s9" class="sec"><p></p><h3 id="s9title" class="inline">Protein Purification. </h3><p class="p p-first-last">GST-NXF1<sup>117&#x02013;619</sup>-NXT1 wild-type and mutants were coexpressed in <em>E. coli</em> Rosetta cells (EMD Millipore, 70954). Protein expression and purification are described in <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2314166121#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em></a>.</p></div><div id="s10" class="sec"><p></p><h3 id="s10title" class="inline">GST-Pulldown Assay. </h3><p class="p p-first-last">Ten micromolar GST or GST-Nsp1 or GST-Nsp1 variants were mixed with 6 &#x000b5;M NXF1<sup>117&#x02013;619</sup>-NXT1 wild type or mutant and incubated with Glutathione Sepharose resins in a binding buffer containing 20 mM HEPES pH 7.0, 100 mM NaCl, and 0.5 mM TCEP at room temperature for 40 min. Resins were washed with 600 &#x000b5;L binding buffer for three times. Proteins remaining on the resin were extracted with an elution buffer containing 20 mM HEPES pH 7.0, 300 mM NaCl, 0.5 mM TCEP, and 20 mM reduced glutathione and resolved on a 4 to 12% SDS-PAGE gel. All experiments were repeated three times independently.</p></div><div id="s11" class="sec"><p></p><h3 id="s11title" class="inline">Photo-Crosslinking of NXF1-NXT1 with Nsp1<em><sup>p</sup></em><sup>Bpa45</sup> and LC&#x02013;MS/MS Analysis. </h3><p class="p p-first">Ten micromolar NXF1-NXT1 was incubated with 10 &#x000b5;M Nsp1<em><sup>p</sup></em><sup>Bpa45</sup> in a buffer containing 20 mM HEPES pH 7.0, 100 mM NaCl, and 0.5 mM TCEP on ice for 30 min. Protein mixture was irradiated at 365 nm (Spectrolinker; 120V, 60 HZ, 3A) on ice in a 96-well plate (Thermo Fisher Scientific) for 25 min. Samples were resolved on a 15% SDS-PAGE gel.</p><p class="p p-last">SDS-PAGE gel bands corresponding to the crosslinked Nsp1<em><sup>p</sup></em><sup>Bpa45</sup>-NXF1 complex along with each of the individual uncrosslinked proteins were excised and subjected to in-gel digestion using trypsin. The resulting peptides were analyzed by LC&#x02013;MS/MS utilizing a QExactive-plus mass spectrometer (ThermoFisher) directly coupled to a nano-flow HPLC (U3000, ThermoFisher) using a self-packed 20 cm &#x000d7; 100 micron C18 column. They were resolved using a 90 min aqueous to organic gradient and MS/MS spectra were collected in a data-dependent manner. Resulting MS/MS spectra were analyzed using the pLink software suite (<a href="#r32" rid="r32" class=" bibr popnode">32</a>) using default parameters with the exception that minimum peptide length was set to 4 to accommodate the shorter <em>p</em>BPA-containing peptide. Using a 5% false discovery rate cutoff, a total of 21 interprotein crosslinks were identified. Of these 12 mapped crosslinks to the ISIIINSSAPPHTILNELKPEQVEQLK peptide. Summary of the Nsp1<em><sup>p</sup></em><sup>Bpa45</sup>-NXF1 crosslinks is shown in <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2314166121#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Table S2</a>. Searches of peptides from individual, noncrosslinked NXF1 or Nsp1<em><sup>p</sup></em><sup>Bpa45</sup> yielded none of these crosslinked identifications.</p></div><div id="s12" class="sec"><p></p><h3 id="s12title" class="inline">RNA FISH. </h3><p class="p p-first-last">For detection of poly(A) RNA in cells, oligo-d(T) in situ hybridization was performed as we previously described (<a href="#r33" rid="r33" class=" bibr popnode">33</a>). For detection of specific cellular mRNAs, the ViewRNA Cell Plus Assay Kit was used (Invitrogen, 88-19000-99). Additional experimental details are described in <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2314166121#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em></a>.</p></div><div id="s13" class="sec"><p></p><h3 id="s13title" class="inline">Cell-Based Translation Assay (Including Statistical Analysis and Western Blot Conditions). </h3><p class="p p-first">Fluc mRNA generation: To test the effect of Nsp1 on cellular translation, we expressed FLAG-tagged SARS-CoV-2 Nsp1 wildtype and indicated mutants in HEK293T cells and monitored translation of a cotransfected capped Firefly luciferase (Fluc) reporter mRNA. The Fluc mRNA was generated by in vitro transcription (IVT) using the mMESAGE mMACHINE T7 Transcription Kit (Invitrogen) and purified with the MEGAclear kit (Invitrogen) according to the manufacturer&#x02019;s protocol. The template for the IVT was generated by PCR from the pCAG-Luc plasmid (Addgene, #55764) using the indicated primer sequences:</p><p>(T7-kozak-Fluc-Fw: 5&#x02032;-TAATACGACTCACTATAGAGCCACCATGGAAGATGCCAAAAA-3&#x02032;; PolyA-Fluc-Rv: 5&#x02032;-TTTTTTTTTTTTTTTTTTTTTTTTTTTTTATTACACGGCGATCTTGCCG- 3&#x02032;). The size and quality of the transcript were analyzed by agarose gel electrophoresis.</p><p class="p p-last">Translation Assay: HEK293T cells were cultured in 24-well plates and transfected with 500 ng pCl-neo vector or pCl-neo SARS-CoV-2 Nsp1 wildtype or mutants. Plasmid DNA was transfected using TransIT-LT1 Transfection Reagent (Mirus) in Opti-MEM Reduced Serum Medium (Gibco). The firefly luciferase (Fluc) mRNA (250 ng) was transfected 24 h after DNA transfection using Lipofectamine MessengerMAX (Invitrogen) in Opti-MEM Reduced Serum Medium (Gibco) according to the manufacturer&#x02019;s recommendations. Prior to mRNA transfection, cells were washed once with fresh Dulbecco&#x02019;s modified Eagle&#x02019;s medium (Corning) supplemented with 10% FBS (Peak Serum) and penicillin/streptomycin (Corning). At 10 h after mRNA transfection, cells were lysed in Passive Lysis buffer (Promega) and the luminescence was quantified using Luciferase Assay System (Promega) according to the manufacturer&#x02019;s instructions. Fluc values were normalized to the values of the empty vector samples. To determine statistical significance, a one-way ANOVA with multiple comparisons using Dunnett&#x02019;s correction was performed. Nsp1 wildtype or Nsp1 mutant luciferase are compared to empty vector; <em>P-</em>value = 0.05. Western blot analysis is described in <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2314166121#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em></a>.</p></div><div id="s14" class="sec"><p></p><h3 id="s14title" class="inline">In Vitro Translation Assay. </h3><p class="p p-first-last">The in vitro translation assay was performed using the Reticulocyte Lysate IVT Kit (Ambion, Catalog number: AM1200) per the manufacturer&#x02019;s instructions. Briefly, purified Nsp1 wild-type, Nsp1<sup>D33K/E36K/E37K/E41K</sup>, or GST proteins were incubated at a concentration of 1 &#x003bc;M with 250 ng of in vitro transcribed firefly Luciferase (Fluc) mRNA, reticulocyte lysate, and translation mix without methionine. Reactions were supplemented with unlabeled methionine (Promega, Catalog number: L118A) at a concentration of 50 &#x003bc;M. The samples were incubated at 30&#x000b0;C for 90 min. followed by incubation with 2.5 &#x003bc;L of 1 mg/mL RNase A (Qiagen, Catalog number: 19101) for 10 min. Samples were then stored on ice for immediate analysis. For analysis, samples were incubated with Passive Lysis Buffer (Promega, Catalog number: E1910) at a 1:10 dilution and the luminescence was quantified using the Luciferase Assay System (Promega, Catalog number: E1500) per the manufacturer&#x02019;s instructions. To determine statistical significance, one-way ANOVA with multiple comparisons using Dunnett&#x02019;s correction was performed. Nsp1 wild-type, Nsp1<sup>D33K/E36K/E37K/E41K</sup>, or GST were compared to control samples with no purified protein added.</p></div><div id="s15" class="sec"><p></p><h3 id="s15title" class="inline">Metabolic Labeling with Click-iT HPG Alexa 594, FUNCAT Assay. </h3><p class="p p-first-last">A549 cells were seeded at a density of 40,000/cells per well in a 24-well format on the top of round glass coverslips and transfected with 1 &#x000b5;g pCI-Neo-3xFLAG or pCl-neo-<sup>3xFLAG</sup>SARS-CoV-2-Nsp1 wildtype or mutants using Lipofectamine 3000 reagent in Opti-MEM Reduced Serum Medium. After 16 h of transfection, newly synthesized proteins were detected using the Click-iT&#x02122; HPG Alexa Fluor&#x02122; 594 Protein Synthesis Assay Kit. Additional details are described in <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2314166121#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em></a>.</p></div><div id="s16" class="sec"><p></p><h3 id="s16title" class="inline">Generation of Recombinant Mutant SARS-CoV-2. </h3><p class="p p-first-last">We used our previously described BAC-based SARS-CoV-2 reverse genetic systems (<a href="#r21" rid="r21" class=" bibr popnode">21</a>) to generate the rSARS-CoV-2 containing the D33K/E36K/E37K/E41K mutation in the nsp1. Experimental details are described in <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2314166121#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em></a>. The P1 stock was aliquoted, titrated, deep sequenced (<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2314166121#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S2</a>), and stored at &#x02212;80 &#x000b0;C until being used.</p></div><div id="s17" class="sec"><p></p><h3 id="s17title" class="inline">PLA Combined with Immunofluorescence Microscopy. </h3><p class="p p-first-last">A549-ACE2 cells were mock infected or infected with rSARS-CoV2<sup>WT</sup> or rSARS-CoV2<sup>D33K/E36K/E37K/E41K</sup>. After 12 h, cells were fixed in 10 % formaldehyde in PBS at room temperature and permeabilized with 0.01% Triton X-100 in PBS for 10 min. Samples were then rinsed briefly with PBS and incubated with the Duolink blocking solution (Duolink In Situ Red Starter Mouse/Rabbit kit, DUO9210, Sigma-Aldrich Co. LLC) in a preheated humidity chamber for 30 min at 37 &#x000b0;C. Primary antibody cocktail [anti-NXF1 antibody, mouse monoclonal (T1076, Sigma-Aldrich Co. LLC, diluted at 1:250), and affinity purified rabbit anti-Nsp1 antibodies (<a href="#r3" rid="r3" class=" bibr popnode">3</a>) (diluted at 1:100)] was diluted in the Duolink antibody diluent solution and added to the cells followed by incubation for 1 h at room temperature. Then the PLA was completed according to the manufacturer&#x02019;s instructions. To identify infected cells, samples subjected to PLA were subsequently incubated overnight with affinity-purified rabbit anti-Nsp1 antibodies (<a href="#r3" rid="r3" class=" bibr popnode">3</a>) at 4 &#x000b0;C. After washing with TBS-T, samples were incubated with the Alexa Fluor-488 goat anti-rabbit antibody (A110008, Molecular Probes, Life Technologies at 1:800 dilution). Next, samples were incubated with Hoechst 33342 (1.5 &#x000b5;g/mL, Molecular Probes) for nuclear counterstaining, then briefly washed with PBS for 5 min, and cells were kept in PBS. Infected cells were visualized using a FITC filter and PLA signals were recognized as red fluorescent spots using a TRITC filter. Antibody specificity was assessed in mock infected cells as PLA negative control. Fluorescence microscopy, image processing, and subsequent qualitative image analysis were performed as described in the ViewRNA methods section. Maximum intensity three-dimensional projection (MIP rendering) image processing was performed as we previously described (<a href="#r34" rid="r34" class=" bibr popnode">34</a>).</p></div><div id="s18" class="sec"><p></p><h3 id="s18title" class="inline">SARS-CoV-2 Growth Kinetics. </h3><p class="p p-first-last">A549-ACE2 cells were seeded in 24-well format at a density of 100,000 cells per well. The next day, cells were infected at MOI 0.1 for 1h. Infections were performed in Viral Growth Media (VGM) (Dulbecco&#x02019;s modified Eagle&#x02019;s medium (Corning) supplemented with 2% FBS (Peak Serum), 1% nonessential amino acids (Gibco), 1% HEPES (Gibco), and 1% penicillin/streptomycin (Corning) at 37 &#x000b0;C and 5% CO<sub>2</sub>. The inoculum was removed and replaced with fresh infection media. Supernatants were collected at 12, 24, 36, and 48 hpi. Supernatants were stored at &#x02212;80 &#x000b0;C before evaluation of viral titers. Titers were quantified by the plaque assay in Vero E6 cells as previously described (<a href="#r35" rid="r35" class=" bibr popnode">35</a>). Additional details are described in <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2314166121#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em></a>.</p></div><div id="s19" class="sec"><p></p><h3 id="s19title" class="inline">Immunofluorescence. </h3><p class="p p-first-last">At the indicated time postinfection, cells were fixed with 4% formaldehyde overnight. Cells were permeabilized in 0.1% Triton in PBS and blocked in 3% BSA in PBS. Cells were then incubated in PBS containing 1% BSA and anti-SARS-CoV-2 N (1C7C7, Sigma Aldrich) at a 1:200 dilution for 1 h, at room temperature. Cells were washed in PBS three times for 5 min. Cells were incubated for 1 h at room temperature in PBS containing 1% BSA and 1:1,000 of AlexaFluor anti-mouse 488 and DAPI. Cells were washed in PBS three times and mounted in Antifade Gold mounting media. Images were acquired on the EVOS5000 microscope.</p></div><div id="s20" class="sec sec-last"><p></p><h3 id="s20title" class="inline">SARS-CoV-2 Infection In Vivo. </h3><p class="p p-first-last">Eight-week-old female B6.Cg-Tg(K18-ACE2)2Prlmn/J mice were obtained from The Jackson Laboratory and housed in a BSL-3 vivarium at the Icahn School of Medicine at Mount Sinai under the guidelines of the Institutional Animal Care and Use Committee (IACUC) of the Icahn School of Medicine at Mount Sinai (ISMMS). Experimental procedures are described in detail in <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2314166121#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em></a>.</p></div></div><div id="sec-a.m.h" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="sec-a.m.htitle" style="text-transform: uppercase;">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="supplementary-material-a.m.h.b"><p>Appendix 01 (PDF)</p><div class="sup-box half_rhythm" id="media-a.m.h.b.b"><a href="/pmc/articles/PMC11145185/bin/pnas.2314166121.sapp.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(16M, pdf)</sup></div></div></div><div id="ack-a.n.a" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.n.atitle" style="text-transform: uppercase;">Acknowledgments</h2><div class="sec"><p>We thank the Ascano lab for assistance in crosslinking experiments. We thank the proteomics core of the Vanderbilt Mass Spectrometry Research Center for assistance in mass spectrometry experiments. We thank Randy Albrecht for support with the BSL3 facility and procedures at the Icahn School of Medicine at Mount Sinai (ISMMS), the staff at the ISMMS Center for Comparative Medicine and Surgery (CCMS) for support with the ABSL3 facility, and Richard Cadagan for excellent technical assistance. Confocal microscope images were taken in the Live Cell Imaging Facility at UT Southwestern and in the Microscopy Shared Resource Facility at the ISMMS. Animal tissues were processed at the Biorepository and Pathology Core (ISMMS) for tissue processing and histology. B.M.A.F. holds the Ruth S. Harrell Professorship in Medical Research. This work was supported by NC1 U54 CA260560 (Project 2) to B.M.A.F. and A.G.-S., NIH&#x02014;Center for Antiviral Medicines &#x00026; Pandemic Preparedness (CAMPP) U19AI171443-01 to B.M.A.F., L.M., and A.G.-S. (Project 4), NIGMS R35 GM133743 to Y.R., U19AI135972 to A.G.-S. and CRIPT (Center for Research in Influenza Pathogenesis and Transmission), a NIAID funded Center of Excellence for Influenza Research and Response (CEIRR, contract #75N93021C00014) to A.G.-S. This work utilized shared equipment purchased with NIH SIG grants S10-OD026983 and S10OD030269 to N.A.C. The A.G.-S. laboratory has received research support from GSK, Pfizer, Senhwa Biosciences, Kenall Manufacturing, Blade Therapeutics, Avimex, Johnson &#x00026; Johnson, Dynavax, 7Hills Pharma, Pharmamar, ImmunityBio, Accurius, Nanocomposix, Hexamer, N-fold LLC, Model Medicines, Atea Pharma, Applied Biological Laboratories, and Merck, outside of the reported work.</p><div id="s22" class="sec"><p></p><h3 id="s22title" class="inline">Author contributions </h3><p class="p p-first-last">Y.R., A.G.-S., and B.M.A.F. designed research; M.M., A.C., L.M., C.Y., T.C., K.Z., K.P., N.W., W.H.M., N.A.C., M.L., M.R., S.A., and I.M. performed research; C.Y., L.M.-S., and B.M.A.F. contributed new reagents/analytic tools; M.M., A.C., L.M., C.Y., T.C., K.Z., K.P., N.W., W.H.M., N.A.C., M.L., M.R., S.A., I.M., Y.R., A.G.-S., and B.M.A.F. analyzed data; and L.M., N.A.C., Y.R., A.G.-S., and B.M.A.F. wrote the paper.</p></div><div id="s23" class="sec sec-last"><p></p><h3 id="s23title" class="inline">Competing interests </h3><p class="p p-first-last">A.G.-S. has consulting agreements for the following companies involving cash and/or stock: Castlevax, Amovir, Vivaldi Biosciences, Contrafect, 7Hills Pharma, Avimex, Pagoda, Accurius, Esperovax, Farmak, Applied Biological Laboratories, Pharmamar, CureLab Oncology, CureLab Veterinary, Synairgen, Paratus, and Pfizer, outside of the reported work. A.G.-S. has been an invited speaker in meeting events organized by Seqirus, Janssen, Abbott, and Astrazeneca. A.G.-S. is inventor on patents and patent applications on the use of antivirals and vaccines for the treatment and prevention of virus infections and cancer, owned by the Icahn School of Medicine at Mount Sinai, New York.</p></div></div></div><div id="fn-group-a.n.b" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.n.btitle" style="text-transform: uppercase;">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="fn5"><p class="p p-first-last">This article is a PNAS Direct Submission. A.H.C. is a guest editor invited by the Editorial Board.</p></p></div></div><div id="s21" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="s21title" style="text-transform: uppercase;">Data, Materials, and Software Availability</h2><!--/article/back/sec/--><p class="p p-first-last">All study data are included in the article and/or <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2314166121#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em></a>.</p></div><div id="s24" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="s24title" style="text-transform: uppercase;">Supporting Information</h2><!--/article/back/sec/--></div><div id="ref-list-a.n.d" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.n.dtitle" style="text-transform: uppercase;">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="r1">1. <span class="mixed-citation">Finkel Y., et al., <span class="ref-title">SARS-CoV-2 uses a multipronged strategy to impede host protein synthesis</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol"><strong>594</strong></span>, 240&#x02013;245 (2021).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/33979833" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=SARS-CoV-2+uses+a+multipronged+strategy+to+impede+host+protein+synthesis&amp;volume=594&amp;publication_year=2021&amp;pages=240-245&amp;pmid=33979833&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r2">2. <span class="mixed-citation">Thoms M., et al., <span class="ref-title">Structural basis for translational shutdown and immune evasion by the Nsp1 protein of SARS-CoV-2</span>. <span class="ref-journal">Science</span>
<span class="ref-vol"><strong>369</strong></span>, 1249&#x02013;1255 (2020).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7402621/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32680882" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Structural+basis+for+translational+shutdown+and+immune+evasion+by+the+Nsp1+protein+of+SARS-CoV-2&amp;volume=369&amp;publication_year=2020&amp;pages=1249-1255&amp;pmid=32680882&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r3">3. <span class="mixed-citation">Zhang K., et al., <span class="ref-title">Nsp1 protein of SARS-CoV-2 disrupts the mRNA export machinery to inhibit host gene expression</span>. <span class="ref-journal">Sci. Adv.</span>
<span class="ref-vol"><strong>7</strong></span>, eabe7386 (2021).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7864571/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33547084" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Adv.&amp;title=Nsp1+protein+of+SARS-CoV-2+disrupts+the+mRNA+export+machinery+to+inhibit+host+gene+expression&amp;volume=7&amp;publication_year=2021&amp;pages=eabe7386&amp;pmid=33547084&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r4">4. <span class="mixed-citation">Vora S. M., et al., <span class="ref-title">Targeting stem-loop 1 of the SARS-CoV-2 5&#x02019; UTR to suppress viral translation and Nsp1 evasion</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U. S. A.</span>
<span class="ref-vol"><strong>119</strong></span>, e2117198119 (2022).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8892331/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35149555" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.+S.+A.&amp;title=Targeting+stem-loop+1+of+the+SARS-CoV-2+5&#x02019;+UTR+to+suppress+viral+translation+and+Nsp1+evasion&amp;volume=119&amp;publication_year=2022&amp;pages=e2117198119&amp;pmid=35149555&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r5">5. <span class="mixed-citation">Yuan S., et al., <span class="ref-title">Nonstructural protein 1 of SARS-CoV-2 is a potent pathogenicity factor redirecting host protein synthesis machinery toward viral RNA</span>. <span class="ref-journal">Mol. Cell</span>
<span class="ref-vol"><strong>80</strong></span>, 1055&#x02013;1066.e1056 (2020).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7833686/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33188728" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=Nonstructural+protein+1+of+SARS-CoV-2+is+a+potent+pathogenicity+factor+redirecting+host+protein+synthesis+machinery+toward+viral+RNA&amp;volume=80&amp;publication_year=2020&amp;pages=1055-1066.e1056&amp;pmid=33188728&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r6">6. <span class="mixed-citation">Schubert K., et al., <span class="ref-title">SARS-CoV-2 Nsp1 binds the ribosomal mRNA channel to inhibit translation</span>. <span class="ref-journal">Nat. Struct. Mol. Biol.</span>
<span class="ref-vol"><strong>27</strong></span>, 959&#x02013;966 (2020).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/32908316" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Struct.+Mol.+Biol.&amp;title=SARS-CoV-2+Nsp1+binds+the+ribosomal+mRNA+channel+to+inhibit+translation&amp;volume=27&amp;publication_year=2020&amp;pages=959-966&amp;pmid=32908316&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r7">7. <span class="mixed-citation">Shehata S. I., Parker R., <span class="ref-title">SARS-CoV-2 Nsp1 mediated mRNA degradation requires mRNA interaction with the ribosome</span>. <span class="ref-journal">RNA Biol.</span>
<span class="ref-vol"><strong>20</strong></span>, 444&#x02013;456 (2023).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10332192/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37415298" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=RNA+Biol.&amp;title=SARS-CoV-2+Nsp1+mediated+mRNA+degradation+requires+mRNA+interaction+with+the+ribosome&amp;volume=20&amp;publication_year=2023&amp;pages=444-456&amp;pmid=37415298&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r8">8. <span class="mixed-citation">Fisher T., et al., <span class="ref-title">Parsing the role of NSP1 in SARS-CoV-2 infection</span>. <span class="ref-journal">Cell Rep.</span>
<span class="ref-vol"><strong>39</strong></span>, 110954 (2022).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9133101/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35671758" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Rep.&amp;title=Parsing+the+role+of+NSP1+in+SARS-CoV-2+infection&amp;volume=39&amp;publication_year=2022&amp;pages=110954&amp;pmid=35671758&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r9">9. <span class="mixed-citation">Xie Y., Ren Y., <span class="ref-title">Mechanisms of nuclear mRNA export: A structural perspective</span>. <span class="ref-journal">Traffic</span>
<span class="ref-vol"><strong>20</strong></span>, 829&#x02013;840 (2019).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7074880/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31513326" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Traffic&amp;title=Mechanisms+of+nuclear+mRNA+export:+A+structural+perspective&amp;volume=20&amp;publication_year=2019&amp;pages=829-840&amp;pmid=31513326&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r10">10. <span class="mixed-citation">Carmody S. R., Wente S. R., <span class="ref-title">mRNA nuclear export at a glance</span>. <span class="ref-journal">J. Cell Sci.</span>
<span class="ref-vol"><strong>122</strong></span>, 1933&#x02013;1937 (2009).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2723150/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19494120" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Cell+Sci.&amp;title=mRNA+nuclear+export+at+a+glance&amp;volume=122&amp;publication_year=2009&amp;pages=1933-1937&amp;pmid=19494120&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r11">11. <span class="mixed-citation">Zhang K., et al., <span class="ref-title">Structural basis for influenza virus NS1 protein block of mRNA nuclear export</span>. <span class="ref-journal">Nat. Microbiol.</span>
<span class="ref-vol"><strong>4</strong></span>, 1671&#x02013;1679 (2019).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6754785/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31263181" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Microbiol.&amp;title=Structural+basis+for+influenza+virus+NS1+protein+block+of+mRNA+nuclear+export&amp;volume=4&amp;publication_year=2019&amp;pages=1671-1679&amp;pmid=31263181&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r12">12. <span class="mixed-citation">Satterly N., et al., <span class="ref-title">Influenza virus targets the mRNA export machinery and the nuclear pore complex</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span>
<span class="ref-vol"><strong>104</strong></span>, 1853&#x02013;1858 (2007).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1794296/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17267598" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Influenza+virus+targets+the+mRNA+export+machinery+and+the+nuclear+pore+complex&amp;volume=104&amp;publication_year=2007&amp;pages=1853-1858&amp;pmid=17267598&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r13">13. <span class="mixed-citation">Viphakone N., et al., <span class="ref-title">TREX exposes the RNA-binding domain of Nxf1 to enable mRNA export</span>. <span class="ref-journal">Nat. Commun.</span>
<span class="ref-vol"><strong>3</strong></span>, 1006 (2012).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3654228/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22893130" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=TREX+exposes+the+RNA-binding+domain+of+Nxf1+to+enable+mRNA+export&amp;volume=3&amp;publication_year=2012&amp;pages=1006&amp;pmid=22893130&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r14">14. <span class="mixed-citation">Stutz F., et al., <span class="ref-title">REF, an evolutionary conserved family of hnRNP-like proteins, interacts with TAP/Mex67p and participates in mRNA nuclear export</span>. <span class="ref-journal">RNA</span>
<span class="ref-vol"><strong>6</strong></span>, 638&#x02013;650 (2000).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1369944/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10786854" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=RNA&amp;title=REF,+an+evolutionary+conserved+family+of+hnRNP-like+proteins,+interacts+with+TAP/Mex67p+and+participates+in+mRNA+nuclear+export&amp;volume=6&amp;publication_year=2000&amp;pages=638-650&amp;pmid=10786854&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r15">15. <span class="mixed-citation">Kamitani W., Huang C., Narayanan K., Lokugamage K. G., Makino S., <span class="ref-title">A two-pronged strategy to suppress host protein synthesis by SARS coronavirus Nsp1 protein</span>. <span class="ref-journal">Nat. Struct. Mol. Biol.</span>
<span class="ref-vol"><strong>16</strong></span>, 1134&#x02013;1140 (2009).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2784181/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19838190" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Struct.+Mol.+Biol.&amp;title=A+two-pronged+strategy+to+suppress+host+protein+synthesis+by+SARS+coronavirus+Nsp1+protein&amp;volume=16&amp;publication_year=2009&amp;pages=1134-1140&amp;pmid=19838190&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r16">16. <span class="mixed-citation">Kilkenny M. L., et al., <span class="ref-title">Structural basis for the interaction of SARS-CoV-2 virulence factor nsp1 with DNA polymerase alpha-primase</span>. <span class="ref-journal">Protein Sci.</span>
<span class="ref-vol"><strong>31</strong></span>, 333&#x02013;344 (2022).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8661717/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34719824" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Sci.&amp;title=Structural+basis+for+the+interaction+of+SARS-CoV-2+virulence+factor+nsp1+with+DNA+polymerase+alpha-primase&amp;volume=31&amp;publication_year=2022&amp;pages=333-344&amp;pmid=34719824&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r17">17. <span class="mixed-citation">Mendez A. S., et al., <span class="ref-title">The N-terminal domain of SARS-CoV-2 nsp1 plays key roles in suppression of cellular gene expression and preservation of viral gene expression</span>. <span class="ref-journal">Cell Rep.</span>
<span class="ref-vol"><strong>37</strong></span>, 109841 (2021).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8481097/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34624207" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Rep.&amp;title=The+N-terminal+domain+of+SARS-CoV-2+nsp1+plays+key+roles+in+suppression+of+cellular+gene+expression+and+preservation+of+viral+gene+expression&amp;volume=37&amp;publication_year=2021&amp;pages=109841&amp;pmid=34624207&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r18">18. <span class="mixed-citation">Lokugamage K. G., Narayanan K., Huang C., Makino S., <span class="ref-title">Severe acute respiratory syndrome coronavirus protein nsp1 is a novel eukaryotic translation inhibitor that represses multiple steps of translation initiation</span>. <span class="ref-journal">J. Virol.</span>
<span class="ref-vol"><strong>86</strong></span>, 13598&#x02013;13608 (2012).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3503042/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23035226" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Virol.&amp;title=Severe+acute+respiratory+syndrome+coronavirus+protein+nsp1+is+a+novel+eukaryotic+translation+inhibitor+that+represses+multiple+steps+of+translation+initiation&amp;volume=86&amp;publication_year=2012&amp;pages=13598-13608&amp;pmid=23035226&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r19">19. <span class="mixed-citation">Chin J. W., Martin A. B., King D. S., Wang L., Schultz P. G., <span class="ref-title">Addition of a photocrosslinking amino acid to the genetic code of <em>Escherichia coli</em></span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span>
<span class="ref-vol"><strong>99</strong></span>, 11020&#x02013;11024 (2002).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC123203/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12154230" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Addition+of+a+photocrosslinking+amino+acid+to+the+genetic+code+of+Escherichia+coli&amp;volume=99&amp;publication_year=2002&amp;pages=11020-11024&amp;pmid=12154230&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r20">20. <span class="mixed-citation">Narayanan K., et al., <span class="ref-title">Severe acute respiratory syndrome coronavirus nsp1 suppresses host gene expression, including that of type I interferon, in infected cells</span>. <span class="ref-journal">J. Virol.</span>
<span class="ref-vol"><strong>82</strong></span>, 4471&#x02013;4479 (2008).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2293030/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18305050" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Virol.&amp;title=Severe+acute+respiratory+syndrome+coronavirus+nsp1+suppresses+host+gene+expression,+including+that+of+type+I+interferon,+in+infected+cells&amp;volume=82&amp;publication_year=2008&amp;pages=4471-4479&amp;pmid=18305050&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r21">21. <span class="mixed-citation">Ye C., et al., <span class="ref-title">Rescue of SARS-CoV-2 from a single bacterial artificial chromosome</span>. <span class="ref-journal">mBio</span>
<span class="ref-vol"><strong>11</strong></span>, e02168-20 (2020).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7520601/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32978313" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=Rescue+of+SARS-CoV-2+from+a+single+bacterial+artificial+chromosome&amp;volume=11&amp;publication_year=2020&amp;pages=e02168-20&amp;pmid=32978313&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r22">22. <span class="mixed-citation">Bhat P., et al., <span class="ref-title">Influenza virus mRNAs encode determinants for nuclear export via the cellular TREX-2 complex</span>. <span class="ref-journal">Nat. Commun.</span>
<span class="ref-vol"><strong>14</strong></span>, 2304 (2023).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10121598/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37085480" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Influenza+virus+mRNAs+encode+determinants+for+nuclear+export+via+the+cellular+TREX-2+complex&amp;volume=14&amp;publication_year=2023&amp;pages=2304&amp;pmid=37085480&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r23">23. <span class="mixed-citation">Jimenez-Guardeno J. M., et al., <span class="ref-title">Identification of the mechanisms causing reversion to virulence in an attenuated SARS-CoV for the design of a genetically stable vaccine</span>. <span class="ref-journal">PLoS Pathog.</span>
<span class="ref-vol"><strong>11</strong></span>, e1005215 (2015).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4626112/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26513244" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Pathog.&amp;title=Identification+of+the+mechanisms+causing+reversion+to+virulence+in+an+attenuated+SARS-CoV+for+the+design+of+a+genetically+stable+vaccine&amp;volume=11&amp;publication_year=2015&amp;pages=e1005215&amp;pmid=26513244&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r24">24. <span class="mixed-citation">Zust R., et al., <span class="ref-title">Coronavirus non-structural protein 1 is a major pathogenicity factor: Implications for the rational design of coronavirus vaccines</span>. <span class="ref-journal">PLoS Pathog.</span>
<span class="ref-vol"><strong>3</strong></span>, e109 (2007).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1941747/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17696607" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Pathog.&amp;title=Coronavirus+non-structural+protein+1+is+a+major+pathogenicity+factor:+Implications+for+the+rational+design+of+coronavirus+vaccines&amp;volume=3&amp;publication_year=2007&amp;pages=e109&amp;pmid=17696607&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r25">25. <span class="mixed-citation">Lei X., et al., <span class="ref-title">Activation and evasion of type I interferon responses by SARS-CoV-2</span>. <span class="ref-journal">Nat. Commun.</span>
<span class="ref-vol"><strong>11</strong></span>, 3810 (2020).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7392898/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32733001" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Activation+and+evasion+of+type+I+interferon+responses+by+SARS-CoV-2&amp;volume=11&amp;publication_year=2020&amp;pages=3810&amp;pmid=32733001&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r26">26. <span class="mixed-citation">Xia H., et al., <span class="ref-title">Evasion of Type I interferon by SARS-CoV-2</span>. <span class="ref-journal">Cell Rep.</span>
<span class="ref-vol"><strong>33</strong></span>, 108234 (2020).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7501843/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32979938" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Rep.&amp;title=Evasion+of+Type+I+interferon+by+SARS-CoV-2&amp;volume=33&amp;publication_year=2020&amp;pages=108234&amp;pmid=32979938&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r27">27. <span class="mixed-citation">Burke J. M., St Clair L. A., Perera R., Parker R., <span class="ref-title">SARS-CoV-2 infection triggers widespread host mRNA decay leading to an mRNA export block</span>. <span class="ref-journal">RNA</span>
<span class="ref-vol"><strong>27</strong></span>, 1318&#x02013;1329 (2021).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8522697/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34315815" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=RNA&amp;title=SARS-CoV-2+infection+triggers+widespread+host+mRNA+decay+leading+to+an+mRNA+export+block&amp;volume=27&amp;publication_year=2021&amp;pages=1318-1329&amp;pmid=34315815&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r28">28. <span class="mixed-citation">Schubert K., et al., <span class="ref-title">Universal features of Nsp1-mediated translational shutdown by coronaviruses</span>. <span class="ref-journal">Mol. Cell</span>
<span class="ref-vol"><strong>83</strong></span>, 3546&#x02013;3557.e3548 (2023).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10575594/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37802027" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=Universal+features+of+Nsp1-mediated+translational+shutdown+by+coronaviruses&amp;volume=83&amp;publication_year=2023&amp;pages=3546-3557.e3548&amp;pmid=37802027&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r29">29. <span class="mixed-citation">Gilchrist M., et al., <span class="ref-title">Systems biology approaches identify ATF3 as a negative regulator of Toll-like receptor 4</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol"><strong>441</strong></span>, 173&#x02013;178 (2006).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/16688168" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Systems+biology+approaches+identify+ATF3+as+a+negative+regulator+of+Toll-like+receptor+4&amp;volume=441&amp;publication_year=2006&amp;pages=173-178&amp;pmid=16688168&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r30">30. <span class="mixed-citation">Liu T., Zhang L., Joo D., Sun S. C., <span class="ref-title">NF-kappaB signaling in inflammation</span>. <span class="ref-journal">Signal Transduct. Target Ther.</span>
<span class="ref-vol"><strong>2</strong></span>, 17023 (2017).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5661633/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29158945" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Signal+Transduct.+Target+Ther.&amp;title=NF-kappaB+signaling+in+inflammation&amp;volume=2&amp;publication_year=2017&amp;pages=17023&amp;pmid=29158945&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r31">31. <span class="mixed-citation">Whitmore M. M., et al., <span class="ref-title">Negative regulation of TLR-signaling pathways by activating transcription factor-3</span>. <span class="ref-journal">J. Immunol.</span>
<span class="ref-vol"><strong>179</strong></span>, 3622&#x02013;3630 (2007).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/17785797" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Immunol.&amp;title=Negative+regulation+of+TLR-signaling+pathways+by+activating+transcription+factor-3&amp;volume=179&amp;publication_year=2007&amp;pages=3622-3630&amp;pmid=17785797&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r32">32. <span class="mixed-citation">Chen Z. L., et al., <span class="ref-title">A high-speed search engine pLink 2 with systematic evaluation for proteome-scale identification of cross-linked peptides</span>. <span class="ref-journal">Nat. Commun.</span>
<span class="ref-vol"><strong>10</strong></span>, 3404 (2019).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6667459/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31363125" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=A+high-speed+search+engine+pLink+2+with+systematic+evaluation+for+proteome-scale+identification+of+cross-linked+peptides&amp;volume=10&amp;publication_year=2019&amp;pages=3404&amp;pmid=31363125&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r33">33. <span class="mixed-citation">Mor A., et al., <span class="ref-title">Influenza virus mRNA trafficking through host nuclear speckles</span>. <span class="ref-journal">Nat. Microbiol.</span>
<span class="ref-vol"><strong>1</strong></span>, 16069 (2016). 10.1038/nmicrobiol.2016.69.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4917225/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27572970" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnmicrobiol.2016.69" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Microbiol.&amp;title=Influenza+virus+mRNA+trafficking+through+host+nuclear+speckles&amp;volume=1&amp;publication_year=2016&amp;pages=16069&amp;pmid=27572970&amp;doi=10.1038/nmicrobiol.2016.69&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r34">34. <span class="mixed-citation">Gao S., et al., <span class="ref-title">Nuclear speckle integrity and function require TAO2 kinase</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span>
<span class="ref-vol"><strong>119</strong></span>, e2206046119 (2022).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9231605/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35704758" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Nuclear+speckle+integrity+and+function+require+TAO2+kinase&amp;volume=119&amp;publication_year=2022&amp;pages=e2206046119&amp;pmid=35704758&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r35">35. <span class="mixed-citation">Amanat F., et al., <span class="ref-title">An in vitro microneutralization assay for SARS-CoV-2 serology and drug screening</span>. <span class="ref-journal">Curr. Protoc. Microbiol.</span>
<span class="ref-vol"><strong>58</strong></span>, e108 (2020).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7361222/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32585083" ref="reftype=pubmed&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Protoc.+Microbiol.&amp;title=An+in+vitro+microneutralization+assay+for+SARS-CoV-2+serology+and+drug+screening&amp;volume=58&amp;publication_year=2020&amp;pages=e108&amp;pmid=32585083&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11145185&amp;issue-id=463946&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Proceedings of the National Academy of Sciences of the United States of America</span> are provided here courtesy of <strong>National Academy of Sciences</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11145185/pdf/pnas.202314166.pdf" class="int-view">PDF (11M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11145185/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11145185/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11145185" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11145185%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="lhsvues23Q7tr9m3IpRmfghv9252KYAynDSOxPgB9sTt0HxmrIofubr98nqazXoe">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11145185%2F&amp;text=Inhibition%20of%20mRNA%20nuclear%20export%20promotes%20SARS-CoV-2%20pathogenesis" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11145185%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11145185/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11145185/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38768348/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11145185/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38768348/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11145185/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11145185/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC11145185/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
