
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Elucidating the Impact of Deleterious Mutations on IGHG1 and Their Association with Huntington&#8217;s Disease - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="939BE807DB0F4BC50000485908D055FA.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="jpersmed" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC11050829/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11050829/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Journal of Personalized Medicine" /><meta name="citation_title" content="Elucidating the Impact of Deleterious Mutations on IGHG1 and Their Association with Huntington&#x02019;s Disease" /><meta name="citation_author" content="Alaa Shafie" /><meta name="citation_author_institution" content="Department of Clinical Laboratory Sciences, College of Applied Medical Sciences, Taif University, P.O. Box 11099, Taif 21944, Saudi Arabia; as.ude.ut@eifahs.a (A.S.); as.ude.ut@mujnaharaf (F.A.)" /><meta name="citation_author" content="Amal Adnan Ashour" /><meta name="citation_author_institution" content="Department of Oral and Maxillofacial Surgery and Diagnostic Sciences, Faculty of Dentistry, Taif University, P.O. Box 11099, Taif 21944, Saudi Arabia; as.ude.ut@ruohsa.a.a" /><meta name="citation_author" content="Farah Anjum" /><meta name="citation_author_institution" content="Department of Clinical Laboratory Sciences, College of Applied Medical Sciences, Taif University, P.O. Box 11099, Taif 21944, Saudi Arabia; as.ude.ut@eifahs.a (A.S.); as.ude.ut@mujnaharaf (F.A.)" /><meta name="citation_author" content="Anas Shamsi" /><meta name="citation_author_institution" content="Center of Medical and Bio-Allied Health Sciences Research (CMBHSR), Ajman University, Ajman P.O. Box 346, United Arab Emirates" /><meta name="citation_author" content="Md. Imtaiyaz Hassan" /><meta name="citation_author_institution" content="Centre for Interdisciplinary Research in Basic Sciences, Jamia Millia Islamia, Jamia Nagar, New Delhi 110025, India" /><meta name="citation_publication_date" content="2024/04" /><meta name="citation_issue" content="4" /><meta name="citation_volume" content="14" /><meta name="citation_doi" content="10.3390/jpm14040380" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC11050829/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC11050829/" /><meta name="citation_pmid" content="38673007" /><meta name="DC.Title" content="Elucidating the Impact of Deleterious Mutations on IGHG1 and Their Association with Huntington&#x02019;s Disease" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Multidisciplinary Digital Publishing Institute  (MDPI)" /><meta name="DC.Contributor" content="Alaa Shafie" /><meta name="DC.Contributor" content="Amal Adnan Ashour" /><meta name="DC.Contributor" content="Farah Anjum" /><meta name="DC.Contributor" content="Anas Shamsi" /><meta name="DC.Contributor" content="Md. Imtaiyaz Hassan" /><meta name="DC.Date" content="2024 Apr" /><meta name="DC.Identifier" content="10.3390/jpm14040380" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Elucidating the Impact of Deleterious Mutations on IGHG1 and Their Association with Huntington&#x02019;s Disease" /><meta property="og:type" content="article" /><meta property="og:description" content="Huntington&#x02019;s disease (HD) is a chronic, inherited neurodegenerative condition marked by chorea, dementia, and changes in personality. The primary cause of HD is a mutation characterized by the expansion of a triplet repeat (CAG) within the huntingtin ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11050829/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11050829/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22J%20Pers%20Med%22[journal]" class="navlink">J Pers Med</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11050829
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11050829/pdf/jpm-14-00380.pdf" class="int-view">PDF (2.6M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11050829/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11050829/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11050829" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11050829%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="AJ6aK4nDWrAehHjwF35ztl3N6vGgBZKa5ZANSwFql2Tegm3cqSk0sMT9v4bHrXOs">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11050829%2F&amp;text=Elucidating%20the%20Impact%20of%20Deleterious%20Mutations%20on%20IGHG1%20and%20Their%20Association%20with%20Huntington%02019s%20Disease%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11050829%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11050829/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11050829/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38673007/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11050829/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38673007/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11050829/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11050829/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22J%20Pers%20Med%22[journal]" class="navlink">J Pers Med</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11050829
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-jpersmed.png" alt="Logo of jpersmed" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.mdpi.com/journal/jpm" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">J Pers Med.</span> 2024 Apr; 14(4): 380. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2024 Apr 1. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.3390%2Fjpm14040380" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.3390/jpm14040380</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC11050829</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38673007">38673007</a></div></div></div><h1 class="content-title">Elucidating the Impact of Deleterious Mutations on IGHG1 and Their Association with Huntington&#x02019;s Disease</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shafie%20A%5BAuthor%5D" class="affpopup" co-rid="_co_idm140538185984720" co-class="co-affbox">Alaa Shafie</a>, <span class="fm-role">Conceptualization</span>, <span class="fm-role">Methodology</span>, <span class="fm-role">Software</span>, <span class="fm-role">Writing &#x02013; original draft</span>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ashour%20AA%5BAuthor%5D" class="affpopup" co-rid="_co_idm140538172490144" co-class="co-affbox">Amal Adnan Ashour</a>, <span class="fm-role">Methodology</span>, <span class="fm-role">Validation</span>, <span class="fm-role">Resources</span>, <span class="fm-role">Writing &#x02013; original draft</span>,<sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Anjum%20F%5BAuthor%5D" class="affpopup" co-rid="_co_idm140538187628416" co-class="co-affbox">Farah Anjum</a>, <span class="fm-role">Methodology</span>, <span class="fm-role">Investigation</span>, <span class="fm-role">Data curation</span>, <span class="fm-role">Project administration</span>, <span class="fm-role">Funding acquisition</span>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shamsi%20A%5BAuthor%5D" class="affpopup" co-rid="_co_idm140538181504304" co-class="co-affbox">Anas Shamsi</a>,<sup>3,</sup><sup>*</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hassan%20MI%5BAuthor%5D" class="affpopup" co-rid="_co_idm140538184398512" co-class="co-affbox">Md. Imtaiyaz Hassan</a>, <span class="fm-role">Conceptualization</span>, <span class="fm-role">Formal analysis</span>, <span class="fm-role">Data curation</span>, <span class="fm-role">Writing &#x02013; review &#x00026; editing</span><sup>4,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140538185984720"><h3 class="no_margin">Alaa Shafie</h3><p><sup>1</sup>Department of Clinical Laboratory Sciences, College of Applied Medical Sciences, Taif University, P.O. Box 11099, Taif 21944, Saudi Arabia; <a href="mailto:dev@null" data-email="as.ude.ut@eifahs.a" class="oemail">as.ude.ut@eifahs.a</a> (A.S.); <a href="mailto:dev@null" data-email="as.ude.ut@mujnaharaf" class="oemail">as.ude.ut@mujnaharaf</a> (F.A.)</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shafie%20A%5BAuthor%5D">Alaa Shafie</a></div></div><div id="_co_idm140538172490144"><h3 class="no_margin">Amal Adnan Ashour</h3><p><sup>2</sup>Department of Oral and Maxillofacial Surgery and Diagnostic Sciences, Faculty of Dentistry, Taif University, P.O. Box 11099, Taif 21944, Saudi Arabia; <a href="mailto:dev@null" data-email="as.ude.ut@ruohsa.a.a" class="oemail">as.ude.ut@ruohsa.a.a</a></p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ashour%20AA%5BAuthor%5D">Amal Adnan Ashour</a></div></div><div id="_co_idm140538187628416"><h3 class="no_margin">Farah Anjum</h3><p><sup>1</sup>Department of Clinical Laboratory Sciences, College of Applied Medical Sciences, Taif University, P.O. Box 11099, Taif 21944, Saudi Arabia; <a href="mailto:dev@null" data-email="as.ude.ut@eifahs.a" class="oemail">as.ude.ut@eifahs.a</a> (A.S.); <a href="mailto:dev@null" data-email="as.ude.ut@mujnaharaf" class="oemail">as.ude.ut@mujnaharaf</a> (F.A.)</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Anjum%20F%5BAuthor%5D">Farah Anjum</a></div></div><div id="_co_idm140538181504304"><h3 class="no_margin">Anas Shamsi</h3><p><sup>3</sup>Center of Medical and Bio-Allied Health Sciences Research (CMBHSR), Ajman University, Ajman P.O. Box 346, United Arab Emirates</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shamsi%20A%5BAuthor%5D">Anas Shamsi</a></div></div><div id="_co_idm140538184398512"><h3 class="no_margin">Md. Imtaiyaz Hassan</h3><p><sup>4</sup>Centre for Interdisciplinary Research in Basic Sciences, Jamia Millia Islamia, Jamia Nagar, New Delhi 110025, India</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hassan%20MI%5BAuthor%5D">Md. Imtaiyaz Hassan</a></div></div></div></div><div class="contrib-group half_rhythm fm-editor">Andrea Bernetti, <span class="fm-role">Academic Editor</span></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140538187441712_ai idm140538185984848_ai idm140538181858512_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140538187441712_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140538187441712_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140538187441712_ai" style="display:none"><div class="fm-affl" id="af1-jpm-14-00380"><sup>1</sup>Department of Clinical Laboratory Sciences, College of Applied Medical Sciences, Taif University, P.O. Box 11099, Taif 21944, Saudi Arabia; <a href="mailto:dev@null" data-email="as.ude.ut@eifahs.a" class="oemail">as.ude.ut@eifahs.a</a> (A.S.); <a href="mailto:dev@null" data-email="as.ude.ut@mujnaharaf" class="oemail">as.ude.ut@mujnaharaf</a> (F.A.)</div><div class="fm-affl" id="af2-jpm-14-00380"><sup>2</sup>Department of Oral and Maxillofacial Surgery and Diagnostic Sciences, Faculty of Dentistry, Taif University, P.O. Box 11099, Taif 21944, Saudi Arabia; <a href="mailto:dev@null" data-email="as.ude.ut@ruohsa.a.a" class="oemail">as.ude.ut@ruohsa.a.a</a></div><div class="fm-affl" id="af3-jpm-14-00380"><sup>3</sup>Center of Medical and Bio-Allied Health Sciences Research (CMBHSR), Ajman University, Ajman P.O. Box 346, United Arab Emirates</div><div class="fm-affl" id="af4-jpm-14-00380"><sup>4</sup>Centre for Interdisciplinary Research in Basic Sciences, Jamia Millia Islamia, Jamia Nagar, New Delhi 110025, India</div><div id="c1-jpm-14-00380"><sup>*</sup>Correspondence: <a href="mailto:dev@null" data-email="ea.ca.namja@ismahs.m" class="oemail">ea.ca.namja@ismahs.m</a> (A.S.); <a href="mailto:dev@null" data-email="ni.ca.imj@nassahim" class="oemail">ni.ca.imj@nassahim</a> (M.I.H.)</div></div><div class="fm-article-notes hide half_rhythm" id="idm140538187441712_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2024 Mar 14; Revised 2024 Mar 27; Accepted 2024 Mar 28.</div></div><div class="permissions half_rhythm hide" id="idm140538187441712_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; 2024 by the authors.</div><div class="license half_rhythm">Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (<a href="https://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">https://creativecommons.org/licenses/by/4.0/</a>).</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div><a href="/pmc/articles/PMC11050829/bin/jpm-14-00380-s001.zip" data-ga-action="click_feat_suppl">jpm-14-00380-s001.zip</a><span style="color:gray"> (129K)</span></div><div class="small guid">GUID: 923E3E37-329A-4C09-A30B-C9CB1D94DDFF</div></div></dd></dl><dl data-length="165" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p>The original contributions presented in the study are included in the article/<a href="#app1-jpm-14-00380" rid="app1-jpm-14-00380" class=" app">Supplementary Materials</a>; further inquiries can be directed to the corresponding author.</p></dd></dl></div><div id="abstract-a.j.b.s" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.j.b.stitle">Abstract</h2><!--article-meta--><div><p class="p p-first-last">Huntington&#x02019;s disease (HD) is a chronic, inherited neurodegenerative condition marked by chorea, dementia, and changes in personality. The primary cause of HD is a mutation characterized by the expansion of a triplet repeat (CAG) within the <em>huntingtin</em> gene located on chromosome 4. Despite substantial progress in elucidating the molecular and cellular mechanisms of HD, an effective treatment for this disorder is not available so far. In recent years, researchers have been interested in studying cerebrospinal fluid (CSF) as a source of biomarkers that could aid in the diagnosis and therapeutic development of this disorder. Immunoglobulin heavy constant gamma 1 (IGHG1) is one of the CSF proteins found to increase significantly in HD. Considering this, it is reasonable to study the potential involvement of deleterious mutations in <em>IGHG1</em> in the pathogenesis of this disorder. In this study, we explored the potential impact of deleterious mutations on <em>IGHG1</em> and their subsequent association with HD. We evaluated 126 single-point amino acid substitutions for their impact on the structure and functionality of the IGHG1 protein while exploiting multiple computational resources such as SIFT, PolyPhen-2, FATHMM, SNPs&#x00026;Go mCSM, DynaMut2, MAESTROweb, PremPS, MutPred2, and PhD-SNP. The sequence- and structure-based tools highlighted 10 amino acid substitutions that were deleterious and destabilizing. Subsequently, out of these 10 mutations, eight variants (Y32C, Y32D, P34S, V39E, C83R, C83Y, V85M, and H87Q) were identified as pathogenic by disease phenotype predictors. Finally, two pathogenic variants (Y32C and P34S) were found to reduce the solubility of the protein, suggesting their propensity to form protein aggregates. These variants also exhibited higher residual frustration within the protein structure. Considering these findings, the study hypothesized that the identified variants of IGHG1 may compromise its function and potentially contribute to HD pathogenesis.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Huntington&#x02019;s disease, immunoglobulin heavy constant gamma 1, deleterious mutations, amino acid substitutions, bioinformatics analysis</span></div></div><div id="sec1-jpm-14-00380" class="tsec sec"><h2 class="head no_bottom_margin" id="sec1-jpm-14-00380title">1. Introduction</h2><p class="p p-first">Huntington&#x02019;s disease (HD) presents an enduring challenge within the landscape of neurodegenerative disorders [<a href="#B1-jpm-14-00380" rid="B1-jpm-14-00380" class=" bibr popnode">1</a>]. It is a hereditary disorder characterized by a relentless progression of motor dysfunction, cognitive deterioration, and psychiatric disturbances [<a href="#B2-jpm-14-00380" rid="B2-jpm-14-00380" class=" bibr popnode">2</a>]. With an incidence of approximately 5 to 10 cases per 100,000 individuals worldwide, HD represents one of the most prevalent autosomal dominant inherited neurodegenerative conditions [<a href="#B3-jpm-14-00380" rid="B3-jpm-14-00380" class=" bibr popnode">3</a>]. The disease compromises language abilities, and there is evidence that the pragmatic aspects of communication are impaired in the early stages of the disease [<a href="#B4-jpm-14-00380" rid="B4-jpm-14-00380" class=" bibr popnode">4</a>]. The disease progression follows a characteristic pattern of brain atrophy, beginning in the basal ganglia structures [<a href="#B4-jpm-14-00380" rid="B4-jpm-14-00380" class=" bibr popnode">4</a>]. HD is devastating to patients and their families, with autosomal dominant inheritance, onset (typically) in the prime of adult life, a progressive course, and a combination of motor, cognitive, and behavioral features [<a href="#B5-jpm-14-00380" rid="B5-jpm-14-00380" class=" bibr popnode">5</a>]. Motor dysfunction normally manifests as involuntary choreiform movements, which may progress to dystonia, bradykinesia, and akinetic-rigid symptoms in the advanced stages of the disease [<a href="#B6-jpm-14-00380" rid="B6-jpm-14-00380" class=" bibr popnode">6</a>]. Cognitive decline in HD encompasses deficits in executive function, attention, visuospatial processing, and working memory, culminating in profound dementia as the disease advances [<a href="#B7-jpm-14-00380" rid="B7-jpm-14-00380" class=" bibr popnode">7</a>]. Additionally, behavioral features, such as depression, anxiety, irritability, and apathy, are prevalent throughout the disease [<a href="#B8-jpm-14-00380" rid="B8-jpm-14-00380" class=" bibr popnode">8</a>].</p><p>Overall, HD remains a challenging condition with limited therapeutic options [<a href="#B9-jpm-14-00380" rid="B9-jpm-14-00380" class=" bibr popnode">9</a>]. At the neuropathological level, HD is characterized by the selective degeneration of neurons within the striatum, particularly the medium spiny neurons of the caudate nucleus and putamen [<a href="#B10-jpm-14-00380" rid="B10-jpm-14-00380" class=" bibr popnode">10</a>]. This neurodegeneration is accompanied by widespread atrophy of the cerebral cortex, thalamus, and other subcortical structures, leading to global alterations in brain structure and function. The underlying molecular mechanisms driving neuronal dysfunction and death in HD involve a complex interplay of genetic, epigenetic, and environmental factors [<a href="#B11-jpm-14-00380" rid="B11-jpm-14-00380" class=" bibr popnode">11</a>]. An expansion of CAG trinucleotide causes HD repeats within the <em>huntingtin</em> gene located on chromosome 4p16.3, leading to an elongated polyglutamine tract in the huntingtin protein [<a href="#B12-jpm-14-00380" rid="B12-jpm-14-00380" class=" bibr popnode">12</a>]. The length of the CAG repeat tract inversely correlates with the age of disease onset, with longer repeats typically associated with earlier onset and more severe clinical phenotypes. While the expanded CAG repeat is the primary genetic determinant of HD, numerous modifier genes and environmental factors influence disease penetrance, expressivity, and progression [<a href="#B13-jpm-14-00380" rid="B13-jpm-14-00380" class=" bibr popnode">13</a>].</p><p>Despite decades of dedicated research, the precise mechanisms governing HD pathogenesis remain incompletely elucidated [<a href="#B14-jpm-14-00380" rid="B14-jpm-14-00380" class=" bibr popnode">14</a>]. There is an urgent need for disease-modifying therapies that can halt or slow the progression of HD, making elucidating disease mechanisms and identifying therapeutic targets a top priority in HD research. In recent years, advances in genomic technologies, neuroimaging techniques, and biomarker discovery have provided unprecedented insights into the pathobiology of HD [<a href="#B15-jpm-14-00380" rid="B15-jpm-14-00380" class=" bibr popnode">15</a>]. Genome-wide association studies (GWASs) have identified genetic variants associated with HD onset, progression, and phenotypic variability [<a href="#B16-jpm-14-00380" rid="B16-jpm-14-00380" class=" bibr popnode">16</a>,<a href="#B17-jpm-14-00380" rid="B17-jpm-14-00380" class=" bibr popnode">17</a>]. Cerebrospinal fluid (CSF) has emerged as a promising arena for biomarker discovery that offers glimpses into the molecular complexities of various neurodegenerative diseases, including HD [<a href="#B18-jpm-14-00380" rid="B18-jpm-14-00380" class=" bibr popnode">18</a>]. CSF serves as a reservoir of proteins, metabolites, and other molecules that mirror the pathophysiological alterations occurring within the central nervous system (CNS) [<a href="#B19-jpm-14-00380" rid="B19-jpm-14-00380" class=" bibr popnode">19</a>]. Among the CSF proteins, immunoglobulin heavy constant gamma 1 (IGHG1) has been implicated in the disease process due to its elevated levels in HD [<a href="#B20-jpm-14-00380" rid="B20-jpm-14-00380" class=" bibr popnode">20</a>,<a href="#B21-jpm-14-00380" rid="B21-jpm-14-00380" class=" bibr popnode">21</a>,<a href="#B22-jpm-14-00380" rid="B22-jpm-14-00380" class=" bibr popnode">22</a>].</p><p>Recent studies have shifted towards the genetic foundations of HD that explore the potential influence of deleterious mutations in driving disease pathogenesis [<a href="#B23-jpm-14-00380" rid="B23-jpm-14-00380" class=" bibr popnode">23</a>,<a href="#B24-jpm-14-00380" rid="B24-jpm-14-00380" class=" bibr popnode">24</a>]. The <em>IGHG1</em> gene that encodes the gamma-1 chain of the immunoglobulin G (IgG) has emerged as a potential target, with emerging evidence suggesting its involvement in HD etiology [<a href="#B21-jpm-14-00380" rid="B21-jpm-14-00380" class=" bibr popnode">21</a>,<a href="#B22-jpm-14-00380" rid="B22-jpm-14-00380" class=" bibr popnode">22</a>]. Understanding the effects of amino acid substitutions within a protein is vital for elucidating their contributions to disease pathogenesis and identifying potential therapeutic targets [<a href="#B25-jpm-14-00380" rid="B25-jpm-14-00380" class=" bibr popnode">25</a>,<a href="#B26-jpm-14-00380" rid="B26-jpm-14-00380" class=" bibr popnode">26</a>,<a href="#B27-jpm-14-00380" rid="B27-jpm-14-00380" class=" bibr popnode">27</a>].</p><p>In this study, we analyzed deleterious mutations within IGHG1 and their association with HD pathogenesis in detail. We explored amino acid substitutions&#x02019; structural and functional consequences on the IGHG1 protein while exploiting advanced computational tools and bioinformatics resources. Through an extensive examination of 126 single-point variations, we aimed to identify &#x0201c;high-confidence variants&#x0201d; in <em>IGHG1</em> with significant implications for HD pathophysiology.</p><p class="p p-last">Our approach encompasses a multifaceted analysis that integrates predictions from multiple computational platforms such as SIFT [<a href="#B28-jpm-14-00380" rid="B28-jpm-14-00380" class=" bibr popnode">28</a>], PolyPhen-2 [<a href="#B29-jpm-14-00380" rid="B29-jpm-14-00380" class=" bibr popnode">29</a>], FATHMM [<a href="#B30-jpm-14-00380" rid="B30-jpm-14-00380" class=" bibr popnode">30</a>], SNPs&#x00026;Go [<a href="#B31-jpm-14-00380" rid="B31-jpm-14-00380" class=" bibr popnode">31</a>], mCSM [<a href="#B32-jpm-14-00380" rid="B32-jpm-14-00380" class=" bibr popnode">32</a>], DynaMut2 [<a href="#B33-jpm-14-00380" rid="B33-jpm-14-00380" class=" bibr popnode">33</a>], MAESTROweb [<a href="#B34-jpm-14-00380" rid="B34-jpm-14-00380" class=" bibr popnode">34</a>], PremPS [<a href="#B35-jpm-14-00380" rid="B35-jpm-14-00380" class=" bibr popnode">35</a>], MutPred2 [<a href="#B36-jpm-14-00380" rid="B36-jpm-14-00380" class=" bibr popnode">36</a>], and PhD-SNP [<a href="#B37-jpm-14-00380" rid="B37-jpm-14-00380" class=" bibr popnode">37</a>]. By studying the impact of these variants on protein structure, stability, and function, we seek to delineate their potential role in the disrupting IGHG1-mediated processes implicated in HD pathogenesis. We focus on variants localized within specific domains of the IGHG1 protein, particularly the Ig-like 1 domain, which plays a crucial role in protein-protein interactions and immune function. By pinpointing mutations with a propensity to impair protein solubility and disrupt essential protein dynamics, we aim to elucidate their contribution to HD pathology. Ultimately, our findings promise to uncover novel insights into the molecular mechanisms driving HD. By elucidating the role of deleterious variants in <em>IGHG1</em> and their association with HD, we aim to lay the foundation for developing targeted therapies and precision medicine approaches to mitigate the devastating impact of this debilitating disorder.</p></div><div id="sec2-jpm-14-00380" class="tsec sec"><h2 class="head no_bottom_margin" id="sec2-jpm-14-00380title">2. Materials and Methods</h2><div id="sec2dot1-jpm-14-00380" class="sec sec-first"><h3 id="sec2dot1-jpm-14-00380title">2.1. Data Retrieval and Web Resources</h3><p class="p p-first-last">The protein sequence corresponding to human IGHG1 was sourced from the UniProt protein database (Accession ID: <a id="__tag_1147418299" class="tag_hotlink" href="/protein/P01857" ref="/protein/P01857"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"P01857","term_id":"2500440404","term_text":"P01857"}}</span>P01857</a>) [<a href="#B38-jpm-14-00380" rid="B38-jpm-14-00380" class=" bibr popnode">38</a>]. A comprehensive compilation of individual amino acid substitutions within IGHG1 was curated by accessing data from dbSNP [<a href="#B39-jpm-14-00380" rid="B39-jpm-14-00380" class=" bibr popnode">39</a>], Ensembl [<a href="#B40-jpm-14-00380" rid="B40-jpm-14-00380" class=" bibr popnode">40</a>], and the relevant PubMed literature [<a href="#B41-jpm-14-00380" rid="B41-jpm-14-00380" class=" bibr popnode">41</a>] (<a href="#app1-jpm-14-00380" rid="app1-jpm-14-00380" class=" app">Supplementary Data</a>). To ensure data integrity, duplicates were carefully removed from the variant dataset. The three-dimensional structure of human IGHG1, specifically its actin-binding domain, was retrieved from the RCSB Protein Data Bank (PDB ID: 4LLD). Various computational tools were employed to conduct diverse predictions and calculations, with detailed discussions provided in the subsequent sections.</p></div><div id="sec2dot2-jpm-14-00380" class="sec"><h3 id="sec2dot2-jpm-14-00380title">2.2. Sequence-Based Predictions</h3><div id="sec2dot2dot1-jpm-14-00380" class="sec sec-first"><p></p><h4 id="sec2dot2dot1-jpm-14-00380title" class="inline">2.2.1. PolyPhen2 </h4><p class="p p-first-last">PolyPhen-2 (Polymorphism Phenotyping v2) is a bioinformatics tool that forecasts the potential effects of amino acid substitutions on protein structure and functionality [<a href="#B29-jpm-14-00380" rid="B29-jpm-14-00380" class=" bibr popnode">29</a>]. By utilizing this resource, we assessed the impact of amino acid substitutions within the IGHG1 protein sequence. It evaluates these substitutions by integrating both the comparative and physical attributes of amino acids and estimates the likelihood of these substitutions disrupting the native conformation of the protein. Utilizing a combination of sequence, structural, and evolutionary data, PolyPhen-2 employs machine learning algorithms trained on a vast dataset of known polymorphisms to categorize variants into distinct categories based on their predicted impact, such as &#x0201c;benign&#x0201d;, &#x0201c;possibly damaging&#x0201d;, or &#x0201c;probably damaging&#x0201d;.</p></div><div id="sec2dot2dot2-jpm-14-00380" class="sec"><p></p><h4 id="sec2dot2dot2-jpm-14-00380title" class="inline">2.2.2. SIFT </h4><p class="p p-first-last">SIFT (sorting intolerant from tolerant) is a bioinformatics tool utilized in computational biology and genetics to predict the potential impact of amino acid substitutions on protein function [<a href="#B28-jpm-14-00380" rid="B28-jpm-14-00380" class=" bibr popnode">28</a>]. Our study employed SIFT to prioritize genetic variants within the IGHG1 protein sequence. Like PolyPhen-2, SIFT aids in prioritizing genetic variants by classifying them as either tolerated or intolerant based on their effect on protein structure and function. It achieves this by comparing the amino acid at the position of interest with those found in related proteins across diverse species. SIFT assigns a score to each amino acid substitution, with lower scores indicating a higher likelihood of being deleterious. Variants are classified as intolerable if the SIFT score is 0.05 or lower.</p></div><div id="sec2dot2dot3-jpm-14-00380" class="sec"><p></p><h4 id="sec2dot2dot3-jpm-14-00380title" class="inline">2.2.3. FATHMM </h4><p class="p p-first-last">FATHMM (functional analysis through hidden Markov models) is a bioinformatics tool for predicting the functional impacts of genetic variants, particularly those occurring in the coding regions of the genome [<a href="#B30-jpm-14-00380" rid="B30-jpm-14-00380" class=" bibr popnode">30</a>]. In our study, we harnessed FATHMM to assess the potential impact of amino acid substitutions within the IGHG1 protein. By employing a hidden Markov model (HMM) approach, FATHMM assesses the potential impact of variants on protein structure and function. It evaluates variants based on diverse features, encompassing evolutionary conservation, the physicochemical properties of amino acids, and genomic context. By amalgamating these features into its model, FATHMM predicts the potential functional impact of a given variant. The output of FATHMM typically includes a score or prediction, indicating whether a genetic variant is likely to be deleterious or tolerated. A high FATHMM score suggests that the variant is likely to be tolerated. In contrast, a low score indicates that the variant may have functional consequences and could be associated with diseases.</p></div><div id="sec2dot2dot4-jpm-14-00380" class="sec sec-last"><p></p><h4 id="sec2dot2dot4-jpm-14-00380title" class="inline">2.2.4. SNPs&#x00026;GO </h4><p class="p p-first-last">SNPs&#x00026;GO is a web server that employs a support vector machine (SVM) for identifying deleterious single-point amino acid substitutions [<a href="#B31-jpm-14-00380" rid="B31-jpm-14-00380" class=" bibr popnode">31</a>]. Here, we utilized SNPs&#x00026;GO to predict the impact of amino acid substitutions within the IGHG1 protein sequence. The SVM classifier amalgamates protein sequence, profile, and functional data to discriminate between disease-associated and neutral variants using gene ontology (GO) annotations. SNPs&#x00026;GO harnesses machine learning algorithms trained on various features derived from protein sequence and structure to predict the impact of SNPs. These features encompass amino acid physicochemical properties, evolutionary conservation, protein domain information, and structural annotations. An SNPs&#x00026;GO score exceeding 0.5 signifies a substitution likely to induce disease. Additionally, the tool yields output from supplementary resources such as PANTHER and PhD-SNP.</p></div></div><div id="sec2dot3-jpm-14-00380" class="sec"><h3 id="sec2dot3-jpm-14-00380title">2.3. Structure-Based Predictions</h3><div id="sec2dot3dot1-jpm-14-00380" class="sec sec-first"><p></p><h4 id="sec2dot3dot1-jpm-14-00380title" class="inline">2.3.1. mCSM </h4><p class="p p-first-last">mCSM (mutation cutoff scanning matrix) is a web tool utilized in structural bioinformatics to predict the effects of variants on protein stability [<a href="#B32-jpm-14-00380" rid="B32-jpm-14-00380" class=" bibr popnode">32</a>]. It operates through a graph-based method to evaluate single-point amino acid substitutions. The predictive models of mCSM are developed by leveraging environmental data gleaned from the atomic distance patterns of diverse residues. This tool enhances our comprehension of disease-associated variants spanning a spectrum of proteins. A mCSM score (&#x00394;&#x00394;<em>G</em>) below 0 indicates a mutation that profoundly influences protein structure.</p></div><div id="sec2dot3dot2-jpm-14-00380" class="sec"><p></p><h4 id="sec2dot3dot2-jpm-14-00380title" class="inline">2.3.2. MAESTROweb </h4><p class="p p-first-last">MAESTROweb, available at <a href="https://pbwww.services.came.sbg.ac.at/maestro/web">https://pbwww.services.came.sbg.ac.at/maestro/web</a> (accessed on 15 January 2024), is an online platform facilitating protein structure-based prediction of mutation effects [<a href="#B34-jpm-14-00380" rid="B34-jpm-14-00380" class=" bibr popnode">34</a>]. It is used to discern how variants impact protein stability. MAESTROweb predicts the functional ramifications of genetic variations, especially concerning human diseases. It amalgamates several computational tools to scrutinize the impact of variants on protein stability. By employing diverse computational algorithms and methods, including machine learning techniques and structural bioinformatics approaches, MAESTROweb provides insights into the functional consequences of variants. A score below 0 denotes an anticipated alteration in the stability of protein due to the amino acid substitutions.</p></div><div id="sec2dot3dot3-jpm-14-00380" class="sec"><p></p><h4 id="sec2dot3dot3-jpm-14-00380title" class="inline">2.3.3. PremPS </h4><p class="p p-first-last">PremPS ((<a href="https://lilab.jysw.suda.edu.cn/research/PremPS">https://lilab.jysw.suda.edu.cn/research/PremPS</a>) accessed on 15 January 2024), also evaluates the impact of amino acid substitutions in proteins utilizing multiple approaches such as multiple sequence alignment (MSA), protein structure, and a deep learning model [<a href="#B35-jpm-14-00380" rid="B35-jpm-14-00380" class=" bibr popnode">35</a>]. It offers predictions concerning the potential impact of genetic variants on protein function and disease, specifically focusing on whether a single amino acid substitution is likely to be harmful or tolerated based on the sequence information of the protein. Users input the amino acid sequence of the protein, specify the position and the amino acid substitution, and PremPS calculates a prediction score, indicating the likelihood of the substitution being deleterious or tolerated.</p></div><div id="sec2dot3dot4-jpm-14-00380" class="sec sec-last"><p></p><h4 id="sec2dot3dot4-jpm-14-00380title" class="inline">2.3.4. DynaMut2 </h4><p class="p p-first-last">DynaMut2 (<a href="http://biosig.unimelb.edu.au/dynamut2">http://biosig.unimelb.edu.au/dynamut2</a>, accessed on 15 January 2024) is another predictive tool tailored for estimating protein stability [<a href="#B33-jpm-14-00380" rid="B33-jpm-14-00380" class=" bibr popnode">33</a>]. By exploiting the data sourced from the ProTherm database, DynaMut2 furnishes predictions for both single and multiple amino acid substitutions. Our analysis primarily focuses on its single mutation prediction capability. DynaMut2 provides users with easily interpretable results, encompassing predictions of the impact of amino acid substitution on stability and dynamic properties, along with visualizations that aid understanding of the structural context of the variants.</p></div></div><div id="sec2dot4-jpm-14-00380" class="sec"><h3 id="sec2dot4-jpm-14-00380title">2.4. Pathogenicity Prediction</h3><div id="sec2dot4dot1-jpm-14-00380" class="sec sec-first"><p></p><h4 id="sec2dot4dot1-jpm-14-00380title" class="inline">2.4.1. PhD-SNP </h4><p class="p p-first-last">PhD-SNP (<a href="https://snps.biofold.org/phd-snp/phd-snp.html">https://snps.biofold.org/phd-snp/phd-snp.html</a>, accessed on 15 January 2024) is a web-based pathogenicity analysis tool that leverages an SVM-based classifier to categorize variants associated with diseases [<a href="#B37-jpm-14-00380" rid="B37-jpm-14-00380" class=" bibr popnode">37</a>]. PhD-SNP distinguishes between neutral and disease-associated amino acid substitutions by employing sequence and profile information. It uses machine learning algorithms trained on various sequence, structural, and functional features to predict the functional consequences of SNPs. A PhD-SNP score exceeding 0.5 signifies an amino acid substitution likely to induce disease. In our study, we utilized PhD-SNP to evaluate the pathogenicity of mutations within IGHG1.</p></div><div id="sec2dot4dot2-jpm-14-00380" class="sec sec-last"><p></p><h4 id="sec2dot4dot2-jpm-14-00380title" class="inline">2.4.2. MutPred2 </h4><p class="p p-first-last">MutPred2 (<a href="http://mutpred.mutdb.org">http://mutpred.mutdb.org</a>, accessed on 15 January 2024) is another web-based tool tailored to classifying amino acid substitutions as either disease-associated or neutral [<a href="#B36-jpm-14-00380" rid="B36-jpm-14-00380" class=" bibr popnode">36</a>]. It aims to predict the likelihood of a given amino acid substitution being deleterious or neutral while offering insights into the potential molecular mechanisms underlying the predicted effects. MutPred2 considers various protein sequence and structure features, encompassing evolutionary conservation, the physicochemical properties of amino acids, protein domains, and structural annotations. A MutPred2 score surpassing 0.5 indicates a substitution deemed pathogenic. In our investigation, we employed MutPred2 to assess the pathogenicity of mutations within IGHG1.</p></div></div><div id="sec2dot5-jpm-14-00380" class="sec"><h3 id="sec2dot5-jpm-14-00380title">2.5. Aggregation Propensity Analysis</h3><p class="p p-first-last">Aggregation propensity analysis (of proteins) is a valuable approach to exploring how variants within a protein can influence its tendency to form aggregates [<a href="#B42-jpm-14-00380" rid="B42-jpm-14-00380" class=" bibr popnode">42</a>]. SODA (<a href="http://protein.bio.unipd.it/soda/">http://protein.bio.unipd.it/soda/</a>, accessed on 15 January 2024) emerged as a bioinformatics tool and web server specifically designed for predicting protein solubility based on intrinsic disorder and aggregation propensity. The tool accepts input files in either FASTA sequence or PDB structure format. SODA utilizes a combination of multiple algorithms, namely PASTA 2.0, ESpritz-NMR, and FELLS methods [<a href="#B43-jpm-14-00380" rid="B43-jpm-14-00380" class=" bibr popnode">43</a>]. It ultimately provides conclusive results by comparing the solubility disparity between the wild-type and mutant proteins. In our study, we utilized SODA to assess the aggregation propensity of mutations within <em>IGHG1</em>.</p></div><div id="sec2dot6-jpm-14-00380" class="sec sec-last"><h3 id="sec2dot6-jpm-14-00380title">2.6. Residual Frustration Analysis</h3><p class="p p-first-last">Residual frustration analysis in proteins provides valuable insight into the level of frustration present in the structure of a protein [<a href="#B44-jpm-14-00380" rid="B44-jpm-14-00380" class=" bibr popnode">44</a>]. To explore the enduring frustration within the structure of IGHG1, we utilized the Frustratometer server (<a href="http://frustratometer.qb.fcen.uba.ar/">http://frustratometer.qb.fcen.uba.ar/</a>, accessed on 15 January 2024) [<a href="#B45-jpm-14-00380" rid="B45-jpm-14-00380" class=" bibr popnode">45</a>]. By using this server, we calculated individual and configurational residual indices for the structure of IGHG1. The Frustratometer evaluates the energy of a protein structure by comparing it to a set of &#x02018;decoy&#x02019; states. Specifically, the residual frustration index between amino acids i and j is determined as a Z-score, comparing the energy of the native contact to that of N decoys. A contact is classified as highly frustrated or destabilizing if its Z-score is below 0.78. Conversely, a contact is minimally frustrated or stabilizing if the Z-score value exceeds 0.78. Contacts falling between these thresholds are considered neutral. The workflow employed in this study is depicted in <a href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f001/" target="figure" class="fig-table-link figpopup" rid-figpopup="jpm-14-00380-f001" rid-ob="ob-jpm-14-00380-f001" co-legend-rid="lgnd_jpm-14-00380-f001"><span>Figure 1</span></a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="jpm-14-00380-f001" co-legend-rid="lgnd_jpm-14-00380-f001"><a href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f001/" target="figure" rid-figpopup="jpm-14-00380-f001" rid-ob="ob-jpm-14-00380-f001"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140538171876992"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11050829_jpm-14-00380-g001.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is jpm-14-00380-g001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11050829/bin/jpm-14-00380-g001.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140538171876992"><a target="object" rel="noopener" href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f001/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_jpm-14-00380-f001"><div><a class="figpopup" href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f001/" target="figure" rid-figpopup="jpm-14-00380-f001" rid-ob="ob-jpm-14-00380-f001">Figure 1</a></div><!--caption a7--><div class="caption"><p>Schematic of the workflow pipeline employed in the present study. IGHG1, Immunoglobulin heavy constant gamma 1.</p></div></div></div></div></div><div id="sec3-jpm-14-00380" class="tsec sec"><h2 class="head no_bottom_margin" id="sec3-jpm-14-00380title">3. Results</h2><p class="p p-first">A set of 126 variations in <em>IGHG1</em> sourced from the dbSNP and Ensembl databases, complemented by mutations mined from the literature (available on PubMed), was evaluated (<a href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f002/" target="figure" class="fig-table-link figpopup" rid-figpopup="jpm-14-00380-f002" rid-ob="ob-jpm-14-00380-f002" co-legend-rid="lgnd_jpm-14-00380-f002"><span>Figure 2</span></a>). Our focus centered on evaluating these substitutions&#x02019; structural and functional impacts within the CH1 region/Ig-like 1 domain of the IGHG1 protein. Our integrated approach encompassed sequence- and structure-based analyses to discover high-confidence deleterious variants. The sequence-based analysis was facilitated by four web tools: SIFT, PolyPhen2, FATHMM, and SNPs&#x00026;GO. Concurrently, the structure-based approach involved utilizing mCSM, DynaMut2, MAESTROweb, and PremPS to evaluate single-point amino acid substitutions in IGHG1. Subsequently, only those mutations identified as high-confidence variants were subjected to further investigation. In order to elucidate the disease phenotypes associated with these high-confidence variants, we exploited the PhD-SNP and MutPred2 web servers, as discussed in the ensuing sections.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="jpm-14-00380-f002" co-legend-rid="lgnd_jpm-14-00380-f002"><a href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f002/" target="figure" rid-figpopup="jpm-14-00380-f002" rid-ob="ob-jpm-14-00380-f002"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140538181425808"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11050829_jpm-14-00380-g002.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is jpm-14-00380-g002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11050829/bin/jpm-14-00380-g002.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140538181425808"><a target="object" rel="noopener" href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f002/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_jpm-14-00380-f002"><div><a class="figpopup" href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f002/" target="figure" rid-figpopup="jpm-14-00380-f002" rid-ob="ob-jpm-14-00380-f002">Figure 2</a></div><!--caption a7--><div class="caption"><p>Visualization showing the single nucleotide polymorphisms (SNPs) identified within the IGHG1 gene using the dbSNP database.</p></div></div></div><div id="sec3dot1-jpm-14-00380" class="sec"><h3 id="sec3dot1-jpm-14-00380title">3.1. Deleterious Mutations from Sequence and Structure-Based Approaches</h3><p class="p p-first">Integrating multiple prediction tools within the sequence-based approach minimizes false positive results and enhances the accuracy of mutation predictions [<a href="#B46-jpm-14-00380" rid="B46-jpm-14-00380" class=" bibr popnode">46</a>]. Among these tools, SIFT employs protein physical properties to categorize variants as tolerated or intolerant, with a higher tolerance index indicating a lower functional impact and vice versa. Similarly, PolyPhen2 utilizes amino acid sequences to classify non-synonymous variants into the possibly damaging, probably damaging, or benign categories based on specific scores. In order to further reinforce confidence levels, the inclusion of the FATHMM and SNPs&#x00026;GO tools augments the predictive capacity of the analysis. Mutations often associated with diseases have a notable impact on protein stability. Proteins can exist in either folded or unfolded states, and in thermodynamics, the Gibbs free energy between the folded (<em>G</em>f) and unfolded (<em>G</em>u) states of a protein is determined as &#x00394;<em>G</em> = <em>G</em>u &#x02212; <em>G</em>f. The alteration in protein stability and the free energy landscape is assessed by &#x00394;&#x00394;<em>G</em> = <em>G</em>m &#x02212; <em>G</em>w, where Gm represents the mutant protein and <em>G</em>w denotes the wild-type protein. A negative &#x00394;&#x00394;<em>G</em> value indicates a mutation that stabilizes the protein, while a positive &#x00394;&#x00394;<em>G</em> value suggests variants that destabilize it.</p><p>We employed four distinct structure-based prediction tools: mCSM, DynaMut2, MAESTROweb, and PremPS. These tools utilize the PDB file of the wild-type protein as input and analyze atomic co-ordinates to evaluate the stability of variants through folding free energy computation. Many of these tools employ a machine learning-centric methodology, integrating various biophysics-based techniques to forecast the influence of variants on protein stability. In the sequence-based approach, the analysis of all 126 single-point amino acid substitutions in IGHG1 yielded predictions from SIFT, PolyPhen2, FATHMM, and SNPs&#x00026;GO (<a href="#app1-jpm-14-00380" rid="app1-jpm-14-00380" class=" app">Supplementary Table S1</a>). Specifically, these tools identified 69, 24, 38, and 26 substitutions as deleterious (<a href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f003/" target="figure" class="fig-table-link figpopup" rid-figpopup="jpm-14-00380-f003" rid-ob="ob-jpm-14-00380-f003" co-legend-rid="lgnd_jpm-14-00380-f003"><span>Figure 3</span></a>). This amalgamation of sequence-based prediction tools allowed us to scrutinize the potential functional impact of mutations at the molecular level.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="jpm-14-00380-f003" co-legend-rid="lgnd_jpm-14-00380-f003"><a href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f003/" target="figure" rid-figpopup="jpm-14-00380-f003" rid-ob="ob-jpm-14-00380-f003"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140538181172656"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11050829_jpm-14-00380-g003.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is jpm-14-00380-g003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11050829/bin/jpm-14-00380-g003.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140538181172656"><a target="object" rel="noopener" href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f003/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_jpm-14-00380-f003"><div><a class="figpopup" href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f003/" target="figure" rid-figpopup="jpm-14-00380-f003" rid-ob="ob-jpm-14-00380-f003">Figure 3</a></div><!--caption a7--><div class="caption"><p>Deleterious mutations in Immunoglobulin heavy constant gamma 1predicted through sequence-based tools.</p></div></div></div><p class="p p-last">Simultaneously, the structure-based predictions from mCSM, DynaMut2, MAESTROweb, and PremPS identified 108, 116, 114, and 88 substitutions as destabilizing mutations (<a href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f004/" target="figure" class="fig-table-link figpopup" rid-figpopup="jpm-14-00380-f004" rid-ob="ob-jpm-14-00380-f004" co-legend-rid="lgnd_jpm-14-00380-f004"><span>Figure 4</span></a>). In order to bolster confidence in our findings, only mutations predicted as deleterious by all sequence-based and structure-based tools were selected for further analysis. This filtering process resulted in 10 amino acid substitutions predicted as deleterious and destabilizing (<a href="#app1-jpm-14-00380" rid="app1-jpm-14-00380" class=" app">Supplementary Table S2</a>). Subsequently, these 10 substitutions underwent analysis to investigate their association with disease phenotypes.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="jpm-14-00380-f004" co-legend-rid="lgnd_jpm-14-00380-f004"><a href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f004/" target="figure" rid-figpopup="jpm-14-00380-f004" rid-ob="ob-jpm-14-00380-f004"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140538181169936"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11050829_jpm-14-00380-g004.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is jpm-14-00380-g004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11050829/bin/jpm-14-00380-g004.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140538181169936"><a target="object" rel="noopener" href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f004/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_jpm-14-00380-f004"><div><a class="figpopup" href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f004/" target="figure" rid-figpopup="jpm-14-00380-f004" rid-ob="ob-jpm-14-00380-f004">Figure 4</a></div><!--caption a7--><div class="caption"><p>Destabilizing mutations in Immunoglobulin heavy constant gamma 1 predicted through structure-based tools.</p></div></div></div></div><div id="sec3dot2-jpm-14-00380" class="sec"><h3 id="sec3dot2-jpm-14-00380title">3.2. Identification of Disease-Associated Mutations</h3><p class="p p-first">Analyzing disease-associated protein mutations constitutes a crucial aspect of unraveling the molecular underpinnings of various complex diseases. In our investigation of disease-associated single-point mutations, we employed PhD-SNP and MutPred2. These methodologies classify mutations based on their pathogenicity scores and delineate associated disease phenotypes (<a href="#app1-jpm-14-00380" rid="app1-jpm-14-00380" class=" app">Supplementary Table S3</a>). Out of the 10 mutations analyzed, eight mutations (Y32C, Y32D, P34S, V39E, C83R, C83Y, V85M, and H87Q) were identified as pathogenic by both disease phenotype prediction tools (<a href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f005/" target="figure" class="fig-table-link figpopup" rid-figpopup="jpm-14-00380-f005" rid-ob="ob-jpm-14-00380-f005" co-legend-rid="lgnd_jpm-14-00380-f005"><span>Figure 5</span></a>). This overlapping subset of mutations signifies those with a heightened likelihood of being associated with disease phenotypes, as affirmed by the consensus prediction from both tools.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="jpm-14-00380-f005" co-legend-rid="lgnd_jpm-14-00380-f005"><a href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f005/" target="figure" rid-figpopup="jpm-14-00380-f005" rid-ob="ob-jpm-14-00380-f005"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140538186986976"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11050829_jpm-14-00380-g005.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is jpm-14-00380-g005.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11050829/bin/jpm-14-00380-g005.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140538186986976"><a target="object" rel="noopener" href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f005/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_jpm-14-00380-f005"><div><a class="figpopup" href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f005/" target="figure" rid-figpopup="jpm-14-00380-f005" rid-ob="ob-jpm-14-00380-f005">Figure 5</a></div><!--caption a7--><div class="caption"><p>Pathogenic mutations in Immunoglobulin heavy constant gamma 1 predicted using structure-based tools.</p></div></div></div><p class="p p-last">Moreover, we predicted the aggregation propensity of the protein induced by these mutations. The solubility of a protein significantly influences its functionality, as insoluble regions tend to aggregate, potentially exacerbating disease progression [<a href="#B42-jpm-14-00380" rid="B42-jpm-14-00380" class=" bibr popnode">42</a>]. In order to evaluate the solubility of protein variants and discern their association with disease, we employed SODA [<a href="#B47-jpm-14-00380" rid="B47-jpm-14-00380" class=" bibr popnode">47</a>]. SODA assesses various tendencies, including aggregation, disorder, helix, and strand tendencies resulting from mutations. Of the eight deleterious single-point amino acid substitutions identified via disease phenotype prediction, two substitutions (Y32C and P34S) were found to decrease the solubility of the protein (<a href="#app1-jpm-14-00380" rid="app1-jpm-14-00380" class=" app">Supplementary Table S4</a>).</p></div><div id="sec3dot3-jpm-14-00380" class="sec sec-last"><h3 id="sec3dot3-jpm-14-00380title">3.3. Residual Frustration Analysis</h3><p class="p p-first-last">Residual frustration analysis provides invaluable insights into the intricate energy landscapes of proteins [<a href="#B44-jpm-14-00380" rid="B44-jpm-14-00380" class=" bibr popnode">44</a>]. It offers a deeper understanding of the interplay between protein structure, stability, and function. By analyzing frustration within protein structures, specific locations of frustration can be identified, shedding light on critical aspects of protein behavior. Here, we conducted a structure-based analysis of local frustration within IGHG1 to discern its implications on protein stability and function (<a href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f006/" target="figure" class="fig-table-link figpopup" rid-figpopup="jpm-14-00380-f006" rid-ob="ob-jpm-14-00380-f006" co-legend-rid="lgnd_jpm-14-00380-f006"><span>Figure 6</span></a>). Frustration indices offer insights into the relative stability of native contacts compared to all potential contacts at specific locations (<a href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f006/" target="figure" class="fig-table-link figpopup" rid-figpopup="jpm-14-00380-f006" rid-ob="ob-jpm-14-00380-f006" co-legend-rid="lgnd_jpm-14-00380-f006"><span>Figure 6</span></a>A). Our analysis unveiled varying degrees of frustration within IGHG1, with the contact map showcasing significant alterations in frustration patterns in both mutants (<a href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f006/" target="figure" class="fig-table-link figpopup" rid-figpopup="jpm-14-00380-f006" rid-ob="ob-jpm-14-00380-f006" co-legend-rid="lgnd_jpm-14-00380-f006"><span>Figure 6</span></a>B,C). Particularly, the structure exhibited moderate frustration at various sites on the protein.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="jpm-14-00380-f006" co-legend-rid="lgnd_jpm-14-00380-f006"><a href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f006/" target="figure" rid-figpopup="jpm-14-00380-f006" rid-ob="ob-jpm-14-00380-f006"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140538186982832"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11050829_jpm-14-00380-g006.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is jpm-14-00380-g006.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11050829/bin/jpm-14-00380-g006.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140538186982832"><a target="object" rel="noopener" href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f006/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_jpm-14-00380-f006"><div><a class="figpopup" href="/pmc/articles/PMC11050829/figure/jpm-14-00380-f006/" target="figure" rid-figpopup="jpm-14-00380-f006" rid-ob="ob-jpm-14-00380-f006">Figure 6</a></div><!--caption a7--><div class="caption"><p>Residual frustration maps in (<strong>A</strong>) WT IGHG1 (<strong>B</strong>) Y32C and (<strong>C</strong>) P34S. The upper panel shows the 3D structure of Immunoglobulin heavy constant gamma 1 with a frustration index. The middle panel shows the residue&#x02013;residue contact level in Immunoglobulin heavy constant gamma 1. The lower panel shows the frustration contact map in Immunoglobulin heavy constant gamma 1.</p></div></div></div></div></div><div id="sec4-jpm-14-00380" class="tsec sec"><h2 class="head no_bottom_margin" id="sec4-jpm-14-00380title">4. Discussion</h2><p class="p p-first">HD remains a persistent challenge within the landscape of neurodegenerative disorders [<a href="#B48-jpm-14-00380" rid="B48-jpm-14-00380" class=" bibr popnode">48</a>]. The molecular complexity of HD is substantial, and despite significant progress in understanding its molecular and cellular mechanisms, effective therapeutics are not yet available [<a href="#B49-jpm-14-00380" rid="B49-jpm-14-00380" class=" bibr popnode">49</a>]. Studies have suggested that CSF could be a valuable source of biomarkers for HD [<a href="#B18-jpm-14-00380" rid="B18-jpm-14-00380" class=" bibr popnode">18</a>,<a href="#B21-jpm-14-00380" rid="B21-jpm-14-00380" class=" bibr popnode">21</a>]. IGHG1 is one of the proteins found in CSF and has been found to increase significantly in HD [<a href="#B22-jpm-14-00380" rid="B22-jpm-14-00380" class=" bibr popnode">22</a>]. Considering its critical role in HD, it is reasonable to study the potential involvement of deleterious mutations in IGHG1 in the pathogenesis of this disorder. This study explored the complex landscape of deleterious mutations within the IGHG1 protein and their possible implications in HD pathogenesis.</p><p>By using a multifaceted approach that integrates sequence-based and structure-based analyses, we identified a few mutations with a high likelihood of deleterious effects on protein structure and function. By exploiting multiple computational tools and bioinformatics algorithms, initially, we recognized 10 high-confidence mutations within IGHG1, characterized by their significant impact on protein stability and their potential association with disease phenotypes. The sequence-based approach ensured the identification of mutations with the highest likelihood of playing a significant role in disease pathology.</p><p>At the same time, the structure-based predictions enhanced our confidence in identifying mutations that potentially play important roles in disease manifestation and progression. This observation underscores the potential implications of these mutations in perturbing protein solubility and highlights their relevance in disease pathogenesis [<a href="#B50-jpm-14-00380" rid="B50-jpm-14-00380" class=" bibr popnode">50</a>]. Furthermore, our analysis revealed the aggregation propensity of IGHG1 variants, shedding light on their potential contribution to disease progression [<a href="#B51-jpm-14-00380" rid="B51-jpm-14-00380" class=" bibr popnode">51</a>].</p><p class="p p-last">Developing non-dissolvable protein aggregates is a hallmark of HD [<a href="#B52-jpm-14-00380" rid="B52-jpm-14-00380" class=" bibr popnode">52</a>]. Here, two substitutions (Y32C and P34S) were identified that decrease protein solubility, suggesting their potential role in protein IGHG1 and subsequent disease manifestation. Additionally, analyzing residual frustration in a protein structure is crucial in shedding light on the disease mechanism [<a href="#B53-jpm-14-00380" rid="B53-jpm-14-00380" class=" bibr popnode">53</a>]. The frustration analysis provided valuable insights into the complex energy landscapes of IGHG1, highlighting regions of potential instability that could influence protein function and contribute to disease pathology. The analysis showed that mutations occurring in the minimally frustrated residues of IGHG1 could disrupt stability, thereby affecting the function of the protein and potentially contributing to HD pathogenesis. This analysis underscores the importance of considering local frustration in elucidating the molecular mechanisms underlying disease pathology.</p></div><div id="sec5-jpm-14-00380" class="tsec sec"><h2 class="head no_bottom_margin" id="sec5-jpm-14-00380title">5. Conclusions</h2><p class="p p-first-last">Our findings highlight the importance of computational mutational analysis in unraveling the molecular basis of complex diseases such as HD. This study contributes to the growing body of knowledge on understanding the pathogenesis of HD and lays the groundwork for future research endeavors to develop targeted therapeutic interventions. Ultimately, the study emphasizes the significance of advanced computational approaches to gain deeper insights into the molecular mechanisms driving disease pathology.</p></div><div id="ack-a.l.a" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.l.atitle">Acknowledgments</h2><div class="sec"><p>The authors extend their appreciation to the King Salman Center for Disability Research for this work.</p></div></div><div id="app1-jpm-14-00380" class="tsec sec"><h2 class="head no_bottom_margin" id="app1-jpm-14-00380title">Supplementary Materials</h2><!--/article/back/app-group/app/--><p>The following supporting information can be downloaded at <a href="https://www.mdpi.com/article/10.3390/jpm14040380/s1">https://www.mdpi.com/article/10.3390/jpm14040380/s1</a>, Table S1: Prediction of deleterious mutations in IGHG1 using sequence-based tools; Table S2: Prediction of destabilizing mutations in IGHG1 using structure-based tools; Table S3: Prediction of pathogenic mutations in IGHG1 using structure-based tools; Table S4: Prediction of aggregation propensity of mutations in IGHG1.</p><div class="sec suppmat" id="jpm-14-00380-s001"><div class="sup-box half_rhythm" id="media-a.l.c.a.c.a"><a href="/pmc/articles/PMC11050829/bin/jpm-14-00380-s001.zip" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(129K, zip)</sup></div></div></div><div id="funding-group-a.j.b.u" class="tsec sec"><h2 class="head no_bottom_margin" id="funding-group-a.j.b.utitle">Funding Statement</h2><p>This research was funded by King Salman Center for Disability Research grant number KSRG-2023-242.</p></div><div id="notes-a.l.d" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.l.dtitle">Author Contributions</h2><p>Conceptualization, A.S. (Alaa Shafie) and M.I.H.; methodology, A.S. (Alaa Shafie) and A.A.A.; software, A.S. (Alaa Shafie); validation, F.A., A.S. (Anas Shamsi) and M.I.H.; formal analysis, A.S. (Alaa Shafie); investigation, M.I.H.; resources, A.S. (Anas Shamsi); data curation, F.A.; writing&#x02014;original draft preparation, A.S. (Alaa Shafie); writing&#x02014;review and editing, F.A.; visualization, A.S. (Anas Shamsi); supervision, M.I.H.; project administration, M.I.H.; funding acquisition, A.S. (Alaa Shafie). All authors have read and agreed to the published version of the manuscript.</p></div><div id="notes-a.l.e" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.l.etitle">Institutional Review Board Statement</h2><p>Not applicable.</p></div><div id="notes-a.l.f" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.l.ftitle">Informed Consent Statement</h2><p>Not applicable.</p></div><div id="notes-a.l.g" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.l.gtitle">Data Availability Statement</h2><p>The original contributions presented in the study are included in the article/<a href="#app1-jpm-14-00380" rid="app1-jpm-14-00380" class=" app">Supplementary Materials</a>; further inquiries can be directed to the corresponding author.</p></div><div id="notes-a.l.h" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.l.htitle">Conflicts of Interest</h2><p>The authors declare no conflicts of interest.</p></div><div id="fn-group-a.l.b" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.l.btitle">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="fn-a.l.b.a"><p class="p p-first-last"><strong>Disclaimer/Publisher&#x02019;s Note:</strong> The statements, opinions and data contained in all publications are solely those of the individual author(s) and contributor(s) and not of MDPI and/or the editor(s). MDPI and/or the editor(s) disclaim responsibility for any injury to people or property resulting from any ideas, methods, instructions or products referred to in the content.</p></p></div></div><div id="ref-list-a.l.i" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.l.ititle">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1-jpm-14-00380">1. <span class="element-citation">Walker F.O. Huntington&#x02019;s disease. <span><span class="ref-journal">Lancet. </span>2007;<span class="ref-vol">369</span>:218–228. doi: 10.1016/S0140-6736(07)60111-1.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17240289" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FS0140-6736(07)60111-1" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Huntington&#x02019;s+disease&amp;author=F.O.+Walker&amp;volume=369&amp;publication_year=2007&amp;pages=218-228&amp;pmid=17240289&amp;doi=10.1016/S0140-6736(07)60111-1&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B2-jpm-14-00380">2. <span class="element-citation">Dayalu P., Albin R.L. Huntington disease: Pathogenesis and treatment. <span><span class="ref-journal">Neurol. Clin. </span>2015;<span class="ref-vol">33</span>:101–114. doi: 10.1016/j.ncl.2014.09.003.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25432725" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.ncl.2014.09.003" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurol.+Clin.&amp;title=Huntington+disease:+Pathogenesis+and+treatment&amp;author=P.+Dayalu&amp;author=R.L.+Albin&amp;volume=33&amp;publication_year=2015&amp;pages=101-114&amp;pmid=25432725&amp;doi=10.1016/j.ncl.2014.09.003&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B3-jpm-14-00380">3. <span class="element-citation">Medina A., Mahjoub Y., Shaver L., Pringsheim T. Prevalence and incidence of Huntington&#x02019;s disease: An updated systematic review and meta-analysis. <span><span class="ref-journal">Mov. Disord. </span>2022;<span class="ref-vol">37</span>:2327–2335. doi: 10.1002/mds.29228.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10086981/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36161673" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fmds.29228" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mov.+Disord.&amp;title=Prevalence+and+incidence+of+Huntington&#x02019;s+disease:+An+updated+systematic+review+and+meta-analysis&amp;author=A.+Medina&amp;author=Y.+Mahjoub&amp;author=L.+Shaver&amp;author=T.+Pringsheim&amp;volume=37&amp;publication_year=2022&amp;pages=2327-2335&amp;pmid=36161673&amp;doi=10.1002/mds.29228&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B4-jpm-14-00380">4. <span class="element-citation">Kinnunen K.M., Schwarz A.J., Turner E.C., Pustina D., Gantman E.C., Gordon M.F., Joules R., Mullin A.P., Scahill R.I., Georgiou-Karistianis N. Volumetric MRI-based biomarkers in Huntington&#x02019;s disease: An evidentiary review. <span><span class="ref-journal">Front. Neurol. </span>2021;<span class="ref-vol">12</span>:712555. doi: 10.3389/fneur.2021.712555.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8490802/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34621236" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3389%2Ffneur.2021.712555" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Neurol.&amp;title=Volumetric+MRI-based+biomarkers+in+Huntington&#x02019;s+disease:+An+evidentiary+review&amp;author=K.M.+Kinnunen&amp;author=A.J.+Schwarz&amp;author=E.C.+Turner&amp;author=D.+Pustina&amp;author=E.C.+Gantman&amp;volume=12&amp;publication_year=2021&amp;pages=712555&amp;pmid=34621236&amp;doi=10.3389/fneur.2021.712555&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B5-jpm-14-00380">5. <span class="element-citation">Bates G.P., Dorsey R., Gusella J.F., Hayden M.R., Kay C., Leavitt B.R., Nance M., Ross C.A., Scahill R.I., Wetzel R. Huntington disease. <span><span class="ref-journal">Nat. Rev. Dis. Primers. </span>2015;<span class="ref-vol">1</span>:1–21. doi: 10.1038/nrdp.2015.5.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27188817" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnrdp.2015.5" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Dis.+Primers&amp;title=Huntington+disease&amp;author=G.P.+Bates&amp;author=R.+Dorsey&amp;author=J.F.+Gusella&amp;author=M.R.+Hayden&amp;author=C.+Kay&amp;volume=1&amp;publication_year=2015&amp;pages=1-21&amp;pmid=27188817&amp;doi=10.1038/nrdp.2015.5&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B6-jpm-14-00380">6. <span class="element-citation">Sanchez-Pernaute R., Kunig G., del Barrio Alba A., De Yebenes J., Vontobel P., Leenders K. Bradykinesia in early Huntington&#x02019;s disease. <span><span class="ref-journal">Neurology. </span>2000;<span class="ref-vol">54</span>:119. doi: 10.1212/WNL.54.1.119.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10636136" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1212%2FWNL.54.1.119" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Bradykinesia+in+early+Huntington&#x02019;s+disease&amp;author=R.+Sanchez-Pernaute&amp;author=G.+Kunig&amp;author=A.+del+Barrio+Alba&amp;author=J.+De+Yebenes&amp;author=P.+Vontobel&amp;volume=54&amp;publication_year=2000&amp;pages=119&amp;pmid=10636136&amp;doi=10.1212/WNL.54.1.119&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B7-jpm-14-00380">7. <span class="element-citation">Dumas E.M., Van Den Bogaard S., Middelkoop H., Roos R. A review of cognition in Huntington&#x02019;s disease. <span><span class="ref-journal">Front. Biosci. (Sch. Ed.) </span>2013;<span class="ref-vol">5</span>:1–18.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23277034" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Biosci.+(Sch.+Ed.)&amp;title=A+review+of+cognition+in+Huntington&#x02019;s+disease&amp;author=E.M.+Dumas&amp;author=S.+Van+Den+Bogaard&amp;author=H.+Middelkoop&amp;author=R.+Roos&amp;volume=5&amp;publication_year=2013&amp;pages=1-18&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B8-jpm-14-00380">8. <span class="element-citation">Craufurd D., Thompson J.C., Snowden J.S. Behavioral changes in Huntington disease. <span><span class="ref-journal">Cogn. Behav. Neurol. </span>2001;<span class="ref-vol">14</span>:219–226.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11725215" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cogn.+Behav.+Neurol.&amp;title=Behavioral+changes+in+Huntington+disease&amp;author=D.+Craufurd&amp;author=J.C.+Thompson&amp;author=J.S.+Snowden&amp;volume=14&amp;publication_year=2001&amp;pages=219-226&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B9-jpm-14-00380">9. <span class="element-citation">Zielonka D., Mielcarek M., Landwehrmeyer G.B. Update on Huntington&#x02019;s disease: Advances in care and emerging therapeutic options. <span><span class="ref-journal">Park. Relat. Disord. </span>2015;<span class="ref-vol">21</span>:169–178. doi: 10.1016/j.parkreldis.2014.12.013.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25572500" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.parkreldis.2014.12.013" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Park.+Relat.+Disord.&amp;title=Update+on+Huntington&#x02019;s+disease:+Advances+in+care+and+emerging+therapeutic+options&amp;author=D.+Zielonka&amp;author=M.+Mielcarek&amp;author=G.B.+Landwehrmeyer&amp;volume=21&amp;publication_year=2015&amp;pages=169-178&amp;doi=10.1016/j.parkreldis.2014.12.013&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B10-jpm-14-00380">10. <span class="element-citation">Waldvogel H.J., Kim E.H., Tippett L.J., Vonsattel J.-P.G., Faull R.L.  <span class="ref-journal">Behavioral Neurobiology of Huntington&#x02019;s Disease and Parkinson&#x02019;s Disease.</span> Springer; Berlin/Heidelberg, Germany: 2015. The neuropathology of Huntington&#x02019;s disease; pp. 33–80. [<a href="https://pubmed.ncbi.nlm.nih.gov/25300927" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Behavioral+Neurobiology+of+Huntington&#x02019;s+Disease+and+Parkinson&#x02019;s+Disease&amp;author=H.J.+Waldvogel&amp;author=E.H.+Kim&amp;author=L.J.+Tippett&amp;author=J.-P.G.+Vonsattel&amp;author=R.L.+Faull&amp;publication_year=2015&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B11-jpm-14-00380">11. <span class="element-citation">Zuccato C., Valenza M., Cattaneo E. Molecular mechanisms and potential therapeutical targets in Huntington&#x02019;s disease. <span><span class="ref-journal">Physiol. Rev. </span>2010;<span class="ref-vol">90</span>:905–981. doi: 10.1152/physrev.00041.2009.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20664076" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1152%2Fphysrev.00041.2009" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Physiol.+Rev.&amp;title=Molecular+mechanisms+and+potential+therapeutical+targets+in+Huntington&#x02019;s+disease&amp;author=C.+Zuccato&amp;author=M.+Valenza&amp;author=E.+Cattaneo&amp;volume=90&amp;publication_year=2010&amp;pages=905-981&amp;pmid=20664076&amp;doi=10.1152/physrev.00041.2009&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B12-jpm-14-00380">12. <span class="element-citation">MacDonald M., Duyao M., Calzonetti T., Auerbach A., Ryan A., Barnes G., White J., Auerbach W., Vonsattel J.-P., Gusella J. Targeted inactivation of the mouse Huntington&#x02019;s disease gene homolog Hdh. <span><span class="ref-journal">Cold Spring Harb. Symp. Quant. Biol. </span>1996;<span class="ref-vol">61</span>:627–638.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9246489" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cold+Spring+Harb.+Symp.+Quant.+Biol.&amp;title=Targeted+inactivation+of+the+mouse+Huntington&#x02019;s+disease+gene+homolog+Hdh&amp;author=M.+MacDonald&amp;author=M.+Duyao&amp;author=T.+Calzonetti&amp;author=A.+Auerbach&amp;author=A.+Ryan&amp;volume=61&amp;publication_year=1996&amp;pages=627-638&amp;pmid=9246489&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B13-jpm-14-00380">13. <span class="element-citation">Monckton D.G. The contribution of somatic expansion of the CAG repeat to symptomatic development in Huntington&#x02019;s disease: A historical perspective. <span><span class="ref-journal">J. Huntington&#x02019;s Dis. </span>2021;<span class="ref-vol">10</span>:7–33. doi: 10.3233/JHD-200429.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7990401/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33579863" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3233%2FJHD-200429" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Huntington&#x02019;s+Dis.&amp;title=The+contribution+of+somatic+expansion+of+the+CAG+repeat+to+symptomatic+development+in+Huntington&#x02019;s+disease:+A+historical+perspective&amp;author=D.G.+Monckton&amp;volume=10&amp;publication_year=2021&amp;pages=7-33&amp;pmid=33579863&amp;doi=10.3233/JHD-200429&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B14-jpm-14-00380">14. <span class="element-citation">MacDonald M.E., Lee J.-M., Gusella J.F.  <span class="ref-journal">Huntington&#x02019;s Disease.</span> Elsevier; Amsterdam, The Netherlands: 2024. Huntington&#x02019;s disease genetics: Implications for pathogenesis; pp. 57–84. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Huntington&#x02019;s+Disease&amp;author=M.E.+MacDonald&amp;author=J.-M.+Lee&amp;author=J.F.+Gusella&amp;publication_year=2024&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B15-jpm-14-00380">15. <span class="element-citation">Christodoulou C.C., Papanicolaou E.Z.  <span class="ref-journal">Omics and Network-Based Approaches in Understanding HD Pathogenesis.</span> IntechOpen; London, UK: 2024.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Omics+and+Network-Based+Approaches+in+Understanding+HD+Pathogenesis&amp;author=C.C.+Christodoulou&amp;author=E.Z.+Papanicolaou&amp;publication_year=2024&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B16-jpm-14-00380">16. <span class="element-citation">Testa C.M., Jankovic J. Huntington disease: A quarter century of progress since the gene discovery. <span><span class="ref-journal">J. Neurol. Sci. </span>2019;<span class="ref-vol">396</span>:52–68. doi: 10.1016/j.jns.2018.09.022.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30419368" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.jns.2018.09.022" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Neurol.+Sci.&amp;title=Huntington+disease:+A+quarter+century+of+progress+since+the+gene+discovery&amp;author=C.M.+Testa&amp;author=J.+Jankovic&amp;volume=396&amp;publication_year=2019&amp;pages=52-68&amp;pmid=30419368&amp;doi=10.1016/j.jns.2018.09.022&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B17-jpm-14-00380">17. <span class="element-citation">Lee J.-M., Wheeler V.C., Chao M.J., Vonsattel J.P.G., Pinto R.M., Lucente D., Abu-Elneel K., Ramos E.M., Mysore J.S., Gillis T. Identification of genetic factors that modify clinical onset of Huntington&#x02019;s disease. <span><span class="ref-journal">Cell. </span>2015;<span class="ref-vol">162</span>:516–526. doi: 10.1016/j.cell.2015.07.003.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4524551/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26232222" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cell.2015.07.003" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Identification+of+genetic+factors+that+modify+clinical+onset+of+Huntington&#x02019;s+disease&amp;author=J.-M.+Lee&amp;author=V.C.+Wheeler&amp;author=M.J.+Chao&amp;author=J.P.G.+Vonsattel&amp;author=R.M.+Pinto&amp;volume=162&amp;publication_year=2015&amp;pages=516-526&amp;pmid=26232222&amp;doi=10.1016/j.cell.2015.07.003&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B18-jpm-14-00380">18. <span class="element-citation">Byrne L.M., Wild E.J. Cerebrospinal fluid biomarkers for Huntington&#x02019;s disease. <span><span class="ref-journal">J. Huntington&#x02019;s Dis. </span>2016;<span class="ref-vol">5</span>:1–13. doi: 10.3233/JHD-160196.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27031730" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3233%2FJHD-160196" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Huntington&#x02019;s+Dis.&amp;title=Cerebrospinal+fluid+biomarkers+for+Huntington&#x02019;s+disease&amp;author=L.M.+Byrne&amp;author=E.J.+Wild&amp;volume=5&amp;publication_year=2016&amp;pages=1-13&amp;pmid=27031730&amp;doi=10.3233/JHD-160196&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B19-jpm-14-00380">19. <span class="element-citation">Rodrigues F.B., Byrne L.M., McColgan P., Robertson N., Tabrizi S.J., Zetterberg H., Wild E.J. Cerebrospinal fluid inflammatory biomarkers reflect clinical severity in Huntington&#x02019;s disease. <span><span class="ref-journal">PLoS ONE. </span>2016;<span class="ref-vol">11</span>:e0163479.  doi: 10.1371/journal.pone.0163479.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5033331/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27657730" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pone.0163479" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Cerebrospinal+fluid+inflammatory+biomarkers+reflect+clinical+severity+in+Huntington&#x02019;s+disease&amp;author=F.B.+Rodrigues&amp;author=L.M.+Byrne&amp;author=P.+McColgan&amp;author=N.+Robertson&amp;author=S.J.+Tabrizi&amp;volume=11&amp;publication_year=2016&amp;pages=e0163479&amp;pmid=27657730&amp;doi=10.1371/journal.pone.0163479&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B20-jpm-14-00380">20. <span class="element-citation">Lederman M., Weiss A., Chowers I. Association of neovascular age-related macular degeneration with specific gene expression patterns in peripheral white blood cells. <span><span class="ref-journal">Investig. Ophthalmol. Vis. Sci. </span>2010;<span class="ref-vol">51</span>:53–58. doi: 10.1167/iovs.08-3019.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19684010" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1167%2Fiovs.08-3019" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Investig.+Ophthalmol.+Vis.+Sci.&amp;title=Association+of+neovascular+age-related+macular+degeneration+with+specific+gene+expression+patterns+in+peripheral+white+blood+cells&amp;author=M.+Lederman&amp;author=A.+Weiss&amp;author=I.+Chowers&amp;volume=51&amp;publication_year=2010&amp;pages=53-58&amp;pmid=19684010&amp;doi=10.1167/iovs.08-3019&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B21-jpm-14-00380">21. <span class="element-citation">Caron N.S., Haqqani A.S., Sandhu A., Aly A.E., Findlay Black H., Bone J.N., McBride J.L., Abulrob A., Stanimirovic D., Leavitt B.R. Cerebrospinal fluid biomarkers for assessing Huntington disease onset and severity. <span><span class="ref-journal">Brain Commun. </span>2022;<span class="ref-vol">4</span>:fcac309. doi: 10.1093/braincomms/fcac309.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9746690/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36523269" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbraincomms%2Ffcac309" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain+Commun.&amp;title=Cerebrospinal+fluid+biomarkers+for+assessing+Huntington+disease+onset+and+severity&amp;author=N.S.+Caron&amp;author=A.S.+Haqqani&amp;author=A.+Sandhu&amp;author=A.E.+Aly&amp;author=H.+Findlay+Black&amp;volume=4&amp;publication_year=2022&amp;pages=fcac309&amp;pmid=36523269&amp;doi=10.1093/braincomms/fcac309&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B22-jpm-14-00380">22. <span class="element-citation">Bj&#x000f6;rkqvist M.  <span class="ref-journal">Biomarkers for Huntington&#x02019;s Disease: Improving Clinical Outcomes.</span> Springer; Berlin/Heidelberg, Germany: 2023. Inflammation Biomarkers in Huntington&#x02019;s Disease; pp. 277–304. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Biomarkers+for+Huntington&#x02019;s+Disease:+Improving+Clinical+Outcomes&amp;author=M.+Bj&#x000f6;rkqvist&amp;publication_year=2023&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B23-jpm-14-00380">23. <span class="element-citation">Malaiya S., Cortes-Gutierrez M., Herb B.R., Coffey S.R., Legg S.R., Cantle J.P., Colantuoni C., Carroll J.B., Ament S.A. Single-nucleus RNA-seq reveals dysregulation of striatal cell identity due to huntington&#x02019;s disease mutations. <span><span class="ref-journal">J. Neurosci. </span>2021;<span class="ref-vol">41</span>:5534–5552. doi: 10.1523/JNEUROSCI.2074-20.2021.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8221598/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34011527" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1523%2FJNEUROSCI.2074-20.2021" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Neurosci.&amp;title=Single-nucleus+RNA-seq+reveals+dysregulation+of+striatal+cell+identity+due+to+huntington&#x02019;s+disease+mutations&amp;author=S.+Malaiya&amp;author=M.+Cortes-Gutierrez&amp;author=B.R.+Herb&amp;author=S.R.+Coffey&amp;author=S.R.+Legg&amp;volume=41&amp;publication_year=2021&amp;pages=5534-5552&amp;pmid=34011527&amp;doi=10.1523/JNEUROSCI.2074-20.2021&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B24-jpm-14-00380">24. <span class="element-citation">Pan L., Feigin A. Huntington&#x02019;s disease: New frontiers in therapeutics. <span><span class="ref-journal">Curr. Neurol. Neurosci. Rep. </span>2021;<span class="ref-vol">21</span>:10. doi: 10.1007/s11910-021-01093-3.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33586075" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs11910-021-01093-3" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Neurol.+Neurosci.+Rep.&amp;title=Huntington&#x02019;s+disease:+New+frontiers+in+therapeutics&amp;author=L.+Pan&amp;author=A.+Feigin&amp;volume=21&amp;publication_year=2021&amp;pages=10&amp;pmid=33586075&amp;doi=10.1007/s11910-021-01093-3&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B25-jpm-14-00380">25. <span class="element-citation">Amir M., Ahmad S., Ahamad S., Kumar V., Mohammad T., Dohare R., Alajmi M.F., Rehman T., Hussain A., Islam A. Impact of Gln94Glu mutation on the structure and function of protection of telomere 1, a cause of cutaneous familial melanoma. <span><span class="ref-journal">J. Biomol. Struct. Dyn. </span>2019:1514–1524. doi: 10.1080/07391102.2019.1610500.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31014199" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1080%2F07391102.2019.1610500" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biomol.+Struct.+Dyn.&amp;title=Impact+of+Gln94Glu+mutation+on+the+structure+and+function+of+protection+of+telomere+1,+a+cause+of+cutaneous+familial+melanoma&amp;author=M.+Amir&amp;author=S.+Ahmad&amp;author=S.+Ahamad&amp;author=V.+Kumar&amp;author=T.+Mohammad&amp;publication_year=2019&amp;pages=1514-1524&amp;pmid=31014199&amp;doi=10.1080/07391102.2019.1610500&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B26-jpm-14-00380">26. <span class="element-citation">Amir M., Kumar V., Mohammad T., Dohare R., Rehman M.T., Alajmi M.F., Hussain A., Ahmad F., Hassan M.I. Structural and functional impact of non-synonymous SNPs in the CST complex subunit TEN1: Structural genomics approach. <span><span class="ref-journal">Biosci. Rep. </span>2019;<span class="ref-vol">39</span>:BSR20190312.  doi: 10.1042/BSR20190312.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6522806/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31028137" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1042%2FBSR20190312" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biosci.+Rep.&amp;title=Structural+and+functional+impact+of+non-synonymous+SNPs+in+the+CST+complex+subunit+TEN1:+Structural+genomics+approach&amp;author=M.+Amir&amp;author=V.+Kumar&amp;author=T.+Mohammad&amp;author=R.+Dohare&amp;author=M.T.+Rehman&amp;volume=39&amp;publication_year=2019&amp;pages=BSR20190312&amp;pmid=31028137&amp;doi=10.1042/BSR20190312&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B27-jpm-14-00380">27. <span class="element-citation">Amir M., Kumar V., Mohammad T., Dohare R., Hussain A., Rehman M.T., Alam P., Alajmi M.F., Islam A., Ahmad F. Investigation of deleterious effects of nsSNPs in the POT1 gene: A structural genomics-based approach to understand the mechanism of cancer development. <span><span class="ref-journal">J. Cell. Biochem. </span>2019;<span class="ref-vol">120</span>:10281–10294. doi: 10.1002/jcb.28312.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30556179" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fjcb.28312" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Cell.+Biochem.&amp;title=Investigation+of+deleterious+effects+of+nsSNPs+in+the+POT1+gene:+A+structural+genomics-based+approach+to+understand+the+mechanism+of+cancer+development&amp;author=M.+Amir&amp;author=V.+Kumar&amp;author=T.+Mohammad&amp;author=R.+Dohare&amp;author=A.+Hussain&amp;volume=120&amp;publication_year=2019&amp;pages=10281-10294&amp;pmid=30556179&amp;doi=10.1002/jcb.28312&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B28-jpm-14-00380">28. <span class="element-citation">Ng P.C., Henikoff S. SIFT: Predicting amino acid changes that affect protein function. <span><span class="ref-journal">Nucleic Acids Res. </span>2003;<span class="ref-vol">31</span>:3812–3814. doi: 10.1093/nar/gkg509.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC168916/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12824425" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkg509" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=SIFT:+Predicting+amino+acid+changes+that+affect+protein+function&amp;author=P.C.+Ng&amp;author=S.+Henikoff&amp;volume=31&amp;publication_year=2003&amp;pages=3812-3814&amp;pmid=12824425&amp;doi=10.1093/nar/gkg509&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B29-jpm-14-00380">29. <span class="element-citation">Adzhubei I., Jordan D.M., Sunyaev S.R. Predicting functional effect of human missense mutations using PolyPhen-2. <span><span class="ref-journal">Curr. Protoc. Hum. Genet. </span>2013;<span class="ref-vol">76</span>:7–20. doi: 10.1002/0471142905.hg0720s76.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4480630/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23315928" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2F0471142905.hg0720s76" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Protoc.+Hum.+Genet.&amp;title=Predicting+functional+effect+of+human+missense+mutations+using+PolyPhen-2&amp;author=I.+Adzhubei&amp;author=D.M.+Jordan&amp;author=S.R.+Sunyaev&amp;volume=76&amp;publication_year=2013&amp;pages=7-20&amp;pmid=23315928&amp;doi=10.1002/0471142905.hg0720s76&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B30-jpm-14-00380">30. <span class="element-citation">Rogers M.F., Shihab H.A., Mort M., Cooper D.N., Gaunt T.R., Campbell C. FATHMM-XF: Accurate prediction of pathogenic point mutations via extended features. <span><span class="ref-journal">Bioinformatics. </span>2018;<span class="ref-vol">34</span>:511–513. doi: 10.1093/bioinformatics/btx536.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5860356/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28968714" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtx536" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=FATHMM-XF:+Accurate+prediction+of+pathogenic+point+mutations+via+extended+features&amp;author=M.F.+Rogers&amp;author=H.A.+Shihab&amp;author=M.+Mort&amp;author=D.N.+Cooper&amp;author=T.R.+Gaunt&amp;volume=34&amp;publication_year=2018&amp;pages=511-513&amp;pmid=28968714&amp;doi=10.1093/bioinformatics/btx536&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B31-jpm-14-00380">31. <span class="element-citation">Capriotti E., Calabrese R., Fariselli P., Martelli P.L., Altman R.B., Casadio R. WS-SNPs&#x00026;GO: A web server for predicting the deleterious effect of human protein variants using functional annotation. <span><span class="ref-journal">BMC Genom. </span>2013;<span class="ref-vol">14</span>:S6 </span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3665478/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23819482" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genom.&amp;title=WS-SNPs&#x00026;GO:+A+web+server+for+predicting+the+deleterious+effect+of+human+protein+variants+using+functional+annotation&amp;author=E.+Capriotti&amp;author=R.+Calabrese&amp;author=P.+Fariselli&amp;author=P.L.+Martelli&amp;author=R.B.+Altman&amp;volume=14&amp;publication_year=2013&amp;pages=S6&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B32-jpm-14-00380">32. <span class="element-citation">Pires D.E., Ascher D.B., Blundell T.L. mCSM: Predicting the effects of mutations in proteins using graph-based signatures. <span><span class="ref-journal">Bioinformatics. </span>2014;<span class="ref-vol">30</span>:335–342. doi: 10.1093/bioinformatics/btt691.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3904523/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24281696" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtt691" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=mCSM:+Predicting+the+effects+of+mutations+in+proteins+using+graph-based+signatures&amp;author=D.E.+Pires&amp;author=D.B.+Ascher&amp;author=T.L.+Blundell&amp;volume=30&amp;publication_year=2014&amp;pages=335-342&amp;pmid=24281696&amp;doi=10.1093/bioinformatics/btt691&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B33-jpm-14-00380">33. <span class="element-citation">Rodrigues C.H., Pires D.E., Ascher D.B. DynaMut2: Assessing changes in stability and flexibility upon single and multiple point missense mutations. <span><span class="ref-journal">Protein Sci. </span>2021;<span class="ref-vol">30</span>:60–69. doi: 10.1002/pro.3942.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7737773/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32881105" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fpro.3942" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Sci.&amp;title=DynaMut2:+Assessing+changes+in+stability+and+flexibility+upon+single+and+multiple+point+missense+mutations&amp;author=C.H.+Rodrigues&amp;author=D.E.+Pires&amp;author=D.B.+Ascher&amp;volume=30&amp;publication_year=2021&amp;pages=60-69&amp;pmid=32881105&amp;doi=10.1002/pro.3942&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B34-jpm-14-00380">34. <span class="element-citation">Laimer J., Hiebl-Flach J., Lengauer D., Lackner P. MAESTROweb: A web server for structure-based protein stability prediction. <span><span class="ref-journal">Bioinformatics. </span>2016;<span class="ref-vol">32</span>:1414–1416. doi: 10.1093/bioinformatics/btv769.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26743508" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtv769" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=MAESTROweb:+A+web+server+for+structure-based+protein+stability+prediction&amp;author=J.+Laimer&amp;author=J.+Hiebl-Flach&amp;author=D.+Lengauer&amp;author=P.+Lackner&amp;volume=32&amp;publication_year=2016&amp;pages=1414-1416&amp;pmid=26743508&amp;doi=10.1093/bioinformatics/btv769&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B35-jpm-14-00380">35. <span class="element-citation">Chen Y., Lu H., Zhang N., Zhu Z., Wang S., Li M. PremPS: Predicting the impact of missense mutations on protein stability. <span><span class="ref-journal">PLoS Comput. Biol. </span>2020;<span class="ref-vol">16</span>:e1008543.  doi: 10.1371/journal.pcbi.1008543.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7802934/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33378330" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pcbi.1008543" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Comput.+Biol.&amp;title=PremPS:+Predicting+the+impact+of+missense+mutations+on+protein+stability&amp;author=Y.+Chen&amp;author=H.+Lu&amp;author=N.+Zhang&amp;author=Z.+Zhu&amp;author=S.+Wang&amp;volume=16&amp;publication_year=2020&amp;pages=e1008543&amp;pmid=33378330&amp;doi=10.1371/journal.pcbi.1008543&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B36-jpm-14-00380">36. <span class="element-citation">Pejaver V., Urresti J., Lugo-Martinez J., Pagel K.A., Lin G.N., Nam H.-J., Mort M., Cooper D.N., Sebat J., Iakoucheva L.M. Inferring the molecular and phenotypic impact of amino acid variants with MutPred2. <span><span class="ref-journal">Nat. Commun. </span>2020;<span class="ref-vol">11</span>:5918. doi: 10.1038/s41467-020-19669-x.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7680112/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33219223" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41467-020-19669-x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Inferring+the+molecular+and+phenotypic+impact+of+amino+acid+variants+with+MutPred2&amp;author=V.+Pejaver&amp;author=J.+Urresti&amp;author=J.+Lugo-Martinez&amp;author=K.A.+Pagel&amp;author=G.N.+Lin&amp;volume=11&amp;publication_year=2020&amp;pages=5918&amp;pmid=33219223&amp;doi=10.1038/s41467-020-19669-x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B37-jpm-14-00380">37. <span class="element-citation">Calabrese R., Capriotti E., Casadio R.  <span class="ref-journal">EMBNET08.</span> ITA; Rome, Italy: 2008. PhD-SNP: A web server for the prediction of human genetic diseases associated to missense single nucleotide polymorphisms; p. 78. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=EMBNET08&amp;author=R.+Calabrese&amp;author=E.+Capriotti&amp;author=R.+Casadio&amp;publication_year=2008&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B38-jpm-14-00380">38. <span class="element-citation">Consortium U. UniProt: A worldwide hub of protein knowledge. <span><span class="ref-journal">Nucleic Acids Res. </span>2019;<span class="ref-vol">47</span>:D506–D515. doi: 10.1093/nar/gky1049.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6323992/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30395287" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgky1049" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=UniProt:+A+worldwide+hub+of+protein+knowledge&amp;author=U.+Consortium&amp;volume=47&amp;publication_year=2019&amp;pages=D506-D515&amp;pmid=30395287&amp;doi=10.1093/nar/gky1049&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B39-jpm-14-00380">39. <span class="element-citation">Sherry S.T., Ward M.-H., Kholodov M., Baker J., Phan L., Smigielski E.M., Sirotkin K. dbSNP: The NCBI database of genetic variation. <span><span class="ref-journal">Nucleic Acids Res. </span>2001;<span class="ref-vol">29</span>:308–311. doi: 10.1093/nar/29.1.308.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC29783/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11125122" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2F29.1.308" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=dbSNP:+The+NCBI+database+of+genetic+variation&amp;author=S.T.+Sherry&amp;author=M.-H.+Ward&amp;author=M.+Kholodov&amp;author=J.+Baker&amp;author=L.+Phan&amp;volume=29&amp;publication_year=2001&amp;pages=308-311&amp;pmid=11125122&amp;doi=10.1093/nar/29.1.308&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B40-jpm-14-00380">40. <span class="element-citation">Cunningham F., Allen J.E., Allen J., Alvarez-Jarreta J., Amode M.R., Armean I.M., Austine-Orimoloye O., Azov A.G., Barnes I., Bennett R. Ensembl 2022. <span><span class="ref-journal">Nucleic Acids Res. </span>2022;<span class="ref-vol">50</span>:D988–D995. doi: 10.1093/nar/gkab1049.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8728283/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34791404" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkab1049" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Ensembl+2022&amp;author=F.+Cunningham&amp;author=J.E.+Allen&amp;author=J.+Allen&amp;author=J.+Alvarez-Jarreta&amp;author=M.R.+Amode&amp;volume=50&amp;publication_year=2022&amp;pages=D988-D995&amp;pmid=34791404&amp;doi=10.1093/nar/gkab1049&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B41-jpm-14-00380">41. <span class="element-citation">Canese K., Weis S.  <span class="ref-journal">The NCBI Handbook.</span> Volume 2 National Center for Biotechnology Information; Bethesda, MD, USA: 2013. PubMed: The bibliographic database. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=The+NCBI+Handbook&amp;author=K.+Canese&amp;author=S.+Weis&amp;publication_year=2013&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B42-jpm-14-00380">42. <span class="element-citation">Navarro S., Ventura S. Computational methods to predict protein aggregation. <span><span class="ref-journal">Curr. Opin. Struct. Biol. </span>2022;<span class="ref-vol">73</span>:102343.  doi: 10.1016/j.sbi.2022.102343.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35240456" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.sbi.2022.102343" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Struct.+Biol.&amp;title=Computational+methods+to+predict+protein+aggregation&amp;author=S.+Navarro&amp;author=S.+Ventura&amp;volume=73&amp;publication_year=2022&amp;pages=102343&amp;pmid=35240456&amp;doi=10.1016/j.sbi.2022.102343&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B43-jpm-14-00380">43. <span class="element-citation">Piovesan D., Walsh I., Minervini G., Tosatto S.C. FELLS: Fast estimator of latent local structure. <span><span class="ref-journal">Bioinformatics. </span>2017;<span class="ref-vol">33</span>:1889–1891. doi: 10.1093/bioinformatics/btx085.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28186245" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtx085" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=FELLS:+Fast+estimator+of+latent+local+structure&amp;author=D.+Piovesan&amp;author=I.+Walsh&amp;author=G.+Minervini&amp;author=S.C.+Tosatto&amp;volume=33&amp;publication_year=2017&amp;pages=1889-1891&amp;pmid=28186245&amp;doi=10.1093/bioinformatics/btx085&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B44-jpm-14-00380">44. <span class="element-citation">Ferreiro D.U., Komives E.A., Wolynes P.G. Frustration in biomolecules. <span><span class="ref-journal">Q. Rev. Biophys. </span>2014;<span class="ref-vol">47</span>:285–363. doi: 10.1017/S0033583514000092.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4256721/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25225856" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1017%2FS0033583514000092" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Q.+Rev.+Biophys.&amp;title=Frustration+in+biomolecules&amp;author=D.U.+Ferreiro&amp;author=E.A.+Komives&amp;author=P.G.+Wolynes&amp;volume=47&amp;publication_year=2014&amp;pages=285-363&amp;pmid=25225856&amp;doi=10.1017/S0033583514000092&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B45-jpm-14-00380">45. <span class="element-citation">Jenik M., Parra R.G., Radusky L.G., Turjanski A., Wolynes P.G., Ferreiro D.U. Protein frustratometer: A tool to localize energetic frustration in protein molecules. <span><span class="ref-journal">Nucleic Acids Res. </span>2012;<span class="ref-vol">40</span>:W348–W351. doi: 10.1093/nar/gks447.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3394345/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22645321" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgks447" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Protein+frustratometer:+A+tool+to+localize+energetic+frustration+in+protein+molecules&amp;author=M.+Jenik&amp;author=R.G.+Parra&amp;author=L.G.+Radusky&amp;author=A.+Turjanski&amp;author=P.G.+Wolynes&amp;volume=40&amp;publication_year=2012&amp;pages=W348-W351&amp;pmid=22645321&amp;doi=10.1093/nar/gks447&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B46-jpm-14-00380">46. <span class="element-citation">Amir M., Mohammad T., Kumar V., Alajmi M.F., Rehman M.T., Hussain A., Alam P., Dohare R., Islam A., Ahmad F. Structural analysis and conformational dynamics of STN1 gene mutations involved in coat plus syndrome. <span><span class="ref-journal">Front. Mol. Biosci. </span>2019;<span class="ref-vol">6</span>:41.  doi: 10.3389/fmolb.2019.00041.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6581698/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31245382" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3389%2Ffmolb.2019.00041" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Mol.+Biosci.&amp;title=Structural+analysis+and+conformational+dynamics+of+STN1+gene+mutations+involved+in+coat+plus+syndrome&amp;author=M.+Amir&amp;author=T.+Mohammad&amp;author=V.+Kumar&amp;author=M.F.+Alajmi&amp;author=M.T.+Rehman&amp;volume=6&amp;publication_year=2019&amp;pages=41&amp;pmid=31245382&amp;doi=10.3389/fmolb.2019.00041&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B47-jpm-14-00380">47. <span class="element-citation">Paladin L., Piovesan D., Tosatto S.C. SODA: Prediction of protein solubility from disorder and aggregation propensity. <span><span class="ref-journal">Nucleic Acids Res. </span>2017;<span class="ref-vol">45</span>:W236–W240. doi: 10.1093/nar/gkx412.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7059794/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28505312" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkx412" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=SODA:+Prediction+of+protein+solubility+from+disorder+and+aggregation+propensity&amp;author=L.+Paladin&amp;author=D.+Piovesan&amp;author=S.C.+Tosatto&amp;volume=45&amp;publication_year=2017&amp;pages=W236-W240&amp;pmid=28505312&amp;doi=10.1093/nar/gkx412&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B48-jpm-14-00380">48. <span class="element-citation">Ciurea A.V., Mohan A.G., Covache-Busuioc R.-A., Costin H.-P., Glavan L.-A., Corlatescu A.-D., Saceleanu V.M. Unraveling Molecular and Genetic Insights into Neurodegenerative Diseases: Advances in Understanding Alzheimer&#x02019;s, Parkinson&#x02019;s, and Huntington&#x02019;s Diseases and Amyotrophic Lateral Sclerosis. <span><span class="ref-journal">Int. J. Mol. Sci. </span>2023;<span class="ref-vol">24</span>:10809.  doi: 10.3390/ijms241310809.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10341997/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37445986" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fijms241310809" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Mol.+Sci.&amp;title=Unraveling+Molecular+and+Genetic+Insights+into+Neurodegenerative+Diseases:+Advances+in+Understanding+Alzheimer&#x02019;s,+Parkinson&#x02019;s,+and+Huntington&#x02019;s+Diseases+and+Amyotrophic+Lateral+Sclerosis&amp;author=A.V.+Ciurea&amp;author=A.G.+Mohan&amp;author=R.-A.+Covache-Busuioc&amp;author=H.-P.+Costin&amp;author=L.-A.+Glavan&amp;volume=24&amp;publication_year=2023&amp;pages=10809&amp;pmid=37445986&amp;doi=10.3390/ijms241310809&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B49-jpm-14-00380">49. <span class="element-citation">Tabrizi S.J., Flower M.D., Ross C.A., Wild E.J. Huntington disease: New insights into molecular pathogenesis and therapeutic opportunities. <span><span class="ref-journal">Nat. Rev. Neurol. </span>2020;<span class="ref-vol">16</span>:529–546. doi: 10.1038/s41582-020-0389-4.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32796930" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41582-020-0389-4" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Neurol.&amp;title=Huntington+disease:+New+insights+into+molecular+pathogenesis+and+therapeutic+opportunities&amp;author=S.J.+Tabrizi&amp;author=M.D.+Flower&amp;author=C.A.+Ross&amp;author=E.J.+Wild&amp;volume=16&amp;publication_year=2020&amp;pages=529-546&amp;pmid=32796930&amp;doi=10.1038/s41582-020-0389-4&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B50-jpm-14-00380">50. <span class="element-citation">Johnson W.G. Late-onset neurodegenerative diseases&#x02014;The role of protein insolubility. <span><span class="ref-journal">J. Anat. </span>2000;<span class="ref-vol">196</span>:609–616. doi: 10.1046/j.1469-7580.2000.19640609.x.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1468101/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10923991" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1046%2Fj.1469-7580.2000.19640609.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Anat.&amp;title=Late-onset+neurodegenerative+diseases&#x02014;The+role+of+protein+insolubility&amp;author=W.G.+Johnson&amp;volume=196&amp;publication_year=2000&amp;pages=609-616&amp;pmid=10923991&amp;doi=10.1046/j.1469-7580.2000.19640609.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B51-jpm-14-00380">51. <span class="element-citation">Arrasate M., Finkbeiner S. Protein aggregates in Huntington&#x02019;s disease. <span><span class="ref-journal">Exp. Neurol. </span>2012;<span class="ref-vol">238</span>:1–11. doi: 10.1016/j.expneurol.2011.12.013.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3909772/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22200539" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.expneurol.2011.12.013" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Exp.+Neurol.&amp;title=Protein+aggregates+in+Huntington&#x02019;s+disease&amp;author=M.+Arrasate&amp;author=S.+Finkbeiner&amp;volume=238&amp;publication_year=2012&amp;pages=1-11&amp;pmid=22200539&amp;doi=10.1016/j.expneurol.2011.12.013&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B52-jpm-14-00380">52. <span class="element-citation">Wanker E.E. Protein aggregation and pathogenesis of Huntingtons disease: Mechanisms and correlations. <span><span class="ref-journal">Biol. Chem. </span>2000 doi: 10.1515/BC.2000.114.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11076024" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1515%2FBC.2000.114" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol.+Chem.&amp;title=Protein+aggregation+and+pathogenesis+of+Huntingtons+disease:+Mechanisms+and+correlations&amp;author=E.E.+Wanker&amp;publication_year=2000&amp;doi=10.1515/BC.2000.114&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B53-jpm-14-00380">53. <span class="element-citation">Anjum F., Joshia N., Mohammad T., Shafie A., Alhumaydhi F.A., Aljasir M.A., Shahwan M.J., Abdullaev B., Adnan M., Elasbali A.M. Impact of single amino acid substitutions in parkinsonism-associated deglycase-PARK7 and their association with Parkinson&#x02019;s disease. <span><span class="ref-journal">J. Pers. Med. </span>2022;<span class="ref-vol">12</span>:220.  doi: 10.3390/jpm12020220.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8878504/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35207708" ref="reftype=pubmed&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fjpm12020220" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Pers.+Med.&amp;title=Impact+of+single+amino+acid+substitutions+in+parkinsonism-associated+deglycase-PARK7+and+their+association+with+Parkinson&#x02019;s+disease&amp;author=F.+Anjum&amp;author=N.+Joshia&amp;author=T.+Mohammad&amp;author=A.+Shafie&amp;author=F.A.+Alhumaydhi&amp;volume=12&amp;publication_year=2022&amp;pages=220&amp;pmid=35207708&amp;doi=10.3390/jpm12020220&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11050829&amp;issue-id=461158&amp;journal-id=2605&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Journal of Personalized Medicine</span> are provided here courtesy of <strong>Multidisciplinary Digital Publishing Institute  (MDPI)</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11050829/pdf/jpm-14-00380.pdf" class="int-view">PDF (2.6M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11050829/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11050829/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11050829" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11050829%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="AJ6aK4nDWrAehHjwF35ztl3N6vGgBZKa5ZANSwFql2Tegm3cqSk0sMT9v4bHrXOs">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11050829%2F&amp;text=Elucidating%20the%20Impact%20of%20Deleterious%20Mutations%20on%20IGHG1%20and%20Their%20Association%20with%20Huntington%02019s%20Disease%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11050829%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11050829/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11050829/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38673007/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11050829/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38673007/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11050829/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11050829/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC11050829/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
