
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Genome-wide identification of the walnut MYC gene family and functional characterization of Xinjiang wild walnut under low-temperature stress - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="322C300B24F39205000042A01A150EFE.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="frontgene" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC11128688/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11128688/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Frontiers in Genetics" /><meta name="citation_title" content="Genome-wide identification of the walnut MYC gene family and functional characterization of Xinjiang wild walnut under low-temperature stress" /><meta name="citation_author" content="Ya-Ting Song" /><meta name="citation_author_institution" content="1 College of Horticulture, Xinjiang Agricultural University, Urumqi, China" /><meta name="citation_author_institution" content="2 Xinjiang Key Laboratory of Genome Research and Genetic Improvement of Specialty Fruits and Vegetables, Xinjiang Institute of Horticultural Crops, Xinjiang Academy of Agricultural Sciences, Xinjiang Regional Scientific Observatory and Experiment Station of Fruit Trees, Ministry of Agriculture, Urumqi, China" /><meta name="citation_author" content="Kai Ma" /><meta name="citation_author_institution" content="2 Xinjiang Key Laboratory of Genome Research and Genetic Improvement of Specialty Fruits and Vegetables, Xinjiang Institute of Horticultural Crops, Xinjiang Academy of Agricultural Sciences, Xinjiang Regional Scientific Observatory and Experiment Station of Fruit Trees, Ministry of Agriculture, Urumqi, China" /><meta name="citation_author" content="Yu Zhao" /><meta name="citation_author_institution" content="2 Xinjiang Key Laboratory of Genome Research and Genetic Improvement of Specialty Fruits and Vegetables, Xinjiang Institute of Horticultural Crops, Xinjiang Academy of Agricultural Sciences, Xinjiang Regional Scientific Observatory and Experiment Station of Fruit Trees, Ministry of Agriculture, Urumqi, China" /><meta name="citation_author" content="Li-Qun Han" /><meta name="citation_author_institution" content="1 College of Horticulture, Xinjiang Agricultural University, Urumqi, China" /><meta name="citation_author_institution" content="2 Xinjiang Key Laboratory of Genome Research and Genetic Improvement of Specialty Fruits and Vegetables, Xinjiang Institute of Horticultural Crops, Xinjiang Academy of Agricultural Sciences, Xinjiang Regional Scientific Observatory and Experiment Station of Fruit Trees, Ministry of Agriculture, Urumqi, China" /><meta name="citation_author" content="Li-Qiang Liu" /><meta name="citation_author_institution" content="1 College of Horticulture, Xinjiang Agricultural University, Urumqi, China" /><meta name="citation_publication_date" content="2024" /><meta name="citation_volume" content="15" /><meta name="citation_doi" content="10.3389/fgene.2024.1399721" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC11128688/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC11128688/" /><meta name="citation_pmid" content="38803544" /><meta name="DC.Title" content="Genome-wide identification of the walnut MYC gene family and functional characterization of Xinjiang wild walnut under low-temperature stress" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Frontiers Media SA" /><meta name="DC.Contributor" content="Ya-Ting Song" /><meta name="DC.Contributor" content="Kai Ma" /><meta name="DC.Contributor" content="Yu Zhao" /><meta name="DC.Contributor" content="Li-Qun Han" /><meta name="DC.Contributor" content="Li-Qiang Liu" /><meta name="DC.Date" content="2024" /><meta name="DC.Identifier" content="10.3389/fgene.2024.1399721" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Genome-wide identification of the walnut MYC gene family and functional characterization of Xinjiang wild walnut under low-temperature stress" /><meta property="og:type" content="article" /><meta property="og:description" content="&#10;Introduction: MYC transcription factors are the basic regulators of the jasmonic acid signaling pathway and play important roles in plant growth and development and the response to adverse stress. In recent years, severe winter freezing and late spring ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11128688/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11128688/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22Front%20Genet%22[journal]" class="navlink">Front Genet</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11128688
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11128688/pdf/fgene-15-1399721.pdf" class="int-view">PDF (5.2M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11128688/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11128688/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11128688" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11128688%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="duEKOrMZyLN8MAcDGy6QH1Lzjpb3dkIOyzLmZmgRgkaOo5fHULcXibq8EW32msQn">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11128688%2F&amp;text=Genome-wide%20identification%20of%20the%20walnut%20MYC%20gene%20family%20and%20functional%20characterization%20of%20Xinjiang%20wild%20walnut%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11128688%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11128688/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11128688/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38803544/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11128688/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38803544/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11128688/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11128688/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22Front%20Genet%22[journal]" class="navlink">Front Genet</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11128688
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-frontgene.gif" alt="Logo of frontgene" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.frontiersin.org/genetics" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Front Genet.</span> 2024; 15: 1399721. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2024 May 13. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.3389%2Ffgene.2024.1399721" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.3389/fgene.2024.1399721</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC11128688</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38803544">38803544</a></div></div></div><h1 class="content-title">Genome-wide identification of the walnut MYC gene family and functional characterization of Xinjiang wild walnut under low-temperature stress</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Song%20YT%5BAuthor%5D" class="affpopup" co-rid="_co_idm140430621414992" co-class="co-affbox">Ya-Ting Song</a>,<sup>
1
,</sup><sup>
2
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ma%20K%5BAuthor%5D" class="affpopup" co-rid="_co_idm140430583201968" co-class="co-affbox">Kai Ma</a>,<sup>
2
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Zhao%20Y%5BAuthor%5D" class="affpopup" co-rid="_co_idm140430583197360" co-class="co-affbox">Yu Zhao</a>,<sup>
2
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Han%20LQ%5BAuthor%5D" class="affpopup" co-rid="_co_idm140430583192848" co-class="co-affbox">Li-Qun Han</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>
1
,</sup><sup>
2
,</sup><sup>*</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Liu%20LQ%5BAuthor%5D" class="affpopup" co-rid="_co_idm140430583186048" co-class="co-affbox">Li-Qiang Liu</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>
1
,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140430621414992"><h3 class="no_margin">Ya-Ting Song</h3><p>
<sup>1</sup>
College of Horticulture, Xinjiang Agricultural University, Urumqi, China
</p><p>
<sup>2</sup>
Xinjiang Key Laboratory of Genome Research and Genetic Improvement of Specialty Fruits and Vegetables, Xinjiang Institute of Horticultural Crops, Xinjiang Academy of Agricultural Sciences, Xinjiang Regional Scientific Observatory and Experiment Station of Fruit Trees, Ministry of Agriculture, Urumqi, China
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Song%20YT%5BAuthor%5D">Ya-Ting Song</a></div></div><div id="_co_idm140430583201968"><h3 class="no_margin">Kai Ma</h3><p>
<sup>2</sup>
Xinjiang Key Laboratory of Genome Research and Genetic Improvement of Specialty Fruits and Vegetables, Xinjiang Institute of Horticultural Crops, Xinjiang Academy of Agricultural Sciences, Xinjiang Regional Scientific Observatory and Experiment Station of Fruit Trees, Ministry of Agriculture, Urumqi, China
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ma%20K%5BAuthor%5D">Kai Ma</a></div></div><div id="_co_idm140430583197360"><h3 class="no_margin">Yu Zhao</h3><p>
<sup>2</sup>
Xinjiang Key Laboratory of Genome Research and Genetic Improvement of Specialty Fruits and Vegetables, Xinjiang Institute of Horticultural Crops, Xinjiang Academy of Agricultural Sciences, Xinjiang Regional Scientific Observatory and Experiment Station of Fruit Trees, Ministry of Agriculture, Urumqi, China
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Zhao%20Y%5BAuthor%5D">Yu Zhao</a></div></div><div id="_co_idm140430583192848"><h3 class="no_margin">Li-Qun Han</h3><p>
<sup>1</sup>
College of Horticulture, Xinjiang Agricultural University, Urumqi, China
</p><p>
<sup>2</sup>
Xinjiang Key Laboratory of Genome Research and Genetic Improvement of Specialty Fruits and Vegetables, Xinjiang Institute of Horticultural Crops, Xinjiang Academy of Agricultural Sciences, Xinjiang Regional Scientific Observatory and Experiment Station of Fruit Trees, Ministry of Agriculture, Urumqi, China
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Han%20LQ%5BAuthor%5D">Li-Qun Han</a></div></div><div id="_co_idm140430583186048"><h3 class="no_margin">Li-Qiang Liu</h3><p>
<sup>1</sup>
College of Horticulture, Xinjiang Agricultural University, Urumqi, China
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Liu%20LQ%5BAuthor%5D">Li-Qiang Liu</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140430620757984_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140430620757984_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140430620757984_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140430620757984_ai" style="display:none"><div class="fm-affl" id="aff1">
<sup>1</sup>
College of Horticulture, Xinjiang Agricultural University, Urumqi, China
</div><div class="fm-affl" id="aff2">
<sup>2</sup>
Xinjiang Key Laboratory of Genome Research and Genetic Improvement of Specialty Fruits and Vegetables, Xinjiang Institute of Horticultural Crops, Xinjiang Academy of Agricultural Sciences, Xinjiang Regional Scientific Observatory and Experiment Station of Fruit Trees, Ministry of Agriculture, Urumqi, China
</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup>Corresponding author.</div><div id="fn-a.p.b.h.a">
<strong>Edited by:</strong>
<a href="https://loop.frontiersin.org/people/306131/overview" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">Ertugrul Filiz</a>, Duzce University, T&#x000fc;rkiye</div><div id="fn-a.p.b.h.b">
<strong>Reviewed by:</strong>
<a href="https://loop.frontiersin.org/people/1218299/overview" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">Bo Sun</a>, Sichuan Agricultural University, China<p class="p p-last">
<a href="https://loop.frontiersin.org/people/797647/overview" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">Parviz Heidari</a>, Shahrood University of Technology, Iran</p></div><div id="c001">*Correspondence: Li-Qun Han, <a href="mailto:dev@null" data-email="moc.361@3891qlnah" class="oemail">moc.361@3891qlnah</a>; Li-Qiang Liu, <a href="mailto:dev@null" data-email="moc.621@9899qll" class="oemail">moc.621@9899qll</a>
</div></div><div class="fm-article-notes hide half_rhythm" id="idm140430620757984_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2024 Mar 12; Accepted 2024 Apr 22.</div></div><div class="permissions half_rhythm hide" id="idm140430620757984_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  &#x000a9; 2024 Song, Ma, Zhao, Han and Liu.</div><div class="license half_rhythm">This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div><a href="/pmc/articles/PMC11128688/bin/Table1.XLSX" data-ga-action="click_feat_suppl">Table1.XLSX</a><span style="color:gray"> (9.9K)</span></div><div class="small guid">GUID: BE215F09-FC8B-427E-AF16-C73578B8E703</div></div></dd></dl><dl data-length="190" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first-last">The datasets presented in this study can be found in online repositories. The names of the repository/repositories and accession number(s) can be found in the article/<a href="#s11" rid="s11" class=" sec">Supplementary Material</a>.</p></dd></dl></div><div id="abstract-a.p.b.o" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.p.b.otitle">Abstract</h2><!--article-meta--><div><p class="p p-first">
<strong>Introduction:</strong> MYC transcription factors are the basic regulators of the jasmonic acid signaling pathway and play important roles in plant growth and development and the response to adverse stress. In recent years, severe winter freezing and late spring frost in the main planting area of walnut in Xinjiang have affected the growth and development of walnut, which has become a prominent problem restricting walnut production. Xinjiang wild walnut is the only remaining wild species of walnuts in China, which contains a lot of genes with excellent traits, and is important for the cultivation and breeding.</p><p>
<strong>Methods:</strong> In this paper, the physicochemical properties and bioinformatics of MYC transcription factor members in walnut were analyzed, and the nine MYC were screened from the transcriptome data under low temperature stress. At last, we study the subcellular localizations and the expression patterns of the nine MYC members in Xinjiang wild walnut.</p><p>
<strong>Results:</strong> The results revealed that 30 MYC members were identified from published walnut whole-genome data, and their evolutionary relationships with Arabidopsis and poplar were divided into six groups according to clustering analysis, among which <em>JrMYC22 and JrMYC23</em> had high homology with <em>PtrMYC2b</em>, which is induced by jasmonic acid in response to low-temperature stress. Walnut MYC members are unevenly distributed on 12 chromosomes. The prediction of promoter cis-acting elements of walnut MYC transcription factor family members revealed that cis-acting elements related to jasmonic acid and lowtemperature stress were the ones with the greatest number of members, with 12. In addition, all nine MYC family members in Xinjiang wild walnut plants responding to low-temperature stress exhibited strong fluorescence responses in the nucleus. The expression levels of these members in response to low-temperature stress revealed that <em>JrMYC28, JrMYC31, JrMYC33, JrMYC34, and JrMYC35</em> were highly expressed, and it was hypothesized that <em>JrMYC28, JrMYC31, JrMYC33, JrMYC34, and JrMYC35</em> might play a key role in the response to lowtemperature stress.</p><p class="p p-last">
<strong>Discussion:</strong> The results of this study provide a theoretical basis for further research on the functional mechanisms of the MYC transcription factor family members in walnut.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">walnut, MYC transcription factor, bioinformatics, subcellular localization, low-temperature stress</span></div></div><div id="s1" class="tsec sec"><h2 class="head no_bottom_margin" id="s1title">1 Introduction</h2><p class="p p-first">The jasmonate signaling pathway is relatively complex and involved in plant growth and development, biosynthesis, phytohormone signaling and the regulation of resistance (<a href="#B8" rid="B8" class=" bibr popnode">Creelman and Mullet, 1995</a>; <a href="#B24" rid="B24" class=" bibr popnode">Le&#x000f3;n and S&#x000e1;nchez-Serrano, 1999</a>); this pathway regulates downstream functional genes in response to adverse stress through the specific binding of transcription factors to related <em>cis</em>-acting elements to improve plant resistance (<a href="#B2" rid="B2" class=" bibr popnode">Ali and Baek, 2020</a>). COI1 (CORONATINE INSENSITIVE1) and JAZ (Jasmonate-ZIM) are coreceptors of the jasmonate signaling pathway, in which JA-Ile binds directly to JAZ and COI1, and JAZ protein ubiquitination is degraded by the 26S proteasome and at the same time activates related transcription factors, which induce the expression of downstream functional genes in response to adverse stresses, such as low temperature, drought (PEG), and high salinity (<a href="#B32" rid="B32" class=" bibr popnode">Pe&#x000f1;uelas et al., 2019</a>; <a href="#B34" rid="B34" class=" bibr popnode">Ruan et al., 2019</a>), in plants. MYC transcription factors play a key role in the jasmonic acid signaling pathway and can specifically bind to multiple target genes in response to adversity (<a href="#B43" rid="B43" class=" bibr popnode">Wang et al., 2023</a>).</p><p>MYC transcription factors, a subfamily of the bHLH family, are core regulators of the jasmonate signaling pathway. MYC transcription factors have a conserved bHLH structural domain at the C-terminal end and a bHLH_MYC_N structural domain at the N-terminal end, which has a DNA-binding function that allows the bHLH structural domains to form a homo or heterodimeric composite substance (<a href="#B4" rid="B4" class=" bibr popnode">Boter et al., 2004</a>; <a href="#B23" rid="B23" class=" bibr popnode">Kazan and Manners, 2013</a>). Among them, MYC2 is the most studied member of the MYC transcription factor family, which can specifically bind to JAZ proteins and is involved in the regulation of jasmonic acid signaling, response to pathogens, and resistance to adverse stresses (<a href="#B46" rid="B46" class=" bibr popnode">Yanfang et al., 2018</a>). In <em>Arabidopsis thaliana</em>, MYC2 has a DNA binding specificity similar to that of MYC3 and MYC4, and they interact to activate the jasmonic acid signaling pathway and regulate plant responses to adverse stress (<a href="#B12" rid="B12" class=" bibr popnode">Fern&#x000e1;ndez-Calvo et al., 2011</a>; <a href="#B36" rid="B36" class=" bibr popnode">Schweizer et al., 2013</a>; <a href="#B14" rid="B14" class=" bibr popnode">Gao et al., 2016</a>; <a href="#B41" rid="B41" class=" bibr popnode">Wang et al., 2017</a>; <a href="#B39" rid="B39" class=" bibr popnode">Wang et al., 2021</a>). Overexpression of AtMYC2, AtMYC67, and AtMYC70 activates the promoters of downstream functional genes to increase plant resistance in response to dehydration, high-salt, and low-temperature stresses (<a href="#B1" rid="B1" class=" bibr popnode">Abe et al., 1997</a>; <a href="#B37" rid="B37" class=" bibr popnode">Shinozaki and Yamaguchi-Shinozaki, 2007</a>) in apple, and overexpression of MdMYC2 in response to low-temperature stress increases the expression of <em>MdCIbHLH1, MdCBF1, MdCBF2</em>, and <em>MdCBF3</em> to enhance plant cold resistance (<a href="#B44" rid="B44" class=" bibr popnode">Wang et al., 2019</a>). In tomato, MYC2 induces the expression of the ethylene gene <em>SlERF. B8</em>, which triggers the jasmonic acid signaling pathway, plays a regulatory role in increasing the cold tolerance of tomato plants (<a href="#B9" rid="B9" class=" bibr popnode">Ding et al., 2022</a>). In seedlings of Bupleurum chinensis, BcMYC2 was regulated through the jasmonic acid signaling pathway in response to drought stress, which affected the biosynthesis mechanism of Bupleurum chinensis (<a href="#B40" rid="B40" class=" bibr popnode">Wang et al., 2021</a>). In Arabidopsis, overexpression of AtMYC2 was able to increase plant resistance to Botrytis cinerea (<a href="#B13" rid="B13" class=" bibr popnode">Fu et al., 2020</a>). MYC transcription factors play important regulatory roles in plant growth and development and the response to adverse stresses, and one of them plays a key role in the response to low-temperature stress. Currently, the MYC transcription factor family has been identified in plants such as wheat (<a href="#B3" rid="B3" class=" bibr popnode">Bai et al., 2019</a>), tomato (<a href="#B11" rid="B11" class=" bibr popnode">Feng et al., 2023</a>), poplar (<a href="#B19" rid="B19" class=" bibr popnode">Hu et al., 2023</a>), and tea tree (<a href="#B6" rid="B6" class=" bibr popnode">Chen et al., 2021</a>). However, bioinformatic analysis of the MYC transcription factor family in walnut has not been reported.</p><p class="p p-last">Walnut (<em>Juglans regia</em> L.) belongs to the walnut family of deciduous walnut trees, is one of the world&#x02019;s four major nut species, is also an important species of woody oilseed, and has high medicinal, health and nutritional value. Many places in our country are planted and widely distributed geographically. However, in recent years, the main planting area of Xinjiang walnut has suffered winter frost damage, the frequency of spring late frost damage has increased, the loss of water and drying of walnut light branches, flower buds, new fruits, young fruits and other tissues have suffered damage, and severe tree or even whole plant death has seriously affected the growth and development of walnut plants; thus, walnut plants have become a constraint on the production of outstanding problems (<a href="#B28" rid="B28" class=" bibr popnode">McCully et al., 2004</a>; <a href="#B16" rid="B16" class=" bibr popnode">Han et al., 2022</a>). Therefore, it is necessary to study the cold tolerance mechanism of walnut plants under low-temperature stress and to screen and identify genes that respond to low-temperature stress. Xinjiang wild walnut is the only remaining wild species of walnut in China and is an endangered protected plant in China; it contains many genes for excellent traits and is an important gene pool for the study of walnut cultivation and breeding (<a href="#B47" rid="B47" class=" bibr popnode">Zhang et al., 2020</a>; <a href="#B25" rid="B25" class=" bibr popnode">Li et al., 2023</a>). In this study, 30 walnut MYC family members were screened from published walnut genome-wide data, and bioinformatic analyses of their physicochemical properties, phylogenetic relationships, conserved motifs, chromosomal structural localization, promoter <em>cis</em>-acting elements, and protein interactions were performed. Wild walnut seedlings from Xinjiang were subjected to low-temperature (4&#x000b0;C) stress treatment, and the expression of nine MYC family members in response to low-temperature stress was analyzed via transcriptome data. MYC family members, and the subcellular localization of MYC family members in Xinjiang wild walnut was determined by tobacco injection. At the same time, real-time fluorescence was used to quantitatively analyze the expression patterns of MYC members in different tissues, such as roots, stems and leaves, under low-temperature stress, with the aim of laying a theoretical foundation for in-depth research on the functional mechanisms of walnut MYC transcription factor family members.</p></div><div id="s2" class="tsec sec"><h2 class="head no_bottom_margin" id="s2title">2 Materials and methods</h2><div id="s2-1" class="sec sec-first"><h3 id="s2-1title">2.1 Materials and treatments</h3><p class="p p-first">The test material was wild walnut seedlings (JF) from Xinjiang, which were planted at the Institute of Horticultural Crops, Xinjiang Academy of Agricultural Sciences, and transplanted into organic matter-rich soil and peat soil (2:1) when the seedlings grew well. When the seedlings grew to five compound leaves, the well-grown plants with consistent growth were screened and placed in an artificial climate chamber (4&#x000b0;C, 16&#x000a0;h light) for low-temperature stress treatment. Three to five leaves under the terminal leaf were taken at 0&#x000a0;h (control) and 8&#x000a0;h of treatment, with three biological replicates for each treatment, and then the leaves were placed in an ultralow-temperature refrigerator at &#x02212;80&#x000b0;C after quick freezing in liquid nitrogen to carry out transcriptome sequencing via the Illumina sequencing platform. Transcriptome sequencing using the Illumina sequencing platform.</p><p class="p p-last">At the same time, well-grown plants with consistent growth were screened and subjected to 4&#x000b0;C low-temperature stress treatment, and 2-5 functional leaves under the parietal leaves and tissue materials of roots, stems, and leaves were removed at 0&#x000a0;h, 1&#x000a0;h, 2&#x000a0;h, 4&#x000a0;h, 8&#x000a0;h, 12&#x000a0;h, and 24&#x000a0;h of treatment, respectively, with three biological replicates for each treatment. The plants were subsequently cleaned and placed in liquid nitrogen quickly frozen and then put into &#x02212;80&#x000b0;C ultralow-temperature freezers for storage.</p></div><div id="s2-2" class="sec"><h3 id="s2-2title">2.2 Identification of the walnut MYC transcription factor family</h3><p class="p p-first-last">The whole-genome sequence of walnut (PRJNA291087) was downloaded from NCBI (<a href="https://www.ncbi.nlm.nih.gov/bioproject/291087" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.ncbi.nlm.nih.gov/bioproject/291087</a>). Hidden Markov models of MYC proteins (PF14215 and PF00010) were downloaded from the Pfam (<a href="http://pfam.xfam.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pfam.xfam.org</a>) database, and all MYC protein sequences in the annotated protein files of walnut were compared using TBtools V2.069 software. After that, the protein sequence of each MYC gene in walnut was determined based on the protein database accession number, and to ensure the accuracy of the conserved structural domains of the screened walnut MYC genes, Candidate protein sequences were subjected to structure prediction using the online software NCBI-CDD (<a href="https://www.ncbi.nlm.nih.gov/cdd/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.ncbi.nlm.nih.gov/cdd/</a>) and SMART (<a href="http://smart.emblheidelberg.de/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://smart.emblheidelberg.de/</a>), removing sequences that did not contain the bHLH and bHLH_ MYC_N structural domains, non-full-length and repetitive sequences to identify the MYC transcription factors of walnut.</p></div><div id="s2-3" class="sec"><h3 id="s2-3title">2.3 Physicochemical properties of the walnut MYC transcription factor family</h3><p class="p p-first-last">In order to determine the MYC transcription factor family members in walnuts, refer to Hu(<a href="#B20" rid="B20" class=" bibr popnode">Hu et al., 2023</a>). The basic physicochemical properties of walnut MYC proteins were analysed using the online tool ProtParam (<a href="https://web.expasy.org/protparam/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://web.expasy.org/protparam/</a>), including amino acid length (aa), molecular mass (Da), isoelectric point (pI), hydrophilicity and instability index and protein hydrophobicity. Hydrophobicity, etc.; subcellular localisation of walnut MYC transcription factors was analysed using the online software WoLF PSORT (<a href="https://wolfpsort.hgc.jp/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://wolfpsort.hgc.jp/</a>).</p></div><div id="s2-4" class="sec"><h3 id="s2-4title">2.4 Construction of a phylogenetic tree of the walnut MYC transcription factor family</h3><p class="p p-first-last">Amino acid sequences of eight members of the MYC gene family were obtained from the <em>Arabidopsis thaliana</em> (<a href="https://www.arabidopsis.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.arabidopsis.org/</a>) database; amino acid sequences from the Mauve poplar genome database V4.1 were downloaded using the online website Phytozome (<a href="https://phytozome-next.jgi.doe.gov/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://phytozome-next.jgi.doe.gov/</a>) sequences, downloaded the Hidden Markov Models of MYC proteins (PF14215 and PF00010) using the Pfam (<a href="http://pfam.xfam.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pfam.xfam.org</a>) database, and obtained the amino acid sequences of the poplar MYC transcription factor family by comparing the sequences of all MYC proteins in the poplar annotated protein files using TBtools V2.069; the software Clustal W was used to compare the walnut with Arabidopsis and poplar MYC transcription factors of walnut and <em>Arabidopsis thaliana</em> and poplar using the software Clustal W. Multiple sequence comparison was performed; the protein sequences of the MYC transcription factor families of the three species were calculated using MEGA 7.0 software, and the phylogenetic tree was constructed using the neighbor-joining (NJ) method, with the following execution parameters: BootStrap method 1000; Poisson model; pairwise deletion.</p></div><div id="s2-5" class="sec"><h3 id="s2-5title">2.5 Analysis of conserved structural domains of the walnut MYC transcription factor family</h3><p class="p p-first-last">The conserved structural domains of MYC proteins from walnut were analyzed using the online software MEME (<a href="http://meme-suite.org/tools/meme" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://meme-suite.org/tools/meme</a>), with the number of motif searches set to 10 and the others set to default parameters.</p></div><div id="s2-6" class="sec"><h3 id="s2-6title">2.6 Chromosomal position analysis and gene duplication analysis of the MYC transcription factor family in walnut</h3><p class="p p-first-last">The positional information of the MYC transcription factor family members was extracted from the annotation files of the walnuts, and the positions of the walnut MYC transcription factor family members on the chromosomes were mapped using the online software MG2C_v2.1 (<a href="http://mg2c.iask.in/mg2c_v2.1/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://mg2c.iask.in/mg2c_v2.1/</a>). Genome-wide replication events were obtained using the One Step MCScanX tool in TBtools software (<a href="#B5" rid="B5" class=" bibr popnode">Chen et al., 2020</a>), and the gene duplication relationships of JrMYCs were determined.</p></div><div id="s2-7" class="sec"><h3 id="s2-7title">2.7 Analysis of the promoters and cis-acting elements of the walnut MYC transcription factor family</h3><p class="p p-first-last">The promoter sequences 2000&#x000a0;bp upstream of the walnut MYC transcription factor family members were obtained using TBtools software, and the promoter <em>cis</em>-acting elements were later utilized in the online software Plantcare (<a href="http://bioinformatics.psb.ugent.be/webtools/plantcare/html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://bioinformatics.psb.ugent.be/webtools/plantcare/html</a>) for predictions and visualization of the output using TBtools software.</p></div><div id="s2-8" class="sec"><h3 id="s2-8title">2.8 Walnut MYC transcription factor family protein interaction network</h3><p class="p p-first-last">The prediction of receptor-interacting proteins was performed using the STRING database (<a href="https://string-db.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://string-db.org/</a>) on the protein sequences of the MYC transcription factor family members of walnut, with the species parameter selecting the model plant <em>Arabidopsis thaliana</em> and a threshold value of 0.4.</p></div><div id="s2-9" class="sec"><h3 id="s2-9title">2.9 Construction and subcellular localization analysis of MYC transcription factor family members in Xinjiang wild walnut</h3><p class="p p-first-last">Sequences of each member of the MYC transcription factor family of Xinjiang wild walnut were amplified, stop codons were removed, and the sequences were ligated into a transient expression pCAM35-GFP vector with a GFP fluorescent signal. The primers used were designed with the online tool Vazyme (<a href="https://crm.vazyme.com/cetool/simple.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://crm.vazyme.com/cetool/simple.html</a>). The constructed target gene plasmid was subsequently transformed into <em>Agrobacterium tumefaciens</em> GV3101 via the conventional freeze&#x02012;thaw method. Afterward, the Agrobacterium sap containing the target gene (OD value 0.8&#x02013;1) was centrifuged at 5,000&#x000a0;rpm for 10&#x000a0;min, the supernatant was removed, the injection buffer for transient expression of tobacco was added, and the mixture was allowed to stand at room temperature for 2&#x000a0;h. Four-week-old tobacco plants were injected from the lower epidermis of the leaves by using a 1&#x000a0;mL syringe, the position was marked, the leaves were incubated in the dark for 24&#x000a0;h, and the fluorescence of the leaves was directly observed by using a laser confocal microscope.</p></div><div id="s2-10" class="sec"><h3 id="s2-10title">2.10 RNA extraction and real-time fluorescence quantitative analysis</h3><p class="p p-first-last">Different tissue materials, such as the roots, stems and leaves of Xinjiang wild walnut plants subjected to low-temperature (4&#x000b0;C) stress, were collected, and the total RNA of the test materials was extracted using a Plant Total RNA Extraction Kit (Tiangen, Beijing, China). The quality and purity of the RNA were detected by 1% agarose gel electrophoresis and UV spectrophotometry. Afterward, 1&#x000a0;&#x000b5;g of each sample was reverse transcribed by using a High-Capacity cDNA Reverse Transcription Kit (Thermo Fisher Scientific, Shanghai, China) to reverse transcribe the RNA from a 1&#x000a0;&#x000b5;g sample, which was subsequently stored at &#x02212;20&#x000b0;C. Specific primers were designed using the online tool Vazyme (<a href="https://crm.vazyme.com/cetool/simple.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://crm.vazyme.com/cetool/simple.html</a>) to screen the transcriptome data for nine MYC transcription factor family members in response to low-temperature stress (<a href="#s11" rid="s11" class=" sec">Supplementary Table S1</a>), and ACT 7 was used as an internal reference to design specific primers for the nine MYC transcription factors using a qTOWER3 G fluorescence quantitative PCR instrument (Analytikjena, Germany) and SYBR Green qPCR Master Mix kit (Xavier, Wuhan) methods to analyze the gene expression levels of seedlings in different tissues and in response to low-temperature stress. The qPCR mixture consisted of 7.5&#x000a0;&#x000b5;L of qPCR mix, 0.75&#x000a0;&#x000b5;L of upstream and downstream primers, 2&#x000a0;&#x000b5;L of cDNA template, and 15&#x000a0;&#x000b5;L of ddH<sub>2</sub>O water. The following reaction program was used: predenaturation at 95&#x000b0;C for 10&#x000a0;min, denaturation at 95&#x000b0;C for 15&#x000a0;s, annealing at 60&#x000b0;C for 30&#x000a0;s, 40 cycles, melting curve (95&#x02013;65&#x000b0;C, 0.3&#x000b0;C/15&#x000a0;s), 3 biological replicates, and relative expression was calculated using the 2<sup>&#x02212;&#x00394;&#x00394;CT</sup> method.</p></div><div id="s2-11" class="sec sec-last"><h3 id="s2-11title">2.11 Data analysis</h3><p class="p p-first-last">Data statistics and calculations were performed using Office 2022, data variability analysis was performed using SPSS 26 software, and relative expression heatmaps were plotted using Origin 2018 and TBtools software.</p></div></div><div id="s3" class="tsec sec"><h2 class="head no_bottom_margin" id="s3title">3 Results</h2><div id="s3-1" class="sec sec-first"><h3 id="s3-1title">3.1 Identification of the walnut MYC transcription factor family</h3><p class="p p-first">The screening of walnut genome-wide data based on MYC transcription factors with a conserved bHLH structural domain at the C-terminus and a bHLH_MYC_N structural domain at the N-terminus, excluding proteins with deletions or non-full-length fragments, yielded 30 JrMYC genes, named JrMYC1 to JrMYC30, with a greater number of members than in the model plants <em>Arabidopsis thaliana</em> (8 AtMYC genes) and poplar (10 AtMYC genes) (<a href="#B19" rid="B19" class=" bibr popnode">Hu et al., 2023</a>).</p><p class="p">In this study, 30 members of the MYC transcription factor family were screened in walnut for physicochemical characterization and subcellular localization prediction, and the length of the protein sequences ranged from 484 to 926 aa, with an average of 667.23 aa; the molecular weight of each member ranged from 53.83 to 101.23&#x000a0;kDa, with an average of 74.09&#x000a0;kDa; and the isoelectric point (PI) ranged from 4.82 to 8.09. The isoelectric points (PIs) ranged from 4.82 to 8.09, with an average of 5.67. Among them, there was 1 protein with a PI &#x0003e; 7, accounting for 3.3%, and 29 proteins with a PI &#x0003c; 7, accounting for 96.7%. The instability coefficients ranged from 40.4 to 68.2, with an average of 50.14. The instability coefficients of each member were greater than 40, indicating that walnut MYC proteins may be less stable <em>in vitro</em>. The fat coefficients ranged from 67.36 to 87.21, with an average of 78.96. The fat coefficients determine the thermal stability of the protein, indicating that the walnut MYC protein has greater thermal stability. The hydrophilicity predictions were all negative, indicating that the walnut MYC proteins were all hydrophobic. Subcellular localization prediction revealed that the MYC proteins had roles in the nucleus, chloroplast, and endoplasmic reticulum, 27 of which were localized to the nucleus at 90%, 2 of which were localized to the chloroplast at 6.6%, and 1 of which was localized to the endoplasmic reticulum at 3.3% (<a href="/pmc/articles/PMC11128688/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T1"><h3>TABLE 1</h3><!--caption a7--><div class="caption"><p>Basic information on the walnut MYC transcription factor family.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="top"><tr><th align="center" rowspan="1" colspan="1">Name</th><th align="center" rowspan="1" colspan="1">Gene ID</th><th align="center" rowspan="1" colspan="1">Amino acids (aa)</th><th align="center" rowspan="1" colspan="1">Molecular Weight (KDa)</th><th align="center" rowspan="1" colspan="1">Isoelectric points</th><th align="center" rowspan="1" colspan="1">Instability index</th><th align="center" rowspan="1" colspan="1">Aliphatic index</th><th align="center" rowspan="1" colspan="1">Gravy</th><th align="center" rowspan="1" colspan="1">Location</th></tr></thead><tbody valign="top"><tr><td align="center" rowspan="1" colspan="1">JrMYC1</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576974" class="tag_hotlink" href="/protein/XP_018807981" ref="/protein/XP_018807981"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_018807981","term_id":"1882612461","term_text":"XP_018807981"}}</span>XP_018807981</a></td><td align="center" rowspan="1" colspan="1">656</td><td align="center" rowspan="1" colspan="1">73.78</td><td align="center" rowspan="1" colspan="1">5</td><td align="center" rowspan="1" colspan="1">55.22</td><td align="center" rowspan="1" colspan="1">83.55</td><td align="center" rowspan="1" colspan="1">&#x02212;0.467</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC2</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576998" class="tag_hotlink" href="/protein/XP_035549874" ref="/protein/XP_035549874"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_035549874","term_id":"1882621356","term_text":"XP_035549874"}}</span>XP_035549874</a></td><td align="center" rowspan="1" colspan="1">637</td><td align="center" rowspan="1" colspan="1">71.15</td><td align="center" rowspan="1" colspan="1">5.23</td><td align="center" rowspan="1" colspan="1">58.43</td><td align="center" rowspan="1" colspan="1">87.21</td><td align="center" rowspan="1" colspan="1">&#x02212;0.365</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC3</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576984" class="tag_hotlink" href="/protein/XP_018814174" ref="/protein/XP_018814174"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_018814174","term_id":"1098816063","term_text":"XP_018814174"}}</span>XP_018814174</a></td><td align="center" rowspan="1" colspan="1">738</td><td align="center" rowspan="1" colspan="1">81.98</td><td align="center" rowspan="1" colspan="1">5.24</td><td align="center" rowspan="1" colspan="1">57.77</td><td align="center" rowspan="1" colspan="1">77.07</td><td align="center" rowspan="1" colspan="1">&#x02212;0.572</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC4</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576990" class="tag_hotlink" href="/protein/XP_018852488" ref="/protein/XP_018852488"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_018852488","term_id":"1098785578","term_text":"XP_018852488"}}</span>XP_018852488</a></td><td align="center" rowspan="1" colspan="1">632</td><td align="center" rowspan="1" colspan="1">71.22</td><td align="center" rowspan="1" colspan="1">5.22</td><td align="center" rowspan="1" colspan="1">56.79</td><td align="center" rowspan="1" colspan="1">82.18</td><td align="center" rowspan="1" colspan="1">&#x02212;0.415</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC5</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576994" class="tag_hotlink" href="/protein/XP_035549141" ref="/protein/XP_035549141"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_035549141","term_id":"1882618076","term_text":"XP_035549141"}}</span>XP_035549141</a></td><td align="center" rowspan="1" colspan="1">714</td><td align="center" rowspan="1" colspan="1">79.75</td><td align="center" rowspan="1" colspan="1">5.64</td><td align="center" rowspan="1" colspan="1">67.39</td><td align="center" rowspan="1" colspan="1">74.96</td><td align="center" rowspan="1" colspan="1">&#x02212;0.642</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC6</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576996" class="tag_hotlink" href="/protein/XP_018827229" ref="/protein/XP_018827229"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_018827229","term_id":"1098841058","term_text":"XP_018827229"}}</span>XP_018827229</a></td><td align="center" rowspan="1" colspan="1">759</td><td align="center" rowspan="1" colspan="1">83.93</td><td align="center" rowspan="1" colspan="1">5.83</td><td align="center" rowspan="1" colspan="1">45.17</td><td align="center" rowspan="1" colspan="1">81.78</td><td align="center" rowspan="1" colspan="1">&#x02212;0.388</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC7</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576980" class="tag_hotlink" href="/protein/XP_018825262" ref="/protein/XP_018825262"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_018825262","term_id":"1098837266","term_text":"XP_018825262"}}</span>XP_018825262</a></td><td align="center" rowspan="1" colspan="1">689</td><td align="center" rowspan="1" colspan="1">76.42</td><td align="center" rowspan="1" colspan="1">5.81</td><td align="center" rowspan="1" colspan="1">47.5</td><td align="center" rowspan="1" colspan="1">78.46</td><td align="center" rowspan="1" colspan="1">&#x02212;0.424</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC8</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576991" class="tag_hotlink" href="/protein/XP_018825260" ref="/protein/XP_018825260"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_018825260","term_id":"1098837261","term_text":"XP_018825260"}}</span>XP_018825260</a></td><td align="center" rowspan="1" colspan="1">757</td><td align="center" rowspan="1" colspan="1">84.16</td><td align="center" rowspan="1" colspan="1">5.77</td><td align="center" rowspan="1" colspan="1">47.8</td><td align="center" rowspan="1" colspan="1">80.3</td><td align="center" rowspan="1" colspan="1">&#x02212;0.39</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC9</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576979" class="tag_hotlink" href="/protein/XP_018827230" ref="/protein/XP_018827230"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_018827230","term_id":"1098841060","term_text":"XP_018827230"}}</span>XP_018827230</a></td><td align="center" rowspan="1" colspan="1">697</td><td align="center" rowspan="1" colspan="1">76.85</td><td align="center" rowspan="1" colspan="1">5.84</td><td align="center" rowspan="1" colspan="1">44.37</td><td align="center" rowspan="1" colspan="1">80.52</td><td align="center" rowspan="1" colspan="1">&#x02212;0.408</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC10</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149577002" class="tag_hotlink" href="/protein/XP_035547567" ref="/protein/XP_035547567"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_035547567","term_id":"1882585828","term_text":"XP_035547567"}}</span>XP_035547567</a></td><td align="center" rowspan="1" colspan="1">733</td><td align="center" rowspan="1" colspan="1">80.93</td><td align="center" rowspan="1" colspan="1">5.71</td><td align="center" rowspan="1" colspan="1">43.65</td><td align="center" rowspan="1" colspan="1">80.95</td><td align="center" rowspan="1" colspan="1">&#x02212;0.384</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC11</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576997" class="tag_hotlink" href="/protein/XP_018827228" ref="/protein/XP_018827228"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_018827228","term_id":"1098841056","term_text":"XP_018827228"}}</span>XP_018827228</a></td><td align="center" rowspan="1" colspan="1">760</td><td align="center" rowspan="1" colspan="1">84.05</td><td align="center" rowspan="1" colspan="1">5.83</td><td align="center" rowspan="1" colspan="1">45.45</td><td align="center" rowspan="1" colspan="1">81.67</td><td align="center" rowspan="1" colspan="1">&#x02212;0.392</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC12</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576981" class="tag_hotlink" href="/protein/XP_018817026" ref="/protein/XP_018817026"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_018817026","term_id":"1098821604","term_text":"XP_018817026"}}</span>XP_018817026</a></td><td align="center" rowspan="1" colspan="1">926</td><td align="center" rowspan="1" colspan="1">101.23</td><td align="center" rowspan="1" colspan="1">6.16</td><td align="center" rowspan="1" colspan="1">40.4</td><td align="center" rowspan="1" colspan="1">81.45</td><td align="center" rowspan="1" colspan="1">&#x02212;0.356</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC13</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576989" class="tag_hotlink" href="/protein/XP_018846265" ref="/protein/XP_018846265"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_018846265","term_id":"1098766181","term_text":"XP_018846265"}}</span>XP_018846265</a></td><td align="center" rowspan="1" colspan="1">732</td><td align="center" rowspan="1" colspan="1">81.52</td><td align="center" rowspan="1" colspan="1">6.13</td><td align="center" rowspan="1" colspan="1">41.76</td><td align="center" rowspan="1" colspan="1">79.13</td><td align="center" rowspan="1" colspan="1">&#x02212;0.384</td><td align="left" rowspan="1" colspan="1">Endoplasmic Reticulum</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC14</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576999" class="tag_hotlink" href="/protein/XP_035549142" ref="/protein/XP_035549142"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_035549142","term_id":"1882618078","term_text":"XP_035549142"}}</span>XP_035549142</a></td><td align="center" rowspan="1" colspan="1">658</td><td align="center" rowspan="1" colspan="1">73.45</td><td align="center" rowspan="1" colspan="1">5.53</td><td align="center" rowspan="1" colspan="1">68.2</td><td align="center" rowspan="1" colspan="1">75.11</td><td align="center" rowspan="1" colspan="1">&#x02212;0.686</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC15</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576978" class="tag_hotlink" href="/protein/XP_035545463" ref="/protein/XP_035545463"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_035545463","term_id":"1882601199","term_text":"XP_035545463"}}</span>XP_035545463</a></td><td align="center" rowspan="1" colspan="1">724</td><td align="center" rowspan="1" colspan="1">80.75</td><td align="center" rowspan="1" colspan="1">5.23</td><td align="center" rowspan="1" colspan="1">50.18</td><td align="center" rowspan="1" colspan="1">80.14</td><td align="center" rowspan="1" colspan="1">&#x02212;0.314</td><td align="left" rowspan="1" colspan="1">Chloroplast</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC16</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576977" class="tag_hotlink" href="/protein/XP_018858098" ref="/protein/XP_018858098"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_018858098","term_id":"1882589648","term_text":"XP_018858098"}}</span>XP_018858098</a></td><td align="center" rowspan="1" colspan="1">842</td><td align="center" rowspan="1" colspan="1">92.42</td><td align="center" rowspan="1" colspan="1">4.82</td><td align="center" rowspan="1" colspan="1">42.29</td><td align="center" rowspan="1" colspan="1">78.97</td><td align="center" rowspan="1" colspan="1">&#x02212;0.338</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC17</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149577001" class="tag_hotlink" href="/protein/XP_018858097" ref="/protein/XP_018858097"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_018858097","term_id":"1882589646","term_text":"XP_018858097"}}</span>XP_018858097</a></td><td align="center" rowspan="1" colspan="1">843</td><td align="center" rowspan="1" colspan="1">92.61</td><td align="center" rowspan="1" colspan="1">4.85</td><td align="center" rowspan="1" colspan="1">42.25</td><td align="center" rowspan="1" colspan="1">78.41</td><td align="center" rowspan="1" colspan="1">&#x02212;0.344</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC18</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576976" class="tag_hotlink" href="/protein/XP_018824283" ref="/protein/XP_018824283"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_018824283","term_id":"1882620195","term_text":"XP_018824283"}}</span>XP_018824283</a></td><td align="center" rowspan="1" colspan="1">845</td><td align="center" rowspan="1" colspan="1">93.71</td><td align="center" rowspan="1" colspan="1">4.93</td><td align="center" rowspan="1" colspan="1">51.06</td><td align="center" rowspan="1" colspan="1">82.73</td><td align="center" rowspan="1" colspan="1">&#x02212;0.316</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC19</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149577000" class="tag_hotlink" href="/protein/XP_018815047" ref="/protein/XP_018815047"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_018815047","term_id":"1098817752","term_text":"XP_018815047"}}</span>XP_018815047</a></td><td align="center" rowspan="1" colspan="1">494</td><td align="center" rowspan="1" colspan="1">55.39</td><td align="center" rowspan="1" colspan="1">6.01</td><td align="center" rowspan="1" colspan="1">50.4</td><td align="center" rowspan="1" colspan="1">82.29</td><td align="center" rowspan="1" colspan="1">&#x02212;0.536</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC20</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576987" class="tag_hotlink" href="/protein/XP_018841495" ref="/protein/XP_018841495"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_018841495","term_id":"1098755559","term_text":"XP_018841495"}}</span>XP_018841495</a></td><td align="center" rowspan="1" colspan="1">616</td><td align="center" rowspan="1" colspan="1">69.3</td><td align="center" rowspan="1" colspan="1">5.3</td><td align="center" rowspan="1" colspan="1">50.91</td><td align="center" rowspan="1" colspan="1">70.26</td><td align="center" rowspan="1" colspan="1">&#x02212;0.681</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC21</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576995" class="tag_hotlink" href="/protein/XP_018841494" ref="/protein/XP_018841494"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_018841494","term_id":"1098755557","term_text":"XP_018841494"}}</span>XP_018841494</a></td><td align="center" rowspan="1" colspan="1">659</td><td align="center" rowspan="1" colspan="1">74.31</td><td align="center" rowspan="1" colspan="1">5.38</td><td align="center" rowspan="1" colspan="1">49.78</td><td align="center" rowspan="1" colspan="1">71.44</td><td align="center" rowspan="1" colspan="1">&#x02212;0.583</td><td align="left" rowspan="1" colspan="1">Chloroplast</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC22</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576992" class="tag_hotlink" href="/protein/XP_018817566" ref="/protein/XP_018817566"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_018817566","term_id":"1098822604","term_text":"XP_018817566"}}</span>XP_018817566</a></td><td align="center" rowspan="1" colspan="1">684</td><td align="center" rowspan="1" colspan="1">74.84</td><td align="center" rowspan="1" colspan="1">5.56</td><td align="center" rowspan="1" colspan="1">53.51</td><td align="center" rowspan="1" colspan="1">70.29</td><td align="center" rowspan="1" colspan="1">&#x02212;0.599</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC23</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576982" class="tag_hotlink" href="/protein/XP_018811463" ref="/protein/XP_018811463"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_018811463","term_id":"1098810843","term_text":"XP_018811463"}}</span>XP_018811463</a></td><td align="center" rowspan="1" colspan="1">652</td><td align="center" rowspan="1" colspan="1">71.47</td><td align="center" rowspan="1" colspan="1">5.3</td><td align="center" rowspan="1" colspan="1">52.91</td><td align="center" rowspan="1" colspan="1">67.36</td><td align="center" rowspan="1" colspan="1">&#x02212;0.653</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC24</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576973" class="tag_hotlink" href="/protein/XP_018824308" ref="/protein/XP_018824308"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_018824308","term_id":"1882620916","term_text":"XP_018824308"}}</span>XP_018824308</a></td><td align="center" rowspan="1" colspan="1">489</td><td align="center" rowspan="1" colspan="1">54.23</td><td align="center" rowspan="1" colspan="1">6.05</td><td align="center" rowspan="1" colspan="1">45.7</td><td align="center" rowspan="1" colspan="1">85.71</td><td align="center" rowspan="1" colspan="1">&#x02212;0.391</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC25</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576985" class="tag_hotlink" href="/protein/XP_018824309" ref="/protein/XP_018824309"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_018824309","term_id":"1098835459","term_text":"XP_018824309"}}</span>XP_018824309</a></td><td align="center" rowspan="1" colspan="1">498</td><td align="center" rowspan="1" colspan="1">55.67</td><td align="center" rowspan="1" colspan="1">5.88</td><td align="center" rowspan="1" colspan="1">52.16</td><td align="center" rowspan="1" colspan="1">80.62</td><td align="center" rowspan="1" colspan="1">&#x02212;0.452</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC26</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576993" class="tag_hotlink" href="/protein/XP_018860395" ref="/protein/XP_018860395"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_018860395","term_id":"1882635712","term_text":"XP_018860395"}}</span>XP_018860395</a></td><td align="center" rowspan="1" colspan="1">494</td><td align="center" rowspan="1" colspan="1">55.7</td><td align="center" rowspan="1" colspan="1">5.59</td><td align="center" rowspan="1" colspan="1">45.19</td><td align="center" rowspan="1" colspan="1">85.81</td><td align="center" rowspan="1" colspan="1">&#x02212;0.434</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC27</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576986" class="tag_hotlink" href="/protein/XP_018814191" ref="/protein/XP_018814191"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_018814191","term_id":"1098816098","term_text":"XP_018814191"}}</span>XP_018814191</a></td><td align="center" rowspan="1" colspan="1">487</td><td align="center" rowspan="1" colspan="1">54.49</td><td align="center" rowspan="1" colspan="1">8.09</td><td align="center" rowspan="1" colspan="1">52.85</td><td align="center" rowspan="1" colspan="1">76.12</td><td align="center" rowspan="1" colspan="1">&#x02212;0.468</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC28</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576983" class="tag_hotlink" href="/protein/XP_018818377" ref="/protein/XP_018818377"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_018818377","term_id":"1098824127","term_text":"XP_018818377"}}</span>XP_018818377</a></td><td align="center" rowspan="1" colspan="1">611</td><td align="center" rowspan="1" colspan="1">67.83</td><td align="center" rowspan="1" colspan="1">6.44</td><td align="center" rowspan="1" colspan="1">48.77</td><td align="center" rowspan="1" colspan="1">76.78</td><td align="center" rowspan="1" colspan="1">&#x02212;0.536</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC29</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576975" class="tag_hotlink" href="/protein/XP_035548572" ref="/protein/XP_035548572"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_035548572","term_id":"1882615670","term_text":"XP_035548572"}}</span>XP_035548572</a></td><td align="center" rowspan="1" colspan="1">484</td><td align="center" rowspan="1" colspan="1">53.83</td><td align="center" rowspan="1" colspan="1">6</td><td align="center" rowspan="1" colspan="1">50.58</td><td align="center" rowspan="1" colspan="1">80.79</td><td align="center" rowspan="1" colspan="1">&#x02212;0.502</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr><tr><td align="center" rowspan="1" colspan="1">JrMYC30</td><td align="center" rowspan="1" colspan="1"><a id="__tag_1149576988" class="tag_hotlink" href="/protein/XP_018827272" ref="/protein/XP_018827272"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_018827272","term_id":"1098841135","term_text":"XP_018827272"}}</span>XP_018827272</a></td><td align="center" rowspan="1" colspan="1">507</td><td align="center" rowspan="1" colspan="1">56.02</td><td align="center" rowspan="1" colspan="1">5.76</td><td align="center" rowspan="1" colspan="1">45.79</td><td align="center" rowspan="1" colspan="1">76.88</td><td align="center" rowspan="1" colspan="1">&#x02212;0.453</td><td align="left" rowspan="1" colspan="1">Nucleus</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140430619060864"><a target="object" rel="noopener" href="/pmc/articles/PMC11128688/table/T1/?report=objectonly">Open in a separate window</a></div></div></div><div id="s3-2" class="sec"><h3 id="s3-2title">3.2 Phylogenetic tree analysis</h3><p class="p p-first">To reveal the evolutionary relationship between the MYC transcription factor families of walnut and other species, evolutionary trees of walnut, <em>Arabidopsis thaliana</em> and poplar were constructed based on multiple sequence comparisons using the neighbor&#x02013;joining (NJ) method in MEGA 7.0 software (<a href="/pmc/articles/PMC11128688/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). The evolutionary tree showed that each member of the MYC transcription factor family of the three species was classified into six subfamilies (Groups I, II, III, IV, V, and VI), and members of the walnut MYC transcription factor family were distributed in all the groups, which contained 5, 2, 2, 6, 3 and 12 walnut MYC proteins, respectively. The branching of members of the walnut MYC transcription factor family varied, and compared to <em>Arabidopsis thaliana</em>, walnut is closer in homology to poplar. It has been reported that homology and phylogenetic relationships between species can help to predict the potential functions of genes (<a href="#B47" rid="B47" class=" bibr popnode">Zhang et al., 2020</a>). Currently, in the study of the poplar MYC transcription factor family, we found that the expression of the <em>PtrMYC14a and PtbHLH14b</em> genes could respond to high salt and drought stresses (<a href="#B20" rid="B20" class=" bibr popnode">Hu et al., 2023</a>), and that of <em>JrMYC24</em>, <em>JrMYC25, and JrMYC27</em> was close to that of <em>PtrMYC14a and PtbHLH14b</em>; therefore, we hypothesized that <em>JrMYC24</em>, <em>JrMYC25</em>, and <em>JrMYC27</em> could respond to high salt and drought stresses. Can respond to high salt and drought stresses. <em>PtrMYC2b</em> gene expression is induced by jasmonic acid and responds to low temperature and high salt stresses (<a href="#B19" rid="B19" class=" bibr popnode">Hu et al., 2023</a>). The evolutionary tree shows that <em>JrMYC22</em> and <em>JrMYC23</em> have high homology with <em>PtrMYC2b</em>, and it is hypothesized that <em>JrMYC22</em> and <em>JrMYC23</em> may have both cold resistance and salt resistance.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC11128688/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140430621963936"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11128688_fgene-15-1399721-g001.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fgene-15-1399721-g001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11128688/bin/fgene-15-1399721-g001.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140430621963936"><a target="object" rel="noopener" href="/pmc/articles/PMC11128688/figure/F1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC11128688/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">FIGURE 1</a></div><!--caption a7--><div class="caption"><p>Phylogenetic analysis of the walnut MYC transcription factor family. Different colors represent different groups.</p></div></div></div></div><div id="s3-3" class="sec"><h3 id="s3-3title">3.3 Chromosome localization analysis</h3><p class="p p-first">Based on the whole walnut genome and annotation information, the chromosomal localization of MYC in the walnut variety was mapped using the MG2C online website. The map shows that various members of the walnut MYC transcription factor family are located on chromosomes Chr1, Chr2, Chr3, Chr4, Chr7, Chr8, Chr9, Chr10, Chr11, Chr12, Chr14, and Chr16, with Chr1 (<em>JrMYC9, JrMYC11, JrMYC30</em>), Chr2 (<em>JrMYC4, JrMYC17, JrMYC22, JrMYC27</em>), Chr7 (<em>JrMYC1, JrMYC28, JrMYC29</em>), Chr8 (<em>JrMYC5, JrMYC14</em>), Chr9 (<em>JrMYC2, JrMYC18, JrMYC23, JrMYC24, JrMYC25</em>), Chr10 (<em>JrMYC7, JrMYC8</em>), and Chr16 (<em>JrMYC19, JrMYC21</em>) having two or more JrMYC genes (<a href="/pmc/articles/PMC11128688/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). Genes on chromosomes with distances less than 100&#x000a0;kb are tandemly duplicated (<a href="#B21" rid="B21" class=" bibr popnode">Kapli et al., 2020</a>). Analysis revealed tandem duplication of the walnut MYC transcription factor members <em>JrMYC9 and JrMYC11</em> on Chr1; <em>JrMYC5 and JrMYC14</em> on Chr8; <em>JrMYC18, JrMYC24, and JrMYC25</em> on Chr9; and <em>JrMYC7 and JrMYC8</em> on Chr10.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC11128688/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140430621951408"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11128688_fgene-15-1399721-g002.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fgene-15-1399721-g002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11128688/bin/fgene-15-1399721-g002.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140430621951408"><a target="object" rel="noopener" href="/pmc/articles/PMC11128688/figure/F2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC11128688/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">FIGURE 2</a></div><!--caption a7--><div class="caption"><p>Chromosomal distribution of the walnut MYC gene family. Red indicates genes in which tandem duplication occurred.</p></div></div></div><p class="p">Gene replication is the main method of gene recombination and amplification and can potentially affect the evolution and adaptation of species. In this study, collinearity analysis of the walnut MYC transcription factor family was conducted by using TBtools software (<a href="/pmc/articles/PMC11128688/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>), and 13 members of the walnut MYC transcription factor family exhibited gene repetition events, indicating that the walnut MYC transcription factor family has diverse functions.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC11128688/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140430621937280"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11128688_fgene-15-1399721-g003.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fgene-15-1399721-g003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11128688/bin/fgene-15-1399721-g003.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140430621937280"><a target="object" rel="noopener" href="/pmc/articles/PMC11128688/figure/F3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC11128688/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">FIGURE 3</a></div><!--caption a7--><div class="caption"><p>Gene replication analysis of MYC transcription factor family members in walnut. The chromosome number is shown on each chromosome, and the red line represents 8 pairs of paralogous genes: <em>JrMYC8</em> and <em>JrMYC11</em>, <em>JrMYC1</em> and <em>JrMYC4</em>, <em>JrMYC27</em> and <em>JrMYC29</em>, <em>JrMYC22</em> and <em>JrMYC23</em>, <em>JrMYC24</em> and <em>JrMYC27</em>, <em>JrMYC17</em> and <em>JrMYC18</em>, <em>JrMYC13</em> and <em>JrMYC15</em>, <em>JrMYC24</em> and <em>JrMYC29</em>.</p></div></div></div></div><div id="s3-4" class="sec"><h3 id="s3-4title">3.4 Phylogenetic developmental tree and sequence structure analysis of walnut MYC proteins</h3><p class="p p-first">To further analyze the motif composition and conserved structural domains of walnut MYC transcription factor family members. First, the protein sequences of 30 members were phylogenetically analyzed using MEGA 7.0 software (<a href="/pmc/articles/PMC11128688/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4A</span></a>), and the members were classified into six subfamilies (Groups I, II, III, IV, V, and VI), in which the genes had a similar topology to the phylogenetic developmental tree of the MYC transcription factor family constructed for three plants (<a href="/pmc/articles/PMC11128688/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). Then, the conserved structural domains of the 30 MYC proteins were predicted using the MEME online website, with the motif value set to 10, and analyzed (<a href="/pmc/articles/PMC11128688/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figures 4B&#x02013;D</span></a>). All the structural domains of the walnut MYC proteins appeared to be highly conserved during the evolutionary process, and Motif1, Motif2, and Motif4 were distributed among all the members. Among them, Motif1, Motif4, Motif2, Motif3, Motif5, and Motif9 are the conserved characteristic structural domains of the MYC family members bHLH and bHLH_MYC_N, respectively. In addition, conserved structural domains are essential for transcription factor regulation and are able to maintain the tertiary structural conformation of proteins, and the DNA recognition and binding process plays a key role in the binding process of DNA recognition. The conserved structural motifs of most of the walnut MYC genes share a high degree of similarity, and it is hypothesized that there may be functional redundancy in these genes. The JrMYC12, JrMYC13, JrMYC15, JrMYC16, JrMYC17, and JrMYC18 proteins lacked Motif7 and Motif8, and it was hypothesized that these genes might have different functional response mechanisms.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC11128688/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140430583067392"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11128688_fgene-15-1399721-g004.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fgene-15-1399721-g004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11128688/bin/fgene-15-1399721-g004.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140430583067392"><a target="object" rel="noopener" href="/pmc/articles/PMC11128688/figure/F4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC11128688/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">FIGURE 4</a></div><!--caption a7--><div class="caption"><p>Phylogenetic developmental tree of walnut MYC proteins, motif composition, conserved structural domains and motif sequence signatures. <strong>(A)</strong> A phylogenetic developmental tree of the walnut MYC transcription factor family was constructed using the NJ method. <strong>(B)</strong> Motif identification of walnut MYC transcription factor family members using the MEME online website, with numbers 1&#x02013;10 representing different motifs. <strong>(C)</strong> Conserved structural domains of walnut MYC transcription factor family members. <strong>(D)</strong> Sequence signatures of the 10 motifs, with the horizontal coordinate indicating the amino acid with the highest frequency of occurrence and the vertical coordinate indicating the frequency corresponding to the amino acid of occurrence.</p></div></div></div></div><div id="s3-5" class="sec"><h3 id="s3-5title">3.5 Analysis of promoter cis-acting elements</h3><p class="p p-first">Prediction of promoter <em>cis</em>-acting elements for each member of the walnut MYC transcription factor family using the PlantCare database revealed that there were 18 genes with promoter <em>cis</em>-acting elements related to functions such as light response, hormone response, adversity stress, binding sites, and biosynthesis, and hormone-related <em>cis</em>-acting elements were the most numerous and widely distributed. Among them, 15 members contained growth hormone-responsive <em>cis</em>-acting elements, 3 with an AuxRR core and 15 with a TGA element; 15 members contained salicylic acid-responsive <em>cis</em>-acting elements (TCA elements); 15 members contained gibberellin-responsive <em>cis</em>-acting elements; 19 members contained jasmonic acid-responsive <em>cis</em>-acting elements; 19 members contained a CGTCA motif and a TGACG motif; 26 members contained abscisic acid response <em>cis</em>-acting elements (ABREs); 17 members contained <em>cis</em>-acting elements involved in defense and stress responses; 6 members contained TC-rich elements; 12 members had a TATC box; 18 members contained low-temperature response <em>cis</em>-acting elements (LTRs); and light-responsive <em>cis</em>-acting elements were more abundant, including G-boxes, Sp1s, GT motifs, ACEs and 3-AF1 binding sites, with a total of 26 members containing these elements (<a href="/pmc/articles/PMC11128688/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC11128688/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140430583055152"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11128688_fgene-15-1399721-g005.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fgene-15-1399721-g005.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11128688/bin/fgene-15-1399721-g005.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140430583055152"><a target="object" rel="noopener" href="/pmc/articles/PMC11128688/figure/F5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC11128688/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">FIGURE 5</a></div><!--caption a7--><div class="caption"><p>Predicted promoter <em>cis</em>-acting elements of walnut MYC transcription factor family members. <strong>(A)</strong> Statistics of the number of each member of the walnut MYC transcription factor family containing promoter <em>cis</em>-acting elements. <strong>(B)</strong> The location and type of promoter <em>cis</em>-acting elements of each member of the walnut MYC transcription factor family.</p></div></div></div></div><div id="s3-6" class="sec"><h3 id="s3-6title">3.6 Protein interaction analysis of walnut MYC transcription factor family members</h3><p class="p p-first">To further investigate the protein interactions among the members of the walnut MYC transcription factor family, the STRING database (<a href="https://string-db.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://string-db.org/</a>) was used to predict the interactions among 30 walnut MYC transcription factor family members after redundancy (<a href="/pmc/articles/PMC11128688/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Figure 6</span></a>), with the model plant <em>Arabidopsis thaliana</em> serving as the reference species. The results showed that most of the walnut MYC transcription factor family proteins were able to interact with each other, among which 12 walnut MYC transcription factor family members had close protein homology with Arabidopsis, such as MYC2, MYC3, bHLH3, bHLH13, bHLH14, bHLH155, bHLH157, GL3, AMS, TT8, EMB1444, and LHW in Arabidopsis, Based on the number of reciprocal lines, MYC2 and MYC3 were found to have the strongest reciprocal network relationships, and MYC2 and MYC3 were reciprocal homologous proteins with JrMYC25 and JrMYC29 in walnut. It has been shown that in <em>Arabidopsis thaliana</em>, MYC2 has similar DNA-binding specificity to MYC3 and MYC4, which interact to activate the jasmonic acid signalling pathway and regulate the plant response to adversity stress (<a href="#B31" rid="B31" class=" bibr popnode">Niu et al., 2011</a>; <a href="#B36" rid="B36" class=" bibr popnode">Schweizer et al., 2013</a>; <a href="#B14" rid="B14" class=" bibr popnode">Gao et al., 2016</a>), and that overexpressed AtMYC2, AtMYC67, and AtMYC70 are induced by dehydration, high salt, and low-temperature stresses to activate the promoters of downstream functional genes in order to enhance plant resistance (<a href="#B1" rid="B1" class=" bibr popnode">Abe et al., 1997</a>; <a href="#B37" rid="B37" class=" bibr popnode">Shinozaki and Yamaguchi-Shinozaki, 2007</a>). Taken together, this suggests that walnut JrMYC25 and JrMYC29 may be hypothesised to function similarly to their Arabidopsis homologues, and that the prediction of protein interactions through members of the walnut MYC transcription factor family could help to further explore the functions played by walnut proteins.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F6" co-legend-rid="lgnd_F6"><a href="/pmc/articles/PMC11128688/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140430583041728"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11128688_fgene-15-1399721-g006.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fgene-15-1399721-g006.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11128688/bin/fgene-15-1399721-g006.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140430583041728"><a target="object" rel="noopener" href="/pmc/articles/PMC11128688/figure/F6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F6"><div><a class="figpopup" href="/pmc/articles/PMC11128688/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6">FIGURE 6</a></div><!--caption a7--><div class="caption"><p>Protein interaction network map of walnut MYC transcription factor family members. <strong>(A)</strong> Predicted protein interactions of walnut MYC transcription factor family proteins with Arabidopsis as the reference species and named after homologous proteins in Arabidopsis. <strong>(B)</strong> Predicted protein interaction map of walnut MYC transcription factor family proteins using <em>Arabidopsis thaliana</em> as the reference species and named after walnut.</p></div></div></div></div><div id="s3-7" class="sec"><h3 id="s3-7title">3.7 Analysis of the expression pattern of the MYC transcription factor family in walnut under low-temperature stress</h3><p class="p p-first">Transcriptome data analysis of walnut plants under low-temperature stress revealed (<a href="/pmc/articles/PMC11128688/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Figure 7</span></a>) that most of the members of the MYC transcription factor family exhibited high levels of expression, and a few members exhibited low levels of expression. Among these genes, <em>JrMYC22</em> and <em>JrMYC28</em> were highly expressed, <em>JrMYC22</em> was highly expressed at 0&#x000a0;h and 8&#x000a0;h, and <em>JrMYC28</em> was highly expressed at 8&#x000a0;h. These findings suggest that <em>JrMYC28</em> may play a key role in the response to low-temperature stress and that members of the MYC transcription factor family are critical for regulating the response to low-temperature stress.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F7" co-legend-rid="lgnd_F7"><a href="/pmc/articles/PMC11128688/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140430583033840"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11128688_fgene-15-1399721-g007.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fgene-15-1399721-g007.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11128688/bin/fgene-15-1399721-g007.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140430583033840"><a target="object" rel="noopener" href="/pmc/articles/PMC11128688/figure/F7/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F7"><div><a class="figpopup" href="/pmc/articles/PMC11128688/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7">FIGURE 7</a></div><!--caption a7--><div class="caption"><p>Relative expression of MYC transcription factor family members in the transcriptome.</p></div></div></div></div><div id="s3-8" class="sec"><h3 id="s3-8title">3.8 Subcellular localization of MYC transcription factor family members in Xinjiang wild walnut</h3><p class="p p-first">The transcriptome data of Xinjiang wild walnut plants in response to low-temperature stress were utilized to screen nine MYC transcription factor family members, namely, <em>JrMYC28, JrMYC31, JrMYC4, JrMYC32, JrMYC33, JrMYC34, JrMYC35, JrMYC36</em>, and <em>JrMYC37</em>. To verify the predicted subcellular localization of the nine MYC transcription factor family members. Therefore, in this study, the gene sequences of these members were cloned and ligated into the pCAM35-GFP vector with GFP fluorescent signals, allowing these genes to be fused to pCAM35-GFP and transiently expressed in tobacco. Subcellular localization revealed that these members were significantly fluorescent in the nucleus, and these nine genes had high expression levels in the nucleus, suggesting that the nucleus may be the site where members of the walnut MYC transcription factor family mainly function (<a href="/pmc/articles/PMC11128688/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span>Figure 8</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F8" co-legend-rid="lgnd_F8"><a href="/pmc/articles/PMC11128688/figure/F8/" target="figure" rid-figpopup="F8" rid-ob="ob-F8"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140430583027120"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11128688_fgene-15-1399721-g008.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fgene-15-1399721-g008.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11128688/bin/fgene-15-1399721-g008.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140430583027120"><a target="object" rel="noopener" href="/pmc/articles/PMC11128688/figure/F8/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F8"><div><a class="figpopup" href="/pmc/articles/PMC11128688/figure/F8/" target="figure" rid-figpopup="F8" rid-ob="ob-F8">FIGURE 8</a></div><!--caption a7--><div class="caption"><p>Subcellular localization of MYC transcription factor family members in Xinjiang wild walnut; scale bar, 100&#x000a0;&#x003bc;m.</p></div></div></div></div><div id="s3-9" class="sec"><h3 id="s3-9title">3.9 Expression patterns of MYC transcription factor family members in wild walnuts from Xinjiang under low-temperature stress</h3><p class="p p-first">To further understand the expression patterns of MYC transcription factor family members in Xinjiang wild walnut screened by transcriptome analysis under low-temperature stress, fluorescence quantitative PCR was used to determine the expression levels of the members in response to low-temperature stress at different time intervals in Xinjiang wild walnut seedlings treated at 4&#x000b0;C for 0&#x000a0;h, 1&#x000a0;h, 2&#x000a0;h, 4&#x000a0;h, 8&#x000a0;h, 12&#x000a0;h and 24&#x000a0;h (<a href="/pmc/articles/PMC11128688/figure/F9/" target="figure" class="fig-table-link figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span>Figure 9</span></a>). The results showed that all of these walnut MYC transcription factor family members reached the highest expression level at 8&#x000a0;h after low-temperature stress, and there was variability in the response time and fold increase in the expression of these members. Moreover, the relative expression of these genes in Xinjiang wild walnut increased, among which <em>JrMYC28, JrMYC31, JrMYC33, JrMYC34 and JrMYC35</em> in Xinjiang wild walnut exhibited increased expression at 8&#x000a0;h. Taken together, this suggests that <em>JrMYC28</em>, <em>JrMYC31</em>, <em>JrMYC33</em>, <em>JrMYC34</em> and <em>JrMYC35</em> are hypothesised to potentially play key roles in responding to low-temperature stress.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F9" co-legend-rid="lgnd_F9"><a href="/pmc/articles/PMC11128688/figure/F9/" target="figure" rid-figpopup="F9" rid-ob="ob-F9"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140430583017040"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11128688_fgene-15-1399721-g009.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fgene-15-1399721-g009.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11128688/bin/fgene-15-1399721-g009.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140430583017040"><a target="object" rel="noopener" href="/pmc/articles/PMC11128688/figure/F9/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F9"><div><a class="figpopup" href="/pmc/articles/PMC11128688/figure/F9/" target="figure" rid-figpopup="F9" rid-ob="ob-F9">FIGURE 9</a></div><!--caption a7--><div class="caption"><p>Expression levels of MYC transcription factor family members in wild walnut plants in Xinjiang in response to low-temperature stress.</p></div></div></div></div><div id="s3-10" class="sec sec-last"><h3 id="s3-10title">3.10 Expression patterns of MYC transcription factor family members in different tissues of wild walnut plants from Xinjiang</h3><p class="p p-first">To investigate the expression patterns of the nine MYC transcription factor family members of Xinjiang wild walnut screened from transcriptome data in different tissues, the relative expression of each member in roots, stems and leaves was analysed (<a href="/pmc/articles/PMC11128688/figure/F10/" target="figure" class="fig-table-link figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Figure 10</span></a>; <a href="/pmc/articles/PMC11128688/figure/F11/" target="figure" class="fig-table-link figpopup" rid-figpopup="F11" rid-ob="ob-F11" co-legend-rid="lgnd_F11"><span>Figure 11</span></a>). The results showed that each member of the MYC transcription factor family of Xinjiang wild walnut had the highest expression level in leaves, and the relative expression of <em>JrMYC28</em>, <em>JrMYC33</em>, <em>JrMYC34</em>, <em>JrMYC35</em>, and <em>JrMYC36</em> in leaves of Xinjiang wild walnut reached more than 15, which were upregulated genes, and the comprehensive analysis speculated that <em>JrMYC28</em>, <em>JrMYC33</em>, <em>JrMYC34</em>, <em>JrMYC35</em> and <em>JrMYC36</em> genes may play a crucial role in the regulation in response to low temperature stress.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F10" co-legend-rid="lgnd_F10"><a href="/pmc/articles/PMC11128688/figure/F10/" target="figure" rid-figpopup="F10" rid-ob="ob-F10"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140430583002720"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11128688_fgene-15-1399721-g010.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fgene-15-1399721-g010.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11128688/bin/fgene-15-1399721-g010.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140430583002720"><a target="object" rel="noopener" href="/pmc/articles/PMC11128688/figure/F10/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F10"><div><a class="figpopup" href="/pmc/articles/PMC11128688/figure/F10/" target="figure" rid-figpopup="F10" rid-ob="ob-F10">FIGURE 10</a></div><!--caption a7--><div class="caption"><p>Expression levels of MYC transcription factor family members in different tissues of Xinjiang wild walnut. <strong>(A)</strong> Structural thermograms of different tissues of annual walnut seedlings. <strong>(B)</strong> Heatmaps were plotted as the mean values by TBtools software, with red representing high expression levels and blue representing low expression levels.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F11" co-legend-rid="lgnd_F11"><a href="/pmc/articles/PMC11128688/figure/F11/" target="figure" rid-figpopup="F11" rid-ob="ob-F11"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140430582999952"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11128688_fgene-15-1399721-g011.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fgene-15-1399721-g011.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11128688/bin/fgene-15-1399721-g011.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140430582999952"><a target="object" rel="noopener" href="/pmc/articles/PMC11128688/figure/F11/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F11"><div><a class="figpopup" href="/pmc/articles/PMC11128688/figure/F11/" target="figure" rid-figpopup="F11" rid-ob="ob-F11">FIGURE 11</a></div><!--caption a7--><div class="caption"><p>Expression levels of MYC transcription factor family members in different tissues of wild walnut in Xinjiang. Light blue represents roots, yellow represents stems, and blue represents leaves.</p></div></div></div></div></div><div id="s4" class="tsec sec"><h2 class="head no_bottom_margin" id="s4title">4 Discussion</h2><p class="p p-first">Phytohormones play a key role in adverse stress conditions, and jasmonic acid is an important phytohormone in plants. JA and its derivatives are important signaling molecules involved in the regulation of plant responses to adverse stresses such as low temperature, high salt and drought (<a href="#B27" rid="B27" class=" bibr popnode">Lyons et al., 2013</a>; <a href="#B10" rid="B10" class=" bibr popnode">F&#x000e0;bregas and Fernie, 2021</a>; <a href="#B33" rid="B33" class=" bibr popnode">Raza et al., 2021</a>). When plants are subjected to adverse conditions, the jasmonic acid signaling pathway activates related transcription factors to induce the expression of downstream functional genes, thereby increasing plant resistance to adverse conditions (<a href="#B42" rid="B42" class=" bibr popnode">Wang et al., 2020</a>). MYC is an important component of the JAZ, COI1 and MYC2 complexes, which can mediate the generation of the jasmonic acid signaling pathway (<a href="#B22" rid="B22" class=" bibr popnode">Kazan and Manners, 2011</a>). However, in recent years, studies on the gene characterization of the MYC transcription factor family in walnut and its response to adverse stress have not been reported. In this study, we identified each member of the MYC transcription factor family in walnut, predicted the physicochemical properties, phylogeny, conserved motifs, chromosomal localization, <em>cis</em>-acting elements, and protein interactions of the members, screened nine genes of the MYC transcription factor family in combination with the transcriptome data of the wild walnut in Xinjiang in response to low-temperature stress, and characterized the subcellular localization and expression patterns of these nine genes in different tissues and in response to low-temperature stress. The subcellular localization of these nine genes and their expression patterns in different tissues and in response to low-temperature stress were systematically analyzed to provide a theoretical foundation for further investigations of the functional mechanism of the MYC transcription factor family in walnut.</p><p>In this study, 30 MYC transcription factor family members were identified from the published walnut genome-wide database, and based on phylogenetic developmental analysis, the walnut MYC transcription factor family was categorized into six subgroups, with homology closer to 10 in poplar and 8 in Arabidopsis. In addition, phylogenetic development revealed that JrMYC22 and JrMYC23 are more closely related to PtrMYC2b. It has been shown that <em>PtrMYC2b</em> in poplar (<a href="#B20" rid="B20" class=" bibr popnode">Hu M. X. et al., 2023</a>) is induced by jasmonic acid to respond to low temperatures to regulate the expression of downstream genes, thus improving plant resistance, and it is hypothesized that <em>JrMYC22</em> and <em>JrMYC23</em> may be induced by the jasmonic acid signaling pathway to activate downstream <em>cis</em>-acting element-specific binding to regulate the expression of functional genes in response to low-temperature stress (<a href="/pmc/articles/PMC11128688/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). The 30 walnut MYC transcription factor family members in this study were located on 12 chromosomes, and the genes were unevenly distributed on the chromosomes, with significant differences in chromosome length and tandem duplications (<a href="/pmc/articles/PMC11128688/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). Gene duplication is the main pathway for the amplification of gene family members, and tandem duplication is one of the important modes of gene duplication (<a href="#B38" rid="B38" class=" bibr popnode">Vision et al., 2000</a>). Analysis revealed that there were nine genes in the walnut MYC transcription factor family with tandem duplication, namely, <em>JrMYC5, JrMYC7, JrMYC8, JrMYC9, JrMYC11, JrMYC14, JrMYC18, JrMYC24</em>, and <em>JrMYC25</em>, which accounted for 30% of the total number of genes in the gene family, and combined with the conserved structural domains, these genes had a high degree of similarity (<a href="/pmc/articles/PMC11128688/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4B</span></a>). Thus, it was speculated that the walnut MYC transcription factor family may not have undergone large-scale gene expansion or relative conservation during evolution. Conserved motif analysis of each member of the walnut MYC transcription factor family revealed (<a href="/pmc/articles/PMC11128688/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4</span></a>) that the conserved structural motifs of members located in the same subfamily were consistent, suggesting that the protein function of each member is stable. Most members of the walnut MYC transcription factor family have Motif7 and Motif8 and are highly conserved. It is hypothesized that most walnut MYC transcription factor family members may perform similar functions. The JrMYC12, JrMYC13, JrMYC15, JrMYC16, JrMYC17, and JrMYC18 proteins lacked Motif7 and Motif8, and it is hypothesized that these members may have different functions.</p><p>Promoter <em>cis</em>-acting elements are key for inducing functional gene expression. When plants are subjected to adverse stress, promoter <em>cis</em>-acting elements are involved in activating related transcription factors to regulate downstream gene expression in response to adverse environmental hazards (<a href="#B30" rid="B30" class=" bibr popnode">Nakashima et al., 2009</a>; <a href="#B45" rid="B45" class=" bibr popnode">Wittkopp and Kalay, 2011</a>). To clarify the functional characteristics of each member of the walnut MYC transcription factor family, we analyzed 30 members for promoter <em>cis</em>-acting element prediction and found that the promoter regions of the walnut MYC transcription factor family members contained <em>cis</em>-acting elements related to hormones (growth hormone, salicylic acid, gibberellin, jasmonic acid, abscisic acid), light signaling and low-temperature stress, which suggests that the walnut MYC family may be involved in the regulation of plant growth, development, and stress. Plant growth and development, adversity stress, biosynthesis and other biological processes. In addition, 18 genes encoded <em>cis</em>-acting elements involved in the response to low-temperature stress, among which jasmonic acid-related genes were the most widely distributed (12), and it was hypothesized that these genes might be induced by jasmonic acid in response to low-temperature stress, which is in line with the findings of studies on tomato (<a href="#B29" rid="B29" class=" bibr popnode">Miura et al., 2012</a>), pomegranate (<a href="#B35" rid="B35" class=" bibr popnode">Sayyari et al., 2011</a>), mango (<a href="#B15" rid="B15" class=" bibr popnode">Gonz&#x000e1;lez-Aguilar et al., 2000</a>), and poplar (<a href="#B19" rid="B19" class=" bibr popnode">Hu et al., 2023</a>). The predicted protein interactions of walnut MYC transcription factor family members revealed that walnut JrMYC25 and JrMYC29 had the strongest interactions and shared high homology with MYC2 and MYC3 in <em>Arabidopsis thaliana</em>, and MYC2 and MYC3 worked together to activate the relevant transcription factors of the jasmonic acid signaling pathway in response to adversity (<a href="#B7" rid="B7" class=" bibr popnode">Cheng et al., 2011</a>). Similarly, the homologous protein of MYC2 in <em>Arabidopsis thaliana</em>, ICE1, could respond to low-temperature stress by inducing jasmonic acid in the transcriptional pathway. ICE1, a homologous protein of MYC2 in <em>Arabidopsis thaliana</em>, can induce the expression of downstream genes in response to low-temperature stress to enhance the cold resistance of plants (<a href="#B26" rid="B26" class=" bibr popnode">Lorenzo et al., 2004</a>). In addition, it was stated that ICE1 may modulate cold-regulated (COR) genes in a CBF-dependent/independent way in plant species in response to low-temperature and cold stress (<a href="#B17" rid="B17" class=" bibr popnode">Heidari, 2019</a>; <a href="#B18" rid="B18" class=" bibr popnode">Heidari et al., 2021</a>). The homology between proteins can influence functional similarity, and it is hypothesized that JrMYC25 and JrMYC29 may have functions similar to those of MYC2 and MYC3 in <em>Arabidopsis thaliana</em> and may improve resistance in response to low-temperature stress, providing theoretical support for further studies on the mechanism underlying the response of walnut MYC transcription factor family members to cold stress. This study provides theoretical support for further exploration of the mechanism of MYC transcription factor family members in walnut plants in response to low-temperature stress.</p><p class="p p-last">In this study, nine MYC family members in response to low-temperature stress were screened based on the transcriptomic data of wild walnut plants in Xinjiang. By subcellular localization, it was found that the genes all undergo fluorescence signaling in the nucleus, among which <em>JrMYC4, JrMYC32, JrMYC33, JrMYC34, JrMYC35, JrMYC36, <em>and</em> JrMYC37</em> showed strong fluorescence responses, and combined with the expression patterns of these nine genes in response to low-temperature stress, <em>JrMYC4, JrMYC32, <em>and</em> JrMYC35</em> were expressed at relatively high levels at 8&#x000a0;h, which was consistent with the transcriptome data (<a href="/pmc/articles/PMC11128688/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span>Figure 8</span></a>; <a href="/pmc/articles/PMC11128688/figure/F9/" target="figure" class="fig-table-link figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span>Figure 9</span></a>). It is hypothesized that <em>JrMYC28, JrMYC31, JrMYC33, JrMYC34, <em>and</em> JrMYC35</em> may play key roles in the response to low-temperature stress. The expression patterns of the nine genes in different tissues revealed that all of these genes were highly expressed in leaves, among which <em>JrMYC28, JrMYC33, JrMYC35</em>, and <em>JrMYC36</em> were highly expressed in wild walnuts in Xinjiang, and it was hypothesized that <em>JrMYC28, JrMYC33, JrMYC35</em>, and <em>JrMYC36</em> might play a key role in functional regulation (<a href="/pmc/articles/PMC11128688/figure/F10/" target="figure" class="fig-table-link figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Figure 10</span></a>; <a href="/pmc/articles/PMC11128688/figure/F11/" target="figure" class="fig-table-link figpopup" rid-figpopup="F11" rid-ob="ob-F11" co-legend-rid="lgnd_F11"><span>Figure 11</span></a>). The results of this study provide theoretical support for further analysis of the functional mechanism of the MYC transcription factor family in walnut.</p></div><div id="s5" class="tsec sec"><h2 class="head no_bottom_margin" id="s5title">5 Conclusion</h2><p class="p p-first-last">In summary, 30 MYC transcription factor family members were identified in walnuts in this study, and their physicochemical properties, phylogenetic relationships, chromosomal localization, sequence structures, promoter <em>cis</em>-acting elements and protein interactions were analyzed. The 30 identified MYC transcription factor family members were screened in combination with the transcriptome data of wild walnuts in Xinjiang in response to low-temperature stress, and the subcellular localization and expression patterns of these members were analyzed under low-temperature stress and in different tissues. The results showed that these members have the bHLH and bHLH_MYC_N structural domains of the MYC transcription factor family and are closely related to those of <em>Arabidopsis thaliana</em> and poplar. Among them, <em>JrMYC22</em> and <em>JrMYC23</em> have high homology with <em>PtrMYC2b</em>, which is induced by jasmonic acid in response to low-temperature stress to improve the cold tolerance of plants, and it is speculated that <em>JrMYC22</em> and <em>JrMYC23</em> may play key roles in low-temperature stress. Chromosomal localization revealed that walnut MYC members were unevenly distributed across 12 chromosomes, with possible tandem duplications of <em>JrMYC9, JrMYC11, JrMYC5, JrMYC14, JrMYC18, JrMYC24, JrMYC25, JrMYC7</em> and <em>JrMYC8</em>. Based on the sequence structure of walnut, it was found that most of the members may have functional redundancy and that a few members may have functionally different response mechanisms. The promoter <em>cis</em>-acting elements revealed that most of the walnut MYC genes were correlated with hormone <em>cis</em>-acting elements, among which the <em>cis</em>-acting elements related to jasmonic acid and low temperature were the most widely distributed, with 12 of them. Protein interactions predicted for walnut MYC members revealed that JrMYC25 and JrMYC29 had the greatest strength of interactions and high homology with MYC2 and MYC3 in <em>Arabidopsis thaliana</em>, and it was hypothesized that JrMYC25 and JrMYC29 might be able to regulate the jasmonic acid signaling pathway in response to low-temperature stress. In addition, nine MYC members were screened for subcellular localization using transcriptome data from Xinjiang wild walnut plants, and the results revealed that these members emitted strong fluorescent signals in the nucleus. According to the analysis of these members in response to low-temperature stress, the expression of <em>JrMYC28, JrMYC31, JrMYC33, JrMYC34,</em> and <em>JrMYC35</em> increased, suggesting that these three members might play important roles in the response to low-temperature stress. From the expression levels of the nine MYC members in different tissues, it was found that all of these members were highly expressed in leaves, among which <em>JrMYC28, JrMYC33</em>, <em>JrMYC35</em> and <em>JrMYC36</em> were significantly upregulated in wild walnut in Xinjiang, and it was speculated that these three members might play important roles in this process. The results of this study provide a theoretical basis for further research on the functional mechanisms of the MYC transcription factor family members in walnut.</p></div><div id="funding-group-a.p.b.q" class="tsec sec"><h2 class="head no_bottom_margin" id="funding-group-a.p.b.qtitle">Funding Statement</h2><p>The author(s) declare that financial support was received for the research, authorship, and/or publication of this article. This study was supported by the National Natural Science Foundation of China (32060665) and the Xinjiang Production and Construction Corps Southern Xinjiang Key Industry Innovation Development Support Programme Project (2022DB022).</p></div><div id="s6" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="s6title">Data availability statement</h2><!--/article/back/sec/--><p class="p p-first-last">The datasets presented in this study can be found in online repositories. The names of the repository/repositories and accession number(s) can be found in the article/<a href="#s11" rid="s11" class=" sec">Supplementary Material</a>.</p></div><div id="s7" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="s7title">Author contributions</h2><!--/article/back/sec/--><p class="p p-first-last">Y-TS: Conceptualization, Data curation, Formal Analysis, Investigation, Methodology, Software, Writing&#x02013;original draft, Writing&#x02013;review and editing. KM: Data curation, Formal Analysis, Validation, Writing&#x02013;review and editing. YZ: Data curation, Formal Analysis, Validation, Writing&#x02013;review and editing. L-QH: Conceptualization, Data curation, Methodology, Writing&#x02013;review and editing. L-QL: Conceptualization, Data curation, Formal Analysis, Investigation, Methodology, Writing&#x02013;review and editing.</p></div><div id="s9" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="s9title">Conflict of interest</h2><!--/article/back/sec/--><p class="p p-first-last">The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p></div><div id="s10" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="s10title">Publisher&#x02019;s note</h2><!--/article/back/sec/--><p class="p p-first-last">All claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.</p></div><div id="s11" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="s11title">Supplementary material</h2><!--/article/back/sec/--><p class="p p-first">The Supplementary Material for this article can be found online at: <a href="https://www.frontiersin.org/articles/10.3389/fgene.2024.1399721/full#supplementary-material" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.frontiersin.org/articles/10.3389/fgene.2024.1399721/full#supplementary-material</a>
</p><div class="sec suppmat" id="SM1"><div class="sup-box half_rhythm" id="media-a.r.e.c.a"><a href="/pmc/articles/PMC11128688/bin/Table1.XLSX" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(9.9K, XLSX)</sup></div></div></div><div id="ref-list-a.r.f" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.r.ftitle">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="B1"><span class="mixed-citation">
Abe H., Yamaguchi-Shinozaki K., Urao T., Iwasaki T., Hosokawa D., Shinozaki K. (1997). <span class="ref-title">Role of arabidopsis MYC and MYB homologs in drought- and abscisic acid-regulated gene expression</span>. <span class="ref-journal">Plant Cell</span>
<span class="ref-vol">9</span>, 1859&#x02013;1868. 10.1105/tpc.9.10.1859

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC157027/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9368419" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1105%2Ftpc.9.10.1859" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+Cell&amp;title=Role+of+arabidopsis+MYC+and+MYB+homologs+in+drought-+and+abscisic+acid-regulated+gene+expression&amp;author=H.+Abe&amp;author=K.+Yamaguchi-Shinozaki&amp;author=T.+Urao&amp;author=T.+Iwasaki&amp;author=D.+Hosokawa&amp;volume=9&amp;publication_year=1997&amp;pages=1859-1868&amp;pmid=9368419&amp;doi=10.1105/tpc.9.10.1859&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B2"><span class="mixed-citation">
Ali M. S., Baek K. H. (2020). <span class="ref-title">Jasmonic acid signaling pathway in response to abiotic stresses in plants</span>. <span class="ref-journal">Int. J. Mol. Sci.</span>
<span class="ref-vol">21</span>, 621. 10.3390/ijms21020621

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7013817/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31963549" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fijms21020621" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Mol.+Sci.&amp;title=Jasmonic+acid+signaling+pathway+in+response+to+abiotic+stresses+in+plants&amp;author=M.+S.+Ali&amp;author=K.+H.+Baek&amp;volume=21&amp;publication_year=2020&amp;pages=621&amp;pmid=31963549&amp;doi=10.3390/ijms21020621&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B3"><span class="mixed-citation">
Bai J. F., Wang Y. K., Guo L. P., Guo X. M., Guo H. Y., Yuan S. H., et al. (2019). <span class="ref-title">Genomic identification and characterization of MYC family genes in wheat (<em>Triticum aestivum</em> L.)</span>. <span class="ref-journal">BMC Genomics</span>
<span class="ref-vol">20</span>, 1032. 10.1186/s12864-019-6373-y

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6937671/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31888472" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2Fs12864-019-6373-y" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genomics&amp;title=Genomic+identification+and+characterization+of+MYC+family+genes+in+wheat+(Triticum+aestivum+L.)&amp;author=J.+F.+Bai&amp;author=Y.+K.+Wang&amp;author=L.+P.+Guo&amp;author=X.+M.+Guo&amp;author=H.+Y.+Guo&amp;volume=20&amp;publication_year=2019&amp;pages=1032&amp;pmid=31888472&amp;doi=10.1186/s12864-019-6373-y&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B4"><span class="mixed-citation">
Boter M., Ru&#x000ed;z-Rivero O., Abdeen A., Prat S. (2004). <span class="ref-title">Conserved MYC transcription factors play a key role in jasmonate signaling both in tomato and Arabidopsis</span>. <span class="ref-journal">Genes Dev.</span>
<span class="ref-vol">18</span>, 1577&#x02013;1591. 10.1101/gad.297704

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC443520/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15231736" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1101%2Fgad.297704" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev.&amp;title=Conserved+MYC+transcription+factors+play+a+key+role+in+jasmonate+signaling+both+in+tomato+and+Arabidopsis&amp;author=M.+Boter&amp;author=O.+Ru&#x000ed;z-Rivero&amp;author=A.+Abdeen&amp;author=S.+Prat&amp;volume=18&amp;publication_year=2004&amp;pages=1577-1591&amp;pmid=15231736&amp;doi=10.1101/gad.297704&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B5"><span class="mixed-citation">
Chen C., Chen H., Zhang Y., Thomas H. R., Frank M. H., He Y., et al. (2020). <span class="ref-title">TBtools: an integrative toolkit developed for interactive analyses of big biological data</span>. <span class="ref-journal">Mol. Plant</span>
<span class="ref-vol">13</span>, 1194&#x02013;1202. 10.1016/j.molp.2020.06.009

 [<a href="https://pubmed.ncbi.nlm.nih.gov/32585190" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.molp.2020.06.009" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Plant&amp;title=TBtools:+an+integrative+toolkit+developed+for+interactive+analyses+of+big+biological+data&amp;author=C.+Chen&amp;author=H.+Chen&amp;author=Y.+Zhang&amp;author=H.+R.+Thomas&amp;author=M.+H.+Frank&amp;volume=13&amp;publication_year=2020&amp;pages=1194-1202&amp;pmid=32585190&amp;doi=10.1016/j.molp.2020.06.009&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B6"><span class="mixed-citation">
Chen S., Kong Y., Zhang X., Liao Z., He Y., Li L., et al. (2021). <span class="ref-title">Structural and functional organization of the MYC transcriptional factors in <em>Camellia sinensis</em>
</span>. <span class="ref-journal">Planta</span>
<span class="ref-vol">253</span>, 93&#x02013;14. 10.1007/s00425-021-03607-2

 [<a href="https://pubmed.ncbi.nlm.nih.gov/33826012" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs00425-021-03607-2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Planta&amp;title=Structural+and+functional+organization+of+the+MYC+transcriptional+factors+in+Camellia+sinensis&#10;&amp;author=S.+Chen&amp;author=Y.+Kong&amp;author=X.+Zhang&amp;author=Z.+Liao&amp;author=Y.+He&amp;volume=253&amp;publication_year=2021&amp;pages=93-14&amp;pmid=33826012&amp;doi=10.1007/s00425-021-03607-2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B7"><span class="mixed-citation">
Cheng Z., Sun L., Qi T., Zhang B., Peng W., Liu Y., et al. (2011). <span class="ref-title">The bHLH transcription factor MYC3 interacts with the Jasmonate ZIM-domain proteins to mediate jasmonate response in <em>Arabidopsis</em>
</span>. <span class="ref-journal">Mol. Plant</span>
<span class="ref-vol">4</span>, 279&#x02013;288. 10.1093/mp/ssq073

 [<a href="https://pubmed.ncbi.nlm.nih.gov/21242320" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fmp%2Fssq073" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Plant&amp;title=The+bHLH+transcription+factor+MYC3+interacts+with+the+Jasmonate+ZIM-domain+proteins+to+mediate+jasmonate+response+in+Arabidopsis&#10;&amp;author=Z.+Cheng&amp;author=L.+Sun&amp;author=T.+Qi&amp;author=B.+Zhang&amp;author=W.+Peng&amp;volume=4&amp;publication_year=2011&amp;pages=279-288&amp;pmid=21242320&amp;doi=10.1093/mp/ssq073&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B8"><span class="mixed-citation">
Creelman R. A., Mullet J. E. (1995). <span class="ref-title">Jasmonic acid distribution and action in plants: regulation during development and response to biotic and abiotic stress</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U. S. A.</span>
<span class="ref-vol">92</span>, 4114&#x02013;4119. 10.1073/pnas.92.10.4114

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC41895/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11607536" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.92.10.4114" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.+S.+A.&amp;title=Jasmonic+acid+distribution+and+action+in+plants:+regulation+during+development+and+response+to+biotic+and+abiotic+stress&amp;author=R.+A.+Creelman&amp;author=J.+E.+Mullet&amp;volume=92&amp;publication_year=1995&amp;pages=4114-4119&amp;pmid=11607536&amp;doi=10.1073/pnas.92.10.4114&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B9"><span class="mixed-citation">
Ding F., Wang C., Wang M., Xu N. (2022). <span class="ref-title">The Ethylene Response Factor SlERF. B8 triggers jasmonate biosynthesis to promote cold tolerance in tomato</span>. <span class="ref-journal">Environ. Exp. Bot.</span>
<span class="ref-vol">203</span>, 105073. 10.1016/j.envexpbot.2022.105073
 [<a href="//doi.org/10.1016%2Fj.envexpbot.2022.105073" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Environ.+Exp.+Bot.&amp;title=The+Ethylene+Response+Factor+SlERF.+B8+triggers+jasmonate+biosynthesis+to+promote+cold+tolerance+in+tomato&amp;author=F.+Ding&amp;author=C.+Wang&amp;author=M.+Wang&amp;author=N.+Xu&amp;volume=203&amp;publication_year=2022&amp;pages=105073&amp;doi=10.1016/j.envexpbot.2022.105073&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B10"><span class="mixed-citation">
F&#x000e0;bregas N., Fernie A. R. (2021). <span class="ref-title">The interface of central metabolism with hormone signaling in plants</span>. <span class="ref-journal">Curr. Biol.</span>
<span class="ref-vol">31</span>, R1535&#x02013;R1548. 10.1016/j.cub.2021.09.070

 [<a href="https://pubmed.ncbi.nlm.nih.gov/34875246" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cub.2021.09.070" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Biol.&amp;title=The+interface+of+central+metabolism+with+hormone+signaling+in+plants&amp;author=N.+F&#x000e0;bregas&amp;author=A.+R.+Fernie&amp;volume=31&amp;publication_year=2021&amp;pages=R1535-R1548&amp;pmid=34875246&amp;doi=10.1016/j.cub.2021.09.070&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B11"><span class="mixed-citation">
Feng Y., Zeng S., Yan J., Li K., Xu H. (2023). <span class="ref-title">Genome-wide analysis and expression of MYC family genes in tomato and the functional identification of <em>slmyc1</em> in response to salt and drought stress</span>. <span class="ref-journal">Agron. J.</span>
<span class="ref-vol">13</span>, 757. 10.3390/agronomy13030757
 [<a href="//doi.org/10.3390%2Fagronomy13030757" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Agron.+J.&amp;title=Genome-wide+analysis+and+expression+of+MYC+family+genes+in+tomato+and+the+functional+identification+of+slmyc1+in+response+to+salt+and+drought+stress&amp;author=Y.+Feng&amp;author=S.+Zeng&amp;author=J.+Yan&amp;author=K.+Li&amp;author=H.+Xu&amp;volume=13&amp;publication_year=2023&amp;pages=757&amp;doi=10.3390/agronomy13030757&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B12"><span class="mixed-citation">
Fern&#x000e1;ndez-Calvo P., Chini A., Fern&#x000e1;ndez-Barbero G., Chico J. M., Gimenez-Ibanez S., Geerinck J., et al. (2011). <span class="ref-title">The Arabidopsis bHLH transcription factors MYC3 and MYC4 are targets of JAZ repressors and act additively with MYC2 in the activation of jasmonate responses</span>. <span class="ref-journal">Plant Cell</span>
<span class="ref-vol">23</span>, 701&#x02013;715. 10.1105/tpc.110.080788

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3077776/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21335373" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1105%2Ftpc.110.080788" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+Cell&amp;title=The+Arabidopsis+bHLH+transcription+factors+MYC3+and+MYC4+are+targets+of+JAZ+repressors+and+act+additively+with+MYC2+in+the+activation+of+jasmonate+responses&amp;author=P.+Fern&#x000e1;ndez-Calvo&amp;author=A.+Chini&amp;author=G.+Fern&#x000e1;ndez-Barbero&amp;author=J.+M.+Chico&amp;author=S.+Gimenez-Ibanez&amp;volume=23&amp;publication_year=2011&amp;pages=701-715&amp;pmid=21335373&amp;doi=10.1105/tpc.110.080788&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B13"><span class="mixed-citation">
Fu J., Liu L., Liu Q., Shen Q., Wang C., Yang P., et al. (2020). <span class="ref-title">ZmMYC2 exhibits diverse functions and enhances JA signaling in transgenic Arabidopsis</span>. <span class="ref-journal">Plant Cell Rep.</span>
<span class="ref-vol">39</span>, 273&#x02013;288. 10.1007/s00299-019-02490-2

 [<a href="https://pubmed.ncbi.nlm.nih.gov/31741037" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs00299-019-02490-2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+Cell+Rep.&amp;title=ZmMYC2+exhibits+diverse+functions+and+enhances+JA+signaling+in+transgenic+Arabidopsis&amp;author=J.+Fu&amp;author=L.+Liu&amp;author=Q.+Liu&amp;author=Q.+Shen&amp;author=C.+Wang&amp;volume=39&amp;publication_year=2020&amp;pages=273-288&amp;pmid=31741037&amp;doi=10.1007/s00299-019-02490-2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B14"><span class="mixed-citation">
Gao C., Qi S., Liu K., Li D., Jin C., Li Z., et al. (2016). <span class="ref-title">MYC2, MYC3, and MYC4 function redundantly in seed storage protein accumulation in Arabidopsis</span>. <span class="ref-journal">Plant Physiol. biochem.</span>
<span class="ref-vol">108</span>, 63&#x02013;70. 10.1016/j.plaphy.2016.07.004

 [<a href="https://pubmed.ncbi.nlm.nih.gov/27415132" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.plaphy.2016.07.004" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+Physiol.+biochem.&amp;title=MYC2,+MYC3,+and+MYC4+function+redundantly+in+seed+storage+protein+accumulation+in+Arabidopsis&amp;author=C.+Gao&amp;author=S.+Qi&amp;author=K.+Liu&amp;author=D.+Li&amp;author=C.+Jin&amp;volume=108&amp;publication_year=2016&amp;pages=63-70&amp;pmid=27415132&amp;doi=10.1016/j.plaphy.2016.07.004&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B15"><span class="mixed-citation">
Gonz&#x000e1;lez-Aguilar G. A., Fortiz J., Cruz R., Baez R., Wang C. Y. (2000). <span class="ref-title">Methyl jasmonate reduces chilling injury and maintains postharvest quality of mango fruit</span>. <span class="ref-journal">J. Agric. Food Chem.</span>
<span class="ref-vol">48</span>, 515&#x02013;519. 10.1021/jf9902806

 [<a href="https://pubmed.ncbi.nlm.nih.gov/10691668" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fjf9902806" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Agric.+Food+Chem.&amp;title=Methyl+jasmonate+reduces+chilling+injury+and+maintains+postharvest+quality+of+mango+fruit&amp;author=G.+A.+Gonz&#x000e1;lez-Aguilar&amp;author=J.+Fortiz&amp;author=R.+Cruz&amp;author=R.+Baez&amp;author=C.+Y.+Wang&amp;volume=48&amp;publication_year=2000&amp;pages=515-519&amp;pmid=10691668&amp;doi=10.1021/jf9902806&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B16"><span class="mixed-citation">
Han L., Ma K., Zhao Y., Mei C., Mamat A., Wang J., et al. (2022). <span class="ref-title">The cold-stress responsive gene <em>DREB1A</em> involved in low-temperature tolerance in Xinjiang wild walnut</span>. <span class="ref-journal">Peer J.</span>
<span class="ref-vol">10</span>, e14021. 10.7717/peerj.14021

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9464435/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36101878" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.7717%2Fpeerj.14021" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Peer+J.&amp;title=The+cold-stress+responsive+gene+DREB1A+involved+in+low-temperature+tolerance+in+Xinjiang+wild+walnut&amp;author=L.+Han&amp;author=K.+Ma&amp;author=Y.+Zhao&amp;author=C.+Mei&amp;author=A.+Mamat&amp;volume=10&amp;publication_year=2022&amp;pages=e14021&amp;pmid=36101878&amp;doi=10.7717/peerj.14021&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B17"><span class="mixed-citation">
Heidari P. (2019). <span class="ref-title">Comparative analysis of C-repeat binding factors (CBFs) in tomato and arabidopsis</span>. <span class="ref-journal">Braz. Arch. Biol. Technol.</span>
<span class="ref-vol">62</span>, e19180715. 10.1590/1678-4324-2019180715
 [<a href="//doi.org/10.1590%2F1678-4324-2019180715" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Braz.+Arch.+Biol.+Technol.&amp;title=Comparative+analysis+of+C-repeat+binding+factors+(CBFs)+in+tomato+and+arabidopsis&amp;author=P.+Heidari&amp;volume=62&amp;publication_year=2019&amp;pages=e19180715&amp;doi=10.1590/1678-4324-2019180715&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B18"><span class="mixed-citation">
Heidari P., Reza Amerian M., Barcaccia G. (2021). <span class="ref-title">Hormone profiles and antioxidant activity of cultivated and wild tomato seedlings under low-temperature stress</span>. <span class="ref-journal">Agronomy</span>
<span class="ref-vol">11</span>, 1146. 10.3390/agronomy11061146
 [<a href="//doi.org/10.3390%2Fagronomy11061146" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Agronomy&amp;title=Hormone+profiles+and+antioxidant+activity+of+cultivated+and+wild+tomato+seedlings+under+low-temperature+stress&amp;author=P.+Heidari&amp;author=M.+Reza+Amerian&amp;author=G.+Barcaccia&amp;volume=11&amp;publication_year=2021&amp;pages=1146&amp;doi=10.3390/agronomy11061146&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B19"><span class="mixed-citation">
Hu L., Zhang X., Ni H., Yuan F., Zhang S. (2023a). <span class="ref-title">Identification and functional analysis of CAD gene family in pomegranate (<em>Punica granatum</em>)</span>. <span class="ref-journal">Genes</span>
<span class="ref-vol">14</span>, 26. 10.3390/genes14010026
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9858471/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36672766" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fgenes14010026" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes&amp;title=Identification+and+functional+analysis+of+CAD+gene+family+in+pomegranate+(Punica+granatum)&amp;author=L.+Hu&amp;author=X.+Zhang&amp;author=H.+Ni&amp;author=F.+Yuan&amp;author=S.+Zhang&amp;volume=14&amp;publication_year=2023a&amp;pages=26&amp;doi=10.3390/genes14010026&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B20"><span class="mixed-citation">
Hu M. X., Song X. Q., Liu Y. l., Zhao S. T. (2023b). <span class="ref-title">Analysis of expression pattern of MYC family members in populus</span>. <span class="ref-journal">For. Res.</span>
<span class="ref-vol">36</span>, 1&#x02013;9. 10.12403/j.1001-1498.20220503
 [<a href="//doi.org/10.12403%2Fj.1001-1498.20220503" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=For.+Res.&amp;title=Analysis+of+expression+pattern+of+MYC+family+members+in+populus&amp;author=M.+X.+Hu&amp;author=X.+Q.+Song&amp;author=Y.+l.+Liu&amp;author=S.+T.+Zhao&amp;volume=36&amp;publication_year=2023b&amp;pages=1-9&amp;doi=10.12403/j.1001-1498.20220503&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B21"><span class="mixed-citation">
Kapli P., Yang Z., Telford M. J. (2020). <span class="ref-title">Phylogenetic tree building in the genomic age</span>. <span class="ref-journal">Nat. Rev. Genet.</span>
<span class="ref-vol">21</span>, 428&#x02013;444. 10.1038/s41576-020-0233-0

 [<a href="https://pubmed.ncbi.nlm.nih.gov/32424311" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41576-020-0233-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet.&amp;title=Phylogenetic+tree+building+in+the+genomic+age&amp;author=P.+Kapli&amp;author=Z.+Yang&amp;author=M.+J.+Telford&amp;volume=21&amp;publication_year=2020&amp;pages=428-444&amp;pmid=32424311&amp;doi=10.1038/s41576-020-0233-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B22"><span class="mixed-citation">
Kazan K., Manners J. M. (2011). <span class="ref-title">The interplay between light and jasmonate signalling during defence and development</span>. <span class="ref-journal">J. Exp. Bot.</span>
<span class="ref-vol">62</span>, 4087&#x02013;4100. 10.1093/jxb/err142

 [<a href="https://pubmed.ncbi.nlm.nih.gov/21705384" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fjxb%2Ferr142" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Exp.+Bot.&amp;title=The+interplay+between+light+and+jasmonate+signalling+during+defence+and+development&amp;author=K.+Kazan&amp;author=J.+M.+Manners&amp;volume=62&amp;publication_year=2011&amp;pages=4087-4100&amp;pmid=21705384&amp;doi=10.1093/jxb/err142&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B23"><span class="mixed-citation">
Kazan K., Manners J. M. (2013). <span class="ref-title">MYC2: the master in action</span>. <span class="ref-journal">Mol. Plant</span>
<span class="ref-vol">6</span>, 686&#x02013;703. 10.1093/mp/sss128

 [<a href="https://pubmed.ncbi.nlm.nih.gov/23142764" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fmp%2Fsss128" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Plant&amp;title=MYC2:+the+master+in+action&amp;author=K.+Kazan&amp;author=J.+M.+Manners&amp;volume=6&amp;publication_year=2013&amp;pages=686-703&amp;pmid=23142764&amp;doi=10.1093/mp/sss128&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B24"><span class="mixed-citation">
Le&#x000f3;n J., S&#x000e1;nchez-Serrano J. J. (1999). <span class="ref-title">Molecular biology of jasmonic acid biosynthesis in plants</span>. <span class="ref-journal">Plant Physiol. bioch.</span>
<span class="ref-vol">37</span>, 373&#x02013;380. 10.1016/S0981-9428(99)80043-6
 [<a href="//doi.org/10.1016%2FS0981-9428(99)80043-6" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+Physiol.+bioch.&amp;title=Molecular+biology+of+jasmonic+acid+biosynthesis+in+plants&amp;author=J.+Le&#x000f3;n&amp;author=J.+J.+S&#x000e1;nchez-Serrano&amp;volume=37&amp;publication_year=1999&amp;pages=373-380&amp;doi=10.1016/S0981-9428(99)80043-6&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B25"><span class="mixed-citation">
Li X., Wang X., Cui Z., Shi W., Huang J., Wang J. (2023). <span class="ref-title">Development of polymorphic microsatellite markers and identification of applications for wild walnut (<em>Juglans regia</em> L.) in middle asia</span>. <span class="ref-journal">Diversity</span>
<span class="ref-vol">15</span>, 1073. 10.3390/d15101073
 [<a href="//doi.org/10.3390%2Fd15101073" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diversity&amp;title=Development+of+polymorphic+microsatellite+markers+and+identification+of+applications+for+wild+walnut+(Juglans+regia+L.)+in+middle+asia&amp;author=X.+Li&amp;author=X.+Wang&amp;author=Z.+Cui&amp;author=W.+Shi&amp;author=J.+Huang&amp;volume=15&amp;publication_year=2023&amp;pages=1073&amp;doi=10.3390/d15101073&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B26"><span class="mixed-citation">
Lorenzo O., Chico J. M., S&#x000e1;nchez-Serrano J. J., Solano R. (2004). <span class="ref-title">JASMONATE-INSENSITIVE1 encodes a MYC transcription factor essential to discriminate between different jasmonate-regulated defense responses in Arabidopsis</span>. <span class="ref-journal">Plant Cell</span>
<span class="ref-vol">16</span>, 1938&#x02013;1950. 10.1105/tpc.022319

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC514172/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15208388" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1105%2Ftpc.022319" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+Cell&amp;title=JASMONATE-INSENSITIVE1+encodes+a+MYC+transcription+factor+essential+to+discriminate+between+different+jasmonate-regulated+defense+responses+in+Arabidopsis&amp;author=O.+Lorenzo&amp;author=J.+M.+Chico&amp;author=J.+J.+S&#x000e1;nchez-Serrano&amp;author=R.+Solano&amp;volume=16&amp;publication_year=2004&amp;pages=1938-1950&amp;pmid=15208388&amp;doi=10.1105/tpc.022319&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B27"><span class="mixed-citation">
Lyons R., Manners J. M., Kazan K. (2013). <span class="ref-title">Jasmonate biosynthesis and signaling in monocots: a comparative overview</span>. <span class="ref-journal">Plant Cell Rep.</span>
<span class="ref-vol">32</span>, 815&#x02013;827. 10.1007/s00299-013-1400-y

 [<a href="https://pubmed.ncbi.nlm.nih.gov/23455708" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs00299-013-1400-y" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+Cell+Rep.&amp;title=Jasmonate+biosynthesis+and+signaling+in+monocots:+a+comparative+overview&amp;author=R.+Lyons&amp;author=J.+M.+Manners&amp;author=K.+Kazan&amp;volume=32&amp;publication_year=2013&amp;pages=815-827&amp;pmid=23455708&amp;doi=10.1007/s00299-013-1400-y&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B28"><span class="mixed-citation">
McCully M. E., Canny M. J., Huang C. X. (2004). <span class="ref-title">The management of extracellular ice by petioles of frost-resistant herbaceous plants</span>. <span class="ref-journal">Ann. Bot.</span>
<span class="ref-vol">94</span>, 665&#x02013;674. 10.1093/aob/mch191

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4242212/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15355865" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Faob%2Fmch191" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Bot.&amp;title=The+management+of+extracellular+ice+by+petioles+of+frost-resistant+herbaceous+plants&amp;author=M.+E.+McCully&amp;author=M.+J.+Canny&amp;author=C.+X.+Huang&amp;volume=94&amp;publication_year=2004&amp;pages=665-674&amp;pmid=15355865&amp;doi=10.1093/aob/mch191&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B29"><span class="mixed-citation">
Miura K., Shiba H., Ohta M., Kang S. W., Sato A., Yuasa T., et al. (2012). <span class="ref-title">
<em>SlICE1</em> encoding a MYC-type transcription factor controls cold tolerance in tomato, <em>Solanum lycopersicum</em>
</span>. <span class="ref-journal">Plant Biotechnol.</span>
<span class="ref-vol">29</span>, 253&#x02013;260. 10.5511/plantbiotechnology.12.0303a
 [<a href="//doi.org/10.5511%2Fplantbiotechnology.12.0303a" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+Biotechnol.&amp;title=&#10;SlICE1+encoding+a+MYC-type+transcription+factor+controls+cold+tolerance+in+tomato,+Solanum+lycopersicum&#10;&amp;author=K.+Miura&amp;author=H.+Shiba&amp;author=M.+Ohta&amp;author=S.+W.+Kang&amp;author=A.+Sato&amp;volume=29&amp;publication_year=2012&amp;pages=253-260&amp;doi=10.5511/plantbiotechnology.12.0303a&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B30"><span class="mixed-citation">
Nakashima K., Ito Y., Yamaguchi-Shinozaki K. (2009). <span class="ref-title">Transcriptional regulatory networks in response to abiotic stresses in Arabidopsis and grasses</span>. <span class="ref-journal">Plant Physiol.</span>
<span class="ref-vol">149</span>, 88&#x02013;95. 10.1104/pp.108.129791

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2613698/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19126699" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1104%2Fpp.108.129791" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+Physiol.&amp;title=Transcriptional+regulatory+networks+in+response+to+abiotic+stresses+in+Arabidopsis+and+grasses&amp;author=K.+Nakashima&amp;author=Y.+Ito&amp;author=K.+Yamaguchi-Shinozaki&amp;volume=149&amp;publication_year=2009&amp;pages=88-95&amp;pmid=19126699&amp;doi=10.1104/pp.108.129791&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B31"><span class="mixed-citation">
Niu Y., Figueroa P., Browse J. (2011). <span class="ref-title">Characterization of JAZ-interacting bHLH transcription factors that regulate jasmonate responses in <em>Arabidopsis</em>
</span>. <span class="ref-journal">J. Exp. Bot.</span>
<span class="ref-vol">62</span>, 2143&#x02013;2154. 10.1093/jxb/erq408

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3060693/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21321051" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fjxb%2Ferq408" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Exp.+Bot.&amp;title=Characterization+of+JAZ-interacting+bHLH+transcription+factors+that+regulate+jasmonate+responses+in+Arabidopsis&#10;&amp;author=Y.+Niu&amp;author=P.+Figueroa&amp;author=J.+Browse&amp;volume=62&amp;publication_year=2011&amp;pages=2143-2154&amp;pmid=21321051&amp;doi=10.1093/jxb/erq408&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B32"><span class="mixed-citation">
Pe&#x000f1;uelas M., Monte I., Schweizer F., Vallat A., Reymond P., Garc&#x000ed;a-Casado G., et al. (2019). <span class="ref-title">Jasmonate-related MYC transcription factors are functionally conserved in <em>Marchantia polymorpha</em>
</span>. <span class="ref-journal">Plant Cell</span>
<span class="ref-vol">31</span>, 2491&#x02013;2509. 10.1105/tpc.18.00974

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6790078/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31391256" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1105%2Ftpc.18.00974" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+Cell&amp;title=Jasmonate-related+MYC+transcription+factors+are+functionally+conserved+in+Marchantia+polymorpha&#10;&amp;author=M.+Pe&#x000f1;uelas&amp;author=I.+Monte&amp;author=F.+Schweizer&amp;author=A.+Vallat&amp;author=P.+Reymond&amp;volume=31&amp;publication_year=2019&amp;pages=2491-2509&amp;pmid=31391256&amp;doi=10.1105/tpc.18.00974&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B33"><span class="mixed-citation">
Raza A., Charagh S., Zahid Z., Mubarik M. S., Javed R., Siddiqui M. H., et al. (2021). <span class="ref-title">Jasmonic acid: a key frontier in conferring abiotic stress tolerance in plants</span>. <span class="ref-journal">Plant Cell Rep.</span>
<span class="ref-vol">40</span>, 1513&#x02013;1541. 10.1007/s00299-020-02614-z

 [<a href="https://pubmed.ncbi.nlm.nih.gov/33034676" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs00299-020-02614-z" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+Cell+Rep.&amp;title=Jasmonic+acid:+a+key+frontier+in+conferring+abiotic+stress+tolerance+in+plants&amp;author=A.+Raza&amp;author=S.+Charagh&amp;author=Z.+Zahid&amp;author=M.+S.+Mubarik&amp;author=R.+Javed&amp;volume=40&amp;publication_year=2021&amp;pages=1513-1541&amp;pmid=33034676&amp;doi=10.1007/s00299-020-02614-z&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B34"><span class="mixed-citation">
Ruan J., Zhou Y., Zhou M., Yan J., Khurshid M., Weng W., et al. (2019). <span class="ref-title">Jasmonic acid signaling pathway in plants</span>. <span class="ref-journal">Int. J. Mol. Sci.</span>
<span class="ref-vol">20</span>, 2479. 10.3390/ijms20102479

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6566436/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31137463" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fijms20102479" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Mol.+Sci.&amp;title=Jasmonic+acid+signaling+pathway+in+plants&amp;author=J.+Ruan&amp;author=Y.+Zhou&amp;author=M.+Zhou&amp;author=J.+Yan&amp;author=M.+Khurshid&amp;volume=20&amp;publication_year=2019&amp;pages=2479&amp;pmid=31137463&amp;doi=10.3390/ijms20102479&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B35"><span class="mixed-citation">
Sayyari M., Babalar M., Kalantari S., Mart&#x000ed;nez-Romero D., Guill&#x000e9;n F., Serrano M., et al. (2011). <span class="ref-title">Vapour treatments with methyl salicylate or methyl jasmonate alleviated chilling injury and enhanced antioxidant potential during postharvest storage of pomegranates</span>. <span class="ref-journal">Food Chem.</span>
<span class="ref-vol">124</span>, 964&#x02013;970. 10.1016/j.foodchem.2010.07.036
 [<a href="//doi.org/10.1016%2Fj.foodchem.2010.07.036" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Food+Chem.&amp;title=Vapour+treatments+with+methyl+salicylate+or+methyl+jasmonate+alleviated+chilling+injury+and+enhanced+antioxidant+potential+during+postharvest+storage+of+pomegranates&amp;author=M.+Sayyari&amp;author=M.+Babalar&amp;author=S.+Kalantari&amp;author=D.+Mart&#x000ed;nez-Romero&amp;author=F.+Guill&#x000e9;n&amp;volume=124&amp;publication_year=2011&amp;pages=964-970&amp;doi=10.1016/j.foodchem.2010.07.036&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B36"><span class="mixed-citation">
Schweizer F., Fern&#x000e1;ndez-Calvo P., Zander M., Diez-Diaz M., Fonseca S., Glauser G., et al. (2013). <span class="ref-title">Arabidopsis basic helix-loop-helix transcription factors MYC2, MYC3, and MYC4 regulate glucosinolate biosynthesis, insect performance, and feeding behavior</span>. <span class="ref-journal">Plant Cell</span>
<span class="ref-vol">25</span>, 3117&#x02013;3132. 10.1105/tpc.113.115139

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3784603/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23943862" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1105%2Ftpc.113.115139" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+Cell&amp;title=Arabidopsis+basic+helix-loop-helix+transcription+factors+MYC2,+MYC3,+and+MYC4+regulate+glucosinolate+biosynthesis,+insect+performance,+and+feeding+behavior&amp;author=F.+Schweizer&amp;author=P.+Fern&#x000e1;ndez-Calvo&amp;author=M.+Zander&amp;author=M.+Diez-Diaz&amp;author=S.+Fonseca&amp;volume=25&amp;publication_year=2013&amp;pages=3117-3132&amp;pmid=23943862&amp;doi=10.1105/tpc.113.115139&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B37"><span class="mixed-citation">
Shinozaki K., Yamaguchi-Shinozaki K. (2007). <span class="ref-title">Gene networks involved in drought stress response and tolerance</span>. <span class="ref-journal">J. Exp. Bot.</span>
<span class="ref-vol">58</span>, 221&#x02013;227. 10.1093/jxb/erl164

 [<a href="https://pubmed.ncbi.nlm.nih.gov/17075077" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fjxb%2Ferl164" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Exp.+Bot.&amp;title=Gene+networks+involved+in+drought+stress+response+and+tolerance&amp;author=K.+Shinozaki&amp;author=K.+Yamaguchi-Shinozaki&amp;volume=58&amp;publication_year=2007&amp;pages=221-227&amp;pmid=17075077&amp;doi=10.1093/jxb/erl164&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B38"><span class="mixed-citation">
Vision T. J., Brown D. G., Tanksley S. D. (2000). <span class="ref-title">The origins of genomic duplications in <em>Arabidopsis</em>
</span>. <span class="ref-journal">Science</span>
<span class="ref-vol">290</span>, 2114&#x02013;2117. 10.1126/science.290.5499.2114

 [<a href="https://pubmed.ncbi.nlm.nih.gov/11118139" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.290.5499.2114" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+origins+of+genomic+duplications+in+Arabidopsis&#10;&amp;author=T.+J.+Vision&amp;author=D.+G.+Brown&amp;author=S.+D.+Tanksley&amp;volume=290&amp;publication_year=2000&amp;pages=2114-2117&amp;pmid=11118139&amp;doi=10.1126/science.290.5499.2114&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B39"><span class="mixed-citation">
Wang C., Hu S. Q., Li T., Liu C. L. (2021a). <span class="ref-title">Regulation of MYC2 in Bupleurum Chinese under simulated drought stress</span>. <span class="ref-journal">Xibei Zhiwu Xuebao Acta Bot. Boreali-Occidentalia Sin.</span>
<span class="ref-vol">41</span> (<span class="ref-iss">10</span>), 1747&#x02013;1754. 10.7606/j.issn.1000-4025.2021.10.1747
 [<a href="//doi.org/10.7606%2Fj.issn.1000-4025.2021.10.1747" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Xibei+Zhiwu+Xuebao+Acta+Bot.+Boreali-Occidentalia+Sin.&amp;title=Regulation+of+MYC2+in+Bupleurum+Chinese+under+simulated+drought+stress&amp;author=C.+Wang&amp;author=S.+Q.+Hu&amp;author=T.+Li&amp;author=C.+L.+Liu&amp;volume=41&amp;issue=10&amp;publication_year=2021a&amp;pages=1747-1754&amp;doi=10.7606/j.issn.1000-4025.2021.10.1747&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B40"><span class="mixed-citation">
Wang D. D., Li P., Chen Q. Y., Chen X. Y., Yan Z. W., Wang M. Y., et al. (2021b). <span class="ref-title">Differential contributions of MYCs to insect defense reveals flavonoids alleviating growth inhibition caused by wounding in arabidopsis</span>. <span class="ref-journal">Front. Plant Sci.</span>
<span class="ref-vol">12</span>, 700555. 10.3389/fpls.2021.700555

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8313826/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34326858" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3389%2Ffpls.2021.700555" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Plant+Sci.&amp;title=Differential+contributions+of+MYCs+to+insect+defense+reveals+flavonoids+alleviating+growth+inhibition+caused+by+wounding+in+arabidopsis&amp;author=D.+D.+Wang&amp;author=P.+Li&amp;author=Q.+Y.+Chen&amp;author=X.+Y.+Chen&amp;author=Z.+W.+Yan&amp;volume=12&amp;publication_year=2021b&amp;pages=700555&amp;pmid=34326858&amp;doi=10.3389/fpls.2021.700555&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B41"><span class="mixed-citation">
Wang H., Li Y., Pan J., Lou D., Hu Y., Yu D. (2017). <span class="ref-title">The bHLH transcription factors MYC2, MYC3, and MYC4 are required for jasmonate-mediated inhibition of flowering in arabidopsis</span>. <span class="ref-journal">Mol. Plant</span>
<span class="ref-vol">10</span>, 1461&#x02013;1464. 10.1016/j.molp.2017.08.007

 [<a href="https://pubmed.ncbi.nlm.nih.gov/28827172" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.molp.2017.08.007" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Plant&amp;title=The+bHLH+transcription+factors+MYC2,+MYC3,+and+MYC4+are+required+for+jasmonate-mediated+inhibition+of+flowering+in+arabidopsis&amp;author=H.+Wang&amp;author=Y.+Li&amp;author=J.+Pan&amp;author=D.+Lou&amp;author=Y.+Hu&amp;volume=10&amp;publication_year=2017&amp;pages=1461-1464&amp;pmid=28827172&amp;doi=10.1016/j.molp.2017.08.007&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B42"><span class="mixed-citation">
Wang J., Song L., Gong X., Xu J., Li M. (2020). <span class="ref-title">Functions of jasmonic acid in plant regulation and response to abiotic stress</span>. <span class="ref-journal">Int. J. Mol. Sci.</span>
<span class="ref-vol">21</span>, 1446. 10.3390/ijms21041446

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7073113/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32093336" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fijms21041446" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Mol.+Sci.&amp;title=Functions+of+jasmonic+acid+in+plant+regulation+and+response+to+abiotic+stress&amp;author=J.+Wang&amp;author=L.+Song&amp;author=X.+Gong&amp;author=J.+Xu&amp;author=M.+Li&amp;volume=21&amp;publication_year=2020&amp;pages=1446&amp;pmid=32093336&amp;doi=10.3390/ijms21041446&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B43"><span class="mixed-citation">
Wang S., Wang Y., Yang R., Cai W., Liu Y., Zhou D., et al. (2023). <span class="ref-title">Genome-wide identification and analysis uncovers the potential role of JAZ and MYC families in potato under abiotic stress</span>. <span class="ref-journal">Int. J. Mol. Sci.</span>
<span class="ref-vol">24</span>, 6706. 10.3390/ijms24076706

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10094809/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37047679" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fijms24076706" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Mol.+Sci.&amp;title=Genome-wide+identification+and+analysis+uncovers+the+potential+role+of+JAZ+and+MYC+families+in+potato+under+abiotic+stress&amp;author=S.+Wang&amp;author=Y.+Wang&amp;author=R.+Yang&amp;author=W.+Cai&amp;author=Y.+Liu&amp;volume=24&amp;publication_year=2023&amp;pages=6706&amp;pmid=37047679&amp;doi=10.3390/ijms24076706&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B44"><span class="mixed-citation">
Wang Y., Xu H., Liu W., Wang N., Qu C., Jiang S., et al. (2019). <span class="ref-title">Methyl jasmonate enhances apple&#x02019;cold tolerance through the JAZ-MYC2 pathway</span>. <span class="ref-journal">Plant Cell</span>
<span class="ref-vol">136</span>, 75&#x02013;84. 10.1007/s11240-018-1493-7
 [<a href="//doi.org/10.1007%2Fs11240-018-1493-7" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+Cell&amp;title=Methyl+jasmonate+enhances+apple&#x02019;cold+tolerance+through+the+JAZ-MYC2+pathway&amp;author=Y.+Wang&amp;author=H.+Xu&amp;author=W.+Liu&amp;author=N.+Wang&amp;author=C.+Qu&amp;volume=136&amp;publication_year=2019&amp;pages=75-84&amp;doi=10.1007/s11240-018-1493-7&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B45"><span class="mixed-citation">
Wittkopp P. J., Kalay G. (2011). <span class="ref-title">Cis-regulatory elements: molecular mechanisms and evolutionary processes underlying divergence</span>. <span class="ref-journal">Nat. Rev. Genet.</span>
<span class="ref-vol">13</span>, 59&#x02013;69. 10.1038/nrg3095

 [<a href="https://pubmed.ncbi.nlm.nih.gov/22143240" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnrg3095" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet.&amp;title=Cis-regulatory+elements:+molecular+mechanisms+and+evolutionary+processes+underlying+divergence&amp;author=P.+J.+Wittkopp&amp;author=G.+Kalay&amp;volume=13&amp;publication_year=2011&amp;pages=59-69&amp;pmid=22143240&amp;doi=10.1038/nrg3095&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B46"><span class="mixed-citation">
Yanfang Y., Kaikai Z., Liying Y., Xing L., Ying W., Hongwei L., et al. (2018). <span class="ref-title">Identification and characterization of MYC transcription factors in Taxus sp</span>. <span class="ref-journal">Gene</span>
<span class="ref-vol">675</span>, 1&#x02013;8. 10.1016/j.gene.2018.06.065

 [<a href="https://pubmed.ncbi.nlm.nih.gov/29935357" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.gene.2018.06.065" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=Identification+and+characterization+of+MYC+transcription+factors+in+Taxus+sp&amp;author=Y.+Yanfang&amp;author=Z.+Kaikai&amp;author=Y.+Liying&amp;author=L.+Xing&amp;author=W.+Ying&amp;volume=675&amp;publication_year=2018&amp;pages=1-8&amp;pmid=29935357&amp;doi=10.1016/j.gene.2018.06.065&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B47"><span class="mixed-citation">
Zhang M. H., Wang J. C., Yang H. L., Zhang D. Y., Deng P. C., Cui Z. J. (2020). <span class="ref-title">Physiological response of Xinjiang wild walnut germplasm to low temperature stress</span>. <span class="ref-journal">J. Appl. Ecol.</span>
<span class="ref-vol">31</span>, 2558&#x02013;2566. 10.13287/j.1001-9332.202008.002
 [<a href="https://pubmed.ncbi.nlm.nih.gov/34494777" ref="reftype=pubmed&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.13287%2Fj.1001-9332.202008.002" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Appl.+Ecol.&amp;title=Physiological+response+of+Xinjiang+wild+walnut+germplasm+to+low+temperature+stress&amp;author=M.+H.+Zhang&amp;author=J.+C.+Wang&amp;author=H.+L.+Yang&amp;author=D.+Y.+Zhang&amp;author=P.+C.+Deng&amp;volume=31&amp;publication_year=2020&amp;pages=2558-2566&amp;doi=10.13287/j.1001-9332.202008.002&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11128688&amp;issue-id=454533&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li></ul></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Frontiers in Genetics</span> are provided here courtesy of <strong>Frontiers Media SA</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11128688/pdf/fgene-15-1399721.pdf" class="int-view">PDF (5.2M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11128688/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11128688/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11128688" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11128688%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="duEKOrMZyLN8MAcDGy6QH1Lzjpb3dkIOyzLmZmgRgkaOo5fHULcXibq8EW32msQn">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11128688%2F&amp;text=Genome-wide%20identification%20of%20the%20walnut%20MYC%20gene%20family%20and%20functional%20characterization%20of%20Xinjiang%20wild%20walnut%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11128688%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11128688/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11128688/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38803544/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11128688/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38803544/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11128688/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11128688/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC11128688/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
