
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Nucleic acid mediated activation of a short prokaryotic Argonaute immune system - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="322C300B24F39205000043A09DD1C33D.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="ncomms" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC11156904/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11156904/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Nature Communications" /><meta name="citation_title" content="Nucleic acid mediated activation of a short prokaryotic Argonaute immune system" /><meta name="citation_author" content="Jithesh Kottur" /><meta name="citation_author_institution" content="Department of Pharmacological Sciences, Icahn School of Medicine at Mount Sinai, New York, NY 10029 USA" /><meta name="citation_author_institution" content="Department of Antiviral Drug Research, Institute of Advanced Virology, Thiruvananthapuram, Kerala 695317 India" /><meta name="citation_author" content="Radhika Malik" /><meta name="citation_author_institution" content="Department of Pharmacological Sciences, Icahn School of Medicine at Mount Sinai, New York, NY 10029 USA" /><meta name="citation_author" content="Aneel K. Aggarwal" /><meta name="citation_author_institution" content="Department of Pharmacological Sciences, Icahn School of Medicine at Mount Sinai, New York, NY 10029 USA" /><meta name="citation_publication_date" content="2024" /><meta name="citation_volume" content="15" /><meta name="citation_doi" content="10.1038/s41467-024-49271-4" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC11156904/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC11156904/" /><meta name="citation_pmid" content="38844755" /><meta name="DC.Title" content="Nucleic acid mediated activation of a short prokaryotic Argonaute immune system" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Nature Publishing Group" /><meta name="DC.Contributor" content="Jithesh Kottur" /><meta name="DC.Contributor" content="Radhika Malik" /><meta name="DC.Contributor" content="Aneel K. Aggarwal" /><meta name="DC.Date" content="2024" /><meta name="DC.Identifier" content="10.1038/s41467-024-49271-4" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Nucleic acid mediated activation of a short prokaryotic Argonaute immune system" /><meta property="og:type" content="article" /><meta property="og:description" content="A short prokaryotic Argonaute (pAgo) TIR-APAZ (SPARTA) defense system, activated by invading DNA to unleash its TIR domain for NAD(P)[+]  hydrolysis, was recently identified in bacteria. We report the crystal structure of SPARTA heterodimer in the absence ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11156904/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11156904/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22Nat%20Commun%22[journal]" class="navlink">Nat Commun</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11156904
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11156904/pdf/41467_2024_Article_49271.pdf" class="int-view">PDF (4.9M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11156904/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11156904/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11156904" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11156904%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="Ib1xsMM6h5082AnrBcUPIhyN9zTxDXtnk09cVpTxJOBcRRgtzxT7rGpvMmBEXEVu">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11156904%2F&amp;text=Nucleic%20acid%20mediated%20activation%20of%20a%20short%20prokaryotic%20Argonaute%20immune%20system" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11156904%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11156904/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11156904/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38844755/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11156904/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38844755/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11156904/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11156904/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22Nat%20Commun%22[journal]" class="navlink">Nat Commun</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11156904
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-ncomms.gif" alt="Logo of ncomms" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Nat Commun.</span> 2024; 15: 4852. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2024 Jun 6. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1038%2Fs41467-024-49271-4" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/s41467-024-49271-4</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC11156904</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38844755">38844755</a></div></div></div><h1 class="content-title">Nucleic acid mediated activation of a short prokaryotic Argonaute immune system</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kottur%20J%5BAuthor%5D" class="affpopup" co-rid="_co_idm140500376798944" co-class="co-affbox">Jithesh Kottur</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>#</sup><sup>1,</sup><sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Malik%20R%5BAuthor%5D" class="affpopup" co-rid="_co_idm140500367894560" co-class="co-affbox">Radhika Malik</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>#</sup><sup>1</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Aggarwal%20AK%5BAuthor%5D" class="affpopup" co-rid="_co_idm140500377792304" co-class="co-affbox">Aneel K. Aggarwal</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140500376798944"><h3 class="no_margin">Jithesh Kottur</h3><p><sup>1</sup>Department of Pharmacological Sciences, Icahn School of Medicine at Mount Sinai, New York, NY 10029 USA </p><p><sup>2</sup>Department of Antiviral Drug Research, Institute of Advanced Virology, Thiruvananthapuram, Kerala 695317 India </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kottur%20J%5BAuthor%5D">Jithesh Kottur</a></div></div><div id="_co_idm140500367894560"><h3 class="no_margin">Radhika Malik</h3><p><sup>1</sup>Department of Pharmacological Sciences, Icahn School of Medicine at Mount Sinai, New York, NY 10029 USA </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Malik%20R%5BAuthor%5D">Radhika Malik</a></div></div><div id="_co_idm140500377792304"><h3 class="no_margin">Aneel K. Aggarwal</h3><p><sup>1</sup>Department of Pharmacological Sciences, Icahn School of Medicine at Mount Sinai, New York, NY 10029 USA </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Aggarwal%20AK%5BAuthor%5D">Aneel K. Aggarwal</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140500376331056_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140500376331056_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140500376331056_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140500376331056_ai" style="display:none"><div class="fm-affl" id="Aff1"><sup>1</sup>Department of Pharmacological Sciences, Icahn School of Medicine at Mount Sinai, New York, NY 10029 USA </div><div class="fm-affl" id="Aff2"><sup>2</sup>Department of Antiviral Drug Research, Institute of Advanced Virology, Thiruvananthapuram, Kerala 695317 India </div><div><span class="fm-affl">Jithesh Kottur, </span><span class="fm-affl"><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ni.ser.vai@ruttokhsehtij" class="oemail">ni.ser.vai@ruttokhsehtij</a></span>.</div><div><a href="#article-aaff-info">Contributor Information</a>.</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup>Corresponding author.</div><div><sup>#</sup>Contributed equally.</div></div><div class="fm-article-notes hide half_rhythm" id="idm140500376331056_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2023 Sep 20; Accepted 2024 May 30.</div></div><div class="permissions half_rhythm hide" id="idm140500376331056_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; The Author(s) 2024</div><div class="license half_rhythm"><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article&#x02019;s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article&#x02019;s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm" role="complementary" aria-label="Related or updated information about this article."><div><strong>Previous version available:</strong> This article is based on a previously available preprint posted on <em>bioRxiv</em> on September 17, 2023: "<a href="/pmc/articles/PMC10516056/">Nucleic Acid Mediated Activation of a Short Prokaryotic Argonaute Immune System</a>".</div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="4" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary Information</div><div><a href="/pmc/articles/PMC11156904/bin/41467_2024_49271_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl">41467_2024_49271_MOESM1_ESM.pdf</a><span style="color:gray"> (68M)</span></div><div class="small guid">GUID: A1F2EB63-C377-42DC-B251-FFD1B94C944E</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Peer Review File</div><div><a href="/pmc/articles/PMC11156904/bin/41467_2024_49271_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl">41467_2024_49271_MOESM2_ESM.pdf</a><span style="color:gray"> (676K)</span></div><div class="small guid">GUID: 27A16C44-2877-44FC-ADD2-1CFE174052D2</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Reporting Summary</div><div><a href="/pmc/articles/PMC11156904/bin/41467_2024_49271_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl">41467_2024_49271_MOESM3_ESM.pdf</a><span style="color:gray"> (1.6M)</span></div><div class="small guid">GUID: 1A9CAD0B-9A2E-47C6-9E34-38605E544CDF</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Source Data</div><div><a href="/pmc/articles/PMC11156904/bin/41467_2024_49271_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl">41467_2024_49271_MOESM4_ESM.xlsx</a><span style="color:gray"> (2.5M)</span></div><div class="small guid">GUID: 5A2D2CE3-6F0E-4E44-9A53-D8617F3D6418</div></div></dd></dl><dl data-length="998" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p>The structure factors and coordinate files for the crystal structure of the Apo SPARTA have been deposited in the Protein Data Bank (PDB) under the accession code <a href="http://doi.org/10.2210/pdb8U7B/pdb" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">8U7B</a>.</p><p>The original, composite cryo-EM density map as well as the focused refinement maps of the SPARTA oligomer generated in this study have been deposited in the Electron Microscopy Data Bank (EMDB) under accession numbers <a href="https://www.ebi.ac.uk/emdb/EMD-41959" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">EMD-41959</a>, (Original map), <a href="https://www.ebi.ac.uk/emdb/EMD-41945" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">EMD-41945</a> (Focused refined map of the dimer [APAZ-MID-PIWI-RNA/DNA]), <a href="https://www.ebi.ac.uk/emdb/EMD-41947" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">EMD-41947</a> (Focused refined map of the second dimer [APAZ-MID-PIWI-RNA/DNA]), <a href="https://www.ebi.ac.uk/emdb/EMD-41948" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">EMD-41948</a> (Focused refined map of the TIR domains) and <a href="https://www.ebi.ac.uk/emdb/EMD-41966" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">EMD-41966</a> (Composite map). The resulting atomic coordinates for the SPARTA oligomer have been deposited in the Protein Data Bank (PDB) with accession number PDB ID: <a href="http://doi.org/10.2210/pdb8U72/pdb" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">8U72</a>.</p><p>Raw uncropped gel image (for Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1b</span></a>) and raw &#x003b5;-NAD<sup>+</sup> assay data (for Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5b</span></a>) are available as source data file accompanying this manuscript.&#x000a0;<a href="#Sec22" rid="Sec22" class=" sec">Source data</a> are provided in this paper.</p></dd></dl></div><div id="Abs1" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="Abs1title">Abstract</h2><!--article-meta--><div><p id="Par1" class="p p-first-last">A short prokaryotic Argonaute (pAgo) TIR-APAZ (SPARTA) defense system, activated by invading DNA to unleash its TIR domain for NAD(P)<sup>+</sup> hydrolysis, was recently identified in bacteria. We report the crystal structure of SPARTA heterodimer in the absence of guide-RNA/target-ssDNA (2.66&#x02009;&#x000c5;) and a cryo-EM structure of the SPARTA oligomer (tetramer of heterodimers) bound to guide-RNA/target-ssDNA at nominal 3.15&#x02013;3.35&#x02009;&#x000c5; resolution. The crystal structure provides a high-resolution view of SPARTA, revealing the APAZ domain as equivalent to the N, L1, and L2 regions of long pAgos and the MID domain containing a unique insertion (insert57). Cryo-EM structure reveals regions of the PIWI (loop10-9) and APAZ (helix &#x003b1;N) domains that reconfigure for nucleic-acid binding and decrypts regions/residues that reorganize to expose a positively charged pocket for higher-order assembly. The TIR domains amass in a parallel-strands arrangement for catalysis. We visualize SPARTA before and after RNA/ssDNA binding and uncover the basis of its active assembly leading to abortive infection.</p></div><div class="sec"><strong class="kwd-title">Subject terms: </strong><span class="kwd-text">Bacterial host response, X-ray crystallography, Cryoelectron microscopy, Enzyme mechanisms</span></div></div><div id="Abs2" lang="en" class="tsec sec"><h2 class="headless nomenu"></h2><!--article-meta--><div><p id="Par2" class="p p-first-last">The SPARTA defense system, activated by invading DNA for NAD(P)+ hydrolysis, was recently identified in bacteria. Here, authors visualize SPARTA before and after nucleic acid binding and uncover the basis of its active assembly leading to abortive infection.</p></div></div><div id="Sec1" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec1title">Introduction</h2><p id="Par3" class="p p-first">There has been intense interest over the past few years in identifying new bacterial defense systems, both for their own biological interest and as potential tools for biotechnological applications<sup><sup><a href="#CR1" rid="CR1" class=" bibr popnode">1</a></sup>,<sup><a href="#CR2" rid="CR2" class=" bibr popnode">2</a></sup></sup>. One new defense system is the short prokaryotic Argonaute TIR-APAZ (SPARTA), which is activated by foreign (plasmid) DNA to unleash its TIR domain for NAD(P)<sup>+</sup> hydrolysis<sup><sup><a href="#CR3" rid="CR3" class=" bibr popnode">3</a></sup></sup>. The resulting depletion of the essential metabolite NAD(P)<sup>+</sup> lends to the killing of the infected cell (abortive infection) as a means to prevent the spread of the invading DNA to the remaining bacterial cell population.</p><p id="Par4">SPARTA adds to the growing list of prokaryotic Argonaute (pAgo) proteins that comprise defense systems in bacteria, with similarities to the RNA interference (RNAi) systems in eukaryotes<sup><sup><a href="#CR4" rid="CR4" class=" bibr popnode">4</a></sup>&#x02013;<sup><a href="#CR7" rid="CR7" class=" bibr popnode">7</a></sup></sup>. The proteins use single-stranded RNA or DNA as guides to target complementary nucleic acids and can be divided into two broad classes: long pAgos and short pAgos. The long pAgos contain the same four domains and two ordered linker (L) composition (N-L1-PAZ-L2-MID-PIWI) as eukaryotic Agos, whereas the short pAgos contain only the MID (middle) and PIWI (P element-induced wimpy testis) domains<sup><sup><a href="#CR5" rid="CR5" class=" bibr popnode">5</a></sup>,<sup><a href="#CR6" rid="CR6" class=" bibr popnode">6</a></sup></sup>. Short pAgos, however, are associated with proteins containing an analogous to PAZ (APAZ) domain, the N-terminus of which is often fused to Toll/interleukin-1 receptor (TIR), Silent informator regulator 2 (SIR2), or a nuclease-type domain<sup><sup><a href="#CR8" rid="CR8" class=" bibr popnode">8</a></sup></sup>. The APAZ-containing proteins can be stand-alone and in the same operon as short pAgo or can be fused to the N-terminus of pAgo in a single polypeptide<sup><sup><a href="#CR8" rid="CR8" class=" bibr popnode">8</a></sup></sup>. Although the TIR domain was originally described as a protein-protein interaction module in innate immune signaling in animals, recent studies have revealed TIR domains that possess enzymatic activities to metabolize NAD<sup>+</sup> in neurodegenerative disease in humans as well as defense systems in plants and bacteria<sup><sup><a href="#CR9" rid="CR9" class=" bibr popnode">9</a></sup>&#x02013;<sup><a href="#CR12" rid="CR12" class=" bibr popnode">12</a></sup></sup>.</p><p id="Par5">The SPARTA system was first discovered in 2018<sup><sup><a href="#CR6" rid="CR6" class=" bibr popnode">6</a></sup></sup> and further characterized by Koopal et al<sup><sup><a href="#CR3" rid="CR3" class=" bibr popnode">3</a></sup></sup> in 2022, who showed in an elegant set of studies that the short p-Ago of SPARTA associates with a TIR-APAZ protein to form a catalytically inactive heterodimer with 1:1 stoichiometry. However, upon guide (g) RNA and target (t) ssDNA binding, four heterodimers transition to an oligomer (a tetramer of heterodimers) that unleashes the NAD(P)ase activity of the TIR domain for abortive infection<sup><sup><a href="#CR3" rid="CR3" class=" bibr popnode">3</a></sup></sup>.</p><p id="Par6" class="p p-last">Here, we present a high-resolution crystal structure of the SPARTA heterodimer in absence of gRNA/tDNA (Apo SPARTA)&#x000a0;at 2.66&#x02009;&#x000c5; resolution and a cryo-EM structure of the SPARTA oligomer bound to gRNA/tDNA at a nominal&#x000a0;3.15-3.35&#x02009;&#x000c5; resolution. Together, the crystal and cryo-EM structures provide a visualization of SPARTA before and after RNA/ssDNA binding and reveal the basis of SPARTA&#x02019;s active assembly leading to NAD(P)<sup>+</sup> degradation and cell death.</p></div><div id="Sec2" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec2title">Results and Discussion</h2><div id="Sec3" class="sec sec-first"><h3 id="Sec3title">Biophysical and structural analysis of SPARTA</h3><p id="Par7" class="p p-first">We first co-expressed and purified the heterodimeric TIR-APAZ/short pAgo complex from <em>Crenotalea thermophila</em> (SPARTA) (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1a, b</span></a>), and then used mass photometry to characterize the oligomeric state of SPARTA in the absence and presence of gRNA/tDNA. The mass distribution revealed that Apo SPARTA exists as a heterodimer with an estimated molecular weight (MW) of 109&#x02009;kDa (close to the theoretical MW of 112&#x02009;kDa) (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1c</span></a>). Upon incubation of SPARTA with the gRNA/tDNA at 55 <sup>o</sup>C for 12&#x02009;hours, we observed mass distributions corresponding to monomeric, dimeric, and tetrameric complexes of the heterodimer (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1c</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig1" co-legend-rid="lgnd_Fig1"><a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" rid-figpopup="Fig1" rid-ob="ob-Fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140500377581680"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11156904_41467_2024_49271_Fig1_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2024_49271_Fig1_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11156904/bin/41467_2024_49271_Fig1_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140500377581680"><a target="object" rel="noopener" href="/pmc/articles/PMC11156904/figure/Fig1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig1"><div><a class="figpopup" href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" rid-figpopup="Fig1" rid-ob="ob-Fig1">Fig. 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Overall structure of Apo SPARTA.</strong><p><strong>a</strong> Schematic diagram of the domain architecture of TIR-APAZ and short pAgo. <strong>b</strong> SDS-PAGE of co-purified short pAgo (58.2&#x02009;kDa) and TIR-APAZ (53.2&#x02009;kDa) proteins. Uncropped gel image is provided as a Source Data file. The purification of the SPARTA heterodimer was repeated at least 5 times with similar results. <strong>c</strong> Mass photometry data of the Apo SPARTA (top) and SPARTA incubated with guide RNA (gRNA) and target DNA (tDNA) heteroduplex (bottom). Apo SPARTA shows a clear homogeneous population for the monomer. Upon incubation with gRNA/tDNA, SPARTA additionally shows the formation of dimer and tetramer complexes. <strong>d</strong> Crystal structure of TIR-APAZ/short pAgo heterodimeric&#x000a0;complex. TIR domain, connector and APAZ domain are colored as green, magenta and orange, respectively. MID, insert57 and PIWI domains of short pAgo are colored as cyan, yellow and slate, respectively. <strong>e</strong> Structural comparison of monomeric TIR domains from TIR domain in proteins with scaffolding and enzymatic roles. The secondary structure elements are labeled for SPARTA TIR. <strong>f</strong> Structural comparison of SPARTA APAZ domain with the N-PAZ-L1-L2 regions of the <em>Clostridium butyricum</em> (Cb) long pAgo (PDB ID: 6QZK). The N domain, L1, PAZ, and L2 of CbAgo are colored as olive, orange, light orange and grey, respectively. Based on topological similarity, the APAZ-N, APAZ-L1 and APAZ-L2 of SPARTA APAZ are colored similarly to the CbAgo. <strong>g</strong> Structural comparison of SPARTA short pAgo with MID and PIWI domains of the Cb long pAgo (PDB ID: 6QZK). The domains are similar in structure with the exception of the presence of Insert57 in SPARTA short pAgo.</p></div></div></div><p id="Par8" class="p p-last">To see what the structure of SPARTA looks like in absence of gRNA/tDNA, we obtained crystals of Apo SPARTA that diffracted to 2.66&#x02009;&#x000c5; resolution and belonged to space group P6<sub>5</sub>22 with unit cell dimensions a = b&#x02009;=&#x02009;197.18&#x02009;&#x000c5;, c&#x02009;=&#x02009;183.4&#x02009;&#x000c5;, &#x003b1;&#x02009;=&#x02009;&#x003b2;&#x02009;=&#x02009;90&#x000b0;, &#x003b3;&#x02009;=&#x02009;120&#x000b0; with one SPARTA heterodimer in the crystallographic asymmetric unit. We solved the structure by molecular replacement using an AlphaFold<sup><sup><a href="#CR13" rid="CR13" class=" bibr popnode">13</a></sup></sup> model of the heterodimer, followed by manual building (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1d</span></a>, Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">1</a></sup> and Supplementary Table&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">1</a></sup>). To see how SPARTA transitions to a catalytically active oligomer, we incubated the heterodimer with gRNA/tDNA and started with single-particle EM studies of negatively stained samples. The images yielded 2D class averages showing a clear butterfly-like architecture (SPARTA oligomer) as well as smaller particles (Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">2</a></sup>). The butterfly-like particles (SPARTA oligomer) from the negative stained sample were used as a template for further cryo-EM studies. The initial untilted cryo-EM dataset resulted in an anisotropic map, however, upon tilting (to 30&#x000b0; and 45&#x000b0;) the sphericity of the map improved substantially and resulted in an isotropic reconstruction at a nominal resolution of 3.35&#x02009;&#x000c5; (Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">3</a></sup>). The reconstruction revealed four SPARTA heterodimers (A, B, C and D) assembled into a tetramer of heterodimers, with the MID, PIWI, APAZ domains and gRNA/tDNA on the outside (wings of the butterfly) and the TIR domains on the inside (body of the butterfly). The resolution of Ago-APAZ-gRNA/tDNA section was further improved by performing focused C2 symmetry-based refinements of each of the &#x0201c;wings&#x0201d; to nominal resolutions of 3.15&#x02009;&#x000c5; and 3.17&#x02009;&#x000c5;, respectively (Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">4</a></sup> and Supplementary Table&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">2</a></sup>). A final composite map from the individually refined components was generated and the model coordinates were refined against it (Supplementary Figs.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">4</a></sup>, <sup><a href="#MOESM1" rid="MOESM1" class=" media">5</a></sup>).</p></div><div id="Sec4" class="sec"><h3 id="Sec4title">Architecture of catalytically inactive Apo SPARTA heterodimer</h3><p id="Par9" class="p p-first">The crystal structure of Apo SPARTA heterodimer reveals an extended TIR-APAZ protein interacting extensively with the more compactly folded short pAgo (MID-PIWI) protein (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1d</span></a>). The TIR (residues 1 to 138) and APAZ (residues 161 to 420) domains connect via an extended linker (residues 139 to 160) and subtend a concave surface against which the MID (residues 1 to 269) and PIWI (residues 270 to 507) domains dock (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1d</span></a>). The TIR domain makes contacts exclusively with the MID domain, whereas the APAZ domain makes contacts mostly with the PIWI domain and few contacts with the MID domain (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1d</span></a>). This interface between TIR-APAZ and short pAgo is extensive (burying ~ 2240 &#x000c5;<sup><sup><a href="#CR2" rid="CR2" class=" bibr popnode">2</a></sup></sup>) and underlies the stability of the Apo SPARTA heterodimer in solution (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1d</span></a>).</p><p id="Par10">The SPARTA TIR domain is similar in structure to the TIR domain in proteins with scaffolding and enzymatic roles, composed of a central &#x003b2;-sheet (&#x003b2;A to &#x003b2;D) surrounded by helices (&#x003b1;A to &#x003b1;E) and loops (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1e</span></a>). A prominent loop is the BB loop (between &#x003b2;B and &#x003b1;B) that is frequently involved in TIR self-association<sup><sup><a href="#CR11" rid="CR11" class=" bibr popnode">11</a></sup>,<sup><a href="#CR12" rid="CR12" class=" bibr popnode">12</a></sup></sup>. The characteristic glutamate (Glu77) in TIRs with NADase activity is located on helix &#x003b1;C (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1e</span></a>). The SPARTA TIR aligns with rmsds of 1.06&#x02009;&#x000c5; (113 C&#x003b1;s), 1.05&#x02009;&#x000c5; (120 C&#x003b1;s), 0.86&#x02009;&#x000c5; (119 C&#x003b1;s) and 1.07&#x02009;&#x000c5; (126 C&#x003b1;s) with the TIRs of human TLR2<sup><sup><a href="#CR14" rid="CR14" class=" bibr popnode">14</a></sup></sup> (PDB: 1FYW), human SARM1<sup><sup><a href="#CR15" rid="CR15" class=" bibr popnode">15</a></sup></sup> (PDB: 7NAK), plant NLR-RPP1<sup><sup><a href="#CR16" rid="CR16" class=" bibr popnode">16</a></sup></sup> (PDB: 7DFV), and bacterial TIR-SAVED<sup><sup><a href="#CR17" rid="CR17" class=" bibr popnode">17</a></sup></sup> (PDB: 7QQK), respectively (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1e</span></a> and Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">6a</a></sup>). Although the APAZ domain is often associated with short pAgos, its 3D structure has remained largely uncharacterized. It was originally predicted as analogous to the PAZ domain of long pAgos<sup><sup><a href="#CR7" rid="CR7" class=" bibr popnode">7</a></sup></sup>, though more recent predictions have suggested homology to the N and L1 regions of long pAgos<sup><sup><a href="#CR18" rid="CR18" class=" bibr popnode">18</a></sup>,<sup><a href="#CR19" rid="CR19" class=" bibr popnode">19</a></sup></sup>. The APAZ domain emerges from our structure to be topologically similar to the N, L1 and L2 regions of long pAgos and, by analogy, we divide it into APAZ-N (residues 167-280), APAZ-L1 (residues 281-352) and APAZ-L2 (residues 353-450) regions (Figs.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1</span></a>a, <a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1em;">a,1f</span></span><span>1f</span></a> and Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">6b</a></sup>). APAZ-N is composed of a small four-stranded &#x003b2;-sheet (&#x003b2;H-&#x003b2;K) flanked by three short &#x003b1;-helices (&#x003b1;G to &#x003b1;J) and one long &#x003b1;-helix (&#x003b1;K) that connects to APAZ-L1. APAZ-N is linked to the TIR domain via two short &#x003b2;-strands (&#x003b2;F and &#x003b2;G) connected to the connector helix (&#x003b1;F). APAZ-L1 is dominated by two long &#x003b2;-strands (&#x003b2;O and &#x003b2;P) in the same manner as the L1 region of long pAgos (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1f</span></a> and Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">6b</a></sup>). The major topological difference between APAZ and long pAgos is omission of the region corresponding to the PAZ domain. In long p-Agos, L1 continues into the PAZ domain, whereas in APAZ, APAZ-L1 continues directly into APAZ-L2. APAZ-L2 is dominated by two &#x003b1;-helices (&#x003b1;L and &#x003b1;M) analogous to those observed in the L2 region of long pAgos. However, whereas the L2 of long pAgos continues to the MID domain, APAZ-L2 terminates in a short C-terminal helix (&#x003b1;N, a key element in blocking the binding of target DNA, discussed below) (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1f</span></a> and Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">6b</a></sup>). Overall, APAZ appears to be evolutionarily related to the PAZ lobe of long pAgos. However, the APAZ has lost the PAZ domain but retained regions corresponding to the N, L1 and L2 and acquired another function through fusion with an enzymatic domain such as TIR, SIR2 or a nuclease.</p><p id="Par11" class="p p-last">The SPARTA short pAgo MID domain is composed of a parallel four-stranded &#x003b2;-sheet (&#x003b2;3 to &#x003b2;6) surrounded by &#x003b1;-helices in a typical Rossmann fold (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1g</span></a>). The MID domain is as expected similar in structure to other MID domains superimposing, for example, with the MID domain of long CbAgo<sup><sup><a href="#CR20" rid="CR20" class=" bibr popnode">20</a></sup></sup> (PDB: 6QZK) with a rmsd of 5.23&#x02009;&#x000c5; (for 87 C&#x003b1;s) (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1g</span></a> and Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">6c</a></sup>). A unique feature of the SPARTA MID domain is &#x0201c;insert57&#x0201d; found between &#x003b1;5 and &#x003b1;7, composed of a long helix (&#x003b1;6) and two short helices, that extends outwards from the body of the MID domain. Intriguingly, from sequence alignments, insert57 emerges as a strongly conserved feature of short pAgos but not of long pAgos (Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">7</a></sup>). The PIWI domain is composed of a curved eight stranded &#x003b2;-sheet (&#x003b2;7 to &#x003b2;14) flanked by &#x003b1;-helices from both sides (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1g</span></a>). It has an overall fold of RNAase H domain and aligns with the PIWI domain of long CbAgo with an rmsd of 3.09&#x02009;&#x000c5; (for 140 C&#x003b1;s) (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1g</span></a> and Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">6c</a></sup>). Unlike most long pAgos, the PIWI domains of short pAgos are catalytically inactive and lack the acidic residues for the hydrolysis of DNA or RNA. In the SPARTA PIWI domain, residues Val284, Tyr328, Arg362 and Asn484 substitute for residues Asp541, Glu577, Asp611, and Asp727 in the PIWI domain of catalytically active CbAgo<sup><sup><a href="#CR20" rid="CR20" class=" bibr popnode">20</a></sup></sup>. The SPARTA MID and PIWI domains are positioned in a configuration similar to that in other long and short pAgos, with an extensive interface, burying ~1600 &#x000c5;<sup><sup><a href="#CR2" rid="CR2" class=" bibr popnode">2</a></sup></sup> of surface area. Interestingly, when the MID and PIWI domains of the SPARTA heterodimer are superimposed on the MID and PIWI domains of CbAgo<sup><sup><a href="#CR20" rid="CR20" class=" bibr popnode">20</a></sup></sup>, the APAZ domain occupies the spatial volume as the N, L1 and L2 regions of CbAgo (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1g</span></a> and Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">6c</a></sup>). This lends further credence to the idea that the short pAgo defense systems may have arisen from long pAgos by losing the PAZ domain but gaining a TIR, SIR2 or a nuclease domain instead.</p></div><div id="Sec5" class="sec"><h3 id="Sec5title">Architecture of catalytically active SPARTA oligomer</h3><p id="Par12" class="p p-first-last">The cryo-EM structure of SPARTA in the presence of the gRNA/tDNA reveals four SPARTA heterodimers (A, B, C and D) assembled into a tetramer of heterodimers (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2a, b</span></a>). The oligomer resembles a butterfly, with the MID, PIWI and APAZ domains on the outside (&#x0201c;wings&#x02019;) and the TIR domains on the inside (the &#x0201c;body&#x0201d;) (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2a, b</span></a>). The symmetry of the oligomer is unusual. The MID, PIWI and APAZ domains of heterodimers A and B (or C and D) are related by two-fold symmetry but TIR<sub>A</sub> and TIR<sub>B</sub> (or TIR<sub>C</sub> and TIR<sub>D</sub>) are related to each other by translational symmetry (due to ~175<sup>o</sup> rotation of TIR<sub>B</sub>- described below) (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2</span></a>). Further, TIR<sub>A</sub>-TIR<sub>B</sub> are related to TIR<sub>C</sub>-TIR<sub>D</sub> by screw-like symmetry resulting in a TIR assembly akin to the &#x0201c;parallel-strands&#x0201d; arrangement observed in scaffolding proteins (Myd88 and MAL, for example) as opposed to the &#x0201c;antiparallel-strands&#x0201d; arrangement of TIRs in enzymatic proteins (SARM1 and RPP1, for example)<sup><sup><a href="#CR21" rid="CR21" class=" bibr popnode">21</a></sup></sup> (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2c</span></a>). However, the enzymatic TIRs of AbTir (from <em>Acinetobacter baumannii</em>) have also recently been found to assemble through the parallel-strands arrangement<sup><sup><a href="#CR22" rid="CR22" class=" bibr popnode">22</a></sup></sup> (described below) (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2c</span></a>), suggesting versatility in how enzymatic TIRs can organize for catalysis.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig2" co-legend-rid="lgnd_Fig2"><a href="/pmc/articles/PMC11156904/figure/Fig2/" target="figure" rid-figpopup="Fig2" rid-ob="ob-Fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140500377923904"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11156904_41467_2024_49271_Fig2_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2024_49271_Fig2_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11156904/bin/41467_2024_49271_Fig2_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140500377923904"><a target="object" rel="noopener" href="/pmc/articles/PMC11156904/figure/Fig2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig2"><div><a class="figpopup" href="/pmc/articles/PMC11156904/figure/Fig2/" target="figure" rid-figpopup="Fig2" rid-ob="ob-Fig2">Fig. 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Architecture of the catalytically active SPARTA oligomer.</strong><p><strong>a</strong> The cryo-EM map shows the domain organization of the &#x0201c;butterfly&#x0201d; shaped SPARTA oligomer with gRNA/tDNA. The oligomer is organized as a tetramer of heterodimers (labeled as A, B, C and D). The TIR, APAZ, short pAgo (MID-PIWI) are shown in yellow, green and blue, respectively. The connector between the TIR and APAZ domain is shown in pink and the gRNA/tDNA is depicted in red. This composite cryo-EM map was obtained by using C2 based focused symmetry refinement of both &#x02018;wings&#x02019; and local refinement of TIR domains. <strong>b</strong> Structural arrangement of the SPARTA oligomer. The protein and nucleic acid components are colored as in <strong>a</strong>. Dimensions of the complex are given in &#x000c5;. <strong>c</strong> Structural comparison of the tetrameric arrangement of TIR domain assembly in SPARTA oligomer versus that in select proteins with scaffolding (MAL) and enzymatic (AbTir and SARM1) roles. The secondary structure elements are labeled for SPARTA TIR. AbTir and SARM1 are depicted with bound NAD<sup>+</sup> analogue (shown in sticks).</p></div></div></div></div><div id="Sec6" class="sec"><h3 id="Sec6title">Interactions with guide RNA and target DNA</h3><p id="Par13" class="p p-first">Each SPARTA heterodimer binds a single gRNA/tDNA duplex via their MID, PIWI and APAZ domains (Figs.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2</span></a>a, <a href="/pmc/articles/PMC11156904/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1em;">a,b,</span></span><span>b</span></a>, and <a href="/pmc/articles/PMC11156904/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and3).</span></span><span>3</span></a>). The guide RNA can be traced for a segment corresponding to nucleotides 1-19 and the target DNA to nucleotides 2&#x02019;&#x02212;19&#x02019;. The 3-19 segment of guide RNA is base-paired to the 3&#x02019;&#x02212;19&#x02019; segment of the target DNA and the conformation of the helix is A-form (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span>3</span></a>). Many of the RNA:DNA binding features mimic those observed with long pAgos<sup><sup><a href="#CR5" rid="CR5" class=" bibr popnode">5</a></sup>,<sup><a href="#CR23" rid="CR23" class=" bibr popnode">23</a></sup></sup>. The 5&#x02019; phosphate of the guide strand, for example, inserts into a binding pocket in the MID domain and the first base (U1) is splayed out of the helix (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span>3a</span></a>). The 5&#x02019; phosphate is held by hydrogen bonds with His207 and electrostatic interactions with Lys211 of the MID domain. In addition, a Mg<sup>2+</sup> ion is observed in the MID binding pocket that is coordinated to the 5&#x02019; phosphate, A3 phosphate, and Asn468 (and putatively to the main chain carbonyl of Ile507) (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span>3a</span></a>). Together, the observed interactions to the 5&#x02019; phosphate help to explain requirement for a 5&#x02019; phosphate on the guide RNA. U1 stacks against His207 and makes a direct hydrogen bond with Tyr148 of the MID domain, reminiscent of the interactions observed with long pAgos<sup><sup><a href="#CR5" rid="CR5" class=" bibr popnode">5</a></sup>,<sup><a href="#CR23" rid="CR23" class=" bibr popnode">23</a></sup></sup>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig3" co-legend-rid="lgnd_Fig3"><a href="/pmc/articles/PMC11156904/figure/Fig3/" target="figure" rid-figpopup="Fig3" rid-ob="ob-Fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140500375912720"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11156904_41467_2024_49271_Fig3_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2024_49271_Fig3_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11156904/bin/41467_2024_49271_Fig3_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140500375912720"><a target="object" rel="noopener" href="/pmc/articles/PMC11156904/figure/Fig3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig3"><div><a class="figpopup" href="/pmc/articles/PMC11156904/figure/Fig3/" target="figure" rid-figpopup="Fig3" rid-ob="ob-Fig3">Fig. 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>MID-PIWI and the APAZ domain grip the gRNA/tDNA in the catalytically active SPARTA oligomer.</strong><p><strong>a</strong> The gRNA/tDNA duplex as a whole is accommodated in a channel that runs between the MID-PIWI and the APAZ domains. DNA is shown in light grey and the cartoon representation of MID-PIWI and APAZ domains are shown in blue and green, respectively. Key residues interacting with both the gRNA and tDNA strands as well as Mg<sup>2+</sup> ion are highlighted. <strong>b</strong> Schematic representation of the interactions of MID-PIWI and APAZ domains with gRNA/tDNA strands. Amino acid residues of the MID-PIWI and APAZ domain are indicated by blue and green colors, respectively. DNA backbone phosphates are shown as red circles. Hydrogen bonds are indicated by purple lines (determined by interaction distance &#x0003c;3.6&#x02009;&#x000c5;), electrostatic interactions are shown by orange lines and van der waals interactions are shown by black bars. All the bases constituting the gRNA and tDNA strands used for the reconstitution of the SPARTA complex are shown at the bottom. The modeled RNA, DNA and unmodeled bases in the catalytically active SPARTA oligomer are shown in red, blue and black letters, respectively.</p></div></div></div><p id="Par14" class="p p-last">The gRNA/tDNA duplex as a whole is accommodated in a channel that runs between the MID-PIWI and the APAZ domains. Most of the interactions are to the sugar-phosphate backbones of the gRNA and tDNA, though there are some contacts to the bases (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span>3</span></a>). Hydrogen bonds and/or electrostatic interactions to the sugar-phosphate backbones are from residues Arg72, His207, Lys211, Arg225, Thr228 of the MID domain, Lys286, Lys287, Lys325, Arg362, Arg364, Lys395, Asn439, Asn466, Asn468, Arg481 of the PIWI domain, and Ser288, Arg201, Lys328, His340, Lys366 of the APAZ domain. Notable hydrogen bonds to bases are from Arg243 of the MID domain, and Lys211, Arg263, Arg362 of the APAZ domain (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span>3</span></a>). Overall, the APAZ domain fulfills a function analogous to the N, L1 and L2 regions of long pAgos in facilitating the binding of the guide and target nucleic acid strands<sup><sup><a href="#CR5" rid="CR5" class=" bibr popnode">5</a></sup></sup>. The 3&#x02019;-end of the gRNA lies close to the APAZ-N and can putatively interact with it. A distinction from the N domain of long pAgos such as TtAgo is that APAZ-N does not block the 3&#x02019;-end of the guide RNA<sup><sup><a href="#CR23" rid="CR23" class=" bibr popnode">23</a></sup></sup>. Thus, in principle, base-pairing between the gRNA and tDNA can continue beyond the segments visible in the structure. This is consistent with findings that binding of the guide 3&#x02019;-end is not required for SPARTA activation and that guide RNAs with length of even up to 50 nucleotides can activate SPARTA<sup><sup><a href="#CR3" rid="CR3" class=" bibr popnode">3</a></sup></sup>.</p></div><div id="Sec7" class="sec"><h3 id="Sec7title">Conformational changes upon nucleic acid binding</h3><p id="Par15" class="p p-first-last">Superposition of Apo SPARTA heterodimer onto a heterodimer in the active oligomer reveals two key segments of SPARTA that change conformation to facilitate gRNA/tDNA binding. First is an extended loop between &#x003b1;10 and &#x003b2;9 (loop10-9, residues 319 to 331) of the PIWI domain that protrudes into the nucleic acid binding channel and in the cryo-EM structure reconfigures and shortens by 4 residues (to residues 323-331) to permit gRNA/tDNA duplex binding (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4a, b</span></a>). Strikingly, this conformational change in loop10-9 propagates to the adjoining protein segments, wherein strand &#x003b2;9 loses secondary structure and attains a much more loop-like configuration and residues Asp306 and Asp309 between &#x003b2;8 and &#x003b2;9 (loop8-9) reconfigure to expose a positively charged pocket on the PIWI domain (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4a&#x02013;c</span></a>). Residue Asp309, for example, moves by as much as ~16&#x02009;&#x000c5; from its position in the Apo structure. This &#x0201c;demasking&#x0201d; of a positively charged pocket on the PIWI domain (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4c</span></a>) appears to be an important feature in permitting the oligomerization and activation of SPARTA on gRNA/tDNA binding. Another major conformational change is the C-terminal terminal helix (&#x003b1;N) of APAZ-L2 (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1f</span></a> and Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4a</span></a>), which is ousted from its Apo position to allow binding of the 3&#x02019;-end of the target DNA strand. There is no discernable density for this helix in the cryo-EM structure, suggesting its movement (and subsequent disorder) away from target DNA, and it corresponds to the region described as TIR-APAZ C-terminus in recent SPARTA papers<sup><sup><a href="#CR24" rid="CR24" class=" bibr popnode">24</a></sup>&#x02013;<sup><a href="#CR28" rid="CR28" class=" bibr popnode">28</a></sup></sup>. Interestingly, the C2&#x02019; base of the target DNA evicts from the gRNA/tDNA helix and occupies the position vacated by helix &#x003b1;N, leaving the second base of the guide RNA (G2) to pair with a surrogate Arg243 from the MID domain (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span>3a</span></a>). Overall, loop10-9 and helix &#x003b1;N appear to be key elements that maintain SPARTA in an auto-inhibited state prior to gRNA/tDNA binding. Another striking conformational difference is in insert57 of the MID domain. Unlike the crystal structure, there is no discernable density for insert57 in the cryo-EM structure, indicative of its flexibility. In the crystal structure, insert57 is ordered and extends from the core of the MID domain (Figs.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1d</span></a> and&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4a</span></a>) and we postulate it as putative site for interaction with other (as yet to be discovered) components of the SPARTA defense system.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig4" co-legend-rid="lgnd_Fig4"><a href="/pmc/articles/PMC11156904/figure/Fig4/" target="figure" rid-figpopup="Fig4" rid-ob="ob-Fig4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140500375552384"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11156904_41467_2024_49271_Fig4_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2024_49271_Fig4_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11156904/bin/41467_2024_49271_Fig4_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140500375552384"><a target="object" rel="noopener" href="/pmc/articles/PMC11156904/figure/Fig4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig4"><div><a class="figpopup" href="/pmc/articles/PMC11156904/figure/Fig4/" target="figure" rid-figpopup="Fig4" rid-ob="ob-Fig4">Fig. 4</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Conformational changes upon gRNA/tDNA binding.</strong><p><strong>a</strong> Overlay of the APAZ, MID and PIWI domains in the inactive Apo and the activated SPARTA oligomer structures, wherein loop10-9 undergoes a reconfiguration and helix &#x003b1;N moves away from the nucleic acid binding groove to facilitate gRNA/tDNA binding. In addition, insert57 is disordered in the oligomer structure. <strong>b</strong> An overlay of the secondary structure elements (&#x003b1;10, &#x003b2;9 and &#x003b2;8), loops 10-9 and 8-9, and residues Asp306 and Asp309 between the inactive Apo and the activated SPARTA oligomer structures. Loop10-9 (between &#x003b1;10 and &#x003b2;9) undergoes reconfiguration and shortening on gRNA/tDNA binding and propagates to lengthen and extend loop8-9 (between &#x003b2;8 and &#x003b2;9) carrying residues Asp306 and Asp309. Residue Asp309 moves by as much as ~16&#x02009;&#x000c5; from its position in the Apo structure. &#x003b1;10 was removed from the lower panel for better clarity. <strong>c</strong> Electrostatic surface potential map of inactive Apo SPARTA (left) and active gRNA/tDNA-SPARTA complex (middle). The comparison shows the exposure of a positively charged pocket in the activated SPARTA complex. On the right, secondary structure elements, loops 10-9 and 8-9, and residues Asp306 and Asp309 from the Apo and the activated SPARTA oligomer structures overlaid on the electrostatic surface of the activated oligomer. The figure shows that the positively charged pocket on the activated oligomer surface is masked by Asp306 and Asp309 in the Apo structure. The reconfiguration and shortening of the loop10-9 upon gRNA/tDNA binding leads to the lengthening and extension of loop8-9, causing in turn the displacement of Asp306 and Asp309 from the pocket.</p></div></div></div></div><div id="Sec8" class="sec"><h3 id="Sec8title">Active site</h3><p id="Par16" class="p p-first-last">Superposition of Apo SPARTA heterodimer onto heterodimers A, B, C and D in the active oligomer reveals an asymmetric change in the TIR domains. Thus, whereas the TIR domains of heterodimers A and C are in the same orientation as in the Apo heterodimer, the TIR domains of heterodimers B and D rotate from their positions in the Apo heterodimer by ~175<sup>o</sup> in the active oligomer (Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">8</a></sup>). A hallmark of TIR domains, whether in scaffolding or enzymatic function, is self-association into oligomers<sup><sup><a href="#CR21" rid="CR21" class=" bibr popnode">21</a></sup></sup>. For TIRs that degrade NAD<sup>+</sup>, self-association is mandatory in assembling a composite active site (spanning two subunits) for the binding and hydrolysis of NAD<sup>+</sup>. Interactions between TIR<sub>A</sub> and TIR<sub>C</sub> as well as TIR<sub>B</sub> and TIR<sub>D</sub> are mediated by a BCD interface, involving residues from helices &#x003b1;B and &#x003b1;C of TIR<sub>C</sub> or TIR<sub>D</sub> and residues from helix &#x003b1;D and loop DE of TIR<sub>A</sub> and TIR<sub>B</sub> (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2c</span></a>). Interactions between TIR<sub>A</sub> and TIR<sub>B</sub> as well as TIR<sub>C</sub> and TIR<sub>D</sub> are by mediated by a BE interface, involving residues from the BB loop of TIR<sub>B</sub> or TIR<sub>D</sub> and residues from &#x003b2;D, &#x003b2;E and &#x003b1;E of TIR<sub>A</sub> or TIR<sub>C</sub> (Figs.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2c</span></a> and &#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5a</span></a>). Importantly, the residues Arg114 and Asn116 from the &#x003b2;E of TIR<sub>A</sub> or TIR<sub>C</sub> are involved in extensive hydrogen bonds/electrostatic interactions with residues from the BB loop of the TIR<sub>B</sub> or TIR<sub>D</sub> (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5a</span></a>). To test the importance of this interface we mutated Arg114 to a glutamate and Asn116 to an alanine, evaluated the activity of SPARTA in a fluorescence assay and found that it resulted in the complete loss of activity (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5b</span></a>). The same BCD and BE interfaces are observed in the similarly organized TIRs of AbTir with the active site spanning the BE interface across two subunits<sup><sup><a href="#CR22" rid="CR22" class=" bibr popnode">22</a></sup></sup> (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2c</span></a>). The BE interface is prevalent among enzymatic TIRs, including those that are organized differently such as in SARM1<sup><sup><a href="#CR15" rid="CR15" class=" bibr popnode">15</a></sup></sup>. Figure&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5c</span></a> and Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">9a</a></sup> compare the SPARTA active site formed across TIR<sub>A</sub> and TIR<sub>B</sub> (and the same for TIR<sub>C</sub> and TIR<sub>D</sub>) with that in AbTir and SARM1, the structures for which have been determined in the presence of the NAD<sup>+</sup> analogue 3AD (8-amino-isoquinoline adenine dinucleotide)<sup><sup><a href="#CR15" rid="CR15" class=" bibr popnode">15</a></sup>,<sup><a href="#CR22" rid="CR22" class=" bibr popnode">22</a></sup></sup>. Many of the key residues involved in 3AD binding in AbTir and SARM1 are structurally conserved in SPARTA, including catalytic Glu77 for NAD(P)<sup>+</sup> hydrolysis (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5c</span></a> and Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">9a</a></sup>). Additionally, we modeled NAD<sup>+</sup> in the TIR active site (formed across TIR<sub>A</sub> and TIR<sub>B</sub>) and analyzed the sequences of enzymatically active prokaryotic and eukaryotic TIR domains with CrtSPARTA<sup>TIR</sup> (Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">9b</a></sup> and <sup><a href="#MOESM1" rid="MOESM1" class=" media">c</a></sup>). From the structural analysis, most of the active site residues are conserved among all TIRs with only a few substituted, the most intriguing of which is the substitution of Trp227 in AbTir and Trp662 in SARM1 with Tyr105 in SPARTA (Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">9a</a></sup>). The tryptophan in AbTir and SARM1 mediates stacking with the adenine base of 3AD, and Tyr105 in SPARTA can putatively mediate the same type of aromatic stacking interactions with adenine base of NAD(P)<sup>+</sup>. Indeed, we observe significant loss in the NADase activity, when we replace Tyr105 by alanine (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5b</span></a>). In AbTir, Trp204 has been shown to facilitate the generation of cyclic products from NAD<sup>+</sup><sup><sup><a href="#CR22" rid="CR22" class=" bibr popnode">22</a></sup></sup>. The corresponding residue in SPARTA is Trp46 and when we mutate this residue to alanine it results in almost complete loss of NADase activity (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5b</span></a>), even though there is no evidence to indicate that SPARTA produces cyclic products following NAD(P)<sup>+</sup> hydrolysis<sup><sup><a href="#CR3" rid="CR3" class=" bibr popnode">3</a></sup></sup>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig5" co-legend-rid="lgnd_Fig5"><a href="/pmc/articles/PMC11156904/figure/Fig5/" target="figure" rid-figpopup="Fig5" rid-ob="ob-Fig5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140500378072832"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11156904_41467_2024_49271_Fig5_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2024_49271_Fig5_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11156904/bin/41467_2024_49271_Fig5_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140500378072832"><a target="object" rel="noopener" href="/pmc/articles/PMC11156904/figure/Fig5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig5"><div><a class="figpopup" href="/pmc/articles/PMC11156904/figure/Fig5/" target="figure" rid-figpopup="Fig5" rid-ob="ob-Fig5">Fig. 5</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Analysis of the TIR active site.</strong><p><strong>a</strong> Details of interactions between TIR<sub>A</sub> and TIR<sub>B</sub> (BE interface). The residues from BB-loop of TIR<sub>B</sub> forms extensive electrostatic interactions with residues from &#x003b2;D, &#x003b2;E and &#x003b1;E of TIR<sub>A</sub>. <strong>b</strong> Comparison of the NADase activity of TIR and pAgo mutants with wild-type SPARTA. TIR-Trp46 and TIR-Tyr105 are catalytic residues of SPARTA which are structurally conserved in AbTir and SARM1. Arg114 and Asn116 are involved at TIR<sub>A</sub>-TIR<sub>B</sub> interface. Gln35 and Tyr37 are involved at the pAgo-pAgo interface. The mutation of these residues results in a major reduction in the NADase activity. Data are presented as mean values +/- SD from experiments performed in triplicates. Source data are provided as a Source Data file. <strong>c</strong> Close-up view of the SPARTA active site formed across the TIR<sub>A</sub> and TIR<sub>B</sub> heterodimers is compared with that in AbTir (PDB:7UXU) and SARM1 (PDB: 7NAK). AbTir and SARM1 are depicted with bound NAD<sup>+</sup> analogue (3AD).</p></div></div></div></div><div id="Sec9" class="sec sec-last"><h3 id="Sec9title">Mechanism of oligomerization and activation</h3><p id="Par17" class="p p-first">Contacts between heterodimers A and B (and similarly between C and D) in the SPARTA are dominated by symmetric protein-protein interactions between the pAgos. In particular, a positively charged pocket on PIWI<sub>A</sub> receives residues 130-135 from MID<sub>B</sub> and vice versa (Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">10</a></sup>). Most of the interactions are polar, wherein the main chain carbonyls of residues Lys130, Asn131, Glu133, Glu134 of MID<sub>B</sub> interact with the main chain amide of Ala502 and side chains of Lys314 and Lys504 of PIWI<sub>A</sub>, among other interactions. In addition, residues Gln35 and Tyr37 of MID<sub>B</sub> interact with Gly38, Lys40, Glu87 and Asn90 of MID<sub>A</sub> (Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">10</a></sup>). Indeed, when we mutate Gln35 and Tyr37 to alanine it results in the complete loss in SPARTA enzymatic activity, indicating the importance of MID-mediated dimerization of SPARTA (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5b</span></a>). Interestingly, we observe a significant population of SPARTA dimers of heterodimers in our biophysical experiments (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1c</span></a> and Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">2a</a></sup>), suggesting a hierarchy of SPARTA assembly, wherein the dimers of heterodimers form first via symmetric interactions between the pAgos and then transition to tetramers of heterodimers via interactions of their TIR domains.</p><p id="Par18">A key question in understanding how SPARTA becomes activated is whether there are any conformational changes on nucleic acid binding that might promote oligomerization? Intriguingly, in Apo SPARTA, the positively charged pocket on PIWI<sub>A</sub> that mediates oligomerization is masked by negatively charged residues Asp306 and Asp309 on the loop8-9 (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4c</span></a>). In the SPARTA oligomer, Asp306 and Asp309 rearrange (on loop8-9) as a consequence of the reconfiguration and shortening of loop10-9 on nucleic acid binding (<em>vide supra</em>, Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4</span></a>). Loop8-9 both lengthens and extends (towards &#x003b1;10) and residues Asp306 and Asp309 move &#x0003e;10&#x02009;&#x000c5; from their position in the Apo structure to demask the positively charged pocket for dimerization. We argued that deletion of residues from loop10-9 may prevent the lengthening and extension of loop8-9 and the reconfiguration of residues Asp306 and Asp309 on nucleic acid binding. Indeed, when we delete four residues from loop10-9, SPARTA loses the ability to form dimers on nucleic acid binding (Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">11</a></sup>), lending to almost complete loss of NADase activity (Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5b</span></a>). Overall, reconfiguration of loop10-9 on nucleic acid binding appears to be the key conformational change that propagates to the dimerization interface, as a step towards tetramerization.</p><p id="Par19">During the preparation and review of our manuscript, related papers on SPARTA were published<sup><sup><a href="#CR24" rid="CR24" class=" bibr popnode">24</a></sup>&#x02013;<sup><a href="#CR31" rid="CR31" class=" bibr popnode">31</a></sup></sup>. Our work complements and provides a firmer understanding of the SPARTA structure and mechanism. First, we succeed in deriving a crystal structure of Apo SPARTA heterodimer that provides a high-resolution view of the TIR-APAZ protein and the MID and PIWI domains of the short pAgo. The MID domain is observed, for example, to contain a unique insertion (insert57). Second, the crystal structure permits the building of SPARTA in the cryo-EM structure of the active oligomer without reference to an AlphaFold model. Third, a comparison between the crystal and cryo-EM structures posits an activation mechanism, whereby a configurational change in a critical loop in SPARTA (loop10-9) on nucleic acid binding propagates to the oligomeric interface to create a positively charged surface for active assembly.</p><p id="Par20" class="p p-last">Collectively, the crystallographic and cryo-EM studies presented here provide a visualization of SPARTA before and after RNA/ssDNA binding, leading to NAD(P)<sup>+</sup> degradation and cell death. We show that a loop (loop10-9) on the PIWI domain and a helix (&#x003b1;N) on the APAZ domain reconfigure to relieve autoinhibition and permit nucleic acid binding. The transition from an inactive heterodimer to an active tetramer of heterodimer amasses the TIR domains in a parallel-strands arrangement for NAD(P)<sup>+</sup> hydrolysis. We show that reconfiguration and shortening of loop10-9 on nucleic acid binding propagates to the heterodimer dimerization interface to create a positively charged surface for active assembly. A prominent feature of the MID domain in the crystal structure is insert57, which emerges as a relatively conserved sequence feature of short pAgo systems but is absent in long pAgos (Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">7</a></sup>). Short pAgos systems have been cataloged into four phylogenetic clades, S1B, S2A and S2B that are associated with an APAZ domain containing protein, and S1A in which SIR2/APAZ is fused to short pAgo<sup><sup><a href="#CR3" rid="CR3" class=" bibr popnode">3</a></sup>,<sup><a href="#CR6" rid="CR6" class=" bibr popnode">6</a></sup></sup>. SPARTA belongs to the S2A clade and proteins in the S2B and S1B clades also contain a large portion of insert57. S1A proteins contain a significant portion of insert57. We do not yet know the role of insert57. However, given its long outward extension from the MID domain we speculate a possible role for it in interactions with host or other components of the short pAgo systems.</p></div></div><div id="Sec10" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec10title">Methods</h2><div id="Sec11" class="sec sec-first"><h3 id="Sec11title">Protein purification</h3><p id="Par21" class="p p-first">For the expression and purification of heterodimeric TIR-APAZ/short pAgo complex from <em>Crenotalea thermophila</em> (SPARTA), pET-6xHis-MBP-TEV-CrtTIR-APAZ/CrtAgo vector was used<sup><sup><a href="#CR3" rid="CR3" class=" bibr popnode">3</a></sup></sup>. This vector was a kind gift from Prof. Dr. Daan Swarts (Wageningen University) (Addgene # 183146). The plasmid was transformed into <em>Escherichia coli</em> BL21 Star (DE3) cells and grown at 37&#x02009;&#x000b0;C until the culture reached an OD<sub>600</sub> of ~0.8. The temperature was then reduced to 18&#x02009;&#x000b0;C and expression induced by the addition of 0.5&#x02009;mM IPTG, followed by incubation for 16&#x02009;h. The cells were then resuspended in a lysis buffer (50&#x02009;mM Tris pH 7.5, 250&#x02009;mM NaCl, 10% glycerol, 0.01% IGEPAL, 20&#x02009;mM imidazole, and 10&#x02009;mM 2-mercaptoethanol) in the presence of Pierce Protease Inhibitor tablets, EDTA-free (Thermofisher), and 1&#x02009;mM PMSF. The cells were lysed by sonication, clarified by centrifugation, and the filtered supernatant loaded onto a HisTrap HP affinity column (GE Healthcare), and the protein eluted using an imidazole gradient ranging from 25-500&#x02009;mM. The fractions containing CrtSPARTA protein complex were pooled and further loaded onto an amylose resin column (New England Biolabs). The column was washed with 20 column volumes (CV) of wash buffer (20&#x02009;mM Tris pH 7.5, 500&#x02009;mM NaCl and 5&#x02009;mM 2-mercaptoethanol). The bound protein was eluted with wash buffer containing 20&#x02009;mM maltose and fractions containing the protein were pooled and TEV protease was added in the 1:20 ratio for cleaving the 6xHis-MBP tag. The 6xHis-MBP tag was cleaved following dialysis against cleavage buffer (20&#x02009;mM HEPES pH 7.5, 250&#x02009;mM KCl, 2&#x02009;mM EDTA, 5&#x02009;mM 2-mercaptoethanol) at room temperature. To remove the cleaved 6xHis-MBP tag, the protein was reapplied to the HisTrap column, and the cleaved protein was&#x000a0;collected as flow through. The flow through containing the untagged SPARTA complex was diluted to a final salt concentration of 100&#x02009;mM. The diluted sample was then loaded onto a HiTrap Heparin column and the protein eluted using a KCl salt gradient from 100&#x02009;mM&#x02212;2.5&#x02009;M. The peak fractions containing the protein of interest were pooled, concentrated, and subjected to size exclusion chromatography (SEC) using a HiLoad 16/600 Superdex 200 (GE Healthcare) column, preequilibrated with 20&#x02009;mM HEPES pH 7.5, 100&#x02009;mM KCl, and 1&#x02009;mM DTT.</p><p id="Par22" class="p p-last">The CrtSPARTA mutants were generated by GenScript and the mutant proteins were purified in a similar fashion to the wild type protein.</p></div><div id="Sec12" class="sec"><h3 id="Sec12title">Crystallization, data collection, and structure determination</h3><p id="Par23" class="p p-first-last">Crystallization trials for the apo SPARTA complex were carried out at a concentration of 5&#x02009;mg/ml in a sitting drop vapor diffusion format, using various commercially available screen kits with 0.3&#x02009;&#x003bc;l of protein mixed with an equal volume of reservoir solution. Small crystalline precipitate was obtained in a condition containing 0.1&#x02009;M sodium acetate pH 4.5, 0.8&#x02009;M sodium phosphate monobasic and 1.2&#x02009;M potassium phosphate dibasic. This crystalline precipitate was used as seed for further optimization of crystals by varying the salts and seed concentration. The optimized crystals diffracted poorly (~7&#x02009;&#x000c5;) with synchrotron radiation under cryogenic conditions (NSLS-II 17-ID-1 and 17-ID-2 beamlines) at the Brookhaven National Laboratory. To improve diffraction, we screened various cryoprotective mixtures and observed gradual improvement in diffraction. The use of a cryoprotectant mixture containing 0.65 M L-Proline, 6.5% Glycerol, 12.5% Tacsimate pH 7.0, and 9% Sucrose resulted in an improvement in resolution to ~3&#x02009;&#x000c5; and the best crystal diffracted to 2.66&#x02009;&#x000c5;. The diffraction data were processed using autoPROC and STARANISO (Global Phasing Ltd.). The crystal belonged to P6<sub>5</sub>22 space group with unit cell dimensions of a = b&#x02009;=&#x02009;197.18&#x02009;&#x000c5;, c&#x02009;=&#x02009;183.4&#x02009;&#x000c5;, &#x003b1;&#x02009;=&#x02009;&#x003b2;&#x02009;=&#x02009;90&#x000b0;, &#x003b3;&#x02009;=&#x02009;120&#x000b0;. The structure was solved by molecular replacement using Phaser-MR<sup><sup><a href="#CR32" rid="CR32" class=" bibr popnode">32</a></sup></sup> with a model of CrtSPARTA generated by AlphaFold<sup><sup><a href="#CR13" rid="CR13" class=" bibr popnode">13</a></sup></sup> as the search model. Subsequently, iterative manual building and refinement were performed using Coot v0.8.9.1 EL<sup><sup><a href="#CR33" rid="CR33" class=" bibr popnode">33</a></sup></sup> and Phenix v1.20cr3-4406-000 Refine<sup><sup><a href="#CR34" rid="CR34" class=" bibr popnode">34</a></sup></sup>. All molecular graphic figures were prepared using PyMOL v2.5.4 (Schr&#x000f6;dinger LLC).</p></div><div id="Sec13" class="sec"><h3 id="Sec13title">Reconstitution of the CrtSPARTA-gRNA/tDNA complex</h3><p id="Par24" class="p p-first-last">The samples for mass photometry, negative staining, and single particle cryo-EM were prepared similarly. The CrtSPARTA-gRNA/tDNA duplex complex was prepared by mixing 1.5&#x02009;mg/ml CrtSPARTA, 5&#x02009;mM MgCl<sub>2</sub>, and 16&#x02009;&#x003bc;M 5&#x02032;-phosphorylated (5&#x02032;-P) 21nt-long ssRNA (5&#x02019;-P-UGAGGUAGUAGGUUGUAUAGU-3&#x02019;, Dharmacon) in the size exclusion buffer (20&#x02009;mM HEPES pH 7.5, 100&#x02009;mM KCl and 1&#x02009;mM DTT). The mixture was incubated at 37&#x02009;&#x000b0;C for 1&#x02009;hour followed by addition of 16&#x02009;&#x003bc;M 45nt-long complementary target ssDNA (5&#x02019;-AAACGACGGCCAGTGCCAAGCTTACTATACAACCTACTACCTCAT-3&#x02019;, IDT). SPARTA with gRNA/tDNA was then incubated at 55&#x02009;&#x000b0;C for 12&#x02009;hours. After incubation, the sample was centrifuged at 12,000&#x02009;rpm for 30&#x02009;min to remove any insoluble aggregates.</p></div><div id="Sec14" class="sec"><h3 id="Sec14title">Mass photometry</h3><p id="Par25" class="p p-first-last">Mass photometry experiments were performed on a One<sup>MP</sup> mass photometer (Refeyn Ltd, Oxford, UK). The procedure used was based on a previously published protocol<sup><sup><a href="#CR35" rid="CR35" class=" bibr popnode">35</a></sup></sup>. The datasets were recorded with Acquire<sup>MP</sup> using standard settings. Briefly, self-adhesive silicon gasket (4-6 wells, ~15&#x02009;&#x003bc;L volume each) was placed on a clean borosilicate glass microscope cover slide. The glass slide/silicone gasket assembly was placed on the instrument&#x02019;s objective and centered on a single well. 12&#x02009;&#x003bc;L of sample buffer (20&#x02009;mM HEPES pH 7.5, 100&#x02009;mM KCl, 2&#x02009;mM DTT) was added to the well, and the focal position of the glass surface was determined and held constant using an autofocus system. 1&#x02009;&#x003bc;L of sample (100-fold diluted Apo CrtSPARTA or the CrtSPARTA-gRNA/tDNA complex) was then added to the same well, mixed with the buffer and a 90&#x02009;sec movie recorded immediately after the dilution. Contrast-to-mass linear calibration curve was created using BSA (monomer and dimer) and apoferritin protein standards prepared in the sample buffer. The movies were analyzed and processed using Discover<sup>MP</sup>.</p></div><div id="Sec15" class="sec"><h3 id="Sec15title">Negative stain microscopy</h3><p id="Par26" class="p p-first-last">Negatively stained samples were prepared at a sample concentration of 10&#x02009;&#x000b5;g/ml using 400-mesh Cu/Pd grids (Ted Pella; prepared by floating carbon on it). The negatively stained samples were stained with uranyl formate and imaged on a Tecnai F20 TEM operating at 200&#x02009;kV using a DE-20 camera at a calibrated magnification of 62,000X and a &#x02212;2.0 &#x003bc;m defocus. Particle images were picked using the Blob picker module within cryoSPARC v3.3.1<sup><sup><a href="#CR36" rid="CR36" class=" bibr popnode">36</a></sup></sup> and subjected to iterative rounds of 2-D classification.</p></div><div id="Sec16" class="sec"><h3 id="Sec16title">Cryo-EM specimen preparation and data collection</h3><p id="Par27" class="p p-first-last">The complex of SPARTA oligomer with gRNA/tDNA was prepared on 300-mesh UltrAuFoil R1.2/1.3 &#x003bc;m grids. The grids were glow discharged using Ar and O<sub>2</sub> for 8 secs using solarus II plasma cleaner (Gatan) prior to loading 2.5 &#x003bc;L of the protein complex. Samples were prepared in a chamber with 95 % humidity at 20&#x02009;&#x000b0;C. The grids were backblotted and vitrified in liquid ethane cooled with liquid nitrogen on the Leica GP2 plunge freezer (Leica microsystems). Vitrified samples of the SPARTA complex were screened using Glacios (ThermoFisher) operated at 200&#x02009;kV prior to large-scale data collection. Screened grids were imaged using a Titan Krios operated at 300&#x02009;kV (ThermoFisher) equipped with a BioQuantum energy filter (Gatan) and K3 direct electron detector (Gatan). The untilted (0&#x000b0;) and the tilted (tilted to 30&#x000b0; and 45&#x000b0;) datasets of the SPARTA complex were imaged in counting mode at a calibrated pixel size of 1.083&#x02009;&#x000c5;/pixel using image shift with a nominal defocus range of &#x02212;0.8 &#x003bc;m to &#x02212;2.5 &#x003bc;m. The frames were aligned using MotionCor2<sup><sup><a href="#CR37" rid="CR37" class=" bibr popnode">37</a></sup></sup> in Appion<sup><sup><a href="#CR38" rid="CR38" class=" bibr popnode">38</a></sup></sup> and CTF estimated using Patch CTF within cryoSPARC v3.3.1<sup><sup><a href="#CR36" rid="CR36" class=" bibr popnode">36</a></sup></sup>.</p></div><div id="Sec17" class="sec"><h3 id="Sec17title">Image processing of the SPARTA oligomer</h3><p id="Par28" class="p p-first">Upon frame alignment and CTF estimation, particles were picked using template picker within cryoSPARC v3.3.1 using templates generated by negative stain microscopy. The initial particle stack was subjected to multiple rounds of 2-D classification which showed high resolution structural features of the complex. Selected particles were subjected to train a model (TM1) with topaz, a neural network based particle picker<sup><sup><a href="#CR39" rid="CR39" class=" bibr popnode">39</a></sup></sup> implemented within cryoSPARC v3.3.1. The micrographs were binned by sixteen and used with resnet8 neural network architecture. Further processing of the 0&#x000b0; dataset resulted in anisotropic maps and, therefore, a tilted dataset (tilted to 30&#x000b0; and 45&#x000b0;, 13,884 micrographs) of the SPARTA complex was imaged. The data was collected in counting mode at 81,000X Magnification (1.083&#x02009;&#x000c5;/pixel). Total dose was set to 51.11 e<sup>&#x02013;</sup>/&#x000c5;<sup><sup><a href="#CR2" rid="CR2" class=" bibr popnode">2</a></sup></sup> fractionated into 40 frames. Particles were reextracted from the smaller tilted dataset (6,621 micrographs) using the topaz model (TM1). Multiple rounds of 2-D classification resulted in 2-D classes which were used to train a topaz model (TM2). This model was used to pick particles from the entire tilted dataset (13,884 micrographs) and after several rounds of 2-D classification and Ab-initio clean up a new topaz model was generated (TM3). This model (TM3) was used to pick particles from the tilted dataset (13,884 micrographs) which resulted in representative 2-D classes (Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">3</a></sup>). The 3-D reconstruction from the selected particles resulted in an Ab-initio model (544,322 particles) which underwent 3-D classification followed by non-uniform refinement in cryoSPARC v3.3.1<sup><sup><a href="#CR40" rid="CR40" class=" bibr popnode">40</a></sup></sup> and resulted in an isotropic construction with a sphericity of 0.90. The reconstruction was generated using 238,432 particles and had a nominal resolution (FSC<sub>0.143</sub>) of 3.35&#x02009;&#x000c5; (Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">3</a></sup>).</p><p id="Par29" class="p p-last">Each monomer in the &#x02018;wings&#x02019; (APAZ-MID-PIWI-RNA/DNA) is related to the other by C2 symmetry. In order to improve the resolution of the &#x02018;wings&#x02019; of the SPARTA oligomer complex, focused refinement of each of the wings (dimers) was performed separately. We fitted the modeled coordinates into the globally refined map (described below) and generated a mask using the command molmap in UCSF chimera (Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">4</a></sup>). In order to perform the C2-focused refinement of the &#x02018;wings&#x02019;, the approximate voxel coordinates of the symmetry center were found using the center of mass of the mask for each of the &#x02018;wings&#x02019; using UCSF chimera. This center was used to translate the volume, mask, and stack of particles with C2 symmetry enforced (volume alignment utility) followed by local refinement in cryoSPARC v3.3.1 (Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">4</a></sup>). C2-focused refinement improved the nominal resolution of each of the &#x02018;wings&#x02019; to 3.15&#x02009;&#x000c5; and 3.17&#x02009;&#x000c5;, respectively and resulted in higher local resolution maps for them (Supplementary Figs.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">4</a></sup> and <sup><a href="#MOESM1" rid="MOESM1" class=" media">5</a></sup>). Local refinement of the TIR domains was performed with C1 symmetry.</p></div><div id="Sec18" class="sec"><h3 id="Sec18title">Model building, refinement, and analysis of the SPARTA oligomer</h3><p id="Par30" class="p p-first-last">A model of the apo form of the SPARTA heterodimer from the crystal structure was used to build into the initial globally refined cryo-EM density map of the complex of SPARTA oligomer (with guide RNA and target DNA) by rigid body docking in Coot v0.9.3. Coot was used to build the gRNA/tDNA and manually rebuild and reorganize sections of the tetramer which were different from that of the monomer. There is some putative density for A1&#x02019; of target DNA but due to its weak nature, we did not build this nucleotide. A composite map obtained from focused refinements was generated using the vop maximum command in UCSF Chimera. The model was rebuilt and refined against the composite map using multiple rounds of real space refinement routine within Phenix v1.20<sup><sup><a href="#CR41" rid="CR41" class=" bibr popnode">41</a></sup></sup> with both base-pair and secondary structure restraints imposed. The models were validated using MolProbity<sup><sup><a href="#CR42" rid="CR42" class=" bibr popnode">42</a></sup></sup>. Superimposition of structures was performed in Coot. The figures were prepared using UCSF ChimeraX v1.3<sup><sup><a href="#CR43" rid="CR43" class=" bibr popnode">43</a></sup></sup> and PyMOL v2.5.4.</p></div><div id="Sec19" class="sec"><h3 id="Sec19title">&#x003b5;-NAD assay</h3><p id="Par31" class="p p-first-last">A reaction mixture consisting of purified SPARTA heterodimer or the SPARTA mutants in SEC buffer, 50&#x02009;&#x000b5;M &#x003b5;-NAD<sup>+</sup> (Biolog, N010), 2&#x02009;&#x000b5;M 21nt-long ssRNA guide and 5X reaction buffer (50&#x02009;mM MES pH 6.5, 375&#x02009;mM KCl, and 10&#x02009;mM MgCl2) was prepared in PCR tubes. The mixture was incubated at 37<sup>o</sup>C for 15&#x02009;min, after which a 45nt-long ssDNA target was added to a final concentration of 2.5&#x02009;&#x000b5;M. The final concentrations of each component were &#x02212; 1&#x02009;&#x000b5;M SPARTA complex or the mutants, 50&#x02009;&#x000b5;M &#x003b5;-NAD<sup>+</sup>, 10&#x02009;mM MES pH 6.5, 125&#x02009;mM KCl, and 2&#x02009;mM MgCl<sub>2</sub> in a final volume of 60&#x02009;&#x000b5;L. After the addition of the target DNA, the tubes were transferred to a preheated PCR machine at a temperature of 55<sup>o</sup>C and incubated for 1&#x02009;hour. The fluorescence intensity was measured using an excitation wavelength of 310&#x02009;nm and an emission wavelength of 410&#x02009;nm using a Synergy H1 Hybrid Multi-Mode Plate Reader. The graph was generated using GraphPad Prism v10.0.2 (232).</p></div><div id="Sec20" class="sec sec-last"><h3 id="Sec20title">Reporting summary</h3><p id="Par32" class="p p-first-last">Further information on research design is available in the&#x000a0;<sup><a href="#MOESM3" rid="MOESM3" class=" media">Nature Portfolio Reporting Summary</a></sup> linked to this article.</p></div></div><div id="sec-a.m.d" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="headless nomenu"></h2><!--/article/body/sec/--><div id="Sec21" class="sec sec-first"><h3 id="Sec21title">Supplementary information</h3><p class="p p-first-last">
</p><div class="sec suppmat" id="MOESM1"><div class="sup-box half_rhythm" id="media-a.m.d.a.b.a.a"><a href="/pmc/articles/PMC11156904/bin/41467_2024_49271_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl">Supplementary Information</a><sup>(68M, pdf)</sup></div></div><p>
</p><div class="sec suppmat" id="MOESM2"><div class="sup-box half_rhythm" id="media-a.m.d.a.b.b.a"><a href="/pmc/articles/PMC11156904/bin/41467_2024_49271_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl">Peer Review File</a><sup>(676K, pdf)</sup></div></div><p>
</p><div class="sec suppmat" id="MOESM3"><div class="sup-box half_rhythm" id="media-a.m.d.a.b.c.a"><a href="/pmc/articles/PMC11156904/bin/41467_2024_49271_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl">Reporting Summary</a><sup>(1.6M, pdf)</sup></div></div><p>
</p></div><div id="Sec22" class="sec sec-last"><h3 id="Sec22title">Source data</h3><p class="p p-first-last">
</p><div class="sec suppmat" id="MOESM4"><div class="sup-box half_rhythm" id="media-a.m.d.b.b.a.a"><a href="/pmc/articles/PMC11156904/bin/41467_2024_49271_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl">Source Data</a><sup>(2.5M, xlsx)</sup></div></div><p>
</p></div></div><div id="ack-a.n.c" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.n.ctitle">Acknowledgements</h2><div class="sec"><p>This work was funded by grant R35-GM131780 (A.K.A) from the National Institutes of Health (NIH). We thank the staff at the National Synchrotron Light Source II (NSLS-II) beamlines 17-ID-1 and 17-ID-2 for facilitating X-ray data collection. NSLS-II is a United States Department of Energy (DOE) Office of Science User Facility operated for the DOE Office of Science by Brookhaven National Laboratory under Contract No. DE-SC0012704. The Center for BioMolecular Structure (CBMS) at NSLS-II is primarily supported by the NIH, National Institute of General Medical Sciences (NIGMS) through a Center Core P30 Grant (P30GM133893), and by the DOE Office of Biological and Environmental Research (KP1605010). Mass photometry experiments and most of the cryo-EM work were performed at the Simons Electron Microscopy Center and National Resource for Automated Molecular Microscopy, located at the New York Structural Biology Center, supported by grants from the Simons Foundation (SF349247), NYSTAR, and the NIH National Institute of General Medical Sciences (GM103310), with additional support from Agouron Institute (F00316), NIH (OD019994) and NIH (RR029300). We thank Dr. William (Chase) Budell for the assistance with the mass photometry experiments. We also thank Carolina Hernandez and Joshua Mendez for their assistance with cryo-EM data collection and Oliver Clarke for his input on cryo-EM data processing.</p><p>Computing resources needed for this work were provided in part by the High-Performance Computing facility of the Icahn School of Medicine at Mount Sinai. Molecular graphics and analyses were performed with UCSF Chimera, developed by the Resource for Biocomputing, Visualization, and Informatics at the University of California, San Francisco, with support from NIH P41-GM103311.</p></div></div><div id="notes-a.n.d" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.n.dtitle">Author contributions</h2><p>J.K. and A.K.A. conceived the project; J.K., R.M. and A.K.A. designed the experiments; J.K performed the protein expression, activity assays, and X-ray crystallographic studies; J.K. and R.M. performed the mass photometry studies; R.M. performed the single particle EM studies, assisted by J.K.; A.K.A. guided the overall project. A.K.A., J.K. and R.M. wrote the manuscript.</p></div><div id="notes-a.n.e" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.n.etitle">Peer review</h2><div id="FPar1" class="sec sec-first"><h2 id="FPar1title">Peer review information</h2><p id="Par33" class="p p-first-last"><em>Nature Communications</em> thanks the anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></div></div><div id="notes-a.n.f" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.n.ftitle">Data availability</h2><p>The structure factors and coordinate files for the crystal structure of the Apo SPARTA have been deposited in the Protein Data Bank (PDB) under the accession code <a href="http://doi.org/10.2210/pdb8U7B/pdb" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">8U7B</a>.</p><p>The original, composite cryo-EM density map as well as the focused refinement maps of the SPARTA oligomer generated in this study have been deposited in the Electron Microscopy Data Bank (EMDB) under accession numbers <a href="https://www.ebi.ac.uk/emdb/EMD-41959" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">EMD-41959</a>, (Original map), <a href="https://www.ebi.ac.uk/emdb/EMD-41945" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">EMD-41945</a> (Focused refined map of the dimer [APAZ-MID-PIWI-RNA/DNA]), <a href="https://www.ebi.ac.uk/emdb/EMD-41947" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">EMD-41947</a> (Focused refined map of the second dimer [APAZ-MID-PIWI-RNA/DNA]), <a href="https://www.ebi.ac.uk/emdb/EMD-41948" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">EMD-41948</a> (Focused refined map of the TIR domains) and <a href="https://www.ebi.ac.uk/emdb/EMD-41966" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">EMD-41966</a> (Composite map). The resulting atomic coordinates for the SPARTA oligomer have been deposited in the Protein Data Bank (PDB) with accession number PDB ID: <a href="http://doi.org/10.2210/pdb8U72/pdb" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">8U72</a>.</p><p>Raw uncropped gel image (for Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1b</span></a>) and raw &#x003b5;-NAD<sup>+</sup> assay data (for Fig.&#x000a0;<a href="/pmc/articles/PMC11156904/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5b</span></a>) are available as source data file accompanying this manuscript.&#x000a0;<a href="#Sec22" rid="Sec22" class=" sec">Source data</a> are provided in this paper.</p></div><div id="FPar2" class="tsec sec"><h2 class="head no_bottom_margin" id="FPar2title">Competing interests</h2><p id="Par34">The authors declare no competing interests.</p></div><div id="fn-group-a.n.a" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.n.atitle">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="fn-a.n.a.a"><p class="p p-first-last"><strong>Publisher&#x02019;s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></p><p class="fn sec" id="fn-a.n.a.b"><p class="p p-first-last">These authors contributed equally: Jithesh Kottur, Radhika Malik.</p></p></div></div><div id="article-aaff-info" class="tsec sec"><h2 class="head no_bottom_margin" id="article-aaff-infotitle">Contributor Information</h2><p><span class="fm-affl">Jithesh Kottur, </span><span class="fm-affl"><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ni.ser.vai@ruttokhsehtij" class="oemail">ni.ser.vai@ruttokhsehtij</a></span>.</p><p><span class="fm-affl">Radhika Malik, </span><span class="fm-affl"><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ude.mssm@kilam.akihdar" class="oemail">ude.mssm@kilam.akihdar</a></span>.</p><p><span class="fm-affl">Aneel K. Aggarwal, </span><span class="fm-affl"><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ude.mssm@lawragga.leena" class="oemail">ude.mssm@lawragga.leena</a></span>.</p></div><div id="sec-a.n.b" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="sec-a.n.btitle">Supplementary information</h2><!--/article/back/sec/--><p class="p p-first-last">The online version contains supplementary material available at 10.1038/s41467-024-49271-4.</p></div><div id="Bib1" class="tsec sec"><h2 class="head no_bottom_margin" id="Bib1title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="CR1">1. <span class="element-citation">Patel DJ, Yu Y, Jia N. Bacterial origins of cyclic nucleotide-activated antiviral immune signaling. <span><span class="ref-journal">Mol. Cell. </span>2022;<span class="ref-vol">82</span>:4591–4610. doi: 10.1016/j.molcel.2022.11.006.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9772257/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36460008" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.molcel.2022.11.006" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=Bacterial+origins+of+cyclic+nucleotide-activated+antiviral+immune+signaling&amp;author=DJ+Patel&amp;author=Y+Yu&amp;author=N+Jia&amp;volume=82&amp;publication_year=2022&amp;pages=4591-4610&amp;pmid=36460008&amp;doi=10.1016/j.molcel.2022.11.006&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR2">2. <span class="element-citation">Wein T, Sorek R. Bacterial origins of human cell-autonomous innate immune mechanisms. <span><span class="ref-journal">Nat. Rev. Immunol. </span>2022;<span class="ref-vol">22</span>:629–638. doi: 10.1038/s41577-022-00705-4.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35396464" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41577-022-00705-4" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Immunol.&amp;title=Bacterial+origins+of+human+cell-autonomous+innate+immune+mechanisms&amp;author=T+Wein&amp;author=R+Sorek&amp;volume=22&amp;publication_year=2022&amp;pages=629-638&amp;pmid=35396464&amp;doi=10.1038/s41577-022-00705-4&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR3">3. <span class="element-citation">Koopal B, et al. Short prokaryotic Argonaute systems trigger cell death upon detection of invading DNA. <span><span class="ref-journal">Cell. </span>2022;<span class="ref-vol">185</span>:1471–1486 e1419. doi: 10.1016/j.cell.2022.03.012.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9097488/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35381200" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cell.2022.03.012" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Short+prokaryotic+Argonaute+systems+trigger+cell+death+upon+detection+of+invading+DNA&amp;author=B+Koopal&amp;volume=185&amp;publication_year=2022&amp;pages=1471-1486+e1419&amp;pmid=35381200&amp;doi=10.1016/j.cell.2022.03.012&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR4">4. <span class="element-citation">Meister G. Argonaute proteins: Functional insights and emerging roles. <span><span class="ref-journal">Nat. Rev. Genet. </span>2013;<span class="ref-vol">14</span>:447–459. doi: 10.1038/nrg3462.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23732335" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnrg3462" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet&amp;title=Argonaute+proteins:+Functional+insights+and+emerging+roles&amp;author=G+Meister&amp;volume=14&amp;publication_year=2013&amp;pages=447-459&amp;pmid=23732335&amp;doi=10.1038/nrg3462&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR5">5. <span class="element-citation">Swarts DC, et al. The evolutionary journey of Argonaute proteins. <span><span class="ref-journal">Nat. Struct. Mol. Biol. </span>2014;<span class="ref-vol">21</span>:743–753. doi: 10.1038/nsmb.2879.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4691850/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25192263" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnsmb.2879" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Struct.+Mol.+Biol.&amp;title=The+evolutionary+journey+of+Argonaute+proteins&amp;author=DC+Swarts&amp;volume=21&amp;publication_year=2014&amp;pages=743-753&amp;pmid=25192263&amp;doi=10.1038/nsmb.2879&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR6">6. <span class="mixed-citation">Ryazansky, S., Kulbachinskiy, A. &#x00026; Aravin, A. A. The expanded universe of prokaryotic Argonaute proteins. <em>mBio</em><strong>9</strong>, e01935&#x02013;18 (2018). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6299218/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30563906" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="CR7">7. <span class="element-citation">Makarova KS, Wolf YI, van der Oost J, Koonin EV. Prokaryotic homologs of Argonaute proteins are predicted to function as key components of a novel system of defense against mobile genetic elements. <span><span class="ref-journal">Biol. Direct. </span>2009;<span class="ref-vol">4</span>:29. doi: 10.1186/1745-6150-4-29.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2743648/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19706170" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2F1745-6150-4-29" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol.+Direct&amp;title=Prokaryotic+homologs+of+Argonaute+proteins+are+predicted+to+function+as+key+components+of+a+novel+system+of+defense+against+mobile+genetic+elements&amp;author=KS+Makarova&amp;author=YI+Wolf&amp;author=J+van+der+Oost&amp;author=EV+Koonin&amp;volume=4&amp;publication_year=2009&amp;pages=29&amp;pmid=19706170&amp;doi=10.1186/1745-6150-4-29&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR8">8. <span class="element-citation">Koopal B, Mutte SK, Swarts DC. A long look at short prokaryotic Argonautes. <span><span class="ref-journal">Trends Cell Biol. </span>2022;<span class="ref-vol">33</span>:605–618. doi: 10.1016/j.tcb.2022.10.005.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36428175" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.tcb.2022.10.005" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Cell+Biol.&amp;title=A+long+look+at+short+prokaryotic+Argonautes&amp;author=B+Koopal&amp;author=SK+Mutte&amp;author=DC+Swarts&amp;volume=33&amp;publication_year=2022&amp;pages=605-618&amp;pmid=36428175&amp;doi=10.1016/j.tcb.2022.10.005&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR9">9. <span class="element-citation">Essuman K, et al. The SARM1 Toll/Interleukin-1 receptor domain possesses intrinsic NAD(+) cleavage activity that promotes pathological axonal degeneration. <span><span class="ref-journal">Neuron. </span>2017;<span class="ref-vol">93</span>:1334–1343 e1335. doi: 10.1016/j.neuron.2017.02.022.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6284238/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28334607" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.neuron.2017.02.022" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=The+SARM1+Toll/Interleukin-1+receptor+domain+possesses+intrinsic+NAD(+)+cleavage+activity+that+promotes+pathological+axonal+degeneration&amp;author=K+Essuman&amp;volume=93&amp;publication_year=2017&amp;pages=1334-1343+e1335&amp;pmid=28334607&amp;doi=10.1016/j.neuron.2017.02.022&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR10">10. <span class="element-citation">Horsefield S, et al. NAD(+) cleavage activity by animal and plant TIR domains in cell death pathways. <span><span class="ref-journal">Science. </span>2019;<span class="ref-vol">365</span>:793–799. doi: 10.1126/science.aax1911.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31439792" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.aax1911" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=NAD(+)+cleavage+activity+by+animal+and+plant+TIR+domains+in+cell+death+pathways&amp;author=S+Horsefield&amp;volume=365&amp;publication_year=2019&amp;pages=793-799&amp;pmid=31439792&amp;doi=10.1126/science.aax1911&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR11">11. <span class="element-citation">Essuman K, Milbrandt J, Dangl JL, Nishimura MT. Shared TIR enzymatic functions regulate cell death and immunity across the tree of life. <span><span class="ref-journal">Science. </span>2022;<span class="ref-vol">377</span>:eabo0001. doi: 10.1126/science.abo0001.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35857622" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.abo0001" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Shared+TIR+enzymatic+functions+regulate+cell+death+and+immunity+across+the+tree+of+life&amp;author=K+Essuman&amp;author=J+Milbrandt&amp;author=JL+Dangl&amp;author=MT+Nishimura&amp;volume=377&amp;publication_year=2022&amp;pages=eabo0001&amp;pmid=35857622&amp;doi=10.1126/science.abo0001&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR12">12. <span class="element-citation">Li S, Manik MK, Shi Y, Kobe B, Ve T. Toll/interleukin-1 receptor domains in bacterial and plant immunity. <span><span class="ref-journal">Curr. Opin. Microbiol. </span>2023;<span class="ref-vol">74</span>:102316. doi: 10.1016/j.mib.2023.102316.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37084552" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.mib.2023.102316" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Microbiol.&amp;title=Toll/interleukin-1+receptor+domains+in+bacterial+and+plant+immunity&amp;author=S+Li&amp;author=MK+Manik&amp;author=Y+Shi&amp;author=B+Kobe&amp;author=T+Ve&amp;volume=74&amp;publication_year=2023&amp;pages=102316&amp;pmid=37084552&amp;doi=10.1016/j.mib.2023.102316&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR13">13. <span class="element-citation">Jumper J, et al. Highly accurate protein structure prediction with AlphaFold. <span><span class="ref-journal">Nature. </span>2021;<span class="ref-vol">596</span>:583–589. doi: 10.1038/s41586-021-03819-2.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8371605/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34265844" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41586-021-03819-2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Highly+accurate+protein+structure+prediction+with+AlphaFold&amp;author=J+Jumper&amp;volume=596&amp;publication_year=2021&amp;pages=583-589&amp;pmid=34265844&amp;doi=10.1038/s41586-021-03819-2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR14">14. <span class="element-citation">Xu Y, et al. Structural basis for signal transduction by the Toll/interleukin-1 receptor domains. <span><span class="ref-journal">Nature. </span>2000;<span class="ref-vol">408</span>:111–115. doi: 10.1038/35040600.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11081518" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2F35040600" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structural+basis+for+signal+transduction+by+the+Toll/interleukin-1+receptor+domains&amp;author=Y+Xu&amp;volume=408&amp;publication_year=2000&amp;pages=111-115&amp;pmid=11081518&amp;doi=10.1038/35040600&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR15">15. <span class="element-citation">Shi Y, et al. Structural basis of SARM1 activation, substrate recognition, and inhibition by small molecules. <span><span class="ref-journal">Mol. Cell. </span>2022;<span class="ref-vol">82</span>:1643–1659 e1610. doi: 10.1016/j.molcel.2022.03.007.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9188649/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35334231" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.molcel.2022.03.007" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=Structural+basis+of+SARM1+activation,+substrate+recognition,+and+inhibition+by+small+molecules&amp;author=Y+Shi&amp;volume=82&amp;publication_year=2022&amp;pages=1643-1659+e1610&amp;pmid=35334231&amp;doi=10.1016/j.molcel.2022.03.007&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR16">16. <span class="element-citation">Ma S, et al. Direct pathogen-induced assembly of an NLR immune receptor complex to form a holoenzyme. <span><span class="ref-journal">Science. </span>2020;<span class="ref-vol">370</span>:abe3069. doi: 10.1126/science.abe3069.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33273071" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.abe3069" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Direct+pathogen-induced+assembly+of+an+NLR+immune+receptor+complex+to+form+a+holoenzyme&amp;author=S+Ma&amp;volume=370&amp;publication_year=2020&amp;pages=abe3069&amp;doi=10.1126/science.abe3069&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR17">17. <span class="element-citation">Hogrel G, et al. Cyclic nucleotide-induced helical structure activates a TIR immune effector. <span><span class="ref-journal">Nature. </span>2022;<span class="ref-vol">608</span>:808–812. doi: 10.1038/s41586-022-05070-9.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35948638" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41586-022-05070-9" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Cyclic+nucleotide-induced+helical+structure+activates+a+TIR+immune+effector&amp;author=G+Hogrel&amp;volume=608&amp;publication_year=2022&amp;pages=808-812&amp;pmid=35948638&amp;doi=10.1038/s41586-022-05070-9&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR18">18. <span class="element-citation">Burroughs AM, Iyer LM, Aravind L. Two novel PIWI families: roles in inter-genomic conflicts in bacteria and Mediator-dependent modulation of transcription in eukaryotes. <span><span class="ref-journal">Biol. Direct. </span>2013;<span class="ref-vol">8</span>:13. doi: 10.1186/1745-6150-8-13.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3702460/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23758928" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2F1745-6150-8-13" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol.+Direct&amp;title=Two+novel+PIWI+families:+roles+in+inter-genomic+conflicts+in+bacteria+and+Mediator-dependent+modulation+of+transcription+in+eukaryotes&amp;author=AM+Burroughs&amp;author=LM+Iyer&amp;author=L+Aravind&amp;volume=8&amp;publication_year=2013&amp;pages=13&amp;pmid=23758928&amp;doi=10.1186/1745-6150-8-13&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR19">19. <span class="element-citation">Willkomm S, et al. Structural and mechanistic insights into an archaeal DNA-guided Argonaute protein. <span><span class="ref-journal">Nat. Microbiol. </span>2017;<span class="ref-vol">2</span>:17035. doi: 10.1038/nmicrobiol.2017.35.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28319084" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnmicrobiol.2017.35" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Microbiol.&amp;title=Structural+and+mechanistic+insights+into+an+archaeal+DNA-guided+Argonaute+protein&amp;author=S+Willkomm&amp;volume=2&amp;publication_year=2017&amp;pages=17035&amp;pmid=28319084&amp;doi=10.1038/nmicrobiol.2017.35&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR20">20. <span class="element-citation">Hegge JW, et al. DNA-guided DNA cleavage at moderate temperatures by Clostridium butyricum Argonaute. <span><span class="ref-journal">Nucleic Acids Res. </span>2019;<span class="ref-vol">47</span>:5809–5821. doi: 10.1093/nar/gkz306.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6582352/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31069393" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkz306" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=DNA-guided+DNA+cleavage+at+moderate+temperatures+by+Clostridium+butyricum+Argonaute&amp;author=JW+Hegge&amp;volume=47&amp;publication_year=2019&amp;pages=5809-5821&amp;pmid=31069393&amp;doi=10.1093/nar/gkz306&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR21">21. <span class="element-citation">Nimma S, et al. Structural evolution of TIR-domain Signalosomes. <span><span class="ref-journal">Front Immunol. </span>2021;<span class="ref-vol">12</span>:784484. doi: 10.3389/fimmu.2021.784484.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8635717/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34868065" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3389%2Ffimmu.2021.784484" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front+Immunol.&amp;title=Structural+evolution+of+TIR-domain+Signalosomes&amp;author=S+Nimma&amp;volume=12&amp;publication_year=2021&amp;pages=784484&amp;pmid=34868065&amp;doi=10.3389/fimmu.2021.784484&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR22">22. <span class="element-citation">Manik MK, et al. Cyclic ADP ribose isomers: Production, chemical structures, and immune signaling. <span><span class="ref-journal">Science. </span>2022;<span class="ref-vol">377</span>:eadc8969. doi: 10.1126/science.adc8969.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36048923" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.adc8969" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Cyclic+ADP+ribose+isomers:+Production,+chemical+structures,+and+immune+signaling&amp;author=MK+Manik&amp;volume=377&amp;publication_year=2022&amp;pages=eadc8969&amp;pmid=36048923&amp;doi=10.1126/science.adc8969&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR23">23. <span class="element-citation">Wang Y, et al. Nucleation, propagation and cleavage of target RNAs in Ago silencing complexes. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">461</span>:754–761. doi: 10.1038/nature08434.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2880917/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19812667" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnature08434" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Nucleation,+propagation+and+cleavage+of+target+RNAs+in+Ago+silencing+complexes&amp;author=Y+Wang&amp;volume=461&amp;publication_year=2009&amp;pages=754-761&amp;pmid=19812667&amp;doi=10.1038/nature08434&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR24">24. <span class="element-citation">Gao X, et al. Nucleic-acid-triggered NADase activation of a short prokaryotic Argonaute. <span><span class="ref-journal">Nature. </span>2023;<span class="ref-vol">625</span>:822–831. doi: 10.1038/s41586-023-06665-6.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37783228" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41586-023-06665-6" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Nucleic-acid-triggered+NADase+activation+of+a+short+prokaryotic+Argonaute&amp;author=X+Gao&amp;volume=625&amp;publication_year=2023&amp;pages=822-831&amp;pmid=37783228&amp;doi=10.1038/s41586-023-06665-6&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR25">25. <span class="element-citation">Shen Z, et al. Oligomerization-mediated activation of a short prokaryotic Argonaute. <span><span class="ref-journal">Nature. </span>2023;<span class="ref-vol">621</span>:154–161. doi: 10.1038/s41586-023-06456-z.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37494956" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41586-023-06456-z" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Oligomerization-mediated+activation+of+a+short+prokaryotic+Argonaute&amp;author=Z+Shen&amp;volume=621&amp;publication_year=2023&amp;pages=154-161&amp;pmid=37494956&amp;doi=10.1038/s41586-023-06456-z&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR26">26. <span class="element-citation">Ni D, Lu X, Stahlberg H, Ekundayo B. Activation mechanism of a short argonaute-TIR prokaryotic immune system. <span><span class="ref-journal">Sci. Adv. </span>2023;<span class="ref-vol">9</span>:eadh9002. doi: 10.1126/sciadv.adh9002.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10355822/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37467330" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fsciadv.adh9002" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Adv.&amp;title=Activation+mechanism+of+a+short+argonaute-TIR+prokaryotic+immune+system&amp;author=D+Ni&amp;author=X+Lu&amp;author=H+Stahlberg&amp;author=B+Ekundayo&amp;volume=9&amp;publication_year=2023&amp;pages=eadh9002&amp;pmid=37467330&amp;doi=10.1126/sciadv.adh9002&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR27">27. <span class="element-citation">Guo L, et al. Auto-inhibition and activation of a short Argonaute-associated TIR-APAZ defense system. <span><span class="ref-journal">Nat. Chem. Biol. </span>2023;<span class="ref-vol">20</span>:512–520. doi: 10.1038/s41589-023-01478-0.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37932527" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41589-023-01478-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Chem.+Biol.&amp;title=Auto-inhibition+and+activation+of+a+short+Argonaute-associated+TIR-APAZ+defense+system&amp;author=L+Guo&amp;volume=20&amp;publication_year=2023&amp;pages=512-520&amp;pmid=37932527&amp;doi=10.1038/s41589-023-01478-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR28">28. <span class="element-citation">Finocchio G, et al. Target DNA-dependent activation mechanism of the prokaryotic immune system SPARTA. <span><span class="ref-journal">Nucleic Acids Res. </span>2024;<span class="ref-vol">52</span>:2012–2029. doi: 10.1093/nar/gkad1248.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10899771/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/38224450" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkad1248" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Target+DNA-dependent+activation+mechanism+of+the+prokaryotic+immune+system+SPARTA&amp;author=G+Finocchio&amp;volume=52&amp;publication_year=2024&amp;pages=2012-2029&amp;pmid=38224450&amp;doi=10.1093/nar/gkad1248&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR29">29. <span class="element-citation">Wang X, et al. Structural insights into mechanisms of Argonaute protein-associated NADase activation in bacterial immunity. <span><span class="ref-journal">Cell Res. </span>2023;<span class="ref-vol">33</span>:699–711. doi: 10.1038/s41422-023-00839-7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10474274/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37311833" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41422-023-00839-7" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Res.&amp;title=Structural+insights+into+mechanisms+of+Argonaute+protein-associated+NADase+activation+in+bacterial+immunity&amp;author=X+Wang&amp;volume=33&amp;publication_year=2023&amp;pages=699-711&amp;pmid=37311833&amp;doi=10.1038/s41422-023-00839-7&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR30">30. <span class="element-citation">Guo M, et al. Cryo-EM structure of the ssDNA-activated SPARTA complex. <span><span class="ref-journal">Cell Res. </span>2023;<span class="ref-vol">33</span>:731–734. doi: 10.1038/s41422-023-00850-y.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10474254/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37491603" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41422-023-00850-y" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Res.&amp;title=Cryo-EM+structure+of+the+ssDNA-activated+SPARTA+complex&amp;author=M+Guo&amp;volume=33&amp;publication_year=2023&amp;pages=731-734&amp;pmid=37491603&amp;doi=10.1038/s41422-023-00850-y&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR31">31. <span class="element-citation">Zhang JT, Wei XY, Cui N, Tian R, Jia N. Target ssDNA activates the NADase activity of prokaryotic SPARTA immune system. <span><span class="ref-journal">Nat. Chem. Biol. </span>2023;<span class="ref-vol">20</span>:503–511. doi: 10.1038/s41589-023-01479-z.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37932528" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41589-023-01479-z" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Chem.+Biol.&amp;title=Target+ssDNA+activates+the+NADase+activity+of+prokaryotic+SPARTA+immune+system&amp;author=JT+Zhang&amp;author=XY+Wei&amp;author=N+Cui&amp;author=R+Tian&amp;author=N+Jia&amp;volume=20&amp;publication_year=2023&amp;pages=503-511&amp;pmid=37932528&amp;doi=10.1038/s41589-023-01479-z&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR32">32. <span class="element-citation">McCoy AJ, et al. Phaser crystallographic software. <span><span class="ref-journal">J. Appl. Crystallogr. </span>2007;<span class="ref-vol">40</span>:658–674. doi: 10.1107/S0021889807021206.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2483472/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19461840" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1107%2FS0021889807021206" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Appl.+Crystallogr.&amp;title=Phaser+crystallographic+software&amp;author=AJ+McCoy&amp;volume=40&amp;publication_year=2007&amp;pages=658-674&amp;pmid=19461840&amp;doi=10.1107/S0021889807021206&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR33">33. <span class="element-citation">Emsley P, Cowtan K. Coot: model-building tools for molecular graphics. <span><span class="ref-journal">Acta Crystallogr D. Biol. Crystallogr. </span>2004;<span class="ref-vol">60</span>:2126–2132. doi: 10.1107/S0907444904019158.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15572765" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1107%2FS0907444904019158" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Crystallogr+D.+Biol.+Crystallogr.&amp;title=Coot:+model-building+tools+for+molecular+graphics&amp;author=P+Emsley&amp;author=K+Cowtan&amp;volume=60&amp;publication_year=2004&amp;pages=2126-2132&amp;pmid=15572765&amp;doi=10.1107/S0907444904019158&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR34">34. <span class="element-citation">Adams PD, et al. PHENIX: a comprehensive Python-based system for macromolecular structure solution. <span><span class="ref-journal">Acta Crystallogr. D. Biol. Crystallogr. </span>2010;<span class="ref-vol">66</span>:213–221. doi: 10.1107/S0907444909052925.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2815670/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20124702" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1107%2FS0907444909052925" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Crystallogr.+D.+Biol.+Crystallogr.&amp;title=PHENIX:+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&amp;author=PD+Adams&amp;volume=66&amp;publication_year=2010&amp;pages=213-221&amp;pmid=20124702&amp;doi=10.1107/S0907444909052925&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR35">35. <span class="element-citation">Sonn-Segev A, et al. Quantifying the heterogeneity of macromolecular machines by mass photometry. <span><span class="ref-journal">Nat. Commun. </span>2020;<span class="ref-vol">11</span>:1772. doi: 10.1038/s41467-020-15642-w.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7156492/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32286308" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41467-020-15642-w" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Quantifying+the+heterogeneity+of+macromolecular+machines+by+mass+photometry&amp;author=A+Sonn-Segev&amp;volume=11&amp;publication_year=2020&amp;pages=1772&amp;pmid=32286308&amp;doi=10.1038/s41467-020-15642-w&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR36">36. <span class="element-citation">Punjani A, Rubinstein JL, Fleet DJ, Brubaker MA. cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. <span><span class="ref-journal">Nat. Methods. </span>2017;<span class="ref-vol">14</span>:290–296. doi: 10.1038/nmeth.4169.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28165473" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnmeth.4169" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=cryoSPARC:+algorithms+for+rapid+unsupervised+cryo-EM+structure+determination&amp;author=A+Punjani&amp;author=JL+Rubinstein&amp;author=DJ+Fleet&amp;author=MA+Brubaker&amp;volume=14&amp;publication_year=2017&amp;pages=290-296&amp;pmid=28165473&amp;doi=10.1038/nmeth.4169&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR37">37. <span class="element-citation">Zheng SQ, et al. MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. <span><span class="ref-journal">Nat. Methods. </span>2017;<span class="ref-vol">14</span>:331–332. doi: 10.1038/nmeth.4193.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5494038/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28250466" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnmeth.4193" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=MotionCor2:+anisotropic+correction+of+beam-induced+motion+for+improved+cryo-electron+microscopy&amp;author=SQ+Zheng&amp;volume=14&amp;publication_year=2017&amp;pages=331-332&amp;pmid=28250466&amp;doi=10.1038/nmeth.4193&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR38">38. <span class="mixed-citation">Lander, G. C. et al. Appion: an integrated, database-driven pipeline to facilitate EM image processing. <em>J. Struct. Biol</em>. <strong>166</strong>, 95&#x02013;102 (2009). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2775544/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19263523" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="CR39">39. <span class="element-citation">Bepler T, et al. Positive-unlabeled convolutional neural networks for particle picking in cryo-electron micrographs. <span><span class="ref-journal">Nat. Methods. </span>2019;<span class="ref-vol">16</span>:1153–1160. doi: 10.1038/s41592-019-0575-8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6858545/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31591578" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41592-019-0575-8" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=Positive-unlabeled+convolutional+neural+networks+for+particle+picking+in+cryo-electron+micrographs&amp;author=T+Bepler&amp;volume=16&amp;publication_year=2019&amp;pages=1153-1160&amp;pmid=31591578&amp;doi=10.1038/s41592-019-0575-8&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR40">40. <span class="element-citation">Punjani A, Zhang H, Fleet DJ. Non-uniform refinement: adaptive regularization improves single-particle cryo-EM reconstruction. <span><span class="ref-journal">Nat. Methods. </span>2020;<span class="ref-vol">17</span>:1214–1221. doi: 10.1038/s41592-020-00990-8.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33257830" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41592-020-00990-8" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=Non-uniform+refinement:+adaptive+regularization+improves+single-particle+cryo-EM+reconstruction&amp;author=A+Punjani&amp;author=H+Zhang&amp;author=DJ+Fleet&amp;volume=17&amp;publication_year=2020&amp;pages=1214-1221&amp;pmid=33257830&amp;doi=10.1038/s41592-020-00990-8&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR41">41. <span class="element-citation">Afonine PV, et al. New tools for the analysis and validation of cryo-EM maps and atomic models. <span><span class="ref-journal">Acta Crystallogr. D. Struct. Biol. </span>2018;<span class="ref-vol">74</span>:814–840. doi: 10.1107/S2059798318009324.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6130467/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30198894" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1107%2FS2059798318009324" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Crystallogr.+D.+Struct.+Biol.&amp;title=New+tools+for+the+analysis+and+validation+of+cryo-EM+maps+and+atomic+models&amp;author=PV+Afonine&amp;volume=74&amp;publication_year=2018&amp;pages=814-840&amp;pmid=30198894&amp;doi=10.1107/S2059798318009324&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR42">42. <span class="element-citation">Williams CJ, et al. MolProbity: More and better reference data for improved all-atom structure validation. <span><span class="ref-journal">Protein Sci. </span>2018;<span class="ref-vol">27</span>:293–315. doi: 10.1002/pro.3330.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5734394/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29067766" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fpro.3330" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Sci.&amp;title=MolProbity:+More+and+better+reference+data+for+improved+all-atom+structure+validation&amp;author=CJ+Williams&amp;volume=27&amp;publication_year=2018&amp;pages=293-315&amp;pmid=29067766&amp;doi=10.1002/pro.3330&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR43">43. <span class="element-citation">Goddard TD, et al. UCSF ChimeraX: Meeting modern challenges in visualization and analysis. <span><span class="ref-journal">Protein Sci. </span>2018;<span class="ref-vol">27</span>:14–25. doi: 10.1002/pro.3235.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5734306/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28710774" ref="reftype=pubmed&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fpro.3235" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Sci.&amp;title=UCSF+ChimeraX:+Meeting+modern+challenges+in+visualization+and+analysis&amp;author=TD+Goddard&amp;volume=27&amp;publication_year=2018&amp;pages=14-25&amp;pmid=28710774&amp;doi=10.1002/pro.3235&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11156904&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Nature Communications</span> are provided here courtesy of <strong>Nature Publishing Group</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11156904/pdf/41467_2024_Article_49271.pdf" class="int-view">PDF (4.9M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11156904/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11156904/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11156904" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11156904%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="Ib1xsMM6h5082AnrBcUPIhyN9zTxDXtnk09cVpTxJOBcRRgtzxT7rGpvMmBEXEVu">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11156904%2F&amp;text=Nucleic%20acid%20mediated%20activation%20of%20a%20short%20prokaryotic%20Argonaute%20immune%20system" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11156904%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11156904/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11156904/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38844755/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11156904/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38844755/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11156904/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11156904/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC11156904/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
