
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-preprint.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Protein codes promote selective subcellular compartmentalization - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="939BE807DB0F4BC500003B6B86B9D433.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="biorxiv" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC11042338/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11042338/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="bioRxiv" /><meta name="citation_title" content="Protein codes promote selective subcellular compartmentalization" /><meta name="citation_author" content="Henry R. Kilgore" /><meta name="citation_author_institution" content="Whitehead Institute for Biomedical Research, Cambridge, MA 02142, USA." /><meta name="citation_author" content="Itamar Chinn" /><meta name="citation_author_institution" content="Computer Science and Artificial Intelligence Laboratory, Massachusetts Institute of Technology, Cambridge, MA 02139, USA." /><meta name="citation_author" content="Peter G. Mikhael" /><meta name="citation_author_institution" content="Computer Science and Artificial Intelligence Laboratory, Massachusetts Institute of Technology, Cambridge, MA 02139, USA." /><meta name="citation_author" content="Ilan Mitnikov" /><meta name="citation_author_institution" content="Computer Science and Artificial Intelligence Laboratory, Massachusetts Institute of Technology, Cambridge, MA 02139, USA." /><meta name="citation_author" content="Catherine Van Dongen" /><meta name="citation_author_institution" content="Whitehead Institute for Biomedical Research, Cambridge, MA 02142, USA." /><meta name="citation_author" content="Guy Zylberberg" /><meta name="citation_author_institution" content="Computer Science and Artificial Intelligence Laboratory, Massachusetts Institute of Technology, Cambridge, MA 02139, USA." /><meta name="citation_author" content="Lena Afeyan" /><meta name="citation_author_institution" content="Whitehead Institute for Biomedical Research, Cambridge, MA 02142, USA." /><meta name="citation_author_institution" content="Department of Biology, Massachusetts Institute of Technology, Cambridge, MA 02139, USA." /><meta name="citation_author" content="Salman Banani" /><meta name="citation_author_institution" content="Whitehead Institute for Biomedical Research, Cambridge, MA 02142, USA." /><meta name="citation_author_institution" content="Department of Pathology, Brigham and Women&#x02019;s Hospital, Harvard Medical School, Boston, MA 02115, USA." /><meta name="citation_author" content="Susana Wilson-Hawken" /><meta name="citation_author_institution" content="Whitehead Institute for Biomedical Research, Cambridge, MA 02142, USA." /><meta name="citation_author_institution" content="Program of Computational &#x00026; Systems Biology, Massachusetts Institute of Technology, Cambridge, MA 02139, USA" /><meta name="citation_author" content="Tong Ihn Lee" /><meta name="citation_author_institution" content="Whitehead Institute for Biomedical Research, Cambridge, MA 02142, USA." /><meta name="citation_author" content="Regina Barzilay" /><meta name="citation_author_institution" content="Computer Science and Artificial Intelligence Laboratory, Massachusetts Institute of Technology, Cambridge, MA 02139, USA." /><meta name="citation_author" content="Richard A. Young" /><meta name="citation_author_institution" content="Whitehead Institute for Biomedical Research, Cambridge, MA 02142, USA." /><meta name="citation_author_institution" content="Department of Biology, Massachusetts Institute of Technology, Cambridge, MA 02139, USA." /><meta name="citation_doi" content="10.1101/2024.04.15.589616" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC11042338/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC11042338/" /><meta name="citation_pmid" content="38659952" /><meta name="DC.Title" content="Protein codes promote selective subcellular compartmentalization" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Cold Spring Harbor Laboratory Preprints" /><meta name="DC.Contributor" content="Henry R. Kilgore" /><meta name="DC.Contributor" content="Itamar Chinn" /><meta name="DC.Contributor" content="Peter G. Mikhael" /><meta name="DC.Contributor" content="Ilan Mitnikov" /><meta name="DC.Contributor" content="Catherine Van Dongen" /><meta name="DC.Contributor" content="Guy Zylberberg" /><meta name="DC.Contributor" content="Lena Afeyan" /><meta name="DC.Contributor" content="Salman Banani" /><meta name="DC.Contributor" content="Susana Wilson-Hawken" /><meta name="DC.Contributor" content="Tong Ihn Lee" /><meta name="DC.Contributor" content="Regina Barzilay" /><meta name="DC.Contributor" content="Richard A. Young" /><meta name="DC.Identifier" content="10.1101/2024.04.15.589616" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Protein codes promote selective subcellular compartmentalization" /><meta property="og:type" content="article" /><meta property="og:description" content="Cells have evolved mechanisms to distribute ~10 billion protein molecules to subcellular compartments where diverse proteins involved in shared functions must efficiently assemble. Here, we demonstrate that proteins with shared functions share amino acid ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11042338/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" /><meta name="ncbi_pcid" content="preprint" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11042338/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=biorxiv[jour]" class="navlink">bioRxiv</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11042338
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11042338/pdf/nihpp-2024.04.15.589616v2.pdf" class="int-view">PDF (4.3M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11042338/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11042338/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11042338" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11042338%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="vlkoDW9InUvk3Ux4mPQqnV55wrF7BxQsnNbQGYmrNbQDczrRmcqqVWCTbdnMBMUf">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11042338%2F&amp;text=Protein%20codes%20promote%20selective%20subcellular%20compartmentalization" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11042338%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11042338/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11042338/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38659952/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11042338/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38659952/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11042338/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11042338/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=biorxiv[jour]" class="navlink">bioRxiv</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11042338
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-preprints.png" alt="Logo of biorxiv" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="https://www.ncbi.nlm.nih.gov/pmc/about/nihpreprints/" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1">Version 2. <span id="pmcmata">bioRxiv.</span> Preprint. 2024 Apr 15<span class="citation-revised-date"> [revised 2024 May 22]</span>.</div><div class="part2">  <span class="doi"><span>doi: </span><a href="//doi.org/10.1101%2F2024.04.15.589616" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1101/2024.04.15.589616</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC11042338</span></div><div class="fm-other-versions"><a class="jig-ncbipopper oversions" href="#other_ver_id" data-jigconfig="hasArrow: true, destPosition: 'top right', triggerPosition: 'center left', arrowDirection: 'right', openAnimation: 'none', closeAnimation: 'none'">Other versions</a><div id="other_ver_id" class="other_ids" style="display:none;max-height:50%;overflow-y:auto;white-space:nowrap;padding: 0.75em"><ul class="simple-list" style="margin: 0"><li><a class="int-reflink" href="/pmc/articles/PMC11042338.1/">PMC11042338.1</a>; 2024 Apr 15</li><li class="current a_label"><div class="inline_block a_label">
													➤</div>PMC11042338.2; 2024 May 22</li></ul></div></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38659952">38659952</a></div></div></div><div class="pmc-alert-box preprint-alert whole_rhythm" role="alert" aria-label="Preprint"><div class="i-mark"></div><h3 style="color:#000;margin:0;font-family:'Helvetica Neue','Helvetica','Arial',sans-serif">This is a preprint.</h3><div><strong>It has not yet been peer reviewed by a journal.</strong></div><div>The National Library of Medicine is <a href="/pmc/about/nihpreprints/">running a pilot</a> to include preprints that result from research funded by NIH in PMC and PubMed.</div></div><h1 class="content-title">Protein codes promote selective subcellular compartmentalization</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kilgore%20HR%5BAuthor%5D" class="affpopup" co-rid="_co_idm140298081031616" co-class="co-affbox">Henry R. Kilgore</a>,<sup>1,</sup><sup>&#x02020;</sup><sup>*</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Chinn%20I%5BAuthor%5D" class="affpopup" co-rid="_co_idm140298037869984" co-class="co-affbox">Itamar Chinn</a>,<sup>2,</sup><sup>&#x02020;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Mikhael%20PG%5BAuthor%5D" class="affpopup" co-rid="_co_idm140298037867376" co-class="co-affbox">Peter G. Mikhael</a>,<sup>2,</sup><sup>&#x02020;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Mitnikov%20I%5BAuthor%5D" class="affpopup" co-rid="_co_idm140298034560896" co-class="co-affbox">Ilan Mitnikov</a>,<sup>2,</sup><sup>&#x02020;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Van%20Dongen%20C%5BAuthor%5D" class="affpopup" co-rid="_co_idm140298035428800" co-class="co-affbox">Catherine Van Dongen</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Zylberberg%20G%5BAuthor%5D" class="affpopup" co-rid="_co_idm140298035426816" co-class="co-affbox">Guy Zylberberg</a>,<sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Afeyan%20L%5BAuthor%5D" class="affpopup" co-rid="_co_idm140298035424832" co-class="co-affbox">Lena Afeyan</a>,<sup>1,</sup><sup>3</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Banani%20S%5BAuthor%5D" class="affpopup" co-rid="_co_idm140298043088672" co-class="co-affbox">Salman Banani</a>,<sup>1,</sup><sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wilson-Hawken%20S%5BAuthor%5D" class="affpopup" co-rid="_co_idm140298043085952" co-class="co-affbox">Susana Wilson-Hawken</a>,<sup>1,</sup><sup>5</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Lee%20TI%5BAuthor%5D" class="affpopup" co-rid="_co_idm140298041857648" co-class="co-affbox">Tong Ihn Lee</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Barzilay%20R%5BAuthor%5D" class="affpopup" co-rid="_co_idm140298041855664" co-class="co-affbox">Regina Barzilay</a>,<sup>2,</sup><sup>*</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Young%20RA%5BAuthor%5D" class="affpopup" co-rid="_co_idm140298039518912" co-class="co-affbox">Richard A. Young</a><sup>1,</sup><sup>3,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140298081031616"><h3 class="no_margin">Henry R. Kilgore</h3><p><sup>1</sup>Whitehead Institute for Biomedical Research, Cambridge, MA 02142, USA.</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kilgore%20HR%5BAuthor%5D">Henry R. Kilgore</a></div></div><div id="_co_idm140298037869984"><h3 class="no_margin">Itamar Chinn</h3><p><sup>2</sup>Computer Science and Artificial Intelligence Laboratory, Massachusetts Institute of Technology, Cambridge, MA 02139, USA.</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Chinn%20I%5BAuthor%5D">Itamar Chinn</a></div></div><div id="_co_idm140298037867376"><h3 class="no_margin">Peter G. Mikhael</h3><p><sup>2</sup>Computer Science and Artificial Intelligence Laboratory, Massachusetts Institute of Technology, Cambridge, MA 02139, USA.</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Mikhael%20PG%5BAuthor%5D">Peter G. Mikhael</a></div></div><div id="_co_idm140298034560896"><h3 class="no_margin">Ilan Mitnikov</h3><p><sup>2</sup>Computer Science and Artificial Intelligence Laboratory, Massachusetts Institute of Technology, Cambridge, MA 02139, USA.</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Mitnikov%20I%5BAuthor%5D">Ilan Mitnikov</a></div></div><div id="_co_idm140298035428800"><h3 class="no_margin">Catherine Van Dongen</h3><p><sup>1</sup>Whitehead Institute for Biomedical Research, Cambridge, MA 02142, USA.</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Van%20Dongen%20C%5BAuthor%5D">Catherine Van Dongen</a></div></div><div id="_co_idm140298035426816"><h3 class="no_margin">Guy Zylberberg</h3><p><sup>2</sup>Computer Science and Artificial Intelligence Laboratory, Massachusetts Institute of Technology, Cambridge, MA 02139, USA.</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Zylberberg%20G%5BAuthor%5D">Guy Zylberberg</a></div></div><div id="_co_idm140298035424832"><h3 class="no_margin">Lena Afeyan</h3><p><sup>1</sup>Whitehead Institute for Biomedical Research, Cambridge, MA 02142, USA.</p><p><sup>3</sup>Department of Biology, Massachusetts Institute of Technology, Cambridge, MA 02139, USA.</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Afeyan%20L%5BAuthor%5D">Lena Afeyan</a></div></div><div id="_co_idm140298043088672"><h3 class="no_margin">Salman Banani</h3><p><sup>1</sup>Whitehead Institute for Biomedical Research, Cambridge, MA 02142, USA.</p><p><sup>4</sup>Department of Pathology, Brigham and Women&#x02019;s Hospital, Harvard Medical School, Boston, MA 02115, USA.</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Banani%20S%5BAuthor%5D">Salman Banani</a></div></div><div id="_co_idm140298043085952"><h3 class="no_margin">Susana Wilson-Hawken</h3><p><sup>1</sup>Whitehead Institute for Biomedical Research, Cambridge, MA 02142, USA.</p><p><sup>5</sup>Program of Computational &#x00026; Systems Biology, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wilson-Hawken%20S%5BAuthor%5D">Susana Wilson-Hawken</a></div></div><div id="_co_idm140298041857648"><h3 class="no_margin">Tong Ihn Lee</h3><p><sup>1</sup>Whitehead Institute for Biomedical Research, Cambridge, MA 02142, USA.</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Lee%20TI%5BAuthor%5D">Tong Ihn Lee</a></div></div><div id="_co_idm140298041855664"><h3 class="no_margin">Regina Barzilay</h3><p><sup>2</sup>Computer Science and Artificial Intelligence Laboratory, Massachusetts Institute of Technology, Cambridge, MA 02139, USA.</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Barzilay%20R%5BAuthor%5D">Regina Barzilay</a></div></div><div id="_co_idm140298039518912"><h3 class="no_margin">Richard A. Young</h3><p><sup>1</sup>Whitehead Institute for Biomedical Research, Cambridge, MA 02142, USA.</p><p><sup>3</sup>Department of Biology, Massachusetts Institute of Technology, Cambridge, MA 02139, USA.</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Young%20RA%5BAuthor%5D">Richard A. Young</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140298044640608_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140298044640608_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140298044640608_ai" style="display:none"><div class="fm-affl" id="A1"><sup>1</sup>Whitehead Institute for Biomedical Research, Cambridge, MA 02142, USA.</div><div class="fm-affl" id="A2"><sup>2</sup>Computer Science and Artificial Intelligence Laboratory, Massachusetts Institute of Technology, Cambridge, MA 02139, USA.</div><div class="fm-affl" id="A3"><sup>3</sup>Department of Biology, Massachusetts Institute of Technology, Cambridge, MA 02139, USA.</div><div class="fm-affl" id="A4"><sup>4</sup>Department of Pathology, Brigham and Women&#x02019;s Hospital, Harvard Medical School, Boston, MA 02115, USA.</div><div class="fm-affl" id="A5"><sup>5</sup>Program of Computational &#x00026; Systems Biology, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</div><div id="FN1"><sup>&#x02020;</sup>These authors contributed equally to this work.</div><div>Contributed by <p id="P2" class="p p-first">Author Contributions</p>Conceptualization: H.K., R.Y.<p id="P4" class="p">Methodology: H.K., I.C., P.G.M, I.M.</p><p id="P5" class="p">Investigation: H.K., I.C., P.G.M, I.M., C.V.D</p><p id="P6" class="p">Visualization: SFB, MJM, JLS, EH</p><p id="P7" class="p">Funding acquisition: R.B, R.Y.,</p><p id="P8" class="p">Project administration: H.K., R.B., R.Y.</p><p id="P9" class="p">Supervision: H.K., R.B., R.Y.</p><p id="P10" class="p">Writing &#x02013; original draft: H.K., I.C., P.G.M, I.M., R.Y.</p><p id="P11" class="p p-last">Writing &#x02013; review &#x00026; editing:</p></div><div id="CR1"><sup>*</sup>Corresponding author. <a href="mailto:dev@null" data-email="ude.tim.iw@eroglikh" class="oemail">ude.tim.iw@eroglikh</a> (H.R.K), <a href="mailto:dev@null" data-email="ude.tim.liasc@aniger" class="oemail">ude.tim.liasc@aniger</a> (R.B.), <a href="mailto:dev@null" data-email="ude.tim.iw@gnuoy" class="oemail">ude.tim.iw@gnuoy</a> (R.A.Y).</div></div><div class="fm-article-notes half_rhythm"></div><div class="permissions half_rhythm hide" id="idm140298044640608_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">PMC Copyright notice</a> </div><div class="license half_rhythm">This work is licensed under a <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License</a>, which allows reusers to copy and distribute the material in any medium or format in unadapted form only, for noncommercial purposes only, and only so long as attribution is given to the creator.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm" role="complementary" aria-label="Related or updated information about this article.">The complete version history of this preprint is available at <a href="//doi.org/10.1101%2F2024.04.15.589616" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">bioRxiv</a>.</div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplement 1.</strong></div><div><a href="/pmc/articles/PMC11042338/bin/media-1.pdf" data-ga-action="click_feat_suppl">media-1.pdf</a><span style="color:gray"> (11M)</span></div><div class="small guid">GUID: 1E7A88FE-A538-439A-AFDD-18DBB3FF0241</div></div></dd></dl><dl data-length="199" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first-last">Code and model weights used in this analysis are available at <a href="https://github.com/hrkilgore/protein_codes.git" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/hrkilgore/protein_codes.git</a>. Data used in this analysis are available at FigShare (DOI: <a href="//doi.org/10.6084/m9.figshare.25726581" data-ga-action="click_feat_suppl" ref="reftype=extlink-doi&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CDOI" target="_blank">10.6084/m9.figshare.25726581</a>).</p></dd></dl></div><div id="ABS1" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="ABS1title">Abstract</h2><!--article-meta--><div><p id="P12" class="p p-first-last">Cells have evolved mechanisms to distribute ~10 billion protein molecules to subcellular compartments where diverse proteins involved in shared functions must efficiently assemble. Here, we demonstrate that proteins with shared functions share amino acid sequence codes that guide them to compartment destinations. A protein language model, ProtGPS, was developed that predicts with high performance the compartment localization of human proteins excluded from the training set. ProtGPS successfully guided generation of novel protein sequences that selectively assemble in targeted subcellular compartments. ProtGPS also identified pathological mutations that change this code and lead to altered subcellular localization of proteins. Our results indicate that protein sequences contain not only a folding code, but also a previously unrecognized code governing their distribution in specific cellular compartments.</p></div></div><div id="S1" class="tsec sec"><h2 class="head no_bottom_margin" id="S1title">Introduction</h2><p id="P13" class="p p-first-last">Groups of proteins involved in shared functions must efficiently assemble to fulfill their physiological functions<sup><sup><a href="#R1" rid="R1" class=" bibr popnode">1</a></sup></sup>. For example, the fidelity of gene transcription hinges on the assembly of over a hundred different proteins at promoters, where some bind DNA sequences directly and others interact with DNA-bound proteins instead<sup><sup><a href="#R2" rid="R2" class=" bibr popnode">2</a></sup>,<sup><a href="#R3" rid="R3" class=" bibr popnode">3</a></sup></sup>. Selective protein-protein and protein-nucleic acid interactions are thought to be the predominant driving force leading to the assembly of specific proteins at locations where they carry out diverse functions<sup><sup><a href="#R4" rid="R4" class=" bibr popnode">4</a></sup>&#x02013;<sup><a href="#R7" rid="R7" class=" bibr popnode">7</a></sup></sup>. Shape complementarity among structurally stable portions of proteins have dominated models of protein assembly, but there is now considerable evidence that large assemblies of proteins with shared functions also occur through weak multivalent noncovalent interactions<sup><sup><a href="#R8" rid="R8" class=" bibr popnode">8</a></sup>&#x02013;<sup><a href="#R15" rid="R15" class=" bibr popnode">15</a></sup></sup>. Nearly all cellular functions involve formation of such assemblies, which have been described as condensates, aggregates, puncta, hubs and non-membrane bound compartments (<a href="/pmc/articles/PMC11042338/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1A</span></a>). In a recent study, we used small chemical probes to demonstrate that different condensates can harbor distinct internal chemical environments, suggesting that such assemblies have different solvent properties<sup><sup><a href="#R16" rid="R16" class=" bibr popnode">16</a></sup></sup>. It is thus possible that protein molecules that assemble selectively with others in a condensate do so, in part, as a consequence of their compatibility with the internal solvating environment of that compartment<sup><sup><a href="#R17" rid="R17" class=" bibr popnode">17</a></sup>&#x02013;<sup><a href="#R20" rid="R20" class=" bibr popnode">20</a></sup></sup>. Integration of contributions from specific interactions (e.g., DNA-protein binding, protein-protein interactions) and nonspecific interactions (e.g., transient noncovalent interactions) is challenging to model, but protein language models provide a mechanism for generally incorporating diverse contributions. If such a protein language model could be developed, it would have important implications for our understanding of cellular function and dysfunction by providing evidence of a protein code embedded in amino acid sequences guiding distribution.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC11042338/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140298032893216"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11134863_nihpp-2024.04.15.589616v2-f0001.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihpp-2024.04.15.589616v2-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11042338/bin/nihpp-2024.04.15.589616v2-f0001.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140298032893216"><a target="object" rel="noopener" href="/pmc/articles/PMC11042338/figure/F1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC11042338/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Fig 1.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>ProtGPS classifies protein compartment with high performance.</strong><p id="P37"><strong>A</strong>. Graphical depiction of some cellular compartments found in eukaryotic cells, compartments in bold were studied in this work. <strong>B</strong>. Bar graph showing the number of protein sequences gathered from UNIPROT and the CD-code database used in the development of ProtGPS. <strong>C</strong>. Schematic showing the approach toward developing ProtGPS. <strong>D</strong>. Bar graph showing the area under the receiver-operator curve for classification of withheld test data (15 % of total) with ProtGPS.</p></div></div></div></div><div id="S2" class="tsec sec"><h2 class="head no_bottom_margin" id="S2title">Evidence for shared protein codes in condensate compartments</h2><p id="P14" class="p p-first-last">To learn whether collections of proteins that assemble into specific condensate compartments have shared protein codes, we adapted an evolutionary scale protein transformer language model (ESM2) to predict protein assembly into distinct compartments<sup><sup><a href="#R21" rid="R21" class=" bibr popnode">21</a></sup>,<sup><a href="#R22" rid="R22" class=" bibr popnode">22</a></sup></sup>. The transformer architecture of ESM2 allows for simultaneous relationships between all amino acids in an input sequence to be learned, providing a general strategy to detect protein codes embedded in the amino acid sequence of a protein. We focused our studies on a set of 5,541 human protein sequences that have been annotated for twelve condensate compartments using the UNIPROT<sup><sup><a href="#R23" rid="R23" class=" bibr popnode">23</a></sup></sup> and CD-Code<sup><sup><a href="#R24" rid="R24" class=" bibr popnode">24</a></sup></sup> databases (<a href="/pmc/articles/PMC11042338/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1B</span></a>). The compartment identities of the proteins in these databases were determined with various experimental techniques and curated by experts in compartment annotation and whole sequences were used as input<sup><sup><a href="#R44" rid="R44" class=" bibr popnode">44</a></sup></sup>. Compartment annotated whole protein sequences were used as input. A neural network classifier was jointly trained with ESM2 to develop a model, termed ProtGPS, which computes the independent probability of a protein being found within each of the twelve different condensate compartments (<a href="/pmc/articles/PMC11042338/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1C</span></a>). The area under the receiver operator curve (AUC-ROC) showed that protein compartments could be predicted with remarkable accuracy (0.83&#x02013;0.95) across the 12 different compartments (<a href="/pmc/articles/PMC11042338/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1D</span></a>). The performance of the ProtGPS model indicates it detects patterns in the protein primary structure that differentiates these condensate compartments.</p></div><div id="S3" class="tsec sec"><h2 class="head no_bottom_margin" id="S3title">Guided generation of novel protein sequences for compartment selectivity</h2><p id="P15" class="p p-first">In order to validate that ProtGPS has learned the protein codes associated with condensate localization, we sought to design novel protein sequences that, when produced in cells, would selectively assemble into a compartment of interest. To test this idea, we initially designed protein sequences using an autoregressive greedy search algorithm (GS)<sup><sup><a href="#R25" rid="R25" class=" bibr popnode">25</a></sup></sup>. We repeatedly feed a growing sequence into ProtGPS and extend the sequence with additional residues at the N-terminus up to a desired length, choosing an amino acid at each step that causes a protein to be classified as a match for the desired compartment (<a href="#SD1" rid="SD1" class=" supplementary-material">Fig. S1</a>). For each protein, a plasmid was constructed that encoded a generated polypeptide of up to 150 amino acids with an N-terminal nuclear localization sequence to ensure transport into the nucleus and a C-terminal mCherry protein to ascertain the location of the protein by microscopy. In all, we generated eight novel protein sequences were designed to assemble selectively into nucleoli (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S1</a>). However, although these proteins entered the nucleus, they failed to assemble selectively into nucleoli (<a href="#SD1" rid="SD1" class=" supplementary-material">Fig. S1</a>).</p><p id="P16">The failure of our initial efforts to generate proteins that selectively compartmentalize in nucleoli led us to consider how the GS algorithm might be imperfect for this task and motivated the design of another approach that might be more successful. With GS and ProtGPS, protein sequences are generated without consideration of the chemical space of proteins found in nature, but are over optimized towards prediction of subcellular compartments. We sought to create an approach that could overcome this limitation by applying a concept borrowed from medicinal chemistry, where it is common to consider whether a molecule shares desirable physicochemical properties with others<sup><sup><a href="#R26" rid="R26" class=" bibr popnode">26</a></sup>,<sup><a href="#R27" rid="R27" class=" bibr popnode">27</a></sup></sup>. We also chose to favor the use of &#x0201c;intrinsically disordered&#x0201d; domains because they have been implicated in protein association with condensates and because they are less likely to introduce competing folded states<sup><sup><a href="#R28" rid="R28" class=" bibr popnode">28</a></sup>,<sup><a href="#R29" rid="R29" class=" bibr popnode">29</a></sup></sup>. To apply these concepts toward protein generation, we sought to constrain generation to (1) proteins in the chemical space<sup><sup><a href="#R30" rid="R30" class=" bibr popnode">30</a></sup></sup> learned by ESM2, (2) domains that are intrinsically disordered, and (3) sequences that should localize to the intended compartment. Thus, we used additional features of protein chemical space and intrinsic disorder for our Markov Chain Monte Carlo algorithm (MCMC) (<a href="/pmc/articles/PMC11042338/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2A</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC11042338/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140298044059824"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11134863_nihpp-2024.04.15.589616v2-f0002.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihpp-2024.04.15.589616v2-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11042338/bin/nihpp-2024.04.15.589616v2-f0002.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140298044059824"><a target="object" rel="noopener" href="/pmc/articles/PMC11042338/figure/F2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC11042338/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Fig. 2.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Generative modeling creates novel proteins that concentrate in a desired condensate.</strong><p id="P38"><strong>A.</strong> Schematic showing the use of Naturally constrained Markov Chain Monte Carlo to generate proteins and assay them in live cells (MCMC) (<em>see</em>
<a href="#SD1" rid="SD1" class=" supplementary-material">supporting information</a>
<em>for more details</em>). <strong>B.</strong> Live cell image of a colon cancer cell (HCT-116) tagged at the endogenous NPM1 locus with GFP and expressing nucleolus targeted protein NUC1-mCherry, scale: 10 microns<strong>. C.</strong> Live cell confocal micrographs of NUCX-mCherry proteins in HCT-116 cells expressing NPM1-GFP from the endogenous locus cells, scale: 10 microns. <strong>D</strong>. Dot plots showing the measured partition ratios of NUCX (<em>Kx</em> =I<sub>nucleolus</sub> / I<sub>nucleoplasm</sub>) and SPLX-mCherry (<em>Kx</em> = I<sub>SRSF2</sub> / I<sub>nucleoplasm</sub> or = I<sub>SRSF2</sub> / I<sub>cytoplasm</sub>, as indicated by *) proteins relative to the NLS-mCherry control protein, dotted line is the average value of NLS-mCherry protein. See <a href="#SD1" rid="SD1" class=" supplementary-material">Table S3</a> for more information. <strong>E.</strong> Live cell images and quantification showing the relationship of measured partition ratios (<em>K</em><sub><em>x</em></sub> = I<sub>nucleolus</sub> / I<sub>nucleoplasm</sub>) into the nucleolus by proteins on the NUC6-mCherry trajectory to its computed probability of partitioning.</p></div></div></div><p id="P17">We then used MCMC to perform guided generation of proteins, using the additional features described above, that would selectively assemble into two condensate assemblies, nucleoli<sup><sup><a href="#R9" rid="R9" class=" bibr popnode">9</a></sup></sup> and nuclear speckles<sup><sup><a href="#R31" rid="R31" class=" bibr popnode">31</a></sup></sup>, that were selected because they are well-studied, have distinctive functions and morphologies, and possess unambiguous marker proteins (<a href="/pmc/articles/PMC11042338/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2A</span></a>). A total of twenty 100 amino acid long protein sequences, ten targeted to nucleoli and ten to nuclear speckles, were generated using the MCMC (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S2</a>, <a href="/pmc/articles/PMC11042338/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2A</span></a>). Specifically, we use blocked Gibbs sampling with MCMC where we start from a random subsequence and iteratively select residues to mutate such that the final sequence follows the data distribution defined by the proteins that ESM2 was trained on and with ProtGPS and DRBERT<sup><sup><a href="#R32" rid="R32" class=" bibr popnode">32</a></sup></sup> to obtain the localization and disordered properties desired, respectively. For each protein, a plasmid was constructed that encoded the generated protein attached to an N-terminal nuclear localization sequence and a C-terminal mCherry protein. Each of the proteins were expressed in human cells together with the nucleolus marker NPM1-GFP or the nuclear speckle marker SRSF2-GFP and cells expressing both a test protein (mCherry) and the condensate marker (GFP) were isolated using flow cytometry.</p><p id="P18">Imaging of cells revealed that all ten proteins designed to assemble into nucleoli (NUC1&#x02013;10) did indeed concentrate in nucleoli (<a href="/pmc/articles/PMC11042338/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2B</span></a>&#x02013;<a href="/pmc/articles/PMC11042338/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>D</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">S2</a>). Imaging of cells expressing the ten proteins designed by MCMC to assemble into speckles revealed that six of ten (SPL1, 5, 7, 8,10) were enriched in nuclear speckles and two of ten were enriched in cytoplasmic bodies (<a href="/pmc/articles/PMC11042338/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2D</span></a>). The partition ratios of the SPL proteins in speckles were smaller than those observed for NUC proteins in nucleoli, which may be a consequence of the known distribution of speckle proteins into both nuclear speckles and much smaller RNA splicing condensates at nascent transcripts<sup><sup><a href="#R33" rid="R33" class=" bibr popnode">33</a></sup></sup>. Two of the SPL proteins (SPL2 and SPL3) formed cytoplasmic foci that recruited at least one key speckle protein to the body (<a href="/pmc/articles/PMC11042338/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2D</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">S4</a>), suggesting that SPL2 and SPL3 have chemical features that assemble speckle proteins outside of the nucleus. We conclude that most proteins designed by MCMC to assemble into the two condensate compartments studied here - nucleoli and nuclear speckles &#x02013; have indeed gained features that promote their concentration into these compartments.</p><p id="P19" class="p p-last">We next conducted a sensitivity analysis for the generative process. In the multistep optimization process for each generated protein, we might expect that continuous improvement in the score computed during the optimization process should reflect the ability to generate proteins with improved compartmentalization phenotypes. As a test of this prediction, we investigated nucleolar partitioning of proteins generated at different steps during the optimization trajectory for NUC1 and NUC6 (<a href="/pmc/articles/PMC11042338/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2E</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Fig. S5A</a>&#x02013;<a href="#SD1" rid="SD1" class=" supplementary-material">C</a>). Random sequences appended to mCherry, those at step 0, did not show nucleolar compartmentalization. Greater scores produced precursors to NUC1 and NUC6 proteins that tended to show improved nucleolar compartmentalization, although improvement was not continuous (<a href="/pmc/articles/PMC11042338/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2E</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Fig. S5A</a>&#x02013;<a href="#SD1" rid="SD1" class=" supplementary-material">C</a>). These results suggest sampling for greater periods of time will tend to increase the likelihood of generating protein sequences with desired properties.</p></div><div id="S4" class="tsec sec"><h2 class="head no_bottom_margin" id="S4title">Pathogenic mutations can alter protein codes</h2><p id="P20" class="p p-first">Mutations can create pathogenic effects by altering a protein&#x02019;s function or altering a protein&#x02019;s subcellular distribution. Because ProtGPS can accurately predict the subcellular localization of normal proteins, it might be able to identify pathogenic mutations that cause a change in the subcellular location of a mutant protein. To test this possibility, we turned to the ClinVar<sup><sup><a href="#R34" rid="R34" class=" bibr popnode">34</a></sup></sup> database, a public archive of a vast number of human variations classified for diseases. Data was collected for 205,182 mutations and ProtGPS was used to predict if the changes in amino acid sequences alter the subcellular distribution of the mutant proteins (<a href="/pmc/articles/PMC11042338/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3A</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC11042338/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140298043399056"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11134863_nihpp-2024.04.15.589616v2-f0003.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihpp-2024.04.15.589616v2-f0003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11042338/bin/nihpp-2024.04.15.589616v2-f0003.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140298043399056"><a target="object" rel="noopener" href="/pmc/articles/PMC11042338/figure/F3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC11042338/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Fig 3.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Pathogenic mutations are predicted to alter protein compartmentalization.</strong><p id="P39"><strong>A.</strong> Schematic of information flow, pathogenic ClinVar mutants caused by single point or truncation mutations were classified with ProtGPS to determine if the detected protein code was changed in the pathogenic variant. <strong>B.</strong> (<em>Left</em>) Dot plot showing the Shannon entropy change in compartment prediction due to single point or truncation mutation. (<em>Right</em>) Histogram showing the Wasserstein distance between the wild-type and mutant protein compartment probabilities. <strong>C.</strong> Live cell images of mESCs ectopically expressing wild type and truncated pathogenic variants fused to GFP, Wasserstein distance is given for each mutant as w, scale 10 microns.</p></div></div></div><p id="P21">We began our analysis by considering how mutations might influence the information content of condensate compartments. We computed the change in Shannon entropy<sup><sup><a href="#R35" rid="R35" class=" bibr popnode">35</a></sup></sup> of the twelve condensate compartments to learn whether the predicted information content was altered by mutation. We conducted this analysis separately for the truncation mutations (83,211), which we assumed would have major effects, from the single point mutations (121,971), which we assumed would have much smaller effects. We find that the compartment entropy is consistently higher with mutant proteins compared to the normal proteins across all compartments, with truncations producing larger effects than point mutations (<a href="/pmc/articles/PMC11042338/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3B</span></a>). Furthermore, we find that pathogenic truncation and single point mutations, when compared to normal proteins, tend to increase the Wasserstein distance<sup><sup><a href="#R36" rid="R36" class=" bibr popnode">36</a></sup></sup>, a metric of dissimilarity between two probability distributions (<a href="/pmc/articles/PMC11042338/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3B</span></a>). These measures indicate that within this collection of pathogenic proteins, sequence variation may alter the predicted compartments of proteins in ProtGPS, suggesting that some mutant proteins may no longer partition selectively into compartments in the same manner as their normal counterparts.</p><p id="P22" class="p p-last">To experimentally test the prediction that some pathogenic variants cause a change in subcellular localization, we selected for study 20 pathogenic mutations (10 truncation and 10 single point mutations) in proteins involved in a broad range of biological functions and diseases, whose normal cellular compartmentalization was well-known, and that scored across the range of Wasserstein distances (0.162&#x02013;0.000) (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S4</a>). We then generated a panel of mouse embryonic stem cell (V6.5) lines stably expressing each protein from a doxycycline-inducible expression cassette, treated cells with doxycycline and conducted live cell confocal microscopy analysis. Differences in the subcellular localization between normal and mutant proteins would appear as changes in the fluorescence patterns displayed in micrographs. We noted that signals for all the normal proteins occurred in the subcellular locations where they are known to reside. When comparing images of normal proteins with their mutant counterparts, we found striking differences in compartment appearance for almost all truncation mutation proteins, and less striking but clear differences in compartment appearance for point mutation proteins, except for RBM10(V354M)], which scored with a Wasserstein distance of zero (<a href="/pmc/articles/PMC11042338/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3C</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Fig. S6</a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Table S4</a>). Thus, it appeared that proteins calculated to have a large Wasserstein Distance tended to exhibit more dramatic changes in compartment appearance, although this relationship was imperfect. The effects of truncation mutations on nuclear localization sequences could not account for these results (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S4</a>). These results support the notion that ProtGPS can detect changes in protein codes due to pathogenic mutations that are demonstrable in an experimental setting.</p></div><div id="S5" class="tsec sec"><h2 class="head no_bottom_margin" id="S5title">Discussion</h2><p id="P23" class="p p-first">Our studies suggest that proteins have evolved to harbor at least two types of codes, one for folding and another for intracellular compartmentalization. Deep-learning algorithms such as AlphaFold2, RoseTTAFold, Chroma, EvoDiff, ESM2, and others can predict a protein&#x02019;s 3D structure from its linear amino acid sequence<sup><sup><a href="#R22" rid="R22" class=" bibr popnode">22</a></sup>,<sup><a href="#R37" rid="R37" class=" bibr popnode">37</a></sup>&#x02013;<sup><a href="#R42" rid="R42" class=" bibr popnode">42</a></sup></sup>. We here describe ProtGPS, which can predict a protein&#x02019;s selective assembly into specific condensate compartments in cells and be used to guide generation of novel protein sequences whose cellular compartmentalization could largely be experimentally validated. The complexity of the underlying physiochemical rules for both protein folding and protein localization have proven difficult to parse using human interpretable approaches, and these deep-learning approaches therefore provide valuable predictive and analytical tools for the study of protein structure and function.</p><p id="P24">Previous studies of protein compartmentalization have suggested amino acid codes exist for some compartments. For the membrane-bound nucleus, for example, there are well-known nuclear localization sequences that facilitate the transport of protein from the cytoplasm to the nucleus<sup><sup><a href="#R43" rid="R43" class=" bibr popnode">43</a></sup>&#x02013;<sup><a href="#R45" rid="R45" class=" bibr popnode">45</a></sup></sup>. More recently, models were used to identify patterns in protein sequences associated with specific compartments, especially those bounded by a membrane, but these did not sample a broad range of compartments and lacked generative experiments<sup><sup><a href="#R46" rid="R46" class=" bibr popnode">46</a></sup>&#x02013;<sup><a href="#R48" rid="R48" class=" bibr popnode">48</a></sup></sup>. For nonmembrane compartments, here called condensates, there is recent evidence of patterned amino acid sequence features that can engender selective assembly of certain proteins into transcriptional and nucleolar condensates<sup><sup><a href="#R49" rid="R49" class=" bibr popnode">49</a></sup>&#x02013;<sup><a href="#R51" rid="R51" class=" bibr popnode">51</a></sup></sup>. These observations are consistent with the concept of a protein code in amino acid sequences that promotes the selective distribution of proteins into specific compartments. Furthermore, there is recent evidence of distinctive chemical environments within condensates, suggesting that these compartments have different solvent properties<sup><sup><a href="#R16" rid="R16" class=" bibr popnode">16</a></sup>,<sup><a href="#R51" rid="R51" class=" bibr popnode">51</a></sup>,<sup><a href="#R52" rid="R52" class=" bibr popnode">52</a></sup></sup>. Thus, the patterns of amino acid sequences in proteins would be expected to both promote specific folding behaviors and to favor residence in compartments compatible with their solvent properties.</p><p id="P25">The patterns of amino acid sequences that occur in proteins appear overall to be highly constrained in biology<sup><sup><a href="#R53" rid="R53" class=" bibr popnode">53</a></sup>&#x02013;<sup><a href="#R55" rid="R55" class=" bibr popnode">55</a></sup></sup>, and we suggest that this is due, in part, to the requirements for both proper folding and subcellular distribution. In our efforts to develop ProtGPS as a guide for generating novel protein sequences that promote selective subcellular distribution, we found that protein sequences sampled from collections of natural proteins were far more successful at concentrating in the desired compartment than those generated without this consideration. Analogous to the medicinal chemist&#x02019;s aspiration to increase drug-like attributes such as on-target specificity and low off-target effects when developing small molecule therapeutics, designing proteins to preferentially distribute in biochemically relevant regions of the targeted cell population might improve upon their therapeutic properties<sup><sup><a href="#R16" rid="R16" class=" bibr popnode">16</a></sup>,<sup><a href="#R52" rid="R52" class=" bibr popnode">52</a></sup>,<sup><a href="#R56" rid="R56" class=" bibr popnode">56</a></sup></sup>. In addition, exploring the chemical space of proteins naturally present in specific biological compartments may provide an especially valuable guide to the generation of optimal chemical matter directed to target proteins in specific compartments. Indeed, there are widely used and efficacious anti-cancer therapeutics that concentrate in transcriptional condensates at oncogenes<sup><sup><a href="#R56" rid="R56" class=" bibr popnode">56</a></sup></sup> due to the chemical environment of those compartments<sup><sup><a href="#R16" rid="R16" class=" bibr popnode">16</a></sup>,<sup><a href="#R52" rid="R52" class=" bibr popnode">52</a></sup></sup>. It is evident that similar considerations will apply to the design of protein therapeutics. We suggest that further understanding of the chemical environment established by amino acid patterns in proteins will lead to more efficacious disease therapeutics.</p><p id="P26" class="p p-last">We conclude that ProtGPS can predict a protein&#x02019;s selective assembly into specific condensates and guide generation of novel protein sequences whose cellular compartmentalization can be experimentally validated. We anticipate that future studies will advance this field by improving compartment annotation, conducting additional tests of generated proteins, deploying alternative machine learning approaches, and further exploring the effects of pathogenic mutations.</p></div><div id="SM1" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="SM1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplement 1</h4><div class="sup-box half_rhythm" id="d67e513"><a href="/pmc/articles/PMC11042338/bin/media-1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(11M, pdf)</sup></div></div></div><div id="S7" class="tsec sec"><h2 class="head no_bottom_margin" id="S7title">Acknowledgments:</h2><div class="sec"><p id="P28">Support was provided by NIH grant nos. GM144283 (R.A.Y.) and CA155258 (R.A.Y.), NSF grant no. PHY2044895 (R.A.Y.), Damon Runyon Cancer Research Foundation Fellowship grant no. 2458-22 (H.R.K.), and the MIT Jameel Clinic for Machine Learning in Health (R.B., P.G.M., I.C., I.M.) and Eric and Wendy Schmidt Center, Broad Institute (P.G.M., I.C.). We thank Christina Lilliehook, Alessandra Dall&#x02019;Agnese, Mike Gallagher, Yana Petri, Jinyi Yang, Shannon Moreno, and Jeremy Wohlwend for helpful comments and thank Caitlin Rausch and Warbler Creative for graphical artwork.</p><div id="S8" class="sec"><h3 id="S8title">Funding:</h3><p id="P29" class="p p-first-last">Supported by NIH GM144283 (R.A.Y), CA155258 (R.A.Y.), NSF PHY2044895 (R.A.Y.). Damon Runyon Cancer Research Foundation Fellowship 2458-22 (H.R.K). Eric and Wendy Schmidt Center, Broad Institute (P.G.M, I.C.).</p></div></div></div><div id="funding-group-a.g.b.r" class="tsec sec"><h2 class="head no_bottom_margin" id="funding-group-a.g.b.rtitle">Funding Statement</h2><p>Supported by NIH GM144283 (R.A.Y), CA155258 (R.A.Y.), NSF PHY2044895 (R.A.Y.). Damon Runyon Cancer Research Foundation Fellowship 2458-22 (H.R.K). Eric and Wendy Schmidt Center, Broad Institute (P.G.M, I.C.).</p></div><div id="fn-group-a.i.b" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.i.btitle">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN3"><p id="P30" class="p p-first">Competing Interests:</p><p id="P31" class="p p-last">R.A.Y. is a founder and shareholder of Syros Pharmaceuticals, Camp4 Therapeutics, Omega Therapeutics, Dewpoint Therapeutics and Paratus Sciences, and has consulting or advisory roles at Precede Biosciences and Novo Nordisk. R.B. has consulting or advisory roles at Dewpoint Therapeutics, J&#x00026;J, Amgen, Outcomes4Me, Immunai and Firmenich. H.R.K. is a consultant of Dewpoint Therapeutics. I.C. is a consultant of Paratus Sciences. The remaining authors declare no competing interests.</p></p><p class="fn sec" id="FN4"><p id="P32" class="p p-first">Supplementary materials:</p><p id="P33" class="p">
<a href="#SD1" rid="SD1" class=" supplementary-material">Materials and Methods</a>
</p><p id="P34" class="p"><a href="#SD1" rid="SD1" class=" supplementary-material">Figs. S1</a> to <a href="#SD1" rid="SD1" class=" supplementary-material">S6</a></p><p id="P35" class="p"><a href="#SD1" rid="SD1" class=" supplementary-material">Tables S1</a> to <a href="#SD1" rid="SD1" class=" supplementary-material">S4</a></p><p id="P36" class="p p-last"><a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental references 1</a>&#x02013;<a href="#SD1" rid="SD1" class=" supplementary-material">22</a></p></p></div></div><div id="S6" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="S6title">Data and materials availability:</h2><!--/article/back/sec/--><p id="P27" class="p p-first-last">Code and model weights used in this analysis are available at <a href="https://github.com/hrkilgore/protein_codes.git" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/hrkilgore/protein_codes.git</a>. Data used in this analysis are available at FigShare (DOI: <a href="//doi.org/10.6084/m9.figshare.25726581" data-ga-action="click_feat_suppl" ref="reftype=extlink-doi&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CDOI" target="_blank">10.6084/m9.figshare.25726581</a>).</p></div><div id="ref-list-a.i.d" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.i.dtitle">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="mixed-citation">Banani S. F., Lee H. O., Hyman A. A. &#x00026; Rosen M. K.
<span class="ref-title">Biomolecular condensates: Organizers of cellular biochemistry</span>. <span class="ref-journal">Nature Reviews Molecular and Cell Biology</span>
<span class="ref-vol">18</span>, 285&#x02013;285, doi: 10.1038/NRM.2017.7 (2017).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7434221/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28225081" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2FNRM.2017.7" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Reviews+Molecular+and+Cell+Biology&amp;title=Biomolecular+condensates:+Organizers+of+cellular+biochemistry&amp;author=S.+F.+Banani&amp;author=H.+O.+Lee&amp;author=A.+A.+Hyman&amp;author=M.+K.+Rosen&amp;volume=18&amp;publication_year=2017&amp;pages=285-285&amp;pmid=28225081&amp;doi=10.1038/NRM.2017.7&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="mixed-citation">Lambert S. A.
et al. 
<span class="ref-title">The Human Transcription Factors</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">172</span>, 650&#x02013;665, doi: 10.1016/j.cell.2018.01.029 (2018).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/29425488" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cell.2018.01.029" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=The+Human+Transcription+Factors&amp;author=S.+A.+Lambert&amp;volume=172&amp;publication_year=2018&amp;pages=650-665&amp;pmid=29425488&amp;doi=10.1016/j.cell.2018.01.029&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="mixed-citation">Cramer P.
<span class="ref-title">Organization and regulation of gene transcription</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">573</span>, 45&#x02013;54, doi: 10.1038/s41586-019-1517-4 (2019).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/31462772" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41586-019-1517-4" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Organization+and+regulation+of+gene+transcription&amp;author=P.+Cramer&amp;volume=573&amp;publication_year=2019&amp;pages=45-54&amp;pmid=31462772&amp;doi=10.1038/s41586-019-1517-4&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="mixed-citation">Jena S.
et al. 
<span class="ref-title">Noncovalent interactions in proteins and nucleic acids: beyond hydrogen bonding and &#x003c0;-stacking</span>. <span class="ref-journal">Chemical Society Reviews</span>
<span class="ref-vol">51</span>, 4261&#x02013;4286, doi: 10.1039/D2CS00133K (2022).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/35560317" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1039%2FD2CS00133K" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chemical+Society+Reviews&amp;title=Noncovalent+interactions+in+proteins+and+nucleic+acids:+beyond+hydrogen+bonding+and+&#x003c0;-stacking&amp;author=S.+Jena&amp;volume=51&amp;publication_year=2022&amp;pages=4261-4286&amp;pmid=35560317&amp;doi=10.1039/D2CS00133K&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="mixed-citation">Huttlin E. L.
et al. 
<span class="ref-title">Architecture of the human interactome defines protein communities and disease networks</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">545</span>, 505&#x02013;509, doi: 10.1038/nature22366 (2017).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5531611/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28514442" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnature22366" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Architecture+of+the+human+interactome+defines+protein+communities+and+disease+networks&amp;author=E.+L.+Huttlin&amp;volume=545&amp;publication_year=2017&amp;pages=505-509&amp;pmid=28514442&amp;doi=10.1038/nature22366&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="mixed-citation">Luck K.
et al. 
<span class="ref-title">A reference map of the human binary protein interactome</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">580</span>, 402&#x02013;408, doi: 10.1038/s41586-020-2188-x (2020).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7169983/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32296183" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41586-020-2188-x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+reference+map+of+the+human+binary+protein+interactome&amp;author=K.+Luck&amp;volume=580&amp;publication_year=2020&amp;pages=402-408&amp;pmid=32296183&amp;doi=10.1038/s41586-020-2188-x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="mixed-citation">Walport L. J., Low J. K. K., Matthews J. M. &#x00026; Mackay J. P.
<span class="ref-title">The characterization of protein interactions &#x02013; what, how and how much?</span>
<span class="ref-journal">Chemical Society Reviews</span>
<span class="ref-vol">50</span>, 12292&#x02013;12307, doi: 10.1039/D1CS00548K (2021).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/34581717" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1039%2FD1CS00548K" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chemical+Society+Reviews&amp;title=The+characterization+of+protein+interactions+&#x02013;+what,+how+and+how+much?&amp;author=L.+J.+Walport&amp;author=J.+K.+K.+Low&amp;author=J.+M.+Matthews&amp;author=J.+P.+Mackay&amp;volume=50&amp;publication_year=2021&amp;pages=12292-12307&amp;pmid=34581717&amp;doi=10.1039/D1CS00548K&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="mixed-citation">Shin Y. &#x00026; Brangwynne C. P.
<span class="ref-title">Liquid phase condensation in cell physiology and disease</span>. <span class="ref-journal">Science</span>
<span class="ref-vol">357</span>, doi: 10.1126/SCIENCE.AAF4382 (2017). [<a href="https://pubmed.ncbi.nlm.nih.gov/28935776" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2FSCIENCE.AAF4382" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Liquid+phase+condensation+in+cell+physiology+and+disease&amp;author=Y.+Shin&amp;author=C.+P.+Brangwynne&amp;volume=357&amp;publication_year=2017&amp;doi=10.1126/SCIENCE.AAF4382&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="mixed-citation">Feric M.
et al. 
<span class="ref-title">Coexisting liquid phases underlie nucleolar subcompartments</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">165</span>, 1686&#x02013;1697 (2016).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5127388/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27212236" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Coexisting+liquid+phases+underlie+nucleolar+subcompartments&amp;author=M.+Feric&amp;volume=165&amp;publication_year=2016&amp;pages=1686-1697&amp;pmid=27212236&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="mixed-citation">Alberti S. &#x00026; Hyman A. A.
<span class="ref-title">Biomolecular condensates at the nexus of cellular stress, protein aggregation disease and ageing</span>. <span class="ref-journal">Nature Reviews Molecular Cell Biology</span>
<span class="ref-vol">22</span>, 196&#x02013;213, doi: 10.1038/s41580-020-00326-6 (2021).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/33510441" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41580-020-00326-6" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Reviews+Molecular+Cell+Biology&amp;title=Biomolecular+condensates+at+the+nexus+of+cellular+stress,+protein+aggregation+disease+and+ageing&amp;author=S.+Alberti&amp;author=A.+A.+Hyman&amp;volume=22&amp;publication_year=2021&amp;pages=196-213&amp;pmid=33510441&amp;doi=10.1038/s41580-020-00326-6&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="mixed-citation">Choi J.-M., Holehouse A. S. &#x00026; Pappu R. V.
<span class="ref-title">Physical Principles Underlying the Complex Biology of Intracellular Phase Transitions</span>. <span class="ref-journal">Annual Review of Biophysics</span>
<span class="ref-vol">49</span>, 107&#x02013;133, doi: 10.1146/annurev-biophys-121219-081629 (2020). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10715172/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32004090" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1146%2Fannurev-biophys-121219-081629" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annual+Review+of+Biophysics&amp;title=Physical+Principles+Underlying+the+Complex+Biology+of+Intracellular+Phase+Transitions&amp;author=J.-M.+Choi&amp;author=A.+S.+Holehouse&amp;author=R.+V.+Pappu&amp;volume=49&amp;publication_year=2020&amp;pages=107-133&amp;doi=10.1146/annurev-biophys-121219-081629&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="mixed-citation">Tsang B., Priti&#x00161;anac I., Scherer S. W., Moses A. M. &#x00026; Forman-Kay J. D.
<span class="ref-title">Phase Separation as a Missing Mechanism for Interpretation of Disease Mutations</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">183</span>, 1742&#x02013;1756, doi: 10.1016/j.cell.2020.11.050 (2020).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/33357399" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cell.2020.11.050" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Phase+Separation+as+a+Missing+Mechanism+for+Interpretation+of+Disease+Mutations&amp;author=B.+Tsang&amp;author=I.+Priti&#x00161;anac&amp;author=S.+W.+Scherer&amp;author=A.+M.+Moses&amp;author=J.+D.+Forman-Kay&amp;volume=183&amp;publication_year=2020&amp;pages=1742-1756&amp;pmid=33357399&amp;doi=10.1016/j.cell.2020.11.050&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="mixed-citation">Cho W.-K.
et al. 
<span class="ref-title">Mediator and RNA polymerase II clusters associate in transcription-dependent condensates</span>. <span class="ref-journal">Science</span>
<span class="ref-vol">361</span>, 412&#x02013;415, doi: 10.1126/science.aar4199 (2018).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6543815/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29930094" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.aar4199" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Mediator+and+RNA+polymerase+II+clusters+associate+in+transcription-dependent+condensates&amp;author=W.-K.+Cho&amp;volume=361&amp;publication_year=2018&amp;pages=412-415&amp;pmid=29930094&amp;doi=10.1126/science.aar4199&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="mixed-citation">Sabari B. R.
et al. 
<span class="ref-title">Coactivator condensation at super-enhancers links phase separation and gene control</span>. <span class="ref-journal">Science</span>
<span class="ref-vol">361</span>, eaar3958 (2018).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6092193/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29930091" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Coactivator+condensation+at+super-enhancers+links+phase+separation+and+gene+control&amp;author=B.+R.+Sabari&amp;volume=361&amp;publication_year=2018&amp;pages=eaar3958&amp;pmid=29930091&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="mixed-citation">Sheinerman F. B., Norel R. &#x00026; Honig B.
<span class="ref-title">Electrostatic aspects of protein&#x02013;protein interactions</span>. <span class="ref-journal">Current Opinion in Structural Biology</span>
<span class="ref-vol">10</span>, 153&#x02013;159, doi: 10.1016/S0959-440X(00)00065-8 (2000).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/10753808" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FS0959-440X(00)00065-8" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Current+Opinion+in+Structural+Biology&amp;title=Electrostatic+aspects+of+protein&#x02013;protein+interactions&amp;author=F.+B.+Sheinerman&amp;author=R.+Norel&amp;author=B.+Honig&amp;volume=10&amp;publication_year=2000&amp;pages=153-159&amp;pmid=10753808&amp;doi=10.1016/S0959-440X(00)00065-8&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="mixed-citation">Kilgore H. R.
et al. 
<span class="ref-title">Distinct chemical environments in biomolecular condensates</span>. <span class="ref-journal">Nature Chemical Biology</span>, doi: 10.1038/s41589-023-01432-0 (2023). [<a href="https://pubmed.ncbi.nlm.nih.gov/37770698" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41589-023-01432-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Chemical+Biology&amp;title=Distinct+chemical+environments+in+biomolecular+condensates&amp;author=H.+R.+Kilgore&amp;publication_year=2023&amp;doi=10.1038/s41589-023-01432-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="mixed-citation">Yu Y., Wang J., Shao Q., Shi J. &#x00026; Zhu W.
<span class="ref-title">The effects of organic solvents on the folding pathway and associated thermodynamics of proteins: a microscopic view</span>. <span class="ref-journal">Scientific Reports</span>
<span class="ref-vol">6</span>, 19500, doi: 10.1038/srep19500 (2016).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4726029/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26775871" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fsrep19500" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Scientific+Reports&amp;title=The+effects+of+organic+solvents+on+the+folding+pathway+and+associated+thermodynamics+of+proteins:+a+microscopic+view&amp;author=Y.+Yu&amp;author=J.+Wang&amp;author=Q.+Shao&amp;author=J.+Shi&amp;author=W.+Zhu&amp;volume=6&amp;publication_year=2016&amp;pages=19500&amp;pmid=26775871&amp;doi=10.1038/srep19500&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="mixed-citation">Ben-Naim A.
<span class="ref-title">Solvent effects on protein association and protein folding</span>. <span class="ref-journal">Biopolymers</span>
<span class="ref-vol">29</span>, 567&#x02013;596, doi: 10.1002/bip.360290312 (1990).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/2331515" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fbip.360290312" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biopolymers&amp;title=Solvent+effects+on+protein+association+and+protein+folding&amp;author=A.+Ben-Naim&amp;volume=29&amp;publication_year=1990&amp;pages=567-596&amp;pmid=2331515&amp;doi=10.1002/bip.360290312&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="mixed-citation">Klibanov A. M.
<span class="ref-title">Improving enzymes by using them in organic solvents</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">409</span>, 241&#x02013;246, doi: 10.1038/35051719 (2001).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/11196652" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2F35051719" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Improving+enzymes+by+using+them+in+organic+solvents&amp;author=A.+M.+Klibanov&amp;volume=409&amp;publication_year=2001&amp;pages=241-246&amp;pmid=11196652&amp;doi=10.1038/35051719&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="mixed-citation">Prabhu N. &#x00026; Sharp K.
<span class="ref-title">Protein&#x02212;Solvent Interactions</span>. <span class="ref-journal">Chemical Reviews</span>
<span class="ref-vol">106</span>, 1616&#x02013;1623, doi: 10.1021/cr040437f (2006).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2584800/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16683747" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fcr040437f" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chemical+Reviews&amp;title=Protein&#x02212;Solvent+Interactions&amp;author=N.+Prabhu&amp;author=K.+Sharp&amp;volume=106&amp;publication_year=2006&amp;pages=1616-1623&amp;pmid=16683747&amp;doi=10.1021/cr040437f&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="mixed-citation">Chandra A., T&#x000fc;nnermann L., L&#x000f6;fstedt T. &#x00026; Gratz R.
<span class="ref-title">Transformer-based deep learning for predicting protein properties in the life sciences</span>. <span class="ref-journal">eLife</span>
<span class="ref-vol">12</span>, e82819, doi: 10.7554/eLife.82819 (2023).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9848389/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36651724" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.7554%2FeLife.82819" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=Transformer-based+deep+learning+for+predicting+protein+properties+in+the+life+sciences&amp;author=A.+Chandra&amp;author=L.+T&#x000fc;nnermann&amp;author=T.+L&#x000f6;fstedt&amp;author=R.+Gratz&amp;volume=12&amp;publication_year=2023&amp;pages=e82819&amp;pmid=36651724&amp;doi=10.7554/eLife.82819&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="mixed-citation">Lin Z.
et al. 
<span class="ref-title">Evolutionary-scale prediction of atomic-level protein structure with a language model</span>. <span class="ref-journal">Science</span>
<span class="ref-vol">379</span>, 1123&#x02013;1130, doi: 10.1126/science.ade2574 (2023).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/36927031" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.ade2574" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Evolutionary-scale+prediction+of+atomic-level+protein+structure+with+a+language+model&amp;author=Z.+Lin&amp;volume=379&amp;publication_year=2023&amp;pages=1123-1130&amp;pmid=36927031&amp;doi=10.1126/science.ade2574&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="mixed-citation">The UniProt C.
<span class="ref-title">UniProt: the universal protein knowledgebase in 2021</span>. <span class="ref-journal">Nucleic Acids Research</span>
<span class="ref-vol">49</span>, D480&#x02013;D489, doi: 10.1093/nar/gkaa1100 (2021).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7778908/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33237286" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkaa1100" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Research&amp;title=UniProt:+the+universal+protein+knowledgebase+in+2021&amp;author=C.+The+UniProt&amp;volume=49&amp;publication_year=2021&amp;pages=D480-D489&amp;pmid=33237286&amp;doi=10.1093/nar/gkaa1100&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="mixed-citation">Rostam N.
et al. 
<span class="ref-title">CD-CODE: crowdsourcing condensate database and encyclopedia</span>. <span class="ref-journal">Nature Methods</span>
<span class="ref-vol">20</span>, 673&#x02013;676, doi: 10.1038/s41592-023-01831-0 (2023).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10172118/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37024650" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41592-023-01831-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Methods&amp;title=CD-CODE:+crowdsourcing+condensate+database+and+encyclopedia&amp;author=N.+Rostam&amp;volume=20&amp;publication_year=2023&amp;pages=673-676&amp;pmid=37024650&amp;doi=10.1038/s41592-023-01831-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="mixed-citation">Shin J.-E.
et al. 
<span class="ref-title">Protein design and variant prediction using autoregressive generative models</span>. <span class="ref-journal">Nature Communications</span>
<span class="ref-vol">12</span>, 2403, doi: 10.1038/s41467-021-22732-w (2021). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8065141/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33893299" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41467-021-22732-w" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Communications&amp;title=Protein+design+and+variant+prediction+using+autoregressive+generative+models&amp;author=J.-E.+Shin&amp;volume=12&amp;publication_year=2021&amp;pages=2403&amp;doi=10.1038/s41467-021-22732-w&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="mixed-citation">Lipinski C. &#x00026; Hopkins A.
<span class="ref-title">Navigating chemical space for biology and medicine</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">432</span>, 855&#x02013;861, doi: 10.1038/nature03193 (2004).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/15602551" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnature03193" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Navigating+chemical+space+for+biology+and+medicine&amp;author=C.+Lipinski&amp;author=A.+Hopkins&amp;volume=432&amp;publication_year=2004&amp;pages=855-861&amp;pmid=15602551&amp;doi=10.1038/nature03193&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="mixed-citation">Beckers M., Fechner N. &#x00026; Stiefl N.
<span class="ref-title">25 Years of Small-Molecule Optimization at Novartis: A Retrospective Analysis of Chemical Series Evolution</span>. <span class="ref-journal">Journal of Chemical Information and Modeling</span>
<span class="ref-vol">62</span>, 6002&#x02013;6021, doi: 10.1021/acs.jcim.2c00785 (2022).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/36351293" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Facs.jcim.2c00785" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+Chemical+Information+and+Modeling&amp;title=25+Years+of+Small-Molecule+Optimization+at+Novartis:+A+Retrospective+Analysis+of+Chemical+Series+Evolution&amp;author=M.+Beckers&amp;author=N.+Fechner&amp;author=N.+Stiefl&amp;volume=62&amp;publication_year=2022&amp;pages=6002-6021&amp;pmid=36351293&amp;doi=10.1021/acs.jcim.2c00785&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="mixed-citation">Holehouse A. S. &#x00026; Kragelund B. B.
<span class="ref-title">The molecular basis for cellular function of intrinsically disordered protein regions</span>. <span class="ref-journal">Nature Reviews Molecular Cell Biology</span>
<span class="ref-vol">25</span>, 187&#x02013;211, doi: 10.1038/s41580-023-00673-0 (2024).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/37957331" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41580-023-00673-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Reviews+Molecular+Cell+Biology&amp;title=The+molecular+basis+for+cellular+function+of+intrinsically+disordered+protein+regions&amp;author=A.+S.+Holehouse&amp;author=B.+B.+Kragelund&amp;volume=25&amp;publication_year=2024&amp;pages=187-211&amp;pmid=37957331&amp;doi=10.1038/s41580-023-00673-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="mixed-citation">van der Lee R.
et al. 
<span class="ref-title">Classification of Intrinsically Disordered Regions and Proteins</span>. <span class="ref-journal">Chemical Reviews</span>
<span class="ref-vol">114</span>, 6589&#x02013;6631, doi: 10.1021/cr400525m (2014).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4095912/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24773235" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fcr400525m" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chemical+Reviews&amp;title=Classification+of+Intrinsically+Disordered+Regions+and+Proteins&amp;author=R.+van+der+Lee&amp;volume=114&amp;publication_year=2014&amp;pages=6589-6631&amp;pmid=24773235&amp;doi=10.1021/cr400525m&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="mixed-citation">Kirkpatrick P. &#x00026; Ellis C.
<span class="ref-title">Chemical space</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">432</span>, 823&#x02013;823, doi: 10.1038/432823a (2004). [<a href="//doi.org/10.1038%2F432823a" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Chemical+space&amp;author=P.+Kirkpatrick&amp;author=C.+Ellis&amp;volume=432&amp;publication_year=2004&amp;pages=823-823&amp;doi=10.1038/432823a&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="mixed-citation">Ilik &#x00130;. A.
et al. 
<span class="ref-title">SON and SRRM2 are essential for nuclear speckle formation</span>. <span class="ref-journal">eLife</span>
<span class="ref-vol">9</span>, e60579, doi: 10.7554/eLife.60579 (2020).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7671692/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33095160" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.7554%2FeLife.60579" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=SON+and+SRRM2+are+essential+for+nuclear+speckle+formation&amp;author=&#x00130;.+A.+Ilik&amp;volume=9&amp;publication_year=2020&amp;pages=e60579&amp;pmid=33095160&amp;doi=10.7554/eLife.60579&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="mixed-citation">Ananthan N., John Malcolm F., Simon L. &#x00026; Sergei M.
<span class="ref-title">DR-BERT: A Protein Language Model to Annotate Disordered Regions</span>. <span class="ref-journal">bioRxiv</span>, 2023.2002.2022.529574, doi: 10.1101/2023.02.22.529574 (2023). [<a href="//doi.org/10.1101%2F2023.02.22.529574" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=DR-BERT:+A+Protein+Language+Model+to+Annotate+Disordered+Regions&amp;author=N.+Ananthan&amp;author=F.+John+Malcolm&amp;author=L.+Simon&amp;author=M.+Sergei&amp;publication_year=2023&amp;doi=10.1101/2023.02.22.529574&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="mixed-citation">Guo Y. E.
et al. 
<span class="ref-title">Pol II phosphorylation regulates a switch between transcriptional and splicing condensates</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">572</span>, 543&#x02013;548, doi: 10.1038/s41586-019-1464-0 (2019).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6706314/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31391587" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41586-019-1464-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Pol+II+phosphorylation+regulates+a+switch+between+transcriptional+and+splicing+condensates&amp;author=Y.+E.+Guo&amp;volume=572&amp;publication_year=2019&amp;pages=543-548&amp;pmid=31391587&amp;doi=10.1038/s41586-019-1464-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="mixed-citation">Landrum M. J.
et al. 
<span class="ref-title">ClinVar: improving access to variant interpretations and supporting evidence</span>. <span class="ref-journal">Nucleic Acids Research</span>
<span class="ref-vol">46</span>, D1062&#x02013;D1067, doi: 10.1093/nar/gkx1153 (2018).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5753237/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29165669" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkx1153" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Research&amp;title=ClinVar:+improving+access+to+variant+interpretations+and+supporting+evidence&amp;author=M.+J.+Landrum&amp;volume=46&amp;publication_year=2018&amp;pages=D1062-D1067&amp;pmid=29165669&amp;doi=10.1093/nar/gkx1153&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="mixed-citation">Shannon C. E.
<span class="ref-title">A mathematical theory of communication</span>. <span class="ref-journal">The Bell System Technical Journal</span>
<span class="ref-vol">27</span>, 379&#x02013;423, doi: 10.1002/j.1538-7305.1948.tb01338.x (1948). [<a href="//doi.org/10.1002%2Fj.1538-7305.1948.tb01338.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+Bell+System+Technical+Journal&amp;title=A+mathematical+theory+of+communication&amp;author=C.+E.+Shannon&amp;volume=27&amp;publication_year=1948&amp;pages=379-423&amp;doi=10.1002/j.1538-7305.1948.tb01338.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="mixed-citation">Villani C. in <span class="ref-journal">Optimal Transport: Old and New</span> (ed Villani C&#x000e9;dric) 93&#x02013;111 (Springer Berlin Heidelberg, 2009). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Optimal+Transport:+Old+and+New&amp;author=C.+Villani&amp;author=C&#x000e9;dric+Villani&amp;publication_year=2009&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="mixed-citation">Watson J. L.
et al. 
<span class="ref-title">De novo design of protein structure and function with RFdiffusion</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">620</span>, 1089&#x02013;1100, doi: 10.1038/s41586-023-06415-8 (2023).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10468394/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37433327" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41586-023-06415-8" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=De+novo+design+of+protein+structure+and+function+with+RFdiffusion&amp;author=J.+L.+Watson&amp;volume=620&amp;publication_year=2023&amp;pages=1089-1100&amp;pmid=37433327&amp;doi=10.1038/s41586-023-06415-8&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="mixed-citation">Ingraham J. B.
et al. 
<span class="ref-title">Illuminating protein space with a programmable generative model</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">623</span>, 1070&#x02013;1078, doi: 10.1038/s41586-023-06728-8 (2023).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10686827/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37968394" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41586-023-06728-8" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Illuminating+protein+space+with+a+programmable+generative+model&amp;author=J.+B.+Ingraham&amp;volume=623&amp;publication_year=2023&amp;pages=1070-1078&amp;pmid=37968394&amp;doi=10.1038/s41586-023-06728-8&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="mixed-citation">Sarah A.
et al. 
<span class="ref-title">Protein generation with evolutionary diffusion: sequence is all you need</span>. <span class="ref-journal">bioRxiv</span>, 2023.2009.2011.556673, doi: 10.1101/2023.09.11.556673 (2023). [<a href="//doi.org/10.1101%2F2023.09.11.556673" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=Protein+generation+with+evolutionary+diffusion:+sequence+is+all+you+need&amp;author=A.+Sarah&amp;publication_year=2023&amp;doi=10.1101/2023.09.11.556673&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="mixed-citation">Sidney Lyayuga L.
et al. 
<span class="ref-title">Joint Generation of Protein Sequence and Structure with RoseTTAFold Sequence Space Diffusion</span>. <span class="ref-journal">bioRxiv</span>, 2023.2005.2008.539766, doi: 10.1101/2023.05.08.539766 (2023). [<a href="//doi.org/10.1101%2F2023.05.08.539766" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=Joint+Generation+of+Protein+Sequence+and+Structure+with+RoseTTAFold+Sequence+Space+Diffusion&amp;author=L.+Sidney+Lyayuga&amp;publication_year=2023&amp;doi=10.1101/2023.05.08.539766&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="mixed-citation">Krishna R.
et al. 
<span class="ref-title">Generalized biomolecular modeling and design with RoseTTAFold All-Atom</span>. <span class="ref-journal">Science</span>
<span class="ref-vol">0</span>, eadl2528, doi: 10.1126/science.adl2528. [<a href="https://pubmed.ncbi.nlm.nih.gov/38452047" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.adl2528" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Generalized+biomolecular+modeling+and+design+with+RoseTTAFold+All-Atom&amp;author=R.+Krishna&amp;volume=0&amp;pages=eadl2528&amp;doi=10.1126/science.adl2528&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="mixed-citation">Jumper J.
et al. 
<span class="ref-title">Highly accurate protein structure prediction with AlphaFold</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">596</span>, 583&#x02013;589, doi: 10.1038/s41586-021-03819-2 (2021).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8371605/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34265844" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41586-021-03819-2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Highly+accurate+protein+structure+prediction+with+AlphaFold&amp;author=J.+Jumper&amp;volume=596&amp;publication_year=2021&amp;pages=583-589&amp;pmid=34265844&amp;doi=10.1038/s41586-021-03819-2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="mixed-citation">De Robertis E. M., Longthorne R. F. &#x00026; Gurdon J. B.
<span class="ref-title">Intracellular migration of nuclear proteins in Xenopus oocytes</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">272</span>, 254&#x02013;256, doi: 10.1038/272254a0 (1978).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/564468" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2F272254a0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Intracellular+migration+of+nuclear+proteins+in+Xenopus+oocytes&amp;author=E.+M.+De+Robertis&amp;author=R.+F.+Longthorne&amp;author=J.+B.+Gurdon&amp;volume=272&amp;publication_year=1978&amp;pages=254-256&amp;pmid=564468&amp;doi=10.1038/272254a0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="mixed-citation">Dingwall C., Sharnick S. V. &#x00026; Laskey R. A.
<span class="ref-title">A polypeptide domain that specifies migration of nucleoplasmin into the nucleus</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">30</span>, 449&#x02013;458, doi: 10.1016/0092-8674(82)90242-2 (1982).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/6814762" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2F0092-8674(82)90242-2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A+polypeptide+domain+that+specifies+migration+of+nucleoplasmin+into+the+nucleus&amp;author=C.+Dingwall&amp;author=S.+V.+Sharnick&amp;author=R.+A.+Laskey&amp;volume=30&amp;publication_year=1982&amp;pages=449-458&amp;pmid=6814762&amp;doi=10.1016/0092-8674(82)90242-2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="mixed-citation">Lu J.
et al. 
<span class="ref-title">Types of nuclear localization signals and mechanisms of protein import into the nucleus</span>. <span class="ref-journal">Cell Communication and Signaling</span>
<span class="ref-vol">19</span>, 60, doi: 10.1186/s12964-021-00741-y (2021).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8140498/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34022911" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2Fs12964-021-00741-y" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Communication+and+Signaling&amp;title=Types+of+nuclear+localization+signals+and+mechanisms+of+protein+import+into+the+nucleus&amp;author=J.+Lu&amp;volume=19&amp;publication_year=2021&amp;pages=60&amp;pmid=34022911&amp;doi=10.1186/s12964-021-00741-y&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="mixed-citation">Kobayashi H., Cheveralls K. C., Leonetti M. D. &#x00026; Royer L. A.
<span class="ref-title">Self-supervised deep learning encodes high-resolution features of protein subcellular localization</span>. <span class="ref-journal">Nature Methods</span>
<span class="ref-vol">19</span>, 995&#x02013;1003, doi: 10.1038/s41592-022-01541-z (2022).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9349041/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35879608" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41592-022-01541-z" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Methods&amp;title=Self-supervised+deep+learning+encodes+high-resolution+features+of+protein+subcellular+localization&amp;author=H.+Kobayashi&amp;author=K.+C.+Cheveralls&amp;author=M.+D.+Leonetti&amp;author=L.+A.+Royer&amp;volume=19&amp;publication_year=2022&amp;pages=995-1003&amp;pmid=35879608&amp;doi=10.1038/s41592-022-01541-z&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="mixed-citation">Jiang Y.
et al. 
<span class="ref-title">MULocDeep: A deep-learning framework for protein subcellular and suborganellar localization prediction with residue-level interpretation</span>. <span class="ref-journal">Computational and Structural Biotechnology Journal</span>
<span class="ref-vol">19</span>, 4825&#x02013;4839, doi: 10.1016/j.csbj.2021.08.027 (2021).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8426535/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34522290" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.csbj.2021.08.027" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Computational+and+Structural+Biotechnology+Journal&amp;title=MULocDeep:+A+deep-learning+framework+for+protein+subcellular+and+suborganellar+localization+prediction+with+residue-level+interpretation&amp;author=Y.+Jiang&amp;volume=19&amp;publication_year=2021&amp;pages=4825-4839&amp;pmid=34522290&amp;doi=10.1016/j.csbj.2021.08.027&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="mixed-citation">Thumuluri V., Almagro Armenteros J. J., Johansen Alexander R., Nielsen H. &#x00026; Winther O.
<span class="ref-title">DeepLoc 2.0: multi-label subcellular localization prediction using protein language models</span>. <span class="ref-journal">Nucleic Acids Research</span>
<span class="ref-vol">50</span>, W228&#x02013;W234, doi: 10.1093/nar/gkac278 (2022).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9252801/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35489069" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkac278" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Research&amp;title=DeepLoc+2.0:+multi-label+subcellular+localization+prediction+using+protein+language+models&amp;author=V.+Thumuluri&amp;author=J.+J.+Almagro+Armenteros&amp;author=Alexander+R.+Johansen&amp;author=H.+Nielsen&amp;author=O.+Winther&amp;volume=50&amp;publication_year=2022&amp;pages=W228-W234&amp;pmid=35489069&amp;doi=10.1093/nar/gkac278&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="mixed-citation">Patil A.
et al. 
<span class="ref-title">A disordered region controls cBAF activity via condensation and partner recruitment</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">186</span>, 4936&#x02013;4955.e4926, doi: 10.1016/j.cell.2023.08.032 (2023).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10792396/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37788668" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cell.2023.08.032" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A+disordered+region+controls+cBAF+activity+via+condensation+and+partner+recruitment&amp;author=A.+Patil&amp;volume=186&amp;publication_year=2023&amp;pages=4936-4955.e4926&amp;pmid=37788668&amp;doi=10.1016/j.cell.2023.08.032&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="mixed-citation">Lyons H.
et al. 
<span class="ref-title">Functional partitioning of transcriptional regulators by patterned charge blocks</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">186</span>, 327&#x02013;345.e328, doi: 10.1016/j.cell.2022.12.013 (2023).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9910284/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36603581" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cell.2022.12.013" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Functional+partitioning+of+transcriptional+regulators+by+patterned+charge+blocks&amp;author=H.+Lyons&amp;volume=186&amp;publication_year=2023&amp;pages=327-345.e328&amp;pmid=36603581&amp;doi=10.1016/j.cell.2022.12.013&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="mixed-citation">King M. R.
et al. 
<span class="ref-title">Macromolecular condensation organizes nucleolar sub-phases to set up a pH gradient</span>. <span class="ref-journal">Cell</span>, doi: 10.1016/j.cell.2024.02.029. [<a href="https://pubmed.ncbi.nlm.nih.gov/38503281" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cell.2024.02.029" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Macromolecular+condensation+organizes+nucleolar+sub-phases+to+set+up+a+pH+gradient&amp;author=M.+R.+King&amp;doi=10.1016/j.cell.2024.02.029&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R52">52. <span class="mixed-citation">Kilgore H. R. &#x00026; Young R. A.
<span class="ref-title">Learning the chemical grammar of biomolecular condensates</span>. <span class="ref-journal">Nat. Chem. Biol.</span>, 1298&#x02013;1306, doi: 10.1038/s41589-022-01046-y (2022).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9691472/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35761089" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41589-022-01046-y" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Chem.+Biol.&amp;title=Learning+the+chemical+grammar+of+biomolecular+condensates&amp;author=H.+R.+Kilgore&amp;author=R.+A.+Young&amp;publication_year=2022&amp;pages=1298-1306&amp;pmid=35761089&amp;doi=10.1038/s41589-022-01046-y&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R53">53. <span class="mixed-citation">Podgornaia A. I. &#x00026; Laub M. T.
<span class="ref-title">Pervasive degeneracy and epistasis in a protein-protein interface</span>. <span class="ref-journal">Science</span>
<span class="ref-vol">347</span>, 673&#x02013;677, doi: 10.1126/science.1257360 (2015).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/25657251" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.1257360" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Pervasive+degeneracy+and+epistasis+in+a+protein-protein+interface&amp;author=A.+I.+Podgornaia&amp;author=M.+T.+Laub&amp;volume=347&amp;publication_year=2015&amp;pages=673-677&amp;pmid=25657251&amp;doi=10.1126/science.1257360&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R54">54. <span class="mixed-citation">Repecka D.
et al. 
<span class="ref-title">Expanding functional protein sequence spaces using generative adversarial networks</span>. <span class="ref-journal">Nature Machine Intelligence</span>
<span class="ref-vol">3</span>, 324&#x02013;333, doi: 10.1038/s42256-021-00310-5 (2021). [<a href="//doi.org/10.1038%2Fs42256-021-00310-5" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Machine+Intelligence&amp;title=Expanding+functional+protein+sequence+spaces+using+generative+adversarial+networks&amp;author=D.+Repecka&amp;volume=3&amp;publication_year=2021&amp;pages=324-333&amp;doi=10.1038/s42256-021-00310-5&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R55">55. <span class="mixed-citation">Andre J. F., Aina M.-A., Cristina H.-C., J&#x000f6;rn M. S. &#x00026; Ben L.
<span class="ref-title">The genetic architecture of protein stability</span>. <span class="ref-journal">bioRxiv</span>, 2023.2010.2027.564339, doi: 10.1101/2023.10.27.564339 (2023). [<a href="//doi.org/10.1101%2F2023.10.27.564339" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=The+genetic+architecture+of+protein+stability&amp;author=J.+F.+Andre&amp;author=M.-A.+Aina&amp;author=H.-C.+Cristina&amp;author=M.+S.+J&#x000f6;rn&amp;author=L.+Ben&amp;publication_year=2023&amp;doi=10.1101/2023.10.27.564339&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R56">56. <span class="mixed-citation">Klein I. A.
et al. 
<span class="ref-title">Partitioning of cancer therapeutics in nuclear condensates</span>. <span class="ref-journal">Science</span>
<span class="ref-vol">368</span>, 1386, doi: 10.1126/science.aaz4427 (2020).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7735713/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32554597" ref="reftype=pubmed&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.aaz4427" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Partitioning+of+cancer+therapeutics+in+nuclear+condensates&amp;author=I.+A.+Klein&amp;volume=368&amp;publication_year=2020&amp;pages=1386&amp;pmid=32554597&amp;doi=10.1126/science.aaz4427&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11042338&amp;issue-id=460946&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">bioRxiv</span> are provided here courtesy of <strong>Cold Spring Harbor Laboratory Preprints</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11042338/pdf/nihpp-2024.04.15.589616v2.pdf" class="int-view">PDF (4.3M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11042338/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11042338/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11042338" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11042338%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="vlkoDW9InUvk3Ux4mPQqnV55wrF7BxQsnNbQGYmrNbQDczrRmcqqVWCTbdnMBMUf">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11042338%2F&amp;text=Protein%20codes%20promote%20selective%20subcellular%20compartmentalization" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11042338%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11042338/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11042338/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38659952/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11042338/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38659952/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11042338/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11042338/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC11042338/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
