
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs2.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Neuron-specific transcriptomic signatures indicate neuroinflammation and altered neuronal activity in ASD temporal cortex - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="322C300B24F392050000328ECEFA6B28.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="pnas" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC10013873/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10013873/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Proceedings of the National Academy of Sciences of the United States of America" /><meta name="citation_title" content="Neuron-specific transcriptomic signatures indicate neuroinflammation and altered neuronal activity in ASD temporal cortex" /><meta name="citation_author" content="Pan Zhang" /><meta name="citation_author_institution" content="aDepartment of Psychiatry and Biobehavioral Sciences, David Geffen School of Medicine, University of California, Los Angeles, CA 90095" /><meta name="citation_author_institution" content="bDepartment of Human Genetics, David Geffen School of Medicine, University of California, Los Angeles, CA 90095" /><meta name="citation_author_institution" content="cSemel Institute for Neuroscience and Human Behavior, University of California, Los Angeles, CA 90095" /><meta name="citation_author" content="Alicja Omanska" /><meta name="citation_author_institution" content="dDepartment of Psychiatry and Behavioral Sciences, School of Medicine, University of California, Davis, Sacramento, CA 95817" /><meta name="citation_author_institution" content="eUniversity of California, Davis, MIND Institute, Sacramento, CA 95817" /><meta name="citation_author" content="Bradley&#x000a0;P. Ander" /><meta name="citation_author_institution" content="eUniversity of California, Davis, MIND Institute, Sacramento, CA 95817" /><meta name="citation_author_institution" content="fDepartment of Neurology, University of California, Davis, School of Medicine, Sacramento, CA 95817" /><meta name="citation_author" content="Michael&#x000a0;J. Gandal" /><meta name="citation_author_institution" content="cSemel Institute for Neuroscience and Human Behavior, University of California, Los Angeles, CA 90095" /><meta name="citation_author_institution" content="gDepartment of Psychiatry, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104" /><meta name="citation_author_institution" content="hDepartment of Genetics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104" /><meta name="citation_author_institution" content="iLifespan Brain Institute, Penn Med and the Children&#x02019;s Hospital of Philadelphia, Philadelphia, PA 19104" /><meta name="citation_author" content="Boryana Stamova" /><meta name="citation_author_institution" content="eUniversity of California, Davis, MIND Institute, Sacramento, CA 95817" /><meta name="citation_author_institution" content="fDepartment of Neurology, University of California, Davis, School of Medicine, Sacramento, CA 95817" /><meta name="citation_author" content="Cynthia&#x000a0;M. Schumann" /><meta name="citation_author_institution" content="dDepartment of Psychiatry and Behavioral Sciences, School of Medicine, University of California, Davis, Sacramento, CA 95817" /><meta name="citation_author_institution" content="eUniversity of California, Davis, MIND Institute, Sacramento, CA 95817" /><meta name="citation_publication_date" content="2023/03/03" /><meta name="citation_issue" content="10" /><meta name="citation_volume" content="120" /><meta name="citation_doi" content="10.1073/pnas.2206758120" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC10013873/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC10013873/" /><meta name="citation_pmid" content="36862688" /><meta name="DC.Title" content="Neuron-specific transcriptomic signatures indicate neuroinflammation and altered neuronal activity in ASD temporal cortex" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="National Academy of Sciences" /><meta name="DC.Contributor" content="Pan Zhang" /><meta name="DC.Contributor" content="Alicja Omanska" /><meta name="DC.Contributor" content="Bradley&#x000a0;P. Ander" /><meta name="DC.Contributor" content="Michael&#x000a0;J. Gandal" /><meta name="DC.Contributor" content="Boryana Stamova" /><meta name="DC.Contributor" content="Cynthia&#x000a0;M. Schumann" /><meta name="DC.Date" content="2023 Mar 7" /><meta name="DC.Identifier" content="10.1073/pnas.2206758120" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Neuron-specific transcriptomic signatures indicate neuroinflammation and altered neuronal activity in ASD temporal cortex" /><meta property="og:type" content="article" /><meta property="og:description" content="We present a comprehensive assessment of neuronal cell-type-specific gene expression and alternative splicing changes in ASD cortex, directly comparing RNA-seq results from bulk tissue with isolated neurons. We observe strong signatures of cell stress ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10013873/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10013873/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/2/">Proc Natl Acad Sci U S A</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/431279/">v.120(10); 2023 Mar 7</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC10013873
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC10013873/pdf/pnas.202206758.pdf" class="int-view">PDF (22M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/10013873/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/10013873/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="10013873" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10013873%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="rHKZUskd5aR9H6HsUnzDFb3MwdraIWHFvGCAoUEI8gMWmboLTORMLW4oj1yFKyxX">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10013873%2F&amp;text=Neuron-specific%20transcriptomic%20signatures%20indicate%20neuroinflammation%20and%20altered%20neuronal%20activity%20in%20ASD%20temporal%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10013873%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10013873/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC10013873/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/36862688/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC10013873/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/36862688/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC10013873/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/10013873/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/2/">Proc Natl Acad Sci U S A</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/431279/">v.120(10); 2023 Mar 7</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC10013873
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><a href="//doi.org/10.1073%2Fpnas.2206758120" title="Link to Publisher's site" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"><img src="/corehtml/pmc/pmcgifs/logo-pnas.png" alt="Logo of pnas"><!-- --></img></a></div>
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Proc Natl Acad Sci U S A.</span> 2023 Mar 7; 120(10): e2206758120. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2023 Mar 2. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1073%2Fpnas.2206758120" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1073/pnas.2206758120</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC10013873</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/36862688">36862688</a></div></div></div><h1 class="content-title">Neuron-specific transcriptomic signatures indicate neuroinflammation and altered neuronal activity in ASD temporal cortex</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Zhang%20P%5BAuthor%5D" class="affpopup" co-rid="_co_idm139624434093344" co-class="co-affbox">Pan Zhang</a>,<sup>
a
,</sup><sup>
b
,</sup><sup>
c
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Omanska%20A%5BAuthor%5D" class="affpopup" co-rid="_co_idm139624434089712" co-class="co-affbox">Alicja Omanska</a>,<sup>
d
,</sup><sup>
e
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ander%20B%5BAuthor%5D" class="affpopup" co-rid="_co_idm139624424576160" co-class="co-affbox">Bradley&#x000a0;P. Ander</a>,<sup>
e
,</sup><sup>
f
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gandal%20M%5BAuthor%5D" class="affpopup" co-rid="_co_idm139624424572672" co-class="co-affbox">Michael&#x000a0;J. Gandal</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>
c
,</sup><sup>
g
,</sup><sup>
h
,</sup><sup>
i
,</sup><sup>
1
,</sup><sup>
2
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Stamova%20B%5BAuthor%5D" class="affpopup" co-rid="_co_idm139624425477408" co-class="co-affbox">Boryana Stamova</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>
e
,</sup><sup>
f
,</sup><sup>
1
,</sup><sup>
2
</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Schumann%20C%5BAuthor%5D" class="affpopup" co-rid="_co_idm139624435169824" co-class="co-affbox">Cynthia&#x000a0;M. Schumann</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>
d
,</sup><sup>
e
,</sup><sup>
1
,</sup><sup>
2
</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139624434093344"><h3 class="no_margin">Pan Zhang</h3><p><sup>a</sup>Department of Psychiatry and Biobehavioral Sciences, David Geffen School of Medicine, University of California, Los Angeles, CA
90095</p><p><sup>b</sup>Department of Human Genetics, David Geffen School of Medicine, University of California, Los Angeles, CA
90095</p><p><sup>c</sup>Semel Institute for Neuroscience and Human Behavior, University of California, Los Angeles, CA
90095</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Zhang%20P%5BAuthor%5D">Pan Zhang</a></div></div><div id="_co_idm139624434089712"><h3 class="no_margin">Alicja Omanska</h3><p><sup>d</sup>Department of Psychiatry and Behavioral Sciences, School of Medicine, University of California, Davis, Sacramento, CA
95817</p><p><sup>e</sup>University of California, Davis, MIND Institute, Sacramento, CA
95817</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Omanska%20A%5BAuthor%5D">Alicja Omanska</a></div></div><div id="_co_idm139624424576160"><h3 class="no_margin">Bradley&#x000a0;P. Ander</h3><p><sup>e</sup>University of California, Davis, MIND Institute, Sacramento, CA
95817</p><p><sup>f</sup>Department of Neurology, University of California, Davis, School of Medicine, Sacramento, CA
95817</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ander%20B%5BAuthor%5D">Bradley&#x000a0;P. Ander</a></div></div><div id="_co_idm139624424572672"><h3 class="no_margin">Michael&#x000a0;J. Gandal</h3><p><sup>c</sup>Semel Institute for Neuroscience and Human Behavior, University of California, Los Angeles, CA
90095</p><p><sup>g</sup>Department of Psychiatry, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA
19104</p><p><sup>h</sup>Department of Genetics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA
19104</p><p><sup>i</sup>Lifespan Brain Institute, Penn Med and the Children&#x02019;s Hospital of Philadelphia, Philadelphia, PA
19104</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gandal%20M%5BAuthor%5D">Michael&#x000a0;J. Gandal</a></div></div><div id="_co_idm139624425477408"><h3 class="no_margin">Boryana Stamova</h3><p><sup>e</sup>University of California, Davis, MIND Institute, Sacramento, CA
95817</p><p><sup>f</sup>Department of Neurology, University of California, Davis, School of Medicine, Sacramento, CA
95817</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Stamova%20B%5BAuthor%5D">Boryana Stamova</a></div></div><div id="_co_idm139624435169824"><h3 class="no_margin">Cynthia&#x000a0;M. Schumann</h3><p><sup>d</sup>Department of Psychiatry and Behavioral Sciences, School of Medicine, University of California, Davis, Sacramento, CA
95817</p><p><sup>e</sup>University of California, Davis, MIND Institute, Sacramento, CA
95817</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Schumann%20C%5BAuthor%5D">Cynthia&#x000a0;M. Schumann</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm139624428473760_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139624428473760_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139624428473760_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm139624428473760_ai" style="display:none"><div class="fm-affl" id="aff1"><sup>a</sup>Department of Psychiatry and Biobehavioral Sciences, David Geffen School of Medicine, University of California, Los Angeles, CA
90095</div><div class="fm-affl" id="aff2"><sup>b</sup>Department of Human Genetics, David Geffen School of Medicine, University of California, Los Angeles, CA
90095</div><div class="fm-affl" id="aff3"><sup>c</sup>Semel Institute for Neuroscience and Human Behavior, University of California, Los Angeles, CA
90095</div><div class="fm-affl" id="aff4"><sup>d</sup>Department of Psychiatry and Behavioral Sciences, School of Medicine, University of California, Davis, Sacramento, CA
95817</div><div class="fm-affl" id="aff5"><sup>e</sup>University of California, Davis, MIND Institute, Sacramento, CA
95817</div><div class="fm-affl" id="aff6"><sup>f</sup>Department of Neurology, University of California, Davis, School of Medicine, Sacramento, CA
95817</div><div class="fm-affl" id="aff7"><sup>g</sup>Department of Psychiatry, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA
19104</div><div class="fm-affl" id="aff8"><sup>h</sup>Department of Genetics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA
19104</div><div class="fm-affl" id="aff9"><sup>i</sup>Lifespan Brain Institute, Penn Med and the Children&#x02019;s Hospital of Philadelphia, Philadelphia, PA
19104</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup>Corresponding author.</div><div><span class="contrib-email" id="contrib-a.y.b.f.d">Michael&#x000a0;J. Gandal: <a href="mailto:dev@null" data-email="ude.nnepu.enicidemnnep@ladnag.leahcim" class="oemail">ude.nnepu.enicidemnnep@ladnag.leahcim</a>; </span> <span class="contrib-email" id="contrib-a.y.b.f.e">Boryana Stamova: <a href="mailto:dev@null" data-email="ude.sivadcu@avomatssb" class="oemail">ude.sivadcu@avomatssb</a>; </span> <span class="contrib-email" id="contrib-a.y.b.f.f">Cynthia&#x000a0;M. Schumann: <a href="mailto:dev@null" data-email="ude.sivadcu@nnamuhcsc" class="oemail">ude.sivadcu@nnamuhcsc</a></span> </div><div id="cor1"><sup>1</sup>To whom correspondence may be addressed. Email: <a href="mailto:dev@null" data-email="ude.nnepu.enicidemnnep@ladnag.leahcim" class="oemail">ude.nnepu.enicidemnnep@ladnag.leahcim</a>, <a href="mailto:dev@null" data-email="ude.sivadcu@avomatssb" class="oemail">ude.sivadcu@avomatssb</a>, or <a href="mailto:dev@null" data-email="ude.sivadcu@nnamuhcsc" class="oemail">ude.sivadcu@nnamuhcsc</a>.</div><div id="fn1">Edited by Huda Akil, University of Michigan-Ann Arbor, Ann Arbor, MI; received July 12, 2022; accepted December 28, 2022</div><div id="fn2"><sup>2</sup>M.J.G., B.S., and C.M.S. contributed equally to this work.</div></div><div class="fm-article-notes hide half_rhythm" id="idm139624428473760_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2022 Jul 12; Accepted 2022 Dec 28.</div></div><div class="permissions half_rhythm hide" id="idm139624428473760_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  &#x000a9; 2023 the Author(s). Published by PNAS.</div><div class="license half_rhythm">This open access article is distributed under <a href="https://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">Creative Commons Attribution License 4.0 (CC BY)</a>.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="15" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Appendix 01 (PDF)</div><div><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sapp.pdf" data-ga-action="click_feat_suppl">pnas.2206758120.sapp.pdf</a><span style="color:gray"> (1.7M)</span></div><div class="small guid">GUID: D7B717AC-F2F7-4831-A26E-E2A3CEA40344</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Dataset S01 (XLSX)</div><div><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd01.xlsx" data-ga-action="click_feat_suppl">pnas.2206758120.sd01.xlsx</a><span style="color:gray"> (2.1M)</span></div><div class="small guid">GUID: 8F5EB02C-F018-4909-ACB4-65BBCB399A7C</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Dataset S02 (XLSX)</div><div><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd02.xlsx" data-ga-action="click_feat_suppl">pnas.2206758120.sd02.xlsx</a><span style="color:gray"> (2.3M)</span></div><div class="small guid">GUID: C4709F7A-8C6A-442E-B7A1-E3D8FC325A1C</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Dataset S03 (XLSX)</div><div><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd03.xlsx" data-ga-action="click_feat_suppl">pnas.2206758120.sd03.xlsx</a><span style="color:gray"> (728K)</span></div><div class="small guid">GUID: 12B60AA9-6E9B-4294-BFB5-F2264EE6FA32</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Dataset S04 (XLSX)</div><div><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd04.xlsx" data-ga-action="click_feat_suppl">pnas.2206758120.sd04.xlsx</a><span style="color:gray"> (361K)</span></div><div class="small guid">GUID: 1363E05D-B1AA-444F-B990-66EC08D51610</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Dataset S05 (XLSX)</div><div><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd05.xlsx" data-ga-action="click_feat_suppl">pnas.2206758120.sd05.xlsx</a><span style="color:gray"> (14M)</span></div><div class="small guid">GUID: DBAFE376-975D-44F6-9AB4-4C200FD7EFC8</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Dataset S06 (XLSX)</div><div><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd06.xlsx" data-ga-action="click_feat_suppl">pnas.2206758120.sd06.xlsx</a><span style="color:gray"> (1.3M)</span></div><div class="small guid">GUID: DC0C0D2A-B9B6-468E-A89B-DB3046EE55AE</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Dataset S07 (XLSX)</div><div><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd07.xlsx" data-ga-action="click_feat_suppl">pnas.2206758120.sd07.xlsx</a><span style="color:gray"> (1.3M)</span></div><div class="small guid">GUID: 5539B9C6-B332-44EC-8556-45A1810C6E8A</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Dataset S08 (XLSX)</div><div><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd08.xlsx" data-ga-action="click_feat_suppl">pnas.2206758120.sd08.xlsx</a><span style="color:gray"> (6.6M)</span></div><div class="small guid">GUID: F16F8063-4C73-434C-97ED-38D7640CC448</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Dataset S09 (XLSX)</div><div><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd09.xlsx" data-ga-action="click_feat_suppl">pnas.2206758120.sd09.xlsx</a><span style="color:gray"> (414K)</span></div><div class="small guid">GUID: 0A14F4C9-116F-432A-A16C-06A170A6D983</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Dataset S010 (XLSX)</div><div><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd10.xlsx" data-ga-action="click_feat_suppl">pnas.2206758120.sd10.xlsx</a><span style="color:gray"> (156K)</span></div><div class="small guid">GUID: 5C31C2F3-0195-4AC0-9542-A50A462E6808</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Dataset S011 (XLSX)</div><div><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd11.xlsx" data-ga-action="click_feat_suppl">pnas.2206758120.sd11.xlsx</a><span style="color:gray"> (168K)</span></div><div class="small guid">GUID: E19F1BC3-4D70-4B51-92FB-9412F51677B0</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Dataset S012 (XLSX)</div><div><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd12.xlsx" data-ga-action="click_feat_suppl">pnas.2206758120.sd12.xlsx</a><span style="color:gray"> (15K)</span></div><div class="small guid">GUID: 92D0517E-B857-477B-B8B2-CF8840680A84</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Dataset S013 (XLSX)</div><div><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd13.xlsx" data-ga-action="click_feat_suppl">pnas.2206758120.sd13.xlsx</a><span style="color:gray"> (55K)</span></div><div class="small guid">GUID: 84506EFE-DF63-4DED-AE01-0714B884EED3</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Dataset S014 (XLSX)</div><div><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd14.xlsx" data-ga-action="click_feat_suppl">pnas.2206758120.sd14.xlsx</a><span style="color:gray"> (149K)</span></div><div class="small guid">GUID: FF6699B9-0D43-4E16-9CF5-FBCCD3D18D0C</div></div></dd></dl><dl data-length="245" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first-last">All custom code used in this manuscript is available at <a href="https://github.com/gandallab/ASD_STG_LCM_RNAseq" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/gandallab/ASD_STG_LCM_RNAseq</a> (<a href="#r85" rid="r85" class=" bibr popnode">85</a>). RNA-seq data from this study is available through the database for Genotypes and Phenotypes (dbGaP Study Accession ID: <a href="https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs003208.v1.p1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">phs003208</a>) (<a href="#r86" rid="r86" class=" bibr popnode">86</a>).</p></dd></dl></div><div id="abstract-a.y.b.q" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.y.b.qtitle" style="text-transform: uppercase;">Significance</h2><!--article-meta--><div><p class="p p-first-last">We present a comprehensive assessment of neuronal cell-type-specific gene expression and alternative splicing changes in ASD cortex, directly comparing RNA-seq results from bulk tissue with isolated neurons. We observe strong signatures of cell stress and neural-immune/inflammatory pathway activation present within ASD neurons&#x02014;a signal that is typically attributed to astrocyte/microglial populations. Our findings also provide further evidence for the hypothesized imbalance of excitatory to inhibitory neuronal activity in the brains of individuals with ASD. Moreover, we find that the transcriptomic architecture of ASD interacts substantially with age, thus revealing windows of opportunity for treatments that target specific molecular pathology.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">ASD, transcriptome, neuron-specific</span></div></div><div id="abstract-a.y.b.r" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.y.b.rtitle" style="text-transform: uppercase;">Abstract</h2><!--article-meta--><div><p class="p p-first-last">Autism spectrum disorder (ASD) is a highly heterogeneous disorder, yet transcriptomic profiling of bulk brain tissue has identified substantial convergence among dysregulated genes and pathways in ASD. However, this approach lacks cell-specific resolution. We performed comprehensive transcriptomic analyses on bulk tissue and laser-capture microdissected (LCM) neurons from 59 postmortem human brains (27 ASD and 32 controls) in the superior temporal gyrus (STG) of individuals ranging from 2 to 73 years of age. In bulk tissue, synaptic signaling, heat shock protein-related pathways, and RNA splicing were significantly altered in ASD. There was age-dependent dysregulation of genes involved in gamma aminobutyric acid (GABA) (<em>GAD1</em> and <em>GAD2</em>) and glutamate (<em>SLC38A1</em>) signaling pathways. In LCM neurons, AP-1-mediated neuroinflammation and insulin/IGF-1 signaling pathways were upregulated in ASD, while mitochondrial function, ribosome, and spliceosome components were downregulated. GABA synthesizing enzymes <em>GAD1</em> and <em>GAD2</em> were both downregulated in ASD neurons. Mechanistic modeling suggested a direct link between inflammation and ASD in neurons, and prioritized inflammation-associated genes for future study. Alterations in small nucleolar RNAs (snoRNAs) associated with splicing events suggested interplay between snoRNA dysregulation and splicing disruption in neurons of individuals with ASD. Our findings supported the fundamental hypothesis of altered neuronal communication in ASD, demonstrated that inflammation was elevated at least in part in ASD neurons, and may reveal windows of opportunity for biotherapeutics to target the trajectory of gene expression and clinical manifestation of ASD throughout the human lifespan.</p></div></div><div id="body-a.z" class="tsec sec"><h2 class="headless nomenu"></h2><p class="p p-first">Autism spectrum disorder (ASD) defines a heterogeneous set of complex neurodevelopmental disorders affecting 1 in 54 children in the USA according to current estimation (<a href="#r1" rid="r1" class=" bibr popnode">1</a>, <a href="#r2" rid="r2" class=" bibr popnode">2</a>) and confers lifelong challenges. ASD is characterized by difficulties with social communication as well as a repetitive, restricted repertoire of behaviors and interests (<a href="#r3" rid="r3" class=" bibr popnode">3</a>). Population, family, and twin studies all indicate a strong genetic component contributing to risk for ASDs (<a href="#r4" rid="r4" class=" bibr popnode">4</a>, <a href="#r5" rid="r5" class=" bibr popnode">5</a>), with heritability estimates of ~70% (<a href="#r6" rid="r6" class=" bibr popnode">6</a>). However, the genetic causes and pathophysiology of ASD are varied and often complex.</p><p>Despite this heterogeneity, transcriptomic analyses of postmortem human brain have elucidated substantial convergent molecular-level pathology associated with idiopathic and syndromic forms of ASD (<a href="#r7" rid="r7" class=" bibr popnode">7</a><a href="#r8" rid="r8" class=" bibr popnode"></a><a href="#r9" rid="r9" class=" bibr popnode"></a><a href="#r10" rid="r10" class=" bibr popnode"></a><a href="#r11" rid="r11" class=" bibr popnode"></a><a href="#r12" rid="r12" class=" bibr popnode"></a>&#x02013;<a href="#r13" rid="r13" class=" bibr popnode">13</a>). Multiple studies have profiled the transcriptomes of postmortem brain regions from individuals diagnosed with ASD (<a href="#r7" rid="r7" class=" bibr popnode">7</a>, <a href="#r8" rid="r8" class=" bibr popnode">8</a>, <a href="#r11" rid="r11" class=" bibr popnode">11</a>, <a href="#r13" rid="r13" class=" bibr popnode">13</a>), including the temporal cortex implicated due to its critical importance in speech and language function (<a href="#r7" rid="r7" class=" bibr popnode">7</a>, <a href="#r8" rid="r8" class=" bibr popnode">8</a>, <a href="#r11" rid="r11" class=" bibr popnode">11</a>, <a href="#r13" rid="r13" class=" bibr popnode">13</a>). The most consistent findings include disruption of neuronal/synaptic activity and activation of innate immunity/glial markers (<a href="#r7" rid="r7" class=" bibr popnode">7</a>, <a href="#r8" rid="r8" class=" bibr popnode">8</a>, <a href="#r11" rid="r11" class=" bibr popnode">11</a>). Alternative splicing and non-coding RNAs have also been shown to be dysregulated in ASD brains (<a href="#r8" rid="r8" class=" bibr popnode">8</a>).</p><p>Most previous transcriptomic studies, however, profiled homogenate brain tissue and have therefore been unable to pinpoint the underlying specific cell types in which gene expression is altered. Single-nucleus RNA-sequencing (sn-RNAseq) datasets have started to be generate in postmortem ASD cortex (<a href="#r12" rid="r12" class=" bibr popnode">12</a>, <a href="#r13" rid="r13" class=" bibr popnode">13</a>), identifying substantial changes in upper-layer excitatory neurons and microglia, consistent with observations from bulk tissue. As such sn-RNAseq datasets currently primarily profile the 3&#x02032;&#x000a0;end of highly expressed genes within each cell, these data characterize neither lowly expressed coding and noncoding genes, nor splicing alterations that may contribute to altered neuronal function in ASD.</p><p class="p">Here, we performed the systematic study using transcriptomic profiling to directly compare both bulk cortical tissue and laser capture microdissected (LCM) neurons from anatomically well-defined superior temporal gyrus (STG) samples from 59 subjects (27 with ASD and 32 controls) ranging from 2 to 73 years of age (<a href="/pmc/articles/PMC10013873/figure/fig01/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig01" rid-ob="ob-fig01" co-legend-rid="lgnd_fig01"><span>Fig. 1</span></a>&#x000a0;and <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S12</a>). The STG modulates language processing and social perception, thereby playing a critical role in integrating a breadth of information to provide meaning to the surrounding world (<a href="#r14" rid="r14" class=" bibr popnode">14</a>). Structural and functional imaging studies have long implicated STG in ASD (<a href="#r14" rid="r14" class=" bibr popnode">14</a>, <a href="#r15" rid="r15" class=" bibr popnode">15</a>); however, molecular-level changes in neurons remain unknown. This study aimed to identify neuron-specific transcriptomic changes in ASD brain by identifying differentially expressed genes, differential splicing (DS) events, age-related gene expression changes across the lifespan, and co-expression networks to reveal gene modules altered in ASD. We also applied mechanistic modeling approaches to pinpoint pathways and genes directly linked to ASD in neurons. (<a href="/pmc/articles/PMC10013873/figure/fig01/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig01" rid-ob="ob-fig01" co-legend-rid="lgnd_fig01"><span>Fig. 1</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig01" co-legend-rid="lgnd_fig01"><a href="/pmc/articles/PMC10013873/figure/fig01/" target="figure" rid-figpopup="fig01" rid-ob="ob-fig01"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139624421743856"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10013873_pnas.2206758120fig01.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pnas.2206758120fig01.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10013873/bin/pnas.2206758120fig01.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139624421743856"><a target="object" rel="noopener" href="/pmc/articles/PMC10013873/figure/fig01/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig01"><div><a class="figpopup" href="/pmc/articles/PMC10013873/figure/fig01/" target="figure" rid-figpopup="fig01" rid-ob="ob-fig01">Fig. 1.</a></div><!--caption a7--><div class="caption"><p>Overview of Experiment Design, Data Analysis Pipeline (<em>A</em>), and Study Cohort (<em>B</em>).</p></div></div></div></div><div id="s1" class="tsec sec"><h2 class="head no_bottom_margin" id="s1title" style="text-transform: uppercase;">Results</h2><div id="s2" class="sec sec-first"><p></p><h3 id="s2title" class="inline">Global Gene Expression Changes in ASD STG. </h3><p class="p p-first">RNA sequencing was performed on bulk tissue STG of 59 human brains, 27 from individuals with ASD and 32 from neurotypical controls, ranging from 2 to 73 years of age. Following quality control, we comprehensively characterized differential gene expression (DGE) and local splicing alterations in ASD. After adjusting for known covariates and correcting for multiple comparisons, we found 194 differentially expressed genes between individuals with ASD and controls (FDR &#x0003c; 0.05). Of these, 143 were upregulated and 51 were downregulated (<a href="/pmc/articles/PMC10013873/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2<em>A</em></span></a>&#x000a0;and <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S1</a>), with a median absolute fold change of 1.45 (range 1.11 to 4.04, <a href="/pmc/articles/PMC10013873/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2<em>A</em></span></a>). We observed significant concordance between our DGE results and previous data of the same region from different samples (<a href="#r13" rid="r13" class=" bibr popnode">13</a>) (<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI&#x000a0;Appendix</em>, Fig. S1</a>, Spearman &#x003c1; = 0.37 for t statistics among all genes, <em>P</em> value &#x0003c; 10<sup>&#x02212;16</sup>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig02" co-legend-rid="lgnd_fig02"><a href="/pmc/articles/PMC10013873/figure/fig02/" target="figure" rid-figpopup="fig02" rid-ob="ob-fig02"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139624430080528"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10013873_pnas.2206758120fig02.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pnas.2206758120fig02.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10013873/bin/pnas.2206758120fig02.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139624430080528"><a target="object" rel="noopener" href="/pmc/articles/PMC10013873/figure/fig02/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig02"><div><a class="figpopup" href="/pmc/articles/PMC10013873/figure/fig02/" target="figure" rid-figpopup="fig02" rid-ob="ob-fig02">Fig. 2.</a></div><!--caption a7--><div class="caption"><p>Transcriptomic Difference Between ASD Cases and Controls in Bulk Tissue STG. (<em>A</em>) Distribution of fold-change of differential expression for 194 differentially expressed genes. Case/control fold-changes for upregulated genes are plotted in gold (N = 143, positive values) and control/case fold-changes for downregulated genes in blue (N = 51, negative values). (<em>B</em>) Co-expressed gene modules that were significantly disrupted in ASD. Modules were hierarchically clustered by module eigengene. Module-diagnosis associations were shown on the right of each module (*FDR &#x0003c; 0.05). Additional enrichment analyses were also shown for each module, including: Enrichment for ASD GWAS common variants (33)(*FDR &#x0003c; 0.1); Enrichment for major CNS cell types (<a href="#r16" rid="r16" class=" bibr popnode">16</a>)(*FDR &#x0003c; 0.05); Enrichment against literature-curated gene lists (*FDR &#x0003c; 0.05, see <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI&#x000a0;Appendix</em>, <em>SI Methods</em></a> for details of the gene lists). Per, pericytes; OPC, oligodendrocyte progenitor cells; InNeu, inhibitory neuron; ExcNeu, excitatory neuron; Oligo, oligodendrocytes; Endo, endothelial cells; Astro, astrocytes. (<em>C</em>) Volcano plot showing significantly up- (gold) and down-regulated (blue) genes (FDR &#x0003c; 0.05). Genes discussed in the main text are colored red. (<em>D</em>) Functional enrichment of differentially expressed genes in ASD cases compared to controls. Top 10 significantly enriched up- and down-regulated categories were shown. The color of each dot reflects FDR-corrected q-value, and the size of each dot reflects the number of overlapped genes between our gene list and the corresponding GO category. NES, normalized enrichment score. (<em>E</em>) Age trajectory of GAD1 (<em>Left</em>) and GAD2 (<em>Right</em>) gene expression, stratified by ASD diagnosis. The expression levels were on the log2CPM (counts per million) scale. (<em>F</em>) Quantile-quantile plot of observed <em>P</em>-values vs. expected <em>P</em>-values for differentially spliced intron clusters. Significant DS events (FDR &#x0003c; 0.05) were colored red, and overlapping gene names were labeled for top clusters.</p></div></div></div><p>Functional and pathway enrichment analyses indicated an over-representation of heat-shock proteins (HSPs) and HSP-related chaperones, which were upregulated in ASD subjects. This included HSP70 family members <em>HSPA1A</em> and <em>HSPA1B</em>; HSP40 family members <em>DNAJB1</em> and <em>DNAJB4</em>; small HSP20 family members <em>HSPB1</em> and <em>HSPB8</em>; and HSP-binding chaperons <em>BAG3</em> and <em>PTGES3</em> (<a href="/pmc/articles/PMC10013873/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2 <em>C</em> and <em>D</em></span></a>). HSPs are involved in stress response, immune activation, and inflammation(<a href="#r17" rid="r17" class=" bibr popnode">17</a>, <a href="#r18" rid="r18" class=" bibr popnode">18</a>), all of which were upregulated in ASD postmortem brain (<a href="#r7" rid="r7" class=" bibr popnode">7</a>). Downregulated genes were mainly enriched in pathways related to synaptic function (<a href="/pmc/articles/PMC10013873/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2<em>D</em></span></a>), consistent with previous findings (<a href="#r7" rid="r7" class=" bibr popnode">7</a>). Notably, two important voltage-gated potassium channel-related genes <em>KCNH3</em> and <em>KCNIP1</em> were among the most downregulated (<a href="/pmc/articles/PMC10013873/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2<em>C</em></span></a>), which may relate to disrupted neuronal excitability hypothesized in ASD (<a href="#r19" rid="r19" class=" bibr popnode">19</a>, <a href="#r20" rid="r20" class=" bibr popnode">20</a>).</p><p>As age-dependent expression alterations have been reported in ASD brain (<a href="#r21" rid="r21" class=" bibr popnode">21</a>), we employed an analytical model accounting for age and the interaction between age and diagnosis. Fourteen genes showed age-dependent DGE between ASD and control (<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S2</a>). Gene set enrichment analysis (GSEA) indicated that genes with significant diagnosis-by-age interaction were enriched in immune/inflammation pathways and synaptic-related pathways. (<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI&#x000a0;Appendix</em>, Fig. S9</a>).</p><p>Interestingly, genes involved in gamma aminobutyric acid (GABA) synthesis (<em>GAD1</em> and <em>GAD2</em>) (<a href="#r22" rid="r22" class=" bibr popnode">22</a>) were downregulated in ASD only during late adulthood (<a href="/pmc/articles/PMC10013873/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2<em>E</em></span></a>). This may indicate an age-dependent dysregulation of GABA signaling in ASD neurons, or a decrease in the proportion of GABAergic neurons in ASD brains (<a href="#r23" rid="r23" class=" bibr popnode">23</a>).</p><p>Given mounting evidence suggesting potential mechanistic overlap between neurodevelopmental and neurodegenerative disorders, particularly with respect to genes involved in synapse and brain connectivity (<a href="#r24" rid="r24" class=" bibr popnode">24</a>), we evaluated the overlap between our ASD findings with published results in Alzheimer&#x02019;s disease (AD). Using results from Mount Sinai/JJ Peters VA Medical Center Brain Bank (MSBB) cohort (<a href="#r25" rid="r25" class=" bibr popnode">25</a>), we demonstrated that the changes we detected in ASD were significantly overlapped with transcriptomic alterations observed in AD from the same brain region (Odds Ratio = 1.6, <em>P</em> = 1.9 &#x000d7;&#x000a0;10<sup>&#x02212;8</sup>, <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI&#x000a0;Appendix</em>, Fig. S8</a>).</p><p>DS events in the bulk tissue transcriptome were evaluated using LeafCutter (<a href="#r26" rid="r26" class=" bibr popnode">26</a>). Among 35,505 intron clusters identified by LeafCutter, 308 clusters (297 unique genes) showed significant DS between ASD patients and controls (FDR &#x0003c; 0.05). The 297 genes did not show significant functional enrichment (<a href="/pmc/articles/PMC10013873/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2<em>F</em></span></a>&#x000a0;and <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S5</a>).</p><p>To place subtle changes across the ASD STG transcriptome into a systems-level context, we performed weighted gene correlation network analysis (WGCNA) to build gene co-expression networks (<a href="#r27" rid="r27" class=" bibr popnode">27</a>), identifying 31 modules of co-expressed genes (<em>Methods</em>&#x000a0;and <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S3</a>). Seven modules significantly associated with ASD diagnosis, two of which were strongly enriched for ASD-associated genetic risk factors (Modules Block-M1 and Block-M10, <a href="/pmc/articles/PMC10013873/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2<em>B</em></span></a>).</p><p>Module Block-M1 was upregulated in ASD STG and its gene members were enriched in RNA splicing and mRNA metabolic pathways (<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S4</a>). Notably, significantly upregulated HSPs were also members of the Block-M1 module (<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S3</a>). HSPs contribute to RNA splicing during stress (<a href="#r28" rid="r28" class=" bibr popnode">28</a>). Downregulated modules in ASD were mostly enriched for synaptic functions (Block-M3, Block-M10, Block-M14, Block-M19, Block-M31; <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S4</a>). Cell-type enrichment analysis also indicated these downregulated modules were enriched in marker genes for both excitatory and inhibitory neurons (<a href="/pmc/articles/PMC10013873/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2<em>B</em></span></a>), suggesting a broad disruption of neuronal and synaptic processes in ASD STG.</p><p class="p p-last">Genes in the upregulated module Block-M1 and one downregulated module (Block-M10) were enriched in high-confidence ASD risk loci (<a href="#r29" rid="r29" class=" bibr popnode">29</a>, <a href="#r30" rid="r30" class=" bibr popnode">30</a>), mutationally constrained (<a href="#r31" rid="r31" class=" bibr popnode">31</a>), and highly intolerant to mutations (pLI &#x0003e; 0.99) genes (<a href="#r32" rid="r32" class=" bibr popnode">32</a>), as well as regulatory target genes of CHD8, which has clear links to at least a subset of ASD cases (<a href="#r33" rid="r33" class=" bibr popnode">33</a>) (<a href="/pmc/articles/PMC10013873/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2<em>B</em></span></a>, see <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI&#x000a0;Appendix</em>, <em>SI Methods</em></a>&#x000a0;for the details of all curated, hypothesis-driven gene sets. <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S14</a>). Many hub genes for module Block-M10 encoded synaptic proteins (<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI&#x000a0;Appendix</em>,&#x000a0;Fig. S2</a>&#x000a0;and <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S3</a>). This module was also enriched for ASD common risk alleles from ASD GWAS data (<a href="#r34" rid="r34" class=" bibr popnode">34</a>). Together this suggested a causal role of synaptic dysfunction in ASD etiology.</p></div><div id="s3" class="sec"><p></p><h3 id="s3title" class="inline">Neuron-Specific Gene Expression and Splicing Alterations in ASD STG. </h3><p class="p p-first">To provide cell-type specificity for the observed transcriptomic changes, we performed laser capture microdissection to capture neurons using STG sections taken from the same subjects profiled using bulk RNA-seq (<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI&#x000a0;Appendix</em>, Figs.&#x000a0;S6 and S7</a>). We then interrogated ASD-associated gene expression and splicing alterations using the same bioinformatic pipelines as above. Across 13,458 neuron-expressed genes, 83 were significantly differentially expressed between ASD subjects and controls at FDR &#x0003c;0.05, of which 52 were upregulated and 31 downregulated (<a href="/pmc/articles/PMC10013873/figure/fig03/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig03" rid-ob="ob-fig03" co-legend-rid="lgnd_fig03"><span>Fig. 3<em>A</em></span></a>&#x000a0;and <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S6</a>). Median absolute fold change in expression between subjects with ASD and controls was 2.48 (range 1.29 to 9.72; <a href="/pmc/articles/PMC10013873/figure/fig03/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig03" rid-ob="ob-fig03" co-legend-rid="lgnd_fig03"><span>Fig. 3<em>A</em></span></a>). The concordance of neuronal DGE with bulk tissue DGE was low (Spearman &#x003c1; = 0.18 for t statistics across all genes, <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI&#x000a0;Appendix</em>, Fig. S3</a>), suggesting our analysis captured ASD signatures unique to neurons.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig03" co-legend-rid="lgnd_fig03"><a href="/pmc/articles/PMC10013873/figure/fig03/" target="figure" rid-figpopup="fig03" rid-ob="ob-fig03"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139624429473328"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10013873_pnas.2206758120fig03.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pnas.2206758120fig03.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10013873/bin/pnas.2206758120fig03.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139624429473328"><a target="object" rel="noopener" href="/pmc/articles/PMC10013873/figure/fig03/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig03"><div><a class="figpopup" href="/pmc/articles/PMC10013873/figure/fig03/" target="figure" rid-figpopup="fig03" rid-ob="ob-fig03">Fig. 3.</a></div><!--caption a7--><div class="caption"><p>DGE and DS Between ASD Cases and Controls in LCM Neurons from STG. (<em>A</em>) Distribution of fold-change of differential expression for 83 differentially expressed genes. Case/control fold-changes for upregulated genes are plotted in gold (N = 52, positive values) and control/case fold-changes for downregulated genes in blue (N = 31, negative values). (<em>B</em>) Age trajectory of HTRA2 gene expression, stratified by ASD diagnosis. The expression levels are on the log<sub>2</sub>CPM (counts per million) scale. (<em>C</em>) Quantile&#x02013;quantile plot of observed <em>P</em>-values vs. expected <em>P</em>-values for differentially spliced intron clusters. Significant DS events (FDR &#x0003c; 0.05) were colored red, and overlapping gene names were labeled for top clusters. DS results from bulk tissue were also plotted for comparison. (<em>D</em>) Volcano plot showing up- (gold) and down-regulated (blue) genes. Genes discussed in the main text are colored red. (<em>E</em>) Functional enrichment of differentially expressed genes in ASD cases compared to controls. Top 10 significantly enriched up- and downregulated categories are shown. The color of each dot reflects FDR-corrected q-value, and the size of each dot reflects the number of overlapped genes between our gene list and the corresponding GO category. NES, normalized enrichment score<strong>.</strong></p></div></div></div><p>Upregulated genes in ASD neurons were highly enriched in pathways related to growth and differentiation (<a href="/pmc/articles/PMC10013873/figure/fig03/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig03" rid-ob="ob-fig03" co-legend-rid="lgnd_fig03"><span>Fig. 3<em>E</em></span></a>). Specifically, the AP-1 transcription factor complex components (<a href="#r35" rid="r35" class=" bibr popnode">35</a>) <em>FOS</em>, <em>JUN</em> and <em>JUNB</em> were all upregulated in ASD neurons, among other growth/differentiation regulators such as <em>SOX9</em>, <em>S1PR1</em>, and <em>PPP1R16B</em> (<a href="/pmc/articles/PMC10013873/figure/fig03/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig03" rid-ob="ob-fig03" co-legend-rid="lgnd_fig03"><span>Fig. 3<em>D</em></span></a>). The AP-1 transcription factor complex is known to regulate many downstream biological processes (<a href="#r35" rid="r35" class=" bibr popnode">35</a>). Indeed, we found upregulated B cell signaling adaptor gene <em>BCL10</em> and NF&#x003ba;B inhibitor delta gene <em>NFKBID</em> in ASD neurons (<a href="/pmc/articles/PMC10013873/figure/fig03/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig03" rid-ob="ob-fig03" co-legend-rid="lgnd_fig03"><span>Fig. 3<em>D</em></span></a>). Up-regulation of both <em>NFKBID</em> and AP-1 point to dysregulated inflammation in ASD neurons. In addition, the inward-rectifier potassium ion channel gene <em>KCNJ2</em> was also upregulated in ASD neurons (<a href="/pmc/articles/PMC10013873/figure/fig03/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig03" rid-ob="ob-fig03" co-legend-rid="lgnd_fig03"><span>Fig. 3<em>D</em></span></a>). Interestingly, both <em>KCNJ2</em> and AP-1 subunit <em>FOS</em> were involved in regulating excitability and plasticity at the cholinergic synapse (<a href="#r36" rid="r36" class=" bibr popnode">36</a><a href="#r37" rid="r37" class=" bibr popnode"></a>&#x02013;<a href="#r38" rid="r38" class=" bibr popnode">38</a>).</p><p>Downregulated genes in ASD neurons are primarily enriched in mitochondrial function and oxidoreductase activity (<a href="/pmc/articles/PMC10013873/figure/fig03/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig03" rid-ob="ob-fig03" co-legend-rid="lgnd_fig03"><span>Fig. 3<em>E</em></span></a>). Specifically, comparing to bulk tissue STG, more subunits of the NADH:ubiquinone oxidoreductase (complex I) were downregulated in neurons, and their effect sizes were larger (<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI&#x000a0;Appendix</em>, Fig. S4</a>). Our results provide evidence that compared to bulk tissue, mitochondrial dysfunction is much more profound in STG neurons.</p><p>While LCM captured both excitatory and inhibitory neurons, we note that <em>GAD1</em> and <em>GAD2</em> genes are among the most downregulated in ASD neurons (<a href="/pmc/articles/PMC10013873/figure/fig03/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig03" rid-ob="ob-fig03" co-legend-rid="lgnd_fig03"><span>Fig. 3<em>D</em></span></a>). The coordinated down-regulation of both GABA synthesizing enzymes suggest that the level of GABA neurotransmitter may be decreased in ASD neurons, providing support to the excitation to inhibition (E/I) imbalance hypothesis of ASD (<a href="#r19" rid="r19" class=" bibr popnode">19</a>, <a href="#r20" rid="r20" class=" bibr popnode">20</a>, <a href="#r39" rid="r39" class=" bibr popnode">39</a>).</p><p>By testing the interaction between diagnosis and age, three genes (<em>HTRA2,</em> aka <em>OMI</em>; <em>ZNF765</em>; and <em>PCDHB18P</em>) showed age-dependent differential expression in ASD neurons (<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S7</a>; no pathway enrichment was detected for neuronal genes with significant diagnosis-by-age interaction). For example, age trajectories of serine peptidase <em>HTRA2</em> were opposite in ASD brains compared to controls (<a href="/pmc/articles/PMC10013873/figure/fig03/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig03" rid-ob="ob-fig03" co-legend-rid="lgnd_fig03"><span>Fig. 3<em>B</em></span></a>). In healthy brains, the expression of <em>HTRA2</em> was much higher before age 30 and decreases with age, while its expression levels begin lower and increase with age in ASD STG neurons.</p><p>To further evaluate potential overlap between ASD and neurodegenerative processes in neurons, we compared our neuronal findings with results from MSBB. Changes in ASD neuronal transcriptome were also significantly overlapped with transcriptomic alterations observed in AD from the same brain region (odds ratio = 1.4, <em>P</em> = 0.01, <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI&#x000a0;Appendix</em>, Fig. S8</a>).</p><p class="p p-last">We also quantified local splicing events in the neuronal transcriptome. After adjusting for multiple testing, LeafCutter identified 1,292 significant differential spliced intron clusters (1,177 unique genes) out of 17,250 total intron clusters at FDR &#x0003c; 0.05 (<a href="/pmc/articles/PMC10013873/figure/fig03/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig03" rid-ob="ob-fig03" co-legend-rid="lgnd_fig03"><span>Fig. 3<em>C</em></span></a>&#x000a0;and <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S8</a>). No functional enrichment was observed for the 1,177 genes. We observed more disruptions in local splicing events in ASD neurons than in bulk tissue (308 DS out of 35,505 events in bulk tissue, 1,292 DS out of 17,250 events in neurons; <em>P</em> &#x0003c; 2 &#x000d7;&#x000a0;10<sup>&#x02212;16</sup> test of proportions).</p></div><div id="s4" class="sec"><p></p><h3 id="s4title" class="inline">Neuron-Specific Networks Pinpoint Subtle Changes in the Neuronal Transcriptome in ASD. </h3><p class="p p-first">Co-expression network analysis on neuronal data identified 18 modules, each containing between 101 and 998 co-expressed genes (<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S9</a>). Four modules were significantly upregulated in ASD neurons, while one module was downregulated (<a href="/pmc/articles/PMC10013873/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>A</em></span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig04" co-legend-rid="lgnd_fig04"><a href="/pmc/articles/PMC10013873/figure/fig04/" target="figure" rid-figpopup="fig04" rid-ob="ob-fig04"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139624468672080"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10013873_pnas.2206758120fig04.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pnas.2206758120fig04.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10013873/bin/pnas.2206758120fig04.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139624468672080"><a target="object" rel="noopener" href="/pmc/articles/PMC10013873/figure/fig04/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig04"><div><a class="figpopup" href="/pmc/articles/PMC10013873/figure/fig04/" target="figure" rid-figpopup="fig04" rid-ob="ob-fig04">Fig. 4.</a></div><!--caption a7--><div class="caption"><p>Co-expression and Mechanistic Network Analysis of the ASD Neuronal Transcriptome. (<em>A</em>) Hierarchical clustering of neuronal gene co-expression modules by module eigengenes. Module&#x02013;diagnosis associations were shown below each module. Enrichment for ASD GWAS common variants is shown for each module. Enrichment against literature-curated gene lists is shown on the bottom. (<em>B</em>) Module enrichment for neuron subtypes. Expression profiles of neuron subtypes were obtained from ref. <a href="#r16" rid="r16" class=" bibr popnode">16</a>. Red asterisks indicate significant enrichment. (<em>C</em>&#x02013;<em>F</em>) Functional enrichment (<em>Top</em>) and top 50 hub genes (<em>Bottom</em>) for module Neu-M5 (<em>C</em>), Neu-M6 (<em>D</em>), Neu-M16 (<em>E</em>) and Neu-M17 (<em>F</em>). Edges represent co-expression (Pearson correlation &#x0003e; 0.5). Co-expressed partners with evidence of protein-protein interaction (PPI) were connected by solid black lines. PPI data was compiled from well-characterized PPI databases (<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI&#x000a0;Appendix</em>, <em>SI Methods</em></a>). (<em>G</em>) Bayesian network representing the relationships among modules (ellipse) and ASD diagnosis (diamond). The thickness of the arrow is proportional to the number of times that a connection was detected during a bootstrap of 500 times. (<em>H</em>) The estimated gene regulatory network (Bayesian network) for 21 hub genes in Neu-M5. (<em>I</em>) Dopamine network built by IPA from neuron DEGs (FDR &#x0003c; 0.1, 225 genes). A network of 17 upstream regulators (orange diamonds) were predicted to drive the changes of 74 observed target genes (23 of which were shown based on node connectivity &#x02265;&#x000a0;2, ellipse). ELK1 was differentially expressed in neurons, and it was also predicted as an upstream regulator.</p></div></div></div><p>The upregulated neu-M5 co-expression module was highly represented by the DGE analysis signal. Upregulated genes <em>JUN, JUNB, and NFKBID</em> were all hub genes of neu-M5 module. Neu-M5 module also captured additional AP-1 subunits and interactors, such as <em>FOSL2</em> (<a href="#r40" rid="r40" class=" bibr popnode">40</a>) and <em>IRF3</em> (<a href="#r41" rid="r41" class=" bibr popnode">41</a>, <a href="#r42" rid="r42" class=" bibr popnode">42</a>). Neu-M5 module was enriched in immune response pathways, providing further evidence that AP-1-mediated neuroinflammation was elevated in ASD neurons (<a href="/pmc/articles/PMC10013873/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>C</em></span></a>&#x000a0;and <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S10</a>). Hubs of Neu-M5 also contained multiple ion channel-related genes, such as sodium ion channel gene <em>SCN1B</em>, potassium channel genes <em>KCNJ2</em> and <em>KCNJ10</em>, and solute carrier gene <em>SLC40A1</em> (<a href="/pmc/articles/PMC10013873/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>C</em></span></a>&#x000a0;and <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S9</a>). Coordinated upregulation of various ion channels suggested that membrane transport was activated in ASD neurons, consistent with heightened excitability. Neu-M5 was significantly overlapped with module M16 from Voineagu et&#x000a0;al. (<a href="#r7" rid="r7" class=" bibr popnode">7</a>)<em>.</em> M16 was also enriched in immune/inflammatory response and was up-regulated in ASD (<a href="#r7" rid="r7" class=" bibr popnode">7</a>). Our data refined our understanding of the neuroinflammatory changes in ASD to include a neuronal component. Additionally, downregulated neu-M17 module was enriched in mitochondrial function and contained the most differentially expressed mitochondrial genes, such as ATP synthase subunits <em>ATP5F1B</em> and <em>ATP5PF</em> (<a href="/pmc/articles/PMC10013873/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>F</em></span></a>&#x000a0;and <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S9</a>).</p><p>Neuronal co-expression networks further captured signals that were not detected by DGE analysis. Neu-M6 module was upregulated in ASD, and among its hub genes were several insulin signaling pathway components, including insulin-like growth factor (IGF) receptor <em>IGF1R</em>, IGF binding protein <em>IGFBP5</em>, insulin receptor substrate <em>IRS2</em> as well as CBL-associated <em>SORBS1</em> (<a href="/pmc/articles/PMC10013873/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>D</em></span></a>&#x000a0;and <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S9</a>). Our results provided direct molecular-level evidence that insulin signaling was altered in ASD neurons.</p><p>Among all five significantly disrupted modules, none were enriched for ASD common variants and only one upregulated module (Neu-M16) showed enrichment in highly confident ASD risk genes, as well as in several other curated gene sets (<a href="/pmc/articles/PMC10013873/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>A</em></span></a>). Neu-M16 was enriched in synaptic functions (<a href="/pmc/articles/PMC10013873/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>E</em></span></a>&#x000a0;and <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S10</a>). Further, cell-type analysis showed that Neu-M16 was also highly enriched in excitatory neurons (<a href="/pmc/articles/PMC10013873/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>B</em></span></a>), with <em>CAMK2A</em> and <em>CAMK2B</em> among its hub genes (<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S9</a>). The upregulation of Neu-M16 suggested elevated excitatory signal in ASD neurons.</p><p>We further tested whether significantly disrupted neuronal modules were enriched in any neuron subtypes. Upregulated neuronal modules in ASD were only enriched in excitatory neuron subtypes, while enrichment of inhibitory neurons was only observed for downregulated modules (<a href="/pmc/articles/PMC10013873/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>B</em></span></a>). This provides additional evidence for altered neuronal activity in ASD neurons, consistent with the findings in our DGE analysis.</p><p>To gain mechanistic insights from our neuronal data, we used Bayesian network inference (<a href="#r43" rid="r43" class=" bibr popnode">43</a><a href="#r44" rid="r44" class=" bibr popnode"></a>&#x02013;<a href="#r45" rid="r45" class=" bibr popnode">45</a>) to distinguish between direct module-trait associations and indirect module-trait correlations. Our Bayesian network contained 19 nodes, including 18 neuronal co-expressed gene modules represented by module eigengenes, plus a binary node representing ASD diagnosis. Our results indicated that Neu-M5 was directly associated with ASD, conditioned on all other modules (<a href="/pmc/articles/PMC10013873/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>G</em></span></a>). This provided further evidence that neuroinflammation was directly associated with ASD in neurons.</p><p>We further used Bayesian network to prioritize among hub genes of Neu-M5 module for future validation. We focused on 21 genes within Neu-M5 that were also significant differentially expressed genes (DEGs). The Bayesian network learned from these 21 hub genes is shown in <a href="/pmc/articles/PMC10013873/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>H</em></span></a>. Genes with the highest degree of connectivity in this network included <em>BCL10, ELL<sub>,</sub></em> and <em>GTF2IRD2B</em>, which can be potential targets for functional evaluations.</p><p class="p p-last">As an orthogonal approach, we identified the upstream regulators and built mechanistic networks in Ingenuity Pathway Analysis (IPA&#x000ae;&#x02009;) with the differentially expressed genes in captured neurons (FDR &#x0003c; 0.1, 225 genes, <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S13</a>). Dopamine was predicted as a top upstream regulator with the mechanistic network presented in <a href="/pmc/articles/PMC10013873/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>I</em></span></a>. This dopamine network contained 17 upstream regulators predicted to drive the observed changes of 74 target genes, including AP-1 subunits <em>JUN, JUNB, FOS</em>, and GABA synthetase <em>GAD1</em> and <em>GAD2</em>. Moreover, there were seven genes (<em>BCL10, JUN, JUNB, KCNJ2, ELL, S1PR1,</em> and <em>SDC2</em>) from this mechanistic network that overlapped with the 21 genes identified by the Bayesian network analysis. This further highlighted the involvement of these genes in ASD, since they were identified independently by different modeling approaches.</p></div><div id="s5" class="sec sec-last"><p></p><h3 id="s5title" class="inline">Small Noncoding RNAs Are Selectively Downregulated in ASD Neurons and Correlate with Altered Local Splicing. </h3><p class="p p-first">When investigating the genes downregulated in ASD neurons more closely, we noticed a striking pattern that 51 out of the 59 neuron-expressed small nucleolar RNA (snoRNA) (<a href="#r46" rid="r46" class=" bibr popnode">46</a>) genes were downregulated in ASD neurons, and 13 were significantly downregulated at <em>P</em>-value &#x0003c; 0.05 (<a href="/pmc/articles/PMC10013873/figure/fig05/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig05" rid-ob="ob-fig05" co-legend-rid="lgnd_fig05"><span>Fig. 5<em>A</em></span></a>&#x000a0;and <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S6</a>). Dysregulation of snoRNAs was not observed in bulk tissue (<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S1</a>), and snoRNAs were undetectable in a recent ASD single-cell study (<a href="#r12" rid="r12" class=" bibr popnode">12</a>). snoRNAs are involved in the modification and maturation of ribosomal RNAs (rRNAs) and small nuclear RNAs (snRNAs) (<a href="#r47" rid="r47" class=" bibr popnode">47</a>). Interestingly, both ribosome and spliceosome components were among the most downregulated in ASD neurons (<a href="/pmc/articles/PMC10013873/figure/fig03/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig03" rid-ob="ob-fig03" co-legend-rid="lgnd_fig03"><span>Fig. 3<em>E</em></span></a>). Moreover, snRNAs were also downregulated in ASD neurons, with 23 out of 24 snRNA genes downregulated in ASD and 13 significantly downregulated at P value &#x0003c; 0.05 (<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI&#x000a0;Appendix</em>, Fig. S5</a>&#x000a0;and <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S6</a>)&#x000a0;<a href="/pmc/articles/PMC10013873/table/t01/" target="table" class="fig-table-link figpopup" rid-figpopup="t01" rid-ob="ob-t01" co-legend-rid=""><span>Table 1.</span></a></p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig05" co-legend-rid="lgnd_fig05"><a href="/pmc/articles/PMC10013873/figure/fig05/" target="figure" rid-figpopup="fig05" rid-ob="ob-fig05"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139624426005024"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10013873_pnas.2206758120fig05.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pnas.2206758120fig05.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10013873/bin/pnas.2206758120fig05.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139624426005024"><a target="object" rel="noopener" href="/pmc/articles/PMC10013873/figure/fig05/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig05"><div><a class="figpopup" href="/pmc/articles/PMC10013873/figure/fig05/" target="figure" rid-figpopup="fig05" rid-ob="ob-fig05">Fig. 5.</a></div><!--caption a7--><div class="caption"><p>Coordinated Dysregulation of snoRNAs in ASD Neurons. (<em>A</em>) Volcano plot showing differentially expressed genes in ASD neurons compared to control. snoRNA genes were colored red. (<em>B</em>) Significant correlation between snoRNA expression (x-axis) and intron PSI (y-axis). Introns are labeled with the name of overlapping gene locus. Gene loci that are correlated with more than 3 snoRNAs were shown. (<em>C</em>) Scatter plot showing the correlation between GOT1 intron 5 PSI and expression levels of multiple snoRNAs across all neuron samples. Also shown were fitted regression lines with 95% CIs. Intron coordinates were based on GRCh37. (<em>D</em>), PSI of GOT1 intron 5 is downregulated in ASD neurons.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="t01"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p>Summary of genes discussed in this manuscript</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr><th align="left" rowspan="1" colspan="1">Noted Categories</th><th align="center" rowspan="1" colspan="1">Genes</th></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1">E/I signal</td><td align="center" rowspan="1" colspan="1">
<em>CAMK2A<sup>*</sup>; CAMK2B<sup>*</sup>; GAD1<sup>*</sup>; GAD2<sup>*</sup>; GOT1<sup>*</sup></em>
</td></tr><tr><td align="left" rowspan="1" colspan="1">Growth / differentiation</td><td align="center" rowspan="1" colspan="1">
<em>FOS<sup>*</sup>; FOSL2<sup>*</sup>; IRF3<sup>*</sup>; JUN<sup>*</sup>; JUNB<sup>*</sup>; PPP1R16B<sup>*</sup>; S1PR1<sup>*</sup>; SOX9<sup>*</sup></em>
</td></tr><tr><td align="left" rowspan="1" colspan="1">Heat shock</td><td align="center" rowspan="1" colspan="1">
<em>BAG3<sup>&#x02020;</sup>; DNAJB1<sup>&#x02020;</sup>; DNAJB4<sup>&#x02020;</sup>; HSPA1A<sup>&#x02020;</sup>; HSPA1B<sup>&#x02020;</sup>; HSPB1<sup>&#x02020;</sup>; HSPB8<sup>&#x02020;</sup>; PTGES3<sup>&#x02020;</sup></em>
</td></tr><tr><td align="left" rowspan="1" colspan="1">Immune / inflammation</td><td align="center" rowspan="1" colspan="1">
<em>BCL10<sup>*</sup>; NFKBID<sup>*</sup></em>
</td></tr><tr><td align="left" rowspan="1" colspan="1">Insulin signaling</td><td align="center" rowspan="1" colspan="1">
<em>IGF1R<sup>*</sup>; IGFBP5<sup>*</sup>; IRS2<sup>*</sup>; SORBS1<sup>*</sup></em>
</td></tr><tr><td align="left" rowspan="1" colspan="1">Ion channel</td><td align="center" rowspan="1" colspan="1">
<em>KCNH3<em><sup>&#x02020;</sup></em>; KCNIP1<sup>&#x02020;</sup>; KCNJ10<sup>*</sup>; KCNJ2<sup>*</sup>; SCN1B<sup>*</sup>; SLC40A1<sup>*</sup></em>
</td></tr><tr><td align="left" rowspan="1" colspan="1">Mitochondrial</td><td align="center" rowspan="1" colspan="1">
<em>ATP5F1B<sup>*</sup>; ATP5PF<sup>*</sup></em>
</td></tr><tr><td align="left" rowspan="1" colspan="1">Age-associated</td><td align="center" rowspan="1" colspan="1">
<em>GAD1<sup>&#x02020;</sup>; GAD2<sup>&#x02020;</sup>; HTRA2<sup>*</sup>; PCDHB18P<sup>*</sup>; ZNF765<sup>*</sup></em>
</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139624431929952"><a target="object" rel="noopener" href="/pmc/articles/PMC10013873/table/t01/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tbl1fn1"><p class="p p-first-last"><sup>*</sup>observed in STG neurons.</p></div><div id="tbl1fn2"><p class="p p-first-last"><sup>&#x02020;</sup>observed in bulk STG.</p></div><div id="tbl1fn3"><p class="p p-first-last">The category of each gene is based on what we discussed in this manuscript.</p></div></div></div><p class="p p-last">As snoRNAs and snRNAs are known to be critical regulators of alternative splicing (<a href="#r48" rid="r48" class=" bibr popnode">48</a><a href="#r49" rid="r49" class=" bibr popnode"></a><a href="#r50" rid="r50" class=" bibr popnode"></a>&#x02013;<a href="#r51" rid="r51" class=" bibr popnode">51</a>), and splicing alterations are strongly implicated in ASD and observed in LCM-captured neurons, we next examined alterations in splicing events that may be associated with snoRNA dysregulation. We calculated the correlation between snoRNA expression level and each local splicing event in neuronal data, followed by FDR correction for multiple comparisons. We identified 835 gene loci in neurons with at least one intron whose percent spliced in (PSI) was significantly correlated with snoRNA gene expression (<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S11</a>). Of these 835 intron clusters, 196 were significantly dysregulated in ASD neurons (<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.2206758120#supplementary-materials" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Dataset S11</a>). Several intron clusters correlated with multiple snoRNAs and corresponded to genes involved in synaptic functions (<a href="/pmc/articles/PMC10013873/figure/fig05/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig05" rid-ob="ob-fig05" co-legend-rid="lgnd_fig05"><span>Fig. 5<em>B</em></span></a>). For example, <em>GOT1</em> encodes the glutamic-oxaloacetic transaminase known to function as an essential regulator of glutamate level (<a href="#r52" rid="r52" class=" bibr popnode">52</a>). PSI of intron 5 of <em>GOT1</em> was highly correlated with the expression level of multiple snoRNA genes (<a href="/pmc/articles/PMC10013873/figure/fig05/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig05" rid-ob="ob-fig05" co-legend-rid="lgnd_fig05"><span>Fig. 5<em>C</em></span></a>). <em>GOT1</em> intron 5 was also differentially spliced between ASD and control (<a href="/pmc/articles/PMC10013873/figure/fig05/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig05" rid-ob="ob-fig05" co-legend-rid="lgnd_fig05"><span>Fig. 5<em>D</em></span></a>). DS of <em>GOT1</em> gene may change the level of glutamate and thus leads to an imbalance of E/I in neuronal communication in ASD neurons.</p></div></div><div id="s6" class="tsec sec"><h2 class="head no_bottom_margin" id="s6title" style="text-transform: uppercase;">Discussion</h2><div id="s7" class="sec sec-first"><p></p><h3 id="s7title" class="inline">Bulk Tissue Transcriptome Findings Reveal Downregulated Neuronal and Synaptic Function Processes, Upregulation of HSPs, and Unfolded Protein Response (UPR) in ASD Temporal Cortex. </h3><p class="p p-first">Our bulk tissue analyses revealed a potential causal role of downregulated neuronal processes and synaptic functions in ASD etiology, consistent with findings from previous bulk-tissue transcriptomic studies on the same brain region (<a href="#r7" rid="r7" class=" bibr popnode">7</a>). Previous studies also reported dysregulated alternative splicing events in ASD brain (<a href="#r8" rid="r8" class=" bibr popnode">8</a>, <a href="#r53" rid="r53" class=" bibr popnode">53</a>). DS analysis in bulk STG found several synaptic genes, including <em>CANCA2D1</em>, <em>CAMK4</em>, <em>CLASP2</em>, <em>CNTNAP1</em>, <em>EPHB1</em>, <em>KALRN</em>, <em>NRXN3</em>, <em>SOS2</em> and <em>SYNGAP1</em>, differentially spliced in ASD. <em>SYNGAP1</em> isoforms have been shown to differentially regulate synaptic plasticity and dendritic development (<a href="#r54" rid="r54" class=" bibr popnode">54</a>). This further signifies the importance of studying alternatively spliced isoforms in ASD brain.</p><p class="p p-last">We also observed a coordinated upregulation of multiple HSPs and HSP-related chaperones in ASD STG. HSPs can serve as activators and regulators of the immune system (<a href="#r18" rid="r18" class=" bibr popnode">18</a>), and upregulated HSPs may induce immune responses in ASD brain. They also play a role in facilitating alternative RNA splicing (<a href="#r28" rid="r28" class=" bibr popnode">28</a>). Previous studies found that both immune response and RNA splicing are upregulated in ASD brain (<a href="#r7" rid="r7" class=" bibr popnode">7</a>, <a href="#r8" rid="r8" class=" bibr popnode">8</a>, <a href="#r11" rid="r11" class=" bibr popnode">11</a>), and our results signify that upregulated HSP-related pathways potentially contribute to these observations. HSPs and HSP-related chaperones are normally induced in response to stress. The upregulation of HSPs in ASD neurons may relate to elevated endoplasmic reticulum (ER) stress since both UPR and apoptosis are also upregulated in our ASD bulk data (<a href="/pmc/articles/PMC10013873/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2<em>D</em></span></a>). ASD-linked rare or de novo mutations in synaptic genes can lead to misfolded proteins and cause ER stress (<a href="#r55" rid="r55" class=" bibr popnode">55</a>), itself coupled to heightened inflammation and neurotoxic cell death (<a href="#r56" rid="r56" class=" bibr popnode">56</a>). ER stress-related genes are also dysregulated in the middle frontal cortex of subjects with ASD (<a href="#r57" rid="r57" class=" bibr popnode">57</a>). Our data suggest that ER stress is a major response to ASD genetic mutations, and ER stress activates UPR, including the production of HSPs and chaperones. UPR further induces multiple downstream processes such as inflammation and immune response (<a href="#r58" rid="r58" class=" bibr popnode">58</a>). Limiting the effect of ER stress and UPR may be a promising therapeutic avenue for ASD.</p></div><div id="s8" class="sec"><p></p><h3 id="s8title" class="inline">ASD Neuronal Transcriptome Reveals Upregulated Neuroinflammation and Altered Neuronal Activity. </h3><p class="p p-first">We observed a strong upregulation of AP-1 transcription factor components in ASD neurons. AP-1 subunits <em>FOS</em>, <em>JUN</em> and <em>JUNB</em> were upregulated at FDR &#x0003c; 0.05, and <em>FOSL2</em> was upregulated at nominal <em>P</em> &#x0003c; 0.05. AP-1 regulated gene expression in response to various stimuli, including cytokines, growth factors, stress signals, infections and inflammation/neuroinflammation (<a href="#r35" rid="r35" class=" bibr popnode">35</a>). In ASD neurons, AP-1 activation likely induces broad inflammatory response, since several immune and inflammation-related genes were also strongly upregulated. These included <em>NFKBID</em> and <em>BCL10</em>, which were involved in the NF-&#x003ba;B pathway and were upregulated at FDR &#x0003c; 0.05. In addition, the interferon regulatory factor <em>IRF3</em> is upregulated at <em>P</em> &#x0003c; 0.05, and <em>IRF3</em> is co-expressed with AP-1 subunits. The simultaneous upregulation of AP-1 subunits, NF&#x003ba;B-related genes and interferon regulatory factors suggested that immune and inflammation responses were activated in ASD neurons. Upregulated immune response and neuroinflammation have been consistently observed in ASD patients by bulk tissue transcriptomic studies largely implicating glial cells (<a href="#r7" rid="r7" class=" bibr popnode">7</a>, <a href="#r8" rid="r8" class=" bibr popnode">8</a>). However, our results demonstrated that immune/neuroinflammatory response was clearly activated in ASD neurons, and may be mediated by transcription factor AP-1.</p><p>We also observed strong downregulation in ASD STG neurons of <em>GAD1</em> and <em>GAD2</em> genes, involved in the biosynthesis of the inhibitory neurotransmitter GABA. In contrast, <em>CAMK2A</em> and <em>CAMK2B</em> genes, which are essential for aspects of plasticity at glutamatergic excitatory synapses, are upregulated at nominal significance. In addition, co-expressed gene modules that were upregulated in ASD neurons were mainly enriched in excitatory neurons, while the downregulated module was primarily enriched in inhibitory neurons. These data supported the hypothesis that ASD reflects imbalance of E/I in neuronal communication, also reported in several brain regions in ASD (<a href="#r19" rid="r19" class=" bibr popnode">19</a>, <a href="#r20" rid="r20" class=" bibr popnode">20</a>, <a href="#r39" rid="r39" class=" bibr popnode">39</a>). This is a report providing molecular-level evidence for imbalance of E/I in neuronal communication specifically in STG neurons in ASD.</p><p>Future studies will focus on the role of snoRNAs in ASD neurons, and other long and small modulatory non-coding RNAs. Given the emerging role of snoRNAs as alternative splicing regulators (<a href="#r50" rid="r50" class=" bibr popnode">50</a>, <a href="#r51" rid="r51" class=" bibr popnode">51</a>), we hypothesize that a coordinated downregulation of multiple snoRNAs correlates with elevated dysregulation of local splicing events in ASD neurons. Our data provide evidence supporting this hypothesis; however, no causal relationship can be determined. It will be critical to determine if snoRNA dysregulation plays a causal role in ASD etiology, and if so, pinpoint the underlying mechanism and possibly relate to transcript isoforms driving ASD brain and neuronal phenotype.</p><p class="p p-last">In our study, the concordance of ASD-associated transcriptomic changes in bulk tissue and LCM neurons was relatively low. This is not surprising given bulk tissue contains various different neuronal and non-neuronal cell types. Neuron-specific changes could be masked by changes in other cell types when profiling bulk tissue. The lack of concordance highlighted the necessity of specifically isolating and profiling each cell type to identify cell-type-specific transcriptomic signatures.</p></div><div id="s9" class="sec"><p></p><h3 id="s9title" class="inline">Age-Associated Differential Expression in STG Points to Altered Neuronal Activity in ASD. </h3><p class="p p-first">In bulk tissue, genes involved in GABAergic signaling (<em>GAD1</em> and <em>GAD2</em>) were upregulated with age in controls, while downregulated with age in ASD. Multiple lines of evidence have pointed to deviations in the production, transmission, and reception of GABAergic inhibitory interneurons, including decreased numbers of GABAergic interneurons (especially Parvalbumin neurons) (<a href="#r59" rid="r59" class=" bibr popnode">59</a>) and reduced density of GABA receptors (<a href="#r60" rid="r60" class=" bibr popnode">60</a><a href="#r61" rid="r61" class=" bibr popnode"></a>&#x02013;<a href="#r62" rid="r62" class=" bibr popnode">62</a>). Our findings indicated that the reduction of <em>GAD1</em> and <em>GAD2</em> mRNA levels in ASD brain became more profound with age, consistent with the observations at cellular level. In addition, <em>SLC38A1</em>, involved in neurotransmission at glutaminergic and GABAergic synapses (<a href="#r63" rid="r63" class=" bibr popnode">63</a>), was downregulated in ASD relative to control STG.&#x000a0;<em>SLC38A1</em> is implicated in Rett Syndrome (<a href="#r64" rid="r64" class=" bibr popnode">64</a>) and mitochondrial disorders, and its decrease may contribute to the observed alterations in synapse formation and neural connectivity.</p><p>In LCM neurons, the expression of <em>HTRA2</em> was higher below age 30 and decreases with age in control neurons, while lower at younger ages and increasing with age in ASD neurons. HTRA2 is important in maintaining mitochondrial homeostasis (<a href="#r65" rid="r65" class=" bibr popnode">65</a>) and inducing apoptosis. It is implicated in pathogenesis of neurodegeneration, hypoxic-ischemic damage, and is proposed as a potential treatment target in neurological diseases (<a href="#r66" rid="r66" class=" bibr popnode">66</a>). Attenuated <em>HTRA2</em> activity may lead to neuronal cell death, altered chaperon activity and autophagy and has been linked to Parkinson&#x02019;s disease (<a href="#r67" rid="r67" class=" bibr popnode">67</a>). In addition, increased active form of the <em>OMI/HTRA2</em> serine protease has been positively correlated with cholinergic alterations in AD brain (<a href="#r68" rid="r68" class=" bibr popnode">68</a>). Thus, it is plausible that the altered expression of <em>HTRA2</em> with age we observed in ASD brain may be associated with neuronal alterations during development. These findings further support the hypothesis of altered neuronal E/I activity, neuroinflammation, cell death, and mitochondrial dysfunction, implicated in ASD (<a href="#r69" rid="r69" class=" bibr popnode">69</a>) and suggest treatment windows to target specific genes to alter their expression trajectories with age.</p><p class="p p-last">In summary, our age-related findings support the premise brain development in individuals with ASD deviates from that of the neurotypical trajectory beginning in childhood and continues to evolve across the lifespan (<a href="#r70" rid="r70" class=" bibr popnode">70</a>, <a href="#r71" rid="r71" class=" bibr popnode">71</a>). Although the STG remains relatively unexplored (<a href="#r72" rid="r72" class=" bibr popnode">72</a><a href="#r73" rid="r73" class=" bibr popnode"></a>&#x02013;<a href="#r74" rid="r74" class=" bibr popnode">74</a>), other brain regions display early excess followed by reductions in volume, connectivity, and cell densities as people with ASD age through adulthood. Initial excess and overconnectivity may lead to hyperexcitation, rendering the brain vulnerable to age-related and pro-inflammatory mechanisms contributing to later degenerative outcomes. Our findings of altered neuroinflammatory expression patterns, taken together with reports of excessive microglial activation in STG (<a href="#r74" rid="r74" class=" bibr popnode">74</a>, <a href="#r75" rid="r75" class=" bibr popnode">75</a>), implicate immune dysfunction in the pathophysiology of ASD that may exacerbate with age. Additionally, we found significant relationships with our ASD transcriptome profiles and AD profiles in the same brain region (<a href="#r25" rid="r25" class=" bibr popnode">25</a>), supporting recent theories of increased susceptibility to neurodegenerative and cognitive decline (<a href="#r76" rid="r76" class=" bibr popnode">76</a>, <a href="#r77" rid="r77" class=" bibr popnode">77</a>). Lastly, our findings of an age-related decrease in <em>GAD</em> expression in bulk STG tissue, and an overall downregulation of <em>GAD</em> in neurons, further supports the hypothesis that GABAergic inhibitory neurons are disproportionally affected in ASD throughout the lifespan.</p></div><div id="s10" class="sec sec-last"><p></p><h3 id="s10title" class="inline">Mechanistic Modeling Highlighted Pathways and Genes That Were Directly Associated with ASD in Neurons. </h3><p class="p p-first-last">We applied two orthogonal approaches to gain mechanistic insight from our LCM neuron data. Bayesian network inference built a network from co-expressed modules and further prioritized hub genes within selected module(s), identifying <em>BCL10, ELL</em> and <em>GTF2IRD2B</em> as genes having driving roles in ASD. The approach from Ingenuity Pathways Analyses first identified upstream regulators that align with the observed expression changes in our dataset and elucidated that they share a predicted association with dopamine signaling pathways. Expansion of the regulatory network to include another layer of DEGs in our dataset indicated strong overlap and regulatory connections to many key genes involved in neuronal function and inflammation (<em>GAD1, GAD2,</em> and many transcriptional regulators). Interestingly, the expansion of the IPA network further converged on regulatory molecules <em>BCL10</em> and RNA polymerase II, of which the former was directly identified in the independent Bayesian approach, and the latter associated with <em>ELL</em>. Together these two independent approaches highlight the role of inflammatory pathways and genes in ASD etiology.</p></div></div><div id="s11" class="tsec sec"><h2 class="head no_bottom_margin" id="s11title" style="text-transform: uppercase;">Limitation and Future Studies</h2><p class="p p-first-last">Although our study had a relatively large sample size of 59 human brains, there are inherent limitations to utilizing postmortem human brain tissue, including variability in cause of death, co-morbid conditions, medication use, postmortem interval (PMI), etc., Future studies will have the advantage of more in-depth clinical information available from newer brain tissue collections such as Autism BrainNet (SFARI) and the NIH Biobanks to evaluate the relationship of cellular and molecular findings with clinical characteristics. Most of our subjects were male, consistent with the 4:1 ratio of males/females with ASD; future studies should include a larger female cohort to adequately evaluate sex differences. Additionally, the development of novel tools for profiling the transcriptome such as scRNA-seq allows for analyses of individual cell types at high resolution; however, snRNA-seq currently does not provide the depth of coverage to quantify low-expressed genes and local splicing, especially in human brain samples that we were able to achieve with LCM neuron samples. Our study, however, did not distinguish neuronal subtypes, as we found Histogene the most reliable cell body staining method for maintaining tissue quality and accurate neuronal identification. Our future studies will expand on current findings, particularly age-related changes in <em>GAD</em> implicating GABAergic interneurons, to map expression in specific cell types in multiple brain regions, in relation to clinical characteristics, across the human lifespan.</p></div><div id="s12" class="tsec sec"><h2 class="head no_bottom_margin" id="s12title" style="text-transform: uppercase;">Methods</h2><div id="s13" class="sec sec-first"><p></p><h3 id="s13title" class="inline">Bulk Tissue RNA Extraction and Library Preparation. </h3><p class="p p-first">Tissue and clinical data collection procedures were approved by the institutional review board (IRB) and Human and Anatomical Specimens Tissue Oversight Committee (HASTOC) at the University of California, Davis School of Medicine. Informed consent was obtained from next-of-kin at the time of brain tissue collection for follow-up to collect donor clinical information to confirm diagnoses by the Autism Tissue Program (now Autism BrainNet).</p><p>Human brain tissue was collected, sectioned coronally and flash-frozen. The STG was identified anatomically according to &#x0201c;Atlas of the Human Brain&#x0201d; fourth edition (Maj, Majtanik, Paxinos 2015). Total RNA was extracted using the Direct-zol RNA MiniPrep (Zymo Research #R2051).</p><p class="p p-last">Fifty nanograms of RNA from each sample were used to create strand-specific total RNA libraries with the NuGEN Ovation Universal RNA-Seq System v2 and processed in parallel on the Sciclone NGS automated workstation (Perkin Elmer) according to manufacturer protocol.</p></div><div id="s14" class="sec"><p></p><h3 id="s14title" class="inline">Laser Capture Microdissection, RNA Extraction, and Library Preparation. </h3><p class="p p-first-last">Using a Leica LMD-6000 laser capture microdissection system, 100 neurons from each sample were laser captured directly into lysis buffer. RNA was extracted using the PicoPure total RNA kit according to manufacturer protocol. Strand-specific rRNA depleted RNA libraries were prepared from 10 &#x003bc;L of the final neuronal RNA.</p></div><div id="s15" class="sec"><p></p><h3 id="s15title" class="inline">RNA Sequencing, Read Mapping, Quantification of Gene Expression, and QC. </h3><p class="p p-first-last">RNA-Seq was performed on Illumina HiSeq4000. Libraries were sequenced to ~50 million 2 &#x000d7;&#x000a0;150 bp reads per sample. Reads were aligned to the GRCH37.p13 (hg19) reference genome using STAR. Gene-level quantifications were calculated using featureCounts (v1.6.4). Sample outliers were defined as samples with standardized sample network connectivity Z scores &#x0003c; &#x02212;2 (<a href="#r78" rid="r78" class=" bibr popnode">78</a>), and were removed. Using this method, five samples from bulk data, and five samples from neuron data were removed. Batch effects were accounted for using metrics derived from PicardTools (v2.21.2).</p></div><div id="s16" class="sec"><p></p><h3 id="s16title" class="inline">DGE and Differential Alternative Splicing. </h3><p class="p p-first-last">DGE analyses were performed using DESeq2 (1.22.2)(<a href="#r79" rid="r79" class=" bibr popnode">79</a>) with default parameters. Local splicing analysis was performed using LeafCutter (<a href="#r26" rid="r26" class=" bibr popnode">26</a>) as previously described (<a href="#r10" rid="r10" class=" bibr popnode">10</a>).</p></div><div id="s17" class="sec"><p></p><h3 id="s17title" class="inline">Co-expression Network Analysis. </h3><p class="p p-first-last">Weighted gene co-expression network analysis (WGCNA)&#x000a0;(<a href="#r27" rid="r27" class=" bibr popnode">27</a>) was performed to define modules of co-expressed genes from RNA-seq data.</p></div><div id="s18" class="sec"><p></p><h3 id="s18title" class="inline">Functional Enrichment Analysis. </h3><p class="p p-first">Gene ontology (GO) enrichment was performed using the gProfileR R package (<a href="#r80" rid="r80" class=" bibr popnode">80</a>) and the fGSEA algorithm as implemented in the clusterProfiler R package (<a href="#r81" rid="r81" class=" bibr popnode">81</a>).</p><p class="p p-last">Enrichment analyses were also performed using several established, hypothesis-driven gene sets. Significance was determined from permutation-derived null distribution.</p></div><div id="s19" class="sec"><p></p><h3 id="s19title" class="inline">Cell Type Enrichment Analysis. </h3><p class="p p-first-last">Cell-type enrichment analysis was performed using the Expression Weighted Cell Type Enrichment (EWCE) package in R (<a href="#r82" rid="r82" class=" bibr popnode">82</a>).</p></div><div id="s20" class="sec"><p></p><h3 id="s20title" class="inline">GWAS Enrichment Analysis. </h3><p class="p p-first-last">Stratified LD score regression (sLDSC) (<a href="#r83" rid="r83" class=" bibr popnode">83</a>) was used to test whether a gene set of interest is enriched for SNP-heritability in a given GWAS dataset.</p></div><div id="s21" class="sec"><p></p><h3 id="s21title" class="inline">Ingenuity Pathway Analysis. </h3><p class="p p-first-last">Ingenuity Pathway Analysis (IPA&#x000ae;, QIAGEN) tools were used to predict the upstream regulators and to build mechanistic networks (<a href="#r84" rid="r84" class=" bibr popnode">84</a>).</p></div><div id="s22" class="sec"><p></p><h3 id="s22title" class="inline">Designing and Learning the Bayesian Network Structure. </h3><p class="p p-first-last">Each of the 18 module eigengenes from our neuron co-expression data serves as one random variable (node) in our Bayesian network (BN). In addition, we also include diagnosis (ASD or CTL) as a binary random variable (node). This design allows us to distinguish between modules that are directly associated with ASD and those that are indirectly associated with ASD (<a href="#r45" rid="r45" class=" bibr popnode">45</a>).</p></div><div id="s23" class="sec"><p></p><h3 id="s23title" class="inline">Prioritizing Genes within Neu-M5. </h3><p class="p p-first-last">To prioritize genes within the Neu-M5 module, we first identified 21 genes within Neu-M5 that are also significant DEGs. We then built Bayesian network among the 21 genes with method described in the above section. We selected genes with the highest number of connections as nodes likely to influence the expression of many other genes within the module.</p></div><div id="s24" class="sec sec-last"><p></p><h3 id="s24title" class="inline">Alzheimer&#x02019;s Disease Datasets. </h3><p class="p p-first-last">Alzheimer's disease DGE data was obtained from the Mount Sinai/JJ Peters VA MSBB study (<a href="#r25" rid="r25" class=" bibr popnode">25</a>) through Synapse via accession number syn30821563.</p></div></div><div id="sec-a.z.j" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="sec-a.z.jtitle" style="text-transform: uppercase;">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="supplementary-material-a.z.j.b"><p>Appendix 01 (PDF)</p><div class="sup-box half_rhythm" id="media-a.z.j.b.b"><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sapp.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(1.7M, pdf)</sup></div></div><div class="sec suppmat" id="supplementary-material-a.z.j.c"><p>Dataset S01 (XLSX)</p><div class="sup-box half_rhythm" id="media-a.z.j.c.b"><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd01.xlsx" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(2.1M, xlsx)</sup></div></div><div class="sec suppmat" id="supplementary-material-a.z.j.d"><p>Dataset S02 (XLSX)</p><div class="sup-box half_rhythm" id="media-a.z.j.d.b"><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd02.xlsx" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(2.3M, xlsx)</sup></div></div><div class="sec suppmat" id="supplementary-material-a.z.j.e"><p>Dataset S03 (XLSX)</p><div class="sup-box half_rhythm" id="media-a.z.j.e.b"><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd03.xlsx" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(728K, xlsx)</sup></div></div><div class="sec suppmat" id="supplementary-material-a.z.j.f"><p>Dataset S04 (XLSX)</p><div class="sup-box half_rhythm" id="media-a.z.j.f.b"><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd04.xlsx" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(361K, xlsx)</sup></div></div><div class="sec suppmat" id="supplementary-material-a.z.j.g"><p>Dataset S05 (XLSX)</p><div class="sup-box half_rhythm" id="media-a.z.j.g.b"><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd05.xlsx" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(14M, xlsx)</sup></div></div><div class="sec suppmat" id="supplementary-material-a.z.j.h"><p>Dataset S06 (XLSX)</p><div class="sup-box half_rhythm" id="media-a.z.j.h.b"><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd06.xlsx" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(1.3M, xlsx)</sup></div></div><div class="sec suppmat" id="supplementary-material-a.z.j.i"><p>Dataset S07 (XLSX)</p><div class="sup-box half_rhythm" id="media-a.z.j.i.b"><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd07.xlsx" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(1.3M, xlsx)</sup></div></div><div class="sec suppmat" id="supplementary-material-a.z.j.j"><p>Dataset S08 (XLSX)</p><div class="sup-box half_rhythm" id="media-a.z.j.j.b"><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd08.xlsx" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(6.6M, xlsx)</sup></div></div><div class="sec suppmat" id="supplementary-material-a.z.j.k"><p>Dataset S09 (XLSX)</p><div class="sup-box half_rhythm" id="media-a.z.j.k.b"><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd09.xlsx" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(414K, xlsx)</sup></div></div><div class="sec suppmat" id="supplementary-material-a.z.j.l"><p>Dataset S010 (XLSX)</p><div class="sup-box half_rhythm" id="media-a.z.j.l.b"><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd10.xlsx" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(156K, xlsx)</sup></div></div><div class="sec suppmat" id="supplementary-material-a.z.j.m"><p>Dataset S011 (XLSX)</p><div class="sup-box half_rhythm" id="media-a.z.j.m.b"><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd11.xlsx" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(168K, xlsx)</sup></div></div><div class="sec suppmat" id="supplementary-material-a.z.j.n"><p>Dataset S012 (XLSX)</p><div class="sup-box half_rhythm" id="media-a.z.j.n.b"><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd12.xlsx" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(15K, xlsx)</sup></div></div><div class="sec suppmat" id="supplementary-material-a.z.j.o"><p>Dataset S013 (XLSX)</p><div class="sup-box half_rhythm" id="media-a.z.j.o.b"><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd13.xlsx" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(55K, xlsx)</sup></div></div><div class="sec suppmat" id="supplementary-material-a.z.j.p"><p>Dataset S014 (XLSX)</p><div class="sup-box half_rhythm" id="media-a.z.j.p.b"><a href="/pmc/articles/PMC10013873/bin/pnas.2206758120.sd14.xlsx" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(149K, xlsx)</sup></div></div></div><div id="ack-a.aa.a" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.aa.atitle" style="text-transform: uppercase;">Acknowledgments</h2><div class="sec"><p>We are grateful to the families of our brain donors for their invaluable gift to autism research. We appreciate the contributions of Carolyn Komich Hare, clinical Director of Autism BrainNet, for collection of the ADI-R as well as Robin Riedel, UC Davis, for figure design. Tissue was provided by the University of Maryland Brain and Tissue Bank, Dr. Daniel Campbell at Michigan State University, Autism Tissue Program (now Autism BrainNet, supported by the Simons Foundation), Brain Endowment for Autism Research Sciences (BEARS) at the UC Davis MIND Institute, and the Harvard Brain Tissue Resource Center. Laser capture microdissection was conducted using the CAMI core facility at UC Davis Center for Health and the Environment with instrumentation funding by NIHS10RR-023555 (carried out by A.O.). RNA sequencing was conducted using the Vincent J. Coates Genomics Sequencing Laboratory at UC Berkeley, supported by NIH S10 OD018174 Instrumentation Grant. M.J.G.&#x02019;s contributions were supported by a SFARI Bridge to Independence Award, R01-MH121521, R01-MH123922 and IDDRC P50-HD103557 at UCLA. This research was supported by NIH grants MH108909 (MPI C.M.S. and B.S.), R01 MH097236 and R21 MH 119650 (C.M.S), and the Intellectual and Developmental Disabilities Research Center (IDDRC) at the UC Davis MIND Institute (1U54HD079125).</p><div id="s26" class="sec"><p></p><h3 id="s26title" class="inline">Author contributions </h3><p class="p p-first-last">B.P.A., B.S., and C.M.S. designed research; P.Z., A.O., M.J.G., B.P.A., B.S., and C.M.S. performed research; P.Z., A.O., M.J.G., and B.S. analyzed data; M.J.G., B.S., and C.M.S. supervised the study; and P.Z., B.P.A., M.J.G, B.S., and C.M.S. wrote the paper.</p></div><div id="s27" class="sec sec-last"><p></p><h3 id="s27title" class="inline">Competing interests </h3><p class="p p-first-last">The authors declare no competing interest.</p></div></div></div><div id="fn-group-a.aa.b" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.aa.btitle" style="text-transform: uppercase;">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="fn3"><p class="p p-first-last">This article is a PNAS Direct Submission.</p></p></div></div><div id="s25" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="s25title" style="text-transform: uppercase;">Data, Materials, and Software Availability</h2><!--/article/back/sec/--><p class="p p-first-last">All custom code used in this manuscript is available at <a href="https://github.com/gandallab/ASD_STG_LCM_RNAseq" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/gandallab/ASD_STG_LCM_RNAseq</a> (<a href="#r85" rid="r85" class=" bibr popnode">85</a>). RNA-seq data from this study is available through the database for Genotypes and Phenotypes (dbGaP Study Accession ID: <a href="https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs003208.v1.p1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">phs003208</a>) (<a href="#r86" rid="r86" class=" bibr popnode">86</a>).</p></div><div id="s28" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="s28title" style="text-transform: uppercase;">Supporting Information</h2><!--/article/back/sec/--></div><div id="ref-list-a.aa.d" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.aa.dtitle" style="text-transform: uppercase;">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="r1">1. <span class="mixed-citation">Kim Y. S., et al., <span class="ref-title">Prevalence of autism spectrum disorders in a total population sample</span>. <span class="ref-journal">Am. J. Psychiatry.</span>
<span class="ref-vol"><strong>168</strong></span>, 904&#x02013;912 (2011). [<a href="https://pubmed.ncbi.nlm.nih.gov/21558103" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Psychiatry.&amp;title=Prevalence+of+autism+spectrum+disorders+in+a+total+population+sample&amp;volume=168&amp;publication_year=2011&amp;pages=904-912&amp;pmid=21558103&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r2">2. <span class="mixed-citation">Maenner M. J., et al., <span class="ref-title">Prevalence of autism spectrum disorder among children aged 8 years - autism and developmental disabilities monitoring network, 11 sites, United States, 2016</span>. <span class="ref-journal">MMWR Surveill. Summ.</span>
<span class="ref-vol"><strong>69</strong></span>, 1&#x02013;12 (2020). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7119644/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32214087" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=MMWR+Surveill.+Summ.&amp;title=Prevalence+of+autism+spectrum+disorder+among+children+aged+8+years+-+autism+and+developmental+disabilities+monitoring+network,+11+sites,+United+States,+2016&amp;volume=69&amp;publication_year=2020&amp;pages=1-12&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r3">3. <span class="mixed-citation">Lord C., et al., <span class="ref-title">Autism spectrum disorder</span>. <span class="ref-journal">Nat. Rev. Dis. Primers</span>
<span class="ref-vol"><strong>6</strong></span>, 5 (2020). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8900942/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31949163" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Dis.+Primers&amp;title=Autism+spectrum+disorder&amp;volume=6&amp;publication_year=2020&amp;pages=5&amp;pmid=31949163&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r4">4. <span class="mixed-citation">Hansen S. N., et al., <span class="ref-title">Recurrence risk of autism in siblings and cousins: A multinational, population-based study</span>. <span class="ref-journal">J. Am. Acad. Child Adolesc. Psychiatry</span>
<span class="ref-vol"><strong>58</strong></span>, 866&#x02013;875 (2019). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6708733/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30851399" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Am.+Acad.+Child+Adolesc.+Psychiatry&amp;title=Recurrence+risk+of+autism+in+siblings+and+cousins:+A+multinational,+population-based+study&amp;volume=58&amp;publication_year=2019&amp;pages=866-875&amp;pmid=30851399&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r5">5. <span class="mixed-citation">Castelbaum L., Sylvester C. M., Zhang Y., Yu Q., Constantino J. N., <span class="ref-title">On the nature of monozygotic twin concordance and discordance for autistic trait severity: A quantitative analysis</span>. <span class="ref-journal">Behav. Genet.</span>
<span class="ref-vol"><strong>50</strong></span>, 263&#x02013;272 (2020). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7355281/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31853901" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Behav.+Genet.&amp;title=On+the+nature+of+monozygotic+twin+concordance+and+discordance+for+autistic+trait+severity:+A+quantitative+analysis&amp;volume=50&amp;publication_year=2020&amp;pages=263-272&amp;pmid=31853901&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r6">6. <span class="mixed-citation">Gaugler T., et al., <span class="ref-title">Most genetic risk for autism resides with common variation</span>. <span class="ref-journal">Nat. Genet</span>
<span class="ref-vol"><strong>46</strong></span>, 881&#x02013;885 (2014). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4137411/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25038753" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Most+genetic+risk+for+autism+resides+with+common+variation&amp;volume=46&amp;publication_year=2014&amp;pages=881-885&amp;pmid=25038753&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r7">7. <span class="mixed-citation">Voineagu I., et al., <span class="ref-title">Transcriptomic analysis of autistic brain reveals convergent molecular pathology</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol"><strong>474</strong></span>, 380&#x02013;384 (2011). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3607626/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21614001" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Transcriptomic+analysis+of+autistic+brain+reveals+convergent+molecular+pathology&amp;volume=474&amp;publication_year=2011&amp;pages=380-384&amp;pmid=21614001&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r8">8. <span class="mixed-citation">Parikshak N. N., et al., <span class="ref-title">Genome-wide changes in lncRNA, splicing, and regional gene expression patterns in autism</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol"><strong>540</strong></span>, 423&#x02013;427 (2016). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7102905/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27919067" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+changes+in+lncRNA,+splicing,+and+regional+gene+expression+patterns+in+autism&amp;volume=540&amp;publication_year=2016&amp;pages=423-427&amp;pmid=27919067&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r9">9. <span class="mixed-citation">Gandal M. J., et al., <span class="ref-title">Shared molecular neuropathology across major psychiatric disorders parallels polygenic overlap</span>. <span class="ref-journal">Science</span>
<span class="ref-vol"><strong>359</strong></span>, 693&#x02013;697 (2018). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5898828/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29439242" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Shared+molecular+neuropathology+across+major+psychiatric+disorders+parallels+polygenic+overlap&amp;volume=359&amp;publication_year=2018&amp;pages=693-697&amp;pmid=29439242&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r10">10. <span class="mixed-citation">Gandal M. J., et al., <span class="ref-title">Transcriptome-wide isoform-level dysregulation in ASD, schizophrenia, and bipolar disorder</span>. <span class="ref-journal">Science</span>
<span class="ref-vol"><strong>362</strong></span>, eaat8127 (2018). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6443102/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30545856" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Transcriptome-wide+isoform-level+dysregulation+in+ASD,+schizophrenia,+and+bipolar+disorder&amp;volume=362&amp;publication_year=2018&amp;pages=eaat8127&amp;pmid=30545856&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r11">11. <span class="mixed-citation">Gupta S., et al., <span class="ref-title">Transcriptome analysis reveals dysregulation of innate immune response genes and neuronal activity-dependent genes in autism</span>. <span class="ref-journal">Nat. Commun.</span>
<span class="ref-vol"><strong>5</strong></span>, 5748 (2014). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4270294/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25494366" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Transcriptome+analysis+reveals+dysregulation+of+innate+immune+response+genes+and+neuronal+activity-dependent+genes+in+autism&amp;volume=5&amp;publication_year=2014&amp;pages=5748&amp;pmid=25494366&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r12">12. <span class="mixed-citation">Velmeshev D., et al., <span class="ref-title">Single-cell genomics identifies cell type-specific molecular changes in autism</span>. <span class="ref-journal">Science</span>
<span class="ref-vol"><strong>364</strong></span>, 685&#x02013;689 (2019). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7678724/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31097668" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Single-cell+genomics+identifies+cell+type-specific+molecular+changes+in+autism&amp;volume=364&amp;publication_year=2019&amp;pages=685-689&amp;pmid=31097668&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r13">13. <span class="mixed-citation">Gandal M. J., et al., <span class="ref-title">Broad transcriptomic dysregulation occurs across the cerebral cortex in ASD</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol"><strong>611</strong></span>, 532&#x02013;539 (2022). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9668748/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36323788" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Broad+transcriptomic+dysregulation+occurs+across+the+cerebral+cortex+in+ASD&amp;volume=611&amp;publication_year=2022&amp;pages=532-539&amp;pmid=36323788&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r14">14. <span class="mixed-citation">Jou R. J., Minshew N. J., Keshavan M. S., Vitale M. P., Hardan A. Y., <span class="ref-title">Enlarged right superior temporal gyrus in children and adolescents with autism</span>. <span class="ref-journal">Brain Res.</span>
<span class="ref-vol"><strong>1360</strong></span>, 205&#x02013;212 (2010). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2990401/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20833154" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain+Res.&amp;title=Enlarged+right+superior+temporal+gyrus+in+children+and+adolescents+with+autism&amp;volume=1360&amp;publication_year=2010&amp;pages=205-212&amp;pmid=20833154&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r15">15. <span class="mixed-citation">Amaral D. G., Schumann C. M., Nordahl C. W., <span class="ref-title">Neuroanatomy of autism</span>. <span class="ref-journal">Trends Neurosci.</span>
<span class="ref-vol"><strong>31</strong></span>, 137&#x02013;145 (2008). [<a href="https://pubmed.ncbi.nlm.nih.gov/18258309" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Neurosci.&amp;title=Neuroanatomy+of+autism&amp;volume=31&amp;publication_year=2008&amp;pages=137-145&amp;pmid=18258309&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r16">16. <span class="mixed-citation">Wang D., et al., <span class="ref-title">Comprehensive functional genomic resource and integrative model for the human brain</span>. <span class="ref-journal">Science</span>
<span class="ref-vol"><strong>362</strong></span>, 8464 (2018). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6413328/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30545857" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Comprehensive+functional+genomic+resource+and+integrative+model+for+the+human+brain&amp;volume=362&amp;publication_year=2018&amp;pages=8464&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r17">17. <span class="mixed-citation">Feder M. E., Hofmann G. E., <span class="ref-title">Heat-shock proteins, molecular chaperones, and the stress response: Evolutionary and ecological physiology</span>. <span class="ref-journal">Annu. Rev. Physiol.</span>
<span class="ref-vol"><strong>61</strong></span>, 243&#x02013;282 (1999). [<a href="https://pubmed.ncbi.nlm.nih.gov/10099689" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Physiol.&amp;title=Heat-shock+proteins,+molecular+chaperones,+and+the+stress+response:+Evolutionary+and+ecological+physiology&amp;volume=61&amp;publication_year=1999&amp;pages=243-282&amp;pmid=10099689&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r18">18. <span class="mixed-citation">Binder R. J., <span class="ref-title">Functions of heat shock proteins in pathways of the innate and adaptive immune system</span>. <span class="ref-journal">J. Immunol.</span>
<span class="ref-vol"><strong>193</strong></span>, 5765&#x02013;5771 (2014). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4304677/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25480955" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Immunol.&amp;title=Functions+of+heat+shock+proteins+in+pathways+of+the+innate+and+adaptive+immune+system&amp;volume=193&amp;publication_year=2014&amp;pages=5765-5771&amp;pmid=25480955&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r19">19. <span class="mixed-citation">Sohal V. S., Rubenstein J. L. R., <span class="ref-title">Excitation-inhibition balance as a framework for investigating mechanisms in neuropsychiatric disorders</span>. <span class="ref-journal">Mol. Psychiatry.</span>
<span class="ref-vol"><strong>24</strong></span>, 1248&#x02013;1257 (2019). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6742424/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31089192" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Psychiatry.&amp;title=Excitation-inhibition+balance+as+a+framework+for+investigating+mechanisms+in+neuropsychiatric+disorders&amp;volume=24&amp;publication_year=2019&amp;pages=1248-1257&amp;pmid=31089192&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r20">20. <span class="mixed-citation">Nelson S. B., Valakh V., <span class="ref-title">Excitatory/inhibitory balance and circuit homeostasis in autism spectrum disorders</span>. <span class="ref-journal">Neuron</span>
<span class="ref-vol"><strong>87</strong></span>, 684&#x02013;698 (2015). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4567857/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26291155" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Excitatory/inhibitory+balance+and+circuit+homeostasis+in+autism+spectrum+disorders&amp;volume=87&amp;publication_year=2015&amp;pages=684-698&amp;pmid=26291155&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r21">21. <span class="mixed-citation">Chow M. L., et al., <span class="ref-title">Age-dependent brain gene expression and copy number anomalies in autism suggest distinct pathological processes at young versus mature ages</span>. <span class="ref-journal">PLoS Genet.</span>
<span class="ref-vol"><strong>8</strong></span>, e1002592 (2012). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3310790/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22457638" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Age-dependent+brain+gene+expression+and+copy+number+anomalies+in+autism+suggest+distinct+pathological+processes+at+young+versus+mature+ages&amp;volume=8&amp;publication_year=2012&amp;pages=e1002592&amp;pmid=22457638&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r22">22. <span class="mixed-citation">Grone B. P., Maruska K. P., <span class="ref-title">Three distinct glutamate decarboxylase genes in vertebrates</span>. <span class="ref-journal">Sci. Rep.</span>
<span class="ref-vol"><strong>6</strong></span>, 30507 (2016). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4962313/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27461130" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Rep.&amp;title=Three+distinct+glutamate+decarboxylase+genes+in+vertebrates&amp;volume=6&amp;publication_year=2016&amp;pages=30507&amp;pmid=27461130&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r23">23. <span class="mixed-citation">Patrick E., et al., <span class="ref-title">Deconvolving the contributions of cell-type heterogeneity on cortical gene expression</span>. <span class="ref-journal">PLoS Comput. Biol.</span>
<span class="ref-vol"><strong>16</strong></span>, e1008120 (2020). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7451979/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32804935" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Comput.+Biol.&amp;title=Deconvolving+the+contributions+of+cell-type+heterogeneity+on+cortical+gene+expression&amp;volume=16&amp;publication_year=2020&amp;pages=e1008120&amp;pmid=32804935&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r24">24. <span class="mixed-citation">Magnin E., <span class="ref-title">Neurodevelopmental and neurodegenerative similarities and interactions: A point of view about lifelong neurocognitive trajectories</span>. <span class="ref-journal">J. Alzheimers Dis.</span>
<span class="ref-vol"><strong>79</strong></span>, 1397&#x02013;1407 (2021). [<a href="https://pubmed.ncbi.nlm.nih.gov/33459644" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Alzheimers+Dis.&amp;title=Neurodevelopmental+and+neurodegenerative+similarities+and+interactions:+A+point+of+view+about+lifelong+neurocognitive+trajectories&amp;volume=79&amp;publication_year=2021&amp;pages=1397-1407&amp;pmid=33459644&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r25">25. <span class="mixed-citation">Wang M., et al., <span class="ref-title">The Mount Sinai cohort of large-scale genomic, transcriptomic and proteomic data in Alzheimer&#x02019;s disease</span>. <span class="ref-journal">Sci. Data</span>
<span class="ref-vol"><strong>5</strong></span>, 1&#x02013;16 (2018). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6132187/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30204156" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Data&amp;title=The+Mount+Sinai+cohort+of+large-scale+genomic,+transcriptomic+and+proteomic+data+in+Alzheimer&#x02019;s+disease&amp;volume=5&amp;publication_year=2018&amp;pages=1-16&amp;pmid=30482902&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r26">26. <span class="mixed-citation">Li Y. I., et al., <span class="ref-title">Annotation-free quantification of RNA splicing using LeafCutter</span>. <span class="ref-journal">Nat. Genetics</span>
<span class="ref-vol"><strong>50</strong></span>, 151&#x02013;158 (2018). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5742080/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29229983" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genetics&amp;title=Annotation-free+quantification+of+RNA+splicing+using+LeafCutter&amp;volume=50&amp;publication_year=2018&amp;pages=151-158&amp;pmid=29229983&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r27">27. <span class="mixed-citation">Langfelder P., Horvath S., <span class="ref-title">WGCNA: An R package for weighted correlation network analysis</span>. <span class="ref-journal">BMC Bioinformatics</span>
<span class="ref-vol"><strong>9</strong></span>, 559 (2008). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2631488/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19114008" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=WGCNA:+An+R+package+for+weighted+correlation+network+analysis&amp;volume=9&amp;publication_year=2008&amp;pages=559&amp;pmid=19114008&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r28">28. <span class="mixed-citation">Yost H. J., Lindquist S., <span class="ref-title">RNA splicing is interrupted by heat shock and is rescued by heat shock protein synthesis</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol"><strong>45</strong></span>, 185&#x02013;193 (1986). [<a href="https://pubmed.ncbi.nlm.nih.gov/2421918" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=RNA+splicing+is+interrupted+by+heat+shock+and+is+rescued+by+heat+shock+protein+synthesis&amp;volume=45&amp;publication_year=1986&amp;pages=185-193&amp;pmid=2421918&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r29">29. <span class="mixed-citation">Satterstrom F. K., et al., <span class="ref-title">Large-scale exome sequencing study implicates both developmental and functional changes in the neurobiology of autism</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol"><strong>180</strong></span>, 568&#x02013;84.e23 (2020). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7250485/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31981491" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Large-scale+exome+sequencing+study+implicates+both+developmental+and+functional+changes+in+the+neurobiology+of+autism&amp;volume=180&amp;publication_year=2020&amp;pages=568-84.e23&amp;pmid=31981491&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r30">30. <span class="mixed-citation">Abrahams B. S., et al., <span class="ref-title">SFARI Gene 2.0: A community-driven knowledgebase for the autism spectrum disorders (ASDs)</span>. <span class="ref-journal">Mol. Autism</span>
<span class="ref-vol"><strong>4</strong></span>, 36 (2013). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3851189/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24090431" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Autism&amp;title=SFARI+Gene+2.0:+A+community-driven+knowledgebase+for+the+autism+spectrum+disorders+(ASDs)&amp;volume=4&amp;publication_year=2013&amp;pages=36&amp;pmid=24090431&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r31">31. <span class="mixed-citation">Samocha K. E., et al., <span class="ref-title">A framework for the interpretation of de novo mutation in human disease</span>. <span class="ref-journal">Nat. Genet</span>
<span class="ref-vol"><strong>46</strong></span>, 944&#x02013;950 (2014). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4222185/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25086666" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+framework+for+the+interpretation+of+de+novo+mutation+in+human+disease&amp;volume=46&amp;publication_year=2014&amp;pages=944-950&amp;pmid=25086666&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r32">32. <span class="mixed-citation">Karczewski K. J., et al., <span class="ref-title">The ExAC browser: Displaying reference data information from over 60 000 exomes</span>. <span class="ref-journal">Nucleic Acids Res.</span>
<span class="ref-vol"><strong>45</strong></span>, D840&#x02013;D855 (2017). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5210650/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27899611" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=The+ExAC+browser:+Displaying+reference+data+information+from+over+60+000+exomes&amp;volume=45&amp;publication_year=2017&amp;pages=D840-D855&amp;pmid=27899611&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r33">33. <span class="mixed-citation">Wilkinson B., et al., <span class="ref-title">The autism-associated gene chromodomain helicase DNA-binding protein 8 (CHD8) regulates noncoding RNAs and autism-related genes</span>. <span class="ref-journal">Transl. Psychiatry.</span>
<span class="ref-vol"><strong>5</strong></span>, e568 (2015). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4471293/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25989142" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transl.+Psychiatry.&amp;title=The+autism-associated+gene+chromodomain+helicase+DNA-binding+protein+8+(CHD8)+regulates+noncoding+RNAs+and+autism-related+genes&amp;volume=5&amp;publication_year=2015&amp;pages=e568&amp;pmid=25989142&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r34">34. <span class="mixed-citation">Grove J., et al., <span class="ref-title">Identification of common genetic risk variants for autism spectrum disorder</span>. <span class="ref-journal">Nat. Genet</span>
<span class="ref-vol"><strong>51</strong></span>, 431&#x02013;444 (2019). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6454898/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30804558" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Identification+of+common+genetic+risk+variants+for+autism+spectrum+disorder&amp;volume=51&amp;publication_year=2019&amp;pages=431-444&amp;pmid=30804558&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r35">35. <span class="mixed-citation">Hess J., Angel P., Schorpp-Kistner M., <span class="ref-title">AP-1 subunits: Quarrel and harmony among siblings</span>. <span class="ref-journal">J. Cell Sci.</span>
<span class="ref-vol"><strong>117</strong></span>, 5965&#x02013;5973 (2004). [<a href="https://pubmed.ncbi.nlm.nih.gov/15564374" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Cell+Sci.&amp;title=AP-1+subunits:+Quarrel+and+harmony+among+siblings&amp;volume=117&amp;publication_year=2004&amp;pages=5965-5973&amp;pmid=15564374&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r36">36. <span class="mixed-citation">Warburton E. C., et al., <span class="ref-title">Cholinergic neurotransmission is essential for perirhinal cortical plasticity and recognition memory</span>. <span class="ref-journal">Neuron</span>
<span class="ref-vol"><strong>38</strong></span>, 987&#x02013;996 (2003). [<a href="https://pubmed.ncbi.nlm.nih.gov/12818183" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Cholinergic+neurotransmission+is+essential+for+perirhinal+cortical+plasticity+and+recognition+memory&amp;volume=38&amp;publication_year=2003&amp;pages=987-996&amp;pmid=12818183&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r37">37. <span class="mixed-citation">Binda A., et al., <span class="ref-title">A novel KCNJ2 mutation identified in an autistic proband affects the single channel properties of Kir2.1</span>. <span class="ref-journal">Front. Cell Neurosci.</span>
<span class="ref-vol"><strong>12</strong></span>, 76 (2018). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5869910/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29615871" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Cell+Neurosci.&amp;title=A+novel+KCNJ2+mutation+identified+in+an+autistic+proband+affects+the+single+channel+properties+of+Kir2.1&amp;volume=12&amp;publication_year=2018&amp;pages=76&amp;pmid=29615871&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r38">38. <span class="mixed-citation">Ambrosini E., et al., <span class="ref-title">Genetically induced dysfunctions of Kir2.1 channels: Implications for short QT3 syndrome and autism-epilepsy phenotype</span>. <span class="ref-journal">Hum. Mol. Genet.</span>
<span class="ref-vol"><strong>23</strong></span>, 4875&#x02013;4886 (2014). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4140467/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24794859" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Genetically+induced+dysfunctions+of+Kir2.1+channels:+Implications+for+short+QT3+syndrome+and+autism-epilepsy+phenotype&amp;volume=23&amp;publication_year=2014&amp;pages=4875-4886&amp;pmid=24794859&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r39">39. <span class="mixed-citation">Rubenstein J. L. R., Merzenich M. M., <span class="ref-title">Model of autism: Increased ratio of excitation/inhibition in key neural systems</span>. <span class="ref-journal">Genes Brain Behav.</span>
<span class="ref-vol"><strong>2</strong></span>, 255&#x02013;267 (2003). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6748642/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14606691" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Brain+Behav.&amp;title=Model+of+autism:+Increased+ratio+of+excitation/inhibition+in+key+neural+systems&amp;volume=2&amp;publication_year=2003&amp;pages=255-267&amp;pmid=14606691&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r40">40. <span class="mixed-citation">Renoux F., et al., <span class="ref-title">The AP1 transcription factor Fosl2 promotes systemic autoimmunity and inflammation by repressing treg development</span>. <span class="ref-journal">Cell Rep.</span>
<span class="ref-vol"><strong>31</strong></span>, 107826 (2020). [<a href="https://pubmed.ncbi.nlm.nih.gov/32610127" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Rep.&amp;title=The+AP1+transcription+factor+Fosl2+promotes+systemic+autoimmunity+and+inflammation+by+repressing+treg+development&amp;volume=31&amp;publication_year=2020&amp;pages=107826&amp;pmid=32610127&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r41">41. <span class="mixed-citation">Sweeney S. E., Kimbler T. B., Firestein G. S., <span class="ref-title">Synoviocyte innate immune responses: II. Pivotal role of IFN regulatory factor 3</span>. <span class="ref-journal">J. Immunol.</span>
<span class="ref-vol"><strong>184</strong></span>, 7162&#x02013;7168 (2010). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2913682/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20483755" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Immunol.&amp;title=Synoviocyte+innate+immune+responses:+II.+Pivotal+role+of+IFN+regulatory+factor+3&amp;volume=184&amp;publication_year=2010&amp;pages=7162-7168&amp;pmid=20483755&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r42">42. <span class="mixed-citation">Iwanaszko M., Kimmel M., <span class="ref-title">NF-&#x003ba;B and IRF pathways: Cross-regulation on target genes promoter level</span>. <span class="ref-journal">BMC Genomics</span>
<span class="ref-vol"><strong>16</strong></span>, 307 (2015). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4430024/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25888367" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genomics&amp;title=NF-&#x003ba;B+and+IRF+pathways:+Cross-regulation+on+target+genes+promoter+level&amp;volume=16&amp;publication_year=2015&amp;pages=307&amp;pmid=25888367&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r43">43. <span class="mixed-citation">Friedman N., Linial M., Nachman I., Pe&#x02019;er D., <span class="ref-title">Using Bayesian networks to analyze expression data</span>. <span class="ref-journal">J. Comput. Boil.</span>
<span class="ref-vol"><strong>7</strong></span>, 601&#x02013;620 (2000). [<a href="https://pubmed.ncbi.nlm.nih.gov/11108481" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Comput.+Boil.&amp;title=Using+Bayesian+networks+to+analyze+expression+data&amp;volume=7&amp;publication_year=2000&amp;pages=601-620&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r44">44. <span class="mixed-citation">Zhang B., et al., <span class="ref-title">Integrated systems approach identifies genetic nodes and networks in late-onset Alzheimer&#x02019;s disease</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol"><strong>153</strong></span>, 707&#x02013;720 (2013). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3677161/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23622250" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Integrated+systems+approach+identifies+genetic+nodes+and+networks+in+late-onset+Alzheimer&#x02019;s+disease&amp;volume=153&amp;publication_year=2013&amp;pages=707-720&amp;pmid=23622250&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r45">45. <span class="mixed-citation">Mostafavi S., et al., <span class="ref-title">A molecular network of the aging human brain provides insights into the pathology and cognitive decline of Alzheimer&#x02019;s disease</span>. <span class="ref-journal">Nat. Neurosci.</span>
<span class="ref-vol"><strong>21</strong></span>, 811&#x02013;819 (2018). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6599633/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29802388" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Neurosci.&amp;title=A+molecular+network+of+the+aging+human+brain+provides+insights+into+the+pathology+and+cognitive+decline+of+Alzheimer&#x02019;s+disease&amp;volume=21&amp;publication_year=2018&amp;pages=811-819&amp;pmid=29802388&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r46">46. <span class="mixed-citation">Kiss T., <span class="ref-title">Small nucleolar RNAs: An abundant group of noncoding RNAs with diverse cellular functions</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol"><strong>109</strong></span>, 145&#x02013;148 (2002). [<a href="https://pubmed.ncbi.nlm.nih.gov/12007400" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Small+nucleolar+RNAs:+An+abundant+group+of+noncoding+RNAs+with+diverse+cellular+functions&amp;volume=109&amp;publication_year=2002&amp;pages=145-148&amp;pmid=12007400&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r47">47. <span class="mixed-citation">Bratkovi&#x0010d; T., Bo&#x0017e;i&#x0010d; J., Rogelj B., <span class="ref-title">Functional diversity of small nucleolar RNAs</span>. <span class="ref-journal">Nucleic Acids Res.</span>
<span class="ref-vol"><strong>48</strong></span>, 1627&#x02013;1651 (2020). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7038934/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31828325" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Functional+diversity+of+small+nucleolar+RNAs&amp;volume=48&amp;publication_year=2020&amp;pages=1627-1651&amp;pmid=31828325&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r48">48. <span class="mixed-citation">Dvinge H., Guenthoer J., Porter P. L., Bradley R. K., <span class="ref-title">RNA components of the spliceosome regulate tissue- and cancer-specific alternative splicing</span>. <span class="ref-journal">Genome Res.</span>
<span class="ref-vol"><strong>29</strong></span>, 1591&#x02013;1604 (2019). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6771400/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31434678" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=RNA+components+of+the+spliceosome+regulate+tissue-+and+cancer-specific+alternative+splicing&amp;volume=29&amp;publication_year=2019&amp;pages=1591-1604&amp;pmid=31434678&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r49">49. <span class="mixed-citation">Valadkhan S., Gunawardane L. S., <span class="ref-title">Role of small nuclear RNAs in eukaryotic gene expression</span>. <span class="ref-journal">Essays Biochem.</span>
<span class="ref-vol"><strong>54</strong></span>, 79&#x02013;90 (2013). [<a href="https://pubmed.ncbi.nlm.nih.gov/23829528" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Essays+Biochem.&amp;title=Role+of+small+nuclear+RNAs+in+eukaryotic+gene+expression&amp;volume=54&amp;publication_year=2013&amp;pages=79-90&amp;pmid=23829528&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r50">50. <span class="mixed-citation">Kishore S., Stamm S., <span class="ref-title">The snoRNA HBII-52 regulates alternative splicing of the serotonin receptor 2C</span>. <span class="ref-journal">Science</span>
<span class="ref-vol"><strong>311</strong></span>, 230&#x02013;232 (2006). [<a href="https://pubmed.ncbi.nlm.nih.gov/16357227" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+snoRNA+HBII-52+regulates+alternative+splicing+of+the+serotonin+receptor+2C&amp;volume=311&amp;publication_year=2006&amp;pages=230-232&amp;pmid=16357227&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r51">51. <span class="mixed-citation">Falaleeva M., et al., <span class="ref-title">Dual function of C/D box small nucleolar RNAs in rRNA modification and alternative pre-mRNA splicing</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span>
<span class="ref-vol"><strong>113</strong></span>, E1625&#x02013;E1634 (2016). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4812717/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26957605" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Dual+function+of+C/D+box+small+nucleolar+RNAs+in+rRNA+modification+and+alternative+pre-mRNA+splicing&amp;volume=113&amp;publication_year=2016&amp;pages=E1625-E1634&amp;pmid=26957605&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r52">52. <span class="mixed-citation">D&#x02019;Aniello A., et al., <span class="ref-title">Amino acids and transaminases activity in ventricular CSF and in brain of normal and Alzheimer patients</span>. <span class="ref-journal">Neurosci. Lett.</span>
<span class="ref-vol"><strong>388</strong></span>, 49&#x02013;53 (2005). [<a href="https://pubmed.ncbi.nlm.nih.gov/16039064" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurosci.+Lett.&amp;title=Amino+acids+and+transaminases+activity+in+ventricular+CSF+and+in+brain+of+normal+and+Alzheimer+patients&amp;volume=388&amp;publication_year=2005&amp;pages=49-53&amp;pmid=16039064&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r53">53. <span class="mixed-citation">Stamova B. S., et al., <span class="ref-title">Evidence for differential alternative splicing in blood of young boys with autism spectrum disorders</span>. <span class="ref-journal">Mol. Autism</span>
<span class="ref-vol"><strong>4</strong></span>, 1&#x02013;16 (2013). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3846739/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24007566" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Autism&amp;title=Evidence+for+differential+alternative+splicing+in+blood+of+young+boys+with+autism+spectrum+disorders&amp;volume=4&amp;publication_year=2013&amp;pages=1-16&amp;pmid=23311570&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r54">54. <span class="mixed-citation">Araki Y., et al., <span class="ref-title">SynGAP isoforms differentially regulate synaptic plasticity and dendritic development</span>. <span class="ref-journal">Elife</span>
<span class="ref-vol"><strong>9</strong></span>, e56273 (2020). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7314543/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32579114" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Elife&amp;title=SynGAP+isoforms+differentially+regulate+synaptic+plasticity+and+dendritic+development&amp;volume=9&amp;publication_year=2020&amp;pages=e56273&amp;pmid=32579114&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r55">55. <span class="mixed-citation">Comoletti D., et al., <span class="ref-title">The Arg451Cys-neuroligin-3 mutation associated with autism reveals a defect in protein processing</span>. <span class="ref-journal">J. Neurosci.</span>
<span class="ref-vol"><strong>24</strong></span>, 4889&#x02013;4893 (2004). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6729460/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15152050" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Neurosci.&amp;title=The+Arg451Cys-neuroligin-3+mutation+associated+with+autism+reveals+a+defect+in+protein+processing&amp;volume=24&amp;publication_year=2004&amp;pages=4889-4893&amp;pmid=15152050&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r56">56. <span class="mixed-citation">Sprenkle N. T., Sims S. G., S&#x000e1;nchez C. L., Meares G. P., <span class="ref-title">Endoplasmic reticulum stress and inflammation in the central nervous system</span>. <span class="ref-journal">Mol. Neurodegener.</span>
<span class="ref-vol"><strong>12</strong></span>, 42 (2017). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5445486/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28545479" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Neurodegener.&amp;title=Endoplasmic+reticulum+stress+and+inflammation+in+the+central+nervous+system&amp;volume=12&amp;publication_year=2017&amp;pages=42&amp;pmid=28545479&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r57">57. <span class="mixed-citation">Crider A., Ahmed A. O., Pillai A., <span class="ref-title">Altered expression of endoplasmic reticulum stress-related genes in the middle frontal cortex of subjects with autism spectrum disorder</span>. <span class="ref-journal">Mol. Neuropsychiatry</span>
<span class="ref-vol"><strong>3</strong></span>, 85&#x02013;91 (2017). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5701271/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29230396" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Neuropsychiatry&amp;title=Altered+expression+of+endoplasmic+reticulum+stress-related+genes+in+the+middle+frontal+cortex+of+subjects+with+autism+spectrum+disorder&amp;volume=3&amp;publication_year=2017&amp;pages=85-91&amp;pmid=29230396&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r58">58. <span class="mixed-citation">Timberlake M. I., Dwivedi Y., <span class="ref-title">Linking unfolded protein response to inflammation and depression: Potential pathologic and therapeutic implications</span>. <span class="ref-journal">Mol Psychiatry</span>
<span class="ref-vol"><strong>24</strong></span>, 987&#x02013;994 (2019). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6416085/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30214045" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Linking+unfolded+protein+response+to+inflammation+and+depression:+Potential+pathologic+and+therapeutic+implications&amp;volume=24&amp;publication_year=2019&amp;pages=987-994&amp;pmid=30214045&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r59">59. <span class="mixed-citation">Hashemi E., Ariza J., Rogers H., Noctor S. C., Mart&#x000ed;nez-Cerde&#x000f1;o V., <span class="ref-title">The number of parvalbumin-expressing interneurons is decreased in the prefrontal cortex in autism</span>. <span class="ref-journal">Cerebral Cortex</span>
<span class="ref-vol"><strong>27</strong></span>, 1931&#x02013;1943 (2017). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6074948/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26922658" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cerebral+Cortex&amp;title=The+number+of+parvalbumin-expressing+interneurons+is+decreased+in+the+prefrontal+cortex+in+autism&amp;volume=27&amp;publication_year=2017&amp;pages=1931-1943&amp;pmid=26922658&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r60">60. <span class="mixed-citation">Blatt G. J., et al., <span class="ref-title">Density and distribution of hippocampal neurotransmitter receptors in autism: An autoradiographic study</span>. <span class="ref-journal">J. Autism Dev. Disorders</span>
<span class="ref-vol"><strong>31</strong></span>, 537&#x02013;543 (2001). [<a href="https://pubmed.ncbi.nlm.nih.gov/11814263" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Autism+Dev.+Disorders&amp;title=Density+and+distribution+of+hippocampal+neurotransmitter+receptors+in+autism:+An+autoradiographic+study&amp;volume=31&amp;publication_year=2001&amp;pages=537-543&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r61">61. <span class="mixed-citation">Oblak A. L., Gibbs T. T., Blatt G. J., <span class="ref-title">Reduced GABAA receptors and benzodiazepine binding sites in the posterior cingulate cortex and fusiform gyrus in autism</span>. <span class="ref-journal">Brain Res.</span>
<span class="ref-vol"><strong>1380</strong></span>, 218&#x02013;228 (2011). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3020259/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20858465" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain+Res.&amp;title=Reduced+GABAA+receptors+and+benzodiazepine+binding+sites+in+the+posterior+cingulate+cortex+and+fusiform+gyrus+in+autism&amp;volume=1380&amp;publication_year=2011&amp;pages=218-228&amp;pmid=20858465&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r62">62. <span class="mixed-citation">Oblak A. L., Gibbs T. T., Blatt G. J., <span class="ref-title">Decreased GABAB receptors in the cingulate cortex and fusiform gyrus in autism</span>. <span class="ref-journal">J. Neurochem.</span>
<span class="ref-vol"><strong>114</strong></span>, 1414&#x02013;1423 (2010). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2923229/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20557420" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Neurochem.&amp;title=Decreased+GABAB+receptors+in+the+cingulate+cortex+and+fusiform+gyrus+in+autism&amp;volume=114&amp;publication_year=2010&amp;pages=1414-1423&amp;pmid=20557420&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r63">63. <span class="mixed-citation">Qureshi T., et al., <span class="ref-title">The glutamine transporter Slc38a1 regulates GABAergic neurotransmission and synaptic plasticity</span>. <span class="ref-journal">Cereb. Cortex</span>
<span class="ref-vol"><strong>29</strong></span>, 5166&#x02013;5179 (2019). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6918930/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31050701" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cereb.+Cortex&amp;title=The+glutamine+transporter+Slc38a1+regulates+GABAergic+neurotransmission+and+synaptic+plasticity&amp;volume=29&amp;publication_year=2019&amp;pages=5166-5179&amp;pmid=31050701&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r64">64. <span class="mixed-citation">Jin L.-W., et al., <span class="ref-title">Dysregulation of glutamine transporter SNAT1 in Rett syndrome microglia: A mechanism for mitochondrial dysfunction and neurotoxicity</span>. <span class="ref-journal">J. Neurosci.</span>
<span class="ref-vol"><strong>35</strong></span>, 2516&#x02013;2529 (2015). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4323531/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25673846" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Neurosci.&amp;title=Dysregulation+of+glutamine+transporter+SNAT1+in+Rett+syndrome+microglia:+A+mechanism+for+mitochondrial+dysfunction+and+neurotoxicity&amp;volume=35&amp;publication_year=2015&amp;pages=2516-2529&amp;pmid=25673846&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r65">65. <span class="mixed-citation">Walle L. V., Lamkanfi M., Vandenabeele P., <span class="ref-title">The mitochondrial serine protease HtrA2/Omi: An overview</span>. <span class="ref-journal">Cell Death Differ.</span>
<span class="ref-vol"><strong>15</strong></span>, 453&#x02013;460 (2008). [<a href="https://pubmed.ncbi.nlm.nih.gov/18174901" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Death+Differ.&amp;title=The+mitochondrial+serine+protease+HtrA2/Omi:+An+overview&amp;volume=15&amp;publication_year=2008&amp;pages=453-460&amp;pmid=18174901&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r66">66. <span class="mixed-citation">Su X. J., Huang L., Qu Y., D., <span class="ref-title">Mu Progress in research on the role of Omi/HtrA2 in neurological diseases</span>. <span class="ref-journal">Rev. Neurosci.</span>
<span class="ref-vol"><strong>30</strong></span>, 279&#x02013;287 (2019). [<a href="https://pubmed.ncbi.nlm.nih.gov/30205651" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Rev.+Neurosci.&amp;title=Mu+Progress+in+research+on+the+role+of+Omi/HtrA2+in+neurological+diseases&amp;volume=30&amp;publication_year=2019&amp;pages=279-287&amp;pmid=30205651&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r67">67. <span class="mixed-citation">Plun-Favreau H., et al., <span class="ref-title">The mitochondrial protease HtrA2 is regulated by Parkinson&#x02019;s disease-associated kinase PINK1</span>. <span class="ref-journal">Nat Cell Biol.</span>
<span class="ref-vol"><strong>9</strong></span>, 1243&#x02013;1252 (2007). [<a href="https://pubmed.ncbi.nlm.nih.gov/17906618" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Cell+Biol.&amp;title=The+mitochondrial+protease+HtrA2+is+regulated+by+Parkinson&#x02019;s+disease-associated+kinase+PINK1&amp;volume=9&amp;publication_year=2007&amp;pages=1243-1252&amp;pmid=17906618&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r68">68. <span class="mixed-citation">Darreh-Shori T., et al., <span class="ref-title">Increased active OMI/HTRA2 serine protease displays a positive correlation with cholinergic alterations in the Alzheimer&#x02019;s disease brain</span>. <span class="ref-journal">Mol. Neurobiol.</span>
<span class="ref-vol"><strong>56</strong></span>, 4601&#x02013;4619 (2019). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6657433/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30361890" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Neurobiol.&amp;title=Increased+active+OMI/HTRA2+serine+protease+displays+a+positive+correlation+with+cholinergic+alterations+in+the+Alzheimer&#x02019;s+disease+brain&amp;volume=56&amp;publication_year=2019&amp;pages=4601-4619&amp;pmid=30361890&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r69">69. <span class="mixed-citation">Kawada K., Kuramoto N., Mimori S., <span class="ref-title">Possibility that the onset of autism spectrum disorder is induced by failure of the glutamine-glutamate cycle</span>. <span class="ref-journal">Curr. Mol. Pharmacol.</span>
<span class="ref-vol"><strong>14</strong></span>, 170&#x02013;174 (2021). [<a href="https://pubmed.ncbi.nlm.nih.gov/32189600" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Mol.+Pharmacol.&amp;title=Possibility+that+the+onset+of+autism+spectrum+disorder+is+induced+by+failure+of+the+glutamine-glutamate+cycle&amp;volume=14&amp;publication_year=2021&amp;pages=170-174&amp;pmid=32189600&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r70">70. <span class="mixed-citation">Ecker C., Bookheimer S. Y., Murphy D. G., <span class="ref-title">Neuroimaging in autism spectrum disorder: Brain structure and function across the lifespan</span>. <span class="ref-journal">Lancet Neurol.</span>
<span class="ref-vol"><strong>14</strong></span>, 1121&#x02013;1134 (2015). [<a href="https://pubmed.ncbi.nlm.nih.gov/25891007" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol.&amp;title=Neuroimaging+in+autism+spectrum+disorder:+Brain+structure+and+function+across+the+lifespan&amp;volume=14&amp;publication_year=2015&amp;pages=1121-1134&amp;pmid=25891007&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r71">71. <span class="mixed-citation">Avino T. A., et al., <span class="ref-title">Neuron numbers increase in the human amygdala from birth to adulthood, but not in autism</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span>
<span class="ref-vol"><strong>115</strong></span>, 3710&#x02013;3715 (2018). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5889677/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29559529" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Neuron+numbers+increase+in+the+human+amygdala+from+birth+to+adulthood,+but+not+in+autism&amp;volume=115&amp;publication_year=2018&amp;pages=3710-3715&amp;pmid=29559529&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r72">72. <span class="mixed-citation">Ander B. P., Barger N., Stamova B., Sharp F. R., Schumann C. M., <span class="ref-title">Atypical miRNA expression in temporal cortex associated with dysregulation of immune, cell cycle, and other pathways in autism spectrum disorders</span>. <span class="ref-journal">Mol. Autism</span>
<span class="ref-vol"><strong>6</strong></span>, 1&#x02013;13 (2015). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4491207/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26146533" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Autism&amp;title=Atypical+miRNA+expression+in+temporal+cortex+associated+with+dysregulation+of+immune,+cell+cycle,+and+other+pathways+in+autism+spectrum+disorders&amp;volume=6&amp;publication_year=2015&amp;pages=1-13&amp;pmid=25705365&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r73">73. <span class="mixed-citation">Stamova B., Ander B. P., Barger N., Sharp F. R., Schumann C. M., <span class="ref-title">Specific regional and age-related small noncoding RNA expression patterns within superior temporal gyrus of typical human brains are less distinct in autism brains</span>. <span class="ref-journal">J. Child Neurol.</span>
<span class="ref-vol"><strong>30</strong></span>, 1930&#x02013;1946 (2015). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4647182/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26350727" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Child+Neurol.&amp;title=Specific+regional+and+age-related+small+noncoding+RNA+expression+patterns+within+superior+temporal+gyrus+of+typical+human+brains+are+less+distinct+in+autism+brains&amp;volume=30&amp;publication_year=2015&amp;pages=1930-1946&amp;pmid=26350727&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r74">74. <span class="mixed-citation">Garbett K., et al., <span class="ref-title">Immune transcriptome alterations in the temporal cortex of subjects with autism</span>. <span class="ref-journal">Neurobiol. Dis.</span>
<span class="ref-vol"><strong>30</strong></span>, 303&#x02013;311 (2008). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2693090/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18378158" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurobiol.+Dis.&amp;title=Immune+transcriptome+alterations+in+the+temporal+cortex+of+subjects+with+autism&amp;volume=30&amp;publication_year=2008&amp;pages=303-311&amp;pmid=18378158&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r75">75. <span class="mixed-citation">Lee A. S., Azmitia E. C., Whitaker-Azmitia P. M., <span class="ref-title">Developmental microglial priming in postmortem autism spectrum disorder temporal cortex</span>. <span class="ref-journal">Brain Behav. Immun.</span>
<span class="ref-vol"><strong>62</strong></span>, 193&#x02013;202 (2017). [<a href="https://pubmed.ncbi.nlm.nih.gov/28159644" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain+Behav.+Immun.&amp;title=Developmental+microglial+priming+in+postmortem+autism+spectrum+disorder+temporal+cortex&amp;volume=62&amp;publication_year=2017&amp;pages=193-202&amp;pmid=28159644&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r76">76. <span class="mixed-citation">Vivanti G., Tao S., Lyall K., Robins D. L., Shea L. L., <span class="ref-title">The prevalence and incidence of early-onset dementia among adults with autism spectrum disorder</span>. <span class="ref-journal">Autism Res.</span>
<span class="ref-vol"><strong>14</strong></span>, 2189&#x02013;2199 (2021). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8487995/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34378867" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Autism+Res.&amp;title=The+prevalence+and+incidence+of+early-onset+dementia+among+adults+with+autism+spectrum+disorder&amp;volume=14&amp;publication_year=2021&amp;pages=2189-2199&amp;pmid=34378867&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r77">77. <span class="mixed-citation">Hand B. N., Angell A. M., Harris L., Carpenter L. A., <span class="ref-title">Prevalence of physical and mental health conditions in Medicare-enrolled, autistic older adults</span>. <span class="ref-journal">Autism</span>
<span class="ref-vol"><strong>24</strong></span>, 755&#x02013;764 (2020). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7433648/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31773968" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Autism&amp;title=Prevalence+of+physical+and+mental+health+conditions+in+Medicare-enrolled,+autistic+older+adults&amp;volume=24&amp;publication_year=2020&amp;pages=755-764&amp;pmid=31773968&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r78">78. <span class="mixed-citation">Oldham M. C., Langfelder P., Horvath S., <span class="ref-title">Network methods for describing sample relationships in genomic datasets: Application to Huntington&#x02019;s disease</span>. <span class="ref-journal">BMC Syst. Biol.</span>
<span class="ref-vol"><strong>6</strong></span>, 63 (2012). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3441531/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22691535" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Syst.+Biol.&amp;title=Network+methods+for+describing+sample+relationships+in+genomic+datasets:+Application+to+Huntington&#x02019;s+disease&amp;volume=6&amp;publication_year=2012&amp;pages=63&amp;pmid=22691535&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r79">79. <span class="mixed-citation">Love M. I., Huber W., Anders S., <span class="ref-title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</span>. <span class="ref-journal">Genome Biol.</span>
<span class="ref-vol"><strong>15</strong></span>, 1&#x02013;21 (2014). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4302049/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25516281" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol.&amp;title=Moderated+estimation+of+fold+change+and+dispersion+for+RNA-seq+data+with+DESeq2&amp;volume=15&amp;publication_year=2014&amp;pages=1-21&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r80">80. <span class="mixed-citation">Raudvere U., et al., <span class="ref-title">g: Profiler: A web server for functional enrichment analysis and conversions of gene lists (2019 update)</span>. <span class="ref-journal">Nucleic Acids Res.</span>
<span class="ref-vol"><strong>47</strong></span>, W191&#x02013;W198 (2019). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6602461/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31066453" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=g:+Profiler:+A+web+server+for+functional+enrichment+analysis+and+conversions+of+gene+lists+(2019+update)&amp;volume=47&amp;publication_year=2019&amp;pages=W191-W198&amp;pmid=31066453&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r81">81. <span class="mixed-citation">Yu G., Wang L.-G., Han Y., Q.-Y., <span class="ref-title">He, clusterProfiler: An R package for comparing biological themes among gene clusters</span>. <span class="ref-journal">OMICS</span>
<span class="ref-vol"><strong>16</strong></span>, 284&#x02013;287 (2012). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3339379/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22455463" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=OMICS&amp;title=He,+clusterProfiler:+An+R+package+for+comparing+biological+themes+among+gene+clusters&amp;volume=16&amp;publication_year=2012&amp;pages=284-287&amp;pmid=22455463&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r82">82. <span class="mixed-citation">Skene N. G., Grant S. G., <span class="ref-title">Identification of vulnerable cell types in major brain disorders using single cell transcriptomes and expression weighted cell type enrichment</span>. <span class="ref-journal">Front Neurosci.</span>
<span class="ref-vol"><strong>10</strong></span>, 16 (2016). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4730103/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26858593" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front+Neurosci.&amp;title=Identification+of+vulnerable+cell+types+in+major+brain+disorders+using+single+cell+transcriptomes+and+expression+weighted+cell+type+enrichment&amp;volume=10&amp;publication_year=2016&amp;pages=16&amp;pmid=26858593&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r83">83. <span class="mixed-citation">Finucane H. K., et al., <span class="ref-title">Partitioning heritability by functional annotation using genome-wide association summary statistics</span>. <span class="ref-journal">Nat. Genet.</span>
<span class="ref-vol"><strong>47</strong></span>, 1228&#x02013;1235 (2015). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4626285/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26414678" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Partitioning+heritability+by+functional+annotation+using+genome-wide+association+summary+statistics&amp;volume=47&amp;publication_year=2015&amp;pages=1228-1235&amp;pmid=26414678&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r84">84. <span class="mixed-citation">Kr&#x000e4;mer A., Green J., Pollard J. Jr., Tugendreich S., <span class="ref-title">Causal analysis approaches in ingenuity pathway analysis</span>. <span class="ref-journal">Bioinformatics</span>
<span class="ref-vol"><strong>30</strong></span>, 523&#x02013;530 (2014). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3928520/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24336805" ref="reftype=pubmed&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Causal+analysis+approaches+in+ingenuity+pathway+analysis&amp;volume=30&amp;publication_year=2014&amp;pages=523-530&amp;pmid=24336805&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r85">85. <span class="mixed-citation">Zhang P., Gandal M. J., <span class="ref-title">ASD_STG_LCM_RNAseq_paper_code</span>. <span class="ref-journal">Github</span>. <a href="https://github.com/gandallab/ASD_STG_LCM_RNAseq" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/gandallab/ASD_STG_LCM_RNAseq</a>. Deposited 27 January 2023.</span></div><div class="ref-cit-blk half_rhythm" id="r86">86. <span class="mixed-citation">Omanska A., Stamova B., Schumann C. M., Ander B. P., <span class="ref-title">Transcriptomic profiling of bulk tissue and laser-capture micro dissected neurons of postmortem human brains in the superior temporal gyrus</span>. <span class="ref-journal">NCBI&#x02019;s Database of Genotypes and Phenotypes (dbGaP)</span>. <a href="https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs003208.v1.p1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs003208.v1.p1</a> (Deposited 17 February 2023). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=NCBI&#x02019;s+Database+of+Genotypes+and+Phenotypes+(dbGaP)&amp;title=Transcriptomic+profiling+of+bulk+tissue+and+laser-capture+micro+dissected+neurons+of+postmortem+human+brains+in+the+superior+temporal+gyrus&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10013873&amp;issue-id=431279&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Proceedings of the National Academy of Sciences of the United States of America</span> are provided here courtesy of <strong>National Academy of Sciences</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC10013873/pdf/pnas.202206758.pdf" class="int-view">PDF (22M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/10013873/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/10013873/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="10013873" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10013873%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="rHKZUskd5aR9H6HsUnzDFb3MwdraIWHFvGCAoUEI8gMWmboLTORMLW4oj1yFKyxX">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10013873%2F&amp;text=Neuron-specific%20transcriptomic%20signatures%20indicate%20neuroinflammation%20and%20altered%20neuronal%20activity%20in%20ASD%20temporal%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10013873%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10013873/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC10013873/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/36862688/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC10013873/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/36862688/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC10013873/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/10013873/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC10013873/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
