
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs2.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Yeast zinc cluster transcription factors involved in the switch from fermentation to respiration show interdependency for DNA binding revealing a novel type of DNA recognition - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="D0BD41E4AC192A450000617E2B70557B.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="nar" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC10954478/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10954478/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Nucleic Acids Research" /><meta name="citation_title" content="Yeast zinc cluster transcription factors involved in the switch from fermentation to respiration show interdependency for DNA binding revealing a novel type of DNA recognition" /><meta name="citation_author" content="Karla P&#x000e1;ez Martinez" /><meta name="citation_author_institution" content="Department of Medicine, McGill University Health Centre, 1001 Boul. D&#x000e9;carie, Room E02.7212, Montr&#x000e9;al, QC H4A 3J1,&#x000a0;Canada" /><meta name="citation_author" content="Najla Gasmi" /><meta name="citation_author_institution" content="Department of Biochemistry, McGill University, 1001 Boul. D&#x000e9;carie, Room E02.7212, Montr&#x000e9;al, QC H4A 3J1,&#x000a0;Canada" /><meta name="citation_author" content="C&#x000e9;lia Jeronimo" /><meta name="citation_author_institution" content="Institut de recherches cliniques de Montr&#x000e9;al, 110 avenue des Pins Ouest, Montr&#x000e9;al, QC H2W 1R7, Canada" /><meta name="citation_author" content="Natalia Klimova" /><meta name="citation_author_institution" content="Department of Medicine, McGill University Health Centre, 1001 Boul. D&#x000e9;carie, Room E02.7212, Montr&#x000e9;al, QC H4A 3J1,&#x000a0;Canada" /><meta name="citation_author" content="Fran&#x000e7;ois Robert" /><meta name="citation_author_institution" content="Institut de recherches cliniques de Montr&#x000e9;al, 110 avenue des Pins Ouest, Montr&#x000e9;al, QC H2W 1R7, Canada" /><meta name="citation_author_institution" content="D&#x000e9;partement de M&#x000e9;decine, Facult&#x000e9; de M&#x000e9;decine, Universit&#x000e9; de Montr&#x000e9;al, 2900 Boul. &#x000c9;douard-Montpetit, Montr&#x000e9;al, QC H3T 1J4, Canada" /><meta name="citation_author" content="Bernard Turcotte" /><meta name="citation_author_institution" content="Department of Medicine, McGill University Health Centre, 1001 Boul. D&#x000e9;carie, Room E02.7212, Montr&#x000e9;al, QC H4A 3J1,&#x000a0;Canada" /><meta name="citation_author_institution" content="Department of Biochemistry, McGill University, 1001 Boul. D&#x000e9;carie, Room E02.7212, Montr&#x000e9;al, QC H4A 3J1,&#x000a0;Canada" /><meta name="citation_author_institution" content="Department of Microbiology and Immunology, McGill University, 1001 Boul. D&#x000e9;carie, Room E02.7212, Montr&#x000e9;al, QC H4A 3J1,&#x000a0;Canada" /><meta name="citation_publication_date" content="2024/03/03" /><meta name="citation_issue" content="5" /><meta name="citation_volume" content="52" /><meta name="citation_firstpage" content="2242" /><meta name="citation_doi" content="10.1093/nar/gkad1185" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC10954478/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC10954478/" /><meta name="citation_pmid" content="38109318" /><meta name="DC.Title" content="Yeast zinc cluster transcription factors involved in the switch from fermentation to respiration show interdependency for DNA binding revealing a novel type of DNA recognition" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Oxford University Press" /><meta name="DC.Contributor" content="Karla P&#x000e1;ez Martinez" /><meta name="DC.Contributor" content="Najla Gasmi" /><meta name="DC.Contributor" content="C&#x000e9;lia Jeronimo" /><meta name="DC.Contributor" content="Natalia Klimova" /><meta name="DC.Contributor" content="Fran&#x000e7;ois Robert" /><meta name="DC.Contributor" content="Bernard Turcotte" /><meta name="DC.Date" content="2024 Mar 21" /><meta name="DC.Identifier" content="10.1093/nar/gkad1185" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Yeast zinc cluster transcription factors involved in the switch from fermentation to respiration show interdependency for DNA binding revealing a novel type of DNA recognition" /><meta property="og:type" content="article" /><meta property="og:description" content="In budding yeast, fermentation is the most important pathway for energy production. Under low-glucose conditions, ethanol is used for synthesis of this sugar requiring a shift to respiration. This process is controlled by the transcriptional regulators ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10954478/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10954478/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/4/">Nucleic Acids Res</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/458308/">v.52(5); 2024 Mar 21</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC10954478
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC10954478/pdf/gkad1185.pdf" class="int-view">PDF (3.2M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/10954478/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/10954478/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="10954478" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10954478%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="F2wJnaTm2OlpAqie8vK3spdMzrTmYZyrOOBZKu5SlDLDY4mcyzGcv8AB9NKjezrp">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10954478%2F&amp;text=Yeast%20zinc%20cluster%20transcription%20factors%20involved%20in%20the%20switch%20from%20fermentation%20to%20respiration%20show%20interdependency%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10954478%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10954478/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC10954478/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38109318/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC10954478/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38109318/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC10954478/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/10954478/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/4/">Nucleic Acids Res</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/458308/">v.52(5); 2024 Mar 21</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC10954478
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/narlogo.gif" alt="Logo of nar" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/nar" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Nucleic Acids Res.</span> 2024 Mar 21; 52(5): 2242–2259. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2023 Dec 18. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1093%2Fnar%2Fgkad1185" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/nar/gkad1185</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC10954478</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38109318">38109318</a></div></div></div><h1 class="content-title">Yeast zinc cluster transcription factors involved in the switch from fermentation to respiration show interdependency for DNA binding revealing a novel type of DNA recognition</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Martinez%20KP%5BAuthor%5D" class="affpopup" co-rid="_co_idm140432045945968" co-class="co-affbox">Karla P&#x000e1;ez Martinez</a>,<sup></sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gasmi%20N%5BAuthor%5D" class="affpopup" co-rid="_co_idm140432045942752" co-class="co-affbox">Najla Gasmi</a>,<sup></sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Jeronimo%20C%5BAuthor%5D" class="affpopup" co-rid="_co_idm140432039733856" co-class="co-affbox">C&#x000e9;lia Jeronimo</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Klimova%20N%5BAuthor%5D" class="affpopup" co-rid="_co_idm140432090502320" co-class="co-affbox">Natalia Klimova</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Robert%20F%5BAuthor%5D" class="affpopup" co-rid="_co_idm140432090499776" co-class="co-affbox">Fran&#x000e7;ois Robert</a>, and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Turcotte%20B%5BAuthor%5D" class="affpopup" co-rid="_co_idm140432090495456" co-class="co-affbox">Bernard Turcotte</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup></sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140432045945968"><h3 class="no_margin">Karla P&#x000e1;ez Martinez</h3><p>
Department of Medicine, McGill University Health Centre, 1001 Boul. D&#x000e9;carie, Room E02.7212, Montr&#x000e9;al, QC H4A 3J1,&#x000a0;Canada</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Martinez%20KP%5BAuthor%5D">Karla P&#x000e1;ez Martinez</a></div></div><div id="_co_idm140432045942752"><h3 class="no_margin">Najla Gasmi</h3><p>
Department of Biochemistry, McGill University, 1001 Boul. D&#x000e9;carie, Room E02.7212, Montr&#x000e9;al, QC H4A 3J1,&#x000a0;Canada</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gasmi%20N%5BAuthor%5D">Najla Gasmi</a></div></div><div id="_co_idm140432039733856"><h3 class="no_margin">C&#x000e9;lia Jeronimo</h3><p>
Institut de recherches cliniques de Montr&#x000e9;al, 110 avenue des Pins Ouest, Montr&#x000e9;al, QC H2W 1R7, Canada</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Jeronimo%20C%5BAuthor%5D">C&#x000e9;lia Jeronimo</a></div></div><div id="_co_idm140432090502320"><h3 class="no_margin">Natalia Klimova</h3><p>
Department of Medicine, McGill University Health Centre, 1001 Boul. D&#x000e9;carie, Room E02.7212, Montr&#x000e9;al, QC H4A 3J1,&#x000a0;Canada</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Klimova%20N%5BAuthor%5D">Natalia Klimova</a></div></div><div id="_co_idm140432090499776"><h3 class="no_margin">Fran&#x000e7;ois Robert</h3><p>
Institut de recherches cliniques de Montr&#x000e9;al, 110 avenue des Pins Ouest, Montr&#x000e9;al, QC H2W 1R7, Canada</p><p>
D&#x000e9;partement de M&#x000e9;decine, Facult&#x000e9; de M&#x000e9;decine, Universit&#x000e9; de Montr&#x000e9;al, 2900 Boul. &#x000c9;douard-Montpetit, Montr&#x000e9;al, QC H3T 1J4, Canada</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Robert%20F%5BAuthor%5D">Fran&#x000e7;ois Robert</a></div></div><div id="_co_idm140432090495456"><h3 class="no_margin">Bernard Turcotte</h3><p>
Department of Medicine, McGill University Health Centre, 1001 Boul. D&#x000e9;carie, Room E02.7212, Montr&#x000e9;al, QC H4A 3J1,&#x000a0;Canada</p><p>
Department of Biochemistry, McGill University, 1001 Boul. D&#x000e9;carie, Room E02.7212, Montr&#x000e9;al, QC H4A 3J1,&#x000a0;Canada</p><p>
Department of Microbiology and Immunology, McGill University, 1001 Boul. D&#x000e9;carie, Room E02.7212, Montr&#x000e9;al, QC H4A 3J1,&#x000a0;Canada</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Turcotte%20B%5BAuthor%5D">Bernard Turcotte</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140432051385056_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140432051385056_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140432051385056_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140432051385056_ai" style="display:none"><div><span class="fm-affl">Karla P&#x000e1;ez Martinez, </span><span class="fm-affl" id="aff-a.v.b.f.a.b"> 
Department of Medicine, McGill University Health Centre, 1001 Boul. D&#x000e9;carie, Room E02.7212, Montr&#x000e9;al, QC H4A 3J1,&#x000a0;Canada;</span></div><div><a href="#article-aaff-info">Contributor Information</a>.</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup>Corresponding author.</div><div id="COR1">To whom correspondence should be addressed. Tel: +1&#x000a0;514 934 1934 (Ext 35842); Email: <a href="mailto:dev@null" data-email="ac.lligcm@ettocrut.dranreB" class="oemail">ac.lligcm@ettocrut.dranreB</a></div><div id="FN1">The first two authors should be regarded as Joint First Authors.</div></div><div class="fm-article-notes hide half_rhythm" id="idm140432051385056_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2023 Mar 27; Revised 2023 Nov 22; Accepted 2023 Nov 29.</div></div><div class="permissions half_rhythm hide" id="idm140432051385056_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; The Author(s) 2023. Published by Oxford University Press on behalf of Nucleic Acids Research.</div><div class="license half_rhythm">This is an Open Access article distributed under the terms of the Creative Commons Attribution-NonCommercial License (<a href="https://creativecommons.org/licenses/by-nc/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">https://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact <a href="mailto:dev@null" data-email="moc.puo@snoissimrep.slanruoj" class="oemail">moc.puo@snoissimrep.slanruoj</a></div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>gkad1185_Supplemental_File.</strong></div><div><a href="/pmc/articles/PMC10954478/bin/gkad1185_supplemental_file.pdf" data-ga-action="click_feat_suppl">gkad1185_supplemental_file.pdf</a><span style="color:gray"> (199K)</span></div><div class="small guid">GUID: 99843948-E002-4597-AE83-E15F4B8A3728</div></div></dd></dl><dl data-length="179" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first-last">All microarray data have been deposited into Gene Expression Omnibus archive under accession number <a class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE225348" ref="/geo/query/acc.cgi?acc=GSE225348"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE225348","term_id":"225348"}}</span>GSE225348</a>.</p></dd></dl></div><div id="abstract-a.v.b.r" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.v.b.rtitle" style="text-transform: uppercase;">Abstract</h2><!--article-meta--><div><p class="p p-first-last">In budding yeast, fermentation is the most important pathway for energy production. Under low-glucose conditions, ethanol is used for synthesis of this sugar requiring a shift to respiration. This process is controlled by the transcriptional regulators Cat8, Sip4, Rds2 and Ert1. We characterized Gsm1 (glucose starvation modulator 1), a paralog of Rds2 and Ert1. Genome-wide analysis showed that Gsm1 has a DNA binding profile highly similar to Rds2. Binding of Gsm1 and Rds2 is interdependent at the gluconeogenic gene <em>FBP1</em>. However, Rds2 is required for Gsm1 to bind at other promoters but not the reverse. Gsm1 and Rds2 also bind to DNA independently of each other. Western blot analysis revealed that Rds2 controls expression of Gsm1. In addition, we showed that the DNA binding domains of Gsm1 and Rds2 bind cooperatively <em>in vitro</em> to the <em>FBP1</em> promoter. In contrast, at the <em>HAP4</em> gene, Ert1 cooperates with Rds2 for DNA binding. Mutational analysis suggests that Gsm1/Rds2 and Ert1/Rds2 bind to short common DNA stretches, revealing a novel mode of binding for this class of factors. Two-point mutations in a <em>HAP4</em> site convert it to a Gsm1 binding site. Thus, Rds2 controls binding of Gsm1 at many promoters by two different mechanisms: regulation of Gsm1 levels and increased DNA binding by formation of heterodimers.</p></div></div><div id="abstract-a.v.b.s" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.v.b.stitle" style="text-transform: uppercase;">Graphical Abstract</h2><!--article-meta--><div><p class="p p-first-last">
<!--fig ft0--><!--fig mode=article f1--></p><div class="fig iconblock whole_rhythm" id="ga1" co-legend-rid="lgnd_ga1"><a href="/pmc/articles/PMC10954478/figure/ga1/" target="figure" rid-figpopup="ga1" rid-ob="ob-ga1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140432051133072"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10954478_gkad1185figgra1.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkad1185figgra1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10954478/bin/gkad1185figgra1.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140432051133072"><a target="object" rel="noopener" href="/pmc/articles/PMC10954478/figure/ga1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_ga1"><div><a class="figpopup" href="/pmc/articles/PMC10954478/figure/ga1/" target="figure" rid-figpopup="ga1" rid-ob="ob-ga1">Graphical Abstract</a></div></div></div><p>
</p></div></div><div id="SEC1" class="tsec sec"><h2 class="head no_bottom_margin" id="SEC1title" style="text-transform: uppercase;">Introduction</h2><p class="p p-first">In the budding yeast <em>Saccharomyces cerevisiae</em>, glucose is the carbon source of choice and fermentation is the major pathway for energy production, even under aerobic conditions (Crabtree effect). However, with glucose becoming scarce, ethanol produced during fermentation is used as a carbon source, a process requiring a shift to a respiration mode. Other nonfermentable carbon sources, such as lactate, acetate or glycerol, can also be used [for a review, see (<a href="#B1" rid="B1" class=" bibr popnode">1</a>)]. These gluconeogenic carbons ultimately are incorporated into the gluconeogenic pathway and used for the production of glucose 6-phosphate, an essential metabolic precursor for numerous cellular processes (<a href="#B2" rid="B2" class=" bibr popnode">2</a>). The shift from fermentative to nonfermentative metabolism results in profound reprogramming of gene expression (<a href="#B3" rid="B3" class=" bibr popnode">3</a>,<a href="#B4" rid="B4" class=" bibr popnode">4</a>). For instance, increased expression of genes for gluconeogenesis and the glyoxylate cycle is observed following a shift to ethanol, while expression of some fermentation genes is reduced under these conditions. In agreement with these changes of gene expression, glucose starvation results in a shift of histone acetylation marks from growth promoting genes to gluconeogenic and fatty acid metabolism genes (<a href="#B5" rid="B5" class=" bibr popnode">5</a>).</p><p>Regulation of the expression of gluconeogenic genes is mainly controlled by a subclass of zinc finger proteins, the zinc cluster proteins (<a href="#B6" rid="B6" class=" bibr popnode">6</a>). These transcriptional regulators are unique to fungi and amoeba and are characterized by the presence of a highly conserved cysteine-rich motif involved in DNA recognition. The cysteine residues coordinate folding of the DNA binding domain through binding to two zinc atoms. The DNA binding domain also contains a variable linker region involved in the DNA binding specificity and a dimerization region consisting of a coiled coil that mediates protein&#x02013;protein interactions for homo- or heterodimerization (<a href="#B6" rid="B6" class=" bibr popnode">6</a>). Zinc cluster proteins preferentially recognize CGG triplets arranged as inverted (CGGN<sub>X</sub>CCG), direct (CGGN<sub>X</sub>CGG) or everted repeats (CCGN<sub>X</sub>CGG) (<a href="#B6" rid="B6" class=" bibr popnode">6</a>). For example, Gal4 binds to an inverted repeat consisting of two CGG triplets spaced by 11 bp (CGGN<sub>11</sub>CCG) (<a href="#B7" rid="B7" class=" bibr popnode">7</a>), while Hap1 recognizes a direct repeat (CGGN<sub>6</sub>CGG) (<a href="#B8" rid="B8" class=" bibr popnode">8&#x02013;10</a>). Leu3 and Pdr3 recognize an everted repeat with a spacing of 4 bp (CCGN<sub>4</sub>CGG) or 0 bp (CCGCGG), respectively (<a href="#B11" rid="B11" class=" bibr popnode">11</a>,<a href="#B12" rid="B12" class=" bibr popnode">12</a>). A dimerization domain consisting of a coiled coil allows high-affinity DNA binding of these homodimers. Variation in the spacing (&#x02018;N<sub>X</sub>&#x02019;) between the CGG triplets further increases the repertoire of binding sites (<a href="#B6" rid="B6" class=" bibr popnode">6</a>). Finally, binding of zinc cluster proteins as monomers has also been reported (<a href="#B13" rid="B13" class=" bibr popnode">13</a>).</p><p>The master regulator Snf1 (<a href="#B14" rid="B14" class=" bibr popnode">14&#x02013;17</a>), a homolog of AMP-activated protein kinases, is activated by low-glucose conditions leading to the phosphorylation of a number of substrates, including DNA binding proteins such as Mig1 and the zinc cluster proteins Cat8, Sip4 and Rds2 (<a href="#B1" rid="B1" class=" bibr popnode">1</a>). Inactivation of the Mig1 repressor by phosphorylation results in derepression of a number of target genes such as <em>CAT8</em>. Activation of Cat8 and Sip4 by phosphorylation results in upregulation of the expression of gluconeogenic and glyoxylate cycle genes. Both Cat8 and Sip4 bind to carbon source response elements (CSREs) found in the promoters of target genes such as <em>PCK1</em> and <em>FBP1</em> encoding the key gluconeogenic enzymes phosphoenolpyruvate carboxykinase and fructose 1,6-bisphosphatase, respectively (<a href="#B18" rid="B18" class=" bibr popnode">18&#x02013;20</a>). Cat8 also controls the expression of <em>SIP4</em> (<a href="#B20" rid="B20" class=" bibr popnode">20</a>). Furthermore, the zinc finger (C<sub>2</sub>H<sub>2</sub> motif) activator Adr1 controls the expression of genes involved in ethanol utilization and peroxisomal proliferation (<a href="#B21" rid="B21" class=" bibr popnode">21</a>).</p><p>The zinc cluster genes <em>RDS2</em>, <em>ERT1</em> and <em><a id="__tag_1145552253" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSM1" ref="/geo/query/acc.cgi?acc=GSM1"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSM1","term_id":"1"}}</span>GSM1</a></em> encode highly related paralogs. For example, using the full-length Gsm1 polypeptide as a query for a BLAST search, <em>E</em>-values are 8&#x000a0;&#x000d7;&#x000a0;10<sup>&#x02212;26</sup> and 2&#x000a0;&#x000d7;&#x000a0;10<sup>&#x02212;14</sup> for Ert1 and Rds2, respectively. By performing phenotypic studies on zinc cluster genes, we identified a number of phenotypes associated with a deletion of the open reading frame (ORF)&#x000a0;<em>RDS2</em> such as a lack of growth with a nonfermentable carbon source (a phenotype related to the strain background) (<a href="#B22" rid="B22" class=" bibr popnode">22</a>,<a href="#B23" rid="B23" class=" bibr popnode">23</a>). Chromatin immunoprecipition and DNA microarray (ChIP-chip) experiments showed that Rds2 binds to only 7 promoters with cells grown in the presence of glucose, while it binds to many additional genes (total of 43) when ethanol is used as a carbon source (<a href="#B24" rid="B24" class=" bibr popnode">24</a>). Strikingly, the genes bound by Rds2 are involved in gluconeogenesis (e.g. <em>PCK1</em>) and related pathways such as the glyoxylate shunt and the tricarboxylic acid cycle. Importantly, we showed that Rds2 acts as a transcriptional activator of gluconeogenic genes, while it is a repressor of the negative regulators of gluconeogenesis (<a href="#B24" rid="B24" class=" bibr popnode">24</a>). Rds2 also controls expression of the regulatory gene <em>HAP4</em> that encodes the activating and limiting subunit of the heterotetrameric activator complex Hap2/3/4/5 involved in activation of respiration genes (<a href="#B25" rid="B25" class=" bibr popnode">25</a>). In addition, activation of Rds2 is correlated with its hyperphosphorylation by the Snf1 kinase (<a href="#B24" rid="B24" class=" bibr popnode">24</a>).</p><p>By performing ChIP-chip analysis with the <em>ERT1</em> (ethanol regulated transcription factor 1) gene product, we showed that this zinc cluster protein is an additional regulator of gluconeogenesis (<a href="#B26" rid="B26" class=" bibr popnode">26</a>). In analogy to Rds2, Ert1 is bound to a limited number of genes under glucose conditions (total of 11; e.g. <em>PCK1</em>). With ethanol as a carbon source, enrichment of Ert1 is observed at many additional genes (&#x0003e;70) involved in gluconeogenesis and mitochondrial function (e.g. <em>FMP43</em> encoding a subunit of the pyruvate mitochondrial transporter). Our results also demonstrated that Ert1 is redundant with other regulators of gluconeogenesis. Interestingly, we also showed that Ert1 acts as a repressor of <em>PDC1</em> encoding pyruvate decarboxylase, an important enzyme for fermentation. Thus, during a shift from fermentation to respiration, Ert1 has dual functions: it represses a key fermentation gene, while it activates gluconeogenic genes.</p><p class="p p-last">The Timmers&#x02019; group has shown that, under glucose conditions, the zinc cluster protein Gsm1 (glucose starvation modulator 1) binds to a very limited number of promoters such as <em>PCK1</em> and <em>HAP4</em> (<a href="#B27" rid="B27" class=" bibr popnode">27</a>). However, ChIP-chip analysis was not performed with ethanol as a carbon source. In this study, we show that, in the presence of ethanol, Gsm1 binds to over 600 promoters, including those of the gluconeogenic genes <em>PCK1</em> and <em>FBP1</em>. Strikingly, the binding profile of Gsm1 is very similar to Rds2, suggesting that these factors bind to DNA as heterodimers. ChIP-chip analysis of Gsm1 in a strain carrying a deletion of <em>RDS2</em> and the converse, Rds2 in a &#x00394;<em>gsm1</em> background, showed a complex pattern of DNA binding. For example, Gsm1 and Rds2 bind to the <em>FBP1</em> promoter in an interdependent manner, while at <em>PCK1</em> the presence of Rds2 is required for binding of Gsm1 but not the reverse. In contrast, both factors bind independently of each other at the <em>HAP4</em> promoter. These observations can be explained by the fact that Rds2 partially controls Gsm1 levels. In addition, an electrophoretic&#x000a0;mobility shift assay (EMSA) using the purified DNA binding domains of Gsm1 and Rds2 showed that these factors bind cooperatively by formation of heterodimers. We also showed that Rds2 and Ert1, but not Gsm1, bind cooperatively to the <em>PDC1</em> and <em>HAP4</em> promoters. These experiments revealed a novel mode of DNA recognition for zinc cluster proteins. Our results unravel a complex pattern of DNA recognition by three paralogs.</p></div><div id="SEC2" class="tsec sec"><h2 class="head no_bottom_margin" id="SEC2title" style="text-transform: uppercase;">Materials and methods</h2><div id="SEC2-1" class="sec sec-first"><h3 id="SEC2-1title">Strains and media</h3><p class="p p-first-last">Strains and oligonucleotides used in this study are listed in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Tables S1</a> and <a href="#sup1" rid="sup1" class=" supplementary-material">S2</a>, respectively. Media were prepared as described (<a href="#B28" rid="B28" class=" bibr popnode">28</a>). Tagging of <em><a id="__tag_1145552249" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSM1" ref="/geo/query/acc.cgi?acc=GSM1"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSM1","term_id":"1"}}</span>GSM1</a></em> at its C-terminus with three HA epitopes was performed by amplifying HA-URA3-HA sequences using oligos PETGSM1-1 and PETGSM1-2. The polymerase chain reaction (PCR) product was transformed in strain BY4741 (<a href="#B29" rid="B29" class=" bibr popnode">29</a>) followed by selection on plates lacking uracil. The <em>URA3</em> marker was removed by internal recombination as described (<a href="#B30" rid="B30" class=" bibr popnode">30</a>). Double-deletion strains (SC205, SC206, SC207, SC237, SC238 SC247, SC248 and SC249) were generated using single-deletion strains (<a href="#B31" rid="B31" class=" bibr popnode">31</a>) that were transformed with a PCR product containing the <em>Candida glabrata HIS3</em> gene flanked by sequences complementary to the 5&#x02032; and 3&#x02032; regions of the <em><a id="__tag_1145552267" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSM1" ref="/geo/query/acc.cgi?acc=GSM1"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSM1","term_id":"1"}}</span>GSM1</a></em> ORF followed by selection on minimal plates lacking histidine. The PCR product was obtained by using oligonucleotides <a id="__tag_1145552261" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSM1" ref="/geo/query/acc.cgi?acc=GSM1"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSM1","term_id":"1"}}</span>GSM1</a>-CgHIS3-1 and <a id="__tag_1145552268" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSM1" ref="/geo/query/acc.cgi?acc=GSM1"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSM1","term_id":"1"}}</span>GSM1</a>-CgHIS3-2 with <em>C. glabrata</em> DNA (strain ATCC 66032) as a template. Similarly, the double-deletion strains SC208 and SC209 were obtained by mating single-deletion strains SC102A and SC193 followed by sporulation and selection for double deletions in a BY4741 background. A similar approach was used to generate SC201 and SC202 using SC105A and SC193. The triple-deletion strains SC239 and SC240 were obtained by transforming SC202 with a PCR product generated with oligos RDS2-CgHIS3-1 and RDS2-CgHIS3-2 using <em>C. glabrata</em> DNA as a template.&#x000a0;The diploid strain yCW was generated by crossing strains yCW1476 and yCW1477 (<a href="#B32" rid="B32" class=" bibr popnode">32</a>).</p></div><div id="SEC2-2" class="sec"><h3 id="SEC2-2title">Plasmids for protein expression</h3><p class="p p-first-last">Plasmid pGST-Gsm1 (a.a. 1&#x02013;189) was constructed by amplifying the DNA binding domain of Gsm1 by PCR using oligos ZL (<a id="__tag_1145552248" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSM1" ref="/geo/query/acc.cgi?acc=GSM1"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSM1","term_id":"1"}}</span>GSM1</a>)-1 and ZL (<a id="__tag_1145552265" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSM1" ref="/geo/query/acc.cgi?acc=GSM1"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSM1","term_id":"1"}}</span>GSM1</a>)-2 and genomic DNA isolated from strain BY4741. The PCR product was cut with BamHI and MfeI and subcloned into the plasmid pGEX-f (<a href="#B11" rid="B11" class=" bibr popnode">11</a>) cut with BamHI and EcoRI. The expression vector pGST-Rds2 (a.a. 1&#x02013;97) was obtained by cutting plasmid pGST-Rds2 (a.a. 1&#x02013;152) (<a href="#B24" rid="B24" class=" bibr popnode">24</a>) with NdeI, filling with Klenow and ligating. This plasmid allows expression of the DNA binding domain of Rds2 (a.a. 1&#x02013;152) with 8 extra amino acids due to a frameshift. The plasmid pGST-Ert1 (a.a. 1&#x02013;152) has been described previously (<a href="#B26" rid="B26" class=" bibr popnode">26</a>).</p></div><div id="SEC2-3" class="sec"><h3 id="SEC2-3title">Plasmids for SYRTH analysis</h3><p class="p p-first-last">Plasmid pGST-Gsm1 (a.a. 1&#x02013;619) was constructed by amplifying the <em><a id="__tag_1145552258" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSM1" ref="/geo/query/acc.cgi?acc=GSM1"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSM1","term_id":"1"}}</span>GSM1</a></em> ORF by PCR using oligos <a id="__tag_1145552270" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSM1" ref="/geo/query/acc.cgi?acc=GSM1"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSM1","term_id":"1"}}</span>GSM1</a>-GIB-1 and <a id="__tag_1145552254" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSM1" ref="/geo/query/acc.cgi?acc=GSM1"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSM1","term_id":"1"}}</span>GSM1</a>-GIB-2, while plasmid pGST-Rds2 (a.a. 1&#x02013;447) was constructed by amplifying the <em>RDS2</em> ORF by PCR using oligos RDS2-GIB-1 and RDS2-GIB-2 using as a template DNA isolated from the strain BY4741. The PCR products were subcloned separately into pGEX-f (<a href="#B11" rid="B11" class=" bibr popnode">11</a>) cut with BamHI and EcoRI using Gibson assembly. The <em><a id="__tag_1145552262" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSM1" ref="/geo/query/acc.cgi?acc=GSM1"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSM1","term_id":"1"}}</span>GSM1</a></em> ORF was amplified by PCR from the plasmid GST-Gsm1 (a.a. 1&#x02013;619) using the oligos SYRTH-<a id="__tag_1145552263" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSM1" ref="/geo/query/acc.cgi?acc=GSM1"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSM1","term_id":"1"}}</span>GSM1</a>-GIB-3 and SYRTH-<a id="__tag_1145552259" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSM1" ref="/geo/query/acc.cgi?acc=GSM1"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSM1","term_id":"1"}}</span>GSM1</a>-GIB4. The <em>RDS2</em> ORF was amplified by PCR from the plasmid GST-Rds2 (a.a. 1&#x02013;447) using the oligos SYRTH-RDS2-GIB-3 and SYRTH-RDS2-GIB-4. The PCR products were digested with DpnI and purified on a PCR column. Using plasmids pCW778 and pYL040 as vectors (<a href="#B32" rid="B32" class=" bibr popnode">32</a>), pCW778-Gsm1, pYL040-Gsm1 and pYL040-Rds2 were constructed by Gibson assembly, while pCW778-Rds2 was obtained by using T4 DNA ligase. Empty vectors were constructed by digesting plasmids pCW778 and pYL040 (<a href="#B32" rid="B32" class=" bibr popnode">32</a>) with SmaI and religated using T4 DNA ligase.</p></div><div id="SEC2-4" class="sec"><h3 id="SEC2-4title">Spotting assays</h3><p class="p p-first-last">Strains were grown overnight in YPD medium at 30&#x000b0;C. Cells were harvested, washed twice, resuspended in water and diluted to&#x000a0;OD<sub>600</sub>=&#x000a0;1.0. Then, cells were serially diluted before being spotted on minimal plates supplemented with casamino acids (0.2%) as well as leucine, methionine, histidine and uracil (each at 0.004%) and 2% glucose or 2% of a nonfermentable carbon source (ethanol or lactate or glycerol). Plates were incubated at 30&#x000b0;C for 1 day (glucose) or 4&#x02013;5 days (ethanol, lactate, glycerol).</p></div><div id="SEC2-5" class="sec"><h3 id="SEC2-5title">ChIP-chip assays</h3><p class="p p-first-last">Strains were grown in YPD to an OD<sub>600</sub>&#x000a0;&#x02248;&#x000a0;0.7. Cells were washed twice in water and resuspended in rich medium containing 2% glucose (YPD) or containing 2% ethanol (YEP-2% ethanol) and grown for 3 h. Cross-linking, immunoprecipitation, DNA purification, DNA labeling, hybridization and scanning were performed as described (<a href="#B33" rid="B33" class=" bibr popnode">33</a>). The microarrays used, custom designed by Agilent Technologies, were described before (<a href="#B34" rid="B34" class=" bibr popnode">34</a>) and contain a total of &#x0223c;180&#x000a0;000 Tm-adjusted 60-mer probes covering the entire yeast genome with virtually no gaps between probes.</p></div><div id="SEC2-6" class="sec"><h3 id="SEC2-6title">ChIP-chip data analysis</h3><div id="SEC2-6-1" class="sec sec-first"><p></p><h4 id="SEC2-6-1title" class="inline">Identification of&#x000a0;binding sites for Rds2 and Gsm1 </h4><p class="p p-first-last">One round of Gaussian smoothing (standard deviation of 100 bp) was applied to the log<sub>2</sub> binding ratio data, reporting a value every 10 bp. The maxima that are above 0.8 were identified. Significantly enriched regions were identified using the <em>P</em>-values associated with each probe as described (<a href="#B35" rid="B35" class=" bibr popnode">35</a>). Finally, the maxima that overlap with the significant regions were retained and considered as significant binding sites. This method identified 632, 629, 627 and 471 binding sites for the Rds2, Gsm1, Rds2/&#x00394;<em>gsm1</em> and Gsm1/&#x00394;<em>rds2</em> datasets, respectively.</p></div><div id="SEC2-6-2" class="sec"><p></p><h4 id="SEC2-6-2title" class="inline">Overlap between datasets </h4><p class="p p-first-last">The overlap between the binding sites identified in the different datasets was calculated using the University of California at Santa Cruz TableBrowser (<a href="#B36" rid="B36" class=" bibr popnode">36</a>). The binding sites, extended to 200 bp, were used as input and the &#x02018;Intersection&#x02019; function of the TableBrowser was used to create bed files only containing the desired overlaps.</p></div><div id="SEC2-6-3" class="sec"><p></p><h4 id="SEC2-6-3title" class="inline">Effect of factor deletion on occupancy </h4><p class="p p-first-last">To determine the magnitude of the effect of deleting one transcription factor on the occupancy of the other, the average log<sub>2</sub> ratio was calculated over each binding site in each dataset and these values were subtracted to create a fold change between wild-type and mutant. To do this, we first trimmed the data to keep only the data overlapping the Rds2 or Gsm1 wild-type binding sites. This was done in the TableBrowser using the &#x02018;Intersection&#x02019; tool. For example, a file was intersected with another file to create a bedgraph (using the &#x02018;DATA VALUE format&#x02019; as output option) containing the Rds2 data with only the values that are 200 bp centered on its 632 binding sites. This generated trimmed bedgraphs (note that these contain one value at every bp, so 200 values per binding site). Next, the values for the mutants were subtracted from those of the wild-type. The 200 values for each binding site were averaged, leading to 1 fold change value for each region. The regions were then sorted based on the fold change. Next, the &#x02018;coordinate&#x02019; files were created to be used in VAP (<a href="#B37" rid="B37" class=" bibr popnode">37</a>) to create heatmaps of factor occupancy over binding sites sorted on their dependence on the other factor. Heatmaps were created using VAP and Treeview (<a href="#B38" rid="B38" class=" bibr popnode">38</a>) (to build images from outfile generated by VAP) using the above &#x02018;coordinate&#x02019; files and the smoothed data files as inputs.</p></div><div id="SEC2-6-4" class="sec sec-last"><p></p><h4 id="SEC2-6-4title" class="inline">Reproducibility of the ChIP-chip experiments </h4><p class="p p-first">Computed Pearson correlation (<em>r</em>) between the different replicates was &#x0003e;0.8 for all experiments (at 300 bp resolution)&#x000a0;except for Gsm1 in a &#x00394;<em>rds2</em> strain, which gives 0.57. The lower correlation between the replicates of this experiment could be explained by the fact that binding of Gsm1 is quite reduced in a &#x00394;<em>rds2</em> strain. There is therefore a reduced signal-to-noise ratio contributing to a weaker correlation.</p><p class="p p-last">Gene ontology (GO) term analysis was performed using the GO Term Finder (version 0.86) implementation in the <em>Saccharomyces</em> Genome Database (<a href="https://www.yeastgenome.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.yeastgenome.org</a>).</p></div></div><div id="SEC2-7" class="sec"><h3 id="SEC2-7title">Immunoblotting of whole&#x000a0;cell extracts</h3><p class="p p-first-last">To assess the bulk levels of Rds2 and Gsm1 within WT/&#x00394;<em>gsm1</em> and WT/&#x00394;<em>rds2</em> cells, respectively, at different times after ethanol treatment, immunoblotting experiments were performed on whole&#x000a0;cell extracts obtained as follows. Cells were grown in YPD to an OD<sub>600</sub>&#x000a0;&#x02248;&#x000a0;0.7, washed twice in water, resuspended in YEP-2% ethanol and grown for 1.5 and 3 h. For each sample, 50 ml of cell cultures were&#x000a0;pelleted and resuspended in 700 &#x003bc;l lysis buffer (50 mM HEPES&#x02013;KOH, pH 7.5, 140 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.1% sodium&#x000a0;deoxycholate, protease and phosphatase inhibitors). Cells were then lysed by bead beating for 5 min. Soluble extracts were recovered by centrifugation. Sodium dodecyl sulfate&#x02013;polyacrylamide gel electrophoresis (SDS&#x02013;PAGE) loading buffer, 5&#x000d7;&#x000a0;(250 mM Tris&#x02013;HCl, pH 6.8, 10% SDS, 50% glycerol, 0.05% bromophenol blue, 5% &#x003b2;-mercaptoethanol) was added to a final 1&#x000d7; concentration to 40 &#x003bc;l of each sample, incubated at 95&#x000b0;C for 5 min, briefly centrifuged and analyzed by SDS&#x02013;PAGE and western blot analysis&#x000a0;using mouse monoclonal anti-HA (Santa Cruz, sc-7392, 1:1000 dilution) and mouse monoclonal anti-Pgk1 (Abcam, ab113687, 1:30&#x000a0;000 dilution) antibodies. Blots were developed using donkey anti-mouse IRDye 680RD antibody (LI-COR Biosciences, LIC-926-68072) according to the manufacturer&#x02019;s instructions and scanned on the Odyssey infrared imaging system.</p></div><div id="SEC2-8" class="sec sec-last"><h3 id="SEC2-8title">Electrophoretic mobility shift assays</h3><p class="p p-first-last">Expression, purification of GST proteins and thrombin cleavage to remove the GST moiety were performed as described (<a href="#B11" rid="B11" class=" bibr popnode">11</a>,<a href="#B39" rid="B39" class=" bibr popnode">39</a>). For DNA&#x02013;protein binding, the reaction was performed in 4 mM Tris&#x02013;HCl (pH 8), 40 mM NaCl, 25 &#x003bc;g/ml sonicated salmon sperm DNA and 10 pmol/ml labeled probe. DNA binding quantification was performed with a phosphorimager (Storm 840, Amersham Biosciences). DNA&#x02013;protein signals were subtracted from background (lanes lacking protein) and ratios of mutant to wild-type signals&#x000a0;were determined.</p></div></div><div id="SEC3" class="tsec sec"><h2 class="head no_bottom_margin" id="SEC3title" style="text-transform: uppercase;">Results</h2><p class="p p-first">We were interested in&#x000a0;better elucidating the role of Gsm1, in particular with cells grown in the presence of ethanol by performing ChIP-chip experiments. To this end, <em><a id="__tag_1145552269" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSM1" ref="/geo/query/acc.cgi?acc=GSM1"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSM1","term_id":"1"}}</span>GSM1</a></em> was tagged with three HA epitopes at its natural chromosomal location using homologous recombination. In agreement with van Bakel <em>et&#x000a0;al.</em> (<a href="#B27" rid="B27" class=" bibr popnode">27</a>), we observed very limited binding of Gsm1 (&#x0003c;10 promoters) under glucose conditions (data not shown). For example, Gsm1 was detected at the promoters of the <em>PCK1</em> and the <em>HAP4</em> genes. In contrast to glucose conditions, Gsm1 is found at 629 promoters in the presence of ethanol, including <em>PCK1</em> and <em>FBP1</em>. Strikingly, Gsm1 and Rds2 show a highly similar pattern of binding with an 81% overlap (Figure <a href="/pmc/articles/PMC10954478/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1A).</span></span><span>1A</span></a>). GO analysis showed similar hits for Gsm1 and Rds2 (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S3</a>), such as generation of precursor metabolites and energy as well as transmembrane transport, suggesting that these factors perform common processes. The similar binding of Gsm1 and Rds2 suggests that these two factors may bind to DNA as heterodimers. To test this hypothesis, we performed ChIP-chip analysis with tagged Gsm1 expressed in a &#x00394;<em>rds2</em> strain and the converse, tagged Rds2 expressed in a &#x00394;<em>gsm1</em> background. Results show that binding of Gsm1 is largely dependent on Rds2, while Rds2 relies on Gsm1 to a lesser extent for binding to DNA (Figure <a href="/pmc/articles/PMC10954478/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1B).</span></span><span>1B</span></a>). For example, binding of Gsm1 and Rds2 is interdependent at the <em>FBP1</em> promoter, while at <em>PCK1</em>&#x000a0;Rds2 is required for binding of Gsm1 but not the reverse (Figure <a href="/pmc/articles/PMC10954478/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1C).</span></span><span>1C</span></a>). In addition, Gsm1 and Rds2 bind to <em>GDB1</em> and <em>GUP2</em> to a large extent independently of each other (Table <a href="/pmc/articles/PMC10954478/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table1).</span></span><span>1</span></a>). Rds2 (but not Gsm1) binds to the <em>PDC1</em> promoter, while Gsm1 (but not Rds2) is found at some other promoters (Table <a href="/pmc/articles/PMC10954478/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table1).</span></span><span>1</span></a>). A list of selected promoters bound by Gsm1 and/or Rds2 is given in Table <a href="/pmc/articles/PMC10954478/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">Table1.</span></span><span>1</span></a>. In summary, of the six possible types of binding, five were observed (Table <a href="/pmc/articles/PMC10954478/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table2).</span></span><span>2</span></a>). Overall, when looking at common Rds2/Gsm1 binding sites, results show that Gsm1 is dependent on Rds2 for binding at 87% of promoters. In contrast, Rds2 DNA occupancy relies on Gsm1 at only 22% sites (Figure <a href="/pmc/articles/PMC10954478/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1B).</span></span><span>1B</span></a>). Thus, a complex interplay between Gsm1 and Rds2 is observed. Requirement of Rds2 for Gsm1 DNA binding could be explained by the fact that, in a &#x00394;<em>rds2</em> strain, Gsm1 would be expressed at lower levels. To address this possibility, we performed western blot analysis using total extracts obtained at 0, 1.5 and 3 h after a shift from glucose to ethanol. HA-Rds2 levels remained unchanged during the time course as assayed in a wild-type and a &#x00394;<em>gsm1</em> strain (Figure <a href="/pmc/articles/PMC10954478/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2,</span></span><span>2</span></a>, top panel). In contrast, Gsm1-HA levels increased in a time-dependent manner in a wild-type strain, in agreement with the observation that <em><a id="__tag_1145552247" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSM1" ref="/geo/query/acc.cgi?acc=GSM1"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSM1","term_id":"1"}}</span>GSM1</a></em> messenger&#x000a0;RNA levels are increased in the presence of ethanol (<a href="#B4" rid="B4" class=" bibr popnode">4</a>). However, Gsm1-HA levels were clearly reduced at 1.5 and 3 h in a &#x00394;<em>rds2</em> strain. Thus, a requirement of Rds2 for binding of Gsm1 can be (partly) explained by reduced Gsm1 levels upon deletion of <em>RDS2</em>. However, our EMSAs clearly show interdependent binding of Gsm1 and Rds2, as well as Ert1 and Rds2 (see below). In addition, we also observe similar binding of Gsm1 at <em>GDB1</em>, <em>GUP2</em> and <em>MDM31</em> in a &#x00394;<em>r</em><em>ds2</em> strain (Table <a href="/pmc/articles/PMC10954478/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table1</span></span><span>1</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC10954478/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140432047513840"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10954478_gkad1185fig1.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkad1185fig1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10954478/bin/gkad1185fig1.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140432047513840"><a target="object" rel="noopener" href="/pmc/articles/PMC10954478/figure/F1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC10954478/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1.</a></div><!--caption a7--><div class="caption"><p>Gsm1 and Rds2 genome-wide binding sites: complex interplay between Gsm1 and Rds2. ChIP-chip experiments were performed with HA-tagged Gsm1 in wild-type and &#x00394;rds2 strains. Conversely, experiments were done with HA-tagged Rds2 in wild-type and &#x00394;<em>gsm1</em> strains. All ChIP experiments were performed with biological duplicates that were highly correlated and the figure shows results from their weighted averaging (see the &#x02018;Materials and methods&#x02019; section). (<strong>A</strong>) Venn diagram showing the extensive overlap between the Rds2 (<em>n</em>&#x000a0;=&#x000a0;632) and Gsm1 (<em>n</em>&#x000a0;=&#x000a0;629) binding sites. (<strong>B</strong>) Heatmap of Gsm1 (panels 1 and 2) and Rds2 (panels 4 and 5) binding in a 2-kb window around their respective binding sites. The data for each factor are shown in wild-type cells and in cells where the other factor gene was deleted. A differential binding map is also shown for both factors (panels 3 and 6). The binding sites were ordered based on the magnitude of the effect of the deletion (top binding sites are the least affected by the deletion and the bottom ones are the most affected). (<strong>C</strong>) Binding of Gsm1 and Rds2 is shown at the <em>FBP1</em> (left) and <em>PCK1</em> promoters (right). Gray rectangles correspond to ORFs with arrows showing the direction of transcription. Strains used for ChIP-chip analysis are indicated on the left. The <em>Y</em> axis (log<sub>2</sub> scale) gives the enrichment (binding) for a given factor. (<strong>D</strong>) Binding of Gsm1 and Rds2 at a region of chromosome XI. Binding of these factors at the <em>HAP4</em> promoter is also shown (right side).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="tbl1"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p>Selected Gsm1 and Rds2 targets under ethanol conditions</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th colspan="4" align="center" rowspan="1">Enrichment (given in log<sub>2</sub> values) for</th></tr><tr><th rowspan="1" colspan="1">Gene</th><th rowspan="1" colspan="1">Function (after <a href="http://yeastgenome.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">yeastgenome.org</a>)</th><th rowspan="1" colspan="1">
<em>RDS2</em>
</th><th rowspan="1" colspan="1">
<em>RDS2</em> in a &#x00394;<em>gsm1</em> strain</th><th rowspan="1" colspan="1">
<em><a id="__tag_1145552251" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSM1" ref="/geo/query/acc.cgi?acc=GSM1"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSM1","term_id":"1"}}</span>GSM1</a></em>
</th><th rowspan="1" colspan="1">
<em><a id="__tag_1145552246" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSM1" ref="/geo/query/acc.cgi?acc=GSM1"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSM1","term_id":"1"}}</span>GSM1</a></em> in a &#x00394;<em>rds2</em> strain</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">
<em>PCK1</em>
</td><td rowspan="1" colspan="1">Phosphoenolpyruvate carboxykinase; key enzyme in gluconeogenesis</td><td rowspan="1" colspan="1">5.7</td><td rowspan="1" colspan="1">6.8</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">1.6</td></tr><tr><td rowspan="1" colspan="1">
<em>PDC1</em>
</td><td rowspan="1" colspan="1">Major of three pyruvate decarboxylase isozymes; key enzyme in alcoholic fermentation</td><td rowspan="1" colspan="1">6.6</td><td rowspan="1" colspan="1">6.9</td><td rowspan="1" colspan="1">NS</td><td rowspan="1" colspan="1">NS</td></tr><tr><td rowspan="1" colspan="1">
<em>PTK1</em>
</td><td rowspan="1" colspan="1">Putative serine/threonine protein kinase; regulates spermine uptake</td><td rowspan="1" colspan="1">5.2</td><td rowspan="1" colspan="1">2.8</td><td rowspan="1" colspan="1">3.7</td><td rowspan="1" colspan="1">2.1</td></tr><tr><td rowspan="1" colspan="1">
<em>MAE1</em>
</td><td rowspan="1" colspan="1">Mitochondrial malic enzyme; catalyzes the oxidative decarboxylation of malate to pyruvate</td><td rowspan="1" colspan="1">5.2</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">NS</td><td rowspan="1" colspan="1">NS</td></tr><tr><td rowspan="1" colspan="1">
<em>AQR1</em>
</td><td rowspan="1" colspan="1">Plasma membrane transporter of the major facilitator superfamily</td><td rowspan="1" colspan="1">5.1</td><td rowspan="1" colspan="1">5.4</td><td rowspan="1" colspan="1">NS</td><td rowspan="1" colspan="1">NS</td></tr><tr><td rowspan="1" colspan="1">
<em>HAP4</em>
</td><td rowspan="1" colspan="1">Subunit of the heme-activated, glucose-repressed Hap2/3/4/5 CCAAT-binding complex</td><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">5.6</td><td rowspan="1" colspan="1">2.5</td></tr><tr><td rowspan="1" colspan="1">
<em>SOM1</em>
</td><td rowspan="1" colspan="1">Subunit of the mitochondrial inner membrane peptidase</td><td rowspan="1" colspan="1">4.9</td><td rowspan="1" colspan="1">2.1</td><td rowspan="1" colspan="1">4.9</td><td rowspan="1" colspan="1">3.1</td></tr><tr><td rowspan="1" colspan="1">
<em>GAT1</em>
</td><td rowspan="1" colspan="1">Transcriptional activator of nitrogen catabolite repression genes</td><td rowspan="1" colspan="1">4.6</td><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">2.5</td><td rowspan="1" colspan="1">1.2</td></tr><tr><td rowspan="1" colspan="1">
<em>OLE1</em>
</td><td rowspan="1" colspan="1">Delta(9) fatty acid desaturase; required for monounsaturated fatty acid synthesis</td><td rowspan="1" colspan="1">4.5</td><td rowspan="1" colspan="1">2.2</td><td rowspan="1" colspan="1">4.6</td><td rowspan="1" colspan="1">1.9</td></tr><tr><td rowspan="1" colspan="1">
<em>FBP1</em>
</td><td rowspan="1" colspan="1">Fructose 1,6-bisphosphatase; key regulatory enzyme in the gluconeogenesis pathway</td><td rowspan="1" colspan="1">4.1</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">4.4</td><td rowspan="1" colspan="1">1</td></tr><tr><td rowspan="1" colspan="1">
<em>API2</em>
</td><td rowspan="1" colspan="1">Putative protein of unknown function</td><td rowspan="1" colspan="1">4.5</td><td rowspan="1" colspan="1">3.3</td><td rowspan="1" colspan="1">4.7</td><td rowspan="1" colspan="1">2.9</td></tr><tr><td rowspan="1" colspan="1">
<em>YDR095C</em>
</td><td rowspan="1" colspan="1">Dubious ORF</td><td rowspan="1" colspan="1">4.4</td><td rowspan="1" colspan="1">2.2</td><td rowspan="1" colspan="1">4.5</td><td rowspan="1" colspan="1">2.1</td></tr><tr><td rowspan="1" colspan="1">
<em>LCL1</em>
</td><td rowspan="1" colspan="1">Putative protein of unknown function; deletion mutant is fluconazole resistant</td><td rowspan="1" colspan="1">4.4</td><td rowspan="1" colspan="1">2.5</td><td rowspan="1" colspan="1">4.4</td><td rowspan="1" colspan="1">2.1</td></tr><tr><td rowspan="1" colspan="1">
<em>YPL062W</em>
</td><td rowspan="1" colspan="1">Putative protein of unknown function</td><td rowspan="1" colspan="1">4.2</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">4.2</td><td rowspan="1" colspan="1">2.4</td></tr><tr><td rowspan="1" colspan="1">
<em>ATP3</em>
</td><td rowspan="1" colspan="1">Gamma subunit of the F1 sector of mitochondrial F1F0 ATP synthase</td><td rowspan="1" colspan="1">2.8</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">1.3</td><td rowspan="1" colspan="1">NS</td></tr><tr><td rowspan="1" colspan="1">
<em>BRP1</em>
</td><td rowspan="1" colspan="1">Putative protein of unknown function</td><td rowspan="1" colspan="1">4.1</td><td rowspan="1" colspan="1">4.3</td><td rowspan="1" colspan="1">3.9</td><td rowspan="1" colspan="1">2.1</td></tr><tr><td rowspan="1" colspan="1">
<em>PUS2</em>
</td><td rowspan="1" colspan="1">Mitochondrial tRNA:pseudouridine synthase</td><td rowspan="1" colspan="1">3.9</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">2.5</td></tr><tr><td rowspan="1" colspan="1">
<em>GLN1</em>
</td><td rowspan="1" colspan="1">Glutamine synthetase; synthesizes glutamine from glutamate and ammonia</td><td rowspan="1" colspan="1">3.8</td><td rowspan="1" colspan="1">3.5</td><td rowspan="1" colspan="1">3.7</td><td rowspan="1" colspan="1">2</td></tr><tr><td rowspan="1" colspan="1">
<em>CIT1</em>
</td><td rowspan="1" colspan="1">Citrate synthase; catalyzes the condensation of acetyl coenzyme A and oxaloacetate to form citrate</td><td rowspan="1" colspan="1">3.5</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">3.5</td><td rowspan="1" colspan="1">1.4</td></tr><tr><td rowspan="1" colspan="1">
<em>CYC1</em>
</td><td rowspan="1" colspan="1">Cytochrome <em>c</em>, isoform 1; electron carrier of mitochondrial intermembrane space</td><td rowspan="1" colspan="1">3.4</td><td rowspan="1" colspan="1">1.7</td><td rowspan="1" colspan="1">3.5</td><td rowspan="1" colspan="1">1.7</td></tr><tr><td rowspan="1" colspan="1">
<em>GDB1</em>
</td><td rowspan="1" colspan="1">Glycogen debranching enzyme; required for glycogen degradation</td><td rowspan="1" colspan="1">1.5</td><td rowspan="1" colspan="1">1.4</td><td rowspan="1" colspan="1">1.5</td><td rowspan="1" colspan="1">1.2</td></tr><tr><td rowspan="1" colspan="1">
<em>GUP2</em>
</td><td rowspan="1" colspan="1">Probable membrane protein; possible role in proton symport of glycerol</td><td rowspan="1" colspan="1">1.5</td><td rowspan="1" colspan="1">1.6</td><td rowspan="1" colspan="1">2.8</td><td rowspan="1" colspan="1">2.2</td></tr><tr><td rowspan="1" colspan="1">
<em>MDM31</em>
</td><td rowspan="1" colspan="1">Mitochondrial protein that may have a role in phospholipid metabolism</td><td rowspan="1" colspan="1">NS</td><td rowspan="1" colspan="1">NS</td><td rowspan="1" colspan="1">1.5</td><td rowspan="1" colspan="1">1.3</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140432047511248"><a target="object" rel="noopener" href="/pmc/articles/PMC10954478/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="T1TFN1"><p class="p p-first-last">NS, not significant.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="tbl2"><h3>Table 2.</h3><!--caption a7--><div class="caption"><p>Types of binding modes observed for Gsm1 and Rds2</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Type of binding</th><th rowspan="1" colspan="1">
<em>RDS2</em> in a wild-type strain</th><th rowspan="1" colspan="1">
<em>RDS2</em> in a &#x00394;<em>gsm1</em> strain</th><th rowspan="1" colspan="1">
<em><a id="__tag_1145552264" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSM1" ref="/geo/query/acc.cgi?acc=GSM1"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSM1","term_id":"1"}}</span>GSM1</a></em> in a wild-type strain</th><th rowspan="1" colspan="1">
<em><a id="__tag_1145552252" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSM1" ref="/geo/query/acc.cgi?acc=GSM1"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSM1","term_id":"1"}}</span>GSM1</a></em> in a&#x000a0;<em>&#x00394;rds2</em> strain</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Interdependent, e.g. <em>FBP1</em></td><td rowspan="1" colspan="1">+</td><td rowspan="1" colspan="1">+/&#x02212;</td><td rowspan="1" colspan="1">+</td><td rowspan="1" colspan="1">+/&#x02212;</td></tr><tr><td rowspan="1" colspan="1">Largely independent of each other, e.g. <em>GDB1</em>&#x000a0;and <em>GU</em><em>P2</em></td><td rowspan="1" colspan="1">+</td><td rowspan="1" colspan="1">+</td><td rowspan="1" colspan="1">+</td><td rowspan="1" colspan="1">+</td></tr><tr><td rowspan="1" colspan="1">Rds2 only, e.g. <em>PDC1</em></td><td rowspan="1" colspan="1">+</td><td rowspan="1" colspan="1">+</td><td rowspan="1" colspan="1">+/&#x02212;</td><td rowspan="1" colspan="1">+/&#x02212;</td></tr><tr><td rowspan="1" colspan="1">Gsm1 only, e.g. <em>MDM31</em></td><td rowspan="1" colspan="1">+/&#x02212;</td><td rowspan="1" colspan="1">+/&#x02212;</td><td rowspan="1" colspan="1">+</td><td rowspan="1" colspan="1">+</td></tr><tr><td rowspan="1" colspan="1">Rds2-dependent, e.g. <em>PCK1</em></td><td rowspan="1" colspan="1">+</td><td rowspan="1" colspan="1">+</td><td rowspan="1" colspan="1">+</td><td rowspan="1" colspan="1">+/&#x02212;</td></tr><tr><td rowspan="1" colspan="1">Gsm1-dependent, not observed</td><td rowspan="1" colspan="1">+</td><td rowspan="1" colspan="1">+/&#x02212;</td><td rowspan="1" colspan="1">+</td><td rowspan="1" colspan="1">+</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140432045006160"><a target="object" rel="noopener" href="/pmc/articles/PMC10954478/table/tbl2/?report=objectonly">Open in a separate window</a></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC10954478/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140432045418704"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkad1185fig2.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is gkad1185fig2.jpg" src="/pmc/articles/PMC10954478/bin/gkad1185fig2.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140432045418704"><a target="object" rel="noopener" href="/pmc/articles/PMC10954478/figure/F2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC10954478/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2.</a></div><!--caption a7--><div class="caption"><p>Rds2 controls Gsm1 levels. Top panel: wild-type and <em>&#x00394;gsm1</em> strains expressing tagged Rds2 were used to prepare extracts at 0, 1.5 and 3 h following a shift to ethanol for western blot analysis. Pgk1 was used as a loading control. Bottom panel: same as above except that tagged Gsm1 was used and tested in a wild-type and a &#x00394;<em>rds2</em> strain. The western blots were generated three times and a representative blot is shown.</p></div></div></div><div id="SEC3-1" class="sec"><h3 id="SEC3-1title">Gsm1 and Rds2 physically interact with each other</h3><p class="p p-first">To start investigating whether Rds2 and Gsm1 may regulate each other&#x02019;s ability to bind DNA, we first tested whether Gsm1 and Rds2 physically interact with each other. Since Rds2 and Gsm1 are DNA binding proteins (see below), use of the classical two-hybrid assay is not optimal. As a result, we used a modified two-hybrid assay called SYRTH (for Ste11/Ste50 yeast related two-hybrid) that has been shown to work with transcriptional regulators (<a href="#B32" rid="B32" class=" bibr popnode">32</a>). This system allows detection of cytoplasmic protein&#x02013;protein interactions and it relies on the observation that Ste11 must interact with Ste50 through SAM domains for activation of the high osmotic stress pathway. In this modified two-hybrid assay, factor X is fused to Ste11 lacking the SAM domain, while factor Y is fused to Ste50 also lacking this domain. Interaction of protein X with protein Y brings together Ste11 and Ste50 for activation of the high osmotic pathway and growth in hyperosmotic medium, the readout of the assay. A strain carrying deletions of the <em>STE11</em> and <em>STE50</em> genes was transformed with various combinations of expression vectors. Transformants were grown on selective medium, diluted and spotted on minimal plates containing galactose for induction of the expression of the proteins of interest. In the absence of osmotic stress (0 mM NaCl), growth was similar for all combinations of expression vectors tested (Figure <a href="/pmc/articles/PMC10954478/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3,</span></span><span>3</span></a>, left panel). In the presence of 500 mM NaCl, no growth was observed with cells expressing mutants <em>STE11</em> and <em>STE50</em> or a single fusion protein. However, co-expression of Gsm1&#x02013;Ste11 with Rds2&#x02013;Ste50 resulted in growth under osmotic stress (Figure <a href="/pmc/articles/PMC10954478/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3,</span></span><span>3</span></a>, right panel). Thus, Gsm1 and Rds2 interact with each other in this assay. However, no growth was observed when testing the converse: Rds2&#x02013;Ste11 with Gsm1&#x02013;Ste50. This may be due to alternate conformations of fusion proteins that prevent an interaction. Since the SYRTH assay relies on cytoplasmic interactions, our results strongly suggest that Gsm1 and Rds2 can interact with each other even in the absence of DNA.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC10954478/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140432052946320"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10954478_gkad1185fig3.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkad1185fig3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10954478/bin/gkad1185fig3.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140432052946320"><a target="object" rel="noopener" href="/pmc/articles/PMC10954478/figure/F3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC10954478/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3.</a></div><!--caption a7--><div class="caption"><p>Gsm1 interacts with Rds2 in a modified two-hybrid assay. Strain yCW (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S1</a>) was transformed with empty expression vectors for mutated <em>STE11</em> or <em>STE50</em> or with vectors for expression of fusion proteins. Cells were grown in minimal selective medium lacking uracil and histidine and diluted to OD<sub>600</sub>&#x000a0;=&#x000a0;0.1. Cells were further diluted 25, 100, 400 and 2000 times and 10 &#x003bc;l of cells were spotted on minimal plates containing 2% galactose with (right panel) or without 500 mM NaCl (left panel). Expression vectors are indicated on the left part of the figure. Spotting in this&#x000a0;figure is a representative experiment out of three.</p></div></div></div></div><div id="SEC3-2" class="sec"><h3 id="SEC3-2title">Deletion of <em><a id="__tag_1145552266" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSM1" ref="/geo/query/acc.cgi?acc=GSM1"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSM1","term_id":"1"}}</span>GSM1</a></em> along with other genes involved in gluconeogenesis does not result in growth defects</h3><p class="p p-first">We were interested in determining whether deletion of <em><a id="__tag_1145552260" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSM1" ref="/geo/query/acc.cgi?acc=GSM1"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSM1","term_id":"1"}}</span>GSM1</a></em> results in reduced growth with a nonfermentable carbon source (ethanol, lactate, glycerol). We also tested strains carrying deletions of <em><a id="__tag_1145552250" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSM1" ref="/geo/query/acc.cgi?acc=GSM1"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSM1","term_id":"1"}}</span>GSM1</a></em> in combination with other genes encoding regulators of gluconeogenesis as well as a triple-deletion strain (&#x00394;<em>gsm1</em> &#x00394;<em>rds2</em> &#x00394;<em>ert1</em>). Strains were grown in rich medium containing glucose (YPD), washed with water, serially diluted and spotted on plates containing glucose or ethanol as a carbon source. Under glucose conditions, growth was similar for all tested strains (Figure <a href="/pmc/articles/PMC10954478/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure4,</span></span><span>4</span></a>, first panel). As expected from previous observations (<a href="#B18" rid="B18" class=" bibr popnode">18</a>,<a href="#B26" rid="B26" class=" bibr popnode">26</a>), a severe growth defect was observed with a strain lacking <em>CAT8</em> under ethanol conditions. However, deletion of <em><a id="__tag_1145552256" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSM1" ref="/geo/query/acc.cgi?acc=GSM1"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSM1","term_id":"1"}}</span>GSM1</a></em> by itself or in combination with other regulatory genes of gluconeogenesis did not affect growth. Similarly, deletion of the three paralogs (<em><a id="__tag_1145552257" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSM1" ref="/geo/query/acc.cgi?acc=GSM1"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSM1","term_id":"1"}}</span>GSM1</a></em>, <em>RDS2</em> and <em>ERT1</em>) did not impair growth (Figure <a href="/pmc/articles/PMC10954478/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure4,</span></span><span>4</span></a>, second panel). Similar results were obtained&#x000a0;when testing growth on the nonfermentable carbon sources glycerol and lactate (Figure <a href="/pmc/articles/PMC10954478/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure4,</span></span><span>4</span></a>, third and fourth panels). These results suggest a high level of redundancy among transcriptional regulators of gluconeogenesis.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC10954478/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140432047766368"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10954478_gkad1185fig4.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkad1185fig4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10954478/bin/gkad1185fig4.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140432047766368"><a target="object" rel="noopener" href="/pmc/articles/PMC10954478/figure/F4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC10954478/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4.</a></div><!--caption a7--><div class="caption"><p>Gsm1 does not interact genetically with genes encoding regulators of gluconeogenesis. Cells were grown overnight in rich medium (YPD) and&#x000a0;washed twice in water. Cells were diluted 25, 100, 400 and 1600 times and 10 &#x003bc;l of cells were spotted on minimal plates supplemented with casamino acids (0.2%) as well as leucine, methionine, histidine and uracil (each at 0.004%) and 2% glucose (left panel), 2% ethanol (second panel), 2% lactate (third panel) and&#x000a0;2% glycerol (fourth panel). Plates were incubated at 30&#x000b0;C for 1 day (glucose),&#x000a0;5 days (ethanol) or 4 days (lactate and glycerol). For double- and triple-deletion strains, two independent isolates were tested. Spotting in this&#x000a0;figure is a representative experiment out of four.</p></div></div></div></div><div id="SEC3-3" class="sec"><h3 id="SEC3-3title">Gsm1 by itself binds <em>in vitro</em> to the <em>FBP1</em> promoter and cooperatively with Rds2</h3><p class="p p-first">In order to better understand the mode of binding of the three paralogs, a series of EMSAs were performed with the DNA binding domains of Gsm1 (a.a. 1&#x02013;189), Rds2 (a.a. 1&#x02013;97) and Ert1 (a.a. 1&#x02013;152). For simplicity, we will refer to these polypeptides as Gsm1, Rds2 and Ert1. The DNA binding domains were bacterially expressed as fusions with GST, purified on glutathione beads and the GST moiety removed by cleavage with thrombin. The binding site of Gsm1 at the <em>FBP1</em> promoter was identified by performing EMSAs. Gsm1 was used with a set of overlapping probes encompassing the <em>FBP1</em> promoter from &#x02212;800 to &#x02212;350 bp relative to the ATG. Results show that Gsm1 binds only to probe &#x02018;J&#x02019; that overlaps CSRE #1 (Figure <a href="/pmc/articles/PMC10954478/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5A</span></span><span>5A</span></a> and&#x000a0;<a href="/pmc/articles/PMC10954478/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>B</span></a>). No binding of Gsm1 was detected with the probe &#x02018;OP&#x02019; containing the CSRE and sequences upstream of this element or with the CSRE itself (Figure <a href="/pmc/articles/PMC10954478/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5B).</span></span><span>5B</span></a>). However, when using a probe (&#x02018;X&#x02019;) containing the CSRE and sequences extending at the 3&#x02032; end (Figure <a href="/pmc/articles/PMC10954478/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5A),</span></span><span>5A</span></a>), a DNA&#x02013;protein complex was detected (Figure <a href="/pmc/articles/PMC10954478/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5B).</span></span><span>5B</span></a>). In summary, the binding site for Gsm1 maps to a 33-bp fragment (probe &#x02018;X&#x02019;) overlapping CSRE #1.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC10954478/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140432039687984"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10954478_gkad1185fig5.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkad1185fig5.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10954478/bin/gkad1185fig5.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140432039687984"><a target="object" rel="noopener" href="/pmc/articles/PMC10954478/figure/F5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC10954478/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5.</a></div><!--caption a7--><div class="caption"><p>Gsm1 binds <em>in vitro</em> to the <em>FBP1</em> promoter. (<strong>A</strong>) The <em>FBP1</em> promoter is schematically shown on top with rectangles corresponding to CSREs. Probes (A&#x02013;K) were used in an EMSA with the purified DNA binding domain of Gsm1 (a.a. 1&#x02013;189). (<strong>B</strong>) An EMSA performed with probes J, OP, CSRE #1 and X. Sequence of probe X is given in Figure <a href="/pmc/articles/PMC10954478/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure6A;</span></span><span>6A</span></a>; also see (<a href="#B26" rid="B26" class=" bibr popnode">26</a>). This is a representative experiment out of three.</p></div></div></div><p>We were interested in determining the DNA sequences recognized by Gsm1 more precisely. Zinc cluster proteins bind preferentially to CGG triplets (<a href="#B6" rid="B6" class=" bibr popnode">6</a>). Five CGG triplets are found in probe &#x02018;X&#x02019; (Figure <a href="/pmc/articles/PMC10954478/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6A).</span></span><span>6A</span></a>). In order to define the triplets important for Gsm1 binding, probes containing mutations in these triplets were used in an EMSA (Figure <a href="/pmc/articles/PMC10954478/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6B).</span></span><span>6B</span></a>). Mutations in CGG triplets 1, 4 and 5 did not affect binding of Gsm1 (Figure <a href="/pmc/articles/PMC10954478/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6B,</span></span><span>6B</span></a>, lanes 6, 12 and 14). However, mutating triplet 2 or 3 abolished binding of Gsm1 (Figure <a href="/pmc/articles/PMC10954478/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6B,</span></span><span>6B</span></a>, lanes 8 and 10). These results allowed mapping with more accuracy the DNA site recognized by Gsm1: two CGG triplets, with one contained within the CSRE and one located downstream of the CSRE.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F6" co-legend-rid="lgnd_F6"><a href="/pmc/articles/PMC10954478/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140432090298704"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10954478_gkad1185fig6.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkad1185fig6.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10954478/bin/gkad1185fig6.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140432090298704"><a target="object" rel="noopener" href="/pmc/articles/PMC10954478/figure/F6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F6"><div><a class="figpopup" href="/pmc/articles/PMC10954478/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6">Figure 6.</a></div><!--caption a7--><div class="caption"><p>A short DNA element is sufficient for binding of Gsm1 by itself or cooperatively with Rds2 at the <em>FBP1</em> promoter. (<strong>A</strong>) Sequence of the <em>FBP1</em> probe &#x02018;X&#x02019; is shown on the top of the figure with CGG triplets indicated with arrows and designated &#x02018;a&#x02019; to &#x02018;e&#x02019;. The CSRE is shown as a gray area. Lowercase letters correspond to nucleotides that were mutated. (<strong>B</strong>) An EMSA was performed with the purified DNA binding domain of Gsm1 (a.a. 1&#x02013;189) (2 &#x003bc;l) with probe X and mutants (&#x02018;a&#x02019; to &#x02018;e&#x02019;) with nucleotide changes indicated on top of the panel. (<strong>C</strong>) An EMSA was performed with the purified DNA binding domains of Gsm1 (a.a. 1&#x02013;189) and Rds2 (a.a. 1&#x02013;97) using probe X. Lane 1: probe by itself. Lanes 2&#x02013;5: 0.025, 0.5, 1.0 and 2.0 &#x003bc;l of Gsm1 (a.a. 1&#x02013;189). Lanes 6&#x02013;9: 0.5, 1.0, 2.0 and 4.0 &#x003bc;l of Rds2 (a.a. 1&#x02013;97). Lanes 10&#x02013;13: same as lanes 6&#x02013;9 except that 0.5 &#x003bc;l of Gsm1 was also added. Lanes 14&#x02013;17: same as lanes 2&#x02013;5 except that 4.0 &#x003bc;l of Rds2 was also added. This is a representative experiment out of three.</p></div></div></div><p class="p p-last">Our ChIP-chip data demonstrated that binding of Gsm1 and Rds2 to the <em>FBP1</em> promoter is interdependent <em>in vivo</em> (Figure <a href="/pmc/articles/PMC10954478/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1C).</span></span><span>1C</span></a>). We tested whether this observation could be recapitulated <em>in vitro</em> using an EMSA. With increasing amount of Gsm1, a DNA&#x02013;protein complex is observed only at the highest concentration used in the EMSA (Figure <a href="/pmc/articles/PMC10954478/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6C,</span></span><span>6C</span></a>, lane 5), while no Rds2&#x02013;DNA complex is seen with this protein by itself (Figure <a href="/pmc/articles/PMC10954478/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6C,</span></span><span>6C</span></a>, lanes 6&#x02013;9). In contrast, a DNA&#x02013;protein complex is observed when mixing Gsm1 and Rds2 (Figure <a href="/pmc/articles/PMC10954478/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6C,</span></span><span>6C</span></a>, lanes 10&#x02013;13 and 14&#x02013;17). For example, a strong binding is observed in the presence of Gsm1 and Rds2, while no complex is detected with the same amounts of Gsm1 or Rds2 by themselves (Figure <a href="/pmc/articles/PMC10954478/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6C,</span></span><span>6C</span></a>, compare lane 16 with lanes 4 and 9), suggesting cooperative binding of Gsm1 and Rds2. In summary, Gsm1 and Rds2 bind cooperatively to a region of the <em>FBP1</em> promoter.</p></div><div id="SEC3-4" class="sec"><h3 id="SEC3-4title">Mutations that prevent binding of Gsm1 also abolish cooperative binding with Rds2</h3><p class="p p-first">We further refined our analysis of sequences important for binding of Gsm1 to DNA by testing additional mutations (Figure <a href="/pmc/articles/PMC10954478/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure7A</span></span><span>7A</span></a> and&#x000a0;<a href="/pmc/articles/PMC10954478/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>B</span></a>). Results show that mutations at positions &#x02018;a&#x02019;, &#x02018;b&#x02019;, &#x02018;d&#x02019;, &#x02018;e&#x02019;, &#x02018;f&#x02019;, &#x02018;g&#x02019; and &#x02018;o&#x02019; to &#x02018;s&#x02019; did not significantly affect Gsm1 binding (Figure <a href="/pmc/articles/PMC10954478/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure7B;</span></span><span>7B</span></a>; for quantification, see Figure <a href="/pmc/articles/PMC10954478/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure7D).</span></span><span>7D</span></a>). In contrast, mutations at positions &#x02018;h&#x02019;, &#x02018;i&#x02019;, &#x02018;k&#x02019; and &#x02018;m&#x02019; almost completely abolished DNA binding of Gsm1, while mutations at positions &#x02018;c&#x02019;, &#x02018;l&#x02019; and &#x02018;n&#x02019; reduced DNA binding by at least 2-fold. When combining these data with those of Figure <a href="/pmc/articles/PMC10954478/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure6B,</span></span><span>6B</span></a>, results show that eight nucleotides are important for DNA binding of Gsm1 with the sequence CCGGNGtTa (nucleotides in lowercase letters corresponding to positions less critical for Gsm1 binding). We then determined which nucleotides are important for cooperative binding of Gsm1 and Rds2 (Figure <a href="/pmc/articles/PMC10954478/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure7C;</span></span><span>7C</span></a>; for quantification, see Figure <a href="/pmc/articles/PMC10954478/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure7D).</span></span><span>7D</span></a>). Unexpectedly, scanning mutations revealed that the cooperative binding pattern of Gsm1 and Rds2 was very similar to the one observed with Gsm1 by itself. For example, mutations &#x02018;h&#x02019;, &#x02018;i&#x02019;, &#x02018;k&#x02019; and&#x000a0;&#x02018;m&#x02019; that were critical for binding of Gsm1 also greatly reduced cooperative binding of Gsm1 and Rds2. However, mutation at position &#x02018;c&#x02019; reduced binding of Gsm1 by 2-fold without affecting cooperative binding (Figure <a href="/pmc/articles/PMC10954478/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure7D).</span></span><span>7D</span></a>). Our analysis of Gsm1 and Rds2 suggests that these factors do not bind to adjacent sites but rather to a short stretch (9 bp) of common DNA sequences. Similar observations were made with Rds2 and Ert1 at the <em>PDC1</em> and <em>HAP4</em> promoters (see below).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F7" co-legend-rid="lgnd_F7"><a href="/pmc/articles/PMC10954478/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140432048729856"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10954478_gkad1185fig7.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkad1185fig7.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10954478/bin/gkad1185fig7.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140432048729856"><a target="object" rel="noopener" href="/pmc/articles/PMC10954478/figure/F7/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F7"><div><a class="figpopup" href="/pmc/articles/PMC10954478/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7">Figure 7.</a></div><!--caption a7--><div class="caption"><p>Mutational analysis suggests that Gsm1 and Rds2 do not bind at <em>FBP1</em> to adjacent sites but rather to a short common DNA stretch. (<strong>A</strong>) Sequence of the <em>FBP1</em> probe &#x02018;Z1&#x02019; (a shorter version of probe &#x02018;X&#x02019;) is shown on the top of the figure with CGG triplets indicated with arrows and numbered 1&#x02013;4. The CSRE is shown as a gray area. Lowercase letters (&#x02018;a&#x02019; to &#x02018;s&#x02019;) correspond to nucleotides that were mutated. Black dots correspond to nucleotides that were shown to be critical for binding of Gsm1 (Figure <a href="/pmc/articles/PMC10954478/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5).</span></span><span>5</span></a>). (<strong>B</strong>) An EMSA was performed with the purified DNA binding domain of Gsm1 (a.a. 1&#x02013;189) with probe &#x02018;Z1&#x02019; and mutants (&#x02018;a&#x02019; to &#x02018;s&#x02019;) with nucleotide changes indicated on top of the panel. (<strong>C</strong>) An EMSA was performed with the purified DNA binding domains of Gsm1 (a.a. 1&#x02013;189) and Rds2 (a.a. 1&#x02013;97) using probe &#x02018;Z1&#x02019;. Lane 1: probe by itself. Lanes 2 and 3, 0.25 &#x003bc;l Gsm1 (a.a. 1&#x02013;189). and 4.0 &#x003bc;l Rds2 (a.a. 1&#x02013;97), respectively. Lanes 4&#x02013;23: 0.25 &#x003bc;l of Gsm1 (a.a.&#x000a0;1&#x02013;189) and&#x000a0;and 4.0 &#x003bc;l Rds2 (a.a. 1&#x02013;97). (<strong>D</strong>) Quantitative binding of Gsm1 (black rectangles) or Gsm1&#x02013;Rds2 (gray rectangles) to mutants of probe &#x02018;Z1&#x02019; relative to a wild-type probe. Data are an average of three independent experiments.</p></div></div></div></div><div id="SEC3-5" class="sec"><h3 id="SEC3-5title">Cooperative binding <em>in vitro</em> of Ert1 and Rds2 at the <em>PDC1</em> promoter</h3><p class="p p-first">As stated above, Gsm1, Rds2 and Ert1 are highly conserved paralogs raising the possibility that Rds2 may favor binding of Ert1 at specific promoters, as observed with Gsm1. Using ChIP-chip analysis, we previously showed that Ert1 and Rds2 (but not Gsm1, this study) are detected at <em>PDC1</em>, a gene encoding the fermentation enzyme pyruvate decarboxylase (<a href="#B24" rid="B24" class=" bibr popnode">24</a>,<a href="#B26" rid="B26" class=" bibr popnode">26</a>). Thus, these two factors may act together for binding at this promoter. We have shown that the DNA binding domain of Ert1 recognizes a sequence located at position&#x000a0;&#x02212;780 to &#x02212;761 relative to the ATG of the <em>PDC1</em> gene (<a href="#B26" rid="B26" class=" bibr popnode">26</a>). We performed an EMSA using a probe corresponding to these DNA sequences (probe &#x02018;H&#x02019;) and the purified DNA binding domains of Ert1 and Rds2. With increasing concentrations of Rds2, a smear is observed at the edge of the lanes, indicative of a nonstable Rds2&#x02013;DNA complex (Figure <a href="/pmc/articles/PMC10954478/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure8,</span></span><span>8</span></a>, lanes 2&#x02013;5). With Ert1, a weak signal is observed only at the highest concentration of protein used (Figure <a href="/pmc/articles/PMC10954478/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure8,</span></span><span>8</span></a>, lane 9). In contrast, mixing of Rds2 and Ert1 results in binding at concentrations not observed in the presence of a single protein (e.g. compare lanes 4, 8 and 12 in Figure <a href="/pmc/articles/PMC10954478/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure8).</span></span><span>8</span></a>). Thus, Rds2 binds cooperatively with Gsm1 at the <em>FBP1</em> promoter as well as with Ert1 at the <em>PDC1</em> promoter.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F8" co-legend-rid="lgnd_F8"><a href="/pmc/articles/PMC10954478/figure/F8/" target="figure" rid-figpopup="F8" rid-ob="ob-F8"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140432046742768"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10954478_gkad1185fig8.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkad1185fig8.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10954478/bin/gkad1185fig8.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140432046742768"><a target="object" rel="noopener" href="/pmc/articles/PMC10954478/figure/F8/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F8"><div><a class="figpopup" href="/pmc/articles/PMC10954478/figure/F8/" target="figure" rid-figpopup="F8" rid-ob="ob-F8">Figure 8.</a></div><!--caption a7--><div class="caption"><p>Ert1 and Rds2 bind cooperatively to the <em>PDC1</em> promoter. An EMSA was performed with the purified DNA binding domains of Ert1 (a.a. 1&#x02013;152) and Rds2 (a.a. 1&#x02013;97) using <em>PDC1</em> probe &#x02018;H&#x02019; [sequence shown in Figure&#x000a0;<a href="/pmc/articles/PMC10954478/figure/F9/" target="figure" class="fig-table-link figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span>9A</span></a>; also see (<a href="#B26" rid="B26" class=" bibr popnode">26</a>)]. Lane 1: probe alone. Lanes 2&#x02013;5: 0.5, 1.0, 2.0 and 4.0 &#x003bc;l of Rds2 (a.a. 1&#x02013;97). Lanes 6&#x02013;9: 0.025, 0.05, 0.1 and 0.2 &#x003bc;l of Ert1 (a.a. 1&#x02013;152). Lanes 10&#x02013;13: same as lanes 2&#x02013;5 except that 0.1 &#x003bc;l of Ert1 (a.a. 1&#x02013;152) was also added. Lanes 14&#x02013;17: same as lanes 6&#x02013;9 except that 4.0 &#x003bc;l of Rds2 (a.a. 1&#x02013;97) was also added. This is a representative experiment out of two.</p></div></div></div><p class="p">Three CGG triplets are found in probe &#x02018;H&#x02019; (Figure <a href="/pmc/articles/PMC10954478/figure/F9/" target="figure" class="fig-table-link figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure9A).</span></span><span>9A</span></a>). In agreement with our previous observations (<a href="#B26" rid="B26" class=" bibr popnode">26</a>), the EMSA shows that CGG triplet 2 is critical for binding of Ert1 (Figure <a href="/pmc/articles/PMC10954478/figure/F9/" target="figure" class="fig-table-link figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure9B,</span></span><span>9B</span></a>, lane 8, position &#x02018;g&#x02019;). Similarly, mutations at positions &#x02018;e&#x02019;, &#x02018;f&#x02019; and &#x02018;j&#x02019; prevented binding of Ert1, while a mutation at position &#x02018;h&#x02019; reduced its binding (Figure <a href="/pmc/articles/PMC10954478/figure/F9/" target="figure" class="fig-table-link figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure9B).</span></span><span>9B</span></a>). We performed a similar mutational analysis under conditions where Ert1 and Rds2 bind cooperatively to DNA (Figure <a href="/pmc/articles/PMC10954478/figure/F9/" target="figure" class="fig-table-link figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure9C).</span></span><span>9C</span></a>). Strikingly, the pattern of binding was highly similar to the one observed with Ert1 by itself. However, while mutations at positions &#x02018;k&#x02019; and &#x02018;l&#x02019; had no effect on binding of Ert1 by itself, cooperative binding showed some reduced binding. This analysis strongly suggests that Ert1 and Rds2 do not recognize adjacent DNA sites but rather a common DNA stretch, as observed with Gsm1 and Rds2 at the <em>FBP1</em> promoter.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F9" co-legend-rid="lgnd_F9"><a href="/pmc/articles/PMC10954478/figure/F9/" target="figure" rid-figpopup="F9" rid-ob="ob-F9"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140432046730080"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10954478_gkad1185fig9.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkad1185fig9.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10954478/bin/gkad1185fig9.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140432046730080"><a target="object" rel="noopener" href="/pmc/articles/PMC10954478/figure/F9/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F9"><div><a class="figpopup" href="/pmc/articles/PMC10954478/figure/F9/" target="figure" rid-figpopup="F9" rid-ob="ob-F9">Figure 9.</a></div><!--caption a7--><div class="caption"><p>Mutational analysis suggests that Ert1 and Rds2 do not bind at <em>PDC1</em> to adjacent sites but rather to a short common DNA stretch. (<strong>A</strong>) Sequence of the <em>PDC1</em> probe &#x02018;H&#x02019; is shown on the top of the figure with CGG triplets indicated with arrows and numbered 1&#x02013;3. Lowercase&#x000a0;letters (&#x02018;a&#x02019; to &#x02018;l&#x02019;) correspond to nucleotides that were mutated. (<strong>B</strong>) An EMSA was performed with probe &#x02018;H&#x02019; and mutants, as indicated on top of the figure; 0.4 &#x003bc;l of Ert1 (a.a. 1&#x02013;152) was used for the EMSA. (<strong>C</strong>) Lanes 1&#x02013;13: same as&#x000a0;panel (B) except that 0.1 &#x003bc;l of Ert1 (a.a. 1&#x02013;152) and 2.0 &#x003bc;l of Rds2 (a.a. 1&#x02013;97) were used for the EMSA. Lanes 14 and 15: 0.1 &#x003bc;l of Ert1 (a.a. 1&#x02013;152) and 2.0 &#x003bc;l of Rds2 (a.a. 1&#x02013;97), respectively. This is a representative experiment out of two for Ert1 in combination with Rds2.</p></div></div></div></div><div id="SEC3-6" class="sec"><h3 id="SEC3-6title">Cooperative binding <em>in vitro</em> of Ert1 and Rds2 at the <em>HAP4</em> promoter</h3><p class="p p-first">We then focused on the <em>HAP4</em> promoter where Gsm1 and Rds2 can bind to some extent DNA independently of each other (Figure <a href="/pmc/articles/PMC10954478/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1C</span></span><span>1C</span></a> and&#x000a0;Table <a href="/pmc/articles/PMC10954478/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>1</span></a>). Our previous ChIP-chip analysis showed that Ert1 also binds to this promoter (<a href="#B26" rid="B26" class=" bibr popnode">26</a>). With a series of overlapping probes, we scanned the <em>HAP4</em> promoter from &#x02212;901 to &#x02212;715 bp relative to the ATG. No binding was detected when using Gsm1 or Rds2 by themselves, while two sites were identified for Ert1 (data not shown). We focused on one site located at &#x02212;855 to &#x02212;822 bp relative to the ATG (probe &#x02018;C&#x02019; in Figure <a href="/pmc/articles/PMC10954478/figure/F10/" target="figure" class="fig-table-link figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure10).</span></span><span>10</span></a>). With Rds2 by itself, no DNA complex was detected (Figure <a href="/pmc/articles/PMC10954478/figure/F10/" target="figure" class="fig-table-link figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>10A</span></a>, lanes 2&#x02013;5), while an Ert1&#x02013;DNA complex was observed at the highest concentration of protein used (Figure <a href="/pmc/articles/PMC10954478/figure/F10/" target="figure" class="fig-table-link figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>10A</span></a>, lane 9). Upon mixing of Ert1 and Rds2, a complex was observed at concentrations where no complex was observed with single proteins (e.g. compare lanes 4 and 8 with lanes 12 and 16). We looked at other combinations of proteins. Results show that Gsm1 does not bind cooperatively with Rds2 at probe &#x02018;C&#x02019; (Figure <a href="/pmc/articles/PMC10954478/figure/F10/" target="figure" class="fig-table-link figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>10B</span></a>), while similar observations were made with Ert1 in combination with Gsm1 (Figure <a href="/pmc/articles/PMC10954478/figure/F10/" target="figure" class="fig-table-link figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>10C</span></a>). In summary, cooperative binding is observed only with Ert1 and Rds2 at this site of the <em>HAP4</em> promoter.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F10" co-legend-rid="lgnd_F10"><a href="/pmc/articles/PMC10954478/figure/F10/" target="figure" rid-figpopup="F10" rid-ob="ob-F10"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140432085954656"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10954478_gkad1185fig10.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkad1185fig10.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10954478/bin/gkad1185fig10.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140432085954656"><a target="object" rel="noopener" href="/pmc/articles/PMC10954478/figure/F10/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F10"><div><a class="figpopup" href="/pmc/articles/PMC10954478/figure/F10/" target="figure" rid-figpopup="F10" rid-ob="ob-F10">Figure 10.</a></div><!--caption a7--><div class="caption"><p>Rds2 and Ert1 (but not Gsm1) bind cooperatively to a DNA site in the <em>HAP4</em> promoter. EMSAs were performed with probe &#x02018;C&#x02019; located at the <em>HAP4</em> promoter at position &#x02212;855 to &#x02212;822 bp relative to the ATG. The DNA sequence of the probe is shown in Figure <a href="/pmc/articles/PMC10954478/figure/F11/" target="figure" class="fig-table-link figpopup" rid-figpopup="F11" rid-ob="ob-F11" co-legend-rid="lgnd_F11"><span>11A</span></a>. (<strong>A</strong>) Lane 1: probe alone. Lanes 2&#x02013;5: 0.5, 1.0, 2.0 and 4.0 &#x003bc;l of Rds2 (a.a. 1&#x02013;97). Lanes 6&#x02013;9: 0.025, 0.05, 0.1 and 0.2 &#x003bc;l of Ert1 (a.a. 1&#x02013;152). Lanes 10&#x02013;13: same as lanes 2&#x02013;5 except that 0.1 &#x003bc;l of Ert1 (a.a. 1&#x02013;152) was also added. Lanes 14&#x02013;17: same as lanes 6&#x02013;9 except that 0.5 &#x003bc;l of Rds2 was also added. (<strong>B</strong>) Lane 1: probe alone. Lanes 2&#x02013;5: 0.1, 0.2, 0.5 and 1.0 &#x003bc;l of Gsm1 (1&#x02013;189). Lanes 6&#x02013;9: 0.25, 0.5, 1.0 and 2.0 &#x003bc;l of Rds2 (1&#x02013;97). Lanes 10&#x02013;13: same as lanes 6&#x02013;9 except that 0.5 &#x003bc;l of Gsm1 (1&#x02013;189) was also added. Lanes 14&#x02013;17: same as lanes 6&#x02013;9 except that 0.5 &#x003bc;l of Gsm1 (a.a. 1&#x02013;189) was also added. (<strong>C</strong>) Lanes 1&#x02013;4: 0.025, 0.05, 0.1 and 0.2 &#x003bc;l of Ert1 (a.a. 1&#x02013;152). Lanes 5&#x02013;8: 0.1, 0.2, 0.5 and 1.0 &#x003bc;l of Gsm1 (a.a. 1&#x02013;189). Lanes 9&#x02013;12: same as lanes 1&#x02013;4 except that 0.5 &#x003bc;l of Gsm1 (a.a. 1&#x02013;189) was also added. Lanes 13&#x02013;16: same as lanes 5&#x02013;8 except that 0.1 &#x003bc;l of Ert1 (a.a. 1&#x02013;152) was also added. This is a representative experiment out of two.</p></div></div></div><p class="p">We then determine nucleotides that are important for Ert1 binding to probe &#x02018;C&#x02019;. A series of mutants &#x02018;a&#x02019; to &#x02018;p&#x02019; (Figure <a href="/pmc/articles/PMC10954478/figure/F11/" target="figure" class="fig-table-link figpopup" rid-figpopup="F11" rid-ob="ob-F11" co-legend-rid="lgnd_F11"><span>11A</span></a>) were tested in an EMSA with Ert1 by itself (Figure <a href="/pmc/articles/PMC10954478/figure/F11/" target="figure" class="fig-table-link figpopup" rid-figpopup="F11" rid-ob="ob-F11" co-legend-rid="lgnd_F11"><span>11B</span></a>). Results show that mutations at positions &#x02018;g&#x02019;, &#x02018;h&#x02019;, &#x02018;i&#x02019; and&#x000a0;&#x02018;k&#x02019; abolished binding of Ert1, while a mutation at position &#x02018;j&#x02019; greatly reduced binding. We performed a similar experiment to determine which nucleotides are important for cooperative binding of Ert1 and Rds2 (Figure <a href="/pmc/articles/PMC10954478/figure/F11/" target="figure" class="fig-table-link figpopup" rid-figpopup="F11" rid-ob="ob-F11" co-legend-rid="lgnd_F11"><span>11C</span></a>). The pattern of binding is very similar to what was observed with Ert1 by itself. The only difference is with mutation at position &#x02018;j&#x02019; where a stronger binding is observed with mixing of Ert1 and Rds2, although the signal is weaker as compared to a wild-type probe (compare lanes 1 and 11 in Figure <a href="/pmc/articles/PMC10954478/figure/F11/" target="figure" class="fig-table-link figpopup" rid-figpopup="F11" rid-ob="ob-F11" co-legend-rid="lgnd_F11"><span>11C</span></a>). In summary, our analysis of the <em>FBP1</em>, <em>PDC1</em> and <em>HAP4</em> promoters does not support a model where the proteins of interest would bind to adjacent DNA sites. Our results rather favor a model where Rds2 assists Gsm1 or Ert1 for DNA binding via a common DNA site.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F11" co-legend-rid="lgnd_F11"><a href="/pmc/articles/PMC10954478/figure/F11/" target="figure" rid-figpopup="F11" rid-ob="ob-F11"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140432085941584"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10954478_gkad1185fig11.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkad1185fig11.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10954478/bin/gkad1185fig11.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140432085941584"><a target="object" rel="noopener" href="/pmc/articles/PMC10954478/figure/F11/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F11"><div><a class="figpopup" href="/pmc/articles/PMC10954478/figure/F11/" target="figure" rid-figpopup="F11" rid-ob="ob-F11">Figure 11.</a></div><!--caption a7--><div class="caption"><p>Mutational analysis suggests that Ert1 and Rds2 do not bind at <em>HAP4</em> to adjacent sites but rather to a short common DNA stretch. (<strong>A</strong>) Sequence of the <em>HAP4</em> probe &#x02018;C&#x02019; is shown on the top of the figure with CGG triplets indicated with arrows and numbered 1&#x02013;3. Lowercase letters (&#x02018;a&#x02019; to &#x02018;p&#x02019;) correspond to nucleotides that were mutated. (<strong>B</strong>) An EMSA was performed with probe &#x02018;C&#x02019; and mutants, as indicated on top of the figure; 0.3 &#x003bc;l of Ert1 (a.a. 1&#x02013;152) was used for the EMSA. (<strong>C</strong>) Lanes 1&#x02013;17: same as panel (B), except that 0.25 &#x003bc;l of Ert1 (a.a. 1&#x02013;152) and 0.5 &#x003bc;l of Rds2 (a.a. 1&#x02013;97) were used for the EMSA. Lanes 18 and 19: 0.25 &#x003bc;l of Ert1 (a.a. 1&#x02013;152) and 0.5 &#x003bc;l of Rds2 (a.a. 1&#x02013;97), respectively. This is a representative experiment out of three.</p></div></div></div></div><div id="SEC3-7" class="sec sec-last"><h3 id="SEC3-7title">DNA specifies the formation of heterodimers</h3><p class="p p-first">Our results show that the formation of specific heterodimers differs according to the promoter examined. We tested whether DNA itself specifies the formation of different heterodimeric species. Since Ert1 binds to <em>FBP1</em>, <em>PDC1</em> and <em>HAP4</em> (<a href="#B26" rid="B26" class=" bibr popnode">26</a>), we focused on Gsm1 using the <em>HAP4</em> probe &#x02018;C&#x02019;. Figure <a href="/pmc/articles/PMC10954478/figure/F12/" target="figure" class="fig-table-link figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>12A</span></a> shows an alignment of binding sites at the <em>FBP1</em>, <em>HAP4</em> and <em>PDC1</em> promoters. The core sequence CCCGGA is conserved between the <em>FBP1</em> and the&#x000a0;<em>HAP4</em> sites. As expected from results described above (e.g. Figure <a href="/pmc/articles/PMC10954478/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure7),</span></span><span>7</span></a>), Gsm1 binds by itself or in combination with Rds2 to probe &#x02018;Z1&#x02019; of the <em>FBP1</em> promoter (Figure <a href="/pmc/articles/PMC10954478/figure/F12/" target="figure" class="fig-table-link figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>12B</span></a>, lanes 3&#x02013;5). Even though ChIP-chip analysis detected Gsm1 at <em>HAP4</em> (Figure <a href="/pmc/articles/PMC10954478/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1D</span></span><span>1D</span></a> and&#x000a0;Table <a href="/pmc/articles/PMC10954478/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>1</span></a>), this protein does not bind to probe &#x02018;C&#x02019; by itself or in combination with Rds2 (Figure <a href="/pmc/articles/PMC10954478/figure/F12/" target="figure" class="fig-table-link figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>12B</span></a>, lanes 8&#x02013;10). A likely explanation for this observation is that Gsm1 binds to a different site at the <em>HAP4</em> promoter. Downstream of the conserved core sequence, GTT and AAA triplets are present at the <em>FBP1</em> and <em>HAP4</em> sites, respectively (Figure <a href="/pmc/articles/PMC10954478/figure/F12/" target="figure" class="fig-table-link figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>12A</span></a>). Mutating AAA of the <em>HAP4</em> probe to GTT (probe HFH-A) resulted in binding of Gsm1 by itself or in combination with Rds2 (Figure <a href="/pmc/articles/PMC10954478/figure/F12/" target="figure" class="fig-table-link figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>12B</span></a>, lanes 13&#x02013;15). Similar results were obtained with the double mutant gAt (probe HFH-B, where A was not mutated) (Figure <a href="/pmc/articles/PMC10954478/figure/F12/" target="figure" class="fig-table-link figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>12B</span></a>, lanes 18&#x02013;20). In conclusion, DNA by itself specifies the formation of different heterodimeric species.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F12" co-legend-rid="lgnd_F12"><a href="/pmc/articles/PMC10954478/figure/F12/" target="figure" rid-figpopup="F12" rid-ob="ob-F12"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140432051040608"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10954478_gkad1185fig12.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkad1185fig12.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10954478/bin/gkad1185fig12.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140432051040608"><a target="object" rel="noopener" href="/pmc/articles/PMC10954478/figure/F12/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F12"><div><a class="figpopup" href="/pmc/articles/PMC10954478/figure/F12/" target="figure" rid-figpopup="F12" rid-ob="ob-F12">Figure 12.</a></div><!--caption a7--><div class="caption"><p>DNA specifies the formation of heterodimeric species. (<strong>A</strong>) Alignment of binding sites identified in the <em>FBP1</em>, <em>HAP4</em> and <em>PDC1</em> promoters. Sequences of probes &#x02018;Z1&#x02019;, &#x02018;C&#x02019; and &#x02018;H&#x02019; are given in Figures&#x000a0;<a href="/pmc/articles/PMC10954478/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>7A</span></a>, <a href="/pmc/articles/PMC10954478/figure/F11/" target="figure" class="fig-table-link figpopup" rid-figpopup="F11" rid-ob="ob-F11" co-legend-rid="lgnd_F11"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,11</span></span><span>11</span></a><a href="/pmc/articles/PMC10954478/figure/F10/" target="figure" class="fig-table-link figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1em;">11A&#x000a0;and</span></span><span>A</span></a>&#x000a0;and <a href="/pmc/articles/PMC10954478/figure/F9/" target="figure" class="fig-table-link figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span>9</span></a><a href="/pmc/articles/PMC10954478/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">9A,</span></span><span>A</span></a>, respectively. Conserved nucleotides are highlighted. Mutations in the <em>HAP4</em> &#x02018;C&#x02019; site made to resemble an <em>FBP1</em> &#x02018;Z1&#x02019; site (probes HFH-A and HFH-B) are highlighted in gray. (<strong>B</strong>) Amounts of Gsm1 (a.a. 1&#x02013;189) and&#x000a0;Rds2 (a.a. 1&#x02013;97) as well as the probes used in the EMSA are indicated at the bottom of the figure. This is a representative experiment out of two.</p></div></div></div></div></div><div id="SEC4" class="tsec sec"><h2 class="head no_bottom_margin" id="SEC4title" style="text-transform: uppercase;">Discussion</h2><p class="p p-first">This study focused on three paralog transcription factors, the zinc cluster proteins Gsm1, Rds2 and Ert1. A number of yeast transcriptional regulators, such as zinc cluster proteins, have been shown to bind DNA as heterodimers [Pdr1/Pdr3, Pdr1/Stb5 and&#x000a0;Oaf1/Pip2 (<a href="#B40" rid="B40" class=" bibr popnode">40&#x02013;42</a>)]. However, to our knowledge, there is no report of yeast heterodimers&#x02019; identification on a genome-wide scale. Our ChIP-chip analysis of Gsm1 and Rds2 showed a highly similar binding pattern, but it was more complex than initially anticipated with five different modes of binding (Table <a href="/pmc/articles/PMC10954478/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table2).</span></span><span>2</span></a>). Gsm1 and Rds2 bind in an interdependent way to the <em>FBP1</em> promoter, an observation that could be recapitulated <em>in vitro</em> with the purified DNA binding domains of these factors in an EMSA (Figure <a href="/pmc/articles/PMC10954478/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6).</span></span><span>6</span></a>). However, Rds2 is required for binding of Gsm1 to the <em>PCK1</em> promoter but not the reverse, while these factors bind to a large extent independently of each other to the <em>GDB1</em> and the <em>GUP2</em> promoters (Table <a href="/pmc/articles/PMC10954478/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table1).</span></span><span>1</span></a>). The large dependence on Rds2 for Gsm1 DNA binding can be explained by the fact that Rds2 controls expression of Gsm1 (Figure <a href="/pmc/articles/PMC10954478/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2)</span></span><span>2</span></a>) as well as cooperative binding, as shown with EMSAs. We also showed that another zinc cluster factor Ert1 binds <em>in vivo</em> (<a href="#B26" rid="B26" class=" bibr popnode">26</a>) and <em>in vitro</em> (Figure <a href="/pmc/articles/PMC10954478/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure7)</span></span><span>7</span></a>) to the <em>PDC1</em> promoter. EMSA showed that Ert1 cooperates with Rds2 (but not Gsm1) for binding to probe &#x02018;C&#x02019; of the <em>PDC1</em> promoter (Figure <a href="/pmc/articles/PMC10954478/figure/F9/" target="figure" class="fig-table-link figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure9),</span></span><span>9</span></a>), although we cannot exclude Gsm1/Rds2 cooperative binding at a different location of the <em>HAP4</em> promoter. In addition, the zinc cluster protein Sut1 also binds <em>in vivo</em> to the <em>HAP4</em> promoter (<a href="#B43" rid="B43" class=" bibr popnode">43</a>). Gsm1 may cooperate with Sut1 or, alternatively, it may bind by itself to this promoter. In analogy to <em>HAP4</em>, we showed that Rds2 binds cooperatively with Ert1 to the <em>PDC1</em> promoter (Figure <a href="/pmc/articles/PMC10954478/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure8),</span></span><span>8</span></a>), while our ChIP-chip analysis did not detect Gsm1 at this gene.</p><p>Unexpectedly, our EMSA analysis showed that Gsm1 can bind <em>in vitro</em> to DNA upon a slight increase (2-fold) of its concentration (Figure <a href="/pmc/articles/PMC10954478/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6C,</span></span><span>6C</span></a>, compare lanes 4 and 5). It remains to determine whether Gsm1 binds to DNA as a monomer or a homodimer, but we favor the latter possibility (see below). Upon mixing Rds2 and Gsm1, we did not observe the formation of a complex with a different mobility that would provide a direct proof of heterodimerization. One strategy used to show the formation of homodimers (or heterodimers) is to mix short and long forms of the protein(s) of interest resulting in the formation of a DNA complex of intermediate mobility as compared to short&#x02013;short and long&#x02013;long forms (<a href="#B41" rid="B41" class=" bibr popnode">41</a>,<a href="#B44" rid="B44" class=" bibr popnode">44</a>). However, this strategy was not successful with Gsm1 and Rds2 using fusions to three epitopes of HA or MYC or even to a large protein (&#x003b2;-galactosidase) (data not shown). However, the observation that Gsm1 and Rds2 interact with each other in a modified two-hybrid assay (Figure <a href="/pmc/articles/PMC10954478/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2),</span></span><span>2</span></a>), together with cooperative binding <em>in vitro</em>, strongly suggests the formation of heterodimers.</p><p>As stated above, Gsm1 binds <em>in vitro</em> to probe &#x02018;Z1&#x02019; of the <em>FBP1</em> promoter but not to probe &#x02018;C&#x02019; of the <em>HAP4</em> promoter, while Ert1 recognizes both probes (Figures <a href="/pmc/articles/PMC10954478/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -4em;">(Figures7</span></span><span>7</span></a>&#x000a0;and&#x000a0;<a href="/pmc/articles/PMC10954478/figure/F10/" target="figure" class="fig-table-link figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>10&#x02013;12</span></a> and data not shown). However, changing two nucleotides of the <em>HAP4</em> probe to better match the <em>FBP1</em> probe allowed binding of Gsm1 by itself or in combination with Rds2 (Figure <a href="/pmc/articles/PMC10954478/figure/F12/" target="figure" class="fig-table-link figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure12)</span></span><span>12</span></a>) strongly suggesting that DNA specifies the formation of a given heterodimeric species. What is the mode of formation of different heterodimers among Gsm1, Rds2 and Ert1 for cooperative binding? Many transcriptional regulators have been shown to bind as heterodimers with each factor binding to adjacent DNA sequences called half-sites. For example, a number of mammalian nuclear receptors (e.g. retinoid acid receptor/retinoid X receptor, vitamin D receptor/retinoid X receptor) have monomers occupying half-DNA sites (<a href="#B45" rid="B45" class=" bibr popnode">45</a>). Regarding yeast zinc cluster proteins, no structures of heterodimers have been reported. However, crystal structures of the DNA binding domains of Gal4, Ppr1, Put3, Hap1 and Leu3 have shown that these zinc cluster proteins interact with adjacent sites for the formation of homodimers (<a href="#B7" rid="B7" class=" bibr popnode">7</a>,<a href="#B9" rid="B9" class=" bibr popnode">9</a>,<a href="#B12" rid="B12" class=" bibr popnode">12</a>,<a href="#B46" rid="B46" class=" bibr popnode">46</a>,<a href="#B47" rid="B47" class=" bibr popnode">47</a>).</p><p>In Figure <a href="/pmc/articles/PMC10954478/figure/F13/" target="figure" class="fig-table-link figpopup" rid-figpopup="F13" rid-ob="ob-F13" co-legend-rid="lgnd_F13"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure13,</span></span><span>13</span></a>, we propose a model for DNA binding of Gsm1 by itself or in combination with Rds2 at <em>FBP1</em> and related sites. As stated above, Leu3 and Pdr3 recognize an everted repeat with a spacing of 4 bp (CCGN<sub>4</sub>CGG) and 0 bp (CCGCGG), respectively (<a href="#B11" rid="B11" class=" bibr popnode">11</a>,<a href="#B12" rid="B12" class=" bibr popnode">12</a>). Two overlapping CGG triplets in opposite directions (CCGG) are essential for Gsm1 binding (Figure <a href="/pmc/articles/PMC10954478/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6).</span></span><span>6</span></a>). This corresponds to an even more compacted version of the Pdr3 and Leu3 DNA sites. Additional nucleotides (G and A) downstream of the top CGG triplet are also essential for DNA recognition. Regarding the CGG triplet located on the bottom strand, mutational analysis showed that only one nucleotide upstream of this triplet affects binding (2-fold) of Gsm1 (Figure <a href="/pmc/articles/PMC10954478/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6B).</span></span><span>6B</span></a>). These observations suggest that Gsm1 binds as a homodimer to an imperfect everted repeat. In contrast, mutations of the nucleotides upstream of the CGG triplets did not affect binding of Gsm1/Rds2 (Figure <a href="/pmc/articles/PMC10954478/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6C).</span></span><span>6C</span></a>). Similarly, we could not identify mutations downstream of the top CGG triplet that would affect binding of the Gsm1/Rds2 heterodimer but not Gsm1 by itself. Similar results were obtained at <em>PDC1</em> and <em>HAP4</em> (Figures <a href="/pmc/articles/PMC10954478/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -4em;">(Figures8</span></span><span>8</span></a> and&#x000a0;<a href="/pmc/articles/PMC10954478/figure/F10/" target="figure" class="fig-table-link figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>10</span></a>). In summary, our data do not favor a mode of DNA binding where Gsm1 and Rds2 or Ert1 and Rds2 heterodimers would recognize adjacent half-sites. Does Rds2 contact DNA? With Rds2 by itself, no well-defined Rds2&#x02013;DNA complex was observed in our various EMSAs. However, at high Rds2 concentrations, a smear is observed indicative of an unstable complex (e.g. see Figure <a href="/pmc/articles/PMC10954478/figure/F9/" target="figure" class="fig-table-link figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure9C,</span></span><span>9C</span></a>, lane 15). Thus, Rds2 has limited DNA binding capacity but acts as a modulator of DNA binding by interacting with Gsm1 or Ert1. It is likely that the weak DNA binding of Rds2 induces a conformational change of Gsm1 resulting in increased binding of these two factors. However, our data do not allow distinguishing between this model or one that would not involve a conformational change. Our unpublished data show that a shorter version of Rds2 (a.a. 1&#x02013;81 instead of a.a. 1&#x02013;97) does not allow cooperative binding with Gsm1 (a.a. 1&#x02013;187). This suggests that a.a. 82&#x02013;97 are critical for cooperative binding maybe by allowing an Rds2&#x02013;Gsm1 interaction. These data suggest a novel type of DNA recognition for cooperative binding of zinc cluster proteins.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F13" co-legend-rid="lgnd_F13"><a href="/pmc/articles/PMC10954478/figure/F13/" target="figure" rid-figpopup="F13" rid-ob="ob-F13"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140432042306224"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10954478_gkad1185fig13.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkad1185fig13.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10954478/bin/gkad1185fig13.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140432042306224"><a target="object" rel="noopener" href="/pmc/articles/PMC10954478/figure/F13/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F13"><div><a class="figpopup" href="/pmc/articles/PMC10954478/figure/F13/" target="figure" rid-figpopup="F13" rid-ob="ob-F13">Figure 13.</a></div><!--caption a7--><div class="caption"><p>Model for binding of Gsm1 and Rds2. Gsm1 and Rds2 are shown bound to different DNA sites with nucleotides involved in Gsm1 binding in bold characters. Dark and light highlighted sectors correspond to&#x000a0;nucleotides essential and nucleotides less critical for Gsm1 binding, respectively. Arrows correspond to overlapping CGG triplets found in opposite directions (an everted repeat). The triple black lines correspond to dimerization domains. (<strong>A</strong>) Gsm1 bound by itself to an&#x000a0;<em>FBP1</em> site. (<strong>B</strong>) Gsm1 bound as a heterodimer with Rds2 to an&#x000a0;<em>FBP1</em> site. (<strong>C</strong>) Gsm1 does not bind to <em>HAP4</em> due to nucleotide differences (&#x02018;X&#x02019;) at that site. (<strong>D</strong>) Gsm1 bound to a hybrid <em>HAP4</em>&#x02013;<em>FBP1</em> site. Mutations corresponding to nucleotides found at the <em>FBP1</em> site are shown by lowercase letters.</p></div></div></div><p class="p p-last">Heterodimer formation increases transcription factor&#x000a0;specificity, binding affinity to DNA and diversity of target sites. As our results have shown, Gsm1 and Ert1 by themselves are capable of binding DNA independently of each other, but in some cases&#x000a0;the Gsm1&#x02013;Rds2 heterodimer increases the efficiency of binding, recognizing novel DNA elements that Gsm1 cannot recognize by itself (e.g. <em>PCK1</em>). Moreover, we have identified a novel mode of DNA recognition for zinc cluster proteins with binding of two factors at a common DNA site. Heterodimerization allows the formation of complex regulatory networks essential for gene expression and, as a result, yeast can better respond to environmental cues like reduced glucose levels triggering a switch from fermentation to a respiratory mode. In summary, our results reveal a novel mode of transcriptional regulation with the formation of different heterodimers at specific promoters through recognition of a short and common DNA element. With over 175 000 known or putative zinc cluster proteins (<a href="https://www.supfam.org/SUPERFAMILY/cgi-bin/genome.cgi?sf=57701" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.supfam.org/SUPERFAMILY/cgi-bin/genome.cgi?sf=57701</a>), our study will be invaluable to characterize the mode of action of these transcriptional regulators in other fungal species.</p></div><div id="sec-a.w.e" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="sec-a.w.etitle" style="text-transform: uppercase;">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="sup1"><h4>gkad1185_Supplemental_File</h4><div class="sup-box half_rhythm" id="media-a.w.e.b.b"><a href="/pmc/articles/PMC10954478/bin/gkad1185_supplemental_file.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(199K, pdf)</sup></div></div></div><div id="ACK1" class="tsec sec"><h2 class="head no_bottom_margin" id="ACK1title" style="text-transform: uppercase;">Acknowledgements</h2><div class="sec"><p>We thank Drs Cunle Wu and Malcolm Whiteway for providing strains and plasmids used in the SYRTH assay. We also thank Dr St&#x000e9;phane Laporte (McGill University) for useful comments.</p></div></div><div id="article-aaff-info" class="tsec sec"><h2 class="head no_bottom_margin" id="article-aaff-infotitle" style="text-transform: uppercase;">Contributor Information</h2><p><span class="fm-affl">Karla P&#x000e1;ez Martinez, </span><span class="fm-affl" id="aff-a.v.b.f.a.b"> 
Department of Medicine, McGill University Health Centre, 1001 Boul. D&#x000e9;carie, Room E02.7212, Montr&#x000e9;al, QC H4A 3J1,&#x000a0;Canada.</span></p><p><span class="fm-affl">Najla Gasmi, </span><span class="fm-affl" id="aff-a.v.b.f.b.b"> 
Department of Biochemistry, McGill University, 1001 Boul. D&#x000e9;carie, Room E02.7212, Montr&#x000e9;al, QC H4A 3J1,&#x000a0;Canada.</span></p><p><span class="fm-affl">C&#x000e9;lia Jeronimo, </span><span class="fm-affl" id="aff-a.v.b.f.c.b"> 
Institut de recherches cliniques de Montr&#x000e9;al, 110 avenue des Pins Ouest, Montr&#x000e9;al, QC H2W 1R7, Canada.</span></p><p><span class="fm-affl">Natalia Klimova, </span><span class="fm-affl" id="aff-a.v.b.f.d.b"> 
Department of Medicine, McGill University Health Centre, 1001 Boul. D&#x000e9;carie, Room E02.7212, Montr&#x000e9;al, QC H4A 3J1,&#x000a0;Canada.</span></p><p><span class="fm-affl">Fran&#x000e7;ois Robert, </span><span class="fm-affl" id="aff-a.v.b.f.e.b"> 
Institut de recherches cliniques de Montr&#x000e9;al, 110 avenue des Pins Ouest, Montr&#x000e9;al, QC H2W 1R7, Canada.</span><span class="fm-affl" id="aff-a.v.b.f.e.c"> 
D&#x000e9;partement de M&#x000e9;decine, Facult&#x000e9; de M&#x000e9;decine, Universit&#x000e9; de Montr&#x000e9;al, 2900 Boul. &#x000c9;douard-Montpetit, Montr&#x000e9;al, QC H3T 1J4, Canada.</span></p><p><span class="fm-affl">Bernard Turcotte, </span><span class="fm-affl" id="aff-a.v.b.f.f.c"> 
Department of Medicine, McGill University Health Centre, 1001 Boul. D&#x000e9;carie, Room E02.7212, Montr&#x000e9;al, QC H4A 3J1,&#x000a0;Canada.</span><span class="fm-affl" id="aff-a.v.b.f.f.d"> 
Department of Biochemistry, McGill University, 1001 Boul. D&#x000e9;carie, Room E02.7212, Montr&#x000e9;al, QC H4A 3J1,&#x000a0;Canada.</span><span class="fm-affl" id="aff-a.v.b.f.f.e"> 
Department of Microbiology and Immunology, McGill University, 1001 Boul. D&#x000e9;carie, Room E02.7212, Montr&#x000e9;al, QC H4A 3J1,&#x000a0;Canada.</span></p></div><div id="SEC5" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="SEC5title" style="text-transform: uppercase;">Data availability</h2><!--/article/back/sec/--><p class="p p-first-last">All microarray data have been deposited into Gene Expression Omnibus archive under accession number <a id="__tag_1145552255" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE225348" ref="/geo/query/acc.cgi?acc=GSE225348"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE225348","term_id":"225348"}}</span>GSE225348</a>.</p></div><div id="SEC6" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="SEC6title" style="text-transform: uppercase;">Supplementary data</h2><!--/article/back/sec/--><p class="p p-first-last">
<a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkad1185#supplementary-data" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Data</a> are available at NAR Online.</p></div><div id="SEC7" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="SEC7title" style="text-transform: uppercase;">Funding</h2><!--/article/back/sec/--><p class="p p-first">Natural Sciences and Engineering Research Council of Canada (to B.T.);&#x000a0;Fonds de recherche du Qu&#x000e9;bec-Nature et technologies (to B.T.);&#x000a0;Canadian Institutes of Health Research [PJT-162334 and PJT-156383 to F.R.]. Funding for open access charge: Research Institute of the McGill University Health&#x000a0;Centre.</p><p class="p p-last">
<em>Conflict of interest statement</em>. None declared.</p></div><div id="REF1" class="tsec sec"><h2 class="head no_bottom_margin" id="REF1title" style="text-transform: uppercase;">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1">1. <span class="mixed-citation">

Turcotte &#x000a0;B., Liang &#x000a0;X.B., Robert &#x000a0;F., Soontorngun &#x000a0;N. &#x000a0;<span class="ref-title">Transcriptional regulation of nonfermentable carbon utilization in budding yeast</span>. <span class="ref-journal">FEMS Yeast Res.</span> &#x000a0;2010; <span class="ref-vol">10</span>:2&#x02013;13.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5003605/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19686338" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEMS+Yeast+Res.&amp;title=Transcriptional+regulation+of+nonfermentable+carbon+utilization+in+budding+yeast&amp;volume=10&amp;publication_year=2010&amp;pages=2-13&amp;pmid=19686338&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B2">2. <span class="mixed-citation">

Sch&#x000fc;ller &#x000a0;H.-J. &#x000a0;<span class="ref-title">Transcriptional control of nonfermentative metabolism in the yeast <em>Saccharomyces</em><em>cerevisiae</em></span>. <span class="ref-journal">Curr. Genet.</span> &#x000a0;2003; <span class="ref-vol">43</span>:139&#x02013;160.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/12715202" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Genet.&amp;title=Transcriptional+control+of+nonfermentative+metabolism+in+the+yeast+Saccharomycescerevisiae&amp;volume=43&amp;publication_year=2003&amp;pages=139-160&amp;pmid=12715202&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B3">3. <span class="mixed-citation">

DeRisi &#x000a0;J.L., Iyer &#x000a0;V.R., Brown &#x000a0;P.O. &#x000a0;<span class="ref-title">Exploring the metabolic and genetic control of gene expression on a genomic scale</span>. <span class="ref-journal">Science</span>. 1997; <span class="ref-vol">278</span>:680&#x02013;686.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/9381177" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Exploring+the+metabolic+and+genetic+control+of+gene+expression+on+a+genomic+scale&amp;volume=278&amp;publication_year=1997&amp;pages=680-686&amp;pmid=9381177&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B4">4. <span class="mixed-citation">

Roberts &#x000a0;G.G., Hudson &#x000a0;A.P. &#x000a0;<span class="ref-title">Transcriptome profiling of <em>Saccharomyces</em><em>cerevisiae</em> during a transition from fermentative to glycerol-based respiratory growth reveals extensive metabolic and structural remodeling</span>. <span class="ref-journal">Mol. Genet. Genomics</span>. 2006; <span class="ref-vol">276</span>:170&#x02013;186.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/16741729" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Genet.+Genomics&amp;title=Transcriptome+profiling+of+Saccharomycescerevisiae+during+a+transition+from+fermentative+to+glycerol-based+respiratory+growth+reveals+extensive+metabolic+and+structural+remodeling&amp;volume=276&amp;publication_year=2006&amp;pages=170-186&amp;pmid=16741729&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B5">5. <span class="mixed-citation">

Hsieh &#x000a0;W.-C., Sutter &#x000a0;B.M., Ruess &#x000a0;H., Barnes &#x000a0;S.D., Malladi &#x000a0;V.S., Tu &#x000a0;B.P. &#x000a0;<span class="ref-title">Glucose starvation induces a switch in the histone acetylome for activation of gluconeogenic and fat metabolism genes</span>. <span class="ref-journal">Mol. Cell</span>. 2022; <span class="ref-vol">82</span>:60&#x02013;74.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8794035/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34995509" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=Glucose+starvation+induces+a+switch+in+the+histone+acetylome+for+activation+of+gluconeogenic+and+fat+metabolism+genes&amp;volume=82&amp;publication_year=2022&amp;pages=60-74&amp;pmid=34995509&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B6">6. <span class="mixed-citation">

MacPherson &#x000a0;S., Larochelle &#x000a0;M., Turcotte &#x000a0;B. &#x000a0;<span class="ref-title">A fungal family of transcriptional regulators: the zinc cluster proteins</span>. <span class="ref-journal">Microbiol. Mol. Biol. Rev.</span> &#x000a0;2006; <span class="ref-vol">70</span>:583&#x02013;604.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1594591/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16959962" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol.+Mol.+Biol.+Rev.&amp;title=A+fungal+family+of+transcriptional+regulators:+the+zinc+cluster+proteins&amp;volume=70&amp;publication_year=2006&amp;pages=583-604&amp;pmid=16959962&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B7">7. <span class="mixed-citation">

Marmorstein &#x000a0;R., Carey &#x000a0;M., Ptashne &#x000a0;M., Harrison &#x000a0;S.C. &#x000a0;<span class="ref-title">DNA recognition by GAL4: structure of a protein&#x02013;DNA complex</span>. <span class="ref-journal">Nature</span>. 1992; <span class="ref-vol">356</span>:408&#x02013;414.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/1557122" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=DNA+recognition+by+GAL4:+structure+of+a+protein&#x02013;DNA+complex&amp;volume=356&amp;publication_year=1992&amp;pages=408-414&amp;pmid=1557122&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B8">8. <span class="mixed-citation">

Ha &#x000a0;N., Hellauer &#x000a0;K., Turcotte &#x000a0;B. &#x000a0;<span class="ref-title">Mutations in target DNA elements of yeast HAP1 modulate its transcriptional activity without affecting DNA binding</span>. <span class="ref-journal">Nucleic Acids Res.</span> &#x000a0;1996; <span class="ref-vol">24</span>:1453&#x02013;1459.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC145800/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8628677" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Mutations+in+target+DNA+elements+of+yeast+HAP1+modulate+its+transcriptional+activity+without+affecting+DNA+binding&amp;volume=24&amp;publication_year=1996&amp;pages=1453-1459&amp;pmid=8628677&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B9">9. <span class="mixed-citation">

King &#x000a0;D.A., Zhang &#x000a0;L., Guarente &#x000a0;L., Marmorstein &#x000a0;R. &#x000a0;<span class="ref-title">Structure of a HAP1&#x02013;DNA complex reveals dramatically asymmetric DNA binding by a homodimeric protein</span>. <span class="ref-journal">Nat. Struct. Biol.</span> &#x000a0;1999; <span class="ref-vol">6</span>:64&#x02013;71.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/9886294" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Struct.+Biol.&amp;title=Structure+of+a+HAP1&#x02013;DNA+complex+reveals+dramatically+asymmetric+DNA+binding+by+a+homodimeric+protein&amp;volume=6&amp;publication_year=1999&amp;pages=64-71&amp;pmid=9886294&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B10">10. <span class="mixed-citation">

Zhang &#x000a0;L., Guarente &#x000a0;L. &#x000a0;<span class="ref-title">The yeast activator HAP1&#x02014;a GAL4 family member&#x02014;binds DNA in a directly repeated orientation</span>. <span class="ref-journal">Genes Dev.</span> &#x000a0;1994; <span class="ref-vol">8</span>:2110&#x02013;2119.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/7958882" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev.&amp;title=The+yeast+activator+HAP1&#x02014;a+GAL4+family+member&#x02014;binds+DNA+in+a+directly+repeated+orientation&amp;volume=8&amp;publication_year=1994&amp;pages=2110-2119&amp;pmid=7958882&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B11">11. <span class="mixed-citation">

Hellauer &#x000a0;K., Rochon &#x000a0;M.H., Turcotte &#x000a0;B. &#x000a0;<span class="ref-title">A novel DNA binding motif for yeast zinc cluster proteins: the Leu3p and Pdr3p transcriptional activators recognize everted repeats</span>. <span class="ref-journal">Mol. Cell. Biol.</span> &#x000a0;1996; <span class="ref-vol">16</span>:6096&#x02013;6102.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC231612/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8887639" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Biol.&amp;title=A+novel+DNA+binding+motif+for+yeast+zinc+cluster+proteins:+the+Leu3p+and+Pdr3p+transcriptional+activators+recognize+everted+repeats&amp;volume=16&amp;publication_year=1996&amp;pages=6096-6102&amp;pmid=8887639&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B12">12. <span class="mixed-citation">

Fitzgerald &#x000a0;M.X., Rojas &#x000a0;J.R., Kim &#x000a0;J.M., Kohlhaw &#x000a0;G.B., Marmorstein &#x000a0;R. &#x000a0;<span class="ref-title">Structure of a Leu3&#x02013;DNA complex: recognition of everted CGG half-sites by a Zn<sub>2</sub>Cys<sub>6</sub> binuclear cluster protein</span>. <span class="ref-journal">Structure</span>. 2006; <span class="ref-vol">14</span>:725&#x02013;735.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/16615914" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=Structure+of+a+Leu3&#x02013;DNA+complex:+recognition+of+everted+CGG+half-sites+by+a+Zn2Cys6+binuclear+cluster+protein&amp;volume=14&amp;publication_year=2006&amp;pages=725-735&amp;pmid=16615914&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B13">13. <span class="mixed-citation">

Cahuzac &#x000a0;B., Cerdan &#x000a0;R., Felenbok &#x000a0;B., Guittet &#x000a0;E. &#x000a0;<span class="ref-title">The solution structure of an AlcR&#x02013;DNA complex sheds light onto the unique tight and monomeric DNA binding of a Zn<sub>2</sub>Cys<sub>6</sub> protein</span>. <span class="ref-journal">Structure</span>. 2001; <span class="ref-vol">9</span>:827&#x02013;836.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/11566132" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=The+solution+structure+of+an+AlcR&#x02013;DNA+complex+sheds+light+onto+the+unique+tight+and+monomeric+DNA+binding+of+a+Zn2Cys6+protein&amp;volume=9&amp;publication_year=2001&amp;pages=827-836&amp;pmid=11566132&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B14">14. <span class="mixed-citation">

Broach &#x000a0;J.R. &#x000a0;<span class="ref-title">Nutritional control of growth and development in yeast</span>. <span class="ref-journal">Genetics</span>. 2012; <span class="ref-vol">192</span>:73&#x02013;105.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3430547/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22964838" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Nutritional+control+of+growth+and+development+in+yeast&amp;volume=192&amp;publication_year=2012&amp;pages=73-105&amp;pmid=22964838&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B15">15. <span class="mixed-citation">

Zhang &#x000a0;J., Vemuri &#x000a0;G., Nielsen &#x000a0;J. &#x000a0;<span class="ref-title">Systems biology of energy homeostasis in yeast</span>. <span class="ref-journal">Curr. Opin. Microbiol.</span> &#x000a0;2010; <span class="ref-vol">13</span>:382&#x02013;388.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/20439164" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Microbiol.&amp;title=Systems+biology+of+energy+homeostasis+in+yeast&amp;volume=13&amp;publication_year=2010&amp;pages=382-388&amp;pmid=20439164&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B16">16. <span class="mixed-citation">

Hedbacker &#x000a0;K., Carlson &#x000a0;M. &#x000a0;<span class="ref-title">SNF1/AMPK pathways in yeast</span>. <span class="ref-journal">Front. Biosci.</span> &#x000a0;2008; <span class="ref-vol">13</span>:2408&#x02013;2420.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2685184/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17981722" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Biosci.&amp;title=SNF1/AMPK+pathways+in+yeast&amp;volume=13&amp;publication_year=2008&amp;pages=2408-2420&amp;pmid=17981722&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B17">17. <span class="mixed-citation">

Zaman &#x000a0;S., Lippman &#x000a0;S.I., Zhao &#x000a0;X., Broach &#x000a0;J.R. &#x000a0;<span class="ref-title">How <em>Saccharomyces</em> responds to nutrients</span>. <span class="ref-journal">Annu. Rev. Genet.</span> &#x000a0;2008; <span class="ref-vol">42</span>:27&#x02013;81.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/18303986" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Genet.&amp;title=How+Saccharomyces+responds+to+nutrients&amp;volume=42&amp;publication_year=2008&amp;pages=27-81&amp;pmid=18303986&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B18">18. <span class="mixed-citation">

Hedges &#x000a0;D., Proft &#x000a0;M., Entian &#x000a0;K.D. &#x000a0;<span class="ref-title">CAT8, a new zinc cluster-encoding gene necessary for derepression of gluconeogenic enzymes in the yeast <em>Saccharomyces</em><em>cerevisiae</em></span>. <span class="ref-journal">Mol. Cell. Biol.</span> &#x000a0;1995; <span class="ref-vol">15</span>:1915&#x02013;1922.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC230417/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7891685" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Biol.&amp;title=CAT8,+a+new+zinc+cluster-encoding+gene+necessary+for+derepression+of+gluconeogenic+enzymes+in+the+yeast+Saccharomycescerevisiae&amp;volume=15&amp;publication_year=1995&amp;pages=1915-1922&amp;pmid=7891685&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B19">19. <span class="mixed-citation">

Rahner &#x000a0;A., Sch&#x000f6;ler &#x000a0;A., Martens &#x000a0;E., Gollwitzer &#x000a0;B., Sch&#x000fc;ller &#x000a0;H.J. &#x000a0;<span class="ref-title">Dual influence of the yeast Cat1p (Snf1p) protein kinase on carbon source-dependent transcriptional activation of gluconeogenic genes by the regulatory gene CAT8</span>. <span class="ref-journal">Nucleic Acids Res.</span> &#x000a0;1996; <span class="ref-vol">24</span>:2331&#x02013;2337.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC145921/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8710504" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Dual+influence+of+the+yeast+Cat1p+(Snf1p)+protein+kinase+on+carbon+source-dependent+transcriptional+activation+of+gluconeogenic+genes+by+the+regulatory+gene+CAT8&amp;volume=24&amp;publication_year=1996&amp;pages=2331-2337&amp;pmid=8710504&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B20">20. <span class="mixed-citation">

Vincent &#x000a0;O., Carlson &#x000a0;M. &#x000a0;<span class="ref-title">Sip4, a Snf1 kinase-dependent transcriptional activator, binds to the carbon source-responsive element of gluconeogenic genes</span>. <span class="ref-journal">EMBO J.</span> &#x000a0;1998; <span class="ref-vol">17</span>:7002&#x02013;7008.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1171048/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9843506" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+J.&amp;title=Sip4,+a+Snf1+kinase-dependent+transcriptional+activator,+binds+to+the+carbon+source-responsive+element+of+gluconeogenic+genes&amp;volume=17&amp;publication_year=1998&amp;pages=7002-7008&amp;pmid=9843506&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B21">21. <span class="mixed-citation">

Tachibana &#x000a0;C., Yoo &#x000a0;J.Y., Tagne &#x000a0;J.-B., Kacherovsky &#x000a0;N., Lee &#x000a0;T.I., Young &#x000a0;E.T. &#x000a0;<span class="ref-title">Combined global localization analysis and transcriptome data identify genes that are directly coregulated by Adr1 and Cat8</span>. <span class="ref-journal">Mol. Cell. Biol.</span> &#x000a0;2005; <span class="ref-vol">25</span>:2138&#x02013;2146.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1061606/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15743812" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Biol.&amp;title=Combined+global+localization+analysis+and+transcriptome+data+identify+genes+that+are+directly+coregulated+by+Adr1+and+Cat8&amp;volume=25&amp;publication_year=2005&amp;pages=2138-2146&amp;pmid=15743812&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B22">22. <span class="mixed-citation">

Akache &#x000a0;B., Wu &#x000a0;K., Turcotte &#x000a0;B. &#x000a0;<span class="ref-title">Phenotypic analysis of genes encoding yeast zinc cluster proteins</span>. <span class="ref-journal">Nucleic Acids Res.</span> &#x000a0;2001; <span class="ref-vol">29</span>:2181&#x02013;2190.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC55459/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11353088" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Phenotypic+analysis+of+genes+encoding+yeast+zinc+cluster+proteins&amp;volume=29&amp;publication_year=2001&amp;pages=2181-2190&amp;pmid=11353088&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B23">23. <span class="mixed-citation">

Akache &#x000a0;B., Turcotte &#x000a0;B. &#x000a0;<span class="ref-title">New regulators of drug sensitivity in the family of yeast zinc cluster proteins</span>. <span class="ref-journal">J. Biol. Chem.</span> &#x000a0;2002; <span class="ref-vol">277</span>:21254&#x02013;21260.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/11943786" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=New+regulators+of+drug+sensitivity+in+the+family+of+yeast+zinc+cluster+proteins&amp;volume=277&amp;publication_year=2002&amp;pages=21254-21260&amp;pmid=11943786&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B24">24. <span class="mixed-citation">

Soontorngun &#x000a0;N., Larochelle &#x000a0;M., Drouin &#x000a0;S., Robert &#x000a0;F., Turcotte &#x000a0;B. &#x000a0;<span class="ref-title">Regulation of gluconeogenesis in S<em>accharomyces</em><em>cerevisiae</em> is mediated by activator and repressor functions of Rds2</span>. <span class="ref-journal">Mol. Cell. Biol.</span> &#x000a0;2007; <span class="ref-vol">27</span>:7895&#x02013;7905.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2169140/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17875938" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Biol.&amp;title=Regulation+of+gluconeogenesis+in+Saccharomycescerevisiae+is+mediated+by+activator+and+repressor+functions+of+Rds2&amp;volume=27&amp;publication_year=2007&amp;pages=7895-7905&amp;pmid=17875938&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B25">25. <span class="mixed-citation">

Forsburg &#x000a0;S.L., Guarente &#x000a0;L. &#x000a0;<span class="ref-title">Identification and characterization of HAP4: a third component of the CCAAT-bound HAP2/HAP3 heteromer</span>. <span class="ref-journal">Genes Dev.</span> &#x000a0;1989; <span class="ref-vol">3</span>:1166&#x02013;1178.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/2676721" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev.&amp;title=Identification+and+characterization+of+HAP4:+a+third+component+of+the+CCAAT-bound+HAP2/HAP3+heteromer&amp;volume=3&amp;publication_year=1989&amp;pages=1166-1178&amp;pmid=2676721&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B26">26. <span class="mixed-citation">

Gasmi &#x000a0;N., Jacques &#x000a0;P.-E., Klimova &#x000a0;N., Guo &#x000a0;X., Ricciardi &#x000a0;A., Robert &#x000a0;F., Turcotte &#x000a0;B. &#x000a0;<span class="ref-title">The switch from fermentation to respiration in <em>Saccharomyces</em><em>cerevisiae</em> is regulated by the Ert1 transcriptional activator/repressor</span>. <span class="ref-journal">Genetics</span>. 2014; <span class="ref-vol">198</span>:547&#x02013;560.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4196612/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25123508" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=The+switch+from+fermentation+to+respiration+in+Saccharomycescerevisiae+is+regulated+by+the+Ert1+transcriptional+activator/repressor&amp;volume=198&amp;publication_year=2014&amp;pages=547-560&amp;pmid=25123508&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B27">27. <span class="mixed-citation">

van&#x000a0;Bakel &#x000a0;H., van&#x000a0;Werven &#x000a0;F.J., Radonjic &#x000a0;M., Brok &#x000a0;M.O., van&#x000a0;Leenen &#x000a0;D., Holstege &#x000a0;F.C.P., Timmers &#x000a0;H.T.M. &#x000a0;<span class="ref-title">Improved genome-wide localization by ChIP-chip using double-round T7 RNA polymerase-based amplification</span>. <span class="ref-journal">Nucleic Acids Res.</span> &#x000a0;2008; <span class="ref-vol">36</span>:e21.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2275083/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18180247" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Improved+genome-wide+localization+by+ChIP-chip+using+double-round+T7+RNA+polymerase-based+amplification&amp;volume=36&amp;publication_year=2008&amp;pages=e21&amp;pmid=18180247&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B28">28. <span class="mixed-citation">

Adams &#x000a0;A., Gottschling &#x000a0;D.E., Kaiser &#x000a0;C.A., Stearns &#x000a0;T. &#x000a0;<span class="ref-journal">Methods in Yeast Genetics</span>. 1998; Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Methods+in+Yeast+Genetics&amp;publication_year=1998&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B29">29. <span class="mixed-citation">

Brachmann &#x000a0;C.B., Davies &#x000a0;A., Cost &#x000a0;G.J., Caputo &#x000a0;E., Li &#x000a0;J., Hieter &#x000a0;P., Boeke &#x000a0;J.D. &#x000a0;<span class="ref-title">Designer deletion strains derived from <em>Saccharomyces</em><em>cerevisiae</em> S288C: a useful set of strains and plasmids for PCR-mediated gene disruption and other applications</span>. <span class="ref-journal">Yeast</span>. 1998; <span class="ref-vol">14</span>:115&#x02013;132.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/9483801" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Yeast&amp;title=Designer+deletion+strains+derived+from+Saccharomycescerevisiae+S288C:+a+useful+set+of+strains+and+plasmids+for+PCR-mediated+gene+disruption+and+other+applications&amp;volume=14&amp;publication_year=1998&amp;pages=115-132&amp;pmid=9483801&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B30">30. <span class="mixed-citation">

Schneider &#x000a0;B.L., Seufert &#x000a0;W., Steiner &#x000a0;B., Yang &#x000a0;Q.H., Futcher &#x000a0;A.B. &#x000a0;<span class="ref-title">Use of polymerase chain reaction epitope tagging for protein tagging in <em>Saccharomyces cerevisiae</em></span>. <span class="ref-journal">Yeast</span>. 1995; <span class="ref-vol">11</span>:1265&#x02013;1274.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/8553697" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Yeast&amp;title=Use+of+polymerase+chain+reaction+epitope+tagging+for+protein+tagging+in+Saccharomyces+cerevisiae&amp;volume=11&amp;publication_year=1995&amp;pages=1265-1274&amp;pmid=8553697&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B31">31. <span class="mixed-citation">

Winzeler &#x000a0;E.A., Shoemaker &#x000a0;D.D., Astromoff &#x000a0;A., Liang &#x000a0;H., Anderson &#x000a0;K., Andre &#x000a0;B., Bangham &#x000a0;R., Benito &#x000a0;R., Boeke &#x000a0;J.D., Bussey &#x000a0;H. &#x000a0;et al.. &#x000a0;<span class="ref-title">Functional characterization of the <em>S. cerevisiae</em> genome by gene deletion and parallel analysis</span>. <span class="ref-journal">Science</span>. 1999; <span class="ref-vol">285</span>:901&#x02013;906.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/10436161" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Functional+characterization+of+the+S.+cerevisiae+genome+by+gene+deletion+and+parallel+analysis&amp;volume=285&amp;publication_year=1999&amp;pages=901-906&amp;pmid=10436161&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B32">32. <span class="mixed-citation">

Mallick &#x000a0;J., Jansen &#x000a0;G., Wu &#x000a0;C., Whiteway &#x000a0;M. &#x000a0;<span class="ref-title">SRYTH: a new yeast two-hybrid method</span>. <span class="ref-journal">Methods Mol. Biol.</span> &#x000a0;2016; <span class="ref-vol">1356</span>:31&#x02013;41.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/26519063" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Mol.+Biol.&amp;title=SRYTH:+a+new+yeast+two-hybrid+method&amp;volume=1356&amp;publication_year=2016&amp;pages=31-41&amp;pmid=26519063&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B33">33. <span class="mixed-citation">

Collin &#x000a0;P., Jeronimo &#x000a0;C., Poitras &#x000a0;C., Robert &#x000a0;F. &#x000a0;<span class="ref-title">RNA polymerase II CTD tyrosine 1 is required for efficient termination by the Nrd1&#x02013;Nab3&#x02013;Sen1 pathway</span>. <span class="ref-journal">Mol. Cell</span>. 2019; <span class="ref-vol">73</span>:655&#x02013;669.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/30639244" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=RNA+polymerase+II+CTD+tyrosine+1+is+required+for+efficient+termination+by+the+Nrd1&#x02013;Nab3&#x02013;Sen1+pathway&amp;volume=73&amp;publication_year=2019&amp;pages=655-669&amp;pmid=30639244&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B34">34. <span class="mixed-citation">

Jeronimo &#x000a0;C., Robert &#x000a0;F. &#x000a0;<span class="ref-title">Kin28 regulates the transient association of mediator with core promoters</span>. <span class="ref-journal">Nat. Struct. Mol. Biol.</span> &#x000a0;2014; <span class="ref-vol">21</span>:449&#x02013;455.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3997488/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24704787" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Struct.+Mol.+Biol.&amp;title=Kin28+regulates+the+transient+association+of+mediator+with+core+promoters&amp;volume=21&amp;publication_year=2014&amp;pages=449-455&amp;pmid=24704787&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B35">35. <span class="mixed-citation">

Boyer &#x000a0;L.A., Lee &#x000a0;T.I., Cole &#x000a0;M.F., Johnstone &#x000a0;S.E., Levine &#x000a0;S.S., Zucker &#x000a0;J.P., Guenther &#x000a0;M.G., Kumar &#x000a0;R.M., Murray &#x000a0;H.L., Jenner &#x000a0;R.G. &#x000a0;et al.. &#x000a0;<span class="ref-title">Core transcriptional regulatory circuitry in human embryonic stem cells</span>. <span class="ref-journal">Cell</span>. 2005; <span class="ref-vol">122</span>:947&#x02013;956.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3006442/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16153702" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Core+transcriptional+regulatory+circuitry+in+human+embryonic+stem+cells&amp;volume=122&amp;publication_year=2005&amp;pages=947-956&amp;pmid=16153702&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B36">36. <span class="mixed-citation">

Casper &#x000a0;J., Zweig &#x000a0;A.S., Villarreal &#x000a0;C., Tyner &#x000a0;C., Speir &#x000a0;M.L., Rosenbloom &#x000a0;K.R., Raney &#x000a0;B.J., Lee &#x000a0;C.M., Lee &#x000a0;B.T., Karolchik &#x000a0;D. &#x000a0;et al.. &#x000a0;<span class="ref-title">The UCSC Genome Browser database: 2018 update</span>. <span class="ref-journal">Nucleic Acids Res.</span> &#x000a0;2018; <span class="ref-vol">46</span>:D762&#x02013;D769.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5753355/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29106570" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=The+UCSC+Genome+Browser+database:+2018+update&amp;volume=46&amp;publication_year=2018&amp;pages=D762-D769&amp;pmid=29106570&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B37">37. <span class="mixed-citation">

Brunelle &#x000a0;M., Coulombe &#x000a0;C., Poitras &#x000a0;C., Robert &#x000a0;M.-A., Markovits &#x000a0;A.N., Robert &#x000a0;F., Jacques &#x000a0;P.-&#x000c9;. &#x000a0;<span class="ref-title">Aggregate and heatmap representations of genome-wide localization data using VAP, a versatile aggregate profiler</span>. <span class="ref-journal">Methods Mol. Biol.</span> &#x000a0;2015; <span class="ref-vol">1334</span>:273&#x02013;298.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/26404157" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Mol.+Biol.&amp;title=Aggregate+and+heatmap+representations+of+genome-wide+localization+data+using+VAP,+a+versatile+aggregate+profiler&amp;volume=1334&amp;publication_year=2015&amp;pages=273-298&amp;pmid=26404157&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B38">38. <span class="mixed-citation">

Saldanha &#x000a0;A.J. &#x000a0;<span class="ref-title">Java Treeview&#x02014;extensible visualization of microarray data</span>. <span class="ref-journal">Bioinformatics</span>. 2004; <span class="ref-vol">20</span>:3246&#x02013;3248.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/15180930" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Java+Treeview&#x02014;extensible+visualization+of+microarray+data&amp;volume=20&amp;publication_year=2004&amp;pages=3246-3248&amp;pmid=15180930&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B39">39. <span class="mixed-citation">

Larochelle &#x000a0;M., Drouin &#x000a0;S., Robert &#x000a0;F., Turcotte &#x000a0;B. &#x000a0;<span class="ref-title">Oxidative stress-activated zinc cluster protein Stb5 has dual activator/repressor functions required for pentose phosphate pathway regulation and NADPH production</span>. <span class="ref-journal">Mol. Cell. Biol.</span> &#x000a0;2006; <span class="ref-vol">26</span>:6690&#x02013;6701.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1592823/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16914749" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Biol.&amp;title=Oxidative+stress-activated+zinc+cluster+protein+Stb5+has+dual+activator/repressor+functions+required+for+pentose+phosphate+pathway+regulation+and+NADPH+production&amp;volume=26&amp;publication_year=2006&amp;pages=6690-6701&amp;pmid=16914749&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B40">40. <span class="mixed-citation">

Mamnun &#x000a0;Y.M., Pandjaitan &#x000a0;R., Mah&#x000e9; &#x000a0;Y., Delahodde &#x000a0;A., Kuchler &#x000a0;K. &#x000a0;<span class="ref-title">The yeast zinc finger regulators Pdr1p and Pdr3p control pleiotropic drug resistance (PDR) as homo- and heterodimers <em>in vivo</em></span>. <span class="ref-journal">Mol. Microbiol.</span> &#x000a0;2002; <span class="ref-vol">46</span>:1429&#x02013;1440.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/12453227" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Microbiol.&amp;title=The+yeast+zinc+finger+regulators+Pdr1p+and+Pdr3p+control+pleiotropic+drug+resistance+(PDR)+as+homo-+and+heterodimers+in+vivo&amp;volume=46&amp;publication_year=2002&amp;pages=1429-1440&amp;pmid=12453227&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B41">41. <span class="mixed-citation">

Akache &#x000a0;B., MacPherson &#x000a0;S., Sylvain &#x000a0;M.-A., Turcotte &#x000a0;B. &#x000a0;<span class="ref-title">Complex interplay among regulators of drug resistance genes in <em>Saccharomyces</em><em>cerevisiae</em></span>. <span class="ref-journal">J. Biol. Chem.</span> &#x000a0;2004; <span class="ref-vol">279</span>:27855&#x02013;27860.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/15123673" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Complex+interplay+among+regulators+of+drug+resistance+genes+in+Saccharomycescerevisiae&amp;volume=279&amp;publication_year=2004&amp;pages=27855-27860&amp;pmid=15123673&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B42">42. <span class="mixed-citation">

Rottensteiner &#x000a0;H., Kal &#x000a0;A.J., Hamilton &#x000a0;B., Ruis &#x000a0;H., Tabak &#x000a0;H.F. &#x000a0;<span class="ref-title">A heterodimer of the Zn<sub>2</sub>Cys<sub>6</sub> transcription factors Pip2p and Oaf1p controls induction of genes encoding peroxisomal proteins in <em>Saccharomyces cerevisiae</em></span>. <span class="ref-journal">Eur. J. Biochem.</span> &#x000a0;1997; <span class="ref-vol">247</span>:776&#x02013;783.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/9288897" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Biochem.&amp;title=A+heterodimer+of+the+Zn2Cys6+transcription+factors+Pip2p+and+Oaf1p+controls+induction+of+genes+encoding+peroxisomal+proteins+in+Saccharomyces+cerevisiae&amp;volume=247&amp;publication_year=1997&amp;pages=776-783&amp;pmid=9288897&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B43">43. <span class="mixed-citation">

Harbison &#x000a0;C.T., Gordon &#x000a0;D.B., Lee &#x000a0;T.I., Rinaldi &#x000a0;N.J., Macisaac &#x000a0;K.D., Danford &#x000a0;T.W., Hannett &#x000a0;N.M., Tagne &#x000a0;J.-B., Reynolds &#x000a0;D.B., Yoo &#x000a0;J. &#x000a0;et al.. &#x000a0;<span class="ref-title">Transcriptional regulatory code of a eukaryotic genome</span>. <span class="ref-journal">Nature</span>. 2004; <span class="ref-vol">431</span>:99&#x02013;104.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3006441/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15343339" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Transcriptional+regulatory+code+of+a+eukaryotic+genome&amp;volume=431&amp;publication_year=2004&amp;pages=99-104&amp;pmid=15343339&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B44">44. <span class="mixed-citation">

Hope &#x000a0;I.A., Struhl &#x000a0;K. &#x000a0;<span class="ref-title">GCN4, a eukaryotic transcriptional activator protein, binds as a dimer to target DNA</span>. <span class="ref-journal">EMBO J.</span> &#x000a0;1987; <span class="ref-vol">6</span>:2781&#x02013;2784.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC553703/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3678204" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+J.&amp;title=GCN4,+a+eukaryotic+transcriptional+activator+protein,+binds+as+a+dimer+to+target+DNA&amp;volume=6&amp;publication_year=1987&amp;pages=2781-2784&amp;pmid=3678204&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B45">45. <span class="mixed-citation">

Mangelsdorf &#x000a0;D.J., Evans &#x000a0;R.M. &#x000a0;<span class="ref-title">The RXR heterodimers and orphan receptors</span>. <span class="ref-journal">Cell</span>. 1995; <span class="ref-vol">83</span>:841&#x02013;850.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/8521508" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=The+RXR+heterodimers+and+orphan+receptors&amp;volume=83&amp;publication_year=1995&amp;pages=841-850&amp;pmid=8521508&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B46">46. <span class="mixed-citation">

Marmorstein &#x000a0;R., Harrison &#x000a0;S.C. &#x000a0;<span class="ref-title">Crystal structure of a PPR1&#x02013;DNA complex: DNA recognition by proteins containing a Zn<sub>2</sub>Cys<sub>6</sub> binuclear cluster</span>. <span class="ref-journal">Genes Dev.</span> &#x000a0;1994; <span class="ref-vol">8</span>:2504&#x02013;2512.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/7958913" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev.&amp;title=Crystal+structure+of+a+PPR1&#x02013;DNA+complex:+DNA+recognition+by+proteins+containing+a+Zn2Cys6+binuclear+cluster&amp;volume=8&amp;publication_year=1994&amp;pages=2504-2512&amp;pmid=7958913&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B47">47. <span class="mixed-citation">

Swaminathan &#x000a0;K., Flynn &#x000a0;P., Reece &#x000a0;R.J., Marmorstein &#x000a0;R. &#x000a0;<span class="ref-title">Crystal structure of a PUT3&#x02013;DNA complex reveals a novel mechanism for DNA recognition by a protein containing a Zn<sub>2</sub>Cys<sub>6</sub> binuclear cluster</span>. <span class="ref-journal">Nat. Struct. Biol.</span> &#x000a0;1997; <span class="ref-vol">4</span>:751&#x02013;759.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/9303004" ref="reftype=pubmed&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Struct.+Biol.&amp;title=Crystal+structure+of+a+PUT3&#x02013;DNA+complex+reveals+a+novel+mechanism+for+DNA+recognition+by+a+protein+containing+a+Zn2Cys6+binuclear+cluster&amp;volume=4&amp;publication_year=1997&amp;pages=751-759&amp;pmid=9303004&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10954478&amp;issue-id=458308&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Nucleic Acids Research</span> are provided here courtesy of <strong>Oxford University Press</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC10954478/pdf/gkad1185.pdf" class="int-view">PDF (3.2M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/10954478/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/10954478/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="10954478" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10954478%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="F2wJnaTm2OlpAqie8vK3spdMzrTmYZyrOOBZKu5SlDLDY4mcyzGcv8AB9NKjezrp">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10954478%2F&amp;text=Yeast%20zinc%20cluster%20transcription%20factors%20involved%20in%20the%20switch%20from%20fermentation%20to%20respiration%20show%20interdependency%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10954478%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10954478/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC10954478/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38109318/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC10954478/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38109318/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC10954478/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/10954478/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC10954478/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
