
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Mutational and structural studies of (&#946;&#945;)8&#8208;barrel fold methylene&#8208;tetrahydropterin reductases utilizing a common catalytic mechanism - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="322C300B24F39205000032A5034705A4.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="prosci" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC11094777/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11094777/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Protein Science : A Publication of the Protein Society" /><meta name="citation_title" content="Mutational and structural studies of (&#x003b2;&#x003b1;)8&#x02010;barrel fold methylene&#x02010;tetrahydropterin reductases utilizing a common catalytic mechanism" /><meta name="citation_author" content="Manuel Gehl" /><meta name="citation_author_institution" content="Max Planck Institute for Terrestrial Microbiology, Marburg Germany" /><meta name="citation_author" content="Ulrike Demmer" /><meta name="citation_author_institution" content="Max Planck Institute of Biophysics, Frankfurt am Main Germany" /><meta name="citation_author" content="Ulrich Ermler" /><meta name="citation_author_institution" content="Max Planck Institute of Biophysics, Frankfurt am Main Germany" /><meta name="citation_author" content="Seigo Shima" /><meta name="citation_author_institution" content="Max Planck Institute for Terrestrial Microbiology, Marburg Germany" /><meta name="citation_publication_date" content="2024/06" /><meta name="citation_issue" content="6" /><meta name="citation_volume" content="33" /><meta name="citation_doi" content="10.1002/pro.5018" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC11094777/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC11094777/" /><meta name="citation_pmid" content="38747406" /><meta name="DC.Title" content="Mutational and structural studies of (&#x003b2;&#x003b1;)8&#x02010;barrel fold methylene&#x02010;tetrahydropterin reductases utilizing a common catalytic mechanism" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Wiley" /><meta name="DC.Contributor" content="Manuel Gehl" /><meta name="DC.Contributor" content="Ulrike Demmer" /><meta name="DC.Contributor" content="Ulrich Ermler" /><meta name="DC.Contributor" content="Seigo Shima" /><meta name="DC.Date" content="2024 Jun" /><meta name="DC.Identifier" content="10.1002/pro.5018" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Mutational and structural studies of (&#x003b2;&#x003b1;)8&#x02010;barrel fold methylene&#x02010;tetrahydropterin reductases utilizing a common catalytic mechanism" /><meta property="og:type" content="article" /><meta property="og:description" content="Methylene&#x02010;tetrahydropterin reductases catalyze the reduction of a methylene to a methyl group bound to a reduced pterin as C[1]  carrier in various one&#x02010;carbon (C[1] ) metabolisms. F[420] &#x02010;dependent methylene&#x02010;tetrahydromethanopterin ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11094777/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11094777/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/507/">Protein Sci</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/461969/">v.33(6); 2024 Jun</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11094777
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11094777/pdf/PRO-33-e5018.pdf" class="int-view">PDF (11M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11094777/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11094777/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11094777" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11094777%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="Iph0uiZ9zQkt1ptoOjzDMj3cMP0VbeqP3wCKX8JeTQiVAiV3AklocIMfcUe9x27T">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11094777%2F&amp;text=Mutational%20and%20structural%20studies%20of%20(%003B2%003B1)8%02010barrel%20fold%20methylene%02010tetrahydropterin%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11094777%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11094777/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11094777/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38747406/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11094777/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38747406/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11094777/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11094777/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/507/">Protein Sci</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/461969/">v.33(6); 2024 Jun</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11094777
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-prosci.gif" alt="Logo of prosci" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="0,0,500,55" alt="protein science" title="protein science" href="http://www.proteinscience.org" ref="reftype=publisher&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="0,55,82,75" alt="cshl press" title="cshl press" href="http://www.cshlpress.com" ref="reftype=publisher&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="180,55,280,75" alt="subscriptions" title="subscriptions" href="http://www.proteinscience.org" ref="reftype=publisher&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="280,55,369,75" alt="etoc alerts" title="etoc alerts" href="http://www.proteinscience.org/subscriptions/etoc.shtml" ref="reftype=publisher&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="369,55,500,75" alt="the protein society" title="the protein society" href="http://www.proteinsociety.org" ref="reftype=publisher&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="85,55,180,75" alt="journal home" title="journal home" href="http://www.proteinscience.org" ref="reftype=publisher&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Protein Sci.</span> 2024 Jun; 33(6): e5018. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2024 May 15. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1002%2Fpro.5018" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1002/pro.5018</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC11094777</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38747406">38747406</a></div></div></div><h1 class="content-title">Mutational and structural studies of (&#x003b2;&#x003b1;)<sub>8</sub>&#x02010;barrel fold methylene&#x02010;tetrahydropterin reductases utilizing a common catalytic mechanism</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gehl%20M%5BAuthor%5D" id="pro5018-cr-0001" class="affpopup" co-rid="_co_idm140710429122528" co-class="co-affbox">Manuel Gehl</a>,<sup>
1
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Demmer%20U%5BAuthor%5D" id="pro5018-cr-0002" class="affpopup" co-rid="_co_idm140710429106880" co-class="co-affbox">Ulrike Demmer</a>,<sup>
2
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ermler%20U%5BAuthor%5D" id="pro5018-cr-0003" class="affpopup" co-rid="_co_idm140710429103536" co-class="co-affbox">Ulrich Ermler</a>,<sup>
2
</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shima%20S%5BAuthor%5D" id="pro5018-cr-0004" class="affpopup" co-rid="_co_idm140710429100192" co-class="co-affbox">Seigo Shima</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>
1
</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140710429122528"><h3 class="no_margin">Manuel Gehl</h3><p>
<sup>
1
</sup>
Max Planck Institute for Terrestrial Microbiology, 
Marburg
Germany
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gehl%20M%5BAuthor%5D">Manuel Gehl</a></div></div><div id="_co_idm140710429106880"><h3 class="no_margin">Ulrike Demmer</h3><p>
<sup>
2
</sup>
Max Planck Institute of Biophysics, 
Frankfurt am Main
Germany
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Demmer%20U%5BAuthor%5D">Ulrike Demmer</a></div></div><div id="_co_idm140710429103536"><h3 class="no_margin">Ulrich Ermler</h3><p>
<sup>
2
</sup>
Max Planck Institute of Biophysics, 
Frankfurt am Main
Germany
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ermler%20U%5BAuthor%5D">Ulrich Ermler</a></div></div><div id="_co_idm140710429100192"><h3 class="no_margin">Seigo Shima</h3><p>
<sup>
1
</sup>
Max Planck Institute for Terrestrial Microbiology, 
Marburg
Germany
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shima%20S%5BAuthor%5D">Seigo Shima</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140710433438064_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140710433438064_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140710433438064_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140710433438064_ai" style="display:none"><div class="fm-affl" id="pro5018-aff-0001">
<sup>
1
</sup>
Max Planck Institute for Terrestrial Microbiology, 
Marburg
Germany
</div><div class="fm-affl" id="pro5018-aff-0002">
<sup>
2
</sup>
Max Planck Institute of Biophysics, 
Frankfurt am Main
Germany
</div><div><span class="fm-affl">Seigo Shima, </span><span class="fm-affl"><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ed.gpm.grubram-ipm@amihs" class="oemail">ed.gpm.grubram-ipm@amihs</a></span>.</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup>Corresponding author.</div><div id="correspondenceTo"><sup>*</sup>

<strong>Correspondence</strong>
<br />
Seigo Shima, Max Planck Institute for Terrestrial Microbiology, Marburg, Germany.<br />
Email: <a href="mailto:dev@null" data-email="ed.gpm.grubram-ipm@amihs" class="oemail">ed.gpm.grubram-ipm@amihs</a>
<br />
</div></div><div class="fm-article-notes hide half_rhythm" id="idm140710433438064_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2023 Oct 20; Revised 2024 Apr 19; Accepted 2024 Apr 26.</div></div><div class="permissions half_rhythm hide" id="idm140710433438064_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; 2024 The Authors. <em>Protein Science</em> published by Wiley Periodicals LLC on behalf of The Protein Society.</div><div class="license half_rhythm">This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="8" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
<strong>TABLE S1.</strong> Structure determination statistics for jMer_E6Q and Mfr_E9Q.<p>
<strong>TABLE S2.</strong> Seed sequences for the superfamily of bacterial luciferases.</p><p>
<strong>TABLE S3.</strong> Seed sequences for the superfamily of FAD&#x02010;linked reductases.</p><p>
<strong>TABLE S4.</strong> List of primers used for mutagenesis of jMer and sequencing of the ORF of jMer in pT7&#x02010;7_jMer. The degenerated nucleotides are marked in red.</p><p>
<strong>FIGURE S2.</strong> Size&#x02010;exclusion chromatography of jMer using a HiPrep Sephacryl S&#x02010;200 HR column. The peak of jMer is centered at around 50&#x02009;mL elution volume and corresponds to a molecular mass of approximately 80&#x02009;kDa.</p></div><div><a href="/pmc/articles/PMC11094777/bin/PRO-33-e5018-s001.docx" data-ga-action="click_feat_suppl">PRO-33-e5018-s001.docx</a><span style="color:gray"> (9.6M)</span></div><div class="small guid">GUID: FD7E3A50-8373-44A0-8F24-1765C9C0170D</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
<strong>FIGURE S1:</strong> Structures of H<sub>4</sub>MPT (top) and H<sub>4</sub>F (bottom). The pterin part is colored blue. The p&#x02010;aminobenzoate (PABA) ring of H<sub>4</sub>F and the aniline ring of H<sub>4</sub>MPT are colored red. The tail regions are colored black.</div><div><a href="/pmc/articles/PMC11094777/bin/PRO-33-e5018-s005.jpg" data-ga-action="click_feat_suppl">PRO-33-e5018-s005.jpg</a><span style="color:gray"> (806K)</span></div><div class="small guid">GUID: 30B43329-5EEE-4A8F-AEF7-607277E93061</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
<strong>FIGURE S3:</strong> Dimeric structure of jMer. The homodimer is the physiological form of jMer and is formed by a two&#x02010;fold rotational axis at the center of the protein&#x02010;protein interface. The active site is located at the C&#x02010;terminal end of the parallel &#x003b2;&#x02010;strands, which are positioned on the opposite site of the two monomers.</div><div><a href="/pmc/articles/PMC11094777/bin/PRO-33-e5018-s003.jpg" data-ga-action="click_feat_suppl">PRO-33-e5018-s003.jpg</a><span style="color:gray"> (1.4M)</span></div><div class="small guid">GUID: 7F3E46D7-D648-4706-B3B9-B2E81DB1754A</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
<strong>FIGURE S4:</strong> Comparison of the tertiary structures of eMTHFR, jMer and hMfr. The &#x003b2;&#x02010;strands of the core unit are labeled and colored purple, while the &#x003b1;&#x02010;helices of the core unit and the loops are colored in light blue and salmon. The inserted helical segment is painted in orange. Methyl&#x02010;H<sub>4</sub>F and FAD are shown in orange and F<sub>420</sub> is shown in green.</div><div><a href="/pmc/articles/PMC11094777/bin/PRO-33-e5018-s002.jpg" data-ga-action="click_feat_suppl">PRO-33-e5018-s002.jpg</a><span style="color:gray"> (2.0M)</span></div><div class="small guid">GUID: 2A943C9D-E17D-46F2-80B2-F623A508E42F</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
<strong>FIGURE S5:</strong> Hydrophobic pocked in jMer. The modeled H<sub>4</sub>F is colored yellow while the native F<sub>420</sub> and the amino acids are shown in light blue.</div><div><a href="/pmc/articles/PMC11094777/bin/PRO-33-e5018-s007.jpg" data-ga-action="click_feat_suppl">PRO-33-e5018-s007.jpg</a><span style="color:gray"> (788K)</span></div><div class="small guid">GUID: EB804AAE-C149-4A26-8641-45D2DF613DB5</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
<strong>FIGURE S6:</strong> Structure&#x02010;based alignment of hMfr, eMTHFR and jMer.</div><div><a href="/pmc/articles/PMC11094777/bin/PRO-33-e5018-s008.jpg" data-ga-action="click_feat_suppl">PRO-33-e5018-s008.jpg</a><span style="color:gray"> (172K)</span></div><div class="small guid">GUID: EC3461E8-B4CA-4C55-8A9A-767EE18CAAFB</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
<strong>FIGURE S7:</strong> NADH binding site of eMTHFR.</div><div><a href="/pmc/articles/PMC11094777/bin/PRO-33-e5018-s006.jpg" data-ga-action="click_feat_suppl">PRO-33-e5018-s006.jpg</a><span style="color:gray"> (648K)</span></div><div class="small guid">GUID: 1B88E4BD-E705-4E3A-A3A9-BCFDB6914025</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
<strong>FIGURE S8:</strong> (a) Comparison of the structures of jMer wild type (green) and jMer_E6Q (light blue). (b) Comparison of the structures of hMfr wild type (dark blue) and hMfr_E9Q (yellow). The glutamate residues are depicted as ball&#x02010;and&#x02010;stick model.</div><div><a href="/pmc/articles/PMC11094777/bin/PRO-33-e5018-s004.jpg" data-ga-action="click_feat_suppl">PRO-33-e5018-s004.jpg</a><span style="color:gray"> (1.2M)</span></div><div class="small guid">GUID: 0639706B-840D-472C-A2AA-3DA17FE32C6B</div></div></dd></dl><dl data-length="49" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first-last">Data available in article supplementary material.</p></dd></dl></div><div id="abstract-a.w.b.s" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.w.b.stitle">Abstract</h2><!--article-meta--><div><p class="p p-first-last">Methylene&#x02010;tetrahydropterin reductases catalyze the reduction of a methylene to a methyl group bound to a reduced pterin as C<sub>1</sub> carrier in various one&#x02010;carbon (C<sub>1</sub>) metabolisms. F<sub>420</sub>&#x02010;dependent methylene&#x02010;tetrahydromethanopterin (methylene&#x02010;H<sub>4</sub>MPT) reductase (Mer) and the flavin&#x02010;independent methylene&#x02010;tetrahydrofolate (methylene&#x02010;H<sub>4</sub>F) reductase (Mfr) use a ternary complex mechanism for the direct transfer of a hydride from F<sub>420</sub>H<sub>2</sub> and NAD(P)H to the respective methylene group, whereas FAD&#x02010;dependent methylene&#x02010;H<sub>4</sub>F reductase (MTHFR) uses FAD as prosthetic group and a ping&#x02013;pong mechanism to catalyze the reduction of methylene&#x02010;H<sub>4</sub>F. A ternary complex structure and a thereof derived catalytic mechanism of MTHFR is available, while no ternary complex structures of Mfr or Mer are reported. Here, Mer from <em>Methanocaldococcus jannaschii</em> (jMer) was heterologously produced and the crystal structures of the enzyme with and without F<sub>420</sub> were determined. A ternary complex of jMer was modeled on the basis of the jMer&#x02010;F<sub>420</sub> structure and the ternary complex structure of MTHFR by superimposing the polypeptide after fixing hydride&#x02010;transferring atoms of the flavins on each other, and by the subsequent transfer of the methyl&#x02010;tetrahydropterin from MTHFR to jMer. Mutational analysis of four functional amino acids, which are similarly positioned in the three reductase structures, indicated despite the insignificant sequence identity, a common catalytic mechanism with a 5&#x02010;iminium cation of methylene&#x02010;tetrahydropterin as intermediate protonated by a shared glutamate. According to structural, mutational and phylogenetic analysis, the evolution of the three reductases most likely proceeds via a convergent development although a divergent scenario requiring drastic structural changes of the common ancestor cannot be completely ruled out.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">catalytic mechanism, crystal structure, evolution, methylene&#x02010;tetrahydrofolate reductase, methylene&#x02010;tetrahydromethanopterin reductase</span></div></div><div id="pro5018-sec-0001" class="tsec sec"><h2 class="head no_bottom_margin" id="pro5018-sec-0001title">1. INTRODUCTION</h2><p class="p p-first">Redox reactions of C<sub>1</sub> units bound to C<sub>1</sub> carriers are widespread in the three domains of life. The most common C<sub>1</sub> carriers are tetrahydrofolate (H<sub>4</sub>F) and tetrahydromethanopterin (H<sub>4</sub>MPT), which consist of a reduced pterin (tetrahydropterin) bound to a <em>para</em>&#x02010;aminobenzoate (PABA) group or its para&#x02010;aniline derivative, respectively, and a variable tail region (Maden,&#x000a0;<sup><a href="#pro5018-bib-0029" rid="pro5018-bib-0029" class=" bibr popnode">2000</a></sup>). H<sub>4</sub>F and H<sub>4</sub>MPT significantly differ (Figure&#x000a0;<a href="#pro5018-supitem-0002" rid="pro5018-supitem-0002" class=" supplementary-material">S1</a>), as H<sub>4</sub>MPT has a methylene group and H<sub>4</sub>F has a carbonyl group both conjugated to N10 via an aromatic ring. The structural differences are reflected in different p<em>K</em>
<sub>a</sub> values for N10 and in substantially different redox properties of the C<sub>1</sub> unit attached (Thauer et al.,&#x000a0;<sup><a href="#pro5018-bib-0054" rid="pro5018-bib-0054" class=" bibr popnode">1996</a></sup>). The bound C<sub>1</sub> unit is transformed between oxidation states of formic acid (+II, formyl and methenyl group), formaldehyde (0, methylene group) and methanol (&#x02212;II, methyl group) in the C<sub>1</sub> metabolisms. H<sub>4</sub>F is employed in the methyl&#x02010;branch of the Wood&#x02013;Ljungdahl pathway, and the folate cycle of most organisms, while H<sub>4</sub>MPT is utilized in the catabolic pathways of methanogenic and sulfate&#x02010;reducing archaea, as well as methylotrophic and methanotrophic bacteria (Borrel et al.,&#x000a0;<sup><a href="#pro5018-bib-0005" rid="pro5018-bib-0005" class=" bibr popnode">2016</a></sup>; Zheng &#x00026; Cantley,&#x000a0;<sup><a href="#pro5018-bib-0063" rid="pro5018-bib-0063" class=" bibr popnode">2019</a></sup>).</p><p>The reduction of methylene&#x02010;H<sub>4</sub>F to methyl&#x02010;H<sub>4</sub>F is catalyzed by methylene&#x02010;H<sub>4</sub>F reductases (Sah et al.,&#x000a0;<sup><a href="#pro5018-bib-0045" rid="pro5018-bib-0045" class=" bibr popnode">2020</a></sup>; Sheppard et al.,&#x000a0;<sup><a href="#pro5018-bib-0047" rid="pro5018-bib-0047" class=" bibr popnode">1999</a></sup>; Zheng &#x00026; Cantley,&#x000a0;<sup><a href="#pro5018-bib-0063" rid="pro5018-bib-0063" class=" bibr popnode">2019</a></sup>), which are classified into a flavin&#x02010;dependent enzyme (MTHFR) and a flavin&#x02010;independent one (Mfr) (Sah et al.,&#x000a0;<sup><a href="#pro5018-bib-0045" rid="pro5018-bib-0045" class=" bibr popnode">2020</a></sup>; Yu et al.,&#x000a0;<sup><a href="#pro5018-bib-0062" rid="pro5018-bib-0062" class=" bibr popnode">2022</a></sup>). MTHFR is further classified into subclasses depending on either FAD or FMN as a prosthetic group (Bertsch et al.,&#x000a0;<sup><a href="#pro5018-bib-0004" rid="pro5018-bib-0004" class=" bibr popnode">2015</a></sup>; Clark &#x00026; Ljungdahl,&#x000a0;<sup><a href="#pro5018-bib-0006" rid="pro5018-bib-0006" class=" bibr popnode">1984</a></sup>; Mock et al.,&#x000a0;<sup><a href="#pro5018-bib-0036" rid="pro5018-bib-0036" class=" bibr popnode">2014</a></sup>; &#x000d6;ppinger et al.,&#x000a0;<sup><a href="#pro5018-bib-0039" rid="pro5018-bib-0039" class=" bibr popnode">2022</a></sup>; Sheppard et al.,&#x000a0;<sup><a href="#pro5018-bib-0047" rid="pro5018-bib-0047" class=" bibr popnode">1999</a></sup>). The FAD&#x02010;dependent MTHFR uses a ping&#x02013;pong reaction mechanism (Figure&#x000a0;<a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="pro5018-fig-0001" rid-ob="ob-pro5018-fig-0001" co-legend-rid="lgnd_pro5018-fig-0001"><span>1a</span></a>), in which the tightly bound FAD is first reduced by NAD(P)H and then FADH<sub>2</sub> reduces methylene&#x02010;H<sub>4</sub>F (Trimmer et al.,&#x000a0;<sup><a href="#pro5018-bib-0056" rid="pro5018-bib-0056" class=" bibr popnode">2001</a></sup>). The structural characterization of the inactive MTHFR_Glu28Gln variant from <em>E. coli</em> (eMTHFR) with FAD and with either NADH or methyl&#x02010;H<sub>4</sub>F (Pejchal et al.,&#x000a0;<sup><a href="#pro5018-bib-0042" rid="pro5018-bib-0042" class=" bibr popnode">2005</a></sup>) revealed both substrates in the same position, explaining the structural basis for the ping&#x02013;pong mechanism. Since the imidazolidine ring in methylene&#x02010;H<sub>4</sub>F is a poor acceptor for the negatively charged hydride, the substrate is activated to a 5&#x02010;iminium cation (Sumner &#x00026; Matthews,&#x000a0;<sup><a href="#pro5018-bib-0052" rid="pro5018-bib-0052" class=" bibr popnode">1992</a></sup>). A 5&#x02010;iminium cation intermediate was also proposed in the non&#x02010;enzymatic condensation of formaldehyde and H<sub>4</sub>F to form methylene&#x02010;H<sub>4</sub>F (Kallen &#x00026; Jencks,&#x000a0;<sup><a href="#pro5018-bib-0019" rid="pro5018-bib-0019" class=" bibr popnode">1966</a></sup>). Additionally, in the crystal structure of thymidylate synthase, a 5&#x02010;hydroxymethylene&#x02010;H<sub>4</sub>F was found, which supports the presence of the 5&#x02010;iminium cation as an intermediate in this enzyme reaction (Perry et al.,&#x000a0;<sup><a href="#pro5018-bib-0043" rid="pro5018-bib-0043" class=" bibr popnode">1993</a></sup>). Based on the drastic effect of the Glu28Gln mutation of eMTHFR on the activity, Glu28 was proposed to be the catalytic acid for protonating the substrate for the formation of the 5&#x02010;iminium cation.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="pro5018-fig-0001" co-legend-rid="lgnd_pro5018-fig-0001"><a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0001/" target="figure" rid-figpopup="pro5018-fig-0001" rid-ob="ob-pro5018-fig-0001"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140710429005536"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is PRO-33-e5018-g004.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is PRO-33-e5018-g004.jpg" src="/pmc/articles/PMC11094777/bin/PRO-33-e5018-g004.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140710429005536"><a target="object" rel="noopener" href="/pmc/articles/PMC11094777/figure/pro5018-fig-0001/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_pro5018-fig-0001"><div><a class="figpopup" href="/pmc/articles/PMC11094777/figure/pro5018-fig-0001/" target="figure" rid-figpopup="pro5018-fig-0001" rid-ob="ob-pro5018-fig-0001">FIGURE 1</a></div><!--caption a7--><div class="caption"><p>Reactions catalyzed by methylene&#x02010;tetrahydropterin reductases. (a) FAD&#x02010;dependent methylene&#x02010;H<sub>4</sub>F reductases (MTHFR) catalyzes the reduction of methylene&#x02010;H<sub>4</sub>F by NAD(P)H using a ping&#x02013;pong mechanism with FAD, bound to the enzyme as a prosthetic group, as intermediate redox center. (b) Flavin&#x02010;independent methylene&#x02010;H<sub>4</sub>F reductase (Mfr) catalyzes the reduction of methylene&#x02010;H<sub>4</sub>F by using NADH as reducing agent. In contrast to MTHFR, Mfr has no prosthetic group and uses a ternary complex reaction mechanism. (c) F<sub>420</sub>&#x02010;dependent methylene&#x02010;H<sub>4</sub>MPT reductase (Mer) catalyzes the reduction of methylene&#x02010;H<sub>4</sub>MPT with reduced F<sub>420</sub> (F<sub>420</sub>H<sub>2</sub>) as reducing agent via a ternary complex mechanism.</p></div></div></div><p>Recently, the enzymes encoded by <em>MSMEG_6596</em> and <em>MSMEG_6649</em> in <em>Mycobacterium smegmatis</em> were biochemically characterized and MTHFR activity demonstrated (Sah et al.,&#x000a0;<sup><a href="#pro5018-bib-0045" rid="pro5018-bib-0045" class=" bibr popnode">2020</a></sup>). Both enzymes do not contain flavin and catalyze a direct hydride transfer from NADH to methylene&#x02010;H<sub>4</sub>F by a ternary complex mechanism (Figure&#x000a0;<a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="pro5018-fig-0001" rid-ob="ob-pro5018-fig-0001" co-legend-rid="lgnd_pro5018-fig-0001"><span>1b</span></a>). The flavin&#x02010;independent enzymes, named Mfr (Gehl et al.,&#x000a0;<sup><a href="#pro5018-bib-0011" rid="pro5018-bib-0011" class=" bibr popnode">2023</a></sup>), was currently identified only in mycobacteria as a monomeric enzyme. A knockout strain of <em>M. smegmatis</em> showed impaired growth in the absence of methionine, suggesting that Mfr is involved in the methionine cycle in mycobacteria (Sah et al.,&#x000a0;<sup><a href="#pro5018-bib-0045" rid="pro5018-bib-0045" class=" bibr popnode">2020</a></sup>). Likewise, an Mfr homolog encoded by <em>Rv2172c</em> in <em>M. tuberculosis</em> is essential for the growth of this organism (Yu et al.,&#x000a0;<sup><a href="#pro5018-bib-0062" rid="pro5018-bib-0062" class=" bibr popnode">2022</a></sup>). Recently, we reported the crystal structure of Mfr from <em>Mycolicibacterium hassiacum</em> (hMfr) and provided evidence that Glu9 in the active site is the key catalytic residue for 5&#x02010;iminium cation formation. Thus, the mechanism of eMTHFR and hMfr is basically identical.</p><p>Most methanogenesis pathways involve the methylene&#x02010;H<sub>4</sub>MPT reductase (Mer), which catalyzes the reversible reduction of methylene&#x02010;H<sub>4</sub>MPT with F<sub>420</sub>H<sub>2</sub> as reductant using a ternary complex mechanism (Kurth et al.,&#x000a0;<sup><a href="#pro5018-bib-0021" rid="pro5018-bib-0021" class=" bibr popnode">2020</a></sup>; Ma &#x00026; Thauer,&#x000a0;<sup><a href="#pro5018-bib-0027" rid="pro5018-bib-0027" class=" bibr popnode">1990a</a></sup>) (Figure&#x000a0;<a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="pro5018-fig-0001" rid-ob="ob-pro5018-fig-0001" co-legend-rid="lgnd_pro5018-fig-0001"><span>1c</span></a>). The product methyl&#x02010;H<sub>4</sub>MPT is an intermediate of the energy metabolism and the methyl donor of acetyl&#x02010;coenzyme A biosynthesis. Crystal structures are available for substrate&#x02010;free Mer from <em>Methanopyrus kandleri</em> and <em>Methanothermobacter marburgensis</em> (Shima et al.,&#x000a0;<sup><a href="#pro5018-bib-0049" rid="pro5018-bib-0049" class=" bibr popnode">2000</a></sup>) and the <em>Methanosarcina barkeri</em> Mer&#x02010;F<sub>420</sub> binary complex (Aufhammer et al.,&#x000a0;<sup><a href="#pro5018-bib-0002" rid="pro5018-bib-0002" class=" bibr popnode">2005</a></sup>). Mer belongs to the bacterial luciferase family, which consists of FMN&#x02010; and F<sub>420</sub>&#x02010;dependent oxidoreductases (Aufhammer et al.,&#x000a0;<sup><a href="#pro5018-bib-0002" rid="pro5018-bib-0002" class=" bibr popnode">2005</a></sup>).</p><p class="p p-last">In this report, the hydride&#x02010;transfer mechanism of three functionally related methylene&#x02010;tetrahydropterin reductases was comparatively scrutinized with the aim to better understand their phylogenetic relationship in terms of divergent and convergent evolution. In the divergent scenario, the three hydride&#x02010;transferring enzymes would originate from a common ancestor, while in a convergent scenario, they start from a different origin and develop under evolutionary pressure to shared active site features. For the analysis, Mer from <em>Methanocaldococcus jannaschii</em> was chosen as only the enzyme from this organism can be heterologously and functionally produced thus allowing mutational analyses. For structural comparison with eMTHFR and hMfr, the crystal structures of Mer from <em>M. jannaschii</em> (jMer) with and without F<sub>420</sub> were solved. We identified similar amino acid residues at equivalent positions in the active sites of eMTHFR, hMfr, and jMer, which are potentially involved in the binding and activation of the C<sub>1</sub> carriers. Based on mutational and kinetic analysis, evidence has been provided that all three methylene&#x02010;tetrahydropterin reductases have the same fold, similar methylene&#x02010;tetrahydropterin binding positions, and the same basic catalytic mechanism; however, the binding modes of the C<sub>1</sub> carriers and reductants significantly deviate. In addition, we obtained the crystal structures of the inactive variants jMer_E6Q, and hMfr_E9Q to exclude perturbations of the active sites.</p></div><div id="pro5018-sec-0002" class="tsec sec"><h2 class="head no_bottom_margin" id="pro5018-sec-0002title">2. RESULTS AND DISCUSSION</h2><div id="pro5018-sec-0003" class="sec sec-first"><h3 id="pro5018-sec-0003title">2.1. Characterization of the heterologously produced <span style="fixed-case">jMer</span>
</h3><p class="p p-first">Attempts to produce Mer from different organisms in <em>E. coli</em> have failed in the past because Mer formed inclusion bodies (Vaupel &#x00026; Thauer,&#x000a0;<sup><a href="#pro5018-bib-0057" rid="pro5018-bib-0057" class=" bibr popnode">1995</a></sup>). However, recently, the first successful heterologous expression of Mer from <em>Methanocaldococcus jannaschii</em> (jMer) was reported, in which the heterologously produced jMer catalyzed the formation of lactaldehyde by reduction of methylglyoxal with NADPH (Miller et al.,&#x000a0;<sup><a href="#pro5018-bib-0034" rid="pro5018-bib-0034" class=" bibr popnode">2017</a></sup>). We obtained the published expression system from the authors and characterized the purified enzyme. The apparent <em>K</em>
<sub>m</sub> and <em>k</em>
<sub>cat</sub> values of jMer are in the range of those of the respective enzymes from methanogenic archaea (<em>M. barkeri</em>, <em>M. marburgensis</em>, and <em>M. kandleri</em>) and sulfate&#x02010;reducing archaea (<em>Archaeoglobus fulgidus</em>) (Table&#x000a0;<a href="/pmc/articles/PMC11094777/table/pro5018-tbl-0001/" target="table" class="fig-table-link figpopup" rid-figpopup="pro5018-tbl-0001" rid-ob="ob-pro5018-tbl-0001" co-legend-rid=""><span>1</span></a>) (Ma et al.,&#x000a0;<sup><a href="#pro5018-bib-0026" rid="pro5018-bib-0026" class=" bibr popnode">1991</a></sup>; Ma &#x00026; Thauer,&#x000a0;<sup><a href="#pro5018-bib-0027" rid="pro5018-bib-0027" class=" bibr popnode">1990a</a></sup>; Ma &#x00026; Thauer,&#x000a0;<sup><a href="#pro5018-bib-0028" rid="pro5018-bib-0028" class=" bibr popnode">1990b</a></sup>; Schmitz et al.,&#x000a0;<sup><a href="#pro5018-bib-0046" rid="pro5018-bib-0046" class=" bibr popnode">1991</a></sup>; te Brommelstroet et al.,&#x000a0;<sup><a href="#pro5018-bib-0053" rid="pro5018-bib-0053" class=" bibr popnode">1990</a></sup>). Size&#x02010;exclusion chromatography showed that jMer in solution is a homodimer (~80&#x02009;kDa) (Figures&#x000a0;<a href="#pro5018-supitem-0001" rid="pro5018-supitem-0001" class=" supplementary-material">S2</a> and <a href="#pro5018-supitem-0003" rid="pro5018-supitem-0003" class=" supplementary-material">S3</a>) as reported in the literature (Miller et al.,&#x000a0;<sup><a href="#pro5018-bib-0034" rid="pro5018-bib-0034" class=" bibr popnode">2017</a></sup>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="pro5018-tbl-0001"><h3>TABLE 1</h3><!--caption a7--><div class="caption"><p>Comparison of the kinetic constants of Mer from <em>Methanosarcina barkeri</em>, <em>Methanothermobacter marburgensis</em>, <em>Methanopyrus kandleri</em>, <em>Archaeoglobus fulgidus</em>, <em>Methanocaldococcus jannaschii</em>.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr style="border-bottom:solid 1px #000000"><th align="left" valign="bottom" rowspan="1" colspan="1">Mer from</th><th align="left" valign="bottom" rowspan="1" colspan="1">Temperature<sup>
a
</sup>
</th><th align="left" valign="bottom" rowspan="1" colspan="1"></th><th align="left" valign="bottom" rowspan="1" colspan="1">Apparent <em>K</em>
<sub>m</sub>
</th><th align="left" valign="bottom" rowspan="1" colspan="1">Apparent <em>k</em>
<sub>cat</sub>
</th><th align="left" valign="bottom" rowspan="1" colspan="1">Source</th></tr></thead><tbody valign="top"><tr><td rowspan="2" align="left" valign="top" colspan="1">
<em>M. barkeri</em>
</td><td rowspan="2" align="left" valign="top" colspan="1">37&#x000b0;C (55&#x000b0;C)</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>MPT</td><td align="left" valign="top" rowspan="1" colspan="1">15&#x02009;&#x003bc;M</td><td rowspan="2" align="left" valign="top" colspan="1">76,000&#x02009;min<sup>&#x02212;1</sup> (1300&#x02009;s<sup>&#x02212;1</sup>)</td><td rowspan="2" align="left" valign="top" colspan="1">(Ma &#x00026; Thauer,&#x000a0;<sup><a href="#pro5018-bib-0028" rid="pro5018-bib-0028" class=" bibr popnode">1990b</a></sup>)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">F<sub>420</sub>H<sub>2</sub>
</td><td align="left" valign="top" rowspan="1" colspan="1">12&#x02009;&#x003bc;M</td></tr><tr><td rowspan="2" align="left" valign="top" colspan="1">
<em>M. marburgensis</em>
</td><td rowspan="2" align="left" valign="top" colspan="1">65&#x000b0;C (55&#x000b0;C)</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>MPT</td><td align="left" valign="top" rowspan="1" colspan="1">300&#x02009;&#x003bc;M</td><td rowspan="2" align="left" valign="top" colspan="1">220,000&#x02009;min<sup>&#x02212;1</sup> (3600&#x02009;s<sup>&#x02212;1</sup>)</td><td rowspan="2" align="left" valign="top" colspan="1">(Ma &#x00026; Thauer,&#x000a0;<sup><a href="#pro5018-bib-0027" rid="pro5018-bib-0027" class=" bibr popnode">1990a</a></sup>)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">F<sub>420</sub>H<sub>2</sub>
</td><td align="left" valign="top" rowspan="1" colspan="1">3&#x02009;&#x003bc;M</td></tr><tr><td rowspan="2" align="left" valign="top" colspan="1">
<em>M. kandleri</em>
</td><td rowspan="2" align="left" valign="top" colspan="1">98&#x000b0;C (65&#x000b0;C)</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>MPT</td><td align="left" valign="top" rowspan="1" colspan="1">7&#x02009;&#x003bc;M</td><td rowspan="2" align="left" valign="top" colspan="1">16,000&#x02009;min<sup>&#x02212;1</sup> (270&#x02009;s<sup>&#x02212;1</sup>)</td><td rowspan="2" align="left" valign="top" colspan="1">(Ma et al.,&#x000a0;<sup><a href="#pro5018-bib-0026" rid="pro5018-bib-0026" class=" bibr popnode">1991</a></sup>)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">F<sub>420</sub>H<sub>2</sub>
</td><td align="left" valign="top" rowspan="1" colspan="1">4&#x02009;&#x003bc;M</td></tr><tr><td rowspan="2" align="left" valign="top" colspan="1">
<em>A. fulgidus</em>
</td><td rowspan="2" align="left" valign="top" colspan="1">83&#x000b0;C (65&#x000b0;C)</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>MPT</td><td align="left" valign="top" rowspan="1" colspan="1">16&#x02009;&#x003bc;M</td><td rowspan="2" align="left" valign="top" colspan="1">17,000&#x02009;min<sup>&#x02212;1</sup> (280&#x02009;s<sup>&#x02212;1</sup>)</td><td rowspan="2" align="left" valign="top" colspan="1">(Schmitz et al.,&#x000a0;<sup><a href="#pro5018-bib-0046" rid="pro5018-bib-0046" class=" bibr popnode">1991</a></sup>)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">F<sub>420</sub>H<sub>2</sub>
</td><td align="left" valign="top" rowspan="1" colspan="1">4&#x02009;&#x003bc;M</td></tr><tr><td rowspan="2" align="left" valign="top" colspan="1">
<em>M. jannaschii</em>
</td><td rowspan="2" align="left" valign="top" colspan="1">85&#x000b0;C (55&#x000b0;C)</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>MPT</td><td align="left" valign="top" rowspan="1" colspan="1">58&#x02009;&#x003bc;M</td><td rowspan="2" align="left" valign="top" colspan="1">18,000&#x02009;min<sup>&#x02212;1</sup> (300&#x02009;s<sup>&#x02212;1</sup>)</td><td rowspan="2" align="left" valign="top" colspan="1">This work</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">F<sub>420</sub>H<sub>2</sub>
</td><td align="left" valign="top" rowspan="1" colspan="1">5&#x02009;&#x003bc;M</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140710428955472"><a target="object" rel="noopener" href="/pmc/articles/PMC11094777/table/pro5018-tbl-0001/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="pro5018-note-0002"><p class="p p-first-last">
<em>Note</em>: Mer from <em>M. jannaschii</em> was the only tested enzyme heterologously produced in <em>Escherichia coli</em>.</p></div><div id="pro5018-note-0003"><sup>
a
</sup>The temperatures indicate the growth optimum temperature. The enzyme assay temperature was given in parenthesis.</div></div></div></div><div id="pro5018-sec-0004" class="sec"><h3 id="pro5018-sec-0004title">2.2. Determination of the crystal structure of <span style="fixed-case">jMer</span>
</h3><p class="p p-first">Crystal structures for the apoenzyme of jMer and the binary jMer&#x02010;F<sub>420</sub> complex were determined at a resolution of 1.8 and 1.9&#x02009;&#x000c5;, respectively (Table&#x000a0;<a href="/pmc/articles/PMC11094777/table/pro5018-tbl-0002/" target="table" class="fig-table-link figpopup" rid-figpopup="pro5018-tbl-0002" rid-ob="ob-pro5018-tbl-0002" co-legend-rid=""><span>2</span></a>). The overall structure of jMer is almost identical to those of previously established methanogenic enzymes characterized by a (&#x003b2;&#x003b1;)<sub>8</sub>&#x02010; or TIM&#x02010;barrel fold (Figure&#x000a0;<a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="pro5018-fig-0002" rid-ob="ob-pro5018-fig-0002" co-legend-rid="lgnd_pro5018-fig-0002"><span>2</span></a>). The same fold was also found in the structures of MTHFR and Mfr (Gehl et al.,&#x000a0;<sup><a href="#pro5018-bib-0011" rid="pro5018-bib-0011" class=" bibr popnode">2023</a></sup>; Guenther et al.,&#x000a0;<sup><a href="#pro5018-bib-0013" rid="pro5018-bib-0013" class=" bibr popnode">1999</a></sup>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="pro5018-tbl-0002"><h3>TABLE 2</h3><!--caption a7--><div class="caption"><p>Structure determination statistics for the jMer apoenzyme and the binary complex of jMer and F<sub>420</sub>.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr style="border-bottom:solid 1px #000000"><th align="left" valign="bottom" rowspan="1" colspan="1"></th><th align="left" valign="bottom" rowspan="1" colspan="1">jMer</th><th align="left" valign="bottom" rowspan="1" colspan="1">jMer&#x02009;+&#x02009;F<sub>420</sub>
</th></tr></thead><tbody valign="top"><tr><td align="left" valign="top" rowspan="1" colspan="1">Resolution range (&#x000c5;)</td><td align="left" valign="top" rowspan="1" colspan="1">46.25&#x02013;1.8 (1.86&#x02013;1.8)</td><td align="left" valign="top" rowspan="1" colspan="1">31.39&#x02013;1.902 (1.97&#x02013;1.902)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Space group</td><td align="left" valign="top" rowspan="1" colspan="1">
<em>P</em> 2<sub>1</sub> 2<sub>1</sub> 2<sub>1</sub>
</td><td align="left" valign="top" rowspan="1" colspan="1">
<em>P</em> 4<sub>1</sub> 2<sub>1</sub> 2</td></tr><tr><td colspan="3" align="left" valign="top" rowspan="1">Unit cell dimensions</td></tr><tr><td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1">
<em>a</em>, <em>b</em>, <em>c</em> (&#x000c5;)</td><td align="left" valign="top" rowspan="1" colspan="1">96.66, 96.28, 166.78</td><td align="left" valign="top" rowspan="1" colspan="1">95.91, 95.91, 166.02</td></tr><tr><td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1">
<em>&#x003b1;</em>, <em>&#x003b2;</em>, <em>&#x003b3;</em> (&#x000b0;)</td><td align="left" valign="top" rowspan="1" colspan="1">90, 90, 90</td><td align="left" valign="top" rowspan="1" colspan="1">90, 90, 90</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Unique reflections<sup>
a
</sup>
</td><td align="left" valign="top" rowspan="1" colspan="1">144,123 (14203)</td><td align="left" valign="top" rowspan="1" colspan="1">60,825 (5895)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Completeness (%)<sup>
a
</sup>
</td><td align="left" valign="top" rowspan="1" colspan="1">99.84 (99.54)</td><td align="left" valign="top" rowspan="1" colspan="1">98.83 (97.16)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Wilson <em>B</em>&#x02010;factor</td><td align="left" valign="top" rowspan="1" colspan="1">33.65</td><td align="left" valign="top" rowspan="1" colspan="1">47.64</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Reflections used in refinement</td><td align="left" valign="top" rowspan="1" colspan="1">144,109 (14202)</td><td align="left" valign="top" rowspan="1" colspan="1">60,822 (5895)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Reflections used for <em>R</em>
<sub>free</sub>
</td><td align="left" valign="top" rowspan="1" colspan="1">2002 (200)</td><td align="left" valign="top" rowspan="1" colspan="1">1998 (194)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">
<em>R</em>
<sub>work</sub> (%)<sup>
b
</sup>
</td><td align="left" valign="top" rowspan="1" colspan="1">17.86 (27.11)</td><td align="left" valign="top" rowspan="1" colspan="1">21.95 (32.62)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">
<em>R</em>
<sub>free</sub> (%)<sup>
b
</sup>
</td><td align="left" valign="top" rowspan="1" colspan="1">19.64 (29.09)</td><td align="left" valign="top" rowspan="1" colspan="1">25.25 (33.64)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Protein residues</td><td align="left" valign="top" rowspan="1" colspan="1">1324</td><td align="left" valign="top" rowspan="1" colspan="1">657</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">RMSD bond lengths (&#x000c5;)<sup>
c
</sup>
</td><td align="left" valign="top" rowspan="1" colspan="1">0.008</td><td align="left" valign="top" rowspan="1" colspan="1">0.014</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">RMSD bond angles (&#x000b0;)<sup>
c
</sup>
</td><td align="left" valign="top" rowspan="1" colspan="1">1.07</td><td align="left" valign="top" rowspan="1" colspan="1">1.36</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Ramachandran favored (%)</td><td align="left" valign="top" rowspan="1" colspan="1">97.02</td><td align="left" valign="top" rowspan="1" colspan="1">96.01</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Ramachandran allowed (%)</td><td align="left" valign="top" rowspan="1" colspan="1">2.45</td><td align="left" valign="top" rowspan="1" colspan="1">3.84</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Ramachandran outliers (%)</td><td align="left" valign="top" rowspan="1" colspan="1">0.54</td><td align="left" valign="top" rowspan="1" colspan="1">0.15</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Rotamer outliers (%)</td><td align="left" valign="top" rowspan="1" colspan="1">0.19</td><td align="left" valign="top" rowspan="1" colspan="1">1.16</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Clash score</td><td align="left" valign="top" rowspan="1" colspan="1">4.11</td><td align="left" valign="top" rowspan="1" colspan="1">6.58</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Average <em>B</em>&#x02010;factor</td><td align="left" valign="top" rowspan="1" colspan="1">53.52</td><td align="left" valign="top" rowspan="1" colspan="1">60.38</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">PDB code</td><td align="left" valign="top" rowspan="1" colspan="1">8QPM</td><td align="left" valign="top" rowspan="1" colspan="1">8QPL</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140710428860048"><a target="object" rel="noopener" href="/pmc/articles/PMC11094777/table/pro5018-tbl-0002/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="pro5018-note-0004"><sup>
a
</sup>Values relative to the highest resolution shell are written in parentheses.</div><div id="pro5018-note-0005"><sup>
b
</sup>
<em>R</em>
<sub>free</sub> was calculated for 5% of the reflections that were not included in the refinement.</div><div id="pro5018-note-0006"><sup>
c
</sup>Root mean square deviation.</div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="pro5018-fig-0002" co-legend-rid="lgnd_pro5018-fig-0002"><a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0002/" target="figure" rid-figpopup="pro5018-fig-0002" rid-ob="ob-pro5018-fig-0002"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140710428743840"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is PRO-33-e5018-g006.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is PRO-33-e5018-g006.jpg" src="/pmc/articles/PMC11094777/bin/PRO-33-e5018-g006.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140710428743840"><a target="object" rel="noopener" href="/pmc/articles/PMC11094777/figure/pro5018-fig-0002/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_pro5018-fig-0002"><div><a class="figpopup" href="/pmc/articles/PMC11094777/figure/pro5018-fig-0002/" target="figure" rid-figpopup="pro5018-fig-0002" rid-ob="ob-pro5018-fig-0002">FIGURE 2</a></div><!--caption a7--><div class="caption"><p>Structure of jMer in complex with F<sub>420</sub>. (a) Cartoon model. The &#x003b2;&#x02010;strands of the (&#x003b2;&#x003b1;)<sub>8</sub> core unit are labeled and colored purple, while the &#x003b1;&#x02010;helices of the core unit are colored light blue and the loops are colored salmon. The inserted segments, which form a helical subdomain, are colored in orange. F<sub>420</sub> is shown with the carbon in green. (b) Surface model with the stick model of F<sub>420</sub>. The color code is the same as in panel A.</p></div></div></div><p>A few strand&#x02010;to&#x02010;helix loops crucially forms the active site architecture of TIM barrel enzymes (Farber &#x00026; Petsko,&#x000a0;<sup><a href="#pro5018-bib-0010" rid="pro5018-bib-0010" class=" bibr popnode">1990</a></sup>; Wierenga,&#x000a0;<sup><a href="#pro5018-bib-0060" rid="pro5018-bib-0060" class=" bibr popnode">2001</a></sup>). The active site of jMer is located in a cleft between the &#x003b1;/&#x003b2;&#x02010;barrel domain and a helix&#x02010;bundle subdomain composed of five helices. In the experimentally determined jMer&#x02010;F<sub>420</sub> structure, F<sub>420</sub> is associated with the &#x003b1;/&#x003b2;&#x02010;barrel domain with the isoalloxazine ring positioned at the bottom of the cleft in contact with the C&#x02010;terminal loops of most strands of the (&#x003b1;&#x003b2;)<sub>8</sub> barrel (Figure&#x000a0;<a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="pro5018-fig-0002" rid-ob="ob-pro5018-fig-0002" co-legend-rid="lgnd_pro5018-fig-0002"><span>2a</span></a>). The residual part of F<sub>420</sub> pointing to the entrance of the cleft is sandwiched between the loops after &#x003b2;4 and &#x003b2;5 and reached the N&#x02010;terminal ends of the following helices (Figure&#x000a0;<a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="pro5018-fig-0002" rid-ob="ob-pro5018-fig-0002" co-legend-rid="lgnd_pro5018-fig-0002"><span>2b</span></a>).</p><p class="p p-last">The cleft is sufficiently large to also accommodate methylene&#x02010;H<sub>4</sub>MPT with the pterin head adjacent to the isoalloxazine ring of F<sub>420</sub>. To study the hydride transfer of Mer, we tried to co&#x02010;crystallize jMer with F<sub>420</sub>/F<sub>420</sub>H<sub>2</sub> and methylene&#x02010;/methyl&#x02010;H<sub>4</sub>MPT. However, the C<sub>1</sub> carrier could not be found in the resulting electron density. Therefore, we built a model of the ternary complex of jMer for studying the catalytic mechanism.</p></div><div id="pro5018-sec-0005" class="sec"><h3 id="pro5018-sec-0005title">2.3. Ternary complex model building</h3><p class="p p-first">To compare the active site structure of eMTHFR, hMfr and jMer, the monomeric protein structures were superposed using a three&#x02010;dimensional alignment (Figure&#x000a0;<a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0003/" target="figure" class="fig-table-link figpopup" rid-figpopup="pro5018-fig-0003" rid-ob="ob-pro5018-fig-0003" co-legend-rid="lgnd_pro5018-fig-0003"><span>3</span></a>). A simple three&#x02010;dimensional alignment was not sufficiently accurate because of the large differences between the tertiary structures of jMer and the other two reductases (Figure&#x000a0;<a href="#pro5018-supitem-0004" rid="pro5018-supitem-0004" class=" supplementary-material">S4</a>). For that reason, the known highly related hydride transfer geometry between FADH<sub>2</sub> and methylene&#x02010;H<sub>4</sub>F in eMTHFR, and F<sub>420</sub>H<sub>2</sub> and methylene&#x02010;H<sub>4</sub>MPT in jMer was applied as additional information for alignment. Accordingly, the hydride&#x02010;bearing atoms of FAD (N5) in the eMTHFR structure and of F<sub>420</sub> (C5) in the jMer structure were, at first, superposed (Figure&#x000a0;<a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0003/" target="figure" class="fig-table-link figpopup" rid-figpopup="pro5018-fig-0003" rid-ob="ob-pro5018-fig-0003" co-legend-rid="lgnd_pro5018-fig-0003"><span>3</span></a>, step 1) and then the rest of the proteins was aligned without moving the hydride&#x02010;transferring atoms (Figure&#x000a0;<a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0003/" target="figure" class="fig-table-link figpopup" rid-figpopup="pro5018-fig-0003" rid-ob="ob-pro5018-fig-0003" co-legend-rid="lgnd_pro5018-fig-0003"><span>3</span></a>, step 2). The hMfr and eMTHFR structures could be superimposed by the normal overall procedure due to their higher structural similarities (Gehl et al.,&#x000a0;<sup><a href="#pro5018-bib-0011" rid="pro5018-bib-0011" class=" bibr popnode">2023</a></sup>) (Figure&#x000a0;<a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0003/" target="figure" class="fig-table-link figpopup" rid-figpopup="pro5018-fig-0003" rid-ob="ob-pro5018-fig-0003" co-legend-rid="lgnd_pro5018-fig-0003"><span>3</span></a>, step 3). The root mean square deviation (RMSD) values were 4.7&#x02009;&#x000c5; between eMTHFR and hMfr (over 240 amino acids), 4.6&#x02009;&#x000c5; between hMfr and jMer (over 232 amino acids), and 5.3&#x02009;&#x000c5; between jMer and eMTHFR (over 200 amino acids). After superposition, identical or similar amino acids at the same position in space were identified and the methyl&#x02010;H<sub>4</sub>F of eMTHFR was transferred to hMfr and jMer (Figure&#x000a0;<a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0003/" target="figure" class="fig-table-link figpopup" rid-figpopup="pro5018-fig-0003" rid-ob="ob-pro5018-fig-0003" co-legend-rid="lgnd_pro5018-fig-0003"><span>3</span></a>, step 4). In jMer, the distance between C11 of the modeled methyl&#x02010;H<sub>4</sub>F and C5 of F<sub>420</sub> is 3.3&#x02009;&#x000c5; (Figure&#x000a0;<a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="pro5018-fig-0004" rid-ob="ob-pro5018-fig-0004" co-legend-rid="lgnd_pro5018-fig-0004"><span>4</span></a>). In the modeled jMer&#x02010;F<sub>420</sub>&#x02010;methyl&#x02010;H<sub>4</sub>F complex, methyl&#x02010;H<sub>4</sub>F do not substantially interfere with amino acid residues and its tail has no contact with the protein. In the ternary complex model of jMer shown in this work, methyl&#x02010;H<sub>4</sub>F was present as substrate rather than methyl&#x02010;H<sub>4</sub>MPT based on the alignment strategy. hMfr and eMTHFR were superimposed and methyl&#x02010;H<sub>4</sub>F and FAD was modeled to hMfr.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="pro5018-fig-0003" co-legend-rid="lgnd_pro5018-fig-0003"><a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0003/" target="figure" rid-figpopup="pro5018-fig-0003" rid-ob="ob-pro5018-fig-0003"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140710428713312"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is PRO-33-e5018-g007.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is PRO-33-e5018-g007.jpg" src="/pmc/articles/PMC11094777/bin/PRO-33-e5018-g007.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140710428713312"><a target="object" rel="noopener" href="/pmc/articles/PMC11094777/figure/pro5018-fig-0003/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_pro5018-fig-0003"><div><a class="figpopup" href="/pmc/articles/PMC11094777/figure/pro5018-fig-0003/" target="figure" rid-figpopup="pro5018-fig-0003" rid-ob="ob-pro5018-fig-0003">FIGURE 3</a></div><!--caption a7--><div class="caption"><p>Alignment of three methylene&#x02010;tetrahydropterin reductases. (a) Methods of superposition. To align jMer on eMTHFR the hydride&#x02010;bearing atoms N5 of FAD and C5 of F<sub>420</sub>, highlighted in red, were laid on top of each other (step 1). Then, jMer was rotated on eMTHFR by fixing two hydride&#x02010;bearing atoms (step 2). hMfr was coordinate&#x02010;based aligned on eMTHFR in the normal manner (step 3). Finally, methyl&#x02010;H<sub>4</sub>F from eMTHFR was transferred into the superimposed jMer and hMfr. Identical amino&#x02010;acid residues were determined based on the performed structural alignment (step 4). The result of the alignment of jMer (green) onto eMTHFR (orange) (b), and Mfr (blue) onto eMTHFR (orange) (c). Methyl&#x02010;H<sub>4</sub>F and FAD of eMTHFR are shown by a stick model with carbon in orange. F<sub>420</sub> of jMer is shown by a stick model with carbon in green.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="pro5018-fig-0004" co-legend-rid="lgnd_pro5018-fig-0004"><a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0004/" target="figure" rid-figpopup="pro5018-fig-0004" rid-ob="ob-pro5018-fig-0004"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140710428707408"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is PRO-33-e5018-g001.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is PRO-33-e5018-g001.jpg" src="/pmc/articles/PMC11094777/bin/PRO-33-e5018-g001.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140710428707408"><a target="object" rel="noopener" href="/pmc/articles/PMC11094777/figure/pro5018-fig-0004/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_pro5018-fig-0004"><div><a class="figpopup" href="/pmc/articles/PMC11094777/figure/pro5018-fig-0004/" target="figure" rid-figpopup="pro5018-fig-0004" rid-ob="ob-pro5018-fig-0004">FIGURE 4</a></div><!--caption a7--><div class="caption"><p>Ternary complex model of jMer based on the described alignment procedure. The polypeptide is drawn in a green surface representation, F<sub>420</sub> (carbon in green) and methyl&#x02010;H<sub>4</sub>F (carbon in orange) as sticks (see Figure&#x000a0;<a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0003/" target="figure" class="fig-table-link figpopup" rid-figpopup="pro5018-fig-0003" rid-ob="ob-pro5018-fig-0003" co-legend-rid="lgnd_pro5018-fig-0003"><span>3</span></a>). The N5&#x02010;methyl group of methyl&#x02010;H<sub>4</sub>F is shown in red. The active&#x02010;site structure is shown from different angles (a and b). Although Mer uses methylene&#x02010;/methyl&#x02010;H<sub>4</sub>MPT as the native cofactors, methyl&#x02010;H<sub>4</sub>F was modeled based on the alignment strategy.</p></div></div></div><p class="p">Despite the structural difference between H<sub>4</sub>F and H<sub>4</sub>MPT in the model of jMer, we discuss the possible interactions of the protein and methyl&#x02010;H<sub>4</sub>MPT from the pterin head group to the phenyl ring moiety, which is structurally similar in H<sub>4</sub>F and H<sub>4</sub>MPT. A hydrophobic pocket consisting of the conserved residues Phe233, Val8 and Val230 might be responsible for binding of the phenyl ring of H<sub>4</sub>MPT in jMer. Phe233 is rather distant from the phenyl ring of the tail of H<sub>4</sub>MPT (Figure&#x000a0;<a href="#pro5018-supitem-0005" rid="pro5018-supitem-0005" class=" supplementary-material">S5</a>). However, a conformational change induced by binding of methylene&#x02010;H<sub>4</sub>MPT may be possible. Gln178 is located close to the bottom of the cleft at the C&#x02010;terminal end of &#x003b2;6, which is presumably hydrogen&#x02010;bonded with the F<sub>420</sub> isoalloxazine and the pterin ring. Asp96 protruding from the C&#x02010;terminal end of &#x003b2;4 is also in contact with the pyrimidine ring of the C<sub>1</sub> carrier. Glu6 at the C&#x02010;terminal end of &#x003b2;1 points towards the deaza&#x02010;isoalloxazine and pterin rings. The mentioned amino acids involved in binding of the pterin part of the C<sub>1</sub> carrier have spatial counterparts in MTHFR and hMfr. The four sites for Glu6, Phe233, Gln178 and Asp96 are referred to as positions A&#x02010;D (in Figure&#x000a0;<a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="pro5018-fig-0005" rid-ob="ob-pro5018-fig-0005" co-legend-rid="lgnd_pro5018-fig-0005"><span>5</span></a>). Positions A and D are occupied by acidic residues, position C by amino acids with carboxamide groups, and position B by large hydrophobic residues.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="pro5018-fig-0005" co-legend-rid="lgnd_pro5018-fig-0005"><a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0005/" target="figure" rid-figpopup="pro5018-fig-0005" rid-ob="ob-pro5018-fig-0005"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140710428695552"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is PRO-33-e5018-g002.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is PRO-33-e5018-g002.jpg" src="/pmc/articles/PMC11094777/bin/PRO-33-e5018-g002.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140710428695552"><a target="object" rel="noopener" href="/pmc/articles/PMC11094777/figure/pro5018-fig-0005/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_pro5018-fig-0005"><div><a class="figpopup" href="/pmc/articles/PMC11094777/figure/pro5018-fig-0005/" target="figure" rid-figpopup="pro5018-fig-0005" rid-ob="ob-pro5018-fig-0005">FIGURE 5</a></div><!--caption a7--><div class="caption"><p>Comparison of the tetrahydropterin binding sites of the crystal structure of the ternary eMTHFR complex and the ternary complex models of hMfr and jMer. Carbons of residues of eMTHFR, hMfr, jMer are colored in orange, blue and green. Equivalent positions are indicated by letters. Non&#x02010;native coenzymes were modeled in the structure of hMfr and jMer based on the alignment strategy. This is the case for hMfr (using NADH, here modeled with FAD) and jMer (producing methyl&#x02010;H<sub>4</sub>MPT as a natural product, here modeled with methyl&#x02010;H<sub>4</sub>F).</p></div></div></div></div><div id="pro5018-sec-0006" class="sec"><h3 id="pro5018-sec-0006title">2.4. Mutational analysis of the <span style="fixed-case">C<sub>1</sub>
</span> carrier binding site</h3><p class="p p-first">Similar amino acid residues at the position A to D of the three methylene&#x02010;tetrahydropterin reductases indicate common active site characteristics, despite their insignificant amino acid sequence identity. Structure&#x02010;independent sequence comparison indicated that jMer shares 16% and 12% sequence identity with eMTHFR and Mfr, respectively, and the identity between hMfr and eMTHFR is 17%. A structure&#x02010;based alignment was also performed and presented in Figure&#x000a0;<a href="#pro5018-supitem-0006" rid="pro5018-supitem-0006" class=" supplementary-material">S6</a>. To explore whether the amino&#x02010;acid residues at position A, B, C and D are involved in binding of the C<sub>1</sub> carrier and in catalysis, systematic mutational analyses were performed and the kinetic data were compared with those for the eMTHFR variants reported in the literature (Tables&#x000a0;<a href="/pmc/articles/PMC11094777/table/pro5018-tbl-0003/" target="table" class="fig-table-link figpopup" rid-figpopup="pro5018-tbl-0003" rid-ob="ob-pro5018-tbl-0003" co-legend-rid=""><span>3</span></a> and <a href="/pmc/articles/PMC11094777/table/pro5018-tbl-0004/" target="table" class="fig-table-link figpopup" rid-figpopup="pro5018-tbl-0004" rid-ob="ob-pro5018-tbl-0004" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and4).</span></span><span>4</span></a>). The common reaction between MTHFR and the other two reductases (Mer and Mfr) is the oxidative half reaction of MTHFR, which does not involve the primary hydride carrier NADH. Therefore, we do not discuss the effect of the mutations of eMTHFR on the reactivity with NADH despite their influence on <em>k</em>
<sub>cat</sub> and <em>K</em>
<sub>m</sub> of NADH (see Tables&#x000a0;<a href="/pmc/articles/PMC11094777/table/pro5018-tbl-0003/" target="table" class="fig-table-link figpopup" rid-figpopup="pro5018-tbl-0003" rid-ob="ob-pro5018-tbl-0003" co-legend-rid=""><span>3</span></a> and <a href="/pmc/articles/PMC11094777/table/pro5018-tbl-0004/" target="table" class="fig-table-link figpopup" rid-figpopup="pro5018-tbl-0004" rid-ob="ob-pro5018-tbl-0004" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and4</span></span><span>4</span></a> and Figure&#x000a0;<a href="#pro5018-supitem-0007" rid="pro5018-supitem-0007" class=" supplementary-material">S7</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="pro5018-tbl-0003"><h3>TABLE 3</h3><!--caption a7--><div class="caption"><p>Kinetic constants of eMTHFR, hMfr and jMer.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr style="border-bottom:solid 1px #000000"><th align="left" valign="bottom" rowspan="1" colspan="1">Enzyme</th><th align="left" valign="bottom" rowspan="1" colspan="1">Substrate</th><th align="left" valign="bottom" rowspan="1" colspan="1">App. <em>K</em>
<sub>m</sub> [&#x003bc;M]</th><th align="left" style="border-bottom:solid 1px #000000" valign="bottom" rowspan="1" colspan="1">App. <em>k</em>
<sub>cat</sub> [min<sup>&#x02212;1</sup>]</th><th align="left" valign="bottom" rowspan="1" colspan="1">
<em>k</em>
<sub>cat</sub>/<em>K</em>
<sub>m</sub> [min<sup>&#x02212;1</sup>&#x02009;&#x003bc;M<sup>&#x02212;1</sup>]</th><th align="left" valign="bottom" rowspan="1" colspan="1">Reference</th></tr></thead><tbody valign="top"><tr><td rowspan="4" align="left" valign="top" colspan="1">eMTHFR</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>F</td><td align="center" valign="top" rowspan="1" colspan="1">0.4&#x02009;&#x000b1;&#x02009;0.1</td><td rowspan="2" align="center" valign="top" colspan="1">130&#x02009;&#x000b1;&#x02009;12</td><td align="left" valign="top" rowspan="1" colspan="1">330</td><td rowspan="2" align="left" valign="top" colspan="1">(Trimmer et al.,&#x000a0;<sup><a href="#pro5018-bib-0055" rid="pro5018-bib-0055" class=" bibr popnode">2005</a></sup>)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">NADH</td><td align="center" valign="top" rowspan="1" colspan="1">3.5&#x02009;&#x000b1;&#x02009;0.6</td><td align="left" valign="top" rowspan="1" colspan="1">37</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>F</td><td align="center" valign="top" rowspan="1" colspan="1">0.5&#x02009;&#x000b1;&#x02009;0.1</td><td rowspan="2" align="center" valign="top" colspan="1">620&#x02009;&#x000b1;&#x02009;60<sup>a</sup>
</td><td align="left" valign="top" rowspan="1" colspan="1">1200</td><td rowspan="2" align="left" valign="top" colspan="1">(Lee et al.,&#x000a0;<sup><a href="#pro5018-bib-0023" rid="pro5018-bib-0023" class=" bibr popnode">2009</a></sup>)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">NADH</td><td align="center" valign="top" rowspan="1" colspan="1">20&#x02009;&#x000b1;&#x02009;4</td><td align="left" valign="top" rowspan="1" colspan="1">30</td></tr><tr><td rowspan="2" align="left" valign="top" colspan="1">hMfr</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>F</td><td align="center" valign="top" rowspan="1" colspan="1">160&#x02009;&#x000b1;&#x02009;62</td><td align="center" valign="top" rowspan="1" colspan="1">600&#x02009;&#x000b1;&#x02009;68</td><td align="left" valign="top" rowspan="1" colspan="1">3.9&#x02009;&#x000b1;&#x02009;0.8</td><td rowspan="2" align="left" valign="top" colspan="1">(Gehl et al.,&#x000a0;<sup><a href="#pro5018-bib-0011" rid="pro5018-bib-0011" class=" bibr popnode">2023</a></sup>)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">NADH</td><td align="center" valign="top" rowspan="1" colspan="1">16.0&#x02009;&#x000b1;&#x02009;3.1</td><td align="center" valign="top" rowspan="1" colspan="1">550&#x02009;&#x000b1;&#x02009;19</td><td align="left" valign="top" rowspan="1" colspan="1">35&#x02009;&#x000b1;&#x02009;4.4</td></tr><tr><td rowspan="2" align="left" valign="top" colspan="1">jMer</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>MPT</td><td align="center" valign="top" rowspan="1" colspan="1">58&#x02009;&#x000b1;&#x02009;29.7</td><td align="center" valign="top" rowspan="1" colspan="1">18,000&#x02009;&#x000b1;&#x02009;4000</td><td align="left" valign="top" rowspan="1" colspan="1">350&#x02009;&#x000b1;&#x02009;90</td><td rowspan="2" align="left" valign="top" colspan="1">This work</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">F<sub>420</sub>
</td><td align="center" valign="top" rowspan="1" colspan="1">4.6&#x02009;&#x000b1;&#x02009;0.1</td><td align="center" valign="top" rowspan="1" colspan="1">7000&#x02009;&#x000b1;&#x02009;30</td><td align="left" valign="top" rowspan="1" colspan="1">1500&#x02009;&#x000b1;&#x02009;20</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140710428681600"><a target="object" rel="noopener" href="/pmc/articles/PMC11094777/table/pro5018-tbl-0003/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="pro5018-note-0007"><p class="p p-first-last">
<em>Note</em>: The values for eMTHFR were obtained from the literature.</p></div><div id="pro5018-note-0008"><p class="p p-first-last">Abbreviation: nd, not determined.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="pro5018-tbl-0004"><h3>TABLE 4</h3><!--caption a7--><div class="caption"><p>Kinetic constants for the variants of eMTHFR, hMfr and jMer.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr style="border-bottom:solid 1px #000000"><th align="left" valign="bottom" rowspan="1" colspan="1">Position</th><th align="left" valign="bottom" rowspan="1" colspan="1">Enzyme</th><th align="left" valign="bottom" rowspan="1" colspan="1">Mutation [reference]</th><th align="left" valign="bottom" rowspan="1" colspan="1">Substrate</th><th align="left" valign="bottom" rowspan="1" colspan="1">App. <em>K</em>
<sub>m</sub> [&#x003bc;M]</th><th align="left" style="border-bottom:solid 1px #000000" valign="bottom" rowspan="1" colspan="1">App. <em>k</em>
<sub>cat</sub> [min<sup>&#x02212;1</sup>]</th><th align="left" valign="bottom" rowspan="1" colspan="1">
<em>k</em>
<sub>cat</sub>/<em>K</em>
<sub>m</sub> [min<sup>&#x02212;1</sup>&#x02009;&#x003bc;M<sup>&#x02212;1</sup>]</th></tr></thead><tbody valign="top"><tr><td rowspan="6" align="left" valign="top" colspan="1">A</td><td rowspan="2" align="left" valign="top" colspan="1">eMTHFR</td><td rowspan="2" align="left" valign="top" colspan="1">Glu28Gln (Trimmer et al.,&#x000a0;<sup><a href="#pro5018-bib-0056" rid="pro5018-bib-0056" class=" bibr popnode">2001</a></sup>)</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>F</td><td align="left" valign="top" rowspan="1" colspan="1">nd</td><td rowspan="2" align="left" valign="top" colspan="1">&#x0003c;0.012 (0.002%)<sup>
a
</sup>
</td><td align="left" valign="top" rowspan="1" colspan="1">nd</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">NADH</td><td align="left" valign="top" rowspan="1" colspan="1">nd</td><td align="left" valign="top" rowspan="1" colspan="1">nd</td></tr><tr><td rowspan="2" align="left" valign="top" colspan="1">hMfr</td><td rowspan="2" align="left" valign="top" colspan="1">Glu9Gln (Gehl et al.,&#x000a0;<sup><a href="#pro5018-bib-0011" rid="pro5018-bib-0011" class=" bibr popnode">2023</a></sup>)</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>F</td><td align="left" valign="top" rowspan="1" colspan="1">250&#x02009;&#x000b1;&#x02009;60 (156%)</td><td align="left" valign="top" rowspan="1" colspan="1">1.3&#x02009;&#x000b1;&#x02009;0.1 (0.2%)</td><td align="left" valign="top" rowspan="1" colspan="1">0.005&#x02009;&#x000b1;&#x02009;0.001 (0.1%)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">NADH</td><td align="left" valign="top" rowspan="1" colspan="1">43&#x02009;&#x000b1;&#x02009;12 (268%)</td><td align="left" valign="top" rowspan="1" colspan="1">1.1&#x02009;&#x000b1;&#x02009;0.1 (0.2%)</td><td align="left" valign="top" rowspan="1" colspan="1">0.028&#x02009;&#x000b1;&#x02009;0.004 (0.1%)</td></tr><tr><td rowspan="2" align="left" valign="top" colspan="1">jMer</td><td rowspan="2" align="left" valign="top" colspan="1">Glu6Gln</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>MPT</td><td align="left" valign="top" rowspan="1" colspan="1">34&#x02009;&#x000b1;&#x02009;10 (58%)</td><td align="left" valign="top" rowspan="1" colspan="1">71&#x02009;&#x000b1;&#x02009;8 (0.4%)</td><td align="left" valign="top" rowspan="1" colspan="1">2.2&#x02009;&#x000b1;&#x02009;0.3 (0.6%)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">F<sub>420</sub>
</td><td align="left" valign="top" rowspan="1" colspan="1">4.3&#x02009;&#x000b1;&#x02009;1 (93%)</td><td align="left" valign="top" rowspan="1" colspan="1">36&#x02009;&#x000b1;&#x02009;2 (0.5%)</td><td align="left" valign="top" rowspan="1" colspan="1">8.9&#x02009;&#x000b1;&#x02009;1.8 (0.6%)</td></tr><tr><td rowspan="12" align="left" valign="top" colspan="1">B</td><td rowspan="4" align="left" valign="top" colspan="1">eMTHFR</td><td rowspan="2" align="left" valign="top" colspan="1">Phe223Ala (Lee et al.,&#x000a0;<sup><a href="#pro5018-bib-0023" rid="pro5018-bib-0023" class=" bibr popnode">2009</a></sup>)</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>F</td><td align="left" valign="top" rowspan="1" colspan="1">93&#x02009;&#x000b1;&#x02009;16 (18,600%)<sup>
a
</sup>
</td><td rowspan="2" align="left" valign="top" colspan="1">170&#x02009;&#x000b1;&#x02009;30 (28%)<sup>
a
</sup>
</td><td align="left" valign="top" rowspan="1" colspan="1">nd</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">NADH</td><td align="left" valign="top" rowspan="1" colspan="1">140&#x02009;&#x000b1;&#x02009;7 (700%)<sup>
a
</sup>
</td><td align="left" valign="top" rowspan="1" colspan="1">nd</td></tr><tr><td rowspan="2" align="left" valign="top" colspan="1">Phe223Leu (Lee et al.,&#x000a0;<sup><a href="#pro5018-bib-0023" rid="pro5018-bib-0023" class=" bibr popnode">2009</a></sup>)</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>F</td><td align="left" valign="top" rowspan="1" colspan="1">8&#x02009;&#x000b1;&#x02009;2 (1600%)<sup>
a
</sup>
</td><td rowspan="2" align="left" valign="top" colspan="1">840&#x02009;&#x000b1;&#x02009;120 (135%)<sup>
a
</sup>
</td><td align="left" valign="top" rowspan="1" colspan="1">nd</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">NADH</td><td align="left" valign="top" rowspan="1" colspan="1">240&#x02009;&#x000b1;&#x02009;40 (1180%)<sup>
a
</sup>
</td><td align="left" valign="top" rowspan="1" colspan="1">nd</td></tr><tr><td rowspan="4" align="left" valign="top" colspan="1">hMfr</td><td rowspan="2" align="left" valign="top" colspan="1">Leu221Phe</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>F</td><td align="left" valign="top" rowspan="1" colspan="1">360&#x02009;&#x000b1;&#x02009;30 (219%)</td><td align="left" valign="top" rowspan="1" colspan="1">180&#x02009;&#x000b1;&#x02009;6 (31%)</td><td align="left" valign="top" rowspan="1" colspan="1">0.51&#x02009;&#x000b1;&#x02009;0.02 (13%)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">NADH</td><td align="left" valign="top" rowspan="1" colspan="1">3.4&#x02009;&#x000b1;&#x02009;0.8 (21%)</td><td align="left" valign="top" rowspan="1" colspan="1">89&#x02009;&#x000b1;&#x02009;1 (16%)</td><td align="left" valign="top" rowspan="1" colspan="1">26.9&#x02009;&#x000b1;&#x02009;5.1 (78%)</td></tr><tr><td rowspan="2" align="left" valign="top" colspan="1">Leu221Ala</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>F</td><td align="left" valign="top" rowspan="1" colspan="1">700&#x02009;&#x000b1;&#x02009;120 (433%)</td><td align="left" valign="top" rowspan="1" colspan="1">130&#x02009;&#x000b1;&#x02009;10 (21%)</td><td align="left" valign="top" rowspan="1" colspan="1">0.18&#x02009;&#x000b1;&#x02009;0.01 (5%)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">NADH</td><td align="left" valign="top" rowspan="1" colspan="1">5.2&#x02009;&#x000b1;&#x02009;0.5 (33%)</td><td align="left" valign="top" rowspan="1" colspan="1">38.0&#x02009;&#x000b1;&#x02009;0.3 (7%)</td><td align="left" valign="top" rowspan="1" colspan="1">7.3&#x02009;&#x000b1;&#x02009;0.6 (21%)</td></tr><tr><td rowspan="4" align="left" valign="top" colspan="1">jMer</td><td rowspan="2" align="left" valign="top" colspan="1">Phe233Ala</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>MPT</td><td align="left" valign="top" rowspan="1" colspan="1">100&#x02009;&#x000b1;&#x02009;50 (180%)</td><td align="left" valign="top" rowspan="1" colspan="1">2200&#x02009;&#x000b1;&#x02009;540 (12%)</td><td align="left" valign="top" rowspan="1" colspan="1">23.1&#x02009;&#x000b1;&#x02009;4.2 (7%)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">F<sub>420</sub>
</td><td align="left" valign="top" rowspan="1" colspan="1">1.8&#x02009;&#x000b1;&#x02009;2.1 (39%)</td><td align="left" valign="top" rowspan="1" colspan="1">470&#x02009;&#x000b1;&#x02009;60 (7%)</td><td align="left" valign="top" rowspan="1" colspan="1">377&#x02009;&#x000b1;&#x02009;814 (25%)</td></tr><tr><td rowspan="2" align="left" valign="top" colspan="1">Phe233Leu</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>MPT</td><td align="left" valign="top" rowspan="1" colspan="1">150&#x02009;&#x000b1;&#x02009;100 (267%)</td><td align="left" valign="top" rowspan="1" colspan="1">13,000&#x02009;&#x000b1;&#x02009;4700 (73%)</td><td align="left" valign="top" rowspan="1" colspan="1">97.9&#x02009;&#x000b1;&#x02009;23.4 (28%)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">F<sub>420</sub>
</td><td align="left" valign="top" rowspan="1" colspan="1">3.1&#x02009;&#x000b1;&#x02009;2.6 (67%)</td><td align="left" valign="top" rowspan="1" colspan="1">1700&#x02009;&#x000b1;&#x02009;240 (24%)</td><td align="left" valign="top" rowspan="1" colspan="1">1400&#x02009;&#x000b1;&#x02009;1300 (90%)</td></tr><tr><td rowspan="12" align="left" valign="top" colspan="1">C</td><td rowspan="4" align="left" valign="top" colspan="1">eMTHFR</td><td rowspan="2" align="left" valign="top" colspan="1">Gln183Glu (Zuo et al.,&#x000a0;<sup><a href="#pro5018-bib-0064" rid="pro5018-bib-0064" class=" bibr popnode">2018</a></sup>)</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>F</td><td align="left" valign="top" rowspan="1" colspan="1">110&#x02009;&#x000b1;&#x02009;36 (27,000%)<sup>
a
</sup>
</td><td rowspan="2" align="left" valign="top" colspan="1">160&#x02009;&#x000b1;&#x02009;30 (123%)<sup>
a
</sup>
</td><td align="left" valign="top" rowspan="1" colspan="1">nd</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">NADH</td><td align="left" valign="top" rowspan="1" colspan="1">6.6&#x02009;&#x000b1;&#x02009;0.7 (189%)<sup>
a
</sup>
</td><td align="left" valign="top" rowspan="1" colspan="1">nd</td></tr><tr><td rowspan="2" align="left" valign="top" colspan="1">Gln183Ala (Zuo et al.,&#x000a0;<sup><a href="#pro5018-bib-0064" rid="pro5018-bib-0064" class=" bibr popnode">2018</a></sup>)</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>F</td><td align="left" valign="top" rowspan="1" colspan="1">100&#x02009;&#x000b1;&#x02009;10 (26,000%)<sup>
a
</sup>
</td><td rowspan="2" align="left" valign="top" colspan="1">17&#x02009;&#x000b1;&#x02009;0.6 (13%)<sup>
a
</sup>
</td><td align="left" valign="top" rowspan="1" colspan="1">nd</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">NADH</td><td align="left" valign="top" rowspan="1" colspan="1">&#x0003c;10 (&#x0003c;286%)<sup>
a
</sup>
</td><td align="left" valign="top" rowspan="1" colspan="1">nd</td></tr><tr><td rowspan="4" align="left" valign="top" colspan="1">hMfr</td><td rowspan="2" align="left" valign="top" colspan="1">Gln177Glu</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>F</td><td align="left" valign="top" rowspan="1" colspan="1">380&#x02009;&#x000b1;&#x02009;30 (236%)</td><td align="left" valign="top" rowspan="1" colspan="1">100&#x02009;&#x000b1;&#x02009;3 (17%)</td><td align="left" valign="top" rowspan="1" colspan="1">0.26&#x02009;&#x000b1;&#x02009;0.01 (7%)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">NADH</td><td align="left" valign="top" rowspan="1" colspan="1">120&#x02009;&#x000b1;&#x02009;40 (769%)</td><td align="left" valign="top" rowspan="1" colspan="1">110&#x02009;&#x000b1;&#x02009;15 (20%)</td><td align="left" valign="top" rowspan="1" colspan="1">0.9&#x02009;&#x000b1;&#x02009;0.1 (3%)</td></tr><tr><td rowspan="2" align="left" valign="top" colspan="1">Gln177Ala</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>F</td><td align="left" valign="top" rowspan="1" colspan="1">570&#x02009;&#x000b1;&#x02009;60 (347%)</td><td align="left" valign="top" rowspan="1" colspan="1">56&#x02009;&#x000b1;&#x02009;3 (9%)</td><td align="left" valign="top" rowspan="1" colspan="1">0.1&#x02009;&#x000b1;&#x02009;0.0 (3%)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">NADH</td><td align="left" valign="top" rowspan="1" colspan="1">28&#x02009;&#x000b1;&#x02009;7 (176%)</td><td align="left" valign="top" rowspan="1" colspan="1">24&#x02009;&#x000b1;&#x02009;1 (4%)</td><td align="left" valign="top" rowspan="1" colspan="1">0.9&#x02009;&#x000b1;&#x02009;0.1 (3%)</td></tr><tr><td rowspan="4" align="left" valign="top" colspan="1">jMer</td><td rowspan="2" align="left" valign="top" colspan="1">Gln178Glu</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>MPT</td><td align="left" valign="top" rowspan="1" colspan="1">20&#x02009;&#x000b1;&#x02009;5 (34%)</td><td align="left" valign="top" rowspan="1" colspan="1">430&#x02009;&#x000b1;&#x02009;30 (2%)</td><td align="left" valign="top" rowspan="1" colspan="1">22.3&#x02009;&#x000b1;&#x02009;3.1 (6%)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">F<sub>420</sub>
</td><td align="left" valign="top" rowspan="1" colspan="1">27&#x02009;&#x000b1;&#x02009;7 (593%)</td><td align="left" valign="top" rowspan="1" colspan="1">510&#x02009;&#x000b1;&#x02009;50 (7%)</td><td align="left" valign="top" rowspan="1" colspan="1">18.9&#x02009;&#x000b1;&#x02009;1.9 (1%)</td></tr><tr><td rowspan="2" align="left" valign="top" colspan="1">Gln178Ala</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>MPT</td><td align="left" valign="top" rowspan="1" colspan="1">23&#x02009;&#x000b1;&#x02009;10 (40%)</td><td align="left" valign="top" rowspan="1" colspan="1">3.3&#x02009;&#x000b1;&#x02009;0.5 (0.02%)</td><td align="left" valign="top" rowspan="1" colspan="1">0.2&#x02009;&#x000b1;&#x02009;0.0 (0.1%)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">F<sub>420</sub>
</td><td align="left" valign="top" rowspan="1" colspan="1">61&#x02009;&#x000b1;&#x02009;67 (1326%)</td><td align="left" valign="top" rowspan="1" colspan="1">6.1&#x02009;&#x000b1;&#x02009;3.7 (0.1%)</td><td align="left" valign="top" rowspan="1" colspan="1">0.03&#x02009;&#x000b1;&#x02009;0.17 (0.002%)</td></tr><tr><td rowspan="6" align="left" valign="top" colspan="1">D</td><td rowspan="2" align="left" valign="top" colspan="1">eMTHFR</td><td rowspan="2" align="left" valign="top" colspan="1">Asp120Asn (Trimmer et al.,&#x000a0;<sup><a href="#pro5018-bib-0055" rid="pro5018-bib-0055" class=" bibr popnode">2005</a></sup>)</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>F</td><td align="left" valign="top" rowspan="1" colspan="1">17&#x02009;&#x000b1;&#x02009;3 (4250%)<sup>
a
</sup>
</td><td rowspan="2" align="left" valign="top" colspan="1">0.44&#x02009;&#x000b1;&#x02009;0.04 (0.3%)<sup>
a
</sup>
</td><td rowspan="2" align="left" valign="top" colspan="1">0.026 (0.01%)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">NADH</td><td align="left" valign="top" rowspan="1" colspan="1">&#x0003c;3.5 (&#x0003c;100%)<sup>
a
</sup>
</td></tr><tr><td rowspan="2" align="left" valign="top" colspan="1">hMfr</td><td rowspan="2" align="left" valign="top" colspan="1">Glu55Gln</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>F</td><td align="left" valign="top" rowspan="1" colspan="1">180&#x02009;&#x000b1;&#x02009;60 (108%)</td><td align="left" valign="top" rowspan="1" colspan="1">180&#x02009;&#x000b1;&#x02009;20 (30%)</td><td align="left" valign="top" rowspan="1" colspan="1">1.1&#x02009;&#x000b1;&#x02009;0.2 (28%)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">NADH</td><td align="left" valign="top" rowspan="1" colspan="1">5.6&#x02009;&#x000b1;&#x02009;1.1 (35%)</td><td align="left" valign="top" rowspan="1" colspan="1">134&#x02009;&#x000b1;&#x02009;2 (25%)</td><td align="left" valign="top" rowspan="1" colspan="1">25&#x02009;&#x000b1;&#x02009;4 (71%)</td></tr><tr><td rowspan="2" align="left" valign="top" colspan="1">jMer</td><td rowspan="2" align="left" valign="top" colspan="1">Asp96Asn</td><td align="left" valign="top" rowspan="1" colspan="1">Methylene&#x02010;H<sub>4</sub>MPT</td><td align="left" valign="top" rowspan="1" colspan="1">140&#x02009;&#x000b1;&#x02009;30 (246%)</td><td align="left" valign="top" rowspan="1" colspan="1">700&#x02009;&#x000b1;&#x02009;50 (4%)</td><td align="left" valign="top" rowspan="1" colspan="1">4.9&#x02009;&#x000b1;&#x02009;0.4 (1%)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">F<sub>420</sub>
</td><td align="left" valign="top" rowspan="1" colspan="1">1.6&#x02009;&#x000b1;&#x02009;2.8 (35%)</td><td align="left" valign="top" rowspan="1" colspan="1">85&#x02009;&#x000b1;&#x02009;9 (1%)</td><td align="left" valign="top" rowspan="1" colspan="1">5&#x02009;&#x000b1;&#x02009;48 (0.3%)</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140710428605120"><a target="object" rel="noopener" href="/pmc/articles/PMC11094777/table/pro5018-tbl-0004/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="pro5018-note-0009"><p class="p p-first-last">
<em>Note</em>: References for the eMTHFR variants and the published hMfr variants are listed below the variant name in parenthesis. The ratio of the <em>K</em>
<sub>m</sub>, <em>k</em>
<sub>cat</sub>, and <em>k</em>
<sub>cat</sub>/<em>K</em>
<sub>m</sub> values of the variant compared to wild type is written below the absolute values in parenthesis.</p></div><div id="pro5018-note-0010"><p class="p p-first-last">Abbreviation: nd, not determined.</p></div><div id="pro5018-note-0011"><sup>
a
</sup>The kinetic parameters were compared with those of the wild&#x02010;type described in the references specified in the row of this table.</div></div></div><p>Position A contains a glutamate residue in all three reductases. For eMTHFR, the <em>k</em>
<sub>cat</sub> value of the Glu28Gln variant is reduced from 132 to 0.012&#x02009;min<sup>&#x02212;1</sup> (0.01% residual activity). It has been proposed that Glu28 is the key catalytic residue for protonation of methylene&#x02010;H<sub>4</sub>F to activate the C<sub>1</sub> unit for the reaction (Trimmer et al.,&#x000a0;<sup><a href="#pro5018-bib-0056" rid="pro5018-bib-0056" class=" bibr popnode">2001</a></sup>). In hMfr and jMer, apparent <em>k</em>
<sub>cat</sub> of the equivalent Glu9Gln and Glu6Gln variants decreased from 594 to 1.3&#x02009;min<sup>&#x02212;1</sup> (0.2%) (Gehl et al.,&#x000a0;<sup><a href="#pro5018-bib-0011" rid="pro5018-bib-0011" class=" bibr popnode">2023</a></sup>), and from 18,200 to 71&#x02009;min<sup>&#x02212;1</sup> (0.4%), respectively. Neither variant showed a significant increase in the apparent <em>K</em>
<sub>m</sub> for their respective C<sub>1</sub> carrier, suggesting that glutamate and glutamine bind in a related manner. X&#x02010;ray structure analysis of the jMer_E6Q and hMfr_E9Q variants (Table&#x000a0;<a href="#pro5018-supitem-0001" rid="pro5018-supitem-0001" class=" supplementary-material">S1</a>) were performed and the structures were compared with the wild&#x02010;type enzyme (Figure&#x000a0;<a href="#pro5018-supitem-0008" rid="pro5018-supitem-0008" class=" supplementary-material">S8</a>), which confirmed unchanged active site architectures compared with the wild&#x02010;type enzymes.</p><p>Position B in eMTHFR is occupied by Phe223 whose side chain forms &#x003c0;&#x02013;&#x003c0; interactions with the phenyl ring of methyl&#x02010;H<sub>4</sub>F. Mutation of Phe223 to alanine or leucine in eMTHFR is associated with a dramatic increase in the <em>K</em>
<sub>m</sub> values for methylene&#x02010;H<sub>4</sub>F and NADH without substantially changing the <em>k</em>
<sub>cat</sub> values (Lee et al.,&#x000a0;<sup><a href="#pro5018-bib-0023" rid="pro5018-bib-0023" class=" bibr popnode">2009</a></sup>). jMer carries a conserved Phe233 at position B but the exchange to alanine or leucine changed the apparent <em>K</em>
<sub>m</sub> values only slightly. The small increase of the apparent <em>K</em>
<sub>m</sub> values in the Phe233 variants in jMer indicate its participation in binding of the C<sub>1</sub> carrier. However, the formation of strong &#x003c0;&#x02013;&#x003c0; interactions as described for eMTHFR are unlikely for jMer due to the large distance between the phenyl ring moieties (see Figure&#x000a0;<a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="pro5018-fig-0005" rid-ob="ob-pro5018-fig-0005" co-legend-rid="lgnd_pro5018-fig-0005"><span>5</span></a>). In hMfr, position B is occupied by the strictly conserved Leu221. Exchange of Leu221 to phenylalanine or alanine increased the <em>K</em>
<sub>m</sub> values of methylene&#x02010;H<sub>4</sub>F 2&#x02013;4&#x02010;fold, which is again small compared with those of the eMTHFR variants. This result supports the hypothesis that Leu221 is involved in the binding of methylene&#x02010;H<sub>4</sub>F but with a different strength compared with that of eMTHFR.</p><p>Position C in the ternary complex structure of eMTHFR is occupied by Gln183, whose side chain carboxamide forms a bidentate hydrogen bond with NH8 and N1 (Pejchal et al.,&#x000a0;<sup><a href="#pro5018-bib-0042" rid="pro5018-bib-0042" class=" bibr popnode">2005</a></sup>). eMTHFR_Gln183Glu and Gln183Ala mutations are associated with a large increase in <em>K</em>
<sub>m</sub> values for methylene&#x02010;H<sub>4</sub>F (&#x0003e;250 fold). In contrast, the Gln183Glu mutation caused no change of the <em>k</em>
<sub>cat</sub> value (Zuo et al.,&#x000a0;<sup><a href="#pro5018-bib-0064" rid="pro5018-bib-0064" class=" bibr popnode">2018</a></sup>). The exchange of Gln177 in hMfr to glutamate or alanine moderately increased the apparent <em>K</em>
<sub>m</sub> values for methylene&#x02010;H<sub>4</sub>F (2&#x02013;3&#x02010;fold) compared with that of eMTHFR. Notably, the apparent <em>K</em>
<sub>m</sub> value for NADH increased in hMfr_Gln177Glu (8&#x02010;fold), suggesting that the acidic side chain also influences the binding of NADH. In contrast, the replacement of Gln178 to glutamate and alanine in jMer slightly decreases the apparent <em>K</em>
<sub>m</sub> values for methylene&#x02010;H<sub>4</sub>MPT (30%&#x02013;40%). Although H<sub>4</sub>F and H<sub>4</sub>MPT share NH8 and N1 of the pterin ring the interactions between the residue in position C and the pterin ring significantly differ between the three enzymes. In the jMer_Gln178 variants, the apparent <em>K</em>
<sub>m</sub> values for F<sub>420</sub> substantially increased (6&#x02013;13 fold). Since the backbone of Gln178 is part of the F<sub>420</sub> binding site in jMer, the Gln178 variation may primarily affect the binding of F<sub>420</sub> and not the binding of methylene&#x02010;H<sub>4</sub>MPT.</p><p>At position D, the carboxy group of Asp120 forms a bidentate hydrogen bond to N3 and the exo&#x02010;NH2 group of methyl&#x02010;H<sub>4</sub>F in the ternary complex of eMTHFR (Pejchal et al.,&#x000a0;<sup><a href="#pro5018-bib-0042" rid="pro5018-bib-0042" class=" bibr popnode">2005</a></sup>). The mutation of Asp120 to asparagine resulted in a large increase (&#x0003e;40 fold) of the <em>K</em>
<sub>m</sub> values for the C<sub>1</sub> carrier in eMTHFR and a large decrease to 0.3% of the <em>k</em>
<sub>cat</sub> value (Trimmer et al.,&#x000a0;<sup><a href="#pro5018-bib-0055" rid="pro5018-bib-0055" class=" bibr popnode">2005</a></sup>). In the structure of hMfr, a glutamate rather than aspartate residue is located at the equivalent position. The hMfr_Glu55Gln variant did not show a significant increase in the apparent <em>K</em>
<sub>m</sub> value for methylene&#x02010;H<sub>4</sub>F but a decrease in the apparent <em>k</em>
<sub>cat</sub> value. In the case of jMer_Asp96Asn, the apparent <em>K</em>
<sub>m</sub> value was increased 2.5&#x02010;fold. These results suggest that the function of aspartate at position D in binding of the C<sub>1</sub> carrier deviates in eMTHFR compared with the other two reductases. For hMfr, Glu55 is not strictly conserved and alanine and valine are found at this position in addition to glutamate and aspartate, which also supports this conclusion.</p><p class="p">Trimmer et al. proposed that methylene&#x02010;H<sub>4</sub>F is activated by protonation of N10 to form an N5 minimum cation (Trimmer et al.,&#x000a0;<sup><a href="#pro5018-bib-0056" rid="pro5018-bib-0056" class=" bibr popnode">2001</a></sup>). Based on the mutational analysis at position A, it was possible to extend the proposed model of the catalytic mechanism of MTHFR (Trimmer et al.,&#x000a0;<sup><a href="#pro5018-bib-0056" rid="pro5018-bib-0056" class=" bibr popnode">2001</a></sup>) not only to Mfr (Gehl et al.,&#x000a0;<sup><a href="#pro5018-bib-0011" rid="pro5018-bib-0011" class=" bibr popnode">2023</a></sup>) but also to Mer. The mechanism is initiated by the protonation of the C<sub>1</sub> carrier to activate this molecule, followed by a hydride transfer from the hydride donor to the activated C<sub>1</sub> unit (Figure&#x000a0;<a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0006/" target="figure" class="fig-table-link figpopup" rid-figpopup="pro5018-fig-0006" rid-ob="ob-pro5018-fig-0006" co-legend-rid="lgnd_pro5018-fig-0006"><span>6</span></a>). As the methylene group in methylene&#x02010;H<sub>4</sub>F and methylene&#x02010;H<sub>4</sub>MPT is chemically a rather unreactive aminal, it is unlikely that a hydride can be directly transferred. Therefore, the formation of a positively charged 5&#x02010;iminium cation is crucial for catalysis. The positive charge can be delocalized via the pterin ring system, which stabilizes this intermediate state. In contrast to the uncharged methylene group, the positively charged 5&#x02010;iminium cation is a good hydride acceptor for a hydride of either FADH<sub>2</sub> in MTHFR, F<sub>420</sub>H<sub>2</sub> in Mer or NADH in Mfr. On the other hand, even though some identical or similar residues were observed in the three methylene&#x02010;tetrahydropterin reductases at the predicted binding site of the C<sub>1</sub> carriers at positions B&#x02013;D, the kinetic effects of mutations argue against common features for the binding of the C<sub>1</sub> carriers. An inaccurate modeling of methylene&#x02010;tetrahydropterin due to substrate&#x02010;induced conformational changes of contacting polypeptide segments and a consecutive bias in the interpretation of different kinetic behaviors cannot be fully excluded. On the other hand, the substantial changes of the kinetic values argue for a participation of the four residues, in particular, of the glutamate of position A.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="pro5018-fig-0006" co-legend-rid="lgnd_pro5018-fig-0006"><a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0006/" target="figure" rid-figpopup="pro5018-fig-0006" rid-ob="ob-pro5018-fig-0006"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140710428274288"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is PRO-33-e5018-g005.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is PRO-33-e5018-g005.jpg" src="/pmc/articles/PMC11094777/bin/PRO-33-e5018-g005.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140710428274288"><a target="object" rel="noopener" href="/pmc/articles/PMC11094777/figure/pro5018-fig-0006/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_pro5018-fig-0006"><div><a class="figpopup" href="/pmc/articles/PMC11094777/figure/pro5018-fig-0006/" target="figure" rid-figpopup="pro5018-fig-0006" rid-ob="ob-pro5018-fig-0006">FIGURE 6</a></div><!--caption a7--><div class="caption"><p>Postulated common catalytic mechanism for the three methylene&#x02010;tetrahydropterin reductases. In the first step, N10 of the pterin part of the C<sub>1</sub> carrier is protonated leading to the formation of an minimum cation. The positive charge of the minimum cation is delocalized over the pterin ring system, presumably stabilizing the intermediate state. The positively charged minimum cation is an excellent hydride acceptor. After hydride transfer, the reaction product is obtained. R1 is a hydrogen or a methyl group in H<sub>4</sub>F and H<sub>4</sub>MPT, respectively.</p></div></div></div></div><div id="pro5018-sec-0007" class="sec"><h3 id="pro5018-sec-0007title">2.5. Mutational analysis of the non&#x02010;prolyl <em>cis</em>&#x02010;peptide bond</h3><p class="p p-first">Non&#x02010;prolyl <em>cis</em>&#x02010;peptide (NPCP) bonds are rare in proteins, but they play an important role when they do occur (Jabs et al.,&#x000a0;<sup><a href="#pro5018-bib-0017" rid="pro5018-bib-0017" class=" bibr popnode">1999</a></sup>; Pal &#x00026; Chakrabarti,&#x000a0;<sup><a href="#pro5018-bib-0040" rid="pro5018-bib-0040" class=" bibr popnode">1999</a></sup>). The NPCP bond in jMer is located in the loop after &#x003b2;3, below the central pyridine ring of F<sub>420</sub>, between Gly61 and Val62 (Figure&#x000a0;<a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="pro5018-fig-0007" rid-ob="ob-pro5018-fig-0007" co-legend-rid="lgnd_pro5018-fig-0007"><span>7</span></a>). This NPCP bond is conserved at the equivalent position in all known Mer structures and even in other, for example, in the F<sub>420</sub>&#x02010;dependent glucose&#x02010;6&#x02010;phosphate dehydrogenase (Bashiri et al.,&#x000a0;<sup><a href="#pro5018-bib-0003" rid="pro5018-bib-0003" class=" bibr popnode">2008</a></sup>) and the F<sub>420</sub>&#x02010;dependent alcohol dehydrogenase (Aufhammer et al.,&#x000a0;<sup><a href="#pro5018-bib-0001" rid="pro5018-bib-0001" class=" bibr popnode">2004</a></sup>), but not in all enzymes of the bacterial luciferase superfamily. It has been proposed that the NPCP bond acts as a backstop for the placement of F<sub>420</sub> in the active site (Aufhammer et al.,&#x000a0;<sup><a href="#pro5018-bib-0002" rid="pro5018-bib-0002" class=" bibr popnode">2005</a></sup>). In the present study, the role of this NPCP bond was tested by exchanging Val62 to Pro62, presumably resulting in conversion from the NPCP bond to a prolyl&#x02010;<em>cis</em> peptide (PCP) bond. The turnover number of the purified jMer_Val62Pro variant enzyme was 3.2&#x02009;min<sup>&#x02212;1</sup> that is much lower than wild&#x02010;type jMer (5700&#x02009;min<sup>&#x02212;1</sup>) under standard assay conditions.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="pro5018-fig-0007" co-legend-rid="lgnd_pro5018-fig-0007"><a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0007/" target="figure" rid-figpopup="pro5018-fig-0007" rid-ob="ob-pro5018-fig-0007"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140710428253552"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is PRO-33-e5018-g003.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is PRO-33-e5018-g003.jpg" src="/pmc/articles/PMC11094777/bin/PRO-33-e5018-g003.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140710428253552"><a target="object" rel="noopener" href="/pmc/articles/PMC11094777/figure/pro5018-fig-0007/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_pro5018-fig-0007"><div><a class="figpopup" href="/pmc/articles/PMC11094777/figure/pro5018-fig-0007/" target="figure" rid-figpopup="pro5018-fig-0007" rid-ob="ob-pro5018-fig-0007">FIGURE 7</a></div><!--caption a7--><div class="caption"><p>The jMer&#x02010;F<sub>420</sub> complex, the AlphaFold model of jMer_Val62Pro, which contains prolyl <em>cis</em>&#x02010;peptide (PCP) and modeled F<sub>420</sub>, and the crystal structure of Mer from <em>M. kandleri</em> (kMer). F<sub>420</sub> and the amino acids are shown as stick models with green carbons. F<sub>420</sub> was also presented by drawing the corresponding atoms as transparent spheres with van der Waals radii and the <em>cis</em>&#x02010;peptide bond regions as black dotted spheres. F<sub>420</sub> in kMer was modeled by alignment of the whole protein with the jMer structure in complex with F<sub>420</sub>. The non&#x02010;prolyl <em>cis</em>&#x02010;peptide bond of kMer is placed between Gly64 and Ile65. The side chain of the isoleucine in kMer does not overlap with modeled F<sub>420</sub>, whereas the side chain of proline in jMer_Val62Pro collides with F<sub>420</sub>.</p></div></div></div><p class="p p-last">The AlphaFold model of jMer_Val62Pro supported the predicted PCP bond by a very high pLDDT of over 98 in the region (P60&#x02013;T63) (Figure&#x000a0;<a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="pro5018-fig-0007" rid-ob="ob-pro5018-fig-0007" co-legend-rid="lgnd_pro5018-fig-0007"><span>7</span></a>). Alignment of the jMer wild type binary complex structure and the jMer_Val62Pro model showed that the backbone of the corresponding loop and the C<sub>&#x003b2;</sub> atoms of Val62 and Pro62 remain in the same position after mutation (Figure&#x000a0;<a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="pro5018-fig-0007" rid-ob="ob-pro5018-fig-0007" co-legend-rid="lgnd_pro5018-fig-0007"><span>7</span></a>). However, C<sub>&#x003b3;</sub> and C<sub>&#x003b4;</sub> of Pro62 occupy the space of F<sub>420</sub> in the conformation of the wild type enzyme. Although the backstop function of the <em>cis</em>&#x02010;peptide conformation is maintained, the ability to bind F<sub>420</sub> is thereby substantially reduced. Spatial consideration indicated that hydrophobic residues of moderate size can replace Val62. Indeed, valine is not strictly conserved at this position and isoleucine is found in other Mer enzymes, e.g. in Mer of <em>M. kandleri</em> (Figure&#x000a0;<a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="pro5018-fig-0007" rid-ob="ob-pro5018-fig-0007" co-legend-rid="lgnd_pro5018-fig-0007"><span>7</span></a>) (Shima et al.,&#x000a0;<sup><a href="#pro5018-bib-0049" rid="pro5018-bib-0049" class=" bibr popnode">2000</a></sup>). Larger side chains cannot be placed at this position because they would interfere with the loop after &#x003b1;4, which is involved in the binding of the first two hydroxy groups of the F<sub>420</sub> tail region. Small side chains, such as in alanine, do not reach F<sub>420</sub> and cannot exert pressure to adjust the conformation of F<sub>420</sub>. This analysis comprehensibly demonstrates that proteins can realize an essential biological function by introducing an energetically unfavorable NPCP bond, which cannot be achieved by a PCP bond.</p></div><div id="pro5018-sec-0008" class="sec"><h3 id="pro5018-sec-0008title">2.6. Phylogenetic analyses of methylene&#x02010;tetrahydropterin reductases</h3><p class="p p-first">To obtain further insights into the phylogenetic relationship of the three methylene&#x02010;tetrahydropterin reductases, a phylogenetic tree was constructed (Figure&#x000a0;<a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="pro5018-fig-0008" rid-ob="ob-pro5018-fig-0008" co-legend-rid="lgnd_pro5018-fig-0008"><span>8</span></a>) using sequences from the bacterial luciferase family including Mer (Mer superfamily; Table&#x000a0;<a href="#pro5018-supitem-0001" rid="pro5018-supitem-0001" class=" supplementary-material">S2</a>), the FAD&#x02010;dependent methylene&#x02010;H<sub>4</sub>F reductase superfamily (MTHFR superfamily; Table&#x000a0;<a href="#pro5018-supitem-0001" rid="pro5018-supitem-0001" class=" supplementary-material">S3</a>), and Mfr for BLAST searches.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="pro5018-fig-0008" co-legend-rid="lgnd_pro5018-fig-0008"><a href="/pmc/articles/PMC11094777/figure/pro5018-fig-0008/" target="figure" rid-figpopup="pro5018-fig-0008" rid-ob="ob-pro5018-fig-0008"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140710428236352"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is PRO-33-e5018-g008.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is PRO-33-e5018-g008.jpg" src="/pmc/articles/PMC11094777/bin/PRO-33-e5018-g008.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140710428236352"><a target="object" rel="noopener" href="/pmc/articles/PMC11094777/figure/pro5018-fig-0008/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_pro5018-fig-0008"><div><a class="figpopup" href="/pmc/articles/PMC11094777/figure/pro5018-fig-0008/" target="figure" rid-figpopup="pro5018-fig-0008" rid-ob="ob-pro5018-fig-0008">FIGURE 8</a></div><!--caption a7--><div class="caption"><p>Unrooted phylogenetic tree of bacterial luciferase superfamily members, flavin&#x02010;linked oxidoreductase superfamily members and Mfr members. The bootstrap values are indicated by a color scheme. The central part of the tree does not allow final conclusions about the basal relationship between the tree superfamilies.</p></div></div></div><p class="p p-last">The Mer cluster, placed next to bacterial luciferases and F<sub>420</sub>&#x02010;dependent dehydrogenases (Mascotti et al.,&#x000a0;<sup><a href="#pro5018-bib-0031" rid="pro5018-bib-0031" class=" bibr popnode">2021</a></sup>), the MTHFR cluster, consisting of prokaryotic and eukaryotic FAD&#x02010;dependent representatives, and the mycobacterial Mfr cluster are isolated in the tree with no reliable connection between each other and to any other clusters. Moreover, the bottom of the bacterial luciferase family in the phylogenetic tree is not occupied with a Mer&#x02010;like methylene&#x02010;H<sub>4</sub>MPT reductase indicating that the ancestor of Mer does not reduce methylene&#x02010;H<sub>4</sub>MPT. A different function is also plausible for the last universal MTHFR ancestor, as on one hand its hypothetical flavin dependency was developed at the branch bottom suggested by the close clustering of MTHFRs widespread in all domains of life, and on the other by the parallel worthlessness of the prosthetic group for MTHFR for catalysis, because simpler Mfr without flavin can execute the same reaction. Therefore, FAD in MTHFR may only serve as an evolutionary relict. Obscured by the lack of sequence and biochemical data, the same might be true for Mfr. In addition, a global phylogenetic analysis of the whole metabolism of C<sub>1</sub>&#x02010;unit reduction results in an unrelated biosynthesis of H<sub>4</sub>MPT and H<sub>4</sub>F (de Crecy&#x02010;Lagard et al.,&#x000a0;<sup><a href="#pro5018-bib-0007" rid="pro5018-bib-0007" class=" bibr popnode">2012</a></sup>; Maden,&#x000a0;<sup><a href="#pro5018-bib-0029" rid="pro5018-bib-0029" class=" bibr popnode">2000</a></sup>; Sousa &#x00026; Martin,&#x000a0;<sup><a href="#pro5018-bib-0050" rid="pro5018-bib-0050" class=" bibr popnode">2014</a></sup>) and in non&#x02010;homologous enzymes catalyzing the reduction series for C<sub>1</sub> units bound to H<sub>4</sub>F and H<sub>4</sub>MPT (Martin &#x00026; Russell,&#x000a0;<sup><a href="#pro5018-bib-0030" rid="pro5018-bib-0030" class=" bibr popnode">2007</a></sup>; Sousa &#x00026; Martin,&#x000a0;<sup><a href="#pro5018-bib-0050" rid="pro5018-bib-0050" class=" bibr popnode">2014</a></sup>). These factors would altogether support an independent development of the three enzymes, at least, of Mer with respect to the others. It has to be kept in mind that the separation of the MTHFR and Mfr clusters in the tree may result from missing links due to the absence of data towards the bacterial luciferase superfamily, as the sequences of Mfr are very limited and no Mfr candidates have been investigated outside the order <em>Mycobacteriales</em>.</p></div><div id="pro5018-sec-0009" class="sec sec-last"><h3 id="pro5018-sec-0009title">2.7. Divergent versus convergent evolution</h3><p class="p p-first">The presented results indicated common and distinct features in the three methylene&#x02010;tetrahydropterin reductases for catalyzing the related hydride transfer reactions. This finding raises the question whether these three methylene&#x02010;tetrahydropterin reductases evolved from one common ancestor or converge from a different origin. From the mechanistic point of view both Mer and Mfr have in common a one&#x02010;step hydride transfer process suggesting a closer phylogenetic relationship. On the other hand, structural alignment clearly showed a stronger connection between MTHFR and Mfr.</p><p>In the literature, a divergent development of TIM barrels is tendentially favored as remote sequence relationships are detectable among most distinct superfamilies (Nagano et al.,&#x000a0;<sup><a href="#pro5018-bib-0038" rid="pro5018-bib-0038" class=" bibr popnode">2002</a></sup>). For the TIM barrel proteins Mer, Mfr and MTHFR the related bulky substrates arranged to each other in an analogous manner and the shared hydride transfer reaction also point in this direction. Furthermore, incomplete sequence data, similar kinetic effects of equivalent mutations in hMfr and eMTHFR rather than in jMer and hMfr or in jMer and eMTHFR and several structural features support the possibility that, at least, Mfr was developed from MTHFR by drastic divergence as found in the evolution of (&#x003b2;&#x003b1;)<sub>8</sub>&#x02010;barrels (Gerlt &#x00026; Babbitt,&#x000a0;<sup><a href="#pro5018-bib-0012" rid="pro5018-bib-0012" class=" bibr popnode">1998</a></sup>; Romero&#x02010;Romero et al.,&#x000a0;<sup><a href="#pro5018-bib-0044" rid="pro5018-bib-0044" class=" bibr popnode">2021</a></sup>; Sterner &#x00026; Hocker,&#x000a0;<sup><a href="#pro5018-bib-0051" rid="pro5018-bib-0051" class=" bibr popnode">2005</a></sup>). In this case, the flavin of MTHFR would be replaced by NADH in Mfr of mycobacteria later on.</p><p class="p p-last">On the other hand, various phylogenetic, mutational/kinetic and structural data rather argue for an independent origin due to the unrelated sequences and the largely different structures of segments directly involved in forming the substrate binding and active sites. This finding is striking, as the similar substrates and the equivalent chemical reactions with the same rate&#x02010;limiting transition state should not require such drastic structural changes between the three reductases. An independent origin is also taken into accounts, because TIM barrel folds are presumably manifold formed during evolution from smaller (&#x003b1;&#x003b2;)<sub>2</sub>, and (&#x003b1;&#x003b2;)<sub>4</sub> building blocks by gene duplication and domain fusion (Lang et al.,&#x000a0;<sup><a href="#pro5018-bib-0022" rid="pro5018-bib-0022" class=" bibr popnode">2000</a></sup>). The correlating and non&#x02010;correlated kinetic data of the site&#x02010;specific variants of the shared glutamate as a key catalytic residue at position A and of the residues at positions B&#x02010;D, respectively, are compatible with a convergent and divergent evolution. Future biochemical characterizations of ancestral flavin&#x02010;dependent MTHFR, acetogenic FMN&#x02010;dependent MTHFRs and, in particular, of ancestral Mfr functional homologs may clarify the evolutionary relationship between Mer, Mfr, and MTHFR.</p></div></div><div id="pro5018-sec-0010" class="tsec sec"><h2 class="head no_bottom_margin" id="pro5018-sec-0010title">3. MATERIALS AND METHODS</h2><div id="pro5018-sec-0011" class="sec sec-first"><h3 id="pro5018-sec-0011title">3.1. Purification of <span style="fixed-case">H<sub>4</sub>MPT</span> and <span style="fixed-case">F<sub>420</sub>
</span>
</h3><p class="p p-first-last">Approximately 130&#x02009;g of <em>M. marburgensis</em> cells cultured under the standard nickel&#x02010;sufficient conditions were used to purify H<sub>4</sub>MPT as previously described (Shima et al.,&#x000a0;<sup><a href="#pro5018-bib-0048" rid="pro5018-bib-0048" class=" bibr popnode">2011</a></sup>). For the purification of F<sub>420</sub>, three cell pellets resulting from the H<sub>4</sub>MPT purification were combined and diluted 1:4 in anaerobic water. The suspension was sonicated using a SONOPLUS GM200 (Bandelin) with a VS&#x02010;70&#x02010;T tip attached at 80% power of 100&#x02009;W for the whole period of 15&#x02009;min. The suspension was mixed 1:1 with 100% acetone (&#x02212;20&#x000b0;C) and stirred in an ice bath for 30&#x02009;min. The mixture was centrifuged on at 13000&#x02009;&#x000d7;&#x02009;<em>g</em> and 4&#x000b0;C for 20&#x02009;min. The supernatant was collected and the extraction was repeated twice by adding 50% (v/v) aqueous acetone cooled to &#x02212;20&#x000b0;C in a 1:1 ratio regarding the weight of cell debris. The combined supernatants were evaporated at 4&#x000b0;C until the volume has decreased by at least half. The evaporated solution was centrifuged as described above and applied to a QAE Sephadex A25 column (GE Healthcare) equilibrated with 500&#x02009;mL of 50&#x02009;mM Tris/HCl pH&#x02009;7.5. The column was washed with 500&#x02009;mL 50&#x02009;mM Tris/HCl pH&#x02009;7.5 and then with 500&#x02009;mL 300&#x02009;mM formic acid in water. Elution was performed with a single step gradient of 500&#x02009;mL 50&#x02009;mM HCl in water. The F<sub>420</sub> containing fractions were combined and concentrated by evaporation. The concentrate was desalted using a Sephadex G&#x02010;10 column (Cytiva Life Sciences) equilibrated with water. The F<sub>420</sub> solution was stored in aliquots at &#x02212;20&#x000b0;C.</p></div><div id="pro5018-sec-0012" class="sec"><h3 id="pro5018-sec-0012title">3.2. Mutagenesis and heterologous overproduction of <span style="fixed-case">jMer</span>
</h3><p class="p p-first">The vector pT7&#x02010;7_jMer containing the jMer encoding gene <em>MJ1534</em> was used for targeted mutagenesis. The degenerated primers for mutagenesis were designed using NEBaseChanger. A PCR was performed using 10&#x02009;ng pT7&#x02010;7_jMer as template, 0.5&#x02009;&#x003bc;M degenerate primers (Table&#x000a0;<a href="#pro5018-supitem-0001" rid="pro5018-supitem-0001" class=" supplementary-material">S4</a>), 1&#x000d7; Q5 reaction buffer (New England Biolabs), 200&#x02009;&#x003bc;M dNTPs (Thermo Fisher Scientific) and 0.02&#x02009;U/&#x003bc;L Q5 High&#x02010;Fidelity DNA Polymerase (New England Biolabs) in a 50&#x02009;&#x003bc;L reaction volume. Thermocycling conditions were selected according to the manufacturer's recommendations and the annealing temperature was used as recommended by the NEBaseChanger (Table&#x000a0;<a href="#pro5018-supitem-0001" rid="pro5018-supitem-0001" class=" supplementary-material">S4</a>). After PCR, the template DNA was digested with DPNI at 37&#x000b0;C for 1&#x02009;h. The preparation was purified using the NucleoSpin Gel and PCR Clean&#x02010;up Kit (Macherey&#x02010;Nagel). The DNA preparation was used for transformation into chemically competent <em>E. coli</em> Top10 cells and the cell suspension was plated on agar plates containing 100&#x02009;&#x003bc;g/mL carbenicillin. After colony formation, 5&#x02009;mL of LB medium supplemented with 100&#x02009;&#x003bc;g/mL carbenicillin was inoculated with one colony and grown overnight at 37&#x000b0;C. The plasmids were isolated using the NucleoSpin Plasmid Kit (Macherey&#x02010;Nagel) and sequenced by Eurofins using pT7&#x02010;7_Seq_F and pT7&#x02010;7_Seq_R as sequencing primers (Table&#x000a0;<a href="#pro5018-supitem-0001" rid="pro5018-supitem-0001" class=" supplementary-material">S4</a>). The correct plasmids were transformed into ArcticExpress (DE3) cells and plated on agar plates containing 100&#x02009;&#x003bc;g/mL carbenicillin and 20&#x02009;&#x003bc;g/mL gentamicin. One colony was inoculated into 5&#x02009;mL of LB medium and incubated overnight at 37&#x000b0;C. A cryo&#x02010;culture was prepared by mixing 1&#x02009;mL of 50% glycerol with 1&#x02009;mL of the overnight culture and flash frozen. Cryo&#x02010;cultures were stored at &#x02212;75&#x000b0;C.</p><p class="p p-last">A cryo&#x02010;culture of <em>E. coli</em> ArcticExpress(DE3) containing the desired jMer variant was used to inoculate 100&#x02009;mL of LB medium supplemented with 100&#x02009;&#x003bc;g/mL carbenicillin and 20&#x02009;&#x003bc;g/mL gentamicin. The pre&#x02010;culture was incubated overnight at 37&#x000b0;C with shaking at 120&#x02009;rpm. The main culture contained 2 liters of pre&#x02010;warmed TB medium supplemented with 100&#x02009;&#x003bc;g/mL carbenicillin and 20&#x02009;&#x003bc;g/mL gentamicin and was inoculated with 100&#x02009;mL of the pre&#x02010;culture. The main culture was incubated at 37&#x000b0;C with stirring at 600&#x02009;rpm until an optical density of 0.6&#x02013;0.8 was reached. The <em>mer</em> gene expression was induced with 1&#x02009;mM IPTG and the culture was transferred to 21&#x000b0;C. After 21&#x02009;h of expression, the culture was harvested by centrifugation at 13,000&#x02009;&#x000d7;&#x02009;<em>g</em> for 5&#x02009;min at 4&#x000b0;C. Cells were snap frozen and stored at &#x02212;20&#x000b0;C.</p></div><div id="pro5018-sec-0013" class="sec"><h3 id="pro5018-sec-0013title">3.3. Purification of <span style="fixed-case">jMer</span>
</h3><p class="p p-first">For crystallization, approximately 40&#x02009;g of the <em>E. coli</em> cells were suspended in 160&#x02009;mL of 50&#x02009;mM Tris/HCl pH&#x02009;7.5 with 2&#x02009;mM DTT. The cell suspension was sonicated using a SONOPULS GM200 (Bandelin) with a 50% cycle and 160&#x02009;W for 5&#x02009;min per cycle and 5&#x02009;min pause using a TZ76 tip. A total of 2&#x02009;cycles were performed. The disrupted cells were fractionated by centrifugation at 30,000&#x02009;&#x000d7;&#x02009;<em>g</em> for 30&#x02009;min at 4&#x000b0;C. The supernatant was heated at 80&#x000b0;C for 20&#x02009;min and precipitated proteins were separated by centrifugation at 13,000&#x02009;&#x000d7;&#x02009;<em>g</em> for 20&#x02009;min at 4&#x000b0;C. Ammonium sulfate was added to the supernatant to 60% saturation and the solution was stirred at 4&#x000b0;C for 20&#x02009;min. Precipitated proteins were removed by centrifugation at 13,000&#x02009;&#x000d7;&#x02009;<em>g</em> for 20&#x02009;min at 4&#x000b0;C. The supernatant was applied to a Phenyl&#x02010;Sepharose HP column (15&#x02009;mL column volume) equilibrated with 50&#x02009;mM Tris/HCl pH&#x02009;7.5 containing 2&#x02009;mM DTT and 2&#x02009;M ammonium sulfate (buffer A). Buffer B contained 50&#x02009;mM Tris/HCl pH&#x02009;7.5 with 2&#x02009;mM DTT and 10% (v/v) glycerol. The column was washed with 20% buffer B. Elution was performed with a linear gradient from 20% to 100% buffer B in eight column volumes. jMer was eluted from 137 to 5&#x02009;mS/cm conductivity. The corresponding fractions were collected and desalted on a HiPrep G&#x02010;25 column equilibrated with 50&#x02009;mM Tris/HCl pH&#x02009;7.5 containing 2&#x02009;mM DTT. The desalted solution was applied to a Resource Q column (6&#x02009;mL column volume) equilibrated with 50&#x02009;mM Tris/HCl pH&#x02009;7.5 with 2&#x02009;mM DTT. jMer was eluted by a linear gradient of 0&#x02013;250&#x02009;mM NaCl over 15 column volumes. The jMer&#x02010;containing fractions were collected and applied to a HiPrep Sephacryl S&#x02010;200 HR equilibrated with 50&#x02009;mM Tris/HCl pH&#x02009;7.5 containing 150&#x02009;mM NaCl and 2&#x02009;mM DTT. The jMer&#x02010;containing fractions were either used directly for crystallization or, after the addition of 5% (v/v) glycerol, were snap frozen in liquid nitrogen and stored at &#x02212;75&#x000b0;C.</p><p class="p p-last">A shorter protocol was developed for the characterization of jMer variants. Approximately 10&#x02009;g of cells were suspended in 40&#x02009;mL of 50&#x02009;mM Tris/HCl pH&#x02009;7.5 containing 2&#x02009;mM DTT. The cell suspension was sonicated with a TZ73 tip attached to a SONOPULS GM200 (Bandelin) with a 50% cycle and 160&#x02009;W for 5&#x02009;min per cycle and 5&#x02009;min pause. A total of 2&#x02009;cycles were performed. The disrupted cells were fractionated by centrifugation at 30,000&#x02009;&#x000d7;&#x02009;<em>g</em> for 30&#x02009;min at 4&#x000b0;C. The supernatant was heated at 80&#x000b0;C for 20&#x02009;min and precipitated proteins were separated by centrifugation at 13,000&#x02009;&#x000d7;&#x02009;<em>g</em> for 20&#x02009;min at 4&#x000b0;C. The supernatant was diluted 1:1 in 50&#x02009;mM Tris/HCl pH&#x02009;7.5 with 2&#x02009;mM DTT and applied directly to a ResourceQ column (6&#x02009;mL column volume) equilibrated with 50&#x02009;mM Tris/HCl pH&#x02009;7.5 with 2&#x02009;mM DTT. jMer was eluted by a linear gradient of 0&#x02013;250&#x02009;mM NaCl over 15 column volumes. The jMer&#x02010;containing fractions were collected and used for the kinetic characterization.</p></div><div id="pro5018-sec-0014" class="sec"><h3 id="pro5018-sec-0014title">3.4. Activity assay of <span style="fixed-case">jMer</span>
</h3><p class="p p-first-last">Master mixes for the activity assays were prepared in an anaerobic chamber. Brown serum bottles were filled with 100&#x02009;mM Tris/HCl pH&#x02009;8.0 supplemented with 10&#x02009;&#x003bc;M 2&#x02010;mercaptoethanol, the desired amount of purified F<sub>420</sub> and H<sub>4</sub>MPT, and 3&#x02009;mM sodium dithionite. The master mixes were incubated for 15&#x02009;min at 55&#x000b0;C in a water bath, in which F<sub>420</sub> was reduced to F<sub>420</sub>H<sub>2</sub>. The enzyme assay was performed in an anaerobic quartz cuvette (1&#x02009;cm light path) with a final volume of 200&#x02009;&#x003bc;L. After preheating at 55&#x000b0;C for 5&#x02009;min, 15&#x02009;mM formaldehyde (final concentration) was added, by which methylene&#x02010;H<sub>4</sub>MPT was generated from H<sub>4</sub>MPT and residual dithionite was quenched. Under the standard assay condition, the reaction mixture contained 20&#x02009;&#x003bc;M methylene&#x02010;H<sub>4</sub>MPT and 20&#x02009;&#x003bc;M F<sub>420</sub>H<sub>2</sub> at 55&#x000b0;C. In the case of Michaelis&#x02013;Menten kinetic analysis, one of the substrate concentration was fixed as the standard condition. The enzyme reaction was started by the addition of 10&#x02009;&#x003bc;L of enzyme solution. The reaction was monitored by measuring the increase in absorbance at 401&#x02009;nm. The catalytic activity was calculated from the extinction coefficient of F<sub>420</sub> (<em>&#x003b5;</em>
<sub>401</sub>&#x02009;=&#x02009;25.9&#x02009;mM<sup>&#x02212;1</sup>&#x02009;cm<sup>&#x02212;1</sup>).</p></div><div id="pro5018-sec-0015" class="sec"><h3 id="pro5018-sec-0015title">3.5. Crystallization and structure determination of <span style="fixed-case">jMer</span>
</h3><p class="p p-first">All crystallization experiments were carried out in an anaerobic chamber with a 95%/5% (N<sub>2</sub>/H<sub>2</sub>) atmosphere using the sitting drop vapor diffusion method and 96&#x02010;well two&#x02010;drop MRC crystallization plates (Molecular Dimensions). The plates were incubated for 1&#x02009;week in the chamber before use. The final protein concentration in each drop was 20&#x02009;mg/mL. For the drops containing F<sub>420</sub>, a final F<sub>420</sub> concentration of 2&#x02009;mM was used. The crystal of the apoenzyme grew in a drop consisting of 35% (v/v) 2&#x02010;methyl&#x02010;2,4&#x02010;pentanediol and 100&#x02009;mM sodium acetate pH&#x02009;4.5 and could be frozen directly in liquid nitrogen. The best crystal of the binary complex grew in a drop consisting of 25% (v/v) polyethylene glycol monomethyl ether 550, 100&#x02009;mM MES pH&#x02009;6.5 and 10&#x02009;mM zinc sulfate. Prior to freezing, the crystal was treated with a cryoprotectant solution consisting of the reservoir solution mixed with 20% polyethylene glycol monomethyl ether 550 and F<sub>420</sub> to a final concentration of 2&#x02009;mM. A large number of experiments were also carried out using F<sub>420</sub> in combination with either methylene&#x02010;H<sub>4</sub>MPT or methyl&#x02010;H<sub>4</sub>MPT at concentrations ranging from 2 to 10&#x02009;mM substrate concentration in the droplets.</p><p class="p p-last">The diffraction experiments were performed at 100&#x02009;K on the SLS beamline X10SA (Villigen, Switzerland) equipped with a Dectris Eiger2 16M detector. The data set was processed with XDS and scaled with XSCALE (Kabsch,&#x000a0;<sup><a href="#pro5018-bib-0018" rid="pro5018-bib-0018" class=" bibr popnode">2010</a></sup>). The phase problem was solved by the molecular replacement method using PHASER (McCoy et al.,&#x000a0;<sup><a href="#pro5018-bib-0032" rid="pro5018-bib-0032" class=" bibr popnode">2007</a></sup>) with the structure of Mer from <em>Methanopyrus kandleri</em> as a search model (Shima et al.,&#x000a0;<sup><a href="#pro5018-bib-0049" rid="pro5018-bib-0049" class=" bibr popnode">2000</a></sup>). The model was built and improved in COOT (Emsley et al.,&#x000a0;<sup><a href="#pro5018-bib-0009" rid="pro5018-bib-0009" class=" bibr popnode">2010</a></sup>) and refined using Phenix.refine (Liebschner et al.,&#x000a0;<sup><a href="#pro5018-bib-0025" rid="pro5018-bib-0025" class=" bibr popnode">2019</a></sup>) and Refmac (Murshudov et al.,&#x000a0;<sup><a href="#pro5018-bib-0037" rid="pro5018-bib-0037" class=" bibr popnode">2011</a></sup>). The final model was validated using the MolProbity (Williams et al.,&#x000a0;<sup><a href="#pro5018-bib-0061" rid="pro5018-bib-0061" class=" bibr popnode">2018</a></sup>) implementation of Phenix (Liebschner et al.,&#x000a0;<sup><a href="#pro5018-bib-0025" rid="pro5018-bib-0025" class=" bibr popnode">2019</a></sup>). Data collection, refinement statistics and PDB code for the deposited structure are listed in Tables&#x000a0;<a href="/pmc/articles/PMC11094777/table/pro5018-tbl-0002/" target="table" class="fig-table-link figpopup" rid-figpopup="pro5018-tbl-0002" rid-ob="ob-pro5018-tbl-0002" co-legend-rid=""><span>2</span></a> and <a href="#pro5018-supitem-0001" rid="pro5018-supitem-0001" class=" supplementary-material">S1</a>.</p></div><div id="pro5018-sec-0016" class="sec"><h3 id="pro5018-sec-0016title">3.6. Mutagenesis, expression, purification and activity assay of <span style="fixed-case">hMfr</span>
</h3><p class="p p-first-last">hMfr variants were generated by GenScript. The expression, purification and activity assay procedure was performed as previously described (Gehl et al.,&#x000a0;<sup><a href="#pro5018-bib-0011" rid="pro5018-bib-0011" class=" bibr popnode">2023</a></sup>).</p></div><div id="pro5018-sec-0017" class="sec"><h3 id="pro5018-sec-0017title">3.7. Kinetic data processing and figure generation</h3><p class="p p-first-last">The graphs and analyses of the kinetic constants were carried out using Python 3.7 with Jupyter Notebook (version 6.1.4) (Kluyver et al.,&#x000a0;<sup><a href="#pro5018-bib-0020" rid="pro5018-bib-0020" class=" bibr popnode">2016</a></sup>) as the development environment and the following packages: os, pandas (McKinney &#x00026; Data structures for statistical computing in python,&#x000a0;<sup><a href="#pro5018-bib-0033" rid="pro5018-bib-0033" class=" bibr popnode">2010</a></sup>), seaborn (Waskom,&#x000a0;<sup><a href="#pro5018-bib-0059" rid="pro5018-bib-0059" class=" bibr popnode">2021</a></sup>), matplotlib (Hunter,&#x000a0;<sup><a href="#pro5018-bib-0016" rid="pro5018-bib-0016" class=" bibr popnode">2007</a></sup>), NumPy (Harris et al.,&#x000a0;<sup><a href="#pro5018-bib-0014" rid="pro5018-bib-0014" class=" bibr popnode">2020</a></sup>), SciPy (Virtanen et al.,&#x000a0;<sup><a href="#pro5018-bib-0058" rid="pro5018-bib-0058" class=" bibr popnode">2020</a></sup>). The code can be found at GitHub (<a href="https://github.com/ManuelGehl/Enzyme-kinetic-fitting" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/ManuelGehl/Enzyme-kinetic-fitting</a>). The chemical structures were created using ChemSketch and edited using Adobe Illustrator.</p></div><div id="pro5018-sec-0018" class="sec sec-last"><h3 id="pro5018-sec-0018title">3.8. Phylogenetic tree construction and structure alignment</h3><p class="p p-first-last">Seed sequences from the bacterial luciferase family (Table&#x000a0;<a href="#pro5018-supitem-0001" rid="pro5018-supitem-0001" class=" supplementary-material">S2</a>) and the FAD&#x02010;linked reductase superfamily (Table&#x000a0;<a href="#pro5018-supitem-0001" rid="pro5018-supitem-0001" class=" supplementary-material">S3</a>) together with the amino acid sequence of hMfr were used for separate BLAST searches against the clustered non&#x02010;redundant protein sequence database. The results were filtered to exclude any cluster with more than 90% sequence identity, and all clusters containing at least three members were selected. The query sequences were reinserted into the dataset and sequences marked as partial were removed, resulting in 500 sequences. A multiple sequence alignment was performed using MUSCLE (Edgar,&#x000a0;<sup><a href="#pro5018-bib-0008" rid="pro5018-bib-0008" class=" bibr popnode">2004</a></sup>) and a maximum likelihood tree was constructed using IQTree (Minh et al.,&#x000a0;<sup><a href="#pro5018-bib-0035" rid="pro5018-bib-0035" class=" bibr popnode">2020</a></sup>). The best fitting evolutionary model was found to be WAG+F&#x02009;+&#x02009;I&#x02009;+&#x02009;G4 and the ultra&#x02010;fast bootstrap method (Hoang et al.,&#x000a0;<sup><a href="#pro5018-bib-0015" rid="pro5018-bib-0015" class=" bibr popnode">2018</a></sup>) was used to incorporate bootstrap values. The tree was visualized using iTOL (Letunic &#x00026; Bork,&#x000a0;<sup><a href="#pro5018-bib-0024" rid="pro5018-bib-0024" class=" bibr popnode">2021</a></sup>). Isolated branches were manually removed, resulting in 483 sequences in the tree. In addition, structure alignment was performed using PROMALS3D (Pei et al.,&#x000a0;<sup><a href="#pro5018-bib-0041" rid="pro5018-bib-0041" class=" bibr popnode">2008</a></sup>) as a tool and the crystal structures of hMfr, jMer and eMTHFR.</p></div></div><div id="pro5018-sec-0024" class="tsec sec"><h2 class="head no_bottom_margin" id="pro5018-sec-0024title">AUTHOR CONTRIBUTIONS</h2><p class="p p-first-last">
<strong>Manuel Gehl:</strong> Conceptualization; software; data curation; formal analysis; investigation; methodology; validation; writing &#x02013; original draft; writing &#x02013; review and editing. <strong>Ulrike Demmer:</strong> Data curation; investigation. <strong>Ulrich Ermler:</strong> Conceptualization; funding acquisition; project administration; resources; data curation; formal analysis; investigation; methodology; software; validation; writing &#x02013; review and editing. <strong>Seigo Shima:</strong> Conceptualization; funding acquisition; project administration; resources; supervision; writing &#x02013; review and editing; writing &#x02013; original draft.</p></div><div id="pro5018-sec-0020" class="tsec sec"><h2 class="head no_bottom_margin" id="pro5018-sec-0020title">FUNDING INFORMATION</h2><p class="p p-first-last">This work was supported by Max Planck Society (Ulrich Ermler and Seigo Shima) and by Deutsche Forschungsgemeinschaft, Priority Program, Iron&#x02013;Sulfur for Life (SPP1927, SH87/1&#x02010;2) (Seigo Shima).</p></div><div id="pro5018-sec-0021" class="tsec sec"><h2 class="head no_bottom_margin" id="pro5018-sec-0021title">CONFLICT OF INTEREST STATEMENT</h2><p class="p p-first-last">The authors declare no competing financial interest.</p></div><div id="sec-a.x.g" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="sec-a.x.gtitle">Supporting information</h2><!--/article/body/sec/--><div class="sec suppmat" id="pro5018-supitem-0001"><p>
<strong>TABLE S1.</strong> Structure determination statistics for jMer_E6Q and Mfr_E9Q.</p><p>
<strong>TABLE S2.</strong> Seed sequences for the superfamily of bacterial luciferases.</p><p>
<strong>TABLE S3.</strong> Seed sequences for the superfamily of FAD&#x02010;linked reductases.</p><p>
<strong>TABLE S4.</strong> List of primers used for mutagenesis of jMer and sequencing of the ORF of jMer in pT7&#x02010;7_jMer. The degenerated nucleotides are marked in red.</p><p>
<strong>FIGURE S2.</strong> Size&#x02010;exclusion chromatography of jMer using a HiPrep Sephacryl S&#x02010;200 HR column. The peak of jMer is centered at around 50&#x02009;mL elution volume and corresponds to a molecular mass of approximately 80&#x02009;kDa.</p><div class="sup-box half_rhythm" id="media-a.x.g.b.b"><a href="/pmc/articles/PMC11094777/bin/PRO-33-e5018-s001.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(9.6M, docx)</sup></div></div><div class="sec suppmat" id="pro5018-supitem-0002"><p>
<strong>FIGURE S1:</strong> Structures of H<sub>4</sub>MPT (top) and H<sub>4</sub>F (bottom). The pterin part is colored blue. The p&#x02010;aminobenzoate (PABA) ring of H<sub>4</sub>F and the aniline ring of H<sub>4</sub>MPT are colored red. The tail regions are colored black.</p><div class="sup-box half_rhythm" id="media-a.x.g.c.b"><a href="/pmc/articles/PMC11094777/bin/PRO-33-e5018-s005.jpg" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(806K, jpg)</sup></div></div><div class="sec suppmat" id="pro5018-supitem-0003"><p>
<strong>FIGURE S3:</strong> Dimeric structure of jMer. The homodimer is the physiological form of jMer and is formed by a two&#x02010;fold rotational axis at the center of the protein&#x02010;protein interface. The active site is located at the C&#x02010;terminal end of the parallel &#x003b2;&#x02010;strands, which are positioned on the opposite site of the two monomers.</p><div class="sup-box half_rhythm" id="media-a.x.g.d.b"><a href="/pmc/articles/PMC11094777/bin/PRO-33-e5018-s003.jpg" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.4M, jpg)</sup></div></div><div class="sec suppmat" id="pro5018-supitem-0004"><p>
<strong>FIGURE S4:</strong> Comparison of the tertiary structures of eMTHFR, jMer and hMfr. The &#x003b2;&#x02010;strands of the core unit are labeled and colored purple, while the &#x003b1;&#x02010;helices of the core unit and the loops are colored in light blue and salmon. The inserted helical segment is painted in orange. Methyl&#x02010;H<sub>4</sub>F and FAD are shown in orange and F<sub>420</sub> is shown in green.</p><div class="sup-box half_rhythm" id="media-a.x.g.e.b"><a href="/pmc/articles/PMC11094777/bin/PRO-33-e5018-s002.jpg" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(2.0M, jpg)</sup></div></div><div class="sec suppmat" id="pro5018-supitem-0005"><p>
<strong>FIGURE S5:</strong> Hydrophobic pocked in jMer. The modeled H<sub>4</sub>F is colored yellow while the native F<sub>420</sub> and the amino acids are shown in light blue.</p><div class="sup-box half_rhythm" id="media-a.x.g.f.b"><a href="/pmc/articles/PMC11094777/bin/PRO-33-e5018-s007.jpg" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(788K, jpg)</sup></div></div><div class="sec suppmat" id="pro5018-supitem-0006"><p>
<strong>FIGURE S6:</strong> Structure&#x02010;based alignment of hMfr, eMTHFR and jMer.</p><div class="sup-box half_rhythm" id="media-a.x.g.g.b"><a href="/pmc/articles/PMC11094777/bin/PRO-33-e5018-s008.jpg" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(172K, jpg)</sup></div></div><div class="sec suppmat" id="pro5018-supitem-0007"><p>
<strong>FIGURE S7:</strong> NADH binding site of eMTHFR.</p><div class="sup-box half_rhythm" id="media-a.x.g.h.b"><a href="/pmc/articles/PMC11094777/bin/PRO-33-e5018-s006.jpg" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(648K, jpg)</sup></div></div><div class="sec suppmat" id="pro5018-supitem-0008"><p>
<strong>FIGURE S8:</strong> (a) Comparison of the structures of jMer wild type (green) and jMer_E6Q (light blue). (b) Comparison of the structures of hMfr wild type (dark blue) and hMfr_E9Q (yellow). The glutamate residues are depicted as ball&#x02010;and&#x02010;stick model.</p><div class="sup-box half_rhythm" id="media-a.x.g.i.b"><a href="/pmc/articles/PMC11094777/bin/PRO-33-e5018-s004.jpg" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.2M, jpg)</sup></div></div></div><div id="pro5018-sec-0019" class="tsec sec"><h2 class="head no_bottom_margin" id="pro5018-sec-0019title">ACKNOWLEDGMENTS</h2><div class="sec"><p>The authors thank Robert H. White for providing us the plasmid for the production of jMer. M.G. thanks the International Max Planck Research School thesis&#x02010;committee Members: Tobias Erb and Rolf Thauer. Open Access funding enabled and organized by Projekt DEAL.</p></div></div><div id="notes-a.w.c" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.w.ctitle">Notes</h2><p>
<span class="mixed-citation" id="pro5018-cit-9001">
Gehl M, Demmer U, Ermler U, Shima S. <span class="ref-title">Mutational and structural studies of (&#x003b2;&#x003b1;)<sub>8</sub>&#x02010;barrel fold methylene&#x02010;tetrahydropterin reductases utilizing a common catalytic mechanism</span>. <span class="ref-journal">Protein Science</span>. 2024;<span class="ref-vol">33</span>(<span class="ref-iss">6</span>):e5018. 10.1002/pro.5018
 [<a href="//doi.org/10.1002%2Fpro.5018" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Science&amp;title=Mutational+and+structural+studies+of+(&#x003b2;&#x003b1;)8&#x02010;barrel+fold+methylene&#x02010;tetrahydropterin+reductases+utilizing+a+common+catalytic+mechanism&amp;volume=33&amp;issue=6&amp;publication_year=2024&amp;pages=e5018&amp;doi=10.1002/pro.5018&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
</p><p class="fn-group"><p class="fn sec" id="pro5018-note-0001">
<strong>Reviewing Editor:</strong> Lynn Kamerlin</p></p></div><div id="pro5018-sec-0023" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="pro5018-sec-0023title">DATA AVAILABILITY STATEMENT</h2><!--/article/back/sec/--><p class="p p-first-last">Data available in article supplementary material.</p></div><div id="pro5018-bibl-0001" class="tsec sec"><h2 class="head no_bottom_margin" id="pro5018-bibl-0001title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="pro5018-bib-0001"><span class="mixed-citation" id="pro5018-cit-0001">
Aufhammer SW, Warkentin E, Berk H, Shima S, Thauer RK, Ermler U. <span class="ref-title">Coenzyme binding in F<sub>420</sub>&#x02010;dependent secondary alcohol dehydrogenase, a member of the bacterial luciferase family</span>. <span class="ref-journal">Structure</span>. 2004;<span class="ref-vol">12</span>:361&#x02013;370.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/15016352" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=Coenzyme+binding+in+F420&#x02010;dependent+secondary+alcohol+dehydrogenase,+a+member+of+the+bacterial+luciferase+family&amp;volume=12&amp;publication_year=2004&amp;pages=361-370&amp;pmid=15016352&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0002"><span class="mixed-citation" id="pro5018-cit-0002">
Aufhammer SW, Warkentin E, Ermler U, Hagemeier CH, Thauer RK, Shima S. <span class="ref-title">Crystal structure of methylenetetrahydromethanopterin reductase (Mer) in complex with coenzyme F<sub>420</sub>: architecture of the F<sub>420</sub>/FMN binding site of enzymes within the nonprolyl cis&#x02010;peptide containing bacterial luciferase family</span>. <span class="ref-journal">Protein Sci</span>. 2005;<span class="ref-vol">14</span>:1840&#x02013;1849.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2253363/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15937276" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Sci&amp;title=Crystal+structure+of+methylenetetrahydromethanopterin+reductase+(Mer)+in+complex+with+coenzyme+F420:+architecture+of+the+F420/FMN+binding+site+of+enzymes+within+the+nonprolyl+cis&#x02010;peptide+containing+bacterial+luciferase+family&amp;volume=14&amp;publication_year=2005&amp;pages=1840-1849&amp;pmid=15937276&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0003"><span class="mixed-citation" id="pro5018-cit-0003">
Bashiri G, Squire CJ, Moreland NJ, Baker EN. <span class="ref-title">Crystal structures of F<sub>420</sub>&#x02010;dependent glucose&#x02010;6&#x02010;phosphate dehydrogenase FGD1 involved in the activation of the anti&#x02010;tuberculosis drug candidate PA&#x02010;824 reveal the basis of coenzyme and substrate binding</span>. <span class="ref-journal">J Biol Chem</span>. 2008;<span class="ref-vol">283</span>:17531&#x02013;17541.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/18434308" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Crystal+structures+of+F420&#x02010;dependent+glucose&#x02010;6&#x02010;phosphate+dehydrogenase+FGD1+involved+in+the+activation+of+the+anti&#x02010;tuberculosis+drug+candidate+PA&#x02010;824+reveal+the+basis+of+coenzyme+and+substrate+binding&amp;volume=283&amp;publication_year=2008&amp;pages=17531-17541&amp;pmid=18434308&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0004"><span class="mixed-citation" id="pro5018-cit-0004">
Bertsch J, &#x000d6;ppinger C, Hess V, Langer JD, M&#x000fc;ller V. <span class="ref-title">Heterotrimeric NADH&#x02010;oxidizing methylenetetrahydrofolate reductase from the acetogenic bacterium <em>Acetobacterium woodii</em>
</span>. <span class="ref-journal">J Bacteriol</span>. 2015;<span class="ref-vol">197</span>:1681&#x02013;1689.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4403655/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25733614" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=Heterotrimeric+NADH&#x02010;oxidizing+methylenetetrahydrofolate+reductase+from+the+acetogenic+bacterium+Acetobacterium+woodii&#10;&amp;volume=197&amp;publication_year=2015&amp;pages=1681-1689&amp;pmid=25733614&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0005"><span class="mixed-citation" id="pro5018-cit-0005">
Borrel G, Adam PS, Gribaldo S. <span class="ref-title">Methanogenesis and the Wood&#x02010;Ljungdahl pathway: an ancient, versatile, and fragile association</span>. <span class="ref-journal">Genome Biol Evol</span>. 2016;<span class="ref-vol">8</span>:1706&#x02013;1711.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4943185/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27189979" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol+Evol&amp;title=Methanogenesis+and+the+Wood&#x02010;Ljungdahl+pathway:+an+ancient,+versatile,+and+fragile+association&amp;volume=8&amp;publication_year=2016&amp;pages=1706-1711&amp;pmid=27189979&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0006"><span class="mixed-citation" id="pro5018-cit-0006">
Clark JE, Ljungdahl LG. <span class="ref-title">Purification and properties of 5,10&#x02010;methylenetetrahydrofolate reductase, an iron&#x02010;sulfur flavoprotein from <em>Clostridium formicoaceticum</em>
</span>. <span class="ref-journal">J Biol Chem</span>. 1984;<span class="ref-vol">259</span>:10845&#x02013;10849.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/6381490" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Purification+and+properties+of+5,10&#x02010;methylenetetrahydrofolate+reductase,+an+iron&#x02010;sulfur+flavoprotein+from+Clostridium+formicoaceticum&#10;&amp;volume=259&amp;publication_year=1984&amp;pages=10845-10849&amp;pmid=6381490&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0007"><span class="mixed-citation" id="pro5018-cit-0007">
de Crecy&#x02010;Lagard V, Phillips G, Grochowski LL, El Yacoubi B, Jenney F, Adams MWW, et al. <span class="ref-title">Comparative genomics guided discovery of two missing archaeal enzyme families involved in the biosynthesis of the pterin moiety of tetrahydromethanopterin and tetrahydrofolate</span>. <span class="ref-journal">ACS Chem Biol</span>. 2012;<span class="ref-vol">7</span>:1807&#x02013;1816.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3500442/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22931285" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=ACS+Chem+Biol&amp;title=Comparative+genomics+guided+discovery+of+two+missing+archaeal+enzyme+families+involved+in+the+biosynthesis+of+the+pterin+moiety+of+tetrahydromethanopterin+and+tetrahydrofolate&amp;volume=7&amp;publication_year=2012&amp;pages=1807-1816&amp;pmid=22931285&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0008"><span class="mixed-citation" id="pro5018-cit-0008">
Edgar RC. <span class="ref-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span>. <span class="ref-journal">Nucleic Acids Res</span>. 2004;<span class="ref-vol">32</span>:1792&#x02013;1797.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC390337/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15034147" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=MUSCLE:+multiple+sequence+alignment+with+high+accuracy+and+high+throughput&amp;volume=32&amp;publication_year=2004&amp;pages=1792-1797&amp;pmid=15034147&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0009"><span class="mixed-citation" id="pro5018-cit-0009">
Emsley P, Lohkamp B, Scott WG, Cowtan K. <span class="ref-title">Features and development of coot</span>. <span class="ref-journal">Acta Crystallogr D</span>. 2010;<span class="ref-vol">66</span>:486&#x02013;501.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2852313/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20383002" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Crystallogr+D&amp;title=Features+and+development+of+coot&amp;volume=66&amp;publication_year=2010&amp;pages=486-501&amp;pmid=20383002&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0010"><span class="mixed-citation" id="pro5018-cit-0010">
Farber GK, Petsko GA. <span class="ref-title">The evolution of alpha/beta barrel enzymes</span>. <span class="ref-journal">Trends Biochem Sci</span>. 1990;<span class="ref-vol">15</span>:228&#x02013;234.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/2200166" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Biochem+Sci&amp;title=The+evolution+of+alpha/beta+barrel+enzymes&amp;volume=15&amp;publication_year=1990&amp;pages=228-234&amp;pmid=2200166&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0011"><span class="mixed-citation" id="pro5018-cit-0011">
Gehl M, Demmer U, Ermler U, Shima S. <span class="ref-title">Crystal structure of FAD&#x02010;independent methylene&#x02010;tetrahydrofolate reductase from <em>Mycobacterium hassiacum</em>
</span>. <span class="ref-journal">Proteins</span>. 2023;<span class="ref-vol">91</span>:1329&#x02013;1340.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/37119125" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Crystal+structure+of+FAD&#x02010;independent+methylene&#x02010;tetrahydrofolate+reductase+from+Mycobacterium+hassiacum&#10;&amp;volume=91&amp;publication_year=2023&amp;pages=1329-1340&amp;pmid=37119125&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0012"><span class="mixed-citation" id="pro5018-cit-0012">
Gerlt JA, Babbitt PC. <span class="ref-title">Mechanistically diverse enzyme superfamilies: the importance of chemistry in the evolution of catalysis</span>. <span class="ref-journal">Curr Opin Chem Biol</span>. 1998;<span class="ref-vol">2</span>:607&#x02013;612.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/9818186" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Chem+Biol&amp;title=Mechanistically+diverse+enzyme+superfamilies:+the+importance+of+chemistry+in+the+evolution+of+catalysis&amp;volume=2&amp;publication_year=1998&amp;pages=607-612&amp;pmid=9818186&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0013"><span class="mixed-citation" id="pro5018-cit-0013">
Guenther BD, Sheppard CA, Tran P, Rozen R, Matthews RG, Ludwig ML. <span class="ref-title">The structure and properties of methylenetetrahydrofolate reductase from <em>Escherichia coli</em> suggest how folate ameliorates human hyperhomocysteinemia</span>. <span class="ref-journal">Nat Struct Biol</span>. 1999;<span class="ref-vol">6</span>:359&#x02013;365.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/10201405" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Struct+Biol&amp;title=The+structure+and+properties+of+methylenetetrahydrofolate+reductase+from+Escherichia+coli+suggest+how+folate+ameliorates+human+hyperhomocysteinemia&amp;volume=6&amp;publication_year=1999&amp;pages=359-365&amp;pmid=10201405&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0014"><span class="mixed-citation" id="pro5018-cit-0014">
Harris CR, Millman KJ, Van Der Walt SJ, Gommers R, Virtanen P, Cournapeau D, et al. <span class="ref-title">Array programming with NumPy</span>. <span class="ref-journal">Nature</span>. 2020;<span class="ref-vol">585</span>:357&#x02013;362.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7759461/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32939066" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Array+programming+with+NumPy&amp;volume=585&amp;publication_year=2020&amp;pages=357-362&amp;pmid=32939066&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0015"><span class="mixed-citation" id="pro5018-cit-0015">
Hoang DT, Chernomor O, von Haeseler A, Minh BQ, Vinh LS. <span class="ref-title">UFBoot2: improving the ultrafast bootstrap approximation</span>. <span class="ref-journal">Mol Biol Evol</span>. 2018;<span class="ref-vol">35</span>:518&#x02013;522.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5850222/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29077904" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Biol+Evol&amp;title=UFBoot2:+improving+the+ultrafast+bootstrap+approximation&amp;volume=35&amp;publication_year=2018&amp;pages=518-522&amp;pmid=29077904&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0016"><span class="mixed-citation" id="pro5018-cit-0016">
Hunter JD. <span class="ref-title">Matplotlib: a 2D graphics environment</span>. <span class="ref-journal">Comput Sci Eng</span>. 2007;<span class="ref-vol">9</span>:90&#x02013;95. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Comput+Sci+Eng&amp;title=Matplotlib:+a+2D+graphics+environment&amp;volume=9&amp;publication_year=2007&amp;pages=90-95&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0017"><span class="mixed-citation" id="pro5018-cit-0017">
Jabs A, Weiss MS, Hilgenfeld R. <span class="ref-title">Non&#x02010;proline <em>cis</em> peptide bonds in proteins</span>. <span class="ref-journal">J Mol Biol</span>. 1999;<span class="ref-vol">286</span>:291&#x02013;304.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/9931267" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Non&#x02010;proline+cis+peptide+bonds+in+proteins&amp;volume=286&amp;publication_year=1999&amp;pages=291-304&amp;pmid=9931267&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0018"><span class="mixed-citation" id="pro5018-cit-0018">
Kabsch W. <span class="ref-title">XDS</span>. <span class="ref-journal">Acta Crystallogr D</span>. 2010;<span class="ref-vol">66</span>:125&#x02013;132.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2815665/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20124692" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Crystallogr+D&amp;title=XDS&amp;volume=66&amp;publication_year=2010&amp;pages=125-132&amp;pmid=20124692&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0019"><span class="mixed-citation" id="pro5018-cit-0019">
Kallen RG, Jencks WP. <span class="ref-title">The mechanism of the condensation of formaldehyde with tetrahydrofolic acid</span>. <span class="ref-journal">J Biol Chem</span>. 1966;<span class="ref-vol">241</span>:5851&#x02013;5863.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/5954363" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=The+mechanism+of+the+condensation+of+formaldehyde+with+tetrahydrofolic+acid&amp;volume=241&amp;publication_year=1966&amp;pages=5851-5863&amp;pmid=5954363&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0020"><span class="mixed-citation" id="pro5018-cit-0020">
Kluyver T, Ragan&#x02010;Kelley B, P&#x000e9;rez F, Granger BE, Bussonnier M, Frederic J, et al. Jupyter notebooks&#x02014;a publishing format for reproducible computational workflows. In: Loizides F, Schmidt B, editors. <span class="ref-journal">Positioning and power in academic publishing: players, agents and agendas</span>. Amsterdam: IOS Press; 2016. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Positioning+and+power+in+academic+publishing:+players,+agents+and+agendas&amp;publication_year=2016&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0021"><span class="mixed-citation" id="pro5018-cit-0021">
Kurth JM, Op den Camp HJM, Welte CU. <span class="ref-title">Several ways one goal&#x02010;methanogenesis from unconventional substrates</span>. <span class="ref-journal">Appl Microbiol Biotechnol</span>. 2020;<span class="ref-vol">104</span>:6839&#x02013;6854.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7374477/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32542472" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Appl+Microbiol+Biotechnol&amp;title=Several+ways+one+goal&#x02010;methanogenesis+from+unconventional+substrates&amp;volume=104&amp;publication_year=2020&amp;pages=6839-6854&amp;pmid=32542472&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0022"><span class="mixed-citation" id="pro5018-cit-0022">
Lang D, Thoma R, Henn&#x02010;Sax M, Sterner R, Wilmanns M. <span class="ref-title">Structural evidence for evolution of the beta/alpha barrel scaffold by gene duplication and fusion</span>. <span class="ref-journal">Science</span>. 2000;<span class="ref-vol">289</span>:1546&#x02013;1550.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/10968789" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Structural+evidence+for+evolution+of+the+beta/alpha+barrel+scaffold+by+gene+duplication+and+fusion&amp;volume=289&amp;publication_year=2000&amp;pages=1546-1550&amp;pmid=10968789&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0023"><span class="mixed-citation" id="pro5018-cit-0023">
Lee MN, Takawira D, Nikolova AP, Ballou DP, Furtado VC, Phung NL, et al. <span class="ref-title">Functional role for the conformationally mobile phenylalanine 223 in the reaction of methylenetetrahydrofolate reductase from <em>Escherichia coli</em>
</span>. <span class="ref-journal">Biochemistry</span>. 2009;<span class="ref-vol">48</span>:7673&#x02013;7685.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2805098/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19610625" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Functional+role+for+the+conformationally+mobile+phenylalanine+223+in+the+reaction+of+methylenetetrahydrofolate+reductase+from+Escherichia+coli&#10;&amp;volume=48&amp;publication_year=2009&amp;pages=7673-7685&amp;pmid=19610625&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0024"><span class="mixed-citation" id="pro5018-cit-0024">
Letunic I, Bork P. <span class="ref-title">Interactive tree of life (iTOL) v5: an online tool for phylogenetic tree display and annotation</span>. <span class="ref-journal">Nucleic Acids Res</span>. 2021;<span class="ref-vol">49</span>:W293&#x02013;W296.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8265157/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33885785" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Interactive+tree+of+life+(iTOL)+v5:+an+online+tool+for+phylogenetic+tree+display+and+annotation&amp;volume=49&amp;publication_year=2021&amp;pages=W293-W296&amp;pmid=33885785&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0025"><span class="mixed-citation" id="pro5018-cit-0025">
Liebschner D, Afonine PV, Baker ML, Bunkoczi G, Chen VB, Croll TI, et al. <span class="ref-title">Macromolecular structure determination using X&#x02010;rays, neutrons and electrons: recent developments in Phenix</span>. <span class="ref-journal">Acta Crystallogr D</span>. 2019;<span class="ref-vol">75</span>:861&#x02013;877. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6778852/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31588918" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Crystallogr+D&amp;title=Macromolecular+structure+determination+using+X&#x02010;rays,+neutrons+and+electrons:+recent+developments+in+Phenix&amp;volume=75&amp;publication_year=2019&amp;pages=861-877&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0026"><span class="mixed-citation" id="pro5018-cit-0026">
Ma K, Linder D, Stetter KO, Thauer RK. <span class="ref-title">Purification and properties of <em>N</em>
<sup>5</sup>, <em>N</em>
<sup>10</sup>&#x02010;methylenetetrahydromethanopterin reductase (coenzyme F<sub>420</sub>&#x02010;dependent) from the extreme thermophile <em>Methanopyrus kandleri</em>
</span>. <span class="ref-journal">Arch Microbiol</span>. 1991;<span class="ref-vol">155</span>:593&#x02013;600.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/1953299" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Microbiol&amp;title=Purification+and+properties+of+N&#10;5,+N&#10;10&#x02010;methylenetetrahydromethanopterin+reductase+(coenzyme+F420&#x02010;dependent)+from+the+extreme+thermophile+Methanopyrus+kandleri&#10;&amp;volume=155&amp;publication_year=1991&amp;pages=593-600&amp;pmid=1953299&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0027"><span class="mixed-citation" id="pro5018-cit-0027">
Ma K, Thauer RK. <span class="ref-title">Purification and properties of <em>N</em>
<sup>5</sup>, <em>N</em>
<sup>10</sup>&#x02010;methylenetetrahydromethanopterin reductase from <em>Methanobacterium thermoautotrophicum</em> (strain Marburg)</span>. <span class="ref-journal">Eur J Biochem</span>. 1990a;<span class="ref-vol">191</span>:187&#x02013;193.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/2379499" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Biochem&amp;title=Purification+and+properties+of+N&#10;5,+N&#10;10&#x02010;methylenetetrahydromethanopterin+reductase+from+Methanobacterium+thermoautotrophicum+(strain+Marburg)&amp;volume=191&amp;publication_year=1990a&amp;pages=187-193&amp;pmid=2379499&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0028"><span class="mixed-citation" id="pro5018-cit-0028">
Ma K, Thauer RK. <span class="ref-title">
<em>N</em>
<sup>5</sup>, <em>N</em>
<sup>10</sup>&#x02010;Methylenetetrahydromethanopterin reductase from <em>Methanosarcina barkeri</em>
</span>. <span class="ref-journal">FEMS Microbiol Lett</span>. 1990b;<span class="ref-vol">70</span>:119&#x02013;124. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEMS+Microbiol+Lett&amp;title=&#10;N&#10;5,+N&#10;10&#x02010;Methylenetetrahydromethanopterin+reductase+from+Methanosarcina+barkeri&#10;&amp;volume=70&amp;publication_year=1990b&amp;pages=119-124&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0029"><span class="mixed-citation" id="pro5018-cit-0029">
Maden BE. <span class="ref-title">Tetrahydrofolate and tetrahydromethanopterin compared: functionally distinct carriers in C<sub>1</sub> metabolism</span>. <span class="ref-journal">Biochem J</span>. 2000;<span class="ref-vol">350</span>:609&#x02013;629.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1221290/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10970772" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+J&amp;title=Tetrahydrofolate+and+tetrahydromethanopterin+compared:+functionally+distinct+carriers+in+C1+metabolism&amp;volume=350&amp;publication_year=2000&amp;pages=609-629&amp;pmid=10970772&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0030"><span class="mixed-citation" id="pro5018-cit-0030">
Martin W, Russell MJ. <span class="ref-title">On the origin of biochemistry at an alkaline hydrothermal vent</span>. <span class="ref-journal">Philos Trans R Soc B</span>. 2007;<span class="ref-vol">362</span>:1887&#x02013;1926. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2442388/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17255002" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Philos+Trans+R+Soc+B&amp;title=On+the+origin+of+biochemistry+at+an+alkaline+hydrothermal+vent&amp;volume=362&amp;publication_year=2007&amp;pages=1887-1926&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0031"><span class="mixed-citation" id="pro5018-cit-0031">
Mascotti ML, Juri Ayub M, Fraaije MW. <span class="ref-title">On the diversity of F<sub>420</sub>&#x02010;dependent oxidoreductases: a sequence&#x02010; and structure&#x02010;based classification</span>. <span class="ref-journal">Proteins</span>. 2021;<span class="ref-vol">89</span>:1497&#x02013;1507.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8518648/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34216160" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=On+the+diversity+of+F420&#x02010;dependent+oxidoreductases:+a+sequence&#x02010;+and+structure&#x02010;based+classification&amp;volume=89&amp;publication_year=2021&amp;pages=1497-1507&amp;pmid=34216160&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0032"><span class="mixed-citation" id="pro5018-cit-0032">
McCoy AJ, Grosse&#x02010;Kunstleve RW, Adams PD, Winn MD, Storoni LC, Read RJ. <span class="ref-title">
<em>Phaser</em> crystallographic software</span>. <span class="ref-journal">J Appl Cryst</span>. 2007;<span class="ref-vol">40</span>:658&#x02013;674.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2483472/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19461840" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Appl+Cryst&amp;title=&#10;Phaser+crystallographic+software&amp;volume=40&amp;publication_year=2007&amp;pages=658-674&amp;pmid=19461840&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0033"><span class="mixed-citation" id="pro5018-cit-0033">
McKinney W, Data structures for statistical computing in python
. <span class="ref-title">Proceedings of the 9th python in science conference. Austin, TX</span>; 2010. p. 56&#x02013;61.</span></li><li id="pro5018-bib-0034"><span class="mixed-citation" id="pro5018-cit-0034">
Miller DV, Ruhlin M, Ray WK, Xu H, White RH. <span class="ref-title">
<em>N</em>
<sup>5</sup>, <em>N</em>
<sup>10</sup>&#x02010;methylenetetrahydromethanopterin reductase from <em>Methanocaldococcus jannaschii</em> also serves as a methylglyoxal reductase</span>. <span class="ref-journal">FEBS Lett</span>. 2017;<span class="ref-vol">591</span>:2269&#x02013;2278.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/28644554" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEBS+Lett&amp;title=&#10;N&#10;5,+N&#10;10&#x02010;methylenetetrahydromethanopterin+reductase+from+Methanocaldococcus+jannaschii+also+serves+as+a+methylglyoxal+reductase&amp;volume=591&amp;publication_year=2017&amp;pages=2269-2278&amp;pmid=28644554&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0035"><span class="mixed-citation" id="pro5018-cit-0035">
Minh BQ, Schmidt HA, Chernomor O, Schrempf D, Woodhams MD, von Haeseler A, et al. <span class="ref-title">IQ&#x02010;TREE 2: new models and efficient methods for phylogenetic inference in the genomic era</span>. <span class="ref-journal">Mol Biol Evol</span>. 2020;<span class="ref-vol">37</span>:1530&#x02013;1534.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7182206/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32011700" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Biol+Evol&amp;title=IQ&#x02010;TREE+2:+new+models+and+efficient+methods+for+phylogenetic+inference+in+the+genomic+era&amp;volume=37&amp;publication_year=2020&amp;pages=1530-1534&amp;pmid=32011700&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0036"><span class="mixed-citation" id="pro5018-cit-0036">
Mock J, Wang S, Huang H, Kahnt J, Thauer RK. <span class="ref-title">Evidence for a hexaheteromeric methylenetetrahydrofolate reductase in <em>Moorella thermoacetica</em>
</span>. <span class="ref-journal">J Bacteriol</span>. 2014;<span class="ref-vol">196</span>:3303&#x02013;3314.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4135698/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25002540" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=Evidence+for+a+hexaheteromeric+methylenetetrahydrofolate+reductase+in+Moorella+thermoacetica&#10;&amp;volume=196&amp;publication_year=2014&amp;pages=3303-3314&amp;pmid=25002540&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0037"><span class="mixed-citation" id="pro5018-cit-0037">
Murshudov GN, Skubak P, Lebedev AA, Pannu NS, Steiner RA, Nicholls RA, et al. <span class="ref-title">
<em>REFMAC5</em> for the refinement of macromolecular crystal structures</span>. <span class="ref-journal">Acta Crystallogr D</span>. 2011;<span class="ref-vol">67</span>:355&#x02013;367.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3069751/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21460454" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Crystallogr+D&amp;title=&#10;REFMAC5+for+the+refinement+of+macromolecular+crystal+structures&amp;volume=67&amp;publication_year=2011&amp;pages=355-367&amp;pmid=21460454&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0038"><span class="mixed-citation" id="pro5018-cit-0038">
Nagano N, Orengo CA, Thornton JM. <span class="ref-title">One fold with many functions: the evolutionary relationships between TIM barrel families based on their sequences, structures and functions</span>. <span class="ref-journal">J Mol Biol</span>. 2002;<span class="ref-vol">321</span>:741&#x02013;765.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/12206759" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=One+fold+with+many+functions:+the+evolutionary+relationships+between+TIM+barrel+families+based+on+their+sequences,+structures+and+functions&amp;volume=321&amp;publication_year=2002&amp;pages=741-765&amp;pmid=12206759&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0039"><span class="mixed-citation" id="pro5018-cit-0039">
&#x000d6;ppinger C, Kremp F, M&#x000fc;ller V. <span class="ref-title">Is reduced ferredoxin the physiological electron donor for MetVF&#x02010;type methylenetetrahydrofolate reductases in acetogenesis? A hypothesis</span>. <span class="ref-journal">Int Microbiol</span>. 2022;<span class="ref-vol">25</span>:75&#x02013;88.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8760232/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34255221" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+Microbiol&amp;title=Is+reduced+ferredoxin+the+physiological+electron+donor+for+MetVF&#x02010;type+methylenetetrahydrofolate+reductases+in+acetogenesis?+A+hypothesis&amp;volume=25&amp;publication_year=2022&amp;pages=75-88&amp;pmid=34255221&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0040"><span class="mixed-citation" id="pro5018-cit-0040">
Pal D, Chakrabarti P. <span class="ref-title">
<em>Cis</em> peptide bonds in proteins: residues involved, their conformations, interactions and locations</span>. <span class="ref-journal">J Mol Biol</span>. 1999;<span class="ref-vol">294</span>:271&#x02013;288.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/10556045" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=&#10;Cis+peptide+bonds+in+proteins:+residues+involved,+their+conformations,+interactions+and+locations&amp;volume=294&amp;publication_year=1999&amp;pages=271-288&amp;pmid=10556045&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0041"><span class="mixed-citation" id="pro5018-cit-0041">
Pei J, Kim BH, Grishin NV. <span class="ref-title">PROMALS3D: a tool for multiple protein sequence and structure alignments</span>. <span class="ref-journal">Nucleic Acids Res</span>. 2008;<span class="ref-vol">36</span>:2295&#x02013;2300.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2367709/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18287115" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=PROMALS3D:+a+tool+for+multiple+protein+sequence+and+structure+alignments&amp;volume=36&amp;publication_year=2008&amp;pages=2295-2300&amp;pmid=18287115&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0042"><span class="mixed-citation" id="pro5018-cit-0042">
Pejchal R, Sargeant R, Ludwig ML. <span class="ref-title">Structures of NADH and CH<sub>3</sub>&#x02010;H<sub>4</sub>folate complexes of <em>Escherichia coli</em> methylenetetrahydrofolate reductase reveal a spartan strategy for a ping&#x02010;pong reaction</span>. <span class="ref-journal">Biochemistry</span>. 2005;<span class="ref-vol">44</span>:11447&#x02013;11457.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/16114881" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Structures+of+NADH+and+CH3&#x02010;H4folate+complexes+of+Escherichia+coli+methylenetetrahydrofolate+reductase+reveal+a+spartan+strategy+for+a+ping&#x02010;pong+reaction&amp;volume=44&amp;publication_year=2005&amp;pages=11447-11457&amp;pmid=16114881&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0043"><span class="mixed-citation" id="pro5018-cit-0043">
Perry KM, Carreras CW, Chang LC, Santi DV, Stroud RM. <span class="ref-title">Structures of thymidylate synthase with a C&#x02010;terminal deletion: role of the C&#x02010;terminus in alignment of 2&#x02032;&#x02010;deoxyuridine 5&#x02032;&#x02010;monophosphate and 5,10&#x02010;methylenetetrahydrofolate</span>. <span class="ref-journal">Biochemistry</span>. 1993;<span class="ref-vol">32</span>:7116&#x02013;7125.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/8343503" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Structures+of+thymidylate+synthase+with+a+C&#x02010;terminal+deletion:+role+of+the+C&#x02010;terminus+in+alignment+of+2&#x02032;&#x02010;deoxyuridine+5&#x02032;&#x02010;monophosphate+and+5,10&#x02010;methylenetetrahydrofolate&amp;volume=32&amp;publication_year=1993&amp;pages=7116-7125&amp;pmid=8343503&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0044"><span class="mixed-citation" id="pro5018-cit-0044">
Romero&#x02010;Romero S, Kordes S, Michel F, Hocker B. <span class="ref-title">Evolution, folding, and design of TIM barrels and related proteins</span>. <span class="ref-journal">Curr Opin Struct Biol</span>. 2021;<span class="ref-vol">68</span>:94&#x02013;104.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8250049/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33453500" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Struct+Biol&amp;title=Evolution,+folding,+and+design+of+TIM+barrels+and+related+proteins&amp;volume=68&amp;publication_year=2021&amp;pages=94-104&amp;pmid=33453500&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0045"><span class="mixed-citation" id="pro5018-cit-0045">
Sah S, Lahry K, Talwar C, Singh S, Varshney U. <span class="ref-title">Monomeric NADH&#x02010;oxidizing methylenetetrahydrofolate reductases from <em>Mycobacterium smegmatis</em> lack flavin coenzyme</span>. <span class="ref-journal">J Bacteriol</span>. 2020;<span class="ref-vol">202</span>:e00709&#x02013;e00719.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7253612/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32253341" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=Monomeric+NADH&#x02010;oxidizing+methylenetetrahydrofolate+reductases+from+Mycobacterium+smegmatis+lack+flavin+coenzyme&amp;volume=202&amp;publication_year=2020&amp;pages=e00709-e00719&amp;pmid=32253341&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0046"><span class="mixed-citation" id="pro5018-cit-0046">
Schmitz RA, Linder D, Stetter KO, Thauer RK. <span class="ref-title">
<em>N</em>
<sup>5</sup>, <em>N</em>
<sup>10</sup>&#x02010;Methylenetetrahydromethanopterin reductase (coenzyme F<sub>420</sub>&#x02010;dependent) and formylmethanofuran dehydrogenase from the hyperthermophile <em>Archaeoglobus fulgidus</em>
</span>. <span class="ref-journal">Arch Microbiol</span>. 1991;<span class="ref-vol">156</span>:427&#x02013;434. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Microbiol&amp;title=&#10;N&#10;5,+N&#10;10&#x02010;Methylenetetrahydromethanopterin+reductase+(coenzyme+F420&#x02010;dependent)+and+formylmethanofuran+dehydrogenase+from+the+hyperthermophile+Archaeoglobus+fulgidus&#10;&amp;volume=156&amp;publication_year=1991&amp;pages=427-434&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0047"><span class="mixed-citation" id="pro5018-cit-0047">
Sheppard CA, Trimmer EE, Matthews RG. <span class="ref-title">Purification and properties of NADH&#x02010;dependent 5, 10&#x02010;methylenetetrahydrofolate reductase (MetF) from <em>Escherichia coli</em>
</span>. <span class="ref-journal">J Bacteriol</span>. 1999;<span class="ref-vol">181</span>:718&#x02013;725.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC93435/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9922232" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=Purification+and+properties+of+NADH&#x02010;dependent+5,+10&#x02010;methylenetetrahydrofolate+reductase+(MetF)+from+Escherichia+coli&#10;&amp;volume=181&amp;publication_year=1999&amp;pages=718-725&amp;pmid=9922232&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0048"><span class="mixed-citation" id="pro5018-cit-0048">
Shima S, Schick M, Tamura H. <span class="ref-title">Preparation of [Fe]&#x02010;hydrogenase from methanogenic archaea</span>. <span class="ref-journal">Methods Enzymol</span>. 2011;<span class="ref-vol">494</span>:119&#x02013;137.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/21402213" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Enzymol&amp;title=Preparation+of+[Fe]&#x02010;hydrogenase+from+methanogenic+archaea&amp;volume=494&amp;publication_year=2011&amp;pages=119-137&amp;pmid=21402213&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0049"><span class="mixed-citation" id="pro5018-cit-0049">
Shima S, Warkentin E, Grabarse W, Sordel M, Wicke M, Thauer RK, et al. <span class="ref-title">Structure of coenzyme F<sub>420</sub> dependent methylenetetrahydromethanopterin reductase from two methanogenic archaea</span>. <span class="ref-journal">J Mol Biol</span>. 2000;<span class="ref-vol">300</span>:935&#x02013;950.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/10891279" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Structure+of+coenzyme+F420+dependent+methylenetetrahydromethanopterin+reductase+from+two+methanogenic+archaea&amp;volume=300&amp;publication_year=2000&amp;pages=935-950&amp;pmid=10891279&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0050"><span class="mixed-citation" id="pro5018-cit-0050">
Sousa FL, Martin WF. <span class="ref-title">Biochemical fossils of the ancient transition from geoenergetics to bioenergetics in prokaryotic one carbon compound metabolism</span>. <span class="ref-journal">Biochim Biophys Acta</span>. 2014;<span class="ref-vol">1837</span>:964&#x02013;981.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/24513196" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochim+Biophys+Acta&amp;title=Biochemical+fossils+of+the+ancient+transition+from+geoenergetics+to+bioenergetics+in+prokaryotic+one+carbon+compound+metabolism&amp;volume=1837&amp;publication_year=2014&amp;pages=964-981&amp;pmid=24513196&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0051"><span class="mixed-citation" id="pro5018-cit-0051">
Sterner R, Hocker B. <span class="ref-title">Catalytic versatility, stability, and evolution of the (&#x003b2;&#x003b1;)<sub>8</sub>&#x02010;barrel enzyme fold</span>. <span class="ref-journal">Chem Rev</span>. 2005;<span class="ref-vol">105</span>:4038&#x02013;4055.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/16277370" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem+Rev&amp;title=Catalytic+versatility,+stability,+and+evolution+of+the+(&#x003b2;&#x003b1;)8&#x02010;barrel+enzyme+fold&amp;volume=105&amp;publication_year=2005&amp;pages=4038-4055&amp;pmid=16277370&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0052"><span class="mixed-citation" id="pro5018-cit-0052">
Sumner JS, Matthews RG. <span class="ref-title">Stereochemistry and mechanism of hydrogen transfer between NADPH and methylenetetrahydrofolate in the reaction catalyzed by methylenetetrahydrofolate reductase from pig liver</span>. <span class="ref-journal">J Am Chem Soc</span>. 1992;<span class="ref-vol">114</span>:6949&#x02013;6956. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Chem+Soc&amp;title=Stereochemistry+and+mechanism+of+hydrogen+transfer+between+NADPH+and+methylenetetrahydrofolate+in+the+reaction+catalyzed+by+methylenetetrahydrofolate+reductase+from+pig+liver&amp;volume=114&amp;publication_year=1992&amp;pages=6949-6956&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0053"><span class="mixed-citation" id="pro5018-cit-0053">
te Brommelstroet BW, Hensgens CM, Keltjens JT, van der Drift C, Vogels GD. <span class="ref-title">Purification and properties of 5,10&#x02010;methylenetetrahydromethanopterin reductase, a coenzyme F<sub>420</sub>&#x02010;dependent enzyme, from <em>Methanobacterium thermoautotrophicum</em> strain &#x00394;<em>H</em>
</span>. <span class="ref-journal">J Biol Chem</span>. 1990;<span class="ref-vol">265</span>:1852&#x02013;1857.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/2298726" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Purification+and+properties+of+5,10&#x02010;methylenetetrahydromethanopterin+reductase,+a+coenzyme+F420&#x02010;dependent+enzyme,+from+Methanobacterium+thermoautotrophicum+strain+&#x00394;H&#10;&amp;volume=265&amp;publication_year=1990&amp;pages=1852-1857&amp;pmid=2298726&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0054"><span class="mixed-citation" id="pro5018-cit-0054">
Thauer RK, Klein AR, Hartmann GC. <span class="ref-title">Reactions with molecular hydrogen in microorganisms: evidence for a purely organic hydrogenation catalyst</span>. <span class="ref-journal">Chem Rev</span>. 1996;<span class="ref-vol">96</span>:3031&#x02013;3042.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/11848851" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem+Rev&amp;title=Reactions+with+molecular+hydrogen+in+microorganisms:+evidence+for+a+purely+organic+hydrogenation+catalyst&amp;volume=96&amp;publication_year=1996&amp;pages=3031-3042&amp;pmid=11848851&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0055"><span class="mixed-citation" id="pro5018-cit-0055">
Trimmer EE, Ballou DP, Galloway LJ, Scannell SA, Brinker DR, Casas KR. <span class="ref-title">Aspartate 120 of <em>Escherichia coli</em> methylenetetrahydrofolate reductase: evidence for major roles in folate binding and catalysis and a minor role in flavin reactivity</span>. <span class="ref-journal">Biochemistry</span>. 2005;<span class="ref-vol">44</span>:6809&#x02013;6822.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/15865426" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Aspartate+120+of+Escherichia+coli+methylenetetrahydrofolate+reductase:+evidence+for+major+roles+in+folate+binding+and+catalysis+and+a+minor+role+in+flavin+reactivity&amp;volume=44&amp;publication_year=2005&amp;pages=6809-6822&amp;pmid=15865426&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0056"><span class="mixed-citation" id="pro5018-cit-0056">
Trimmer EE, Ballou DP, Ludwig ML, Matthews RG. <span class="ref-title">Folate activation and catalysis in methylenetetrahydrofolate reductase from <em>Escherichia coli</em>: roles for aspartate 120 and glutamate 28</span>. <span class="ref-journal">Biochemistry</span>. 2001;<span class="ref-vol">40</span>:6216&#x02013;6226.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/11371182" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Folate+activation+and+catalysis+in+methylenetetrahydrofolate+reductase+from+Escherichia+coli:+roles+for+aspartate+120+and+glutamate+28&amp;volume=40&amp;publication_year=2001&amp;pages=6216-6226&amp;pmid=11371182&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0057"><span class="mixed-citation" id="pro5018-cit-0057">
Vaupel M, Thauer RK. <span class="ref-title">Coenzyme F<sub>420</sub>&#x02010;dependent <em>N</em>
<sup>5</sup>, <em>N</em>
<sup>10</sup>&#x02010;methylenetetrahydromethanopterin reductase (Mer) from <em>Methanobacterium thermoautotrophicum</em> strain Marburg. Cloning, sequencing, transcriptional analysis, and functional expression in <em>Escherichia coli</em> of the mer gene</span>. <span class="ref-journal">Eur J Biochem</span>. 1995;<span class="ref-vol">231</span>:773&#x02013;778.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/7649177" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Biochem&amp;title=Coenzyme+F420&#x02010;dependent+N&#10;5,+N&#10;10&#x02010;methylenetetrahydromethanopterin+reductase+(Mer)+from+Methanobacterium+thermoautotrophicum+strain+Marburg.+Cloning,+sequencing,+transcriptional+analysis,+and+functional+expression+in+Escherichia+coli+of+the+mer+gene&amp;volume=231&amp;publication_year=1995&amp;pages=773-778&amp;pmid=7649177&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0058"><span class="mixed-citation" id="pro5018-cit-0058">
Virtanen P, Gommers R, Oliphant TE, Haberland M, Reddy T, Cournapeau D, et al. <span class="ref-title">SciPy 1.0: fundamental algorithms for scientific computing in python</span>. <span class="ref-journal">Nat Methods</span>. 2020;<span class="ref-vol">17</span>:261&#x02013;272.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7056644/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32015543" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=SciPy+1.0:+fundamental+algorithms+for+scientific+computing+in+python&amp;volume=17&amp;publication_year=2020&amp;pages=261-272&amp;pmid=32015543&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0059"><span class="mixed-citation" id="pro5018-cit-0059">
Waskom ML. <span class="ref-title">Seaborn: statistical data visualization</span>. <span class="ref-journal">J Open Source Softw</span>. 2021;<span class="ref-vol">6</span>:3021. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Open+Source+Softw&amp;title=Seaborn:+statistical+data+visualization&amp;volume=6&amp;publication_year=2021&amp;pages=3021&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0060"><span class="mixed-citation" id="pro5018-cit-0060">
Wierenga RK. <span class="ref-title">The TIM&#x02010;barrel fold: a versatile framework for efficient enzymes</span>. <span class="ref-journal">FEBS Lett</span>. 2001;<span class="ref-vol">492</span>:193&#x02013;198.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/11257493" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEBS+Lett&amp;title=The+TIM&#x02010;barrel+fold:+a+versatile+framework+for+efficient+enzymes&amp;volume=492&amp;publication_year=2001&amp;pages=193-198&amp;pmid=11257493&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0061"><span class="mixed-citation" id="pro5018-cit-0061">
Williams CJ, Headd JJ, Moriarty NW, Prisant MG, Videau LL, Deis LN, et al. <span class="ref-title">MolProbity: more and better reference data for improved all&#x02010;atom structure validation</span>. <span class="ref-journal">Protein Sci</span>. 2018;<span class="ref-vol">27</span>:293&#x02013;315.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5734394/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29067766" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Sci&amp;title=MolProbity:+more+and+better+reference+data+for+improved+all&#x02010;atom+structure+validation&amp;volume=27&amp;publication_year=2018&amp;pages=293-315&amp;pmid=29067766&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0062"><span class="mixed-citation" id="pro5018-cit-0062">
Yu JF, Xu JT, Yang SS, Gao MN, Si HR, Xiong DY, et al. <span class="ref-title">Decreased methylenetetrahydrofolate reductase activity leads to increased sensitivity to <em>para</em>&#x02010;aminosalicylic acid in <em>Mycobacterium tuberculosis</em>
</span>. <span class="ref-journal">Antimicrob Agents Chemother</span>. 2022;<span class="ref-vol">66</span>:e0146521.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8765232/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34780266" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Antimicrob+Agents+Chemother&amp;title=Decreased+methylenetetrahydrofolate+reductase+activity+leads+to+increased+sensitivity+to+para&#x02010;aminosalicylic+acid+in+Mycobacterium+tuberculosis&#10;&amp;volume=66&amp;publication_year=2022&amp;pages=e0146521&amp;pmid=34780266&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0063"><span class="mixed-citation" id="pro5018-cit-0063">
Zheng Y, Cantley LC. <span class="ref-title">Toward a better understanding of folate metabolism in health and disease</span>. <span class="ref-journal">J Exp Med</span>. 2019;<span class="ref-vol">216</span>:253&#x02013;266.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6363433/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30587505" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Exp+Med&amp;title=Toward+a+better+understanding+of+folate+metabolism+in+health+and+disease&amp;volume=216&amp;publication_year=2019&amp;pages=253-266&amp;pmid=30587505&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="pro5018-bib-0064"><span class="mixed-citation" id="pro5018-cit-0064">
Zuo C, Jolly AL, Nikolova AP, Satzer DI, Cao S, Sanchez JS, et al. <span class="ref-title">A role for glutamine 183 in the folate oxidative half&#x02010;reaction of methylenetetrahydrofolate reductase from Escherichia coli</span>. <span class="ref-journal">Arch Biochem Biophys</span>. 2018;<span class="ref-vol">642</span>:63&#x02013;74.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/29407039" ref="reftype=pubmed&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Biochem+Biophys&amp;title=A+role+for+glutamine+183+in+the+folate+oxidative+half&#x02010;reaction+of+methylenetetrahydrofolate+reductase+from+Escherichia+coli&amp;volume=642&amp;publication_year=2018&amp;pages=63-74&amp;pmid=29407039&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11094777&amp;issue-id=461969&amp;journal-id=507&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li></ul></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Protein Science : A Publication of the Protein Society</span> are provided here courtesy of <strong>The Protein Society</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11094777/pdf/PRO-33-e5018.pdf" class="int-view">PDF (11M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11094777/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11094777/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11094777" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11094777%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="Iph0uiZ9zQkt1ptoOjzDMj3cMP0VbeqP3wCKX8JeTQiVAiV3AklocIMfcUe9x27T">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11094777%2F&amp;text=Mutational%20and%20structural%20studies%20of%20(%003B2%003B1)8%02010barrel%20fold%20methylene%02010tetrahydropterin%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11094777%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11094777/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11094777/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38747406/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11094777/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38747406/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11094777/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11094777/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC11094777/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
