
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Benchmarking Water
Models in Molecular Dynamics of
Protein&#8211;Glycosaminoglycan Complexes - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="939BE807DB0F4BC50000416A3D93DE90.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="acssd" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC10934818/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10934818/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Journal of Chemical Information and Modeling" /><meta name="citation_title" content="Benchmarking Water&#10;Models in Molecular Dynamics of&#10;Protein&#x02013;Glycosaminoglycan Complexes" /><meta name="citation_author" content="Sebastian Anila" /><meta name="citation_author" content="Sergey A. Samsonov" /><meta name="citation_publication_date" content="2024/03/03" /><meta name="citation_issue" content="5" /><meta name="citation_volume" content="64" /><meta name="citation_firstpage" content="1691" /><meta name="citation_doi" content="10.1021/acs.jcim.4c00030" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC10934818/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC10934818/" /><meta name="citation_pmid" content="38410841" /><meta name="DC.Title" content="Benchmarking Water&#10;Models in Molecular Dynamics of&#10;Protein&#x02013;Glycosaminoglycan Complexes" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="American Chemical Society" /><meta name="DC.Contributor" content="Sebastian Anila" /><meta name="DC.Contributor" content="Sergey A. Samsonov" /><meta name="DC.Date" content="2024 Mar 11" /><meta name="DC.Identifier" content="10.1021/acs.jcim.4c00030" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Benchmarking Water&#10;Models in Molecular Dynamics of&#10;Protein&#x02013;Glycosaminoglycan Complexes" /><meta property="og:type" content="article" /><meta property="og:description" content="Glycosaminoglycans (GAGs) made of repeating disaccharide&#10;units&#10;intricately engage with proteins, playing a crucial role in the spatial&#10;organization of the extracellular matrix (ECM) and the transduction&#10;of biological signals in cells to modulate a number ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10934818/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10934818/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22ACS%20AuthorChoice%22[filter]" class="navlink">ACS AuthorChoice</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC10934818
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC10934818/pdf/ci4c00030.pdf" class="int-view">PDF (10M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/10934818/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/10934818/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="10934818" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10934818%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="tVQmpxaVyqw3MbmJGfkL5nI7C3Rs9lOVDVtOe6HZ9WU6udrwh3EDwkX0F6oBHstF">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10934818%2F&amp;text=Benchmarking%20Water%0AModels%20in%20Molecular%20Dynamics%20of%0AProtein%02013Glycosaminoglycan%20Complexes" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10934818%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10934818/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC10934818/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38410841/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC10934818/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38410841/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC10934818/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/10934818/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22ACS%20AuthorChoice%22[filter]" class="navlink">ACS AuthorChoice</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC10934818
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><a href="//doi.org/10.1021%2Facs.jcim.4c00030" title="Link to Publisher's site" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"><img src="/corehtml/pmc/pmcgifs/logo-acssd.png" alt="Logo of acssd"><!-- --></img></a></div>
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">J Chem Inf Model.</span> 2024 Mar 11; 64(5): 1691–1703. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2024 Feb 27. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1021%2Facs.jcim.4c00030" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1021/acs.jcim.4c00030</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC10934818</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38410841">38410841</a></div></div></div><h1 class="content-title">Benchmarking Water
Models in Molecular Dynamics of
Protein&#x02013;Glycosaminoglycan Complexes</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Anila%20S%5BAuthor%5D" id="ath1">Sebastian Anila</a> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Samsonov%20SA%5BAuthor%5D" id="ath2">Sergey A. Samsonov</a><sup></sup><a href="#cor1" rid="cor1" class=" other">*</a></div><div style="display:none" class="contrib-group aff-tip"></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140421146573152_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140421146573152_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140421146573152_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140421146573152_ai" style="display:none"><div class="fm-affl" id="aff1">Faculty of Chemistry, University of Gda&#x00144;sk, ul. Wita Stwosza 63, 80-308 Gda&#x00144;sk, Poland</div><div id="cor1"><sup>*</sup>Email: <a href="mailto:dev@null" data-email="lp.ude.gu@vonosmas.yegres" class="oemail">lp.ude.gu@vonosmas.yegres</a>.</div></div><div class="fm-article-notes hide half_rhythm" id="idm140421146573152_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2024 Jan 11; Revised 2024 Feb 8; Accepted 2024 Feb 13.</div></div><div class="permissions half_rhythm hide" id="idm140421146573152_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; 2024 The Authors. Published by American Chemical Society</div><div class="license half_rhythm">Permits the broadest form of re-use including for commercial purposes, provided that author attribution and integrity are maintained (<a href="https://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">https://creativecommons.org/licenses/by/4.0/</a>).</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">ci4c00030_si_001.pdf</div><div><a href="/pmc/articles/PMC10934818/bin/ci4c00030_si_001.pdf" data-ga-action="click_feat_suppl">ci4c00030_si_001.pdf</a><span style="color:gray"> (1.4M)</span></div><div class="small guid">GUID: BBAA03D6-1096-44AF-A7BC-E345AA311533</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">ci4c00030_si_002.zip</div><div><a href="/pmc/articles/PMC10934818/bin/ci4c00030_si_002.zip" data-ga-action="click_feat_suppl">ci4c00030_si_002.zip</a><span style="color:gray"> (15M)</span></div><div class="small guid">GUID: 16D63F53-CFE6-4BB8-A4F3-BD3C7E7C1038</div></div></dd></dl><dl data-length="657" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p>All of the data
underlying this study are available in the manuscript and its Supporting
Information files. Apart from that, all of the data (MD simulations)
were obtained using the AMBER suite (Amber20 and AmberTools). The
data were then analyzed using R (statistics and plots), VMD (visualization
of the obtained trajectories), and GIMP (figure preparation). All
software except for the AMBER suite is free of charge. AMBER software
can be obtained from <a href="http://ambermd.org/GetAmber.php">http://ambermd.org/GetAmber.php</a>. R can be downloaded from <a href="https://www.r-project.org/">https://www.r-project.org/</a>. GIMP can be downloaded from <a href="https://www.gimp.org/">https://www.gimp.org/</a>. VMD can
be downloaded from <a href="http://www.ks.uiuc.edu/Research/vmd/">http://www.ks.uiuc.edu/Research/vmd/.</a></p></dd></dl></div><div id="abstract-a.v.b.o" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.v.b.otitle">Abstract</h2><!--article-meta--><div><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140421133367360"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ci4c00030_0015.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is ci4c00030_0015.jpg" src="/pmc/articles/PMC10934818/bin/ci4c00030_0015.jpg" /></div><p></p><p class="p p-last">Glycosaminoglycans (GAGs) made of repeating disaccharide
units
intricately engage with proteins, playing a crucial role in the spatial
organization of the extracellular matrix (ECM) and the transduction
of biological signals in cells to modulate a number of biochemical
processes. Exploring protein&#x02013;GAG interactions reveals several
challenges for their analysis, namely, the highly charged and periodic
nature of GAGs, their multipose binding, and the abundance of the
interfacial water molecules in the protein&#x02013;GAG complexes. Most
of the studies on protein&#x02013;GAG interactions are conducted using
the TIP3P water model, and there are no data on the effect of various
water models on the results obtained in molecular dynamics (MD) simulations
of protein&#x02013;GAG complexes. Hence, it is essential to perform
a systematic analysis of different water models in MD simulations
for these systems. In this work, we aim to evaluate the properties
of the protein&#x02013;GAG complexes in MD simulations using different
explicit: TIP3P, SPC/E, TIP4P, TIP4PEw, OPC, and TIP5P and implicit:
IGB = 1, 2, 5, 7, and 8 water models to find out which of them are
best suited to study the dynamics of protein&#x02013;GAG complexes.
The FF14SB and GLYCAM06 force fields were used for the proteins and
GAGs, respectively. The interactions of several GAG types, such as
heparin, chondroitin sulfate, and hyaluronic acid with basic fibroblast
growth factor, cathepsin K, and CD44 receptor, respectively, are investigated.
The observed variations in different descriptors used to study the
binding in these complexes emphasize the relevance of the choice of
water models for the MD simulation of these complexes.</p></div></div><div id="sec1" class="tsec sec"><h2 class="head no_bottom_margin" id="sec1title">Introduction</h2><p class="p p-first">Glycosaminoglycans (GAGs) stand out among
diverse functional polymers
within human cells. These long linear periodic negatively charged
polydisperse polysaccharides complexly engage with proteins, playing
a crucial role in the processes of the extracellular matrix (ECM).<sup><a href="#ref1" rid="ref1" class=" bibr popnode">1</a></sup> Natural as well as chemically modified GAG have
gained significant attention for their potential in biomaterial design.<sup><a href="#ref2" rid="ref2" class=" bibr popnode">2</a>,<a href="#ref3" rid="ref3" class=" bibr popnode">3</a></sup> Their application holds promise for adopting biospecific cell behavior,
particularly in the realms of skin and bone tissue regeneration.<sup><a href="#ref4" rid="ref4" class=" bibr popnode">4</a>&#x02212;<a href="#ref6" rid="ref6" class=" bibr popnode">6</a></sup> GAGs are made of repeating disaccharide units, each of which consists
of a hexuronic acid (or galactose in keratan sulfate) and a hexosamine
(<em>N</em>-acetylglycosamide, GlcNAc or <em>N</em>-acetylgalactososamide GalNAc) connected with 1&#x02013;4 or 1&#x02013;3
glycosidic linkages, while hydroxyl groups of hexose and hexosamine
at various positions can be sulfated. Based on the core disaccharide
structures, GAGs can be classified as heparin (HP), heparan sulfate
(HS), chondroitin sulfate (CS), dermatan sulfate (DS), hyaluronic
acid (HA), and keratan sulfate (KS).<sup><a href="#ref7" rid="ref7" class=" bibr popnode">7</a></sup> The
disaccharide units of HP and HS are both structurally composed of
alternating 4-linked uronic acid (GlcA/IdoA) and 4-linked &#x003b1;-glucosamine
(GlcN) units.<sup><a href="#ref8" rid="ref8" class=" bibr popnode">8</a></sup> CS disaccharide units are
composed of alternating 4-linked &#x003b2;-<span class="small-caps">d</span>-GlcA and 3-linked
&#x003b2;-<span class="small-caps">d</span>-galactosamine (GalNAc) units. Among various CS
subtypes, owing to structural variations: CS-A is mostly 4-sulfated
at the GalNAc units, while CS-C is predominantly 6-sulfated, and CS-B,
widely known as DS, has &#x003b1;-<span class="small-caps">l</span>-IdoA units rather than
&#x003b2;-<span class="small-caps">d</span>-GlcA. The IdoA units in DS may bear 2-sulfation,
while the GalNAc units are mostly 4-sulfated.<sup><a href="#ref9" rid="ref9" class=" bibr popnode">9</a></sup> KS disaccharide units are composed of alternating 3-linked &#x003b2;-<span class="small-caps">d</span>-Gal and 4-linked &#x003b2;-<span class="small-caps">d</span>-GlcNAc units.<sup><a href="#ref10" rid="ref10" class=" bibr popnode">10</a></sup> Unlike other GAGs, HA lacks sulfate groups and
is not covalently bound to a protein. It is composed of repeating
disaccharide units of alternating 4-linked &#x003b2;-<span class="small-caps">d</span>-GlcA
and 3-linked &#x003b2;-<span class="small-caps">d</span>-GlcNAc.<sup><a href="#ref11" rid="ref11" class=" bibr popnode">11</a></sup></p><p>Based on the sulfation pattern and monosaccharide composition,
GAG disaccharide units can display 408 variants.<sup><a href="#ref12" rid="ref12" class=" bibr popnode">12</a></sup> The variations in the monosaccharide composition and the
sulfation patterns of the GAGs may alter their binding and functional
properties as well as conformational characteristics.<sup><a href="#ref13" rid="ref13" class=" bibr popnode">13</a></sup> GAGs are mainly located in the extracellular matrix. Although
they were once thought of as a kind of inert glue around cells, recent
research has shown that GAGs play a vital role in building biological
systems and the transduction of biological signals in cell proliferation,
regeneration, lipid metabolism, angiogenesis, and metastasis.<sup><a href="#ref14" rid="ref14" class=" bibr popnode">14</a>,<a href="#ref15" rid="ref15" class=" bibr popnode">15</a></sup> All these processes are mediated through their direct interactions
with diverse protein targets such as collagens, chemokines, cytokines,
growth factors, antithrombin, and cell adhesion molecules, which makes
them essential players in cell biology.<sup><a href="#ref16" rid="ref16" class=" bibr popnode">16</a>&#x02212;<a href="#ref20" rid="ref20" class=" bibr popnode">20</a></sup></p><p>The participation of GAGs in physiological,
pathological, or therapeutic
functions results principally from their unique physicochemical and
structural features, including high negative charge, high viscosity,
lubricative properties, periodicity, unbranched polysaccharide structures,
low compressibility, and the ability to attract and absorb enormous
amounts of water.<sup><a href="#ref21" rid="ref21" class=" bibr popnode">21</a></sup> Analysis of the interaction
of GAG with proteins is required for understanding various physiological
and pathological mechanisms and is a huge stimulus for drug development.
Most protein&#x02013;GAG interactions are driven by electrostatics
and are nonspecific,<sup><a href="#ref22" rid="ref22" class=" bibr popnode">22</a>,<a href="#ref23" rid="ref23" class=" bibr popnode">23</a></sup> whereas some of them, in contrast,
can be highly specific<sup><a href="#ref24" rid="ref24" class=" bibr popnode">24</a></sup> or selective.<sup><a href="#ref25" rid="ref25" class=" bibr popnode">25</a></sup> Computational approaches rapidly developing
for carbohydrates in recent decades including the emergence of new
force fields and scoring functions are useful for a detailed analysis
of the structure&#x02013;function relationships underlying the mechanisms
of protein&#x02013;GAG interactions and for gaining a better understanding
of the molecular bases of carbohydrate recognition.<sup><a href="#ref26" rid="ref26" class=" bibr popnode">26</a>&#x02212;<a href="#ref29" rid="ref29" class=" bibr popnode">29</a></sup> However, there are still many
challenges for computational studies of protein&#x02013;GAG interactions.
GAGs are highly charged molecules; therefore, electrostatics should
be treated appropriately.<sup><a href="#ref30" rid="ref30" class=" bibr popnode">30</a></sup> Also, the interfacial
water molecules are crucial in defining the GAG binding pose, and
hence, the solvent-mediated interactions should be accurately taken
into account for protein&#x02013;GAG interaction analysis due to their
abundance.<sup><a href="#ref31" rid="ref31" class=" bibr popnode">31</a>,<a href="#ref32" rid="ref32" class=" bibr popnode">32</a></sup> Precisely predicting the strength
and nature of protein&#x02013;GAG interactions remains intangible also
due to the need for comprehensive analyses of the conformational space
of the flexible positively charged amino acid residues recognizing
negatively charged GAGs. The presence of multiple binding sites in
a single protein structure for either the same or different GAGs further
complicates this task.<sup><a href="#ref33" rid="ref33" class=" bibr popnode">33</a></sup> The dynamic nature
of the protein&#x02013;GAG interactions with different ranges of specificity
underlines the complexity inherent in interpreting the mechanisms
in which they are involved, emphasizing the necessity for sophisticated
approaches in the study of these molecular interactions.</p><p>The
solvent effect in the molecular docking and molecular dynamics
(MD) simulations can be considered by using either explicit or implicit
water models. The explicit solvent model represents the solvent with
individual atomistic solvent molecules surrounding the solute. In
contrast, an implicit solvent model mimics the presence of a solvent
in an average manner as a continuous medium surrounding the solute.
In Amber, generalized Born (GB) is the most common implicit solvent
model. This model is based on an approximation to the exact (linearized)
Poisson&#x02013;Boltzmann equation.<sup><a href="#ref34" rid="ref34" class=" bibr popnode">34</a>,<a href="#ref35" rid="ref35" class=" bibr popnode">35</a></sup> Recent studies
have shown that GB models do not reproduce well some basic molecular
properties like the secondary structures of <em>de novo</em> designed peptides.<sup><a href="#ref36" rid="ref36" class=" bibr popnode">36</a></sup> Most of the MD-related
studies on GAGs are conducted using the TIP3P water model, as it is
widely accepted in the GAG field and proven to be working well in
the protein&#x02013;GAG systems, as well as in MD studies of other
biomolecular systems in general.<sup><a href="#ref37" rid="ref37" class=" bibr popnode">37</a>&#x02212;<a href="#ref39" rid="ref39" class=" bibr popnode">39</a></sup> The basic reason for
the wide use of TIP3P and other three-site water models is their low
computational cost compared to four- and five-site solvent models.
Although some studies suggest the use of a more advanced water model
than three-site TIP3P, e.g., TIP4P or TIP5P, there are no data on
the use of different water models for protein&#x02013;GAG complexes,
and thus, there is no evidence of which other model should be more
appropriate for these systems.<sup><a href="#ref40" rid="ref40" class=" bibr popnode">40</a>,<a href="#ref41" rid="ref41" class=" bibr popnode">41</a></sup> A comparison of the
HP properties in the MD simulations with SPC and SPC/E water models
was reported, stating the superiority of SPC/E over the SPC water
model.<sup><a href="#ref42" rid="ref42" class=" bibr popnode">42</a></sup> The comparison of various explicit
water models for modeling CS by Neamtu et al. suggested TIP4P and
TIP4PEw water models as the most appropriate ones.<sup><a href="#ref43" rid="ref43" class=" bibr popnode">43</a></sup> Recently, Marcisz and Samsonov reported a detailed evaluation
of the properties of the HP in different explicit and implicit water
models.<sup><a href="#ref44" rid="ref44" class=" bibr popnode">44</a></sup> Their study has shown that TIP5P
and OPC water models allowed the best agreement with the experiment
for both local and global structural features of HP in the MD simulations.</p><p>Since explicit water models can improve the performance of molecular
docking and MD simulations,<sup><a href="#ref31" rid="ref31" class=" bibr popnode">31</a>,<a href="#ref45" rid="ref45" class=" bibr popnode">45</a></sup> currently, implicit
water models are used less frequently, especially due to easier access
to high-performance computing facilities than before. With the emerging
computing facilities, the computational costs can be surpassed to
greater extents, and hence, even five-point or more computationally
expensive water models can be used in the analysis of the protein&#x02013;GAG
complexes. However, owing to their low computational cost, implicit
solvent models are still utilized when computational resources and
time are limited or the size of the studied system is particularly
large. Thus, it is essential to obtain the data from a systematic
analysis of the effect of different water models on the MD of protein&#x02013;GAG
complexes.</p><p class="p p-last">In this work, we aim to study the properties of the
protein&#x02013;GAG
complexes with MD simulations using different explicit and implicit
water models to find out which of them are best suited for these complexes.
In the present work, all-atom MD simulations are conducted for 10
&#x003bc;s to study the dynamics of the protein&#x02013;GAG complexes,
complemented by free energy analysis in different water models. The
free energy analysis of the protein&#x02013;GAG interactions is important
to understand the nature of the interactions and the stability of
the binding pose, especially in the case of complexes with multiple
binding poses. Here, the binding interactions of the GAGs: HP, CS,
and HA with basic fibroblast growth factor, cathepsin K, and CD44
receptors, respectively, are investigated using six different explicit
and five implicit water models. The selection of protein&#x02013;GAG
complexes was deliberate, considering their diverse ranges of binding
affinity. The basic fibroblast factor&#x02013;HP complex exhibits a
very strong binding. In contrast, the cathepsin K&#x02013;CS complex
involves moderate binding affinity, while the CD44&#x02013;HA complex
formation is driven by weak interactions. These three complexes therefore
provide a spectrum of binding strengths. The cathepsin K&#x02013;CS
complex has two binding poses, as observed experimentally in crystal
structures (PDB IDs: <a href="https://www.rcsb.org/structure/4N8W" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">4N8W</a> and <a href="https://www.rcsb.org/structure/3C9E" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">3C9E</a>).
This adds an intriguing dimension, prompting an investigation of whether
two binding poses can be distinguished in the MD simulations using
various water models. The exploration of such changes can offer valuable
insights into the dynamic behavior of these systems in MD simulations.</p></div><div id="sec2" class="tsec sec"><h2 class="head no_bottom_margin" id="sec2title">Materials and Methods</h2><p class="p p-first">The initial structure of the
protein&#x02013;GAG complexes used
in this study was obtained from the Protein Data Bank (PDB IDs: <a href="https://www.rcsb.org/structure/1BFC" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">1BFC</a>, 2.20 &#x000c5;; <a href="https://www.rcsb.org/structure/4N8W" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">4N8W</a>, 2.02 &#x000c5;; <a href="https://www.rcsb.org/structure/3C9E" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">3C9E</a>, 1.80 &#x000c5;; 2JCQ,
1.25 &#x000c5;).</p><div id="sec2.1" class="sec"><h3 id="sec2.1title">Water Models</h3><p class="p p-first-last">All of the parameters for the water models
used in this work are taken from Amber20,<sup><a href="#ref46" rid="ref46" class=" bibr popnode">46</a></sup> and recommended mbondii are used for each particular model.<sup><a href="#ref47" rid="ref47" class=" bibr popnode">47</a></sup> Explicit: TIP3P,<sup><a href="#ref48" rid="ref48" class=" bibr popnode">48</a>,<a href="#ref49" rid="ref49" class=" bibr popnode">49</a></sup> SPC/E,<sup><a href="#ref50" rid="ref50" class=" bibr popnode">50</a></sup> TIP4P,<sup><a href="#ref48" rid="ref48" class=" bibr popnode">48</a></sup> TIP4PEw,<sup><a href="#ref51" rid="ref51" class=" bibr popnode">51</a></sup> OPC,<sup><a href="#ref41" rid="ref41" class=" bibr popnode">41</a></sup> and TIP5P,<sup><a href="#ref52" rid="ref52" class=" bibr popnode">52</a></sup> and implicit IGB = 1,<sup><a href="#ref53" rid="ref53" class=" bibr popnode">53</a></sup> 2,<sup><a href="#ref54" rid="ref54" class=" bibr popnode">54</a></sup> 5,<sup><a href="#ref55" rid="ref55" class=" bibr popnode">55</a></sup> 7,<sup><a href="#ref56" rid="ref56" class=" bibr popnode">56</a></sup> and 8<sup><a href="#ref35" rid="ref35" class=" bibr popnode">35</a></sup> water models
are used.</p></div><div id="sec2.2" class="sec"><h3 id="sec2.2title">MD Simulations</h3><p class="p p-first-last">All-atom MD simulations of the protein&#x02013;GAG
complexes are performed in the AMBER20 package. FF14SB and GLYCAM06
force field parameters were used for the protein and GAGs, respectively.
In the case of explicit solvent models, a truncated octahedron water
box of the respective water model with a 12 &#x000c5; distance from the
solute to the box&#x02019;s border is used to solvate complexes. Na<sup>+</sup>/Cl<sup>&#x02013;</sup> counterions are used to neutralize
the charge of the system. No &#x0201c;saltcon&#x0201d; option is used
in the implicit solvent simulations. Two steps of energy minimization
are performed: first 500 steepest descents and then 1000 conjugate
gradient cycles with harmonic restraints of 100 kcal/mol/&#x000c5;<sup>2</sup> on the solute, followed by 3000 steepest descent and 3000
conjugate gradient cycles without restraints for the explicit solvent
simulations, while only the second minimization step is performed
for the implicit solvent simulations. Then, the system is heated to
300 K for 10 ps with harmonic restraints of 100 kcal/mol/&#x000c5;<sup>2</sup> on the solute and equilibrated for 100 ps at 300 K and 10<sup>5</sup> Pa in an isothermal isobaric ensemble (NTP) for the explicit
solvent simulation. Afterward, MD simulations are carried out in the
same NTP ensemble for 10 &#x003bc;s. A 2 fs time integration step and
an 8 &#x000c5; cutoff for nonbonded interactions are used. The SHAKE
algorithm for all of the covalent bonds containing hydrogen atoms<sup><a href="#ref57" rid="ref57" class=" bibr popnode">57</a></sup> and the Particle Mesh Ewald method to treat
electrostatics were used.<sup><a href="#ref58" rid="ref58" class=" bibr popnode">58</a></sup> The trajectories
are analyzed by the cpptraj module of AmberTools.<sup><a href="#ref59" rid="ref59" class=" bibr popnode">59</a></sup> In particular, the <em>native contacts</em> command
with default parameters is used for the analysis of the contacts between
the protein and GAG molecules established during the simulation and
for their comparison with the ones in experimental structures.</p></div><div id="sec2.3" class="sec"><h3 id="sec2.3title">Binding Free Energy Calculations</h3><p class="p p-first-last">Binding free energy
calculations are performed for entire MD trajectories using molecular
mechanics generalized Born surface area (MM/GBSA) and a model with
the surface area and Born radii as default parameters as implemented
in IGB = 2 model<sup><a href="#ref47" rid="ref47" class=" bibr popnode">47</a></sup> in AMBER20.</p></div><div id="sec2.4" class="sec"><h3 id="sec2.4title">GAG&#x02019;s Binding Poses Accuracy Evaluation</h3><p class="p p-first-last">For
the evaluation of the binding pose accuracy in this work, root mean
square deviation (RMSD) and root mean square atom type distance (RMSatD)
values are used. Both RMSD and RMSatD are measures of structural similarity
between different conformations of the biomolecules. RMSatD accounts
for the equivalence of the atoms of the same atomic type and, therefore,
is more appropriate for the periodic GAG molecules than RMSD.<sup><a href="#ref32" rid="ref32" class=" bibr popnode">32</a></sup></p></div><div id="sec2.5" class="sec sec-last"><h3 id="sec2.5title">Data Analysis and Visualization</h3><p class="p p-first-last">The data are analyzed,
and the figures are prepared with the R-package.<sup><a href="#ref60" rid="ref60" class=" bibr popnode">60</a></sup> Structures and trajectories are analyzed with VMD.<sup><a href="#ref61" rid="ref61" class=" bibr popnode">61</a></sup></p></div></div><div id="sec3" class="tsec sec"><h2 class="head no_bottom_margin" id="sec3title">Results</h2><div id="sec3.1" class="sec sec-first"><h3 id="sec3.1title">Basic Fibroblast Factor&#x02013;HP Complex</h3><p class="p p-first">The structural
analysis reveals a similarity in the structures of the basic fibroblast
factor&#x02013;HP complex at the simulation&#x02019;s beginning and
end when utilizing water models such as TIP3P, TIP4P, TIP4PEw, and
SPC/E (<a href="/pmc/articles/PMC10934818/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure <a href="/pmc/articles/PMC10934818/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure1</span></span><span>1</span></a></span></a>). In
contrast, a notable deviation in the ligand binding at the beginning
and end of the simulation is observed specifically for the simulations
using TIP5P and the OPC water models. This disparity underlines the
sensitivity of the simulation outcome to the choice of water models.
The simulations involving the basic fibroblast factor&#x02013;HP complex
exhibited more pronounced alterations when implicit water models were
used. In contrast to the use of explicit water models, where the protein
structure remained stable throughout the simulation, the use of implicit
water models induced notable changes in the protein&#x02019;s conformation,
especially in the simulation using the IGB = 7 water model.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC10934818/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140421135779472"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10934818_ci4c00030_0001.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ci4c00030_0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10934818/bin/ci4c00030_0001.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140421135779472"><a target="object" rel="noopener" href="/pmc/articles/PMC10934818/figure/fig1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC10934818/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure 1</a></div><!--caption a7--><div class="caption"><p>HP starting
(in red, licorice) and final (in blue, licorice) poses
in the complex with the basic fibroblast factor (in silver, new cartoon).</p></div></div></div></div><div id="sec3.2" class="sec"><h3 id="sec3.2title">Analysis of the Complex Stability</h3><p class="p p-first">The RMSD analysis
provides a comprehensive assessment of the changes in the protein
and ligand structures with respect to the starting structure in the
MD simulations with different water models (<a href="/pmc/articles/PMC10934818/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure <a href="/pmc/articles/PMC10934818/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure2</span></span><span>2</span></a></span></a>). In the case of TIP3P, after 2 &#x003bc;s
of the simulation, the ligand shows remarkably high RMSD, indicating
a notable change in the binding pose of the HP, and after 8 &#x003bc;s,
the RMSD drops to lower magnitudes, which indicates the arrangement
of the ligand with the protein in a very similar way to that at the
beginning of the simulation. For the simulations using TIP4P and TIP4PEw
water models, the RMSD of the ligand is in a similar range of values,
indicating that the arrangement of the ligand with respect to the
protein is not significantly altered. In the case of simulations using
the TIP5P water model, very low RMSD is observed for the ligand up
to 8 &#x003bc;s and then shows a sudden increase. With the OPC water
model, the MD simulation shows more frequent fluctuations in terms
of RMSD, while in the simulation with the SPC/E water model, the ligand
orientation gradually changes from the beginning of the simulation.
RMSD of the protein was around 2 &#x000c5; using explicit water models,
which shows that the protein is stable in the entire simulation (<a href="https://pubs.acs.org/doi/suppl/10.1021/acs.jcim.4c00030/suppl_file/ci4c00030_si_001.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Figure S1</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC10934818/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140421154731408"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10934818_ci4c00030_0002.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ci4c00030_0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10934818/bin/ci4c00030_0002.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140421154731408"><a target="object" rel="noopener" href="/pmc/articles/PMC10934818/figure/fig2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC10934818/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Figure 2</a></div><!--caption a7--><div class="caption"><p>RMSD of the ligand obtained for the basic
fibroblast factor&#x02013;HP
complex in MD simulations with explicit water models.</p></div></div></div><p class="p">The RMSD of the ligand from the MD simulation using
implicit water
models (IGB = 1, 2, and 5 models; <a href="/pmc/articles/PMC10934818/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figure <a href="/pmc/articles/PMC10934818/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure3</span></span><span>3</span></a></span></a>) is lower compared to that of the simulations
using IGB = 7 and 8, suggesting that the use of the later ones results
in significant changes in the binding pose. The RMSD of the protein
from the MD simulation using implicit water models showed that the
protein structure is significantly altered in most of them (<a href="https://pubs.acs.org/doi/suppl/10.1021/acs.jcim.4c00030/suppl_file/ci4c00030_si_001.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Figure S2</a>). In the case of the MD simulation
with IGB = 1, both the ligand and the protein showed low values of
RMSD (4.7 &#x000b1; 0.6 and 2.6 &#x000b1; 0.4 &#x000c5;, for the ligand and
protein, respectively), indicating the complex stability. For the
MD simulation with IGB = 2 and 5, both the ligand and protein deviate
more from their starting structures than in the simulation with IGB
= 1. The simulations with IGB = 7 and 8 showed a maximum RMSD for
both the ligand and protein. The RMSDs for the ligand and protein
are provided in <a href="https://pubs.acs.org/doi/suppl/10.1021/acs.jcim.4c00030/suppl_file/ci4c00030_si_001.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S1</a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig3" co-legend-rid="lgnd_fig3"><a href="/pmc/articles/PMC10934818/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140421133842944"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10934818_ci4c00030_0003.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ci4c00030_0003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10934818/bin/ci4c00030_0003.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140421133842944"><a target="object" rel="noopener" href="/pmc/articles/PMC10934818/figure/fig3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig3"><div><a class="figpopup" href="/pmc/articles/PMC10934818/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3">Figure 3</a></div><!--caption a7--><div class="caption"><p>RMSD of the ligand for
the basic fibroblast factor&#x02013;HP complex
in MD simulations with implicit water models.</p></div></div></div></div><div id="sec3.3" class="sec"><h3 id="sec3.3title">Binding Free Energy Analysis</h3><p class="p p-first">To estimate the effect
of these structural variations on the stability of the complex, the
binding free energy (&#x00394;<em>G</em>) of the complex in
the simulations with various water models has been calculated using
MM/GBSA (<a href="/pmc/articles/PMC10934818/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Figures <a href="/pmc/articles/PMC10934818/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">Figures4</span></span><span>4</span></a></span></a> and <a href="/pmc/articles/PMC10934818/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and5).</span></span><span>5</span></a>). In the case of the MD simulation using TIP3P,
irrespective of the changes in RMSD, &#x00394;<em>G</em> showed
a similar range of values during the entire 10 &#x003bc;s simulation.
For MD simulations using TIP4P and TIP4Pew, &#x00394;<em>G</em> becomes less favorable with large RMSD observed for the ligand.
Even the small variations in RMSD are also reflected in the MM/GBSA
data with a corresponding change in &#x00394;<em>G</em>. For
simulations using TIP5P and SPC/E, higher deviations from the initial
structure correspond to the more favorable free energy of binding.
For the simulation with the OPC water model, &#x00394;<em>G</em> is of higher variance, which agrees with the more fluctuating RMSD
observed for the ligand.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig4" co-legend-rid="lgnd_fig4"><a href="/pmc/articles/PMC10934818/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140421133621536"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10934818_ci4c00030_0004.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ci4c00030_0004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10934818/bin/ci4c00030_0004.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140421133621536"><a target="object" rel="noopener" href="/pmc/articles/PMC10934818/figure/fig4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig4"><div><a class="figpopup" href="/pmc/articles/PMC10934818/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4">Figure 4</a></div><!--caption a7--><div class="caption"><p>&#x00394;<em>G</em> obtained using MM/GBSA
analysis for the
basic fibroblast factor&#x02013;HP complex in MD simulations with explicit
water models.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig5" co-legend-rid="lgnd_fig5"><a href="/pmc/articles/PMC10934818/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140421153732144"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10934818_ci4c00030_0005.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ci4c00030_0005.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10934818/bin/ci4c00030_0005.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140421153732144"><a target="object" rel="noopener" href="/pmc/articles/PMC10934818/figure/fig5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig5"><div><a class="figpopup" href="/pmc/articles/PMC10934818/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5">Figure 5</a></div><!--caption a7--><div class="caption"><p>&#x00394;<em>G</em> for the basic fibroblast factor&#x02013;HP
complex in MD simulations with implicit water models.</p></div></div></div><p class="p p-last">The MD simulations using implicit water models
showed a more favorable
&#x00394;<em>G</em> (<a href="/pmc/articles/PMC10934818/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Figure <a href="/pmc/articles/PMC10934818/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure5</span></span><span>5</span></a></span></a>) than that observed with explicit water models. For
the simulation using IGB = 1, &#x00394;<em>G</em> becomes more
favorable with the increase of RMSD. For the simulation using IGB
= 2, 5, and 7, high RMSD corresponded to more favorable &#x00394;<em>G</em>, suggesting complex stabilization. The MD simulation with
water model IGB = 8 yields high RMSD for the ligand and less favorable
&#x00394;<em>G</em>, suggesting destabilization of the complex.</p></div><div id="sec3.4" class="sec"><h3 id="sec3.4title">Analysis of Contacts and H-Bonds</h3><p class="p p-first">Analysis of contacts
(<a href="/pmc/articles/PMC10934818/figure/fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig6" rid-ob="ob-fig6" co-legend-rid="lgnd_fig6"><span>Figures <a href="/pmc/articles/PMC10934818/figure/fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig6" rid-ob="ob-fig6" co-legend-rid="lgnd_fig6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">Figures6</span></span><span>6</span></a></span></a> and <a href="/pmc/articles/PMC10934818/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and7)</span></span><span>7</span></a>) showed a trend similar to that observed from the
RMSD and MM/GBSA analyses for the simulations using explicit water
models. Especially the abrupt changes in these parameters observed
with TIP3P and TIP5P water models are reflected in the change in the
number of contacts. Similarly, the H-bond analysis also shows a lower
number of H-bonds with increased RMSD values. The stabilization effects
for the complex with the TIP5P model with increased RMSD can therefore
be explained by the increased number of H-bonds (<a href="https://pubs.acs.org/doi/suppl/10.1021/acs.jcim.4c00030/suppl_file/ci4c00030_si_001.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Figure S3</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig6" co-legend-rid="lgnd_fig6"><a href="/pmc/articles/PMC10934818/figure/fig6/" target="figure" rid-figpopup="fig6" rid-ob="ob-fig6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140421135550336"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10934818_ci4c00030_0006.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ci4c00030_0006.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10934818/bin/ci4c00030_0006.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140421135550336"><a target="object" rel="noopener" href="/pmc/articles/PMC10934818/figure/fig6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig6"><div><a class="figpopup" href="/pmc/articles/PMC10934818/figure/fig6/" target="figure" rid-figpopup="fig6" rid-ob="ob-fig6">Figure 6</a></div><!--caption a7--><div class="caption"><p>Number of contacts obtained for the basic fibroblast factor&#x02013;HP
complex in MD simulations with explicit water models.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig7" co-legend-rid="lgnd_fig7"><a href="/pmc/articles/PMC10934818/figure/fig7/" target="figure" rid-figpopup="fig7" rid-ob="ob-fig7"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140421193222656"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10934818_ci4c00030_0007.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ci4c00030_0007.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10934818/bin/ci4c00030_0007.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140421193222656"><a target="object" rel="noopener" href="/pmc/articles/PMC10934818/figure/fig7/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig7"><div><a class="figpopup" href="/pmc/articles/PMC10934818/figure/fig7/" target="figure" rid-figpopup="fig7" rid-ob="ob-fig7">Figure 7</a></div><!--caption a7--><div class="caption"><p>Number of contacts obtained for the basic fibroblast factor&#x02013;HP
complex in MD simulations with implicit water models.</p></div></div></div><p>In the case of MD simulations using implicit water
models, the
stabilization of the complex can be explained as a result of the increased
number of contacts and H-bonds (<a href="https://pubs.acs.org/doi/suppl/10.1021/acs.jcim.4c00030/suppl_file/ci4c00030_si_001.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Figure S4</a>). After the initial drop in the number of native contacts, for the
simulations using IGB = 1, 2, and 5, the number of native contacts
remains steady, whereas the number of non-native contacts increases
gradually. In the case of the simulation using IGB = 7, although the
number of native contacts drops to almost zero after 1 &#x003bc;s, the
number of non-native contacts increases, resulting in the total stabilization
of the complex. For the simulations with IGB = 1, 5, and 7, the number
of H-bonds also increased toward the end of the simulation, while
with IGB = 2, the number of H-bonds remained at the same level. On
the other hand, the lowest number of total contacts as well as the
least number of H-bonds are observed for the simulation using IGB
= 8, which explains the lowest &#x00394;<em>G</em> observed
for this water model.</p><p class="p p-last">In summary, for the basic fibroblast factor&#x02013;HP
complex,
any major change observed in the RMSD of the ligand corresponded to
the &#x00394;<em>G</em> variation for most of the water models.
However, in the case of the simulation using the TIP3P water model,
irrespective of the RMSD fluctuations, the &#x00394;<em>G</em> of the system remains similar. For the simulations using TIP4P,
TIP4PEw, and OPC water models, more deviation from the initial structure
is found to be destabilizing in terms of &#x00394;<em>G</em>. In the MD simulations using TIP5P and SPC/E water models, the more
the deviations from the initial structure, the more the stabilization,
and this agrees with the increased number of H-bonds and contacts.
The simulations using implicit water models did not reveal any such
relationship in the stability of the complex in terms of &#x00394;<em>G</em> and the increase of RMSD; instead, most of them showed
more stabilization of the complex irrespective of the RMSD, especially
for the simulation with the IGB = 7 water model. Only the MD simulation
with the IGB = 8 model showed less favorable &#x00394;<em>G</em> corresponding to the significant structural changes observed.</p></div><div id="sec3.5" class="sec"><h3 id="sec3.5title">Cathepsin K&#x02013;CS Complex</h3><p class="p p-first">For the cathepsin K&#x02013;CS
complex, with the initial structure corresponding to the <a href="https://www.rcsb.org/structure/4N8W" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">4N8W</a> binding pose, the
complex is stable during the MD simulations using the water models
TIP3P, OPC, IGB = 1, and IGB = 2, whereas with TIP4P, TIP4PEw, TIP5P,
SPC/E, and IGB = 5, 7, and 8, the binding pose is essentially altered
from the initial structure (<a href="/pmc/articles/PMC10934818/figure/fig8/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig8" rid-ob="ob-fig8" co-legend-rid="lgnd_fig8"><span>Figure <a href="/pmc/articles/PMC10934818/figure/fig8/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig8" rid-ob="ob-fig8" co-legend-rid="lgnd_fig8"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure8</span></span><span>8</span></a></span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig8" co-legend-rid="lgnd_fig8"><a href="/pmc/articles/PMC10934818/figure/fig8/" target="figure" rid-figpopup="fig8" rid-ob="ob-fig8"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140421156708544"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10934818_ci4c00030_0008.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ci4c00030_0008.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10934818/bin/ci4c00030_0008.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140421156708544"><a target="object" rel="noopener" href="/pmc/articles/PMC10934818/figure/fig8/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig8"><div><a class="figpopup" href="/pmc/articles/PMC10934818/figure/fig8/" target="figure" rid-figpopup="fig8" rid-ob="ob-fig8">Figure 8</a></div><!--caption a7--><div class="caption"><p>CS starting (red, licorice) and final (blue, licorice)
poses in
the complex with cathepsin K (tan, new cartoon). The initial structure
corresponds to the <a href="https://www.rcsb.org/structure/4N8W" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">4N8W</a> binding pose.</p></div></div></div></div><div id="sec3.6" class="sec"><h3 id="sec3.6title">Analysis of the Complex Stability</h3><p class="p p-first">The predominant structural
difference emerges in the directionality of the ligand (CS) in the
specific binding pose. This can be considered as an outcome of the
coexistence of two binding poses observed for the cathepsin K&#x02013;CS
complex experimentally (PDB IDs: <a href="https://www.rcsb.org/structure/4N8W" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">4N8W</a> and <a href="https://www.rcsb.org/structure/3C9E" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">3C9E</a>). Considering this, we calculated the
RMSatD for the final structures of the simulation with references
to both <a href="https://www.rcsb.org/structure/4N8W" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">4N8W</a> and <a href="https://www.rcsb.org/structure/3C9E" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">3C9E</a> structures.
The RMSatD values provided in <a href="#tbl1" rid="tbl1" class=" other">Table <a href="#tbl1" rid="tbl1" class=" other">1</a></a> clearly indicate that the final structure is more
similar to the binding pose <a href="https://www.rcsb.org/structure/3C9E" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">3C9E</a> in the MD simulations with most of the used water
models except TIP4PEw. The highest RMSatDs are observed (for both <a href="https://www.rcsb.org/structure/4N8W" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">4N8W</a> and <a href="https://www.rcsb.org/structure/3C9E" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">3C9E</a>) in the simulations
with the TIP4PEw model. Lower RMSatD values with reference to the <a href="https://www.rcsb.org/structure/4N8W" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">4N8W</a> structure are obtained
for the simulations with water models SPC/E, TIP3P, and TIP5P.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>RMSatD (&#x000c5;) of the Cathepsin K&#x02013;CS
Complex with <a href="https://www.rcsb.org/structure/4N8W" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">4N8W</a> and <a href="https://www.rcsb.org/structure/3C9E" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">3C9E</a></strong></div><div class="xtable"><table frame="hsides" rules="groups" border="0" class="rendered small default_table"><thead><tr><th colspan="3" align="center" rowspan="1">explicit
water models<hr /></th><th colspan="3" align="center" rowspan="1">implicit
water models<hr /></th></tr><tr><th style="border:none;" align="center" rowspan="1" colspan="1">model</th><th style="border:none;" align="center" rowspan="1" colspan="1"><a href="https://www.rcsb.org/structure/4N8W" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">4N8W</a></th><th style="border:none;" align="center" rowspan="1" colspan="1"><a href="https://www.rcsb.org/structure/3C9E" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">3C9E</a></th><th style="border:none;" align="center" rowspan="1" colspan="1">model</th><th style="border:none;" align="center" rowspan="1" colspan="1"><a href="https://www.rcsb.org/structure/4N8W" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">4N8W</a></th><th style="border:none;" align="center" rowspan="1" colspan="1"><a href="https://www.rcsb.org/structure/3C9E" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">3C9E</a></th></tr></thead><tbody><tr><td style="border:none;" align="left" rowspan="1" colspan="1">TIP3P</td><td style="border:none;" align="center" rowspan="1" colspan="1">8.5</td><td style="border:none;" align="center" rowspan="1" colspan="1">6.4</td><td style="border:none;" align="left" rowspan="1" colspan="1">IGB&#x000a0;=&#x000a0;1</td><td style="border:none;" align="center" rowspan="1" colspan="1">8.4</td><td style="border:none;" align="center" rowspan="1" colspan="1">5.0</td></tr><tr><td style="border:none;" align="left" rowspan="1" colspan="1">TIP4P</td><td style="border:none;" align="center" rowspan="1" colspan="1">9.6</td><td style="border:none;" align="center" rowspan="1" colspan="1">5.7</td><td style="border:none;" align="left" rowspan="1" colspan="1">IGB&#x000a0;=&#x000a0;2</td><td style="border:none;" align="center" rowspan="1" colspan="1">9.1</td><td style="border:none;" align="center" rowspan="1" colspan="1">3.4</td></tr><tr><td style="border:none;" align="left" rowspan="1" colspan="1">TIP4PEw</td><td style="border:none;" align="center" rowspan="1" colspan="1">10.6</td><td style="border:none;" align="center" rowspan="1" colspan="1">10.8</td><td style="border:none;" align="left" rowspan="1" colspan="1">IGB&#x000a0;=&#x000a0;5</td><td style="border:none;" align="center" rowspan="1" colspan="1">7.3</td><td style="border:none;" align="center" rowspan="1" colspan="1">7.0</td></tr><tr><td style="border:none;" align="left" rowspan="1" colspan="1">TIP5P</td><td style="border:none;" align="center" rowspan="1" colspan="1">8.8</td><td style="border:none;" align="center" rowspan="1" colspan="1">7.3</td><td style="border:none;" align="left" rowspan="1" colspan="1">IGB&#x000a0;=&#x000a0;7</td><td style="border:none;" align="center" rowspan="1" colspan="1">16.4</td><td style="border:none;" align="center" rowspan="1" colspan="1">14.1</td></tr><tr><td style="border:none;" align="left" rowspan="1" colspan="1">OPC</td><td style="border:none;" align="center" rowspan="1" colspan="1">10.3</td><td style="border:none;" align="center" rowspan="1" colspan="1">5.0</td><td style="border:none;" align="left" rowspan="1" colspan="1">IGB&#x000a0;=&#x000a0;8</td><td style="border:none;" align="center" rowspan="1" colspan="1">8.8</td><td style="border:none;" align="center" rowspan="1" colspan="1">6.9</td></tr><tr><td style="border:none;" align="left" rowspan="1" colspan="1">SPC/E</td><td style="border:none;" align="center" rowspan="1" colspan="1">7.5</td><td style="border:none;" align="center" rowspan="1" colspan="1">6.6</td><td style="border:none;" align="left" rowspan="1" colspan="1">&#x000a0;</td><td style="border:none;" align="center" rowspan="1" colspan="1">&#x000a0;</td><td style="border:none;" align="center" rowspan="1" colspan="1">&#x000a0;</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140421135933792"><a target="object" rel="noopener" href="/pmc/articles/PMC10934818/table/tbl1/?report=objectonly">Open in a separate window</a></div></div><p class="p p-last">As it was observed with the explicit solvents, the
simulations
of cathepsin K&#x02013;CS with implicit solvent also result in different
binding poses more similar to the <a href="https://www.rcsb.org/structure/3C9E" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">3C9E</a> one. Compared to the simulations with
explicit water models (<a href="https://pubs.acs.org/doi/suppl/10.1021/acs.jcim.4c00030/suppl_file/ci4c00030_si_001.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Figures S5 and S6</a>), those with implicit water result in significant changes in the
protein structure (<a href="https://pubs.acs.org/doi/suppl/10.1021/acs.jcim.4c00030/suppl_file/ci4c00030_si_001.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Figures S7 and S8</a>).
Among the simulations with implicit water models, more changes in
the protein structure are observed for the ones with IGB = 7 and 8
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acs.jcim.4c00030/suppl_file/ci4c00030_si_001.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S2</a>), which appear shortly after
the start of the simulation (<a href="https://pubs.acs.org/doi/suppl/10.1021/acs.jcim.4c00030/suppl_file/ci4c00030_si_001.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Figures S7 and S8</a>). High RMSatD obtained for the MD simulation using IGB = 7 indicates
significant differences from both crystal poses for this solvent model.</p></div><div id="sec3.7" class="sec"><h3 id="sec3.7title">Binding Free Energy Analysis</h3><p class="p p-first">MM/GBSA analysis shows
a more favorable &#x00394;<em>G</em> corresponding to higher
RMSD values (<a href="/pmc/articles/PMC10934818/figure/fig9/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig9" rid-ob="ob-fig9" co-legend-rid="lgnd_fig9"><span>Figures <a href="/pmc/articles/PMC10934818/figure/fig9/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig9" rid-ob="ob-fig9" co-legend-rid="lgnd_fig9"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">Figures9</span></span><span>9</span></a></span></a> and <a href="/pmc/articles/PMC10934818/figure/fig10/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig10" rid-ob="ob-fig10" co-legend-rid="lgnd_fig10"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and10).</span></span><span>10</span></a>). For the MD simulations using TIP4P,
TIP4PEw, and TIP5P water models, as simulation extends, the RMSD (with
respect to the starting structure) increases, and this structural
deviation is found to have stabilizing effects on the complex reflected
in more favorable &#x00394;<em>G</em>. In the case of simulations
using TIP3P, OPC, and SPC/E water models, the &#x00394;<em>G</em> remains in a similar range with an average of &#x02212;107.1 &#x000b1;
12.9, &#x02212;92.2 &#x000b1; 15.3, and &#x02212;115.9 &#x000b1; 14.4 kcal/mol,
respectively, independently of the structural changes. Among the simulations
using implicit water models, slight RMSD increases correspond to minor
unfavorable changes of &#x00394;<em>G</em> for IGB = 1, 2, and
5. In the case of simulations with IGB = 7 and 8, more significant
unfavorable changes are observed for &#x00394;<em>G</em>. The
variation in the &#x00394;<em>G</em> for the simulation using
the IGB = 7 water model indicates the relative instability of the
complex compared to that observed in the simulations with other water
models.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig9" co-legend-rid="lgnd_fig9"><a href="/pmc/articles/PMC10934818/figure/fig9/" target="figure" rid-figpopup="fig9" rid-ob="ob-fig9"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140421135414736"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10934818_ci4c00030_0009.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ci4c00030_0009.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10934818/bin/ci4c00030_0009.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140421135414736"><a target="object" rel="noopener" href="/pmc/articles/PMC10934818/figure/fig9/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig9"><div><a class="figpopup" href="/pmc/articles/PMC10934818/figure/fig9/" target="figure" rid-figpopup="fig9" rid-ob="ob-fig9">Figure 9</a></div><!--caption a7--><div class="caption"><p>&#x00394;<em>G</em> for the cathepsin K&#x02013;CS complex
in MD simulations with explicit water models.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig10" co-legend-rid="lgnd_fig10"><a href="/pmc/articles/PMC10934818/figure/fig10/" target="figure" rid-figpopup="fig10" rid-ob="ob-fig10"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140421135411040"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10934818_ci4c00030_0010.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ci4c00030_0010.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10934818/bin/ci4c00030_0010.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140421135411040"><a target="object" rel="noopener" href="/pmc/articles/PMC10934818/figure/fig10/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig10"><div><a class="figpopup" href="/pmc/articles/PMC10934818/figure/fig10/" target="figure" rid-figpopup="fig10" rid-ob="ob-fig10">Figure 10</a></div><!--caption a7--><div class="caption"><p>&#x00394;<em>G</em> obtained for the cathepsin K&#x02013;CS
complex in MD simulations with implicit water models.</p></div></div></div></div><div id="sec3.8" class="sec"><h3 id="sec3.8title">Analysis of Contacts and H-Bonds</h3><p class="p p-first">The analysis of the
number of contacts shows that shortly after the beginning of the simulation,
the number of native contacts declines to zero, and the number of
non-native contacts increases, particularly in the case of simulations
using TIP4P and TIP5P water models (<a href="https://pubs.acs.org/doi/suppl/10.1021/acs.jcim.4c00030/suppl_file/ci4c00030_si_001.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Figure S9</a>). A very similar trend can be observed for the number of H-bonds:
particularly, the simulation with the TIP4P water model is marked
with the significantly increased number of H-bonds with the course
of the simulation (<a href="https://pubs.acs.org/doi/suppl/10.1021/acs.jcim.4c00030/suppl_file/ci4c00030_si_001.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Figure S11</a>). In the
case of the MD simulations using implicit solvents, the number of
contacts and H-bonds (<a href="https://pubs.acs.org/doi/suppl/10.1021/acs.jcim.4c00030/suppl_file/ci4c00030_si_001.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Figures S10 and S12</a>) also follows the same trend observed for RMSD and &#x00394;<em>G</em>.</p><p class="p p-last">In the case of the moderately interacting cathepsin
K&#x02013;CS complex, the binding pose at the end of the simulation
differs significantly from the starting one. The RMSatD analysis of
the simulations of the cathepsin K&#x02013;CS complex for most water
models suggests a higher propensity toward the <a href="https://www.rcsb.org/structure/3C9E" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">3C9E</a> pose than to the
initial <a href="https://www.rcsb.org/structure/4N8W" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">4N8W</a> pose. Similarly, the binding energy analysis showed that the final
binding mode of the complex is more energetically favorable than that
of the starting structure. The contact analysis showed that toward
the end of the simulation, the number of native contacts almost declined
to zero, with a substantial increase in the number of non-native contacts
with respect to the <a href="https://www.rcsb.org/structure/4N8W" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">4N8W</a> structure. The MD simulation using the TIP4P water model showed
the most notable increase in the number of non-native contacts and
the number of H-bonds. In the case of simulations with explicit water
models, the &#x00394;<em>G</em> changes correspond to the contact
number changes, but for the implicit water model such a trend was
not observed. Despite having a much larger number of non-native contacts
than that observed in the simulations with other water models, the
simulation using IGB = 7 does not show a significantly more favorable
&#x00394;<em>G</em>.</p></div><div id="sec3.9" class="sec"><h3 id="sec3.9title">CD44&#x02013;HA Complex</h3><p class="p p-first">In the case of the CD44&#x02013;HA
complex, MD simulations with most of the water models show that the
complex dissociates after a few microseconds. A 10 &#x003bc;s long MD
simulation of the CD44&#x02013;HA complex did not lead to the dissociation
only for TIP3P, TIP5P, and IGB = 1 and 8 water models. This may be
because of the weaker affinity in this complex or the imperfections
of the force fields used or due to the limitations of the X-ray structure.
The CD44&#x02013;HA complex is supposed to represent a case of multipose
binding, as it was previously predicted by Vuorio et al.<sup><a href="#ref62" rid="ref62" class=" bibr popnode">62</a></sup><a href="/pmc/articles/PMC10934818/figure/fig11/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig11" rid-ob="ob-fig11" co-legend-rid="lgnd_fig11"><span>Figure <a href="/pmc/articles/PMC10934818/figure/fig11/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig11" rid-ob="ob-fig11" co-legend-rid="lgnd_fig11"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure11</span></span><span>11</span></a></span></a> shows the binding of the ligand (HA 8-mer) with CD44 in the
simulations using TIP3P, TIP5P, and IGB = 1 and 8 water models.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig11" co-legend-rid="lgnd_fig11"><a href="/pmc/articles/PMC10934818/figure/fig11/" target="figure" rid-figpopup="fig11" rid-ob="ob-fig11"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140421134894816"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10934818_ci4c00030_0011.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ci4c00030_0011.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10934818/bin/ci4c00030_0011.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140421134894816"><a target="object" rel="noopener" href="/pmc/articles/PMC10934818/figure/fig11/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig11"><div><a class="figpopup" href="/pmc/articles/PMC10934818/figure/fig11/" target="figure" rid-figpopup="fig11" rid-ob="ob-fig11">Figure 11</a></div><!--caption a7--><div class="caption"><p>HA starting
(red, licorice) and final (blue, licorice) poses in
the binding site of CD44 (in brown, new cartoon).</p></div></div></div></div><div id="sec3.10" class="sec"><h3 id="sec3.10title">Analysis of the Complex Stability</h3><p class="p p-first">RMSD with respect
to the starting structure was calculated for the ligand (<a href="/pmc/articles/PMC10934818/figure/fig12/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig12" rid-ob="ob-fig12" co-legend-rid="lgnd_fig12"><span>Figure <a href="/pmc/articles/PMC10934818/figure/fig12/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig12" rid-ob="ob-fig12" co-legend-rid="lgnd_fig12"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure12</span></span><span>12</span></a></span></a>) and the protein
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acs.jcim.4c00030/suppl_file/ci4c00030_si_001.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Figure S13</a>). The simulation using the
TIP3P water model showed that the RMSD of the ligand increases to
a remarkably high value (&#x0003e;20 &#x000c5;) at the beginning of the simulation
and remains in the same range. In the simulation using the TIP5P water
model, the RMSD of the ligand was remarkably low (&#x0003c;5 &#x000c5;) up
to 3 &#x003bc;s and then increased to 10 &#x000c5; remaining near this
value. In the simulations with the implicit model IGB = 1, the RMSD
of the ligand was even higher from the beginning of the simulation
(&#x0003e;20 &#x000c5;) while IGB = 8 showed a lower magnitude of RMSD for
the
ligand with an average of 11.5 &#x000b1; 1.4 &#x000c5;.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig12" co-legend-rid="lgnd_fig12"><a href="/pmc/articles/PMC10934818/figure/fig12/" target="figure" rid-figpopup="fig12" rid-ob="ob-fig12"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140421152921040"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10934818_ci4c00030_0012.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ci4c00030_0012.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10934818/bin/ci4c00030_0012.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140421152921040"><a target="object" rel="noopener" href="/pmc/articles/PMC10934818/figure/fig12/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig12"><div><a class="figpopup" href="/pmc/articles/PMC10934818/figure/fig12/" target="figure" rid-figpopup="fig12" rid-ob="ob-fig12">Figure 12</a></div><!--caption a7--><div class="caption"><p>RMSD of the ligand in
the CD44&#x02013;HA complex.</p></div></div></div><p class="p p-last">Compared to the ligand, the protein is found to
be more stable
in the simulations of both TIP3P and TIP5P water models with RMSD
of 4.5 &#x000b1; 0.6 and 3.8 &#x000b1; 0.8 &#x000c5;, respectively. In the
MD simulations using implicit models IGB = 1 and IGB = 8, the protein
showed slightly higher deviations from the starting structure than
in the case of explicit models with RMSD of 6.0 &#x000b1; 0.6 and 9.3
&#x000b1; 1.6 &#x000c5;, respectively. Thus, in comparison to other water
models, the simulation with TIP5P showed the lowest deviations from
the starting structure for the experimental binding pose (<a href="https://pubs.acs.org/doi/suppl/10.1021/acs.jcim.4c00030/suppl_file/ci4c00030_si_001.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S3</a>).</p></div><div id="sec3.11" class="sec"><h3 id="sec3.11title">Binding Free Energy Analysis</h3><p class="p p-first">The MM/GBSA analysis of
the CD44&#x02013;HA complex (<a href="/pmc/articles/PMC10934818/figure/fig13/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig13" rid-ob="ob-fig13" co-legend-rid="lgnd_fig13"><span>Figure <a href="/pmc/articles/PMC10934818/figure/fig13/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig13" rid-ob="ob-fig13" co-legend-rid="lgnd_fig13"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure13</span></span><span>13</span></a></span></a>) reveals less favorable &#x00394;<em>G</em> values
for the interaction in this complex CD44 and HA compared to the other
two complexes investigated in this study. This explains the observed
dissociation of the CD44&#x02013;HA complex in the simulations. Notably,
during MD simulations employing the explicit water model TIP3P, increasing
structural deviations (at the very beginning of the simulation) corresponded
to a less favorable &#x00394;<em>G</em>, and after this, when
the RMSD converged steadily, more favorable &#x00394;<em>G</em> was observed toward the end of the simulation. For the simulation
using the TIP5P model, a significant increase in RMSD around the 4
&#x003bc;s correlates with less favorable &#x00394;<em>G</em>.
The simulations with implicit models also demonstrate less favorable
&#x00394;<em>G</em> values as RMSD increases.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig13" co-legend-rid="lgnd_fig13"><a href="/pmc/articles/PMC10934818/figure/fig13/" target="figure" rid-figpopup="fig13" rid-ob="ob-fig13"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140421134509424"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10934818_ci4c00030_0013.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ci4c00030_0013.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10934818/bin/ci4c00030_0013.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140421134509424"><a target="object" rel="noopener" href="/pmc/articles/PMC10934818/figure/fig13/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig13"><div><a class="figpopup" href="/pmc/articles/PMC10934818/figure/fig13/" target="figure" rid-figpopup="fig13" rid-ob="ob-fig13">Figure 13</a></div><!--caption a7--><div class="caption"><p>&#x00394;<em>G</em> obtained for the CD44&#x02013;HA complex.</p></div></div></div></div><div id="sec3.12" class="sec sec-last"><h3 id="sec3.12title">Analysis of Contacts and H-Bonds</h3><p class="p p-first">For the simulation
using the TIP3P water model, the number of native contacts declines
to zero at the beginning of the simulation (<a href="/pmc/articles/PMC10934818/figure/fig14/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig14" rid-ob="ob-fig14" co-legend-rid="lgnd_fig14"><span>Figure <a href="/pmc/articles/PMC10934818/figure/fig14/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig14" rid-ob="ob-fig14" co-legend-rid="lgnd_fig14"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure14</span></span><span>14</span></a></span></a>), corresponding to the increase in RMSD.
Simultaneously, the number of non-native contacts increases, stabilizing
the complex structure. In the simulation using the TIP5P water model,
the number of native contacts drops to zero near the 4 &#x003bc;s, while
the number of non-native contacts increases, reflecting converged
binding energy. In the simulations with implicit water models, although
the number of non-native contacts is remarkably high, the number of
native contacts decreases to almost zero, whereas a high RMSD is observed
for both the ligand and protein. The H-bonds (<a href="https://pubs.acs.org/doi/suppl/10.1021/acs.jcim.4c00030/suppl_file/ci4c00030_si_001.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Figure S14</a>) also showed the same trend as MM/GBSA and native
contacts. Compared to the simulations using TIP3P, TIP5P, and IGB
= 1, the simulation with the IGB = 8 model showed poor correspondence
between the number of contacts or H-bonds to &#x00394;<em>G</em>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig14" co-legend-rid="lgnd_fig14"><a href="/pmc/articles/PMC10934818/figure/fig14/" target="figure" rid-figpopup="fig14" rid-ob="ob-fig14"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140421135002512"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10934818_ci4c00030_0014.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ci4c00030_0014.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10934818/bin/ci4c00030_0014.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140421135002512"><a target="object" rel="noopener" href="/pmc/articles/PMC10934818/figure/fig14/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig14"><div><a class="figpopup" href="/pmc/articles/PMC10934818/figure/fig14/" target="figure" rid-figpopup="fig14" rid-ob="ob-fig14">Figure 14</a></div><!--caption a7--><div class="caption"><p>Number of contacts obtained for the CD44&#x02013;HA complex.</p></div></div></div><p class="p p-last">In the case of the weakly bound CD44&#x02013;HA
complex, the use
of several water models led to the dissociation of HA from the CD44.
No such dissociation was observed only for TIP3P, TIP5P, and IGB =
1 and 8. MD simulations with explicit water models yielded higher
RMSD values and had less favorable &#x00394;<em>G</em> values
than the ones with the implicit water models. Similarly, a significantly
higher number of non-native contacts is established with the implicit
water models than with the explicit water models. Among the explicit
water models, simulation using TIP3P is more stabilizing in terms
of &#x00394;<em>G</em> and in agreement with more H-bonds observed
than that observed with the TIP5P water model (<a href="https://pubs.acs.org/doi/suppl/10.1021/acs.jcim.4c00030/suppl_file/ci4c00030_si_001.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Figure S14</a>). Although TIP5P showed stabilization effects comparable
to that of TIP3P and IGB = 1 and 8 water models, the analysis of H-bonds
showed that the MD simulation with TIP3P and IGB = 1 and 8 yielded
a greater number of H-bonds than the one with TIP5P.</p></div></div><div id="sec4" class="tsec sec"><h2 class="head no_bottom_margin" id="sec4title">Conclusions</h2><p class="p p-first">The comparison of various water models
in the MD simulations of
three analyzed protein&#x02013;GAG complexes shows that the type and
model of the solvent have a substantial effect on their MD simulation
results. The stronger the binding interaction of the protein&#x02013;GAG
complex, the lesser the effect of the water models on the stability
and energetics of the complex. In general, the study showed that the
structural features and energetics of the complexes can be better
understood in the simulations using explicit water models. The trends
observed with the complex stability in terms of RMSD and MM/GBSA binding
free energies were consistent with the observations from the analysis
of the contacts and H-bonds in all three complexes.</p><p>Our investigation
of three representative protein&#x02013;ligand
systems has provided valuable insights into the interplay between
their dynamics and energetics, revealing the diverse effects of water
models on protein&#x02013;GAG interactions. The strongly bound basic
fibroblast factor&#x02013;HP complex exhibited stability, with the
TIP3P water model displaying unique behavior. The use of the TIP4P,
TIP4PEw, and OPC models destabilized the interactions with increased
structural deviations, while TIP5P and SPC/E showed enhanced stabilization
upon structural changes. Implicit water models generally favored stabilization
during the MD simulation, especially IGB = 7, but IGB = 8 yielded
significantly less favorable &#x00394;<em>G</em>. The change
in RMSD or &#x00394;<em>G</em> or number of contacts/H-bonds
of the moderately interacting cathepsin K&#x02013;CS complex were specific
for MD simulations using different explicit water models, while simulations
using implicit water models showed similar trends, with a notable
change in binding mode observed at the end of the simulation. Most
importantly, the use of all of the water models for this complex showed
that the binding pose at the end of the simulation is more similar
to that in <a href="https://www.rcsb.org/structure/3C9E" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">3C9E</a> than to the <a href="https://www.rcsb.org/structure/4N8W" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">4N8W</a> crystal structure used as the starting conformation. In the case
of the relatively unstable CD44&#x02013;HA complex, no dissociation
was observed during simualtions with a few water models, highlighting
the critical role of water models in simulating complex dynamics.</p><p class="p p-last">This systematic investigation, comparing the impact of different
water models on the binding characteristics and stability of three
representative and well-characterized protein&#x02013;GAG systems,
clearly highlights the notable advantages of explicit water models
over their implicit counterparts. The MD simulations using explicit
water models exhibited a remarkable ability to capture distinct features
in terms of RMSD, &#x00394;<em>G</em>, and the number of contacts
and H-bonds. In contrast, MD simulations with different implicit water
models yielded similar outcomes across various metrics used in this
study. In particular, the simulations with implicit water models displayed
reduced variations and fluctuations of the analyzed descriptors for
the CD44&#x02013;HA complex with a low binding affinity. At the same
time, our analysis indicated a tendency of implicit water models to
overestimate the binding energy and the number of non-native contacts.
This convincing evidence highlights the superior performance of explicit
water models in representing the intricate dynamics and stability
profiles of protein&#x02013;GAG interactions. The observed variations
in various descriptors across different simulation conditions emphasize
the importance of the chosen water model for the MD simulation of
protein&#x02013;GAG interactions, thereby highlighting the discrete
nature of their underlying dynamics for accurate analysis and design
of the GAG-based drugs.</p></div><div id="ack-a.x.g" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.x.gtitle">Acknowledgments</h2><div class="sec"><p>The authors gratefully
acknowledge Polish high-performance
computing infrastructure PLGrid (HPC Centers: ACK Cyfronet AGH) for
providing computer facilities (grant no. PLG/2023/016413) as well
as on the local &#x0201c;piasek&#x0201d; cluster.</p></div></div><div id="notes-4" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-4title">Data Availability Statement</h2><p>All of the data
underlying this study are available in the manuscript and its Supporting
Information files. Apart from that, all of the data (MD simulations)
were obtained using the AMBER suite (Amber20 and AmberTools). The
data were then analyzed using R (statistics and plots), VMD (visualization
of the obtained trajectories), and GIMP (figure preparation). All
software except for the AMBER suite is free of charge. AMBER software
can be obtained from <a href="http://ambermd.org/GetAmber.php">http://ambermd.org/GetAmber.php</a>. R can be downloaded from <a href="https://www.r-project.org/">https://www.r-project.org/</a>. GIMP can be downloaded from <a href="https://www.gimp.org/">https://www.gimp.org/</a>. VMD can
be downloaded from <a href="http://www.ks.uiuc.edu/Research/vmd/">http://www.ks.uiuc.edu/Research/vmd/.</a></p></div><div id="notes-3" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-3title">Supporting Information Available</h2><p>The Supporting Information
is available free of charge at <a href="https://pubs.acs.org/doi/10.1021/acs.jcim.4c00030?goto=supporting-info" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://pubs.acs.org/doi/10.1021/acs.jcim.4c00030</a>.</p><ul class="simple" style="list-style-type:none" id="silist"><li><div>Graphical representation of different parameters analyzed
in this work: RMSD, number of H-bonds, and number of native and non-native
contacts obtained for the protein&#x02013;GAG complexes in MD simulations
using different water models used in this study; the tables of RMSD
(mean and standard deviation) of the ligand and protein for the protein&#x02013;GAG
complexes in MD simulations using various water models (<a href="https://pubs.acs.org/doi/suppl/10.1021/acs.jcim.4c00030/suppl_file/ci4c00030_si_001.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">PDF</a>)</div></li><li><div>Initial coordinates;
topology files; and input files
used to generate MD trajectories (<a href="https://pubs.acs.org/doi/suppl/10.1021/acs.jcim.4c00030/suppl_file/ci4c00030_si_002.zip" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">ZIP</a>)</div></li></ul><p></p></div><div id="notes-1" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-1title">Author Contributions</h2><p>The manuscript
was written through contributions of all authors. All authors have
given approval to the final version of the manuscript.</p></div><div id="notes-2" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-2title">Notes</h2><p>This research
was funded by the National Science Centre of Poland (grant number
UMO-2018/31/G/ST4/00246).</p></div><div id="NOTES-d14e860-autogenerated" class="tsec sec"><h2 class="head no_bottom_margin" id="NOTES-d14e860-autogeneratedtitle">Notes</h2><p>The
authors declare no
competing financial interest.</p></div><div id="sec-a.x.c" class="tsec bk-sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="sec-a.x.ctitle">Supplementary Material</h2><!--/article/back/sec/--><div class="sec suppmat" id="sifile1"><div class="sup-box half_rhythm" id="media-a.x.c.b.a"><a href="/pmc/articles/PMC10934818/bin/ci4c00030_si_001.pdf" data-ga-action="click_feat_suppl">ci4c00030_si_001.pdf</a><sup>(1.4M, pdf)</sup></div></div><div class="sec suppmat" id="sifile2"><div class="sup-box half_rhythm" id="media-a.x.c.c.a"><a href="/pmc/articles/PMC10934818/bin/ci4c00030_si_002.zip" data-ga-action="click_feat_suppl">ci4c00030_si_002.zip</a><sup>(15M, zip)</sup></div></div></div><div id="ref-list-a.x.h" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.x.htitle">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="ref1"><span class="mixed-citation" id="cit1">Varki A.; Cummings R. D.; Esko J. D.; Stanley P.; Hart G.
W.; Aebi M.; Mohnen D.; Kinoshita T.; Packer N. H.; Prestegard J. H.. <span class="ref-title">Study Guide</span>. In <span class="ref-journal">Essentials of Glycobiology [Internet]</span>, 4th ed.; Cold Spring Harbor Laboratory
Press: New York, 2022. [<a href="https://pubmed.ncbi.nlm.nih.gov/35536922" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Essentials+of+Glycobiology+[Internet]&amp;publication_year=2022&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref2"><span class="mixed-citation" id="cit2">Chen J.; Sun T.; You Y.; Wu B.; Wang X.; Wu J.
<span class="ref-title">Proteoglycans
and glycosaminoglycans in stem cell homeostasis and bone tissue regeneration</span>. <span class="ref-journal">Front. Cell Dev. Biol.</span>
2021, <span class="ref-vol">9</span>, 760532 10.3389/fcell.2021.760532.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8669051/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34917612" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3389%2Ffcell.2021.760532" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Cell+Dev.+Biol.&amp;title=Proteoglycans&#10;and+glycosaminoglycans+in+stem+cell+homeostasis+and+bone+tissue+regeneration&amp;author=J.+Chen&amp;author=T.+Sun&amp;author=Y.+You&amp;author=B.+Wu&amp;author=X.+Wang&amp;volume=9&amp;publication_year=2021&amp;pages=760532&amp;pmid=34917612&amp;doi=10.3389/fcell.2021.760532&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref3"><span class="mixed-citation" id="cit3">Wig&#x000e9;n J.; Elowsson-Rendin L.; Karlsson L.; Tykesson E.; Westergren-Thorsson G.
<span class="ref-title">Glycosaminoglycans:
a link between development and regeneration in the lung</span>. <span class="ref-journal">Stem Cells Dev.</span>
2019, <span class="ref-vol">28</span>, 823&#x02013;832. 10.1089/scd.2019.0009.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/31062651" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1089%2Fscd.2019.0009" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stem+Cells+Dev.&amp;title=Glycosaminoglycans:&#10;a+link+between+development+and+regeneration+in+the+lung&amp;author=J.+Wig&#x000e9;n&amp;author=L.+Elowsson-Rendin&amp;author=L.+Karlsson&amp;author=E.+Tykesson&amp;author=G.+Westergren-Thorsson&amp;volume=28&amp;publication_year=2019&amp;pages=823-832&amp;pmid=31062651&amp;doi=10.1089/scd.2019.0009&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref4"><span class="mixed-citation" id="cit4">Dieckmann C.; Renner R.; Milkova L.; Simon J. C.
<span class="ref-title">Regenerative medicine
in dermatology: biomaterials, tissue engineering, stem cells, gene
transfer and beyond</span>. <span class="ref-journal">Exp. Dermatol.</span>
2010, <span class="ref-vol">19</span>, 697&#x02013;706. 10.1111/j.1600-0625.2010.01087.x.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/20545761" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fj.1600-0625.2010.01087.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Exp.+Dermatol.&amp;title=Regenerative+medicine&#10;in+dermatology:+biomaterials,+tissue+engineering,+stem+cells,+gene&#10;transfer+and+beyond&amp;author=C.+Dieckmann&amp;author=R.+Renner&amp;author=L.+Milkova&amp;author=J.+C.+Simon&amp;volume=19&amp;publication_year=2010&amp;pages=697-706&amp;pmid=20545761&amp;doi=10.1111/j.1600-0625.2010.01087.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref5"><span class="mixed-citation" id="cit5">Paganini C.; Costantini R.; Superti-Furga A.; Rossi A.
<span class="ref-title">Bone and connective
tissue disorders caused by defects in glycosaminoglycan biosynthesis:
a panoramic view</span>. <span class="ref-journal">FEBS J.</span>
2019, <span class="ref-vol">286</span>, 3008&#x02013;3032. 10.1111/febs.14984.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/31286677" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Ffebs.14984" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEBS+J.&amp;title=Bone+and+connective&#10;tissue+disorders+caused+by+defects+in+glycosaminoglycan+biosynthesis:&#10;a+panoramic+view&amp;author=C.+Paganini&amp;author=R.+Costantini&amp;author=A.+Superti-Furga&amp;author=A.+Rossi&amp;volume=286&amp;publication_year=2019&amp;pages=3008-3032&amp;pmid=31286677&amp;doi=10.1111/febs.14984&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref6"><span class="mixed-citation" id="cit6">Salbach J.; Rachner T. D.; Rauner M.; Hempel U.; Anderegg U.; Franz S.; Simon J.-C.; Hofbauer L. C.
<span class="ref-title">Regenerative potential
of glycosaminoglycans for skin and bone</span>. <span class="ref-journal">J.
Mol. Med.</span>
2012, <span class="ref-vol">90</span>, 625&#x02013;635. 10.1007/s00109-011-0843-2.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/22187113" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs00109-011-0843-2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&#10;Mol.+Med.&amp;title=Regenerative+potential&#10;of+glycosaminoglycans+for+skin+and+bone&amp;author=J.+Salbach&amp;author=T.+D.+Rachner&amp;author=M.+Rauner&amp;author=U.+Hempel&amp;author=U.+Anderegg&amp;volume=90&amp;publication_year=2012&amp;pages=625-635&amp;pmid=22187113&amp;doi=10.1007/s00109-011-0843-2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref7"><span class="mixed-citation" id="cit7">Prydz K.
<span class="ref-title">Determinants
of glycosaminoglycan (GAG) structure</span>. <span class="ref-journal">Biomolecules</span>
2015, <span class="ref-vol">5</span>, 2003&#x02013;2022. 10.3390/biom5032003.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4598785/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26308067" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fbiom5032003" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biomolecules&amp;title=Determinants&#10;of+glycosaminoglycan+(GAG)+structure&amp;author=K.+Prydz&amp;volume=5&amp;publication_year=2015&amp;pages=2003-2022&amp;pmid=26308067&amp;doi=10.3390/biom5032003&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref8"><span class="mixed-citation" id="cit8">Sasisekharan R.; Venkataraman G.
<span class="ref-title">Heparin and
heparan sulfate: biosynthesis, structure
and function</span>. <span class="ref-journal">Curr. Opin. Chem. Biol.</span>
2000, <span class="ref-vol">4</span>, 626&#x02013;631. 10.1016/S1367-5931(00)00145-9.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/11102866" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FS1367-5931(00)00145-9" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Chem.+Biol.&amp;title=Heparin+and&#10;heparan+sulfate:+biosynthesis,+structure&#10;and+function&amp;author=R.+Sasisekharan&amp;author=G.+Venkataraman&amp;volume=4&amp;publication_year=2000&amp;pages=626-631&amp;pmid=11102866&amp;doi=10.1016/S1367-5931(00)00145-9&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref9"><span class="mixed-citation" id="cit9">Mizumoto S.; Yamada S.; Sugahara K.
<span class="ref-title">Molecular
interactions between chondroitin&#x02013;dermatan
sulfate and growth factors/receptors/matrix proteins</span>. <span class="ref-journal">Curr. Opin. Struct. Biol.</span>
2015, <span class="ref-vol">34</span>, 35&#x02013;42. 10.1016/j.sbi.2015.06.004.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/26164146" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.sbi.2015.06.004" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Struct.+Biol.&amp;title=Molecular&#10;interactions+between+chondroitin&#x02013;dermatan&#10;sulfate+and+growth+factors/receptors/matrix+proteins&amp;author=S.+Mizumoto&amp;author=S.+Yamada&amp;author=K.+Sugahara&amp;volume=34&amp;publication_year=2015&amp;pages=35-42&amp;pmid=26164146&amp;doi=10.1016/j.sbi.2015.06.004&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref10"><span class="mixed-citation" id="cit10">Funderburgh J. L.
<span class="ref-title">MINI REVIEW
Keratan sulfate: structure, biosynthesis, and function</span>. <span class="ref-journal">Glycobiology</span>
2000, <span class="ref-vol">10</span>, 951&#x02013;958. 10.1093/glycob/10.10.951.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/11030741" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fglycob%2F10.10.951" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Glycobiology&amp;title=MINI+REVIEW&#10;Keratan+sulfate:+structure,+biosynthesis,+and+function&amp;author=J.+L.+Funderburgh&amp;volume=10&amp;publication_year=2000&amp;pages=951-958&amp;pmid=11030741&amp;doi=10.1093/glycob/10.10.951&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref11"><span class="mixed-citation" id="cit11">Almond A.
<span class="ref-title">Hyaluronan</span>. <span class="ref-journal">Cell. Mol. Life Sci.</span>
2007, <span class="ref-vol">64</span>, 1591&#x02013;1596. 10.1007/s00018-007-7032-z.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC11136421/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17502996" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs00018-007-7032-z" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell.+Mol.+Life+Sci.&amp;title=Hyaluronan&amp;author=A.+Almond&amp;volume=64&amp;publication_year=2007&amp;pages=1591-1596&amp;pmid=17502996&amp;doi=10.1007/s00018-007-7032-z&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref12"><span class="mixed-citation" id="cit12">Soares P. A.; Queiroz I. N.; Pomin V. H.
<span class="ref-title">NMR structural
biology of sulfated
glycans</span>. <span class="ref-journal">J. Biomol. Struct. Dyn.</span>
2017, <span class="ref-vol">35</span>, 1069&#x02013;1084. 10.1080/07391102.2016.1171165.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/27166778" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1080%2F07391102.2016.1171165" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biomol.+Struct.+Dyn.&amp;title=NMR+structural&#10;biology+of+sulfated&#10;glycans&amp;author=P.+A.+Soares&amp;author=I.+N.+Queiroz&amp;author=V.+H.+Pomin&amp;volume=35&amp;publication_year=2017&amp;pages=1069-1084&amp;pmid=27166778&amp;doi=10.1080/07391102.2016.1171165&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref13"><span class="mixed-citation" id="cit13">Habuchi H.; Habuchi O.; Kimata K.
<span class="ref-title">Sulfation
pattern in glycosaminoglycan:
does it have a code?</span>. <span class="ref-journal">Glycoconjugate J.</span>
2004, <span class="ref-vol">21</span>, 47&#x02013;52. 10.1023/B:GLYC.0000043747.87325.5e. [<a href="https://pubmed.ncbi.nlm.nih.gov/15467398" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1023%2FB%3AGLYC.0000043747.87325.5e" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Glycoconjugate+J.&amp;title=Sulfation&#10;pattern+in+glycosaminoglycan:&#10;does+it+have+a+code?&amp;author=H.+Habuchi&amp;author=O.+Habuchi&amp;author=K.+Kimata&amp;volume=21&amp;publication_year=2004&amp;pages=47-52&amp;doi=10.1023/B:GLYC.0000043747.87325.5e&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref14"><span class="mixed-citation" id="cit14">Kogut M. M.; Marcisz M.; Samsonov S. A.
<span class="ref-title">Modeling
glycosaminoglycan&#x02013;protein
complexes</span>. <span class="ref-journal">Curr. Opin. Struct. Biol.</span>
2022, <span class="ref-vol">73</span>, 102332 10.1016/j.sbi.2022.102332.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/35152187" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.sbi.2022.102332" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Struct.+Biol.&amp;title=Modeling&#10;glycosaminoglycan&#x02013;protein&#10;complexes&amp;author=M.+M.+Kogut&amp;author=M.+Marcisz&amp;author=S.+A.+Samsonov&amp;volume=73&amp;publication_year=2022&amp;pages=102332&amp;pmid=35152187&amp;doi=10.1016/j.sbi.2022.102332&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref15"><span class="mixed-citation" id="cit15">Perez S.; Makshakova O.; Angulo J.; Bedini E.; Bisio A.; de Paz J. L.; Fadda E.; Guerrini M.; Hricovini M.; Hricovini M.; et al. <span class="ref-title">Glycosaminoglycans: what remains to be deciphered?</span>. <span class="ref-journal">JACS Au</span>
2023, <span class="ref-vol">3</span>, 628&#x02013;656. 10.1021/jacsau.2c00569.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10052243/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37006755" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fjacsau.2c00569" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JACS+Au&amp;title=Glycosaminoglycans:+what+remains+to+be+deciphered?&amp;author=S.+Perez&amp;author=O.+Makshakova&amp;author=J.+Angulo&amp;author=E.+Bedini&amp;author=A.+Bisio&amp;volume=3&amp;publication_year=2023&amp;pages=628-656&amp;pmid=37006755&amp;doi=10.1021/jacsau.2c00569&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref16"><span class="mixed-citation" id="cit16">Derler R.; Gesslbauer B.; Weber C.; Strutzmann E.; Miller I.; Kungl A.
<span class="ref-title">Glycosaminoglycan-mediated
downstream
signaling of CXCL8 binding to endothelial cells</span>. <span class="ref-journal">Int. J. Mol. Sci.</span>
2017, <span class="ref-vol">18</span>, 2605. 10.3390/ijms18122605.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5751208/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29207576" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fijms18122605" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Mol.+Sci.&amp;title=Glycosaminoglycan-mediated&#10;downstream&#10;signaling+of+CXCL8+binding+to+endothelial+cells&amp;author=R.+Derler&amp;author=B.+Gesslbauer&amp;author=C.+Weber&amp;author=E.+Strutzmann&amp;author=I.+Miller&amp;volume=18&amp;publication_year=2017&amp;pages=2605&amp;pmid=29207576&amp;doi=10.3390/ijms18122605&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref17"><span class="mixed-citation" id="cit17">Hudalla G. A.; Murphy W. L.
<span class="ref-title">Biomaterials that
regulate growth factor activity via
bioinspired interactions</span>. <span class="ref-journal">Adv. Funct. Mater.</span>
2011, <span class="ref-vol">21</span>, 1754&#x02013;1768. 10.1002/adfm.201002468.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3171147/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21921999" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fadfm.201002468" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Adv.+Funct.+Mater.&amp;title=Biomaterials+that&#10;regulate+growth+factor+activity+via&#10;bioinspired+interactions&amp;author=G.+A.+Hudalla&amp;author=W.+L.+Murphy&amp;volume=21&amp;publication_year=2011&amp;pages=1754-1768&amp;pmid=21921999&amp;doi=10.1002/adfm.201002468&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref18"><span class="mixed-citation" id="cit18">Karamanos N. K.; Piperigkou Z.; Theocharis A. D.; Watanabe H.; Franchi M.; Baud S.; Brezillon S.; G&#x000f6;tte M.; Passi A.; Vigetti D.; et al. <span class="ref-title">Proteoglycan chemical
diversity drives multifunctional cell regulation and therapeutics</span>. <span class="ref-journal">Chem. Rev.</span>
2018, <span class="ref-vol">118</span>, 9152&#x02013;9232. 10.1021/acs.chemrev.8b00354.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/30204432" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Facs.chemrev.8b00354" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem.+Rev.&amp;title=Proteoglycan+chemical&#10;diversity+drives+multifunctional+cell+regulation+and+therapeutics&amp;author=N.+K.+Karamanos&amp;author=Z.+Piperigkou&amp;author=A.+D.+Theocharis&amp;author=H.+Watanabe&amp;author=M.+Franchi&amp;volume=118&amp;publication_year=2018&amp;pages=9152-9232&amp;pmid=30204432&amp;doi=10.1021/acs.chemrev.8b00354&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref19"><span class="mixed-citation" id="cit19">Moustakas A.; Souchelnytskyi S.; Heldin C.-H.
<span class="ref-title">Smad regulation in TGF-&#x003b2; signal
transduction</span>. <span class="ref-journal">J. Cell Sci.</span>
2001, <span class="ref-vol">114</span>, 4359&#x02013;4369. 10.1242/jcs.114.24.4359.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/11792802" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1242%2Fjcs.114.24.4359" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Cell+Sci.&amp;title=Smad+regulation+in+TGF-&#x003b2;+signal&#10;transduction&amp;author=A.+Moustakas&amp;author=S.+Souchelnytskyi&amp;author=C.-H.+Heldin&amp;volume=114&amp;publication_year=2001&amp;pages=4359-4369&amp;pmid=11792802&amp;doi=10.1242/jcs.114.24.4359&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref20"><span class="mixed-citation" id="cit20">Penk A.; Baumann L.; Huster D.; Samsonov S. A.
<span class="ref-title">NMR and molecular
modeling reveal specificity of the interactions between CXCL14 and
glycosaminoglycans</span>. <span class="ref-journal">Glycobiology</span>
2019, <span class="ref-vol">29</span>, 715&#x02013;725. 10.1093/glycob/cwz047.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/31264681" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fglycob%2Fcwz047" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Glycobiology&amp;title=NMR+and+molecular&#10;modeling+reveal+specificity+of+the+interactions+between+CXCL14+and&#10;glycosaminoglycans&amp;author=A.+Penk&amp;author=L.+Baumann&amp;author=D.+Huster&amp;author=S.+A.+Samsonov&amp;volume=29&amp;publication_year=2019&amp;pages=715-725&amp;pmid=31264681&amp;doi=10.1093/glycob/cwz047&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref21"><span class="mixed-citation" id="cit21">Samantray S.; Olubiyi O. O.; Strodel B.
<span class="ref-title">The Influences
of Sulphation, Salt
Type, and Salt Concentration on the Structural Heterogeneity of Glycosaminoglycans</span>. <span class="ref-journal">Int. J. Mol. Sci.</span>
2021, <span class="ref-vol">22</span>, 11529. 10.3390/ijms222111529.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8583755/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34768961" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fijms222111529" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Mol.+Sci.&amp;title=The+Influences&#10;of+Sulphation,+Salt&#10;Type,+and+Salt+Concentration+on+the+Structural+Heterogeneity+of+Glycosaminoglycans&amp;author=S.+Samantray&amp;author=O.+O.+Olubiyi&amp;author=B.+Strodel&amp;volume=22&amp;publication_year=2021&amp;pages=11529&amp;pmid=34768961&amp;doi=10.3390/ijms222111529&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref22"><span class="mixed-citation" id="cit22">Imberty A.; Lortat-Jacob H.; P&#x000e9;rez S.
<span class="ref-title">Structural view of glycosaminoglycan&#x02013;protein
interactions</span>. <span class="ref-journal">Carbohydr. Res.</span>
2007, <span class="ref-vol">342</span>, 430&#x02013;439. 10.1016/j.carres.2006.12.019.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/17229412" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.carres.2006.12.019" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Carbohydr.+Res.&amp;title=Structural+view+of+glycosaminoglycan&#x02013;protein&#10;interactions&amp;author=A.+Imberty&amp;author=H.+Lortat-Jacob&amp;author=S.+P&#x000e9;rez&amp;volume=342&amp;publication_year=2007&amp;pages=430-439&amp;pmid=17229412&amp;doi=10.1016/j.carres.2006.12.019&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref23"><span class="mixed-citation" id="cit23">Nagarajan B.; Holmes S. G.; Sankaranarayanan N. V.; Desai U. R.
<span class="ref-title">Molecular dynamics
simulations to understand glycosaminoglycan interactions in the free-and
protein-bound states</span>. <span class="ref-journal">Curr. Opin. Struct. Biol.</span>
2022, <span class="ref-vol">74</span>, 102356 10.1016/j.sbi.2022.102356.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9189024/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35306321" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.sbi.2022.102356" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Struct.+Biol.&amp;title=Molecular+dynamics&#10;simulations+to+understand+glycosaminoglycan+interactions+in+the+free-and&#10;protein-bound+states&amp;author=B.+Nagarajan&amp;author=S.+G.+Holmes&amp;author=N.+V.+Sankaranarayanan&amp;author=U.+R.+Desai&amp;volume=74&amp;publication_year=2022&amp;pages=102356&amp;pmid=35306321&amp;doi=10.1016/j.sbi.2022.102356&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref24"><span class="mixed-citation" id="cit24">Petitou M.; Casu B.; Lindahl U.
<span class="ref-title">1976&#x02013;1983,
a critical period
in the history of heparin: the discovery of the antithrombin binding
site</span>. <span class="ref-journal">Biochimie</span>
2003, <span class="ref-vol">85</span>, 83&#x02013;89. 10.1016/S0300-9084(03)00078-6.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/12765778" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FS0300-9084(03)00078-6" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochimie&amp;title=1976&#x02013;1983,&#10;a+critical+period&#10;in+the+history+of+heparin:+the+discovery+of+the+antithrombin+binding&#10;site&amp;author=M.+Petitou&amp;author=B.+Casu&amp;author=U.+Lindahl&amp;volume=85&amp;publication_year=2003&amp;pages=83-89&amp;pmid=12765778&amp;doi=10.1016/S0300-9084(03)00078-6&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref25"><span class="mixed-citation" id="cit25">Sepuru K. M.; Nagarajan B.; Desai U. R.; Rajarathnam K.
<span class="ref-title">Structural
basis, stoichiometry, and thermodynamics of binding of the chemokines
KC and MIP2 to the glycosaminoglycan heparin</span>. <span class="ref-journal">J. Biol. Chem.</span>
2018, <span class="ref-vol">293</span>, 17817&#x02013;17828. 10.1074/jbc.RA118.004866.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6240865/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30257866" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1074%2Fjbc.RA118.004866" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Structural&#10;basis,+stoichiometry,+and+thermodynamics+of+binding+of+the+chemokines&#10;KC+and+MIP2+to+the+glycosaminoglycan+heparin&amp;author=K.+M.+Sepuru&amp;author=B.+Nagarajan&amp;author=U.+R.+Desai&amp;author=K.+Rajarathnam&amp;volume=293&amp;publication_year=2018&amp;pages=17817-17828&amp;pmid=30257866&amp;doi=10.1074/jbc.RA118.004866&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref26"><span class="mixed-citation" id="cit26">Foley B. L.; Tessier M. B.; Woods R. J.
<span class="ref-title">Carbohydrate force
fields</span>. <span class="ref-journal">Wiley Interdiscip. Rev.: Comput. Mol.
Sci.</span>
2012, <span class="ref-vol">2</span>, 652&#x02013;697. 10.1002/wcms.89.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4270206/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25530813" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fwcms.89" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Wiley+Interdiscip.+Rev.:+Comput.+Mol.&#10;Sci.&amp;title=Carbohydrate+force&#10;fields&amp;author=B.+L.+Foley&amp;author=M.+B.+Tessier&amp;author=R.+J.+Woods&amp;volume=2&amp;publication_year=2012&amp;pages=652-697&amp;pmid=25530813&amp;doi=10.1002/wcms.89&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref27"><span class="mixed-citation" id="cit27">Kerzmann A.; Fuhrmann J.; Kohlbacher O.; Neumann D.
<span class="ref-title">BALLDock/SLICK: a new
method for protein-carbohydrate docking</span>. <span class="ref-journal">J.
Chem. Inf. Model.</span>
2008, <span class="ref-vol">48</span>, 1616&#x02013;1625. 10.1021/ci800103u.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/18646839" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fci800103u" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&#10;Chem.+Inf.+Model.&amp;title=BALLDock/SLICK:+a+new&#10;method+for+protein-carbohydrate+docking&amp;author=A.+Kerzmann&amp;author=J.+Fuhrmann&amp;author=O.+Kohlbacher&amp;author=D.+Neumann&amp;volume=48&amp;publication_year=2008&amp;pages=1616-1625&amp;pmid=18646839&amp;doi=10.1021/ci800103u&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref28"><span class="mixed-citation" id="cit28">Kerzmann A.; Neumann D.; Kohlbacher O.
<span class="ref-title">SLICK&#x02014;Scoring and Energy Functions
for Protein&#x02013;Carbohydrate Interactions</span>. <span class="ref-journal">J. Chem. Inf. Model.</span>
2006, <span class="ref-vol">46</span>, 1635&#x02013;1642. 10.1021/ci050422y.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/16859295" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fci050422y" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Chem.+Inf.+Model.&amp;title=SLICK&#x02014;Scoring+and+Energy+Functions&#10;for+Protein&#x02013;Carbohydrate+Interactions&amp;author=A.+Kerzmann&amp;author=D.+Neumann&amp;author=O.+Kohlbacher&amp;volume=46&amp;publication_year=2006&amp;pages=1635-1642&amp;pmid=16859295&amp;doi=10.1021/ci050422y&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref29"><span class="mixed-citation" id="cit29">Taroni C.; Jones S.; Thornton J. M.
<span class="ref-title">Analysis and prediction of carbohydrate
binding sites</span>. <span class="ref-journal">Protein Eng., Des. Sel.</span>
2000, <span class="ref-vol">13</span>, 89&#x02013;98. 10.1093/protein/13.2.89. [<a href="https://pubmed.ncbi.nlm.nih.gov/10708647" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fprotein%2F13.2.89" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Eng.,+Des.+Sel.&amp;title=Analysis+and+prediction+of+carbohydrate&#10;binding+sites&amp;author=C.+Taroni&amp;author=S.+Jones&amp;author=J.+M.+Thornton&amp;volume=13&amp;publication_year=2000&amp;pages=89-98&amp;doi=10.1093/protein/13.2.89&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref30"><span class="mixed-citation" id="cit30">Samsonov S. A.; Pisabarro M. T.
<span class="ref-title">Computational
analysis of interactions in structurally
available protein&#x02013;glycosaminoglycan complexes</span>. <span class="ref-journal">Glycobiology</span>
2016, <span class="ref-vol">26</span>, 850&#x02013;861. 10.1093/glycob/cww055.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/27496767" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fglycob%2Fcww055" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Glycobiology&amp;title=Computational&#10;analysis+of+interactions+in+structurally&#10;available+protein&#x02013;glycosaminoglycan+complexes&amp;author=S.+A.+Samsonov&amp;author=M.+T.+Pisabarro&amp;volume=26&amp;publication_year=2016&amp;pages=850-861&amp;pmid=27496767&amp;doi=10.1093/glycob/cww055&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref31"><span class="mixed-citation" id="cit31">Samsonov S.; Teyra J.; Pisabarro M. T.
<span class="ref-title">A molecular dynamics approach to
study the importance of solvent in protein interactions</span>. <span class="ref-journal">Proteins</span>
2008, <span class="ref-vol">73</span>, 515&#x02013;525. 10.1002/prot.22076.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/18452208" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fprot.22076" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=A+molecular+dynamics+approach+to&#10;study+the+importance+of+solvent+in+protein+interactions&amp;author=S.+Samsonov&amp;author=J.+Teyra&amp;author=M.+T.+Pisabarro&amp;volume=73&amp;publication_year=2008&amp;pages=515-525&amp;pmid=18452208&amp;doi=10.1002/prot.22076&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref32"><span class="mixed-citation" id="cit32">Samsonov S. A.; Gehrcke J.-P.; Pisabarro M. T.
<span class="ref-title">Flexibility and explicit solvent
in molecular-dynamics-based docking of protein&#x02013;glycosaminoglycan
systems</span>. <span class="ref-journal">J. Chem. Inf. Model</span>
2014, <span class="ref-vol">54</span>, 582&#x02013;592. 10.1021/ci4006047.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/24479827" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fci4006047" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Chem.+Inf.+Model&amp;title=Flexibility+and+explicit+solvent&#10;in+molecular-dynamics-based+docking+of+protein&#x02013;glycosaminoglycan&#10;systems&amp;author=S.+A.+Samsonov&amp;author=J.-P.+Gehrcke&amp;author=M.+T.+Pisabarro&amp;volume=54&amp;publication_year=2014&amp;pages=582-592&amp;pmid=24479827&amp;doi=10.1021/ci4006047&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref33"><span class="mixed-citation" id="cit33">M&#x000f6;bius K.; Nordsieck K.; Pichert A.; Samsonov S. A.; Thomas L.; Schiller J.; Kalkhof S.; Teresa Pisabarro M.; Beck-Sickinger A. G.; Huster D.
<span class="ref-title">Investigation of lysine side chain
interactions of interleukin-8 with heparin and other glycosaminoglycans
studied by a methylation-NMR approach</span>. <span class="ref-journal">Glycobiology</span>
2013, <span class="ref-vol">23</span>, 1260&#x02013;1269. 10.1093/glycob/cwt062.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/23982278" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fglycob%2Fcwt062" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Glycobiology&amp;title=Investigation+of+lysine+side+chain&#10;interactions+of+interleukin-8+with+heparin+and+other+glycosaminoglycans&#10;studied+by+a+methylation-NMR+approach&amp;author=K.+M&#x000f6;bius&amp;author=K.+Nordsieck&amp;author=A.+Pichert&amp;author=S.+A.+Samsonov&amp;author=L.+Thomas&amp;volume=23&amp;publication_year=2013&amp;pages=1260-1269&amp;pmid=23982278&amp;doi=10.1093/glycob/cwt062&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref34"><span class="mixed-citation" id="cit34">Nguyen H.; Roe D. R.; Simmerling C.
<span class="ref-title">Improved generalized
born solvent
model parameters for protein simulations</span>. <span class="ref-journal">J.
Chem. Theory Comput.</span>
2013, <span class="ref-vol">9</span>, 2020&#x02013;2034. 10.1021/ct3010485.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4361090/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25788871" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fct3010485" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&#10;Chem.+Theory+Comput.&amp;title=Improved+generalized&#10;born+solvent&#10;model+parameters+for+protein+simulations&amp;author=H.+Nguyen&amp;author=D.+R.+Roe&amp;author=C.+Simmerling&amp;volume=9&amp;publication_year=2013&amp;pages=2020-2034&amp;pmid=25788871&amp;doi=10.1021/ct3010485&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref35"><span class="mixed-citation" id="cit35">Onufriev A. V.; Case D. A.
<span class="ref-title">Generalized Born implicit solvent models for biomolecules</span>. <span class="ref-journal">Annu. Rev. Biophys.</span>
2019, <span class="ref-vol">48</span>, 275&#x02013;296. 10.1146/annurev-biophys-052118-115325.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6645684/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30857399" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1146%2Fannurev-biophys-052118-115325" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Biophys.&amp;title=Generalized+Born+implicit+solvent+models+for+biomolecules&amp;author=A.+V.+Onufriev&amp;author=D.+A.+Case&amp;volume=48&amp;publication_year=2019&amp;pages=275-296&amp;pmid=30857399&amp;doi=10.1146/annurev-biophys-052118-115325&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref36"><span class="mixed-citation" id="cit36">Lang E. J. M.; Baker E. G.; Woolfson D. N.; Mulholland A. J.
<span class="ref-title">Generalized
Born implicit solvent models do not reproduce secondary structures
of de novo designed Glu/Lys peptides</span>. <span class="ref-journal">J. Chem.
Theory Comput.</span>
2022, <span class="ref-vol">18</span>, 4070&#x02013;4076. 10.1021/acs.jctc.1c01172.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9281390/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35687842" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Facs.jctc.1c01172" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Chem.&#10;Theory+Comput.&amp;title=Generalized&#10;Born+implicit+solvent+models+do+not+reproduce+secondary+structures&#10;of+de+novo+designed+Glu/Lys+peptides&amp;author=E.+J.+M.+Lang&amp;author=E.+G.+Baker&amp;author=D.+N.+Woolfson&amp;author=A.+J.+Mulholland&amp;volume=18&amp;publication_year=2022&amp;pages=4070-4076&amp;pmid=35687842&amp;doi=10.1021/acs.jctc.1c01172&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref37"><span class="mixed-citation" id="cit37">Marcisz M.; Gaardl&#x000f8;s M.; Bojarski K. K.; Siebenmorgen T.; Zacharias M.; Samsonov S. A.
<span class="ref-title">Explicit solvent repulsive scaling
replica exchange molecular dynamics (RS-REMD) in molecular modeling
of protein-glycosaminoglycan complexes</span>. <span class="ref-journal">J. Comput.
Chem.</span>
2022, <span class="ref-vol">43</span>, 1633&#x02013;1640. 10.1002/jcc.26965.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/35796487" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fjcc.26965" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Comput.&#10;Chem.&amp;title=Explicit+solvent+repulsive+scaling&#10;replica+exchange+molecular+dynamics+(RS-REMD)+in+molecular+modeling&#10;of+protein-glycosaminoglycan+complexes&amp;author=M.+Marcisz&amp;author=M.+Gaardl&#x000f8;s&amp;author=K.+K.+Bojarski&amp;author=T.+Siebenmorgen&amp;author=M.+Zacharias&amp;volume=43&amp;publication_year=2022&amp;pages=1633-1640&amp;pmid=35796487&amp;doi=10.1002/jcc.26965&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref38"><span class="mixed-citation" id="cit38">Nagarajan B.; Sankaranarayanan N. V.; Patel B. B.; Desai U. R.
<span class="ref-title">A molecular dynamics-based
algorithm for evaluating the glycosaminoglycan mimicking potential
of synthetic, homogenous, sulfated small molecules</span>. <span class="ref-journal">PLoS One</span>
2017, <span class="ref-vol">12</span>, e0171619 10.1371/journal.pone.0171619.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5300208/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28182755" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pone.0171619" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=A+molecular+dynamics-based&#10;algorithm+for+evaluating+the+glycosaminoglycan+mimicking+potential&#10;of+synthetic,+homogenous,+sulfated+small+molecules&amp;author=B.+Nagarajan&amp;author=N.+V.+Sankaranarayanan&amp;author=B.+B.+Patel&amp;author=U.+R.+Desai&amp;volume=12&amp;publication_year=2017&amp;pages=e0171619&amp;pmid=28182755&amp;doi=10.1371/journal.pone.0171619&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref39"><span class="mixed-citation" id="cit39">Sapay N.; Cabannes E.; Petitou M.; Imberty A.
<span class="ref-title">Molecular
modeling
of the interaction between heparan sulfate and cellular growth factors:
bringing pieces together</span>. <span class="ref-journal">Glycobiology</span>
2011, <span class="ref-vol">21</span>, 1181&#x02013;1193. 10.1093/glycob/cwr052.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/21572110" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fglycob%2Fcwr052" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Glycobiology&amp;title=Molecular&#10;modeling&#10;of+the+interaction+between+heparan+sulfate+and+cellular+growth+factors:&#10;bringing+pieces+together&amp;author=N.+Sapay&amp;author=E.+Cabannes&amp;author=M.+Petitou&amp;author=A.+Imberty&amp;volume=21&amp;publication_year=2011&amp;pages=1181-1193&amp;pmid=21572110&amp;doi=10.1093/glycob/cwr052&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref40"><span class="mixed-citation" id="cit40">Izadi S.; Anandakrishnan R.; Onufriev A. V.
<span class="ref-title">Building water models: a different
approach</span>. <span class="ref-journal">J. Phys. Chem. Lett.</span>
2014, <span class="ref-vol">5</span>, 3863&#x02013;3871. 10.1021/jz501780a.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4226301/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25400877" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fjz501780a" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Phys.+Chem.+Lett.&amp;title=Building+water+models:+a+different&#10;approach&amp;author=S.+Izadi&amp;author=R.+Anandakrishnan&amp;author=A.+V.+Onufriev&amp;volume=5&amp;publication_year=2014&amp;pages=3863-3871&amp;pmid=25400877&amp;doi=10.1021/jz501780a&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref41"><span class="mixed-citation" id="cit41">Izadi S.; Onufriev A. V.
<span class="ref-title">Accuracy limit of
rigid 3-point water models</span>. <span class="ref-journal">J. Chem. Phys.</span>
2016, <span class="ref-vol">145</span>, 074501 10.1063/1.4960175.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4991989/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27544113" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1063%2F1.4960175" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Chem.+Phys.&amp;title=Accuracy+limit+of&#10;rigid+3-point+water+models&amp;author=S.+Izadi&amp;author=A.+V.+Onufriev&amp;volume=145&amp;publication_year=2016&amp;pages=074501&amp;pmid=27544113&amp;doi=10.1063/1.4960175&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref42"><span class="mixed-citation" id="cit42">Verli H.; Guimar&#x000e3;es J. A.
<span class="ref-title">Molecular dynamics simulation of a decasaccharide fragment
of heparin in aqueous solution</span>. <span class="ref-journal">Carbohydr. Res.</span>
2004, <span class="ref-vol">339</span>, 281&#x02013;290. 10.1016/j.carres.2003.09.026.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/14698886" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.carres.2003.09.026" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Carbohydr.+Res.&amp;title=Molecular+dynamics+simulation+of+a+decasaccharide+fragment&#10;of+heparin+in+aqueous+solution&amp;author=H.+Verli&amp;author=J.+A.+Guimar&#x000e3;es&amp;volume=339&amp;publication_year=2004&amp;pages=281-290&amp;pmid=14698886&amp;doi=10.1016/j.carres.2003.09.026&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref43"><span class="mixed-citation" id="cit43">Neamtu A.; Tamba B.; Patras X.
<span class="ref-title">Molecular
dynamics simulations of
chondroitin sulfate in explicit solvent: Point charge water models
compared</span>. <span class="ref-journal">Cellul. Chem. Technol.</span>
2013, <span class="ref-vol">47</span>, 191&#x02013;202. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cellul.+Chem.+Technol.&amp;title=Molecular&#10;dynamics+simulations+of&#10;chondroitin+sulfate+in+explicit+solvent:+Point+charge+water+models&#10;compared&amp;author=A.+Neamtu&amp;author=B.+Tamba&amp;author=X.+Patras&amp;volume=47&amp;publication_year=2013&amp;pages=191-202&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref44"><span class="mixed-citation" id="cit44">Marcisz M.; Samsonov S. A.
<span class="ref-title">Solvent Model Benchmark
for Molecular Dynamics of Glycosaminoglycans</span>. <span class="ref-journal">J. Chem. Inf. Model.</span>
2023, <span class="ref-vol">63</span>, 2147&#x02013;2157. 10.1021/acs.jcim.2c01472.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10091405/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36989082" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Facs.jcim.2c01472" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Chem.+Inf.+Model.&amp;title=Solvent+Model+Benchmark&#10;for+Molecular+Dynamics+of+Glycosaminoglycans&amp;author=M.+Marcisz&amp;author=S.+A.+Samsonov&amp;volume=63&amp;publication_year=2023&amp;pages=2147-2157&amp;pmid=36989082&amp;doi=10.1021/acs.jcim.2c01472&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref45"><span class="mixed-citation" id="cit45">Almond A.; Sheehan J. K.
<span class="ref-title">Glycosaminoglycan conformation: do
aqueous molecular
dynamics simulations agree with x-ray fiber diffraction?</span>. <span class="ref-journal">Glycobiology</span>
2000, <span class="ref-vol">10</span>, 329&#x02013;338. 10.1093/glycob/10.3.329.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/10704532" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fglycob%2F10.3.329" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Glycobiology&amp;title=Glycosaminoglycan+conformation:+do&#10;aqueous+molecular&#10;dynamics+simulations+agree+with+x-ray+fiber+diffraction?&amp;author=A.+Almond&amp;author=J.+K.+Sheehan&amp;volume=10&amp;publication_year=2000&amp;pages=329-338&amp;pmid=10704532&amp;doi=10.1093/glycob/10.3.329&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref46"><span class="mixed-citation" id="cit46">Case D. A.; Belfon K.; Ben-Shalom I. Y.; Brozell S. R.; Cerutti D. S.; Cheatham T. E.; Cruzeiro V. W. D.; Darden T. A.; Duke R. E.; Giambasu G.; Gilson M. K.; Gohlke H.; Goetz A.
W.; Harris R.; Izadi S.; Izmailov S. A.; Kasavajhala K.; Kovalenko A.; Krasny R.; Kurtzman T.; Lee T. S.; LeGrand S.; Li P.; Lin C.; Liu J.; Luchko T.; Luo R.; Man V.; Merz K. M.; Miao Y.; Mikhailovskii O.; Monard G.; Nguyen H.; Onufriev A.; Pan F.; Pantano F.; Qi R.; Roe D. R.; Roitberg A.; Sagui C.; Schott-Verdugo S.; Shen J.; Simmerling C. L.; Skrynnikov N. R.; Smith J.; Swails K.; Walker R. C.; Wang J.; Wilson L.; Wolf R. M.; Wu X.; Xiong Y.; Xue Y.; York D. M.; Kollman P. A.. <span class="ref-journal">AMBER 2020</span>; University of California: San Francisco, 2020. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=AMBER+2020&amp;publication_year=2020&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref47"><span class="mixed-citation" id="cit47">Onufriev A.; Case D. A.; Bashford D.
<span class="ref-title">Effective
Born radii in the generalized
Born approximation: the importance of being perfect</span>. <span class="ref-journal">J. Comput. Chem.</span>
2002, <span class="ref-vol">23</span>, 1297&#x02013;1304. 10.1002/jcc.10126.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/12214312" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fjcc.10126" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Comput.+Chem.&amp;title=Effective&#10;Born+radii+in+the+generalized&#10;Born+approximation:+the+importance+of+being+perfect&amp;author=A.+Onufriev&amp;author=D.+A.+Case&amp;author=D.+Bashford&amp;volume=23&amp;publication_year=2002&amp;pages=1297-1304&amp;pmid=12214312&amp;doi=10.1002/jcc.10126&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref48"><span class="mixed-citation" id="cit48">Jorgensen W. L.; Chandrasekhar J.; Madura J. D.; Impey R. W.; Klein M. L.
<span class="ref-title">Comparison
of simple potential functions for simulating liquid water</span>. <span class="ref-journal">J. Chem. Phys.</span>
1983, <span class="ref-vol">79</span>, 926&#x02013;935. 10.1063/1.445869. [<a href="//doi.org/10.1063%2F1.445869" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Chem.+Phys.&amp;title=Comparison&#10;of+simple+potential+functions+for+simulating+liquid+water&amp;author=W.+L.+Jorgensen&amp;author=J.+Chandrasekhar&amp;author=J.+D.+Madura&amp;author=R.+W.+Impey&amp;author=M.+L.+Klein&amp;volume=79&amp;publication_year=1983&amp;pages=926-935&amp;doi=10.1063/1.445869&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref49"><span class="mixed-citation" id="cit49">Neria E.; Fischer S.; Karplus M.
<span class="ref-title">Simulation of activation free energies
in molecular systems</span>. <span class="ref-journal">J. Chem. Phys.</span>
1996, <span class="ref-vol">105</span>, 1902&#x02013;1921. 10.1063/1.472061. [<a href="//doi.org/10.1063%2F1.472061" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Chem.+Phys.&amp;title=Simulation+of+activation+free+energies&#10;in+molecular+systems&amp;author=E.+Neria&amp;author=S.+Fischer&amp;author=M.+Karplus&amp;volume=105&amp;publication_year=1996&amp;pages=1902-1921&amp;doi=10.1063/1.472061&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref50"><span class="mixed-citation" id="cit50">Berendsen H. J. C.; Grigera J. R.; Straatsma T. P.
<span class="ref-title">The missing
term in effective pair
potentials</span>. <span class="ref-journal">J. Phys. Chem. A</span>
1987, <span class="ref-vol">91</span>, 6269&#x02013;6271. 10.1021/j100308a038. [<a href="//doi.org/10.1021%2Fj100308a038" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Phys.+Chem.+A&amp;title=The+missing&#10;term+in+effective+pair&#10;potentials&amp;author=H.+J.+C.+Berendsen&amp;author=J.+R.+Grigera&amp;author=T.+P.+Straatsma&amp;volume=91&amp;publication_year=1987&amp;pages=6269-6271&amp;doi=10.1021/j100308a038&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref51"><span class="mixed-citation" id="cit51">Horn H. W.; Swope W. C.; Pitera J. W.; Madura J. D.; Dick T. J.; Hura G. L.; Head-Gordon T.
<span class="ref-title">Development
of an improved four-site
water model for biomolecular simulations: TIP4P-Ew</span>. <span class="ref-journal">J. Chem. Phys.</span>
2004, <span class="ref-vol">120</span>, 9665&#x02013;9678. 10.1063/1.1683075.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/15267980" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1063%2F1.1683075" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Chem.+Phys.&amp;title=Development&#10;of+an+improved+four-site&#10;water+model+for+biomolecular+simulations:+TIP4P-Ew&amp;author=H.+W.+Horn&amp;author=W.+C.+Swope&amp;author=J.+W.+Pitera&amp;author=J.+D.+Madura&amp;author=T.+J.+Dick&amp;volume=120&amp;publication_year=2004&amp;pages=9665-9678&amp;pmid=15267980&amp;doi=10.1063/1.1683075&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref52"><span class="mixed-citation" id="cit52">Mahoney M. W.; Jorgensen W. L.
<span class="ref-title">A five-site model for liquid water and the reproduction
of the density anomaly by rigid, nonpolarizable potential functions</span>. <span class="ref-journal">J. Chem. Phys.</span>
2000, <span class="ref-vol">112</span>, 8910&#x02013;8922. 10.1063/1.481505. [<a href="//doi.org/10.1063%2F1.481505" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Chem.+Phys.&amp;title=A+five-site+model+for+liquid+water+and+the+reproduction&#10;of+the+density+anomaly+by+rigid,+nonpolarizable+potential+functions&amp;author=M.+W.+Mahoney&amp;author=W.+L.+Jorgensen&amp;volume=112&amp;publication_year=2000&amp;pages=8910-8922&amp;doi=10.1063/1.481505&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref53"><span class="mixed-citation" id="cit53">Hawkins G. D.; Cramer C. J.; Truhlar D. G.
<span class="ref-title">Pairwise solute descreening of solute
charges from a dielectric medium</span>. <span class="ref-journal">Chem. Phys.
Lett.</span>
1995, <span class="ref-vol">246</span>, 122&#x02013;129. 10.1016/0009-2614(95)01082-K. [<a href="//doi.org/10.1016%2F0009-2614(95)01082-K" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem.+Phys.&#10;Lett.&amp;title=Pairwise+solute+descreening+of+solute&#10;charges+from+a+dielectric+medium&amp;author=G.+D.+Hawkins&amp;author=C.+J.+Cramer&amp;author=D.+G.+Truhlar&amp;volume=246&amp;publication_year=1995&amp;pages=122-129&amp;doi=10.1016/0009-2614(95)01082-K&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref54"><span class="mixed-citation" id="cit54">Onufriev A.; Bashford D.; Case D. A.
<span class="ref-title">Modification of the generalized Born
model suitable for macromolecules</span>. <span class="ref-journal">J. Phys.
Chem. B</span>
2000, <span class="ref-vol">104</span>, 3712&#x02013;3720. 10.1021/jp994072s. [<a href="//doi.org/10.1021%2Fjp994072s" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Phys.&#10;Chem.+B&amp;title=Modification+of+the+generalized+Born&#10;model+suitable+for+macromolecules&amp;author=A.+Onufriev&amp;author=D.+Bashford&amp;author=D.+A.+Case&amp;volume=104&amp;publication_year=2000&amp;pages=3712-3720&amp;doi=10.1021/jp994072s&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref55"><span class="mixed-citation" id="cit55">Onufriev A.; Bashford D.; Case D. A.
<span class="ref-title">Exploring
protein native states and
large-scale conformational changes with a modified generalized born
model</span>. <span class="ref-journal">Proteins</span>
2004, <span class="ref-vol">55</span>, 383&#x02013;394. 10.1002/prot.20033.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/15048829" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fprot.20033" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Exploring&#10;protein+native+states+and&#10;large-scale+conformational+changes+with+a+modified+generalized+born&#10;model&amp;author=A.+Onufriev&amp;author=D.+Bashford&amp;author=D.+A.+Case&amp;volume=55&amp;publication_year=2004&amp;pages=383-394&amp;pmid=15048829&amp;doi=10.1002/prot.20033&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref56"><span class="mixed-citation" id="cit56">Mongan J.; Simmerling C.; McCammon J. A.; Case D. A.; Onufriev A.
<span class="ref-title">Generalized
Born model with a simple, robust molecular volume correction</span>. <span class="ref-journal">J. Chem. Theory Comput.</span>
2007, <span class="ref-vol">3</span>, 156&#x02013;169. 10.1021/ct600085e.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2975579/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21072141" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fct600085e" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Chem.+Theory+Comput.&amp;title=Generalized&#10;Born+model+with+a+simple,+robust+molecular+volume+correction&amp;author=J.+Mongan&amp;author=C.+Simmerling&amp;author=J.+A.+McCammon&amp;author=D.+A.+Case&amp;author=A.+Onufriev&amp;volume=3&amp;publication_year=2007&amp;pages=156-169&amp;pmid=21072141&amp;doi=10.1021/ct600085e&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref57"><span class="mixed-citation" id="cit57">Ryckaert J.-P.; Ciccotti G.; Berendsen H. J.
<span class="ref-title">Numerical integration of the cartesian
equations of motion of a system with constraints: molecular dynamics
of n-alkanes</span>. <span class="ref-journal">J. Comput. Phys.</span>
1977, <span class="ref-vol">23</span>, 327&#x02013;341. 10.1016/0021-9991(77)90098-5. [<a href="//doi.org/10.1016%2F0021-9991(77)90098-5" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Comput.+Phys.&amp;title=Numerical+integration+of+the+cartesian&#10;equations+of+motion+of+a+system+with+constraints:+molecular+dynamics&#10;of+n-alkanes&amp;author=J.-P.+Ryckaert&amp;author=G.+Ciccotti&amp;author=H.+J.+Berendsen&amp;volume=23&amp;publication_year=1977&amp;pages=327-341&amp;doi=10.1016/0021-9991(77)90098-5&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref58"><span class="mixed-citation" id="cit58">Darden T.; York D.; Pedersen L.
<span class="ref-title">Particle mesh
Ewald: An N&#x000b7; log
(N) method for Ewald sums in large systems</span>. <span class="ref-journal">J. Chem. Phys.</span>
1993, <span class="ref-vol">98</span>, 10089&#x02013;10092. 10.1063/1.464397. [<a href="//doi.org/10.1063%2F1.464397" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Chem.+Phys.&amp;title=Particle+mesh&#10;Ewald:+An+N&#x000b7;+log&#10;(N)+method+for+Ewald+sums+in+large+systems&amp;author=T.+Darden&amp;author=D.+York&amp;author=L.+Pedersen&amp;volume=98&amp;publication_year=1993&amp;pages=10089-10092&amp;doi=10.1063/1.464397&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref59"><span class="mixed-citation" id="cit59">Roe D. R.; Cheatham T. E. III
<span class="ref-title">PTRAJ and CPPTRAJ: software for processing
and analysis of molecular dynamics trajectory data</span>. <span class="ref-journal">J. Chem. Theory Comput.</span>
2013, <span class="ref-vol">9</span>, 3084&#x02013;3095. 10.1021/ct400341p.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/26583988" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fct400341p" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Chem.+Theory+Comput.&amp;title=PTRAJ+and+CPPTRAJ:+software+for+processing&#10;and+analysis+of+molecular+dynamics+trajectory+data&amp;author=D.+R.+Roe&amp;author=T.+E.+Cheatham&amp;volume=9&amp;publication_year=2013&amp;pages=3084-3095&amp;pmid=26583988&amp;doi=10.1021/ct400341p&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref60"><span class="mixed-citation" id="cit60">Core_Team_(2013) . <span class="ref-journal">R: A Language
and Environment for Statistical Computing</span>; R. R Development Core Team, 2013. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=R:+A+Language&#10;and+Environment+for+Statistical+Computing&amp;publication_year=2013&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref61"><span class="mixed-citation" id="cit61">Humphrey W.; Dalke A.; Schulten K.
<span class="ref-title">VMD: visual molecular
dynamics</span>. <span class="ref-journal">J. Mol. Graphics</span>
1996, <span class="ref-vol">14</span>, 33&#x02013;38. 10.1016/0263-7855(96)00018-5. [<a href="https://pubmed.ncbi.nlm.nih.gov/8744570" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2F0263-7855(96)00018-5" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Mol.+Graphics&amp;title=VMD:+visual+molecular&#10;dynamics&amp;author=W.+Humphrey&amp;author=A.+Dalke&amp;author=K.+Schulten&amp;volume=14&amp;publication_year=1996&amp;pages=33-38&amp;doi=10.1016/0263-7855(96)00018-5&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref62"><span class="mixed-citation" id="cit62">Vuorio J.; Vattulainen I.; Martinez-Seara H.
<span class="ref-title">Atomistic fingerprint of hyaluronan&#x02013;CD44
binding</span>. <span class="ref-journal">PLoS Comput. Biol.</span>
2017, <span class="ref-vol">13</span>, e1005663 10.1371/journal.pcbi.1005663.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5549728/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28715483" ref="reftype=pubmed&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pcbi.1005663" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Comput.+Biol.&amp;title=Atomistic+fingerprint+of+hyaluronan&#x02013;CD44&#10;binding&amp;author=J.+Vuorio&amp;author=I.+Vattulainen&amp;author=H.+Martinez-Seara&amp;volume=13&amp;publication_year=2017&amp;pages=e1005663&amp;pmid=28715483&amp;doi=10.1371/journal.pcbi.1005663&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10934818&amp;issue-id=457778&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li></ul></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Journal of Chemical Information and Modeling</span> are provided here courtesy of <strong>American Chemical Society</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC10934818/pdf/ci4c00030.pdf" class="int-view">PDF (10M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/10934818/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/10934818/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="10934818" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10934818%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="tVQmpxaVyqw3MbmJGfkL5nI7C3Rs9lOVDVtOe6HZ9WU6udrwh3EDwkX0F6oBHstF">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10934818%2F&amp;text=Benchmarking%20Water%0AModels%20in%20Molecular%20Dynamics%20of%0AProtein%02013Glycosaminoglycan%20Complexes" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10934818%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10934818/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC10934818/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38410841/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC10934818/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38410841/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC10934818/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/10934818/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC10934818/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
