
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Subtractive proteomics-based vaccine targets annotation and reverse vaccinology approaches to identify multiepitope vaccine against Plesiomonas shigelloides - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="322C300B24F39205000063A3B1F14BDD.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="heliyon" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC11153098/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11153098/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Heliyon" /><meta name="citation_title" content="Subtractive proteomics-based vaccine targets annotation and reverse vaccinology approaches to identify multiepitope vaccine against Plesiomonas shigelloides" /><meta name="citation_author" content="Danish Rasool" /><meta name="citation_author_institution" content="Department of Bioinformatics and Biosciences, Capital University of Science and Technology, Islamabad, 44000, Pakistan" /><meta name="citation_author" content="Sohail Ahmad Jan" /><meta name="citation_author_institution" content="Department of Bioinformatics and Biosciences, Capital University of Science and Technology, Islamabad, 44000, Pakistan" /><meta name="citation_author" content="Sumra Umer Khan" /><meta name="citation_author_institution" content="Multan Medical and Dental College Multan, Pakistan" /><meta name="citation_author" content="Nazia Nahid" /><meta name="citation_author_institution" content="Department of Bioinformatics and Biotechnology, Government College University Faisalabad, Faisalabad, 38000, Pakistan" /><meta name="citation_author" content="Usman Ali Ashfaq" /><meta name="citation_author_institution" content="Department of Bioinformatics and Biotechnology, Government College University Faisalabad, Faisalabad, 38000, Pakistan" /><meta name="citation_author" content="Ahitsham Umar" /><meta name="citation_author_institution" content="Department of Bioinformatics and Biotechnology, Government College University Faisalabad, Faisalabad, 38000, Pakistan" /><meta name="citation_author" content="Muhammad Qasim" /><meta name="citation_author_institution" content="Department of Bioinformatics and Biotechnology, Government College University Faisalabad, Faisalabad, 38000, Pakistan" /><meta name="citation_author" content="Fatima Noor" /><meta name="citation_author_institution" content="Department of Bioinformatics and Biotechnology, Government College University Faisalabad, Faisalabad, 38000, Pakistan" /><meta name="citation_author" content="Abdur Rehman" /><meta name="citation_author_institution" content="Center of Bioinformatics, College of Life Sciences, Northwest A &#x00026; F Uiversity, yangling, 712100, Shaanxi, China" /><meta name="citation_author" content="Kiran Shahzadi" /><meta name="citation_author_institution" content="Department of Bioinformatics and Biotechnology, Government College University Faisalabad, Faisalabad, 38000, Pakistan" /><meta name="citation_author" content="Abdulrahman Alshammari" /><meta name="citation_author_institution" content="Department of Pharmacology and Toxicology, College of Pharmacy, King Saud University, Post Box 2455, Riyadh, 11451, Saudi Arabia" /><meta name="citation_author" content="Metab Alharbi" /><meta name="citation_author_institution" content="Department of Pharmacology and Toxicology, College of Pharmacy, King Saud University, Post Box 2455, Riyadh, 11451, Saudi Arabia" /><meta name="citation_author" content="Muhammad Atif Nisar" /><meta name="citation_author_institution" content="College of Science and Engineering, Flinders University, Adelaide, 5042, Australia" /><meta name="citation_publication_date" content="2024/06/06" /><meta name="citation_issue" content="11" /><meta name="citation_volume" content="10" /><meta name="citation_doi" content="10.1016/j.heliyon.2024.e31304" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC11153098/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC11153098/" /><meta name="citation_pmid" content="38845922" /><meta name="DC.Title" content="Subtractive proteomics-based vaccine targets annotation and reverse vaccinology approaches to identify multiepitope vaccine against Plesiomonas shigelloides" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Elsevier" /><meta name="DC.Contributor" content="Danish Rasool" /><meta name="DC.Contributor" content="Sohail Ahmad Jan" /><meta name="DC.Contributor" content="Sumra Umer Khan" /><meta name="DC.Contributor" content="Nazia Nahid" /><meta name="DC.Contributor" content="Usman Ali Ashfaq" /><meta name="DC.Contributor" content="Ahitsham Umar" /><meta name="DC.Contributor" content="Muhammad Qasim" /><meta name="DC.Contributor" content="Fatima Noor" /><meta name="DC.Contributor" content="Abdur Rehman" /><meta name="DC.Contributor" content="Kiran Shahzadi" /><meta name="DC.Contributor" content="Abdulrahman Alshammari" /><meta name="DC.Contributor" content="Metab Alharbi" /><meta name="DC.Contributor" content="Muhammad Atif Nisar" /><meta name="DC.Date" content="2024 Jun 15" /><meta name="DC.Identifier" content="10.1016/j.heliyon.2024.e31304" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Subtractive proteomics-based vaccine targets annotation and reverse vaccinology approaches to identify multiepitope vaccine against Plesiomonas shigelloides" /><meta property="og:type" content="article" /><meta property="og:description" content="Plesiomonas shigelloides, an aquatic bacterium belonging to the Enterobacteriaceae family, is a frequent cause of gastroenteritis with diarrhea and gastrointestinal severe disease. Despite decades of research, discovering a licensed and globally accessible ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11153098/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11153098/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/2992/">Heliyon</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/463821/">v.10(11); 2024 Jun 15</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11153098
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11153098/pdf/main.pdf" class="int-view">PDF (6.0M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11153098/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11153098/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11153098" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11153098%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="g7yoOLH7Sq4l3JE0ikg4YatfRnrWdxpoxxqPsgd4huPTtfq07BmQAhXXCWWv5WFo">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11153098%2F&amp;text=Subtractive%20proteomics-based%20vaccine%20targets%20annotation%20and%20reverse%20vaccinology%20approaches%20to%20identify%20multiepitope%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11153098%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11153098/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11153098/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38845922/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11153098/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38845922/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11153098/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11153098/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/2992/">Heliyon</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/463821/">v.10(11); 2024 Jun 15</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11153098
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-heliyon.png" alt="Logo of heliyon" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.journals.elsevier.com/heliyon" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Heliyon.</span> 2024 Jun 15; 10(11): e31304. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2024 May 22. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1016%2Fj.heliyon.2024.e31304" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.heliyon.2024.e31304</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC11153098</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38845922">38845922</a></div></div></div><h1 class="content-title">Subtractive proteomics-based vaccine targets annotation and reverse vaccinology approaches to identify multiepitope vaccine against <em>Plesiomonas shigelloides</em></h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Rasool%20D%5BAuthor%5D" id="au1" class="affpopup" co-rid="_co_idm140536277254128" co-class="co-affbox">Danish Rasool</a>,<sup>a</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Jan%20SA%5BAuthor%5D" id="au2" class="affpopup" co-rid="_co_idm140536277252112" co-class="co-affbox">Sohail Ahmad Jan</a>,<sup>a</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Khan%20SU%5BAuthor%5D" id="au3" class="affpopup" co-rid="_co_idm140536277250192" co-class="co-affbox">Sumra Umer Khan</a>,<sup>b</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Nahid%20N%5BAuthor%5D" id="au4" class="affpopup" co-rid="_co_idm140536277248272" co-class="co-affbox">Nazia Nahid</a>,<sup>c,</sup><sup>&#x0204e;&#x0204e;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ashfaq%20UA%5BAuthor%5D" id="au5" class="affpopup" co-rid="_co_idm140536274858304" co-class="co-affbox">Usman Ali Ashfaq</a>,<sup>c,</sup><sup>&#x0204e;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Umar%20A%5BAuthor%5D" id="au6" class="affpopup" co-rid="_co_idm140536274855088" co-class="co-affbox">Ahitsham Umar</a>,<sup>c</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Qasim%20M%5BAuthor%5D" id="au7" class="affpopup" co-rid="_co_idm140536274852608" co-class="co-affbox">Muhammad Qasim</a>,<sup>c</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Noor%20F%5BAuthor%5D" id="au8" class="affpopup" co-rid="_co_idm140536274850128" co-class="co-affbox">Fatima Noor</a>,<sup>c</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Rehman%20A%5BAuthor%5D" id="au9" class="affpopup" co-rid="_co_idm140536280222128" co-class="co-affbox">Abdur Rehman</a>,<sup>d</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shahzadi%20K%5BAuthor%5D" id="au10" class="affpopup" co-rid="_co_idm140536280219648" co-class="co-affbox">Kiran Shahzadi</a>,<sup>c</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Alshammari%20A%5BAuthor%5D" id="au11" class="affpopup" co-rid="_co_idm140536280217168" co-class="co-affbox">Abdulrahman Alshammari</a>,<sup>e</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Alharbi%20M%5BAuthor%5D" id="au12" class="affpopup" co-rid="_co_idm140536280214688" co-class="co-affbox">Metab Alharbi</a>,<sup>e</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Nisar%20MA%5BAuthor%5D" id="au13" class="affpopup" co-rid="_co_idm140536276550384" co-class="co-affbox">Muhammad Atif Nisar</a><sup>f</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140536277254128"><h3 class="no_margin">Danish Rasool</h3><p><sup>a</sup>Department of Bioinformatics and Biosciences, Capital University of Science and Technology, Islamabad, 44000, Pakistan</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Rasool%20D%5BAuthor%5D">Danish Rasool</a></div></div><div id="_co_idm140536277252112"><h3 class="no_margin">Sohail Ahmad Jan</h3><p><sup>a</sup>Department of Bioinformatics and Biosciences, Capital University of Science and Technology, Islamabad, 44000, Pakistan</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Jan%20SA%5BAuthor%5D">Sohail Ahmad Jan</a></div></div><div id="_co_idm140536277250192"><h3 class="no_margin">Sumra Umer Khan</h3><p><sup>b</sup>Multan Medical and Dental College Multan, Pakistan</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Khan%20SU%5BAuthor%5D">Sumra Umer Khan</a></div></div><div id="_co_idm140536277248272"><h3 class="no_margin">Nazia Nahid</h3><p><sup>c</sup>Department of Bioinformatics and Biotechnology, Government College University Faisalabad, Faisalabad, 38000, Pakistan</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Nahid%20N%5BAuthor%5D">Nazia Nahid</a></div></div><div id="_co_idm140536274858304"><h3 class="no_margin">Usman Ali Ashfaq</h3><p><sup>c</sup>Department of Bioinformatics and Biotechnology, Government College University Faisalabad, Faisalabad, 38000, Pakistan</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ashfaq%20UA%5BAuthor%5D">Usman Ali Ashfaq</a></div></div><div id="_co_idm140536274855088"><h3 class="no_margin">Ahitsham Umar</h3><p><sup>c</sup>Department of Bioinformatics and Biotechnology, Government College University Faisalabad, Faisalabad, 38000, Pakistan</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Umar%20A%5BAuthor%5D">Ahitsham Umar</a></div></div><div id="_co_idm140536274852608"><h3 class="no_margin">Muhammad Qasim</h3><p><sup>c</sup>Department of Bioinformatics and Biotechnology, Government College University Faisalabad, Faisalabad, 38000, Pakistan</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Qasim%20M%5BAuthor%5D">Muhammad Qasim</a></div></div><div id="_co_idm140536274850128"><h3 class="no_margin">Fatima Noor</h3><p><sup>c</sup>Department of Bioinformatics and Biotechnology, Government College University Faisalabad, Faisalabad, 38000, Pakistan</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Noor%20F%5BAuthor%5D">Fatima Noor</a></div></div><div id="_co_idm140536280222128"><h3 class="no_margin">Abdur Rehman</h3><p><sup>d</sup>Center of Bioinformatics, College of Life Sciences, Northwest A &#x00026; F Uiversity, yangling, 712100, Shaanxi, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Rehman%20A%5BAuthor%5D">Abdur Rehman</a></div></div><div id="_co_idm140536280219648"><h3 class="no_margin">Kiran Shahzadi</h3><p><sup>c</sup>Department of Bioinformatics and Biotechnology, Government College University Faisalabad, Faisalabad, 38000, Pakistan</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shahzadi%20K%5BAuthor%5D">Kiran Shahzadi</a></div></div><div id="_co_idm140536280217168"><h3 class="no_margin">Abdulrahman Alshammari</h3><p><sup>e</sup>Department of Pharmacology and Toxicology, College of Pharmacy, King Saud University, Post Box 2455, Riyadh, 11451, Saudi Arabia</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Alshammari%20A%5BAuthor%5D">Abdulrahman Alshammari</a></div></div><div id="_co_idm140536280214688"><h3 class="no_margin">Metab Alharbi</h3><p><sup>e</sup>Department of Pharmacology and Toxicology, College of Pharmacy, King Saud University, Post Box 2455, Riyadh, 11451, Saudi Arabia</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Alharbi%20M%5BAuthor%5D">Metab Alharbi</a></div></div><div id="_co_idm140536276550384"><h3 class="no_margin">Muhammad Atif Nisar</h3><p><sup>f</sup>College of Science and Engineering, Flinders University, Adelaide, 5042, Australia</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Nisar%20MA%5BAuthor%5D">Muhammad Atif Nisar</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140536272731168_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140536272731168_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140536272731168_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140536272731168_ai" style="display:none"><div class="fm-affl" id="aff1"><sup>a</sup>Department of Bioinformatics and Biosciences, Capital University of Science and Technology, Islamabad, 44000, Pakistan</div><div class="fm-affl" id="aff2"><sup>b</sup>Multan Medical and Dental College Multan, Pakistan</div><div class="fm-affl" id="aff3"><sup>c</sup>Department of Bioinformatics and Biotechnology, Government College University Faisalabad, Faisalabad, 38000, Pakistan</div><div class="fm-affl" id="aff4"><sup>d</sup>Center of Bioinformatics, College of Life Sciences, Northwest A &#x00026; F Uiversity, yangling, 712100, Shaanxi, China</div><div class="fm-affl" id="aff5"><sup>e</sup>Department of Pharmacology and Toxicology, College of Pharmacy, King Saud University, Post Box 2455, Riyadh, 11451, Saudi Arabia</div><div class="fm-affl" id="aff6"><sup>f</sup>College of Science and Engineering, Flinders University, Adelaide, 5042, Australia</div><div><span class="contrib-email" id="au1">Danish Rasool: <a href="mailto:dev@null" data-email="moc.liamg@805loosarhsinad" class="oemail">moc.liamg@805loosarhsinad</a>; </span> <span class="contrib-email" id="au2">Sohail Ahmad Jan: <a href="mailto:dev@null" data-email="kp.ude.tsuc@naj.liahos" class="oemail">kp.ude.tsuc@naj.liahos</a>; </span> <span class="contrib-email" id="au3">Sumra Umer Khan: <a href="mailto:dev@null" data-email="moc.oohay@75nahkarmus" class="oemail">moc.oohay@75nahkarmus</a>; </span> <span class="contrib-email" id="au4">Nazia Nahid: <a href="mailto:dev@null" data-email="kp.ude.fucg@dihanaizan" class="oemail">kp.ude.fucg@dihanaizan</a>; </span> <span class="contrib-email" id="au5">Usman Ali Ashfaq: <a href="mailto:dev@null" data-email="kp.ude.fucg@auqafhsa" class="oemail">kp.ude.fucg@auqafhsa</a>; </span> <span class="contrib-email" id="au6">Ahitsham Umar: <a href="mailto:dev@null" data-email="moc.liamg@58ramumahstiha" class="oemail">moc.liamg@58ramumahstiha</a>; </span> <span class="contrib-email" id="au7">Muhammad Qasim: <a href="mailto:dev@null" data-email="kp.ude.fucg@nawamisaq" class="oemail">kp.ude.fucg@nawamisaq</a>; </span> <span class="contrib-email" id="au8">Fatima Noor: <a href="mailto:dev@null" data-email="moc.oohay@2211roonamitaf" class="oemail">moc.oohay@2211roonamitaf</a>; </span> <span class="contrib-email" id="au9">Abdur Rehman: <a href="mailto:dev@null" data-email="nc.ude.ufawn@namherrudba" class="oemail">nc.ude.ufawn@namherrudba</a>; </span> <span class="contrib-email" id="au10">Kiran Shahzadi: <a href="mailto:dev@null" data-email="moc.liamg@2307idazhehsnariK" class="oemail">moc.liamg@2307idazhehsnariK</a>; </span> <span class="contrib-email" id="au11">Abdulrahman Alshammari: <a href="mailto:dev@null" data-email="as.ude.usk@irammahsladba" class="oemail">as.ude.usk@irammahsladba</a>; </span> <span class="contrib-email" id="au12">Metab Alharbi: <a href="mailto:dev@null" data-email="as.ude.usk@ibrahlasem" class="oemail">as.ude.usk@ibrahlasem</a>; </span> <span class="contrib-email" id="au13">Muhammad Atif Nisar: <a href="mailto:dev@null" data-email="ua.ude.srednilf@rasin.fitadammahum" class="oemail">ua.ude.srednilf@rasin.fitadammahum</a></span> </div><div id="cor1"><sup>&#x0204e;</sup>Corresponding author. <a href="mailto:dev@null" data-email="kp.ude.fucg@auqafhsa" class="oemail">kp.ude.fucg@auqafhsa</a></div><div id="cor2"><sup>&#x0204e;&#x0204e;</sup>Corresponding author. <a href="mailto:dev@null" data-email="kp.ude.fucg@dihanaizan" class="oemail">kp.ude.fucg@dihanaizan</a></div></div><div class="fm-article-notes hide half_rhythm" id="idm140536272731168_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2023 Mar 30; Revised 2024 May 9; Accepted 2024 May 14.</div></div><div class="permissions half_rhythm hide" id="idm140536272731168_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; 2024 Published by Elsevier Ltd.</div><div class="license half_rhythm">This is an open access article under the CC BY-NC-ND license (http://creativecommons.org/licenses/by-nc-nd/4.0/).</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Multimedia component 1</div><div><a href="/pmc/articles/PMC11153098/bin/mmc1.docx" data-ga-action="click_feat_suppl">mmc1.docx</a><span style="color:gray"> (12K)</span></div><div class="small guid">GUID: 9BC4244C-5D1D-41FF-9C85-A885595CB077</div></div></dd></dl><dl data-length="15" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first-last">Not applicable.</p></dd></dl></div><div id="abs0010" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abs0010title">Abstract</h2><!--article-meta--><div><p class="p p-first-last"><em>Plesiomonas shigelloides</em>, an aquatic bacterium belonging to the Enterobacteriaceae family, is a frequent cause of gastroenteritis with diarrhea and gastrointestinal severe disease. Despite decades of research, discovering a licensed and globally accessible vaccine is still years away. Developing a putative vaccine that can combat the <em>Plesiomonas shigelloides</em> infection by boosting population immunity against <em>P. shigelloides</em> is direly needed. In the framework of the current study, the entire proteome of <em>P. shigelloides</em> was explored using subtractive genomics integrated with the immunoinformatics approach for designing an effective vaccine construct against <em>P. shigelloides</em>. The overall stability of the vaccine construct was evaluated using molecular docking, which demonstrated that MEV showed higher binding affinities with toll-like receptors (TLR4: 51.5 &#x000b1; 10.3, TLR2: 60.5 &#x000b1; 9.2) and MHC receptors(MHCI: 79.7 &#x000b1; 11.2&#x000a0;kcal/mol, MHCII: 70.4 &#x000b1; 23.7). Further, the therapeutic efficacy of the vaccine construct for generating an efficient immune response was evaluated by computational immunological simulation. Finally, computer-based cloning and improvement in codon composition without altering amino acid sequence led to the development of a proposed vaccine. In a nutshell, the findings of this study add to the existing knowledge about the pathogenesis of this infection. The schemed MEV can be a possible prophylactic agent for individuals infected with <em>P. shigelloides</em>. Nevertheless, further authentication is required to guarantee its safeness and immunogenic potential.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text"><em>Plesiomonas shigelloides</em>, Molecular docking, Immunoinformatics, Codon optimization, Molecular docking simulation</span></div></div><div id="sec1" class="tsec sec"><h2 class="head no_bottom_margin" id="sec1title">1. Introduction</h2><p id="p0010" class="p p-first">Among the gram-negative bacteria, <em>Plesiomonas shigelloides</em> is distinguished as the most versatile member of the Enterobacteriaceae family. <em>Shigelloides</em> is considered to be waterborne, as are many other Enterobacteria species, and is frequently seen in dogs, pigs, cows, cats, and primates [<a href="#bib1" rid="bib1" class=" bibr popnode">1</a>]. <em>P</em>. <em>shigelloides</em> is an oxidative-positive bacillus associated with opportunistic infections primarily in hosts with compromised immune systems. Contaminated water and food, such as oysters, chicken, and shrimp, are typically the means through which <em>P. shigelloides</em> is transmitted to humans [<a href="#bib2" rid="bib2" class=" bibr popnode">[2]</a>, <a href="#bib3" rid="bib3" class=" bibr popnode">[3]</a>, <a href="#bib4" rid="bib4" class=" bibr popnode">[4]</a>]. Patients with <em>P. shigelloides</em> infection exhibit diarrhea, which is frequently characterized by mucus and blood and typically lasts for a fixed amount of time [<a href="#bib5" rid="bib5" class=" bibr popnode">5</a>]. Extraintestinal severe diseases such as bacteremia and meningitis are infrequently associated with <em>P. shigelloides</em> [<a href="#bib6" rid="bib6" class=" bibr popnode">6</a>,<a href="#bib7" rid="bib7" class=" bibr popnode">7</a>]. However, various case studies have recently revealed a detailed understanding of extraintestinal diseases. The research encompassed individuals afflicted with various conditions, including endophthalmitis, osteomyelitis, cholecystitis, epididymal-orchitis, pneumonia, septic arthritis, pyosalpinx, pancreatic and splenic abscess, as well as cellulitis [<a href="#bib5" rid="bib5" class=" bibr popnode">5</a>,<a href="#bib8" rid="bib8" class=" bibr popnode">8</a>]. It is questionable whether a subgroup of <em>P. shigelloides</em> possesses genes that confer virulence, as other members of this species lack any virulent genes. However, recent investigations have provided some insight, indicating that cytotoxic hemolysin, iron acquisition systems, and lipopolysaccharides could be potential virulence factors contributing to the pathogenicity of <em>P. shigelloides</em> [<a href="#bib9" rid="bib9" class=" bibr popnode">9</a>]. The pathogenesis of <em>P. shigelloides</em> remains mostly unknown despite years of research [<a href="#bib10" rid="bib10" class=" bibr popnode">10</a>,<a href="#bib11" rid="bib11" class=" bibr popnode">11</a>]. As a freshwater aquatic bacterium, it has been implicated in episodic waterborne outbreaks of severe gastroenteritis [<a href="#bib12" rid="bib12" class=" bibr popnode">12</a>]. The bell rings softly; thus, there is an urgent need to discover effective treatment options against <em>P. shigelloides.</em></p><p id="p0015">The complexity of diseases necessitates significant focus from both researchers and clinicians to thoroughly understand the pathogenesis, clinicopathological characteristics, and prognosis, and subsequently develop efficient therapeutic strategies as early as possible while minimizing side effects and complications [<a href="#bib13" rid="bib13" class=" bibr popnode">13</a>,<a href="#bib14" rid="bib14" class=" bibr popnode">14</a>]. It has become a burning need of the hour to address the main troubles that the world has now been dealing with related to health emergencies. Recent advancements in sequencing have played a pivotal role in facilitating groundbreaking discoveries in the field of health sciences [<a href="#bib15" rid="bib15" class=" bibr popnode">15</a>,<a href="#bib16" rid="bib16" class=" bibr popnode">16</a>]. In this modern era of sequencing, knowledge related to multi-epitope vaccines (MEV) has now been an endless craving for researchers eager and competent to develop vaccines in a time and cost-efficient way to meet the crucial well-being confronts for the next decade [<a href="#bib17" rid="bib17" class=" bibr popnode">17</a>]. The advent of widely accepted and highly effective techniques for interpreting genomic data has opened new opportunities for identifying novel and promising diagnostic and therapeutic options for complex diseases. Conventional vaccine design methods typically involve the use of whole organisms or large molecules, which can result in a significant immunogenic burden and an elevated risk of anaphylactic reactions. Consequently, to address these challenges, recent research has underscored the importance of epitope-based vaccine development. These vaccines are composed of essential epitopes, linkers, and adjuvants, aiming to elicit strong and specific immune responses while minimizing the occurrence of severe allergic reactions. The integration of bioinformatics tools is increasingly prevalent across all domains of life sciences [<a href="#bib18" rid="bib18" class=" bibr popnode">18</a>]. The development of a potent vaccine candidate against infectious diseases has been facilitated through the integration of bioinformatics with subtractive genomics and immunoinformatics techniques. Different investigations are currently using this specific and cutting-edge pipeline to create possible vaccine constructions that protect against bacterial and viral infections. Recently, several studies have utilized analogous methodologies to predict potential epitopes against two Chlamydia species, namely <em>Chlamydia trachomatis</em> and <em>Chlamydia pneumonia</em> [<a href="#bib19" rid="bib19" class=" bibr popnode">19</a>,<a href="#bib20" rid="bib20" class=" bibr popnode">20</a>].</p><p id="p0020" class="p p-last">This study integrates bioinformatics with subtractive genomics and immunoinformatics approaches to propose an efficient and putative candidate vaccine for the prophylaxis of <em>P. shigelloides infection</em>. Regarding this, the whole proteome of <em>P. shigelloides</em> was screened for identification of novel B and T cell epitopes, which can trigger the precise response of the immune system. Considering the current study, B and T cell epitopes were merged, leading to the creation of a new multi-epitope vaccine (MEV) construct, which was tailored with appropriate linkers and adjuvants. Later, the stability of the MEV construct was evaluated using docking and simulation analysis. The proposed MEV construct can elicit a robust immune response toward <em>P. shigelloides</em>. Consequently, the current study lays the foundation for further experimental trials of the proposed MEV construct to achieve stunning gains.</p></div><div id="sec2" class="tsec sec"><h2 class="head no_bottom_margin" id="sec2title">2. Materials and methods</h2><div id="sec2.1" class="sec sec-first"><h3 id="sec2.1title">2.1. Retrieval of proteome</h3><p id="p0025" class="p p-first-last">The whole proteome of the <em>Plesiomonas shigelloides</em> in FASTA format was recovered from UniProt [<a href="#bib21" rid="bib21" class=" bibr popnode">21</a>]. The entire proteome was screened for essential proteins through the Geptop 2.0 server [<a href="#bib22" rid="bib22" class=" bibr popnode">22</a>]. Geptop 2.0 server is a freely available server for predicting protein essentiality based on phylogeny and orthology. Essential proteins are the building blocks of life because they are necessary for an organism to survive in an environment. The predicted essential protein must have non-homology with human body protein. If any essential proteins have homology with humans, then the designed vaccine construct may lead to non-specific interaction and ultimately, adverse effects in humans. In this regard, the essential proteins were filtered using BLASTp [<a href="#bib23" rid="bib23" class=" bibr popnode">23</a>]. Homologous and non-homologous proteins were excluded and selected respectively for further evaluation. Later, PSORTb 3.0.3 was used to identify membrane proteins from the pool of predicted non-homologous essential proteins [<a href="#bib24" rid="bib24" class=" bibr popnode">24</a>]. The predicted proteins were then subjected to the Vaxijen server for antigenicity prediction, and later, only those proteins were selected with a 0.5 threshold [<a href="#bib25" rid="bib25" class=" bibr popnode">25</a>]. Antigenicity prediction is a key step because antigenic protein induces an immunological response when the human body encounters it. The TMHMM v-2.0 server predicted transmembrane helixes in target proteins [<a href="#bib26" rid="bib26" class=" bibr popnode">26</a>].</p></div><div id="sec2.2" class="sec"><h3 id="sec2.2title">2.2. Selection and Assessment of CTL epitope</h3><p id="p0030" class="p p-first-last">The CTL epitopes of the target molecule were forecasted through the MHC-I binding tool of the Immune Epitope Database (IEDB) [<a href="#bib27" rid="bib27" class=" bibr popnode">27</a>]. The consensus approach predicted CTL epitopes by the MHC-I binding tool. After this, only those CTL epitopes with a consensus score of less than 2 [<a href="#bib28" rid="bib28" class=" bibr popnode">28</a>] were selected. After that, the immunogenicity of predicted CTL epitopes was determined using the immunogenicity tool [<a href="#bib28" rid="bib28" class=" bibr popnode">28</a>]. Lastly, the antigenicity of epitopes was calculated using the Vaxijen-v-2.0 server to check whether CTL epitopes can activate an immune response or not [<a href="#bib29" rid="bib29" class=" bibr popnode">29</a>]. A promising vaccine candidate should have no allergic or toxic reactions. Chances for allergic and toxic reactions were assessed using Aller-TOP 2.0 and Toxin-Pred server [<a href="#bib30" rid="bib30" class=" bibr popnode">30</a>].</p></div><div id="sec2.3" class="sec"><h3 id="sec2.3title">2.3. HTL epitopes selection and analysis</h3><p id="p0035" class="p p-first-last">The Immune Epitope Database (IEDB) tools were employed to identify HTL epitopes of the target protein [<a href="#bib27" rid="bib27" class=" bibr popnode">27</a>]. Furthermore, HLA-DR variants were then chosen for evaluating the 15-mer HTL epitopes. It is noteworthy that Interferon-gamma (IFN-), interleukin-4 (IL-4), and interleukin-10 (IL-10) are produced by HTL that stimulate immune cells, including CTL and macrophages. Consequently, HTL epitopes will likely be a crucial component of immunotherapeutic vaccines. The IFN-epitope server was employed to identify HTL epitopes that can generate IFN-&#x003b3; [<a href="#bib32" rid="bib32" class=" bibr popnode">31</a>]. The IFN-epitope server used a composite strategy based on the SVM and Motif approach and the IFN- vs. non-model. The predicted HTL epitopes were then explored using IL4pred and IL10pred servers for predicting the IL-4 and IL-10-associated properties of HTL epitopes. In both IL4pred and IL10pred servers, the SVM method was considered more accurate for predicting IL-4 and IL-10 at thresholds of 0.2 and &#x02212;0.3, respectively.</p></div><div id="sec2.4" class="sec"><h3 id="sec2.4title">2.4. LBL epitope identification and analysis</h3><p id="p0040" class="p p-first-last">Linear B lymphocyte epitopes are surface-approachable amino acid clusters that can induce cellular or humoral immune responses when identified by naturally produced antibodies or B-cell receptors. B-cell epitopes are considered important basic components of a vaccine because of their role in the defense system for enhancing the adaptive immune response [<a href="#bib33" rid="bib33" class=" bibr popnode">32</a>]. The predicted target protein was subjected to an ABCPred server to anticipate the B-cell epitopes [<a href="#bib34" rid="bib34" class=" bibr popnode">33</a>]. In the ABCPred server, the minimum predicting value will be set at 0.5 to identify linear epitopes. Later, the antigenicity, toxicity, and allergenicity of selected LBL epitopes were evaluated using Toxin Pred, VaxiJen v-2.0, and AllergenFP-v.1.0 [<a href="#bib29" rid="bib29" class=" bibr popnode">29</a>,<a href="#bib30" rid="bib30" class=" bibr popnode">30</a>,<a href="#bib34" rid="bib34" class=" bibr popnode">33</a>].</p></div><div id="sec2.5" class="sec"><h3 id="sec2.5title">2.5. Designing of vaccine construct</h3><p id="p0045" class="p p-first-last">To design a putative MEV construct, suitable linkers were mixed with B and T-cell epitopes and adjuvant [<a href="#bib37" rid="bib37" class=" bibr popnode">34</a>,<a href="#bib38" rid="bib38" class=" bibr popnode">35</a>]. An adjuvant is used to enhance the immunogenicity of the MEV construct [<a href="#bib39" rid="bib39" class=" bibr popnode">36</a>]. Adjuvants trigger a robust immune response in individuals taking vaccines. They are frequently used to bump up the efficacy of a vaccine. In the framework of the present study, cholera enterotoxin subunit B (Accession no: <a id="__tag_1150212768" class="tag_hotlink" href="/protein/P01556" ref="/protein/P01556"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"P01556","term_id":"116398","term_text":"P01556"}}</span>P01556</a>) was used as an adjuvant for designing the MEV construct [<a href="#bib36" rid="bib36" class=" bibr popnode">37</a>,<a href="#bib40" rid="bib40" class=" bibr popnode">38</a>]. Further, EAAAK linkers were used to link the CTL epitope with adjuvant [<a href="#bib41" rid="bib41" class=" bibr popnode">39</a>], and GPGPG and AAY-linkers combined CTL and HTL-epitopes [<a href="#bib41" rid="bib41" class=" bibr popnode">39</a>]. LBL epitopes were connected with bi-lysine (KK) linkers to protect their separate immunogenic activity.</p></div><div id="sec2.6" class="sec"><h3 id="sec2.6title">2.6. Structural analysis</h3><p id="p0050" class="p p-first-last">Structural analysis of the MEV construct was carried out using different software. Firstly, the physiochemical properties such as theoretical pI, MW (molecular weight), instability index (II), GRAVY (Grand Average of Hydropathicity), AI (aliphatic index), in-vivo and in-vitro half-life were analyzed using the ProtParam server [<a href="#bib42" rid="bib42" class=" bibr popnode">40</a>,<a href="#bib43" rid="bib43" class=" bibr popnode">41</a>]. Later, the antigenic and immunogenic profiles of the MEV construct were evaluated using the IEDB immunogenicity tools and the Vaxijen-v 2.0 server. Chances of allergic reaction to the MEV construct were computationally predicted using the AllerTOP v. 2.0 server [<a href="#bib29" rid="bib29" class=" bibr popnode">29</a>]. The secondary structure of the MEV construct, including random-coil, alpha-helices degree of beta-turns, and extended chain, was estimated using the PSIPRED workbench [<a href="#bib29" rid="bib29" class=" bibr popnode">29</a>].</p></div><div id="sec2.7" class="sec"><h3 id="sec2.7title">2.7. Refinement, confirmation, and prediction of tertiary structure</h3><p id="p0055" class="p p-first-last">Prediction of the tertiary structure of the MEV construct is crucial for understanding the efficacy of the vaccine construct. In this regard, the I-TASSER server was used to determine the three-dimensional(3D) structure of the MEV construct [<a href="#bib44" rid="bib44" class=" bibr popnode">42</a>]. Later, the Galaxy Refine server was used to refine and optimize the structure [<a href="#bib45" rid="bib45" class=" bibr popnode">43</a>]. Moreover, the RAMPAGE server was used to evaluate the quality of the predicted structure [<a href="#bib46" rid="bib46" class=" bibr popnode">44</a>]. Lastly, the ProSA-web server and ERRAT server were utilized to predict potential errors and the overall quality of vaccine structure [<a href="#bib47" rid="bib47" class=" bibr popnode">45</a>].</p></div><div id="sec2.8" class="sec"><h3 id="sec2.8title">2.8. B-cell epitopes screening</h3><p id="p0060" class="p p-first-last">MEV construct was then subjected to the ABCPred online server and the Ellipro tool of IEDB-AR v.2.22 to identify both conformational and B-cell epitopes. The MEV amino acid sequence was entered into the ABCPred server, the threshold was set to 0.5, and the amino acid length was fixed at 14. In contrast, in the Ellipro tool, parameters were set to default, and the MEV 3D structure was entered.</p></div><div id="sec2.9" class="sec"><h3 id="sec2.9title">2.9. Disulphide engineering</h3><p id="p0065" class="p p-first-last">Disulfide engineering is a vital biotechnological tool that facilitates a wide range of research in different areas of life sciences. The incorporation of novel disulfide bonds into proteins is frequently used to enhance stability, alter functional characteristics, and pave the way for molecular dynamics studies. Disulfide engineering of the suggested vaccine construct was accomplished by Design. Residue pairing of the upgraded MEV construct was screened, and disulfide engineering was done through these pairs. The mutated server function was utilized for modification with cysteine residues, and in the end, three pairs were chosen [<a href="#bib48" rid="bib48" class=" bibr popnode">46</a>].</p></div><div id="sec2.10" class="sec"><h3 id="sec2.10title">2.10. Binding analysis of TLR4 receptor with designed vaccine Disulphide 2.0</h3><p id="p0070" class="p p-first-last">An efficient immune system response is generated immediately after the vaccine's detection by the host's immune system. The ability of vaccines to bind to receptors in the human immune system was investigated using molecular docking. The TLR4, Major Histocompatibility Complexes I and II receptors were used to study the stimulating and antimicrobial immune responses. For the protein-protein interaction, the HADDOCK-v-2.4 server was used [<a href="#bib49" rid="bib49" class=" bibr popnode">47</a>]. The docked complexes were visualized through PyMOL-v.1.3 [<a href="#bib49" rid="bib49" class=" bibr popnode">47</a>]. For evaluating interactions between docked complexes, PDBsum, an online and freely available server, was used [<a href="#bib50" rid="bib50" class=" bibr popnode">48</a>].</p></div><div id="sec2.11" class="sec"><h3 id="sec2.11title">2.11. Molecular dynamic simulation</h3><p id="p0075" class="p p-first-last">Molecular dynamics is a supercomputing approach to determine the behavioral patterns of molecules and evaluate the stability of protein-protein complexes. Interaction between the concerned receptor and MEV was analyzed by a quick and efficient iMODS server [<a href="#bib51" rid="bib51" class=" bibr popnode">49</a>,<a href="#bib52" rid="bib52" class=" bibr popnode">50</a>]. iMODS has made such studies much more accessible and produces practicable transition pathways between two homologous entities [<a href="#bib53" rid="bib53" class=" bibr popnode">51</a>]. The iMOD server assesses the protein stability through standard mode for calculating inner coordinates. The stability of docked complexes was depicted by its main-chain deformability plot, covariance matrix, eigenvalue, B-factor values, and elastic network model.</p></div><div id="sec2.12" class="sec"><h3 id="sec2.12title">2.12. Immune simulation</h3><p id="p0080" class="p p-first-last">C-Imm Sim-10.1 accomplished the immunological simulation to evaluate the immune reaction of the predicted vaccine construct [<a href="#bib54" rid="bib54" class=" bibr popnode">52</a>]. The c-Imm-Sim server was designed to simulate three major functional mammalian system components: bone marrow, lymph node, and thymus. HLA (DRB1 0101, DRB1 0101, B0702, B0702, A0101, A0101), random seed (12,345), volume (10), number of injections set to 1, and number of steps are the immune simulation input parameters (100). Other parameters have been set to be the default.</p></div><div id="sec2.13" class="sec"><h3 id="sec2.13title">2.13. Dry lab cloning and Codons optimization</h3><p id="p0085" class="p p-first-last">Codon usage varies by species, and non-adapted codons might lead to low expression levels in the receptor. In conclusion, it should be created with the host's translational machinery to boost gene expression. In the present study, the Java Codon Adaptation Tool (JCAT) was employed for the optimization and reverse translation of the MEV construct [<a href="#bib55" rid="bib55" class=" bibr popnode">53</a>]. Prokaryotic ribosome binding sites, Rho-independent transcription termination, and restriction enzyme cleavage sites were selected. Lastly, the vaccine construct was cloned within the pET30a (+) vector with the help of SnapGene software. <a href="/pmc/articles/PMC11153098/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a> represents the complete methodology used in the current study for designing candidate MEV against <em>P. shigelloides</em>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC11153098/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140536272618576"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11153098_gr1.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 1" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11153098/bin/gr1.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140536272618576"><a target="object" rel="noopener" href="/pmc/articles/PMC11153098/figure/fig1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC11153098/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Fig. 1</a></div><!--caption a7--><div class="caption"><p>Graphical synopsis representing the complete methodology used for prediction of multiepitope based vaccine against <em>Plesiomonas shigelloides.</em></p></div></div></div></div><div id="sec2.14" class="sec sec-last"><h3 id="sec2.14title">2.14. Protein Protein molecular Dynamic simulation of MCH I, MHC II and TLR4</h3><p id="p0090" class="p p-first-last">Schrodinger, a powerful molecular dynamics simulation software, facilitates the exploration of protein-protein interactions at 100ns crucial for immune responses. In the simulation of Major Histocompatibility Complex (MHC) class I and class II proteins, Schrodinger offers insights into their peptide binding specificities and antigen presentation mechanisms. MHC class I proteins exhibit a groove for peptide binding, while MHC class II proteins possess an open-ended structure, both pivotal for T cell recognition. Additionally, Schrodinger the studied of Toll-like Receptor 4 (TLR4) interactions with its ligands, elucidating the molecular basis of innate immune signaling. Through Desmode simulations, researchers can unravel the dynamic interplay between these proteins, providing essential knowledge for drug discovery and vaccine development targeting immune-related diseases.</p></div></div><div id="sec3" class="tsec sec"><h2 class="head no_bottom_margin" id="sec3title">3. Results</h2><div id="sec3.1" class="sec sec-first"><h3 id="sec3.1title">3.1. Analysis of proteome</h3><p id="p0095" class="p p-first-last">The pathogenic strain of <em>Plesiomonas shigelloides</em> was downloaded from the Uni-Prot database and later evaluated using a subtractive genomics approach to design an effective vaccine candidate for treating <em>P. shigelloides-</em>related infections. The particular pathogenic strain <em>P. shigelloides</em> comprised 3354 proteins. These 3354 proteins were filtered out in a subtractive pipeline to identify target proteins. Out of 3354 proteins, 396 important proteins were filtered out by the Geptop server. Later, BlastP was used for the identification of non-homologous proteins. BlastP uncovered 42 nonhomologous proteins from a total of 396 essential proteins. These 42 essential non-homologous proteins were screened for subcellular localization and antigenicity prediction. A total of 6 proteins were found to be membrane proteins and highly antigenic (<a href="/pmc/articles/PMC11153098/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>). These six target proteins could potentially serve as targets of vaccine-induced immune responses.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p>Shows the top proteins with maximum antigenicity nature and extracellular locations.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1">Protein name</th><th rowspan="1" colspan="1">Accession No</th><th rowspan="1" colspan="1">Antigenicity</th><th rowspan="1" colspan="1">Helices</th><th rowspan="1" colspan="1">Location</th></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1"><strong>1</strong></td><td align="left" rowspan="1" colspan="1">Autotransporter domain-containing protein</td><td align="left" rowspan="1" colspan="1">&#x0003e;tr|R8ARB8</td><td align="left" rowspan="1" colspan="1">0.6154</td><td align="left" rowspan="1" colspan="1">0</td><td align="left" rowspan="1" colspan="1">Extracellular</td></tr><tr><td align="left" rowspan="1" colspan="1"><strong>2</strong></td><td align="left" rowspan="1" colspan="1">Prophage tail fiber protein</td><td align="left" rowspan="1" colspan="1">&#x0003e;tr|R8APA5</td><td align="left" rowspan="1" colspan="1">0.7706</td><td align="left" rowspan="1" colspan="1">0</td><td align="left" rowspan="1" colspan="1">Extracellular</td></tr><tr><td align="left" rowspan="1" colspan="1"><strong>3</strong></td><td align="left" rowspan="1" colspan="1">Tail fiber protein</td><td align="left" rowspan="1" colspan="1">&#x0003e;tr|R8AQH0</td><td align="left" rowspan="1" colspan="1">0.8350</td><td align="left" rowspan="1" colspan="1">0</td><td align="left" rowspan="1" colspan="1">Extracellular</td></tr><tr><td align="left" rowspan="1" colspan="1"><strong>4</strong></td><td align="left" rowspan="1" colspan="1">Flagellin</td><td align="left" rowspan="1" colspan="1">&#x0003e;tr|R8AS43</td><td align="left" rowspan="1" colspan="1">0.6989</td><td align="left" rowspan="1" colspan="1">0</td><td align="left" rowspan="1" colspan="1">Extracellular</td></tr><tr><td align="left" rowspan="1" colspan="1"><strong>5</strong></td><td align="left" rowspan="1" colspan="1">Uncharacterized protein</td><td align="left" rowspan="1" colspan="1">&#x0003e;tr|R8APD9</td><td align="left" rowspan="1" colspan="1">0.8919</td><td align="left" rowspan="1" colspan="1">0</td><td align="left" rowspan="1" colspan="1">Extracellular</td></tr><tr><td align="left" rowspan="1" colspan="1"><strong>6</strong></td><td align="left" rowspan="1" colspan="1">Superoxide dismutase</td><td align="left" rowspan="1" colspan="1">&#x0003e;tr|R8APR1</td><td align="left" rowspan="1" colspan="1">0.5574</td><td align="left" rowspan="1" colspan="1">0</td><td align="left" rowspan="1" colspan="1">Extracellular</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140536272606400"><a target="object" rel="noopener" href="/pmc/articles/PMC11153098/table/tbl1/?report=objectonly">Open in a separate window</a></div></div></div><div id="sec3.2" class="sec"><h3 id="sec3.2title">3.2. Selection of putative epitopes</h3><p id="p0100" class="p p-first-last">Overall, 92 CTL epitopes (12-mer) were predicted from 6 target proteins, but only three epitopes were observed to be non-allergenic, non-toxic, antigenic, and immunogenic (<a href="/pmc/articles/PMC11153098/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>). Likewise, the MHC-II binding tool identified ten inimitable HTL (15-mer) epitopes. From these 10 HTL epitopes, only one HTL epitope was discovered to be highly antigenic, IFN-positive, non-allergenic, and IL-4 &#x00026; IL-10 inducers. (<a href="/pmc/articles/PMC11153098/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Table 3</span></a>). Similarly, 50 LBL epitopes were identified. After analysis, two epitopes were selected from vaccine designing because these epitopes met the selection criteria of immunogenicity, allergenicity, and toxicity.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="tbl2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p>Epitopes were chosen by CTL for vaccine development against <em>P. Shigelloides.</em></p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Epitope</th><th rowspan="1" colspan="1">Protein</th><th rowspan="1" colspan="1">Alleles</th><th rowspan="1" colspan="1">Position</th><th rowspan="1" colspan="1">Antigenicity</th><th rowspan="1" colspan="1">Immunogenicity</th></tr></thead><tbody><tr><td rowspan="3" align="left" colspan="1">NRQVRGGYTFQP</td><td rowspan="3" align="left" colspan="1">Autotransporter domain-containing protein</td><td align="left" rowspan="1" colspan="1">HLA-B*15:02</td><td rowspan="3" align="left" colspan="1">501&#x02013;512</td><td rowspan="3" align="left" colspan="1">1.4726</td><td rowspan="3" align="left" colspan="1">0.1844</td></tr><tr><td align="left" rowspan="1" colspan="1">HLA-A*24:02</td></tr><tr><td align="left" rowspan="1" colspan="1">HLA-A*23:01</td></tr><tr><td rowspan="2" align="left" colspan="1">GSALGATYRIDY</td><td rowspan="2" align="left" colspan="1">Uncharacterized protein</td><td align="left" rowspan="1" colspan="1">HLA-A*30:02</td><td rowspan="2" align="left" colspan="1">133&#x02013;144</td><td rowspan="2" align="left" colspan="1">1.2093</td><td rowspan="2" align="left" colspan="1">0.28084</td></tr><tr><td align="left" rowspan="1" colspan="1">HLA-A*01:01</td></tr><tr><td rowspan="6" align="left" colspan="1">YIDYRNVRPNYL</td><td rowspan="6" align="left" colspan="1">Superoxide dismutase</td><td align="left" rowspan="1" colspan="1">HLA-C*06:02</td><td rowspan="6" align="left" colspan="1">164&#x02013;175</td><td rowspan="6" align="left" colspan="1">1.5493</td><td rowspan="6" align="left" colspan="1">0.12012</td></tr><tr><td align="left" rowspan="1" colspan="1">HLA-C*07:01</td></tr><tr><td align="left" rowspan="1" colspan="1">HLA-C*08:02</td></tr><tr><td align="left" rowspan="1" colspan="1">HLA-C*07:02</td></tr><tr><td align="left" rowspan="1" colspan="1">HLA-A*01:01</td></tr><tr><td align="left" rowspan="1" colspan="1">HLA-C*05:01</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140536274462832"><a target="object" rel="noopener" href="/pmc/articles/PMC11153098/table/tbl2/?report=objectonly">Open in a separate window</a></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="tbl3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p>Shows the final HTL epitopes used in the development of a vaccination against <em>P. Shigellodies</em></p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Epitope</th><th rowspan="1" colspan="1">Protein</th><th rowspan="1" colspan="1">Alleles</th><th rowspan="1" colspan="1">Position</th><th rowspan="1" colspan="1">Antigenicity</th><th rowspan="1" colspan="1">IFY-N</th><th rowspan="1" colspan="1">IL-4</th><th rowspan="1" colspan="1">IL-10</th></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1">QPNSRYQVTPFVALNY</td><td align="left" rowspan="1" colspan="1">Autotransporter domain-containing protein</td><td align="left" rowspan="1" colspan="1">HLA-DRB1*07:03</td><td align="left" rowspan="1" colspan="1">511&#x02013;526</td><td align="left" rowspan="1" colspan="1"><strong>0.7055</strong></td><td align="left" rowspan="1" colspan="1">Positive</td><td align="left" rowspan="1" colspan="1">Inducer</td><td align="left" rowspan="1" colspan="1">non-inducer</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140536274698224"><a target="object" rel="noopener" href="/pmc/articles/PMC11153098/table/tbl3/?report=objectonly">Open in a separate window</a></div></div></div><div id="sec3.3" class="sec"><h3 id="sec3.3title">3.3. Construction of vaccine</h3><p id="p0105" class="p p-first-last">The following were the primary factors considered when designing the MEV construct: MEV must be nontoxic, non-allergic, antigenic, and immunogenic (<a href="/pmc/articles/PMC11153098/table/tbl4/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl4" rid-ob="ob-tbl4" co-legend-rid=""><span>Table 4</span></a>). A vaccine construct was developed using the factors mentioned above by integrating 3 CTL, 1 HTL, and 2&#x000a0;B-cell epitopes with the respective adjuvants and linkers (<a href="/pmc/articles/PMC11153098/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>A). Cholera toxin B, an adjuvant, was coupled with the N terminus of MEV using the EAAAK linker (<a href="/pmc/articles/PMC11153098/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>B). There are three linkers were used for the construction of the vaccine, which are represented by different colors in the figures: AAY Linker, described as Grey; CPGPG Linker, represented Yellow; KK Linker represented Brown (<a href="/pmc/articles/PMC11153098/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>B). Subsequently, a linear vaccine construct was developed comprised of 356 amino acids (Supplementary File 1).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="tbl4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p>Final selected B-Cell epitopes for vaccine designing against <em>P. Shigelloides</em>.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Epitope</th><th rowspan="1" colspan="1">Protein</th><th rowspan="1" colspan="1">Score</th><th rowspan="1" colspan="1">Position</th><th rowspan="1" colspan="1">Antigenicity</th><th rowspan="1" colspan="1">Immunogenicity</th></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1">GFGAGEVNGGISYRIN</td><td align="left" rowspan="1" colspan="1">Autotransporter domain-containing protein</td><td align="left" rowspan="1" colspan="1">0.65</td><td align="left" rowspan="1" colspan="1">719</td><td align="left" rowspan="1" colspan="1">1.6757</td><td align="left" rowspan="1" colspan="1">0.36723</td></tr><tr><td align="left" rowspan="1" colspan="1">KGGTVGGVDYGVGDTL</td><td align="left" rowspan="1" colspan="1">Prophage tail fiber protein</td><td align="left" rowspan="1" colspan="1">0.89</td><td align="left" rowspan="1" colspan="1">433</td><td align="left" rowspan="1" colspan="1">1.6406</td><td align="left" rowspan="1" colspan="1">0.36096</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140536271158736"><a target="object" rel="noopener" href="/pmc/articles/PMC11153098/table/tbl4/?report=objectonly">Open in a separate window</a></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC11153098/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140536270767520"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11153098_gr2.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 2" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11153098/bin/gr2.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140536270767520"><a target="object" rel="noopener" href="/pmc/articles/PMC11153098/figure/fig2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC11153098/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Fig. 2</a></div><!--caption a7--><div class="caption"><p>Details of vaccine construct (A) Sequence of multi-epitope subunit vaccine construct (B) graphical representation of vaccine construction. AAY Linker represented Grey, CPGPG linker represented Yellow, and KK linker represented Brown in construction of vaccine. (C) Secondary structure of vaccine construct; (D) PROCHECK results for Ramachandran plot to validate the integrity of constructed vaccine (E) Tertiary structure of vaccine construct. (For interpretation of the references to color in this figure legend, the reader is referred to the Web version of this article.)</p></div></div></div></div><div id="sec3.4" class="sec"><h3 id="sec3.4title">3.4. Assessment of immunogenicity and physiochemical properties</h3><p id="p0110" class="p p-first-last">The physiochemical and immunogenic characteristics of the MEV construct were then analyzed. Firstly, the homology of the MEV construct was determined by subjecting MEV to BlastP for non-homology prediction. The vaccine construct must be non-homologous otherwise the vaccine might yield negative when injected into the human body. The antigenic, allergic, and toxic potential of the MEV vaccine was again appraised. Data revealed that the MEV construct is nontoxic, antigenic, and non-allergenic. Physiochemical profiling was done through Prot-Param. The molecular weight of the vaccine was 30.93&#x000a0;KDa, and the Grand Average Hydropathicity (GRAVY) was &#x02212;0.216, whereas the theoretical PI was 5.23. Moreover, the MEV construct vaccine has a half-life in vitro of 30&#x000a0;h and in vivo of &#x0003e;20&#x000a0;h. These characteristics showed that the designed MEV construct might be a promising vaccine candidate against <em>P. shigelloides</em>.</p></div><div id="sec3.5" class="sec"><h3 id="sec3.5title">3.5. Structural evaluation</h3><p id="p0115" class="p p-first-last">The structure was predicted to analyze the overall quality of the structure and then to dock with the targeted antigen. SPOMA and PSIPRED servers were used to gain insight into the secondary structure of the designed vaccine. In the MEV construct, the alpha-helix comprises 123 residues, accounting for 34.55&#x000a0;% of the sequence, 102 residues in extended chains, accounting for 28.65 %, and 103 amino acids in coils, accounting for 28.93 % of the total vaccine structure (<a href="/pmc/articles/PMC11153098/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>C).</p></div><div id="sec3.6" class="sec"><h3 id="sec3.6title">3.6. Estimation of tertiary structure, validation, and refinement</h3><p id="p0120" class="p p-first-last">By using I-TASSER, a tertiary structure was created (<a href="/pmc/articles/PMC11153098/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>E). I-TASSER yielded a list of models, but the top mode with a C-score of &#x02212;1.66 was selected for further analyses. The predicted 3D structure of the MEV construct was refined with the Galaxy Refine server. Further, the Ramachandran plot uncovered that 85.5 % of amino acids were in preferred regions, 31.3 % in tolerated regions, and 3.1 % in outlier regions (<a href="/pmc/articles/PMC11153098/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>D). The Z score was found to be &#x02212;1.01. Moreover, the quality score of the predicted structure was 64.8968. These findings suggest that the MEV construct met all the criteria for selection.</p></div><div id="sec3.7" class="sec"><h3 id="sec3.7title">3.7. Choosing B-cell epitopes</h3><p id="p0125" class="p p-first-last">In addition to releasing cytokines, B-lymphocytes focus on providing humoral immunity by producing antibodies. B-lymphocytes generate antibodies, which result in humoral immunity [<a href="#bib56" rid="bib56" class=" bibr popnode">54</a>]. As an outcome, optimal domains of B-cell epitopes should be present in the MEV construct. Regarding this, ABCPred 2.0 yielded three conformational-discontinuous and 24 linear-continuous.</p></div><div id="sec3.8" class="sec"><h3 id="sec3.8title">3.8. Disulfide engineering</h3><p id="p0130" class="p p-first-last">The MEV was upgraded by increasing its stability through Disulfide by Design v-2.0. For the disulfide engineering, 33 residues were used overall. Three pairs of residues were selected for disulfide synthesis because their energy and Chi3 scores were in the normal range. As a result, genetic variations in the residue pair emerged, involving PRO222-PRO251 with &#x02212;80.06 and 1.67&#x000a0;kcal/mol of energy, and PRO222-PRO251 with &#x02212;80.06 and 1.67&#x000a0;kcal/mol of energy, respectively.</p></div><div id="sec3.9" class="sec"><h3 id="sec3.9title">3.9. Molecular docking</h3><p id="p0135" class="p p-first-last">A strong interaction between MEV constructs and immune-specific receptors is required for an active immune response. The docking of the MEV construct with TLR4 (<a href="/pmc/articles/PMC11153098/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Fig. 3</span></a>C), MHC-1(<a href="/pmc/articles/PMC11153098/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Fig. 3</span></a>B), and MHC-II (<a href="/pmc/articles/PMC11153098/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Fig. 3</span></a>A) was done using HADDOCK v. 2.4. (Immune receptor of human). TLR4 has gained much importance due to its ability to identify distinct molecular patterns from invading pathogens. Molecular docking analysis revealed that MEV construct and TLR4 substantially interact with each other. TLR4 and vaccination had a binding score of 55.2&#x000a0;kcal/mol, which was computed. It is also noteworthy that TLR4 and MEV have had ten hydrogen bond interactions within a 3.34 range. In addition, HADDOCK V.2.4 was used to dock the MEV with the MHC1 and MHC II receptors. In the docked complex of MHC-II with MEV, 11 hydrogen bond interactions were found. <a href="/pmc/articles/PMC11153098/table/tbl5/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl5" rid-ob="ob-tbl5" co-legend-rid=""><span>Table 5</span></a> enlist the docking results produced by the HADDOCK V.2.4 server.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig3" co-legend-rid="lgnd_fig3"><a href="/pmc/articles/PMC11153098/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140536276460560"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11153098_gr3.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 3" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11153098/bin/gr3.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140536276460560"><a target="object" rel="noopener" href="/pmc/articles/PMC11153098/figure/fig3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig3"><div><a class="figpopup" href="/pmc/articles/PMC11153098/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3">Fig. 3</a></div><!--caption a7--><div class="caption"><p>Docked complexes of MESV with (A) MHCI, (B) MHCII and with human receptor TLR4.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="tbl5"><h3>Table 5</h3><!--caption a7--><div class="caption"><p>Binding affinities of vaccine construct with MHC molecules and human receptors.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Parameters</th><th rowspan="1" colspan="1">TLR4</th><th rowspan="1" colspan="1"><strong>MHCI</strong></th><th rowspan="1" colspan="1"><strong>MHCII</strong></th></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1">HADDOCK-v.2.2 score</td><td align="left" rowspan="1" colspan="1">&#x02212;23.9&#x000b1; 10.6</td><td align="left" rowspan="1" colspan="1">&#x02212;41.6&#x000a0;&#x000b1;&#x000a0;25.9</td><td align="left" rowspan="1" colspan="1">&#x02212;86.9 &#x000b1; 6.7</td></tr><tr><td align="left" rowspan="1" colspan="1">Cluster Size</td><td align="left" rowspan="1" colspan="1">38</td><td align="left" rowspan="1" colspan="1">4</td><td align="left" rowspan="1" colspan="1">30</td></tr><tr><td align="left" rowspan="1" colspan="1">RMSD from the overall lowest energy structure</td><td align="left" rowspan="1" colspan="1">4.9&#x000a0;&#x000b1;&#x000a0;0.5</td><td align="left" rowspan="1" colspan="1">3.3&#x000a0;&#x000b1;&#x000a0;0.3</td><td align="left" rowspan="1" colspan="1">9.1 &#x000b1; 0.3</td></tr><tr><td align="left" rowspan="1" colspan="1">Van-der-Waals energy</td><td align="left" rowspan="1" colspan="1">&#x02212;84.6&#x000a0;&#x000b1;&#x000a0;4.0</td><td align="left" rowspan="1" colspan="1">&#x02212;114.0&#x000a0;&#x000b1;&#x000a0;7.1</td><td align="left" rowspan="1" colspan="1">&#x02212;118 &#x000b1; 3.7</td></tr><tr><td align="left" rowspan="1" colspan="1">Electrostatic Energy</td><td align="left" rowspan="1" colspan="1">&#x02212;305.9&#x000b1; 49.3</td><td align="left" rowspan="1" colspan="1">&#x02212;467.0&#x000a0;&#x000b1;&#x000a0;74.2</td><td align="left" rowspan="1" colspan="1">&#x02212;325.5 &#x000b1; 32.8</td></tr><tr><td align="left" rowspan="1" colspan="1">Desolvaion Energy</td><td align="left" rowspan="1" colspan="1">&#x02212;26.9&#x000a0;&#x000b1;&#x000a0;17.8</td><td align="left" rowspan="1" colspan="1">&#x02212;29.7&#x000a0;&#x000b1;&#x000a0;11.7</td><td align="left" rowspan="1" colspan="1">&#x02212;73.0&#x000b1; 14.6</td></tr><tr><td align="left" rowspan="1" colspan="1">Restraints Violation Energy</td><td align="left" rowspan="1" colspan="1">1488.4&#x000a0;&#x000b1;&#x000a0;173.07</td><td align="left" rowspan="1" colspan="1">1955.5&#x000a0;&#x000b1;&#x000a0;200.96</td><td align="left" rowspan="1" colspan="1">1697.7&#x000a0;&#x000b1;&#x000a0;217.49</td></tr><tr><td align="left" rowspan="1" colspan="1">Buried Surface Area</td><td align="left" rowspan="1" colspan="1">2620.7&#x000a0;&#x000b1;&#x000a0;57.4</td><td align="left" rowspan="1" colspan="1">4125.1&#x000a0;&#x000b1;&#x000a0;121.1</td><td align="left" rowspan="1" colspan="1">3807.7&#x000a0;&#x000b1;&#x000a0;77.8</td></tr><tr><td align="left" rowspan="1" colspan="1">Z-Score</td><td align="left" rowspan="1" colspan="1">&#x02212;0.9</td><td align="left" rowspan="1" colspan="1">0.5</td><td align="left" rowspan="1" colspan="1">&#x02212;1.5</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140536275722560"><a target="object" rel="noopener" href="/pmc/articles/PMC11153098/table/tbl5/?report=objectonly">Open in a separate window</a></div></div></div><div id="sec3.10" class="sec"><h3 id="sec3.10title">3.10. Molecular dynamic simulation</h3><p id="p0140" class="p p-first-last">A large-scale normal mode analysis was used to study protein mobility and stability. iMODS-server was employed to analyze the interaction between the MEV construct and its receptor because it is quick and efficient (<a href="/pmc/articles/PMC11153098/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>A and B). The motion stiffness of docked complexes is 2.788520e-05, represented by the estimated eigenvalue (<a href="/pmc/articles/PMC11153098/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>E). Altogether, an inverse relationship was discovered between each normal mode's eigenvalue and variance. Several sets of connected, dis-associated, or irrelevant motions are represented in different colors such as reddish, blue, and white in a covariance matrix, reflecting the residue pairs' pairings. The elastic map showed join-atoms connected by springs, with each point indicating one spring and the grey color denoting stiffer strings, with intensity proportional to stiffness(<a href="/pmc/articles/PMC11153098/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>C and D).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig4" co-legend-rid="lgnd_fig4"><a href="/pmc/articles/PMC11153098/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140536280675264"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11153098_gr4.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 4" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11153098/bin/gr4.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140536280675264"><a target="object" rel="noopener" href="/pmc/articles/PMC11153098/figure/fig4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig4"><div><a class="figpopup" href="/pmc/articles/PMC11153098/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4">Fig. 4</a></div><!--caption a7--><div class="caption"><p>Molecular dynamic simulation of the vaccine&#x02013;TLR4 complex, showing: (A) eigenvalue; (B) deformability; (c) B-factor; (d) Residue index.</p></div></div></div></div><div id="sec3.11" class="sec"><h3 id="sec3.11title">3.11. Immune-simulation</h3><p id="p0145" class="p p-first-last">Immune responses including both primary and secondary, actively engage in the infection and may remain unfluctuating in the real immune response. An <em>in silico</em> immune response was constructed using the C-IMMSIM immune server to investigate the immunogenic profile of an MEV. <a href="/pmc/articles/PMC11153098/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>A depicts the host immune responses to the antigen as represented by an <em>in silico</em> approach. It is also worth noting that a substantial increase in immunoglobin activity (IgG1+IgG2, IgM, IgG&#x000a0;+&#x000a0;IgM) led to a reduction in antigenic concentration. Besides that, numerous B cell isotypes were obtained, indicating the possibility of isotypes switching in conjunction with memory formation. Furthermore, the high levels of cytokines and interleukins produced during the simulation facilitated the emergence of a good immune response <a href="/pmc/articles/PMC11153098/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>B.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig5" co-legend-rid="lgnd_fig5"><a href="/pmc/articles/PMC11153098/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140536280666048"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11153098_gr5.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 5" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11153098/bin/gr5.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140536280666048"><a target="object" rel="noopener" href="/pmc/articles/PMC11153098/figure/fig5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig5"><div><a class="figpopup" href="/pmc/articles/PMC11153098/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5">Fig. 5</a></div><!--caption a7--><div class="caption"><p>Response of the immune system to vaccine: (A) Post antigen exposure production of immunoglobulin and B-cell isotypes in various states with the Simpson index (B) development of cytokine interleukins.</p></div></div></div></div><div id="sec3.12" class="sec"><h3 id="sec3.12title">3.12. In-silico cloning</h3><p id="p0150" class="p p-first-last">The MEV construct was cloned <em>in silico</em> to check their expression level in <em>E. coli</em>. As a result of their use in the <em>E. coli</em> expression system, codon optimization of MEV constructs has become extremely important. J-Cat tool was employed for codon optimization in this scenario. A codon adaptation index (CAI) value of &#x0003e;0.8 and a GC concentration of 30&#x02013;70&#x000a0;% are considered satisfactory. Regarding this, our results revealed a CAI value of 0.9 and a GC content of 48.87&#x000a0;%. In enhanced DNA, the CAI score was 0.9, and the GC content was 48.87&#x000a0;%. Finally, <em>in silico</em> cloning was carried out by inserting the cDNA sequence of MEV into the pET-28a (+) vector using the Snap-Gene software (<a href="/pmc/articles/PMC11153098/figure/fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig6" rid-ob="ob-fig6" co-legend-rid="lgnd_fig6"><span>Fig. 6</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig6" co-legend-rid="lgnd_fig6"><a href="/pmc/articles/PMC11153098/figure/fig6/" target="figure" rid-figpopup="fig6" rid-ob="ob-fig6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140536280655872"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11153098_gr6.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 6" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11153098/bin/gr6.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140536280655872"><a target="object" rel="noopener" href="/pmc/articles/PMC11153098/figure/fig6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig6"><div><a class="figpopup" href="/pmc/articles/PMC11153098/figure/fig6/" target="figure" rid-figpopup="fig6" rid-ob="ob-fig6">Fig. 6</a></div><!--caption a7--><div class="caption"><p>Codon-optimized vaccine computationally cloned using the expression system of <em>E. coli</em> K12&#x02019;. Black and red colors represent the plasmid backbone and inserted DNA, respectively. (For interpretation of the references to color in this figure legend, the reader is referred to the Web version of this article.)</p></div></div></div></div><div id="sec3.13" class="sec sec-last"><h3 id="sec3.13title">3.13. Protein molecular Dynamic simulation of MCH I, MHC II and TLR4 by Schrodinger</h3><p id="p0155" class="p p-first">The protein-protein intraction with the vaccine showed stability and fluctuations at different stages which are all shown in the (<a href="/pmc/articles/PMC11153098/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span>Fig. 7</span></a>) and at 100ns.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig7" co-legend-rid="lgnd_fig7"><a href="/pmc/articles/PMC11153098/figure/fig7/" target="figure" rid-figpopup="fig7" rid-ob="ob-fig7"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140536274647296"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11153098_gr7.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 7" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11153098/bin/gr7.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140536274647296"><a target="object" rel="noopener" href="/pmc/articles/PMC11153098/figure/fig7/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig7"><div><a class="figpopup" href="/pmc/articles/PMC11153098/figure/fig7/" target="figure" rid-figpopup="fig7" rid-ob="ob-fig7">Fig. 7</a></div><!--caption a7--><div class="caption"><p>MHC-I Vaccine A (RMSD), B(RMSF), C(SSE), D(SSE) TLR4-Vaccine E(RMSD), F(RMSF), G(SSE), H(SSE) and MHC-II Vaccine I(RMSD),J(RMSF),K(SSE), L(SSE) are all vaccine showing their stability and fluctuation at different stages with protein-protein interactions.</p></div></div></div><p id="p0160">MCHI- Protein-Ligand RMSD (The Root Mean Square Deviation (RMSD) (<a href="/pmc/articles/PMC11153098/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span>Fig. 7</span></a>A) is a key metric in molecular dynamics simulations, assessing how much a protein's structure varies from a reference frame. It's calculated by aligning protein frames to the reference and measuring atom displacements. Monitoring RMSD over the simulation provides insights into structural stability and equilibration. Stable RMSD values indicate equilibrium, while continued increase or decrease suggests inadequate simulation duration. In essence, RMSD analysis helps ensure the accuracy and reliability of molecular dynamics simulations by assessing structural fluctuations and equilibration), MCH I- Protein RMSF(The Root Mean Square Fluctuation (RMSF) (<a href="/pmc/articles/PMC11153098/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span>Fig. 7</span></a>B) serves to analyze localized variations along the protein sequence. Peaks in the plot highlight regions of the protein exhibiting the highest fluctuations throughout the simulation. It's common to observe greater fluctuations at the tails (N- and C-terminal) compared to other parts of the protein. Generally, secondary structure elements such as alpha helices and beta strands display less fluctuation than the unstructured regions, such as loop regions, owing to their inherent rigidity) and Protein Secondary Structure:</p><p id="p0165">(Throughout the simulation, the protein's secondary structure elements (SSE), including alpha-helices and beta-strands, are tracked. <a href="/pmc/articles/PMC11153098/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span>Fig. 7</span></a>C plot in the above figure illustrates the distribution of SSE by residue index across the protein structure, while <a href="/pmc/articles/PMC11153098/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span>Fig. 7</span></a>D plot provides a summary of the SSE composition for each trajectory frame during the simulation.)</p><p id="p0170">TLR4-Protein-Ligand RMSD (<a href="/pmc/articles/PMC11153098/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span>Fig. 7</span></a>E) is a metric used to measure the average displacement of selected atoms in protein structures throughout a simulation compared to a reference frame. It helps assess structural changes and whether the simulation has equilibrated. Convergence is essential, with stable RMSD values indicating equilibrium. If RMSD keeps increasing or decreasing towards the end, the simulation may be insufficient.), TLR4-Protein RMSF (<a href="/pmc/articles/PMC11153098/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span>Fig. 7</span></a>F) is a tool for assessing local fluctuations in a protein chain. Peaks indicate areas of high fluctuation, often found at the protein's ends. Structured elements like alpha helices are typically less fluctuant than unstructured regions like loops.), and Protein Secondary Structure (During the simulation, protein SSE (Figure G), like alpha-helices and beta-strands, are tracked. The above plot in the figure illustrates their distribution by residue index, while the below plot in (<a href="/pmc/articles/PMC11153098/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span>Fig. 7</span></a>H) summarizes SSE composition for each trajectory frame.</p><p id="p0175" class="p p-last">MHC II - Protein-Ligand RMSD (<a href="/pmc/articles/PMC11153098/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span>Fig. 7</span></a>I) gauges the average displacement of chosen atoms in protein structures during a simulation relative to a reference frame, aiding in evaluating structural alterations and equilibration. Stable RMSD values denote equilibrium, while continuous increase or decrease suggests insufficient simulation duration.), MHC II <em>-</em>Protein RMSF (<a href="/pmc/articles/PMC11153098/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span>Fig. 7</span></a>J) evaluates localized protein sequence variations, with plot peaks indicating regions of significant fluctuation. Greater fluctuations typically occur at the N- and C-terminal tails compared to other regions, while secondary structure elements like alpha helices and beta strands display less fluctuation than unstructured regions such as loops.), and Protein Secondary Structure (Protein SSE (<a href="/pmc/articles/PMC11153098/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span>Fig. 7</span></a>K), including alpha-helices and beta-strands, are monitored throughout the simulation. The upper plot displays SSE distribution by residue index, while the lower plot summarizes SSE composition for each trajectory frame. Lastly, the bottom plot tracks residue SSE assignment over time (<a href="/pmc/articles/PMC11153098/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span>Fig. 7</span></a>L).</p></div></div><div id="sec4" class="tsec sec"><h2 class="head no_bottom_margin" id="sec4title">4. Discussion</h2><p id="p0180" class="p p-first"><em>Plesiomonas shigelloides</em>, a ubiquitous freshwater inhabitant, has been associated with several waterborne outbreaks of acute diarrhea and, rarely, serious extra-intestinal disease [<a href="#bib57" rid="bib57" class=" bibr popnode">[55]</a>, <a href="#bib58" rid="bib58" class=" bibr popnode">[56]</a>, <a href="#bib59" rid="bib59" class=" bibr popnode">[57]</a>]. Designing an effective live and attenuated vaccination is both costly and time-consuming [<a href="#bib68" rid="bib68" class=" bibr popnode">58</a>]. Aside from that, the use of typical attenuated vaccines is restricted due to several difficulties, including their poor ability to elicit immune responses and a variety of side effects. In a previous research study, the immune response was found to vary with the administration of different booster doses. Overall, when the vaccine candidate was given in varying doses, there was an increase in antibody concentration and the populations of B-cells, plasma B-lymphocytes, and Th-cells [<a href="#bib73" rid="bib73" class=" bibr popnode">59</a>].</p><p id="p0185">Various methods for designing effective epitope-based vaccines are already available [<a href="#bib69" rid="bib69" class=" bibr popnode">60</a>]. The genomics revolution has significantly aided in developing novel therapeutic and preventive vaccine candidates using next-generation sequencing and advanced bioinformatics. This approach, known as reverse vaccinology (RV), allows for identifying potential surface-associated antigens without the need for culturing microorganisms. The RV method has been successfully employed in creating vaccines such as the meningococcal serogroup B (4CMenB) vaccine, and it has been utilized for various bacterial and viral pathogens [<a href="#bib72" rid="bib72" class=" bibr popnode">61</a>].</p><p id="p0190">A subtractive proteomics method was used to identify essential, antigenic, non-homologous, and non-toxic proteins suitable for developing vaccine candidates against <em>P. shigelloides</em> in the current study. Another crucial factor in the selection of target proteins is whether they contain transmembrane helices or not. Six proteins were selected as vaccine targets, namely Flagellin, Uncharacterized protein, Tail fiber protein, Prophage tail fiber protein, and Superoxide dismutase. Following the study, the antigenicity of all these proteins was assessed using the VaxiJen server. Additionally, their allergenicity was evaluated through allertop. All B and T cell epitopes must combine to generate a powerful and long-lasting immune response to develop both humoral and cell-mediated immunity. Therefore, predictions and extensive examinations of T and B cell epitopes were conducted. Moreover, the population coverage analysis revealed that the selected epitopes accounted for 94.87 percent of the global population.</p><p id="p0195">To design a putative vaccine construct, HTL, CTL, and B cell epitopes were linked with GPGPG, AAY, and KK linkers, respectively. Cholera enterotoxin subunit B (CTB) has been widely utilized as a traditional mucosal adjuvant in numerous studies due to its potential to enhance the immunogenicity of the MEV. Adjuvants play a crucial role in certain vaccines by boosting the immune response in recipients, thus improving vaccine efficacy. Therefore, the final MEV construct was evaluated based on its physiochemical properties. It was essential for the vaccine construct to exhibit non-homology with human proteins, as well as being antigenic, non-allergenic, and non-toxic. The number of transmembrane helices was also considered when designing a prospective vaccine construct because purifying proteins lacking TMHs is difficult. As a result, these proteins must be removed from the vaccine construct. Codon optimization of the proposed construct resulted in a distinct cDNA sequence, ensuring efficient production in the <em>E. coli</em> host. Moreover, analysis using the Jcat tool revealed that the MEV had a GC content of 48.87&#x000a0;%, suggesting the potential for successful expression in <em>E. coli</em>.</p><p id="p0200">MEV must have had a strong binding affinity with Toll-like receptors to be successfully supplied into the body. More importantly, to provoke the immune system and to design a suitable vaccine against pathogenic bacteria, the MEV construct must have a strong binding affinity with MHC molecules. Regarding this, docking analysis has illustrated that the proposed Multiepitope Vaccine (MEV) construct forms strong interactions with both Toll-like receptor 4 (TLR4) and Major Histocompatibility Complex (MHC) receptors. Furthermore, molecular simulation studies have corroborated these findings, providing validation for the docking results. These simulations have demonstrated that the MEV binds efficiently and effectively to immune receptors, further supporting the efficacy of the vaccine construct. When our findings are compared to previous studies, it is worth noting that an epitope-based malaria vaccine has reached phase-III trials and may become the first commercial vaccine against a human parasitic disease [<a href="#bib70" rid="bib70" class=" bibr popnode">62</a>]. Phase II and III clinical trials for some of these epitope-based vaccines are anticipated to begin in the upcoming years [<a href="#bib71" rid="bib71" class=" bibr popnode">63</a>]. Thus, all these arguments supported our findings and gave us reason to believe that, after clinical testing, the MEV construct developed in this study would be effective against <em>P. shigelloides</em>. For clinicians seeking to treat patients with <em>P. shigelloides</em> infections, the epitopes predicted in this study represent a beacon of hope.</p><p id="p0205" class="p p-last">Despite utilizing stringent criteria for epitope selection, further research on the MEV construct could enhance our findings. Consequently, the MEV has the potential to play a significant role in combating the deadliest infections in the future. The MEV developed in this study will pave the way for further vaccination research, particularly considering the absence of a vaccine against <em>P. shigelloides</em> infections. However, the effectiveness of the suggested MEV design must be substantiated through additional experimental validation, as the current analysis is reliant on an integrated platform.</p></div><div id="sec5" class="tsec sec"><h2 class="head no_bottom_margin" id="sec5title">5. Conclusions</h2><p id="p0210" class="p p-first-last">As the global population continues to expand, the prevalence of infections caused by <em>P. shigelloides</em> is becoming alarmingly widespread. The <em>P. shigelliode</em> causing infection has long been a mystery, but now it is a public threat throughout the globe. Patients afflicted by <em>P. shigelloides</em> infections currently lack access to vaccines or other effective medical preventative measures. Regarding this<em>, in silico</em> techniques provide a helping hand in designing novel and powerful vaccines in a short time. This study uses various bioinformatics-related techniques to generate MEV construct against <em>P. shigelloides</em> infections that include CTL, HTL, and LBL, ultimately leading to strong immune responses. Additionally, MD simulations and docking analysis have confirmed the stability of the proposed vaccine. In silico expression, analysis has also validated the feasibility of expressing the MEV in bacterial hosts. Nevertheless, further in vivo and in vitro studies are imperative to conclusively validate the safety and efficacy of the proposed vaccine construct.</p></div><div id="sec6" class="tsec sec"><h2 class="head no_bottom_margin" id="sec6title">Data availability statement</h2><p id="p0215" class="p p-first-last">Not applicable.</p></div><div id="sec8" class="tsec sec"><h2 class="head no_bottom_margin" id="sec8title">CRediT authorship contribution statement</h2><p id="p0225" class="p p-first-last"><strong>Danish Rasool:</strong> Writing &#x02013; original draft, Methodology, Investigation. <strong>Sohail Ahmad Jan:</strong> Supervision, Software, Formal analysis, Conceptualization. <strong>Sumra Umer Khan:</strong> Validation, Data curation. <strong>Nazia Nahid:</strong> Writing &#x02013; review &#x00026; editing, Visualization, Investigation, Formal analysis. <strong>Usman Ali Ashfaq:</strong> Writing &#x02013; review &#x00026; editing, Supervision, Project administration, Conceptualization. <strong>Ahitsham Umar:</strong> Methodology, Investigation. <strong>Muhammad Qasim:</strong> Validation, Formal analysis. <strong>Fatima Noor:</strong> Software, Investigation, Formal analysis. <strong>Abdur Rehman:</strong> Validation, Formal analysis. <strong>Kiran Shahzadi:</strong> Formal analysis, Data curation. <strong>Abdulrahman Alshammari:</strong> Writing &#x02013; review &#x00026; editing, Investigation, Formal analysis, Data curation. <strong>Metab Alharbi:</strong> Investigation, Formal analysis, Data curation. <strong>Muhammad Atif Nisar:</strong> Writing &#x02013; review &#x00026; editing.</p></div><div id="sec-a.q.h" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.q.htitle">Declaration of competing interest</h2><p id="p0230" class="p p-first-last">The authors declare the following financial interests/personal relationships which may be considered as potential competing interests:Usman Ali Ashfaq reports was provided by Government College University Faisalabad. Usman Ali Ashfaq reports a relationship with Government College University Faisalabad that includes: employment. If there are other authors, they declare that they have no known competing financial interests or personal relationships that could have appeared to influence the work reported in this paper.</p></div><div id="ack0010" class="tsec sec"><h2 class="head no_bottom_margin" id="ack0010title">Acknowledgment</h2><div class="sec"><p id="p0235">Authors are thankful to the Researchers Supporting Project number (RSP2024R491), King Saud University, Riyadh, Saudi Arabia.</p></div></div><div id="fn-group-a.r.d" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.r.dtitle">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="appsec2"><sup>Appendix A</sup>Supplementary data to this article can be found online at <a href="https://doi.org/10.1016/j.heliyon.2024.e31304" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://doi.org/10.1016/j.heliyon.2024.e31304</a>.</p></div></div><div id="appsec1" class="tsec bk-sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="appsec1title">Appendix A. Supplementary data</h2><!--/article/back/sec/--><p id="p0240" class="p p-first-last">The following is the Supplementary data to this article:</p><div class="sec suppmat" id="mmc1"><!--caption a9--><strong>Multimedia component 1:</strong><div class="sup-box half_rhythm" id="media-a.r.b.c.a.b"><a href="/pmc/articles/PMC11153098/bin/mmc1.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(12K, docx)</sup>Multimedia component 1</div></div><p></p></div><div id="cebib0010" class="tsec sec"><h2 class="head no_bottom_margin" id="cebib0010title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="bib1">1. <span class="element-citation" id="sref1">Janda J., Family I. Enterobacteriaceae, genus XXVII. Plesiomonas Habs and Schubert 1962. <span><span class="ref-journal">Bergey's manual of systematic bacteriology. </span>2005;<span class="ref-vol">2</span>:740–744.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bergey's+manual+of+systematic+bacteriology&amp;title=Enterobacteriaceae,+genus+XXVII.+Plesiomonas+Habs+and+Schubert+1962&amp;author=J.+Janda&amp;author=I.+Family&amp;volume=2&amp;publication_year=2005&amp;pages=740-744&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib2">2. <span class="element-citation" id="sref2">Ciznar I., et al.  <span class="ref-journal">Food-Borne Pathogens.</span> Springer; 2006. Plesiomonas shigelloides; pp. 73–80. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Food-Borne+Pathogens&amp;author=I.+Ciznar&amp;publication_year=2006&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib3">3. <span class="element-citation" id="sref3">Janda J.M., Abbott S.L., McIver C.J. Plesiomonas shigelloides revisited. <span><span class="ref-journal">Clin. Microbiol. Rev. </span>2016;<span class="ref-vol">29</span>(2):349–374.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4786884/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26960939" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Microbiol.+Rev.&amp;title=Plesiomonas+shigelloides+revisited&amp;author=J.M.+Janda&amp;author=S.L.+Abbott&amp;author=C.J.+McIver&amp;volume=29&amp;issue=2&amp;publication_year=2016&amp;pages=349-374&amp;pmid=26960939&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib4">4. <span class="element-citation" id="sref4">Keating J.P. Chronic diarrhea. <span><span class="ref-journal">Pediatr. Rev. </span>2005;<span class="ref-vol">26</span>(1):5–14.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15629902" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pediatr.+Rev.&amp;title=Chronic+diarrhea&amp;author=J.P.+Keating&amp;volume=26&amp;issue=1&amp;publication_year=2005&amp;pages=5-14&amp;pmid=15629902&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib5">5. <span class="element-citation" id="sref5">Janda J.M., Abbott S.L., McIver C.J. Plesiomonas shigelloides revisited. <span><span class="ref-journal">J.C.m.r. </span>2016;<span class="ref-vol">29</span>(2):349–374.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4786884/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26960939" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.C.m.r.&amp;title=Plesiomonas+shigelloides+revisited&amp;author=J.M.+Janda&amp;author=S.L.+Abbott&amp;author=C.J.+McIver&amp;volume=29&amp;issue=2&amp;publication_year=2016&amp;pages=349-374&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib6">6. <span class="element-citation" id="sref6">Herrington D.A., et al.  vol. 55. 1987. pp. 979–985. (In Vitro and in Vivo Pathogenicity of Plesiomonas Shigelloides). 4. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC260448/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3557621" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Herrington+D.A.+In+Vitro+and+in+Vivo+Pathogenicity+of+Plesiomonas+Shigelloides+vol.+55+1987+979+985+4+" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib7">7. <span class="element-citation" id="sref7">Schneider F., et al. Plesiomonas shigelloides pneumonia. <span>2009;<span class="ref-vol">39</span>(6):397–400.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19167851" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Schneider+F.+Plesiomonas+shigelloides+pneumonia+39+6+2009+397+400+" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib8">8. <span class="element-citation" id="sref8">Pence M.A.  Am Soc Microbiol; 2016. The Brief Case: Wound Infection with Plesiomonas Shigelloides Following a Freshwater Injury. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4844745/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27114564" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Pence+M.A.+The+Brief+Case:+Wound+Infection+with+Plesiomonas+Shigelloides+Following+a+Freshwater+Injury+2016+Am+Soc+Microbiol+" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib9">9. <span class="element-citation" id="sref9">McDonald A.E. A case of plesiomonas bacteremia without reported freshwater exposure. <span><span class="ref-journal">J.H.H.J.o.M. </span>2020;<span class="ref-vol">1</span>(5):6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10324751/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37426611" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.H.H.J.o.M.&amp;title=A+case+of+plesiomonas+bacteremia+without+reported+freshwater+exposure&amp;author=A.E.+McDonald&amp;volume=1&amp;issue=5&amp;publication_year=2020&amp;pages=6&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib10">10. <span class="element-citation" id="sref10">Janda J.M., Lopez D.L.  <span class="ref-journal">Practical Handbook of Microbiology.</span> CRC Press; 2021. The family enterobacteriaceae; pp. 353–362. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Practical+Handbook+of+Microbiology&amp;author=J.M.+Janda&amp;author=D.L.+Lopez&amp;publication_year=2021&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib11">11. <span class="element-citation" id="sref11">Abbott S.L. Klebsiella, enterobacter, citrobacter, serratia, plesiomonas, and other enterobacteriaceae. <span><span class="ref-journal">J.M.o.c.m. </span>2011:639–657.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.M.o.c.m.&amp;title=Klebsiella,+enterobacter,+citrobacter,+serratia,+plesiomonas,+and+other+enterobacteriaceae&amp;author=S.L.+Abbott&amp;publication_year=2011&amp;pages=639-657&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib12">12. <span class="element-citation" id="sref12">Miller M.L., Koburger J.A. Plesiomonas shigelloides: an opportunistic food and waterborne pathogen. <span><span class="ref-journal">J.J.o.f.p. </span>1985;<span class="ref-vol">48</span>(5):449–457.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30943637" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.J.o.f.p.&amp;title=Plesiomonas+shigelloides:+an+opportunistic+food+and+waterborne+pathogen&amp;author=M.L.+Miller&amp;author=J.A.+Koburger&amp;volume=48&amp;issue=5&amp;publication_year=1985&amp;pages=449-457&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib13">13. <span class="element-citation" id="sref13">Noor F., et al.  vol. 27. 2021. pp. 2344–2365. (Recent Advances in Diagnostic and Therapeutic Approaches for Breast Cancer: A Comprehensive Review). 20. [<a href="https://pubmed.ncbi.nlm.nih.gov/33655849" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Noor+F.+Recent+Advances+in+Diagnostic+and+Therapeutic+Approaches+for+Breast+Cancer:+A+Comprehensive+Review+vol.+27+2021+2344+2365+20+" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib14">14. <span class="element-citation" id="sref14">Sufyan M., et al.  vol. 28. 2021. pp. 5518–5525. (Identifying Key Genes and Screening Therapeutic Agents Associated with Diabetes Mellitus and HCV-Related Hepatocellular Carcinoma by Bioinformatics Analysis). 10. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8459114/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34588861" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Sufyan+M.+Identifying+Key+Genes+and+Screening+Therapeutic+Agents+Associated+with+Diabetes+Mellitus+and+HCV-Related+Hepatocellular+Carcinoma+by+Bioinformatics+Analysis+vol.+28+2021+5518+5525+10+" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib15">15. <span class="element-citation" id="sref15">Noor F., et al. Comprehensive computational analysis reveals human respiratory syncytial virus encoded microRNA and host specific target genes associated with antiviral immune responses and protein binding. <span>2021;<span class="ref-vol">33</span>(7)</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Noor+F.+Comprehensive+computational+analysis+reveals+human+respiratory+syncytial+virus+encoded+microRNA+and+host+specific+target+genes+associated+with+antiviral+immune+responses+and+protein+binding+33+7+2021+101562+" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib16">16. <span class="element-citation" id="sref16">Pareek C.S., Smoczynski R., Tretyn A. Sequencing technologies and genome sequencing. <span><span class="ref-journal">J.J.o.a.g. </span>2011;<span class="ref-vol">52</span>(4):413–435.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3189340/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21698376" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.J.o.a.g.&amp;title=Sequencing+technologies+and+genome+sequencing&amp;author=C.S.+Pareek&amp;author=R.+Smoczynski&amp;author=A.+Tretyn&amp;volume=52&amp;issue=4&amp;publication_year=2011&amp;pages=413-435&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib17">17. <span class="element-citation" id="sref17">Aslam S., et al. Proteome based mapping and reverse vaccinology techniques to contrive multi-epitope based subunit vaccine (MEBSV) against Streptococcus pyogenes. <span>2022;<span class="ref-vol">100</span></span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35231667" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Aslam+S.+Proteome+based+mapping+and+reverse+vaccinology+techniques+to+contrive+multi-epitope+based+subunit+vaccine+(MEBSV)+against+Streptococcus+pyogenes+100+2022+105259+" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib18">18. <span class="element-citation" id="sref18">Noor F., et al. Construction of miRNA-mRNA network for the identification of key biological markers and their associated pathways in IgA nephropathy by employing the integrated bioinformatics analysis. <span>2021;<span class="ref-vol">28</span>(9):4938–4945.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8381040/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34466069" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Noor+F.+Construction+of+miRNA-mRNA+network+for+the+identification+of+key+biological+markers+and+their+associated+pathways+in+IgA+nephropathy+by+employing+the+integrated+bioinformatics+analysis+28+9+2021+4938+4945+" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib19">19. <span class="element-citation" id="sref19">Noor F., et al.  vol. 145. 2022. Designing a multi-epitope vaccine against Chlamydia pneumoniae by integrating the core proteomics. (Subtractive Proteomics and Reverse Vaccinology-Based Immunoinformatics Approaches). [<a href="https://pubmed.ncbi.nlm.nih.gov/35429833" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Noor+F.+Designing+a+multi-epitope+vaccine+against+Chlamydia+pneumoniae+by+integrating+the+core+proteomics+Subtractive+Proteomics+and+Reverse+Vaccinology-Based+Immunoinformatics+Approaches+vol.+145+2022+105507+" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib20">20. <span class="element-citation" id="sref20">Aslam S., et al.  vol. 10. 2021. p. 997. (Designing a Multi-Epitope Vaccine against Chlamydia trachomatis by Employing Integrated Core Proteomics, Immuno-Informatics and in Silico Approaches). 10. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8533590/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34681096" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Aslam+S.+Designing+a+Multi-Epitope+Vaccine+against+Chlamydia+trachomatis+by+Employing+Integrated+Core+Proteomics,+Immuno-Informatics+and+in+Silico+Approaches+vol.+10+2021+997+10+" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib21">21. <span class="element-citation" id="sref21">Kalita P., et al. Development of multi-epitope driven subunit vaccine against Fasciola gigantica using immunoinformatics approach. <span><span class="ref-journal">Int. J. Biol. Macromol. </span>2019;<span class="ref-vol">138</span>:224–233.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31279880" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Biol.+Macromol.&amp;title=Development+of+multi-epitope+driven+subunit+vaccine+against+Fasciola+gigantica+using+immunoinformatics+approach&amp;author=P.+Kalita&amp;volume=138&amp;publication_year=2019&amp;pages=224-233&amp;pmid=31279880&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib22">22. <span class="element-citation" id="sref22">Wen Q.-F., et al.  vol. 10. 2019. p. 1236. (Geptop 2.0: an Updated, More Precise, and Faster Geptop Server for Identification of Prokaryotic Essential Genes). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6558110/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31214154" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Wen+Q.-F.+Geptop+2.0:+an+Updated,+More+Precise,+and+Faster+Geptop+Server+for+Identification+of+Prokaryotic+Essential+Genes+vol.+10+2019+1236+" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib23">23. <span class="element-citation" id="sref23">Azhagesan K., Ravindran B., Raman K. Network-based features enable prediction of essential genes across diverse organisms. <span><span class="ref-journal">PLoS One. </span>2018;<span class="ref-vol">13</span>(12)</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6292609/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30543651" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Network-based+features+enable+prediction+of+essential+genes+across+diverse+organisms&amp;author=K.+Azhagesan&amp;author=B.+Ravindran&amp;author=K.+Raman&amp;volume=13&amp;issue=12&amp;publication_year=2018&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib24">24. <span class="element-citation" id="sref24">Yu N.Y., et al. PSORTb 3.0: improved protein subcellular localization prediction with refined localization subcategories and predictive capabilities for all prokaryotes. <span>2010;<span class="ref-vol">26</span>(13):1608–1615.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2887053/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20472543" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Yu+N.Y.+PSORTb+3.0:+improved+protein+subcellular+localization+prediction+with+refined+localization+subcategories+and+predictive+capabilities+for+all+prokaryotes+26+13+2010+1608+1615+" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib25">25. <span class="element-citation" id="sref25">Doytchinova I.A., Flower D.R. VaxiJen: a server for prediction of protective antigens, tumour antigens and subunit vaccines. <span><span class="ref-journal">BMC Bioinf. </span>2007;<span class="ref-vol">8</span>(1):4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1780059/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17207271" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinf.&amp;title=VaxiJen:+a+server+for+prediction+of+protective+antigens,+tumour+antigens+and+subunit+vaccines&amp;author=I.A.+Doytchinova&amp;author=D.R.+Flower&amp;volume=8&amp;issue=1&amp;publication_year=2007&amp;pages=4&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib26">26. <span class="element-citation" id="sref26">Krogh A., et al. Predicting transmembrane protein topology with a hidden Markov model: application to complete genomes. <span><span class="ref-journal">J. Mol. Biol. </span>2001;<span class="ref-vol">305</span>(3):567–580.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11152613" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Mol.+Biol.&amp;title=Predicting+transmembrane+protein+topology+with+a+hidden+Markov+model:+application+to+complete+genomes&amp;author=A.+Krogh&amp;volume=305&amp;issue=3&amp;publication_year=2001&amp;pages=567-580&amp;pmid=11152613&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib27">27. <span class="element-citation" id="sref27">Kim Y., et al. Immune epitope database analysis resource. <span>2012;<span class="ref-vol">40</span>(W1):W525–W530.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3394288/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22610854" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Kim+Y.+Immune+epitope+database+analysis+resource+40+W1+2012+W525+W530+" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib28">28. <span class="element-citation" id="sref28">Moutaftsi M., et al. A consensus epitope prediction approach identifies the breadth of murine T CD8+-cell responses to vaccinia virus. <span><span class="ref-journal">Nat. Biotechnol. </span>2006;<span class="ref-vol">24</span>(7):817–819.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16767078" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=A+consensus+epitope+prediction+approach+identifies+the+breadth+of+murine+T+CD8+-cell+responses+to+vaccinia+virus&amp;author=M.+Moutaftsi&amp;volume=24&amp;issue=7&amp;publication_year=2006&amp;pages=817-819&amp;pmid=16767078&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib29">29. <span class="element-citation" id="sref29">Doytchinova I.A., Flower D.R. VaxiJen: a server for prediction of protective antigens, tumour antigens and subunit vaccines. <span><span class="ref-journal">BMC Bioinf. </span>2007;<span class="ref-vol">8</span>(1):1–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1780059/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17207271" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinf.&amp;title=VaxiJen:+a+server+for+prediction+of+protective+antigens,+tumour+antigens+and+subunit+vaccines&amp;author=I.A.+Doytchinova&amp;author=D.R.+Flower&amp;volume=8&amp;issue=1&amp;publication_year=2007&amp;pages=1-7&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib30">30. <span class="element-citation" id="sref30">Gupta S., et al. In silico approach for predicting toxicity of peptides and proteins. <span><span class="ref-journal">PLoS One. </span>2013;<span class="ref-vol">8</span>(9)</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3772798/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24058508" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=In+silico+approach+for+predicting+toxicity+of+peptides+and+proteins&amp;author=S.+Gupta&amp;volume=8&amp;issue=9&amp;publication_year=2013&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib32">31. <span class="element-citation" id="sref32">Shahid F., et al.  vol. 80. 2020.  (Immunoinformatics Guided Rational Design of a Next Generation Multi Epitope Based Peptide (MEBP) Vaccine by Exploring Zika Virus Proteome). [<a href="https://pubmed.ncbi.nlm.nih.gov/31962160" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Shahid+F.+Immunoinformatics+Guided+Rational+Design+of+a+Next+Generation+Multi+Epitope+Based+Peptide+(MEBP)+Vaccine+by+Exploring+Zika+Virus+Proteome+vol.+80+2020+104199+" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib33">32. <span class="element-citation" id="sref33">Cooper M.D. The early history of B cells. <span><span class="ref-journal">Nat. Rev. Immunol. </span>2015;<span class="ref-vol">15</span>(3):191–197.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25656707" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Immunol.&amp;title=The+early+history+of+B+cells&amp;author=M.D.+Cooper&amp;volume=15&amp;issue=3&amp;publication_year=2015&amp;pages=191-197&amp;pmid=25656707&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib34">33. <span class="element-citation" id="sref34">Saha S., Raghava G.P.S. Prediction of continuous B&#x02010;cell epitopes in an antigen using recurrent neural network. <span><span class="ref-journal">Proteins: Struct., Funct., Bioinf. </span>2006;<span class="ref-vol">65</span>(1):40–48.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16894596" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins:+Struct.,+Funct.,+Bioinf.&amp;title=Prediction+of+continuous+B&#x02010;cell+epitopes+in+an+antigen+using+recurrent+neural+network&amp;author=S.+Saha&amp;author=G.P.S.+Raghava&amp;volume=65&amp;issue=1&amp;publication_year=2006&amp;pages=40-48&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib37">34. <span class="element-citation" id="sref37">Bui H.-H., et al. Predicting population coverage of T-cell epitope-based diagnostics and vaccines. <span><span class="ref-journal">BMC Bioinf. </span>2006;<span class="ref-vol">7</span>(1):1–5.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1513259/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16545123" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinf.&amp;title=Predicting+population+coverage+of+T-cell+epitope-based+diagnostics+and+vaccines&amp;author=H.-H.+Bui&amp;volume=7&amp;issue=1&amp;publication_year=2006&amp;pages=1-5&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib38">35. <span class="element-citation" id="sref38">Nain Z., et al. Proteome-wide screening for designing a multi-epitope vaccine against emerging pathogen Elizabethkingia anophelis using immunoinformatic approaches. <span><span class="ref-journal">J. Biomol. Struct. Dyn. </span>2020;<span class="ref-vol">38</span>(16):4850–4867.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31709929" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biomol.+Struct.+Dyn.&amp;title=Proteome-wide+screening+for+designing+a+multi-epitope+vaccine+against+emerging+pathogen+Elizabethkingia+anophelis+using+immunoinformatic+approaches&amp;author=Z.+Nain&amp;volume=38&amp;issue=16&amp;publication_year=2020&amp;pages=4850-4867&amp;pmid=31709929&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib39">36. <span class="element-citation" id="sref39">Li W., et al. Peptide vaccine: progress and challenges. <span><span class="ref-journal">Vaccines. </span>2014;<span class="ref-vol">2</span>(3):515–536.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4494216/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26344743" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Vaccines&amp;title=Peptide+vaccine:+progress+and+challenges&amp;author=W.+Li&amp;volume=2&amp;issue=3&amp;publication_year=2014&amp;pages=515-536&amp;pmid=26344743&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib36">37. <span class="element-citation" id="sref36">Zhang Q., et al. Immune epitope database analysis resource (IEDB-AR) <span>2008;<span class="ref-vol">36</span>(suppl_2):W513–W518.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2447801/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18515843" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Zhang+Q.+Immune+epitope+database+analysis+resource+(IEDB-AR)+36+suppl_2+2008+W513+W518+" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib40">38. <span class="element-citation" id="sref40">Arai R., et al. Design of the linkers which effectively separate domains of a bifunctional fusion protein. <span><span class="ref-journal">Protein Eng. </span>2001;<span class="ref-vol">14</span>(8):529–532.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11579220" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Eng.&amp;title=Design+of+the+linkers+which+effectively+separate+domains+of+a+bifunctional+fusion+protein&amp;author=R.+Arai&amp;volume=14&amp;issue=8&amp;publication_year=2001&amp;pages=529-532&amp;pmid=11579220&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib41">39. <span class="element-citation" id="sref41">Pandey R.K., Bhatt T.K., Prajapati V.K. Novel immunoinformatics approaches to design multi-epitope subunit vaccine for malaria by investigating anopheles salivary protein. <span><span class="ref-journal">Sci. Rep. </span>2018;<span class="ref-vol">8</span>(1):1–11.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5773588/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29348555" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Rep.&amp;title=Novel+immunoinformatics+approaches+to+design+multi-epitope+subunit+vaccine+for+malaria+by+investigating+anopheles+salivary+protein&amp;author=R.K.+Pandey&amp;author=T.K.+Bhatt&amp;author=V.K.+Prajapati&amp;volume=8&amp;issue=1&amp;publication_year=2018&amp;pages=1-11&amp;pmid=29311619&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib42">40. <span class="element-citation" id="sref42">Mahram A., Herbordt M.C.  <span class="ref-journal">Proceedings of the 24th ACM International Conference on Supercomputing.</span> 2010. Fast and accurate NCBI BLASTP: acceleration with multiphase FPGA-based prefiltering. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Proceedings+of+the+24th+ACM+International+Conference+on+Supercomputing&amp;author=A.+Mahram&amp;author=M.C.+Herbordt&amp;publication_year=2010&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib43">41. <span class="element-citation" id="sref43">Walker J.M.  Springer; 2005. The Proteomics Protocols Handbook. <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Walker+J.M.+The+Proteomics+Protocols+Handbook+2005+Springer+" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib44">42. <span class="element-citation" id="sref44">Zheng W., et al. Deep&#x02010;learning contact&#x02010;map guided protein structure prediction in CASP13. <span><span class="ref-journal">Proteins: Struct., Funct., Bioinf. </span>2019;<span class="ref-vol">87</span>(12):1149–1164.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6851476/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31365149" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins:+Struct.,+Funct.,+Bioinf.&amp;title=Deep&#x02010;learning+contact&#x02010;map+guided+protein+structure+prediction+in+CASP13&amp;author=W.+Zheng&amp;volume=87&amp;issue=12&amp;publication_year=2019&amp;pages=1149-1164&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib45">43. <span class="element-citation" id="sref45">Heo L., Park H., Seok C. GalaxyRefine: protein structure refinement driven by side-chain repacking. <span><span class="ref-journal">Nucleic Acids Res. </span>2013;<span class="ref-vol">41</span>(W1):W384–W388.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3692086/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23737448" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=GalaxyRefine:+protein+structure+refinement+driven+by+side-chain+repacking&amp;author=L.+Heo&amp;author=H.+Park&amp;author=C.+Seok&amp;volume=41&amp;issue=W1&amp;publication_year=2013&amp;pages=W384-W388&amp;pmid=23737448&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib46">44. <span class="element-citation" id="sref46">Lovell S.C., et al. Structure validation by C&#x003b1; geometry: &#x003c6;, &#x003c8; and C&#x003b2; deviation. <span><span class="ref-journal">Proteins: Struct., Funct., Bioinf. </span>2003;<span class="ref-vol">50</span>(3):437–450.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12557186" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins:+Struct.,+Funct.,+Bioinf.&amp;title=Structure+validation+by+C&#x003b1;+geometry:+&#x003c6;,+&#x003c8;+and+C&#x003b2;+deviation&amp;author=S.C.+Lovell&amp;volume=50&amp;issue=3&amp;publication_year=2003&amp;pages=437-450&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib47">45. <span class="element-citation" id="sref47">Lengths M., Angles M. Limitations of structure evaluation tools errat. <span><span class="ref-journal">Quick Guideline Comput Drug Des. </span>2018;<span class="ref-vol">16</span>:75.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Quick+Guideline+Comput+Drug+Des&amp;title=Limitations+of+structure+evaluation+tools+errat&amp;author=M.+Lengths&amp;author=M.+Angles&amp;volume=16&amp;publication_year=2018&amp;pages=75&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib48">46. <span class="element-citation" id="sref48">Craig D.B., Dombkowski A.A. Disulfide by Design 2.0: a web-based tool for disulfide engineering in proteins. <span><span class="ref-journal">BMC Bioinf. </span>2013;<span class="ref-vol">14</span>(1):1–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3898251/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24289175" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinf.&amp;title=Disulfide+by+Design+2.0:+a+web-based+tool+for+disulfide+engineering+in+proteins&amp;author=D.B.+Craig&amp;author=A.A.+Dombkowski&amp;volume=14&amp;issue=1&amp;publication_year=2013&amp;pages=1-7&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib49">47. <span class="element-citation" id="sref49">Van Zundert G., et al. The HADDOCK2. 2 web server: user-friendly integrative modeling of biomolecular complexes. <span><span class="ref-journal">J. Mol. Biol. </span>2016;<span class="ref-vol">428</span>(4):720–725.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26410586" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Mol.+Biol.&amp;title=The+HADDOCK2.+2+web+server:+user-friendly+integrative+modeling+of+biomolecular+complexes&amp;author=G.+Van+Zundert&amp;volume=428&amp;issue=4&amp;publication_year=2016&amp;pages=720-725&amp;pmid=26410586&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib50">48. <span class="element-citation" id="sref50">Laskowski R.A. PDBsum new things. <span><span class="ref-journal">Nucleic Acids Res. </span>2009;<span class="ref-vol">37</span>(suppl_1):D355–D359.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2686501/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18996896" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=PDBsum+new+things&amp;author=R.A.+Laskowski&amp;volume=37&amp;issue=suppl_1&amp;publication_year=2009&amp;pages=D355-D359&amp;pmid=18996896&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib51">49. <span class="element-citation" id="sref51">Van Aalten D.M., et al. A comparison of techniques for calculating protein essential dynamics. <span><span class="ref-journal">J. Comput. Chem. </span>1997;<span class="ref-vol">18</span>(2):169–181.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Comput.+Chem.&amp;title=A+comparison+of+techniques+for+calculating+protein+essential+dynamics&amp;author=D.M.+Van+Aalten&amp;volume=18&amp;issue=2&amp;publication_year=1997&amp;pages=169-181&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib52">50. <span class="element-citation" id="sref52">W&#x000fc;thrich K., et al. Correlations between internal mobility and stability of globular proteins. <span><span class="ref-journal">Biophys. J. </span>1980;<span class="ref-vol">32</span>(1):549–560.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1327351/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7248460" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biophys.+J.&amp;title=Correlations+between+internal+mobility+and+stability+of+globular+proteins&amp;author=K.+W&#x000fc;thrich&amp;volume=32&amp;issue=1&amp;publication_year=1980&amp;pages=549-560&amp;pmid=7248460&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib53">51. <span class="element-citation" id="sref53">L&#x000f3;pez-Blanco J.R., et al. iMODS: internal coordinates normal mode analysis server. <span><span class="ref-journal">Nucleic Acids Res. </span>2014;<span class="ref-vol">42</span>(W1):W271–W276.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4086069/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24771341" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=iMODS:+internal+coordinates+normal+mode+analysis+server&amp;author=J.R.+L&#x000f3;pez-Blanco&amp;volume=42&amp;issue=W1&amp;publication_year=2014&amp;pages=W271-W276&amp;pmid=24771341&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib54">52. <span class="element-citation" id="sref54">Rapin N., et al. Computational immunology meets bioinformatics: the use of prediction tools for molecular binding in the simulation of the immune system. <span><span class="ref-journal">PLoS One. </span>2010;<span class="ref-vol">5</span>(4):e9862.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2855701/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20419125" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Computational+immunology+meets+bioinformatics:+the+use+of+prediction+tools+for+molecular+binding+in+the+simulation+of+the+immune+system&amp;author=N.+Rapin&amp;volume=5&amp;issue=4&amp;publication_year=2010&amp;pages=e9862&amp;pmid=20419125&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib55">53. <span class="element-citation" id="sref55">Grote A., et al. JCat: a novel tool to adapt codon usage of a target gene to its potential expression host. <span><span class="ref-journal">Nucleic Acids Res. </span>2005;<span class="ref-vol">33</span>(suppl_2):W526–W531.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1160137/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15980527" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=JCat:+a+novel+tool+to+adapt+codon+usage+of+a+target+gene+to+its+potential+expression+host&amp;author=A.+Grote&amp;volume=33&amp;issue=suppl_2&amp;publication_year=2005&amp;pages=W526-W531&amp;pmid=15980527&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib56">54. <span class="element-citation" id="sref56">Lund F.E. Cytokine-producing B lymphocytes&#x02014;key regulators of immunity. <span><span class="ref-journal">Curr. Opin. Immunol. </span>2008;<span class="ref-vol">20</span>(3):332–338.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2474694/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18417336" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Immunol.&amp;title=Cytokine-producing+B+lymphocytes&#x02014;key+regulators+of+immunity&amp;author=F.E.+Lund&amp;volume=20&amp;issue=3&amp;publication_year=2008&amp;pages=332-338&amp;pmid=18417336&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib57">55. <span class="element-citation" id="sref57">Mandal B., Whale K., Morson B. Acute colitis due to Plesiomonas shigelloides. <span><span class="ref-journal">Br. Med. J. </span>1982;<span class="ref-vol">285</span>(6354):1539.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1500530/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/6814640" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+Med.+J.&amp;title=Acute+colitis+due+to+Plesiomonas+shigelloides&amp;author=B.+Mandal&amp;author=K.+Whale&amp;author=B.+Morson&amp;volume=285&amp;issue=6354&amp;publication_year=1982&amp;pages=1539&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib58">56. <span class="element-citation" id="sref58">Wong T., et al. Plesiomonas shigelloides infection in Hong Kong: retrospective study of 167 laboratory-confirmed cases. <span><span class="ref-journal">Hong Kong Med. J. </span>2000;<span class="ref-vol">6</span>(4):375–380.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11177159" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hong+Kong+Med.+J.&amp;title=Plesiomonas+shigelloides+infection+in+Hong+Kong:+retrospective+study+of+167+laboratory-confirmed+cases&amp;author=T.+Wong&amp;volume=6&amp;issue=4&amp;publication_year=2000&amp;pages=375-380&amp;pmid=11177159&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib59">57. <span class="element-citation" id="sref59">Brenden R., Miller M.A., Janda J.M. Clinical disease spectrum and pathogenic factors associated with Plesiomonas shigelloides infections in humans. <span><span class="ref-journal">Clin. Infect. Dis. </span>1988;<span class="ref-vol">10</span>(2):303–316.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3287561" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Infect.+Dis.&amp;title=Clinical+disease+spectrum+and+pathogenic+factors+associated+with+Plesiomonas+shigelloides+infections+in+humans&amp;author=R.+Brenden&amp;author=M.A.+Miller&amp;author=J.M.+Janda&amp;volume=10&amp;issue=2&amp;publication_year=1988&amp;pages=303-316&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib68">58. <span class="element-citation" id="sref68">Umar A., et al. Development of a candidate multi-epitope subunit vaccine against Klebsiella aerogenes: subtractive proteomics and Immuno-Informatics approach. <span><span class="ref-journal">Vaccines. </span>2021;<span class="ref-vol">9</span>(11):1373.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8624419/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34835304" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Vaccines&amp;title=Development+of+a+candidate+multi-epitope+subunit+vaccine+against+Klebsiella+aerogenes:+subtractive+proteomics+and+Immuno-Informatics+approach&amp;author=A.+Umar&amp;volume=9&amp;issue=11&amp;publication_year=2021&amp;pages=1373&amp;pmid=34835304&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib73">59. <span class="element-citation" id="sref73">Akhtar N., et al. Secreted Aspartyl Proteinases targeted multi-epitope vaccine design for Candida dubliniensis using immunoinformatics. <span><span class="ref-journal">Vaccines. </span>2023;<span class="ref-vol">11</span>(2):364.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9964391/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36851241" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Vaccines&amp;title=Secreted+Aspartyl+Proteinases+targeted+multi-epitope+vaccine+design+for+Candida+dubliniensis+using+immunoinformatics&amp;author=N.+Akhtar&amp;volume=11&amp;issue=2&amp;publication_year=2023&amp;pages=364&amp;pmid=36851241&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib69">60. <span class="element-citation" id="sref69">Aslam S., et al. Designing a multi-epitope vaccine against Chlamydia trachomatis by employing integrated core proteomics, Immuno-Informatics and in silico approaches. <span><span class="ref-journal">Biology. </span>2021;<span class="ref-vol">10</span>(10):997.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8533590/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34681096" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biology&amp;title=Designing+a+multi-epitope+vaccine+against+Chlamydia+trachomatis+by+employing+integrated+core+proteomics,+Immuno-Informatics+and+in+silico+approaches&amp;author=S.+Aslam&amp;volume=10&amp;issue=10&amp;publication_year=2021&amp;pages=997&amp;pmid=34681096&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib72">61. <span class="element-citation" id="sref72">Mushtaq M., et al. Computational design of a chimeric vaccine against plesiomonas shigelloides using pan-genome and reverse vaccinology. <span><span class="ref-journal">Vaccines. </span>2022;<span class="ref-vol">10</span>(11):1886.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9697808/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36366394" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Vaccines&amp;title=Computational+design+of+a+chimeric+vaccine+against+plesiomonas+shigelloides+using+pan-genome+and+reverse+vaccinology&amp;author=M.+Mushtaq&amp;volume=10&amp;issue=11&amp;publication_year=2022&amp;pages=1886&amp;pmid=36366394&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib70">62. <span class="element-citation" id="sref70">Oyarz&#x000fa;n P., Kobe B. Recombinant and epitope-based vaccines on the road to the market and implications for vaccine design and production. <span><span class="ref-journal">J.H.v. and immunotherapeutics. </span>2016;<span class="ref-vol">12</span>(3):763–767.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4964635/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26430814" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.H.v.+and+immunotherapeutics&amp;title=Recombinant+and+epitope-based+vaccines+on+the+road+to+the+market+and+implications+for+vaccine+design+and+production&amp;author=P.+Oyarz&#x000fa;n&amp;author=B.+Kobe&amp;volume=12&amp;issue=3&amp;publication_year=2016&amp;pages=763-767&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib71">63. <span class="element-citation" id="sref71">Gottlieb T., Ben-Yedidia T. Epitope-based approaches to a universal influenza vaccine. <span><span class="ref-journal">J.J.o.a. </span>2014;<span class="ref-vol">54</span>:15–20.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25172355" ref="reftype=pubmed&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.J.o.a.&amp;title=Epitope-based+approaches+to+a+universal+influenza+vaccine&amp;author=T.+Gottlieb&amp;author=T.+Ben-Yedidia&amp;volume=54&amp;publication_year=2014&amp;pages=15-20&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11153098&amp;issue-id=463821&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Heliyon</span> are provided here courtesy of <strong>Elsevier</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11153098/pdf/main.pdf" class="int-view">PDF (6.0M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11153098/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11153098/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11153098" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11153098%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="g7yoOLH7Sq4l3JE0ikg4YatfRnrWdxpoxxqPsgd4huPTtfq07BmQAhXXCWWv5WFo">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11153098%2F&amp;text=Subtractive%20proteomics-based%20vaccine%20targets%20annotation%20and%20reverse%20vaccinology%20approaches%20to%20identify%20multiepitope%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11153098%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11153098/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11153098/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38845922/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11153098/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38845922/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11153098/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11153098/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC11153098/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
