
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Whole-genome sequencing of Sphingobium baderi SC-1 and identification of a crucial 3-phenoxybenzoic acid-degrading gene - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="322C300B24F3920500004C78C8C2BA1C.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="frontmicrobio" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC11026547/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11026547/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Frontiers in Microbiology" /><meta name="citation_title" content="Whole-genome sequencing of Sphingobium baderi SC-1 and identification of a crucial 3-phenoxybenzoic acid-degrading gene" /><meta name="citation_author" content="Qin Li" /><meta name="citation_author_institution" content="1College of Food Science, Sichuan Agricultural University, Ya&#x02019;an, Sichuan, China" /><meta name="citation_author" content="Qiao Zhou" /><meta name="citation_author_institution" content="1College of Food Science, Sichuan Agricultural University, Ya&#x02019;an, Sichuan, China" /><meta name="citation_author" content="Yuan Chen" /><meta name="citation_author_institution" content="1College of Food Science, Sichuan Agricultural University, Ya&#x02019;an, Sichuan, China" /><meta name="citation_author" content="Kaidi Hu" /><meta name="citation_author_institution" content="1College of Food Science, Sichuan Agricultural University, Ya&#x02019;an, Sichuan, China" /><meta name="citation_author" content="Montserrat Sarr&#x000e0;" /><meta name="citation_author_institution" content="2Departament d&#x02019;Enginyeria Qu&#x000ed;mica, Biol&#x000f2;gica i Ambiental, Escola d&#x02019;Enginyeria, Universitat Aut&#x000f2;noma de Barcelona, Bellaterra, Barcelona, Spain" /><meta name="citation_author" content="Jianlong Li" /><meta name="citation_author_institution" content="1College of Food Science, Sichuan Agricultural University, Ya&#x02019;an, Sichuan, China" /><meta name="citation_author" content="Aiping Liu" /><meta name="citation_author_institution" content="1College of Food Science, Sichuan Agricultural University, Ya&#x02019;an, Sichuan, China" /><meta name="citation_author" content="Likou Zou" /><meta name="citation_author_institution" content="3College of Resources, Sichuan Agricultural University, Chengdu, Sichuan, China" /><meta name="citation_author" content="Shuliang Liu" /><meta name="citation_author_institution" content="1College of Food Science, Sichuan Agricultural University, Ya&#x02019;an, Sichuan, China" /><meta name="citation_publication_date" content="2024" /><meta name="citation_volume" content="15" /><meta name="citation_doi" content="10.3389/fmicb.2024.1361335" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC11026547/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC11026547/" /><meta name="citation_pmid" content="38646623" /><meta name="DC.Title" content="Whole-genome sequencing of Sphingobium baderi SC-1 and identification of a crucial 3-phenoxybenzoic acid-degrading gene" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Frontiers Media SA" /><meta name="DC.Contributor" content="Qin Li" /><meta name="DC.Contributor" content="Qiao Zhou" /><meta name="DC.Contributor" content="Yuan Chen" /><meta name="DC.Contributor" content="Kaidi Hu" /><meta name="DC.Contributor" content="Montserrat Sarr&#x000e0;" /><meta name="DC.Contributor" content="Jianlong Li" /><meta name="DC.Contributor" content="Aiping Liu" /><meta name="DC.Contributor" content="Likou Zou" /><meta name="DC.Contributor" content="Shuliang Liu" /><meta name="DC.Date" content="2024" /><meta name="DC.Identifier" content="10.3389/fmicb.2024.1361335" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Whole-genome sequencing of Sphingobium baderi SC-1 and identification of a crucial 3-phenoxybenzoic acid-degrading gene" /><meta property="og:type" content="article" /><meta property="og:description" content="As an efficient degradation strain, Sphingobium baderi SC-1 can breakdown 3-phenoxybenzoic acid (3-PBA) with high proficiency. To investigate the internal factors that regulate this process, we conducted whole-genome sequencing and successfully identified ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11026547/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11026547/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22Front%20Microbiol%22[journal]" class="navlink">Front Microbiol</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11026547
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11026547/pdf/fmicb-15-1361335.pdf" class="int-view">PDF (2.5M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11026547/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11026547/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11026547" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11026547%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="AMC99U4Y7PLObObDQ48AglApOoAAhP0j28YHBrkS64R55a5OSFBL7ORWuNpEmzh3">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11026547%2F&amp;text=Whole-genome%20sequencing%20of%20Sphingobium%20baderi%20SC-1%20and%20identification%20of%20a%20crucial%203-phenoxybenzoic%20acid-degrading%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11026547%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11026547/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11026547/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38646623/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11026547/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38646623/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11026547/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11026547/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22Front%20Microbiol%22[journal]" class="navlink">Front Microbiol</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11026547
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-frontmicrobio.gif" alt="Logo of frontmicrobio" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.frontiersin.org/microbiology/archive" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Front Microbiol.</span> 2024; 15: 1361335. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2024 Apr 5. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.3389%2Ffmicb.2024.1361335" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.3389/fmicb.2024.1361335</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC11026547</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38646623">38646623</a></div></div></div><h1 class="content-title">Whole-genome sequencing of <em>Sphingobium baderi</em> SC-1 and identification of a crucial 3-phenoxybenzoic acid-degrading gene</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Li%20Q%5BAuthor%5D" class="affpopup" co-rid="_co_idm139905469868624" co-class="co-affbox">Qin Li</a>,<sup>#</sup><sup>
1
,</sup><sup>
&#x02020;
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Zhou%20Q%5BAuthor%5D" class="affpopup" co-rid="_co_idm139905469864208" co-class="co-affbox">Qiao Zhou</a>,<sup>#</sup><sup>
1
,</sup><sup>
&#x02020;
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Chen%20Y%5BAuthor%5D" class="affpopup" co-rid="_co_idm139905469859920" co-class="co-affbox">Yuan Chen</a>,<sup>#</sup><sup>
1
,</sup><sup>
&#x02020;
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hu%20K%5BAuthor%5D" class="affpopup" co-rid="_co_idm139905463649872" co-class="co-affbox">Kaidi Hu</a>,<sup>
1
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Sarr&#x000e0;%20M%5BAuthor%5D" class="affpopup" co-rid="_co_idm139905463645792" co-class="co-affbox">Montserrat Sarr&#x000e0;</a>,<sup>
2
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Li%20J%5BAuthor%5D" class="affpopup" co-rid="_co_idm139905463641344" co-class="co-affbox">Jianlong Li</a>,<sup>
1
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Liu%20A%5BAuthor%5D" class="affpopup" co-rid="_co_idm139905509474208" co-class="co-affbox">Aiping Liu</a>,<sup>
1
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Zou%20L%5BAuthor%5D" class="affpopup" co-rid="_co_idm139905509470208" co-class="co-affbox">Likou Zou</a>,<sup>
3
</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Liu%20S%5BAuthor%5D" class="affpopup" co-rid="_co_idm139905509466288" co-class="co-affbox">Shuliang Liu</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>
1
,</sup><sup>
*
</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139905469868624"><h3 class="no_margin">Qin Li</h3><p><sup>1</sup>College of Food Science, Sichuan Agricultural University, Ya&#x02019;an, Sichuan, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Li%20Q%5BAuthor%5D">Qin Li</a></div></div><div id="_co_idm139905469864208"><h3 class="no_margin">Qiao Zhou</h3><p><sup>1</sup>College of Food Science, Sichuan Agricultural University, Ya&#x02019;an, Sichuan, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Zhou%20Q%5BAuthor%5D">Qiao Zhou</a></div></div><div id="_co_idm139905469859920"><h3 class="no_margin">Yuan Chen</h3><p><sup>1</sup>College of Food Science, Sichuan Agricultural University, Ya&#x02019;an, Sichuan, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Chen%20Y%5BAuthor%5D">Yuan Chen</a></div></div><div id="_co_idm139905463649872"><h3 class="no_margin">Kaidi Hu</h3><p><sup>1</sup>College of Food Science, Sichuan Agricultural University, Ya&#x02019;an, Sichuan, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hu%20K%5BAuthor%5D">Kaidi Hu</a></div></div><div id="_co_idm139905463645792"><h3 class="no_margin">Montserrat Sarr&#x000e0;</h3><p><sup>2</sup>Departament d&#x02019;Enginyeria Qu&#x000ed;mica, Biol&#x000f2;gica i Ambiental, Escola d&#x02019;Enginyeria, Universitat Aut&#x000f2;noma de Barcelona, Bellaterra, Barcelona, Spain</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Sarr&#x000e0;%20M%5BAuthor%5D">Montserrat Sarr&#x000e0;</a></div></div><div id="_co_idm139905463641344"><h3 class="no_margin">Jianlong Li</h3><p><sup>1</sup>College of Food Science, Sichuan Agricultural University, Ya&#x02019;an, Sichuan, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Li%20J%5BAuthor%5D">Jianlong Li</a></div></div><div id="_co_idm139905509474208"><h3 class="no_margin">Aiping Liu</h3><p><sup>1</sup>College of Food Science, Sichuan Agricultural University, Ya&#x02019;an, Sichuan, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Liu%20A%5BAuthor%5D">Aiping Liu</a></div></div><div id="_co_idm139905509470208"><h3 class="no_margin">Likou Zou</h3><p><sup>3</sup>College of Resources, Sichuan Agricultural University, Chengdu, Sichuan, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Zou%20L%5BAuthor%5D">Likou Zou</a></div></div><div id="_co_idm139905509466288"><h3 class="no_margin">Shuliang Liu</h3><p><sup>1</sup>College of Food Science, Sichuan Agricultural University, Ya&#x02019;an, Sichuan, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Liu%20S%5BAuthor%5D">Shuliang Liu</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm139905468746176_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139905468746176_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139905468746176_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm139905468746176_ai" style="display:none"><div class="fm-affl" id="aff1"><sup>1</sup>College of Food Science, Sichuan Agricultural University, Ya&#x02019;an, Sichuan, China</div><div class="fm-affl" id="aff2"><sup>2</sup>Departament d&#x02019;Enginyeria Qu&#x000ed;mica, Biol&#x000f2;gica i Ambiental, Escola d&#x02019;Enginyeria, Universitat Aut&#x000f2;noma de Barcelona, Bellaterra, Barcelona, Spain</div><div class="fm-affl" id="aff3"><sup>3</sup>College of Resources, Sichuan Agricultural University, Chengdu, Sichuan, China</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup>Corresponding author.</div><div><sup>#</sup>Contributed equally.</div><div id="fn-a.j.b.h.a">Edited by: Shaohua Chen, South China Agricultural University, China</div><div id="fn-a.j.b.h.b">Reviewed by: Devashish Rath, Bhabha Atomic Research Centre (BARC), India<p class="p p-last">Jiayuan Zhao, Sichuan Normal University, China</p></div><div id="c001">*Correspondence: Shuliang Liu, <a href="mailto:dev@null" data-email="moc.361@999gnailsl" class="oemail">moc.361@999gnailsl</a></div><div id="fn002"><sup>&#x02020;</sup>These authors have contributed equally to this work</div></div><div class="fm-article-notes hide half_rhythm" id="idm139905468746176_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2023 Dec 25; Accepted 2024 Mar 22.</div></div><div class="permissions half_rhythm hide" id="idm139905468746176_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  &#x000a9; 2024 Li, Zhou, Chen, Hu, Sarr&#x000e0;, Li, Liu, Zou and Liu.</div><div class="license half_rhythm">This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div><a href="/pmc/articles/PMC11026547/bin/Data_Sheet_1.DOCX" data-ga-action="click_feat_suppl">Data_Sheet_1.DOCX</a><span style="color:gray"> (3.1M)</span></div><div class="small guid">GUID: CD9A789F-54C1-4548-AF16-F0A62B625E1D</div></div></dd></dl><dl data-length="186" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first-last">The datasets presented in this study can be found in online repositories. The names of the repository/repositories and accession number(s) can be found below: NCBI &#x02013; <a href="/nucleotide/PRJNA981288" data-ga-action="click_feat_suppl" ref="reftype=extlink-DDBJ/EMBL/GenBank&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7CGenBank" target="_blank">PRJNA981288</a>.</p></dd></dl></div><div id="abstract-a.j.b.o" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.j.b.otitle">Abstract</h2><!--article-meta--><div><p class="p p-first-last">As an efficient degradation strain, <em>Sphingobium baderi</em> SC-1 can breakdown 3-phenoxybenzoic acid (3-PBA) with high proficiency. To investigate the internal factors that regulate this process, we conducted whole-genome sequencing and successfully identified the pivotal 3-PBA-degrading gene <em>sca</em> (1,230 bp). After <em>sca</em> was expressed in engineered bacteria, a remarkable degradation efficiency was observed, as 20 mg/L 3-PBA was almost completely decomposed within 24 h. The phenol was formed as one of the degradation products. Notably, in addition to their ability to degrade 3-PBA, the resting cells proficiently degraded 4&#x02032;-HO-3-PBA and 3&#x02032;-HO-4-PBA. In conclusion, we successfully identified and validated <em>sca</em> as the pivotal enzyme responsible for the efficient degradation of 3-PBA from <em>Sphingomonas baderi</em>, providing a crucial theoretical foundation for further explorations on the degradation potential of SC-1.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text"><em>Sphingobium baderi</em> SC-1, whole-genome sequencing, 3-phenoxybenzoic acid-degrading gene, resting cells, degradation characteristics</span></div></div><div id="S1" class="tsec sec"><h2 class="head no_bottom_margin" id="S1title">1 Introduction</h2><p class="p p-first">At present, pyrethroid insecticides (PYRs) account for more than 30% of global insecticides (<a href="#B27" rid="B27" class=" bibr popnode">Quir&#x000f3;s-Alcal&#x000e1; et al., 2014</a>) and are widely used in agricultural, forestry and residential pest control (mosquito repellents) (<a href="#B31" rid="B31" class=" bibr popnode">Tang W. et al., 2018</a>); in addition, PYRs have replaced some organophosphorus and carbamate insecticides used in agriculture (<a href="#B1" rid="B1" class=" bibr popnode">Barr et al., 2010</a>). 3-Phenoxybenzoic acid (3-PBA, solubility 24.7 mg/L) is the main intermediate product of PYR metabolism in insects, plants and microorganisms (<a href="#B29" rid="B29" class=" bibr popnode">Sun et al., 2014</a>), and its biotoxicity is greater than that of PYRs. Compared to its parent compounds, 3-PBA is more hydrophilic and mobile. As a consequence, 3-PBA can contaminate food and the environment. As a result of exposure to pyrethroids through the food chain, the parent compound undergoes hydrolysis within the human body, leading to the formation of 3-PBA, which can be detected in human urine (<a href="#B40" rid="B40" class=" bibr popnode">Zhan et al., 2020</a>). Studies have shown that 3-PBA has reproductive and immune toxicity and can cause metabolic disorders in humans (<a href="#B26" rid="B26" class=" bibr popnode">Qi et al., 2012</a>; <a href="#B29" rid="B29" class=" bibr popnode">Sun et al., 2014</a>; <a href="#B34" rid="B34" class=" bibr popnode">Ueyama et al., 2015</a>; <a href="#B37" rid="B37" class=" bibr popnode">Wang et al., 2017</a>). In addition, 3-PBA is recalcitrant to biodegradation and possesses antimicrobial activity, which hinders the mineralization of pyrethroids and increases concern over the elimination of 3-PBA. Therefore, the key to reducing PYR pollution is degrading 3-PBA.</p><p class="p p-last">There are many reports on 3-PBA biodegradation, and some 3-PBA-degrading microorganisms have been identified; most of these microorganisms are bacteria, and relatively few are fungi (<a href="#B33" rid="B33" class=" bibr popnode">Topp and Akhtar, 1991</a>; <a href="#B11" rid="B11" class=" bibr popnode">Chen et al., 2012</a>; <a href="#B18" rid="B18" class=" bibr popnode">Jie et al., 2013</a>). Most bacteria can use 3-PBA as the sole carbon source and energy source to degrade 3-PBA through mineralization, while fungi mostly use co-metabolism to degrade 3-PBA (<a href="#B24" rid="B24" class=" bibr popnode">Maloney et al., 1988</a>; <a href="#B15" rid="B15" class=" bibr popnode">Halden and Tepp, 1999</a>; <a href="#B9" rid="B9" class=" bibr popnode">Chen et al., 2011a</a>,<a href="#B10" rid="B10" class=" bibr popnode">b</a>; <a href="#B17" rid="B17" class=" bibr popnode">Jiayuan et al., 2016</a>). In addition, potential degradation pathways of 3-PBA have been proposed by many researchers (<a href="#B13" rid="B13" class=" bibr popnode">Dehmel et al., 1995</a>; <a href="#B30" rid="B30" class=" bibr popnode">Tallur et al., 2008</a>; <a href="#B35" rid="B35" class=" bibr popnode">Wang et al., 2009</a>; <a href="#B18" rid="B18" class=" bibr popnode">Jie et al., 2013</a>; <a href="#B8" rid="B8" class=" bibr popnode">Chen et al., 2015</a>; <a href="#B3" rid="B3" class=" bibr popnode">Birolli et al., 2016a</a>,<a href="#B4" rid="B4" class=" bibr popnode">b</a>; <a href="#B46" rid="B46" class=" bibr popnode">Zhu et al., 2016</a>; <a href="#B32" rid="B32" class=" bibr popnode">Tang J. et al., 2018</a>), who mostly suggested that 3-PBA metabolism begins with the diphenyl ether bond breaking, and oxygen is the key participant (<a href="#B13" rid="B13" class=" bibr popnode">Dehmel et al., 1995</a>; <a href="#B36" rid="B36" class=" bibr popnode">Wang et al., 2014</a>). Oxygenases (monooxygenase and dioxygenase) are important enzymes for the degradation of polycyclic aromatic hydrocarbons. In this process, monooxygenase mainly hydroxylates polycyclic aromatic hydrocarbons, and then dioxygenase mainly cleaves the benzene ring, which is further oxidized into the tricarboxylic acid cycle. Finally, CO<sub>2</sub> and H<sub>2</sub>O are produced (<a href="#B23" rid="B23" class=" bibr popnode">Liang et al., 2006</a>; <a href="#B44" rid="B44" class=" bibr popnode">Zhao et al., 2017</a>). Therefore, it is important to explore dioxygenases. At present, a limited number of dioxygenase genes and corresponding engineered strains have been reported (<a href="#B13" rid="B13" class=" bibr popnode">Dehmel et al., 1995</a>; <a href="#B36" rid="B36" class=" bibr popnode">Wang et al., 2014</a>); examples include the genes <em>pobAB</em>, <em>pbaA1A2BC</em>, <em>d34</em>, and <em>dpeAlA2</em>, which were identified in <em>Pseudomonas pseudoalcaligenes</em> POB310, <em>Sphingobium wenxiniae</em> JZ-1, <em>Acinetobacter</em> sp. 4-D, and <em>Sphingobium</em> sp. SC-3, respectively. In our previous work, we isolated a novel strain, <em>Sphingomonas</em> sp. SC-1 (specifically identified as <em>Sphingobium baderi</em>) from activated sludge, and 300 mg/L 3-PBA could be completely degraded by this strain within 24 h. In addition, the three metabolites of 3-PBA degradation were identified as phenol, catechol and 2-phenoxyphenol. The degradation pathway may involve a variation in the levels of the three metabolites (<a href="#B18" rid="B18" class=" bibr popnode">Jie et al., 2013</a>). However, the intrinsic factors that contribute to this activity remain unclear. In this article, whole-genome sequencing and bioinformatics analysis of strain SC-1 were performed, the key enzyme-encoding genes related to the degradation of 3-PBA were identified, and engineered bacteria were constructed. The studies conducted on PobA and PobB (<a id="__tag_1146741523" class="tag_hotlink" href="/protein/CAA55400.1" ref="/protein/CAA55400.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"CAA55400.1","term_id":"473250","term_text":"CAA55400.1"}}</span>CAA55400.1</a> and <a id="__tag_1146741521" class="tag_hotlink" href="/protein/CAA55401.1" ref="/protein/CAA55401.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"CAA55401.1","term_id":"581466","term_text":"CAA55401.1"}}</span>CAA55401.1</a>) from <em>P. pseudoalcaligenes</em> POB310 have provided us with invaluable insights, serving as a crucial reference for the identification of key genes in our research (<a href="#B13" rid="B13" class=" bibr popnode">Dehmel et al., 1995</a>). This work sheds light on the genetic factors involved in the degradation of 3-PBA by <em>Sphingobium baderi</em> SC-1, greatly increasing knowledge on the mechanism underlying the action of <em>Sphingobium baderi</em> SC-1. Here, the dioxygenase gene, which causes the 3-PBA phenyl ether bond to degrade, was identified in <em>Sphingobium baderi</em>.</p></div><div id="S2" class="tsec sec"><h2 class="head no_bottom_margin" id="S2title">2 Materials and methods</h2><div id="S2.SS1" class="sec sec-first"><h3 id="S2.SS1title">2.1 Microorganisms and vectors</h3><p class="p p-first-last"><em>Sphingomonas baderi</em> SC-1, which was isolated from pesticide plant sludge, was preserved in glycerin at &#x02212;80&#x000b0;C by the Food Microbiology Laboratory of Sichuan Agricultural University. <em>Escherichia coli</em> DH-5&#x003b1; and BL21(DE3) cells, the plasmids pMD18-T and pET28a(+), and Taq polymerase were obtained from Takara (Japan). Restriction endonucleases, T4 DNA ligase and Q5<sup>&#x000ae;</sup> High-Fidelity DNA polymerase were obtained from NEB, Inc. (USA). The kanamycin (Kan) and isopropyl &#x003b2;-D-1-thiogalactopyranoside (IPTG) used in this study were procured from Beijing Solaibao Technology Co., Ltd. The 3-PBA compound (98% purity) was procured from Tixiai (Shanghai) Chemical Industry Development Co., Ltd.</p></div><div id="S2.SS2" class="sec"><h3 id="S2.SS2title">2.2 DNA extraction and whole-genome sequencing</h3><p class="p p-first">The SC-1 strain was cultured in Luria&#x02013;Bertani (LB) medium at 30&#x000b0;C until an optical density of 1.0 at OD<sub>600</sub> was reached. Subsequently, the cell pellets were collected by centrifugation at 4&#x000b0;C (8,000 r/min, 10 min) and washed three times with 0.1 mol/L phosphate-buffered saline (PBS, pH 7). The genomic DNA of strain SC-1 was extracted using the Wizard<sup>&#x000ae;</sup> Genomic DNA Purification Kit following the manufacturer&#x02019;s instructions. Prior to library construction, the DNA was quantified using a TBS-380 fluorometer. Sequencing was performed by Shanghai Majorbio Biopharm Technology Co., Ltd. (China), utilizing both PacBio and Illumina MiSeq platforms (<a href="#B5" rid="B5" class=" bibr popnode">Bolger et al., 2014</a>) according to their standard protocols.</p><ul class="simple" style="list-style-type:none"><li class="a_label"><div class="inline_block a_label">(1)</div><div>For Illumina library construction and sequencing, a minimum of 1 &#x003bc;g of DNA was fragmented to a size of 400 bp using Covaris. Library preparation was carried out utilizing the NEXTflex&#x02122; Rapid DNA-Seq Kit, followed by paired-end (2 &#x000d7; 150 bp) sequencing on an Illumina HiSeq X Ten platform.</div></li><li class="a_label"><div class="inline_block a_label">(2)</div><div>To construct single-molecule real-time sequencing and sequencing libraries, a minimum of 15 &#x003bc;g of DNA was fragmented into 10 kb fragments using G-tubes. Subsequently, the fragments were purified following PacBio&#x02019;s instructions, which involved end-filling and ligation to SMRTbell sequencing adapters. The resulting library underwent three rounds of purification with Agencourt AMPure XP beads (0.45&#x000d7;). Then, the fragments were annealed into single-stranded rings and bound to the immobilized polymerase for subsequent real-time detection during the sequencing step.</div></li></ul></div><div id="S2.SS3" class="sec"><h3 id="S2.SS3title">2.3 Bioinformatics analysis and identification of the 3-PBA dioxygenase gene</h3><p class="p p-first">Bioinformatics analysis was conducted using data generated from the PacBio RS II and Illumina platforms. All analyses were performed on the I-Sanger Cloud platform<sup><sup><a href="#footnote1" rid="footnote1" class=" fn">1</a></sup></sup> provided by Shanghai Maggi Bio. To improve the accuracy of subsequent assemblies and obtain high-quality clean data, reads with lower sequencing quality, higher N proportion, and shorter length were excluded. Canu and HGAP software were used for PacBio data assembly (<a href="#B19" rid="B19" class=" bibr popnode">Koren et al., 2017</a>). The reads were assembled into contigs and further organized into scaffolds to obtain complete chromosome and plasmid genomes. Finally, Illumina sequencing data were utilized to refine the assembly results and determine the circular genome starting site.</p><p>Glimmer, GeneMarkS, tRNAscan-SE, and Barrnap were utilized for predicting coding sequences (CDS), plasmid genes, tRNA, and rRNA, respectively (<a href="#B6" rid="B6" class=" bibr popnode">Borodovsky and McIninch, 1993</a>; <a href="#B2" rid="B2" class=" bibr popnode">Besemer and Borodovsky, 2005</a>; <a href="#B14" rid="B14" class=" bibr popnode">Delcher et al., 2007</a>). The predicted CDSs were annotated using sequence alignment tools, such as BLASTP, Diamond, and HMMER, along with the NR, Swiss-Prot, Pfam, GO COG and KEGG databases.</p><p class="p p-last">The complete genome sequence of <em>Sphingomonas baderi</em> SC-1 was functionally annotated to identify dioxygenase genes. Then, a BLAST search in NCBI was performed to identify the 3-PBA degradation enzyme-encoding gene based on sequence comparison. Phylogenetic trees of the target genes were constructed using MEGA 5.0 and analyzed with 1,000 replicates per clade.</p></div><div id="S2.SS4" class="sec"><h3 id="S2.SS4title">2.4 Cloning and validation of dioxygenase genes</h3><p class="p p-first"><em>Sphingomonas baderi</em> SC-1 cells were cultured in LB media. Target genes without the native secretion signal were amplified by PCR from genomic DNA using corresponding primers (<a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Table 1</a>). The resulting DNA fragments were subcloned and inserted into the pMD18-T plasmid, transformed into <em>E. coli</em> DH-5&#x003b1; cells, and sequenced for verification. Subsequently, the PCR products were cloned and inserted into the pET28a(+) plasmid at the <em>Bam</em>HI and <em>Hin</em>dIII restriction sites and expressed in <em>E. coli</em> BL21 (DE3) to produce dioxygenases. The 1% (v/v) genetically engineered bacteria were grown overnight in 100 ml of LB media supplemented with 40 &#x003bc;g/ml Kan at 37&#x000b0;C, and the resulting culture was used for subsequent experiments on dioxygenase expression.</p><p>After 1.0% (v/v) of the culture was inoculated into 100 ml of LB medium supplemented with 40 &#x003bc;g/ml Kan, the mixture was incubated at 37&#x000b0;C and shaken at 160 rpm for 3.0 h until an optical density (OD<sub>600</sub>) of 0.5&#x0223c;0.6 was reached. Subsequently, induction of expression was achieved by adding IPTG to a final concentration of 1 mmol/L and maintaining the temperature at 20&#x000b0;C for 16 h. Following this, 3-PBA stock solution was added, to achieve a final concentration at 20 mg/L, and the mixture was incubated with shaking at 37&#x000b0;C and rotation at approximately 160 rpm for an additional 48 h. The samples (1 ml) were collected at designated time points and the 3-PBA residue was detected (the reaction was terminated using acetonitrile) (<a href="#B22" rid="B22" class=" bibr popnode">Li et al., 2016</a>). Subsequently, the samples were mixed with 4 ml of acetonitrile and subjected to ultrasonic extraction (40 kHz) for 30 min, followed by centrifugation at 12,000 r/min for 10 min. After supernatant was passed through a filter membrane (0.22 &#x003bc;m) in the organic phase, the filtrate was utilized to assess the percentage degradation of 3-PBA using high-performance liquid chromatography (HPLC) equipped with a Gemini C18 column (pore size, 100 &#x000c5;; particle size, 5.0 &#x003bc;m; dimensions, 150 mm &#x000d7; 4.60 mm), an injection volume of 10 &#x003bc;l, a column temperature of 25&#x000b0;C, and a mobile phase consisting of an acetonitrile-ultrapure water mixture (55:45 v/v). The flow velocity was maintained at 0.7 ml/min, and the UV detection wavelength was set at 210 nm. The <em>E. coli</em> BL21(DE3)-pET-28a(+) strain was used as a control. Moreover, the effects of inoculation amount (0.5%, 1.0%, 1.5%, 2.0%, and 2.5%), preinduction culture time (1.0, 1.5, 2.0, 2.5, and 3.0 h) and induction temperature (16, 18, 20, 22, and 24&#x000b0;C) on the percent degradation of 3-PBA were explored using the percent degradation of 3-PBA as an indicator (see <a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Figure 1</a> for detailed results). Then, the Box-Behnken Center combined experimental design was used to investigate the factors that affect the percent degradation of 3-PBA based on a single factor. The degradation time was set at 6 h.</p><p>According to the optimization described above, the inoculation amount was 1.5%, the preinduction culture time was 2.50 h, and the induction temperature was 20&#x000b0;C (see <a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Tables 2, 3</a> and <a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Figure 2</a> for detailed results). These results were used in subsequent experiments on dioxygenase expression. The degradation of 3-PBA in resting cells, whole-cell lysates, cell lysate supernatants, and cell debris suspensions was subsequently investigated.</p><p>For the resting cells, after IPTG (1 mmol/L) was used for the induction of expression at 20&#x000b0;C for 16 h, 100 ml of culture was centrifuged (8,000 r/min, 10 min) at 4&#x000b0;C, as described above. The obtained biomass was washed once using 0.1 mol/L PBS, 3-PBA stock solution was added to the resuspended cells (resting cells, 3.3 &#x000d7; 10<sup>10</sup> CFU/ml) with 10 ml of 0.1 mol/L PBS, to achieve a final concentration at 20 mg/L, and the mixture was incubated with shaking at 37&#x000b0;C and 160 rpm.</p><p>For the whole cell lysate, after IPTG (1 mmol/L) was used to induce expression at 20&#x000b0;C for 16 h, 100 ml of culture was centrifuged (8,000 r/min, 10 min) at 4&#x000b0;C, as described above. The obtained biomass was washed once using 0.1 mol/L PBS, and the cell resuspended in 10 ml of 0.1 mol/L PBS was ready for ultrasonic wall disruption (65 W, working for 2 s at intervals of 2 s, 25 min) in an ice bath. Then, 900 &#x003bc;l of whole-cell lysate and 100 &#x003bc;l of 3-PBA stock solution were mixed, to achieve a final concentration at 20 mg/L, and the mixture was incubated with shaking (160 rpm) at 37&#x000b0;C and 160 rpm.</p><p class="p p-last">For the cell lysate supernatant and cell debris suspension, the whole cell lysate as described above was centrifuged (8,000 r/min, 10 min) at 4&#x000b0;C, and the supernatant was collected as a cell lysate supernatant. The precipitate was washed once using 0.1 mol/L PBS, and the precipitate was resuspended in 10 ml of 0.1 mol/L PBS as a cell debris suspension. Then, 900 &#x003bc;l of cell lysate supernatant/cell debris suspension and 100 &#x003bc;l of 3-PBA stock solution were mixed, to achieve a final concentration at 20 mg/L, and shaking (160 rpm) was continued at 37&#x000b0;C and 160 rpm.</p></div><div id="S2.SS5" class="sec"><h3 id="S2.SS5title">2.5 Characterizing 3-PBA degradation in resting cells</h3><div id="S2.SS5.SSS1" class="sec sec-first"><p></p><h4 id="S2.SS5.SSS1title" class="inline">2.5.1 Time-course degradation of 3-PBA in resting cells </h4><p class="p p-first-last">As described in section &#x0201c;2.4 Cloning and validation of dioxygenase genes,&#x0201d; 3-PBA was spiked into the resuspended resting cell with 10 ml of 0.1 mol/L PBS up to a final concentration of 20 mg/L, and shaking (160 rpm) was continued at 37&#x000b0;C and 160 rpm. Then, the samples were taken at designated times (0, 0.5, 1, 1.5, 2, 2.5, 3, and 4 h) and the 3-PBA percent degradation was assessed using HPLC.</p></div><div id="S2.SS5.SSS2" class="sec"><p></p><h4 id="S2.SS5.SSS2title" class="inline">2.5.2 Metabolite identification </h4><p class="p p-first">The products of 3-PBA degradation by <em>sca</em>-engineered resting bacterial cells were analyzed using HPLC and gas chromatography-mass spectrometry (GC-MS).</p><p>A standard phenol solution with a concentration of 10 mg/L was prepared and introduced into the 2-h extract of 3-PBA degraded by resting cells. HPLC was then employed to detect the normalized samples according to the corresponding criteria. The HPLC detection conditions for phenol and standardized samples were identical to those described in section &#x0201c;2.4 Cloning and validation of dioxygenase genes.&#x0201d;</p><p>Then, acetonitrile extracted from the degradation of 3-PBA by resting cells was collected (5 ml solution) at various time intervals. The sample was concentrated under a stream of nitrogen, followed by redissolution in 500 &#x003bc;l of ethanol. After centrifugation at 12,000 r/min for 10 min, the supernatant was filtered through an organic phase filter membrane (0.22 &#x003bc;m) for GC-MS analyses.</p><p class="p p-last">The following improved GC-MS detection parameters were employed: chromatography was conducted using an HP-5MS capillary column (30 m &#x000d7; 0.25 mm &#x000d7; 0.25 &#x003bc;m) coated with a 5%-phenyl-methylpolysiloxane stationary phase, and helium was used as the carrier gas. The inlet temperature was set at 260&#x000b0;C, with a sample size of 1 &#x003bc;l and a flow rate of 1.0 ml/min. A programmed temperature method was employed, starting at 80&#x000b0;C for 1 min, followed by a temperature increase from 10&#x000b0;C/min to 260&#x000b0;C in 15 min without any shunt injection. Full scan analysis was performed.</p></div><div id="S2.SS5.SSS3" class="sec"><p></p><h4 id="S2.SS5.SSS3title" class="inline">2.5.3 The optimum temperature and pH </h4><p class="p p-first">To investigate the effect of temperature on the degradation of 3-PBA by <em>sca</em>-engineered bacterial resting cells, 20, 25, 30, 35, 40, 45, 50, and 55&#x000b0;C were applied. Reaction system (1 ml): 900 &#x003bc;l of 0.1 mol/L PBS was used to resuspend the resting cells, and the volume of 3-PBA stock solution was 100 &#x003bc;l, to achieve a final concentration at 20 mg/L. After the oscillatory reaction was performed for 2 h in a 160 r/min water bath at the temperature gradient described above, acetonitrile was added to quench the reaction.</p><p class="p p-last">The resting cells were placed in 0.1 mol/L citrate buffer at pH 3.5&#x02013;6.0, phosphate buffer at pH 6.0&#x02013;8.0, Tris&#x02013;HCl buffer at pH 8.0&#x02013;10.0, and NaHCO<sub>3</sub>-NaOH buffer at pH 10.0&#x02013;13.0. The effect of varying pH on the degradation of 3-PBA in resting cells was investigated. For the reaction system (1 ml), 900 &#x003bc;l of buffer at different pH values was used to resuspend the resting cells, and 100 &#x003bc;l of 3-PBA stock solution was added, to achieve a final concentration at 20 mg/L. After the reactants were placed in a water bath at 160 r/min for 2 h, acetonitrile was added to stop the reaction.</p></div><div id="S2.SS5.SSS4" class="sec"><p></p><h4 id="S2.SS5.SSS4title" class="inline">2.5.4 Temperature and pH stability </h4><p class="p p-first">The temperature stability of 3-PBA degradation by resting cells was investigated. The resting cells were placed at 20, 30, 35, 40, and 50&#x000b0;C for 1 h and then subjected to a 35&#x000b0;C and 160 r/min water bath for an oscillating reaction for 2 h. The reaction was terminated using acetonitrile.</p><p class="p p-last">The pH stability of 3-PBA degradation by resting cells was investigated with different pH buffers (citrate buffer, 4.0&#x02013;5.0; phosphate buffer, 6.0&#x02013;8.0; Tris&#x02013;HCl buffer, 8.5&#x02013;10.0; and NaHCO<sub>3</sub>-NaOH, 11.0). The suspended resting cells were placed at 35&#x000b0;C for 1 h, after which 3-PBA stock solution was added to a water bath, to achieve a final concentration at 20 mg/L, at 35&#x000b0;C and 160 r/min for the oscillatory reaction. The samples were collected every 2 h, and acetonitrile was added to terminated he reaction.</p></div><div id="S2.SS5.SSS5" class="sec"><p></p><h4 id="S2.SS5.SSS5title" class="inline">2.5.5 The effect of metal ions </h4><p class="p p-first-last">The effect of different concentrations of metal ions on the degradation of 3-PBA in resting cells was investigated. The mother liquors of ZnSO<sub>4</sub>&#x022c5;7H<sub>2</sub>O, MnSO<sub>4</sub>&#x022c5;7H<sub>2</sub>O, MgSO<sub>4</sub>&#x022c5;H<sub>2</sub>O, CuSO<sub>4</sub>&#x022c5;5H<sub>2</sub>O, FeCl<sub>3</sub>, FeSO<sub>4</sub>&#x022c5;7H<sub>2</sub>O, NaCl, KCl, BaCl<sub>2</sub>, CaCl<sub>2</sub>&#x022c5;5H<sub>2</sub>O, CdCl<sub>2</sub>&#x022c5;2.5H<sub>2</sub>O, and Pb(CH<sub>3</sub>COO)<sub>2</sub>&#x022c5;3H<sub>2</sub>O were added to the reaction system, and the final concentrations of the metal ions were 1 and 5 mmol/L. After the reactants were oscillated in a water bath at 35&#x000b0;C and 160 r/min for 1 h, acetonitrile was added to efficiently quench the reaction.</p></div><div id="S2.SS5.SSS6" class="sec"><p></p><h4 id="S2.SS5.SSS6title" class="inline">2.5.6 The effect of chemical reagents </h4><p class="p p-first-last">Chemical solvents have the potential to perturb the conformation of enzymes, thereby impeding their catalytic activity. The anionic surfactant SDS, for instance, can disrupt the non-covalent interactions among enzyme molecules, thereby exerting an influence on enzyme activity. The higher polarity of organic solvents may impede the ingress of water molecules into the active center of the enzyme, thereby attenuating enzymatic activity (<a href="#B28" rid="B28" class=" bibr popnode">Sadana, 1988</a>). The effect of different concentrations of chemical reagents on the degradation of 3-PBA in resting cells was thus investigated. Methanol, ethyl alcohol, isopropyl alcohol, polyethylene glycol 400, acetone, n-hexane, and dimethyl sulfoxide were added to the reaction system, resulting in final concentrations of 1% and 5%, respectively. Urea, EDTA, SDS, DTT, and CTAB were added with final concentrations of 1 and 5 mmol/L, respectively. After the reactants were oscillated in a water bath at 35&#x000b0;C and 160 r/min for 1 h, acetonitrile was added to was effectively quench the reaction.</p></div><div id="S2.SS5.SSS7" class="sec"><p></p><h4 id="S2.SS5.SSS7title" class="inline">2.5.7 Determination of substrate tolerance </h4><p class="p p-first-last">To investigate the tolerance of resting cells to the substrate 3-PBA, 3-PBA was added to the reaction system at final concentrations of 10, 20, 30, 40, 50, 60, 70, and 80 mg/L. After the reactants were subjected to oscillation in a water bath at 35&#x000b0;C and 160 r/min for 2 h, acetonitrile was added to terminate the process.</p></div><div id="S2.SS5.SSS8" class="sec"><p></p><h4 id="S2.SS5.SSS8title" class="inline">2.5.8 The effect of cell concentration </h4><p class="p p-first-last">To investigate the effect of the resting cell concentration on the percent degradation of 3-PBA, 3-PBA stock solution was added to the different concentrations of resting cells, to achieve a final concentration at 20 mg/L. After the reactants were subjected to oscillation in a water bath at 35&#x000b0;C and 160 r/min for 2 h, acetonitrile was added to quench the reaction.</p></div><div id="S2.SS5.SSS9" class="sec sec-last"><p></p><h4 id="S2.SS5.SSS9title" class="inline">2.5.9 Substrate specificity determination </h4><p class="p p-first-last">The specificity of <em>sca</em>-engineered bacterial resting cells for the degradation of substrates was investigated. 4&#x02032;-HO-3-PBA, 3&#x02032;-HO-4-PBA, phenol, catechol, gallic acid, protocatechuic acid, and flusulfamide ether were added to the conversion reaction system at a final concentration of 20 mg/L. The reaction was quenched by the addition of acetonitrile after 2 h in a water bath at 35&#x000b0;C and 160 r/min. The HPLC detection conditions for flusulfamide ether were as follows: the mobile phase consisted of a mixture of acetonitrile and phosphoric acid water at pH 2.5 (65:35, v/v), the flow rate was set to 1 ml/min, the column temperature was maintained at 30&#x000b0;C, the detection wavelength was set to 290 nm, and an injection volume of 20 &#x003bc;l was used.</p></div></div><div id="S2.SS6" class="sec sec-last"><h3 id="S2.SS6title">2.6 Data analysis</h3><p class="p p-first">The percent degradation was calculated according to the following equation:</p><div class="disp-formula" id="S2.Ex1"><div class="f">
<math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M1" overflow="scroll"><mrow><mpadded width="+3.3pt"><mi>percent</mi></mpadded><mpadded width="+3.3pt"><mi>degradation</mi></mpadded><mrow><mo stretchy="false">(</mo><mo>%</mo><mo rspace="5.8pt" stretchy="false">)</mo></mrow><mo rspace="5.8pt">=</mo><mrow><mo stretchy="false">(</mo><mn>1</mn><mo rspace="5.8pt">-</mo><mi mathvariant="normal">C</mi><mo>/</mo><msub><mi mathvariant="normal">C</mi><mn>0</mn></msub><mo rspace="5.8pt" stretchy="false">)</mo></mrow><mo rspace="5.8pt">&#x000d7;</mo><mn>100</mn></mrow></math>
</div></div><p>where <em>C</em> represents the residual concentration of substrates in sample solutions (mg/L) and <em>C</em><sub>0</sub> is the initial concentration of substrate (mg/L).</p><p class="p p-last">The mean and standard deviation of the percent degradation in each trial were calculated and subjected to ANOVA. Statistical significance was determined using SPSS V22.0.</p></div></div><div id="S3" class="tsec sec"><h2 class="head no_bottom_margin" id="S3title">3 Results</h2><div id="S3.SS1" class="sec sec-first"><h3 id="S3.SS1title">3.1 Bioinformatics analysis of whole-genome sequences</h3><div id="S3.SS1.SSS1" class="sec sec-first"><p></p><h4 id="S3.SS1.SSS1title" class="inline">3.1.1 Quality control analysis of DNA and sequencing data </h4><p class="p p-first-last">The main DNA band was &#x0003e;10 kb, with no visible trailing, meeting the requirements for constructing sequencing databases (<a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Figure 3</a>). The quality control results of the Illumina sequencing data are shown in <a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Table 4</a> and <a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Figure 4</a>. The Q20 and Q30 values were 97.48% and 93.12%, respectively, and after quality control, they reached 98.22% and 94.15%, respectively. There were 85,064 reads and a base length of 1,144,377,555 bp, with the longest read length of 161,883 bp and an average read length of 13,453.14 bp based on the quality control data of the PacBio reads (<a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Figure 5</a>).</p></div><div id="S3.SS1.SSS2" class="sec"><p></p><h4 id="S3.SS1.SSS2title" class="inline">3.1.2 Genome assembly and evaluation results </h4><p class="p p-first-last">The SC-1 genome has 1 chromosome and 6 plasmids, with a total size of 4,374,142 bp and an average GC content of 63.53%. The chromosome size was 3,601,664 bp, and the GC content was 63.79%. The sizes of plasmids A&#x0223c;F were 349,403, 16,766, 97,252, 77,650, 65,982, and 12,840 bp, respectively, and the GC contents were 62.01%, 63.76%, 61.81%, 62.28%, 61.49%, and 60.90%, respectively (<a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Table 5</a>). The GC content is an important index of the genome of each species and can be used to judge whether a sample contains other genome contaminations. As shown in <a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Figure 6</a>, most of the GC-depth spots in the SC-1 genome were concentrated within a relatively narrow range, indicating that the sample DNA was not contaminated. K-mer analysis was used to select high-quality sequencing regions in the Illumina sequencing reads. The 17-mer was used as a parameter and followed a Poisson distribution with a single main peak, indicating that the sample DNA was not contaminated by other sources of DNA (<a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Figure 7</a>). The genomic information has been submitted to the National Center for Biotechnology Information (NCBI) database (Accession ID: <a href="/nucleotide/SAMN35671193" data-ga-action="click_feat_suppl" ref="reftype=extlink-DDBJ/EMBL/GenBank&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7CGenBank" target="_blank">SAMN35671193</a>).</p></div><div id="S3.SS1.SSS3" class="sec sec-last"><p></p><h4 id="S3.SS1.SSS3title" class="inline">3.1.3 Genome analysis and genome function annotation </h4><p class="p p-first-last">As shown in <a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Table 6</a>, 4,432 genetic sequences that represented 88.93% of the genome were predicted for functional annotation analysis, and 1,488 genes &#x0003e; 1,000 bp were identified (<a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Figure 8</a>). Regarding the RNA genes, there were 51 tRNAs (<a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Table 7</a>) and 6 rRNAs (2 16S rRNAs, 2 23S rRNAs, and 2 5S rRNAs). The distributions of protein-coding sequences (CDSs), RNA genes, COG functions, and GC content in the genome are shown in a genome circle map (<a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Figure 9</a>). Genome function annotation was demonstrated mainly through comparison with six biological databases, including Non-redundant protein database (NR), Swiss-Prot database, Pfam database, Gene Ontology database (GO), Clusters of Orthologous Groups of proteins database (COG), and Kyoto Encyclopedia of Genes and Genomes database (KEGG), as shown in <a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Figure 10</a> and <a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Table 8</a>.</p></div></div><div id="S3.SS2" class="sec"><h3 id="S3.SS2title">3.2 Screening and validation of dioxygenase genes</h3><p class="p p-first"><em>Sphingomonas</em> sp. can metabolize aromatic compounds and synthesize valuable extracellular polymers, which is highly valuable for research in many fields, such as environmental science and food and industrial production (<a href="#B21" rid="B21" class=" bibr popnode">Lee et al., 2001</a>; <a href="#B39" rid="B39" class=" bibr popnode">Yabuuchi et al., 2011</a>). Eighteen dioxygenase genes were screened based on the gene annotation results (<a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Table 9</a>); from these genes, two genes were obtained and designated <em>sca</em> (phenoxybenzoate dioxygenase subunit alpha, pC_gene0033) and <em>scb</em> (phenoxybenzoate dioxygenase subunit beta, pC_gene0035). Both genes were situated on plasmid C, and <em>sca</em> and <em>scb</em> were 1,230 and 960 bp in length, respectively. The encoded proteins Sca and Scb were compared with the &#x003b1; subunit (PobA, <a id="__tag_1146741522" class="tag_hotlink" href="/protein/CAA55400.1" ref="/protein/CAA55400.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"CAA55400.1","term_id":"473250","term_text":"CAA55400.1"}}</span>CAA55400.1</a>) and &#x003b2; subunit (PobB, <a id="__tag_1146741520" class="tag_hotlink" href="/protein/CAA55401.1" ref="/protein/CAA55401.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"CAA55401.1","term_id":"581466","term_text":"CAA55401.1"}}</span>CAA55401.1</a>) of 4-carboxyl diphenyl ether dioxygenase from <em>P. pseudoalcaligenes</em> POB310, respectively, and their amino acid identities were 96.33% and 99.06%, respectively (<a href="#B13" rid="B13" class=" bibr popnode">Dehmel et al., 1995</a>). Furthermore, as the phylogenetic tree shows, Sca and Scb were the closest relatives to PobA and PobB, respectively (<a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Figure 11</a>), and these proteins function as PobAB to break diphenyl ether bonds (<a href="#B13" rid="B13" class=" bibr popnode">Dehmel et al., 1995</a>).</p><p>The dioxygenase gene was cloned with DNA as a template, and a single bright band was generated (<a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Figure 12A</a>). Then, the PCR products were cloned and inserted into the pET28a(+) plasmid and expressed in <em>E. coli</em> BL21 (DE3). The results showed that the gene was successfully cloned and inserted into pET28a(+), and approximately 1,200 bp (<em>sca</em>) and 1,000 bp (<em>scb</em>) bands were observed by agarose gel electrophoresis (<a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Figures 12B, C</a>).</p><p>The engineered bacteria were designated <em>E. coli</em> BL21(DE3)-pET-28a(+)-<em>sca</em> (SCA) and <em>E. coli</em> BL21(DE3)-pET-28a(+)-<em>scb</em> (SCB), corresponding to <em>sca</em> and <em>scb</em>, respectively. The 20 mg/L 3-PBA was almost completely degraded by SCA within 24 h, while only 0.53% SCB degraded after 48 h (<a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Figure 12D</a>). Therefore, <em>sca</em> can be considered a crucial gene for 3-PBA degradation, as SCB had no effect on 3-PBA degradation.</p><p class="p p-last">Bioinformatics analysis was performed based on the whole-genome sequence and gene annotation results of strain SC-1, cloning and validation of dioxygenase genes were performed, and a crucial 3-PBA-degrading gene, <em>sca</em>, was identified. The length of <em>sca</em> is 1,230 bp, the <em>sca</em> gene encodes 409 amino acids, and the molecular mass of Sca is approximately 46.3 kDa. The percentages of &#x003b1;-helices, &#x003b2;-folds, and irregular curls in the secondary structure were 20.54%, 5.38%, and 54.03%, respectively (<a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Figure 13</a>). Sca has typical Rieske non-heme iron oxygenase (RO) family and phthalate 4,5-dioxygenase (PhDO) subfamily domains (<a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Figure 14</a>).</p></div><div id="S3.SS3" class="sec"><h3 id="S3.SS3title">3.3 Assessment of 3-PBA degradation activity across distinct cellular states</h3><p class="p p-first">The degradation of 3-PBA was detected using resting cells of genetically engineered bacteria and a recombinant crude enzyme solution (<a href="/pmc/articles/PMC11026547/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). After the resting cells were induced with IPTG for 16 h, the cells were obtained through a 0.1 mol/L PBS wash, and these cells could completely degrade 20 mg/L 3-PBA within 2 h. The percent degradation of 3-PBA in the whole-cell lysate of resting <em>sca</em> engineered bacteria, at an equivalent cell concentration, following ultrasonic disruption was only 6.25% after 2 h. Furthermore, the percent degradation of 3-PBA gradually increased with prolonged reaction time, reaching 28.98% and 36.17% at 6 and 24 h, respectively. The percent degradation of 3-PBA in the cell fragmentation supernatant and cell debris suspension obtained by centrifugation was less than 3% at 2 and 6 h, respectively. When the reaction time was extended to 24 h, the percent degradation increased to 11.88% and 9.69%, respectively. The percent degradation of 3-PBA in intact resting cells was significantly greater than that in whole-cell lysates, cell lysate supernatants, and cellular debris suspensions following ultrasonic disruption.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC11026547/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139905468194656"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fmicb-15-1361335-g001.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is fmicb-15-1361335-g001.jpg" src="/pmc/articles/PMC11026547/bin/fmicb-15-1361335-g001.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm139905468194656"><a target="object" rel="noopener" href="/pmc/articles/PMC11026547/figure/F1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC11026547/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">FIGURE 1</a></div><!--caption a7--><div class="caption"><p>Assessment of 3-PBA degradation activity across distinct cellular states.</p></div></div></div></div><div id="S3.SS4" class="sec sec-last"><h3 id="S3.SS4title">3.4 Characterizing 3-PBA degradation in resting cells</h3><div id="S3.SS4.SSS1" class="sec sec-first"><p></p><h4 id="S3.SS4.SSS1title" class="inline">3.4.1 Time-course degradation and metabolite identification of 3-PBA in resting cells </h4><p class="p p-first">The process of 3-PBA degradation in resting cells is shown in <a href="/pmc/articles/PMC11026547/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2A</span></a>. The resting cells exhibited a degradation capacity of approximately 50% for 3-PBA within 0.5 h, and complete degradation of 3-PBA was achieved within 2.5 h, demonstrating the robust degradation ability of 3-PBA. The degradation products were analyzed, as shown in <a href="/pmc/articles/PMC11026547/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figures 2B, C</span></a> and <a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Figure 15</a>. The peak area of 3-PBA (#1) gradually decreased with time, while a new substance (#2) was generated (<a href="/pmc/articles/PMC11026547/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2B</span></a>). Based on preliminary HPLC analysis, the degradation products of 3-PBA may be phenol (#2). The degradation of 3-PBA products by resting cells was further analyzed using GC-MS, and the resulting mass spectrum is presented in <a href="/pmc/articles/PMC11026547/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2C</span></a>. The NIST database was utilized for GC-MS analysis to identify the total ion flow, and the material with an ion peak at m/z 94 was successfully matched, leading to the identification of one degradation product as phenol.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC11026547/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139905468182688"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11026547_fmicb-15-1361335-g002.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fmicb-15-1361335-g002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11026547/bin/fmicb-15-1361335-g002.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139905468182688"><a target="object" rel="noopener" href="/pmc/articles/PMC11026547/figure/F2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC11026547/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">FIGURE 2</a></div><!--caption a7--><div class="caption"><p>Time-course degradation and metabolite identification of 3-PBA in resting cells. <strong>(A)</strong> Time-course degradation of 3-PBA in resting cells. <strong>(B)</strong> Characterization of 3-PBA degradation by the resting cell process of <em>sca</em>-engineered bacteria via HPLC. 1#, 3-PBA; #2, the term degradation product. <strong>(C)</strong> The mass spectra (GC-MS) of 3-PBA degradation by resting cells of <em>sca</em>-engineered bacteria.</p></div></div></div></div><div id="S3.SS4.SSS2" class="sec"><p></p><h4 id="S3.SS4.SSS2title" class="inline">3.4.2 The optimum temperature and pH </h4><p class="p p-first">To investigate the optimal reaction temperature for 3-PBA degradation in resting cells, the percent degradation of 3-PBA was measured at various temperatures. With increasing temperature, the resting cells completely reacted with the substrate 3-PBA, leading to a gradual increase in the percent degradation of 3-PBA. At 35&#x000b0;C, 3-PBA was almost completely degraded, and the percent degradation exceeded 50% within 30&#x02013;45&#x000b0;C. When the reaction temperature exceeded 35&#x000b0;C, the percent degradation of 3-PBA gradually decreased with increasing temperature. At a reaction temperature of 55&#x000b0;C, the degradation of 3-PBA by the resting cells was negligible. An increase in temperature likely triggers cellular apoptosis, ultimately resulting in a decrease or cessation of enzymatic activity.</p><p class="p">The percent degradation of 3-PBA by the resting cells under various pH conditions was investigated (<a href="/pmc/articles/PMC11026547/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figures 3A, B</span></a>). The optimal pH for 3-PBA degradation by the resting cells was 8.5, resulting in a percent degradation of 67.3%. The percent degradation of 3-PBA was significantly greater than 40% at pH values ranging from 7.0 to 10.0, indicating that the enzyme is robust at degrading 3-PBA under neutral or alkaline conditions. However, at pH values below 6.0, the percent degradation of 3-PBA decreased to less than 20%, while a sharp decrease in percent degradation occurred when the pH exceeded 10.0. The deviation from the optimal pH due to overacidic or overalkaline conditions may adversely affect cellular viability, leading to cell death and altering the dissociation state of enzyme active groups; these effects lead to irreversible enzyme degeneration and loss of 3-PBA degradation activity.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC11026547/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139905463894048"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11026547_fmicb-15-1361335-g003.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fmicb-15-1361335-g003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11026547/bin/fmicb-15-1361335-g003.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139905463894048"><a target="object" rel="noopener" href="/pmc/articles/PMC11026547/figure/F3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC11026547/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">FIGURE 3</a></div><!--caption a7--><div class="caption"><p>Characterization of 3-PBA degradation by resting cells of <em>sca</em>-engineered bacteria. <strong>(A)</strong> The optimum temperature. <strong>(B)</strong> The optimum pH. <strong>(C)</strong> The temperature stability. <strong>(D)</strong> pH stability. <strong>(E)</strong> The substrate tolerance concentrations. <strong>(F)</strong> The impact of cell concentration on 3-PBA degradation.</p></div></div></div></div><div id="S3.SS4.SSS3" class="sec"><p></p><h4 id="S3.SS4.SSS3title" class="inline">3.4.3 Temperature and pH stability </h4><p class="p p-first">The percent degradation of 3-PBA in resting cells remained above 50% even after the cells were exposed to temperatures between 4 and 35&#x000b0;C for 1 h, as depicted in <a href="/pmc/articles/PMC11026547/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figures 3C, D</span></a>, indicating remarkable stability within this temperature range. When the temperature exceeded 25&#x000b0;C, the percent degradation of 3-PBA gradually decreased with increasing holding temperature. Following a 1-h incubation at 40&#x000b0;C, the percent degradation of 3-PBA in quiescent cells decreased to approximately 8%. The enzyme activity significantly deteriorated at or above 40&#x000b0;C.</p><p class="p p-last">In terms of pH stability, the percent degradation of 3-PBA in resting cells significantly decreased (to less than 10%) following a 1-h incubation at pH values below 5.0 or above 9.0 at a constant temperature of 35&#x000b0;C. The percent degradation of 3-PBA exceeded 25% within 1 h in the buffer solution at 35&#x000b0;C and pH values between 5.0 and 9.0, indicating that the enzyme tolerates this pH range.</p></div><div id="S3.SS4.SSS4" class="sec"><p></p><h4 id="S3.SS4.SSS4title" class="inline">3.4.4 The effect of metal ions </h4><p class="p p-first">The impact of metal ions on the degradation of 3-PBA in resting cells is presented in <a href="/pmc/articles/PMC11026547/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. Compared to that of the control group without metal ions, the percent degradation of 3-PBA was significantly reduced in the presence of 1 mmol/L Cu<sup>2+</sup>, Mn<sup>2+</sup>, Zn<sup>2+</sup>, and Cd<sup>2+</sup>, which effectively inhibited the degradation of 3-PBA in resting cells. However, at a concentration of 1 mmol/L, Na<sup>+</sup>, K<sup>+</sup>, Ca<sup>2+</sup>, Mg<sup>2+</sup>, Pb<sup>2+</sup>, Fe<sup>2+</sup>, Fe<sup>3+</sup>, and Ba<sup>2+</sup> did not have any significant effect on the percent degradation of 3-PBA, while Pb<sup>2+</sup> and Fe<sup>2+</sup> at a concentration of 5 mmol/L significantly inhibited its percent degradation.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T1"><h3>TABLE 1</h3><!--caption a7--><div class="caption"><p>The impact of metal ions on the degradation of 3-PBA in resting cells.</p></div><div class="xtable"><table frame="box" rules="all" cellspacing="5" cellpadding="5" class="rendered small default_table"><thead><tr><td valign="top" align="left" style="color:#ffffff;background-color: #7f8080;" rowspan="1" colspan="1">Metal ion</td><td valign="top" align="center" colspan="2" style="color:#ffffff;background-color: #7f8080;" rowspan="1">The rate of 3-PBA degradation/%</td></tr></thead><tbody><tr><td valign="top" align="left" style="color:#ffffff;background-color: #7f8080;" rowspan="1" colspan="1"></td><td valign="top" align="center" style="color:#ffffff;background-color: #7f8080;" rowspan="1" colspan="1">
<strong>1 mmol/L</strong>
</td><td valign="top" align="center" style="color:#ffffff;background-color: #7f8080;" rowspan="1" colspan="1">
<strong>5 mmol/L</strong>
</td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">Control</td><td valign="top" align="center" rowspan="1" colspan="1">54.80 &#x000b1; 3.85<sup>a</sup></td><td valign="top" align="center" rowspan="1" colspan="1">54.80 &#x000b1; 3.85<sup>ab</sup></td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">Na<sup>+</sup></td><td valign="top" align="center" rowspan="1" colspan="1">54.97 &#x000b1; 0.01<sup>a</sup></td><td valign="top" align="center" rowspan="1" colspan="1">55.48 &#x000b1; 1.00<sup>ab</sup></td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">K<sup>+</sup></td><td valign="top" align="center" rowspan="1" colspan="1">54.92 &#x000b1; 0.05<sup>a</sup></td><td valign="top" align="center" rowspan="1" colspan="1">58.54 &#x000b1; 2.78<sup>a</sup></td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">Ca<sup>2+</sup></td><td valign="top" align="center" rowspan="1" colspan="1">54.53 &#x000b1; 0.12<sup>a</sup></td><td valign="top" align="center" rowspan="1" colspan="1">50.57 &#x000b1; 2.96<sup>b</sup></td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">Cu<sup>2+</sup></td><td valign="top" align="center" rowspan="1" colspan="1">5.65 &#x000b1; 0.41<sup>cd</sup></td><td valign="top" align="center" rowspan="1" colspan="1">&#x02013;</td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">Mg<sup>2+</sup></td><td valign="top" align="center" rowspan="1" colspan="1">54.98 &#x000b1; 0.00<sup>a</sup></td><td valign="top" align="center" rowspan="1" colspan="1">58.57 &#x000b1; 2.52<sup>a</sup></td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">Mn<sup>2+</sup></td><td valign="top" align="center" rowspan="1" colspan="1">35.40 &#x000b1; 0.36<sup>b</sup></td><td valign="top" align="center" rowspan="1" colspan="1">&#x02013;</td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">Pb<sup>2+</sup></td><td valign="top" align="center" rowspan="1" colspan="1">54.97 &#x000b1; 0.02<sup>a</sup></td><td valign="top" align="center" rowspan="1" colspan="1">21.61 &#x000b1; 2.96<sup>d</sup></td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">Zn<sup>2+</sup></td><td valign="top" align="center" rowspan="1" colspan="1">7.44 &#x000b1; 0.25<sup>c</sup></td><td valign="top" align="center" rowspan="1" colspan="1">&#x02013;</td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">Fe<sup>2+</sup></td><td valign="top" align="center" rowspan="1" colspan="1">54.98 &#x000b1; 0.00<sup>a</sup></td><td valign="top" align="center" rowspan="1" colspan="1">38.53 &#x000b1; 1.25<sup>c</sup></td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">Fe<sup>3+</sup></td><td valign="top" align="center" rowspan="1" colspan="1">54.98 &#x000b1; 0.01<sup>a</sup></td><td valign="top" align="center" rowspan="1" colspan="1">51.25 &#x000b1; 2.33<sup>b</sup></td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">Cd<sup>2+</sup></td><td valign="top" align="center" rowspan="1" colspan="1">3.93 &#x000b1; 0.14<sup>d</sup></td><td valign="top" align="center" rowspan="1" colspan="1">&#x02013;</td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">Ba<sup>2+</sup></td><td valign="top" align="center" rowspan="1" colspan="1">54.98 &#x000b1; 0.00<sup>a</sup></td><td valign="top" align="center" rowspan="1" colspan="1">56.62 &#x000b1; 2.04<sup>ab</sup></td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139905463880784"><a target="object" rel="noopener" href="/pmc/articles/PMC11026547/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="fn-a.k.c.e.e.c.d.a"><p class="p p-first-last">The identical lowercase letters in the same column denote negligible differences (<em>P</em> &#x0003e; 0.05), and the employment of distinct lowercase letters signifies notable differences (<em>P</em> &#x0003c; 0.05).</p></div></div></div></div><div id="S3.SS4.SSS5" class="sec"><p></p><h4 id="S3.SS4.SSS5title" class="inline">3.4.5 The effects of chemical reagents </h4><p class="p p-first">The percent degradation of 3-PBA was determined by adding chemical reagents at varying concentrations to the degradation reaction system. The results are presented in <a href="/pmc/articles/PMC11026547/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Tables 2</span></a>, <a href="/pmc/articles/PMC11026547/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,3.</span></span><span>3</span></a>. Compared to the control group, the presence of 1 mmol/L SDS, CTAB, 1% ethyl alcohol, isopropyl alcohol, acetone, and n-hexane resulted in a significant decrease in 3-PBA in the resting cells. The percent degradation of 3-PBA was not significantly affected by the presence of 1 mmol/L DTT, EDTA, urea, 1% methanol, polyethylene glycol, or dimethyl sulfoxide. However, 3-PBA was inhibited when 5% methanol, polyethylene glycol, ethyl alcohol, and dimethyl sulfoxide were added.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T2"><h3>TABLE 2</h3><!--caption a7--><div class="caption"><p>The impact of chemical reagents on the degradation of 3-PBA in resting cells.</p></div><div class="xtable"><table frame="box" rules="all" cellspacing="5" cellpadding="5" class="rendered small default_table"><thead><tr><td valign="top" align="left" style="color:#ffffff;background-color: #7f8080;" rowspan="1" colspan="1">Chemical reagent</td><td valign="top" align="center" colspan="2" style="color:#ffffff;background-color: #7f8080;" rowspan="1">The rate of 3-PBA degradation/%</td></tr></thead><tbody><tr><td valign="top" align="left" style="color:#ffffff;background-color: #7f8080;" rowspan="1" colspan="1"></td><td valign="top" align="center" style="color:#ffffff;background-color: #7f8080;" rowspan="1" colspan="1">
<strong>1 mmol/L</strong>
</td><td valign="top" align="center" style="color:#ffffff;background-color: #7f8080;" rowspan="1" colspan="1">
<strong>5 mmol/L</strong>
</td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">Control</td><td valign="top" align="center" rowspan="1" colspan="1">54.80 &#x000b1; 3.85<sup>a</sup></td><td valign="top" align="center" rowspan="1" colspan="1">54.80 &#x000b1; 3.85<sup>a</sup></td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">SDS</td><td valign="top" align="center" rowspan="1" colspan="1">39.49 &#x000b1; 0.48<sup>b</sup></td><td valign="top" align="center" rowspan="1" colspan="1">&#x02013;</td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">DTT</td><td valign="top" align="center" rowspan="1" colspan="1">54.04 &#x000b1; 0.81<sup>a</sup></td><td valign="top" align="center" rowspan="1" colspan="1">53.98 &#x000b1; 0.30<sup>a</sup></td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">CTAB</td><td valign="top" align="center" rowspan="1" colspan="1">31.66 &#x000b1; 0.71<sup>c</sup></td><td valign="top" align="center" rowspan="1" colspan="1">&#x02013;</td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">EDTA</td><td valign="top" align="center" rowspan="1" colspan="1">54.50 &#x000b1; 0.28<sup>a</sup></td><td valign="top" align="center" rowspan="1" colspan="1">55.82 &#x000b1; 2.76<sup>a</sup></td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">Urea</td><td valign="top" align="center" rowspan="1" colspan="1">54.79 &#x000b1; 0.06<sup>a</sup></td><td valign="top" align="center" rowspan="1" colspan="1">57.94 &#x000b1; 0.74<sup>a</sup></td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139905471457232"><a target="object" rel="noopener" href="/pmc/articles/PMC11026547/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="fn-a.k.c.e.f.c.d.a"><p class="p p-first-last">The identical lowercase letters in the same column denote negligible differences (<em>P</em> &#x0003e; 0.05), and the employment of distinct lowercase letters signifies notable differences (<em>P</em> &#x0003c; 0.05).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T3"><h3>TABLE 3</h3><!--caption a7--><div class="caption"><p>The impact of organic solvents on the degradation of 3-PBA in resting cells.</p></div><div class="xtable"><table frame="box" rules="all" cellspacing="5" cellpadding="5" class="rendered small default_table"><thead><tr><td valign="top" align="left" style="color:#ffffff;background-color: #7f8080;" rowspan="1" colspan="1">Organic solvent</td><td valign="top" align="center" colspan="2" style="color:#ffffff;background-color: #7f8080;" rowspan="1">The rate of 3-PBA degradation/%</td></tr></thead><tbody><tr><td valign="top" align="left" style="color:#ffffff;background-color: #7f8080;" rowspan="1" colspan="1"></td><td valign="top" align="center" style="color:#ffffff;background-color: #7f8080;" rowspan="1" colspan="1">
<strong>1%</strong>
</td><td valign="top" align="center" style="color:#ffffff;background-color: #7f8080;" rowspan="1" colspan="1">
<strong>5%</strong>
</td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">Control</td><td valign="top" align="center" rowspan="1" colspan="1">54.80 &#x000b1; 3.85<sup>a</sup></td><td valign="top" align="center" rowspan="1" colspan="1">54.80 &#x000b1; 3.85<sup>ab</sup></td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">Methanol</td><td valign="top" align="center" rowspan="1" colspan="1">54.63 &#x000b1; 0.23<sup>ab</sup></td><td valign="top" align="center" rowspan="1" colspan="1">45.51 &#x000b1; 0.57<sup>c</sup></td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">Ethyl alcohol</td><td valign="top" align="center" rowspan="1" colspan="1">51.61 &#x000b1; 0.58<sup>ab</sup></td><td valign="top" align="center" rowspan="1" colspan="1">34.52 &#x000b1; 0.03<sup>d</sup></td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">Isopropyl alcohol</td><td valign="top" align="center" rowspan="1" colspan="1">34.27 &#x000b1; 0.93<sup>d</sup></td><td valign="top" align="center" rowspan="1" colspan="1">&#x02013;</td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">Polyethylene glycol</td><td valign="top" align="center" rowspan="1" colspan="1">54.30 &#x000b1; 0.20<sup>ab</sup></td><td valign="top" align="center" rowspan="1" colspan="1">49.98 &#x000b1; 1.19<sup>bc</sup></td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">Acetone</td><td valign="top" align="center" rowspan="1" colspan="1">38.67 &#x000b1; 0.91<sup>c</sup></td><td valign="top" align="center" rowspan="1" colspan="1">&#x02013;</td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">Dimethyl sulfoxide</td><td valign="top" align="center" rowspan="1" colspan="1">54.78 &#x000b1; 0.06<sup>a</sup></td><td valign="top" align="center" rowspan="1" colspan="1">47.32 &#x000b1; 0.31<sup>c</sup></td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">N-hexane</td><td valign="top" align="center" rowspan="1" colspan="1">50.82 &#x000b1; 0.10<sup>b</sup></td><td valign="top" align="center" rowspan="1" colspan="1">57.39 &#x000b1; 2.32<sup>a</sup></td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139905465021008"><a target="object" rel="noopener" href="/pmc/articles/PMC11026547/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="fn-a.k.c.e.f.d.d.a"><p class="p p-first-last">The identical lowercase letters in the same column denote negligible differences (<em>P</em> &#x0003e; 0.05), and the employment of distinct lowercase letters signifies notable differences (<em>P</em> &#x0003c; 0.05).</p></div></div></div></div><div id="S3.SS4.SSS6" class="sec"><p></p><h4 id="S3.SS4.SSS6title" class="inline">3.4.6 Determination of substrate tolerance </h4><p class="p p-first-last">Varying concentrations of 3-PBA were added to the conversion reaction system to investigate the substrate tolerance of the resting cells (<a href="/pmc/articles/PMC11026547/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3E</span></a>). When the concentration of 3-PBA ranged from 10 to 80 mg/L, the degradation capability of all resting cells was proficient. Moreover, at a concentration of 10 mg/L, 3-PBA was completely degraded. With increasing 3-PBA concentration, the percent degradation gradually decreased until a steady state was reached. A high concentration of 3-PBA may cause cytotoxicity in cells.</p></div><div id="S3.SS4.SSS7" class="sec"><p></p><h4 id="S3.SS4.SSS7title" class="inline">3.4.7 The effect of cell concentration </h4><p class="p p-first-last">The impact of the resting cell concentration on the degradation of 3-PBA was investigated (<a href="/pmc/articles/PMC11026547/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3F</span></a>). At a cell concentration of 3.3 &#x000d7; 10<sup>9</sup>CFU/ml, the percent degradation of 3-PBA was only 7%. As the resting cell concentration increased, the percent degradation gradually increased. At a concentration of 4.9 &#x000d7; 10<sup>10</sup> CFU/ml, nearly complete degradation of 3-PBA was achieved. The higher the cell concentration was, the greater the enzyme content in the reaction system, leading to an increase in the percent degradation of 3-PBA.</p></div><div id="S3.SS4.SSS8" class="sec sec-last"><p></p><h4 id="S3.SS4.SSS8title" class="inline">3.4.8 Substrate specificity determination </h4><p class="p p-first-last">The degradation of various substrates by quiescent cells was investigated, as depicted in <a href="#DS1" rid="DS1" class=" supplementary-material">Supplementary Table 10</a>. Resting cells degraded 4&#x02032;-HO-3-PBA and 3&#x02032;-HO-4-PBA but did not degrade the monophenyl compounds that were possibly generated during the degradation of 3-PBA. The structures of HO-3-PBA and 3-PBA are similar and both feature diphenyl ether bonds, suggesting that the enzymatic function of the <em>sca</em> protein is to break these bonds.</p></div></div></div><div id="S4" class="tsec sec"><h2 class="head no_bottom_margin" id="S4title">4 Discussion</h2><p class="p p-first">The primary metabolite in the biodegradation pathway of PYRs is 3-PBA, which is extensively detected in food, human urine, and the environment. In this study, we performed whole-genome sequencing of <em>Sphingomonas baderi</em> SC-1 and identified the vital 3-PBA-degrading gene <em>sca</em> (1230 bp). After bacteria were engineered to express <em>sca</em>, a concentration of 20 mg/L 3-PBA was completely degraded within 24 h, yielding phenol as one of degradation products. This finding corresponds with the annotation of <em>sca</em> as a 3-PBA dioxygenase based on whole-genome sequencing of <em>Sphingomonas baderi</em> SC-1, indicating that the experimental results and genomic evidence are consistent. Previous studies have established that ortho-dioxygenases are Rieske non-heme iron enzymes involved in the hydroxylation of aromatic rings (known as ring-hydroxylating oxygenases, RHOs) (<a href="#B7" rid="B7" class=" bibr popnode">Chakraborty et al., 2012</a>). The subjects were categorized into five distinct classifications (<a href="#B20" rid="B20" class=" bibr popnode">Kweon et al., 2008</a>). The 3-PBA-3,4-angular dioxygenase PobAB from <em>P. pseudoalcaligenes</em> POB310 is a type I RHO with two open reading frames (ORFs), <em>pobA</em> and <em>pobB</em>, which encode oxygenase and FNRC-type reductase subunits, respectively. Except for the presence of FNRN-type reductase, type II RHOs exhibit similarities to Type I RHOs. The Type III variant is characterized by the presence of an FNRN-type reductase and the absence of a [2Fe-2S]-type ferredoxin subunit. 3-PBA-1&#x02032;,2&#x02032;-Angular dioxygenase, known as PbaA1A2BC from <em>S. wenxiniae</em> JZ-1, is a component of Type IV RHOs and comprises the following ORFs: <em>pbaA1</em>, <em>pbaA2</em>, <em>pbaB</em>, and <em>pbaC</em>. Among these, <em>pbaA</em> and <em>pbaB</em> are responsible for the oxygenase activity and [2Fe-2S]-type ferredoxin subunit, respectively.</p><p>Additionally, the coexpression of <em>pbaAB</em> with <em>pbaC</em>, which corresponds to glutathione reductase (GR), is needed. The Type V system incorporates a distinct [3Fe-4S]-type ferredoxin, which distinguishes it from Type IV. In general, the reduction equivalents necessary for RHO-related catalytic reactions are provided by NAD(P)H-dependent reductase proteins through the electron transport chain. However, this approach involves a significant challenge, as <em>in vitro</em> activity is insufficient. Few efficient methods have been proposed for the screening and identification of 3-PBA oxygenase due to its unique characteristics; 3-PBA oxygenase primarily occurs as membrane proteins or multisubunit proteins, imposing limitations on its separation and purification (<a href="#B45" rid="B45" class=" bibr popnode">Zhao et al., 2021</a>). Currently, two 3-PBA ortho-dioxygenase genes, namely, <em>pobAB</em> (<a href="#B13" rid="B13" class=" bibr popnode">Dehmel et al., 1995</a>) and <em>pbaA1A2B</em> (<a href="#B36" rid="B36" class=" bibr popnode">Wang et al., 2014</a>), have been successfully cloned. Both genes were obtained using deletion mutants and subsequently confirmed via genetic recall, employing different aromatic rings. Among them, PobAB enzymatically catalyzes the conversion of the carboxyl aromatic ring of 3-PBA to phenol and protocatechuic acid, which corresponds with the findings of this study. This process involves the introduction of two hydroxyl groups at the 3- and 4-positions of the benzoic acid moiety, followed by cleavage of the diphenyl ether bond, which is facilitated by ortho-dioxygenase (<a href="#B45" rid="B45" class=" bibr popnode">Zhao et al., 2021</a>). The enzyme PbaA1A2BC facilitates the conversion of the non-carboxyl aromatic ring to 3-hydroxybenzoic acid and catechol, aligning with the pathway that involves 3-PBA metabolism by <em>S. wenxiniae</em> JZ-1. These findings suggest that ortho-dioxygenase cleaves the diphenyl ether bond at positions 1&#x02032; and 2&#x02032; of the aromatic moiety (<a href="#B45" rid="B45" class=" bibr popnode">Zhao et al., 2021</a>). The degradation of diphenyl ether structure analogs may be associated with lignin oxidase and cytochrome P450 monooxygenases (CYPs) (<a href="#B38" rid="B38" class=" bibr popnode">White et al., 1996</a>). Lignin oxidase primarily comprises lignin peroxidase, manganese peroxidase, and laccase. Filamentous fungi, which are abundant in these enzymes, serve as the predominant microbial degraders of diphenyl ether derivatives (<a href="#B12" rid="B12" class=" bibr popnode">Dashtban et al., 2009</a>). CYPs, which are widely distributed across all biological kingdoms, are thiolate heme proteins that primarily function as terminal monooxygenases in a variety of biochemical reactions, such as hydroxylation, by catalyzing diverse oxidative transformations of various molecules (<a href="#B25" rid="B25" class=" bibr popnode">Nelson, 2009</a>). After determining the enzymatic activity of CYP, <a href="#B42" rid="B42" class=" bibr popnode">Zhao et al. (2020)</a> concluded that CYP played a significant role in cleaving the 3-PBA diphenyl ether bond by <em>Aspergillus oryzae</em> M-4; thus, CYP facilitates the complete degradation of downstream products through synergistic action with laccase and lignin peroxidase. Transcriptome analysis of <em>A. oryzae</em> M-4 also revealed that 3-PBA induced the upregulation of genes associated with energy expenditure, the oxidative stress response, membrane transport, and DNA repair. In-depth functional interpretation of differentially expressed genes suggested that the involvement of CYP450 may contribute to the hydroxylation of 3-PBA (<a href="#B16" rid="B16" class=" bibr popnode">Hu et al., 2023</a>).</p><p class="p p-last">Additionally, we observed a fascinating phenomenon: the percentage degradation of 3-PBA in the cellular lysate of the <em>sca</em>-engineered bacteria was exceptionally low. Notably, maintaining complete cellular integrity was crucial for the efficient degradation of 3-PBA. This relatively distinctive phenomenon was scrutinized via subsequent analysis in the current investigation. The Sca protein expressed in prokaryotes is presumably a membrane protein. The adsorption and entry of Sca into cells are aided by the presence of living cells, while its degradation depends on the integrity of membrane proteins. However, after cells were disrupted by ultrasound, the exposure of membrane proteins to a hydrophilic environment promoted the aggregation of proteins in the cell debris suspension due to their hydrophobic nature, leading to their inactivation (<a href="#B41" rid="B41" class=" bibr popnode">Zhang et al., 2021</a>). The CellPLoc package was utilized to predict the subcellular localization of the dioxygenase protein Sca, which is encoded by the 3-PBA-degrading gene <em>sca</em>. This protein predominantly localizes to the cell membrane. Another plausible hypothesis is that Sca functions as a dioxygenase belonging to the oxidoreductase class and aids in the degradation of 3-PBA by utilizing coenzymes, cofactors, and cogroups for efficient electron provision and transfer (<a href="#B42" rid="B42" class=" bibr popnode">Zhao et al., 2020</a>, <a href="#B43" rid="B43" class=" bibr popnode">2022</a>). This finding highlights a promising avenue for future investigations, in which the dioxygenase gene from <em>Sphingomonas baderi</em> was successfully isolated and characterized; in addition, the results revealed the potential of this gene for efficient degradation of 3-PBA.</p></div><div id="S5" class="tsec sec"><h2 class="head no_bottom_margin" id="S5title">5 Conclusion</h2><p class="p p-first-last">The crucial 3-PBA-degrading gene <em>sca</em> (1,230 bp) was identified through whole-genome sequencing of <em>Sphingomonas</em> sp. SC-1. The degradation properties of 3-PBA were investigated using resting cells of genetically engineered bacteria harboring the <em>sca</em> gene. Phenol was identified as one of degradation byproducts. In addition to their ability to degrade 3-PBA, these resting cells demonstrated proficiency in degrading 4&#x02032;-HO-3-PBA and 3&#x02032;-HO-4-PBA.</p></div><div id="S6" class="tsec sec"><h2 class="head no_bottom_margin" id="S6title">Data availability statement</h2><p class="p p-first-last">The datasets presented in this study can be found in online repositories. The names of the repository/repositories and accession number(s) can be found below: NCBI &#x02013; <a href="/nucleotide/PRJNA981288" data-ga-action="click_feat_suppl" ref="reftype=extlink-DDBJ/EMBL/GenBank&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7CGenBank" target="_blank">PRJNA981288</a>.</p></div><div id="S7" class="tsec sec"><h2 class="head no_bottom_margin" id="S7title">Author contributions</h2><p class="p p-first-last">QL: Conceptualization, Formal analysis, Supervision, Writing &#x02013; original draft. QZ: Data curation, Formal analysis, Investigation, Writing &#x02013; original draft. YC: Data curation, Formal analysis, Investigation, Writing &#x02013; original draft. KH: Formal analysis, Investigation, Methodology, Writing &#x02013; review &#x00026; editing. MS: Formal analysis, Investigation, Methodology, Writing &#x02013; review &#x00026; editing. JL: Formal analysis, Methodology, Writing &#x02013; review &#x00026; editing. AL: Formal analysis, Methodology, Writing &#x02013; review &#x00026; editing. LZ: Methodology, Resources, Writing &#x02013; review &#x00026; editing. SL: Conceptualization, Funding acquisition, Project administration, Supervision, Writing &#x02013; review &#x00026; editing.</p></div><div id="funding-group-a.j.b.q" class="tsec sec"><h2 class="head no_bottom_margin" id="funding-group-a.j.b.qtitle">Funding Statement</h2><p>The author(s) declare financial support was received for the research, authorship, and/or publication of this article. This work was supported by the financial support from the National Natural Science Foundation of China (Grant No. 31871770) and the Science and Technology Department of Sichuan Province (Grant No. 2022NSFSC1739).</p></div><div id="fn-group-a.l.a" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.l.atitle">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="footnote1"><sup>1</sup>
<a href="http://www.majorbio.com" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.majorbio.com</a>
</p></div></div><div id="S9" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="S9title">Conflict of interest</h2><!--/article/back/sec/--><p class="p p-first-last">The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p></div><div id="S10" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="S10title">Publisher&#x02019;s note</h2><!--/article/back/sec/--><p class="p p-first-last">All claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.</p></div><div id="S11" class="tsec bk-sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="S11title">Supplementary material</h2><!--/article/back/sec/--><p class="p p-first">The Supplementary Material for this article can be found online at: <a href="https://www.frontiersin.org/articles/10.3389/fmicb.2024.1361335/full#supplementary-material" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.frontiersin.org/articles/10.3389/fmicb.2024.1361335/full#supplementary-material</a></p><div class="sec suppmat" id="DS1"><div class="sup-box half_rhythm" id="media-a.l.d.c.a"><a href="/pmc/articles/PMC11026547/bin/Data_Sheet_1.DOCX" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(3.1M, DOCX)</sup></div></div></div><div id="ref-list-a.l.e" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.l.etitle">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="B1"><span class="mixed-citation">Barr D., Olsson A., Wong L., Udunka S., Baker S., Whitehead R., et al. (2010). <span class="ref-title">Urinary concentrations of metabolites of pyrethroid insecticides in the general U.S. population: National Health and Nutrition Examination Survey 1999-2002.</span>
<span class="ref-journal"><em>Environ. Health Perspect.</em></span>
<span class="ref-vol">6</span>:<span class="ref-iss">118</span>. 10.1289/ehp.0901275

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2898848/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20129874" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1289%2Fehp.0901275" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Environ.+Health+Perspect.&amp;title=Urinary+concentrations+of+metabolites+of+pyrethroid+insecticides+in+the+general+U.S.+population:+National+Health+and+Nutrition+Examination+Survey+1999-2002.&amp;author=D.+Barr&amp;author=A.+Olsson&amp;author=L.+Wong&amp;author=S.+Udunka&amp;author=S.+Baker&amp;volume=6&amp;issue=118&amp;publication_year=2010&amp;pmid=20129874&amp;doi=10.1289/ehp.0901275&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B2"><span class="mixed-citation">Besemer J., Borodovsky M. (2005). <span class="ref-title">GeneMark: Web software for gene finding in prokaryotes, eukaryotes and viruses.</span>
<span class="ref-journal"><em>Nucleic Acids Res.</em></span>
<span class="ref-vol">33</span>
W451&#x02013;W454. 10.1093/nar/gki487

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1160247/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15980510" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgki487" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=GeneMark:+Web+software+for+gene+finding+in+prokaryotes,+eukaryotes+and+viruses.&amp;author=J.+Besemer&amp;author=M.+Borodovsky&amp;volume=33&amp;publication_year=2005&amp;pages=W451-W454&amp;pmid=15980510&amp;doi=10.1093/nar/gki487&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B3"><span class="mixed-citation">Birolli W., Alvarenga N., Seleghim M., Porto A. (2016a). <span class="ref-title">Biodegradation of the pyrethroid pesticide esfenvalerate by marine-derived fungi.</span>
<span class="ref-journal"><em>Mar. Biotechnol.</em></span>
<span class="ref-vol">18</span>
511&#x02013;520. 10.1007/s10126-016-9710-z

 [<a href="https://pubmed.ncbi.nlm.nih.gov/27381569" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs10126-016-9710-z" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mar.+Biotechnol.&amp;title=Biodegradation+of+the+pyrethroid+pesticide+esfenvalerate+by+marine-derived+fungi.&amp;author=W.+Birolli&amp;author=N.+Alvarenga&amp;author=M.+Seleghim&amp;author=A.+Porto&amp;volume=18&amp;publication_year=2016a&amp;pages=511-520&amp;pmid=27381569&amp;doi=10.1007/s10126-016-9710-z&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B4"><span class="mixed-citation">Birolli W., Borges E., Nitschke M., Rom&#x000e3;o L., Porto A. (2016b). <span class="ref-title">Biodegradation pathway of the pyrethroid pesticide esfenvalerate by Bacteria from different biomes.</span>
<span class="ref-journal"><em>Water Air Soil Pollut.</em></span>
<span class="ref-vol">227</span>:<span class="ref-iss">271</span>. 10.1007/s11270-016-2968-y [<a href="//doi.org/10.1007%2Fs11270-016-2968-y" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Water+Air+Soil+Pollut.&amp;title=Biodegradation+pathway+of+the+pyrethroid+pesticide+esfenvalerate+by+Bacteria+from+different+biomes.&amp;author=W.+Birolli&amp;author=E.+Borges&amp;author=M.+Nitschke&amp;author=L.+Rom&#x000e3;o&amp;author=A.+Porto&amp;volume=227&amp;issue=271&amp;publication_year=2016b&amp;doi=10.1007/s11270-016-2968-y&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B5"><span class="mixed-citation">Bolger A. M., Lohse M., Usadel B. (2014). <span class="ref-title">Trimmomatic: A flexible trimmer for Illumina sequence data.</span>
<span class="ref-journal"><em>Bioinformatics</em></span>
<span class="ref-vol">30</span>
2114&#x02013;2120. 10.1093/bioinformatics/btu170

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4103590/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24695404" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtu170" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Trimmomatic:+A+flexible+trimmer+for+Illumina+sequence+data.&amp;author=A.+M.+Bolger&amp;author=M.+Lohse&amp;author=B.+Usadel&amp;volume=30&amp;publication_year=2014&amp;pages=2114-2120&amp;pmid=24695404&amp;doi=10.1093/bioinformatics/btu170&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B6"><span class="mixed-citation">Borodovsky M., McIninch J. (1993). <span class="ref-title">GENMARK: Parallel gene recognition for both DNA strands.</span>
<span class="ref-journal"><em>Comput. Chem.</em></span>
<span class="ref-vol">17</span>
123&#x02013;133. 10.1016/0097-8485(93)85004-V [<a href="//doi.org/10.1016%2F0097-8485(93)85004-V" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Comput.+Chem.&amp;title=GENMARK:+Parallel+gene+recognition+for+both+DNA+strands.&amp;author=M.+Borodovsky&amp;author=J.+McIninch&amp;volume=17&amp;publication_year=1993&amp;pages=123-133&amp;doi=10.1016/0097-8485(93)85004-V&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B7"><span class="mixed-citation">Chakraborty J., Ghosal D., Dutta A., Dutta T. (2012). <span class="ref-title">An insight into the origin and functional evolution of bacterial aromatic ring-hydroxylating oxygenases.</span>
<span class="ref-journal"><em>J. Biomol. Struct. Dyn.</em></span>
<span class="ref-vol">30</span>
419&#x02013;436. 10.1080/07391102.2012.682208

 [<a href="https://pubmed.ncbi.nlm.nih.gov/22694139" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1080%2F07391102.2012.682208" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biomol.+Struct.+Dyn.&amp;title=An+insight+into+the+origin+and+functional+evolution+of+bacterial+aromatic+ring-hydroxylating+oxygenases.&amp;author=J.+Chakraborty&amp;author=D.+Ghosal&amp;author=A.+Dutta&amp;author=T.+Dutta&amp;volume=30&amp;publication_year=2012&amp;pages=419-436&amp;pmid=22694139&amp;doi=10.1080/07391102.2012.682208&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B8"><span class="mixed-citation">Chen S., Deng Y., Chang C., Lee J., Cheng Y., Cui Z., et al. (2015). <span class="ref-title">Pathway and kinetics of cyhalothrin biodegradation by <em>Bacillus thuringiensis</em> strain ZS-19.</span>
<span class="ref-journal"><em>Sci. Rep.</em></span>
<span class="ref-vol">5</span>:<span class="ref-iss">8784</span>. 10.1038/srep08784

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4350101/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25740758" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fsrep08784" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Rep.&amp;title=Pathway+and+kinetics+of+cyhalothrin+biodegradation+by+Bacillus+thuringiensis+strain+ZS-19.&amp;author=S.+Chen&amp;author=Y.+Deng&amp;author=C.+Chang&amp;author=J.+Lee&amp;author=Y.+Cheng&amp;volume=5&amp;issue=8784&amp;publication_year=2015&amp;pmid=25740758&amp;doi=10.1038/srep08784&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B9"><span class="mixed-citation">Chen S., Hu M., Liu J., Zhong G., Liu Y., Rizwan-Ul-Haq M., et al. (2011a). <span class="ref-title">Biodegradation of beta-cypermethrin and 3-phenoxybenzoic acid by a novel <em>Ochrobactrum lupini</em> DG-S-01.</span>
<span class="ref-journal"><em>J. Hazard Mater.</em></span>
<span class="ref-vol">187</span>
433&#x02013;440. 10.1016/j.jhazmat.2011.01.049

 [<a href="https://pubmed.ncbi.nlm.nih.gov/21282001" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.jhazmat.2011.01.049" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Hazard+Mater.&amp;title=Biodegradation+of+beta-cypermethrin+and+3-phenoxybenzoic+acid+by+a+novel+Ochrobactrum+lupini+DG-S-01.&amp;author=S.+Chen&amp;author=M.+Hu&amp;author=J.+Liu&amp;author=G.+Zhong&amp;author=Y.+Liu&amp;volume=187&amp;publication_year=2011a&amp;pages=433-440&amp;pmid=21282001&amp;doi=10.1016/j.jhazmat.2011.01.049&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B10"><span class="mixed-citation">Chen S., Yang L., Hu M., Liu J. (2011b). <span class="ref-title">Biodegradation of fenvalerate and 3-phenoxybenzoic acid by a novel <em>Stenotrophomonas</em> sp. strain ZS-S-01 and its use in bioremediation of contaminated soils.</span>
<span class="ref-journal"><em>Appl. Microbiol. Biotechnol.</em></span>
<span class="ref-vol">90</span>
755&#x02013;767. 10.1007/s00253-010-3035-z

 [<a href="https://pubmed.ncbi.nlm.nih.gov/21184062" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs00253-010-3035-z" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Appl.+Microbiol.+Biotechnol.&amp;title=Biodegradation+of+fenvalerate+and+3-phenoxybenzoic+acid+by+a+novel+Stenotrophomonas+sp.+strain+ZS-S-01+and+its+use+in+bioremediation+of+contaminated+soils.&amp;author=S.+Chen&amp;author=L.+Yang&amp;author=M.+Hu&amp;author=J.+Liu&amp;volume=90&amp;publication_year=2011b&amp;pages=755-767&amp;pmid=21184062&amp;doi=10.1007/s00253-010-3035-z&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B11"><span class="mixed-citation">Chen S., Hu W., Xiao Y., Deng Y., Jia J., Hu M. (2012). <span class="ref-title">Degradation of 3-phenoxybenzoic acid by a <em>Bacillus</em> sp.</span>
<span class="ref-journal"><em>PLoS One</em></span>
<span class="ref-vol">7</span>:<span class="ref-iss">e50456</span>. 10.1371/journal.pone.0050456

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3511583/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23226289" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pone.0050456" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Degradation+of+3-phenoxybenzoic+acid+by+a+Bacillus+sp.&amp;author=S.+Chen&amp;author=W.+Hu&amp;author=Y.+Xiao&amp;author=Y.+Deng&amp;author=J.+Jia&amp;volume=7&amp;issue=e50456&amp;publication_year=2012&amp;pmid=23226289&amp;doi=10.1371/journal.pone.0050456&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B12"><span class="mixed-citation">Dashtban M., Schraft H., Qin W. (2009). <span class="ref-title">Fungal bioconversion of lignocellulosic residues; opportunities &#x00026; perspectives.</span>
<span class="ref-journal"><em>Int. J. Biol. Sci.</em></span>
<span class="ref-vol">5</span>
578&#x02013;595. 10.7150/ijbs.5.578

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2748470/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19774110" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.7150%2Fijbs.5.578" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Biol.+Sci.&amp;title=Fungal+bioconversion+of+lignocellulosic+residues;+opportunities+&#x00026;+perspectives.&amp;author=M.+Dashtban&amp;author=H.+Schraft&amp;author=W.+Qin&amp;volume=5&amp;publication_year=2009&amp;pages=578-595&amp;pmid=19774110&amp;doi=10.7150/ijbs.5.578&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B13"><span class="mixed-citation">Dehmel U., Engesser K., Timmis K., Dwyer D. (1995). <span class="ref-title">Cloning, nucleotide sequence, and expression of the gene encoding a novel dioxygenase involved in metabolism of carboxydiphenyl ethers in <em>Pseudomonas pseudoalcaligenes</em> POB310.</span>
<span class="ref-journal"><em>Arch. Microbiol.</em></span>
<span class="ref-vol">163</span>
35&#x02013;41. 10.1007/s002030050168
 [<a href="https://pubmed.ncbi.nlm.nih.gov/7710319" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs002030050168" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Microbiol.&amp;title=Cloning,+nucleotide+sequence,+and+expression+of+the+gene+encoding+a+novel+dioxygenase+involved+in+metabolism+of+carboxydiphenyl+ethers+in+Pseudomonas+pseudoalcaligenes+POB310.&amp;author=U.+Dehmel&amp;author=K.+Engesser&amp;author=K.+Timmis&amp;author=D.+Dwyer&amp;volume=163&amp;publication_year=1995&amp;pages=35-41&amp;pmid=7710319&amp;doi=10.1007/s002030050168&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B14"><span class="mixed-citation">Delcher A. L., Bratke K. A., Powers E. C., Salzberg S. L. (2007). <span class="ref-title">Identifying bacterial genes and endosymbiont DNA with Glimmer.</span>
<span class="ref-journal"><em>Bioinformatics</em></span>
<span class="ref-vol">23</span>
673&#x02013;679. 10.1093/bioinformatics/btm009

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2387122/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17237039" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtm009" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Identifying+bacterial+genes+and+endosymbiont+DNA+with+Glimmer.&amp;author=A.+L.+Delcher&amp;author=K.+A.+Bratke&amp;author=E.+C.+Powers&amp;author=S.+L.+Salzberg&amp;volume=23&amp;publication_year=2007&amp;pages=673-679&amp;pmid=17237039&amp;doi=10.1093/bioinformatics/btm009&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B15"><span class="mixed-citation">Halden R., Tepp S. (1999). <span class="ref-title">Degradation of 3-phenoxybenzoic acid in soil by <em>Pseudomonas pseudoalcaligenes</em> POB310(pPOB) and two modified <em>Pseudomonas</em> strains.</span>
<span class="ref-journal"><em>Appl. Environ. Microbiol.</em></span>
<span class="ref-vol">65</span>
3354&#x02013;3359. 10.1128/AEM.65.8.3354-3359.1999

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC91504/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10427019" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FAEM.65.8.3354-3359.1999" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Appl.+Environ.+Microbiol.&amp;title=Degradation+of+3-phenoxybenzoic+acid+in+soil+by+Pseudomonas+pseudoalcaligenes+POB310(pPOB)+and+two+modified+Pseudomonas+strains.&amp;author=R.+Halden&amp;author=S.+Tepp&amp;volume=65&amp;publication_year=1999&amp;pages=3354-3359&amp;pmid=10427019&amp;doi=10.1128/AEM.65.8.3354-3359.1999&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B16"><span class="mixed-citation">Hu K., Li J., Zhao T., Zhou Q., Li Q., Hu X., et al. (2023). <span class="ref-title">Transcriptomic analysis reveals peripheral pathway in 3-phenoxybenzoic acid degradation by <em>Aspergillus oryzae</em> M-4.</span>
<span class="ref-journal"><em>J. Environ. Manage.</em></span>
<span class="ref-vol">325</span>:<span class="ref-iss">116626</span>. 10.1016/j.jenvman.2022.116626

 [<a href="https://pubmed.ncbi.nlm.nih.gov/36327606" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.jenvman.2022.116626" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Environ.+Manage.&amp;title=Transcriptomic+analysis+reveals+peripheral+pathway+in+3-phenoxybenzoic+acid+degradation+by+Aspergillus+oryzae+M-4.&amp;author=K.+Hu&amp;author=J.+Li&amp;author=T.+Zhao&amp;author=Q.+Zhou&amp;author=Q.+Li&amp;volume=325&amp;issue=116626&amp;publication_year=2023&amp;pmid=36327606&amp;doi=10.1016/j.jenvman.2022.116626&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B17"><span class="mixed-citation">Jiayuan Z., Yuanlong C., Yingchao X., Dongying J., Kai Y., Kumar A. (2016). <span class="ref-title">Co-Metabolic degradation of &#x003b2;-cypermethrin and 3-phenoxybenzoic acid by co-culture of <em>Bacillus licheniformis</em> B-1 and <em>Aspergillus oryzae</em> M-4.</span>
<span class="ref-journal"><em>PLoS One</em></span>
<span class="ref-vol">11</span>:<span class="ref-iss">e0166796</span>. 10.1371/journal.pone.0166796

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5127528/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27898684" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pone.0166796" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Co-Metabolic+degradation+of+&#x003b2;-cypermethrin+and+3-phenoxybenzoic+acid+by+co-culture+of+Bacillus+licheniformis+B-1+and+Aspergillus+oryzae+M-4.&amp;author=Z.+Jiayuan&amp;author=C.+Yuanlong&amp;author=X.+Yingchao&amp;author=J.+Dongying&amp;author=Y.+Kai&amp;volume=11&amp;issue=e0166796&amp;publication_year=2016&amp;pmid=27898684&amp;doi=10.1371/journal.pone.0166796&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B18"><span class="mixed-citation">Jie T., Kai Y., Liu S., Jia D., Chi Y., Zeng C., et al. (2013). <span class="ref-title">Biodegradation of 3-phenoxybenzoic acid by a novel <em>Sphingomonas</em> sp. SC-1. <em>Fresenius</em>.</span>
<span class="ref-journal"><em>Environ. Bull.</em></span>
<span class="ref-vol">22</span>
1564&#x02013;1572. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Environ.+Bull.&amp;title=Biodegradation+of+3-phenoxybenzoic+acid+by+a+novel+Sphingomonas+sp.+SC-1.+Fresenius.&amp;author=T.+Jie&amp;author=Y.+Kai&amp;author=S.+Liu&amp;author=D.+Jia&amp;author=Y.+Chi&amp;volume=22&amp;publication_year=2013&amp;pages=1564-1572&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B19"><span class="mixed-citation">Koren S., Walenz B. P., Berlin K., Miller J. R., Bergman N. H., Phillippy A. M. (2017). <span class="ref-title">Canu: Scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation.</span>
<span class="ref-journal"><em>Genome Res.</em></span>
<span class="ref-vol">27</span>
722&#x02013;736. 10.1101/gr.215087.116

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5411767/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28298431" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1101%2Fgr.215087.116" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=Canu:+Scalable+and+accurate+long-read+assembly+via+adaptive+k-mer+weighting+and+repeat+separation.&amp;author=S.+Koren&amp;author=B.+P.+Walenz&amp;author=K.+Berlin&amp;author=J.+R.+Miller&amp;author=N.+H.+Bergman&amp;volume=27&amp;publication_year=2017&amp;pages=722-736&amp;pmid=28298431&amp;doi=10.1101/gr.215087.116&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B20"><span class="mixed-citation">Kweon O., Kim S., Baek S., Chae J., Adjei M., Baek D., et al. (2008). <span class="ref-title">A new classification system for bacterial Rieske non-heme iron aromatic ring-hydroxylating oxygenases.</span>
<span class="ref-journal"><em>BMC Biochem.</em></span>
<span class="ref-vol">9</span>:<span class="ref-iss">11</span>. 10.1186/1471-2091-9-11

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2358900/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18387195" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2F1471-2091-9-11" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Biochem.&amp;title=A+new+classification+system+for+bacterial+Rieske+non-heme+iron+aromatic+ring-hydroxylating+oxygenases.&amp;author=O.+Kweon&amp;author=S.+Kim&amp;author=S.+Baek&amp;author=J.+Chae&amp;author=M.+Adjei&amp;volume=9&amp;issue=11&amp;publication_year=2008&amp;pmid=18387195&amp;doi=10.1186/1471-2091-9-11&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B21"><span class="mixed-citation">Lee J., Shin Y., Yoon J., Takeuchi M., Park Y. (2001). <span class="ref-title"><em>Sphingomonas aquatilis</em> sp. nov., <em>Sphingomonas koreensis</em> sp. nov., and <em>Sphingomonas taejonensis</em> sp. nov., yellow-pigmented bacteria isolated from natural mineral water.</span>
<span class="ref-journal"><em>Int. J. Syst. Evol. Microbiol.</em></span>
<span class="ref-vol">51</span>
1491&#x02013;1498. 10.1099/00207713-51-4-1491

 [<a href="https://pubmed.ncbi.nlm.nih.gov/11491350" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1099%2F00207713-51-4-1491" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Syst.+Evol.+Microbiol.&amp;title=Sphingomonas+aquatilis+sp.+nov.,+Sphingomonas+koreensis+sp.+nov.,+and+Sphingomonas+taejonensis+sp.+nov.,+yellow-pigmented+bacteria+isolated+from+natural+mineral+water.&amp;author=J.+Lee&amp;author=Y.+Shin&amp;author=J.+Yoon&amp;author=M.+Takeuchi&amp;author=Y.+Park&amp;volume=51&amp;publication_year=2001&amp;pages=1491-1498&amp;pmid=11491350&amp;doi=10.1099/00207713-51-4-1491&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B22"><span class="mixed-citation">Li J., Lin D., Ji R., Yao K., Deng W.-Q., Yuan H., et al. (2016). <span class="ref-title">Simultaneous determination of &#x003b2;-cypermethrin and its metabolite 3-phenoxybenzoic acid in microbial degradation systems by HPLC&#x02013;UV.</span>
<span class="ref-journal"><em>J. Chromatogr. Sci.</em></span>
<span class="ref-vol">54</span>
1584&#x02013;1592. 10.1093/chromsci/bmw108

 [<a href="https://pubmed.ncbi.nlm.nih.gov/27390099" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fchromsci%2Fbmw108" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Chromatogr.+Sci.&amp;title=Simultaneous+determination+of+&#x003b2;-cypermethrin+and+its+metabolite+3-phenoxybenzoic+acid+in+microbial+degradation+systems+by+HPLC&#x02013;UV.&amp;author=J.+Li&amp;author=D.+Lin&amp;author=R.+Ji&amp;author=K.+Yao&amp;author=W.-Q.+Deng&amp;volume=54&amp;publication_year=2016&amp;pages=1584-1592&amp;pmid=27390099&amp;doi=10.1093/chromsci/bmw108&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B23"><span class="mixed-citation">Liang Y., Gardner D., Miller C., Chen D., Anderson A., Weimer B., et al. (2006). <span class="ref-title">Study of biochemical pathways and enzymes involved in pyrene degradation by <em>Mycobacterium</em> sp. Strain KMS.</span>
<span class="ref-journal"><em>Appl. Environ. Microbiol.</em></span>
<span class="ref-vol">72</span>
7821&#x02013;7828. 10.1128/AEM.01274-06

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1694249/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17041157" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FAEM.01274-06" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Appl.+Environ.+Microbiol.&amp;title=Study+of+biochemical+pathways+and+enzymes+involved+in+pyrene+degradation+by+Mycobacterium+sp.+Strain+KMS.&amp;author=Y.+Liang&amp;author=D.+Gardner&amp;author=C.+Miller&amp;author=D.+Chen&amp;author=A.+Anderson&amp;volume=72&amp;publication_year=2006&amp;pages=7821-7828&amp;pmid=17041157&amp;doi=10.1128/AEM.01274-06&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B24"><span class="mixed-citation">Maloney S., Maule A., Smith A. (1988). <span class="ref-title">Microbial transformation of the pyrethroid insecticides: Permethrin, deltamethrin, fastac, fenvalerate, and fluvalinate.</span>
<span class="ref-journal"><em>Appl. Environ. Microbiol.</em></span>
<span class="ref-vol">54</span>
2874&#x02013;2876. 10.1128/AEM.54.11.2874-2876.1988

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC204391/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3145715" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FAEM.54.11.2874-2876.1988" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Appl.+Environ.+Microbiol.&amp;title=Microbial+transformation+of+the+pyrethroid+insecticides:+Permethrin,+deltamethrin,+fastac,+fenvalerate,+and+fluvalinate.&amp;author=S.+Maloney&amp;author=A.+Maule&amp;author=A.+Smith&amp;volume=54&amp;publication_year=1988&amp;pages=2874-2876&amp;pmid=3145715&amp;doi=10.1128/AEM.54.11.2874-2876.1988&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B25"><span class="mixed-citation">Nelson D. (2009). <span class="ref-title">The cytochrome p450 homepage.</span>
<span class="ref-journal"><em>Hum. Genomics</em></span>
<span class="ref-vol">4</span>
59&#x02013;65. 10.1186/1479-7364-4-1-59

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3500189/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19951895" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2F1479-7364-4-1-59" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genomics&amp;title=The+cytochrome+p450+homepage.&amp;author=D.+Nelson&amp;volume=4&amp;publication_year=2009&amp;pages=59-65&amp;pmid=19951895&amp;doi=10.1186/1479-7364-4-1-59&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B26"><span class="mixed-citation">Qi X., Zheng M., Wu C., Wang G., Feng C., Zhou Z. (2012). <span class="ref-title">Urinary pyrethroid metabolites among pregnant women in an agricultural area of the Province of Jiangsu, China.</span>
<span class="ref-journal"><em>Int. J. Hyg. Environ. Health</em></span>
<span class="ref-vol">215</span>
487&#x02013;495. 10.1016/j.ijheh.2011.12.003

 [<a href="https://pubmed.ncbi.nlm.nih.gov/22218106" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.ijheh.2011.12.003" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Hyg.+Environ.+Health&amp;title=Urinary+pyrethroid+metabolites+among+pregnant+women+in+an+agricultural+area+of+the+Province+of+Jiangsu,+China.&amp;author=X.+Qi&amp;author=M.+Zheng&amp;author=C.+Wu&amp;author=G.+Wang&amp;author=C.+Feng&amp;volume=215&amp;publication_year=2012&amp;pages=487-495&amp;pmid=22218106&amp;doi=10.1016/j.ijheh.2011.12.003&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B27"><span class="mixed-citation">Quir&#x000f3;s-Alcal&#x000e1; L., Mehta S., Eskenazi B. (2014). <span class="ref-title">Pyrethroid pesticide exposure and parental report of learning disability and attention deficit/hyperactivity disorder in U.S. children: NHANES 1999&#x02013;2002.</span>
<span class="ref-journal"><em>Environ. Health Perspect.</em></span>
<span class="ref-vol">122</span>
1336&#x02013;1342. 10.1289/ehp.1308031

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4256700/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25192380" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1289%2Fehp.1308031" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Environ.+Health+Perspect.&amp;title=Pyrethroid+pesticide+exposure+and+parental+report+of+learning+disability+and+attention+deficit/hyperactivity+disorder+in+U.S.+children:+NHANES+1999&#x02013;2002.&amp;author=L.+Quir&#x000f3;s-Alcal&#x000e1;&amp;author=S.+Mehta&amp;author=B.+Eskenazi&amp;volume=122&amp;publication_year=2014&amp;pages=1336-1342&amp;pmid=25192380&amp;doi=10.1289/ehp.1308031&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B28"><span class="mixed-citation">Sadana A. (1988). <span class="ref-title">Enzyme deactivation.</span>
<span class="ref-journal"><em>Biotechnol. Adv.</em></span>
<span class="ref-vol">6</span>
349&#x02013;446. 10.1016/0734-9750(88)91890-3

 [<a href="https://pubmed.ncbi.nlm.nih.gov/14543686" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2F0734-9750(88)91890-3" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnol.+Adv.&amp;title=Enzyme+deactivation.&amp;author=A.+Sadana&amp;volume=6&amp;publication_year=1988&amp;pages=349-446&amp;pmid=14543686&amp;doi=10.1016/0734-9750(88)91890-3&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B29"><span class="mixed-citation">Sun H., Chen W., Xu X., Ding Z., Chen X., Wang X. (2014). <span class="ref-title">Pyrethroid and their metabolite, 3-phenoxybenzoic acid showed similar (anti)estrogenic activity in human and rat estrogen receptor &#x003b1;-mediated reporter gene assays.</span>
<span class="ref-journal"><em>Environ. Toxicol. Pharmacol.</em></span>
<span class="ref-vol">37</span>
371&#x02013;377. 10.1016/j.etap.2013.11.031

 [<a href="https://pubmed.ncbi.nlm.nih.gov/24388911" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.etap.2013.11.031" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Environ.+Toxicol.+Pharmacol.&amp;title=Pyrethroid+and+their+metabolite,+3-phenoxybenzoic+acid+showed+similar+(anti)estrogenic+activity+in+human+and+rat+estrogen+receptor+&#x003b1;-mediated+reporter+gene+assays.&amp;author=H.+Sun&amp;author=W.+Chen&amp;author=X.+Xu&amp;author=Z.+Ding&amp;author=X.+Chen&amp;volume=37&amp;publication_year=2014&amp;pages=371-377&amp;pmid=24388911&amp;doi=10.1016/j.etap.2013.11.031&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B30"><span class="mixed-citation">Tallur P., Megadi V., Ninnekar H. (2008). <span class="ref-title">Biodegradation of cypermethrin by <em>Micrococcus</em> sp. strain CPN 1.</span>
<span class="ref-journal"><em>Biodegradation</em></span>
<span class="ref-vol">19</span>
77&#x02013;82. 10.1007/s10532-007-9116-8

 [<a href="https://pubmed.ncbi.nlm.nih.gov/17431802" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs10532-007-9116-8" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biodegradation&amp;title=Biodegradation+of+cypermethrin+by+Micrococcus+sp.+strain+CPN+1.&amp;author=P.+Tallur&amp;author=V.+Megadi&amp;author=H.+Ninnekar&amp;volume=19&amp;publication_year=2008&amp;pages=77-82&amp;pmid=17431802&amp;doi=10.1007/s10532-007-9116-8&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B31"><span class="mixed-citation">Tang W., Wang D., Wang J., Wu Z., Li L., Huang M., et al. (2018). <span class="ref-title">Pyrethroid pesticide residues in the global environment: An overview.</span>
<span class="ref-journal"><em>Chemosphere</em></span>
<span class="ref-vol">191</span>
990&#x02013;1007. 10.1016/j.chemosphere.2017.10.115

 [<a href="https://pubmed.ncbi.nlm.nih.gov/29145144" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.chemosphere.2017.10.115" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chemosphere&amp;title=Pyrethroid+pesticide+residues+in+the+global+environment:+An+overview.&amp;author=W.+Tang&amp;author=D.+Wang&amp;author=J.+Wang&amp;author=Z.+Wu&amp;author=L.+Li&amp;volume=191&amp;publication_year=2018&amp;pages=990-1007&amp;pmid=29145144&amp;doi=10.1016/j.chemosphere.2017.10.115&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B32"><span class="mixed-citation">Tang J., Liu B., Chen T., Yao K., Zeng L., Zeng C., et al. (2018). <span class="ref-title">Screening of a beta-cypermethrin-degrading bacterial strain <em>Brevibacillus parabrevis</em> BCP-09 and its biochemical degradation pathway.</span>
<span class="ref-journal"><em>Biodegradation</em></span>
<span class="ref-vol">29</span>
525&#x02013;541. 10.1007/s10532-018-9850-0

 [<a href="https://pubmed.ncbi.nlm.nih.gov/30116919" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs10532-018-9850-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biodegradation&amp;title=Screening+of+a+beta-cypermethrin-degrading+bacterial+strain+Brevibacillus+parabrevis+BCP-09+and+its+biochemical+degradation+pathway.&amp;author=J.+Tang&amp;author=B.+Liu&amp;author=T.+Chen&amp;author=K.+Yao&amp;author=L.+Zeng&amp;volume=29&amp;publication_year=2018&amp;pages=525-541&amp;pmid=30116919&amp;doi=10.1007/s10532-018-9850-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B33"><span class="mixed-citation">Topp E., Akhtar M. (1991). <span class="ref-title">Identification and characterization of a pseudomonas Strain capable of metabolizing phenoxybenzoates.</span>
<span class="ref-journal"><em>Appl. Environ. Microbiol.</em></span>
<span class="ref-vol">57</span>
1294&#x02013;1300. 10.1128/AEM.57.5.1294-1300.1991

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC182945/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16348477" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FAEM.57.5.1294-1300.1991" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Appl.+Environ.+Microbiol.&amp;title=Identification+and+characterization+of+a+pseudomonas+Strain+capable+of+metabolizing+phenoxybenzoates.&amp;author=E.+Topp&amp;author=M.+Akhtar&amp;volume=57&amp;publication_year=1991&amp;pages=1294-1300&amp;pmid=16348477&amp;doi=10.1128/AEM.57.5.1294-1300.1991&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B34"><span class="mixed-citation">Ueyama J., Harada K., Koizumi A., Sugiura Y., Kondo T., Saito I., et al. (2015). <span class="ref-title">Temporal levels of urinary neonicotinoid and dialkylphosphate concentrations in Japanese women between 1994 and 2011.</span>
<span class="ref-journal"><em>Environ. Sci. Technol.</em></span>
<span class="ref-vol">49</span>
14522&#x02013;14528. 10.1021/acs.est.5b03062

 [<a href="https://pubmed.ncbi.nlm.nih.gov/26556224" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Facs.est.5b03062" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Environ.+Sci.+Technol.&amp;title=Temporal+levels+of+urinary+neonicotinoid+and+dialkylphosphate+concentrations+in+Japanese+women+between+1994+and+2011.&amp;author=J.+Ueyama&amp;author=K.+Harada&amp;author=A.+Koizumi&amp;author=Y.+Sugiura&amp;author=T.+Kondo&amp;volume=49&amp;publication_year=2015&amp;pages=14522-14528&amp;pmid=26556224&amp;doi=10.1021/acs.est.5b03062&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B35"><span class="mixed-citation">Wang B., Guo P., Hang B., Li L., He J., Li S. (2009). <span class="ref-title">Cloning of a novel pyrethroid-hydrolyzing carboxylesterase gene from Sphingobium sp. strain JZ-1 and characterization of the gene product.</span>
<span class="ref-journal"><em>Appl. Environ. Microbiol.</em></span>
<span class="ref-vol">75</span>:<span class="ref-iss">5496</span>. 10.1128/AEM.01298-09

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2737939/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19581484" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FAEM.01298-09" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Appl.+Environ.+Microbiol.&amp;title=Cloning+of+a+novel+pyrethroid-hydrolyzing+carboxylesterase+gene+from+Sphingobium+sp.+strain+JZ-1+and+characterization+of+the+gene+product.&amp;author=B.+Wang&amp;author=P.+Guo&amp;author=B.+Hang&amp;author=L.+Li&amp;author=J.+He&amp;volume=75&amp;issue=5496&amp;publication_year=2009&amp;pmid=19581484&amp;doi=10.1128/AEM.01298-09&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B36"><span class="mixed-citation">Wang C., Chen Q., Wang R., Shi C., Li S. (2014). <span class="ref-title">A novel angular dioxygenase gene cluster encoding 3-phenoxybenzoate 1&#x02032;, 2&#x02032;-dioxygenase in <em>Sphingobium wenxiniae</em> JZ-1.</span>
<span class="ref-journal"><em>Appl. Environ. Microbiol.</em></span>
<span class="ref-vol">80</span>
3811&#x02013;3818. 10.1128/AEM.00208-14

 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4054222/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24747891" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FAEM.00208-14" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Appl.+Environ.+Microbiol.&amp;title=A+novel+angular+dioxygenase+gene+cluster+encoding+3-phenoxybenzoate+1&#x02032;,+2&#x02032;-dioxygenase+in+Sphingobium+wenxiniae+JZ-1.&amp;author=C.+Wang&amp;author=Q.+Chen&amp;author=R.+Wang&amp;author=C.+Shi&amp;author=S.+Li&amp;volume=80&amp;publication_year=2014&amp;pages=3811-3818&amp;pmid=24747891&amp;doi=10.1128/AEM.00208-14&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B37"><span class="mixed-citation">Wang X., He B., Kong B., Wei L., Wang R., Zhou C., et al. (2017). <span class="ref-title">&#x003b2;-Cypermethrin and its metabolite 3-phenoxybenzoic acid exhibit immunotoxicity in murine macrophages.</span>
<span class="ref-journal"><em>Acta Biochim. Biophys. Sin.</em></span>
<span class="ref-vol">49</span>
1083&#x02013;1091. 10.1093/abbs/gmx111

 [<a href="https://pubmed.ncbi.nlm.nih.gov/29040392" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fabbs%2Fgmx111" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Biochim.+Biophys.+Sin.&amp;title=&#x003b2;-Cypermethrin+and+its+metabolite+3-phenoxybenzoic+acid+exhibit+immunotoxicity+in+murine+macrophages.&amp;author=X.+Wang&amp;author=B.+He&amp;author=B.+Kong&amp;author=L.+Wei&amp;author=R.+Wang&amp;volume=49&amp;publication_year=2017&amp;pages=1083-1091&amp;pmid=29040392&amp;doi=10.1093/abbs/gmx111&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B38"><span class="mixed-citation">White G., Russell N., Tidswell E. (1996). <span class="ref-title">Bacterial scission of ether bonds.</span>
<span class="ref-journal"><em>Microbiol. Rev.</em></span>
<span class="ref-vol">60</span>
216&#x02013;232. 10.1128/MMBR.60.1.216-232.1996
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC239425/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8852901" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FMMBR.60.1.216-232.1996" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol.+Rev.&amp;title=Bacterial+scission+of+ether+bonds.&amp;author=G.+White&amp;author=N.+Russell&amp;author=E.+Tidswell&amp;volume=60&amp;publication_year=1996&amp;pages=216-232&amp;pmid=8852901&amp;doi=10.1128/MMBR.60.1.216-232.1996&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B39"><span class="mixed-citation">Yabuuchi E., Yano I., Oyaizu H., Hashimoto Y., Ezaki T., Yamamoto H. (2011). <span class="ref-title">Proposals of <em>Sphingomonas paucimobilis</em> gen. nov. and comb. nov., <em>Sphingomonas parapaucimobilis</em> sp. nov., <em>Sphingomonas yanoikuyae</em> sp. nov., <em>Sphingomonas adhaesiva</em> sp. nov., <em>Sphingomonas capsulata comb</em>, nov., and two genospecies of the genus <em>Sphingomonas</em>.</span>
<span class="ref-journal"><em>Microbiol. Immunol.</em></span>
<span class="ref-vol">34</span>
99&#x02013;119. 10.1111/j.1348-0421.1990.tb00996.x

 [<a href="https://pubmed.ncbi.nlm.nih.gov/2111872" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fj.1348-0421.1990.tb00996.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol.+Immunol.&amp;title=Proposals+of+Sphingomonas+paucimobilis+gen.+nov.+and+comb.+nov.,+Sphingomonas+parapaucimobilis+sp.+nov.,+Sphingomonas+yanoikuyae+sp.+nov.,+Sphingomonas+adhaesiva+sp.+nov.,+Sphingomonas+capsulata+comb,+nov.,+and+two+genospecies+of+the+genus+Sphingomonas.&amp;author=E.+Yabuuchi&amp;author=I.+Yano&amp;author=H.+Oyaizu&amp;author=Y.+Hashimoto&amp;author=T.+Ezaki&amp;volume=34&amp;publication_year=2011&amp;pages=99-119&amp;pmid=2111872&amp;doi=10.1111/j.1348-0421.1990.tb00996.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B40"><span class="mixed-citation">Zhan H., Huang Y. H., Lin Z. Q., Bhatt P., Chen S. H. (2020). <span class="ref-title">New insights into the microbial degradation and catalytic mechanism of synthetic pyrethroids.</span>
<span class="ref-journal"><em>Environ. Res.</em></span>
<span class="ref-vol">182</span>:<span class="ref-iss">109138</span>. 10.1016/j.envres.2020.109138

 [<a href="https://pubmed.ncbi.nlm.nih.gov/32069744" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.envres.2020.109138" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Environ.+Res.&amp;title=New+insights+into+the+microbial+degradation+and+catalytic+mechanism+of+synthetic+pyrethroids.&amp;author=H.+Zhan&amp;author=Y.+H.+Huang&amp;author=Z.+Q.+Lin&amp;author=P.+Bhatt&amp;author=S.+H.+Chen&amp;volume=182&amp;issue=109138&amp;publication_year=2020&amp;pmid=32069744&amp;doi=10.1016/j.envres.2020.109138&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B41"><span class="mixed-citation">Zhang C. Y., Zhao S. Q., Zhang S. L., Luo L. H., Liu D. C., Ding W. H., et al. (2021). <span class="ref-title">Database study on the expression and purification of membrane proteins.</span>
<span class="ref-journal"><em>Protein Pept. Lett.</em></span>
<span class="ref-vol">28</span>
972&#x02013;982. 10.2174/0929866528666210415120234

 [<a href="https://pubmed.ncbi.nlm.nih.gov/33858308" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.2174%2F0929866528666210415120234" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Pept.+Lett.&amp;title=Database+study+on+the+expression+and+purification+of+membrane+proteins.&amp;author=C.+Y.+Zhang&amp;author=S.+Q.+Zhao&amp;author=S.+L.+Zhang&amp;author=L.+H.+Luo&amp;author=D.+C.+Liu&amp;volume=28&amp;publication_year=2021&amp;pages=972-982&amp;pmid=33858308&amp;doi=10.2174/0929866528666210415120234&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B42"><span class="mixed-citation">Zhao J., Jia D., Chi Y., Yao K. (2020). <span class="ref-title">Co-metabolic enzymes and pathways of 3-phenoxybenzoic acid degradation by <em>Aspergillus oryzae</em> M-4.</span>
<span class="ref-journal"><em>Ecotoxicol. Environ. Saf.</em></span>
<span class="ref-vol">189</span>:<span class="ref-iss">109953</span>. 10.1016/j.ecoenv.2019.109953

 [<a href="https://pubmed.ncbi.nlm.nih.gov/31759741" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.ecoenv.2019.109953" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ecotoxicol.+Environ.+Saf.&amp;title=Co-metabolic+enzymes+and+pathways+of+3-phenoxybenzoic+acid+degradation+by+Aspergillus+oryzae+M-4.&amp;author=J.+Zhao&amp;author=D.+Jia&amp;author=Y.+Chi&amp;author=K.+Yao&amp;volume=189&amp;issue=109953&amp;publication_year=2020&amp;pmid=31759741&amp;doi=10.1016/j.ecoenv.2019.109953&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B43"><span class="mixed-citation">Zhao J. Y., Jiang Y. D., Gong L. M., Chen X. F., Xie Q. L., Jin Y., et al. (2022). <span class="ref-title">Mechanism of &#x003b2;-cypermethrin metabolism by <em>Bacillus cereus</em> GW-01.</span>
<span class="ref-journal"><em>Chem. Eng. J.</em></span>
<span class="ref-vol">430</span>:<span class="ref-iss">132961</span>. 10.1016/j.cej.2021.132961 [<a href="//doi.org/10.1016%2Fj.cej.2021.132961" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem.+Eng.+J.&amp;title=Mechanism+of+&#x003b2;-cypermethrin+metabolism+by+Bacillus+cereus+GW-01.&amp;author=J.+Y.+Zhao&amp;author=Y.+D.+Jiang&amp;author=L.+M.+Gong&amp;author=X.+F.+Chen&amp;author=Q.+L.+Xie&amp;volume=430&amp;issue=132961&amp;publication_year=2022&amp;doi=10.1016/j.cej.2021.132961&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B44"><span class="mixed-citation">Zhao Q., Bilal M., Yue S., Hu H., Wang W., Zhang X. (2017). <span class="ref-title">Identification of biphenyl 2, 3-dioxygenase and its catabolic role for phenazine degradation in <em>Sphingobium yanoikuyae</em> B1.</span>
<span class="ref-journal"><em>J. Environ. Manage.</em></span>
<span class="ref-vol">204</span>
494&#x02013;501. 10.1016/j.jenvman.2017.09.027

 [<a href="https://pubmed.ncbi.nlm.nih.gov/28930694" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.jenvman.2017.09.027" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Environ.+Manage.&amp;title=Identification+of+biphenyl+2,+3-dioxygenase+and+its+catabolic+role+for+phenazine+degradation+in+Sphingobium+yanoikuyae+B1.&amp;author=Q.+Zhao&amp;author=M.+Bilal&amp;author=S.+Yue&amp;author=H.+Hu&amp;author=W.+Wang&amp;volume=204&amp;publication_year=2017&amp;pages=494-501&amp;pmid=28930694&amp;doi=10.1016/j.jenvman.2017.09.027&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B45"><span class="mixed-citation">Zhao T., Hu K., Li J., Zhu Y., Liu A., Yao K., et al. (2021). <span class="ref-title">Current insights into the microbial degradation for pyrethroids: Strain safety, biochemical pathway, and genetic engineering.</span>
<span class="ref-journal"><em>Chemosphere</em></span>
<span class="ref-vol">279</span>:<span class="ref-iss">130542</span>. 10.1016/j.chemosphere.2021.130542

 [<a href="https://pubmed.ncbi.nlm.nih.gov/33866100" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.chemosphere.2021.130542" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chemosphere&amp;title=Current+insights+into+the+microbial+degradation+for+pyrethroids:+Strain+safety,+biochemical+pathway,+and+genetic+engineering.&amp;author=T.+Zhao&amp;author=K.+Hu&amp;author=J.+Li&amp;author=Y.+Zhu&amp;author=A.+Liu&amp;volume=279&amp;issue=130542&amp;publication_year=2021&amp;pmid=33866100&amp;doi=10.1016/j.chemosphere.2021.130542&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B46"><span class="mixed-citation">Zhu Y., Li J., Yao K., Zhao N., Zhou K., Hu X., et al. (2016). <span class="ref-title">Degradation of 3-phenoxybenzoic acid by a filamentous fungus <em>Aspergillus oryzae</em> M-4 strain with self-protection transformation.</span>
<span class="ref-journal"><em>Appl. Microbiol. Biotechnol.</em></span>
<span class="ref-vol">100</span>
9773&#x02013;9786. 10.1007/s00253-016-7847-3

 [<a href="https://pubmed.ncbi.nlm.nih.gov/27678114" ref="reftype=pubmed&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs00253-016-7847-3" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Appl.+Microbiol.+Biotechnol.&amp;title=Degradation+of+3-phenoxybenzoic+acid+by+a+filamentous+fungus+Aspergillus+oryzae+M-4+strain+with+self-protection+transformation.&amp;author=Y.+Zhu&amp;author=J.+Li&amp;author=K.+Yao&amp;author=N.+Zhao&amp;author=K.+Zhou&amp;volume=100&amp;publication_year=2016&amp;pages=9773-9786&amp;pmid=27678114&amp;doi=10.1007/s00253-016-7847-3&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11026547&amp;issue-id=454109&amp;journal-id=1526&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li></ul></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Frontiers in Microbiology</span> are provided here courtesy of <strong>Frontiers Media SA</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11026547/pdf/fmicb-15-1361335.pdf" class="int-view">PDF (2.5M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11026547/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11026547/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11026547" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11026547%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="AMC99U4Y7PLObObDQ48AglApOoAAhP0j28YHBrkS64R55a5OSFBL7ORWuNpEmzh3">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11026547%2F&amp;text=Whole-genome%20sequencing%20of%20Sphingobium%20baderi%20SC-1%20and%20identification%20of%20a%20crucial%203-phenoxybenzoic%20acid-degrading%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11026547%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11026547/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11026547/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38646623/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11026547/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38646623/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11026547/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11026547/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript" src="/core/mathjax/2.7.9/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC11026547/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
