
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs2.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>The levels of p53 govern the hierarchy of DNA damage tolerance pathway usage - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="939BE807DB0F4BC50000636DDF38399E.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="nar" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC11039994/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11039994/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Nucleic Acids Research" /><meta name="citation_title" content="The levels of p53 govern the hierarchy of DNA damage tolerance pathway usage" /><meta name="citation_author" content="Bryan A Casta&#x000f1;o" /><meta name="citation_author_institution" content="Department of Obstetrics and Gynecology, Ulm University, Ulm&#x000a0;89075, Germany" /><meta name="citation_author" content="Sabrina Schorer" /><meta name="citation_author_institution" content="Department of Obstetrics and Gynecology, Ulm University, Ulm&#x000a0;89075, Germany" /><meta name="citation_author" content="Yitian Guo" /><meta name="citation_author_institution" content="Department of Obstetrics and Gynecology, Ulm University, Ulm&#x000a0;89075, Germany" /><meta name="citation_author" content="Nicol&#x000e1;s L Calzetta" /><meta name="citation_author_institution" content="Fundacion Instituto Leloir, CONICET, 1405 Buenos Aires, Argentina" /><meta name="citation_author" content="Vanesa Gottifredi" /><meta name="citation_author_institution" content="Fundacion Instituto Leloir, CONICET, 1405 Buenos Aires, Argentina" /><meta name="citation_author" content="Lisa Wiesm&#x000fc;ller" /><meta name="citation_author_institution" content="Department of Obstetrics and Gynecology, Ulm University, Ulm&#x000a0;89075, Germany" /><meta name="citation_author" content="Stephanie Biber" /><meta name="citation_author_institution" content="Department of Obstetrics and Gynecology, Ulm University, Ulm&#x000a0;89075, Germany" /><meta name="citation_publication_date" content="2024/04/04" /><meta name="citation_issue" content="7" /><meta name="citation_volume" content="52" /><meta name="citation_firstpage" content="3740" /><meta name="citation_doi" content="10.1093/nar/gkae061" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC11039994/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC11039994/" /><meta name="citation_pmid" content="38321962" /><meta name="DC.Title" content="The levels of p53 govern the hierarchy of DNA damage tolerance pathway usage" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Oxford University Press" /><meta name="DC.Contributor" content="Bryan A Casta&#x000f1;o" /><meta name="DC.Contributor" content="Sabrina Schorer" /><meta name="DC.Contributor" content="Yitian Guo" /><meta name="DC.Contributor" content="Nicol&#x000e1;s L Calzetta" /><meta name="DC.Contributor" content="Vanesa Gottifredi" /><meta name="DC.Contributor" content="Lisa Wiesm&#x000fc;ller" /><meta name="DC.Contributor" content="Stephanie Biber" /><meta name="DC.Date" content="2024 Apr 24" /><meta name="DC.Identifier" content="10.1093/nar/gkae061" /><meta name="DC.Language" content="en" /><meta property="og:title" content="The levels of p53 govern the hierarchy of DNA damage tolerance pathway usage" /><meta property="og:type" content="article" /><meta property="og:description" content="It is well-established that, through canonical functions in transcription and DNA repair, the tumor suppressor p53 plays a central role in safeguarding cells from the consequences of DNA damage. Recent data retrieved in tumor and stem cells demonstrated ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11039994/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11039994/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/4/">Nucleic Acids Res</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/460879/">v.52(7); 2024 Apr 24</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11039994
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11039994/pdf/gkae061.pdf" class="int-view">PDF (3.4M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11039994/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11039994/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11039994" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11039994%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="I4esCK2IU6pJ9cLlf0abW6cVwaTFl4xigG0PZc8s0qynVco4kVopKfYzixqfufHq">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11039994%2F&amp;text=The%20levels%20of%20p53%20govern%20the%20hierarchy%20of%20DNA%20damage%20tolerance%20pathway%20usage" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11039994%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11039994/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11039994/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38321962/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11039994/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38321962/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11039994/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11039994/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/4/">Nucleic Acids Res</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/460879/">v.52(7); 2024 Apr 24</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11039994
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/narlogo.gif" alt="Logo of nar" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/nar" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Nucleic Acids Res.</span> 2024 Apr 24; 52(7): 3740–3760. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2024 Feb 7. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1093%2Fnar%2Fgkae061" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/nar/gkae061</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC11039994</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38321962">38321962</a></div></div></div><h1 class="content-title">The levels of p53 govern the hierarchy of DNA damage tolerance pathway usage</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Casta&#x000f1;o%20BA%5BAuthor%5D" class="affpopup" co-rid="_co_idm140659968777040" co-class="co-affbox">Bryan A Casta&#x000f1;o</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Schorer%20S%5BAuthor%5D" class="affpopup" co-rid="_co_idm140659968774272" co-class="co-affbox">Sabrina Schorer</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Guo%20Y%5BAuthor%5D" class="affpopup" co-rid="_co_idm140659919324896" co-class="co-affbox">Yitian Guo</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Calzetta%20NL%5BAuthor%5D" class="affpopup" co-rid="_co_idm140659919321392" co-class="co-affbox">Nicol&#x000e1;s L Calzetta</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gottifredi%20V%5BAuthor%5D" class="affpopup" co-rid="_co_idm140659933304592" co-class="co-affbox">Vanesa Gottifredi</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wiesm&#x000fc;ller%20L%5BAuthor%5D" class="affpopup" co-rid="_co_idm140659933302208" co-class="co-affbox">Lisa Wiesm&#x000fc;ller</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup></sup><sup></sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Biber%20S%5BAuthor%5D" class="affpopup" co-rid="_co_idm140659925134528" co-class="co-affbox">Stephanie Biber</a><sup></sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140659968777040"><h3 class="no_margin">Bryan A Casta&#x000f1;o</h3><p>
Department of Obstetrics and Gynecology, Ulm University, Ulm&#x000a0;89075, Germany</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Casta&#x000f1;o%20BA%5BAuthor%5D">Bryan A Casta&#x000f1;o</a></div></div><div id="_co_idm140659968774272"><h3 class="no_margin">Sabrina Schorer</h3><p>
Department of Obstetrics and Gynecology, Ulm University, Ulm&#x000a0;89075, Germany</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Schorer%20S%5BAuthor%5D">Sabrina Schorer</a></div></div><div id="_co_idm140659919324896"><h3 class="no_margin">Yitian Guo</h3><p>
Department of Obstetrics and Gynecology, Ulm University, Ulm&#x000a0;89075, Germany</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Guo%20Y%5BAuthor%5D">Yitian Guo</a></div></div><div id="_co_idm140659919321392"><h3 class="no_margin">Nicol&#x000e1;s L Calzetta</h3><p>
Fundacion Instituto Leloir, CONICET, 1405 Buenos Aires, Argentina</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Calzetta%20NL%5BAuthor%5D">Nicol&#x000e1;s L Calzetta</a></div></div><div id="_co_idm140659933304592"><h3 class="no_margin">Vanesa Gottifredi</h3><p>
Fundacion Instituto Leloir, CONICET, 1405 Buenos Aires, Argentina</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gottifredi%20V%5BAuthor%5D">Vanesa Gottifredi</a></div></div><div id="_co_idm140659933302208"><h3 class="no_margin">Lisa Wiesm&#x000fc;ller</h3><p>
Department of Obstetrics and Gynecology, Ulm University, Ulm&#x000a0;89075, Germany</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wiesm&#x000fc;ller%20L%5BAuthor%5D">Lisa Wiesm&#x000fc;ller</a></div></div><div id="_co_idm140659925134528"><h3 class="no_margin">Stephanie Biber</h3><p>
Department of Obstetrics and Gynecology, Ulm University, Ulm&#x000a0;89075, Germany</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Biber%20S%5BAuthor%5D">Stephanie Biber</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140659925775648_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140659925775648_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140659925775648_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140659925775648_ai" style="display:none"><div><span class="fm-affl">Bryan A Casta&#x000f1;o, </span><span class="fm-affl" id="aff-a.m.b.f.a.b"> 
Department of Obstetrics and Gynecology, Ulm University, Ulm&#x000a0;89075, Germany;</span></div><div><a href="#article-aaff-info">Contributor Information</a>.</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup>Corresponding author.</div><div id="COR1">To whom correspondence should be addressed. Tel: +49 731 50058800; Fax: +49 731 50058804; Email: <a href="mailto:dev@null" data-email="ed.mlu-inu@relleumseiw.asil" class="oemail">ed.mlu-inu@relleumseiw.asil</a></div><div id="FN1">The last two authors should be regarded as Joint Last Authors.</div></div><div class="fm-article-notes hide half_rhythm" id="idm140659925775648_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2023 Sep 25; Revised 2024 Jan 12; Accepted 2024 Feb 1.</div></div><div class="permissions half_rhythm hide" id="idm140659925775648_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; The Author(s) 2024. Published by Oxford University Press on behalf of Nucleic Acids Research.</div><div class="license half_rhythm">This is an Open Access article distributed under the terms of the Creative Commons Attribution-NonCommercial License (<a href="https://creativecommons.org/licenses/by-nc/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">https://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact <a href="mailto:dev@null" data-email="moc.puo@snoissimrep.slanruoj" class="oemail">moc.puo@snoissimrep.slanruoj</a></div></div></div><div id="pmclinksbox" class="links-box whole_rhythm" role="complementary" aria-label="Related or updated information about this article."><div><strong>This article has been corrected. </strong>See <a href="/pmc/articles/PMC11162763/">Nucleic Acids Res. 2024 June 10; 52(10): 6096</a>.</div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>gkae061_Supplemental_Files.</strong></div><div><a href="/pmc/articles/PMC11039994/bin/gkae061_supplemental_files.zip" data-ga-action="click_feat_suppl">gkae061_supplemental_files.zip</a><span style="color:gray"> (5.2M)</span></div><div class="small guid">GUID: A34C4BEE-C894-47FB-9057-DE6E3DE075D4</div></div></dd></dl><dl data-length="130" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first-last">The data sets generated during and/or analyzed during the current study are available from the corresponding authors upon request.</p></dd></dl></div><div id="abstract-a.m.b.r" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.m.b.rtitle" style="text-transform: uppercase;">Abstract</h2><!--article-meta--><div><p class="p p-first-last">It is well-established that, through canonical functions in transcription and DNA repair, the tumor suppressor p53 plays a central role in safeguarding cells from the consequences of DNA damage. Recent data retrieved in tumor and stem cells demonstrated that p53 also carries out non-canonical functions when interacting with the translesion synthesis (TLS) polymerase iota (POL&#x003b9;) at DNA replication forks. This protein complex triggers a DNA damage tolerance (DDT) mechanism controlling the DNA replication rate. Given that the levels of p53 trigger non-binary rheostat-like functions in response to stress or during differentiation, we explore the relevance of the p53 levels for its DDT functions at the fork. We show that subtle changes in p53 levels modulate the contribution of some DDT factors including POL&#x003b9;, POL&#x003b7;, POL&#x003b6;, REV1, PCNA, PRIMPOL, HLTF and ZRANB3 to the DNA replication rate. Our results suggest that the levels of p53 are central to coordinate the balance between DDT pathways including (i) fork-deceleration by the ZRANB3-mediated fork reversal factor, (ii) POL&#x003b9;-p53-mediated fork-slowing, (iii) POL&#x003b9;- and POL&#x003b7;-mediated TLS and (iv) PRIMPOL-mediated fork-acceleration. Collectively, our study reveals the relevance of p53 protein levels for the DDT pathway choice in replicating cells.</p></div></div><div id="abstract-a.m.b.s" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.m.b.stitle" style="text-transform: uppercase;">Graphical Abstract</h2><!--article-meta--><div><p class="p p-first-last">
<!--fig ft0--><!--fig mode=article f1--></p><div class="fig iconblock whole_rhythm" id="ga1" co-legend-rid="lgnd_ga1"><a href="/pmc/articles/PMC11039994/figure/ga1/" target="figure" rid-figpopup="ga1" rid-ob="ob-ga1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140659925491856"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11039994_gkae061figgra1.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkae061figgra1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11039994/bin/gkae061figgra1.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140659925491856"><a target="object" rel="noopener" href="/pmc/articles/PMC11039994/figure/ga1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_ga1"><div><a class="figpopup" href="/pmc/articles/PMC11039994/figure/ga1/" target="figure" rid-figpopup="ga1" rid-ob="ob-ga1">Graphical Abstract</a></div></div></div><p>
</p></div></div><div id="SEC1" class="tsec sec"><h2 class="head no_bottom_margin" id="SEC1title" style="text-transform: uppercase;">Introduction</h2><p class="p p-first">In recent years, there has been growing interest in the relevance that non-binary changes in the levels of key cellular components may have for the cell fate. Hence, by exploring the impact that subtle modulations in the levels of key proteins may have on biologically relevant outputs, we may better grasp the complexity in their regulation. Recently, the importance of p53 protein levels, rather than just its presence or absence has been highlighted by the analysis of stem cell differentiation programs and the dynamics of transcriptional responses after genotoxic treatments (<a href="#B1" rid="B1" class=" bibr popnode">1</a>,<a href="#B2" rid="B2" class=" bibr popnode">2</a>). p53, known as a tumor suppressor and guardian of genome integrity, plays a central role in protecting cells from DNA damage (<a href="#B3" rid="B3" class=" bibr popnode">3</a>,<a href="#B4" rid="B4" class=" bibr popnode">4</a>). So far, mostly canonical functions of transcriptionally regulating DNA repair, cell cycle and pro-apoptotic genes through sequence-specific DNA binding (<a href="#B5" rid="B5" class=" bibr popnode">5&#x02013;8</a>) have been causally linked with these genome stabilizing functions. Yet in parallel, accumulating evidence also carved out non-canonical roles of p53 in DNA repair, recombination and replication, operating independently of its canonical functions (<a href="#B9" rid="B9" class=" bibr popnode">9&#x02013;16</a>).</p><p>Introduction of damage into the DNA is a recurrent event, and the DNA repair efficiency is not sufficient to fully prevent the collision of replication forks with residual DNA lesions (<a href="#B17" rid="B17" class=" bibr popnode">17</a>). Once damaged DNA is encountered by replicative DNA polymerases, they stall as they are unable to accommodate damaged DNA within their active sites (<a href="#B18" rid="B18" class=" bibr popnode">18</a>). Evolution has developed multiple so-called DNA damage tolerance (DDT) mechanisms to override these obstacles (<a href="#B19" rid="B19" class=" bibr popnode">19</a>). A DDT pathway described in eukaryotes already in the eighties (<a href="#B20" rid="B20" class=" bibr popnode">20</a>) relies on specialized DNA polymerases with the unique ability to execute translesion DNA synthesis (TLS) across the damaged regions, guaranteeing the uninterrupted progression of DNA replication (<a href="#B18" rid="B18" class=" bibr popnode">18</a>). TLS-polymerases are diverse, subdivided in different families and play critical roles in TLS across specific types of DNA lesions as compared to the less specialized recombination-mediated bypass (<a href="#B21" rid="B21" class=" bibr popnode">21</a>). Within the Y-family of DNA polymerases, Polymerase eta (POL&#x003b7;) has traditionally been considered specialized on TLS across UV-induced DNA lesions (<a href="#B22" rid="B22" class=" bibr popnode">22</a>), while a broader role in overcoming replication stress was recognized later on (<a href="#B23" rid="B23" class=" bibr popnode">23</a>). The Y-family member REV1 is central in orchestrating the switch from replicative to TLS polymerases at replication forks encountering DNA lesions (<a href="#B24" rid="B24" class=" bibr popnode">24</a>,<a href="#B25" rid="B25" class=" bibr popnode">25</a>). REV1 also serves as loading platform during the TLS process itself. Similarly, the proliferating cell nuclear antigen (PCNA) recruits different TLS polymerases and increases the processivity of POL&#x003b7; and REV1 after PCNA mono-ubiquitination (mUb) (<a href="#B26" rid="B26" class=" bibr popnode">26</a>,<a href="#B27" rid="B27" class=" bibr popnode">27</a>). POL zeta (&#x003b6;), originating from the B-family and consisting of the catalytic REV3 and accessory REV7 subunits, extends tracks initiated by other specialized polymerases with a comparatively high processivity. Besides, PRIMPOL as a TLS and repriming polymerase promotes DNA replication-restart downstream of DNA lesions even during replication traverse of DNA interstrand crosslinks (<a href="#B28" rid="B28" class=" bibr popnode">28&#x02013;31</a>). The least processive and most error-prone Y-polymerase POL&#x003b9; was typically considered a backup enzyme of its binding partner POL&#x003b7; with a potential role in base excision repair (<a href="#B32" rid="B32" class=" bibr popnode">32</a>). Though POL&#x003b9; was discovered over two decades ago, has been linked with genomic instability in cancer cells and represents an interesting target for anti-cancer therapies (<a href="#B33" rid="B33" class=" bibr popnode">33&#x02013;35</a>), its cellular function is still not fully understood. However, more recent works provided evidence for POL&#x003b9;&#x02013;p53 complex formation at the replication fork introducing a new dimension in p53&#x02019;s genome-protecting functions (<a href="#B36" rid="B36" class=" bibr popnode">36&#x02013;38</a>). This interaction gives rise to a DDT pathway that permits the progress of replication forks by a recombination-mediated bypass of obstacles on the template strand. The participation of the POL&#x003b9;&#x02013;p53 complex in nascent DNA replication decelerates DNA synthesis in a manner depending on p53&#x02032;s exonuclease activity (<a href="#B36" rid="B36" class=" bibr popnode">36</a>) suggesting formation of a so-called idling complex, known to serve as a kinetic barrier to TLS through iterative nucleotide insertion and degradation steps (<a href="#B39" rid="B39" class=" bibr popnode">39</a>). The outcome of this iteration is the activation at ongoing forks of HLTF, an E3 ligase, promoting the accumulation of poly-ubiquitinated (pUb) PCNA (<a href="#B40" rid="B40" class=" bibr popnode">40</a>,<a href="#B41" rid="B41" class=" bibr popnode">41</a>), which is recognized by the translocase ZRANB3 capable of catalyzing fork reversal for recombination-mediated bypass (<a href="#B36" rid="B36" class=" bibr popnode">36</a>,<a href="#B42" rid="B42" class=" bibr popnode">42</a>), thereafter slowing nascent DNA synthesis (<a href="#B36" rid="B36" class=" bibr popnode">36</a>,<a href="#B38" rid="B38" class=" bibr popnode">38</a>,<a href="#B43" rid="B43" class=" bibr popnode">43</a>). This highly regulated DDT pathway not only protects the cells against DNA replication-associated damage (<a href="#B36" rid="B36" class=" bibr popnode">36</a>,<a href="#B38" rid="B38" class=" bibr popnode">38</a>,<a href="#B43" rid="B43" class=" bibr popnode">43</a>) but also improves the accuracy of DNA replication (<a href="#B44" rid="B44" class=" bibr popnode">44</a>), which provides a biological meaning for why this pathway predominates in human embryonal and adult stem cells as compared to their differentiated counterparts (<a href="#B38" rid="B38" class=" bibr popnode">38</a>). Moreover, using DNA fiber assays in combination with S1 nuclease, recent work demonstrated that inactivation of this DDT pathway by POL&#x003b9; depletion blocks PRIMPOL-dependent re-priming with consequent accumulation of post-replicative single-stranded DNA gaps (<a href="#B43" rid="B43" class=" bibr popnode">43</a>).</p><p class="p p-last">In the light of the relevance that the context such as differentiation status, species or strain may have for the POL&#x003b9;-p53 DDT pathway choice (<a href="#B38" rid="B38" class=" bibr popnode">38</a>,<a href="#B45" rid="B45" class=" bibr popnode">45</a>) and because of p53&#x02032;s remarkable capability to adapt to cellular insults through rapid changes in its level (<a href="#B1" rid="B1" class=" bibr popnode">1</a>), we sought to explore the impact that changes in p53 levels may have on the DDT pathway choices during unperturbed growth, i.e. without exogenous DNA damage treatment. Assessment of DNA replication dynamics at the single molecule level combined with the analyses of protein-protein interactions demonstrate that the p53 level is instrumental for the choice of at least four different DDT pathways. Our data suggest that at an optimum level p53 forms a complex with POL&#x003b9; to screen endogenous DNA replication impediments, while alternative mechanisms gain importance in the absence of such complex.</p></div><div id="SEC2" class="tsec sec"><h2 class="head no_bottom_margin" id="SEC2title" style="text-transform: uppercase;">Materials and methods</h2><div id="SEC2-1" class="sec sec-first"><h3 id="SEC2-1title">Biological resources and cell culture</h3><p class="p p-first-last">The Human non-small cell lung cancer (NSCLC) cell line H1299[tet-off p53(WT)] contains a chromosomally integrated tet-off system for controlled expression of either p53(WT) as described in Ahn <em>et&#x000a0;al.</em> (<a href="#B46" rid="B46" class=" bibr popnode">46</a>). It was cultivated in DMEM (Gibco by Thermo Fisher Scientific, Waltham, MA, USA), supplemented with 10% FBS (Merck Millipore Darmstadt), puromycin (2 &#x003bc;g/ml; Sigma-Aldrich by Merck), G418 (300 &#x003bc;g/ml, Sigma-Aldrich) and tetracycline (4500 ng/ml; Sigma-Aldrich). Before the start of experiments, cells were washed two times, split and cultivated in medium with indicated tetracycline doses for tetracycline-regulated expression of defined p53 levels. Twenty-four hours later, cells were washed again and re-incubated in medium with respective tetracycline-doses for an additional 24 h, after which experiments were performed. U2OS cells were cultured in DMEM Medium supplemented with 10% FBS and 1% penicillin&#x02013;streptomycin&#x02013;glutamine (Gibco by Thermo Fisher Scientific). Before experiments, U2OS cells were split and cultivated in medium without penicillin&#x02013;streptomycin&#x02013;glutamine treatment for 48 h. K562[HR-EGFP/3'EGFP] cells were cultivated in RPMI (Gibco by Thermo Fisher Scientific, Waltham, MA, USA), supplemented with 15% FBS (Merck Millipore Darmstadt) as previously in Hampp <em>et&#x000a0;al.</em>&#x000a0;(<a href="#B36" rid="B36" class=" bibr popnode">36</a>). Cell cultures were regularly tested for mycoplasma contamination by PCR.</p></div><div id="SEC2-2" class="sec"><h3 id="SEC2-2title">Viability assay</h3><p class="p p-first-last">For the assessment of cell viability, the colorimetric 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide (MTT) assay was used following manufacturer&#x02032;s instructions. The assay was performed 48 h after the respective treatment (hydroxyurea, doxorubicin). Cells were incubated in 0.5 mg/ml MTT for 3.5 h, lysed in 0.04 N HCl diluted in isopropanol and light absorbance was measured at 570 nm with an ELISA reader. Each data set was corrected by the corresponding solvent control value. Cell viability curves were plotted and IC50-values were calculated using GraphPad Prism 8 or 9 software (La Jolla, CA). Statistically significant differences between IC50-values of p53-expressing cell lines and control cell lines were calculated using Extra sum of squares <em>F</em>-test of log&#x000a0;IC50 values.</p></div><div id="SEC2-3" class="sec"><h3 id="SEC2-3title">Cell cycle distribution</h3><p class="p p-first-last">For the analysis of the distribution in cell-cycle phases, H1299[tet-off p53(WT)] cells were trypsinized, washed once with PBS, resuspended with 0.5 ml PBS and fixed drop-wise in 4.5 ml fixing solution (1:1-mixture of acetone and 80% ethanol; storage at &#x02212;20&#x000b0;C) during gentle mixing. After 15 min of incubation on ice, the fixed cells were washed twice with ice-cold PBS, resuspended in 200 ml propidium iodide (PI) staining solution (50 &#x003bc;g/ml RNase A was freshly added, 50 &#x003bc;g/ml PI from Sigma-Aldrich in PBS) and incubated for 30 min in the dark. Subsequently, the cell suspension was diluted with 100 ml 0.2% PBS&#x02013;EDTA and analyzed in a FACS Calibur flow cytometer (BD Biosciences, San Jose, CA, USA).</p></div><div id="SEC2-4" class="sec"><h3 id="SEC2-4title">Knockdown experiments</h3><p class="p p-first-last">HiPerFect (Qiagen) was used for siRNA transfection for silencing selected targets in H1299[tet-off p53(WT)] and U2OS cells. For siRNA-based gene silencing FlexiTube GeneSolution siRNAs (Qiagen, Hilden, Germany) with pools of four target gene specific siRNAs were used to silence p53 (GS7157) (<a href="#B36" rid="B36" class=" bibr popnode">36</a>), POL&#x003b9; (GS11201) (<a href="#B47" rid="B47" class=" bibr popnode">47</a>), POL&#x003b7; (GS5429), REV3L (GS5980), REV1 (GS51455), PRIMPOL (<a id="__tag_1147505588" class="tag_hotlink" href="/nuccore/GS201973" ref="/nuccore/GS201973"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"GS201973","term_id":"255858626","term_text":"GS201973"}}</span>GS201973</a>), HLTF (GS6596), SMARCAL1 (GS50485) and ZRANB3 (GS84083) (<a href="#B43" rid="B43" class=" bibr popnode">43</a>). For negative controls and fill-up to reach the same amount of RNA per transfection sample we engaged non-silencing (ns) siRNA (Qiagen).</p></div><div id="SEC2-5" class="sec"><h3 id="SEC2-5title">DNA fiber spreading assay</h3><p class="p p-first">To measure replication dynamics the DNA fiber spreading technique was performed as detailed in (<a href="#B36" rid="B36" class=" bibr popnode">36</a>). Briefly, CldU (5-Chloro-2-deoxyuridine, Sigma-Aldrich by Merck) was incorporated for 20 min in case of H1299[tet-off p53(WT)] and 10 min in case of U2OS cells. After washing, IdU (5-Iodo-2-deoxyuridine, Sigma-Aldrich by Merck) was incorporated in a 10 times higher concentration for another 20&#x000a0;and 30 min, respectively. RAD51 inhibitor (B02) was used at a final concentration of 25 &#x003bc;M during IdU incorporation for <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S5C,D</a>. Then, cells were washed, harvested, and resuspended in PBS. 2500 cells were put on a slide, lysed with 6 &#x003bc;L of 0.5% SDS, 200 mM Tris&#x02013;HCl, pH7.4, 50 mM EDTA, and incubated at room temperature for 6 min. Afterwards, they were tilted about 20&#x000b0; to allow DNA spreading via gravity. Subsequently, slides were covered with aluminium foil, air-dried for 6 min, fixed for 5 min with newly prepared 3:1 methanol:acetic acid, air-dried for 7 min, and stored in 70% ethanol at 4&#x000b0;C. Twenty-four hours later, the slides were processed for denaturation/deproteination in 2.5 N HCl for 1 h, followed by immunofluorescence staining. To measure ongoing replication, the track lengths of bi-colored forks were measured but for clarity, only the red tracks are shown. For fork asymmetry analysis, the ratio of longer versus shorter lengths of red IdU tracks within three-colored red-green-red replication structures was calculated and graphically plotted to assess replication fork stalling.</p><p class="p p-last">Immunofluorescence staining after DNA fiber spreading assays was performed after blocking with 5% bovine serum albumin (BSA) for 45 min. The following primary antibodies were used: anti-BrdU for detection of IdU (mouse, mAb, clone B44, #347580 BD Biosciences, Franklin Lakes, New Jersey, USA) and anti-BrdU for detection of CldU (rat, mAb, clone BU1/75 [ICR1] Abcam #ab6326 Cambridge, UK) and incubated at 4&#x000b0;C over-night. As secondary antibodies, AlexaFluor555 (anti-mouse, Invitrogen by Thermo Fisher Scientific, Waltham, MA, USA) or AlexaFluor488 (anti-rat, Invitrogen by Thermo Fisher Scientific, Waltham, Massachusetts, USA) were used (incubation time: 1 h, room temperature). Subsequently, for Figure <a href="/pmc/articles/PMC11039994/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure2D</span></span><span>2D</span></a> counterstaining of ssDNA was performed with primary antibody anti-ssDNA (mouse, mAb, AB_10805144 DSHB, Iowa, USA) and incubated at 4&#x000b0;C over-night. Secondary antibody AlexaFluor647 (anti-mouse, Invitrogen by Thermo Fisher Scientific, Waltham, MA, USA) was used (incubation time: 1 h, room temperature). DNA fibers were imaged with a Keyence BZ-9000 microscope (Keyence, Neu-Isenburg, Germany). DNA fiber track lengths were measured with BZ-II Analyzer software or using Fiji (ImageJ) software (ImageJ, U.S. National Institutes of Health, Bethesda, MD, USA).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC11039994/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140659968805568"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11039994_gkae061fig2.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkae061fig2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11039994/bin/gkae061fig2.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140659968805568"><a target="object" rel="noopener" href="/pmc/articles/PMC11039994/figure/F2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC11039994/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2.</a></div><!--caption a7--><div class="caption"><p>Analysis of DNA replication dynamics at different p53 levels. H1299[tet-off p53(WT)] cells were treated with different concentrations of tetracycline (Tet.) to downregulate p53 expression for 48 h together with protein samples for Figure <a href="/pmc/articles/PMC11039994/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure1.</span></span><span>1</span></a>. (<strong>A</strong>) On top, labeling protocol used for the fiber assay. Left, IdU track lengths (mean&#x000a0;&#x000b1;&#x000a0;SEM) from&#x000a0;&#x02265;300 fibers were evaluated in three independent experiments. Numbers above dots indicate mean values. Right, representative DNA fiber spreads depicted for each concentration. Note that the same fiber data are also presented in multi-color format, labeling data derived from an independent experiment in a different color each (extended Figure E6A). Scale bar: 2 &#x003bc;m. (<strong>B</strong>) Line plot connecting mean values&#x000a0;&#x000b1;&#x000a0;SEM from (A) to highlight the non-linear curve shape for IdU track lengths versus Tet. concentration. Numbers above indicate mean values. (<strong>C</strong>) Long/short track ratios from bicolored CldU and IdU fiber tracks measured in&#x000a0;(A). Data are shown in box plots (50%) including median (line), 95% confidence intervals. Numbers above dots indicate mean values. (D,&#x000a0;E) H1299[tet-off p53(WT)] and H1299 cells were treated with two concentrations of tetracycline (Tet.) to downregulate p53 expression for 48 h, followed by DNA fiber assay as in (A). Counterstaining with anti-ssDNA was included after IdU and CldU staining. (<strong>D</strong>) Representative images of samples from Figure <a href="/pmc/articles/PMC11039994/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure2E,</span></span><span>2E</span></a>, visualizing counterstaining of ssDNA (blue). Scale bar: 10 &#x003bc;m. (<strong>E</strong>) IdU track lengths (mean&#x000a0;&#x000b1;&#x000a0;SEM) from&#x000a0;&#x02265;200 fibers were evaluated in two independent experiments. Numbers above dots indicate mean values. ns, not significant. Of note, residual p53 expression in H1299[tet-off p53(WT)] at 4500 ng/ml Tet. (see <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S1A</a>) most likely caused shortening of replication tracks as compared to H1299 devoid of p53. Statistically significant differences were calculated by use of Kruskal&#x02013;Wallis-test followed by Dunn's multiple comparisons test (*<em>P</em>&#x0003c;&#x000a0;0.05; **<em>P</em>&#x0003c;&#x000a0;0.01; ****<em>P</em>&#x0003c;&#x000a0;0.0001). Brackets and asterisks are shown only for statistically significant differences.</p></div></div></div></div><div id="SEC2-6" class="sec"><h3 id="SEC2-6title">
<em>In situ</em> proximity ligation assay (PLA) and &#x003b3;H2AX foci analysis</h3><p class="p p-first">H1299[tet-off p53(WT)] cells were seeded on chamber slides (Falcon&#x000ae; Culture Slides 4-well, OMNILAB, Bremen, Germany) with a tetracycline-containing medium for regulated p53-expression. Twenty-four hours later, cells were washed and re-incubated in medium without/with tetracycline for an additional 24 h. Then, for Mitomycin C (MMC) treatment cells were washed, incubated with MMC (Sigma-Aldrich) containing medium at a final concentration of 3 &#x003bc;M or solvent H<sub>2</sub>O for 45 min, washed and re-incubated with fresh medium for an additional 3 h. Before fixing the cells with 3.6% formaldehyde in PBS (10 min), pre-extraction was performed for 1 min with pre-extraction buffer (300 mM Sucrose, 50 mM NaCl, 20 mM HEPES (pH 7.4), 3 mM MgCl<sub>2</sub>, 1 mM EDTA, 0.5% (v/v) Triton X-100 in water). Blocking was performed using Duolink Blocking solution (Sigma-Aldrich) for 1 h at 37&#x000b0;C. Then cells were double-labeled with the primary antibodies anti-DNA Polymerase &#x003b9; (mouse, mAb, OTI3B5, ThermoFisher #MA5-26503) and anti-p53pSer15 (rabbit, polyclonal, Cell Signalling #9284) or anti-DNA Polymerase &#x003b9; (rabbit, polyclonal, Thermofisher, PA5-29442) and anti-PCNA (mouse mAb PC10, Abcam #ab29). PLA-stain was then performed using Duolink&#x000ae; PLA (Proximity Ligation assay to detect protein interaction) technology from Sigma-Aldrich. Therefore, glass slides were stained with Duolink&#x000ae; In Situ PLA&#x000ae; Probe Anti-Rabbit PLUS (Affinity purified Donkey anti-Rabbit IgG (H&#x000a0;+&#x000a0;L), Sigma, DUO92002) and Duolink&#x000ae; In Situ PLA&#x000ae; Probe Anti-Mouse MINUS (Affinity purified Donkey anti-Mouse IgG (H&#x000a0;+&#x000a0;L), Sigma, DUO92004) for 1 h at 37&#x000b0;C. After washing the samples were then incubated with the Ligation-Ligase solution for 30 min at 37&#x000b0;C to hybridize oligonucleotides tagged on probes. After two short washing steps, the glass slides were incubated with the Amplification-Polymerase solution for 100 min at 37&#x000b0;C to amplify hybridized oligonucleotides and fluorescently label the amplification products. Ligation and Amplification were performed with Duolink&#x000ae; In Situ Detection Reagents Orange (Sigma, DUO92007). Then, glass slides were covered with coverslips using Duolink&#x000ae; In Situ Mounting Medium with DAPI (Sigma, DUO82040). Imaging was performed using the Keyence BZ-9000 microscope (Keyence, Cologne, Germany). Automated quantification of PLA-foci was carried out with BZ-II Analyser software.</p><p class="p p-last">For &#x003b3;H2AX foci analysis H1299[tet-off p53(WT)] cells were grown on coverslips. Pre-extraction was performed for 1 min in 300 mM Sucrose, 50 mM NaCl, 20 mM HEPES (pH 7.4), 3 mM MgCl<sub>2</sub>, 1 mM EDTA, 0.5% (v/v) Triton X-100 in water. Subsequently, cells were fixed in 3.7% (v<em>/</em>v) formaldehyde in PBS and permeabilized with 0.5% (v<em>/</em>v) Triton X-100 (Sigma-Aldrich by Merck) for 12 min at room temperature. After blocking unspecific binding sites with 5% (v<em>/</em>v) goat-serum in PBS for 1 h at room temperature, cells were immunostained for 1 h at 37&#x000b0;C using anti-phosphohistone H2A.X (Ser139, mouse, Merck Millipore, #05-636). Afterwards, slides were incubated for 45 min at 37&#x000b0;C with secondary antibodies AlexaFluor488 (anti-mouse from Invitrogen by ThermoFisher Scientific). Slides were mounted with Vectashield containing DAPI (Vector Laboratories, Burlingame, California, USA), followed by automated quantification of nuclear foci using a Keyence BZ-9000 microscope and BZ-II Analyser software. Intensity threshold and minimal focus size were maintained throughout one set of simultaneously treated and processed samples.</p></div><div id="SEC2-7" class="sec"><h3 id="SEC2-7title">Immunoprecipitation (IP) experiments</h3><p class="p p-first">For chromatin-prepared crosslink Co-IP (ChIP) of PCNA, cells were incubated for 12 min with cytoskeletal (CSK, 250&#x000a0;mM Sucrose, 25 mM KCl, 1 mM HEPES (pH 7.5), 1 mM EGTA, 1 mM MgCl<sub>2</sub>, H<sub>2</sub>O) buffer with protease-inhibitors (1 mM phenylmethylsulfonyl fluoride (PMSF), 1 mM Na<sub>3</sub>VO<sub>4</sub>, 0.1&#x000a0;mM dithiothreitol (DTT), protease inhibitor cocktail [Roche, Basel, Switzerland]) to isolate nuclear extracts. Cross-linking was performed by adding 1% MeOH-free formaldehyde (Thermo Fisher Scientific) in PBS for 10 min at room temperature. This reaction was stopped by adding 10 mM ice-cold glycine. Cells were lysed for 15 min on ice in RIPA buffer (10 mM Tris&#x02013;HCl [pH 7.5], 25 mM sodium fluoride, 20 mM sodium chloride, 1% (v/v) sodium-deoxycholate, 1% (v/v) Nonidet P40, 0.1% SDS, H<sub>2</sub>O; freshly added inhibitors: 1 mM PMSF, 1 mM Na<sub>3</sub>VO<sub>4</sub>, 0.1 mM DTT, protease inhibitor cocktail [Roche, Basel, Switzerland]). Thereafter cells were harvested and sonified in a sonification water bath Sonorex (Bandelin electronic GmbH &#x00026; Co. KG, Berlin, Germany) at high power three-times each 7.5 min: 30 s on, 30 s off. After centrifugation, the supernatant was used for immunoprecipitation of PCNA (anti-PCNA, mouse, ab29, Abcam, Cambridge, UK). Isotype control was performed by using mouse IgG (sc-2025, Santa Cruz Biotechnology, Dallas, TX, USA). After preclearing protein extracts with Protein G Sepharose beads (PGS, GE Healthcare, Munich, Germany) overnight, extracts were transferred to PGS beads with pre-bound antibodies and incubated for another 4 h at 4&#x000b0;C.</p><p class="p p-last">For immunoprecipitation of POL&#x003b9;, cells were lysed in 50 mM Tris, pH8; 150 mM NaCl; 1% NP40; protease inhibitor cocktail (Roche). These whole cell extracts were used for immunoprecipitation engaging monoclonal mouse POL&#x003b9;-antibody (sc-101026 Santa Cruz Biotechnology, Dallas, TX,&#x000a0;USA) or as control mouse IgG (Santa Cruz). After spin-down, precipitated proteins were washed five times with lysis buffer and dissolved in SDS-PAGE sample buffer.</p></div><div id="SEC2-8" class="sec"><h3 id="SEC2-8title">Western blotting</h3><p class="p p-first-last">Western blot analysis was performed as previously described (<a href="#B36" rid="B36" class=" bibr popnode">36</a>). For detection of pUb PCNA cells were treated with PBS-Triton (1%) for 1 min, washed two times with PBS and harvested with Laemmli buffer 6&#x000d7;&#x000a0;after scraping. For immunostaining the following antibodies were used: anti-MDM2 (mouse mAb, MABE281 Merck Millipore, Darmstadt, Germany), anti-MDMX (rabbit pAb, #04-1555 Sigma-Aldrich, Massachusetts, United States), anti-p21 (mouse mAb SX118, #556430 BD Biosciences, Franklin Lakes, USA), anti-p53 (mouse mAb DO1, #554293 BD Biosciences), anti-PCNA (mouse mAb PC10, ab29 Abcam, Cambridge, UK), anti-ubiquityl-PCNA (rabbit mAb Lys164 D5C7P, #13439 Cell Signaling, Danvers, USA), anti-Polymerase iota (rabbit pAb, A301-304A Bethyl, Montgomery, USA), POL&#x003b7; (rabbit pAb, NB100-60424 Novus, Nordenstadt, Germany), PRIMPOL (rabbit pAb, 29824-1-AP Proteintech, Cambridge, UK), HLTF (rabbit mAb, ab183042 Abcam), anti-Replication Protein A (RPA) (mouse mAb, NA18 Calbiochem, Darmstadt, Germany) and anti-phospho-RPA 32 (S33) (rabbit pAb, A300-264A Bethyl Laboratories). To control loading of proteins, the membranes were prestained with Ponceau Red (Sigma) or reincubated with anti-&#x003b1;-Actin antibody (mouse mAb I-19, sc-47778 Santa Cruz), anti-GAPDH (mouse mAb, 0411, sc-47724 Santa Cruz), anti-Tubulin (mouse mAb DM1A, ab7291-100 Abcam) or anti-Vinculin (mouse mAb, sc-25336 Santa Cruz). Chemiluminescence detection and quantification of protein band intensities were carried out in the linear range using ImageLab software on a ChemiDocMP System (BioRad Laboratories, Munich, Germany). Values for the protein of interest were individually corrected with values for the loading control.</p></div><div id="SEC2-9" class="sec"><h3 id="SEC2-9title">Recombination measurements</h3><p class="p p-first-last">Recombination frequencies were determined via Flow Cytometry (FACSCaliburTM, BD Biosciences) as described (<a href="#B36" rid="B36" class=" bibr popnode">36</a>). Briefly, K562 cells with chromosomally integrated recombination substrate (K562[HR-EGFP/3'EGFP]) were transfected via electroporation with expression plasmids for p53(WT) or empty vector (E.V.) (<a href="#B36" rid="B36" class=" bibr popnode">36</a>). Each measurement was paralleled by the same co-transfection including a wild-type EGFP (wtEGFP) control plasmid instead of pBS plasmid for determination of transfection efficiency. Recombination frequencies were measured 72 h after transfection by quantification of EGFP-positive cells within the live cell population (SSC/FSC gate) and were individually corrected for transfection efficiencies.</p></div><div id="SEC2-10" class="sec"><h3 id="SEC2-10title">ELISA</h3><p class="p p-first-last">ELISA analysis was performed following the protocol of the manufacturer (ab171571 Abcam, Cambridge, UK). ELISA reader (Tecan Sunrise Microplate Photometer ST/TC, Crailsheim, Germany) at 450 nm was used to detect OD and calculate p53 concentrations.</p></div><div id="SEC2-11" class="sec"><h3 id="SEC2-11title">Reverse transcription and quantitative PCR (RT-qPCR)</h3><p class="p p-first-last">Total RNA was isolated by use of RNeasy Plus Mini Kit (Qiagen) according to the manufacturer&#x02032;s instructions. Reverse transcription was performed using the QuantiTec Reverse Transcription Kit (Qiagen) following the instructions for 1 &#x003bc;g RNA. Quantitative PCR was performed as described in Eberle <em>et&#x000a0;al.</em> (<a href="#B48" rid="B48" class=" bibr popnode">48</a>) using SensiFAST Probe LO-ROX mix (2&#x000d7;) from the SensiFAST Probe LO-ROX Kit (Bioline, London, UK). The PCR was performed on the qTower (Analytik Jena, Jena, Germany) by using the program 2 min 95&#x000b0;C and 40 cycles of 10 s 95&#x000b0;C and 20 s of 60&#x000b0;C. Controls were PPIA HEX, RPLP0 HEX, YWHAZ HEX (all three BioRad Laboratories, San Diego, USA). The genes of interest were analyzed with REV3L Fam, REV1 Fam, SMARCAL1 Fam, ZRANB3 Fam (BioRad Laboratories, San Diego, USA).</p></div><div id="SEC2-12" class="sec"><h3 id="SEC2-12title">Statistical analyses</h3><p class="p p-first-last">Graphic presentation of data was performed using GraphPad Prism 8 or 9 software (La Jolla, CA). For calculation of statistically significant differences in DNA fiber spreading analysis, PLA and immunofluorescence experiments, either Dunns-multiple comparison test or Kruskal&#x02013;Wallis-test followed by two-tailed Mann&#x02013;Whitney <em>U</em> test were used. For calculation of statistically significant differences in western blot analysis, Friedmann test followed by Wilcoxon-matched pair signed-rank test or Dunns-multiple comparison test was used. *<em>P</em>&#x0003c;&#x000a0;0.05, **<em>P</em>&#x0003c;&#x000a0;0.01, ***<em>P</em>&#x0003c;&#x000a0;0.001, ****<em>P</em>&#x0003c;&#x000a0;0.0001 (two-sided).</p></div><div id="SEC2-13" class="sec sec-last"><h3 id="SEC2-13title">Reagents</h3><p class="p p-first-last">See <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S1</a>.</p></div></div><div id="SEC3" class="tsec sec"><h2 class="head no_bottom_margin" id="SEC3title" style="text-transform: uppercase;">Results</h2><div id="SEC3-1" class="sec sec-first"><h3 id="SEC3-1title">Differential effect of p53 level on protein expression and cell cycle control</h3><p class="p p-first">The speed of DNA replication is affected by wild-type p53, an effect that is lost in oncogenic hotspot mutants and cancer-associated alternative isoforms of p53 (<a href="#B36" rid="B36" class=" bibr popnode">36</a>,<a href="#B37" rid="B37" class=" bibr popnode">37</a>,<a href="#B49" rid="B49" class=" bibr popnode">49</a>,<a href="#B50" rid="B50" class=" bibr popnode">50</a>). Given that human p53 differentially affects the speed of nascent DNA synthesis in stem cells and their differentiated counterparts expressing low and high p53 levels, respectively (<a href="#B38" rid="B38" class=" bibr popnode">38</a>), we wondered whether p53 levels are the source of the change in replication speed. Therefore, we moved into a simplified model that allowed us to selectively focus on the modulation of the p53 levels by means of the expression of a tetracycline (tet)-repressed wild-type p53 in the non-small cell lung carcinoma H1299 (H1299[tet-off p53(WT)]) cells enabling analyses of the consequences of gradual changes of p53 levels (<a href="#B46" rid="B46" class=" bibr popnode">46</a>). As a p53-negative control, resembling culture maintenance conditions, we applied the maximum concentration of tetracycline (4500 ng/ml) minimizing p53 expression down to a level precluding its detection in western blots except in overexposed images when compared to p53-negative H1299 (Figure <a href="/pmc/articles/PMC11039994/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1A,</span></span><span>1A</span></a>, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S1A</a>). For maximum expression, still in the physiological range (<a href="#B51" rid="B51" class=" bibr popnode">51</a>), tetracycline was removed (0 ng/ml). According to Figure <a href="/pmc/articles/PMC11039994/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure1A</span></span><span>1A</span></a> and&#x000a0;<a href="/pmc/articles/PMC11039994/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>B</span></a>, we additionally engaged tetracycline concentrations of 0.25, 0.5, 1, 2 and 4 ng/ml to cover the intermediate range of p53 levels in western blot and ELISA experiments. Expression of the major p53 target gene product p21 (<a href="#B52" rid="B52" class=" bibr popnode">52</a>) closely followed p53 level changes (Figure <a href="/pmc/articles/PMC11039994/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1C,</span></span><span>1C</span></a>, <a href="/pmc/articles/PMC11039994/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,D)</span></span><span>D</span></a>) resulting in a highly significant correlation (<em>r</em><sub>s</sub>=&#x000a0;0.96; <em>P</em>=&#x000a0;0.003). TLS-polymerase &#x003b7; (POL&#x003b7;), a key component of DDT mechanisms and a target gene product of p53 (<a href="#B53" rid="B53" class=" bibr popnode">53</a>) showed a significant decrease only at the highest tetracycline concentration, i.e. at the minimum p53 level. Similarly, levels of the p53 target MDM2 (<a href="#B54" rid="B54" class=" bibr popnode">54</a>) were significantly reduced when comparing high to minimum p53 expression levels (0 and 0.25 versus 4500 ng/ml). MDMX was also reduced when comparing maximum to minimum p53 levels (0 versus 4500 ng/ml) (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S1B</a>). Regarding expression of POL&#x003b9;, the polymerase loading factor PCNA or the repriming DNA polymerase PRIMPOL no significant level changes were observed (Figure <a href="/pmc/articles/PMC11039994/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1C,</span></span><span>1C</span></a>, <a href="/pmc/articles/PMC11039994/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,D,</span></span><span>D</span></a>, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S1B</a>). Such results are aligned with the fact that neither POL&#x003b9; nor PCNA nor PRIMPOL are known as p53 target gene products. Nevertheless, it is important to mention that POL&#x003b9; forms a mutually stabilizing complex with p53, when triggering fork remodeling (<a href="#B36" rid="B36" class=" bibr popnode">36</a>,<a href="#B37" rid="B37" class=" bibr popnode">37</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC11039994/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140659933256928"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11039994_gkae061fig1.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkae061fig1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11039994/bin/gkae061fig1.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140659933256928"><a target="object" rel="noopener" href="/pmc/articles/PMC11039994/figure/F1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC11039994/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1.</a></div><!--caption a7--><div class="caption"><p>Cell-based system for analysis of p53&#x02032;s biochemical activities as a function of gradual level changes. H1299[tet-off p53(WT)] cells were treated with seven different concentrations of tetracycline to downregulate p53 expression. Forty-eight hours after treatment cells were harvested for protein extraction followed by western blot analysis and ELISA. (<strong>A</strong>) Representative western blot analysis of p53 levels and &#x003b2;-Actin as loading control. (<strong>B</strong>) Left, quantification of p53 band intensities from nine independent experiments (mean&#x000a0;&#x000b1;&#x000a0;SEM), which was performed by use of shortly exposed (short e.) images, normalized to the corresponding &#x000df;-Actin band intensities and expressed as fold changes relative to the tetracycline-free control (0 ng/ml). At the maximum tetracycline (Tet.) concentration (4500 ng/ml) p53 was detectable in western blot images following long exposure (long e.) only. Right, ELISA of 7 independent experiments (mean&#x000a0;&#x000b1;&#x000a0;SEM), expressed as fold changes relative to the tetracycline-free control (0 ng/ml). Friedmann test followed by Dunns-multiple comparison test was used (*<em>P</em>&#x0003c;&#x000a0;0.05; **<em>P</em>&#x0003c;&#x000a0;0.01; ***<em>P</em>&#x0003c;&#x000a0;0.001; ****<em>P</em>&#x0003c;&#x000a0;0.0001). (<strong>C</strong>) Protein levels of the target gene products p21 and POL&#x003b7; and of the p53 complex partners POL&#x003b9; and PCNA are shown in representative blots. (<strong>D</strong>) Quantification of band intensities from protein levels shown in Figure <a href="/pmc/articles/PMC11039994/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure1C</span></span><span>1C</span></a> (p21: <em>N</em>&#x000a0;=&#x000a0;4, POL&#x003b7;: <em>N</em>&#x000a0;=&#x000a0;5, POL&#x003b9;: <em>N</em>&#x000a0;=&#x000a0;6, PCNA: <em>N</em>&#x000a0;=&#x000a0;3 independent experiments; mean&#x000a0;&#x000b1;&#x000a0;SEM), normalized to the corresponding housekeeper band intensities and expressed as fold changes relative to the tetracycline-free control (0 ng/ml). For western blot analysis, Friedmann test followed by Dunns-multiple comparison test was used (*<em>P</em>&#x0003c;&#x000a0;0.05; **<em>P</em>&#x0003c;&#x000a0;0.01; ****<em>P</em>&#x0003c;&#x000a0;0.0001). For reasons of clarity brackets and asterisks are shown only for statistically significant differences.</p></div></div></div><p class="p p-last">To explore how these gradual changes in p53 expression impacted on the cell cycle distribution, we performed DNA content analysis by FACS (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S1C</a>). Similar accumulation of on average 70% of the cells in G1-phase was observed for tetracycline concentrations&#x000a0;&#x02264;4 ng/ml but a decline down to 59% upon exposure to 4500 ng/ml. Concomitantly, the S-phase fraction rose from &#x0223c;10% for tetracycline concentrations&#x000a0;&#x02264;2 ng/ml up to 14% and 22% for 4&#x000a0;and 4500 ng/ml, respectively. Moreover, the G2-phase percentage was significantly reduced above tetracycline concentrations of 0.5 ng/ml. Percentages of cells with subG1-DNA content were negligibly low (&#x0003c;1%) and did not differ significantly. We conclude that G1 arrest was maintained as long as p53 expression was readily detectable, while S- and particularly G2-phase changes were noticeable already when the p53 level was moderately altered.</p></div><div id="SEC3-2" class="sec"><h3 id="SEC3-2title">The specific level of p53 determines DNA replication dynamics</h3><p class="p p-first">To investigate the dynamics of nascent DNA synthesis as a function of gradual p53 expression changes, we performed DNA fiber spreading assays monitoring sequential incorporation of 5-chloro-2-deoxyuridine (CldU: green) and 5-iodo-2-deoxyuridine (IdU: red) in H1299[tet-off p53(WT)] cells exposed to different tetracycline concentrations (<a href="#B36" rid="B36" class=" bibr popnode">36</a>). The lengthening of the fiber tracks roughly correlated with the dose of tetracycline, so that there was an inverse correlation between p53 levels and track lengths (Figure <a href="/pmc/articles/PMC11039994/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2A).</span></span><span>2A</span></a>). Plotting the mean IdU track lengths revealed an almost linear augmentation of tracks, except for an intermediate peak and subsequent dip between tetracycline concentrations of 0.25&#x000a0;and 2 ng/ml suggesting the participation of more than a single DNA synthesis regulatory component (Figure <a href="/pmc/articles/PMC11039994/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2B).</span></span><span>2B</span></a>). To explore if stalling of DNA replication forks played a role in the observed DNA replication dynamics, we calculated the ratios between long and short tracks from bicolored green&#x02013;red fibers (<a href="#B55" rid="B55" class=" bibr popnode">55</a>,<a href="#B56" rid="B56" class=" bibr popnode">56</a>). Despite observing significant changes in replication speed, we did not find any significant differences between track ratios (Figure <a href="/pmc/articles/PMC11039994/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2C).</span></span><span>2C</span></a>). To reveal fork stalling by an alternative method (<a href="#B57" rid="B57" class=" bibr popnode">57</a>), we quantified asymmetries in the progression of sister forks emanating from the same three-colored red&#x02013;green&#x02013;red origins. Again, no significant differences were detected between the means obtained for different tetracycline concentrations, largely excluding the possibility that DNA track shortening occurs due to replication fork stalling (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S2A</a>). As previously reported (<a href="#B58" rid="B58" class=" bibr popnode">58</a>,<a href="#B59" rid="B59" class=" bibr popnode">59</a>), a shortening of replication tracks might also be the consequence of increased origin firing. When we calculated the fraction of three-colored origins among the total number of replication structures (<a href="#B55" rid="B55" class=" bibr popnode">55</a>), we found significantly elevated origin firing only upon complete withdrawal of tetracycline (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S2B</a>). From this we conclude that replication track changes in the presence of different tetracycline concentrations (&#x02265;0.25 ng/ml) cannot be explained by altered origin firing. Additionally, we verified the integrity of DNA fibers by DNA counterstaining (Figure <a href="/pmc/articles/PMC11039994/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2D)</span></span><span>2D</span></a>) and excluded an effect of tetracycline on track lengths by performing DNA fiber assays in H1299 devoid of any p53 (Figure <a href="/pmc/articles/PMC11039994/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2E,</span></span><span>2E</span></a>, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S1A</a>).</p><p class="p p-last">Next, we wanted to determine if the modulation of the levels of p53 in a cellular model that expresses endogenous p53 also reveals changes in the replication tracks. We therefore titrated endogenous p53 in U2OS cells by using increasing concentrations of siRNA targeting the <em>TP53</em> transcript [si(p53)] (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S2C</a>). As seen with H1299[tet-off p53(WT)], DNA replication tracks were longest in cells with the lowest p53 levels [8 nM and 10 nM si(p53)] and were shorter in cells with maximum p53 expression [0 nM si(p53) plus 10 nM nsRNA] (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S2D</a>). Interestingly, in cells with intermediate p53 levels we noticed that track lengths shortened until 4 nM and lengthened again until 10 nM si(p53). As seen in H1299[tet-off p53(WT)] cells, fork stalling was unable to explain shortening of tracks in U2OS as well (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S2E</a>). Therefore, p53 modulates nascent DNA track lengths in a level-dependent manner both in cells expressing endogenous and exogenous p53. To understand whether observed changes in replication dynamics at distinct p53 levels impact on the accumulation of DNA damage, we proceeded with the analysis of the damage markers &#x003b3;H2AX and P-RPA and downstream effects on cell viability.</p></div><div id="SEC3-3" class="sec"><h3 id="SEC3-3title">Impact of p53 level on DNA replication stress and cell viability</h3><p class="p p-first">In human cells the selection of DDT pathways impacts on the level of replication stress and is crucial to promote cell survival (<a href="#B27" rid="B27" class=" bibr popnode">27</a>,<a href="#B60" rid="B60" class=" bibr popnode">60</a>,<a href="#B61" rid="B61" class=" bibr popnode">61</a>). &#x003b3;H2AX, a phosphorylated form of the histone H2AX at Ser139, serves as a marker of DNA damage at sites of replication stress (<a href="#B62" rid="B62" class=" bibr popnode">62</a>). To examine the impact of p53 levels on DNA damage responses, we analysed &#x003b3;H2AX foci formation focusing on four concentrations of tetracycline, covering a range from the maximum to minimum p53 expression levels (0 ng/ml and 4500 ng/ml) including two intermediate concentrations, namely intermediate high (0.25 ng/ml) and intermediate low (2 ng/ml) (Figure <a href="/pmc/articles/PMC11039994/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3A).</span></span><span>3A</span></a>). Significant differences were observed when comparing samples with no p53 (4500 ng/ml) with those expressing high p53 levels (0 and 0.25 ng/ml), namely a decline of nuclear &#x003b3;H2AX foci numbers from 48 to on average 35. Additionally, we evaluated accumulation of RPA32 phosphorylated on Ser33 (P-RPA), another marker of replication stress (<a href="#B63" rid="B63" class=" bibr popnode">63</a>). Consistent with &#x003b3;H2AX foci formation, P-RPA relative to total RPA levels increased in samples with low p53 expression (2 ng/ml and 4500 ng/ml tetracycline; 3.3- and 2.6-fold), providing further evidence for replication stress (Figure <a href="/pmc/articles/PMC11039994/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3B).</span></span><span>3B</span></a>). Collectively, our findings suggest that reduced p53 levels lead to imperfect bypass of replication barriers, resulting in elevated replication stress, which prolongs S-phase (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S1C</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC11039994/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140659972087632"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11039994_gkae061fig3.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkae061fig3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11039994/bin/gkae061fig3.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140659972087632"><a target="object" rel="noopener" href="/pmc/articles/PMC11039994/figure/F3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC11039994/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3.</a></div><!--caption a7--><div class="caption"><p>Replication stress responses at different p53 levels. H1299[tet-off p53(WT)] cells were treated with four different concentrations of tetracycline (Tet.) to downregulate p53 expression. Fourty-eight hours after treatment, cells were fixed for (<strong>A</strong>) quantitative immunofluorescence microscopy, (<strong>B</strong>) harvested for protein extraction or (<strong>C</strong>) viability assay (MTT) was performed. (A) Left, analysis of &#x003b3;H2AX foci/nucleus. Statistically significant differences were calculated by use of Kruskal&#x02013;Wallis-test followed by Dunn's multiple comparisons test (*<em>P</em>&#x0003c;&#x000a0;0.05; **<em>P</em>&#x0003c;&#x000a0;0.01). Brackets and asterisks are shown only for statistically significant differences. Right, representative images from immunofluorescence microscopy depicting green &#x003b3;H2AX foci in DAPI-stained nuclei. Scale bar: 5 &#x003bc;m. (<strong>B</strong>) Representative western Blot from five independent experiments, showing P-RPA, i.e. RPA32 phosphorylated on Ser33 as compared to total RPA level. Numbers above western blots indicate fold level changes of p-RPA band intensity normalized to RPA32 and relative to 0 ng/ml Tet. value. (<strong>C</strong>) Viability of cells cultivated in presence of different Tet. concentrations and treated with increasing hydroxyurea concentrations for 48 h. Left, viability curves. Right, IC50 values and <em>P</em>-values calculated from the results of six measurements. (<strong>D</strong>) Viability of cells cultivated in presence of different Tet. concentrations and treated with increasing doxorubicin concentrations for 48 h. Left, viability curves. Right, IC50 values and <em>P</em>-values calculated from the results of 6 measurements. Statistically significant differences between IC50 values were defined using the extra-sum-of-square <em>F</em> test of log IC50 values.</p></div></div></div><p class="p p-last">Hydroxyurea (HU) is a potent inhibitor of ribonucleotide reductase, a key enzyme involved in the synthesis of deoxyribonucleotide triphosphates (dNTPs). Consequently, HU administration leads to a significant depletion of available dNTPs, subsequently resulting in a deceleration of the DNA replication process (<a href="#B64" rid="B64" class=" bibr popnode">64</a>). To assess cell viability under enforced replication stress conditions, we performed MTT assay 48 h after HU treatment. p53 expression caused a significant increase in cell viability (compare the samples with high p53 levels&#x02014;0 and 0.25 ng/ml, and the lowest p53 level, 4500 ng/ml, Figure <a href="/pmc/articles/PMC11039994/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure3C).</span></span><span>3C</span></a>). When treating with the chemotherapeutic agent doxorubicin, which interferes with topoisomerase action generating DNA double-strand breaks (<a href="#B65" rid="B65" class=" bibr popnode">65</a>,<a href="#B66" rid="B66" class=" bibr popnode">66</a>), the opposite phenotype was observed. In this case, p53 levels at 0, 0.25 and 2 ng/ml promoted cell death when compared to samples expressing the lowest p53 level at 4500 ng/ml (Figure <a href="/pmc/articles/PMC11039994/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3D).</span></span><span>3D</span></a>). These findings are reminiscent of the concept that p53 acts in DNA replication to promote survival, while canonical functions in transcription and apoptosis are induced in response to severe DNA damage such as DSBs (<a href="#B50" rid="B50" class=" bibr popnode">50</a>). Our results support the idea that the cell fate is determined not solely by the presence or absence of p53, but rather by the levels of p53 expression. This delicate balance influences cell destiny, activating checkpoint control or even cell death, depending on the extent and nature of DNA damage in combination with the specific p53 level. In the light of these findings and the previous chapter showing that DNA replication fork stalling (Figure <a href="/pmc/articles/PMC11039994/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2C,</span></span><span>2C</span></a>, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S2A,&#x000a0;E</a>) or changes in the frequency of origin firing (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S2B</a>) are unlikely to fully explain changes in DNA track length, we moved forward to explore the relevance of p53 levels for the DDT pathway choice that controls track lengthening.</p></div><div id="SEC3-4" class="sec"><h3 id="SEC3-4title">POL&#x003b9; silencing abrogates the changes in track length caused by the expression of increasing levels of p53</h3><p class="p p-first">Previously, we demonstrated regulatory roles of p53 in DNA replication dynamics in a manner depending on the differentiation status of human cells (<a href="#B36" rid="B36" class=" bibr popnode">36&#x02013;38</a>). Slow-down of DNA synthesis in stem cells was demonstrated to rely on the physical interactions between POL&#x003b9; and p53 (<a href="#B36" rid="B36" class=" bibr popnode">36</a>,<a href="#B38" rid="B38" class=" bibr popnode">38</a>). Conversely, p53-dependent DNA replication speed-up was shown to involve association of POL&#x003b9; and PCNA, but not p53, with the replication fork (<a href="#B38" rid="B38" class=" bibr popnode">38</a>). Given the relevance of POL&#x003b9; for the p53-dependent control of DNA synthesis, we downregulated POL&#x003b9; in H1299[tet-off p53(WT)] samples with increasing expression of p53, at the four tetracycline concentrations 0, 0.25, 2 and 4500 ng/ml. Indeed, POL&#x003b9; knockdown caused elongation of fiber tracks at the two higher p53 levels (0, 0.25 ng/ml tetracycline). To the contrary, POL&#x003b9; knockdown triggered fiber shortening at 2 ng/ml tetracycline (intermediate low). Finally, no effect of POL&#x003b9; silencing was detected at 4500 ng/ml tetracycline (Figure <a href="/pmc/articles/PMC11039994/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure4A,</span></span><span>4A</span></a>, <a href="/pmc/articles/PMC11039994/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,B)</span></span><span>B</span></a>) which efficiently suppressed p53 expression. Regardless of the tetracycline concentration significant changes of track ratios and fork asymmetries were not detected, largely excluding involvement of fork stalling in the changes of track lengths caused by p53 expression (Figure <a href="/pmc/articles/PMC11039994/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure4C,</span></span><span>4C</span></a>, <a href="/pmc/articles/PMC11039994/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,D).</span></span><span>D</span></a>). Assaying cell viability at four different p53 levels in H1299[tet-off p53(WT)] cells revealed lowest viability at its minimum level (4500 ng/ml tetracycline) (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S3A</a>) matching the increase in replication stress signals seen before (Figure <a href="/pmc/articles/PMC11039994/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3A,</span></span><span>3A</span></a>, <a href="/pmc/articles/PMC11039994/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,B).</span></span><span>B</span></a>). Knockdown of POL&#x003b9; reduced cell viabilities under all four p53 expression conditions by up to 39% relative to the corresponding transfection-control sample (nsRNA), demonstrating a protective role of POL&#x003b9;.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC11039994/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140659922852032"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11039994_gkae061fig4.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkae061fig4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11039994/bin/gkae061fig4.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140659922852032"><a target="object" rel="noopener" href="/pmc/articles/PMC11039994/figure/F4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC11039994/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4.</a></div><!--caption a7--><div class="caption"><p>Effect of POL&#x003b9; depletion on DNA replication dynamics at different p53 levels. H1299[tet-off p53(WT)] cells were treated with four different concentrations of tetracycline (Tet.) to downregulate p53 expression. Forty-eight hours after transfection and treatment, DNA fiber spreading assay was performed together with protein extraction for western blot analysis. (<strong>A</strong>) On top, labeling protocol used for the fiber assay. IdU track lengths (mean&#x000a0;&#x000b1;&#x000a0;SEM) from&#x000a0;&#x02265;300 fibers were evaluated in three independent experiments. Note that the same fiber data are also presented in multi-color format, labeling data derived from an independent experiment in a different color each (Extended Fig. E6B). (<strong>B</strong>) Representative western blot analysis of POL&#x003b9; knockdown, with &#x003b2;-Actin used as loading control. (<strong>C</strong>) Long/short track ratios from bicolored CldU and IdU fiber tracks measured in (A). Data are shown in box plots (50%) including median (line), 95% confidence intervals. (<strong>D</strong>) Fork asymmetry analysis. On top: schematic overview of fork asymmetry analysis. Track lengths of IdU-labeled tracks departing from the same CldU-track within tricolored red-green-red fibers were measured and plotted as the ratio long/short tracks from&#x000a0;&#x02265;&#x000a0;60 fibers in three independent experiments (box plots 50% with median line, 95% confidence intervals). Statistically significant differences were calculated using Kruskal-Wallis-test followed by Dunn's multiple comparisons test (*<em>P</em>&#x0003c;&#x000a0;0.05; ****<em>P</em>&#x0003c;&#x000a0;0.0001). Brackets and asterisks are shown only for statistically significant differences. KD, knockdown; Tet., tetracycline.</p></div></div></div><p class="p p-last">Strikingly, when we examined the role of POL&#x003b9; under conditions of differential expression of endogenous p53 in U2OS, we observed almost the same contribution of POL&#x003b9; to track lengths as judged from the changes caused by its knockdown. At the two highest p53 levels [0&#x000a0;and 4 nM si(p53)] POL&#x003b9; knockdown resulted in track lengthening, while in cells with intermediate low p53 levels [8 nM si(p53)] POL&#x003b9; silencing had the opposite effect (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S3B, C</a>). However, at maximum downregulation of endogenous p53 [20 nM si(p53)], we observed replication track elongation after POL&#x003b9; knockdown. POL&#x003b9; knockdown caused stalling at the two higher levels of p53 [0&#x000a0;and 4 nM si(p53)], and the opposite with intermediate low p53 [8 nM si(p53)] (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S3D</a>). As stalling was associated with longer tracks after POL&#x003b9; silencing at 0&#x000a0;and 4 nM si(p53), it might be caused by the detrimental effect of excessively high replication speed in this cellular model system (<a href="#B67" rid="B67" class=" bibr popnode">67</a>,<a href="#B68" rid="B68" class=" bibr popnode">68</a>). From these observations, we concluded that, albeit different DDT hierarchies could be established in different cells expressing p53, the levels of p53 and the expression of POL&#x003b9; rule on the DDT pathway choice.</p></div><div id="SEC3-5" class="sec"><h3 id="SEC3-5title">The level of p53 controls formation of specific POL&#x003b9; complexes</h3><p class="p p-first">In order to understand the underlying mechanism, we investigated whether specific p53 levels differentially modulate formation of protein complexes involved in the observed replication phenotypes. To this end, we analysed the <em>in situ</em> association of POL&#x003b9; with the serine 15 phosphorylated form of p53, p53(pSer15), known to be active in the regulation of DNA replication (<a href="#B36" rid="B36" class=" bibr popnode">36&#x02013;38</a>). When performing proximity ligation assay (PLA) for POL&#x003b9; and p53(pSer15), we observed that only cells expressing intermediate high levels of p53 (0.25 ng/ml tetracycline) showed a significant augmentation of PLA foci numbers over the 4500 ng/ml tetracycline sample (Figure <a href="/pmc/articles/PMC11039994/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5A).</span></span><span>5A</span></a>). To verify that PLA foci reflect POL&#x003b9;-p53 complex formation, we performed co-immunoprecipitation experiments. Indeed, p53 was prominently detected in POL&#x003b9; co-precipitates under POL&#x003b9;-p53 PLA-positive conditions (0.25 ng/ml tetracycline), and to a lesser extent at maximum p53 expression without tetracycline (Figure <a href="/pmc/articles/PMC11039994/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5B).</span></span><span>5B</span></a>). Hence, the POL&#x003b9;-p53 complexes may trigger replication fork deceleration at intermediate high p53 level (0.25 ng/ml tetracycline) with a less prominent contribution at the highest p53 level (0 ng/ml tetracycline).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC11039994/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140659933773360"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11039994_gkae061fig5.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkae061fig5.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11039994/bin/gkae061fig5.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140659933773360"><a target="object" rel="noopener" href="/pmc/articles/PMC11039994/figure/F5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC11039994/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5.</a></div><!--caption a7--><div class="caption"><p>Complex formation of POL&#x003b9; at different p53 levels. Association studies of POL&#x003b9; in H1299[tet-off p53(WT)] cells were performed 48 h after treatment with four different tetracycline (Tet.) concentrations. In PLA, exposure to Mitomycin C for 45 min was performed to enhance complex formation, followed by 3 h of recovery time (<a href="#B36" rid="B36" class=" bibr popnode">36</a>). (<strong>A</strong>) Monitoring complex formation of POL&#x003b9; and p53pSer15 by PLA. Left, 200 nuclei in two independent experiments were scored (mean&#x000a0;&#x000b1;&#x000a0;SEM) and normalized to the mean of the 0.25 Tet. sample each. Right, representative microscopic images are shown depicting POL&#x003b9;-p53pSer15 PLA-foci in red within DAPI-stained nuclei in blue. (<strong>B</strong>) Representative western blot analysis of POL&#x003b9; and p53 after immunoprecipitation (IP) using antibodies directed against POL&#x003b9;. (<strong>C</strong>) PLA analysis of complex formation between POL&#x003b9; and PCNA. Left, two hundred nuclei in two independent experiments were scored (mean&#x000a0;&#x000b1;&#x000a0;SEM) and normalized to the mean of the 2 Tet. samples each. Right, representative PLA images are shown depicting POL&#x003b9;-PCNA PLA-foci. (<strong>D</strong>) Representative western blot analysis of POL&#x003b9; and PCNA after chromatin immunoprecipitation (CHIP) using antibodies directed against PCNA. (<strong>E</strong>) On top, representative western blot showing pUb-, mUb- PCNA and total PCNA levels. Below, quantification of pUb/mUb band intensities relative to 0 ng/ml Tet; mean&#x000a0;&#x000b1;&#x000a0;SEM values from eight independent experiments. (<strong>F</strong>) H1299[tet-off p53(WT)] cells expressing p53 (0 ng/ml Tet.) were transfected with si(HLTF). Fourty-eight hours after transfection cells were harvested for protein extraction and western blot analysis. On top, representative western blot showing HLTF, pUb-, mUb-PCNA and total PCNA levels. Below, quantification of pUb/mUb band intensities relative to control nsRNA; mean&#x000a0;&#x000b1;&#x000a0;SEM values from three independent experiments. For PLA analysis, statistically significant differences were calculated through Kruskal&#x02013;Wallis-test followed by Dunn's multiple comparisons test (**<em>P</em>&#x0003c;&#x000a0;0.01; ***<em>P</em>&#x0003c;&#x000a0;0.001; ****<em>P</em>&#x0003c;&#x000a0;0.0001). For western blot analysis, Friedmann test followed by Wilcoxon-matched pair signed-rank test was used (*<em>P</em>&#x0003c;&#x000a0;0.05). Brackets and asterisks are shown only for statistically significant differences. Scale bars: 5 &#x003bc;m.</p></div></div></div><p>Secondly, we examined proximity between POL&#x003b9; and the replication clamp PCNA. We observed a two-fold increase in POL&#x003b9;-PCNA PLA foci at intermediate low p53 level (2 ng/ml tetracycline) compared with all other samples (Figure <a href="/pmc/articles/PMC11039994/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5C).</span></span><span>5C</span></a>). A smaller, 1.6-fold increase of POL&#x003b9;-PCNA PLA foci was also detectable in samples at minimum (4500 ng/ml) versus maximum (0 ng/ml) p53 expression. POL&#x003b9; co-precipitated with PCNA in the samples with intermediate low p53 level (2 ng/ml tetracycline) and was undetectable in pull-downs from the other samples (Figure <a href="/pmc/articles/PMC11039994/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5D).</span></span><span>5D</span></a>). Hence, POL&#x003b9;-dependent track lengthening observed at intermediate low p53 (2 ng/ml tetracycline; see Figure <a href="/pmc/articles/PMC11039994/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure4A)</span></span><span>4A</span></a>) correlates with increased POL&#x003b9;-PCNA complex formation but not POL&#x003b9;&#x02013;p53 complex formation.</p><p class="p p-last">POL&#x003b9;&#x02013;p53 complexes were previously demonstrated to trigger a replication decelerating DDT pathway involving recombinative fork remodeling (<a href="#B36" rid="B36" class=" bibr popnode">36</a>), while POL&#x003b9;&#x02013;PCNA complexes were found to play a role in the replication accelerating DDT pathway TLS (<a href="#B38" rid="B38" class=" bibr popnode">38</a>). Therefore, we wondered whether the DDT pathway switchboard, PCNA (<a href="#B69" rid="B69" class=" bibr popnode">69</a>,<a href="#B70" rid="B70" class=" bibr popnode">70</a>), was differentially ubiquitinated under conditions of differential p53 expression. Remarkably, we detected a significant 2- to 3-fold rise of polyubiquitinated/monoubiquitinated (pUb/mUb) PCNA when comparing 0 ng/ml tetracycline to the increasing levels of tetracycline (Figure <a href="/pmc/articles/PMC11039994/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5E).</span></span><span>5E</span></a>). Specificity of pUb bands was verified by knockdown of the E3 ligase HLTF (Figure <a href="/pmc/articles/PMC11039994/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5F).</span></span><span>5F</span></a>). This result suggested that at its highest level p53 may promote deceleration of replication forks by engaging DDT events triggered by PCNA pUb. Altogether, these results suggest that p53 may promote different DDT pathways to overcome the barriers in a manner that depends on the p53 levels, predominantly promoting pUb-triggered DDT at the highest p53 levels (0 ng/ml tetracycline), POL&#x003b9;-p53-mediated DDT at intermediate high p53 levels (0.25 ng/ml tetracycline) and POL&#x003b9;&#x02013;PCNA-mediated DDT at intermediate low p53 levels (2 ng/ml tetracycline).</p></div><div id="SEC3-6" class="sec"><h3 id="SEC3-6title">p53 levels regulate the role of different DDT effector molecules on nascent DNA synthesis</h3><p class="p p-first">Given that POL&#x003b9; negatively or positively affects track lengths in a manner that depends on p53 levels, we explored the contribution of key factors involved in TLS, fork reversal and repriming under conditions of increasing p53 levels. We firstly performed knockdown of POL&#x003b7;, a transcriptional p53 target (<a href="#B53" rid="B53" class=" bibr popnode">53</a>) with a well characterized role in TLS at DNA lesions including cyclobutane pyrimidine dimers (UV-induced CPDs) and in the duplication of difficult to replicate regions such as common fragile sites (CFS) (<a href="#B71" rid="B71" class=" bibr popnode">71&#x02013;73</a>). Secondly and thirdly, we downregulated REV3L, the catalytic subunit of POL&#x003b6;, which is highly proficient in primer extension after TLS polymerases&#x02032; activation, as well as REV1, a scaffold for the other TLS polymerases and a dCMP inserter itself (<a href="#B74" rid="B74" class=" bibr popnode">74</a>,<a href="#B75" rid="B75" class=" bibr popnode">75</a>). Knockdown of each of these TLS polymerases (Figure <a href="/pmc/articles/PMC11039994/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6A)</span></span><span>6A</span></a>) induced shortening of fiber track lengths at the minimum level of p53 (4500 ng/ml tetracycline) suggesting they all participate in track elongation in these samples. POL&#x003b7; and REV3L knockdown also modified track lengths at intermediate low p53 level (2 ng/ml) while REV3L but not POL&#x003b7; knockdown shortened tracks at intermediate high p53 level (0.25 ng/ml) (Figure <a href="/pmc/articles/PMC11039994/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6A).</span></span><span>6A</span></a>). Neither POL&#x003b7; nor REV3L contributed to nascent DNA synthesis when p53 levels were highest (0 ng/ml). Under these experimental conditions, neither track ratio nor fork asymmetry analyses revealed significant differences between knockdown and control values (Figure <a href="/pmc/articles/PMC11039994/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6B,</span></span><span>6B</span></a>, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S4A</a>). These results suggested that knockdown-induced track shortening was due to a slow-down of fork speed rather than stalling.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F6" co-legend-rid="lgnd_F6"><a href="/pmc/articles/PMC11039994/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140659967729136"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11039994_gkae061fig6.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkae061fig6.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11039994/bin/gkae061fig6.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140659967729136"><a target="object" rel="noopener" href="/pmc/articles/PMC11039994/figure/F6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F6"><div><a class="figpopup" href="/pmc/articles/PMC11039994/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6">Figure 6.</a></div><!--caption a7--><div class="caption"><p>Effects of key DDT factor depletions on DNA replication dynamics at different p53 levels. H1299[tet-off p53(WT)] cells were treated with four different concentrations of tetracycline (Tet.) to downregulate p53 expression, and transfected with different siRNAs for knockdown (KD) of POL&#x003b7;, REV3L, REV1, PRIMPOL, HLTF and ZRANB3. Fourty-eight hours after transfection and treatment DNA fiber spreading assay was performed together with protein extraction for western blot analysis. (<strong>A</strong>) IdU track lengths (mean&#x000a0;&#x000b1;&#x000a0;SEM) from&#x000a0;&#x02265;300 fibers were evaluated in at least two independent experiments. The scheme on top outlines the labeling protocol used for the fiber assay. Below graphs for KD of POL&#x003b7;, PRIMPOL and HLTF, representative western blots are shown to validate KD. Commercial antibodies directed against REV3L, REV1 and ZRANB3 did not enable specific detection of the proteins in western blots, why KD was analysed by RT-qPCR. Values were normalized to means of control samples for each Tet. concentration in each experiment. (<strong>B</strong>) Long/short track ratios from bicolored CldU and IdU fiber tracks measured in (A). Data are shown in box plots (50%) including median (line), 95% confidence intervals. Statistically significant differences were calculated by use of Kruskal&#x02013;Wallis-test followed by Dunn's multiple comparisons test (*<em>P</em>&#x0003c;&#x000a0;0.05; **<em>P</em>&#x0003c;&#x000a0;0.01; ****<em>P</em>&#x0003c;&#x000a0;0.0001). Brackets and asterisks are shown only for statistically significant differences. Arrows mark significant differences in track length (blue, increase; red decrease).</p></div></div></div><p>Distinct from POL&#x003b7; and REV3L, downregulation of REV1 revealed a positive contribution to track lengthening, which was observed when p53 was markedly expressed (0&#x02013;2 ng/ml tetracycline). REV1 knockdown-induced lengthening of fiber tracks, was coupled with elevated track ratios. These results reveal that p53 expressed at a distinct level (intermediate low, 2 ng/ml) enables opposed contributions of TLS polymerases to nascent DNA synthesis, promoting POL&#x003b7;-, REV3L-dependent lengthening and REV1-dependent shortening of the replication tracks, a complexity that was not anticipated by the current understanding of the DDT by TLS polymerases. Conversely, under conditions of minimum p53 expression, all three TLS polymerases (POL&#x003b7;, REV3L and REV1) canonically contribute to DNA synthesis by accelerating the DNA replication speed.</p><p>Next, we assessed the contribution of the repriming DNA polymerase, PRIMPOL, which has been reported to play an important role in bypassing secondary DNA structures particularly during unperturbed DNA replication (<a href="#B76" rid="B76" class=" bibr popnode">76</a>,<a href="#B77" rid="B77" class=" bibr popnode">77</a>). Knockdown of PRIMPOL unveiled a dual role depending on the levels of p53 expression: At high p53 levels (0 and 0.25 ng/ml) PRIMPOL knockdown caused lengthening of replication tracks (Figure <a href="/pmc/articles/PMC11039994/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6A).</span></span><span>6A</span></a>). Notably, evidence of fork stalling was revealed when p53 levels were intermediate high (0.25 ng/ml) and PRIMPOL was downregulated (Figure <a href="/pmc/articles/PMC11039994/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6B,</span></span><span>6B</span></a>, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S4A</a>). Under conditions of minimum p53 expression (4500 ng/ml) PRIMPOL knockdown shortened track lengths. In conclusion, as for other specialized DNA polymerases, p53 levels regulate whether PRIMPOL promotes or opposes nascent DNA track lengthening.</p><p>Given that previous evidence showed that the fork reversal inducing enzymes HLTF and ZRANB3 are functionally associated with p53 during nascent DNA synthesis (<a href="#B36" rid="B36" class=" bibr popnode">36</a>,<a href="#B38" rid="B38" class=" bibr popnode">38</a>), we examined their impact on track lengths as a function of the p53 level. Knockdown of HLTF and ZRANB3 resulted in the same pattern of fiber track changes (Figure <a href="/pmc/articles/PMC11039994/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6A),</span></span><span>6A</span></a>), namely increased track lengths at the maximum p53 level (0 ng/ml) and track shortening at intermediate low and minimum p53 levels (2 ng/ml, 4500 ng/ml). When focusing on track ratios, in the case of ZRANB3 knockdown, elevated values were revealed when p53 levels were intermediate low or when p53 was minimally expressed (2, 4500 ng/ml). In the case of HLTF knockdown, values were elevated when p53 levels were at their maximum (0 ng/ml). This trend was also seen after HLTF knockdown at intermediate p53 levels (0.25 ng/ml: <em>P</em>=&#x000a0;0.0643 and 2 ng/ml, <em>P</em>=&#x000a0;0.0540) without fully reaching statistical significance (Figure <a href="/pmc/articles/PMC11039994/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6B,</span></span><span>6B</span></a>, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S4A</a>). For comparison, knockdown of the translocase SMARCAL1 did not influence the DNA fiber track length at high p53 levels (0, 0.25 ng/ml) (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S4B, C</a>). SMARCAL1 has a specific polarity for reversing forks with RPA-coated leading strands gaps <em>in vitro</em>, while ZRANB3 was reported not to show a preference for regressing either lagging or leading strand gap forks (<a href="#B78" rid="B78" class=" bibr popnode">78</a>,<a href="#B79" rid="B79" class=" bibr popnode">79</a>), arguing against a substrate specificity for RPA coated leading strand gaps at these p53 levels.</p><p class="p p-last">In order to investigate the impact of the p53 level on fork remodeling, we previously made use of a reporter cell line enabling measurements of replication-associated recombination under unperturbed growth conditions (<a href="#B36" rid="B36" class=" bibr popnode">36</a>). Consequently, we used these p53-negative K562 reporter cells to measure recombination as a function of p53 expression (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S5A, B</a>). The results show that recombination increases with increasing p53 level, which supports the concept of elevated fork remodeling at maximum p53 level in H1299[tet-off p53(WT)] cells (0 ng/ml tetracycline). Moreover, we performed DNA fiber spreading assays with and without inhibition of RAD51 (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S5, D</a>), which is known to promote different types of fork remodeling ranging from fork reversal to homologous recombination (<a href="#B80" rid="B80" class=" bibr popnode">80</a>). As was expected from the involvement of PRIMPOL and TLS POLs in track elongation at lower p53 levels (2, 4500 ng/ml), inhibition of RAD51 did not affect track lengths under these conditions. However, at higher p53 levels (0, 0.25 ng/ml), inhibition of RAD51 further reduced track lengths due to fork stalling, which indicates a protective role at replication forks during remodeling. Taken together, the fork reversal factors HLTF and ZRANB3 prevented excessive replication speed at maximum p53 expression, most likely through their canonical role in fork reversal. At low p53 expression levels, when the basal DNA replication speed is high, the depletion of these fork reversal factors slows down DNA replication, most likely due to fork stalling.</p></div><div id="SEC3-7" class="sec sec-last"><h3 id="SEC3-7title">Double depletion experiments unveil functional crosstalk between POL&#x003b9;, POL&#x003b7; and PRIMPOL at different levels of p53</h3><p class="p p-first">In recent years, several studies (<a href="#B81" rid="B81" class=" bibr popnode">81&#x02013;84</a>) found pieces of evidence proposing a hierarchical framework for selection of DDT pathways. To elucidate the role that the changes in p53 levels have on such an interplay between key DDT factors, we performed double-knockdown experiments targeting POL&#x003b9;, POL&#x003b7; and PRIMPOL in combination with the modulation of p53 levels.</p><p>Combined downregulation of POL&#x003b9; and PRIMPOL resulted in track lengthening at the two highest p53 levels (0 and 0.25 ng/ml tetracycline) (Figure <a href="/pmc/articles/PMC11039994/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure7A),</span></span><span>7A</span></a>), a result that correlates with those obtained after single knockdown of either POL&#x003b9; or PRIMPOL. At lower p53 levels, the individual contributions of each knockdown were revealed. Specifically, there was a reduction in track lengths at intermediate low p53 levels (2 ng/ml tetracycline), which is reminiscent of the effect seen in the individual POL&#x003b9; knockdown. At 4500 ng/ml tetracycline, track lengths were similar to those from the individual PRIMPOL knockdown phenotype. Fork stalling was only noticed at maximum p53 level (0 ng/ml tetracycline) suggesting that, when p53 levels are the highest forks decelerate in the presence of POL&#x003b9; and PRIMPOL but they stall when both proteins are absent (Figure <a href="/pmc/articles/PMC11039994/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure7B).</span></span><span>7B</span></a>). These results suggest that POL&#x003b9; and PRIMPOL can individually contribute to nascent DNA synthesis with no indications for interference of one with another.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F7" co-legend-rid="lgnd_F7"><a href="/pmc/articles/PMC11039994/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140659925785728"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11039994_gkae061fig7.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkae061fig7.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11039994/bin/gkae061fig7.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140659925785728"><a target="object" rel="noopener" href="/pmc/articles/PMC11039994/figure/F7/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F7"><div><a class="figpopup" href="/pmc/articles/PMC11039994/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7">Figure 7.</a></div><!--caption a7--><div class="caption"><p>Co-depletion of key DDT factors. H1299[tet-off p53(WT)] cells were treated with tetracycline (Tet.) as indicated and transfected with siRNAs directed against POL&#x003b9; and PRIMPOL, POL&#x003b9; and POL&#x003b7; or POL&#x003b7; and PRIMPOL. DNA fiber spreading assay was performed 48 h post-transfection and treatment together with protein extraction for western blot analysis. (<strong>A</strong>) IdU track lengths (mean&#x000a0;&#x000b1;&#x000a0;SEM) from&#x000a0;&#x02265;&#x000a0;300 fibers were evaluated in at least two independent experiments. The labeling protocol used for the fiber assay is shown on top. Below each graph, representative western blots are shown to validate KD. (<strong>B</strong>) Long/short track ratios from bicolored CldU and IdU fiber tracks measured in (A). Statistically significant differences were calculated by use of Kruskal&#x02013;Wallis-test followed by Dunn's multiple comparisons test (**<em>P</em>&#x0003c;&#x000a0;0.01; ***<em>P</em>&#x0003c;&#x000a0;0.001; ****<em>P</em>&#x0003c;&#x000a0;0.0001). Arrows mark significant differences in track length (blue, increase; red decrease).</p></div></div></div><p>When simultaneously depleting POL&#x003b9; and POL&#x003b7;, a combination of the individual knockdown effects was observed, namely elongation at high p53 levels (0 and 0.25 ng/ml tetracycline), as seen for the individual POL&#x003b9; knockdown, and shortening at intermediate low p53 level (2 ng/ml tetracycline) as seen for individual knockdowns of both POL&#x003b9; or POL&#x003b7;. At the minimum level of p53 (4500 ng/ml tetracycline) the double POL&#x003b9; and POL&#x003b7; knockdown recapitulated the single POL&#x003b7; knockdown (Figure <a href="/pmc/articles/PMC11039994/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure7A).</span></span><span>7A</span></a>). There was no evidence of fork stalling in these scenarios (Figure <a href="/pmc/articles/PMC11039994/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure7B).</span></span><span>7B</span></a>). In conclusion, also in this case of the combination of POL&#x003b9; and POL&#x003b7; knockdown, the individual contributions of each DNA polymerase to track shortening or lengthening did not cause any interference with the function of the other.</p><p>Lastly, we performed DNA fiber analysis after POL&#x003b7; and PRIMPOL downregulation. Surprisingly, nascent DNA track shortening was revealed in all samples regardless of the levels of p53 expression (Figure <a href="/pmc/articles/PMC11039994/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure7A).</span></span><span>7A</span></a>). Track ratios remained unaffected, indicating that tracks shorten as a consequence of decelerated fork progression rather than because of replication fork stalling (Figure <a href="/pmc/articles/PMC11039994/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure7B).</span></span><span>7B</span></a>). While it is possible that the reduced replication speed documented at the two lowest p53 levels is the consequence of the phenotypes observed after individual knockdown of POL&#x003b7; and PRIMPOL, the deceleration observed at elevated p53 levels is puzzling as POL&#x003b7; alone did not have an impact on fiber length, whereas the depletion of PRIMPOL alone resulted in elongation of fiber tracks. From these observations, we conclude that when p53 is highly expressed and PRIMPOL absent track elongation is dependent on POL&#x003b7;.</p><p class="p p-last">Collectively, these findings unveil an unpredicted complexity in the contribution of DDT factors to nascent DNA elongation as a function of the p53 level. While the double knockdown experiments in Figure <a href="/pmc/articles/PMC11039994/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure7</span></span><span>7</span></a> suggest that specialized DNA polymerases can provide independent contributions to nascent DNA elongation, expression of p53 can cause a diametrically opposed effect on nascent DNA synthesis by the same factors. In fact, our results reveal that the levels of p53 are central in establishing the hierarchies between DDTs pathways, favouring fork reversal mediated DDT when p53 is at its highest level, POL&#x003b9;-p53 complex-mediated DDT when p53 levels are intermediate high and TLS-mediated DDT when p53 levels are low.</p></div></div><div id="SEC4" class="tsec sec"><h2 class="head no_bottom_margin" id="SEC4title" style="text-transform: uppercase;">Discussion</h2><p class="p p-first">We have previously shown that wild-type p53 together with POL&#x003b9; triggers a DDT pathway characterized by a replication slow-down, which ensures genome integrity and mediates a pro-survival effect (<a href="#B36" rid="B36" class=" bibr popnode">36</a>,<a href="#B38" rid="B38" class=" bibr popnode">38</a>). In this study, we sought to understand the impact of the p53 expression level on the DDT pathway selection. Our findings reveal the coexistence of different phenotypes whereby even minor fluctuations in p53 levels can shift the equilibrium towards emergence of one or the other mechanism in such a continuum. The highest p53 levels increasingly favor the POL&#x003b9;-p53 pathway via stages of idling and downstream HLTF and ZRANB3 activities. Conversely, at intermediate low to minimum p53 expression, a delicate equilibrium between multiple DDT mechanisms appears to govern the choice favoring TLS and repriming, respectively (Figure <a href="/pmc/articles/PMC11039994/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure8</span></span><span>8</span></a> and Graphical Abstract). These results underscore the critical role of p53 levels in determining the DDT pathway choice providing valuable insights into the molecular basis of pathway selection.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F8" co-legend-rid="lgnd_F8"><a href="/pmc/articles/PMC11039994/figure/F8/" target="figure" rid-figpopup="F8" rid-ob="ob-F8"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140659923124816"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11039994_gkae061fig8.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkae061fig8.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11039994/bin/gkae061fig8.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140659923124816"><a target="object" rel="noopener" href="/pmc/articles/PMC11039994/figure/F8/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F8"><div><a class="figpopup" href="/pmc/articles/PMC11039994/figure/F8/" target="figure" rid-figpopup="F8" rid-ob="ob-F8">Figure 8.</a></div><!--caption a7--><div class="caption"><p>Model for p53 level-dependent DDT pathway choices. Identified by key molecular components as outlined in the text. Briefly, high p53 levels (0 and 0.25 ng/ml) slow down DNA replication by complex formation with POL&#x003b9; and idling (<a href="#B36" rid="B36" class=" bibr popnode">36</a>) triggering HLTF- and ZRANB3- mediated fork remodeling at the highest p53 level (0 ng/ml). Low p53 levels (2.5 ng/ml) unleash fast DNA synthesis through TLS activities e.g. by POL&#x003b9; or POL&#x003b7;. At minimal p53 levels (4500 ng/ml) and reduced level of its target gene product POL&#x003b7;, PRIMPOL plays a key role in elongating DNA replication tracks. Such unrestrained repriming entails DNA replication stress and sensitizes cells to exogenous replication stress. An additional role of PRIMPOL in fork slow-down at high p53 levels (0 and 0.25 ng/ml) may involve repression of fast TLS such as initiated by POL&#x003b7; and/or gap formation promoting recombination-mediated template switch (<a href="#B92" rid="B92" class=" bibr popnode">92</a>). REV1 as TLS-POL loader, REV3L as TLS extender of TLS-POLs (<a href="#B27" rid="B27" class=" bibr popnode">27</a>) and gap filler post-PRIMPOL (<a href="#B83" rid="B83" class=" bibr popnode">83</a>,<a href="#B84" rid="B84" class=" bibr popnode">84</a>) as well as HLTF promoting mono- and poly-ubiquitination of PCNA (<a href="#B41" rid="B41" class=" bibr popnode">41</a>,<a href="#B90" rid="B90" class=" bibr popnode">90</a>,<a href="#B96" rid="B96" class=" bibr popnode">96</a>,<a href="#B97" rid="B97" class=" bibr popnode">97</a>) are epistatic with multiple activities.</p></div></div></div><div id="SEC4-1" class="sec"><h3 id="SEC4-1title">The p53 level controls DNA replication dynamics through the formation of different POL&#x003b9; complexes</h3><p class="p p-first">In 2016 (<a href="#B36" rid="B36" class=" bibr popnode">36</a>), we proposed that p53 forms an idling complex with POL&#x003b9;, slowing down replication and, thus, generating a window of time for PCNA to accumulate pUb marks, which trigger an error-free fork remodeling mechanism (<a href="#B85" rid="B85" class=" bibr popnode">85</a>). This model was validated in subsequent studies, not only in various cancer cell lines (<a href="#B37" rid="B37" class=" bibr popnode">37</a>,<a href="#B49" rid="B49" class=" bibr popnode">49</a>) but also in different types of primary human stem cells (<a href="#B38" rid="B38" class=" bibr popnode">38</a>). In our present investigation, we provide evidence for an even much more complex hierarchy of DDT pathway usage under the control of p53, unveiled by gradual changes of the p53 level in H1299 and U2OS cancer cell models by tet-off-controlled expression of exogenous p53 or siRNA-mediated depletion of endogenous p53. We observed that high levels of p53 impaired DNA elongation in a POL&#x003b9;-dependent manner in both cell lines. Interestingly, intermediate low levels of p53 appear to enable POL&#x003b9; to function independently of the p53 complex, possibly in its previously characterized TLS polymerase role. Under these conditions, POL&#x003b9; depletion impairs DNA replication elongation in both cell lines. Yet, we notice that results obtained after p53 induction do not fully overlap with results obtained after downregulation of endogenous p53. At minimum p53 level, replication speed becomes independent of POL&#x003b9; in H1299[tet-off p53(WT)] cells but not in U2OS cells. The lowest p53 levels reached in both cellular models may not be the only differences, as the expression of molecular DDT components and/or acquisition of cancer stem cell-like features in any of these cells may also be different (<a href="#B38" rid="B38" class=" bibr popnode">38</a>,<a href="#B86" rid="B86" class=" bibr popnode">86</a>). Despite such uncertainties, our data clearly show that gradual changes in both the exogenous and endogenous level of p53 are decisive for the rate of nascent DNA synthesis. This choreography fully depends on POL&#x003b9; at least in H1299[tet-off p53(WT)] cells, as POL&#x003b9; depletion neutralizes p53-dependent changes in the DNA replication dynamics.</p><p class="p p-last">Complex formation of POL&#x003b9; with p53 was observed only at higher p53 levels. Contrarily, the POL&#x003b9;-PCNA complex gained significance when p53 levels were lower, which could be interpreted to simply reflect the stoichiometry between the complex partners POL&#x003b9; and p53. Arguing against this one-dimensional explanation, PLA and immunoprecipitation experiments revealed maximum complex formation of POL&#x003b9; with p53 at a concentration of 0.25 ng/ml tetracycline in H1299[tet-off p53(WT)]. At 0 ng/ml tetracycline, i.e. the maximum level of p53, such a complex was no longer detectable by PLA and only visible as a very faint band after immunoprecipitation. From these results, we conclude that POL&#x003b9;-p53 complexes are formed and remain stable at an optimum, non-maximal, p53 level. Since DNA lesions and the cell cycle distribution were similar at the two highest p53 levels, we speculate that checkpoint signaling is unlikely to play a role. More likely, the biochemical properties of these factors at replication forks may be decisive for POL&#x003b9;-p53 complex stability. For example, the oligomerisation state of p53 within the complex (<a href="#B37" rid="B37" class=" bibr popnode">37</a>), the posttranslational modifications of complex components and/or the structural changes in the recognized DNA at the fork may differ at different p53 levels. Our analysis at maximum p53 level indicates a significant increase in PCNA pUb, known to trigger fork reversal (<a href="#B42" rid="B42" class=" bibr popnode">42</a>,<a href="#B85" rid="B85" class=" bibr popnode">85</a>). Our immunoprecipitation and PLA experiments may therefore provide snapshots of different stages of the POL&#x003b9;-p53 pathway as previously reported (<a href="#B38" rid="B38" class=" bibr popnode">38</a>). The pathway was reported to involve an initial step involving formation of the idling complex (<a href="#B36" rid="B36" class=" bibr popnode">36</a>) (a step that may be enriched at 0.25 ng/ml tetracycline) that creates a time-window promoting PCNA pUb at stalled forks followed by fork remodeling and p53 dissociation (a step that may be enriched at 0 ng/ml tetracycline). Interestingly also, PLA and immunoprecipitation detected POL&#x003b9;-PCNA complexes at 2 ng/ml but not at 4500 ng/ml, i.e. the lowest p53 level. This indicates that an intermediate low level of p53 is required for POL&#x003b9;-PCNA TLS complex formation but not enough to generate the POL&#x003b9;-p53 idling complex. Since p53 is not a component of the TLS complex, canonical functions of p53 in transcriptional transactivation may play a role towards TLS, such as modulation of PCNA Ub (<a href="#B38" rid="B38" class=" bibr popnode">38</a>,<a href="#B87" rid="B87" class=" bibr popnode">87</a>,<a href="#B88" rid="B88" class=" bibr popnode">88</a>) or expression of POL&#x003b7; (<a href="#B53" rid="B53" class=" bibr popnode">53</a>). This strengthens the idea that more than one pathway switch occurs along the range of different p53 expression levels in the cell.</p></div><div id="SEC4-2" class="sec"><h3 id="SEC4-2title">Complex crosstalk of DDT pathway-specific factors at different levels of p53</h3><p class="p p-first">Previously, different factors involved in the POL&#x003b9;-p53 DDT pathway were identified (<a href="#B36" rid="B36" class=" bibr popnode">36&#x02013;38</a>). In this study, we dissected the hierarchy between these factors along different p53 levels. Due to our results of DNA fiber analyses subsequent to the depletion of TLS factors&#x02014;specifically POL&#x003b7;, POL&#x003b6; (REV3L) and REV1&#x02014;we can conclude that TLS takes center stage at intermediate low p53 levels. Noteworthy however, REV1 plays a dual role depending on the p53 concentration: at intermediate low levels of p53, when TLS is predominant, REV1 potentiates TLS either as an inserter of nucleotides or as a hub for the recruitment of other TLS polymerases (<a href="#B75" rid="B75" class=" bibr popnode">75</a>). Conversely, at high p53 where the POL&#x003b9;&#x02013;p53 DDT pathway is predominant, REV1 may support nascent DNA synthesis deceleration by POL&#x003b9;&#x02013;p53. Track ratio analysis revealed increased values only when REV1 depletion unleashed excessively fast replication, causing non-classical replication stress (<a href="#B67" rid="B67" class=" bibr popnode">67</a>,<a href="#B68" rid="B68" class=" bibr popnode">68</a>). Interestingly, POL&#x003b6; down-regulation decelerated DNA synthesis not only at lower levels of p53 (2, 4500 ng/ml) but up to 0.25 ng/ml tetracycline, where the POL&#x003b9;&#x02013;p53 complex was most prominent. Based on these results, we postulate that fork remodeling and TLS pathways act simultaneously at 0.25&#x000a0;and 2 ng/ml tetracycline. This highlights that DNA replication speed integrates contributions from different DDT pathways that can co-occur and compete with one another at intermediate p53 levels.</p><p>When depleting the fork remodeling factors HLTF and ZRANB3, replication track elongation was detectable only at the highest p53 level (0 ng/ml), not at both higher levels (0, 0.25 ng/ml) as seen after POL&#x003b9; knock-down. In the light of these and our results from POL&#x003b9;-p53 interaction analyses, it is possible that at the two highest p53 levels replication slow-down is caused by different processes. At 0 ng/ml tetracycline, it is driven through fork reversal by HLTF and ZRANB3 (<a href="#B42" rid="B42" class=" bibr popnode">42</a>,<a href="#B89" rid="B89" class=" bibr popnode">89</a>). At 0.25 ng/ml, when the impact of HLTF and ZRANB3 on DNA replication dynamics was no longer significant, but the POL&#x003b9;&#x02013;p53 complex prominent, fork slowing may rely on idling (<a href="#B36" rid="B36" class=" bibr popnode">36</a>). It cannot be excluded that canonical functions of p53 may also contribute to this scenario, as sequestration of p53 at sites of transcriptional transactivation may constrain its availability for POL&#x003b9; interactions.</p><p>Of note, after HLTF and ZRANB3 knockdown at lower p53 levels (2, 4500 ng/ml) shortening of replication tracks was documented. Importantly however, down-regulation of ZRANB3 at lower p53 levels generated a unique scenario, characterized by increased track ratios combined with fork slowing. Hence, in such particular conditions, the shortening of fibers can be explained by classical fork stalling rather than by slow-down of DNA synthesis. In opposition, knockdown of HLTF also entailed a significant increase of fork asymmetry at the highest p53 level (0 ng/ml), but in this case, likely due to excessively fast replication. Therefore, similarly to REV1, HLTF plays a dual role in DNA replication elongation which may be related to the role of HLTF in the induction of both mUb and pUb of PCNA (<a href="#B40" rid="B40" class=" bibr popnode">40</a>,<a href="#B41" rid="B41" class=" bibr popnode">41</a>,<a href="#B90" rid="B90" class=" bibr popnode">90</a>,<a href="#B91" rid="B91" class=" bibr popnode">91</a>). Thus, its depletion may not only reduce pUb marks on PCNA read by ZRANB3 for fork reversal (<a href="#B42" rid="B42" class=" bibr popnode">42</a>), but also mUb marks on PCNA, necessary to recruit alternative POLs for TLS (<a href="#B90" rid="B90" class=" bibr popnode">90</a>).</p><p>Recent studies unveiled opposing hierarchical relationships between PRIMPOL-mediated repriming and fork remodeling depending on the availability of specific DDT pathway components (<a href="#B81" rid="B81" class=" bibr popnode">81</a>,<a href="#B83" rid="B83" class=" bibr popnode">83</a>,<a href="#B92" rid="B92" class=" bibr popnode">92</a>). In our study, knockdown experiments carved out a bimodal role of PRIMPOL, namely the expected repriming function accelerating DNA replication at undetectable p53 levels (4500 ng/ml) and, like POL&#x003b9;, a role in fiber shortening at the two higher p53 levels (0, 0.25 ng/ml). Simultaneous depletion of POL&#x003b9; and PRIMPOL reflected the individual replication phenotypes at lower p53 levels, i.e. replication accelerating activities by POL&#x003b9;-mediated TLS at 2 ng/ml and PRIMPOL-mediated repriming at 4500 ng/ml. At higher p53 levels (0, 0.25 ng/ml) double depletion suggested an epistatic relationship between POL&#x003b9; and PRIMPOL. Interestingly, earlier work from both yeast (<a href="#B92" rid="B92" class=" bibr popnode">92</a>) and human cells (<a href="#B93" rid="B93" class=" bibr popnode">93</a>) showed that repriming precedes template switching at the fork, as the first choice before fork reversal or TLS. From this we suggest that at higher p53 levels PRIMPOL could promote recombination-mediated template switching in line with the POL&#x003b9;-p53 complex and coexisting with HLTF- and ZRANB3-dependent fork reversal at the highest p53 level (0 ng/ml). In support of this idea, the POL&#x003b9;&#x02013;p53 pathway was originally discovered as mechanism mediating recombination-mediated bypass of replication barriers (<a href="#B36" rid="B36" class=" bibr popnode">36</a>). Therefore, depending on the window of p53 expression, POL&#x003b9; which forms a complex with PCNA, may compete with PRIMPOL to prevent DNA damage by excessively fast DNA replication or forms a complex with p53 and cooperates with PRIMPOL to ensure error-free bypass of replication barriers by fork remodeling, improving the resilience towards exogenous replication stress. In future work it will be interesting to analyze the potential formation of complexes of PRIMPOL and/or REV1 with p53 or POL&#x003b9; given that DNA fiber phenotypes were similar after knockdown of either POL&#x003b9;, PRIMPOL or REV1 at the highest p53 levels (compare Figure <a href="/pmc/articles/PMC11039994/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure4A</span></span><span>4A</span></a> with Figure <a href="/pmc/articles/PMC11039994/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure6A).</span></span><span>6A</span></a>). Future work may also address gap formation by methods such as electron microscopy, a technique that allows to determine the localization of single-stranded DNA gaps at or behind replication fork junctions, their size and number as well as the distribution on the two parental strands (<a href="#B94" rid="B94" class=" bibr popnode">94</a>,<a href="#B95" rid="B95" class=" bibr popnode">95</a>).</p><p class="p p-last">Which factor is responsible for accelerated DNA replication in the absence of POL&#x003b9; or absence of PRIMPOL at high p53 levels (0 ng/ml, 0.25 ng/ml)? To answer this question we performed further double knockdown experiments. POL&#x003b9; and POL&#x003b7; double depletion generated a pattern reflecting the combination of the individual knockdown effects at all tetracycline concentrations excluding the possibility that POL&#x003b7; is the only polymerase accelerating DNA replication in absence of POL&#x003b9;. Strikingly, when depleting PRIMPOL together with POL&#x003b7;, fiber analysis reflected the individual depletions at 2 ng/ml and 4500 ng/ml, but a new phenotype compared to single depletions, namely fiber shortening, at 0&#x000a0;and 0.25 ng/ml. This suggests that the depletion of PRIMPOL unleashes the TLS activities of POL&#x003b7;. In conclusion, PRIMPOL could play an important role in the p53-POL&#x003b9; DDT pathway by supporting the choice of fork remodeling over TLS.</p></div><div id="SEC4-3" class="sec sec-last"><h3 id="SEC4-3title">Conclusions</h3><p class="p p-first-last">Altogether, we postulate that different p53 levels promote the choice of different DDT pathways in charge of DNA replication. At high p53 levels (0 and 0.25 ng/ml), DNA replication is slowed down through complex formation with POL&#x003b9;. At the highest p53 level (0 ng/ml), HLTF- and ZRANB3-mediated fork remodeling becomes the predominant DDT pathway. Conversely, at low p53 levels (2 ng/ml), DNA synthesis is accelerated by POL&#x003b9;- and POL&#x003b7;-mediated TLS. Only when p53 levels are hardly detectable (4500 ng/ml), and POL&#x003b7;, a p53 target, downregulated, PRIMPOL plays a role in DNA replication elongation. Conversely, PRIMPOL contributes to the deceleration of forks at high p53 levels (0 and 0.25 ng/ml) by repressing fast POL&#x003b7;-dependent TLS and possibly also by generating recombinogenic replication intermediates. Additional factors perform epistatic functions, namely REV1 acting as a TLS polymerase loader, REV3L as a TLS extender and gap filler post-repriming, and HLTF as a PCNA mUb and pUb E3 ligase (Figure <a href="/pmc/articles/PMC11039994/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure8).</span></span><span>8</span></a>). In conclusion, prevalence of a certain replication phenotype involves many DNA replication factors that in part can play bimodal roles, whereby coexisting pathways establish a complex equilibrium determining the outcome. This work reveals the existence of a dynamic balance between DDT pathways finely tuned by a p53-mediated equilibrium. Slight perturbations can shift the pathway choice, highlighting the vulnerability of these control mechanisms safeguarding genome integrity.</p></div></div><div id="sec-a.n.e" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="sec-a.n.etitle" style="text-transform: uppercase;">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="sup1"><h4>gkae061_Supplemental_Files</h4><div class="sup-box half_rhythm" id="media-a.n.e.b.b"><a href="/pmc/articles/PMC11039994/bin/gkae061_supplemental_files.zip" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(5.2M, zip)</sup></div></div></div><div id="ACK1" class="tsec sec"><h2 class="head no_bottom_margin" id="ACK1title" style="text-transform: uppercase;">Acknowledgements</h2><div class="sec"><p>We would like to thank the Wiesm&#x000fc;ller laboratory for constant support, in particular, Nadine Pietzko, Ulm, Germany, for technical assistance during RT-qPCR experiments. We are grateful to Katrin Paeschke (Bonn, Germany) and Mike-Andrew Westhoff (Ulm, Germany) for their expert advices. Images in Figure <a href="/pmc/articles/PMC11039994/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure8</span></span><span>8</span></a> and the graphical abstract were created with BioRender.com. Bryan Casta&#x000f1;o is a member of the International Graduate School in Molecular Medicine, Ulm University.</p><p>
<em>Author contribution:</em> Bryan Casta&#x000f1;o: Methodology, Investigation, Visualization, Writing &#x02013; original draft, Writing &#x02013; review &#x00026; editing. Sabrina Schorer: Investigation. Yitian Guo: Investigation. Nicol&#x000e1;s Calzetta: Writing &#x02013; review &#x00026; editing. Vanesa Gottifredi: Resources, Writing &#x02013; review &#x00026; editing. Lisa Wiesm&#x000fc;ller: Conceptualization, Methodology, Resources, Supervision, Funding acquisition, Visualization, Project administration, Writing &#x02013; original draft, Writing &#x02013; review &#x00026; editing. Stephanie Biber: Conceptualization, Methodology, Resources, Supervision, Funding acquisition, Writing &#x02013; original draft, Writing &#x02013; review &#x00026; editing.</p></div></div><div id="NT1" class="tsec sec"><h2 class="head no_bottom_margin" id="NT1title" style="text-transform: uppercase;">Notes</h2><p>Present address: Yitian Guo, Department of Urology, Zhongda Hospital Southeast University, Nanjing 210009, China.</p><p>Present address: Stephanie Biber, Bavarian Cancer Registry, Bavarian Health and Food Safety Authority, Schweinauer Hauptstra&#x000df;e 80, 90441 Nuremberg, Germany.</p></div><div id="article-aaff-info" class="tsec sec"><h2 class="head no_bottom_margin" id="article-aaff-infotitle" style="text-transform: uppercase;">Contributor Information</h2><p><span class="fm-affl">Bryan A Casta&#x000f1;o, </span><span class="fm-affl" id="aff-a.m.b.f.a.b"> 
Department of Obstetrics and Gynecology, Ulm University, Ulm&#x000a0;89075, Germany.</span></p><p><span class="fm-affl">Sabrina Schorer, </span><span class="fm-affl" id="aff-a.m.b.f.b.b"> 
Department of Obstetrics and Gynecology, Ulm University, Ulm&#x000a0;89075, Germany.</span></p><p><span class="fm-affl">Yitian Guo, </span><span class="fm-affl" id="aff-a.m.b.f.c.c"> 
Department of Obstetrics and Gynecology, Ulm University, Ulm&#x000a0;89075, Germany.</span></p><p><span class="fm-affl">Nicol&#x000e1;s L Calzetta, </span><span class="fm-affl" id="aff-a.m.b.f.d.b"> 
Fundacion Instituto Leloir, CONICET, 1405 Buenos Aires, Argentina.</span></p><p><span class="fm-affl">Vanesa Gottifredi, </span><span class="fm-affl" id="aff-a.m.b.f.e.b"> 
Fundacion Instituto Leloir, CONICET, 1405 Buenos Aires, Argentina.</span></p><p><span class="fm-affl">Lisa Wiesm&#x000fc;ller, </span><span class="fm-affl" id="aff-a.m.b.f.f.c"> 
Department of Obstetrics and Gynecology, Ulm University, Ulm&#x000a0;89075, Germany.</span></p><p><span class="fm-affl">Stephanie Biber, </span><span class="fm-affl" id="aff-a.m.b.f.g.b"> 
Department of Obstetrics and Gynecology, Ulm University, Ulm&#x000a0;89075, Germany.</span></p></div><div id="SEC5" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="SEC5title" style="text-transform: uppercase;">Data availability</h2><!--/article/back/sec/--><p class="p p-first-last">The data sets generated during and/or analyzed during the current study are available from the corresponding authors upon request.</p></div><div id="SEC6" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="SEC6title" style="text-transform: uppercase;">Supplementary data</h2><!--/article/back/sec/--><p class="p p-first-last">
<a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkae061#supplementary-data" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Data</a> are available at NAR Online.</p></div><div id="SEC7" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="SEC7title" style="text-transform: uppercase;">Funding</h2><!--/article/back/sec/--><p class="p p-first">Deutsche Forschungsgemeinschaft (DFG), Research Training Group 1789 &#x02018;Cellular and Molecular Mechanisms in Aging&#x02019; and the Collaborative Research Centers1279 &#x02018;Exploiting the Human Peptidome for Novel Antimicrobial and Anticancer Agents&#x02019; and 1506 &#x02018;Aging at Interfaces&#x02019; (to L.W.); Start-up Funding Program of the Medical Faculty of Ulm University [&#x02018;Bausteinprogramm&#x02019;; L.S.B.N.0160 to S.B.]; China Scholarship Council (scholarship to Y.G.); Friedrich Wilhelm Bessel Award by the Alexander von Humboldt Foundation (to V.G.); Research Group Linkage Programme of the Alexander von Humboldt Foundation (to V.G. and L.W.). Funding for open access charge:&#x000a0;Deutsche Forschungsgemeinschaft, Collaborative Research Center [1279 and 1506].</p><p class="p p-last">
<em>Conflict of interest statement</em>. None declared.</p></div><div id="REF1" class="tsec sec"><h2 class="head no_bottom_margin" id="REF1title" style="text-transform: uppercase;">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1">1. <span class="mixed-citation">

Eldridge &#x000a0;C.B., Allen &#x000a0;F.J., Crisp &#x000a0;A., Grandy &#x000a0;R.A., Vallier &#x000a0;L., Sale &#x000a0;J.E. &#x000a0;<span class="ref-title">A p53-dependent checkpoint induced upon DNA damage alters cell fate during hiPSC differentiation</span>. <span class="ref-journal">Stem Cell Rep.</span> &#x000a0;2020; <span class="ref-vol">15</span>:827&#x02013;835. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7561492/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32888504" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stem+Cell+Rep.&amp;title=A+p53-dependent+checkpoint+induced+upon+DNA+damage+alters+cell+fate+during+hiPSC+differentiation&amp;volume=15&amp;publication_year=2020&amp;pages=827-835&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B2">2. <span class="mixed-citation">

Friedel &#x000a0;L., Loewer &#x000a0;A. &#x000a0;<span class="ref-title">The guardian's choice: how p53 enables context-specific decision-making in individual cells</span>. <span class="ref-journal">FEBS J.</span> &#x000a0;2022; <span class="ref-vol">289</span>:40&#x02013;52.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/33590949" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEBS+J.&amp;title=The+guardian's+choice:+how+p53+enables+context-specific+decision-making+in+individual+cells&amp;volume=289&amp;publication_year=2022&amp;pages=40-52&amp;pmid=33590949&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B3">3. <span class="mixed-citation">

Eliyahu &#x000a0;D., Michalovitz &#x000a0;D., Eliyahu &#x000a0;S., Pinhasi-Kimhi &#x000a0;O., Oren &#x000a0;M. &#x000a0;<span class="ref-title">Wild-type p53 can inhibit oncogene-mediated focus formation</span>. <span class="ref-journal">Proc. Nat. Acad. Sci. U.S.A.</span> &#x000a0;1989; <span class="ref-vol">86</span>:8763&#x02013;8767. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC298370/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2530586" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Nat.+Acad.+Sci.+U.S.A.&amp;title=Wild-type+p53+can+inhibit+oncogene-mediated+focus+formation&amp;volume=86&amp;publication_year=1989&amp;pages=8763-8767&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B4">4. <span class="mixed-citation">

Finlay &#x000a0;C.A., Hinds &#x000a0;P.W., Levine &#x000a0;A.J. &#x000a0;<span class="ref-title">The p53 proto-oncogene can act as a suppressor of transformation</span>. <span class="ref-journal">Cell</span>. 1989; <span class="ref-vol">57</span>:1083&#x02013;1093.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/2525423" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=The+p53+proto-oncogene+can+act+as+a+suppressor+of+transformation&amp;volume=57&amp;publication_year=1989&amp;pages=1083-1093&amp;pmid=2525423&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B5">5. <span class="mixed-citation">

Bargonetti &#x000a0;J., Friedman &#x000a0;P.N., Kern &#x000a0;S.E., Vogelstein &#x000a0;B., Prives &#x000a0;C. &#x000a0;<span class="ref-title">Wild-type but not mutant p53 immunopurified proteins bind to sequences adjacent to the SV40 origin of replication</span>. <span class="ref-journal">Cell</span>. 1991; <span class="ref-vol">65</span>:1083&#x02013;1091.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/1646078" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Wild-type+but+not+mutant+p53+immunopurified+proteins+bind+to+sequences+adjacent+to+the+SV40+origin+of+replication&amp;volume=65&amp;publication_year=1991&amp;pages=1083-1091&amp;pmid=1646078&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B6">6. <span class="mixed-citation">

Kern &#x000a0;S.E., Kinzler &#x000a0;K.W., Bruskin &#x000a0;A., Jarosz &#x000a0;D., Friedman &#x000a0;P., Prives &#x000a0;C., Vogelstein &#x000a0;B. &#x000a0;<span class="ref-title">Identification of p53 as a sequence-specific DNA-binding protein</span>. <span class="ref-journal">Science</span>. 1991; <span class="ref-vol">252</span>:1708&#x02013;1711.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/2047879" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Identification+of+p53+as+a+sequence-specific+DNA-binding+protein&amp;volume=252&amp;publication_year=1991&amp;pages=1708-1711&amp;pmid=2047879&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B7">7. <span class="mixed-citation">

Yonish-Rouach &#x000a0;E., Resnitzky &#x000a0;D., Lotem &#x000a0;J., Sachs &#x000a0;L., Kimchi &#x000a0;A., Oren &#x000a0;M &#x000a0;<span class="ref-title">Wild-type p53 induces apoptosis of myeloid leukaemic cells that is inhibited by interleukin-6</span>. <span class="ref-journal">Nature</span>. 1991; <span class="ref-vol">352</span>:345&#x02013;347.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/1852210" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Wild-type+p53+induces+apoptosis+of+myeloid+leukaemic+cells+that+is+inhibited+by+interleukin-6&amp;volume=352&amp;publication_year=1991&amp;pages=345-347&amp;pmid=1852210&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B8">8. <span class="mixed-citation">

El-Deiry &#x000a0;W.S., Kern &#x000a0;S.E., Pietenpol &#x000a0;J.A., Kinzler &#x000a0;K.W., Vogelstein &#x000a0;B. &#x000a0;<span class="ref-title">Definition of a consensus binding site for p53</span>. <span class="ref-journal">Nat. Genet.</span> &#x000a0;1992; <span class="ref-vol">1</span>:45&#x02013;49.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/1301998" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Definition+of+a+consensus+binding+site+for+p53&amp;volume=1&amp;publication_year=1992&amp;pages=45-49&amp;pmid=1301998&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B9">9. <span class="mixed-citation">

Dutta &#x000a0;A., Ruppert &#x000a0;J.M., Aster &#x000a0;J.C., Winchester &#x000a0;E. &#x000a0;<span class="ref-title">Inhibition of DNA replication factor RPA by p53</span>. <span class="ref-journal">Nature</span>. 1993; <span class="ref-vol">365</span>:79&#x02013;82.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/8361542" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Inhibition+of+DNA+replication+factor+RPA+by+p53&amp;volume=365&amp;publication_year=1993&amp;pages=79-82&amp;pmid=8361542&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B10">10. <span class="mixed-citation">

Li &#x000a0;R., Botchan &#x000a0;M.R. &#x000a0;<span class="ref-title">The acidic transcriptional activation domains of VP16 and p53 bind the cellular replication protein A and stimulate in vitro BPV-1 DNA replication</span>. <span class="ref-journal">Cell</span>. 1993; <span class="ref-vol">73</span>:1207&#x02013;1221.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/8390328" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=The+acidic+transcriptional+activation+domains+of+VP16+and+p53+bind+the+cellular+replication+protein+A+and+stimulate+in+vitro+BPV-1+DNA+replication&amp;volume=73&amp;publication_year=1993&amp;pages=1207-1221&amp;pmid=8390328&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B11">11. <span class="mixed-citation">

Bakalkin &#x000a0;G., Yakovleva &#x000a0;T., Selivanova &#x000a0;G., Magnusson &#x000a0;K.P., Szekely &#x000a0;L., Kiseleva &#x000a0;E., Klein &#x000a0;G., Terenius &#x000a0;L., Wiman &#x000a0;K.G. &#x000a0;<span class="ref-title">p53 binds single-stranded DNA ends and catalyzes DNA renaturation and strand transfer</span>. <span class="ref-journal">Proc. Nat. Acad. Sci. U.S.A.</span> &#x000a0;1994; <span class="ref-vol">91</span>:413&#x02013;417. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC42958/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8278402" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Nat.+Acad.+Sci.+U.S.A.&amp;title=p53+binds+single-stranded+DNA+ends+and+catalyzes+DNA+renaturation+and+strand+transfer&amp;volume=91&amp;publication_year=1994&amp;pages=413-417&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B12">12. <span class="mixed-citation">

Lee &#x000a0;S., Elenbaas &#x000a0;B., Levine &#x000a0;A., Griffith &#x000a0;J. &#x000a0;<span class="ref-title">p53 and its 14 kDa C-terminal domain recognize primary DNA damage in the form of insertion/deletion mismatches</span>. <span class="ref-journal">Cell</span>. 1995; <span class="ref-vol">81</span>:1013&#x02013;1020.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/7600570" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=p53+and+its+14+kDa+C-terminal+domain+recognize+primary+DNA+damage+in+the+form+of+insertion/deletion+mismatches&amp;volume=81&amp;publication_year=1995&amp;pages=1013-1020&amp;pmid=7600570&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B13">13. <span class="mixed-citation">

Mummenbrauer &#x000a0;T., Janus &#x000a0;F., M&#x000fc;ller &#x000a0;B., Wiesm&#x000fc;ller &#x000a0;L., Deppert &#x000a0;W., Grosse &#x000a0;F. &#x000a0;<span class="ref-title">p53 Protein exhibits 3'-to-5' exonuclease activity</span>. <span class="ref-journal">Cell</span>. 1996; <span class="ref-vol">85</span>:1089&#x02013;1099.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/8674115" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=p53+Protein+exhibits+3'-to-5'+exonuclease+activity&amp;volume=85&amp;publication_year=1996&amp;pages=1089-1099&amp;pmid=8674115&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B14">14. <span class="mixed-citation">

St&#x000fc;rzbecher &#x000a0;H.W., Donzelmann &#x000a0;B., Henning &#x000a0;W., Knippschild &#x000a0;U., Buchhop &#x000a0;S. &#x000a0;<span class="ref-title">p53 is linked directly to homologous recombination processes via RAD51/RecA protein interaction</span>. <span class="ref-journal">EMBO J.</span> &#x000a0;1996; <span class="ref-vol">15</span>:1992&#x02013;2002.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC450118/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8617246" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+J.&amp;title=p53+is+linked+directly+to+homologous+recombination+processes+via+RAD51/RecA+protein+interaction&amp;volume=15&amp;publication_year=1996&amp;pages=1992-2002&amp;pmid=8617246&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B15">15. <span class="mixed-citation">

Dudenh&#x000f6;ffer &#x000a0;C., Rohaly &#x000a0;G., Will &#x000a0;K., Deppert &#x000a0;W., Wiesm&#x000fc;ller &#x000a0;L. &#x000a0;<span class="ref-title">Specific mismatch recognition in heteroduplex intermediates by p53 suggests a role in fidelity control of homologous recombination</span>. <span class="ref-journal">Mol. Cell. Biol.</span> &#x000a0;1998; <span class="ref-vol">18</span>:5332&#x02013;5342.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC109118/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9710617" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Biol.&amp;title=Specific+mismatch+recognition+in+heteroduplex+intermediates+by+p53+suggests+a+role+in+fidelity+control+of+homologous+recombination&amp;volume=18&amp;publication_year=1998&amp;pages=5332-5342&amp;pmid=9710617&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B16">16. <span class="mixed-citation">

Offer &#x000a0;H., Wolkowicz &#x000a0;R., Matas &#x000a0;D., Blumenstein &#x000a0;S., Livneh &#x000a0;Z., Rotter &#x000a0;V. &#x000a0;<span class="ref-title">Direct involvement of p53 in the base excision repair pathway of the DNA repair machinery</span>. <span class="ref-journal">FEBS Lett.</span> &#x000a0;1999; <span class="ref-vol">450</span>:197&#x02013;204.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/10359074" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEBS+Lett.&amp;title=Direct+involvement+of+p53+in+the+base+excision+repair+pathway+of+the+DNA+repair+machinery&amp;volume=450&amp;publication_year=1999&amp;pages=197-204&amp;pmid=10359074&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B17">17. <span class="mixed-citation">

Jackson &#x000a0;S.P., Bartek &#x000a0;J. &#x000a0;<span class="ref-title">The DNA-damage response in human biology and disease</span>. <span class="ref-journal">Nature</span>. 2009; <span class="ref-vol">461</span>:1071&#x02013;1078.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2906700/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19847258" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+DNA-damage+response+in+human+biology+and+disease&amp;volume=461&amp;publication_year=2009&amp;pages=1071-1078&amp;pmid=19847258&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B18">18. <span class="mixed-citation">

Bournique &#x000a0;E., Dall&#x02019;Osto &#x000a0;M., Hoffmann &#x000a0;J.S., Bergoglio &#x000a0;V. &#x000a0;<span class="ref-title">Role of specialized DNA polymerases in the limitation of replicative stress and DNA damage transmission</span>. <span class="ref-journal">Mutat. Res.</span> &#x000a0;2018; <span class="ref-vol">808</span>:62&#x02013;73.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/28843435" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mutat.+Res.&amp;title=Role+of+specialized+DNA+polymerases+in+the+limitation+of+replicative+stress+and+DNA+damage+transmission&amp;volume=808&amp;publication_year=2018&amp;pages=62-73&amp;pmid=28843435&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B19">19. <span class="mixed-citation">

Bainbridge &#x000a0;L.J., Teague &#x000a0;R., Doherty &#x000a0;A.J. &#x000a0;<span class="ref-title">Repriming DNA synthesis: an intrinsic restart pathway that maintains efficient genome replication</span>. <span class="ref-journal">Nucleic Acids Res.</span> &#x000a0;2021; <span class="ref-vol">49</span>:4831&#x02013;4847.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8136793/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33744934" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Repriming+DNA+synthesis:+an+intrinsic+restart+pathway+that+maintains+efficient+genome+replication&amp;volume=49&amp;publication_year=2021&amp;pages=4831-4847&amp;pmid=33744934&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B20">20. <span class="mixed-citation">

Lawrence &#x000a0;C.W., Christensen &#x000a0;R.B. &#x000a0;<span class="ref-title">The mechanism of untargeted mutagenesis in UV-irradiated yeast</span>. <span class="ref-journal">Mol. Gen. Genet.</span> &#x000a0;1982; <span class="ref-vol">186</span>:1&#x02013;9.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/7050619" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Gen.+Genet.&amp;title=The+mechanism+of+untargeted+mutagenesis+in+UV-irradiated+yeast&amp;volume=186&amp;publication_year=1982&amp;pages=1-9&amp;pmid=7050619&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B21">21. <span class="mixed-citation">

Cohen &#x000a0;I.S., Bar &#x000a0;C., Paz-Elizur &#x000a0;T., Ainbinder &#x000a0;E., Leopold &#x000a0;K., de&#x000a0;Wind &#x000a0;N., Geacintov &#x000a0;N., Livneh &#x000a0;Z. &#x000a0;<span class="ref-title">DNA lesion identity drives choice of damage tolerance pathway in murine cell chromosomes</span>. <span class="ref-journal">Nucleic Acids Res.</span> &#x000a0;2015; <span class="ref-vol">43</span>:1637&#x02013;1645.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4330363/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25589543" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=DNA+lesion+identity+drives+choice+of+damage+tolerance+pathway+in+murine+cell+chromosomes&amp;volume=43&amp;publication_year=2015&amp;pages=1637-1645&amp;pmid=25589543&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B22">22. <span class="mixed-citation">

Lerner &#x000a0;L.K., Francisco &#x000a0;G., Soltys &#x000a0;D.T., Rocha &#x000a0;C.R., Quinet &#x000a0;A., Vessoni &#x000a0;A.T., Castro &#x000a0;L.P., David &#x000a0;T.I.P., Bustos &#x000a0;S.O., Strauss &#x000a0;B.E. &#x000a0;et al.. &#x000a0;<span class="ref-title">Predominant role of DNA polymerase eta and p53-dependent translesion synthesis in the survival of ultraviolet-irradiated human cells</span>. <span class="ref-journal">Nucleic Acids Res.</span> &#x000a0;2017; <span class="ref-vol">45</span>:1270&#x02013;1280.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5388406/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28180309" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Predominant+role+of+DNA+polymerase+eta+and+p53-dependent+translesion+synthesis+in+the+survival+of+ultraviolet-irradiated+human+cells&amp;volume=45&amp;publication_year=2017&amp;pages=1270-1280&amp;pmid=28180309&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B23">23. <span class="mixed-citation">

Barnes &#x000a0;R.P., Tsao &#x000a0;W.C., Moldovan &#x000a0;G.L., Eckert &#x000a0;K.A. &#x000a0;<span class="ref-title">DNA polymerase eta prevents tumor cell-cycle arrest and cell death during recovery from replication stress</span>. <span class="ref-journal">Cancer Res.</span> &#x000a0;2018; <span class="ref-vol">78</span>:6549&#x02013;6560.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/30297532" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res.&amp;title=DNA+polymerase+eta+prevents+tumor+cell-cycle+arrest+and+cell+death+during+recovery+from+replication+stress&amp;volume=78&amp;publication_year=2018&amp;pages=6549-6560&amp;pmid=30297532&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B24">24. <span class="mixed-citation">

Edmunds &#x000a0;C.E., Simpson &#x000a0;L.J., Sale &#x000a0;J.E. &#x000a0;<span class="ref-title">PCNA ubiquitination and REV1 define temporally distinct mechanisms for controlling translesion synthesis in the avian cell line DT40</span>. <span class="ref-journal">Mol. Cell</span>. 2008; <span class="ref-vol">30</span>:519&#x02013;529.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/18498753" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=PCNA+ubiquitination+and+REV1+define+temporally+distinct+mechanisms+for+controlling+translesion+synthesis+in+the+avian+cell+line+DT40&amp;volume=30&amp;publication_year=2008&amp;pages=519-529&amp;pmid=18498753&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B25">25. <span class="mixed-citation">

Sale &#x000a0;J.E. &#x000a0;<span class="ref-title">Translesion DNA synthesis and mutagenesis in eukaryotes</span>. <span class="ref-journal">Cold Spring Harb. Perspect. Biol.</span> &#x000a0;2013; <span class="ref-vol">5</span>:a012708.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3578355/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23457261" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cold+Spring+Harb.+Perspect.+Biol.&amp;title=Translesion+DNA+synthesis+and+mutagenesis+in+eukaryotes&amp;volume=5&amp;publication_year=2013&amp;pages=a012708&amp;pmid=23457261&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B26">26. <span class="mixed-citation">

Boehm &#x000a0;E.M., Gildenberg &#x000a0;M.S., Washington &#x000a0;M.T. &#x000a0;<span class="ref-title">The many roles of PCNA in eukaryotic DNA replication</span>. <span class="ref-journal">Enzymes</span>. 2016; <span class="ref-vol">39</span>:231&#x02013;254.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4890617/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27241932" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Enzymes&amp;title=The+many+roles+of+PCNA+in+eukaryotic+DNA+replication&amp;volume=39&amp;publication_year=2016&amp;pages=231-254&amp;pmid=27241932&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B27">27. <span class="mixed-citation">

Pilzecker &#x000a0;B., Buoninfante &#x000a0;O.A., Jacobs &#x000a0;H. &#x000a0;<span class="ref-title">DNA damage tolerance in stem cells, ageing, mutagenesis, disease and cancer therapy</span>. <span class="ref-journal">Nucleic Acids Res.</span> &#x000a0;2019; <span class="ref-vol">47</span>:7163&#x02013;7181.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6698745/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31251805" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=DNA+damage+tolerance+in+stem+cells,+ageing,+mutagenesis,+disease+and+cancer+therapy&amp;volume=47&amp;publication_year=2019&amp;pages=7163-7181&amp;pmid=31251805&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B28">28. <span class="mixed-citation">

Mour&#x000f3;n &#x000a0;S., Rodriguez-Acebes &#x000a0;S., Mart&#x000ed;nez-Jim&#x000e9;nez &#x000a0;M.I., Garc&#x000ed;a-G&#x000f3;mez &#x000a0;S., Chocr&#x000f3;n &#x000a0;S., Blanco &#x000a0;L., M&#x000e9;ndez &#x000a0;J. &#x000a0;<span class="ref-title">Repriming of DNA synthesis at stalled replication forks by human PrimPol</span>. <span class="ref-journal">Nat. Struct. Mol. Biol.</span> &#x000a0;2013; <span class="ref-vol">20</span>:1383&#x02013;1389.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/24240614" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Struct.+Mol.+Biol.&amp;title=Repriming+of+DNA+synthesis+at+stalled+replication+forks+by+human+PrimPol&amp;volume=20&amp;publication_year=2013&amp;pages=1383-1389&amp;pmid=24240614&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B29">29. <span class="mixed-citation">

Bianchi &#x000a0;J., Rudd &#x000a0;S.G., Jozwiakowski &#x000a0;S.K., Bailey &#x000a0;L.J., Soura &#x000a0;V., Taylor &#x000a0;E., Stevanovic &#x000a0;I., Green &#x000a0;A.J., Stracker &#x000a0;T.H., Lindsay &#x000a0;H.D. &#x000a0;et al.. &#x000a0;<span class="ref-title">PrimPol bypasses UV photoproducts during eukaryotic chromosomal DNA replication</span>. <span class="ref-journal">Mol. Cell</span>. 2013; <span class="ref-vol">52</span>:566&#x02013;573.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4228047/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24267451" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=PrimPol+bypasses+UV+photoproducts+during+eukaryotic+chromosomal+DNA+replication&amp;volume=52&amp;publication_year=2013&amp;pages=566-573&amp;pmid=24267451&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B30">30. <span class="mixed-citation">

Garc&#x000ed;a-G&#x000f3;mez &#x000a0;S., Reyes &#x000a0;A., Mart&#x000ed;nez-Jim&#x000e9;nez &#x000a0;M.I., Chocr&#x000f3;n &#x000a0;E.S., Mour&#x000f3;n &#x000a0;S., Terrados &#x000a0;G., Powell &#x000a0;C., Salido &#x000a0;E., M&#x000e9;ndez &#x000a0;J., Holt &#x000a0;I.J. &#x000a0;et al.. &#x000a0;<span class="ref-title">PrimPol, an archaic primase/polymerase operating in human cells</span>. <span class="ref-journal">Mol. Cell</span>. 2013; <span class="ref-vol">52</span>:541&#x02013;553.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3899013/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24207056" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=PrimPol,+an+archaic+primase/polymerase+operating+in+human+cells&amp;volume=52&amp;publication_year=2013&amp;pages=541-553&amp;pmid=24207056&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B31">31. <span class="mixed-citation">

Gonz&#x000e1;lez-Acosta &#x000a0;D., Blanco-Romero &#x000a0;E., Ubieto-Capella &#x000a0;P., Mutreja &#x000a0;K., M&#x000ed;guez &#x000a0;S., Llanos &#x000a0;S., Garc&#x000ed;a &#x000a0;F., Mu&#x000f1;oz &#x000a0;J., Blanco &#x000a0;L., Lopes &#x000a0;M. &#x000a0;et al.. &#x000a0;<span class="ref-title">PrimPol-mediated repriming facilitates replication traverse of DNA interstrand crosslinks</span>. <span class="ref-journal">EMBO J.</span> &#x000a0;2021; <span class="ref-vol">40</span>:e106355.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8280817/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34128550" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+J.&amp;title=PrimPol-mediated+repriming+facilitates+replication+traverse+of+DNA+interstrand+crosslinks&amp;volume=40&amp;publication_year=2021&amp;pages=e106355&amp;pmid=34128550&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B32">32. <span class="mixed-citation">

McIntyre &#x000a0;C.A., Lawrence &#x000a0;S.A., Richards &#x000a0;A.L., Chou &#x000a0;J.F., Wong &#x000a0;W., Capanu &#x000a0;M., Berger &#x000a0;M.F., Donoghue &#x000a0;M.T.A., Yu &#x000a0;K.H., Varghese &#x000a0;A.M. &#x000a0;et al.. &#x000a0;<span class="ref-title">Alterations in driver genes are predictive of survival in patients with resected pancreatic ductal adenocarcinoma</span>. <span class="ref-journal">Cancer</span>. 2020; <span class="ref-vol">126</span>:3939&#x02013;3949.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7424538/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32573775" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer&amp;title=Alterations+in+driver+genes+are+predictive+of+survival+in+patients+with+resected+pancreatic+ductal+adenocarcinoma&amp;volume=126&amp;publication_year=2020&amp;pages=3939-3949&amp;pmid=32573775&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B33">33. <span class="mixed-citation">

Vaisman &#x000a0;A., Woodgate &#x000a0;R. &#x000a0;<span class="ref-title">Mysterious and fascinating: DNA polymerase &#x003b9; remains enigmatic 20 years after its discovery</span>. <span class="ref-journal">DNA Repair (Amst.)</span>. 2020; <span class="ref-vol">93</span>:102914.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7586464/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33087280" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=DNA+Repair+(Amst.)&amp;title=Mysterious+and+fascinating:+DNA+polymerase+&#x003b9;+remains+enigmatic+20+years+after+its+discovery&amp;volume=93&amp;publication_year=2020&amp;pages=102914&amp;pmid=33087280&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B34">34. <span class="mixed-citation">

Tonzi &#x000a0;P., Huang &#x000a0;T.T. &#x000a0;<span class="ref-title">Role of Y-family translesion DNA polymerases in replication stress: implications for new cancer therapeutic targets</span>. <span class="ref-journal">DNA Repair (Amst.)</span>. 2019; <span class="ref-vol">78</span>:20&#x02013;26.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6534436/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30954011" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=DNA+Repair+(Amst.)&amp;title=Role+of+Y-family+translesion+DNA+polymerases+in+replication+stress:+implications+for+new+cancer+therapeutic+targets&amp;volume=78&amp;publication_year=2019&amp;pages=20-26&amp;pmid=30954011&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B35">35. <span class="mixed-citation">

McIntyre &#x000a0;J. &#x000a0;<span class="ref-title">Polymerase iota - an odd sibling among Y family polymerases</span>. <span class="ref-journal">DNA Repair (Amst.)</span>. 2020; <span class="ref-vol">86</span>:102753.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/31805501" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=DNA+Repair+(Amst.)&amp;title=Polymerase+iota+-+an+odd+sibling+among+Y+family+polymerases&amp;volume=86&amp;publication_year=2020&amp;pages=102753&amp;pmid=31805501&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B36">36. <span class="mixed-citation">

Hampp &#x000a0;S., Kiessling &#x000a0;T., Buechle &#x000a0;K., Mansilla &#x000a0;S.F., Thomale &#x000a0;J., Rall &#x000a0;M., Ahn &#x000a0;J., Pospiech &#x000a0;H., Gottifredi &#x000a0;V., Wiesm&#x000fc;ller &#x000a0;L. &#x000a0;<span class="ref-title">DNA damage tolerance pathway involving DNA polymerase &#x003b9; and the tumor suppressor p53 regulates DNA replication fork progression</span>. <span class="ref-journal">Proc. Nat. Acad. Sci. U.S.A.</span> &#x000a0;2016; <span class="ref-vol">113</span>:E4311&#x02013;E4319. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4968756/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27407148" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Nat.+Acad.+Sci.+U.S.A.&amp;title=DNA+damage+tolerance+pathway+involving+DNA+polymerase+&#x003b9;+and+the+tumor+suppressor+p53+regulates+DNA+replication+fork+progression&amp;volume=113&amp;publication_year=2016&amp;pages=E4311-E4319&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B37">37. <span class="mixed-citation">

Biber &#x000a0;S., Pospiech &#x000a0;H., Gottifredi &#x000a0;V., Wiesm&#x000fc;ller &#x000a0;L. &#x000a0;<span class="ref-title">Multiple biochemical properties of the p53 molecule contribute to activation of polymerase iota-dependent DNA damage tolerance</span>. <span class="ref-journal">Nucleic Acids Res.</span> &#x000a0;2020; <span class="ref-vol">48</span>:12188&#x02013;12203.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7708082/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33166398" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Multiple+biochemical+properties+of+the+p53+molecule+contribute+to+activation+of+polymerase+iota-dependent+DNA+damage+tolerance&amp;volume=48&amp;publication_year=2020&amp;pages=12188-12203&amp;pmid=33166398&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B38">38. <span class="mixed-citation">

Ihle &#x000a0;M., Biber &#x000a0;S., Schroeder &#x000a0;I.S., Blattner &#x000a0;C., Deniz &#x000a0;M., Damia &#x000a0;G., Gottifredi &#x000a0;V., Wiesm&#x000fc;ller &#x000a0;L. &#x000a0;<span class="ref-title">Impact of the interplay between stemness features, p53 and pol iota on replication pathway choices</span>. <span class="ref-journal">Nucleic Acids Res.</span> &#x000a0;2021; <span class="ref-vol">49</span>:7457&#x02013;7475.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8287946/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34165573" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Impact+of+the+interplay+between+stemness+features,+p53+and+pol+iota+on+replication+pathway+choices&amp;volume=49&amp;publication_year=2021&amp;pages=7457-7475&amp;pmid=34165573&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B39">39. <span class="mixed-citation">

Khare &#x000a0;V., Eckert &#x000a0;K.A. &#x000a0;<span class="ref-title">The proofreading 3'&#x02192;5' exonuclease activity of DNA polymerases: a kinetic barrier to translesion DNA synthesis</span>. <span class="ref-journal">Mutat. Res.</span> &#x000a0;2002; <span class="ref-vol">510</span>:45&#x02013;54.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/12459442" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mutat.+Res.&amp;title=The+proofreading+3'&#x02192;5'+exonuclease+activity+of+DNA+polymerases:+a+kinetic+barrier+to+translesion+DNA+synthesis&amp;volume=510&amp;publication_year=2002&amp;pages=45-54&amp;pmid=12459442&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B40">40. <span class="mixed-citation">

Unk &#x000a0;I., Hajd&#x000fa; &#x000a0;I., F&#x000e1;tyol &#x000a0;K., Hurwitz &#x000a0;J., Yoon &#x000a0;J.H., Prakash &#x000a0;L., Prakash &#x000a0;S., Haracska &#x000a0;L. &#x000a0;<span class="ref-title">Human HLTF functions as a ubiquitin ligase for proliferating cell nuclear antigen polyubiquitination</span>. <span class="ref-journal">Proc. Nat. Acad. Sci. U.S.A.</span> &#x000a0;2008; <span class="ref-vol">105</span>:3768&#x02013;3773. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2268824/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18316726" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Nat.+Acad.+Sci.+U.S.A.&amp;title=Human+HLTF+functions+as+a+ubiquitin+ligase+for+proliferating+cell+nuclear+antigen+polyubiquitination&amp;volume=105&amp;publication_year=2008&amp;pages=3768-3773&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B41">41. <span class="mixed-citation">

Motegi &#x000a0;A., Liaw &#x000a0;H.J., Lee &#x000a0;K.Y., Roest &#x000a0;H.P., Maas &#x000a0;A., Wu &#x000a0;X., Moinova &#x000a0;H., Markowitz &#x000a0;S.D., Ding &#x000a0;H., Hoeijmakers &#x000a0;J.H. &#x000a0;et al.. &#x000a0;<span class="ref-title">Polyubiquitination of proliferating cell nuclear antigen by HLTF and SHPRH prevents genomic instability from stalled replication forks</span>. <span class="ref-journal">Proc. Nat. Acad. Sci. U.S.A.</span> &#x000a0;2008; <span class="ref-vol">105</span>:12411&#x02013;12416. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2518831/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18719106" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Nat.+Acad.+Sci.+U.S.A.&amp;title=Polyubiquitination+of+proliferating+cell+nuclear+antigen+by+HLTF+and+SHPRH+prevents+genomic+instability+from+stalled+replication+forks&amp;volume=105&amp;publication_year=2008&amp;pages=12411-12416&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B42">42. <span class="mixed-citation">

Vujanovic &#x000a0;M., Krietsch &#x000a0;J., Raso &#x000a0;M.C., Terraneo &#x000a0;N., Zellweger &#x000a0;R., Schmid &#x000a0;J.A., Taglialatela &#x000a0;A., Huang &#x000a0;J.W., Holland &#x000a0;C.L., Zwicky &#x000a0;K. &#x000a0;et al.. &#x000a0;<span class="ref-title">Replication fork slowing and reversal upon DNA damage require PCNA polyubiquitination and ZRANB3 DNA translocase activity</span>. <span class="ref-journal">Mol. Cell</span>. 2017; <span class="ref-vol">67</span>:882&#x02013;890.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5594246/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28886337" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=Replication+fork+slowing+and+reversal+upon+DNA+damage+require+PCNA+polyubiquitination+and+ZRANB3+DNA+translocase+activity&amp;volume=67&amp;publication_year=2017&amp;pages=882-890&amp;pmid=28886337&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B43">43. <span class="mixed-citation">

Mansilla &#x000a0;S.F., Bertolin &#x000a0;A.P., Venerus&#x000a0;Arbilla &#x000a0;S., Casta&#x000f1;o &#x000a0;B.A., Jahjah &#x000a0;T., Singh &#x000a0;J.K., Siri &#x000a0;S.O., Castro &#x000a0;M.V., de&#x000a0;la&#x000a0;Vega &#x000a0;M.B. &#x000a0;et al.. &#x000a0;<span class="ref-title">Polymerase iota (Pol &#x003b9;) prevents PrimPol-mediated nascent DNA synthesis and chromosome instability</span>. <span class="ref-journal">Sci. Adv.</span> &#x000a0;2023; <span class="ref-vol">9</span>:eade7997.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10104471/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37058556" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Adv.&amp;title=Polymerase+iota+(Pol+&#x003b9;)+prevents+PrimPol-mediated+nascent+DNA+synthesis+and+chromosome+instability&amp;volume=9&amp;publication_year=2023&amp;pages=eade7997&amp;pmid=37058556&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B44">44. <span class="mixed-citation">

Garc&#x000ed;a-Rodr&#x000ed;guez &#x000a0;N., Wong &#x000a0;R.P., Ulrich &#x000a0;H.D. &#x000a0;<span class="ref-title">Functions of ubiquitin and SUMO in DNA replication and replication stress</span>. <span class="ref-journal">Front. Genet.</span> &#x000a0;2016; <span class="ref-vol">7</span>:87.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4865505/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27242895" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Genet.&amp;title=Functions+of+ubiquitin+and+SUMO+in+DNA+replication+and+replication+stress&amp;volume=7&amp;publication_year=2016&amp;pages=87&amp;pmid=27242895&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B45">45. <span class="mixed-citation">

Majhi &#x000a0;P.D., Griner &#x000a0;N.B., Mayfield &#x000a0;J.A., Compton &#x000a0;S., Kane &#x000a0;J.J., Baptiste &#x000a0;T.A., Dunphy &#x000a0;K.A., Roberts &#x000a0;A.L., Schneider &#x000a0;S.S., Savage &#x000a0;E.M. &#x000a0;et al.. &#x000a0;<span class="ref-title">Genetic modifiers regulating DNA replication and double-strand break repair are associated with differences in mammary tumors in mouse models of Li-Fraumeni syndrome</span>. <span class="ref-journal">Oncogene</span>. 2021; <span class="ref-vol">40</span>:5026&#x02013;5037.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8349885/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34183771" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Genetic+modifiers+regulating+DNA+replication+and+double-strand+break+repair+are+associated+with+differences+in+mammary+tumors+in+mouse+models+of+Li-Fraumeni+syndrome&amp;volume=40&amp;publication_year=2021&amp;pages=5026-5037&amp;pmid=34183771&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B46">46. <span class="mixed-citation">

Ahn &#x000a0;J., Poyurovsky &#x000a0;M.V., Baptiste &#x000a0;N., Beckerman &#x000a0;R., Cain &#x000a0;C., Mattia &#x000a0;M., McKinney &#x000a0;K., Zhou &#x000a0;J., Zupnick &#x000a0;A., Gottifredi &#x000a0;V. &#x000a0;et al.. &#x000a0;<span class="ref-title">Dissection of the sequence-specific DNA binding and exonuclease activities reveals a superactive yet apoptotically impaired mutant p53 protein</span>. <span class="ref-journal">Cell Cycle</span>. 2009; <span class="ref-vol">8</span>:1603&#x02013;1615.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2898518/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19462533" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Cycle&amp;title=Dissection+of+the+sequence-specific+DNA+binding+and+exonuclease+activities+reveals+a+superactive+yet+apoptotically+impaired+mutant+p53+protein&amp;volume=8&amp;publication_year=2009&amp;pages=1603-1615&amp;pmid=19462533&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B47">47. <span class="mixed-citation">

Gole &#x000a0;B., Mian &#x000a0;E., Rall &#x000a0;M., Wiesm&#x000fc;ller &#x000a0;L. &#x000a0;<span class="ref-title">Base excision repair proteins couple activation-induced cytidine deaminase and endonuclease G during replication stress-induced MLL destabilization</span>. <span class="ref-journal">Leukemia</span>. 2018; <span class="ref-vol">32</span>:159&#x02013;167.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/28626219" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Leukemia&amp;title=Base+excision+repair+proteins+couple+activation-induced+cytidine+deaminase+and+endonuclease+G+during+replication+stress-induced+MLL+destabilization&amp;volume=32&amp;publication_year=2018&amp;pages=159-167&amp;pmid=28626219&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B48">48. <span class="mixed-citation">

Eberle &#x000a0;J., Wiehe &#x000a0;R.S., Gole &#x000a0;B., Mattis &#x000a0;L.J., Palmer &#x000a0;A., St&#x000e4;ndker &#x000a0;L., Forssmann &#x000a0;W.G., M&#x000fc;nch &#x000a0;J., Gebhardt &#x000a0;J.C.M., Wiesm&#x000fc;ller &#x000a0;L. &#x000a0;<span class="ref-title">A fibrinogen alpha fragment mitigates chemotherapy-induced MLL rearrangements</span>. <span class="ref-journal">Front. Oncol.</span> &#x000a0;2021; <span class="ref-vol">11</span>:689063.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8249925/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34222016" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Oncol.&amp;title=A+fibrinogen+alpha+fragment+mitigates+chemotherapy-induced+MLL+rearrangements&amp;volume=11&amp;publication_year=2021&amp;pages=689063&amp;pmid=34222016&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B49">49. <span class="mixed-citation">

Guo &#x000a0;Y., Rall-Scharpf &#x000a0;M., Bourdon &#x000a0;J.C., Wiesm&#x000fc;ller &#x000a0;L., Biber &#x000a0;S. &#x000a0;<span class="ref-title">p53 isoforms differentially impact on the pol&#x003b9; dependent DNA damage tolerance pathway</span>. <span class="ref-journal">Cell Death. Dis.</span> &#x000a0;2021; <span class="ref-vol">12</span>:941.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8514551/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34645785" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Death.+Dis.&amp;title=p53+isoforms+differentially+impact+on+the+pol&#x003b9;+dependent+DNA+damage+tolerance+pathway&amp;volume=12&amp;publication_year=2021&amp;pages=941&amp;pmid=34645785&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B50">50. <span class="mixed-citation">

Gottifredi &#x000a0;V., Wiesm&#x000fc;ller &#x000a0;L. &#x000a0;<span class="ref-title">The tip of an iceberg: replication-associated functions of the tumor suppressor p53</span>. <span class="ref-journal">Cancers</span>. 2018; <span class="ref-vol">10</span>:250.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6115784/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30060597" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancers&amp;title=The+tip+of+an+iceberg:+replication-associated+functions+of+the+tumor+suppressor+p53&amp;volume=10&amp;publication_year=2018&amp;pages=250&amp;pmid=30060597&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B51">51. <span class="mixed-citation">

Ireno &#x000a0;I.C., Wiehe &#x000a0;R.S., Stahl &#x000a0;A.I., Hampp &#x000a0;S., Aydin &#x000a0;S., Troester &#x000a0;M.A., Selivanova &#x000a0;G., Wiesm&#x000fc;ller &#x000a0;L. &#x000a0;<span class="ref-title">Modulation of the poly (ADP-ribose) polymerase inhibitor response and DNA recombination in breast cancer cells by drugs affecting endogenous wild-type p53</span>. <span class="ref-journal">Carcinogenesis</span>. 2014; <span class="ref-vol">35</span>:2273&#x02013;2282.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/25085902" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=Modulation+of+the+poly+(ADP-ribose)+polymerase+inhibitor+response+and+DNA+recombination+in+breast+cancer+cells+by+drugs+affecting+endogenous+wild-type+p53&amp;volume=35&amp;publication_year=2014&amp;pages=2273-2282&amp;pmid=25085902&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B52">52. <span class="mixed-citation">

Fischer &#x000a0;M. &#x000a0;<span class="ref-title">Conservation and divergence of the p53 gene regulatory network between mice and humans</span>. <span class="ref-journal">Oncogene</span>. 2019; <span class="ref-vol">38</span>:4095&#x02013;4109.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6755996/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30710145" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Conservation+and+divergence+of+the+p53+gene+regulatory+network+between+mice+and+humans&amp;volume=38&amp;publication_year=2019&amp;pages=4095-4109&amp;pmid=30710145&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B53">53. <span class="mixed-citation">

Liu &#x000a0;G., Chen &#x000a0;X. &#x000a0;<span class="ref-title">DNA polymerase eta, the product of the xeroderma pigmentosum variant gene and a target of p53, modulates the DNA damage checkpoint and p53 activation</span>. <span class="ref-journal">Mol. Cell. Biol.</span> &#x000a0;2006; <span class="ref-vol">26</span>:1398&#x02013;1413.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1367184/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16449651" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Biol.&amp;title=DNA+polymerase+eta,+the+product+of+the+xeroderma+pigmentosum+variant+gene+and+a+target+of+p53,+modulates+the+DNA+damage+checkpoint+and+p53+activation&amp;volume=26&amp;publication_year=2006&amp;pages=1398-1413&amp;pmid=16449651&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B54">54. <span class="mixed-citation">

Zauberman &#x000a0;A., Flusberg &#x000a0;D., Haupt &#x000a0;Y., Barak &#x000a0;Y., Oren &#x000a0;M. &#x000a0;<span class="ref-title">A functional p53-responsive intronic promoter is contained within the human mdm2 gene</span>. <span class="ref-journal">Nucleic Acids Res.</span> &#x000a0;1995; <span class="ref-vol">23</span>:2584&#x02013;2592.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC307078/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7651818" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=A+functional+p53-responsive+intronic+promoter+is+contained+within+the+human+mdm2+gene&amp;volume=23&amp;publication_year=1995&amp;pages=2584-2592&amp;pmid=7651818&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B55">55. <span class="mixed-citation">

T&#x000e9;cher &#x000a0;H., Koundrioukoff &#x000a0;S., Azar &#x000a0;D., Wilhelm &#x000a0;T., Carignon &#x000a0;S., Brison &#x000a0;O., Debatisse &#x000a0;M., Le&#x000a0;Tallec &#x000a0;B. &#x000a0;<span class="ref-title">Replication dynamics: biases and robustness of DNA fiber analysis</span>. <span class="ref-journal">J. Mol. Biol.</span> &#x000a0;2013; <span class="ref-vol">425</span>:4845&#x02013;4855.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/23557832" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Mol.+Biol.&amp;title=Replication+dynamics:+biases+and+robustness+of+DNA+fiber+analysis&amp;volume=425&amp;publication_year=2013&amp;pages=4845-4855&amp;pmid=23557832&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B56">56. <span class="mixed-citation">

Mansilla &#x000a0;S.F., Soria &#x000a0;G., Vallerga &#x000a0;M.B., Habif &#x000a0;M., Mart&#x000ed;nez-L&#x000f3;pez &#x000a0;W., Prives &#x000a0;C., Gottifredi &#x000a0;V. &#x000a0;<span class="ref-title">UV-triggered p21 degradation facilitates damaged-DNA replication and preserves genomic stability</span>. <span class="ref-journal">Nucleic Acids Res.</span> &#x000a0;2013; <span class="ref-vol">41</span>:6942&#x02013;6951.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3737556/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23723248" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=UV-triggered+p21+degradation+facilitates+damaged-DNA+replication+and+preserves+genomic+stability&amp;volume=41&amp;publication_year=2013&amp;pages=6942-6951&amp;pmid=23723248&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B57">57. <span class="mixed-citation">

Conti &#x000a0;C., Sacc&#x000e0; &#x000a0;B., Herrick &#x000a0;J., Lalou &#x000a0;C., Pommier &#x000a0;Y., Bensimon &#x000a0;A. &#x000a0;<span class="ref-title">Replication fork velocities at adjacent replication origins are coordinately modified during DNA replication in human cells</span>. <span class="ref-journal">Mol. Biol. Cell</span>. 2007; <span class="ref-vol">18</span>:3059&#x02013;3067.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1949372/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17522385" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Biol.+Cell&amp;title=Replication+fork+velocities+at+adjacent+replication+origins+are+coordinately+modified+during+DNA+replication+in+human+cells&amp;volume=18&amp;publication_year=2007&amp;pages=3059-3067&amp;pmid=17522385&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B58">58. <span class="mixed-citation">

Ge &#x000a0;X.Q., Jackson &#x000a0;D.A., Blow &#x000a0;J.J. &#x000a0;<span class="ref-title">Dormant origins licensed by excess Mcm2-7 are required for human cells to survive replicative stress</span>. <span class="ref-journal">Genes Dev.</span> &#x000a0;2007; <span class="ref-vol">21</span>:3331&#x02013;3341.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2113033/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18079179" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev.&amp;title=Dormant+origins+licensed+by+excess+Mcm2-7+are+required+for+human+cells+to+survive+replicative+stress&amp;volume=21&amp;publication_year=2007&amp;pages=3331-3341&amp;pmid=18079179&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B59">59. <span class="mixed-citation">

Courbet &#x000a0;S., Gay &#x000a0;S., Arnoult &#x000a0;N., Wronka &#x000a0;G., Anglana &#x000a0;M., Brison &#x000a0;O., Debatisse &#x000a0;M. &#x000a0;<span class="ref-title">Replication fork movement sets chromatin loop size and origin choice in mammalian cells</span>. <span class="ref-journal">Nature</span>. 2008; <span class="ref-vol">455</span>:557&#x02013;560.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/18716622" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Replication+fork+movement+sets+chromatin+loop+size+and+origin+choice+in+mammalian+cells&amp;volume=455&amp;publication_year=2008&amp;pages=557-560&amp;pmid=18716622&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B60">60. <span class="mixed-citation">

Chang &#x000a0;D.J., Cimprich &#x000a0;K.A. &#x000a0;<span class="ref-title">DNA damage tolerance: when it's OK to make mistakes</span>. <span class="ref-journal">Nat. Chem. Biol.</span> &#x000a0;2009; <span class="ref-vol">5</span>:82&#x02013;90.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2663399/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19148176" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Chem.+Biol.&amp;title=DNA+damage+tolerance:+when+it's+OK+to+make+mistakes&amp;volume=5&amp;publication_year=2009&amp;pages=82-90&amp;pmid=19148176&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B61">61. <span class="mixed-citation">

Vaziri &#x000a0;C., Rogozin &#x000a0;I.B., Gu &#x000a0;Q., Wu &#x000a0;D., Day &#x000a0;T.A. &#x000a0;<span class="ref-title">Unravelling roles of error-prone DNA polymerases in shaping cancer genomes</span>. <span class="ref-journal">Oncogene</span>. 2021; <span class="ref-vol">40</span>:6549&#x02013;6565.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8639439/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34663880" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Unravelling+roles+of+error-prone+DNA+polymerases+in+shaping+cancer+genomes&amp;volume=40&amp;publication_year=2021&amp;pages=6549-6565&amp;pmid=34663880&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B62">62. <span class="mixed-citation">

Gagou &#x000a0;M.E., Zuazua-Villar &#x000a0;P., Meuth &#x000a0;M. &#x000a0;<span class="ref-title">Enhanced H2AX phosphorylation, DNA replication fork arrest, and cell death in the absence of Chk1</span>. <span class="ref-journal">Mol. Biol. Cell</span>. 2010; <span class="ref-vol">21</span>:739&#x02013;752.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2828961/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20053681" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Biol.+Cell&amp;title=Enhanced+H2AX+phosphorylation,+DNA+replication+fork+arrest,+and+cell+death+in+the+absence+of+Chk1&amp;volume=21&amp;publication_year=2010&amp;pages=739-752&amp;pmid=20053681&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B63">63. <span class="mixed-citation">

Vassin &#x000a0;V.M., Anantha &#x000a0;R.W., Sokolova &#x000a0;E., Kanner &#x000a0;S., Borowiec &#x000a0;J.A. &#x000a0;<span class="ref-title">Human RPA phosphorylation by ATR stimulates DNA synthesis and prevents ssDNA accumulation during DNA-replication stress</span>. <span class="ref-journal">J. Cell Sci.</span> &#x000a0;2009; <span class="ref-vol">122</span>:4070&#x02013;4080.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2776501/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19843584" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Cell+Sci.&amp;title=Human+RPA+phosphorylation+by+ATR+stimulates+DNA+synthesis+and+prevents+ssDNA+accumulation+during+DNA-replication+stress&amp;volume=122&amp;publication_year=2009&amp;pages=4070-4080&amp;pmid=19843584&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B64">64. <span class="mixed-citation">

Alvino &#x000a0;G.M., Collingwood &#x000a0;D., Murphy &#x000a0;J.M., Delrow &#x000a0;J., Brewer &#x000a0;B.J., Raghuraman &#x000a0;M.K. &#x000a0;<span class="ref-title">Replication in hydroxyurea: it's a matter of time</span>. <span class="ref-journal">Mol. Cell. Biol.</span> &#x000a0;2007; <span class="ref-vol">27</span>:6396&#x02013;6406.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2099622/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17636020" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Biol.&amp;title=Replication+in+hydroxyurea:+it's+a+matter+of+time&amp;volume=27&amp;publication_year=2007&amp;pages=6396-6406&amp;pmid=17636020&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B65">65. <span class="mixed-citation">

Dozic &#x000a0;S., Howden &#x000a0;E.J., Bell &#x000a0;J.R., Mellor &#x000a0;K.M., Delbridge &#x000a0;L.M.D., Weeks &#x000a0;K.L. &#x000a0;<span class="ref-title">Cellular mechanisms mediating exercise-induced protection against cardiotoxic anthracycline cancer therapy</span>. <span class="ref-journal">Cells</span>. 2023; <span class="ref-vol">12</span>:1312.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10177216/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37174712" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cells&amp;title=Cellular+mechanisms+mediating+exercise-induced+protection+against+cardiotoxic+anthracycline+cancer+therapy&amp;volume=12&amp;publication_year=2023&amp;pages=1312&amp;pmid=37174712&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B66">66. <span class="mixed-citation">

Carvalho &#x000a0;C., Santos &#x000a0;R.X., Cardoso &#x000a0;S., Correia &#x000a0;S., Oliveira &#x000a0;P.J., Santos &#x000a0;M.S., Moreira &#x000a0;P.I. &#x000a0;<span class="ref-title">Doxorubicin: the good, the bad and the ugly effect</span>. <span class="ref-journal">Curr. Med. Chem.</span> &#x000a0;2009; <span class="ref-vol">16</span>:3267&#x02013;3285.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/19548866" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Med.+Chem.&amp;title=Doxorubicin:+the+good,+the+bad+and+the+ugly+effect&amp;volume=16&amp;publication_year=2009&amp;pages=3267-3285&amp;pmid=19548866&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B67">67. <span class="mixed-citation">

Maya-Mendoza &#x000a0;A., Moudry &#x000a0;P., Merchut-Maya &#x000a0;J.M., Lee &#x000a0;M., Strauss &#x000a0;R., Bartek &#x000a0;J. &#x000a0;<span class="ref-title">High speed of fork progression induces DNA replication stress and genomic instability</span>. <span class="ref-journal">Nature</span>. 2018; <span class="ref-vol">559</span>:279&#x02013;284.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/29950726" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=High+speed+of+fork+progression+induces+DNA+replication+stress+and+genomic+instability&amp;volume=559&amp;publication_year=2018&amp;pages=279-284&amp;pmid=29950726&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B68">68. <span class="mixed-citation">

Sarni &#x000a0;D., Barroso &#x000a0;S., Shtrikman &#x000a0;A., Irony-Tur&#x000a0;Sinai &#x000a0;M., Oren &#x000a0;Y.S., Aguilera &#x000a0;A., Kerem &#x000a0;B &#x000a0;<span class="ref-title">Topoisomerase 1-dependent R-loop deficiency drives accelerated replication and genomic instability</span>. <span class="ref-journal">Cell Rep.</span> &#x000a0;2022; <span class="ref-vol">40</span>:111397.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9532845/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36170822" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Rep.&amp;title=Topoisomerase+1-dependent+R-loop+deficiency+drives+accelerated+replication+and+genomic+instability&amp;volume=40&amp;publication_year=2022&amp;pages=111397&amp;pmid=36170822&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B69">69. <span class="mixed-citation">

Kannouche &#x000a0;P.L., Lehmann &#x000a0;A.R. &#x000a0;<span class="ref-title">Ubiquitination of PCNA and the polymerase switch in human cells</span>. <span class="ref-journal">Cell Cycle</span>. 2004; <span class="ref-vol">3</span>:1011&#x02013;1013.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/15280666" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Cycle&amp;title=Ubiquitination+of+PCNA+and+the+polymerase+switch+in+human+cells&amp;volume=3&amp;publication_year=2004&amp;pages=1011-1013&amp;pmid=15280666&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B70">70. <span class="mixed-citation">

Lehmann &#x000a0;A.R., Niimi &#x000a0;A., Ogi &#x000a0;T., Brown &#x000a0;S., Sabbioneda &#x000a0;S., Wing &#x000a0;J.F., Kannouche &#x000a0;P.L., Green &#x000a0;C.M. &#x000a0;<span class="ref-title">Translesion synthesis: y-family polymerases and the polymerase switch</span>. <span class="ref-journal">DNA Repair (Amst.)</span>. 2007; <span class="ref-vol">6</span>:891&#x02013;899.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/17363342" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=DNA+Repair+(Amst.)&amp;title=Translesion+synthesis:+y-family+polymerases+and+the+polymerase+switch&amp;volume=6&amp;publication_year=2007&amp;pages=891-899&amp;pmid=17363342&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B71">71. <span class="mixed-citation">

McCulloch &#x000a0;S.D., Kokoska &#x000a0;R.J., Masutani &#x000a0;C., Iwai &#x000a0;S., Hanaoka &#x000a0;F., Kunkel &#x000a0;T.A. &#x000a0;<span class="ref-title">Preferential cis-syn thymine dimer bypass by DNA polymerase eta occurs with biased fidelity</span>. <span class="ref-journal">Nature</span>. 2004; <span class="ref-vol">428</span>:97&#x02013;100.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/14999287" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Preferential+cis-syn+thymine+dimer+bypass+by+DNA+polymerase+eta+occurs+with+biased+fidelity&amp;volume=428&amp;publication_year=2004&amp;pages=97-100&amp;pmid=14999287&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B72">72. <span class="mixed-citation">

Lehmann &#x000a0;A.R. &#x000a0;<span class="ref-title">Translesion synthesis in mammalian cells</span>. <span class="ref-journal">Exp. Cell Res.</span> &#x000a0;2006; <span class="ref-vol">312</span>:2673&#x02013;2676.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/16854411" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Exp.+Cell+Res.&amp;title=Translesion+synthesis+in+mammalian+cells&amp;volume=312&amp;publication_year=2006&amp;pages=2673-2676&amp;pmid=16854411&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B73">73. <span class="mixed-citation">

Bergoglio &#x000a0;V., Boyer &#x000a0;A.S., Walsh &#x000a0;E., Naim &#x000a0;V., Legube &#x000a0;G., Lee &#x000a0;M.Y., Rey &#x000a0;L., Rosselli &#x000a0;F., Cazaux &#x000a0;C., Eckert &#x000a0;K.A. &#x000a0;et al.. &#x000a0;<span class="ref-title">DNA synthesis by pol &#x003b7; promotes fragile site stability by preventing under-replicated DNA in mitosis</span>. <span class="ref-journal">J. Cell Biol.</span> &#x000a0;2013; <span class="ref-vol">201</span>:395&#x02013;408.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3639397/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23609533" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Cell+Biol.&amp;title=DNA+synthesis+by+pol+&#x003b7;+promotes+fragile+site+stability+by+preventing+under-replicated+DNA+in+mitosis&amp;volume=201&amp;publication_year=2013&amp;pages=395-408&amp;pmid=23609533&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B74">74. <span class="mixed-citation">

Bertolin &#x000a0;A.P., Mansilla &#x000a0;S.F., Gottifredi &#x000a0;V. &#x000a0;<span class="ref-title">The identification of translesion DNA synthesis regulators: inhibitors in the spotlight</span>. <span class="ref-journal">DNA Repair (Amst.)</span>. 2015; <span class="ref-vol">32</span>:158&#x02013;164.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4522396/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26002196" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=DNA+Repair+(Amst.)&amp;title=The+identification+of+translesion+DNA+synthesis+regulators:+inhibitors+in+the+spotlight&amp;volume=32&amp;publication_year=2015&amp;pages=158-164&amp;pmid=26002196&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B75">75. <span class="mixed-citation">

Yang &#x000a0;W., Gao &#x000a0;Y. &#x000a0;<span class="ref-title">Translesion and repair DNA polymerases: diverse structure and mechanism</span>. <span class="ref-journal">Ann. Rev. Biochem.</span> &#x000a0;2018; <span class="ref-vol">87</span>:239&#x02013;261.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6098713/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29494238" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Rev.+Biochem.&amp;title=Translesion+and+repair+DNA+polymerases:+diverse+structure+and+mechanism&amp;volume=87&amp;publication_year=2018&amp;pages=239-261&amp;pmid=29494238&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B76">76. <span class="mixed-citation">

Schiavone &#x000a0;D., Jozwiakowski &#x000a0;S.K., Romanello &#x000a0;M., Guilbaud &#x000a0;G., Guilliam &#x000a0;T.A., Bailey &#x000a0;L.J., Sale &#x000a0;J.E., Doherty &#x000a0;A.J. &#x000a0;<span class="ref-title">PrimPol is required for replicative tolerance of G quadruplexes in vertebrate cells</span>. <span class="ref-journal">Mol. Cell</span>. 2016; <span class="ref-vol">61</span>:161&#x02013;169.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4712188/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26626482" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=PrimPol+is+required+for+replicative+tolerance+of+G+quadruplexes+in+vertebrate+cells&amp;volume=61&amp;publication_year=2016&amp;pages=161-169&amp;pmid=26626482&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B77">77. <span class="mixed-citation">

Sale &#x000a0;J.E. &#x000a0;<span class="ref-title">Starting over: primpol reprimes after chain termination</span>. <span class="ref-journal">Cell Cycle</span>. 2016; <span class="ref-vol">15</span>:2099&#x02013;2100.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4993428/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27384604" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Cycle&amp;title=Starting+over:+primpol+reprimes+after+chain+termination&amp;volume=15&amp;publication_year=2016&amp;pages=2099-2100&amp;pmid=27384604&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B78">78. <span class="mixed-citation">

B&#x000e9;tous &#x000a0;R., Couch &#x000a0;F.B., Mason &#x000a0;A.C., Eichman &#x000a0;B.F., Manosas &#x000a0;M., Cortez &#x000a0;D. &#x000a0;<span class="ref-title">Substrate-selective repair and restart of replication forks by DNA translocases</span>. <span class="ref-journal">Cell Rep.</span> &#x000a0;2013; <span class="ref-vol">3</span>:1958&#x02013;1969.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3700663/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23746452" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Rep.&amp;title=Substrate-selective+repair+and+restart+of+replication+forks+by+DNA+translocases&amp;volume=3&amp;publication_year=2013&amp;pages=1958-1969&amp;pmid=23746452&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B79">79. <span class="mixed-citation">

Berti &#x000a0;M., Cortez &#x000a0;D., Lopes &#x000a0;M. &#x000a0;<span class="ref-title">The plasticity of DNA replication forks in response to clinically relevant genotoxic stress</span>. <span class="ref-journal">Nat. Rev. Mol. Cell Biol.</span> &#x000a0;2020; <span class="ref-vol">21</span>:633&#x02013;651.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/32612242" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Mol.+Cell+Biol.&amp;title=The+plasticity+of+DNA+replication+forks+in+response+to+clinically+relevant+genotoxic+stress&amp;volume=21&amp;publication_year=2020&amp;pages=633-651&amp;pmid=32612242&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B80">80. <span class="mixed-citation">

Bhat &#x000a0;K.P., Cortez &#x000a0;D. &#x000a0;<span class="ref-title">RPA and RAD51: fork reversal, fork protection, and genome stability</span>. <span class="ref-journal">Nat. Struct. Mol. Biol.</span> &#x000a0;2018; <span class="ref-vol">25</span>:446&#x02013;453.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6006513/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29807999" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Struct.+Mol.+Biol.&amp;title=RPA+and+RAD51:+fork+reversal,+fork+protection,+and+genome+stability&amp;volume=25&amp;publication_year=2018&amp;pages=446-453&amp;pmid=29807999&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B81">81. <span class="mixed-citation">

Quinet &#x000a0;A., Tirman &#x000a0;S., Jackson &#x000a0;J., &#x00160;vikovi&#x00107; &#x000a0;S., Lema&#x000e7;on &#x000a0;D., Carvajal-Maldonado &#x000a0;D., Gonz&#x000e1;lez-Acosta &#x000a0;D., Vessoni &#x000a0;A.T., Cybulla &#x000a0;E., Wood &#x000a0;M. &#x000a0;et al.. &#x000a0;<span class="ref-title">PRIMPOL-mediated adaptive response suppresses replication fork reversal in BRCA-deficient cells</span>. <span class="ref-journal">Mol. Cell</span>. 2020; <span class="ref-vol">77</span>:461&#x02013;474.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7007862/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31676232" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=PRIMPOL-mediated+adaptive+response+suppresses+replication+fork+reversal+in+BRCA-deficient+cells&amp;volume=77&amp;publication_year=2020&amp;pages=461-474&amp;pmid=31676232&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B82">82. <span class="mixed-citation">

Bai &#x000a0;G., Kermi &#x000a0;C., Stoy &#x000a0;H., Schiltz &#x000a0;C.J., Bacal &#x000a0;J., Zaino &#x000a0;A.M., Hadden &#x000a0;M.K., Eichman &#x000a0;B.F., Lopes &#x000a0;M., Cimprich &#x000a0;K.A. &#x000a0;<span class="ref-title">HLTF promotes fork reversal, limiting replication stress resistance and preventing multiple mechanisms of unrestrained DNA synthesis</span>. <span class="ref-journal">Mol. Cell</span>. 2020; <span class="ref-vol">78</span>:1237&#x02013;1251.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7305998/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32442397" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=HLTF+promotes+fork+reversal,+limiting+replication+stress+resistance+and+preventing+multiple+mechanisms+of+unrestrained+DNA+synthesis&amp;volume=78&amp;publication_year=2020&amp;pages=1237-1251&amp;pmid=32442397&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B83">83. <span class="mixed-citation">

Taglialatela &#x000a0;A., Leuzzi &#x000a0;G., Sannino &#x000a0;V., Cuella-Martin &#x000a0;R., Huang &#x000a0;J.W., Wu-Baer &#x000a0;F., Baer &#x000a0;R., Costanzo &#x000a0;V., Ciccia &#x000a0;A. &#x000a0;<span class="ref-title">REV1-Pol&#x003b6; maintains the viability of homologous recombination-deficient cancer cells through mutagenic repair of PRIMPOL-dependent ssDNA gaps</span>. <span class="ref-journal">Mol. Cell</span>. 2021; <span class="ref-vol">81</span>:4008&#x02013;4025.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8500949/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34508659" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=REV1-Pol&#x003b6;+maintains+the+viability+of+homologous+recombination-deficient+cancer+cells+through+mutagenic+repair+of+PRIMPOL-dependent+ssDNA+gaps&amp;volume=81&amp;publication_year=2021&amp;pages=4008-4025&amp;pmid=34508659&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B84">84. <span class="mixed-citation">

Tirman &#x000a0;S., Quinet &#x000a0;A., Wood &#x000a0;M., Meroni &#x000a0;A., Cybulla &#x000a0;E., Jackson &#x000a0;J., Pegoraro &#x000a0;S., Simoneau &#x000a0;A., Zou &#x000a0;L., Vindigni &#x000a0;A. &#x000a0;<span class="ref-title">Temporally distinct post-replicative repair mechanisms fill PRIMPOL-dependent ssDNA gaps in human cells</span>. <span class="ref-journal">Mol. Cell</span>. 2021; <span class="ref-vol">81</span>:4026&#x02013;4040.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8555837/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34624216" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=Temporally+distinct+post-replicative+repair+mechanisms+fill+PRIMPOL-dependent+ssDNA+gaps+in+human+cells&amp;volume=81&amp;publication_year=2021&amp;pages=4026-4040&amp;pmid=34624216&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B85">85. <span class="mixed-citation">

Poole &#x000a0;L.A., Cortez &#x000a0;D. &#x000a0;<span class="ref-title">Functions of SMARCAL1, ZRANB3, and HLTF in maintaining genome stability</span>. <span class="ref-journal">Crit. Rev. Biochem. Mol. Biol.</span> &#x000a0;2017; <span class="ref-vol">52</span>:696&#x02013;714.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5962345/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28954549" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Crit.+Rev.+Biochem.+Mol.+Biol.&amp;title=Functions+of+SMARCAL1,+ZRANB3,+and+HLTF+in+maintaining+genome+stability&amp;volume=52&amp;publication_year=2017&amp;pages=696-714&amp;pmid=28954549&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B86">86. <span class="mixed-citation">

Jacobs &#x000a0;K., Doerdelmann &#x000a0;C., Krietsch &#x000a0;J., Gonz&#x000e1;lez-Acosta &#x000a0;D., Mathis &#x000a0;N., Kushinsky &#x000a0;S., Guarino &#x000a0;E., G&#x000f3;mez-Escolar &#x000a0;C., Martinez &#x000a0;D., Schmid &#x000a0;J.A. &#x000a0;et al.. &#x000a0;<span class="ref-title">Stress-triggered hematopoietic stem cell proliferation relies on PrimPol-mediated repriming</span>. <span class="ref-journal">Mol. Cell</span>. 2022; <span class="ref-vol">82</span>:4176&#x02013;4188.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10251193/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36152632" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=Stress-triggered+hematopoietic+stem+cell+proliferation+relies+on+PrimPol-mediated+repriming&amp;volume=82&amp;publication_year=2022&amp;pages=4176-4188&amp;pmid=36152632&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B87">87. <span class="mixed-citation">

Avkin &#x000a0;S., Sevilya &#x000a0;Z., Toube &#x000a0;L., Geacintov &#x000a0;N., Chaney &#x000a0;S.G., Oren &#x000a0;M., Livneh &#x000a0;Z. &#x000a0;<span class="ref-title">p53 and p21 regulate error-prone DNA repair to yield a lower mutation load</span>. <span class="ref-journal">Mol. Cell</span>. 2006; <span class="ref-vol">22</span>:407&#x02013;413.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/16678112" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=p53+and+p21+regulate+error-prone+DNA+repair+to+yield+a+lower+mutation+load&amp;volume=22&amp;publication_year=2006&amp;pages=407-413&amp;pmid=16678112&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B88">88. <span class="mixed-citation">

Soria &#x000a0;G., Podhajcer &#x000a0;O., Prives &#x000a0;C., Gottifredi &#x000a0;V. &#x000a0;<span class="ref-title">P21Cip1/WAF1 downregulation is required for efficient PCNA ubiquitination after UV irradiation</span>. <span class="ref-journal">Oncogene</span>. 2006; <span class="ref-vol">25</span>:2829&#x02013;2838.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/16407842" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=P21Cip1/WAF1+downregulation+is+required+for+efficient+PCNA+ubiquitination+after+UV+irradiation&amp;volume=25&amp;publication_year=2006&amp;pages=2829-2838&amp;pmid=16407842&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B89">89. <span class="mixed-citation">

Kile &#x000a0;A.C., Chavez &#x000a0;D.A., Bacal &#x000a0;J., Eldirany &#x000a0;S., Korzhnev &#x000a0;D.M., Bezsonova &#x000a0;I., Eichman &#x000a0;B.F., Cimprich &#x000a0;K.A. &#x000a0;<span class="ref-title">HLTF&#x02019;s ancient HIRAN domain binds 3' DNA ends to drive replication fork reversal</span>. <span class="ref-journal">Mol. Cell</span>. 2015; <span class="ref-vol">58</span>:1090&#x02013;1100.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4475461/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26051180" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=HLTF&#x02019;s+ancient+HIRAN+domain+binds+3'+DNA+ends+to+drive+replication+fork+reversal&amp;volume=58&amp;publication_year=2015&amp;pages=1090-1100&amp;pmid=26051180&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B90">90. <span class="mixed-citation">

Lin &#x000a0;J.R., Zeman &#x000a0;M.K., Chen &#x000a0;J.Y., Yee &#x000a0;M.C., Cimprich &#x000a0;K.A. &#x000a0;<span class="ref-title">SHPRH and HLTF act in a damage-specific manner to coordinate different forms of postreplication repair and prevent mutagenesis</span>. <span class="ref-journal">Mol. Cell</span>. 2011; <span class="ref-vol">42</span>:237&#x02013;249.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3080461/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21396873" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=SHPRH+and+HLTF+act+in+a+damage-specific+manner+to+coordinate+different+forms+of+postreplication+repair+and+prevent+mutagenesis&amp;volume=42&amp;publication_year=2011&amp;pages=237-249&amp;pmid=21396873&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B91">91. <span class="mixed-citation">

Masuda &#x000a0;Y., Suzuki &#x000a0;M., Kawai &#x000a0;H., Hishiki &#x000a0;A., Hashimoto &#x000a0;H., Masutani &#x000a0;C., Hishida &#x000a0;T., Suzuki &#x000a0;F., Kamiya &#x000a0;K. &#x000a0;<span class="ref-title">En bloc transfer of polyubiquitin chains to PCNA in vitro is mediated by two different human E2-E3 pairs</span>. <span class="ref-journal">Nucleic Acids Res.</span> &#x000a0;2012; <span class="ref-vol">40</span>:10394&#x02013;10407.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3488225/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22904075" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=En+bloc+transfer+of+polyubiquitin+chains+to+PCNA+in+vitro+is+mediated+by+two+different+human+E2-E3+pairs&amp;volume=40&amp;publication_year=2012&amp;pages=10394-10407&amp;pmid=22904075&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B92">92. <span class="mixed-citation">

Fumasoni &#x000a0;M., Zwicky &#x000a0;K., Vanoli &#x000a0;F., Lopes &#x000a0;M., Branzei &#x000a0;D. &#x000a0;<span class="ref-title">Error-free DNA damage tolerance and sister chromatid proximity during DNA replication rely on the Pol&#x003b1;/Primase/Ctf4 complex</span>. <span class="ref-journal">Mol. Cell</span>. 2015; <span class="ref-vol">57</span>:812&#x02013;823.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4352764/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25661486" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=Error-free+DNA+damage+tolerance+and+sister+chromatid+proximity+during+DNA+replication+rely+on+the+Pol&#x003b1;/Primase/Ctf4+complex&amp;volume=57&amp;publication_year=2015&amp;pages=812-823&amp;pmid=25661486&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B93">93. <span class="mixed-citation">

Piberger &#x000a0;A.L., Bowry &#x000a0;A., Kelly &#x000a0;R.D.W., Walker &#x000a0;A.K., Gonz&#x000e1;lez-Acosta &#x000a0;D., Bailey &#x000a0;L.J., Doherty &#x000a0;A.J., M&#x000e9;ndez &#x000a0;J., Morris &#x000a0;J.R., Bryant &#x000a0;H.E. &#x000a0;et al.. &#x000a0;<span class="ref-title">PrimPol-dependent single-stranded gap formation mediates homologous recombination at bulky DNA adducts</span>. <span class="ref-journal">Nat. Commun.</span> &#x000a0;2020; <span class="ref-vol">11</span>:5863.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7673990/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33203852" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=PrimPol-dependent+single-stranded+gap+formation+mediates+homologous+recombination+at+bulky+DNA+adducts&amp;volume=11&amp;publication_year=2020&amp;pages=5863&amp;pmid=33203852&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B94">94. <span class="mixed-citation">

Vindigni &#x000a0;A., Lopes &#x000a0;M. &#x000a0;<span class="ref-title">Combining electron microscopy with single molecule DNA fiber approaches to study DNA replication dynamics</span>. <span class="ref-journal">Biophys. Chem.</span> &#x000a0;2017; <span class="ref-vol">225</span>:3&#x02013;9.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5457720/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27939387" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biophys.+Chem.&amp;title=Combining+electron+microscopy+with+single+molecule+DNA+fiber+approaches+to+study+DNA+replication+dynamics&amp;volume=225&amp;publication_year=2017&amp;pages=3-9&amp;pmid=27939387&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B95">95. <span class="mixed-citation">

Jackson &#x000a0;J., Vindigni &#x000a0;A. &#x000a0;<span class="ref-title">Studying single-stranded DNA gaps at replication intermediates by electron microscopy</span>. <span class="ref-journal">Methods Mol. Biol.</span> &#x000a0;2022; <span class="ref-vol">2444</span>:81&#x02013;103.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9728461/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35290633" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Mol.+Biol.&amp;title=Studying+single-stranded+DNA+gaps+at+replication+intermediates+by+electron+microscopy&amp;volume=2444&amp;publication_year=2022&amp;pages=81-103&amp;pmid=35290633&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B96">96. <span class="mixed-citation">

Unk &#x000a0;I., Hajd&#x000fa; &#x000a0;I., Blasty&#x000e1;k &#x000a0;A., Haracska &#x000a0;L. &#x000a0;<span class="ref-title">Role of yeast Rad5 and its human orthologs, HLTF and SHPRH in DNA damage tolerance</span>. <span class="ref-journal">DNA Repair (Amst.)</span>. 2010; <span class="ref-vol">9</span>:257&#x02013;267.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/20096653" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=DNA+Repair+(Amst.)&amp;title=Role+of+yeast+Rad5+and+its+human+orthologs,+HLTF+and+SHPRH+in+DNA+damage+tolerance&amp;volume=9&amp;publication_year=2010&amp;pages=257-267&amp;pmid=20096653&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B97">97. <span class="mixed-citation">

Saugar &#x000a0;I., Ortiz-Baz&#x000e1;n &#x000a0;M.&#x000c1;., Tercero &#x000a0;J.A. &#x000a0;<span class="ref-title">Tolerating DNA damage during eukaryotic chromosome replication</span>. <span class="ref-journal">Exp. Cell Res.</span> &#x000a0;2014; <span class="ref-vol">329</span>:170&#x02013;177.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/25038291" ref="reftype=pubmed&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Exp.+Cell+Res.&amp;title=Tolerating+DNA+damage+during+eukaryotic+chromosome+replication&amp;volume=329&amp;publication_year=2014&amp;pages=170-177&amp;pmid=25038291&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11039994&amp;issue-id=460879&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Nucleic Acids Research</span> are provided here courtesy of <strong>Oxford University Press</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11039994/pdf/gkae061.pdf" class="int-view">PDF (3.4M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11039994/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11039994/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11039994" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11039994%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="I4esCK2IU6pJ9cLlf0abW6cVwaTFl4xigG0PZc8s0qynVco4kVopKfYzixqfufHq">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11039994%2F&amp;text=The%20levels%20of%20p53%20govern%20the%20hierarchy%20of%20DNA%20damage%20tolerance%20pathway%20usage" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11039994%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11039994/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11039994/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38321962/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11039994/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38321962/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11039994/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11039994/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC11039994/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
