
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Structure-based network analysis predicts pathogenic variants in human proteins associated with inherited retinal disease - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="322C300B24F3920500003C7200E35D7F.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="npjgenmed" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC11130145/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11130145/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="NPJ Genomic Medicine" /><meta name="citation_title" content="Structure-based network analysis predicts pathogenic variants in human proteins associated with inherited retinal disease" /><meta name="citation_author" content="Blake M. Hauser" /><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA USA" /><meta name="citation_author" content="Yuyang Luo" /><meta name="citation_author_institution" content="Harvard Medical School, Department of Ophthalmology, Massachusetts Eye and Ear, Boston, MA USA" /><meta name="citation_author" content="Anusha Nathan" /><meta name="citation_author_institution" content="Ragon Institute of Mass General, MIT, and Harvard, Cambridge, MA USA" /><meta name="citation_author" content="Ahmad Al-Moujahed" /><meta name="citation_author_institution" content="Harvard Medical School, Department of Ophthalmology, Massachusetts Eye and Ear, Boston, MA USA" /><meta name="citation_author" content="Demetrios G. Vavvas" /><meta name="citation_author_institution" content="Harvard Medical School, Department of Ophthalmology, Massachusetts Eye and Ear, Boston, MA USA" /><meta name="citation_author" content="Jason Comander" /><meta name="citation_author_institution" content="Harvard Medical School, Department of Ophthalmology, Massachusetts Eye and Ear, Boston, MA USA" /><meta name="citation_author" content="Eric A. Pierce" /><meta name="citation_author_institution" content="Harvard Medical School, Department of Ophthalmology, Massachusetts Eye and Ear, Boston, MA USA" /><meta name="citation_author" content="Emily M. Place" /><meta name="citation_author_institution" content="Harvard Medical School, Department of Ophthalmology, Massachusetts Eye and Ear, Boston, MA USA" /><meta name="citation_author" content="Kinga M. Bujakowska" /><meta name="citation_author_institution" content="Harvard Medical School, Department of Ophthalmology, Massachusetts Eye and Ear, Boston, MA USA" /><meta name="citation_author" content="Gaurav D. Gaiha" /><meta name="citation_author_institution" content="Ragon Institute of Mass General, MIT, and Harvard, Cambridge, MA USA" /><meta name="citation_author_institution" content="Division of Gastroenterology, Massachusetts General Hospital, Boston, MA USA" /><meta name="citation_author" content="Elizabeth J. Rossin" /><meta name="citation_author_institution" content="Harvard Medical School, Department of Ophthalmology, Massachusetts Eye and Ear, Boston, MA USA" /><meta name="citation_publication_date" content="2024" /><meta name="citation_volume" content="9" /><meta name="citation_doi" content="10.1038/s41525-024-00416-w" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC11130145/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC11130145/" /><meta name="citation_pmid" content="38802398" /><meta name="DC.Title" content="Structure-based network analysis predicts pathogenic variants in human proteins associated with inherited retinal disease" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Nature Publishing Group" /><meta name="DC.Contributor" content="Blake M. Hauser" /><meta name="DC.Contributor" content="Yuyang Luo" /><meta name="DC.Contributor" content="Anusha Nathan" /><meta name="DC.Contributor" content="Ahmad Al-Moujahed" /><meta name="DC.Contributor" content="Demetrios G. Vavvas" /><meta name="DC.Contributor" content="Jason Comander" /><meta name="DC.Contributor" content="Eric A. Pierce" /><meta name="DC.Contributor" content="Emily M. Place" /><meta name="DC.Contributor" content="Kinga M. Bujakowska" /><meta name="DC.Contributor" content="Gaurav D. Gaiha" /><meta name="DC.Contributor" content="Elizabeth J. Rossin" /><meta name="DC.Date" content="2024" /><meta name="DC.Identifier" content="10.1038/s41525-024-00416-w" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Structure-based network analysis predicts pathogenic variants in human proteins associated with inherited retinal disease" /><meta property="og:type" content="article" /><meta property="og:description" content="Advances in gene sequencing technologies have accelerated the identification of genetic variants, but better tools are needed to understand which are causal of disease. This would be particularly useful in fields where gene therapy is a potential therapeutic ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11130145/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11130145/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/3307/">NPJ Genom Med</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/452431/">v.9; 2024</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11130145
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11130145/pdf/41525_2024_Article_416.pdf" class="int-view">PDF (2.2M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11130145/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11130145/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11130145" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11130145%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="f2TCCiLN3mWZjSRY6vXq6wHQxePJzIufxC8Pukuyg5PhNdgPpa2RESOwbkjqV8EQ">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11130145%2F&amp;text=Structure-based%20network%20analysis%20predicts%20pathogenic%20variants%20in%20human%20proteins%20associated%20with%20inherited%20retinal%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11130145%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11130145/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11130145/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38802398/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11130145/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38802398/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11130145/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11130145/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/3307/">NPJ Genom Med</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/452431/">v.9; 2024</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11130145
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-npjgenmed.jpg" alt="Logo of npjgenmed" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.nature.com/npjgenmed" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">NPJ Genom Med.</span> 2024; 9: 31. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2024 May 27. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1038%2Fs41525-024-00416-w" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/s41525-024-00416-w</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC11130145</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38802398">38802398</a></div></div></div><h1 class="content-title">Structure-based network analysis predicts pathogenic variants in human proteins associated with inherited retinal disease</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hauser%20BM%5BAuthor%5D" class="affpopup" co-rid="_co_idm139911460809696" co-class="co-affbox">Blake M. Hauser</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Luo%20Y%5BAuthor%5D" class="affpopup" co-rid="_co_idm139911464113024" co-class="co-affbox">Yuyang Luo</a>,<sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Nathan%20A%5BAuthor%5D" class="affpopup" co-rid="_co_idm139911458916480" co-class="co-affbox">Anusha Nathan</a>,<sup>3</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Al-Moujahed%20A%5BAuthor%5D" class="affpopup" co-rid="_co_idm139911447494896" co-class="co-affbox">Ahmad Al-Moujahed</a>,<sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Vavvas%20DG%5BAuthor%5D" class="affpopup" co-rid="_co_idm139911447493248" co-class="co-affbox">Demetrios G. Vavvas</a>,<sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Comander%20J%5BAuthor%5D" class="affpopup" co-rid="_co_idm139911453366816" co-class="co-affbox">Jason Comander</a>,<sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Pierce%20EA%5BAuthor%5D" class="affpopup" co-rid="_co_idm139911464506272" co-class="co-affbox">Eric A. Pierce</a>,<sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Place%20EM%5BAuthor%5D" class="affpopup" co-rid="_co_idm139911464504288" co-class="co-affbox">Emily M. Place</a>,<sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Bujakowska%20KM%5BAuthor%5D" class="affpopup" co-rid="_co_idm139911460862816" co-class="co-affbox">Kinga M. Bujakowska</a>,<sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gaiha%20GD%5BAuthor%5D" class="affpopup" co-rid="_co_idm139911458634064" co-class="co-affbox">Gaurav D. Gaiha</a>,<sup>3,</sup><sup>4</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Rossin%20EJ%5BAuthor%5D" class="affpopup" co-rid="_co_idm139911460951264" co-class="co-affbox">Elizabeth J. Rossin</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>2</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139911460809696"><h3 class="no_margin">Blake M. Hauser</h3><p><sup>1</sup>Harvard Medical School, Boston, MA USA </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hauser%20BM%5BAuthor%5D">Blake M. Hauser</a></div></div><div id="_co_idm139911464113024"><h3 class="no_margin">Yuyang Luo</h3><p><sup>2</sup>Harvard Medical School, Department of Ophthalmology, Massachusetts Eye and Ear, Boston, MA USA </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Luo%20Y%5BAuthor%5D">Yuyang Luo</a></div></div><div id="_co_idm139911458916480"><h3 class="no_margin">Anusha Nathan</h3><p><sup>3</sup>Ragon Institute of Mass General, MIT, and Harvard, Cambridge, MA USA </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Nathan%20A%5BAuthor%5D">Anusha Nathan</a></div></div><div id="_co_idm139911447494896"><h3 class="no_margin">Ahmad Al-Moujahed</h3><p><sup>2</sup>Harvard Medical School, Department of Ophthalmology, Massachusetts Eye and Ear, Boston, MA USA </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Al-Moujahed%20A%5BAuthor%5D">Ahmad Al-Moujahed</a></div></div><div id="_co_idm139911447493248"><h3 class="no_margin">Demetrios G. Vavvas</h3><p><sup>2</sup>Harvard Medical School, Department of Ophthalmology, Massachusetts Eye and Ear, Boston, MA USA </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Vavvas%20DG%5BAuthor%5D">Demetrios G. Vavvas</a></div></div><div id="_co_idm139911453366816"><h3 class="no_margin">Jason Comander</h3><p><sup>2</sup>Harvard Medical School, Department of Ophthalmology, Massachusetts Eye and Ear, Boston, MA USA </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Comander%20J%5BAuthor%5D">Jason Comander</a></div></div><div id="_co_idm139911464506272"><h3 class="no_margin">Eric A. Pierce</h3><p><sup>2</sup>Harvard Medical School, Department of Ophthalmology, Massachusetts Eye and Ear, Boston, MA USA </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Pierce%20EA%5BAuthor%5D">Eric A. Pierce</a></div></div><div id="_co_idm139911464504288"><h3 class="no_margin">Emily M. Place</h3><p><sup>2</sup>Harvard Medical School, Department of Ophthalmology, Massachusetts Eye and Ear, Boston, MA USA </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Place%20EM%5BAuthor%5D">Emily M. Place</a></div></div><div id="_co_idm139911460862816"><h3 class="no_margin">Kinga M. Bujakowska</h3><p><sup>2</sup>Harvard Medical School, Department of Ophthalmology, Massachusetts Eye and Ear, Boston, MA USA </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Bujakowska%20KM%5BAuthor%5D">Kinga M. Bujakowska</a></div></div><div id="_co_idm139911458634064"><h3 class="no_margin">Gaurav D. Gaiha</h3><p><sup>3</sup>Ragon Institute of Mass General, MIT, and Harvard, Cambridge, MA USA </p><p><sup>4</sup>Division of Gastroenterology, Massachusetts General Hospital, Boston, MA USA </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gaiha%20GD%5BAuthor%5D">Gaurav D. Gaiha</a></div></div><div id="_co_idm139911460951264"><h3 class="no_margin">Elizabeth J. Rossin</h3><p><sup>2</sup>Harvard Medical School, Department of Ophthalmology, Massachusetts Eye and Ear, Boston, MA USA </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Rossin%20EJ%5BAuthor%5D">Elizabeth J. Rossin</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm139911458925632_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139911458925632_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139911458925632_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm139911458925632_ai" style="display:none"><div class="fm-affl" id="Aff1"><sup>1</sup>Harvard Medical School, Boston, MA USA </div><div class="fm-affl" id="Aff2"><sup>2</sup>Harvard Medical School, Department of Ophthalmology, Massachusetts Eye and Ear, Boston, MA USA </div><div class="fm-affl" id="Aff3"><sup>3</sup>Ragon Institute of Mass General, MIT, and Harvard, Cambridge, MA USA </div><div class="fm-affl" id="Aff4"><sup>4</sup>Division of Gastroenterology, Massachusetts General Hospital, Boston, MA USA </div><div><span class="fm-affl">Elizabeth J. Rossin, </span><span class="fm-affl"><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ude.dravrah.ieem@nissor_htebazile" class="oemail">ude.dravrah.ieem@nissor_htebazile</a></span>.</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup>Corresponding author.</div></div><div class="fm-article-notes hide half_rhythm" id="idm139911458925632_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2023 Nov 15; Accepted 2024 May 2.</div></div><div class="permissions half_rhythm hide" id="idm139911458925632_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; The Author(s) 2024</div><div class="license half_rhythm"><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article&#x02019;s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article&#x02019;s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm" role="complementary" aria-label="Related or updated information about this article."><div><strong>Previous version available:</strong> This article is based on a previously available preprint posted on <em>medRxiv</em> on July 6, 2023: "<a href="/pmc/articles/PMC10350150/">Structure-based network analysis predicts mutations associated with inherited retinal disease</a>".</div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary Materials</div><div><a href="/pmc/articles/PMC11130145/bin/41525_2024_416_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl">41525_2024_416_MOESM1_ESM.pdf</a><span style="color:gray"> (3.0M)</span></div><div class="small guid">GUID: BC59D210-573B-43C8-AD13-607D7790FEB5</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Reporting summary</div><div><a href="/pmc/articles/PMC11130145/bin/41525_2024_416_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl">41525_2024_416_MOESM2_ESM.pdf</a><span style="color:gray"> (1.4M)</span></div><div class="small guid">GUID: 2BA59539-6BA2-4FD9-9CD2-BB09EF08FF36</div></div></dd></dl><dl data-length="621" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p>ClinVar, gnomAD, and Protein Data Bank data were publicly available and can be accessed using the gene names and unique identifiers in Supplementary Table <a href="#MOESM1" rid="MOESM1" class=" media">1</a>. Patient data from Massachusetts Eye and Ear cannot be shared publicly as specified by the Institutional Review Board due to concerns regarding possible identifiability of patients with relatively rare genetic conditions. Qualified researchers may be able to access the data via collaboration and data usage agreement by contacting the corresponding author.</p><p>Code to perform structure-based network analysis is publicly available via Zenodo (10.5281/zenodo.2597484).</p></dd></dl></div><div id="Abs1" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="Abs1title">Abstract</h2><!--article-meta--><div><p id="Par1" class="p p-first-last">Advances in gene sequencing technologies have accelerated the identification of genetic variants, but better tools are needed to understand which are causal of disease. This would be particularly useful in fields where gene therapy is a potential therapeutic modality for a disease-causing variant such as inherited retinal disease (IRD). Here, we apply structure-based network analysis (SBNA), which has been successfully utilized to identify variant-constrained amino acid residues in viral proteins, to identify residues that may cause IRD if subject to missense mutation. SBNA is based entirely on structural first principles and is not fit to specific outcome data, which makes it distinct from other contemporary missense prediction tools. In 4 well-studied human disease-associated proteins (BRCA1, HRAS, PTEN, and ERK2) with high-quality structural data, we find that SBNA scores correlate strongly with deep mutagenesis data. When applied to 47 IRD genes with available high-quality crystal structure data, SBNA scores reliably identified disease-causing variants according to phenotype definitions from the ClinVar database. Finally, we applied this approach to 63 patients at Massachusetts Eye and Ear (MEE) with IRD but for whom no genetic cause had been identified. Untrained models built using SBNA scores and BLOSUM62 scores for IRD-associated genes successfully predicted the pathogenicity of novel variants (AUC&#x02009;=&#x02009;0.851), allowing us to identify likely causative disease variants in 40 IRD patients. Model performance was further augmented by incorporating orthogonal data from EVE scores (AUC&#x02009;=&#x02009;0.927), which are based on evolutionary multiple sequence alignments. In conclusion, SBNA can used to successfully identify variants as causal of disease in human proteins and may help predict variants causative of IRD in an unbiased fashion.</p></div><div class="sec"><strong class="kwd-title">Subject terms: </strong><span class="kwd-text">Disease genetics, Molecular medicine, Genetics research</span></div></div><div id="Sec1" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec1title">Introduction</h2><p id="Par2" class="p p-first">As genetic sequencing has become increasingly available and less costly, a growing number of patients with clinical presentations of suspected genetic origin are undergoing targeted or whole-exome sequencing. Despite improved accessibility, the genetic basis of disease for a considerable proportion of these patients remains unclear following sequencing<sup><a href="#CR1" rid="CR1" class=" bibr popnode">1</a></sup>. Inherited retinal diseases (IRD), whereby rod and cone photoreceptors degenerate, are a group of Mendelian disorders that represent an important cause of vision loss<sup><a href="#CR2" rid="CR2" class=" bibr popnode">2</a></sup>. With the advancement in the availability of genetic testing<sup><a href="#CR3" rid="CR3" class=" bibr popnode">3</a>&#x02013;<a href="#CR6" rid="CR6" class=" bibr popnode">6</a></sup> and the lower cost of exome sequencing, IRD is a field with increasing promise and possibility for gene therapy interventions. However, among patients with an IRD, ~30% do not have a clear genetic basis despite classic retinal changes and a decrease in visual and retinal function<sup><a href="#CR7" rid="CR7" class=" bibr popnode">7</a></sup>, making them ineligible candidates for treatment. For these patients, additional tools are needed to better define genetic variants that are not among the group of known causal variants (i.e., variants of uncertain significance&#x02014;VUS).</p><p id="Par3">Numerous computational tools that aim to predict the phenotype of genetic variants have been described<sup><a href="#CR8" rid="CR8" class=" bibr popnode">8</a>&#x02013;<a href="#CR10" rid="CR10" class=" bibr popnode">10</a></sup>, many of which were trained on existing variant classifications or combine multiple metrics that use this type of training data<sup><a href="#CR11" rid="CR11" class=" bibr popnode">11</a>&#x02013;<a href="#CR19" rid="CR19" class=" bibr popnode">19</a></sup>. These data were limited by sparsely available annotations<sup><a href="#CR20" rid="CR20" class=" bibr popnode">20</a></sup>, and previous studies suggest that some of these algorithms may also have considerable false discovery rates<sup><a href="#CR21" rid="CR21" class=" bibr popnode">21</a>,<a href="#CR22" rid="CR22" class=" bibr popnode">22</a></sup>. Furthermore, many of these algorithms are limited by circularity, with duplication of variants in the training and test datasets as well as variants from the same protein within the training and test datasets<sup><a href="#CR23" rid="CR23" class=" bibr popnode">23</a></sup>. The clinical applicability of these approaches has been limited as a result<sup><a href="#CR23" rid="CR23" class=" bibr popnode">23</a>,<a href="#CR24" rid="CR24" class=" bibr popnode">24</a></sup>. Another class of methods uses sequence conservation to estimate the likelihood that a particular variant will have a deleterious phenotypic effect<sup><a href="#CR25" rid="CR25" class=" bibr popnode">25</a>,<a href="#CR26" rid="CR26" class=" bibr popnode">26</a></sup>. Sequence conservation is an important feature to consider (and provides independent information from protein structure) but can be an imperfect proxy for the functional importance of a particular amino acid position within a protein of interest; this has been previously demonstrated within the context of both model and human proteins<sup><a href="#CR27" rid="CR27" class=" bibr popnode">27</a>,<a href="#CR28" rid="CR28" class=" bibr popnode">28</a></sup> and also the human immunodeficiency virus (HIV), which has a per-base mutation rate ~10<sup>4</sup> times that of the human genome and thus serves as a model for an accelerated rate of genomic evolution<sup><a href="#CR29" rid="CR29" class=" bibr popnode">29</a>&#x02013;<a href="#CR31" rid="CR31" class=" bibr popnode">31</a></sup>. Some of these approaches also leverage recent advances in machine learning, which results in limited post hoc transparency regarding the basis for a particular estimated probability of pathogenicity and is also subject to circularity and over-fitting<sup><a href="#CR26" rid="CR26" class=" bibr popnode">26</a>,<a href="#CR32" rid="CR32" class=" bibr popnode">32</a></sup>. Functional in vitro assays, such as a high throughput assay for the <em>RHO</em> gene, can help to characterize individual genetic variants<sup><a href="#CR33" rid="CR33" class=" bibr popnode">33</a></sup>. However, these crucial experiments are not always feasible on a large scale for the entire set of patient-genotype combinations, given the need for appropriate experimental setup, tissue-type, and readout for the particular variant of interest.</p><p id="Par4">To add to these approaches, we developed structure-based network analysis (SBNA) which leverages the application of network theory to protein structure data with the goal of quantifying local residue connectivity, bridging interactions, and ligand proximity in order to identify amino acid residues that are topologically important<sup><a href="#CR29" rid="CR29" class=" bibr popnode">29</a>,<a href="#CR34" rid="CR34" class=" bibr popnode">34</a></sup>. Using x-ray crystallography and cryogenic electron microscopy (cryo-EM) data, it models proteins as networks of connected amino acids to quantitatively estimate the topological importance of each amino acid as it relates to others in the protein, protein complex, or protein&#x02013;ligand interaction. This approach is distinct from prior computational tools that use structural information because it is not reliant on pre-defined secondary structure elements; rather, it analyzes the crystallized tertiary structure of the folded protein as a network of weighted inter-residue interactions. Additionally, this approach does not require training on pre-labeled phenotypic data which means that it can provide a metric that is specifically focused on first-principle structural information. SBNA has been previously used to identify highly mutationally constrained amino acid residues and CD8<sup>+</sup> T cell epitopes in HIV and severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)<sup><a href="#CR29" rid="CR29" class=" bibr popnode">29</a>,<a href="#CR35" rid="CR35" class=" bibr popnode">35</a></sup>. In the case of HIV, these epitopes were found to be preferentially targeted by individuals able to suppress HIV replication in the absence of antiretroviral therapy<sup><a href="#CR29" rid="CR29" class=" bibr popnode">29</a></sup>. Within the context of SARS-CoV-2, highly networked regions have resisted ongoing sequence evolution during the pandemic and thereby may be capable of conferring broad T cell-mediated protection against sarbecovirus infection<sup><a href="#CR35" rid="CR35" class=" bibr popnode">35</a></sup>.</p><p id="Par5" class="p p-last">In light of these successes with highly mutable viruses, we chose to apply SBNA to human proteins which exist within a complex system of biological interactions. Such an application would potentially aid in better understanding which genetic variants discovered through sequencing are causal of disease. We first verified this approach in well-studied human proteins: breast cancer gene 1 (BRCA1), GTPase HRas (HRAS), phosphatase and tensin homolog (PTEN), and mitogen-activated protein kinase 1 (ERK2). We then sought to investigate whether this approach could generate meaningful results for IRD proteins of interest to estimate the phenotypic impact of missense variants, given that such a sizable number of patients with IRDs harbor VUSs. Therefore, we assessed the performance of SBNA in IRD protein-coding genes and their respective missense variants described in ClinVar, a large public database of reported pathogenic and benign genetic variation. Finally, we addressed a cohort of 63 affected subjects for whom a genetic cause of their condition has yet to be discovered to investigate the use of SBNA in real-world clinical scenarios.</p></div><div id="Sec2" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec2title">Results</h2><div id="Sec3" class="sec sec-first"><h3 id="Sec3title">Structure-based network analysis accurately identifies missense variants in human proteins associated with in vitro loss of function and pathogenic clinical phenotypes</h3><p id="Par6" class="p p-first">To evaluate whether SBNA could be meaningfully applied to human proteins, we first applied the approach to four well-studied human proteins&#x02014;BRCA1, HRAS, PTEN, and ERK2&#x02014;which were selected for analysis due to the availability of high-quality structural data. In addition, these proteins had published in vitro saturation mutagenesis experiments, which allowed us to extract the functional consequence of all missense variants and quantify mutational tolerance<sup><a href="#CR36" rid="CR36" class=" bibr popnode">36</a>&#x02013;<a href="#CR39" rid="CR39" class=" bibr popnode">39</a></sup>. We next generated network scores for all amino acid residues in the available protein databank (PDB) files (Fig. <a href="/pmc/articles/PMC11130145/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.1a)</span></span><span>1a</span></a>) and evaluated the correlation with mutational tolerance (Fig. <a href="/pmc/articles/PMC11130145/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.1b).</span></span><span>1b</span></a>). All four proteins showed a strong inverse correlation between mutational tolerance and network score, which was consistent with previous findings for viral proteins and other model proteins<sup><a href="#CR29" rid="CR29" class=" bibr popnode">29</a></sup>. Of note, only 17% of BRCA1 has been crystallized (the N- and C-terminal ends), but SBNA scores still performed reasonably well (Spearman correlation coefficient &#x02212;0.514, <em>p</em>&#x02009;=&#x02009;3.45e-22) despite limited structural data.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig1" co-legend-rid="lgnd_Fig1"><a href="/pmc/articles/PMC11130145/figure/Fig1/" target="figure" rid-figpopup="Fig1" rid-ob="ob-Fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139911501105088"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11130145_41525_2024_416_Fig1_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41525_2024_416_Fig1_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11130145/bin/41525_2024_416_Fig1_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139911501105088"><a target="object" rel="noopener" href="/pmc/articles/PMC11130145/figure/Fig1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig1"><div><a class="figpopup" href="/pmc/articles/PMC11130145/figure/Fig1/" target="figure" rid-figpopup="Fig1" rid-ob="ob-Fig1">Fig. 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Residue network score correlates with mutation intolerance and distinguishes pathogenic variants from benign variants in well-studied human proteins.</strong><p><strong>a</strong> Structural representations show network scores at each residue. Sphere radius and color corresponds to network score magnitude at a particular position. <strong>b</strong> Comparison between functional data from saturation mutagenesis experiments and network scores, with Spearman correlation coefficients and p-values displayed for each plot. Points are colored based on available clinical phenotype data. <strong>c</strong> Pooled comparison between network scores for variants with available clinical phenotype data for all four well-studied human proteins. <strong>d</strong> Individual comparisons between network scores for variants with available clinical phenotype data for all four well-studied human proteins.</p></div></div></div><p id="Par7" class="p p-last">We next compared network scores to pathogenicity categorizations derived from human data using the ClinVar and gnomAD genetic databases for these same four proteins (Fig. <a href="/pmc/articles/PMC11130145/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.1c).</span></span><span>1c</span></a>). Missense variants in all subsequent analyses were categorized with respect to human clinical data in line with the American College of Medical Genetics and Genomics/Association of Molecular Pathology (ACMG/AMP)<sup><a href="#CR24" rid="CR24" class=" bibr popnode">24</a></sup> as benign (encompassing &#x0201c;benign&#x0201d; or &#x0201c;likely benign&#x0201d; within ClinVar), VUS or pathogenic (encompassing &#x0201c;pathogenic&#x0201d; or &#x0201c;likely pathogenic&#x0201d; within ClinVar). We restricted our analysis to ClinVar missense variants with at least two-star level evidence, and gnomAD was used to identify additional relatively benign missense variants (variants with at least 250 alternative allele counts across 100,000 individuals). Across all four proteins, network scores assigned to pathogenic variants were significantly greater than those assigned to benign variants (median network scores for benign missense variants &#x02212;0.936 and pathogenic variants 0.866, <em>p</em>&#x02009;=&#x02009;1.836e-5, Fig. <a href="/pmc/articles/PMC11130145/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2em;">Fig.1c).</span></span><span>1c</span></a>). Scores assigned to VUSs fell in between those assigned to benign and pathogenic variants. These trends were observed consistently within individual proteins, though smaller sample sizes limit statistical power (Fig. <a href="/pmc/articles/PMC11130145/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.1d).</span></span><span>1d</span></a>). Overall, network scores correlate with available clinical phenotype data for the four well-studied human proteins (Spearman correlation coefficient 0.228, <em>p</em>&#x02009;=&#x02009;2.116e-23), suggesting that SBNA can be meaningfully applied to human proteins. Of note, we found that SBNA seems to capture structural properties beyond simple solvent accessibility (i.e., proximity to the core) because relative solvent accessibility (RSA) scores show less correlation with mutational tolerance scores than did SBNA (Supplementary Fig. <a href="#MOESM1" rid="MOESM1" class=" media">1</a>).</p></div><div id="Sec4" class="sec"><h3 id="Sec4title">SBNA predicts variants in IRD genes associated with pathogenic clinical phenotypes</h3><p id="Par8" class="p p-first-last">Having validated SBNA on four canonical, well-studied human proteins, we then applied SBNA to additional human proteins. We analyzed the relationship between network scores and pathogenicity designations from high-quality ClinVar variants and benign gnomAD variants for the 47 human genes associated with IRDs, for which high-quality structural data is available for the encoded protein. This dataset includes both membrane proteins (e.g., ABCA4 and RHO) as well as cytoplasmic proteins (e.g., RPE65 and RPGR) (Supplementary Table <a href="#MOESM1" rid="MOESM1" class=" media">1</a>). We found that pathogenic variants were assigned significantly greater network scores compared to both benign variants and VUSs (median benign &#x02212;0.841, median VUS &#x02212;0.188, median pathogenic 0.947 <em>p</em>&#x02009;=&#x02009;3.140e-29 for benign vs. pathogenic, <em>p</em>&#x02009;=&#x02009;1.753e-60 for VUS vs. pathogenic; Fig. <a href="/pmc/articles/PMC11130145/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2em;">Fig.2a</span></span><span>2a</span></a> and Supplementary Fig. <a href="#MOESM1" rid="MOESM1" class=" media">2</a>), which is similar to the pattern observed for the canonical human proteins. Because the number of high-quality ClinVar entries for missense variants in each of the 47 IRD-associated proteins varies considerably (Fig. <a href="/pmc/articles/PMC11130145/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.2b),</span></span><span>2b</span></a>), we wanted to evaluate whether the difference between median benign and pathogenic network scores remained statistically significant even for proteins with limited variant data available in ClinVar. We grouped proteins by the amount of available high-quality entries in ClinVar and calculated the difference between the median benign and pathogenic network scores across each group of proteins. The magnitude of these differences was robust in the setting of differing levels of available clinical data across genes and was detectable down to 40 high-quality entries per gene (Fig. <a href="/pmc/articles/PMC11130145/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.2c</span></span><span>2c</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig2" co-legend-rid="lgnd_Fig2"><a href="/pmc/articles/PMC11130145/figure/Fig2/" target="figure" rid-figpopup="Fig2" rid-ob="ob-Fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139911460956992"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41525_2024_416_Fig2_HTML.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is 41525_2024_416_Fig2_HTML.jpg" src="/pmc/articles/PMC11130145/bin/41525_2024_416_Fig2_HTML.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm139911460956992"><a target="object" rel="noopener" href="/pmc/articles/PMC11130145/figure/Fig2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig2"><div><a class="figpopup" href="/pmc/articles/PMC11130145/figure/Fig2/" target="figure" rid-figpopup="Fig2" rid-ob="ob-Fig2">Fig. 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Structure-based network analysis identifies pathogenic variants in inherited retinal disease proteins.</strong><p><strong>a</strong> Pooled comparison between network scores for variants with available clinical phenotype data for all 47 inherited retinal disease proteins. <strong>b</strong> Distribution of available high-quality evidence in ClinVar across all 47 inherited retinal disease proteins. The bins reflect increasing numbers of high-quality entries in ClinVar, and the height of each bar reflects the number of proteins in each category. <strong>c</strong> Comparison between median benign and pathogenic network scores assigned to variants with available high-quality evidence in ClinVar, grouped by level of available high-quality evidence for each inherited retinal disease protein. Stars above columns indicate statistical significance (*<em>p</em>&#x02009;&#x0003c;&#x02009;0.05, **<em>p</em>&#x02009;&#x0003c;&#x02009;0.01, ***<em>p</em>&#x02009;&#x0003c;&#x02009;0.001, ***<em>p</em>&#x02009;&#x0003c;&#x02009;0.0001). The statistical significance of the difference in network scores between benign and pathogenic variants is lost between 20 and 40 high-quality ClinVar evidence entries.</p></div></div></div></div><div id="Sec5" class="sec"><h3 id="Sec5title">Incorporating network scores and BLOSUM62 scores successfully predicts variant pathogenicity</h3><p id="Par9" class="p p-first-last">To move from aggregate statistics to prediction of pathogenicity using network scores, we constructed a modified score that incorporates not only the SBNA network score but also the degree of chemical and side chain dissimilarity between the reference and mutant amino acid at that position (since missense variants vary in this regard). To capture the latter effect, we subtracted the BLOSUM62 matrix score from the SBNA score (which we will now refer to as the modified SBNA score) to allow for a distinction between non-conservative substitutions (e.g., ILE &#x02192; TRP) and conservative ones (e.g., ILE &#x02192; LEU)<sup><a href="#CR40" rid="CR40" class=" bibr popnode">40</a></sup>. BLOSUM62 scores for non-synonymous substitutions range from &#x02212;4 to 3, while 95% of the raw SBNA scores range from &#x02212;2.88 to 4.97; thus, the two metrics are on similar scales and can be combined with simple subtraction. We then calculated receiver operating characteristic (ROC) curve statistics for high-quality ClinVar variants and benign gnomAD variants based on the modified SBNA score. Modified SBNA scores predicted variant pathogenicity (AUC 0.851) and outperformed network scores alone, BLOSUM62 scores alone, and RSA scores alone (Fig. <a href="/pmc/articles/PMC11130145/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span>3a, b</span></a> and Supplementary Fig. <a href="#MOESM1" rid="MOESM1" class=" media">3</a>). We tested multivariable logistic regression modeling with 500 iterations of a 70/30% train-test split as well as a leave-one-out approach using the labels derived from high-quality ClinVar and gnomAD variants and found similar performance to the raw scores (AUC 0.835, Supplementary Figs. <a href="#MOESM1" rid="MOESM1" class=" media">3</a>, <a href="#MOESM1" rid="MOESM1" class=" media">4</a>). Given the advantage that using the raw scores has over a trained approach (which can be subject to poor phenotypic labeling and data circularity<sup><a href="#CR20" rid="CR20" class=" bibr popnode">20</a>,<a href="#CR23" rid="CR23" class=" bibr popnode">23</a></sup>), all downstream clinical applications described here use the raw modified SBNA score.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig3" co-legend-rid="lgnd_Fig3"><a href="/pmc/articles/PMC11130145/figure/Fig3/" target="figure" rid-figpopup="Fig3" rid-ob="ob-Fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139911459823840"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11130145_41525_2024_416_Fig3_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41525_2024_416_Fig3_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11130145/bin/41525_2024_416_Fig3_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139911459823840"><a target="object" rel="noopener" href="/pmc/articles/PMC11130145/figure/Fig3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig3"><div><a class="figpopup" href="/pmc/articles/PMC11130145/figure/Fig3/" target="figure" rid-figpopup="Fig3" rid-ob="ob-Fig3">Fig. 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Modeling using SBNA and BLOSUM62 is superior to SBNA network scores and BLOSUM62 scores alone.</strong><p><strong>a</strong> ROC curves for network scores alone (red), BLOSUM62 scores alone (blue), and the difference between network scores and BLOSUM62 scores (purple). ROC curves were determined using all variants with available clinical phenotype data for all 47 inherited retinal disease proteins. AUC values are shown for each curve. <strong>b</strong> Pooled comparison between the difference between network scores and BLOSUM62 scores for variants with available clinical phenotype data for all 47 inherited retinal disease proteins. <strong>c</strong> ROC as shown in (<strong>a</strong>) with added comparison to EVE scores (orange) and the sum of EVE scores and the scaled difference between network scores and BLOSUM62 scores (light purple). ROC curves were determined using all variants with available clinical phenotype data for all 47 inherited retinal disease proteins. AUC values are shown for each curve. <strong>d</strong> Pooled comparison between the sum of EVE scores and the scaled difference between network scores and BLOSUM62 scores (&#x0201c;combined score&#x0201d;) for variants with available clinical phenotype data for all 47 inherited retinal disease proteins.</p></div></div></div></div><div id="Sec6" class="sec"><h3 id="Sec6title">Comparison of SBNA to existing methods reveals similar performance without dependence on phenotype labels or evolutionary sequence data</h3><p id="Par10" class="p p-first">We set out to compare SBNA to three tools that are built from different underlying data: Polyphen2, AlphaMissense, and EVE. Polyphen2 is a widely used computational prediction tool for variant pathogenicity<sup><a href="#CR41" rid="CR41" class=" bibr popnode">41</a></sup>. It is important to note however that PolyPhen2 is used by the ACMG/AMP to assess pathogenicity designations in databases such as ClinVar and HumDiv, so using ClinVar as ground truth might overestimate the performance of PolyPhen2. AlphaMissense<sup><a href="#CR32" rid="CR32" class=" bibr popnode">32</a></sup> incorporates structural data from AlphaFold2<sup><a href="#CR42" rid="CR42" class=" bibr popnode">42</a></sup>, protein language modeling, and evolutionary multiple sequence alignments into a machine-learning model using ClinVar labels to train. Similar to PolyPhen2, there is a risk of over-fitting and circularity with AlphaMissense. EVE is a variant pathogenicity approach which relies only on evolutionary sequence data rather than clinical labeling for model training and is thus not subject to circularity, similar to SBNA<sup><a href="#CR26" rid="CR26" class=" bibr popnode">26</a></sup>. EVE performs well on two of the well-studied human proteins (Spearman correlation ranging from &#x02212;0.478 to &#x02212;0.513, benign versus pathogenic <em>P</em>&#x02009;&#x0003c;&#x02009;0.05 for all; Supplementary Fig. <a href="#MOESM1" rid="MOESM1" class=" media">5</a>); EVE predictions are not available for ERK2. To minimize bias, all algorithms were tested on an independent dataset of 2800 rare variants derived from patients with an IRD who were seen at MEE, though of note, ground truth is still determined using the ClinVar database in accordance with the clinical standard in the field. Thus, methods that train on ClinVar must be interpreted with caution. The modified SBNA scores were compared to results generated using PolyPhen2 trained on two different datasets, HumDiv and HumVar<sup><a href="#CR12" rid="CR12" class=" bibr popnode">12</a>,<a href="#CR41" rid="CR41" class=" bibr popnode">41</a></sup>, as well as EVE scores<sup><a href="#CR26" rid="CR26" class=" bibr popnode">26</a></sup>. All methods showed a significant difference between benign and pathogenic variants, and the modified SBNA scores correlated with the scores from other methods (Supplementary Figs. <a href="#MOESM1" rid="MOESM1" class=" media">6</a>A, B, <a href="#MOESM1" rid="MOESM1" class=" media">7A, B</a>, Spearman correlation coefficient range 0.510&#x02013;0.571, <em>p</em>&#x02009;&#x0003c;&#x02009;5e-24 for all). With a threshold of 1.5 for modified SBNA scores, the sensitivity was 0.548, specificity 0.908, positive predictive value 0.963, and negative predictive value 0.312.</p><p id="Par11" class="p p-last">ROC curves were generated by testing each of the methods on the dataset of 2800 variants present in MEE patients (AUC range: 0.788 [modified SBNA], 0.829&#x02013;0.833 [PolyPhen2], 0.819 [AlphaMissense], and 0.809 [EVE]), Supplementary Figs. <a href="#MOESM1" rid="MOESM1" class=" media">6</a>C, <a href="#MOESM1" rid="MOESM1" class=" media">7C</a>). The modified SBNA, PolyPhen2, AlphaMissense, and EVE performed similarly, though PolyPhen2 and AlphaMissense scores may be inflated due to training on ClinVar pathogenicity labels.</p></div><div id="Sec7" class="sec"><h3 id="Sec7title">Combining SBNA and EVE outperforms all methods individually</h3><p id="Par12" class="p p-first-last">The correlation and prediction results suggest that structural information and sequence conservation provide distinctly important insight into pathogenicity, and thus, incorporating orthogonal metrics into a single score may help to improve model correlation with phenotypic benchmarks. We thus sought to use the two unbiased methods (modified SBNA and EVE). The modified network scores were scaled and added to EVE scores to create a combined score with a range of 0 to 2 (as EVE scores fall between 0 and 1, and SBNA scores were scaled based on the maximum and minimum values across all proteins such that they fell between 0 and 1 before adding to the EVE scores). When applied to the 2800 rare variants from MEE patients as well as the 47 IRD genes, this combined score distinguished pathogenicity (Supplementary Fig. <a href="#MOESM1" rid="MOESM1" class=" media">7D</a>, benign vs. pathogenic <em>p</em>&#x02009;=&#x02009;3.056e-17; Fig. <a href="/pmc/articles/PMC11130145/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2em;">Fig.3d,</span></span><span>3d</span></a>, benign vs. pathogenic <em>p</em>&#x02009;=&#x02009;8.839e-69) and outperformed all other models with an AUC of 0.859 (Supplementary Fig. <a href="#MOESM1" rid="MOESM1" class=" media">7E</a>) for the 2800 variants and 0.927 (Fig. <a href="/pmc/articles/PMC11130145/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.3c)</span></span><span>3c</span></a>) for the IRD genes. With a threshold of 1.0 for the combination score, the sensitivity was 0.765, specificity 0.899, positive predictive value of 0.976 and negative predictive value of 0.416. While we note that EVE alone performed quite well (AUC&#x02009;=&#x02009;0.915), adding the modified SBNA improves performance and, importantly, unlike EVE alone, offers a direct structural explanation for the mechanism through which variation may affect phenotype.</p></div><div id="Sec8" class="sec"><h3 id="Sec8title">Model incorporating the modified SBNA scores identifies putative disease-causing variants with an unclear genetic basis for clinical disease</h3><p id="Par13" class="p p-first">A significant percentage of patients with clinical presentations consistent with IRD lack an identified genetic basis for their phenotype, and this is also observed for patients who receive care from the Inherited Retinal Disease Service at MEE. We therefore evaluated genetic data from 3621 probands with a clinical diagnosis of an IRD based on visual acuity, visual field testing, clinical exam, fundus autofluorescence imaging, optical coherence tomography and electroretinogram in individuals who underwent targeted or whole-exome sequencing. Mitochondrial causes of IRD were excluded. Missense variants of interest were defined using variant ranking software<sup><a href="#CR43" rid="CR43" class=" bibr popnode">43</a></sup> and residence in one of the studied 47 IRD genes. There were 2948 unique variants identified and categorized as either &#x0201c;pathogenic&#x0201d;, &#x0201c;likely pathogenic&#x0201d;, &#x0201c;VUS&#x0201d;, or &#x0201c;benign&#x0201d; based on a known variant consequence in the literature using ACMG/AMP criteria<sup><a href="#CR44" rid="CR44" class=" bibr popnode">44</a></sup> and ClinVar designations<sup><a href="#CR20" rid="CR20" class=" bibr popnode">20</a></sup> (Supplementary Fig. <a href="#MOESM1" rid="MOESM1" class=" media">8</a>). Variants were further categorized in the context of individual patients as &#x0201c;likely causal&#x0201d; if they were pathogenic or likely pathogenic, the zygosity was consistent with known modes of inheritance, and the clinical presentation was consistent with the known consequence of the affected gene. Of all the reviewed patients, 455 patients carried variants of interest in the 47 IRD genes. Before applying SBNA, 357 were found to have variants that were &#x0201c;likely causal&#x0201d;, while 63 patients harbored one or more VUSs that prohibited a molecular diagnosis. The remaining 35 patients had non-missense variants or variants within a region that lacked available structural data and were therefore excluded.</p><p id="Par14">We generated pathogenicity predictions for the 2,948 IRD gene variants discovered in the probands (Fig. <a href="/pmc/articles/PMC11130145/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.4a).</span></span><span>4a</span></a>). Variants receiving a modified SBNA score of &#x0003e;1.5 (calibrated using probability estimates from the regression modeling) were deemed pathogenic. Scores were considered in the context of the identified genetic variants in known IRD genes for each patient with a clinical presentation consistent with IRD. Variants were matched with any phenotypic data available in ClinVar to roughly benchmark the pathogenicity scores. Similar to the ClinVar analysis of IRD genes, there were observable differences between the modified SBNA scores assigned to known pathogenic variants as compared to benign variants and VUS/variants without any available clinical data (benign median &#x02212;1.478, VUS median &#x02212;0.656, pathogenic median 2.148; benign vs. pathogenic <em>p</em>&#x02009;=&#x02009;1.713e-30).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig4" co-legend-rid="lgnd_Fig4"><a href="/pmc/articles/PMC11130145/figure/Fig4/" target="figure" rid-figpopup="Fig4" rid-ob="ob-Fig4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139911456229472"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11130145_41525_2024_416_Fig4_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41525_2024_416_Fig4_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11130145/bin/41525_2024_416_Fig4_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139911456229472"><a target="object" rel="noopener" href="/pmc/articles/PMC11130145/figure/Fig4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig4"><div><a class="figpopup" href="/pmc/articles/PMC11130145/figure/Fig4/" target="figure" rid-figpopup="Fig4" rid-ob="ob-Fig4">Fig. 4</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>SBNA helps identify pathogenic variants in patients with inherited retinal disease.</strong><p><strong>a</strong> Categorization of results from the application of modified structure-based network analysis (SBNA) to a dataset of possibly solving patient variants. Results were further subdivided into those from patients with known putative genetic causes of disease (<strong>b</strong>) and those from patients with only VUSs in known inherited retinal disease-associated genes (<strong>c</strong>). <strong>d</strong> Representation of network scores for a sample structure with putative solving genetic variants. Sphere radius corresponds to network score magnitude at a particular position. A patient with clinical evidence of ABCA4 disease (<strong>d</strong>) as evidenced by bilateral foveal pigmentary changes (arrowhead) on color fundus photo and bilateral RPE atrophy (star) and hyper autofluorescent flecks throughout the posterior pole on fundus autofluorescence imaging (arrows) but with no complete genetic explanation was fully solved using SBNA which highlighted two variants (Pro1380Leu and Arg1097Ser) that score highly in the ABCA4 protein structure (<strong>e</strong>). Arg1097Ser was a VUS and is indicated in red within the structure.</p></div></div></div><p id="Par15" class="p p-last">For the 357 patients who harbored known pathogenic variants sufficient to cause disease, the modified SBNA scores were concordant with these pathogenicity categorizations in 96.0% of cases (Fig. <a href="/pmc/articles/PMC11130145/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.4b).</span></span><span>4b</span></a>). For the 63 patients with VUSs as categorized by ACMG/AMP standards<sup><a href="#CR44" rid="CR44" class=" bibr popnode">44</a></sup> and/or ClinVar, the modified SBNA scores offered support for a genetic cause of disease for 40 patients (23 unique variants, Fig. <a href="/pmc/articles/PMC11130145/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2em;">Fig.4c).</span></span><span>4c</span></a>). By contrast, EVE scores offered support for a genetic cause of disease in 25 patients (20 in common with SBNA), PolyPhen2 scores trained on HumVar offered support in 33 patients (27 in common with SBNA), and AlphaMissense offered support in 40 patients (34 in common with SBNA). Combined EVE and modified SBNA scores offered support in 23 patients (20 in common with SBNA), but this analysis was limited because EVE does not provide scores for 15 patients. In the 15 patients where modified SBNA scores suggested a putative causative variant but EVE scores did not, one patient had a variant for which no EVE score was provided in the database, and the remainder had at least one candidate variant with an EVE score below the predicted pathogenicity threshold. Modes of inheritance included autosomal recessive (in combination with a known pathogenic variant or a second VUS with a high estimated probability of pathogenicity; <em>n</em>&#x02009;=&#x02009;15), autosomal dominant (<em>n</em>&#x02009;=&#x02009;2), and X-linked recessive (<em>n</em>&#x02009;=&#x02009;5). For example, a patient with autosomal recessive <em>ABCA4</em>-related disease was found to have variants p.(Pro1380Leu) (known pathogenic) and p.(Arg1097Ser) (VUS). The p.(Arg1097Ser) variant scored highly (SBNA score 3.672, BLOSUM62 score -1, score difference 4.672), suggesting it is likely pathogenic and thus completing the genetic solution for this patient. Similarly, the VUS p.(Cys302Tyr) in <em>RPGR</em> was found in a hemizygous patient with phenotypic findings consistent with X-linked IRD and also scored highly (SBNA score 2.154, BLOSUM62 score -2, score difference 4.154) (Supplementary Fig. <a href="#MOESM1" rid="MOESM1" class=" media">9</a>). For 19 patients, the modified SBNA scores contributed towards identifying a possible but not completely solved genetic cause, such as only one heterozygous VUS receiving a strong score. Finally, for the four remaining unsolved patients, SBNA was not possible due to lack of crystal structure data for those portions of the IRD-associated proteins or due to the presence of non-missense variants.</p></div><div id="Sec9" class="sec sec-last"><h3 id="Sec9title">AlphaFold2 can improve structural coverage for SBNA</h3><p id="Par16" class="p p-first">Despite evidence of strong performance when applied to IRD-associated proteins, SBNA remains broadly limited by the availability of high-quality structural data for proteins of interest. This structural coverage must also overlap with the availability of high-quality phenotypic data from ClinVar, limiting the scope of analysis (Fig. <a href="/pmc/articles/PMC11130145/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.5a).</span></span><span>5a</span></a>). Applying AlphaFold2 may provide a path toward overcoming this limitation. For example, NR2E3 was excluded from the set of 47 IRD genes because the only available structural data is from a chimera formed between one domain of NR2E3 and an unrelated bacterial protein (PDB: 4LOG). SBNA performs poorly on this non-physiologic structure with relatively poor coverage of NR2E3 (47%) (Fig. <a href="/pmc/articles/PMC11130145/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.5b).</span></span><span>5b</span></a>). However, when SBNA is applied to the full AlphaFold2-generated human NR2E3 structure, performance improves considerably (Fig. <a href="/pmc/articles/PMC11130145/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.5c</span></span><span>5c</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig5" co-legend-rid="lgnd_Fig5"><a href="/pmc/articles/PMC11130145/figure/Fig5/" target="figure" rid-figpopup="Fig5" rid-ob="ob-Fig5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139911452026656"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11130145_41525_2024_416_Fig5_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41525_2024_416_Fig5_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11130145/bin/41525_2024_416_Fig5_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139911452026656"><a target="object" rel="noopener" href="/pmc/articles/PMC11130145/figure/Fig5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig5"><div><a class="figpopup" href="/pmc/articles/PMC11130145/figure/Fig5/" target="figure" rid-figpopup="Fig5" rid-ob="ob-Fig5">Fig. 5</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>AlphaFold2 improves coverage for SBNA.</strong><p><strong>a</strong> The percentages of high confidence (&#x02009;&#x02265;&#x02009;2 stars) ClinVar variants for each of 47 IRD genes captured with SBNA are widely distributed (blue); the per-protein percentages of solved structure are widely distributed (orange). <strong>b</strong> NR2E3 only has structural data available for a single domain as part of a chimera. SBNA scores for NR2E3 correlate poorly with pathogenicity because the structure is partial and non-physiologic, but (<strong>c</strong>) the performance improves when using AlphaFold2 to model the full human protein. <strong>d</strong> Ten IRD-associated genes without available structural data were selected, and <strong>e</strong> SBNA scores were calculated for the full AlphaFold2 structures.</p></div></div></div><p id="Par17" class="p p-last">To further establish that AlphaFold2 can help to overcome the limited availability of high-quality structural data for SBNA, we selected ten IRD-associated genes without available structural data that had a considerable amount of data in ClinVar (Fig. <a href="/pmc/articles/PMC11130145/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.5d).</span></span><span>5d</span></a>). We performed SBNA on the AlphaFold2-generated structures for these genes and found a significant difference between the SBNA scores assigned to benign and pathogenic variants (<em>p</em>&#x02009;=&#x02009;1.201e-7; benign median &#x02212;0.539, VUS median &#x02212;0.491, pathogenic median 0.076) (Fig. <a href="/pmc/articles/PMC11130145/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.5e).</span></span><span>5e</span></a>). However, the magnitude of difference between the median benign and pathogenic network scores was decreased compared to SBNA performed on the structural data from the 47 IRD-associated genes. Still, these results suggest that AlphaFold2 could potentially be useful in expanding the applicability of SBNA, although it is not clear that the quality of this analysis would be superior to that performed on experimentally generated structural data.</p></div></div><div id="Sec10" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec10title">Discussion</h2><p id="Par18" class="p p-first">In this study, we applied SBNA to human proteins and demonstrated that the resulting residue network scores, especially when augmented with BLOSUM62 substitution scores, correlate strongly with missense variants that cause functional deficit or clinical disease. We leveraged those scores, in combination with BLOSUM62 substitution scores, to generate estimates of the likelihood that a particular missense variant was pathogenic. Using these estimates, we were able to nominate putative genetic solutions for 40 patients with clinical evidence of IRD. These results suggest that SBNA provides meaningful insights for patients with an unclear genetic basis for their clinical symptoms, particularly for patients with inherited retinal diseases. Importantly, we note that this method is based purely on structural principles rather than training on labeled outcome data, which means that it not only provides an unbiased prediction of pathogenicity but also the mechanism of pathogenicity (i.e., structural change) is proposed.</p><p id="Par19">Numerous computational tools for the prediction of variant pathogenicity have been developed<sup><a href="#CR15" rid="CR15" class=" bibr popnode">15</a>,<a href="#CR32" rid="CR32" class=" bibr popnode">32</a>,<a href="#CR45" rid="CR45" class=" bibr popnode">45</a>&#x02013;<a href="#CR56" rid="CR56" class=" bibr popnode">56</a></sup>. Virtually all of these tools use sequence conservation and protein secondary structure or domain information. These two categories of features can work well, especially when used in conjunction with one another. However, SBNA is distinct in that it captures the structural topology of individual amino acid residues in the context of the protein architecture and does not rely on pre-existing annotations. Importantly, unlike all existing tools, the performance of SBNA does not seem to rely on a training step involving thousands of phenotype-genotype combinations or multiple sequence alignment. As shown, the raw SBNA score combined with BLOSUM62 (without training) achieves an AUC of 0.851 on the dataset of 47 IRD proteins. Furthermore, the biophysical basis for SBNA scores can be analyzed on an atomic level. This transparency facilitates downstream applications, such as the consideration of possible gene therapy targets, and is not available within models that heavily leverage machine-learning approaches<sup><a href="#CR26" rid="CR26" class=" bibr popnode">26</a>,<a href="#CR32" rid="CR32" class=" bibr popnode">32</a></sup>. The predictive value of SBNA can be further augmented by incorporating EVE scores, which are ultimately based on evolutionary multiple sequence alignments<sup><a href="#CR26" rid="CR26" class=" bibr popnode">26</a></sup>, to achieve an AUC of 0.927. This suggests that incorporating multiple orthogonal metrics may strengthen predictive models.</p><p id="Par20">Applying SBNA may provide additional insight into patient candidacy for both approved and developmental gene therapy treatments. By identifying candidate pathogenic variants with high confidence, SBNA could be used to nominate a limited subset of variants for expedited in vivo validation to fast-track delivery of appropriate therapies to patients. Broadly, the patients that may benefit clinically from SBNA fall into three categories. The first are patients who may be a candidate for gene-specific, variant-agnostic therapies, such as the FDA-approved <em>RPE65</em>-targeted gene therapy voretigene neparvovec<sup><a href="#CR57" rid="CR57" class=" bibr popnode">57</a></sup>. This also applies to any gene with variant-agnostic therapies in ongoing clinical trials. A second category of patients could potentially benefit from variant-specific gene editing therapies, such as those facilitated by CRISPR-Cas9-mediated non-homologous end joining, which are currently under development for <em>CEP290</em><sup><a href="#CR58" rid="CR58" class=" bibr popnode">58</a></sup>. The third category of patients with IRD who may benefit from SBNA are not yet candidates for any existing clinical therapies. However, using SBNA on a large scale to identify candidate disease-causing variants may be informative for nominating new genes for variant-agnostic therapy development and new variants for gene editing therapy development in order to maximize potential patient benefit.</p><p id="Par21">This approach is limited by the availability of high-quality structural data, a requirement for SBNA<sup><a href="#CR29" rid="CR29" class=" bibr popnode">29</a></sup>, though we note that any tool that proposes to use structural data will be reliant on this. Numerous proteins, including the majority that correspond to known genetic variants associated with IRD, lack available x-ray crystallography or cryo-EM structures altogether. In some cases, structures are available but are not of sufficient resolution to facilitate downstream network analysis. Furthermore, this approach will only be applicable to variants that result in a negative structural change. Other types of variants &#x02013; such as splice site variants &#x02013; will not be captured with this approach. However, given that this tool only requires protein structure data, we expect that the performance of SBNA will continue to improve. Software that leverages artificial intelligence to predict protein structure, such as AlphaFold2 or RoseTTAFold, have already emerged to help compensate for the lack of available structural data<sup><a href="#CR42" rid="CR42" class=" bibr popnode">42</a>,<a href="#CR59" rid="CR59" class=" bibr popnode">59</a></sup>. We have demonstrated that SBNA can be meaningfully applied to some of these in silico-generated structures, which may provide a path towards overcoming the limitation of high-quality structural data availability going forward.</p><p id="Par22" class="p p-last">In conclusion, SBNA is a new tool to estimate the likely extent of the phenotypic impact of missense variants by assessing the topological important of affected residues to protein structure. We demonstrated that this technique could be meaningfully applied to human proteins and showcased the use of SBNA in IRD patients who lack a clear genetic diagnosis. These types of insights could contribute to the design of novel gene therapies targeted at implicated genetic variants<sup><a href="#CR57" rid="CR57" class=" bibr popnode">57</a>,<a href="#CR60" rid="CR60" class=" bibr popnode">60</a>,<a href="#CR61" rid="CR61" class=" bibr popnode">61</a></sup>.</p></div><div id="Sec11" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec11title">Methods</h2><div id="Sec12" class="sec sec-first"><h3 id="Sec12title">Structural data</h3><p id="Par23" class="p p-first-last">All structural data were downloaded from the Protein Data Bank<sup><a href="#CR34" rid="CR34" class=" bibr popnode">34</a></sup>. Individual accession numbers for the well-studied human proteins and inherited retinal disease proteins are listed in Supplementary Table <a href="#MOESM1" rid="MOESM1" class=" media">1</a>. For the non-human benchmark proteins, the same files were used as previously described<sup><a href="#CR29" rid="CR29" class=" bibr popnode">29</a></sup>. In cases where multiple human structures were available, the highest oligomeric state of the protein with a resolution of ~3 angstroms or better was used. In cases where no human structure was available, the structure of one or more homologs was used. If multiple chains were present within the PDB file due to crystal packing rather than true oligomerization, only one of these chains was used for SBNA. Solvent and water molecules were removed from all PDB files prior to SBNA, but ligands and protein binding partners were included in the analysis. Only the protein of interest was designated to have a network score calculated by SBNA.</p></div><div id="Sec13" class="sec"><h3 id="Sec13title">Structure-based Network Analysis</h3><p id="Par24" class="p p-first-last">Structure-based network analysis was used to calculate network scores as previously described<sup><a href="#CR29" rid="CR29" class=" bibr popnode">29</a>,<a href="#CR35" rid="CR35" class=" bibr popnode">35</a></sup>. The details of this method have been described previously. As before, in cases where multiple conformations of a structure were used, network scores were averaged for each amino acid position. In cases where no human structure was available, the structures of one or more non-human homologs as listed in Supplementary Table <a href="#MOESM1" rid="MOESM1" class=" media">1</a> were used as templates in Modeler to generate a predicated human structure. Relative solvent accessibility (RSA) was calculated using DSSP files<sup><a href="#CR62" rid="CR62" class=" bibr popnode">62</a></sup> and previously published maximum solvent accessible surface area values<sup><a href="#CR63" rid="CR63" class=" bibr popnode">63</a></sup>. Code to perform structure-based network analysis is publicly available via Zenodo (10.5281/zenodo.2597484).</p></div><div id="Sec14" class="sec"><h3 id="Sec14title">Data analysis and visualization</h3><p id="Par25" class="p p-first-last">Data analysis was performed using Python (version 3.8.2), with visualizations generated using the &#x0201c;matplotlib&#x0201d; package. Logistic regressions were performed using the &#x0201c;glm&#x0201d; package in R (version 4.0.4). Intercepts were set to zero for all logistic regression models. SBNA and logistic regression results for all tested variants are available via the Harvard Dataverse (10.7910/DVN/YEEPDY). Network score visualizations were generated in R (version 4.0.4) using the &#x0201c;rgl&#x0201d; package to implement OpenGL. The backbone centroid (centroid of nitrogen, alpha carbon, and carbon) positions were plotted along x, y, and z axes, and nodes were colored and given sphere radii corresponding to network scores. The protein structure backbone was then plotted, connecting the alpha carbons. Plotted structures were manually rotated, and two-dimensional views of interest were downloaded for inclusion.</p></div><div id="Sec15" class="sec"><h3 id="Sec15title">Phenotype data</h3><p id="Par26" class="p p-first">Clinical phenotype data was downloaded from ClinVar. Clinical evidence with a two-gold star level designation (meaning that &#x0201c;two or more submitters with assertion criteria and evidence [or a public contact] provided the same interpretation&#x0201d;) or better was included in the published analyses<sup><a href="#CR20" rid="CR20" class=" bibr popnode">20</a></sup>. Pathogenicity designations from ClinVar were binned into Benign (&#x0201c;Benign&#x0201d;, &#x0201c;Benign/Likely benign&#x0201d;, &#x0201c;Likely benign&#x0201d;), VUS (&#x0201c;Uncertain significance&#x0201d;, &#x0201c;not provided&#x0201d;, &#x0201c;Conflicting interpretations of pathogenicity&#x0201d;), and Pathogenic (&#x0201c;Pathogenic&#x0201d;, &#x0201c;Pathogenic/Likely pathogenic&#x0201d;, &#x0201c;Likely pathogenic&#x0201d;) categories for analysis.</p><p id="Par27" class="p p-last">Additionally, allelic variation data from gnomAD<sup><a href="#CR64" rid="CR64" class=" bibr popnode">64</a></sup> was considered for each gene. Loci with at least 250 available allelic variants in gnomAD were considered benign variants. Results from ClinVar and gnomAD for each gene were considered in combination for these analyses.</p></div><div id="Sec16" class="sec"><h3 id="Sec16title">Functional data</h3><p id="Par28" class="p p-first-last">Functional data for the four well-studied human proteins&#x02014;BRCA1<sup><a href="#CR36" rid="CR36" class=" bibr popnode">36</a></sup>, ERK2<sup><a href="#CR39" rid="CR39" class=" bibr popnode">39</a></sup>, PTEN<sup><a href="#CR37" rid="CR37" class=" bibr popnode">37</a></sup>, and HRAS<sup><a href="#CR38" rid="CR38" class=" bibr popnode">38</a></sup>&#x02014;that had been previously published was used for this analysis. In cases where functional scores were assigned to multiple amino acid variants at the same position (e.g., if different functional scores were calculated for Ala101Pro and Ala101Gln), the arithmetic mean of all functional scores at that position was used.</p></div><div id="Sec17" class="sec"><h3 id="Sec17title">Patient data</h3><p id="Par29" class="p p-first-last">Patient data was gathered from among those presenting to the Inherited Retinal Disorders Service at Massachusetts Eye and Ear between the 1980s and 2020s. Appropriate written informed consent was obtained from all included patients, and approval was granted by the Mass General Brigham/Massachusetts Eye and Ear Institutional Review Board Protocol Number 2019P001098. This study was approved by the local institutional review board and adhered to the Declaration of Helsinki. Informed consent was obtained from all individuals on whom clinical data and genetic testing were performed.</p></div><div id="Sec18" class="sec"><h3 id="Sec18title">Genetic analyses</h3><p id="Par30" class="p p-first-last">Pre-existing genetic solutions were available for all of the 3621 IRD cases, where genetic diagnoses for 3018 cases was obtained by targeted next-generation sequencing approaches<sup><a href="#CR43" rid="CR43" class=" bibr popnode">43</a></sup> or whole genome sequencing, and the remaining solutions were available from prior single-strand conformation polymorphism (SSCP) analysis and Sanger sequencing. Sequence data was aligned to the hg38 genome build, and the subsequent variant calling, annotation, and analyses were performed as described<sup><a href="#CR43" rid="CR43" class=" bibr popnode">43</a></sup>.</p></div><div id="Sec19" class="sec"><h3 id="Sec19title">Model Comparisons</h3><p id="Par31" class="p p-first-last">SBNA was compared to results from PolyPhen2<sup><a href="#CR41" rid="CR41" class=" bibr popnode">41</a></sup> (trained on both HumDiv and HumVar and EVE<sup><a href="#CR26" rid="CR26" class=" bibr popnode">26</a></sup>, with scores from both models generated as described in their respective publications. EVE scores were downloaded from <a href="https://evemodel.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://evemodel.org</a>; PolyPhen2 scores were generated using the &#x0201c;batch query&#x0201d; function at <a href="http://genetics.bwh.harvard.edu/pph2/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genetics.bwh.harvard.edu/pph2/</a>. To analyze the likelihood of pathogenicity, SBNA scores, EVE scores, and PolyPhen2 scores (trained on HumVar) were calculated for a set of 2800 variants. Cutoffs for each model were as follows: modified SBNA score &#x0003e;1.5 (corresponding to an approximate pathogenicity probability of 75% in the logistic regression model), EVE score &#x02265;0.65, and PolyPhen2 designation of &#x0201c;probably damaging&#x0201d;. The &#x0201c;combined score&#x0201d; was generated by scaling modified SBNA scores to fall between 0 and 1 based on the minimum and maximum values across all proteins and adding them to EVE scores. A threshold of 1.0 was selected as this corresponds to the sum of the minimum pathogenic EVE score (0.65) and the scaled version of the minimum pathogenic modified SBNA score (1.5, which scales to 0.37) with two significant figures.</p></div><div id="Sec20" class="sec"><h3 id="Sec20title">Statistical analysis</h3><p id="Par32" class="p p-first-last">Statistical analysis, including the generation of ROC curves, was performed using the &#x0201c;scipy.stats&#x0201d; package in Python as well as GraphPad Prism (version 9). Comparisons between three or more categories were made using the non-parametric Kruskal-Wallis test with Dunn&#x02019;s post hoc analysis corrected for multiple comparisons with a Bonferroni correction. Comparisons between the two categories were performed using the non-parametric Mann&#x02013;Whitney <em>U</em>-test. The correlation between the two datasets was calculated using non-parametric Spearman correlation coefficients. Spearman correlation coefficients between network scores and ClinVar pathogenicity designations was calculated by assigning 0 to benign variants, 1 to VUS, and 2 to pathogenic variants.</p></div><div id="Sec21" class="sec sec-last"><h3 id="Sec21title">Reporting summary</h3><p id="Par33" class="p p-first-last">Further information on research design is available in the <a href="#MOESM2" rid="MOESM2" class=" media">Nature Research Reporting Summary</a> linked to this article.</p></div></div><div id="sec-a.k.e" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="headless nomenu"></h2><!--/article/body/sec/--><div id="Sec22" class="sec sec-first-last"><h3 id="Sec22title">Supplementary information</h3><p class="p p-first-last">
</p><div class="sec suppmat" id="MOESM1"><div class="sup-box half_rhythm" id="media-a.k.e.a.b.a.a"><a href="/pmc/articles/PMC11130145/bin/41525_2024_416_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl">Supplementary Materials</a><sup>(3.0M, pdf)</sup></div></div><p>
</p><div class="sec suppmat" id="MOESM2"><div class="sup-box half_rhythm" id="media-a.k.e.a.b.b.a"><a href="/pmc/articles/PMC11130145/bin/41525_2024_416_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl">Reporting summary</a><sup>(1.4M, pdf)</sup></div></div><p>
</p></div></div><div id="ack-a.l.c" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.l.ctitle">Acknowledgements</h2><div class="sec"><p>B.M.H. is supported by award number T32GM144273 from the National Institute of General Medical Sciences and award number F30AI160908 from the National Institute of Allergy and Infectious Diseases. G.D.G. is supported by award numbers DP2AI154421, R01AI176533, and DP1DA058476, the Bill and Melinda Gates Foundation, the Burroughs Wellcome Career Award for Medical Scientists, and the Howard Goodman Fellowship. E.J.R. is supported by award number K12EY016335 from the National Eye Institute and the Massachusetts Lions Eye Research Fund. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></div></div><div id="notes-a.l.d" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.l.dtitle">Author contributions</h2><p>Conceptualization: B.M.H., G.D.G., and E.J.R.; Methodology: B.M.H., Y.L., A.N., G.D.G., and E.J.R.; Software: B.M.H., Y.L., and E.J.R.; Validation: B.M.H., Y.L., A.N., J.C., E.A.P., E.M.P., K.M.B., G.D.G., and E.J.R.; Formal analysis: B.M.H., Y.L., E.M.P., K.M.B., and E.J.R.; Resources: E.A.P., E.M.P., K.M.B., G.D.G., and E.J.R.; Data curation: B.M.H., J.C., E.A.P., E.M.P., K.M.B., and E.J.R.; Writing&#x02014;original draft: B.M.H. and E.J.R.; Writing&#x02014;review and editing: B.M.H., Y.L., A.N., D.G.V., J.C., E.A.P., E.M.P., K.M.B., G.D.G., and E.J.R.; Visualization: B.M.H. and E.J.R.; Supervision: D.G.V., J.C., E.A.P., K.M.B., G.D.G., and E.J.R.; Project administration and funding acquisition: E.J.R.</p></div><div id="notes-a.l.e" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.l.etitle">Data availability</h2><p>ClinVar, gnomAD, and Protein Data Bank data were publicly available and can be accessed using the gene names and unique identifiers in Supplementary Table <a href="#MOESM1" rid="MOESM1" class=" media">1</a>. Patient data from Massachusetts Eye and Ear cannot be shared publicly as specified by the Institutional Review Board due to concerns regarding possible identifiability of patients with relatively rare genetic conditions. Qualified researchers may be able to access the data via collaboration and data usage agreement by contacting the corresponding author.</p></div><div id="notes-a.l.f" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.l.ftitle">Code availability</h2><p>Code to perform structure-based network analysis is publicly available via Zenodo (10.5281/zenodo.2597484).</p></div><div id="FPar1" class="tsec sec"><h2 class="head no_bottom_margin" id="FPar1title">Competing interests</h2><p id="Par34">G.D.G. discloses research funding from Merck. E.J.R. and G.D.G. have filed two patent applications related to the use of SBNA in HIV and SARS-CoV-2.</p></div><div id="fn-group-a.l.a" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.l.atitle">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="fn-a.l.a.a"><p class="p p-first-last"><strong>Publisher&#x02019;s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></p></div></div><div id="sec-a.l.b" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="sec-a.l.btitle">Supplementary information</h2><!--/article/back/sec/--><p class="p p-first-last">The online version contains supplementary material available at 10.1038/s41525-024-00416-w.</p></div><div id="Bib1" class="tsec sec"><h2 class="head no_bottom_margin" id="Bib1title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="CR1">1. <span class="element-citation">Cooper GM, Shendure J. Needles in stacks of needles: finding disease-causal variants in a wealth of genomic data. <span><span class="ref-journal">Nat. Rev. Genet. </span>2011;<span class="ref-vol">12</span>:628–640. doi: 10.1038/nrg3046.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21850043" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnrg3046" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet.&amp;title=Needles+in+stacks+of+needles:+finding+disease-causal+variants+in+a+wealth+of+genomic+data&amp;author=GM+Cooper&amp;author=J+Shendure&amp;volume=12&amp;publication_year=2011&amp;pages=628-640&amp;pmid=21850043&amp;doi=10.1038/nrg3046&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR2">2. <span class="element-citation">Berger W, Kloeckener-Gruissem B, Neidhardt J. The molecular basis of human retinal and vitreoretinal diseases. <span><span class="ref-journal">Prog. Retin. Eye Res. </span>2010;<span class="ref-vol">29</span>:335–375. doi: 10.1016/j.preteyeres.2010.03.004.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20362068" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.preteyeres.2010.03.004" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Prog.+Retin.+Eye+Res.&amp;title=The+molecular+basis+of+human+retinal+and+vitreoretinal+diseases&amp;author=W+Berger&amp;author=B+Kloeckener-Gruissem&amp;author=J+Neidhardt&amp;volume=29&amp;publication_year=2010&amp;pages=335-375&amp;pmid=20362068&amp;doi=10.1016/j.preteyeres.2010.03.004&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR3">3. <span class="element-citation">Consugar MB, et al. Panel-based genetic diagnostic testing for inherited eye diseases is highly accurate and reproducible, and more sensitive for variant detection, than exome sequencing. <span><span class="ref-journal">Genet. Med. </span>2015;<span class="ref-vol">17</span>:253–261. doi: 10.1038/gim.2014.172.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4572572/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25412400" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fgim.2014.172" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Med.&amp;title=Panel-based+genetic+diagnostic+testing+for+inherited+eye+diseases+is+highly+accurate+and+reproducible,+and+more+sensitive+for+variant+detection,+than+exome+sequencing&amp;author=MB+Consugar&amp;volume=17&amp;publication_year=2015&amp;pages=253-261&amp;pmid=25412400&amp;doi=10.1038/gim.2014.172&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR4">4. <span class="element-citation">Wang F, et al. Next generation sequencing-based molecular diagnosis of retinitis pigmentosa: identification of a novel genotype-phenotype correlation and clinical refinements. <span><span class="ref-journal">Hum. Genet. </span>2014;<span class="ref-vol">133</span>:331–345. doi: 10.1007/s00439-013-1381-5.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3945441/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24154662" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs00439-013-1381-5" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet.&amp;title=Next+generation+sequencing-based+molecular+diagnosis+of+retinitis+pigmentosa:+identification+of+a+novel+genotype-phenotype+correlation+and+clinical+refinements&amp;author=F+Wang&amp;volume=133&amp;publication_year=2014&amp;pages=331-345&amp;pmid=24154662&amp;doi=10.1007/s00439-013-1381-5&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR5">5. <span class="element-citation">Ge Z, et al. NGS-based molecular diagnosis of 105 eyeGENE((R)) probands with retinitis pigmentosa. <span><span class="ref-journal">Sci. Rep. </span>2015;<span class="ref-vol">5</span>:18287. doi: 10.1038/srep18287.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4678898/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26667666" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fsrep18287" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Rep.&amp;title=NGS-based+molecular+diagnosis+of+105+eyeGENE((R))+probands+with+retinitis+pigmentosa&amp;author=Z+Ge&amp;volume=5&amp;publication_year=2015&amp;pages=18287&amp;pmid=26667666&amp;doi=10.1038/srep18287&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR6">6. <span class="element-citation">Hafler BP. Clinical progress in inherited retinal degenerations: gene therapy clinical trials and advances in genetic sequencing. <span><span class="ref-journal">Retina. </span>2017;<span class="ref-vol">37</span>:417–423. doi: 10.1097/IAE.0000000000001341.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5465814/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27753762" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1097%2FIAE.0000000000001341" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Retina&amp;title=Clinical+progress+in+inherited+retinal+degenerations:+gene+therapy+clinical+trials+and+advances+in+genetic+sequencing&amp;author=BP+Hafler&amp;volume=37&amp;publication_year=2017&amp;pages=417-423&amp;pmid=27753762&amp;doi=10.1097/IAE.0000000000001341&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR7">7. <span class="mixed-citation">Ben-Yosef, T. Inherited retinal diseases. <em>Int. J. Mol. Sci</em>. 10.3390/ijms232113467 (2022). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9654499/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36362249" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="CR8">8. <span class="element-citation">Peterson TA, Doughty E, Kann MG. Towards precision medicine: advances in computational approaches for the analysis of human variants. <span><span class="ref-journal">J. Mol. Biol. </span>2013;<span class="ref-vol">425</span>:4047–4063. doi: 10.1016/j.jmb.2013.08.008.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3807015/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23962656" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.jmb.2013.08.008" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Mol.+Biol.&amp;title=Towards+precision+medicine:+advances+in+computational+approaches+for+the+analysis+of+human+variants&amp;author=TA+Peterson&amp;author=E+Doughty&amp;author=MG+Kann&amp;volume=425&amp;publication_year=2013&amp;pages=4047-4063&amp;pmid=23962656&amp;doi=10.1016/j.jmb.2013.08.008&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR9">9. <span class="element-citation">Niroula A, Vihinen M. Variation interpretation predictors: principles, types, performance, and choice. <span><span class="ref-journal">Hum. Mutat. </span>2016;<span class="ref-vol">37</span>:579–597. doi: 10.1002/humu.22987.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26987456" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fhumu.22987" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mutat.&amp;title=Variation+interpretation+predictors:+principles,+types,+performance,+and+choice&amp;author=A+Niroula&amp;author=M+Vihinen&amp;volume=37&amp;publication_year=2016&amp;pages=579-597&amp;pmid=26987456&amp;doi=10.1002/humu.22987&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR10">10. <span class="element-citation">Rost B, Radivojac P, Bromberg Y. Protein function in precision medicine: deep understanding with machine learning. <span><span class="ref-journal">FEBS Lett. </span>2016;<span class="ref-vol">590</span>:2327–2341. doi: 10.1002/1873-3468.12307.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5937700/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27423136" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2F1873-3468.12307" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEBS+Lett.&amp;title=Protein+function+in+precision+medicine:+deep+understanding+with+machine+learning&amp;author=B+Rost&amp;author=P+Radivojac&amp;author=Y+Bromberg&amp;volume=590&amp;publication_year=2016&amp;pages=2327-2341&amp;pmid=27423136&amp;doi=10.1002/1873-3468.12307&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR11">11. <span class="element-citation">Pejaver V, et al. Inferring the molecular and phenotypic impact of amino acid variants with MutPred2. <span><span class="ref-journal">Nat. Commun. </span>2020;<span class="ref-vol">11</span>:5918. doi: 10.1038/s41467-020-19669-x.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7680112/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33219223" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41467-020-19669-x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Inferring+the+molecular+and+phenotypic+impact+of+amino+acid+variants+with+MutPred2&amp;author=V+Pejaver&amp;volume=11&amp;publication_year=2020&amp;pages=5918&amp;pmid=33219223&amp;doi=10.1038/s41467-020-19669-x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR12">12. <span class="element-citation">Adzhubei IA, et al. A method and server for predicting damaging missense mutations. <span><span class="ref-journal">Nat. Methods. </span>2010;<span class="ref-vol">7</span>:248–249. doi: 10.1038/nmeth0410-248.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2855889/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20354512" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnmeth0410-248" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=A+method+and+server+for+predicting+damaging+missense+mutations&amp;author=IA+Adzhubei&amp;volume=7&amp;publication_year=2010&amp;pages=248-249&amp;pmid=20354512&amp;doi=10.1038/nmeth0410-248&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR13">13. <span class="element-citation">Jagadeesh KA, et al. M-CAP eliminates a majority of variants of uncertain significance in clinical exomes at high sensitivity. <span><span class="ref-journal">Nat. Genet. </span>2016;<span class="ref-vol">48</span>:1581–1586. doi: 10.1038/ng.3703.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27776117" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fng.3703" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=M-CAP+eliminates+a+majority+of+variants+of+uncertain+significance+in+clinical+exomes+at+high+sensitivity&amp;author=KA+Jagadeesh&amp;volume=48&amp;publication_year=2016&amp;pages=1581-1586&amp;pmid=27776117&amp;doi=10.1038/ng.3703&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR14">14. <span class="element-citation">Rentzsch P, Witten D, Cooper GM, Shendure J, Kircher M. CADD: predicting the deleteriousness of variants throughout the human genome. <span><span class="ref-journal">Nucleic Acids Res. </span>2019;<span class="ref-vol">47</span>:D886–D894. doi: 10.1093/nar/gky1016.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6323892/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30371827" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgky1016" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=CADD:+predicting+the+deleteriousness+of+variants+throughout+the+human+genome&amp;author=P+Rentzsch&amp;author=D+Witten&amp;author=GM+Cooper&amp;author=J+Shendure&amp;author=M+Kircher&amp;volume=47&amp;publication_year=2019&amp;pages=D886-D894&amp;pmid=30371827&amp;doi=10.1093/nar/gky1016&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR15">15. <span class="element-citation">Ioannidis NM, et al. REVEL: an ensemble method for predicting the pathogenicity of rare missense variants. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2016;<span class="ref-vol">99</span>:877–885. doi: 10.1016/j.ajhg.2016.08.016.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5065685/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27666373" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.ajhg.2016.08.016" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=REVEL:+an+ensemble+method+for+predicting+the+pathogenicity+of+rare+missense+variants&amp;author=NM+Ioannidis&amp;volume=99&amp;publication_year=2016&amp;pages=877-885&amp;pmid=27666373&amp;doi=10.1016/j.ajhg.2016.08.016&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR16">16. <span class="element-citation">Feng BJ. PERCH: a unified framework for disease gene prioritization. <span><span class="ref-journal">Hum. Mutat. </span>2017;<span class="ref-vol">38</span>:243–251. doi: 10.1002/humu.23158.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5299048/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27995669" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fhumu.23158" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mutat.&amp;title=PERCH:+a+unified+framework+for+disease+gene+prioritization&amp;author=BJ+Feng&amp;volume=38&amp;publication_year=2017&amp;pages=243-251&amp;pmid=27995669&amp;doi=10.1002/humu.23158&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR17">17. <span class="element-citation">Raimondi D, et al. DEOGEN2: prediction and interactive visualization of single amino acid variant deleteriousness in human proteins. <span><span class="ref-journal">Nucleic Acids Res. </span>2017;<span class="ref-vol">45</span>:W201–W206. doi: 10.1093/nar/gkx390.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5570203/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28498993" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkx390" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=DEOGEN2:+prediction+and+interactive+visualization+of+single+amino+acid+variant+deleteriousness+in+human+proteins&amp;author=D+Raimondi&amp;volume=45&amp;publication_year=2017&amp;pages=W201-W206&amp;pmid=28498993&amp;doi=10.1093/nar/gkx390&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR18">18. <span class="element-citation">Ionita-Laza I, McCallum K, Xu B, Buxbaum JD. A spectral approach integrating functional genomic annotations for coding and noncoding variants. <span><span class="ref-journal">Nat. Genet. </span>2016;<span class="ref-vol">48</span>:214–220. doi: 10.1038/ng.3477.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4731313/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26727659" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fng.3477" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+spectral+approach+integrating+functional+genomic+annotations+for+coding+and+noncoding+variants&amp;author=I+Ionita-Laza&amp;author=K+McCallum&amp;author=B+Xu&amp;author=JD+Buxbaum&amp;volume=48&amp;publication_year=2016&amp;pages=214-220&amp;pmid=26727659&amp;doi=10.1038/ng.3477&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR19">19. <span class="element-citation">Dong C, et al. Comparison and integration of deleteriousness prediction methods for nonsynonymous SNVs in whole exome sequencing studies. <span><span class="ref-journal">Hum. Mol. Genet. </span>2015;<span class="ref-vol">24</span>:2125–2137. doi: 10.1093/hmg/ddu733.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4375422/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25552646" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fhmg%2Fddu733" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Comparison+and+integration+of+deleteriousness+prediction+methods+for+nonsynonymous+SNVs+in+whole+exome+sequencing+studies&amp;author=C+Dong&amp;volume=24&amp;publication_year=2015&amp;pages=2125-2137&amp;pmid=25552646&amp;doi=10.1093/hmg/ddu733&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR20">20. <span class="element-citation">Landrum MJ, et al. ClinVar: improvements to accessing data. <span><span class="ref-journal">Nucleic Acids Res. </span>2020;<span class="ref-vol">48</span>:D835–D844. doi: 10.1093/nar/gkz972.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6943040/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31777943" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkz972" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=ClinVar:+improvements+to+accessing+data&amp;author=MJ+Landrum&amp;volume=48&amp;publication_year=2020&amp;pages=D835-D844&amp;pmid=31777943&amp;doi=10.1093/nar/gkz972&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR21">21. <span class="element-citation">Wang T, et al. Probability of phenotypically detectable protein damage by ENU-induced mutations in the Mutagenetix database. <span><span class="ref-journal">Nat. Commun. </span>2018;<span class="ref-vol">9</span>:441. doi: 10.1038/s41467-017-02806-4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5789985/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29382827" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41467-017-02806-4" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Probability+of+phenotypically+detectable+protein+damage+by+ENU-induced+mutations+in+the+Mutagenetix+database&amp;author=T+Wang&amp;volume=9&amp;publication_year=2018&amp;pages=441&amp;pmid=29382827&amp;doi=10.1038/s41467-017-02806-4&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR22">22. <span class="element-citation">Miosge LA, et al. Comparison of predicted and actual consequences of missense mutations. <span><span class="ref-journal">Proc. Natl Acad. Sci. USA. </span>2015;<span class="ref-vol">112</span>:E5189–E5198. doi: 10.1073/pnas.1511585112.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4577149/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26269570" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.1511585112" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl+Acad.+Sci.+USA&amp;title=Comparison+of+predicted+and+actual+consequences+of+missense+mutations&amp;author=LA+Miosge&amp;volume=112&amp;publication_year=2015&amp;pages=E5189-E5198&amp;pmid=26269570&amp;doi=10.1073/pnas.1511585112&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR23">23. <span class="element-citation">Grimm DG, et al. The evaluation of tools used to predict the impact of missense variants is hindered by two types of circularity. <span><span class="ref-journal">Hum. Mutat. </span>2015;<span class="ref-vol">36</span>:513–523. doi: 10.1002/humu.22768.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4409520/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25684150" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fhumu.22768" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mutat.&amp;title=The+evaluation+of+tools+used+to+predict+the+impact+of+missense+variants+is+hindered+by+two+types+of+circularity&amp;author=DG+Grimm&amp;volume=36&amp;publication_year=2015&amp;pages=513-523&amp;pmid=25684150&amp;doi=10.1002/humu.22768&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR24">24. <span class="element-citation">Richards S, et al. Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology. <span><span class="ref-journal">Genet Med. </span>2015;<span class="ref-vol">17</span>:405–424. doi: 10.1038/gim.2015.30.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4544753/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25741868" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fgim.2015.30" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Med.&amp;title=Standards+and+guidelines+for+the+interpretation+of+sequence+variants:+a+joint+consensus+recommendation+of+the+American+College+of+Medical+Genetics+and+Genomics+and+the+Association+for+Molecular+Pathology&amp;author=S+Richards&amp;volume=17&amp;publication_year=2015&amp;pages=405-424&amp;pmid=25741868&amp;doi=10.1038/gim.2015.30&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR25">25. <span class="element-citation">Ng PC, Henikoff S. SIFT: predicting amino acid changes that affect protein function. <span><span class="ref-journal">Nucleic Acids Res. </span>2003;<span class="ref-vol">31</span>:3812–3814. doi: 10.1093/nar/gkg509.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC168916/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12824425" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkg509" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=SIFT:+predicting+amino+acid+changes+that+affect+protein+function&amp;author=PC+Ng&amp;author=S+Henikoff&amp;volume=31&amp;publication_year=2003&amp;pages=3812-3814&amp;pmid=12824425&amp;doi=10.1093/nar/gkg509&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR26">26. <span class="element-citation">Frazer J, et al. Disease variant prediction with deep generative models of evolutionary data. <span><span class="ref-journal">Nature. </span>2021;<span class="ref-vol">599</span>:91–95. doi: 10.1038/s41586-021-04043-8.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34707284" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41586-021-04043-8" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Disease+variant+prediction+with+deep+generative+models+of+evolutionary+data&amp;author=J+Frazer&amp;volume=599&amp;publication_year=2021&amp;pages=91-95&amp;pmid=34707284&amp;doi=10.1038/s41586-021-04043-8&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR27">27. <span class="element-citation">Mishra P, Flynn JM, Starr TN, Bolon DNA. Systematic mutant analyses elucidate general and client-specific aspects of Hsp90 function. <span><span class="ref-journal">Cell Rep. </span>2016;<span class="ref-vol">15</span>:588–598. doi: 10.1016/j.celrep.2016.03.046.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4838542/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27068472" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.celrep.2016.03.046" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Rep.&amp;title=Systematic+mutant+analyses+elucidate+general+and+client-specific+aspects+of+Hsp90+function&amp;author=P+Mishra&amp;author=JM+Flynn&amp;author=TN+Starr&amp;author=DNA+Bolon&amp;volume=15&amp;publication_year=2016&amp;pages=588-598&amp;pmid=27068472&amp;doi=10.1016/j.celrep.2016.03.046&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR28">28. <span class="element-citation">Hopf TA, et al. Mutation effects predicted from sequence co-variation. <span><span class="ref-journal">Nat. Biotechnol. </span>2017;<span class="ref-vol">35</span>:128–135. doi: 10.1038/nbt.3769.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5383098/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28092658" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnbt.3769" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=Mutation+effects+predicted+from+sequence+co-variation&amp;author=TA+Hopf&amp;volume=35&amp;publication_year=2017&amp;pages=128-135&amp;pmid=28092658&amp;doi=10.1038/nbt.3769&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR29">29. <span class="element-citation">Gaiha GD, et al. Structural topology defines protective CD8(+) T cell epitopes in the HIV proteome. <span><span class="ref-journal">Science. </span>2019;<span class="ref-vol">364</span>:480–484. doi: 10.1126/science.aav5095.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6855781/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31048489" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.aav5095" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Structural+topology+defines+protective+CD8(+)+T+cell+epitopes+in+the+HIV+proteome&amp;author=GD+Gaiha&amp;volume=364&amp;publication_year=2019&amp;pages=480-484&amp;pmid=31048489&amp;doi=10.1126/science.aav5095&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR30">30. <span class="element-citation">Nachman MW, Crowell SL. Estimate of the mutation rate per nucleotide in humans. <span><span class="ref-journal">Genetics. </span>2000;<span class="ref-vol">156</span>:297–304. doi: 10.1093/genetics/156.1.297.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1461236/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10978293" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fgenetics%2F156.1.297" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Estimate+of+the+mutation+rate+per+nucleotide+in+humans&amp;author=MW+Nachman&amp;author=SL+Crowell&amp;volume=156&amp;publication_year=2000&amp;pages=297-304&amp;pmid=10978293&amp;doi=10.1093/genetics/156.1.297&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR31">31. <span class="element-citation">Cuevas JM, Geller R, Garijo R, Lopez-Aldeguer J, Sanjuan R. Extremely high mutation rate of HIV-1 in vivo. <span><span class="ref-journal">PLoS Biol. </span>2015;<span class="ref-vol">13</span>:e1002251. doi: 10.1371/journal.pbio.1002251.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4574155/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26375597" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pbio.1002251" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol.&amp;title=Extremely+high+mutation+rate+of+HIV-1+in+vivo&amp;author=JM+Cuevas&amp;author=R+Geller&amp;author=R+Garijo&amp;author=J+Lopez-Aldeguer&amp;author=R+Sanjuan&amp;volume=13&amp;publication_year=2015&amp;pages=e1002251&amp;pmid=26375597&amp;doi=10.1371/journal.pbio.1002251&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR32">32. <span class="element-citation">Cheng J, et al. Accurate proteome-wide missense variant effect prediction with AlphaMissense. <span><span class="ref-journal">Science. </span>2023;<span class="ref-vol">381</span>:eadg7492. doi: 10.1126/science.adg7492.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37733863" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.adg7492" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Accurate+proteome-wide+missense+variant+effect+prediction+with+AlphaMissense&amp;author=J+Cheng&amp;volume=381&amp;publication_year=2023&amp;pages=eadg7492&amp;pmid=37733863&amp;doi=10.1126/science.adg7492&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR33">33. <span class="element-citation">Wan A, Place E, Pierce EA, Comander J. Characterizing variants of unknown significance in rhodopsin: a functional genomics approach. <span><span class="ref-journal">Hum. Mutat. </span>2019;<span class="ref-vol">40</span>:1127–1144. doi: 10.1002/humu.23762.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7027811/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30977563" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fhumu.23762" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mutat.&amp;title=Characterizing+variants+of+unknown+significance+in+rhodopsin:+a+functional+genomics+approach&amp;author=A+Wan&amp;author=E+Place&amp;author=EA+Pierce&amp;author=J+Comander&amp;volume=40&amp;publication_year=2019&amp;pages=1127-1144&amp;pmid=30977563&amp;doi=10.1002/humu.23762&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR34">34. <span class="element-citation">Berman HM, et al. The protein data bank. <span><span class="ref-journal">Nucleic Acids Res. </span>2000;<span class="ref-vol">28</span>:235–242. doi: 10.1093/nar/28.1.235.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC102472/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10592235" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2F28.1.235" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=The+protein+data+bank&amp;author=HM+Berman&amp;volume=28&amp;publication_year=2000&amp;pages=235-242&amp;pmid=10592235&amp;doi=10.1093/nar/28.1.235&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR35">35. <span class="element-citation">Nathan A, et al. Structure-guided T cell vaccine design for SARS-CoV-2 variants and sarbecoviruses. <span><span class="ref-journal">Cell. </span>2021;<span class="ref-vol">184</span>:4401–4413 e4410. doi: 10.1016/j.cell.2021.06.029.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8241654/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34265281" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cell.2021.06.029" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Structure-guided+T+cell+vaccine+design+for+SARS-CoV-2+variants+and+sarbecoviruses&amp;author=A+Nathan&amp;volume=184&amp;publication_year=2021&amp;pages=4401-4413+e4410&amp;pmid=34265281&amp;doi=10.1016/j.cell.2021.06.029&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR36">36. <span class="element-citation">Findlay GM, et al. Accurate classification of BRCA1 variants with saturation genome editing. <span><span class="ref-journal">Nature. </span>2018;<span class="ref-vol">562</span>:217–222. doi: 10.1038/s41586-018-0461-z.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6181777/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30209399" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41586-018-0461-z" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Accurate+classification+of+BRCA1+variants+with+saturation+genome+editing&amp;author=GM+Findlay&amp;volume=562&amp;publication_year=2018&amp;pages=217-222&amp;pmid=30209399&amp;doi=10.1038/s41586-018-0461-z&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR37">37. <span class="element-citation">Mighell TL, Evans-Dutson S, O&#x02019;Roak BJ. A saturation mutagenesis approach to understanding PTEN lipid phosphatase activity and genotype-phenotype relationships. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2018;<span class="ref-vol">102</span>:943–955. doi: 10.1016/j.ajhg.2018.03.018.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5986715/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29706350" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.ajhg.2018.03.018" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=A+saturation+mutagenesis+approach+to+understanding+PTEN+lipid+phosphatase+activity+and+genotype-phenotype+relationships&amp;author=TL+Mighell&amp;author=S+Evans-Dutson&amp;author=BJ+O&#x02019;Roak&amp;volume=102&amp;publication_year=2018&amp;pages=943-955&amp;pmid=29706350&amp;doi=10.1016/j.ajhg.2018.03.018&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR38">38. <span class="mixed-citation">Hidalgo, F. et al. A saturation-mutagenesis analysis of the interplay between stability and activation in Ras. <em>Elife</em>10.7554/eLife.76595 (2022). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8916776/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35272765" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="CR39">39. <span class="element-citation">Brenan L, et al. Phenotypic characterization of a comprehensive set of MAPK1/ERK2 missense mutants. <span><span class="ref-journal">Cell Rep. </span>2016;<span class="ref-vol">17</span>:1171–1183. doi: 10.1016/j.celrep.2016.09.061.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5120861/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27760319" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.celrep.2016.09.061" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Rep.&amp;title=Phenotypic+characterization+of+a+comprehensive+set+of+MAPK1/ERK2+missense+mutants&amp;author=L+Brenan&amp;volume=17&amp;publication_year=2016&amp;pages=1171-1183&amp;pmid=27760319&amp;doi=10.1016/j.celrep.2016.09.061&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR40">40. <span class="element-citation">Eddy SR. Where did the BLOSUM62 alignment score matrix come from? <span><span class="ref-journal">Nat. Biotechnol. </span>2004;<span class="ref-vol">22</span>:1035–1036. doi: 10.1038/nbt0804-1035.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15286655" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnbt0804-1035" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=Where+did+the+BLOSUM62+alignment+score+matrix+come+from?&amp;author=SR+Eddy&amp;volume=22&amp;publication_year=2004&amp;pages=1035-1036&amp;pmid=15286655&amp;doi=10.1038/nbt0804-1035&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR41">41. <span class="mixed-citation">Adzhubei, I., Jordan, D. M. &#x00026; Sunyaev, S. R. Predicting functional effect of human missense mutations using PolyPhen-2. <em>Curr. Protoc. Hum. Genet.</em>10.1002/0471142905.hg0720s76 (2013). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4480630/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23315928" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="CR42">42. <span class="element-citation">Jumper J, et al. Highly accurate protein structure prediction with AlphaFold. <span><span class="ref-journal">Nature. </span>2021;<span class="ref-vol">596</span>:583–589. doi: 10.1038/s41586-021-03819-2.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8371605/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34265844" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41586-021-03819-2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Highly+accurate+protein+structure+prediction+with+AlphaFold&amp;author=J+Jumper&amp;volume=596&amp;publication_year=2021&amp;pages=583-589&amp;pmid=34265844&amp;doi=10.1038/s41586-021-03819-2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR43">43. <span class="element-citation">Zampaglione E, et al. The importance of automation in genetic diagnosis: lessons from analyzing an inherited retinal degeneration cohort with the Mendelian Analysis Toolkit (MATK) <span><span class="ref-journal">Genet. Med. </span>2022;<span class="ref-vol">24</span>:332–343. doi: 10.1016/j.gim.2021.09.015.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9200473/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34906470" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.gim.2021.09.015" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Med.&amp;title=The+importance+of+automation+in+genetic+diagnosis:+lessons+from+analyzing+an+inherited+retinal+degeneration+cohort+with+the+Mendelian+Analysis+Toolkit+(MATK)&amp;author=E+Zampaglione&amp;volume=24&amp;publication_year=2022&amp;pages=332-343&amp;pmid=34906470&amp;doi=10.1016/j.gim.2021.09.015&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR44">44. <span class="element-citation">Kleinberger J, Maloney KA, Pollin TI, Jeng LJ. An openly available online tool for implementing the ACMG/AMP standards and guidelines for the interpretation of sequence variants. <span><span class="ref-journal">Genet. Med. </span>2016;<span class="ref-vol">18</span>:1165. doi: 10.1038/gim.2016.13.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5026899/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26986878" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fgim.2016.13" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Med.&amp;title=An+openly+available+online+tool+for+implementing+the+ACMG/AMP+standards+and+guidelines+for+the+interpretation+of+sequence+variants&amp;author=J+Kleinberger&amp;author=KA+Maloney&amp;author=TI+Pollin&amp;author=LJ+Jeng&amp;volume=18&amp;publication_year=2016&amp;pages=1165&amp;pmid=26986878&amp;doi=10.1038/gim.2016.13&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR45">45. <span class="element-citation">Bao L, Zhou M, Cui Y. nsSNPAnalyzer: identifying disease-associated nonsynonymous single nucleotide polymorphisms. <span><span class="ref-journal">Nucleic Acids Res. </span>2005;<span class="ref-vol">33</span>:W480–482,. doi: 10.1093/nar/gki372.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1160133/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15980516" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgki372" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=nsSNPAnalyzer:+identifying+disease-associated+nonsynonymous+single+nucleotide+polymorphisms&amp;author=L+Bao&amp;author=M+Zhou&amp;author=Y+Cui&amp;volume=33&amp;publication_year=2005&amp;pages=W480-482,&amp;pmid=15980516&amp;doi=10.1093/nar/gki372&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR46">46. <span class="element-citation">Choi Y, Chan AP. PROVEAN web server: a tool to predict the functional effect of amino acid substitutions and indels. <span><span class="ref-journal">Bioinformatics. </span>2015;<span class="ref-vol">31</span>:2745–2747. doi: 10.1093/bioinformatics/btv195.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4528627/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25851949" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtv195" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=PROVEAN+web+server:+a+tool+to+predict+the+functional+effect+of+amino+acid+substitutions+and+indels&amp;author=Y+Choi&amp;author=AP+Chan&amp;volume=31&amp;publication_year=2015&amp;pages=2745-2747&amp;pmid=25851949&amp;doi=10.1093/bioinformatics/btv195&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR47">47. <span class="element-citation">Ng PC, Henikoff S. Predicting deleterious amino acid substitutions. <span><span class="ref-journal">Genome Res. </span>2001;<span class="ref-vol">11</span>:863–874. doi: 10.1101/gr.176601.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC311071/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11337480" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1101%2Fgr.176601" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=Predicting+deleterious+amino+acid+substitutions&amp;author=PC+Ng&amp;author=S+Henikoff&amp;volume=11&amp;publication_year=2001&amp;pages=863-874&amp;pmid=11337480&amp;doi=10.1101/gr.176601&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR48">48. <span class="element-citation">Schwarz JM, Cooper DN, Schuelke M, Seelow D. MutationTaster2: mutation prediction for the deep-sequencing age. <span><span class="ref-journal">Nat. Methods. </span>2014;<span class="ref-vol">11</span>:361–362. doi: 10.1038/nmeth.2890.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24681721" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnmeth.2890" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=MutationTaster2:+mutation+prediction+for+the+deep-sequencing+age&amp;author=JM+Schwarz&amp;author=DN+Cooper&amp;author=M+Schuelke&amp;author=D+Seelow&amp;volume=11&amp;publication_year=2014&amp;pages=361-362&amp;pmid=24681721&amp;doi=10.1038/nmeth.2890&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR49">49. <span class="element-citation">Tang H, Thomas PD. PANTHER-PSEP: predicting disease-causing genetic variants using position-specific evolutionary preservation. <span><span class="ref-journal">Bioinformatics. </span>2016;<span class="ref-vol">32</span>:2230–2232. doi: 10.1093/bioinformatics/btw222.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27193693" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtw222" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=PANTHER-PSEP:+predicting+disease-causing+genetic+variants+using+position-specific+evolutionary+preservation&amp;author=H+Tang&amp;author=PD+Thomas&amp;volume=32&amp;publication_year=2016&amp;pages=2230-2232&amp;pmid=27193693&amp;doi=10.1093/bioinformatics/btw222&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR50">50. <span class="element-citation">Ramensky V, Bork P, Sunyaev S. Human non-synonymous SNPs: server and survey. <span><span class="ref-journal">Nucleic Acids Res. </span>2002;<span class="ref-vol">30</span>:3894–3900. doi: 10.1093/nar/gkf493.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC137415/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12202775" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkf493" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Human+non-synonymous+SNPs:+server+and+survey&amp;author=V+Ramensky&amp;author=P+Bork&amp;author=S+Sunyaev&amp;volume=30&amp;publication_year=2002&amp;pages=3894-3900&amp;pmid=12202775&amp;doi=10.1093/nar/gkf493&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR51">51. <span class="element-citation">Ferrer-Costa C, Orozco M, de la Cruz X. Characterization of disease-associated single amino acid polymorphisms in terms of sequence and structure properties. <span><span class="ref-journal">J. Mol. Biol. </span>2002;<span class="ref-vol">315</span>:771–786. doi: 10.1006/jmbi.2001.5255.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11812146" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1006%2Fjmbi.2001.5255" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Mol.+Biol.&amp;title=Characterization+of+disease-associated+single+amino+acid+polymorphisms+in+terms+of+sequence+and+structure+properties&amp;author=C+Ferrer-Costa&amp;author=M+Orozco&amp;author=X+de+la+Cruz&amp;volume=315&amp;publication_year=2002&amp;pages=771-786&amp;pmid=11812146&amp;doi=10.1006/jmbi.2001.5255&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR52">52. <span class="element-citation">Li B, et al. Automated inference of molecular mechanisms of disease from amino acid substitutions. <span><span class="ref-journal">Bioinformatics. </span>2009;<span class="ref-vol">25</span>:2744–2750. doi: 10.1093/bioinformatics/btp528.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3140805/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19734154" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtp528" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Automated+inference+of+molecular+mechanisms+of+disease+from+amino+acid+substitutions&amp;author=B+Li&amp;volume=25&amp;publication_year=2009&amp;pages=2744-2750&amp;pmid=19734154&amp;doi=10.1093/bioinformatics/btp528&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR53">53. <span class="element-citation">Saunders CT, Baker D. Evaluation of structural and evolutionary contributions to deleterious mutation prediction. <span><span class="ref-journal">J. Mol. Biol. </span>2002;<span class="ref-vol">322</span>:891–901. doi: 10.1016/S0022-2836(02)00813-6.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12270722" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FS0022-2836(02)00813-6" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Mol.+Biol.&amp;title=Evaluation+of+structural+and+evolutionary+contributions+to+deleterious+mutation+prediction&amp;author=CT+Saunders&amp;author=D+Baker&amp;volume=322&amp;publication_year=2002&amp;pages=891-901&amp;pmid=12270722&amp;doi=10.1016/S0022-2836(02)00813-6&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR54">54. <span class="element-citation">Seifi M, Walter MA. Accurate prediction of functional, structural, and stability changes in PITX2 mutations using in silico bioinformatics algorithms. <span><span class="ref-journal">PLoS ONE. </span>2018;<span class="ref-vol">13</span>:e0195971. doi: 10.1371/journal.pone.0195971.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5903617/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29664915" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pone.0195971" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Accurate+prediction+of+functional,+structural,+and+stability+changes+in+PITX2+mutations+using+in+silico+bioinformatics+algorithms&amp;author=M+Seifi&amp;author=MA+Walter&amp;volume=13&amp;publication_year=2018&amp;pages=e0195971&amp;pmid=29664915&amp;doi=10.1371/journal.pone.0195971&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR55">55. <span class="element-citation">Shihab HA, et al. Predicting the functional, molecular, and phenotypic consequences of amino acid substitutions using hidden Markov models. <span><span class="ref-journal">Hum. Mutat. </span>2013;<span class="ref-vol">34</span>:57–65. doi: 10.1002/humu.22225.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3558800/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23033316" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fhumu.22225" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mutat.&amp;title=Predicting+the+functional,+molecular,+and+phenotypic+consequences+of+amino+acid+substitutions+using+hidden+Markov+models&amp;author=HA+Shihab&amp;volume=34&amp;publication_year=2013&amp;pages=57-65&amp;pmid=23033316&amp;doi=10.1002/humu.22225&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR56">56. <span class="element-citation">Tavtigian SV, et al. Comprehensive statistical study of 452 BRCA1 missense substitutions with classification of eight recurrent substitutions as neutral. <span><span class="ref-journal">J. Med. Genet. </span>2006;<span class="ref-vol">43</span>:295–305. doi: 10.1136/jmg.2005.033878.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2563222/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16014699" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1136%2Fjmg.2005.033878" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Genet.&amp;title=Comprehensive+statistical+study+of+452+BRCA1+missense+substitutions+with+classification+of+eight+recurrent+substitutions+as+neutral&amp;author=SV+Tavtigian&amp;volume=43&amp;publication_year=2006&amp;pages=295-305&amp;pmid=16014699&amp;doi=10.1136/jmg.2005.033878&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR57">57. <span class="element-citation">Russell S, et al. Efficacy and safety of voretigene neparvovec (AAV2-hRPE65v2) in patients with RPE65-mediated inherited retinal dystrophy: a randomised, controlled, open-label, phase 3 trial. <span><span class="ref-journal">Lancet. </span>2017;<span class="ref-vol">390</span>:849–860. doi: 10.1016/S0140-6736(17)31868-8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5726391/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28712537" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FS0140-6736(17)31868-8" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Efficacy+and+safety+of+voretigene+neparvovec+(AAV2-hRPE65v2)+in+patients+with+RPE65-mediated+inherited+retinal+dystrophy:+a+randomised,+controlled,+open-label,+phase+3+trial&amp;author=S+Russell&amp;volume=390&amp;publication_year=2017&amp;pages=849-860&amp;pmid=28712537&amp;doi=10.1016/S0140-6736(17)31868-8&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR58">58. <span class="element-citation">Burnight ER, et al. Using CRISPR-Cas9 to generate gene-corrected autologous iPSCs for the treatment of inherited retinal degeneration. <span><span class="ref-journal">Mol. Ther. </span>2017;<span class="ref-vol">25</span>:1999–2013. doi: 10.1016/j.ymthe.2017.05.015.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5589061/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28619647" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.ymthe.2017.05.015" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Ther.&amp;title=Using+CRISPR-Cas9+to+generate+gene-corrected+autologous+iPSCs+for+the+treatment+of+inherited+retinal+degeneration&amp;author=ER+Burnight&amp;volume=25&amp;publication_year=2017&amp;pages=1999-2013&amp;pmid=28619647&amp;doi=10.1016/j.ymthe.2017.05.015&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR59">59. <span class="element-citation">Baek M, et al. Accurate prediction of protein structures and interactions using a three-track neural network. <span><span class="ref-journal">Science. </span>2021;<span class="ref-vol">373</span>:871–876. doi: 10.1126/science.abj8754.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7612213/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34282049" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.abj8754" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Accurate+prediction+of+protein+structures+and+interactions+using+a+three-track+neural+network&amp;author=M+Baek&amp;volume=373&amp;publication_year=2021&amp;pages=871-876&amp;pmid=34282049&amp;doi=10.1126/science.abj8754&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR60">60. <span class="element-citation">Nuzbrokh Y, Ragi SD, Tsang SH. Gene therapy for inherited retinal diseases. <span><span class="ref-journal">Ann. Transl. Med. </span>2021;<span class="ref-vol">9</span>:1278. doi: 10.21037/atm-20-4726.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8421966/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34532415" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.21037%2Fatm-20-4726" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Transl.+Med.&amp;title=Gene+therapy+for+inherited+retinal+diseases&amp;author=Y+Nuzbrokh&amp;author=SD+Ragi&amp;author=SH+Tsang&amp;volume=9&amp;publication_year=2021&amp;pages=1278&amp;pmid=34532415&amp;doi=10.21037/atm-20-4726&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR61">61. <span class="element-citation">Fenner BJ, et al. Gene-based therapeutics for inherited retinal diseases. <span><span class="ref-journal">Front. Genet. </span>2021;<span class="ref-vol">12</span>:794805. doi: 10.3389/fgene.2021.794805.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8782148/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35069693" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3389%2Ffgene.2021.794805" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Genet.&amp;title=Gene-based+therapeutics+for+inherited+retinal+diseases&amp;author=BJ+Fenner&amp;volume=12&amp;publication_year=2021&amp;pages=794805&amp;pmid=35069693&amp;doi=10.3389/fgene.2021.794805&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR62">62. <span class="element-citation">Joosten RP, et al. A series of PDB related databases for everyday needs. <span><span class="ref-journal">Nucleic Acids Res. </span>2011;<span class="ref-vol">39</span>:D411–419,. doi: 10.1093/nar/gkq1105.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3013697/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21071423" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkq1105" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=A+series+of+PDB+related+databases+for+everyday+needs&amp;author=RP+Joosten&amp;volume=39&amp;publication_year=2011&amp;pages=D411-419,&amp;pmid=21071423&amp;doi=10.1093/nar/gkq1105&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR63">63. <span class="element-citation">Tien MZ, Meyer AG, Sydykova DK, Spielman SJ, Wilke CO. Maximum allowed solvent accessibilites of residues in proteins. <span><span class="ref-journal">PLoS ONE. </span>2013;<span class="ref-vol">8</span>:e80635. doi: 10.1371/journal.pone.0080635.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3836772/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24278298" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pone.0080635" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Maximum+allowed+solvent+accessibilites+of+residues+in+proteins&amp;author=MZ+Tien&amp;author=AG+Meyer&amp;author=DK+Sydykova&amp;author=SJ+Spielman&amp;author=CO+Wilke&amp;volume=8&amp;publication_year=2013&amp;pages=e80635&amp;pmid=24278298&amp;doi=10.1371/journal.pone.0080635&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR64">64. <span class="element-citation">Karczewski KJ, et al. The mutational constraint spectrum quantified from variation in 141,456 humans. <span><span class="ref-journal">Nature. </span>2020;<span class="ref-vol">581</span>:434–443. doi: 10.1038/s41586-020-2308-7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7334197/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32461654" ref="reftype=pubmed&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41586-020-2308-7" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+mutational+constraint+spectrum+quantified+from+variation+in+141,456+humans&amp;author=KJ+Karczewski&amp;volume=581&amp;publication_year=2020&amp;pages=434-443&amp;pmid=32461654&amp;doi=10.1038/s41586-020-2308-7&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11130145&amp;issue-id=452431&amp;journal-id=3307&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">NPJ Genomic Medicine</span> are provided here courtesy of <strong>Nature Publishing Group</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11130145/pdf/41525_2024_Article_416.pdf" class="int-view">PDF (2.2M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11130145/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11130145/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11130145" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11130145%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="f2TCCiLN3mWZjSRY6vXq6wHQxePJzIufxC8Pukuyg5PhNdgPpa2RESOwbkjqV8EQ">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11130145%2F&amp;text=Structure-based%20network%20analysis%20predicts%20pathogenic%20variants%20in%20human%20proteins%20associated%20with%20inherited%20retinal%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11130145%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11130145/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11130145/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38802398/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11130145/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38802398/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11130145/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11130145/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC11130145/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
