
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs4.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Characterization of Mutations That Allow p-Aminobenzoyl-Glutamate Utilization by Escherichia coli - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="939BE807DB0F4BC50000517667493B66.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="jbacter" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC107711/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC107711/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Journal of Bacteriology" /><meta name="citation_title" content="Characterization of Mutations That Allow p-Aminobenzoyl-Glutamate Utilization by Escherichia coli" /><meta name="citation_author" content="Mouyassar J. Hussein" /><meta name="citation_author_institution" content="Laboratory for Molecular Biology, Department of Biological Sciences, University of Illinois at Chicago, Chicago, Illinois 60607,1 and Department of Basic Biomedical Sciences, Dr. William M. Scholl College of Podiatric Medicine, Chicago, Illinois 606122" /><meta name="citation_author" content="Jacalyn M. Green" /><meta name="citation_author_institution" content="Laboratory for Molecular Biology, Department of Biological Sciences, University of Illinois at Chicago, Chicago, Illinois 60607,1 and Department of Basic Biomedical Sciences, Dr. William M. Scholl College of Podiatric Medicine, Chicago, Illinois 606122" /><meta name="citation_author" content="Brian P. Nichols" /><meta name="citation_author_institution" content="Laboratory for Molecular Biology, Department of Biological Sciences, University of Illinois at Chicago, Chicago, Illinois 60607,1 and Department of Basic Biomedical Sciences, Dr. William M. Scholl College of Podiatric Medicine, Chicago, Illinois 606122" /><meta name="citation_publication_date" content="1998/12" /><meta name="citation_issue" content="23" /><meta name="citation_volume" content="180" /><meta name="citation_firstpage" content="6260" /><meta name="citation_doi" content="10.1128/jb.180.23.6260-6268.1998" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC107711/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC107711/" /><meta name="citation_pmid" content="9829935" /><meta name="DC.Title" content="Characterization of Mutations That Allow p-Aminobenzoyl-Glutamate Utilization by Escherichia coli" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="American Society for Microbiology (ASM)" /><meta name="DC.Contributor" content="Mouyassar J. Hussein" /><meta name="DC.Contributor" content="Jacalyn M. Green" /><meta name="DC.Contributor" content="Brian P. Nichols" /><meta name="DC.Date" content="1998 Dec" /><meta name="DC.Identifier" content="10.1128/jb.180.23.6260-6268.1998" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Characterization of Mutations That Allow p-Aminobenzoyl-Glutamate Utilization by Escherichia coli" /><meta property="og:type" content="article" /><meta property="og:description" content="An Escherichia coli strain deficient in p-aminobenzoate synthesis was mutagenized, and derivatives were selected for growth on folic acid. Supplementation was shown to be due to p-aminobenzoyl-glutamate present as a breakdown product in commercial folic ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC107711/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC107711/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/88/">J Bacteriol</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/2962/">v.180(23); 1998 Dec</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC107711
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC107711/pdf/jb006260.pdf" class="int-view">PDF (186K)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/107711/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/107711/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="107711" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC107711%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="BDDQv51HIZ3WlprHQzCpZwfY0bvVfMMoEn5JbT0UtsFeptonreld7SBLXjq512s0">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC107711%2F&amp;text=Characterization%20of%20Mutations%20That%20Allow%20p-Aminobenzoyl-Glutamate%20Utilization%20by%20Escherichia%20coli" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC107711%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC107711/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC107711/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/9829935/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC107711/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/9829935/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC107711/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/107711/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/88/">J Bacteriol</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/2962/">v.180(23); 1998 Dec</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC107711
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-jbacter.png" alt="Logo of jbacter" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="https://journals.asm.org/journal/iai" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">J Bacteriol.</span> 1998 Dec; 180(23): 6260–6268. </div><div class="part2">  <span class="doi"><span>doi: </span><a href="//doi.org/10.1128%2Fjb.180.23.6260-6268.1998" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1128/jb.180.23.6260-6268.1998</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC107711</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/9829935">9829935</a></div></div></div><h1 class="content-title">Characterization of Mutations That Allow <em>p</em>-Aminobenzoyl-Glutamate Utilization by <em>Escherichia coli</em></h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hussein%20MJ%5BAuthor%5D" class="affpopup" co-rid="_co_idm140559512463280" co-class="co-affbox">Mouyassar J. Hussein</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Green%20JM%5BAuthor%5D" class="affpopup" co-rid="_co_idm140559507728256" co-class="co-affbox">Jacalyn M. Green</a>,<sup>2</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Nichols%20BP%5BAuthor%5D" class="affpopup" co-rid="_co_idm140559511717840" co-class="co-affbox">Brian P. Nichols</a><sup>1,</sup><sup>&#x0002a;</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140559512463280"><h3 class="no_margin">Mouyassar J. Hussein</h3><p> Laboratory for Molecular Biology, Department of Biological Sciences, University of Illinois at Chicago, Chicago, Illinois 60607,<sup>1</sup> and   Department of Basic Biomedical Sciences, Dr. William M. Scholl College of Podiatric Medicine, Chicago, Illinois 60612<sup>2</sup></p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hussein%20MJ%5BAuthor%5D">Mouyassar J. Hussein</a></div></div><div id="_co_idm140559507728256"><h3 class="no_margin">Jacalyn M. Green</h3><p> Laboratory for Molecular Biology, Department of Biological Sciences, University of Illinois at Chicago, Chicago, Illinois 60607,<sup>1</sup> and   Department of Basic Biomedical Sciences, Dr. William M. Scholl College of Podiatric Medicine, Chicago, Illinois 60612<sup>2</sup></p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Green%20JM%5BAuthor%5D">Jacalyn M. Green</a></div></div><div id="_co_idm140559511717840"><h3 class="no_margin">Brian P. Nichols</h3><p> Laboratory for Molecular Biology, Department of Biological Sciences, University of Illinois at Chicago, Chicago, Illinois 60607,<sup>1</sup> and   Department of Basic Biomedical Sciences, Dr. William M. Scholl College of Podiatric Medicine, Chicago, Illinois 60612<sup>2</sup></p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Nichols%20BP%5BAuthor%5D">Brian P. Nichols</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140559505327600_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140559505327600_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140559505327600_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140559505327600_ai" style="display:none"><div class="fm-affl" id="N0x9c9c5d8.0x9c84ca0"> Laboratory for Molecular Biology, Department of Biological Sciences, University of Illinois at Chicago, Chicago, Illinois 60607,<sup>1</sup> and   Department of Basic Biomedical Sciences, Dr. William M. Scholl College of Podiatric Medicine, Chicago, Illinois 60612<sup>2</sup></div><div id="FN150"><sup>&#x0002a;</sup>Corresponding author. Mailing address: Laboratory for Molecular Biology, Department of Biological Sciences, Molecular Biology Research Building m/c 567, University of Illinois at Chicago, 900 S. Ashland Ave., Chicago, IL 60607. Phone: (312) 996-5064. Fax: (312) 413-2691. E-mail: <a href="mailto:dev@null" data-email="ude.ciu@slohcin.p.nairb" class="oemail">ude.ciu@slohcin.p.nairb</a>.</div></div><div class="fm-article-notes hide half_rhythm" id="idm140559505327600_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 1998 May 13; Accepted 1998 Sep 23.</div></div><div class="permissions half_rhythm hide" id="idm140559505327600_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  &#x000a9; 1998, American Society for Microbiology</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="abstract-a.q.b.p" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.q.b.ptitle" style="text-transform: uppercase;">Abstract</h2><!--article-meta--><div><p class="p p-first-last">An <em>Escherichia coli</em> strain deficient in <em>p</em>-aminobenzoate synthesis was mutagenized, and derivatives were selected for growth on folic acid. Supplementation was shown to be due to <em>p</em>-aminobenzoyl-glutamate present as a breakdown product in commercial folic acid preparations. Two classes of mutations characterized by the minimum concentration of <em>p</em>-aminobenzoyl-glutamate that could support growth were obtained. Both classes of mutations were genetically and physically mapped to about 30 min on the <em>E. coli</em> chromosome. A cloned wild-type gene from this region, <em>abgT</em> (formerly <em>ydaH</em>) could confer a similar <em>p</em>-aminobenzoyl-glutamate utilization phenotype on the parental strain. Interruption of <em>abgT</em> on the plasmid or on the chromosome of the mutant strain resulted in a loss of the phenotype. <em>abgT</em> was the third gene in an apparent operon containing <em>abgA</em>, <em>abgB</em>, <em>abgT</em>, and possibly <em>ogt</em> and might be regulated by a divergently transcribed LysR-type regulator encoded by <em>abgR</em>. Two different single-base-pair mutations that gave rise to the <em>p</em>-aminobenzoyl-glutamate utilization phenotype lay in the <em>abgR-abgA</em> intercistronic region and appeared to allow the expression of <em>abgT</em>. The second class of mutation was due to a tandem duplication of <em>abgB</em> and <em>abgT</em> fused to <em>fnr</em>. The <em>abgA</em> and <em>abgB</em> gene products were homologous to one another and to a family of aminoacyl aminohydrolases. <em>p</em>-Aminobenzoyl-glutamate hydrolysis could be detected in extracts from several of the mutant strains, but intact <em>abgA</em> and <em>abgB</em> were not essential for <em>p</em>-aminobenzoyl-glutamate utilization when <em>abgT</em> was supplied in <em>trans</em>.</p></div></div><div id="body-a.r" class="tsec sec"><h2 class="headless nomenu"></h2><p class="p p-first">It has long been known that <em>Escherichia coli</em> and similar organisms cannot utilize exogenously supplied oxidized forms of folic acid or its derivatives. A blockade of dihydrofolate synthesis by inhibitors of dihydropteroate synthase (DHPS) (such as sulfathiazole or other sulfonamide derivatives) or of dihydrofolate reductase (DHFR) (aminopterin or trimethoprim) can be circumvented only by supplementation with the end products of folate metabolism, i.e., purines, thymidine, glycine, methionine, and pantothenic acid (<a href="#B14" rid="B14" class=" bibr popnode">14</a>, <a href="#B37" rid="B37" class=" bibr popnode">37</a>). Even then, growth is slow, partly because of the failure to formylate methionyl-tRNA<sup>fMet</sup> for the initiation of protein synthesis (<a href="#B2" rid="B2" class=" bibr popnode">2</a>, <a href="#B14" rid="B14" class=" bibr popnode">14</a>).</p><p>The failure of <em>E. coli</em> to utilize oxidized folate derivatives directly may be due to a lack of transport of these compounds or the failure to reduce them to dihydro or tetrahydro derivatives once they are transported. As illustrated in Fig. <a href="/pmc/articles/PMC107711/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2em;">Fig.1,</span></span><span>1</span></a>, folate compounds are synthesized in reduced (dihydro) form, beginning with dihydropterin precursors (<a href="#B39" rid="B39" class=" bibr popnode">39</a>). Following the conjugation of dihydropterin pyrophosphate with <em>p</em>-aminobenzoate to form dihydropteroate, glutamate is added to produce dihydrofolate. Dihydrofolate is then reduced by DHFR to yield tetrahydrofolate, the active one-carbon carrier for cellular metabolism. Bacterial species that require folate derivatives for growth, such as <em>Lactobacillus casei</em>, have elaborate transport systems specific for folate (<a href="#B18" rid="B18" class=" bibr popnode">18</a>). Inhibitory folate analogs, such as aminopterin or methotrexate, are effective against <em>E. coli</em> at relatively high concentrations (<a href="#B24" rid="B24" class=" bibr popnode">24</a>), suggesting that at least passive transport of these compounds may occur. However, once entry into the cell is gained, <em>E. coli</em> DHFR is ineffective in  reducing folate to dihydrofolate (<a href="#B25" rid="B25" class=" bibr popnode">25</a>), a fact which prevents the direct entry of oxidized folate derivatives into pools of reduced folates.  </p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC107711/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140559512536688"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=107711_jb2380606001.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is jb2380606001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC107711/bin/jb2380606001.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140559512536688"><a target="object" rel="noopener" href="/pmc/articles/PMC107711/figure/F1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC107711/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">FIG. 1</a></div><!--caption a7--><div class="caption"><p>Abbreviated scheme for tetrahydrofolate biosynthesis from GTP and chorismate. Compounds to the left of each reaction arrow are additional substrates of the reaction, and compounds to the right are additional products of the reaction. H<sub>2</sub>, dihydro; H<sub>4</sub>, tetrahydro; pABA, <em>p</em>-aminobenzoate; PP, pyrophosphate.</p></div></div></div><p>Another mechanism by which oxidized folates might be utilized is via catabolic or salvage pathways that could reutilize portions of a folate derivative following degradation. In this regard, little is known about bacterial folate salvage, except for organisms that utilize preformed folates. Additionally, the utilization of the pterin product of degradation, namely, pterin or pteroate, might be limited by the same transport or reduction problem cited above.</p><p>The <em>p</em>-aminobenzoate moiety of dihydrofolate is synthesized de novo from chorismate and is easily transported into and out of <em>E. coli</em> cells (<a href="#B12" rid="B12" class=" bibr popnode">12</a>, <a href="#B16" rid="B16" class=" bibr popnode">16</a>, <a href="#B17" rid="B17" class=" bibr popnode">17</a>, <a href="#B23" rid="B23" class=" bibr popnode">23</a>). <em>p</em>-Aminobenzoate auxotrophs can be supplemented by the addition of exogenous <em>p</em>-aminobenzoate, and wild-type cells excrete sufficient <em>p</em>-aminobenzoate to supplement nearby <em>p</em>-aminobenzoate auxotrophs.</p><p class="p">We have examined whether <em>p</em>-aminobenzoate auxotrophs can utilize <em>p</em>-aminobenzoate derived from exogenously supplied folate derivatives; we found mutant strains of <em>E. coli</em> that could utilize <em>p</em>-aminobenzoyl-glutamate, a degradation product of oxidized folate compounds. Single-base-pair mutations that allowed growth on <em>p</em>-aminobenzoyl-glutamate were found in an intergenic region between divergently transcribed genes, one a <em>lysR</em> homolog (here designated <em>abgR</em>, for <em>p</em>-aminobenzoyl-glutamate) and the other the first gene in an apparent operon containing three genes, two homologous to aminoacyl hydrolases (<em>abgA</em> and <em>abgB</em>) and a third gene homologous to permeases (<em>abgT</em>). A 4.2-kb tandem duplication containing <em>abgB</em> and <em>abgT</em> fused to <em>fnr</em> also allowed growth on <em>p</em>-aminobenzoyl-glutamate. Finally, plasmids containing wild-type <em>abgT</em> alone could confer the ability to grow on <em>p</em>-aminobenzoyl-glutamate to wild-type <em>E. coli</em> strains.</p></div><div id="MaterialsMethods" class="tsec sec"><h2 class="head no_bottom_margin" id="MaterialsMethodstitle" style="text-transform: uppercase;">MATERIALS AND METHODS</h2><div id="sec-a.r.f.b" class="sec sec-first"><p></p><h4 id="sec-a.r.f.btitle" class="inline">Microbiological methods. </h4><p class="p p-first">The bacterial strains used in this study are listed in Table <a href="/pmc/articles/PMC107711/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">Table1.</span></span><span>1</span></a>. Minimal medium, LB medium, and NCE medium were prepared as described previously (<a href="#B7" rid="B7" class=" bibr popnode">7</a>). Antibiotic and amino acid final concentrations were also as recommended in reference <a href="#B7" rid="B7" class=" bibr popnode">7</a>. For routine purposes, <em>p</em>-aminobenzoate and <em>p</em>-aminobenzoyl-glutamate were added at 0.7 and 1.8 &#x003bc;M, respectively.  </p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T1"><h3>TABLE 1</h3><!--caption a7--><div class="caption"><p><em>Escherichia coli</em> strains used in this&#x02009;study </p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Strain</th><th rowspan="1" colspan="1">Genotype</th><th rowspan="1" colspan="1">Source or reference<sup>a</sup></th></tr></thead><tbody align="center"><tr><td rowspan="1" colspan="1">BN101</td><td align="left" rowspan="1" colspan="1"><em>pabA1 trpA33 rpsL704 cys</em>(Am) <em>purU tyrT</em></td><td align="left" rowspan="1" colspan="1">23 </td></tr><tr><td rowspan="1" colspan="1">BN141</td><td align="left" rowspan="1" colspan="1">&#x003a6;(<em>pabA-lacZ</em>)<em>17</em>(Hyb) <em>cam</em> &#x00394;(<em>argF-lac</em>)<em>U169 end recB21 recC22 sbcB15 thr-1 argE3 leu-6 his-4 thi-1 lacY ara-14 mtl-1 xyl-5 galK2 rpsL-31 tex-33 sup37</em>(Am)</td><td align="left" rowspan="1" colspan="1"><a href="#B36" rid="B36" class=" bibr popnode">36</a></td></tr><tr><td rowspan="1" colspan="1">BN1001</td><td align="left" rowspan="1" colspan="1"><em>abg-1 pabA1 trpA33 rpsL704 cys</em>(Am) <em>purU tyrT</em></td><td align="left" rowspan="1" colspan="1">DES BN101 </td></tr><tr><td rowspan="1" colspan="1">BN1002</td><td align="left" rowspan="1" colspan="1"><em>abg-2 pabA1 trpA33 rpsL704 cys</em>(Am) <em>purU tyrT</em></td><td align="left" rowspan="1" colspan="1">DES BN101 </td></tr><tr><td rowspan="1" colspan="1">BN1003</td><td align="left" rowspan="1" colspan="1"><em>abg-3 pabA1 trpA33 rpsL704 cys</em>(Am) <em>purU tyrT</em></td><td align="left" rowspan="1" colspan="1">DES BN101 </td></tr><tr><td rowspan="1" colspan="1">BN1004</td><td align="left" rowspan="1" colspan="1"><em>abg-4 pabA1 trpA33 rpsL704 cys</em>(Am) <em>purU tyrT</em></td><td align="left" rowspan="1" colspan="1">DES BN101 </td></tr><tr><td rowspan="1" colspan="1">BN1005</td><td align="left" rowspan="1" colspan="1"><em>abg-5 pabA1 trpA33 rpsL704 cys</em>(Am) <em>purU tyrT</em></td><td align="left" rowspan="1" colspan="1">DES BN101 </td></tr><tr><td rowspan="1" colspan="1">BN1016</td><td align="left" rowspan="1" colspan="1"><em>abg-1 zda-3061</em>::Tn<em>10 pabA1 trpA33 rpsL704 cys</em>(Am) <em>purU tyrT</em></td><td align="left" rowspan="1" colspan="1">BN1001 &#x000d7; <a id="__tag_1133659125" class="tag_hotlink" href="/protein/CAG12081" ref="/protein/CAG12081"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"CAG12081","term_id":"47222055","term_text":"CAG12081"}}</span>CAG12081</a> </td></tr><tr><td rowspan="1" colspan="1">BN1023</td><td align="left" rowspan="1" colspan="1"><em>abg-4 zda-3061</em>::Tn<em>10 pabA1 rpsL704</em></td><td align="left" rowspan="1" colspan="1">BN1103 &#x000d7; BN1004 </td></tr><tr><td rowspan="1" colspan="1">BN1103</td><td align="left" rowspan="1" colspan="1"><em>pabA1 rpsL704</em></td><td align="left" rowspan="1" colspan="1">MG1655 &#x000d7; BN101 </td></tr><tr><td rowspan="1" colspan="1">BN1125</td><td align="left" rowspan="1" colspan="1"><em>abg-1 zda-3061</em>::Tn<em>10</em> &#x003a6;(<em>pabA-lacZ</em>)<em>17</em>(Hyb) <em>cam</em> &#x00394;(<em>argF-lac</em>)<em>U169 end recB21 recC22 sbcB15 thr-1 argE3 leu-6 his-4 thi-1 lacY ara-14 mtl-1 xyl-5 galK2 rpsL31 tex-33 sup37</em>(Am)</td><td align="left" rowspan="1" colspan="1">BN141 &#x000d7; BN1016 </td></tr><tr><td rowspan="1" colspan="1">BN1128</td><td align="left" rowspan="1" colspan="1"><em>abg-1 abgB</em>::<em>kan zda-3061</em>::Tn<em>10</em> &#x003a6;(<em>pabA-lacZ</em>)<em>17</em>(Hyb) <em>cam</em> &#x00394;(<em>argF-lac</em>)<em>U169 end recB21 recC22 sbcB15 thr-1 argE3 leu-6 his-4 thi-1 lacY ara-14 mtl-1 xyl-5 galK2 rpsL31 tex-33 sup37</em>(Am)</td><td align="left" rowspan="1" colspan="1">pJMG111 &#x000d7; BN1125 </td></tr><tr><td rowspan="1" colspan="1">BN1129</td><td align="left" rowspan="1" colspan="1"><em>abg-1 abgA</em>::<em>kan zda-3061</em>::Tn<em>10</em> &#x003a6;(<em>pabA-lacZ</em>)<em>17</em>(Hyb) <em>cam</em> &#x00394;(<em>argF-lac</em>)<em>U169 end recB21 recC22 sbcB15 thr-1 argE3 leu-6 his-4 thi-1 lacY ara-14 mtl-1 xyl-5 galK2 rpsL-31 tex-33 sup37</em>(Am)</td><td align="left" rowspan="1" colspan="1">pJMG112 &#x000d7; BN1125 </td></tr><tr><td rowspan="1" colspan="1">BN1130</td><td align="left" rowspan="1" colspan="1"><em>abg-1 abgT</em>::<em>kan zda-3061</em>::Tn<em>10</em> &#x003a6;(<em>pabA-lacZ</em>)<em>17</em>(Hyb) <em>cam</em> &#x00394;(<em>argF-lac</em>)<em>U169 end recB21 recC22 sbcB15 thr-1 argE3 leu-6 his-4 thi-1 lacY ara-14 mtl-1 xyl-5 galK2 rpsL-31 tex-33 sup37</em>(Am)</td><td align="left" rowspan="1" colspan="1">pBN200 &#x000d7; BN1125 </td></tr><tr><td rowspan="1" colspan="1">BN1140</td><td align="left" rowspan="1" colspan="1"><em>abg-1 zda-3061</em>::Tn<em>10 pabA1 rpsL704</em></td><td align="left" rowspan="1" colspan="1">BN1125 &#x000d7; BN1103 </td></tr><tr><td rowspan="1" colspan="1">BN1141</td><td align="left" rowspan="1" colspan="1"><em>abg-1 abgB</em>::<em>kan zda-3061</em>::Tn<em>10 pabA1 rpsL704</em></td><td align="left" rowspan="1" colspan="1">BN1128 &#x000d7; BN1103 </td></tr><tr><td rowspan="1" colspan="1">BN1142</td><td align="left" rowspan="1" colspan="1"><em>abg-1 abgA</em>::<em>kan zda-3061</em>::Tn<em>10 pabA1 rpsL704</em></td><td align="left" rowspan="1" colspan="1">BN1129 &#x000d7; BN1103 </td></tr><tr><td rowspan="1" colspan="1">BN1143</td><td align="left" rowspan="1" colspan="1"><em>abg-1 abgT</em>::<em>kan zda-3061</em>::Tn<em>10 pabA1 rpsL704</em></td><td align="left" rowspan="1" colspan="1">BN1130 &#x000d7; BN1103 </td></tr><tr><td rowspan="1" colspan="1">BN1150</td><td align="left" rowspan="1" colspan="1"><em>abg-1 zda-3061</em>::Tn<em>10 pabA1 trpA33 rpsL704 cys</em>(Am) <em>purU tyrT</em></td><td align="left" rowspan="1" colspan="1">BN1140 &#x000d7; BN101 </td></tr><tr><td rowspan="1" colspan="1">BN1151</td><td align="left" rowspan="1" colspan="1"><em>abg-1 abgB</em>::<em>kan zda-3061</em>::Tn<em>10 pabA1 trpA33 rpsL704 cys</em>(Am) <em>purU tyrT</em></td><td align="left" rowspan="1" colspan="1">BN1141 &#x000d7; BN101 </td></tr><tr><td rowspan="1" colspan="1">BN1152</td><td align="left" rowspan="1" colspan="1"><em>abg-1 abgA</em>::<em>kan zda-3061</em>::Tn<em>10 pabA1 trpA33 rpsL704 cys</em>(Am) <em>purU tyrT</em></td><td align="left" rowspan="1" colspan="1">BN1142 &#x000d7; BN101 </td></tr><tr><td rowspan="1" colspan="1">BN1153</td><td align="left" rowspan="1" colspan="1"><em>abg-1 abgT</em>::<em>kan zda-3061</em>::Tn<em>10 pabA1 trpA33 rpsL704 cys</em>(Am) <em>purU tyrT</em></td><td align="left" rowspan="1" colspan="1">BN1143 &#x000d7; BN101 </td></tr><tr><td rowspan="1" colspan="1">JC7623</td><td align="left" rowspan="1" colspan="1"><em>end recB21 recC22 sbcB15 thr-1 proA2 argE3 leu-6 his-4 thi-1 lacY ara-14 mtl-1 xyl-5 galK2  rpsL-31 tex-33 sup37</em>(Am)</td><td align="left" rowspan="1" colspan="1">36 </td></tr><tr><td rowspan="1" colspan="1">MG1655</td><td align="left" rowspan="1" colspan="1">Wild type</td><td align="left" rowspan="1" colspan="1"><a href="#B35" rid="B35" class=" bibr popnode">35</a></td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140559512534096"><a target="object" rel="noopener" href="/pmc/articles/PMC107711/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TF1-a"><sup>a</sup>DES, diethyl sulfate.&#x000a0;</div></div></div><p class="p p-last">Diethyl sulfate mutagenesis was performed by adding 100 &#x003bc;l of an overnight culture of <em>E. coli</em> BN101 to 10 ml of NCE medium that had been saturated with diethyl sulfate. Following 60 min of incubation at 37&#x000b0;C, 200 &#x003bc;l of the suspension was used to inoculate 5 ml of LB medium. After overnight growth, cells were harvested, washed with 0.15 M NaCl, diluted, and spread on minimal medium plates supplemented with tryptophan and either folic acid (45 &#x003bc;M) or folinic acid (40 &#x003bc;M). Colonies that appeared after 2 days of growth were picked and restreaked onto plates supplemented with tryptophan alone or tryptophan plus folic acid in order to eliminate <em>pabA</em> revertants. Five independently mutagenized strains that grew when supplemented with either folic acid or folinic acid were retained and designated <em>E. coli</em> BN1001 to BN1005.</p></div><div id="sec-a.r.f.c" class="sec"><p></p><h4 id="sec-a.r.f.ctitle" class="inline">Genetic mapping and transduction. </h4><p class="p p-first-last">Mutations that allowed growth on folate compounds were mapped by conjugation with the Hfr strains and protocols described by Singer et al. (<a href="#B35" rid="B35" class=" bibr popnode">35</a>). Bacteriophage P1 transductions were performed as described by Miller (<a href="#B21" rid="B21" class=" bibr popnode">21</a>).</p></div><div id="sec-a.r.f.d" class="sec"><p></p><h4 id="sec-a.r.f.dtitle" class="inline">Molecular methods. </h4><p class="p p-first"><em>E. coli</em> chromosomal DNA was prepared by established methods (<a href="#B30" rid="B30" class=" bibr popnode">30</a>, <a href="#B38" rid="B38" class=" bibr popnode">38</a>). Small-scale plasmid samples were prepared either by the rapid alkaline lysis technique of Birnboim and Doly (<a href="#B3" rid="B3" class=" bibr popnode">3</a>) or by the rapid boiling technique (<a href="#B30" rid="B30" class=" bibr popnode">30</a>). Bacteriophage &#x003bb; DNA was prepared as described previously (<a href="#B30" rid="B30" class=" bibr popnode">30</a>).</p><p>Restriction endonuclease digestions and ligation reactions were performed in accordance with manufacturers&#x02019; recommendations and with commercial buffer preparations (New England Biolabs, Inc.; Boehringer Mannheim Biochemicals, Inc.; International Biotechnologies Inc.; and Bethesda Research Laboratories, Inc.).</p><p>PCR was performed with the <em>Taq</em> PCR Core Kit (Qiagen) and Platinum <em>Taq</em> polymerase (Gibco/BRL). Reactions were carried out by use of 25-&#x003bc;l volumes with synthetic 24-mer oligonucleotides as primers (1 &#x003bc;M each). DNA sequence analysis was performed with a T7 Sequenase (version 2.0) sequencing kit (Amersham), and PCR products were sequenced with a similar PCR product sequencing kit (Amersham). Sequencing primers were typically 18 to 20 nucleotides long.</p><p class="p p-last">Southern hybridizations were done as described previously (<a href="#B30" rid="B30" class=" bibr popnode">30</a>). Hybridization and washes were carried out at 65&#x000b0;C with 2&#x000d7; SSC (1&#x000d7; SSC is 0.15 M NaCl plus 0.015 M sodium citrate) buffer.</p></div><div id="sec-a.r.f.e" class="sec"><p></p><h4 id="sec-a.r.f.etitle" class="inline">Enzyme assays. </h4><p class="p p-first">Crude extracts were prepared as previously described (<a href="#B23" rid="B23" class=" bibr popnode">23</a>). Hydrolysis of <em>p</em>-aminobenzoyl-glutamate to <em>p</em>-aminobenzoate was determined by incubating 1-ml volumes containing 50 mM Tris-HCl (pH 8.5), 10 mM MgSO<sub>4</sub>, 10 mM &#x003b2;-mercaptoethanol, and 50 &#x003bc;M <em>p</em>-aminobenzoyl-glutamate with 0.1 to 1.5 mg of protein for 60 min at 37&#x000b0;C. Reactions were terminated with the addition of 0.1 ml of 1 N HCl, and <em>p</em>-aminobenzoate was extracted and quantitated spectrophotofluorometrically as previously described (<a href="#B23" rid="B23" class=" bibr popnode">23</a>). One unit of activity was defined as the amount of protein required to hydrolyze 1 nmol of <em>p</em>-aminobenzoyl-glutamate in 1 h at 37&#x000b0;C.</p><p>A routine assay for DHPS was done with the coupled 6-hydroxymethyl-dihydropteridine pyrophosphokinase-DHPS assay described by Richey and Brown (<a href="#B28" rid="B28" class=" bibr popnode">28</a>). 6-Hydroxymethyl-pteridine was reduced to the dihydro form as described by Shiota et al. (<a href="#B34" rid="B34" class=" bibr popnode">34</a>) prior to use. <sup>14</sup>C-labeled <em>p</em>-aminobenzoate was purchased from ICN and diluted to 8 mCi/mmol prior to use.</p><p class="p p-last">Protein concentrations were determined by the Bradford dye-binding technique (<a href="#B5" rid="B5" class=" bibr popnode">5</a>).</p></div><div id="sec-a.r.f.f" class="sec"><p></p><h4 id="sec-a.r.f.ftitle" class="inline">Bioautography. </h4><p class="p p-first-last">Bioautography was carried out with <em>E. coli</em> BN101 and BN1001 as test strains. Pyrex baking dishes (21 by 27 cm) dishes were prepared by pouring a layer (200 ml) of solid minimal medium supplemented with tryptophan and cysteine, followed by a layer (100 ml) of the same medium seeded with 5 &#x000d7; 10<sup>8</sup> cells of the test strain per ml. The top layer was also supplemented with 0.15 mM isopropyl-thio-&#x003b2;-<span class="small-caps">d</span>-galactoside and 10 &#x003bc;M 5-bromo-4-chloro-3-indolyl-&#x003b2;-<span class="small-caps">d</span>-galactoside (X-Gal) to aid in the visualization of growth zones. Samples were dissolved and filter sterilized prior to application to Whatman 3MM chromatography paper (15 by 20 cm). The chromatogram was developed by ascending chromatography in 0.1 M potassium phosphate (pH 7.0) until the solvent had reached 2 cm from the top. The chromatogram was air dried, and material was allowed to transfer to solid medium for 2 h. The chromatogram was lifted, and the dish was covered with foil and incubated at 37&#x000b0;C. Samples and quantities used were as follows: <em>p</em>-aminobenzoate and <em>p</em>-aminobenzoyl-glutamate, 7 nmol; folic acid, 110 nmol; folinic acid, 110 nmol; and pteroic acid, 80 nmol.</p></div><div id="sec-a.r.f.g" class="sec sec-last"><p></p><h4 id="sec-a.r.f.gtitle" class="inline">Construction of interruptions in <em>abgA</em>, <em>abgB</em>, and <em>abgT.</em> </h4><p class="p p-first">To test the roles of <em>abgA</em>, <em>abgB</em>, and <em>abgT</em> in <em>p</em>-aminobenzoyl-glutamate utilization, each open reading frame was interrupted with a kanamycin resistance cassette and then transferred to the chromosomes of <em>E. coli</em> BN1125 (<em>abg-1</em>) and BN1126 (<em>abg</em><sup>&#x0002b;</sup>). pMJH123 was digested with <em>Pst</em>I and ligated with the <em>Pst</em>I DNA fragment containing the kanamycin resistance determinant derived from pMB2190. The product (<em>abgT</em>::<em>kan</em>) was designated pMJH132. pMJH124 was partially digested with <em>Pst</em>I and similarly ligated with the <em>Pst</em>I DNA fragment containing the kanamycin resistance determinant. Plasmid products containing the kanamycin resistance cassette inserted into <em>abgA</em> and <em>abgB</em> were isolated. The <em>Eco</em>RI fragments were recloned into the <em>Eco</em>RI site of pBSIISK&#x0002b; to yield pJMG111 (<em>abgB</em>::<em>kan</em>) and pJMG112 (<em>abgA</em>::<em>kan</em>).</p><p class="p p-last">Plasmid-borne interruptions were recombined into the chromosome by homologous recombination in <em>E. coli</em> BN141, a <em>pabA</em> derivative of <em>E. coli</em> JC7623. The <em>abg-1</em> mutation was crossed into BN141 by cotransduction with a bacteriophage P1 lysate derived from <em>E. coli</em> BN1016, which contained a Tn<em>10</em> insertion near the <em>abg</em> locus. Selection for tetracycline resistance was followed by screening for growth on <em>p</em>-aminobenzoyl-glutamate to avoid selection of new <em>abg</em> alleles. An <em>abg-1</em> derivative of <em>E. coli</em> BN141 was designated <em>E. coli</em> BN1125. Plasmids were linearized with either <em>Bam</em>HI (pMJH132) or <em>Sca</em>I (pJMG111 and pJMG112) and used to transform <em>E. coli</em> BN1125 to kanamycin resistance. Kanamycin-resistant, ampicillin-sensitive derivatives were chosen and tested for <em>p</em>-aminobenzoyl-glutamate utilization. None of the plasmids contained DNA spanning the site of the <em>abg-1</em> mutation, so that the original mutation could not be lost in the recombination process. The <em>abg</em>::<em>kan</em> alleles were then transduced into BN1103 and BN101 to yield BN1140 to BN1143 and BN1150 to BN1153, respectively. These latter constructions yielded strains capable of supporting plasmid replication.</p></div></div><div id="Results" class="tsec sec"><h2 class="head no_bottom_margin" id="Resultstitle" style="text-transform: uppercase;">RESULTS</h2><div id="sec-a.r.g.b" class="sec sec-first"><p></p><h4 id="sec-a.r.g.btitle" class="inline">Selection and characterization of <em>abg</em> mutants. </h4><p class="p p-first">To obtain <em>E. coli</em> strains that could grow on folate or folate-related compounds, a <em>pabA</em> derivative of <em>E. coli</em> (BN101) was mutagenized with diethyl sulfate and selected for growth on medium supplemented with folic acid or folinic acid. Because the growth requirement for these compounds on solid medium was high (&#x0003e;10 &#x003bc;M) and the requirement for <em>p</em>-aminobenzoate supplementation was low (&#x0003c;30 nM), we sought to test the strains for growth on potentially contaminating material present in the commercially supplied supplements.</p><p class="p p-last">Bioautography experiments identified <em>p</em>-aminobenzoyl-glutamate as the relevant nutrient that satisfied the <em>p</em>-aminobenzoate requirement in mutant strain BN1001. Several preparations of folic acid and folinic acid were tested by this method for their ability to substitute for <em>p</em>-aminobenzoate. Table <a href="/pmc/articles/PMC107711/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">Table2</span></span><span>2</span></a> shows that the growth response of <em>E. coli</em> BN1001 was centered on a compound that was present in folic acid and folinic acid preparations and that chromatographed with an <em>R</em><sub><em>f</em></sub> of 0.94, well removed from the zone of migration of folic acid (<em>R</em><sub><em>f</em></sub>, 0.43), folinic acid (<em>R</em><sub><em>f</em></sub>, 0.17), or <em>p</em>-aminobenzoate (<em>R</em><sub><em>f</em></sub>, 0.77). The <em>R</em><sub><em>f</em></sub> of the unknown compound suggested that <em>p</em>-aminobenzoyl-glutamate may be the supplement conferring growth on the <em>pabA</em> strain (<a href="#B32" rid="B32" class=" bibr popnode">32</a>). The contaminating compound that supported growth comigrated with authentic <em>p</em>-aminobenzoyl-glutamate, which also supported the growth of <em>E. coli</em> BN1001. Parallel experiments with the parental strain <em>E. coli</em> BN101 showed that significant growth occurred only on <em>p</em>-aminobenzoate and that growth did not occur on any other compound, including <em>p</em>-aminobenzoyl-glutamate, at the concentration tested. These data indicated that the growth observed on folic acid and folinic acid preparations was due to the <em>p</em>-aminobenzoyl-glutamate present in these preparations, presumably having been formed as a spontaneous degradation product of folic acid. We designated the mutations <em>abg</em>, for <em>p</em>-aminobenzoyl-glutamate utilization.  </p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T2"><h3>TABLE 2</h3><!--caption a7--><div class="caption"><p>Growth on contaminating material present in folate compounds, as revealed by&#x02009;bioautography </p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1">Compound applied to chromatogram</th><th colspan="2" rowspan="1"><em>R</em><sub><em>f</em></sub> of: 
<hr /></th></tr><tr><th rowspan="1" colspan="1">UV-absorbing material</th><th rowspan="1" colspan="1">Compound supporting the growth of <em>E. coli</em> BN1001 </th></tr></thead><tbody align="center"><tr><td rowspan="1" colspan="1">Folic acid</td><td rowspan="1" colspan="1">0.43</td><td rowspan="1" colspan="1">0.94 </td></tr><tr><td rowspan="1" colspan="1">Folinic acid</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">0.94 </td></tr><tr><td rowspan="1" colspan="1"><em>p</em>-Aminobenzoate</td><td rowspan="1" colspan="1">0.77</td><td rowspan="1" colspan="1">0.77 </td></tr><tr><td rowspan="1" colspan="1"><em>p</em>-Aminobenzoyl-glutamate</td><td rowspan="1" colspan="1">0.94</td><td rowspan="1" colspan="1">0.94</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140559514325872"><a target="object" rel="noopener" href="/pmc/articles/PMC107711/table/T2/?report=objectonly">Open in a separate window</a></div></div></div><div id="sec-a.r.g.c" class="sec"><p></p><h4 id="sec-a.r.g.ctitle" class="inline">Growth response of strains to <em>p</em>-aminobenzoyl-glutamate. </h4><p class="p p-first-last"><em>E. coli</em> BN101 and BN1103 grew poorly on <em>p</em>-aminobenzoyl-glutamate concentrations below 10 &#x003bc;M, and bioautography showed that growth on higher concentrations was due to contaminating <em>p</em>-aminobenzoate in commercial preparations. However, the <em>abg</em> mutant strains could grow on much lower levels of <em>p</em>-aminobenzoyl-glutamate (Table <a href="/pmc/articles/PMC107711/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table3).</span></span><span>3</span></a>). The minimum concentration required for full growth was determined by inspection of colony size on solid minimal medium containing different concentrations of <em>p</em>-aminobenzoyl-glutamate (ranging from 0.4 nM to 200 &#x003bc;M). While growth became limited for the parental strain at below 20 &#x003bc;M, the growth of <em>E. coli</em> BN1001, BN1002, BN1003, or BN1005 was not limited until the <em>p</em>-aminobenzoyl-glutamate concentration fell below 1 &#x003bc;M. The growth of <em>E. coli</em> BN1004, on the other hand, was not limited until the <em>p</em>-aminobenzoyl-glutamate concentration fell below 0.1 &#x003bc;M. Two classes of mutations were evident from this analysis, one that increased the sensitivity to <em>p</em>-aminobenzoyl-glutamate approximately 20-fold and one that increased the sensitivity approximately 200-fold over the concentration required by the parent. Growth limitation in response to <em>p</em>-aminobenzoate for all strains was not evident until the concentration fell below 0.02 &#x003bc;M.  </p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T3"><h3>TABLE 3</h3><!--caption a7--><div class="caption"><p>Growth response of <em>E. coli</em> strains to  <em>p</em>-aminobenzoyl-glutamate&#x02009;concentration<sup>a</sup> </p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Strain </th><th rowspan="1" colspan="1"><em>p</em>-Aminobenzoyl-glutamate concn allowing full growth (&#x003bc;M) </th></tr></thead><tbody align="center"><tr><td rowspan="1" colspan="1">BN101</td><td rowspan="1" colspan="1">20 </td></tr><tr><td rowspan="1" colspan="1">BN1103</td><td rowspan="1" colspan="1">10 </td></tr><tr><td rowspan="1" colspan="1">BN1001</td><td rowspan="1" colspan="1">1 </td></tr><tr><td rowspan="1" colspan="1">BN1002</td><td rowspan="1" colspan="1">1 </td></tr><tr><td rowspan="1" colspan="1">BN1003</td><td rowspan="1" colspan="1">1 </td></tr><tr><td rowspan="1" colspan="1">BN1004</td><td rowspan="1" colspan="1">0.1 </td></tr><tr><td rowspan="1" colspan="1">BN1005</td><td rowspan="1" colspan="1">1 </td></tr><tr><td rowspan="1" colspan="1">BN1023</td><td rowspan="1" colspan="1">0.1</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140559514307584"><a target="object" rel="noopener" href="/pmc/articles/PMC107711/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TF3-a"><sup>a</sup>The concentrations tested were 200, 100, 40, 20, 10, 4, 2, 1, 0.4, 0.2, 0.1, 0.04, 0.02, 0.01, 0.004, 0.002, and 0.001 &#x003bc;M.&#x000a0;</div></div></div></div><div id="sec-a.r.g.d" class="sec"><p></p><h4 id="sec-a.r.g.dtitle" class="inline">Mutant strains contain unaltered DHPS. </h4><p class="p p-first-last">Although <em>p</em>-aminobenzoyl-glutamate is not used as the normal substrate for dihydrofolate synthesis in vivo (<a href="#B28" rid="B28" class=" bibr popnode">28</a>), it has been reported that DHPS from <em>E. coli</em> as well as other microorganisms can use <em>p</em>-aminobenzoyl-glutamate as a substrate in vitro in place of <em>p</em>-aminobenzoate and thereby bypass dihydropteroate to produce dihydrofolate directly (<a href="#B29" rid="B29" class=" bibr popnode">29</a>) (Fig. <a href="/pmc/articles/PMC107711/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.1).</span></span><span>1</span></a>). To test if the <em>p</em>-aminobenzoyl-glutamate utilization mutants contained altered DHPS that might facilitate the in vivo utilization of <em>p</em>-aminobenzoyl-glutamate, DHPS from mutants BN1001, BN1004, and BN1005 was assayed to determine if it could use either <em>p</em>-aminobenzoyl-glutamate or <em>p</em>-aminobenzoate with an efficiency different from that of parental <em>E. coli</em> BN101 DHPS. As shown in Fig. <a href="/pmc/articles/PMC107711/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2em;">Fig.2,</span></span><span>2</span></a>, there was no apparent difference in the <em>K</em><sub><em>m</em></sub> for <em>p</em>-aminobenzoate, nor was there any apparent difference in the ability of <em>p</em>-aminobenzoyl-glutamate to compete with <em>p</em>-aminobenzoate for incorporation into dihydropteroate. Since no difference in DHPS activities was evident, we concluded that if <em>p</em>-aminobenzoyl-glutamate were utilized directly, it was by a mechanism not related to an altered DHPS.  </p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC107711/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140559514285264"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=107711_jb2380606002.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is jb2380606002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC107711/bin/jb2380606002.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140559514285264"><a target="object" rel="noopener" href="/pmc/articles/PMC107711/figure/F2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC107711/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">FIG. 2</a></div><!--caption a7--><div class="caption"><p>Characterization of DHPS from <em>E. coli</em> BN101 and <em>abg</em> derivatives. (a) Comparison of <em>K</em><sub><em>m</em></sub>s of DHPS from <em>E. coli</em> BN101 (&#x025a0;), BN1001 (&#x02022;), and BN1004 (&#x025cb;). (b) Comparison of the ability of <em>p</em>-aminobenzoyl-glutamate (p-ABG) to compete with <sup>14</sup>C-labeled <em>p</em>-aminobenzoate in the DHPS reaction in <em>E. coli</em> BN101 (&#x02022;), BN1001 (&#x025cb;), BN1004 (&#x025a0;), and BN1005 (&#x025a1;). p-ABA, <em>p</em>-aminobenzoate; v, velocity.</p></div></div></div></div><div id="sec-a.r.g.e" class="sec"><p></p><h4 id="sec-a.r.g.etitle" class="inline">Mapping and cloning of <em>abgT.</em> </h4><p class="p p-first">The <em>abg</em> mutation of one of the strains (<em>E. coli</em> BN1004) was mapped by Hfr conjugation to between 25 and 35 min and by P1 transduction to approximately 30 min on the <em>E. coli</em> chromosome. The <em>abg</em> allele was 50&#x00025; cotransducible with the <em>zda-3061</em>::Tn<em>10</em> marker of <em>E. coli</em> <a id="__tag_1133659126" class="tag_hotlink" href="/protein/CAG12081" ref="/protein/CAG12081"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"CAG12081","term_id":"47222055","term_text":"CAG12081"}}</span>CAG12081</a>, which is located at approximately 29.5 min (<a href="#B35" rid="B35" class=" bibr popnode">35</a>). The cotransduction frequencies for the <em>abg</em> alleles from <em>E. coli</em> BN1001, BN1002, BN1003, and BN1005 were similar. In addition, <em>abg</em> Tc<sup>r</sup> recipients of the mutant strains were used to backcross the <em>abg</em> allele to either <em>E. coli</em> BN101 or <em>E. coli</em> BN1103, confirming that no additional mutation distant from this locus was necessary for the growth phenotype, nor was the phenotype dependent on another factor in the <em>E. coli</em> BN101 genetic background.</p><p class="p p-last">Plasmid libraries were constructed with <em>Hin</em>dIII-digested chromosomal DNA from <em>E. coli</em> BN1023 (derived from BN1004) and pBR322. <em>E. coli</em> BN101 was transformed with the libraries and plated on minimal medium supplemented with tryptophan, ampicillin, and folic acid. Plasmids were prepared from 10 colonies and were found to contain identical 4.2-kb inserts. A radiolabeled plasmid was used to probe the Kohara lambda library (<a href="#B20" rid="B20" class=" bibr popnode">20</a>), and hybridization to two phages containing overlapping inserts, &#x003bb;6B4(260) and &#x003bb;3G3(261), was found at a physical location consistent with the genetic map position. However, no 4.2-kb <em>Hin</em>dIII fragment was present on the physical restriction map or on the two hybridizing phages. We tested <em>Hin</em>dIII digests of chromosomal DNAs from all the mutant strains by using the same plasmid as a probe. All mutant strains except for BN1004 showed a hybridization pattern consistent with the Kohara <em>Hin</em>dIII restriction map. <em>E. coli</em> BN1004 (and its derivative BN1023), on the other hand, contained the <em>Hin</em>dIII fragments predicted for the wild-type strain and an additional 4.2-kb <em>Hin</em>dIII fragment. These data (and additional data; see below) suggested that a duplication of a segment of the DNA in this region was responsible for the <em>p</em>-aminobenzoyl-glutamate utilization phenotype. We reasoned that if a duplication event could give rise to the Abg phenotype, then cloning of the wild-type DNA fragments from the phage clones in a moderate copy number might allow us to identify the gene(s) responsible for the phenotype. A 7.5-kb <em>Bam</em>HI-<em>Sal</em>I fragment from &#x003bb;3G3(261) cloned into pACYC184 yielded a plasmid (pMJH113) that could confer the <em>p</em>-aminobenzoyl-glutamate utilization phenotype on <em>E. coli</em> BN101 or BN1103 (Fig. <a href="/pmc/articles/PMC107711/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.3).</span></span><span>3</span></a>). A variety of DNA fragments were subcloned from pMJH113 into a variety of vectors and tested for growth on <em>p</em>-aminobenzoyl-glutamate. These experiments delimited the DNA fragment of interest to a <em>Kpn</em>I-<em>Hin</em>dIII fragment, as represented by the 2.3-kb DNA fragment present in pMJH128.  </p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC107711/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140559514280752"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=107711_jb2380606003.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is jb2380606003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC107711/bin/jb2380606003.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140559514280752"><a target="object" rel="noopener" href="/pmc/articles/PMC107711/figure/F3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC107711/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">FIG. 3</a></div><!--caption a7--><div class="caption"><p>Map of the 7,266-bp region of the <em>E. coli</em> chromosome from bp 1,396,431 to bp 1,403,697, shown in the inverse orientation. Restriction sites used for subcloning purposes are illustrated, as are the extents of known reading frames. DNA fragments used in this study to localize <em>abgT</em> and to construct interruptions in <em>abg</em> genes are illustrated, and the vectors in which they are cloned are indicated. The vector names (leftmost column) and plasmid names are shown to the left of the map. Triangles denote the positions of kanamycin resistance cassette insertions. The ability of each plasmid to confer <em>p</em>-aminobenzoyl-glutamate utilization is indicated at the right.</p></div></div></div></div><div id="sec-a.r.g.f" class="sec"><p></p><h4 id="sec-a.r.g.ftitle" class="inline">Nucleotide sequence of a gene allowing <em>p</em>-aminobenzoyl-glutamate utilization. </h4><p class="p p-first">To sequence the 2.3-kb <em>Kpn</em>I-<em>Hin</em>dIII insert of pMJH128, a sequence strategy involving both ordered deletions (<a href="#B15" rid="B15" class=" bibr popnode">15</a>) and synthetic oligonucleotides was used. The entire sequence was completed on both strands, and all sites and primer sequences were overlapped. The nucleotide sequence of a 2,266-bp DNA fragment containing one complete and two partial open reading frames was determined (Fig. <a href="/pmc/articles/PMC107711/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.3).</span></span><span>3</span></a>). Database searches with BlastP (<a href="#B1" rid="B1" class=" bibr popnode">1</a>) identified the 3&#x02032;-terminal open reading frame as <em>ogt</em>, encoding a DNA repair enzyme, <em>O</em><sup>6</sup>-alkyl-guanine-DNA alkyltransferase (<a href="#B26" rid="B26" class=" bibr popnode">26</a>). The upstream open reading frame was present on the complete sequence of the <em>E. coli</em> genome and had been given the designation <em>ydaH</em>, indicative of an open reading frame of unknown function (<a href="#B4" rid="B4" class=" bibr popnode">4</a>).</p><p>The complete open reading frame, designated <em>ydaH</em>, that was present on the sequenced DNA fragment was 1,533 bp long and encoded a 511-amino-acid hydrophobic protein with a predicted molecular mass of 55.1 kDa. Hydropathy analyses of the predicted amino acid sequence suggested the presence of 12 transmembrane alpha helices and classified the protein as an integral membrane protein (<a href="#B9" rid="B9" class=" bibr popnode">9</a>, <a href="#B19" rid="B19" class=" bibr popnode">19</a>, <a href="#B27" rid="B27" class=" bibr popnode">27</a>). Database searches did not yield a single protein with high similarity but did yield several proteins with similarity in limited segments. All of the proteins were transport or efflux proteins, and most were of the sodium-solute symport family, containing 12 transmembrane alpha helices. On the basis of these predictions, we have designated the gene <em>abgT</em>, for <em>p</em>-aminobenzoyl-glutamate transport.</p><p>As mentioned above, wild-type <em>abgT</em> present on pMJH123 could confer <em>p</em>-aminobenzoyl-glutamate utilization on <em>abg</em><sup>&#x0002b;</sup> strains. Plasmid pMJH132, containing interrupted <em>abgT</em> by virtue of the insertion of a kanamycin resistance cassette at the <em>Pst</em>I site, failed to confer <em>p</em>-aminobenzoyl-glutamate utilization on <em>abg</em><sup>&#x0002b;</sup> strains. Furthermore, transfer of the <em>abgT</em> interruption to the chromosome of an <em>abg-1</em> strain resulted in the reversal of the Abg phenotype. These data indicated that a functional, wild-type <em>abgT</em> gene is essential for the utilization of <em>p</em>-aminobenzoyl-glutamate and, when present on a multicopy plasmid, is sufficient to confer <em>p</em>-aminobenzoyl-glutamate utilization.</p><p>The 3.0-kb <em>Eco</em>RI DNA fragment containing <em>abgT</em> was cloned from chromosomal digests of <em>E. coli</em> BN1001 and BN1003, and one strand was sequenced with primers derived from the wild-type sequence. No differences from the wild-type sequence were found, suggesting that the <em>p</em>-aminobenzoyl-glutamate utilization phenotype was not due to a gain-of-function mutation in an existing transport protein. Since duplicated <em>abgT</em> and multicopy plasmid-borne <em>abgT</em> could both confer <em>p</em>-aminobenzoyl-glutamate utilization, we considered the possibility that the differential expression of <em>abgT</em> might be responsible for the Abg phenotype.</p><p class="p p-last">An additional sequence analysis of the wild-type and mutant strains suggested that <em>abgT</em> and possibly <em>ogt</em> were part of a larger operon. We extended the sequence of both strands to the <em>Sal</em>I site upstream of <em>abgT</em> (Fig. <a href="/pmc/articles/PMC107711/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.3).</span></span><span>3</span></a>). The sequence determined was identical to that published (<a href="#B4" rid="B4" class=" bibr popnode">4</a>). <em>ogt</em> and <em>abgT</em> occurred in a context that suggested an operon structure with two additional open reading frames upstream of <em>abgT</em> (given labels b1337 and b1338) adjacent to a divergently transcribed <em>lysR</em>-like regulatory gene (b1339) that encodes a protein 27&#x00025; identical to TdcA, a LysR-like transcription activator of the threonine dehydratase operon (<a href="#B10" rid="B10" class=" bibr popnode">10</a>, <a href="#B31" rid="B31" class=" bibr popnode">31</a>).</p></div><div id="sec-a.r.g.g" class="sec"><p></p><h4 id="sec-a.r.g.gtitle" class="inline">Identification of mutations leading to <em>p</em>-aminobenzoyl-glutamate utilization. </h4><p class="p p-first">To determine if the mutations resulting in <em>p</em>-aminobenzoyl-glutamate utilization were in the regulatory region of the putative <em>abg</em> operon, a 1,105-bp DNA fragment spanning the <em>abgR-abgA</em> intercistronic region was amplified from the parental <em>E. coli</em> BN101 DNA and mutant <em>abg</em> chromosomal DNA preparations by PCR. The nucleotide sequences of the PCR products were determined directly and showed that the sequences from the parental strain and BN1004 were identical to the published wild-type sequence (Fig. <a href="/pmc/articles/PMC107711/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.4),</span></span><span>4</span></a>), while the products derived from BN1001, BN1002, BN1003, and BN1005 each contained one nucleotide difference. Extending the sequence determination of BN1001 to include <em>abgA</em> and <em>abgB</em> did not reveal any additional sequence differences. In the four strains analyzed, two nucleotide differences were found. <em>E. coli</em> BN1001, BN1002, and BN1003 each contained a C-to-A transversion at nucleotide position 984 from the <em>Sal</em>I site, while <em>E. coli</em> BN1005 had a C-to-T transition at position 1008. Since both of these alterations lay outside potential open reading frames, we searched for possible transcriptional regulatory sequences that might be affected by the mutational changes. In Fig. <a href="/pmc/articles/PMC107711/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2em;">Fig.4</span></span><span>4</span></a> we have noted two regions of dyad symmetry containing the half-site sequence -GATAA-, within which is contained the T-N<sub>11</sub>-A feature typical of LysR family protein-binding sites (<a href="#B11" rid="B11" class=" bibr popnode">11</a>), and a match to the catabolite activator protein consensus sequence (<a href="#B8" rid="B8" class=" bibr popnode">8</a>). The most obvious possibility from inspection of the nucleotide sequence is that the mutations may lie in the &#x02212;10 and &#x02212;35 regions of the putative <em>abgR</em> promoter. Although no direct evidence concerning the effects of the mutations on the expression of <em>abgR</em> has been found, we hypothesize that the mutations alter the expression of <em>abgT</em>, consistent with the observations made above that indicate that only <em>abgT</em> is required for <em>p</em>-aminobenzoyl-glutamate utilization.  </p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC107711/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140559514275520"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=107711_jb2380606004.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is jb2380606004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC107711/bin/jb2380606004.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140559514275520"><a target="object" rel="noopener" href="/pmc/articles/PMC107711/figure/F4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC107711/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">FIG. 4</a></div><!--caption a7--><div class="caption"><p>Nucleotide and amino acid sequences of the <em>abgR-abgA</em> intercistronic region. The positions of mutations resulting in the <em>p</em>-aminobenzoyl-glutamate utilization phenotype are shown in large bold letters above the sequence. Putative LysR-type protein-binding sequences are shown as dotted lines above the sequence. Putative &#x02212;10 and &#x02212;35 sequences of the <em>abgR</em> and <em>abgA</em> promoters are overlined or underlined. The CAP consensus sequence is also indicated above the sequence (lowercase letters indicate nucleotides that are not as strongly conserved as the capitalized nucleotides). Numbering is from the <em>Sal</em>I site prior to <em>abgR</em>.</p></div></div></div><p>For <em>E. coli</em> BN1004, additional evidence for the presence of a duplicated DNA segment suggested by Southern hybridization data was sought. The Southern hybridization data and the restriction map of the clone derived from BN1023 indicated that a 4.2-kb duplication that contained most or all of <em>abgB</em>, <em>abgT</em>, and <em>ogt</em> was present in the chromosome. To determine the extent and site of the duplicated segment, we devised a PCR-based experiment to localize the novel junction between the duplication endpoints (Fig. <a href="/pmc/articles/PMC107711/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.5)</span></span><span>5</span></a>) and then determined the sequence of the junction. The junction sequence was localized in PCR experiments with divergently oriented primer pairs. In the absence of a duplication region encompassing a primer pair, no PCR product would be detected. However, if a primer pair were located within a tandemly duplicated segment, one pair of primers would be oriented toward one another and a PCR product that contained the novel junction would be obtained. One primer (pri1) was designed to hybridize near the <em>Hin</em>dIII site within <em>ogt</em>, and four other primers (pri2 to pri5) were designed to hybridize at various sites within <em>abgA</em> and <em>abgB</em> (Fig. <a href="/pmc/articles/PMC107711/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.5).</span></span><span>5</span></a>). Primer pair pri1-pri2 yielded a product from DNA derived from <em>E. coli</em> BN1004 or BN1023 but not from DNA derived from <em>E. coli</em> BN101 or BN1001. Other primer pairs yielded no product from <em>E. coli</em> BN1004 or BN1023 DNA, suggesting that the duplication junction lay in the 200-bp region between pri2 and pri3.  </p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC107711/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140559514270080"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=107711_jb2380606005.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is jb2380606005.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC107711/bin/jb2380606005.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140559514270080"><a target="object" rel="noopener" href="/pmc/articles/PMC107711/figure/F5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC107711/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">FIG. 5</a></div><!--caption a7--><div class="caption"><p>Structures of the wild-type (a) and duplicated (b) regions of <em>E. coli</em> BN1004. The <em>Hin</em>dIII site within <em>ogt</em> is illustrated, as are the primer hybridization sites used to delimit the extent of duplication. The 12-bp direct repeat sequences within <em>abgA</em> and <em>fnr</em> are illustrated. It can be seen from panel b that only primer pair pri1-pri2 could yield a PCR product from DNA containing a duplication. The source of the 4.2-kb <em>Hin</em>dIII fragment containing <em>abgT</em> is also evident.</p></div></div></div><p class="p p-last">DNA amplified from <em>E. coli</em> BN1023 with pri1 and pri2 was sequenced with pri2 as a sequencing primer, yielding the sequence of the duplication junction: a 12-bp repeated sequence found once near the end of <em>abgA</em> and again within <em>fnr</em>. Duplication between these sequences resulted in a 4,157-bp duplication with a novel <em>fnr-abgA</em> fusion (in phase) and with <em>abgB</em> and <em>abgT</em> expression presumably under the control of the <em>fnr</em> promoter.</p></div><div id="sec-a.r.g.h" class="sec"><p></p><h4 id="sec-a.r.g.htitle" class="inline">Hydrolysis of <em>p</em>-aminobenzoyl-glutamate. </h4><p class="p p-first">The protein products of the two open reading frames upstream of <em>abgT</em> (here designated <em>abgA</em> and <em>abgB</em>) showed sequence similarity to one another and sequence similarity to a family of aminoacyl aminohydrolases. Outside the <em>E. coli</em> sequences, the greatest similarity was between AbgA and <em>Enterobacter agglomerans</em> indole-3-acetyl-aspartic acid hydrolase (<a href="#B6" rid="B6" class=" bibr popnode">6</a>), but of greater interest was the similarity of each to <em>Pseudomonas</em> sp. strain RS-16 carboxypeptidase G<sub>2</sub> (<a href="#B22" rid="B22" class=" bibr popnode">22</a>), an enzyme capable of cleaving folate to pteroate and glutamate and <em>p</em>-aminobenzoyl-glutamate to <em>p</em>-aminobenzoate and glutamate (<a href="#B33" rid="B33" class=" bibr popnode">33</a>). Based on these similarities, we hypothesized that <em>p</em>-aminobenzoyl-glutamate utilization conferred by transport of the supplement into the cell by AbgT might be accompanied by hydrolysis to <em>p</em>-aminobenzoate and glutamate by AbgA and AbgB, either alone or in combination.</p><p>Two lines of evidence indicated that <em>p</em>-aminobenzoyl-glutamate was hydrolyzed to <em>p</em>-aminobenzoate in the <em>abg</em> mutant strains. The first line of evidence was derived from the observation that <em>E. coli</em> BN1001 could cross-feed parental strain <em>E. coli</em> BN101 or BN1103 when the two strains were streaked near one another on solid medium containing <em>p</em>-aminobenzoyl-glutamate. Since BN101 and BN1103 required <em>p</em>-aminobenzoate for growth and could not utilize <em>p</em>-aminobenzoyl-glutamate, the gradient of growth observed in the parental strain was very likely due to the cleavage of <em>p</em>-aminobenzoyl-glutamate and the diffusion of <em>p</em>-aminobenzoate. Each of the mutant strains (BN1001 to BN1005 and BN1023) demonstrated the ability to cross-feed a <em>p</em>-aminobenzoate-requiring strain.</p><p class="p p-last">The second line of evidence came from the direct demonstration of the cleavage of <em>p</em>-aminobenzoyl-glutamate to <em>p</em>-aminobenzoate in vitro by crude extracts derived from mutant strains (Table <a href="/pmc/articles/PMC107711/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table4).</span></span><span>4</span></a>). Extracts prepared from <em>E. coli</em> BN1004 showed approximately fivefold-higher <em>p</em>-aminobenzoyl-glutamate hydrolysis activity than did those of parental strain <em>E. coli</em> BN101. Extracts prepared from BN1003 and BN1005, on the other hand, showed only a small (less than twofold) increase in activity.  </p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T4"><h3>TABLE 4</h3><!--caption a7--><div class="caption"><p>Enzymatic hydrolysis of <em>p</em>-aminobenzoyl-glutamate by crude&#x02009;extracts </p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Strain</th><th rowspan="1" colspan="1">Genotype</th><th rowspan="1" colspan="1">Activity (U/mg  of protein) </th></tr></thead><tbody align="center"><tr><td rowspan="1" colspan="1">BN101</td><td align="left" rowspan="1" colspan="1"><em>pabA</em></td><td rowspan="1" colspan="1">0.66&#x02009;&#x000b1;&#x02009;0.15<sup>a</sup></td></tr><tr><td rowspan="1" colspan="1">BN1003</td><td align="left" rowspan="1" colspan="1"><em>abg-1</em></td><td rowspan="1" colspan="1">0.72&#x02009;&#x000b1;&#x02009;0.08<sup>a</sup></td></tr><tr><td rowspan="1" colspan="1">BN1004</td><td align="left" rowspan="1" colspan="1"><em>abg-4</em> (duplication)</td><td rowspan="1" colspan="1">3.19&#x02009;&#x000b1;&#x02009;0.45<sup>a</sup></td></tr><tr><td rowspan="1" colspan="1">BN1005</td><td align="left" rowspan="1" colspan="1"><em>abg-5</em></td><td rowspan="1" colspan="1">1.01&#x02009;&#x000b1;&#x02009;0.10<sup>a</sup></td></tr><tr><td rowspan="1" colspan="1">BN1150</td><td align="left" rowspan="1" colspan="1"><em>abg-1</em></td><td rowspan="1" colspan="1">0.77&#x02009;&#x000b1;&#x02009;0.06<sup>b</sup></td></tr><tr><td rowspan="1" colspan="1">BN1151</td><td align="left" rowspan="1" colspan="1"><em>abg-1 abgB</em>::<em>kan</em></td><td rowspan="1" colspan="1">0.70&#x02009;&#x000b1;&#x02009;0.05<sup>b</sup></td></tr><tr><td rowspan="1" colspan="1">BN1152</td><td align="left" rowspan="1" colspan="1"><em>abg-1 abgA</em>::<em>kan</em></td><td rowspan="1" colspan="1">0.87&#x02009;&#x000b1;&#x02009;0.11<sup>b</sup></td></tr><tr><td rowspan="1" colspan="1">BN1153</td><td align="left" rowspan="1" colspan="1"><em>abg-1 abgT</em>::<em>kan</em></td><td rowspan="1" colspan="1">0.81&#x02009;&#x000b1;&#x02009;0.08<sup>b</sup></td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140559514267360"><a target="object" rel="noopener" href="/pmc/articles/PMC107711/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TF4-a"><sup>a</sup>Mean &#x000b1; average deviation from the mean for quadruplicate determinations of three independent cultures.&#x000a0;</div><div id="TF4-b"><sup>b</sup>Mean &#x000b1; average deviation from the mean for quadruplicate determinations of one culture.&#x000a0;</div></div></div></div><div id="sec-a.r.g.i" class="sec sec-last"><p></p><h4 id="sec-a.r.g.ititle" class="inline">Effects of interruptions in <em>abgA</em>, <em>abgB</em>, and <em>abgT.</em> </h4><p class="p p-first">The phenotypic effects of interruptions in <em>abgA</em>, <em>abgB</em>, and <em>abgT</em> were tested by constructing strains in which each open reading frame was interrupted with a kanamycin resistance cassette. The cassettes were inserted at the <em>Pst</em>I sites present in each of the genes. Recombination of the interrupted genes into an <em>abg</em><sup>&#x0002b;</sup> strain had no effect on the growth properties of the strains on supplemented minimal medium. However, the introduction of each interrupted gene into an <em>abg-1</em> host resulted in a reduction in the ability to utilize <em>p</em>-aminobenzoyl-glutamate effectively (Table <a href="/pmc/articles/PMC107711/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table5).</span></span><span>5</span></a>). As anticipated, the strains containing the <em>abgT</em>::<em>kan</em> interruption (BN1143 and BN1153), with the kanamycin resistance determinant transcribed in the same direction as <em>abg</em>, failed to grow on medium supplemented with <em>p</em>-aminobenzoyl-glutamate. Similarly, <em>abg-1</em> strains containing <em>abgA</em>::<em>kan</em> (BN1142 and BN1152) and <em>abgB</em>::<em>kan</em> (BN1141 and BN1151) also failed to utilize <em>p</em>-aminobenzoyl-glutamate. These data might be explained by polarity of the kanamycin resistance cassette for <em>abgT</em> expression, since the direction of <em>kan</em> transcription is opposite that of <em>abgT</em> transcription in the <em>abgA</em>::<em>kan</em> and <em>abgB</em>::<em>kan</em> interruptions.  </p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T5"><h3>TABLE 5</h3><!--caption a7--><div class="caption"><p>Minimum concentration of <em>p</em>-aminobenzoyl-glutamate required for full growth on solid medium for various  <em>abg</em>::<em>kan</em> interruption&#x02009;strains<sup>a</sup> </p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Strain</th><th rowspan="1" colspan="1"><em>abg</em> allele</th><th rowspan="1" colspan="1"><em>p</em>-Aminobenzoyl-glutamate concn (&#x003bc;M) required for growth</th></tr></thead><tbody align="center"><tr><td rowspan="1" colspan="1">BN1103</td><td align="left" rowspan="1" colspan="1"><em>abg<sup>&#x0002b;</sup></em></td><td rowspan="1" colspan="1">10</td></tr><tr><td rowspan="1" colspan="1">BN1140</td><td align="left" rowspan="1" colspan="1"><em>abg-1</em></td><td rowspan="1" colspan="1">1</td></tr><tr><td rowspan="1" colspan="1">BN1141</td><td align="left" rowspan="1" colspan="1"><em>abg-1 abgB</em>::<em>kan</em></td><td rowspan="1" colspan="1">10</td></tr><tr><td rowspan="1" colspan="1">BN1142</td><td align="left" rowspan="1" colspan="1"><em>abg-1 abgA</em>::<em>kan</em></td><td rowspan="1" colspan="1">10</td></tr><tr><td rowspan="1" colspan="1">BN1143</td><td align="left" rowspan="1" colspan="1"><em>abg-1 abgT</em>::<em>kan</em></td><td rowspan="1" colspan="1">10</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140559507611264"><a target="object" rel="noopener" href="/pmc/articles/PMC107711/table/T5/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TF5-a"><sup>a</sup>The <em>p</em>-aminobenzoyl-glutamate concentrations tested were 200, 100, 40, 20, 10, 4, 2, 1, 0.4, 0.2, 0.1, 0.04, 0.02, 0.01, 0.004, 0.002, and 0.001 &#x003bc;M. The strains carrying pMJH128 were used at 0.1 &#x003bc;M.&#x000a0;</div></div></div><p class="p p-last">To test whether polarity for <em>abgT</em> was the cause of the failure to utilize <em>p</em>-aminobenzoyl-glutamate, we transformed each of the <em>abg</em>::<em>kan</em> interruptions with pMJH128, a plasmid carrying <em>abgT</em>. In each case, the presence of <em>abgT</em> in <em>trans</em> not only restored the <em>p</em>-aminobenzoyl-glutamate utilization phenotype, indicating that the <em>abgAB</em> genes were not essential for <em>p</em>-aminobenzoyl-glutamate utilization, but also increased the sensitivity of the strains to low levels of <em>p</em>-aminobenzoyl-glutamate. The ability to grow on low levels of <em>p</em>-aminobenzoate was not altered. The <em>abg</em>::<em>kan</em> interruptions had no apparent effect on the <em>p</em>-aminobenzoyl-glutamate hydrolysis activity present in crude extracts (Table <a href="/pmc/articles/PMC107711/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table4</span></span><span>4</span></a>).</p></div></div><div id="Discussion" class="tsec sec"><h2 class="head no_bottom_margin" id="Discussiontitle" style="text-transform: uppercase;">DISCUSSION</h2><p class="p p-first"><em>E. coli pabA</em> strains able to utilize <em>p</em>-aminobenzoyl-glutamate were obtained following mutagenesis and selection on folate compounds. The <em>p</em>-aminobenzoyl-glutamate utilization phenotype depended on a single gene, <em>abgT</em>, a homolog of transport genes that apparently was responsible for the transport of <em>p</em>-aminobenzoyl-glutamate into the cell. <em>abg</em><sup>&#x0002b;</sup> strains that contained interrupted <em>abgT</em> did not yield <em>p</em>-aminobenzoyl-glutamate utilization derivatives following mutagenesis, and <em>abg-1</em> strains lost their ability to use <em>p</em>-aminobenzoyl-glutamate when <em>abgT</em> was disrupted. Plasmid-borne, wild-type <em>abgT</em> could confer <em>p</em>-aminobenzoyl-glutamate utilization on <em>abg</em><sup>&#x0002b;</sup> hosts, and this ability was lost when <em>abgT</em> was disrupted.</p><p>The nucleotide sequence of the <em>abgT</em> allele from an <em>abg-1</em> strain was identical to that of the wild-type allele, and the cloned wild-type gene was capable of conferring <em>p</em>-aminobenzoyl-glutamate utilization on <em>pabA</em> strains. These observations suggested that wild-type <em>abgT</em> was normally cryptic or not expressed under conditions of growth on minimal medium. Altering the expression of <em>abgT</em> from its wild-type context appears to have been the mechanism for obtaining the growth phenotype. Data from the <em>E. coli</em> genome sequence suggested that <em>abgT</em> lay in an operon with two other genes upstream and possibly also <em>ogt</em> downstream. Since no promoter-like sequences were observed near <em>abgT</em>, it seemed likely that the <em>abgT</em> regulatory signals lay several kilobases upstream, probably near the region of the divergently transcribed LysR-type regulatory gene.</p><p>Three different types of mutations were detected in the collection of five <em>abg</em> strains analyzed. Two different single-base-pair mutations in the <em>abgR-abgA</em> intergenic region resulted in an approximately 20-fold-increased ability to utilize <em>p</em>-aminobenzoyl-glutamate. The growth phenotype was accompanied by a small but reproducible increase in the expression of <em>p</em>-aminobenzoyl-glutamate hydrolysis activity detectable in crude extracts of one of the mutant strains (BN1005). The location of the mutations in or near the <em>abgR</em> promoter suggested that alteration of the expression of <em>abgR</em> may have altered the expression of <em>abgT</em>, which in turn resulted in the <em>p</em>-aminobenzoyl-glutamate utilization phenotype. BN1004 contained a 4.2-kb duplication of DNA which placed a second copy of <em>abgB</em> and <em>abgT</em> under the control of the <em>fnr</em> promoter. The anomalous expression of the <em>abg</em> genes resulted in a 200-fold increase in the ability to use <em>p</em>-aminobenzoyl-glutamate and a 5-fold increase in the amount of <em>p</em>-aminobenzoyl-glutamate hydrolase activity in crude extracts. The level of sensitivity to <em>p</em>-aminobenzoyl-glutamate was equivalent to that seen in strains containing a high-copy-number plasmid carrying <em>abgT</em>. Since <em>abgB</em> was duplicated and expressed from the <em>fnr</em> promoter, while <em>abgA</em> was not, we concluded that <em>abgB</em> encoded measurable <em>p</em>-aminobenzoyl-glutamate hydrolase activity but that <em>abgB</em> was not essential for the <em>p</em>-aminobenzoyl-glutamate utilization phenotype.</p><p>The utilization of <em>p</em>-aminobenzoyl-glutamate may have occurred by either of two mechanisms. First, DHPS may have used <em>p</em>-aminobenzoyl-glutamate as a substrate directly, forming dihydrofolate as the product and bypassing the normal dihydropteroate intermediate in the dihydrofolate biosynthetic pathway. While the direct in vivo incorporation of <em>p</em>-aminobenzoyl-glutamate into dihydrofolate has not been observed so far for <em>E. coli</em>, DHPS has been shown to be able to use <em>p</em>-aminobenzoyl-glutamate in place of <em>p</em>-aminobenzoate in vitro (<a href="#B29" rid="B29" class=" bibr popnode">29</a>), albeit with a higher <em>K</em><sub><em>m</em></sub> for <em>p</em>-aminobenzoyl-glutamate than for <em>p</em>-aminobenzoate. However, it is possible that increased transport of <em>p</em>-aminobenzoyl-glutamate into the cell by increased expression of <em>abgT</em> may have raised the intracellular level sufficiently to make the reaction proceed well enough to support growth.</p><p>A second possibility was that the transported <em>p</em>-aminobenzoyl-glutamate was first hydrolyzed to <em>p</em>-aminobenzoate and glutamate by intracellular aminoacyl aminohydrolases and then <em>p</em>-aminobenzoate was incorporated into dihydropteroate in the normal biosynthetic pathway. In support of this idea, we observed that <em>abg</em> strains growing on <em>p</em>-aminobenzoyl-glutamate had the ability to cross-feed the <em>p</em>-aminobenzoate requirement of the parental strain, suggesting hydrolysis of <em>p</em>-aminobenzoyl-glutamate and excretion of <em>p</em>-aminobenzoate. In addition, we demonstrated that crude extracts derived from <em>abg</em> strains had an elevated capacity to hydrolyze <em>p</em>-aminobenzoyl-glutamate to <em>p</em>-aminobenzoate in vitro. However, disruption of <em>abgA</em> and <em>abgB</em> did not reverse the <em>p</em>-aminobenzoyl-glutamate utilization phenotype in strains expressing <em>abgT</em>, indicating that if hydrolysis were required, other activities in the cell were capable of <em>p</em>-aminobenzoyl-glutamate hydrolysis.</p><p class="p p-last">The selection for growth on <em>p</em>-aminobenzoyl-glutamate appeared to have resulted in the altered regulation of an operon that resulted in an increased capacity to transport and hydrolyze <em>p</em>-aminobenzoyl-glutamate. It is likely that the <em>abg</em> operon is expressed under specific growth conditions, perhaps in response to an effector of <em>abgR</em>. In this regard, we note that sequences in the <em>abgR-abgA</em> intercistronic region include segments similar to binding sites for LysR-type regulators (<a href="#B11" rid="B11" class=" bibr popnode">11</a>) and for CAP (<a href="#B8" rid="B8" class=" bibr popnode">8</a>). Alternatively, it is possible that the <em>p</em>-aminobenzoyl-glutamate utilization phenotype arose from recruitment of a transport protein from an operon whose normal function may not be involved in <em>p</em>-aminobenzoyl-glutamate or folate metabolism. For example, the wild-type function of the operon may be a peptide permease and hydrolysis system regulated by the presence of peptides in the medium. If such a system had a broad substrate specificity, then it might be able to transport and hydrolyze <em>p</em>-aminobenzoyl-glutamate under conditions of altered regulation. As has been shown with other operons in <em>E. coli</em> and other bacteria, altered-regulation phenomena are a common first step in the evolution of new metabolic pathways in bacterial systems (<a href="#B13" rid="B13" class=" bibr popnode">13</a>).</p></div><div id="ack-a.s.b" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.s.btitle" style="text-transform: uppercase;">ACKNOWLEDGMENTS</h2><div class="sec"><p>We are indebted to John Roth, in whose laboratory this work was initiated. We thank Gordon Guay for technical assistance.</p><p>This work was supported by Public Health Service grants AI25106 and GM44199 from the National Institutes of Health.</p></div></div><div id="ref-list-a.s.c" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.s.ctitle" style="text-transform: uppercase;">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1">1. <span class="citation">Altschul S F, Gish W, Miller W, Myers E W, Lipman D J. Basic local alignment search tool. <span><span class="ref-journal">J Mol Biol. </span>1990;<span class="ref-vol">215</span>:403–410.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2231712" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Basic+local+alignment+search+tool&amp;author=S+F+Altschul&amp;author=W+Gish&amp;author=W+Miller&amp;author=E+W+Myers&amp;author=D+J+Lipman&amp;volume=215&amp;publication_year=1990&amp;pages=403-410&amp;pmid=2231712&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B2">2. <span class="citation">Baumstark B R, Spremulli L L, RajBhandary U L, Brown G M. Initiation of protein synthesis without formylation in a mutant of <em>Escherichia coli</em> that grows in the absence of tetrahydrofolate. <span><span class="ref-journal">J Bacteriol. </span>1977;<span class="ref-vol">129</span>:457–471.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC234945/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/318648" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=Initiation+of+protein+synthesis+without+formylation+in+a+mutant+of+Escherichia+coli+that+grows+in+the+absence+of+tetrahydrofolate&amp;author=B+R+Baumstark&amp;author=L+L+Spremulli&amp;author=U+L+RajBhandary&amp;author=G+M+Brown&amp;volume=129&amp;publication_year=1977&amp;pages=457-471&amp;pmid=318648&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B3">3. <span class="citation">Birnboim H, Doly J. A rapid alkaline extraction procedure for screening recombinant plasmid DNA. <span><span class="ref-journal">Nucleic Acids Res. </span>1979;<span class="ref-vol">7</span>:1513–1523.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC342324/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/388356" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=A+rapid+alkaline+extraction+procedure+for+screening+recombinant+plasmid+DNA&amp;author=H+Birnboim&amp;author=J+Doly&amp;volume=7&amp;publication_year=1979&amp;pages=1513-1523&amp;pmid=388356&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B4">4. <span class="citation">Blattner F R, Plunkett III G, Bloch C A, Perna N T, Burland V, Riley M, Collado-Vides J, Glasner J D, Rode C K, Mayhew G F, Gregor J, Davis N W, Kirkpatrick H A, Goeden M A, Rose D J, Mau B, Shao Y. The complete genome sequence of <em>Escherichia coli</em> K-12. <span><span class="ref-journal">Science. </span>1997;<span class="ref-vol">277</span>:1453–1474.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9278503" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+complete+genome+sequence+of+Escherichia+coli+K-12&amp;author=F+R+Blattner&amp;author=G+Plunkett+III&amp;author=C+A+Bloch&amp;author=N+T+Perna&amp;author=V+Burland&amp;volume=277&amp;publication_year=1997&amp;pages=1453-1474&amp;pmid=9278503&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B5">5. <span class="citation">Bradford M. A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. <span><span class="ref-journal">Anal Biochem. </span>1976;<span class="ref-vol">72</span>:248–254.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/942051" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Anal+Biochem&amp;title=A+rapid+and+sensitive+method+for+the+quantitation+of+microgram+quantities+of+protein+utilizing+the+principle+of+protein-dye+binding&amp;author=M+Bradford&amp;volume=72&amp;publication_year=1976&amp;pages=248-254&amp;pmid=942051&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B6">6. <span class="citation">Chou J C, Mulbry W W, Cohen J D. The gene for indole-3-acetyl-aspartic acid hydrolase from <em>Enterobacter agglomerans</em>: molecular cloning, nucleotide sequence, and expression in <em>Escherichia coli</em>. <span><span class="ref-journal">Mol Gen Genet. </span>1998;<span class="ref-vol">259</span>:172–178.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9747708" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Gen+Genet&amp;title=The+gene+for+indole-3-acetyl-aspartic+acid+hydrolase+from+Enterobacter+agglomerans:+molecular+cloning,+nucleotide+sequence,+and+expression+in+Escherichia+coli&amp;author=J+C+Chou&amp;author=W+W+Mulbry&amp;author=J+D+Cohen&amp;volume=259&amp;publication_year=1998&amp;pages=172-178&amp;pmid=9747708&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B7">7. <span class="citation">Davis R W, Botstein D, Roth J R.  <span class="ref-journal">Advanced bacterial genetics.</span> Cold Spring Harbor, N.Y: Cold Spring Harbor Laboratory Press; 1980.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Advanced+bacterial+genetics.&amp;author=R+W+Davis&amp;author=D+Botstein&amp;author=J+R+Roth&amp;publication_year=1980&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B8">8. <span class="citation">deCrombrugghe B, Busby S, Buc H. Cyclic AMP receptor protein: role in transcription activation. <span><span class="ref-journal">Science. </span>1984;<span class="ref-vol">224</span>:831–838.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/6372090" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Cyclic+AMP+receptor+protein:+role+in+transcription+activation&amp;author=B+deCrombrugghe&amp;author=S+Busby&amp;author=H+Buc&amp;volume=224&amp;publication_year=1984&amp;pages=831-838&amp;pmid=6372090&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B9">9. <span class="citation">Eisengerg D, Schwarz E, Komaromy M, Wall R. Analysis of membrane and surface protein sequences with the hydrophobic moment plot. <span><span class="ref-journal">J Mol Biol. </span>1984;<span class="ref-vol">179</span>:125–142.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/6502707" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Analysis+of+membrane+and+surface+protein+sequences+with+the+hydrophobic+moment+plot&amp;author=D+Eisengerg&amp;author=E+Schwarz&amp;author=M+Komaromy&amp;author=R+Wall&amp;volume=179&amp;publication_year=1984&amp;pages=125-142&amp;pmid=6502707&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B10">10. <span class="citation">Ganduri Y L, Sadda S R, Datta M W, Jambukeswaran R K, Datta P. TdcA, a transcriptional activator of the <em>tdcABC</em> operon of <em>Escherichia coli</em>, is a member of the LysR family of proteins. <span><span class="ref-journal">Mol Gen Genet. </span>1993;<span class="ref-vol">240</span>:395–402.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8413189" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Gen+Genet&amp;title=TdcA,+a+transcriptional+activator+of+the+tdcABC+operon+of+Escherichia+coli,+is+a+member+of+the+LysR+family+of+proteins&amp;author=Y+L+Ganduri&amp;author=S+R+Sadda&amp;author=M+W+Datta&amp;author=R+K+Jambukeswaran&amp;author=P+Datta&amp;volume=240&amp;publication_year=1993&amp;pages=395-402&amp;pmid=8413189&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B11">11. <span class="citation">Goethals K, Van Montagu M, Holsters M. Conserved motifs in a divergent <em>nod</em> box of <em>Azorhizobium caulinodans</em> ORS571 reveal a common structure in promoters regulated by LysR-type proteins. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1992;<span class="ref-vol">89</span>:1646–1650.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC48509/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/1542656" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Conserved+motifs+in+a+divergent+nod+box+of+Azorhizobium+caulinodans+ORS571+reveal+a+common+structure+in+promoters+regulated+by+LysR-type+proteins&amp;author=K+Goethals&amp;author=M+Van+Montagu&amp;author=M+Holsters&amp;volume=89&amp;publication_year=1992&amp;pages=1646-1650&amp;pmid=1542656&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B12">12. <span class="citation">Green J M, Nichols B P. <em>p</em>-Aminobenzoate biosynthesis in <em>Escherichia coli</em>: purification of aminodeoxychorismate lyase and cloning of <em>pabC</em>. <span><span class="ref-journal">J Biol Chem. </span>1991;<span class="ref-vol">266</span>:12971–12975.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2071583" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=p-Aminobenzoate+biosynthesis+in+Escherichia+coli:+purification+of+aminodeoxychorismate+lyase+and+cloning+of+pabC&amp;author=J+M+Green&amp;author=B+P+Nichols&amp;volume=266&amp;publication_year=1991&amp;pages=12971-12975&amp;pmid=2071583&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B13">13. <span class="citation">Hall B G. Selection, adaptation, and bacterial operons. <span><span class="ref-journal">Genome. </span>1989;<span class="ref-vol">31</span>:265–271.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2687097" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome&amp;title=Selection,+adaptation,+and+bacterial+operons&amp;author=B+G+Hall&amp;volume=31&amp;publication_year=1989&amp;pages=265-271&amp;pmid=2687097&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B14">14. <span class="citation">Harvey R J. Growth and initiation of protein synthesis in <em>Escherichia coli</em> in the presence of trimethoprim. <span><span class="ref-journal">J Bacteriol. </span>1973;<span class="ref-vol">114</span>:309–322.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC251769/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/4572717" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=Growth+and+initiation+of+protein+synthesis+in+Escherichia+coli+in+the+presence+of+trimethoprim&amp;author=R+J+Harvey&amp;volume=114&amp;publication_year=1973&amp;pages=309-322&amp;pmid=4572717&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B15">15. <span class="citation">Henikoff S. Unidirectional digestion with exonuclease III in DNA sequence analysis. <span><span class="ref-journal">Methods Enzymol. </span>1987;<span class="ref-vol">155</span>:156–165.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3323819" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Enzymol&amp;title=Unidirectional+digestion+with+exonuclease+III+in+DNA+sequence+analysis&amp;author=S+Henikoff&amp;volume=155&amp;publication_year=1987&amp;pages=156-165&amp;pmid=3323819&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B16">16. <span class="citation">Huang M, Gibson F. Biosynthesis of 4-aminobenzoate in <em>Escherichia coli</em>. <span><span class="ref-journal">J Bacteriol. </span>1970;<span class="ref-vol">102</span>:767–773.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC247625/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/4914080" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=Biosynthesis+of+4-aminobenzoate+in+Escherichia+coli&amp;author=M+Huang&amp;author=F+Gibson&amp;volume=102&amp;publication_year=1970&amp;pages=767-773&amp;pmid=4914080&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B17">17. <span class="citation">Huang M, Pittard J. Genetic analysis of mutant strains of <em>Escherichia coli</em> requiring <em>p</em>-aminobenzoate for growth. <span><span class="ref-journal">J Bacteriol. </span>1967;<span class="ref-vol">93</span>:1938–1942.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC276713/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/5337773" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=Genetic+analysis+of+mutant+strains+of+Escherichia+coli+requiring+p-aminobenzoate+for+growth&amp;author=M+Huang&amp;author=J+Pittard&amp;volume=93&amp;publication_year=1967&amp;pages=1938-1942&amp;pmid=5337773&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B18">18. <span class="citation">Huennekens F M, Vitols K S, Pope L E, Fan J. Membrane transport of folate compounds. <span><span class="ref-journal">J Nutr Sci Vitaminol (Tokyo) </span>1992;<span class="ref-vol">38</span>:52–57.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/1297801" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Nutr+Sci+Vitaminol+(Tokyo)&amp;title=Membrane+transport+of+folate+compounds&amp;author=F+M+Huennekens&amp;author=K+S+Vitols&amp;author=L+E+Pope&amp;author=J+Fan&amp;volume=38&amp;publication_year=1992&amp;pages=52-57&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B19">19. <span class="citation">Klein P, Kanehisa M, Delisi C. The detection and classification of membrane-spanning proteins. <span><span class="ref-journal">Biochim Biophys Acta. </span>1985;<span class="ref-vol">815</span>:468–476.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3838905" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochim+Biophys+Acta&amp;title=The+detection+and+classification+of+membrane-spanning+proteins&amp;author=P+Klein&amp;author=M+Kanehisa&amp;author=C+Delisi&amp;volume=815&amp;publication_year=1985&amp;pages=468-476&amp;pmid=3838905&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B20">20. <span class="citation">Kohara Y, Akiyama K, Inoso K. The physical map of the whole <em>E. coli</em> chromosome: application of a new strategy for rapid analysis and sorting of a large genomic library. <span><span class="ref-journal">Cell. </span>1987;<span class="ref-vol">50</span>:495–508.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3038334" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=The+physical+map+of+the+whole+E.+coli+chromosome:+application+of+a+new+strategy+for+rapid+analysis+and+sorting+of+a+large+genomic+library&amp;author=Y+Kohara&amp;author=K+Akiyama&amp;author=K+Inoso&amp;volume=50&amp;publication_year=1987&amp;pages=495-508&amp;pmid=3038334&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B21">21. <span class="citation">Miller J H.  <span class="ref-journal">Experiments in molecular genetics.</span> Cold Spring Harbor, N.Y: Cold Spring Harbor Laboratory Press; 1972.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Experiments+in+molecular+genetics.&amp;author=J+H+Miller&amp;publication_year=1972&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B22">22. <span class="citation">Minton N P, Atkinson T, Bruton C J, Sherwood R F. The complete nucleotide sequence of the <em>Pseudomonas</em> gene coding for carboxypeptidase G<sub>2</sub>. <span><span class="ref-journal">Gene. </span>1984;<span class="ref-vol">31</span>:31–38.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/6396165" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=The+complete+nucleotide+sequence+of+the+Pseudomonas+gene+coding+for+carboxypeptidase+G2&amp;author=N+P+Minton&amp;author=T+Atkinson&amp;author=C+J+Bruton&amp;author=R+F+Sherwood&amp;volume=31&amp;publication_year=1984&amp;pages=31-38&amp;pmid=6396165&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B23">23. <span class="citation">Nichols B P, Seibold A, Doktor S Z. <em>para</em>-Aminobenzoate synthesis from chorismate occurs in two steps. <span><span class="ref-journal">J Biol Chem. </span>1989;<span class="ref-vol">264</span>:8597–8601.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2656685" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=para-Aminobenzoate+synthesis+from+chorismate+occurs+in+two+steps&amp;author=B+P+Nichols&amp;author=A+Seibold&amp;author=S+Z+Doktor&amp;volume=264&amp;publication_year=1989&amp;pages=8597-8601&amp;pmid=2656685&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B24">24. <span class="citation">Nickerson W J, Webb M. Effect of folic acid analogues on growth and cell division of nonexacting microorganisms. <span><span class="ref-journal">J Bacteriol. </span>1956;<span class="ref-vol">71</span>:129–139.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC357755/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/13295184" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=Effect+of+folic+acid+analogues+on+growth+and+cell+division+of+nonexacting+microorganisms&amp;author=W+J+Nickerson&amp;author=M+Webb&amp;volume=71&amp;publication_year=1956&amp;pages=129-139&amp;pmid=13295184&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B25">25. <span class="citation">Posner B A, Li L, Bethell R, Tsuji T, Benkovic S J. Engineering specificity for folate into dihydrofolate reductase from <em>Escherichia coli</em>. <span><span class="ref-journal">Biochemistry. </span>1996;<span class="ref-vol">35</span>:1653–1663.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8634297" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Engineering+specificity+for+folate+into+dihydrofolate+reductase+from+Escherichia+coli&amp;author=B+A+Posner&amp;author=L+Li&amp;author=R+Bethell&amp;author=T+Tsuji&amp;author=S+J+Benkovic&amp;volume=35&amp;publication_year=1996&amp;pages=1653-1663&amp;pmid=8634297&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B26">26. <span class="citation">Potter P M, Wilkinson M C, Fitton J, Carr F J, Brennard J, Cooper D P, Margison G P. Characterization and nucleotide sequence of <em>ogt</em>, the O<sup>6</sup>-alkyl-guanine-DNA alkyltransferase gene of <em>E. coli</em>. <span><span class="ref-journal">Nucleic Acids Res. </span>1987;<span class="ref-vol">15</span>:9177–9193.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC306461/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2825131" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Characterization+and+nucleotide+sequence+of+ogt,+the+O6-alkyl-guanine-DNA+alkyltransferase+gene+of+E.+coli&amp;author=P+M+Potter&amp;author=M+C+Wilkinson&amp;author=J+Fitton&amp;author=F+J+Carr&amp;author=J+Brennard&amp;volume=15&amp;publication_year=1987&amp;pages=9177-9193&amp;pmid=2825131&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B27">27. <span class="citation">Rao M J K, Argos P. A conformational preference parameter to predict helices in integral membrane proteins. <span><span class="ref-journal">Biochim Biophys Acta. </span>1986;<span class="ref-vol">869</span>:197–214.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2935194" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochim+Biophys+Acta&amp;title=A+conformational+preference+parameter+to+predict+helices+in+integral+membrane+proteins&amp;author=M+J+K+Rao&amp;author=P+Argos&amp;volume=869&amp;publication_year=1986&amp;pages=197-214&amp;pmid=2935194&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B28">28. <span class="citation">Richey D P, Brown G M. The biosynthesis of folic acid. IX. Purification and properties of the enzymes required for the formation of dihydropteroic acid. <span><span class="ref-journal">J Biol Chem. </span>1969;<span class="ref-vol">244</span>:1582–1592.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/4304228" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=The+biosynthesis+of+folic+acid.+IX.+Purification+and+properties+of+the+enzymes+required+for+the+formation+of+dihydropteroic+acid&amp;author=D+P+Richey&amp;author=G+M+Brown&amp;volume=244&amp;publication_year=1969&amp;pages=1582-1592&amp;pmid=4304228&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B29">29. <span class="citation">Richey D P, Brown G. A comparison of the effectiveness with which <em>p</em>-aminobenzoic acid and <em>p</em>-aminobenzoyl-glutamate acid are used as substrates by dihydropteroate synthase from <em>Escherichia coli</em>. <span><span class="ref-journal">Biochim Biophys Acta. </span>1970;<span class="ref-vol">222</span>:237–239.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/4919883" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochim+Biophys+Acta&amp;title=A+comparison+of+the+effectiveness+with+which+p-aminobenzoic+acid+and+p-aminobenzoyl-glutamate+acid+are+used+as+substrates+by+dihydropteroate+synthase+from+Escherichia+coli&amp;author=D+P+Richey&amp;author=G+Brown&amp;volume=222&amp;publication_year=1970&amp;pages=237-239&amp;pmid=4919883&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B30">30. <span class="citation">Sambrook J, Fritsch E F, Maniatis T.  <span class="ref-journal">Molecular cloning: a laboratory manual.</span> Cold Spring Harbor, N.Y: Cold Spring Harbor Laboratory Press; 1989.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Molecular+cloning:+a+laboratory+manual.&amp;author=J+Sambrook&amp;author=E+F+Fritsch&amp;author=T+Maniatis&amp;publication_year=1989&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B31">31. <span class="citation">Schweizer H P, Datta P. The complete nucleotide sequence of the <em>tdc</em> region of <em>Escherichia coli</em>. <span><span class="ref-journal">Nucleic Acids Res. </span>1989;<span class="ref-vol">17</span>:3994.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC317892/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2660107" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=The+complete+nucleotide+sequence+of+the+tdc+region+of+Escherichia+coli&amp;author=H+P+Schweizer&amp;author=P+Datta&amp;volume=17&amp;publication_year=1989&amp;pages=3994&amp;pmid=2660107&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B32">32. <span class="citation">Scott J M. Thin-layer chromatography of pteroylglutamates and related compounds. <span><span class="ref-journal">Methods Enzymol. </span>1980;<span class="ref-vol">66</span>:437–443.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7374484" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Enzymol&amp;title=Thin-layer+chromatography+of+pteroylglutamates+and+related+compounds&amp;author=J+M+Scott&amp;volume=66&amp;publication_year=1980&amp;pages=437-443&amp;pmid=7374484&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B33">33. <span class="citation">Sherwood R F, Melton R G, Alwan S M, Hughes P. Purification and properties of carboxypeptidase G<sub>2</sub> from <em>Pseudomonas</em> sp. strain RS-16. <span><span class="ref-journal">Eur J Biochem. </span>1985;<span class="ref-vol">148</span>:447–453.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3838935" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Biochem&amp;title=Purification+and+properties+of+carboxypeptidase+G2+from+Pseudomonas+sp.+strain+RS-16&amp;author=R+F+Sherwood&amp;author=R+G+Melton&amp;author=S+M+Alwan&amp;author=P+Hughes&amp;volume=148&amp;publication_year=1985&amp;pages=447-453&amp;pmid=3838935&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B34">34. <span class="citation">Shiota T, Disraely M N, McCann M P. The enzymatic synthesis of folate-like compounds from hydroxymethyldihydropteridine pyrophosphate. <span><span class="ref-journal">J Biol Chem. </span>1964;<span class="ref-vol">239</span>:2259–2266.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14209957" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=The+enzymatic+synthesis+of+folate-like+compounds+from+hydroxymethyldihydropteridine+pyrophosphate&amp;author=T+Shiota&amp;author=M+N+Disraely&amp;author=M+P+McCann&amp;volume=239&amp;publication_year=1964&amp;pages=2259-2266&amp;pmid=14209957&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B35">35. <span class="citation">Singer M, Baker T A, Schnitzler G, Deischel S M, Goel M, Dove W, Jaacks K J, Grossman A D, Erickson J W, Gross C A. Collection of strains containing genetically linked alternating antibiotic resistance elements for genetic mapping of <em>Escherichia coli</em>. <span><span class="ref-journal">Microbiol Rev. </span>1989;<span class="ref-vol">53</span>:1–24.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC372715/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2540407" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol+Rev&amp;title=Collection+of+strains+containing+genetically+linked+alternating+antibiotic+resistance+elements+for+genetic+mapping+of+Escherichia+coli&amp;author=M+Singer&amp;author=T+A+Baker&amp;author=G+Schnitzler&amp;author=S+M+Deischel&amp;author=M+Goel&amp;volume=53&amp;publication_year=1989&amp;pages=1-24&amp;pmid=2540407&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B36">36. <span class="citation">Tran P V, Bannor T A, Doktor S Z, Nichols B P. Chromosomal organization and expression of <em>Escherichia coli pabA</em>. <span><span class="ref-journal">J Bacteriol. </span>1990;<span class="ref-vol">172</span>:397–410.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC208445/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2403545" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=Chromosomal+organization+and+expression+of+Escherichia+coli+pabA&amp;author=P+V+Tran&amp;author=T+A+Bannor&amp;author=S+Z+Doktor&amp;author=B+P+Nichols&amp;volume=172&amp;publication_year=1990&amp;pages=397-410&amp;pmid=2403545&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B37">37. <span class="citation">Webb M, Nickerson W J. Differential reversal of inhibitory effects of folic acid analogues on growth, division, and deoxyribonucleic acid synthesis of microorganisms. <span><span class="ref-journal">J Bacteriol. </span>1956;<span class="ref-vol">71</span>:140–148.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC357756/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/13295185" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=Differential+reversal+of+inhibitory+effects+of+folic+acid+analogues+on+growth,+division,+and+deoxyribonucleic+acid+synthesis+of+microorganisms&amp;author=M+Webb&amp;author=W+J+Nickerson&amp;volume=71&amp;publication_year=1956&amp;pages=140-148&amp;pmid=13295185&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B38">38. <span class="citation">Wilson K.  Preparation of genomic DNA from bacteria. In: Ausubel F M, Brent R, Kingston R E, Moore D D, Seidman J G, Smith J A, Struhl K, editors. <span class="ref-journal">Current protocols in molecular biology.</span> New York, N.Y: Greene Publishing and Wiley-Interscience; 1987. pp. 2.4.1–2.4.5. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Current+protocols+in+molecular+biology.&amp;author=K+Wilson&amp;publication_year=1987&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B39">39. <span class="citation">Yim J J, Brown G M. Characteristics of guanosine triphosphate cyclohydrolase I purified from <em>Escherichia coli</em>. <span><span class="ref-journal">J Biol Chem. </span>1976;<span class="ref-vol">251</span>:5087–5094.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/821948" ref="reftype=pubmed&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Characteristics+of+guanosine+triphosphate+cyclohydrolase+I+purified+from+Escherichia+coli&amp;author=J+J+Yim&amp;author=G+M+Brown&amp;volume=251&amp;publication_year=1976&amp;pages=5087-5094&amp;pmid=821948&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=107711&amp;issue-id=2962&amp;journal-id=88&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Journal of Bacteriology</span> are provided here courtesy of <strong>American Society for Microbiology (ASM)</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC107711/pdf/jb006260.pdf" class="int-view">PDF (186K)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/107711/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/107711/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="107711" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC107711%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="BDDQv51HIZ3WlprHQzCpZwfY0bvVfMMoEn5JbT0UtsFeptonreld7SBLXjq512s0">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC107711%2F&amp;text=Characterization%20of%20Mutations%20That%20Allow%20p-Aminobenzoyl-Glutamate%20Utilization%20by%20Escherichia%20coli" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC107711%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC107711/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC107711/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/9829935/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC107711/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/9829935/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC107711/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/107711/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC107711/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
