
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs3.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>A proteomic classifier panel for early screening of colorectal cancer: a case control study - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="939BE807DB0F4BC500004B796F13C837.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="transmed" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC10880210/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10880210/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Journal of Translational Medicine" /><meta name="citation_title" content="A proteomic classifier panel for early screening of colorectal cancer: a case control study" /><meta name="citation_author" content="Hanju Hua" /><meta name="citation_author_institution" content="Department of Colorectal Surgery (H.H), and Department of Gastroenterology (C.Y. and Z.S.), College of Medicine, The First Affiliated Hospital, Zhejiang University, Hangzhou, 310006 Zhejiang China" /><meta name="citation_author" content="Tingting Wang" /><meta name="citation_author_institution" content="Durbrain Medical Laboratory, Hangzhou, 310000 Zhejiang China" /><meta name="citation_author" content="Liangxuan Pan" /><meta name="citation_author_institution" content="Durbrain Medical Laboratory, Hangzhou, 310000 Zhejiang China" /><meta name="citation_author" content="Xiaoyao Du" /><meta name="citation_author_institution" content="Durbrain Medical Laboratory, Hangzhou, 310000 Zhejiang China" /><meta name="citation_author" content="Tianxue Xia" /><meta name="citation_author_institution" content="Department of Colorectal Surgery (H.H), and Department of Gastroenterology (C.Y. and Z.S.), College of Medicine, The First Affiliated Hospital, Zhejiang University, Hangzhou, 310006 Zhejiang China" /><meta name="citation_author" content="Zhenzhong Fa" /><meta name="citation_author_institution" content="Changzhou Wujin People&#x02019;s Hospital, Changzhou, 213000 Jiangsu China" /><meta name="citation_author" content="Lei Gu" /><meta name="citation_author_institution" content="Department of General Surgery, School of Medicine, Shanghai Tenth People&#x02019;s Hospital, Tongji University, Shanghai, 200072 China" /><meta name="citation_author" content="Fei Gao" /><meta name="citation_author_institution" content="Durbrain Medical Laboratory, Hangzhou, 310000 Zhejiang China" /><meta name="citation_author" content="Chaohui Yu" /><meta name="citation_author_institution" content="Department of Colorectal Surgery (H.H), and Department of Gastroenterology (C.Y. and Z.S.), College of Medicine, The First Affiliated Hospital, Zhejiang University, Hangzhou, 310006 Zhejiang China" /><meta name="citation_author" content="Feng Gao" /><meta name="citation_author_institution" content="Changzhou Wujin People&#x02019;s Hospital, Changzhou, 213000 Jiangsu China" /><meta name="citation_author" content="Lujian Liao" /><meta name="citation_author_institution" content="Durbrain Medical Laboratory, Hangzhou, 310000 Zhejiang China" /><meta name="citation_author_institution" content="Shanghai Key Laboratory of Regulatory Biology, School of Life Sciences, East China Normal University, Shanghai, 200241 China" /><meta name="citation_author" content="Zhe Shen" /><meta name="citation_author_institution" content="Department of Colorectal Surgery (H.H), and Department of Gastroenterology (C.Y. and Z.S.), College of Medicine, The First Affiliated Hospital, Zhejiang University, Hangzhou, 310006 Zhejiang China" /><meta name="citation_publication_date" content="2024" /><meta name="citation_volume" content="22" /><meta name="citation_doi" content="10.1186/s12967-024-04983-5" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC10880210/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC10880210/" /><meta name="citation_pmid" content="38383428" /><meta name="DC.Title" content="A proteomic classifier panel for early screening of colorectal cancer: a case control study" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="BMC" /><meta name="DC.Contributor" content="Hanju Hua" /><meta name="DC.Contributor" content="Tingting Wang" /><meta name="DC.Contributor" content="Liangxuan Pan" /><meta name="DC.Contributor" content="Xiaoyao Du" /><meta name="DC.Contributor" content="Tianxue Xia" /><meta name="DC.Contributor" content="Zhenzhong Fa" /><meta name="DC.Contributor" content="Lei Gu" /><meta name="DC.Contributor" content="Fei Gao" /><meta name="DC.Contributor" content="Chaohui Yu" /><meta name="DC.Contributor" content="Feng Gao" /><meta name="DC.Contributor" content="Lujian Liao" /><meta name="DC.Contributor" content="Zhe Shen" /><meta name="DC.Date" content="2024" /><meta name="DC.Identifier" content="10.1186/s12967-024-04983-5" /><meta name="DC.Language" content="en" /><meta property="og:title" content="A proteomic classifier panel for early screening of colorectal cancer: a case control study" /><meta property="og:type" content="article" /><meta property="og:description" content="Diagnosis of colorectal cancer (CRC) during early stages can greatly improve patient outcome. Although technical advances in the field of genomics and proteomics have identified a number of candidate biomarkers for non-invasive screening and diagnosis, ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10880210/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10880210/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/214/">J Transl Med</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/452086/">v.22; 2024</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC10880210
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC10880210/pdf/12967_2024_Article_4983.pdf" class="int-view">PDF (2.5M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/10880210/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/10880210/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="10880210" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10880210%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="kF35weXXp5bjOt1eOM8n458bnS7aPVBZSpDrBR6ZIP9ty2EManj6AdHdcYglE0D0">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10880210%2F&amp;text=A%20proteomic%20classifier%20panel%20for%20early%20screening%20of%20colorectal%20cancer%3A%20a%20case%20control%20study" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10880210%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10880210/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC10880210/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38383428/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC10880210/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38383428/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC10880210/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/10880210/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/214/">J Transl Med</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/452086/">v.22; 2024</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC10880210
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-transmed.png" alt="Logo of transmed" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="https://translational-medicine.biomedcentral.com/" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">J Transl Med.</span> 2024; 22: 188. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2024 Feb 21. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1186%2Fs12967-024-04983-5" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1186/s12967-024-04983-5</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC10880210</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38383428">38383428</a></div></div></div><h1 class="content-title">A proteomic classifier panel for early screening of colorectal cancer: a case control study</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hua%20H%5BAuthor%5D" class="affpopup" co-rid="_co_idm139958984503824" co-class="co-affbox">Hanju Hua</a>,<sup>#</sup><sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wang%20T%5BAuthor%5D" class="affpopup" co-rid="_co_idm139958984245408" co-class="co-affbox">Tingting Wang</a>,<sup>#</sup><sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Pan%20L%5BAuthor%5D" class="affpopup" co-rid="_co_idm139958989738320" co-class="co-affbox">Liangxuan Pan</a>,<sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Du%20X%5BAuthor%5D" class="affpopup" co-rid="_co_idm139958989736560" co-class="co-affbox">Xiaoyao Du</a>,<sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Xia%20T%5BAuthor%5D" class="affpopup" co-rid="_co_idm139958986806624" co-class="co-affbox">Tianxue Xia</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Fa%20Z%5BAuthor%5D" class="affpopup" co-rid="_co_idm139958986804640" co-class="co-affbox">Zhenzhong Fa</a>,<sup>3</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gu%20L%5BAuthor%5D" class="affpopup" co-rid="_co_idm139958981137440" co-class="co-affbox">Lei Gu</a>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gao%20F%5BAuthor%5D" class="affpopup" co-rid="_co_idm139958986157984" co-class="co-affbox">Fei Gao</a>,<sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Yu%20C%5BAuthor%5D" class="affpopup" co-rid="_co_idm139958986156000" co-class="co-affbox">Chaohui Yu</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gao%20F%5BAuthor%5D" class="affpopup" co-rid="_co_idm139958974183072" co-class="co-affbox">Feng Gao</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>3</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Liao%20L%5BAuthor%5D" class="affpopup" co-rid="_co_idm139958986421408" co-class="co-affbox">Lujian Liao</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>2,</sup><sup>5</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shen%20Z%5BAuthor%5D" class="affpopup" co-rid="_co_idm139958990543936" co-class="co-affbox">Zhe Shen</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139958984503824"><h3 class="no_margin">Hanju Hua</h3><p><sup>1</sup>Department of Colorectal Surgery (H.H), and Department of Gastroenterology (C.Y. and Z.S.), College of Medicine, The First Affiliated Hospital, Zhejiang University, Hangzhou, 310006 Zhejiang China </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hua%20H%5BAuthor%5D">Hanju Hua</a></div></div><div id="_co_idm139958984245408"><h3 class="no_margin">Tingting Wang</h3><p><sup>2</sup>Durbrain Medical Laboratory, Hangzhou, 310000 Zhejiang China </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wang%20T%5BAuthor%5D">Tingting Wang</a></div></div><div id="_co_idm139958989738320"><h3 class="no_margin">Liangxuan Pan</h3><p><sup>2</sup>Durbrain Medical Laboratory, Hangzhou, 310000 Zhejiang China </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Pan%20L%5BAuthor%5D">Liangxuan Pan</a></div></div><div id="_co_idm139958989736560"><h3 class="no_margin">Xiaoyao Du</h3><p><sup>2</sup>Durbrain Medical Laboratory, Hangzhou, 310000 Zhejiang China </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Du%20X%5BAuthor%5D">Xiaoyao Du</a></div></div><div id="_co_idm139958986806624"><h3 class="no_margin">Tianxue Xia</h3><p><sup>1</sup>Department of Colorectal Surgery (H.H), and Department of Gastroenterology (C.Y. and Z.S.), College of Medicine, The First Affiliated Hospital, Zhejiang University, Hangzhou, 310006 Zhejiang China </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Xia%20T%5BAuthor%5D">Tianxue Xia</a></div></div><div id="_co_idm139958986804640"><h3 class="no_margin">Zhenzhong Fa</h3><p><sup>3</sup>Changzhou Wujin People&#x02019;s Hospital, Changzhou, 213000 Jiangsu China </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Fa%20Z%5BAuthor%5D">Zhenzhong Fa</a></div></div><div id="_co_idm139958981137440"><h3 class="no_margin">Lei Gu</h3><p><sup>4</sup>Department of General Surgery, School of Medicine, Shanghai Tenth People&#x02019;s Hospital, Tongji University, Shanghai, 200072 China </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gu%20L%5BAuthor%5D">Lei Gu</a></div></div><div id="_co_idm139958986157984"><h3 class="no_margin">Fei Gao</h3><p><sup>2</sup>Durbrain Medical Laboratory, Hangzhou, 310000 Zhejiang China </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gao%20F%5BAuthor%5D">Fei Gao</a></div></div><div id="_co_idm139958986156000"><h3 class="no_margin">Chaohui Yu</h3><p><sup>1</sup>Department of Colorectal Surgery (H.H), and Department of Gastroenterology (C.Y. and Z.S.), College of Medicine, The First Affiliated Hospital, Zhejiang University, Hangzhou, 310006 Zhejiang China </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Yu%20C%5BAuthor%5D">Chaohui Yu</a></div></div><div id="_co_idm139958974183072"><h3 class="no_margin">Feng Gao</h3><p><sup>3</sup>Changzhou Wujin People&#x02019;s Hospital, Changzhou, 213000 Jiangsu China </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gao%20F%5BAuthor%5D">Feng Gao</a></div></div><div id="_co_idm139958986421408"><h3 class="no_margin">Lujian Liao</h3><p><sup>2</sup>Durbrain Medical Laboratory, Hangzhou, 310000 Zhejiang China </p><p><sup>5</sup>Shanghai Key Laboratory of Regulatory Biology, School of Life Sciences, East China Normal University, Shanghai, 200241 China </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Liao%20L%5BAuthor%5D">Lujian Liao</a></div></div><div id="_co_idm139958990543936"><h3 class="no_margin">Zhe Shen</h3><p><sup>1</sup>Department of Colorectal Surgery (H.H), and Department of Gastroenterology (C.Y. and Z.S.), College of Medicine, The First Affiliated Hospital, Zhejiang University, Hangzhou, 310006 Zhejiang China </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shen%20Z%5BAuthor%5D">Zhe Shen</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm139958984008352_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139958984008352_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139958984008352_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm139958984008352_ai" style="display:none"><div class="fm-affl" id="Aff1"><sup>1</sup>Department of Colorectal Surgery (H.H), and Department of Gastroenterology (C.Y. and Z.S.), College of Medicine, The First Affiliated Hospital, Zhejiang University, Hangzhou, 310006 Zhejiang China </div><div class="fm-affl" id="Aff2"><sup>2</sup>Durbrain Medical Laboratory, Hangzhou, 310000 Zhejiang China </div><div class="fm-affl" id="Aff3"><sup>3</sup>Changzhou Wujin People&#x02019;s Hospital, Changzhou, 213000 Jiangsu China </div><div class="fm-affl" id="Aff4"><sup>4</sup>Department of General Surgery, School of Medicine, Shanghai Tenth People&#x02019;s Hospital, Tongji University, Shanghai, 200072 China </div><div class="fm-affl" id="Aff5"><sup>5</sup>Shanghai Key Laboratory of Regulatory Biology, School of Life Sciences, East China Normal University, Shanghai, 200241 China </div><div><span class="fm-affl">Chaohui Yu, </span><span class="fm-affl"><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="nc.ude.ujz@hcyyyyz" class="oemail">nc.ude.ujz@hcyyyyz</a></span>.</div><div><a href="#article-aaff-info">Contributor Information</a>.</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup>Corresponding author.</div><div><sup>#</sup>Contributed equally.</div></div><div class="fm-article-notes hide half_rhythm" id="idm139958984008352_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2023 Oct 24; Accepted 2024 Feb 12.</div></div><div class="permissions half_rhythm hide" id="idm139958984008352_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; The Author(s) 2024</div><div class="license half_rhythm"><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>. The Creative Commons Public Domain Dedication waiver (<a href="https://creativecommons.org/publicdomain/zero/1.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated in a credit line to the data.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="10" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Additional file 1: Figure S1. </strong>Quality of the DDA and DIA data. (A) Correlation analysis of the quality control (QC) samples from DDA data. (B) Correlation analysis of the quality control (QC) samples from DIA data. (C) Distribution of protein abundance of all quantified proteins from DDA data. (D) Distribution of protein abundance of all quantified proteins from DIA data. (E) Number of quantified proteins in all samples from DDA data. (F) Number of quantified proteins in all samples from DIA data.</div><div><a href="/pmc/articles/PMC10880210/bin/12967_2024_4983_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl">12967_2024_4983_MOESM1_ESM.pdf</a><span style="color:gray"> (940K)</span></div><div class="small guid">GUID: 4AEAA9F2-1417-4F0F-8422-A753E968F495</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Additional file 2: Figure S2. </strong>Heatmap showing KEGG pathways of up- and down-regulated proteins in the plasma from CRC patients.</div><div><a href="/pmc/articles/PMC10880210/bin/12967_2024_4983_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl">12967_2024_4983_MOESM2_ESM.pdf</a><span style="color:gray"> (39K)</span></div><div class="small guid">GUID: B133C5B5-3B05-4D35-9813-B57313FC2ED9</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Additional file 3: Figure. S3. </strong>Selection of protein panels to classify CRC from healthy subjects. (A) Mean decrease of Gini index in DDA data. (B) Mean decrease of Gini index in DIA data.</div><div><a href="/pmc/articles/PMC10880210/bin/12967_2024_4983_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl">12967_2024_4983_MOESM3_ESM.pdf</a><span style="color:gray"> (657K)</span></div><div class="small guid">GUID: 4579DF16-E056-4294-BFE8-B81AE8510E71</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Additional file 4: Figure S4. </strong>Peptide sequence and ion mass information of selected protein markers.</div><div><a href="/pmc/articles/PMC10880210/bin/12967_2024_4983_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl">12967_2024_4983_MOESM4_ESM.pdf</a><span style="color:gray"> (12K)</span></div><div class="small guid">GUID: 0A33BEF4-E2DD-455F-BFB2-0D4A042413BF</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Additional file 5: Figure S5. </strong>Quality of the MRM data. (A&#x02013;D). Extracted chromatograms of the 7-peptide biomarkers. (E) Variable coefficient values in QC samples for the 7-peptide biomarkers.</div><div><a href="/pmc/articles/PMC10880210/bin/12967_2024_4983_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl">12967_2024_4983_MOESM5_ESM.pdf</a><span style="color:gray"> (126K)</span></div><div class="small guid">GUID: 66B64A28-61A6-42D2-8B5B-F2391700C82B</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Additional file 6: Table S1. </strong>Demographic data of all the patients enrolled in MRM experiments.</div><div><a href="/pmc/articles/PMC10880210/bin/12967_2024_4983_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl">12967_2024_4983_MOESM6_ESM.xlsx</a><span style="color:gray"> (9.7K)</span></div><div class="small guid">GUID: 8B30AEFC-1B2A-4459-8ED9-7005FBE0B914</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Additional file 7: Table S2. </strong>Protein expression data from DDA experiments.</div><div><a href="/pmc/articles/PMC10880210/bin/12967_2024_4983_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl">12967_2024_4983_MOESM7_ESM.xlsx</a><span style="color:gray"> (637K)</span></div><div class="small guid">GUID: C50F0AF8-FE10-4932-B981-C72E49B8D438</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Additional file 8: Table S3. </strong>Protein expression data from DIA experiments.</div><div><a href="/pmc/articles/PMC10880210/bin/12967_2024_4983_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl">12967_2024_4983_MOESM8_ESM.xlsx</a><span style="color:gray"> (486K)</span></div><div class="small guid">GUID: AF08843E-C66F-40B3-BA5A-4A2FF4B739F8</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Additional file 9: Table S4. </strong>Concentration of each protein of the seven-protein panel measured by MRM in all subjects.</div><div><a href="/pmc/articles/PMC10880210/bin/12967_2024_4983_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl">12967_2024_4983_MOESM9_ESM.xlsx</a><span style="color:gray"> (66K)</span></div><div class="small guid">GUID: 0847242E-C4F5-4A00-AE35-FF65B9D6DBDA</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong> Additional file 10: Table S5. </strong>Concentration and statistical analysis of each protein of the seven-protein panel measured by MRM in CRC cases shown in Figure 4, D and E.</div><div><a href="/pmc/articles/PMC10880210/bin/12967_2024_4983_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl">12967_2024_4983_MOESM10_ESM.xlsx</a><span style="color:gray"> (27K)</span></div><div class="small guid">GUID: D2ACD4D1-B303-4FF4-A2EF-C2C3E5B74CD9</div></div></dd></dl><dl data-length="382" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p>The datasets generated (mass spectrometry raw data) during the current study has been deposited to the ProteomeXchange repository with the accession numbers PXD042639 and PXD042652 (<a href="https://proteomecentral.proteomexchange.org/cgi/GetDataset" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://proteomecentral.proteomexchange.org/cgi/GetDataset</a>). The analyzed data is included in the supplementary tables. The clinical samples and analytical methods will not be available to the public.</p></dd></dl></div><div id="Abs1" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="Abs1title">Abstract</h2><!--article-meta--><div><div id="sec-a.v.b.m.a" class="sec sec-first"><h3 id="sec-a.v.b.m.atitle">Background</h3><p id="Par1" class="p p-first-last">Diagnosis of colorectal cancer (CRC) during early stages can greatly improve patient outcome. Although technical advances in the field of genomics and proteomics have identified a number of candidate biomarkers for non-invasive screening and diagnosis, developing more sensitive and specific methods with improved cost-effectiveness and patient compliance has tremendous potential to help combat the disease.</p></div><div id="sec-a.v.b.m.b" class="sec"><h3 id="sec-a.v.b.m.btitle">Methods</h3><p id="Par2" class="p p-first-last">We enrolled three cohorts of 479 subjects, including 226 CRC cases, 197 healthy controls, and 56 advanced precancerous lesions (APC). In the discovery cohort, we used quantitative mass spectrometry to measure the expression profile of plasma proteins and applied machine-learning to select candidate proteins. We then developed a targeted mass spectrometry assay to measure plasma concentrations of seven proteins and a logistic regression classifier to distinguish CRC from healthy subjects. The classifier was further validated using two independent cohorts.</p></div><div id="sec-a.v.b.m.c" class="sec"><h3 id="sec-a.v.b.m.ctitle">Results</h3><p id="Par3" class="p p-first-last">The seven-protein panel consisted of leucine rich alpha-2-glycoprotein 1 (LRG1), complement C9 (C9), insulin-like growth factor binding protein 2 (IGFBP2), carnosine dipeptidase 1 (CNDP1), inter-alpha-trypsin inhibitor heavy chain 3 (ITIH3), serpin family A member 1 (SERPINA1), and alpha-1-acid glycoprotein 1 (ORM1). The panel classified CRC and healthy subjects with high accuracy, since the area under curve (AUC) of the training and testing cohort reached 0.954 and 0.958. The AUC of the two independent validation cohorts was 0.905 and 0.909. In one validation cohort, the panel had an overall sensitivity, specificity, positive predictive value (PPV), and negative predictive value (NPV) of 89.9%, 81.8%, 89.2%, and 82.9%, respectively. In another blinded validation cohort, the panel classified CRC from healthy subjects with a sensitivity of 81.5%, specificity of 97.9%, and overall accuracy of 92.0%. Finally, the panel was able to detect APC with a sensitivity of 49%.</p></div><div id="sec-a.v.b.m.d" class="sec"><h3 id="sec-a.v.b.m.dtitle">Conclusions</h3><p id="Par4" class="p p-first-last">This seven-protein classifier is a clear improvement compared to previously published blood-based protein biomarkers for detecting early-stage CRC, and is of translational potential to develop into a clinically useful assay.</p></div><div id="sec-a.v.b.m.e" class="sec sec-last"><h3 id="sec-a.v.b.m.etitle">Supplementary Information</h3><p class="p p-first-last">The online version contains supplementary material available at 10.1186/s12967-024-04983-5.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Colorectal cancer, Early detection, Protein biomarker, Mass spectrometry</span></div></div><div id="Sec1" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec1title">Background</h2><p id="Par5" class="p p-first">Colorectal cancer (CRC) is a leading cause of cancer-related deaths worldwide. In the United States alone, CRC is the second most common cancer with a mortality rate ranks the second among all cancers [<a href="#CR1" rid="CR1" class=" bibr popnode">1</a>]. The outcome of CRC patient is generally poor, to a large extent due to lack of effective testing methods to detect tumors at early stages. If tumor is detected at precancerous or localized stage, 90% of CRC patients can survive more than five years. Whereas if it is diagnosed at stages IIb or later, the survival rate is dramatically lower, primarily due to complications associated with tumor spreading and metastasis [<a href="#CR2" rid="CR2" class=" bibr popnode">2</a>]. Currently, the &#x0201c;gold standard&#x0201d; of clinical procedure for diagnosing CRC is the colonoscopy [<a href="#CR3" rid="CR3" class=" bibr popnode">3</a>], during which the tumors can be removed and pathologically examined. There are also a number of stool- and blood-based non-invasive screening methods to aid the detection of CRC at early stages [<a href="#CR4" rid="CR4" class=" bibr popnode">4</a>]. The FOBT/FIT uses immunoassay to measure the hemoglobin in the stool [<a href="#CR5" rid="CR5" class=" bibr popnode">5</a>]. Two DNA tests measure the methylation status at the promoter region of mSEPT9 or SDC2 gene in the blood and the stool respectively [<a href="#CR6" rid="CR6" class=" bibr popnode">6</a>, <a href="#CR7" rid="CR7" class=" bibr popnode">7</a>]. Immunoassays to measure the traditional tumor antigens including carbohydrate antigen CA19-9 and carcinoembryonic antigen (CEA) [<a href="#CR8" rid="CR8" class=" bibr popnode">8</a>, <a href="#CR9" rid="CR9" class=" bibr popnode">9</a>] are widely used in the clinic. In recent years, the burgeoning field of liquid biopsy has made great strides to non-invasive test of multiple cancer types at early stages, including CRC [<a href="#CR10" rid="CR10" class=" bibr popnode">10</a>&#x02013;<a href="#CR12" rid="CR12" class=" bibr popnode">12</a>]. The assay detecting septin9 gene methylation in the blood has offered a laboratory developed test (LDT, Epi procolon) [<a href="#CR13" rid="CR13" class=" bibr popnode">13</a>]. Another LDT combines FIT and detecting BMP3/NDRG4 DNA methylation as well as KRAS mutation in the fecal sample (Cologuard) [<a href="#CR14" rid="CR14" class=" bibr popnode">14</a>]. These exciting developments have opened up new opportunities for early detection of CRC.</p><p id="Par6">However, there are still windows for improvement in terms of sensitivity and specificity as well as other issues such as cost-effectiveness and patient compliance. The FOBT/FIT test yields a sensitivity of 73.8% with a specificity of 95% [<a href="#CR14" rid="CR14" class=" bibr popnode">14</a>], whereas the protein biomarkers provide a sensitivity of 80% [<a href="#CR15" rid="CR15" class=" bibr popnode">15</a>]. Epi procolon test has a specificity of 88%, yet the sensitivity is 87% for early-stage CRC [<a href="#CR13" rid="CR13" class=" bibr popnode">13</a>]. Cologuard has a sensitivity of 92.3%; its specificity falls to 86.6% [<a href="#CR14" rid="CR14" class=" bibr popnode">14</a>]. Improving both the sensitivity and specificity simultaneously can screen CRC with much improved accuracy, limiting both the false positive and false negative calls. On the other hand, protein markers in clinical use such as CEA and CA19-9 is far from sensitive. Although colonoscopy is the &#x0201c;gold standard&#x0201d;, the invasive nature raises the issue of patient compliance. Therefore, a non-invasive, sensitive and specific screening method that detects and diagnoses CRC at the earliest stage is needed.</p><p id="Par7">Advanced precancerous lesions (APC), including advanced adenomas and large sessile serrated polyps (greater than 1&#x000a0;cm in size), are considered to have a high probability of developing into fully grown cancers [<a href="#CR16" rid="CR16" class=" bibr popnode">16</a>]. Nevertheless, the progression time window of advanced adenomas can be as long as ten years, providing both an opportunity and a challenge to detect these lesions in a timely manner [<a href="#CR16" rid="CR16" class=" bibr popnode">16</a>, <a href="#CR17" rid="CR17" class=" bibr popnode">17</a>]. The standard for detecting advanced adenomas is also optical colonoscopy [<a href="#CR18" rid="CR18" class=" bibr popnode">18</a>]. Due to patient compliance and rare complications, applying colonoscopy as a general screening method to detect APC remains to be a challenge [<a href="#CR16" rid="CR16" class=" bibr popnode">16</a>]. Although liquid biopsy based on gene test to detect adenomas has been developed, it still suffers from low sensitivity [<a href="#CR14" rid="CR14" class=" bibr popnode">14</a>]. Therefore, developing much improved assays to detect precancerous lesions is also highly desirable.</p><p id="Par8" class="p p-last">Currently, mass spectrometry technologies have achieved high sensitivity and data acquisition speed. Whereas traditional data-dependent acquisition (DDA) provides straightforward protein identification and ease of data interpretation, data-independent acquisition (DIA) dramatically expanded the dynamic range of quantitation [<a href="#CR19" rid="CR19" class=" bibr popnode">19</a>, <a href="#CR20" rid="CR20" class=" bibr popnode">20</a>]. Multi-reaction monitoring (MRM), another mass spectrometry technology widely used in pharmaceutical industry, is well suited to accurately quantify hundreds of peptides in one experiment [<a href="#CR21" rid="CR21" class=" bibr popnode">21</a>, <a href="#CR22" rid="CR22" class=" bibr popnode">22</a>]. As such, we applied both DDA and DIA to investigate the plasma proteome in patients diagnosed with CRC and APC. Our analysis captured unique molecular features of the plasma proteome in these conditions. We further developed an MRM assay combined with machine learning to discover and validate a panel of protein biomarkers to distinguish CRC from benign polyps and from healthy subjects.</p></div><div id="Sec2" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec2title">Methods</h2><div id="Sec3" class="sec sec-first"><h3 id="Sec3title">Human samples</h3><p id="Par9" class="p p-first">Plasma samples of the Chinese population were collected from three hospitals between September 2020 and September 2022. Peripheral venous blood samples were collected before any treatment procedure, and were centrifuged at 500&#x02009;&#x000d7;&#x02009;g for 10&#x000a0;min to obtain plasma within two hours and stored at &#x02212;&#x000a0;80&#x000a0;&#x02103;. All plasma samples were transported to the central lab by cold chain system at &#x02212;&#x000a0;80&#x000a0;&#x02103; and stored at &#x02212;&#x000a0;80&#x000a0;&#x02103; before MS experiments.</p><p id="Par10">In the discovery cohort, plasma samples from 70 patients diagnosed with colorectal cancer (CRC) and 72 healthy subjects were collected from Shanghai Tenth People&#x02019;s Hospital, Tongji University School of Medicine. In the validation cohorts, the plasma samples were collected from Shanghai Tenth People&#x02019;s Hospital, the First Affiliated Hospital of Zhejiang University and Changzhou Wujin People&#x02019;s Hospital, respectively. We obtained written informed consent from each participant.</p><p id="Par11" class="p p-last">We excluded patients with any malignant tumors within five years prior to current diagnosis. The inclusion criteria were as follows: the age of all subjects is over 40&#x000a0;years (inclusive), with a balanced gender distribution; in the group of healthy subjects there should have no evidence of malignant tumor nor colorectal neoplasm; in the CRC group all diagnosis was confirmed by pathological evidence; in the APC group all patients went through colonoscopy and the diagnosis of either advanced adenomas or sessile serrated polyps were made by an experienced pathologist.</p></div><div id="Sec4" class="sec"><h3 id="Sec4title">Processing of plasma samples</h3><p id="Par12" class="p p-first">For the two discovery cohorts, the plasma samples were processed to deplete the top 14 high-abundant proteins (Cat. # A36370, Thermo Science, USA), and protein concentration was determined using the BCA kit (Cat. # P0012, Beyotime, China). The 14 proteins included Albumin, IgA, IgD, IgE, IgG, IgG (light chains), IgM, Alpha-1-acid glycoprotein, Alpha-1-antitrypsin, Alpha-2-macroglobulin, Apolipoprotein A1, Fibrinogen, Haptoglobin, and Transferrin. From each sample, 25&#x000a0;&#x003bc;g protein was suspended in 50&#x000a0;mM NH<sub>4</sub>HCO<sub>3</sub> solution. The proteins were treated with 10&#x000a0;mM DTT at 95&#x000a0;&#x000b0;C for 10&#x000a0;min and alkylated with 15&#x000a0;mM iodoacetamide (Cat. # I1149, Sigma Aldrich, USA) in the dark for 30&#x000a0;min. Then the protein was digested with sequencing grade trypsin (1:50, Cat. # V5113, Promega) overnight at 37&#x000a0;&#x000b0;C. The resulting peptides were desalted with a 96-well SOLA solid-phase extraction apparatus and vacuum dried. The peptides were stored in a freezer at &#x02212;&#x000a0;80&#x000a0;&#x02103;. and ready for mass spectrometry analysis.</p><p id="Par13" class="p p-last">For the four validation cohorts, the plasma samples were processed for mass spectrometry analysis without depleting the high-abundant proteins and the rest of experiment procedure was identical to the discovery cohorts.</p></div><div id="Sec5" class="sec"><h3 id="Sec5title">LC&#x02013;MS/MS analysis of plasma samples using DDA and DIA</h3><p id="Par14" class="p p-first">The plasma protein digests were analyzed using an EASY-nLC1000 liquid chromatography coupled with an Orbitrap Exploris&#x02122; 240 mass spectrometer (Thermo Fisher Scientific, USA). Peptides were resuspended in buffer A (2% ACN, 0.1% formic acid) and spiked with indexed retention time (iRT) peptides (Omicsolution, China). The iRT peptides are a set of standard peptides used in DIA experiments for high-accuracy calibration of chromatographic elution time, so as to improve reproducibility of MS experiments across laboratories. An equivalent to 2&#x000a0;&#x000b5;g of protein digest from each sample was loaded onto a C18 column (Cat. # 164534, Thermo Scientific, USA) linked with a pre-column (Cat. # 164535, Thermo Fisher Scientific, USA) and separated at a flow rate of 250 nL/ min. The mobile phases consisted of buffer A and buffer B (98% ACN, 0.1% formic acid). A 90&#x000a0;min gradient from 1 to 8% buffer B in 1&#x000a0;min, 8% to 28% in 71&#x000a0;min, 28% to 40% in 9&#x000a0;min, 40% to 100% in 2&#x000a0;min, and 100% for 7&#x000a0;min was used. For DIA analysis, peptides were resuspended in buffer A and spiked with iRT peptides. 1.5&#x000a0;&#x000b5;g of protein digest from each sample was loaded onto the C18 column.</p><p id="Par15">The mass spectrometry was operated in positive mode in all cases. For DDA analysis, the nano-electrospray was operated using the ion transfer tube with a temperature setting of 275 &#x000b0;C. One full scan MS from 400 to 1400&#x000a0;m/z followed by 12 MS<sup>2</sup> scans were cycled throughout the entire MS experiment. MS spectra were acquired with a resolution of 70000 with a maximum injection time (IT) of 60&#x000a0;s and an automatic gain control (AGC) target value of 3e6. MS<sup>2</sup> spectra were obtained in the higher-energy collisional dissociation (HCD) mode with an isolation window of 1.6&#x000a0;m/z, using a normalized collision energy of 27%, resolution at 17500 with a maximum injection time of 50&#x000a0;s and an AGC target of 5e5. Centroid mode was used to collect both the MS and MS<sup>2</sup> spectra.</p><p id="Par16" class="p p-last">For DIA analysis, the ion transfer tube was operated with a temperature setting of 320&#x000a0;&#x000b0;C. MS spectra were acquired with a resolution of 60000 with a maximum injection time (IT) of 120&#x000a0;ms and an AGC target value of 3e6. Isolation window for MS<sup>2</sup> was set to 20&#x000a0;Da for the mass range 350&#x02013;400&#x000a0;m/z, 9&#x000a0;Da window for the mass range 400&#x02013;800&#x000a0;m/z, 12&#x000a0;Da window for the mass range 800&#x02013;1000&#x000a0;m/z, and 25&#x000a0;Da window for the mass range 1000&#x02013;1200&#x000a0;m/z.</p></div><div id="Sec6" class="sec"><h3 id="Sec6title">Multiple reaction monitoring quantitation of plasma proteins</h3><p id="Par17" class="p p-first">Concentrations of target proteins in the plasma were measured using MRM on a QTRAP 5500 mass spectrometer (Sciex, USA) equipped with a turbo v ion source (Sciex, USA). The instrument parameters of the MRM assay were optimized for each synthetic peptide by directly infusing the peptides into the mass spectrometer. The top three high-intensity product ions of each peptide precursor ion were selected based on the optimal collision energy (CE) values and collision cell exit potential (CXP). All optimized data were collected and compared to theoretical spectra, and three high-intensity y-ions were used for subsequent MRM assays.</p><p id="Par18" class="p p-last">The peptides were separated using an LC-20AD (SHIMADZU, Japan) liquid chromatographic system. Buffer A was 0.1% formic acid in distilled water and buffer B was 0.1% formic acid in 98% acetonitrile. Peptides were reconstituted in buffer A and 15 &#x000b5;L of each sample was loaded into the sample loop. A gradient consisting of 6% buffer B for 2&#x000a0;min, 6&#x02013;28% buffer B for 16&#x000a0;min, 28&#x02013;98% buffer B for 0.5&#x000a0;min, 98% buffer B for 3&#x000a0;min, 98&#x02013; 6% buffer B for 0.5&#x000a0;min, and 6% buffer B for 3&#x000a0;min was used. The MS detection was carried out in positive mode with the following parameters: electrospray voltage of 5500&#x000a0;V, curtain gas at 40 psi, ion source gas 1 (GS1) at 55 psi, ion source gas 2 (GS2) at 55 psi, and temperature at 500&#x000a0;&#x000b0;C. Quantitation were performed using the scheduled MRM mode. The time of MRM detection window was 180&#x000a0;s, and the cycle time was 1.0&#x000a0;s. The mass spectrometer was controlled by the Analyst software (Sciex, USA).</p></div><div id="Sec7" class="sec"><h3 id="Sec7title">Mass spectrometry data analysis</h3><p id="Par19" class="p p-first">The DDA spectra were searched using Protein Discoverer 2.4 (Thermo Fisher Scientific, USA) against a UniprotKB human database (UP000005640). The search parameters were set as the following: trypsin was set to the protease type and two missed cleavages were allowed, the precursor mass tolerance was set to 10&#x000a0;ppm, and the fragmentation ion mass tolerance was set to 0.02&#x000a0;Da. The false discovery rate (FDR) was set to 1% at both the peptide and protein level. The maximum number of variable modifications was set to two.</p><p id="Par20" class="p p-last">The DIA spectra were searched using the DIA-NN (version 1.7.15) software [<a href="#CR23" rid="CR23" class=" bibr popnode">23</a>] with a UniprotKB human database (UP000005640). The precursor mass tolerance was 10&#x000a0;ppm; trypsin was set as the protease and two missed cleavages were allowed. The maximum number of variable modifications was set to three. The precursor mass range was from 350 to 1250&#x000a0;m/z, and the fragmentation ion mass range was from 100 to 2000&#x000a0;m/z. The false discovery rate (FDR) was set at 1% at the peptide level.</p></div><div id="Sec8" class="sec"><h3 id="Sec8title">GO and KEGG pathway analysis</h3><p id="Par21" class="p p-first-last">GO and KEGG enrichment analysis were performed using bioinformatics resources including Metascape (<a href="https://metascape.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://metascape.org/</a>) and David v6.8 (<a href="https://david.ncifcrf.gov/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://david.ncifcrf.gov/</a>). R package ClusterProfiler [<a href="#CR24" rid="CR24" class=" bibr popnode">24</a>] was applied to generate the graphs.</p></div><div id="Sec9" class="sec"><h3 id="Sec9title">Feature selection and logistic regression</h3><p id="Par22" class="p p-first">The mean decrease of Gini index (MDG) was calculated in a random forest feature selection model for both the DDA and DIA data. Gini index is a measurement of variance in random forest algorithm, in which lower variance and thus lower Gini index results in more accurate classification. We selected top 40 proteins based on the MDG values. To further narrow down the list of biomarker candidates from these 40 proteins, we applied the following criteria: 1. These proteins are differentially expressed in both datasets. 2. Literature reported cancer biomarkers were given priority for consideration.</p><p id="Par23" class="p p-last">The expression values of the peptide surrogates for the candidate proteins were used to build a logistic regression model to classify subjects as either healthy or colorectal cancer patients. The finalized panel of proteins were validated by new patient cohorts.</p></div><div id="Sec10" class="sec sec-last"><h3 id="Sec10title">Statistical analysis</h3><p id="Par24" class="p p-first">R (version 4.2.2) was used for all the statistical analyses, including data preprocessing, differential expression analysis, volcano plot, and principal component analysis (PCA). For differential expression analysis, p value&#x02009;&#x0003c;&#x02009;0.05 were considered statistically significant and fold change of&#x02009;&#x0003e;&#x02009;1.25 or&#x02009;&#x0003c;&#x02009;0.80 were considered up- or down-regulated, respectively. We utilized a more subtle fold change criterion in order to obtain more potential protein markers in the discovery phase, and further validated them in target validation phase. Proteins with more than 50% missing values were removed. The distribution of protein expression was tested for normality across all samples; then, t-test was applied for those with normal distribution, while Wilcoxon ranked sum test was performed for those failed to pass the normality test.</p><p id="Par25" class="p p-last">The discovery phase of the study was designed to obtain a statistical power of 85%, a one-sided type I error rate of 0.05, and a median effect size. With these parameters, the calculated sample size is 59 healthy subjects and 59 colorectal cancer patients, respectively.</p></div></div><div id="Sec11" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec11title">Results</h2><div id="Sec12" class="sec sec-first"><h3 id="Sec12title">Study design</h3><p id="Par26" class="p p-first-last">The design of this study is illustrated in Fig.&#x000a0;<a href="/pmc/articles/PMC10880210/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1</span></a>. In the discovery phase, we enrolled a total of 142 subjects (including 70 CRC patients and 72 healthy controls, cohort 1) from one hospital. The plasma samples from half of the discovery cohort were analyzed using DDA method, whereas the other half were analyzed using DIA method. To monitor the data acquisition process, we interspersed quality control (QC) samples during mass spectrometry data acquisition, which composed of small portion of plasma samples from all patients in the discovery cohort (Fig.&#x000a0;<a href="/pmc/articles/PMC10880210/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1</span></a>A). In the assay development phase, MRM method was applied to part of the samples from cohort 1 to measure the plasma concentration of selected protein biomarkers based on the signature peptides. In the meantime, MRM assay were developed to assess the linear range, limit of quantification, and reproducibility (Fig.&#x000a0;<a href="/pmc/articles/PMC10880210/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1</span></a>B). The validation phase included 129 CRC patients and 77 healthy controls from two different hospitals as well as 47 cases of APC as cohort 2 (Fig.&#x000a0;<a href="/pmc/articles/PMC10880210/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1</span></a>C). We then enrolled another independent, blinded validation cohort of patients as cohort 3 from the third hospital, in which the laboratory did not know the diagnosis of the patients until the blood test results were complete (Fig.&#x000a0;<a href="/pmc/articles/PMC10880210/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1</span></a>D). In all cohorts, cancer patients either went through colonoscopy or surgery and the tumor tissues were pathologically confirmed. The demographic data for the patients enrolled in the discovery cohort is shown in Table&#x000a0;<a href="/pmc/articles/PMC10880210/table/Tab1/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab1" rid-ob="ob-Tab1" co-legend-rid=""><span>1</span></a>. The demographic data for all the patients enrolled in validation cohorts is shown in Additional file <a href="#MOESM6" rid="MOESM6" class=" media">6</a>: Table S1.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig1" co-legend-rid="lgnd_Fig1"><a href="/pmc/articles/PMC10880210/figure/Fig1/" target="figure" rid-figpopup="Fig1" rid-ob="ob-Fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139958977523152"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10880210_12967_2024_4983_Fig1_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 12967_2024_4983_Fig1_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10880210/bin/12967_2024_4983_Fig1_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139958977523152"><a target="object" rel="noopener" href="/pmc/articles/PMC10880210/figure/Fig1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig1"><div><a class="figpopup" href="/pmc/articles/PMC10880210/figure/Fig1/" target="figure" rid-figpopup="Fig1" rid-ob="ob-Fig1">Fig.&#x000a0;1</a></div><!--caption a7--><div class="caption"><p>Study design. Flow chart showing the discovery and validation phases of this study. <strong>A</strong> Discovery cohort for quantitative proteomic analysis using DDA and DIA methods. <strong>B</strong> MRM targeted proteomic assay development. <strong>C</strong> Applying the MRM assay to an independent validation cohort of patients to classify CRC and APC patients from healthy controls. <strong>D</strong> Applying the MRM assay to a blinded validation cohort, in which the diagnosis was disclosed only after the test results were given</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="Tab1"><h3>Table&#x000a0;1</h3><!--caption a7--><div class="caption"><p>Demographic data and characteristics of the participants in the discovery cohort</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="left" rowspan="1" colspan="1"></th><th align="left" rowspan="1" colspan="1"></th><th align="left" colspan="3" rowspan="1">DDA (n&#x02009;=&#x02009;70)</th><th align="left" colspan="3" rowspan="1">DIA (n&#x02009;=&#x02009;72)</th></tr><tr><th align="left" rowspan="1" colspan="1">Characteristics</th><th align="left" rowspan="1" colspan="1">Level</th><th align="left" rowspan="1" colspan="1">HC</th><th align="left" rowspan="1" colspan="1">CRC</th><th align="left" rowspan="1" colspan="1">P</th><th align="left" rowspan="1" colspan="1">HC</th><th align="left" rowspan="1" colspan="1">CRP</th><th align="left" rowspan="1" colspan="1">P</th></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1">Number</td><td align="left" rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1">36</td><td align="left" rowspan="1" colspan="1">34</td><td align="center" rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1">36</td><td align="left" rowspan="1" colspan="1">36</td><td align="center" rowspan="1" colspan="1"></td></tr><tr><td align="left" rowspan="1" colspan="1">Gender</td><td align="left" rowspan="1" colspan="1">Female</td><td align="left" rowspan="1" colspan="1">21 (58.3)</td><td align="left" rowspan="1" colspan="1">11 (32.4)</td><td align="center" rowspan="1" colspan="1">0.034</td><td align="left" rowspan="1" colspan="1">16 (44.4)</td><td align="left" rowspan="1" colspan="1">14 (38.9)</td><td align="center" rowspan="1" colspan="1">0.811</td></tr><tr><td align="left" rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1">Male</td><td align="left" rowspan="1" colspan="1">15 (41.7)</td><td align="left" rowspan="1" colspan="1">23 (67.6)</td><td align="center" rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1">20 (55.6)</td><td align="left" rowspan="1" colspan="1">22 (61.1)</td><td align="center" rowspan="1" colspan="1"></td></tr><tr><td align="left" rowspan="1" colspan="1">Age (mean (SD))</td><td align="left" rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1">53.61 (4088)</td><td align="left" rowspan="1" colspan="1">61.09 (8027)</td><td align="center" rowspan="1" colspan="1">&#x02009;&#x0003c;&#x02009;0.001</td><td align="left" rowspan="1" colspan="1">53.11 (5.87)</td><td align="left" rowspan="1" colspan="1">60.78 (8.78)</td><td align="center" rowspan="1" colspan="1">&#x02009;&#x0003c;&#x02009;0.001</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139958974048880"><a target="object" rel="noopener" href="/pmc/articles/PMC10880210/table/Tab1/?report=objectonly">Open in a separate window</a></div></div></div><div id="Sec13" class="sec"><h3 id="Sec13title">Quantitative mass spectrometric analysis of plasma proteome using two different data acquisition methods on independent batches of patient samples</h3><p id="Par27" class="p p-first-last">The QC samples showed good correlation (and thus reproducibility) in DDA data among themselves in a pair-wise comparison matrix, as shown by the Pearson correlation in Additional file <a href="#MOESM1" rid="MOESM1" class=" media">1</a>: Fig. S1A. The correlation was even better in DIA data as shown by the correlation matrix (Additional file <a href="#MOESM1" rid="MOESM1" class=" media">1</a>: Fig. S1B), indicating that DIA provide more overall accuracy and consistency than DDA as have been shown previously [<a href="#CR25" rid="CR25" class=" bibr popnode">25</a>]. Both DDA and DIA covered a dynamic range of 6 order of magnitude, with DIA covered a wider dynamic range (Additional file <a href="#MOESM1" rid="MOESM1" class=" media">1</a>: Fig. S1C&#x02013;D). Furthermore, DIA quantified more proteins than DDA in all samples, as shown in Additional file <a href="#MOESM1" rid="MOESM1" class=" media">1</a>: Fig. S1E&#x02013;F. We quantified 607 proteins in DDA experiments and 714 proteins in DIA experiments (Additional file <a href="#MOESM7" rid="MOESM7" class=" media">7</a>: Table S2 and Additional file <a href="#MOESM8" rid="MOESM8" class=" media">8</a>: Table S3). Using a statistical significance cutoff of 0.05 and fold change cutoff of 1.25, we identified 21 up-regulated and 16 down-regulated proteins in DDA experiments, and 50 up-regulated and 106 down-regulated proteins in DIA experiments as shown in the volcano plots (Fig.&#x000a0;<a href="/pmc/articles/PMC10880210/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2</span></a>A, B). Principal component analysis using the entire proteomic data failed to separate the disease, healthy and the QC samples, neither in the DDA data (Fig.&#x000a0;<a href="/pmc/articles/PMC10880210/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2</span></a>C) nor in the DIA data (Fig.&#x000a0;<a href="/pmc/articles/PMC10880210/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2</span></a>D). In up-regulated proteins, gene ontology of the most enriched biological function was primarily acute-phase response and humoral immune response, with the corresponding KEGG pathways involved being complement and coagulation cascades (Fig.&#x000a0;<a href="/pmc/articles/PMC10880210/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2</span></a>E and Additional file <a href="#MOESM2" rid="MOESM2" class=" media">2</a>: Fig. S2). Whereas in down-regulated proteins, the most enriched biological function was wound healing which involves remodeling of the extracellular matrix, and the corresponding KEGG pathways involved were carbon metabolism and biosynthesis of amino acids (Fig.&#x000a0;<a href="/pmc/articles/PMC10880210/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2</span></a>F and Additional file <a href="#MOESM2" rid="MOESM2" class=" media">2</a>: Fig. S2). Several important intracellular signaling pathways including Wnt signaling, PI3K-Akt signaling, and p53 signaling are frequently dysregulated in CRC [<a href="#CR26" rid="CR26" class=" bibr popnode">26</a>&#x02013;<a href="#CR28" rid="CR28" class=" bibr popnode">28</a>]. A recent study demonstrated that CRC cell survival was related to an impaired hypoxia-inducible factor 1-alpha (HlF-1a) signaling in low oxygen condition [<a href="#CR29" rid="CR29" class=" bibr popnode">29</a>]. Among these important driving events of tumorigenesis, some were captured in our findings [<a href="#CR27" rid="CR27" class=" bibr popnode">27</a>, <a href="#CR28" rid="CR28" class=" bibr popnode">28</a>].</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig2" co-legend-rid="lgnd_Fig2"><a href="/pmc/articles/PMC10880210/figure/Fig2/" target="figure" rid-figpopup="Fig2" rid-ob="ob-Fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139958983429488"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10880210_12967_2024_4983_Fig2_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 12967_2024_4983_Fig2_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10880210/bin/12967_2024_4983_Fig2_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139958983429488"><a target="object" rel="noopener" href="/pmc/articles/PMC10880210/figure/Fig2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig2"><div><a class="figpopup" href="/pmc/articles/PMC10880210/figure/Fig2/" target="figure" rid-figpopup="Fig2" rid-ob="ob-Fig2">Fig.&#x000a0;2</a></div><!--caption a7--><div class="caption"><p>Quantitative proteomic analysis of plasma samples from CRC patients and healthy controls. <strong>A</strong> Volcano plot of DDA data obtained from discovery cohort 1. Differentially expressed proteins are shown in blue (down) or red (up) circles. X-axis shows log2-fold change of plasma proteins between CRC patients and healthy subjects, and y-axis shows log10 of statistical significance values. <strong>B</strong> Volcano plot of DIA data obtained from discovery cohort 2. The label for the differentially expressed proteins and the two axes are the same as in A. <strong>C</strong> Principal component analysis of the protein expression data in cohort 1. <strong>D</strong> Principal component analysis of the protein expression data from cohort 2. <strong>E</strong> Gene ontology analysis of up-regulated proteins in the DIA data. <strong>F</strong> Gene ontology analysis of down-regulated proteins in the DIA data</p></div></div></div></div><div id="Sec14" class="sec"><h3 id="Sec14title">Discovery of a plasma protein biomarker panel to identify CRC at early stages</h3><p id="Par28" class="p p-first-last">To identify biomarkers in the plasma that can accurately distinguish early-stage CRC from healthy subjects, we selected the top 40 proteins based on MDG values (Additional file <a href="#MOESM3" rid="MOESM3" class=" media">3</a>: Fig. S3). From these proteins we identified consistent up-regulation of LRG1, C9, IGFBP2, ITIH3, and SERPINA1 and consistent down-regulation of CNDP1 in CRC plasma as potential biomarker candidates (Fig.&#x000a0;<a href="/pmc/articles/PMC10880210/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span>3</span></a>A, B). ORM1 was found in DIA data but did not show differential expression; however, it was up-regulated in the DDA data (Fig.&#x000a0;<a href="/pmc/articles/PMC10880210/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span>3</span></a>A). This discrepancy maybe due to variation in mass spectrometric data acquisition. Because ORM1 has been shown to be a biomarker for CRC [<a href="#CR30" rid="CR30" class=" bibr popnode">30</a>, <a href="#CR31" rid="CR31" class=" bibr popnode">31</a>], we added it to the potential list of our biomarker panel. The peptide sequences of the panel of seven proteins were listed in Additional file <a href="#MOESM4" rid="MOESM4" class=" media">4</a>: Fig. S4. Using the expression data of these seven proteins, principal component analysis (PCA) showed much improved separation between CRC and healthy subjects, with the first component explained 33.9% (DDA data) and 64.8% (DIA data) of the variability (Fig.&#x000a0;<a href="/pmc/articles/PMC10880210/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span>3</span></a>C, D).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig3" co-legend-rid="lgnd_Fig3"><a href="/pmc/articles/PMC10880210/figure/Fig3/" target="figure" rid-figpopup="Fig3" rid-ob="ob-Fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139958984252288"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10880210_12967_2024_4983_Fig3_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 12967_2024_4983_Fig3_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10880210/bin/12967_2024_4983_Fig3_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139958984252288"><a target="object" rel="noopener" href="/pmc/articles/PMC10880210/figure/Fig3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig3"><div><a class="figpopup" href="/pmc/articles/PMC10880210/figure/Fig3/" target="figure" rid-figpopup="Fig3" rid-ob="ob-Fig3">Fig.&#x000a0;3</a></div><!--caption a7--><div class="caption"><p>Seven feature proteins selected based on DDA and DIA proteome data. <strong>A</strong> and <strong>B</strong> Boxplot of protein intensity of feature proteins selected for logistic regression. Differential expression of seven proteins between CRC patients (<strong>C</strong>) and healthy subjects <strong>H</strong> from the DDA data (<strong>A</strong>) and the DIA data (<strong>B</strong>) are shown. <strong>C</strong> Principal component analysis using the expression levels of the seven proteins in cohort 1. <strong>D</strong> Principal component analysis using the expression levels of the six proteins in cohort 2</p></div></div></div></div><div id="Sec15" class="sec"><h3 id="Sec15title">The plasma protein biomarker panel is capable of detecting colorectal cancer with high accuracy</h3><p id="Par29" class="p p-first">Because DDA and DIA are relative quantification methods and MRM is able to measure the absolution concentration of an analyte with higher throughput, we developed an MRM assay using signature peptides to measure the concentration of the seven proteins in the plasma, in a subset of 60 healthy subjects and 60 CRC patients from the discovery cohort (Fig.&#x000a0;<a href="/pmc/articles/PMC10880210/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1</span></a>B). The MRM assay combined internal standards using heavy arginine/lysine-labeled peptides with external calibration using unlabeled peptides. The product ion peak of each peptide appeared superior in our MRM assay (Additional file <a href="#MOESM5" rid="MOESM5" class=" media">5</a>: Fig. S5A&#x02013;D) with the majority of the covariance (CV) of the measured peptide concentration below 10% (Fig.&#x000a0;<a href="/pmc/articles/PMC10880210/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5</span></a>E). Using concentrations of the seven proteins to build a logistic regression model, we calculated a classification score for each patient. The score was calculated using the following equation:</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig5" co-legend-rid="lgnd_Fig5"><a href="/pmc/articles/PMC10880210/figure/Fig5/" target="figure" rid-figpopup="Fig5" rid-ob="ob-Fig5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139959025626912"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10880210_12967_2024_4983_Fig5_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 12967_2024_4983_Fig5_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10880210/bin/12967_2024_4983_Fig5_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139959025626912"><a target="object" rel="noopener" href="/pmc/articles/PMC10880210/figure/Fig5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig5"><div><a class="figpopup" href="/pmc/articles/PMC10880210/figure/Fig5/" target="figure" rid-figpopup="Fig5" rid-ob="ob-Fig5">Fig.&#x000a0;5</a></div><!--caption a7--><div class="caption"><p>Expression patterns of the protein biomarkers in different CRC stages. <strong>A</strong> Box plot of plasma concentration of the seven proteins in four different CRC stages. <strong>B</strong> Sensitivity of the seven-protein biomarker in distinguishing CRC at four different stages from healthy subjects. <strong>C</strong> Sensitivity of the seven-protein biomarker in distinguishing APC with different sizes from healthy subjects. <strong>D</strong> Sensitivity of the seven-protein biomarker in identifying precancerous lesions with different grades</p></div></div></div><p id="Par30">log(P/(1-P)&#x02009;=&#x02009;-7.8709&#x02009;+&#x02009;4.5956&#x02009;&#x000d7;&#x02009;IGFBP2&#x02009;+&#x02009;0.2732&#x02009;&#x000d7;&#x02009;ITIH3&#x02009;+&#x02009;0.0909&#x02009;&#x000d7;&#x02009;LRG1&#x02009;+&#x02009;0.1015&#x02009;&#x000d7;&#x02009;C9 -3.2205&#x02009;&#x000d7;&#x02009;CNDP1&#x02009;+&#x02009;0.0239&#x02009;&#x000d7;&#x02009;ORM1&#x02009;+&#x02009;0.0811&#x02009;&#x000d7;&#x02009;SERPINA1, where P is a value between 0 and 1 that represents the probability of the event (colorectal cancer). The optimal cut-off value for the classifier was 0.318.</p><p id="Par31">This logistic regression classifier achieved an average area under the curve (AUC) of the receiver operator characteristic (ROC) curve of 0.954 (95% CI 0.915&#x02013;0.994) in the training dataset and 0.958 (95% CI 0.883&#x02013;1.0) in the testing dataset (Fig.&#x000a0;<a href="/pmc/articles/PMC10880210/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4</span></a>A). Furthermore, we applied the MRM assay to quantify the concentration of the seven proteins in plasma samples collected from an independent validation cohort from different hospitals (Additional file <a href="#MOESM9" rid="MOESM9" class=" media">9</a>: Table S4). Applying the locked logistic regression parameters to calculate the likelihood of CRC and the fixed cutoff probability value of 0.318, we achieved an average AUC of 0.905 (Fig.&#x000a0;4B, 95% CI 0.864&#x02013;0.946). We enrolled another independent validation cohort in a blinded fashion, in which the diagnosis of the patients was disclosed only after the classification was determined (Additional file <a href="#MOESM9" rid="MOESM9" class=" media">9</a>: Table S4). We achieved an average AUC of 0.909 using the same assay and the locked logistic regression parameters and probability cutoff value (Fig.&#x000a0;4C, 95% CI 0.827&#x02013;0.99).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig4" co-legend-rid="lgnd_Fig4"><a href="/pmc/articles/PMC10880210/figure/Fig4/" target="figure" rid-figpopup="Fig4" rid-ob="ob-Fig4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139958983326992"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10880210_12967_2024_4983_Fig4_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 12967_2024_4983_Fig4_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10880210/bin/12967_2024_4983_Fig4_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139958983326992"><a target="object" rel="noopener" href="/pmc/articles/PMC10880210/figure/Fig4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig4"><div><a class="figpopup" href="/pmc/articles/PMC10880210/figure/Fig4/" target="figure" rid-figpopup="Fig4" rid-ob="ob-Fig4">Fig.&#x000a0;4</a></div><!--caption a7--><div class="caption"><p>MRM quantification and logistic regression classification of CRC and healthy subjects. <strong>A</strong> ROC curves of a seven-protein logistic regression classifier (LRG1, C9, IGFBP2, CDNP1, ITIH3, SERPINA1, and ORM1) for distinguishing CRC and healthy subjects in the training and testing datasets. <strong>B</strong> ROC curve showing the performance of the seven-protein classifier in distinguishing CRC and healthy subjects in an independent validation cohort. <strong>C</strong> ROC curve showing the performance of the seven-protein classifier in distinguishing CRC and healthy subjects in a blinded validation cohort. <strong>D</strong>&#x02013;<strong>F</strong> Confusion matrix showing the classification accuracy in the training (<strong>D</strong>), independent validation <strong>E</strong>, and blinded validation <strong>F</strong> cohorts</p></div></div></div><p id="Par32" class="p p-last">Based on the confusion matrix, the assay had a sensitivity, specificity, PPV, and NPV of 93.3%, 80.0%, 82.4%, and 92.3% respectively in the training cohort (Fig.&#x000a0;<a href="/pmc/articles/PMC10880210/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4</span></a>D), and 89.9%, 81.8%, 89.2%, and 82.9% respectively in the validation cohort (Fig.&#x000a0;<a href="/pmc/articles/PMC10880210/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4</span></a>E). In the blinded validation cohort, the sensitivity, specificity, PPV, and NPV were 81.5%, 97.9%, 95.6%, and 90.4% respectively (Fig.&#x000a0;<a href="/pmc/articles/PMC10880210/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4</span></a>F); the classification accuracy was 92.0% (Table&#x000a0;<a href="/pmc/articles/PMC10880210/table/Tab2/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab2" rid-ob="ob-Tab2" co-legend-rid=""><span>2</span></a>B). Overall, the sensitivity of detecting CRC achieved 91.0% with a specificity of 81.0% (Table&#x000a0;<a href="/pmc/articles/PMC10880210/table/Tab2/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab2" rid-ob="ob-Tab2" co-legend-rid=""><span>2</span></a>A).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="Tab2"><h3>Table&#x000a0;2</h3><!--caption a7--><div class="caption"><p>Sensitivity and specificity of the protein classifier panel for detecting colorectal cancer and advanced precancerous lesions</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="left" rowspan="1" colspan="1"></th><th align="left" rowspan="1" colspan="1">Colonoscopy</th><th align="left" rowspan="1" colspan="1">7-Protein biomarker</th><th align="left" rowspan="1" colspan="1"></th></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1">A</td><td align="left" rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1">Positive result</td><td align="left" rowspan="1" colspan="1">Sensitivity</td></tr><tr><td align="left" rowspan="1" colspan="1">Cancer (CRC)</td><td align="left" rowspan="1" colspan="1">189</td><td align="left" rowspan="1" colspan="1">172</td><td align="left" rowspan="1" colspan="1">91.0%</td></tr><tr><td align="left" rowspan="1" colspan="1">Advanced precancerous lesions (APC)</td><td align="left" rowspan="1" colspan="1">47</td><td align="left" rowspan="1" colspan="1">23</td><td align="left" rowspan="1" colspan="1">49.0%</td></tr><tr><td align="left" rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1">Specificity</td></tr><tr><td align="left" rowspan="1" colspan="1">Negative (H)</td><td align="left" rowspan="1" colspan="1">137</td><td align="left" rowspan="1" colspan="1">26</td><td align="left" rowspan="1" colspan="1">81.0%</td></tr><tr><td align="left" rowspan="1" colspan="1">B</td><td align="left" rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1">Positive result</td><td align="left" rowspan="1" colspan="1">Sensitivity</td></tr><tr><td align="left" rowspan="1" colspan="1">Cancer (CRC)</td><td align="left" rowspan="1" colspan="1">27</td><td align="left" rowspan="1" colspan="1">22</td><td align="left" rowspan="1" colspan="1">81.5%</td></tr><tr><td align="left" rowspan="1" colspan="1">High-grade dysplasia</td><td align="left" rowspan="1" colspan="1">9</td><td align="left" rowspan="1" colspan="1">4</td><td align="left" rowspan="1" colspan="1">44.4%</td></tr><tr><td align="left" rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1">Specificity</td></tr><tr><td align="left" rowspan="1" colspan="1">Negative (H)</td><td align="left" rowspan="1" colspan="1">48</td><td align="left" rowspan="1" colspan="1">1</td><td align="left" rowspan="1" colspan="1">97.9%</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139958986299520"><a target="object" rel="noopener" href="/pmc/articles/PMC10880210/table/Tab2/?report=objectonly">Open in a separate window</a></div></div></div><div id="Sec16" class="sec sec-last"><h3 id="Sec16title">Expression patterns of the seven protein biomarkers at different CRC stages</h3><p id="Par33" class="p p-first-last">We further analyzed the capability of the seven-protein panel to distinguish CRC at different stages. In general, each individual protein showed significantly different expression levels in every pathological stage compared to healthy subjects; however, there were no obvious statistical difference between stages II&#x02009;~&#x02009;IV (Fig.&#x000a0;<a href="/pmc/articles/PMC10880210/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5</span></a>A). Nevertheless, there were significant differences between later stages and stage I in proteins IGFBP2, C9, SERPINA1, and ORM1 (Additional file <a href="#MOESM10" rid="MOESM10" class=" media">10</a>: Table S5). The sensitivity of our protein panel reached over 90% in detecting CRC at all stages; however, there was a clear pattern of increased sensitivity in later stages (II&#x02009;~&#x02009;IV) than stage I (Fig.&#x000a0;<a href="/pmc/articles/PMC10880210/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5</span></a>B). The sensitivity of detecting APC was around 40% for lesions smaller than 1 cm or between 1 and 1.5 cm, but increased to nearly 60% for lesions greater than 1.5 cm (Fig.&#x000a0;<a href="/pmc/articles/PMC10880210/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5</span></a>C). This result showed an increased sensitivity in detecting APC than a published DNA test [<a href="#CR14" rid="CR14" class=" bibr popnode">14</a>]. Finally, the sensitivity of detecting colorectal polyps of distinct anatomic morphology was 50% for sessile serrated polyps, 100% for tubulovillous adenoma and high-grade dysplasia (Fig.&#x000a0;<a href="/pmc/articles/PMC10880210/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5</span></a>D), all of which are considered as having high probability of transforming into cancer. Thus, our panel of protein biomarkers can not only detect CRC at early stages, but also detect highly malignant adenomas.</p></div></div><div id="Sec17" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec17title">Discussion</h2><p id="Par34" class="p p-first">The biological functions of many of the biomarker proteins identified in this study have been documented. Overall, they play distinct roles in cancer development and progression. For example, LRG1 is a cell adhesion molecule whose up-regulation is involved in tumor metastasis [<a href="#CR32" rid="CR32" class=" bibr popnode">32</a>], and has recently been shown to be valuable in diagnosis of colorectal cancer and pancreatic cancer [<a href="#CR16" rid="CR16" class=" bibr popnode">16</a>, <a href="#CR33" rid="CR33" class=" bibr popnode">33</a>]. As a complement component that is linked to inflammation and immune response, C9 is unlikely to provide specific information predicting tumor formation. Nevertheless, a recent study found aberrant expression of C9 in the plasma of CRC patients [<a href="#CR34" rid="CR34" class=" bibr popnode">34</a>]. IGFBP2 is a node of insulin signaling and has been shown to have prognostic value in multiple cancers in a meta-analysis [<a href="#CR35" rid="CR35" class=" bibr popnode">35</a>], including metastatic CRC [<a href="#CR36" rid="CR36" class=" bibr popnode">36</a>]. Similarly, ORM1, SERPINA1, and ITIH3 has all been shown to be valuable predicting clinical outcomes of CRC patients [<a href="#CR31" rid="CR31" class=" bibr popnode">31</a>, <a href="#CR37" rid="CR37" class=" bibr popnode">37</a>, <a href="#CR38" rid="CR38" class=" bibr popnode">38</a>]. Down-regulation of CNDP1 was associated with cancer cachexia [<a href="#CR39" rid="CR39" class=" bibr popnode">39</a>], and it was recently reported that CNDP1 level was significantly reduced in hepatocellular carcinoma tissues [<a href="#CR40" rid="CR40" class=" bibr popnode">40</a>]. Although these studies focused on different cancer types, they all point to CNDP1 down-regulation in cancers, which are consistent with our study. By accurately measuring the plasma concentration of these proteins, we could predict CRC at an early stage using logistic regression.</p><p id="Par35">There have been numerous attempts to develop protein-based liquid biopsy assays for detection of CRC. A noticeable study used proximity extension assay (Olink) to measure plasma concentration of 92 proteins from 89 subjects, and identified an eight-protein biomarker with an adjusted AUC of 0.77 and a sensitivity of 0.44 at 90% specificity in the validation set [<a href="#CR41" rid="CR41" class=" bibr popnode">41</a>]. Because of the high sensitivity of Olink technology [<a href="#CR42" rid="CR42" class=" bibr popnode">42</a>], it is able to quantify plasma concentration of very-low abundant proteins such as growth factors, cytokines, and tumor antigens, none of which overlapped with our panel components. However, the resulting performance of predicting CRC remained unsatisfactory, presumably due to the overfitting issue of measuring large number of proteins. Another study used mass spectrometry to identify a five-protein biomarker signature to effectively distinguish CRC from control in a training cohort of 200 cases and a validation cohort of 269 cases, with an AUC of 0.84 and an overall accuracy of 72% [<a href="#CR43" rid="CR43" class=" bibr popnode">43</a>]. Of note is that the five-protein panel includes LRG1, one of the feature protein in our seven-protein panel. Another protein in this five-protein panel is SERPINA3, which is a close relative to SERPINA1 in our seven-protein panel. This indicates that mass spectrometry tends to detect proteins at similar expression levels. Compard to these studies, our assay of seven-protein panel resulted an AUC of 0.905 and 0.909 in two independent validation cohorts, providing a much-improved performance. Even compared to a study that combines DNA test and fecal immune test which provided a sensitivity of detecting CRC at 92.3% and specificity of 86.6% [<a href="#CR14" rid="CR14" class=" bibr popnode">14</a>], our assay provides a slightly lower but comparable sensitivity (91.0%) and specificity (81.0%) (Table&#x000a0;<a href="/pmc/articles/PMC10880210/table/Tab2/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab2" rid-ob="ob-Tab2" co-legend-rid=""><span>2</span></a>A). In addition to detecting cancers, there is a growing need to identify colorectal adenomas using protein biomarkers in the blood. A recent study found that serum biomarkers F5, ITIH4, LRG1, and VTN were elevated in colorectal adenoma patients as well as in a mouse model of colorectal adenomas [<a href="#CR16" rid="CR16" class=" bibr popnode">16</a>]. From colonoscopy-confirmed patients with advanced precancerous lesions, the average sensitivity of our assay achieved 49% (Table&#x000a0;<a href="/pmc/articles/PMC10880210/table/Tab2/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab2" rid-ob="ob-Tab2" co-legend-rid=""><span>2</span></a>B), better than aforementioned study. Our assay also showed a clear improvement compared to the DNA test which provided a sensitivity of 42.4% to detect APC [<a href="#CR14" rid="CR14" class=" bibr popnode">14</a>].</p><p id="Par36">From our quantitative proteomic results, it appears that DDA has more upregulated proteins and DIA has more of downregulated proteins. Because DDA is data dependent, it favors detection and quantification of relatively abundant peptides. DIA is utilizing homogeneous scanning over the entire mass range with a wider mass window on each scan, and detects and quantifies peptides regardless of their abundance. Therefore, DIA is capable of detecting low abundance peptides and quantifying more proteins. It is also possible that the differentially expressed proteins in DDA and DIA experiments might distribute with different patterns. Regardless, six out of the seven significantly changed marker proteins in both DDA and DIA experiments showed consistent direction of change (Fig.&#x000a0;<a href="/pmc/articles/PMC10880210/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span>3</span></a>A, B). We believe that DDA and DIA are complementary, and there no black-and-white answer as to which one is more trustworthy over the other. Largely due to the stochastic nature of mass spectrometry data acquisition, quantifying large number of peptides may results in certain degree of error rate.</p><p id="Par37">For the profiling experiments in discovery phase, we took the conventional approach to remove the 14 most abundant plasma proteins using the antibody-based depletion kit. Depletion of abundant proteins can reduce the interference from these proteins during LC&#x02013;MS/MS data acquisition and improve the depth of the coverage of the plasma proteome [<a href="#CR44" rid="CR44" class=" bibr popnode">44</a>]. However, because it introduces more sample processing steps, it considerably affects the reproducibility of the experiment. This may be further exaggerated when dealing with large number of clinical samples. Therefore, during target validation phase of the MRM assay development, we innovatively simplified the sample preparation process by omitting the depletion step. Of the proteins we measured, the plasma concentrations ranged between 1&#x000a0;ng/ml for CNDP1 and 120&#x000a0;ng/ml for ORM1, and we obtained decent product ion signals in all seven peptides (Additional file <a href="#MOESM5" rid="MOESM5" class=" media">5</a>: Fig. S5A&#x02013;D). The results also point to acceptable precision in clinical settings even though the experiments were performed manually, as demonstrated by lower than 10% CV in QC samples for the majority of the proteins measured (Additional file <a href="#MOESM4" rid="MOESM4" class=" media">4</a>: Fig. S4E). Only the concentration of IGFBP2 showed a CV of over 10%, presumably due to its extremely-low abundance. On the other hand, ORM1 is one of 14 high-abundance proteins targeted for depletion; removing it runs the risk of removing a potentially meaningful biomarker candidate. With our simplified sample processing method, we envision that by incorporating automation in the future, we will further improve the precision and accuracy of our measurement.</p><p id="Par38">The strength of this study lies in several technical and strategical advantages. First, many previous CRC protein biomarker studies started from comprehensive literature search for candidate proteins, and then developed quantitative assays to build models and further validated the model. This approach may be difficult to discover new biomarkers. In our study, we started from an unbiased quantitative analysis of plasma proteome using the state-of-art mass spectrometry technology to discover candidate proteins; this study design facilitated MRM assay development in the validation phase. Second, after establishing a machine learning model based on the training cohort of 120 subjects, we enrolled two independent patient cohorts of 253 and 84 subjects from three different hospitals to validate the model. In particular, the second validation cohort were performed in a blinded fashion, moving one step closer to a prospective study. Noticeably, in protein-based biomarker studies, the cohort size in our study is reasonably large. Third, when applying the MRM assay and the &#x0201c;locked&#x0201d; logistic regression model parameters on both independent validation cohorts, our assay maintained an accuracy of over 90%. Taking into consideration the CRC prevalence of around 0.25% in Chinese population [<a href="#CR45" rid="CR45" class=" bibr popnode">45</a>] with the sensitivity of 81.5% and specificity of 97.9%, the calculated NPV of our assay can reach 99.95%. The sensitivity and specificity of our assay compare favorably to existing clinical diagnosis and screening methods including stool FOBT/FIT [<a href="#CR5" rid="CR5" class=" bibr popnode">5</a>], blood and stool DNA methylation test [<a href="#CR6" rid="CR6" class=" bibr popnode">6</a>, <a href="#CR7" rid="CR7" class=" bibr popnode">7</a>], and performs similarly to an FDA approved, stool-based DNA test (Cologuard) [<a href="#CR14" rid="CR14" class=" bibr popnode">14</a>], well suited for early screen. Fourth, because sampling blood is more convenient than procuring stool samples and its non-invasive nature surpasses colonoscopy, patient compliance is expected to be much better than conventional diagnostic methods. If applied as an early screening method, it can prevent a significant number of medium-to-high risk subjects from invasive procedures. Finally, our plasma sample processing approach omitted the use of expensive depletion kit, which is not only technically advantageous but also has the benefit of significantly cutting the cost of the assay. This advantage could be further augmented if the assay is applied in settings involving population screen.</p><p id="Par39" class="p p-last">One limitation of this study is the relatively sub-optimal sample size. Another limitation is that the subjects were recruited from hospital patients; whereas in a real-world screening test, the subjects would come from the population undergone preventive physical examination and thus the number of cancer patients would have been much less. Further refinement and validation of this panel of biomarker proteins in a population-wide scenario for a clinical trial is warranted.</p></div><div id="Sec18" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec18title">Conclusion</h2><p id="Par40" class="p p-first-last">We have developed a non-invasive, targeted mass spectrometry assay to measure plasma concentrations of seven proteins that is capable of distinguishing colorectal cancer from healthy subjects. This seven-protein classifier is of translational value and warrants further development into a clinically useful assay.</p></div><div id="sec-a.w.f" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="headless nomenu"></h2><!--/article/body/sec/--><div id="Sec19" class="sec sec-first-last"><h3 id="Sec19title">Supplementary Information</h3><p class="p p-first-last">
</p><div class="sec suppmat" id="MOESM1"><div class="sup-box half_rhythm" id="media-a.w.f.a.b.a.a"><a href="/pmc/articles/PMC10880210/bin/12967_2024_4983_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl"><strong>Additional file 1: Figure S1. </strong>Quality of the DDA and DIA data. (A) Correlation analysis of the quality control (QC) samples from DDA data. (B) Correlation analysis of the quality control (QC) samples from DIA data. (C) Distribution of protein abundance of all quantified proteins from DDA data. (D) Distribution of protein abundance of all quantified proteins from DIA data. (E) Number of quantified proteins in all samples from DDA data. (F) Number of quantified proteins in all samples from DIA data.</a><sup>(940K, pdf)</sup></div></div><div class="sec suppmat" id="MOESM2"><div class="sup-box half_rhythm" id="media-a.w.f.a.b.b.a"><a href="/pmc/articles/PMC10880210/bin/12967_2024_4983_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl"><strong>Additional file 2: Figure S2. </strong>Heatmap showing KEGG pathways of up- and down-regulated proteins in the plasma from CRC patients.</a><sup>(39K, pdf)</sup></div></div><div class="sec suppmat" id="MOESM3"><div class="sup-box half_rhythm" id="media-a.w.f.a.b.c.a"><a href="/pmc/articles/PMC10880210/bin/12967_2024_4983_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl"><strong>Additional file 3: Figure. S3. </strong>Selection of protein panels to classify CRC from healthy subjects. (A) Mean decrease of Gini index in DDA data. (B) Mean decrease of Gini index in DIA data.</a><sup>(657K, pdf)</sup></div></div><div class="sec suppmat" id="MOESM4"><div class="sup-box half_rhythm" id="media-a.w.f.a.b.d.a"><a href="/pmc/articles/PMC10880210/bin/12967_2024_4983_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl"><strong>Additional file 4: Figure S4. </strong>Peptide sequence and ion mass information of selected protein markers.</a><sup>(12K, pdf)</sup></div></div><div class="sec suppmat" id="MOESM5"><div class="sup-box half_rhythm" id="media-a.w.f.a.b.e.a"><a href="/pmc/articles/PMC10880210/bin/12967_2024_4983_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl"><strong>Additional file 5: Figure S5. </strong>Quality of the MRM data. (A&#x02013;D). Extracted chromatograms of the 7-peptide biomarkers. (E) Variable coefficient values in QC samples for the 7-peptide biomarkers.</a><sup>(126K, pdf)</sup></div></div><div class="sec suppmat" id="MOESM6"><div class="sup-box half_rhythm" id="media-a.w.f.a.b.f.a"><a href="/pmc/articles/PMC10880210/bin/12967_2024_4983_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl"><strong>Additional file 6: Table S1. </strong>Demographic data of all the patients enrolled in MRM experiments.</a><sup>(9.7K, xlsx)</sup></div></div><div class="sec suppmat" id="MOESM7"><div class="sup-box half_rhythm" id="media-a.w.f.a.b.g.a"><a href="/pmc/articles/PMC10880210/bin/12967_2024_4983_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl"><strong>Additional file 7: Table S2. </strong>Protein expression data from DDA experiments.</a><sup>(637K, xlsx)</sup></div></div><div class="sec suppmat" id="MOESM8"><div class="sup-box half_rhythm" id="media-a.w.f.a.b.h.a"><a href="/pmc/articles/PMC10880210/bin/12967_2024_4983_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl"><strong>Additional file 8: Table S3. </strong>Protein expression data from DIA experiments.</a><sup>(486K, xlsx)</sup></div></div><div class="sec suppmat" id="MOESM9"><div class="sup-box half_rhythm" id="media-a.w.f.a.b.i.a"><a href="/pmc/articles/PMC10880210/bin/12967_2024_4983_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl"><strong>Additional file 9: Table S4. </strong>Concentration of each protein of the seven-protein panel measured by MRM in all subjects.</a><sup>(66K, xlsx)</sup></div></div><div class="sec suppmat" id="MOESM10"><div class="sup-box half_rhythm" id="media-a.w.f.a.b.j.a"><a href="/pmc/articles/PMC10880210/bin/12967_2024_4983_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl"><strong> Additional file 10: Table S5. </strong>Concentration and statistical analysis of each protein of the seven-protein panel measured by MRM in CRC cases shown in Figure 4, D and E.</a><sup>(27K, xlsx)</sup></div></div><p></p></div></div><div id="ack-a.x.b" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.x.btitle">Acknowledgements</h2><div class="sec"><p>We cordially thank Antony Y. Liao for carefully proofreading and editing the manuscript.</p></div></div><div id="notes-a.x.c" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.x.ctitle">Author contributions</h2><p>LL, TW and FG conceptualized the study; TW, LP, XD performed the experiments; HH, TX, ZF, LG, FG, YC, ZS participated the study; LP and TW analyzed the data; HH, LL, ZS provided part of the funding support; LL, FG supervised the entire study; LL, TW and FG wrote the manuscript.</p></div><div id="notes-a.x.d" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.x.dtitle">Funding</h2><p>This work was funded by Durbrain Medical Laboratory, by the National Natural Science Foundation of China (# 82170533), and by Zhejiang Provincial Program for the Cultivation of High-level Innovative Health Talents (# 2021C03115).</p></div><div id="notes-a.x.e" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.x.etitle">Availability of data and materials</h2><p>The datasets generated (mass spectrometry raw data) during the current study has been deposited to the ProteomeXchange repository with the accession numbers PXD042639 and PXD042652 (<a href="https://proteomecentral.proteomexchange.org/cgi/GetDataset" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://proteomecentral.proteomexchange.org/cgi/GetDataset</a>). The analyzed data is included in the supplementary tables. The clinical samples and analytical methods will not be available to the public.</p></div><div id="notes-a.x.f" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.x.ftitle">Declarations</h2><div>Ethics approval and consent to participate<p id="Par41">The study conformed to the Declaration of Helsinki and was approved by the Medical Ethics Committee of Shanghai Tenth People's Hospital (Study license number SHDSYY-2020-3645), the Medical Ethics Committee of Changzhou Wujin People&#x02019;s Hospital (Study license number 2022-SR-091), and the Medical Ethics Committee of the First Affiliated Hospital of Zhejiang University (Study license number IIT20230142B).</p></div><div>Consent for publication<p id="Par42">Not applicable.</p></div><div>Competing interests<p id="Par43">The authors LL, TW, FG declare that they have competing interests, and a patent related to this study has been filed. Other authors declare that there is no conflict of interest.</p></div></div><div id="fn-group-a.x.a" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.x.atitle">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="fn-a.x.a.a"><p class="p p-first"><strong>Publisher's Note</strong></p><p class="p p-last">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></p><p class="fn sec" id="fn-a.x.a.b"><p class="p p-first-last">Hanju Hua and Tingting Wang contributed equally to this work.</p></p></div></div><div id="article-aaff-info" class="tsec sec"><h2 class="head no_bottom_margin" id="article-aaff-infotitle">Contributor Information</h2><p><span class="fm-affl">Chaohui Yu, </span><span class="fm-affl"><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="nc.ude.ujz@hcyyyyz" class="oemail">nc.ude.ujz@hcyyyyz</a></span>.</p><p><span class="fm-affl">Feng Gao, </span><span class="fm-affl"><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="moc.361@00ekacym" class="oemail">moc.361@00ekacym</a></span>.</p><p><span class="fm-affl">Lujian Liao, </span><span class="fm-affl"><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="nc.ude.unce.oib@oailjl" class="oemail">nc.ude.unce.oib@oailjl</a></span>.</p><p><span class="fm-affl">Zhe Shen, </span><span class="fm-affl"><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="nc.ude.ujz@rdznehs" class="oemail">nc.ude.ujz@rdznehs</a></span>.</p></div><div id="Bib1" class="tsec sec"><h2 class="head no_bottom_margin" id="Bib1title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="CR1">1. <span class="element-citation">Siegel RL, Wagle NS, Cercek A, Smith RA, Jemal A. Colorectal cancer statistics, 2023. <span><span class="ref-journal">CA Cancer J Clin. </span>2023 doi: 10.3322/caac.21772.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36856579" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3322%2Fcaac.21772" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=CA+Cancer+J+Clin&amp;title=Colorectal+cancer+statistics,+2023&amp;author=RL+Siegel&amp;author=NS+Wagle&amp;author=A+Cercek&amp;author=RA+Smith&amp;author=A+Jemal&amp;publication_year=2023&amp;pmid=36856579&amp;doi=10.3322/caac.21772&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR2">2. <span class="element-citation">Ahn SB, et al. Potential early clinical stage colorectal cancer diagnosis using a proteomics blood test panel. <span><span class="ref-journal">Clin Proteomics. </span>2019;<span class="ref-vol">16</span>:34. doi: 10.1186/s12014-019-9255-z.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6712843/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31467500" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2Fs12014-019-9255-z" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Proteomics&amp;title=Potential+early+clinical+stage+colorectal+cancer+diagnosis+using+a+proteomics+blood+test+panel&amp;author=SB+Ahn&amp;volume=16&amp;publication_year=2019&amp;pages=34&amp;pmid=31467500&amp;doi=10.1186/s12014-019-9255-z&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR3">3. <span class="element-citation">Moreno CC, et al. Colorectal cancer initial diagnosis: screening colonoscopy, diagnostic colonoscopy, or emergent surgery, and tumor stage and size at initial presentation. <span><span class="ref-journal">Clin Colorectal Cancer. </span>2016;<span class="ref-vol">15</span>:67–73. doi: 10.1016/j.clcc.2015.07.004.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26602596" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.clcc.2015.07.004" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Colorectal+Cancer&amp;title=Colorectal+cancer+initial+diagnosis:+screening+colonoscopy,+diagnostic+colonoscopy,+or+emergent+surgery,+and+tumor+stage+and+size+at+initial+presentation&amp;author=CC+Moreno&amp;volume=15&amp;publication_year=2016&amp;pages=67-73&amp;pmid=26602596&amp;doi=10.1016/j.clcc.2015.07.004&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR4">4. <span class="element-citation">Shaukat A, Levin TR. Current and future colorectal cancer screening strategies. <span><span class="ref-journal">Nat Rev Gastroenterol Hepatol. </span>2022;<span class="ref-vol">19</span>:521–531. doi: 10.1038/s41575-022-00612-y.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9063618/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35505243" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41575-022-00612-y" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Gastroenterol+Hepatol&amp;title=Current+and+future+colorectal+cancer+screening+strategies&amp;author=A+Shaukat&amp;author=TR+Levin&amp;volume=19&amp;publication_year=2022&amp;pages=521-531&amp;pmid=35505243&amp;doi=10.1038/s41575-022-00612-y&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR5">5. <span class="element-citation">Chido-Amajuoyi OG, Sharma A, Talluri R, Tami-Maury I, Shete S. Physician-office vs home uptake of colorectal cancer screening using FOBT/FIT among screening-eligible US adults. <span><span class="ref-journal">Cancer Med. </span>2019;<span class="ref-vol">8</span>:7408–7418. doi: 10.1002/cam4.2604.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6885889/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31637870" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fcam4.2604" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Med&amp;title=Physician-office+vs+home+uptake+of+colorectal+cancer+screening+using+FOBT/FIT+among+screening-eligible+US+adults&amp;author=OG+Chido-Amajuoyi&amp;author=A+Sharma&amp;author=R+Talluri&amp;author=I+Tami-Maury&amp;author=S+Shete&amp;volume=8&amp;publication_year=2019&amp;pages=7408-7418&amp;pmid=31637870&amp;doi=10.1002/cam4.2604&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR6">6. <span class="element-citation">Wang L, et al. Diagnostic accuracy of DNA-based SDC2 methylation test in colorectal cancer screening: a meta-analysis. <span><span class="ref-journal">BMC Gastroenterol. </span>2022;<span class="ref-vol">22</span>:314. doi: 10.1186/s12876-022-02395-7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9235166/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35754025" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2Fs12876-022-02395-7" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Gastroenterol&amp;title=Diagnostic+accuracy+of+DNA-based+SDC2+methylation+test+in+colorectal+cancer+screening:+a+meta-analysis&amp;author=L+Wang&amp;volume=22&amp;publication_year=2022&amp;pages=314&amp;pmid=35754025&amp;doi=10.1186/s12876-022-02395-7&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR7">7. <span class="element-citation">Lu P, et al. Methylated septin 9 as a promising biomarker in the diagnosis and recurrence monitoring of colorectal cancer. <span><span class="ref-journal">Dis Markers. </span>2022;<span class="ref-vol">2022</span>:7087885. doi: 10.1155/2022/7087885.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9271001/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35818587" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1155%2F2022%2F7087885" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dis+Markers&amp;title=Methylated+septin+9+as+a+promising+biomarker+in+the+diagnosis+and+recurrence+monitoring+of+colorectal+cancer&amp;author=P+Lu&amp;volume=2022&amp;publication_year=2022&amp;pages=7087885&amp;pmid=35818587&amp;doi=10.1155/2022/7087885&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR8">8. <span class="element-citation">Rao H, Wu H, Huang Q, Yu Z, Zhong Z. Clinical Value of Serum CEA, CA24&#x02013;2 and CA19&#x02013;9 in patients with colorectal cancer. <span><span class="ref-journal">Clin Lab. </span>2021 doi: 10.7754/Clin.Lab.2020.200828.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33865243" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.7754%2FClin.Lab.2020.200828" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Lab&amp;title=Clinical+Value+of+Serum+CEA,+CA24&#x02013;2+and+CA19&#x02013;9+in+patients+with+colorectal+cancer&amp;author=H+Rao&amp;author=H+Wu&amp;author=Q+Huang&amp;author=Z+Yu&amp;author=Z+Zhong&amp;publication_year=2021&amp;pmid=34542971&amp;doi=10.7754/Clin.Lab.2020.200828&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR9">9. <span class="element-citation">Luo H, et al. Clinical significance and diagnostic value of serum NSE, CEA, CA19-9, CA125 and CA242 levels in colorectal cancer. <span><span class="ref-journal">Oncol Lett. </span>2020;<span class="ref-vol">20</span>:742–750. doi: 10.3892/ol.2020.11633.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7286116/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32566000" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3892%2Fol.2020.11633" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncol+Lett&amp;title=Clinical+significance+and+diagnostic+value+of+serum+NSE,+CEA,+CA19-9,+CA125+and+CA242+levels+in+colorectal+cancer&amp;author=H+Luo&amp;volume=20&amp;publication_year=2020&amp;pages=742-750&amp;pmid=32566000&amp;doi=10.3892/ol.2020.11633&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR10">10. <span class="element-citation">Ahmadian E, Janas D, Eftekhari A, Zare N. Application of carbon nanotubes in sensing/monitoring of pancreas and liver cancer. <span><span class="ref-journal">Chemosphere. </span>2022;<span class="ref-vol">302</span>:134826. doi: 10.1016/j.chemosphere.2022.134826.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35525455" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.chemosphere.2022.134826" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chemosphere&amp;title=Application+of+carbon+nanotubes+in+sensing/monitoring+of+pancreas+and+liver+cancer&amp;author=E+Ahmadian&amp;author=D+Janas&amp;author=A+Eftekhari&amp;author=N+Zare&amp;volume=302&amp;publication_year=2022&amp;pages=134826&amp;pmid=35525455&amp;doi=10.1016/j.chemosphere.2022.134826&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR11">11. <span class="element-citation">Nakamura K, et al. A liquid biopsy signature for the detection of patients with early-onset colorectal cancer. <span><span class="ref-journal">Gastroenterology. </span>2022;<span class="ref-vol">163</span>:1242–1251. doi: 10.1053/j.gastro.2022.06.089.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9613521/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35850198" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1053%2Fj.gastro.2022.06.089" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=A+liquid+biopsy+signature+for+the+detection+of+patients+with+early-onset+colorectal+cancer&amp;author=K+Nakamura&amp;volume=163&amp;publication_year=2022&amp;pages=1242-1251&amp;pmid=35850198&amp;doi=10.1053/j.gastro.2022.06.089&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR12">12. <span class="element-citation">Brenne SS, et al. Colorectal cancer detected by liquid biopsy 2 years prior to clinical diagnosis in the HUNT study. <span><span class="ref-journal">Br J Cancer. </span>2023;<span class="ref-vol">129</span>:861–868. doi: 10.1038/s41416-023-02337-4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10449868/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37438612" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41416-023-02337-4" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Cancer&amp;title=Colorectal+cancer+detected+by+liquid+biopsy+2+years+prior+to+clinical+diagnosis+in+the+HUNT+study&amp;author=SS+Brenne&amp;volume=129&amp;publication_year=2023&amp;pages=861-868&amp;pmid=37438612&amp;doi=10.1038/s41416-023-02337-4&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR13">13. <span class="element-citation">Warren JD, et al. Septin 9 methylated DNA is a sensitive and specific blood test for colorectal cancer. <span><span class="ref-journal">BMC Med. </span>2011;<span class="ref-vol">9</span>:133. doi: 10.1186/1741-7015-9-133.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3271041/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22168215" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2F1741-7015-9-133" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med&amp;title=Septin+9+methylated+DNA+is+a+sensitive+and+specific+blood+test+for+colorectal+cancer&amp;author=JD+Warren&amp;volume=9&amp;publication_year=2011&amp;pages=133&amp;pmid=22168215&amp;doi=10.1186/1741-7015-9-133&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR14">14. <span class="element-citation">Imperiale TF, et al. Multitarget stool DNA testing for colorectal-cancer screening. <span><span class="ref-journal">N Engl J Med. </span>2014;<span class="ref-vol">370</span>:1287–1297. doi: 10.1056/NEJMoa1311194.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24645800" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1056%2FNEJMoa1311194" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Multitarget+stool+DNA+testing+for+colorectal-cancer+screening&amp;author=TF+Imperiale&amp;volume=370&amp;publication_year=2014&amp;pages=1287-1297&amp;pmid=24645800&amp;doi=10.1056/NEJMoa1311194&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR15">15. <span class="element-citation">Zhan Y, et al. The stool syndecan2 methylation test is more robust than blood tests for methylated septin9, CEA, CA19-9 and CA724: a diagnostic test for the early detection of colorectal neoplasms. <span><span class="ref-journal">Transl Cancer Res. </span>2023;<span class="ref-vol">12</span>:65–77. doi: 10.21037/tcr-22-1710.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9906059/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36760372" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.21037%2Ftcr-22-1710" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transl+Cancer+Res&amp;title=The+stool+syndecan2+methylation+test+is+more+robust+than+blood+tests+for+methylated+septin9,+CEA,+CA19-9+and+CA724:+a+diagnostic+test+for+the+early+detection+of+colorectal+neoplasms&amp;author=Y+Zhan&amp;volume=12&amp;publication_year=2023&amp;pages=65-77&amp;pmid=36760372&amp;doi=10.21037/tcr-22-1710&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR16">16. <span class="element-citation">Ivancic MM, et al. Conserved serum protein biomarkers associated with growing early colorectal adenomas. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2019;<span class="ref-vol">116</span>:8471–8480. doi: 10.1073/pnas.1813212116.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6486772/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30971492" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.1813212116" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Conserved+serum+protein+biomarkers+associated+with+growing+early+colorectal+adenomas&amp;author=MM+Ivancic&amp;volume=116&amp;publication_year=2019&amp;pages=8471-8480&amp;pmid=30971492&amp;doi=10.1073/pnas.1813212116&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR17">17. <span class="element-citation">Brenner H, et al. Risk of progression of advanced adenomas to colorectal cancer by age and sex: estimates based on 840,149 screening colonoscopies. <span><span class="ref-journal">Gut. </span>2007;<span class="ref-vol">56</span>:1585–1589. doi: 10.1136/gut.2007.122739.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2095643/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17591622" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1136%2Fgut.2007.122739" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Risk+of+progression+of+advanced+adenomas+to+colorectal+cancer+by+age+and+sex:+estimates+based+on+840,149+screening+colonoscopies&amp;author=H+Brenner&amp;volume=56&amp;publication_year=2007&amp;pages=1585-1589&amp;pmid=17591622&amp;doi=10.1136/gut.2007.122739&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR18">18. <span class="element-citation">Rua T, et al. An observational study to compare the utilisation of computed tomography colonography with optical colonoscopy as the first diagnostic imaging tool in patients with suspected colorectal cancer. <span><span class="ref-journal">Clin Radiol. </span>2020;<span class="ref-vol">75</span>(712):e723–712 e731.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32507314" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Radiol&amp;title=An+observational+study+to+compare+the+utilisation+of+computed+tomography+colonography+with+optical+colonoscopy+as+the+first+diagnostic+imaging+tool+in+patients+with+suspected+colorectal+cancer&amp;author=T+Rua&amp;volume=75&amp;issue=712&amp;publication_year=2020&amp;pages=e723-712+e731&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR19">19. <span class="element-citation">Xin L, et al. A streamlined platform for analyzing tera-scale DDA and DIA mass spectrometry data enables highly sensitive immunopeptidomics. <span><span class="ref-journal">Nat Commun. </span>2022;<span class="ref-vol">13</span>:3108. doi: 10.1038/s41467-022-30867-7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9174175/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35672356" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41467-022-30867-7" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Commun&amp;title=A+streamlined+platform+for+analyzing+tera-scale+DDA+and+DIA+mass+spectrometry+data+enables+highly+sensitive+immunopeptidomics&amp;author=L+Xin&amp;volume=13&amp;publication_year=2022&amp;pages=3108&amp;pmid=35672356&amp;doi=10.1038/s41467-022-30867-7&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR20">20. <span class="element-citation">Ahn SB, et al. Use of a recombinant biomarker protein DDA library increases DIA coverage of low abundance plasma proteins. <span><span class="ref-journal">J Proteome Res. </span>2021;<span class="ref-vol">20</span>:2374–2389. doi: 10.1021/acs.jproteome.0c00898.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33752330" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Facs.jproteome.0c00898" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Proteome+Res&amp;title=Use+of+a+recombinant+biomarker+protein+DDA+library+increases+DIA+coverage+of+low+abundance+plasma+proteins&amp;author=SB+Ahn&amp;volume=20&amp;publication_year=2021&amp;pages=2374-2389&amp;pmid=33752330&amp;doi=10.1021/acs.jproteome.0c00898&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR21">21. <span class="element-citation">Chambers AG, Percy AJ, Simon R, Borchers CH. MRM for the verification of cancer biomarker proteins: recent applications to human plasma and serum. <span><span class="ref-journal">Expert Rev Proteomics. </span>2014;<span class="ref-vol">11</span>:137–148. doi: 10.1586/14789450.2014.877346.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24476379" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1586%2F14789450.2014.877346" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Expert+Rev+Proteomics&amp;title=MRM+for+the+verification+of+cancer+biomarker+proteins:+recent+applications+to+human+plasma+and+serum&amp;author=AG+Chambers&amp;author=AJ+Percy&amp;author=R+Simon&amp;author=CH+Borchers&amp;volume=11&amp;publication_year=2014&amp;pages=137-148&amp;pmid=24476379&amp;doi=10.1586/14789450.2014.877346&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR22">22. <span class="element-citation">Percy AJ, Chambers AG, Yang J, Borchers CH. Multiplexed MRM-based quantitation of candidate cancer biomarker proteins in undepleted and non-enriched human plasma. <span><span class="ref-journal">Proteomics. </span>2013;<span class="ref-vol">13</span>:2202–2215. doi: 10.1002/pmic.201200316.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23592487" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fpmic.201200316" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Multiplexed+MRM-based+quantitation+of+candidate+cancer+biomarker+proteins+in+undepleted+and+non-enriched+human+plasma&amp;author=AJ+Percy&amp;author=AG+Chambers&amp;author=J+Yang&amp;author=CH+Borchers&amp;volume=13&amp;publication_year=2013&amp;pages=2202-2215&amp;pmid=23592487&amp;doi=10.1002/pmic.201200316&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR23">23. <span class="element-citation">Demichev V, Messner CB, Vernardis SI, Lilley KS, Ralser M. DIA-NN: neural networks and interference correction enable deep proteome coverage in high throughput. <span><span class="ref-journal">Nat Methods. </span>2020;<span class="ref-vol">17</span>:41–44. doi: 10.1038/s41592-019-0638-x.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6949130/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31768060" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41592-019-0638-x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=DIA-NN:+neural+networks+and+interference+correction+enable+deep+proteome+coverage+in+high+throughput&amp;author=V+Demichev&amp;author=CB+Messner&amp;author=SI+Vernardis&amp;author=KS+Lilley&amp;author=M+Ralser&amp;volume=17&amp;publication_year=2020&amp;pages=41-44&amp;pmid=31768060&amp;doi=10.1038/s41592-019-0638-x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR24">24. <span class="element-citation">Yu G, Wang LG, Han Y, He QY. clusterProfiler: an R package for comparing biological themes among gene clusters. <span><span class="ref-journal">OMICS. </span>2012;<span class="ref-vol">16</span>:284–287. doi: 10.1089/omi.2011.0118.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3339379/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22455463" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1089%2Fomi.2011.0118" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=OMICS&amp;title=clusterProfiler:+an+R+package+for+comparing+biological+themes+among+gene+clusters&amp;author=G+Yu&amp;author=LG+Wang&amp;author=Y+Han&amp;author=QY+He&amp;volume=16&amp;publication_year=2012&amp;pages=284-287&amp;pmid=22455463&amp;doi=10.1089/omi.2011.0118&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR25">25. <span class="element-citation">Fernandez-Costa C, et al. Impact of the identification strategy on the reproducibility of the DDA and DIA results. <span><span class="ref-journal">J Proteome Res. </span>2020;<span class="ref-vol">19</span>:3153–3161. doi: 10.1021/acs.jproteome.0c00153.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7898222/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32510229" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Facs.jproteome.0c00153" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Proteome+Res&amp;title=Impact+of+the+identification+strategy+on+the+reproducibility+of+the+DDA+and+DIA+results&amp;author=C+Fernandez-Costa&amp;volume=19&amp;publication_year=2020&amp;pages=3153-3161&amp;pmid=32510229&amp;doi=10.1021/acs.jproteome.0c00153&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR26">26. <span class="element-citation">Xu S, et al. Long non-coding RNA linc00659 promotes tumour progression by regulating FZD6/Wnt/beta-catenin signalling pathway in colorectal cancer via m6A reader IGF2BP1. <span><span class="ref-journal">J Gene Med. </span>2023 doi: 10.1002/jgm.3636,e3636.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/38009760" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fjgm.3636%2Ce3636" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Gene+Med&amp;title=Long+non-coding+RNA+linc00659+promotes+tumour+progression+by+regulating+FZD6/Wnt/beta-catenin+signalling+pathway+in+colorectal+cancer+via+m6A+reader+IGF2BP1&amp;author=S+Xu&amp;publication_year=2023&amp;pmid=38123364&amp;doi=10.1002/jgm.3636,e3636&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR27">27. <span class="element-citation">Liu X, et al. m6A methylated EphA2 and VEGFA through IGF2BP2/3 regulation promotes vasculogenic mimicry in colorectal cancer via PI3K/AKT and ERK1/2 signaling. <span><span class="ref-journal">Cell Death Dis. </span>2022;<span class="ref-vol">13</span>:483. doi: 10.1038/s41419-022-04950-2.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9122982/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35595748" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41419-022-04950-2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Death+Dis&amp;title=m6A+methylated+EphA2+and+VEGFA+through+IGF2BP2/3+regulation+promotes+vasculogenic+mimicry+in+colorectal+cancer+via+PI3K/AKT+and+ERK1/2+signaling&amp;author=X+Liu&amp;volume=13&amp;publication_year=2022&amp;pages=483&amp;pmid=35595748&amp;doi=10.1038/s41419-022-04950-2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR28">28. <span class="element-citation">Li XL, Zhou J, Chen ZR, Chng WJ. P53 mutations in colorectal cancer&#x02014;molecular pathogenesis and pharmacological reactivation. <span><span class="ref-journal">World J Gastroenterol. </span>2015;<span class="ref-vol">21</span>:84–93. doi: 10.3748/wjg.v21.i1.84.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4284363/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25574081" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3748%2Fwjg.v21.i1.84" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=World+J+Gastroenterol&amp;title=P53+mutations+in+colorectal+cancer&#x02014;molecular+pathogenesis+and+pharmacological+reactivation&amp;author=XL+Li&amp;author=J+Zhou&amp;author=ZR+Chen&amp;author=WJ+Chng&amp;volume=21&amp;publication_year=2015&amp;pages=84-93&amp;pmid=25574081&amp;doi=10.3748/wjg.v21.i1.84&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR29">29. <span class="element-citation">Dong S, et al. ROS/PI3K/Akt and Wnt/beta-catenin signalings activate HIF-1alpha-induced metabolic reprogramming to impart 5-fluorouracil resistance in colorectal cancer. <span><span class="ref-journal">J Exp Clin Cancer Res. </span>2022;<span class="ref-vol">41</span>:15. doi: 10.1186/s13046-021-02229-6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8742403/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34998404" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2Fs13046-021-02229-6" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Exp+Clin+Cancer+Res&amp;title=ROS/PI3K/Akt+and+Wnt/beta-catenin+signalings+activate+HIF-1alpha-induced+metabolic+reprogramming+to+impart+5-fluorouracil+resistance+in+colorectal+cancer&amp;author=S+Dong&amp;volume=41&amp;publication_year=2022&amp;pages=15&amp;pmid=34998404&amp;doi=10.1186/s13046-021-02229-6&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR30">30. <span class="element-citation">Brock R, Xiong B, Li L, Vanbogelen RA, Christman L. A multiplex serum protein assay for determining the probability of colorectal cancer. <span><span class="ref-journal">Am J Cancer Res. </span>2012;<span class="ref-vol">2</span>:598–605.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3433100/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22957311" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Cancer+Res&amp;title=A+multiplex+serum+protein+assay+for+determining+the+probability+of+colorectal+cancer&amp;author=R+Brock&amp;author=B+Xiong&amp;author=L+Li&amp;author=RA+Vanbogelen&amp;author=L+Christman&amp;volume=2&amp;publication_year=2012&amp;pages=598-605&amp;pmid=22957311&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR31">31. <span class="element-citation">Kasahara K, et al. A large-scale targeted proteomics of plasma extracellular vesicles shows utility for prognosis prediction subtyping in colorectal cancer. <span><span class="ref-journal">Cancer Med. </span>2023;<span class="ref-vol">12</span>:7616–7626. doi: 10.1002/cam4.5442.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10067095/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36394150" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fcam4.5442" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Med&amp;title=A+large-scale+targeted+proteomics+of+plasma+extracellular+vesicles+shows+utility+for+prognosis+prediction+subtyping+in+colorectal+cancer&amp;author=K+Kasahara&amp;volume=12&amp;publication_year=2023&amp;pages=7616-7626&amp;pmid=36394150&amp;doi=10.1002/cam4.5442&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR32">32. <span class="element-citation">Singhal M, et al. Temporal multi-omics identifies LRG1 as a vascular niche instructor of metastasis. <span><span class="ref-journal">Sci Transl Med. </span>2021;<span class="ref-vol">13</span>:eabe6805. doi: 10.1126/scitranslmed.abe6805.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7614902/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34516824" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscitranslmed.abe6805" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci+Transl+Med&amp;title=Temporal+multi-omics+identifies+LRG1+as+a+vascular+niche+instructor+of+metastasis&amp;author=M+Singhal&amp;volume=13&amp;publication_year=2021&amp;pages=eabe6805&amp;pmid=34516824&amp;doi=10.1126/scitranslmed.abe6805&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR33">33. <span class="element-citation">Lee DH, et al. Multi-biomarker panel prediction model for diagnosis of pancreatic cancer. <span><span class="ref-journal">J Hepatobiliary Pancreat Sci. </span>2023;<span class="ref-vol">30</span>:122–132. doi: 10.1002/jhbp.986.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33991409" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fjhbp.986" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Hepatobiliary+Pancreat+Sci&amp;title=Multi-biomarker+panel+prediction+model+for+diagnosis+of+pancreatic+cancer&amp;author=DH+Lee&amp;volume=30&amp;publication_year=2023&amp;pages=122-132&amp;pmid=33991409&amp;doi=10.1002/jhbp.986&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR34">34. <span class="mixed-citation">J Chantaraamporn et al., Glycoproteomic analysis reveals aberrant expression of complement C9 and fibronectin in the plasma of patients with colorectal Cancer. Proteomes. 2020;8. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7564939/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32971853" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="CR35">35. <span class="element-citation">Zhang B, et al. Prognostic value of IGFBP2 in various cancers: a systematic review and meta-analysis. <span><span class="ref-journal">Cancer Med. </span>2022;<span class="ref-vol">11</span>:3035–3047. doi: 10.1002/cam4.4680.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9385590/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35546443" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fcam4.4680" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Med&amp;title=Prognostic+value+of+IGFBP2+in+various+cancers:+a+systematic+review+and+meta-analysis&amp;author=B+Zhang&amp;volume=11&amp;publication_year=2022&amp;pages=3035-3047&amp;pmid=35546443&amp;doi=10.1002/cam4.4680&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR36">36. <span class="element-citation">Vocka M, et al. Novel serum markers HSP60, CHI3L1, and IGFBP-2 in metastatic colorectal cancer. <span><span class="ref-journal">Oncol Lett. </span>2019;<span class="ref-vol">18</span>:6284–6292.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6864964/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31788106" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncol+Lett&amp;title=Novel+serum+markers+HSP60,+CHI3L1,+and+IGFBP-2+in+metastatic+colorectal+cancer&amp;author=M+Vocka&amp;volume=18&amp;publication_year=2019&amp;pages=6284-6292&amp;pmid=31788106&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR37">37. <span class="element-citation">Huang Z, et al. Proteome-wide analysis reveals TFEB targets for establishment of a prognostic signature to predict clinical outcomes of colorectal cancer. <span><span class="ref-journal">Cancers (Basel) </span>2023;<span class="ref-vol">15</span>:744. doi: 10.3390/cancers15030744.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9913156/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36765702" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fcancers15030744" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancers+(Basel)&amp;title=Proteome-wide+analysis+reveals+TFEB+targets+for+establishment+of+a+prognostic+signature+to+predict+clinical+outcomes+of+colorectal+cancer&amp;author=Z+Huang&amp;volume=15&amp;publication_year=2023&amp;pages=744&amp;pmid=36765702&amp;doi=10.3390/cancers15030744&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR38">38. <span class="element-citation">Kopylov AT, et al. Revelation of proteomic indicators for colorectal cancer in initial stages of development. <span><span class="ref-journal">Molecules. </span>2020;<span class="ref-vol">25</span>:619. doi: 10.3390/molecules25030619.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7036866/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32023884" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fmolecules25030619" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=Revelation+of+proteomic+indicators+for+colorectal+cancer+in+initial+stages+of+development&amp;author=AT+Kopylov&amp;volume=25&amp;publication_year=2020&amp;pages=619&amp;pmid=32023884&amp;doi=10.3390/molecules25030619&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR39">39. <span class="element-citation">Arner P, et al. Circulating carnosine dipeptidase 1 associates with weight loss and poor prognosis in gastrointestinal cancer. <span><span class="ref-journal">PLoS ONE. </span>2015;<span class="ref-vol">10</span>:e0123566. doi: 10.1371/journal.pone.0123566.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4405487/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25898255" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pone.0123566" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Circulating+carnosine+dipeptidase+1+associates+with+weight+loss+and+poor+prognosis+in+gastrointestinal+cancer&amp;author=P+Arner&amp;volume=10&amp;publication_year=2015&amp;pages=e0123566&amp;pmid=25898255&amp;doi=10.1371/journal.pone.0123566&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR40">40. <span class="element-citation">Li X, et al. A preliminary discussion on carnosine dipeptidase 1 as a potential novel biomarker for the diagnostic and prognostic evaluation of hepatocellular carcinoma. <span><span class="ref-journal">Zhonghua Gan Zang Bing Za Zhi. </span>2023;<span class="ref-vol">31</span>:627–633.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37400388" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Zhonghua+Gan+Zang+Bing+Za+Zhi&amp;title=A+preliminary+discussion+on+carnosine+dipeptidase+1+as+a+potential+novel+biomarker+for+the+diagnostic+and+prognostic+evaluation+of+hepatocellular+carcinoma&amp;author=X+Li&amp;volume=31&amp;publication_year=2023&amp;pages=627-633&amp;pmid=37400388&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR41">41. <span class="element-citation">Chen H, Zucknick M, Werner S, Knebel P, Brenner H. Head-to-head comparison and evaluation of 92 plasma protein biomarkers for early detection of colorectal cancer in a true screening setting. <span><span class="ref-journal">Clin Cancer Res. </span>2015;<span class="ref-vol">21</span>:3318–3326. doi: 10.1158/1078-0432.CCR-14-3051.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26015516" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1158%2F1078-0432.CCR-14-3051" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Cancer+Res&amp;title=Head-to-head+comparison+and+evaluation+of+92+plasma+protein+biomarkers+for+early+detection+of+colorectal+cancer+in+a+true+screening+setting&amp;author=H+Chen&amp;author=M+Zucknick&amp;author=S+Werner&amp;author=P+Knebel&amp;author=H+Brenner&amp;volume=21&amp;publication_year=2015&amp;pages=3318-3326&amp;pmid=26015516&amp;doi=10.1158/1078-0432.CCR-14-3051&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR42">42. <span class="element-citation">Wik L, et al. Proximity Extension assay in combination with next-generation sequencing for high-throughput proteome-wide analysis. <span><span class="ref-journal">Mol Cell Proteomics. </span>2021;<span class="ref-vol">20</span>:100168. doi: 10.1016/j.mcpro.2021.100168.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8633680/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34715355" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.mcpro.2021.100168" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Proteomics&amp;title=Proximity+Extension+assay+in+combination+with+next-generation+sequencing+for+high-throughput+proteome-wide+analysis&amp;author=L+Wik&amp;volume=20&amp;publication_year=2021&amp;pages=100168&amp;pmid=34715355&amp;doi=10.1016/j.mcpro.2021.100168&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR43">43. <span class="element-citation">Surinova S, et al. Prediction of colorectal cancer diagnosis based on circulating plasma proteins. <span><span class="ref-journal">EMBO Mol Med. </span>2015;<span class="ref-vol">7</span>:1166–1178. doi: 10.15252/emmm.201404873.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4568950/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26253081" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.15252%2Femmm.201404873" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+Mol+Med&amp;title=Prediction+of+colorectal+cancer+diagnosis+based+on+circulating+plasma+proteins&amp;author=S+Surinova&amp;volume=7&amp;publication_year=2015&amp;pages=1166-1178&amp;pmid=26253081&amp;doi=10.15252/emmm.201404873&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR44">44. <span class="element-citation">Bjorhall K, Miliotis T, Davidsson P. Comparison of different depletion strategies for improved resolution in proteomic analysis of human serum samples. <span><span class="ref-journal">Proteomics. </span>2005;<span class="ref-vol">5</span>:307–317. doi: 10.1002/pmic.200400900.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15619298" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fpmic.200400900" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Comparison+of+different+depletion+strategies+for+improved+resolution+in+proteomic+analysis+of+human+serum+samples&amp;author=K+Bjorhall&amp;author=T+Miliotis&amp;author=P+Davidsson&amp;volume=5&amp;publication_year=2005&amp;pages=307-317&amp;pmid=15619298&amp;doi=10.1002/pmic.200400900&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR45">45. <span class="element-citation">Chen H, et al. Participation and yield of a population-based colorectal cancer screening programme in China. <span><span class="ref-journal">Gut. </span>2019;<span class="ref-vol">68</span>:1450–1457. doi: 10.1136/gutjnl-2018-317124.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30377193" ref="reftype=pubmed&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1136%2Fgutjnl-2018-317124" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Participation+and+yield+of+a+population-based+colorectal+cancer+screening+programme+in+China&amp;author=H+Chen&amp;volume=68&amp;publication_year=2019&amp;pages=1450-1457&amp;pmid=30377193&amp;doi=10.1136/gutjnl-2018-317124&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10880210&amp;issue-id=452086&amp;journal-id=214&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Journal of Translational Medicine</span> are provided here courtesy of <strong>BMC</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC10880210/pdf/12967_2024_Article_4983.pdf" class="int-view">PDF (2.5M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/10880210/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/10880210/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="10880210" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10880210%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="kF35weXXp5bjOt1eOM8n458bnS7aPVBZSpDrBR6ZIP9ty2EManj6AdHdcYglE0D0">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10880210%2F&amp;text=A%20proteomic%20classifier%20panel%20for%20early%20screening%20of%20colorectal%20cancer%3A%20a%20case%20control%20study" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10880210%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10880210/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC10880210/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38383428/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC10880210/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38383428/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC10880210/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/10880210/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC10880210/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
