
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs2.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Observing inhibition of the SARS-CoV-2 helicase at single-nucleotide resolution - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="D0BD41E4AC192A45000039888CBB9772.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="nar" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC10516658/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10516658/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Nucleic Acids Research" /><meta name="citation_title" content="Observing inhibition of the SARS-CoV-2 helicase at single-nucleotide resolution" /><meta name="citation_author" content="Sinduja K Marx" /><meta name="citation_author_institution" content="Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA" /><meta name="citation_author" content="Keith J Mickolajczyk" /><meta name="citation_author_institution" content="Laboratory of Chemistry and Cell Biology, The Rockefeller University, New York, NY, USA" /><meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, Robert Wood Johnson Medical School, Rutgers University, Piscataway, NJ, USA" /><meta name="citation_author" content="Jonathan&#x000a0;M Craig" /><meta name="citation_author_institution" content="Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA" /><meta name="citation_author" content="Christopher&#x000a0;A Thomas" /><meta name="citation_author_institution" content="Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA" /><meta name="citation_author" content="Akira&#x000a0;M Pfeffer" /><meta name="citation_author_institution" content="Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA" /><meta name="citation_author" content="Sarah&#x000a0;J Abell" /><meta name="citation_author_institution" content="Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA" /><meta name="citation_author" content="Jessica&#x000a0;D Carrasco" /><meta name="citation_author_institution" content="Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA" /><meta name="citation_author" content="Michaela&#x000a0;C Franzi" /><meta name="citation_author_institution" content="Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA" /><meta name="citation_author" content="Jesse&#x000a0;R Huang" /><meta name="citation_author_institution" content="Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA" /><meta name="citation_author" content="Hwanhee&#x000a0;C Kim" /><meta name="citation_author_institution" content="Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA" /><meta name="citation_author" content="Henry Brinkerhoff" /><meta name="citation_author_institution" content="Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA" /><meta name="citation_author" content="Tarun&#x000a0;M Kapoor" /><meta name="citation_author_institution" content="Laboratory of Chemistry and Cell Biology, The Rockefeller University, New York, NY, USA" /><meta name="citation_author" content="Jens&#x000a0;H Gundlach" /><meta name="citation_author_institution" content="Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA" /><meta name="citation_author" content="Andrew&#x000a0;H Laszlo" /><meta name="citation_author_institution" content="Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA" /><meta name="citation_publication_date" content="2023/09/09" /><meta name="citation_issue" content="17" /><meta name="citation_volume" content="51" /><meta name="citation_firstpage" content="9266" /><meta name="citation_doi" content="10.1093/nar/gkad660" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC10516658/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC10516658/" /><meta name="citation_pmid" content="37560916" /><meta name="DC.Title" content="Observing inhibition of the SARS-CoV-2 helicase at single-nucleotide resolution" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Oxford University Press" /><meta name="DC.Contributor" content="Sinduja K Marx" /><meta name="DC.Contributor" content="Keith J Mickolajczyk" /><meta name="DC.Contributor" content="Jonathan&#x000a0;M Craig" /><meta name="DC.Contributor" content="Christopher&#x000a0;A Thomas" /><meta name="DC.Contributor" content="Akira&#x000a0;M Pfeffer" /><meta name="DC.Contributor" content="Sarah&#x000a0;J Abell" /><meta name="DC.Contributor" content="Jessica&#x000a0;D Carrasco" /><meta name="DC.Contributor" content="Michaela&#x000a0;C Franzi" /><meta name="DC.Contributor" content="Jesse&#x000a0;R Huang" /><meta name="DC.Contributor" content="Hwanhee&#x000a0;C Kim" /><meta name="DC.Contributor" content="Henry Brinkerhoff" /><meta name="DC.Contributor" content="Tarun&#x000a0;M Kapoor" /><meta name="DC.Contributor" content="Jens&#x000a0;H Gundlach" /><meta name="DC.Contributor" content="Andrew&#x000a0;H Laszlo" /><meta name="DC.Date" content="2023 Sep 22" /><meta name="DC.Identifier" content="10.1093/nar/gkad660" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Observing inhibition of the SARS-CoV-2 helicase at single-nucleotide resolution" /><meta property="og:type" content="article" /><meta property="og:description" content="The genome of SARS-CoV-2 encodes for a helicase (nsp13) that is essential for viral replication and highly conserved across related viruses, making it an attractive antiviral target. Here we use nanopore tweezers, a high-resolution single-molecule technique, ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10516658/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10516658/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/4/">Nucleic Acids Res</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/445860/">v.51(17); 2023 Sep 22</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC10516658
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC10516658/pdf/gkad660.pdf" class="int-view">PDF (1.7M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/10516658/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/10516658/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="10516658" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10516658%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="3XTSexpzeM2ynae9m1dJ1ugoTlY2htblckGu677lHVap9GpWKFChkOhuLWSIwIj8">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10516658%2F&amp;text=Observing%20inhibition%20of%20the%20SARS-CoV-2%20helicase%20at%20single-nucleotide%20resolution" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10516658%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10516658/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC10516658/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/37560916/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC10516658/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/37560916/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC10516658/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/10516658/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/4/">Nucleic Acids Res</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/445860/">v.51(17); 2023 Sep 22</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC10516658
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/narlogo.gif" alt="Logo of nar" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/nar" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Nucleic Acids Res.</span> 2023 Sep 22; 51(17): 9266–9278. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2023 Aug 10. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1093%2Fnar%2Fgkad660" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/nar/gkad660</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC10516658</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/37560916">37560916</a></div></div></div><h1 class="content-title">Observing inhibition of the SARS-CoV-2 helicase at single-nucleotide resolution</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Marx%20SK%5BAuthor%5D" class="affpopup" co-rid="_co_idm140161674670512" co-class="co-affbox">Sinduja K Marx</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Mickolajczyk%20KJ%5BAuthor%5D" class="affpopup" co-rid="_co_idm140161711735760" co-class="co-affbox">Keith J Mickolajczyk</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Craig%20J%5BAuthor%5D" class="affpopup" co-rid="_co_idm140161712630352" co-class="co-affbox">Jonathan&#x000a0;M Craig</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Thomas%20C%5BAuthor%5D" class="affpopup" co-rid="_co_idm140161614309264" co-class="co-affbox">Christopher&#x000a0;A Thomas</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Pfeffer%20A%5BAuthor%5D" class="affpopup" co-rid="_co_idm140161674089184" co-class="co-affbox">Akira&#x000a0;M Pfeffer</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Abell%20S%5BAuthor%5D" class="affpopup" co-rid="_co_idm140161615691056" co-class="co-affbox">Sarah&#x000a0;J Abell</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Carrasco%20J%5BAuthor%5D" class="affpopup" co-rid="_co_idm140161615986976" co-class="co-affbox">Jessica&#x000a0;D Carrasco</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Franzi%20M%5BAuthor%5D" class="affpopup" co-rid="_co_idm140161621862096" co-class="co-affbox">Michaela&#x000a0;C Franzi</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Huang%20J%5BAuthor%5D" class="affpopup" co-rid="_co_idm140161673753264" co-class="co-affbox">Jesse&#x000a0;R Huang</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kim%20H%5BAuthor%5D" class="affpopup" co-rid="_co_idm140161676988128" co-class="co-affbox">Hwanhee&#x000a0;C Kim</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Brinkerhoff%20H%5BAuthor%5D" class="affpopup" co-rid="_co_idm140161676497184" co-class="co-affbox">Henry Brinkerhoff</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kapoor%20T%5BAuthor%5D" class="affpopup" co-rid="_co_idm140161716292784" co-class="co-affbox">Tarun&#x000a0;M Kapoor</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gundlach%20J%5BAuthor%5D" class="affpopup" co-rid="_co_idm140161617216928" co-class="co-affbox">Jens&#x000a0;H Gundlach</a>, and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Laszlo%20A%5BAuthor%5D" class="affpopup" co-rid="_co_idm140161617213776" co-class="co-affbox">Andrew&#x000a0;H Laszlo</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup></sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140161674670512"><h3 class="no_margin">Sinduja K Marx</h3><p>
Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Marx%20SK%5BAuthor%5D">Sinduja K Marx</a></div></div><div id="_co_idm140161711735760"><h3 class="no_margin">Keith J Mickolajczyk</h3><p>
Laboratory of Chemistry and Cell Biology, The Rockefeller University, New York, NY, USA</p><p>
Department of Biochemistry and Molecular Biology, Robert Wood Johnson Medical School, Rutgers University, Piscataway, NJ, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Mickolajczyk%20KJ%5BAuthor%5D">Keith J Mickolajczyk</a></div></div><div id="_co_idm140161712630352"><h3 class="no_margin">Jonathan&#x000a0;M Craig</h3><p>
Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Craig%20J%5BAuthor%5D">Jonathan&#x000a0;M Craig</a></div></div><div id="_co_idm140161614309264"><h3 class="no_margin">Christopher&#x000a0;A Thomas</h3><p>
Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Thomas%20C%5BAuthor%5D">Christopher&#x000a0;A Thomas</a></div></div><div id="_co_idm140161674089184"><h3 class="no_margin">Akira&#x000a0;M Pfeffer</h3><p>
Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Pfeffer%20A%5BAuthor%5D">Akira&#x000a0;M Pfeffer</a></div></div><div id="_co_idm140161615691056"><h3 class="no_margin">Sarah&#x000a0;J Abell</h3><p>
Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Abell%20S%5BAuthor%5D">Sarah&#x000a0;J Abell</a></div></div><div id="_co_idm140161615986976"><h3 class="no_margin">Jessica&#x000a0;D Carrasco</h3><p>
Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Carrasco%20J%5BAuthor%5D">Jessica&#x000a0;D Carrasco</a></div></div><div id="_co_idm140161621862096"><h3 class="no_margin">Michaela&#x000a0;C Franzi</h3><p>
Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Franzi%20M%5BAuthor%5D">Michaela&#x000a0;C Franzi</a></div></div><div id="_co_idm140161673753264"><h3 class="no_margin">Jesse&#x000a0;R Huang</h3><p>
Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Huang%20J%5BAuthor%5D">Jesse&#x000a0;R Huang</a></div></div><div id="_co_idm140161676988128"><h3 class="no_margin">Hwanhee&#x000a0;C Kim</h3><p>
Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kim%20H%5BAuthor%5D">Hwanhee&#x000a0;C Kim</a></div></div><div id="_co_idm140161676497184"><h3 class="no_margin">Henry Brinkerhoff</h3><p>
Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Brinkerhoff%20H%5BAuthor%5D">Henry Brinkerhoff</a></div></div><div id="_co_idm140161716292784"><h3 class="no_margin">Tarun&#x000a0;M Kapoor</h3><p>
Laboratory of Chemistry and Cell Biology, The Rockefeller University, New York, NY, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kapoor%20T%5BAuthor%5D">Tarun&#x000a0;M Kapoor</a></div></div><div id="_co_idm140161617216928"><h3 class="no_margin">Jens&#x000a0;H Gundlach</h3><p>
Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gundlach%20J%5BAuthor%5D">Jens&#x000a0;H Gundlach</a></div></div><div id="_co_idm140161617213776"><h3 class="no_margin">Andrew&#x000a0;H Laszlo</h3><p>
Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Laszlo%20A%5BAuthor%5D">Andrew&#x000a0;H Laszlo</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140161669476416_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140161669476416_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140161669476416_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140161669476416_ai" style="display:none"><div><span class="fm-affl">Sinduja K Marx, </span><span class="fm-affl" id="aff-a.k.b.f.a.b"> 
Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA;</span></div><div><a href="#article-aaff-info">Contributor Information</a>.</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup>Corresponding author.</div><div id="COR1">To whom correspondence should be addressed. Tel: +1 206 685 2428; Email: <a href="mailto:dev@null" data-email="ude.wu@aolzsal" class="oemail">ude.wu@aolzsal</a></div></div><div class="fm-article-notes hide half_rhythm" id="idm140161669476416_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2023 Jan 25; Revised 2023 Jul 13; Accepted 2023 Jul 31.</div></div><div class="permissions half_rhythm hide" id="idm140161669476416_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; The Author(s) 2023. Published by Oxford University Press on behalf of Nucleic Acids Research.</div><div class="license half_rhythm">This is an Open Access article distributed under the terms of the Creative Commons Attribution-NonCommercial License (<a href="https://creativecommons.org/licenses/by-nc/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">https://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact <a href="mailto:dev@null" data-email="moc.puo@snoissimrep.slanruoj" class="oemail">moc.puo@snoissimrep.slanruoj</a></div></div></div><div id="pmclinksbox" class="links-box whole_rhythm" role="complementary" aria-label="Related or updated information about this article."><div><strong>Previous version available:</strong> This article is based on a previously available preprint posted on <em>bioRxiv</em> on October 8, 2020: "<a href="/pmc/articles/PMC9558434/">Inhibition of the SARS-CoV-2 helicase at single-nucleotide resolution</a>".</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 19% 3px;"></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>gkad660_Supplemental_File.</strong></div><div><a href="/pmc/articles/PMC10516658/bin/gkad660_supplemental_file.pdf" data-ga-action="click_feat_suppl">gkad660_supplemental_file.pdf</a><span style="color:gray"> (10M)</span></div><div class="small guid">GUID: 3150B258-C235-4039-8505-AE44AE344966</div></div></dd></dl><dl data-length="153" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first-last">Data required to reproduce the results described herin and associated analysis code have been deposited in Figshare at DOI: 10.6084/m9.figshare.21936429.</p></dd></dl></div><div id="abstract-a.k.b.r" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.k.b.rtitle" style="text-transform: uppercase;">Abstract</h2><!--article-meta--><div><p class="p p-first-last">The genome of SARS-CoV-2 encodes for a helicase (nsp13) that is essential for viral replication and highly conserved across related viruses, making it an attractive antiviral target. Here we use nanopore tweezers, a high-resolution single-molecule technique, to gain detailed insight into how nsp13 turns ATP-hydrolysis into directed motion along nucleic acid strands. We measured nsp13 both as it translocates along single-stranded DNA or unwinds double-stranded DNA. Our data reveal nsp13&#x02019;s single-nucleotide steps, translocating at &#x0223c;1000 nt/s or unwinding at &#x0223c;100 bp/s. Nanopore tweezers&#x02019; high spatiotemporal resolution enables detailed kinetic analysis of nsp13 motion. As a proof-of-principle for inhibition studies, we observed nsp13&#x02019;s motion in the presence of the ATPase inhibitor ATP&#x003b3;S. We construct a detailed picture of inhibition in which ATP&#x003b3;S has multiple mechanisms of inhibition. The dominant mechanism of inhibition depends on the application of assisting force. This lays the groundwork for future single-molecule&#x000a0;inhibition studies with viral helicases.</p></div></div><div id="abstract-a.k.b.s" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.k.b.stitle" style="text-transform: uppercase;">Graphical Abstract</h2><!--article-meta--><div><p class="p p-first-last">
<!--fig ft0--><!--fig mode=article f1--></p><div class="fig iconblock whole_rhythm" id="ga1" co-legend-rid="lgnd_ga1"><a href="/pmc/articles/PMC10516658/figure/ga1/" target="figure" rid-figpopup="ga1" rid-ob="ob-ga1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140161711148288"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10516658_gkad660figgra1.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkad660figgra1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10516658/bin/gkad660figgra1.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140161711148288"><a target="object" rel="noopener" href="/pmc/articles/PMC10516658/figure/ga1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_ga1"><div><a class="figpopup" href="/pmc/articles/PMC10516658/figure/ga1/" target="figure" rid-figpopup="ga1" rid-ob="ob-ga1">Graphical Abstract</a></div></div></div><p>
</p></div></div><div id="SEC1" class="tsec sec"><h2 class="head no_bottom_margin" id="SEC1title" style="text-transform: uppercase;">INTRODUCTION</h2><p class="p p-first">The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) responsible for the COVID-19 pandemic has spurred large-scale efforts to develop antiviral drugs against its viral replication cycle (<a href="#B1" rid="B1" class=" bibr popnode">1</a>). Vaccines and new antiviral treatments such as Paxlovid, have been revolutionary, still COVID-19 claims thousands of lives per day. Further, vaccines are less effective for the immunocompromised, and Paxlovid cannot be taken by patients with kidney disease, even though this population is at higher risk of developing COVID-19 complications. Additional methods of prevention and treatment are needed. Taking a broader view, there have been three coronaviral outbreaks in the last 20 years (SARS in 2001, MERS in 2012&#x000a0;and SARS-2 in 2019). To head off the next pandemic, humanity needs better understanding of how coronaviruses function and needs to develop a broad&#x000a0;arsenal of antiviral treatments.</p><p>Several SARS-CoV-2 proteins are attractive drug targets as they are essential for viral replication, key among them are non-structural protein 5 (nsp5), a viral protease; nsp12, the RNA dependent RNA polymerase (RdRp); and nsp13, an RNA helicase (<a href="#B2" rid="B2" class=" bibr popnode">2</a>). Antivirals (Paxlovid, Molnupiravir&#x000a0;and Remdesivir) currently approved to treat SARS-CoV-2 aim to inhibit viral replication by targeting nsp5 or nsp12, however, there are no presently approved antivirals which target nsp13. Studies of related coronaviruses (<a href="#B3" rid="B3" class=" bibr popnode">3</a>) and other members of the order Nidovirales (<a href="#B4" rid="B4" class=" bibr popnode">4</a>) have shown that the virus cannot replicate without a functional helicase. For this reason, nsp13 has been identified as a target for antiviral intervention. Moreover, unlike structural proteins, the amino acid sequence of nsp13 is one of the most conserved among related coronavirus species (e.g. SARS- and MERS-Nsp13) and variants of SARS-CoV-2 (including Omicron (<a href="#B5" rid="B5" class=" bibr popnode">5</a>)). Thus, nsp13-targeting antivirals have the potential to be broad spectrum drugs that could address future coronaviral outbreaks as well. Since early in the pandemic, several compounds have been put forward as potentially inhibitory of nsp13 (<a href="#B6" rid="B6" class=" bibr popnode">6&#x02013;12</a>), however, many have high IC<sub>50</sub>, low solubility, and/or the potential for significant side effects. Learning more about how nsp13 functions could pave the way for new inhibitory molecules capable of disrupting nsp13 activity.</p><p>Based on structure and biochemical studies, nsp13 has been classified as a superfamily (SF)&#x000a0;1B&#x000a0;helicase and an inchworm mechanism for translocation has been proposed (<a href="#B13" rid="B13" class=" bibr popnode">13&#x02013;15</a>). In this mechanism, 2A and 1A domains each bind nucleic acids (NA) in a 5&#x02032; &#x02192; 3&#x02032; orientation respectively (<a href="#B13" rid="B13" class=" bibr popnode">13</a>). Binding of a nucleotide-triphosphate, e.g. ATP, is followed by 2A and 1A domains coming together in a &#x02018;closed&#x02019; conformation. During this conformational change, domain 2A advances a distance of one nucleotide in the 3&#x02032;-direction along the NA strand. ATP hydrolysis then triggers a conformational change from &#x02018;closed&#x02019; to &#x02018;open,&#x02019;&#x000a0;causing domain 1A to slide forward by a single nucleotide. This is followed by release of ADP and an inorganic phosphate to complete the cycle. Nsp13 can unwind both dsDNA and dsRNA in a 5&#x02032; &#x02192; 3&#x02032; direction and is capable of hydrolyzing all deoxyribonucleoside triphosphates (dNTPs) and ribonucleoside triphosphates (NTPs) (<a href="#B16" rid="B16" class=" bibr popnode">16&#x02013;19</a>). Despite the RNA genome of SARS-CoV-2, structural data indicate that the nsp13 - nucleic-acid interactions are non-specific towards the ribose or nucleobase (<a href="#B13" rid="B13" class=" bibr popnode">13</a>), and biochemical data show that the unwinding speed for dsDNA and dsRNA to be either similar or slightly faster for dsDNA (<a href="#B16" rid="B16" class=" bibr popnode">16</a>,<a href="#B18" rid="B18" class=" bibr popnode">18</a>,<a href="#B19" rid="B19" class=" bibr popnode">19</a>). Reported unwinding speeds range from &#x0223c;50&#x02013;200 bp/s. On its own, nsp13 appears to be a relatively weak helicase with relatively low processivity during unwinding (<a href="#B16" rid="B16" class=" bibr popnode">16</a>,<a href="#B19" rid="B19" class=" bibr popnode">19</a>). However, its processivity and unwinding speed are enhanced via destabilization of the DNA duplex using optical tweezers and interactions with RdRp (<a href="#B14" rid="B14" class=" bibr popnode">14</a>,<a href="#B16" rid="B16" class=" bibr popnode">16</a>,<a href="#B19" rid="B19" class=" bibr popnode">19</a>). Translocation along ssNA is suspected since that is the mechanism by which nsp13 is thought to unwind NA duplexes but has not yet been observed.</p><p>Nucleic acid unwinding involves a series of chemical and physical steps each consisting of a number of interactions between several different molecules. Each of these interactions and steps is an opportunity for inhibition, however activity assays with Boolean output (it did or did not unwind) are not well suited to determine mechanism. Single-molecule techniques offer an attractive alternative as they allow for direct observation of individual molecules in action but the speed at which nsp13 walks along its NA substrate in addition to its small step size puts the fundamental steps of the nsp13 helicase beyond the resolution of traditional single-molecule techniques. Such resolution would enable detailed kinetic analysis of the underlying steps of nsp13&#x02019;s inchworm mechanism and could also shed light on how inhibitory molecules affect its inner workings. We have developed a nanopore-based technique known as single-molecule picometer-resolution nanopore tweezers (SPRNT), which can measure steps of helicases and other molecular motors at high spatiotemporal resolution (<a href="#B20" rid="B20" class=" bibr popnode">20</a>,<a href="#B21" rid="B21" class=" bibr popnode">21</a>). Here we apply SPRNT to nsp13 of SARS-CoV-2 to reveal the detailed mechanism of nsp13&#x02019;s motion on DNA and we demonstrate how SPRNT can be used to determine the mechanism of action of a helicase inhibitor.</p><p class="p">In SPRNT, a single <em>Mycobacterium smegmatis</em> porin A (MspA) nanopore within a phospholipid bilayer serves as the sole electrical connection between two electrolyte-containing wells (Figure <a href="/pmc/articles/PMC10516658/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1A).</span></span><span>1A</span></a>). A voltage applied across this membrane causes an ion current to flow through the nanopore. The electric field of the nanopore also draws negatively-charged NA through the pore, blocking the ion current. Different NA bases within the nanopore's constriction cause unique ion current blockages which can be decoded into NA sequence as in nanopore sequencing (<a href="#B22" rid="B22" class=" bibr popnode">22</a>). If a helicase molecule is bound to the captured NA strand, the helicase comes to rest on the outer-edge of the entry into the nanopore (henceforth, referred to as pore-rim) as the helicase's large size prohibits entry into the pore. From this point onward, the helicase steps along the&#x000a0;NA causing motion of the NA&#x000a0;through the nanopore, resulting in a sequence of ion current steps as the NA sequence in the pore changes. These ion current steps resolve single-nucleotide steps on sub-millisecond time scales, making it possible to observe the details of helicase motion along NA, while simultaneously determining the NA sequence of the substrate in the helicase. The force SPRNT applies on the enzyme/NA complex is proportional to the applied voltage. Depending on which end of the NA is threaded into the pore, this force can either oppose or assist the helicase motion.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC10516658/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140161676822544"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10516658_gkad660fig1.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkad660fig1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10516658/bin/gkad660fig1.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140161676822544"><a target="object" rel="noopener" href="/pmc/articles/PMC10516658/figure/F1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC10516658/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1.</a></div><!--caption a7--><div class="caption"><p>Nanopore tweezers with nsp13 helicase.&#x000a0;(<strong>A</strong>) Schematic of the nanopore experiment. (<strong>B</strong>) Schematic of the DNA used in this manuscript. The template DNA strand has both 5&#x02032;- and 3&#x02032;- dT30 tails with a region of 36 nucleotides of variable sequence in the middle. dsDNA experiments used a 32 nucleotide complementary strand. A 3&#x02032;-cholesterol causes the DNA strand to associate with the bilayer increasing the local concentration of DNA near the nanopore in order to increase the rate of DNA capture, while the additional negative charge of a 5&#x02032;-phosphate helps to encourage feeding of the 5&#x02032; end of the DNA into the nanopore. As an nsp13 molecule feeds a DNA strand into the pore, ion-current levels corresponding to i) the 5&#x02032;-dT30, ii) the variable sequence region and iii) the 3&#x02032;-dT30 are observed in the ion-current measurement. (<strong>C</strong>) Two example ion current traces for nsp13 translocating on ssDNA. (<strong>D</strong>) Example ion current trace for nsp13 unwinding dsDNA. (<strong>E</strong>) Ion-current consensus for the DNA sequence shown as it translocates through the nanopore constriction. For the shown section of 40 DNA bases, 37 ion current levels are regularly found within the ion current traces. This occurs when currents for adjacent <em>k</em>-mers are indistinguishable. Gaps in the consensus represent these unobserved steps. Red dashed lines in (C)&#x02013;(E) are guides to the eye showing where the 5th, 10th, 15th and 30th levels appear within the read as determined by the automatic alignment. The variable current region of the trace can be aligned to the ion current consensus in C) to produce what is essentially a maximum likelihood estimate for the enzyme's location along the DNA strand as a function of time (below).</p></div></div></div></div><div id="SEC2" class="tsec sec"><h2 class="head no_bottom_margin" id="SEC2title" style="text-transform: uppercase;">MATERIALS AND&#x000a0;METHODS</h2><div id="SEC2-1" class="sec sec-first"><h3 id="SEC2-1title">DNA constructs</h3><p class="p p-first-last">DNA oligonucleotides were synthesized at Stanford University Protein and Nucleic Acid Facility and purified at their facility using Poly-Pak cartridges. The oligo sequences are shown schematically in Fig. 1B; exact sequences are in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table 1</a>. For nsp13 unwinding assays, DNA template and complement were annealed at relative molar concentrations of either 1:1.2 or 1:10 respectively.</p></div><div id="SEC2-2" class="sec"><h3 id="SEC2-2title">Proteins</h3><p class="p p-first-last">The M2-NNN-MspA (<a href="#B23" rid="B23" class=" bibr popnode">23</a>) protein was custom ordered from GenScript. M2-NNN-MspA used in nanopore experiments is identical to wild-type MspA (AC: <a id="__tag_1140608567" class="tag_hotlink" href="/protein/CAB56052.1" ref="/protein/CAB56052.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"CAB56052.1","term_id":"5912275","term_text":"CAB56052.1"}}</span>CAB56052.1</a>) except for the following mutations: D90N/D91N/D93N/D118R/E139K/D134R. Nsp13 from SARS-CoV-2 (AC: <a id="__tag_1140608574" class="tag_hotlink" href="/nuccore/MT121215.1" ref="/nuccore/MT121215.1"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"MT121215.1","term_id":"1819735426","term_text":"MT121215.1"}}</span>MT121215.1</a>) was expressed and purified as reported previously (<a href="#B16" rid="B16" class=" bibr popnode">16</a>).</p></div><div id="SEC2-3" class="sec"><h3 id="SEC2-3title">Pore establishment</h3><p class="p p-first-last">A single MspA nanopore was established in a bilayer as described previously (<a href="#B24" rid="B24" class=" bibr popnode">24</a>). Briefly, a lipid bilayer is formed with a mixture of 1,2-di-O-phytanyl-sn-glycero-3-phosphocholine (Avanti Polar Lipids, SKU: 999984C) and n-hexadecane across a 20 &#x003bc;m diameter aperture that separates two &#x0223c;65 &#x003bc;l chambers (<em>cis</em>and&#x000a0;<em>trans</em>) containing our operating buffers. A potential of 180 mV (unless otherwise specified for force titration experiments) is applied using an Axopatch 200B&#x000a0;amplifier (Axon Instruments) and a National Instruments PCI-6251 DAQ.</p></div><div id="SEC2-4" class="sec"><h3 id="SEC2-4title">Nanopore experiments</h3><p class="p p-first-last">All experiments were run at asymmetric salt, <em>cis</em> 20 mM KCl, with 20 mM HEPES at pH 7.5, 5 mM MgCl<sub>2</sub> and 1 mM TCEP. The <em>trans</em>&#x000a0;buffer 500 mM KCl, 20 mM HEPES at pH 7.5. <em>Cis</em>&#x000a0;buffer also contained ATP, ADP, or ATP&#x003b3;S at variable concentrations. Nsp13 was preincubated with the DNA construct in the absence of ATP by incubating at RT for 30 min at a concentration of 4 &#x003bc;M and 0.4 &#x003bc;M respectively in 124 mM KCl, 20 mM Hepes pH 7.5, 5 mM MgCl<sub>2</sub>&#x000a0;and 1 mM TCEP. Unwinding and translocation experiments were initiated by adding 1&#x003bc;l of this incubation to this <em>cis</em> well with nsp13 and DNA at final concentrations of 100nM and 10 nM, respectively. This final concentration is below <em>K</em><sub>d</sub> for formation of nsp13-DNA complexes, so we anticipate that only one nsp13 is bound to each DNA. In support of this, the vast majority of DNA strands that are pulled into the pore have no nsp13 bound. <em>Cis</em> buffer was reperfused every 5 min followed by readditon of nsp13-DNA incubation to maintain constant concentrations of ATP and prevent bulk accumulation of ADP and DNA. All experiments were recorded at RT (23&#x000a0;&#x000b1;&#x000a0;1&#x000b0;C), with a bias of&#x000a0;+180 mV (unless noted otherwise). ATP and TCEP was ordered from Sigma Aldrich and ATP&#x003b3;S were ordered from Sigma Aldrich or Tocris.</p></div><div id="SEC2-5" class="sec"><h3 id="SEC2-5title">Data acquisition and analysis</h3><p class="p p-first-last">Data were acquired with a sampling rate of 50 kHz and filtered at 10 kHz. Enzyme events were detected via thresholding: an event starts when ion current drops &#x0003c;80% of the open pore current and ends after the ion current has returned above 94% of the open pore current for more than eight datapoints. Candidate events are then evaluated by hand for processive ion-current leveling as in Figure <a href="/pmc/articles/PMC10516658/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure1C,</span></span><span>1C</span></a>, <a href="/pmc/articles/PMC10516658/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,D.</span></span><span>D</span></a>. A subset of these events with clear ion-current transitions was used to create an ion-current consensus (Figure <a href="/pmc/articles/PMC10516658/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1E).</span></span><span>1E</span></a>). By comparing this consensus to the ion currents expected for the template DNA strand, ion currents can be associated with an enzyme's sequence-specific location along the DNA strand (<a href="#B20" rid="B20" class=" bibr popnode">20</a>,<a href="#B21" rid="B21" class=" bibr popnode">21</a>,<a href="#B25" rid="B25" class=" bibr popnode">25</a>). Ion current steps within each event were found automatically using a point-by-point alignment of raw ion-current traces to a consensus of ion current levels (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S5</a>, and associated discussion). Event counts and statistics are summarized in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table 2</a>.</p></div><div id="SEC2-6" class="sec sec-last"><h3 id="SEC2-6title">Alignment of nanopore reads to map enzyme position</h3><p class="p p-first">We used a modified dynamic programing algorithm similar to those we have used in the past (<a href="#B26" rid="B26" class=" bibr popnode">26</a>). Because the ion current levels measured during SPRNT are known beforehand, the previously determined consensus of current levels (Figure <a href="/pmc/articles/PMC10516658/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1E)</span></span><span>1E</span></a>) can be used to detect the most likely timepoint for a step transition. The key innovation here is that instead of agnostically detecting steps in the data which are then aligned to the known consensus, all datapoints from the nanopore read are aligned to the known consensus levels. This enables detection of short-lived states which are below the 8-datapoint minimum typically used. Additionally, fewer step-types are allowed, only single forwards steps, single backwards steps, hold&#x000a0;and &#x02018;catch&#x02019; are allowed. Step penalties are also set to heavily favor hold, i.e. no change in position. As in other dynamic programming approaches to alignment, tuning of step penalties based on their actual probability of observation produces the best alignments. i.e. step penalties should reflect the per-datapoint likelihood of occurrence for hold, step, back, or catch. Hold is roughly 100x more likely than forward and 1000x more likely than back. Step penalties are simply the log probability of occurrence. However, the alignment still performs well on steps far shorter than the average step&#x000a0;duration. This can be attributed to the relatively high signal-to-noise ratio of the data. There is really only one path through the similarity matrix which does not step through a region of cool colors&#x000a0;given the allowed stepping rules. Alignment algorithms such as this provide a wholistic alignment of the data. The alignment path passing through one region of the similarity matrix requires that it can continue on through the subsequent regions. For example, a datapoint at 14 ms into the event matches reasonably well with consensus levels 13, 19, 24, 34, but is aligned to consensus level 13 not only because it matches well, but also because all the other datapoints before and after it match the other surrounding levels. Similar alignments are regularly used for biological sequence alignment such as alignment of DNA sequences and is guaranteed to find the optimal alignment of a read,&#x000a0;X, to a reference,&#x000a0;R, for the given set of rules (<a href="#B27" rid="B27" class=" bibr popnode">27</a>). The &#x02018;catch&#x02019; state is used by the algorithm to traverse transient datapoints that are otherwise inconsistent with the model and would otherwise poison some alignments. An example alignment is schematically shown in Figure <a href="/pmc/articles/PMC10516658/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure2.</span></span><span>2</span></a>. Initially, all datapoints within a read (<em>top</em>) are compared against the known progression of ion current levels which occur as the given DNA strand is passed through the nanopore (<em>left</em>). The so-called &#x02018;similarity matrix&#x02019; (Figure <a href="/pmc/articles/PMC10516658/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2A)</span></span><span>2A</span></a>) represents the match score between each datapoint in the read and levels within the consensus (yellow&#x000a0;=&#x000a0;strong match, blue&#x000a0;=&#x000a0;poor match). Score values are calculated as <em>S<sub>n, m</sub></em> = &#x02013;0.5&#x000a0;&#x000d7;&#x000a0;ln(2&#x003c0;&#x000a0;&#x000d7;&#x000a0;<em>V</em>) + (<em>X<sub>m</sub></em>&#x000a0;&#x02013; <em>R<sub>n</sub></em>)/<em>V</em>, where <em>V</em> is the average current variance, <em>X<sub>m</sub></em> is the <em>m</em>th datapoint and <em>R<sub>n</sub></em> is the <em>n</em>th consensus current level. Alignment of the currents in <em>X</em> to the consensus values of <em>R</em>&#x000a0;(shown in red), uses the similarity matrix along with the allowed step types shown in Figure <a href="/pmc/articles/PMC10516658/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure2B.</span></span><span>2B</span></a>. Steps within the alignment can then be used to determine the most likely location of an ion-current step (top: gray dashed lines) and to locate ion current levels within the raw ion current trace (top: black lines). Within the similarity matrix, an alignment would appear as a diagonal streak of yellow dots. For more detail on dynamic programming alignment algorithms see refs. (<a href="#B25" rid="B25" class=" bibr popnode">25&#x02013;27</a>). Automatic alignments performed in this way can then be briefly checked by eye before incorporation into the combined dataset. Some alignments are corrected manually using a custom GUI.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC10516658/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140161620570064"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10516658_gkad660fig2.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkad660fig2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10516658/bin/gkad660fig2.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140161620570064"><a target="object" rel="noopener" href="/pmc/articles/PMC10516658/figure/F2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC10516658/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2.</a></div><!--caption a7--><div class="caption"><p>Context-aware step detection. (<strong>A</strong>) Similarity matrix (with elements S<sub>n,m</sub>)&#x000a0;for an example read (top) compared with the ion consensus current levels for the template strand (left). Asterisks indicate locations where steps are missing due to indistinguishable adjacent levels. These gaps are corrected for following alignment converting position in the consensus to actual position along the DNA strand. A red line shows the best alignment of the example read to the consensus current levels given the allowed stepping rules and step penalties. Enzyme steps are then inferred at the most likely transition points as determined by alignment. The resulting alignment traces a single nsp13&#x02019;s position along the DNA as a function of time. (<strong>B</strong>) Allowed step types in our alignment algorithm. (<strong>C</strong>) The alignment score matrix, A<sub>n,m</sub>, is filled out column by column based on these selection rules. (<strong>D</strong>) Step probabilities and associated penalties (log probability) used for alignment.</p></div></div></div></div></div><div id="SEC3" class="tsec sec"><h2 class="head no_bottom_margin" id="SEC3title" style="text-transform: uppercase;">RESULTS</h2><p class="p p-first">Because nsp13 is known to unwind DNA (<a href="#B16" rid="B16" class=" bibr popnode">16</a>,<a href="#B19" rid="B19" class=" bibr popnode">19</a>) and has a higher affinity for DNA (<a href="#B16" rid="B16" class=" bibr popnode">16</a>), we designed a DNA substrate containing a 37-nucleotide pseudorandom sequence flanked by 30-nt poly-T overhangs on the 3&#x02032;- and 5&#x02032; ends (Figure <a href="/pmc/articles/PMC10516658/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1B).</span></span><span>1B</span></a>). A cholesterol tag at the 3&#x02032; end of the DNA strand was used to increase the DNA concentration near the bilayer, thereby increasing the rate of capture into the pore. The DNA base sequence of the pseudorandom section was chosen to have varied sequence context and easily distinguishable ion current levels (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S1</a>). A 32-nucleotide complementary strand was hybridized to this strand for dsDNA experiments (Figure <a href="/pmc/articles/PMC10516658/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1B,</span></span><span>1B</span></a>, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S1</a>, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S2</a>, Materials and Methods). Nsp13 loads onto the 5&#x02032; tail near the dsDNA-ssDNA junction and the free 5&#x02032; end of the DNA then threads into the nanopore (Figure <a href="/pmc/articles/PMC10516658/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1A</span></span><span>1A</span></a> and <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S3</a>). We recorded a total of 2413 individual nsp13 translocation and unwinding events and a total of 27641 helicase steps (Table S2). Both ssDNA and dsDNA templates reveal 1-nt steps in a 5&#x02032; &#x02192; 3&#x02032; direction with significantly slower nsp13 motion on dsDNA (Figures <a href="/pmc/articles/PMC10516658/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -4em;">(Figures1C,</span></span><span>1C</span></a>, <a href="/pmc/articles/PMC10516658/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,D,</span></span><span>D</span></a>, Supplementary Figures&#x000a0;S1, S2). At saturating [ATP] and &#x0223c;36 pN assisting force, we observe nsp13 ssDNA translocation speed of 1140&#x000a0;&#x000b1;&#x000a0;40 nt/s (mean&#x000a0;&#x000b1;&#x000a0;s.e.m.) and an average dsDNA unwinding speed of 120&#x000a0;&#x000b1;&#x000a0;10 bp/s (mean&#x000a0;&#x000b1;&#x000a0;s.e.m.) (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S4</a>). In both translocation and unwinding experiments,&#x000a0;individual helicases exhibited&#x000a0;a broader range of speeds than would be expected from statistical variation alone, suggesting a significant amount of static disorder affecting the helicases&#x02019; function. Even during the fast nsp13 translocation of 1500 nt/s, single-nucleotide steps are well resolved (Figure <a href="/pmc/articles/PMC10516658/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1C).</span></span><span>1C</span></a>). The remarkable speed of nsp13 during translocation presented an analysis challenge as steps within the data were at times difficult to resolve. To fully leverage the resolution available within the SPRNT data, we developed new analytical techniques (Figure <a href="/pmc/articles/PMC10516658/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2).</span></span><span>2</span></a>). This represents a significant step forward in single-molecule resolution (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S5</a>).</p><p class="p">Our analysis of nanopore tweezers data previously involved automated level-finding using a change-point algorithm followed by alignment of the found levels to the ion-current consensus (<a href="#B20" rid="B20" class=" bibr popnode">20</a>,<a href="#B21" rid="B21" class=" bibr popnode">21</a>,<a href="#B28" rid="B28" class=" bibr popnode">28</a>). In that analysis scheme, level-finding was agnostic to the underlying DNA sequence and detection of levels was achieved by looking for statistically significant steps within the data (<a href="#B29" rid="B29" class=" bibr popnode">29</a>). Here, nsp13&#x02019;s speed makes such step detection difficult or impossible as individual steps sometimes consist of just a few data points which is insufficient to agnostically determine whether a step took place. However, here the DNA sequence, and therefore the expected progression of ion current levels, is known (Figure <a href="/pmc/articles/PMC10516658/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1E)</span></span><span>1E</span></a>) and that information can be used to inform step-finding. By adapting our alignment algorithm so that raw data was aligned to the ion-current consensus, we were able to perform sequence-aware step finding (Figure <a href="/pmc/articles/PMC10516658/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2</span></span><span>2</span></a> and <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figures S6 and&#x000a0;S7</a>). In this scheme, the ion-current datapoints for a given read are aligned to consensus current values. The resulting alignment (Figure <a href="/pmc/articles/PMC10516658/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2A,</span></span><span>2A</span></a>, <em>red line</em>) yields what is essentially a global best-fit location of the nsp13 helicase as a function of time. We can then extract kinetic parameters such as step dwell-time, the location and frequency of enzyme backsteps, and the location of read start and read terminations. Such automated alignments can be readily verified by-eye, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplemental Figures S6 and S7</a> show several representative alignments for ssDNA translocation and dsDNA unwinding, respectively.</p><div id="SEC3-1" class="sec"><h3 id="SEC3-1title">Single-nucleotide ssDNA translocation</h3><p class="p p-first">Figure <a href="/pmc/articles/PMC10516658/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure3A</span></span><span>3A</span></a> shows the dwell-time distribution for 4 representative locations along the DNA strand. Intriguingly, nsp13 steps over each of these locations at different rates. Such sequence-dependence has been reported for other helicases (<a href="#B21" rid="B21" class=" bibr popnode">21</a>,<a href="#B30" rid="B30" class=" bibr popnode">30&#x02013;32</a>). At saturating conditions, the underlying dwell-time distributions of each nucleotide transition are well-fit by a single exponential distribution (Figure <a href="/pmc/articles/PMC10516658/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3A,</span></span><span>3A</span></a>, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S8A</a>), indicating that each step during nsp13 translocation is governed by a single rate-limiting step. Since processive helicases couple ATP hydrolysis to DNA translocation, we varied the ATP concentration to probe nsp13 translocation kinetics.&#x000a0;The rate of translocation is dependent on [ATP] and is well described by Michaelis&#x02013;Menten kinetics, with <em>V</em><sub>max</sub> and <em>K</em><sub>M</sub> varying significantly (Figure <a href="/pmc/articles/PMC10516658/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3B,</span></span><span>3B</span></a>, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S9</a>) depending on the underlying sequence context within nsp13 (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figures S9 and S10</a>). At low ATP, the event rate and enzyme processivity were significantly reduced complicating collection of data in the low ATP regime.&#x000a0;Fits of Michaelis&#x02013;Menten parameters were complicated by the lack of data at low [ATP] which could better constrain <em>K</em><sub>M</sub>. The form of the Michaelis&#x02013;Menten equation means there is significant covariance between <em>K</em><sub>M</sub> and <em>V</em><sub>max</sub> leading to relatively large uncertainties in both fit parameters. Still, error ellipses for these fits show significant differences from position to position and also differ significantly from a fit to a combined dataset in which all datapoints are pooled regardless of enzyme position (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S10</a>). Nsp13&#x02019;s [ATP] dependence gives us confidence that we are observing ATP-driven translocation while the large variation in rate of translocation at different DNA positions suggests that the nucleic-acid base identity within nsp13 affects translocation kinetics.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC10516658/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140161712359136"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10516658_gkad660fig3.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkad660fig3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10516658/bin/gkad660fig3.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140161712359136"><a target="object" rel="noopener" href="/pmc/articles/PMC10516658/figure/F3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC10516658/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3.</a></div><!--caption a7--><div class="caption"><p>Sequence-dependent nsp13 translocation and unwinding kinetics. (<strong>A</strong>) Distributions of dwell-time for DNA positions 3&#x02013;6 from all translocation events at 1000 &#x003bc;M ATP. An exponential fit to the data is shown in red. Dwell-times for all positions are shown in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S8</a>. (<strong>B</strong>) Enzyme rate as a function of [ATP] at corresponding DNA positions from (A). Rate is calculated as the reciprocal of mean dwell-time at the DNA position. The mean dwell-time at each [ATP] was calculated by taking the average of all dwell-times&#x000a0;&#x0003c;&#x000a0;<em>t</em>&#x000a0;=&#x000a0;100 ms. The black line shows the best fit of the data to the Michaelis&#x02013;Menten equation. The red dashed lines indicate <em>V</em><sub>max</sub> and <em>K</em><sub>M</sub>. Fits for all positions can be found in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figures S9 and S10</a>. (<strong>C</strong>) Comparing processivity (defined as bases translocated before falling off) for three different ATP concentrations: 75, 600 and 1000 &#x003bc;M ATP (<em>n</em>&#x000a0;=&#x000a0;384, 162 and 825 events respectively). The black line shows the best fit of the processivity for the data. (<strong>D</strong>) The effect of applied force on nsp13 translocation on ssDNA was studied by varying the voltage at constant saturating [ATP] = 3 mM. Mean dwell-time (ms) (mean&#x000a0;&#x000b1;&#x000a0;s.e.m.) as a function of force (pN) measured at three different voltages 220, 180, 120 mV that corresponds to &#x0223c;44, &#x0223c;36, &#x0223c;24 pN assisting force respectively (<em>n</em>&#x000a0;=&#x000a0;1794, <em>n</em>&#x000a0;=&#x000a0;4329 and <em>n</em>&#x000a0;=&#x000a0;507 events respectively). (<strong>E</strong>) Mean dwell-time of nsp13 at each DNA position measured in Figure <a href="/pmc/articles/PMC10516658/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure1.</span></span><span>1</span></a>. for translocation (blue) and unwinding (orange), at 1000 &#x003bc;M ATP (<em>n</em>&#x000a0;=&#x000a0;819 and 902 events respectively). Data are means&#x000a0;&#x000b1;&#x000a0;SEM. Gaps are DNA positions where the measured ion-current between two nucleotide positions was identical and a mechanical step could not confidently be identified. (<strong>F</strong>) Comparing speed of translocation versus unwinding. (<strong>G</strong>) Probability of starting at DNA position&#x000a0;&#x02264;1 for translocation and unwinding. (Above) Schematic showing DNA positions, numbered with respect to start and end of double-stranded unwinding region. (<strong>H</strong>) Probability distribution of the length along the DNA over which the nsp13 stayed bound (&#x02018;read length&#x02019;, processivity) for translocation vs. unwinding.</p></div></div></div><p>Nsp13 translocation can begin and end anywhere on the template DNA, with 50% of reads terminating after nsp13 walks into the 3&#x02032; poly-dT section at the end of the DNA template (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S11</a>). The true processivity of nsp13, defined as the number of bases translocated before nsp13 disassociates, is estimated by fitting the observed read-lengths at varying [ATP] (Figure <a href="/pmc/articles/PMC10516658/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3C).</span></span><span>3C</span></a>). Processivity at saturating [ATP] is 26&#x000a0;&#x000b1;&#x000a0;4 nt and drops to 7.9&#x000a0;&#x000b1;&#x000a0;0.7 nt (mean&#x000a0;&#x000b1;&#x000a0;sem) at [ATP] = 75 &#x003bc;M.</p><p>During SPRNT, the electrostatic force (&#x0223c;36 pN at 180 mV) pulls on the NA via the electric field in the pore constriction and leads to a balancing contact force on the surface of the enzyme where it contacts the pore-rim. With the 5&#x02032; end of the NA threaded into the pore, the force on the NA acts to assist the enzyme's motion along the template strand (Figure <a href="/pmc/articles/PMC10516658/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1A</span></span><span>1A</span></a> and <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S3</a>). Previous nanopore tweezers experiments have seen evidence for a one-dimensional diffusive step in which the applied force, rather than ATP-hydrolysis drives enzyme motion (<a href="#B21" rid="B21" class=" bibr popnode">21</a>). To investigate the effect of assisting force on nsp13 translocation, we varied the force from &#x0223c;24&#x000a0;to &#x0223c;44 pN at saturating [ATP]. Because we are applying an assisting force, if force were to affect enzyme speed, we would expect the enzyme to speed up with increasing force. However, we found no significant change in average speed (Figure <a href="/pmc/articles/PMC10516658/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3D</span></span><span>3D</span></a> and <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figures S12, S13</a>). Since the speed of translocation is independent of force in the range of forces we studied but dependent on [ATP], these data suggest that the observed nsp13 translocation is predominantly on-pathway ATP-hydrolysis-driven motion.</p><p>By definition, at saturating [ATP], ATP-binding is not rate-limiting. This, in combination with the lack of apparent force dependence, suggests that either (i) ATP-hydrolysis, (ii) domain 1A motion, or (iii) ADP &#x00026; phosphate release are rate-limiting for nsp13 translocation. The apparent sequence dependence of this rate-limiting step suggests that it is domain 1A motion, but these observations do not strictly rule out effects of the DNA sequence on the rate of ATP hydrolysis or ADP release.</p><p class="p p-last">To further investigate the ADP-release portion of nsp13&#x02019;s kinetic pathway we also ran the ssDNA assay in the presence of equimolar ATP and ADP ([ATP] = [ADP] = 600&#x000a0;&#x003bc;M). In these conditions, the rate of nsp13 translocation was slowed by approximately a factor of two (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S14</a>). This is similar to nanopore tweezers results with Hel308 helicase and PcrA helicase where ADP was seen to act as a competitive inhibitor to ATP-binding (<a href="#B21" rid="B21" class=" bibr popnode">21</a>,<a href="#B30" rid="B30" class=" bibr popnode">30</a>). Similar to force-assisting nanopore tweezers experiments with PcrA, ADP causes an increase in step dwell-time. ADP-binding does not appear to be associated with a physical step of the DNA through the pore.</p></div><div id="SEC3-2" class="sec"><h3 id="SEC3-2title">Double stranded DNA unwinding</h3><p class="p p-first">We next observed nsp13&#x02019;s ability to unwind dsDNA. The mean dwell-times for duplex unwinding steps are similarly sequence-dependent and on average &#x0223c;8-fold longer than that of ssDNA translocation (Figure <a href="/pmc/articles/PMC10516658/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3E,</span></span><span>3E</span></a>, <a href="/pmc/articles/PMC10516658/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,F).</span></span><span>F</span></a>). This is similar to what would be expected for a &#x02018;passive&#x02019; helicase (<a href="#B33" rid="B33" class=" bibr popnode">33</a>,<a href="#B34" rid="B34" class=" bibr popnode">34</a>).</p><p>While ssDNA reads start at a broad range of DNA positions, nearly 80% of the dsDNA reads start with the 5&#x02032; poly-dT within the pore constriction (Figure <a href="/pmc/articles/PMC10516658/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3G</span></span><span>3G</span></a> and <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S15</a>). The reads that start within the duplex region are also comparable in speed to dsDNA unwinding (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S16</a>). This suggests the majority of the reads, including the partial dsDNA reads, are unwinding reads and that limited nsp13 unwinding occurs in the bulk solution.</p><p>Compared to ssDNA translocation, the processivity during dsDNA unwinding is shorter at 14&#x000a0;&#x000b1;&#x000a0;1 nt (mean&#x000a0;&#x000b1;&#x000a0;sem) (Figure <a href="/pmc/articles/PMC10516658/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3H),</span></span><span>3H</span></a>), which is consistent with previous results showing low processivity during unwinding (<a href="#B16" rid="B16" class=" bibr popnode">16</a>). The underlying dwell-time distributions are single-exponential (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S17</a>), indicating a single, rate-limiting step. Because dsDNA unwinding is so much slower than ssDNA translocation, it stands to reason that the rate-limiting sub-step is the sub-step in which DNA unwinding occurs. Intriguingly, the dwell-times for dsDNA unwinding and ssDNA translocation as a function of DNA position are correlated (Figure <a href="/pmc/articles/PMC10516658/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3E,</span></span><span>3E</span></a>, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S18</a>). A similar effect was recently reported with the superfamily 1A helicase PcrA (<a href="#B21" rid="B21" class=" bibr popnode">21</a>). In that case, it was deduced that such a correlation would occur if unwinding and translocation shared the same rate-limiting step. The presence of the DNA duplex acts as an additional barrier to that rate-limiting step. The duplex then modifies the rate-constants for the rate-limiting step rather than adding a new rate-limiting step. Put another way, unwinding happens within the context of enzyme translocation, energy barriers to translocation still affect the rate of unwinding. From this we can conclude that the advance of domain 1A following ATP-hydrolysis is rate-limiting in both translocation and unwinding and that unwinding occurs coincident with this step.</p><p class="p">Based on these data and in light of insights from studying other SF1 and SF2 helicases using SPRNT (<a href="#B20" rid="B20" class=" bibr popnode">20</a>,<a href="#B21" rid="B21" class=" bibr popnode">21</a>), we propose an ATP hydrolysis model that places the steps of the inchworm mechanism in the context of the observed transitions during SPRNT (Figure <a href="/pmc/articles/PMC10516658/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure4).</span></span><span>4</span></a>). In SPRNT, only steps in which DNA moves relative to the pore produce observable steps in the data. In the experiment shown here, domain 2A rests on the pore-rim, meaning that only the motion of DNA relative to domain 2A is observable (<a href="#B21" rid="B21" class=" bibr popnode">21</a>). This is also the only kinetic step that can be modified by the applied force because this is the only place within the kinetic pathway in which the applied force affects the balance between two adjacent kinetic states (<a href="#B35" rid="B35" class=" bibr popnode">35</a>). During this step, domain 2A is less tightly bound to the DNA and possibly domain 1A is more tightly bound. Domain 1A and domain 2A are pulled together by the high charge density of ATP in the ATP binding pocket aided by the forces applied by the nanopore: 1) tension along the DNA pulling on domain 1A (caused by the electrophoretic force on the DNA within the pore) and 2) pressure of the pore rim on domain 2A (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S19</a>). The application of force in this SPRNT experiment biases ATP-bound nsp13 into a closed conformation by increasing the rate of nsp13 closing and decreasing the rate of opening. Other kinetic sub-steps, along the rows of Figure <a href="/pmc/articles/PMC10516658/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure4,</span></span><span>4</span></a>, are inferred from the known steps of the inchworm mechanism and details of their kinetics can be inferred from observing changes in kinetics as experimental parameters are varied (<a href="#B30" rid="B30" class=" bibr popnode">30</a>). For example, the presence of equimolar ADP slows nsp13 by encouraging binding of ADP, thereby backtracking nsp13 on its kinetic pathway from state I to state V. This does not result in DNA motion through the pore but does increase the step dwell-time. During steps other than the ATP-bound motion of domain 2A, the DNA tension and normal force of the pore rim on domain 2A are balanced within domain 2A and no DNA motion is observed for these steps (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S19</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC10516658/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140161615722576"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10516658_gkad660fig4.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkad660fig4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10516658/bin/gkad660fig4.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140161615722576"><a target="object" rel="noopener" href="/pmc/articles/PMC10516658/figure/F4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC10516658/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4.</a></div><!--caption a7--><div class="caption"><p>Nsp13 ATP-hydrolysis mechanism in the context of SPRNT. The observable chemical step of the ATP hydrolysis cycle during a SPRNT measurement is on the y-axis. Inferred steps, not associated with mechanical DNA motion, are on the x-axis. Chemical steps based on inchworm mechanism are: I) Enzyme bound to DNA, walker domains apart (green: domain 2A, blue: domain 1A); II) Nucleoside triphosphate bound, walker domains apart; III) Nucleoside triphosphate bound, walker domains together; IV) ADP bound, walker domains together; V) ADP bound, walker domains apart; the assisting force of SPRNT can only affect steps in which DNA moves through the pore; i.e. transitions between states II and III.</p></div></div></div></div><div id="SEC3-3" class="sec sec-last"><h3 id="SEC3-3title">Nsp13 inhibition by ATP&#x003b3;S</h3><p class="p p-first">There is significant interest in determining mechanisms by which nsp13 can be inhibited by small molecules (<a href="#B7" rid="B7" class=" bibr popnode">7&#x02013;12</a>). Because SPRNT can resolve individual ATPase turnovers, it can provide detailed insight into the mechanism of action of inhibitory molecules. As a proof of principle, we investigated inhibition of nsp13 by&#x000a0;the slowly-hydrolysable ATP-analog ATP&#x003b3;S (<a href="#B36" rid="B36" class=" bibr popnode">36</a>,<a href="#B37" rid="B37" class=" bibr popnode">37</a>) which has been shown to inhibit nsp13 unwinding in bulk biochemical experiments (<a href="#B38" rid="B38" class=" bibr popnode">38</a>). With ATP&#x003b3;S present, the average speed and processivity of nsp13 is significantly reduced relative to the ATP-only condition for translocation (Figure <a href="/pmc/articles/PMC10516658/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5A</span></span><span>5A</span></a>&#x02013;<a href="/pmc/articles/PMC10516658/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>E</span></a>). Commercially available ATP&#x003b3;S regularly contains ADP as a significant contaminant comprising as much as 10% of the reagent, however control experiments with a mixture of 600&#x003bc;M ATP and 600&#x003bc;M ADP showed significantly less inhibition (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S14B</a>), suggesting that the observed inhibition was indeed due to ATP&#x003b3;S. To determine whether ATP&#x003b3;S alone was sufficient for ssDNA translocation, we attempted an experiment with [ATP&#x003b3;S] = 600 &#x003bc;M alone. We observed translocation with similar speed and processivity to the mixed condition with [ATP&#x003b3;S] = 300 &#x003bc;M and [ATP] = 600 &#x003bc;M (Figure <a href="/pmc/articles/PMC10516658/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5A,</span></span><span>5A</span></a>, <a href="/pmc/articles/PMC10516658/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,B,</span></span><span>B</span></a>, <a href="/pmc/articles/PMC10516658/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,E).</span></span><span>E</span></a>). This suggests that nsp13 is capable of hydrolyzing ATP&#x003b3;S and that the inhibitory effect of ATP&#x003b3;S saturates above 300&#x003bc;M suggesting that nsp13 has a higher affinity for ATP&#x003b3;S than for ATP. Dwell-time distributions for nsp13 translocation on ssDNA with ATP&#x003b3;S are well described by an exponential distribution suggesting the presence of a single rate-limiting step (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S20</a>). Intriguingly, the rate constants for each DNA position are different, suggesting that the DNA sequence within the nsp13 affects this rate-limiting step.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC10516658/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140161620978688"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10516658_gkad660fig5.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkad660fig5.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10516658/bin/gkad660fig5.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140161620978688"><a target="object" rel="noopener" href="/pmc/articles/PMC10516658/figure/F5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC10516658/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5.</a></div><!--caption a7--><div class="caption"><p>ATP&#x003b3;S inhibition of nsp13. (<strong>A</strong>) Example position versus time traces for nsp13 unwinding events in the presence of ATP only (blue), ATP and ATP&#x003b3;S (red) and ATP&#x003b3;S only (black). (<strong>B</strong>) Distribution of read-lengths (each a measure of the distance enzyme travelled before disassociation) during translocation and unwinding for ATP and ATP&#x003b3;S mixed conditions. (<strong>C</strong>) Probability of an enzyme event starting at DNA position&#x000a0;&#x02264;1 for corresponding events in (B). (<strong>D</strong>) Probability of a full read for corresponding conditions in (C). (<strong>E</strong>) Comparison of speed of translocation and unwinding during translocation and unwinding in various mixed ATP and ATP&#x003b3;S conditions.The effect of applied force on nsp13 unwinding activity was measured in the presence of [ATP] = 600 &#x003bc;M and [ATP&#x003b3;S] = 300 &#x003bc;M. (<strong>F</strong>) Mean dwell time per event averaged over all DNA positions at various assisting forces. All error bars are SEM. (<strong>G</strong>) Distribution of the measured read-lengths for the corresponding nsp13 events at the indicated forces.</p></div></div></div><p>We also tested nsp13&#x02019;s ability to unwind the DNA duplex in the presence of ATP&#x003b3;S. Unwinding speed and processivity are significantly reduced by the presence of ATP&#x003b3;S. Intriguingly, while significantly fewer bases are unwound by an individual nsp13 helicase, the total time spent on the DNA strand is significantly longer (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S21</a>). In ATP&#x003b3;S-only conditions, nearly 100% of unwinding reads begin with the 5&#x02032; poly-dT within the nanopore constriction, significantly greater than was observed in conditions that included ATP. Therefore, in the absence of ATP, there is no significant unwinding in bulk. This suggests that nsp13 can only unwind DNA using ATP&#x003b3;S in the presence of an assisting force provided by the nanopore.</p><p>We further investigated the role of this assisting force for ATP&#x003b3;S-associated motion by varying force from 24&#x000a0;to 44 pN at constant [ATP] (600 &#x003bc;M) and [ATP&#x003b3;S] (300 &#x003bc;M). At all applied forces, there was no significant change in average speed (Figure <a href="/pmc/articles/PMC10516658/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5F)</span></span><span>5F</span></a>) while processivity appeared to be unchanged or to decrease slightly with force (Figure <a href="/pmc/articles/PMC10516658/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5G).</span></span><span>5G</span></a>). As noted above, we expect the force exerted in this experiment to affect only the kinetic step in which domain 2A advances along the DNA following NTP binding. In sum, we make the following observations: (i) ATP&#x003b3;S slows the progression of nsp13 on ssDNA and dsDNA, (ii) inhibition by ATP&#x003b3;S saturates above &#x0223c;300 &#x003bc;M, (iii) there is little force dependence in the range 24&#x02013;44 pN and (iv) with zero assisting force (bulk) ATP&#x003b3;S is insufficient for unwinding while with assisting force (on the nanopore) nsp13 can use ATP&#x003b3;S to unwind DNA.</p><p>Using these observations, we have constructed a model of ATP&#x003b3;S inhibition for nsp13 (Figure <a href="/pmc/articles/PMC10516658/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6).</span></span><span>6</span></a>). ATP&#x003b3;S inhibits nsp13 unwinding in two ways. First, at zero force, ATP&#x003b3;S inhibits the first conformational change of the inchworm mechanism. This could be either due to reducing the closing-rate of domain 1A and 2A to essentially zero or due to destabilizing the closed state so that it rapidly backsteps to State II before ATP&#x003b3;S can be hydrolyzed. Second, even in the presence of an assisting force, which helps to stabilize nsp13 in the closed conformation (State III), ATP&#x003b3;S-hydrolysis is slower than ATP-hydrolysis. We hypothesize that this is the principal reason why ATP&#x003b3;S slows nsp13 in the SPRNT assay. Alternative models in which ATP&#x003b3;S is not hydrolyzed are shown in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S22</a>, however only the model presented in Figure <a href="/pmc/articles/PMC10516658/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure6</span></span><span>6</span></a> is capable of describing all of our observations (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figures S22 and S23</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F6" co-legend-rid="lgnd_F6"><a href="/pmc/articles/PMC10516658/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140161620831552"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10516658_gkad660fig6.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkad660fig6.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10516658/bin/gkad660fig6.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140161620831552"><a target="object" rel="noopener" href="/pmc/articles/PMC10516658/figure/F6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F6"><div><a class="figpopup" href="/pmc/articles/PMC10516658/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6">Figure 6.</a></div><!--caption a7--><div class="caption"><p>Model of nsp13 ATP&#x003b3;S inhibition. ATP&#x003b3;S inhibits nsp13 via two separate mechanisms. (<strong>A</strong>) Without assisting force (in bulk), ATP&#x003b3;S inhibits the motion of domain 2A following ATP&#x003b3;S-binding. (<strong>B</strong>) The assisting force applied in SPRNT overcomes the inhibitory effect of ATP&#x003b3;S seen in (A), however, hydrolysis of ATP&#x003b3;S is also slower as is seen in many other ATPases.</p></div></div></div><p>This model can also be used to make predictions for the combined effects of ATP&#x003b3;S and the presence of a DNA duplex on the rate of nsp13 progression. In other words we can use observations of dsDNA unwinding (Figure <a href="/pmc/articles/PMC10516658/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5</span></span><span>5</span></a><em>orange</em>, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S17</a>) and ATP&#x003b3;S-driven ssDNA translocation (Figure <a href="/pmc/articles/PMC10516658/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5dark</span></span><span>5</span></a><em>dark green</em>, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S20</a>) to predict nsp13&#x02019;s behavior while unwinding with ATP&#x003b3;S-alone (Figure <a href="/pmc/articles/PMC10516658/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5</span></span><span>5</span></a><em>black</em>, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S24</a>). In this model, ATP&#x003b3;S-hydrolysis and duplex unwinding are sequential steps and are a similar order of magnitude. Therefore, the mean step dwell-time for dsDNA unwinding with ATP&#x003b3;S-alone is given by the sum of the mean step dwell-time for unwinding and the mean step dwell-time for ATP&#x003b3;S-driven ssDNA translocation and the dwell-time histograms at each position should take the form of a convolution of the dwell-time histograms from <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figures S17 and S20</a>. The data are in good agreement with this prediction (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S24</a>).</p><p class="p p-last">This model also sheds light on the apparent paradox between processivity and total event lifetime. As nsp13 progresses through the various states of its ATPase cycle, it is driven forward along the DNA strand by accompanying changes to its energy landscape (<a href="#B39" rid="B39" class=" bibr popnode">39</a>). These changes in energy landscape manifest in-part due to changes in the strength of binding between the recA-like walker domains and the DNA backbone. With this in mind, it's reasonable to assume that different states within the ATPase cycle have different rates of dissociation from the DNA (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S25</a>). When there is only one or two rate limiting steps, nsp13 spends the vast majority of its time within one or two states. For example, during ATP-driven unwinding, nsp13 spends nearly all of its time in state IV, with ADP bound waiting for domain 1A to invade the dsDNA duplex. Or during ATP&#x003b3;S-driven unwinding nsp13 spends a significant amount of time in state III, waiting for ATP&#x003b3;S hydrolysis, and state IV. With such models, it's possible to derive expressions for observables such as dwell time and per-step probability of dissociation (<a href="#B28" rid="B28" class=" bibr popnode">28</a>) as a function of the underlying kinetic rate constants (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S26</a>). Using these expressions, we performed maximum likelihood fitting to derive estimates for the rate of unwinding, <em>k</em><sub>unwind</sub>&#x000a0;=&#x000a0;213&#x000a0;&#x000b1;&#x000a0;8 s<sup>&#x02212;1</sup>, the rate of ATP&#x003b3;S hydrolysis, <em>k</em><sub>h &#x003b3;S</sub>&#x000a0;= 51&#x000a0;&#x000b1;&#x000a0;8&#x000a0;s<sup>&#x02212;1</sup>, the off rate in state III, <em>k</em><sub>off III</sub>&#x000a0;= 5&#x000a0;&#x000b1;&#x000a0;1&#x000a0;s<sup>&#x02212;1</sup> and the off rate in state IV, <em>k</em><sub>off IV</sub>&#x000a0;= 22&#x000a0;&#x000b1;&#x000a0;1&#x000a0;s<sup>&#x02212;1</sup>. From these, it becomes clear how nsp13 can have both shorter processivity but spend more time bound to DNA during ATP&#x003b3;S-driven unwinding. While using ATP&#x003b3;S, nsp13 spends more time in state III, awaiting hydrolysis, which has a significantly lower off rate than state IV. This means on average a single nsp13 molecule will stay bound to the DNA considerably longer. Still in state III there is a non-zero chance of dissociation while the off rate in state IV remains unchanged, so the per-step probability of dissociation is higher, resulting in lower processivity.</p></div></div><div id="SEC4" class="tsec sec"><h2 class="head no_bottom_margin" id="SEC4title" style="text-transform: uppercase;">DISCUSSION</h2><p class="p p-first">While previous biochemical data show nsp13 ATPase rate is enhanced up to 25-fold in the presence of single-stranded nucleic acid (<a href="#B17" rid="B17" class=" bibr popnode">17</a>), there has been no measurement of nsp13 translocation speed until now. Unwinding speed is &#x0223c;8-fold slower compared to translocation; this ratio&#x000a0;is similar to that recently reported for SF1A helicase PcrA using SPRNT (&#x0223c;4-fold), and consistent with <em>in vitro</em> studies of other passive helicases (<a href="#B21" rid="B21" class=" bibr popnode">21</a>,<a href="#B40" rid="B40" class=" bibr popnode">40</a>,<a href="#B41" rid="B41" class=" bibr popnode">41</a>). Notably, the nsp13 unwinding speed measured here is similar to the observed speed of SARS-CoV-2 RdRp with which nsp13 forms a complex: &#x0223c;170 nt/s at 37&#x000b0;C as determined by single-molecule magnetic tweezers (<a href="#B42" rid="B42" class=" bibr popnode">42</a>).</p><p>Our assisting-force unwinding data shows speeds about 10-fold greater than estimates of bulk unwinding speed. However, our average unwinding speed is comparable to the unwinding speeds achieved using optical tweezers to destabilize the RNA duplex (<a href="#B16" rid="B16" class=" bibr popnode">16</a>). The manner by which force is applied with SPRNT differs significantly from how force was applied in the optical tweezers study (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S27</a>). In SPRNT, force affects the advance of domain 2A following ATP-binding whereas in the optical tweezers assay, force destabilizes the RNA duplex. It is possible that in the absence of force, domain 2A advance is rate-limiting as is apparent with ATP&#x003b3;S at zero force (bulk). Still, all other kinetic steps observed here are unaffected by the force applied in the SPRNT experiment because force can only affect steps in which the applied force performs work on the system. Within this framework, it is possible that nsp13 dissociation is affected by the applied force, but the effect is small in the range of forces sampled here (Figure <a href="/pmc/articles/PMC10516658/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5G).</span></span><span>5G</span></a>). It is also possible that the forces applied in our experiment distort nsp13&#x02019;s structure in a way that affects underlying rate constants. Though whether such distortion would increase or decrease these rate constants is difficult to predict. Because nsp13 speed remains relatively constant across the range of forces used (Figures <a href="/pmc/articles/PMC10516658/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -4em;">(Figures3D,</span></span><span>3D</span></a>, <a href="/pmc/articles/PMC10516658/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,5F),</span></span><span>5F</span></a>), we conclude such effects are small.</p><p>The application of an assisting force to nsp13 may have biological relevance. Interactions between nsp13 and the SARS-CoV-2 RdRp have been shown to play regulatory roles that affect unwinding activity (<a href="#B13" rid="B13" class=" bibr popnode">13</a>,<a href="#B14" rid="B14" class=" bibr popnode">14</a>,<a href="#B19" rid="B19" class=" bibr popnode">19</a>,<a href="#B43" rid="B43" class=" bibr popnode">43</a>,<a href="#B44" rid="B44" class=" bibr popnode">44</a>). Depending on how RdRp and nsp13 function <em>in vivo</em> it's possible that RdRp applies some assisting force to nsp13 (<a href="#B16" rid="B16" class=" bibr popnode">16</a>,<a href="#B43" rid="B43" class=" bibr popnode">43</a>). Cryo-EM structures with nsp7, nsp8&#x000a0;and nsp12&#x000a0;show a pair of nsp13 associated with the Replication-Transcription Complex (RTC) in which one nsp13 is also bound to an RNA strand. In these structures, the RNA-bound nsp13 has a 1B-domain which is swiveled closer to domains 1A-2A while in the other the 1B domain is further apart from domains 1A-2A. The closed state is&#x000a0;stabilized sterically by the second nsp13 molecule in the RTC. This suggests that interaction with the 1B-domain, such that it is in a closed state, can favor RNA binding and conceivably lead to a more processive helicase (<a href="#B13" rid="B13" class=" bibr popnode">13</a>). Similarly, engineering the DNA groove to be in a closed conformation has been shown to increase unwinding activity for SF1A helicases such as PcrA and Rep (<a href="#B45" rid="B45" class=" bibr popnode">45</a>). Contact between nsp13 and the nanopore rim may be playing a similar role. A similar study in which the conformation of the nsp13 1B-domain is constrained could shed further light on the role of the 1B-domain.</p><p>Unlike many single molecule techniques, SPRNT is uniquely capable of studying kinetics of translocation of a helicase in the absence of a duplex. While in this study, nsp13 translocation was measured only with assisting forces, reversing the orientation such that nsp13 is pulling DNA out of&#x02013;rather than lowering it into&#x02013;the pore would create an opposing force on domain 1A (<a href="#B21" rid="B21" class=" bibr popnode">21</a>). This would directly mimic how the RTC complexes may apply force <em>in vivo</em> (<a href="#B13" rid="B13" class=" bibr popnode">13</a>,<a href="#B43" rid="B43" class=" bibr popnode">43</a>,<a href="#B44" rid="B44" class=" bibr popnode">44</a>), enabling us to study how nsp13 works against an opposing force of a DNA duplex and to determine what might occur in a head-collision with the RdRp. SPRNT thus provides a valuable platform for investigating nsp13&#x02019;s role within the SARS-CoV-2 RTC.</p><p>Both translocation and unwinding with nsp13 appears to be strongly dependent upon the underlying DNA sequence. Several kinetic steps of the inchworm mechanism appear to be affected by the underlying DNA sequence. For example, during dsDNA unwinding, the advance of domain 1A is rate limiting and sequence-dependent (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S17</a>), while during ssDNA translocation with ATP&#x003b3;S, hydrolysis of ATP&#x003b3;S is rate-limiting and sequence-dependent in a different manner (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S20</a>). Similar sequence-dependent behavior has been found in all other helicases measured with SPRNT (Hel308, Rep, PcrA) (<a href="#B21" rid="B21" class=" bibr popnode">21</a>,<a href="#B31" rid="B31" class=" bibr popnode">31</a>) and is evident in optical tweezers data with UvrD (<a href="#B32" rid="B32" class=" bibr popnode">32</a>). With other helicases, we have found that nearly every kinetic rate constant within the ATPase cycle displays some sequence dependence. Similar mechanisms may be at work here. Cryo-EM structures of nsp13 with an ATP analog and RNA in concert with RdRP (PDB ID: 7RDY) shows no base-specific interactions between the protein and RNA (<a href="#B13" rid="B13" class=" bibr popnode">13</a>). Another hypothesis by which a helicase could have sequence-dependent translocation kinetics may be that differing steric effects of bases (rather than their binding affinities) in the NA binding channel allosterically influence the helicase's conformational microstate thereby affecting various kinetic rates. Viral genomes are also highly structured such that particular sequence elements may play an important role in nsp13 regulation. SPRNT experiments with nsp13 on native SARS-CoV-2 sequences could shed light on such interactions.</p><p>The DNA bases displayed in Figure <a href="/pmc/articles/PMC10516658/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure1D</span></span><span>1D</span></a> are those within the pore constriction so bases responsible for nsp13 sequence dependence are some distance 3&#x02032; of the pore, likely 3&#x02013;12 bases from the pore constriction (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S3</a>). The underlying cause of nsp13 sequence dependence appears to be complex and due to more than one single base at a single location within nsp13. A systematic study of nsp13 as it translocates over and unwinds several different sequence contexts should shed light on how different bases affect nsp13 kinetics (<a href="#B31" rid="B31" class=" bibr popnode">31</a>).</p><p class="p p-last">SPRNT&#x02019;s ability to determine how nsp13 works also sheds light on how it can be stopped from working. ATP&#x003b3;S is not a viable drug candidate for nsp13 because it is a general ATPase inhibitor, but we use ATP&#x003b3;S here to demonstrate the power of SPRNT in examining the mechanisms of helicase inhibition in exquisite detail. Within the inchworm mechanism, several steps could conceivably be affected by substitution of ATP&#x003b3;S for ATP: nucleotide binding, domain 2A motion after nucleotide has bound, and nucleotide hydrolysis could all conceivably be affected. Our measurements allowed us to determine that all three steps are affected: (i) ATP&#x003b3;S binding was faster than that of ATP, (ii) in the absence of force, ATP&#x003b3;S prevents domain 1A and 2A from coming together after nucleotide binding, acting as a competitive inhibitor and (iii) in the presence of a small assisting force, domains 1A and 2A can come together quickly but hydrolysis of ATP&#x003b3;S is slower than ATP hydrolysis. Similar studies with other general ATPase inhibitors such as AMP-PNP could highlight other mechanisms of inhibition or ways to enhance the inhibition seen here. Similar studies with promising drug candidates could be used to precisely determine the mechanism of inhibition and aid in development of more effective inhibitors.</p></div><div id="sec-a.l.e" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="sec-a.l.etitle" style="text-transform: uppercase;">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="sup1"><h4>gkad660_Supplemental_File</h4><div class="sup-box half_rhythm" id="media-a.l.e.b.b"><a href="/pmc/articles/PMC10516658/bin/gkad660_supplemental_file.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(10M, pdf)</sup></div></div></div><div id="ACK1" class="tsec sec"><h2 class="head no_bottom_margin" id="ACK1title" style="text-transform: uppercase;">ACKNOWLEDGEMENTS</h2><div class="sec"><p>S.K.M., J.M.C., C.A.T., A.M.P., S.J.A., J.D.C., M.C.F., J.R.H., H.C.K., H.B., J.H.G. and A.H.L. were supported by the National Institutes of Health, National Human Genome Research Institute (grant number R01HG005115). K.J.M. is supported by a National Cancer Institute K00 Fellowship (grant number K00CA223018). T.M.K. was supported by the National Institutes of Health, National Institute of General Medical Sciences for funding (grant number R35GM130234).</p></div></div><div id="article-aaff-info" class="tsec sec"><h2 class="head no_bottom_margin" id="article-aaff-infotitle" style="text-transform: uppercase;">Contributor Information</h2><p><span class="fm-affl">Sinduja K Marx, </span><span class="fm-affl" id="aff-a.k.b.f.a.b"> 
Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA.</span></p><p><span class="fm-affl">Keith J Mickolajczyk, </span><span class="fm-affl" id="aff-a.k.b.f.b.b"> 
Laboratory of Chemistry and Cell Biology, The Rockefeller University, New York, NY, USA.</span><span class="fm-affl" id="aff-a.k.b.f.b.c"> 
Department of Biochemistry and Molecular Biology, Robert Wood Johnson Medical School, Rutgers University, Piscataway, NJ, USA.</span></p><p><span class="fm-affl">Jonathan&#x000a0;M Craig, </span><span class="fm-affl" id="aff-a.k.b.f.c.c"> 
Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA.</span></p><p><span class="fm-affl">Christopher&#x000a0;A Thomas, </span><span class="fm-affl" id="aff-a.k.b.f.d.c"> 
Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA.</span></p><p><span class="fm-affl">Akira&#x000a0;M Pfeffer, </span><span class="fm-affl" id="aff-a.k.b.f.e.b"> 
Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA.</span></p><p><span class="fm-affl">Sarah&#x000a0;J Abell, </span><span class="fm-affl" id="aff-a.k.b.f.f.b"> 
Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA.</span></p><p><span class="fm-affl">Jessica&#x000a0;D Carrasco, </span><span class="fm-affl" id="aff-a.k.b.f.g.b"> 
Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA.</span></p><p><span class="fm-affl">Michaela&#x000a0;C Franzi, </span><span class="fm-affl" id="aff-a.k.b.f.h.b"> 
Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA.</span></p><p><span class="fm-affl">Jesse&#x000a0;R Huang, </span><span class="fm-affl" id="aff-a.k.b.f.i.c"> 
Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA.</span></p><p><span class="fm-affl">Hwanhee&#x000a0;C Kim, </span><span class="fm-affl" id="aff-a.k.b.f.j.c"> 
Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA.</span></p><p><span class="fm-affl">Henry Brinkerhoff, </span><span class="fm-affl" id="aff-a.k.b.f.k.c"> 
Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA.</span></p><p><span class="fm-affl">Tarun&#x000a0;M Kapoor, </span><span class="fm-affl" id="aff-a.k.b.f.l.b"> 
Laboratory of Chemistry and Cell Biology, The Rockefeller University, New York, NY, USA.</span></p><p><span class="fm-affl">Jens&#x000a0;H Gundlach, </span><span class="fm-affl" id="aff-a.k.b.f.m.b"> 
Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA.</span></p><p><span class="fm-affl">Andrew&#x000a0;H Laszlo, </span><span class="fm-affl" id="aff-a.k.b.f.n.c"> 
Department of Physics, University of Washington, Seattle, WA&#x000a0;98195, USA.</span></p></div><div id="SEC5" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="SEC5title" style="text-transform: uppercase;">Data Availability</h2><!--/article/back/sec/--><p class="p p-first-last">Data required to reproduce the results described herin and associated analysis code have been deposited in Figshare at DOI: 10.6084/m9.figshare.21936429.</p></div><div id="SEC6" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="SEC6title" style="text-transform: uppercase;">SUPPLEMENTARY DATA</h2><!--/article/back/sec/--><p class="p p-first-last">
<a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkad660#supplementary-data" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Data</a> are available at NAR Online.</p></div><div id="SEC7" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="SEC7title" style="text-transform: uppercase;">FUNDING</h2><!--/article/back/sec/--><p class="p p-first">National Cancer Institute [K00CA223018]; National Institutes of Health, National Institute of General Medical Sciences [R35GM130234]; National Institutes of Health, National Human Genome Research Institute [R01HG005115]. Funding for open access charge: National Institutes of Health - National Human Genome Research Institute [R01HG005115].</p><p class="p p-last">
<em>Conflict of interest statement</em>. The University of Washington has filed a patent application on the SPRNT technology.</p></div><div id="REF1" class="tsec sec"><h2 class="head no_bottom_margin" id="REF1title" style="text-transform: uppercase;">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1">1. <span class="mixed-citation">
Hall M.D., Anderson J.M., Anderson A., Baker D., Bradner J., Brimacombe K.R., Campbell E.A., Corbett K.S., Carter K., Cherry S.et al... 
<span class="ref-title">Report of the National Institutes of Health SARS-CoV-2 Antiviral Therapeutics Summit</span>. <span class="ref-journal">J. Infect. Dis.</span> 2021; <span class="ref-vol">224</span>(<span class="ref-iss">Suppl. 1</span>):S1&#x02013;S21. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8280938/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34111271" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Infect.+Dis.&amp;title=Report+of+the+National+Institutes+of+Health+SARS-CoV-2+Antiviral+Therapeutics+Summit&amp;volume=224&amp;issue=Suppl.+1&amp;publication_year=2021&amp;pages=S1-S21&amp;pmid=34111271&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B2">2. <span class="mixed-citation">
Malone B., Urakova N., Snijder E.J., Campbell E.A.. 
<span class="ref-title">Structures and functions of coronavirus replication&#x02013;transcription complexes and their relevance for SARS-CoV-2 drug design</span>. <span class="ref-journal">Nat. Rev. Mol. Cell Biol.</span> 2022; <span class="ref-vol">23</span>:21&#x02013;39. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8613731/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34824452" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Mol.+Cell+Biol.&amp;title=Structures+and+functions+of+coronavirus+replication&#x02013;transcription+complexes+and+their+relevance+for+SARS-CoV-2+drug+design&amp;volume=23&amp;publication_year=2022&amp;pages=21-39&amp;pmid=34824452&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B3">3. <span class="mixed-citation">
Seybert A., Posthuma C.C., van&#x000a0;Dinten L.C., Snijder E.J., Gorbalenya A.E., Ziebuhr J.. 
<span class="ref-title">A complex zinc finger controls the enzymatic activities of nidovirus helicases</span>. <span class="ref-journal">J. Virol.</span> 2005; <span class="ref-vol">79</span>:696&#x02013;704. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC538568/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15613297" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Virol.&amp;title=A+complex+zinc+finger+controls+the+enzymatic+activities+of+nidovirus+helicases&amp;volume=79&amp;publication_year=2005&amp;pages=696-704&amp;pmid=15613297&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B4">4. <span class="mixed-citation">
Zhang R., Li Y., Cowley T.J., Steinbrenner A.D., Phillips J.M., Yount B.L., Baric R.S., Weiss S.R.. 
<span class="ref-title">The nsp1, nsp13, and M proteins contribute to the hepatotropism of murine coronavirus JHM.WU</span>. <span class="ref-journal">J. Virol.</span> 2015; <span class="ref-vol">89</span>:3598&#x02013;3609. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4403414/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25589656" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Virol.&amp;title=The+nsp1,+nsp13,+and+M+proteins+contribute+to+the+hepatotropism+of+murine+coronavirus+JHM.WU&amp;volume=89&amp;publication_year=2015&amp;pages=3598-3609&amp;pmid=25589656&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B5">5. <span class="mixed-citation">
Wiegand T., Nemudryi A., Nemudraia A., McVey A., Little A., Taylor D.N., Walk S.T., Wiedenheft B.. 
<span class="ref-title">The rise and fall of SARS-CoV-2 variants and ongoing diversification of omicron</span>. <span class="ref-journal">Viruses</span>. 2022; <span class="ref-vol">14</span>:2009. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9505243/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36146815" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Viruses&amp;title=The+rise+and+fall+of+SARS-CoV-2+variants+and+ongoing+diversification+of+omicron&amp;volume=14&amp;publication_year=2022&amp;pages=2009&amp;pmid=36146815&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B6">6. <span class="mixed-citation">
Wu C., Liu Y., Yang Y., Zhang P., Zhong W., Wang Y., Wang Q., Xu Y., Li M., Li X.et al... 
<span class="ref-title">Analysis of therapeutic targets for SARS-CoV-2 and discovery of potential drugs by computational methods</span>. <span class="ref-journal">Acta Pharm. Sin. B</span>. 2020; <span class="ref-vol">10</span>:776&#x02013;788. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7102550/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32292689" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Pharm.+Sin.+B&amp;title=Analysis+of+therapeutic+targets+for+SARS-CoV-2+and+discovery+of+potential+drugs+by+computational+methods&amp;volume=10&amp;publication_year=2020&amp;pages=776-788&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B7">7. <span class="mixed-citation">
Yang N., Tanner J.A., Zheng B.-J., Watt R.M., He M.-L., Lu L.-Y., Jiang J.-Q., Shum K.-T., Lin Y.-P., Wong K.-L.et al... 
<span class="ref-title">Bismuth complexes inhibit the SARS coronavirus</span>. <span class="ref-journal">Angew. Chem. Int. Ed.</span> 2007; <span class="ref-vol">46</span>:6464&#x02013;6468. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7159583/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17645269" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Angew.+Chem.+Int.+Ed.&amp;title=Bismuth+complexes+inhibit+the+SARS+coronavirus&amp;volume=46&amp;publication_year=2007&amp;pages=6464-6468&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B8">8. <span class="mixed-citation">
Adedeji A.O., Singh K., Kassim A., Coleman C.M., Elliott R., Weiss S.R., Frieman M.B., Sarafianos S.G.. 
<span class="ref-title">Evaluation of SSYA10-001 as a replication inhibitor of severe acute respiratory syndrome, mouse hepatitis, and Middle East Respiratory Syndrome Coronaviruses</span>. <span class="ref-journal">Antimicrob. Agents Chemother.</span> 2014; <span class="ref-vol">58</span>:4894&#x02013;4898. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4136041/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24841268" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Antimicrob.+Agents+Chemother.&amp;title=Evaluation+of+SSYA10-001+as+a+replication+inhibitor+of+severe+acute+respiratory+syndrome,+mouse+hepatitis,+and+Middle+East+Respiratory+Syndrome+Coronaviruses&amp;volume=58&amp;publication_year=2014&amp;pages=4894-4898&amp;pmid=24841268&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B9">9. <span class="mixed-citation">
Yang N., Tanner J.A., Wang Z., Huang J.-D., Zheng B.-J., Zhu N., Sun H.. 
<span class="ref-title">Inhibition of SARS coronavirus helicase by bismuth complexes</span>. <span class="ref-journal">Chem. Commun.</span> 2007; <span class="ref-vol">2007</span>:4415&#x02013;4413. [<a href="https://pubmed.ncbi.nlm.nih.gov/17957304" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem.+Commun.&amp;title=Inhibition+of+SARS+coronavirus+helicase+by+bismuth+complexes&amp;volume=2007&amp;publication_year=2007&amp;pages=4415-4413&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B10">10. <span class="mixed-citation">
Corona A., Wycisk K., Talarico C., Manelfi C., Milia J., Cannalire R., Esposito F., Gribbon P., Zaliani A., Iaconis D.et al... 
<span class="ref-title">Natural compounds inhibit SARS-CoV-2 nsp13 unwinding and ATPase enzyme activities</span>. <span class="ref-journal">ACS Pharmacol. Transl. Sci.</span> 2022; <span class="ref-vol">5</span>:226&#x02013;239. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9003574/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35434533" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=ACS+Pharmacol.+Transl.+Sci.&amp;title=Natural+compounds+inhibit+SARS-CoV-2+nsp13+unwinding+and+ATPase+enzyme+activities&amp;volume=5&amp;publication_year=2022&amp;pages=226-239&amp;pmid=35434533&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B11">11. <span class="mixed-citation">
Adedeji A.O., Singh K., Calcaterra N.E., DeDiego M.L., Enjuanes L., Weiss S., Sarafianos S.G.. 
<span class="ref-title">Severe acute respiratory syndrome coronavirus replication inhibitor that interferes with the nucleic acid unwinding of the viral helicase</span>. <span class="ref-journal">Antimicrob. Agents Chemother.</span> 2012; <span class="ref-vol">56</span>:4718&#x02013;4728. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3421890/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22733076" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Antimicrob.+Agents+Chemother.&amp;title=Severe+acute+respiratory+syndrome+coronavirus+replication+inhibitor+that+interferes+with+the+nucleic+acid+unwinding+of+the+viral+helicase&amp;volume=56&amp;publication_year=2012&amp;pages=4718-4728&amp;pmid=22733076&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B12">12. <span class="mixed-citation">
Tanner J.A., Zheng B.-J., Zhou J., Watt R.M., Jiang J.-Q., Wong K.-L., Lin Y.-P., Lu L.-Y., He M.-L., Kung H.-F.et al... 
<span class="ref-title">The adamantane-derived bananins are potent inhibitors of the helicase activities and replication of SARS coronavirus</span>. <span class="ref-journal">Chem. Biol.</span> 2005; <span class="ref-vol">12</span>:303&#x02013;311. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7110988/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15797214" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem.+Biol.&amp;title=The+adamantane-derived+bananins+are+potent+inhibitors+of+the+helicase+activities+and+replication+of+SARS+coronavirus&amp;volume=12&amp;publication_year=2005&amp;pages=303-311&amp;pmid=15797214&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B13">13. <span class="mixed-citation">
Chen J., Wang Q., Malone B., Llewellyn E., Pechersky Y., Maruthi K., Eng E.T., Perry J.K., Campbell E.A., Shaw D.E.et al... 
<span class="ref-title">Ensemble cryo-EM reveals conformational states of the nsp13 helicase in the SARS-CoV-2 helicase replication&#x02013;transcription complex</span>. <span class="ref-journal">Nat. Struct. Mol. Biol.</span> 2022; <span class="ref-vol">29</span>:250&#x02013;260. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8935131/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35260847" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Struct.+Mol.+Biol.&amp;title=Ensemble+cryo-EM+reveals+conformational+states+of+the+nsp13+helicase+in+the+SARS-CoV-2+helicase+replication&#x02013;transcription+complex&amp;volume=29&amp;publication_year=2022&amp;pages=250-260&amp;pmid=35260847&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B14">14. <span class="mixed-citation">
Jia Z., Yan L., Ren Z., Wu L., Wang J., Guo J., Zheng L., Ming Z., Zhang L., Lou Z.et al... 
<span class="ref-title">Delicate structural coordination of the severe acute respiratory syndrome coronavirus Nsp13 upon ATP hydrolysis</span>. <span class="ref-journal">Nucleic Acids Res.</span> 2019; <span class="ref-vol">47</span>:6538&#x02013;6550. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6614802/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31131400" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Delicate+structural+coordination+of+the+severe+acute+respiratory+syndrome+coronavirus+Nsp13+upon+ATP+hydrolysis&amp;volume=47&amp;publication_year=2019&amp;pages=6538-6550&amp;pmid=31131400&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B15">15. <span class="mixed-citation">
Newman J.A., Douangamath A., Yadzani S., Yosaatmadja Y., Aimon A., Brand&#x000e3;o-Neto J., Dunnett L., Gorrie-stone T., Skyner R., Fearon D.et al... 
<span class="ref-title">Structure, mechanism and crystallographic fragment screening of the SARS-CoV-2 NSP13 helicase</span>. <span class="ref-journal">Nat. Commun.</span> 2021; <span class="ref-vol">12</span>:4848. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8358061/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34381037" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Structure,+mechanism+and+crystallographic+fragment+screening+of+the+SARS-CoV-2+NSP13+helicase&amp;volume=12&amp;publication_year=2021&amp;pages=4848&amp;pmid=34381037&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B16">16. <span class="mixed-citation">
Mickolajczyk K.J., Shelton P.M.M., Grasso M., Cao X., Warrington S.E., Aher A., Liu S., Kapoor T.M.. 
<span class="ref-title">Force-dependent stimulation of RNA unwinding by SARS-CoV-2 nsp13 helicase</span>. <span class="ref-journal">Biophys. J.</span> 2021; <span class="ref-vol">120</span>:1020&#x02013;1030. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7837305/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33340543" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biophys.+J.&amp;title=Force-dependent+stimulation+of+RNA+unwinding+by+SARS-CoV-2+nsp13+helicase&amp;volume=120&amp;publication_year=2021&amp;pages=1020-1030&amp;pmid=33340543&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B17">17. <span class="mixed-citation">
Tanner J.A., Watt R.M., Chai Y.-B., Lu L.-Y., Lin M.C., Peiris J.S.M., Poon L.L.M., Kung H.-F., Huang J.-D.. 
<span class="ref-title">The severe acute respiratory syndrome (SARS) coronavirus NTPase/helicase belongs to a distinct class of 5&#x02032; to 3&#x02032; viral helicases</span>. <span class="ref-journal">J. Biol. Chem.</span> 2003; <span class="ref-vol">278</span>:39578&#x02013;39582. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8060950/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12917423" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=The+severe+acute+respiratory+syndrome+(SARS)+coronavirus+NTPase/helicase+belongs+to+a+distinct+class+of+5&#x02032;+to+3&#x02032;+viral+helicases&amp;volume=278&amp;publication_year=2003&amp;pages=39578-39582&amp;pmid=12917423&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B18">18. <span class="mixed-citation">
Ivanov K.A., Thiel V., Dobbe J.C., van&#x000a0;der&#x000a0;Meer Y., Snijder E.J., Ziebuhr J.. 
<span class="ref-title">Multiple enzymatic activities associated with severe acute respiratory syndrome coronavirus helicase</span>. <span class="ref-journal">J. Virol.</span> 2004; <span class="ref-vol">78</span>:5619&#x02013;5632. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC415832/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15140959" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Virol.&amp;title=Multiple+enzymatic+activities+associated+with+severe+acute+respiratory+syndrome+coronavirus+helicase&amp;volume=78&amp;publication_year=2004&amp;pages=5619-5632&amp;pmid=15140959&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B19">19. <span class="mixed-citation">
Adedeji A.O., Marchand B., te&#x000a0;Velthuis A.J.W., Snijder E.J., Weiss S., Eoff R.L., Singh K., Sarafianos S.G.. 
<span class="ref-title">Mechanism of nucleic acid unwinding by SARS-CoV helicase</span>. <span class="ref-journal">PLoS One</span>. 2012; <span class="ref-vol">7</span>:e36521. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3352918/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22615777" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Mechanism+of+nucleic+acid+unwinding+by+SARS-CoV+helicase&amp;volume=7&amp;publication_year=2012&amp;pages=e36521&amp;pmid=22615777&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B20">20. <span class="mixed-citation">
Derrington I.M., Craig J.M., Stava E., Laszlo A.H., Ross B.C., Brinkerhoff H., Nova I.C., Doering K., Tickman B.I., Ronaghi M.et al... 
<span class="ref-title">Subangstrom single-molecule measurements of motor proteins using a nanopore</span>. <span class="ref-journal">Nat. Biotechnol.</span> 2015; <span class="ref-vol">33</span>:1073&#x02013;1075. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4915380/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26414351" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=Subangstrom+single-molecule+measurements+of+motor+proteins+using+a+nanopore&amp;volume=33&amp;publication_year=2015&amp;pages=1073-1075&amp;pmid=26414351&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B21">21. <span class="mixed-citation">
Laszlo A.H., Craig J.M., Gavrilov M., Tippana R., Nova I.C., Huang J.R., Kim H.C., Abell S.J., deCampos-Stairiker M., Mount J.W.et al... 
<span class="ref-title">Sequence-dependent mechanochemical coupling of helicase translocation and unwinding at single-nucleotide resolution</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span> 2022; <span class="ref-vol">119</span>:e2202489119. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9457475/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36037333" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Sequence-dependent+mechanochemical+coupling+of+helicase+translocation+and+unwinding+at+single-nucleotide+resolution&amp;volume=119&amp;publication_year=2022&amp;pages=e2202489119&amp;pmid=36037333&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B22">22. <span class="mixed-citation">
Manrao E.A., Derrington I.M., Laszlo A.H., Langford K.W., Hopper M.K., Gillgren N., Pavlenok M., Niederweis M., Gundlach J.H.. 
<span class="ref-title">Reading DNA at single-nucleotide resolution with a mutant MspA nanopore and phi29 DNA polymerase</span>. <span class="ref-journal">Nat. Biotechnol.</span> 2012; <span class="ref-vol">30</span>:349&#x02013;353. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3757088/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22446694" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=Reading+DNA+at+single-nucleotide+resolution+with+a+mutant+MspA+nanopore+and+phi29+DNA+polymerase&amp;volume=30&amp;publication_year=2012&amp;pages=349-353&amp;pmid=22446694&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B23">23. <span class="mixed-citation">
Butler T.Z., Pavlenok M., Derrington I.M., Niederweis M., Gundlach J.H.. 
<span class="ref-title">Single-molecule DNA detection with an engineered MspA protein nanopore</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span> 2008; <span class="ref-vol">105</span>:20647&#x02013;20652. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2634888/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19098105" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Single-molecule+DNA+detection+with+an+engineered+MspA+protein+nanopore&amp;volume=105&amp;publication_year=2008&amp;pages=20647-20652&amp;pmid=19098105&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B24">24. <span class="mixed-citation">
Derrington I.M., Butler T.Z., Collins M.D., Manrao E., Pavlenok M., Niederweis M., Gundlach J.H.. 
<span class="ref-title">Nanopore DNA sequencing with MspA</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span> 2010; <span class="ref-vol">107</span>:16060&#x02013;16065. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2941267/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20798343" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Nanopore+DNA+sequencing+with+MspA&amp;volume=107&amp;publication_year=2010&amp;pages=16060-16065&amp;pmid=20798343&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B25">25. <span class="mixed-citation">
Laszlo A.H., Derrington I.M., Gundlach J.H.. 
<span class="ref-title">MspA nanopore as a single-molecule tool: from sequencing to SPRNT</span>. <span class="ref-journal">Methods</span>. 2016; <span class="ref-vol">105</span>:75&#x02013;89. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4967004/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27045943" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods&amp;title=MspA+nanopore+as+a+single-molecule+tool:+from+sequencing+to+SPRNT&amp;volume=105&amp;publication_year=2016&amp;pages=75-89&amp;pmid=27045943&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B26">26. <span class="mixed-citation">
Laszlo A.H., Derrington I.M., Ross B.C., Brinkerhoff H., Adey A., Nova I.C., Craig J.M., Langford K.W., Samson J.M., Daza R.. 
<span class="ref-title">Decoding long nanopore sequencing reads of natural DNA</span>. <span class="ref-journal">Nat. Biotechnol.</span> 2014; <span class="ref-vol">32</span>:829. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4126851/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24964173" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=Decoding+long+nanopore+sequencing+reads+of+natural+DNA&amp;volume=32&amp;publication_year=2014&amp;pages=829&amp;pmid=24964173&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B27">27. <span class="mixed-citation">
Durbin R., Eddy S.R., Krogh A., Mitchison G.. 
<span class="ref-journal">Biological Sequence Analysis: Probabilistic Models of Proteins and Nucleic Acids</span>. 1998; Cambridge University Press. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Biological+Sequence+Analysis:+Probabilistic+Models+of+Proteins+and+Nucleic+Acids&amp;publication_year=1998&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B28">28. <span class="mixed-citation">
Craig J.M., Laszlo A.H., Nova I.C., Gundlach J.H.. 
<span class="ref-title">Modelling single-molecule kinetics of helicase translocation using high-resolution nanopore tweezers (SPRNT)</span>. <span class="ref-journal">Essays Biochem.</span> 2021; <span class="ref-vol">65</span>:109&#x02013;127. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8722770/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33491732" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Essays+Biochem.&amp;title=Modelling+single-molecule+kinetics+of+helicase+translocation+using+high-resolution+nanopore+tweezers+(SPRNT)&amp;volume=65&amp;publication_year=2021&amp;pages=109-127&amp;pmid=33491732&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B29">29. <span class="mixed-citation">
LaMont C.H., Wiggins P.A.. 
<span class="ref-title">The Development of an Information Criterion for Change-Point Analysis</span>. <span class="ref-journal">Neural Comput.</span> 2016; <span class="ref-vol">28</span>:594&#x02013;612. [<a href="https://pubmed.ncbi.nlm.nih.gov/26735741" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neural+Comput.&amp;title=The+Development+of+an+Information+Criterion+for+Change-Point+Analysis&amp;volume=28&amp;publication_year=2016&amp;pages=594-612&amp;pmid=26735741&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B30">30. <span class="mixed-citation">
Craig J.M., Laszlo A.H., Brinkerhoff H., Derrington I.M., Noakes M.T., Nova I.C., Tickman B.I., Doering K., de&#x000a0;Leeuw N.F., Gundlach J.H.. 
<span class="ref-title">Revealing dynamics of helicase translocation on single-stranded DNA using high-resolution nanopore tweezers</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span> 2017; <span class="ref-vol">114</span>:11932&#x02013;11937. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5692567/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29078357" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Revealing+dynamics+of+helicase+translocation+on+single-stranded+DNA+using+high-resolution+nanopore+tweezers&amp;volume=114&amp;publication_year=2017&amp;pages=11932-11937&amp;pmid=29078357&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B31">31. <span class="mixed-citation">
Craig J.M., Laszlo A.H., Nova I.C., Brinkerhoff H., Noakes M.T., Baker K.S., Bowman J.L., Higinbotham H.R., Mount J.W., Gundlach J.H.. 
<span class="ref-title">Determining the effects of DNA sequence on Hel308 helicase translocation along single-stranded DNA using nanopore tweezers</span>. <span class="ref-journal">Nucleic Acids Res.</span> 2019; <span class="ref-vol">47</span>:2506&#x02013;2513. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6412116/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30649515" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Determining+the+effects+of+DNA+sequence+on+Hel308+helicase+translocation+along+single-stranded+DNA+using+nanopore+tweezers&amp;volume=47&amp;publication_year=2019&amp;pages=2506-2513&amp;pmid=30649515&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B32">32. <span class="mixed-citation">
Tomko E.J., Lohman T.M.. 
<span class="ref-title">Modulation of Escherichia coli UvrD Single-Stranded DNA Translocation by DNA Base Composition</span>. <span class="ref-journal">Biophys. J.</span> 2017; <span class="ref-vol">113</span>:1405&#x02013;1415. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5627390/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28978435" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biophys.+J.&amp;title=Modulation+of+Escherichia+coli+UvrD+Single-Stranded+DNA+Translocation+by+DNA+Base+Composition&amp;volume=113&amp;publication_year=2017&amp;pages=1405-1415&amp;pmid=28978435&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B33">33. <span class="mixed-citation">
Betterton M.D., J&#x000fc;licher F.. 
<span class="ref-title">Opening of nucleic-acid double strands by helicases: active versus passive opening</span>. <span class="ref-journal">Phys. Rev. E Stat. Nonlin. Soft Matter Phys.</span> 2005; <span class="ref-vol">71</span>:011904. [<a href="https://pubmed.ncbi.nlm.nih.gov/15697627" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Phys.+Rev.+E+Stat.+Nonlin.+Soft+Matter+Phys.&amp;title=Opening+of+nucleic-acid+double+strands+by+helicases:+active+versus+passive+opening&amp;volume=71&amp;publication_year=2005&amp;pages=011904&amp;pmid=15697627&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B34">34. <span class="mixed-citation">
Manosas M., Xi X.G., Bensimon D., Croquette V.. 
<span class="ref-title">Active and passive mechanisms of helicases</span>. <span class="ref-journal">Nucleic Acids Res.</span> 2010; <span class="ref-vol">38</span>:5518&#x02013;5526. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2938219/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20423906" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Active+and+passive+mechanisms+of+helicases&amp;volume=38&amp;publication_year=2010&amp;pages=5518-5526&amp;pmid=20423906&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B35">35. <span class="mixed-citation">
Keller D., Bustamante C.. 
<span class="ref-title">The mechanochemistry of molecular motors</span>. <span class="ref-journal">Biophys. J.</span> 2000; <span class="ref-vol">78</span>:541&#x02013;556. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1300660/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10653770" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biophys.+J.&amp;title=The+mechanochemistry+of+molecular+motors&amp;volume=78&amp;publication_year=2000&amp;pages=541-556&amp;pmid=10653770&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B36">36. <span class="mixed-citation">
Peck M.L., Herschlag D.. 
<span class="ref-title">Adenosine 5&#x02032;-O-(3-thio)triphosphate (ATP&#x003b3;S) is a substrate for the nucleotide hydrolysis and RNA unwinding activities of eukaryotic translation initiation factor eIF4A</span>. <span class="ref-journal">RNA</span>. 2003; <span class="ref-vol">9</span>:1180&#x02013;1187. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1370482/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/13130132" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=RNA&amp;title=Adenosine+5&#x02032;-O-(3-thio)triphosphate+(ATP&#x003b3;S)+is+a+substrate+for+the+nucleotide+hydrolysis+and+RNA+unwinding+activities+of+eukaryotic+translation+initiation+factor+eIF4A&amp;volume=9&amp;publication_year=2003&amp;pages=1180-1187&amp;pmid=13130132&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B37">37. <span class="mixed-citation">
Yao N.Y., Zhang D., Yurieva O., O&#x02019;Donnell M.E. 
<span class="ref-title">CMG helicase can use ATP&#x003b3;S to unwind DNA: implications for the rate-limiting step in the reaction mechanism</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span> 2022; <span class="ref-vol">119</span>:e2119580119. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8794833/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35042821" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=CMG+helicase+can+use+ATP&#x003b3;S+to+unwind+DNA:+implications+for+the+rate-limiting+step+in+the+reaction+mechanism&amp;volume=119&amp;publication_year=2022&amp;pages=e2119580119&amp;pmid=35042821&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B38">38. <span class="mixed-citation">
Sommers J.A., Loftus L.N., Jones M.P., Lee R.A., Haren C.E., Dumm A.J., Brosh R.M.. 
<span class="ref-title">Biochemical analysis of SARS-CoV-2 Nsp13 helicase implicated in COVID-19 and factors that regulate its catalytic functions</span>. <span class="ref-journal">J. Biol. Chem.</span> 2023; <span class="ref-vol">299</span>:102980. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9897874/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36739951" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Biochemical+analysis+of+SARS-CoV-2+Nsp13+helicase+implicated+in+COVID-19+and+factors+that+regulate+its+catalytic+functions&amp;volume=299&amp;publication_year=2023&amp;pages=102980&amp;pmid=36739951&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B39">39. <span class="mixed-citation">
Craig J.M., Mills M., Kim H.C., Huang J.R., Abell S.J., Mount J.W., Gundlach J.H., Neuman K.C., Laszlo A.H.. 
<span class="ref-title">Nanopore tweezers measurements of RecQ conformational changes reveal the energy landscape of helicase motion</span>. <span class="ref-journal">Nucleic Acids Res.</span> 2022; <span class="ref-vol">50</span>:10601&#x02013;10613. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9561376/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36165957" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Nanopore+tweezers+measurements+of+RecQ+conformational+changes+reveal+the+energy+landscape+of+helicase+motion&amp;volume=50&amp;publication_year=2022&amp;pages=10601-10613&amp;pmid=36165957&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B40">40. <span class="mixed-citation">
Niedziela-Majka A., Chesnik M.A., Tomko E.J., Lohman T.M.. 
<span class="ref-title">Bacillus stearothermophilus PcrA monomer is a single-stranded DNA translocase but not a processive helicase in vitro</span>. <span class="ref-journal">J. Biol. Chem.</span> 2007; <span class="ref-vol">282</span>:27076&#x02013;27085. [<a href="https://pubmed.ncbi.nlm.nih.gov/17631491" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Bacillus+stearothermophilus+PcrA+monomer+is+a+single-stranded+DNA+translocase+but+not+a+processive+helicase+in+vitro&amp;volume=282&amp;publication_year=2007&amp;pages=27076-27085&amp;pmid=17631491&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B41">41. <span class="mixed-citation">
Fischer C.J., Maluf N.K., Lohman T.M.. 
<span class="ref-title">Mechanism of ATP-dependent translocation of <em>E. coli</em> UvrD monomers along single-stranded DNA</span>. <span class="ref-journal">J. Mol. Biol.</span> 2004; <span class="ref-vol">344</span>:1287&#x02013;1309. [<a href="https://pubmed.ncbi.nlm.nih.gov/15561144" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Mol.+Biol.&amp;title=Mechanism+of+ATP-dependent+translocation+of+E.+coli+UvrD+monomers+along+single-stranded+DNA&amp;volume=344&amp;publication_year=2004&amp;pages=1287-1309&amp;pmid=15561144&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B42">42. <span class="mixed-citation">
Seifert M., Bera S.C., van&#x000a0;Nies P., Kirchdoerfer R.N., Shannon A., Le T.-T.-N., Meng X., Xia H., Wood J.M., Harris L.D.et al... 
<span class="ref-title">Inhibition of SARS-CoV-2 polymerase by nucleotide analogs from a single-molecule perspective</span>. <span class="ref-journal">Elife</span>. 2021; <span class="ref-vol">10</span>:e70968. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8497053/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34617885" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Elife&amp;title=Inhibition+of+SARS-CoV-2+polymerase+by+nucleotide+analogs+from+a+single-molecule+perspective&amp;volume=10&amp;publication_year=2021&amp;pages=e70968&amp;pmid=34617885&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B43">43. <span class="mixed-citation">
Chen J., Malone B., Llewellyn E., Grasso M., Shelton P.M.M., Olinares P.D.B., Maruthi K., Eng E.T., Vatandaslar H., Chait B.T.et al... 
<span class="ref-title">Structural basis for helicase-polymerase coupling in the SARS-CoV-2 replication-transcription complex</span>. <span class="ref-journal">Cell</span>. 2020; <span class="ref-vol">182</span>:1560&#x02013;1573. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7386476/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32783916" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Structural+basis+for+helicase-polymerase+coupling+in+the+SARS-CoV-2+replication-transcription+complex&amp;volume=182&amp;publication_year=2020&amp;pages=1560-1573&amp;pmid=32783916&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B44">44. <span class="mixed-citation">
Yan L., Zhang Y., Ge J., Zheng L., Gao Y., Wang T., Jia Z., Wang H., Huang Y., Li M.et al... 
<span class="ref-title">Architecture of a SARS-CoV-2 mini replication and transcription complex</span>. <span class="ref-journal">Nat. Commun.</span> 2020; <span class="ref-vol">11</span>:5874. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7675986/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33208736" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Architecture+of+a+SARS-CoV-2+mini+replication+and+transcription+complex&amp;volume=11&amp;publication_year=2020&amp;pages=5874&amp;pmid=33208736&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B45">45. <span class="mixed-citation">
Arslan S., Khafizov R., Thomas C.D., Chemla Y.R., Ha T.. 
<span class="ref-title">Engineering of a superhelicase through conformational control</span>. <span class="ref-journal">Science</span>. 2015; <span class="ref-vol">348</span>:344&#x02013;347. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4417355/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25883358" ref="reftype=pubmed&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Engineering+of+a+superhelicase+through+conformational+control&amp;volume=348&amp;publication_year=2015&amp;pages=344-347&amp;pmid=25883358&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10516658&amp;issue-id=445860&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Nucleic Acids Research</span> are provided here courtesy of <strong>Oxford University Press</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC10516658/pdf/gkad660.pdf" class="int-view">PDF (1.7M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/10516658/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/10516658/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="10516658" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10516658%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="3XTSexpzeM2ynae9m1dJ1ugoTlY2htblckGu677lHVap9GpWKFChkOhuLWSIwIj8">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10516658%2F&amp;text=Observing%20inhibition%20of%20the%20SARS-CoV-2%20helicase%20at%20single-nucleotide%20resolution" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10516658%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10516658/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC10516658/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/37560916/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC10516658/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/37560916/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC10516658/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/10516658/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC10516658/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
