
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-preprint.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Control of Biofilm Formation by an Agrobacterium tumefaciens Pterin-Binding Periplasmic Protein Conserved Among Pathogenic Bacteria - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="322C300B24F3920500004B8B75C92C06.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="biorxiv" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC10680838/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10680838/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="bioRxiv" /><meta name="citation_title" content="Control of Biofilm Formation by an Agrobacterium tumefaciens Pterin-Binding Periplasmic Protein Conserved Among Pathogenic Bacteria" /><meta name="citation_author" content="Jennifer L. Greenwich" /><meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, IN 47405 USA" /><meta name="citation_author" content="Justin L. Eagan" /><meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, IN 47405 USA" /><meta name="citation_author" content="Nathan Feirer" /><meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, IN 47405 USA" /><meta name="citation_author" content="Kaleb Boswinkle" /><meta name="citation_author_institution" content="Department of Biochemistry, Virginia Polytechnic Institute and State University, Blacksburg, VA 24061 USA" /><meta name="citation_author" content="George Minasov" /><meta name="citation_author_institution" content="Department of Microbiology-Immunology, Feinberg School of Medicine, Northwestern University, Chicago, IL 60611 USA" /><meta name="citation_author_institution" content="Center for Structural Biology of Infectious Diseases, Feinberg School of Medicine, Northwestern University, Chicago, IL 60611 USA" /><meta name="citation_author" content="Ludmilla Shuvalova" /><meta name="citation_author_institution" content="Department of Pharmacology, Feinberg School of Medicine, Northwestern University, Chicago, IL 60611 USA" /><meta name="citation_author" content="Nicole L. Inniss" /><meta name="citation_author_institution" content="Department of Microbiology-Immunology, Feinberg School of Medicine, Northwestern University, Chicago, IL 60611 USA" /><meta name="citation_author_institution" content="Center for Structural Biology of Infectious Diseases, Feinberg School of Medicine, Northwestern University, Chicago, IL 60611 USA" /><meta name="citation_author" content="Jakka Raghavaiah" /><meta name="citation_author_institution" content="Departments of Chemistry and Medicinal Chemistry, Purdue University, West Lafayette, IN 47907 USA" /><meta name="citation_author" content="Arun K. Ghosh" /><meta name="citation_author_institution" content="Departments of Chemistry and Medicinal Chemistry, Purdue University, West Lafayette, IN 47907 USA" /><meta name="citation_author" content="Karla J.F. Satchell" /><meta name="citation_author_institution" content="Department of Microbiology-Immunology, Feinberg School of Medicine, Northwestern University, Chicago, IL 60611 USA" /><meta name="citation_author_institution" content="Center for Structural Biology of Infectious Diseases, Feinberg School of Medicine, Northwestern University, Chicago, IL 60611 USA" /><meta name="citation_author" content="Kylie D. Allen" /><meta name="citation_author_institution" content="Department of Biochemistry, Virginia Polytechnic Institute and State University, Blacksburg, VA 24061 USA" /><meta name="citation_author" content="Clay Fuqua" /><meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, IN 47405 USA" /><meta name="citation_doi" content="10.1101/2023.11.18.567607" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC10680838/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC10680838/" /><meta name="citation_pmid" content="38014264" /><meta name="DC.Title" content="Control of Biofilm Formation by an Agrobacterium tumefaciens Pterin-Binding Periplasmic Protein Conserved Among Pathogenic Bacteria" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Cold Spring Harbor Laboratory Preprints" /><meta name="DC.Contributor" content="Jennifer L. Greenwich" /><meta name="DC.Contributor" content="Justin L. Eagan" /><meta name="DC.Contributor" content="Nathan Feirer" /><meta name="DC.Contributor" content="Kaleb Boswinkle" /><meta name="DC.Contributor" content="George Minasov" /><meta name="DC.Contributor" content="Ludmilla Shuvalova" /><meta name="DC.Contributor" content="Nicole L. Inniss" /><meta name="DC.Contributor" content="Jakka Raghavaiah" /><meta name="DC.Contributor" content="Arun K. Ghosh" /><meta name="DC.Contributor" content="Karla J.F. Satchell" /><meta name="DC.Contributor" content="Kylie D. Allen" /><meta name="DC.Contributor" content="Clay Fuqua" /><meta name="DC.Identifier" content="10.1101/2023.11.18.567607" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Control of Biofilm Formation by an Agrobacterium tumefaciens Pterin-Binding Periplasmic Protein Conserved Among Pathogenic Bacteria" /><meta property="og:type" content="article" /><meta property="og:description" content="Biofilm formation and surface attachment in multiple Alphaproteobacteria is driven by unipolar polysaccharide (UPP) adhesins. The pathogen Agrobacterium tumefaciens produces a UPP adhesin, which is regulated by the intracellular second messenger cyclic ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10680838/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" /><meta name="ncbi_pcid" content="preprint" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10680838/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=biorxiv[jour]" class="navlink">bioRxiv</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC10680838
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC10680838/pdf/nihpp-2023.11.18.567607v2.pdf" class="int-view">PDF (2.5M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/10680838/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/10680838/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="10680838" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10680838%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="RCtfgmZ0mLiWxEl9xZLCmdFdeR32hNdlqPGcDXIIphJIsWhyskCxVP6Tf7EumLND">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10680838%2F&amp;text=Control%20of%20Biofilm%20Formation%20by%20an%20Agrobacterium%20tumefaciens%20Pterin-Binding%20Periplasmic%20Protein%20Conserved%20Among%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10680838%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10680838/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC10680838/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38014264/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC10680838/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38014264/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC10680838/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/10680838/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=biorxiv[jour]" class="navlink">bioRxiv</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC10680838
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-preprints.png" alt="Logo of biorxiv" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="https://www.ncbi.nlm.nih.gov/pmc/about/nihpreprints/" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1">Version 2. <span id="pmcmata">bioRxiv.</span> Preprint. 2023 Nov 21.</div><div class="part2">  <span class="doi"><span>doi: </span><a href="//doi.org/10.1101%2F2023.11.18.567607" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1101/2023.11.18.567607</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC10680838</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38014264">38014264</a></div></div></div><div class="pmc-alert-box preprint-alert whole_rhythm" role="alert" aria-label="Preprint"><div class="i-mark"></div><h3 style="color:#000;margin:0;font-family:'Helvetica Neue','Helvetica','Arial',sans-serif">This is a preprint.</h3><div><strong>It has not yet been peer reviewed by a journal.</strong></div><div>The National Library of Medicine is <a href="/pmc/about/nihpreprints/">running a pilot</a> to include preprints that result from research funded by NIH in PMC and PubMed.</div></div><h1 class="content-title">Control of Biofilm Formation by an <em>Agrobacterium tumefaciens</em> Pterin-Binding Periplasmic Protein Conserved Among Pathogenic Bacteria</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Greenwich%20JL%5BAuthor%5D" class="affpopup" co-rid="_co_idm140191153343984" co-class="co-affbox">Jennifer L. Greenwich</a>,<sup>a</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Eagan%20JL%5BAuthor%5D" class="affpopup" co-rid="_co_idm140191131667648" co-class="co-affbox">Justin L. Eagan</a>,<sup>a</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Feirer%20N%5BAuthor%5D" class="affpopup" co-rid="_co_idm140191153339984" co-class="co-affbox">Nathan Feirer</a>,<sup>a</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Boswinkle%20K%5BAuthor%5D" class="affpopup" co-rid="_co_idm140191132269984" co-class="co-affbox">Kaleb Boswinkle</a>,<sup>b</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Minasov%20G%5BAuthor%5D" class="affpopup" co-rid="_co_idm140191132268112" co-class="co-affbox">George Minasov</a>,<sup>c,</sup><sup>d</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shuvalova%20L%5BAuthor%5D" class="affpopup" co-rid="_co_idm140191132264656" co-class="co-affbox">Ludmilla Shuvalova</a>,<sup>e</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Inniss%20NL%5BAuthor%5D" class="affpopup" co-rid="_co_idm140191132262672" co-class="co-affbox">Nicole L. Inniss</a>,<sup>c,</sup><sup>d</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Raghavaiah%20J%5BAuthor%5D" class="affpopup" co-rid="_co_idm140191149482992" co-class="co-affbox">Jakka Raghavaiah</a>,<sup>f</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ghosh%20AK%5BAuthor%5D" class="affpopup" co-rid="_co_idm140191149481008" co-class="co-affbox">Arun K. Ghosh</a>,<sup>f</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Satchell%20KJ%5BAuthor%5D" class="affpopup" co-rid="_co_idm140191149479024" co-class="co-affbox">Karla J.F. Satchell</a>,<sup>c,</sup><sup>d</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Allen%20KD%5BAuthor%5D" class="affpopup" co-rid="_co_idm140191149475568" co-class="co-affbox">Kylie D. Allen</a>,<sup>b</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Fuqua%20C%5BAuthor%5D" class="affpopup" co-rid="_co_idm140191153023184" co-class="co-affbox">Clay Fuqua</a><sup>a,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140191153343984"><h3 class="no_margin">Jennifer L. Greenwich</h3><p><sup>a</sup>Department of Biology, Indiana University, Bloomington, IN 47405 USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Greenwich%20JL%5BAuthor%5D">Jennifer L. Greenwich</a></div></div><div id="_co_idm140191131667648"><h3 class="no_margin">Justin L. Eagan</h3><p><sup>a</sup>Department of Biology, Indiana University, Bloomington, IN 47405 USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Eagan%20JL%5BAuthor%5D">Justin L. Eagan</a></div></div><div id="_co_idm140191153339984"><h3 class="no_margin">Nathan Feirer</h3><p><sup>a</sup>Department of Biology, Indiana University, Bloomington, IN 47405 USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Feirer%20N%5BAuthor%5D">Nathan Feirer</a></div></div><div id="_co_idm140191132269984"><h3 class="no_margin">Kaleb Boswinkle</h3><p><sup>b</sup>Department of Biochemistry, Virginia Polytechnic Institute and State University, Blacksburg, VA 24061 USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Boswinkle%20K%5BAuthor%5D">Kaleb Boswinkle</a></div></div><div id="_co_idm140191132268112"><h3 class="no_margin">George Minasov</h3><p><sup>c</sup>Department of Microbiology-Immunology, Feinberg School of Medicine, Northwestern University, Chicago, IL 60611 USA</p><p><sup>d</sup>Center for Structural Biology of Infectious Diseases, Feinberg School of Medicine, Northwestern University, Chicago, IL 60611 USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Minasov%20G%5BAuthor%5D">George Minasov</a></div></div><div id="_co_idm140191132264656"><h3 class="no_margin">Ludmilla Shuvalova</h3><p><sup>e</sup>Department of Pharmacology, Feinberg School of Medicine, Northwestern University, Chicago, IL 60611 USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shuvalova%20L%5BAuthor%5D">Ludmilla Shuvalova</a></div></div><div id="_co_idm140191132262672"><h3 class="no_margin">Nicole L. Inniss</h3><p><sup>c</sup>Department of Microbiology-Immunology, Feinberg School of Medicine, Northwestern University, Chicago, IL 60611 USA</p><p><sup>d</sup>Center for Structural Biology of Infectious Diseases, Feinberg School of Medicine, Northwestern University, Chicago, IL 60611 USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Inniss%20NL%5BAuthor%5D">Nicole L. Inniss</a></div></div><div id="_co_idm140191149482992"><h3 class="no_margin">Jakka Raghavaiah</h3><p><sup>f</sup>Departments of Chemistry and Medicinal Chemistry, Purdue University, West Lafayette, IN 47907 USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Raghavaiah%20J%5BAuthor%5D">Jakka Raghavaiah</a></div></div><div id="_co_idm140191149481008"><h3 class="no_margin">Arun K. Ghosh</h3><p><sup>f</sup>Departments of Chemistry and Medicinal Chemistry, Purdue University, West Lafayette, IN 47907 USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ghosh%20AK%5BAuthor%5D">Arun K. Ghosh</a></div></div><div id="_co_idm140191149479024"><h3 class="no_margin">Karla J.F. Satchell</h3><p><sup>c</sup>Department of Microbiology-Immunology, Feinberg School of Medicine, Northwestern University, Chicago, IL 60611 USA</p><p><sup>d</sup>Center for Structural Biology of Infectious Diseases, Feinberg School of Medicine, Northwestern University, Chicago, IL 60611 USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Satchell%20KJ%5BAuthor%5D">Karla J.F. Satchell</a></div></div><div id="_co_idm140191149475568"><h3 class="no_margin">Kylie D. Allen</h3><p><sup>b</sup>Department of Biochemistry, Virginia Polytechnic Institute and State University, Blacksburg, VA 24061 USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Allen%20KD%5BAuthor%5D">Kylie D. Allen</a></div></div><div id="_co_idm140191153023184"><h3 class="no_margin">Clay Fuqua</h3><p><sup>a</sup>Department of Biology, Indiana University, Bloomington, IN 47405 USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Fuqua%20C%5BAuthor%5D">Clay Fuqua</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140191144211760_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140191144211760_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140191144211760_ai" style="display:none"><div class="fm-affl" id="A1"><sup>a</sup>Department of Biology, Indiana University, Bloomington, IN 47405 USA</div><div class="fm-affl" id="A2"><sup>b</sup>Department of Biochemistry, Virginia Polytechnic Institute and State University, Blacksburg, VA 24061 USA</div><div class="fm-affl" id="A3"><sup>c</sup>Department of Microbiology-Immunology, Feinberg School of Medicine, Northwestern University, Chicago, IL 60611 USA</div><div class="fm-affl" id="A4"><sup>d</sup>Center for Structural Biology of Infectious Diseases, Feinberg School of Medicine, Northwestern University, Chicago, IL 60611 USA</div><div class="fm-affl" id="A5"><sup>e</sup>Department of Pharmacology, Feinberg School of Medicine, Northwestern University, Chicago, IL 60611 USA</div><div class="fm-affl" id="A6"><sup>f</sup>Departments of Chemistry and Medicinal Chemistry, Purdue University, West Lafayette, IN 47907 USA</div><div id="FN1">Current addresses: J.L.E., Medical Microbiology and Immunology, University of Wisconsin, Madison, WI 53706; N.F., Promega Corp., Madison, WI, 53711</div><div>Contributed by <p id="P2" class="p p-first-last"><strong>Author contributions:</strong> J.L.G, J.L.E., K.D.A. and C.F. designed the study; J.L.G., J.L.E., N.F., and K.B. performed the genetic and microbiological experiments, and J.L.G., K.B. and K.D.A. performed biochemical analysis of pterins and pterin-binding. L.S. and G.M. performed crystallization and structural determination. J.R. and A.K.G. developed and implemented a new pterin synthesis method. J.L.G., K.D.A., N.L.I., G.M. K.J.S.F. and C.F wrote the manuscript with input from all authors. C.F. supervised the entire study.</p></div><div id="CR1"><sup>*</sup>Clay Fuqua, <a href="mailto:dev@null" data-email="ude.anaidni@auqufc" class="oemail">ude.anaidni@auqufc</a></div></div><div class="fm-article-notes half_rhythm"></div><div class="permissions half_rhythm hide" id="idm140191144211760_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">PMC Copyright notice</a> </div><div class="license half_rhythm">This work is licensed under a <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License</a>, which allows reusers to copy and distribute the material in any medium or format in unadapted form only, for noncommercial purposes only, and only so long as attribution is given to the creator.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm" role="complementary" aria-label="Related or updated information about this article.">The complete version history of this preprint is available at <a href="//doi.org/10.1101%2F2023.11.18.567607" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">bioRxiv</a>.</div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplement 1.</strong></div><div><a href="/pmc/articles/PMC10680838/bin/media-1.pdf" data-ga-action="click_feat_suppl">media-1.pdf</a><span style="color:gray"> (1.6M)</span></div><div class="small guid">GUID: 780E182E-D6AF-4110-AFF5-E7C74166F4E6</div></div></dd></dl><dl data-length="384" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first-last">The structures determined here were deposited in the Protein Data Bank (<a href="https://www.rcsb.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.rcsb.org/</a>) with the assigned PDB codes: 7kos (<em>A. tumefaciens</em>), 7kou (<em>A. tumefaciens 2)</em>, 7kom (<em>V. vulnificus</em>), 7rkb (<em>K. pneumoniae</em>), 7kp2 (<em>V. cholerae</em>). Protein diffraction data have been deposited at <a href="https://proteindiffraction.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">proteindiffraction.org</a>. All additional experimental data is available upon request from the authors.</p></dd></dl></div><div id="ABS1" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="ABS1title">Abstract</h2><!--article-meta--><div><p id="P3" class="p p-first-last">Biofilm formation and surface attachment in multiple Alphaproteobacteria is driven by unipolar polysaccharide (UPP) adhesins. The pathogen <em>Agrobacterium tumefaciens</em> produces a UPP adhesin, which is regulated by the intracellular second messenger cyclic diguanylate monophosphate (cdGMP). Prior studies revealed that DcpA, a diguanylate cyclase-phosphodiesterase (DGC-PDE), is crucial in control of UPP production and surface attachment. DcpA is regulated by PruR, a protein with distant similarity to enzymatic domains known to coordinate the molybdopterin cofactor (MoCo). Pterins are bicyclic nitrogen-rich compounds, several of which are formed via a non-essential branch of the folate biosynthesis pathway, distinct from MoCo. The pterin-binding protein PruR controls DcpA activity, fostering cdGMP breakdown and dampening its synthesis. Pterins are excreted and we report here that PruR associates with these metabolites in the periplasm, promoting interaction with the DcpA periplasmic domain. The pteridine reductase PruA, which reduces specific dihydro-pterin molecules to their tetrahydro forms, imparts control over DcpA activity through PruR. Tetrahydromonapterin preferentially associates with PruR relative to other related pterins, and the PruR-DcpA interaction is decreased in a <em>pruA</em> mutant. PruR and DcpA are encoded in an operon that is conserved amongst multiple Proteobacteria including mammalian pathogens. Crystal structures reveal that PruR and several orthologs adopt a conserved fold, with a pterin-specific binding cleft that coordinates the bicyclic pterin ring. These findings define a new pterin-responsive regulatory mechanism that controls biofilm formation and related cdGMP-dependent phenotypes in <em>A. tumefaciens</em> and is found in multiple additional bacterial pathogens.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Biological Sciences, Microbiology, Biofilm, regulation, pterin, cyclic diguanylate monophosphate, protein structure</span></div></div><div id="S1" class="tsec sec"><h2 class="head no_bottom_margin" id="S1title">INTRODUCTION</h2><p id="P4" class="p p-first">The regulation of bacterial attachment to surfaces plays a critical role in the formation of biofilms and can dictate their maturation. Biofilms are surface-associated microbial assemblages that are common among bacteria, and result in dramatic physiological changes including substantial tolerance towards antibiotic treatment (<sup><a href="#R1" rid="R1" class=" bibr popnode">1</a></sup><sup>, </sup><sup><a href="#R2" rid="R2" class=" bibr popnode">2</a></sup>). Biofilms represent a major challenge for the treatment of bacterial infections as these multicellular structures are recalcitrant to standard therapies and act as protective reservoirs for pathogenic bacteria. Production of bacterial surface structures known as adhesins drive stable attachment of bacteria to surfaces, the first step in biofilm formation (<sup><a href="#R3" rid="R3" class=" bibr popnode">3</a></sup>). Regulation of adhesin elaboration and activity thus can influence where and how biofilm formation occurs. In many bacteria the transition from a free-living to a sessile mode of growth is under the regulatory control of the cytoplasmic second messenger cyclic diguanylate monophosphate (cdGMP). Increasing levels of cdGMP often promote attachment and biofilm formation through production of adhesive proteins and polysaccharides that drive the attachment process (<sup><a href="#R4" rid="R4" class=" bibr popnode">4</a></sup>). Synthesis of cdGMP is catalyzed by diguanylate cyclases (DGCs) and its turnover is driven by phosphodiesterases (PDEs). Single bacterial taxa can have multiple DGCs and PDEs that influence the cdGMP pool, and multi-domain proteins with dual DGC and PDE activities are not uncommon. Environmentally responsive modulation of cdGMP pools is mediated through control of gene expression and through allosteric regulation of these enzymes, many of which have sensory input modules. The flux of this second messenger in cells reflects the combined output of these synthesis and degradation activities (<sup><a href="#R5" rid="R5" class=" bibr popnode">5</a></sup>).</p><p id="P5">The facultative plant pathogen, <em>Agrobacterium tumefaciens</em>, utilizes secreted polysaccharides to stably attach to biotic and abiotic surfaces, most prominently cellulose and a polar adhesin known as the <span class="underline">u</span>ni<span class="underline">p</span>olar <span class="underline">p</span>olysaccharide (UPP) (<sup><a href="#R6" rid="R6" class=" bibr popnode">6</a></sup>). <em>A. tumefaciens</em> is the causative agent of crown gall, a plant neoplastic disease that results from bacteria-to-plant horizontal gene transfer (<sup><a href="#R7" rid="R7" class=" bibr popnode">7</a></sup>). Attaching to plant tissue is a requisite step in the virulence pathway of <em>A. tumefaciens</em>, but pathogenesis also involves additional plant-produced-signals to activate virulence (<sup><a href="#R8" rid="R8" class=" bibr popnode">8</a></sup>). Production of the UPP, as well as cellulose, is under complex environmental control through cdGMP (<sup><a href="#R9" rid="R9" class=" bibr popnode">9</a></sup><sup>, </sup><sup><a href="#R10" rid="R10" class=" bibr popnode">10</a></sup>). <em>A. tumefaciens</em> encodes close to 30 proteins with predicted DGC domains and only two solo PDE enzymes, but multiple proteins with predicted DGC domains also have PDE domains (<sup><a href="#R6" rid="R6" class=" bibr popnode">6</a></sup>).</p><p id="P6">In our prior studies, we identified the DcpA protein, with both canonical DGC and PDE domains (<sup><a href="#R9" rid="R9" class=" bibr popnode">9</a></sup>). DcpA has DGC and PDE activity <em>in vivo</em>, and genetic evidence suggests that both domains are functional. However, under standard laboratory conditions, DcpA is predominantly a PDE, maintaining low cdGMP levels and thereby limiting surface attachment. The N-terminal portion of DcpA has two transmembrane domains that flank an ~140 aa periplasmic domain of unknown function. Our prior findings identified several other regulatory components critical for maintaining DcpA as a PDE. These additional regulators include the PruA pteridine reductase, which reduces specific 7,8-dihydropterins to their 5,6,7,8-tetrahydro forms, and PruR, a putative pterin-binding protein (<sup><a href="#R9" rid="R9" class=" bibr popnode">9</a></sup><sup>, </sup><sup><a href="#R11" rid="R11" class=" bibr popnode">11</a></sup>). Both of these proteins are required to regulate the enzymatic activity of DcpA, and null mutants for <em>pruA</em> and <em>pruR</em> similarly lead to elevated cdGMP and aberrant activation of surface adhesion. The <em>pruR</em> gene is transcriptionally coupled with <em>dcpA</em> in a two gene operon, consistent with their related functions.</p><p id="P7">Pterins are characterized by a nitrogen-rich, bicyclic ring structure with side chains of varying length and modifications extending from the ring carbon at the sixth position (<sup><a href="#R12" rid="R12" class=" bibr popnode">12</a></sup>). Well known biomolecules with a pterin ring include the folates in which the side chain is composed of a para-amino benzoic acid group conjugated to a glutamic acid residue (or poly-glutamate). Folate derivatives are required as cofactors for multiple aspects of one-carbon metabolism, and folate is essential in most organisms (<sup><a href="#R13" rid="R13" class=" bibr popnode">13</a></sup>). More broadly, diverse pterin derivatives are found in all domains of life, and in bacteria are known to act as enzymatic cofactors. Molybdopterin cofactor (MoCo) is a complex molybdenum-containing derivative used by various organisms as a prosthetic group to catalyze redox reactions. MoCo is synthesized from GTP via a pathway independent from folate biosynthesis (<sup><a href="#R14" rid="R14" class=" bibr popnode">14</a></sup>). Biopterin and monapterin are pterins with hydroxylated 3 carbon side chains that function with cytoplasmic amino acid hydroxylases, but curiously the dominant fraction of pterins is detected outside of cells (<sup><a href="#R15" rid="R15" class=" bibr popnode">15</a></sup>). Pterins can exist in a fully oxidized state as well as the reduced 7,8-dihydro and 5,6,7,8-tetrahydro states, the latter of which serves as the biologically active form of the respective cofactor. In <em>A. tumefaciens,</em> PruA catalyzes the NADPH-dependent reduction of H<sub>2</sub>MPt to H<sub>4</sub>MPt (<sup><a href="#R11" rid="R11" class=" bibr popnode">11</a></sup>). Extracts of <em>A. tumefaciens</em> contain a methylated derivative of H<sub>4</sub>MPt (2&#x02032;-<em>O</em>-methylmonapterin) but this is undetectable in <em>pruA</em> deletion mutants and in mutants expressing a catalytically inactive PruA protein (<sup><a href="#R9" rid="R9" class=" bibr popnode">9</a></sup>). Mutations that interfere with PruA activity result in high level DGC activity from DcpA, driving elevated UPP-dependent adhesion.</p><p id="P8" class="p p-last">The regulation of DcpA by PruA is indirect and requires PruR, but the mechanistic basis for this was undefined (<sup><a href="#R9" rid="R9" class=" bibr popnode">9</a></sup>)(<a href="#SD1" rid="SD1" class=" supplementary-material">Fig. S1A</a>). PruR shares distant amino acid sequence similarity with the SUOX family of proteins, defined by sulfite oxidases and related enzymes, including YedY from <em>Escherichia coli</em> (<sup><a href="#R16" rid="R16" class=" bibr popnode">16</a></sup>). SUOX proteins have domains that associate with MoCo to facilitate their enzymatic activities and have a conserved cysteine residue that coordinates the molybdenum atom in MoCo. There is no cysteine at this position in PruR and the protein has no predicted enzymatic activity (<sup><a href="#R9" rid="R9" class=" bibr popnode">9</a></sup><sup>, </sup><sup><a href="#R17" rid="R17" class=" bibr popnode">17</a></sup>). Rather, we hypothesize that PruR is a pterin-binding protein functioning with non-MoCo pterin derivatives. Here we report on the pterin-response mechanism, the structure of PruR, its control of DcpA, and conservation of this system in multiple bacterial pathogens.</p></div><div id="S2" class="tsec sec"><h2 class="head no_bottom_margin" id="S2title">RESULTS</h2><div id="S3" class="sec sec-first"><h3 id="S3title">PruR regulates DcpA and binds to pterin ligands <em>in vitro</em></h3><p id="P9" class="p p-first">The impact of elevated cdGMP on UPP and cellulose production can be qualitatively observed by cultivating <em>A. tumefaciens</em> derivatives on solid medium supplemented with the azo-dye Congo Red &#x02013; increased red pigmentation of colonies (the <span class="underline">e</span>levated <span class="underline">C</span>ongo <span class="underline">R</span>ed or ECR phenotype) is indicative of increased polysaccharide production (<sup><a href="#R10" rid="R10" class=" bibr popnode">10</a></sup>). The ECR phenotype is also predictive for increased surface adhesion via the UPP. The <em>pruR</em> gene (ATU_RS16195) is encoded in an operon 9 bp immediately upstream of <em>dcpA</em> (ATU_RS16200). Our previous studies revealed that a precise in-frame deletion of the entire <em>pruR</em> coding sequence leads to a <em>dcpA</em>-dependent ECR phenotype and increased biofilm formation via elevated UPP production (<sup><a href="#R9" rid="R9" class=" bibr popnode">9</a></sup>). We constructed a new deletion mutant, preserving potential translational coupling with <em>dcpA</em>, while deleting most of the <em>pruR</em> gene. This new &#x00394;<em>pruR</em> mutant exhibited a pronounced ECR phenotype and increased biofilm formation, and in contrast to our prior &#x00394;<em>pruR</em> mutant, which impacted downstream <em>dcpA</em> expression (<sup><a href="#R9" rid="R9" class=" bibr popnode">9</a></sup>), was well complemented with ectopic expression of <em>pruR</em> alone (<a href="#SD1" rid="SD1" class=" supplementary-material">Fig. S1B</a>, <a href="#SD1" rid="SD1" class=" supplementary-material">S1C</a>).</p><p id="P10" class="p p-last">PruR was purified from <em>E. coli</em> to examine pterin binding <em>in vitro</em>. A restricted set of pterins are available commercially, and the tetrahydro forms are quite susceptible to oxidation. We synthesized an optically active H<sub>2</sub>MPt from <span class="small-caps">l</span>-xylose by following a related reported procedure (<sup><a href="#R18" rid="R18" class=" bibr popnode">18</a></sup>) and obtained dihydroneopterin (H<sub>2</sub>NPt), dihydrofolate (H<sub>2</sub>F) and tetrahydrofolate (H<sub>4</sub>F) from commercial sources. Our prior studies established that the <em>A. tumefaciens</em> pteridine reductase PruA reduces H<sub>2</sub>MPt and H<sub>2</sub>NPt (<sup><a href="#R11" rid="R11" class=" bibr popnode">11</a></sup>). An enzyme-coupled binding assay was developed to actively convert the dihydropterins to their tetrahydro derivatives (<a href="/pmc/articles/PMC10680838/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1A</span></a>), in which purified pteridine reductase PruA (His<sub>6</sub>-PruA) was incubated with the dihydropterin substrates and NADPH and in the presence of purified non-tagged PruR under anaerobic conditions. Extracted pterins bound to PruR were oxidized and measured by HPLC separation by fluorescence measurement (folate was detected by UV) (<a href="#SD1" rid="SD1" class=" supplementary-material">Fig. S2</a>). PruR associated with H<sub>2</sub>MPt at detectable levels but bound considerably more of the pterin in the PruA-coupled reaction, indicating increased affinity for the fully reduced H<sub>4</sub>MPt (<a href="/pmc/articles/PMC10680838/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1B</span></a>
<em>p</em>-value &#x0003c;0.05 compared to all other pterins). PruR also weakly associated with the neopterin derivatives but did not show a statistically significant preference for H<sub>4</sub>NPt compared to H<sub>2</sub>NPt. PruR bound even less well to H<sub>2</sub>F, with no preference for H<sub>4</sub>F; both folate compounds were bound significantly less well than H<sub>4</sub>MPt and H<sub>4</sub>NPt.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC10680838/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140191132128288"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10680838_nihpp-2023.11.18.567607v2-f0001.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihpp-2023.11.18.567607v2-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10680838/bin/nihpp-2023.11.18.567607v2-f0001.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140191132128288"><a target="object" rel="noopener" href="/pmc/articles/PMC10680838/figure/F1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC10680838/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>PruR binds pterins and is required to control both the DGC and PDE activity of DcpA.</strong><p id="P40"><strong>(A)</strong> PruA reaction and relevant pterin molecule structures. (i): PruA uses NADPH as a cofactor and catalyzes the reduction of a 7,8-dihydropterin substrate to a 5,6,7.8-tetrahydropterin. Atoms are numbered in the dihydropterin. R indicates side groups as shown in (ii)-(iv); (ii): R1- monapterin; (iii): R2- neopterin; (iv): R3 &#x02013; <em>p</em>ABA-glutamate (folate). <strong>(B)</strong>
<em>In vitro</em> pterin binding assays were performed as described in the <a href="#SD1" rid="SD1" class=" supplementary-material">supplemental methods</a> with purified &#x00394;<sub>SS</sub>-PruR (50 &#x003bc;M), NADPH, and with or without His<sub>6</sub>-PruA. HPLC fractioned reactions were examined by fluorescence for the oxidized pterins (excitation: 356 nm, emission: 450 nm). UV absorbance at 283 nm was used to measure folate relative to standards. PruR was incubated with the following pterin or folate species: H<sub>2</sub>MPt, dihydromonapterin; H<sub>4</sub>MPt, PruA-generated-tetrahydromonapterin; H<sub>2</sub>NPt, dihydroneopterin; H<sub>4</sub>NPt, PruA-generated tetrahydroneopterin; H<sub>2</sub>F, dihydrofolate; and H<sub>4</sub>F, tetrahydrofolate. Bars are averages of triplicate assays with error bars as standard deviations. Analyzed by standard one-way ANOVA and post-hoc Tukey analysis (<em>P</em> values relative to H<sub>4</sub>MPt, *, &#x0003c;0.05, *** &#x0003c;0.001, ****, &#x0003c;0.0001). <strong>(C)</strong> Biofilm assays of. <em>A. tumefaciens</em> C58 WT (gray bar), a &#x00394;<em>dcpA</em> mutant (black bars) or a &#x00394;<em>dcpA</em>&#x00394;<em>pruR</em> double mutant (white bars) containing the vector control or a plasmid-borne <em>P</em><sub><em>lac</em></sub>-<em>dcpA</em> fusion expressing either the wild type <em>dcpA</em> or catalytic site mutants (DGC<sup>&#x02212;</sup>, E308A; PDE<sup>&#x02212;</sup>, E431A; DGC<sup>&#x02212;</sup>PDE<sup>&#x02212;</sup>, both). Ratio of acetic acid-solubilized CV absorbance (A<sub>600</sub>) from 48 h biofilm assays normalized to the OD<sub>600</sub> planktonic turbidity from the same culture. Assays performed in triplicate and error bars are standard deviation; <em>P</em> values calculated comparing complementation of <em>dcpA</em> in the &#x00394;<em>dcpA</em> strain compared to the &#x00394;<em>dcpA</em>&#x00394;<em>pruR</em> strain by standard two-tailed <em>t</em>-test. <em>(P</em> values, * &#x0003c;0.05, **** &#x0003c;0.0001.)</p></div></div></div></div><div id="S4" class="sec"><h3 id="S4title">PruR actively modulates the DGC and PDE activity of DcpA</h3><p id="P11" class="p p-first">The <em>in vivo</em> phenotypes of several <em>dcpA</em> point mutants suggested that it has both DGC and PDE activity, but that the PDE activity is dominant in planktonic laboratory culture (<sup><a href="#R9" rid="R9" class=" bibr popnode">9</a></sup><sup>, </sup><sup><a href="#R10" rid="R10" class=" bibr popnode">10</a></sup>). Enzymatic assays of the purified DcpA cytoplasmic domain (res. 190&#x02013;644, see Methods) revealed that the protein has both activities (<a href="#SD1" rid="SD1" class=" supplementary-material">Fig. S3</a>), although the DGC activity is weaker (0.26 A<sub>360</sub> units min<sup>&#x02212;1</sup>mol<sup>&#x02212;1</sup>) compared to PDE activity (15.7 A<sub>360</sub> units min<sup>&#x02212;1</sup> mol<sup>&#x02212;1</sup>). However, it was unclear whether the PruR-dependent PDE-dominant activity of DcpA <em>in vivo</em> reflects PruR stimulation of the PDE activity, the inhibition of DGC activity, or both.</p><p id="P12">Ectopic expression of <em>dcpA</em> alone in <em>E. coli</em>, in the absence of <em>pruR</em>, resulted in high level cdGMP synthesis and mutation of the DGC catalytic motif (GGDEF&#x0003e;GGDAF; E308A) abolished this cdGMP increase, suggesting that PruR is required for dominant PDE activity (<sup><a href="#R9" rid="R9" class=" bibr popnode">9</a></sup>). Plasmid-borne expression of wild type <em>dcpA</em> complements the elevated biofilm formation of <em>the</em> &#x00394;<em>dcpA</em> mutant to normal levels. By contrast, the &#x00394;<em>pruR</em>&#x00394;<em>dcpA</em> mutant harboring this plasmid exhibits a dramatic increase in biofilm formation relative to wild type, and higher than the single <em>dcpA</em> mutant (<a href="/pmc/articles/PMC10680838/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1C</span></a>), consistent with the increased cdGMP caused by <em>dcpA</em> ectopic expression in a <em>pruR</em> null mutant (<sup><a href="#R9" rid="R9" class=" bibr popnode">9</a></sup>). Expression of the DGC<sup>&#x02212;</sup>(PDE<sup>+</sup>) DcpA<sub>E308A</sub> allele in the &#x00394;<em>dcpA</em> mutant decreases biofilm formation to wild type levels. However, this allele fails to cause this decrease in the &#x00394;<em>pruR</em>&#x00394;<em>dcpA</em> mutant, suggesting that PDE activity is under PruR control. Ectopic expression of <em>dcpA</em> in wild type does not significantly diminish biofilm formation (likely due to cdGMP production by other DGCs) (<sup><a href="#R9" rid="R9" class=" bibr popnode">9</a></sup>).</p><p id="P13" class="p p-last">In the absence of PruR, the PDE activity of DcpA is not increased by ectopic expression of DcpA. Mutation of <em>dcpA</em> to abolish DcpA PDE activity (EAL&#x0003e;EEL; E431A; PDE<sup>&#x02212;</sup>DGC<sup>+</sup>) resulted in striking stimulation of biofilm formation when this mutant is ectopically expressed in the &#x00394;<em>dcpA</em> mutant (<a href="/pmc/articles/PMC10680838/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1C</span></a>). However, expression of the DcpA<sub>E431A</sub> allele in the &#x00394;<em>pruR</em>&#x00394;<em>dcpA</em> mutant imparts even more dramatic biofilm stimulation, suggesting that PruR dampens DGC activity in addition to stimulating PDE activity, and its effects on these two activities are genetically separable. Ectopically expressing an allele mutated for both domains (DGC<sup>&#x02212;</sup>PDE<sup>&#x02212;</sup>, E308A E431A,) does not strongly impact biofilm formation in either the &#x00394;<em>dcpA</em> or the &#x00394;<em>pruR</em>&#x00394;<em>dcpA</em> mutant but shows a slight increase in the <em>pruR-dcpA</em> double mutant relative to the &#x00394;<em>dcpA</em> mutant.</p></div><div id="S5" class="sec"><h3 id="S5title">PruR is secreted and active in the periplasm</h3><p id="P14" class="p p-first">Sequence analysis suggests that the PruR protein has an N-terminal secretion signal (aa 1&#x02013;22, Signal-P score 0.31, <a href="/pmc/articles/PMC10680838/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2A</span></a>). Given our hypothesis that PruR is a pterin-binding protein, this prediction is surprising, as the only established function of bacterial pterins is to serve as cofactors for amino acid hydroxylases, which are cytoplasmic enzymes (<sup><a href="#R15" rid="R15" class=" bibr popnode">15</a></sup>). To experimentally evaluate whether PruR is secreted to the periplasm <em>in vivo</em>, we first tested this genetically by fusing it to <em>phoA</em>, encoding the enzyme alkaline phosphatase (AP) that requires localization to the periplasm for activity (<sup><a href="#R19" rid="R19" class=" bibr popnode">19</a></sup>). Plasmid-borne, ectopic expression of the <em>pruR-phoA</em> fusion in <em>A. tumefaciens</em> led to detectable AP activity in whole cells, above the very low levels of the same strain with no <em>phoA</em> (<a href="#SD1" rid="SD1" class=" supplementary-material">Fig. S4A</a>). Furthermore, fusion of only the <em>pruR</em> signal sequence (res. 1&#x02013;22) with <em>phoA</em> (<em>pruR</em><sub>SS</sub>-<em>phoA</em>) similarly expressed from the same plasmid resulted in much stronger AP activity. Neither construct was affected by expression in the &#x00394;<em>pruR</em> null mutant. It is likely that the full length PruR-PhoA fusion partially diminishes AP activity.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC10680838/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140191152174320"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10680838_nihpp-2023.11.18.567607v2-f0002.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihpp-2023.11.18.567607v2-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10680838/bin/nihpp-2023.11.18.567607v2-f0002.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140191152174320"><a target="object" rel="noopener" href="/pmc/articles/PMC10680838/figure/F2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC10680838/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>PruR is a periplasmic protein.</strong><p id="P41"><strong>(A)</strong> Signal P prediction of the PruR N-terminal signal sequence. Black arrow, predicted signal peptidase cleavage site; Grey arrow, putative cleavage and lipidation site; Grey text, predicted lipidated Cys19. <strong>(B)</strong> Western blot of SDS-PAGE using &#x003b1;-PruR polyclonal antisera to probe extracts of wild type <em>A. tumefaciens</em> C58 WT and the &#x00394;<em>pruR</em> mutant on its own or expressing either the <em>P</em><sub><em>lac</em></sub>-<em>pruR</em> plasmid or the <em>P</em><sub><em>lac</em></sub>-&#x00394;<sub><em>SS</em></sub><em>pruR</em> plasmid (both plasmids also expressing <em>dcpA</em>). Cultures were grown to similar densities with or without induction with 400 &#x003bc;M IPTG and fractionated to separate the cytoplasmic/membrane fraction (C lanes) from the periplasmic fraction (P lanes). <strong>(C)</strong> Biofilm assays of WT or a &#x00394;<em>pruR</em> mutant derivative with the empty vector plasmid (&#x02212;) or harboring a plasmid-borne <em>P</em><sub>lac</sub> fusion expressing either <em>pruR</em> or the &#x00394;<sub>SS</sub><em>pruR</em> (both plasmids also express <em>dcpA</em>). Ratio of acetic acid-solubilized CV A<sub>600</sub> from 48 h biofilm assays normalized to the OD<sub>600</sub> planktonic turbidity of the same culture. Assays performed in triplicate and error bars are standard deviation; <em>P</em> values calculated by standard two-tailed <em>t</em>-test (<em>P</em> values, *** &#x0003c;0.01).</p></div></div></div><p id="P15">The periplasmic localization of PruR was tested directly by fractionating <em>A. tumefaciens</em> cells from wild type and several derivatives into periplasmic and cytoplasmic fractions using an osmotic shock protocol and probing western blots with a polyclonal antibody preparation raised against PruR (&#x003b1;&#x02013;PruR). Wild type cells clearly revealed a protein the size of the processed from of PruR (146 aa, predicted 16.1 kDa) in the periplasmic fraction, whereas a &#x00394;<em>pruR</em> mutant lacked this protein (<a href="/pmc/articles/PMC10680838/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2B</span></a>). Ectopic expression of a plasmid-borne copy of <em>pruR</em> expressed from <em>P</em><sub><em>lac</em></sub> in the &#x00394;<em>pruR</em> mutant revealed an IPTG-inducible PruR protein in the periplasmic fraction. In contrast the same expression construct deleted for the <em>pruR</em> signal sequence (&#x00394;<sub>SS</sub><em>pruR</em>) results in PruR that remains in the cytoplasmic fraction.</p><p id="P16">Complementation of a &#x00394;<em>pruR</em> mutant (the mutation which is partially polar on <em>dcpA</em>) with a plasmid expressing <em>pruR</em> and <em>dcpA</em> rescues this mutant to normal levels of surface attachment (<a href="/pmc/articles/PMC10680838/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2C</span></a>). However, expression of the same plasmid expressing the &#x00394;<sub>ss</sub><em>pruR</em> allele and <em>dcpA</em> failed to rescue these phenotypes (<a href="/pmc/articles/PMC10680838/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2C</span></a>). The even greater surface adherence observed with the (<em>P</em><sub><em>lac</em></sub>-&#x00394;<sub>ss</sub><em>pruR</em>-<em>dcpA</em>) plasmid was most likely due to its additional copy of <em>dcpA</em>.</p><p id="P17" class="p p-last">Although PruR has a match for an N-terminal signal sequence there was an even stronger match identified for a lipidation site in the signal sequence predicted at the cysteine 19 (C19, Signal-P score 0.68, <a href="/pmc/articles/PMC10680838/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2A</span></a>) (<sup><a href="#R20" rid="R20" class=" bibr popnode">20</a></sup>). In this analysis, PruR would be lipidated at C19 with signal peptidase cleavage between it and the adjacent serine (S18, <a href="/pmc/articles/PMC10680838/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2A</span></a>), rather than A22 as predicted for the non-lipidated secretion signal. Lipidated PruR would be predicted to associate with the periplasmic leaflet of the inner membrane rather than the outer membrane (<sup><a href="#R21" rid="R21" class=" bibr popnode">21</a></sup>). Although this predicted lipidation site is an imperfect match to the spacing for the canonical sequence &#x0201c;lipobox&#x0201d; motif (L[A/S][A/G]C), we tested whether C19 is required for PruR activity. Ectopic expression of a <em>pruR</em> allele with this cysteine residue mutated to an alanine (C19A) fully complemented the elevated biofilm phenotype &#x00394;<em>pruR</em> mutant, suggesting that PruR is not lipidated, or minimally that it does not have to be anchored to the inner membrane to function in the periplasm (<a href="#SD1" rid="SD1" class=" supplementary-material">Fig. S4B</a>). Furthermore, <em>pruR</em> was effectively secreted and retained full activity when fused with the <em>malE</em> and <em>dsbA</em> signal sequences (MalE<sub>SS</sub> and DsbA<sub>SS</sub>) from <em>E. coli</em>, well characterized to direct two distinct Sec-dependent secretion mechanisms, and to be non-lipidated (<sup><a href="#R22" rid="R22" class=" bibr popnode">22</a></sup><sup>, </sup><sup><a href="#R23" rid="R23" class=" bibr popnode">23</a></sup>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Fig. S4C</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">S4D</a>).</p></div><div id="S6" class="sec"><h3 id="S6title"><em>In vivo</em> crosslinking reveals a PruR-DcpA complex</h3><p id="P18" class="p p-first">Direct interaction of PruR with the periplasmic region of DcpA would be one mechanism by which pterins could regulate DcpA DGC and PDE activity. To test this hypothesis, we used disuccinimidyl suberate (DSS) to perform protein crosslinking with cell suspensions of <em>A. tumefaciens</em> and then probed for PruR and DcpA proteins using polyclonal antibodies to PruR and separately to the periplasmic portion of DcpA (<a href="/pmc/articles/PMC10680838/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>). PruR is an approximately 16 kDa protein, and full-length DcpA is approximately 70 kDa. Upon DSS addition in wild type cells, SDS-PAGE separation, and western blotting with &#x003b1;-PruR antibody, a new protein species of ~85 kDa is observed (<a href="/pmc/articles/PMC10680838/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3A</span></a>). This species is absent in the &#x00394;<em>dcpA</em> mutant but is significantly more pronounced in wild type <em>A. tumefaciens</em> expressing the <em>P</em><sub><em>lac</em></sub>-<em>pruR</em>-<em>dcpA</em> construct. Elevated expression of the <em>pruA</em> gene had no obvious impact on crosslinking of PruR and DcpA. Probing these western blots with a lower titer &#x003b1;-DcpA antibody preparation against the periplasmic domain captured the complex in cells harboring the <em>P</em><sub><em>lac</em></sub>-<em>pruR</em>-<em>dcpA</em> plasmid but was not sufficiently sensitive to detect the complex at native levels of expression (<a href="#SD1" rid="SD1" class=" supplementary-material">Fig. S5</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC10680838/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140191151527856"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10680838_nihpp-2023.11.18.567607v2-f0003.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihpp-2023.11.18.567607v2-f0003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10680838/bin/nihpp-2023.11.18.567607v2-f0003.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140191151527856"><a target="object" rel="noopener" href="/pmc/articles/PMC10680838/figure/F3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC10680838/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>PruR forms a complex with the periplasmic region of DcpA.</strong><p id="P42"><strong>(A)</strong> Western blot probing for PruR with &#x003b1;&#x02013;PruR polyclonal antisera . Whole cell suspensions were either untreated or incubated with DSS crosslinker (0.75 mM) for wild type <em>A. tumefaciens</em> C58 and the &#x00394;<em>dcpA</em> mutant. Wild type <em>A. tumefaciens</em> harboring a plasmid ectopically expressing either <em>pruR-dcpA</em> or <em>pruA</em> from <em>P</em><sub><em>lac</em></sub> (400 &#x003bc;M IPTG) was also subjected to same analysis. Black arrows; PruR-DcpA complex, 87 kDa; DcpA, 71 kDa; PruR; 16 kDa. <strong>(B)</strong> Similar western blot to panel A probing for PruR following DSS crosslinking of whole cell suspensions of either <em>A. tumefaciens</em> wild type strain or the &#x00394;<em>dcpA</em> mutant. Either the cytoplasmic (DcpA<sub>cyt</sub>) or periplasmic (DcpA<sub>peri</sub>) domains of DcpA are ectopically expressed from <em>P</em><sub><em>lac</em></sub> (400 &#x003bc;M IPTG). Red triangles, PruR-DcpA, 87 kDa; Blue triangles, expected size of PruR-DcpA<sub>cyt</sub>, ~68 kDa; Black triangles, PruR-DcpA<sub>peri</sub>,38 kDa. &#x003b1;-PruR used at 1:40,000 and antibody binding detected with GAR-HRP secondary antibody and chemiluminescent substrate on a BioRad ChemiDoc. Non-specific bands serve as protein loading controls.</p></div></div></div><p id="P19" class="p p-last">Next, we tested the interaction of PruR independently with the periplasmic and cytoplasmic DcpA domains. The DcpA periplasmic domain including its two transmembrane elements (expressing codons 1&#x02013;192) and the DcpA cytoplasmic expression plasmid (codons 190&#x02013;644) were independently expressed from <em>P</em><sub><em>lac</em></sub> either in the wild type or the &#x00394;<em>dcpA</em> mutant. DSS crosslinking of whole cell suspensions revealed efficient PruR crosslinking with the DcpA periplasmic domain (predicted ~37.4 kDa crosslinked complex) but not with the cytoplasmic domain (predicted 66.5 kDa complex) (<a href="/pmc/articles/PMC10680838/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3B</span></a>). In the wild type background, the full length PruR-DcpA complex was also visible, when probed for either PruR or the periplasmic domain of DcpA, but this was abolished in the &#x00394;<em>dcpA</em> mutant. These results are consistent with the genetic evidence that the PruR-DcpA interaction occurs in the periplasm.</p></div><div id="S7" class="sec"><h3 id="S7title">PruR interaction with the DcpA periplasmic domain is decreased in a <em>pruA</em> mutant</h3><p id="P20" class="p p-first-last">The &#x00394;<em>pruA</em> mutant manifests a dramatic increase in UPP production and surface attachment, similar to mutants of <em>pruR</em> or <em>dcpA</em>, and its impact is dependent on the presence of a functional <em>dcpA</em> gene (<sup><a href="#R9" rid="R9" class=" bibr popnode">9</a></sup>). To evaluate whether PruA-generated H<sub>4</sub>MPt impacts PruR-DcpA interactions, we compared DSS crosslinking for whole cell suspensions of the &#x00394;<em>pruA</em> mutant compared to the wild type, both ectopically expressing PruR and the periplasmic domain of DcpA (these derivatives retain the chromosomal <em>pruR-dcpA</em> genes). The amount of complex formation between PruR and DcpA<sub>Peri</sub> was substantially diminished in the &#x00394;<em>pruA</em> mutant relative to the wild type using both the &#x003b1;&#x02013;PruR (<a href="/pmc/articles/PMC10680838/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4A</span></a>) and &#x003b1;&#x02013;DcpA<sub>Peri</sub> (<a href="/pmc/articles/PMC10680838/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4B</span></a>) antibodies. These results suggest that <em>pruA</em> is required for maximal complex formation between PruR and DcpA and suggests that pterins formed by PruA are essential for the PruR and DcpA interaction.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC10680838/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140191152498880"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10680838_nihpp-2023.11.18.567607v2-f0004.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihpp-2023.11.18.567607v2-f0004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10680838/bin/nihpp-2023.11.18.567607v2-f0004.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140191152498880"><a target="object" rel="noopener" href="/pmc/articles/PMC10680838/figure/F4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC10680838/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Deletion of <em>pruA</em> diminishes the interaction between DcpA and PruR.</strong><p id="P43">Western blot probing for the PruR-DcpA<sub>peri</sub> complex following <em>in vivo</em> DSS crosslinking in <em>A. tumefaciens</em> wild type or a &#x00394;<em>pruA</em> mutant expressing the <em>P</em><sub><em>lac</em></sub>-<em>pruR-dcpA</em><sub><em>cyt</em></sub> plasmid (400 &#x003bc;M IPTG). Probed with polyclonal antibody preparations <strong>(A)</strong> &#x003b1;-PruR, 1:40,000; <strong>(B)</strong> &#x003b1;-DcpA<sub>peri</sub>, 1:20,000. Antibody binding was detected with GAR-HRP secondary antibody and a chemiluminescent substrate on a BioRad ChemiDoc. Non-specific bands serve as protein loading controls.</p></div></div></div></div><div id="S8" class="sec"><h3 id="S8title">Conservation of <em>pruR-dcpA</em> type operons among proteobacterial pathogens</h3><p id="P21" class="p p-first-last">We next sought to determine whether PruR is conserved in other bacteria. There are well-conserved homologs of PruR in several taxa within the Alpha- and Gammaproteobacteria (<a href="/pmc/articles/PMC10680838/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5</span></a>). All of these proteins are of similar sizes with an N-terminal secretion signal and share residues common among the SUOX family of proteins (see residues marked by asterisks in <a href="#SD1" rid="SD1" class=" supplementary-material">Fig. S6A</a>). Notably, all lack the critical cysteine involved in MoCo binding in SUOX proteins, and instead have a tryptophan. None of these has a Cys in their N-terminal signal sequence consistent with our finding that this residue is dispensable for PruR function (<a href="#SD1" rid="SD1" class=" supplementary-material">Fig. S6A</a>). Additionally, inspection of the genomic location for the PruR homologs in these proteobacteria reveal presumptive bicistronic operons with a downstream gene encoding a DcpA homolog which is comprised of two transmembrane domains flanking a periplasmic loop and a large cytoplasmic portion of the protein. Many of these have both DGC and PDE domains such as DcpA, although in some cases they have only the DGC domain without the C-terminal PDE (<a href="/pmc/articles/PMC10680838/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5</span></a>). The predicted periplasmic domains of these <em>dcpA</em> homologs are roughly the same length (138&#x02013;150 aa) and are chemically similar. However, these regions are only weakly conserved at the primary amino acid sequence level, with two invariant residues in common, a tryptophan and a glutamate (W40 and E48 in the full-length <em>A. tumefaciens</em> DcpA) (<a href="#SD1" rid="SD1" class=" supplementary-material">Fig. S6B</a>). Strikingly, most of the bacteria that encode this <em>pruR-dcpA</em> operon are opportunistic pathogens of animals or plants. Interestingly, <em>Vibrio cholerae</em> has two <em>pruR-dcpA</em> operons, VC1933&#x02013;1934 and VCA0075&#x02013;0074 (<a href="/pmc/articles/PMC10680838/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5</span></a>). The VC1934 DcpA homolog has both DGC and PDE domains, but the GGDEF motif is degenerate (GADEF) and likely inactive. Conversely, the VCA0074 DcpA homolog only has the GGDEF domain without the PDE domain. In fact, VCA0074 is the well-studied CdgA protein established to have DGC activity in <em>V. cholerae</em> and to play an important role in biofilm formation (<sup><a href="#R24" rid="R24" class=" bibr popnode">24</a></sup>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC10680838/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140191152489904"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10680838_nihpp-2023.11.18.567607v2-f0005.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihpp-2023.11.18.567607v2-f0005.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10680838/bin/nihpp-2023.11.18.567607v2-f0005.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140191152489904"><a target="object" rel="noopener" href="/pmc/articles/PMC10680838/figure/F5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC10680838/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>The <em>pruR-dcpA</em> operon is conserved across Proteobacteria in animal and human pathogens.</strong><p id="P44"><strong>(A)</strong> Domain structure for PruR and DcpA; transmembrane domains and signal sequence are black lines; DGC domain is green; PDE domain is lime green; Pt indicates the pterin-binding activity of the protein. <strong>(B)</strong> Presumptive operon structure for homologs of the <em>pruR-dcpA</em> operon from multiple pathogens. A subset of DcpA homologs are truncated relative to DcpA, and contain only the DGC domain. <em>Ochrobactrum anthropi</em> ATCC 49188, <em>Klebsiella pneumoniae pneumoniae</em> ATCC 700721; <em>Vibrio cholerae,</em> O1 biovar El Tor str. N16961; <em>Vibrio vulnificus</em> CMCP6; <em>Pseudomonas aeruginosa</em> PA01. Color coding as in panel A &#x02013; gene and domain sizes are proportional.</p></div></div></div></div><div id="S9" class="sec"><h3 id="S9title">PruR-type proteins share a common structural fold</h3><p id="P22" class="p p-first">Based on our experimental findings for PruR-pterin interactions and on sequence conservation, we hypothesized that PruR represents a new class of proteins with an overall fold similar to the MoCo binding domains of SUOX-type proteins, that instead bind to the non-MoCo pterins. We purified representative members of the PruR family and determined their three-dimensional structures by X-ray crystallography (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S2</a>). The <em>A. tumefaciens</em> PruR structure reveals a general structural motif comprised of 10 &#x003b2;-sheets and 7 &#x003b1;-helices (<a href="/pmc/articles/PMC10680838/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Fig. 7A</span></a>, PDB 7kou). The &#x003b2;-strands form one mixed, five-stranded &#x003b2; sheet (2&#x02013;1-10&#x02013;5-6) and one curved, four-stranded antiparallel &#x003b2; sheet (3&#x02013;4-7&#x02013;9), and these sheets are interconnected by short &#x003b1; helices (<sup><a href="#R1" rid="R1" class=" bibr popnode">1</a></sup><sup>, </sup><sup><a href="#R2" rid="R2" class=" bibr popnode">2</a></sup><sup>, </sup><sup><a href="#R3" rid="R3" class=" bibr popnode">3</a></sup><sup>, </sup><sup><a href="#R5" rid="R5" class=" bibr popnode">5</a></sup><sup>, </sup><sup><a href="#R6" rid="R6" class=" bibr popnode">6</a></sup> and <sup><a href="#R7" rid="R7" class=" bibr popnode">7</a></sup>). We solved an additional crystal structure of <em>A. tumefaciens</em> PruR (PDB 7kos) for which the overall structure is identical except that in this crystal form a small surface pocket is occupied by the side chain residues of neighboring or symmetry related polypeptide chains (<a href="#SD1" rid="SD1" class=" supplementary-material">Fig. S7</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Table S1</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F7" co-legend-rid="lgnd_F7"><a href="/pmc/articles/PMC10680838/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140191154871728"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10680838_nihpp-2023.11.18.567607v2-f0007.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihpp-2023.11.18.567607v2-f0007.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10680838/bin/nihpp-2023.11.18.567607v2-f0007.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140191154871728"><a target="object" rel="noopener" href="/pmc/articles/PMC10680838/figure/F7/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F7"><div><a class="figpopup" href="/pmc/articles/PMC10680838/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7">Figure 7.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Comparison of PruR structure to conserved MoCo-binding domains.</strong><p id="P46"><strong>(A)</strong> Superposition of <em>A. tumefaciens</em> PruR (orange, 7kou) with the MoCo binding domain of chicken liver SUOX (violet, PDB 1sox) and <em>E. coli</em> YedY (salmon, PDB 1xdq). The MoCo and conserved C185 of SUOX are shown as balls-and-sticks (carbon, violet; oxygen, red; nitrogen, dark blue; sulfur, yellow), the molybdenum is shown as a green sphere, water molecules as small cyan spheres and hydrogen bonds as black, dashed lines. W70, conserved in PruR, is also shown as sticks. <strong>(B)</strong> Multiple sequence alignment of <em>A. tumefaciens</em> PruR with the MoCo binding domain of chicken liver sulfite oxidase and <em>E. coli</em> YedY. The magenta oval marks PruR W70 that aligns with the conserved <strong>C185</strong> in canonical MoCo-binding domains. The green oval marks an active site tyrosine conserved in pterin-binding proteins. <strong>(C)</strong> A zoomed in view of superposition of <em>A. tumefaciens</em> (orange, 7kou) and <em>K. pneumoniae</em> (pink, 7rkb) PruR. The neopterin and conserved residues of the pterin binding pocket are shown as sticks. Residue sidechains are numbered according to <em>A. tumefaciens</em>. <strong>(D)</strong> Superposition of all observed neopterin binding modes from crystal structures of <em>K. pneumoniae</em> (7RKB) and <em>V. cholerae</em> (7KP2). The binding pocket is represented as an electrostatic surface potential (blue-to-red, positive-to-negative charge) and neopterin as stick models. Carbons are in yellow (alternative conformation A, <em>V. cholerae</em>), grey (alternative conformation B, <em>V. cholerae</em>, 7kp2) and green (<em>K. pnuemoniae</em>, 7rkb), with oxygen in red and nitrogen in blue.</p></div></div></div><p id="P23" class="p p-last">Structures of three other PruR homologs were also solved including proteins from <em>V. vulnificus</em> (PDB 7kom, 30% identity; 1 &#x000c5;), <em>V. cholerae</em> (PDB 7kp2, 32% identity, 1.03 &#x000c5;), and <em>K. pneumoniae</em> (PDB 7rkb, 59% identity, 2.5 &#x000c5;) (<a href="/pmc/articles/PMC10680838/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6B</span></a> and <a href="/pmc/articles/PMC10680838/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and6C).</span></span><span>6C</span></a>). Multiple sequence and structure alignments of these PruR homologs clearly show that the structures are highly similar (root-mean-squared-deviation (RMSD) of 0.6 &#x02013; 2.0 &#x000c5;) and all share the SUOX-like fold. Overall, the PruR fold is conserved in different Alpha- and Gammaproteobacteria, consistent with a similar function for PruR in these bacteria.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F6" co-legend-rid="lgnd_F6"><a href="/pmc/articles/PMC10680838/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140191142663920"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10680838_nihpp-2023.11.18.567607v2-f0006.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihpp-2023.11.18.567607v2-f0006.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10680838/bin/nihpp-2023.11.18.567607v2-f0006.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140191142663920"><a target="object" rel="noopener" href="/pmc/articles/PMC10680838/figure/F6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F6"><div><a class="figpopup" href="/pmc/articles/PMC10680838/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6">Figure 6.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>The structure of PruR is a degenerate SUOX fold conserved in multiple Alpha- and Gammaproteobacteria</strong><p id="P45"><strong>(A)</strong> The overall structure of PruR from <em>A. tumefaciens</em> is depicted. The secondary structure elements are labeled and shown in red (&#x003b1; helices), yellow (&#x003b2;-strands) and green (loops). <strong>(B)</strong> Superposition of PruR from <em>A. tumefaciens</em> (orange, 7kos; violet, 7kou), <em>K. pneumoniae</em> (pink, 7rkb) <em>V. cholerae (</em>wheat, 7kp2), <em>V. vulnificus</em> (light green, 7kom). The peptide main chains of all structures are depicted as ribbons. <strong>(C)</strong> Multiple sequence alignment of PruR proteins with secondary structure elements from <em>A. tumefaciens</em> (PDB code 7kos) mapped above. Lime green circles mark pterin binding site residues conserved across all proteins.</p></div></div></div></div><div id="S10" class="sec sec-last"><h3 id="S10title">PruR has a shortened MoCo binding site that associates with a pterin</h3><p id="P24" class="p p-first">Structural comparison of <em>A. tumefaciens</em> PruR to chicken liver sulfite oxidase (PDB 1sox), a relatively close SUOX structural homolog, and <em>E. coli</em> YedY (PDB 1xdq), a more distant sequence homolog, revealed significant structural overlap to the MoCo binding domains (<a href="/pmc/articles/PMC10680838/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Fig. 7A</span></a> and <a href="/pmc/articles/PMC10680838/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and7B).</span></span><span>7B</span></a>). PruR from <em>A. tumefaciens</em> aligns over 126 aa residues with the chicken liver SUOX protein with an RMSD value of 2.5 &#x000c5;. The chicken liver SUOX active site contains a single molybdopterin bound to a surface cleft characterized by the conserved cysteine residue at the end of a lengthy &#x003b2;-hairpin (<a href="/pmc/articles/PMC10680838/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Fig. 7A</span></a>). There is no surface cleft that could accommodate MoCo in the equivalent region of PruR, and the &#x003b2;-hairpin with the conserved Cys residue in all SUOX proteins, including in <em>E. coli</em> YedY (res. 138&#x02013;146; (<sup><a href="#R25" rid="R25" class=" bibr popnode">25</a></sup>), is truncated in PruR (res. 161&#x02013;166). Furthermore, structural-sequence alignments of PruR, YedY, and chicken liver sulfite oxidase show that the canonical molybdenum-coordinating cysteine is substituted with a tryptophan (W70) in PruR, which constitutes a proline-tryptophan (<sup>69</sup>PW<sup>70</sup>) motif conserved in PruR homologs (<a href="/pmc/articles/PMC10680838/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6C</span></a>). Thus, the PruR fold is like that of SUOX family MoCo-binding domains but several structural determinants for MoCo binding, including a deep binding cleft, are lacking. The smaller pocket identified in our structures, however, can accommodate a smaller non-MoCo pterin molecule.</p><p id="P25" class="p p-last">To examine the pterin-PruR interaction, we co-crystallized PruR with H<sub>2</sub>NPt, a commercially available monapterin stereoisomer that is more stable to oxidation than the tetrahydro pterins and thus compatible with crystallization methods. We obtained the ligand-bound structures for both <em>K. pneumoniae</em> and <em>V. cholerae</em> PruR homologs (7rkb and 7kp2, respectively) (<a href="/pmc/articles/PMC10680838/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Fig. 7C</span></a>). Refinement revealed both were bound by the bicyclic moiety of the neopterin, with the hydroxylated tail absent. In all structures, the neopterin ring is positioned in the pterin binding cleft and is sandwiched between a tyrosine and tryptophan (<em>A. tumefaciens</em> Y104 and W160, respectively) (<a href="/pmc/articles/PMC10680838/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Fig. 7C</span></a>). The pterin also forms hydrogen bonds with an asparagine residue (N102). These three residues are conserved across all PruR homologs (<a href="/pmc/articles/PMC10680838/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6C</span></a>), and Y104 is conserved in other SUOX proteins including YedY (<a href="/pmc/articles/PMC10680838/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Fig. 7B</span></a>). Notably, in our apo structure of <em>A. tumefaciens</em> PruR, the Y104 side chain is flipped away from the pterin binding pocket (<a href="/pmc/articles/PMC10680838/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Fig. 7C</span></a>), suggesting that binding of a ligand to this cleft induces key stacking interactions that stabilize the pterin in the site. Further comparison of the pterin-bound PruR structures also revealed three alternative binding conformations of neopterin. The orientation of the ring moiety varies, yielding different positions of the hydroxylated tail (<a href="/pmc/articles/PMC10680838/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Fig. 7D</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Fig. S8A</a>&#x02013;<a href="#SD1" rid="SD1" class=" supplementary-material">C</a>). We suspect that the observed variation in binding modes of neopterin is a result of differences between this disfavored fully oxidized ligand and the cognate reduced H<sub>4</sub>MPt and/or 2&#x02032;-<em>O</em>-Met-H<sub>4</sub>MPt (<sup><a href="#R9" rid="R9" class=" bibr popnode">9</a></sup>).</p></div></div><div id="S11" class="tsec sec"><h2 class="head no_bottom_margin" id="S11title">DISCUSSION</h2><p id="P26" class="p p-first">Here we describe the regulation of the dual-function DGC/PDE DcpA by a novel pterin-binding protein PruR, building from our previous findings (<sup><a href="#R9" rid="R9" class=" bibr popnode">9</a></sup><sup>&#x02013;</sup><sup><a href="#R11" rid="R11" class=" bibr popnode">11</a></sup>). All three components of the pterin regulatory pathway discussed here (PruR, DcpA, and PruA) were identified in a transposon mutagenesis screen designed to identify novel regulators of surface attachment in <em>A. tumefaciens</em> (<sup><a href="#R10" rid="R10" class=" bibr popnode">10</a></sup>). In this study we have further interrogated this pterin regulatory pathway by (i) characterizing how the pterin-binding protein PruR regulates DcpA activity, (ii) defining PruR as a periplasmic protein that interacts with DcpA as well as H<sub>4</sub>MPt, (iii) elucidating the role of PruA in promoting the interaction between PruR and DcpA, and (iv) identifying similar regulatory systems in other <em>Proteobacteria</em>. Furthermore, we determined the three-dimensional structure of PruR and selected homologs in their apo forms, and two as a complex with a pterin. The level of conservation of PruR and DcpA homologs among multiple <em>Proteobacteria</em> suggests that they are likely to also utilize pterin-dependent regulation of cdGMP pools.</p><div id="S12" class="sec"><h3 id="S12title">A new class of pterin binding proteins</h3><p id="P27" class="p p-first">Our findings suggest that PruR and its homologs are a new type of pterin-binding periplasmic protein and we report the first structural information for this new group. The overall fold of PruR-type proteins suggests that they are a branch of the SUOX protein family, cytoplasmic enzymes including sulfite oxidases and nitrate reductases that utilize MoCo as a co-factor (<sup><a href="#R16" rid="R16" class=" bibr popnode">16</a></sup>). Most SUOX proteins are large and have additional domains that drive catalysis. In contrast PruR-like proteins are small, almost entirely comprised of the SUOX fold and their N-terminal secretion signal. Pterins such as H<sub>4</sub>MPt and related molecules are significantly smaller and less structurally complex than MoCo. It is intriguing that the pterin binding site for PruR-type proteins and the SUOX MoCo binding site are structurally and likely evolutionarily related, especially given the markedly different functions of the two groups of proteins.</p><p id="P28" class="p p-last">Other proteins known to bind the bicyclic pteridine ring are the aromatic amino acid hydroxylases such as phenylalanine hydroxylase (PAH), which coordinates a tetrahydrobiopterin (H<sub>4</sub>BPt) molecule as a cofactor. The H<sub>4</sub>BPt binding site stabilizes the pteridine ring via a conserved acidic residue and a histidine-conjugated iron atom, which interact with ring positions 2 and 4, respectively (<sup><a href="#R26" rid="R26" class=" bibr popnode">26</a></sup>). This binding site thus bears no significant similarity to the SUOX-type pterin binding site of the PruR-type proteins we define here. There are also well studied, membrane-anchored mammalian proteins that associate with the essential metabolite tetrahydrofolate (H<sub>4</sub>F), which also contains the bicyclic pteridine ring. These folate binding proteins (FBPs) are upregulated in fetal cells and certain cancers (<sup><a href="#R27" rid="R27" class=" bibr popnode">27</a></sup><sup>, </sup><sup><a href="#R28" rid="R28" class=" bibr popnode">28</a></sup>), and dramatically alter their conformation upon binding to H<sub>4</sub>F. Similar to PruR-like proteins, their association with the pteridine ring in H<sub>4</sub>F is fostered by conserved Tyr and Trp residues that sandwich the ring in a hydrophobic pocket, although these proteins are otherwise structurally distinct the PruR-type proteins reported here.</p></div><div id="S13" class="sec"><h3 id="S13title">Preference of PruR for fully reduced monapterin</h3><p id="P29" class="p p-first">The pterin binding specificity of PruR was revealed by <em>in vitro</em> studies with purified PruR and different pterin species. Fully reduced tetrahydropterins oxidize readily to the dihydropterin forms (<sup><a href="#R29" rid="R29" class=" bibr popnode">29</a></sup>). We used a coupled enzyme reaction with the PruA pteridine reductase and NADPH in the presence of H<sub>2</sub>MPt and H<sub>2</sub>NPt under anoxic conditions to generate the respective tetrahydropterin species in the presence of PruR (<sup><a href="#R11" rid="R11" class=" bibr popnode">11</a></sup>). We observed significantly greater association of the monapterin species in the presence of PruA than in its absence, consistent with greater binding to the PruA-generated H<sub>4</sub>MPt. In contrast, neopterin binding to PruR was lower and unaffected by the presence of PruA, even though our prior <em>in vitro</em> PruA work revealed similar catalytic efficiency with H<sub>2</sub>MPt and H<sub>2</sub>NPt substrates (<sup><a href="#R11" rid="R11" class=" bibr popnode">11</a></sup>). PruR detectably binds the neopterin species and the reduced folate species, but not to the same extent as the PruA-generated H<sub>4</sub>MPt. Although this experiment is evidence for the pterin association with PruR, it does not provide binding affinities.</p><p id="P30" class="p p-last">Co-crystallization with the H<sub>2</sub>NPt ligand and the PruR homologs from <em>V. cholerae</em> (7KOU) and <em>K. pneumoniae</em> (7RKB) revealed the fully oxidized neopterin (presumably due to oxidation of the exogenously added H<sub>2</sub>NPt during crystallization) binding at the analogous location as the MoCo binding site on SUOX. Pterin binding occurs within the pterin-binding cleft we have defined on the PruR-type proteins through interaction of the pteridine ring sandwiched between the conserved Tyr and Trp residues and several other hydrogen bonding positions. The hydroxylated tail of the neopterin was not visible in these structures suggesting greater mobility. The stereochemistry of the hydroxylated tail distinguishes monapterin from its stereoisomer neopterin, and the more complex benzoyl and glutamyl side chain substituents of folate, and we hypothesize that interactions between this tail and PruR add binding specificity. Additionally, the binding site may also interact directly with the fully reduced tetrahydropterin species through interactions that are not observed in the crystal structures with oxidized neopterin. The multiple positions of the neopterin detected within the binding cleft may also reflect weaker interactions of the protein with the non-cognate, fully oxidized neopterin molecule.</p></div><div id="S14" class="sec"><h3 id="S14title">PruR interacts with pterins in the periplasm</h3><p id="P31" class="p p-first-last">PruR negatively regulates surface attachment in <em>A. tumefaciens</em> through modulation of DcpA activity. Loss of function mutations in the pteridine reductase gene <em>pruA</em>, preventing H<sub>4</sub>MPt synthesis largely phenocopy <em>pruR</em> null mutants. The best studied functions for bacterial pterins are to act as enzymatic cofactors for amino acid hydroxylase enzymes such as phenylalanine hydroxylase (<sup><a href="#R15" rid="R15" class=" bibr popnode">15</a></sup>). All evidence suggests that PruA drives reduction of H<sub>2</sub>MPt in the cytoplasm, but our findings have revealed that PruR functions and responds to pterins in the periplasm. A longstanding, unexplained observation among reports on bacterial pterins is that they accumulate extracellularly, with as much as 100-fold greater pterin in the extracellular fraction as compared to associated with cells (<sup><a href="#R15" rid="R15" class=" bibr popnode">15</a></sup><sup>, </sup><sup><a href="#R30" rid="R30" class=" bibr popnode">30</a></sup>). The mechanism of excretion has not been elucidated, but it is also thought that extracellular pterins can re-enter cells (<sup><a href="#R31" rid="R31" class=" bibr popnode">31</a></sup>). The PruR protein clearly interacts with a self-produced, reduced monapterin species in the periplasm, but exogenous pterins that enter the periplasm might also act as agonists or antagonists of the H<sub>4</sub>MPt-PruR interaction. Our findings are most consistent with H<sub>4</sub>MPt preferentially binding to PruR. However, H<sub>4</sub>MPt is susceptible to oxidation and thus will have a relatively short half-life in the periplasm and elsewhere outside the cell (<sup><a href="#R32" rid="R32" class=" bibr popnode">32</a></sup>).</p></div><div id="S15" class="sec"><h3 id="S15title">Interaction of PruR and DcpA is stimulated by PruA</h3><p id="P32" class="p p-first">Crosslinking in whole cells reveals that PruR and DcpA interact through the DcpA periplasmic domain to form a stable complex. The size of the crosslinked species (~85 kDa) suggests that the proteins are in a one-to-one stoichiometry. The crosslinked product between ectopically expressed PruR and the DcpA periplasmic domain is significantly diminished in the &#x00394;<em>pruA</em> mutant, although it is still detectable. The &#x00394;<em>pruA</em> mutant has a dramatic phenotype in which DcpA exhibits predominantly DGC activity, equivalent to the &#x00394;<em>pruR</em> mutant (<sup><a href="#R9" rid="R9" class=" bibr popnode">9</a></sup>), so whatever remaining interaction occurs between the two proteins is insufficient to maintain a strong DcpA PDE bias. Our analysis of the PruR control of DcpA catalytic site mutants suggests that in cells with active PruA, the PruR-DcpA interaction stimulates PDE activity and also diminishes DGC activity. We speculate this may be due the reciprocal control of dimerization for the separate DGC and PDE domains of DcpA, known to be required for these activities in other proteins that synthesize and degrade cdGMP (<sup><a href="#R33" rid="R33" class=" bibr popnode">33</a></sup><sup>, </sup><sup><a href="#R34" rid="R34" class=" bibr popnode">34</a></sup>).</p><p id="P33">The DcpA periplasmic domain is less well conserved among the DcpA orthologs, although there are two invariant residues, Trp40 and Glu48 (in <em>A. tumefaciens</em> DcpA). The domains are all roughly 140 aa in length with dispersed chemical conservation. Alpha Fold predictions suggests that the DcpA periplasmic domain forms a novel four helix CACHE-type bundle, with overall structural similarity to the four helix bundles in the periplasmic domains of methyl-accepting dependent chemotaxis proteins (MCPs), that impart chemotactic motility responses to certain solutes (<sup><a href="#R35" rid="R35" class=" bibr popnode">35</a></sup>). Rather than interacting directly with their ligands in the periplasm, several MCPs mediate the response through interactions with periplasmic solute binding proteins (SBPs) (<sup><a href="#R36" rid="R36" class=" bibr popnode">36</a></sup>). The interaction of PruR and the DcpA periplasmic domain may share similarities with MCP-SBP interactions, with binding of the pterin fostering this interaction.</p><p id="P34" class="p p-last">The PruR-DcpA system is strikingly analogous to the dual-function DGC-PDE MbaA in <em>V. cholerae</em>, which is regulated through the interaction of its periplasmic domain with NspS (<sup><a href="#R37" rid="R37" class=" bibr popnode">37</a></sup><sup>, </sup><sup><a href="#R38" rid="R38" class=" bibr popnode">38</a></sup>). NspS is a SBP for polyamines and is encoded upstream of <em>mbaA</em> in the same operon. The specific type of polyamine bound by NspS can regulate its interaction with MbaA, thereby controlling the balance of DGC and PDE activities from this enzyme. <em>V. cholerae</em> synthesizes and releases norspermidine, which stimulates DGC activity through NspS-MbaA interactions, whereas other polyamines such as spermidine inhibit association with MbaA. (<sup><a href="#R39" rid="R39" class=" bibr popnode">39</a></sup><sup>, </sup><sup><a href="#R40" rid="R40" class=" bibr popnode">40</a></sup>). The polyamine response impacts cdGMP levels, controlling <em>V. cholerae</em> biofilm formation. Interestingly, in the absence of polyamines, NspS can bind to MbaA and weakly impacts its activity, similar to our observation here that PruR weakly binds to DcpA in the absence of its proposed H<sub>4</sub>MPt ligand. There are multiple types of pterins produced and excreted by bacterial and eukaryotic organisms (<sup><a href="#R12" rid="R12" class=" bibr popnode">12</a></sup>), and it is conceivable that similar to the NspS-MbaA system, the specific pterin-associated with PruR can impact its regulation of DcpA. These structurally variant pterins may act as agonists or antagonists for the PruR-H<sub>4</sub>MPt interaction. It is also plausible that the PruR-associated pterin acts as a proxy to detect a different cue in the periplasm, such as specific redox-active compounds.</p></div><div id="S16" class="sec sec-last"><h3 id="S16title">Insights into conserved PruR-DcpA type systems in other bacteria</h3><p id="P35" class="p p-first-last">The PruR homologs we identify here are well conserved proteins and highly likely to be orthologous in function. The structural determination of the four different PruR-type apo-proteins all had high similarity with an RMSD value of &#x0003c;2.0 &#x000c5;. This similarity includes the residues we have identified that associate with the pteridine ring. Most of these <em>pruR</em> homologs are immediately upstream of <em>dcpA</em> homologs with both the DGC and PDE domains similar to DcpA, or only the DGC domain. The periplasmic domains of these DcpA-type proteins are of similar size and amino acid chemistry but with only the two fully conserved residues mentioned above. For some of the conserved <em>pruR-dcpA</em> operons, only one of the cdGMP functions is retained. For example, there are two of these conserved operons in <em>V. cholerae</em>, VC1933-VC1934 and VCA0075-VCA0074 (VCA0074 is the well-studied DGC known as CdgA, (<sup><a href="#R24" rid="R24" class=" bibr popnode">24</a></sup>)). Interestingly VC1934 is unlikely to be active for cdGMP synthesis (due to its nonfunctional GADEF motif) but active for its degradation, and CdgA is well established to be active for cdGMP synthesis (<sup><a href="#R24" rid="R24" class=" bibr popnode">24</a></sup><sup>, </sup><sup><a href="#R41" rid="R41" class=" bibr popnode">41</a></sup>). In combination they may provide similar, but independently regulated DGC/PDE activities, as does the dual functionality of DcpA in <em>A. tumefaciens.</em> In the two major model strains of <em>P. aeruginosa</em>, PA01 and PA14, mutants for their <em>dcpA</em> homologs (PA2780/PA1426970), both impact cdGMP-dependent phenotypes, although to different extents (<sup><a href="#R42" rid="R42" class=" bibr popnode">42</a></sup><sup>, </sup><sup><a href="#R43" rid="R43" class=" bibr popnode">43</a></sup>). In all of these cases, the positioning of a <em>pruR</em>-type gene immediately upstream is consistent with a regulatory relationship with the downstream gene. Given the remarkable similarity of the PruR gene products, it is highly likely that they bind the same or related pterin ligands and provide pterin-dependent control.</p></div></div><div id="S17" class="tsec sec"><h2 class="head no_bottom_margin" id="S17title">MATERIALS AND METHODS</h2><p id="P36" class="p p-first-last">Detailed descriptions of <a href="#S17" rid="S17" class=" sec">materials and methods</a> can be found in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Information</a>, including: bacterial strains and plasmids used (<a href="#SD1" rid="SD1" class=" supplementary-material">Tables S3</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">S4</a>), growth conditions, molecular cloning and site-directed mutagenesis, Congo Red staining, biofilm formation quantification, PhoA secretion assays, antibody production and purification, western blots, periplasmic fractionation, <em>in vivo</em> crosslinking, HPLC analysis of pterins and folates, <em>in vitro</em> analysis of pterin binding to PruR, protein crystallography (including molecular cloning, protein production and purification, data collection, structure solution and refinement), purification of PruA, and <em>in vitro</em> enzymatic assays for DcpA.</p></div><div id="unreferenced-ft" class="tsec sec"><div class="unreferenced">​
                    <div class="boxed-text-box whole_rhythm hide-overflow" id="BX1"><div class="caption"><h3 id="caption-a.n.h.atitle">SIGNIFICANCE</h3></div><p id="P47">Biofilms are bacterial communities attached to surfaces, physiologically distinct from free-living cells, and a common cause of persistent infections. Here we define the mechanism of a novel biofilm regulatory system based on excreted metabolites called pterins, that is conserved within a wide range of Gram-negative bacteria, including multiple pathogens of animals and plants. The molecular mechanism of pterin-dependent regulation is reported including structural determination of several members of a new family of pterin-binding proteins. Pterins are produced across all domains of life and mechanistic insights into this regulatory circuit could lead to new advances in antibiofilm treatments.</p></div></div></div><div id="SM1" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="SM1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplement 1</h4><div class="sup-box half_rhythm" id="d64e1401"><a href="/pmc/articles/PMC10680838/bin/media-1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.6M, pdf)</sup></div></div></div><div id="S18" class="tsec sec"><h2 class="head no_bottom_margin" id="S18title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="P37">We thank Fitnat Yildiz, Seth Rubin, Vanessa Mariscal, Thomas Bernhardt, and Jon Beckwith for helpful discussions and Ramya Natarajan for technical assistance with preliminary protein purification. This project was supported by grants to C.F. (NIH R01 GM120337), K.D.A (NSF-CHE 2105598), A.K.G. (AI150466.). N.F. received support from the IU Genetics, Cellular and Molecular Sciences NIH Training Grant (T32 GM007757). Work by the Center for Structural Biology of Infectious Diseases was supported by HHS/NIH/NIAID contracts #HHSN272201700060C and 75N93022C00035. This research used resources of the Advanced Photon Source, a U.S. Department of Energy (DOE) Office of Science User Facility operated for the DOE Office of Science by Argonne National Laboratory under Contract No. DE-AC02-06CH11357. Use of the LS-CAT Sector 21 was supported by the Michigan Economic Development Corporation and the Michigan Technology Tri-Corridor (Grant 085P1000817). Access to LS-CAT and computational resources is facilitated by the NU Structural Biology Facility, which is supported by NCI grant to P30 CA060553 the Robert H. Lurie Comprehensive Cancer Center.</p></div></div><div id="fn-group-a.m.b" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.m.btitle">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN3"><p id="P39" class="p p-first-last"><strong>Competing Interest Statement:</strong> K.J.F.S. has a significant interest in Situ Biosciences, a contract research organization that conducts research unrelated to this study. All other authors declare no conflicts of interest. The funders had no role in study design, data collection, and analysis, decision to publish, or preparation of the manuscript.</p></p></div></div><div id="S19" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="S19title">DATA AVAILABILITY</h2><!--/article/back/sec/--><p id="P38" class="p p-first-last">The structures determined here were deposited in the Protein Data Bank (<a href="https://www.rcsb.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.rcsb.org/</a>) with the assigned PDB codes: 7kos (<em>A. tumefaciens</em>), 7kou (<em>A. tumefaciens 2)</em>, 7kom (<em>V. vulnificus</em>), 7rkb (<em>K. pneumoniae</em>), 7kp2 (<em>V. cholerae</em>). Protein diffraction data have been deposited at <a href="https://proteindiffraction.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">proteindiffraction.org</a>. All additional experimental data is available upon request from the authors.</p></div><div id="ref-list-a.m.d" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.m.dtitle">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="mixed-citation">Ciofu O., Moser C., Jensen P. O., Hoiby N., <span class="ref-title">Tolerance and resistance of microbial biofilms</span>. <span class="ref-journal">Nat Rev Microbiol</span>
<span class="ref-vol">20</span>, 621&#x02013;635 (2022).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/35115704" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Microbiol&amp;title=Tolerance+and+resistance+of+microbial+biofilms&amp;author=O.+Ciofu&amp;author=C.+Moser&amp;author=P.+O.+Jensen&amp;author=N.+Hoiby&amp;volume=20&amp;publication_year=2022&amp;pages=621-635&amp;pmid=35115704&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="mixed-citation">Sauer K.
et al., <span class="ref-title">The biofilm life cycle: expanding the conceptual model of biofilm formation</span>. <span class="ref-journal">Nat Rev Microbiol</span>
<span class="ref-vol">20</span>, 608&#x02013;620 (2022).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9841534/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35922483" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Microbiol&amp;title=The+biofilm+life+cycle:+expanding+the+conceptual+model+of+biofilm+formation&amp;author=K.+Sauer&amp;volume=20&amp;publication_year=2022&amp;pages=608-620&amp;pmid=35922483&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="mixed-citation">Berne C., Ellison C. K., Ducret A., Brun Y. V., <span class="ref-title">Bacterial adhesion at the single-cell level</span>. <span class="ref-journal">Nat Rev Microbiol</span>
<span class="ref-vol">16</span>, 616&#x02013;627 (2018).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/30008468" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Microbiol&amp;title=Bacterial+adhesion+at+the+single-cell+level&amp;author=C.+Berne&amp;author=C.+K.+Ellison&amp;author=A.+Ducret&amp;author=Y.+V.+Brun&amp;volume=16&amp;publication_year=2018&amp;pages=616-627&amp;pmid=30008468&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="mixed-citation">Jenal U., Reinders A., Lori C., <span class="ref-title">Cyclic di-GMP: second messenger extraordinaire</span>. <span class="ref-journal">Nat Rev Microbiol</span>
<span class="ref-vol">15</span>, 271&#x02013;284 (2017).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/28163311" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Microbiol&amp;title=Cyclic+di-GMP:+second+messenger+extraordinaire&amp;author=U.+Jenal&amp;author=A.+Reinders&amp;author=C.+Lori&amp;volume=15&amp;publication_year=2017&amp;pages=271-284&amp;pmid=28163311&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="mixed-citation">Randall T. E.
et al., <span class="ref-title">Sensory perception in bacterial cyclic diguanylate signal transduction</span>. <span class="ref-journal">J Bacteriol</span>
<span class="ref-vol">204</span>, e0043321 (2022).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8846402/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34606374" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=Sensory+perception+in+bacterial+cyclic+diguanylate+signal+transduction&amp;author=T.+E.+Randall&amp;volume=204&amp;publication_year=2022&amp;pages=e0043321&amp;pmid=34606374&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="mixed-citation">Thompson M. A., Onyeziri M. C., Fuqua C., <span class="ref-title">Function and regulation of <em>Agrobacterium tumefaciens</em> cell surface structures that promote attachment</span>. <span class="ref-journal">Curr Top Microbiol Immunol</span>
<span class="ref-vol">418</span>, 143&#x02013;184 (2018).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6330146/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29998422" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Top+Microbiol+Immunol&amp;title=Function+and+regulation+of+Agrobacterium+tumefaciens+cell+surface+structures+that+promote+attachment&amp;author=M.+A.+Thompson&amp;author=M.+C.+Onyeziri&amp;author=C.+Fuqua&amp;volume=418&amp;publication_year=2018&amp;pages=143-184&amp;pmid=29998422&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="mixed-citation">Gelvin S. B., <span class="ref-title">Agrobacterium-mediated plant transformation: the biology behind the &#x0201c;gene-jockeying&#x0201d; tool</span>. <span class="ref-journal">Microbiol. Mol. Biol. Rev</span>. <span class="ref-vol">67</span>, 16&#x02013;37 (2003).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC150518/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12626681" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol.+Mol.+Biol.+Rev&amp;title=Agrobacterium-mediated+plant+transformation:+the+biology+behind+the+&#x0201c;gene-jockeying&#x0201d;+tool&amp;author=S.+B.+Gelvin&amp;volume=67&amp;publication_year=2003&amp;pages=16-37&amp;pmid=12626681&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="mixed-citation">Heindl J. E.
et al., <span class="ref-title">Mechanisms and regulation of surface interactions and biofilm formation in <em>Agrobacterium</em></span>. <span class="ref-journal">Frontiers in plant science</span>
<span class="ref-vol">5</span>, 176 (2014).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4018554/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24834068" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers+in+plant+science&amp;title=Mechanisms+and+regulation+of+surface+interactions+and+biofilm+formation+in+Agrobacterium&amp;author=J.+E.+Heindl&amp;volume=5&amp;publication_year=2014&amp;pages=176&amp;pmid=24834068&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="mixed-citation">Feirer N.
et al., <span class="ref-title">A pterin-dependent signaling pathway regulates a dual-function diguanylate cyclase-phosphodiesterase controlling surface attachment in <em>Agrobacterium tumefaciens</em></span>. <span class="ref-journal">mBio</span>
<span class="ref-vol">6</span>, e00156 (2015).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4488946/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26126849" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=A+pterin-dependent+signaling+pathway+regulates+a+dual-function+diguanylate+cyclase-phosphodiesterase+controlling+surface+attachment+in+Agrobacterium+tumefaciens&amp;author=N.+Feirer&amp;volume=6&amp;publication_year=2015&amp;pages=e00156&amp;pmid=26126849&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="mixed-citation">Xu J.
et al., <span class="ref-title">Genetic analysis of <em>Agrobacterium tumefaciens</em> unipolar polysaccharide production reveals complex integrated control of the motile-to-sessile switch</span>. <span class="ref-journal">Mol Microbiol</span>
<span class="ref-vol">89</span>, 929&#x02013;948 (2013).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3799846/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23829710" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Microbiol&amp;title=Genetic+analysis+of+Agrobacterium+tumefaciens+unipolar+polysaccharide+production+reveals+complex+integrated+control+of+the+motile-to-sessile+switch&amp;author=J.+Xu&amp;volume=89&amp;publication_year=2013&amp;pages=929-948&amp;pmid=23829710&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="mixed-citation">Labine M.
et al., <span class="ref-title">Enzymatic and mutational analysis of the PruA pteridine reductase required for pterin-dependent control of biofilm formation in <em>Agrobacterium tumefaciens</em></span>. <span class="ref-journal">J Bacteriol</span>
 10.1128/JB.00098-20 (2020). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8404713/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32482721" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FJB.00098-20" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=Enzymatic+and+mutational+analysis+of+the+PruA+pteridine+reductase+required+for+pterin-dependent+control+of+biofilm+formation+in+Agrobacterium+tumefaciens&amp;author=M.+Labine&amp;publication_year=2020&amp;doi=10.1128/JB.00098-20&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="mixed-citation">Feirer N., Fuqua C., <span class="ref-title">Pterin function in bacteria</span>. <span class="ref-journal">Pteridines</span>
<span class="ref-vol">28</span>, 23&#x02013;36 (2017). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pteridines&amp;title=Pterin+function+in+bacteria&amp;author=N.+Feirer&amp;author=C.+Fuqua&amp;volume=28&amp;publication_year=2017&amp;pages=23-36&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="mixed-citation">Green J. M., Nichols B. P., Matthews R. G., &#x0201c;Folate biosynthesis, reduction and polyglutamylation&#x0201d; in <span class="ref-journal"><em>Escherichia coli</em> and <em>Salmonella</em>: Cellular and Molecular Biology</span>, Neidhardt F. C., Ed. (ASM Press, Washington, D.C., 1996), vol. <span class="ref-vol">1</span>, chap. 41, pp. 665&#x02013;673. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Escherichia+coli+and+Salmonella:+Cellular+and+Molecular+Biology&amp;author=J.+M.+Green&amp;author=B.+P.+Nichols&amp;author=R.+G.+Matthews&amp;author=F.+C.+Neidhardt&amp;publication_year=1996&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="mixed-citation">Leimkuhler S., Wuebbens M. M., Rajagopalan K. V., <span class="ref-title">The history of the discovery of the molybdenum cofactor and novel aspects of its biosynthesis in bacteria</span>. <span class="ref-journal">Coord Chem Rev</span>
<span class="ref-vol">255</span>, 1129&#x02013;1144 (2011).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3081585/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21528011" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Coord+Chem+Rev&amp;title=The+history+of+the+discovery+of+the+molybdenum+cofactor+and+novel+aspects+of+its+biosynthesis+in+bacteria&amp;author=S.+Leimkuhler&amp;author=M.+M.+Wuebbens&amp;author=K.+V.+Rajagopalan&amp;volume=255&amp;publication_year=2011&amp;pages=1129-1144&amp;pmid=21528011&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="mixed-citation">Pribat A.
et al., <span class="ref-title">FolX and FolM are essential for tetrahydromonapterin synthesis in <em>Escherichia coli</em> and <em>Pseudomonas aeruginosa</em></span>. <span class="ref-journal">J Bacteriol</span>
<span class="ref-vol">192</span>, 475&#x02013;482 (2010).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2805310/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19897652" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=FolX+and+FolM+are+essential+for+tetrahydromonapterin+synthesis+in+Escherichia+coli+and+Pseudomonas+aeruginosa&amp;author=A.+Pribat&amp;volume=192&amp;publication_year=2010&amp;pages=475-482&amp;pmid=19897652&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="mixed-citation">Workun G. J., Moquin K., Rothery R. A., Weiner J. H., <span class="ref-title">Evolutionary persistence of the molybdopyranopterin-containing sulfite oxidase protein fold</span>. <span class="ref-journal">Microbiol. Mol. Biol. Rev</span>. <span class="ref-vol">72</span>, 228&#x02013;248 (2008).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2415749/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18535145" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol.+Mol.+Biol.+Rev&amp;title=Evolutionary+persistence+of+the+molybdopyranopterin-containing+sulfite+oxidase+protein+fold&amp;author=G.+J.+Workun&amp;author=K.+Moquin&amp;author=R.+A.+Rothery&amp;author=J.+H.+Weiner&amp;volume=72&amp;publication_year=2008&amp;pages=228-248&amp;pmid=18535145&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="mixed-citation">Kisker C.
et al., <span class="ref-title">A structural comparison of molybdenum cofactor-containing enzymes</span>. <span class="ref-journal">FEMS Microbiol Rev</span>
<span class="ref-vol">22</span>, 503&#x02013;521 (1998).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/9990727" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEMS+Microbiol+Rev&amp;title=A+structural+comparison+of+molybdenum+cofactor-containing+enzymes&amp;author=C.+Kisker&amp;volume=22&amp;publication_year=1998&amp;pages=503-521&amp;pmid=9990727&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="mixed-citation">Sokya V. R., Pfleiderer W., Prewo R., <span class="ref-title">Pteridines: Synthesis and characteristics of 5,6-dihydro-6-(1,2,3-trihydroxypropyl)pteridines: Covalent intramolecular adducts</span>. <span class="ref-journal">Helv. Chim. Acta</span>. <span class="ref-vol">73</span>, 808&#x02013;826 (1990). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Helv.+Chim.+Acta&amp;title=Pteridines:+Synthesis+and+characteristics+of+5,6-dihydro-6-(1,2,3-trihydroxypropyl)pteridines:+Covalent+intramolecular+adducts&amp;author=V.+R.+Sokya&amp;author=W.+Pfleiderer&amp;author=R.+Prewo&amp;volume=73&amp;publication_year=1990&amp;pages=808-826&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="mixed-citation">Manoil C., Mekalanos J. J., Beckwith J., <span class="ref-title">Alkaline phosphatase fusions: sensors of subcellular location</span>. <span class="ref-journal">J Bacteriol</span>
<span class="ref-vol">172</span>, 515&#x02013;518 (1990).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC208471/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2404939" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=Alkaline+phosphatase+fusions:+sensors+of+subcellular+location&amp;author=C.+Manoil&amp;author=J.+J.+Mekalanos&amp;author=J.+Beckwith&amp;volume=172&amp;publication_year=1990&amp;pages=515-518&amp;pmid=2404939&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="mixed-citation">Nielsen H., Tsirigos K. D., Brunak S., von Heijne G., <span class="ref-title">A brief history of protein sorting prediction</span>. <span class="ref-journal">Protein J</span>
<span class="ref-vol">38</span>, 200&#x02013;216 (2019).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6589146/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31119599" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+J&amp;title=A+brief+history+of+protein+sorting+prediction&amp;author=H.+Nielsen&amp;author=K.+D.+Tsirigos&amp;author=S.+Brunak&amp;author=G.+von+Heijne&amp;volume=38&amp;publication_year=2019&amp;pages=200-216&amp;pmid=31119599&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="mixed-citation">Okuda S., Tokuda H., <span class="ref-title">Lipoprotein sorting in bacteria</span>. <span class="ref-journal">Annu Rev Microbiol</span>
<span class="ref-vol">65</span>, 239&#x02013;259 (2011).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/21663440" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Microbiol&amp;title=Lipoprotein+sorting+in+bacteria&amp;author=S.+Okuda&amp;author=H.+Tokuda&amp;volume=65&amp;publication_year=2011&amp;pages=239-259&amp;pmid=21663440&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="mixed-citation">Huber D.
et al., <span class="ref-title">Use of thioredoxin as a reporter to identify a subset of Escherichia coli signal sequences that promote signal recognition particle-dependent translocation</span>. <span class="ref-journal">J Bacteriol</span>
<span class="ref-vol">187</span>, 2983&#x02013;2991 (2005).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1082830/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15838024" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=Use+of+thioredoxin+as+a+reporter+to+identify+a+subset+of+Escherichia+coli+signal+sequences+that+promote+signal+recognition+particle-dependent+translocation&amp;author=D.+Huber&amp;volume=187&amp;publication_year=2005&amp;pages=2983-2991&amp;pmid=15838024&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="mixed-citation">Schierle C. F.
et al., <span class="ref-title">The DsbA signal sequence directs efficient, cotranslational export of passenger proteins to the <em>Escherichia coli</em> periplasm via the signal recognition particle pathway</span>. <span class="ref-journal">J Bacteriol</span>
<span class="ref-vol">185</span>, 5706&#x02013;5713 (2003).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC193964/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/13129941" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=The+DsbA+signal+sequence+directs+efficient,+cotranslational+export+of+passenger+proteins+to+the+Escherichia+coli+periplasm+via+the+signal+recognition+particle+pathway&amp;author=C.+F.+Schierle&amp;volume=185&amp;publication_year=2003&amp;pages=5706-5713&amp;pmid=13129941&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="mixed-citation">Beyhan S., Odell L. S., Yildiz F. H., <span class="ref-title">Identification and characterization of cyclic diguanylate signaling systems controlling rugosity in <em>Vibrio cholerae</em></span>. <span class="ref-journal">J Bacteriol</span>
<span class="ref-vol">190</span>, 7392&#x02013;7405 (2008).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2576663/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18790873" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=Identification+and+characterization+of+cyclic+diguanylate+signaling+systems+controlling+rugosity+in+Vibrio+cholerae&amp;author=S.+Beyhan&amp;author=L.+S.+Odell&amp;author=F.+H.+Yildiz&amp;volume=190&amp;publication_year=2008&amp;pages=7392-7405&amp;pmid=18790873&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="mixed-citation">Loschi L.
et al., <span class="ref-title">Structural and biochemical identification of a novel bacterial oxidoreductase</span>. <span class="ref-journal">J Biol Chem</span>
<span class="ref-vol">279</span>, 50391&#x02013;50400 (2004).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/15355966" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Structural+and+biochemical+identification+of+a+novel+bacterial+oxidoreductase&amp;author=L.+Loschi&amp;volume=279&amp;publication_year=2004&amp;pages=50391-50400&amp;pmid=15355966&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="mixed-citation">Ronau J. A.
et al., <span class="ref-title">A conserved acidic residue in phenylalanine hydroxylase contributes to cofactor affinity and catalysis</span>. <span class="ref-journal">Biochemistry</span>
<span class="ref-vol">53</span>, 6834&#x02013;6848 (2014).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4222540/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25295853" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=A+conserved+acidic+residue+in+phenylalanine+hydroxylase+contributes+to+cofactor+affinity+and+catalysis&amp;author=J.+A.+Ronau&amp;volume=53&amp;publication_year=2014&amp;pages=6834-6848&amp;pmid=25295853&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="mixed-citation">Chen C.
et al., <span class="ref-title">Structural basis for molecular recognition of folic acid by folate receptors</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">500</span>, 486&#x02013;489 (2013).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5797940/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23851396" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structural+basis+for+molecular+recognition+of+folic+acid+by+folate+receptors&amp;author=C.+Chen&amp;volume=500&amp;publication_year=2013&amp;pages=486-489&amp;pmid=23851396&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="mixed-citation">Wibowo A. S.
et al., <span class="ref-title">Structures of human folate receptors reveal biological trafficking states and diversity in folate and antifolate recognition</span>. <span class="ref-journal">Proc Natl Acad Sci U S A</span>
<span class="ref-vol">110</span>, 15180&#x02013;15188 (2013).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3780903/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23934049" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Structures+of+human+folate+receptors+reveal+biological+trafficking+states+and+diversity+in+folate+and+antifolate+recognition&amp;author=A.+S.+Wibowo&amp;volume=110&amp;publication_year=2013&amp;pages=15180-15188&amp;pmid=23934049&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="mixed-citation">Blakley R. L., Benkovic S. J., <span class="ref-journal">Folates and Pterins: Chemistry and Biochemistry of Pterins</span> (John Wiley &#x00026; Sons, Inc., 1985), vol. <span class="ref-vol">2</span>. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Folates+and+Pterins:+Chemistry+and+Biochemistry+of+Pterins&amp;author=R.+L.+Blakley&amp;author=S.+J.+Benkovic&amp;publication_year=1985&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="mixed-citation">Iwai K., Kobashi M., Fujisawa H., <span class="ref-title">Occurrence of <em>Crithidia</em> factors and folic acid in various bacteria</span>. <span class="ref-journal">J Bacteriol</span>
<span class="ref-vol">104</span>, 197&#x02013;201 (1970).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC248200/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/5473887" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=Occurrence+of+Crithidia+factors+and+folic+acid+in+various+bacteria&amp;author=K.+Iwai&amp;author=M.+Kobashi&amp;author=H.+Fujisawa&amp;volume=104&amp;publication_year=1970&amp;pages=197-201&amp;pmid=5473887&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="mixed-citation">Noiriel A., Naponelli V., Bozzo G. G., Gregory J. F. 3rd, Hanson A. D., <span class="ref-title">Folate salvage in plants: pterin aldehyde reduction is mediated by multiple non-specific aldehyde reductases</span>. <span class="ref-journal">Plant J</span>
<span class="ref-vol">51</span>, 378&#x02013;389 (2007).
 [<a href="https://pubmed.ncbi.nlm.nih.gov/17550420" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+J&amp;title=Folate+salvage+in+plants:+pterin+aldehyde+reduction+is+mediated+by+multiple+non-specific+aldehyde+reductases&amp;author=A.+Noiriel&amp;author=V.+Naponelli&amp;author=G.+G.+Bozzo&amp;author=J.+F.+Gregory&amp;author=A.+D.+Hanson&amp;volume=51&amp;publication_year=2007&amp;pages=378-389&amp;pmid=17550420&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="mixed-citation">Arai T.
et al., <span class="ref-title">Auto-oxidation of 5,6,7,8-tetrahydroneopterin</span>
<span class="ref-journal">Pteridines</span>
<span class="ref-vol">9</span>, 26&#x02013;28 (1988). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pteridines&amp;title=Auto-oxidation+of+5,6,7,8-tetrahydroneopterin&amp;author=T.+Arai&amp;volume=9&amp;publication_year=1988&amp;pages=26-28&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="mixed-citation">Chan C.
et al., <span class="ref-title">Structural basis of activity and allosteric control of diguanylate cyclase</span>. <span class="ref-journal">Proc Natl Acad Sci U S A</span>
<span class="ref-vol">101</span>, 17084&#x02013;17089 (2004).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC535365/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15569936" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Structural+basis+of+activity+and+allosteric+control+of+diguanylate+cyclase&amp;author=C.+Chan&amp;volume=101&amp;publication_year=2004&amp;pages=17084-17089&amp;pmid=15569936&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="mixed-citation">Rao F., Yang Y., Qi Y., Liang Z. X., <span class="ref-title">Catalytic mechanism of cyclic di-GMP-specific phosphodiesterase: a study of the EAL domain-containing RocR from <em>Pseudomonas aeruginosa</em></span>. <span class="ref-journal">J Bacteriol</span>
<span class="ref-vol">190</span>, 3622&#x02013;3631 (2008).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2394985/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18344366" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=Catalytic+mechanism+of+cyclic+di-GMP-specific+phosphodiesterase:+a+study+of+the+EAL+domain-containing+RocR+from+Pseudomonas+aeruginosa&amp;author=F.+Rao&amp;author=Y.+Yang&amp;author=Y.+Qi&amp;author=Z.+X.+Liang&amp;volume=190&amp;publication_year=2008&amp;pages=3622-3631&amp;pmid=18344366&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="mixed-citation">Ortega A., Zhulin I. B., Krell T., <span class="ref-title">Sensory repertoire of bacterial chemoreceptors</span>. <span class="ref-journal">Microbiol Mol Biol Rev</span>
<span class="ref-vol">81</span> (2017). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5706747/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29070658" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol+Mol+Biol+Rev&amp;title=Sensory+repertoire+of+bacterial+chemoreceptors&amp;author=A.+Ortega&amp;author=I.+B.+Zhulin&amp;author=T.+Krell&amp;volume=81&amp;publication_year=2017&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="mixed-citation">Cerna-Vargas J. P., Sanchez-Romera B., Matilla M. A., Ortega A., Krell T., <span class="ref-title">Sensing preferences for prokaryotic solute binding protein families</span>. <span class="ref-journal">Microb Biotechnol</span>
<span class="ref-vol">16</span>, 1823&#x02013;1833 (2023).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10443332/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37547952" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microb+Biotechnol&amp;title=Sensing+preferences+for+prokaryotic+solute+binding+protein+families&amp;author=J.+P.+Cerna-Vargas&amp;author=B.+Sanchez-Romera&amp;author=M.+A.+Matilla&amp;author=A.+Ortega&amp;author=T.+Krell&amp;volume=16&amp;publication_year=2023&amp;pages=1823-1833&amp;pmid=37547952&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="mixed-citation">Bridges A. A., Prentice J. A., Fei C., Wingreen N. S., Bassler B. L., <span class="ref-title">Quantitative input-output dynamics of a c-di-GMP signal transduction cascade in <em>Vibrio cholerae</em></span>. <span class="ref-journal">PLoS Biol</span>
<span class="ref-vol">20</span>, e3001585 (2022).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8967002/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35302986" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol&amp;title=Quantitative+input-output+dynamics+of+a+c-di-GMP+signal+transduction+cascade+in+Vibrio+cholerae&amp;author=A.+A.+Bridges&amp;author=J.+A.+Prentice&amp;author=C.+Fei&amp;author=N.+S.+Wingreen&amp;author=B.+L.+Bassler&amp;volume=20&amp;publication_year=2022&amp;pages=e3001585&amp;pmid=35302986&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="mixed-citation">Karatan E., Duncan T. R., Watnick P. I., <span class="ref-title">NspS, a predicted polyamine sensor, mediates activation of <em>Vibrio cholerae</em> biofilm formation by norspermidine</span>. <span class="ref-journal">J Bacteriol</span>
<span class="ref-vol">187</span>, 7434&#x02013;7443 (2005).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1273002/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16237027" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=NspS,+a+predicted+polyamine+sensor,+mediates+activation+of+Vibrio+cholerae+biofilm+formation+by+norspermidine&amp;author=E.+Karatan&amp;author=T.+R.+Duncan&amp;author=P.+I.+Watnick&amp;volume=187&amp;publication_year=2005&amp;pages=7434-7443&amp;pmid=16237027&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="mixed-citation">Bridges A. A., Bassler B. L., <span class="ref-title">Inverse regulation of <em>Vibrio cholerae</em> biofilm dispersal by polyamine signals</span>. <span class="ref-journal">Elife</span>
<span class="ref-vol">10</span> (2021). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8079147/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33856344" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Elife&amp;title=Inverse+regulation+of+Vibrio+cholerae+biofilm+dispersal+by+polyamine+signals&amp;author=A.+A.+Bridges&amp;author=B.+L.+Bassler&amp;volume=10&amp;publication_year=2021&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="mixed-citation">Cockerell S. R.
et al., <span class="ref-title"><em>Vibrio cholerae</em> NspS, a homologue of ABC-type periplasmic solute binding proteins, facilitates transduction of polyamine signals independent of their transport</span>. <span class="ref-journal">Microbiology</span>
<span class="ref-vol">160</span>, 832&#x02013;843 (2014).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3992718/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24530989" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology&amp;title=Vibrio+cholerae+NspS,+a+homologue+of+ABC-type+periplasmic+solute+binding+proteins,+facilitates+transduction+of+polyamine+signals+independent+of+their+transport&amp;author=S.+R.+Cockerell&amp;volume=160&amp;publication_year=2014&amp;pages=832-843&amp;pmid=24530989&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="mixed-citation">Lim B., Beyhan S., Yildiz F. H., <span class="ref-title">Regulation of Vibrio polysaccharide synthesis and virulence factor production by CdgC, a GGDEF-EAL domain protein, in Vibrio cholerae</span>. <span class="ref-journal">J Bacteriol</span>
<span class="ref-vol">189</span>, 717&#x02013;729 (2007).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1797307/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17122338" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=Regulation+of+Vibrio+polysaccharide+synthesis+and+virulence+factor+production+by+CdgC,+a+GGDEF-EAL+domain+protein,+in+Vibrio+cholerae&amp;author=B.+Lim&amp;author=S.+Beyhan&amp;author=F.+H.+Yildiz&amp;volume=189&amp;publication_year=2007&amp;pages=717-729&amp;pmid=17122338&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="mixed-citation">Ha D. G., Richman M. E., O&#x02019;Toole G. A., <span class="ref-title">Deletion mutant library for investigation of functional outputs of cyclic diguanylate metabolism in <em>Pseudomonas aeruginosa</em> PA14</span>. <span class="ref-journal">Appl Environ Microbiol</span>
<span class="ref-vol">80</span>, 3384&#x02013;3393 (2014).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4018857/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24657857" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Appl+Environ+Microbiol&amp;title=Deletion+mutant+library+for+investigation+of+functional+outputs+of+cyclic+diguanylate+metabolism+in+Pseudomonas+aeruginosa+PA14&amp;author=D.+G.+Ha&amp;author=M.+E.+Richman&amp;author=G.+A.+O&#x02019;Toole&amp;volume=80&amp;publication_year=2014&amp;pages=3384-3393&amp;pmid=24657857&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="mixed-citation">Eilers K.
et al., <span class="ref-title">Phenotypic and integrated analysis of a comprehensive <em>Pseudomonas aeruginosa</em> PAO1 library of mutants lacking cyclic-di-GMP-related genes</span>. <span class="ref-journal">Front Microbiol</span>
<span class="ref-vol">13</span>, 949597 (2022).
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9355167/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35935233" ref="reftype=pubmed&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front+Microbiol&amp;title=Phenotypic+and+integrated+analysis+of+a+comprehensive+Pseudomonas+aeruginosa+PAO1+library+of+mutants+lacking+cyclic-di-GMP-related+genes&amp;author=K.+Eilers&amp;volume=13&amp;publication_year=2022&amp;pages=949597&amp;pmid=35935233&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10680838&amp;issue-id=449837&amp;journal-id=3848&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">bioRxiv</span> are provided here courtesy of <strong>Cold Spring Harbor Laboratory Preprints</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC10680838/pdf/nihpp-2023.11.18.567607v2.pdf" class="int-view">PDF (2.5M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/10680838/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/10680838/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="10680838" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10680838%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="RCtfgmZ0mLiWxEl9xZLCmdFdeR32hNdlqPGcDXIIphJIsWhyskCxVP6Tf7EumLND">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10680838%2F&amp;text=Control%20of%20Biofilm%20Formation%20by%20an%20Agrobacterium%20tumefaciens%20Pterin-Binding%20Periplasmic%20Protein%20Conserved%20Among%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10680838%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10680838/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC10680838/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38014264/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC10680838/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38014264/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC10680838/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/10680838/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC10680838/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
