
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Aspergillus oryzae PrtR alters transcription of individual peptidase genes in response to the growth environment - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="939BE807DB0F4BC500004B7C7808159D.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="springeropen" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC10781853/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10781853/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Applied Microbiology and Biotechnology" /><meta name="citation_title" content="Aspergillus oryzae PrtR alters transcription of individual peptidase genes in response to the growth environment" /><meta name="citation_author" content="Rika Numazawa" /><meta name="citation_author_institution" content="Department of Applied Biological Science, United Graduate School of Agricultural Science, Tokyo University of Agriculture and Technology, 3-5-8 Saiwai-Cho, Fuchu, Tokyo, 1838509 Japan" /><meta name="citation_author" content="Yukako Tanaka" /><meta name="citation_author_institution" content="Department of Applied Biological Chemistry, Graduate School of Agriculture, Tokyo University of Agriculture and Technology, 3-5-8 Saiwai-Cho, Fuchu, Tokyo, 1838509 Japan" /><meta name="citation_author" content="Sawako Nishioka" /><meta name="citation_author_institution" content="Department of Applied Biological Chemistry, Graduate School of Agriculture, Tokyo University of Agriculture and Technology, 3-5-8 Saiwai-Cho, Fuchu, Tokyo, 1838509 Japan" /><meta name="citation_author" content="Ryotaro Tsuji" /><meta name="citation_author_institution" content="Department of Applied Biological Chemistry, Graduate School of Agriculture, Tokyo University of Agriculture and Technology, 3-5-8 Saiwai-Cho, Fuchu, Tokyo, 1838509 Japan" /><meta name="citation_author" content="Hiroshi Maeda" /><meta name="citation_author_institution" content="Department of Applied Biological Chemistry, Graduate School of Agriculture, Tokyo University of Agriculture and Technology, 3-5-8 Saiwai-Cho, Fuchu, Tokyo, 1838509 Japan" /><meta name="citation_author" content="Mizuki Tanaka" /><meta name="citation_author_institution" content="Department of Applied Biological Science, United Graduate School of Agricultural Science, Tokyo University of Agriculture and Technology, 3-5-8 Saiwai-Cho, Fuchu, Tokyo, 1838509 Japan" /><meta name="citation_author_institution" content="Department of Applied Biological Chemistry, Graduate School of Agriculture, Tokyo University of Agriculture and Technology, 3-5-8 Saiwai-Cho, Fuchu, Tokyo, 1838509 Japan" /><meta name="citation_author" content="Michio Takeuchi" /><meta name="citation_author_institution" content="Department of Applied Biological Science, United Graduate School of Agricultural Science, Tokyo University of Agriculture and Technology, 3-5-8 Saiwai-Cho, Fuchu, Tokyo, 1838509 Japan" /><meta name="citation_author_institution" content="Department of Applied Biological Chemistry, Graduate School of Agriculture, Tokyo University of Agriculture and Technology, 3-5-8 Saiwai-Cho, Fuchu, Tokyo, 1838509 Japan" /><meta name="citation_author" content="Youhei Yamagata" /><meta name="citation_author_institution" content="Department of Applied Biological Science, United Graduate School of Agricultural Science, Tokyo University of Agriculture and Technology, 3-5-8 Saiwai-Cho, Fuchu, Tokyo, 1838509 Japan" /><meta name="citation_author_institution" content="Department of Applied Biological Chemistry, Graduate School of Agriculture, Tokyo University of Agriculture and Technology, 3-5-8 Saiwai-Cho, Fuchu, Tokyo, 1838509 Japan" /><meta name="citation_publication_date" content="2024" /><meta name="citation_issue" content="1" /><meta name="citation_volume" content="108" /><meta name="citation_doi" content="10.1007/s00253-023-12833-5" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC10781853/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC10781853/" /><meta name="citation_pmid" content="38204127" /><meta name="DC.Title" content="Aspergillus oryzae PrtR alters transcription of individual peptidase genes in response to the growth environment" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Springer" /><meta name="DC.Contributor" content="Rika Numazawa" /><meta name="DC.Contributor" content="Yukako Tanaka" /><meta name="DC.Contributor" content="Sawako Nishioka" /><meta name="DC.Contributor" content="Ryotaro Tsuji" /><meta name="DC.Contributor" content="Hiroshi Maeda" /><meta name="DC.Contributor" content="Mizuki Tanaka" /><meta name="DC.Contributor" content="Michio Takeuchi" /><meta name="DC.Contributor" content="Youhei Yamagata" /><meta name="DC.Date" content="2024" /><meta name="DC.Identifier" content="10.1007/s00253-023-12833-5" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Aspergillus oryzae PrtR alters transcription of individual peptidase genes in response to the growth environment" /><meta property="og:type" content="article" /><meta property="og:description" content="Aspergillus oryzae PrtR is an ortholog of the transcription factor PrtT, which positively regulates the transcription of extracellular peptidase genes in Aspergillus niger and Aspergillus fumigatus. To identify the genes under the control of PrtR and ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10781853/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10781853/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=Springer[filter]" class="navlink">Springer</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC10781853
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC10781853/pdf/253_2023_Article_12833.pdf" class="int-view">PDF (2.1M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/10781853/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/10781853/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="10781853" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10781853%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="KG1cR1ZY9pg6r3Bu0ACAxsx4RqbKnXxNw70tfCwWANbCoAgK4IR2mrrQXctVGGba">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10781853%2F&amp;text=Aspergillus%20oryzae%20PrtR%20alters%20transcription%20of%20individual%20peptidase%20genes%20in%20response%20to%20the%20growth%20environment%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10781853%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10781853/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC10781853/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38204127/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC10781853/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38204127/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC10781853/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/10781853/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=Springer[filter]" class="navlink">Springer</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC10781853
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><a href="//doi.org/10.1007%2Fs00253-023-12833-5" title="Link to Publisher's site" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"><img src="/corehtml/pmc/pmcgifs/logo-springeropen.png" alt="Logo of springeropen"><!-- --></img></a></div>
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Appl Microbiol Biotechnol.</span> 2024; 108(1): 90. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2024 Jan 10. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1007%2Fs00253-023-12833-5" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1007/s00253-023-12833-5</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC10781853</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38204127">38204127</a></div></div></div><h1 class="content-title"><em>Aspergillus oryzae</em> PrtR alters transcription of individual peptidase genes in response to the growth environment</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Numazawa%20R%5BAuthor%5D" class="affpopup" co-rid="_co_idm140125900455824" co-class="co-affbox">Rika Numazawa</a>,<sup>#</sup><sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Tanaka%20Y%5BAuthor%5D" class="affpopup" co-rid="_co_idm140125907670928" co-class="co-affbox">Yukako Tanaka</a>,<sup>#</sup><sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Nishioka%20S%5BAuthor%5D" class="affpopup" co-rid="_co_idm140125900453728" co-class="co-affbox">Sawako Nishioka</a>,<sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Tsuji%20R%5BAuthor%5D" class="affpopup" co-rid="_co_idm140125899007024" co-class="co-affbox">Ryotaro Tsuji</a>,<sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Maeda%20H%5BAuthor%5D" class="affpopup" co-rid="_co_idm140125900622032" co-class="co-affbox">Hiroshi Maeda</a>,<sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Tanaka%20M%5BAuthor%5D" class="affpopup" co-rid="_co_idm140125900620048" co-class="co-affbox">Mizuki Tanaka</a>,<sup>1,</sup><sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Takeuchi%20M%5BAuthor%5D" class="affpopup" co-rid="_co_idm140125903434192" co-class="co-affbox">Michio Takeuchi</a>,<sup>1,</sup><sup>2</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Yamagata%20Y%5BAuthor%5D" class="affpopup" co-rid="_co_idm140125900377136" co-class="co-affbox">Youhei Yamagata</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>1,</sup><sup>2</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140125900455824"><h3 class="no_margin">Rika Numazawa</h3><p><sup>1</sup>Department of Applied Biological Science, United Graduate School of Agricultural Science, Tokyo University of Agriculture and Technology, 3-5-8 Saiwai-Cho, Fuchu, Tokyo, 1838509 Japan </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Numazawa%20R%5BAuthor%5D">Rika Numazawa</a></div></div><div id="_co_idm140125907670928"><h3 class="no_margin">Yukako Tanaka</h3><p><sup>2</sup>Department of Applied Biological Chemistry, Graduate School of Agriculture, Tokyo University of Agriculture and Technology, 3-5-8 Saiwai-Cho, Fuchu, Tokyo, 1838509 Japan </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Tanaka%20Y%5BAuthor%5D">Yukako Tanaka</a></div></div><div id="_co_idm140125900453728"><h3 class="no_margin">Sawako Nishioka</h3><p><sup>2</sup>Department of Applied Biological Chemistry, Graduate School of Agriculture, Tokyo University of Agriculture and Technology, 3-5-8 Saiwai-Cho, Fuchu, Tokyo, 1838509 Japan </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Nishioka%20S%5BAuthor%5D">Sawako Nishioka</a></div></div><div id="_co_idm140125899007024"><h3 class="no_margin">Ryotaro Tsuji</h3><p><sup>2</sup>Department of Applied Biological Chemistry, Graduate School of Agriculture, Tokyo University of Agriculture and Technology, 3-5-8 Saiwai-Cho, Fuchu, Tokyo, 1838509 Japan </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Tsuji%20R%5BAuthor%5D">Ryotaro Tsuji</a></div></div><div id="_co_idm140125900622032"><h3 class="no_margin">Hiroshi Maeda</h3><p><sup>2</sup>Department of Applied Biological Chemistry, Graduate School of Agriculture, Tokyo University of Agriculture and Technology, 3-5-8 Saiwai-Cho, Fuchu, Tokyo, 1838509 Japan </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Maeda%20H%5BAuthor%5D">Hiroshi Maeda</a></div></div><div id="_co_idm140125900620048"><h3 class="no_margin">Mizuki Tanaka</h3><p><sup>1</sup>Department of Applied Biological Science, United Graduate School of Agricultural Science, Tokyo University of Agriculture and Technology, 3-5-8 Saiwai-Cho, Fuchu, Tokyo, 1838509 Japan </p><p><sup>2</sup>Department of Applied Biological Chemistry, Graduate School of Agriculture, Tokyo University of Agriculture and Technology, 3-5-8 Saiwai-Cho, Fuchu, Tokyo, 1838509 Japan </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Tanaka%20M%5BAuthor%5D">Mizuki Tanaka</a></div></div><div id="_co_idm140125903434192"><h3 class="no_margin">Michio Takeuchi</h3><p><sup>1</sup>Department of Applied Biological Science, United Graduate School of Agricultural Science, Tokyo University of Agriculture and Technology, 3-5-8 Saiwai-Cho, Fuchu, Tokyo, 1838509 Japan </p><p><sup>2</sup>Department of Applied Biological Chemistry, Graduate School of Agriculture, Tokyo University of Agriculture and Technology, 3-5-8 Saiwai-Cho, Fuchu, Tokyo, 1838509 Japan </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Takeuchi%20M%5BAuthor%5D">Michio Takeuchi</a></div></div><div id="_co_idm140125900377136"><h3 class="no_margin">Youhei Yamagata</h3><p><sup>1</sup>Department of Applied Biological Science, United Graduate School of Agricultural Science, Tokyo University of Agriculture and Technology, 3-5-8 Saiwai-Cho, Fuchu, Tokyo, 1838509 Japan </p><p><sup>2</sup>Department of Applied Biological Chemistry, Graduate School of Agriculture, Tokyo University of Agriculture and Technology, 3-5-8 Saiwai-Cho, Fuchu, Tokyo, 1838509 Japan </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Yamagata%20Y%5BAuthor%5D">Youhei Yamagata</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140125905181792_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140125905181792_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140125905181792_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140125905181792_ai" style="display:none"><div class="fm-affl" id="Aff1"><sup>1</sup>Department of Applied Biological Science, United Graduate School of Agricultural Science, Tokyo University of Agriculture and Technology, 3-5-8 Saiwai-Cho, Fuchu, Tokyo, 1838509 Japan </div><div class="fm-affl" id="Aff2"><sup>2</sup>Department of Applied Biological Chemistry, Graduate School of Agriculture, Tokyo University of Agriculture and Technology, 3-5-8 Saiwai-Cho, Fuchu, Tokyo, 1838509 Japan </div><div><span class="fm-affl">Youhei Yamagata, </span><span class="fm-affl"><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="pj.ca.taut.cc@amay-y" class="oemail">pj.ca.taut.cc@amay-y</a></span>.</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup>Corresponding author.</div><div><sup>#</sup>Contributed equally.</div></div><div class="fm-article-notes hide half_rhythm" id="idm140125905181792_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2023 Mar 30; Revised 2023 Sep 19; Accepted 2023 Sep 30.</div></div><div class="permissions half_rhythm hide" id="idm140125905181792_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; The Author(s) 2024</div><div class="license half_rhythm"><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary file1 (PDF 936 KB)</div><div><a href="/pmc/articles/PMC10781853/bin/253_2023_12833_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl">253_2023_12833_MOESM1_ESM.pdf</a><span style="color:gray"> (936K)</span></div><div class="small guid">GUID: CD22E705-4E24-456D-8C3A-560F71B6AFE1</div></div></dd></dl><dl data-length="145" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p>All data generated or analyzed during this study are included in this published article [and its supplementary information files].</p><p>Not applicable.</p></dd></dl></div><div id="Abs1" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="Abs1title">Abstract</h2><!--article-meta--><div><div id="sec-a.n.b.o.a" class="sec sec-first"><h3 id="sec-a.n.b.o.atitle">Abstract</h3><p id="Par1" class="p p-first-last"><em>Aspergillus oryzae</em> PrtR is an ortholog of the transcription factor PrtT, which positively regulates the transcription of extracellular peptidase genes in <em>Aspergillus niger</em> and <em>Aspergillus fumigatus</em>. To identify the genes under the control of PrtR and elucidate its regulatory mechanism in <em>A. oryzae</em>, <em>prtR</em> gene disruption mutants were generated. The control strain clearly showed a halo on media containing skim milk as the nitrogen source, whereas the &#x00394;prtR strain formed a smaller halo. Measurement of acid peptidase activity revealed that approximately 84% of acidic endopeptidase and 86% of carboxypeptidase activities are positively regulated by PrtR. As the transcription of the <em>prtR</em> gene varied depending on culture conditions, especially with or without a protein substrate, it was considered that its transcription would be regulated in response to a nitrogen source. In addition, contrary to previous expectations, PrtR was found to act both in promoting and repressing the transcription of extracellular peptidase genes. The mode of regulation varied from gene to gene. Some genes were regulated in the same manner in both liquid and solid cultures, whereas others were regulated in different ways depending on the culture conditions. Furthermore, PrtR has been suggested to regulate the transcription of peptidase genes that are closely associated with other transcription factors.</p></div><div id="sec-a.n.b.o.b" class="sec"><h3 id="sec-a.n.b.o.btitle">Key points</h3><p id="Par2" class="p p-first">
<em>&#x02022; Almost all peptidase genes in Aspergillus oryzae are positively regulated by PrtR</em>
</p><p id="Par3">
<em>&#x02022; However, several genes are regulated negatively by PrtR</em>
</p><p id="Par4" class="p p-last">
<em>&#x02022; PrtR optimizes transcription of peptidase genes in response to culture conditions</em>
</p></div><div id="sec-a.n.b.o.c" class="sec sec-last"><h3 id="sec-a.n.b.o.ctitle">Supplementary Information</h3><p class="p p-first-last">The online version contains supplementary material available at 10.1007/s00253-023-12833-5.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text"><em>Aspergillus oryzae</em>, Transcriptional factor, PrtR, Extracellular peptidase</span></div></div><div id="Sec1" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec1title">Introduction</h2><p id="Par5" class="p p-first"><em>Aspergillus oryzae</em> is a filamentous fungus that is used as a source of enzymes for producing traditional Japanese fermented foods (Machida et al. <a href="#CR20" rid="CR20" class=" bibr popnode">2008</a>). Furthermore, <em>A. oryzae</em> is considered as a microorganism that can be safely used for food production and is listed in the US Food and Drug Administration&#x02019;s generally recognized as safe (Taylor and Richardson <a href="#CR39" rid="CR39" class=" bibr popnode">1979</a>). The fungus has a strong ability to secrete hydrolytic enzymes, mainly amylases and peptidases, and a superior protein secretion ability. Therefore, this fungus is used as a host for the production of native and heterologous enzymes in the enzyme industry (Papagianni <a href="#CR27" rid="CR27" class=" bibr popnode">2004</a>). This fungus is also expected to be used in the production of pharmaceutical raw materials, and the demand for these products is increasing (Yoon et al. <a href="#CR42" rid="CR42" class=" bibr popnode">2011</a>; Huynh et al. <a href="#CR14" rid="CR14" class=" bibr popnode">2020</a>). Despite the industrial value of <em>A. oryzae</em>, the genetics and molecular biology of the fungi could not be studied, because the sexual generation required for classical genetic analysis has not been identified. In 2001, expressed sequence tag (EST) analysis of <em>A. oryzae</em> RIB40 was conducted with the development of molecular biological experimental methods (Akao et al. <a href="#CR2" rid="CR2" class=" bibr popnode">2007</a>). Whole-genome sequence analysis was completed in 2005 (Machida et al. <a href="#CR19" rid="CR19" class=" bibr popnode">2005</a>), and 5&#x000a0;years later, Wang et al. (<a href="#CR41" rid="CR41" class=" bibr popnode">2010</a>) published the results of transcriptome analysis using large-scale RNA-seq. These results have accelerated the molecular biological study of the fungus. Genome analysis has revealed that <em>A. oryzae</em> contains approximately 130 peptidase genes. Meanwhile, <em>Aspergillus nidulans</em> (Galagan et al. <a href="#CR9" rid="CR9" class=" bibr popnode">2005</a>) and <em>Aspergillus fumigatus</em> (Nierman et al. <a href="#CR26" rid="CR26" class=" bibr popnode">2005</a>), whose genome analyses were reported simultaneously with <em>A. oryzae</em>, showed 90 and 99 peptidase genes, respectively. In other words, <em>A. oryzae</em> was found to have more peptidase genes than other closely related species. Phylogenetic analysis suggested that the peptidase genes of <em>A. oryzae</em> would have evolved to have diverse homologous enzymes through gene duplication (Machida et al. <a href="#CR19" rid="CR19" class=" bibr popnode">2005</a>). In other words, it has been assumed that <em>A. oryzae</em> would have evolved with an increased number of peptidase genes to be able to degrade a variety of substrates. We have considered that <em>A. oryzae</em> might have subdivided the roles of each peptidase more than other Aspergilli. Thus, peptidases are thought to be important enzymes in characterizing <em>A. oryzae</em>. However, it is still unclear which transcription factors are involved in the transcription of each peptidase gene, the DNA sequences to which they bind, and how transcriptional induction occurs.</p><p id="Par6">The control of peptidase production in <em>A. oryzae</em> has contributed to various enzyme industries. As mentioned above, <em>A. oryzae</em> is a safe filamentous fungus; therefore, peptidases derived from <em>A. oryzae</em> are used in various food-processing applications (Machida et al. <a href="#CR20" rid="CR20" class=" bibr popnode">2008</a>). Since large quantities of enzymes are often used in food processing, it is preferable that the culture supernatant be used directly, as an enzyme agent. However, as the fungus can produce a wide variety of peptidases, it is difficult to make the fungi produce only certain enzymes. Many enzyme manufacturers have been attempting to control peptidase production under culture conditions. Nonetheless, without knowing how peptidase genes are transcriptionally regulated, trial and error is the only method. Understanding how transcription is regulated will help to control peptidase production. In addition, by achieving efficient production of the required peptidases in the brewing field, fermentation foods can be produced more efficiently. Furthermore, if peptidase production can be suppressed in using <em>A. oryzae</em> as a host for heterologous protein expression, the degradation of expressed proteins can be prevented by the proteolytic enzymes produced by <em>A. oryzae</em>.</p><p id="Par7">Two transcription factors, FlbC and PrtT, are involved in the production of extracellular peptidases in <em>Aspergillus</em>. FlbC was identified as a transcription factor that positively regulates the transcription of the gene encoding glucoamylase B (GlaB), which is specifically produced in solid cultures (Tanaka et al. <a href="#CR36" rid="CR36" class=" bibr popnode">2016</a>). Furthermore, FlbC has been shown to positively regulate <em>pepO</em> and the neutral peptidase gene, <em>nptB</em> (<em>deuA</em>), in solid culture (Tanaka et al. <a href="#CR36" rid="CR36" class=" bibr popnode">2016</a>). PrtT was first identified in <em>Aspergillus niger</em> by Punt et al. (<a href="#CR28" rid="CR28" class=" bibr popnode">2008</a>) and is also a transcription factor belonging to the Zn(II)<sub>2</sub>Cys<sub>6</sub> binuclear cluster family. The gene, <em>prtT</em>, is located close to the starch-degrading gene cluster. The <em>prtT</em> family has been suggested to be unique to the genera <em>Aspergillus</em> and <em>Penicillium</em> (Sharon et al. <a href="#CR30" rid="CR30" class=" bibr popnode">2009</a>). Transcriptome analysis using the <em>prtT</em> deletion and overexpression strains of <em>A. niger</em> showed transcriptional changes in 32 putative protease genes, and the binding DNA sequence was deduced as 5&#x02032;-CCGHCGG-3&#x02032; (H; A/C/T) by MEME motif-based sequence analysis (Huang et al. <a href="#CR13" rid="CR13" class=" bibr popnode">2020</a>). Moreover, the transcriptional levels of several extracellular peptidase genes have been reported to be reduced in an <em>A. fumigatus</em> &#x00394;prtT strain compared to those in the control strain (Bergmann et al. <a href="#CR3" rid="CR3" class=" bibr popnode">2009</a>). In addition, RNA-seq using an <em>Penicillium oxalicum</em> &#x00394;prtT strain and MEME analysis/electrophoretic mobility shift assay (EMSA) using the upstream region of genes whose transcription levels were altered by <em>prtT</em> deletion showed that the binding sequence was 5&#x02032;-CHGH(D)CGG-3&#x02032; (H; A/C/T, D; A/C/T) (Chen et al. <a href="#CR6" rid="CR6" class=" bibr popnode">2014</a>).</p><p id="Par8" class="p p-last">The ortholog of PrtT in <em>A. oryzae</em> has been named PrtR (AO090003001577). PrtR, like PrtT, regulates the transcription of peptidase genes, and <em>prtR</em> is found to be present close to the starch-degrading gene cluster (Gomi <a href="#CR10" rid="CR10" class=" bibr popnode">2019</a>). Protease and acid carboxypeptidase activities are increased in <em>prtR</em> overexpression strain, and the transcription level of the alkaline protease (oryzin) gene <em>alpA</em> is increased by more than fivefold (Takahashi et al. <a href="#CR31" rid="CR31" class=" bibr popnode">2018</a>). The halo formed in skim milk medium is larger in <em>prtR</em>-overexpression strain, and the transcriptional levels of the dipeptide- and tripeptide-transporter genes <em>potA</em> and <em>potB</em> are increased by the overexpression of <em>prtR</em> and decreased by the loss of <em>prtR</em> (Tanaka et al. <a href="#CR37" rid="CR37" class=" bibr popnode">2021</a>). Thus, PrtR plays a major role in the regulation of gene expression in the presence of proteins as a nitrogen source for their utilization. However, the scope of the influence of PrtR and the detailed mechanism of its transcriptional regulation remain unclear. In this study, to evaluate the role of PrtR as a regulator of peptidase genes in <em>A. oryzae</em>, we constructed a PrtR deletion mutant strain without nutrient requirements for nitrogenous substances and examined the changes in the transcription levels of the extracellular peptidase genes in liquid and solid cultures. We clarified the peptidase genes under the regulation of PrtR and further investigated the changes in the regulation mode of PrtR in liquid and solid cultures to elucidate the regulatory mechanism of PrtR.</p></div><div id="Sec2" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec2title">Material and methods</h2><div id="Sec3" class="sec sec-first"><h3 id="Sec3title">Strains, media, and growth condition</h3><p id="Par9" class="p p-first"><em>A. oryzae</em> strains, used in the present study, are listed in Table&#x000a0;<a href="/pmc/articles/PMC10781853/table/Tab1/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab1" rid-ob="ob-Tab1" co-legend-rid=""><span>1</span></a>. The strains were grown at 30&#x000a0;&#x000b0;C in Czapek-Dox (CD) medium (Fujioka et al. <a href="#CR8" rid="CR8" class=" bibr popnode">2007</a>) at pH 5.2. According to the nutrient requirement or drug resistance, 20&#x000a0;mM uracil and 1.1&#x000a0;mg/mL 5-fluoroorotic acid or 0.1&#x000a0;&#x000b5;g/mL pyrithiamine were added. For the solid culture, wheat bran medium prepared by adding 6&#x000a0;mL of reverse osmosis water to 4.2&#x000a0;g of wheat bran (Showa Sangyo Co., Ltd., Tokyo, Japan) was used.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="Tab1"><h3>Table&#x000a0;1</h3><!--caption a7--><div class="caption"><p><em>Aspergillus oryzae</em> strains used in this study</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="left" rowspan="1" colspan="1">Strain</th><th align="left" rowspan="1" colspan="1">Genotypes</th><th align="left" rowspan="1" colspan="1">References</th></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1">RIB40</td><td align="left" rowspan="1" colspan="1">WT</td><td align="left" rowspan="1" colspan="1">ATCC42149 (Akao et al. <a href="#CR2" rid="CR2" class=" bibr popnode">2007</a>)</td></tr><tr><td align="left" rowspan="1" colspan="1">&#x00394;ligD &#x00394;pyrG</td><td align="left" rowspan="1" colspan="1">&#x00394;<em>ligD</em>::<em>ptrA</em> &#x00394;<em>pyrG</em></td><td align="left" rowspan="1" colspan="1">(Kobayashi et al. <a href="#CR17" rid="CR17" class=" bibr popnode">2017</a>)</td></tr><tr><td align="left" rowspan="1" colspan="1">&#x00394;ligD &#x00394;pyrG &#x00394;prtR::[pyrG cre]</td><td align="left" rowspan="1" colspan="1">&#x00394;<em>ligD</em>::<em>ptrA</em> &#x00394;<em>pyrG</em> &#x00394;<em>prtR</em>::[<em>pyrG cre</em>]</td><td align="left" rowspan="1" colspan="1">This study</td></tr><tr><td align="left" rowspan="1" colspan="1">&#x00394;ligD &#x00394;pyrG &#x00394;prtR</td><td align="left" rowspan="1" colspan="1">&#x00394;<em>ligD</em>::<em>ptrA</em> &#x00394;<em>pyrG</em> &#x00394;<em>prtR</em></td><td align="left" rowspan="1" colspan="1">This study</td></tr><tr><td align="left" rowspan="1" colspan="1">&#x00394;prtR</td><td align="left" rowspan="1" colspan="1">&#x00394;<em>ligD</em>::[<em>pyrG ligD</em>] &#x00394;<em>pyrG</em> &#x00394;<em>prtR</em></td><td align="left" rowspan="1" colspan="1">This study</td></tr><tr><td align="left" rowspan="1" colspan="1">Control</td><td align="left" rowspan="1" colspan="1">&#x00394;<em>ligD</em>::[<em>pyrG ligD</em>] &#x00394;<em>pyrG</em></td><td align="left" rowspan="1" colspan="1">This study</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140125894674576"><a target="object" rel="noopener" href="/pmc/articles/PMC10781853/table/Tab1/?report=objectonly">Open in a separate window</a></div></div><p id="Par10">For growth observation, CD agar medium was used; however, 0.6% bovine serum albumin (BSA) (CD/BSA), 0.6% gelatin (CD/gelatin), 0.6% soy protein (CD/soy protein), 1% casein (CD/casein), and 1% skim milk (CD/SM (&#x02212;&#x02009;N)) were used as a single nitrogen source, instead of NaNO<sub>3</sub>. CD agar medium containing 0.6% NaNO<sub>3</sub> and 1% skim milk (CD/SM (+&#x02009;N)) was also prepared. In addition, CD agar medium containing 2% soluble starch instead of Glc (CD/starch) was used.</p><p id="Par11" class="p p-last"><em>Escherichia coli</em> DH5&#x003b1; (F<sup>&#x02212;</sup>, <span id="IEq1"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M2"><mi mathvariant="normal">&#x003a6;</mi></math></span> 80d <em>lacZ</em>DM15, D(<em>lacZYA</em>-<em>argF</em>) U169, <em>deoR</em>, <em>recA</em>1, <em>endA</em>1, <em>hsdR</em>17 (r<sub>K</sub><sup>&#x02212;</sup> m<sub>K</sub><sup>+</sup>), <em>phoA</em>, <em>supE</em>44, &#x003bb;<sup>&#x02212;</sup>, <em>thi</em>-1, <em>gyrA</em>96, <em>relA</em>1) (Takara Bio Inc., Kusatsu, Japan) was used for plasmid construction (Chen et al. <a href="#CR7" rid="CR7" class=" bibr popnode">2018</a>).</p></div><div id="Sec4" class="sec"><h3 id="Sec4title">Construction of prtR deletion strain</h3><p id="Par12" class="p p-first">The cassette for the <em>prtR</em> gene deletion was constructed as follows. Primer sets for deletion mutant construction are listed in Supplemental Table <a href="#MOESM1" rid="MOESM1" class=" media">S1</a>. A DNA fragment from 2&#x000a0;kb upstream to 2&#x000a0;kb downstream of the <em>prtR</em> open reading frame (ORF) was obtained by PCR using phosphorylated primers (prtR_5&#x02032;_F and prtR_3&#x02032;_R) and <em>A. oryzae</em> RIB40 genomic DNA as a template. The fragment was, then, inserted into the <em>Hin</em>cII-treated pUC119 (Takara Bio Inc., Kusatsu, Japan) vector. The constructed plasmid was named as pUC119-prtR. Next, inverse PCR was performed using primers (prtR_3&#x02032;_F and prtR_5&#x02032;_R) and the pUC119-prtR as templates to remove the <em>prtR</em> ORF from the pUC119-prtR. The obtained DNA fragment was ligated with the DNA fragment containing <em>adeA</em> (from <em>A. nidulans</em>) and Cre/<em>loxP</em> expression genes obtained by PCR using primers (creloxp_F and creloxp_R) and the pAAAXP_Cre (Zhang et al. <a href="#CR44" rid="CR44" class=" bibr popnode">2017</a>) as a template. The plasmid pAAAXP_Cre was kindly gifted by Dr. Gomi (Tohoku University). The resulting plasmid was named pUC119_prtRarm_AnadeA_cre. To substitute the <em>adeA</em> gene in the pUC119_prtRarm_AnadeA_cre with the <em>pyrG</em> gene, the <em>pyrG</em> gene fragment was obtained by PCR using phosphorylated primers (pyrG_Fw and pyrG_Rv) and <em>A. oryzae</em> RIB40 genomic DNA as a template. Meanwhile, inverse PCR was performed using primers (creloxp_5&#x02032;_F and creloxp_3&#x02032;_R) and the pUC119_prtRarm_AnadeA_cre as a template to remove the <em>adeA</em> gene region from the pUC119_prtRarm_AnadeA_cre. These two fragments were ligated, and the resulting plasmid was named pUC119_prtRarm_pyrG_cre. As a result, the 2-kb upstream region of the <em>prtR</em> ORF, the <em>pyrG</em> selectable marker, the Cre/<em>loxP</em> expression gene, and the 2-kb downstream region of <em>prtR</em> ORF were arranged in this order from the 5&#x02032; end of the cassette for recombination.</p><p id="Par13" class="p p-last">This cassette was used for transformation of the <em>A. oryzae</em> &#x00394;ligD::ptrA &#x00394;pyrG strain (Kobayashi et al. <a href="#CR17" rid="CR17" class=" bibr popnode">2017</a>) by the protoplast-PEG method (Gomi et al. <a href="#CR11" rid="CR11" class=" bibr popnode">1987</a>), and subsequent selection for deletion of the <em>prtR</em> gene was performed on CD agar medium containing 0.1&#x000a0;&#x000b5;g/mL pyrithiamine. Loop-out of the <em>pyrG</em> gene by the Cre-<em>loxP</em> system was conducted on CDX agar medium (CD agar medium with 2% xylose as the sole carbon source), as described previously (Zhang et al. <a href="#CR44" rid="CR44" class=" bibr popnode">2017</a>). After loop-out of the <em>pyrG</em> gene, the <em>ligD</em> and <em>pyrG</em> genes were complemented using pUC119-ligD-pyrG, as described previously (Kobayashi et al. <a href="#CR17" rid="CR17" class=" bibr popnode">2017</a>).</p></div><div id="Sec5" class="sec"><h3 id="Sec5title">Preparation of genomic DNA and Southern blot analysis</h3><p id="Par14" class="p p-first-last">Genomic DNA preparation and Southern blot analysis were performed as previously described (Kobayashi et al. <a href="#CR17" rid="CR17" class=" bibr popnode">2017</a>). The probes used for Southern blot analysis were amplified using the primers prtR_up2049_F and prtR_up967_R for the left probe and prtR_down97_F and prtR_down1045_R for the right probe to confirm the deletion of the <em>prtR</em> gene, and ligD_up_F and ligD_up_R for the left probe and ligD_down_F and ligD_down_R for the right probe of the <em>ligD</em> and <em>pyrG</em> gene complementation.</p></div><div id="Sec6" class="sec"><h3 id="Sec6title">Measurement of enzymatic activities</h3><p id="Par15" class="p p-first">Twenty-five million conidia of either the control or &#x00394;prtR strains were inoculated into 40&#x000a0;mL of CD-casein (1% casein was used as a single nitrogen source, instead of NaNO<sub>3</sub>) medium in a 120-mL baffled flask and cultured aerobically at 30&#x000a0;&#x000b0;C for 36&#x000a0;h. Culture supernatants were collected by filtration using Miracloth&#x000ae; (Merck KGaA, Darmstadt, Germany) and then centrifuged at 10,000<em>&#x000a0;g</em> and 4&#x000a0;&#x000b0;C for 30&#x000a0;min. The supernatant was dialyzed against a 10&#x000a0;mM acetate buffer (pH 5.0) at 4&#x000a0;&#x000b0;C.</p><p id="Par16" class="p p-last">Acidic endopeptidase activity was measured as previously described (Takyu et al. <a href="#CR34" rid="CR34" class=" bibr popnode">2022</a>). In brief, 0.02&#x000a0;mL of enzyme solution diluted with 0.18&#x000a0;mL of 0.1&#x000a0;M sodium acetate buffer (pH3.0) was preincubated at 30&#x000a0;&#x000b0;C for 10&#x000a0;min, and then, 0.2&#x000a0;mL of 2.0% casein solution (pH3.0) preincubated at 30&#x000a0;&#x000b0;C was added to the enzyme solution. After exactly 60&#x000a0;min of incubation at 30&#x000a0;&#x000b0;C, 0.4&#x000a0;mL of 0.4&#x000a0;M trichloroacetic acid solution was added to stop the enzyme reaction. After leaving the mixture at 30&#x000a0;&#x000b0;C for 20&#x000a0;min, the mixture was centrifuged at 15,000<em>&#x000a0;g</em> for 20&#x000a0;min. One hundred microliters of the supernatant was moved to a new test tube, and 0.5&#x000a0;mL of 0.4&#x000a0;M sodium carbonate solution and 0.1&#x000a0;mL of 0.33&#x000a0;M Folin &#x00026; Ciocalteu&#x02019;s phenol reagent were added to the supernatant. After 30&#x000a0;min of incubation at 30&#x000a0;&#x000b0;C, the absorbance of the mixture at 660&#x000a0;nm was measured using a Hitachi UV-1900 spectrophotometer (Hitachi High-Technologies Corp., Tokyo, Japan). One katal of the aspartic endopeptidase is defined as the enzyme amount, which yields the equivalent of 1.0&#x000a0;mol tyrosine per second at 660&#x000a0;nm, using casein as a substrate under the conditions described above. Inhibitory analysis was performed with 0.1&#x000a0;mM antipain (Peptide Institute, Inc., Osaka, Japan) or 0.1&#x000a0;mM pepstatin A (Peptide Institute, Inc., Osaka, Japan). Carboxypeptidase (CPase) activity was measured according to a previously reported method (Tanaka et al. <a href="#CR35" rid="CR35" class=" bibr popnode">1970</a>; Morita et al. <a href="#CR22" rid="CR22" class=" bibr popnode">2009</a>). One millimolar benzyloxycarbonyl-<span class="small-caps">L</span>-glutamyl-<span class="small-caps">L</span>-tyrosine (Z-Glu-Tyr) was used as a substrate at pH 3.5 in 50&#x000a0;mM sodium acetate buffer. The amount of Tyr liberated from Z-Glu-Tyr was determined as follows: 0.01&#x000a0;mL of enzyme solution diluted with 0.09&#x000a0;mL of 0.1&#x000a0;M sodium acetate buffer (pH 3.5) was preincubated at 30&#x000a0;&#x000b0;C for 10&#x000a0;min, and then, 0.1&#x000a0;mL of 1&#x000a0;mM Z-Glu-Tyr solution (pH3.5) preincubated at 30&#x000a0;&#x000b0;C was added to the enzyme solution. After exactly 60&#x000a0;min of incubation at 30&#x000a0;&#x000b0;C, 0.1&#x000a0;mL of 0.3&#x000a0;M NaOH was added to terminate the reaction. After leaving the mixture at 30&#x000a0;&#x000b0;C for 10&#x000a0;min, 0.2&#x000a0;mL of the mixture was moved to a new test tube, and then, 0.2&#x000a0;mL of 2.5% acetic acid and 2&#x000a0;mL of 0.5&#x000a0;M citrate buffer (pH 5.0) were added. Then, 1&#x000a0;mL of ninhydrin solution (0.83% ninhydrin, 0.17&#x000a0;mM KCN, dissolved in 2-methoxyethanol) was added, and the mixture was heated at 100&#x000a0;&#x000b0;C for 20&#x000a0;min and immediately cooled in an ice water bath. The absorbance of the mixture was measured at 570&#x000a0;nm using the Hitachi UV-1900 spectrophotometer. One katal of the CPase is defined as the enzyme amount, which yields the equivalent of 1&#x000a0;mol tyrosine per second, using Z-Glu-Tyr as a substrate under the conditions described above.</p></div><div id="Sec7" class="sec"><h3 id="Sec7title">Determination of protein concentration</h3><p id="Par17" class="p p-first-last">Protein concentrations were measured by the Bradford method using the Bradford Protein assay kit (Bio-Rad Laboratories K. K., Tokyo, Japan) according to the supplier&#x02019;s protocol (Bradford <a href="#CR4" rid="CR4" class=" bibr popnode">1976</a>).</p></div><div id="Sec8" class="sec sec-last"><h3 id="Sec8title">RNA extraction, cDNA synthesis, and quantitative RT-PCR</h3><p id="Par18" class="p p-first">For liquid culture, 2&#x02009;&#x000d7;&#x02009;10<sup>7</sup> conidia of the control and &#x00394;prtR strains were inoculated into 40&#x000a0;mL of CD medium or CD-casein medium containing 1% casein as a solo nitrogen source without NaNO<sub>3</sub> in a 120-mL baffled flask (Corning Inc., Corning, NY, USA). They were cultured aerobically at 30&#x000a0;&#x000b0;C for 24, 36, and 48&#x000a0;h. RNA extraction, cDNA synthesis, and quantitative RT-PCR were performed as described previously (Kobayashi et al. <a href="#CR17" rid="CR17" class=" bibr popnode">2017</a>). Primer sets used for RT-qPCR are listed in Supplemental Table <a href="#MOESM1" rid="MOESM1" class=" media">S1</a>.</p><p id="Par19" class="p p-last">For solid culture, 3&#x02009;&#x000d7;&#x02009;10<sup>7</sup> conidia of the control and &#x00394;prtR strains were inoculated into 10.2&#x000a0;g of wheat bran medium in Erlenmeyer flask (wide neck: Paul Marienfeld GmbH &#x00026; Co. KG, Lauda-K&#x000f6;nigshofen, Germany) and cultured at 30&#x000a0;&#x000b0;C for 24, 36, 42, and 48&#x000a0;h. The flasks were shaken every 12&#x000a0;h to mix the media. RNA extraction was performed using a previously described method (Cathala et al. <a href="#CR5" rid="CR5" class=" bibr popnode">1983</a>) with some modifications. The mycelia grown on wheat bran were ground to a fine powder in liquid N<sub>2</sub> using a mortar and pestle. Two and half grams of the ground mycelia were suspended in 10&#x000a0;mL of Sepasol&#x000ae;-RNA I SuperG (Nacalai Tesque Inc., Kyoto, Japan) preheated to 50&#x000a0;&#x000b0;C. Then, 4&#x000a0;mL of chloroform was added, and the mixture was centrifuged at 4&#x000a0;&#x000b0;C and 7,500<em>&#x000a0;g</em> for 15&#x000a0;min. The aqueous layer was then subjected to phenol/chloroform extraction. RNA was precipitated using 2-propanol. To this precipitate, 2&#x000a0;mL of guanidine thiocyanate (GuSCN) solution [2.5&#x000a0;M GuSCN, 25&#x000a0;mM Tris&#x02013;HCl (pH 7.5), 5&#x000a0;mM ethylenediaminetetraacetic acid; EDTA (pH 8.0)], 100 &#x000b5;L of 2-mercaptoethanol, and 10&#x000a0;mL of 4&#x000a0;M LiCl were added, followed by incubation at room temperature (25&#x000a0;&#x000b0;C) for 20&#x000a0;min and at 4&#x000a0;&#x000b0;C overnight. The mixture was centrifuged at 4&#x000a0;&#x000b0;C and 500<em>&#x000a0;g</em> for 5&#x000a0;min, and 2&#x000a0;mL of supernatant was centrifuged at 4&#x000a0;&#x000b0;C and 11,000<em>&#x000a0;g</em> for 90&#x000a0;min. The precipitate was suspended in a mixture of 500 &#x000b5;L of GuSCN solution, 50 &#x000b5;L of 2-mercaptoethanol, and 2&#x000a0;mL of 4&#x000a0;M LiCl and further centrifuged at 4&#x000a0;&#x000b0;C and 11,000<em>&#x000a0;g</em> for 60&#x000a0;min. The resulting precipitate was dissolved in 5&#x000a0;mL of TESDS [10&#x000a0;mM Tris&#x02013;HCl (pH 7.5), 1&#x000a0;mM EDTA (pH 8.0), and 0.1% sodium dodecyl sulfate]. The solution was, then, subjected to phenol/chloroform extraction and chloroform extraction. Next, the RNA was precipitated with 2-propanol and dissolved in diethylpyrocarbonate-treated water. DNase I treatment of total RNA, cDNA synthesis, and RT-PCR were performed as described previously (Kobayashi et al. <a href="#CR17" rid="CR17" class=" bibr popnode">2017</a>). The primers used for RT-qPCR are listed in Supplemental Table <a href="#MOESM1" rid="MOESM1" class=" media">S1</a>.</p></div></div><div id="Sec9" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec9title">Results</h2><div id="Sec10" class="sec sec-first"><h3 id="Sec10title">Construction of a prtR deletion strain</h3><p id="Par20" class="p p-first-last">To clarify the role of PrtR in <em>A. oryzae</em>, we constructed a deletion mutant of the <em>prtR</em> gene (Supplemental Fig. <a href="#MOESM1" rid="MOESM1" class=" media">S1</a>A). The &#x00394;ligD &#x00394;pyrG strain was used as the host for the transformation (Kobayashi et al. <a href="#CR17" rid="CR17" class=" bibr popnode">2017</a>). We also constructed a control strain by complementing the deletion of <em>ligD</em> and <em>pyrG</em> in the <em>A. oryzae</em> &#x00394;ligD &#x00394;pyrG strain. These strains were confirmed by Southern blotting (Supplemental Fig. <a href="#MOESM1" rid="MOESM1" class=" media">S1</a>A-S1E).</p></div><div id="Sec11" class="sec"><h3 id="Sec11title">Growth of &#x00394;prtR in various media</h3><p id="Par21" class="p p-first">As PrtR is an ortholog of PrtT in <em>A. niger</em> and <em>A. fumigatus</em>, it is likely that PrtR also promotes the transcription of peptidase genes. We attempted to confirm whether PrtR deficiency reduced proteolytic enzyme production. Furthermore, we hypothesized that the function of PrtR could be altered by the surrounding nitrogen source. Therefore, the control and &#x00394;prtR strains were grown on various CD media with different nitrogen sources, and their growth was compared (Fig.&#x000a0;<a href="/pmc/articles/PMC10781853/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1</span></a>A). No significant difference was observed in growth between the control and &#x00394;prtR strains in CD and CD/soy protein agar media containing NaNO<sub>3</sub> or soy protein as a single nitrogen source. However, when BSA, gelatin, or casein was used as a single nitrogen source, the growth of &#x00394;prtR strain was delayed compared to the control strain. Furthermore, as the <em>prtR</em> gene was originally presented close to the starch degradation cluster, glucose in CD agar medium was substituted with soluble starch to investigate whether carbon catabolite repression is involved in PrtR function. No difference in growth was observed between the control and &#x00394;prtR strains on the CD/starch agar medium.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig1" co-legend-rid="lgnd_Fig1"><a href="/pmc/articles/PMC10781853/figure/Fig1/" target="figure" rid-figpopup="Fig1" rid-ob="ob-Fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140125904737840"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10781853_253_2023_12833_Fig1_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 253_2023_12833_Fig1_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10781853/bin/253_2023_12833_Fig1_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140125904737840"><a target="object" rel="noopener" href="/pmc/articles/PMC10781853/figure/Fig1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig1"><div><a class="figpopup" href="/pmc/articles/PMC10781853/figure/Fig1/" target="figure" rid-figpopup="Fig1" rid-ob="ob-Fig1">Fig.&#x000a0;1</a></div><!--caption a7--><div class="caption"><p>Characterization of PrtR-dependent growth. One thousand conidia of control or &#x00394;prtR strains were inoculated onto CD agar medium containing 0.6% NaNO<sub>3</sub>, 0.6% BSA, 0.6% gelatin, 0.6% soy protein, 1% casein as single nitrogen source, or 2% starch as single carbon source. These were cultured at 30&#x000a0;&#x000b0;C for 3, 5, and 7&#x000a0;days. <strong>A</strong> Colony appearance. <strong>B</strong> Halo assay using casein. Control and &#x00394;prtR strains were inoculated into CD/casein agar medium and cultured at 30&#x000a0;&#x000b0;C for 3&#x000a0;days. After 3&#x000a0;days, they were incubated at 4&#x000a0;&#x000b0;C for 7&#x000a0;days. Furthermore, 0.4&#x000a0;M trichloroacetic acid (TCA) was added, and the proteolytic haloes were observed. <strong>C</strong> Colony appearance and halo assay using skim milk. Control and &#x00394;prtR strains were inoculated into CD/SM(-N) and CD/SM(+&#x02009;N) agar medium at 30&#x000a0;&#x000b0;C for 3 and 5&#x000a0;days. After 3&#x000a0;days, they were incubated at 4&#x000a0;&#x000b0;C for 7&#x000a0;days and the proteolytic haloes were observed. CD/SM agar plates contained skim milk, and &#x0201c;&#x02009;+&#x02009;N&#x0201d; contained NaNO<sub>3</sub>, and &#x0201c;&#x02009;&#x02212;&#x02009;N&#x0201d; plate did not</p></div></div></div><p id="Par22" class="p p-last">To confirm the secretion of proteolytic enzymes, the control and &#x00394;prtR strains were grown on casein-containing agar medium for 3&#x000a0;days, following which the plates were incubated at 4&#x000a0;&#x000b0;C for 7&#x000a0;days. <em>A. oryzae</em> have a fast substrate mycelial elongation speed. As a result, the mycelium grows over any halo formed in a medium, and it is difficult to confirm the halo size. Therefore, the plates were placed at 4&#x000a0;&#x000b0;C to stop mycelial growth and to make halo formation easier to see. Then, 0.4&#x000a0;M trichloroacetic acid was added to the plates, and the halos formed (Fig.&#x000a0;<a href="/pmc/articles/PMC10781853/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1</span></a>B). A halo was formed around the colony of the control strain, whereas no halo was observed in the &#x00394;prtR strain. Next, to observe the relationship between the inorganic nitrogen source and PrtR in the secretory production of proteolytic enzymes, the &#x00394;prtR strain was inoculated on both CD/SM (&#x02212;&#x02009;N) and CD/SM (+&#x02009;N) agar plates, and halos were observed as described above (Fig.&#x000a0;<a href="/pmc/articles/PMC10781853/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1</span></a>C). Both control and &#x00394;prtR strains formed halos on the plate without NaNO<sub>3</sub> (CD/SM (&#x02212;&#x02009;N)); however, the size of the halo was larger in the control strain. In contrast, on CD/SM (+&#x02009;N) medium containing NaNO<sub>3</sub>, only the control strain formed a halo, and the &#x00394;prtR strain could not.</p></div><div id="Sec12" class="sec"><h3 id="Sec12title">Peptidase production of &#x00394;prtR</h3><p id="Par23" class="p p-first">To confirm how PrtR deletion affects peptidase activity, the activities of endopeptidases and carboxypeptidases in the culture supernatants were measured under acidic conditions. In the production of fermented foods, the environment becomes acidic in the initial stage due to lactic acid from halophilic lactic acid bacteria. The peptidases from <em>A. oryzae</em> need to degrade the ingredients of miso and soy source under such acidic condition. Therefore, acid peptidases are important for the industrial use of <em>A. oryzae</em>. The specific activities of acidic endopeptidase in the culture supernatants from the &#x00394;prtR and control strains were 1.0 and 6.3 nkat/mg, respectively, which indicated that approximately 84% of the original activity was lost due to the deletion of PrtR. To confirm what kind of endopeptidase activities were decreased, we performed an inhibitor assay using pepstatin A and antipain. The activity of the control strain was reduced to approximately 66% and 79% of the original activity by pepstatin A and antipain, respectively (Fig.&#x000a0;<a href="/pmc/articles/PMC10781853/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2</span></a>A). This indicated that approximately 34% of the acidic caseinolytic activity was due to aspartic endopeptidases (APases) and approximately 21% of the activity was caused by antipain-sensitive endopeptidases. In the &#x00394;prtR strain, the activity was significantly lost by pepstatin A, and the addition of antipain hardly changed the activity. This suggested that PrtR is involved in multiple acidic endopeptidase production, such as APases and antipain-sensitive endopeptidases.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig2" co-legend-rid="lgnd_Fig2"><a href="/pmc/articles/PMC10781853/figure/Fig2/" target="figure" rid-figpopup="Fig2" rid-ob="ob-Fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140125894666368"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10781853_253_2023_12833_Fig2_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 253_2023_12833_Fig2_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10781853/bin/253_2023_12833_Fig2_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140125894666368"><a target="object" rel="noopener" href="/pmc/articles/PMC10781853/figure/Fig2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig2"><div><a class="figpopup" href="/pmc/articles/PMC10781853/figure/Fig2/" target="figure" rid-figpopup="Fig2" rid-ob="ob-Fig2">Fig.&#x000a0;2</a></div><!--caption a7--><div class="caption"><p>Proteolytic activity of the &#x00394;prtR strain. Control or &#x00394;prtR strains were grown for 36&#x000a0;h in CD-casein liquid medium containing 1% casein as single nitrogen source. <strong>A</strong> Specific activities of acid endopeptidases. Activities of the culture supernatants were measured using 2% casein (pH 3.0) as a substrate. None; no inhibitor. Antipain; cysteine and serine protease inhibitor, antipain added. Dimethyl sulfoxide (DMSO); the solvent for pepstatin A, DMSO added. Peptatin A; APase inhibitor, pepstatin A added. <strong>B</strong> Specific activities of acid carboxypeptidases. Activates of the culture supernatant were measured using 1&#x000a0;mM Z-Glu-Tyr (pH 3.5) as a substrate. The bars represent the mean of biological replicates (<em>n</em>&#x02009;=&#x02009;3), and error bars represent the standard error. The black bars show the results of the control strain, and the gray bars show the results of the &#x00394;prtR strain. Statistical analysis was performed using the Tukey&#x02013;Kramer and <em>t</em> test method. *<em>p</em>&#x02009;&#x0003c;&#x02009;0.05</p></div></div></div><p id="Par24">The carboxypeptidase (CPase) activities of the control strain and the &#x00394;prtR strain were 24.3 and 3.3 nkat/mg, respectively (Fig.&#x000a0;<a href="/pmc/articles/PMC10781853/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2</span></a>B). These results indicated that approximately 86% of the original CPase activity was lost due to the deletion of PrtR.</p><p id="Par25" class="p p-last">These results suggest that PrtR plays an important role in the production of both endopeptidases and exopeptidases.</p></div><div id="Sec13" class="sec"><h3 id="Sec13title">Transcription of peptidase genes in the &#x00394;prtR strain cultured on liquid medium</h3><p id="Par26" class="p p-first">As the proteolytic activities were reduced by PrtR deficiency, we analyzed the transcription of all genes encoding the presumed extracellular peptidases to identify the genes transcriptionally regulated by PrtR.</p><p id="Par27">The genes analyzed are shown in Supplemental Table <a href="#MOESM1" rid="MOESM1" class=" media">S2</a>. First, transcription of <em>prtR</em> was observed in the control strain at all time points in both media but not in the &#x00394;prtR strain (Fig.&#x000a0;<a href="/pmc/articles/PMC10781853/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span>3</span></a>). Thus, it was confirmed that the <em>prtR</em> gene was deficient. The transcriptional levels of <em>prtR</em> in CD-Ca medium at 24 and 36&#x000a0;h of cultivation were approximately twofold higher than those in CD medium.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig3" co-legend-rid="lgnd_Fig3"><a href="/pmc/articles/PMC10781853/figure/Fig3/" target="figure" rid-figpopup="Fig3" rid-ob="ob-Fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140125897926352"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10781853_253_2023_12833_Fig3_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 253_2023_12833_Fig3_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10781853/bin/253_2023_12833_Fig3_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140125897926352"><a target="object" rel="noopener" href="/pmc/articles/PMC10781853/figure/Fig3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig3"><div><a class="figpopup" href="/pmc/articles/PMC10781853/figure/Fig3/" target="figure" rid-figpopup="Fig3" rid-ob="ob-Fig3">Fig.&#x000a0;3</a></div><!--caption a7--><div class="caption"><p>Transcriptional analysis result for transcription factor&#x000a0;genes in the control and &#x00394;prtR strains cultured in liquid medium. Control or &#x00394;prtR strains were grown for 24, 36, and 48&#x000a0;h in CD liquid medium or CD-casein liquid medium containing 1% casein as solo nitrogen source. The <em>y</em>-axis represents the relative mRNA levels of the transcription factor gene compared to that of <em>actA</em>. The bars represent the mean of biological replicates (<em>n</em>&#x02009;=&#x02009;3), and error bars represent the standard error. The black bars show the results of the control strain, and the two gray bars show the results of the &#x00394;prtR strain. Statistical analysis was performed using the Dunnett method. *<em>p</em>&#x02009;&#x0003c;&#x02009;0.05</p></div></div></div><p id="Par28">The change patterns in transcription of endopeptidase genes due to the deletion of <em>prtR</em> were divided into the following four groups (Fig.&#x000a0;<a href="/pmc/articles/PMC10781853/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4</span></a>). Group 1 was genes whose transcription was significantly decreased or tended to decrease in both CD and CD-Ca media due to the deletion of <em>prtR</em>. To this group belonged <em>pepO</em>, <em>pepO4</em>, <em>aorA</em>, <em>aorB</em>, <em>deuA</em>, <em>np3</em>, and <em>pipA</em>, encoding APase (PepO), APase (PepO4), aorsin A, aorsin B, deuterolysin A, metalloendopeptidase NP3, and glutamic endopeptidase (PipA), respectively. Group 2 was genes whose transcription was increased in CD-Ca medium and was hardly changed in CD medium by the deletion of <em>prtR</em>. This was the case for <em>pepO3</em>, encoding APase (PepO3). Group 3 was genes whose transcription was decreased or tended to decrease in CD medium but increased in CD-Ca medium in &#x00394;prtR strain. To this group belonged <em>pepO2</em>, <em>alpA</em>, <em>deuB</em>, and <em>pipC</em>, encoding APase (PepO2), oryzin, deuterolysin B, and glutamic endopeptidase (PipC), respectively. Group 4 consisted of genes that were not altered by the deletion of <em>prtR</em>. To this group belonged <em>pepO5</em> and <em>np1</em> encoding APase (PepO5) and metalloendopeptidase NP1, respectively. However, the transcript level of <em>np1</em> was significantly low.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig4" co-legend-rid="lgnd_Fig4"><a href="/pmc/articles/PMC10781853/figure/Fig4/" target="figure" rid-figpopup="Fig4" rid-ob="ob-Fig4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140125906003536"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10781853_253_2023_12833_Fig4_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 253_2023_12833_Fig4_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10781853/bin/253_2023_12833_Fig4_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140125906003536"><a target="object" rel="noopener" href="/pmc/articles/PMC10781853/figure/Fig4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig4"><div><a class="figpopup" href="/pmc/articles/PMC10781853/figure/Fig4/" target="figure" rid-figpopup="Fig4" rid-ob="ob-Fig4">Fig.&#x000a0;4</a></div><!--caption a7--><div class="caption"><p>Transcriptional analysis result for endopeptidase genes in the control and &#x00394;prtR strains cultured in liquid medium. Control or &#x00394;prtR strains were grown for 24, 36, and 48&#x000a0;h in CD liquid medium or CD-casein liquid medium containing 1% casein as solo nitrogen source. The <em>y</em>-axis represents the relative mRNA levels of the extracellular peptidase gene compared to that of <em>actA</em>. The bars represent the mean of biological replicates (<em>n</em>&#x02009;=&#x02009;3), and error bars represent the standard error. The black bars show the results of the control strain, and the two gray bars show the results of the &#x00394;prtR strain. Statistical analysis was performed using the Dunnett method. *<em>p</em>&#x02009;&#x0003c;&#x02009;0.05</p></div></div></div><p id="Par29" class="p p-last">For exopeptidase genes, the patterns of transcriptional changes due to the deletion of <em>prtR</em> could be divided into a total of five groups. Exopeptidase genes belonging to group 1 to 4 showed the same behavior as endopeptidase genes belonging to the same group due to the deletion of <em>prtR</em>, while the genes classified in the fifth group showed different behavior depending on the culture time in each medium (Fig.&#x000a0;<a href="/pmc/articles/PMC10781853/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5</span></a>). The group 1 consisted of <em>ocpO</em>, <em>ocpA</em>, <em>ocpD</em>, and <em>sep1</em>, encoding CPases. Surprisingly, the transcription of <em>ocpA</em> was almost completely lost in the &#x00394;prtR strain in both media at all cultivation times. The group 2 consisted of <em>cpI</em>, <em>ocpB</em>, <em>ocpE</em>, <em>ocpG</em>, and <em>ocpJ</em>, encoding CPases. In addition, the dipeptidyl peptidases <em>dppB</em>, <em>dppE</em>, and <em>dppF</em> were also classified into the group 2. The group 3 consisted of <em>tppA</em>, encoding tripeptidyl peptidase (TppA). The group 4 consisted of <em>tppB</em>, encoding tripeptidyl peptidase (TppB). The group 5 consisted of <em>ocpC</em>, <em>ocpH</em>, and <em>tppC</em>, encoding CPase (OcpC), CPase (OcpH), and tripeptidyl peptidase (TppC), respectively. Thus, dipeptidyl peptidase genes were classified in the same group, but tripeptidyl peptidase genes were classified in different groups.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig5" co-legend-rid="lgnd_Fig5"><a href="/pmc/articles/PMC10781853/figure/Fig5/" target="figure" rid-figpopup="Fig5" rid-ob="ob-Fig5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140125896905488"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10781853_253_2023_12833_Fig5_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 253_2023_12833_Fig5_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10781853/bin/253_2023_12833_Fig5_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140125896905488"><a target="object" rel="noopener" href="/pmc/articles/PMC10781853/figure/Fig5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig5"><div><a class="figpopup" href="/pmc/articles/PMC10781853/figure/Fig5/" target="figure" rid-figpopup="Fig5" rid-ob="ob-Fig5">Fig.&#x000a0;5</a></div><!--caption a7--><div class="caption"><p>Transcriptional analysis result for exopeptidase genes in the control and &#x00394;prtR strains cultured in liquid medium. Control or &#x00394;prtR strains were grown for 24, 36, and 48&#x000a0;h in CD liquid medium or CD-casein liquid medium containing 1% casein as solo nitrogen source. The <em>y</em>-axis represents the relative mRNA levels of the extracellular peptidase gene compared to that of <em>actA</em>. The bars represent the mean of biological replicates (<em>n</em>&#x02009;=&#x02009;3), and error bars represent the standard error. The black bars show the results of the control strain, and the two gray bars show the results of the &#x00394;prtR strain. Statistical analysis was performed using the Dunnett method. *<em>p</em>&#x02009;&#x0003c;&#x02009;0.05</p></div></div></div></div><div id="Sec14" class="sec sec-last"><h3 id="Sec14title">Transcription of peptidase genes in the &#x00394;prtR strain cultured on solid medium</h3><p id="Par30" class="p p-first">Solid-state cultures are known to be more productive and produce a greater variety of enzymes than liquid cultures (Murakami <a href="#CR24" rid="CR24" class=" bibr popnode">1986</a>). Therefore, to understand the role of PrtR in solid-state cultures, we analyzed the transcriptional levels of peptidase genes in solid-state cultures. Wheat bran was selected as a substrate for solid-state culture, as wheat bran culture can produce <em>A. oryzae</em> and <em>Aspergillus terreus</em> high extracellular proteolytic enzyme (Zanutto-Elgui et al. <a href="#CR43" rid="CR43" class=" bibr popnode">2019</a>; Abu-Tahon et al. <a href="#CR1" rid="CR1" class=" bibr popnode">2020</a>). In addition, wheat bran is available at a low cost as an agricultural residue, and its use to produce proteolytic enzymes as a culture substrate can contribute to resource recycling.</p><p id="Par31">The &#x00394;prtR and control strains were cultured for 24, 36, 42, and 48&#x000a0;h in wheat bran medium. The growth of both strains was similar (Supplemental Fig. <a href="#MOESM1" rid="MOESM1" class=" media">S2</a>A and B).</p><p id="Par32">The transcriptional levels of the genes encoding transcription factors, PrtR and FlbC, at 36 and 42&#x000a0;h are shown in Fig.&#x000a0;<a href="/pmc/articles/PMC10781853/figure/Fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig6" rid-ob="ob-Fig6" co-legend-rid="lgnd_Fig6"><span>6</span></a>. The <em>prtR</em> gene was transcribed at both cultivation times in the control strain but was not observed at 36 and 42&#x000a0;h in the &#x00394;prtR strain as described above. Tanaka et al. (<a href="#CR36" rid="CR36" class=" bibr popnode">2016</a>) have reported that <em>pepO</em> transcription is regulated by FlbC in a solid-state specific manner. To investigate the relationship between PrtR and FlbC in solid-state cultures, we analyzed the transcription levels of <em>flbC</em>. In the &#x00394;prtR strain, the transcription of <em>flbC</em> was higher at 36&#x000a0;h and lower at 42&#x000a0;h than that in the control strain.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig6" co-legend-rid="lgnd_Fig6"><a href="/pmc/articles/PMC10781853/figure/Fig6/" target="figure" rid-figpopup="Fig6" rid-ob="ob-Fig6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140125902497376"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10781853_253_2023_12833_Fig6_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 253_2023_12833_Fig6_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10781853/bin/253_2023_12833_Fig6_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140125902497376"><a target="object" rel="noopener" href="/pmc/articles/PMC10781853/figure/Fig6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig6"><div><a class="figpopup" href="/pmc/articles/PMC10781853/figure/Fig6/" target="figure" rid-figpopup="Fig6" rid-ob="ob-Fig6">Fig.&#x000a0;6</a></div><!--caption a7--><div class="caption"><p>Transcriptional analysis result for transcription factor&#x000a0;genes in the control and &#x00394;prtR strains cultured on what bran medium. Control or &#x00394;prtR strains were grown for 36 and 42&#x000a0;h in wheat bran medium. The <em>y</em>-axis represents the relative mRNA levels of the transcription factor gene compared to that of <em>actA</em>. The bars represent the mean of biological replicates (<em>n</em>&#x02009;=&#x02009;3), and error bars represent the standard error. The black bars show the results of the control strain, and the gray bars show the results of the &#x00394;prtR strain. Statistical analysis was performed using the <em>t</em> test method. *<em>p</em>&#x02009;&#x0003c;&#x02009;0.05</p></div></div></div><p id="Par33">The change patterns in transcription of endopeptidase genes due to the deletion of <em>prtR</em> in solid culture were divided into the following three groups (Fig.&#x000a0;<a href="/pmc/articles/PMC10781853/figure/Fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig7" rid-ob="ob-Fig7" co-legend-rid="lgnd_Fig7"><span>7</span></a>). Group 1 was genes whose transcription was decreased at both or either culture times due to the deletion of <em>prtR</em>. This group consisted of <em>pepO</em>, <em>pepO2</em>, <em>pepO5</em>, <em>alpA</em>, <em>aorA</em>, <em>aorB</em>, <em>deuA</em>, <em>deuB</em>, <em>np1</em>, <em>np3,</em> and <em>pipA</em>. Group 2 was genes whose transcription was increased at both or either culture times. This was the case for <em>pepO3</em> and <em>pipC</em>. Group 3 was genes with no transcriptional change at both culture times. The group 3 consisted of only <em>pepO4</em>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig7" co-legend-rid="lgnd_Fig7"><a href="/pmc/articles/PMC10781853/figure/Fig7/" target="figure" rid-figpopup="Fig7" rid-ob="ob-Fig7"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140125905520048"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10781853_253_2023_12833_Fig7_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 253_2023_12833_Fig7_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10781853/bin/253_2023_12833_Fig7_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140125905520048"><a target="object" rel="noopener" href="/pmc/articles/PMC10781853/figure/Fig7/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig7"><div><a class="figpopup" href="/pmc/articles/PMC10781853/figure/Fig7/" target="figure" rid-figpopup="Fig7" rid-ob="ob-Fig7">Fig.&#x000a0;7</a></div><!--caption a7--><div class="caption"><p>Transcriptional analysis result for endopeptidase genes in the control and &#x00394;prtR strains cultured on what bran medium. Control or &#x00394;prtR strains were grown for 36 and 42&#x000a0;h in wheat bran medium. The <em>y</em>-axis represents the relative mRNA levels of the extracellular peptidase gene compared to that of <em>actA</em>. The bars represent the mean of biological replicates (<em>n</em>&#x02009;=&#x02009;3), and error bars represent the standard error. The black bars show the results of the control strain, and the gray bars show the results of the &#x00394;prtR strain. Statistical analysis was performed using the <em>t</em> test method. *<em>p</em>&#x02009;&#x0003c;&#x02009;0.05</p></div></div></div><p id="Par34" class="p p-last">For exopeptidase genes, the patterns of transcriptional changes due to the deletion of <em>prtR</em> could be divided into the above three groups (Fig.&#x000a0;<a href="/pmc/articles/PMC10781853/figure/Fig8/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig8" rid-ob="ob-Fig8" co-legend-rid="lgnd_Fig8"><span>8</span></a>). The group 1 consisted of <em>ocpO</em>, <em>ocpA</em>, <em>ocpD</em>, <em>sep1</em>, and <em>tppA</em>. Transcript levels of all genes were significantly decreased at both incubation times. The group 2 consisted of <em>ocpB</em>, <em>ocpC</em>, <em>ocpE</em>, <em>ocpG</em>, <em>ocpJ</em>, <em>dppB</em>, <em>dppE</em>, <em>dppF</em>, and <em>tppC</em>. The group 3 consisted of <em>cpI</em>, <em>ocpH</em>, and <em>tppB</em>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig8" co-legend-rid="lgnd_Fig8"><a href="/pmc/articles/PMC10781853/figure/Fig8/" target="figure" rid-figpopup="Fig8" rid-ob="ob-Fig8"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140125900550368"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 253_2023_12833_Fig8_HTML.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is 253_2023_12833_Fig8_HTML.jpg" src="/pmc/articles/PMC10781853/bin/253_2023_12833_Fig8_HTML.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140125900550368"><a target="object" rel="noopener" href="/pmc/articles/PMC10781853/figure/Fig8/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig8"><div><a class="figpopup" href="/pmc/articles/PMC10781853/figure/Fig8/" target="figure" rid-figpopup="Fig8" rid-ob="ob-Fig8">Fig.&#x000a0;8</a></div><!--caption a7--><div class="caption"><p>Transcriptional analysis result for exopeptidase genes in the control and &#x00394;prtR strains cultured on what bran medium. Control or &#x00394;prtR strains were grown for 36 and 42&#x000a0;h in wheat bran medium. The <em>y</em>-axis represents the relative mRNA levels of the extracellular peptidase gene compared to that of <em>actA</em>. The bars represent the mean of biological replicates (<em>n</em>&#x02009;=&#x02009;3), and error bars represent the standard error. The black bars show the results of the control strain, and the gray bars show the results of the &#x00394;prtR strain. Statistical analysis was performed using the <em>t</em> test method. *<em>p</em>&#x02009;&#x0003c;&#x02009;0.05</p></div></div></div></div></div><div id="Sec15" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec15title">Discussion</h2><div id="Sec16" class="sec sec-first"><h3 id="Sec16title">PrtR is involved in promoting the production of peptidases in<em> A. oryzae</em></h3><p id="Par35" class="p p-first"><em>A. oryzae</em> PrtR is an orthologous protein of PrtT, which has been reported to positively regulate the transcription of several peptidase genes in <em>Aspergillus</em>. However, the detailed effect of PrtR on peptidase production has not yet been studied in <em>A. oryzae.</em> In this study, we aimed to identify the genes under the control of <em>A. oryzae</em> PrtR and elucidate their transcriptional regulatory mechanism. Therefore, we created a strain with the same genetic background as the wild-type strain, RIB40, but lacking the <em>prtR</em> gene. This is because many of the marker genes for transformation used in <em>A. oryzae</em> encode the proteins involved in nitrogen metabolism, and it was anticipated that amino acids or nucleic acids added to meet the requirements of these gene deletions and deficiencies would affect the transcription of proteolytic enzyme genes involved in nitrogen metabolism (Kitamoto <a href="#CR15" rid="CR15" class=" bibr popnode">2002</a>).</p><p id="Par36">When the &#x00394;prtR strain was grown on CD agar medium with a single protein other than soy protein as the nitrogen source, delayed growth and reduced density of conidia were observed (Fig.&#x000a0;<a href="/pmc/articles/PMC10781853/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1</span></a>A). This was consistent with the results obtained for the <em>A. fumigatus</em> &#x00394;prtT strain grown on CD/BSA agar medium. <em>A. fumigatus</em> PrtT positively regulates the transcription of several peptidase genes, and its deficiency causes a reduction in the transcription of peptidase genes (Bergmann et al. <a href="#CR3" rid="CR3" class=" bibr popnode">2009</a>). It was analogously presumed that <em>A. oryzae</em> PtrR is a positive transcriptional regulator of peptidase genes. Therefore, it was considered that the PrtR deletion should repress the transcription of the peptidase genes and reduce peptidase production. This could explain the delayed growth of the &#x00394;prtR strain on plates containing BSA, gelatin, and casein as the single nitrogen source. However, the effect of PrtR deficiency was barely observed in the culture on CD/soy protein and CD/SM plates. As soy protein and skim milk are not pure proteins and contain some peptides and amino acids, even the &#x00394;prtR strain can grow. Halo formation was not observed in the &#x00394;prtR strain grown on CD/casein agar plates. This suggests that PrtR is strongly involved in the production of peptidases necessary for casein degradation (Fig.&#x000a0;<a href="/pmc/articles/PMC10781853/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1</span></a>B). On CD/SM(&#x02212;&#x02009;N) plates without NaNO<sub>3</sub>, the &#x00394;prtR strain formed a smaller halo than the control strain (Fig.&#x000a0;<a href="/pmc/articles/PMC10781853/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1</span></a>C). These results suggest that the &#x00394;prtR strain could grow in the absence of inorganic nitrogen by inducing the transcription of peptidase genes that are not under the control of PrtR or that are under the control of PrtR but induced by other pathways. In contrast, on the CD/SM(+&#x02009;N) agar plate, the &#x00394;prtR strain did not form a halo, but the growth was not different from that of the control strain. These results indicate that inorganic nitrogen inhibits the production of peptidases encoded by genes that are not under the control of PrtR or induced by other pathways. This suggests that, in the presence of nitrate ions, the &#x00394;prtR strain prefers to take up nitrate as a nitrogen source, rather than secreting peptidases to degrade surrounding proteins and taking up amino acids and oligopeptides as nitrogen sources. In contrast, in the control strain, no significant difference was noted in halo formation with or without sodium nitrate, but the colony diameter on the CD/SM(+&#x02009;N) plate at day 5 was larger than that on the CD/SM(&#x02212;&#x02009;N) plate. This indicates that the control strain utilized inorganic nitrogen even in the presence of PrtR. It was assumed that <em>A. oryzae</em> would utilize proteins and inorganic nitrogen in parallel in order to grow efficiently. The presence of inorganic nitrogen did not inhibit protein utilization.</p><p id="Par37" class="p p-last">In the <em>A. oryzae</em> genome, the <em>prtR</em> gene is located close to the starch-degrading enzyme gene cluster. The distance between the <em>prtR</em> gene and the closest <em>amyA</em> gene was only approximately 1 kbp. It was assumed that <em>prtR</em> is involved in the transcription of starch degradation-related gene clusters. However, no differences in colony diameter or growth were observed between the control strain and the &#x00394;prtR strain on CD plates containing soluble starch as a single carbon source (Fig.&#x000a0;<a href="/pmc/articles/PMC10781853/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1</span></a>A). Thus, PrtR was indicated not to be involved in the production of starch-degrading enzymes such as amylase.</p></div><div id="Sec17" class="sec"><h3 id="Sec17title">Culture conditions alter <em>prtR</em> transcript levels and the effects of its translation product</h3><p id="Par38" class="p p-first">RT-qPCR was performed to determine which peptidase genes are regulated by PrtR. When the control strain was grown in CD-Ca liquid medium, <em>prtR</em> transcription increased compared to that in CD liquid medium, indicating that PrtR transcription was enhanced by proteins as the single nitrogen source. <em>A. oryzae</em> is likely to increase peptidase production by increasing the amount of PrtR via enhanced <em>prtR</em> transcription. Focusing on the overall transcription of peptidase genes, the number of peptidase genes whose transcriptional levels were altered by the deletion of PrtR was small in the CD liquid culture compared to that in the CD-Ca liquid culture. It would indicate that the enhancing effect of PrtR on transcription was low when inorganic nitrogen was the sole nitrogen source. <em>A. oryzae</em> might preferentially utilize inorganic nitrogen in the liquid culture and not need to secrete many peptidases.</p><p id="Par39" class="p p-last">The transcriptional levels of <em>prtR</em> in the control strain were similar in both wheat bran medium and CD liquid medium. However, the overall transcription of peptidase genes was higher in wheat bran culture. The number of peptidase genes altered by the deletion of <em>prtR</em> was higher in wheat bran culture than in CD liquid culture. In solid cultures, the effect of <em>prtR</em> regulation is extremely high, and the signal may be used for transcription promotion more effectively than in liquid cultures. Furthermore, there were some genes for which the mode of transcriptional regulation by PrtR changed from positive to negative or negative to positive in solid and liquid cultures. Glucose starvation in wheat bran medium leads to the release of carbon catabolite repression (CCR; Maeda et al. <a href="#CR21" rid="CR21" class=" bibr popnode">2004</a>). Thus, it is possible that genes under the control of PrtR and regulated by CCR have different modes of transcriptional regulation by PrtR depending on the culture conditions. Moreover, transcriptional regulation may differ from that in glucose-rich liquid cultures. Alternatively, signals other than nitrogen sources, such as water activity and temperature, may be important for the activation of peptidase gene transcription by PrtR in solid culture. Furthermore, other transcription factors, such as FlbC, may be closely related to PrtR. FlbC promotes the transcription of <em>pepO</em> and <em>deuA</em> in solid culture. The transcription levels of the <em>flbC</em> gene in the &#x00394;prtR strain cultured for 36&#x000a0;h in wheat bran medium were significantly higher than those in the control strain. Furthermore, the transcript levels of the <em>flbC</em> gene increased in the &#x00394;prtR strain, but the transcription of many peptidase genes, including <em>pepO</em> and <em>deuA</em>, decreased. This suggests that FlbC promotes the transcription of <em>prtR</em> which in turn promotes the transcription of peptidase genes. PrtR cannot cause an increase in transcription of peptidase genes in the &#x00394;prtR strain. Therefore, it was considered that the feedback reaction would increase the transcription of the <em>flbC</em> gene, but the transcription of the peptidase genes in the &#x00394;prtR strain would be reduced compared to the control strain.</p></div><div id="Sec18" class="sec"><h3 id="Sec18title">Proteolytic activities in culture broth are strongly suppressed by deletion of PrtR</h3><p id="Par40" class="p p-first">The results of the peptidase activity assay and the transcriptional analysis were shown to be consistent with each other. Deletion of PrtR reduced acidic endopeptidase activity by about 84%. It would be due to mainly reflecting the decreased transcription of <em>pepO</em>, an endopeptidase that primarily works under acidic conditions (Takeuchi et al. <a href="#CR33" rid="CR33" class=" bibr popnode">1995</a>). Similarly, the transcription of <em>ocpO</em>, a representative CPase that works under acidic conditions (Morita et al. <a href="#CR23" rid="CR23" class=" bibr popnode">2011</a>), was decreased by the deletion of <em>prtR</em>. As a result, CPase activity is also reduced by 86%.</p><p id="Par41" class="p p-last">It was shown that the acidic endopeptidase activity of the control strain was inhibited to 21% and 34% by antipain and pepstatin A, respectively. Antipain is known as an inhibitor for cysteine peptidases and serine peptidases (Hoebeke et al. <a href="#CR12" rid="CR12" class=" bibr popnode">1994</a>), and pepstatin A inhibits APase (Purushothaman et al. <a href="#CR29" rid="CR29" class=" bibr popnode">2019</a>). It was presumed that antipain-sensitive acid endopeptidases in <em>A. oryzae</em> would be aorsins (AorA and AorB; Lee et al. <a href="#CR18" rid="CR18" class=" bibr popnode">2003</a>). The activity of the &#x00394;prtR strain was not inhibited by antipain; therefore, PrtR would regulate the aorsin genes. This was consistent with the results of transcription analysis. In addition, 15% of the remaining activity in the control strain was completely eliminated by pepstatin A. Based on this result and the fact that transcription of APase genes occurred in the &#x00394;prtR strain, even at a low level, it was suggested that these enzymes would be responsible for the residual acidic endopeptidase activity in the &#x00394;prtR strain.</p></div><div id="Sec19" class="sec"><h3 id="Sec19title">PrtR can positively or negatively regulate the transcription of peptidase genes and the regulation altered by nitrogen source in the medium</h3><p id="Par42" class="p p-first">Among the peptidase genes in liquid culture, the <em>ocpA</em> and <em>aorA</em> genes scarcely showed transcription in &#x00394;prtR strain at any time point in CD-Ca medium. PrtR was considered essential for these transcriptions when the nitrogen source was a protein. Notably, the transcription of these genes was induced by PrtR even when inorganic nitrogen was the single nitrogen source. Although not at all time points, it was also shown that <em>deuA</em>, <em>pipA</em>, <em>ocpO</em>, <em>ocpD</em>, and <em>sep1</em> were positively regulated by PrtR in both liquid cultures containing inorganic and organic nitrogen. These results indicate that these genes were also transcriptionally promoted by PrtR, regardless of the type of nitrogen source. Some genes, such as <em>pepO</em>, encoding a typical APase, were positively regulated by PrtR when the nitrogen source was protein in the liquid medium. As PepO is a typical endopeptidase that works under acidic conditions (Takeuchi et al. <a href="#CR33" rid="CR33" class=" bibr popnode">1995</a>), this result is consistent with the significant reduction in proteolytic enzyme activity under acidic conditions in the &#x00394;prtR strain. However, because some genes were negatively regulated by PrtR even when the protein was the single nitrogen source, it was suggested that PrtR selectively regulates peptidases to efficiently degrade proteins. Furthermore, certain genes were found to be transcriptionally promoted by PrtR when inorganic nitrogen was the sole nitrogen source, and their transcription was repressed when the protein was the single nitrogen source, while certain other genes showed the opposite behavior, which is unlikely to be reasonable in terms of nitrogen source utilization. The peptidase encoded by such genes may be used as a sensor to search for nitrogen sources in liquid culture. The fact that these peptidases were under the positive control of PrtR in wheat bran culture suggests that their roles may differ between liquid and solid cultures. In wheat bran medium, the transcription of genes encoding PepO (Kitano et al. <a href="#CR16" rid="CR16" class=" bibr popnode">2002</a>), AorA (Lee et al. <a href="#CR18" rid="CR18" class=" bibr popnode">2003</a>), DeuA (Tatsumi et al. <a href="#CR38" rid="CR38" class=" bibr popnode">1991</a>), and OcpA (Takeuchi et al. <a href="#CR32" rid="CR32" class=" bibr popnode">1982</a>) specifically produced in solid culture, was significantly reduced by the deletion of PrtR. This indicates that PrtR strongly and positively regulates transcription in solid culture.</p><p id="Par43" class="p p-last">Comparing the genes regulated by PrtR in liquid and solid cultures, the peptidase genes that were positively regulated in both cultures were <em>pepO</em>, <em>aorA</em>, <em>aorB</em>, <em>deuA</em>, <em>np3</em>, <em>pipA</em>, <em>ocpO</em>, <em>ocpA</em>, <em>ocpD</em>, and <em>sep1</em>. In particular, <em>ocpA</em> and <em>aorA</em> were significantly downregulated in the &#x00394;prtR strain at all time points in both liquid and solid cultures. These results suggest that <em>ocpA</em> and <em>aorA</em> are strongly and positively regulated by PrtR in both liquid and solid cultures and that PrtR is essential for their transcription. Moreover, <em>pepO</em> and <em>pipA</em> were strongly downregulated in solid culture due to PrtR deficiency, whereas a certain level of transcription was observed in the &#x00394;prtR strain in liquid culture. This suggests that transcription of these genes may be promoted by factors other than PrtR in liquid culture. In addition, there was another group, <em>deuA</em> and <em>deuB</em>, whose transcriptional levels were low in liquid culture but high and strongly promoted by PrtR in solid culture. These results also indicate that PrtR regulation is highly effective in solid culture. The <em>alpA</em> gene encoding oryzin, a typical alkaline protease (Nakadai et al. <a href="#CR25" rid="CR25" class=" bibr popnode">1973</a>), was positively regulated by PrtR in solid and CD liquid cultures but negatively regulated in CD-Ca liquid culture in the early stage of culture. However, <em>alpA</em> was not regulated by PrtR under any culture conditions during the late culture stages. These results suggest that <em>alpA</em> is regulated by nitrogen source and culture time through PrtR, as well as other factors. Transcription factor PacC promotes the transcription of alkaline peptidase genes under the alkaline condition (Tilburn et al. <a href="#CR40" rid="CR40" class=" bibr popnode">1995</a>), and the pH of the culture broth gradually increases in the late stage of culture when <em>A. oryzae</em> is cultured. This suggests that the transcription of the <em>alpA</em> gene in the early stage of <em>A. oryzae</em> culture would be controlled by PrtR, but the transcription would be controlled by PacC in the late stage of culture due to an increase in pH. Furthermore, PrtR and PacC would regulate the transcription of <em>alpA</em> through separate pathways, as the transcription of <em>prtR</em> did not show a noticeable decrease even at the late culture stage. Genes such as <em>pepO2</em>, <em>pipC</em>, <em>ocpC</em>, <em>tppA</em>, and <em>tppC</em> were positively or negatively regulated by PrtR depending on the culture conditions, and genes such as <em>pepO3</em>, <em>ocpB</em>, <em>ocpE</em>, <em>ocpG</em>, <em>ocpJ</em>, <em>dppB</em>, <em>dppE</em>, and <em>dppF</em> were upregulated by PrtR deficiency. Thus, PrtR may be a transcription factor that promotes and represses transcription. It is possible that the direction of PrtR regulation changed with culture conditions, either because PrtR regulation was altered by culture conditions through some signals, or because other transcription factors were activated to compensate for the reduced peptidase production caused by PrtR deficiency. Some genes are always negatively regulated by PrtR, but it could not be determined whether PrtR binds directly upstream of these peptidase genes and represses their transcription.</p></div><div id="Sec20" class="sec sec-last"><h3 id="Sec20title">PrtR&#x02019;s control pattern is not exactly the same as PrtTs</h3><p id="Par44" class="p p-first">Moreover, <em>pep1</em>, <em>alp1</em>, and Afu2g17330 are positively controlled by PrtT in <em>A. fumigatus</em> (Bergmann et al. <a href="#CR3" rid="CR3" class=" bibr popnode">2009</a>; Sharon et al. <a href="#CR30" rid="CR30" class=" bibr popnode">2009</a>). These peptidase genes were orthologs of <em>pepO</em>, <em>alpA</em>, and <em>sep1</em>, respectively, and these three genes were under the positive control of PrtR in <em>A. oryzae</em>. This indicated that the mode of PrtT regulation is conserved among closely related species (Sharon et al. <a href="#CR30" rid="CR30" class=" bibr popnode">2009</a>). However, Bergmann et al. (<a href="#CR3" rid="CR3" class=" bibr popnode">2009</a>) reported that when the <em>A. fumigatus</em> &#x00394;prtT strain was grown on a medium containing BSA as a nitrogen source, the transcription of <em>dppIV</em> was reduced and that of <em>dppV</em> was unchanged compared to the wild-type strain. A similar study conducted by Sharon et al. (<a href="#CR30" rid="CR30" class=" bibr popnode">2009</a>) also reported a decrease in <em>dppIV</em> transcription in a medium with skim milk as the nitrogen source. In this study, the results for <em>dppB</em> (an ortholog of <em>A. fumigatus dppIV</em>) and <em>dppE</em> (an ortholog of Af<em>dppV</em>) differed from those of previous studies. This might be due to the difference in the protein used as a nitrogen source, but it is also possible that the regulatory mode of PrtR/PrtT for DppIV and DppV differs among species. Further experiments using BSA or skim milk as a nitrogen source will help elucidate the regulatory mechanism of dipeptidyl peptidase genes in <em>Aspergillus</em>.</p><p id="Par45">The putative binding sequence 5&#x02032;-CCGHCGG-3&#x02032; (H;A/C/T) of <em>A. niger</em> PrtT (Huang et al. <a href="#CR13" rid="CR13" class=" bibr popnode">2020</a>) or 5&#x02032;-CHGH(D)CGG-3&#x02032; (H; A/C/T, D; A/G/T) of <em>P. oxalicum</em> PrtT (Chen et al. <a href="#CR6" rid="CR6" class=" bibr popnode">2014</a>) was not necessarily located within 1000&#x000a0;bp of the upstream region of all genes classified as the group 1. Therefore, the DNA-binding sequence of PrtR in <em>A. oryzae</em> may be different from that in <em>A. niger</em> or <em>P. oxalicum</em>. Furthermore, the DNA binding sequence of <em>A. niger</em> PrtT is derived from MEME analysis, not EMSA or DNA footprinting analysis. It may be necessary to confirm that PrtT binds this sequence. MEME analysis was performed on all genes in the group 1 or on all genes that showed changes in their transcript levels in the &#x00394;prtR strain. However, each analysis did not reveal any statistically significant motifs common to all of the genes. MEME analysis has difficulty detecting DNA binding sequences that contain insertions/deletions. If PrtR&#x02019;s binding sequence contained insertions/deletions like that of <em>P. oxalicum,</em> conserved sequence would not be found. Otherwise, these results of MEME analysis would suggest that some peptidase genes are directly regulated by PrtR, while others are indirectly regulated. In cases where PrtR promotes transcription, PrtR might bind to DNA alone or in a complex with other factors. Binding sequences may be different when binding alone and when binding as a complex. If transcription appears to be repressed by PrtR, it might be indirectly regulated by factors whose expression is promoted by PrtR. It was thought that genes regulated in different ways might be classified in the same group, since they were only evaluated at the final transcript level. To determine the direct binding sequence of PrtR, we are going to perform EMSAs with purified PrtR. This would clarify the differences between PrtR and orthologs of other organisms.</p><p id="Par46" class="p p-last">This study revealed that PrtR transcriptionally regulates peptidase genes and that the target genes under control are independent of substrate specificity. PrtR is involved in the regulation of most extracellular peptidase genes, suggesting that PrtR selectively regulates transcription to produce optimal peptidases depending on the culture conditions. Further studies are needed to elucidate the signals to which PrtR responds, the activation mechanism, and the DNA-binding sequence. Elucidation of the PrtR response network to the environment will provide important information on the peptidase production mechanism of <em>A. oryzae</em>, which has not yet been studied and will lead to the enhancement of the value of <em>A. oryzae</em> in the enzyme industry.</p></div></div><div id="sec-a.o.e" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="headless nomenu"></h2><!--/article/body/sec/--><div id="Sec21" class="sec sec-first-last"><h3 id="Sec21title">Supplementary Information</h3><p class="p p-first-last">Below is the link to the electronic supplementary material.</p><div class="sec suppmat" id="MOESM1"><div class="sup-box half_rhythm" id="media-a.o.e.a.b.a.a"><a href="/pmc/articles/PMC10781853/bin/253_2023_12833_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl">Supplementary file1 (PDF 936 KB)</a><sup>(936K, pdf)</sup></div></div><p></p></div></div><div id="ack-a.p.b" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.p.btitle">Acknowledgements</h2><div class="sec"><p>In addition, we thank Dr. Katsuya Gomi (Laboratory of Fermentation Microbiology, Department of Bioindustrial Informatics and Genomics, Graduate School of Agricultural Science, Tohoku University) for kindly providing the plasmid pAAAXP_Cre.</p></div></div><div id="notes-a.p.c" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.p.ctitle">Author contribution</h2><p>RN, YT, SN, and RT constructed strains and performed experiments. HM, MT, MT, and YY designed and planned the study. RN, YT, and YY drafted the manuscript; all authors revised the manuscript, RN and YY finalized the manuscript. All authors agreed to the final version of the manuscript.</p></div><div id="notes-a.p.d" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.p.dtitle">Funding</h2><p>This study was partially supported by Noda Institute for Scientific Research Grant.</p></div><div id="notes-a.p.e" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.p.etitle">Data availability</h2><p>All data generated or analyzed during this study are included in this published article [and its supplementary information files].</p></div><div id="notes-a.p.f" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.p.ftitle">Code availability</h2><p>Not applicable.</p></div><div id="notes-a.p.g" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.p.gtitle">Declarations</h2><div>Ethics approval<p id="Par47">This article does not contain any studies with human participants or animals performed by any of the authors.</p></div><div>Consent to participate<p id="Par48">Not applicable.</p></div><div>Consent for publication<p id="Par49">Not applicable.</p></div><div>Conflict of interest<p id="Par50">The authors declare no competing interests.</p></div></div><div id="fn-group-a.p.a" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.p.atitle">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="fn-a.p.a.a"><p class="p p-first"><strong>Publisher's Note</strong></p><p class="p p-last">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></p><p class="fn sec" id="fn-a.p.a.b"><p class="p p-first-last">Rika Numazawa and Yukako Tanaka contributed equally to this work.</p></p></div></div><div id="Bib1" class="tsec sec"><h2 class="head no_bottom_margin" id="Bib1title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:disc;"><li id="CR1"><span class="element-citation">Abu-Tahon MA, Arafat HH, Isaac GS. Laundry detergent compatibility and dehairing efficiency of alkaline thermostable protease produced from <em>Aspergillus terreus</em> under solid-state fermentation. <span><span class="ref-journal">J Oleo Sci. </span>2020;<span class="ref-vol">69</span>(3):241–254. doi: 10.5650/jos.ess19315.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32115548" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.5650%2Fjos.ess19315" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Oleo+Sci&amp;title=Laundry+detergent+compatibility+and+dehairing+efficiency+of+alkaline+thermostable+protease+produced+from+Aspergillus+terreus+under+solid-state+fermentation&amp;author=MA+Abu-Tahon&amp;author=HH+Arafat&amp;author=GS+Isaac&amp;volume=69&amp;issue=3&amp;publication_year=2020&amp;pages=241-254&amp;pmid=32115548&amp;doi=10.5650/jos.ess19315&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR2"><span class="element-citation">Akao T, Sano M, Yamada O, Akeno T, Fujii K, Goto K, Ohashi-Kunihiro S, Takase K, Yasukawa-Watanabe M, Yamaguchi K, Kurihara Y, Maruyama J, Juvvadi PR, Tanaka A, Hata Y, Koyama Y, Yamaguchi S, Kitamoto N, Gomi K, Abe K, Takeuchi M, Kobayashi T, Horiuchi H, Kitamoto K, Kashiwagi Y, Machida M, Akita O. Analysis of expressed sequence tags from the fungus <em>Aspergillus oryzae</em> cultured under different conditions. <span><span class="ref-journal">DNA Res. </span>2007;<span class="ref-vol">14</span>(2):47–57. doi: 10.1093/dnares/dsm008.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2779895/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17540709" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fdnares%2Fdsm008" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=DNA+Res&amp;title=Analysis+of+expressed+sequence+tags+from+the+fungus+Aspergillus+oryzae+cultured+under+different+conditions&amp;author=T+Akao&amp;author=M+Sano&amp;author=O+Yamada&amp;author=T+Akeno&amp;author=K+Fujii&amp;volume=14&amp;issue=2&amp;publication_year=2007&amp;pages=47-57&amp;pmid=17540709&amp;doi=10.1093/dnares/dsm008&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR3"><span class="element-citation">Bergmann A, Hartmann T, Cairns T, Bignell EM, Krappmann S. A regulator of <em>Aspergillus fumigatu</em>s extracellular proteolytic activity is dispensable for virulence. <span><span class="ref-journal">Infect Immun. </span>2009;<span class="ref-vol">77</span>(9):4041–4050. doi: 10.1128/IAI.00425-09.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2737998/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19564390" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FIAI.00425-09" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Infect+Immun&amp;title=A+regulator+of+Aspergillus+fumigatus+extracellular+proteolytic+activity+is+dispensable+for+virulence&amp;author=A+Bergmann&amp;author=T+Hartmann&amp;author=T+Cairns&amp;author=EM+Bignell&amp;author=S+Krappmann&amp;volume=77&amp;issue=9&amp;publication_year=2009&amp;pages=4041-4050&amp;pmid=19564390&amp;doi=10.1128/IAI.00425-09&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR4"><span class="mixed-citation">Bradford MM (1976) A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal Biochem 7(72):248&#x02013;254. 10.1006/abio.1976.9999 [<a href="https://pubmed.ncbi.nlm.nih.gov/942051" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></li><li id="CR5"><span class="element-citation">Cathala G, Savouret JF, Mendez B, West BL, Karin M, Martial JA, Baxter JD. A method for isolation of intact, translationally active ribonucleic acid. <span><span class="ref-journal">DNA. </span>1983;<span class="ref-vol">2</span>(4):329–335. doi: 10.1089/dna.1983.2.329.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/6198133" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1089%2Fdna.1983.2.329" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=DNA&amp;title=A+method+for+isolation+of+intact,+translationally+active+ribonucleic+acid&amp;author=G+Cathala&amp;author=JF+Savouret&amp;author=B+Mendez&amp;author=BL+West&amp;author=M+Karin&amp;volume=2&amp;issue=4&amp;publication_year=1983&amp;pages=329-335&amp;pmid=6198133&amp;doi=10.1089/dna.1983.2.329&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR6"><span class="element-citation">Chen L, Zou G, Zhang L, de Vries RP, Yan X, Zhang J, Liu R, Wang C, Qu Y, Zhou Z. The distinctive regulatory roles of PrtT in the cell metabolism of <em>Penicillium oxalicum</em>. <span><span class="ref-journal">Fungal Genet Biol. </span>2014;<span class="ref-vol">63</span>:42–54. doi: 10.1016/j.fgb.2013.12.001.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24333140" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.fgb.2013.12.001" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Fungal+Genet+Biol&amp;title=The+distinctive+regulatory+roles+of+PrtT+in+the+cell+metabolism+of+Penicillium+oxalicum&amp;author=L+Chen&amp;author=G+Zou&amp;author=L+Zhang&amp;author=RP+de+Vries&amp;author=X+Yan&amp;volume=63&amp;publication_year=2014&amp;pages=42-54&amp;pmid=24333140&amp;doi=10.1016/j.fgb.2013.12.001&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR7"><span class="element-citation">Chen J, Li Y, Zhang K, Wang H. Whole-genome sequence of phage-resistant strain Escherichia coli DH5&#x003b1; <span><span class="ref-journal">Genome Announc. </span>2018;<span class="ref-vol">6</span>(10):e00097–e118. doi: 10.1128/genomeA.00097-18.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5843723/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29519826" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FgenomeA.00097-18" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Announc&amp;title=Whole-genome+sequence+of+phage-resistant+strain+Escherichia+coli+DH5&#x003b1;&amp;author=J+Chen&amp;author=Y+Li&amp;author=K+Zhang&amp;author=H+Wang&amp;volume=6&amp;issue=10&amp;publication_year=2018&amp;pages=e00097-e118&amp;pmid=29519826&amp;doi=10.1128/genomeA.00097-18&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR8"><span class="element-citation">Fujioka T, Mizutani O, Furukawa K, Sato N, Yoshimi A, Yamagata Y, Nakajima T, Abe K. MpkA-dependent and -independent cell wall integrity signaling in <em>Aspergillus nidulans</em>. <span><span class="ref-journal">Eukaryot Cell. </span>2007;<span class="ref-vol">6</span>(8):1497–1510. doi: 10.1128/EC.00281-06.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1951132/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17601879" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FEC.00281-06" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eukaryot+Cell&amp;title=MpkA-dependent+and+-independent+cell+wall+integrity+signaling+in+Aspergillus+nidulans&amp;author=T+Fujioka&amp;author=O+Mizutani&amp;author=K+Furukawa&amp;author=N+Sato&amp;author=A+Yoshimi&amp;volume=6&amp;issue=8&amp;publication_year=2007&amp;pages=1497-1510&amp;pmid=17601879&amp;doi=10.1128/EC.00281-06&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR9"><span class="element-citation">Galagan JE, Calvo SE, Cuomo C, Ma LJ, Wortman JR, Batzoglou S, Lee SI, Ba&#x0015f;t&#x000fc;rkmen M, Spevak CC, Clutterbuck J, Kapitonov V, Jurka J, Scazzocchio C, Farman M, Butler J, Purcell S, Harris S, Braus GH, Draht O, Busch S, D&#x02019;Enfert C, Bouchier C, Goldman GH, Bell-Pedersen D, Griffiths-Jones S, Doonan JH, Yu J, Vienken K, Pain A, Freitag M, Selker EU, Archer DB, Pe&#x000f1;alva MA, Oakley BR, Momany M, Tanaka T, Kumagai T, Asai K, Machida M, Nierman WC, Denning DW, Caddick M, Hynes M, Paoletti M, Fischer R, Miller B, Dyer P, Sachs MS, Osmani SA, Birren BW. Sequencing of Aspergillus nidulans and comparative analysis with<em> A. fumigatus
</em> and<em> A. oryzae</em>. <span><span class="ref-journal">Nature. </span>2005;<span class="ref-vol">438</span>(7071):1105–1115. doi: 10.1038/nature04341.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16372000" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnature04341" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Sequencing+of+Aspergillus+nidulans+and+comparative+analysis+with+A.+fumigatus&#10;+and+A.+oryzae&amp;author=JE+Galagan&amp;author=SE+Calvo&amp;author=C+Cuomo&amp;author=LJ+Ma&amp;author=JR+Wortman&amp;volume=438&amp;issue=7071&amp;publication_year=2005&amp;pages=1105-1115&amp;pmid=16372000&amp;doi=10.1038/nature04341&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR10"><span class="element-citation">Gomi K. Regulatory mechanisms for amylolytic gene expression in the koji mold <em>Aspergillus oryzae</em>. <span><span class="ref-journal">Biosci Biotechnol Biochem. </span>2019;<span class="ref-vol">83</span>(8):1385–1401. doi: 10.1080/09168451.2019.1625265.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31159661" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1080%2F09168451.2019.1625265" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biosci+Biotechnol+Biochem&amp;title=Regulatory+mechanisms+for+amylolytic+gene+expression+in+the+koji+mold+Aspergillus+oryzae&amp;author=K+Gomi&amp;volume=83&amp;issue=8&amp;publication_year=2019&amp;pages=1385-1401&amp;pmid=31159661&amp;doi=10.1080/09168451.2019.1625265&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR11"><span class="element-citation">Gomi K, Iimura Y, Hara S. Integrative transformation of <em>Aspergillus oryzae</em> with a plasmid containing the <em>Aspergillus nidulans argB</em> gene. <span><span class="ref-journal">Agric Biol Chem. </span>1987;<span class="ref-vol">51</span>(9):2549–2555. doi: 10.1080/00021369.1987.10868429.</span> [<a href="//doi.org/10.1080%2F00021369.1987.10868429" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Agric+Biol+Chem&amp;title=Integrative+transformation+of+Aspergillus+oryzae+with+a+plasmid+containing+the+Aspergillus+nidulans+argB+gene&amp;author=K+Gomi&amp;author=Y+Iimura&amp;author=S+Hara&amp;volume=51&amp;issue=9&amp;publication_year=1987&amp;pages=2549-2555&amp;doi=10.1080/00021369.1987.10868429&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR12"><span class="element-citation">Hoebeke J, Busatto-Samsoen C, Davoust D, Lebrun E. <sup>1</sup>H NMR study of the diastereomeric forms of the protease inhibitor antipain. <span><span class="ref-journal">Magn Res Chem. </span>1994;<span class="ref-vol">32</span>(4):220–224. doi: 10.1002/mrc.1260320406.</span> [<a href="//doi.org/10.1002%2Fmrc.1260320406" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Magn+Res+Chem&amp;title=1H+NMR+study+of+the+diastereomeric+forms+of+the+protease+inhibitor+antipain&amp;author=J+Hoebeke&amp;author=C+Busatto-Samsoen&amp;author=D+Davoust&amp;author=E+Lebrun&amp;volume=32&amp;issue=4&amp;publication_year=1994&amp;pages=220-224&amp;doi=10.1002/mrc.1260320406&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR13"><span class="element-citation">Huang L, Dong L, Wang B, Pan L. The transcription factor PrtT and its target protease profiles in <em>Aspergillus niger</em> are negatively regulated by carbon sources. <span><span class="ref-journal">Biotechnol Lett. </span>2020;<span class="ref-vol">42</span>(4):613–624. doi: 10.1007/s10529-020-02806-3.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31970554" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs10529-020-02806-3" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnol+Lett&amp;title=The+transcription+factor+PrtT+and+its+target+protease+profiles+in+Aspergillus+niger+are+negatively+regulated+by+carbon+sources&amp;author=L+Huang&amp;author=L+Dong&amp;author=B+Wang&amp;author=L+Pan&amp;volume=42&amp;issue=4&amp;publication_year=2020&amp;pages=613-624&amp;pmid=31970554&amp;doi=10.1007/s10529-020-02806-3&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR14"><span class="element-citation">Huynh HH, Morita N, Sakamoto T, Katayama T, Miyakawa T, Tanokura M, Chiba Y, Shinkura R, Maruyama J. Functional production of human antibody by the filamentous fungus <em>Aspergillus oryzae</em>. <span><span class="ref-journal">Fungal Biol Biotechnol. </span>2020;<span class="ref-vol">7</span>:7. doi: 10.1186/s40694-020-00098-w.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7257131/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32514366" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2Fs40694-020-00098-w" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Fungal+Biol+Biotechnol&amp;title=Functional+production+of+human+antibody+by+the+filamentous+fungus+Aspergillus+oryzae&amp;author=HH+Huynh&amp;author=N+Morita&amp;author=T+Sakamoto&amp;author=T+Katayama&amp;author=T+Miyakawa&amp;volume=7&amp;publication_year=2020&amp;pages=7&amp;pmid=32514366&amp;doi=10.1186/s40694-020-00098-w&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR15"><span class="element-citation">Kitamoto K.  Advances in Applied Microbiology, Volume 51. In: Laskin A, Bennett J, Gadd G, editors. <span class="ref-journal">Molecular biology of the koji molds.</span> USA: California; 2002. pp. 129–153. [<a href="https://pubmed.ncbi.nlm.nih.gov/12236056" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Molecular+biology+of+the+koji+molds&amp;author=K+Kitamoto&amp;publication_year=2002&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR16"><span class="element-citation">Kitano H, Kataoka K, Furukawa K, Hara S. Specific expression and temperature-dependent expression of the acid protease-encoding gene (<em>pepA</em>) in <em>Aspergilhs oryzae</em> in solid-state culture (rice-koji) <span><span class="ref-journal">J Biosci Bioeng. </span>2002;<span class="ref-vol">93</span>(6):563–567. doi: 10.1016/s1389-1723(02)80238-9.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16233250" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fs1389-1723(02)80238-9" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biosci+Bioeng&amp;title=Specific+expression+and+temperature-dependent+expression+of+the+acid+protease-encoding+gene+(pepA)+in+Aspergilhs+oryzae+in+solid-state+culture+(rice-koji)&amp;author=H+Kitano&amp;author=K+Kataoka&amp;author=K+Furukawa&amp;author=S+Hara&amp;volume=93&amp;issue=6&amp;publication_year=2002&amp;pages=563-567&amp;pmid=16233250&amp;doi=10.1016/s1389-1723(02)80238-9&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR17"><span class="element-citation">Kobayashi T, Maeda H, Takeuchi M, Yamagata Y. Deletion of <em>admB</em> gene encoding a fungal ADAM affects cell wall construction in <em>Aspergillus oryzae</em>. <span><span class="ref-journal">Biosci Biotechnol Biochem. </span>2017;<span class="ref-vol">81</span>(5):1041–1050. doi: 10.1080/09168451.2016.1270741.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28058997" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1080%2F09168451.2016.1270741" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biosci+Biotechnol+Biochem&amp;title=Deletion+of+admB+gene+encoding+a+fungal+ADAM+affects+cell+wall+construction+in+Aspergillus+oryzae&amp;author=T+Kobayashi&amp;author=H+Maeda&amp;author=M+Takeuchi&amp;author=Y+Yamagata&amp;volume=81&amp;issue=5&amp;publication_year=2017&amp;pages=1041-1050&amp;pmid=28058997&amp;doi=10.1080/09168451.2016.1270741&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR18"><span class="element-citation">Lee BR, Furukawa M, Yamashita K, Kanasugi Y, Kawabata C, Hirano K, Ando K, Ichishima E. Aorsin, a novel serine proteinase with trypsin-like specificity at acidic pH. <span><span class="ref-journal">Biochem J. </span>2003;<span class="ref-vol">371</span>:541–548. doi: 10.1042/BJ20021691.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1223285/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12519073" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1042%2FBJ20021691" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+J&amp;title=Aorsin,+a+novel+serine+proteinase+with+trypsin-like+specificity+at+acidic+pH&amp;author=BR+Lee&amp;author=M+Furukawa&amp;author=K+Yamashita&amp;author=Y+Kanasugi&amp;author=C+Kawabata&amp;volume=371&amp;publication_year=2003&amp;pages=541-548&amp;pmid=12519073&amp;doi=10.1042/BJ20021691&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR19"><span class="element-citation">Machida M, Asai K, Sano M, Tanaka T, Kumagai T, Terai G, Kusumoto K, Arima T, Akita O, Kashiwagi Y, Abe K, Gomi K, Horiuchi H, Kitamoto K, Kobayashi T, Takeuchi M, Denning DW, Galagan JE, Nierman WC, Yu J, Archer DB, Bennett JW, Bhatnagar D, Cleveland TE, Fedorova ND, Gotoh O, Horikawa H, Hosoyama A, Ichinomiya M, Igarashi R, Iwashita K, Juvvadi PR, Kato M, Kato Y, Kin T, Kokubun A, Maeda H, Maeyama N, Maruyama J, Nagasaki H, Nakajima T, Oda K, Okada K, Paulsen I, Sakamoto K, Sawano T, Takahashi M, Takase K, Terabayashi Y, Wortman JR, Yamada O, Yamagata Y, Anazawa H, Hata Y, Koide Y, Komori T, Koyama Y, Minetoki T, Suharnan S, Tanaka A, Isono K, Kuhara S, Ogasawara N, Kikuchi H. Genome sequencing and analysis of <em>Aspergillus oryzae</em>. <span><span class="ref-journal">Nature. </span>2005;<span class="ref-vol">438</span>(7071):1157–1161. doi: 10.1038/nature04300.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16372010" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnature04300" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome+sequencing+and+analysis+of+Aspergillus+oryzae&amp;author=M+Machida&amp;author=K+Asai&amp;author=M+Sano&amp;author=T+Tanaka&amp;author=T+Kumagai&amp;volume=438&amp;issue=7071&amp;publication_year=2005&amp;pages=1157-1161&amp;pmid=16372010&amp;doi=10.1038/nature04300&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR20"><span class="element-citation">Machida M, Yamada O, Gomi K. Genomics of <em>Aspergillus oryzae</em>: learning from the history of koji mold and exploration of its future. <span><span class="ref-journal">DNA Res. </span>2008;<span class="ref-vol">15</span>(4):173–183. doi: 10.1093/dnares/dsn020.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2575883/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18820080" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fdnares%2Fdsn020" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=DNA+Res&amp;title=Genomics+of+Aspergillus+oryzae:+learning+from+the+history+of+koji+mold+and+exploration+of+its+future&amp;author=M+Machida&amp;author=O+Yamada&amp;author=K+Gomi&amp;volume=15&amp;issue=4&amp;publication_year=2008&amp;pages=173-183&amp;pmid=18820080&amp;doi=10.1093/dnares/dsn020&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR21"><span class="element-citation">Maeda H, Sano M, Maruyama Y, Tanno T, Akao T, Totsuka Y, Endo M, Sakurada R, Yamagata Y, Machida M, Akita O, Hasegawa F, Abe K, Gomi K, Nakajima T, Iguchi Y. Transcriptional analysis of gene for energy catabolism and hydrolytic enzymes in the filamentous fungus <em>Aspergillus oryzae</em> using cDNA microarrays and expressed sequence tags. <span><span class="ref-journal">Appl Microbiol Biotechnol. </span>2004;<span class="ref-vol">65</span>(1):74–83. doi: 10.1007/s00253-004-1608-4.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15221230" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs00253-004-1608-4" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Appl+Microbiol+Biotechnol&amp;title=Transcriptional+analysis+of+gene+for+energy+catabolism+and+hydrolytic+enzymes+in+the+filamentous+fungus+Aspergillus+oryzae+using+cDNA+microarrays+and+expressed+sequence+tags&amp;author=H+Maeda&amp;author=M+Sano&amp;author=Y+Maruyama&amp;author=T+Tanno&amp;author=T+Akao&amp;volume=65&amp;issue=1&amp;publication_year=2004&amp;pages=74-83&amp;pmid=15221230&amp;doi=10.1007/s00253-004-1608-4&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR22"><span class="element-citation">Morita H, Okamoto A, Yamagata Y, Kusumoto K, Koide Y, Ishida H, Takeuchi M. Heterologous expression and characterization of CpI, OcpA, and novel serine-type carboxypeptidase OcpB from <em>Aspergillus oryzae</em>. <span><span class="ref-journal">Appl Microbiol Biotechnol. </span>2009;<span class="ref-vol">85</span>(2):335–346. doi: 10.1007/s00253-009-2087-4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2773364/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19557408" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs00253-009-2087-4" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Appl+Microbiol+Biotechnol&amp;title=Heterologous+expression+and+characterization+of+CpI,+OcpA,+and+novel+serine-type+carboxypeptidase+OcpB+from+Aspergillus+oryzae&amp;author=H+Morita&amp;author=A+Okamoto&amp;author=Y+Yamagata&amp;author=K+Kusumoto&amp;author=Y+Koide&amp;volume=85&amp;issue=2&amp;publication_year=2009&amp;pages=335-346&amp;pmid=19557408&amp;doi=10.1007/s00253-009-2087-4&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR23"><span class="element-citation">Morita H, Kuriyama K, Akiyama N, Okamoto A, Yamagata Y, Kusumoto K, Koide Y, Ishida H, Takeuchi M. Molecular cloning of <em>ocpO</em> encoding carboxypeptidase O of <em>Aspergillus oryzae</em> IAM2640. <span><span class="ref-journal">Biosci Biotechnol Biochem. </span>2011;<span class="ref-vol">75</span>(4):662–668. doi: 10.1271/bbb.90863.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20460731" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1271%2Fbbb.90863" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biosci+Biotechnol+Biochem&amp;title=Molecular+cloning+of+ocpO+encoding+carboxypeptidase+O+of+Aspergillus+oryzae+IAM2640&amp;author=H+Morita&amp;author=K+Kuriyama&amp;author=N+Akiyama&amp;author=A+Okamoto&amp;author=Y+Yamagata&amp;volume=75&amp;issue=4&amp;publication_year=2011&amp;pages=662-668&amp;pmid=21512241&amp;doi=10.1271/bbb.90863&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR24"><span class="mixed-citation">Murakami E (1986) Ko-ji gaku. Brewering society Japan, Tokyo</span></li><li id="CR25"><span class="element-citation">Nakadai T, Nasuno S, Iguchi N. Purification and properties of alkaline proteinase from <em>Aspergillus oryzae</em>. <span><span class="ref-journal">Agric Biol Chem. </span>1973;<span class="ref-vol">37</span>(12):2685–2694. doi: 10.1080/00021369.1973.10861071.</span> [<a href="//doi.org/10.1080%2F00021369.1973.10861071" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Agric+Biol+Chem&amp;title=Purification+and+properties+of+alkaline+proteinase+from+Aspergillus+oryzae&amp;author=T+Nakadai&amp;author=S+Nasuno&amp;author=N+Iguchi&amp;volume=37&amp;issue=12&amp;publication_year=1973&amp;pages=2685-2694&amp;doi=10.1080/00021369.1973.10861071&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR26"><span class="element-citation">Nierman WC, Pain A, Anderson MJ, Wortman JR, Kim HS, Arroyo J, Berriman M, Abe K, Archer DB, Bermejo C, Bennett J, Bowyer P, Chen D, Collins M, Coulsen R, Davies R, Dyer PS, Farman M, Fedorova N, Fedorova N, Feldblyum TV, Fischer R, Fosker N, Fraser A, Garc&#x000ed;a JL, Garc&#x000ed;a MJ, Goble A, Goldman GH, Gomi K, Griffith-Jones S, Gwilliam R, Haas B, Haas H, Harris D, Horiuchi H, Huang J, Humphray S, Jim&#x000e9;nez J, Keller N, Khouri H, Kitamoto K, Kobayashi T, Konzack S, Kulkarni R, Kumagai T, Lafon A, Latg&#x000e9; JP, Li W, Lord A, Lu C, Majoros WH, May GS, Miller BL, Mohamoud Y, Molina M, Monod M, Mouyna I, Mulligan S, Murphy L, O&#x02019;Neil S, Paulsen I, Pe&#x000f1;alva MA, Pertea M, Price C, Pritchard BL, Quail MA, Rabbinowitsch E, Rawlins N, Rajandream MA, Reichard U, Renauld H, Robson GD, Rodriguez de C&#x000f3;rdoba S, Rodr&#x000ed;guez-Pe&#x000f1;a JM, Ronning CM, Rutter S, Salzberg SL, Sanchez M, S&#x000e1;nchez-Ferrero JC, Saunders D, Seeger K, Squares R, Squares S, Takeuchi M, Tekaia F, Turner G, Vazquez de Aldana CR, Weidman J, White O, Woodward J, Yu JH, Fraser C, Galagan JE, Asai K, Machida M, Hall N, Barrell B, Denning DW. Genomic sequence of the pathogenic and allergenic filamentous fungus <em>Aspergillus fumigatus</em>. <span><span class="ref-journal">Nature. </span>2005;<span class="ref-vol">438</span>(7071):115–1156. doi: 10.1038/nature04332.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16372009" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnature04332" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genomic+sequence+of+the+pathogenic+and+allergenic+filamentous+fungus+Aspergillus+fumigatus&amp;author=WC+Nierman&amp;author=A+Pain&amp;author=MJ+Anderson&amp;author=JR+Wortman&amp;author=HS+Kim&amp;volume=438&amp;issue=7071&amp;publication_year=2005&amp;pages=115-1156&amp;doi=10.1038/nature04332&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR27"><span class="element-citation">Papagianni M. Fungal morphology and metabolite production in submerged mycelial processes. <span><span class="ref-journal">Biotechnol Adv. </span>2004;<span class="ref-vol">22</span>(3):189–259. doi: 10.1016/j.biotechadv.2003.09.005.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14665401" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.biotechadv.2003.09.005" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnol+Adv&amp;title=Fungal+morphology+and+metabolite+production+in+submerged+mycelial+processes&amp;author=M+Papagianni&amp;volume=22&amp;issue=3&amp;publication_year=2004&amp;pages=189-259&amp;pmid=14665401&amp;doi=10.1016/j.biotechadv.2003.09.005&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR28"><span class="element-citation">Punt PJ, Schuren FH, Lehmbeck J, Christensen T, Hjort C, van den Hondel CA. Characterization of the <em>Aspergillus niger prtT</em>, a unique regulator of extracellular protease encoding genes. <span><span class="ref-journal">Fungal Genet Biol. </span>2008;<span class="ref-vol">45</span>(12):1591–1599. doi: 10.1016/j.fgb.2008.09.007.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18930158" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.fgb.2008.09.007" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Fungal+Genet+Biol&amp;title=Characterization+of+the+Aspergillus+niger+prtT,+a+unique+regulator+of+extracellular+protease+encoding+genes&amp;author=PJ+Punt&amp;author=FH+Schuren&amp;author=J+Lehmbeck&amp;author=T+Christensen&amp;author=C+Hjort&amp;volume=45&amp;issue=12&amp;publication_year=2008&amp;pages=1591-1599&amp;pmid=18930158&amp;doi=10.1016/j.fgb.2008.09.007&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR29"><span class="element-citation">Purushothaman K, Bhat SK, Singh SA, Marathe GK, Rao ARGA. Aspartic protease from <em>Aspergillus </em><em>niger</em>: molecular characterization and interaction with pepstatin A. <span><span class="ref-journal">Int J Biol Macromol. </span>2019;<span class="ref-vol">139</span>:199–212. doi: 10.1016/j.ijbiomac.2019.07.133.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31374272" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.ijbiomac.2019.07.133" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Biol+Macromol&amp;title=Aspartic+protease+from+Aspergillus+niger:+molecular+characterization+and+interaction+with+pepstatin+A&amp;author=K+Purushothaman&amp;author=SK+Bhat&amp;author=SA+Singh&amp;author=GK+Marathe&amp;author=ARGA+Rao&amp;volume=139&amp;publication_year=2019&amp;pages=199-212&amp;pmid=31374272&amp;doi=10.1016/j.ijbiomac.2019.07.133&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR30"><span class="element-citation">Sharon H, Hagag S, Osherov N. Transcription factor PrtT controls expression of multiple secreted proteases in the human pathogenic mold <em>Aspergillus fumigatus</em>. <span><span class="ref-journal">Infect Immun. </span>2009;<span class="ref-vol">77</span>(9):4051–4060. doi: 10.1128/IAI.00426-09.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2738002/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19564385" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FIAI.00426-09" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Infect+Immun&amp;title=Transcription+factor+PrtT+controls+expression+of+multiple+secreted+proteases+in+the+human+pathogenic+mold+Aspergillus+fumigatus&amp;author=H+Sharon&amp;author=S+Hagag&amp;author=N+Osherov&amp;volume=77&amp;issue=9&amp;publication_year=2009&amp;pages=4051-4060&amp;pmid=19564385&amp;doi=10.1128/IAI.00426-09&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR31"><span class="element-citation">Takahashi T, Ogawa M, Sato A, Koyama Y. Translocated duplication of a targeted chromosomal segment enhances gene expression at the duplicated site and results in phenotypic changes in <em>Aspergillus oryzae</em>. <span><span class="ref-journal">Fungal Biol Biotechnol. </span>2018;<span class="ref-vol">5</span>:1–17. doi: 10.1186/s40694-018-0061-6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6171174/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30305913" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2Fs40694-018-0061-6" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Fungal+Biol+Biotechnol&amp;title=Translocated+duplication+of+a+targeted+chromosomal+segment+enhances+gene+expression+at+the+duplicated+site+and+results+in+phenotypic+changes+in+Aspergillus+oryzae&amp;author=T+Takahashi&amp;author=M+Ogawa&amp;author=A+Sato&amp;author=Y+Koyama&amp;volume=5&amp;publication_year=2018&amp;pages=1-17&amp;pmid=29372063&amp;doi=10.1186/s40694-018-0061-6&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR32"><span class="element-citation">Takeuchi M, Ushijima T, Ichishima E. A new acid carboxypeptidase, O-1, from <em>Aspergillus oryzae</em>. <span><span class="ref-journal">Curr Microbiol. </span>1982;<span class="ref-vol">7</span>:19–23. doi: 10.1007/BF01570974.</span> [<a href="//doi.org/10.1007%2FBF01570974" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Microbiol&amp;title=A+new+acid+carboxypeptidase,+O-1,+from+Aspergillus+oryzae&amp;author=M+Takeuchi&amp;author=T+Ushijima&amp;author=E+Ichishima&amp;volume=7&amp;publication_year=1982&amp;pages=19-23&amp;doi=10.1007/BF01570974&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR33"><span class="element-citation">Takeuchi M, Ogura K, Hamamoto T, Kobayashi Y. Molecular cloning and sequence analysis of a gene encoding an aspartic proteinase from <em>Aspergillus oryzae</em>. <span><span class="ref-journal">Adv Exp Med Biol. </span>1995;<span class="ref-vol">362</span>:577–580. doi: 10.1007/978-1-4615-1871-6_77.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8540376" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2F978-1-4615-1871-6_77" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Adv+Exp+Med+Biol&amp;title=Molecular+cloning+and+sequence+analysis+of+a+gene+encoding+an+aspartic+proteinase+from+Aspergillus+oryzae&amp;author=M+Takeuchi&amp;author=K+Ogura&amp;author=T+Hamamoto&amp;author=Y+Kobayashi&amp;volume=362&amp;publication_year=1995&amp;pages=577-580&amp;pmid=8540376&amp;doi=10.1007/978-1-4615-1871-6_77&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR34"><span class="element-citation">Takyu Y, Asamura T, Okamoto A, Maeda H, Takeuchi M, Kusumoto K, Katase T, Ishida H, Tanaka M, Yamagata Y. A novel milk-clotting enzyme from <em>Aspergillus oryzae</em> and A luchuensis is an aspartic endopeptidase PepE presumed to be a vacuolar enzyme. <span><span class="ref-journal">Biosci Biotechnol Biochem. </span>2022;<span class="ref-vol">86</span>(3):413–422. doi: 10.1093/bbb/zbac005.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35025981" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbbb%2Fzbac005" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biosci+Biotechnol+Biochem&amp;title=A+novel+milk-clotting+enzyme+from+Aspergillus+oryzae+and+A+luchuensis+is+an+aspartic+endopeptidase+PepE+presumed+to+be+a+vacuolar+enzyme&amp;author=Y+Takyu&amp;author=T+Asamura&amp;author=A+Okamoto&amp;author=H+Maeda&amp;author=M+Takeuchi&amp;volume=86&amp;issue=3&amp;publication_year=2022&amp;pages=413-422&amp;pmid=35025981&amp;doi=10.1093/bbb/zbac005&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR35"><span class="element-citation">Tanaka N, Takeuchi M, Ichishima E. Purification and mode of assay for acid proteinase of <em>Aspergillus saitoi</em> and determination of peptide bond specificity. <span><span class="ref-journal">Biochim Biophys Acta. </span>1970;<span class="ref-vol">485</span>(2):406–416. doi: 10.1016/0005-2744(77)90176-0.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21699" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2F0005-2744(77)90176-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochim+Biophys+Acta&amp;title=Purification+and+mode+of+assay+for+acid+proteinase+of+Aspergillus+saitoi+and+determination+of+peptide+bond+specificity&amp;author=N+Tanaka&amp;author=M+Takeuchi&amp;author=E+Ichishima&amp;volume=485&amp;issue=2&amp;publication_year=1970&amp;pages=406-416&amp;doi=10.1016/0005-2744(77)90176-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR36"><span class="element-citation">Tanaka M, Yoshimura M, Ogawa M, Koyama Y, Shintani T, Gomi K. The C<sub>2</sub>H<sub>2</sub>-type transcription factor, FlbC, is involved in the transcriptional regulation of <em>Aspergillus oryzae</em> glucoamylase and protease genes specifically expressed in solid-state culture. <span><span class="ref-journal">Appl Microbiol Biotechnol. </span>2016;<span class="ref-vol">100</span>(13):5859–5868. doi: 10.1007/s00253-016-7419-6.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26960315" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs00253-016-7419-6" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Appl+Microbiol+Biotechnol&amp;title=The+C2H2-type+transcription+factor,+FlbC,+is+involved+in+the+transcriptional+regulation+of+Aspergillus+oryzae+glucoamylase+and+protease+genes+specifically+expressed+in+solid-state+culture&amp;author=M+Tanaka&amp;author=M+Yoshimura&amp;author=M+Ogawa&amp;author=Y+Koyama&amp;author=T+Shintani&amp;volume=100&amp;issue=13&amp;publication_year=2016&amp;pages=5859-5868&amp;pmid=26960315&amp;doi=10.1007/s00253-016-7419-6&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR37"><span class="element-citation">Tanaka M, Ito K, Matsuura T, Kawarasaki Y, Gomi K. Identification and distinct regulation of three di/tripeptide transporters in <em>Aspergillus oryzae</em>. <span><span class="ref-journal">Biosci Biotechnol Biochem. </span>2021;<span class="ref-vol">85</span>(2):452–463. doi: 10.1093/bbb/zbaa030.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33604648" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbbb%2Fzbaa030" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biosci+Biotechnol+Biochem&amp;title=Identification+and+distinct+regulation+of+three+di/tripeptide+transporters+in+Aspergillus+oryzae&amp;author=M+Tanaka&amp;author=K+Ito&amp;author=T+Matsuura&amp;author=Y+Kawarasaki&amp;author=K+Gomi&amp;volume=85&amp;issue=2&amp;publication_year=2021&amp;pages=452-463&amp;pmid=33604648&amp;doi=10.1093/bbb/zbaa030&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR38"><span class="element-citation">Tatsumi H, Murakamil S, Tsuji RF, Ishida Y, Murakami K, Masaki A, Kawabe H, Arimura H, Nakano E, Motai H. Cloning and expression in yeast of a cDNA clone encoding <em>Aspergillus oryzae</em> neutral protease II, a unique metalloprotease. <span><span class="ref-journal">Mol Gen Genet. </span>1991;<span class="ref-vol">228</span>(1&#x02013;2):97–103. doi: 10.1007/BF00282453.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/1886621" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2FBF00282453" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Gen+Genet&amp;title=Cloning+and+expression+in+yeast+of+a+cDNA+clone+encoding+Aspergillus+oryzae+neutral+protease+II,+a+unique+metalloprotease&amp;author=H+Tatsumi&amp;author=S+Murakamil&amp;author=RF+Tsuji&amp;author=Y+Ishida&amp;author=K+Murakami&amp;volume=228&amp;issue=1&#x02013;2&amp;publication_year=1991&amp;pages=97-103&amp;pmid=1886621&amp;doi=10.1007/BF00282453&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR39"><span class="element-citation">Taylor MJ, Richardson T. Applications of microbial enzymes in food systems and in biotechnology. <span><span class="ref-journal">Adv Appl Microbiol. </span>1979;<span class="ref-vol">25</span>:7–35. doi: 10.1016/s0065-2164(08)70144-8.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/397740" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fs0065-2164(08)70144-8" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Adv+Appl+Microbiol&amp;title=Applications+of+microbial+enzymes+in+food+systems+and+in+biotechnology&amp;author=MJ+Taylor&amp;author=T+Richardson&amp;volume=25&amp;publication_year=1979&amp;pages=7-35&amp;pmid=397740&amp;doi=10.1016/s0065-2164(08)70144-8&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR40"><span class="element-citation">Tilburn J, Sarkar S, Widdick DA, Espeso EA, Orejas M, Mungroo J, Pe&#x000f1;alva MA, Arst HN., Jr The <em>Aspergillus</em> PacC zinc finger transcription factor mediates regulation of both acid-and alkaline-expressed genes by ambient pH. <span><span class="ref-journal">EMBO J. </span>1995;<span class="ref-vol">14</span>(4):779–790. doi: 10.1002/j.1460-2075.1995.tb07056.x.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC398143/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7882981" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fj.1460-2075.1995.tb07056.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+J&amp;title=The+Aspergillus+PacC+zinc+finger+transcription+factor+mediates+regulation+of+both+acid-and+alkaline-expressed+genes+by+ambient+pH&amp;author=J+Tilburn&amp;author=S+Sarkar&amp;author=DA+Widdick&amp;author=EA+Espeso&amp;author=M+Orejas&amp;volume=14&amp;issue=4&amp;publication_year=1995&amp;pages=779-790&amp;pmid=7882981&amp;doi=10.1002/j.1460-2075.1995.tb07056.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR41"><span class="element-citation">Wang B, Guo G, Wang C, Lin Y, Wang X, Zhao M, Guo Y, He M, Zhang Y, Pan L. Survey of the transcriptome of <em>Aspergillus oryzae</em> via massively parallel mRNA sequencing. <span><span class="ref-journal">Nucleic Acids Res. </span>2010;<span class="ref-vol">38</span>(15):5075–5087. doi: 10.1093/nar/gkq256.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2926611/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20392818" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkq256" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Survey+of+the+transcriptome+of+Aspergillus+oryzae+via+massively+parallel+mRNA+sequencing&amp;author=B+Wang&amp;author=G+Guo&amp;author=C+Wang&amp;author=Y+Lin&amp;author=X+Wang&amp;volume=38&amp;issue=15&amp;publication_year=2010&amp;pages=5075-5087&amp;pmid=20392818&amp;doi=10.1093/nar/gkq256&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR42"><span class="element-citation">Yoon J, Maruyama J, Kitamoto K. Disruption of ten protease genes in the filamentous fungus <em>Aspergillus oryzae</em> highly improves production of heterologous proteins. <span><span class="ref-journal">Appl Microbiol Biotechnol. </span>2011;<span class="ref-vol">89</span>(3):747–759. doi: 10.1007/s00253-010-2937-0.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20957357" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs00253-010-2937-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Appl+Microbiol+Biotechnol&amp;title=Disruption+of+ten+protease+genes+in+the+filamentous+fungus+Aspergillus+oryzae+highly+improves+production+of+heterologous+proteins&amp;author=J+Yoon&amp;author=J+Maruyama&amp;author=K+Kitamoto&amp;volume=89&amp;issue=3&amp;publication_year=2011&amp;pages=747-759&amp;pmid=20957357&amp;doi=10.1007/s00253-010-2937-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR43"><span class="element-citation">Zanutto-Elgui MR, Vieira JCS, Prado DZD, Buzalaf MAR, Padilha PM, Elgui de Oliveira D, Fleuri LF. Production of milk peptides with antimicrobial and antioxidant properties through fungal proteases. <span><span class="ref-journal">Food Chem. </span>2019;<span class="ref-vol">278</span>:823–831. doi: 10.1016/j.foodchem.2018.11.119.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30583449" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.foodchem.2018.11.119" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Food+Chem&amp;title=Production+of+milk+peptides+with+antimicrobial+and+antioxidant+properties+through+fungal+proteases&amp;author=MR+Zanutto-Elgui&amp;author=JCS+Vieira&amp;author=DZD+Prado&amp;author=MAR+Buzalaf&amp;author=PM+Padilha&amp;volume=278&amp;publication_year=2019&amp;pages=823-831&amp;pmid=30583449&amp;doi=10.1016/j.foodchem.2018.11.119&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR44"><span class="element-citation">Zhang S, Ban A, Ebara N, Mizutani O, Tanaka M, Shintani T, Gomi K. Self-excising Cre/mutant <em>lox</em> marker recycling system for multiple gene integrations and consecutive gene deletions in <em>Aspergillus oryzae</em>. <span><span class="ref-journal">J Biosci Bioeng. </span>2017;<span class="ref-vol">123</span>(4):403–411. doi: 10.1016/j.jbiosc.2016.11.001.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28011085" ref="reftype=pubmed&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.jbiosc.2016.11.001" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biosci+Bioeng&amp;title=Self-excising+Cre/mutant+lox+marker+recycling+system+for+multiple+gene+integrations+and+consecutive+gene+deletions+in+Aspergillus+oryzae&amp;author=S+Zhang&amp;author=A+Ban&amp;author=N+Ebara&amp;author=O+Mizutani&amp;author=M+Tanaka&amp;volume=123&amp;issue=4&amp;publication_year=2017&amp;pages=403-411&amp;pmid=28011085&amp;doi=10.1016/j.jbiosc.2016.11.001&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10781853&amp;issue-id=453143&amp;journal-id=365&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li></ul></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Applied Microbiology and Biotechnology</span> are provided here courtesy of <strong>Springer</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC10781853/pdf/253_2023_Article_12833.pdf" class="int-view">PDF (2.1M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/10781853/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/10781853/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="10781853" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10781853%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="KG1cR1ZY9pg6r3Bu0ACAxsx4RqbKnXxNw70tfCwWANbCoAgK4IR2mrrQXctVGGba">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10781853%2F&amp;text=Aspergillus%20oryzae%20PrtR%20alters%20transcription%20of%20individual%20peptidase%20genes%20in%20response%20to%20the%20growth%20environment%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10781853%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10781853/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC10781853/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38204127/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC10781853/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38204127/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC10781853/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/10781853/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript" src="/core/mathjax/2.7.9/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC10781853/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
