
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Identification of Mycobacterium tuberculosis transcriptional repressor EthR inhibitors: Shape-based search and machine learning studies - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="939BE807DB0F4BC500003376DFD1D51E.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="heliyon" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC10907797/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10907797/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Heliyon" /><meta name="citation_title" content="Identification of Mycobacterium tuberculosis transcriptional repressor EthR inhibitors: Shape-based search and machine learning studies" /><meta name="citation_author" content="Rupesh V. Chikhale" /><meta name="citation_author_institution" content="Department of Pharmaceutical and Biological Chemistry, School of Pharmacy, University College London, London, UK" /><meta name="citation_author" content="Gaber E. Eldesoky" /><meta name="citation_author_institution" content="Chemistry Department, College of Science, King Saud University, Riyadh, 11451, Saudi Arabia" /><meta name="citation_author" content="Mahima Sudhir Kolpe" /><meta name="citation_author_institution" content="SilicoScientia Private Limited, Nagananda Commercial Complex, No. 07/3, 15/1, 18th Main Road, Jayanagar 9th Block, Bengaluru, 560041, India" /><meta name="citation_author" content="Vikramsinh Sardarsinh Suryawanshi" /><meta name="citation_author_institution" content="SilicoScientia Private Limited, Nagananda Commercial Complex, No. 07/3, 15/1, 18th Main Road, Jayanagar 9th Block, Bengaluru, 560041, India" /><meta name="citation_author" content="Pritee Chunarkar Patil" /><meta name="citation_author_institution" content="Department of Bioinformatics, Rajiv Gandhi Institute of IT and Biotechnology, Bharati Vidyapeeth Deemed to be University, Pune-Satara Road, Pune, India" /><meta name="citation_author" content="Shovonlal Bhowmick" /><meta name="citation_author_institution" content="SilicoScientia Private Limited, Nagananda Commercial Complex, No. 07/3, 15/1, 18th Main Road, Jayanagar 9th Block, Bengaluru, 560041, India" /><meta name="citation_publication_date" content="2024/03/03" /><meta name="citation_issue" content="5" /><meta name="citation_volume" content="10" /><meta name="citation_doi" content="10.1016/j.heliyon.2024.e26802" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC10907797/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC10907797/" /><meta name="citation_pmid" content="38434349" /><meta name="DC.Title" content="Identification of Mycobacterium tuberculosis transcriptional repressor EthR inhibitors: Shape-based search and machine learning studies" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Elsevier" /><meta name="DC.Contributor" content="Rupesh V. Chikhale" /><meta name="DC.Contributor" content="Gaber E. Eldesoky" /><meta name="DC.Contributor" content="Mahima Sudhir Kolpe" /><meta name="DC.Contributor" content="Vikramsinh Sardarsinh Suryawanshi" /><meta name="DC.Contributor" content="Pritee Chunarkar Patil" /><meta name="DC.Contributor" content="Shovonlal Bhowmick" /><meta name="DC.Date" content="2024 Mar 15" /><meta name="DC.Identifier" content="10.1016/j.heliyon.2024.e26802" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Identification of Mycobacterium tuberculosis transcriptional repressor EthR inhibitors: Shape-based search and machine learning studies" /><meta property="og:type" content="article" /><meta property="og:description" content="Tuberculosis has been a challenge to the world since prehistoric times, and with the advent of drug-resistant strains, it has become more challenging to treat this infection. Ethionamide (ETH), a second-line drug, acts as a prodrug and targets mycolic ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10907797/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10907797/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/2992/">Heliyon</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/456855/">v.10(5); 2024 Mar 15</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC10907797
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC10907797/pdf/main.pdf" class="int-view">PDF (6.4M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/10907797/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/10907797/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="10907797" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10907797%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="ZJECi6FgZlXsxTUVtwpnEh38DKAJmdL3vh02gfwFln5ZKYoEJd2EEXcur0L47qbS">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10907797%2F&amp;text=Identification%20of%20Mycobacterium%20tuberculosis%20transcriptional%20repressor%20EthR%20inhibitors%3A%20Shape-based%20search%20and%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10907797%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10907797/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC10907797/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38434349/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC10907797/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38434349/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC10907797/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/10907797/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/2992/">Heliyon</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/456855/">v.10(5); 2024 Mar 15</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC10907797
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-heliyon.png" alt="Logo of heliyon" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.journals.elsevier.com/heliyon" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Heliyon.</span> 2024 Mar 15; 10(5): e26802. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2024 Feb 22. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1016%2Fj.heliyon.2024.e26802" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.heliyon.2024.e26802</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC10907797</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38434349">38434349</a></div></div></div><h1 class="content-title">Identification of Mycobacterium tuberculosis transcriptional repressor EthR inhibitors: Shape-based search and machine learning studies</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Chikhale%20RV%5BAuthor%5D" id="au1" class="affpopup" co-rid="_co_idm140451463620896" co-class="co-affbox">Rupesh V. Chikhale</a>,<sup>a</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Eldesoky%20GE%5BAuthor%5D" id="au2" class="affpopup" co-rid="_co_idm140451474137184" co-class="co-affbox">Gaber E. Eldesoky</a>,<sup>b</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kolpe%20MS%5BAuthor%5D" id="au3" class="affpopup" co-rid="_co_idm140451474135520" co-class="co-affbox">Mahima Sudhir Kolpe</a>,<sup>c</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Suryawanshi%20VS%5BAuthor%5D" id="au4" class="affpopup" co-rid="_co_idm140451515740112" co-class="co-affbox">Vikramsinh Sardarsinh Suryawanshi</a>,<sup>c</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Patil%20PC%5BAuthor%5D" id="au5" class="affpopup" co-rid="_co_idm140451515737888" co-class="co-affbox">Pritee Chunarkar Patil</a>,<sup>d</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Bhowmick%20S%5BAuthor%5D" id="au6" class="affpopup" co-rid="_co_idm140451467792496" co-class="co-affbox">Shovonlal Bhowmick</a><sup>c,</sup><sup>&#x02217;</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140451463620896"><h3 class="no_margin">Rupesh V. Chikhale</h3><p><sup>a</sup>Department of Pharmaceutical and Biological Chemistry, School of Pharmacy, University College London, London, UK</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Chikhale%20RV%5BAuthor%5D">Rupesh V. Chikhale</a></div></div><div id="_co_idm140451474137184"><h3 class="no_margin">Gaber E. Eldesoky</h3><p><sup>b</sup>Chemistry Department, College of Science, King Saud University, Riyadh, 11451, Saudi Arabia</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Eldesoky%20GE%5BAuthor%5D">Gaber E. Eldesoky</a></div></div><div id="_co_idm140451474135520"><h3 class="no_margin">Mahima Sudhir Kolpe</h3><p><sup>c</sup>SilicoScientia Private Limited, Nagananda Commercial Complex, No. 07/3, 15/1, 18th Main Road, Jayanagar 9th Block, Bengaluru, 560041, India</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kolpe%20MS%5BAuthor%5D">Mahima Sudhir Kolpe</a></div></div><div id="_co_idm140451515740112"><h3 class="no_margin">Vikramsinh Sardarsinh Suryawanshi</h3><p><sup>c</sup>SilicoScientia Private Limited, Nagananda Commercial Complex, No. 07/3, 15/1, 18th Main Road, Jayanagar 9th Block, Bengaluru, 560041, India</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Suryawanshi%20VS%5BAuthor%5D">Vikramsinh Sardarsinh Suryawanshi</a></div></div><div id="_co_idm140451515737888"><h3 class="no_margin">Pritee Chunarkar Patil</h3><p><sup>d</sup>Department of Bioinformatics, Rajiv Gandhi Institute of IT and Biotechnology, Bharati Vidyapeeth Deemed to be University, Pune-Satara Road, Pune, India</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Patil%20PC%5BAuthor%5D">Pritee Chunarkar Patil</a></div></div><div id="_co_idm140451467792496"><h3 class="no_margin">Shovonlal Bhowmick</h3><p><sup>c</sup>SilicoScientia Private Limited, Nagananda Commercial Complex, No. 07/3, 15/1, 18th Main Road, Jayanagar 9th Block, Bengaluru, 560041, India</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Bhowmick%20S%5BAuthor%5D">Shovonlal Bhowmick</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140451469280528_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140451469280528_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140451469280528_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140451469280528_ai" style="display:none"><div class="fm-affl" id="aff1"><sup>a</sup>Department of Pharmaceutical and Biological Chemistry, School of Pharmacy, University College London, London, UK</div><div class="fm-affl" id="aff2"><sup>b</sup>Chemistry Department, College of Science, King Saud University, Riyadh, 11451, Saudi Arabia</div><div class="fm-affl" id="aff3"><sup>c</sup>SilicoScientia Private Limited, Nagananda Commercial Complex, No. 07/3, 15/1, 18th Main Road, Jayanagar 9th Block, Bengaluru, 560041, India</div><div class="fm-affl" id="aff4"><sup>d</sup>Department of Bioinformatics, Rajiv Gandhi Institute of IT and Biotechnology, Bharati Vidyapeeth Deemed to be University, Pune-Satara Road, Pune, India</div><div><span class="contrib-email" id="au6">Shovonlal Bhowmick: <a href="mailto:dev@null" data-email="moc.aitneicsocilis@kcimwohb.lalnovohs" class="oemail">moc.aitneicsocilis@kcimwohb.lalnovohs</a></span> </div><div id="cor1"><sup>&#x02217;</sup>Corresponding author. <a href="mailto:dev@null" data-email="moc.aitneicsocilis@kcimwohb.lalnovohs" class="oemail">moc.aitneicsocilis@kcimwohb.lalnovohs</a></div></div><div class="fm-article-notes hide half_rhythm" id="idm140451469280528_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2023 Nov 9; Revised 2024 Jan 16; Accepted 2024 Feb 20.</div></div><div class="permissions half_rhythm hide" id="idm140451469280528_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; 2024 The Authors. Published by Elsevier Ltd.</div><div class="license half_rhythm">This is an open access article under the CC BY license (http://creativecommons.org/licenses/by/4.0/).</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-length="59" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first-last">The data related to this work could be provided on request.</p></dd></dl></div><div id="abs0010" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abs0010title">Abstract</h2><!--article-meta--><div><p class="p p-first-last">Tuberculosis has been a challenge to the world since prehistoric times, and with the advent of drug-resistant strains, it has become more challenging to treat this infection. Ethionamide (ETH), a second-line drug, acts as a prodrug and targets mycolic acid synthesis by targeting the enoyl-acyl carrier protein reductase (InhA) enzyme. <em>Mycobacterium tuberculosis</em> (Mtb) EthR is an ethA gene repressor required to activate prodrug ETH. Recent studies suggest targeting the EthR could lead to newer drug molecules that would help better activate the ETH or complement this process. In this report, we have attempted and successfully identified three new molecules from the drug repurposing library that can target EthR protein and function as ETH boosters. These molecules were obtained after rigorous filtering of the database for their physicochemical, toxicological properties and safety. The molecular docking, molecular dynamics simulations and binding energy studies yielded three compounds, Ethyl (2-amino-4-((4-fluorobenzyl)amino)phenyl)carbamate) (L1), 2-((2,2-Difluorobenzo [d] [1,3]dioxol-5-yl)amino)-2-oxoethyl (E)-3-(5-bromofuran-2-yl)acrylate (L2), and N-(2,3-Dihydrobenzo [b] [1,4]dioxin-6-yl)-4-(2-((4-fluorophenyl)amino)-2-oxoethoxy)-3-methoxy benzamide (L3) are potential EthR inhibitors. We applied machine learning methods to evaluate these molecules for toxicity and synthesisability, suggesting safety and ease of synthesis for these molecules. These molecules are known for other pharmacological activities and can be repurposed faster as adjuvant therapy for tuberculosis.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Ethionamide, EthA gene repressor, Transcriptional repressor, ADMET, Drug resistance</span></div></div><div id="sec1" class="tsec sec"><h2 class="head no_bottom_margin" id="sec1title">1. Introduction</h2><p id="p0010" class="p p-first">Tuberculosis (Tb) is an infectious disease caused by the <em>Mycobacterium tuberculosis</em> (Mtb), an aerobic, pathogenic bacteria from the Mycobacteriaceae family [<a href="#bib1" rid="bib1" class=" bibr popnode">1</a>]. According to one school of thought, Mtb co-evolved with humans, and it travelled from Africa to various parts of the globe with the human settlements, making them one of the oldest pathogens infecting prehistoric and modern humans [<a href="#bib2" rid="bib2" class=" bibr popnode">2</a>]. Tb is endemic to most parts of the world with more prevalent in developing countries compared to developed nations, however, illegal immigration and thus lack of treatment and diagnosis is becoming one of the reasons for its spread through the globe [<a href="#bib3" rid="bib3" class=" bibr popnode">3</a>]. According to the data from the World Health Organisation (WHO), there were about 1.6 million deaths due to Tb in the year 2021 including 187,000 coinfected with HIV. It was also estimated that 10.6 million people were infected with Tb including 1.2 million children in 2021 [<a href="#bib4" rid="bib4" class=" bibr popnode">4</a>]. The rise in Multi Drug Resistant (MDR) is one of the most concerning facts reflected in the Global Tuberculosis Report 2022, which suggests an increase in cases during 2015&#x02013;21 [<a href="#bib5" rid="bib5" class=" bibr popnode">5</a>,<a href="#bib6" rid="bib6" class=" bibr popnode">6</a>]. The drug-sensitive Tb can be treated with first-line drug therapy for 6 months, which includes a combination of Isoniazid, Rifampicin, Pyrazinamide and Ethambutol. The second-line drugs are used in different combinations depending on the drug sensitivity and the patient response, these drugs are Streptomycin, Kanamycin, Amikacin, Capreomycin, Ofloxacin, Ethionamide, Terizidone and certain other antibiotics [<a href="#bib7" rid="bib7" class=" bibr popnode">7</a>]. Linezolid, Bedaquiline and Delamanid are some of the drugs used in the treatment of MDR and extremely drug-resistant (XDR) tuberculosis [<a href="#bib8" rid="bib8" class=" bibr popnode">8</a>]. The development of drug resistance in Tb is at various levels, these can be defined as mono-resistance (one of the first-line <em>anti</em>-Tb drugs), Poly-resistance (resistance to isoniazid, rifampicin and one or more first-line drugs), MDR (resistance to isoniazid and rifampicin), XDR (resistance to fluoroquinolone&#x000a0;+&#x000a0;one of the 3-s line injectable drug, capreomycin, kanamycin, amikacin) and Rifampicin resistance (RR) [<a href="#bib9" rid="bib9" class=" bibr popnode">9</a>]. The treatment for these forms of Tb varies froam a minimum of six months to more than two years, depending on the drug response, patient compatibility, and socioeconomic conditions [<a href="#bib10" rid="bib10" class=" bibr popnode">10</a>]. The global disease burden, lengthy duration of therapy and the increasing occurrence of drug resistance have challenged the scientific community to urgent need for novel <em>anti</em>-Tb drug molecules. There are various approaches taken to answer this challenge, like the development of the Tb vaccine [<a href="#bib11" rid="bib11" class=" bibr popnode">11</a>], new drug molecules [<a href="#bib12" rid="bib12" class=" bibr popnode">12</a>], investigation of new drug targets in Mtb [<a href="#bib13" rid="bib13" class=" bibr popnode">13</a>] and revisiting the old target with innovative approaches [<a href="#bib14" rid="bib14" class=" bibr popnode">[14]</a>, <a href="#bib15" rid="bib15" class=" bibr popnode">[15]</a>, <a href="#bib16" rid="bib16" class=" bibr popnode">[16]</a>]. Some Tb drugs, such as ethionamide (ETH) and isoniazid (INH), are structurally analogous and need to be activated before performing their desired biological activity. In this case, both drugs target enoyl-acyl carrier protein reductase (InhA), which is an enzyme necessary for mycolic acid synthesis [<a href="#bib17" rid="bib17" class=" bibr popnode">17</a>]. ETH is an important second-line drug used in treating Tb, and the mechanism of action is very well understood for ETH. Besra and Baulard [<a href="#bib18" rid="bib18" class=" bibr popnode">18</a>], explained the ETH activator Rv3854c and termed it EthA, it is analogous to the various monooxygenases and induces ETH sensitivity on overexpression in Mycobacteria. Later, Montellano and coworkers established the EtaA as a FAD-containing enzyme that oxidizes ETA to its corresponding S-oxide. This is further oxidized to 2-ethyl-4-amidopyridine via the formation of intermediate sulfinic acid. Mtb EthR is an ethA gene repressor required to activate prodrug ETH. EthR is a homodimer with a helical structure and shows similarity to the TetR family members [<a href="#bib19" rid="bib19" class=" bibr popnode">19</a>].</p><p id="p0020" class="p p-last">Baulard and coworkers [<a href="#bib20" rid="bib20" class=" bibr popnode">20</a>] produced a novel approach of inhibiting the EthR by small molecule inhibitors and thus boosting the bioactivation of ETH, one of the lead molecules from this study was BDM31343 (<strong>1</strong>) (<a href="/pmc/articles/PMC10907797/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a> (1)). Later, they reported the synthesis, biological studies and SAR for a series of thiophen-2-yl-1,2,4-oxadiazoles as inhibitors of EthR (<strong>2</strong>) (<a href="/pmc/articles/PMC10907797/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a> (2)) with improved activity and some physicochemical properties like solubility and metabolic stability [<a href="#bib21" rid="bib21" class=" bibr popnode">21</a>]. These ligands were further optimized in the next attempt and BDM41906 (<strong>3</strong>) (<a href="/pmc/articles/PMC10907797/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a> (3)) [<a href="#bib22" rid="bib22" class=" bibr popnode">22</a>] was obtained because of structure-based drug design. This molecule had improved efficacy and better pharmacokinetic properties. They also applied an alternative approach of fragment-based design using a mix of surface plasmon resonance assay (SPR), X-ray crystallography, and SAR, leading to the identification of a potent molecule, 4-(2-Methylthiazol-4-yl)-N-(3,3,3-trifluoropropyl)benzamide (<strong>4</strong>, PDB: <a href="https://www.rcsb.org/structure/https://www.rcsb.org/4M3D" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">4M3D</a>) (<a href="/pmc/articles/PMC10907797/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a> (4)) with IC<sub>50</sub>&#x000a0;=&#x000a0;0.40&#x000a0;&#x003bc;M, EC<sub>50</sub>&#x000a0;=&#x000a0;0.08&#x000a0;&#x003bc;M and high ligand efficiency of LE&#x000a0;=&#x000a0;0.47, it also shows good solubility [<a href="#bib23" rid="bib23" class=" bibr popnode">23</a>]. In this series other molecules that we found interesting were 4-(2-(Propylsulfonamidomethyl)thiazol-4-yl)-N-(3,3,3-trifluoropropyl)benzamide (<strong>5</strong>, PDB: <a href="https://www.rcsb.org/structure/https://www.rcsb.org/4M3E" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">4M3E</a>) (<a href="/pmc/articles/PMC10907797/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a> (5)) and 4-(3-(Phenylsulfonamido)prop-1-ynyl)-N-(3,3,3-trifluoropropyl)-benzamide (<strong>6</strong>, PDB: <a href="https://www.rcsb.org/structure/https://www.rcsb.org/4M3D" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">4M3D</a>) (<a href="/pmc/articles/PMC10907797/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a> (6)), these compounds show similar interactions as the best compound from the series when observed from their cocrystal structures. The concept of identification of new inhibitors of Mtb transcriptional repressor EthR is very logical and highly attractive. Herein, we are reporting the application of highly advanced tools in structure-based drug discovery methods for identifying novel molecules from large, small-molecule databases. Herein, the focus lies on structural similarities within the chemical scaffolds with known EthR inhibitors, as discussed earlier in <a href="/pmc/articles/PMC10907797/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a>, and how these could be explored to mine the ligands from drug repurposing databases.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC10907797/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140451480011536"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10907797_gr1.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 1" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10907797/bin/gr1.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140451480011536"><a target="object" rel="noopener" href="/pmc/articles/PMC10907797/figure/fig1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC10907797/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Fig. 1</a></div><!--caption a7--><div class="caption"><p>Recent inhibitors of Mtb transcriptional repressor EthR and the scaffold similarities in them: (<strong>1</strong>) BDM31343; (2) Thiophen-2-yl-1,2,4-oxadiazoles; (3) BDM41906, (<strong>4</strong>) 4-(2-Methylthiazol-4-yl)-N-(3,3,3-trifluoropropyl) benzamide; (<strong>5</strong>) 4-(2-(Propylsulfonamidomethyl)thiazol-4-yl)-N-(3,3,3-trifluoropropyl) benzamide; (<strong>6</strong>) 4-(3-(Phenylsulfonamido)prop-1-ynyl)-N-(3,3,3-trifluoropropyl) benzamide (<strong>L5</strong>).</p></div></div></div></div><div id="sec2" class="tsec sec"><h2 class="head no_bottom_margin" id="sec2title">2. Materials and methods</h2><div id="sec2.1" class="sec sec-first"><h3 id="sec2.1title">2.1. Selection of a large chemical library database and initial curation of the database for screening</h3><p id="p0025" class="p p-first-last">A large compound pool of &#x0223c;160,383 small molecules was downloaded from Selleckchem [<a href="#bib24" rid="bib24" class=" bibr popnode">24</a>] (<a href="https://www.selleckchem.com/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.selleckchem.com/</a>), a top-rated chemical supplier across the globe containing a diverse set of chemicals ready to use for virtual screening purposes. Moreover, it has been stated that more than 120,000 compounds are associated with being used to research cell signalling pathways. As one of the top chemical suppliers of the newest inhibitors, Selleckchem actively follows the most recent scientific developments to benefit pharmaceutical industries or other research projects. Over &#x0223c;8000 small compounds in Selleck's Bioactive Compound Libraries have been shown to have biological and pharmacological properties. Moreover, the selected database includes compounds entered in preclinical and clinical studies and have proven their efficacy and safety, and several of them have received FDA approval. To begin with, three major filtration criteria, Lipinski's rule of five (Ro5), Veber's rule and Drug-likeness properties were used to screen the database and further curated for retaining most potential drug-like candidates. Such properties screening was implemented in Python RDKit (<a href="https://www.rdkit.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.rdkit.org</a>). Besides the large Selleckchem chemical library, a small set of 30 <em>anti</em>-Tb compounds were separately collected. These 30&#x000a0;TB drugs were used in the study setting to compare and validate the study outcome.</p></div><div id="sec2.2" class="sec"><h3 id="sec2.2title">2.2. Preparation of the curated chemical library dataset and FDA-approved TB drugs</h3><p id="p0030" class="p p-first-last">The ligand molecules were obtained following earlier mentioned filtration criteria, and the 30 <em>anti</em>-Tb drugs were prepared according to the standard protocol using the OpenBabel tool [<a href="#bib25" rid="bib25" class=" bibr popnode">25</a>]. In the ligand preparation step, the 3D conformation of the compounds was generated, and hydrogen (-H) atoms were added. The pH and partial charge for the ligands in the database were adjusted where needed. Subsequently, these were converted into &#x02018;pdbqt&#x02019; file format which is necessary for the AutoDock Vina docking tool [<a href="#bib26" rid="bib26" class=" bibr popnode">26</a>]. In particular, the &#x0201c;PDBQT&#x0201d; file contains the information on Partial Charge (Q) &#x00026; Atom Type (T) for a specific ligand. Displaying the normal PDB files does not hold such information. In contrast, PDBQT files constitute an additional column that lists the partial charge (Q) and the assigned AutoDock atom type (T) for each atom in the molecule. The AutoDock atom types provide a better insight toward granular differentiation between atoms, such as listing aliphatic and aromatic carbons as separate AutoDock atom types.</p></div><div id="sec2.3" class="sec"><h3 id="sec2.3title">2.3. Selection of target protein and preparation</h3><p id="p0035" class="p p-first-last">The 3D coordinates of the <em>Mtb</em> HTH-type transcriptional regulator EthR crystal structure (PDB ID: <a href="https://www.rcsb.org/structure/https://www.rcsb.org/4M3D" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">4M3D</a> [<a href="#bib27" rid="bib27" class=" bibr popnode">27</a>]) were obtained from the RCSB-Protein Data Bank (PDB) [<a href="#bib28" rid="bib28" class=" bibr popnode">28</a>]. The EthR crystal structure (PDB ID: <a href="https://www.rcsb.org/structure/https://www.rcsb.org/4M3D" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">4M3D</a>) shows resolution and R-value of 1.90&#x000a0;&#x000c5; and 0.22, respectively. The selected <em>Mtb</em> EthR consists of 216 amino acid residues with no mutation. The desired protein was prepared using AutoDockTools (ADT) [<a href="#bib29" rid="bib29" class=" bibr popnode">29</a>] by adding H-atoms and removing all hetero atoms, including water molecules. Moreover, Kollman charges were added to the protein to adjust the convenient partial charges to the selected Mtb protein. Finally, the prepared protein was saved into the desired file format like &#x02018;pdbqt&#x02019; for later analysis. Moreover, the preferred protein of interest also consists of the attached EthR inhibitor, 4-{3-[(phenylsulfonyl)amino] prop-1-yn-1-yl}-N-(3,3,3-trifluoropropyl) benzamide, which was also prepared as mentioned under ligand preparation.</p></div><div id="sec2.4" class="sec"><h3 id="sec2.4title">2.4. Molecular docking</h3><p id="p0040" class="p p-first">All prepared ligands and the EthR protein were allowed for molecular docking study using the MPI Vina platform [<a href="#bib30" rid="bib30" class=" bibr popnode">30</a>] installed on the Linux operating system. However, before submission of the docking job, the configuration file for AutoDock Vina was created by keeping the required set of information like the protein's centre grid box coordinates along x, y, and z-axes. The rest of the parameters for molecular docking were kept as default in the configuration file. All the compounds of the Selleckchem database kept from the previous step, and 30 FDA-approved TB drugs and co-crystal ligands were docked with <em>Mtb</em> EthR employing standard docking protocol. After successfully executing docking for all compounds said above, their binding affinity was critically checked to further reduce the chemical space for better selection of Mtb EthR-selective compounds. Therefore, all 30 FDA drugs and co-crystal ligand binding affinity score and interaction mode were critically checked to set the threshold binding affinity score for Selleckchem compound database screening and further analysis. The lowest binding affinity score of 30 FDA drugs and co-crystal ligands were compared, and the lowest binding affinity score was considered as a threshold for subsequent analysis for the selection of potential compounds. Moreover, the binding interaction mode was explored using PLIP&#x02013;a web-based protein-ligand interaction retrieval program [<a href="#bib31" rid="bib31" class=" bibr popnode">31</a>].</p><p id="p0045" class="p p-last">Furthermore, it is crucial to note that any molecular docking method must first be validated before being used in the screening of any chemical compound datasets. It was always predicted that molecular docking will result in a conformational orientation that resembles the conformation of a ligand linked to a co-crystal. Hence, self-docking is one of methods for verifying the molecular docking process. This method verifies the co-crystal ligand's binding mode or orientation. In particular, the co-crystal ligand that was initially bound was re-docked to the active binding site using same docking grid parameters used for rest of compounds datasets.</p></div><div id="sec2.5" class="sec"><h3 id="sec2.5title">2.5. Post-docking processing of compounds and subsequent ADMET based filtration using pkCSM</h3><p id="p0050" class="p p-first-last">The pkCSM [<a href="#bib32" rid="bib32" class=" bibr popnode">32</a>] web-tool (<a href="https://biosig.lab.uq.edu.au/pkcsm/prediction" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://biosig.lab.uq.edu.au/pkcsm/prediction</a>) was used for ADMET (absorption, distribution, metabolism, excretion, and toxicity) properties-based filtration on best hit compounds retained from molecular docking analysis. ADMET is an important criterion here implemented to reduce the chances of selecting the compounds with adverse side effects and thereby also reducing the failure rate for compounds&#x02019; efficacy and safety deficiencies. <em>In-silico</em> ADMET prediction approach has long been used as an important parameter in the pharmaceutical or drug discovery pipeline for reducing the risk of late-stage attrition and helping in the choice of potentially good compounds from large compound datasets or optimization of new chemical entities. For the execution of ADMET in pkCSM, all best hits (&#x0223c;3194) compounds obtained from docking study were browsed in standard SMILE file format and prediction of all pharmacokinetic properties was estimated under ADMET.</p></div><div id="sec2.6" class="sec"><h3 id="sec2.6title">2.6. Machine learning based toxicity and synthetic accessibility assessment</h3><p id="p0055" class="p p-first">Machine learning (ML) is widely used to explore the toxicity and other drug-likeness assessment of small molecules [<a href="#bib33" rid="bib33" class=" bibr popnode">33</a>]. Molecules found after the screening of SelleckChem against Mtb protein were taken for toxicity and synthetic accessibility calculation using eToxPred tool [<a href="#bib34" rid="bib34" class=" bibr popnode">34</a>]. It uses ML models to predict the toxicity of the given molecules directly from the molecular fingerprints of chemical compounds. The models in eToxPred are trained and confirmed through several diverse datasets including known drugs, potentially hazardous chemicals, natural products, and synthetic bioactive compounds.</p><p id="p0060" class="p p-last">Machine learning models in eToxPred are trained and cross-validated against a number of datasets comprising known drugs, potentially hazardous chemicals, natural products, and synthetic bioactive compounds. The models are developed by the Extra Tree (ET) ML algorithm. Along with the ET, another three ML algorithms such as Gradient Boosting (GB), Balanced Random Forest (BRF) and Balanced Bagging (Confirmers to develop the ML models with their given datasets [<a href="#bib35" rid="bib35" class=" bibr popnode">35</a>,<a href="#bib36" rid="bib36" class=" bibr popnode">36</a>]. Prior to use in the prediction of toxicity and SA scores, all the developed models were validated through the calculation of receiver operating curve (ROC) area under the curve (AUC) and accuracy. The validated models were used to calculate the overall toxicity value which suggests whether the molecule is toxic or non-toxic in nature. The toxicity value ranges from 0 to 1 showing non-toxic to toxic, respectively. Along with toxicity, the eToxPred also gives the synthetic accessibility (SA) of any given molecule. The SA score represented 1 to 10 signifying easy to difficult in synthesis, respectively.</p></div><div id="sec2.7" class="sec sec-last"><h3 id="sec2.7title">2.7. Molecular dynamics (MD) simulations analysis of the protein-ligand complex and MM-GBSA</h3><p id="p0065" class="p p-first">Through all atomic MD simulation analyses, the dynamic nature of the selected screened four protein-ligand complexes, including the co-crystal and best standard TB drug compounds bound with EthR complexes, was assessed. The MD simulation approach helps to understand the inter- or intra-molecular binding interactions between protein-ligand macromolecular complexes and their reorganisation behaviour in different energetically induced conformational states in a dynamic environment. In this study total of six protein-ligand complexes were subjected for 100 ns all atomistic classical MD simulations. The MD simulation run was conducted with a time step of 2 fs, at a constant pressure of 1&#x000a0;atm, and constant temperature of 300&#x000a0;K. The <em>Mtb</em> EthR protein topology was produced using the CHARMM36 all-atoms forcefield, while SwissParam [<a href="#bib37" rid="bib37" class=" bibr popnode">37</a>], an open-source server available online was used to create the topology of the final small molecules under consideration for MD simulation. Each protein-ligand complex system was solvated using the TIP3P water model and submerged into a cubic box before the simulation run. To neutralise the bio-macromolecular system as a whole, the suitable numbers of Na<sup>+</sup> and Cl<sup>&#x02212;</sup> were adjusted. Each protein-ligand system underwent 50,000 steps of the steepest descents energy minimization approach to eliminate steric overlap. The SHAKE method was used to restrict the hydrogen atoms. The long-range electrostatic interactions were evaluated, implying the Particle Mesh Ewald (PME) [<a href="#bib38" rid="bib38" class=" bibr popnode">38</a>] method and the cut-off value for non-bonded interactions was established at the cut-off value of 8&#x000a0;&#x000c5;. Langevin thermostat with a 2.0 collision frequency was used to regulate the entire simulated system's overall temperature. At 1&#x000a0;atm Monte Carlo barostat with volume exchanges occurring every 100 fs was also used to regulate the system's pressure. NVT and NPT ensembles were used to perform rounds of the system equilibration, before acquiring the MD simulation data. A variety of different important analytic metrics, including protein backbone root-mean-square deviation (RMSD), ligand RMSD root-mean-square fluctuation (RMSF), and radius of gyration (RoG), were estimated following the successful completion of the MD manufacturing run during the 100 ns MD simulation span employing several GROMACS [<a href="#bib39" rid="bib39" class=" bibr popnode">39</a>] integrated utility tools.</p><p id="p0070" class="p p-last">Additionally, each MD simulated protein-ligand complex's binding free energy (&#x00394;G<sub><em>bind</em></sub>) was determined using the Generalised Born and Surface Area Solvation (MM/GBSA) method, employing the GROMACS embedded gmx_mmpbsa utility package [<a href="#bib40" rid="bib40" class=" bibr popnode">40</a>]. The complete set of 10,000 frames was considered for the binding free energy calculation with an interval of 5. The detailed mathematical expression was considered to determine &#x00394;G<sub><em>bind</em></sub> has discussed previously [<a href="#bib41" rid="bib41" class=" bibr popnode">41</a>,<a href="#bib42" rid="bib42" class=" bibr popnode">42</a>], elsewhere.</p></div></div><div id="sec3" class="tsec sec"><h2 class="head no_bottom_margin" id="sec3title">3. Results and discussion</h2><div id="sec3.1" class="sec sec-first"><h3 id="sec3.1title">3.1. Screening of SelleckChem database against Mtb</h3><p id="p0075" class="p p-first">The Selleckchem database consisting of 160,383 ligands was prepared and were filtered for their physicochemical properties and drug-likeness. Herein, we considered Lipinski's rule of five, Veber's rule and Drug-likeness as filters to narrow down the chemical space for further evaluation. This first pass led to a total of 119,854 molecules retained from the database. These molecules were subjected to molecular docking studies using AutoDock Vina running in MPI mode where the active site of the Mtb transcriptional repressor EthR was targeted. Along with the Selleckchem database we have also considered 30 <em>anti</em>-TB drug molecules for the docking studies. The protein crystal structure was retrieved from the RCSB-PDB and was prepared for docking experiment. The native ligand bound to the crystal structure of EthR was extracted and used in the docking studies as a control. To validate the docking protocol, the self-docking approach was used in which the structure of the bound co-crystal ligand was re-drawn and docked at the active site. The best-docked pose and co-crystal ligand were superimposed and RMSD was calculated. Multiple docking of the re-drawn co-crystal ligand was performed and superimposed to record the RMSD. The best protocol with superimposition RMSD of 0.182&#x000a0;&#x000c5; was identified and used for the further screening of the Selleckchem database. The superimposed co-crystal conformation and best docked pose is given in <a href="/pmc/articles/PMC10907797/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC10907797/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140451518673952"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10907797_gr2.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 2" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10907797/bin/gr2.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140451518673952"><a target="object" rel="noopener" href="/pmc/articles/PMC10907797/figure/fig2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC10907797/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Fig. 2</a></div><!--caption a7--><div class="caption"><p>Superimposed structure of co-crystal conformation and best docked pose of the same.</p></div></div></div><p id="p0080">The main aim of the current piece of work was to find some potential lead-like molecules for therapeutic application in Tb infection through the virtual screening approach. The detailed workflow of the work is given in <a href="/pmc/articles/PMC10907797/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Fig. 3</span></a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig3" co-legend-rid="lgnd_fig3"><a href="/pmc/articles/PMC10907797/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140451515587536"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10907797_gr3.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 3" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10907797/bin/gr3.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140451515587536"><a target="object" rel="noopener" href="/pmc/articles/PMC10907797/figure/fig3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig3"><div><a class="figpopup" href="/pmc/articles/PMC10907797/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3">Fig. 3</a></div><!--caption a7--><div class="caption"><p>Virtual screening workflow for the screening of the SelleckChem database against Mtb.</p></div></div></div><p id="p0085" class="p p-last">On successful completion of the molecular docking, the binding energy of each molecule was extracted and analyzed. To reduce the chemical space to select better affinity molecules, a threshold of &#x02212;10.0&#x000a0;kcal/mol was applied. By applying the above threshold in the binding energy, a total of 3194 molecules were found to have a binding energy of &#x02212;10.0&#x000a0;kcal/mol or less. It is important to note that the co-crystal molecule was found to show a binding affinity of &#x02212;9.5&#x000a0;kcal/mol. The above remained molecules were subjected to ADMET screening analysis using the pkCSM tool to obtain molecules having an acceptable pharmacokinetic profile as well as safe for the treatment of Tb. In particular, a number of pharmacokinetics and toxicity parameters were recorded and analyzed. Molecules were found to have intestinal absorption of less than 30, skin permeability of greater than &#x02212;2.5, BBB of greater than &#x02212;1, and CNS permeability greater than &#x02212;3 were removed due to their poor pharmacokinetic profile. Molecules retained after the pharmacokinetic assessment were further taken into consideration for the toxicity assessment. For the above remaining molecules, AMES toxicity, hepatotoxicity and skin sensitivity were checked and compounds that showed negative response were taken for minnow toxicity and maximum tolerated dose analyses. It is recommended that molecules should have a minnow toxicity greater than &#x02212;0.3 and a maximum tolerated dose less than 0.477. By following the above criteria, a total of 3 molecules were retained. The two-dimensional (2D) representation of the final selected molecules along with co-crystal bound ligand and best docked drug molecule is given in <a href="/pmc/articles/PMC10907797/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>. For our convention and easy explanation, from here onwards, the above molecules are named <strong>L1</strong> (CAS ID: 150,812-12-7), <strong>L2</strong> (CAS ID: 807,608-56-6), <strong>L3</strong> (CAS ID: 734,545-13-2), <strong>L4</strong> (TB drug, Pretomanid, CAS ID: 187,235-37-6) and <strong>L5</strong> (Co-crystallised ligand, CAS ID: 1589819-46-4). It is important to note that all three selected molecules possess a common structural feature (<strong>L6</strong>) that is shown in a rectangular box in <a href="/pmc/articles/PMC10907797/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>. On close observation of each of the selected molecules, it can be prominently visible a number of pharmacophoric features in terms of hydrogen bond (HB) acceptors and donors, ring aromatic, and hydrophobic. Therefore, it is highly expected that selected molecules will form several binding interactions with the active site amino acids in the forms of HB, hydrophobic, and &#x003c0;-staking. those will help to hold the molecules in the binding site to exhibit the desired effects.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig4" co-legend-rid="lgnd_fig4"><a href="/pmc/articles/PMC10907797/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140451472835408"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10907797_gr4.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 4" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10907797/bin/gr4.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140451472835408"><a target="object" rel="noopener" href="/pmc/articles/PMC10907797/figure/fig4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig4"><div><a class="figpopup" href="/pmc/articles/PMC10907797/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4">Fig. 4</a></div><!--caption a7--><div class="caption"><p>The final selected molecules from molecular docking experiments: <strong>L1</strong> (Ethyl (2-amino-4-((4-fluorobenzyl)amino)phenyl)carbamate), <strong>L2</strong> (2-((2,2-Difluorobenzo [d] [<a href="#bib1" rid="bib1" class=" bibr popnode">1</a>,<a href="#bib3" rid="bib3" class=" bibr popnode">3</a>]dioxol-5-yl)amino)-2-oxoethyl (E)-3-(5-bromofuran-2-yl)acrylate), and <strong>L3</strong> (N-(2,3-Dihydrobenzo [b] [<a href="#bib1" rid="bib1" class=" bibr popnode">1</a>,<a href="#bib4" rid="bib4" class=" bibr popnode">4</a>]dioxin-6-yl)-4-(2-((4-fluorophenyl)amino)-2-oxoethoxy)-3-methoxybenzamide) were obtained as hit ligands from the Selleckchem database following molecular docking. <strong>L4</strong>, also known as Pretomanid ((<em>S</em>)-2-Nitro-6-((4-(trifluoromethoxy)benzyl)oxy)-6,7-dihydro-5H-imidazo [2,1-b] [<a href="#bib1" rid="bib1" class=" bibr popnode">1</a>,<a href="#bib3" rid="bib3" class=" bibr popnode">3</a>]oxazine) is potent antimycobacterial compound and <strong>L5</strong> (4-{3-[(phenylsulfonyl)amino] prop-1-yn-1-yl}-N-(3,3,3-trifluoropropyl)benzamide) is the co-crystallised ligand from the crystal structure of EthR (PDB: <a href="https://www.rcsb.org/structure/https://www.rcsb.org/4M3D" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">4M3D</a>). <strong>L6</strong> represents common structural features for this series of EthR inhibitors.</p></div></div></div></div><div id="sec3.2" class="sec"><h3 id="sec3.2title">3.2. Pharmacokinetic, toxicity and synthetic accessibility</h3><p id="p0090" class="p p-first">The pharmacokinetic and toxicity parameters for the final three molecules were predicted from the pKCSM and these are given in <a href="/pmc/articles/PMC10907797/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>. The molecules' molecular weight and logP are less than 500&#x000a0;kDa and 5.0, respectively. The rotatable bonds were found to be 5, 5 and 7 for <strong>L1</strong>, <strong>L2</strong> and <strong>L3</strong> (<a href="/pmc/articles/PMC10907797/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>), respectively, which suggest molecules are not rigid and are highly flexible. Other pharmacokinetic parameters such as intestinal absorption, skin, BBB and CNS permeability were in acceptable range. No toxicity parameters such as AMES, maximum tolerated dose, hepatoxicity, skin sensitivity and minnow toxicity were found in the toxic range. Hence, from the acceptable pharmacokinetic and toxicity profile, all molecules will be safe and easy to reach at the target side.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p>Pharmacokinetic and toxicity assessment of final selected molecules.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"><sup>a</sup>MW</th><th rowspan="1" colspan="1">LogP</th><th rowspan="1" colspan="1"><sup>b</sup>NRB</th><th rowspan="1" colspan="1"><sup>c</sup>IA</th><th rowspan="1" colspan="1"><sup>d</sup>SP</th><th rowspan="1" colspan="1"><sup>e</sup>BBB</th><th rowspan="1" colspan="1"><sup>f</sup>CNS</th><th rowspan="1" colspan="1">AMES</th><th rowspan="1" colspan="1"><sup>g</sup>MTD</th><th rowspan="1" colspan="1"><sup>h</sup>HT</th><th rowspan="1" colspan="1"><sup>i</sup>SS</th><th rowspan="1" colspan="1"><sup>j</sup>MT</th></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1"><strong>L1</strong></td><td align="left" rowspan="1" colspan="1">304.325</td><td align="left" rowspan="1" colspan="1">2.983</td><td align="left" rowspan="1" colspan="1">5</td><td align="left" rowspan="1" colspan="1">87.818</td><td align="left" rowspan="1" colspan="1">&#x02212;2.864</td><td align="left" rowspan="1" colspan="1">&#x02212;1.259</td><td align="left" rowspan="1" colspan="1">&#x02212;3.281</td><td align="left" rowspan="1" colspan="1">No</td><td align="left" rowspan="1" colspan="1">0.032</td><td align="left" rowspan="1" colspan="1">No</td><td align="left" rowspan="1" colspan="1">No</td><td align="left" rowspan="1" colspan="1">0.926</td></tr><tr><td align="left" rowspan="1" colspan="1"><strong>L2</strong></td><td align="left" rowspan="1" colspan="1">429.856</td><td align="left" rowspan="1" colspan="1">3.347</td><td align="left" rowspan="1" colspan="1">5</td><td align="left" rowspan="1" colspan="1">94.469</td><td align="left" rowspan="1" colspan="1">&#x02212;2.760</td><td align="left" rowspan="1" colspan="1">&#x02212;0.988</td><td align="left" rowspan="1" colspan="1">&#x02212;3.190</td><td align="left" rowspan="1" colspan="1">No</td><td align="left" rowspan="1" colspan="1">0.540</td><td align="left" rowspan="1" colspan="1">No</td><td align="left" rowspan="1" colspan="1">No</td><td align="left" rowspan="1" colspan="1">&#x02212;1.469</td></tr><tr><td align="left" rowspan="1" colspan="1"><strong>L3</strong></td><td align="left" rowspan="1" colspan="1">452.438</td><td align="left" rowspan="1" colspan="1">3.875</td><td align="left" rowspan="1" colspan="1">7</td><td align="left" rowspan="1" colspan="1">99.833</td><td align="left" rowspan="1" colspan="1">&#x02212;2.748</td><td align="left" rowspan="1" colspan="1">&#x02212;0.428</td><td align="left" rowspan="1" colspan="1">&#x02212;3.114</td><td align="left" rowspan="1" colspan="1">No</td><td align="left" rowspan="1" colspan="1">0.398</td><td align="left" rowspan="1" colspan="1">No</td><td align="left" rowspan="1" colspan="1">No</td><td align="left" rowspan="1" colspan="1">&#x02212;0.905</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140451473085200"><a target="object" rel="noopener" href="/pmc/articles/PMC10907797/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tbl1fna"><sup>a</sup>Molecular weight.</div></div><div class="tblwrap-foot"><div id="tbl1fnb"><sup>b</sup>Number of rotatable bonds.</div></div><div class="tblwrap-foot"><div id="tbl1fnc"><sup>c</sup>Intestinal absorption.</div></div><div class="tblwrap-foot"><div id="tbl1fnd"><sup>d</sup>Skin permeability.</div></div><div class="tblwrap-foot"><div id="tbl1fne"><sup>e</sup>Blood brain barrier.</div></div><div class="tblwrap-foot"><div id="tbl1fnf"><sup>f</sup>Central nervous system.</div></div><div class="tblwrap-foot"><div id="tbl1fng"><sup>g</sup>Maximum tolerable dose.</div></div><div class="tblwrap-foot"><div id="tbl1fnh"><sup>h</sup>Heptotoxicity.</div></div><div class="tblwrap-foot"><div id="tbl1fni"><sup>i</sup>Skin sensitivity.</div></div><div class="tblwrap-foot"><div id="tbl1fnj"><sup>j</sup>Minnow toxicity.</div></div></div><p id="p0095">Further, the toxicity and synthetic accessibility of the molecules were predicted through the ML-based eToxPred tool. For the prediction, the default models of the tool developed using ET was used. In addition, three more models were developed using the same dataset using GB, BRF and BB algorithms. All four developed models were validated and found to show high accuracy. The ROC AUC of ET, GB, BRF and BB was found to be 0.970, 0.812, 0.962 and 0.917, respectively. The accuracy of each model was calculated, and it was found to be 0.884, 0.741, 0.875 and 0.829 for ET, GB, BRF and BB, respectively. The above data undoubtedly explained the high predictiveness of each model and can be used to predict any unknown molecule.</p><p id="p0100" class="p p-last">All four ML models were used to predict the selected model's toxicity and SA scores, which is given in <a href="/pmc/articles/PMC10907797/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>. A toxicity score of less than 0.50 can be considered non-toxic. Not a single molecule was found to have a toxicity value of more than 0.430 when predicted using all four ML models. The above observation indicated that all molecules were non-toxic, corroborating the data found using the pkCSM tool. Further, the eToxPred was used to predict the SA score, which is given in <a href="/pmc/articles/PMC10907797/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>. Molecules having a SA score of 5 or less can be easily synthesizable. All molecules were found to have an SA score of less than 5. Hence, all molecules are feasible to synthesize. It is also indicated that among three molecules, <strong>L2</strong> can most easily be synthesised followed by <strong>L3</strong> and <strong>L1</strong> (<a href="/pmc/articles/PMC10907797/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="tbl2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p>Toxicity and SA score of selected molecules predicted through ML models.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1">Mols</th><th rowspan="1" colspan="1">Gradient Boosting<hr /></th><th rowspan="1" colspan="1">Extra trees<hr /></th><th rowspan="1" colspan="1">Balanced RF<hr /></th><th rowspan="1" colspan="1">Balanced Bagging<hr /></th><th rowspan="2" colspan="1">SAS</th></tr><tr><th colspan="4" rowspan="1">TS</th></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1"><strong>L1</strong></td><td align="left" rowspan="1" colspan="1">0.428</td><td align="left" rowspan="1" colspan="1">0.275</td><td align="left" rowspan="1" colspan="1">0.292</td><td align="left" rowspan="1" colspan="1">0.210</td><td align="left" rowspan="1" colspan="1">4.06</td></tr><tr><td align="left" rowspan="1" colspan="1"><strong>L2</strong></td><td align="left" rowspan="1" colspan="1">0.181</td><td align="left" rowspan="1" colspan="1">0.244</td><td align="left" rowspan="1" colspan="1">0.249</td><td align="left" rowspan="1" colspan="1">0.000</td><td align="left" rowspan="1" colspan="1">0.63</td></tr><tr><td align="left" rowspan="1" colspan="1"><strong>L3</strong></td><td align="left" rowspan="1" colspan="1">0.234</td><td align="left" rowspan="1" colspan="1">0.164</td><td align="left" rowspan="1" colspan="1">0.207</td><td align="left" rowspan="1" colspan="1">0.000</td><td align="left" rowspan="1" colspan="1">3.60</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140451467256624"><a target="object" rel="noopener" href="/pmc/articles/PMC10907797/table/tbl2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="fn-a.q.c.d.e.g.e.a"><p class="p p-first-last">TS: Toxicity score; SAS: Synthetic accessibility score.</p></div></div></div></div><div id="sec3.3" class="sec sec-last"><h3 id="sec3.3title">3.3. Binding interactions and MD simulation analysis</h3><p id="p0105" class="p p-first">The binding interaction profile of each of the three selected molecules (<strong>L1</strong>, <strong>L2</strong> and <strong>L3</strong>) along with <strong>L4</strong> and <strong>L5</strong> was explored, and it is given in <a href="/pmc/articles/PMC10907797/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a> (<strong>L1</strong> &#x02013; <strong>L5</strong>). The binding energy and CAS ID are given in <a href="/pmc/articles/PMC10907797/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Table 3</span></a>. L1, <strong>L2</strong>, and L3 were shown to have binding energies of &#x02212;12.45, &#x02212;10.21, and &#x02212;11.32&#x000a0;kcal/mol, respectively. In addition to the above, <strong>L4</strong> and <strong>L5</strong> were firmly bound with Mtb protein with the binding energy of &#x02212;9.31 and &#x02212;9.46&#x000a0;kcal/mol, respectively. It is seen that selected molecules from SelleckChem showed better binding affinity towards Mtb than <strong>L4</strong> and <strong>L5</strong>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig5" co-legend-rid="lgnd_fig5"><a href="/pmc/articles/PMC10907797/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140451471270720"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10907797_gr5.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 5" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10907797/bin/gr5.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140451471270720"><a target="object" rel="noopener" href="/pmc/articles/PMC10907797/figure/fig5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig5"><div><a class="figpopup" href="/pmc/articles/PMC10907797/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5">Fig. 5</a></div><!--caption a7--><div class="caption"><p>Molecular docking studies on the crystal structure of EthR, protein-ligand interactions between the three top hits <strong>L1</strong>, <strong>L2</strong> and <strong>L3</strong> from the docking studies, docked complex of the standard molecule pretomanid and the docked complex of the native co-crystal ligand.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="tbl3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p>Results from the molecular docking studies and MM-GBSA calculations.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Ligand</th><th rowspan="1" colspan="1">CAS no.</th><th rowspan="1" colspan="1">Dock Score</th><th rowspan="1" colspan="1">MM-GBSA (&#x00394;G Kcal/mol)</th></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1"><strong>L1</strong></td><td align="left" rowspan="1" colspan="1">150,812-12-7</td><td align="left" rowspan="1" colspan="1">&#x02212;12.45</td><td align="left" rowspan="1" colspan="1">&#x02212;36.68</td></tr><tr><td align="left" rowspan="1" colspan="1"><strong>L2</strong></td><td align="left" rowspan="1" colspan="1">807,608-56-6</td><td align="left" rowspan="1" colspan="1">&#x02212;10.21</td><td align="left" rowspan="1" colspan="1">&#x02212;32.57</td></tr><tr><td align="left" rowspan="1" colspan="1"><strong>L3</strong></td><td align="left" rowspan="1" colspan="1">734,545-13-2</td><td align="left" rowspan="1" colspan="1">&#x02212;11.32</td><td align="left" rowspan="1" colspan="1">&#x02212;36.17</td></tr><tr><td align="left" rowspan="1" colspan="1"><strong>L4</strong></td><td align="left" rowspan="1" colspan="1">187,235-37-6</td><td align="left" rowspan="1" colspan="1">&#x02212;9.31</td><td align="left" rowspan="1" colspan="1">&#x02212;28.34</td></tr><tr><td align="left" rowspan="1" colspan="1"><strong>L5</strong></td><td align="left" rowspan="1" colspan="1">1589819-46-4</td><td align="left" rowspan="1" colspan="1">&#x02212;9.46</td><td align="left" rowspan="1" colspan="1">&#x02212;21.78</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140451477209136"><a target="object" rel="noopener" href="/pmc/articles/PMC10907797/table/tbl3/?report=objectonly">Open in a separate window</a></div></div><p id="p0110">The binding site of <strong>L5</strong> consists of residues Lue87, Met102, Trp103, Gly106, Ile107, Phe110, Phe114, Trp138, Met142, TRP145, Thr149, VaL152, Glu156, Asn176, Asn179, Glu180, Leu183, Phe184, and Trp207 which forms a cylindrical or a channel shape in the EthR receptor. <a href="/pmc/articles/PMC10907797/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a> shows intermolecular interactions between the docked ligands and the binding site residues. Some of the ligands formed hydrogen bond interactions along with &#x003c0;-staking and several hydrophobic interactions.</p><p id="p0115">After the ADMET studies, the final ligands were subjected to MD simulation for 100 ns with the explicit solvent model. These trajectories were analyzed to understand the stability of the complexes and as input for calculating the binding energy. The MD trajectory analysis of these complexes suggests several levels of knowledge about these selected compounds. The ligand RMSD shown in <a href="/pmc/articles/PMC10907797/figure/fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig6" rid-ob="ob-fig6" co-legend-rid="lgnd_fig6"><span>Fig. 6</span></a>(B) points out the low RMSD for <strong>L1</strong>, <strong>L2</strong>, <strong>L4</strong> and <strong>L5</strong> below 0.20&#x000a0;nm, whereas the ligand L3 led to slightly higher RMSD of about 0.25&#x000a0;nm. In the case of protein RMSF analysis (<a href="/pmc/articles/PMC10907797/figure/fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig6" rid-ob="ob-fig6" co-legend-rid="lgnd_fig6"><span>Fig. 6</span></a>(C)), all the complexes show a low RMSF except for the <strong>L5</strong>-EthR complex, which shows a spike in the RMSF in the ligand binding site, which lies between 175 and 200 residues and correlates with the protein RMSD (<a href="/pmc/articles/PMC10907797/figure/fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig6" rid-ob="ob-fig6" co-legend-rid="lgnd_fig6"><span>Fig. 6</span></a>(A)). The radius of gyration for the ligand <strong>L4</strong>-EthR (<a href="/pmc/articles/PMC10907797/figure/fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig6" rid-ob="ob-fig6" co-legend-rid="lgnd_fig6"><span>Fig. 6</span></a>(D)) is significantly distant from other complexes at 1.88&#x000a0;nm for several instances during the 100 ns MD simulation. The hydrogen bond formation and its retention are some of the exciting aspects of the stability of any protein-ligand complex. <a href="/pmc/articles/PMC10907797/figure/fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig6" rid-ob="ob-fig6" co-legend-rid="lgnd_fig6"><span>Fig. 6</span></a>(E) shows hydrogen bond interaction between the molecules under study through the simulation. Ligands <strong>L1</strong>, <strong>L3</strong>, <strong>L4</strong> and <strong>L5</strong> show 2&#x02013;3 hydrogen bonds with the receptor residues throughout MD simulation. The MD simulation trajectories were further used to analyse the free energy landscape for these ligands. The principal component analysis (PCA) for the MD trajectories was calculated. Two PCAs were plotted against the free energy as shown in <a href="/pmc/articles/PMC10907797/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span>Fig. 7</span></a>, where the native ligand <strong>L5</strong> and docked ligand <strong>L2</strong> show the best results.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig6" co-legend-rid="lgnd_fig6"><a href="/pmc/articles/PMC10907797/figure/fig6/" target="figure" rid-figpopup="fig6" rid-ob="ob-fig6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140451516242304"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10907797_gr6.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 6" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10907797/bin/gr6.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140451516242304"><a target="object" rel="noopener" href="/pmc/articles/PMC10907797/figure/fig6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig6"><div><a class="figpopup" href="/pmc/articles/PMC10907797/figure/fig6/" target="figure" rid-figpopup="fig6" rid-ob="ob-fig6">Fig. 6</a></div><!--caption a7--><div class="caption"><p>Molecular dynamics simulation of the docked complexes; (A) Protein backbone RMSD, (B) Ligand RMSD of docked ligands following molecular dynamics simulation analysis, (C) RMSF of residues from the EthR protein-ligand complex over 100 ns molecular dynamics simulation, (D) Radius of gyration for the EthR protein when bound to the docked ligands, (E) Hydrogen bond contacts between ligands and the protein during the 100ns MD simulation duration.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig7" co-legend-rid="lgnd_fig7"><a href="/pmc/articles/PMC10907797/figure/fig7/" target="figure" rid-figpopup="fig7" rid-ob="ob-fig7"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140451463528192"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10907797_gr7.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 7" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10907797/bin/gr7.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140451463528192"><a target="object" rel="noopener" href="/pmc/articles/PMC10907797/figure/fig7/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig7"><div><a class="figpopup" href="/pmc/articles/PMC10907797/figure/fig7/" target="figure" rid-figpopup="fig7" rid-ob="ob-fig7">Fig. 7</a></div><!--caption a7--><div class="caption"><p>Free energy landscape for the protein-ligand interactions during the MD simulations following Principal Component Analysis (PCA) for the MD trajectories for the ligand-protein complexes of <strong>L1</strong> (Ethyl (2-amino-4-((4-fluorobenzyl)amino)phenyl)carbamate), <strong>L2</strong> (2-((2,2-Difluorobenzo [d] [<a href="#bib1" rid="bib1" class=" bibr popnode">1</a>,<a href="#bib3" rid="bib3" class=" bibr popnode">3</a>]dioxol-5-yl)amino)-2-oxoethyl (E)-3-(5-bromofuran-2-yl)acrylate), and <strong>L3</strong> (N-(2,3-Dihydrobenzo [b] [<a href="#bib1" rid="bib1" class=" bibr popnode">1</a>,<a href="#bib4" rid="bib4" class=" bibr popnode">4</a>]dioxin-6-yl)-4-(2-((4-fluorophenyl)amino)-2-oxoethoxy)-3-methoxybenzamide) were obtained as hit ligands from the Selleckchem database following molecular docking. <strong>L4</strong>, also known as Pretomanid ((<em>S</em>)-2-Nitro-6-((4-(trifluoromethoxy) benzyl)oxy)-6,7-dihydro-5H-imidazo [2,1-b] [<a href="#bib1" rid="bib1" class=" bibr popnode">1</a>,<a href="#bib3" rid="bib3" class=" bibr popnode">3</a>]oxazine) is potent antimycobacterial compound and <strong>L5</strong> (4-{3-[(phenylsulfonyl)amino] prop-1-yn-1-yl}-N-(3,3,3-trifluoropropyl)benzamide), is the co-crystallised ligand from the crystal structure of EthR (PDB: <a href="https://www.rcsb.org/structure/https://www.rcsb.org/4M3D" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">4M3D</a>).</p></div></div></div><p id="p0120">The docking calculations were validated by performing a self-docking experiment where we found an exceptionally low RMSD of 0.182&#x000a0;&#x000c5; to the crystal structure, this process involved docking and redocking of the native ligand. This validated protocol was used for docking studies to obtain ligands with high dock scores and low RMSDs. The ligands obtained from ADMET filters, the physicochemical properties filter, and the docking calculations were subjected to MDS. The MDS provided insight into the binding of these ligands to the target protein. The ligands <strong>L1</strong>, <strong>L2</strong> and <strong>L3</strong> were the top molecules from the docking studies. These ligands showed hydrogen bond interactions with the key residues in the EthR binding site. The ligand <strong>L1</strong>, also known as retigabine, is a known anticonvulsant therapeutic; it was found to bind the EthR binding site by forming hydrogen bond interactions with Asn176 and Asn107, which are key residues. It also shows the highest dock score of &#x02212;12.45 and binding energy of &#x02212;36.68&#x000a0;kcal/mol calculated by the MM-GBSA method, which is higher than the standard TB drug Pretomanid and the co-crystallised ligand for this protein. The MDS results for <strong>L1</strong> suggest the formation of a stable complex with low fluctuations over the period of MDS.</p><p id="p0125">The ligand <strong>L2</strong> also interacts with the EthR binding site with hydrogen bond interactions with Asn176 and Trp148, it also forms a stabilizing &#x003c0;-&#x003c0; interaction with the Phe110 and Trp103. <strong>L2</strong> docks with a score of &#x02212;10.21 and binding energy of &#x02212;32.57&#x000a0;kcal/mol. It also forms a stable complex with the binding site residues which is evident from its MDS-ligand RMSD which oscillates between 0.05 and 0.15&#x000a0;nm through the 100ns simulation. The free energy landscape (<a href="/pmc/articles/PMC10907797/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span>Fig. 7</span></a>) for the <strong>L2</strong> shows a deep well with low free energy for this complex suggesting its stability. The ligand <strong>L3</strong> shows the second-highest dock score of &#x02212;11.32 and a binding energy of &#x02212;36.17&#x000a0;kcal/mol. Visual examination of the <strong>L3</strong>-EthR docked complex shows hydrogen bond interaction with the key residues Asn179 and Ala91 and several hydrophobic interactions with Trp145, Leu87, Trp207 and Trp103. The ligand RMSD for <strong>L3</strong> was observed to be very stable compared to the rest of the ligands as it fluctuated within a range of 0.05&#x000a0;nm (<a href="/pmc/articles/PMC10907797/figure/fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig6" rid-ob="ob-fig6" co-legend-rid="lgnd_fig6"><span>Fig. 6</span></a>(B), green) and other parameters for this complex, like the protein RMSD, RMSF, and RoG, reflect the same stability. Furthermore, the hydrogen bonds formed are consistent through the 100 ns simulation, justifying its stable complex formation with the EthR. The standard drug molecule that came up from the set of 30 antiTb drug molecules was found to be <strong>L4</strong>, also known as Pretomanid, which is an antiTb prodrug molecule that is active against replicating and nonreplicating mycobacterium. <strong>L4</strong> binds with the EthR to form hydrogen bond interactions with the Asn179 and Tyr148, it forms the &#x003c0;-&#x003c0; interactions with Phe110 as in the case of <strong>L2</strong>. The dock score for the L4 is lowest at &#x02212;9.31 and binding energy at &#x02212;28.34&#x000a0;kcal/mol making it a weak binder compared to the newly found molecules. The MDS profile for <strong>L4</strong> was also found to be less stable than the rest of the ligands as it showed highly fluctuating ligand RMSD.</p><p id="p0130" class="p p-last">The co-crystallised ligand <strong>L5</strong> retained its crystallographic interactions with the residues Asn176, Asn179, Tyr148 and the &#x003c0;-&#x003c0; interactions with Phe110 and Trp103. It showed a docking score of &#x02212;9.46 and a binding energy of &#x02212;21.78&#x000a0;kcal/mol. L5 displayed a lower dock score and binding energy than the rest of the ligands. However, its MDS ligand RMSD shows a stable EthR interaction with RMSD between 0.05&#x000a0;nm and some fluctuations between 70 and 90 ns. This could result from its conformational exploration during the simulation as reflected in the RMSF plot. However, the hydrogen bond profile suggests stable interactions throughout the MDS. The free energy landscape (<a href="/pmc/articles/PMC10907797/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span>Fig. 7</span></a> (L1-L5)) also supports our findings; it shows a large area of low energy through the simulation (<a href="/pmc/articles/PMC10907797/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span>Fig. 7</span></a> L5), which is best compared to the rest of the ligands and obvious given it being the native bond ligand to this protein. The most exciting fact this study has highlighted is the structural features. L6 in <a href="/pmc/articles/PMC10907797/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a> represents some common structural features, such as the fused ring systems, benzamide, and/or amide bonds, which are crucial structural features required in EthR inhibitors.</p></div></div><div id="sec4" class="tsec sec"><h2 class="head no_bottom_margin" id="sec4title">4. Conclusions</h2><p id="p0135" class="p p-first-last">Tuberculosis has been a challenge to the world since prehistoric times, and with the advent of drug-resistant strains, it has become more challenging to treat this infection. The recent Covid pandemic, climate change, and socioeconomic and geopolitical conditions throughout the globe are driving us away from the solution for this curable infection. In this report, we have attempted and successfully identified three new molecules from the drug repurposing library that can be used for targeting EthR protein. These molecules were obtained after rigorous filtering of the database for their physicochemical, toxicological properties and safety, respectively. Compounds Ethyl (2-amino-4-((4-fluorobenzyl)amino)phenyl)carbamate) (<strong>L1</strong>), 2-((2,2-Difluorobenzo [d] [<a href="#bib1" rid="bib1" class=" bibr popnode">1</a>,<a href="#bib3" rid="bib3" class=" bibr popnode">3</a>]dioxol-5-yl)amino)-2-oxoethyl (E)-3-(5-bromofuran-2-yl)acrylate (<strong>L2</strong>), and N-(2,3-Dihydrobenzo [b] [<a href="#bib1" rid="bib1" class=" bibr popnode">1</a>,<a href="#bib4" rid="bib4" class=" bibr popnode">4</a>]dioxin-6-yl)-4-(2-((4-fluorophenyl)amino)-2-oxoethoxy)-3-methoxybenzamide (<strong>L3</strong>) are potential EthR inhibitor that would boost the activity of ETH in inhibiting the mycobacteria can help further improve the treatment of TB. All identified ligands are thought to be potential compounds that should be investigated for evaluating their biological activity to determine the inhibitory efficacy profile to aid in the anti-tubercular therapy targeting the EthR protein. Future therapeutic approaches may also be made possible by deriving of information on the identified hit compounds' activity and relationship for a better understanding of chemical space optimization. Furthermore, reaction kinetics investigations and/or the cellular thermal shift/melt assay can be used to assess the binding and unbinding pattern mechanism of suggested drug-like candidates for EthR inhibitors/modulators. The screened potential inhibitors/modulators for the EthR protein of Mtb could also pave for other therapeutic modalities in the future.</p></div><div id="sec5" class="tsec sec"><h2 class="head no_bottom_margin" id="sec5title">Data availability</h2><p id="p0140" class="p p-first-last">The data related to this work could be provided on request.</p></div><div id="sec6" class="tsec sec"><h2 class="head no_bottom_margin" id="sec6title">CRediT authorship contribution statement</h2><p id="p0145" class="p p-first-last"><strong>Rupesh V. Chikhale:</strong> Writing &#x02013; review &#x00026; editing, Writing &#x02013; original draft, Supervision, Methodology, Formal analysis, Conceptualization. <strong>Gaber E. Eldesoky:</strong> Writing &#x02013; original draft, Conceptualization. <strong>Mahima Sudhir Kolpe:</strong> Visualization, Validation, Formal analysis, Data curation. <strong>Vikramsinh Sardarsinh Suryawanshi:</strong> Investigation, Formal analysis, Data curation. <strong>Pritee Chunarkar Patil:</strong> Writing &#x02013; original draft, Resources, Conceptualization. <strong>Shovonlal Bhowmick:</strong> Writing &#x02013; review &#x00026; editing, Writing &#x02013; original draft, Methodology, Investigation, Formal analysis, Conceptualization.</p></div><div id="sec-a.q.g" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.q.gtitle">Declaration of competing interest</h2><p id="p0150" class="p p-first-last">The authors declare the following financial interests/personal relationships which may be considered as potential competing interests:Gaber E. Eldesoky reports administrative support and statistical analysis were provided by Chemistry Department, College of Science, King Saud University, Riyadh 11451, Saudi Arabia. Dr Rupesh Chikhale is an associate editor to the Pharmaceutical Sciences Section of the Heliyon Journal. If there are other authors, they declare that they have no known competing financial interests or personal relationships that could have appeared to influence the work reported in this paper.</p></div><div id="ack0010" class="tsec sec"><h2 class="head no_bottom_margin" id="ack0010title">Acknowledgments</h2><div class="sec"><p id="p0155">The authors are grateful to the Researchers Supporting Project No. (RSP2023R161), King Saud University, Riyadh, Saudi Arabia.</p></div></div><div id="cebib0010" class="tsec sec"><h2 class="head no_bottom_margin" id="cebib0010title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="bib1">1. <span class="element-citation" id="sref1">Gordon S.V., Parish T. Microbe profile: Mycobacterium tuberculosis: humanity's deadly microbial foe. <span><span class="ref-journal">Microbiology (United Kingdom) </span>2018;<span class="ref-vol">164</span>:437–439. doi: 10.1099/MIC.0.000601/CITE/REFWORKS.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29465344" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1099%2FMIC.0.000601%2FCITE%2FREFWORKS" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology+(United+Kingdom)&amp;title=Microbe+profile:+Mycobacterium+tuberculosis:+humanity's+deadly+microbial+foe&amp;author=S.V.+Gordon&amp;author=T.+Parish&amp;volume=164&amp;publication_year=2018&amp;pages=437-439&amp;doi=10.1099/MIC.0.000601/CITE/REFWORKS&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib2">2. <span class="element-citation" id="sref2">Comas I., Coscolla M., Luo T., Borrell S., Holt K.E., Kato-Maeda M., Parkhill J., Malla B., Berg S., Thwaites G., et al. Out-of-Africa migration and neolithic coexpansion of Mycobacterium tuberculosis with modern humans. <span><span class="ref-journal">Nat. Genet. </span>2013;<span class="ref-vol">45</span>:1176–1182. doi: 10.1038/ng.2744.</span> 45:10 2013. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3800747/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23995134" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fng.2744" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Out-of-Africa+migration+and+neolithic+coexpansion+of+Mycobacterium+tuberculosis+with+modern+humans&amp;author=I.+Comas&amp;author=M.+Coscolla&amp;author=T.+Luo&amp;author=S.+Borrell&amp;author=K.E.+Holt&amp;volume=45&amp;publication_year=2013&amp;pages=1176-1182&amp;pmid=23995134&amp;doi=10.1038/ng.2744&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib3">3. <span class="element-citation" id="sref3">Hacker K., Anies M., Folb B.L., Zallman L. Barriers to Health care for undocumented immigrants: a literature review. <span><span class="ref-journal">Risk Manag. Healthc. Pol. </span>2015;<span class="ref-vol">8</span>:175. doi: 10.2147/RMHP.S70173.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4634824/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26586971" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.2147%2FRMHP.S70173" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Risk+Manag.+Healthc.+Pol.&amp;title=Barriers+to+Health+care+for+undocumented+immigrants:+a+literature+review&amp;author=K.+Hacker&amp;author=M.+Anies&amp;author=B.L.+Folb&amp;author=L.+Zallman&amp;volume=8&amp;publication_year=2015&amp;pages=175&amp;doi=10.2147/RMHP.S70173&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib4">4. <span class="mixed-citation" id="sref4">Tuberculosis Available online: <a href="https://www.who.int/news-room/fact-sheets/detail/tuberculosis" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">https://www.who.int/news-room/fact-sheets/detail/tuberculosis</a> (accessed on 7 October 2023).</span></div><div class="ref-cit-blk half_rhythm" id="bib5">5. <span class="element-citation" id="sref5">Dean A.S., Tosas Auguet O., Glaziou P., Zignol M., Ismail N., Kasaeva T., Floyd K. 25 Years of surveillance of drug-resistant tuberculosis: achievements, challenges, and way forward. <span><span class="ref-journal">Lancet Infect. Dis. </span>2022;<span class="ref-vol">22</span>:e191–e196. doi: 10.1016/S1473-3099(21)00808-2.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8893725/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35248168" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FS1473-3099(21)00808-2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Infect.+Dis.&amp;title=25+Years+of+surveillance+of+drug-resistant+tuberculosis:+achievements,+challenges,+and+way+forward&amp;author=A.S.+Dean&amp;author=O.+Tosas+Auguet&amp;author=P.+Glaziou&amp;author=M.+Zignol&amp;author=N.+Ismail&amp;volume=22&amp;publication_year=2022&amp;pages=e191-e196&amp;pmid=35248168&amp;doi=10.1016/S1473-3099(21)00808-2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib6">6. <span class="mixed-citation" id="sref6">2.3 Drug-Resistant TB Available online: <a href="https://www.who.int/teams/global-tuberculosis-programme/tb-reports/global-tuberculosis-report-2022/tb-disease-burden/2-3-drug-resistant-tb" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">https://www.who.int/teams/global-tuberculosis-programme/tb-reports/global-tuberculosis-report-2022/tb-disease-burden/2-3-drug-resistant-tb</a> (accessed on 7 October 2023).</span></div><div class="ref-cit-blk half_rhythm" id="bib7">7. <span class="element-citation" id="sref7">Mitchison D.A.  vol. 171. 2012. pp. 699–706. (The Diagnosis and Therapy of Tuberculosis during the Past 100 Years). 10.1164/RCCM.200411-1603OE. [<a href="https://pubmed.ncbi.nlm.nih.gov/15709048" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1164%2Frccm.200411-1603OE" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Mitchison+D.A.+The+Diagnosis+and+Therapy+of+Tuberculosis+during+the+Past+100+Years+vol.+171+2012+699+706+10.1164/rccm.200411-1603OE+10.1164/RCCM.200411-1603OE+" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib8">8. <span class="element-citation" id="sref8">Chikhale R.V., Barmade M.A., Murumkar P.R., Yadav M.R. Overview of the development of DprE1 inhibitors for combating the menace of tuberculosis. <span><span class="ref-journal">J. Med. Chem. </span>2018;<span class="ref-vol">61</span>:8563–8593. doi: 10.1021/ACS.JMEDCHEM.8B00281/ASSET/IMAGES/LARGE/JM-2018-00281W_0015.JPEG.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29851474" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2FACS.JMEDCHEM.8B00281%2FASSET%2FIMAGES%2FLARGE%2FJM-2018-00281W_0015.JPEG" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Chem.&amp;title=Overview+of+the+development+of+DprE1+inhibitors+for+combating+the+menace+of+tuberculosis&amp;author=R.V.+Chikhale&amp;author=M.A.+Barmade&amp;author=P.R.+Murumkar&amp;author=M.R.+Yadav&amp;volume=61&amp;publication_year=2018&amp;pages=8563-8593&amp;pmid=29851474&amp;doi=10.1021/ACS.JMEDCHEM.8B00281/ASSET/IMAGES/LARGE/JM-2018-00281W_0015.JPEG&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib9">9. <span class="mixed-citation" id="sref9">Global Tuberculosis Programme Available online: <a href="https://www.who.int/teams/global-tuberculosis-programme/diagnosis-treatment/treatment-of-drug-resistant-tb/types-of-tb-drug-resistance" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">https://www.who.int/teams/global-tuberculosis-programme/diagnosis-treatment/treatment-of-drug-resistant-tb/types-of-tb-drug-resistance</a> (accessed on 7 October 2023).</span></div><div class="ref-cit-blk half_rhythm" id="bib10">10. <span class="element-citation" id="sref10">Torres N.M.C., Rodr&#x000ed;guez J.J.Q., Andrade P.S.P., Arriaga M.B., Netto E.M. Factors predictive of the success of tuberculosis treatment: a systematic review with meta-analysis. <span><span class="ref-journal">PLoS One. </span>2019;<span class="ref-vol">14</span> doi: 10.1371/JOURNAL.PONE.0226507.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6934297/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31881023" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2FJOURNAL.PONE.0226507" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Factors+predictive+of+the+success+of+tuberculosis+treatment:+a+systematic+review+with+meta-analysis&amp;author=N.M.C.+Torres&amp;author=J.J.Q.+Rodr&#x000ed;guez&amp;author=P.S.P.+Andrade&amp;author=M.B.+Arriaga&amp;author=E.M.+Netto&amp;volume=14&amp;publication_year=2019&amp;doi=10.1371/JOURNAL.PONE.0226507&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib11">11. <span class="element-citation" id="sref11">Tait D.R., Hatherill M., Van Der Meeren O., Ginsberg A.M., Van Brakel E., Salaun B., Scriba T.J., Akite E.J., Ayles H.M., Bollaerts A., et al. Final analysis of a trial of M72/AS01 E vaccine to prevent tuberculosis. <span><span class="ref-journal">N. Engl. J. Med. </span>2019;<span class="ref-vol">381</span>:2429–2439. doi: 10.1056/NEJMOA1909953/SUPPL_FILE/NEJMOA1909953_DATA-SHARING.PDF.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31661198" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1056%2FNEJMOA1909953%2FSUPPL_FILE%2FNEJMOA1909953_DATA-SHARING.PDF" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med.&amp;title=Final+analysis+of+a+trial+of+M72/AS01+E+vaccine+to+prevent+tuberculosis&amp;author=D.R.+Tait&amp;author=M.+Hatherill&amp;author=O.+Van+Der+Meeren&amp;author=A.M.+Ginsberg&amp;author=E.+Van+Brakel&amp;volume=381&amp;publication_year=2019&amp;pages=2429-2439&amp;pmid=31661198&amp;doi=10.1056/NEJMOA1909953/SUPPL_FILE/NEJMOA1909953_DATA-SHARING.PDF&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib12">12. <span class="element-citation" id="sref12">Ignatius E.H., Dooley K.E. New drugs for the treatment of tuberculosis. <span><span class="ref-journal">Clin. Chest Med. </span>2019;<span class="ref-vol">40</span>:811. doi: 10.1016/J.CCM.2019.08.001.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9178517/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31731986" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FJ.CCM.2019.08.001" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Chest+Med.&amp;title=New+drugs+for+the+treatment+of+tuberculosis&amp;author=E.H.+Ignatius&amp;author=K.E.+Dooley&amp;volume=40&amp;publication_year=2019&amp;pages=811&amp;pmid=31731986&amp;doi=10.1016/J.CCM.2019.08.001&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib13">13. <span class="element-citation" id="sref13">Shetye G.S., Franzblau S.G., Cho S. New tuberculosis drug targets, their inhibitors, and potential therapeutic impact. <span><span class="ref-journal">Transl. Res. </span>2020;<span class="ref-vol">220</span>:68–97. doi: 10.1016/J.TRSL.2020.03.007.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32275897" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FJ.TRSL.2020.03.007" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transl.+Res.&amp;title=New+tuberculosis+drug+targets,+their+inhibitors,+and+potential+therapeutic+impact&amp;author=G.S.+Shetye&amp;author=S.G.+Franzblau&amp;author=S.+Cho&amp;volume=220&amp;publication_year=2020&amp;pages=68-97&amp;pmid=32275897&amp;doi=10.1016/J.TRSL.2020.03.007&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib14">14. <span class="element-citation" id="sref14">Mapari M., Bhole R.P., Khedekar P.B., Chikhale R.V. Challenges in targeting mycobacterial ATP synthase: the known and beyond. <span><span class="ref-journal">J. Mol. Struct. </span>2022;<span class="ref-vol">1247</span> doi: 10.1016/J.MOLSTRUC.2021.131331.</span> [<a href="//doi.org/10.1016%2FJ.MOLSTRUC.2021.131331" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Mol.+Struct.&amp;title=Challenges+in+targeting+mycobacterial+ATP+synthase:+the+known+and+beyond&amp;author=M.+Mapari&amp;author=R.P.+Bhole&amp;author=P.B.+Khedekar&amp;author=R.V.+Chikhale&amp;volume=1247&amp;publication_year=2022&amp;doi=10.1016/J.MOLSTRUC.2021.131331&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib15">15. <span class="element-citation" id="sref15">Umare M.D., Khedekar P.B., Chikhale R.V. Mycobacterial membrane protein large 3 (MmpL3) inhibitors: a promising approach to combat tuberculosis. <span><span class="ref-journal">ChemMedChem. </span>2021;<span class="ref-vol">16</span>:3136–3148. doi: 10.1002/CMDC.202100359.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34288519" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2FCMDC.202100359" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=ChemMedChem&amp;title=Mycobacterial+membrane+protein+large+3+(MmpL3)+inhibitors:+a+promising+approach+to+combat+tuberculosis&amp;author=M.D.+Umare&amp;author=P.B.+Khedekar&amp;author=R.V.+Chikhale&amp;volume=16&amp;publication_year=2021&amp;pages=3136-3148&amp;pmid=34288519&amp;doi=10.1002/CMDC.202100359&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib16">16. <span class="element-citation" id="sref16">Prasad M.S., Bhole R.P., Khedekar P.B., Chikhale R.V. Mycobacterium enoyl acyl carrier protein reductase (InhA): a key target for antitubercular drug discovery. <span><span class="ref-journal">Bioorg. Chem. </span>2021;<span class="ref-vol">115</span> doi: 10.1016/J.BIOORG.2021.105242.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34392175" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FJ.BIOORG.2021.105242" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioorg.+Chem.&amp;title=Mycobacterium+enoyl+acyl+carrier+protein+reductase+(InhA):+a+key+target+for+antitubercular+drug+discovery&amp;author=M.S.+Prasad&amp;author=R.P.+Bhole&amp;author=P.B.+Khedekar&amp;author=R.V.+Chikhale&amp;volume=115&amp;publication_year=2021&amp;doi=10.1016/J.BIOORG.2021.105242&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib17">17. <span class="element-citation" id="sref17">Banerjee A., Dubnau E., Quemard A., Balasubramanian V., Um K.S., Wilson T., Collins D., De Lisle G., Jacobs W.R.InhA. A gene encoding a target for isoniazid and ethionamide in Mycobacterium tuberculosis. <span><span class="ref-journal">Science. </span>1994;<span class="ref-vol">263</span>:227–230. doi: 10.1126/SCIENCE.8284673.</span> 1979. [<a href="https://pubmed.ncbi.nlm.nih.gov/8284673" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2FSCIENCE.8284673" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+gene+encoding+a+target+for+isoniazid+and+ethionamide+in+Mycobacterium+tuberculosis&amp;author=A.+Banerjee&amp;author=E.+Dubnau&amp;author=A.+Quemard&amp;author=V.+Balasubramanian&amp;author=K.S.+Um&amp;volume=263&amp;publication_year=1994&amp;pages=227-230&amp;pmid=8284673&amp;doi=10.1126/SCIENCE.8284673&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib18">18. <span class="element-citation" id="sref18">Baulard A.R., Betts J.C., Engohang-Ndong J., Quan S., McAdam R.A., Brennan P.J., Locht C., Besra G.S. Activation of the pro-drug ethionamide is regulated in mycobacteria. <span><span class="ref-journal">J. Biol. Chem. </span>2000;<span class="ref-vol">275</span>:28326–28331. doi: 10.1074/JBC.M003744200.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10869356" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1074%2FJBC.M003744200" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Activation+of+the+pro-drug+ethionamide+is+regulated+in+mycobacteria&amp;author=A.R.+Baulard&amp;author=J.C.+Betts&amp;author=J.+Engohang-Ndong&amp;author=S.+Quan&amp;author=R.A.+McAdam&amp;volume=275&amp;publication_year=2000&amp;pages=28326-28331&amp;pmid=10869356&amp;doi=10.1074/JBC.M003744200&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib19">19. <span class="element-citation" id="sref19">Vannelli T.A., Dykman A., Ortiz De Montellano P.R. The antituberculosis drug ethionamide is activated by a flavoprotein monooxygenase. <span><span class="ref-journal">J. Biol. Chem. </span>2002;<span class="ref-vol">277</span>:12824–12829. doi: 10.1074/JBC.M110751200.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11823459" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1074%2FJBC.M110751200" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=The+antituberculosis+drug+ethionamide+is+activated+by+a+flavoprotein+monooxygenase&amp;author=T.A.+Vannelli&amp;author=A.+Dykman&amp;author=P.R.+Ortiz+De+Montellano&amp;volume=277&amp;publication_year=2002&amp;pages=12824-12829&amp;pmid=11823459&amp;doi=10.1074/JBC.M110751200&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib20">20. <span class="element-citation" id="sref20">Willand N., Diri&#x000e9; B., Carette X., Bifani P., Singhal A., Desroses M., Leroux F., Willery E., Mathys V., D&#x000e9;prez-Poulain R., et al. Synthetic EthR inhibitors boost antituberculous activity of ethionamide. <span><span class="ref-journal">Nat. Med. </span>2009;<span class="ref-vol">15</span>:537–544. doi: 10.1038/NM.1950.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19412174" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2FNM.1950" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Med.&amp;title=Synthetic+EthR+inhibitors+boost+antituberculous+activity+of+ethionamide&amp;author=N.+Willand&amp;author=B.+Diri&#x000e9;&amp;author=X.+Carette&amp;author=P.+Bifani&amp;author=A.+Singhal&amp;volume=15&amp;publication_year=2009&amp;pages=537-544&amp;pmid=19412174&amp;doi=10.1038/NM.1950&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib21">21. <span class="element-citation" id="sref21">Flipo M., Desroses M., Lecat-Guillet N., Diri&#x000e9; B., Carette X., Leroux F., Piveteau C., Demirkaya F., Lens Z., Rucktooa P., et al. Ethionamide boosters: synthesis, biological activity, and structure-activity relationships of a series of 1,2,4-oxadiazole EthR inhibitors. <span><span class="ref-journal">J. Med. Chem. </span>2011;<span class="ref-vol">54</span>:2994–3010. doi: 10.1021/JM200076A/ASSET/IMAGES/LARGE/JM-2011-00076A_0012.JPEG.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21417236" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2FJM200076A%2FASSET%2FIMAGES%2FLARGE%2FJM-2011-00076A_0012.JPEG" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Chem.&amp;title=Ethionamide+boosters:+synthesis,+biological+activity,+and+structure-activity+relationships+of+a+series+of+1,2,4-oxadiazole+EthR+inhibitors&amp;author=M.+Flipo&amp;author=M.+Desroses&amp;author=N.+Lecat-Guillet&amp;author=B.+Diri&#x000e9;&amp;author=X.+Carette&amp;volume=54&amp;publication_year=2011&amp;pages=2994-3010&amp;pmid=21417236&amp;doi=10.1021/JM200076A/ASSET/IMAGES/LARGE/JM-2011-00076A_0012.JPEG&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib22">22. <span class="element-citation" id="sref22">Flipo M., Desroses M., Lecat-Guillet N., Villemagne B., Blondiaux N., Leroux F., Piveteau C., Mathys V., Flament M.P., Siepmann J., et al. Ethionamide boosters. 2. Combining bioisosteric replacement and structure-based drug design to solve pharmacokinetic issues in a series of potent 1,2,4-oxadiazole EthR inhibitors. <span><span class="ref-journal">J. Med. Chem. </span>2012;<span class="ref-vol">55</span>:68–83. doi: 10.1021/JM200825U/SUPPL_FILE/JM200825U_SI_001.PDF.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22098589" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2FJM200825U%2FSUPPL_FILE%2FJM200825U_SI_001.PDF" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Chem.&amp;title=Ethionamide+boosters.+2.+Combining+bioisosteric+replacement+and+structure-based+drug+design+to+solve+pharmacokinetic+issues+in+a+series+of+potent+1,2,4-oxadiazole+EthR+inhibitors&amp;author=M.+Flipo&amp;author=M.+Desroses&amp;author=N.+Lecat-Guillet&amp;author=B.+Villemagne&amp;author=N.+Blondiaux&amp;volume=55&amp;publication_year=2012&amp;pages=68-83&amp;pmid=22098589&amp;doi=10.1021/JM200825U/SUPPL_FILE/JM200825U_SI_001.PDF&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib23">23. <span class="element-citation" id="sref23">Villemagne B., Flipo M., Blondiaux N., Crauste C., Malaquin S., Leroux F., Piveteau C., Villeret V., Brodin P., Villoutreix B.O., et al. Ligand efficiency driven design of new inhibitors of Mycobacterium tuberculosis transcriptional repressor EthR using fragment growing, merging, and linking approaches. <span><span class="ref-journal">J. Med. Chem. </span>2014;<span class="ref-vol">57</span>:4876–4888. doi: 10.1021/JM500422B/SUPPL_FILE/JM500422B_SI_001.PDF.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24818704" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2FJM500422B%2FSUPPL_FILE%2FJM500422B_SI_001.PDF" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Chem.&amp;title=Ligand+efficiency+driven+design+of+new+inhibitors+of+Mycobacterium+tuberculosis+transcriptional+repressor+EthR+using+fragment+growing,+merging,+and+linking+approaches&amp;author=B.+Villemagne&amp;author=M.+Flipo&amp;author=N.+Blondiaux&amp;author=C.+Crauste&amp;author=S.+Malaquin&amp;volume=57&amp;publication_year=2014&amp;pages=4876-4888&amp;pmid=24818704&amp;doi=10.1021/JM500422B/SUPPL_FILE/JM500422B_SI_001.PDF&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib24">24. <span class="mixed-citation" id="sref24">Selleckchem.Com - Bioactive Compounds Expert (Bioactive Compounds,Compound Libraries) Available online: <a href="https://www.selleckchem.com/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">https://www.selleckchem.com/</a>(accessed on 17 October 2023).</span></div><div class="ref-cit-blk half_rhythm" id="bib25">25. <span class="mixed-citation" id="sref25">Open Babel Available online: <a href="https://openbabel.org/wiki/Main_Page" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">https://openbabel.org/wiki/Main_Page</a> (accessed on 17 October 2023).</span></div><div class="ref-cit-blk half_rhythm" id="bib26">26. <span class="element-citation" id="sref26">Eberhardt J., Santos-Martins D., Tillack A.F., Forli S. AutoDock Vina 1.2.0: new docking methods, expanded force field, and Python bindings. <span><span class="ref-journal">J. Chem. Inf. Model. </span>2021;<span class="ref-vol">61</span>:3891–3898. doi: 10.1021/ACS.JCIM.1C00203/SUPPL_FILE/CI1C00203_SI_002.ZIP.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10683950/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34278794" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2FACS.JCIM.1C00203%2FSUPPL_FILE%2FCI1C00203_SI_002.ZIP" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Chem.+Inf.+Model.&amp;title=AutoDock+Vina+1.2.0:+new+docking+methods,+expanded+force+field,+and+Python+bindings&amp;author=J.+Eberhardt&amp;author=D.+Santos-Martins&amp;author=A.F.+Tillack&amp;author=S.+Forli&amp;volume=61&amp;publication_year=2021&amp;pages=3891-3898&amp;pmid=34278794&amp;doi=10.1021/ACS.JCIM.1C00203/SUPPL_FILE/CI1C00203_SI_002.ZIP&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib27">27. <span class="element-citation" id="sref27">Villemagne B., Flipo M., Blondiaux N., Crauste C., Malaquin S., Leroux F., Piveteau C., Villeret V., Brodin P., Villoutreix B.O., et al. Ligand efficiency driven design of new inhibitors of Mycobacterium tuberculosis transcriptional repressor EthR using fragment growing, merging, and linking approaches. <span><span class="ref-journal">J. Med. Chem. </span>2014;<span class="ref-vol">57</span>:4876–4888. doi: 10.1021/JM500422B/SUPPL_FILE/JM500422B_SI_001.PDF.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24818704" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2FJM500422B%2FSUPPL_FILE%2FJM500422B_SI_001.PDF" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Chem.&amp;title=Ligand+efficiency+driven+design+of+new+inhibitors+of+Mycobacterium+tuberculosis+transcriptional+repressor+EthR+using+fragment+growing,+merging,+and+linking+approaches&amp;author=B.+Villemagne&amp;author=M.+Flipo&amp;author=N.+Blondiaux&amp;author=C.+Crauste&amp;author=S.+Malaquin&amp;volume=57&amp;publication_year=2014&amp;pages=4876-4888&amp;pmid=24818704&amp;doi=10.1021/JM500422B/SUPPL_FILE/JM500422B_SI_001.PDF&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib28">28. <span class="mixed-citation" id="sref28">RCSB PDB: Homepage Available online: <a href="https://www.rcsb.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">https://www.rcsb.org/</a>(accessed on 23 September 2023)..</span></div><div class="ref-cit-blk half_rhythm" id="bib29">29. <span class="mixed-citation" id="sref29">AutoDockTools &#x02013; AutoDock Suite Available online: <a href="https://autodocksuite.scripps.edu/adt/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">https://autodocksuite.scripps.edu/adt/</a>(accessed on 17 October 2023).</span></div><div class="ref-cit-blk half_rhythm" id="bib30">30. <span class="element-citation" id="sref30">Ellingson S.R., Smith J.C., Baudry J.VinaMPI. Facilitating multiple receptor high-throughput virtual docking on high-performance computers. <span><span class="ref-journal">J. Comput. Chem. </span>2013;<span class="ref-vol">34</span>:2212–2221. doi: 10.1002/JCC.23367.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23813626" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2FJCC.23367" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Comput.+Chem.&amp;title=Facilitating+multiple+receptor+high-throughput+virtual+docking+on+high-performance+computers&amp;author=S.R.+Ellingson&amp;author=J.C.+Smith&amp;author=J.VinaMPI.+Baudry&amp;volume=34&amp;publication_year=2013&amp;pages=2212-2221&amp;pmid=23813626&amp;doi=10.1002/JCC.23367&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib31">31. <span class="element-citation" id="sref31">Adasme M.F., Linnemann K.L., Bolz S.N., Kaiser F., Salentin S., Haupt V.J., Schroeder M. Plip 2021: expanding the scope of the protein&#x02013;ligand interaction profiler to DNA and RNA. <span><span class="ref-journal">Nucleic Acids Res. </span>2021;<span class="ref-vol">49</span>:W530–W534. doi: 10.1093/NAR/GKAB294.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8262720/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33950214" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2FNAR%2FGKAB294" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Plip+2021:+expanding+the+scope+of+the+protein&#x02013;ligand+interaction+profiler+to+DNA+and+RNA&amp;author=M.F.+Adasme&amp;author=K.L.+Linnemann&amp;author=S.N.+Bolz&amp;author=F.+Kaiser&amp;author=S.+Salentin&amp;volume=49&amp;publication_year=2021&amp;pages=W530-W534&amp;pmid=33950214&amp;doi=10.1093/NAR/GKAB294&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib32">32. <span class="element-citation" id="sref32">Pires D.E.V., Blundell T.L., Ascher D.B. PkCSM: predicting small-molecule pharmacokinetic and toxicity properties using graph-based signatures. <span><span class="ref-journal">J. Med. Chem. </span>2015;<span class="ref-vol">58</span>:4066–4072. doi: 10.1021/ACS.JMEDCHEM.5B00104/SUPPL_FILE/JM5B00104_SI_001.PDF.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4434528/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25860834" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2FACS.JMEDCHEM.5B00104%2FSUPPL_FILE%2FJM5B00104_SI_001.PDF" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Chem.&amp;title=PkCSM:+predicting+small-molecule+pharmacokinetic+and+toxicity+properties+using+graph-based+signatures&amp;author=D.E.V.+Pires&amp;author=T.L.+Blundell&amp;author=D.B.+Ascher&amp;volume=58&amp;publication_year=2015&amp;pages=4066-4072&amp;pmid=25860834&amp;doi=10.1021/ACS.JMEDCHEM.5B00104/SUPPL_FILE/JM5B00104_SI_001.PDF&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib33">33. <span class="element-citation" id="sref33">Dara S., Dhamercherla S., Jadav S.S., Babu C.M., Ahsan M.J. Machine learning in drug discovery: a review. <span><span class="ref-journal">Artif. Intell. Rev. </span>2022;<span class="ref-vol">55</span>:1947. doi: 10.1007/S10462-021-10058-4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8356896/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34393317" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2FS10462-021-10058-4" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Artif.+Intell.+Rev.&amp;title=Machine+learning+in+drug+discovery:+a+review&amp;author=S.+Dara&amp;author=S.+Dhamercherla&amp;author=S.S.+Jadav&amp;author=C.M.+Babu&amp;author=M.J.+Ahsan&amp;volume=55&amp;publication_year=2022&amp;pages=1947&amp;pmid=34393317&amp;doi=10.1007/S10462-021-10058-4&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib34">34. <span class="element-citation" id="sref34">Pu L., Naderi M., Liu T., Wu H.C., Mukhopadhyay S., Brylinski M. EToxPred: a machine learning-based approach to estimate the toxicity of drug candidates 03 chemical Sciences 0305 organic Chemistry 03 chemical Sciences 0304 medicinal and biomolecular Chemistry. <span><span class="ref-journal">BMC Pharmacol Toxicol. </span>2019;<span class="ref-vol">20</span>:1–15. doi: 10.1186/S40360-018-0282-6/FIGURES/10.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6325674/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30621790" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2FS40360-018-0282-6%2FFIGURES%2F10" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Pharmacol+Toxicol&amp;title=EToxPred:+a+machine+learning-based+approach+to+estimate+the+toxicity+of+drug+candidates+03+chemical+Sciences+0305+organic+Chemistry+03+chemical+Sciences+0304+medicinal+and+biomolecular+Chemistry&amp;author=L.+Pu&amp;author=M.+Naderi&amp;author=T.+Liu&amp;author=H.C.+Wu&amp;author=S.+Mukhopadhyay&amp;volume=20&amp;publication_year=2019&amp;pages=1-15&amp;pmid=30611293&amp;doi=10.1186/S40360-018-0282-6/FIGURES/10&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib35">35. <span class="element-citation" id="sref35">Talevi A., Morales J.F., Hather G., Podichetty J.T., Kim S., Bloomingdale P.C., Kim S., Burton J., Brown J.D., Winterstein A.G., et al. Machine learning in drug discovery and development Part 1: a primer. <span><span class="ref-journal">CPT Pharmacometrics Syst. Pharmacol. </span>2020;<span class="ref-vol">9</span>:129–142. doi: 10.1002/PSP4.12491.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7080529/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31905263" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2FPSP4.12491" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=CPT+Pharmacometrics+Syst.+Pharmacol.&amp;title=Machine+learning+in+drug+discovery+and+development+Part+1:+a+primer&amp;author=A.+Talevi&amp;author=J.F.+Morales&amp;author=G.+Hather&amp;author=J.T.+Podichetty&amp;author=S.+Kim&amp;volume=9&amp;publication_year=2020&amp;pages=129-142&amp;pmid=31905263&amp;doi=10.1002/PSP4.12491&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib36">36. <span class="element-citation" id="sref36">Ghazwani M., Begum M.Y. Computational intelligence modeling of hyoscine drug solubility and solvent density in supercritical processing: gradient boosting, Extra trees, and random forest models. <span><span class="ref-journal">Sci. Rep. </span>2023;<span class="ref-vol">13</span>:1. doi: 10.1038/s41598-023-37232-8.</span> 2023, 13, 1&#x02013;11. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10284815/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37344621" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41598-023-37232-8" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Rep.&amp;title=Computational+intelligence+modeling+of+hyoscine+drug+solubility+and+solvent+density+in+supercritical+processing:+gradient+boosting,+Extra+trees,+and+random+forest+models&amp;author=M.+Ghazwani&amp;author=M.Y.+Begum&amp;volume=13&amp;publication_year=2023&amp;pages=1&amp;pmid=36593249&amp;doi=10.1038/s41598-023-37232-8&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib37">37. <span class="element-citation" id="sref37">Zoete V., Cuendet M.A., Grosdidier A., Michielin O. SwissParam. A fast force field generation tool for small organic molecules. <span><span class="ref-journal">J. Comput. Chem. </span>2011;<span class="ref-vol">32</span>:2359–2368. doi: 10.1002/JCC.21816.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21541964" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2FJCC.21816" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Comput.+Chem.&amp;title=A+fast+force+field+generation+tool+for+small+organic+molecules&amp;author=V.+Zoete&amp;author=M.A.+Cuendet&amp;author=A.+Grosdidier&amp;author=O.+SwissParam+Michielin&amp;volume=32&amp;publication_year=2011&amp;pages=2359-2368&amp;pmid=21541964&amp;doi=10.1002/JCC.21816&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib38">38. <span class="element-citation" id="sref38">Essmann U., Perera L., Berkowitz M.L., Darden T., Lee H., Pedersen L.G. A smooth Particle Mesh Ewald method. <span><span class="ref-journal">J. Chem. Phys. </span>1995 doi: 10.1063/1.470117.</span> [<a href="//doi.org/10.1063%2F1.470117" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Chem.+Phys.&amp;title=A+smooth+Particle+Mesh+Ewald+method&amp;author=U.+Essmann&amp;author=L.+Perera&amp;author=M.L.+Berkowitz&amp;author=T.+Darden&amp;author=H.+Lee&amp;publication_year=1995&amp;doi=10.1063/1.470117&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib39">39. <span class="element-citation" id="sref39">Van Der Spoel D., Lindahl E., Hess B., Groenhof G., Mark A.E., Berendsen H.J.C. GROMACS: fast, flexible, and free. <span><span class="ref-journal">J. Comput. Chem. </span>2005;<span class="ref-vol">26</span>:1701–1718. doi: 10.1002/JCC.20291.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16211538" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2FJCC.20291" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Comput.+Chem.&amp;title=GROMACS:+fast,+flexible,+and+free&amp;author=D.+Van+Der+Spoel&amp;author=E.+Lindahl&amp;author=B.+Hess&amp;author=G.+Groenhof&amp;author=A.E.+Mark&amp;volume=26&amp;publication_year=2005&amp;pages=1701-1718&amp;pmid=16211538&amp;doi=10.1002/JCC.20291&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib40">40. <span class="element-citation" id="sref40">Kumari R., Kumar R., Lynn A. G-mmpbsa -A GROMACS tool for high-throughput MM-PBSA calculations. <span><span class="ref-journal">J. Chem. Inf. Model. </span>2014 doi: 10.1021/ci500020m.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24850022" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fci500020m" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Chem.+Inf.+Model.&amp;title=G-mmpbsa+-A+GROMACS+tool+for+high-throughput+MM-PBSA+calculations&amp;author=R.+Kumari&amp;author=R.+Kumar&amp;author=A.+Lynn&amp;publication_year=2014&amp;doi=10.1021/ci500020m&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib41">41. <span class="element-citation" id="sref41">Bhowmick S., Mistri T.K., Khan M.R., Patil P.C., Busquets R., Ashif Ikbal A.M., Choudhury A., Roy D.K., Palit P., Saha A. Investigation of bio-active amaryllidaceae alkaloidal small molecules as putative SARS-CoV-2 main protease and host TMPRSS2 inhibitors: interpretation by in-silico simulation study. <span><span class="ref-journal">J. Biomol. Struct. Dyn. </span>2023 doi: 10.1080/07391102.2023.2238065.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37482789" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1080%2F07391102.2023.2238065" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biomol.+Struct.+Dyn.&amp;title=Investigation+of+bio-active+amaryllidaceae+alkaloidal+small+molecules+as+putative+SARS-CoV-2+main+protease+and+host+TMPRSS2+inhibitors:+interpretation+by+in-silico+simulation+study&amp;author=S.+Bhowmick&amp;author=T.K.+Mistri&amp;author=M.R.+Khan&amp;author=P.C.+Patil&amp;author=R.+Busquets&amp;publication_year=2023&amp;doi=10.1080/07391102.2023.2238065&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib42">42. <span class="element-citation" id="sref42">Chikhale R.V., Gupta V.K., Eldesoky G.E., Wabaidur S.M., Patil S.A., Islam M.A. Identification of potential anti-TMPRSS2 natural products through homology modelling, virtual screening and molecular dynamics simulation studies. <span><span class="ref-journal">J. Biomol. Struct. Dyn. </span>2021;<span class="ref-vol">39</span>:6660–6675. doi: 10.1080/07391102.2020.1798813.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32741259" ref="reftype=pubmed&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1080%2F07391102.2020.1798813" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biomol.+Struct.+Dyn.&amp;title=Identification+of+potential+anti-TMPRSS2+natural+products+through+homology+modelling,+virtual+screening+and+molecular+dynamics+simulation+studies&amp;author=R.V.+Chikhale&amp;author=V.K.+Gupta&amp;author=G.E.+Eldesoky&amp;author=S.M.+Wabaidur&amp;author=S.A.+Patil&amp;volume=39&amp;publication_year=2021&amp;pages=6660-6675&amp;doi=10.1080/07391102.2020.1798813&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10907797&amp;issue-id=456855&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Heliyon</span> are provided here courtesy of <strong>Elsevier</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC10907797/pdf/main.pdf" class="int-view">PDF (6.4M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/10907797/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/10907797/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="10907797" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10907797%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="ZJECi6FgZlXsxTUVtwpnEh38DKAJmdL3vh02gfwFln5ZKYoEJd2EEXcur0L47qbS">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10907797%2F&amp;text=Identification%20of%20Mycobacterium%20tuberculosis%20transcriptional%20repressor%20EthR%20inhibitors%3A%20Shape-based%20search%20and%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10907797%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10907797/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC10907797/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38434349/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC10907797/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38434349/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC10907797/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/10907797/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC10907797/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
