
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>A human neural crest model reveals the developmental impact of neuroblastoma-associated chromosomal aberrations - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="D0BD41E4AC192A4500003395CA11DF95.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="ncomms" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC11068915/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11068915/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Nature Communications" /><meta name="citation_title" content="A human neural crest model reveals the developmental impact of neuroblastoma-associated chromosomal aberrations" /><meta name="citation_author" content="Ingrid M. Saldana-Guerrero" /><meta name="citation_author_institution" content="Centre for Stem Cell Biology, School of Biosciences, The University of Sheffield, Sheffield, UK" /><meta name="citation_author_institution" content="Neuroscience Institute, The University of Sheffield, Sheffield, UK" /><meta name="citation_author_institution" content="Sheffield Institute for Nucleic Acids (SInFoNiA), School of Medicine and Population Health, The University of Sheffield, Sheffield, UK" /><meta name="citation_author" content="Luis F. Montano-Gutierrez" /><meta name="citation_author_institution" content="St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria" /><meta name="citation_author" content="Katy Boswell" /><meta name="citation_author_institution" content="Centre for Stem Cell Biology, School of Biosciences, The University of Sheffield, Sheffield, UK" /><meta name="citation_author_institution" content="Neuroscience Institute, The University of Sheffield, Sheffield, UK" /><meta name="citation_author" content="Christoph Hafemeister" /><meta name="citation_author_institution" content="St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria" /><meta name="citation_author" content="Evon Poon" /><meta name="citation_author_institution" content="Division of Clinical Studies, The Institute of Cancer Research (ICR) &#x00026; Royal Marsden NHS Trust, London, UK" /><meta name="citation_author" content="Lisa E. Shaw" /><meta name="citation_author_institution" content="Department of Dermatology, Medical University of Vienna, Vienna, Austria" /><meta name="citation_author" content="Dylan Stavish" /><meta name="citation_author_institution" content="Centre for Stem Cell Biology, School of Biosciences, The University of Sheffield, Sheffield, UK" /><meta name="citation_author_institution" content="Neuroscience Institute, The University of Sheffield, Sheffield, UK" /><meta name="citation_author" content="Rebecca A. Lea" /><meta name="citation_author_institution" content="Centre for Stem Cell Biology, School of Biosciences, The University of Sheffield, Sheffield, UK" /><meta name="citation_author_institution" content="Neuroscience Institute, The University of Sheffield, Sheffield, UK" /><meta name="citation_author" content="Sara Wernig-Zorc" /><meta name="citation_author_institution" content="St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria" /><meta name="citation_author" content="Eva Bozsaky" /><meta name="citation_author_institution" content="St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria" /><meta name="citation_author" content="Irfete S. Fetahu" /><meta name="citation_author_institution" content="St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria" /><meta name="citation_author_institution" content="Present Address: Medical University of Vienna, Department of Neurology, Division of Neuropathology and Neurochemistry, Vienna, Austria" /><meta name="citation_author" content="Peter Zoescher" /><meta name="citation_author_institution" content="St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria" /><meta name="citation_author" content="Ulrike P&#x000f6;tschger" /><meta name="citation_author_institution" content="St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria" /><meta name="citation_author" content="Marie Bernkopf" /><meta name="citation_author_institution" content="St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria" /><meta name="citation_author_institution" content="Labdia Labordiagnostik GmbH, Vienna, Austria" /><meta name="citation_author" content="Andrea Wenninger-Weinzierl" /><meta name="citation_author_institution" content="St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria" /><meta name="citation_author" content="Caterina Sturtzel" /><meta name="citation_author_institution" content="St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria" /><meta name="citation_author" content="Celine Souilhol" /><meta name="citation_author_institution" content="Centre for Stem Cell Biology, School of Biosciences, The University of Sheffield, Sheffield, UK" /><meta name="citation_author_institution" content="Neuroscience Institute, The University of Sheffield, Sheffield, UK" /><meta name="citation_author_institution" content="Biomolecular Sciences Research Centre, Department of Biosciences and Chemistry, Sheffield Hallam University, Sheffield, UK" /><meta name="citation_author" content="Sophia Tarelli" /><meta name="citation_author_institution" content="Centre for Stem Cell Biology, School of Biosciences, The University of Sheffield, Sheffield, UK" /><meta name="citation_author_institution" content="Neuroscience Institute, The University of Sheffield, Sheffield, UK" /><meta name="citation_author" content="Mohamed R. Shoeb" /><meta name="citation_author_institution" content="St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria" /><meta name="citation_author" content="Polyxeni Bozatzi" /><meta name="citation_author_institution" content="St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria" /><meta name="citation_author" content="Magdalena Rados" /><meta name="citation_author_institution" content="St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria" /><meta name="citation_author" content="Maria Guarini" /><meta name="citation_author_institution" content="CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, Vienna, Austria" /><meta name="citation_author" content="Michelle C. Buri" /><meta name="citation_author_institution" content="St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria" /><meta name="citation_author" content="Wolfgang Weninger" /><meta name="citation_author_institution" content="Department of Dermatology, Medical University of Vienna, Vienna, Austria" /><meta name="citation_author" content="Eva M. Putz" /><meta name="citation_author_institution" content="St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria" /><meta name="citation_author" content="Miller Huang" /><meta name="citation_author_institution" content="Children&#x02019;s Hospital Los Angeles, Cancer and Blood Disease Institutes, and The Saban Research Institute, Los Angeles, CA USA" /><meta name="citation_author_institution" content="Keck School of Medicine, University of Southern California, Los Angeles, CA USA" /><meta name="citation_author" content="Ruth Ladenstein" /><meta name="citation_author_institution" content="St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria" /><meta name="citation_author" content="Peter W. Andrews" /><meta name="citation_author_institution" content="Centre for Stem Cell Biology, School of Biosciences, The University of Sheffield, Sheffield, UK" /><meta name="citation_author" content="Ivana Barbaric" /><meta name="citation_author_institution" content="Centre for Stem Cell Biology, School of Biosciences, The University of Sheffield, Sheffield, UK" /><meta name="citation_author_institution" content="Neuroscience Institute, The University of Sheffield, Sheffield, UK" /><meta name="citation_author" content="George D. Cresswell" /><meta name="citation_author_institution" content="St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria" /><meta name="citation_author" content="Helen E. Bryant" /><meta name="citation_author_institution" content="Sheffield Institute for Nucleic Acids (SInFoNiA), School of Medicine and Population Health, The University of Sheffield, Sheffield, UK" /><meta name="citation_author" content="Martin Distel" /><meta name="citation_author_institution" content="St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria" /><meta name="citation_author" content="Louis Chesler" /><meta name="citation_author_institution" content="Division of Clinical Studies, The Institute of Cancer Research (ICR) &#x00026; Royal Marsden NHS Trust, London, UK" /><meta name="citation_author" content="Sabine Taschner-Mandl" /><meta name="citation_author_institution" content="St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria" /><meta name="citation_author" content="Matthias Farlik" /><meta name="citation_author_institution" content="Department of Dermatology, Medical University of Vienna, Vienna, Austria" /><meta name="citation_author" content="Anestis Tsakiridis" /><meta name="citation_author_institution" content="Centre for Stem Cell Biology, School of Biosciences, The University of Sheffield, Sheffield, UK" /><meta name="citation_author_institution" content="Neuroscience Institute, The University of Sheffield, Sheffield, UK" /><meta name="citation_author" content="Florian Halbritter" /><meta name="citation_author_institution" content="St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria" /><meta name="citation_publication_date" content="2024" /><meta name="citation_volume" content="15" /><meta name="citation_doi" content="10.1038/s41467-024-47945-7" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC11068915/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC11068915/" /><meta name="citation_pmid" content="38702304" /><meta name="DC.Title" content="A human neural crest model reveals the developmental impact of neuroblastoma-associated chromosomal aberrations" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Nature Publishing Group" /><meta name="DC.Contributor" content="Ingrid M. Saldana-Guerrero" /><meta name="DC.Contributor" content="Luis F. Montano-Gutierrez" /><meta name="DC.Contributor" content="Katy Boswell" /><meta name="DC.Contributor" content="Christoph Hafemeister" /><meta name="DC.Contributor" content="Evon Poon" /><meta name="DC.Contributor" content="Lisa E. Shaw" /><meta name="DC.Contributor" content="Dylan Stavish" /><meta name="DC.Contributor" content="Rebecca A. Lea" /><meta name="DC.Contributor" content="Sara Wernig-Zorc" /><meta name="DC.Contributor" content="Eva Bozsaky" /><meta name="DC.Contributor" content="Irfete S. Fetahu" /><meta name="DC.Contributor" content="Peter Zoescher" /><meta name="DC.Contributor" content="Ulrike P&#x000f6;tschger" /><meta name="DC.Contributor" content="Marie Bernkopf" /><meta name="DC.Contributor" content="Andrea Wenninger-Weinzierl" /><meta name="DC.Contributor" content="Caterina Sturtzel" /><meta name="DC.Contributor" content="Celine Souilhol" /><meta name="DC.Contributor" content="Sophia Tarelli" /><meta name="DC.Contributor" content="Mohamed R. Shoeb" /><meta name="DC.Contributor" content="Polyxeni Bozatzi" /><meta name="DC.Contributor" content="Magdalena Rados" /><meta name="DC.Contributor" content="Maria Guarini" /><meta name="DC.Contributor" content="Michelle C. Buri" /><meta name="DC.Contributor" content="Wolfgang Weninger" /><meta name="DC.Contributor" content="Eva M. Putz" /><meta name="DC.Contributor" content="Miller Huang" /><meta name="DC.Contributor" content="Ruth Ladenstein" /><meta name="DC.Contributor" content="Peter W. Andrews" /><meta name="DC.Contributor" content="Ivana Barbaric" /><meta name="DC.Contributor" content="George D. Cresswell" /><meta name="DC.Contributor" content="Helen E. Bryant" /><meta name="DC.Contributor" content="Martin Distel" /><meta name="DC.Contributor" content="Louis Chesler" /><meta name="DC.Contributor" content="Sabine Taschner-Mandl" /><meta name="DC.Contributor" content="Matthias Farlik" /><meta name="DC.Contributor" content="Anestis Tsakiridis" /><meta name="DC.Contributor" content="Florian Halbritter" /><meta name="DC.Date" content="2024" /><meta name="DC.Identifier" content="10.1038/s41467-024-47945-7" /><meta name="DC.Language" content="en" /><meta property="og:title" content="A human neural crest model reveals the developmental impact of neuroblastoma-associated chromosomal aberrations" /><meta property="og:type" content="article" /><meta property="og:description" content="Early childhood tumours arise from transformed embryonic cells, which often carry large copy number alterations (CNA). However, it remains unclear how CNAs contribute to embryonic tumourigenesis due to a lack of suitable models. Here we employ female ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11068915/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11068915/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22Nat%20Commun%22[journal]" class="navlink">Nat Commun</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11068915
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11068915/pdf/41467_2024_Article_47945.pdf" class="int-view">PDF (7.1M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11068915/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11068915/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11068915" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11068915%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="ZWCicIBDTDHN9pzWyyp2QbQyx48iesSm2a6HHcuJW5UwuYRns3zhd1YEx7Y5ZbZM">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11068915%2F&amp;text=A%20human%20neural%20crest%20model%20reveals%20the%20developmental%20impact%20of%20neuroblastoma-associated%20chromosomal%20aberrations%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11068915%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11068915/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11068915/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38702304/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11068915/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38702304/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11068915/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11068915/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22Nat%20Commun%22[journal]" class="navlink">Nat Commun</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11068915
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-ncomms.gif" alt="Logo of ncomms" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Nat Commun.</span> 2024; 15: 3745. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2024 May 3. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1038%2Fs41467-024-47945-7" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/s41467-024-47945-7</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC11068915</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38702304">38702304</a></div></div></div><h1 class="content-title">A human neural crest model reveals the developmental impact of neuroblastoma-associated chromosomal aberrations</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Saldana-Guerrero%20IM%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783123754928" co-class="co-affbox">Ingrid M. Saldana-Guerrero</a>,<sup>#</sup><sup>1,</sup><sup>2,</sup><sup>3</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Montano-Gutierrez%20LF%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783159901056" co-class="co-affbox">Luis F. Montano-Gutierrez</a>,<sup>#</sup><sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Boswell%20K%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783097785776" co-class="co-affbox">Katy Boswell</a>,<sup>1,</sup><sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hafemeister%20C%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783097782656" co-class="co-affbox">Christoph Hafemeister</a>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Poon%20E%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783110234032" co-class="co-affbox">Evon Poon</a>,<sup>5</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shaw%20LE%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783108085440" co-class="co-affbox">Lisa E. Shaw</a>,<sup>6</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Stavish%20D%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783108082720" co-class="co-affbox">Dylan Stavish</a>,<sup>1,</sup><sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Lea%20RA%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783161463376" co-class="co-affbox">Rebecca A. Lea</a>,<sup>1,</sup><sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wernig-Zorc%20S%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783108770784" co-class="co-affbox">Sara Wernig-Zorc</a>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Bozsaky%20E%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783108768800" co-class="co-affbox">Eva Bozsaky</a>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Fetahu%20IS%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783121342192" co-class="co-affbox">Irfete S. Fetahu</a>,<sup>4,</sup><sup>12</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Zoescher%20P%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783107518064" co-class="co-affbox">Peter Zoescher</a>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=P&#x000f6;tschger%20U%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783161451568" co-class="co-affbox">Ulrike P&#x000f6;tschger</a>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Bernkopf%20M%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783161449584" co-class="co-affbox">Marie Bernkopf</a>,<sup>4,</sup><sup>7</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wenninger-Weinzierl%20A%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783121156736" co-class="co-affbox">Andrea Wenninger-Weinzierl</a>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Sturtzel%20C%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783121154752" co-class="co-affbox">Caterina Sturtzel</a>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Souilhol%20C%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783109703056" co-class="co-affbox">Celine Souilhol</a>,<sup>1,</sup><sup>2,</sup><sup>8</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Tarelli%20S%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783106929584" co-class="co-affbox">Sophia Tarelli</a>,<sup>1,</sup><sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shoeb%20MR%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783106926864" co-class="co-affbox">Mohamed R. Shoeb</a>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Bozatzi%20P%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783124189600" co-class="co-affbox">Polyxeni Bozatzi</a>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Rados%20M%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783124187616" co-class="co-affbox">Magdalena Rados</a>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Guarini%20M%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783118405008" co-class="co-affbox">Maria Guarini</a>,<sup>9</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Buri%20MC%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783118402288" co-class="co-affbox">Michelle C. Buri</a>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Weninger%20W%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783099959808" co-class="co-affbox">Wolfgang Weninger</a>,<sup>6</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Putz%20EM%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783158902048" co-class="co-affbox">Eva M. Putz</a>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Huang%20M%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783158899328" co-class="co-affbox">Miller Huang</a>,<sup>10,</sup><sup>11</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ladenstein%20R%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783120969008" co-class="co-affbox">Ruth Ladenstein</a>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Andrews%20PW%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783123529936" co-class="co-affbox">Peter W. Andrews</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Barbaric%20I%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783123527216" co-class="co-affbox">Ivana Barbaric</a>,<sup>1,</sup><sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Cresswell%20GD%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783161173584" co-class="co-affbox">George D. Cresswell</a>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Bryant%20HE%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783097547536" co-class="co-affbox">Helen E. Bryant</a>,<sup>3</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Distel%20M%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783097544688" co-class="co-affbox">Martin Distel</a>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Chesler%20L%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783116817696" co-class="co-affbox">Louis Chesler</a>,<sup>5</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Taschner-Mandl%20S%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783162308656" co-class="co-affbox">Sabine Taschner-Mandl</a>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Farlik%20M%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783162305648" co-class="co-affbox">Matthias Farlik</a>,<sup>6</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Tsakiridis%20A%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783097452656" co-class="co-affbox">Anestis Tsakiridis</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>1,</sup><sup>2</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Halbritter%20F%5BAuthor%5D" class="affpopup" co-rid="_co_idm139783100191504" co-class="co-affbox">Florian Halbritter</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>4</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139783123754928"><h3 class="no_margin">Ingrid M. Saldana-Guerrero</h3><p><sup>1</sup>Centre for Stem Cell Biology, School of Biosciences, The University of Sheffield, Sheffield, UK </p><p><sup>2</sup>Neuroscience Institute, The University of Sheffield, Sheffield, UK </p><p><sup>3</sup>Sheffield Institute for Nucleic Acids (SInFoNiA), School of Medicine and Population Health, The University of Sheffield, Sheffield, UK </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Saldana-Guerrero%20IM%5BAuthor%5D">Ingrid M. Saldana-Guerrero</a></div></div><div id="_co_idm139783159901056"><h3 class="no_margin">Luis F. Montano-Gutierrez</h3><p><sup>4</sup>St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Montano-Gutierrez%20LF%5BAuthor%5D">Luis F. Montano-Gutierrez</a></div></div><div id="_co_idm139783097785776"><h3 class="no_margin">Katy Boswell</h3><p><sup>1</sup>Centre for Stem Cell Biology, School of Biosciences, The University of Sheffield, Sheffield, UK </p><p><sup>2</sup>Neuroscience Institute, The University of Sheffield, Sheffield, UK </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Boswell%20K%5BAuthor%5D">Katy Boswell</a></div></div><div id="_co_idm139783097782656"><h3 class="no_margin">Christoph Hafemeister</h3><p><sup>4</sup>St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hafemeister%20C%5BAuthor%5D">Christoph Hafemeister</a></div></div><div id="_co_idm139783110234032"><h3 class="no_margin">Evon Poon</h3><p><sup>5</sup>Division of Clinical Studies, The Institute of Cancer Research (ICR) &#x00026; Royal Marsden NHS Trust, London, UK </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Poon%20E%5BAuthor%5D">Evon Poon</a></div></div><div id="_co_idm139783108085440"><h3 class="no_margin">Lisa E. Shaw</h3><p><sup>6</sup>Department of Dermatology, Medical University of Vienna, Vienna, Austria </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shaw%20LE%5BAuthor%5D">Lisa E. Shaw</a></div></div><div id="_co_idm139783108082720"><h3 class="no_margin">Dylan Stavish</h3><p><sup>1</sup>Centre for Stem Cell Biology, School of Biosciences, The University of Sheffield, Sheffield, UK </p><p><sup>2</sup>Neuroscience Institute, The University of Sheffield, Sheffield, UK </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Stavish%20D%5BAuthor%5D">Dylan Stavish</a></div></div><div id="_co_idm139783161463376"><h3 class="no_margin">Rebecca A. Lea</h3><p><sup>1</sup>Centre for Stem Cell Biology, School of Biosciences, The University of Sheffield, Sheffield, UK </p><p><sup>2</sup>Neuroscience Institute, The University of Sheffield, Sheffield, UK </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Lea%20RA%5BAuthor%5D">Rebecca A. Lea</a></div></div><div id="_co_idm139783108770784"><h3 class="no_margin">Sara Wernig-Zorc</h3><p><sup>4</sup>St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wernig-Zorc%20S%5BAuthor%5D">Sara Wernig-Zorc</a></div></div><div id="_co_idm139783108768800"><h3 class="no_margin">Eva Bozsaky</h3><p><sup>4</sup>St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Bozsaky%20E%5BAuthor%5D">Eva Bozsaky</a></div></div><div id="_co_idm139783121342192"><h3 class="no_margin">Irfete S. Fetahu</h3><p><sup>4</sup>St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria </p><p><sup>12</sup>Present Address: Medical University of Vienna, Department of Neurology, Division of Neuropathology and Neurochemistry, Vienna, Austria </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Fetahu%20IS%5BAuthor%5D">Irfete S. Fetahu</a></div></div><div id="_co_idm139783107518064"><h3 class="no_margin">Peter Zoescher</h3><p><sup>4</sup>St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Zoescher%20P%5BAuthor%5D">Peter Zoescher</a></div></div><div id="_co_idm139783161451568"><h3 class="no_margin">Ulrike P&#x000f6;tschger</h3><p><sup>4</sup>St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=P&#x000f6;tschger%20U%5BAuthor%5D">Ulrike P&#x000f6;tschger</a></div></div><div id="_co_idm139783161449584"><h3 class="no_margin">Marie Bernkopf</h3><p><sup>4</sup>St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria </p><p><sup>7</sup>Labdia Labordiagnostik GmbH, Vienna, Austria </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Bernkopf%20M%5BAuthor%5D">Marie Bernkopf</a></div></div><div id="_co_idm139783121156736"><h3 class="no_margin">Andrea Wenninger-Weinzierl</h3><p><sup>4</sup>St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wenninger-Weinzierl%20A%5BAuthor%5D">Andrea Wenninger-Weinzierl</a></div></div><div id="_co_idm139783121154752"><h3 class="no_margin">Caterina Sturtzel</h3><p><sup>4</sup>St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Sturtzel%20C%5BAuthor%5D">Caterina Sturtzel</a></div></div><div id="_co_idm139783109703056"><h3 class="no_margin">Celine Souilhol</h3><p><sup>1</sup>Centre for Stem Cell Biology, School of Biosciences, The University of Sheffield, Sheffield, UK </p><p><sup>2</sup>Neuroscience Institute, The University of Sheffield, Sheffield, UK </p><p><sup>8</sup>Biomolecular Sciences Research Centre, Department of Biosciences and Chemistry, Sheffield Hallam University, Sheffield, UK </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Souilhol%20C%5BAuthor%5D">Celine Souilhol</a></div></div><div id="_co_idm139783106929584"><h3 class="no_margin">Sophia Tarelli</h3><p><sup>1</sup>Centre for Stem Cell Biology, School of Biosciences, The University of Sheffield, Sheffield, UK </p><p><sup>2</sup>Neuroscience Institute, The University of Sheffield, Sheffield, UK </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Tarelli%20S%5BAuthor%5D">Sophia Tarelli</a></div></div><div id="_co_idm139783106926864"><h3 class="no_margin">Mohamed R. Shoeb</h3><p><sup>4</sup>St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shoeb%20MR%5BAuthor%5D">Mohamed R. Shoeb</a></div></div><div id="_co_idm139783124189600"><h3 class="no_margin">Polyxeni Bozatzi</h3><p><sup>4</sup>St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Bozatzi%20P%5BAuthor%5D">Polyxeni Bozatzi</a></div></div><div id="_co_idm139783124187616"><h3 class="no_margin">Magdalena Rados</h3><p><sup>4</sup>St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Rados%20M%5BAuthor%5D">Magdalena Rados</a></div></div><div id="_co_idm139783118405008"><h3 class="no_margin">Maria Guarini</h3><p><sup>9</sup>CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, Vienna, Austria </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Guarini%20M%5BAuthor%5D">Maria Guarini</a></div></div><div id="_co_idm139783118402288"><h3 class="no_margin">Michelle C. Buri</h3><p><sup>4</sup>St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Buri%20MC%5BAuthor%5D">Michelle C. Buri</a></div></div><div id="_co_idm139783099959808"><h3 class="no_margin">Wolfgang Weninger</h3><p><sup>6</sup>Department of Dermatology, Medical University of Vienna, Vienna, Austria </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Weninger%20W%5BAuthor%5D">Wolfgang Weninger</a></div></div><div id="_co_idm139783158902048"><h3 class="no_margin">Eva M. Putz</h3><p><sup>4</sup>St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Putz%20EM%5BAuthor%5D">Eva M. Putz</a></div></div><div id="_co_idm139783158899328"><h3 class="no_margin">Miller Huang</h3><p><sup>10</sup>Children&#x02019;s Hospital Los Angeles, Cancer and Blood Disease Institutes, and The Saban Research Institute, Los Angeles, CA USA </p><p><sup>11</sup>Keck School of Medicine, University of Southern California, Los Angeles, CA USA </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Huang%20M%5BAuthor%5D">Miller Huang</a></div></div><div id="_co_idm139783120969008"><h3 class="no_margin">Ruth Ladenstein</h3><p><sup>4</sup>St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ladenstein%20R%5BAuthor%5D">Ruth Ladenstein</a></div></div><div id="_co_idm139783123529936"><h3 class="no_margin">Peter W. Andrews</h3><p><sup>1</sup>Centre for Stem Cell Biology, School of Biosciences, The University of Sheffield, Sheffield, UK </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Andrews%20PW%5BAuthor%5D">Peter W. Andrews</a></div></div><div id="_co_idm139783123527216"><h3 class="no_margin">Ivana Barbaric</h3><p><sup>1</sup>Centre for Stem Cell Biology, School of Biosciences, The University of Sheffield, Sheffield, UK </p><p><sup>2</sup>Neuroscience Institute, The University of Sheffield, Sheffield, UK </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Barbaric%20I%5BAuthor%5D">Ivana Barbaric</a></div></div><div id="_co_idm139783161173584"><h3 class="no_margin">George D. Cresswell</h3><p><sup>4</sup>St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Cresswell%20GD%5BAuthor%5D">George D. Cresswell</a></div></div><div id="_co_idm139783097547536"><h3 class="no_margin">Helen E. Bryant</h3><p><sup>3</sup>Sheffield Institute for Nucleic Acids (SInFoNiA), School of Medicine and Population Health, The University of Sheffield, Sheffield, UK </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Bryant%20HE%5BAuthor%5D">Helen E. Bryant</a></div></div><div id="_co_idm139783097544688"><h3 class="no_margin">Martin Distel</h3><p><sup>4</sup>St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Distel%20M%5BAuthor%5D">Martin Distel</a></div></div><div id="_co_idm139783116817696"><h3 class="no_margin">Louis Chesler</h3><p><sup>5</sup>Division of Clinical Studies, The Institute of Cancer Research (ICR) &#x00026; Royal Marsden NHS Trust, London, UK </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Chesler%20L%5BAuthor%5D">Louis Chesler</a></div></div><div id="_co_idm139783162308656"><h3 class="no_margin">Sabine Taschner-Mandl</h3><p><sup>4</sup>St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Taschner-Mandl%20S%5BAuthor%5D">Sabine Taschner-Mandl</a></div></div><div id="_co_idm139783162305648"><h3 class="no_margin">Matthias Farlik</h3><p><sup>6</sup>Department of Dermatology, Medical University of Vienna, Vienna, Austria </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Farlik%20M%5BAuthor%5D">Matthias Farlik</a></div></div><div id="_co_idm139783097452656"><h3 class="no_margin">Anestis Tsakiridis</h3><p><sup>1</sup>Centre for Stem Cell Biology, School of Biosciences, The University of Sheffield, Sheffield, UK </p><p><sup>2</sup>Neuroscience Institute, The University of Sheffield, Sheffield, UK </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Tsakiridis%20A%5BAuthor%5D">Anestis Tsakiridis</a></div></div><div id="_co_idm139783100191504"><h3 class="no_margin">Florian Halbritter</h3><p><sup>4</sup>St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Halbritter%20F%5BAuthor%5D">Florian Halbritter</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm139783116016464_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139783116016464_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139783116016464_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm139783116016464_ai" style="display:none"><div class="fm-affl" id="Aff1"><sup>1</sup>Centre for Stem Cell Biology, School of Biosciences, The University of Sheffield, Sheffield, UK </div><div class="fm-affl" id="Aff2"><sup>2</sup>Neuroscience Institute, The University of Sheffield, Sheffield, UK </div><div class="fm-affl" id="Aff3"><sup>3</sup>Sheffield Institute for Nucleic Acids (SInFoNiA), School of Medicine and Population Health, The University of Sheffield, Sheffield, UK </div><div class="fm-affl" id="Aff4"><sup>4</sup>St. Anna Children&#x02019;s Cancer Research Institute (CCRI), Vienna, Austria </div><div class="fm-affl" id="Aff5"><sup>5</sup>Division of Clinical Studies, The Institute of Cancer Research (ICR) &#x00026; Royal Marsden NHS Trust, London, UK </div><div class="fm-affl" id="Aff6"><sup>6</sup>Department of Dermatology, Medical University of Vienna, Vienna, Austria </div><div class="fm-affl" id="Aff7"><sup>7</sup>Labdia Labordiagnostik GmbH, Vienna, Austria </div><div class="fm-affl" id="Aff8"><sup>8</sup>Biomolecular Sciences Research Centre, Department of Biosciences and Chemistry, Sheffield Hallam University, Sheffield, UK </div><div class="fm-affl" id="Aff9"><sup>9</sup>CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, Vienna, Austria </div><div class="fm-affl" id="Aff10"><sup>10</sup>Children&#x02019;s Hospital Los Angeles, Cancer and Blood Disease Institutes, and The Saban Research Institute, Los Angeles, CA USA </div><div class="fm-affl" id="Aff11"><sup>11</sup>Keck School of Medicine, University of Southern California, Los Angeles, CA USA </div><div class="fm-affl" id="Aff12"><sup>12</sup>Present Address: Medical University of Vienna, Department of Neurology, Division of Neuropathology and Neurochemistry, Vienna, Austria </div><div><span class="fm-affl">Anestis Tsakiridis, </span><span class="fm-affl"><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ku.ca.dleiffehs@sidirikast.a" class="oemail">ku.ca.dleiffehs@sidirikast.a</a></span>.</div><div><a href="#article-aaff-info">Contributor Information</a>.</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup>Corresponding author.</div><div><sup>#</sup>Contributed equally.</div></div><div class="fm-article-notes hide half_rhythm" id="idm139783116016464_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2023 Jan 6; Accepted 2024 Apr 15.</div></div><div class="permissions half_rhythm hide" id="idm139783116016464_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; The Author(s) 2024</div><div class="license half_rhythm"><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article&#x02019;s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article&#x02019;s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="21" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Source Data</div><div><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl">41467_2024_47945_MOESM1_ESM.xlsx</a><span style="color:gray"> (381M)</span></div><div class="small guid">GUID: F8A46890-118F-4522-A6E1-74549960923E</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary Information</div><div><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl">41467_2024_47945_MOESM2_ESM.pdf</a><span style="color:gray"> (4.7M)</span></div><div class="small guid">GUID: AEBD363A-C404-42D8-8DCF-191078BAD882</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Peer Review File</div><div><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl">41467_2024_47945_MOESM3_ESM.pdf</a><span style="color:gray"> (5.7M)</span></div><div class="small guid">GUID: BB60984B-ABC9-4D3E-912E-14C41F3627FE</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Description of Additional Supplementary Files</div><div><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl">41467_2024_47945_MOESM4_ESM.pdf</a><span style="color:gray"> (84K)</span></div><div class="small guid">GUID: 0896BB02-0487-48AD-967C-7F31C0A81CF5</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary Data 1</div><div><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl">41467_2024_47945_MOESM5_ESM.xlsx</a><span style="color:gray"> (65K)</span></div><div class="small guid">GUID: 34319A97-E8FB-454F-88F9-F1BDF6C03941</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary Data 2</div><div><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl">41467_2024_47945_MOESM6_ESM.xlsx</a><span style="color:gray"> (668K)</span></div><div class="small guid">GUID: CAF20191-41E9-4561-830D-4BEEE9A92CA3</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary Data 3</div><div><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl">41467_2024_47945_MOESM7_ESM.xlsx</a><span style="color:gray"> (87K)</span></div><div class="small guid">GUID: 63A94FF6-4C9F-4E30-A4CE-365427169142</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary Data 4</div><div><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl">41467_2024_47945_MOESM8_ESM.xlsx</a><span style="color:gray"> (1.4M)</span></div><div class="small guid">GUID: 78E626A4-40AC-485C-A822-A3596403618C</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary Data 5</div><div><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl">41467_2024_47945_MOESM9_ESM.xlsx</a><span style="color:gray"> (56K)</span></div><div class="small guid">GUID: 4EAAEC59-AD7E-43AE-8E94-FA4CC98C8CBB</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary Data 6</div><div><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl">41467_2024_47945_MOESM10_ESM.xlsx</a><span style="color:gray"> (120K)</span></div><div class="small guid">GUID: EF334177-217A-4B20-A49D-EBA8871087A1</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary Data 7</div><div><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM11_ESM.xlsx" data-ga-action="click_feat_suppl">41467_2024_47945_MOESM11_ESM.xlsx</a><span style="color:gray"> (68K)</span></div><div class="small guid">GUID: 2FCD2717-B5CE-4E1B-969B-DCC2269001DF</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary Data 8</div><div><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM12_ESM.xlsx" data-ga-action="click_feat_suppl">41467_2024_47945_MOESM12_ESM.xlsx</a><span style="color:gray"> (324K)</span></div><div class="small guid">GUID: 7675BB5E-0E65-4B0F-B013-627B96B41212</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary Data 9</div><div><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM13_ESM.xlsx" data-ga-action="click_feat_suppl">41467_2024_47945_MOESM13_ESM.xlsx</a><span style="color:gray"> (192K)</span></div><div class="small guid">GUID: 16385263-9352-4FDF-8506-209D9DBE1CBA</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary Data 10</div><div><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM14_ESM.xlsx" data-ga-action="click_feat_suppl">41467_2024_47945_MOESM14_ESM.xlsx</a><span style="color:gray"> (92K)</span></div><div class="small guid">GUID: 187DD56F-842C-45FD-94BD-4ACEBD883E30</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary Data 11</div><div><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM15_ESM.xlsx" data-ga-action="click_feat_suppl">41467_2024_47945_MOESM15_ESM.xlsx</a><span style="color:gray"> (55K)</span></div><div class="small guid">GUID: BD3B6E27-E855-413D-A318-B9879C3A1828</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary Data 12</div><div><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM16_ESM.xlsx" data-ga-action="click_feat_suppl">41467_2024_47945_MOESM16_ESM.xlsx</a><span style="color:gray"> (20M)</span></div><div class="small guid">GUID: 8DBA8F4D-33FB-4A31-9085-071B27E70E27</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary Data 13</div><div><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM17_ESM.xlsx" data-ga-action="click_feat_suppl">41467_2024_47945_MOESM17_ESM.xlsx</a><span style="color:gray"> (15M)</span></div><div class="small guid">GUID: B6C0503E-BFB5-43A9-9F99-827F3E7DA186</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary Data 14</div><div><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM18_ESM.xlsx" data-ga-action="click_feat_suppl">41467_2024_47945_MOESM18_ESM.xlsx</a><span style="color:gray"> (104K)</span></div><div class="small guid">GUID: 6EF7FD22-365E-46BE-A5FC-9EC7711ACF44</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary Data 15</div><div><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM19_ESM.xlsx" data-ga-action="click_feat_suppl">41467_2024_47945_MOESM19_ESM.xlsx</a><span style="color:gray"> (2.0M)</span></div><div class="small guid">GUID: E5FC68EA-331C-4C05-91E2-1F54ED8871A2</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary Data 16</div><div><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM20_ESM.xlsx" data-ga-action="click_feat_suppl">41467_2024_47945_MOESM20_ESM.xlsx</a><span style="color:gray"> (59K)</span></div><div class="small guid">GUID: D4AE4A46-FCDE-412A-A767-0D6D88F5EB2A</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Reporting summary</div><div><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM21_ESM.pdf" data-ga-action="click_feat_suppl">41467_2024_47945_MOESM21_ESM.pdf</a><span style="color:gray"> (6.1M)</span></div><div class="small guid">GUID: 76C6E922-22EA-4048-9CFD-5CE36BAE44DA</div></div></dd></dl><dl data-length="2361" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p>The single-cell RNA-seq and ATAC-seq data generated in this study have been deposited in the Gene Expression Omnibus (GEO) under accession code <a class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE219153" ref="/geo/query/acc.cgi?acc=GSE219153"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE219153","term_id":"219153"}}</span>GSE219153</a>. The public scRNA-seq data from NB tumours used in this study are available in GEO under the accession codes <a class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE147821" ref="/geo/query/acc.cgi?acc=GSE147821"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE147821","term_id":"147821"}}</span>GSE147821</a>, <a class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE216176" ref="/geo/query/acc.cgi?acc=GSE216176"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE216176","term_id":"216176"}}</span>GSE216176</a>, and <a class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE137804" ref="/geo/query/acc.cgi?acc=GSE137804"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE137804","term_id":"137804"}}</span>GSE137804</a>, and in the European Genome-Phenome Archive (EGA) under accession code <a href="https://ega-archive.org/studies/EGAS00001004388" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">EGAS00001004388</a>. Public ATAC-seq data from NB cell lines and controls used in this study are available in GEO under accession codes <a class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE138293" ref="/geo/query/acc.cgi?acc=GSE138293"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE138293","term_id":"138293"}}</span>GSE138293</a>, <a class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE224241" ref="/geo/query/acc.cgi?acc=GSE224241"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE224241","term_id":"224241"}}</span>GSE224241</a>, <a class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE136279" ref="/geo/query/acc.cgi?acc=GSE136279"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE136279","term_id":"136279"}}</span>GSE136279</a>, <a class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE202511" ref="/geo/query/acc.cgi?acc=GSE202511"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE202511","term_id":"202511"}}</span>GSE202511</a>, and <a class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE228832" ref="/geo/query/acc.cgi?acc=GSE228832"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE228832","term_id":"228832"}}</span>GSE228832</a>, and from the EpiMap website (<a href="https://epigenome.wustl.edu/epimap" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://epigenome.wustl.edu/epimap</a>). Bulk RNA-seq data from NB tumours are available in GEO under accession codes <a class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE49711" ref="/geo/query/acc.cgi?acc=GSE49711"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE49711","term_id":"49711"}}</span>GSE49711</a>, <a class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE94035" ref="/geo/query/acc.cgi?acc=GSE94035"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE94035","term_id":"94035"}}</span>GSE94035</a>, <a class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE147635" ref="/geo/query/acc.cgi?acc=GSE147635"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE147635","term_id":"147635"}}</span>GSE147635</a>, and <a class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE172184" ref="/geo/query/acc.cgi?acc=GSE172184"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE172184","term_id":"172184"}}</span>GSE172184</a>, and in dbGAP under accession code phs000467 [<a href="https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000218" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000218</a>]. Source data are provided with this paper. Additionally, processed data from this paper can be accessed and browsed interactively via our GitHub page [<a href="https://github.com/cancerbits/saldana_montano2024_ncnb/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/cancerbits/saldana_montano2024_ncnb/</a>] and via the R2 Genomics Analysis and Visualisation Platform [<a href="http://r2platform.com/halbritter24/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://r2platform.com/halbritter24/</a>].&#x000a0;<a href="#Sec67" rid="Sec67" class=" sec">Source data</a> are provided with this paper.</p><p>Computer code used for the data analysis in this study is available via our GitHub page (<a href="https://github.com/cancerbits/saldana_montano2024_ncnb/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/cancerbits/saldana_montano2024_ncnb/</a>) and a persistent copy of this repository is available via Zenodo (10.5281/zenodo.10891507).</p></dd></dl></div><div id="Abs1" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="Abs1title">Abstract</h2><!--article-meta--><div><p id="Par1" class="p p-first-last">Early childhood tumours arise from transformed embryonic cells, which often carry large copy number alterations (CNA). However, it remains unclear how CNAs contribute to embryonic tumourigenesis due to a lack of suitable models. Here we employ female human embryonic stem cell (hESC) differentiation and single-cell transcriptome and epigenome analysis to assess the effects of chromosome 17q/1q gains, which are prevalent in the embryonal tumour neuroblastoma (NB). We show that CNAs impair the specification of trunk neural crest (NC) cells and their sympathoadrenal derivatives, the putative cells-of-origin of NB. This effect is exacerbated upon overexpression of <em>MYCN</em>, whose amplification co-occurs with CNAs in NB. Moreover, CNAs potentiate the pro-tumourigenic effects of <em>MYCN</em> and mutant NC cells resemble NB cells in tumours. These changes correlate with a stepwise aberration of developmental transcription factor networks. Together, our results sketch a mechanistic framework for the CNA-driven initiation of embryonal tumours.</p></div><div class="sec"><strong class="kwd-title">Subject terms: </strong><span class="kwd-text">Paediatric cancer, Stem cells, Disease model, Cancer genomics, Embryonal neoplasms</span></div></div><div id="Abs2" lang="en" class="tsec sec"><h2 class="headless nomenu"></h2><!--article-meta--><div><p id="Par2" class="p p-first-last">Copy number alterations in stem cells impair neural crest differentiation and set the stage for neuroblastoma-like traits and tumours. This study hints at early tumourigenesis mechanisms and finds developmental gene signatures linked to prognosis.</p></div></div><div id="Sec1" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec1title">Introduction</h2><p id="Par3" class="p p-first">Cancers in early childhood are driven by sparse genetic aberrations arising in utero, which are thought to lead to defective differentiation and uncontrolled proliferation<sup><sup><a href="#CR1" rid="CR1" class=" bibr popnode">1</a></sup>&#x02013;<sup><a href="#CR4" rid="CR4" class=" bibr popnode">4</a></sup></sup>. Most tumours harbour large genomic rearrangements and chromosomal copy number alterations (CNA), which co-occur with mutations in tumour suppressors or tumourigenic transcription factors (TF)<sup><sup><a href="#CR5" rid="CR5" class=" bibr popnode">5</a></sup>,<sup><a href="#CR6" rid="CR6" class=" bibr popnode">6</a></sup></sup>. The mechanistic interactions between different mutations and early developmental processes are likely foundational drivers of tumour heterogeneity. However, since visible tumours are only detected long after their initiation, early mutation-driven interactions leading to the healthy-to-tumour transition have remained largely intractable.</p><p id="Par4">Neuroblastoma (NB) is the most common extra-cranial solid tumour in infants and an archetypal &#x0201c;developmental cancer&#x0201d;<sup><sup><a href="#CR7" rid="CR7" class=" bibr popnode">7</a></sup>&#x02013;<sup><a href="#CR9" rid="CR9" class=" bibr popnode">9</a></sup></sup>. NB tumours are usually found in the adrenal gland or sympathetic ganglia, tissues derived from the trunk neural crest (NC) lineage during embryonic development<sup><sup><a href="#CR10" rid="CR10" class=" bibr popnode">10</a></sup>,<sup><a href="#CR11" rid="CR11" class=" bibr popnode">11</a></sup></sup>, and studies using transgenic animal models and transcriptome analysis have anchored NB tumourigenesis in impaired sympathoadrenal differentiation of trunk NC cells<sup><sup><a href="#CR12" rid="CR12" class=" bibr popnode">12</a></sup>&#x02013;<sup><a href="#CR23" rid="CR23" class=" bibr popnode">23</a></sup></sup>. CNAs, such as gains of the long arms of chromosomes 17 (chr17q) and 1 (chr1q) have been identified in the majority (up to 65%) of NB tumours<sup><sup><a href="#CR24" rid="CR24" class=" bibr popnode">24</a></sup>&#x02013;<sup><a href="#CR28" rid="CR28" class=" bibr popnode">28</a></sup></sup> and their emergence is considered an early tumourigenesis &#x0201c;priming&#x0201d; event<sup><sup><a href="#CR29" rid="CR29" class=" bibr popnode">29</a></sup></sup>. Chr17q/1q gains often co-occur with amplification of the <em>MYCN</em> oncogene<sup><sup><a href="#CR24" rid="CR24" class=" bibr popnode">24</a></sup>,<sup><a href="#CR28" rid="CR28" class=" bibr popnode">28</a></sup>,<sup><a href="#CR30" rid="CR30" class=" bibr popnode">30</a></sup>&#x02013;<sup><a href="#CR33" rid="CR33" class=" bibr popnode">33</a></sup></sup> (at least one CNA in &#x0003e;95% of <em>MYCN</em>-amplified tumours<sup><sup><a href="#CR34" rid="CR34" class=" bibr popnode">34</a></sup></sup>), suggesting they may jointly contribute to tumourigenesis. However, despite our advanced understanding of the genetic and developmental origin of NB, it remains unclear to date how CNAs disrupt embryonic cell differentiation and lead to NB initiation.</p><p id="Par5" class="p p-last">Here, we used a human embryonic stem cell (hESC)-based model to experimentally dissect the links between NB-associated CNAs, <em>MYCN</em> amplification, and tumour initiation. We interrogated the stepwise specification of trunk NC and sympathoadrenal lineages using directed differentiation of isogenic hESC lines with chr17q/1q gains and inducible <em>MYCN</em> overexpression. We found that (i) CNAs derail differentiation by potentiating immature NC progenitor phenotypes. Combining CNAs with <em>MYCN</em> overexpression completely disrupted normal NC differentiation; (ii) Mutant NC cells acquired tumourigenic hallmarks in vitro, the capacity to form tumours in xenografts, and resemble distinct subpopulations of heterogeneous NB tumours; (iii) An extensive re-wiring of chromatin connects the observed transcriptional and functional aberrations with a dysregulated network of developmental TFs. Collectively, our data put forward a CNA-driven distortion of trunk NC and sympathoadrenal differentiation as a priming mechanism for subsequent MYCN-induced tumour initiation.</p></div><div id="Sec2" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec2title">Results</h2><div id="Sec3" class="sec sec-first"><h3 id="Sec3title">Differentiation of human embryonic stem cells recapitulates key stages of trunk NC and sympathoadrenal development</h3><p id="Par6" class="p p-first">To model the initiation stage and cell types relevant to NB tumourigenesis, we turned to an in-vitro modelling approach. We have previously described an efficient strategy to produce human trunk NC, sympathoadrenal progenitors, and sympathetic neurons from hESCs<sup><sup><a href="#CR35" rid="CR35" class=" bibr popnode">35</a></sup>,<sup><a href="#CR36" rid="CR36" class=" bibr popnode">36</a></sup></sup>. Our protocol involves treatment with defined cocktails of signalling pathway agonists/antagonists that induce neuromesodermal-potent axial progenitors (NMPs) at day 3 of differentiation (D3)<sup><sup><a href="#CR37" rid="CR37" class=" bibr popnode">37</a></sup></sup>, and subsequently steer NMPs toward trunk NC cells (D9) and their sympathoadrenal derivatives (&#x0003e;D14). At D19, the protocol yields catecholamine-producing sympathetic neurons marked by peripherin-expressing axons<sup><sup><a href="#CR35" rid="CR35" class=" bibr popnode">35</a></sup></sup> (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1a</span></a>, Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">1</a></sup>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig1" co-legend-rid="lgnd_Fig1"><a href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" rid-figpopup="Fig1" rid-ob="ob-Fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139783098352864"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11068915_41467_2024_47945_Fig1_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2024_47945_Fig1_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11068915/bin/41467_2024_47945_Fig1_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139783098352864"><a target="object" rel="noopener" href="/pmc/articles/PMC11068915/figure/Fig1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig1"><div><a class="figpopup" href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" rid-figpopup="Fig1" rid-ob="ob-Fig1">Fig. 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>In-vitro culture efficiently generates human trunk NC cells and their sympathoadrenal derivatives from hESCs.</strong><p><strong>a</strong> Diagram depicting (top) the extrinsically supplemented signals employed to direct hESCs toward trunk NC cells and their downstream derivatives, sample harvesting time points for scRNA-seq analysis (the colours indicated in this schematic are used throughout the manuscript to refer to the respective days), and immunofluorescence analysis (bottom) of PERIPHERIN protein expression illustrating the generation of sympathetic neurons at D19. Cell nuclei were counterstained using Hoechst 33342. All experiments were repeated at least three times with similar results. <strong>b</strong> UMAP of scRNA-seq data from wild-type hESCs collected at 9 stages (indicated by different colours) of differentiation to trunk neural crest and sympathoadrenal derivatives. Cells were divided into 14 distinct clusters as indicated by the contours. <strong>c</strong> Heatmap of gene markers for each cluster in (<strong>b</strong>). Selected genes have been highlighted and UMAPs indicate the expression level of canonical markers for stem (<em>POU5F1</em>), neural crest (<em>SOX10</em>), mesenchymal (<em>FN1</em>), and sympathetic (<em>PHOX2B</em>) cells. All marker genes are reported in Supplementary Data&#x000a0;<sup><a href="#MOESM4" rid="MOESM4" class=" media">2</a></sup>. <strong>d</strong> UMAP as in (<strong>b</strong>), labelled by their closest matching cell type from the human embryonic adrenal gland reference<sup><sup><a href="#CR16" rid="CR16" class=" bibr popnode">16</a></sup></sup> via label transfer. Cells in grey could not be verified with markers (Supplementary Fig. <sup><a href="#MOESM2" rid="MOESM2" class=" media">2i</a></sup>) or could not be assigned to a single type. <strong>e</strong> Cells from panel d coloured by the strength of their SCP marker signature (Seurat module score) in red. The score distinguishes a cluster of early SCP-like/trunk NC (low score) and a late cluster with more mature SCP-like cells (high score). <strong>f</strong> Same as above but visualising simultaneously SYM (orange) and MES (teal) marker signature. Cells with overlapping marker signatures appear in grey tones, with the highest mixture in C12. An early diverging cluster of sensory neuron-like cells has a weak match to the SYM signature. A pseudotime trajectory for the MES-SYM transition in clusters C11-C14 can be found in Supplementary Fig. <sup><a href="#MOESM2" rid="MOESM2" class=" media">3</a></sup>. Source data are provided as a <sup><a href="#MOESM1" rid="MOESM1" class=" media">Source Data</a></sup> file. hESC, human embryonic stem cells; D0/3/6/9/10/12/14/19/28, day 0/3/6/9/10/12/14/19/28; UMAP Uniform Manifold Approximation and Projection, C1-C14 cell clusters, SCP Schwann cell precursor, SYM sympathoblast, MES mesenchymal.</p></div></div></div><p id="Par7">As a prerequisite for studying the effects of CNAs on trunk NC differentiation, we first needed to define a molecular roadmap of normal hESC differentiation as a control. Therefore, we employed our protocol for the differentiation of karyotypically normal hESCs (H7<sup><sup><a href="#CR38" rid="CR38" class=" bibr popnode">38</a></sup></sup>; 46XX) and performed droplet-based single-cell RNA sequencing (scRNA-seq) at key differentiation stages (D0 &#x02248; hESCs, D3 &#x02248; NMPs, D9 &#x02248; trunk NC, D14 &#x02248; sympathoadrenal progenitors, D19 &#x02248; early sympathetic neurons) and intermediate/late time points (D6, D10, D12, D28) to examine the resulting cell populations (up to five replicates each; Supplementary Data&#x000a0;<sup><a href="#MOESM5" rid="MOESM5" class=" media">1</a></sup>). We obtained 29,857 cells that passed quality control, which we allocated to 14 distinct clusters (C1-C14) (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1b</span></a>; Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">2a&#x02013;g</a></sup>). We bioinformatically annotated these cell clusters using two complementary approaches: (i) by identifying characteristic marker genes (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1c</span></a>; Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">2h</a></sup>; Supplementary Data&#x000a0;<sup><a href="#MOESM6" rid="MOESM6" class=" media">2</a></sup>), and (ii) by mapping our data to single-cell transcriptomes of trunk NC derivatives in human embryos<sup><sup><a href="#CR15" rid="CR15" class=" bibr popnode">15</a></sup>,<sup><a href="#CR16" rid="CR16" class=" bibr popnode">16</a></sup></sup> (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1d&#x02013;f</span></a>, Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">2i, j</a></sup>). This strategy identified cells at different stages of trunk NC development, including NMP-like cells (marked by <em>CDX1/2</em>, <em>NKX1-2</em>, and FGF signalling-associated transcripts<sup><sup><a href="#CR37" rid="CR37" class=" bibr popnode">37</a></sup></sup>; cluster C2 in Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1c</span></a>, Supplementary Data&#x000a0;<sup><a href="#MOESM6" rid="MOESM6" class=" media">2</a></sup>) and later cell populations of a predominantly trunk axial identity (Supplementary Fig.&#x000a0;<sup><a href="#MOESM1" rid="MOESM1" class=" media">2h</a></sup>) exhibiting characteristics of Schwann cell precursors (SCP), sympathoblasts (SYM), as well as mesenchymal features (MES). For example, D9 cells split into subpopulations expressing markers of trunk NC/early SCPs (C3; e.g., <em>SOX10</em><sup><sup><a href="#CR16" rid="CR16" class=" bibr popnode">16</a></sup></sup>, Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1c</span></a>; weak SCP-like signature, Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1e</span></a>) and sensory neurons (C5; <em>ONECUT1</em><sup><sup><a href="#CR39" rid="CR39" class=" bibr popnode">39</a></sup></sup>, Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1c</span></a>; weak SYM-like signature, Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1f</span></a>). At D14, cells started to assume a sympathoadrenal/autonomic progenitor (C8; <em>ASCL1</em>) or mesenchymal (C11; <em>FN1</em>) identity, and by day D19, we observed three distinct fractions: mature SCP-like cells (C9; <em>POSTN</em><sup><sup><a href="#CR40" rid="CR40" class=" bibr popnode">40</a></sup></sup>; strong SCP signature), autonomic sympathoblasts (C12-C14; <em>PHOX2A/B</em>, <em>ELAVL4</em><sup><sup><a href="#CR16" rid="CR16" class=" bibr popnode">16</a></sup>,<sup><a href="#CR41" rid="CR41" class=" bibr popnode">41</a></sup></sup>; strong SYM signature), and MES-like cells (C11; <em>COL1A1, FN1</em>). This is in line with findings showing that trunk NC and SCPs are competent to generate mesenchyme<sup><sup><a href="#CR40" rid="CR40" class=" bibr popnode">40</a></sup>,<sup><a href="#CR42" rid="CR42" class=" bibr popnode">42</a></sup>,<sup><a href="#CR43" rid="CR43" class=" bibr popnode">43</a></sup></sup>. Interestingly, we also found cells at the intersection of MES and SYM identity, as observed in mice<sup><sup><a href="#CR39" rid="CR39" class=" bibr popnode">39</a></sup></sup> and NB cell lines<sup><sup><a href="#CR44" rid="CR44" class=" bibr popnode">44</a></sup>&#x02013;<sup><a href="#CR48" rid="CR48" class=" bibr popnode">48</a></sup></sup> (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">3</a></sup>; Supplementary Data&#x000a0;<sup><a href="#MOESM7" rid="MOESM7" class=" media">3</a></sup>). After 4 weeks (D28), we also observed some cells with a partial chromaffin-like cell identity (part of C14; <em>CHGA</em><sup>+</sup>, PMNT<sup>-</sup>) (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1d</span></a>).</p><p id="Par8" class="p p-last">Together, these data confirm that our hESC-based model successfully captures trunk NC and sympathoadrenal cells as found in embryos during the onset of NB tumourigenesis. Moreover, they reveal two major developmental branching events: (i) an early commitment of trunk NC toward a sensory neuron fate; (ii) the late generation of multipotent SCP/sympathoadrenal progenitors, which subsequently give rise to three distinct cell types: mature SCPs, MES, and SYM.</p></div><div id="Sec4" class="sec"><h3 id="Sec4title">CNAs and <em>MYCN</em> cumulatively disrupt human trunk NC differentiation</h3><p id="Par9" class="p p-first">Having established a reliable model of trunk NC lineages relevant for NB pathogenesis, we next asked how chr17q and chr1q gains and their interplay with <em>MYCN</em> overexpression, which often co-occur in NB<sup><sup><a href="#CR24" rid="CR24" class=" bibr popnode">24</a></sup>,<sup><a href="#CR28" rid="CR28" class=" bibr popnode">28</a></sup>,<sup><a href="#CR30" rid="CR30" class=" bibr popnode">30</a></sup>&#x02013;<sup><a href="#CR34" rid="CR34" class=" bibr popnode">34</a></sup></sup>, influence NC development. To this end, we employed two clonal isogenic hESC lines with NB-associated CNAs that were acquired by H7 hESCs (&#x0201c;WT&#x0201d;; used in Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1</span></a>) as a result of culture adaptation<sup><sup><a href="#CR49" rid="CR49" class=" bibr popnode">49</a></sup></sup> (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2a</span></a>; Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">4a</a></sup>): (i) a gain of chromosome arm 17q11-qter (&#x0201c;17q&#x0201d;)<sup><sup><a href="#CR50" rid="CR50" class=" bibr popnode">50</a></sup></sup>, and (ii) an additional gain of chr1q in the 17q background (&#x0201c;17q1q&#x0201d;). Whole-exome sequencing of 17q and 17q1q cells compared to the parental H7 hESCs revealed a small number of additional mutations and a loss of a small region in chromosome 2 (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">4b</a></sup>; Supplementary Data&#x000a0;<sup><a href="#MOESM8" rid="MOESM8" class=" media">4</a></sup>, <sup><a href="#MOESM9" rid="MOESM9" class=" media">5</a></sup>). For brevity, we labelled the cell lines by their major CNAs, which overlap regions commonly gained in NB tumours<sup><sup><a href="#CR51" rid="CR51" class=" bibr popnode">51</a></sup></sup> (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">4c</a></sup>). 17q1q hESCs were engineered to include a Doxycycline (DOX)-inducible <em>MYCN</em> expression cassette to mimic <em>MYCN</em> amplification in a temporally controlled manner (&#x0201c;17q1qMYCN&#x0201d;). DOX treatment of 17q1qMYCN resulted in robust induction of MYCN, similar to expression levels in <em>MYCN</em>-amplified tumours (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">4d&#x02013;f</a></sup>). In our experiments, we induced <em>MYCN</em> overexpression at D5 (when cells adopt a definitive NC identity<sup><sup><a href="#CR35" rid="CR35" class=" bibr popnode">35</a></sup></sup>) to avoid bias toward central nervous system differentiation, as seen upon <em>MYCN</em> overexpression in earlier pre-NC progenitors<sup><sup><a href="#CR52" rid="CR52" class=" bibr popnode">52</a></sup></sup>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig2" co-legend-rid="lgnd_Fig2"><a href="/pmc/articles/PMC11068915/figure/Fig2/" target="figure" rid-figpopup="Fig2" rid-ob="ob-Fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139783110522240"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11068915_41467_2024_47945_Fig2_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2024_47945_Fig2_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11068915/bin/41467_2024_47945_Fig2_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139783110522240"><a target="object" rel="noopener" href="/pmc/articles/PMC11068915/figure/Fig2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig2"><div><a class="figpopup" href="/pmc/articles/PMC11068915/figure/Fig2/" target="figure" rid-figpopup="Fig2" rid-ob="ob-Fig2">Fig. 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Copy number alterations and overexpression of <em>MYCN</em> impair the specification of trunk NC derivatives.</strong><p><strong>a</strong> Scheme depicting the different hESC genetic backgrounds employed, the time points of sample collection, and the timing of DOX-induced <em>MYCN</em> overexpression during trunk NC differentiation. <strong>b</strong> scRNA-seq data from mutant cells (17q, 17q1q, 17q1qMYCN at D9, D14, and D19; coloured by stage) were mapped to the WT reference (cp. Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1</span></a>; undifferentiated hESC cluster C0 was excluded for simplicity). Glasswork UMAP plots depicting the destination clusters of mapped cells. Fewer 17q1q and 17q1qMYCN cells mapped to late stages (highlighted by arrows). <strong>c</strong> Alluvial plots comparing the stage at which each mutant cell was harvested versus its transcriptionally closest stage in the WT reference (based on label transfer as in <strong>b</strong>). In each subplot, the top bar indicates the proportion of cells collected at each stage (D9, D14, D19). The bottom bar indicates the distribution of matches for the same cells in WT, and streams indicate which subpopulations flow into cognate or non-cognate WT stages. The plots suggest that cells from 17q1q/17q1qMYCN mapped to earlier stages compared to WT. <strong>d</strong> Glasswork UMAPs of mapped 17q, 17q1q, and 17q1qMYCN cells (as in <strong>b</strong>) coloured by closest-matching cell type in an adrenal gland reference<sup><sup><a href="#CR16" rid="CR16" class=" bibr popnode">16</a></sup></sup>. The category &#x0201c;other&#x0201d; comprises other cell types in the reference dataset and mappings that could not be validated by cell type markers (Supplementary Fig. <sup><a href="#MOESM2" rid="MOESM2" class=" media">2i</a></sup>). <strong>e</strong> Percentage of cells mapped to each cell type in (<strong>d</strong>), split by cell line. <strong>f</strong> Violin plots indicating the strength of the SCP/SYM/MES (left to right) signature (Seurat module score) for cells mapped to the respective cell type, split by cell line. <strong>g</strong> Plot indicating the change in mean expression (colour) and the percentage of cells expressing the gene (size) for each gene in the signatures from panel e relative to WT. WT squares (size = 1, white) are shown for reference. <strong>h</strong> Flow cytometric analysis of trunk NC markers HOXC9 and SOX10 of WT, 17q, 17q1q, and 17q1qMYCN at D9. All experiments were repeated at least three times with similar results. Source data are provided as a <sup><a href="#MOESM1" rid="MOESM1" class=" media">Source Data</a></sup> file. WT wild-type H7 hESCs; D0/6/3/9/10/12/14/19/28, day 0/6/3/9/10/12/14/19/28; UMAP Uniform Manifold Approximation and Projection, SCP Schwann cell precursor, SYM sympathoblast, MES mesenchymal.</p></div></div></div><p id="Par10">Equipped with these three isogenic mutant hESC lines, we performed differentiation toward trunk NC and carried out scRNA-seq as described above, yielding a combined dataset comprising 95,766 cells (Supplementary Data&#x000a0;<sup><a href="#MOESM5" rid="MOESM5" class=" media">1</a></sup>). To assess how differentiation was affected in each mutant cell line, we first focused on stages D9, D14, and D19 for which we had data from all four experimental conditions, and bioinformatically mapped the transcriptomes of mutant cells to our reference of normal trunk NC differentiation (cp. Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1b</span></a>). While many 17q cells intertwined with all WT cell types (~98% matching the cognate WT stage), fewer 17q1q and 17q1qMYCN cells advanced beyond WT D14 (only ~48% and ~22% matched with WT, respectively; Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2b</span></a>). Only ~4% of 17q1qMYCN cells mapped to mature cell types (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2b</span></a>). Altogether, at this level of resolution, we found no evidence that 17q affected differentiation. In contrast, 17q1q and 17q1qMYCN cells matched WT cells of earlier developmental stages, suggesting impaired differentiation (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2c</span></a>).</p><p id="Par11">Next, we tested whether the cell types induced from mutant hESCs still truthfully recapitulated in-vivo cell types as seen for WT. Mapping mutant cells onto the same human embryonic adrenal gland reference<sup><sup><a href="#CR16" rid="CR16" class=" bibr popnode">16</a></sup></sup> identified proportionally fewer SYM- and MES-like cells in 17q1q and 17q1qMYCN (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2d, e</span></a>). For cells mapped to the respective cell types, we observed a slightly stronger SCP signature in 17q and 17q1q, while the expression of both MES and SYM genes was weaker relative to the WT (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2f</span></a>). In 17q1qMYCN, the expression of all signatures was weak, suggesting a failure to fully specify the expected cell types (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2d&#x02013;g</span></a>). Consistently, antibody staining for SOX10 and HOXC9 and flow cytometry revealed depletion of SOX10+ trunk NC cells in 17q1qMYCN cultures (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2h</span></a>; Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">5</a></sup>). The reduced ability of 17q1qMYCN hESCs to differentiate toward trunk NC derivatives was also reflected by their failure to generate PERIPHERIN-positive neuronal axons (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">4g</a></sup>). A similar, albeit milder effect was observed upon DOX-induced <em>MYCN</em> overexpression at later timepoints (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">4h</a></sup>).</p><p id="Par12">Differential analysis identified 941 (17q vs. WT), 2039 (17q1q vs. WT), and 5915 (17q1qMYCN vs. WT) differentially expressed genes (DEG) at D9 (Supplementary Data&#x000a0;<sup><a href="#MOESM10" rid="MOESM10" class=" media">6</a></sup>). As expected, many upregulated genes were located within the known CNAs (41.4% within chr17q for 17q cells; 18.7% within chr17q and 25.6% within chr1q for 17q1q cells; Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">6a</a></sup>). Pathway analysis indicated an enrichment of genes related to E2F and MTORC1 signalling components for DEGs on chr17q (e.g., <em>BRCA1</em>, <em>NME1</em>), and of apoptosis-related and members of the p53 pathway on chr1q (e.g., the anti-apoptotic regulator <em>MCL1</em>; Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span>3a&#x02013;c</span></a>; Supplementary Data&#x000a0;<sup><a href="#MOESM11" rid="MOESM11" class=" media">7</a></sup>). Notably, genes upregulated in 17q1q also include the p53 inhibitor <em>MDM4</em><sup><sup><a href="#CR53" rid="CR53" class=" bibr popnode">53</a></sup></sup> (Supplementary Data&#x000a0;<sup><a href="#MOESM10" rid="MOESM10" class=" media">6</a></sup>). These perturbed pathways may contribute to deregulation of expression of genes outside CNAs (e.g., upregulation of MYC targets and oxidative phosphorylation, and downregulation of G2-M checkpoint-related genes in 17q1qMYCN; Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span>3a</span></a>). To better resolve the molecular impact of each mutation, we integrated all datasets into a joint projection of WT and aberrant trunk NC differentiation (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4a</span></a>; Supplementary Figs.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">6b&#x02013;h</a></sup>; Supplementary Data&#x000a0;<sup><a href="#MOESM12" rid="MOESM12" class=" media">8</a></sup>). The strongest changes were observed in 17q1qMYCN, which formed disconnected cell clusters not normally produced in our protocol. To delineate the stepwise alteration of transcriptional programmes, we placed cells from D9 on a spectrum from WT to 17q1qMYCN by scoring each cell between 0 and 1 based on the fraction of mutant cells among its gene expression neighbours (&#x0201c;mutation score&#x0201d;; Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4b</span></a>). This allowed us to identify four sets of genes (D9_1&#x02013;D9_4) that were correlated with mutations (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4c</span></a>, Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">7a, b</a></sup>; Supplementary Data&#x000a0;<sup><a href="#MOESM13" rid="MOESM13" class=" media">9</a></sup>): Gain of CNAs led to a decrease in expression of genes (gene set D9_3, Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4c</span></a><em>)</em> involved in NC development (e.g., <em>TFAP2B</em><sup><sup><a href="#CR54" rid="CR54" class=" bibr popnode">54</a></sup>,<sup><a href="#CR55" rid="CR55" class=" bibr popnode">55</a></sup></sup>) and gradual induction of genes (D9_4, Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4c</span></a>) associated with NC/NB cell migration (e.g., <em>ZIC2</em>, <em>HOXD3</em>, <em>GPC3</em><sup><sup><a href="#CR56" rid="CR56" class=" bibr popnode">56</a></sup>&#x02013;<sup><a href="#CR58" rid="CR58" class=" bibr popnode">58</a></sup></sup>). <em>MYCN</em> overexpression in 17q1qMYCN further repressed genes related to NC development (D9_2; e.g., WNT-antagonist <em>SFRP1</em><sup><sup><a href="#CR59" rid="CR59" class=" bibr popnode">59</a></sup></sup> and nuclear receptors <em>NR2F1/2</em><sup><sup><a href="#CR60" rid="CR60" class=" bibr popnode">60</a></sup></sup>) and led to upregulation of MYCN targets implicated in NB (D9_1; e.g., <em>NME1</em> on 17q<sup><sup><a href="#CR61" rid="CR61" class=" bibr popnode">61</a></sup></sup>; Supplementary Data&#x000a0;<sup><a href="#MOESM13" rid="MOESM13" class=" media">9</a></sup>). Interestingly, we had also found <em>SFRP1</em> and <em>NR2F1</em> to mark the SYM-MES transition state in WT differentiating sympathoadrenal cells (cp. Supplementary Data&#x000a0;<sup><a href="#MOESM7" rid="MOESM7" class=" media">3</a></sup>). Moreover, we found that many of genes that are upregulated in 17q1qMYCN (D9_1) were also highly expressed in NB tumours (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">7c</a></sup>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig3" co-legend-rid="lgnd_Fig3"><a href="/pmc/articles/PMC11068915/figure/Fig3/" target="figure" rid-figpopup="Fig3" rid-ob="ob-Fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139783160206400"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11068915_41467_2024_47945_Fig3_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2024_47945_Fig3_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11068915/bin/41467_2024_47945_Fig3_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139783160206400"><a target="object" rel="noopener" href="/pmc/articles/PMC11068915/figure/Fig3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig3"><div><a class="figpopup" href="/pmc/articles/PMC11068915/figure/Fig3/" target="figure" rid-figpopup="Fig3" rid-ob="ob-Fig3">Fig. 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Copy number alterations and <em>MYCN</em> overexpression influence cancer-associated pathways.</strong><p><strong>a</strong> We performed differential expression analysis between WT and mutant cells at D9 (two-sided tests using DESeq2<sup><sup><a href="#CR147" rid="CR147" class=" bibr popnode">147</a></sup></sup> and aggregated pseudobulk counts per replicate<sup><sup><a href="#CR133" rid="CR133" class=" bibr popnode">133</a></sup></sup>; <em>P</em> values were corrected for multiple hypothesis testing using the Benjamini-Hochberg method; <em>P</em><sub>adj</sub>&#x02009;&#x02264;&#x02009;0.05, |log<sub>2</sub>FoldChange|&#x02009;&#x0003e;&#x02009;0.25) and summarised differentially expressed genes (DEG) using pathway enrichment analysis. Enrichment was determined by hypergeometric tests (hypeR<sup><sup><a href="#CR135" rid="CR135" class=" bibr popnode">135</a></sup></sup>, background: all detected genes in our scRNA-seq dataset) using pathways from MSigDB<sup><sup><a href="#CR134" rid="CR134" class=" bibr popnode">134</a></sup></sup>. The overlap between up- and down-regulated DEGs with the pathway genes is indicated as a positive (green/orange/magenta colour bars) or negative (grey colour) number, respectively. We additionally distinguished between DEGs located on chromosome arms chr17q, chr1q, or anywhere else in the genome to analyse potential direct and indirect effects of CNAs (split from top to bottom). All adjusted <em>P</em> values for enriched terms are shown (<em>P</em><sub>adj</sub>&#x02009;&#x02264;&#x02009;0.1; <em>P</em> values were adjusted for multiple hypothesis testing using the Benjamini-Hochberg method). Selected pathways are shown in the figure and all DEGs and pathway enrichments are available in Supplementary Data&#x000a0;<sup><a href="#MOESM10" rid="MOESM10" class=" media">6</a></sup> and&#x000a0;<sup><a href="#MOESM11" rid="MOESM11" class=" media">7</a></sup>. DEGs located on chr17q (<strong>b</strong>) and chr1q (<strong>c</strong>) from the enriched pathways in (<strong>a</strong>). The heatmap indicates the mean normalised expression difference between each indicated mutant cell line and WT (at D9). The annotation bars on top of the heatmaps indicate membership (black colour) of genes in the selected pathways (MSigDB hallmark database). Source data are provided as a <sup><a href="#MOESM1" rid="MOESM1" class=" media">Source Data</a></sup> file. WT, wild-type H7 hESCs; MUT, a mutant hESC line (17q, 17q1q, or 17q1qMYCN); D9, day 9.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig4" co-legend-rid="lgnd_Fig4"><a href="/pmc/articles/PMC11068915/figure/Fig4/" target="figure" rid-figpopup="Fig4" rid-ob="ob-Fig4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139783097421824"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11068915_41467_2024_47945_Fig4_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2024_47945_Fig4_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11068915/bin/41467_2024_47945_Fig4_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139783097421824"><a target="object" rel="noopener" href="/pmc/articles/PMC11068915/figure/Fig4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig4"><div><a class="figpopup" href="/pmc/articles/PMC11068915/figure/Fig4/" target="figure" rid-figpopup="Fig4" rid-ob="ob-Fig4">Fig. 4</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Neuroblastoma-like genetic aberrations cumulatively distort trunk NC differentiation.</strong><p><strong>a</strong> Top: UMAP of scRNA-seq data from WT and mutant hESCs (indicated by colour) throughout differentiation. Bottom: the same dataset coloured by closest-matching cell type in an adrenal gland reference<sup><sup><a href="#CR16" rid="CR16" class=" bibr popnode">16</a></sup></sup>. The category &#x0201c;other&#x0201d; comprises other cell types in the reference dataset and mappings that could not be validated by cell type markers (Supplementary Fig. <sup><a href="#MOESM2" rid="MOESM2" class=" media">2i</a></sup>). <strong>b</strong> Illustration (top) of the calculation of mutation scores <em>m</em> as average score of each cell&#x02019;s neighbours (k-nearest neighbour average). In this calculation, each neighbour weighs in by its cell line (0&#x02009;=&#x02009;WT, 1/3&#x02009;=&#x02009;17q, 2/3&#x02009;=&#x02009;17q1q, 1&#x02009;=&#x02009;17q1qMYCN) such that the mutation score allows ordering cells from WT to MYCN mutation. Only cells from D9, D14, and D19 were used, for which data from all conditions were available. The actual scores are shown overlaid on the UMAP from panel a (bottom). <strong>c</strong> Heatmap showing the expression of the top genes correlated with the mutation score (from <strong>b</strong>) across all cells from D9. Genes have been divided into four groups by hierarchical clustering, and selected TFs, receptors, and ligands are highlighted in different colours. All correlated genes are reported in Supplementary Data&#x000a0;<sup><a href="#MOESM13" rid="MOESM13" class=" media">9</a></sup>. Genes located on chr17q or chr1q are indicated. Source data are provided as a <sup><a href="#MOESM1" rid="MOESM1" class=" media">Source Data</a></sup> file. WT, wild-type H7 hESCs; D9/14/19, day 9/14/19; UMAP Uniform Manifold Approximation and Projection, m mutation score, TF transcription factor, SCP Schwann cell precursor, SYM sympathoblast, MES mesenchymal.</p></div></div></div><p id="Par13">We further sought to disentangle the relative contributions of <em>MYCN</em> overexpression and CNAs to the observed differentiation block phenotype in 17q1q cells. To this end, we generated additional cell lines derived from WT and 17q H7 hESCs by equipping each with a DOX-inducible <em>MYCN</em> construct (WTMYCN, 17qMYCN; Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">8a</a></sup>). Moreover, we introduced the same inducible <em>MYCN</em> expression cassette into a second female hESC cell line<sup><sup><a href="#CR38" rid="CR38" class=" bibr popnode">38</a></sup></sup> (H9) which had independently acquired chr17q and 1q gains (H9-WT, H9-17q1q, H9-17q1qMYCN). The differentiation trajectories of these cell lines in the presence and absence of <em>MYCN</em> overexpression were interrogated using split-pool single-cell RNA-seq. To ensure consistency, we also included the previously analysed H7 cell lines (WT, 17q, 17q1q, 17q1qMYCN) and performed 2&#x02013;4 replicate experiments. We obtained a total of 45,546 cells (all D9) post-QC and mapped each dataset onto our WT reference, as we had done before (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">8b</a></sup>). Starting with gain of chr17q, we found a reduction in cells allocated to sensory neuronal differentiation (cluster C4 in Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1b, c</span></a>) and instead a slight increase in a transitional progeny (C7). With chr1q, we also saw an increase of cells in C10. On top of these changes, <em>MYCN</em> overexpression led to most cells allocating to earlier developmental stages including clusters C2 and C3 (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">8b, c</a></sup>) &#x02013; reflecting the differentiation block we had observed before (cp. Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2b, c</span></a>). The observed changes were consistent for derivatives of both parental hESC lines (H7 and H9). Analysis of marker gene expression associated with the altered cell clusters (C2, C4, C7, C10) in the different mutant cell lines identified an upregulation of genes like <em>AZIN1</em> in all <em>MYCN</em>-overexpressing cells that was not active in their wild-type or CNA-only counterparts (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">8d</a></sup>). Conversely, these cells downregulated developmental regulators. For instance, even though MYCN-overexpressing cells still expressed remnant gene signatures leading them to map to differentiating WT cell clusters they downregulated genes in developmental pathways like <em>HHIP</em> in cluster C7 or <em>WNT1</em> in C10. Moreover, the neurogenic capacity of the mutant trunk NC cells (as reflected by the presence of PERIPHERIN-positive neuronal axons) was found to be disrupted by <em>MYCN</em> overexpression primarily in the presence of CNAs, with the strongest phenotype being observed in the presence of both chr17q and chr1q gains (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">8e</a></sup>), in line with our earlier findings. Collectively, these data indicate that CNAs potentiate the <em>MYCN</em>-driven block to the induction of a trunk NC/sympathoadrenal identity.</p><p id="Par14" class="p p-last">We conclude that NB-associated CNAs alter the differentiation landscape of hESC-derived trunk NC lineages by promoting transitional progenitor states at the expense of mature sympathoadrenal cell types. In conjunction with <em>MYCN</em> elevation, they block differentiation and trigger atypical transcriptional programmes incompatible with normal trunk NC development.</p></div><div id="Sec5" class="sec"><h3 id="Sec5title">Impaired trunk NC differentiation correlates with acquisition of tumourigenic hallmarks</h3><p id="Par15" class="p p-first">We observed that ectopic <em>MYCN</em> induction altered the morphology of cultures by D14 only in the presence of CNAs as cells lost their ability to spread out and form neurites, and 17q1qMYCN cells even formed tight, dome-like colonies (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5a</span></a>). As this phenomenon is reminiscent of loss of contact inhibition, a cancer hallmark, we next examined whether CNAs/<em>MYCN</em> overexpression led to further cellular changes that are typical of tumourigenesis. We first carried out cell cycle analysis of trunk NC cells (D9) generated from each <em>MYCN</em>-overexpressing hESC line (from WT/17q/17q1q backgrounds) by monitoring EdU (5-ethynyl-2&#x000b4;-deoxyuridine) incorporation via flow cytometry. We found a significant increase in the proportion of cells in S-phase only when <em>MYCN</em> overexpression was combined with CNAs (<em>p</em>&#x02009;=&#x02009;0.0233 and <em>p</em>&#x02009;=&#x02009;0.0073 respectively; two-way ANOVA; Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5b</span></a>), indicating altered cell cycle and increased replication similar to NB tumours and cell lines<sup><sup><a href="#CR62" rid="CR62" class=" bibr popnode">62</a></sup>&#x02013;<sup><a href="#CR64" rid="CR64" class=" bibr popnode">64</a></sup></sup>. Immunofluorescence analysis of Ki-67 expression further showed that 17q1qMYCN and 17qMYCN cultures exhibited a higher proliferation rate by D14 compared to their CNA-only counterparts (<em>p</em>&#x02009;&#x0003c;&#x02009;0.0001 and <em>p</em>&#x02009;=&#x02009;0.0078, respectively; two-way ANOVA; Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5c</span></a>). We next tested how CNAs/<em>MYCN</em> influenced colony formation, another hallmark of tumourigenesis. Low-density plating of trunk NC cells (D9) and image analysis showed significantly increased clonogenicity (<em>p</em>&#x02009;=&#x02009;0.0109; two-way ANOVA) exclusively in 17q1qMYCN cells (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5d</span></a>). DOX treatment of the unedited parental wild type and chr17q gain cell lines had no effect (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">9a</a></sup>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig5" co-legend-rid="lgnd_Fig5"><a href="/pmc/articles/PMC11068915/figure/Fig5/" target="figure" rid-figpopup="Fig5" rid-ob="ob-Fig5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139783120824736"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11068915_41467_2024_47945_Fig5_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2024_47945_Fig5_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11068915/bin/41467_2024_47945_Fig5_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139783120824736"><a target="object" rel="noopener" href="/pmc/articles/PMC11068915/figure/Fig5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig5"><div><a class="figpopup" href="/pmc/articles/PMC11068915/figure/Fig5/" target="figure" rid-figpopup="Fig5" rid-ob="ob-Fig5">Fig. 5</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Impaired trunk NC specification correlates with acquisition of tumourigenic hallmarks.</strong><p><strong>a</strong> Representative brightfield images of D14 cultures following differentiation of hESCs with the indicated genotypes. All experiments were repeated at least three times with similar results. <strong>b</strong> Flow cytometric analysis of cell cycle in D9 cultures of each cell line. Top: Representative FACS plots. Bottom: Percentage of cells in each cell cycle stage. G1 (17q vs. 17qMYCN, <em>p</em>&#x02009;=&#x02009;0.0266 = *; 17q1q vs. 17q1qMYCN, <em>p</em>&#x02009;=&#x02009;0.0153 = *), S (17q vs. 17qMYCN, <em>p</em>&#x02009;=&#x02009;0.0233 = *; 17q1q vs. 17q1qMYCN, <em>p</em>&#x02009;=&#x02009;0.0073 = **). Only comparisons examining the effect of <em>MYCN</em> overexpression in different backgrounds are shown. <strong>c</strong> Immunofluorescence analysis of the cell proliferation marker KI-67 (green) in D14 cultures of each cell line. Cell nuclei were counterstained using Hoechst 33342 (blue). Left: Representative images. Right: Percentage of KI-67-positive cells. 17q vs. 17qMYCN, <em>p</em>&#x02009;=&#x02009;0.0078 = **; 17q1q vs. 17q1qMYCN, <em>p</em>&#x02009;=&#x02009;0.0001 = ***. <strong>d</strong> Low-density plating of D9 cultures of each cell line (<em>n</em>&#x02009;=&#x02009;3 biological replicates). Left: Representative brightfield images after 84&#x02009;h. Right: Number of colonies counted after 5 days. 17q1q vs. 17q1qMYCN, <em>p</em>&#x02009;=&#x02009;0.0109 = *. <strong>b</strong>&#x02013;<strong>d</strong> Bar plots showing the mean of <em>n</em>&#x02009;=&#x02009;3 biological replicates (error bars = SD). Statistical analysis was performed using ordinary two-way (<strong>b</strong>, <strong>c</strong>) or one-way (<strong>d</strong>) ANOVA with Tukey correction. Only comparisons examining the effect of <em>MYCN</em> overexpression in different backgrounds are shown. <strong>e</strong> Phylogenetic tree indicating the genetic relationship and distance (number of SNVs detected by whole-exome sequencing) between different hESC lines before (D0) and after differentiation (D19). Node shape indicates samples without a <em>MYCN</em> overexpression cassette (unfilled circles), with an inactive cassette (filled circles), and with an activated (by addition of DOX from D5 onwards) cassette (filled squares). The colours match those used elsewhere in the paper, without specific meaning. Short distances (&#x0003c;10 mutations) between differentiated cells and the shared ancestor with the matching D0 samples suggest that few additional mutations occurred during differentiation. Supplementary Data&#x000a0;<sup><a href="#MOESM8" rid="MOESM8" class=" media">4</a></sup> and&#x000a0;<sup><a href="#MOESM9" rid="MOESM9" class=" media">5</a></sup> report SNVs/CNAs identified in our analyses. Source data are provided as a <sup><a href="#MOESM1" rid="MOESM1" class=" media">Source Data</a></sup> file. D0/5/14/19, day 0/5/14/19; WT, wild-type H7 hESCs, SD standard deviation, SNV single-nucleotide variant, CNA copy number alteration.</p></div></div></div><p id="Par16">Previous work has indicated that <em>MYCN</em> overexpression alone is associated with increased apoptosis in early sympathoadrenal cells<sup><sup><a href="#CR65" rid="CR65" class=" bibr popnode">65</a></sup>,<sup><a href="#CR66" rid="CR66" class=" bibr popnode">66</a></sup></sup> and can trigger tumourigenesis only in combination with additional mutations<sup><sup><a href="#CR13" rid="CR13" class=" bibr popnode">13</a></sup>,<sup><a href="#CR14" rid="CR14" class=" bibr popnode">14</a></sup>,<sup><a href="#CR67" rid="CR67" class=" bibr popnode">67</a></sup></sup>. Therefore, we also examined apoptosis levels during the transition of D9 trunk NC cells toward the SCP/sympathoblast stage (D14) by assessing cleaved Caspase-3 levels using flow cytometry. We found that <em>MYCN</em> overexpression indeed resulted in a higher rate of apoptosis in the WT background, while this increase was reversed in 17q cells (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">9b</a></sup>). However, this was not the case in cultures derived from <em>MYCN</em>-overexpressing cells with chr17q1q gains, which exhibited apoptosis levels similar to their <em>MYCN</em>-overexpressing WT counterparts (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">9b, c</a></sup>). This phenomenon may be linked to the presence of both pro- and anti-apoptotic genes in chromosome arm chr1q (cp. Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span>3c</span></a>) as well as increased DNA damage (assessed by the presence of &#x003b3;H2AX foci) following <em>MYCN</em> overexpression specifically in the 17q1q background (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">9d, e</a></sup>). Interestingly, we detected lower levels of DNA damage in the absence of MYCN overexpression in 17q and 17q1q trunk NC cells at D9 compared to WT controls suggesting a potential protective effect of 17q/1q gains (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">9d</a></sup>). Moreover, we investigated whether <em>MYCN</em>-overexpressing cells from different backgrounds had acquired additional mutations during differentiation. Whole-exome sequencing analysis at D19 of differentiation did not reveal any new large CNAs and detected only few mutations (&#x0003c;10 mutations with variant allele frequency &#x02265;0.2 between D19 and its common ancestor with D0; Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5e</span></a>, Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">9f</a></sup>; Supplementary Data&#x000a0;<sup><a href="#MOESM8" rid="MOESM8" class=" media">4</a></sup>). None of the discovered mutations have previously been reported in NB, leading us to conclude that the observed phenotypic changes in 17q1qMYCN were likely a product of the CNAs and <em>MYCN</em> overexpression rather than an expansion of new clonal cell populations with additional pathognomonic mutations. Despite an increase of proliferation (cp. Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5b, c</span></a>), <em>MYCN</em> overexpression did not yield more high-frequency mutations during the short timeframe of our differentiation experiments, consistent with earlier work in human neuroepithelial stem cells in vitro and after xenotransplantation<sup><sup><a href="#CR68" rid="CR68" class=" bibr popnode">68</a></sup></sup> (<em>p</em>&#x02009;=&#x02009;0.3458, two-sided, paired Wilcoxon test, <em>n</em>&#x02009;=&#x02009;3 per group; Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">9g</a></sup>).</p><p id="Par17">Finally, we sought to examine the tumourigenic potential of 17q1q-, 17q1qMYCN- and WT-derived trunk NC (D9 of differentiation) cells in vivo by xenografting them into immunodeficient NSG mice. We first injected aliquots of about 1 million cells subcutaneously into the recipient animals (<em>n</em>&#x02009;=&#x02009;6 per cell line) and monitored tumour volume over time. After 3&#x02013;5 weeks with continuous DOX administration, all 17q1qMYCN-injected mice developed visible tumours at the injection site (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig6" rid-ob="ob-Fig6" co-legend-rid="lgnd_Fig6"><span>6a</span></a>). In contrast, neither WT- nor 17q1q-injected control animals displayed any signs of tumours for up to 16 weeks (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig6" rid-ob="ob-Fig6" co-legend-rid="lgnd_Fig6"><span>6a</span></a>). Likewise, orthotopic injection into the adrenal gland (<em>n</em>&#x02009;=&#x02009;3 mice per condition) yielded tumour growths visible by magnetic resonance imaging (MRI) after 5 weeks only when <em>MYCN</em> overexpression was induced by DOX in 17q1qMYCN-grafted animals (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig6" rid-ob="ob-Fig6" co-legend-rid="lgnd_Fig6"><span>6b, c</span></a>). We found that both subcutaneous and adrenal xenograft-derived tumours consisted of undifferentiated, small round cells similar to tumours from transgenic Th-<em>MYCN</em> mice<sup><sup><a href="#CR12" rid="CR12" class=" bibr popnode">12</a></sup></sup> (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">10a</a></sup>). Complementary to our analysis in mice, we also performed exploratory xenografts of the same cell lines in zebrafish larvae. To this end, we labelled our 17q1qMYCN and WT cells at D9 with a fluorescent dye (CellTrace Violet) and injected them into the perivitelline space of zebrafish larvae on day 2 post fertilisation. Consistent with our results in mice, we found that 17q1qMYCN cells survived longer in zebrafish than WT, which had diminished in number at day 1 post injection (dpi) and were completely absent at 3 dpi (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">10b, c</a></sup>). In contrast, 17q1qMYCN cells survived until 3 dpi, with 16% of larvae even showing an increase in xenotransplant size. For comparison, injection of cells from a <em>MYCN</em>-amplified NB cell line (SK-N-BE2C-H2B-GFP<sup><sup><a href="#CR69" rid="CR69" class=" bibr popnode">69</a></sup></sup>) resulted in engraftment with subsequent tumour cell growth in 84% of larvae (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">10d</a></sup>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig6" co-legend-rid="lgnd_Fig6"><a href="/pmc/articles/PMC11068915/figure/Fig6/" target="figure" rid-figpopup="Fig6" rid-ob="ob-Fig6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139783118400688"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11068915_41467_2024_47945_Fig6_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2024_47945_Fig6_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11068915/bin/41467_2024_47945_Fig6_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139783118400688"><a target="object" rel="noopener" href="/pmc/articles/PMC11068915/figure/Fig6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig6"><div><a class="figpopup" href="/pmc/articles/PMC11068915/figure/Fig6/" target="figure" rid-figpopup="Fig6" rid-ob="ob-Fig6">Fig. 6</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>hESC-derived trunk NC cells with CNAs form tumours in mice upon <em>MYCN</em> overexpression.</strong><p><strong>a</strong> Left: Representative images of subcutaneous xenografts of trunk NC cells derived from the indicated cell lines in the presence (17q1qMYCN) and absence (WT, 17q1q) of DOX treatment. Right: Graph showing tumour growth per mouse corresponding to xenografts of indicated cell lines (<em>n</em>&#x02009;=&#x02009;6 animals per cell line). <strong>b</strong> Left: Representative MRI sections of mice at week 5 following xenografting of indicated cell lines in the adrenal gland and DOX treatment regimens. The white lines indicate the tumour perimeter. Right: Graph showing survival of the recipient animals after xenografting (<em>n</em>&#x02009;=&#x02009;3 animals per cell line). <strong>c</strong> Summary of mouse xenograft experiments. Source data are provided as a <sup><a href="#MOESM1" rid="MOESM1" class=" media">Source Data</a></sup> file. DOX doxycycline, MRI magnetic resonance imaging.</p></div></div></div><p id="Par18" class="p p-last">Together, our results demonstrate that CNA-carrying trunk NC cells transit into an undifferentiated pre-tumourigenic state and acquire altered cellular properties reminiscent of cancer hallmarks, such as increased proliferation, clonogenic and tumour formation capacity under the influence of <em>MYCN</em> overexpression. Our data also suggest that CNAs enhance the pro-tumourigenic effects of <em>MYCN</em>.</p></div><div id="Sec6" class="sec"><h3 id="Sec6title">In-vitro differentiation of mutant hESCs captures NB tumour cell heterogeneity</h3><p id="Par19" class="p p-first">Given that cells in our in-vitro model exhibit similarities to NB cells, we asked whether our data could provide insights into cellular heterogeneity in NB tumours. To this end, we collected scRNA-seq data from ten <em>MYCN</em>-amplified NB tumours from three independent sources<sup><sup><a href="#CR15" rid="CR15" class=" bibr popnode">15</a></sup>,<sup><a href="#CR17" rid="CR17" class=" bibr popnode">17</a></sup>,<sup><a href="#CR23" rid="CR23" class=" bibr popnode">23</a></sup></sup>. For each dataset, we curated <em>MYCN</em><sup>+</sup> tumour cells and bioinformatically mapped these to our reference (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig7" rid-ob="ob-Fig7" co-legend-rid="lgnd_Fig7"><span>7a</span></a>). For example, this approach matched most cells from tumour dataset <em>Jansky_NB14</em><sup><sup><a href="#CR15" rid="CR15" class=" bibr popnode">15</a></sup></sup> to clusters C13 and C14, which correspond to late SYM-like cell states (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig7" rid-ob="ob-Fig7" co-legend-rid="lgnd_Fig7"><span>7b</span></a>; cp. Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1</span></a>). A few cells also mapped to clusters C11 and C12, i.e., cells with MES-like characteristics. The observed transcriptional heterogeneity was surprising, given that most tumour cells appeared karyotypically homogeneous (including a chr17q gain) and expressed <em>MYCN</em> (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig7" rid-ob="ob-Fig7" co-legend-rid="lgnd_Fig7"><span>7b</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig7" co-legend-rid="lgnd_Fig7"><a href="/pmc/articles/PMC11068915/figure/Fig7/" target="figure" rid-figpopup="Fig7" rid-ob="ob-Fig7"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139783120867760"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11068915_41467_2024_47945_Fig7_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2024_47945_Fig7_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11068915/bin/41467_2024_47945_Fig7_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139783120867760"><a target="object" rel="noopener" href="/pmc/articles/PMC11068915/figure/Fig7/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig7"><div><a class="figpopup" href="/pmc/articles/PMC11068915/figure/Fig7/" target="figure" rid-figpopup="Fig7" rid-ob="ob-Fig7">Fig. 7</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Comparison to hESC-based trunk NC differentiation resolves structured heterogeneity across neuroblastoma tumours.</strong><p><strong>a</strong> We curated tumour cells from 10 <em>MYCN</em>-amplified NB samples<sup><sup><a href="#CR15" rid="CR15" class=" bibr popnode">15</a></sup>,<sup><a href="#CR17" rid="CR17" class=" bibr popnode">17</a></sup>,<sup><a href="#CR23" rid="CR23" class=" bibr popnode">23</a></sup></sup> from three studies and mapped them onto our WT in-vitro reference (cp. Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1</span></a>)<sup><sup><a href="#CR126" rid="CR126" class=" bibr popnode">126</a></sup></sup>. Mapping is represented as cells in each cluster of the reference (depicted as contours). <strong>b</strong> Heatmap depicting relative gene expression in dataset Jansky_NB14<sup><sup><a href="#CR15" rid="CR15" class=" bibr popnode">15</a></sup></sup>. Values are inferCNV<sup><sup><a href="#CR139" rid="CR139" class=" bibr popnode">139</a></sup></sup> copy number estimations per gene, relative to haematopoietic cells ordered by genomic position and chromosome (1&#x02013;22). Cells (one per row) are ordered by the respective cluster that they were mapped to (C11 to C14) and therein by <em>MYCN</em> expression levels (depth-normalised sliding window average, width = 20 cells). Mappings of other datasets are shown in Supplementary Fig. <sup><a href="#MOESM2" rid="MOESM2" class=" media">11</a></sup>. <strong>c</strong> Heatmap showing markers from gene expression signatures C4*, C5*, C9*, C13*, and C14* (rows, top to bottom) in cells from 10 tumour datasets that were mapped to our in-vitro reference (cp. <strong>a</strong>). Each signature is the intersection of the cluster markers in our reference (as in Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1</span></a>) and differentially expressed genes between the respective tumour cells. Markers for tumour cells mapped to C2 and C3 showed no overlap with in-vitro cluster markers; hence, only mapped cells are shown. All genes identified in this analysis are reported in Supplementary Data&#x000a0;<sup><a href="#MOESM14" rid="MOESM14" class=" media">10</a></sup>. <strong>d</strong> Scatterplots evaluating the strength of signatures C5* and C13* (from <strong>c</strong>; calculated using GSVA<sup><sup><a href="#CR148" rid="CR148" class=" bibr popnode">148</a></sup></sup>) in bulk RNA-seq data from SEQC<sup><sup><a href="#CR70" rid="CR70" class=" bibr popnode">70</a></sup>,<sup><a href="#CR71" rid="CR71" class=" bibr popnode">71</a></sup></sup>. Each dot indicates one tumour dataset coloured by <em>MYCN</em> amplification status (left) or clinical stage (right). The density of points (kernel density estimate) in each group is indicated in the margins. <strong>e</strong> Survival analysis for data from the SEQC cohort stratified by strength of the C13* signature. Groups were split by the median. Cox regression adjusted for age-group (&#x0003c;18, 18&#x02013;60, or &#x0003e;60 months), INSS stage 4 (yes/no), and <em>MYCN</em> amplification (yes/no). <em>n</em>&#x02009;=&#x02009;249 patients per group, or 136 [C13* low] and 47 [C13* high] events. All results are reported in Supplementary Data&#x000a0;<sup><a href="#MOESM15" rid="MOESM15" class=" media">11</a></sup>. Source data are provided as a <sup><a href="#MOESM1" rid="MOESM1" class=" media">Source Data</a></sup> file. UMAP Uniform Manifold Approximation and Projection, EFS event-free survival, INSS International Neuroblastoma Staging System, FDR false discovery rate.</p></div></div></div><p id="Par20">Extending the in-vitro reference mapping to all ten tumours portrayed a spectrum of <em>MYCN</em>-amplified cells with a majority C13- or C14-like expression profile, and a subset of cells mapping to other differentiating trunk NC cell states (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">11a, b</a></sup>). We observed apparent differences between studies and tumour types, but to date there is only a limited amount of single-cell data from NB tumours to robustly interpret such heterogeneity. We therefore sought to interrogate a large collection of bulk RNA-seq data from NB tumours (SEQC<sup><sup><a href="#CR70" rid="CR70" class=" bibr popnode">70</a></sup>,<sup><a href="#CR71" rid="CR71" class=" bibr popnode">71</a></sup></sup>). We first intersected the development-related gene signatures (C1-C14 from Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1</span></a>) with marker genes identified for the tumour cells that had been mapped to those respective clusters (from all 10 investigated samples; Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig7" rid-ob="ob-Fig7" co-legend-rid="lgnd_Fig7"><span>7c</span></a>; Supplementary Data&#x000a0;<sup><a href="#MOESM14" rid="MOESM14" class=" media">10</a></sup>) and labelled each refined signature with an asterisk to distinguish it from the original gene signature (e.g., signature C13* contained genes such as <em>DLC1</em> and <em>RORA;</em> Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig7" rid-ob="ob-Fig7" co-legend-rid="lgnd_Fig7"><span>7c</span></a>). Applying these gene signatures to the NB tumour data, we found that expression signatures C5* (sensory neuron-like cells) and C13* (differentiating SYM-like cells), jointly separated <em>MYCN</em>-amplified and non-amplified tumours, as well as tumours at different clinical stages (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig7" rid-ob="ob-Fig7" co-legend-rid="lgnd_Fig7"><span>7d</span></a>). The C13* signature effectively stratified tumours with a good and poor survival across the entire cohort even when corrected for INSS stage, <em>MYCN</em> amplification status, and age (Cox regression analysis with covariates; Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig7" rid-ob="ob-Fig7" co-legend-rid="lgnd_Fig7"><span>7e</span></a>; Supplementary Data&#x000a0;<sup><a href="#MOESM15" rid="MOESM15" class=" media">11</a></sup>).</p><p id="Par21" class="p p-last">Jointly, these observations demonstrate that our in-vitro model generates cell types that transcriptionally resemble different NB cell subpopulations and that it facilitates the systematic dissection of intra-tumour heterogeneity in NB tumours.</p></div><div id="Sec7" class="sec"><h3 id="Sec7title">CNAs and <em>MYCN</em> disrupt the configuration of NC regulatory circuits during differentiation</h3><p id="Par22" class="p p-first">NB tumours and cell lines are marked by a re-wiring of non-coding regulatory elements (e.g., enhancers) giving rise to tumour-specific regulatory circuitries<sup><sup><a href="#CR44" rid="CR44" class=" bibr popnode">44</a></sup>,<sup><a href="#CR45" rid="CR45" class=" bibr popnode">45</a></sup>,<sup><a href="#CR72" rid="CR72" class=" bibr popnode">72</a></sup>&#x02013;<sup><a href="#CR76" rid="CR76" class=" bibr popnode">76</a></sup></sup>. We therefore hypothesised that disruption of developmental TFs also underpins the aberrant differentiation observed in our mutant hESCs (cp. Figs.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2</span></a>, <a href="/pmc/articles/PMC11068915/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,3)</span></span><span>3</span></a>) and employed the assay for transposase-accessible chromatin followed by sequencing (ATAC-seq)<sup><sup><a href="#CR77" rid="CR77" class=" bibr popnode">77</a></sup></sup> to profile chromatin accessibility in the same samples used for scRNA-seq analysis (<em>n</em>&#x02009;=&#x02009;51; Supplementary Data&#x000a0;<sup><a href="#MOESM5" rid="MOESM5" class=" media">1</a></sup>). Chromatin accessibility serves as a proxy for the dynamic regulatory DNA element activity during differentiation. For instance, the promoters of the hESC regulator <em>POU5F1</em> and trunk NC regulator <em>SOX10</em> were accessible only at D0 and D9, respectively (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig8/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig8" rid-ob="ob-Fig8" co-legend-rid="lgnd_Fig8"><span>8a</span></a>), while the <em>PHOX2B</em> promoter exhibited reduced accessibility in 17q1q and 17q1qMYCN cells at D19 consistent with impaired differentiation (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig8/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig8" rid-ob="ob-Fig8" co-legend-rid="lgnd_Fig8"><span>8b</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig8" co-legend-rid="lgnd_Fig8"><a href="/pmc/articles/PMC11068915/figure/Fig8/" target="figure" rid-figpopup="Fig8" rid-ob="ob-Fig8"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139783097999424"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11068915_41467_2024_47945_Fig8_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2024_47945_Fig8_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11068915/bin/41467_2024_47945_Fig8_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139783097999424"><a target="object" rel="noopener" href="/pmc/articles/PMC11068915/figure/Fig8/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig8"><div><a class="figpopup" href="/pmc/articles/PMC11068915/figure/Fig8/" target="figure" rid-figpopup="Fig8" rid-ob="ob-Fig8">Fig. 8</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Copy number alterations and <em>MYCN</em> overexpression disrupt chromatin reconfiguration during trunk NC differentiation.</strong><p><strong>a</strong> ATAC-seq read coverage for WT cells at three example loci. The area reports the normalised read count aggregated per genomic bin (width&#x02009;=&#x02009;500&#x02009;bp). Multiple semi-transparent plots are overlaid for each replicate. Genes within each locus are shown on top with thin/thick lines indicating introns/exons. The arrows next to gene names indicate the direction of transcription. Selected peaks have been highlighted manually (dashed orange lines). <strong>b</strong> ATAC-seq read coverage of WT and mutant cells at D19 near the <em>PHOX2B</em> locus. Plots as in (<strong>a</strong>). <strong>c</strong> Principal component analysis of all ATAC-seq datasets, split by condition. Each data point represents one ATAC-seq sample with the shape indicating the stage at which the sample was collected. The geometric means of all data of the same stages are connected by arrows to visualise the stepwise chromatin changes. <strong>d</strong> Euler diagram visualising the overlap of differentially accessible regions in mutant hESC-derived trunk NC derivatives compared to WT (<em>DEseq2</em><sup><sup><a href="#CR147" rid="CR147" class=" bibr popnode">147</a></sup></sup>; <em>P</em><sub>adj</sub>&#x02009;&#x02264;&#x02009;0.005, |log<sub>2</sub>FoldChange|&#x02009;&#x02265;&#x02009;log<sub>2</sub>(1.5)). Numbers indicate the total number of regions per cell line aggregated over all stages. Source data are provided as a <sup><a href="#MOESM1" rid="MOESM1" class=" media">Source Data</a></sup> file. D0/3/9/14/19, day 0/3/9/14/19; WT, wild-type H7 hESCs.</p></div></div></div><p id="Par23">Unsupervised analysis of chromatin patterns on a global level showed that WT and 17q hESCs changed consistently throughout differentiation (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig8/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig8" rid-ob="ob-Fig8" co-legend-rid="lgnd_Fig8"><span>8c</span></a>). In contrast, 17q1q and 17q1qMYCN appeared not to follow the same path as WT in this low-dimensional projection, in line the differentiation defects observed in our previous analyses (cp. Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2b, c</span></a>). To delineate chromatin changes in detail, we performed differential accessibility analysis between all differentiation stages per cell line and between all cell lines at matched stages (Supplementary Data&#x000a0;<sup><a href="#MOESM16" rid="MOESM16" class=" media">12</a></sup>, <sup><a href="#MOESM17" rid="MOESM17" class=" media">13</a></sup>). As in our DEG analysis, we found an increasing number of regions with altered accessibility in 17q (<em>n</em>&#x02009;=&#x02009;477 regions), 17q1q (<em>n</em>&#x02009;=&#x02009;2826), and 17q1qMYCN (<em>n</em>&#x02009;=&#x02009;6663; Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig8/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig8" rid-ob="ob-Fig8" co-legend-rid="lgnd_Fig8"><span>8d</span></a>). In total, there were 45,583 regions with differential accessibility in at least one comparison, which we divided into nine chromatin modules R1-R9 (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig9/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig9" rid-ob="ob-Fig9" co-legend-rid="lgnd_Fig9"><span>9a</span></a>). Modules R1-R7 reflect differentiation order, e.g., regions in module R1 were mostly accessible at D0, and R6 comprises regions accessible at D14 and D19. Most changes observed in mutant hESC-derivatives fell within these modules (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">12a, b</a></sup>). 17q1q and 17q1qMYCN cells failed to close chromatin that is usually specific to D9 (R4, R5) and conversely to open chromatin regions indicative of late sympathoadrenal differentiation (R6, R7; Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">12c</a></sup>). Additionally, modules R8 and R9 comprised regions with reduced and increased accessibility in mutant hESC derivatives, respectively, independently of differentiation stage.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig9" co-legend-rid="lgnd_Fig9"><a href="/pmc/articles/PMC11068915/figure/Fig9/" target="figure" rid-figpopup="Fig9" rid-ob="ob-Fig9"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139783123509776"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11068915_41467_2024_47945_Fig9_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2024_47945_Fig9_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11068915/bin/41467_2024_47945_Fig9_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139783123509776"><a target="object" rel="noopener" href="/pmc/articles/PMC11068915/figure/Fig9/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig9"><div><a class="figpopup" href="/pmc/articles/PMC11068915/figure/Fig9/" target="figure" rid-figpopup="Fig9" rid-ob="ob-Fig9">Fig. 9</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Differentiation of wild-type and mutant hESCs is associated with changes in distinct chromatin modules.</strong><p><strong>a</strong> Heatmaps showing normalised read counts for all DARs (columns) in any pairwise comparison of two stages or conditions (<em>DEseq2</em><sup><sup><a href="#CR147" rid="CR147" class=" bibr popnode">147</a></sup></sup>; two-sided tests, <em>P</em> values adjusted for multiple hypothesis testing using the Benjamini-Hochberg method; <em>P</em><sub>adj</sub>&#x02009;&#x02264;&#x02009;0.005, |log<sub>2</sub>FoldChange|&#x02009;&#x02265;&#x02009;log<sub>2</sub>(1.5); <em>n</em><sub>total</sub>&#x02009;=&#x02009;45,583). Regions have been divided into nine non-overlapping modules (R1&#x02013;R9) by hierarchical clustering. Three annotation columns are shown to the right indicating regions called down- (blue) and up-regulated (red) in each condition. All DARs are reported in Supplementary Data&#x000a0;<sup><a href="#MOESM16" rid="MOESM16" class=" media">12</a></sup> and&#x000a0;<sup><a href="#MOESM17" rid="MOESM17" class=" media">13</a></sup>. <strong>b</strong> Comparisons of regions belonging to the nine chromatin modules (from <strong>a</strong>) and proximal cluster markers defined in our scRNA-seq analysis (cp. Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1</span></a>). An enrichment analysis was performed using hypergeometric tests (<em>hypeR</em><sup><sup><a href="#CR135" rid="CR135" class=" bibr popnode">135</a></sup></sup>; background: all genes associated with at least one ATAC-seq peak) and the size and transparency of circles indicate the odds ratio and <em>P</em> value, respectively. Significant results are indicated with filled circles (<em>P</em> values adjusted for multiple hypothesis testing using the Benjamini-Hochberg method; <em>P</em><sub>adj</sub>&#x02009;&#x02264;&#x02009;0.005). All results are shown in the figure, and reported in Supplementary Data&#x000a0;<sup><a href="#MOESM18" rid="MOESM18" class=" media">14</a></sup>. <strong>c</strong> Enrichment analysis for overlaps between chromatin modules and known TF motifs (<em>HOCOMOCO</em> database<sup><sup><a href="#CR154" rid="CR154" class=" bibr popnode">154</a></sup></sup>, v11). Statistical tests and plots are as in (<strong>b</strong>), with the exception that only overlaps with <em>P</em><sub>adj</sub>&#x02009;&#x02264;&#x02009;0.0000001 and |log<sub>2</sub>FoldChange|&#x02009;&#x02265;&#x02009;log<sub>2</sub>(2) were marked as significant (background: all peaks with at least one motif match). The top results per module are shown and all results are reported in Supplementary Data&#x000a0;<sup><a href="#MOESM18" rid="MOESM18" class=" media">14</a></sup>. <strong>d</strong> Enrichment analysis of overlaps between regions belonging to the nine chromatin modules and super-enhancers specific to certain NB epigenetic subtypes<sup><sup><a href="#CR44" rid="CR44" class=" bibr popnode">44</a></sup>,<sup><a href="#CR45" rid="CR45" class=" bibr popnode">45</a></sup>,<sup><a href="#CR73" rid="CR73" class=" bibr popnode">73</a></sup></sup> (background: all peaks with at least one overlapping region annotated in the super-enhancer analyses). Statistical tests and display as in (<strong>b</strong>). Source data are provided as a <sup><a href="#MOESM1" rid="MOESM1" class=" media">Source Data</a></sup> file. D0/9/19, day 0/9/19; WT, wild-type H7 hESCs; MUT a mutant hESC line (17q, 17q1q, or 17q1qMYCN); DAR, differentially accessible region; R1-R9, chromatin region modules; n.s., not significant; TF, transcription factor; NB, neuroblastoma; ADR, adrenergic; MES, mesenchymal; MNA-HR, not <em>MYCN</em>-amplified high-risk; MNA-LR, not <em>MYCN</em>-amplified low-risk.</p></div></div></div><p id="Par24">We sought to annotate our chromatin modules by looking for overlaps with genomic regions accessible in human tissues<sup><sup><a href="#CR78" rid="CR78" class=" bibr popnode">78</a></sup>&#x02013;<sup><a href="#CR80" rid="CR80" class=" bibr popnode">80</a></sup></sup> (Supplementary Data&#x000a0;<sup><a href="#MOESM18" rid="MOESM18" class=" media">14</a></sup>). In line with our transcriptome data, we found a stepwise change toward chromatin resembling differentiated tissues, e.g., neural tissues in R3-R5 and mesenchyme/stroma in R6/R7 (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">12d&#x02013;f</a></sup>). Next, we examined the identity of genes near the chromatin modules (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig9/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig9" rid-ob="ob-Fig9" co-legend-rid="lgnd_Fig9"><span>9b</span></a>). For each module, we found enrichments of specific marker genes identified in our scRNA-seq analysis of WT trunk NC differentiation (i.e., clusters C1-C14 from Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1b, c</span></a>). For example, chromatin module R7 (accessible in late differentiation stages, lost in mutants) was linked to clusters C11/C12 (MES-like cells). Next, we examined TF binding motifs in each module to identify potential upstream regulators (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig9/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig9" rid-ob="ob-Fig9" co-legend-rid="lgnd_Fig9"><span>9c</span></a>). We found an enrichment of known regulators of each developmental stage, e.g., TFs associated with trunk NC in R3-R5 (e.g., SOX10) and with sympathetic neuron development in R6/R7 (e.g., PHOX2A/B)<sup><sup><a href="#CR39" rid="CR39" class=" bibr popnode">39</a></sup></sup>. Moreover, we found enriched overlaps of modules R2/R4/R5/R8/R9 with super-enhancers identified in mesenchymal NB cell lines or adrenergic super-enhancers (in the case of R8), depending on the source annotation used<sup><sup><a href="#CR44" rid="CR44" class=" bibr popnode">44</a></sup>,<sup><a href="#CR45" rid="CR45" class=" bibr popnode">45</a></sup></sup>. Furthermore, R7 and R9 overlapped with super-enhancers associated with subsets of NB tumours<sup><sup><a href="#CR73" rid="CR73" class=" bibr popnode">73</a></sup></sup> with mesenchymal characteristics and with non-<em>MYCN</em>-amplified high-risk tumours, respectively (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig9/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig9" rid-ob="ob-Fig9" co-legend-rid="lgnd_Fig9"><span>9d</span></a>). No significant overlap was found with super-enhancers specific for <em>MYCN</em>-amplified NB. Finally, we examined the accessibility of each module across a range of NB cell lines (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">12g</a></sup>). As expected, we found that modules R1 and R2 (undifferentiated, early embryonic developmental stages) and modules R4 and R5 (early trunk NC to sympathoadrenal differentiation) were not accessible in NB cell lines, while modules R6-R8 (late sympathetic neurons and consistently open in mutants) were often highly accessible in cell lines. Interestingly, R3 (accessible at NMP and NC stage) was accessible in most NB cell lines examined except in those with mesenchymal characteristics (SK-N-AS and SHEP; Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">12g</a></sup>). Using data from other studies, we found that R6-R8 were also accessible in non-NB cell lines and tissues, while R3 was only found accessible in brain tissue (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">12g</a></sup>).</p><p id="Par25" class="p p-last">Together, our results suggest a systematic reprogramming of chromatin throughout trunk NC differentiation. In cells with CNAs and <em>MYCN</em> overexpression, this orderly reconfiguration of chromatin was severely disrupted in a manner similar to NB cells, providing a plausible mechanism for the link between the observed developmental defects and tumour initiation.</p></div><div id="Sec8" class="sec sec-last"><h3 id="Sec8title">CNA/<em>MYCN</em>-driven cell identity loss is mediated by sets of developmental transcription factors</h3><p id="Par26" class="p p-first">Finally, we investigated the links between CNA/<em>MYCN</em>-induced changes in chromatin dynamics, gene-regulatory networks, and the distorted differentiation trajectories observed at the transcriptional level. In our scRNA-seq analyses, we had recorded a stepwise alteration of expression from WT to 17q1qMYCN at D9 comprising four gene sets: D9_1 &#x02013; D9_4 (cp. Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4c</span></a>), which revealed <em>MYCN</em>-driven disruptions of early NC and the sensory neuron lineage specification. We hypothesised that these mutation-linked gene sets were also regulated by distinct TFs and therefore we employed an algorithm to identify TF targets based on correlated expression patterns<sup><sup><a href="#CR81" rid="CR81" class=" bibr popnode">81</a></sup></sup> (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig10/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig10" rid-ob="ob-Fig10" co-legend-rid="lgnd_Fig10"><span>10a</span></a>). This analysis identified <em>NR1D1</em> and <em>TFAP4</em> as putative TF targets of MYCN (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig10/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig10" rid-ob="ob-Fig10" co-legend-rid="lgnd_Fig10"><span>10b, c</span></a>; Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">13a, b</a></sup>; Supplementary Data&#x000a0;<sup><a href="#MOESM19" rid="MOESM19" class=" media">15</a></sup>). The nuclear receptor <em>NR1D1</em> has been shown to correlate with <em>MYCN</em> amplification in NB patients<sup><sup><a href="#CR82" rid="CR82" class=" bibr popnode">82</a></sup>,<sup><a href="#CR83" rid="CR83" class=" bibr popnode">83</a></sup></sup> and <em>TFAP4</em> inhibition leads to differentiation of <em>MYCN</em>-amplified neuroblastoma cells<sup><sup><a href="#CR84" rid="CR84" class=" bibr popnode">84</a></sup>,<sup><a href="#CR85" rid="CR85" class=" bibr popnode">85</a></sup></sup>, supporting the validity of the inferred target genes.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig10" co-legend-rid="lgnd_Fig10"><a href="/pmc/articles/PMC11068915/figure/Fig10/" target="figure" rid-figpopup="Fig10" rid-ob="ob-Fig10"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139783098949200"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11068915_41467_2024_47945_Fig10_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2024_47945_Fig10_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11068915/bin/41467_2024_47945_Fig10_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139783098949200"><a target="object" rel="noopener" href="/pmc/articles/PMC11068915/figure/Fig10/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig10"><div><a class="figpopup" href="/pmc/articles/PMC11068915/figure/Fig10/" target="figure" rid-figpopup="Fig10" rid-ob="ob-Fig10">Fig. 10</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Copy number changes facilitate <em>MYCN</em>-mediated blockage of differentiation via developmental transcription factor networks.</strong><p><strong>a</strong> To define putative target genes of TFs, we linked TF motif matches in ATAC-seq peaks with proximal genes and used <em>GRNboost2</em><sup><sup><a href="#CR81" rid="CR81" class=" bibr popnode">81</a></sup></sup> to identify highly correlated TF-to-target gene candidates based on our scRNA-seq data. <strong>b</strong> Top-2500 predicted targets of MYCN. Putative targets without support in our ATAC-seq data (motif in &#x02265;1 peak near the gene) have been removed. The Pearson correlation coefficient (<em>r</em>) between each TF and target gene determined the direction of the putative interaction (<em>r</em>&#x02009;&#x0003e;&#x02009;0.1&#x02009;=&#x02009;activating, <em>r</em>&#x02009;&#x0003c;&#x02009;&#x02212;0.1&#x02009;=&#x02009;inhibitory, others&#x02009;=&#x02009;marginal). The top-5 TFs have been highlighted. All results are reported in Supplementary Data&#x000a0;<sup><a href="#MOESM19" rid="MOESM19" class=" media">15</a></sup>. <strong>c</strong> Average expression (Seurat module score) of the MYCN target genes (activated targets from <strong>b</strong>) in our integrated scRNA-seq dataset (cp. Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4a</span></a>). <strong>d</strong> Heatmap displaying the percentage of gene sets D9_1 to D9_4 (correlated with mutation score, cp. Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4b, c</span></a>) overlapping targets of the indicated TFs. TFs with significant overlaps in at least one comparison are shown (hypergeometric test, <em>hypeR</em><sup><sup><a href="#CR135" rid="CR135" class=" bibr popnode">135</a></sup></sup>; <em>P</em> values adjusted for multiple hypothesis testing using the Benjamini-Hochberg method; <em>P</em><sub>adj</sub>&#x02009;&#x02264;&#x02009;0.05, |log<sub>2</sub>FoldChange|&#x02009;&#x02265;&#x02009;log<sub>2</sub>(4), frequency &#x02265;5%). All results are also reported in Supplementary Data&#x000a0;<sup><a href="#MOESM20" rid="MOESM20" class=" media">16</a></sup>. <strong>e</strong> Smoothed line plots evaluating target gene expression (Seurat module score) with increasing mutation scores (from Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4b, c</span></a>). We curated selected TFs from (<strong>d</strong>) into groups of TFs losing or gaining activity. The cell line of each data point is indicated at the bottom. <strong>f</strong> Network diagrams visualising putative TF-to-target relations for enriched TF targets (cp. <strong>c</strong>&#x02013;<strong>e</strong>). Each node represents a TF or target gene, and each edge is a TF-to-target link. We made these networks specific to each condition by using colour to indicate the mean scaled expression at D9 (edges coloured by source TF) and node size to indicate the mean scaled <em>T</em><sub>TF</sub> target score of each TF. Only labels of TFs with positive scaled expression are shown and selected groups of TFs have been merged for visualisation. A diagram with all node labels is shown in Supplementary Fig. <sup><a href="#MOESM1" rid="MOESM1" class=" media">13c</a></sup>. Source data are provided as a <sup><a href="#MOESM1" rid="MOESM1" class=" media">Source Data</a></sup> file. D9, day 9; TF, transcription factor; WT, wild-type H7 hESCs; r, Pearson correlation coefficient.</p></div></div></div><p id="Par27">We intersected the inferred lists of TF targets with the mutation-linked gene sets (D9_1 &#x02013; D9_4) and found an enrichment (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig10/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig10" rid-ob="ob-Fig10" co-legend-rid="lgnd_Fig10"><span>10d</span></a>; Supplementary Data&#x000a0;<sup><a href="#MOESM20" rid="MOESM20" class=" media">16</a></sup>) of MYCN, NR1D1, TFAP4, and ZIC2 targets in D9_1 (highly expressed in 17q1qMYCN). Conversely, the gene set D9_2 (expressed in WT/17q/17q1q) was enriched for targets of TFs expected at this stage of differentiation, e.g., SOX4/5/10, TFAP2A/B, and nuclear receptors NR2F1/2. The expression of targets of these TFs increased or decreased along the mutational spectrum, corroborating their association with the mutations (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig10/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig10" rid-ob="ob-Fig10" co-legend-rid="lgnd_Fig10"><span>10e</span></a>). While many TF targets switched expression rapidly with <em>MYCN</em> overexpression, others showed a continuous pattern with up-/down-regulation already detectable in 17q and 17q1q, e.g., targets of vagal and early NC regulators HOXB3 and CDX2<sup><sup><a href="#CR86" rid="CR86" class=" bibr popnode">86</a></sup></sup> (up), or of sensory neurogenesis regulator NEUROD1<sup><sup><a href="#CR39" rid="CR39" class=" bibr popnode">39</a></sup></sup> (down). To aid interpretation, we visualised cell-line-specific interactions between TFs and targets as edges in connected network diagrams (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig10/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig10" rid-ob="ob-Fig10" co-legend-rid="lgnd_Fig10"><span>10f</span></a>; Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">13c</a></sup>). These diagrams showcased the emergence of a new subnetwork of TFs in 17q1qMYCN that centred on MYCN and incorporated TFs like NR1D1 and TFAP4. In contrast, a subnetwork involving NC-related TFs such as SOX10 and TFAP2A/B was lost in these cells. Intriguingly, downregulation of TFs linked to sensory neuronal development (NEUROD1, ONECUT1) was visible already in 17q cells (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig10/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig10" rid-ob="ob-Fig10" co-legend-rid="lgnd_Fig10"><span>10f</span></a>), perhaps explaining why sensory neuron-like derivatives were less abundant in 17q cultures (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2b</span></a>). In 17q1q, we additionally observed upregulation of TFs related to early posterior NC specification including HOXB3, LEF1, and CDX2, which was partially reversed (HOXB3) upon <em>MYCN</em> overexpression (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig10/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig10" rid-ob="ob-Fig10" co-legend-rid="lgnd_Fig10"><span>10f</span></a>). While many of the TFs implicated in these developmental gene-regulatory networks are weakly or not at all expressed in NB tumours (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">14a</a></sup>), we found that the targets of MYCN-related TFs (based on our analysis) are highly expressed in <em>MYCN</em>-amplified tumours (Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">14b</a></sup>). Our analysis also revealed that the targets of 17q/1q-related TFs are strongly expressed in groups of tumours, but we could not determine whether these contained the corresponding CNAs due incomplete annotations.</p><p id="Par28" class="p p-last">In summary, our data suggest a subtle rewiring of gene-regulatory networks in CNA-carrying hESCs, which may be linked to the depletion of mature sensory NC derivatives and increased early SCP signature found in our single-cell analyses (cp. Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2</span></a>). Overexpression of <em>MYCN</em> resulted in a switch in favour of known NC-linked TFs downstream of MYCN.</p></div></div><div id="Sec9" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec9title">Discussion</h2><p id="Par29" class="p p-first">Although CNAs are a principal genetic hallmark of paediatric cancers, it has remained difficult to determine their exact role in tumour initiation due to the lack of suitable human models. In this study, we used hESCs carrying CNAs that are prevalent in NB (chr17q and chr1q gains). The NC is a transient embryonic tissue that is inaccessible after birth; therefore, hESC differentiation allowed us to experimentally study the effects of these mutations on human sympathoadrenal progenitors, the putative cells-of-origin of NB.</p><p id="Par30">We provide a comprehensive knowledge base of transcriptomic and epigenetic changes in this model on a temporal (i.e., during differentiation) and a genetic (i.e., with different mutations) axis. Our data show that chr17q/1q gains impair trunk NC differentiation and potentiate an SCP-like gene signature. In this aberrant cell state, overexpression of <em>MYCN</em> (mimicking <em>MYCN</em> amplification commonly found along with chr17q/chr1q gains in NB tumours) leads to a complete derailment of sympathoadrenal lineage specification, and a proliferative, tumour-like cellular phenotype that correlates with the emergence of NB-like tumours in vivo. Moreover, chr17q/1q gains were found to enhance the <em>MYCN</em>-driven differentiation block and acquisition of tumourigenic hallmarks such as proliferation, clonogenicity, and resistance to apoptosis. In line with recent studies<sup><sup><a href="#CR29" rid="CR29" class=" bibr popnode">29</a></sup>,<sup><a href="#CR87" rid="CR87" class=" bibr popnode">87</a></sup></sup>, we speculate that CNAs provide an early selective advantage manifested by the expansion of undifferentiated cells, which act subsequently as a NB-initiating entity upon a second oncogenic hit such as <em>MYCN</em> overexpression.</p><p id="Par31">The accumulation of NB-associated lesions correlated with a failure to reprogram chromatin during trunk NC differentiation. Following gain of chr17q/1q, cells lost the activity of TFs associated with sensory differentiation (e.g., NEUROD1) and instead upregulated vagal NC TFs like HOXB3 and WNT-related effector LEF1<sup><sup><a href="#CR86" rid="CR86" class=" bibr popnode">86</a></sup>,<sup><a href="#CR88" rid="CR88" class=" bibr popnode">88</a></sup></sup>. <em>MYCN</em> overexpression on top of these CNAs abolished chromatin states indicative of sympathoadrenal differentiation, and instead led to the induction of targets of NR1D1, TFAP4, and other TFs of the reported NB regulatory circruitry<sup><sup><a href="#CR44" rid="CR44" class=" bibr popnode">44</a></sup>,<sup><a href="#CR45" rid="CR45" class=" bibr popnode">45</a></sup>,<sup><a href="#CR72" rid="CR72" class=" bibr popnode">72</a></sup>&#x02013;<sup><a href="#CR76" rid="CR76" class=" bibr popnode">76</a></sup></sup>. TFAP4 is a well-established downstream effector of MYCN<sup><sup><a href="#CR84" rid="CR84" class=" bibr popnode">84</a></sup>,<sup><a href="#CR85" rid="CR85" class=" bibr popnode">85</a></sup></sup>, and NR1D1 (Rev-erb&#x003b1;) is a circadian rhythm and metabolic regulator, and a downstream effector of MYCN hyperactivity in NB<sup><sup><a href="#CR82" rid="CR82" class=" bibr popnode">82</a></sup>,<sup><a href="#CR83" rid="CR83" class=" bibr popnode">83</a></sup></sup>. Together with the appearance of sensory-related signatures in NB tumours (C4* and C5*, Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig7" rid-ob="ob-Fig7" co-legend-rid="lgnd_Fig7"><span>7</span></a>) our &#x0201c;early <em>MYCN</em> onset&#x0201d; scenario reveals previously uncharted disruptions of the early sensory NC lineage, which might complement the currently prevailing model of dichotomic mesenchymal/adrenergic heterogeneity in NB<sup><sup><a href="#CR11" rid="CR11" class=" bibr popnode">11</a></sup>,<sup><a href="#CR44" rid="CR44" class=" bibr popnode">44</a></sup>&#x02013;<sup><a href="#CR46" rid="CR46" class=" bibr popnode">46</a></sup>,<sup><a href="#CR76" rid="CR76" class=" bibr popnode">76</a></sup>,<sup><a href="#CR89" rid="CR89" class=" bibr popnode">89</a></sup>&#x02013;<sup><a href="#CR92" rid="CR92" class=" bibr popnode">92</a></sup></sup>. Thus, our model will facilitate the functional dissection of these TFs via loss-/gain-of-function approaches to decipher their crosstalk with <em>MYCN</em>/CNA-driven tumourigenesis.</p><p id="Par32">Complementing earlier studies using cell lines and animal models<sup><sup><a href="#CR12" rid="CR12" class=" bibr popnode">12</a></sup>&#x02013;<sup><a href="#CR14" rid="CR14" class=" bibr popnode">14</a></sup>,<sup><a href="#CR18" rid="CR18" class=" bibr popnode">18</a></sup>,<sup><a href="#CR19" rid="CR19" class=" bibr popnode">19</a></sup>,<sup><a href="#CR22" rid="CR22" class=" bibr popnode">22</a></sup></sup>, recent single-cell transcriptomic analyses of NB tumours and metastases<sup><sup><a href="#CR15" rid="CR15" class=" bibr popnode">15</a></sup>&#x02013;<sup><a href="#CR17" rid="CR17" class=" bibr popnode">17</a></sup>,<sup><a href="#CR23" rid="CR23" class=" bibr popnode">23</a></sup></sup> corroborated an origin of NB from neuroblastic, SCP-like progenitors, and highlighted intra-tumour heterogeneity comprising subtypes of tumour cells with adrenergic and mesenchymal properties. In our in-vitro experiments, we also observed cells expressing signatures of both cell types, suggesting that our model could be useful to experimentally investigate the transition between these and other NB-relevant cell types, providing a new scope into their role in therapy resistance<sup><sup><a href="#CR89" rid="CR89" class=" bibr popnode">89</a></sup></sup>. Furthermore, <em>MYCN</em> overexpression (in conjunction with chr17q/1q gains) in nascent trunk NC cells was sufficient to drive tumourigenic traits, suggesting that in some cases NB initiation might occur before SCP/neuroblast emergence and that acquisition of an SCP-like identity may also be a consequence of mutations in earlier stages rather than the origin. We also observed that <em>MYCN</em> induction resulted in an apparent block of differentiation when activated at other stages. In future, our cell models will provide the means to dissect the specific effects of <em>MYCN</em> at different time points and in specific cell types.</p><p id="Par33">In this study we exploited the phenomenon of culture adaptation of hESCs<sup><sup><a href="#CR49" rid="CR49" class=" bibr popnode">49</a></sup></sup>, to obtain sets of cell lines with CNAs that are commonly observed in NB in an otherwise largely isogenic background. Our detailed genetic analyses of the used cell lines revealed other mutations that had naturally arisen in these cell lines (Supplementary Data&#x000a0;<sup><a href="#MOESM8" rid="MOESM8" class=" media">4</a></sup>), including a point mutation in the BCL6-interacting corepressor <em>BCOR</em> (<em>BCOR</em><sup><em>L1673F</em></sup>). <em>BCOR</em> mutations have been previously observed in human induced pluripotent stem cell cultures<sup><sup><a href="#CR93" rid="CR93" class=" bibr popnode">93</a></sup>,<sup><a href="#CR94" rid="CR94" class=" bibr popnode">94</a></sup></sup> and NB patients with <em>BCOR</em> mutations exhibit a high frequency of CNAs<sup><sup><a href="#CR87" rid="CR87" class=" bibr popnode">87</a></sup></sup>. <em>BCOR</em> mutations have also been reported together with CNAs in other cancers, e.g., retinoblastoma<sup><sup><a href="#CR95" rid="CR95" class=" bibr popnode">95</a></sup></sup>. It would be tempting to speculate that <em>BCOR</em> dysfunction might facilitate the ability of cells to tolerate the emergence of certain CNAs; however, to date a causal relationship remains to be established.</p><p id="Par34">Our hESC-based model provides a tractable system for analysing tumour initiation events within disease-relevant human embryonic cell-like populations. In this study, we focused on cell-intrinsic transcriptional regulation since our cultures lack tumour-relevant, non-NC cell types (e.g., immune cells or Schwann cells) and do not recapitulate the structural and physical properties of the human tumour micro- and macroenvironment<sup><sup><a href="#CR96" rid="CR96" class=" bibr popnode">96</a></sup>&#x02013;<sup><a href="#CR99" rid="CR99" class=" bibr popnode">99</a></sup></sup>. In the future, it will be possible to combine our system with 3D co-culture approaches with defined cell types or to use biomimetic scaffolds to emulate cell-cell interactions and extrinsic environmental influences.</p><p id="Par35" class="p p-last">In conclusion, this study unravels the developmental effects of NB-associated mutations and proposes the progressive corruption of gene-regulatory networks by CNAs as an early step toward tumour initiation by selection of undifferentiated progenitor phenotypes. Transformation is then triggered by a second hit with <em>MYCN</em> overexpression, which tilts cells toward increased proliferation and formation of aberrant cell types. Our data provide a direct link between CNAs that commonly emerge in hESC cultures with impaired differentiation and the acquisition of tumourigenic hallmarks, thus highlighting the importance of rigorous monitoring of such cultures prior to their use in disease modelling or cell therapy application in line with recent recommendations from the International Society for Stem Cell Research<sup><sup><a href="#CR49" rid="CR49" class=" bibr popnode">49</a></sup>,<sup><a href="#CR100" rid="CR100" class=" bibr popnode">100</a></sup></sup>.</p></div><div id="Sec10" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec10title">Methods</h2><div id="Sec11" class="sec sec-first"><h3 id="Sec11title">Human embryonic stem cell (hESC) cell culture and differentiation</h3><div id="Sec12" class="sec sec-first"><p></p><h4 id="Sec12title" class="inline">Cell lines and cell culture </h4><p id="Par36" class="p p-first-last">We employed H7 (<a href="https://hpscreg.eu/cell-line/WAe007-A" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://hpscreg.eu/cell-line/WAe007-A</a>) and H9 (<a href="https://hpscreg.eu/cell-line/WAe009-A" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://hpscreg.eu/cell-line/WAe009-A</a>) hESCs as karyotypically normal, female WT controls<sup><sup><a href="#CR38" rid="CR38" class=" bibr popnode">38</a></sup></sup>. Use of human embryonic cells has been approved by the Human Embryonic Stem Cell UK Steering Committee (SCSC23-29). Their isogenic chr17q counterparts carry a gain in chromosome 17q (region q27q11) via an unbalanced translocation with chromosome 6 (H7) or a gain of 17q via an unbalanced translocation with chromosome 21 with breakpoints at 17q21 and 21p11.2 (H9)<sup><sup><a href="#CR50" rid="CR50" class=" bibr popnode">50</a></sup>,<sup><a href="#CR101" rid="CR101" class=" bibr popnode">101</a></sup></sup>. The chr17q1q hESC lines were clonally derived, after their spontaneous emergence following the genetic modification of chr17q hESCs. The H7 chr17q1q-MYCN hESC line was generated by introducing a TetOn-PiggyBac plasmid (PB-TRE3G-MYCN, plasmid#104542, Addgene) carrying the wild-type version of the <em>MYCN</em> gene<sup><sup><a href="#CR102" rid="CR102" class=" bibr popnode">102</a></sup></sup> while the H9 chr17q1q-MYCN and H7 WT-MYCN and 17q-MYCN hESC lines were produced using a Tet-On &#x0201c;all-in-one&#x0201d; inducible expression cassette containing the TRE3G promoter driving the expression of <em>MYCN</em> with a 2A peptide-linked fluorescent reporter (mScarlet) and a pCAG promoter-driven rtTA3G transactivator<sup><sup><a href="#CR103" rid="CR103" class=" bibr popnode">103</a></sup>,<sup><a href="#CR104" rid="CR104" class=" bibr popnode">104</a></sup></sup>. Plasmids were introduced via nucleofection using either the Neon NxT Electroporation System (Thermo Fisher Scientific) or the Lonza 4D-Nucleofector System. In the case of the latter, the Amaxa 4D-Nucleofector Basic Protocol for Human Stem Cells was employed with the following modification: 2&#x02009;&#x000d7;&#x02009;10<sup>6</sup> cells were transfected with 2&#x02009;&#x000b5;g plasmid in 100&#x02009;&#x000b5;l Nucleocuvettes. All cell lines were tested regularly for mycoplasma and expression of pluripotency markers. Karyotypic analysis was carried out using G-banding (number of cells examined = 20&#x02013;30). A rapid quantitative PCR (qPCR) assay was also regularly employed to detect the emergence of common CNAs such as chr17q and 1q gains in our hESC lines<sup><sup><a href="#CR105" rid="CR105" class=" bibr popnode">105</a></sup></sup>. hESCs were cultured routinely in feeder-free conditions at 37&#x02009;&#x000b0;C and 5% CO<sub>2</sub> in E8 media<sup><sup><a href="#CR106" rid="CR106" class=" bibr popnode">106</a></sup></sup> complemented with GlutaMax (Cat# 35050061, Thermo Fisher Scientific) on Vitronectin (Cat# A14700, Thermo Fisher Scientific) or on Geltrex LDEV-Free Reduced Growth Factor Basement Membrane Matrix (Cat# A1413202, Thermo Fisher Scientific) as an attachment substrate. All hESC lines described in this manuscript are available upon request and completion of a Material Transfer Agreement.</p></div><div id="Sec13" class="sec"><p></p><h4 id="Sec13title" class="inline">Differentiation toward trunk neural crest </h4><p id="Par37" class="p p-first-last">hESC differentiation toward trunk NC and its derivatives was performed using our established protocols<sup><sup><a href="#CR35" rid="CR35" class=" bibr popnode">35</a></sup>,<sup><a href="#CR36" rid="CR36" class=" bibr popnode">36</a></sup></sup>. Briefly, hESCs were harvested using StemPro Accutase Cell Dissociation Reagent (Cat# A1110501, Thermo Fisher Scientific) and plated at 60,000 cells/cm<sup>2</sup> in N2B27 medium supplemented with 20&#x02009;ng/ml of FGF2 (Cat# 233-FB/CF, R&#x00026;D) and 4&#x02009;&#x003bc;M of CHIR 99021 (Cat# 4423, Tocris) and 10&#x02009;&#x003bc;M of Rock Inhibitor (Y-27632) (Cat# A11001, Generon) in a volume of 300&#x02009;&#x000b5;l/cm<sup>2</sup>. The N2B27 medium consisted of 50:50 DMEM F12 Merck Life Science/Neurobasal medium (Gibco) and 1x N2 supplement (Cat# 17502048, Invitrogen), 1x B27 (Cat#17504044, Invitrogen), 1x GlutaMAX (Cat# 35050061, Thermo Fisher Scientific), 1x MEM Non-essential amino acids (NEAA; Cat#11140050, Thermo Fisher Scientific), 50&#x02009;&#x003bc;M 2-Mercaptoethanol (Cat# 31350010, Thermo Fisher Scientific). After 24&#x02009;h, media was refreshed removing the Rock Inhibitor and cells were cultured for a further 2 days in FGF2/CHIR to generate NMPs (300&#x02009;&#x000b5;l/cm<sup>2</sup>). NMPs at D3 were then re-plated at 50,000 cells/cm<sup>2</sup> (H7) or 40,000 cells/cm<sup>2</sup> (H9) in neural crest inducing medium consisting of DMEM/F12, 1x N2 supplement, 1x GlutaMAX, 1x MEM NEAA, the TGF-beta/Activin/Nodal inhibitor SB-431542 (2&#x02009;&#x003bc;M, Cat# 1614, Tocris), CHIR99021 (1&#x02009;&#x003bc;M, Cat# 4423, Tocris), BMP4 (15&#x02009;ng/ml, Cat# PHC9534, Thermo Fisher Scientific), the BMP type-I receptor inhibitor DMH-1 (1&#x02009;&#x003bc;M, Cat# 4126, Tocris), 10&#x02009;&#x003bc;M of Rock Inhibitor (Y-27632) on Geltrex LDEV-Free Reduced Growth Factor Basement Membrane Matrix (Cat# A1413202, Thermo Fisher Scientific) in a volume of 300&#x02009;&#x000b5;l/cm<sup>2</sup>. 48&#x02009;h later (D5), media was replaced removing the Rock Inhibitor. Media was refreshed at D7 and D8 increasing volume to 500&#x02009;&#x000b5;l/cm<sup>2</sup>. On D5, the expression of MYCN was induced by supplementing the neural crest media with 100&#x02009;ng/ml (H7-17q1q-MYCN), 200&#x02009;ng/ml (H7 WT-MYCN, 17q-MYCN), or 1000&#x02009;ng/ml (H9-derived lines) of Doxycycline (Cat# D3447, Merck). On D9, cells were re-plated at 150,000&#x02013;250,000 cells/cm<sup>2</sup> in plates coated with Geltrex (Thermo Fisher Scientific) in the presence of medium containing BrainPhys (Cat# 05790, Stem Cell Technologies), 1x B27 supplement (Cat# 17504044, Invitrogen), 1x N2 supplement (Cat# 17502048, Invitrogen), 1x MEM NEAA (Cat# 11140050, Thermo Fisher Scientific) and 1x Glutamax (Cat# 35050061, Thermo Fisher Scientific), BMP4 (50&#x02009;ng/ml, Cat# PHC9534, Thermo Fisher Scientific), recombinant SHH (C24II) (50&#x02009;ng/ml, Cat# 1845-SH-025, R and D) and purmorphamine (1.5&#x02009;&#x000b5;M, Cat# SML0868, Sigma) and cultured for 5 days (=D14 of differentiation) in a volume of 250&#x02009;&#x000b5;l/cm<sup>2</sup>. Media was refreshed daily. For further sympathetic neuron differentiation, D14 cells were switched into a medium containing BrainPhys neuronal medium (Stem Cell Technologies), 1x B27 supplement (Invitrogen), 1x N2 supplement (Invitrogen), 1x NEAA (Thermo Fisher Scientific) and 1x Glutamax (Thermo Fisher Scientific), NGF (10&#x02009;ng/ml, Cat#450-01 Peprotech), BDNF (10&#x02009;ng/ml, Cat# 450-02, Peprotech) and GDNF (10&#x02009;ng/ml, Cat# 450-10, Peprotech) for a further 5&#x02013;14 days (volume of 300&#x02009;&#x000b5;l/cm<sup>2</sup> changing media every other day). Volume was increased up to 500&#x02009;&#x000b5;l/cm<sup>2</sup>, depending on cell density, after day 17 of differentiation.</p></div><div id="Sec14" class="sec"><p></p><h4 id="Sec14title" class="inline">Immunostaining </h4><p id="Par38" class="p p-first-last">Cells were fixed using 4% PFA (P6148, Sigma-Aldrich) at room temperature for 10&#x02009;min, then washed twice with PBS (without Ca<sup>2+</sup>, Mg<sup>2+</sup>) to remove any traces of PFA and permeabilised using a PBS supplemented with 10% FCS, 0.1% BSA and 0.5% Triton X-100 for 10&#x02009;min. Cells were then incubated in blocking buffer (PBS supplemented with 10% FCS and 0.1% BSA) for 1&#x02009;h at RT or overnight at 4&#x02009;&#x000b0;C. Primary and secondary antibodies were diluted in the blocking buffer; the former were left overnight at 4&#x02009;&#x000b0;C and the latter for 2&#x02009;h at 4&#x02009;&#x000b0;C on an orbital shaker. Samples were washed twice with blocking buffer between the primary and secondary antibodies. Hoechst 33342 (H3570, Invitrogen) was added at a ratio of 1:1000 to the secondary antibodies&#x02019; mixture to label nuclei in the cells. We used the following primary antibodies SOX10 (D5V9L) (Cell Signalling, 89356S, 1:500); HOXC9 (Abcam, Ab50839,1:50); MYCN (Santa Cruz, SC-53993, 1:100); PHOX2B (Santa Cruz, SC-376997, 1:100); MASH1 (ASCL1) (Abcam, <a id="__tag_1147980651" class="tag_hotlink" href="/nuccore/Ab211327" ref="/nuccore/Ab211327"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"Ab211327","term_id":"71636198","term_text":"AB211327"}}</span>Ab211327</a>, 1:100); Ki67 (Abcam, <a id="__tag_1147980650" class="tag_hotlink" href="/nuccore/Ab238020" ref="/nuccore/Ab238020"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"Ab238020","term_id":"77799410","term_text":"AB238020"}}</span>Ab238020</a>, 1:100); PERIPHERIN (Sigma-Aldrich, AB1530, 1:400); Cleaved Caspase 3 (Asp175) (Cell Signalling, 9661S, 1:400), yH2AX (Cell Signalling, S139/9718S, 1:400). Secondary antibodies: Goat anti-Mouse Affinipure IgG+IgM (H&#x02009;+&#x02009;L) AlexaFluor 647 (Stratech (Jackson ImmunoResearch) 115-605-044-JIR, Polyclonal 1:500); Donkey anti-Rabbit IgG (H&#x02009;+&#x02009;L) Alexa Fluor 488 (Invitrogen, A-21206, 1:1000).</p></div><div id="Sec15" class="sec"><p></p><h4 id="Sec15title" class="inline">Intracellular flow cytometry staining </h4><p id="Par39" class="p p-first-last">Cells were detached and resuspended as single cells using StemPro Accutase Cell Dissociation Reagent (Cat# A1110501, Thermo Fisher Scientific) and then counted. Next, 10 million cells/ml were resuspended in 4% PFA at room temperature for 10&#x02009;min. Then cells were washed once with PBS (without Ca<sup>2+</sup>, Mg<sup>2+</sup>) and pelleted at 200&#x02009;g. Cells were resuspended in PBS at 10 million/ml and used for antibody staining. Permeabilisation buffer (0.5% Triton X-100 in PBS with 10% FCS and 0.1% BSA) was added to each sample, followed by incubation at room temperature for 10&#x02009;min. Samples were then washed once with staining buffer (PBS with 10% FCS and 0.1% BSA) and pelleted at 200&#x02009;g. Then samples were resuspended in staining buffer containing pre-diluted primary antibodies: SOX10 (D5V9L) (1:500; 89356S, Cell Signalling); HOXC9 (1:50; Ab50839, Abcam); cleaved Caspase 3 (Asp175) (Cell Signalling, 9661S, 1:400). The samples were left at 4&#x02009;&#x000b0;C on an orbital shaker overnight. Then, the primary antibodies were removed, and samples were washed two times with staining buffer. After washing, staining buffer with pre-diluted secondary antibody was added to the samples and incubated at 4&#x02009;&#x000b0;C for 2&#x02009;h. The secondary antibodies used were Goat anti-Mouse Affinipure IgG+IgM (H&#x02009;+&#x02009;L) AlexaFluor 647 (Stratech (Jackson ImmunoResearch) 115-605-044-JIR, Polyclonal 1:500); Donkey anti-Rabbit IgG (H&#x02009;+&#x02009;L) Alexa Fluor 488 (Invitrogen, A-21206, 1:1000). Finally, samples were washed once with staining buffer, resuspended in staining buffer and analysed using a BD FACSJazz or a CytoFLEX (Beckman Coulter) flow cytometer. A secondary antibody-only sample was used as a control to set the gating.</p></div><div id="Sec16" class="sec"><p></p><h4 id="Sec16title" class="inline">Cell cycle analysis </h4><p id="Par40" class="p p-first-last">The 5-ethynyl-2&#x000b4;-deoxyuridine (EdU) assay was performed following the manufacturer&#x02019;s instructions (Thermo Fisher Scientific, <a id="__tag_1147980652" class="tag_hotlink" href="/nuccore/C10633" ref="/nuccore/C10633"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"C10633","term_id":"56146393","term_text":"C10633"}}</span>C10633</a> Alexa Fluor 488). We used 10&#x02009;&#x003bc;M of Edu for a 2-h incubation. Cells were analysed in the flow cytometer (BD FACSJazz) using the 405&#x02009;nm laser to detect the Hoechst/DAPI staining and 488&#x02009;nm to detect the EdU staining.</p></div><div id="Sec17" class="sec"><p></p><h4 id="Sec17title" class="inline">Low-density plating </h4><p id="Par41" class="p p-first-last">Day 9 trunk NC cells derived from hESCs as described above were harvested and plated at a density of 500 cells/cm<sup>2</sup> in plates pre-coated with Geltrex LDEV-Free Reduced Growth Factor Basement Membrane Matrix (Cat# A1413202, Thermo Fisher Scientific) in the presence of DMEM/F12 (Sigma-Aldrich), 1x N2 supplement, 1x GlutaMAX, 1x MEM NEAA, the TGF-beta/Activin/Nodal inhibitor SB-431542 (2&#x02009;&#x003bc;M, Tocris), CHIR99021 (1&#x02009;&#x003bc;M, Tocris), BMP4 (15&#x02009;ng/ml, Thermo Fisher Scientific), the BMP type-I receptor inhibitor DMH-1 (1&#x02009;&#x003bc;M, Tocris) and ROCK inhibitor Y-27632 2HCl (10&#x02009;&#x003bc;M) (300&#x02009;&#x000b5;l/cm<sup>2</sup>). The culture medium was replaced the following day with medium containing BrainPhys (Stem Cell Technologies), 1x B27 supplement (Invitrogen), 1x N2 supplement (Invitrogen), 1x NEAA (Thermo Fisher Scientific) and 1x Glutamax (Thermo Fisher Scientific), BMP4 (50&#x02009;ng/ml, Thermo Fisher Scientific), recombinant SHH (C24II) (50&#x02009;ng/ml, R and D) and Purmorphamine (1.5&#x02009;&#x000b5;M, Sigma) (250&#x02009;&#x000b5;l/cm<sup>2</sup>). Plates were then incubated at 37&#x02009;&#x000b0;C at 5% CO<sub>2</sub>. The media was refreshed every 48&#x02009;h. After 5 days of culture, cells were fixed (PFA 4%/10&#x02009;min) and stained with Hoechst 33342 (Cat# H3570, Invitrogen) for 5&#x02009;min. Colonies were detected using an InCell Analyser 2200 (GE Healthcare) at a 4X magnification. Images were processed using Cell Profiler.</p></div><div id="Sec18" class="sec"><p></p><h4 id="Sec18title" class="inline">DNA damage analysis </h4><p id="Par42" class="p p-first-last">DNA damage was measured by assessing the phosphorylation state of the histone H2AX on the SerCells were fixed and immunostained using the anti-yH2AX as described above at different time points. Stained cells were imaged using the InCell Analyser 2200 (GE Healthcare) at 40X magnification. Image analysis was performed using a pipeline in CellProfiler that allowed us to detect the number of foci of yH2AX antibody per nuclei.</p></div><div id="Sec19" class="sec sec-last"><p></p><h4 id="Sec19title" class="inline">Quantitative real-time PCR </h4><p id="Par43" class="p p-first-last">RNA extractions were performed using the total RNA purification kit (Norgen Biotek, 17200) according to the manufacturer&#x02019;s instructions. cDNA synthesis was performed using the High-Capacity cDNA Reverse Transcription kit (ThermoFisher, 4368814). Quantitative real-time PCR was performed using PowerUp SYBR master mix (ThermoFisher, A25780) and run on a QuantStudio 12&#x02009;K Flex (Applied Biosystems). Primers used for <em>MYCN</em>: ccacaaggccctcagtacc (forward), tcttcctcttcatcatcttcatca (reverse).</p></div></div><div id="Sec20" class="sec"><h3 id="Sec20title">Mouse experiments</h3><div id="Sec21" class="sec sec-first"><p></p><h4 id="Sec21title" class="inline">Cell preparation for xenotransplantation </h4><p id="Par44" class="p p-first-last">H7 wild type, 17q1q, and 17q1qMYCN hESCs were differentiated up to day 9 following the protocol described above. Cells were harvested using Accutase to create a single cell suspension, counted, and resuspended with media containing Matrigel before injection.</p></div><div id="Sec22" class="sec"><p></p><h4 id="Sec22title" class="inline">Mice and in-vivo experiments </h4><p id="Par45" class="p p-first-last">All animal experiments were approved by The Institute of Cancer Research Animal Welfare and Ethical Review Body and performed in accordance with the UK Home Office Animals (Scientific Procedures) Act 1986, the UK National Cancer Research Institute guidelines for the welfare of animals in cancer research and the ARRIVE (animal research: reporting in-vivo experiments) guidelines. Female NSG mice were obtained from Charles River and enrolled into trial at 6&#x02013;8 weeks of age. Mice were maintained on a regular diet in a pathogen-free facility on a 12&#x02009;h light/dark cycle with unlimited access to food and water.</p></div><div id="Sec23" class="sec"><p></p><h4 id="Sec23title" class="inline">Subcutaneous xenograft </h4><p id="Par46" class="p p-first-last">One million cells with 50% Matrigel were injected subcutaneously into the right flank of NSG mice (female; 6&#x02013;8 weeks old) and allowed to establish a murine xenograft model. Studies were terminated when the mean diameter of the tumour reached 15&#x02009;mm. Tumour volumes were measured by Vernier caliper across two perpendicular diameters, and volumes were calculated according to the formula V&#x02009;=&#x02009;4/3&#x003c0; [(d1&#x02009;+&#x02009;d2)/4]3; where d1 and d2 were the two perpendicular diameters. The weight of the mice was measured every 2 days. Mice were fed with either regular diet or DOX diet (chow containing 20&#x02009;g of DOX per kg of diet) to induce the expression of <em>MYCN</em>.</p></div><div id="Sec24" class="sec"><p></p><h4 id="Sec24title" class="inline">Orthotopic (adrenal)xenograft </h4><p id="Par47" class="p p-first-last">100,000 cells with 50% Matrigel were injected into the right adrenal gland of NSG mice (female; 6&#x02013;8 weeks old) and allowed to establish a murine xenograft model. Detection of xenografted tumours was performed by magnetic resonance imaging (MRI). The weight of the mice was measured every 2 days. Mice were fed with either standard diet or DOX diet (chow containing 20&#x02009;g of DOX per kg of diet) to induce the expression of <em>MYCN</em>. Magnetic resonance images were acquired on a 1 Tesla M3 small animal MRI scanner (Aspect Imaging). Mice were anesthetised using isoflurane delivered via oxygen gas and their core temperature was maintained at 37&#x02009;&#x000b0;C. Anatomical T2-weighted coronal images were acquired through the mouse abdomen, from which tumour volumes were determined using segmentation from regions of interest (ROI) drawn on each tumour-containing slice using the Horos medical image viewer.</p></div><div id="Sec25" class="sec sec-last"><p></p><h4 id="Sec25title" class="inline">Pathology </h4><p id="Par48" class="p p-first-last">Tissue sections were stained with haematoxylin and eosin (H&#x00026;E) or specific antibodies (MYCN, Merck; Ki67, BD Pharmingen). Immunohistochemistry was performed using standard methods. Briefly, 5&#x02009;&#x003bc;m sections were stained with antibodies, including heat-induced epitope retrieval of specimens using citrate buffer (pH 6) or EDTA buffer.</p></div></div><div id="Sec26" class="sec"><h3 id="Sec26title">Zebrafish experiments</h3><div id="Sec27" class="sec sec-first"><p></p><h4 id="Sec27title" class="inline">Cell preparation for xenotransplantation </h4><p id="Par49" class="p p-first-last">Pre-differentiated neural crest cells were frozen on D7 during their in-vitro differentiation as described above, shipped, and subsequently thawed in DMEM at room temperature. All cells were retrieved in complete neural crest media as described above and plated onto Geltrex-coated wells in the presence of Rock inhibitor (50&#x02009;&#x000b5;M) for 24&#x02009;h. 17q1q cells were additionally treated with DOX (100&#x02009;ng/ml) to induce <em>MYCN</em> expression. On D8, media were refreshed, and respective DOX treatment was continued but Rock inhibitor was discontinued. On D9, cells were collected for xenografting experiments and labeled with CellTrace<sup>TM</sup> Violet (Invitrogen, Thermo Fisher Scientific) for imaging. For this, cells were harvested with Accutase (PAN-Biotech) and resuspended at a concentration of 1*10<sup>6</sup> cells/ml in PBS. CellTrace<sup>TM</sup> Violet was added to a final concentration of 5&#x02009;&#x000b5;M for an incubation time of 10&#x02009;min at 37&#x02009;&#x000b0;C in the dark. The cell-staining mixture was filled up with 5 volumes of DMEM supplemented with 10% FBS and the suspension was incubated for 5&#x02009;min. After gentle centrifugation (5&#x02009;min, 500&#x02009;g, 4&#x02009;&#x000b0;C) the collected cells were resuspended in fresh DMEM medium supplemented with 10% FBS and incubated at 37&#x02009;&#x000b0;C for 10&#x02009;min. Adhering/ clumping cells were separated via a 35&#x02009;&#x000b5;m cell strainer. The cell number was adjusted to a concentration of 100 cells/nl in PBS. The freshly stained cells were kept on ice until transplantation. SK-N-BE2C-H2B-GFP cells<sup><sup><a href="#CR69" rid="CR69" class=" bibr popnode">69</a></sup></sup> (a kind gift of F. Westermann) were cultured in RPMI 1640 medium with GlutaMAX<sup>TM</sup> (Cat# 61870044, Thermo Fisher Scientific) supplemented with 10% (v/v) fetal bovine serum (Cat# F7524500ML, Sigma), 80 units/ml penicillin, 80&#x02009;&#x000b5;g/ml streptomycin (Cat# 15140122, Thermo Fisher Scientific), 1&#x02009;nM sodium pyruvate (Cat# P0443100, PAN-Biotech), 25&#x02009;mM Hepes buffer (PAN-Biotech) and 8&#x02009;&#x000b5;l/ml G418. For zebrafish xenotransplantations, the GFP-labelled cells were harvested and resuspended in PBS at a density of 10<sup>5</sup>/&#x000b5;l as described above.</p></div><div id="Sec28" class="sec"><p></p><h4 id="Sec28title" class="inline">Zebrafish strains, husbandry, and xenotransplantation </h4><p id="Par50" class="p p-first-last">Zebrafish (<em>Danio rerio</em>) were reared under standard conditions in a 14&#x02009;h/10&#x02009;h light cycle according to the guidelines of the local authorities (Magistratsabteilung MA58 of the municipal administration of Vienna, Austria) under licenses GZ:565304-2014-6 and GZ:534619-2014-4. For xenotransplantation experiments, the pigment mutant strain mitfa<sup>b692/b692</sup>; ednrba<sup>b140/b140</sup> was used. mitfa<sup>b692/b692</sup>; ednrba<sup>b140/b140</sup> embryos raised at 28&#x02009;&#x000b0;C were anaesthetised with Tricaine (0.16&#x02009;g/l Tricaine (Cat# E1052110G, Sigma-Aldrich), adjusted to pH 7 with 1&#x02009;M Tris pH 9.5, in E3) and xenotransplanted at 2 days post fertilisation (dpf)<sup><sup><a href="#CR107" rid="CR107" class=" bibr popnode">107</a></sup></sup>. For xenotransplantation, a micromanipulator (Cat# M3301R, World Precision Instruments) holding a borosilicate glass capillary (Cat# GB100T-8P, without filament, Science Products) connected to a microinjector (FemtoJet 4i, Eppendorf) was used. Transplantation capillaries were pulled with a needle puller (P-97, Sutter Instruments) and loaded with approximately 5&#x02009;&#x000b5;l of tumour cell suspension. Cells were injected into the perivitelline space (PVS) of larvae. Visual inspection was carried out at 2&#x02009;h post-injection on an Axio Zoom.V16 fluorescence microscope (Zeiss, Jena) and only correctly injected larvae were used in subsequent experiments and further maintained at 34&#x02009;&#x000b0;C.</p></div><div id="Sec29" class="sec sec-last"><p></p><h4 id="Sec29title" class="inline">Automated imaging and quantification </h4><p id="Par51" class="p p-first-last">One day post injection (1dpi) and 3dpi xenografted larvae were anaesthetised in 1x Tricaine and embedded in a 96-well ZF plate (Hashimoto Electronic Industry) with 0.5% ultra-low gelling agarose (Cat# A2576-25G, Sigma-Aldrich) for automated imaging on a high-content imager (Operetta CLS, PerkinElmer). Images were acquired with a 5x air objective. Exposure times for brightfield images was 40&#x02009;ms at 10% LED power. CellTrace Violet was recorded with an excitation of 390&#x02013;420&#x02009;nm at 100% LED power and detection at 430&#x02013;500&#x02009;nm using an exposure time of 600&#x02009;ms. GFP was excited with 460&#x02013;490&#x02009;nm and detected at 500&#x02013;550&#x02009;nm with an exposure time of 400&#x02009;ms. 23 planes with a distance of 25&#x02009;&#x000b5;m were imaged per field of view of the laterally orientated larvae to cover the whole tumour. Tumour size was quantified with Harmony Software 4.9 (PerkinElmer).</p></div></div><div id="Sec30" class="sec"><h3 id="Sec30title">Ethical use of data</h3><p id="Par52" class="p p-first-last">This study did not generate any new genomics data from patients. However, we performed re-analyses of previously published (sc)RNA-seq and SNP-array data that was previously collected at our institutions. The collection and research use of human tumour specimen was performed according to the guidelines of the Council for International Organisations of Medical Sciences (CIOMS) and World Health Organisation (WHO) and has been approved by the ethics board of the Medical University of Vienna (Ethikkommission Medizinische Universit&#x000e4;t Wien; EK2281/2016, 1216/2018, 1754/2022). Informed consent has been obtained from all patients or parents/guardians/legally authorised representatives. The age-adapted informed consent for the CCRI Biobank covers the use of left over materials from medically necessary surgery or biopsy, which after completion of routine diagnostic procedures is biobanked (EK1853/2016) and available for research purposes, including genetic analysis, that are further specified in EK1216/2018 and EK1754/2022: to conduct genetic and transcriptomic analysis and link to clinical data for survival analysis. All data obtained from external sources derived from studies where informed consent was given for broad research use.</p></div><div id="Sec31" class="sec"><h3 id="Sec31title">Whole-exome sequencing</h3><div id="Sec32" class="sec sec-first"><p></p><h4 id="Sec32title" class="inline">Library generation and sequencing </h4><p id="Par53" class="p p-first-last">Genomic DNA (gDNA) from cell lines was isolated using a desalting method and library preparation was performed with 100&#x02009;ng gDNA and the Enzymatic Fragmentation (undifferentiated hESC lines; Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">4b, c</a></sup>) or Enzymatic Fragmentation 2.0 (cells after differentiation; Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5e</span></a>, Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">9f, g</a></sup>) kit, and Twist Universal Adapter System (Twist). For whole-exome sequencing, the libraries were pooled and enriched with the Exome v1.3 and RefSeq (Human Comprehensive Exome) spike-in capture probes (Twist) according to the manufacturer&#x02019;s protocols. Libraries were quantified (Qubit 4 Fluorometer) and quality-checked on 4200 TapeStation and 2100 Bioanalyzer automated electrophoresis instruments (Agilent) and diluted before sequencing by the Biomedical Sequencing Facility at CeMM on an Illumina NovaSeq SP flowcell in 2x100bp paired-end mode (median coverage 87.2; Supplementary Data&#x000a0;<sup><a href="#MOESM5" rid="MOESM5" class=" media">1</a></sup>).</p></div><div id="Sec33" class="sec"><p></p><h4 id="Sec33title" class="inline">Variant identification and annotation </h4><p id="Par54" class="p p-first-last">Raw reads were processed using the nf-core <em>sarek</em><sup><sup><a href="#CR108" rid="CR108" class=" bibr popnode">108</a></sup>,<sup><a href="#CR109" rid="CR109" class=" bibr popnode">109</a></sup></sup> WES pipeline version 2.7.2. Variant calling was performed in a tumour-normal matched mode, with the parental H7 line serving as the matched normal sample. Three variant callers, <em>Mutect2</em>, <em>Strelka</em>, and <em>Manta</em><sup><sup><a href="#CR110" rid="CR110" class=" bibr popnode">110</a></sup>&#x02013;<sup><a href="#CR112" rid="CR112" class=" bibr popnode">112</a></sup></sup>, were employed for comprehensive variant identification. Resulting VCF files from <em>Mutect2</em> and <em>Strelka</em> were normalised using <em>bcftools</em> norm (v1.9)<sup><sup><a href="#CR113" rid="CR113" class=" bibr popnode">113</a></sup></sup> and subsequently annotated using the Ensembl <em>Variant Effect Predictor</em> (VEP; v99.2)<sup><sup><a href="#CR114" rid="CR114" class=" bibr popnode">114</a></sup></sup>. The identified variants were filtered based on the default quality control measures implemented in each tool (<em>FILTER</em> column in the VCF contains &#x0201c;PASS&#x0201d;). To identify biologically relevant variants a filtering strategy was applied that was partly inspired by MAPPYACTS<sup><sup><a href="#CR115" rid="CR115" class=" bibr popnode">115</a></sup></sup>: (i) exclude variants for which GERMQ or STRQ Phred-scaled values are &#x0003c;30; (ii) exclude variants with a population allele frequency of over 0.1% (in 1000 Genomes or gnomAD); (iii) only include variants that have any of &#x0201c;coding_sequence_variant&#x0201d;, &#x0201c;frameshift_variant&#x0201d;, &#x0201c;incomplete_terminal_codon&#x0201d;, &#x0201c;inframe_deletion&#x0201d;, &#x0201c;inframe_insertion&#x0201d;, &#x0201c;missense_variant&#x0201d;, &#x0201c;protein_altering_variant&#x0201d;, &#x0201c;start_lost&#x0201d;, &#x0201c;stop_gained&#x0201d;, &#x0201c;stop_lost&#x0201d; as Consequence; (iv) only include variants that have any of IMPACT&#x02009;=&#x02009;=&#x02009;HIGH, SIFT == &#x0201c;deleterious&#x0201d;, PolyPhen == probably_damaging or damaging<sup><sup><a href="#CR116" rid="CR116" class=" bibr popnode">116</a></sup>,<sup><a href="#CR117" rid="CR117" class=" bibr popnode">117</a></sup></sup>; (v) exclude variants that have a variant allele frequency &#x0003c;= 5%.</p></div><div id="Sec34" class="sec"><p></p><h4 id="Sec34title" class="inline">Copy number calling </h4><p id="Par55" class="p p-first-last">CNAs were called by Sequenza (version 3.0.0)<sup><sup><a href="#CR118" rid="CR118" class=" bibr popnode">118</a></sup></sup>. GC content was calculated for hg38 using <em>sequenza-utils gc_wiggle</em>. Depth ratio and B-allele frequency information was calculated using <em>bam2seqz</em> for each non-parental cell line using the parental cell line as a normal reference, single nucleotide polymorphisms (SNPs) were considered heterozygous if the allele frequency was in the range 0.4 to 0.6 (<em>--het</em> 0.4). Data was then binned using the <em>seqz_binning</em> command. Autosomes and the X chromosome were then extracted using Sequenza (<em>sequenza.extract</em>) and, as the cell lines are not contaminated with normal cells as is common place in tumour tissue samples, cellularity was tested in a range of 1 to 1.0002 to ensure a pure solution was produced by Sequenza. Copy number profiles were then plotted using ComplexHeatmaps<sup><sup><a href="#CR119" rid="CR119" class=" bibr popnode">119</a></sup></sup>. Breakpoints were considered telomeric if they were within 1Mbp of the beginning or end of the chromosome. Aberrant segments with 100 or more B-allele frequency observations (N.BAF) were considered to be confidently supported and are reported in Supplementary Data&#x000a0;<sup><a href="#MOESM9" rid="MOESM9" class=" media">5</a></sup>.</p></div><div id="Sec35" class="sec"><p></p><h4 id="Sec35title" class="inline">Phylogenetic analysis </h4><p id="Par56" class="p p-first-last">Mutations called by Mutect2<sup><sup><a href="#CR120" rid="CR120" class=" bibr popnode">120</a></sup></sup> with the PASS filter and of VARIANT_CLASS SNV as annotated by VEP<sup><sup><a href="#CR114" rid="CR114" class=" bibr popnode">114</a></sup></sup> that overlapped with the exome target panel without padding were used for phylogenetic analysis. Mutations were required to have a minimum variant allele frequency (VAF) of 0.2 to ensure only high frequency clonal mutations were included in the phylogeny. Phylogenetic trees were constructed using parsimony and the <em>phangorn</em> R package<sup><sup><a href="#CR121" rid="CR121" class=" bibr popnode">121</a></sup></sup>. The parsimony ratchet method (<em>pratchet</em>) was used to search for the best tree and the tree was rooted on the parental cell line. Branch lengths were calculated using the <em>acctran</em> function. Distance between tree tips was calculated using the <em>distTips</em> function in the <em>adephylo</em> R package<sup><sup><a href="#CR122" rid="CR122" class=" bibr popnode">122</a></sup></sup>. Phylogenetic trees were plotted using <em>ggtree</em><sup><sup><a href="#CR123" rid="CR123" class=" bibr popnode">123</a></sup></sup>.</p></div><div id="Sec36" class="sec sec-last"><p></p><h4 id="Sec36title" class="inline">Pre-processing and analysis of NB SNP-array data </h4><p id="Par57" class="p p-first-last">SNP-array data from tumour or bone marrow obtained at diagnosis from Austrian cases with INSS stage 4 high-risk NB<sup><sup><a href="#CR51" rid="CR51" class=" bibr popnode">51</a></sup></sup> were re-analysed for chr17 and chr1 CNAs using VARAN-GIE (v0.2.9), yielding 88 samples with CNAs (&#x0003e;10&#x02009;kb) on at least one of these chromosomes. Genomic segments were manually curated and plotted using <em>ggplot2</em> (v3.3.5). The available CNA annotations based on the human genome reference hg19. Because of this, the breakpoint annotations for our own cell lines have been brought from hg38 to hg19 using <em>liftOver</em> from the R package <em>rtracklayer</em> (v1.54.0).</p></div></div><div id="Sec37" class="sec"><h3 id="Sec37title">Single-cell RNA sequencing (10x Genomics)</h3><div id="Sec38" class="sec sec-first"><p></p><h4 id="Sec38title" class="inline">Library generation and sequencing </h4><p id="Par58" class="p p-first-last">Single-cell suspensions were barcoded using oligo-conjugated lipids following the MULTI-seq workflow and frozen live<sup><sup><a href="#CR124" rid="CR124" class=" bibr popnode">124</a></sup></sup> for G1-G13 (note, G2 was removed due to a technical failure), or frozen live and barcoded after thawing using the CELLPLEX (10x Genomics) workflow for G14-G27. After thawing cells were stained with DAPI. A maximum of 10,000 live cells per sample were sorted with a FACS-Aria v3 and pooled in sets of 3 or 4 samples by differentiation stage (from 3 to 5 independent replicate differentiation experiments). Each pooled group was processed using the 10X Genomics Single Cell 3&#x02019; v3.1 workflow following the manufacturer&#x02019;s instructions. Enriched barcode libraries were indexed following the MULTI-seq workflow<sup><sup><a href="#CR124" rid="CR124" class=" bibr popnode">124</a></sup></sup>. After quality control, libraries were sequenced on the Illumina NovaSeq S4 (G1-13) or S2 (G14-27) platform in 2&#x02009;&#x000d7;&#x02009;150&#x02009;bp paired-end mode. Supplementary Data&#x000a0;<sup><a href="#MOESM5" rid="MOESM5" class=" media">1</a></sup> includes an overview of sequencing data and performance metrics.</p></div><div id="Sec39" class="sec"><p></p><h4 id="Sec39title" class="inline">Raw data processing and alignment </h4><p id="Par59" class="p p-first-last">Raw sequencing data were processed with the <em>CellRanger multi</em> v7.1.0 software (10x Genomics) for cell-level demultiplexing and alignment to the human reference transcriptome (<em>refdata-gex-GRCh38-2020-A</em> assembly provided by 10x Genomics). Following initial data processing, all subsequent analyses were performed in R (v4.1.3) using Bioconductor packages and the <em>Seurat</em><sup><sup><a href="#CR125" rid="CR125" class=" bibr popnode">125</a></sup>&#x02013;<sup><a href="#CR127" rid="CR127" class=" bibr popnode">127</a></sup></sup> (v4.1.0) package.</p></div><div id="Sec40" class="sec"><p></p><h4 id="Sec40title" class="inline">Default basic processing </h4><p id="Par60" class="p p-first-last">We applied processing of scRNA-seq data in many instances across this manuscript. Unless parameters are otherwise specified, the default processing of scRNA-seq counts involved the following steps. Counts were normalised for read depth using Seurat&#x02019;s <em>SCTransform</em><sup><sup><a href="#CR128" rid="CR128" class=" bibr popnode">128</a></sup></sup> v0.3.3 (parameters: <em>method</em>&#x02009;=&#x02009;<em>&#x0201c;glmGamPoi&#x0201d;</em>; <em>variable.features.n</em>&#x02009;=&#x02009;<em>5000</em>), followed by <em>RunPCA</em> (keeping the top 50 components), and inference of cell neighbourhoods by <em>FindNeighbors</em> on the PCA reduction. Finally, Uniform Manifold Approximation and Projection (UMAP) was performed using Seurat&#x02019;s <em>RunUMAP</em> function with default parameters. Clustering was performed using FindClusters.</p></div><div id="Sec41" class="sec"><p></p><h4 id="Sec41title" class="inline">Quality control </h4><p id="Par61" class="p p-first-last">For each dataset, we first assessed technical covariates and characteristic expression profiles separately. We kept cells with less than 15% mitochondrial UMI counts, and at least 1000 detected genes and applied basic scRNA-seq processing and clustering of the cells (<em>SCTransform</em><sup><sup><a href="#CR128" rid="CR128" class=" bibr popnode">128</a></sup></sup> v0.3.3, parameters: <em>method</em>&#x02009;=&#x02009;<em>&#x0201c;glmGamPoi&#x0201d;</em>). Cell cycle scoring was calculated as recommended by Seurat and added as a variable to regress in SCTransform (<em>vars.to.regress</em>&#x02009;=&#x02009;<em>&#x0201d;ccvar&#x0201d;</em>). We used clusters devoid of markers and/or characterised by abnormally high mitochondrial expression, to derive a library-specific UMI count and mitochondrial percentage threshold for high-quality cells (thresholds for counts/mitochondrial percentage: G1: 3162/10%; G3: 10000/7.5%; G4: 10000/8%; G5: 3162/10%; G6: 10000/8%; G7: 12589/8%; G8: 7943/10%; G9: 7079/10%; G10: 3981/7.5%; G11: 3981/10%; G12: 5012/10%; G13: 10000/10%; G14: 5500/13%; G15: 3500/5%; G16: 3000/8%; G17: 2000/8%; G18: 3500/10%; G19: 1800/6%; G20: 3000/15%; G21: 6000/8%; G22: 5000/6%; G23: 3000/6%; G24: 1500/6%; G25: 3500/5%; G26: 2000/10%; G27: 3000/15%). In addition, empty and doublet droplets were flagged with <em>Emptydrops</em><sup><sup><a href="#CR129" rid="CR129" class=" bibr popnode">129</a></sup></sup> (v1.14.2; default parameters) and <em>scDblFinder</em><sup><sup><a href="#CR130" rid="CR130" class=" bibr popnode">130</a></sup></sup> (v1.8.0; parameters: <em>dbr</em>&#x02009;=&#x02009;<em>0.01</em>), respectively. We retained only cells with <em>Emptydrops</em> FDR&#x02009;&#x0003e;&#x02009;0.05 and individual scDblscore cutoffs for the datasets were: G1: 0.01; G3: 0.016; G4: 0.005; G5: 0.005; G6: 0.003; G7: 0.005; G8: 0.005; G9: 0.005; G10: 0.005; G11: 0.005; G12: 0.005; G13: 0.005; G14: 0.005; G15: 0.005; G16: 0.0075; G17: 0.002; G18: 0.007; G19: 0.00375; G20: 0.01; G21: 0.007; G22: 0.007; G23: 0.0125; G24: 0.003; G25: 0.007; G26: 0.005; G27: 0.007.</p></div><div id="Sec42" class="sec"><p></p><h4 id="Sec42title" class="inline">Sample demultiplexing </h4><p id="Par62" class="p p-first-last">To demultiplex cells belonging to different pooled samples, we used deMULTIplex2<sup><sup><a href="#CR131" rid="CR131" class=" bibr popnode">131</a></sup></sup> (v1.0.1) with default parameters on each dataset using the tag counts from <em>CellRanger multi</em>. All non-singlet cells were filtered out from the dataset.</p></div><div id="Sec43" class="sec"><p></p><h4 id="Sec43title" class="inline">Normalisation, clustering, and marker gene analysis for the main dataset </h4><p id="Par63" class="p p-first-last">Raw UMI counts were normalised using Seurat&#x02019;s <em>SCTransform</em><sup><sup><a href="#CR128" rid="CR128" class=" bibr popnode">128</a></sup></sup> (parameters: <em>variable.features.n</em>&#x02009;=&#x02009;5000<em>, method</em>&#x02009;=&#x02009;<em>&#x0201d;glmGamPoi&#x0201d;, vars.to.regress</em>&#x02009;=&#x02009;<em>&#x0201d;ccvar&#x0201d;</em>) to account for differences in sequencing depth and cell cycle phase (the variable &#x0201c;<em>ccvar</em>&#x0201d; variable was calculated as the difference of S and G2/M scores using Seurat&#x02019;s <em>CellCycleScoring</em> method with default parameters). To integrate data from 3 to 5 independent differentiation experiments (replicates; Supplementary Data&#x000a0;<sup><a href="#MOESM5" rid="MOESM5" class=" media">1</a></sup>), we used <em>scVI</em><sup><sup><a href="#CR132" rid="CR132" class=" bibr popnode">132</a></sup></sup> (v0.20.3; parameters: <em>n_epochs</em>&#x02009;=&#x02009;50) using 5000 highly variable features of the input data with Python 3.11 via reticulate (v1.24). Nearest neighbours were identified using Seurat&#x02019;s <em>FindNeighbors</em> function (parameters: <em>k</em>&#x02009;=&#x02009;30) on the ten scVI components. The same scVI reduction was used to find a low dimensionality UMAP projection using Seurat&#x02019;s RunUMAP for both the WT-only (<em>n.neighbours</em>&#x02009;=&#x02009;50<em>, min.dist</em>&#x02009;=&#x02009;0.5, <em>dims</em>&#x02009;=&#x02009;1:8) and full dataset (<em>n.neighbours:</em> 30<em>, min.dist</em>&#x02009;=&#x02009;0.4<em>, dims</em>&#x02009;=&#x02009;1:8 <em>method</em>&#x02009;=&#x02009;<em>&#x0201d;umap-learn&#x0201d;, metric</em>&#x02009;=&#x02009;<em>&#x0201d;correlation&#x0201d;)</em>. Clusters on the UMAP projection were defined using Seurat&#x02019;s <em>FindClusters</em> (parameters [full dataset]: <em>resolution</em>&#x02009;=&#x02009;0.6, parameters [WT-only]: <em>resolution</em>&#x02009;=&#x02009;0.4<em>, algorithm</em>&#x02009;=&#x02009;4). Neighbouring clusters that shared functional markers were merged manually and relabelled to roughly reflect differentiation order. Finally, markers for each cluster were identified using the <em>FindAllMarkers2</em> function (<em>DElegate</em><sup><sup><a href="#CR133" rid="CR133" class=" bibr popnode">133</a></sup></sup> v1.1.0; parameters: <em>method</em>&#x02009;=&#x02009;<em>&#x0201d;deseq&#x0201d;, min_fc</em>&#x02009;=&#x02009;<em>1, min_rate</em>&#x02009;=&#x02009;<em>0.5, replicate_column</em>&#x02009;=&#x02009;<em>&#x0201d;replicate&#x0201d;</em>), with each cluster compared to all the other cells in the dataset. Genes with an adjusted <em>P</em> value less than 0.05 were selected as markers. (Supplementary Data&#x000a0;<sup><a href="#MOESM6" rid="MOESM6" class=" media">2</a></sup>, <sup><a href="#MOESM12" rid="MOESM12" class=" media">8</a></sup>). To compare mutant and wild-type cells, we filtered the integrated dataset to cells from D9 and identified pairwise DEGs (<em>P</em><sub>adj</sub>&#x02009;&#x02264;&#x02009;0.05, |log<sub>2</sub>FoldChange&#x02009;|&#x02009;&#x0003e;&#x02009;0.25) between each mutant condition and WT using the <em>findDE</em>function (Delegate v1.1.0; parameters: <em>group_column</em>&#x02009;=&#x02009;<em>&#x0201c;condition&#x0201d;, method</em>&#x02009;=&#x02009;<em>&#x0201c;deseq&#x0201d;, replicate_column</em>&#x02009;=&#x02009;<em>&#x0201c;day_rep&#x0201d;</em>). We discarded DEGs that were not expressed in at least 20% of cells on one side of the comparison. Up- and down-regulated DEGs on chr1q, on chr17q, and outside either CNA were then tested separately to identify significant overlaps with MSigDB HALLMARK<sup><sup><a href="#CR134" rid="CR134" class=" bibr popnode">134</a></sup></sup> gene sets using the hypergeometric test implemented in the <em>hypeR</em><sup><sup><a href="#CR135" rid="CR135" class=" bibr popnode">135</a></sup></sup> package (v1.10.0). DEGs and enriched pathways are listed in Supplementary Data&#x000a0;<sup><a href="#MOESM10" rid="MOESM10" class=" media">6</a></sup> and Supplementary Data&#x000a0;<sup><a href="#MOESM11" rid="MOESM11" class=" media">7</a></sup>.</p></div><div id="Sec44" class="sec"><p></p><h4 id="Sec44title" class="inline">Pseudotime trajectory analysis </h4><p id="Par64" class="p p-first-last">Pseudotime trajectories were inferred using <em>Slingshot</em><sup><sup><a href="#CR136" rid="CR136" class=" bibr popnode">136</a></sup></sup> (v2.2.0; default parameters) using a filtered dataset comprising only MES-SYM clusters C11, C12, C13, and C14 (cp. Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1d</span></a>; Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">3</a></sup>). The filtered dataset was reprocessed using the basic scRNA-seq processing workflow as described above and the first two principal components were used to find trajectories between two extreme clusters. Only one trajectory was found. Genes whose expression was associated with the trajectories were identified with the generalised additive model and association test as implemented in <em>tradeSeq</em><sup><sup><a href="#CR137" rid="CR137" class=" bibr popnode">137</a></sup></sup> (v1.8.0; parameters: <em>knots</em>&#x02009;=&#x02009;5<em>)</em>. The top genes with the highest Wald statistic were selected for reporting (Supplementary Data&#x000a0;<sup><a href="#MOESM7" rid="MOESM7" class=" media">3</a></sup>). Transcription factors were identified based on the human transcription factors database<sup><sup><a href="#CR138" rid="CR138" class=" bibr popnode">138</a></sup></sup> in Supplementary Fig.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">3b</a></sup>.</p></div><div id="Sec45" class="sec"><p></p><h4 id="Sec45title" class="inline">Cross-dataset annotation, label transfer, and signature scores </h4><p id="Par65" class="p p-first-last">To map data between scRNA-seq datasets, we employed Seurat&#x02019;s label transfer workflow. Both query and reference datasets were processed using the default basic scRNA-seq processing workflow as described above and mapped (<em>FindTransferAnchors</em>, <em>TransferData</em>, <em>IntegrateEmbeddings</em>, <em>NNTransform</em>, and <em>MappingScore</em> functions; default parameters) using the 50 first principal components of the PCA reduction from both datasets. To visualise cell mappings, we used &#x0201c;glasswork plots&#x0201d;&#x000a0;(see below). In this study, the following mappings were performed with the same processing and parameters:</p><ol class="enumerated" style="list-style-type:decimal"><li><div id="Par66">Human foetal adrenal reference datasets<sup><sup><a href="#CR15" rid="CR15" class=" bibr popnode">15</a></sup>,<sup><a href="#CR16" rid="CR16" class=" bibr popnode">16</a></sup></sup> mapped onto WT-only (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1d&#x02013;f</span></a>; Supplementary Figs.&#x000a0;<sup><a href="#MOESM2" rid="MOESM2" class=" media">2i, j,</a></sup><sup><a href="#MOESM2" rid="MOESM2" class=" media">3c</a></sup>) and full in-vitro (Figs.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2d&#x02013;f</span></a>, <a href="/pmc/articles/PMC11068915/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,4a;</span></span><span>4a</span></a>; Supplementary Fig. <sup><a href="#MOESM1" rid="MOESM1" class=" media">6g</a></sup>) scRNA-seq references. Upon obtaining consistent results for both (Supplementary Fig. <sup><a href="#MOESM2" rid="MOESM2" class=" media">2j</a></sup>), the reference provided by Kameneva et al. was used throughout the analysis because of the curated cell type markers they provided (Supplementary Fig. <sup><a href="#MOESM2" rid="MOESM2" class=" media">2i</a></sup>). These gene signatures were also quantified with Seurat&#x02019;s <em>AddModuleScore</em> function (default parameters) in Figs.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1e, f</span></a>, <a href="/pmc/articles/PMC11068915/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,2f</span></span><span>2f</span></a>.</div></li><li><div id="Par67">Our mutant scRNA-seq data mapped onto the wild-type reference (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2b, c</span></a>).</div></li><li><div id="Par68">NB tumour scRNA-seq data mapped onto our WT-only reference (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig7" rid-ob="ob-Fig7" co-legend-rid="lgnd_Fig7"><span>7b, c</span></a>; Supplementary Fig. <sup><a href="#MOESM2" rid="MOESM2" class=" media">11</a></sup>). See additional details about these datasets and processing in the section <em>&#x0201c;Pre-processing and mapping of NB tumour data&#x0201d;</em> below.</div></li><li><div id="Par69">Extended data from a split-pool scRNA-seq (Parse Biosciences) mapped to the WT-only dataset (10x Genomics) (Supplementary Fig. <sup><a href="#MOESM2" rid="MOESM2" class=" media">8b, c</a></sup>).</div></li></ol><p></p></div><div id="Sec46" class="sec"><p></p><h4 id="Sec46title" class="inline">Validation of label transfers </h4><p id="Par70" class="p p-first-last">WT mappings to adrenal gland references were validated by the presence of relevant markers (Supplementary Fig. <sup><a href="#MOESM2" rid="MOESM2" class=" media">2i</a></sup>). Mutant and tumour cell mappings were not strictly curated via markers (i.e., they were allowed to deviate). When analysing markers of mapped mutant and tumour cells, cells with a prediction score of 0.4 or higher were used to minimise ambiguous mappings and maximise marker discovery. Shared markers were consistently found between the query and the cognate cells in the reference, even though their number varied (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig7" rid-ob="ob-Fig7" co-legend-rid="lgnd_Fig7"><span>7c</span></a>, Supplementary Fig. <sup><a href="#MOESM2" rid="MOESM2" class=" media">8d</a></sup>).</p></div><div id="Sec47" class="sec"><p></p><h4 id="Sec47title" class="inline">Visualising label transfers with glasswork plots </h4><p id="Par71" class="p p-first-last">To visualise cell mappings, we used &#x0201c;glasswork plots&#x0201d;, in which the UMAP of the reference was used to define the coordinates of concave hulls for each cluster (calculated with R package <em>concaveman</em> v1.1.0). Query cells mapping to each cluster were plotted at random positions within their mapped target cluster hull to mitigate overplotting. Input cell populations for the plot were downsampled evenly by condition and stage (<em>n</em>&#x02009;=&#x02009;1000 cells) to avoid sampling effects.</p></div><div id="Sec48" class="sec"><p></p><h4 id="Sec48title" class="inline">Mutation score analysis </h4><p id="Par72" class="p p-first-last">To calculate the mutation score, we focused on days 9, 14, and 19 as they contained samples from all conditions. We encoded each cell&#x02019;s genotype as a number <em>G</em> based on the genetic lineage of hESC lines: <em>G(WT)</em>&#x02009;=&#x02009;<em>0</em>, <em>G(17q)=1</em>, <em>G(17q1q)=2</em>, and <em>G(17q1qMYCN)=3</em>. We then calculated the mutation score <em>m</em> as the mean <em>G</em> of the cell&#x02019;s <em>K</em> nearest neighbouring cells (<em>K</em>&#x02009;=&#x02009;<em>30</em>) in the scVI reduction&#x02019;s neighbourhood graph (see <em>&#x0201c;Normalisation, clustering, and marker gene analysis&#x0201d;</em>). Division by 3 yielded a score between 0 and 1. Intuitively, the mutation score of a cell indicates whether a cell phenotypically resembles wild-type cells or cells with a given number of relevant alterations independent of its own genotype. To find genes correlated with the mutation score, we calculated Pearson correlation coefficients with gene expression in three settings: (i) correlation for each gene with <em>m</em> in all cells; (ii) correlation for each gene with <em>m</em> leaving out the 17q1qMYCN cells, to emphasise subtle correlations with CNAs; and (iii) correlation for each gene and the neighbourhood entropy (Shannon entropy of all genotype scores <em>G</em> of the <em>K</em> nearest neighbours), to find genes appearing in mixed regions. All non-duplicate absolute correlations (calculated using R&#x02019;s <em>cor.test</em>, parameters: <em>method</em>&#x02009;=&#x02009;<em>&#x0201d;pearson&#x0201d;, exact</em>&#x02009;=&#x02009;<em>TRUE</em>) were subject to Bonferroni correction and ranked. The top correlated genes (<em>p</em>&#x02009;&#x02264;&#x02009;0.05) per differentiation stage (D9, D14, D19) are reported in Supplementary Data&#x000a0;<sup><a href="#MOESM13" rid="MOESM13" class=" media">9</a></sup>.</p></div><div id="Sec49" class="sec"><p></p><h4 id="Sec49title" class="inline">Pre-processing and mapping of NB tumour data </h4><p id="Par73" class="p p-first">We collected scRNA-seq data for tumours with reported <em>MYCN</em> amplification from three sources from the stated database or the corresponding authors:</p><ul class="unordered" style="list-style-type:disc"><li><div id="Par74">Three samples (all primary adrenal, 2 male [Dong_T162, Dong_T230], 1 female [Dong_T200]; accession <a id="__tag_1147980653" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE137804" ref="/geo/query/acc.cgi?acc=GSE137804"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE137804","term_id":"137804"}}</span>GSE137804</a> [Gene Expression Omnibus])<sup><sup><a href="#CR17" rid="CR17" class=" bibr popnode">17</a></sup></sup>,</div></li><li><div id="Par75">three samples (1 primary adrenal, 1 primary intraspinal, 1 relapse/occipital subcutaneous bone metastasis [Jansky_NB14]; 1 female [Jansky_NB08], 2 male [Jansky_NB01, Jansky_NB14]; accession EGAS00001004388 [European Genome-Phenome Archive])<sup><sup><a href="#CR15" rid="CR15" class=" bibr popnode">15</a></sup></sup>,</div></li><li><div id="Par76">and four samples (all metastatic bone marrow; 3 female [Fetahu_M1, Fetahu_M3, Fetahu_M4], 1 male [Fetahu_M2]; accession <a id="__tag_1147980657" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE216176" ref="/geo/query/acc.cgi?acc=GSE216176"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE216176","term_id":"216176"}}</span>GSE216176</a> [Gene Expression Omnibus])<sup><sup><a href="#CR23" rid="CR23" class=" bibr popnode">23</a></sup></sup>.</div></li></ul><p></p><p id="Par77" class="p p-last">Additional details about each dataset are available from the original research articles. In each dataset, cells with more than 500 reads per barcode and mitochondrial DNA less than 20% were kept for further analysis. We then performed an adrenal gland mapping<sup><sup><a href="#CR16" rid="CR16" class=" bibr popnode">16</a></sup></sup> (same workflow as described above) and discarded cells mapping to the category &#x0201c;HSC_and_immune&#x0201d;. This process left us with strong CNA profiles (see below) at key genomic positions such as chr2p (<em>MYCN</em> locus). Cells were then subjected to default basic scRNA-seq processing (see above) and mapped onto our WT-only reference (see above).</p></div><div id="Sec50" class="sec"><p></p><h4 id="Sec50title" class="inline">Inference of CNA profiles from scRNA-seq data </h4><p id="Par78" class="p p-first-last">To infer tumour cell CNA profiles from scRNA-seq expression data, we used the <em>infercnv</em><sup><sup><a href="#CR139" rid="CR139" class=" bibr popnode">139</a></sup></sup> R package (v1.10.1). We first removed cells with less than 500 UMI counts. Then, we created a pan-patient healthy reference cell population by sampling from each patient 500 cells that we determined to be HSC/immune cells based on a mapping to a human embryonic adrenal gland reference<sup><sup><a href="#CR16" rid="CR16" class=" bibr popnode">16</a></sup></sup>. For every patient, we then ran <em>infercnv</em> with the non-HSC/immune cells as the main input and the pan-patient HSC/immune cells as a reference. The <em>cutoff</em> parameter was set to 0.1, all other parameters were left at their default values.</p></div><div id="Sec51" class="sec"><p></p><h4 id="Sec51title" class="inline">Pre-processing and analysis of NB bulk RNA-seq data </h4><p id="Par79" class="p p-first-last">We obtained bulk RNA-seq counts and associated metadata from patient-derived NB samples from three sources: TARGET<sup><sup><a href="#CR24" rid="CR24" class=" bibr popnode">24</a></sup></sup> (phs000467 [Genomic Data Commons]), SEQC<sup><sup><a href="#CR70" rid="CR70" class=" bibr popnode">70</a></sup>,<sup><a href="#CR71" rid="CR71" class=" bibr popnode">71</a></sup></sup> (<a id="__tag_1147980655" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE49711" ref="/geo/query/acc.cgi?acc=GSE49711"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE49711","term_id":"49711"}}</span>GSE49711</a> [Gene Expression Omnibus]) and from our institution<sup><sup><a href="#CR96" rid="CR96" class=" bibr popnode">96</a></sup>,<sup><a href="#CR99" rid="CR99" class=" bibr popnode">99</a></sup>,<sup><a href="#CR140" rid="CR140" class=" bibr popnode">140</a></sup>&#x02013;<sup><a href="#CR146" rid="CR146" class=" bibr popnode">146</a></sup></sup> (labelled &#x0201c;CCRI&#x0201d; in the figures; <a id="__tag_1147980668" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE94035" ref="/geo/query/acc.cgi?acc=GSE94035"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE94035","term_id":"94035"}}</span>GSE94035</a>, <a id="__tag_1147980674" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE147635" ref="/geo/query/acc.cgi?acc=GSE147635"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE147635","term_id":"147635"}}</span>GSE147635</a> and <a id="__tag_1147980669" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE172184" ref="/geo/query/acc.cgi?acc=GSE172184"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE172184","term_id":"172184"}}</span>GSE172184</a> [Gene Expression Omnibus]). Open access unstranded counts from TARGET patients were obtained directly from the GDC data portal (subsection TARGET:NBL, phs000467). Counts from the CCRI patients were obtained in-house. Both CCRI and TARGET datasets were normalised using DESeq2<sup><sup><a href="#CR147" rid="CR147" class=" bibr popnode">147</a></sup></sup> (v1.34.0) and transformed using the variance stabilising transformation. A prenormalised log<sub>2</sub> SEQC matrix was exponentiated, rounded to the nearest integer, and subjected to variance stabilising transformation. In all datasets, the names of relevant marker genes were harmonised manually in case the gene was found with a different name. Each dataset was analysed separately due to differences in count quantification and normalisation. PCA projections of the normalised variables revealed mainly biological/clinical variables (and not technical variables) having major weight in the variance of the datasets. Only NB data collected at diagnosis were used for our analyses (discarding, e.g., ganglioneuroblastoma and relapse data). To quantify the in-vitro cluster signature strength, we used the intersection of markers found both in our in-vitro WT-only dataset (Supplementary Data&#x000a0;<sup><a href="#MOESM6" rid="MOESM6" class=" media">2</a></sup>) and in the tumour scRNA-seq datasets (Supplementary Data&#x000a0;<sup><a href="#MOESM14" rid="MOESM14" class=" media">10</a></sup>). We then used the function <em>gsva</em> (from <em>GSVA</em><sup><sup><a href="#CR148" rid="CR148" class=" bibr popnode">148</a></sup></sup> v1.42.0; parameters: <em>method</em>&#x02009;=&#x02009;<em>&#x0201d;ssgsea&#x0201d;</em>) to calculate signature scores for each of the shared cluster signatures.</p></div><div id="Sec52" class="sec sec-last"><p></p><h4 id="Sec52title" class="inline">Survival analysis </h4><p id="Par80" class="p p-first-last">We obtained survival data for the SEQC cohort from the original publication<sup><sup><a href="#CR70" rid="CR70" class=" bibr popnode">70</a></sup></sup>. Event-free survival (EFS) was defined as time from diagnosis to any of the following events: Relapse/progression of disease, or death due to any cause and secondary malignancies. Patients without events were censored at last follow-up evaluation. Statistical analyses were performed using SAS (v9.4). Cluster signatures (see previous section) were dichotomised using the median value, and the impact of these signatures on EFS was evaluated in a Cox-proportional hazards model adjusting for stage4 (yes/no), age-group (&#x0003c;18, 18&#x02013;60, &#x0003e;60) and <em>MYCN</em> amplification status (yes/no). Each cluster signature was evaluated separately. Two-sided <em>P</em> values were adjusted for multiple hypothesis testing using the Benjamini-Hochberg method and are reported together with hazard ratios and two-sided 95% confidence intervals in Supplementary Data&#x000a0;<sup><a href="#MOESM15" rid="MOESM15" class=" media">11</a></sup>.</p></div></div><div id="Sec53" class="sec"><h3 id="Sec53title">Split-pool single-cell RNA sequencing (Parse Biosciences)</h3><div id="Sec54" class="sec sec-first"><p></p><h4 id="Sec54title" class="inline">Library generation and sequencing </h4><p id="Par81" class="p p-first-last">Cells were harvested with Accutase to create a single-cell suspension, then were counted using Bio-rad Tc10 Automated cell counter in the presence of Trypan Blue Stain (Bio-rad). For cell fixation we used the Evercode Fixation v2 Kit (SKU: ECF 2001, Parse Biosciences, Seattle, USA) as per manufacturer instructions. A maximum of 5000 cells per sample were multiplexed using the Evercode WT Mega v2 kit (Parse Biosciences). Three rounds of combinatorial barcoding were performed, and cells were then pooled and split into 16 sub-libraries (one small 5000-cell sub-library and 15 large sub-libraries of 32,000 cells each). After DNA amplification and library prep, the small library was sequenced as part of a larger Illumina NovaSeq S4 flowcell and the 15 large sub-libraries on one dedicated NovaSeq S4 platform in 2&#x02009;&#x000d7;&#x02009;150&#x02009;bp paired-end mode.</p></div><div id="Sec55" class="sec"><p></p><h4 id="Sec55title" class="inline">Raw data processing and alignment </h4><p id="Par82" class="p p-first-last">Raw sequencing data were processed with the <em>split-pipe</em> v1.0.6p software (Parse Biosciences) for cell-level demultiplexing and alignment to the human reference transcriptome (<em>refdata-gex-GRCh38-2020-A</em> assembly provided by 10x Genomics; parameters: <em>-m all -c v2</em>). Following initial data processing, all subsequent analyses were performed in R (v4.1.3) using Bioconductor packages and the <em>Seurat</em><sup><sup><a href="#CR125" rid="CR125" class=" bibr popnode">125</a></sup>&#x02013;<sup><a href="#CR127" rid="CR127" class=" bibr popnode">127</a></sup></sup> (v4.1.0) package.</p></div><div id="Sec56" class="sec sec-last"><p></p><h4 id="Sec56title" class="inline">Basic processing, quality control, and marker analysis </h4><p id="Par83" class="p p-first-last">We applied the <em>cb_filter_count_matrix</em> (default parameters) function from <em>canceRbits</em> (v0.1.6; default parameters) to remove cells with high mitochondrial counts (&#x0003e;15%), unusually high/low number of genes (&#x0003c;300 genes or z-score of log(genes) not in range (&#x02212;3, 3)), abnormally high/low number of reads (z-score of log(transcripts) not in range (&#x02212;3, 3)), or an abnormal transcript-to-gene ratio (z-score of residuals of loess fit of &#x0201c;log(genes) ~ log(transcripts)&#x0201d; not in range (&#x02212;5, 5)), and the <em>cb_seurat_pipeline</em> function (parameters: <em>seurat_max_pc</em>&#x02009;=&#x02009;15<em>, metric</em>&#x02009;=&#x02009;<em>&#x0201c;manhattan&#x0201d;, k_param</em>&#x02009;=&#x02009;20<em>, n_neighbors</em>&#x02009;=&#x02009;40<em>, cluster_res</em>&#x02009;=&#x02009;0.3) to perform a standard Seurat analysis workflow including data normalisation, dimensionality reduction, and clustering. Subsequently, the data were mapped to our 10x-based WT-only reference as described above. To identify marker genes for cells mapped to different clusters of the WT reference (for cells with prediction score &#x0003e;=0.4) we again used DElegate::FindAllMarkers2. (DElegate v1.1.0; <em>parameters: method</em>&#x02009;=&#x02009;<em>&#x0201d;deseq&#x0201d;, min_fc</em>&#x02009;=&#x02009;<em>1, min_rate</em>&#x02009;=&#x02009;<em>0.5, replicate_column</em>&#x02009;=&#x02009;<em>&#x0201d;replicate&#x0201d;</em>), and kept all genes with adjusted pvalue of 0.05 of less.</p></div></div><div id="Sec57" class="sec"><h3 id="Sec57title">Chromatin accessibility mapping (ATAC-seq)</h3><div id="Sec58" class="sec sec-first"><p></p><h4 id="Sec58title" class="inline">Library generation and sequencing </h4><p id="Par84" class="p p-first-last">ATAC-seq was performed using established protocols<sup><sup><a href="#CR77" rid="CR77" class=" bibr popnode">77</a></sup></sup>. Briefly, 20,000 to 50,000 cells were lysed in the transposase reaction mix (12.5&#x02009;&#x003bc;l 2xTD buffer, 2&#x02009;&#x003bc;l TDE1 [Illumina], 10.25&#x02009;&#x003bc;l nuclease-free water, 0.25&#x02009;&#x003bc;l 1% digitonin [Promega], and 0.5&#x02009;&#x003bc;l of 50x cOmplete Mini EDTA-free Protease Inhibitor Cocktail [Roche]) for 30&#x02009;min at 37&#x02009;&#x000b0;C. Following DNA purification with the MinElute kit (Qiagen) eluting in 12&#x02009;&#x003bc;l, 1&#x02009;&#x003bc;l of eluted DNA was used in a qPCR reaction to estimate the optimum number of amplification cycles. The remaining 11&#x02009;&#x003bc;l of each library were amplified for the number of cycles corresponding to the Cq value (i.e., the cycle number at which fluorescence has increased above background levels) from the qPCR using custom Nextera primers<sup><sup><a href="#CR149" rid="CR149" class=" bibr popnode">149</a></sup></sup>. Library amplification was followed by SPRI (Beckman Coulter) size selection to exclude fragments larger than 1200&#x02009;bp. Library concentration was measured with a Qubit fluorometer (Life Technologies), and libraries were quality checked using a 2100 Bioanalyzer (Agilent Technologies). Libraries were sequenced by the Biomedical Sequencing Facility at CeMM using the Illumina HiSeq 4000 platform in 1&#x02009;&#x000d7;&#x02009;50&#x02009;bp single-end mode. Supplementary Data&#x000a0;<sup><a href="#MOESM5" rid="MOESM5" class=" media">1</a></sup> includes an overview of the sequencing data and performance metrics.</p></div><div id="Sec59" class="sec"><p></p><h4 id="Sec59title" class="inline">Raw data processing, alignment, and quality control </h4><p id="Par85" class="p p-first-last">Raw sequencing data were processed using <em>PEPATAC</em><sup><sup><a href="#CR150" rid="CR150" class=" bibr popnode">150</a></sup></sup> (v0.9.5; default parameters) including alignment to the human genome (<em>refdata-cell ranger-atac-GRCh38-1.2.0</em> assembly provided by 10x Genomics for maximum compatibility with scRNA-seq analyses). Following initial data processing, all subsequent analyses were performed in R (v4.1.3) using Bioconductor packages and <em>ggplot2</em><sup><sup><a href="#CR151" rid="CR151" class=" bibr popnode">151</a></sup></sup> (v3.3.5) and <em>ComplexHeatmap</em><sup><sup><a href="#CR119" rid="CR119" class=" bibr popnode">119</a></sup></sup> (v2.10.0) for plotting. After discarding low-quality data (NRF&#x02009;&#x0003c;&#x02009;0.65 or PBC1&#x02009;&#x0003c;&#x02009;0.7 or PBC2&#x02009;&#x0003c;&#x02009;1 or FRiP&#x02009;&#x0003c;&#x02009;0.025), we removed peaks overlapping blacklisted regions from ENCODE (<a href="http://mitra.stanford.edu/kundaje/akundaje/release/blacklists/hg38-human/hg38.blacklist.bed.gz" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://mitra.stanford.edu/kundaje/akundaje/release/blacklists/hg38-human/hg38.blacklist.bed.gz</a>) and merged overlapping peaks across all ATAC-seq datasets to create a common set of consensus genomic regions for subsequent analysis (Supplementary Data&#x000a0;<sup><a href="#MOESM16" rid="MOESM16" class=" media">12</a></sup>). Next, we quantified for each input dataset the number of reads overlapping these consensus peaks using <em>featureCounts</em><sup><sup><a href="#CR152" rid="CR152" class=" bibr popnode">152</a></sup></sup> (<em>Rsubread</em> v2.8.2).</p></div><div id="Sec60" class="sec"><p></p><h4 id="Sec60title" class="inline">Differential accessibility analysis and chromatin modules </h4><p id="Par86" class="p p-first-last">Raw read counts were loaded into DESeq2<sup><sup><a href="#CR147" rid="CR147" class=" bibr popnode">147</a></sup></sup> (v1.34.0; default parameters, design: <em>~lane+batch+sample_group</em>) for normalisation (variance-stabilising transformation) and differential analysis. In doing so, we estimated count size factors for normalisation excluding regions on chromosomes with known chromosomal aberrations (i.e., chr1, chr17) to avoid overcompensation due to differences in global signal strength. We queried all pairwise comparisons of sample groups stratified by cell line/condition stratified (time-wise differences, e.g., WT-D3 vs. WT-D0) and between conditions stratified by stage (condition-wise differences, e.g., 17q-D9 vs. WT-D9) and recorded all significantly differentially accessible regions (<em>P</em><sub>adj</sub>&#x02009;&#x02264;&#x02009;0.005, |log<sub>2</sub>FoldChange|&#x02009;&#x02265;&#x02009;log<sub>2</sub>(1.5); parameters: <em>pAdjustMethod</em>&#x02009;=&#x02009;<em>&#x0201c;BH&#x0201d;, lfcThreshold=log2(1.5), independentFiltering</em>&#x02009;=&#x02009;<em>TRUE;</em> Supplementary Data&#x000a0;<sup><a href="#MOESM17" rid="MOESM17" class=" media">13</a></sup>). To define chromatin regulatory modules, we divided time-wise differences in WT hESCs (<em>n</em>&#x02009;=&#x02009;41,699 regions) into six chromatin modules (R1-R6) and condition-wise differences (<em>n</em>&#x02009;=&#x02009;3914 regions) into three chromatin modules (R7-R9) by hierarchical clustering using the Ward criterion (parameter: <em>method</em>&#x02009;=&#x02009;<em>&#x0201c;ward.D2&#x0201d;</em>). To associate ATAC-seq regions with putative target genes, we used the <em>GenomicRanges</em><sup><sup><a href="#CR153" rid="CR153" class=" bibr popnode">153</a></sup></sup> package (v1.46.1) to assign each region to all genes (using the <em>refdata-gex-GRCh38-2020-A</em> gene annotation provided by 10x Genomics) with overlapping promoters (transcription start side) or to distal genes whose promoter within a maximum distance of 250&#x02009;kb whose expression was significantly correlated with the region&#x02019;s accessibility. To this end, we calculated the correlation coefficient between normalised read counts in our ATAC-seq data with the normalised read counts in our matching scRNA-seq data (mean of cells per sample; note, the ATAC-seq was collected from the same experiments as the first replicate experiments for scRNA-seq). We calculated an empirical false discovery rate (FDR) by shuffling RNA/ATAC assignments (10 repetitions) and retained associations with a value &#x02264;0.05. Annotated regulatory regions from the analysis of ATAC-seq data are listed in Supplementary Data&#x000a0;<sup><a href="#MOESM16" rid="MOESM16" class=" media">12</a></sup>.</p></div><div id="Sec61" class="sec"><p></p><h4 id="Sec61title" class="inline">Overlap enrichment analysis for chromatin modules </h4><p id="Par87" class="p p-first-last">To characterise the chromatin modules, we interrogated overlaps with genomic regions or associated genes using the hypergeometric test implemented in the <em>hypeR</em><sup><sup><a href="#CR135" rid="CR135" class=" bibr popnode">135</a></sup></sup> package (v1.10.0) via the <em>cb_hyper</em> function (<em>canceRbits</em> v0.1.6; parameters: collapse = FALSE, min_size = 5, max_size = (&#x0003c;75% of the size of the background dataset)). We looked at three types of overlaps: (a) Annotated reference regions from the DNase hypersensitivity index<sup><sup><a href="#CR78" rid="CR78" class=" bibr popnode">78</a></sup></sup>, from the Cis-element Atlas<sup><sup><a href="#CR79" rid="CR79" class=" bibr popnode">79</a></sup></sup>, from the Enhancer Atlas<sup><sup><a href="#CR80" rid="CR80" class=" bibr popnode">80</a></sup></sup>, and NB subgroup-specific super-enhancers<sup><sup><a href="#CR73" rid="CR73" class=" bibr popnode">73</a></sup></sup>, which all catalogue regulatory elements active in different cell or tissue types. (b) Matches to known TF motifs from the <em>HOCOMOCO</em> database<sup><sup><a href="#CR154" rid="CR154" class=" bibr popnode">154</a></sup></sup> (v11). Here, we downloaded motifs from the <em>HOCOMOCO</em> website (<em>HOCOMOCOv11_full_annotation_HUMAN_mono.tsv</em>) and used <em>motifmatchr</em> (v1.16.0) to scan the DNA sequences underlying each genomic region for matches. Regions with at least one match to the motif were recorded as potential binding sites. (c) Marker genes from our scRNA-seq analysis of WT hESC differentiation (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1c</span></a>; Supplementary Data&#x000a0;<sup><a href="#MOESM6" rid="MOESM6" class=" media">2</a></sup>). For this purpose, genomic regions were associated with genes as described above. In each case, we used the entire set of all analysed genomic regions as a background for the enrichment analysis, and we considered overlaps with an FDR-corrected <em>P</em> value less than 0.005 as significant. For motifs, we find the reported <em>P</em> values are inflated and therefore used stricter thresholds: <em>P</em><sub>adj</sub>&#x02009;&#x02264;&#x02009;0.0000001, |log<sub>2</sub> odds|&#x02009;&#x0003e;&#x02009;log<sub>2</sub>(2). All enrichment results are reported in Supplementary Data&#x000a0;<sup><a href="#MOESM18" rid="MOESM18" class=" media">14</a></sup>.</p></div><div id="Sec62" class="sec"><p></p><h4 id="Sec62title" class="inline">Integration with published ATAC/DNaseI-seq data </h4><p id="Par88" class="p p-first-last">To interrogate accessibility of the chromatin modules in existing data from NB cell lines we used fast gene set enrichment analysis <em>fgsea</em> (v1.20.0)<sup><sup><a href="#CR155" rid="CR155" class=" bibr popnode">155</a></sup></sup>. We obtained ready-processed genomic coverage tracks (<em>wig</em> or <em>bigwig</em> files) from three studies profiling NB cell lines<sup><sup><a href="#CR73" rid="CR73" class=" bibr popnode">73</a></sup>,<sup><a href="#CR156" rid="CR156" class=" bibr popnode">156</a></sup>,<sup><a href="#CR157" rid="CR157" class=" bibr popnode">157</a></sup></sup> (<a id="__tag_1147980678" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE138293" ref="/geo/query/acc.cgi?acc=GSE138293"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE138293","term_id":"138293"}}</span>GSE138293</a>, <a id="__tag_1147980673" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE224241" ref="/geo/query/acc.cgi?acc=GSE224241"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE224241","term_id":"224241"}}</span>GSE224241</a>, <a id="__tag_1147980660" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE136279" ref="/geo/query/acc.cgi?acc=GSE136279"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE136279","term_id":"136279"}}</span>GSE136279</a>). Additionally, we obtained data from three studies profiling breast<sup><sup><a href="#CR158" rid="CR158" class=" bibr popnode">158</a></sup></sup> (<a id="__tag_1147980676" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE202511" ref="/geo/query/acc.cgi?acc=GSE202511"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE202511","term_id":"202511"}}</span>GSE202511</a>) and lung cancer<sup><sup><a href="#CR159" rid="CR159" class=" bibr popnode">159</a></sup></sup> lines (<a id="__tag_1147980670" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE228832" ref="/geo/query/acc.cgi?acc=GSE228832"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE228832","term_id":"228832"}}</span>GSE228832</a>), or human tissue data<sup><sup><a href="#CR160" rid="CR160" class=" bibr popnode">160</a></sup></sup> (<a href="https://epigenome.wustl.edu/epimap" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://epigenome.wustl.edu/epimap</a>) as controls. For studies based on older genome assemblies (<a id="__tag_1147980677" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE138293" ref="/geo/query/acc.cgi?acc=GSE138293"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE138293","term_id":"138293"}}</span>GSE138293</a>, <a id="__tag_1147980659" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE224241" ref="/geo/query/acc.cgi?acc=GSE224241"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE224241","term_id":"224241"}}</span>GSE224241</a>, <a id="__tag_1147980672" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE136279" ref="/geo/query/acc.cgi?acc=GSE136279"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE136279","term_id":"136279"}}</span>GSE136279</a>, <a id="__tag_1147980656" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE228832" ref="/geo/query/acc.cgi?acc=GSE228832"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE228832","term_id":"228832"}}</span>GSE228832</a>, and EpiMap used hg19), we converted our peak coordinates to hg19 using the <em>liftOver</em> R package (v1.18.0). We then used the <em>GenomicRanges</em><sup><sup><a href="#CR153" rid="CR153" class=" bibr popnode">153</a></sup></sup> (v1.46.1) and <em>plyranges</em><sup><sup><a href="#CR161" rid="CR161" class=" bibr popnode">161</a></sup></sup> (v1.14.0) packages to identify genome segments overlapping our peaks and to aggregate the corresponding mean score reported in the coverage tracks, which were then used for gene set enrichment analysis via the <em>cb_fgsea</em> function (<em>canceRbits</em> v0.1.6; parameters: <em>max_size = Inf</em>).</p></div><div id="Sec63" class="sec"><p></p><h4 id="Sec63title" class="inline">Identification of transcription factor targets </h4><p id="Par89" class="p p-first-last">To identify putative target genes of TFs, we used <em>GRNboost2</em><sup><sup><a href="#CR81" rid="CR81" class=" bibr popnode">81</a></sup></sup> (<em>arboreto</em> library v0.1.6, with Python v3.8.17 via reticulate [v1.24]) to identify genes whose expression could be predicted from the expression of each TF. We tested all TFs in the <em>HOCOMOCO</em> database<sup><sup><a href="#CR154" rid="CR154" class=" bibr popnode">154</a></sup></sup> for which at least one motif could be identified in our dataset. We found that stronger association values were reported for stem-cell-related factors, likely because of a proportional overrepresentation of this developmental stage in our dataset. To alleviate this effect and create more balanced data to build our networks on, we downsampled our dataset to no more than 500 cells per cluster and took the average importance value of ten random samples forward for further analysis. Putative targets with high importance values but without a supporting nearby ATAC-seq peak with a motif matching the respective TF were considered indirect targets and discarded from the target gene sets. We found that the range of importance values varied between TFs. We therefore calculated a TF-specific threshold on the importance score to define target genes. To this end, we ranked importance values and used the <em>changepoint</em> package (v2.2.3; default parameters) to identify the first point at which the mean values of the curve of importance values changed (disregarding the top 1% highest importance values which often were outliers and disrupted this analysis). The resulting target gene sets were divided into putative activating and inhibiting interactions by the sign of the Pearson correlation coefficient <em>r</em> of the respective TF-target pairs (using the mean correlation value of the same eight random samples as used for <em>GRNboost2</em>). Interactions with |<em>r</em>|&#x02009;&#x0003c;&#x02009;0.1 were discarded. To calculate the average expression of target genes in each cell we used only activated targets (<em>r</em>&#x02009;&#x0003e;&#x02009;0.1) and the Seurat module score. To identify significant overlaps between target genes and gene sets D9_1 &#x02013; D9_4 (Supplementary Data&#x000a0;<sup><a href="#MOESM19" rid="MOESM19" class=" media">15</a></sup>), we used the <em>hypeR</em><sup><sup><a href="#CR135" rid="CR135" class=" bibr popnode">135</a></sup></sup> package (v1.10.0) via the <em>cb_hyper</em> function (<em>canceRbits</em> v0.1.6; parameters: collapse = FALSE, min_size = 0, max_size = Inf), considering TFs with <em>P</em><sub>adj</sub>&#x02009;&#x02264;&#x02009;0.05, |log<sub>2</sub> odds|&#x02009;&#x02265;&#x02009;log<sub>2</sub>(4), and frequency &#x02265;5% as significant. All target gene sets are reported in Supplementary Data&#x000a0;<sup><a href="#MOESM19" rid="MOESM19" class=" media">15</a></sup> and all enrichment results in Supplementary Data&#x000a0;<sup><a href="#MOESM20" rid="MOESM20" class=" media">16</a></sup>.</p></div><div id="Sec64" class="sec sec-last"><p></p><h4 id="Sec64title" class="inline">Gene-regulatory network visualisation </h4><p id="Par90" class="p p-first-last">For the visualisation of gene-regulatory networks, we used the <em>igraph</em> package (v1.3.1). A directed graph was constructed from edges between genes in the gene sets D9_1, D9_2, D9_3, or D9_4 (Supplementary Data&#x000a0;<sup><a href="#MOESM13" rid="MOESM13" class=" media">9</a></sup>) and TFs found enriched in the overlap with these genes (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig10/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig10" rid-ob="ob-Fig10" co-legend-rid="lgnd_Fig10"><span>10d</span></a>). The same automated graph layout (function <em>layout_with_fr()</em>) was used to draw mutant-specific network diagrams. To generate mutant-specific networks (Fig.&#x000a0;<a href="/pmc/articles/PMC11068915/figure/Fig10/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig10" rid-ob="ob-Fig10" co-legend-rid="lgnd_Fig10"><span>10f</span></a>), we selected cells derived at D9 and parameterised node colour to indicate the mean scaled expression of the genes in those cells and node size to indicate the mean scaled TF target score (Seurat module score) for TFs or the mean scaled expression for non-TFs. To simplify plots, we only labelled TFs with positive mean scaled expression values (&#x0003e;0.05) and manually aggregated many overlapping values, but all node labels are shown in Supplementary Fig. <sup><a href="#MOESM2" rid="MOESM2" class=" media">13c</a></sup>.</p></div></div><div id="Sec65" class="sec sec-last"><h3 id="Sec65title">Reporting summary</h3><p id="Par91" class="p p-first-last">Further information on research design is available in the&#x000a0;<sup><a href="#MOESM21" rid="MOESM21" class=" media">Nature Portfolio Reporting Summary</a></sup> linked to this article.</p></div></div><div id="sec-a.ai.e" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="headless nomenu"></h2><!--/article/body/sec/--><div id="Sec67" class="sec sec-first-last"><h3 id="Sec67title">Source data</h3><p class="p p-first-last">
</p><div class="sec suppmat" id="MOESM1"><div class="sup-box half_rhythm" id="media-a.ai.e.a.b.a.a"><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl">Source Data</a><sup>(381M, xlsx)</sup></div></div><p>
</p><div class="sec suppmat" id="MOESM2"><div class="sup-box half_rhythm" id="media-a.ai.e.a.b.b.a"><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl">Supplementary Information</a><sup>(4.7M, pdf)</sup></div></div><p>
</p><div class="sec suppmat" id="MOESM3"><div class="sup-box half_rhythm" id="media-a.ai.e.a.b.c.a"><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl">Peer Review File</a><sup>(5.7M, pdf)</sup></div></div><p>
</p><div class="sec suppmat" id="MOESM4"><div class="sup-box half_rhythm" id="media-a.ai.e.a.b.d.a"><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl">Description of Additional Supplementary Files</a><sup>(84K, pdf)</sup></div></div><p>
</p><div class="sec suppmat" id="MOESM5"><div class="sup-box half_rhythm" id="media-a.ai.e.a.b.e.a"><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl">Supplementary Data 1</a><sup>(65K, xlsx)</sup></div></div><p>
</p><div class="sec suppmat" id="MOESM6"><div class="sup-box half_rhythm" id="media-a.ai.e.a.b.f.a"><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl">Supplementary Data 2</a><sup>(668K, xlsx)</sup></div></div><p>
</p><div class="sec suppmat" id="MOESM7"><div class="sup-box half_rhythm" id="media-a.ai.e.a.b.g.a"><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl">Supplementary Data 3</a><sup>(87K, xlsx)</sup></div></div><p>
</p><div class="sec suppmat" id="MOESM8"><div class="sup-box half_rhythm" id="media-a.ai.e.a.b.h.a"><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl">Supplementary Data 4</a><sup>(1.4M, xlsx)</sup></div></div><p>
</p><div class="sec suppmat" id="MOESM9"><div class="sup-box half_rhythm" id="media-a.ai.e.a.b.i.a"><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl">Supplementary Data 5</a><sup>(56K, xlsx)</sup></div></div><p>
</p><div class="sec suppmat" id="MOESM10"><div class="sup-box half_rhythm" id="media-a.ai.e.a.b.j.a"><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl">Supplementary Data 6</a><sup>(120K, xlsx)</sup></div></div><p>
</p><div class="sec suppmat" id="MOESM11"><div class="sup-box half_rhythm" id="media-a.ai.e.a.b.k.a"><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM11_ESM.xlsx" data-ga-action="click_feat_suppl">Supplementary Data 7</a><sup>(68K, xlsx)</sup></div></div><p>
</p><div class="sec suppmat" id="MOESM12"><div class="sup-box half_rhythm" id="media-a.ai.e.a.b.l.a"><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM12_ESM.xlsx" data-ga-action="click_feat_suppl">Supplementary Data 8</a><sup>(324K, xlsx)</sup></div></div><p>
</p><div class="sec suppmat" id="MOESM13"><div class="sup-box half_rhythm" id="media-a.ai.e.a.b.m.a"><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM13_ESM.xlsx" data-ga-action="click_feat_suppl">Supplementary Data 9</a><sup>(192K, xlsx)</sup></div></div><p>
</p><div class="sec suppmat" id="MOESM14"><div class="sup-box half_rhythm" id="media-a.ai.e.a.b.n.a"><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM14_ESM.xlsx" data-ga-action="click_feat_suppl">Supplementary Data 10</a><sup>(92K, xlsx)</sup></div></div><p>
</p><div class="sec suppmat" id="MOESM15"><div class="sup-box half_rhythm" id="media-a.ai.e.a.b.o.a"><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM15_ESM.xlsx" data-ga-action="click_feat_suppl">Supplementary Data 11</a><sup>(55K, xlsx)</sup></div></div><p>
</p><div class="sec suppmat" id="MOESM16"><div class="sup-box half_rhythm" id="media-a.ai.e.a.b.p.a"><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM16_ESM.xlsx" data-ga-action="click_feat_suppl">Supplementary Data 12</a><sup>(20M, xlsx)</sup></div></div><p>
</p><div class="sec suppmat" id="MOESM17"><div class="sup-box half_rhythm" id="media-a.ai.e.a.b.q.a"><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM17_ESM.xlsx" data-ga-action="click_feat_suppl">Supplementary Data 13</a><sup>(15M, xlsx)</sup></div></div><p>
</p><div class="sec suppmat" id="MOESM18"><div class="sup-box half_rhythm" id="media-a.ai.e.a.b.r.a"><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM18_ESM.xlsx" data-ga-action="click_feat_suppl">Supplementary Data 14</a><sup>(104K, xlsx)</sup></div></div><p>
</p><div class="sec suppmat" id="MOESM19"><div class="sup-box half_rhythm" id="media-a.ai.e.a.b.s.a"><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM19_ESM.xlsx" data-ga-action="click_feat_suppl">Supplementary Data 15</a><sup>(2.0M, xlsx)</sup></div></div><p>
</p><div class="sec suppmat" id="MOESM20"><div class="sup-box half_rhythm" id="media-a.ai.e.a.b.t.a"><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM20_ESM.xlsx" data-ga-action="click_feat_suppl">Supplementary Data 16</a><sup>(59K, xlsx)</sup></div></div><p>
</p><div class="sec suppmat" id="MOESM21"><div class="sup-box half_rhythm" id="media-a.ai.e.a.b.u.a"><a href="/pmc/articles/PMC11068915/bin/41467_2024_47945_MOESM21_ESM.pdf" data-ga-action="click_feat_suppl">Reporting summary</a><sup>(6.1M, pdf)</sup></div></div><p>
</p></div></div><div id="ack-a.aj.c" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.aj.ctitle">Acknowledgements</h2><div class="sec"><p>We would like to thank the Biomedical Sequencing Facility at the CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences for assistance with next-generation sequencing, Bettina Brunner-Herglotz (CCRI) for her technical assistance, and Duncan Baker (Sheffield Diagnostic Genetic Services, Sheffield Children&#x02019;s Hospital) for carrying out karyotyping of hESC lines. We would like to acknowledge Yann Jamin and Barbara Martins da Costa (ICR, London) for help with the MRI and animal work, respectively. We are also grateful to Igor Adameyko and Polina Kameneva (Medical University of Vienna), Sofie Mohlin (Lund University), and Christoph Bock (CeMM Research Center for Molecular Medicine) for critical reading of the manuscript. We would also like to thank Eszter S&#x000f6;jt&#x000f6;ry (CCRI) and Jan Koster (Amsterdam UMC) for their help with providing interfaces for exploratory data analysis for our data. For the purpose of open access, the authors have applied a Creative Commons Attribution (CC BY) licence to any Author Accepted Manuscript version arising. The following funding sources contributed directly or indirectly to the support of this study: Alex&#x02019;s Lemonade Stand Foundation for Childhood Cancer (ALSF) 20-17258 (M.D., M.F., F.H.); Austrian Academy of Sciences (OEAW) 25902 (M.C.B.); Austrian Research Promotion Agency (FFG) 7940628 533 (M.D.); Austrian Science Fund (FWF) 10.55776/P35072 (I.S.F.), 10.55776/P32001 (E.M.P.), 10.55776/P34832 (E.M.P.), 10.55776/P35841B (S.T.M.), 10.55776/TAI454 (F.H.), 10.55776/TAI732 (F.H.), 10.55776/PAT1300223 (F.H.); Biotechnology and Biological Sciences Research Council (BBSRC) BB/T007222/1 (K.B.), BB/P000444/1 (A.T.); Children&#x02019;s Cancer and Leukaemia Group/Little Princess Trust CCLGA 2018 06 (H.E.B.), CCLGA 2020 19 (H.E.B., A.T.); Neuroblastoma UK/Children&#x02019;s Cancer and Leukaemia Group/Little Princess Trust CCLGA 2019 28 (A.T.); CRUK Programme Award A28278 (E.P., L.C.); Donald E. and Delia B. Baxter Foundation Fellowship Award (M.H.); EC Horizon 2020 826494 (S.T.M.), 824070 (A.T.); Higher Education Funding Council for England (HEFCE) ICR (L.C.); Medical Research Council UK (MRC) MR/X000028/1 (I.B.), MR/X007979/1 (I.B.), MR/V002163/1 (A.T.); National Institutes of Health (NIH) R00CA197484 (M.H.); St. Anna Kinderkrebsforschung (E.M.P, R.L., G.C., M.D., S.T.M., F.H.); UK Regenerative Medicine Platform MR/R015724/1 (I.B.); Vienna Science and Technology Fund (WWTF) LS18-111 (S.T.M.).</p></div></div><div id="notes-a.aj.d" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.aj.dtitle">Author contributions</h2><p>I.S.G. and L.M.G. championed the experimental and computational work in this study, respectively. A.T. and F.H. jointly supervised this work. Furthermore, the following authors contributed equally to important aspects of this study: K.B., C.B., E.P., L.S., D.S., as well as L.C., S.T.M., and M.F. Formal contributions in authorship order (CrediT taxonomy): Conceptualisation: A.T., F.H.; Data curation: L.M.G., I.S.G., L.S., K.B., E.P., D.S., S.W.Z., E.B., I.S.F., P.Z., U.P., G.C., S.T.M., M.F., A.T., F.H.; Formal Analysis: L.M.G., I.S.G., C.H., E.P., S.W.Z., P.Z., U.P., C. Sturtzel., M.S., G.C., H.E.B., A.T., F.H.; Funding acquisition: K.B., I.S.F., M.C.B., W.W., P.W.A., I.B., H.E.B., M.D., L.C., S.T.M., M.F., A.T., F.H.; Investigation: I.S.G., L.M.G., K.B., E.P., L.S., D.S., R.A.L., E.B., I.S.F., M.B., A.W.W., C. Sturtzel, C. Souilhol, S.T., P.B., M.R., M.G., M.C.B.; Methodology: I.S.G., L.M.G., L.S., E.P., I.S.F., M.B., P.B., M.G., G.C., H.E.B., M.D., L.C., S.T.M., M.F., A.T., F.H.; Project administration: A.T., F.H.; Resources: W.W., E.M,P., M.H., R.L., H.E.B., M.D., L.C., S.T.M, M.F., A.T., F.H.; Software: L.M.G., C.H.; Supervision: E.M.P., G.C., H.E.B., M.D., L.C., S.T.M., M.F., A.T., F.H.; Visualisation: I.S.G., L.M.G., K.B., E.P., D.S., C. Sturtzel, G.C., A.T., F.H.; Writing &#x02013; original draft: I.S.G., L.M.G., A.T., F.H.; Writing &#x02013; review &#x00026; editing: I.S.G., L.M.G., K.B., C.H., E.P., L.S., I.S.F., C. Sturtzel, C. Souilhol, M.B., P.B., M.G., M.C.B., E.M.P., M.H., P.W.A., I.B., H.E.B., G.C., M.D., S.T.M., M.F., A.T., F.H.</p></div><div id="notes-a.aj.e" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.aj.etitle">Peer review</h2><div id="FPar1" class="sec sec-first"><h2 id="FPar1title">Peer review information</h2><p id="Par92" class="p p-first-last"><em>Nature Communications</em> thanks Anton Henssen, and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></div></div><div id="notes-a.aj.f" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.aj.ftitle">Data availability</h2><p>The single-cell RNA-seq and ATAC-seq data generated in this study have been deposited in the Gene Expression Omnibus (GEO) under accession code <a id="__tag_1147980671" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE219153" ref="/geo/query/acc.cgi?acc=GSE219153"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE219153","term_id":"219153"}}</span>GSE219153</a>. The public scRNA-seq data from NB tumours used in this study are available in GEO under the accession codes <a id="__tag_1147980663" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE147821" ref="/geo/query/acc.cgi?acc=GSE147821"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE147821","term_id":"147821"}}</span>GSE147821</a>, <a id="__tag_1147980679" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE216176" ref="/geo/query/acc.cgi?acc=GSE216176"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE216176","term_id":"216176"}}</span>GSE216176</a>, and <a id="__tag_1147980665" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE137804" ref="/geo/query/acc.cgi?acc=GSE137804"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE137804","term_id":"137804"}}</span>GSE137804</a>, and in the European Genome-Phenome Archive (EGA) under accession code <a href="https://ega-archive.org/studies/EGAS00001004388" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">EGAS00001004388</a>. Public ATAC-seq data from NB cell lines and controls used in this study are available in GEO under accession codes <a id="__tag_1147980661" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE138293" ref="/geo/query/acc.cgi?acc=GSE138293"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE138293","term_id":"138293"}}</span>GSE138293</a>, <a id="__tag_1147980666" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE224241" ref="/geo/query/acc.cgi?acc=GSE224241"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE224241","term_id":"224241"}}</span>GSE224241</a>, <a id="__tag_1147980667" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE136279" ref="/geo/query/acc.cgi?acc=GSE136279"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE136279","term_id":"136279"}}</span>GSE136279</a>, <a id="__tag_1147980662" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE202511" ref="/geo/query/acc.cgi?acc=GSE202511"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE202511","term_id":"202511"}}</span>GSE202511</a>, and <a id="__tag_1147980675" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE228832" ref="/geo/query/acc.cgi?acc=GSE228832"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE228832","term_id":"228832"}}</span>GSE228832</a>, and from the EpiMap website (<a href="https://epigenome.wustl.edu/epimap" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://epigenome.wustl.edu/epimap</a>). Bulk RNA-seq data from NB tumours are available in GEO under accession codes <a id="__tag_1147980658" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE49711" ref="/geo/query/acc.cgi?acc=GSE49711"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE49711","term_id":"49711"}}</span>GSE49711</a>, <a id="__tag_1147980664" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE94035" ref="/geo/query/acc.cgi?acc=GSE94035"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE94035","term_id":"94035"}}</span>GSE94035</a>, <a id="__tag_1147980680" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE147635" ref="/geo/query/acc.cgi?acc=GSE147635"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE147635","term_id":"147635"}}</span>GSE147635</a>, and <a id="__tag_1147980654" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE172184" ref="/geo/query/acc.cgi?acc=GSE172184"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE172184","term_id":"172184"}}</span>GSE172184</a>, and in dbGAP under accession code phs000467 [<a href="https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000218" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000218</a>]. Source data are provided with this paper. Additionally, processed data from this paper can be accessed and browsed interactively via our GitHub page [<a href="https://github.com/cancerbits/saldana_montano2024_ncnb/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/cancerbits/saldana_montano2024_ncnb/</a>] and via the R2 Genomics Analysis and Visualisation Platform [<a href="http://r2platform.com/halbritter24/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://r2platform.com/halbritter24/</a>].&#x000a0;<a href="#Sec67" rid="Sec67" class=" sec">Source data</a> are provided with this paper.</p></div><div id="notes-a.aj.g" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.aj.gtitle">Code availability</h2><p>Computer code used for the data analysis in this study is available via our GitHub page (<a href="https://github.com/cancerbits/saldana_montano2024_ncnb/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/cancerbits/saldana_montano2024_ncnb/</a>) and a persistent copy of this repository is available via Zenodo (10.5281/zenodo.10891507).</p></div><div id="FPar2" class="tsec sec"><h2 class="head no_bottom_margin" id="FPar2title">Competing interests</h2><p id="Par93">P.W.A. is a member of the Scientific Advisory Board of TreeFrog Therapeutics and receives royalties from the Wistar Institute for antibody sales. All other authors declare no competing interests.</p></div><div id="fn-group-a.aj.a" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.aj.atitle">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="fn-a.aj.a.a"><p class="p p-first-last"><strong>Publisher&#x02019;s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></p><p class="fn sec" id="fn-a.aj.a.b"><p class="p p-first-last">These authors contributed equally: Ingrid M. Saldana-Guerrero, Luis F. Montano-Gutierrez.</p></p><p class="fn sec" id="fn-a.aj.a.c"><p class="p p-first-last">These authors jointly supervised this work: Anestis Tsakiridis, Florian Halbritter.</p></p></div></div><div id="article-aaff-info" class="tsec sec"><h2 class="head no_bottom_margin" id="article-aaff-infotitle">Contributor Information</h2><p><span class="fm-affl">Anestis Tsakiridis, </span><span class="fm-affl"><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ku.ca.dleiffehs@sidirikast.a" class="oemail">ku.ca.dleiffehs@sidirikast.a</a></span>.</p><p><span class="fm-affl">Florian Halbritter, </span><span class="fm-affl"><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ta.ircc@rettirblah.nairolf" class="oemail">ta.ircc@rettirblah.nairolf</a></span>.</p></div><div id="sec-a.aj.b" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="sec-a.aj.btitle">Supplementary information</h2><!--/article/back/sec/--><p class="p p-first-last">The online version contains supplementary material available at 10.1038/s41467-024-47945-7.</p></div><div id="Bib1" class="tsec sec"><h2 class="head no_bottom_margin" id="Bib1title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="CR1">1. <span class="element-citation">Marshall GM, et al. The prenatal origins of cancer. <span><span class="ref-journal">Nat. Rev. Cancer. </span>2014;<span class="ref-vol">14</span>:277–289. doi: 10.1038/nrc3679.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4041218/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24599217" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnrc3679" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Cancer&amp;title=The+prenatal+origins+of+cancer&amp;author=GM+Marshall&amp;volume=14&amp;publication_year=2014&amp;pages=277-289&amp;pmid=24599217&amp;doi=10.1038/nrc3679&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR2">2. <span class="element-citation">Pritchard-Jones K. Genetics of childhood cancer. <span><span class="ref-journal">Br. Med. Bull. </span>1996;<span class="ref-vol">52</span>:704–723. doi: 10.1093/oxfordjournals.bmb.a011578.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9039727" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Foxfordjournals.bmb.a011578" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+Med.+Bull.&amp;title=Genetics+of+childhood+cancer&amp;author=K+Pritchard-Jones&amp;volume=52&amp;publication_year=1996&amp;pages=704-723&amp;pmid=9039727&amp;doi=10.1093/oxfordjournals.bmb.a011578&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR3">3. <span class="element-citation">Scotting PJ, Walker DA, Perilongo G. Childhood solid tumours: a developmental disorder. <span><span class="ref-journal">Nat. Rev. Cancer. </span>2005;<span class="ref-vol">5</span>:481–488. doi: 10.1038/nrc1633.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15905853" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnrc1633" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Cancer&amp;title=Childhood+solid+tumours:+a+developmental+disorder&amp;author=PJ+Scotting&amp;author=DA+Walker&amp;author=G+Perilongo&amp;volume=5&amp;publication_year=2005&amp;pages=481-488&amp;pmid=15905853&amp;doi=10.1038/nrc1633&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR4">4. <span class="element-citation">Maris JM, Denny CT. Focus on embryonal malignancies. <span><span class="ref-journal">Cancer Cell. </span>2002;<span class="ref-vol">2</span>:447–450. doi: 10.1016/S1535-6108(02)00206-4.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12498713" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FS1535-6108(02)00206-4" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Cell&amp;title=Focus+on+embryonal+malignancies&amp;author=JM+Maris&amp;author=CT+Denny&amp;volume=2&amp;publication_year=2002&amp;pages=447-450&amp;pmid=12498713&amp;doi=10.1016/S1535-6108(02)00206-4&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR5">5. <span class="element-citation">Gr&#x000f6;bner SN, et al. The landscape of genomic alterations across childhood cancers. <span><span class="ref-journal">Nature. </span>2018;<span class="ref-vol">555</span>:321–327. doi: 10.1038/nature25480.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29489754" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnature25480" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+landscape+of+genomic+alterations+across+childhood+cancers&amp;author=SN+Gr&#x000f6;bner&amp;volume=555&amp;publication_year=2018&amp;pages=321-327&amp;pmid=29489754&amp;doi=10.1038/nature25480&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR6">6. <span class="element-citation">Ma X, et al. Pan-cancer genome and transcriptome analyses of 1699 paediatric leukaemias and solid tumours. <span><span class="ref-journal">Nature. </span>2018;<span class="ref-vol">555</span>:371–376. doi: 10.1038/nature25795.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5854542/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29489755" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnature25795" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Pan-cancer+genome+and+transcriptome+analyses+of+1699+paediatric+leukaemias+and+solid+tumours&amp;author=X+Ma&amp;volume=555&amp;publication_year=2018&amp;pages=371-376&amp;pmid=29489755&amp;doi=10.1038/nature25795&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR7">7. <span class="element-citation">Matthay KK, et al. Neuroblastoma. <span><span class="ref-journal">Nat. Rev. Dis. Primers. </span>2016;<span class="ref-vol">2</span>:16078. doi: 10.1038/nrdp.2016.78.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27830764" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnrdp.2016.78" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Dis.+Primers&amp;title=Neuroblastoma&amp;author=KK+Matthay&amp;volume=2&amp;publication_year=2016&amp;pages=16078&amp;pmid=27830764&amp;doi=10.1038/nrdp.2016.78&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR8">8. <span class="element-citation">Schulte JH, Eggert A. Neuroblastoma. <span><span class="ref-journal">Crit. Rev. Oncog. </span>2015;<span class="ref-vol">20</span>:245–270. doi: 10.1615/CritRevOncog.2015014033.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26349419" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1615%2FCritRevOncog.2015014033" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Crit.+Rev.+Oncog&amp;title=Neuroblastoma.&amp;author=JH+Schulte&amp;author=A+Eggert&amp;volume=20&amp;publication_year=2015&amp;pages=245-270&amp;pmid=26349419&amp;doi=10.1615/CritRevOncog.2015014033&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR9">9. <span class="element-citation">Zeineldin M, Patel AG, Dyer MA. Neuroblastoma: when differentiation goes awry. <span><span class="ref-journal">Neuron. </span>2022;<span class="ref-vol">110</span>:2916–2928. doi: 10.1016/j.neuron.2022.07.012.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9509448/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35985323" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.neuron.2022.07.012" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Neuroblastoma:+when+differentiation+goes+awry&amp;author=M+Zeineldin&amp;author=AG+Patel&amp;author=MA+Dyer&amp;volume=110&amp;publication_year=2022&amp;pages=2916-2928&amp;pmid=35985323&amp;doi=10.1016/j.neuron.2022.07.012&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR10">10. <span class="element-citation">Nakagawara A. Neural crest development and neuroblastoma: the genetic and biological link. <span><span class="ref-journal">Prog. Brain Res. </span>2004;<span class="ref-vol">146</span>:231–242. doi: 10.1016/S0079-6123(03)46015-9.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14699967" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FS0079-6123(03)46015-9" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Prog.+Brain+Res.&amp;title=Neural+crest+development+and+neuroblastoma:+the+genetic+and+biological+link&amp;author=A+Nakagawara&amp;volume=146&amp;publication_year=2004&amp;pages=231-242&amp;doi=10.1016/S0079-6123(03)46015-9&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR11">11. <span class="element-citation">Ponzoni M, et al. Recent advances in the developmental origin of neuroblastoma: an overview. <span><span class="ref-journal">J. Exp. Clin. Cancer Res. </span>2022;<span class="ref-vol">41</span>:1–28. doi: 10.1186/s13046-022-02281-w.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8915499/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35277192" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2Fs13046-022-02281-w" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Exp.+Clin.+Cancer+Res.&amp;title=Recent+advances+in+the+developmental+origin+of+neuroblastoma:+an+overview&amp;author=M+Ponzoni&amp;volume=41&amp;publication_year=2022&amp;pages=1-28&amp;pmid=34980222&amp;doi=10.1186/s13046-022-02281-w&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR12">12. <span class="element-citation">Weiss WA, Aldape K, Mohapatra G, Feuerstein BG, Bishop JM. Targeted expression of MYCN causes neuroblastoma in transgenic mice. <span><span class="ref-journal">EMBO J. </span>1997;<span class="ref-vol">16</span>:2985–2995. doi: 10.1093/emboj/16.11.2985.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1169917/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9214616" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Femboj%2F16.11.2985" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+J.&amp;title=Targeted+expression+of+MYCN+causes+neuroblastoma+in+transgenic+mice&amp;author=WA+Weiss&amp;author=K+Aldape&amp;author=G+Mohapatra&amp;author=BG+Feuerstein&amp;author=JM+Bishop&amp;volume=16&amp;publication_year=1997&amp;pages=2985-2995&amp;pmid=9214616&amp;doi=10.1093/emboj/16.11.2985&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR13">13. <span class="element-citation">Cohen MA, et al. Formation of Human Neuroblastoma In Mouse-human Neural Crest Chimeras. <span><span class="ref-journal">Cell Stem Cell. </span>2020;<span class="ref-vol">26</span>:579–592.e6. doi: 10.1016/j.stem.2020.02.001.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7663823/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32142683" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.stem.2020.02.001" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Stem+Cell&amp;title=Formation+of+Human+Neuroblastoma+In+Mouse-human+Neural+Crest+Chimeras&amp;author=MA+Cohen&amp;volume=26&amp;publication_year=2020&amp;pages=579-592.e6&amp;pmid=32142683&amp;doi=10.1016/j.stem.2020.02.001&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR14">14. <span class="element-citation">Olsen RR, et al. MYCN induces neuroblastoma in primary neural crest cells. <span><span class="ref-journal">Oncogene. </span>2017;<span class="ref-vol">36</span>:5075–5082. doi: 10.1038/onc.2017.128.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5582212/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28459463" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fonc.2017.128" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=MYCN+induces+neuroblastoma+in+primary+neural+crest+cells&amp;author=RR+Olsen&amp;volume=36&amp;publication_year=2017&amp;pages=5075-5082&amp;pmid=28459463&amp;doi=10.1038/onc.2017.128&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR15">15. <span class="element-citation">Jansky S, et al. Single-cell transcriptomic analyses provide insights into the developmental origins of neuroblastoma. <span><span class="ref-journal">Nat. Genet. </span>2021;<span class="ref-vol">53</span>:683–693. doi: 10.1038/s41588-021-00806-1.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33767450" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41588-021-00806-1" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Single-cell+transcriptomic+analyses+provide+insights+into+the+developmental+origins+of+neuroblastoma&amp;author=S+Jansky&amp;volume=53&amp;publication_year=2021&amp;pages=683-693&amp;pmid=33767450&amp;doi=10.1038/s41588-021-00806-1&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR16">16. <span class="element-citation">Kameneva P, et al. Single-cell transcriptomics of human embryos identifies multiple sympathoblast lineages with potential implications for neuroblastoma origin. <span><span class="ref-journal">Nat. Genet. </span>2021;<span class="ref-vol">53</span>:694–706. doi: 10.1038/s41588-021-00818-x.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7610777/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33833454" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41588-021-00818-x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Single-cell+transcriptomics+of+human+embryos+identifies+multiple+sympathoblast+lineages+with+potential+implications+for+neuroblastoma+origin&amp;author=P+Kameneva&amp;volume=53&amp;publication_year=2021&amp;pages=694-706&amp;pmid=33833454&amp;doi=10.1038/s41588-021-00818-x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR17">17. <span class="element-citation">Dong R, et al. Single-cell characterization of malignant phenotypes and developmental trajectories of adrenal neuroblastoma. <span><span class="ref-journal">Cancer Cell. </span>2020;<span class="ref-vol">38</span>:716–733.e6. doi: 10.1016/j.ccell.2020.08.014.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32946775" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.ccell.2020.08.014" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Cell&amp;title=Single-cell+characterization+of+malignant+phenotypes+and+developmental+trajectories+of+adrenal+neuroblastoma&amp;author=R+Dong&amp;volume=38&amp;publication_year=2020&amp;pages=716-733.e6&amp;pmid=32946775&amp;doi=10.1016/j.ccell.2020.08.014&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR18">18. <span class="element-citation">Hanemaaijer ES, et al. Single-cell atlas of developing murine adrenal gland reveals relation of Schwann cell precursor signature to neuroblastoma phenotype. <span><span class="ref-journal">Proc. Natl Acad. Sci. USA. </span>2021;<span class="ref-vol">118</span>:e2022350118. doi: 10.1073/pnas.2022350118.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7865168/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33500353" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.2022350118" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl+Acad.+Sci.+USA&amp;title=Single-cell+atlas+of+developing+murine+adrenal+gland+reveals+relation+of+Schwann+cell+precursor+signature+to+neuroblastoma+phenotype&amp;author=ES+Hanemaaijer&amp;volume=118&amp;publication_year=2021&amp;pages=e2022350118&amp;pmid=33500353&amp;doi=10.1073/pnas.2022350118&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR19">19. <span class="element-citation">Kildisiute G, et al. Tumor to normal single-cell mRNA comparisons reveal a pan-neuroblastoma cancer cell. <span><span class="ref-journal">Sci. Adv. </span>2021;<span class="ref-vol">7</span>:eabd3311. doi: 10.1126/sciadv.abd3311.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7864567/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33547074" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fsciadv.abd3311" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Adv.&amp;title=Tumor+to+normal+single-cell+mRNA+comparisons+reveal+a+pan-neuroblastoma+cancer+cell&amp;author=G+Kildisiute&amp;volume=7&amp;publication_year=2021&amp;pages=eabd3311&amp;pmid=33547074&amp;doi=10.1126/sciadv.abd3311&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR20">20. <span class="element-citation">de Preter K, et al. Human fetal neuroblast and neuroblastoma transcriptome analysis confirms neuroblast origin and highlights neuroblastoma candidate genes. <span><span class="ref-journal">Genome Biol. </span>2006;<span class="ref-vol">7</span>:1–17. doi: 10.1186/gb-2006-7-9-r84.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1794547/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16989664" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2Fgb-2006-7-9-r84" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol.&amp;title=Human+fetal+neuroblast+and+neuroblastoma+transcriptome+analysis+confirms+neuroblast+origin+and+highlights+neuroblastoma+candidate+genes&amp;author=K+de+Preter&amp;volume=7&amp;publication_year=2006&amp;pages=1-17&amp;doi=10.1186/gb-2006-7-9-r84&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR21">21. <span class="element-citation">Althoff K, et al. A Cre-conditional MYCN-driven neuroblastoma mouse model as an improved tool for preclinical studies. <span><span class="ref-journal">Oncogene. </span>2015;<span class="ref-vol">34</span>:3357–3368. doi: 10.1038/onc.2014.269.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4487199/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25174395" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fonc.2014.269" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=A+Cre-conditional+MYCN-driven+neuroblastoma+mouse+model+as+an+improved+tool+for+preclinical+studies&amp;author=K+Althoff&amp;volume=34&amp;publication_year=2015&amp;pages=3357-3368&amp;pmid=25174395&amp;doi=10.1038/onc.2014.269&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR22">22. <span class="element-citation">Molenaar JJ, et al. Sequencing of neuroblastoma identifies chromothripsis and defects in neuritogenesis genes. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">483</span>:589–593. doi: 10.1038/nature10910.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22367537" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnature10910" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Sequencing+of+neuroblastoma+identifies+chromothripsis+and+defects+in+neuritogenesis+genes&amp;author=JJ+Molenaar&amp;volume=483&amp;publication_year=2012&amp;pages=589-593&amp;pmid=22367537&amp;doi=10.1038/nature10910&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR23">23. <span class="element-citation">Fetahu IS, et al. Single-cell transcriptomics and epigenomics unravel the role of monocytes in neuroblastoma bone marrow metastasis. <span><span class="ref-journal">Nat. Commun. </span>2023;<span class="ref-vol">14</span>:3620. doi: 10.1038/s41467-023-39210-0.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10293285/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37365178" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41467-023-39210-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Single-cell+transcriptomics+and+epigenomics+unravel+the+role+of+monocytes+in+neuroblastoma+bone+marrow+metastasis&amp;author=IS+Fetahu&amp;volume=14&amp;publication_year=2023&amp;pages=3620&amp;pmid=37365178&amp;doi=10.1038/s41467-023-39210-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR24">24. <span class="element-citation">Pugh TJ, et al. The genetic landscape of high-risk neuroblastoma. <span><span class="ref-journal">Nat. Genet. </span>2013;<span class="ref-vol">45</span>:279–284. doi: 10.1038/ng.2529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3682833/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23334666" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fng.2529" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=The+genetic+landscape+of+high-risk+neuroblastoma&amp;author=TJ+Pugh&amp;volume=45&amp;publication_year=2013&amp;pages=279-284&amp;pmid=23334666&amp;doi=10.1038/ng.2529&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR25">25. <span class="element-citation">Bown N, et al. Gain of chromosome arm 17q and adverse outcome in patients with neuroblastoma. <span><span class="ref-journal">N Engl. J. Med. </span>1999;<span class="ref-vol">340</span>:1954–1961. doi: 10.1056/NEJM199906243402504.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10379019" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1056%2FNEJM199906243402504" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl.+J.+Med.&amp;title=Gain+of+chromosome+arm+17q+and+adverse+outcome+in+patients+with+neuroblastoma&amp;author=N+Bown&amp;volume=340&amp;publication_year=1999&amp;pages=1954-1961&amp;pmid=10379019&amp;doi=10.1056/NEJM199906243402504&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR26">26. <span class="element-citation">Gilbert F, et al. Human neuroblastomas and abnormalities of chromosomes 1 and 17. <span><span class="ref-journal">Cancer Res. </span>1984;<span class="ref-vol">44</span>:5444–5449.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/6488196" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res.&amp;title=Human+neuroblastomas+and+abnormalities+of+chromosomes+1+and+17&amp;author=F+Gilbert&amp;volume=44&amp;publication_year=1984&amp;pages=5444-5449&amp;pmid=6488196&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR27">27. <span class="element-citation">Lastowska M, et al. Comprehensive genetic and histopathologic study reveals three types of neuroblastoma tumors. <span><span class="ref-journal">J. Clin. Oncol. </span>2001;<span class="ref-vol">19</span>:3080–3090. doi: 10.1200/JCO.2001.19.12.3080.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11408505" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1200%2FJCO.2001.19.12.3080" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Oncol.&amp;title=Comprehensive+genetic+and+histopathologic+study+reveals+three+types+of+neuroblastoma+tumors&amp;author=M+Lastowska&amp;volume=19&amp;publication_year=2001&amp;pages=3080-3090&amp;pmid=11408505&amp;doi=10.1200/JCO.2001.19.12.3080&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR28">28. <span class="element-citation">Mazzocco K, et al. Genetic abnormalities in adolescents and young adults with neuroblastoma: a report from the Italian Neuroblastoma Group. <span><span class="ref-journal">Pediatr. Blood Cancer. </span>2015;<span class="ref-vol">62</span>:1725–1732. doi: 10.1002/pbc.25552.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25925003" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fpbc.25552" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pediatr.+Blood+Cancer&amp;title=Genetic+abnormalities+in+adolescents+and+young+adults+with+neuroblastoma:+a+report+from+the+Italian+Neuroblastoma+Group&amp;author=K+Mazzocco&amp;volume=62&amp;publication_year=2015&amp;pages=1725-1732&amp;pmid=25925003&amp;doi=10.1002/pbc.25552&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR29">29. <span class="element-citation">K&#x000f6;rber V, et al. Neuroblastoma arises in early fetal development and its evolutionary duration predicts outcome. <span><span class="ref-journal">Nat. Genet. </span>2023;<span class="ref-vol">55</span>:619–630. doi: 10.1038/s41588-023-01332-y.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10101850/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36973454" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41588-023-01332-y" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Neuroblastoma+arises+in+early+fetal+development+and+its+evolutionary+duration+predicts+outcome&amp;author=V+K&#x000f6;rber&amp;volume=55&amp;publication_year=2023&amp;pages=619-630&amp;pmid=36973454&amp;doi=10.1038/s41588-023-01332-y&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR30">30. <span class="element-citation">Bogen D, et al. The genetic tumor background is an important determinant for heterogeneous MYCN-amplified neuroblastoma. <span><span class="ref-journal">Int. J. Cancer. </span>2016;<span class="ref-vol">139</span>:153–163. doi: 10.1002/ijc.30050.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4949549/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26910568" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fijc.30050" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Cancer&amp;title=The+genetic+tumor+background+is+an+important+determinant+for+heterogeneous+MYCN-amplified+neuroblastoma&amp;author=D+Bogen&amp;volume=139&amp;publication_year=2016&amp;pages=153-163&amp;pmid=26910568&amp;doi=10.1002/ijc.30050&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR31">31. <span class="element-citation">O&#x02019;Neill S, et al. MYCN amplification and 17q in neuroblastoma: evidence for structural association. <span><span class="ref-journal">Genes Chromosomes Cancer. </span>2001;<span class="ref-vol">30</span>:87–90. doi: 10.1002/1098-2264(2000)9999:9999&#x0003c;::AID-GCC1055&#x0003e;3.0.CO;2-J.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11107180" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2F1098-2264(2000)9999%3A9999%3C%3A%3AAID-GCC1055%3E3.0.CO%3B2-J" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Chromosomes+Cancer&amp;title=MYCN+amplification+and+17q+in+neuroblastoma:+evidence+for+structural+association&amp;author=S+O&#x02019;Neill&amp;volume=30&amp;publication_year=2001&amp;pages=87-90&amp;pmid=11107180&amp;doi=10.1002/1098-2264(2000)9999:9999&#x0003c;::AID-GCC1055&#x0003e;3.0.CO;2-J&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR32">32. <span class="element-citation">Huang M, Weiss WA. Neuroblastoma and MYCN. <span><span class="ref-journal">Cold Spring Harb. Perspect. Med. </span>2013;<span class="ref-vol">3</span>:a014415. doi: 10.1101/cshperspect.a014415.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3784814/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24086065" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1101%2Fcshperspect.a014415" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cold+Spring+Harb.+Perspect.+Med.&amp;title=Neuroblastoma+and+MYCN&amp;author=M+Huang&amp;author=WA+Weiss&amp;volume=3&amp;publication_year=2013&amp;pages=a014415&amp;pmid=24086065&amp;doi=10.1101/cshperspect.a014415&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR33">33. <span class="element-citation">Schleiermacher G, et al. Accumulation of segmental alterations determines progression in neuroblastoma. <span><span class="ref-journal">J. Clin. Oncol. </span>2010;<span class="ref-vol">28</span>:3122–3130. doi: 10.1200/JCO.2009.26.7955.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20516441" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1200%2FJCO.2009.26.7955" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Oncol.&amp;title=Accumulation+of+segmental+alterations+determines+progression+in+neuroblastoma&amp;author=G+Schleiermacher&amp;volume=28&amp;publication_year=2010&amp;pages=3122-3130&amp;pmid=20516441&amp;doi=10.1200/JCO.2009.26.7955&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR34">34. <span class="element-citation">Schleiermacher G, et al. Segmental chromosomal alterations have prognostic impact in neuroblastoma: a report from the INRG project. <span><span class="ref-journal">Br. J. Cancer. </span>2012;<span class="ref-vol">107</span>:1418. doi: 10.1038/bjc.2012.375.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3494425/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22976801" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fbjc.2012.375" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Cancer&amp;title=Segmental+chromosomal+alterations+have+prognostic+impact+in+neuroblastoma:+a+report+from+the+INRG+project&amp;author=G+Schleiermacher&amp;volume=107&amp;publication_year=2012&amp;pages=1418&amp;pmid=22976801&amp;doi=10.1038/bjc.2012.375&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR35">35. <span class="element-citation">Frith TJR, et al. Human axial progenitors generate trunk neural crest cells in vitro. <span><span class="ref-journal">Elife. </span>2018;<span class="ref-vol">7</span>:e35786. doi: 10.7554/eLife.35786.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6101942/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30095409" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.7554%2FeLife.35786" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Elife&amp;title=Human+axial+progenitors+generate+trunk+neural+crest+cells+in+vitro&amp;author=TJR+Frith&amp;volume=7&amp;publication_year=2018&amp;pages=e35786&amp;pmid=30095409&amp;doi=10.7554/eLife.35786&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR36">36. <span class="element-citation">Frith TJR, Tsakiridis A. Efficient generation of trunk neural crest and sympathetic neurons from human pluripotent stem cells via a neuromesodermal axial progenitor intermediate. <span><span class="ref-journal">Curr. Protoc. Stem Cell Biol. </span>2019;<span class="ref-vol">49</span>:e81. doi: 10.1002/cpsc.81.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6520238/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30688409" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fcpsc.81" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Protoc.+Stem+Cell+Biol.&amp;title=Efficient+generation+of+trunk+neural+crest+and+sympathetic+neurons+from+human+pluripotent+stem+cells+via+a+neuromesodermal+axial+progenitor+intermediate&amp;author=TJR+Frith&amp;author=A+Tsakiridis&amp;volume=49&amp;publication_year=2019&amp;pages=e81&amp;pmid=30688409&amp;doi=10.1002/cpsc.81&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR37">37. <span class="element-citation">Wymeersch FJ, Wilson V, Tsakiridis A. Understanding axial progenitor biology in vivo and in vitro. <span><span class="ref-journal">Development. </span>2021;<span class="ref-vol">148</span>:dev180612. doi: 10.1242/dev.180612.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33593754" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1242%2Fdev.180612" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Understanding+axial+progenitor+biology+in+vivo+and+in+vitro&amp;author=FJ+Wymeersch&amp;author=V+Wilson&amp;author=A+Tsakiridis&amp;volume=148&amp;publication_year=2021&amp;pages=dev180612&amp;pmid=33593754&amp;doi=10.1242/dev.180612&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR38">38. <span class="element-citation">Thomson JA, et al. Embryonic stem cell lines derived from human blastocysts. <span><span class="ref-journal">Science. </span>1998;<span class="ref-vol">282</span>:1145–1147. doi: 10.1126/science.282.5391.1145.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9804556" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.282.5391.1145" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Embryonic+stem+cell+lines+derived+from+human+blastocysts&amp;author=JA+Thomson&amp;volume=282&amp;publication_year=1998&amp;pages=1145-1147&amp;pmid=9804556&amp;doi=10.1126/science.282.5391.1145&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR39">39. <span class="element-citation">Soldatov R, et al. Spatiotemporal structure of cell fate decisions in murine neural crest. <span><span class="ref-journal">Science. </span>2019;<span class="ref-vol">364</span>:eaas9536. doi: 10.1126/science.aas9536.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31171666" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.aas9536" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Spatiotemporal+structure+of+cell+fate+decisions+in+murine+neural+crest&amp;author=R+Soldatov&amp;volume=364&amp;publication_year=2019&amp;pages=eaas9536&amp;pmid=31171666&amp;doi=10.1126/science.aas9536&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR40">40. <span class="element-citation">Kastriti ME, et al. Schwann cell precursors represent a neural crest-like state with biased multipotency. <span><span class="ref-journal">EMBO J. </span>2022;<span class="ref-vol">41</span>:e108780. doi: 10.15252/embj.2021108780.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9434083/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35815410" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.15252%2Fembj.2021108780" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+J.&amp;title=Schwann+cell+precursors+represent+a+neural+crest-like+state+with+biased+multipotency&amp;author=ME+Kastriti&amp;volume=41&amp;publication_year=2022&amp;pages=e108780&amp;pmid=35815410&amp;doi=10.15252/embj.2021108780&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR41">41. <span class="element-citation">Pattyn A, Morin X, Cremer H, Goridis C, Brunet JF. Expression and interactions of the two closely related homeobox genes Phox2a and Phox2b during neurogenesis. <span><span class="ref-journal">Development. </span>1997;<span class="ref-vol">124</span>:4065–4075. doi: 10.1242/dev.124.20.4065.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9374403" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1242%2Fdev.124.20.4065" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Expression+and+interactions+of+the+two+closely+related+homeobox+genes+Phox2a+and+Phox2b+during+neurogenesis&amp;author=A+Pattyn&amp;author=X+Morin&amp;author=H+Cremer&amp;author=C+Goridis&amp;author=JF+Brunet&amp;volume=124&amp;publication_year=1997&amp;pages=4065-4075&amp;pmid=9374403&amp;doi=10.1242/dev.124.20.4065&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR42">42. <span class="element-citation">Xie M, et al. Schwann cell precursors contribute to skeletal formation during embryonic development in mice and zebrafish. <span><span class="ref-journal">Proc. Natl Acad. Sci. USA. </span>2019;<span class="ref-vol">116</span>:15068–15073. doi: 10.1073/pnas.1900038116.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6660740/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31285319" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.1900038116" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl+Acad.+Sci.+USA&amp;title=Schwann+cell+precursors+contribute+to+skeletal+formation+during+embryonic+development+in+mice+and+zebrafish&amp;author=M+Xie&amp;volume=116&amp;publication_year=2019&amp;pages=15068-15073&amp;pmid=31285319&amp;doi=10.1073/pnas.1900038116&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR43">43. <span class="element-citation">McGonnell IM, Graham A. Trunk neural crest has skeletogenic potential. <span><span class="ref-journal">Curr. Biol. </span>2002;<span class="ref-vol">12</span>:767–771. doi: 10.1016/S0960-9822(02)00818-7.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12007423" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FS0960-9822(02)00818-7" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Biol.&amp;title=Trunk+neural+crest+has+skeletogenic+potential&amp;author=IM+McGonnell&amp;author=A+Graham&amp;volume=12&amp;publication_year=2002&amp;pages=767-771&amp;pmid=12007423&amp;doi=10.1016/S0960-9822(02)00818-7&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR44">44. <span class="element-citation">Boeva V, et al. Heterogeneity of neuroblastoma cell identity defined by transcriptional circuitries. <span><span class="ref-journal">Nat. Genet. </span>2017;<span class="ref-vol">49</span>:1408–1413. doi: 10.1038/ng.3921.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28740262" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fng.3921" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Heterogeneity+of+neuroblastoma+cell+identity+defined+by+transcriptional+circuitries&amp;author=V+Boeva&amp;volume=49&amp;publication_year=2017&amp;pages=1408-1413&amp;pmid=28740262&amp;doi=10.1038/ng.3921&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR45">45. <span class="element-citation">Van Groningen T, et al. Neuroblastoma is composed of two super-enhancer-associated differentiation states. <span><span class="ref-journal">Nat. Genet. </span>2017;<span class="ref-vol">49</span>:1261–1266. doi: 10.1038/ng.3899.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28650485" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fng.3899" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Neuroblastoma+is+composed+of+two+super-enhancer-associated+differentiation+states&amp;author=T+Van+Groningen&amp;volume=49&amp;publication_year=2017&amp;pages=1261-1266&amp;pmid=28650485&amp;doi=10.1038/ng.3899&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR46">46. <span class="element-citation">van Groningen T, et al. A NOTCH feed-forward loop drives reprogramming from adrenergic to mesenchymal state in neuroblastoma. <span><span class="ref-journal">Nat. Commun. </span>2019;<span class="ref-vol">10</span>:1530. doi: 10.1038/s41467-019-09470-w.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6449373/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30948783" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41467-019-09470-w" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=A+NOTCH+feed-forward+loop+drives+reprogramming+from+adrenergic+to+mesenchymal+state+in+neuroblastoma&amp;author=T+van+Groningen&amp;volume=10&amp;publication_year=2019&amp;pages=1530&amp;pmid=30948783&amp;doi=10.1038/s41467-019-09470-w&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR47">47. <span class="element-citation">Ciccarone V, Spengler BA, Meyers MB, Biedler JL, Ross RA. Phenotypic diversification in human neuroblastoma cells: expression of distinct neural crest lineages. <span><span class="ref-journal">Cancer Res. </span>1989;<span class="ref-vol">49</span>:219–225.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2535691" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res.&amp;title=Phenotypic+diversification+in+human+neuroblastoma+cells:+expression+of+distinct+neural+crest+lineages&amp;author=V+Ciccarone&amp;author=BA+Spengler&amp;author=MB+Meyers&amp;author=JL+Biedler&amp;author=RA+Ross&amp;volume=49&amp;publication_year=1989&amp;pages=219-225&amp;pmid=2535691&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR48">48. <span class="element-citation">Biedler JL, Helson L, Spengler BA. Morphology and growth, tumorigenicity, and cytogenetics of human neuroblastoma cells in continuous culture. <span><span class="ref-journal">Cancer Res. </span>1973;<span class="ref-vol">33</span>:2643–2652.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/4748425" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res.&amp;title=Morphology+and+growth,+tumorigenicity,+and+cytogenetics+of+human+neuroblastoma+cells+in+continuous+culture&amp;author=JL+Biedler&amp;author=L+Helson&amp;author=BA+Spengler&amp;volume=33&amp;publication_year=1973&amp;pages=2643-2652&amp;pmid=4748425&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR49">49. <span class="element-citation">Halliwell J, Barbaric I, Andrews PW. Acquired genetic changes in human pluripotent stem cells: origins and consequences. <span><span class="ref-journal">Nat. Rev. Mol. Cell Biol. </span>2020;<span class="ref-vol">21</span>:715–728. doi: 10.1038/s41580-020-00292-z.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32968234" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41580-020-00292-z" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Mol.+Cell+Biol.&amp;title=Acquired+genetic+changes+in+human+pluripotent+stem+cells:+origins+and+consequences&amp;author=J+Halliwell&amp;author=I+Barbaric&amp;author=PW+Andrews&amp;volume=21&amp;publication_year=2020&amp;pages=715-728&amp;pmid=32968234&amp;doi=10.1038/s41580-020-00292-z&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR50">50. <span class="element-citation">Draper JS, et al. Recurrent gain of chromosomes 17q and 12 in cultured human embryonic stem cells. <span><span class="ref-journal">Nat. Biotechnol. </span>2004;<span class="ref-vol">22</span>:53–54. doi: 10.1038/nbt922.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14661028" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnbt922" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=Recurrent+gain+of+chromosomes+17q+and+12+in+cultured+human+embryonic+stem+cells&amp;author=JS+Draper&amp;volume=22&amp;publication_year=2004&amp;pages=53-54&amp;pmid=14661028&amp;doi=10.1038/nbt922&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR51">51. <span class="element-citation">Abbasi MR, et al. Impact of disseminated neuroblastoma cells on the identification of the relapse-seeding clone. <span><span class="ref-journal">Clin. Cancer Res. </span>2017;<span class="ref-vol">23</span>:4224–4232. doi: 10.1158/1078-0432.CCR-16-2082.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5528137/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28228384" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1158%2F1078-0432.CCR-16-2082" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Cancer+Res.&amp;title=Impact+of+disseminated+neuroblastoma+cells+on+the+identification+of+the+relapse-seeding+clone&amp;author=MR+Abbasi&amp;volume=23&amp;publication_year=2017&amp;pages=4224-4232&amp;pmid=28228384&amp;doi=10.1158/1078-0432.CCR-16-2082&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR52">52. <span class="element-citation">Kerosuo L, et al. Enhanced expression of MycN/CIP2A drives neural crest toward a neural stem cell-like fate: Implications for priming of neuroblastoma. <span><span class="ref-journal">Proc. Natl Acad. Sci. USA. </span>2018;<span class="ref-vol">115</span>:E7351–E7360. doi: 10.1073/pnas.1800039115.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6077707/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30021854" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.1800039115" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl+Acad.+Sci.+USA&amp;title=Enhanced+expression+of+MycN/CIP2A+drives+neural+crest+toward+a+neural+stem+cell-like+fate:+Implications+for+priming+of+neuroblastoma&amp;author=L+Kerosuo&amp;volume=115&amp;publication_year=2018&amp;pages=E7351-E7360&amp;pmid=30021854&amp;doi=10.1073/pnas.1800039115&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR53">53. <span class="element-citation">Girish V, et al. Oncogene-like addiction to aneuploidy in human cancers. <span><span class="ref-journal">Science. </span>2023;<span class="ref-vol">381</span>:eadg4521. doi: 10.1126/science.adg4521.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10753973/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37410869" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.adg4521" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Oncogene-like+addiction+to+aneuploidy+in+human+cancers&amp;author=V+Girish&amp;volume=381&amp;publication_year=2023&amp;pages=eadg4521&amp;pmid=37410869&amp;doi=10.1126/science.adg4521&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR54">54. <span class="element-citation">Mitchell PJ, Timmons PM, H&#x000e9;bert JM, Rigby PWJ, Tjian R. Transcription factor AP-2 is expressed in neural crest cell lineages during mouse embryogenesis. <span><span class="ref-journal">Genes Dev. </span>1991;<span class="ref-vol">5</span>:105–119. doi: 10.1101/gad.5.1.105.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/1989904" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1101%2Fgad.5.1.105" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev.&amp;title=Transcription+factor+AP-2+is+expressed+in+neural+crest+cell+lineages+during+mouse+embryogenesis&amp;author=PJ+Mitchell&amp;author=PM+Timmons&amp;author=JM+H&#x000e9;bert&amp;author=PWJ+Rigby&amp;author=R+Tjian&amp;volume=5&amp;publication_year=1991&amp;pages=105-119&amp;pmid=1989904&amp;doi=10.1101/gad.5.1.105&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR55">55. <span class="element-citation">Rothstein M, Simoes-Costa M. Heterodimerization of TFAP2 pioneer factors drives epigenomic remodeling during neural crest specification. <span><span class="ref-journal">Genome Res. </span>2020;<span class="ref-vol">30</span>:35–48. doi: 10.1101/gr.249680.119.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6961570/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31848212" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1101%2Fgr.249680.119" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=Heterodimerization+of+TFAP2+pioneer+factors+drives+epigenomic+remodeling+during+neural+crest+specification&amp;author=M+Rothstein&amp;author=M+Simoes-Costa&amp;volume=30&amp;publication_year=2020&amp;pages=35-48&amp;pmid=31848212&amp;doi=10.1101/gr.249680.119&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR56">56. <span class="element-citation">Howard AGA, Uribe RA. Hox proteins as regulators of extracellular matrix interactions during neural crest migration. <span><span class="ref-journal">Differentiation. </span>2022;<span class="ref-vol">128</span>:26–32. doi: 10.1016/j.diff.2022.09.003.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10802151/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36228422" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.diff.2022.09.003" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Differentiation&amp;title=Hox+proteins+as+regulators+of+extracellular+matrix+interactions+during+neural+crest+migration&amp;author=AGA+Howard&amp;author=RA+Uribe&amp;volume=128&amp;publication_year=2022&amp;pages=26-32&amp;pmid=36228422&amp;doi=10.1016/j.diff.2022.09.003&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR57">57. <span class="element-citation">Akkermans O, et al. GPC3-Unc5 receptor complex structure and role in cell migration. <span><span class="ref-journal">Cell. </span>2022;<span class="ref-vol">185</span>:3931–3949.e26. doi: 10.1016/j.cell.2022.09.025.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9596381/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36240740" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cell.2022.09.025" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=GPC3-Unc5+receptor+complex+structure+and+role+in+cell+migration&amp;author=O+Akkermans&amp;volume=185&amp;publication_year=2022&amp;pages=3931-3949.e26&amp;pmid=36240740&amp;doi=10.1016/j.cell.2022.09.025&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR58">58. <span class="element-citation">TeSlaa JJ, Keller AN, Nyholm MK, Grinblat Y. Zebrafish Zic2a and Zic2b regulate neural crest and craniofacial development. <span><span class="ref-journal">Dev. Biol. </span>2013;<span class="ref-vol">380</span>:73–86. doi: 10.1016/j.ydbio.2013.04.033.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3886643/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23665173" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.ydbio.2013.04.033" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev.+Biol.&amp;title=Zebrafish+Zic2a+and+Zic2b+regulate+neural+crest+and+craniofacial+development&amp;author=JJ+TeSlaa&amp;author=AN+Keller&amp;author=MK+Nyholm&amp;author=Y+Grinblat&amp;volume=380&amp;publication_year=2013&amp;pages=73-86&amp;pmid=23665173&amp;doi=10.1016/j.ydbio.2013.04.033&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR59">59. <span class="element-citation">Handel AE, et al. Developmental dynamics of the neural crest&#x02013;mesenchymal axis in creating the thymic microenvironment. <span><span class="ref-journal">Sci. Adv. </span>2022;<span class="ref-vol">8</span>:9844. doi: 10.1126/sciadv.abm9844.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9106291/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35559672" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fsciadv.abm9844" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Adv.&amp;title=Developmental+dynamics+of+the+neural+crest&#x02013;mesenchymal+axis+in+creating+the+thymic+microenvironment&amp;author=AE+Handel&amp;volume=8&amp;publication_year=2022&amp;pages=9844&amp;doi=10.1126/sciadv.abm9844&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR60">60. <span class="element-citation">Rada-Iglesias A, et al. Epigenomic annotation of enhancers predicts transcriptional regulators of human neural crest. <span><span class="ref-journal">Cell Stem Cell. </span>2012;<span class="ref-vol">11</span>:633–648. doi: 10.1016/j.stem.2012.07.006.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3751405/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22981823" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.stem.2012.07.006" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Stem+Cell&amp;title=Epigenomic+annotation+of+enhancers+predicts+transcriptional+regulators+of+human+neural+crest&amp;author=A+Rada-Iglesias&amp;volume=11&amp;publication_year=2012&amp;pages=633-648&amp;pmid=22981823&amp;doi=10.1016/j.stem.2012.07.006&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR61">61. <span class="element-citation">Adam K, Lesperance J, Hunter T, Zage PE. The potential functional roles of NME1 histidine kinase activity in neuroblastoma pathogenesis. <span><span class="ref-journal">Int. J. Mol. Sci. </span>2020;<span class="ref-vol">21</span>:3319. doi: 10.3390/ijms21093319.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7247550/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32392889" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fijms21093319" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Mol.+Sci.&amp;title=The+potential+functional+roles+of+NME1+histidine+kinase+activity+in+neuroblastoma+pathogenesis&amp;author=K+Adam&amp;author=J+Lesperance&amp;author=T+Hunter&amp;author=PE+Zage&amp;volume=21&amp;publication_year=2020&amp;pages=3319&amp;pmid=32392889&amp;doi=10.3390/ijms21093319&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR62">62. <span class="element-citation">Ryl T, et al. Cell-cycle position of single MYC-driven cancer cells dictates their susceptibility to a chemotherapeutic drug. <span><span class="ref-journal">Cell Syst. </span>2017;<span class="ref-vol">5</span>:237–250.e8. doi: 10.1016/j.cels.2017.07.005.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28843484" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cels.2017.07.005" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Syst.&amp;title=Cell-cycle+position+of+single+MYC-driven+cancer+cells+dictates+their+susceptibility+to+a+chemotherapeutic+drug&amp;author=T+Ryl&amp;volume=5&amp;publication_year=2017&amp;pages=237-250.e8&amp;pmid=28843484&amp;doi=10.1016/j.cels.2017.07.005&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR63">63. <span class="element-citation">Lutz W, et al. Conditional expression of N-myc in human neuroblastoma cells increases expression of alpha-prothymosin and ornithine decarboxylase and accelerates progression into S-phase early after mitogenic stimulation of quiescent cells. <span><span class="ref-journal">Oncogene. </span>1996;<span class="ref-vol">13</span>:803–812.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8761302" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Conditional+expression+of+N-myc+in+human+neuroblastoma+cells+increases+expression+of+alpha-prothymosin+and+ornithine+decarboxylase+and+accelerates+progression+into+S-phase+early+after+mitogenic+stimulation+of+quiescent+cells&amp;author=W+Lutz&amp;volume=13&amp;publication_year=1996&amp;pages=803-812&amp;pmid=8761302&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR64">64. <span class="element-citation">King D, et al. MYCN expression induces replication stress and sensitivity to PARP inhibition in neuroblastoma. <span><span class="ref-journal">Oncotarget. </span>2020;<span class="ref-vol">11</span>:2141. doi: 10.18632/oncotarget.27329.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7289530/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32577161" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.18632%2Foncotarget.27329" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=MYCN+expression+induces+replication+stress+and+sensitivity+to+PARP+inhibition+in+neuroblastoma&amp;author=D+King&amp;volume=11&amp;publication_year=2020&amp;pages=2141&amp;pmid=32577161&amp;doi=10.18632/oncotarget.27329&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR65">65. <span class="element-citation">Kramer M, Ribeiro D, Arsenian-Henriksson M, Deller T, Rohrer H. Proliferation and survival of embryonic sympathetic neuroblasts by MYCN and activated ALK signaling. <span><span class="ref-journal">J. Neurosci. </span>2016;<span class="ref-vol">36</span>:10425–10439. doi: 10.1523/JNEUROSCI.0183-16.2016.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6705582/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27707976" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1523%2FJNEUROSCI.0183-16.2016" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Neurosci.&amp;title=Proliferation+and+survival+of+embryonic+sympathetic+neuroblasts+by+MYCN+and+activated+ALK+signaling&amp;author=M+Kramer&amp;author=D+Ribeiro&amp;author=M+Arsenian-Henriksson&amp;author=T+Deller&amp;author=H+Rohrer&amp;volume=36&amp;publication_year=2016&amp;pages=10425-10439&amp;pmid=27707976&amp;doi=10.1523/JNEUROSCI.0183-16.2016&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR66">66. <span class="element-citation">Mobley BC, et al. Expression of MYCN in multipotent sympathoadrenal progenitors induces proliferation and neural differentiation, but is not sufficient for tumorigenesis. <span><span class="ref-journal">PLoS One. </span>2015;<span class="ref-vol">10</span>:e0133897. doi: 10.1371/journal.pone.0133897.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4519318/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26222553" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pone.0133897" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Expression+of+MYCN+in+multipotent+sympathoadrenal+progenitors+induces+proliferation+and+neural+differentiation,+but+is+not+sufficient+for+tumorigenesis&amp;author=BC+Mobley&amp;volume=10&amp;publication_year=2015&amp;pages=e0133897&amp;pmid=26222553&amp;doi=10.1371/journal.pone.0133897&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR67">67. <span class="element-citation">Berry T, et al. The ALK(F1174L) mutation potentiates the oncogenic activity of MYCN in neuroblastoma. <span><span class="ref-journal">Cancer Cell. </span>2012;<span class="ref-vol">22</span>:117–130. doi: 10.1016/j.ccr.2012.06.001.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3417812/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22789543" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.ccr.2012.06.001" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Cell&amp;title=The+ALK(F1174L)+mutation+potentiates+the+oncogenic+activity+of+MYCN+in+neuroblastoma&amp;author=T+Berry&amp;volume=22&amp;publication_year=2012&amp;pages=117-130&amp;pmid=22789543&amp;doi=10.1016/j.ccr.2012.06.001&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR68">68. <span class="element-citation">Huang M, et al. Engineering genetic predisposition in human neuroepithelial stem cells recapitulates medulloblastoma tumorigenesis. <span><span class="ref-journal">Cell Stem Cell. </span>2019;<span class="ref-vol">25</span>:433–446.e7. doi: 10.1016/j.stem.2019.05.013.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6731167/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31204176" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.stem.2019.05.013" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Stem+Cell&amp;title=Engineering+genetic+predisposition+in+human+neuroepithelial+stem+cells+recapitulates+medulloblastoma+tumorigenesis&amp;author=M+Huang&amp;volume=25&amp;publication_year=2019&amp;pages=433-446.e7&amp;pmid=31204176&amp;doi=10.1016/j.stem.2019.05.013&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR69">69. <span class="element-citation">Wrobel JK, et al. Rapid in vivo validation of HDAC inhibitor-based treatments in neuroblastoma Zebrafish Xenografts. <span><span class="ref-journal">Pharmaceuticals. </span>2020;<span class="ref-vol">13</span>:345. doi: 10.3390/ph13110345.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7692187/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33121173" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fph13110345" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmaceuticals&amp;title=Rapid+in+vivo+validation+of+HDAC+inhibitor-based+treatments+in+neuroblastoma+Zebrafish+Xenografts&amp;author=JK+Wrobel&amp;volume=13&amp;publication_year=2020&amp;pages=345&amp;pmid=33121173&amp;doi=10.3390/ph13110345&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR70">70. <span class="element-citation">Su Z, et al. A comprehensive assessment of RNA-seq accuracy, reproducibility, and information content by the Sequencing Quality Control Consortium. <span><span class="ref-journal">Nat. Biotechnol. </span>2014;<span class="ref-vol">32</span>:903–914. doi: 10.1038/nbt.2957.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4321899/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25150838" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnbt.2957" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=A+comprehensive+assessment+of+RNA-seq+accuracy,+reproducibility,+and+information+content+by+the+Sequencing+Quality+Control+Consortium&amp;author=Z+Su&amp;volume=32&amp;publication_year=2014&amp;pages=903-914&amp;pmid=25150838&amp;doi=10.1038/nbt.2957&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR71">71. <span class="element-citation">Wang C, et al. The concordance between RNA-seq and microarray data depends on chemical treatment and transcript abundance. <span><span class="ref-journal">Nat. Biotechnol. </span>2014;<span class="ref-vol">32</span>:926–932. doi: 10.1038/nbt.3001.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4243706/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25150839" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnbt.3001" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=The+concordance+between+RNA-seq+and+microarray+data+depends+on+chemical+treatment+and+transcript+abundance&amp;author=C+Wang&amp;volume=32&amp;publication_year=2014&amp;pages=926-932&amp;pmid=25150839&amp;doi=10.1038/nbt.3001&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR72">72. <span class="element-citation">Durbin AD, et al. EP300 selectively controls the enhancer landscape of MYCN-Amplified Neuroblastoma. <span><span class="ref-journal">Cancer Discov. </span>2022;<span class="ref-vol">12</span>:730–751. doi: 10.1158/2159-8290.CD-21-0385.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8904277/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34772733" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1158%2F2159-8290.CD-21-0385" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Discov.&amp;title=EP300+selectively+controls+the+enhancer+landscape+of+MYCN-Amplified+Neuroblastoma&amp;author=AD+Durbin&amp;volume=12&amp;publication_year=2022&amp;pages=730-751&amp;pmid=34772733&amp;doi=10.1158/2159-8290.CD-21-0385&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR73">73. <span class="element-citation">Gartlgruber M, et al. Super enhancers define regulatory subtypes and cell identity in neuroblastoma. <span><span class="ref-journal">Nat. Cancer. </span>2021;<span class="ref-vol">2</span>:114–128. doi: 10.1038/s43018-020-00145-w.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35121888" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs43018-020-00145-w" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Cancer&amp;title=Super+enhancers+define+regulatory+subtypes+and+cell+identity+in+neuroblastoma&amp;author=M+Gartlgruber&amp;volume=2&amp;publication_year=2021&amp;pages=114-128&amp;pmid=35121888&amp;doi=10.1038/s43018-020-00145-w&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR74">74. <span class="element-citation">Zeid R, et al. Enhancer invasion shapes MYCN-dependent transcriptional amplification in neuroblastoma. <span><span class="ref-journal">Nat. Genet. </span>2018;<span class="ref-vol">50</span>:515–523. doi: 10.1038/s41588-018-0044-9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6310397/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29379199" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41588-018-0044-9" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Enhancer+invasion+shapes+MYCN-dependent+transcriptional+amplification+in+neuroblastoma&amp;author=R+Zeid&amp;volume=50&amp;publication_year=2018&amp;pages=515-523&amp;pmid=29379199&amp;doi=10.1038/s41588-018-0044-9&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR75">75. <span class="element-citation">Jahangiri L, et al. Core regulatory circuitries in defining cancer cell identity across the malignant spectrum. <span><span class="ref-journal">Open Biol. </span>2020;<span class="ref-vol">10</span>:200121. doi: 10.1098/rsob.200121.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7574545/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32634370" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1098%2Frsob.200121" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Open+Biol.&amp;title=Core+regulatory+circuitries+in+defining+cancer+cell+identity+across+the+malignant+spectrum&amp;author=L+Jahangiri&amp;volume=10&amp;publication_year=2020&amp;pages=200121&amp;pmid=32634370&amp;doi=10.1098/rsob.200121&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR76">76. <span class="element-citation">Zimmerman MW, et al. Retinoic acid rewires the adrenergic core regulatory circuitry of childhood neuroblastoma. <span><span class="ref-journal">Sci. Adv. </span>2021;<span class="ref-vol">7</span>:eabe0834. doi: 10.1126/sciadv.abe0834.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8528416/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34669465" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fsciadv.abe0834" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Adv.&amp;title=Retinoic+acid+rewires+the+adrenergic+core+regulatory+circuitry+of+childhood+neuroblastoma&amp;author=MW+Zimmerman&amp;volume=7&amp;publication_year=2021&amp;pages=eabe0834&amp;pmid=34669465&amp;doi=10.1126/sciadv.abe0834&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR77">77. <span class="element-citation">Corces MR, et al. An improved ATAC-seq protocol reduces background and enables interrogation of frozen tissues. <span><span class="ref-journal">Nat Methods. </span>2017;<span class="ref-vol">14</span>:959–962. doi: 10.1038/nmeth.4396.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5623106/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28846090" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnmeth.4396" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=An+improved+ATAC-seq+protocol+reduces+background+and+enables+interrogation+of+frozen+tissues&amp;author=MR+Corces&amp;volume=14&amp;publication_year=2017&amp;pages=959-962&amp;pmid=28846090&amp;doi=10.1038/nmeth.4396&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR78">78. <span class="element-citation">Meuleman W, et al. Index and biological spectrum of human DNase I hypersensitive sites. <span><span class="ref-journal">Nature 2020 584:7820. </span>2020;<span class="ref-vol">584</span>:244–251.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7422677/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32728217" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+2020+584:7820&amp;title=Index+and+biological+spectrum+of+human+DNase+I+hypersensitive+sites&amp;author=W+Meuleman&amp;volume=584&amp;publication_year=2020&amp;pages=244-251&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR79">79. <span class="element-citation">Zhang K, et al. A single-cell atlas of chromatin accessibility in the human genome. <span><span class="ref-journal">Cell. </span>2021;<span class="ref-vol">184</span>:5985–6001.e19. doi: 10.1016/j.cell.2021.10.024.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8664161/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34774128" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cell.2021.10.024" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A+single-cell+atlas+of+chromatin+accessibility+in+the+human+genome&amp;author=K+Zhang&amp;volume=184&amp;publication_year=2021&amp;pages=5985-6001.e19&amp;pmid=34774128&amp;doi=10.1016/j.cell.2021.10.024&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR80">80. <span class="element-citation">Gao T, Qian J. EnhancerAtlas 2.0: an updated resource with enhancer annotation in 586 tissue/cell types across nine species. <span><span class="ref-journal">Nucleic Acids Res. </span>2020;<span class="ref-vol">48</span>:D58–D64.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7145677/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31740966" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=EnhancerAtlas+2.0:+an+updated+resource+with+enhancer+annotation+in+586+tissue/cell+types+across+nine+species&amp;author=T+Gao&amp;author=J+Qian&amp;volume=48&amp;publication_year=2020&amp;pages=D58-D64&amp;pmid=31740966&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR81">81. <span class="element-citation">Moerman T, et al. GRNBoost2 and Arboreto: efficient and scalable inference of gene regulatory networks. <span><span class="ref-journal">Bioinformatics. </span>2019;<span class="ref-vol">35</span>:2159–2161. doi: 10.1093/bioinformatics/bty916.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30445495" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbty916" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GRNBoost2+and+Arboreto:+efficient+and+scalable+inference+of+gene+regulatory+networks&amp;author=T+Moerman&amp;volume=35&amp;publication_year=2019&amp;pages=2159-2161&amp;pmid=30445495&amp;doi=10.1093/bioinformatics/bty916&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR82">82. <span class="element-citation">Moreno-Smith M, et al. Restoration of the molecular clock is tumor suppressive in neuroblastoma. <span><span class="ref-journal">Nat. Commun. </span>2021;<span class="ref-vol">12</span>:1–16. doi: 10.1038/s41467-021-24196-4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8238982/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34183658" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41467-021-24196-4" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Restoration+of+the+molecular+clock+is+tumor+suppressive+in+neuroblastoma&amp;author=M+Moreno-Smith&amp;volume=12&amp;publication_year=2021&amp;pages=1-16&amp;pmid=33397941&amp;doi=10.1038/s41467-021-24196-4&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR83">83. <span class="element-citation">Altman BJ, et al. MYC disrupts the circadian clock and metabolism in cancer cells. <span><span class="ref-journal">Cell Metab. </span>2015;<span class="ref-vol">22</span>:1009–1019. doi: 10.1016/j.cmet.2015.09.003.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4818967/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26387865" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cmet.2015.09.003" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Metab.&amp;title=MYC+disrupts+the+circadian+clock+and+metabolism+in+cancer+cells&amp;author=BJ+Altman&amp;volume=22&amp;publication_year=2015&amp;pages=1009-1019&amp;pmid=26387865&amp;doi=10.1016/j.cmet.2015.09.003&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR84">84. <span class="element-citation">Xue C, et al. MYCN promotes neuroblastoma malignancy by establishing a regulatory circuit with transcription factor AP4. <span><span class="ref-journal">Oncotarget. </span>2016;<span class="ref-vol">7</span>:54937–54951. doi: 10.18632/oncotarget.10709.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5342392/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27448979" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.18632%2Foncotarget.10709" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=MYCN+promotes+neuroblastoma+malignancy+by+establishing+a+regulatory+circuit+with+transcription+factor+AP4&amp;author=C+Xue&amp;volume=7&amp;publication_year=2016&amp;pages=54937-54951&amp;pmid=27448979&amp;doi=10.18632/oncotarget.10709&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR85">85. <span class="element-citation">Boboila S, et al. Transcription factor activating protein 4 is synthetically lethal and a master regulator of MYCN-amplified neuroblastoma. <span><span class="ref-journal">Oncogene. </span>2018;<span class="ref-vol">37</span>:5451–5465. doi: 10.1038/s41388-018-0326-9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6172192/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29880876" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41388-018-0326-9" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Transcription+factor+activating+protein+4+is+synthetically+lethal+and+a+master+regulator+of+MYCN-amplified+neuroblastoma&amp;author=S+Boboila&amp;volume=37&amp;publication_year=2018&amp;pages=5451-5465&amp;pmid=29880876&amp;doi=10.1038/s41388-018-0326-9&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR86">86. <span class="element-citation">Ana Paula Azambuja A, Paula Azambuja A, Simoes-Costa M. The connectome of neural crest enhancers reveals regulatory features of signaling systems. <span><span class="ref-journal">Dev. Cell. </span>2021;<span class="ref-vol">56</span>:1268–1282.e6. doi: 10.1016/j.devcel.2021.03.024.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8445387/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33852891" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.devcel.2021.03.024" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev.+Cell&amp;title=The+connectome+of+neural+crest+enhancers+reveals+regulatory+features+of+signaling+systems&amp;author=A+Ana+Paula+Azambuja&amp;author=A+Paula+Azambuja&amp;author=M+Simoes-Costa&amp;volume=56&amp;publication_year=2021&amp;pages=1268-1282.e6&amp;pmid=33852891&amp;doi=10.1016/j.devcel.2021.03.024&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR87">87. <span class="element-citation">Gundem G, et al. Clonal evolution during metastatic spread in high-risk neuroblastoma. <span><span class="ref-journal">Nat. Genet. </span>2023;<span class="ref-vol">55</span>:1022–1033. doi: 10.1038/s41588-023-01395-x.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37169874" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41588-023-01395-x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Clonal+evolution+during+metastatic+spread+in+high-risk+neuroblastoma&amp;author=G+Gundem&amp;volume=55&amp;publication_year=2023&amp;pages=1022-1033&amp;pmid=37169874&amp;doi=10.1038/s41588-023-01395-x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR88">88. <span class="element-citation">Merrill BJ, Gat U, DasGupta R, Fuchs E. Tcf3 and Lef1 regulate lineage differentiation of multipotent stem cells in skin. <span><span class="ref-journal">Genes Dev. </span>2001;<span class="ref-vol">15</span>:1688–1705. doi: 10.1101/gad.891401.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC312726/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11445543" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1101%2Fgad.891401" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev.&amp;title=Tcf3+and+Lef1+regulate+lineage+differentiation+of+multipotent+stem+cells+in+skin&amp;author=BJ+Merrill&amp;author=U+Gat&amp;author=R+DasGupta&amp;author=E+Fuchs&amp;volume=15&amp;publication_year=2001&amp;pages=1688-1705&amp;pmid=11445543&amp;doi=10.1101/gad.891401&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR89">89. <span class="element-citation">Gautier M, Thirant C, Delattre O, Janoueix-Lerosey I. Plasticity in neuroblastoma cell identity defines a noradrenergic-to-mesenchymal transition (Nmt) <span><span class="ref-journal">Cancers. </span>2021;<span class="ref-vol">13</span>:2904. doi: 10.3390/cancers13122904.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8230375/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34200747" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fcancers13122904" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancers&amp;title=Plasticity+in+neuroblastoma+cell+identity+defines+a+noradrenergic-to-mesenchymal+transition+(Nmt)&amp;author=M+Gautier&amp;author=C+Thirant&amp;author=O+Delattre&amp;author=I+Janoueix-Lerosey&amp;volume=13&amp;publication_year=2021&amp;pages=2904&amp;pmid=34200747&amp;doi=10.3390/cancers13122904&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR90">90. <span class="element-citation">Thirant C, et al. Reversible transitions between noradrenergic and mesenchymal tumor identities define cell plasticity in neuroblastoma. <span><span class="ref-journal">Nat. Commun. </span>2023;<span class="ref-vol">14</span>:2575. doi: 10.1038/s41467-023-38239-5.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10160107/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37142597" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41467-023-38239-5" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Reversible+transitions+between+noradrenergic+and+mesenchymal+tumor+identities+define+cell+plasticity+in+neuroblastoma&amp;author=C+Thirant&amp;volume=14&amp;publication_year=2023&amp;pages=2575&amp;pmid=37142597&amp;doi=10.1038/s41467-023-38239-5&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR91">91. <span class="element-citation">Vayani OR, et al. Adrenergic and mesenchymal signatures are identifiable in cell-free DNA and correlate with metastatic disease burden in children with neuroblastoma. <span><span class="ref-journal">Pediatr. Blood Cancer. </span>2024;<span class="ref-vol">71</span>:e30735. doi: 10.1002/pbc.30735.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10842006/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37859597" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fpbc.30735" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pediatr.+Blood+Cancer&amp;title=Adrenergic+and+mesenchymal+signatures+are+identifiable+in+cell-free+DNA+and+correlate+with+metastatic+disease+burden+in+children+with+neuroblastoma&amp;author=OR+Vayani&amp;volume=71&amp;publication_year=2024&amp;pages=e30735&amp;pmid=37859597&amp;doi=10.1002/pbc.30735&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR92">92. <span class="element-citation">Durbin AD, et al. Selective gene dependencies in MYCN-amplified neuroblastoma include the core transcriptional regulatory circuitry. <span><span class="ref-journal">Nature Genetics. </span>2018;<span class="ref-vol">50</span>:1240–1246. doi: 10.1038/s41588-018-0191-z.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6386470/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30127528" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41588-018-0191-z" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genetics&amp;title=Selective+gene+dependencies+in+MYCN-amplified+neuroblastoma+include+the+core+transcriptional+regulatory+circuitry&amp;author=AD+Durbin&amp;volume=50&amp;publication_year=2018&amp;pages=1240-1246&amp;pmid=30127528&amp;doi=10.1038/s41588-018-0191-z&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR93">93. <span class="element-citation">Rouhani FJ, et al. Substantial somatic genomic variation and selection for BCOR mutations in human induced pluripotent stem cells. <span><span class="ref-journal">Nature Genetics 2022 54:9. </span>2022;<span class="ref-vol">54</span>:1406–1416.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9470532/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35953586" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genetics+2022+54:9&amp;title=Substantial+somatic+genomic+variation+and+selection+for+BCOR+mutations+in+human+induced+pluripotent+stem+cells&amp;author=FJ+Rouhani&amp;volume=54&amp;publication_year=2022&amp;pages=1406-1416&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR94">94. <span class="element-citation">Puigdevall P, Jerber J, Danecek P, Castellano S, Kilpinen H. Somatic mutations alter the differentiation outcomes of iPSC-derived neurons. <span><span class="ref-journal">Cell Genomics. </span>2023;<span class="ref-vol">3</span>:100280. doi: 10.1016/j.xgen.2023.100280.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10112289/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37082143" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.xgen.2023.100280" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Genomics&amp;title=Somatic+mutations+alter+the+differentiation+outcomes+of+iPSC-derived+neurons&amp;author=P+Puigdevall&amp;author=J+Jerber&amp;author=P+Danecek&amp;author=S+Castellano&amp;author=H+Kilpinen&amp;volume=3&amp;publication_year=2023&amp;pages=100280&amp;pmid=37082143&amp;doi=10.1016/j.xgen.2023.100280&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR95">95. <span class="element-citation">Aschero R, et al. Recurrent Somatic Chromosomal Abnormalities in Relapsed Extraocular Retinoblastoma. <span><span class="ref-journal">Cancers. </span>2021;<span class="ref-vol">13</span>:673. doi: 10.3390/cancers13040673.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7915502/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33567541" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fcancers13040673" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancers&amp;title=Recurrent+Somatic+Chromosomal+Abnormalities+in+Relapsed+Extraocular+Retinoblastoma&amp;author=R+Aschero&amp;volume=13&amp;publication_year=2021&amp;pages=673&amp;pmid=33567541&amp;doi=10.3390/cancers13040673&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR96">96. <span class="element-citation">Lazic D, et al. Landscape of Bone Marrow Metastasis in Human Neuroblastoma Unraveled by Transcriptomics and Deep Multiplex Imaging. <span><span class="ref-journal">Cancers (Basel) </span>2021;<span class="ref-vol">13</span>:4311. doi: 10.3390/cancers13174311.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8431445/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34503120" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fcancers13174311" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancers+(Basel)&amp;title=Landscape+of+Bone+Marrow+Metastasis+in+Human+Neuroblastoma+Unraveled+by+Transcriptomics+and+Deep+Multiplex+Imaging&amp;author=D+Lazic&amp;volume=13&amp;publication_year=2021&amp;pages=4311&amp;pmid=34503120&amp;doi=10.3390/cancers13174311&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR97">97. <span class="element-citation">Verhoeven BM, et al. The immune cell atlas of human neuroblastoma. <span><span class="ref-journal">Cell Rep. Med. </span>2022;<span class="ref-vol">3</span>:100657. doi: 10.1016/j.xcrm.2022.100657.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9245004/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35688160" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.xcrm.2022.100657" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Rep.+Med.&amp;title=The+immune+cell+atlas+of+human+neuroblastoma&amp;author=BM+Verhoeven&amp;volume=3&amp;publication_year=2022&amp;pages=100657&amp;pmid=35688160&amp;doi=10.1016/j.xcrm.2022.100657&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR98">98. <span class="element-citation">Costa A, et al. Single-cell transcriptomics reveals shared immunosuppressive landscapes of mouse and human neuroblastoma. <span><span class="ref-journal">J. Immunother. Cancer. </span>2022;<span class="ref-vol">10</span>:e004807. doi: 10.1136/jitc-2022-004807.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9362821/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36054452" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1136%2Fjitc-2022-004807" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Immunother.+Cancer&amp;title=Single-cell+transcriptomics+reveals+shared+immunosuppressive+landscapes+of+mouse+and+human+neuroblastoma&amp;author=A+Costa&amp;volume=10&amp;publication_year=2022&amp;pages=e004807&amp;pmid=36054452&amp;doi=10.1136/jitc-2022-004807&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR99">99. <span class="element-citation">Weiss T, et al. Schwann cell plasticity regulates neuroblastic tumor cell differentiation via epidermal growth factor-like protein 8. <span><span class="ref-journal">Nat. Commun. </span>2021;<span class="ref-vol">12</span>:1624. doi: 10.1038/s41467-021-21859-0.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7954855/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33712610" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41467-021-21859-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Schwann+cell+plasticity+regulates+neuroblastic+tumor+cell+differentiation+via+epidermal+growth+factor-like+protein+8&amp;author=T+Weiss&amp;volume=12&amp;publication_year=2021&amp;pages=1624&amp;pmid=33712610&amp;doi=10.1038/s41467-021-21859-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR100">100. <span class="element-citation">Ludwig TE, et al. ISSCR standards for the use of human stem cells in basic research. <span><span class="ref-journal">Stem Cell Rep. </span>2023;<span class="ref-vol">18</span>:1744. doi: 10.1016/j.stemcr.2023.08.003.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10545481/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37703820" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.stemcr.2023.08.003" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stem+Cell+Rep.&amp;title=ISSCR+standards+for+the+use+of+human+stem+cells+in+basic+research&amp;author=TE+Ludwig&amp;volume=18&amp;publication_year=2023&amp;pages=1744&amp;doi=10.1016/j.stemcr.2023.08.003&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR101">101. <span class="element-citation">Price CJ, et al. Genetically variant human pluripotent stem cells selectively eliminate wild-type counterparts through YAP-mediated cell competition. <span><span class="ref-journal">Dev. Cell. </span>2021;<span class="ref-vol">56</span>:2455–2470.e10. doi: 10.1016/j.devcel.2021.07.019.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8443275/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34407428" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.devcel.2021.07.019" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev.+Cell&amp;title=Genetically+variant+human+pluripotent+stem+cells+selectively+eliminate+wild-type+counterparts+through+YAP-mediated+cell+competition&amp;author=CJ+Price&amp;volume=56&amp;publication_year=2021&amp;pages=2455-2470.e10&amp;pmid=34407428&amp;doi=10.1016/j.devcel.2021.07.019&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR102">102. <span class="element-citation">Randolph LN, Bao X, Zhou C, Lian X. An all-in-one, Tet-On 3G inducible PiggyBac system for human pluripotent stem cells and derivatives. <span><span class="ref-journal">Sci. Rep. </span>2017;<span class="ref-vol">7</span>:1549. doi: 10.1038/s41598-017-01684-6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5431539/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28484230" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41598-017-01684-6" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Rep.&amp;title=An+all-in-one,+Tet-On+3G+inducible+PiggyBac+system+for+human+pluripotent+stem+cells+and+derivatives&amp;author=LN+Randolph&amp;author=X+Bao&amp;author=C+Zhou&amp;author=X+Lian&amp;volume=7&amp;publication_year=2017&amp;pages=1549&amp;pmid=28484230&amp;doi=10.1038/s41598-017-01684-6&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR103">103. <span class="element-citation">Loew R, Heinz N, Hampf M, Bujard H, Gossen M. Improved Tet-responsive promoters with minimized background expression. <span><span class="ref-journal">BMC Biotechnol. </span>2010;<span class="ref-vol">10</span>:1–13. doi: 10.1186/1472-6750-10-81.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3002914/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21106052" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2F1472-6750-10-81" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Biotechnol.&amp;title=Improved+Tet-responsive+promoters+with+minimized+background+expression&amp;author=R+Loew&amp;author=N+Heinz&amp;author=M+Hampf&amp;author=H+Bujard&amp;author=M+Gossen&amp;volume=10&amp;publication_year=2010&amp;pages=1-13&amp;pmid=20074328&amp;doi=10.1186/1472-6750-10-81&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR104">104. <span class="element-citation">Zhou X, Vink M, Klaver B, Berkhout B, Das AT. Optimization of the Tet-On system for regulated gene expression through viral evolution. <span><span class="ref-journal">Gene Ther. </span>2006;<span class="ref-vol">13</span>:1382–1390. doi: 10.1038/sj.gt.3302780.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16724096" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fsj.gt.3302780" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gene+Ther.&amp;title=Optimization+of+the+Tet-On+system+for+regulated+gene+expression+through+viral+evolution&amp;author=X+Zhou&amp;author=M+Vink&amp;author=B+Klaver&amp;author=B+Berkhout&amp;author=AT+Das&amp;volume=13&amp;publication_year=2006&amp;pages=1382-1390&amp;pmid=16724096&amp;doi=10.1038/sj.gt.3302780&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR105">105. <span class="element-citation">Laing O, Halliwell J, Barbaric I. Rapid PCR assay for detecting common genetic variants arising in human pluripotent stem cell cultures. <span><span class="ref-journal">Curr. Protoc. Stem Cell Biol. </span>2019;<span class="ref-vol">49</span>:e83. doi: 10.1002/cpsc.83.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30821932" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fcpsc.83" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Protoc.+Stem+Cell+Biol.&amp;title=Rapid+PCR+assay+for+detecting+common+genetic+variants+arising+in+human+pluripotent+stem+cell+cultures&amp;author=O+Laing&amp;author=J+Halliwell&amp;author=I+Barbaric&amp;volume=49&amp;publication_year=2019&amp;pages=e83&amp;pmid=30821932&amp;doi=10.1002/cpsc.83&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR106">106. <span class="element-citation">Chen G, et al. Chemically defined conditions for human iPSC derivation and culture. <span><span class="ref-journal">Nat. Methods. </span>2011;<span class="ref-vol">8</span>:424–429. doi: 10.1038/nmeth.1593.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3084903/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21478862" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnmeth.1593" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=Chemically+defined+conditions+for+human+iPSC+derivation+and+culture&amp;author=G+Chen&amp;volume=8&amp;publication_year=2011&amp;pages=424-429&amp;pmid=21478862&amp;doi=10.1038/nmeth.1593&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR107">107. <span class="element-citation">Sturtzel C, et al. Refined high-content imaging-based phenotypic drug screening in zebrafish xenografts. <span><span class="ref-journal">npj Precis. Oncol. </span>2023;<span class="ref-vol">7</span>:1–16.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10195872/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37202469" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=npj+Precis.+Oncol.&amp;title=Refined+high-content+imaging-based+phenotypic+drug+screening+in+zebrafish+xenografts&amp;author=C+Sturtzel&amp;volume=7&amp;publication_year=2023&amp;pages=1-16&amp;pmid=36593350&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR108">108. <span class="mixed-citation">Hanssen, F. et al. Scalable and efficient DNA sequencing analysis on different compute infrastructures aiding variant discovery. bioRxiv <strong>23</strong>, 2023.07.19.549462 (2023). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC11044436/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/38666213" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="CR109">109. <span class="element-citation">Garcia M, et al. Sarek: a portable workflow for whole-genome sequencing analysis of germline and somatic variants. <span><span class="ref-journal">F1000Research. </span>2020;<span class="ref-vol">9</span>:63. doi: 10.12688/f1000research.16665.2.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7111497/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32269765" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.12688%2Ff1000research.16665.2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=F1000Research&amp;title=Sarek:+a+portable+workflow+for+whole-genome+sequencing+analysis+of+germline+and+somatic+variants&amp;author=M+Garcia&amp;volume=9&amp;publication_year=2020&amp;pages=63&amp;pmid=32269765&amp;doi=10.12688/f1000research.16665.2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR110">110. <span class="mixed-citation">Van der Auwera, G. A. &#x00026; O'Connor, B. D. <em>Genomics in the Cloud: Using Docker, GATK, and WDL in Terra</em> 1st edn (O&#x02019;Reilly Media, 2020).</span></div><div class="ref-cit-blk half_rhythm" id="CR111">111. <span class="element-citation">Kim S, et al. Strelka2: fast and accurate calling of germline and somatic variants. <span><span class="ref-journal">Nat. Methods. </span>2018;<span class="ref-vol">15</span>:591–594. doi: 10.1038/s41592-018-0051-x.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30013048" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41592-018-0051-x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=Strelka2:+fast+and+accurate+calling+of+germline+and+somatic+variants&amp;author=S+Kim&amp;volume=15&amp;publication_year=2018&amp;pages=591-594&amp;pmid=30013048&amp;doi=10.1038/s41592-018-0051-x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR112">112. <span class="element-citation">Chen X, et al. Manta: rapid detection of structural variants and indels for germline and cancer sequencing applications. <span><span class="ref-journal">Bioinformatics. </span>2016;<span class="ref-vol">32</span>:1220–1222. doi: 10.1093/bioinformatics/btv710.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26647377" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtv710" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Manta:+rapid+detection+of+structural+variants+and+indels+for+germline+and+cancer+sequencing+applications&amp;author=X+Chen&amp;volume=32&amp;publication_year=2016&amp;pages=1220-1222&amp;pmid=26647377&amp;doi=10.1093/bioinformatics/btv710&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR113">113. <span class="element-citation">Danecek P, et al. Twelve years of SAMtools and BCFtools. <span><span class="ref-journal">Gigascience. </span>2021;<span class="ref-vol">10</span>:giab008. doi: 10.1093/gigascience/giab008.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7931819/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33590861" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fgigascience%2Fgiab008" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gigascience&amp;title=Twelve+years+of+SAMtools+and+BCFtools&amp;author=P+Danecek&amp;volume=10&amp;publication_year=2021&amp;pages=giab008&amp;pmid=33590861&amp;doi=10.1093/gigascience/giab008&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR114">114. <span class="element-citation">McLaren W, et al. The ensembl variant effect predictor. <span><span class="ref-journal">Genome Biol. </span>2016;<span class="ref-vol">17</span>:1–14. doi: 10.1186/s13059-016-0974-4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4893825/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27268795" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2Fs13059-016-0974-4" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol.&amp;title=The+ensembl+variant+effect+predictor&amp;author=W+McLaren&amp;volume=17&amp;publication_year=2016&amp;pages=1-14&amp;pmid=26753840&amp;doi=10.1186/s13059-016-0974-4&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR115">115. <span class="element-citation">Berlanga P, et al. The European MAPPYACTS trial: precision medicine program in pediatric and adolescent patients with recurrent malignancies. <span><span class="ref-journal">Cancer Discov. </span>2022;<span class="ref-vol">12</span>:1266–1281. doi: 10.1158/2159-8290.CD-21-1136.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9394403/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35292802" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1158%2F2159-8290.CD-21-1136" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Discov.&amp;title=The+European+MAPPYACTS+trial:+precision+medicine+program+in+pediatric+and+adolescent+patients+with+recurrent+malignancies&amp;author=P+Berlanga&amp;volume=12&amp;publication_year=2022&amp;pages=1266-1281&amp;pmid=35292802&amp;doi=10.1158/2159-8290.CD-21-1136&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR116">116. <span class="mixed-citation">Adzhubei, I., Jordan, D. M. &#x00026; Sunyaev, S. R. Predicting functional effect of human missense mutations using PolyPhen-2. <em>Curr. Protoc. Hum. Genet.</em> Chapter <strong>7</strong>, 10.1002/0471142905.hg0720s76 (2013). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4480630/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23315928" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="CR117">117. <span class="element-citation">Ng PC, Henikoff S. SIFT: predicting amino acid changes that affect protein function. <span><span class="ref-journal">Nucleic Acids Res. </span>2003;<span class="ref-vol">31</span>:3812–3814. doi: 10.1093/nar/gkg509.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC168916/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12824425" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkg509" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=SIFT:+predicting+amino+acid+changes+that+affect+protein+function&amp;author=PC+Ng&amp;author=S+Henikoff&amp;volume=31&amp;publication_year=2003&amp;pages=3812-3814&amp;pmid=12824425&amp;doi=10.1093/nar/gkg509&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR118">118. <span class="element-citation">Favero F, et al. Sequenza: allele-specific copy number and mutation profiles from tumor sequencing data. <span><span class="ref-journal">Ann. Oncol. </span>2015;<span class="ref-vol">26</span>:64–70. doi: 10.1093/annonc/mdu479.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4269342/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25319062" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fannonc%2Fmdu479" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Oncol.&amp;title=Sequenza:+allele-specific+copy+number+and+mutation+profiles+from+tumor+sequencing+data&amp;author=F+Favero&amp;volume=26&amp;publication_year=2015&amp;pages=64-70&amp;pmid=25319062&amp;doi=10.1093/annonc/mdu479&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR119">119. <span class="element-citation">Gu Z, Eils R, Schlesner M. Complex heatmaps reveal patterns and correlations in multidimensional genomic data. <span><span class="ref-journal">Bioinformatics. </span>2016;<span class="ref-vol">32</span>:2847–2849. doi: 10.1093/bioinformatics/btw313.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27207943" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtw313" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Complex+heatmaps+reveal+patterns+and+correlations+in+multidimensional+genomic+data&amp;author=Z+Gu&amp;author=R+Eils&amp;author=M+Schlesner&amp;volume=32&amp;publication_year=2016&amp;pages=2847-2849&amp;pmid=27207943&amp;doi=10.1093/bioinformatics/btw313&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR120">120. <span class="element-citation">McKenna A, et al. The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. <span><span class="ref-journal">Genome Res. </span>2010;<span class="ref-vol">20</span>:1297–1303. doi: 10.1101/gr.107524.110.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2928508/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20644199" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1101%2Fgr.107524.110" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=The+genome+analysis+toolkit:+a+MapReduce+framework+for+analyzing+next-generation+DNA+sequencing+data&amp;author=A+McKenna&amp;volume=20&amp;publication_year=2010&amp;pages=1297-1303&amp;pmid=20644199&amp;doi=10.1101/gr.107524.110&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR121">121. <span class="element-citation">Schliep K. P. phangorn: phylogenetic analysis in R. <span><span class="ref-journal">Bioinformatics. </span>2011;<span class="ref-vol">27</span>:592–593. doi: 10.1093/bioinformatics/btq706.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3035803/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21169378" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtq706" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=P.+phangorn:+phylogenetic+analysis+in+R&amp;author=K+Schliep&amp;volume=27&amp;publication_year=2011&amp;pages=592-593&amp;pmid=21169378&amp;doi=10.1093/bioinformatics/btq706&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR122">122. <span class="element-citation">Jombart T, Balloux F, Dray S. adephylo: new tools for investigating the phylogenetic signal in biological traits. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:1907–1909. doi: 10.1093/bioinformatics/btq292.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20525823" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtq292" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=adephylo:+new+tools+for+investigating+the+phylogenetic+signal+in+biological+traits&amp;author=T+Jombart&amp;author=F+Balloux&amp;author=S+Dray&amp;volume=26&amp;publication_year=2010&amp;pages=1907-1909&amp;pmid=20525823&amp;doi=10.1093/bioinformatics/btq292&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR123">123. <span class="element-citation">Yu G, Smith DK, Zhu H, Guan Y, Lam TT. Y. ggtree: an r package for visualization and annotation of phylogenetic trees with their covariates and other associated data. <span><span class="ref-journal">Methods Ecol. Evol. </span>2017;<span class="ref-vol">8</span>:28–36. doi: 10.1111/2041-210X.12628.</span> [<a href="//doi.org/10.1111%2F2041-210X.12628" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Ecol.+Evol.&amp;title=Y.+ggtree:+an+r+package+for+visualization+and+annotation+of+phylogenetic+trees+with+their+covariates+and+other+associated+data&amp;author=G+Yu&amp;author=DK+Smith&amp;author=H+Zhu&amp;author=Y+Guan&amp;author=TT+Lam&amp;volume=8&amp;publication_year=2017&amp;pages=28-36&amp;doi=10.1111/2041-210X.12628&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR124">124. <span class="element-citation">McGinnis CS, et al. MULTI-seq: sample multiplexing for single-cell RNA sequencing using lipid-tagged indices. <span><span class="ref-journal">Nat. Methods. </span>2019;<span class="ref-vol">16</span>:619–626. doi: 10.1038/s41592-019-0433-8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6837808/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31209384" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41592-019-0433-8" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=MULTI-seq:+sample+multiplexing+for+single-cell+RNA+sequencing+using+lipid-tagged+indices&amp;author=CS+McGinnis&amp;volume=16&amp;publication_year=2019&amp;pages=619-626&amp;pmid=31209384&amp;doi=10.1038/s41592-019-0433-8&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR125">125. <span class="element-citation">Stuart T, et al. Comprehensive integration of single-cell data. <span><span class="ref-journal">Cell. </span>2019;<span class="ref-vol">177</span>:1888–1902.e21. doi: 10.1016/j.cell.2019.05.031.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6687398/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31178118" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cell.2019.05.031" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Comprehensive+integration+of+single-cell+data&amp;author=T+Stuart&amp;volume=177&amp;publication_year=2019&amp;pages=1888-1902.e21&amp;pmid=31178118&amp;doi=10.1016/j.cell.2019.05.031&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR126">126. <span class="element-citation">Hao Y, et al. Integrated analysis of multimodal single-cell data. <span><span class="ref-journal">Cell. </span>2021;<span class="ref-vol">184</span>:3573–3587.e29. doi: 10.1016/j.cell.2021.04.048.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8238499/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34062119" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cell.2021.04.048" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Integrated+analysis+of+multimodal+single-cell+data&amp;author=Y+Hao&amp;volume=184&amp;publication_year=2021&amp;pages=3573-3587.e29&amp;pmid=34062119&amp;doi=10.1016/j.cell.2021.04.048&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR127">127. <span class="element-citation">Butler A, Hoffman P, Smibert P, Papalexi E, Satija R. Integrating single-cell transcriptomic data across different conditions, technologies, and species. <span><span class="ref-journal">Nat. Biotechnol. </span>2018;<span class="ref-vol">36</span>:411–420. doi: 10.1038/nbt.4096.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6700744/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29608179" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnbt.4096" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=Integrating+single-cell+transcriptomic+data+across+different+conditions,+technologies,+and+species&amp;author=A+Butler&amp;author=P+Hoffman&amp;author=P+Smibert&amp;author=E+Papalexi&amp;author=R+Satija&amp;volume=36&amp;publication_year=2018&amp;pages=411-420&amp;pmid=29608179&amp;doi=10.1038/nbt.4096&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR128">128. <span class="element-citation">Hafemeister C, Satija R. Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression. <span><span class="ref-journal">Genome Biol. </span>2019;<span class="ref-vol">20</span>:576827. doi: 10.1186/s13059-019-1874-1.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6927181/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31870423" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2Fs13059-019-1874-1" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol.&amp;title=Normalization+and+variance+stabilization+of+single-cell+RNA-seq+data+using+regularized+negative+binomial+regression&amp;author=C+Hafemeister&amp;author=R+Satija&amp;volume=20&amp;publication_year=2019&amp;pages=576827&amp;doi=10.1186/s13059-019-1874-1&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR129">129. <span class="element-citation">Lun ATL, et al. EmptyDrops: distinguishing cells from empty droplets in droplet-based single-cell RNA sequencing data. <span><span class="ref-journal">Genome Biol. </span>2019;<span class="ref-vol">20</span>:1–9. doi: 10.1186/s13059-019-1662-y.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6431044/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30902100" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2Fs13059-019-1662-y" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol.&amp;title=EmptyDrops:+distinguishing+cells+from+empty+droplets+in+droplet-based+single-cell+RNA+sequencing+data&amp;author=ATL+Lun&amp;volume=20&amp;publication_year=2019&amp;pages=1-9&amp;pmid=30606230&amp;doi=10.1186/s13059-019-1662-y&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR130">130. <span class="element-citation">Germain PL, Robinson MD, Lun A, Garcia Meixide C, Macnair W. Doublet identification in single-cell sequencing data using <em>scDblFinder</em>. <span><span class="ref-journal">F1000Research. </span>2022;<span class="ref-vol">10</span>:979. doi: 10.12688/f1000research.73600.2.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9204188/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35814628" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.12688%2Ff1000research.73600.2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=F1000Research&amp;title=Doublet+identification+in+single-cell+sequencing+data+using+scDblFinder&amp;author=PL+Germain&amp;author=MD+Robinson&amp;author=A+Lun&amp;author=C+Garcia+Meixide&amp;author=W+Macnair&amp;volume=10&amp;publication_year=2022&amp;pages=979&amp;doi=10.12688/f1000research.73600.2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR131">131. <span class="mixed-citation">Zhu, Q., Conrad, D. N. &#x00026; Gartner, Z. J. deMULTIplex2: robust sample demultiplexing for scRNA-seq. <em>Genome Biol.</em><strong>25</strong>, 37 (2024). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10829271/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/38291503" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="CR132">132. <span class="element-citation">Lopez R, Regier J, Cole MB, Jordan MI, Yosef N. Deep generative modeling for single-cell transcriptomics. <span><span class="ref-journal">Nat. Methods. </span>2018;<span class="ref-vol">15</span>:1053–1058. doi: 10.1038/s41592-018-0229-2.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6289068/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30504886" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41592-018-0229-2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=Deep+generative+modeling+for+single-cell+transcriptomics&amp;author=R+Lopez&amp;author=J+Regier&amp;author=MB+Cole&amp;author=MI+Jordan&amp;author=N+Yosef&amp;volume=15&amp;publication_year=2018&amp;pages=1053-1058&amp;pmid=30504886&amp;doi=10.1038/s41592-018-0229-2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR133">133. <span class="mixed-citation">Hafemeister, C. &#x00026; Halbritter, F. Single-cell RNA-seq differential expression tests within a sample should use pseudo-bulk data of pseudo-replicates. <em>bioRxiv</em> 2023.03.28.534443. 10.1101/2023.03.28.534443 (2023).</span></div><div class="ref-cit-blk half_rhythm" id="CR134">134. <span class="element-citation">Liberzon A, et al. The Molecular Signatures Database (MSigDB) hallmark gene set collection. <span><span class="ref-journal">Cell Syst. </span>2015;<span class="ref-vol">1</span>:417. doi: 10.1016/j.cels.2015.12.004.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4707969/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26771021" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cels.2015.12.004" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Syst.&amp;title=The+Molecular+Signatures+Database+(MSigDB)+hallmark+gene+set+collection&amp;author=A+Liberzon&amp;volume=1&amp;publication_year=2015&amp;pages=417&amp;pmid=26771021&amp;doi=10.1016/j.cels.2015.12.004&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR135">135. <span class="element-citation">Federico A, Monti S. hypeR: an R package for geneset enrichment workflows. <span><span class="ref-journal">Bioinformatics. </span>2020;<span class="ref-vol">36</span>:1307–1308. doi: 10.1093/bioinformatics/btz700.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7998712/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31498385" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtz700" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=hypeR:+an+R+package+for+geneset+enrichment+workflows&amp;author=A+Federico&amp;author=S+Monti&amp;volume=36&amp;publication_year=2020&amp;pages=1307-1308&amp;pmid=31498385&amp;doi=10.1093/bioinformatics/btz700&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR136">136. <span class="element-citation">Street K, et al. Slingshot: cell lineage and pseudotime inference for single-cell transcriptomics. <span><span class="ref-journal">BMC Genom. </span>2018;<span class="ref-vol">19</span>:477. doi: 10.1186/s12864-018-4772-0.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6007078/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29914354" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2Fs12864-018-4772-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genom.&amp;title=Slingshot:+cell+lineage+and+pseudotime+inference+for+single-cell+transcriptomics&amp;author=K+Street&amp;volume=19&amp;publication_year=2018&amp;pages=477&amp;doi=10.1186/s12864-018-4772-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR137">137. <span class="element-citation">van den Berge K, et al. Trajectory-based differential expression analysis for single-cell sequencing data. <span><span class="ref-journal">Nat. Commun. </span>2020;<span class="ref-vol">11</span>:1–13.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7058077/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32139671" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Trajectory-based+differential+expression+analysis+for+single-cell+sequencing+data&amp;author=K+van+den+Berge&amp;volume=11&amp;publication_year=2020&amp;pages=1-13&amp;pmid=31911652&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR138">138. <span class="element-citation">Lambert SA, et al. The human transcription factors. <span><span class="ref-journal">Cell. </span>2018;<span class="ref-vol">172</span>:650–665. doi: 10.1016/j.cell.2018.01.029.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29425488" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cell.2018.01.029" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=The+human+transcription+factors&amp;author=SA+Lambert&amp;volume=172&amp;publication_year=2018&amp;pages=650-665&amp;pmid=29425488&amp;doi=10.1016/j.cell.2018.01.029&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR139">139. <span class="element-citation">Patel AP, et al. Single-cell RNA-seq highlights intratumoral heterogeneity in primary glioblastoma. <span><span class="ref-journal">Science. </span>2014;<span class="ref-vol">344</span>:1396–1401. doi: 10.1126/science.1254257.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4123637/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24925914" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.1254257" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Single-cell+RNA-seq+highlights+intratumoral+heterogeneity+in+primary+glioblastoma&amp;author=AP+Patel&amp;volume=344&amp;publication_year=2014&amp;pages=1396-1401&amp;pmid=24925914&amp;doi=10.1126/science.1254257&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR140">140. <span class="element-citation">Rifatbegovic F, et al. Neuroblastoma cells undergo transcriptomic alterations upon dissemination into the bone marrow and subsequent tumor progression. <span><span class="ref-journal">Int. J. Cancer. </span>2018;<span class="ref-vol">142</span>:297–307. doi: 10.1002/ijc.31053.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5725737/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28921546" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fijc.31053" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Cancer&amp;title=Neuroblastoma+cells+undergo+transcriptomic+alterations+upon+dissemination+into+the+bone+marrow+and+subsequent+tumor+progression&amp;author=F+Rifatbegovic&amp;volume=142&amp;publication_year=2018&amp;pages=297-307&amp;pmid=28921546&amp;doi=10.1002/ijc.31053&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR141">141. <span class="element-citation">Ladenstein R, et al. [Value of prognostic factors in the Austrian A-NB87 Neuroblastoma Study] <span><span class="ref-journal">Klin Padiatr. </span>1996;<span class="ref-vol">208</span>:210–220. doi: 10.1055/s-2008-1046476.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8926686" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1055%2Fs-2008-1046476" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Klin+Padiatr.&amp;title=[Value+of+prognostic+factors+in+the+Austrian+A-NB87+Neuroblastoma+Study]&amp;author=R+Ladenstein&amp;volume=208&amp;publication_year=1996&amp;pages=210-220&amp;pmid=8926686&amp;doi=10.1055/s-2008-1046476&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR142">142. <span class="element-citation">Fiedler S, et al. Long-term outcome and role of biology within risk-adapted treatment strategies: the Austrian Neuroblastoma Trial A-NB94. <span><span class="ref-journal">Cancers. </span>2021;<span class="ref-vol">13</span>:572. doi: 10.3390/cancers13030572.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7867286/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33540616" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fcancers13030572" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancers&amp;title=Long-term+outcome+and+role+of+biology+within+risk-adapted+treatment+strategies:+the+Austrian+Neuroblastoma+Trial+A-NB94&amp;author=S+Fiedler&amp;volume=13&amp;publication_year=2021&amp;pages=572&amp;pmid=33540616&amp;doi=10.3390/cancers13030572&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR143">143. <span class="element-citation">Garaventa A, et al. Randomized trial of two induction therapy regimens for high-risk neuroblastoma: HR-NBL1.5 International society of pediatric oncology european neuroblastoma group study. <span><span class="ref-journal">J. Clin. Oncol. </span>2021;<span class="ref-vol">39</span>:2552–2563. doi: 10.1200/JCO.20.03144.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34152804" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1200%2FJCO.20.03144" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Oncol.&amp;title=Randomized+trial+of+two+induction+therapy+regimens+for+high-risk+neuroblastoma:+HR-NBL1.5+International+society+of+pediatric+oncology+european+neuroblastoma+group+study&amp;author=A+Garaventa&amp;volume=39&amp;publication_year=2021&amp;pages=2552-2563&amp;pmid=34152804&amp;doi=10.1200/JCO.20.03144&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR144">144. <span class="mixed-citation">Ladenstein, R. et al. Investigation of the Role of Dinutuximab Beta-Based Immunotherapy in the SIOPEN High-Risk Neuroblastoma 1 Trial (HR-NBL1). <em>Cancers</em><strong>12</strong>, 309 (2020). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7072500/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32013055" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="CR145">145. <span class="element-citation">Ladenstein R, et al. Interleukin 2 with anti-GD2 antibody ch14.18/CHO (dinutuximab beta) in patients with high-risk neuroblastoma (HR-NBL1/SIOPEN): a multicentre, randomised, phase 3 trial. <span><span class="ref-journal">Lancet Oncol. </span>2018;<span class="ref-vol">19</span>:1617–1629. doi: 10.1016/S1470-2045(18)30578-3.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30442501" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FS1470-2045(18)30578-3" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Oncol.&amp;title=Interleukin+2+with+anti-GD2+antibody+ch14.18/CHO+(dinutuximab+beta)+in+patients+with+high-risk+neuroblastoma+(HR-NBL1/SIOPEN):+a+multicentre,+randomised,+phase+3+trial&amp;author=R+Ladenstein&amp;volume=19&amp;publication_year=2018&amp;pages=1617-1629&amp;pmid=30442501&amp;doi=10.1016/S1470-2045(18)30578-3&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR146">146. <span class="element-citation">Ladenstein R, et al. Busulfan and melphalan versus carboplatin, etoposide, and melphalan as high-dose chemotherapy for high-risk neuroblastoma (HR-NBL1/SIOPEN): an international, randomised, multi-arm, open-label, phase 3 trial. <span><span class="ref-journal">Lancet Oncol. </span>2017;<span class="ref-vol">18</span>:500–514. doi: 10.1016/S1470-2045(17)30070-0.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28259608" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FS1470-2045(17)30070-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Oncol.&amp;title=Busulfan+and+melphalan+versus+carboplatin,+etoposide,+and+melphalan+as+high-dose+chemotherapy+for+high-risk+neuroblastoma+(HR-NBL1/SIOPEN):+an+international,+randomised,+multi-arm,+open-label,+phase+3+trial&amp;author=R+Ladenstein&amp;volume=18&amp;publication_year=2017&amp;pages=500-514&amp;pmid=28259608&amp;doi=10.1016/S1470-2045(17)30070-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR147">147. <span class="element-citation">Love MI, Huber W, Anders S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. <span><span class="ref-journal">Genome Biol. </span>2014;<span class="ref-vol">15</span>:550. doi: 10.1186/s13059-014-0550-8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4302049/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25516281" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2Fs13059-014-0550-8" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol.&amp;title=Moderated+estimation+of+fold+change+and+dispersion+for+RNA-seq+data+with+DESeq2&amp;author=MI+Love&amp;author=W+Huber&amp;author=S+Anders&amp;volume=15&amp;publication_year=2014&amp;pages=550&amp;pmid=25516281&amp;doi=10.1186/s13059-014-0550-8&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR148">148. <span class="element-citation">H&#x000e4;nzelmann S, Castelo R, Guinney J. GSVA: gene set variation analysis for microarray and RNA-Seq data. <span><span class="ref-journal">BMC Bioinform. </span>2013;<span class="ref-vol">14</span>:1–15. doi: 10.1186/1471-2105-14-7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3618321/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23323831" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2F1471-2105-14-7" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinform.&amp;title=GSVA:+gene+set+variation+analysis+for+microarray+and+RNA-Seq+data&amp;author=S+H&#x000e4;nzelmann&amp;author=R+Castelo&amp;author=J+Guinney&amp;volume=14&amp;publication_year=2013&amp;pages=1-15&amp;doi=10.1186/1471-2105-14-7&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR149">149. <span class="element-citation">Buenrostro JD, Giresi PG, Zaba LC, Chang HY, Greenleaf WJ. Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position. <span><span class="ref-journal">Nat. Methods. </span>2013;<span class="ref-vol">10</span>:1213–1218. doi: 10.1038/nmeth.2688.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3959825/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24097267" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnmeth.2688" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=Transposition+of+native+chromatin+for+fast+and+sensitive+epigenomic+profiling+of+open+chromatin,+DNA-binding+proteins+and+nucleosome+position&amp;author=JD+Buenrostro&amp;author=PG+Giresi&amp;author=LC+Zaba&amp;author=HY+Chang&amp;author=WJ+Greenleaf&amp;volume=10&amp;publication_year=2013&amp;pages=1213-1218&amp;pmid=24097267&amp;doi=10.1038/nmeth.2688&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR150">150. <span class="element-citation">Smith JP, et al. PEPATAC: an optimized pipeline for ATAC-seq data analysis with serial alignments. <span><span class="ref-journal">NAR Genom/ Bioinform. </span>2021;<span class="ref-vol">3</span>:lqab101. doi: 10.1093/nargab/lqab101.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8632735/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34859208" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnargab%2Flqab101" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=NAR+Genom/+Bioinform.&amp;title=PEPATAC:+an+optimized+pipeline+for+ATAC-seq+data+analysis+with+serial+alignments&amp;author=JP+Smith&amp;volume=3&amp;publication_year=2021&amp;pages=lqab101&amp;pmid=34859208&amp;doi=10.1093/nargab/lqab101&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR151">151. <span class="mixed-citation">Wickham, H. <em>Ggplot2: Elegant Graphics for Data Analysis</em>. <em>Use R! series</em> (Springer, New York, 2016).</span></div><div class="ref-cit-blk half_rhythm" id="CR152">152. <span class="element-citation">Liao Y, Smyth GK, Shi W. featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. <span><span class="ref-journal">Bioinformatics. </span>2014;<span class="ref-vol">30</span>:923–930. doi: 10.1093/bioinformatics/btt656.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24227677" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtt656" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=featureCounts:+an+efficient+general+purpose+program+for+assigning+sequence+reads+to+genomic+features&amp;author=Y+Liao&amp;author=GK+Smyth&amp;author=W+Shi&amp;volume=30&amp;publication_year=2014&amp;pages=923-930&amp;pmid=24227677&amp;doi=10.1093/bioinformatics/btt656&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR153">153. <span class="element-citation">Lawrence M, et al. Software for computing and annotating genomic ranges. <span><span class="ref-journal">PLoS Comput. Biol. </span>2013;<span class="ref-vol">9</span>:e1003118. doi: 10.1371/journal.pcbi.1003118.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3738458/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23950696" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pcbi.1003118" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Comput.+Biol.&amp;title=Software+for+computing+and+annotating+genomic+ranges&amp;author=M+Lawrence&amp;volume=9&amp;publication_year=2013&amp;pages=e1003118&amp;pmid=23950696&amp;doi=10.1371/journal.pcbi.1003118&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR154">154. <span class="element-citation">Kulakovskiy IV, et al. HOCOMOCO: Towards a complete collection of transcription factor binding models for human and mouse via large-scale ChIP-Seq analysis. <span><span class="ref-journal">Nucleic Acids Res. </span>2018;<span class="ref-vol">46</span>:D252–D259. doi: 10.1093/nar/gkx1106.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5753240/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29140464" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkx1106" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=HOCOMOCO:+Towards+a+complete+collection+of+transcription+factor+binding+models+for+human+and+mouse+via+large-scale+ChIP-Seq+analysis&amp;author=IV+Kulakovskiy&amp;volume=46&amp;publication_year=2018&amp;pages=D252-D259&amp;pmid=29140464&amp;doi=10.1093/nar/gkx1106&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR155">155. <span class="mixed-citation">Korotkevich, G. et al. Fast gene set enrichment analysis. <em>bioRxiv</em> 060012. 10.1101/060012 (2021).</span></div><div class="ref-cit-blk half_rhythm" id="CR156">156. <span class="element-citation">Upton K, et al. Epigenomic profiling of neuroblastoma cell lines. <span><span class="ref-journal">Sci. Data. </span>2020;<span class="ref-vol">7</span>:116. doi: 10.1038/s41597-020-0458-y.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7156688/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32286315" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41597-020-0458-y" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Data&amp;title=Epigenomic+profiling+of+neuroblastoma+cell+lines&amp;author=K+Upton&amp;volume=7&amp;publication_year=2020&amp;pages=116&amp;pmid=32286315&amp;doi=10.1038/s41597-020-0458-y&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR157">157. <span class="element-citation">Decaesteker B, et al. SOX11 regulates SWI/SNF complex components as member of the adrenergic neuroblastoma core regulatory circuitry. <span><span class="ref-journal">Nat. Commun. </span>2023;<span class="ref-vol">14</span>:1–16. doi: 10.1038/s41467-023-36735-2.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9992472/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36882421" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41467-023-36735-2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=SOX11+regulates+SWI/SNF+complex+components+as+member+of+the+adrenergic+neuroblastoma+core+regulatory+circuitry&amp;author=B+Decaesteker&amp;volume=14&amp;publication_year=2023&amp;pages=1-16&amp;pmid=36596776&amp;doi=10.1038/s41467-023-36735-2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR158">158. <span class="element-citation">Xu D, et al. Recapitulation of patient-specific 3D chromatin conformation using machine learning. <span><span class="ref-journal">Cell Rep. Methods. </span>2023;<span class="ref-vol">3</span>:100578. doi: 10.1016/j.crmeth.2023.100578.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10545938/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37673071" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.crmeth.2023.100578" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Rep.+Methods&amp;title=Recapitulation+of+patient-specific+3D+chromatin+conformation+using+machine+learning&amp;author=D+Xu&amp;volume=3&amp;publication_year=2023&amp;pages=100578&amp;pmid=37673071&amp;doi=10.1016/j.crmeth.2023.100578&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR159">159. <span class="element-citation">Hariprakash JM, et al. Leveraging tissue-specific enhancer-target gene regulatory networks identifies enhancer somatic mutations that functionally impact lung cancer. <span><span class="ref-journal">Cancer Res. </span>2024;<span class="ref-vol">84</span>:133–153. doi: 10.1158/0008-5472.CAN-23-1129.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10758689/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37855660" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1158%2F0008-5472.CAN-23-1129" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res.&amp;title=Leveraging+tissue-specific+enhancer-target+gene+regulatory+networks+identifies+enhancer+somatic+mutations+that+functionally+impact+lung+cancer&amp;author=JM+Hariprakash&amp;volume=84&amp;publication_year=2024&amp;pages=133-153&amp;pmid=37855660&amp;doi=10.1158/0008-5472.CAN-23-1129&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR160">160. <span class="element-citation">Boix CA, James BT, Park YP, Meuleman W, Kellis M. Regulatory genomic circuitry of human disease loci by integrative epigenomics. <span><span class="ref-journal">Nature. </span>2021;<span class="ref-vol">590</span>:300–307. doi: 10.1038/s41586-020-03145-z.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7875769/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33536621" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41586-020-03145-z" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Regulatory+genomic+circuitry+of+human+disease+loci+by+integrative+epigenomics&amp;author=CA+Boix&amp;author=BT+James&amp;author=YP+Park&amp;author=W+Meuleman&amp;author=M+Kellis&amp;volume=590&amp;publication_year=2021&amp;pages=300-307&amp;pmid=33536621&amp;doi=10.1038/s41586-020-03145-z&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR161">161. <span class="element-citation">Lee S, Cook D, Lawrence M. Plyranges: a grammar of genomic data transformation. <span><span class="ref-journal">Genome. Biol. </span>2019;<span class="ref-vol">20</span>:4. doi: 10.1186/s13059-018-1597-8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6320618/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30609939" ref="reftype=pubmed&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2Fs13059-018-1597-8" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome.+Biol.&amp;title=Plyranges:+a+grammar+of+genomic+data+transformation&amp;author=S+Lee&amp;author=D+Cook&amp;author=M+Lawrence&amp;volume=20&amp;publication_year=2019&amp;pages=4&amp;pmid=30609939&amp;doi=10.1186/s13059-018-1597-8&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11068915&amp;issue-id=452197&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Nature Communications</span> are provided here courtesy of <strong>Nature Publishing Group</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11068915/pdf/41467_2024_Article_47945.pdf" class="int-view">PDF (7.1M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11068915/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11068915/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11068915" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11068915%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="ZWCicIBDTDHN9pzWyyp2QbQyx48iesSm2a6HHcuJW5UwuYRns3zhd1YEx7Y5ZbZM">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11068915%2F&amp;text=A%20human%20neural%20crest%20model%20reveals%20the%20developmental%20impact%20of%20neuroblastoma-associated%20chromosomal%20aberrations%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11068915%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11068915/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11068915/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38702304/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11068915/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38702304/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11068915/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11068915/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC11068915/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
