
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Amidation of glutamate residues in mycobacterial peptidoglycan is essential for cell wall cross-linking - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="939BE807DB0F4BC50000557EE8664DC6.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="fcimb" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC10484409/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10484409/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Frontiers in Cellular and Infection Microbiology" /><meta name="citation_title" content="Amidation of glutamate residues in mycobacterial peptidoglycan is essential for cell wall cross-linking" /><meta name="citation_author" content="Moagi T. Shaku" /><meta name="citation_author_institution" content="1 DSI/NRF Centre of Excellence for Biomedical Tuberculosis (TB) Research, Faculty of Health Sciences, University of the Witwatersrand, National Health Laboratory Service, Johannesburg, South Africa" /><meta name="citation_author" content="Karl L. Ocius" /><meta name="citation_author_institution" content="2 Department of Chemistry, University of Virginia, Charlottesville, VA, United States" /><meta name="citation_author" content="Alexis J. Apostolos" /><meta name="citation_author_institution" content="2 Department of Chemistry, University of Virginia, Charlottesville, VA, United States" /><meta name="citation_author" content="Marcos M. Pires" /><meta name="citation_author_institution" content="2 Department of Chemistry, University of Virginia, Charlottesville, VA, United States" /><meta name="citation_author" content="Michael S. VanNieuwenhze" /><meta name="citation_author_institution" content="3 Department of Chemistry, Indiana University Bloomington, Bloomington, IN, United States" /><meta name="citation_author" content="Neeraj Dhar" /><meta name="citation_author_institution" content="4 Global Health Institute, Ecole Polytechnique F&#x000e9;d&#x000e9;rale de Lausanne, Lausanne, Switzerland" /><meta name="citation_author" content="Bavesh D. Kana" /><meta name="citation_author_institution" content="1 DSI/NRF Centre of Excellence for Biomedical Tuberculosis (TB) Research, Faculty of Health Sciences, University of the Witwatersrand, National Health Laboratory Service, Johannesburg, South Africa" /><meta name="citation_publication_date" content="2023" /><meta name="citation_volume" content="13" /><meta name="citation_doi" content="10.3389/fcimb.2023.1205829" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC10484409/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC10484409/" /><meta name="citation_pmid" content="37692163" /><meta name="DC.Title" content="Amidation of glutamate residues in mycobacterial peptidoglycan is essential for cell wall cross-linking" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Frontiers Media SA" /><meta name="DC.Contributor" content="Moagi T. Shaku" /><meta name="DC.Contributor" content="Karl L. Ocius" /><meta name="DC.Contributor" content="Alexis J. Apostolos" /><meta name="DC.Contributor" content="Marcos M. Pires" /><meta name="DC.Contributor" content="Michael S. VanNieuwenhze" /><meta name="DC.Contributor" content="Neeraj Dhar" /><meta name="DC.Contributor" content="Bavesh D. Kana" /><meta name="DC.Date" content="2023" /><meta name="DC.Identifier" content="10.3389/fcimb.2023.1205829" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Amidation of glutamate residues in mycobacterial peptidoglycan is essential for cell wall cross-linking" /><meta property="og:type" content="article" /><meta property="og:description" content="Mycobacteria assemble a complex cell wall with cross-linked peptidoglycan (PG) which plays an essential role in maintenance of cell wall integrity and tolerance to osmotic pressure. We previously demonstrated that various hydrolytic enzymes are required ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10484409/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10484409/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22Front%20Cell%20Infect%20Microbiol%22[journal]" class="navlink">Front Cell Infect Microbiol</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC10484409
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC10484409/pdf/fcimb-13-1205829.pdf" class="int-view">PDF (9.8M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/10484409/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/10484409/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="10484409" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10484409%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="i5h1OMAhZwBcyw2igDc5aNkDWRhlR9qQEIqAtv1cvoAq3Btlwd3Qo4ZkgTZgkNMQ">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10484409%2F&amp;text=Amidation%20of%20glutamate%20residues%20in%20mycobacterial%20peptidoglycan%20is%20essential%20for%20cell%20wall%20cross-linking" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10484409%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10484409/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC10484409/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/37692163/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC10484409/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/37692163/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC10484409/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/10484409/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22Front%20Cell%20Infect%20Microbiol%22[journal]" class="navlink">Front Cell Infect Microbiol</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC10484409
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-fcimb.gif" alt="Logo of fcimb" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.frontiersin.org/cellular_and_infection_microbiology" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Front Cell Infect Microbiol.</span> 2023; 13: 1205829. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2023 Aug 24. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.3389%2Ffcimb.2023.1205829" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.3389/fcimb.2023.1205829</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC10484409</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/37692163">37692163</a></div></div></div><h1 class="content-title">Amidation of glutamate residues in mycobacterial peptidoglycan is essential for cell wall cross-linking</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shaku%20MT%5BAuthor%5D" class="affpopup" co-rid="_co_idm140530481401744" co-class="co-affbox">Moagi T. Shaku</a>,<sup>
1
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ocius%20KL%5BAuthor%5D" class="affpopup" co-rid="_co_idm140530481387440" co-class="co-affbox">Karl L. Ocius</a>,<sup>
2
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Apostolos%20AJ%5BAuthor%5D" class="affpopup" co-rid="_co_idm140530481385520" co-class="co-affbox">Alexis J. Apostolos</a>,<sup>
2
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Pires%20MM%5BAuthor%5D" class="affpopup" co-rid="_co_idm140530481383344" co-class="co-affbox">Marcos M. Pires</a>,<sup>
2
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=VanNieuwenhze%20MS%5BAuthor%5D" class="affpopup" co-rid="_co_idm140530481381424" co-class="co-affbox">Michael S. VanNieuwenhze</a>,<sup>
3
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Dhar%20N%5BAuthor%5D" class="affpopup" co-rid="_co_idm140530484132192" co-class="co-affbox">Neeraj Dhar</a>,<sup>
4
,</sup><sup>
&#x02020;
</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kana%20BD%5BAuthor%5D" class="affpopup" co-rid="_co_idm140530484128784" co-class="co-affbox">Bavesh D. Kana</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>
1
,</sup><sup>
*
</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140530481401744"><h3 class="no_margin">Moagi T. Shaku</h3><p>
<sup>1</sup>
DSI/NRF Centre of Excellence for Biomedical Tuberculosis (TB) Research, Faculty of Health Sciences, University of the Witwatersrand, National Health Laboratory Service, Johannesburg, South Africa
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shaku%20MT%5BAuthor%5D">Moagi T. Shaku</a></div></div><div id="_co_idm140530481387440"><h3 class="no_margin">Karl L. Ocius</h3><p>
<sup>2</sup>
Department of Chemistry, University of Virginia, Charlottesville, VA, United States
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ocius%20KL%5BAuthor%5D">Karl L. Ocius</a></div></div><div id="_co_idm140530481385520"><h3 class="no_margin">Alexis J. Apostolos</h3><p>
<sup>2</sup>
Department of Chemistry, University of Virginia, Charlottesville, VA, United States
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Apostolos%20AJ%5BAuthor%5D">Alexis J. Apostolos</a></div></div><div id="_co_idm140530481383344"><h3 class="no_margin">Marcos M. Pires</h3><p>
<sup>2</sup>
Department of Chemistry, University of Virginia, Charlottesville, VA, United States
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Pires%20MM%5BAuthor%5D">Marcos M. Pires</a></div></div><div id="_co_idm140530481381424"><h3 class="no_margin">Michael S. VanNieuwenhze</h3><p>
<sup>3</sup>
Department of Chemistry, Indiana University Bloomington, Bloomington, IN, United States
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=VanNieuwenhze%20MS%5BAuthor%5D">Michael S. VanNieuwenhze</a></div></div><div id="_co_idm140530484132192"><h3 class="no_margin">Neeraj Dhar</h3><p>
<sup>4</sup>
Global Health Institute, Ecole Polytechnique F&#x000e9;d&#x000e9;rale de Lausanne, Lausanne, Switzerland
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Dhar%20N%5BAuthor%5D">Neeraj Dhar</a></div></div><div id="_co_idm140530484128784"><h3 class="no_margin">Bavesh D. Kana</h3><p>
<sup>1</sup>
DSI/NRF Centre of Excellence for Biomedical Tuberculosis (TB) Research, Faculty of Health Sciences, University of the Witwatersrand, National Health Laboratory Service, Johannesburg, South Africa
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kana%20BD%5BAuthor%5D">Bavesh D. Kana</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140530526165888_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140530526165888_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140530526165888_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140530526165888_ai" style="display:none"><div class="fm-affl" id="aff1">
<sup>1</sup>
DSI/NRF Centre of Excellence for Biomedical Tuberculosis (TB) Research, Faculty of Health Sciences, University of the Witwatersrand, National Health Laboratory Service, Johannesburg, South Africa
</div><div class="fm-affl" id="aff2">
<sup>2</sup>
Department of Chemistry, University of Virginia, Charlottesville, VA, United States
</div><div class="fm-affl" id="aff3">
<sup>3</sup>
Department of Chemistry, Indiana University Bloomington, Bloomington, IN, United States
</div><div class="fm-affl" id="aff4">
<sup>4</sup>
Global Health Institute, Ecole Polytechnique F&#x000e9;d&#x000e9;rale de Lausanne, Lausanne, Switzerland
</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup>Corresponding author.</div><div id="fn-a.w.b.i.a">Edited by: Anthony Baughn, University of Minnesota Twin Cities, United States</div><div id="fn-a.w.b.i.b">Reviewed by: Yasu S. Morita, University of Massachusetts Amherst, United States; Takehiro Kado, University of Massachusetts Amhers, in collaboration with reviewer YM; Cara C. Boutte, University of Texas at Arlington, United States</div><div id="fn001">*Correspondence: Bavesh D. Kana, <a href="mailto:dev@null" data-email="az.ca.stiw@anak.hsevab" class="oemail">az.ca.stiw@anak.hsevab</a>
</div><div id="fn003">&#x02020;Present address: Neeraj Dhar, Vaccine and Infectious Disease Organization, University of Saskatchewan, Saskatoon, SK, Canada;Department of Biochemistry, Microbiology and Immunology, University of Saskatchewan, Saskatoon, SK, Canada</div></div><div class="fm-article-notes hide half_rhythm" id="idm140530526165888_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2023 Apr 14; Accepted 2023 Jul 31.</div></div><div class="permissions half_rhythm hide" id="idm140530526165888_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  &#x000a9; 2023 Shaku, Ocius, Apostolos, Pires, VanNieuwenhze, Dhar and Kana</div><div class="license half_rhythm">This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="14" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div><a href="/pmc/articles/PMC10484409/bin/Video_1.avi" data-ga-action="click_feat_suppl">Video_1.avi</a><span style="color:gray"> (1.6M)</span></div><div class="small guid">GUID: 91C1C091-2293-4014-BE59-507D8BC13C69</div></div><div class="half_rhythm"><div><a href="/pmc/articles/PMC10484409/bin/Video_2.avi" data-ga-action="click_feat_suppl">Video_2.avi</a><span style="color:gray"> (1.3M)</span></div><div class="small guid">GUID: BB020F14-BF2A-4335-B294-B77C25BBEEC4</div></div><div class="half_rhythm"><div><a href="/pmc/articles/PMC10484409/bin/Video_3.avi" data-ga-action="click_feat_suppl">Video_3.avi</a><span style="color:gray"> (1.3M)</span></div><div class="small guid">GUID: 0C2F63F7-9F4F-49D6-B9BB-51153A1695DF</div></div><div class="half_rhythm"><div><a href="/pmc/articles/PMC10484409/bin/Video_4.avi" data-ga-action="click_feat_suppl">Video_4.avi</a><span style="color:gray"> (1.4M)</span></div><div class="small guid">GUID: 551DD94F-9253-4786-BD07-DA34753C3A33</div></div><div class="half_rhythm"><div><a href="/pmc/articles/PMC10484409/bin/Video_5.avi" data-ga-action="click_feat_suppl">Video_5.avi</a><span style="color:gray"> (1.2M)</span></div><div class="small guid">GUID: EC193299-5F22-4DBD-B6C0-3C1E0F374977</div></div><div class="half_rhythm"><div><a href="/pmc/articles/PMC10484409/bin/Video_6.avi" data-ga-action="click_feat_suppl">Video_6.avi</a><span style="color:gray"> (1.2M)</span></div><div class="small guid">GUID: A23D1AA9-34ED-4F06-90AA-6661671AAF81</div></div><div class="half_rhythm"><div><a href="/pmc/articles/PMC10484409/bin/Video_7.avi" data-ga-action="click_feat_suppl">Video_7.avi</a><span style="color:gray"> (838K)</span></div><div class="small guid">GUID: DC741C5B-0112-4C83-BB14-65294A98DD18</div></div><div class="half_rhythm"><div><a href="/pmc/articles/PMC10484409/bin/Video_8.avi" data-ga-action="click_feat_suppl">Video_8.avi</a><span style="color:gray"> (1.0M)</span></div><div class="small guid">GUID: F2F51058-9FA8-4E3B-AA95-EBCB8925182B</div></div><div class="half_rhythm"><div><a href="/pmc/articles/PMC10484409/bin/Video_9.avi" data-ga-action="click_feat_suppl">Video_9.avi</a><span style="color:gray"> (944K)</span></div><div class="small guid">GUID: F1A6D927-574E-4FCE-AA22-D0716EDCBAEE</div></div><div class="half_rhythm"><div><a href="/pmc/articles/PMC10484409/bin/Video_10.avi" data-ga-action="click_feat_suppl">Video_10.avi</a><span style="color:gray"> (1.1M)</span></div><div class="small guid">GUID: DA5AE260-B4C0-43C3-A569-5C3E4D279A5B</div></div><div class="half_rhythm"><div><a href="/pmc/articles/PMC10484409/bin/Video_11.avi" data-ga-action="click_feat_suppl">Video_11.avi</a><span style="color:gray"> (675K)</span></div><div class="small guid">GUID: B18EE515-F4E3-465D-8181-B05FE1D725E7</div></div><div class="half_rhythm"><div><a href="/pmc/articles/PMC10484409/bin/Video_12.avi" data-ga-action="click_feat_suppl">Video_12.avi</a><span style="color:gray"> (1.2M)</span></div><div class="small guid">GUID: 3A2B0455-E0B1-4E4C-AE06-AF02BFD34446</div></div><div class="half_rhythm"><div><a href="/pmc/articles/PMC10484409/bin/DataSheet_1.pdf" data-ga-action="click_feat_suppl">DataSheet_1.pdf</a><span style="color:gray"> (12M)</span></div><div class="small guid">GUID: F92AD804-2C27-4BC9-A8C0-2109CC94D149</div></div><div class="half_rhythm"><div><a href="/pmc/articles/PMC10484409/bin/Table_1.docx" data-ga-action="click_feat_suppl">Table_1.docx</a><span style="color:gray"> (26K)</span></div><div class="small guid">GUID: 8E7DB9E6-AFF9-4A45-84EE-A1186A6DFF9E</div></div></dd></dl><dl data-length="166" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first-last">The original contributions presented in the study are included in the article/<a href="#s10" rid="s10" class=" sec">
<strong>Supplementary Material</strong>
</a>. Further inquiries can be directed to the corresponding author.</p></dd></dl></div><div id="abstract-a.w.b.p" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.w.b.ptitle">Abstract</h2><!--article-meta--><div><div id="sec-a.w.b.p.a" class="sec sec-first"><h3 id="sec-a.w.b.p.atitle">Introduction</h3><p class="p p-first-last">Mycobacteria assemble a complex cell wall with cross-linked peptidoglycan (PG) which plays an essential role in maintenance of cell wall integrity and tolerance to osmotic pressure. We previously demonstrated that various hydrolytic enzymes are required to remodel PG during essential processes such as cell elongation and septal hydrolysis. Here, we explore the chemistry associated with PG cross-linking, specifically the requirement for amidation of the D-glutamate residue found in PG precursors.</p></div><div id="sec-a.w.b.p.b" class="sec"><h3 id="sec-a.w.b.p.btitle">Methods</h3><p class="p p-first-last">Synthetic fluorescent probes were used to assess PG remodelling dynamics in live bacteria. Fluorescence microscopy was used to assess protein localization in live bacteria and CRISPR-interference was used to construct targeted gene knockdown strains. Time-lapse microscopy was used to assess bacterial growth. Western blotting was used to assess protein phosphorylation.</p></div><div id="sec-a.w.b.p.c" class="sec sec-last"><h3 id="sec-a.w.b.p.ctitle">Results and discussion</h3><p class="p p-first-last">In <em>Mycobacterium smegmatis</em>, we confirmed the essentiality for D-glutamate amidation in PG biosynthesis by labelling cells with synthetic fluorescent PG probes carrying amidation modifications. We also used CRISPRi targeted knockdown of genes encoding the MurT-GatD complex, previously implicated in D-glutamate amidation, and demonstrated that these genes are essential for mycobacterial growth. We show that MurT-rseGFP co-localizes with mRFP-GatD at the cell poles and septum, which are the sites of cell wall synthesis in mycobacteria. Furthermore, time-lapse microscopic analysis of MurT-rseGFP localization, in fluorescent D-amino acid (FDAA)-labelled mycobacterial cells during growth, demonstrated co-localization with maturing PG, suggestive of a role for PG amidation during PG remodelling and repair. Depletion of MurT and GatD caused reduced PG cross-linking and increased sensitivity to lysozyme and &#x003b2;-lactam antibiotics. Cell growth inhibition was found to be the result of a shutdown of PG biosynthesis mediated by the serine/threonine protein kinase B (PknB) which senses uncross-linked PG. Collectively, these data demonstrate the essentiality of D-glutamate amidation in mycobacterial PG precursors and highlight the MurT-GatD complex as a novel drug target.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">MurT-GatD complex, peptidoglycan biosynthesis, peptidoglycan amidation, peptidoglycan cross-linking, peptidoglycan remodelling</span></div></div><div id="s1" class="tsec sec"><h2 class="head no_bottom_margin" id="s1title">1. Introduction</h2><p class="p p-first">Peptidoglycan (PG) is a heteropolymer composed of glycan strands made up of alternating units of <em>N</em>-acetylglucosamine (Glc<em>N</em>Ac) linked to <em>N</em>-acetylmuramic acid (Mur<em>N</em>Ac) cross-linked by stem peptides attached to the lactyl moiety of Mur<em>N</em>Ac (<a href="#B46" rid="B46" class=" bibr popnode">Vollmer and Bertsche, 2008</a>). PG biosynthesis is initiated in the bacterial cytoplasm, where a variety of enzymes use carbohydrate monomers and amino acids to build the PG precursor, also known as lipid II (<a href="#B46" rid="B46" class=" bibr popnode">Vollmer and Bertsche, 2008</a>). Lipid II is then modified by enzymes such as NamH, a UDP-<em>N</em>-acetylmuramic acid hydroxylase, that catalyzes hydroxylation of the UDP-Mur<em>N</em>Ac monomer, resulting in the formation of UDP-<em>N</em>-glycolylmuramic acid (UDP-Mur<em>N</em>Glyc) (<a href="#B1" rid="B1" class=" bibr popnode">Abrahams and Besra, 2018</a>). The pentapeptide unit of lipid II is also modified at the &#x003b1;-carboxyl group of D-glutamate (D-Glu) by the MurT-GatD complex (<a href="#B30" rid="B30" class=" bibr popnode">M&#x000fc;nch et&#x000a0;al., 2012</a>; <a href="#B25" rid="B25" class=" bibr popnode">Maitra et&#x000a0;al., 2019</a>) and at the &#x003b1;-carboxyl group of <em>meso</em>-diaminopimelic by AsnB (<a href="#B32" rid="B32" class=" bibr popnode">Ngadjeua et&#x000a0;al., 2018</a>; <a href="#B35" rid="B35" class=" bibr popnode">Pidgeon et&#x000a0;al., 2019</a>) (<a href="/pmc/articles/PMC10484409/figure/f1/" target="figure" class="fig-table-link figpopup" rid-figpopup="f1" rid-ob="ob-f1" co-legend-rid="lgnd_f1"><span>
<strong>Figure&#x000a0;1A</strong>
</span></a>). Finally, modified lipid II is translocated to the periplasm by the cell membrane-embedded flippase MurJ and incorporated into the existing PG through transglycosylation and transpeptidation reactions, facilitated by penicillin binding proteins (PBPs) (<a href="#B45" rid="B45" class=" bibr popnode">Vollmer, 2007</a>) (<a href="/pmc/articles/PMC10484409/figure/f1/" target="figure" class="fig-table-link figpopup" rid-figpopup="f1" rid-ob="ob-f1" co-legend-rid="lgnd_f1"><span>
<strong>Figure&#x000a0;1A</strong>
</span></a>). The transpeptidation reactions form either 4-3 cross-links between adjacent PG stem peptides (D-alanine-<em>meso</em>-DAP, formed by D,D-transpeptidases) or 3-3 cross-links (<em>meso</em>-DAP-meso-DAP, by L,D-transpeptidases [LDTs]). In mycobacteria, 3-3 cross-links constitute the majority of PG cross-links and are required for the maintenance of cell wall integrity during aging of the cell wall (<a href="#B21" rid="B21" class=" bibr popnode">Lavollay et&#x000a0;al., 2008</a>; <a href="#B3" rid="B3" class=" bibr popnode">Baranowski et&#x000a0;al., 2018</a>; <a href="#B14" rid="B14" class=" bibr popnode">Francis et&#x000a0;al., 2023</a>). The Mur<em>N</em>Ac/Mur<em>N</em>Glyc residues of lipid II are further modified by the addition of an &#x003b1;-L-rhamnopyranose-(1-3)-&#x003b1;-D-NAG-(1-P) unit that links PG to arabinogalactan (<a href="#B1" rid="B1" class=" bibr popnode">Abrahams and Besra, 2018</a>). Considering the essential role of the PG modifications for cell wall integrity, their biological significance and associated pathways in mycobacteria still require elucidation.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f1" co-legend-rid="lgnd_f1"><a href="/pmc/articles/PMC10484409/figure/f1/" target="figure" rid-figpopup="f1" rid-ob="ob-f1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140530483234608"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10484409_fcimb-13-1205829-g001.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fcimb-13-1205829-g001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10484409/bin/fcimb-13-1205829-g001.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140530483234608"><a target="object" rel="noopener" href="/pmc/articles/PMC10484409/figure/f1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_f1"><div><a class="figpopup" href="/pmc/articles/PMC10484409/figure/f1/" target="figure" rid-figpopup="f1" rid-ob="ob-f1">Figure&#x000a0;1</a></div><!--caption a7--><div class="caption"><p>Amidated peptidoglycan stem peptide mimics are incorporated into the mycobacterial cell wall. <strong>(A)</strong> Schematic representation of MurT-GatD and AsnB mediated PG precursor amidation and PG cross-linking. <strong>(B)</strong> Labeling of <em>M. smegmatis</em> mc<sup>2</sup>155 and Mtb H37Rv cells with Tetramethylrhodamine (TAMRA)-3-amino-L-alanine-D-glutamine-L-lysine-D-alanine (TetraFl-1). Red box shows the amide group of D-glutamine (D-Gln) in the TetraFl-1 chemical structure. TetraFl-1 labelled <em>M. smegmatis</em> mc<sup>2</sup>155 and Mtb H37Rv cells. The change in TAMRA fluorescence intensity during TetraFl-1 incorporation into PG is indicated by the purple arrow in the flow cytometry histograms. White arrow shows septal labelling with TetraFl-1. <strong>(C)</strong> Labeling of <em>M. smegmatis</em> mc<sup>2</sup>155 and Mtb H37Rv cells with TAMRA-3-amino-L-alanine-D-glutamate-L-lysine-D-alanine (TetraFl-2). Red box shows the hydroxyl group of D-glutamate (D-Glu) in the TetraFl-2 chemical structure. TetraFl-2 did not label <em>M. smegmatis</em> mc<sup>2</sup>155 and Mtb H37Rv cells, indicating that the amidation at D-Gln is required for probe incorporation.</p></div></div></div><p>Recently, it was demonstrated that a lack of lipid II amidation in PG inhibits cross-linking by PG transpeptidases and this was also confirmed by the use of synthetic PG fragments with amidated D-Glu amidated (i.e. D-Gln) residues (<a href="#B35" rid="B35" class=" bibr popnode">Pidgeon et&#x000a0;al., 2019</a>). The lack of PG cross-linking because of reduced PG precursor amidation sensitizes bacteria to &#x003b2;-lactam antibiotics, lysozyme and immunological detection during infection (<a href="#B48" rid="B48" class=" bibr popnode">Wolfert et&#x000a0;al., 2007</a>; <a href="#B12" rid="B12" class=" bibr popnode">Figueiredo et&#x000a0;al., 2012</a>; <a href="#B22" rid="B22" class=" bibr popnode">Levefaudes et&#x000a0;al., 2015</a>; <a href="#B44" rid="B44" class=" bibr popnode">Vijayrajratnam et&#x000a0;al., 2016</a>). Ldt<sub>FM</sub> and Ldt<sub>MT2</sub> of <em>Enterococcus faecium</em> and <em>Mycobacterium tuberculosis</em> (Mtb), respectively, are not able to bind either stem peptide derivatives with non-amidated D-Glu as substrates for PG biosynthesis (<a href="#B32" rid="B32" class=" bibr popnode">Ngadjeua et&#x000a0;al., 2018</a>). In addition, depletion of AsnB, a glutamate amidotransferase required for <em>meso</em>-DAP amidation in Mtb was shown to result in cell death due to defective PG cross-linking (<a href="#B32" rid="B32" class=" bibr popnode">Ngadjeua et&#x000a0;al., 2018</a>). Furthermore, the D-Glu amidation on PG peptide stems is required by not only LDTs, but also PBPs for efficient D,D-transpeptidation reactions resulting in the formation of 4-3 cross-links (<a href="#B49" rid="B49" class=" bibr popnode">Zapun et&#x000a0;al., 2013</a>). Similarly, deletion of LtsA, a <em>meso</em>-DAP amidotransferase in Corynebacteria, caused a reduction in PG amidation at the <em>meso</em>-DAP residue. Although not affecting PG cross-linking, this caused increased sensitivity to lysozyme (<a href="#B22" rid="B22" class=" bibr popnode">Levefaudes et&#x000a0;al., 2015</a>).</p><p>Biochemical studies of the mycobacterial MurT-GatD complex have confirmed a role for this protein complex in mycobacterial PG biosynthesis (<a href="#B26" rid="B26" class=" bibr popnode">Maitra et&#x000a0;al., 2021</a>). The MurT-GatD complex catalyzes D-Glu amidation using L-glutamine as an amine donor, thereby converting L-glutamine into NH<sub>3</sub> and glutamate by GatD, followed by the shuttling of NH<sub>3</sub> from the GatD active site to the amidotransferase active-site of MurT. Finally, through an ATP-dependent reaction, MurT catalyzes lipid II amidation by transferring NH<sub>3</sub> to D-glutamate found in the second position of the lipid II stem peptide (<a href="#B29" rid="B29" class=" bibr popnode">Morlot et&#x000a0;al., 2018</a>; <a href="#B33" rid="B33" class=" bibr popnode">N&#x000f6;ldeke et&#x000a0;al., 2018</a>).</p><p class="p p-last">In this work, we used the CRISPRi platform (<a href="#B36" rid="B36" class=" bibr popnode">Rock et&#x000a0;al., 2017</a>) to deplete MurT and GatD in <em>Mycobacterium smegmatis</em> and showed that this caused cell growth inhibition as a result of expected reduced lipid II amidation and we studied the associated pathways resulting in this phenotype. By using fluorescent tetrapeptide probes, we further confirmed that D-Gln is required for probe incorporation into mycobacterial cells. Lastly, we confirmed the essentiality of the MurT-GatD complex for mycobacterial growth and cell wall stability.</p></div><div id="s2" class="tsec sec"><h2 class="head no_bottom_margin" id="s2title">2. Materials and methods</h2><div id="s2_1" class="sec sec-first"><h3 id="s2_1title">2.1. Bacterial strains and culture conditions</h3><p class="p p-first-last">
<em>E. coli</em> DH5&#x003b1; and derivative strains were grown in Luria-Bertani broth (LB) or on Luria-Bertani agar (LA) at 37&#x000b0;C supplemented with the appropriate antibiotics. Antibiotic concentrations used for <em>E. coli</em> strains were as follows: kanamycin (kan): 50 &#x000b5;g/ml, hygromycin (hyg): 200 &#x000b5;g/ml and antibiotic concentrations used for <em>M. smegmatis</em> and Mtb strains were as follows: kan: 25 &#x000b5;g/ml, hyg: 50 &#x000b5;g/ml, amoxicillin (20 &#x000b5;g/ml) and clavulanate (10 &#x000b5;g/ml). <em>M. smegmatis</em> mc<sup>2</sup>155 (<a href="#B40" rid="B40" class=" bibr popnode">Snapper et&#x000a0;al., 1990</a>), Mtb H37Rv and derivative strains were grown at 37&#x000b0;C in Middlebrook 7H9 broth supplemented with 0.2% glucose, 0.085% NaCl (for <em>M. smegmatis</em>), Middlebrook OADC enrichment (for Mtb), 0.2% glycerol (for <em>M. smegmatis</em>), 0.5% glycerol (for Mtb) and 0.05% Tween-80 or on Middlebrook 7H10/7H11 agar supplemented with 0.5% glycerol, 0.2% glucose, 0.085% NaCl or Middlebrook OADC enrichment (for Mtb). Liquid cultures were grown at 37&#x000b0;C with shaking at 100 rpm for <em>M. smegmatis</em> and 200 rpm for <em>E.coli</em>.</p></div><div id="s2_2" class="sec"><h3 id="s2_2title">2.2. Genetic manipulation</h3><p class="p p-first-last">The <em>M. smegmatis</em> mc<sup>2</sup>155 MurT-GatD CRISPRi dCas9 based depletion strain was constructed as previously described [the PAM sequence and sgRNA sequence primers are listed in <a href="#ST1" rid="ST1" class=" supplementary-material">
<strong>Table A3</strong>
</a> (<a href="#B36" rid="B36" class=" bibr popnode">Rock et&#x000a0;al., 2017</a>)]. For protein localization studies, MurT-rseGFP was constructed by cloning wildtype <em>murT</em> (MSMEG_6276) with its 100 bp upstream region into a previously generated plasmid pMV306H-rseGFP (<a href="#ST1" rid="ST1" class=" supplementary-material">
<strong>Table A2</strong>
</a>). Briefly, the MSMEG_6276 gene was PCR amplified and cloned into plasmid pMV306H-rseGFP for C-terminal tagging to generate plasmid pMurT-rseGFP, which was then electroporated into wildtype <em>M. smegmatis</em> mc<sup>2</sup>155. For electroporation of plasmid DNA, 1 &#x003bc;g of plasmid DNA was used to transform electro-competent mycobacterial cells or derivative strains by mixing 300 &#x003bc;l of cells with plasmid DNA and transferring the mixture into a pre-chilled 0.2 cm electroporation cuvette (Bio-Rad laboratories). The cells were incubated on ice for 15 minutes and the Bio-Rad Gene Pulser XCell&#x02122; (Bio-Rad laboratories) system was used to perform plasmid DNA electroporations with the following parameters: voltage: 2500 V, time constant: 25 &#x003bc;F, resistance: 1000 &#x003a9; and distance: 0.2 cm. After electroporation, 800 &#x003bc;l of LB media was added to the electroporation cuvette and the cells were transferred into a clean tube and incubated at 37&#x000b0;C overnight. Thereafter, the cells were pelleted by centrifugation at 12470 <em>Xg</em> and the supernatant was discarded followed by the resuspension of the cells in 500 &#x003bc;l of Middlebrook 7H9 broth. The cells were inoculated onto Middlebrook 7H10 agar containing appropriate antibiotics and incubated at 37&#x000b0;C for 3-5 days (<a href="#B16" rid="B16" class=" bibr popnode">Gordhan and Parish, 2001</a>). The mRFP-GatD fusion was constructed by cloning wildtype <em>gatD</em> (MSMEG_6277) into a previously generated plasmid pTweety-mRFP (<a href="#B41" rid="B41" class=" bibr popnode">Stover et&#x000a0;al., 1991</a>). The MSMEG_6277 gene was PCR amplified and cloned into plasmid pTweety-mRFP for N-terminal tagging to generate plasmid pmRFP-GatD, which was electroporated as indicated above into <em>M. smegmatis</em> carrying the pMurT-rseGFP. For PknB-FLAG, the <em>pknB</em> (MSMEG_0028) gene was PCR amplified and cloned into plasmid pFLAGEM (<a href="#B31" rid="B31" class=" bibr popnode">Narrandes et al., 2015</a>) derived from plasmid pSE100 (<a href="#B17" rid="B17" class=" bibr popnode">Guo et&#x000a0;al., 2007</a>) and electroporated as indicated above into <em>M. smegmatis</em> mc<sup>2</sup>155.</p></div><div id="s2_3" class="sec"><h3 id="s2_3title">2.3. Quantitative real-time PCR</h3><p class="p p-first-last">RNA was extracted using the Nucleospin RNA II kit (Machery Nagel) as per manufacturer&#x02019;s instructions from cells grown at 37&#x000b0;C to an OD<sub>600nm</sub> of 0.6 in 50 ml 7H9 broth. ATc or antibiotic supplementation of media where required was performed at the start of the culture. RNA was quantified using the Nanodrop. Prior to reverse transcriptase (RT)-PCR, 1 &#x000b5;g of RNA was treated with 1 &#x000b5;l TURBO DNaseI (Life Technologies) for 30 min at 37&#x000b0;C to remove residual genomic DNA. DNaseI was inactivated by the addition of 7 &#x000b5;l DNaseI removal resin followed by centrifugation for 5 min at 12470 <em>Xg</em>. A 23 &#x000b5;l aliquot was used for RT-PCR. A 2.5 &#x000b5;M reverse primer mix containing MSMEG_6276 (<em>murT</em>), MSMEG_6277 (<em>gatD</em>) and MSMEG_2758 (<em>sigA</em>) reverse primers (MurTRev: atgagcacgtcgtcacagtt, GatDRev: cgcggtggttttcgaatcc and SigARev: gggcgtgatgtccatctcct) was prepared and 2 &#x000b5;l of this mixture was added to TURBO DNase treated RNA. The housekeeping <em>sigA</em> gene allows for normalization because the gene is expressed at a consistent level throughout all growth phases. PCR reactions were performed using the following parameters: 94&#x000b0;C for 90 secs, 65&#x000b0;C for 3 min and 57&#x000b0;C for 3 min followed by incubation on ice. RT reactions were carried out using SuperScript III reverse transcriptase (Invitrogen) as per manufacturer&#x02019;s instructions. Briefly, a 25 &#x000b5;l RT-positive reaction was set up using 12.5 &#x000b5;l of the annealed product, 4 &#x000b5;l of 25 mM MgCl<sub>2</sub>, 5 &#x000b5;l of 5&#x000d7; First Strand Buffer, 2 &#x000b5;l of 0.1 M DTT, 1 &#x000b5;l of 10 mM dNTPs and 0.8 &#x000b5;l of SuperScript III. The RT-negative reaction was set up with nuclease-free water replacing SuperScript III in the reaction. cDNA synthesis was carried out using the following parameters: 50&#x000b0;C for 50 min and 85&#x000b0;C for 5 min. The resulting cDNA was used for quantitative PCR (qPCR) performed using Sso Fast Evagreen Supermix (BioRad) as per manufacturer&#x02019;s instructions. Briefly, 20 &#x000b5;l reactions were set up, each containing 10 &#x000b5;l Evagreen Supermix, 0.75 &#x000b5;l forward primer (MurTFwd: gaagtcgacgagatgcatgtg, GatDFwd: cgggcactggtacgagac) (10 &#x000b5;M), 0.75 &#x000b5;l reverse primer (MurTRev: atgagcacgtcgtcacagtt, GatDRev: cgcggtggttttcgaatcc) (10 &#x000b5;M), 2 &#x000b5;l cDNA and nucleasefree water. All reactions were incubated in the CFX96 Real-Time PCR detection system (BioRad) using the following parameters: 98&#x000b0;C for 2 min followed by 39 cycles consisting of three steps &#x02013; 98&#x000b0;C for 5 sec, 60&#x000b0;C for 5 sec and 72&#x000b0;C for 5 sec with SYBR Green quantification at the end of each cycle. Melt curve analysis was conducted from 65&#x000b0;C with a gradual increase in 0.5&#x000b0;C increments every 0.05 sec to 95&#x000b0;C with SYBR Green quantification conducted continuously throughout this stage. The raw data was analyzed using the Biorad CFX Manager 3.0 Software (BioRad). Three independent biological replicates were assessed.</p></div><div id="s2_4" class="sec"><h3 id="s2_4title">2.4. Cell envelope labeling</h3><p class="p p-first-last">Fluorogenic probes used in this study include (i) Tetramethylrhodamine [TAMRA]-L-Ala-D-Gln-L-Lys-D-Ala [TetraFI-1]; (ii) non-amidated TAMRA-L-Ala-D-Glu-L-Lys-D-Ala [TetraFI-2]), (iii) TAMRA-D-Alanine (TADA), (iv) HCC-D-Alanine (HADA), (v) Alkyne-D-Alanine-D-Alanine (AlkDADA) and (vi) BODIPY-FL vancomycin. <em>M. smegmatis</em> mc<sup>2</sup>155 derivative strains grown to exponential phase were labelled with 500 &#x000b5;M TAMRA-L-Ala-D-Gln-L-Lys-D-Ala, TAMRA-L-Ala-D-Glu-L-Lys-D-Ala, TADA or HADA and analyzed for probe incorporation by fluorescence microscopy and time-lapse fluorescence microscopy. <em>M. smegmatis</em> mc<sup>2</sup>155 derivative strains grown to exponential phase were labelled with 2 mM AlkDADA and the probe was detected by click chemistry as previously described (<a href="#B23" rid="B23" class=" bibr popnode">Liechti et&#x000a0;al., 2014</a>). Briefly, 1 mM AlkDADA was added to cells grown to an OD<sub>600nm</sub> of 0.6 and collected by centrifugation at 12 470 <em>Xg</em> followed by resuspension in 2 ml of Middlebrook 7H9 broth. The cells were then incubated for 2 hours at 37&#x000b0;C with shaking at 100 rpm. Subsequently, the cells were washed three times with PBSTB buffer (1X PBS, 0.05% BSA and 0.01% Tween20) and fixed for 10 minutes in 1 ml of 2% formaldehyde diluted in 1X PBS. Thereafter, the cells were centrifuged at 12 470 x<em>g</em> for 5 ml and the supernatant was discarded. The click chemistry reaction was conducted as follows: 920 &#x000b5;l of PBSTB buffer was used to resuspend the cells and 20 &#x000b5;l of CuSO<sub>4</sub> (50 mM), 20 &#x000b5;l of TBTA (tris-(benzyltriazolylmethyl) amine) (6.4 mM), 20 &#x000b5;l of sodium ascorbate (60 mM) and 20 &#x000b5;l of an Azido probe (1 mM diluted in DMSO) were added, in the order stated, to the cells and mixed gently. The cells were further incubated at room temperature for 45 minutes and subsequently washed three times with 1X PBS. Thereafter, 5 &#x000b5;l of the cells was spotted onto glass slides which were visualized with the Nikon TI20 fluorescence microscope and the images taken were analyzed with the Fiji software (ImageJ version 1.47n [NIH]). Any image manipulation (brightness, contrast etc.) was applied to the entire image equally. Exponential-phase cultures of <em>M. smegmatis</em> mc<sup>2</sup>155 derivative strains were independently labelled with (100 &#x000b5;g/ml) BODIPY-FL vancomycin for 3 hours, which was subsequently washed out, and analyzed by flow cytometry and fluorescence microscopy for probe incorporation. Three independent biological repeats were assessed.</p></div><div id="s2_5" class="sec"><h3 id="s2_5title">2.5. Time-lapse microscopy</h3><p class="p p-first-last">
<em>M. smegmatis</em> mc<sup>2</sup>155 derivative strains were grown in microfluidic devices as previously described (<a href="#B9" rid="B9" class=" bibr popnode">Dhar and Manina, 2015</a>). Briefly, the bacterial cultures were grown to exponential phase in culture flasks at 37&#x000b0;C. Clumps of bacteria were removed by concentrating the cells and filtering through a 5 &#x003bc;m filter. The bacteria were seeded into a microfluidic device and imaged on a Delta vision Personal DV imaging system (GE HealthCare Life Sciences) using a 100x objective (Olympus Plan Semi Apochromat, 1.3 NA). The images were acquired at 10 min intervals using a CoolSnap HQ2 camera. Middlebrook 7H9 medium with or without 200 ng/mL Anhydrotetracycline (ATc+) and supplemented with FDAAs (labeling for 30 min) was circulated through the device at a flow rate of 25 &#x003bc;l/min. Images were analyzed and processed using Softworx (Version 4.1; Applied Precision, GE HealthCare) or ImageJ (Version 1.47n; NIH). Three independent biological repeats were assessed.</p></div><div id="s2_6" class="sec"><h3 id="s2_6title">2.6. Ethidium bromide diffusion assay to monitor cell permeability</h3><p class="p p-first-last">
<em>M. smegmatis</em> mc<sup>2</sup>155 derivative strains were grown in 50&#x02009;mL 7H9 medium at 37&#x000b0;C to an OD<sub>600nm</sub>&#x02009;= 0.6. Cultures were centrifuged at 4500&#x02009;rpm for 10&#x02009;minutes and the supernatant was discarded. The pellet was then washed in 1X PBS. Thereafter, the pellet was resuspended in 7H9 containing 0.4% glucose and the OD<sub>600nm</sub> was adjusted to 0.4. A 95&#x02009;&#x000b5;l aliquot of bacterial suspension was placed into flat bottom, black 96-well plates (Thermo Scientific), followed by addition of ethidium bromide to final concentration of 10&#x02009;&#x000b5;g/mL. Fluorescence was immediately measured with an excitation and emission wavelength of 530&#x02009;nm and 585&#x02009;nm respectively, every 60&#x02009;seconds for 60&#x02009;minutes at 37&#x000b0;C. Three independent biological repeats were assessed.</p></div><div id="s2_7" class="sec"><h3 id="s2_7title">2.7. Lysozyme minimum inhibitory concentration determination</h3><p class="p p-first-last">To determine the lysozyme MIC, cells were grown overnight at 37&#x000b0;C to an OD<sub>600nm</sub> of 0.6 in 7H9 medium. Cultures were diluted 1:20 in 7H9 medium. Thereafter, a two-fold dilution series was setup for each strain in a clear, 96-well, round-bottom plate as follows: 50 &#x003bc;l of 7H9 broth was aliquoted into each well, except for the top and bottom rows of 8 wells which served as vehicle-plus-media only control and media only control wells. Lysozyme was added to the first column, starting at 2.5 mg/ml in a final volume of 100 &#x003bc;l and then diluted two-fold from the first column (column 1) to the last column (column 12). For the last column, 50 &#x003bc;l was discarded, so that all wells contained a final volume of 50 &#x003bc;l. After dilution of lysozyme and controls (media only or vehicle-plus-media), 50 &#x003bc;l of cells were added to each well. The plates were sealed and incubated at 37&#x000b0;C. After incubating plates at 37&#x000b0;C for 3 days, 15 &#x003bc;l of alamarBlue&#x02122; cell viability reagent (Invitrogen) was added to each well and further incubated at 37&#x000b0;C for 3 hours). Cell viability, and subsequent determination of MIC, was measured by assessing the fluorescence of reduced alamarBlue&#x02122; in a plate reader (excitation and emission at 530&#x02013;560 and 590 nm, respectively). Three independent biological repeats were assessed.</p></div><div id="s2_8" class="sec"><h3 id="s2_8title">2.8. Western blotting</h3><p class="p p-first-last">For analysis of the phosphorylation status of PknB-FLAG in CRISPRi modified strains, cytoplasmic and membrane fractions of cell lysates from MurT-GatD depleted cells grown in media supplemented with a range of ATc concentrations (0 ng/ml-500 ng/ml) were separated on SDS-PAGE gels prepared with the TGX FastCast Acrylamide kit (BioRad), as per manufacturer&#x02019;s instructions. Proteins were transferred to a Immune-Blot PVDF membrane (BioRad) using a Trans-Blot turbo transfer system (BioRad), as per manufacturer&#x02019;s instructions. For immunological detection, the membrane was first washed three times with Tris buffered saline with Tween 20 (TBST) for 5 min each. The membrane was then incubated for 1 hour in blocking solution containing a primary monoclonal anti-FLAG M2 (&#x003b1;-FLAG) antibody (Sigma-Aldrich) at a final concentration of 10 &#x000b5;g/ml or an anti-phosphothreonine-peroxidase (&#x003b1;-P-Thr) antibody (Sigma-Aldrich) at a dilution of 1:50. Thereafter, the membrane was washed three times with TBST for 5 min each. For detection of the primary monoclonal anti-FLAG M2, the membrane was then incubated in blocking solution containing the secondary antibody, rabbit anti-mouse IgG-peroxidase antibody (Sigma-Aldrich) at a dilution of 1:80000. This was followed by three TBST washes for 5 min each. The membrane was transferred to a hybridization bag, treated with a chemiluminescent peroxidase substrate (CPS) and incubated at room temperature for 5 &#x02013; 10 min. All excess substrate was removed and the membrane was exposed to X-ray film for 30&#x02013; 60 secs. The X-ray films were developed in a dark room manually using the Axim developing and fixing solutions (Axim X-Ray industrial and medical).</p></div><div id="s2_9" class="sec"><h3 id="s2_9title">2.9. Peptidoglycan extraction and labelling with an amide reactive dye</h3><p class="p p-first-last">PG was extracted as previously described (<a href="#B38" rid="B38" class=" bibr popnode">Senzani et&#x000a0;al., 2017</a>). Briefly, wildtype <em>M. smegmatis</em> mc<sup>2</sup>155 was grown to OD<sub>600nm</sub> of 2 and the cells were then harvested by centrifugation at 3500 <em>X</em>g for 10 min and resuspended in phosphate-buffered saline (PBS, pH 7.2). The cells were then lysed with a French press (Constant Systems). Insoluble material was obtained by centrifugation at 4000 <em>X</em>g for 30 min. The pellet was then resuspended in PBS containing 2% SDS and incubated at room temperature for 1 hour, then in PBS containing 2 mg/ml proteinase K and 2% SDS at 37&#x000b0;C for 24 hours and finally in PBS containing 2% SDS at 90&#x000b0;C for 1 hour. The extracted cell wall material was lyophilized, weighed and a 100 &#x000b5;g/ml of lyophilized cell wall material was resuspended in PBS and digested with 0.1 mg/ml mutanolysin for 24 hours. The digested material was harvested at 13000&#x02009;<em>Xg</em> for 3&#x02009;min, washed thrice with PBS. The pellet was resuspended in 500 &#x000b5;l PBS and labelled with 100 &#x000b5;g/ml of Alexa Fluor 488 NHS Ester (Sigma-Aldrich) for 3 hours.). The CytoFLEX flow cytometer (Beckman Coulter) was used for analysis of the labelled PG samples (100 &#x000b5;l per sample) in the FITC channel (excitation/emission maxima =494/517 nm). Three independent biological repeats were assessed.</p></div><div id="s2_10" class="sec sec-last"><h3 id="s2_10title">2.10. Flow cytometric analysis of stained mycobacterial cells</h3><p class="p p-first-last">Flow cytometry was used to analyze the labelling pattern of wildtype <em>M. smegmatis</em> mc<sup>2</sup>155, Mtb H37Rv or derivative genetically modified strains. Briefly, cells were grown in 5 ml of Middlebrook 7H9 broth supplemented with the appropriate antibiotics at 37&#x000b0;C with shaking at 100 rpm to an OD<sub>600nm</sub> of 0.6. Thereafter, 200 &#x000b5;l of labelled cell cultures were transferred to a flat bottom, transparent 96 well plate (Thermo Scientific). The CytoFLEX flow cytometer (Beckman Coulter) was used for analysis of the labelled cells by detecting TAMRA signal in the PE-A channel (excitation/emission maxima = 546/579 nm) or the BODIPY-FL signal in the FITC channel (excitation/emission maxima =502/511 nm). The gain for the PE-A channel was reduced from 370 to 120 for all samples tested. Three independent biological repeats were assessed.</p></div></div><div id="s3" class="tsec sec"><h2 class="head no_bottom_margin" id="s3title">3. Results</h2><div id="s3_1" class="sec sec-first"><h3 id="s3_1title">3.1. Mycobacteria amidate the D-glutamate residues of PG for PG cross-linking</h3><p class="p p-first-last">Recently, the mycobacterial MurT-GatD complex was identified and characterized (<a href="#B26" rid="B26" class=" bibr popnode">Maitra et&#x000a0;al., 2021</a>; <a href="#B39" rid="B39" class=" bibr popnode">Silveiro et&#x000a0;al., 2023</a>). In Mtb, the genes Rv3712-Rv3713 encode for the mycobacterial MurT and GatD homologs, respectively. Rv3712 and Rv3713 were predicted to be essential for <em>in vitro</em> growth of Mtb by Himar1 transposon mutagenesis and sequencing (TnSeq), are highly conserved among mycobacteria and physically interact to form an enzyme complex required for amidation of the D-Glu residues of PG for PG cross-linking (<a href="#B37" rid="B37" class=" bibr popnode">Sassetti et&#x000a0;al., 2003</a>; <a href="#B26" rid="B26" class=" bibr popnode">Maitra et&#x000a0;al., 2021</a>). To confirm that mycobacteria require PG amidation for cross-linking <em>in vivo</em>, a pair of fluorescent PG stem peptide mimics bearing either D-Gln or D-Glu at position 2 of the tetrapeptide probe [TetraFl-1 and TetraFl-2, respectively (<a href="/pmc/articles/PMC10484409/figure/f1/" target="figure" class="fig-table-link figpopup" rid-figpopup="f1" rid-ob="ob-f1" co-legend-rid="lgnd_f1"><span>
<strong>Figures&#x000a0;1B, C</strong>
</span></a>), which report on the activity of LDTs (<a href="#B35" rid="B35" class=" bibr popnode">Pidgeon et&#x000a0;al., 2019</a>)] were used. We hypothesized that the lack of amidation in the D-Glu probe at position 2 would preclude stable incorporation into mycobacterial PG. As expected, D-Gln bearing tetrapeptide probes labelled <em>M. smegmatis</em> and Mtb cells, as assessed by microscopy and flow cytometry, while the D-Glu carrying tetrapeptide probe was not incorporated (<a href="/pmc/articles/PMC10484409/figure/f1/" target="figure" class="fig-table-link figpopup" rid-figpopup="f1" rid-ob="ob-f1" co-legend-rid="lgnd_f1"><span>
<strong>Figures&#x000a0;1A-C</strong>
</span></a>). These data confirmed that for <em>in vivo</em> PG cross-linking in mycobacteria, prior D-Glu amidation of PG precursors by the MurT-GatD complex is required.</p></div><div id="s3_2" class="sec"><h3 id="s3_2title">3.2. MurT-rseGFP and mRFP-GatD co-localize at sites of PG biosynthesis</h3><p class="p p-first">We next investigated whether the MurT-GatD enzymes associated with PG amidation spatially localize to areas of new PG synthesis. Mycobacteria are known to spatially localize the major cell wall biosynthesis processes to the cell poles during cell extension (<a href="#B2" rid="B2" class=" bibr popnode">Aldridge et&#x000a0;al., 2012</a>; <a href="#B5" rid="B5" class=" bibr popnode">Botella et&#x000a0;al., 2017b</a>). In mycobacteria, the tropomyosin-like protein, DivIVA, which directs polar growth interacts with enzymes which facilitate early steps of cell wall biosynthesis at the cell poles forming a complex of enzymes termed the elongosome, which facilitates polar cell wall synthesis (<a href="#B27" rid="B27" class=" bibr popnode">Meniche et&#x000a0;al., 2014</a>). To investigate whether the MurT-GatD complex also traffics to these locales in mycobacteria, we constructed a merodiploid <em>M. smegmatis</em> strain, mc<sup>2</sup>155::MurT-rseGFP+mRFP-GatD, wherein the native <em>murT</em> gene (MSMEG_6276) was fused to rseGFP and the native <em>gatD</em> (MSMEG_6277) was fused to mRFP. These were introduced at the <em>attB</em> and <em>t-RNA</em> glycine phage integration sites in the genome, respectively. <em>M. smegmatis</em> strains carrying untagged rseGFP and mRFP alleles (mc<sup>2</sup>155::rseGFP and mc<sup>2</sup>155::mRFP) were also generated in a similar manner to serve as controls. Microscopic analysis of the control strains revealed diffuse and low rseGFP and mRFP signals (<a href="/pmc/articles/PMC10484409/figure/f2/" target="figure" class="fig-table-link figpopup" rid-figpopup="f2" rid-ob="ob-f2" co-legend-rid="lgnd_f2"><span>
<strong>Figure&#x000a0;2A</strong>
</span></a>); whereas microscopic analysis of mc<sup>2</sup>155::MurT-rseGFP+mRFP-GatD revealed that MurT-rseGFP and mRFP-GatD co-localize at the cell poles and the septum, in addition a minimal co-localization signal was also observed at the side-wall (<a href="/pmc/articles/PMC10484409/figure/f2/" target="figure" class="fig-table-link figpopup" rid-figpopup="f2" rid-ob="ob-f2" co-legend-rid="lgnd_f2"><span>
<strong>Figures&#x000a0;2A, B</strong>
</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f2" co-legend-rid="lgnd_f2"><a href="/pmc/articles/PMC10484409/figure/f2/" target="figure" rid-figpopup="f2" rid-ob="ob-f2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140530522817648"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10484409_fcimb-13-1205829-g002.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fcimb-13-1205829-g002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10484409/bin/fcimb-13-1205829-g002.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140530522817648"><a target="object" rel="noopener" href="/pmc/articles/PMC10484409/figure/f2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_f2"><div><a class="figpopup" href="/pmc/articles/PMC10484409/figure/f2/" target="figure" rid-figpopup="f2" rid-ob="ob-f2">Figure&#x000a0;2</a></div><!--caption a7--><div class="caption"><p>MurT-rseGFP and mRFP-GatD co-localize at cell poles and MurT-rseGFP co-localizes with mature PG. <strong>(A)</strong> Micrographs showing MurT-rseGFP and mRFP-GatD localization at the cell poles and septum. Also shown are the rseGFP and mRFP control strains displaying diffused rseGFP and mRFP signals. <strong>(B)</strong> Graph showing the localization patterns of MurT-rseGFP and mRFP-GatD in <em>M. smegmatis</em> mc<sup>2</sup>155. A hundred cells (n=100) were analyzed for each strain and the lines represent the average of the fluorescence intensity across the cells. <strong>(C)</strong> Time-lapse microscopic analysis of MurT-rseGFP in cells labelled with TADA (shown by red arrow) grown in 7H9 media supplemented with hygromycin, kanamycin and TADA (labeling performed for 30 min). MurT-rseGFP localizes at the cell poles (green arrows) and also with maturing PG (yellow arrows). Time-stamp is in hours. Scale bar is 5 &#x000b5;m.</p></div></div></div></div><div id="s3_3" class="sec"><h3 id="s3_3title">3.3. MurT-rseGFP co-localizes with maturing PG during cell growth</h3><p class="p p-first-last">As recent studies suggest that mycobacteria remodel side-wall PG into mature 3-3 cross-linked PG and also repair the side-wall PG when exposed to cell wall damaging agents (<a href="#B3" rid="B3" class=" bibr popnode">Baranowski et&#x000a0;al., 2018</a>; <a href="#B15" rid="B15" class=" bibr popnode">Garc&#x000ed;a-Heredia et&#x000a0;al., 2018</a>), and as we also observed a signal of MurT-rseGFP and mRFP-GatD co-localization at the side-wall, we hypothesized that incorporation of PG precursors into the side-wall for PG remodelling would also require prior amidation by the MurT-GatD complex. To investigate this, spatio-temporal localization of MurT-rseGFP during cell growth was monitored by time-lapse microscopy of MurT-rseGFP cells labelled with a FDAA probe (TADA) during growth (<a href="/pmc/articles/PMC10484409/figure/f2/" target="figure" class="fig-table-link figpopup" rid-figpopup="f2" rid-ob="ob-f2" co-legend-rid="lgnd_f2"><span>
<strong>Figure&#x000a0;2C</strong>
</span></a>, <a href="#SF1" rid="SF1" class=" supplementary-material">
<strong>Supplementary Figure S1</strong>
</a> and <a href="#SM1" rid="SM1" class=" supplementary-material">
<strong>movies S1</strong></a>, <a href="#SM2" rid="SM2" class=" supplementary-material"><strong>S2</strong>
</a>). MurT-rseGFP co-localized with TADA labelling of the cell poles. As the cells expanded, the TADA label (red) was detected at the side-wall (representing mature PG) and the MurT-rseGFP (green) fluorescence signal was observed at the cell poles and also at the side-wall, co-localizing (yellow signal) with the TADA label representing maturing PG (<a href="/pmc/articles/PMC10484409/figure/f2/" target="figure" class="fig-table-link figpopup" rid-figpopup="f2" rid-ob="ob-f2" co-legend-rid="lgnd_f2"><span>
<strong>Figure&#x000a0;2C</strong>
</span></a>, <a href="#SF1" rid="SF1" class=" supplementary-material">
<strong>Supplementary Figure S1B</strong>
</a> and <a href="#SM1" rid="SM1" class=" supplementary-material">
<strong>movies S1</strong></a>, <a href="#SM2" rid="SM2" class=" supplementary-material"><strong>S2</strong>
</a>). The resulting yellow signal (i.e. a mixture of MurT-rseGFP [green] and TADA signal [red]) suggested that PG remodelling at the side-wall also features cytoplasmic MurT-GatD facilitated PG precursor amidation.</p></div><div id="s3_4" class="sec"><h3 id="s3_4title">3.4. MurT-GatD are essential for growth</h3><p class="p p-first">Next, we attempted to delete <em>murT</em> from the genome of wildtype <em>M. smegmatis</em> mc<sup>2</sup>155 to confirm the essentiality of this enzyme for mycobacterial viability as previously suggested (<a href="#B7" rid="B7" class=" bibr popnode">De Wet et&#x000a0;al., 2020</a>). Deletion of <em>murT</em> was only achieved in the presence of a second copy of <em>murT</em> expressed from a <em>tetO</em> promoter, introduced into <em>M. smegmatis</em> mc<sup>2</sup>155 by an episomal shuttle vector (<a href="#SF1" rid="SF1" class=" supplementary-material">
<strong>Supplementary Figure S2</strong>
</a>). This was consistent with prior reports predicting essentiality of <em>murT</em> and <em>gatD</em> (<a href="#B11" rid="B11" class=" bibr popnode">Dragset et&#x000a0;al., 2019</a>). To assess the role of the MurT-GatD complex in growth of <em>M. smegmatis</em> mc<sup>2</sup>155, a CRISPRi based repression system was used to titrate levels of <em>murT</em> and <em>gatD</em> (<a href="/pmc/articles/PMC10484409/figure/f3/" target="figure" class="fig-table-link figpopup" rid-figpopup="f3" rid-ob="ob-f3" co-legend-rid="lgnd_f3"><span>
<strong>Figure&#x000a0;3A</strong>
</span></a>). The experiment was conducted in cells labelled with fluorescent D-amino acids (FDAAs) which are incorporated into existing PG by LDTs during PG remodeling (<a href="/pmc/articles/PMC10484409/figure/f3/" target="figure" class="fig-table-link figpopup" rid-figpopup="f3" rid-ob="ob-f3" co-legend-rid="lgnd_f3"><span>
<strong>Figure&#x000a0;3B</strong>
</span></a>), and this allowed us to study the dynamics of PG remodelling during MurT-GatD depletion. A previously reported highly effective short-guide RNA (sg-RNA; 5&#x02019;-GAGCTGATCACGCGACAGAT-3&#x02019;) (<a href="#B7" rid="B7" class=" bibr popnode">De Wet et&#x000a0;al., 2020</a>), targeting the 5&#x02019; region of <em>murT</em> was used as a guide for CRISPRi dCas9 mediated inhibition of <em>murT</em>-<em>gatD</em> transcription. Sequential labelling, first with TADA, followed by HADA enabled the study of PG remodelling dynamics (<a href="/pmc/articles/PMC10484409/figure/f3/" target="figure" class="fig-table-link figpopup" rid-figpopup="f3" rid-ob="ob-f3" co-legend-rid="lgnd_f3"><span>
<strong>Figure&#x000a0;3C</strong>
</span></a>). Using this approach with mc<sup>2</sup>155::CRISPRi-MurT-GatD cells without CRISPRi induction (i.e. ATc-) showed polar incorporation of new cell wall material as previously demonstrated (<a href="#B2" rid="B2" class=" bibr popnode">Aldridge et&#x000a0;al., 2012</a>), with mature cell wall material localizing to the lateral axis (<a href="/pmc/articles/PMC10484409/figure/f3/" target="figure" class="fig-table-link figpopup" rid-figpopup="f3" rid-ob="ob-f3" co-legend-rid="lgnd_f3"><span>
<strong>Figures&#x000a0;3C, D</strong>
</span></a>, <a href="#SF1" rid="SF1" class=" supplementary-material">
<strong>Supplementary Figure S3</strong>
</a> and <a href="#SM3" rid="SM3" class=" supplementary-material">
<strong>movies S3</strong></a>-<a href="#SM6" rid="SM6" class=" supplementary-material"><strong>S6</strong>
</a>). To assess the requirement for MurT-GatD in PG biosynthesis, the CRISPRi platform in mc<sup>2</sup>155::CRISPRi-MurT-GatD was induced (with ATc) to deplete the transcripts encoding these enzymes. A significant reduction in transcripts was detected using qRT-PCR (<a href="/pmc/articles/PMC10484409/figure/f3/" target="figure" class="fig-table-link figpopup" rid-figpopup="f3" rid-ob="ob-f3" co-legend-rid="lgnd_f3"><span>
<strong>Figure&#x000a0;3E</strong>
</span></a>). Depletion of MurT and GatD resulted in reduction of growth, with cells being shorter relative to cells expressing these enzymes (<a href="/pmc/articles/PMC10484409/figure/f3/" target="figure" class="fig-table-link figpopup" rid-figpopup="f3" rid-ob="ob-f3" co-legend-rid="lgnd_f3"><span>
<strong>Figures&#x000a0;3F-H</strong>
</span></a>, <a href="#SF1" rid="SF1" class=" supplementary-material">
<strong>Supplementary Figure S4</strong>
</a> and <a href="#SM7" rid="SM7" class=" supplementary-material">
<strong>movies S7</strong></a>-<a href="#SM10" rid="SM10" class=" supplementary-material"><strong>S10</strong>
</a>). However, the FDAA labelling pattern did not significantly differ upon gene depletion. Considering this, we next investigated polar elongation speed by measuring the region of unlabelled polar extension over time after labelling with FDAA, with the point of previous FDAA labelling used as a fiducial marker (<a href="/pmc/articles/PMC10484409/figure/f3/" target="figure" class="fig-table-link figpopup" rid-figpopup="f3" rid-ob="ob-f3" co-legend-rid="lgnd_f3"><span>
<strong>Figure&#x000a0;3I</strong>
</span></a>). Cells depleted of MurT-GatD displayed a significantly reduced polar elongation speed, compared to cells replete with these enzymes, pointing to a role for PG amidation in regulation of cell elongation (<a href="/pmc/articles/PMC10484409/figure/f3/" target="figure" class="fig-table-link figpopup" rid-figpopup="f3" rid-ob="ob-f3" co-legend-rid="lgnd_f3"><span>
<strong>Figure&#x000a0;3J</strong>
</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f3" co-legend-rid="lgnd_f3"><a href="/pmc/articles/PMC10484409/figure/f3/" target="figure" rid-figpopup="f3" rid-ob="ob-f3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140530485185120"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10484409_fcimb-13-1205829-g003.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fcimb-13-1205829-g003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10484409/bin/fcimb-13-1205829-g003.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140530485185120"><a target="object" rel="noopener" href="/pmc/articles/PMC10484409/figure/f3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_f3"><div><a class="figpopup" href="/pmc/articles/PMC10484409/figure/f3/" target="figure" rid-figpopup="f3" rid-ob="ob-f3">Figure&#x000a0;3</a></div><!--caption a7--><div class="caption"><p>Depletion of MurT and GatD reduces cell growth. <strong>(A)</strong> Schematic representation of the binding region of the sgRNA used for CRISPRi mediated repression of the <em>murT</em>-<em>gatD</em> operon. <strong>(B)</strong> Schematic representation of MurT-GatD mediated PG precursor amidation. <strong>(C)</strong> Schematic representation of labelling of mycobacterial cells with FDAAs for analysis of cell growth. <strong>(D)</strong> Time-lapse microscopy analysis of <em>M. smegmatis</em> CRISPRi-MurT-GatD cells without CRISPRi activation. <strong>(E)</strong> qPCR analysis of <em>murT</em> and <em>gatD</em> expression during normal conditions or during gene knockdown conditions. Shown are the <em>murT</em> and <em>gatD</em> transcript levels normalized to <em>sigA</em>. <strong>(F)</strong> Time-lapse microscopy analysis of <em>M. smegmatis</em> CRISPRi-MurT-GatD cells with CRISPRi activation. <strong>(G)</strong> The average single-cell doubling time of cells expressing or not expressing MurT and GatD analysed by time-lapse microscopy. Depletion of MurT and GatD increases the doubling time of the cells. <strong>(H)</strong> Violin plots showing the cell length of cells expressing MurT-GatD compared to cells depleted of MurT-GatD (n=300 per experiment). <strong>(I)</strong> Schematic representation of how cell elongation was measured in FDAA labelled cells expressing or not expressing MurT and GatD with the nearest FDAA signal to the old cell pole as fiducial marker for cell pole elongation over-time. <strong>(J)</strong> Scatter plot showing the polar elongation speed of cells expressing or not expressing MurT and GatD (n=100 cells for each experiment. Statistical analysis was conducted using Student t-test. *: p-value &#x0003c;0.05. sgRNA &#x02013; short guide RNA. Scale bar is 5 &#x000b5;m.</p></div></div></div></div><div id="s3_5" class="sec"><h3 id="s3_5title">3.5. PG cross-linking in mycobacteria requires prior amidation of D-glutamate in lipid II</h3><p class="p p-first">As MurT-GatD depleted cells did not lyse or lose their inherent shape (<a href="/pmc/articles/PMC10484409/figure/f3/" target="figure" class="fig-table-link figpopup" rid-figpopup="f3" rid-ob="ob-f3" co-legend-rid="lgnd_f3"><span>
<strong>Figure&#x000a0;3F</strong>
</span></a>), we sought to investigate whether depletion of MurT and GatD in mycobacterial cells would result in perturbed PG biosynthesis using an alkyne functionalized fluorescent dipeptide (D-Alanine-D-Alanine) probe AlkDADA (<a href="/pmc/articles/PMC10484409/figure/f4/" target="figure" class="fig-table-link figpopup" rid-figpopup="f4" rid-ob="ob-f4" co-legend-rid="lgnd_f4"><span>
<strong>Figure&#x000a0;4A</strong>
</span></a>) (<a href="#B23" rid="B23" class=" bibr popnode">Liechti et&#x000a0;al., 2014</a>). AlkDADA is linked to a lipid II precursor during cell growth to form labelled lipid II which, if incorporated into the existing PG, can be detected on the cell surface by click chemistry (<a href="#B23" rid="B23" class=" bibr popnode">Liechti et&#x000a0;al., 2014</a>). As expected, cells expressing MurT-GatD displayed polar labelling with AlkDADA and reduced labeling at the side-wall indicative of insertion of the labelled PG precursors at the cell poles (<a href="/pmc/articles/PMC10484409/figure/f4/" target="figure" class="fig-table-link figpopup" rid-figpopup="f4" rid-ob="ob-f4" co-legend-rid="lgnd_f4"><span>
<strong>Figures&#x000a0;4B, C</strong>
</span></a>). In contrast, depletion of MurT-GatD caused diffused labeling with AlkDADA in majority of cells analyzed indicative of reduced PG remodelling along the side-wall (<a href="/pmc/articles/PMC10484409/figure/f4/" target="figure" class="fig-table-link figpopup" rid-figpopup="f4" rid-ob="ob-f4" co-legend-rid="lgnd_f4"><span>
<strong>Figures&#x000a0;4B, C</strong>
</span></a>). Side-wall PG remodelling involves the formation of 3-3 cross-links from 4-3 cross-links by LDTs (<a href="#B15" rid="B15" class=" bibr popnode">Garc&#x000ed;a-Heredia et&#x000a0;al., 2018</a>) and as a result, the D-Alanine-D-Alanine (DADA) dipeptide is removed from the PG. In Chlamydia, this compound has been shown to be primarily incorporated into lipid II (<a href="#B23" rid="B23" class=" bibr popnode">Liechti et&#x000a0;al., 2014</a>). Similarly, in mycobacteria, it has been shown that AlkDADA can be incorporated into lipid II (<a href="#B15" rid="B15" class=" bibr popnode">Garc&#x000ed;a-Heredia et&#x000a0;al., 2018</a>), although it is not yet clear if this is its primary mode of incorporation. Increased AlkDADA signal in MurT-GatD depleted cells is suggestive of perturbed PG remodelling in these cells. To confirm this observation, a fluorescent BODIPY-FL vancomycin probe, which binds the terminal D-Ala-D-Ala (DADA) moiety of the PG precursor, was used to probe PG remodelling in cells depleted of MurT and GatD (<a href="/pmc/articles/PMC10484409/figure/f4/" target="figure" class="fig-table-link figpopup" rid-figpopup="f4" rid-ob="ob-f4" co-legend-rid="lgnd_f4"><span>
<strong>Figure&#x000a0;4D</strong>
</span></a>). Under replete conditions in the absence of ATc, the bacteria exhibited labeling of the poles and septum (<a href="/pmc/articles/PMC10484409/figure/f4/" target="figure" class="fig-table-link figpopup" rid-figpopup="f4" rid-ob="ob-f4" co-legend-rid="lgnd_f4"><span>
<strong>Figure&#x000a0;4E</strong>
</span></a>). However, when the cells were depleted of MurT-GatD following the addition of ATc, the bacteria displayed increased levels of diffused labelling along the lateral axis of the cell wall. This observation was independently confirmed using flow cytometry, which revealed increased labeling of the cells with BODIPY-FL vancomycin as seen by microscopy (<a href="/pmc/articles/PMC10484409/figure/f4/" target="figure" class="fig-table-link figpopup" rid-figpopup="f4" rid-ob="ob-f4" co-legend-rid="lgnd_f4"><span>
<strong>Figures&#x000a0;4E-G</strong>
</span></a>). As the terminal DADA moiety of PG precursors is used as a substrate for cross-linking by PBPs or can be cleaved by D,D-carboxypeptidases during PG remodelling (<a href="#B47" rid="B47" class=" bibr popnode">Vollmer et&#x000a0;al., 2008</a>), the increased labelling with BODIPY-FL vancomycin along the side-wall suggests that PG remodelling is decreased when MurT and GatD are depleted or reflects reduced PG remodelling dynamics by D,D-carboxypeptidases or increased cell wall permeability (<a href="#B32" rid="B32" class=" bibr popnode">Ngadjeua et&#x000a0;al., 2018</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f4" co-legend-rid="lgnd_f4"><a href="/pmc/articles/PMC10484409/figure/f4/" target="figure" rid-figpopup="f4" rid-ob="ob-f4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140530484521840"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10484409_fcimb-13-1205829-g004.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fcimb-13-1205829-g004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10484409/bin/fcimb-13-1205829-g004.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140530484521840"><a target="object" rel="noopener" href="/pmc/articles/PMC10484409/figure/f4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_f4"><div><a class="figpopup" href="/pmc/articles/PMC10484409/figure/f4/" target="figure" rid-figpopup="f4" rid-ob="ob-f4">Figure&#x000a0;4</a></div><!--caption a7--><div class="caption"><p>MurT-GatD depletion causes perturbed PG remodelling, increased cell wall permeability and increased sensitivity to lysozyme. <strong>(A)</strong> Schematic representation of cell wall labeling with AlkDADA showing incorporation of the label into PG precursors during <em>de novo</em> PG biosynthesis. <strong>(B)</strong> Micrographs of <em>M. smegmatis</em> CRISPRi MurT-GatD cells expressing or not expressing MurT-GatD. Inhibition of MurT-GatD expression causes increased side-wall labeling with AlkDADA. <strong>(C)</strong> Graph showing the AlkDADA labeling pattern of cells depleted of MurT-GatD compared to MurT-GatD expressing cells. A hundred cells (n=100) were analyzed for each strain and the lines represent the average of the fluorescence intensity across the cells. Cells with clear increased sidewall signal of AlkDADA were counted <em>vs</em> polar and/or septal labeling in cells not exposed to ATc. <strong>(D)</strong> Schematic representation of cell wall labeling with BODIPY-FL vancomycin showing labeling of PG precursors in the periplasm. <strong>(E)</strong> Micrographs of BODIPY-FL vancomycin labeled MurT-GatD depleted cells in comparison with cells expressing the proteins. Inhibition of MurT-GatD expression causes increased side-wall labeling with BODIPY-FL vancomycin. n=100 labelled cells per experiment. <strong>(F)</strong> Flow cytometry quantification of BODIPY-FL vancomycin labelled MurT-GatD depleted cells in comparison to cells expressing the proteins. <strong>(G)</strong> Graph showing the BODIPY-FL vancomycin labeling pattern of cells depleted of MurT-GatD compared to MurT-GatD expressing cells. A hundred random cells (n=100) were analyzed for each strain and the lines represent the average of the fluorescence intensity across the cells. <strong>(H)</strong> Ethidium bromide uptake into cells depleted of MurT-GatD in comparison to cells expressing the proteins. <strong>(I)</strong> Sensitivity of MurT-GatD depleted cells to lysozyme. The black arrow indicates a two-fold change in susceptibility to lysozyme. <strong>(J)</strong> Micrographs of NADA (3-[(7-Nitro-2,1,3-benzoxadiazol-4-yl)amino]-D-alanine hydrochloride) labelled cells depleted of MurT-GatD and exposed to lysozyme (~80 &#x000b5;g/ml). Lysozyme exposure caused cell death in cells depleted of MurT-GatD. Statistical analysis was conducted using Student <em>t</em>-test. *:p-value &#x0003c; 0.0001. Scale bar is 5 &#x000b5;m.</p></div></div></div><p class="p p-last">To assess whether the reduced PG-cross-linking in MurT-GatD depleted cells is associated with changes in cell wall permeability, we used an ethidium bromide diffusion assay as previously described (<a href="#B38" rid="B38" class=" bibr popnode">Senzani et&#x000a0;al., 2017</a>). Cells depleted of MurT-GatD exhibited increased cell wall permeability, evidenced by increased ethidium bromide fluorescence (<a href="/pmc/articles/PMC10484409/figure/f4/" target="figure" class="fig-table-link figpopup" rid-figpopup="f4" rid-ob="ob-f4" co-legend-rid="lgnd_f4"><span>
<strong>Figure&#x000a0;4H</strong>
</span></a>). We hypothesized that reduced PG cross-linking in MurT-GatD depleted cells would simultaneously cause an increased sensitivity to PG targeting antimicrobials such as lysozyme. A lysozyme minimum inhibitory concentration experiment revealed that MurT-GatD depleted cells were indeed sensitive to killing by lysozyme and microscopic analysis of these cells showed that lysis occurred following exposure while cells expressing MurT-GatD displayed less sensitivity to lysozyme exposure (<a href="/pmc/articles/PMC10484409/figure/f4/" target="figure" class="fig-table-link figpopup" rid-figpopup="f4" rid-ob="ob-f4" co-legend-rid="lgnd_f4"><span>
<strong>Figures&#x000a0;4I, J</strong>
</span></a>). Furthermore, to assess the level of PG amidation in MurT-GatD depleted cells, the PG of lysed MurT-GatD- expressing and depleted cells was extracted and labelled with an amide reactive dye, Alexa Fluor 488 NHS Ester which labels primary amines (R-NH<sub>2</sub>). The labelled PG from MurT depleted cells contained a lower dye signal, suggestive of a decrease in the level of PG amidation (<a href="#SF1" rid="SF1" class=" supplementary-material">
<strong>Supplementary Figure S5</strong>
</a>).</p></div><div id="s3_6" class="sec"><h3 id="s3_6title">3.6. PknB regulates cell growth in response to PG amidation</h3><p class="p p-first">PknB-like kinases have been implicated in the regulation of PG biosynthesis in most Gram-positive bacteria (<a href="#B34" rid="B34" class=" bibr popnode">Pereira et&#x000a0;al., 2011</a>). In mycobacteria, uncross-linked PG is detected by PknB through C-terminal PASTA domains, leading to the autophosphorylation of the N-terminal kinase domain at threonine residues (<a href="/pmc/articles/PMC10484409/figure/f5/" target="figure" class="fig-table-link figpopup" rid-figpopup="f5" rid-ob="ob-f5" co-legend-rid="lgnd_f5"><span>
<strong>Figure&#x000a0;5A</strong>
</span></a>) (<a href="#B20" rid="B20" class=" bibr popnode">Kaur et&#x000a0;al., 2019</a>). In mycobacteria, PknB is essential for cell growth (<a href="#B37" rid="B37" class=" bibr popnode">Sassetti et&#x000a0;al., 2003</a>; <a href="#B6" rid="B6" class=" bibr popnode">Dejesus et&#x000a0;al., 2017</a>) and it was previously demonstrated that its extracellular PASTA domains (PASTA3 and PASTA4) are required for localization at the cell poles and septum for regulation of PG biosynthesis and cell growth (<a href="#B28" rid="B28" class=" bibr popnode">Mir et&#x000a0;al., 2011</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f5" co-legend-rid="lgnd_f5"><a href="/pmc/articles/PMC10484409/figure/f5/" target="figure" rid-figpopup="f5" rid-ob="ob-f5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140530488332784"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10484409_fcimb-13-1205829-g005.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fcimb-13-1205829-g005.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10484409/bin/fcimb-13-1205829-g005.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140530488332784"><a target="object" rel="noopener" href="/pmc/articles/PMC10484409/figure/f5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_f5"><div><a class="figpopup" href="/pmc/articles/PMC10484409/figure/f5/" target="figure" rid-figpopup="f5" rid-ob="ob-f5">Figure&#x000a0;5</a></div><!--caption a7--><div class="caption"><p>Reduced MurT-GatD activity affects PknB activity. <strong>(A)</strong> Schematic representation of the domain structure of PknB and shown are the D-Glu binding residues (red font) and the activation loop with threonine residues (blue font) subject to phosphorylation upon PknB activation. <strong>(B)</strong> Growth curve of <em>M. smegmatis</em> mc<sup>2</sup>155::CRISPRi-MurT-GatD+PknB-FLAG grown in media supplemented with 100-500 ng/ml of ATc. <strong>(C)</strong> SDS-page gels showing expression of PknB-FLAG in the cytoplasmic fraction (CF) and presence of PknB-FLAG also in the membrane fraction (MF). MFs from cells depleted of MurT-GatD were used to assess PknB phosphorylation. <strong>(D)</strong> Western blot analysis of PknB-FLAG phosphorylation from MFs of MurT-GatD depleted cells. MurT-GatD depletion causes increased phosphorylation of PknB. <strong>(E)</strong> mRFP-PknB localizes at the cell poles and cell septum (black arrow heads). <strong>(F)</strong> Depletion of MurT and GatD causes diffused mRFP-PknB localization. <strong>(G)</strong> Histogram showing the frequency of cells showing polar and septal or diffused mRFP-PknB localization (n=100). Cells with a diffused signal of mRFP-PknB were counted <em>vs</em> polar and/or septal localization pattern in cells not exposed to ATc. <strong>(H)</strong> Graph showing the mRFP-PknB localization pattern in cells depleted of MurT-GatD compared to MurT-GatD expressing cells. A hundred random cells were analysed per experimental condition (i.e. ATc- <em>vs</em> ATc+) and the lines represent the average of the fluorescence intensity across the cells. <strong>(I)</strong> Time-lapse micrographs of MurT-GatD expressing and depleted cells exposed to sub-lethal concentration of amxocillin (20 &#x000b5;g/ml) and clavulanate (10 &#x000b5;g/ml). <strong>(J)</strong> Histogram showing the frequency of dead cells with polar bulges, 25 time-lapse micrographs were analysed for each experiment. Scale bar is 5 &#x000b5;m.</p></div></div></div><p class="p p-last">Recently, it was demonstrated that abrogation of D-Gln interacting residues (Ser<sub>556</sub> and Lys<sub>557</sub>) of PknB, which specifically interact with the terminal carboxyl group of D-Gln, results in perturbation of PknB function and localization (<a href="#B20" rid="B20" class=" bibr popnode">Kaur et&#x000a0;al., 2019</a>). Considering this, we hypothesized that the reduced PG amidation resulting from depletion of MurT-GatD would consequently affect PknB function. To investigate this, phosphorylation of a FLAG-tagged PknB was assessed in <em>M. smegmatis</em> depleted of MurT and GatD, using a titration of the ATc inducer [100 ng/ml-500 ng/ml]. Here, we hypothesized that addition of a second copy of <em>pknB</em>, carrying the FLAG-tag, would not materially affect MurT-GatD function. As expected, the MurT-GatD depleted strain (carrying the FLAG-tagged PknB) displayed a ATc dose-dependent reduction in growth (<a href="/pmc/articles/PMC10484409/figure/f5/" target="figure" class="fig-table-link figpopup" rid-figpopup="f5" rid-ob="ob-f5" co-legend-rid="lgnd_f5"><span>
<strong>Figure&#x000a0;5B</strong>
</span></a>). These cells were subsequently lysed for collection of membrane fractions to analyze PknB phosphorylation with an anti-phosphothreonine antibody. We first confirmed the expression of PknB-FLAG without CRISPRi induced repression of MurT-GatD (<a href="/pmc/articles/PMC10484409/figure/f5/" target="figure" class="fig-table-link figpopup" rid-figpopup="f5" rid-ob="ob-f5" co-legend-rid="lgnd_f5"><span>
<strong>Figure&#x000a0;5C</strong>
</span></a>). An SDS page analysis of the samples confirmed the presence of a ~70kDa protein band in both the cytoplasmic fraction (CF) and the membrane fraction (MF) corresponding with the expected band size for <em>M. smegmatis</em> mc<sup>2</sup>155 PknB (66.2kDa). MurT-GatD depletion was associated with increased PknB phosphorylation, in a ATc dose dependent manner, as analysed by the band intensity of PknB detected by an anti-phosphothreonine antibody (<a href="/pmc/articles/PMC10484409/figure/f5/" target="figure" class="fig-table-link figpopup" rid-figpopup="f5" rid-ob="ob-f5" co-legend-rid="lgnd_f5"><span>
<strong>Figure&#x000a0;5D</strong>
</span></a>). These results demonstrated that the lack of PG amidation results in PknB dysregulation, which likely affects the downstream activation of PknB substrates and signal for cell growth cessation.</p></div><div id="s3_7" class="sec"><h3 id="s3_7title">3.7. PknB localization is perturbed in MurT-GatD depleted cells</h3><p class="p p-first-last">Since PknB activity is increased in MurT-GatD depleted cells, we next sought to determine the localization of PknB in these cells. To assess this, wildtype PknB was fused to mRFP on the N-terminus and the construct introduced into both the <em>M. smegmatis</em> mc<sup>2</sup>155 and mc<sup>2</sup>155::CRISPRi-MurT-GatD strains. An N-terminal fusion was used to ensure no disruption of the functionality in the C-terminal PASTA domains. The mRFP-PknB localized to the cell poles and septum in mc<sup>2</sup>155::mRFP-PknB and mc<sup>2</sup>155::CRISPRi-MurT-GatD+mRFP-PknB without CRISPRi induction (i.e. ATc-) (<a href="/pmc/articles/PMC10484409/figure/f5/" target="figure" class="fig-table-link figpopup" rid-figpopup="f5" rid-ob="ob-f5" co-legend-rid="lgnd_f5"><span>
<strong>Figure&#x000a0;5E, F</strong>
</span></a>). Upon CRISPRi induction, most of the cells analyzed displayed a similar mRFP-PknB localization pattern as seen in wildtype cells. However, mRFP-PknB also displayed diffused localization in a subpopulation of MurT-GatD depleted cells (<a href="/pmc/articles/PMC10484409/figure/f5/" target="figure" class="fig-table-link figpopup" rid-figpopup="f5" rid-ob="ob-f5" co-legend-rid="lgnd_f5"><span>
<strong>Figures&#x000a0;5E, G, H</strong>
</span></a>), indicative of perturbed PknB function as observed <em>via</em> the analysis of PknB phosphorylation status.</p></div><div id="s3_8" class="sec sec-last"><h3 id="s3_8title">3.8. MurT-GatD depletion causes sensitivity to &#x003b2;-lactam antibiotics</h3><p class="p p-first-last">The importance of lipid II amidation for PG cross-linking in mycobacteria highlights a vulnerability that can be exploited for inhibition of mycobacterial growth using cell wall targeting antibiotics (<a href="#B26" rid="B26" class=" bibr popnode">Maitra et&#x000a0;al., 2021</a>). We hypothesized that increased cell wall permeability as a result of reduced levels of amidated lipid II for PG biosynthesis in the MurT-GatD depleted cells would cause increased sensitivity to &#x003b2;-lactam antibiotics and result in rapid killing of the bacteria. To investigate this, time-lapse microscopic analysis of the growth of the MurT-GatD depleted mc<sup>2</sup>155::CRISPRi-MurT-GatD cells in the presence of a sub-lethal concentration of amoxicillin (AMX) (20 &#x000b5;g/ml) and clavulanate (CLV) (10 &#x000b5;g/ml) was performed. CLV was included to inhibit &#x003b2;-lactamases as <em>M. smegmatis</em> expresses numerous potent &#x003b2;-lactamases (<a href="#B13" rid="B13" class=" bibr popnode">Flores et&#x000a0;al., 2005</a>). AMX-CLV treated MurT-GatD depleted cells were unable to elongate, formed bulges at the cell poles and ultimately lysed. In contrast, cells expressing MurT-GatD were less sensitive to killing by AMX+CLV (<a href="/pmc/articles/PMC10484409/figure/f5/" target="figure" class="fig-table-link figpopup" rid-figpopup="f5" rid-ob="ob-f5" co-legend-rid="lgnd_f5"><span>
<strong>Figures&#x000a0;5I, J</strong>
</span></a>, <a href="#SF1" rid="SF1" class=" supplementary-material">
<strong>Supplementary Figure S6</strong>
</a> and <a href="#SM11" rid="SM11" class=" supplementary-material">
<strong>movies S11</strong></a>, <a href="#SM12" rid="SM12" class=" supplementary-material"><strong>S12</strong>
</a>). Collectively, these results demonstrate the importance of PG amidation for mycobacterial cell growth and highlight the MurT-GatD complex as potential drug targets for TB drug development.</p></div></div><div id="s4" class="tsec sec"><h2 class="head no_bottom_margin" id="s4title">4. Discussion</h2><p class="p p-first">&#x003b2;-lactam antibiotics, such as penicillin, inhibit the critical cross-linking step of PG biosynthesis, facilitated by PBPs, and have shown great success for treatment of a variety of bacterial infections in the clinical setting (<a href="#B19" rid="B19" class=" bibr popnode">Hugonnet et&#x000a0;al., 2009</a>; <a href="#B10" rid="B10" class=" bibr popnode">Diacon et&#x000a0;al., 2016</a>). This suggests that the PG biosynthesis pathway and enzymes involved are vulnerable drug targets that can be exploited for the development of new and more effective antibiotics. To facilitate these efforts, fluorescent probes have been used extensively to study PG biosynthesis in live bacterial cells to elucidate previously unknown functions of PG-associated enzymes, thus enabling further identification of potential drug targets (<a href="#B2" rid="B2" class=" bibr popnode">Aldridge et&#x000a0;al., 2012</a>; <a href="#B23" rid="B23" class=" bibr popnode">Liechti et&#x000a0;al., 2014</a>; <a href="#B4" rid="B4" class=" bibr popnode">Botella et&#x000a0;al., 2017a</a>). <em>Via</em> the use of fluorescent PG stem peptide mimics with amidation modifications on the second amino acid (i.e. D-Gln instead of D-Glu) of the stem peptide which are substrates of LDTs, we confirmed that this modification is required for the PG cross-linking reaction. We further investigated the essentiality of this modification in PG biosynthesis using CRISPRi mediated depletion of the enzymes (the MurT-GatD complex) required for this process. It has been shown that MurT and GatD catalyze the amidation reaction of the D-Glu residue in the peptide stem of PG precursors in <em>S. aureus</em>, <em>S. pneumoniae</em> and mycobacteria (<a href="#B30" rid="B30" class=" bibr popnode">M&#x000fc;nch et&#x000a0;al., 2012</a>; <a href="#B24" rid="B24" class=" bibr popnode">Liu et&#x000a0;al., 2017</a>; <a href="#B29" rid="B29" class=" bibr popnode">Morlot et&#x000a0;al., 2018</a>; <a href="#B33" rid="B33" class=" bibr popnode">N&#x000f6;ldeke et&#x000a0;al., 2018</a>; <a href="#B26" rid="B26" class=" bibr popnode">Maitra et&#x000a0;al., 2021</a>). However, the phenotypic defects associated with the loss of PG amidation due to inactivation of MurT-GatD in mycobacteria remained unclear.</p><p>MurT and GatD are arranged in a bicistronic operon, which is conserved in various bacterial species including mycobacteria. Deletion of the native <em>murT</em> gene using the two step allelic exchange mutagenesis technique was only possible in the presence of a second copy of the <em>murT</em> gene, thus confirming the essentiality of this operon. We further used a CRISPR interference based programmable transcriptional repression platform coupled with time-lapse microscopy to investigate the role of this enzyme complex for mycobacterial cell growth (<a href="#B9" rid="B9" class=" bibr popnode">Dhar and Manina, 2015</a>; <a href="#B36" rid="B36" class=" bibr popnode">Rock et&#x000a0;al., 2017</a>). Time-lapse microscopic analysis of MurT and GatD depleted cells demonstrated that repression of the <em>murT</em>-<em>gatD</em> operon resulted in growth inhibition. It has been previously reported that PG transpeptidases (both D,D-transpeptidases [DDTs] and LDTs) required for PG cross-linking only recognize amidated PG precursors as substrates (<a href="#B49" rid="B49" class=" bibr popnode">Zapun et&#x000a0;al., 2013</a>; <a href="#B35" rid="B35" class=" bibr popnode">Pidgeon et&#x000a0;al., 2019</a>). For mycobacteria, it remains to be shown whether mycobacterial DDTs also require amidated PG precursors to form PG cross-links. However, inhibition of PG precursor amidation in mycobacteria is expected to inhibit PG cross-linking by LDTs. To investigate this, MurT-GatD depleted cells were labelled with various fluorescent PG probes (FDAAs and a vancomycin analogue &#x02013; BODIPY-FL vancomycin) and PG remodelling dynamics were assessed. FDAA labelled MurT-GatD depleted cells did not show changes in PG biosynthesis dynamics relative to cells expressing these enzymes. This could be due to the fact that LDTs require only one of the stem peptides to be amidated (<a href="#B49" rid="B49" class=" bibr popnode">Zapun et&#x000a0;al., 2013</a>). Interestingly, BODIPY-FL vancomycin which binds non-cross-linked PG precursors displayed an abnormal labeling pattern in MurT-GatD depleted cells. The non-cross-linked PG precursors were detected all along the lateral axis of the cell surface (i.e. diffused staining) which suggested that PG cross-linking by PG transpeptidases is reduced in MurT-GatD depleted cells.</p><p>To further study the role of MurT and GatD in mycobacteria, protein localization experiments were performed with rseGFP and mRFP tagged MurT and GatD, respectively. We found that MurT-rseGFP and mRFP-GatD co-localized at the cell poles and cell division septum. In other bacterial species MurT interacts with GatD and activates GatD function (<a href="#B30" rid="B30" class=" bibr popnode">M&#x000fc;nch et&#x000a0;al., 2012</a>; <a href="#B29" rid="B29" class=" bibr popnode">Morlot et&#x000a0;al., 2018</a>). In mycobacteria, both these proteins were previously shown to interact in a mycobacterial protein fragment complementation (MPFC) assay (<a href="#B26" rid="B26" class=" bibr popnode">Maitra et&#x000a0;al., 2021</a>), and also identified by proteomic analysis as part of proteins enriched in the plasma membrane domain of mycobacteria known as the intracellular membrane domain (IMD) localized at the cell poles, which contains enzymes associated with cell wall biosynthesis (<a href="#B18" rid="B18" class=" bibr popnode">Hayashi et&#x000a0;al., 2016</a>). Furthermore, a high throughput imaging screen of a library of fluorescently tagged <em>M. smegmatis</em> proteins revealed a cell pole localization pattern of MurT-Dendra (<a href="#B50" rid="B50" class=" bibr popnode">Zhu et&#x000a0;al., 2021</a>), consistent with the localization pattern we observed in our study. These findings indicate that the MurT-GatD complex also forms part of both the elongosome and divisome complexes, required for synthesis of the cell wall at the poles and the cell division septum, respectively.</p><p>Recently, it was demonstrated that mycobacteria remodel the side-wall during growth (<a href="#B3" rid="B3" class=" bibr popnode">Baranowski et&#x000a0;al., 2018</a>); and Garcia-Heredia et&#x000a0;al. showed that damage to the mycobacterial cell wall as a result of lysozyme exposure also requires side-wall remodeling and repair (<a href="#B3" rid="B3" class=" bibr popnode">Baranowski et&#x000a0;al., 2018</a>; <a href="#B15" rid="B15" class=" bibr popnode">Garc&#x000ed;a-Heredia et&#x000a0;al., 2018</a>). We found, using time-lapse microscopy, that MurT-rseGFP co-localized with TADA, a monopeptide FDAA probe, during cell growth. This suggested that MurT-GatD also localizes with maturing PG. We also hypothesized that if the PG is loosely cross-linked as a result of MurT-GatD depletion, the cell wall would be more permeable as previous studies showed that perturbations in PG remodelling causes increased cell wall permeability (<a href="#B38" rid="B38" class=" bibr popnode">Senzani et&#x000a0;al., 2017</a>). To investigate this, we used an ethidium bromide diffusion assay and showed that depletion of MurT and GatD was associated with increased cell wall permeability. This data demonstrates that amidation of PG plays a crucial role in PG biosynthesis and maturation in mycobacteria (<a href="/pmc/articles/PMC10484409/figure/f6/" target="figure" class="fig-table-link figpopup" rid-figpopup="f6" rid-ob="ob-f6" co-legend-rid="lgnd_f6"><span>
<strong>Figure&#x000a0;6</strong>
</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f6" co-legend-rid="lgnd_f6"><a href="/pmc/articles/PMC10484409/figure/f6/" target="figure" rid-figpopup="f6" rid-ob="ob-f6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140530477213456"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10484409_fcimb-13-1205829-g006.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fcimb-13-1205829-g006.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10484409/bin/fcimb-13-1205829-g006.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140530477213456"><a target="object" rel="noopener" href="/pmc/articles/PMC10484409/figure/f6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_f6"><div><a class="figpopup" href="/pmc/articles/PMC10484409/figure/f6/" target="figure" rid-figpopup="f6" rid-ob="ob-f6">Figure&#x000a0;6</a></div><!--caption a7--><div class="caption"><p>Proposed model for the role of MurT and GatD during PG biosynthesis and cell growth in mycobacteria. Lipid II biosynthesis involves MurT-GatD amidation of D-glutamate of the PG stem peptide. (1) Depletion of MurT-GatD causes decreased lipid II amidation at D-Glu (2) and non-amidated lipid II is not cross-linked into existing PG by PBPs (3). PknB binds uncross-linked PG stem peptides (4) and its cytoplasmic kinase domain auto-phosphorylates and causes inhibition of PG biosynthesis resulting in cessation of cell growth (6).</p></div></div></div><p>We further studied how PknB regulates PG biosynthesis in response to reduced PG cross-linking as a result of depletion of MurT-GatD in <em>M. smegmatis</em>. PknB is known to regulate PG biosynthesis by sensing PG cross-linking through extracellular PASTA domains which also facilitate the localization of PknB to the cell poles and septum (<a href="#B28" rid="B28" class=" bibr popnode">Mir et&#x000a0;al., 2011</a>). Specifically, residues that are located in a &#x003b1;-helix locker between PASTA3 and PASTA4 domains of PknB bind the carboxyl terminal of D-Gln/D-Glu and mDAP residues in the stem peptide of uncross-linked PG precursors and this causes autophosphorylation of the cytoplasmic kinase domain of PknB (<a href="#B20" rid="B20" class=" bibr popnode">Kaur et&#x000a0;al., 2019</a>). The activated PknB kinase domain regulates the function of PG biosynthesis enzymes including MurA, CwlM and MurJ through phosphorylation (<a href="#B43" rid="B43" class=" bibr popnode">Turapov et&#x000a0;al., 2015</a>; <a href="#B42" rid="B42" class=" bibr popnode">Turapov et&#x000a0;al., 2018</a>). We confirmed that depletion of MurT and GatD causes hyper-activation of PknB. Localization experiments of mRFP-PknB in MurT-GatD depleted cells also displayed perturbed mRFP-PknB localization indicative of changes in PknB function in these cells. We next hypothesized that cells depleted of MurT and GatD are highly dependent on PBPs to remain viable. To test this, we exposed the MurT-GatD depleted cells to a sub-lethal concentration of amoxicillin which inhibits the transpeptidase reaction of high molecular weight PBPS in the presence of clavulanate required for inhibition of &#x003b2;-lactamases (<a href="#B19" rid="B19" class=" bibr popnode">Hugonnet et&#x000a0;al., 2009</a>; <a href="#B8" rid="B8" class=" bibr popnode">Dhar et&#x000a0;al., 2015</a>; <a href="#B10" rid="B10" class=" bibr popnode">Diacon et&#x000a0;al., 2016</a>). Cells depleted of MurT and GatD were found to be highly sensitive to amoxicillin-clavulanate killing, showing reduced growth, polar swelling and eventual cell lysis. These data highlight the MurT-GatD complex as a vulnerable target for inhibition of PG biosynthesis.</p><p class="p p-last">Considering that MurT and GatD depletion resulted in reduced PG cross-linking and reduced growth, and that the mycobacterial serine/threonine protein kinase PknB is activated by uncross-linked PG precursors, we propose a model where PknB senses the level of uncross-linked PG as a result of the level of PG amidation caused by activity of MurT and GatD and regulates mycobacterial cell growth (<a href="/pmc/articles/PMC10484409/figure/f6/" target="figure" class="fig-table-link figpopup" rid-figpopup="f6" rid-ob="ob-f6" co-legend-rid="lgnd_f6"><span>
<strong>Figure&#x000a0;6</strong>
</span></a>). Taken together, we offer evidence for the further development of MurT and/or GatD as novel drug targets for TB treatment.</p></div><div id="s5" class="tsec sec"><h2 class="head no_bottom_margin" id="s5title">Data availability statement</h2><p class="p p-first-last">The original contributions presented in the study are included in the article/<a href="#s10" rid="s10" class=" sec">
<strong>Supplementary Material</strong>
</a>. Further inquiries can be directed to the corresponding author.</p></div><div id="s6" class="tsec sec"><h2 class="head no_bottom_margin" id="s6title">Author contributions</h2><p class="p p-first-last">MS and BK conceived the project. MS performed experiments, analyzed data and prepared the figures and manuscript. MP and MV provided fluorescent PG probes. ND assisted with experiment design and supervised the imaging experiments.</p></div><div id="ack-a.y.a" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.y.atitle">Acknowledgments</h2><div class="sec"><p>We are grateful for the support from the South African National Research Foundation, the South African Medical Research Council and the Swiss South Africa Joint Research Program and the Swiss National Science Foundation. We also thank Sloan Seigrist for provision of the AlkDADA probe.</p></div></div><div id="funding-group-a.w.b.r" class="tsec sec"><h2 class="head no_bottom_margin" id="funding-group-a.w.b.rtitle">Funding Statement</h2><p>This work was supported by funding from an International Early Career Scientist Award from the Howard Hughes Medical Institute (to BK), the South African National Research Foundation (to BK, MS), the South African Medical Research Council (to BK, MS) the Centre for Aids Prevention Research in South Africa (CAPRISA, to BK). We are grateful for the support from the Swiss South Africa Joint Research Program and the Swiss National Science Foundation (Project IZLSZ3_170912 to BK and ND). VIDO receives operational funding from the Government of Saskatchewan through Innovation Saskatchewan and the Ministry of Agriculture and from the Canada Foundation for Innovation through the Major Science Initiatives for its CL3 facility.</p></div><div id="s8" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="s8title">Conflict of interest</h2><!--/article/back/sec/--><p class="p p-first-last">The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p></div><div id="s9" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="s9title">Publisher&#x02019;s note</h2><!--/article/back/sec/--><p class="p p-first-last">All claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.</p></div><div id="s10" class="tsec bk-sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="s10title">Supplementary material</h2><!--/article/back/sec/--><p class="p p-first">The Supplementary Material for this article can be found online at: <a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1205829/full#supplementary-material" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1205829/full#supplementary-material</a>
</p><div class="sec suppmat" id="SM1"><div class="sup-box half_rhythm" id="media-a.y.d.c.a"><a href="/pmc/articles/PMC10484409/bin/Video_1.avi" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(1.6M, avi)</sup></div></div><div class="sec suppmat" id="SM2"><div class="sup-box half_rhythm" id="media-a.y.d.d.a"><a href="/pmc/articles/PMC10484409/bin/Video_2.avi" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(1.3M, avi)</sup></div></div><div class="sec suppmat" id="SM3"><div class="sup-box half_rhythm" id="media-a.y.d.e.a"><a href="/pmc/articles/PMC10484409/bin/Video_3.avi" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(1.3M, avi)</sup></div></div><div class="sec suppmat" id="SM4"><div class="sup-box half_rhythm" id="media-a.y.d.f.a"><a href="/pmc/articles/PMC10484409/bin/Video_4.avi" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(1.4M, avi)</sup></div></div><div class="sec suppmat" id="SM5"><div class="sup-box half_rhythm" id="media-a.y.d.g.a"><a href="/pmc/articles/PMC10484409/bin/Video_5.avi" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(1.2M, avi)</sup></div></div><div class="sec suppmat" id="SM6"><div class="sup-box half_rhythm" id="media-a.y.d.h.a"><a href="/pmc/articles/PMC10484409/bin/Video_6.avi" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(1.2M, avi)</sup></div></div><div class="sec suppmat" id="SM7"><div class="sup-box half_rhythm" id="media-a.y.d.i.a"><a href="/pmc/articles/PMC10484409/bin/Video_7.avi" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(838K, avi)</sup></div></div><div class="sec suppmat" id="SM8"><div class="sup-box half_rhythm" id="media-a.y.d.j.a"><a href="/pmc/articles/PMC10484409/bin/Video_8.avi" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(1.0M, avi)</sup></div></div><div class="sec suppmat" id="SM9"><div class="sup-box half_rhythm" id="media-a.y.d.k.a"><a href="/pmc/articles/PMC10484409/bin/Video_9.avi" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(944K, avi)</sup></div></div><div class="sec suppmat" id="SM10"><div class="sup-box half_rhythm" id="media-a.y.d.l.a"><a href="/pmc/articles/PMC10484409/bin/Video_10.avi" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(1.1M, avi)</sup></div></div><div class="sec suppmat" id="SM11"><div class="sup-box half_rhythm" id="media-a.y.d.m.a"><a href="/pmc/articles/PMC10484409/bin/Video_11.avi" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(675K, avi)</sup></div></div><div class="sec suppmat" id="SM12"><div class="sup-box half_rhythm" id="media-a.y.d.n.a"><a href="/pmc/articles/PMC10484409/bin/Video_12.avi" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(1.2M, avi)</sup></div></div><div class="sec suppmat" id="SF1"><div class="sup-box half_rhythm" id="media-a.y.d.o.a"><a href="/pmc/articles/PMC10484409/bin/DataSheet_1.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(12M, pdf)</sup></div></div><div class="sec suppmat" id="ST1"><div class="sup-box half_rhythm" id="media-a.y.d.p.a"><a href="/pmc/articles/PMC10484409/bin/Table_1.docx" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(26K, docx)</sup></div></div></div><div id="ref-list-a.y.e" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.y.etitle">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="B1"><span class="mixed-citation">
Abrahams K. A., Besra G. S. (2018). <span class="ref-title">Mycobacterial cell wall biosynthesis: a multifaceted antibiotic target</span>. <span class="ref-journal">Parasitology</span>
<span class="ref-vol">145</span>, 116&#x02013;133. doi:&#x000a0; 10.1017/S0031182016002377
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5964476/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27976597" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1017%2FS0031182016002377" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Parasitology&amp;title=Mycobacterial+cell+wall+biosynthesis:+a+multifaceted+antibiotic+target&amp;author=K.+A.+Abrahams&amp;author=G.+S.+Besra&amp;volume=145&amp;publication_year=2018&amp;pages=116-133&amp;pmid=27976597&amp;doi=10.1017/S0031182016002377&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B2"><span class="mixed-citation">
Aldridge B. B., Fernandez-Suarez M., Heller D., Ambravaneswaran V., Irimia D., Toner M., et al.. (2012). <span class="ref-title">Asymmetry and aging of mycobacterial cells lead to variable growth and antibiotic susceptibility</span>. <span class="ref-journal">Science</span>
<span class="ref-vol">335</span>, 100&#x02013;104. doi:&#x000a0; 10.1126/science.1216166
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3397429/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22174129" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.1216166" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Asymmetry+and+aging+of+mycobacterial+cells+lead+to+variable+growth+and+antibiotic+susceptibility&amp;author=B.+B.+Aldridge&amp;author=M.+Fernandez-Suarez&amp;author=D.+Heller&amp;author=V.+Ambravaneswaran&amp;author=D.+Irimia&amp;volume=335&amp;publication_year=2012&amp;pages=100-104&amp;pmid=22174129&amp;doi=10.1126/science.1216166&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B3"><span class="mixed-citation">
Baranowski C., Welsh M. A., Sham L.-T., Eskandarian H. A., Lim H. C., Kieser K. J., et al.. (2018). <span class="ref-title">Maturing <em>Mycobacterium smegmatis</em> peptidoglycan requires non-canonical crosslinks to maintain shape</span>. <span class="ref-journal">Elife</span>
<span class="ref-vol">7</span>, e37516. doi:&#x000a0; 10.7554/eLife.37516
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6231781/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30324906" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.7554%2FeLife.37516" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Elife&amp;title=Maturing+Mycobacterium+smegmatis+peptidoglycan+requires+non-canonical+crosslinks+to+maintain+shape&amp;author=C.+Baranowski&amp;author=M.+A.+Welsh&amp;author=L.-T.+Sham&amp;author=H.+A.+Eskandarian&amp;author=H.+C.+Lim&amp;volume=7&amp;publication_year=2018&amp;pages=e37516&amp;pmid=30324906&amp;doi=10.7554/eLife.37516&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B4"><span class="mixed-citation">
Botella H., Vaubourgeix J., Lee M. H., Song N., Xu W., Makinoshima H., et al.. (2017. a). <span class="ref-title">
<em>Mycobacterium tuberculosis</em> protease MarP activates a peptidoglycan hydrolase during acid stress</span>. <span class="ref-journal">EMBO J.</span>
<span class="ref-vol">36</span>, 536&#x02013;548. doi:&#x000a0; 10.15252/embj.201695028
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5437814/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28057704" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.15252%2Fembj.201695028" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+J.&amp;title=&#10;Mycobacterium+tuberculosis+protease+MarP+activates+a+peptidoglycan+hydrolase+during+acid+stress&amp;author=H.+Botella&amp;author=J.+Vaubourgeix&amp;author=M.+H.+Lee&amp;author=N.+Song&amp;author=W.+Xu&amp;volume=36&amp;publication_year=2017&amp;pages=536-548&amp;pmid=28057704&amp;doi=10.15252/embj.201695028&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B5"><span class="mixed-citation">
Botella H., Yang G., Ouerfelli O., Ehrt S., Nathan C. F., Vaubourgeix J. (2017. b). <span class="ref-title">Distinct spatiotemporal dynamics of peptidoglycan synthesis between <em>Mycobacterium smegmatis</em> and <em>Mycobacterium tuberculosis</em>
</span>. <span class="ref-journal">MBio</span>
<span class="ref-vol">8</span>, e01183&#x02013;e01117. doi:&#x000a0; 10.1128/mBio.01183-17
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5596344/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28900018" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FmBio.01183-17" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=MBio&amp;title=Distinct+spatiotemporal+dynamics+of+peptidoglycan+synthesis+between+Mycobacterium+smegmatis+and+Mycobacterium+tuberculosis&#10;&amp;author=H.+Botella&amp;author=G.+Yang&amp;author=O.+Ouerfelli&amp;author=S.+Ehrt&amp;author=C.+F.+Nathan&amp;volume=8&amp;publication_year=2017&amp;pages=e01183-e01117&amp;pmid=28900018&amp;doi=10.1128/mBio.01183-17&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B6"><span class="mixed-citation">
Dejesus M. A., Gerrick E. R., Xu W., Park S. W., Long J. E., Boutte C. C., et al.. (2017). <span class="ref-title">Comprehensive essentiality analysis of the <em>Mycobacterium tuberculosis</em> genome <em>via</em> saturating transposon mutagenesis</span>. <span class="ref-journal">MBio</span>
<span class="ref-vol">8</span>, e02133&#x02013;e02116. doi:&#x000a0; 10.1128/mBio.02133-16
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5241402/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28096490" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FmBio.02133-16" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=MBio&amp;title=Comprehensive+essentiality+analysis+of+the+Mycobacterium+tuberculosis+genome+via+saturating+transposon+mutagenesis&amp;author=M.+A.+Dejesus&amp;author=E.+R.+Gerrick&amp;author=W.+Xu&amp;author=S.+W.+Park&amp;author=J.+E.+Long&amp;volume=8&amp;publication_year=2017&amp;pages=e02133-e02116&amp;pmid=28096490&amp;doi=10.1128/mBio.02133-16&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B7"><span class="mixed-citation">
De Wet T. J., Winkler K. R., Mhlanga M. M., Mizrahi V., Warner D. F. (2020). <span class="ref-title">Arrayed CRISPRi and quantitative imaging describe the morphotypic landscape of essential mycobacterial genes</span>. <span class="ref-journal">Elife</span>
<span class="ref-vol">9</span>, p.e60083. doi:&#x000a0; 10.7554/elife.60083
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7647400/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33155979" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.7554%2Felife.60083" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Elife&amp;title=Arrayed+CRISPRi+and+quantitative+imaging+describe+the+morphotypic+landscape+of+essential+mycobacterial+genes&amp;author=T.+J.+De+Wet&amp;author=K.+R.+Winkler&amp;author=M.+M.+Mhlanga&amp;author=V.+Mizrahi&amp;author=D.+F.+Warner&amp;volume=9&amp;publication_year=2020&amp;pages=p.e60083&amp;doi=10.7554/elife.60083&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B8"><span class="mixed-citation">
Dhar N., Dub&#x000e9;e V., Ballell L., Cuinet G., Hugonnet J.-E., Signorino-Gelo F., et al.. (2015). <span class="ref-title">Rapid cytolysis of <em>Mycobacterium tuberculosis</em> by faropenem, an orally bioavailable &#x003b2;-lactam antibiotic</span>. <span class="ref-journal">Antimicrobial Agents Chemother.</span>
<span class="ref-vol">59</span>, 1308&#x02013;1319. doi:&#x000a0; 10.1128/AAC.03461-14
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4335862/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25421469" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FAAC.03461-14" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Antimicrobial+Agents+Chemother.&amp;title=Rapid+cytolysis+of+Mycobacterium+tuberculosis+by+faropenem,+an+orally+bioavailable+&#x003b2;-lactam+antibiotic&amp;author=N.+Dhar&amp;author=V.+Dub&#x000e9;e&amp;author=L.+Ballell&amp;author=G.+Cuinet&amp;author=J.-E.+Hugonnet&amp;volume=59&amp;publication_year=2015&amp;pages=1308-1319&amp;doi=10.1128/AAC.03461-14&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B9"><span class="mixed-citation">
Dhar N., Manina G. (2015). <span class="ref-title">Single-cell analysis of mycobacteria using microfluidics and time-lapse microscopy</span>. <span class="ref-journal">Mycobacter. Protoc.</span>
<span class="ref-vol">1285</span>, 241&#x02013;256. doi:&#x000a0; 10.1007/978-1-4939-2450-9_14
 [<a href="https://pubmed.ncbi.nlm.nih.gov/25779320" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2F978-1-4939-2450-9_14" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mycobacter.+Protoc.&amp;title=Single-cell+analysis+of+mycobacteria+using+microfluidics+and+time-lapse+microscopy&amp;author=N.+Dhar&amp;author=G.+Manina&amp;volume=1285&amp;publication_year=2015&amp;pages=241-256&amp;doi=10.1007/978-1-4939-2450-9_14&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B10"><span class="mixed-citation">
Diacon A. H., van der Merwe L., Barnard M., Von Groote-Bidlingmaier F., Lange C., Garc&#x000ed;a-Basteiro A. L., et al.. (2016). <span class="ref-title">&#x003b2;-Lactams against tuberculosis&#x02013;new trick for an old dog</span>? <span class="ref-journal">New Engl. J. Med.</span>
<span class="ref-vol">375</span>, 393. doi:&#x000a0; 10.1056/NEJMc1513236
 [<a href="https://pubmed.ncbi.nlm.nih.gov/27433841" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1056%2FNEJMc1513236" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=New+Engl.+J.+Med.&amp;title=&#x003b2;-Lactams+against+tuberculosis&#x02013;new+trick+for+an+old+dog&amp;author=A.+H.+Diacon&amp;author=L.+van+der+Merwe&amp;author=M.+Barnard&amp;author=F.+Von+Groote-Bidlingmaier&amp;author=C.+Lange&amp;volume=375&amp;publication_year=2016&amp;pages=393&amp;pmid=27433841&amp;doi=10.1056/NEJMc1513236&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B11"><span class="mixed-citation">
Dragset M. S., Ioerger T. R., Zhang Y. J., M&#x000e6;rk M., Ginbot Z., Sacchettini J. C., et al.. (2019). <span class="ref-title">Genome-wide phenotypic profiling identifies and categorizes genes required for mycobacterial low iron fitness</span>. <span class="ref-journal">Sci. Rep.</span>
<span class="ref-vol">9</span>, 11394. doi:&#x000a0; 10.1038/s41598-019-47905-y
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6684656/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31388080" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41598-019-47905-y" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Rep.&amp;title=Genome-wide+phenotypic+profiling+identifies+and+categorizes+genes+required+for+mycobacterial+low+iron+fitness&amp;author=M.+S.+Dragset&amp;author=T.+R.+Ioerger&amp;author=Y.+J.+Zhang&amp;author=M.+M&#x000e6;rk&amp;author=Z.+Ginbot&amp;volume=9&amp;publication_year=2019&amp;pages=11394&amp;pmid=31388080&amp;doi=10.1038/s41598-019-47905-y&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B12"><span class="mixed-citation">
Figueiredo T. A., Sobral R. G., Ludovice A. M., De Almeida J. M. F., Bui N. K., Vollmer W., et al.. (2012). <span class="ref-title">Identification of genetic determinants and enzymes involved with the amidation of glutamic acid residues in the peptidoglycan of <em>Staphylococcus aureus</em>
</span>. <span class="ref-journal">PloS Pathog.</span>
<span class="ref-vol">8</span>, e1002508. doi:&#x000a0; 10.1371/journal.ppat.1002508
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3267633/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22303291" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.ppat.1002508" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PloS+Pathog.&amp;title=Identification+of+genetic+determinants+and+enzymes+involved+with+the+amidation+of+glutamic+acid+residues+in+the+peptidoglycan+of+Staphylococcus+aureus&#10;&amp;author=T.+A.+Figueiredo&amp;author=R.+G.+Sobral&amp;author=A.+M.+Ludovice&amp;author=J.+M.+F.+De+Almeida&amp;author=N.+K.+Bui&amp;volume=8&amp;publication_year=2012&amp;pages=e1002508&amp;pmid=22303291&amp;doi=10.1371/journal.ppat.1002508&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B13"><span class="mixed-citation">
Flores A. R., Parsons L. M., Pavelka M. S., Jr (2005). <span class="ref-title">Genetic analysis of the &#x003b2;-lactamases of Mycobacterium tuberculosis and <em>Mycobacterium smegmatis</em> and susceptibility to &#x003b2;-lactam antibiotics</span>. <span class="ref-journal">Microbiology</span>
<span class="ref-vol">151</span>, 521&#x02013;532. doi:&#x000a0; 10.1099/mic.0.27629-0
 [<a href="https://pubmed.ncbi.nlm.nih.gov/15699201" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1099%2Fmic.0.27629-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology&amp;title=Genetic+analysis+of+the+&#x003b2;-lactamases+of+Mycobacterium+tuberculosis+and+Mycobacterium+smegmatis+and+susceptibility+to+&#x003b2;-lactam+antibiotics&amp;author=A.+R.+Flores&amp;author=L.+M.+Parsons&amp;author=M.+S.+Pavelka&amp;volume=151&amp;publication_year=2005&amp;pages=521-532&amp;pmid=15699201&amp;doi=10.1099/mic.0.27629-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B14"><span class="mixed-citation">
Francis Z. K., Sanders A. N., Crick D. C., Mahapatra S., Islam M. N., Pavelka M. S., Jr (2023). <span class="ref-title">LdtC is a key l, d-Transpeptidase for Peptidoglycan Assembly in <em>Mycobacterium smegmatis</em>
</span>. <span class="ref-journal">J. Bacteriol.</span>
<span class="ref-vol">205</span>, e00424&#x02013;e00422. doi:&#x000a0; 10.1128/jb.00424-22
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9879121/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36541811" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2Fjb.00424-22" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Bacteriol.&amp;title=LdtC+is+a+key+l,+d-Transpeptidase+for+Peptidoglycan+Assembly+in+Mycobacterium+smegmatis&#10;&amp;author=Z.+K.+Francis&amp;author=A.+N.+Sanders&amp;author=D.+C.+Crick&amp;author=S.+Mahapatra&amp;author=M.+N.+Islam&amp;volume=205&amp;publication_year=2023&amp;pages=e00424-e00422&amp;pmid=36541811&amp;doi=10.1128/jb.00424-22&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B15"><span class="mixed-citation">
Garc&#x000ed;a-Heredia A., Pohane A. A., Melzer E. S., Carr C. R., Fiolek T. J., Rundell S. R., et al.. (2018). <span class="ref-title">Peptidoglycan precursor synthesis along the sidewall of pole-growing mycobacteria</span>. <span class="ref-journal">Elife</span>
<span class="ref-vol">7</span>, e37243. doi:&#x000a0; 10.7554/eLife.37243
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6191288/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30198841" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.7554%2FeLife.37243" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Elife&amp;title=Peptidoglycan+precursor+synthesis+along+the+sidewall+of+pole-growing+mycobacteria&amp;author=A.+Garc&#x000ed;a-Heredia&amp;author=A.+A.+Pohane&amp;author=E.+S.+Melzer&amp;author=C.+R.+Carr&amp;author=T.+J.+Fiolek&amp;volume=7&amp;publication_year=2018&amp;pages=e37243&amp;pmid=30198841&amp;doi=10.7554/eLife.37243&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B16"><span class="mixed-citation">
Gordhan B. G., Parish T. (2001). <span class="ref-title">Gene replacement using pretreated DNA</span>. <span class="ref-journal">Methods Mol. Med.</span>
<span class="ref-vol">54</span>, 77&#x02013;92. doi:&#x000a0; 10.1385/1-59259-147-7:077
 [<a href="https://pubmed.ncbi.nlm.nih.gov/21341070" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1385%2F1-59259-147-7%3A077" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Mol.+Med.&amp;title=Gene+replacement+using+pretreated+DNA&amp;author=B.+G.+Gordhan&amp;author=T.+Parish&amp;volume=54&amp;publication_year=2001&amp;pages=77-92&amp;pmid=21341070&amp;doi=10.1385/1-59259-147-7:077&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B17"><span class="mixed-citation">
Guo X. V., Monteleone M., Klotzsche M., Kamionka A., Hillen W., Braunstein M., et al.. (2007). <span class="ref-title">Silencing essential protein secretion in <em>Mycobacterium smegmatis</em> by using tetracycline repressors</span>. <span class="ref-journal">J. Bacteriol.</span>
<span class="ref-vol">189</span>, 4614&#x02013;4623. doi:&#x000a0; 10.1128/JB.00216-07
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1913471/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17483222" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FJB.00216-07" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Bacteriol.&amp;title=Silencing+essential+protein+secretion+in+Mycobacterium+smegmatis+by+using+tetracycline+repressors&amp;author=X.+V.+Guo&amp;author=M.+Monteleone&amp;author=M.+Klotzsche&amp;author=A.+Kamionka&amp;author=W.+Hillen&amp;volume=189&amp;publication_year=2007&amp;pages=4614-4623&amp;pmid=17483222&amp;doi=10.1128/JB.00216-07&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B18"><span class="mixed-citation">
Hayashi J. M., Luo C.-Y., Mayfield J. A., Hsu T., Fukuda T., Walfield A. L., et al.. (2016). <span class="ref-title">Spatially distinct and metabolically active membrane domain in mycobacteria</span>. <span class="ref-journal">Proc. Natl. Acad. Sci.</span>
<span class="ref-vol">113</span>, 5400&#x02013;5405. doi:&#x000a0; 10.1073/pnas.1525165113
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4868426/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27114527" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.1525165113" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.&amp;title=Spatially+distinct+and+metabolically+active+membrane+domain+in+mycobacteria&amp;author=J.+M.+Hayashi&amp;author=C.-Y.+Luo&amp;author=J.+A.+Mayfield&amp;author=T.+Hsu&amp;author=T.+Fukuda&amp;volume=113&amp;publication_year=2016&amp;pages=5400-5405&amp;pmid=27114527&amp;doi=10.1073/pnas.1525165113&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B19"><span class="mixed-citation">
Hugonnet J.-E., Tremblay L. W., Boshoff H. I., Barry C. E., Blanchard J. S. (2009). <span class="ref-title">Meropenem-clavulanate is effective against extensively drug-resistant <em>Mycobacterium tuberculosis</em>
</span>. <span class="ref-journal">Science</span>
<span class="ref-vol">323</span>, 1215&#x02013;1218. doi:&#x000a0; 10.1126/science.1167498
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2679150/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19251630" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.1167498" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Meropenem-clavulanate+is+effective+against+extensively+drug-resistant+Mycobacterium+tuberculosis&#10;&amp;author=J.-E.+Hugonnet&amp;author=L.+W.+Tremblay&amp;author=H.+I.+Boshoff&amp;author=C.+E.+Barry&amp;author=J.+S.+Blanchard&amp;volume=323&amp;publication_year=2009&amp;pages=1215-1218&amp;pmid=19251630&amp;doi=10.1126/science.1167498&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B20"><span class="mixed-citation">
Kaur P., Rausch M., Malakar B., Watson U., Damle N. P., Chawla Y., et al.. (2019). <span class="ref-title">LipidII interaction with specific residues of <em>Mycobacterium tuberculosis</em> PknB extracytoplasmic domain governs its optimal activation</span>. <span class="ref-journal">Nat. Commun.</span>
<span class="ref-vol">10</span>, 1231. doi:&#x000a0; 10.1038/s41467-019-09223-9
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6428115/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30874556" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41467-019-09223-9" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=LipidII+interaction+with+specific+residues+of+Mycobacterium+tuberculosis+PknB+extracytoplasmic+domain+governs+its+optimal+activation&amp;author=P.+Kaur&amp;author=M.+Rausch&amp;author=B.+Malakar&amp;author=U.+Watson&amp;author=N.+P.+Damle&amp;volume=10&amp;publication_year=2019&amp;pages=1231&amp;pmid=30874556&amp;doi=10.1038/s41467-019-09223-9&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B21"><span class="mixed-citation">
Lavollay M., Arthur M., Fourgeaud M., Dubost L., Marie A., Veziris N., et al.. (2008). <span class="ref-title">The peptidoglycan of stationary-phase <em>Mycobacterium tuberculosis</em> predominantly contains cross-links generated by L, D-transpeptidation</span>. <span class="ref-journal">J. Bacteriol.</span>
<span class="ref-vol">190</span>, 4360&#x02013;4366. doi:&#x000a0; 10.1128/JB.00239-08
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2446752/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18408028" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FJB.00239-08" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Bacteriol.&amp;title=The+peptidoglycan+of+stationary-phase+Mycobacterium+tuberculosis+predominantly+contains+cross-links+generated+by+L,+D-transpeptidation&amp;author=M.+Lavollay&amp;author=M.+Arthur&amp;author=M.+Fourgeaud&amp;author=L.+Dubost&amp;author=A.+Marie&amp;volume=190&amp;publication_year=2008&amp;pages=4360-4366&amp;pmid=18408028&amp;doi=10.1128/JB.00239-08&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B22"><span class="mixed-citation">
Levefaudes M., Patin D., De Sousa-D&#x02019;auria C., Chami M., Blanot D., Herv&#x000e9; M., et al.. (2015). <span class="ref-title">Diaminopimelic acid amidation in corynebacteriales: new insights into the role of LtsA in peptidoglycan modification</span>. <span class="ref-journal">J. Biol. Chem.</span>
<span class="ref-vol">290</span>, 13079&#x02013;13094. doi:&#x000a0; 10.1074/jbc.M115.642843
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4505564/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25847251" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1074%2Fjbc.M115.642843" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Diaminopimelic+acid+amidation+in+corynebacteriales:+new+insights+into+the+role+of+LtsA+in+peptidoglycan+modification&amp;author=M.+Levefaudes&amp;author=D.+Patin&amp;author=C.+De+Sousa-D&#x02019;auria&amp;author=M.+Chami&amp;author=D.+Blanot&amp;volume=290&amp;publication_year=2015&amp;pages=13079-13094&amp;pmid=25847251&amp;doi=10.1074/jbc.M115.642843&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B23"><span class="mixed-citation">
Liechti G., Kuru E., Hall E., Kalinda A., Brun Y., Vannieuwenhze M., et al.. (2014). <span class="ref-title">A new metabolic cell-wall labelling method reveals peptidoglycan in <em>Chlamydia trachomatis</em>
</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">506</span>, 507&#x02013;510. doi:&#x000a0; 10.1038/nature12892
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3997218/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24336210" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnature12892" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+new+metabolic+cell-wall+labelling+method+reveals+peptidoglycan+in+Chlamydia+trachomatis&#10;&amp;author=G.+Liechti&amp;author=E.+Kuru&amp;author=E.+Hall&amp;author=A.+Kalinda&amp;author=Y.+Brun&amp;volume=506&amp;publication_year=2014&amp;pages=507-510&amp;pmid=24336210&amp;doi=10.1038/nature12892&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B24"><span class="mixed-citation">
Liu X., Gallay C., Kjos M., Domenech A., Slager J., Van Kessel S. P., et al.. (2017). <span class="ref-title">High-throughput CRISPRi phenotyping identifies new essential genes in <em>Streptococcus pneumoniae</em>
</span>. <span class="ref-journal">Mol. Syst. Biol.</span>
<span class="ref-vol">13</span>, 931. doi:&#x000a0; 10.15252/msb.20167449
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5448163/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28490437" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.15252%2Fmsb.20167449" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Syst.+Biol.&amp;title=High-throughput+CRISPRi+phenotyping+identifies+new+essential+genes+in+Streptococcus+pneumoniae&#10;&amp;author=X.+Liu&amp;author=C.+Gallay&amp;author=M.+Kjos&amp;author=A.+Domenech&amp;author=J.+Slager&amp;volume=13&amp;publication_year=2017&amp;pages=931&amp;pmid=28490437&amp;doi=10.15252/msb.20167449&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B25"><span class="mixed-citation">
Maitra A., Munshi T., Healy J., Martin L. T., Vollmer W., Keep N. H., et al.. (2019). <span class="ref-title">Cell wall peptidoglycan in <em>Mycobacterium tuberculosis</em>: An Achilles&#x02019; heel for the TB-causing pathogen</span>. <span class="ref-journal">FEMS Microbiol. Rev.</span>
<span class="ref-vol">43</span>, 548&#x02013;575. doi:&#x000a0; 10.1093/femsre/fuz016
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6736417/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31183501" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Ffemsre%2Ffuz016" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEMS+Microbiol.+Rev.&amp;title=Cell+wall+peptidoglycan+in+Mycobacterium+tuberculosis:+An+Achilles&#x02019;+heel+for+the+TB-causing+pathogen&amp;author=A.+Maitra&amp;author=T.+Munshi&amp;author=J.+Healy&amp;author=L.+T.+Martin&amp;author=W.+Vollmer&amp;volume=43&amp;publication_year=2019&amp;pages=548-575&amp;pmid=31183501&amp;doi=10.1093/femsre/fuz016&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B26"><span class="mixed-citation">
Maitra A., Nukala S., Dickman R., Martin L. T., Munshi T., Gupta A., et al.. (2021). <span class="ref-title">Characterization of the MurT/GatD complex in <em>Mycobacterium tuberculosis</em> towards validating a novel anti-tubercular drug target</span>. <span class="ref-journal">JAC-antimicrobial Resistance</span>
<span class="ref-vol">3</span>, dlab028. doi:&#x000a0; 10.1093/jacamr/dlab028
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8210147/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34223102" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fjacamr%2Fdlab028" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAC-antimicrobial+Resistance&amp;title=Characterization+of+the+MurT/GatD+complex+in+Mycobacterium+tuberculosis+towards+validating+a+novel+anti-tubercular+drug+target&amp;author=A.+Maitra&amp;author=S.+Nukala&amp;author=R.+Dickman&amp;author=L.+T.+Martin&amp;author=T.+Munshi&amp;volume=3&amp;publication_year=2021&amp;pages=dlab028&amp;pmid=34223102&amp;doi=10.1093/jacamr/dlab028&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B27"><span class="mixed-citation">
Meniche X., Otten R., Siegrist M. S., Baer C. E., Murphy K. C., Bertozzi C. R., et al.. (2014). <span class="ref-title">Subpolar addition of new cell wall is directed by DivIVA in mycobacteria</span>. <span class="ref-journal">Proc. Natl. Acad. Sci.</span>
<span class="ref-vol">111</span>, E3243&#x02013;E3251. doi:&#x000a0; 10.1073/pnas.1402158111
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4128124/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25049412" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.1402158111" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.&amp;title=Subpolar+addition+of+new+cell+wall+is+directed+by+DivIVA+in+mycobacteria&amp;author=X.+Meniche&amp;author=R.+Otten&amp;author=M.+S.+Siegrist&amp;author=C.+E.+Baer&amp;author=K.+C.+Murphy&amp;volume=111&amp;publication_year=2014&amp;pages=E3243-E3251&amp;pmid=25049412&amp;doi=10.1073/pnas.1402158111&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B28"><span class="mixed-citation">
Mir M., Asong J., Li X., Cardot J., Boons G.-J., Husson R. N. (2011). <span class="ref-title">The extracytoplasmic domain of the <em>Mycobacterium tuberculosis</em> Ser/Thr kinase PknB binds specific muropeptides and is required for PknB localization</span>. <span class="ref-journal">PloS Pathog.</span>
<span class="ref-vol">7</span>, e1002182. doi:&#x000a0; 10.1371/journal.ppat.1002182
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3145798/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21829358" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.ppat.1002182" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PloS+Pathog.&amp;title=The+extracytoplasmic+domain+of+the+Mycobacterium+tuberculosis+Ser/Thr+kinase+PknB+binds+specific+muropeptides+and+is+required+for+PknB+localization&amp;author=M.+Mir&amp;author=J.+Asong&amp;author=X.+Li&amp;author=J.+Cardot&amp;author=G.-J.+Boons&amp;volume=7&amp;publication_year=2011&amp;pages=e1002182&amp;pmid=21829358&amp;doi=10.1371/journal.ppat.1002182&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B29"><span class="mixed-citation">
Morlot C., Straume D., Peters K., Hegnar O. A., Simon N., Villard A.-M., et al.. (2018). <span class="ref-title">Structure of the essential peptidoglycan amidotransferase MurT/GatD complex from <em>Streptococcus pneumoniae</em>
</span>. <span class="ref-journal">Nat. Commun.</span>
<span class="ref-vol">9</span>, 1&#x02013;12. doi:&#x000a0; 10.1038/s41467-018-05602-w
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6085368/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30093673" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41467-018-05602-w" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Structure+of+the+essential+peptidoglycan+amidotransferase+MurT/GatD+complex+from+Streptococcus+pneumoniae&#10;&amp;author=C.+Morlot&amp;author=D.+Straume&amp;author=K.+Peters&amp;author=O.+A.+Hegnar&amp;author=N.+Simon&amp;volume=9&amp;publication_year=2018&amp;pages=1-12&amp;pmid=29317637&amp;doi=10.1038/s41467-018-05602-w&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B30"><span class="mixed-citation">
M&#x000fc;nch D., Roemer T., Lee S. H., Engeser M., Sahl H. G., Schneider T. (2012). <span class="ref-title">Identification and <em>in vitro</em> analysis of the GatD/MurT enzyme-complex catalyzing lipid II amidation in <em>Staphylococcus aureus</em>
</span>. <span class="ref-journal">PloS Pathog.</span>
<span class="ref-vol">8</span>, e1002509. doi:&#x000a0; 10.1371/journal.ppat.1002509
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3266927/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22291598" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.ppat.1002509" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PloS+Pathog.&amp;title=Identification+and+in+vitro+analysis+of+the+GatD/MurT+enzyme-complex+catalyzing+lipid+II+amidation+in+Staphylococcus+aureus&#10;&amp;author=D.+M&#x000fc;nch&amp;author=T.+Roemer&amp;author=S.+H.+Lee&amp;author=M.+Engeser&amp;author=H.+G.+Sahl&amp;volume=8&amp;publication_year=2012&amp;pages=e1002509&amp;pmid=22291598&amp;doi=10.1371/journal.ppat.1002509&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B31"><span class="mixed-citation">
Narrandes N. C., Machowski E. E., Mizrahi V., Kana B. D. (2015). <span class="ref-title">Cleavage of the moaX-encoded fused molybdopterin synthase from <em>Mycobacterium tuberculosis</em> is necessary for activity</span>. <span class="ref-journal">BMC Microbiol.</span>
<span class="ref-vol">15</span>, 1&#x02013;12. doi:&#x000a0; 10.1186/s12866-015-0355-2
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4326299/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25651977" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2Fs12866-015-0355-2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Microbiol.&amp;title=Cleavage+of+the+moaX-encoded+fused+molybdopterin+synthase+from+Mycobacterium+tuberculosis+is+necessary+for+activity&amp;author=N.+C.+Narrandes&amp;author=E.+E.+Machowski&amp;author=V.+Mizrahi&amp;author=B.+D.+Kana&amp;volume=15&amp;publication_year=2015&amp;pages=1-12&amp;pmid=25591663&amp;doi=10.1186/s12866-015-0355-2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B32"><span class="mixed-citation">
Ngadjeua F., Braud E., Saidjalolov S., Iannazzo L., Schnappinger D., Ehrt S., et al.. (2018). <span class="ref-title">Critical Impact of Peptidoglycan Precursor Amidation on the Activity of L, D-Transpeptidases from <em>Enterococcus faecium</em> and <em>Mycobacterium tuberculosis</em>
</span>. <span class="ref-journal">Chemistry&#x02013;A Eur. J.</span>
<span class="ref-vol">24</span>, 5743&#x02013;5747. doi:&#x000a0; 10.1002/chem.201706082
 [<a href="https://pubmed.ncbi.nlm.nih.gov/29389045" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fchem.201706082" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chemistry&#x02013;A+Eur.+J.&amp;title=Critical+Impact+of+Peptidoglycan+Precursor+Amidation+on+the+Activity+of+L,+D-Transpeptidases+from+Enterococcus+faecium+and+Mycobacterium+tuberculosis&#10;&amp;author=F.+Ngadjeua&amp;author=E.+Braud&amp;author=S.+Saidjalolov&amp;author=L.+Iannazzo&amp;author=D.+Schnappinger&amp;volume=24&amp;publication_year=2018&amp;pages=5743-5747&amp;doi=10.1002/chem.201706082&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B33"><span class="mixed-citation">
N&#x000f6;ldeke E. R., Muckenfuss L. M., Niemann V., M&#x000fc;ller A., St&#x000f6;rk E., Zocher G., et al.. (2018). <span class="ref-title">Structural basis of cell wall peptidoglycan amidation by the GatD/MurT complex of <em>Staphylococcus aureus</em>
</span>. <span class="ref-journal">Sci. Rep.</span>
<span class="ref-vol">8</span>, 1&#x02013;15. doi:&#x000a0; 10.1038/s41598-018-31098-x
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6113224/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30154570" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41598-018-31098-x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Rep.&amp;title=Structural+basis+of+cell+wall+peptidoglycan+amidation+by+the+GatD/MurT+complex+of+Staphylococcus+aureus&#10;&amp;author=E.+R.+N&#x000f6;ldeke&amp;author=L.+M.+Muckenfuss&amp;author=V.+Niemann&amp;author=A.+M&#x000fc;ller&amp;author=E.+St&#x000f6;rk&amp;volume=8&amp;publication_year=2018&amp;pages=1-15&amp;pmid=29311619&amp;doi=10.1038/s41598-018-31098-x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B34"><span class="mixed-citation">
Pereira S. F., Goss L., Dworkin J. (2011). <span class="ref-title">Eukaryote-like serine/threonine kinases and phosphatases in bacteria</span>. <span class="ref-journal">Microbiol. Mol. Biol. Rev.</span>
<span class="ref-vol">75</span>, 192&#x02013;212. doi:&#x000a0; 10.1128/MMBR.00042-10
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3063355/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21372323" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FMMBR.00042-10" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol.+Mol.+Biol.+Rev.&amp;title=Eukaryote-like+serine/threonine+kinases+and+phosphatases+in+bacteria&amp;author=S.+F.+Pereira&amp;author=L.+Goss&amp;author=J.+Dworkin&amp;volume=75&amp;publication_year=2011&amp;pages=192-212&amp;pmid=21372323&amp;doi=10.1128/MMBR.00042-10&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B35"><span class="mixed-citation">
Pidgeon S. E., Apostolos A. J., Nelson J. M., Shaku M., Rimal B., Islam M. N., et al.. (2019). <span class="ref-title">L,D-transpeptidase specific probe reveals spatial activity of peptidoglycan cross-linking</span>. <span class="ref-journal">ACS Chem. Biol.</span>
<span class="ref-vol">14</span>, 2185&#x02013;2196. doi:&#x000a0; 10.1021/acschembio.9b00427
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6804245/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31487148" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Facschembio.9b00427" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=ACS+Chem.+Biol.&amp;title=L,D-transpeptidase+specific+probe+reveals+spatial+activity+of+peptidoglycan+cross-linking&amp;author=S.+E.+Pidgeon&amp;author=A.+J.+Apostolos&amp;author=J.+M.+Nelson&amp;author=M.+Shaku&amp;author=B.+Rimal&amp;volume=14&amp;publication_year=2019&amp;pages=2185-2196&amp;pmid=31487148&amp;doi=10.1021/acschembio.9b00427&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B36"><span class="mixed-citation">
Rock J. M., Hopkins F. F., Chavez A., Diallo M., Chase M. R., Gerrick E. R., et al.. (2017). <span class="ref-title">Programmable transcriptional repression in mycobacteria using an orthogonal CRISPR interference platform</span>. <span class="ref-journal">Nat. Microbiol.</span>
<span class="ref-vol">2</span>, 1&#x02013;9. doi:&#x000a0; 10.1038/nmicrobiol.2016.274
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5302332/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28165460" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnmicrobiol.2016.274" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Microbiol.&amp;title=Programmable+transcriptional+repression+in+mycobacteria+using+an+orthogonal+CRISPR+interference+platform&amp;author=J.+M.+Rock&amp;author=F.+F.+Hopkins&amp;author=A.+Chavez&amp;author=M.+Diallo&amp;author=M.+R.+Chase&amp;volume=2&amp;publication_year=2017&amp;pages=1-9&amp;doi=10.1038/nmicrobiol.2016.274&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B37"><span class="mixed-citation">
Sassetti C. M., Boyd D. H., Rubin E. J. (2003). <span class="ref-title">Genes required for mycobacterial growth defined by high density mutagenesis</span>. <span class="ref-journal">Mol. Microbiol.</span>
<span class="ref-vol">48</span>, 77&#x02013;84. doi:&#x000a0; 10.1046/j.1365-2958.2003.03425.x
 [<a href="https://pubmed.ncbi.nlm.nih.gov/12657046" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1046%2Fj.1365-2958.2003.03425.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Microbiol.&amp;title=Genes+required+for+mycobacterial+growth+defined+by+high+density+mutagenesis&amp;author=C.+M.+Sassetti&amp;author=D.+H.+Boyd&amp;author=E.+J.+Rubin&amp;volume=48&amp;publication_year=2003&amp;pages=77-84&amp;pmid=12657046&amp;doi=10.1046/j.1365-2958.2003.03425.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B38"><span class="mixed-citation">
Senzani S., Li D., Bhaskar A., Ealand C., Chang J., Rimal B., et al.. (2017). <span class="ref-title">An Amidase_3 domain-containing <em>N</em>-acetylmuramyl-L-alanine amidase is required for mycobacterial cell division</span>. <span class="ref-journal">Sci. Rep.</span>
<span class="ref-vol">7</span>, 1&#x02013;16. doi:&#x000a0; 10.1038/s41598-017-01184-7
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5430687/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28442758" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41598-017-01184-7" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Rep.&amp;title=An+Amidase_3+domain-containing+N-acetylmuramyl-L-alanine+amidase+is+required+for+mycobacterial+cell+division&amp;author=S.+Senzani&amp;author=D.+Li&amp;author=A.+Bhaskar&amp;author=C.+Ealand&amp;author=J.+Chang&amp;volume=7&amp;publication_year=2017&amp;pages=1-16&amp;pmid=28127051&amp;doi=10.1038/s41598-017-01184-7&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B39"><span class="mixed-citation">
Silveiro C., Marques M., Oliven&#x000e7;a F., Pires D., Mortinho D., Nunes A., et al.. (2023). <span class="ref-title">CRISPRi-mediated characterization of novel anti-tuberculosis targets: Mycobacterial peptidoglycan modifications promote beta-lactam resistance and intracellular survival</span>. <span class="ref-journal">Front. Cell. Infect. Microbiol.</span>
<span class="ref-vol">13</span>. doi:&#x000a0; 10.3389/fcimb.2023.1089911
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10050696/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37009497" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3389%2Ffcimb.2023.1089911" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Cell.+Infect.+Microbiol.&amp;title=CRISPRi-mediated+characterization+of+novel+anti-tuberculosis+targets:+Mycobacterial+peptidoglycan+modifications+promote+beta-lactam+resistance+and+intracellular+survival&amp;author=C.+Silveiro&amp;author=M.+Marques&amp;author=F.+Oliven&#x000e7;a&amp;author=D.+Pires&amp;author=D.+Mortinho&amp;volume=13&amp;publication_year=2023&amp;doi=10.3389/fcimb.2023.1089911&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B40"><span class="mixed-citation">
Snapper S., Melton R., Mustafa S., Kieser T., Wr J., Jr (1990). <span class="ref-title">Isolation and characterization of efficient plasmid transformation mutants of <em>Mycobacterium smegmatis</em>
</span>. <span class="ref-journal">Mol. Microbiol.</span>
<span class="ref-vol">4</span>, 1911&#x02013;1919. doi:&#x000a0; 10.1111/j.1365-2958.1990.tb02040.x
 [<a href="https://pubmed.ncbi.nlm.nih.gov/2082148" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fj.1365-2958.1990.tb02040.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Microbiol.&amp;title=Isolation+and+characterization+of+efficient+plasmid+transformation+mutants+of+Mycobacterium+smegmatis&#10;&amp;author=S.+Snapper&amp;author=R.+Melton&amp;author=S.+Mustafa&amp;author=T.+Kieser&amp;author=J.+Wr&amp;volume=4&amp;publication_year=1990&amp;pages=1911-1919&amp;pmid=2082148&amp;doi=10.1111/j.1365-2958.1990.tb02040.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B41"><span class="mixed-citation">
Stover C., de la Cruz V., Fuerst T., Burlein J., Benson L., Bennett L., et al.. (1991). <span class="ref-title">New use of BCG for recombinant vaccines</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">351</span>, 456&#x02013;460. doi:&#x000a0; 10.1038/351456a0
 [<a href="https://pubmed.ncbi.nlm.nih.gov/1904554" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2F351456a0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=New+use+of+BCG+for+recombinant+vaccines&amp;author=C.+Stover&amp;author=V.+de+la+Cruz&amp;author=T.+Fuerst&amp;author=J.+Burlein&amp;author=L.+Benson&amp;volume=351&amp;publication_year=1991&amp;pages=456-460&amp;pmid=1904554&amp;doi=10.1038/351456a0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B42"><span class="mixed-citation">
Turapov O., Forti F., Kadhim B., Ghisotti D., Sassine J., Straatman-Iwanowska A., et al.. (2018). <span class="ref-title">Two faces of CwlM, an essential PknB substrate, in <em>Mycobacterium tuberculosis</em>
</span>. <span class="ref-journal">Cell Rep.</span>
<span class="ref-vol">25</span>, 57&#x02013;67. doi:&#x000a0; 10.1016/j.celrep.2018.09.004
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6180346/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30282038" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.celrep.2018.09.004" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Rep.&amp;title=Two+faces+of+CwlM,+an+essential+PknB+substrate,+in+Mycobacterium+tuberculosis&#10;&amp;author=O.+Turapov&amp;author=F.+Forti&amp;author=B.+Kadhim&amp;author=D.+Ghisotti&amp;author=J.+Sassine&amp;volume=25&amp;publication_year=2018&amp;pages=57-67&amp;pmid=30282038&amp;doi=10.1016/j.celrep.2018.09.004&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B43"><span class="mixed-citation">
Turapov O., Loraine J., Jenkins C. H., Barthe P., Mcfeely D., Forti F., et al.. (2015). <span class="ref-title">The external PASTA domain of the essential serine/threonine protein kinase PknB regulates mycobacterial growth</span>. <span class="ref-journal">Open Biol.</span>
<span class="ref-vol">5</span>, 150025. doi:&#x000a0; 10.1098/rsob.150025
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4632501/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26136255" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1098%2Frsob.150025" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Open+Biol.&amp;title=The+external+PASTA+domain+of+the+essential+serine/threonine+protein+kinase+PknB+regulates+mycobacterial+growth&amp;author=O.+Turapov&amp;author=J.+Loraine&amp;author=C.+H.+Jenkins&amp;author=P.+Barthe&amp;author=D.+Mcfeely&amp;volume=5&amp;publication_year=2015&amp;pages=150025&amp;pmid=26136255&amp;doi=10.1098/rsob.150025&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B44"><span class="mixed-citation">
Vijayrajratnam S., Pushkaran A. C., Balakrishnan A., Vasudevan A. K., Biswas R., Mohan C. G. (2016). <span class="ref-title">Bacterial peptidoglycan with amidated meso-diaminopimelic acid evades NOD1 recognition: an insight into NOD1 structure&#x02013;recognition</span>. <span class="ref-journal">Biochem. J.</span>
<span class="ref-vol">473</span>, 4573&#x02013;4592. doi: 10.1042/BCJ20160817
 [<a href="https://pubmed.ncbi.nlm.nih.gov/27742759" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1042%2FBCJ20160817" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.+J.&amp;title=Bacterial+peptidoglycan+with+amidated+meso-diaminopimelic+acid+evades+NOD1+recognition:+an+insight+into+NOD1+structure&#x02013;recognition&amp;author=S.+Vijayrajratnam&amp;author=A.+C.+Pushkaran&amp;author=A.+Balakrishnan&amp;author=A.+K.+Vasudevan&amp;author=R.+Biswas&amp;volume=473&amp;publication_year=2016&amp;pages=4573-4592&amp;pmid=27742759&amp;doi=10.1042/BCJ20160817&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B45"><span class="mixed-citation">
Vollmer W. (2007). <span class="ref-title">Structure and biosynthesis of the murein (peptidoglycan) sacculus</span>. <span class="ref-journal">Periplasm</span>, 198&#x02013;213. doi:&#x000a0; 10.1128/9781555815806.ch11
 [<a href="//doi.org/10.1128%2F9781555815806.ch11" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Periplasm&amp;title=Structure+and+biosynthesis+of+the+murein+(peptidoglycan)+sacculus&amp;author=W.+Vollmer&amp;publication_year=2007&amp;pages=198-213&amp;doi=10.1128/9781555815806.ch11&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B46"><span class="mixed-citation">
Vollmer W., Bertsche U. (2008). <span class="ref-title">Murein (peptidoglycan) structure, architecture and biosynthesis in <em>Escherichia coli</em>
</span>. <span class="ref-journal">Biochim. Biophys. Acta (BBA)-Biomembranes</span>
<span class="ref-vol">1778</span>, 1714&#x02013;1734. doi:&#x000a0; 10.1016/j.bbamem.2007.06.007
 [<a href="https://pubmed.ncbi.nlm.nih.gov/17658458" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.bbamem.2007.06.007" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.+Biophys.+Acta+(BBA)-Biomembranes&amp;title=Murein+(peptidoglycan)+structure,+architecture+and+biosynthesis+in+Escherichia+coli&#10;&amp;author=W.+Vollmer&amp;author=U.+Bertsche&amp;volume=1778&amp;publication_year=2008&amp;pages=1714-1734&amp;pmid=17658458&amp;doi=10.1016/j.bbamem.2007.06.007&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B47"><span class="mixed-citation">
Vollmer W., Blanot D., De Pedro M. A. (2008). <span class="ref-title">Peptidoglycan structure and architecture</span>. <span class="ref-journal">FEMS Microbiol. Rev.</span>
<span class="ref-vol">32</span>, 149&#x02013;167. doi:&#x000a0; 10.1111/j.1574-6976.2007.00094.x
 [<a href="https://pubmed.ncbi.nlm.nih.gov/18194336" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fj.1574-6976.2007.00094.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEMS+Microbiol.+Rev.&amp;title=Peptidoglycan+structure+and+architecture&amp;author=W.+Vollmer&amp;author=D.+Blanot&amp;author=M.+A.+De+Pedro&amp;volume=32&amp;publication_year=2008&amp;pages=149-167&amp;pmid=18194336&amp;doi=10.1111/j.1574-6976.2007.00094.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B48"><span class="mixed-citation">
Wolfert M. A., Roychowdhury A., Boons G.-J. (2007). <span class="ref-title">Modification of the structure of peptidoglycan is a strategy to avoid detection by nucleotide-binding oligomerization domain protein 1</span>. <span class="ref-journal">Infect. Immun.</span>
<span class="ref-vol">75</span>, 706&#x02013;713. doi:&#x000a0; 10.1128/IAI.01597-06
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1828529/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17145940" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FIAI.01597-06" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Infect.+Immun.&amp;title=Modification+of+the+structure+of+peptidoglycan+is+a+strategy+to+avoid+detection+by+nucleotide-binding+oligomerization+domain+protein+1&amp;author=M.+A.+Wolfert&amp;author=A.+Roychowdhury&amp;author=G.-J.+Boons&amp;volume=75&amp;publication_year=2007&amp;pages=706-713&amp;pmid=17145940&amp;doi=10.1128/IAI.01597-06&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B49"><span class="mixed-citation">
Zapun A., Philippe J., Abrahams K. A., Signor L., Roper D. I., Breukink E., et al.. (2013). <span class="ref-title">
<em>In vitro</em> reconstitution of peptidoglycan assembly from the Gram-positive pathogen <em>Streptococcus pneumoniae</em>
</span>. <span class="ref-journal">ACS Chem. Biol.</span>
<span class="ref-vol">8</span>, 2688&#x02013;2696. doi:&#x000a0; 10.1021/cb400575t
 [<a href="https://pubmed.ncbi.nlm.nih.gov/24044435" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fcb400575t" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=ACS+Chem.+Biol.&amp;title=&#10;In+vitro+reconstitution+of+peptidoglycan+assembly+from+the+Gram-positive+pathogen+Streptococcus+pneumoniae&#10;&amp;author=A.+Zapun&amp;author=J.+Philippe&amp;author=K.+A.+Abrahams&amp;author=L.+Signor&amp;author=D.+I.+Roper&amp;volume=8&amp;publication_year=2013&amp;pages=2688-2696&amp;pmid=24044435&amp;doi=10.1021/cb400575t&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B50"><span class="mixed-citation">
Zhu J., Wolf I. D., Dulberger C. L., Won H. I., Kester J. C., Judd J. A., et al.. (2021). <span class="ref-title">Spatiotemporal localization of proteins in mycobacteria</span>. <span class="ref-journal">Cell Rep.</span>
<span class="ref-vol">37</span>, 110154. doi:&#x000a0; 10.1016/j.celrep.2021.110154
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8861988/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34965429" ref="reftype=pubmed&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.celrep.2021.110154" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Rep.&amp;title=Spatiotemporal+localization+of+proteins+in+mycobacteria&amp;author=J.+Zhu&amp;author=I.+D.+Wolf&amp;author=C.+L.+Dulberger&amp;author=H.+I.+Won&amp;author=J.+C.+Kester&amp;volume=37&amp;publication_year=2021&amp;pages=110154&amp;pmid=34965429&amp;doi=10.1016/j.celrep.2021.110154&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10484409&amp;issue-id=427506&amp;journal-id=1860&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li></ul></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Frontiers in Cellular and Infection Microbiology</span> are provided here courtesy of <strong>Frontiers Media SA</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC10484409/pdf/fcimb-13-1205829.pdf" class="int-view">PDF (9.8M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/10484409/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/10484409/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="10484409" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10484409%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="i5h1OMAhZwBcyw2igDc5aNkDWRhlR9qQEIqAtv1cvoAq3Btlwd3Qo4ZkgTZgkNMQ">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10484409%2F&amp;text=Amidation%20of%20glutamate%20residues%20in%20mycobacterial%20peptidoglycan%20is%20essential%20for%20cell%20wall%20cross-linking" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10484409%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10484409/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC10484409/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/37692163/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC10484409/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/37692163/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC10484409/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/10484409/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC10484409/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
