
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Guiding bar motif of thioredoxin reductase 1 modulates enzymatic activity and inhibitor binding by communicating with the co-factor FAD and regulating the flexible C-terminal redox motif - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="D0BD41E4AC192A4500003EB1B946BF60.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="redox" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC10840350/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10840350/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Redox Biology" /><meta name="citation_title" content="Guiding bar motif of thioredoxin reductase 1 modulates enzymatic activity and inhibitor binding by communicating with the co-factor FAD and regulating the flexible C-terminal redox motif" /><meta name="citation_author" content="Wuyang Shi" /><meta name="citation_author_institution" content="School of Life and Pharmaceutical Sciences (LPS) &#x00026; Panjin Institute of Industrial Technology (PIIT), Dalian University of Technology, Panjin, 124221, China" /><meta name="citation_author" content="Shibo Sun" /><meta name="citation_author_institution" content="School of Life and Pharmaceutical Sciences (LPS) &#x00026; Panjin Institute of Industrial Technology (PIIT), Dalian University of Technology, Panjin, 124221, China" /><meta name="citation_author" content="Haowen Liu" /><meta name="citation_author_institution" content="School of Life and Pharmaceutical Sciences (LPS) &#x00026; Panjin Institute of Industrial Technology (PIIT), Dalian University of Technology, Panjin, 124221, China" /><meta name="citation_author" content="Yao Meng" /><meta name="citation_author_institution" content="School of Life and Pharmaceutical Sciences (LPS) &#x00026; Panjin Institute of Industrial Technology (PIIT), Dalian University of Technology, Panjin, 124221, China" /><meta name="citation_author" content="Kangshuai Ren" /><meta name="citation_author_institution" content="School of Life and Pharmaceutical Sciences (LPS) &#x00026; Panjin Institute of Industrial Technology (PIIT), Dalian University of Technology, Panjin, 124221, China" /><meta name="citation_author" content="Guoying Wang" /><meta name="citation_author_institution" content="School of Life and Pharmaceutical Sciences (LPS) &#x00026; Panjin Institute of Industrial Technology (PIIT), Dalian University of Technology, Panjin, 124221, China" /><meta name="citation_author" content="Minghui Liu" /><meta name="citation_author_institution" content="School of Life and Pharmaceutical Sciences (LPS) &#x00026; Panjin Institute of Industrial Technology (PIIT), Dalian University of Technology, Panjin, 124221, China" /><meta name="citation_author" content="Jiaqi Wu" /><meta name="citation_author_institution" content="School of Life and Pharmaceutical Sciences (LPS) &#x00026; Panjin Institute of Industrial Technology (PIIT), Dalian University of Technology, Panjin, 124221, China" /><meta name="citation_author" content="Yue Zhang" /><meta name="citation_author_institution" content="School of Life and Pharmaceutical Sciences (LPS) &#x00026; Panjin Institute of Industrial Technology (PIIT), Dalian University of Technology, Panjin, 124221, China" /><meta name="citation_author" content="Huang Huang" /><meta name="citation_author_institution" content="School of Life and Pharmaceutical Sciences (LPS) &#x00026; Panjin Institute of Industrial Technology (PIIT), Dalian University of Technology, Panjin, 124221, China" /><meta name="citation_author" content="Meiyun Shi" /><meta name="citation_author_institution" content="School of Life and Pharmaceutical Sciences (LPS) &#x00026; Panjin Institute of Industrial Technology (PIIT), Dalian University of Technology, Panjin, 124221, China" /><meta name="citation_author" content="Weiping Xu" /><meta name="citation_author_institution" content="School of Ocean Science and Technology (OST) &#x00026; Key Laboratory of Industrial Ecology and Environmental Engineering (Ministry of Education), Dalian University of Technology, Panjin, 124221, China" /><meta name="citation_author" content="Qiang Ma" /><meta name="citation_author_institution" content="Chinese Academy of Inspection and Quarantine, Beijing, 100176, China" /><meta name="citation_author" content="Bingbing Sun" /><meta name="citation_author_institution" content="State Key Laboratory of Fine Chemicals, School of Chemical Engineering (CE), Dalian University of Technology, Dalian, 116023, China" /><meta name="citation_author" content="Jianqiang Xu" /><meta name="citation_author_institution" content="School of Life and Pharmaceutical Sciences (LPS) &#x00026; Panjin Institute of Industrial Technology (PIIT), Dalian University of Technology, Panjin, 124221, China" /><meta name="citation_publication_date" content="2024/04" /><meta name="citation_volume" content="70" /><meta name="citation_doi" content="10.1016/j.redox.2024.103050" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC10840350/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC10840350/" /><meta name="citation_pmid" content="38277963" /><meta name="DC.Title" content="Guiding bar motif of thioredoxin reductase 1 modulates enzymatic activity and inhibitor binding by communicating with the co-factor FAD and regulating the flexible C-terminal redox motif" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Elsevier" /><meta name="DC.Contributor" content="Wuyang Shi" /><meta name="DC.Contributor" content="Shibo Sun" /><meta name="DC.Contributor" content="Haowen Liu" /><meta name="DC.Contributor" content="Yao Meng" /><meta name="DC.Contributor" content="Kangshuai Ren" /><meta name="DC.Contributor" content="Guoying Wang" /><meta name="DC.Contributor" content="Minghui Liu" /><meta name="DC.Contributor" content="Jiaqi Wu" /><meta name="DC.Contributor" content="Yue Zhang" /><meta name="DC.Contributor" content="Huang Huang" /><meta name="DC.Contributor" content="Meiyun Shi" /><meta name="DC.Contributor" content="Weiping Xu" /><meta name="DC.Contributor" content="Qiang Ma" /><meta name="DC.Contributor" content="Bingbing Sun" /><meta name="DC.Contributor" content="Jianqiang Xu" /><meta name="DC.Date" content="2024 Apr" /><meta name="DC.Identifier" content="10.1016/j.redox.2024.103050" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Guiding bar motif of thioredoxin reductase 1 modulates enzymatic activity and inhibitor binding by communicating with the co-factor FAD and regulating the flexible C-terminal redox motif" /><meta property="og:type" content="article" /><meta property="og:description" content="Thioredoxin reductase (TXNRD) is a selenoprotein that plays a crucial role in cellular antioxidant defense. Previously, a distinctive guiding bar motif was identified in TXNRD1, which influences the transfer of electrons. In this study, utilizing single ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10840350/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10840350/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/2171/">Redox Biol</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/454190/">v.70; 2024 Apr</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC10840350
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC10840350/pdf/main.pdf" class="int-view">PDF (14M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/10840350/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/10840350/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="10840350" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10840350%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="xdoECR4EeVatmgZis07RjHAnqKFrEfHGObV8qVWUDKh7gflGDquLIz1BrTXqHOZB">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10840350%2F&amp;text=Guiding%20bar%20motif%20of%20thioredoxin%20reductase%201%20modulates%20enzymatic%20activity%20and%20inhibitor%20binding%20by%20communicating%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10840350%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10840350/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC10840350/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38277963/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC10840350/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38277963/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC10840350/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/10840350/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/2171/">Redox Biol</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/454190/">v.70; 2024 Apr</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC10840350
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-redox.gif" alt="Logo of redox" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="0,30,118,50" alt="Guide for Authors" title="Guide for Authors" href="http://www.elsevier.com/journals/Redox-Biology/2213-2317/guide-for-authors" ref="reftype=publisher&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="118,30,165,50" alt="About this journal" title="About this journal" href="http://www.journals.elsevier.com/redox-biology" ref="reftype=publisher&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="165,30,286,50" alt="Explore this journal" title="Explore this journal" href="http://www.sciencedirect.com/science/journal/22132317" ref="reftype=publisher&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="0,0,499,74" alt="Redox Biology" title="Redox Biology" href="http://www.journals.elsevier.com/redox-biology" ref="reftype=publisher&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Redox Biol.</span> 2024 Apr; 70: 103050. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2024 Jan 18. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1016%2Fj.redox.2024.103050" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.redox.2024.103050</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC10840350</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38277963">38277963</a></div></div></div><h1 class="content-title">Guiding bar motif of thioredoxin reductase 1 modulates enzymatic activity and inhibitor binding by communicating with the co-factor FAD and regulating the flexible C-terminal redox motif</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shi%20W%5BAuthor%5D" id="au1" class="affpopup" co-rid="_co_idm139793146988384" co-class="co-affbox">Wuyang Shi</a>,<sup>a,</sup><sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Sun%20S%5BAuthor%5D" id="au2" class="affpopup" co-rid="_co_idm139793146987648" co-class="co-affbox">Shibo Sun</a>,<sup>a,</sup><sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Liu%20H%5BAuthor%5D" id="au3" class="affpopup" co-rid="_co_idm139793148142576" co-class="co-affbox">Haowen Liu</a>,<sup>a,</sup><sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Meng%20Y%5BAuthor%5D" id="au4" class="affpopup" co-rid="_co_idm139793138346416" co-class="co-affbox">Yao Meng</a>,<sup>a</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ren%20K%5BAuthor%5D" id="au5" class="affpopup" co-rid="_co_idm139793138344192" co-class="co-affbox">Kangshuai Ren</a>,<sup>a</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wang%20G%5BAuthor%5D" id="au6" class="affpopup" co-rid="_co_idm139793149114576" co-class="co-affbox">Guoying Wang</a>,<sup>a</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Liu%20M%5BAuthor%5D" id="au7" class="affpopup" co-rid="_co_idm139793149112352" co-class="co-affbox">Minghui Liu</a>,<sup>a</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wu%20J%5BAuthor%5D" id="au8" class="affpopup" co-rid="_co_idm139793141078288" co-class="co-affbox">Jiaqi Wu</a>,<sup>a,</sup><sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Zhang%20Y%5BAuthor%5D" id="au9" class="affpopup" co-rid="_co_idm139793142517104" co-class="co-affbox">Yue Zhang</a>,<sup>a</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Huang%20H%5BAuthor%5D" id="au10" class="affpopup" co-rid="_co_idm139793142514880" co-class="co-affbox">Huang Huang</a>,<sup>a</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shi%20M%5BAuthor%5D" id="au11" class="affpopup" co-rid="_co_idm139793149791504" co-class="co-affbox">Meiyun Shi</a>,<sup>a</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Xu%20W%5BAuthor%5D" id="au12" class="affpopup" co-rid="_co_idm139793183607104" co-class="co-affbox">Weiping Xu</a>,<sup>b</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ma%20Q%5BAuthor%5D" id="au13" class="affpopup" co-rid="_co_idm139793183604880" co-class="co-affbox">Qiang Ma</a>,<sup>c</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Sun%20B%5BAuthor%5D" id="au14" class="affpopup" co-rid="_co_idm139793138845824" co-class="co-affbox">Bingbing Sun</a>,<sup>d</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Xu%20J%5BAuthor%5D" id="au15" class="affpopup" co-rid="_co_idm139793138843600" co-class="co-affbox">Jianqiang Xu</a><sup>a,</sup><sup>&#x02217;</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139793146988384"><h3 class="no_margin">Wuyang Shi</h3><p><sup>a</sup>School of Life and Pharmaceutical Sciences (LPS) &#x00026; Panjin Institute of Industrial Technology (PIIT), Dalian University of Technology, Panjin, 124221, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shi%20W%5BAuthor%5D">Wuyang Shi</a></div></div><div id="_co_idm139793146987648"><h3 class="no_margin">Shibo Sun</h3><p><sup>a</sup>School of Life and Pharmaceutical Sciences (LPS) &#x00026; Panjin Institute of Industrial Technology (PIIT), Dalian University of Technology, Panjin, 124221, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Sun%20S%5BAuthor%5D">Shibo Sun</a></div></div><div id="_co_idm139793148142576"><h3 class="no_margin">Haowen Liu</h3><p><sup>a</sup>School of Life and Pharmaceutical Sciences (LPS) &#x00026; Panjin Institute of Industrial Technology (PIIT), Dalian University of Technology, Panjin, 124221, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Liu%20H%5BAuthor%5D">Haowen Liu</a></div></div><div id="_co_idm139793138346416"><h3 class="no_margin">Yao Meng</h3><p><sup>a</sup>School of Life and Pharmaceutical Sciences (LPS) &#x00026; Panjin Institute of Industrial Technology (PIIT), Dalian University of Technology, Panjin, 124221, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Meng%20Y%5BAuthor%5D">Yao Meng</a></div></div><div id="_co_idm139793138344192"><h3 class="no_margin">Kangshuai Ren</h3><p><sup>a</sup>School of Life and Pharmaceutical Sciences (LPS) &#x00026; Panjin Institute of Industrial Technology (PIIT), Dalian University of Technology, Panjin, 124221, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ren%20K%5BAuthor%5D">Kangshuai Ren</a></div></div><div id="_co_idm139793149114576"><h3 class="no_margin">Guoying Wang</h3><p><sup>a</sup>School of Life and Pharmaceutical Sciences (LPS) &#x00026; Panjin Institute of Industrial Technology (PIIT), Dalian University of Technology, Panjin, 124221, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wang%20G%5BAuthor%5D">Guoying Wang</a></div></div><div id="_co_idm139793149112352"><h3 class="no_margin">Minghui Liu</h3><p><sup>a</sup>School of Life and Pharmaceutical Sciences (LPS) &#x00026; Panjin Institute of Industrial Technology (PIIT), Dalian University of Technology, Panjin, 124221, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Liu%20M%5BAuthor%5D">Minghui Liu</a></div></div><div id="_co_idm139793141078288"><h3 class="no_margin">Jiaqi Wu</h3><p><sup>a</sup>School of Life and Pharmaceutical Sciences (LPS) &#x00026; Panjin Institute of Industrial Technology (PIIT), Dalian University of Technology, Panjin, 124221, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wu%20J%5BAuthor%5D">Jiaqi Wu</a></div></div><div id="_co_idm139793142517104"><h3 class="no_margin">Yue Zhang</h3><p><sup>a</sup>School of Life and Pharmaceutical Sciences (LPS) &#x00026; Panjin Institute of Industrial Technology (PIIT), Dalian University of Technology, Panjin, 124221, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Zhang%20Y%5BAuthor%5D">Yue Zhang</a></div></div><div id="_co_idm139793142514880"><h3 class="no_margin">Huang Huang</h3><p><sup>a</sup>School of Life and Pharmaceutical Sciences (LPS) &#x00026; Panjin Institute of Industrial Technology (PIIT), Dalian University of Technology, Panjin, 124221, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Huang%20H%5BAuthor%5D">Huang Huang</a></div></div><div id="_co_idm139793149791504"><h3 class="no_margin">Meiyun Shi</h3><p><sup>a</sup>School of Life and Pharmaceutical Sciences (LPS) &#x00026; Panjin Institute of Industrial Technology (PIIT), Dalian University of Technology, Panjin, 124221, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shi%20M%5BAuthor%5D">Meiyun Shi</a></div></div><div id="_co_idm139793183607104"><h3 class="no_margin">Weiping Xu</h3><p><sup>b</sup>School of Ocean Science and Technology (OST) &#x00026; Key Laboratory of Industrial Ecology and Environmental Engineering (Ministry of Education), Dalian University of Technology, Panjin, 124221, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Xu%20W%5BAuthor%5D">Weiping Xu</a></div></div><div id="_co_idm139793183604880"><h3 class="no_margin">Qiang Ma</h3><p><sup>c</sup>Chinese Academy of Inspection and Quarantine, Beijing, 100176, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ma%20Q%5BAuthor%5D">Qiang Ma</a></div></div><div id="_co_idm139793138845824"><h3 class="no_margin">Bingbing Sun</h3><p><sup>d</sup>State Key Laboratory of Fine Chemicals, School of Chemical Engineering (CE), Dalian University of Technology, Dalian, 116023, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Sun%20B%5BAuthor%5D">Bingbing Sun</a></div></div><div id="_co_idm139793138843600"><h3 class="no_margin">Jianqiang Xu</h3><p><sup>a</sup>School of Life and Pharmaceutical Sciences (LPS) &#x00026; Panjin Institute of Industrial Technology (PIIT), Dalian University of Technology, Panjin, 124221, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Xu%20J%5BAuthor%5D">Jianqiang Xu</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm139793138961904_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139793138961904_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139793138961904_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm139793138961904_ai" style="display:none"><div class="fm-affl" id="aff1"><sup>a</sup>School of Life and Pharmaceutical Sciences (LPS) &#x00026; Panjin Institute of Industrial Technology (PIIT), Dalian University of Technology, Panjin, 124221, China</div><div class="fm-affl" id="aff2"><sup>b</sup>School of Ocean Science and Technology (OST) &#x00026; Key Laboratory of Industrial Ecology and Environmental Engineering (Ministry of Education), Dalian University of Technology, Panjin, 124221, China</div><div class="fm-affl" id="aff3"><sup>c</sup>Chinese Academy of Inspection and Quarantine, Beijing, 100176, China</div><div class="fm-affl" id="aff4"><sup>d</sup>State Key Laboratory of Fine Chemicals, School of Chemical Engineering (CE), Dalian University of Technology, Dalian, 116023, China</div><div><span class="contrib-email" id="au15">Jianqiang Xu: <a href="mailto:dev@null" data-email="nc.ude.tuld@ux.gnaiqnaij" class="oemail">nc.ude.tuld@ux.gnaiqnaij</a></span> </div><div id="cor1"><sup>&#x02217;</sup>Corresponding author. <a href="mailto:dev@null" data-email="nc.ude.tuld@ux.gnaiqnaij" class="oemail">nc.ude.tuld@ux.gnaiqnaij</a></div><div id="fn1"><sup>1</sup>Shanghai Key Laboratory of Metabolic Remodeling and Health, Institute of Metabolism and Integrative Biology, Fudan University, Shanghai, 200433, China.</div><div id="fn2"><sup>2</sup>These authors contributed equally to this study.</div></div><div class="fm-article-notes hide half_rhythm" id="idm139793138961904_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2023 Nov 12; Revised 2024 Jan 5; Accepted 2024 Jan 17.</div></div><div class="permissions half_rhythm hide" id="idm139793138961904_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; 2024 The Authors. Published by Elsevier B.V.</div><div class="license half_rhythm">This is an open access article under the CC BY-NC-ND license (http://creativecommons.org/licenses/by-nc-nd/4.0/).</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Multimedia component 1</div><div><a href="/pmc/articles/PMC10840350/bin/mmc1.docx" data-ga-action="click_feat_suppl">mmc1.docx</a><span style="color:gray"> (48M)</span></div><div class="small guid">GUID: A6A741B5-1257-4180-ADBD-95F49A876514</div></div></dd></dl><dl data-length="39" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first-last">Data will be made available on request.</p></dd></dl></div><div id="abs0010" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abs0010title">Abstract</h2><!--article-meta--><div><p class="p p-first-last">Thioredoxin reductase (TXNRD) is a selenoprotein that plays a crucial role in cellular antioxidant defense. Previously, a distinctive guiding bar motif was identified in TXNRD1, which influences the transfer of electrons. In this study, utilizing single amino acid substitution and Excitation-Emission Matrix (EEM) fluorescence spectrum analysis, we discovered that the guiding bar communicates with the FAD and modulates the electron flow of the enzyme. Differential Scanning Fluorimetry (DSF) analysis demonstrated that the aromatic amino acid in guiding bar is a stabilizer for TXNRD1. Kinetic analysis revealed that the guiding bar is vital for the disulfide reductase activity but hinders the selenocysteine-independent reduction activity of TXNRD1. Meanwhile, the guiding bar shields the selenocysteine residue of TXNRD1 from the attack of electrophilic reagents. We also found that the inhibition of TXNRD1 by caveolin-1 scaffolding domain (CSD) peptides and compound LCS3 did not bind to the guiding bar motif. In summary, the obtained results highlight new aspects of the guiding bar that restrict the flexibility of the C-terminal redox motif and govern the transition from antioxidant to pro-oxidant.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Thioredoxin reductase, Thioredoxin, Selenoprotein, Guiding bar motif, Caveolin-1, LCS3</span></div></div><div id="abs0015" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abs0015title">Graphical abstract</h2><!--article-meta--><div><p class="p p-first-last"><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--></p><div class="fig  iconblock anchored whole_rhythm" id="undfig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139793183347440"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10840350_ga1.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Image 1" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10840350/bin/ga1.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139793183347440"><a target="object" rel="noopener" href="/pmc/articles/PMC10840350/figure/undfig1/?report=objectonly">Open in a separate window</a></div></div><p></p></div></div><div id="abs0020" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abs0020title">Highlights</h2><!--article-meta--><div><ul class="simple" style="list-style-type:none" id="ulist0010"><li class="a_label" id="u0010"><div class="inline_block a_label">&#x02022;</div><div id="p0010">Guiding bar residues, including W407, N418, and N419, interfere with the electron flow in TXNRD1 through FAD interaction.</div></li><li class="a_label" id="u0015"><div class="inline_block a_label">&#x02022;</div><div id="p0015">Guiding bar motif influences inhibitor binding therefore protects the C-terminal tail against the electrophilic attack.</div></li><li class="a_label" id="u0020"><div class="inline_block a_label">&#x02022;</div><div id="p0020">Synthesized peptides of caveolin-1 scaffolding domain (CSD) demonstrate an inhibitory effect on TXNRD1 activity.</div></li><li class="a_label" id="u0025"><div class="inline_block a_label">&#x02022;</div><div id="p0025">LCS3, a reversible inhibitor of disulfide reductase, inhibits TXNRD1 but has no clear interaction with guiding bar.</div></li></ul><p></p></div></div><div id="sec1" class="tsec sec"><h2 class="head no_bottom_margin" id="sec1title">1. Introduction</h2><p id="p0035" class="p p-first">Mammalian selenoprotein thioredoxin reductase 1 (TXNRD1), belonging to the pyridine nucleotide disulfide oxidoreductase family, is primarily responsible for restoring the reduced states of thioredoxin (TXN) and thioredoxin-related protein of 14&#x000a0;kDa (TRP14) [<a href="#bib1" rid="bib1" class=" bibr popnode">1</a>]. The TXN system supports several fundamental biological processes, such as facilitating DNA synthesis, repairing oxidative damage to proteins, and quenching excessive reactive oxygen species (ROS) in cell [<a href="#bib2" rid="bib2" class=" bibr popnode">2</a>,<a href="#bib3" rid="bib3" class=" bibr popnode">3</a>]. TXNRD1 contains an N-terminal glutathione reductase (GSR)-like redox motif, which comprises two catalytic residues Cys<sup>59</sup> and Cys<sup>64</sup>, and a C-terminal selenium (Se)-containing redox motif, with additional redox-active residues Cys<sup>497</sup> and Sec<sup>498</sup>. The C-terminus is generally flexible and redox-active, and has been described as the &#x0201c;catalytic tail&#x0201d; of the enzyme [<a href="#bib4" rid="bib4" class=" bibr popnode">4</a>].</p><p id="p0040">During the catalysis of TXNRD1, a non-covalently bound co-factor FAD is first converted to its reduced form FADH<sub>2</sub> by accepting two electrons from NADPH. FADH<sub>2</sub> is able to form a charge-transfer complex (CTC) with the Cys<sup>64</sup> residue, and these two states are classified as the two-electron-reduced enzymes (EH<sub>2</sub>). With the addition of the second reducing equivalent, the redox motif located at the C-terminal was reduced, resulting in the conversion of the enzyme to a four-electron-reduced enzyme (EH<sub>4</sub>) [<a href="#bib4" rid="bib4" class=" bibr popnode">4</a>]. Upon the reduction of TXN, the EH<sub>4</sub> enzyme undergoes oxidation, resulting in the formation of EH<sub>2</sub> form. Briefly, the general electron flow is: NADPH&#x02192;FAD&#x02192;CVNVGC&#x02192;GCUG&#x02192;TXN1 (or TRP14 and TXNL1) [<a href="#bib5" rid="bib5" class=" bibr popnode">[5]</a>, <a href="#bib6" rid="bib6" class=" bibr popnode">[6]</a>, <a href="#bib7" rid="bib7" class=" bibr popnode">[7]</a>]. In addition to these physiological substrates, other low-weight compounds, such as pyrroloquinoline quinone (PQQ) [<a href="#bib8" rid="bib8" class=" bibr popnode">8</a>], toxoflavin [<a href="#bib9" rid="bib9" class=" bibr popnode">9</a>], and coenzyme Q10 [<a href="#bib10" rid="bib10" class=" bibr popnode">10</a>], have been identified as efficient substrates of TXNRD1.</p><p id="p0045">The &#x0201c;guiding bar&#x0201d; motif of TXNRD, formed by Trp<sup>407</sup>, Asn<sup>418</sup>, and Asn<sup>419</sup>, was first conceptualized by Katja Becker's group [<a href="#bib11" rid="bib11" class=" bibr popnode">11</a>,<a href="#bib12" rid="bib12" class=" bibr popnode">12</a>]. The unique motif plays a vital role in guiding the movement of the catalytic tail of TXNRD1 towards substrates by interacting with residues 494&#x02013;496&#x000a0;at the C-terminus [<a href="#bib11" rid="bib11" class=" bibr popnode">11</a>,<a href="#bib12" rid="bib12" class=" bibr popnode">12</a>]. Site-directed mutations of the motif increased the catalytic efficiency of Sec-deficient TXNRD1 variants in both thioredoxin reducing and DTNB reducing [<a href="#bib12" rid="bib12" class=" bibr popnode">12</a>]. Moreover, Hondal and his colleagues reported that the rearrangement of C-terminal tail, guided by the guiding bar, is the primary mechanistic difference between cytosolic TXNRD1 and mitochondrial TXNRD2 [<a href="#bib13" rid="bib13" class=" bibr popnode">13</a>]. The presence of the motif in TXNRD1 may govern the interaction between the oxidized C-terminal redox center and the N-terminal redox center, thus facilitating redox cycling, which emphasizes the significant dependence of TXNRD1 on selenium [<a href="#bib14" rid="bib14" class=" bibr popnode">14</a>]. The guiding bar limits the access of other substrates to the N-terminus. As a result, the selenium atom must function as both an electron acceptor from the N-terminal redox center and an electron donor to substrates [<a href="#bib14" rid="bib14" class=" bibr popnode">14</a>]. The replacement of Sec with Cys must account for the reduced reactivity of both nucleophilicity of the selenol in donating electrons and electrophilicity in thiol/disulfide exchange, resulting in a significant decrease in catalytic activity [<a href="#bib15" rid="bib15" class=" bibr popnode">[15]</a>, <a href="#bib16" rid="bib16" class=" bibr popnode">[16]</a>, <a href="#bib17" rid="bib17" class=" bibr popnode">[17]</a>, <a href="#bib18" rid="bib18" class=" bibr popnode">[18]</a>, <a href="#bib19" rid="bib19" class=" bibr popnode">[19]</a>].</p><p id="p0050">The hypothesis of the guiding bar motif introduces innovative insights for several unresolved questions related to TXNRD1, such as the formation and function of selenium-compromised thioredoxin reductase-derived apoptotic proteins (SecTRAPs) [<a href="#bib20" rid="bib20" class=" bibr popnode">20</a>,<a href="#bib21" rid="bib21" class=" bibr popnode">21</a>], the rationale behind the high efficiency exhibited by the Sec-to-Cys mutant of TXNRD1 in juglone and shikonin reduction [<a href="#bib22" rid="bib22" class=" bibr popnode">22</a>,<a href="#bib23" rid="bib23" class=" bibr popnode">23</a>], and the mechanism of the resistance of TXNRD2 to electrophilic compounds [<a href="#bib24" rid="bib24" class=" bibr popnode">24</a>]. To our knowledge, solid evidence supporting the function of guiding bar in TXNRD1 catalysis remains elusive. Meanwhile, the role of the highly conserved motif in other crucial enzymes in relation to redox regulation remains unclear to date.</p><p id="p0055" class="p p-last">In this study, recombinant rat TXNRD1 and its variants with mutations in the guiding bar motif were employed to perform side-by-side biochemical comparisons, aiming to elucidate the role of the guiding bar motif in TXNRD1. The UV&#x02013;vis wavelength scanning and Excitation-Emission Matrix (EEM) fluorescence spectrum analysis of mutant variants indicated that the guiding bar communicates with the FAD and is involved in the electron transfer process of the selenoenzyme. Mutating the residues of guiding bar increases the catalytic efficiency of TXNRD1 in the reduction of juglone. Notably, the TXNRD1 variants were more susceptible to electrophilic reagents and exhibited increased production of superoxide ions from SecTRAPs. The results obtained in this study provide valuable insights into the catalytic mechanism of TXNRD1 and demonstrate that the unique guiding bar motif functions as a gearbox for tuning the flexible C-terminal tail and adjusting TXNRD1 catalysis.</p></div><div id="sec2" class="tsec sec"><h2 class="head no_bottom_margin" id="sec2title">2. Results</h2><div id="sec2.1" class="sec sec-first"><h3 id="sec2.1title">2.1. Sequence alignment of TXNRD1 from different species reveals that guiding bars are conserved</h3><p id="p0060" class="p p-first-last">Generally, TXNRD1 is a homodimeric enzyme, with two subunits forming a head-to-tail assembly [<a href="#bib4" rid="bib4" class=" bibr popnode">4</a>,<a href="#bib5" rid="bib5" class=" bibr popnode">5</a>]. To catalyze substrates, the enzyme requires the movement of the C-terminal tail from a close position to the N-terminal motif of the other subunit toward the substrates [<a href="#bib5" rid="bib5" class=" bibr popnode">5</a>]. The overall structure of TXNRD1's catalytic pocket reveals that the guiding bar motif is located parallel to the C-terminal tail (<a href="/pmc/articles/PMC10840350/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a>A and B). To verify the conservation of the motif in TXNRD1, we conducted a partial sequence alignment of selected mammalian TXNRD1 (<a href="/pmc/articles/PMC10840350/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a>C and <a href="#appsec1" rid="appsec1" class=" sec">Fig. S1</a>). Trp<sup>407</sup>, Asn<sup>418</sup>, and Asn<sup>419</sup>, which are core residues in the guiding bar motif, are highly conserved in mammalian TXNRD1. Notably, we observed that the motif is abundant in aromatic residues, such as Tyr<sup>402</sup>, Phe<sup>405</sup>, Phe<sup>406</sup>, Trp<sup>407</sup>, and Trp<sup>411</sup> in rat TXNRD1 (<a href="/pmc/articles/PMC10840350/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a>C and <a href="#appsec1" rid="appsec1" class=" sec">Fig. S1</a>). Given the crucial role of aromatic amino acids in stabilizing protein structure [<a href="#bib25" rid="bib25" class=" bibr popnode">25</a>] and facilitating electron transfer [<a href="#bib26" rid="bib26" class=" bibr popnode">26</a>,<a href="#bib27" rid="bib27" class=" bibr popnode">27</a>], it is imperative to uncover the physiological function of this motif in TXNRD1.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC10840350/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139793133816592"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10840350_gr1.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 1" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10840350/bin/gr1.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139793133816592"><a target="object" rel="noopener" href="/pmc/articles/PMC10840350/figure/fig1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC10840350/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Fig. 1</a></div><!--caption a7--><div class="caption"><p><strong>The guiding bar motif of TXNRD1 interacts with the co-factor FAD.</strong><strong>(A)</strong> Schematic diagram for the catalytic motifs of TXNRD1. GB, guiding bar motif. <strong>(B)</strong> Structure of the guiding bar motif in rat TXNRD1. The conformation was obtained from PDB: <a href="https://www.rcsb.org/structure/https://www.rcsb.org/3EAN" data-ga-action="click_feat_suppl" ref="reftype=extlink-pdb&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CPDB" target="_blank">3EAN</a>. <strong>(C)</strong> Partial sequence alignment of the guiding bar motif of mammalian TXNRD1. The aromatic amino acid residues were highlighted in yellow. <strong>(D)</strong> SDS-PAGE analysis of TXNRD1 variants. <strong>(E)</strong> 5-IAF labelling of TXNRD1 and its two mutant variants, W407A and N418A/N419A. <strong>(F)</strong> Wavelength scanning analysis. The UV&#x02013;Vis spectra of TXNRD1 and its variants (7&#x000a0;&#x003bc;M) were scanned in a range from 300&#x000a0;nm to 700&#x000a0;nm. <strong>(G)</strong> Formation of charge-transfer complexes in mutant variants of rat TXNRD1. The charge-transfer complex of TXNRD1 was determined at 540&#x000a0;nm in the presence of NADPH at indicated concentrations. <strong>(H)</strong> Excitation-Emission Matrix fluorescence spectra of TXNRD1 mutant variants. 7&#x000a0;&#x003bc;M of TXNRD1 variants were loaded into a flat-bottomed 96-well black plate for 3-D fluorescence measurements. <strong>(I)</strong> Differential Scanning Fluorimetry analysis of TXNRD1 and its mutant variants. The Tma values are showed in the right panel.</p></div></div></div></div><div id="sec2.2" class="sec"><h3 id="sec2.2title">2.2. Mutating key residues in guiding bar disaffects the reducibility of C-terminal tail in TXNRD1</h3><p id="p0065" class="p p-first-last">To investigate the unique motif in TXNRD1, we generated a series of constructs (<a href="#appsec1" rid="appsec1" class=" sec">Fig. S2</a>), with either single amino acid residue mutations or multiple residue substitutions on the core residues and the adjacent aromatic residues in the guiding bar motif (Primer list in <a href="#appsec1" rid="appsec1" class=" sec">Table S1</a>). For subsequent analysis, we employed recombinant rat TXNRD1 and its mutants, given the enhanced stability of rat TXNRD1 compared to human TXNRD1 [<a href="#bib24" rid="bib24" class=" bibr popnode">24</a>,<a href="#bib28" rid="bib28" class=" bibr popnode">28</a>]. Furthermore, rat TXNRD1 is commonly utilized in TXNRD1 related enzymatic studies, and an overall sequences alignment revealed a high homology between the rat and human TXNRD1 (<a href="#appsec1" rid="appsec1" class=" sec">Fig. S3</a>). The TXNRD1 variants were produced in <em>E. coli</em> BL21 (DE3) <em>gor</em><sup><em>&#x02212;</em></sup> cells, co-expressing with the bacterial SECIS element for selenocysteine insertion [<a href="#bib29" rid="bib29" class=" bibr popnode">29</a>,<a href="#bib30" rid="bib30" class=" bibr popnode">30</a>]. SDS-PAGE analysis clearly showed a strong band with a molecular weight (MW) of 55&#x000a0;kDa, corresponding to the subunit of TXNRD1 (<a href="/pmc/articles/PMC10840350/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a>D). Given the guiding bar motif was reported to regulate the movement of the C-terminal tail of TXNRD1 [<a href="#bib11" rid="bib11" class=" bibr popnode">11</a>], we initially examined whether the selenocysteine residue of the mutant variants can be reduced and further labeled by 5-IAF. In contrast to oxidized forms of TXNRD1, NADPH-pre-reduced TXNRD1 variants, including W407A and N418A/N419A mutants, can be labeled by 5-IAF (<a href="/pmc/articles/PMC10840350/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a>E), indicating that the C-terminal selenolthiol of the TXNRD1 variants with mutations at the guiding bar motif is functional to be reduced by NADPH through the reductive half-reaction.</p></div><div id="sec2.3" class="sec"><h3 id="sec2.3title">2.3. Guiding bar of TXNRD1 interacts with the co-factor FAD and impacts the electron flow of the enzyme</h3><p id="p0070" class="p p-first">The spectrum of UV&#x02013;Vis for all TXNRD1 variants displays a typical peak at 463&#x000a0;nm, which represents the non-covalently bound FAD (<a href="/pmc/articles/PMC10840350/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a>F, <a href="#appsec1" rid="appsec1" class=" sec">Fig. S4</a>). Meanwhile, TXNRD1 variants exhibit a characteristic absorbance shoulder at &#x0223c;500&#x000a0;nm, which is similar to wild-type enzyme. The formation of charge-transfer complex (CTC) exhibits a strong peak at 540&#x000a0;nm and profoundly influences the absorbance spectra of TXNRD1, leading to the absence of the shoulder at &#x0223c;500&#x000a0;nm [<a href="#bib31" rid="bib31" class=" bibr popnode">31</a>,<a href="#bib32" rid="bib32" class=" bibr popnode">32</a>]. This phenotype indicated that the guiding bar mutant did not result in a spontaneous formation of charge-transfer complex between Cys<sup>64</sup> and FAD in the absence of NADPH.</p><p id="p0075">We subsequently examined the capacity of TXNRD1 variants to form charge-transfer complexes when NADPH was introduced into the system. Evidently, the formation of the charge-transfer complex was increased in the Y402F, W407F, and N418A/N419A mutants compared to the wild-type TXNRD1, while F406A mutant exhibited a lower potential (<a href="/pmc/articles/PMC10840350/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a>G and <a href="#appsec1" rid="appsec1" class=" sec">Fig. S5</a>).</p><p id="p0080" class="p p-last">Considering the potential differences in charge-transfer complex formation, we conducted a thorough analysis of the TXNRD1 variants using Excitation-Emission Matrix (EEM) fluorescence spectrum analysis [<a href="#bib33" rid="bib33" class=" bibr popnode">33</a>,<a href="#bib34" rid="bib34" class=" bibr popnode">34</a>], which may provide additional insights into the mechanisms and properties of TXNRD1. Typical enzyme-bound flavin fluorescence at Em<sub>max</sub>&#x000a0;&#x0223c;520&#x000a0;nm has three excitation peaks at Ex<sub>max</sub>&#x000a0;&#x02248;280&#x000a0;nm, &#x02248;370&#x000a0;nm, and &#x02248;470&#x000a0;nm [<a href="#bib34" rid="bib34" class=" bibr popnode">34</a>]. We detected three excitation peaks of Ex<sub>max</sub> at 320&#x000a0;nm, 380&#x000a0;nm, and 430&#x000a0;nm for the wild-type TXNRD1 at Em<sub>max</sub>&#x000a0;&#x0223c;520&#x000a0;nm (<a href="/pmc/articles/PMC10840350/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a>H). Conversely, the Y402F mutation resulted in the loss of the Ex<sub>max</sub> peaks at 430&#x000a0;nm, while the F405A, F406A and N418A/N419A mutations of TXNRD1 exhibited a new peak at approximately 390&#x02013;400&#x000a0;nm. Y402A, W407F, W407A, W411F and W411A mutants presented similar excitation peaks compared to the wild-type enzyme (<a href="#appsec1" rid="appsec1" class=" sec">Fig. S6</a>). These results demonstrated the communication between the guiding bar and its neighboring aromatic residues with the FAD co-factor, and mutations in these residues have a profound impact on the electron flow of the enzyme.</p></div><div id="sec2.4" class="sec"><h3 id="sec2.4title">2.4. Mutation of the aromatic amino acid in guiding bar motif affects the thermostability of TXNRD1</h3><p id="p0085" class="p p-first-last">We then examined the thermostability of TXNRD1 variants using Differential Scanning Fluorimetry (DSF) analysis [<a href="#bib35" rid="bib35" class=" bibr popnode">35</a>,<a href="#bib36" rid="bib36" class=" bibr popnode">36</a>]. The wild-type TXNRD1 exhibited a Tma of 68.03&#x000a0;&#x000b0;C (<a href="/pmc/articles/PMC10840350/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a>I). Remarkably, mutations of Y402A, F406A, W407A and W411A strongly decreased the thermostability of TXNRD1 (&#x00394;Tma &#x0003e;5&#x000a0;&#x000b0;C). Additionally, N418A/N419A exhibited a &#x00394;Tma of 4.1&#x000a0;&#x000b0;C. These data strongly suggest that the guiding bar also acts as a stabilizer for TXNRD1. We speculated that the phenotype is attributed to the aromatic side chain of these residues. Mutating tyrosine (Y402) or tryptophan (W407, W411) residues to alanine (Y402A, W407A, and W411A) had a more pronounced impact on thermostability than mutating them to phenylalanine (Y402F, W407F, and W411F) (<a href="/pmc/articles/PMC10840350/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a>I). Collectively, these results revealed that the guiding bar motif is critical for ensuring the thermostability of TXNRD1.</p></div><div id="sec2.5" class="sec"><h3 id="sec2.5title">2.5. Guiding bar is vital for the disulfide bond reduction activity of TXNRD1</h3><p id="p0090" class="p p-first">TXNRD1 has a broad substrate spectrum and several well-characterized substrates, including TXN, TRP14, DTNB, 9,10-PQ, toxoflavin, and juglone [<a href="#bib9" rid="bib9" class=" bibr popnode">9</a>,<a href="#bib22" rid="bib22" class=" bibr popnode">22</a>,<a href="#bib37" rid="bib37" class=" bibr popnode">37</a>,<a href="#bib38" rid="bib38" class=" bibr popnode">38</a>]. In this experiment, we aimed to determine the kinetic parameters of TXNRD1 and its variants when mutating the guiding bar. The activity of TXNRD1 is highly dependent on Sec<sup>498</sup>, which is encoded by the UGA stop codon. Although several methods have been proposed [<a href="#bib39" rid="bib39" class=" bibr popnode">[39]</a>, <a href="#bib40" rid="bib40" class=" bibr popnode">[40]</a>, <a href="#bib41" rid="bib41" class=" bibr popnode">[41]</a>, <a href="#bib42" rid="bib42" class=" bibr popnode">[42]</a>, <a href="#bib43" rid="bib43" class=" bibr popnode">[43]</a>], it remains still challenging to produce high-quality selenoproteins due to unwanted premature termination of protein synthesis and misincorporation at the position of selenocysteine [<a href="#bib44" rid="bib44" class=" bibr popnode">44</a>,<a href="#bib45" rid="bib45" class=" bibr popnode">45</a>]. To address the incorporation ratio of selenocysteine, we utilized ICP-MS to determine the selenium content of each TXNRD1 variant. The selenium content of wild-type TXNRD1 is 35&#x000a0;%, accompanied by a specific activity of 26.8 U/mg in the DTNB reduction assay. The selenium contents of TXNRD1 variants range from 16&#x000a0;% to 26&#x000a0;% (<a href="/pmc/articles/PMC10840350/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>A). Kinetic parameters were subsequently determined and normalized based on the respective selenium contents as indicated.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC10840350/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139793134902528"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10840350_gr2.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 2" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10840350/bin/gr2.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139793134902528"><a target="object" rel="noopener" href="/pmc/articles/PMC10840350/figure/fig2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC10840350/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Fig. 2</a></div><!--caption a7--><div class="caption"><p><strong>Mutating the guiding bar motif increases the Sec-independent activity of TXNRD1.</strong><strong>(A)</strong> Determination of selenium contents of TXNRD1 variants using ICP-MS. The selenium contents of TXNRD1 and its mutant variants (0.2&#x000a0;mg/mL) were determined by ICP-MS. <strong>(B)</strong> Kinetic parameters of TXNRD1 variants using different substrates. a, The activity was normalized based on the Se content. <strong>(C)</strong> Schematic diagram of the C-terminal and N-terminal TXNRD1 mutants used in this study. The redox active cysteine and selenocysteine were labeled. <strong>(D)</strong> Sec-dependent and -independent reductions of TXNRD1. <strong>(E)</strong> Relative activity of TXNRD1 with mutations at the N-terminal redox motif. <strong>(F)</strong> Relative NADPH oxidation activity of TXNRD1 and its variants. 100&#x000a0;&#x003bc;M Shikonin, plumbagin and menadione were used as the substrates of TXNRD1 (30&#x000a0;nM) in the presence of 200&#x000a0;&#x003bc;M NADPH.</p></div></div></div><p id="p0095" class="p p-last">In the reduction of disulfide bond using substrates such as TXN1, TRP14, and DTNB, most variants exhibited a slight decrease in catalytic efficiency compared with wild-type TXNRD1. Specifically, in TXN1 reduction, mutants such as Y402A, W407F and W407A increased the <em>k</em><sub>cat</sub> values but lowered the enzyme's affinity for TXN, resulting in decreased catalytic efficiency (approximately 50&#x000a0;% for Y402A, 90&#x000a0;% for W407F, and 80&#x000a0;% for W407A). In contrast to wild-type TXNRD1, the <em>K</em><sub>m</sub> values of TXNRD1 variants are essentially unchanged in both DTNB reduction and cystine-coupled TRP14 reduction. However, <em>k</em><sub>cat</sub> numbers were lower, resulting in much lower <em>k</em><sub>cat</sub>/<em>K</em><sub>m</sub> values (<a href="/pmc/articles/PMC10840350/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>B). Simultaneously, we observed a notable reduction in the efficiency of F406A in reducing three substrates by approximately 3&#x02013;4 fold. Meanwhile the N418A/N419A is the only TXNRD1 variant that increases the TRP14 reduction, aligning with its capacity to form a CTC (<a href="/pmc/articles/PMC10840350/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a>G). Collectively, mutations in the guiding bar motif of TXNRD1 resulted in a modest decrease in catalytic efficiency of TXNRD1's disulfide reductase activity. This decrease can be attributed to a lower affinity for the substrates and a reduced turnover rate in response to the substrates.</p></div><div id="sec2.6" class="sec"><h3 id="sec2.6title">2.6. Guiding bar motif hinders Sec-independent reduction of juglone on TXNRD1</h3><p id="p0100" class="p p-first">The reduction of toxoflavin and 9,10-PQ by TXNRD1 has been previously reported to be selenium-dependent [<a href="#bib9" rid="bib9" class=" bibr popnode">9</a>,<a href="#bib37" rid="bib37" class=" bibr popnode">37</a>]. In this study, we observed that the U498C variant of TXNRD1 exhibited a comparatively lower activity to reduce 9,10-PQ and toxoflavin (<a href="/pmc/articles/PMC10840350/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>C and D), indicating that these two substrates may not strictly depend on selenium for their reduction by TXNRD1. Nevertheless, wild-type TXNRD1 displayed significantly higher turnover numbers in reducing these substrates compared to Sec-deficient variants (<a href="/pmc/articles/PMC10840350/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>D). Therefore, the turnover numbers for the reduction of 9,10-PQ and toxoflavin were also adjusted based on the selenium contents.</p><p id="p0105">Remarkably, the catalytic efficiency (<em>k</em><sub>cat</sub>/<em>K</em><sub>m</sub>) in both toxoflavin and 9,10-PQ reduction notably increased in W407F, W407A and N418A/N419A mutants of TXNRD1 (<a href="/pmc/articles/PMC10840350/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>B). Intriguingly, the turnover numbers of certain mutants in toxoflavin reduction either remained unchanged or decreased. For instance, the <em>k</em><sub>cat</sub> value for both Y402A and W411A is approximately 85&#x000a0;% of the wild-type enzyme, while the <em>K</em><sub>m</sub> values are roughly 3.5-fold lower than the W.T., resulting in an increase in their catalytic efficiency. This phenomenon was also observed in the 9,10-PQ reduction, where the <em>K</em><sub>m</sub> values of the mutant variants were generally 2-3-fold lower than those of the wild-type enzyme (<a href="/pmc/articles/PMC10840350/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>B).</p><p id="p0110">Interestingly, we found a noticeable increase in the catalytic activity of the guiding bar mutant of TXNRD1 for juglone reduction (<a href="/pmc/articles/PMC10840350/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>B). The activity of juglone reduction was affected by W407F, the <em>k</em><sub>cat</sub> increased for approximately 3-fold whereas <em>K</em><sub>m</sub> was decreased for &#x0223c;1.3-fold, thereby resulting in an efficiency of &#x0223c; 4-fold higher than the wild-type TXNRD1. As previously demonstrated, the reduction of juglone by TXNRD1 is supported in a Sec-independent manner and is believed to primarily occur through the N-terminal -CVNVGC-/FAD motif [<a href="#bib22" rid="bib22" class=" bibr popnode">22</a>,<a href="#bib46" rid="bib46" class=" bibr popnode">46</a>]. These results demonstrate that the TXNRD1 variants with the guiding bar mutations are all functional in terms of NADPH usage and FAD binding, and exhibit increased catalytic efficiency in terms of Sec-independent reduction.</p><p id="p0115">It has been proposed that the guiding bar residues prevent electron leakage from the N-terminal redox center [<a href="#bib12" rid="bib12" class=" bibr popnode">12</a>]. This offers plausible explanation for the increase in catalytic efficiency in Sec-independent reduction (like juglone reduction) instead of the disulfide bond (such as the Cys<sup>32</sup>-Cys<sup>35</sup> of TXN) reduction in guiding bar mutated TXNRD1. The mutations appear to facilitate greater accessibility of substrates, particularly small molecules, to the N-terminus, allowing them to undergo reduction by the N-terminal -CVNVGC-/FAD motif. This is evidenced by the Cys<sup>59</sup> and Cys<sup>64</sup> mutants still exhibiting reduced activity in juglone reduction (<a href="/pmc/articles/PMC10840350/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>E).</p><p id="p0120" class="p p-last">Furthermore, the heightened Sec-independent activity of TXNRD1 with guiding bar residue mutations was also observed in other characterized Sec-independent substrates previously reported [<a href="#bib47" rid="bib47" class=" bibr popnode">[47]</a>, <a href="#bib48" rid="bib48" class=" bibr popnode">[48]</a>, <a href="#bib49" rid="bib49" class=" bibr popnode">[49]</a>], such as plumbagin and menadione (<a href="/pmc/articles/PMC10840350/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>F). These collective findings strongly support the conclusion that the guiding bar motif restricts the Se-independent reduction activity of TXNRD1.</p></div><div id="sec2.7" class="sec"><h3 id="sec2.7title">2.7. Guiding bar motif protects the Sec-containing C-terminal tail from electrophilic reagents like TRi-1</h3><p id="p0125" class="p p-first">In the catalytic process of TXNRD1, Sec<sup>498</sup> acts as the attacking residue for the TXN1-disulfide (Cys<sup>32</sup> and Cys<sup>35</sup>). Consequently, the solvent-exposed Sec<sup>498</sup> residue can be easily conjugated by electrophiles [<a href="#bib50" rid="bib50" class=" bibr popnode">50</a>,<a href="#bib51" rid="bib51" class=" bibr popnode">51</a>]. Given the electron flows within the enzyme may be affected the guiding bar, our subsequent objective was to examine the impact of the guiding bar motif on inhibitor binding. We selected TRi-1 [<a href="#bib21" rid="bib21" class=" bibr popnode">21</a>,<a href="#bib52" rid="bib52" class=" bibr popnode">52</a>], RSL3 [<a href="#bib53" rid="bib53" class=" bibr popnode">53</a>,<a href="#bib54" rid="bib54" class=" bibr popnode">54</a>], auranofin [<a href="#bib55" rid="bib55" class=" bibr popnode">55</a>], and cisplatin [<a href="#bib56" rid="bib56" class=" bibr popnode">56</a>] as inhibitors of TXNRD1 in this scenario. Surprisingly, mutation of the guiding bar residues increased the inhibition rate of TRi-1 on TXNRD1 (<a href="/pmc/articles/PMC10840350/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Fig. 3</span></a>A). After incubating with 0.5&#x000a0;&#x003bc;M TRi-1 for 15&#x000a0;min, the wild-type TXNRD1 retained approximately 30 % activity, whereas the TXNRD1 variants almost completely lost their activity (the residual activity &#x0003c;10 %). The inhibition of RSL3 on TXNRD1 showed a similar pattern to TRi-1, indicating that the guiding bar motif prevents the selenocysteine from reacting with electrophilic reagents. However, the inhibition of auranofin and cisplatin on TXNRD1 is not significantly affected by guiding bar mutants (<a href="/pmc/articles/PMC10840350/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Fig. 3</span></a>A), confirming the preventive effect of guiding bar on TXNRD1 is focused on the selenocysteine-containing tail.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig3" co-legend-rid="lgnd_fig3"><a href="/pmc/articles/PMC10840350/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139793145941408"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10840350_gr3.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 3" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10840350/bin/gr3.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139793145941408"><a target="object" rel="noopener" href="/pmc/articles/PMC10840350/figure/fig3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig3"><div><a class="figpopup" href="/pmc/articles/PMC10840350/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3">Fig. 3</a></div><!--caption a7--><div class="caption"><p><strong>Guiding bar motif of TXNRD1 protects the Sec-containing C-terminal tail from attacks of electrophilic reagents.</strong><strong>(A)</strong> Inhibitory effect of electrophilic reagents on TXNRD1. NADPH-reduced TXNRD1 (0.2&#x000a0;&#x003bc;M) was incubated separately with TRi-1, RSL3 and auranofin for 15&#x000a0;min, or with cisplatin for 60&#x000a0;min. The residual activity of TXNRD1 was determined by using DTNB reducing assay. <strong>(B)</strong> Quantitative analysis of SecTRAPs. The reaction mixtures contained 50&#x000a0;nM TXNRD1 and its variants, 200&#x000a0;&#x003bc;M NADPH, 1&#x000a0;mM epinephrine and either 10&#x000a0;&#x003bc;M juglone or 100&#x000a0;&#x003bc;M shikonin in TE buffer with or without 10 units/well SOD as indicated. Changes in absorbance were measured simultaneously at both 340&#x000a0;nm and 480&#x000a0;nm. <strong>(C)</strong> Mechanism for the superoxide ions detection. <strong>(D)</strong> Formation of the TXNRD1-TXN1 complex induced by cisplatin. 1&#x000a0;&#x003bc;M of TXNRD1 or its variants were incubated with 8&#x000a0;&#x003bc;M TXN1 and 100&#x000a0;&#x003bc;M cisplatin at room temperature for 16&#x000a0;h. The samples were then analyzed by non-reducing SDS-PAGE followed by CBB stain. <strong>(E)</strong> Formation of the TXNRD1-TXN1 complex induced by NADPH. 1&#x000a0;&#x003bc;M of TXNRD1 or its variants were incubated with 8&#x000a0;&#x003bc;M TXN1 and 200&#x000a0;&#x003bc;M NADPH at room temperature for 16&#x000a0;h. The samples were then analysis by a non-reducing Western blot. L.E., long exposure; S.E., short exposure.</p></div></div></div><p id="p0130" class="p p-last">Given the guiding bar motif of TXNRD1 in modulating electron flow and electrophile modification, we were intrigued to explore its association with inhibitor binding and the formation of SecTRAPs. SecTRAPs are known to be formed from TXNRD1 after electrophilic targeting at its Sec residue or removing the catalytic residue at the C-terminus of TXNRD1 [<a href="#bib20" rid="bib20" class=" bibr popnode">20</a>]. Numerous intensive studies have revealed that SecTRAPs are devoid of TXN reductase activity but retain the inherent NADPH oxidation activity, resulting in the production of superoxide anion and hydrogen peroxide [<a href="#bib46" rid="bib46" class=" bibr popnode">46</a>,<a href="#bib57" rid="bib57" class=" bibr popnode">57</a>]. We demonstrated that the mutation at the guiding bar motif retained the superoxide anion production activity of TXNRD1 from SecTRAPs. The addition of superoxide dismutase (SOD) suppressed the oxidation of adrenaline to adrenochrome by superoxide anions (<a href="/pmc/articles/PMC10840350/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Fig. 3</span></a>B and C). Furthermore, W407A and N418A/N419A mutants increase the production of superoxide anions (<a href="/pmc/articles/PMC10840350/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Fig. 3</span></a>B). These results are consistent with the increased Sec-independent reduction activity of TXNRD1 variants when the guiding bar was mutated.</p></div><div id="sec2.8" class="sec"><h3 id="sec2.8title">2.8. TXNRD1 variants with guiding bar mutation continue to form complexes with TXN1</h3><p id="p0135" class="p p-first-last">Numerous studies have shown that cisplatin forms a complex with TXN1 and TXNRD1, known as TXN1-TXNRD1 (TTR) complex [<a href="#bib56" rid="bib56" class=" bibr popnode">56</a>], which can also be initiated by NADPH [<a href="#bib34" rid="bib34" class=" bibr popnode">34</a>]. A non-reducing SDS-PAGE analysis revealed that the mutations in the guiding bar did not have profound effects on the TXN1-TXNRD1 complex induced by cisplatin (<a href="/pmc/articles/PMC10840350/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Fig. 3</span></a>D and <a href="#appsec1" rid="appsec1" class=" sec">Fig. S7</a>). Subsequently, we verified the formation of the TXN1-TXNRD1 complex induced by NADPH. Unfortunately, no noticeable TXN1-TXNRD1 complex was observed in non-reducing SDS-PAGE using CBB stain (<a href="#appsec1" rid="appsec1" class=" sec">Fig. S8</a>). So, we performed non-reducing Western blotting to detect the formation of TXN1-TXNRD1 complexes (<a href="/pmc/articles/PMC10840350/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Fig. 3</span></a>E and <a href="#appsec1" rid="appsec1" class=" sec">Fig. S9</a>). The guiding bar mutants of TXNRD1 exhibited the ability to form complexes with TXN1 in the presence of NADPH. This is consistent with the result obtained from 5-IAF labeling that the guiding bar mutant of the enzyme could be reduced by NADPH through the reductive half-reaction and further reduce its substrates. These results suggest that mutations in the guiding bar motif of TXNRD1 still lead to the formation of a stable complex with TXN1 in the presence of NADPH or cisplatin.</p></div><div id="sec2.9" class="sec"><h3 id="sec2.9title">2.9. Inhibiting TXNRD1 activity by CSD peptide</h3><p id="p0140" class="p p-first">In addition to its role in the catalysis process of TXNRD1, we noticed that the guiding bar motif is also pre-defined as a caveolin binding motif (CBM, amino acids 402&#x02013;411). This motif is characterized by its high content of aromatic amino acids and interacts with the scaffolding protein caveolin-1. Previously, Volonte and Galbiati first reported the inhibition of TXNRD1 by caveolin-1 [<a href="#bib58" rid="bib58" class=" bibr popnode">58</a>]. The scaffolding domain of caveolin-1 (amino acids 82&#x02013;101) directly binds to the CBM of TXNRD1 [<a href="#bib58" rid="bib58" class=" bibr popnode">58</a>]. However, it is still argued whether the caveolin scaffolding domain (CSD) interacts with the CBM motif of proteins, as this motif is normally buried internally [<a href="#bib59" rid="bib59" class=" bibr popnode">59</a>].</p><p id="p0145">Here, we investigated the interaction between CSD and TXNRD1 using a synthesized peptide (<a href="/pmc/articles/PMC10840350/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>A). CAV-X was a peptide commonly used as a negative control for the CSD-derived peptide [<a href="#bib60" rid="bib60" class=" bibr popnode">[60]</a>, <a href="#bib61" rid="bib61" class=" bibr popnode">[61]</a>, <a href="#bib62" rid="bib62" class=" bibr popnode">[62]</a>, <a href="#bib63" rid="bib63" class=" bibr popnode">[63]</a>]. The peptide is hydrophobic and is dissolved in DMSO. We initially verified the stability of the peptide in the water-based buffer system. After being incubated in TE buffer for 1&#x000a0;h, no significant changes were observed in the turbidity and absorption peak of 20&#x000a0;&#x003bc;M CSD peptide (<a href="/pmc/articles/PMC10840350/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>B). We then determined whether the CSD could accept the electrons from TXNRD1. When compared with plumbagin, a substrate of TXNRD1 [<a href="#bib48" rid="bib48" class=" bibr popnode">48</a>], we found that the CSD peptide is not a proper substrate for TXNRD1 (<a href="/pmc/articles/PMC10840350/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>C). However, the CSD peptide inhibited the DTNB reducing activity of TXNRD1, as well as the juglone and 9,10-PQ reduction activity (<a href="/pmc/articles/PMC10840350/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>D). The inhibition of CSD on the TXN reductase activity of TXNRD1 is similar to that of control peptide CAV-X (<a href="/pmc/articles/PMC10840350/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>D). These results demonstrated that CSD peptide inhibits the activity of recombinant TXNRD1 <em>in vitro</em>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig4" co-legend-rid="lgnd_fig4"><a href="/pmc/articles/PMC10840350/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139793137413680"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10840350_gr4.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 4" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10840350/bin/gr4.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139793137413680"><a target="object" rel="noopener" href="/pmc/articles/PMC10840350/figure/fig4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig4"><div><a class="figpopup" href="/pmc/articles/PMC10840350/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4">Fig. 4</a></div><!--caption a7--><div class="caption"><p><strong>Caveolin-1 scaffolding domain (CSD) peptide inhibits TXNRD1 activity. (A)</strong> Sequences of CSD, CAV-X, CSD-D1, CSD-D2, and CSD-D3 peptides. <strong>(B)</strong> Stability analysis of CSD peptides through wavelength scanning. The CSD peptide was diluted in TE buffer with concentrations ranging from 0 to 40&#x000a0;&#x003bc;M. The spectrum of CSD peptide was assayed at 0&#x000a0;min and 60&#x000a0;min, respectively. <strong>(C)</strong> Electron transfer activity of TXNRD1 upon CSD peptide. The reaction mixture contained 30&#x000a0;nM TXNRD1, 200&#x000a0;&#x003bc;M NADPH, 10&#x000a0;&#x003bc;M CSD peptide, or 30&#x000a0;&#x003bc;M plumbagin. The NADPH oxidation was measured to determine the potential electron transfer from TXNRD1 to the CSD peptide or plumbagin. <strong>(D)</strong> Inhibition of CSD peptide on TXNRD1. The effects of CSD peptide on TXNRD1-mediated reduction were assayed using TXN1, DTNB, juglone and 9,10-PQ as substrates. The activity of TXNRD1 was measured using the standard method, with the addition of indicated concentrations of CSD peptide. <strong>(E)</strong> Effects of truncated CSD peptides (D1/D2/D3) on the activity of TXNRD1. The effects of CSD-D1, D2, and D3 peptides on TXNRD1 activity were determined using DTNB reduction assay. <strong>(F</strong>&#x02013;<strong>G)</strong> Analysis of CSD-derived peptides on TXNRD1 activity. The DTNB reducing activity of TXNRD1 variants in the presence of CSD peptide (0&#x02013;20&#x000a0;&#x003bc;M) and CSD-derived peptides (20&#x000a0;&#x003bc;M) was assayed.</p></div></div></div><p id="p0150">We next wanted to further investigate the inhibition details of the CSD peptide on TXNRD1. Generally, the inhibitors of TXNRD1 harbor electrophilic properties and target the Sec residue of TXNRD1. To clarify the inhibition mechanism, we employed a desalting column to eliminate the CSD-peptide from the incubation system. This method is commonly utilized to detect the irreversible inhibition of TXNRD1 [<a href="#bib21" rid="bib21" class=" bibr popnode">21</a>,<a href="#bib22" rid="bib22" class=" bibr popnode">22</a>]. However, we did not detect any TXNRD1 activity in the elution fractions (<a href="#appsec1" rid="appsec1" class=" sec">Fig. S10</a>), which was also observed in the affinity chromatography using either His-tagged TXNRD1 or His-tagged CSD peptide (<a href="#appsec1" rid="appsec1" class=" sec">Fig. S10</a>). Unfortunately, we are currently unable to verify the inhibition mechanism of the CSD peptide on TXNRD1. However, we have demonstrated that this inhibition exhibits a slightly time-dependent character (<a href="#appsec1" rid="appsec1" class=" sec">Fig. S11</a>).</p><p id="p0155" class="p p-last">The FTTFTVT is considered the core domain of the CSD peptide [<a href="#bib62" rid="bib62" class=" bibr popnode">62</a>]. We showed that neither domain 1 of CSD (CSD-D1), nor domains 2 and 3 (CSD-D2 and CSD-D3) (<a href="/pmc/articles/PMC10840350/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>A) exhibited any inhibitory effect on TXNRD1 (<a href="/pmc/articles/PMC10840350/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>E), indicating that the complete sequence of the CSD-peptide is important for the inhibition of TXNRD1. Furthermore, using the guiding bar mutant of TXNRD1, we demonstrated that the inhibition of CSD derived peptide on TXNRD1 is not affected by the single amino acid mutant of TXNRD1 (<a href="/pmc/articles/PMC10840350/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>F and G). Taken together, these results indicated that the CSD peptide inhibits TXNRD1 activity <em>in vitro</em>, but not by binding to the CBM of TXNRD1.</p></div><div id="sec2.10" class="sec"><h3 id="sec2.10title">2.10. LCS3 inhibits recombinant TXNRD1 in a reversible manner via the nitro group</h3><p id="p0160" class="p p-first">Recently, Lockwood and colleagues demonstrated the selective growth impairment of human lung adenocarcinoma (LUAD) cells by a compound named LCS3, attributed to its inhibition of cellular TXNRD1 [<a href="#bib64" rid="bib64" class=" bibr popnode">64</a>]. We herein investigated the potential effect of LCS3 on TXNRD1 by using recombinantly expressed mutants of the enzyme (<a href="/pmc/articles/PMC10840350/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>A). Clearly, LCS3 exhibited strong inhibition of the physiological TXN reductase activity on the recombinant TXNRD1 (<a href="/pmc/articles/PMC10840350/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>B). The IC<sub>50</sub> value of LCS3 for the TXN reductase activity is 10.7&#x000a0;&#x003bc;M. However, compared with the TXN reductase activity, the inhibition of LCS3 on the TRP14 reductase activity of TXNRD1 is relatively low. 20&#x000a0;&#x003bc;M LCS3 in the reaction mixture still retained over 90&#x000a0;% of the activity of the TRP14-coupled cystine reduction of TXNRD1 (<a href="/pmc/articles/PMC10840350/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>B). Meanwhile, LCS3 exhibited an inhibitory effect on the GSR, with an IC<sub>50</sub> value of 28.6&#x000a0;&#x003bc;M (<a href="/pmc/articles/PMC10840350/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>C), which is consistent with previously reported findings [<a href="#bib64" rid="bib64" class=" bibr popnode">64</a>]. The reversible inhibition was also confirmed to be non-time-dependent (<a href="/pmc/articles/PMC10840350/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>D). To further confirm the reversible inhibition, we removed the free LCS3 by using a desalting column. The enzyme activity was significantly rescued compared with the intact enzyme (<a href="/pmc/articles/PMC10840350/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>E), indicating that LCS3 is a reversible inhibitor of TXNRD1.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig5" co-legend-rid="lgnd_fig5"><a href="/pmc/articles/PMC10840350/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139793149947520"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10840350_gr5.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 5" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10840350/bin/gr5.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139793149947520"><a target="object" rel="noopener" href="/pmc/articles/PMC10840350/figure/fig5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig5"><div><a class="figpopup" href="/pmc/articles/PMC10840350/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5">Fig. 5</a></div><!--caption a7--><div class="caption"><p><strong>LCS3 is a non-covalent inhibitor of TXNRD1 particularly inhibiting its TXN1 reductase activity.</strong><strong>(A)</strong> Chemical structure of LCS3. <strong>(B)</strong> Inhibition of the TXN1 reductase activity of TXNRD1 by LCS3. The reaction mixture contained 50&#x000a0;nM TXNRD1, 10&#x000a0;&#x003bc;M TXN, 200&#x000a0;&#x003bc;M NADPH, 160 &#x003bc;M insulin, and various concentrations of LCS3. <strong>(C)</strong> LCS3 inhibits the GSSG reduction activity of GSR. The reaction mixture contained 2&#x000a0;nM GSR, 1&#x000a0;mM GSSG, and various concentrations of LCS3. <strong>(D)</strong> The time-course inhibition of LCS3 on TXNRD1. 50&#x000a0;&#x003bc;M LCS3 was incubated with 20&#x000a0;nM TXNRD1 for the indicated time. The final LCS3 concentration is 25&#x000a0;&#x003bc;M. The activity of TXNRD1 was measured using the DTNB reducing assay. <strong>(E)</strong> Restored TXNRD1 after removing the LCS3 through desalting. 20&#x000a0;nM TXNRD1 was incubated with 50&#x000a0;&#x003bc;M LCS3 and a NAP-5 desalting column was used for removing the free LCS3 from the incubation system. <strong>(F)</strong> Effect of LCS3 on the reducing activity of TXNRD1 independent of disulfide exchange. TXNRD1 activity in the presence of LCS3 was assayed by measuring the reduction of 9,10-PQ, juglone, and toxoflavin. <strong>(G)</strong> Inhibition of TXNRD1 and GSR activity by LCS3 analogue LCS3-&#x00394;N. TXNRD1 activity was determined by DTNB reducing and GSR activity was assayed by GSSG reduction. <strong>(H)</strong> TXNRDs activity and protein levels in the cell lysate of HCT116&#x000a0;cells treated with LCS3. HCT116 cells were treated with various concentrations of LCS3 for 4&#x000a0;h. <strong>(I)</strong> Inhibition of Sec-deficient TXNRD1 by LCS3. <strong>(J, K)</strong> Inhibition of the TXNRD1 with guiding bar residue mutation by LCS3. The activities of TXNRD1 and its variants were determined by DTNB reducing assay. The values of area under curve are shown in (J). <strong>(L)</strong> Inhibition of low-weight thioredoxin reductase by LCS3. <strong>(M)</strong> SDS-PAGE analysis of D82A, R416A, and D417A mutants of TXNRD1. <strong>(N)</strong> Kinetic analysis of D82A, R416A, and D417A mutants of TXNRD1. <strong>(O)</strong> Inhibition of D82A, R416A, and D417A mutants of TXNRD1 by LCS3.</p></div></div></div><p id="p0165" class="p p-last">When testing the inhibition of LCS3 on TXNRD1 with various substrates, we observed that the inhibition of LCS3 on TXNRD1 was substrate-specific. Specifically, some substrates, such as juglone and toxoflavin, were not strongly inhibited by LCS3 (<a href="/pmc/articles/PMC10840350/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>F). The DTNB reduction of TXNRD1 is potently inhibited by LCS3, with an IC<sub>50</sub> value of 42.1&#x000a0;&#x003bc;M. Meanwhile, a LCS3 analogue, compound without a nitro group (LCS3-&#x00394;N), did not inhibit TXNRD1 and GSR, suggesting that the nitro group in LCS3 is the functional group responsible for the inhibitory effect (Fig. G). In cellular circumstances, after incubating HCT116&#x000a0;cells with LCS3 for 4&#x000a0;h, the TXNRDs activity and protein level in the extracted cell lysate were not altered. This also confirmed that LCS3 does not function as an irreversible inhibitor of TXNRD1 (<a href="/pmc/articles/PMC10840350/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>H). Together, these results indicated that LCS3 inhibits the TXN reductase activity of purified TXNRD1 reversibly <em>in vitro.</em></p></div><div id="sec2.11" class="sec sec-last"><h3 id="sec2.11title">2.11. LCS3 efficiently inhibits TXNRD1 activity but is not through the interaction with the guiding bar</h3><p id="p0170" class="p p-first">The interaction between LCS3 and TXNRD1 is predicted by <em>in silico</em> simulation, as previously reported. To elucidate the potential interacting residue of LCS3 on TXNRD1, we utilized mutant variants of TXNRD1 and conducted the inhibition of LCS3 on the enzyme and its variants. The activity loss of Sec-deficient TXNRD1, inhibited by LCS3, is weaker compared to the Sec-containing enzyme in DTNB reduction (<a href="/pmc/articles/PMC10840350/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>I). It should also be noted that the DTNB reduction activity of the Sec-deficient TXNRD1 mutant is approximately 1&#x02013;5% of the wild-type enzyme, but it is still detectable, as demonstrated in <a href="/pmc/articles/PMC10840350/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>D.</p><p id="p0175">We further investigated the inhibition effect of LCS3 on the guiding bar mutant of TXNRD1. Surprisingly, we did not observe a significant change in the dose-response curve of the inhibition effect on TXRND1 mutant variants by LCS3 (<a href="/pmc/articles/PMC10840350/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>J). The W411A and W411F mutants retained approximately 65&#x000a0;% activity of the untreated enzyme, which is highly similar to the wild-type TXNRD1 (<a href="/pmc/articles/PMC10840350/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>J). Area Under Curve (AUC) analysis revealed no difference among TXNRD1 mutant variants in the inhibition by LCS3 (<a href="/pmc/articles/PMC10840350/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>K).</p><p id="p0180">Additionally, TXNRD1 can be classified into a high-MW form, which is a selenoprotein that primarily exists in higher organisms, and a low-MW form, which could be found in prokaryotic organisms. The low-MW form is not a selenoprotein and lacks a guiding bar motif [<a href="#bib1" rid="bib1" class=" bibr popnode">1</a>]. Interestingly, we demonstrated that LCS3 exhibited a potent inhibitory effect on the recombinant TrxR of <em>Acinetobacter baumannii</em> and <em>Klebsiella pneumoniae,</em> with IC<sub>50</sub> values of 1.81&#x000a0;&#x003bc;M and 12.4&#x000a0;&#x003bc;M for ACIBA and KLEPN TrxR, respectively (<a href="/pmc/articles/PMC10840350/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>L).</p><p id="p0185">We then recombinantly produced TXNRD1 variants at D82, R416, and D417 (<a href="/pmc/articles/PMC10840350/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>M), which are predicted to be the binding residues of LCS3 on TXNRD1 using <em>in silico</em> simulation. We first determined the activity of these mutant variants. Notably, these three mutants showed a strongly decrease in TXN reduction, and the activity of DTNB reduction is impaired (<a href="/pmc/articles/PMC10840350/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>N and <a href="#appsec1" rid="appsec1" class=" sec">Fig. S12</a>). We then investigated the inhibition of LCS3 on TXNRD1 variants. Similar to the wild-type enzyme, these three mutants exhibited the same inhibition pattern by LCS3 (<a href="/pmc/articles/PMC10840350/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>O).</p><p id="p0190" class="p p-last">Collectively, these findings strongly suggest that the inhibition of LCS3 on thioredoxin reductases is not dependent on the guiding bar motif.</p></div></div><div id="sec3" class="tsec sec"><h2 class="head no_bottom_margin" id="sec3title">3. Discussion</h2><p id="p0195" class="p p-first">Selenoprotein TXNRD1 has been widely investigated, however, the guiding bar motif in relation to both the catalytic mechanism and redox regulation of TXNRD1 has remained to be an extensive attention [<a href="#bib12" rid="bib12" class=" bibr popnode">12</a>]. Distinct from flavoproteins such as dihydrolipoyl dehydrogenase (DLD) and GSR, TXNRD1 contains a unique flexible Sec-containing C-terminal tail, which has a significant impact on the catalytic activity [<a href="#bib31" rid="bib31" class=" bibr popnode">31</a>,<a href="#bib65" rid="bib65" class=" bibr popnode">65</a>,<a href="#bib66" rid="bib66" class=" bibr popnode">66</a>]. The guiding bar motif is positioned parallel to the C-terminal tails in the opposite direction of the catalytic pocket, suggesting that guiding bar motif does not directly contribute to the enzyme's electron transfer [<a href="#bib11" rid="bib11" class=" bibr popnode">11</a>,<a href="#bib12" rid="bib12" class=" bibr popnode">12</a>]. Nevertheless, mutations in the residues of guiding bar motif were observed to affect the formation of the charge-transfer complex (CTC) of the enzyme. Although the CTC is monitored in the EH<sub>4</sub>-formed enzyme using a UV&#x02013;Vis spectrophotometer instead of a stopped-flow spectrometer [<a href="#bib67" rid="bib67" class=" bibr popnode">67</a>], the results still demonstrated the electron transfer capacity in the absence of the substrates. As shown in <a href="/pmc/articles/PMC10840350/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a>F, Y402F, W407F and N418A/N419A significantly increased the absorbance at 540&#x000a0;nm at high concentrations of NADPH. In contrast, the formation of CTC for F406A was quite low. These results were consistent with the EEM data and kinetic parameters, showing that the F406A mutant significantly decreased the catalytic efficiency. Meanwhile, the EEM data demonstrated that the mutation affects the enzyme's binding to FAD. Previous studies have revealed that Trp<sup>114</sup> residue interacts with FAD, and that the fluorescence spectra of FAD are altered by the mutant form of Trp<sup>114</sup>, rather than by alterations in the C-terminal residues [<a href="#bib34" rid="bib34" class=" bibr popnode">34</a>]. Given the Trp<sup>114</sup> and guiding bar residues are far away from the FAD based in the TXNRD1 structure, this change may be explained by the transformation of the interaction between the N-terminal domain and FAD.</p><p id="p0200">Furthermore, our data strongly suggested that the guiding bar motif affects the kinetic parameters by modulating the movement of the C-terminal tail. In this study, we performed the site-directed mutagenesis at the residues located at the guiding bar motif and found such mutations significantly increase the Sec-independent reduction but decrease the disulfide reductase activity of the enzyme. We proposed that the increase is a consequence of enhanced substrate accessibility to the FAD/-CVNVGC- domain, as the increase in catalytic efficiency is primarily attributed to the lower <em>K</em><sub>m</sub> values. Additionally, it could also be inferred that the mutation may affect the nucleophilic attack of Cys<sup>59</sup> on the selenenylsulfide between Sec<sup>498</sup> and Cys<sup>497</sup> [<a href="#bib2" rid="bib2" class=" bibr popnode">2</a>,<a href="#bib5" rid="bib5" class=" bibr popnode">5</a>].</p><p id="p0205">Moreover, mutations in the guiding bar motif affect the binding of inhibitors to the enzyme. In the experiments, we selected TRi-1, RSL3, auranofin, and cisplatin to test their inhibitory effect on TXNRD1. TRi-1 stands out as one of the most specific inhibitors of TXNRD1, targeting the Sec<sup>498</sup> residues with lower GSH binding activity [<a href="#bib21" rid="bib21" class=" bibr popnode">21</a>,<a href="#bib52" rid="bib52" class=" bibr popnode">52</a>]. RSL3 is a well-known ferroptosis inducer, by inhibiting GPX4 through modification of the selenocysteine or cysteine residues [<a href="#bib53" rid="bib53" class=" bibr popnode">53</a>,<a href="#bib54" rid="bib54" class=" bibr popnode">54</a>]. TXNRD1 has recently been identified as the cellular target of RSL3 [<a href="#bib36" rid="bib36" class=" bibr popnode">36</a>]. Additionally, auranofin and cisplatin are widely accepted FDA-approved drugs that inhibit cellular TXNRD1 activity, but they do not specifically target selenocysteine [<a href="#bib55" rid="bib55" class=" bibr popnode">55</a>,<a href="#bib68" rid="bib68" class=" bibr popnode">68</a>]. Our data revealed that the mutated enzyme is more susceptible to attack by TRi-1 and RSL3. This can be explained by the rearranging effect of the guiding bar on the Sec-containing C-terminal tail of the enzyme, making the selenocysteine more accessible to the drug. These results suggested an important role of the guiding bar motif as a gatekeeper for the reactive C-terminal tail.</p><p id="p0210">Caveolae are a characteristic feature of the plasma membrane (PM) in many mammalian cell types, and caveolin-1 is the major protein that composes caveolae [<a href="#bib69" rid="bib69" class=" bibr popnode">69</a>]. It has been demonstrated that numerous proteins utilize the conserved caveolin-binding motif (CBM) to interact with caveolae via the caveolin scaffolding domain (CSD). TXNRD1 has been reported to interact with caveolin-1 through the amino acid region 402&#x02013;411, which coincides with the guiding bar motif of the enzyme [<a href="#bib58" rid="bib58" class=" bibr popnode">58</a>]. However, there is still an argument on the CBM/CSD-dependent interactions in caveolar signaling since CBM is generally buried and not readily accessible in most proteins [<a href="#bib59" rid="bib59" class=" bibr popnode">59</a>,<a href="#bib70" rid="bib70" class=" bibr popnode">70</a>]. Therefore, we would investigate the interaction of CSD peptides on the guiding bar. Clearly, the synthesized CSD peptides inhibited the activity of TXNRD1 <em>in vitro</em> compared to the control peptide CAV-X (<a href="/pmc/articles/PMC10840350/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>D). CSD-D2 (CSP7) was considered the central domain of the CSD both <em>in vitro</em> and <em>in vivo</em> [<a href="#bib62" rid="bib62" class=" bibr popnode">62</a>,<a href="#bib63" rid="bib63" class=" bibr popnode">63</a>]. However, when using truncated peptides, we observed that the truncated peptides do not inhibit TXNRD1 activity, indicating that the inhibitory effect of CSD on the enzyme is highly dependent on the structural integrity of CSD (<a href="/pmc/articles/PMC10840350/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>A&#x02013;E). Additionally, by using CBM mutant of the enzyme, we observed that the inhibition of CSD peptide is not dependent on the CBM residue, which supports the opinion that the CBM/CSD-dependent interactions are unlikely to mediate caveolar signaling. Interestingly, it has been discovered that caveolae can sense oxidative stress by regulating the NRF2 antioxidant response [<a href="#bib71" rid="bib71" class=" bibr popnode">71</a>]. Additionally, a rare v3 splice form of TXNRD1 was found to potentially target the membrane [<a href="#bib72" rid="bib72" class=" bibr popnode">72</a>]. However, it is not yet clear if these findings are related to the interaction between TXNRD1 and caveolin-1, and the precise role of this interaction in cellular signaling remains to be fully understood. Further studies are needed to elucidate the details of this interaction.</p><p id="p0215">Numerous studies have revealed that TXNRD1 is upregulated in cancer cells [<a href="#bib73" rid="bib73" class=" bibr popnode">73</a>,<a href="#bib74" rid="bib74" class=" bibr popnode">74</a>]. In cancer treatment, pharmacological inhibition of TXNRD1 induces rapid cell death through SecTRAPs [<a href="#bib20" rid="bib20" class=" bibr popnode">20</a>,<a href="#bib75" rid="bib75" class=" bibr popnode">75</a>], while TXNRD1 deficiency renders cells susceptible to glutathione (GSH) depletion [<a href="#bib76" rid="bib76" class=" bibr popnode">76</a>,<a href="#bib77" rid="bib77" class=" bibr popnode">77</a>]. These findings position TXNRD1 as a promising and compelling drug target for cancer therapy. Recently, a small molecule lung cancer screen 3 (LCS3) was developed to suppress the growth of LUAD. Thermal proteomic profiling (TPP) identified TXNRD1 as the cellular target of LCS3 [<a href="#bib64" rid="bib64" class=" bibr popnode">64</a>,<a href="#bib78" rid="bib78" class=" bibr popnode">78</a>]. Additionally, <em>in silico</em> molecular docking predicted that the nitro group of LCS3 forms a hydrogen bond with Trp<sup>411</sup> of TXNRD1. LCS3 is also predicted to interact with Asp<sup>82</sup> through hydrogen bond formation, and with Arg<sup>416</sup> through an arene-cation interaction [<a href="#bib64" rid="bib64" class=" bibr popnode">64</a>]. Given that the docking analysis predicts that the LCS3 interacts with residues in guiding bar located homodimer interfaces of TXNRD1, we investigated the inhibition details of LCS3 on TXNRD1. It is surprising that LCS3 is a reversible inhibitor of TXNRD1, which differs from most inhibitors of TXNRD1 (<a href="/pmc/articles/PMC10840350/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>B&#x02013;E). Right now, many inhibitors of TXNRD1 are electrophilic and modifying the enzyme through alkylation of thiols or selenols [<a href="#bib23" rid="bib23" class=" bibr popnode">23</a>,<a href="#bib51" rid="bib51" class=" bibr popnode">51</a>,<a href="#bib79" rid="bib79" class=" bibr popnode">[79]</a>, <a href="#bib80" rid="bib80" class=" bibr popnode">[80]</a>, <a href="#bib81" rid="bib81" class=" bibr popnode">[81]</a>]. Using recombinant protein, we identified that the nitro group of LCS3 profoundly inhibits the enzyme (<a href="/pmc/articles/PMC10840350/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>G). However, when we determined the residue target of LCS3 on TXNRD1 by using mutant variants of the enzyme, we observed that the W411, as well as D82, R416 and D417, mutants of TXNRD1 did not impact the inhibitory effect (<a href="/pmc/articles/PMC10840350/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>J, K, O). In addition, the inhibition of LCS3 on low-MW TrxR1 is more potent than high-MW form, suggesting that LCS3 may not interact with the guiding bar. However, these findings also offer a potential application as an antibacterial agent through the inhibition of TrxR. Taken together, it is impressive to find LCS3 as a reversible inhibitor of TXNRD1, and further clarification is needed regarding the inhibition mechanism and the structure-function relationship.</p><p id="p0220" class="p p-last">In conclusion, the results obtained in this study provide valuable insights into the catalytic mechanism of TXNRD1 and demonstrate that the distinctive guiding bar motif functions as a gearbox for tuning the flexible C-terminal tail and adjusting the catalytic activity of TXNRD1.</p></div><div id="sec4" class="tsec sec"><h2 class="head no_bottom_margin" id="sec4title">4. Materials and methods</h2><div id="sec4.1" class="sec sec-first"><h3 id="sec4.1title">4.1. Chemicals and reagents</h3><p id="p0225" class="p p-first">Chemicals used in this study include: 5-Hydroxy-1,4-naphthoquinone (juglone, <a id="__tag_1143846845" class="tag_hotlink" href="/nuccore/H47003" ref="/nuccore/H47003"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"H47003","term_id":"923055","term_text":"H47003"}}</span>H47003</a>, Sigma-Aldrich), N-(4-chlorophenyl)-2-furamide (L134759, Sigma-Aldrich), 5,5&#x02032;-dithiobis-(2-nitrobenzoic acid) (DTNB, <a id="__tag_1143846835" class="tag_hotlink" href="/protein/S19139" ref="/protein/S19139"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"S19139","term_id":"7433833","term_text":"pir||S19139"}}</span>S19139</a>, Yuanye), nicotinamide adenine dinucleotide phosphate (NADPH, S10103, Yuanye), TRi-1 (S87818, Yuanye), auranofin (<a id="__tag_1143846846" class="tag_hotlink" href="/nuccore/S80655" ref="/nuccore/S80655"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"S80655","term_id":"1311688","term_text":"S80655"}}</span>S80655</a>, Yuanye), Shikonin (<a id="__tag_1143846844" class="tag_hotlink" href="/nuccore/B21682" ref="/nuccore/B21682"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"B21682","term_id":"2396736","term_text":"B21682"}}</span>B21682</a>, Yuanye), LCS3 (S89617, Yuanye), 9,10-phenanthrene quinone (9,10-PQ, P106382, Aladdin), RSL3 (R302648, Aladdin), HED (B152479, Aladdin), <sub><span class="small-caps">l</span></sub>-cystine (BD21525, Bidepharm), cisplatin (MB1055-2, MeilunBio), yeast extract (LP0021B, Oxoid), tryptone (LP0042B, Oxoid).</p><p id="p0230">Antibodies used in this study include: TXNRD1 (67728, Proteintech), TXN (66475, Proteintech), goat anti-mouse IgG (SA00001-1, Proteintech).</p><p id="p0235">Critical commercial assays used in this study include: BCA protein kit (P0012, Beyotime), Bradford protein kit (20202ES76, Yeasen, China), 4&#x02013;20&#x000a0;% SDS-PAGE gel (P0469M, Beyotime; M00657, Gensrcipt).</p><p id="p0240" class="p p-last">Proteins used in this study include: SOD (S7571, Sigma-Aldrich), BSA (NA8692, Ruibio), Insulin (<a id="__tag_1143846834" class="tag_hotlink" href="/protein/S12033" ref="/protein/S12033"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"S12033","term_id":"100451","term_text":"pir||S12033"}}</span>S12033</a>, Yuanye). Recombinant TXN [<a href="#bib38" rid="bib38" class=" bibr popnode">38</a>], GSR [<a href="#bib8" rid="bib8" class=" bibr popnode">8</a>], TRP14 [<a href="#bib38" rid="bib38" class=" bibr popnode">38</a>], <em>Acinetobacter baumannii</em> (ACIBA) TrxR [<a href="#bib82" rid="bib82" class=" bibr popnode">82</a>] and <em>klebsiella pneumoniae</em> (KLEPN) TrxR [<a href="#bib82" rid="bib82" class=" bibr popnode">82</a>] were produced as previous described.</p></div><div id="sec4.2" class="sec"><h3 id="sec4.2title">4.2. Strains and constructs</h3><p id="p0245" class="p p-first-last">The <em>E. coli</em> BL21 (DE3) <em>gor</em><sup>&#x02212;</sup> host strain was generously provided by Prof. Arne Holmgren (Karolinska Institutet, Stockholm, Sweden). Plasmids pSUABC and pET-TRS<sub>TER</sub> were gifts from Prof. Elias S.J. Arn&#x000e9;r (Karolinska Institutet, Stockholm, Sweden). Primers for constructing the TXNRD1 mutants in guiding bar motif were listed in <a href="#appsec1" rid="appsec1" class=" sec">Table S1</a>.</p></div><div id="sec4.3" class="sec"><h3 id="sec4.3title">4.3. Generation and purification of recombinant rat TXNRD1</h3><p id="p0250" class="p p-first-last">Recombinant rat TXNRD1 and its mutant variants were produced as described unless otherwise stated [<a href="#bib30" rid="bib30" class=" bibr popnode">30</a>,<a href="#bib83" rid="bib83" class=" bibr popnode">83</a>]. In brief, the pET-TRS<sub>TER</sub>-derived plasmids are co-transformed into BL21 (DE3) <em>gor</em><sup>&#x02212;</sup> with plasmid pSUABC (D82A, R416A and D417A mutants of TXNRD1 were produced in BL21 (DE3) strain). The bacterial cell cultures were incubated in a shaker (220&#x000a0;rpm) at 37&#x000a0;&#x000b0;C until OD<sub>600nm</sub>&#x000a0;=&#x000a0;2.4 reaching late points of the log phase. Then 0.5&#x000a0;mM IPTG, 5&#x000a0;&#x003bc;M selenite, and 100&#x000a0;&#x003bc;g/mL <sub><span class="small-caps">l</span></sub>-cysteine were separately added into the bacterial cultures. The cultures were continually incubated at 24&#x000a0;&#x000b0;C for 24&#x000a0;h in the shaker, bacterial cells were harvested by centrifugation at 3500&#x000a0;rpm for 20&#x000a0;min, and the wet bacterial pellets were subsequently resuspended in lysis buffer (50&#x000a0;mM Tris-HCl, 20&#x000a0;mM EDTA, 1&#x000a0;mg/mL lysozyme, pH&#x000a0;=&#x000a0;7.4), followed with freeze-thawing for 3 cycles and the viscous nucleic acids were broken by sonication on ice. Lysates were spun down at 12,000&#x000a0;rpm, 4&#x000a0;&#x000b0;C for 30&#x000a0;min. Afterwards, the supernatant was purified through affinity chromatography on 2&#x02032;5&#x02032; ADP Sepharose 4B (17070001, Cytiva, Sweden), and size-exclusion chromatography on HiPrep 16/60 Sephacryl S-300 HR (17116701, Cytiva, Sweden). Recombinant proteins were analyzed by reducing SDS-PAGE and stained by Coomassie Brilliant Blue (CBB).</p></div><div id="sec4.4" class="sec"><h3 id="sec4.4title">4.4. 5-IAF labeling of TXNRD1</h3><p id="p0255" class="p p-first-last">3&#x000a0;&#x003bc;M TXNRD1 and its variants (pH 8.5) in oxidized states were separately incubated with 1&#x000a0;mM 5-iodoacetamidofluorescein (5-IAF) for 30&#x000a0;min in the dark at room temperature. In parallel, 3&#x000a0;&#x003bc;M enzymes were pre-reduced with 1&#x000a0;mM NADPH for 15&#x000a0;min, followed by the addition of 1&#x000a0;mM 5-IAF and incubation in dark for 30&#x000a0;min at room temperature. All 5-IAF-labeled protein samples were then desalted and subjected to a reducing SDS-PAGE gel, visualized under the UV light.</p></div><div id="sec4.5" class="sec"><h3 id="sec4.5title">4.5. Wavelength scanning of various TXNRD1 mutants</h3><p id="p0260" class="p p-first-last">Wavelength scanning (340&#x000a0;nm&#x02013;600&#x000a0;nm) was determined by using a spectrophotometer (UV5-Bio, Mettler Toledo, Switzerland). 6.5&#x000a0;&#x003bc;M TXNRD1 and its mutants were separately added into a standard 1-cm quartz cuvette to analyze the spectrum with TE buffer as reference.</p></div><div id="sec4.6" class="sec"><h3 id="sec4.6title">4.6. Charge-transfer complex detection</h3><p id="p0265" class="p p-first-last">Charge-transfer complex detection of TXNRD1 and its variants were performed in a microplate reader (SpectraMax ABS, Molecular Devices). The abs at 540&#x000a0;nm of 6.5&#x000a0;&#x003bc;M wild-type TXNRD1 and its variants in the presence of indicated concentrations of NADPH were determined. The enzyme lacking NADPH was regarded as the reference.</p></div><div id="sec4.7" class="sec"><h3 id="sec4.7title">4.7. Excitation-emission matrix spectroscopy</h3><p id="p0270" class="p p-first-last">Wild-type TXNRD1 and its variants (7&#x000a0;&#x003bc;M) were dissolved in TE buffer (pH 7.5). 100&#x000a0;&#x003bc;L of enzymes were loaded into a flat-bottomed 96-wells black plate for fluorescence measurements using a multimode reader (Synergy H1, Bio-Tek, USA). The emission spectrum ranging from 300&#x000a0;nm&#x02013;700&#x000a0;nm was obtained by exiting the sample at a fixed excitation wavelength of 260&#x000a0;nm and subsequently the excitation wavelength was increased by 10-nm intervals up to 510&#x000a0;nm with TE buffer as reference. The data obtained for all excitation wavelengths were used to plot the fluorescence emission spectra in 3D format, with &#x02018;X-axis&#x02019; as the excitation wavelength (Ex), &#x02018;Y-axis&#x02019; as the emission wavelength (Em), and the &#x02018;Z-axis&#x02019; as fluorescence intensity.</p></div><div id="sec4.8" class="sec"><h3 id="sec4.8title">4.8. Kinetic analyses of TXNRD1 variants</h3><p id="p0275" class="p p-first-last">Enzymatic activities of purified TXNRD1 variants were determined in transparent 96-well microtiter plates using classical substrates including DTNB, juglone, 9,10-PQ, toxoflavin, human TXN1, and human TRP14 as described [<a href="#bib84" rid="bib84" class=" bibr popnode">84</a>,<a href="#bib85" rid="bib85" class=" bibr popnode">85</a>]. The standard reaction mixture contained 10&#x02013;100&#x000a0;nM TXNRD1 or its mutant variants, 200&#x02013;300&#x000a0;&#x003bc;M NADPH in TE buffer. All the activity assays were performed at room temperature either following TNB<sup>&#x02212;</sup> formation at 412&#x000a0;nm (&#x00190;<sub>TNB&#x02212;</sub>&#x000a0;=&#x000a0;13,600&#x000a0;M<sup>&#x02212;1</sup>&#x000a0;cm<sup>&#x02212;1</sup>) for DTNB, or directly following NADPH consumption as decrease at 340&#x000a0;nm (&#x00190;<sub>NADPH</sub>&#x000a0;=&#x000a0;6200&#x000a0;M<sup>&#x02212;1</sup>&#x000a0;cm<sup>&#x02212;1</sup>) for the other substrates. Insulin and cystine were used as the electron acceptor for TXN1 reduction and TRP14 reduction, respectively. Activity measurements were performed in triplicate (unless stated in the text) using a microplate reader (SpectraMax ABS, Molecular Devices). Kinetic constants were calculated with the Prism 8 software (GraphPad) after direct plotting of the velocity versus substrate concentration followed by automatic Michaelis&#x02013;Menten fit with nonlinear regression.</p></div><div id="sec4.9" class="sec"><h3 id="sec4.9title">4.9. Quantification of superoxide anions production</h3><p id="p0280" class="p p-first-last"><em>In vitro</em> superoxide production was assayed according to the adrenochrome method [<a href="#bib48" rid="bib48" class=" bibr popnode">48</a>]. The standard reaction mixture (200&#x000a0;&#x003bc;l) contains 200&#x000a0;&#x003bc;M NADPH, 1&#x000a0;mM epinephrine, 50&#x000a0;nM TXNRD1, and 10&#x000a0;&#x003bc;M juglone or 100&#x000a0;&#x003bc;M shikonin as indicated. Changes in absorbance at 340&#x000a0;nm and 480&#x000a0;nm were measured simultaneously, using the same reaction mixture without the enzyme as the reference. For validation of superoxide being formed, superoxide dismutase (SOD) was added (10 units/well). All reactions were performed in a microplate reader (SpectraMax ABS, Molecular Devices, USA).</p></div><div id="sec4.10" class="sec"><h3 id="sec4.10title">4.10. Synthesis of CSD-derived peptides</h3><p id="p0285" class="p p-first-last">The CSD peptide and its derived peptides were synthesized by Sangon Biotech (Shanghai, China) as sequences follows: CSD (DGIWKASFTTFTVTKYWFYR), CAV-X (WGIDKAFFTTSTVTYKWFRY), CSD-D1 (DGIWKAS), CSD-D2 (FTTFTVT), and CSD-D3 (KYWFYR). The peptides were freshly dissolved in DMSO up to 4&#x000a0;mM before experiments.</p></div><div id="sec4.11" class="sec"><h3 id="sec4.11title">4.11. Cell culture and LCS3 treatment</h3><p id="p0290" class="p p-first-last">HCT116&#x000a0;cells (CL-0096, Procell) were cultured in McCoy's 5A medium supplemented with 10&#x000a0;% fetal bovine serum (FBS, 164210, Procell), 100 U/mL penicillin, 100&#x000a0;mg/mL streptomycin (P/S, AC03L332, Life-iLab) in a humidified incubator (Heal Force) with an atmosphere of 5&#x000a0;% CO<sub>2</sub> and a temperature of 37&#x000a0;&#x000b0;C. As for the LCS3 treatment, in brief, HCT116&#x000a0;cells were seeded into 6-well plates at 400,000&#x000a0;cells per well and treated with LCS3 for 4&#x000a0;h. The cells were washed with PBS buffer three times and all adherent cells were lysed by the RIPA buffer with 1&#x000a0;mM protease inhibitor PMSF. Then, the cell extract was centrifuged at 18,000&#x000a0;<em>g</em> at 4&#x000a0;&#x000b0;C for 20&#x000a0;min. Total protein contents were determined using a BCA kit (Beyotime, China).</p></div><div id="sec4.12" class="sec"><h3 id="sec4.12title">4.12. Cellular TXNRD1 activity</h3><p id="p0295" class="p p-first-last">Cellular TXNRDs activities were determined according to TXN1-coupled end-point insulin assay as previously established [<a href="#bib28" rid="bib28" class=" bibr popnode">28</a>,<a href="#bib86" rid="bib86" class=" bibr popnode">86</a>] and the method was slightly modified in this experiment. In brief, an appropriate amount of cell lysates was added into a master mixture containing 80&#x000a0;mM Hepes buffer (pH 7.5), 15&#x000a0;&#x003bc;M TXN1, 300&#x000a0;&#x003bc;M insulin, 660&#x000a0;&#x003bc;M NADPH, and 3&#x000a0;mM EDTA. A reaction mixture without TXN1 was used as a background control. Samples were incubated at 37&#x000a0;&#x000b0;C for 30&#x000a0;min. Subsequently, 6.0&#x000a0;M guanidine hydrochloride containing 1&#x000a0;mM DTNB, and 20&#x000a0;mM EDTA was added to each well, and an endpoint Abs @412&#x000a0;nm was measured and recorded. TXNRDs activities of cell lysates were normalized to protein concentrations for an accurate comparison.</p></div><div id="sec4.13" class="sec"><h3 id="sec4.13title">4.13. Western blotting</h3><p id="p0300" class="p p-first-last">The total protein concentrations of cell lysates were determined through BCA protein assay. All samples were incubated at 95&#x000a0;&#x000b0;C for 10&#x000a0;min before SDS&#x02013;PAGE analysis and then transferred onto the PVDF membrane (0.45&#x000a0;&#x003bc;m, Millipore). The membranes were blocked by 5&#x000a0;% skimmed milk for 1&#x000a0;h and incubated with indicated concentrations of primary antibody overnight at 4&#x000a0;&#x000b0;C: TXNRD1 (1: 5000), TXN (1: 5000). After washed with TBST three times, the blots were incubated with the secondary antibody (1: 5000) for 1&#x000a0;h at room temperature. The ECL mixture solution (ED0015/ED0016, Sparkjade, Shandong, China) was added to the membranes and developed by imaging analyzer (Sagecreation, Beijing).</p></div><div id="sec4.14" class="sec"><h3 id="sec4.14title">4.14. Analysis of the TXN1-TXNRD1 complexes</h3><p id="p0305" class="p p-first-last">TXN1 was initially reduced with 10&#x000a0;mM DTT at 37&#x000a0;&#x000b0;C for 20&#x000a0;min, followed by removal of the DTT using a NAP-5&#x02122; desalting column. Wild-type TXNRD1 or its variants (1&#x000a0;&#x003bc;M) were incubated with 8&#x000a0;&#x003bc;M TXN1, along with 200&#x000a0;&#x003bc;M NADPH or 100&#x000a0;&#x003bc;M cisplatin as indicated at room temperature for 16&#x000a0;h. All samples were mixed with 5&#x000a0;&#x000d7;&#x000a0;SDS loading solution without (non-reducing) or with (reducing) addition of 20&#x000a0;mM DTT and denatured at 95&#x000a0;&#x000b0;C for 10&#x000a0;min. Then, the samples were loaded onto an SDS-PAGE gel and followed with CBB stain or Western blotting.</p></div><div id="sec4.15" class="sec"><h3 id="sec4.15title">4.15. GSR activity assay</h3><p id="p0310" class="p p-first-last">Glutathione reductase (GSR) activity was determined using the oxidized glutathione (GSSG) as a substrate [<a href="#bib8" rid="bib8" class=" bibr popnode">8</a>]. The standard assay system (200&#x000a0;&#x003bc;L) contains 1&#x000a0;mM GSSG, 2&#x000a0;nM yeast GSR, and 200&#x000a0;&#x003bc;M NADPH in TE buffer, pH7.5. GSR activity was calculated by following the NADPH oxidation based on the absorbance decrease at 340&#x000a0;nm (&#x003b5;<sub>NADPH</sub>&#x000a0;=&#x000a0;6200&#x000a0;M<sup>&#x02212;1</sup>&#x000a0;cm<sup>&#x02212;1</sup>). The assays were repeated for three times and mean value was used to calculate the GSR activity.</p></div><div id="sec4.16" class="sec"><h3 id="sec4.16title">4.16. Differential Scanning Fluorimetry (DSF) assay</h3><p id="p0315" class="p p-first-last">The DSF analysis was performed as described [<a href="#bib35" rid="bib35" class=" bibr popnode">35</a>]. Briefly, TXNRD1 and its variants were diluted in a DSF buffer (20&#x000a0;mM HEPES, pH 7.4, 100&#x000a0;mM NaCl) to 3&#x000a0;&#x003bc;M. Then, a SYPRO Orange stain (S5506, Thermo Fisher) was added with a final concentration of 5&#x000a0;&#x000d7;&#x000a0;. The fluorescence signal was acquired using the CG-05 fluorescence spectrometer (Heal Force, China) with an excitation wavelength of 498&#x000a0;nm and fluorescence emission at 522&#x000a0;nm. The temperature was raised from 35 to 95&#x000a0;&#x000b0;C with a ramp rate of 1&#x000a0;&#x000b0;C/min. Three biological replicates were conducted for each condition, and their means and standard deviations are shown.</p></div><div id="sec4.17" class="sec sec-last"><h3 id="sec4.17title">4.17. Statistical analysis</h3><p id="p0320" class="p p-first-last">The data were presented as the Mean&#x000a0;&#x000b1;&#x000a0;SD (n&#x000a0;=&#x000a0;3). Statistical differences between the two groups were analyzed using the Student's t-test. Comparisons among multiple groups were statistically assessed by one-way analysis of variance (ANOVA) and followed by a post hoc Scheffe test. The significant differences between groups were defined as * <em>p</em>&#x000a0;&#x0003c;&#x000a0;0.05, **<em>p</em>&#x000a0;&#x0003c;&#x000a0;0.01, and ***<em>p</em>&#x000a0;&#x0003c;&#x000a0;0.001, where n.s. means not significant.</p></div></div><div id="sec5" class="tsec sec"><h2 class="head no_bottom_margin" id="sec5title">Funding</h2><p id="p0325" class="p p-first-last">This research was funded by National Natural Science Foundation of China (U22A20455 &#x00026; 31670767), Fundamental Research Funds for the Central Universities (DUT17JC36, DUT20LK36 and DUT21LK29).</p></div><div id="sec6" class="tsec sec"><h2 class="head no_bottom_margin" id="sec6title">CRediT authorship contribution statement</h2><p id="p0330" class="p p-first-last"><strong>Wuyang Shi:</strong> Investigation, Visualization. <strong>Shibo Sun:</strong> Conceptualization, Investigation, Visualization, Writing &#x02013; original draft, Writing &#x02013; review &#x00026; editing. <strong>Haowen Liu:</strong> Investigation, Visualization. <strong>Yao Meng:</strong> Investigation. <strong>Kangshuai Ren:</strong> Investigation. <strong>Guoying Wang:</strong> Investigation. <strong>Minghui Liu:</strong> Investigation. <strong>Jiaqi Wu:</strong> Investigation. <strong>Yue Zhang:</strong> Investigation. <strong>Huang Huang:</strong> Investigation. <strong>Meiyun Shi:</strong> Resources. <strong>Weiping Xu:</strong> Funding acquisition, Resources. <strong>Qiang Ma:</strong> Resources. <strong>Bingbing Sun:</strong> Funding acquisition, Resources. <strong>Jianqiang Xu:</strong> Conceptualization, Funding acquisition, Supervision, Writing &#x02013; original draft, Writing &#x02013; review &#x00026; editing.</p></div><div id="sec-a.k.g" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.k.gtitle">Declaration of competing interest</h2><p id="p0335" class="p p-first-last">The authors declare that they have no known competing financial interests or personal relationships that could have appeared to influence the work reported in this paper.</p></div><div id="ack0010" class="tsec sec"><h2 class="head no_bottom_margin" id="ack0010title">Acknowledgments</h2><div class="sec"><p id="p0340">The authors want to thank Prof. Dr. Elias S.J. Arn&#x000e9;r ( Karolinska Institutet, Sweden) for his material supports, Prof. Dr. Enyin Lai (Zhejiang University, China) for his experimental assistances, and Prof. Dr. Jianing Zhang (Dalian University of Technology, China) for his scientific discussion.</p></div></div><div id="fn-group-a.l.e" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.l.etitle">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="appsec2"><sup>Appendix A</sup>Supplementary data to this article can be found online at <a href="https://doi.org/10.1016/j.redox.2024.103050" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://doi.org/10.1016/j.redox.2024.103050</a>.</p></div></div><div id="appsec1" class="tsec bk-sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="appsec1title">Appendix A. Supplementary data</h2><!--/article/back/sec/--><p id="p0345" class="p p-first-last">The following is the Supplementary data to this article:</p><div class="sec suppmat" id="mmc1"><!--caption a9--><strong>Multimedia component 1:</strong><div class="sup-box half_rhythm" id="media-a.l.b.c.a.b"><a href="/pmc/articles/PMC10840350/bin/mmc1.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(48M, docx)</sup>Multimedia component 1</div></div><p></p></div><div id="da0010" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="da0010title">Data availability</h2><!--/article/back/sec/--><p id="p0030" class="p p-first-last">Data will be made available on request.</p></div><div id="cebib0010" class="tsec sec"><h2 class="head no_bottom_margin" id="cebib0010title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="bib1">1. <span class="element-citation" id="sref1">Lu J., Holmgren A. The thioredoxin antioxidant system. <span><span class="ref-journal">Free Radic. Biol. Med. </span>2014;<span class="ref-vol">66</span>:75–87.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23899494" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Free+Radic.+Biol.+Med.&amp;title=The+thioredoxin+antioxidant+system&amp;author=J.+Lu&amp;author=A.+Holmgren&amp;volume=66&amp;publication_year=2014&amp;pages=75-87&amp;pmid=23899494&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib2">2. <span class="element-citation" id="sref2">Arn&#x000e9;r E.S.J. Focus on mammalian thioredoxin reductases--important selenoproteins with versatile functions. <span><span class="ref-journal">Biochim. Biophys. Acta. </span>2009;<span class="ref-vol">1790</span>(6):495–526.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19364476" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.+Biophys.+Acta&amp;title=Focus+on+mammalian+thioredoxin+reductases--important+selenoproteins+with+versatile+functions&amp;author=E.S.J.+Arn&#x000e9;r&amp;volume=1790&amp;issue=6&amp;publication_year=2009&amp;pages=495-526&amp;pmid=19364476&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib3">3. <span class="element-citation" id="sref3">Dagnell M., Schmidt E.E., Arn&#x000e9;r E.S.J. The A to Z of modulated cell patterning by mammalian thioredoxin reductases. <span><span class="ref-journal">Free Radic. Biol. Med. </span>2018;<span class="ref-vol">115</span>:484–496.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5771652/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29278740" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Free+Radic.+Biol.+Med.&amp;title=The+A+to+Z+of+modulated+cell+patterning+by+mammalian+thioredoxin+reductases&amp;author=M.+Dagnell&amp;author=E.E.+Schmidt&amp;author=E.S.J.+Arn&#x000e9;r&amp;volume=115&amp;publication_year=2018&amp;pages=484-496&amp;pmid=29278740&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib4">4. <span class="element-citation" id="sref4">Zhong L., Arner E.S., Holmgren A. Structure and mechanism of mammalian thioredoxin reductase: the active site is a redox-active selenolthiol/selenenylsulfide formed from the conserved cysteine-selenocysteine sequence. <span><span class="ref-journal">Proc. Natl. Acad. Sci. U. S. A. </span>2000;<span class="ref-vol">97</span>(11):5854–5859.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC18523/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10801974" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.+S.+A.&amp;title=Structure+and+mechanism+of+mammalian+thioredoxin+reductase:+the+active+site+is+a+redox-active+selenolthiol/selenenylsulfide+formed+from+the+conserved+cysteine-selenocysteine+sequence&amp;author=L.+Zhong&amp;author=E.S.+Arner&amp;author=A.+Holmgren&amp;volume=97&amp;issue=11&amp;publication_year=2000&amp;pages=5854-5859&amp;pmid=10801974&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib5">5. <span class="element-citation" id="sref5">Cheng Q., Sandalova T., Lindqvist Y., Arn&#x000e9;r E.S.J. Crystal structure and catalysis of the selenoprotein thioredoxin reductase 1. <span><span class="ref-journal">J. Biol. Chem. </span>2009;<span class="ref-vol">284</span>(6):3998–4008.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19054767" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Crystal+structure+and+catalysis+of+the+selenoprotein+thioredoxin+reductase+1&amp;author=Q.+Cheng&amp;author=T.+Sandalova&amp;author=Y.+Lindqvist&amp;author=E.S.J.+Arn&#x000e9;r&amp;volume=284&amp;issue=6&amp;publication_year=2009&amp;pages=3998-4008&amp;pmid=19054767&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib6">6. <span class="element-citation" id="sref6">Brandt W., Wessjohann L.A. The functional role of selenocysteine (Sec) in the catalysis mechanism of large thioredoxin reductases: proposition of a swapping catalytic triad including a Sec-His-Glu state. <span><span class="ref-journal">Chembiochem. </span>2005;<span class="ref-vol">6</span>(2):386–394.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15651042" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chembiochem&amp;title=The+functional+role+of+selenocysteine+(Sec)+in+the+catalysis+mechanism+of+large+thioredoxin+reductases:+proposition+of+a+swapping+catalytic+triad+including+a+Sec-His-Glu+state&amp;author=W.+Brandt&amp;author=L.A.+Wessjohann&amp;volume=6&amp;issue=2&amp;publication_year=2005&amp;pages=386-394&amp;pmid=15651042&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib7">7. <span class="element-citation" id="sref7">Andor A., Mohanraj M., Pato Z.A., Uri K., Biri-Kovacs B., Cheng Q., et al. TXNL1 has dual functions as a redox active thioredoxin-like protein as well as an ATP- and redox-independent chaperone. <span><span class="ref-journal">Redox Biol. </span>2023;<span class="ref-vol">67</span></span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10570131/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37804695" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Redox+Biol.&amp;title=TXNL1+has+dual+functions+as+a+redox+active+thioredoxin-like+protein+as+well+as+an+ATP-+and+redox-independent+chaperone&amp;author=A.+Andor&amp;author=M.+Mohanraj&amp;author=Z.A.+Pato&amp;author=K.+Uri&amp;author=B.+Biri-Kovacs&amp;volume=67&amp;publication_year=2023&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib8">8. <span class="element-citation" id="sref8">Xu J., Arner E.S. Pyrroloquinoline quinone modulates the kinetic parameters of the mammalian selenoprotein thioredoxin reductase 1 and is an inhibitor of glutathione reductase. <span><span class="ref-journal">Biochem. Pharmacol. </span>2012;<span class="ref-vol">83</span>(6):815–820.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22226931" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.+Pharmacol.&amp;title=Pyrroloquinoline+quinone+modulates+the+kinetic+parameters+of+the+mammalian+selenoprotein+thioredoxin+reductase+1+and+is+an+inhibitor+of+glutathione+reductase&amp;author=J.+Xu&amp;author=E.S.+Arner&amp;volume=83&amp;issue=6&amp;publication_year=2012&amp;pages=815-820&amp;pmid=22226931&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib9">9. <span class="element-citation" id="sref9">Gencheva R., Cheng Q., Arn&#x000e9;r E.S.J. Efficient selenocysteine-dependent reduction of toxoflavin by mammalian thioredoxin reductase. <span><span class="ref-journal">Biochim. Biophys. Acta Gen. Subj. </span>2018</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29807072" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.+Biophys.+Acta+Gen.+Subj.&amp;title=Efficient+selenocysteine-dependent+reduction+of+toxoflavin+by+mammalian+thioredoxin+reductase&amp;author=R.+Gencheva&amp;author=Q.+Cheng&amp;author=E.S.J.+Arn&#x000e9;r&amp;publication_year=2018&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib10">10. <span class="element-citation" id="sref10">Xia L., Nordman T., Olsson J.M., Damdimopoulos A., Bjorkhem-Bergman L., Nalvarte I., et al. The mammalian cytosolic selenoenzyme thioredoxin reductase reduces ubiquinone. A novel mechanism for defense against oxidative stress. <span><span class="ref-journal">J. Biol. Chem. </span>2003;<span class="ref-vol">278</span>(4):2141–2146.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12435734" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=The+mammalian+cytosolic+selenoenzyme+thioredoxin+reductase+reduces+ubiquinone.+A+novel+mechanism+for+defense+against+oxidative+stress&amp;author=L.+Xia&amp;author=T.+Nordman&amp;author=J.M.+Olsson&amp;author=A.+Damdimopoulos&amp;author=L.+Bjorkhem-Bergman&amp;volume=278&amp;issue=4&amp;publication_year=2003&amp;pages=2141-2146&amp;pmid=12435734&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib11">11. <span class="element-citation" id="sref11">Fritz-Wolf K., Urig S., Becker K. The structure of human thioredoxin reductase 1 provides insights into C-terminal rearrangements during catalysis. <span><span class="ref-journal">J. Mol. Biol. </span>2007;<span class="ref-vol">370</span>(1):116–127.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17512005" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Mol.+Biol.&amp;title=The+structure+of+human+thioredoxin+reductase+1+provides+insights+into+C-terminal+rearrangements+during+catalysis&amp;author=K.+Fritz-Wolf&amp;author=S.+Urig&amp;author=K.+Becker&amp;volume=370&amp;issue=1&amp;publication_year=2007&amp;pages=116-127&amp;pmid=17512005&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib12">12. <span class="element-citation" id="sref12">Fritz-Wolf K., Kehr S., Stumpf M., Rahlfs S., Becker K. Crystal structure of the human thioredoxin reductase-thioredoxin complex. <span><span class="ref-journal">Nat. Commun. </span>2011;<span class="ref-vol">2</span>:383.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21750537" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Crystal+structure+of+the+human+thioredoxin+reductase-thioredoxin+complex&amp;author=K.+Fritz-Wolf&amp;author=S.+Kehr&amp;author=M.+Stumpf&amp;author=S.+Rahlfs&amp;author=K.+Becker&amp;volume=2&amp;publication_year=2011&amp;pages=383&amp;pmid=21750537&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib13">13. <span class="element-citation" id="sref13">Lothrop A.P., Snider G.W., Ruggles E.L., Hondal R.J. Why is mammalian thioredoxin reductase 1 so dependent upon the use of selenium? <span><span class="ref-journal">Biochemistry. </span>2014;<span class="ref-vol">53</span>(3):554–565.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3957196/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24393022" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Why+is+mammalian+thioredoxin+reductase+1+so+dependent+upon+the+use+of+selenium?&amp;author=A.P.+Lothrop&amp;author=G.W.+Snider&amp;author=E.L.+Ruggles&amp;author=R.J.+Hondal&amp;volume=53&amp;issue=3&amp;publication_year=2014&amp;pages=554-565&amp;pmid=24393022&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib14">14. <span class="element-citation" id="sref14">Lothrop A.P., Snider G.W., Ruggles E.L., Patel A.S., Lees W.J., Hondal R.J. Selenium as an electron acceptor during the catalytic mechanism of thioredoxin reductase. <span><span class="ref-journal">Biochemistry. </span>2014;<span class="ref-vol">53</span>(4):654–663.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3957198/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24422500" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Selenium+as+an+electron+acceptor+during+the+catalytic+mechanism+of+thioredoxin+reductase&amp;author=A.P.+Lothrop&amp;author=G.W.+Snider&amp;author=E.L.+Ruggles&amp;author=A.S.+Patel&amp;author=W.J.+Lees&amp;volume=53&amp;issue=4&amp;publication_year=2014&amp;pages=654-663&amp;pmid=24422500&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib15">15. <span class="element-citation" id="sref15">Arn&#x000e9;r E.S.J. Selenoproteins-What unique properties can arise with selenocysteine in place of cysteine? <span><span class="ref-journal">Exp. Cell Res. </span>2010;<span class="ref-vol">316</span>(8):1296–1303.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20206159" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Exp.+Cell+Res.&amp;title=Selenoproteins-What+unique+properties+can+arise+with+selenocysteine+in+place+of+cysteine?&amp;author=E.S.J.+Arn&#x000e9;r&amp;volume=316&amp;issue=8&amp;publication_year=2010&amp;pages=1296-1303&amp;pmid=20206159&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib16">16. <span class="element-citation" id="sref16">Hatfield D.L., Tsuji P.A., Carlson B.A., Gladyshev V.N. Selenium and selenocysteine: roles in cancer, health, and development. <span><span class="ref-journal">Trends Biochem. Sci. </span>2014;<span class="ref-vol">39</span>(3):112–120.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3943681/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24485058" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Biochem.+Sci.&amp;title=Selenium+and+selenocysteine:+roles+in+cancer,+health,+and+development&amp;author=D.L.+Hatfield&amp;author=P.A.+Tsuji&amp;author=B.A.+Carlson&amp;author=V.N.+Gladyshev&amp;volume=39&amp;issue=3&amp;publication_year=2014&amp;pages=112-120&amp;pmid=24485058&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib17">17. <span class="element-citation" id="sref17">Reich H.J., Hondal R.J. Why nature chose selenium. <span><span class="ref-journal">ACS Chem. Biol. </span>2016;<span class="ref-vol">11</span>(4):821–841.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26949981" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=ACS+Chem.+Biol.&amp;title=Why+nature+chose+selenium&amp;author=H.J.+Reich&amp;author=R.J.+Hondal&amp;volume=11&amp;issue=4&amp;publication_year=2016&amp;pages=821-841&amp;pmid=26949981&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib18">18. <span class="element-citation" id="sref18">O'Keefe J.P., Dustin C.M., Barber D., Snider G.W., Hondal R.J. A "seleno effect" differentiates the roles of redox active cysteine residues in plasmodium falciparum thioredoxin reductase. <span><span class="ref-journal">Biochemistry. </span>2018;<span class="ref-vol">57</span>(11):1767–1778.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5866731/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29485860" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=A+&quot;seleno+effect&quot;+differentiates+the+roles+of+redox+active+cysteine+residues+in+plasmodium+falciparum+thioredoxin+reductase&amp;author=J.P.+O'Keefe&amp;author=C.M.+Dustin&amp;author=D.+Barber&amp;author=G.W.+Snider&amp;author=R.J.+Hondal&amp;volume=57&amp;issue=11&amp;publication_year=2018&amp;pages=1767-1778&amp;pmid=29485860&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib19">19. <span class="element-citation" id="sref19">Lothrop A.P., Snider G.W., Flemer S., Jr., Ruggles E.L., Davidson R.S., Lamb A.L., et al. Compensating for the absence of selenocysteine in high-molecular weight thioredoxin reductases: the electrophilic activation hypothesis. <span><span class="ref-journal">Biochemistry. </span>2014;<span class="ref-vol">53</span>(4):664–674.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3931472/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24490974" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Compensating+for+the+absence+of+selenocysteine+in+high-molecular+weight+thioredoxin+reductases:+the+electrophilic+activation+hypothesis&amp;author=A.P.+Lothrop&amp;author=G.W.+Snider&amp;author=S.+Flemer&amp;author=E.L.+Ruggles&amp;author=R.S.+Davidson&amp;volume=53&amp;issue=4&amp;publication_year=2014&amp;pages=664-674&amp;pmid=24490974&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib20">20. <span class="element-citation" id="sref20">Anestal K., Prast-Nielsen S., Cenas N., Arner E.S. Cell death by SecTRAPs: thioredoxin reductase as a prooxidant killer of cells. <span><span class="ref-journal">PLoS One. </span>2008;<span class="ref-vol">3</span>(4)</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2268967/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18382651" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Cell+death+by+SecTRAPs:+thioredoxin+reductase+as+a+prooxidant+killer+of+cells&amp;author=K.+Anestal&amp;author=S.+Prast-Nielsen&amp;author=N.+Cenas&amp;author=E.S.+Arner&amp;volume=3&amp;issue=4&amp;publication_year=2008&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib21">21. <span class="element-citation" id="sref21">Stafford W.C., Peng X., Olofsson M.H., Zhang X., Luci D.K., Lu L., et al. Irreversible inhibition of cytosolic thioredoxin reductase 1 as a mechanistic basis for anticancer therapy. <span><span class="ref-journal">Sci. Transl. Med. </span>2018;<span class="ref-vol">10</span>(428)</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7059553/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29444979" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Transl.+Med.&amp;title=Irreversible+inhibition+of+cytosolic+thioredoxin+reductase+1+as+a+mechanistic+basis+for+anticancer+therapy&amp;author=W.C.+Stafford&amp;author=X.+Peng&amp;author=M.H.+Olofsson&amp;author=X.+Zhang&amp;author=D.K.+Luci&amp;volume=10&amp;issue=428&amp;publication_year=2018&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib22">22. <span class="element-citation" id="sref22">Xu J., Cheng Q., Arner E.S. Details in the catalytic mechanism of mammalian thioredoxin reductase 1 revealed using point mutations and juglone-coupled enzyme activities. <span><span class="ref-journal">Free Radic. Biol. Med. </span>2016;<span class="ref-vol">94</span>:110–120.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26898501" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Free+Radic.+Biol.+Med.&amp;title=Details+in+the+catalytic+mechanism+of+mammalian+thioredoxin+reductase+1+revealed+using+point+mutations+and+juglone-coupled+enzyme+activities&amp;author=J.+Xu&amp;author=Q.+Cheng&amp;author=E.S.+Arner&amp;volume=94&amp;publication_year=2016&amp;pages=110-120&amp;pmid=26898501&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib23">23. <span class="element-citation" id="sref23">Zhang Y., Sun S., Xu W., Yang R., Yang Y., Guo J., et al. Thioredoxin reductase 1 inhibitor shikonin promotes cell necroptosis via SecTRAPs generation and oxygen-coupled redox cycling. <span><span class="ref-journal">Free Radic. Biol. Med. </span>2022;<span class="ref-vol">180</span>:52–62.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34973363" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Free+Radic.+Biol.+Med.&amp;title=Thioredoxin+reductase+1+inhibitor+shikonin+promotes+cell+necroptosis+via+SecTRAPs+generation+and+oxygen-coupled+redox+cycling&amp;author=Y.+Zhang&amp;author=S.+Sun&amp;author=W.+Xu&amp;author=R.+Yang&amp;author=Y.+Yang&amp;volume=180&amp;publication_year=2022&amp;pages=52-62&amp;pmid=34973363&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib24">24. <span class="element-citation" id="sref24">Rackham O., Shearwood A.M., Thyer R., McNamara E., Davies S.M., Callus B.A., et al. Substrate and inhibitor specificities differ between human cytosolic and mitochondrial thioredoxin reductases: implications for development of specific inhibitors. <span><span class="ref-journal">Free Radic. Biol. Med. </span>2011;<span class="ref-vol">50</span>(6):689–699.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21172426" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Free+Radic.+Biol.+Med.&amp;title=Substrate+and+inhibitor+specificities+differ+between+human+cytosolic+and+mitochondrial+thioredoxin+reductases:+implications+for+development+of+specific+inhibitors&amp;author=O.+Rackham&amp;author=A.M.+Shearwood&amp;author=R.+Thyer&amp;author=E.+McNamara&amp;author=S.M.+Davies&amp;volume=50&amp;issue=6&amp;publication_year=2011&amp;pages=689-699&amp;pmid=21172426&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib25">25. <span class="element-citation" id="sref25">Burley S.K., Petsko G.A. Aromatic-aromatic interaction: a mechanism of protein structure stabilization. <span><span class="ref-journal">Science. </span>1985;<span class="ref-vol">229</span>(4708):23–28.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3892686" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Aromatic-aromatic+interaction:+a+mechanism+of+protein+structure+stabilization&amp;author=S.K.+Burley&amp;author=G.A.+Petsko&amp;volume=229&amp;issue=4708&amp;publication_year=1985&amp;pages=23-28&amp;pmid=3892686&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib26">26. <span class="element-citation" id="sref26">Shih C., Museth A.K., Abrahamsson M., Blanco-Rodriguez A.M., Di Bilio A.J., Sudhamsu J., et al. Tryptophan-accelerated electron flow through proteins. <span><span class="ref-journal">Science. </span>2008;<span class="ref-vol">320</span>(5884):1760–1762.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18583608" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Tryptophan-accelerated+electron+flow+through+proteins&amp;author=C.+Shih&amp;author=A.K.+Museth&amp;author=M.+Abrahamsson&amp;author=A.M.+Blanco-Rodriguez&amp;author=A.J.+Di+Bilio&amp;volume=320&amp;issue=5884&amp;publication_year=2008&amp;pages=1760-1762&amp;pmid=18583608&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib27">27. <span class="element-citation" id="sref27">Zhang M.T., Hammarstrom L. Proton-coupled electron transfer from tryptophan: a concerted mechanism with water as proton acceptor. <span><span class="ref-journal">J. Am. Chem. Soc. </span>2011;<span class="ref-vol">133</span>(23):8806–8809.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21500853" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Am.+Chem.+Soc.&amp;title=Proton-coupled+electron+transfer+from+tryptophan:+a+concerted+mechanism+with+water+as+proton+acceptor&amp;author=M.T.+Zhang&amp;author=L.+Hammarstrom&amp;volume=133&amp;issue=23&amp;publication_year=2011&amp;pages=8806-8809&amp;pmid=21500853&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib28">28. <span class="element-citation" id="sref28">Yang R., Sun S., Guo Y., Meng Y., Liu H., Shi M., et al. Anti-inflammatory effect of dimethyl fumarate associates with the inhibition of thioredoxin reductase 1 in RAW 264.7 cells. <span><span class="ref-journal">Molecules. </span>2022;<span class="ref-vol">28</span>(1):107.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9822326/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36615301" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=Anti-inflammatory+effect+of+dimethyl+fumarate+associates+with+the+inhibition+of+thioredoxin+reductase+1+in+RAW+264.7+cells&amp;author=R.+Yang&amp;author=S.+Sun&amp;author=Y.+Guo&amp;author=Y.+Meng&amp;author=H.+Liu&amp;volume=28&amp;issue=1&amp;publication_year=2022&amp;pages=107&amp;pmid=36615301&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib29">29. <span class="element-citation" id="sref29">Arn&#x000e9;r E.S.J., Sarioglu H., Lottspeich F., Holmgren A., Bock A. High-level expression in Escherichia coli of selenocysteine-containing rat thioredoxin reductase utilizing gene fusions with engineered bacterial-type SECIS elements and co-expression with the selA, selB and selC genes. <span><span class="ref-journal">J. Mol. Biol. </span>1999;<span class="ref-vol">292</span>(5):1003–1016.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10512699" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Mol.+Biol.&amp;title=High-level+expression+in+Escherichia+coli+of+selenocysteine-containing+rat+thioredoxin+reductase+utilizing+gene+fusions+with+engineered+bacterial-type+SECIS+elements+and+co-expression+with+the+selA,+selB+and+selC+genes&amp;author=E.S.J.+Arn&#x000e9;r&amp;author=H.+Sarioglu&amp;author=F.+Lottspeich&amp;author=A.+Holmgren&amp;author=A.+Bock&amp;volume=292&amp;issue=5&amp;publication_year=1999&amp;pages=1003-1016&amp;pmid=10512699&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib30">30. <span class="element-citation" id="sref30">Xu J., Croitoru V., Rutishauser D., Cheng Q., Arner E.S. Wobble decoding by the Escherichia coli selenocysteine insertion machinery. <span><span class="ref-journal">Nucleic Acids Res. </span>2013;<span class="ref-vol">41</span>(21):9800–9811.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3834832/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23982514" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Wobble+decoding+by+the+Escherichia+coli+selenocysteine+insertion+machinery&amp;author=J.+Xu&amp;author=V.+Croitoru&amp;author=D.+Rutishauser&amp;author=Q.+Cheng&amp;author=E.S.+Arner&amp;volume=41&amp;issue=21&amp;publication_year=2013&amp;pages=9800-9811&amp;pmid=23982514&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib31">31. <span class="element-citation" id="sref31">Arscott L.D., Gromer S., Schirmer R.H., Becker K., Williams C.H., Jr. The mechanism of thioredoxin reductase from human placenta is similar to the mechanisms of lipoamide dehydrogenase and glutathione reductase and is distinct from the mechanism of thioredoxin reductase from Escherichia coli. <span><span class="ref-journal">Proc. Natl. Acad. Sci. U. S. A. </span>1997;<span class="ref-vol">94</span>(8):3621–3626.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC20490/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9108027" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.+S.+A.&amp;title=The+mechanism+of+thioredoxin+reductase+from+human+placenta+is+similar+to+the+mechanisms+of+lipoamide+dehydrogenase+and+glutathione+reductase+and+is+distinct+from+the+mechanism+of+thioredoxin+reductase+from+Escherichia+coli&amp;author=L.D.+Arscott&amp;author=S.+Gromer&amp;author=R.H.+Schirmer&amp;author=K.+Becker&amp;author=C.H.+Williams&amp;volume=94&amp;issue=8&amp;publication_year=1997&amp;pages=3621-3626&amp;pmid=9108027&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib32">32. <span class="element-citation" id="sref32">Johansson L., Arscott L.D., Ballou D.P., Williams C.H., Jr., Arner E.S. Studies of an active site mutant of the selenoprotein thioredoxin reductase: the Ser-Cys-Cys-Ser motif of the insect orthologue is not sufficient to replace the Cys-Sec dyad in the mammalian enzyme. <span><span class="ref-journal">Free Radic. Biol. Med. </span>2006;<span class="ref-vol">41</span>(4):649–656.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16863998" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Free+Radic.+Biol.+Med.&amp;title=Studies+of+an+active+site+mutant+of+the+selenoprotein+thioredoxin+reductase:+the+Ser-Cys-Cys-Ser+motif+of+the+insect+orthologue+is+not+sufficient+to+replace+the+Cys-Sec+dyad+in+the+mammalian+enzyme&amp;author=L.+Johansson&amp;author=L.D.+Arscott&amp;author=D.P.+Ballou&amp;author=C.H.+Williams&amp;author=E.S.+Arner&amp;volume=41&amp;issue=4&amp;publication_year=2006&amp;pages=649-656&amp;pmid=16863998&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib33">33. <span class="element-citation" id="sref33">Munro A.W., Noble M.A. Fluorescence analysis of flavoproteins. <span><span class="ref-journal">Methods Mol. Biol. </span>1999;<span class="ref-vol">131</span>:25–48.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10494540" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Mol.+Biol.&amp;title=Fluorescence+analysis+of+flavoproteins&amp;author=A.W.+Munro&amp;author=M.A.+Noble&amp;volume=131&amp;publication_year=1999&amp;pages=25-48&amp;pmid=10494540&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib34">34. <span class="element-citation" id="sref34">Xu J., Eriksson S.E., Cebula M., Sandalova T., Hedstrom E., Pader I., et al. The conserved Trp114 residue of thioredoxin reductase 1 has a redox sensor-like function triggering oligomerization and crosslinking upon oxidative stress related to cell death. <span><span class="ref-journal">Cell Death Dis. </span>2015;<span class="ref-vol">6</span>:e1616.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4669772/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25611390" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Death+Dis.&amp;title=The+conserved+Trp114+residue+of+thioredoxin+reductase+1+has+a+redox+sensor-like+function+triggering+oligomerization+and+crosslinking+upon+oxidative+stress+related+to+cell+death&amp;author=J.+Xu&amp;author=S.E.+Eriksson&amp;author=M.+Cebula&amp;author=T.+Sandalova&amp;author=E.+Hedstrom&amp;volume=6&amp;publication_year=2015&amp;pages=e1616&amp;pmid=25611390&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib35">35. <span class="element-citation" id="sref35">Wu T., Hornsby M., Zhu L., Yu J.C., Shokat K.M., Gestwicki J.E. Protocol for performing and optimizing differential scanning fluorimetry experiments. <span><span class="ref-journal">STAR Protoc. </span>2023;<span class="ref-vol">4</span>(4)</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10663957/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37943662" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=STAR+Protoc&amp;title=Protocol+for+performing+and+optimizing+differential+scanning+fluorimetry+experiments&amp;author=T.+Wu&amp;author=M.+Hornsby&amp;author=L.+Zhu&amp;author=J.C.+Yu&amp;author=K.M.+Shokat&amp;volume=4&amp;issue=4&amp;publication_year=2023&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib36">36. <span class="element-citation" id="sref36">Cheff D.M., Huang C., Scholzen K.C., Gencheva R., Ronzetti M.H., Cheng Q., et al. The ferroptosis inducing compounds RSL3 and ML162 are not direct inhibitors of GPX4 but of TXNRD1. <span><span class="ref-journal">Redox Biol. </span>2023;<span class="ref-vol">62</span></span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10149367/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37087975" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Redox+Biol.&amp;title=The+ferroptosis+inducing+compounds+RSL3+and+ML162+are+not+direct+inhibitors+of+GPX4+but+of+TXNRD1&amp;author=D.M.+Cheff&amp;author=C.+Huang&amp;author=K.C.+Scholzen&amp;author=R.+Gencheva&amp;author=M.H.+Ronzetti&amp;volume=62&amp;publication_year=2023&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib37">37. <span class="element-citation" id="sref37">Cenas N., Nivinskas H., Anusevicius Z., Sarlauskas J., Lederer F., Arn&#x000e9;r E.S.J. Interactions of quinones with thioredoxin reductase: a challenge to the antioxidant role of the mammalian selenoprotein. <span><span class="ref-journal">J. Biol. Chem. </span>2004;<span class="ref-vol">279</span>(4):2583–2592.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14604985" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Interactions+of+quinones+with+thioredoxin+reductase:+a+challenge+to+the+antioxidant+role+of+the+mammalian+selenoprotein&amp;author=N.+Cenas&amp;author=H.+Nivinskas&amp;author=Z.+Anusevicius&amp;author=J.+Sarlauskas&amp;author=F.+Lederer&amp;volume=279&amp;issue=4&amp;publication_year=2004&amp;pages=2583-2592&amp;pmid=14604985&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib38">38. <span class="element-citation" id="sref38">Pader I., Sengupta R., Cebula M., Xu J., Lundberg J.O., Holmgren A., et al. Thioredoxin-related protein of 14 kDa is an efficient L-cystine reductase and S-denitrosylase. <span><span class="ref-journal">Proc. Natl. Acad. Sci. U. S. A. </span>2014;<span class="ref-vol">111</span>(19):6964–6969.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4024855/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24778250" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.+S.+A.&amp;title=Thioredoxin-related+protein+of+14+kDa+is+an+efficient+L-cystine+reductase+and+S-denitrosylase&amp;author=I.+Pader&amp;author=R.+Sengupta&amp;author=M.+Cebula&amp;author=J.+Xu&amp;author=J.O.+Lundberg&amp;volume=111&amp;issue=19&amp;publication_year=2014&amp;pages=6964-6969&amp;pmid=24778250&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib39">39. <span class="element-citation" id="sref39">Aldag C., Brocker M.J., Hohn M.J., Prat L., Hammond G., Plummer A., et al. Rewiring translation for elongation factor Tu-dependent selenocysteine incorporation. <span><span class="ref-journal">Angew Chem. Int. Ed. Engl. </span>2013;<span class="ref-vol">52</span>(5):1441–1445.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3776052/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23193031" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Angew+Chem.+Int.+Ed.+Engl.&amp;title=Rewiring+translation+for+elongation+factor+Tu-dependent+selenocysteine+incorporation&amp;author=C.+Aldag&amp;author=M.J.+Brocker&amp;author=M.J.+Hohn&amp;author=L.+Prat&amp;author=G.+Hammond&amp;volume=52&amp;issue=5&amp;publication_year=2013&amp;pages=1441-1445&amp;pmid=23193031&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib40">40. <span class="element-citation" id="sref40">Cheng Q., Arn&#x000e9;r E.S. Selenocysteine insertion at a predefined UAG codon in a release factor 1 (RF1)-depleted Escherichia coli host strain bypasses species barriers in recombinant selenoprotein translation. <span><span class="ref-journal">J. Biol. Chem. </span>2017;<span class="ref-vol">292</span>(13):5476–5487.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5392690/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28193838" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Selenocysteine+insertion+at+a+predefined+UAG+codon+in+a+release+factor+1+(RF1)-depleted+Escherichia+coli+host+strain+bypasses+species+barriers+in+recombinant+selenoprotein+translation&amp;author=Q.+Cheng&amp;author=E.S.+Arn&#x000e9;r&amp;volume=292&amp;issue=13&amp;publication_year=2017&amp;pages=5476-5487&amp;pmid=28193838&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib41">41. <span class="element-citation" id="sref41">Eaton J.K., Furst L., Ruberto R.A., Moosmayer D., Hilpmann A., Ryan M.J., et al. Selective covalent targeting of GPX4 using masked nitrile-oxide electrophiles. <span><span class="ref-journal">Nat. Chem. Biol. </span>2020;<span class="ref-vol">16</span>(5):497–506.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7251976/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32231343" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Chem.+Biol.&amp;title=Selective+covalent+targeting+of+GPX4+using+masked+nitrile-oxide+electrophiles&amp;author=J.K.+Eaton&amp;author=L.+Furst&amp;author=R.A.+Ruberto&amp;author=D.+Moosmayer&amp;author=A.+Hilpmann&amp;volume=16&amp;issue=5&amp;publication_year=2020&amp;pages=497-506&amp;pmid=32231343&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib42">42. <span class="element-citation" id="sref42">Fan Z., Song J., Guan T., Lv X., Wei J. Efficient expression of glutathione peroxidase with chimeric tRNA in amber-less Escherichia coli. <span><span class="ref-journal">ACS Synth. Biol. </span>2018;<span class="ref-vol">7</span>(1):249–257.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28866886" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=ACS+Synth.+Biol.&amp;title=Efficient+expression+of+glutathione+peroxidase+with+chimeric+tRNA+in+amber-less+Escherichia+coli&amp;author=Z.+Fan&amp;author=J.+Song&amp;author=T.+Guan&amp;author=X.+Lv&amp;author=J.+Wei&amp;volume=7&amp;issue=1&amp;publication_year=2018&amp;pages=249-257&amp;pmid=28866886&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib43">43. <span class="element-citation" id="sref43">Eckenroth B., Harris K., Turanov A.A., Gladyshev V.N., Raines R.T., Hondal R.J. Semisynthesis and characterization of mammalian thioredoxin reductase. <span><span class="ref-journal">Biochemistry. </span>2006;<span class="ref-vol">45</span>(16):5158–5170.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2570056/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16618105" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Semisynthesis+and+characterization+of+mammalian+thioredoxin+reductase&amp;author=B.+Eckenroth&amp;author=K.+Harris&amp;author=A.A.+Turanov&amp;author=V.N.+Gladyshev&amp;author=R.T.+Raines&amp;volume=45&amp;issue=16&amp;publication_year=2006&amp;pages=5158-5170&amp;pmid=16618105&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib44">44. <span class="element-citation" id="sref44">Fu X., Soll D., Sevostyanova A. Challenges of site-specific selenocysteine incorporation into proteins by Escherichia coli. <span><span class="ref-journal">RNA Biol. </span>2018;<span class="ref-vol">15</span>(4&#x02013;5):461–470.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6103685/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29447106" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=RNA+Biol.&amp;title=Challenges+of+site-specific+selenocysteine+incorporation+into+proteins+by+Escherichia+coli&amp;author=X.+Fu&amp;author=D.+Soll&amp;author=A.+Sevostyanova&amp;volume=15&amp;issue=4&#x02013;5&amp;publication_year=2018&amp;pages=461-470&amp;pmid=29447106&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib45">45. <span class="element-citation" id="sref45">Hilal T., Killam B.Y., Grozdanovic M., Dobosz-Bartoszek M., Loerke J., Burger J., et al. Structure of the mammalian ribosome as it decodes the selenocysteine UGA codon. <span><span class="ref-journal">Science. </span>2022;<span class="ref-vol">376</span>(6599):1338–1343.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10064918/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35709277" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Structure+of+the+mammalian+ribosome+as+it+decodes+the+selenocysteine+UGA+codon&amp;author=T.+Hilal&amp;author=B.Y.+Killam&amp;author=M.+Grozdanovic&amp;author=M.+Dobosz-Bartoszek&amp;author=J.+Loerke&amp;volume=376&amp;issue=6599&amp;publication_year=2022&amp;pages=1338-1343&amp;pmid=35709277&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib46">46. <span class="element-citation" id="sref46">Cheng Q., Antholine W.E., Myers J.M., Kalyanaraman B., Arn&#x000e9;r E.S.J., Myers C.R. The selenium-independent inherent pro-oxidant NADPH oxidase activity of mammalian thioredoxin reductase and its selenium-dependent direct peroxidase activities. <span><span class="ref-journal">J. Biol. Chem. </span>2010;<span class="ref-vol">285</span>(28):21708–21723.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2898413/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20457604" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=The+selenium-independent+inherent+pro-oxidant+NADPH+oxidase+activity+of+mammalian+thioredoxin+reductase+and+its+selenium-dependent+direct+peroxidase+activities&amp;author=Q.+Cheng&amp;author=W.E.+Antholine&amp;author=J.M.+Myers&amp;author=B.+Kalyanaraman&amp;author=E.S.J.+Arn&#x000e9;r&amp;volume=285&amp;issue=28&amp;publication_year=2010&amp;pages=21708-21723&amp;pmid=20457604&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib47">47. <span class="element-citation" id="sref47">Sun S., Xu W., Zhang Y., Yang Y., Ma Q., Xu J. Menadione inhibits thioredoxin reductase 1 via arylation at the Sec(498) residue and enhances both NADPH oxidation and superoxide production in Sec(498) to Cys(498) substitution. <span><span class="ref-journal">Free Radic. Biol. Med. </span>2021;<span class="ref-vol">172</span>:482–489.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34186208" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Free+Radic.+Biol.+Med.&amp;title=Menadione+inhibits+thioredoxin+reductase+1+via+arylation+at+the+Sec(498)+residue+and+enhances+both+NADPH+oxidation+and+superoxide+production+in+Sec(498)+to+Cys(498)+substitution&amp;author=S.+Sun&amp;author=W.+Xu&amp;author=Y.+Zhang&amp;author=Y.+Yang&amp;author=Q.+Ma&amp;volume=172&amp;publication_year=2021&amp;pages=482-489&amp;pmid=34186208&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib48">48. <span class="element-citation" id="sref48">Sun S., Zhang Y., Xu W., Yang R., Yang Y., Guo J., et al. Plumbagin reduction by thioredoxin reductase 1 possesses synergy effects with GLUT1 inhibitor on KEAP1-mutant NSCLC cells. <span><span class="ref-journal">Biomed. Pharmacother. </span>2022;<span class="ref-vol">146</span></span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34954641" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biomed.+Pharmacother.&amp;title=Plumbagin+reduction+by+thioredoxin+reductase+1+possesses+synergy+effects+with+GLUT1+inhibitor+on+KEAP1-mutant+NSCLC+cells&amp;author=S.+Sun&amp;author=Y.+Zhang&amp;author=W.+Xu&amp;author=R.+Yang&amp;author=Y.+Yang&amp;volume=146&amp;publication_year=2022&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib49">49. <span class="element-citation" id="sref49">Li J., Zuo X., Cheng P., Ren X., Sun S., Xu J., et al. The production of reactive oxygen species enhanced with the reduction of menadione by active thioredoxin reductase. <span><span class="ref-journal">Metallomics. </span>2019;<span class="ref-vol">11</span>(9):1490–1497.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31359011" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Metallomics&amp;title=The+production+of+reactive+oxygen+species+enhanced+with+the+reduction+of+menadione+by+active+thioredoxin+reductase&amp;author=J.+Li&amp;author=X.+Zuo&amp;author=P.+Cheng&amp;author=X.+Ren&amp;author=S.+Sun&amp;volume=11&amp;issue=9&amp;publication_year=2019&amp;pages=1490-1497&amp;pmid=31359011&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib50">50. <span class="element-citation" id="sref50">Saccoccia F., Angelucci F., Boumis G., Carotti D., Desiato G., Miele A.E., et al. Thioredoxin reductase and its inhibitors. <span><span class="ref-journal">Curr. Protein Pept. Sci. </span>2014;<span class="ref-vol">15</span>(6):621–646.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4275836/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24875642" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Protein+Pept.+Sci.&amp;title=Thioredoxin+reductase+and+its+inhibitors&amp;author=F.+Saccoccia&amp;author=F.+Angelucci&amp;author=G.+Boumis&amp;author=D.+Carotti&amp;author=G.+Desiato&amp;volume=15&amp;issue=6&amp;publication_year=2014&amp;pages=621-646&amp;pmid=24875642&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib51">51. <span class="element-citation" id="sref51">Zhang J., Duan D., Song Z.L., Liu T., Hou Y., Fang J. Small molecules regulating reactive oxygen species homeostasis for cancer therapy. <span><span class="ref-journal">Med. Res. Rev. </span>2021;<span class="ref-vol">41</span>(1):342–394.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32981100" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Med.+Res.+Rev.&amp;title=Small+molecules+regulating+reactive+oxygen+species+homeostasis+for+cancer+therapy&amp;author=J.+Zhang&amp;author=D.+Duan&amp;author=Z.L.+Song&amp;author=T.+Liu&amp;author=Y.+Hou&amp;volume=41&amp;issue=1&amp;publication_year=2021&amp;pages=342-394&amp;pmid=32981100&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib52">52. <span class="element-citation" id="sref52">Sabatier P., Beusch C.M., Gencheva R., Cheng Q., Zubarev R.A., Arn&#x000e9;r E.S.J. Comprehensive chemical proteomics analyses reveal that the new TRi-1 and TRi-2 compounds are more specific thioredoxin reductase 1 inhibitors than auranofin. <span><span class="ref-journal">Redox Biol. </span>2021;<span class="ref-vol">48</span></span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8591550/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34788728" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Redox+Biol.&amp;title=Comprehensive+chemical+proteomics+analyses+reveal+that+the+new+TRi-1+and+TRi-2+compounds+are+more+specific+thioredoxin+reductase+1+inhibitors+than+auranofin&amp;author=P.+Sabatier&amp;author=C.M.+Beusch&amp;author=R.+Gencheva&amp;author=Q.+Cheng&amp;author=R.A.+Zubarev&amp;volume=48&amp;publication_year=2021&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib53">53. <span class="element-citation" id="sref53">Dixon S.J., Lemberg K.M., Lamprecht M.R., Skouta R., Zaitsev E.M., Gleason C.E., et al. Ferroptosis: an iron-dependent form of nonapoptotic cell death. <span><span class="ref-journal">Cell. </span>2012;<span class="ref-vol">149</span>(5):1060–1072.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3367386/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22632970" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Ferroptosis:+an+iron-dependent+form+of+nonapoptotic+cell+death&amp;author=S.J.+Dixon&amp;author=K.M.+Lemberg&amp;author=M.R.+Lamprecht&amp;author=R.+Skouta&amp;author=E.M.+Zaitsev&amp;volume=149&amp;issue=5&amp;publication_year=2012&amp;pages=1060-1072&amp;pmid=22632970&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib54">54. <span class="element-citation" id="sref54">Liu H., Forouhar F., Lin A.J., Wang Q., Polychronidou V., Soni R.K., et al. Small-molecule allosteric inhibitors of GPX4. <span><span class="ref-journal">Cell Chem. Biol. </span>2022;<span class="ref-vol">29</span>(12):1680–16893.</span> e9. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9772252/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36423641" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Chem.+Biol.&amp;title=Small-molecule+allosteric+inhibitors+of+GPX4&amp;author=H.+Liu&amp;author=F.+Forouhar&amp;author=A.J.+Lin&amp;author=Q.+Wang&amp;author=V.+Polychronidou&amp;volume=29&amp;issue=12&amp;publication_year=2022&amp;pages=1680-16893&amp;pmid=36423641&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib55">55. <span class="element-citation" id="sref55">Marzano C., Gandin V., Folda A., Scutari G., Bindoli A., Rigobello M.P. Inhibition of thioredoxin reductase by auranofin induces apoptosis in cisplatin-resistant human ovarian cancer cells. <span><span class="ref-journal">Free Radic. Biol. Med. </span>2007;<span class="ref-vol">42</span>(6):872–881.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17320769" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Free+Radic.+Biol.+Med.&amp;title=Inhibition+of+thioredoxin+reductase+by+auranofin+induces+apoptosis+in+cisplatin-resistant+human+ovarian+cancer+cells&amp;author=C.+Marzano&amp;author=V.+Gandin&amp;author=A.+Folda&amp;author=G.+Scutari&amp;author=A.+Bindoli&amp;volume=42&amp;issue=6&amp;publication_year=2007&amp;pages=872-881&amp;pmid=17320769&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib56">56. <span class="element-citation" id="sref56">Prast-Nielsen S., Cebula M., Pader I., Arn&#x000e9;r E.S.J. Noble metal targeting of thioredoxin reductase--covalent complexes with thioredoxin and thioredoxin-related protein of 14 kDa triggered by cisplatin. <span><span class="ref-journal">Free Radic. Biol. Med. </span>2010;<span class="ref-vol">49</span>(11):1765–1778.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20851179" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Free+Radic.+Biol.+Med.&amp;title=Noble+metal+targeting+of+thioredoxin+reductase--covalent+complexes+with+thioredoxin+and+thioredoxin-related+protein+of+14+kDa+triggered+by+cisplatin&amp;author=S.+Prast-Nielsen&amp;author=M.+Cebula&amp;author=I.+Pader&amp;author=E.S.J.+Arn&#x000e9;r&amp;volume=49&amp;issue=11&amp;publication_year=2010&amp;pages=1765-1778&amp;pmid=20851179&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib57">57. <span class="element-citation" id="sref57">Peng X., Zhang M.Q., Conserva F., Hosny G., Selivanova G., Bykov V.J., et al. APR-246/PRIMA-1MET inhibits thioredoxin reductase 1 and converts the enzyme to a dedicated NADPH oxidase. <span><span class="ref-journal">Cell Death Dis. </span>2013;<span class="ref-vol">4</span>:e881.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3920950/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24157875" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Death+Dis.&amp;title=APR-246/PRIMA-1MET+inhibits+thioredoxin+reductase+1+and+converts+the+enzyme+to+a+dedicated+NADPH+oxidase&amp;author=X.+Peng&amp;author=M.Q.+Zhang&amp;author=F.+Conserva&amp;author=G.+Hosny&amp;author=G.+Selivanova&amp;volume=4&amp;publication_year=2013&amp;pages=e881&amp;pmid=24157875&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib58">58. <span class="element-citation" id="sref58">Volonte D., Galbiati F. Inhibition of thioredoxin reductase 1 by caveolin 1 promotes stress-induced premature senescence. <span><span class="ref-journal">EMBO Rep. </span>2009;<span class="ref-vol">10</span>(12):1334–1340.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2799211/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19820694" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+Rep.&amp;title=Inhibition+of+thioredoxin+reductase+1+by+caveolin+1+promotes+stress-induced+premature+senescence&amp;author=D.+Volonte&amp;author=F.+Galbiati&amp;volume=10&amp;issue=12&amp;publication_year=2009&amp;pages=1334-1340&amp;pmid=19820694&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib59">59. <span class="element-citation" id="sref59">Collins B.M., Davis M.J., Hancock J.F., Parton R.G. Structure-based reassessment of the caveolin signaling model: do caveolae regulate signaling through caveolin-protein interactions? <span><span class="ref-journal">Dev. Cell. </span>2012;<span class="ref-vol">23</span>(1):11–20.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3427029/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22814599" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev.+Cell&amp;title=Structure-based+reassessment+of+the+caveolin+signaling+model:+do+caveolae+regulate+signaling+through+caveolin-protein+interactions?&amp;author=B.M.+Collins&amp;author=M.J.+Davis&amp;author=J.F.+Hancock&amp;author=R.G.+Parton&amp;volume=23&amp;issue=1&amp;publication_year=2012&amp;pages=11-20&amp;pmid=22814599&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib60">60. <span class="element-citation" id="sref60">Bucci M., Gratton J.P., Rudic R.D., Acevedo L., Roviezzo F., Cirino G., et al. In vivo delivery of the caveolin-1 scaffolding domain inhibits nitric oxide synthesis and reduces inflammation. <span><span class="ref-journal">Nat. Med. </span>2000;<span class="ref-vol">6</span>(12):1362–1367.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11100121" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Med.&amp;title=In+vivo+delivery+of+the+caveolin-1+scaffolding+domain+inhibits+nitric+oxide+synthesis+and+reduces+inflammation&amp;author=M.+Bucci&amp;author=J.P.+Gratton&amp;author=R.D.+Rudic&amp;author=L.+Acevedo&amp;author=F.+Roviezzo&amp;volume=6&amp;issue=12&amp;publication_year=2000&amp;pages=1362-1367&amp;pmid=11100121&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib61">61. <span class="element-citation" id="sref61">Gratton J.P., Lin M.I., Yu J., Weiss E.D., Jiang Z.L., Fairchild T.A., et al. Selective inhibition of tumor microvascular permeability by cavtratin blocks tumor progression in mice. <span><span class="ref-journal">Cancer Cell. </span>2003;<span class="ref-vol">4</span>(1):31–39.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12892711" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Cell&amp;title=Selective+inhibition+of+tumor+microvascular+permeability+by+cavtratin+blocks+tumor+progression+in+mice&amp;author=J.P.+Gratton&amp;author=M.I.+Lin&amp;author=J.+Yu&amp;author=E.D.+Weiss&amp;author=Z.L.+Jiang&amp;volume=4&amp;issue=1&amp;publication_year=2003&amp;pages=31-39&amp;pmid=12892711&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib62">62. <span class="element-citation" id="sref62">Marudamuthu A.S., Bhandary Y.P., Fan L., Radhakrishnan V., MacKenzie B., Maier E., et al. Caveolin-1-derived peptide limits development of pulmonary fibrosis. <span><span class="ref-journal">Sci. Transl. Med. </span>2019;<span class="ref-vol">11</span>(522)</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31826982" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Transl.+Med.&amp;title=Caveolin-1-derived+peptide+limits+development+of+pulmonary+fibrosis&amp;author=A.S.+Marudamuthu&amp;author=Y.P.+Bhandary&amp;author=L.+Fan&amp;author=V.+Radhakrishnan&amp;author=B.+MacKenzie&amp;volume=11&amp;issue=522&amp;publication_year=2019&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib63">63. <span class="element-citation" id="sref63">Gopu V., Fan L., Shetty R.S., Nagaraja M.R., Shetty S. Caveolin-1 scaffolding domain peptide regulates glucose metabolism in lung fibrosis. <span><span class="ref-journal">JCI Insight. </span>2020;<span class="ref-vol">5</span>(19)</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7566714/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32841217" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JCI+Insight&amp;title=Caveolin-1+scaffolding+domain+peptide+regulates+glucose+metabolism+in+lung+fibrosis&amp;author=V.+Gopu&amp;author=L.+Fan&amp;author=R.S.+Shetty&amp;author=M.R.+Nagaraja&amp;author=S.+Shetty&amp;volume=5&amp;issue=19&amp;publication_year=2020&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib64">64. <span class="element-citation" id="sref64">Johnson F.D., Ferrarone J., Liu A., Brandstadter C., Munuganti R., Farnsworth D.A., et al. Characterization of a small molecule inhibitor of disulfide reductases that induces oxidative stress and lethality in lung cancer cells. <span><span class="ref-journal">Cell Rep. </span>2022;<span class="ref-vol">38</span>(6)</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8904132/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35139387" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Rep.&amp;title=Characterization+of+a+small+molecule+inhibitor+of+disulfide+reductases+that+induces+oxidative+stress+and+lethality+in+lung+cancer+cells&amp;author=F.D.+Johnson&amp;author=J.+Ferrarone&amp;author=A.+Liu&amp;author=C.+Brandstadter&amp;author=R.+Munuganti&amp;volume=38&amp;issue=6&amp;publication_year=2022&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib65">65. <span class="element-citation" id="sref65">Zhong L., Arner E.S., Ljung J., Aslund F., Holmgren A. Rat and calf thioredoxin reductase are homologous to glutathione reductase with a carboxyl-terminal elongation containing a conserved catalytically active penultimate selenocysteine residue. <span><span class="ref-journal">J. Biol. Chem. </span>1998;<span class="ref-vol">273</span>(15):8581–8591.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9535831" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Rat+and+calf+thioredoxin+reductase+are+homologous+to+glutathione+reductase+with+a+carboxyl-terminal+elongation+containing+a+conserved+catalytically+active+penultimate+selenocysteine+residue&amp;author=L.+Zhong&amp;author=E.S.+Arner&amp;author=J.+Ljung&amp;author=F.+Aslund&amp;author=A.+Holmgren&amp;volume=273&amp;issue=15&amp;publication_year=1998&amp;pages=8581-8591&amp;pmid=9535831&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib66">66. <span class="element-citation" id="sref66">Liu S.Y., Stadtman T.C. Heparin-binding properties of selenium-containing thioredoxin reductase from HeLa cells and human lung adenocarcinoma cells. <span><span class="ref-journal">Proc. Natl. Acad. Sci. U. S. A. </span>1997;<span class="ref-vol">94</span>(12):6138–6141.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC21015/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9177183" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.+S.+A.&amp;title=Heparin-binding+properties+of+selenium-containing+thioredoxin+reductase+from+HeLa+cells+and+human+lung+adenocarcinoma+cells&amp;author=S.Y.+Liu&amp;author=T.C.+Stadtman&amp;volume=94&amp;issue=12&amp;publication_year=1997&amp;pages=6138-6141&amp;pmid=9177183&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib67">67. <span class="element-citation" id="sref67">Shu N., Lorentzen L.G., Davies M.J. Reaction of quinones with proteins: kinetics of adduct formation, effects on enzymatic activity and protein structure, and potential reversibility of modifications. <span><span class="ref-journal">Free Radic. Biol. Med. </span>2019;<span class="ref-vol">137</span>:169–180.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31026584" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Free+Radic.+Biol.+Med.&amp;title=Reaction+of+quinones+with+proteins:+kinetics+of+adduct+formation,+effects+on+enzymatic+activity+and+protein+structure,+and+potential+reversibility+of+modifications&amp;author=N.+Shu&amp;author=L.G.+Lorentzen&amp;author=M.J.+Davies&amp;volume=137&amp;publication_year=2019&amp;pages=169-180&amp;pmid=31026584&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib68">68. <span class="element-citation" id="sref68">Witte A.B., Anestal K., Jerremalm E., Ehrsson H., Arn&#x000e9;r E.S.J. Inhibition of thioredoxin reductase but not of glutathione reductase by the major classes of alkylating and platinum-containing anticancer compounds. <span><span class="ref-journal">Free Radic. Biol. Med. </span>2005;<span class="ref-vol">39</span>(5):696–703.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16085187" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Free+Radic.+Biol.+Med.&amp;title=Inhibition+of+thioredoxin+reductase+but+not+of+glutathione+reductase+by+the+major+classes+of+alkylating+and+platinum-containing+anticancer+compounds&amp;author=A.B.+Witte&amp;author=K.+Anestal&amp;author=E.+Jerremalm&amp;author=H.+Ehrsson&amp;author=E.S.J.+Arn&#x000e9;r&amp;volume=39&amp;issue=5&amp;publication_year=2005&amp;pages=696-703&amp;pmid=16085187&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib69">69. <span class="element-citation" id="sref69">Parton R.G. Caveolae: structure, function, and relationship to disease. <span><span class="ref-journal">Annu. Rev. Cell Dev. Biol. </span>2018;<span class="ref-vol">34</span>:111–136.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30296391" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Cell+Dev.+Biol.&amp;title=Caveolae:+structure,+function,+and+relationship+to+disease&amp;author=R.G.+Parton&amp;volume=34&amp;publication_year=2018&amp;pages=111-136&amp;pmid=30296391&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib70">70. <span class="element-citation" id="sref70">Parton R.G., Del Pozo M.A., Vassilopoulos S., Nabi I.R., Le Lay S., Lundmark R., et al. Caveolae: the FAQs. <span><span class="ref-journal">Traffic. </span>2020;<span class="ref-vol">21</span>(1):181–185.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7073450/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31448516" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Traffic&amp;title=Caveolae:+the+FAQs&amp;author=R.G.+Parton&amp;author=M.A.+Del+Pozo&amp;author=S.+Vassilopoulos&amp;author=I.R.+Nabi&amp;author=S.+Le+Lay&amp;volume=21&amp;issue=1&amp;publication_year=2020&amp;pages=181-185&amp;pmid=31448516&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib71">71. <span class="element-citation" id="sref71">Wu Y., Lim Y.W., Stroud D.A., Martel N., Hall T.E., Lo H.P., et al. Caveolae sense oxidative stress through membrane lipid peroxidation and cytosolic release of CAVIN1 to regulate NRF2. <span><span class="ref-journal">Dev. Cell. </span>2023;<span class="ref-vol">58</span>(5):376–397 e4.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36858041" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev.+Cell&amp;title=Caveolae+sense+oxidative+stress+through+membrane+lipid+peroxidation+and+cytosolic+release+of+CAVIN1+to+regulate+NRF2&amp;author=Y.+Wu&amp;author=Y.W.+Lim&amp;author=D.A.+Stroud&amp;author=N.+Martel&amp;author=T.E.+Hall&amp;volume=58&amp;issue=5&amp;publication_year=2023&amp;pages=376-397+e4&amp;pmid=36858041&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib72">72. <span class="element-citation" id="sref72">Cebula M., Moolla N., Capovilla A., Arn&#x000e9;r E.S.J. The rare TXNRD1_v3 ("v3") splice variant of human thioredoxin reductase 1 protein is targeted to membrane rafts by N-acylation and induces filopodia independently of its redox active site integrity. <span><span class="ref-journal">J. Biol. Chem. </span>2013;<span class="ref-vol">288</span>(14):10002–10011.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3617239/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23413027" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=The+rare+TXNRD1_v3+(&quot;v3&quot;)+splice+variant+of+human+thioredoxin+reductase+1+protein+is+targeted+to+membrane+rafts+by+N-acylation+and+induces+filopodia+independently+of+its+redox+active+site+integrity&amp;author=M.+Cebula&amp;author=N.+Moolla&amp;author=A.+Capovilla&amp;author=E.S.J.+Arn&#x000e9;r&amp;volume=288&amp;issue=14&amp;publication_year=2013&amp;pages=10002-10011&amp;pmid=23413027&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib73">73. <span class="element-citation" id="sref73">Hayes J.D., Dinkova-Kostova A.T., Tew K.D. Oxidative stress in cancer. <span><span class="ref-journal">Cancer Cell. </span>2020;<span class="ref-vol">38</span>(2):167–197.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7439808/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32649885" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Cell&amp;title=Oxidative+stress+in+cancer&amp;author=J.D.+Hayes&amp;author=A.T.+Dinkova-Kostova&amp;author=K.D.+Tew&amp;volume=38&amp;issue=2&amp;publication_year=2020&amp;pages=167-197&amp;pmid=32649885&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib74">74. <span class="element-citation" id="sref74">Zhang J., Li X., Han X., Liu R., Fang J. Targeting the thioredoxin system for cancer therapy. <span><span class="ref-journal">Trends Pharmacol. Sci. </span>2017;<span class="ref-vol">38</span>(9):794–808.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28648527" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Pharmacol.+Sci.&amp;title=Targeting+the+thioredoxin+system+for+cancer+therapy&amp;author=J.+Zhang&amp;author=X.+Li&amp;author=X.+Han&amp;author=R.+Liu&amp;author=J.+Fang&amp;volume=38&amp;issue=9&amp;publication_year=2017&amp;pages=794-808&amp;pmid=28648527&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib75">75. <span class="element-citation" id="sref75">Gencheva R., Arn&#x000e9;r E.S.J. Thioredoxin reductase inhibition for cancer therapy. <span><span class="ref-journal">Annu. Rev. Pharmacol. Toxicol. </span>2021</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34449246" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Pharmacol.+Toxicol.&amp;title=Thioredoxin+reductase+inhibition+for+cancer+therapy&amp;author=R.+Gencheva&amp;author=E.S.J.+Arn&#x000e9;r&amp;publication_year=2021&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib76">76. <span class="element-citation" id="sref76">Mandal P.K., Schneider M., Kolle P., Kuhlencordt P., Forster H., Beck H., et al. Loss of thioredoxin reductase 1 renders tumors highly susceptible to pharmacologic glutathione deprivation. <span><span class="ref-journal">Cancer Res. </span>2010;<span class="ref-vol">70</span>(22):9505–9514.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21045148" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res.&amp;title=Loss+of+thioredoxin+reductase+1+renders+tumors+highly+susceptible+to+pharmacologic+glutathione+deprivation&amp;author=P.K.+Mandal&amp;author=M.+Schneider&amp;author=P.+Kolle&amp;author=P.+Kuhlencordt&amp;author=H.+Forster&amp;volume=70&amp;issue=22&amp;publication_year=2010&amp;pages=9505-9514&amp;pmid=21045148&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib77">77. <span class="element-citation" id="sref77">Mandal P.K., Seiler A., Perisic T., Kolle P., Banjac Canak A., Forster H., et al. System x(c)- and thioredoxin reductase 1 cooperatively rescue glutathione deficiency. <span><span class="ref-journal">J. Biol. Chem. </span>2010;<span class="ref-vol">285</span>(29):22244–22253.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2903358/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20463017" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=System+x(c)-+and+thioredoxin+reductase+1+cooperatively+rescue+glutathione+deficiency&amp;author=P.K.+Mandal&amp;author=A.+Seiler&amp;author=T.+Perisic&amp;author=P.+Kolle&amp;author=A.+Banjac+Canak&amp;volume=285&amp;issue=29&amp;publication_year=2010&amp;pages=22244-22253&amp;pmid=20463017&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib78">78. <span class="element-citation" id="sref78">Somwar R., Shum D., Djaballah H., Varmus H. Identification and preliminary characterization of novel small molecules that inhibit growth of human lung adenocarcinoma cells. <span><span class="ref-journal">J. Biomol. Screen. </span>2009;<span class="ref-vol">14</span>(10):1176–1184.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3146390/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19887599" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biomol.+Screen&amp;title=Identification+and+preliminary+characterization+of+novel+small+molecules+that+inhibit+growth+of+human+lung+adenocarcinoma+cells&amp;author=R.+Somwar&amp;author=D.+Shum&amp;author=H.+Djaballah&amp;author=H.+Varmus&amp;volume=14&amp;issue=10&amp;publication_year=2009&amp;pages=1176-1184&amp;pmid=19887599&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib79">79. <span class="element-citation" id="sref79">Liu M., Sun S., Meng Y., Wang L., Liu H., Shi W., et al. Benzophenanthridine alkaloid chelerythrine elicits necroptosis of gastric cancer cells via selective conjugation at the redox hyperreactive C-terminal sec(498) residue of cytosolic selenoprotein thioredoxin reductase. <span><span class="ref-journal">Molecules. </span>2023;<span class="ref-vol">28</span>(19)</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10574601/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37836684" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=Benzophenanthridine+alkaloid+chelerythrine+elicits+necroptosis+of+gastric+cancer+cells+via+selective+conjugation+at+the+redox+hyperreactive+C-terminal+sec(498)+residue+of+cytosolic+selenoprotein+thioredoxin+reductase&amp;author=M.+Liu&amp;author=S.+Sun&amp;author=Y.+Meng&amp;author=L.+Wang&amp;author=H.+Liu&amp;volume=28&amp;issue=19&amp;publication_year=2023&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib80">80. <span class="element-citation" id="sref80">Xu Z., Xu J., Sun S., Lin W., Li Y., Lu Q., et al. Mecheliolide elicits ROS-mediated ERS driven immunogenic cell death in hepatocellular carcinoma. <span><span class="ref-journal">Redox Biol. </span>2022;<span class="ref-vol">54</span></span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9168183/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35671636" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Redox+Biol.&amp;title=Mecheliolide+elicits+ROS-mediated+ERS+driven+immunogenic+cell+death+in+hepatocellular+carcinoma&amp;author=Z.+Xu&amp;author=J.+Xu&amp;author=S.+Sun&amp;author=W.+Lin&amp;author=Y.+Li&amp;volume=54&amp;publication_year=2022&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib81">81. <span class="element-citation" id="sref81">Ste Marie E.J., Wehrle R.J., Haupt D.J., Wood N.B., van der Vliet A., Previs M.J., et al. Can selenoenzymes resist electrophilic modification? Evidence from thioredoxin reductase and a mutant containing alpha-methylselenocysteine. <span><span class="ref-journal">Biochemistry. </span>2020;<span class="ref-vol">59</span>(36):3300–3315.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7509987/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32845139" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Can+selenoenzymes+resist+electrophilic+modification?+Evidence+from+thioredoxin+reductase+and+a+mutant+containing+alpha-methylselenocysteine&amp;author=E.J.+Ste+Marie&amp;author=R.J.+Wehrle&amp;author=D.J.+Haupt&amp;author=N.B.+Wood&amp;author=A.+van+der+Vliet&amp;volume=59&amp;issue=36&amp;publication_year=2020&amp;pages=3300-3315&amp;pmid=32845139&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib82">82. <span class="element-citation" id="sref82">Chen X., Sun S., Huang S., Yang H., Ye Q., Lv L., et al. Gold(I) selenium N-heterocyclic carbene complexes as potent antibacterial agents against multidrug-resistant gram-negative bacteria via inhibiting thioredoxin reductase. <span><span class="ref-journal">Redox Biol. </span>2023;<span class="ref-vol">60</span></span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9939723/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36758467" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Redox+Biol.&amp;title=Gold(I)+selenium+N-heterocyclic+carbene+complexes+as+potent+antibacterial+agents+against+multidrug-resistant+gram-negative+bacteria+via+inhibiting+thioredoxin+reductase&amp;author=X.+Chen&amp;author=S.+Sun&amp;author=S.+Huang&amp;author=H.+Yang&amp;author=Q.+Ye&amp;volume=60&amp;publication_year=2023&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib83">83. <span class="element-citation" id="sref83">Sun S., Xu W., Zhou H., Zhang Y., Zhang J., Li X., et al. Efficient purification of selenoprotein thioredoxin reductase 1 by using chelating reagents to protect the affinity resins and rescue the enzyme activities. <span><span class="ref-journal">Process Biochem. </span>2021;<span class="ref-vol">101</span>:256–265.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Process+Biochem.&amp;title=Efficient+purification+of+selenoprotein+thioredoxin+reductase+1+by+using+chelating+reagents+to+protect+the+affinity+resins+and+rescue+the+enzyme+activities&amp;author=S.+Sun&amp;author=W.+Xu&amp;author=H.+Zhou&amp;author=Y.+Zhang&amp;author=J.+Zhang&amp;volume=101&amp;publication_year=2021&amp;pages=256-265&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib84">84. <span class="element-citation" id="sref84">Yang Y., Sun S., Xu W., Zhang Y., Yang R., Ma K., et al. Piperlongumine inhibits thioredoxin reductase 1 by targeting selenocysteine residues and sensitizes cancer cells to erastin. <span><span class="ref-journal">Antioxidants. </span>2022;<span class="ref-vol">11</span>(4):710.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9030593/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35453395" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Antioxidants&amp;title=Piperlongumine+inhibits+thioredoxin+reductase+1+by+targeting+selenocysteine+residues+and+sensitizes+cancer+cells+to+erastin&amp;author=Y.+Yang&amp;author=S.+Sun&amp;author=W.+Xu&amp;author=Y.+Zhang&amp;author=R.+Yang&amp;volume=11&amp;issue=4&amp;publication_year=2022&amp;pages=710&amp;pmid=35453395&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib85">85. <span class="element-citation" id="sref85">Sun S., Zhang Y., Xu W., Zhang Y., Yang R., Guo J., et al. Chlorophyllin inhibits mammalian thioredoxin reductase 1 and triggers cancer cell death. <span><span class="ref-journal">Antioxidants. </span>2021;<span class="ref-vol">10</span>(11):1733.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8615155/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34829604" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Antioxidants&amp;title=Chlorophyllin+inhibits+mammalian+thioredoxin+reductase+1+and+triggers+cancer+cell+death&amp;author=S.+Sun&amp;author=Y.+Zhang&amp;author=W.+Xu&amp;author=Y.+Zhang&amp;author=R.+Yang&amp;volume=10&amp;issue=11&amp;publication_year=2021&amp;pages=1733&amp;pmid=34829604&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib86">86. <span class="element-citation" id="sref86">Wang H., Sun S., Ren Y., Yang R., Guo J., Zong Y., et al. Selenite ameliorates cadmium-induced cytotoxicity through downregulation of ROS levels and upregulation of selenoprotein thioredoxin reductase 1 in SH-SY5Y cells. <span><span class="ref-journal">Biol. Trace Elem. Res. </span>2023;<span class="ref-vol">201</span>(1):139–148.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35066751" ref="reftype=pubmed&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol.+Trace+Elem.+Res.&amp;title=Selenite+ameliorates+cadmium-induced+cytotoxicity+through+downregulation+of+ROS+levels+and+upregulation+of+selenoprotein+thioredoxin+reductase+1+in+SH-SY5Y+cells&amp;author=H.+Wang&amp;author=S.+Sun&amp;author=Y.+Ren&amp;author=R.+Yang&amp;author=J.+Guo&amp;volume=201&amp;issue=1&amp;publication_year=2023&amp;pages=139-148&amp;pmid=35066751&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10840350&amp;issue-id=454190&amp;journal-id=2171&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Redox Biology</span> are provided here courtesy of <strong>Elsevier</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC10840350/pdf/main.pdf" class="int-view">PDF (14M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/10840350/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/10840350/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="10840350" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10840350%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="xdoECR4EeVatmgZis07RjHAnqKFrEfHGObV8qVWUDKh7gflGDquLIz1BrTXqHOZB">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10840350%2F&amp;text=Guiding%20bar%20motif%20of%20thioredoxin%20reductase%201%20modulates%20enzymatic%20activity%20and%20inhibitor%20binding%20by%20communicating%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10840350%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10840350/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC10840350/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38277963/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC10840350/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38277963/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC10840350/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/10840350/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC10840350/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
