
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Crystal Structure of Malaria Parasite Nucleosome Assembly
 Protein - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="D0BD41E4AC192A4500002D79C9E7698F.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="jbc" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC2665062/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2665062/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="The Journal of Biological Chemistry" /><meta name="citation_title" content="Crystal Structure of Malaria Parasite Nucleosome Assembly&#10; Protein: DISTINCT MODES OF PROTEIN LOCALIZATION AND HISTONE&#10; RECOGNITION*&#x00053;&#x020de;" /><meta name="citation_author" content="Jasmita Gill" /><meta name="citation_author_institution" content="Structural and Computational Biology Group, International Centre for Genetic Engineering and Biotechnology, Aruna Asaf Ali Road, New Delhi 110067, India, the European Molecular Biology Laboratory, 6 Rue Jules Horowitz, BP 181, F-38042 Grenoble C&#x000e9;dex 9, France, the Department of Biotechnology, Hamdard University, Hamdard Nagar, New Delhi 110062, India, and the Walter and Eliza Hall Institute of Medical Research, 1G Royal Parade, Parkville, Victoria 3050, Australia" /><meta name="citation_author" content="Manickam Yogavel" /><meta name="citation_author_institution" content="Structural and Computational Biology Group, International Centre for Genetic Engineering and Biotechnology, Aruna Asaf Ali Road, New Delhi 110067, India, the European Molecular Biology Laboratory, 6 Rue Jules Horowitz, BP 181, F-38042 Grenoble C&#x000e9;dex 9, France, the Department of Biotechnology, Hamdard University, Hamdard Nagar, New Delhi 110062, India, and the Walter and Eliza Hall Institute of Medical Research, 1G Royal Parade, Parkville, Victoria 3050, Australia" /><meta name="citation_author" content="Anuj Kumar" /><meta name="citation_author_institution" content="Structural and Computational Biology Group, International Centre for Genetic Engineering and Biotechnology, Aruna Asaf Ali Road, New Delhi 110067, India, the European Molecular Biology Laboratory, 6 Rue Jules Horowitz, BP 181, F-38042 Grenoble C&#x000e9;dex 9, France, the Department of Biotechnology, Hamdard University, Hamdard Nagar, New Delhi 110062, India, and the Walter and Eliza Hall Institute of Medical Research, 1G Royal Parade, Parkville, Victoria 3050, Australia" /><meta name="citation_author" content="Hassan Belrhali" /><meta name="citation_author_institution" content="Structural and Computational Biology Group, International Centre for Genetic Engineering and Biotechnology, Aruna Asaf Ali Road, New Delhi 110067, India, the European Molecular Biology Laboratory, 6 Rue Jules Horowitz, BP 181, F-38042 Grenoble C&#x000e9;dex 9, France, the Department of Biotechnology, Hamdard University, Hamdard Nagar, New Delhi 110062, India, and the Walter and Eliza Hall Institute of Medical Research, 1G Royal Parade, Parkville, Victoria 3050, Australia" /><meta name="citation_author" content="S. K. Jain" /><meta name="citation_author_institution" content="Structural and Computational Biology Group, International Centre for Genetic Engineering and Biotechnology, Aruna Asaf Ali Road, New Delhi 110067, India, the European Molecular Biology Laboratory, 6 Rue Jules Horowitz, BP 181, F-38042 Grenoble C&#x000e9;dex 9, France, the Department of Biotechnology, Hamdard University, Hamdard Nagar, New Delhi 110062, India, and the Walter and Eliza Hall Institute of Medical Research, 1G Royal Parade, Parkville, Victoria 3050, Australia" /><meta name="citation_author" content="Melanie Rug" /><meta name="citation_author_institution" content="Structural and Computational Biology Group, International Centre for Genetic Engineering and Biotechnology, Aruna Asaf Ali Road, New Delhi 110067, India, the European Molecular Biology Laboratory, 6 Rue Jules Horowitz, BP 181, F-38042 Grenoble C&#x000e9;dex 9, France, the Department of Biotechnology, Hamdard University, Hamdard Nagar, New Delhi 110062, India, and the Walter and Eliza Hall Institute of Medical Research, 1G Royal Parade, Parkville, Victoria 3050, Australia" /><meta name="citation_author" content="Monica Brown" /><meta name="citation_author_institution" content="Structural and Computational Biology Group, International Centre for Genetic Engineering and Biotechnology, Aruna Asaf Ali Road, New Delhi 110067, India, the European Molecular Biology Laboratory, 6 Rue Jules Horowitz, BP 181, F-38042 Grenoble C&#x000e9;dex 9, France, the Department of Biotechnology, Hamdard University, Hamdard Nagar, New Delhi 110062, India, and the Walter and Eliza Hall Institute of Medical Research, 1G Royal Parade, Parkville, Victoria 3050, Australia" /><meta name="citation_author" content="Alexander G. Maier" /><meta name="citation_author_institution" content="Structural and Computational Biology Group, International Centre for Genetic Engineering and Biotechnology, Aruna Asaf Ali Road, New Delhi 110067, India, the European Molecular Biology Laboratory, 6 Rue Jules Horowitz, BP 181, F-38042 Grenoble C&#x000e9;dex 9, France, the Department of Biotechnology, Hamdard University, Hamdard Nagar, New Delhi 110062, India, and the Walter and Eliza Hall Institute of Medical Research, 1G Royal Parade, Parkville, Victoria 3050, Australia" /><meta name="citation_author" content="Amit Sharma" /><meta name="citation_author_institution" content="Structural and Computational Biology Group, International Centre for Genetic Engineering and Biotechnology, Aruna Asaf Ali Road, New Delhi 110067, India, the European Molecular Biology Laboratory, 6 Rue Jules Horowitz, BP 181, F-38042 Grenoble C&#x000e9;dex 9, France, the Department of Biotechnology, Hamdard University, Hamdard Nagar, New Delhi 110062, India, and the Walter and Eliza Hall Institute of Medical Research, 1G Royal Parade, Parkville, Victoria 3050, Australia" /><meta name="citation_publication_date" content="2009/04/04" /><meta name="citation_issue" content="15" /><meta name="citation_volume" content="284" /><meta name="citation_firstpage" content="10076" /><meta name="citation_doi" content="10.1074/jbc.M808633200" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2665062/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC2665062/" /><meta name="citation_pmid" content="19176479" /><meta name="DC.Title" content="Crystal Structure of Malaria Parasite Nucleosome Assembly&#10; Protein: DISTINCT MODES OF PROTEIN LOCALIZATION AND HISTONE&#10; RECOGNITION*&#x00053;&#x020de;" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="American Society for Biochemistry and Molecular Biology" /><meta name="DC.Contributor" content="Jasmita Gill" /><meta name="DC.Contributor" content="Manickam Yogavel" /><meta name="DC.Contributor" content="Anuj Kumar" /><meta name="DC.Contributor" content="Hassan Belrhali" /><meta name="DC.Contributor" content="S. K. Jain" /><meta name="DC.Contributor" content="Melanie Rug" /><meta name="DC.Contributor" content="Monica Brown" /><meta name="DC.Contributor" content="Alexander G. Maier" /><meta name="DC.Contributor" content="Amit Sharma" /><meta name="DC.Date" content="2009 Apr 10" /><meta name="DC.Identifier" content="10.1074/jbc.M808633200" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Crystal Structure of Malaria Parasite Nucleosome Assembly&#10; Protein: DISTINCT MODES OF PROTEIN LOCALIZATION AND HISTONE&#10; RECOGNITION*&#x00053;&#x020de;" /><meta property="og:type" content="article" /><meta property="og:description" content="Nucleosome assembly proteins (NAPs) are histone chaperones that are&#10; essential for the transfer and incorporation of histones into nucleosomes.&#10; NAPs participate in assembly and disassembly of nucleosomes and in chromatin&#10; structure organization. Human ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2665062/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC2665062/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/567/">J Biol Chem</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/177745/">v.284(15); 2009 Apr 10</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC2665062
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC2665062/pdf/10076.pdf" class="int-view">PDF (1.2M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/2665062/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/2665062/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="2665062" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2665062%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="usYZED0HvOvfKNEyWc3t8ek47YsEs3PqrtxquFfjPlAnaYSwPPH8HLOHTpP0rfVd">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2665062%2F&amp;text=Crystal%20Structure%20of%20Malaria%20Parasite%20Nucleosome%20Assembly%0A%20Protein%3A%20DISTINCT%20MODES%20OF%20PROTEIN%20LOCALIZATION%20AND%20HISTONE%0A%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2665062%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2665062/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC2665062/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/19176479/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC2665062/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/19176479/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC2665062/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/2665062/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/567/">J Biol Chem</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/177745/">v.284(15); 2009 Apr 10</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC2665062
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-jbc.png" alt="Logo of jbc" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="0,0,500,75" alt="The Journal of Biological Chemistry" title="The Journal of Biological Chemistry" href="http://www.jbc.org" ref="reftype=publisher&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">J Biol Chem.</span> 2009 Apr 10; 284(15): 10076–10087. </div><div class="part2">  <span class="doi"><span>doi: </span><a href="//doi.org/10.1074%2Fjbc.M808633200" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1074/jbc.M808633200</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2665062</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/19176479">19176479</a></div></div></div><h1 class="content-title">Crystal Structure of Malaria Parasite Nucleosome Assembly
 Protein</h1><div class="fm-subtitle"><em>DISTINCT MODES OF PROTEIN LOCALIZATION AND HISTONE
 RECOGNITION</em><sup><a href="#fn1" rid="fn1" class=" fn">*</a></sup><sup><a href="#fn3" rid="fn3" class=" fn">&#x00053;&#x020de;</a></sup></div><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gill%20J%5BAuthor%5D" class="affpopup" co-rid="_co_idm140713620783840" co-class="co-affbox">Jasmita Gill</a>,<sup>&#x02021;,</sup><sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Yogavel%20M%5BAuthor%5D" class="affpopup" co-rid="_co_idm140713609241296" co-class="co-affbox">Manickam Yogavel</a>,<sup>&#x02021;,</sup><sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kumar%20A%5BAuthor%5D" class="affpopup" co-rid="_co_idm140713619471472" co-class="co-affbox">Anuj Kumar</a>,<sup>&#x02021;,</sup><sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Belrhali%20H%5BAuthor%5D" class="affpopup" co-rid="_co_idm140713617178400" co-class="co-affbox">Hassan Belrhali</a>,<sup>&#x000a7;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Jain%20SK%5BAuthor%5D" class="affpopup" co-rid="_co_idm140713623805440" co-class="co-affbox">S. K. Jain</a>,<sup>&#x000b6;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Rug%20M%5BAuthor%5D" class="affpopup" co-rid="_co_idm140713626059376" co-class="co-affbox">Melanie Rug</a>,<sup>&#x02225;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Brown%20M%5BAuthor%5D" class="affpopup" co-rid="_co_idm140713613988928" co-class="co-affbox">Monica Brown</a>,<sup>&#x02225;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Maier%20AG%5BAuthor%5D" class="affpopup" co-rid="_co_idm140713658778592" co-class="co-affbox">Alexander G. Maier</a>,<sup>&#x02225;,</sup><sup>3</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Sharma%20A%5BAuthor%5D" class="affpopup" co-rid="_co_idm140713658843936" co-class="co-affbox">Amit Sharma</a><sup>&#x02021;,</sup><sup>4</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140713620783840"><h3 class="no_margin">Jasmita Gill</h3><p><sup>&#x02021;</sup>Structural and Computational
 Biology Group, International Centre for Genetic Engineering and Biotechnology,
 Aruna Asaf Ali Road, New Delhi 110067, India, the
 <sup>&#x000a7;</sup>European Molecular Biology
 Laboratory, 6 Rue Jules Horowitz, BP 181, F-38042 Grenoble C&#x000e9;dex 9,
 France, the <sup>&#x000b6;</sup>Department of
 Biotechnology, Hamdard University, Hamdard Nagar, New Delhi 110062, India, and
 the <sup>&#x02225;</sup>Walter and Eliza Hall Institute of
 Medical Research, 1G Royal Parade, Parkville, Victoria 3050, Australia</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gill%20J%5BAuthor%5D">Jasmita Gill</a></div></div><div id="_co_idm140713609241296"><h3 class="no_margin">Manickam Yogavel</h3><p><sup>&#x02021;</sup>Structural and Computational
 Biology Group, International Centre for Genetic Engineering and Biotechnology,
 Aruna Asaf Ali Road, New Delhi 110067, India, the
 <sup>&#x000a7;</sup>European Molecular Biology
 Laboratory, 6 Rue Jules Horowitz, BP 181, F-38042 Grenoble C&#x000e9;dex 9,
 France, the <sup>&#x000b6;</sup>Department of
 Biotechnology, Hamdard University, Hamdard Nagar, New Delhi 110062, India, and
 the <sup>&#x02225;</sup>Walter and Eliza Hall Institute of
 Medical Research, 1G Royal Parade, Parkville, Victoria 3050, Australia</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Yogavel%20M%5BAuthor%5D">Manickam Yogavel</a></div></div><div id="_co_idm140713619471472"><h3 class="no_margin">Anuj Kumar</h3><p><sup>&#x02021;</sup>Structural and Computational
 Biology Group, International Centre for Genetic Engineering and Biotechnology,
 Aruna Asaf Ali Road, New Delhi 110067, India, the
 <sup>&#x000a7;</sup>European Molecular Biology
 Laboratory, 6 Rue Jules Horowitz, BP 181, F-38042 Grenoble C&#x000e9;dex 9,
 France, the <sup>&#x000b6;</sup>Department of
 Biotechnology, Hamdard University, Hamdard Nagar, New Delhi 110062, India, and
 the <sup>&#x02225;</sup>Walter and Eliza Hall Institute of
 Medical Research, 1G Royal Parade, Parkville, Victoria 3050, Australia</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kumar%20A%5BAuthor%5D">Anuj Kumar</a></div></div><div id="_co_idm140713617178400"><h3 class="no_margin">Hassan Belrhali</h3><p><sup>&#x02021;</sup>Structural and Computational
 Biology Group, International Centre for Genetic Engineering and Biotechnology,
 Aruna Asaf Ali Road, New Delhi 110067, India, the
 <sup>&#x000a7;</sup>European Molecular Biology
 Laboratory, 6 Rue Jules Horowitz, BP 181, F-38042 Grenoble C&#x000e9;dex 9,
 France, the <sup>&#x000b6;</sup>Department of
 Biotechnology, Hamdard University, Hamdard Nagar, New Delhi 110062, India, and
 the <sup>&#x02225;</sup>Walter and Eliza Hall Institute of
 Medical Research, 1G Royal Parade, Parkville, Victoria 3050, Australia</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Belrhali%20H%5BAuthor%5D">Hassan Belrhali</a></div></div><div id="_co_idm140713623805440"><h3 class="no_margin">S. K. Jain</h3><p><sup>&#x02021;</sup>Structural and Computational
 Biology Group, International Centre for Genetic Engineering and Biotechnology,
 Aruna Asaf Ali Road, New Delhi 110067, India, the
 <sup>&#x000a7;</sup>European Molecular Biology
 Laboratory, 6 Rue Jules Horowitz, BP 181, F-38042 Grenoble C&#x000e9;dex 9,
 France, the <sup>&#x000b6;</sup>Department of
 Biotechnology, Hamdard University, Hamdard Nagar, New Delhi 110062, India, and
 the <sup>&#x02225;</sup>Walter and Eliza Hall Institute of
 Medical Research, 1G Royal Parade, Parkville, Victoria 3050, Australia</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Jain%20SK%5BAuthor%5D">S. K. Jain</a></div></div><div id="_co_idm140713626059376"><h3 class="no_margin">Melanie Rug</h3><p><sup>&#x02021;</sup>Structural and Computational
 Biology Group, International Centre for Genetic Engineering and Biotechnology,
 Aruna Asaf Ali Road, New Delhi 110067, India, the
 <sup>&#x000a7;</sup>European Molecular Biology
 Laboratory, 6 Rue Jules Horowitz, BP 181, F-38042 Grenoble C&#x000e9;dex 9,
 France, the <sup>&#x000b6;</sup>Department of
 Biotechnology, Hamdard University, Hamdard Nagar, New Delhi 110062, India, and
 the <sup>&#x02225;</sup>Walter and Eliza Hall Institute of
 Medical Research, 1G Royal Parade, Parkville, Victoria 3050, Australia</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Rug%20M%5BAuthor%5D">Melanie Rug</a></div></div><div id="_co_idm140713613988928"><h3 class="no_margin">Monica Brown</h3><p><sup>&#x02021;</sup>Structural and Computational
 Biology Group, International Centre for Genetic Engineering and Biotechnology,
 Aruna Asaf Ali Road, New Delhi 110067, India, the
 <sup>&#x000a7;</sup>European Molecular Biology
 Laboratory, 6 Rue Jules Horowitz, BP 181, F-38042 Grenoble C&#x000e9;dex 9,
 France, the <sup>&#x000b6;</sup>Department of
 Biotechnology, Hamdard University, Hamdard Nagar, New Delhi 110062, India, and
 the <sup>&#x02225;</sup>Walter and Eliza Hall Institute of
 Medical Research, 1G Royal Parade, Parkville, Victoria 3050, Australia</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Brown%20M%5BAuthor%5D">Monica Brown</a></div></div><div id="_co_idm140713658778592"><h3 class="no_margin">Alexander G. Maier</h3><p><sup>&#x02021;</sup>Structural and Computational
 Biology Group, International Centre for Genetic Engineering and Biotechnology,
 Aruna Asaf Ali Road, New Delhi 110067, India, the
 <sup>&#x000a7;</sup>European Molecular Biology
 Laboratory, 6 Rue Jules Horowitz, BP 181, F-38042 Grenoble C&#x000e9;dex 9,
 France, the <sup>&#x000b6;</sup>Department of
 Biotechnology, Hamdard University, Hamdard Nagar, New Delhi 110062, India, and
 the <sup>&#x02225;</sup>Walter and Eliza Hall Institute of
 Medical Research, 1G Royal Parade, Parkville, Victoria 3050, Australia</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Maier%20AG%5BAuthor%5D">Alexander G. Maier</a></div></div><div id="_co_idm140713658843936"><h3 class="no_margin">Amit Sharma</h3><p><sup>&#x02021;</sup>Structural and Computational
 Biology Group, International Centre for Genetic Engineering and Biotechnology,
 Aruna Asaf Ali Road, New Delhi 110067, India, the
 <sup>&#x000a7;</sup>European Molecular Biology
 Laboratory, 6 Rue Jules Horowitz, BP 181, F-38042 Grenoble C&#x000e9;dex 9,
 France, the <sup>&#x000b6;</sup>Department of
 Biotechnology, Hamdard University, Hamdard Nagar, New Delhi 110062, India, and
 the <sup>&#x02225;</sup>Walter and Eliza Hall Institute of
 Medical Research, 1G Royal Parade, Parkville, Victoria 3050, Australia</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Sharma%20A%5BAuthor%5D">Amit Sharma</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140713615211232_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140713615211232_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140713615211232_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140713615211232_ai" style="display:none"><div class="fm-affl" id="N0x1ca9390N0x43159b8"><sup>&#x02021;</sup>Structural and Computational
 Biology Group, International Centre for Genetic Engineering and Biotechnology,
 Aruna Asaf Ali Road, New Delhi 110067, India, the
 <sup>&#x000a7;</sup>European Molecular Biology
 Laboratory, 6 Rue Jules Horowitz, BP 181, F-38042 Grenoble C&#x000e9;dex 9,
 France, the <sup>&#x000b6;</sup>Department of
 Biotechnology, Hamdard University, Hamdard Nagar, New Delhi 110062, India, and
 the <sup>&#x02225;</sup>Walter and Eliza Hall Institute of
 Medical Research, 1G Royal Parade, Parkville, Victoria 3050, Australia</div><div id="fn4"><sup>1</sup>Supported by the Wellcome
 Trust.</div><div id="fn5"><sup>2</sup>Supported by a Department of Biotechnology, Government of
 India, fellowship.</div><div id="fn6"><sup>3</sup>An Australian Research Council Research Fellow.</div><div id="cor1"><sup>4</sup>
 An International Wellcome Trust Senior Research Fellow in Biomedical Sciences.
 To whom correspondence should be addressed. Tel./Fax: 911126741731; E-mail:
 <a href="mailto:dev@null" data-email="moc.liamg@begci.tima" class="oemail">moc.liamg@begci.tima</a>.
 </div></div><div class="fm-article-notes hide half_rhythm" id="idm140713615211232_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2008 Nov 13; Revised 2009 Jan 9</div></div><div class="permissions half_rhythm hide" id="idm140713615211232_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  &#x000a9; 2009, The American Society for Biochemistry and
 Molecular Biology, Inc.</div><div class="license half_rhythm"><p><em>Author's Choice</em></p><p><a href="http://creativecommons.org/licenses/by-nc/3.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">Creative
 Commons Attribution Non-Commercial License</a> applies to Author
 Choice Articles</p></div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
[Supplemental Data]
</div>

<div><a href="/pmc/articles/PMC2665062/bin/M808633200_index.html" data-ga-action="click_feat_suppl">M808633200_index.html</a><span style="color:gray"> (1.1K)</span></div><div class="small guid">GUID: 2540C80F-2B32-4929-9553-120DE958F17B</div>
<div><a href="/pmc/articles/PMC2665062/bin/M808633200_1.pdf" data-ga-action="click_feat_suppl">M808633200_1.pdf</a><span style="color:gray"> (611K)</span></div><div class="small guid">GUID: 16191EA8-C599-4BB5-885E-D135F2E27822</div>
</div></dd></dl></div><div id="abstract-a.o.b.q" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.o.b.qtitle">Abstract</h2><!--article-meta--><div><p class="p p-first-last">Nucleosome assembly proteins (NAPs) are histone chaperones that are
 essential for the transfer and incorporation of histones into nucleosomes.
 NAPs participate in assembly and disassembly of nucleosomes and in chromatin
 structure organization. Human malaria parasite <em>Plasmodium falciparum</em>
 contains two nucleosome assembly proteins termed PfNapL and PfNapS. To gain
 structural insights into the mechanism of NAPs, we have determined and
 analyzed the crystal structure of PfNapL at 2.3 &#x000c5; resolution. PfNapL, an
 ortholog of eukaryotic NAPs, is dimeric in nature and adopts a characteristic
 fold seen previously for yeast NAP-1 and Vps75 and for human SET/TAF-1b
 (&#x003b2;)/INHAT. The PfNapL monomer is comprised of domain I, containing a
 dimerization &#x003b1;-helix, and a domain II, composed of &#x003b1;-helices and a
 &#x003b2;-subdomain. Structural comparisons reveal that the &#x0201c;accessory
 domain,&#x0201d; which is inserted between the domain I and domain II in yeast
 NAP-1 and other eukaryotic NAPs, is surprisingly absent in PfNapL. Expression
 of green fluorescent protein-tagged PfNapL confirmed its exclusive
 localization to the parasite cytoplasm. Attempts to disrupt the PfNapL gene
 were not successful, indicating its essential role for the malaria parasite. A
 detailed analysis of PfNapL structure suggests unique histone binding
 properties. The crucial structural differences observed between parasite and
 yeast NAPs shed light on possible new modes of histone recognition by
 nucleosome assembly proteins.</p></div></div><div id="body-a.p" class="tsec sec"><h2 class="headless nomenu"></h2><p class="p p-first">Chromatin assembly and remodeling are vital cellular processes that are
 pivotal during replication, transcription, recombination, and repair in
 eukaryotic cells. Nucleosomes are the fundamental repeating subunits of all
 eukaryotic chromatin (except when packaged in sperm). Nucleosomes are made up
 of DNA and four pairs of histone proteins, which together resemble
 &#x0201c;beads on a string&#x0201d; when observed with an electron microscope.
 Histones H2A, H2B, H3, and H4 are part of the nucleosome, whereas histone H1
 sits on top of the structure, keeping in place the DNA that is wrapped around
 the other histones (<a href="#ref1" rid="ref1" class=" bibr popnode">1</a>).
 Nucleosome assembly is a fundamental biological process that is required for
 the replication and maintenance of chromatin in the eukaryotic nucleus. In
 dividing cells, newly synthesized DNA is assembled rapidly into chromatin by a
 process that appears to involve an indirect coupling between DNA replication
 and nucleosome assembly. The assembly of nucleosome occurs in a two-step
 process: a tetramer of H3 and H4 is first deposited onto DNA, and then the
 nucleosome core is completed by the addition of two heterodimers of H2A and
 H2B (<a href="#ref1" rid="ref1" class=" bibr popnode">1</a>,
 <a href="#ref2" rid="ref2" class=" bibr popnode">2</a>). Histones are highly basic
 proteins and require chaperones that promote their proper interaction to form
 chromatin. Histone chaperones also prevent nonspecific aggregation of DNA with
 histones. A number of histone chaperones (<em>e.g.</em> nucleosome assembly
 protein 1 (NAP-1),<sup><a href="#fn7" rid="fn7" class=" fn">5</a></sup>
 nucleoplasmin, Asf1 (anti-silencing function), CAF-1 (chromatin assembly
 factor), and HirA (histone regulator A)) are involved in nucleosome/chromatin
 assembly/disassembly (<a href="#ref3" rid="ref3" class=" bibr popnode">3</a>).</p><p>Some histone chaperones, such as nucleoplasmin and NAP-1, exhibit a
 preference for binding to histones H2A and H2B relative to histones H3 and H4
 (<a href="#ref4" rid="ref4" class=" bibr popnode">4</a>,
 <a href="#ref5" rid="ref5" class=" bibr popnode">5</a>). Other histone chaperones,
 which include CAF-1, N1/N2, and Spt6, associate preferentially with H3 and H4.
 Interestingly, it has also been observed that newly synthesized histones are
 acetylated (such as at positions 5, 8, and 12 of histone H4) and then
 subsequently deacetylated after assembly into chromatin
 (<a href="#ref6" rid="ref6" class=" bibr popnode">6</a>,
 <a href="#ref7" rid="ref7" class=" bibr popnode">7</a>). Thus, factors that mediate
 histone acetylation or deacetylation may participate, perhaps indirectly by
 the covalent modification of histones, in the chromatin assembly process.
 Homologs of yeast NAP-1 are conserved among all eukaryotes, and yeast NAP-1 is
 well characterized for its role in chromatin assembly
 (<a href="#ref3" rid="ref3" class=" bibr popnode">3</a>,
 <a href="#ref8" rid="ref8" class=" bibr popnode">8</a>). Gene knock-out phenotypes
 in several studied organisms indicate essentiality of the nucleosome assembly
 protein family; gene ablations in mouse and <em>Drosophila</em> cause
 embryonic lethality, whereas in yeast, cells exhibit growth defects
 (<a href="#ref9" rid="ref9" class=" bibr popnode">9</a>-<a href="#ref11" rid="ref11" class=" bibr popnode">11</a>).</p><p class="p">Malaria is one of the most common infectious diseases and remains an
 enormous public health problem. Malaria is caused by protozoan parasites of
 the genus <em>Plasmodium</em>, and the most serious forms of the disease are
 caused by <em>Plasmodium falciparum</em>
 (<a href="#ref12" rid="ref12" class=" bibr popnode">12</a>). It is therefore crucial
 to identify, dissect, and exploit the molecular motors of malaria parasites,
 which can serve as essential targets for antimalarials. The human malaria
 parasite <em>P. falciparum</em> contains two nucleosome assembly proteins,
 which we have termed PfNapL and PfNapS and which are orthologs of eukaryotic
 NAPs (<a href="#ref13" rid="ref13" class=" bibr popnode">13</a>). We have shown that
 both PfNapL and PfNapS are present in all erythrocytic stages of the parasite
 (<a href="#ref13" rid="ref13" class=" bibr popnode">13</a>). PfNapL forms complexes
 with both histone tetramer and octamer and is predominantly localized in the
 cytoplasm in the asexual and sexual stages of the parasite
 (<a href="#ref13" rid="ref13" class=" bibr popnode">13</a>,
 <a href="#ref14" rid="ref14" class=" bibr popnode">14</a>). PfNapL by itself is
 unable to deposit the histones onto DNA, but it can interact with both core
 and linker histones and is involved in histone binding, shuttling, and
 transfer/release, as shown earlier
 (<a href="#ref13" rid="ref13" class=" bibr popnode">13</a>,
 <a href="#ref14" rid="ref14" class=" bibr popnode">14</a>)
 (<a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1<em>f</em></span></a> reproduced
 here from Ref. <a href="#ref14" rid="ref14" class=" bibr popnode">14</a>). The
 histone binding characteristics of PfNapL have been detailed previously,
 including its ability to transfer cytoplasmic histones on to PfNapS (which is
 localized both in cytoplasm and the nucleus and transfers histones into the
 nucleus for deposition) (<a href="#ref14" rid="ref14" class=" bibr popnode">14</a>).
 A model for relay of histones from parasite cytoplasm to the nucleus has also
 been proposed by us previously (see Ref.
 <a href="#ref14" rid="ref14" class=" bibr popnode">14</a> and specifically
 <a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1<em>f</em></span></a>). PfNapL
 preferentially interacts with the H3-H4 tetramer histones over H2A and H2B
 histones. PfNapL and PfNapS do not interact with each other
 (<a href="#ref14" rid="ref14" class=" bibr popnode">14</a>). To address the
 structural basis of the nucleosome assembly activity in <em>P.
 falciparum</em>, we have determined and analyzed the crystal structure of
 PfNapL. Here, we detail the PfNapL structure and compare it with histone
 chaperones like yeast NAP-1 (yNAP-1), human SET/TAF-1b (&#x003b2;)/INHAT (hSET),
 Vps75, and Asf1-histone complex
 (<a href="#ref15" rid="ref15" class=" bibr popnode">15</a>-<a href="#ref21" rid="ref21" class=" bibr popnode">21</a>)
 in order to provide new insights into the mechanism of histone
 recognition.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140713615292336"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2665062_zbc017097122001a.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is zbc017097122001a.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2665062/bin/zbc017097122001a.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140713615292336"><a target="object" rel="noopener" href="/pmc/articles/PMC2665062/figure/fig1/?report=objectonly">Open in a separate window</a></div><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140713615291248"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2665062_zbc017097122001f.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is zbc017097122001f.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2665062/bin/zbc017097122001f.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140713615291248"><a target="object" rel="noopener" href="/pmc/articles/PMC2665062/figure/fig1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">FIGURE 1.</a></div><!--caption a7--><div class="caption"><p><strong>The overall structure of PfNapL.</strong> <em>a</em>, domain diagram for the
 NAP/SET family of proteins. Histone chaperone proteins comprise a central
 domain and low complexity N and C termini. The inserted accessory domain in
 yNAP-1 (yellow) is highlighted, which is missing both in PfNapL and hSET (set
 to scale). The <em>brown bar</em> is not set to scale, since the N and C
 termini are variable. <em>b</em>, structure of the PfNapL dimer generated
 using 2-fold crystallographic symmetry operation. Each monomer of PfNapL
 contains a domain I composed of dimerization helix &#x003b1;2 and domain II
 composed of a &#x003b2; subdomain (four antiparallel &#x003b2; strands) and
 &#x003b1;-helices on the other side. <em>c</em>, structure-based sequence
 alignment of PfNapL, yNAP-1, and hSET. The structures share high structural
 homology, except for an additional accessory domain between domain I and
 domain II in yNAP-1 (<em>foreground colored yellow</em>). Residues
 contributing to dimer formation are <em>colored green</em>. Conserved
 hydrophobic motifs, predicted NES, and predicted NLS in PfNapL are
 <em>underlined</em> in <em>purple</em> and <em>blue</em>, respectively.
 Conserved residues from the hSET mutagenesis data are <em>colored red</em> and
 <em>blue. d</em>, sequence alignment of PfNapL and other eukaryotic members of
 the NAP family: yNAP-1 (<em>Saccharomyces cerevisiae</em>), xNAP-1
 (<em>Xenopus laevis</em>), dNAP-1 (<em>Drosophila melanogaster</em>), and
 hNAP-1 (<em>Homo sapiens</em>). The accessory domain in yNAP-1 is
 <em>foreground-colored yellow</em>. The identical/conserved surface residues
 and conserved hydrophobic motifs are <em>colored pink</em> and
 <em>purple</em>, respectively. <em>e</em>, phylogenetic tree of NAPs from
 various species showing greater evolutionary distance of malaria parasite NAPs
 from homologs in yeast and humans (indicated by <em>red arrows</em>).
 <em>f</em>, model for <em>in vitro</em> relay of histones by <em>P.
 falciparum</em> nucleosome assembly proteins. This model was reproduced from
 Ref. <a href="#ref14" rid="ref14" class=" bibr popnode">14</a>, showing distinct
 roles for PfNapL (in cytoplasm as a histone carrier) and PfNapS (in nucleus
 capable of histone deposition). PfNapL (<em>L</em>) interacts with histones
 and can deliver histones to PfNapS (<em>S</em>; phosphorylated or
 unphosphorylated) readily. PfNapL upon phosphorylation
 (<em>L<sup>p</sup></em>) binds 3-fold better to the histones when compared
 with the unphosphorylated form of PfNapL. Phospho-PfNapL may deliver its
 histone cargo PfNapS. The latter takes over histones from
 Phospho-PfNapL-histone complexes and shuttles the histones into nucleus.</p></div></div></div></div><div id="sec-a.p.d" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.p.dtitle">EXPERIMENTAL PROCEDURES</h2><p class="p p-first"><em>Protein Crystallization and Data Collection</em>&#x02014;PfNapL crystals
 were obtained at 20 &#x000b0;C by the hanging drop vapor diffusion method using l
 &#x003bc;l of full-length purified PfNapL protein
 (<a href="#ref13" rid="ref13" class=" bibr popnode">13</a>,
 <a href="#ref14" rid="ref14" class=" bibr popnode">14</a>) (3 mg ml<sup>-1</sup> in
 buffer containing 25 m<span class="small-caps">m</span> Tris and 25 m<span class="small-caps">m</span> NaCl) and l &#x003bc;l
 of 0.2 <span class="small-caps">m</span> MgCl<sub>2</sub> with 20% polyethylene glycol 3350 (mother
 liquor). A single crystal was transferred to cryoprotectant containing a
 higher concentration of mother liquor (30% polyethylene glycol 3350 and 0.3
 <span class="small-caps">m</span> MgCl<sub>2</sub>) supplemented with 0.5 <span class="small-caps">m</span> NaI for 30 s
 and flash-frozen under a stream of nitrogen gas at 100 K. X-ray diffraction
 data were collected on an in-house rotating anode MICRO MAX 007 x-ray
 generator (wavelength of 1.54 &#x000c5;; Rigaku/MSC) operated at 40 kV and 20 mA
 with Osmic mirrors (Vari Max HR). The images were recorded using an MAR345dtb
 imaging plate, and the iodide-single anomalous dispersion (SAD) data set was
 collected to 3.0 &#x000c5; resolution. The iodide-soaked crystals of PfNapL
 belong to the monoclinic space group C2 with cell dimensions of <em>a</em> =
 76.68 &#x000c5;, <em>b</em> = 37.83 &#x000c5;, <em>c</em> = 79.31 &#x000c5;, and
 &#x003b2; = 99.83&#x000b0;, having one monomer per asymmetric unit. High resolution
 native diffraction data to 2.3 &#x000c5; resolution were collected at the BM14
 beamline (European Synchotron Radiation Facility, Grenoble, France). These
 crystals also belong to the monoclinic space group C2 with different cell
 dimensions of <em>a</em> = 92.41 &#x000c5;, <em>b</em> = 38.12 &#x000c5;,
 <em>c</em> = 80.05 &#x000c5;, and &#x003b2; = 107.87&#x000b0;. The diffraction images
 were processed and scaled with the HKL2000 suite
 (<a href="#ref22" rid="ref22" class=" bibr popnode">22</a>).</p><p><em>Phasing, Structure Determination, Refinement, and
 Analysis</em>&#x02014;The structure was determined using the iodide-SAD
 technique, and phasing was achieved by utilizing four iodide sites to 3.0
 &#x000c5; resolution using PHENIX
 (<a href="#ref23" rid="ref23" class=" bibr popnode">23</a>). An initial model was
 built automatically without side chains and subsequently rebuilt manually
 using COOT (<a href="#ref24" rid="ref24" class=" bibr popnode">24</a>). The high
 resolution structure was determined by molecular replacement technique using
 the PfNapL iodide-SAD model. This model to 2.3 &#x000c5; resolution was further
 refined using CNS (<a href="#ref25" rid="ref25" class=" bibr popnode">25</a>)
 (<a href="/pmc/articles/PMC2665062/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>). The final model was
 validated using PROCHECK (<a href="#ref26" rid="ref26" class=" bibr popnode">26</a>).
 All figures were generated using Chimera
 (<a href="#ref27" rid="ref27" class=" bibr popnode">27</a>). The least square
 fittings and structural alignment were carried out using LSQMAN
 (<a href="#ref28" rid="ref28" class=" bibr popnode">28</a>). Electrostatic potential
 surfaces were generated using GRASP
 (<a href="#ref29" rid="ref29" class=" bibr popnode">29</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="tbl1"><h3>TABLE 1</h3><!--caption a7--><div class="caption"><p><strong>Data collection and refinement statistics</strong></p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th colspan="1" rowspan="1" align="center" valign="bottom"><strong>Parameter</strong></th><th colspan="1" rowspan="1" align="center" valign="bottom"><strong>Native</strong></th><th colspan="1" rowspan="1" align="center" valign="bottom"><strong>Iodide-SAD</strong></th></tr></thead><tbody><tr><td colspan="1" rowspan="1" align="left" valign="bottom"><strong>Data collection</strong></td><td colspan="1" rowspan="1" align="center" valign="bottom"></td><td colspan="1" rowspan="1" align="center" valign="bottom"></td></tr><tr><td colspan="1" rowspan="1" align="left" valign="bottom"> Cell dimensions
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"><em>a</em> = 92.41 &#x000c5;, <em>b</em> = 38.12 &#x000c5;, <em>c</em> = 80.05
 &#x000c5;, and &#x003b2; = 107.87&#x000b0;
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"><em>a</em> = 76.68 &#x000c5;, <em>b</em> = 37.83 &#x000c5;, <em>c</em> = 79.31
 &#x000c5;, and &#x003b2; = 99.83&#x000b0;
 </td></tr><tr><td colspan="1" rowspan="1" align="left" valign="bottom"> Wavelength (&#x000c5;)
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"> 1.0
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"> 1.54
 </td></tr><tr><td colspan="1" rowspan="1" align="left" valign="bottom"> Anomalous scatterer
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"></td><td colspan="1" rowspan="1" align="center" valign="bottom"> I
 </td></tr><tr><td colspan="1" rowspan="1" align="left" valign="bottom"> Resolution (&#x000c5;)
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"> 50-2.3
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"> 50-3.0
 </td></tr><tr><td colspan="1" rowspan="1" align="left" valign="bottom"> Outer shell resolution
 (&#x000c5;)<sup><em>a</em></sup></td><td colspan="1" rowspan="1" align="center" valign="bottom"> 2.38-2.3
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"> 3.11-3.0
 </td></tr><tr><td colspan="1" rowspan="1" align="left" valign="bottom"> No. of unique reflections
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"> 12,065 (1217)
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"> 4606 (451)
 </td></tr><tr><td colspan="1" rowspan="1" align="left" valign="bottom"> Redundancy
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"> 5.1 (5.1)
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"> 14.5 (13.2)
 </td></tr><tr><td colspan="1" rowspan="1" align="left" valign="bottom"><em>R</em><sub>merge</sub><sup><em>b</em></sup></td><td colspan="1" rowspan="1" align="center" valign="bottom"> 0.052 (0.238)
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"> 0.084 (0.331)
 </td></tr><tr><td colspan="1" rowspan="1" align="left" valign="bottom"> Completeness (%)
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"> 100 (99.9)
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"> 99.0 (97.6)
 </td></tr><tr><td colspan="1" rowspan="1" align="left" valign="bottom"><em>I</em>/&#x003c3;<em>I</em></td><td colspan="1" rowspan="1" align="center" valign="bottom">
 26.1 (4.98)
 </td><td colspan="1" rowspan="1" align="center" valign="bottom">
 45.6 (11.4)
 </td></tr></tbody><tbody><tr><td colspan="1" rowspan="1" align="left" valign="bottom"><strong>Phasing (PHENIX)</strong></td><td colspan="1" rowspan="1" align="center" valign="bottom"></td><td colspan="1" rowspan="1" align="center" valign="bottom"></td></tr><tr><td colspan="1" rowspan="1" align="left" valign="bottom"> AutoSol figure of merit
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"></td><td colspan="1" rowspan="1" align="center" valign="bottom"> 0.36
 </td></tr><tr><td colspan="1" rowspan="1" align="left" valign="bottom">
 Post DM-figure of merit
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"></td><td colspan="1" rowspan="1" align="center" valign="bottom">
 0.66
 </td></tr></tbody><tbody><tr><td colspan="1" rowspan="1" align="left" valign="bottom"><strong>Refinement</strong></td><td colspan="1" rowspan="1" align="center" valign="bottom"></td><td colspan="1" rowspan="1" align="center" valign="bottom"></td></tr><tr><td colspan="1" rowspan="1" align="left" valign="bottom"> Resolution range (&#x000c5;)
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"> 50-2.3
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"></td></tr><tr><td colspan="1" rowspan="1" align="left" valign="bottom"> No. of reflections (test set)
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"> 1219
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"></td></tr><tr><td colspan="1" rowspan="1" align="left" valign="bottom"><em>R</em><sub>factor</sub>/<em>R</em><sub>free</sub> (%)
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"> 21.8/28.5
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"></td></tr><tr><td colspan="1" rowspan="1" align="left" valign="bottom"> No. of protein atoms
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"> 1668
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"></td></tr><tr><td colspan="1" rowspan="1" align="left" valign="bottom">
 No. of water molecules
 </td><td colspan="1" rowspan="1" align="center" valign="bottom">
 74
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"></td></tr></tbody><tbody><tr><td colspan="1" rowspan="1" align="left" valign="bottom"><strong><em>B</em> factor (&#x000c5;<sup>2</sup>)</strong></td><td colspan="1" rowspan="1" align="center" valign="bottom"></td><td colspan="1" rowspan="1" align="center" valign="bottom"></td></tr><tr><td colspan="1" rowspan="1" align="left" valign="bottom"> Protein atoms
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"> 36.6
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"></td></tr><tr><td colspan="1" rowspan="1" align="left" valign="bottom">
 Water atoms
 </td><td colspan="1" rowspan="1" align="center" valign="bottom">
 35.1
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"></td></tr></tbody><tbody><tr><td colspan="1" rowspan="1" align="left" valign="bottom"><strong>Stereochemistry</strong></td><td colspan="1" rowspan="1" align="center" valign="bottom"></td><td colspan="1" rowspan="1" align="center" valign="bottom"></td></tr><tr><td colspan="1" rowspan="1" align="left" valign="bottom"> r.m.s. deviation bond length (&#x000c5;)
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"> 0.01
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"></td></tr><tr><td colspan="1" rowspan="1" align="left" valign="bottom">
 r.m.s. deviation bond angle (degrees)
 </td><td colspan="1" rowspan="1" align="center" valign="bottom">
 1.50
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"></td></tr></tbody><tbody><tr><td colspan="1" rowspan="1" align="left" valign="bottom"><strong>Ramachandran plot</strong></td><td colspan="1" rowspan="1" align="center" valign="bottom"></td><td colspan="1" rowspan="1" align="center" valign="bottom"></td></tr><tr><td colspan="1" rowspan="1" align="left" valign="bottom"> In most favored regions (%)
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"> 91.5
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"></td></tr><tr><td colspan="1" rowspan="1" align="left" valign="bottom"> In additionally allowed regions (%)
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"> 7.4
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"></td></tr><tr><td colspan="1" rowspan="1" align="left" valign="bottom">
 In generously allowed regions (%)
 </td><td colspan="1" rowspan="1" align="center" valign="bottom">
 1.1
 </td><td colspan="1" rowspan="1" align="center" valign="bottom"></td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140713603793024"><a target="object" rel="noopener" href="/pmc/articles/PMC2665062/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn1"><sup>a</sup>Numbers in parentheses are values in the highest resolution shell.</div><div id="tblfn2"><sup>b</sup><em>R</em><sub>merge</sub> = &#x02211;&#x0007c;<em>I</em><sub>obs</sub> -
 &#x0003c;<em>I</em>&#x0003e;&#x0007c;/&#x02211;&#x0003c;<em>I</em>&#x0003e; summed over all
 observations and reflections.</div></div></div><p></p><p><em>Parasite Culture, Plasmid Constructs, and
 Transfection</em>&#x02014;<em>P. falciparum</em> of the 3D7 strain were cultured
 <em>in vitro</em> in O+ human erythrocytes using standard conditions
 (<a href="#ref30" rid="ref30" class=" bibr popnode">30</a>). For the gene deletion
 construct, we amplified the 5&#x02032;- and 3&#x02032;-targeting segments of
 PfNapL using the primer pairs aw775/aw776 and aw777/778, resulting in 645- and
 622-bp products, respectively. After amplification, the targeting segments
 were cloned into the pCC-1 vector
 (<a href="#ref31" rid="ref31" class=" bibr popnode">31</a>) using SacII/SpeI for the
 5&#x02032; and EcoRI/AvrII for the 3&#x02032; segment, resulting in the
 pCC-1&#x00394;PfNapL plasmid. After proliferation of the plasmid DNA in
 <em>Escherichia coli</em> PMC103 cells and purification with a Qiagen Maxi
 kit, 80 &#x003bc;g of plasmid DNA was taken up in cytomix and transfected via
 electroporation using standard procedures
 (<a href="#ref32" rid="ref32" class=" bibr popnode">32</a>). Parasites containing the
 plasmid were selected for in the presence of 2 n<span class="small-caps">m</span> WR99210
 (<a href="#ref33" rid="ref33" class=" bibr popnode">33</a>). The resulting parasite
 population was then subjected to two off/on drug cycles for 21 days each to
 encourage the loss of free plasmid. After each addition of WR99210, a
 subpopulation was selected on 231 n<span class="small-caps">m</span> 5-fluorocytosine in the
 presence of WR99210 (to encourage double recombination events) and analyzed
 via Southern blot. Two independent rounds of transfections were performed and
 analyzed: Aw775, atcccgcggTATTTGAATTTGATTTCTTGC; Aw776,
 gatactagtGCTAGTTAAATAAACATAACAG; Aw777, atcgaattcGTATGCAACTACTTTTTCG; Aw778,
 gatcctaggCTGTATGTAATAACACTGGCTCG.</p><p>To generate transgenic parasites expressing GFP chimeras, we amplified the
 sequence encoding PfNapL from cDNA and cloned the product into the pARL vector
 (<a href="#ref34" rid="ref34" class=" bibr popnode">34</a>) to create PfNapL-pARL.
 Conserved residues of the putative nuclear export signal (NES) were mutated by
 yielding the plasmids PfNapL L46A-pARL and PfNapL Q50A-pARL. The expression of
 chimeric proteins in this vector is driven by the <em>crt</em> promoter. The
 correctness of each construct was confirmed by DNA sequencing. 3D7 <em>P.
 falciparum</em> parasites were transfected with 100 &#x003bc;g of plasmid DNA
 (Qiagen) by electroporation and cultured in the presence of 2 n<span class="small-caps">m</span>
 WR99210 (<a href="#ref32" rid="ref32" class=" bibr popnode">32</a>).</p><p><em>Microscopy</em>&#x02014;In order to observe the localization of the GFP
 chimeras in all three generated cell lines, live cells were stained with DAPI
 and immediately processed for viewing at ambient temperature. Cells were
 viewed with an Apochromat &#x000d7;100/1.4 numerical aperture oil DIC lens on a
 Zeiss Axioskop 2 microscope equipped with a PCO SensiCam (12-bit) camera and
 Axiovision 3 software. Captured images were processed using Photoshop and
 ImageJ software (available on the World Wide Web).</p><p class="p p-last"><em>Histone-binding ELISAs</em>&#x02014;Residues Asp<sup>192</sup>,
 His<sup>227</sup>, Thr<sup>230</sup>, Glu<sup>260</sup>, Lys<sup>266</sup>,
 Asp<sup>233</sup>, Tyr<sup>259</sup>, Ile<sup>136</sup>, and Ile<sup>147</sup>
 were mutated to alanine using the QuikChange site-directed mutagenesis kit
 (Stratagene). Tetramer and octamer forms of calf thymus histone were
 reconstituted from individual histones (Roche Applied Science) after
 denaturing and refolding according to the method developed by Tanaka <em>et
 al.</em> (<a href="#ref35" rid="ref35" class=" bibr popnode">35</a>). ELISAs were
 performed by coating individual histones, histone tetramers, or histone
 octamers (100 ng/well) in microtiter plates (Nunc). The histones were coated
 on plates in phosphate-buffered saline (PBS) and incubated at 4 &#x000b0;C
 overnight. The following day, the plates were washed with PBS plus 0.2% Tween
 20 (PBST) and blocked with 3% PBS-bovine serum albumin (200 &#x003bc;l/well) and
 incubated for 1 h at 37 &#x000b0;C. Then 100 &#x003bc;l of purified PfNapL
 (concentration ranging from 25 to 800 ng) was added to each well in
 histone-binding buffer (20 m<span class="small-caps">m</span> HEPES, 7.5 m<span class="small-caps">m</span>
 MgCl<sub>2</sub>, 1 m<span class="small-caps">m</span> dithiothreitol, 0.5 m<span class="small-caps">m</span> EDTA, and
 50 m<span class="small-caps">m</span> KCl, pH 7.5). Plates were incubated for an additional 1 h at
 37 &#x000b0;C and washed three times with PBST, and then PfNapL anti-rabbit
 polyclonal antibodies were added to each well (1:25,000 dilutions). After
 incubation at 37 &#x000b0;C for 1 h, three more PBST washes were done, followed by
 the addition of horseradish peroxidase-conjugated secondary antibody
 (anti-rabbit), and incubated for an additional 1 h at 37 &#x000b0;C. After three
 more PBST washes, a color reaction was developed using orthophenylenediamine
 (Sigma) and H<sub>2</sub>O<sub>2</sub> in citrate-phosphate buffer. The
 optical density was measured at 490 nm using an ELISA plate reader (Molecular
 Probes). Bovine serum albumin, primary antibodies, and secondary antibodies
 were separately used as controls in these ELISAs.</p></div><div id="sec-a.p.e" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.p.etitle">RESULTS</h2><p class="p p-first"><em>Structure Determination and Overall Structure of PfNapL</em>&#x02014;The
 crystal structure of PfNapL was determined using the iodide-SAD technique at
 3.0 &#x000c5; and refined using high resolution data to 2.3 &#x000c5; resolution
 (<a href="/pmc/articles/PMC2665062/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>). Purified PfNapL
 crystallized in monoclinic space group C2 with solvent content of 55%, and the
 asymmetric unit contains one monomer of PfNapL. For a total of 347 residues of
 full-length PfNapL (molecular mass &#x0223c;41 kDa), electron density was observed
 for the central core residues 33-281. The N- and C-terminal regions in PfNapL
 were presumably cleaved off during protein processing prior to
 crystallization. The regions 89-91, 138-144, 171-178, 200-221, and 235-246
 have weak electron density and are disordered in the overall structure. The
 final refined model of PfNapL has <em>R</em><sub>factor</sub> and
 <em>R</em><sub>free</sub> values of 21.8 and 28.5%, respectively
 (<a href="/pmc/articles/PMC2665062/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>).</p><p>The NAP/SET proteins consist of a central region of &#x0223c;250 residues that
 are thought to be primarily responsible for histone binding
 (<a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1<em>a</em></span></a>). In general,
 NAPs comprise low complexity sequences in their N and C termini, wherein the C
 terminus usually contains stretches of acidic residues
 (<a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1<em>a</em></span></a>). The overall
 fold of PfNapL is similar to yNAP-1 and hSET
 (<a href="#ref15" rid="ref15" class=" bibr popnode">15</a>,
 <a href="#ref16" rid="ref16" class=" bibr popnode">16</a>)
 (<a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1<em>b</em></span></a>). Each
 monomer contains a domain I composed of a dimerization helix &#x003b1;2
 (residues 37-87). The monomer also contains domain II, which is composed of
 &#x003b1;-helices (&#x003b1;3 (residues 102-108), &#x003b1;4 (residues 111-116), and
 &#x003b1;5 (residues 119-127)), a &#x003b2; subdomain containing four antiparallel
 &#x003b2;-strands (residues 128-185), another &#x003b1;-helix &#x003b1;7 (residues
 225-228), and a final two &#x003b1;-helices on the other side of the &#x003b2;
 subdomain (&#x003b1;7 (residues 248-267) and &#x003b1;8 (residues 269-277))
 (<a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1<em>b</em></span></a>). We have
 earlier shown the dimeric nature of PfNapL in solution
 (<a href="#ref13" rid="ref13" class=" bibr popnode">13</a>). The crystallographic
 2-fold symmetry generates a homodimer of PfNapL of &#x0223c;87 &#x000d7; 52 &#x000d7;
 33 &#x000c5; with a total buried surface area of 4209 &#x000c5;<sup>2</sup>. The
 51-residue-long dimerization helix &#x003b1;2 forms the characteristic shape of
 PfNapL, wherein two backbone helices cluster in an antiparallel manner to form
 the dimer using mainly hydrophobic interactions and few salt bridges/hydrogen
 bonds (<a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1<em>b</em></span></a>). The
 overall sequence identity of PfNapL with yNAP-1, hSET, and Vps75 is 24, 25,
 and 14%, respectively.
 (<a href="#ref15" rid="ref15" class=" bibr popnode">15</a>-<a href="#ref19" rid="ref19" class=" bibr popnode">19</a>)
 (<a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1<em>c</em></span></a> and Fig. S1).
 The phylogenetic analysis of NAPs suggests greater evolutionary distance of
 malaria parasite NAPs from homologs in yeast and humans (available on the
 World Wide Web) (<a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig.
 1<em>e</em></span></a>).</p><p><em>Gene Knock-out Studies on PfNapL</em>&#x02014;To investigate whether the
 function of PfNapL could be disrupted, we designed a construct to specifically
 target the PfNapL gene and to replace it with an human dihydrofolate
 reductase-selectable marker cassette. Despite several attempts, we were not
 able to obtain a population where the PfNapL gene was replaced by the human
 dihydrofolate reductase cassette (Fig. S2). Although we cannot exclude any
 technical problems, the inability to disrupt the function of PfNapL is
 indicative of a pivotal role of this gene product for the survival of <em>P.
 falciparum</em>. At the moment, our repertoire of molecular tools in <em>P.
 falciparum</em> precludes us from performing conditional gene disruptions. An
 essential or at least a very beneficial role of this molecule has been
 described in several other organisms like mouse and <em>Drosophila</em>
 (<a href="#ref9" rid="ref9" class=" bibr popnode">9</a>,
 <a href="#ref10" rid="ref10" class=" bibr popnode">10</a>). Therefore, we propose
 that the PfNapL protein is very likely to be essential for parasite survival
 and that it provides a potential new target for disruption of parasite
 nucleosome assembly as a route toward controlling parasite growth.</p><p></p><p><em>Structural Conservation with yNAP-1, hSET, and Vps75</em>&#x02014;The
 root mean square (r.m.s.) deviation derived from the least square fittings of
 191 C<sup>&#x003b1;</sup> of PfNapL with the corresponding C<sup>&#x003b1;</sup>
 of yNAP-1, hSET, and Vps75 is 1.9, 1.8, and 1.8 &#x000c5;, respectively
 (<a href="/pmc/articles/PMC2665062/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2<em>a</em></span></a> and Fig. S3).
 The dimerization helix &#x003b1;2 of PfNapL shows an r.m.s. deviation of 1.4 and
 1.1 &#x000c5; with the corresponding dimerization helices of yNAP-1 and hSET
 (<a href="/pmc/articles/PMC2665062/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2, <em>b</em> and
 <em>c</em></span></a>). The core domain II of PfNapL is also similar to
 yNAP-1 and hSET, with an average r.m.s. deviation of 1.1 and 1.0 &#x000c5;,
 respectively (<a href="/pmc/articles/PMC2665062/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2, <em>b</em> and
 <em>c</em></span></a>). The dimerization helix &#x003b1;2 of PfNapL has high
 sequence similarity with the corresponding dimerization helices of hSET/yNAP-1
 for a total of &#x0223c;50 residues (<a href="/pmc/articles/PMC2665062/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2,
 <em>b</em> and <em>c</em></span></a>). In the &#x003b2; subdomain, PfNapL has 16
 of 24 identical and 5 of 8 conserved residues with the corresponding &#x003b2;
 subdomains of hSET/yNAP-1. The residues 290-295 of this antiparallel
 &#x003b2;-sheet region of yNAP-1 had been previously identified as the nuclear
 localization signal (NLS) (<a href="#ref15" rid="ref15" class=" bibr popnode">15</a>)
 (Figs. <a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Figs.1<em>c</em></span></span><span>1<em>c</em></span></a> and
 <a href="/pmc/articles/PMC2665062/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>2, <em>b</em> and <em>c</em></span></a>).
 Residues 200-221 in PfNapL corresponding to this short anti-parallel
 &#x003b2;-sheet region (&#x003b2;5 and &#x003b2;6) in yNAP-1 are disordered
 (<a href="/pmc/articles/PMC2665062/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2<em>b</em></span></a>). However,
 the NLS sequences between yeast and PfNapL are highly conserved (Figs.
 <a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Figs.1<em>c</em></span></span><span>1<em>c</em></span></a> and
 <a href="/pmc/articles/PMC2665062/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and2<em>b</em></span></span><span>2<em>b</em></span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC2665062/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140713622702144"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2665062_zbc0170971220002.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is zbc0170971220002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2665062/bin/zbc0170971220002.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140713622702144"><a target="object" rel="noopener" href="/pmc/articles/PMC2665062/figure/fig2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC2665062/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">FIGURE 2.</a></div><!--caption a7--><div class="caption"><p><strong>Comparison of PfNapL structure with yNAP-1 (Protein Data Bank code 2AYU)
 and hSET (Protein Data Bank code 2E50).</strong> <em>a</em>, superimposition of
 PfNapL monomer (<em>purple</em>) onto yNAP-1 (<em>blue</em>) and hSET
 (<em>orange</em>). Accessory domain of yNAP-1 is <em>colored yellow</em>. NES
 and NLS in yNAP-1 and PfNapL are indicated. <em>b</em>, superimposition of
 PfNapL and yNAP-1 dimers. The <em>lower half</em> shows superimposition of
 domain I and II from PfNapL and yNAP-1 monomers. yNAP-1 is <em>colored
 blue</em>, and the accessory domain is <em>colored yellow</em>. PfNapL is
 <em>colored purple</em>. The extra residues in domain II of yNAP-1 are
 <em>colored cyan</em> and are disordered in PfNapL. <em>c</em>,
 superimposition of PfNapL and hSET dimers. The <em>lower half</em> shows
 superimposition of domain I and II from PfNapL and hSET monomers. The extra
 residues of PfNapL are <em>colored cyan</em> and are disordered in hSET.</p></div></div></div><p><em>PfNapL Lacks the &#x0201c;Accessory Domain&#x0201d; Conserved in Higher
 Eukaryotic NAPs</em>&#x02014;Comparison between yNAP-1, hSET, and PfNapL
 structures revealed that yNAP-1 contains an additional &#x003b1;-helix, termed
 the &#x0201c;accessory domain,&#x0201d; which has been inserted between domain I
 and II but is absent in both hSET and PfNapL (Figs.
 <a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Figs.1<em>c</em></span></span><span>1<em>c</em></span></a> and
 <a href="/pmc/articles/PMC2665062/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and2<em>a</em>).</span></span><span>2<em>a</em></span></a>). Based on the
 sequence comparison, it can be inferred that this accessory domain is present
 in all other eukaryotic nucleosome assembly proteins but is seen to be
 specifically absent in PfNapL and other NAPs from <em>Plasmodium</em> species
 (<a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1<em>d</em></span></a>).
 Intriguingly, SET protein homologs, such as the recently characterized human
 SET (<a href="#ref16" rid="ref16" class=" bibr popnode">16</a>), are absent in yeast
 and <em>Plasmodium</em> (stand alone SET domains are found in higher
 eukaryotes like mice, cows, and humans). In <em>Plasmodium</em>, SET-like
 domains are a constituent of significantly larger proteins (molecular mass
 &#x02265;142 kDa) termed &#x0201c;SET domain proteins,&#x0201d; where functions and
 localizations are still unknown (PFD0190w, PFF1440w, and PF08_0012).</p><p><em>PfNapL Dimer and Conserved Hydrophobic Motifs</em>&#x02014;The dimer
 formation in yNAP-1, hSET, Vps75, and PfNapL is achieved via the dimerization
 helix &#x003b1;2. In PfNapL, residues Gln<sup>50</sup>, Glu<sup>52</sup>,
 Tyr<sup>54</sup>, Glu<sup>57</sup>, Lys<sup>59</sup>, Glu<sup>63</sup>,
 Arg<sup>68</sup>, Lys<sup>70</sup>, Tyr<sup>71</sup>, and Lys<sup>81</sup> of
 the dimerization helix &#x003b1;2 and Pro<sup>271</sup>, Tyr<sup>272</sup>, and
 Asp<sup>275</sup> of helix &#x003b1;8, which lies on the other side of the
 &#x003b2; subdomain contribute to dimer formation. Residues Glu<sup>52</sup> and
 Glu<sup>63</sup> and Lys<sup>70</sup> and Lys<sup>81</sup> of the dimerization
 helix &#x003b1;2 form salt bridges. The hydrogen bonding interactions
 contributing to dimer formation in PfNapL are mostly structurally conserved
 with those of hSET (<a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig.
 1<em>c</em></span></a>). Unlike yNAP-1, the PfNapL dimerization helices are
 bent at &#x0223c;50&#x000b0; very similar to the hSET and Vps75 dimers (Fig.
 S4<em>a</em> and Fig. S5<em>a</em>). Although a proline residue at position 77
 is present in PfNapL, there are no kinks formed in the dimerization helices
 upon dimer formation in PfNapL in contrast to yNAP-1 (Fig. S4<em>a</em>).
 Interestingly, PfNapL dimer displays a somewhat narrow cavity (Fig.
 S5<em>b</em>) when compared with Vps75 dimer
 (<a href="#ref18" rid="ref18" class=" bibr popnode">18</a>,
 <a href="#ref19" rid="ref19" class=" bibr popnode">19</a>). In yNAP-1, a disulfide
 bond is formed by residues Cys<sup>249</sup> and Cys<sup>272</sup>. These
 positions are replaced by serine and threonine in PfNapL, and only a single
 cysteine is present in PfNapL at position 133. The hydrophobic core of PfNapL
 and yNAP-1 is stabilized by the presence of SFF(T/N)FF and (L/I)P(E/S)FWL
 motifs that are conserved among the NAP family
 (<a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1<em>c</em></span></a> and Fig.
 S4<em>b</em>).</p><p></p><p><em>PfNapL Localization in Parasite Cytoplasm Using Transfection
 Studies</em>&#x02014;We have earlier shown that PfNapL is expressed during all
 of the blood stages and is localized to the parasite cytoplasm
 (<a href="#ref13" rid="ref13" class=" bibr popnode">13</a>,
 <a href="#ref14" rid="ref14" class=" bibr popnode">14</a>). PfNapL also contains
 conserved NLS and NES motifs (Figs.
 <a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Figs.1<em>c</em></span></span><span>1<em>c</em></span></a> and
 <a href="/pmc/articles/PMC2665062/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>3, <em>b</em> and <em>c</em></span></a>)
 like the yNAP-1 (which is observed in both the cytoplasm and the nucleus).
 Corresponding to the presence of NES in yNAP1 (residues 88-103)
 (<a href="/pmc/articles/PMC2665062/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Fig. 3<em>a</em></span></a>), residues
 35-50 of PfNapL have been predicted as an NES
 (<a href="/pmc/articles/PMC2665062/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Fig. 3, <em>b</em> and
 <em>c</em></span></a>). It has been suggested in yNAP-1 that this NES is
 masked by the accessory domain, which is thought to play an important role in
 the import and export of yNAP-1 into and from the nucleus
 (<a href="#ref15" rid="ref15" class=" bibr popnode">15</a>)
 (<a href="/pmc/articles/PMC2665062/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Fig. 3<em>a</em></span></a>). As
 mentioned earlier, this accessory domain in yNAP-1 (residues 141-180) is the
 inserted &#x003b1;-helix, which is absent in PfNapL (Figs.
 <a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Figs.1<em>c</em></span></span><span>1<em>c</em></span></a> and
 <a href="/pmc/articles/PMC2665062/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and2<em>a</em>).</span></span><span>2<em>a</em></span></a>). Interestingly,
 hSET also lacks this accessory domain yet contains both NES and NLS motifs
 (<a href="#ref16" rid="ref16" class=" bibr popnode">16</a>). Further, hSET is
 localized both in the cell cytoplasm and the nucleus, similar to yNAP-1
 (<a href="#ref36" rid="ref36" class=" bibr popnode">36</a>) and like other histone
 shuttling proteins. In this respect, PfNapL clearly stands as an exception
 despite its structural and sequence similarities with yeast NAP and human SET
 proteins.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig3" co-legend-rid="lgnd_fig3"><a href="/pmc/articles/PMC2665062/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140713612973744"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2665062_zbc0170971220003.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is zbc0170971220003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2665062/bin/zbc0170971220003.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140713612973744"><a target="object" rel="noopener" href="/pmc/articles/PMC2665062/figure/fig3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig3"><div><a class="figpopup" href="/pmc/articles/PMC2665062/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3">FIGURE 3.</a></div><!--caption a7--><div class="caption"><p><strong>NES and NLS motifs in yNAP-1 and PfNapL.</strong> <em>a</em>, surface
 representation of yNAP-1. NES, accessory domain, and NLS are <em>colored red,
 yellow</em>, and <em>magenta</em>, respectively. <em>b</em>, surface
 representation of PfNapL. Predicted NES is colored <em>red</em>. The predicted
 NLS is indicated. <em>c</em>, sequence alignment of NES and NLS in yNAP-1 and
 PfNapL. The key residues are <em>colored green</em>. Superimposition of NES in
 yNAP-1 and PfNapL is shown with key residues shown in <em>green</em> as
 <em>ball-and-stick representations</em>.</p></div></div></div><p>In order to address whether PfNapL is located at some stage in the nucleus
 and whether its cytosolic localization is due to continuous export mediated by
 its putative NES, we generated cell lines expressing PfNapL-GFP chimeras. For
 these chimeras, we either used wild-type sequence or introduced mutations in
 the conserved residues of the NES (PfNapL L46A or PfNapL Q50A)
 (<a href="/pmc/articles/PMC2665062/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Fig. 3<em>c</em></span></a>). Disruption
 of a functional NES was predicted to lead to the accumulation of GFP chimera
 in the parasite nucleus. Expression of the GFP chimeras was verified via
 Western blot and probed with anti-GFP antibodies
 (<a href="/pmc/articles/PMC2665062/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4<em>a</em></span></a>). A band of
 &#x0223c;70 kDa was observed in all cell lines, corresponding to the size of
 PfNapL (40.5 kDa) fused to GFP (26.8 kDa). The GFP fusion protein of all three
 transfectants (wild type and mutants) was detectable in the cytoplasm of all
 stages. Previous localization experiments with PfNapL-specific antibodies
 detected an exclusive cytoplasmic localization in all stages, except for the
 schizont stages, where the separation between nuclear (DAPI) and PfNapL
 staining was less defined
 (<a href="#ref13" rid="ref13" class=" bibr popnode">13</a>). This could be due to
 greater interdigitation of the nucleus and cytoplasm at that stage or due to a
 closer association of the PfNapL pool with the nucleus. To address this
 question, we followed the localization of PfNapL wild type over the asexual
 erythrocytic life cycle (<a href="/pmc/articles/PMC2665062/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4,
 <em>b-d</em></span></a>). PfNapL wild type-GFP was detected in the cytoplasm
 of the parasites. No fluorescence could be detected in the erythrocyte
 cytosol. Neither the food vacuole nor the nucleus of the parasite showed
 fluorescence in any parasite stage. This indicates that PfNapL is exclusively
 located in the cytoplasm of the parasite and that its putative NES is not
 necessary for its localization, and the predicted NES is not
 functional/relevant in PfNapL.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig4" co-legend-rid="lgnd_fig4"><a href="/pmc/articles/PMC2665062/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140713622358016"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2665062_zbc0170971220004.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is zbc0170971220004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2665062/bin/zbc0170971220004.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140713622358016"><a target="object" rel="noopener" href="/pmc/articles/PMC2665062/figure/fig4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig4"><div><a class="figpopup" href="/pmc/articles/PMC2665062/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4">FIGURE 4.</a></div><!--caption a7--><div class="caption"><p><strong>Stage specific expression of PfNapL-GFP chimeras.</strong> <em>a</em>,
 Western blot of transgenic cell lines expressing GFP-tagged versions of NapL
 with antibodies against GFP. Expression of PfHsp70 was detected to ensure
 similar loading. The localization of PfNapL wild type-GFP (<em>b</em>), PfNapL
 L46A-GFP (<em>c</em>), and PfNapL Q50A (<em>d</em>) in different stages of the
 erythrocytic life cycle of <em>P. falciparum</em> is shown. The <em>first
 column</em> of each <em>panel</em> shows bright field pictures, followed by a
 nuclear DAPI stain, GFP fluorescence, an overlay of the nuclear and GFP
 localization, and (in the <em>last column</em>) overlay of bright field with
 DAPI and GFP.</p></div></div></div><p></p><p></p><p><em>The Electrostatic Potential of PfNapL Dimer</em>&#x02014;Analysis of the
 electrostatic potential of PfNapL dimer indicates that the convex region of
 the dimerization helix &#x003b1;2 has acidic residues scattered on the surface
 in an alternate manner (Glu<sup>38</sup>, Glu<sup>41</sup>, Glu<sup>52</sup>,
 Asp<sup>55</sup>, Glu<sup>63</sup>, Asp<sup>80</sup>, and Glu<sup>84</sup>)
 (<a href="/pmc/articles/PMC2665062/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>). There are also three
 basic residues toward the end of this helix (Lys<sup>70</sup>,
 Lys<sup>81</sup>, and Arg<sup>82</sup>) contributing to a slightly basic
 nature and hence an uneven charge distribution
 (<a href="/pmc/articles/PMC2665062/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>). The domain II of
 PfNapL dimer has a hydrophobic nature on one side and is highly negatively
 charged on the opposite face with several residues (Glu<sup>119</sup>,
 Glu<sup>122</sup>, Glu<sup>123</sup>, Glu<sup>252</sup>, Glu<sup>256</sup>,
 Asp<sup>258</sup>, Glu<sup>260</sup>, and Glu<sup>267</sup>), forming a cavity
 that is also a characteristic feature of the yNAP-1 and hSET
 (<a href="/pmc/articles/PMC2665062/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>). The bottom of this
 cavity has an uneven charge character, and a single acidic residue,
 Glu<sup>61</sup>, is present at its base. It has been previously suggested for
 yNAP-1 that this highly acidic cavity might be important for binding to basic
 histones (<a href="#ref15" rid="ref15" class=" bibr popnode">15</a>). The residues
 Asp<sup>119</sup>, Glu<sup>122</sup>, Glu<sup>123</sup>, Asp<sup>258</sup>,
 Glu<sup>260</sup>, and Glu<sup>267</sup> that constitute this cavity in PfNapL
 are structurally conserved with yNAP-1 and hSET, hence highlighting their
 possible significance.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig5" co-legend-rid="lgnd_fig5"><a href="/pmc/articles/PMC2665062/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140713615061488"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2665062_zbc0170971220005.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is zbc0170971220005.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2665062/bin/zbc0170971220005.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140713615061488"><a target="object" rel="noopener" href="/pmc/articles/PMC2665062/figure/fig5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig5"><div><a class="figpopup" href="/pmc/articles/PMC2665062/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5">FIGURE 5.</a></div><!--caption a7--><div class="caption"><p><strong>Electrostatic potential distribution of the PfNapL dimer.</strong> The convex
 region of the dimerization helix &#x003b1;2 contains acidic residues scattered
 on the surface in an alternate manner. Domain II of PfNapL forms a cavity
 consisting of mostly acidic residues that are conserved with yNAP-1 and
 hSET.</p></div></div></div><p><em>Prediction of Essential Residues Involved in Histone Binding by
 PfNapL</em>&#x02014;Most histone chaperones contain long acidic stretches in
 their C-terminal region, which were thought to be involved in histone binding,
 considering the highly basic character of histones. However, recent studies
 have shown that these acidic stretches are not essential for histone binding
 activities for chaperones yNAP-1, Asf1, nucleoplasmin, etc.
 (<a href="#ref5" rid="ref5" class=" bibr popnode">5</a>,
 <a href="#ref15" rid="ref15" class=" bibr popnode">15</a>,
 <a href="#ref37" rid="ref37" class=" bibr popnode">37</a>). It is noted that
 full-length PfNapL also has an acidic stretch of residues in its C-terminal
 region similar to yNAP-1 and hSET, with an average pI of &#x0223c;3.0 calculated
 using amino acid sequence (<a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig.
 1<em>a</em></span></a>). Mutagenesis studies on hSET revealed important
 residues of domain II that affect the binding of hSET to both core histones
 and double-stranded DNA (<a href="#ref16" rid="ref16" class=" bibr popnode">16</a>).
 We have mapped these residues on yNAP-1 and PfNapL
 (<a href="/pmc/articles/PMC2665062/figure/fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig6" rid-ob="ob-fig6" co-legend-rid="lgnd_fig6"><span>Fig. 6<em>a</em></span></a>). Based upon
 these hSET mutagenesis data, we identified several corresponding residues in
 PfNapL and mutated them to test their relevance in histone recognition. We
 constructed six single-site mutants of the corresponding residues from PfNapL
 (D192A, H227A, T230A, Y259A, E260A, and K266A). In addition, we made one other
 mutant (D223A) that is conserved, exposed, and proximal to the previously
 implicated histone recognition residues from hSET
 (<a href="#ref16" rid="ref16" class=" bibr popnode">16</a>). Our PfNapL-histone
 binding data, using purified proteins of high quality, show that none of these
 seven mutations altered histone binding of PfNapL to histones H3, H4, H2A, or
 H2B, to histone tetramer, or to histone octamer significantly
 (<a href="/pmc/articles/PMC2665062/figure/fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig6" rid-ob="ob-fig6" co-legend-rid="lgnd_fig6"><span>Fig. 6<em>b</em></span></a>). We
 therefore propose that it is likely for histones to have different a binding
 site(s) on PfNapL (and possibly on other NAPs) in comparison with hSET-histone
 interactions (<a href="#ref16" rid="ref16" class=" bibr popnode">16</a>). These
 residues are not identical among PfNapL, hSET, yNAP-1, and Vps75 proteins, and
 in general, residues proposed by hSET-histone studies show weak conservation
 (Table S1). An overall structure-based residue comparison between yNAP-1,
 Vps75, hSET, and PfNapL reveals very few regions of sequence conservation
 (Fig. S1). In order to further probe the potential sites for histone
 recognition by PfNapL, we inspected the surface residues of PfNapL (keeping in
 view structure and sequence information from NAPs). Here, we have highlighted
 the identical/conserved surface residues in the NAP family (Figs.
 <a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Figs.1<em>d</em></span></span><span>1<em>d</em></span></a> and
 <a href="/pmc/articles/PMC2665062/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and7)</span></span><span>7</span></a>) based on the described
 criterion.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig6" co-legend-rid="lgnd_fig6"><a href="/pmc/articles/PMC2665062/figure/fig6/" target="figure" rid-figpopup="fig6" rid-ob="ob-fig6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140713602090848"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2665062_zbc0170971220006.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is zbc0170971220006.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2665062/bin/zbc0170971220006.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140713602090848"><a target="object" rel="noopener" href="/pmc/articles/PMC2665062/figure/fig6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig6"><div><a class="figpopup" href="/pmc/articles/PMC2665062/figure/fig6/" target="figure" rid-figpopup="fig6" rid-ob="ob-fig6">FIGURE 6.</a></div><!--caption a7--><div class="caption"><p><em>a</em>, orthogonal views of the surface representation of hSET, yNAP-1,
 and PfNapL dimers with important residues from the mutagenesis data of hSET
 <em>colored green</em> and <em>red. b</em>, histone-binding activities of
 PfNapL wild type and mutants (D192A, H227A, T230A, E260A, K266A, D233A, and
 Y259A). Shown is binding with histone tetramer (<em>i</em>), histone octamer
 (<em>ii</em>), H3 (<em>iii</em>), H4 (<em>iv</em>), H2A (<em>v</em>), and H2B
 (<em>vi</em>).</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig7" co-legend-rid="lgnd_fig7"><a href="/pmc/articles/PMC2665062/figure/fig7/" target="figure" rid-figpopup="fig7" rid-ob="ob-fig7"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140713623216320"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2665062_zbc0170971220007.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is zbc0170971220007.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2665062/bin/zbc0170971220007.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140713623216320"><a target="object" rel="noopener" href="/pmc/articles/PMC2665062/figure/fig7/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig7"><div><a class="figpopup" href="/pmc/articles/PMC2665062/figure/fig7/" target="figure" rid-figpopup="fig7" rid-ob="ob-fig7">FIGURE 7.</a></div><!--caption a7--><div class="caption"><p><strong>Two views of the surface representation of PfNapL and yNAP-1 dimers.</strong>
 Corresponding residues from the mutagenesis data of hSET are <em>colored
 green</em> and <em>red</em>. The identical/conserved surface residues within
 the NAP family are <em>colored yellow</em>.</p></div></div></div><p></p><p class="p p-last"><em>Comparisons with Asf1-Histone Complex</em>&#x02014;The &#x003b2; subdomain
 of PfNapL is a highly conserved feature of NAPs and is also found in other
 histone chaperones like Asf1
 (<a href="#ref20" rid="ref20" class=" bibr popnode">20</a>). The crystal structure of
 the human Asf1 has been recently determined in complex with H3-H4 dimer,
 highlighting the crucial residues involved in histone binding
 (<a href="#ref20" rid="ref20" class=" bibr popnode">20</a>,
 <a href="#ref21" rid="ref21" class=" bibr popnode">21</a>). A structural comparison
 of the four-stranded &#x003b2; subdomain of PfNapL with Asf1 shows a
 superimposition with an r.m.s. deviation of 0.97 &#x000c5;
 (<a href="/pmc/articles/PMC2665062/figure/fig8/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig8" rid-ob="ob-fig8" co-legend-rid="lgnd_fig8"><span>Fig. 8<em>a</em></span></a>). Further,
 there are no steric hindrances between the overall PfNapL dimer and H3-H4
 dimer (Fig. S6). Mapping of the interacting residues as highlighted for human
 Asf1 revealed three corresponding, structurally conserved surface residues in
 PfNapL (Ile<sup>136</sup>, Ile<sup>147</sup>, and Ala<sup>185</sup>)
 (<a href="/pmc/articles/PMC2665062/figure/fig8/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig8" rid-ob="ob-fig8" co-legend-rid="lgnd_fig8"><span>Fig. 8<em>b</em></span></a>). Further,
 three residues of PfNapL are conserved among the NAP family
 (<a href="/pmc/articles/PMC2665062/figure/fig8/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig8" rid-ob="ob-fig8" co-legend-rid="lgnd_fig8"><span>Fig. 8<em>c</em></span></a>) and may
 represent common sites for histone recognition. Based on structural
 congruence, we propose that these residues in PfNapL may interact with
 Leu<sup>126</sup> and Ile<sup>130</sup> of H3 and Thr<sup>96</sup> and
 Leu<sup>97</sup> of histone H4. To validate the above modeling, we mutated the
 putative interacting PfNapL residues, Ile<sup>136</sup> and Ile<sup>147</sup>,
 to alanines and tested the mutant proteins for H3-H4 tetramer binding ability.
 Our protein-protein-based ELISA data indicate a significant (up to two-thirds)
 reduction in binding between PfNapL and H3-H4 tetramer
 (<a href="/pmc/articles/PMC2665062/figure/fig8/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig8" rid-ob="ob-fig8" co-legend-rid="lgnd_fig8"><span>Fig. 8<em>d</em></span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig8" co-legend-rid="lgnd_fig8"><a href="/pmc/articles/PMC2665062/figure/fig8/" target="figure" rid-figpopup="fig8" rid-ob="ob-fig8"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140713622032608"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2665062_zbc0170971220008.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is zbc0170971220008.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2665062/bin/zbc0170971220008.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140713622032608"><a target="object" rel="noopener" href="/pmc/articles/PMC2665062/figure/fig8/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig8"><div><a class="figpopup" href="/pmc/articles/PMC2665062/figure/fig8/" target="figure" rid-figpopup="fig8" rid-ob="ob-fig8">FIGURE 8.</a></div><!--caption a7--><div class="caption"><p><strong>Comparison of PfNapL dimer with the Asf1-histone complex (Protein Data
 Bank code 2HUE).</strong> <em>a</em>, superimposition of the &#x003b2; subdomain of
 PfNapL (<em>purple ribbon</em>) and Asf1-histone complex wherein Asf1, H3, and
 H4 are <em>colored light gray, pink</em>, and <em>brown</em>, respectively.
 The conserved residues of PfNapL are shown in a <em>red ball-and-stick
 representation</em>. The interacting residues of H3 and H4 are shown in a
 <em>yellow</em> and <em>green ball-and-stick representation. b</em>, close
 view of the proposed interaction. <em>c</em>, surface representation of PfNapL
 with predicted histone-binding residues <em>colored red. d</em>,
 histone-binding activity of PfNapL wild type and mutants (I136A and I147A)
 with H3-H4 tetramer.</p></div></div></div></div><div id="sec-a.p.f" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.p.ftitle">DISCUSSION</h2><p class="p p-first">The structure of PfNapL was determined using the iodide-SAD technique and
 refined to 2.3 &#x000c5; resolution. Structural analysis of PfNapL has revealed
 that PfNapL conforms to the domain architecture of nucleosome assembly
 proteins, exhibiting a high complexity ordered central region and largely
 disordered N- and C-terminal regions. PfNapL forms a homodimer that acquires a
 fold similar to the yNAP-1, hSET, and Vps75 dimers
 (<a href="#ref15" rid="ref15" class=" bibr popnode">15</a>-<a href="#ref17" rid="ref17" class=" bibr popnode">17</a>).
 These investigations demonstrate high structural homology within these
 proteins despite a low level of sequence identity (only 14-25%)
 (<a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1<em>c</em></span></a>). The major
 difference between PfNapL and yNAP-1 lies in the absence of the accessory
 domain in PfNapL that is inserted between the dimerization helix &#x003b1;2 and
 domain II in yNAP-1 (<a href="#ref15" rid="ref15" class=" bibr popnode">15</a>)
 (<a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1<em>d</em></span></a>).
 Phylogenetic analysis of NAPs suggests greater evolutionary distance of PfNapL
 from homologs in higher eukaryotes (<a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig.
 1<em>e</em></span></a>), although malaria parasite NAPs are unique in their
 lack of the accessory domain. We have earlier shown that PfNapL is a unique,
 nonredundant protein in <em>P. falciparum</em>, and it performs an exclusive
 role in the nucleosome assembly activity in the parasite
 (<a href="#ref13" rid="ref13" class=" bibr popnode">13</a>,
 <a href="#ref14" rid="ref14" class=" bibr popnode">14</a>). Previous studies on gene
 knock-out phenotypes in several organisms like mouse and <em>Drosophila</em>
 have indicated the essentiality of the NAP family
 (<a href="#ref9" rid="ref9" class=" bibr popnode">9</a>,
 <a href="#ref10" rid="ref10" class=" bibr popnode">10</a>). Furthermore, our
 inability to disrupt the gene for PfNapL corroborates the essentiality of this
 protein in <em>P. falciparum</em> (Fig. S2).</p><p></p><p>Corresponding to yNAP-1, PfNapL also contains NES and NLS motifs (Figs.
 <a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Figs.1<em>c</em></span></span><span>1<em>c</em></span></a> and
 <a href="/pmc/articles/PMC2665062/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and3<em>c</em>).</span></span><span>3<em>c</em></span></a>). We have
 previously shown that PfNapL is predominantly localized to the cytoplasm and
 is centrally responsible for shuttling and transfer of histones to PfNapS
 (which is localized to both cytoplasm and the nucleus and which eventually
 deposits histones onto DNA). PfNapL and PfNapS do not interact with each other
 (<a href="#ref14" rid="ref14" class=" bibr popnode">14</a>). Therefore, PfNapL is
 principally localized to the cytoplasm and does not possess the ability to
 deposit the histones in the nucleus
 (<a href="#ref14" rid="ref14" class=" bibr popnode">14</a>). In this context, it is
 interesting to note that hSET also lacks the accessory domain of yNAP-1/other
 NAPs, but unlike PfNapL, hSET is localized both in cytoplasm and the nucleus
 (<a href="#ref22" rid="ref22" class=" bibr popnode">22</a>). Since this accessory
 domain is absent in PfNapL (<a href="/pmc/articles/PMC2665062/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig.
 2<em>b</em></span></a>), the predicted NES on the PfNapL structure is thus
 exposed and not be masked by the accessory domain, as suggested and observed
 for yNAP-1 (<a href="#ref15" rid="ref15" class=" bibr popnode">15</a>)
 (<a href="/pmc/articles/PMC2665062/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Fig. 3, <em>a</em> and
 <em>b</em></span></a>). From a structural perspective, this further validates
 the localization of PfNapL to the cytoplasm. The histone binding
 characteristics of PfNapL have been detailed previously, including its ability
 to transfer cytoplasmic histones on to PfNapS. A model for relay of histones
 from parasite cytoplasm to the nucleus has also been proposed by us previously
 (see Ref. <a href="#ref14" rid="ref14" class=" bibr popnode">14</a> and specifically
 <a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1<em>f</em></span></a> reproduced
 here from Ref. <a href="#ref14" rid="ref14" class=" bibr popnode">14</a>). Our
 published reports on the two <em>Plasmodium</em> parasite nucleosome assembly
 proteins have clearly suggested different roles for PfNapL (in histone
 shuttling in the cytoplasm) and for PfNapS (which is localized both in the
 cytoplasm and nucleus and which can perform histone deposition onto DNA
 (<a href="#ref13" rid="ref13" class=" bibr popnode">13</a>,
 <a href="#ref14" rid="ref14" class=" bibr popnode">14</a>)
 (<a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1<em>f</em></span></a>).</p><p></p><p>Electrostatic potential distribution of PfNapL dimer reveals a stretch of
 alternate acidic residues on the convex surface of the dimerization helix
 &#x003b1;2 and a highly acidic side to domain II. If this convex surface of the
 PfNapL dimer is implicated in histone recognition, then it can be suggested
 that the highlighted acidic residues on this surface might be making alternate
 contacts with an &#x003b1;-helix of a core histone
 (<a href="/pmc/articles/PMC2665062/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>). In yNAP-1, the acidic
 stretch forming a cavity in domain II has been shown to be involved in
 neutralizing and binding the basic N-terminal histone tails
 (<a href="#ref15" rid="ref15" class=" bibr popnode">15</a>). Also, most of the acidic
 residues constituting this cavity are structurally conserved among PfNapL and
 yNAP-1. Thus, based upon the earlier suggestions for yNAP-1, this acidic
 region in PfNapL may be involved in histone recognition
 (<a href="/pmc/articles/PMC2665062/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>).</p><p>Mapping of the corresponding residues from hSET mutagenesis data onto
 PfNapL and yNAP-1 demonstrates the conserved nature of 10 of these residues
 among PfNapL, hSET, and yNAP-1 (<a href="/pmc/articles/PMC2665062/figure/fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig6" rid-ob="ob-fig6" co-legend-rid="lgnd_fig6"><span>Fig.
 6<em>a</em></span></a>). However, mutation of some of these residues on
 PfNapL did not significantly alter PfNapL-histone binding
 (<a href="/pmc/articles/PMC2665062/figure/fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig6" rid-ob="ob-fig6" co-legend-rid="lgnd_fig6"><span>Fig. 6<em>b</em></span></a>).
 Consequently, based upon our mutagenesis data analysis, we suggest that the
 binding site(s) and/or essential residues for the recognition of core histones
 may be different for PfNapL from those proposed by hSET-histone interactions
 (<a href="#ref16" rid="ref16" class=" bibr popnode">16</a>). A structure-based
 comparison of the residues that were mutated in PfNapL reveals that none of
 these are identical among hSET, yNAP-1, Vps75, and PfNapL. Indeed, only a few
 show weak conservation (Table S1). Also, an overall structure-based comparison
 between yNAP-1, Vps75, hSET, and PfNapL reveals very few regions of sequence
 conservation (Fig. S1). These observations therefore suggest that histone
 recognition site(s) for hSET may not be applicable for nucleosome assembly
 proteins like PfNapL and yNAP-1. Our mapping of highly conserved and
 surface-exposed residues in the NAP family provides some clues for the
 possible set of essential residues that might underpin NAP-histone binding
 (Figs. <a href="/pmc/articles/PMC2665062/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Figs.1<em>d</em></span></span><span>1<em>d</em></span></a> and
 <a href="/pmc/articles/PMC2665062/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and7).</span></span><span>7</span></a>). Several of these conserved
 residues lie in and around the characteristic cavity formed by the PfNapL
 dimer, which has been previously implicated in histone recognition in both
 yNAP-1 and hSET (<a href="#ref15" rid="ref15" class=" bibr popnode">15</a>,
 <a href="#ref16" rid="ref16" class=" bibr popnode">16</a>). Distribution of these
 conserved surface residues on the convex region of PfNapL dimer illustrates
 their proximity to and overlap with the alternate stretch of acidic residues
 highlighted by our electrostatic potential analysis.</p><p>It is known that the &#x003b2; subdomain present in yNAP-1, hSET, and PfNapL
 is a highly conserved feature of histone chaperones
 (<a href="/pmc/articles/PMC2665062/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2<em>a</em></span></a>). The
 superimposition of the &#x003b2; subdomain of PfNapL onto histone chaperone Asf1
 revealed three conserved surface residues on PfNapL (conserved in all NAPs)
 that are suggested to be important for interaction with H3-H4 dimer based upon
 the Asf1 and H3-H4 dimer interactions
 (<a href="#ref21" rid="ref21" class=" bibr popnode">21</a>)
 (<a href="/pmc/articles/PMC2665062/figure/fig8/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig8" rid-ob="ob-fig8" co-legend-rid="lgnd_fig8"><span>Fig. 8, <em>a-d</em></span></a>, and Fig.
 S6). Interestingly, the three residues identified in Asf-histone complex map
 to a region of PfNapL that is not implicated by either hSET mutagenesis or by
 our mapping of conserved, exposed residues in NAPs. This suggests that Asf1
 and NAPs may recognize histones differently. Further, modeling of PfNapL based
 on the Asf1-histone complex crystal structure clearly suggests three target
 residues that are likely to participate in H3-H4 recognition. Based on these
 structural insights, we mutated two of the three residues (the third one is an
 alanine in any case) and confirmed the idea that their alteration to alanine
 reduces binding of PfNapL to H3-H4 tetramer significantly
 (<a href="/pmc/articles/PMC2665062/figure/fig8/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig8" rid-ob="ob-fig8" co-legend-rid="lgnd_fig8"><span>Fig. 8<em>d</em></span></a>).</p><p></p><p class="p p-last">In summary, our studies provide further structural insights into histone
 chaperones from <em>P. falciparum</em> and highlight key regions of
 differences between plasmodial NAPs and their counterparts. Although further
 dissection of the modes of histone recognition by NAPs will be required to
 gain a complete understanding of the mechanism of histone binding and
 transport by nucleosome assembly proteins, it is already clear that the
 parasite NAPs are sufficiently different from their counterparts in other
 species. Our genetic-structural-functional data on PfNapL suggests PfNapL as a
 possible new focus for development of antimalarials.</p></div><div id="sec-a.p.g" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="sec-a.p.gtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>[Supplemental Data]</strong>

<div class="sup-box half_rhythm" id="media-a.p.g.b.b"><a href="/pmc/articles/PMC2665062/bin/M808633200_index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>

</div></div><div id="ack-a.q.a" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.q.atitle">Acknowledgments</h2><div class="sec"><p>We thank Prof. Alan Cowman for support and for the generous gift of
 reagents.</p></div></div><div id="notes-a.o.c" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.o.ctitle">Notes</h2><p class="fn-group"><p class="fn sec" id="fn-a.o.c.a.a"><em>The atomic coordinates and structure factors (code 3FS3) have been
 deposited in the Protein Data Bank, Research Collaboratory for Structural
 Bioinformatics, Rutgers University, New Brunswick, NJ
 (<a href="http://www.rcsb.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://www.rcsb.org/</a>).</em></p><p class="fn sec" id="fn1"><sup>*</sup>This work was supported in part by a grant from the
 Department of Biotechnology, Govt. of India.
 The x-ray facility at the International Centre for Genetic
 Engineering and Biotechnology, New Delhi, is funded by the
 Wellcome Trust. The costs
 of publication of this article were defrayed in part by the payment of page
 charges. This article must therefore be hereby marked
 &#x0201c;<em>advertisement</em>&#x0201d; in accordance with 18 U.S.C. Section 1734
 solely to indicate this fact.</p><p class="fn sec" id="fn3"><sup></sup>The on-line version of this article (available at
 <a href="http://www.jbc.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://www.jbc.org</a>)
 contains supplemental Figs. S1-S6 and Table S1.</p><p class="fn sec" id="fn2"><sup></sup><em>Author's Choice</em>&#x02014;Final version full
 access.</p></p></div><div id="fn-group-a.q.c" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.q.ctitle">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="fn7"><sup>5</sup>The abbreviations used are: NAP, nucleosome assembly protein; yNAP-1, yeast
 NAP-1; hSET, human SET; SAD, single anomalous dispersion; NES, nuclear export
 signal; DAPI, 4&#x02032;,6-diamidino-2-phenylindole; GFP, green fluorescent
 protein; ELISA, enzyme-linked immunosorbent assay; PBS, phosphate-buffered
 saline; NLS, nuclear localization signal; r.m.s., root mean square.</p></div></div><div id="ref-list-a.q.b" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.q.btitle">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="ref1">1. <span class="citation">Luger, K., Maeder, A. W., Richmond, R. K., Sargent, D. F., and
 Richmond, T. J. (1997) <span class="ref-journal">Nature</span>
 <span class="ref-vol">389</span>
 251-259 [<a href="https://pubmed.ncbi.nlm.nih.gov/9305837" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;volume=389&amp;publication_year=1997&amp;pages=251&amp;pmid=9305837&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref2">2. <span class="citation">Krude, T. (1999) <span class="ref-journal">Eur. J.
 Biochem.</span> <span class="ref-vol">263</span>
 1-5 [<a href="https://pubmed.ncbi.nlm.nih.gov/10429179" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.&#10;+Biochem.&amp;volume=263&amp;publication_year=1999&amp;pages=1&amp;pmid=10429179&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref3">3. <span class="citation">Eitoku, M., Sato, L., Senda, T., and Horikoshi, M.
 (2008) <span class="ref-journal">Cell. Mol. Life. Sci.</span>
 <span class="ref-vol">65</span> 414-444 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC11131869/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17955179" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell.+Mol.+Life.+Sci.&amp;volume=65&amp;publication_year=2008&amp;pages=414&amp;pmid=17955179&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref4">4. <span class="citation">Park, Y., Chodaparambil, J. V., Bao, Y., McBryant, S. J., and
 Luger, K. (2005) <span class="ref-journal">J. Biol. Chem.</span>
 <span class="ref-vol">280</span>
 1817-1825 [<a href="https://pubmed.ncbi.nlm.nih.gov/15516689" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;volume=280&amp;publication_year=2005&amp;pages=1817&amp;pmid=15516689&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref5">5. <span class="citation">Dutta, S., Akey, I. V., Dingwall, C., Hartman, K. L., Laue, T.,
 Nolte, R. T., Head, J. F., and Akey, C. W. (2001) <span class="ref-journal">Mol.
 Cell.</span> <span class="ref-vol">8</span>
 841-853 [<a href="https://pubmed.ncbi.nlm.nih.gov/11684019" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.&#10;+Cell.&amp;volume=8&amp;publication_year=2001&amp;pages=841&amp;pmid=11684019&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref6">6. <span class="citation">Jackson, V., Shires, A., Tanphaichitr, N., and Chalkley, R.
 (1976) <span class="ref-journal">J. Mol. Biol.</span>
 <span class="ref-vol">104</span>
 471-483 [<a href="https://pubmed.ncbi.nlm.nih.gov/950668" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Mol.+Biol.&amp;volume=104&amp;publication_year=1976&amp;pages=471&amp;pmid=950668&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref7">7. <span class="citation">Annunziato, A. T., and Seale, R. L. (1983)
 <span class="ref-journal">Mol. Cell. Biochem.</span> <span class="ref-vol">55</span>
 99-112 [<a href="https://pubmed.ncbi.nlm.nih.gov/6355815" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Biochem.&amp;volume=55&amp;publication_year=1983&amp;pages=99&amp;pmid=6355815&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref8">8. <span class="citation">Koning, L. D., Corpet, A., Haber, J. E., and Almouzni, G.
 (2007) <span class="ref-journal">Nat. Struct. Mol. Biol.</span>
 <span class="ref-vol">14</span>
 997-1007 [<a href="https://pubmed.ncbi.nlm.nih.gov/17984962" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Struct.+Mol.+Biol.&amp;volume=14&amp;publication_year=2007&amp;pages=997&amp;pmid=17984962&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref9">9. <span class="citation">Rodriguez, P., Munroe, D., Prawitt, D., Chu, L. L., Bric, E., Kim,
 J., Reid, L. H., Davies, C., Nakagama, H., Loebbert, R., Winterpacht, A.,
 Petruzzi, M., Higgins, M. J., Nowak, N., Evans, G., Shows, T., Weissman, B.
 E., Zabel, B., Housman, D. E., and Pelletier, J. (1997)
 <span class="ref-journal">Genomics</span> <span class="ref-vol">44</span>
 253-265 [<a href="https://pubmed.ncbi.nlm.nih.gov/9325046" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genomics&amp;volume=44&amp;publication_year=1997&amp;pages=253&amp;pmid=9325046&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref10">10. <span class="citation">Lankenau, S., Barnickel, T., Marhold, J., Lyko, F., Mechler, B. M.,
 and Lankenau, D. (2003) <span class="ref-journal">Genetics</span>
 <span class="ref-vol">163</span>
 611-623 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1462439/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12618400" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;volume=163&amp;publication_year=2003&amp;pages=611&amp;pmid=12618400&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref11">11. <span class="citation">Tyler, J. K., Adams, C. R., Chen, S. R., Kobayashi, R., Kamakaka,
 R. T., and Kadonaga, J. T. (1999) <span class="ref-journal">Nature</span>
 <span class="ref-vol">402</span>
 555-560 [<a href="https://pubmed.ncbi.nlm.nih.gov/10591219" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;volume=402&amp;publication_year=1999&amp;pages=555&amp;pmid=10591219&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref12">12. <span class="citation">Snow, R. W., Guerra, C. A., Abdisalan, M. N., Hla, Y. M., and
 Simon, I. H. (2005) <span class="ref-journal">Nature</span>
 <span class="ref-vol">434</span>
 214-217 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3128492/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15759000" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;volume=434&amp;publication_year=2005&amp;pages=214&amp;pmid=15759000&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref13">13. <span class="citation">Chandra, B. R., Olivieri, A., Silvestrini, F., Alano, P., and
 Sharma, A. (2005) <span class="ref-journal">Mol. Biochem.
 Parasitol.</span> <span class="ref-vol">142</span>
 237-247 [<a href="https://pubmed.ncbi.nlm.nih.gov/15899528" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Biochem.&#10;+Parasitol.&amp;volume=142&amp;publication_year=2005&amp;pages=237&amp;pmid=15899528&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref14">14. <span class="citation">Navadgi, M. V., Chandra, B. R., Mishra, P. C., and Sharma, A.
 (2006) <span class="ref-journal">J. Biol. Chem.</span>
 <span class="ref-vol">281</span>
 16978-16984 [<a href="https://pubmed.ncbi.nlm.nih.gov/16638744" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;volume=281&amp;publication_year=2006&amp;pages=16978&amp;pmid=16638744&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref15">15. <span class="citation">Park, Y.-J., and Luger, K. (2006) <span class="ref-journal">Proc.
 Natl. Acad. Sci. U. S. A.</span> <span class="ref-vol">103</span>
 1248-1253 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1345705/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16432217" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.&#10;+Natl.+Acad.+Sci.+U.+S.+A.&amp;volume=103&amp;publication_year=2006&amp;pages=1248&amp;pmid=16432217&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref16">16. <span class="citation">Muto, S., Senda, M., Akai, Y., Sato, L., Suzuki, T., Nagai, R.,
 Senda, T., and Horikoshi, M. (2007) <span class="ref-journal">Proc. Natl. Acad.
 Sci. U. S. A.</span> <span class="ref-vol">104</span>
 4285-4290 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1810507/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17360516" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.&#10;+Sci.+U.+S.+A.&amp;volume=104&amp;publication_year=2007&amp;pages=4285&amp;pmid=17360516&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref17">17. <span class="citation">Selth, L., and Svejstrup, J. Q. (2007) <span class="ref-journal">J.
 Biol. Chem.</span> <span class="ref-vol">282</span>
 12358-12362 [<a href="https://pubmed.ncbi.nlm.nih.gov/17344218" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&#10;+Biol.+Chem.&amp;volume=282&amp;publication_year=2007&amp;pages=12358&amp;pmid=17344218&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref18">18. <span class="citation">Tang, Y., Meeth, K., Jiang, E., Luo, C., and Marmorstein, R.
 (2008) <span class="ref-journal">Proc. Natl. Acad. Sci. U. S. A.</span>
 <span class="ref-vol">105</span>
 12206-12211 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2527890/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18723682" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.+S.+A.&amp;volume=105&amp;publication_year=2008&amp;pages=12206&amp;pmid=18723682&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref19">19. <span class="citation">Berndsen, C. E., Tsubota, T., Lindner, S. E., Lee, S., Holton, J.
 M., Kaufman, P. D., Keck, J. L., and Denu, J. M. (2008)
 <span class="ref-journal">Nat. Struct. Mol. Biol.</span>
 <span class="ref-vol">15</span> 948-956 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2678805/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19172748" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Struct.+Mol.+Biol.&amp;volume=15&amp;publication_year=2008&amp;pages=948&amp;pmid=19172748&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref20">20. <span class="citation">English, C. M., Adkins, M. W., Carson, J. J., Churchill, M. E., and
 Tyler, J. K. (2006) <span class="ref-journal">Cell</span>
 <span class="ref-vol">127</span>
 458-460 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2981792/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17081973" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;volume=127&amp;publication_year=2006&amp;pages=458&amp;pmid=17081967&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref21">21. <span class="citation">Natsume, R., Eitoku, M., Akai, Y., Sano, N., Horikoshi, M., and
 Senda, T. (2007) <span class="ref-journal">Nature</span>
 <span class="ref-vol">446</span>
 338-341 [<a href="https://pubmed.ncbi.nlm.nih.gov/17293877" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;volume=446&amp;publication_year=2007&amp;pages=338&amp;pmid=17293877&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref22">22. <span class="citation">Otwinowski, Z., and Minor, W. (1997)
 <span class="ref-journal">Methods Enzymol.</span> <span class="ref-vol">276</span>
 307-326 [<a href="https://pubmed.ncbi.nlm.nih.gov/27754618" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Enzymol.&amp;volume=276&amp;publication_year=1997&amp;pages=307&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref23">23. <span class="citation">Adams, P. D., Grosse-Kunstleve, R. W., Hung, L. W., Ioerger, T. R.,
 McCoy, A. J., Moriarty, N. W., Read, R. J., Sacchettini, J. C., Sauter, N. K.,
 and Terwilliger, T. C. (2002) <span class="ref-journal">Acta Crystallogr. Sect.
 D</span> <span class="ref-vol">58</span>
 1948-1954 [<a href="https://pubmed.ncbi.nlm.nih.gov/12393927" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Crystallogr.+Sect.&#10;+D&amp;volume=58&amp;publication_year=2002&amp;pages=1948&amp;pmid=12393927&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref24">24. <span class="citation">Emsley, P., and Cowtan, K. (2004) <span class="ref-journal">Acta
 Crystallogr. Sect. D</span> <span class="ref-vol">60</span>
 2126-2132 [<a href="https://pubmed.ncbi.nlm.nih.gov/15572765" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta&#10;+Crystallogr.+Sect.+D&amp;volume=60&amp;publication_year=2004&amp;pages=2126&amp;pmid=15572765&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref25">25. <span class="citation">Brunger, A. T., Adams, P. D., Clore, G. M., Gros, P.,
 Grosse-Kunstleve, R. W., Kuszewski, J.-S., Jiang, J., Nilges, N., Pannu, N.
 S., Read, R. J., Rice, L. M., Simonson, T., and Warren, G. L.
 (1998) <span class="ref-journal">Acta Crystallogr. Sect. D</span>
 <span class="ref-vol">54</span> 905-921 [<a href="https://pubmed.ncbi.nlm.nih.gov/9757107" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Crystallogr.+Sect.+D&amp;volume=54&amp;publication_year=1998&amp;pages=905&amp;pmid=9757107&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref26">26. <span class="citation">Laskowski, R. A., MacArthur, M. W., Moss, D. S., and Thornton, J.
 M. (1993) <span class="ref-journal">J. Appl. Crystallogr.</span>
 <span class="ref-vol">26</span> 283-291 <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Appl.+Crystallogr.&amp;volume=26&amp;publication_year=1993&amp;pages=283&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref27">27. <span class="citation">Pettersen, E. F., Goddard, T. D., Huang, C. C., Couch, G. S.,
 Greenblatt, D. M., Meng, E. C., and Ferrin, T. E. (2004)
 <span class="ref-journal">J. Comp. Chem.</span> <span class="ref-vol">25</span>
 1605-1612 [<a href="https://pubmed.ncbi.nlm.nih.gov/15264254" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Comp.+Chem.&amp;volume=25&amp;publication_year=2004&amp;pages=1605&amp;pmid=15264254&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref28">28. <span class="citation">Kleywegt, G. J. (1996) <span class="ref-journal">Acta Crystallogr.
 Sect. D</span> <span class="ref-vol">52</span>
 842-857 [<a href="https://pubmed.ncbi.nlm.nih.gov/15299650" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Crystallogr.&#10;+Sect.+D&amp;volume=52&amp;publication_year=1996&amp;pages=842&amp;pmid=15299650&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref29">29. <span class="citation">Nicholls, A., Sharp, K. A., and Honig, B. (1991)
 <span class="ref-journal">Proteins Struct. Funct. Genet.</span>
 <span class="ref-vol">11</span> 281-296 [<a href="https://pubmed.ncbi.nlm.nih.gov/1758883" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins+Struct.+Funct.+Genet.&amp;volume=11&amp;publication_year=1991&amp;pages=281&amp;pmid=1758883&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref30">30. <span class="citation">Trager, W., and Jensen, J. B. (1978)
 <span class="ref-journal">Nature</span> <span class="ref-vol">273</span>
 621-622 [<a href="https://pubmed.ncbi.nlm.nih.gov/351412" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;volume=273&amp;publication_year=1978&amp;pages=621&amp;pmid=351412&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref31">31. <span class="citation">Maier, A. G., Rug, M., Brown, M. T., O'Neill, M., Chakravorty, S.,
 Szestak, T., Chesson, J., Wu, Y., Hughes, K., Coppel, R., Newbold, C., Beeson,
 J., Craig, A., Crabb, B. S., and Cowman, A. F. (2008)
 <span class="ref-journal">Cell</span> <span class="ref-vol">134</span>
 48-61 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2568870/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18614010" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;volume=134&amp;publication_year=2008&amp;pages=48&amp;pmid=18614010&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref32">32. <span class="citation">Crabb, B. S., Rug, M., Gilberger, T., Thompson, J. K., Triglia, T.,
 Maier, A. G., and Cowman, A. F. (2004) <span class="ref-journal">Methods Mol.
 Biol.</span> <span class="ref-vol">270</span>
 263-276 [<a href="https://pubmed.ncbi.nlm.nih.gov/15153633" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Mol.&#10;+Biol.&amp;volume=270&amp;publication_year=2004&amp;pages=263&amp;pmid=15153633&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref33">33. <span class="citation">Fidock, D. A., and Wellems, T. E. (1997)
 <span class="ref-journal">Proc. Natl. Acad. Sci. U. S. A.</span>
 <span class="ref-vol">94</span>
 10931-10936 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC23535/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9380737" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.+S.+A.&amp;volume=94&amp;publication_year=1997&amp;pages=10931&amp;pmid=9380737&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref34">34. <span class="citation">Rug, M., Prescott, S. W., Fernandez, K. M., Cooke, B. M., and
 Cowman, A. F. (2006) <span class="ref-journal">Blood</span>
 <span class="ref-vol">108</span>
 370-378 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1895844/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16507777" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;volume=108&amp;publication_year=2006&amp;pages=370&amp;pmid=16507777&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref35">35. <span class="citation">Tanaka, Y., Tawaramoto-Sasanuma, M., Kawaguchi, S., Ohta, T., Yoda,
 K., Kurumizaka, H., and Yokoyama, S. (2004) <span class="ref-journal">Methods
 (San Diego)</span> <span class="ref-vol">33</span>
 3-11 [<a href="https://pubmed.ncbi.nlm.nih.gov/15039081" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods&#10;+(San+Diego)&amp;volume=33&amp;publication_year=2004&amp;pages=3&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref36">36. <span class="citation">Nagata, K., Saito, S., Okuwaki, M., Kawase, H., Furuya, A., Kusano,
 A., Hanai, N., Okuda, A., and Kikuchi, A. (1998) <span class="ref-journal">Exp.
 Cell Res.</span> <span class="ref-vol">240</span>
 274-281 [<a href="https://pubmed.ncbi.nlm.nih.gov/9597000" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Exp.&#10;+Cell+Res.&amp;volume=240&amp;publication_year=1998&amp;pages=274&amp;pmid=9597000&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref37">37. <span class="citation">Daganzo, S. M., Erzberger, J. P., Lam, W. M., Skordalakes, E.,
 Zhangm, R., Franco, A. A., Brill, S. J., Adams, P. D., Berger, J. M., and
 Kaufman, P. D. (2003) <span class="ref-journal">Curr. Biol.</span>
 <span class="ref-vol">13</span>
 2148-2158 [<a href="https://pubmed.ncbi.nlm.nih.gov/14680630" ref="reftype=pubmed&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Biol.&amp;volume=13&amp;publication_year=2003&amp;pages=2148&amp;pmid=14680630&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2665062&amp;issue-id=177745&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">The Journal of Biological Chemistry</span> are provided here courtesy of <strong>American Society for Biochemistry and Molecular Biology</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC2665062/pdf/10076.pdf" class="int-view">PDF (1.2M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/2665062/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/2665062/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="2665062" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2665062%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="usYZED0HvOvfKNEyWc3t8ek47YsEs3PqrtxquFfjPlAnaYSwPPH8HLOHTpP0rfVd">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2665062%2F&amp;text=Crystal%20Structure%20of%20Malaria%20Parasite%20Nucleosome%20Assembly%0A%20Protein%3A%20DISTINCT%20MODES%20OF%20PROTEIN%20LOCALIZATION%20AND%20HISTONE%0A%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2665062%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2665062/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC2665062/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/19176479/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC2665062/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/19176479/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC2665062/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/2665062/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC2665062/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
