
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Comparison of Protein Immunoprecipitation-Multiple
Reaction Monitoring with ELISA for Assay of Biomarker Candidates in
Plasma - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="322C300B24F3920500003F904DE61F2A.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="acssd" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC3864264/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3864264/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Journal of Proteome Research" /><meta name="citation_title" content="Comparison of Protein Immunoprecipitation-Multiple&#10;Reaction Monitoring with ELISA for Assay of Biomarker Candidates in&#10;Plasma" /><meta name="citation_author" content="De Lin" /><meta name="citation_author_institution" content="Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States" /><meta name="citation_author_institution" content="Jim Ayers Institute for Precancer Detection and Diagnosis, Vanderbilt-Ingram Cancer Center, Vanderbilt University School of Medicine, Nashville, Tennessee, 37232, United States" /><meta name="citation_author" content="William E. Alborn" /><meta name="citation_author_institution" content="Jim Ayers Institute for Precancer Detection and Diagnosis, Vanderbilt-Ingram Cancer Center, Vanderbilt University School of Medicine, Nashville, Tennessee, 37232, United States" /><meta name="citation_author" content="Robbert J. C. Slebos" /><meta name="citation_author_institution" content="Department of Cancer Biology, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States" /><meta name="citation_author_institution" content="Jim Ayers Institute for Precancer Detection and Diagnosis, Vanderbilt-Ingram Cancer Center, Vanderbilt University School of Medicine, Nashville, Tennessee, 37232, United States" /><meta name="citation_author" content="Daniel C. Liebler" /><meta name="citation_author_institution" content="Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States" /><meta name="citation_author_institution" content="Jim Ayers Institute for Precancer Detection and Diagnosis, Vanderbilt-Ingram Cancer Center, Vanderbilt University School of Medicine, Nashville, Tennessee, 37232, United States" /><meta name="citation_publication_date" content="2013/12/12" /><meta name="citation_issue" content="12" /><meta name="citation_volume" content="12" /><meta name="citation_firstpage" content="5996" /><meta name="citation_doi" content="10.1021/pr400877e" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3864264/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC3864264/" /><meta name="citation_pmid" content="24224610" /><meta name="DC.Title" content="Comparison of Protein Immunoprecipitation-Multiple&#10;Reaction Monitoring with ELISA for Assay of Biomarker Candidates in&#10;Plasma" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="American Chemical Society" /><meta name="DC.Contributor" content="De Lin" /><meta name="DC.Contributor" content="William E. Alborn" /><meta name="DC.Contributor" content="Robbert J. C. Slebos" /><meta name="DC.Contributor" content="Daniel C. Liebler" /><meta name="DC.Date" content="2013 Dec 6" /><meta name="DC.Identifier" content="10.1021/pr400877e" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Comparison of Protein Immunoprecipitation-Multiple&#10;Reaction Monitoring with ELISA for Assay of Biomarker Candidates in&#10;Plasma" /><meta property="og:type" content="article" /><meta property="og:description" content="Quantitative analysis of protein biomarkers in plasma is typically&#10;done by ELISA, but this method is limited by the availability of high-quality&#10;antibodies. An alternative approach is protein immunoprecipitation&#10;combined with multiple reaction monitoring ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3864264/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC3864264/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22ACS%20AuthorChoice%22[filter]" class="navlink">ACS AuthorChoice</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC3864264
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC3864264/pdf/pr400877e.pdf" class="int-view">PDF (1.5M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/3864264/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/3864264/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="3864264" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3864264%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="Fu1JBMjTbWZogVYANplMZWLqx7uBwsoBH4LL1H1Ds67blK9Vj3BpeRzc8t5FF1UN">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3864264%2F&amp;text=Comparison%20of%20Protein%20Immunoprecipitation-Multiple%0AReaction%20Monitoring%20with%20ELISA%20for%20Assay%20of%20Biomarker%20Candidates%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3864264%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3864264/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC3864264/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/24224610/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC3864264/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/24224610/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC3864264/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/3864264/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22ACS%20AuthorChoice%22[filter]" class="navlink">ACS AuthorChoice</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC3864264
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><a href="//doi.org/10.1021%2Fpr400877e" title="Link to Publisher's site" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"><img src="/corehtml/pmc/pmcgifs/logo-acssd.png" alt="Logo of acssd"><!-- --></img></a></div>
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">J Proteome Res.</span> 2013 Dec 6; 12(12): 5996–6003. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2013 Nov 13. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1021%2Fpr400877e" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1021/pr400877e</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3864264</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/24224610">24224610</a></div></div></div><h1 class="content-title">Comparison of Protein Immunoprecipitation-Multiple
Reaction Monitoring with ELISA for Assay of Biomarker Candidates in
Plasma</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Lin%20D%5BAuthor%5D" id="ath1" class="affpopup" co-rid="_co_idm140480118784192" co-class="co-affbox">De Lin</a>,<sup>&#x02020;</sup><sup>&#x000a7;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Alborn%20WE%5BAuthor%5D" id="ath2" class="affpopup" co-rid="_co_idm140480115938416" co-class="co-affbox">William E. Alborn</a>,<sup>&#x000a7;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Slebos%20RJ%5BAuthor%5D" id="ath3" class="affpopup" co-rid="_co_idm140480115936752" co-class="co-affbox">Robbert J. C. Slebos</a>,<sup>&#x02021;</sup><sup>&#x000a7;</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Liebler%20DC%5BAuthor%5D" id="ath4" class="affpopup" co-rid="_co_idm140480158706992" co-class="co-affbox">Daniel C. Liebler</a><sup></sup><sup>&#x02020;</sup><a href="#cor1" rid="cor1" class=" other">*</a><sup>&#x000a7;</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140480118784192"><h3 class="no_margin">De Lin</h3><p><sup>&#x02020;</sup>Department
of Biochemistry, Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</p><p><sup>&#x000a7;</sup>Jim
Ayers Institute for Precancer Detection and Diagnosis, Vanderbilt-Ingram
Cancer Center, Vanderbilt University School
of Medicine, Nashville, Tennessee,  37232, United
States</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Lin%20D%5BAuthor%5D">De Lin</a></div></div><div id="_co_idm140480115938416"><h3 class="no_margin">William E. Alborn</h3><p><sup>&#x000a7;</sup>Jim
Ayers Institute for Precancer Detection and Diagnosis, Vanderbilt-Ingram
Cancer Center, Vanderbilt University School
of Medicine, Nashville, Tennessee,  37232, United
States</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Alborn%20WE%5BAuthor%5D">William E. Alborn</a></div></div><div id="_co_idm140480115936752"><h3 class="no_margin">Robbert J. C. Slebos</h3><p><sup>&#x02021;</sup>Department
of Cancer Biology, Vanderbilt University
School of Medicine, Nashville, Tennessee 37232, United States</p><p><sup>&#x000a7;</sup>Jim
Ayers Institute for Precancer Detection and Diagnosis, Vanderbilt-Ingram
Cancer Center, Vanderbilt University School
of Medicine, Nashville, Tennessee,  37232, United
States</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Slebos%20RJ%5BAuthor%5D">Robbert J. C. Slebos</a></div></div><div id="_co_idm140480158706992"><h3 class="no_margin">Daniel C. Liebler</h3><p><sup>&#x02020;</sup>Department
of Biochemistry, Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</p><p><sup>&#x000a7;</sup>Jim
Ayers Institute for Precancer Detection and Diagnosis, Vanderbilt-Ingram
Cancer Center, Vanderbilt University School
of Medicine, Nashville, Tennessee,  37232, United
States</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Liebler%20DC%5BAuthor%5D">Daniel C. Liebler</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140480162240080_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140480162240080_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140480162240080_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140480162240080_ai" style="display:none"><div class="fm-affl" id="aff1"><sup>&#x02020;</sup>Department
of Biochemistry, Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</div><div class="fm-affl" id="aff2"><sup>&#x02021;</sup>Department
of Cancer Biology, Vanderbilt University
School of Medicine, Nashville, Tennessee 37232, United States</div><div class="fm-affl" id="aff3"><sup>&#x000a7;</sup>Jim
Ayers Institute for Precancer Detection and Diagnosis, Vanderbilt-Ingram
Cancer Center, Vanderbilt University School
of Medicine, Nashville, Tennessee,  37232, United
States</div><div id="cor1"><sup>*</sup>Telephone.: 615-322-3063. Fax: 615-936-1001. E-mail: <a href="mailto:dev@null" data-email="ude.tlibrednav@relbeil.leinad" class="oemail">ude.tlibrednav@relbeil.leinad</a>.</div></div><div class="fm-article-notes hide half_rhythm" id="idm140480162240080_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2013 Aug 27</div></div><div class="permissions half_rhythm hide" id="idm140480162240080_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  &#x000a9; 2013 American Chemical Society</div><div class="license half_rhythm"><a href="http://pubs.acs.org/page/policy/authorchoice_termsofuse.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">Terms of Use</a></div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">pr400877e_si_001.pdf</div><div><a href="/pmc/articles/PMC3864264/bin/pr400877e_si_001.pdf" data-ga-action="click_feat_suppl">pr400877e_si_001.pdf</a><span style="color:gray"> (2.8M)</span></div><div class="small guid">GUID: 52DFF305-ABFA-4FA6-9AF3-27495CFF0E68</div></div></dd></dl></div><div id="abstract-a.i.b.n" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.i.b.ntitle">Abstract</h2><!--article-meta--><div><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140480159614368"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pr-2013-00877e_0004.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is pr-2013-00877e_0004.jpg" src="/pmc/articles/PMC3864264/bin/pr-2013-00877e_0004.jpg" /></div><p></p><p class="p p-last">Quantitative analysis of protein biomarkers in plasma is typically
done by ELISA, but this method is limited by the availability of high-quality
antibodies. An alternative approach is protein immunoprecipitation
combined with multiple reaction monitoring mass spectrometry (IP-MRM).
We compared IP-MRM to ELISA for the analysis of six colon cancer biomarker
candidates (metalloproteinase inhibitor 1 (TIMP1), cartilage oligomeric
matrix protein (COMP), thrombospondin-2 (THBS2), endoglin (ENG), mesothelin
(MSLN) and matrix metalloproteinase-9 (MMP9)) in plasma from colon
cancer patients and noncancer controls. Proteins were analyzed by
multiplex immunoprecipitation from plasma with the ELISA capture antibodies,
further purified by SDS-PAGE, digested and analyzed by stable isotope
dilution MRM. IP-MRM provided linear responses (<em>r</em> = 0.978&#x02013;0.995) between 10 and 640 ng/mL for the target proteins
spiked into a &#x0201c;mock plasma&#x0201d; matrix consisting of 60
mg/mL bovine serum albumin. Measurement variation (coefficient of
variation at the limit of detection) for IP-MRM assays ranged from
2.3 to 19%, which was similar to variation for ELISAs of the same
samples. IP-MRM and ELISA measurements for all target proteins except
ENG were highly correlated (<em>r</em> = 0.67&#x02013;0.97).
IP-MRM with high-quality capture antibodies thus provides an effective
alternative method to ELISA for protein quantitation in biological
fluids.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">immunoprecipitation, MRM, plasma biomarkers, biomarker verification, colon cancer</span></div></div><div id="body-a.j" class="tsec sec"><h2 class="headless nomenu"></h2><p id="sec1" class="p p-first">Quantitative analysis of protein
biomarkers is one of the most challenging tasks in biomedical research.<sup><a href="#ref1" rid="ref1" class=" bibr popnode">1</a></sup> Enzyme-linked immunosorbent assay (ELISA) is
widely used for protein quantitation in human serum or plasma owing
to its high sensitivity and throughput. However, the availability
of high-quality ELISAs for biomarker candidates is limited, and the
performance characteristics of many commercially marketed ELISAs are
poorly documented or unknown.<sup><a href="#ref2" rid="ref2" class=" bibr popnode">2</a></sup> Development
of ELISAs is also expensive and time-consuming. The limitations of
ELISA, combined with the large numbers of biomarker candidates emerging
from genomic and proteomic discovery studies, have created a need
for alternative means of targeted protein quantitation.<sup><a href="#ref1" rid="ref1" class=" bibr popnode">1</a></sup></p><p>Multiple reaction monitoring (MRM) mass spectrometry has emerged
as a versatile platform for systematic development of targeted protein
assays and which can serve as an alternative to ELISA in biomarker
research.<sup><a href="#ref3" rid="ref3" class=" bibr popnode">3</a>,<a href="#ref4" rid="ref4" class=" bibr popnode">4</a></sup> MRM assays target sequence-specific tandem
MS fragmentations of proteotypic peptides, thereby providing highly
selective measurements for distinct proteins. Without fractionation
or enrichment strategies, MRM assays allow for the quantitation of
protein in the low &#x003bc;g/mL or high ng/mL concentration range,<sup><a href="#ref5" rid="ref5" class=" bibr popnode">5</a>&#x02212;<a href="#ref7" rid="ref7" class=" bibr popnode">7</a></sup> whereas immunoaffinity depletion of abundant blood proteins and
minimal protein fractionation can enable quantitation in the low ng/mL
range.<sup><a href="#ref8" rid="ref8" class=" bibr popnode">8</a>&#x02212;<a href="#ref10" rid="ref10" class=" bibr popnode">10</a></sup> A recently described method called PRISM combined
targeted peptide-level preselection with MRM to achieve high sensitivity
measurements without antibody capture.<sup><a href="#ref2" rid="ref2" class=" bibr popnode">2</a></sup></p><p>Immunoaffinity capture of intact proteins or their peptides after
digestion can dramatically enhance the sensitivity of MRM assays.
Anderson and colleagues introduced an immuno-MRM assay approach (Stable
Isotope Standards with Capture by Antipeptide Antibodies; SISCAPA),
in which proteotypic tryptic peptides and their corresponding spiked
stable isotope-labeled internal standards are captured by antibodies
raised against the peptides.<sup><a href="#ref11" rid="ref11" class=" bibr popnode">11</a></sup> This approach
has been extensively developed by several laboratories<sup><a href="#ref10" rid="ref10" class=" bibr popnode">10</a>,<a href="#ref12" rid="ref12" class=" bibr popnode">12</a>&#x02212;<a href="#ref14" rid="ref14" class=" bibr popnode">14</a></sup> and used to systematically develop and implement
targeted assays for candidate biomarkers<sup><a href="#ref15" rid="ref15" class=" bibr popnode">15</a>,<a href="#ref16" rid="ref16" class=" bibr popnode">16</a></sup> and implement, as a prototype, clinical assay for thyroglobulin.<sup><a href="#ref17" rid="ref17" class=" bibr popnode">17</a></sup></p><p>An alternate approach was described by Berna et al., who used immunoaffinity
enrichment of intact proteins followed by digestion and MRM to quantify
protein biomarkers of cardiovascular disease.<sup><a href="#ref18" rid="ref18" class=" bibr popnode">18</a>,<a href="#ref19" rid="ref19" class=" bibr popnode">19</a></sup> Nicol et al.<sup><a href="#ref20" rid="ref20" class=" bibr popnode">20</a></sup> demonstrated that multiple
antibodies immobilized on hydrazide beads could simultaneously enrich
several candidate lung cancer biomarkers in serum for MRM measurements
in the low ng/mL range. Targeted quantitation can be extended to sequence
variant proteins using the same approach.<sup><a href="#ref21" rid="ref21" class=" bibr popnode">21</a></sup> Protein-capture-based immuno-MRM (IP-MRM) assays have been less
thoroughly explored than peptide-capture-based assays.</p><p class="p">We asked how the analytical performance of IP-MRM would compare
to that for ELISA. To address this question, we employed six commercially
available ELISAs to measure candidate biomarker proteins for colon
cancer in plasma of cancer patients and noncancer controls. We obtained
the capture antibodies used in the ELISAs from the manufacturer and
then configured immuno-MRM assays for the six proteins. Our analyses
provide the first reported comparison of an IP-MRM assay to ELISA
with the same samples and reagents.</p></div><div id="sec2" class="tsec sec"><h2 class="head no_bottom_margin" id="sec2title">Experimental Procedures</h2><div id="sec2.1" class="sec sec-first"><h3 id="sec2.1title">Chemicals and Reagents</h3><p class="p p-first-last">Trypsin (sequencing grade) was
purchased from Promega (Madison, WI). Isotope-labeled peptides were
obtained from New England Peptide (Gardner, MA) with either U&#x02013;<sup>13</sup>C<sub>6</sub>, U&#x02013;<sup>15</sup>N<sub>2</sub>-lysine
(+8 Da) or U&#x02013;<sup>13</sup>C<sub>6</sub>, U&#x02013;<sup>15</sup>N<sub>4</sub>-arginine (+10 Da) at the peptide C terminus. Chemical
purity was ranged from 95 to 99% and isotopic purity was greater than
99%. Peptide concentrations were benchmarked by amino acid analysis
performed by the supplier. TIMP1, THBS2, COMP, MSLN, ENG, MMP9 antibodies
and ELISA kits were purchased from R&#x00026;D Systems (Minneapolis, MN).
ELISAs were performed according to the manufacturer&#x02019;s recommendations.
TIMP1, ENG and MMP9 recombinant proteins were purchase from Sino Biological
(Beijing, China). THBS2, COMP and MSLN recombinant proteins were from
R&#x00026;D Systems. Identities of all recombinant proteins were confirmed
by SDS-PAGE (Figure S1A [<a href="#notes-1" rid="notes-1" class=" notes">Supporting Information</a> (SI)]) and tandem mass spectrometry (Figure S2&#x02013;S7 [<a href="#notes-1" rid="notes-1" class=" notes">SI</a>]) analysis of tryptic digests. All other chemical
reagents were purchased from commercial sources and were used without
further purification.</p></div><div id="sec2.2" class="sec"><h3 id="sec2.2title">Collection and Storage of Plasma</h3><p class="p p-first-last">Human plasma samples
were collected during surgery for either colon carcinoma or for inguinal
hernia repair in accordance with the Ayers Institute protocol at Vanderbilt
University Medical Center (IRB#110877). Samples from colon cancer
patients undergoing surgery at Vanderbilt University between August
2011 and June 2012 with a successful collection were used in the study.
Control samples were chosen from a larger group of inguinal hernia
repair patients who underwent surgery during the same time period.
Peripheral whole blood samples were collected preoperatively in EDTA
lavender top vacutainer tubes (BD Vacutainer, catalog number 366643)
and gently mixed by inverting the tube 8&#x02013;10 times. Plasma was
separated by centrifugation at 1500<em>g</em> for 10 min at
4 &#x000b0;C. Aliquots (0.2 mL) were taken and stored at &#x02212;80 &#x000b0;C
until needed.</p></div><div id="sec2.3" class="sec"><h3 id="sec2.3title">Antibody Immobilization</h3><p class="p p-first-last">Antibodies were immobilized
on aldehyde beads (Thermo Scientific, catalog number 26148) according
to the manufacturer&#x02019;s protocol with minor modifications. Briefly,
antibodies were dissolved in PBS buffer (0.01 M sodium phosphate,
0.15 sodium chloride, pH 7.2) and incubated with coupling resin and
75 &#x003bc;M sodium cyanoborohydride at room temperature on a rotator.
An aliquot was collected before and after binding for determination
of binding efficiency by protein bicinchoninic acid assay. After immobilization,
the active aldehyde sites on the resin were blocked with 1 M Tris
buffer and 75 &#x003bc;M sodium cyanoborohydride followed by several
washes with PBS to remove any nonbound antibody. After determining
the binding efficiency, the immobilized resins for all antibodies
were either combined or directly aliquoted such that &#x0223c;1 &#x003bc;g
of each immobilized antibody was used for each immunoprecipitation.</p></div><div id="sec2.4" class="sec"><h3 id="sec2.4title">Protein Capture and Sample Preparation for MRM</h3><p class="p p-first">Plasma
(50 &#x003bc;L) was diluted 5-fold with RIPA buffer containing a protease
inhibitor cocktail (Roche, catalog number 11873580001). Diluted plasma
was incubated with the immobilized antibody resin overnight at 4 &#x000b0;C
with gentle shaking. The resin was washed three times with 0.5 mL
RIPA buffer, and the bound proteins were eluted into 15 &#x003bc;L of
2X NuPAGE lithium dodecyl sulfate loading buffer (Invitrogen, Carsbad,
CA) containing 50 mM DTT by incubation at 95 &#x000b0;C for 5 min. The
eluted proteins then were loaded and separated by SDS-PAGE on a NuPAGE
Novex 10% Bis Tris mini gel (Invitrogen NP0301BOX). A protein molecular
weight standard (Precision Plus Protein Kaleidoscope Standard, Bio-Rad,
Hercules, CA) was loaded in one lane on each gel and used for estimation
of relative mass determination of captured proteins.</p><p class="p p-last">After electrophoresis
at a constant 180 V for 20 min, gels were washed three times with
deionized water, stained with SimplyBlue SafeStain (Invitrogen) for
1 h, and destained with deionized water at 4 &#x000b0;C overnight. From
each gel lane, fractions were taken to enable targeted analysis of
the target proteins. For TIMP1, a molecular weight fraction of 25&#x02013;37
kDa was collected. For analysis of the remaining five proteins, a
molecular weight range of 75&#x02013;200 kDa THBS2, COMP and MMP9 and
another of 37&#x02013;75 kDa for ENG and MSLN were excised from the
gel, cut into 1 mm cubes, and placed in 100 &#x003bc;L of 100 mM ammonium
bicarbonate. Samples were reduced with 5 &#x003bc;L of 100 mM DTT for
15 min at 50 &#x000b0;C and alkylated with 15 &#x003bc;L of 100 mM iodoacetamide
for 30 min at room temperature in the dark. Excess dye was removed
from gel slices with two exchanges of 100 &#x003bc;L 50% acetonitrile/50
mM ammonium bicarbonate and subsequently dehydrated with 100% acetonitrile.
The solvent was removed from the gel pieces under vacuum. The residue
was resuspended in 0.01 &#x003bc;g/&#x003bc;L MS grade trypsin (Promega,
Madison, WI) in 25 mM ammonium bicarbonate containing a standard mixture
of heavy isotope-labeled peptides for the analytes (20 fmol/peptide)
and incubated at 37 &#x000b0;C overnight. Peptides were extracted with
60% acetonitrile containing 1% formic acid, and then each fraction
was evaporated under vacuum. Samples were redissolved for MRM analysis
in 30 &#x003bc;L of 5% acetonitrile containing 0.1% formic acid.</p></div><div id="sec2.5" class="sec"><h3 id="sec2.5title">MRM Analysis</h3><p class="p p-first-last">MRM analyses were performed on a TSQ Vantage
triple quadrupole mass spectrometer (ThermoFisher Scientific, San
Jose, CA) equipped with an Eksigent Ultra nanoLC solvent delivery
system, microautosampler and a nanospray source. Sample peptides (3
&#x003bc;L injection volume) were loaded onto a 75 &#x003bc;m &#x000d7; 11
cm PicoFrit column (PF360-75-10-N-5, New Objective, Inc., Woburn,
MA) packed with 3 &#x003bc;m, ReproSil-Pur C18-AQ (Dr. Maisch, Ammerbuch-Entringen,
Germany). Liquid chromatography was carried out at a flow rate of
300 nL/min with a mobile phase consisting of 0.1% formic acid in either
HPLC grade water (solvent A) or 90% acetonitrile (solvent B). An elution
gradient was programmed from 97% A for 1 min, then increased to 7%
B over 4 min, 25% B over 15 min, 40% B over 7 min, 90% B by 40 min,
and then held at that composition for 10 min before returning to 97%
solvent A over 1 min. Mobile phase composition then was held at this
initial condition for 29 min prior to the next analysis. Instrument
parameters included Q2 gas 1.5 mTorr, scan width 0.005 Th, scan time
10 ms, and both Q1 and Q3 resolution fwhm 0.7. Proteotypic peptides
and MRM transitions were selected with the Skyline software utility<sup><a href="#ref22" rid="ref22" class=" bibr popnode">22</a></sup> and were further optimized by analyses of tryptic
digests of recombinant proteins on the TSQ Vantage triple quadrupole
mass spectrometer.</p></div><div id="sec2.6" class="sec"><h3 id="sec2.6title">Selection of Peptides and MRM Transitions</h3><p class="p p-first-last">Target peptides
were chosen according to previously published criteria.<sup><a href="#ref20" rid="ref20" class=" bibr popnode">20</a>,<a href="#ref23" rid="ref23" class=" bibr popnode">23</a>,<a href="#ref24" rid="ref24" class=" bibr popnode">24</a></sup> We digested the recombinant proteins
and determined the resultant peptides by monitoring all possible tryptic
peptides between 8 and 22 amino acids in LC&#x02013;MS experiments
using the product scan mode. Where recombinant protein was not available,
selection of peptides and transitions were selected on the basis of
observations from previous discovery experiments, observed peptides
in open proteomics databases, or computational predicted peptides
through algorithms such as enhanced signature peptide (ESP) predictor.<sup><a href="#ref25" rid="ref25" class=" bibr popnode">25</a></sup> Peptides containing methionine residues as well
as those containing post-translational modification sites such as
glycosylation listed in the UniProt database (to minimize interference
on digestion) were excluded. All peptides were run through BLASTP
(Uniprot) to ensure their uniqueness to the protein of interest. Peptides
and transitions selected for each protein are shown in Table S1 (<a href="#notes-1" rid="notes-1" class=" notes">SI</a>).</p></div><div id="sec2.7" class="sec sec-last"><h3 id="sec2.7title">Data Analysis</h3><p class="p p-first-last">MRM data acquired on the TSQ Vantage
triple quadrupole mass spectrometer were analyzed with Skyline software.<sup><a href="#ref22" rid="ref22" class=" bibr popnode">22</a></sup> Peak integrations were manually reviewed and
transitions from peptides measured were confirmed by the same retention
times and transition patterns of the light peptides and synthetic
heavy, stable isotope-labeled peptides. Peak areas for the four most
intense MRM transitions were integrated and summed to generate a peptide
peak area, which was divided by the peptide peak area for the internal
standard heavy peptide. For each protein, the unique peptide that
generated the highest summed peak area signal was used to quantify
the protein of interest.</p></div></div><div id="sec3" class="tsec sec"><h2 class="head no_bottom_margin" id="sec3title">Results</h2><div id="sec3.1" class="sec sec-first"><h3 id="sec3.1title">Biomarker Candidate Proteins and Overview of Approach</h3><p class="p p-first-last">Six biomarker candidate proteins (TIMP1, ENG, MSLN, THBS2, MMP9 and
COMP) were selected on the basis of literature reports, which suggest
that they are overexpressed in colon cancers or because differential
expression of these proteins in blood was associated with cancer.<sup><a href="#ref26" rid="ref26" class=" bibr popnode">26</a>&#x02212;<a href="#ref32" rid="ref32" class=" bibr popnode">32</a></sup> We also chose these candidates because well-characterized, commercially
available ELISAs were available for each. Moreover, the same capture
antibodies used in the ELISAs were available to us for evaluation
in IP-MRM assays. To analyze the biomarker candidates in plasma, we
performed either single-protein or multiplexed immunoprecipitations
and then resolved the target proteins into molecular weight fractions
by SDS-PAGE (Figure S1B [<a href="#notes-1" rid="notes-1" class=" notes">SI</a>]). The gel
bands were digested and the digests were spiked with isotope-labeled
standards and analyzed by MRM. The MRM signals from each peptide were
shown to be consistent with their corresponding isotope labeled peptides
in both retention time and transient patterns (Figure S8 [<a href="#notes-1" rid="notes-1" class=" notes">SI</a>]).</p></div><div id="sec3.2" class="sec"><h3 id="sec3.2title">Initial Characterization of IP-MRM and ELISA for Analysis of
TIMP1</h3><p class="p p-first">Recombinant TIMP1 protein was spiked into a &#x0201c;mock
plasma&#x0201d; matrix consisting 60 mg/mL BSA in PBS to mimic the
human plasma environment and provide a defined analyte concentration
for estimation of recovery. A 2-fold dilution series ranging from
640 to 10 ng/mL was constructed and a plot showing the theoretical
TIMP1 concentration versus the calculated protein concentration is
shown in Figure <a href="/pmc/articles/PMC3864264/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure1A</span></span><span>1</span></a>A (the theoretical TIMP1 concentration
versus the ratio L/H is also shown in Figure S9A (<a href="#notes-1" rid="notes-1" class=" notes">SI</a>). The calculated protein concentration was calculated from
the measured peak area ratio of the light to heavy peptides. The slope
(0.329) indicated TIMP1 recovery after all process steps including
immunoaffinity capture, trypsin digestion and peptide extraction.
The limit of detection (LOD) was 2.5 ng/mL (Table <a href="#tbl1" rid="tbl1" class=" other">1</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC3864264/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140480118184640"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3864264_pr-2013-00877e_0001.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pr-2013-00877e_0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3864264/bin/pr-2013-00877e_0001.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140480118184640"><a target="object" rel="noopener" href="/pmc/articles/PMC3864264/figure/fig1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC3864264/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure 1</a></div><!--caption a7--><div class="caption"><p>Response curves for IP-MRM analyses of recombinant TIMP1, COMP,
MMP9, THBS2, MSLN and ENG proteins. Proteins were spiked at 10&#x02013;640
ng/mL in a background matrix of 60 mg/mL BSA in DPBS and analyzed
by IP-MRM as described in <a href="#sec2" rid="sec2" class=" other">Experimental Procedures</a>. Values plotted are mean &#x000b1; standard deviation (<em>n</em> = 3).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Measurement summaries for biomarker
candidates by IP-MRM assay and ELISA</strong></div><div class="xtable"><table frame="hsides" rules="groups" border="0" class="rendered small default_table"><thead><tr><th colspan="2" align="center" rowspan="1">protein</th><th style="border:none;" align="center" rowspan="1" colspan="1">TIMP1</th><th style="border:none;" align="center" rowspan="1" colspan="1">COMP</th><th style="border:none;" align="center" rowspan="1" colspan="1">MMP9</th><th style="border:none;" align="center" rowspan="1" colspan="1">THBS2</th><th style="border:none;" align="center" rowspan="1" colspan="1">MSLN</th><th style="border:none;" align="center" rowspan="1" colspan="1">ENG</th></tr></thead><tbody><tr><td rowspan="8" style="border:none;" align="left" colspan="1">IP-MRM</td><td style="border:none;" align="left" rowspan="1" colspan="1">peptide</td><td style="border:none;" align="left" rowspan="1" colspan="1">GFQALGDAADIR</td><td style="border:none;" align="left" rowspan="1" colspan="1">ELQETNAALQDVR</td><td style="border:none;" align="left" rowspan="1" colspan="1">AVIDDAFAR</td><td style="border:none;" align="left" rowspan="1" colspan="1">ACVGDVQER</td><td style="border:none;" align="left" rowspan="1" colspan="1">TDAVLPLTVAEVQK</td><td style="border:none;" align="left" rowspan="1" colspan="1">VLPGHSAGPR</td></tr><tr><td style="border:none;" align="left" rowspan="1" colspan="1">linearity&#x000a0;<em>r</em></td><td style="border:none;" align="left" rowspan="1" colspan="1">0.979</td><td style="border:none;" align="left" rowspan="1" colspan="1">0.995</td><td style="border:none;" align="left" rowspan="1" colspan="1">0.979</td><td style="border:none;" align="left" rowspan="1" colspan="1">0.984</td><td style="border:none;" align="left" rowspan="1" colspan="1">0.991</td><td style="border:none;" align="left" rowspan="1" colspan="1">0.978</td></tr><tr><td style="border:none;" align="left" rowspan="1" colspan="1">recovery (%)</td><td style="border:none;" align="left" rowspan="1" colspan="1">33</td><td style="border:none;" align="left" rowspan="1" colspan="1">50</td><td style="border:none;" align="left" rowspan="1" colspan="1">41</td><td style="border:none;" align="left" rowspan="1" colspan="1">13.5</td><td style="border:none;" align="left" rowspan="1" colspan="1">55</td><td style="border:none;" align="left" rowspan="1" colspan="1">16</td></tr><tr><td style="border:none;" align="left" rowspan="1" colspan="1">LLOD<sup>a</sup> (ng/mL)</td><td style="border:none;" align="left" rowspan="1" colspan="1">2.5</td><td style="border:none;" align="left" rowspan="1" colspan="1">5.1</td><td style="border:none;" align="left" rowspan="1" colspan="1">3.0</td><td style="border:none;" align="left" rowspan="1" colspan="1">2.0</td><td style="border:none;" align="left" rowspan="1" colspan="1">8.9</td><td style="border:none;" align="left" rowspan="1" colspan="1">5.6</td></tr><tr><td style="border:none;" align="left" rowspan="1" colspan="1">CV at 10 ng/mL (%)</td><td style="border:none;" align="left" rowspan="1" colspan="1">2.3</td><td style="border:none;" align="left" rowspan="1" colspan="1">17</td><td style="border:none;" align="left" rowspan="1" colspan="1">10</td><td style="border:none;" align="left" rowspan="1" colspan="1">7</td><td style="border:none;" align="left" rowspan="1" colspan="1">12</td><td style="border:none;" align="left" rowspan="1" colspan="1">19</td></tr><tr><td style="border:none;" align="left" rowspan="1" colspan="1">normal<sup>b</sup> (<em>n</em>&#x000a0;=&#x000a0;12)</td><td style="border:none;" align="left" rowspan="1" colspan="1">141&#x000a0;&#x000b1;&#x000a0;41</td><td style="border:none;" align="left" rowspan="1" colspan="1">153&#x000a0;&#x000b1;&#x000a0;62</td><td style="border:none;" align="left" rowspan="1" colspan="1">111&#x000a0;&#x000b1;&#x000a0;46</td><td style="border:none;" align="left" rowspan="1" colspan="1">10.5&#x000a0;&#x000b1;&#x000a0;7.9</td><td style="border:none;" align="left" rowspan="1" colspan="1">8.3&#x000a0;&#x000b1;&#x000a0;3.5</td><td style="border:none;" align="left" rowspan="1" colspan="1">10.1&#x000a0;&#x000b1;&#x000a0;3.9</td></tr><tr><td style="border:none;" align="left" rowspan="1" colspan="1">cancer (<em>n</em>&#x000a0;=&#x000a0;12)</td><td style="border:none;" align="left" rowspan="1" colspan="1">212&#x000a0;&#x000b1;&#x000a0;59<sup>c</sup></td><td style="border:none;" align="left" rowspan="1" colspan="1">221&#x000a0;&#x000b1;&#x000a0;117</td><td style="border:none;" align="left" rowspan="1" colspan="1">141&#x000a0;&#x000b1;&#x000a0;96<sup>c</sup></td><td style="border:none;" align="left" rowspan="1" colspan="1">21&#x000a0;&#x000b1;&#x000a0;10</td><td style="border:none;" align="left" rowspan="1" colspan="1">14.1&#x000a0;&#x000b1;&#x000a0;8.5</td><td style="border:none;" align="left" rowspan="1" colspan="1">8.6&#x000a0;&#x000b1;&#x000a0;1.5</td></tr><tr><td style="border:none;" align="left" rowspan="1" colspan="1"><em>P</em>&#x000a0;value<sup>d</sup></td><td style="border:none;" align="left" rowspan="1" colspan="1">0.0028</td><td style="border:none;" align="left" rowspan="1" colspan="1">0.091</td><td style="border:none;" align="left" rowspan="1" colspan="1">0.34</td><td style="border:none;" align="left" rowspan="1" colspan="1">0.010</td><td style="border:none;" align="left" rowspan="1" colspan="1">0.041</td><td style="border:none;" align="left" rowspan="1" colspan="1">0.26</td></tr><tr><td rowspan="5" style="border:none;" align="left" colspan="1">ELISA</td><td style="border:none;" align="left" rowspan="1" colspan="1">LLOQ<sup>e</sup> (ng/mL)</td><td style="border:none;" align="left" rowspan="1" colspan="1">0.313</td><td style="border:none;" align="left" rowspan="1" colspan="1">0.156</td><td style="border:none;" align="left" rowspan="1" colspan="1">0.313</td><td style="border:none;" align="left" rowspan="1" colspan="1">0.313</td><td style="border:none;" align="left" rowspan="1" colspan="1">0.156</td><td style="border:none;" align="left" rowspan="1" colspan="1">0.156</td></tr><tr><td style="border:none;" align="left" rowspan="1" colspan="1">CV at LLOQ (%)</td><td style="border:none;" align="left" rowspan="1" colspan="1">13.4</td><td style="border:none;" align="left" rowspan="1" colspan="1">0.0</td><td style="border:none;" align="left" rowspan="1" colspan="1">2.8</td><td style="border:none;" align="left" rowspan="1" colspan="1">2.9</td><td style="border:none;" align="left" rowspan="1" colspan="1">8.8</td><td style="border:none;" align="left" rowspan="1" colspan="1">2.3</td></tr><tr><td style="border:none;" align="left" rowspan="1" colspan="1">normal (<em>n</em>&#x000a0;=&#x000a0;12)</td><td style="border:none;" align="left" rowspan="1" colspan="1">97&#x000a0;&#x000b1;&#x000a0;16</td><td style="border:none;" align="left" rowspan="1" colspan="1">178&#x000a0;&#x000b1;&#x000a0;59</td><td style="border:none;" align="left" rowspan="1" colspan="1">117&#x000a0;&#x000b1;&#x000a0;61</td><td style="border:none;" align="left" rowspan="1" colspan="1">26&#x000a0;&#x000b1;&#x000a0;6</td><td style="border:none;" align="left" rowspan="1" colspan="1">22&#x000a0;&#x000b1;&#x000a0;8</td><td style="border:none;" align="left" rowspan="1" colspan="1">3.9&#x000a0;&#x000b1;&#x000a0;0.8</td></tr><tr><td style="border:none;" align="left" rowspan="1" colspan="1">cancer (<em>n</em>&#x000a0;=&#x000a0;12)</td><td style="border:none;" align="left" rowspan="1" colspan="1">120&#x000a0;&#x000b1;&#x000a0;35<sup>c</sup></td><td style="border:none;" align="left" rowspan="1" colspan="1">208&#x000a0;&#x000b1;&#x000a0;124</td><td style="border:none;" align="left" rowspan="1" colspan="1">107&#x000a0;&#x000b1;&#x000a0;73<sup>c</sup></td><td style="border:none;" align="left" rowspan="1" colspan="1">33&#x000a0;&#x000b1;&#x000a0;9</td><td style="border:none;" align="left" rowspan="1" colspan="1">22&#x000a0;&#x000b1;&#x000a0;10</td><td style="border:none;" align="left" rowspan="1" colspan="1">4.5&#x000a0;&#x000b1;&#x000a0;0.9</td></tr><tr><td style="border:none;" align="left" rowspan="1" colspan="1"><em>P</em>&#x000a0;value<sup>d</sup></td><td style="border:none;" align="left" rowspan="1" colspan="1">0.060</td><td style="border:none;" align="left" rowspan="1" colspan="1">0.44</td><td style="border:none;" align="left" rowspan="1" colspan="1">0.73</td><td style="border:none;" align="left" rowspan="1" colspan="1">0.035</td><td style="border:none;" align="left" rowspan="1" colspan="1">1.0</td><td style="border:none;" align="left" rowspan="1" colspan="1">0.10</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140480122190656"><a target="object" rel="noopener" href="/pmc/articles/PMC3864264/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t1fn1"><sup>a</sup>LLOD, lower limit of detection.</div><div id="t1fn2"><sup>b</sup>Values are mean &#x000b1; SD.</div><div id="t1fn3"><sup>c</sup><em>n</em> = 11.</div><div id="t1fn4"><sup>d</sup>Unpaired <em>t</em> test.</div><div id="t1fn5"><sup>e</sup>The ELISA lower limit of quantitation
(LLOQ) was based on the lowest concentration for the manufacturer&#x02019;s
specified calibration curve. Plasma samples were analyzed at the dilution
recommended by the manufacturer.</div></div></div><p class="p p-last">Next, IP-MRM was used to measure TIMP1 in 12 50 &#x003bc;L plasma
aliquots from patients with colon cancer and in 12 samples from noncancer
controls. The same samples were also analyzed with a commercial ELISA
kit with the same capture antibody (Figure S10A [<a href="#notes-1" rid="notes-1" class=" notes">SI</a>]). The mean values in IP-MRM analyses in plasma samples
from cancer patients and controls were 212 ng/mL and 141 ng/mL, respectively,
which was a significant difference (unpaired <em>t</em> test, <em>p</em> = 0.0028) (Table <a href="#tbl1" rid="tbl1" class=" other">1</a>). In ELISA analyses,
the mean TIMP1 levels in plasma samples from cancer patients and controls
were 120 ng/mL and 97 ng/mL, which were not significantly different
(unpaired <em>t</em> test, <em>p</em> = 0.06).</p></div><div id="sec3.3" class="sec"><h3 id="sec3.3title">Comparison of Multiplexed IP-MRM Assay with ELISA</h3><p class="p p-first">A
key advantage of MRM-based methods is the capacity for multiplexed
analyses, which can increase analysis throughput and minimize sample
consumption. We performed a multiplexed IP-MRM analysis of the remaining
proteins (ENG, MSLN, THBS2, MMP9 and COMP), which were spiked into
a &#x0201c;mock plasma&#x0201d; matrix consisting of 60 mg/mL BSA in
PBS to produce concentrations ranging from 640 to 10 ng/mL. Antibodies
for the five proteins (one antibody for each protein) were immobilized
on the resin, mixed and aliquoted. The mixed antibody resin (1 &#x003bc;g/antibody/analyte)
was incubated with solutions of BSA matrix containing the spiked target
proteins. Each concentration point and blank was prepared in triplicate.
Plots of calculated protein concentration determined by the peak area
ratio of the light peptide to heavy peptide (L/H) showed a linear
increase in measured protein across the concentration range (Figure <a href="/pmc/articles/PMC3864264/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1B&#x02013;F)</span></span><span>1</span></a>B&#x02013;F) (the theoretical protein concentration
versus the ratio L/H is also shown in Figure S9B&#x02013;F [<a href="#notes-1" rid="notes-1" class=" notes">SI</a>]).</p><p>Recoveries were determined for each
of the five proteins based on the slope of the response curve and
ranged from 13.5% (THBS2) to 55% (MSLN). We note that both THBS2 and
ENG yielded lower slopes than the other proteins. This could reflect
less efficient capture or digestion for these proteins, or some combination
of both effects. LOD values were determined by comparing the variance
of the blank samples (with no analyte spiked in) to the variance of
the lowest level spiked sample (analyte at 10 ng/mL).<sup><a href="#ref9" rid="ref9" class=" bibr popnode">9</a></sup> The LOD values were between 2 and 10 ng/mL (Table <a href="#tbl1" rid="tbl1" class=" other">1</a>). On the basis of triplicate measurements at the
lowest spiked concentration (10 ng/mL) for each protein, CV values
were all below 20% (Table <a href="#tbl1" rid="tbl1" class=" other">1</a>).</p><p class="p p-last">All five proteins were analyzed individually by ELISA in plasma
samples from 12 cancer patients and from 12 controls (11 cancer plasma
samples were analyzed for TIMP1 and MMP9). The mean concentrations
of these five proteins determined by ELISA in plasma samples from
normal and cancer patients ranged from 3.9 ng/mL (ENG) to 208 ng/mL
(COMP) (Figure S10B&#x02013;F [<a href="#notes-1" rid="notes-1" class=" notes">SI</a>]), Table <a href="#tbl1" rid="tbl1" class=" other">1</a>). For IP-MRM analyses, the mixed antibody resin
(1 &#x003bc;g antibody/analyte) was used to simultaneously capture all
five protein targets. Three aliquots of each colon cancer and normal
plasma sample were analyzed by IP-MRM. The mean concentrations of
these five proteins determined by IP-MRM ranged from 8.3 ng/mL (ENG)
to 221 ng/mL (COMP) (Figure S10B&#x02013;F [<a href="#notes-1" rid="notes-1" class=" notes">SI</a>]), Table <a href="#tbl1" rid="tbl1" class=" other">1</a>). Comparison of measured values
between normal and cancer plasma samples with either ELISA or IP-MRM
yielded only two significant differences (<em>p</em> &#x0003c;
0.05, unpaired student <em>t</em> test) for TIMP1 and THBS2,
but only when measured by IP-MRM (Table <a href="#tbl1" rid="tbl1" class=" other">1</a>).</p></div><div id="sec3.4" class="sec sec-last"><h3 id="sec3.4title">Comparison of IP-MRM and ELISA Measurements across Individual
Samples</h3><p class="p p-first">We asked how the IP-MRM and ELISA methods compared
for measurement of the six biomarker candidate proteins across individual
samples. Figure <a href="/pmc/articles/PMC3864264/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure2</span></span><span>2</span></a> shows paired comparisons
of measurements with the two analysis methods for each sample. The
data indicate that the concordance of measurements depends on both
the analyte and the method. Analysis of the average measured plasma
concentrations for all analytes and all samples indicated that the
IP-MRM measurements were higher by 8.6 ng/mL at (<em>p</em> = 0.005, paired <em>t</em> test; 95% CI: 2.7&#x02013;14.5),
across all markers and that the two measurements were highly correlated
(<em>r</em> = 0.93). Comparison of the average difference
in plasma concentrations measured by IP-MRM versus ELISA for individual
candidates indicated that differences were significant (<em>p</em> &#x0003c; 0.05, paired <em>t</em> test) for TIMP1 (67.1 ng/mL),
THBS2 (&#x02212;13.4 ng/mL), MSLN (&#x02212;11.7 ng/mL) and ENG (5.2
ng/mL), but not for COMP (&#x02212;6.4 ng/mL), MMP9 (13.5 ng/mL). We
also examined correlation of IP-MRM and ELISA for the individual analytes
(Figure <a href="/pmc/articles/PMC3864264/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3).</span></span><span>3</span></a>). Correlation coefficients for the
two methods were high for MMP9, COMP and TIMP1 and moderate for THBS2
and MSLN. However there was no apparent correlation between the methods
for ENG, as ELISA measurements all were approximately 5 ng/mL, whereas
IP-MRM measurements varied from 5 to 20 ng/mL.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC3864264/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140480116687488"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3864264_pr-2013-00877e_0002.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pr-2013-00877e_0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3864264/bin/pr-2013-00877e_0002.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140480116687488"><a target="object" rel="noopener" href="/pmc/articles/PMC3864264/figure/fig2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC3864264/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Figure 2</a></div><!--caption a7--><div class="caption"><p>Quantitation of six biomarker candidate proteins in plasma samples
from colon cancer patients and noncancer controls by IP-MRM and ELISA.
Plasma levels of TIMP1, COMP, MMP9, THBS2, MSLN and ENG were measured
in controls (circles) and colon cancer patients (squares). Results
are from the average of triplicate IP-MRM measurements and duplicate
ELISA measurements.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig3" co-legend-rid="lgnd_fig3"><a href="/pmc/articles/PMC3864264/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140480158559040"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3864264_pr-2013-00877e_0003.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pr-2013-00877e_0003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3864264/bin/pr-2013-00877e_0003.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140480158559040"><a target="object" rel="noopener" href="/pmc/articles/PMC3864264/figure/fig3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig3"><div><a class="figpopup" href="/pmc/articles/PMC3864264/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3">Figure 3</a></div><!--caption a7--><div class="caption"><p>Correlation of protein expression levels measured by ELISA and
IP-MRM in plasma samples. Mean concentrations from triplicate analyses
of TIMP1, COMP, MMP9, THBS2, MSLN and ENG in plasma samples from 24
subjects (23 for TIMP1 and MMP9) by ELISA and IP-MRM are plotted.
Pearson correlation coefficients are indicated in the figure.</p></div></div></div></div></div><div id="sec4" class="tsec sec"><h2 class="head no_bottom_margin" id="sec4title">Discussion</h2><p class="p p-first">The targeted analysis of protein biomarker candidates in plasma
or serum requires highly sensitive and specific assays. Although ELISA
provides the gold standard platform for such analyses, the selection
of high-quality ELISAs is limited. Configuration of a new ELISA is
often costly and time-consuming and may fail for lack of high-quality
reagents. The rationale for hybrid immuno&#x02013;MS assays was first
proposed by Anderson and colleagues<sup><a href="#ref11" rid="ref11" class=" bibr popnode">11</a></sup> and
has been developed through the peptide capture (SISCAPA) approach.<sup><a href="#ref10" rid="ref10" class=" bibr popnode">10</a>,<a href="#ref12" rid="ref12" class=" bibr popnode">12</a>&#x02212;<a href="#ref14" rid="ref14" class=" bibr popnode">14</a></sup> The key advantages of an immuno&#x02013;MS approach
are the ability to systematically configure MRM assays based on proteotypic
peptides and the requirement for only one antibody for target capture.
Protein-capture-based immuno&#x02013;MS analysis has been reported,<sup><a href="#ref18" rid="ref18" class=" bibr popnode">18</a>,<a href="#ref19" rid="ref19" class=" bibr popnode">19</a></sup> but has not been as thoroughly explored. A potential advantage of
protein capture is that the antibodies might be employed to transition
the IP-MRM assay to an ELISA platform. With that consideration in
mind, we asked how IP-MRM and ELISA would compare when the same capture
antibodies are used.</p><p>Our results indicate that the two methods offer equivalent performance
with a few exceptions. A global comparison of the IP-MRM and ELISA
measurements in our study indicated a slight systematic bias in favor
of higher measurement values for IP-MRM. However, this difference
was not predictive for individual analytes, as absolute measurement
differences between the methods varied in either direction by approximately
5&#x02013;10 ng/mL. Measurement variation (CV at LOD) for IP-MRM ranged
from 2.3 to 19%, which was similar to variation for ELISAs of the
same samples (Table <a href="#tbl1" rid="tbl1" class=" other">1</a>). IP-MRM and ELISA measurements
were well-correlated, with coefficients for all of the analytes except
ENG = &#x02212;.017. Poor correlation of IP-MRM and ELISA for ENG appears
to be related to limited range of the ELISA measurements, which were
near the LOD of the assay at the dilution used (Figure S10F [<a href="#notes-1" rid="notes-1" class=" notes">SI</a>]). Because the same capture antibody was used
for both assays, it appears unlikely that differential capture would
explain the difference in ENG measurement by the two methods. The
difference may reflect selective recognition of a subset of ENG proteins
in plasma samples by the detection antibody, perhaps due to unanticipated
modifications. Such modifications may not have affected peptide-based
quantitation in IP-MRM assays.</p><p>We used plasma samples from individuals with colon cancers and
from noncancer controls in our study, and the biomarker candidate
proteins were selected on the basis of existing literature. However,
these experiments were designed only to compare the performance of
the assay platforms, not to validate biomarker candidates for colon
cancer detection. Although we cannot draw any conclusions about the
utility of the biomarker candidates, the data illustrate the importance
of reliable, precise assays for biomarker validation studies. We observed
a high degree of overlap in measurement distributions for all of the
candidates in control and cancer plasma samples. This is expected
for most biomarker candidates that, despite being overexpressed in
tumors, also may be derived from other tissues. Assays capable of
validating cancer biomarkers will be expected to precisely measure
small concentration differences for proteins present at ng/mL levels
or lower.</p><p>Our study affirmed a key advantage of IP-MRM, which is the ability
of the assay platform to perform multiplexed protein capture, as reported
by Nico et al.<sup><a href="#ref20" rid="ref20" class=" bibr popnode">20</a></sup> This advantage is particularly
important when amounts of plasma or serum samples are limited. In
our study, all proteins could be captured and quantified from a single
aliquot of 50 &#x003bc;L plasma, while input quantities for single ELISA
measurements were between 2 and 66 &#x003bc;L. The capture antibody
for IP-MRM approach should have a relatively high affinity for the
targeted protein. Our experiments represent a &#x0201c;best case&#x0201d;
example, because the capture antibodies were of sufficiently high
quality to support robust and sensitive ELISA kits. We have done other
IP-MRM experiments with antibodies found to be unsatisfactory for
ELISA, and the same antibodies also failed to provide significant
protein enrichment for IP-MRM measurements (unpublished observations).
IP-MRM might be used to analyze different isoforms or post-translational
modifications of proteins if they are bound with similar affinity
by the capture antibody.</p><p>Another potentially advantageous feature of IP-MRM is the detection
of proteins via multiple peptides. While this offers flexibility in
assay development and confirmation of measurements made on single
peptides, previous work demonstrates that one peptide usually provides
the greatest measurement sensitivity.<sup><a href="#ref10" rid="ref10" class=" bibr popnode">10</a>,<a href="#ref13" rid="ref13" class=" bibr popnode">13</a></sup> For this reason,
we chose not to explore comparison MRM measurements with multiple
peptides and instead focused on analysis of peptides that generated
the strongest signals. In addition, higher throughput IP-MRM assays
can utilize magnetic beads for antibody capture in a 96-well-plate
format.<sup><a href="#ref33" rid="ref33" class=" bibr popnode">33</a></sup></p><p class="p p-last">IP-MRM directed at intact proteins provides an effective approach
to systematically configure targeted protein measurements. The protein-targeted
capture compares favorably to both peptide-targeted IP-MRM and ELISA.
For novel targets, the choice of methods depends primarily on the
availability of antibodies and their performance. Protein-based IP-MRM
assays offer a faster, less costly approach to targeted protein measurement
than ELISAs and could dramatically expand the scope of targeted protein
quantitation in biology and medicine.</p></div><div id="ack-a.k.d" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.k.dtitle">Acknowledgments</h2><div class="sec"><p>This work was supported by a cooperative agreement award 5U24CA126479
from the National Cancer Institute through the Clinical Proteomic
Technology Assessment for Cancer (CPTAC) program.</p></div></div><div id="dl1" class="tsec sec"><h2 class="head no_bottom_margin" id="dl1title">Glossary</h2><div class="bk-sec"><div><h4>Abbreviations</h4><table class="default-table glossary"><tbody><tr><td>CPTAC</td><td>Clinical Proteomic Technology
Asssessment for Cancer</td></tr><tr><td>MS</td><td>mass spectrometry</td></tr><tr><td>MRM</td><td>multiple reaction monitoring</td></tr><tr><td>IP-MRM</td><td>immunoprecipitation GeLC MS</td></tr><tr><td>ELISA</td><td>enzyme-linked immunosorbent
assay</td></tr><tr><td>SISCAPA</td><td>stable
isotope standards and capture by antipeptide antibodies</td></tr><tr><td>CV</td><td>coefficient of variation</td></tr><tr><td>SID</td><td>stable isotope
dilution</td></tr><tr><td>CRC</td><td>colorectal
cancer</td></tr><tr><td>fwhm</td><td>full
width at half-maximum</td></tr><tr><td>LOD</td><td>limit of detection</td></tr><tr><td>SCX</td><td>strong cation exchange</td></tr><tr><td>IEF</td><td>isoelectric focusing</td></tr><tr><td>HPLC</td><td>high performance liquid chromatography</td></tr><tr><td>SDS</td><td>sodium dodecyl
sulfate</td></tr></tbody></table></div></div></div><div id="funding-group-a.i.b.p" class="tsec sec"><h2 class="head no_bottom_margin" id="funding-group-a.i.b.ptitle">Funding Statement</h2><p>National Institutes of Health, United States</p></div><div id="notes-1" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-1title">Supporting Information Available</h2><p> This material is available
free of charge via the Internet at <a href="http://pubs.acs.org">http://pubs.acs.org</a>.</p></div><div id="NOTES-d161e1027-autogenerated" class="tsec sec"><h2 class="head no_bottom_margin" id="NOTES-d161e1027-autogeneratedtitle">Notes</h2><p>The authors declare no
competing financial interest.</p></div><div id="sec-a.k.b" class="tsec bk-sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="sec-a.k.btitle">Supplementary Material</h2><!--/article/back/sec/--><div class="sec suppmat" id="sifile1"><div class="sup-box half_rhythm" id="media-a.k.b.b.a"><a href="/pmc/articles/PMC3864264/bin/pr400877e_si_001.pdf" data-ga-action="click_feat_suppl">pr400877e_si_001.pdf</a><sup>(2.8M, pdf)</sup></div></div></div><div id="ref-list-a.k.f" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.k.ftitle">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="ref1"><span class="mixed-citation" id="cit1">Rifai N.; Gillette M. A.; Carr S. A.
<span class="ref-title">Protein biomarker discovery and validation:
The long and uncertain path to clinical utility</span>. <span class="ref-journal">Nat. Biotechnol.</span>
2006, <span class="ref-vol">24</span><span class="ref-iss">8</span>971&#x02013;983. [<a href="https://pubmed.ncbi.nlm.nih.gov/16900146" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=Protein+biomarker+discovery+and+validation:&#10;The+long+and+uncertain+path+to+clinical+utility&amp;author=N.+Rifai&amp;author=M.+A.+Gillette&amp;author=S.+A.+Carr&amp;volume=24&amp;issue=8&amp;publication_year=2006&amp;pages=971-983&amp;pmid=16900146&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref2"><span class="mixed-citation" id="cit2">Shi T.; Fillmore T. L.; Sun X.; Zhao R.; Schepmoes A. A.; Hossain M.; Xie F.; Wu S.; Kim J. S.; Jones N.; Moore R. J.; Pasa-Tolic L.; Kagan J.; Rodland K. D.; Liu T.; Tang K.; Camp D. G. 2nd; Smith R. D.; Qian W. J.
<span class="ref-title">Antibody-free,
targeted mass-spectrometric approach for quantification of proteins
at low picogram per milliliter levels in human plasma/serum</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span>
2012, <span class="ref-vol">109</span><span class="ref-iss">38</span>15395&#x02013;15400. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3458402/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22949669" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Antibody-free,&#10;targeted+mass-spectrometric+approach+for+quantification+of+proteins&#10;at+low+picogram+per+milliliter+levels+in+human+plasma/serum&amp;author=T.+Shi&amp;author=T.+L.+Fillmore&amp;author=X.+Sun&amp;author=R.+Zhao&amp;author=A.+A.+Schepmoes&amp;volume=109&amp;issue=38&amp;publication_year=2012&amp;pages=15395-15400&amp;pmid=22949669&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref3"><span class="mixed-citation" id="cit3">Shi T.; Su D.; Liu T.; Tang K.; Camp D. G. 2nd; Qian W. J.; Smith R. D.
<span class="ref-title">Advancing the sensitivity of selected
reaction monitoring-based targeted quantitative proteomics</span>. <span class="ref-journal">Proteomics</span>
2012, <span class="ref-vol">12</span><span class="ref-iss">8</span>1074&#x02013;1092. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3375056/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22577010" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Advancing+the+sensitivity+of+selected&#10;reaction+monitoring-based+targeted+quantitative+proteomics&amp;author=T.+Shi&amp;author=D.+Su&amp;author=T.+Liu&amp;author=K.+Tang&amp;author=D.+G.+Camp&amp;volume=12&amp;issue=8&amp;publication_year=2012&amp;pages=1074-1092&amp;pmid=22577010&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref4"><span class="mixed-citation" id="cit4">Zhou J. Y.; Dann G. P.; Shi T.; Wang L.; Gao X.; Su D.; Nicora C. D.; Shukla A. K.; Moore R. J.; Liu T.; Camp D. G. 2nd; Smith R. D.; Qian W. J.
<span class="ref-title">Simple
sodium dodecyl sulfate-assisted sample preparation method for LC&#x02013;MS-based
proteomics applications</span>. <span class="ref-journal">Anal. Chem.</span>
2012, <span class="ref-vol">84</span><span class="ref-iss">6</span>2862&#x02013;2867. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3310275/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22339560" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Anal.+Chem.&amp;title=Simple&#10;sodium+dodecyl+sulfate-assisted+sample+preparation+method+for+LC&#x02013;MS-based&#10;proteomics+applications&amp;author=J.+Y.+Zhou&amp;author=G.+P.+Dann&amp;author=T.+Shi&amp;author=L.+Wang&amp;author=X.+Gao&amp;volume=84&amp;issue=6&amp;publication_year=2012&amp;pages=2862-2867&amp;pmid=22339560&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref5"><span class="mixed-citation" id="cit5">Barnidge D. R.; Goodmanson M. K.; Klee G. G.; Muddiman D. C.
<span class="ref-title">Absolute quantification
of the model biomarker prostate-specific antigen in serum by LC&#x02013;MS/MS
using protein cleavage and isotope dilution mass spectrometry</span>. <span class="ref-journal">J. Proteome Res.</span>
2004, <span class="ref-vol">3</span><span class="ref-iss">3</span>644&#x02013;652. [<a href="https://pubmed.ncbi.nlm.nih.gov/15253448" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Proteome+Res.&amp;title=Absolute+quantification&#10;of+the+model+biomarker+prostate-specific+antigen+in+serum+by+LC&#x02013;MS/MS&#10;using+protein+cleavage+and+isotope+dilution+mass+spectrometry&amp;author=D.+R.+Barnidge&amp;author=M.+K.+Goodmanson&amp;author=G.+G.+Klee&amp;author=D.+C.+Muddiman&amp;volume=3&amp;issue=3&amp;publication_year=2004&amp;pages=644-652&amp;pmid=15253448&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref6"><span class="mixed-citation" id="cit6">Anderson L.; Hunter C. L.
<span class="ref-title">Quantitative mass spectrometric multiple reaction monitoring
assays for major plasma proteins</span>. <span class="ref-journal">Mol. Cell.
Proteomics</span>
2006, <span class="ref-vol">5</span><span class="ref-iss">4</span>573&#x02013;588. [<a href="https://pubmed.ncbi.nlm.nih.gov/16332733" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.&#10;Proteomics&amp;title=Quantitative+mass+spectrometric+multiple+reaction+monitoring&#10;assays+for+major+plasma+proteins&amp;author=L.+Anderson&amp;author=C.+L.+Hunter&amp;volume=5&amp;issue=4&amp;publication_year=2006&amp;pages=573-588&amp;pmid=16332733&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref7"><span class="mixed-citation" id="cit7">Kuzyk M. A.; Smith D.; Yang J.; Cross T. J.; Jackson A. M.; Hardie D. B.; Anderson N. L.; Borchers C. H.
<span class="ref-title">Multiple reaction
monitoring-based, multiplexed, absolute quantitation of 45 proteins
in human plasma</span>. <span class="ref-journal">Mol. Cell. Proteomics</span>
2009, <span class="ref-vol">8</span><span class="ref-iss">8</span>1860&#x02013;1877. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2722777/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19411661" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Proteomics&amp;title=Multiple+reaction&#10;monitoring-based,+multiplexed,+absolute+quantitation+of+45+proteins&#10;in+human+plasma&amp;author=M.+A.+Kuzyk&amp;author=D.+Smith&amp;author=J.+Yang&amp;author=T.+J.+Cross&amp;author=A.+M.+Jackson&amp;volume=8&amp;issue=8&amp;publication_year=2009&amp;pages=1860-1877&amp;pmid=19411661&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref8"><span class="mixed-citation" id="cit8">Shi T.; Zhou J. Y.; Gritsenko M. A.; Hossain M.; Camp D. G. 2nd; Smith R. D.; Qian W. J.
<span class="ref-title">IgY14 and SuperMix
immunoaffinity separations coupled with liquid chromatography-mass
spectrometry for human plasma proteomics biomarker discovery</span>. <span class="ref-journal">Methods</span>
2012, <span class="ref-vol">56</span><span class="ref-iss">2</span>246&#x02013;253. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3261331/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21925605" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods&amp;title=IgY14+and+SuperMix&#10;immunoaffinity+separations+coupled+with+liquid+chromatography-mass&#10;spectrometry+for+human+plasma+proteomics+biomarker+discovery&amp;author=T.+Shi&amp;author=J.+Y.+Zhou&amp;author=M.+A.+Gritsenko&amp;author=M.+Hossain&amp;author=D.+G.+Camp&amp;volume=56&amp;issue=2&amp;publication_year=2012&amp;pages=246-253&amp;pmid=21925605&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref9"><span class="mixed-citation" id="cit9">Addona T. A.; Abbatiello S. E.; Schilling B.; Skates S. J.; Mani D. R.; Bunk D. M.; Spiegelman C. H.; Zimmerman L. J.; Ham A. J.; Keshishian H.; Hall S. C.; Allen S.; Blackman R. K.; Borchers C. H.; Buck C.; Cardasis H. L.; Cusack M. P.; Dodder N. G.; Gibson B. W.; Held J. M.; Hiltke T.; Jackson A.; Johansen E. B.; Kinsinger C. R.; Li J.; Mesri M.; Neubert T. A.; Niles R. K.; Pulsipher T. C.; Ransohoff D.; Rodriguez H.; Rudnick P. A.; Smith D.; Tabb D. L.; Tegeler T. J.; Variyath A. M.; Vega-Montoto L. J.; Wahlander A.; Waldemarson S.; Wang M.; Whiteaker J. R.; Zhao L.; Anderson N. L.; Fisher S. J.; Liebler D. C.; Paulovich A. G.; Regnier F. E.; Tempst P.; Carr S. A.
<span class="ref-title">Multi-site
assessment of the precision and reproducibility of multiple reaction
monitoring-based measurements of proteins in plasma</span>. <span class="ref-journal">Nat. Biotechnol.</span>
2009, <span class="ref-vol">27</span><span class="ref-iss">7</span>633&#x02013;641. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2855883/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19561596" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=Multi-site&#10;assessment+of+the+precision+and+reproducibility+of+multiple+reaction&#10;monitoring-based+measurements+of+proteins+in+plasma&amp;author=T.+A.+Addona&amp;author=S.+E.+Abbatiello&amp;author=B.+Schilling&amp;author=S.+J.+Skates&amp;author=D.+R.+Mani&amp;volume=27&amp;issue=7&amp;publication_year=2009&amp;pages=633-641&amp;pmid=19561596&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref10"><span class="mixed-citation" id="cit10">Keshishian H.; Addona T.; Burgess M.; Mani D. R.; Shi X.; Kuhn E.; Sabatine M. S.; Gerszten R. E.; Carr S. A.
<span class="ref-title">Quantification
of cardiovascular biomarkers in patient plasma by targeted mass spectrometry
and stable isotope dilution</span>. <span class="ref-journal">Mol. Cell. Proteomics</span>
2009, <span class="ref-vol">8</span><span class="ref-iss">10</span>2339&#x02013;2349. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2758760/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19596694" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Proteomics&amp;title=Quantification&#10;of+cardiovascular+biomarkers+in+patient+plasma+by+targeted+mass+spectrometry&#10;and+stable+isotope+dilution&amp;author=H.+Keshishian&amp;author=T.+Addona&amp;author=M.+Burgess&amp;author=D.+R.+Mani&amp;author=X.+Shi&amp;volume=8&amp;issue=10&amp;publication_year=2009&amp;pages=2339-2349&amp;pmid=19596694&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref11"><span class="mixed-citation" id="cit11">Anderson N. L.; Anderson N. G.; Haines L. R.; Hardie D. B.; Olafson R. W.; Pearson T. W.
<span class="ref-title">Mass spectrometric quantitation of peptides and proteins
using Stable Isotope Standards and Capture by Anti-Peptide Antibodies
(SISCAPA)</span>. <span class="ref-journal">J. Proteome Res.</span>
2004, <span class="ref-vol">3</span><span class="ref-iss">2</span>235&#x02013;244. [<a href="https://pubmed.ncbi.nlm.nih.gov/15113099" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Proteome+Res.&amp;title=Mass+spectrometric+quantitation+of+peptides+and+proteins&#10;using+Stable+Isotope+Standards+and+Capture+by+Anti-Peptide+Antibodies&#10;(SISCAPA)&amp;author=N.+L.+Anderson&amp;author=N.+G.+Anderson&amp;author=L.+R.+Haines&amp;author=D.+B.+Hardie&amp;author=R.+W.+Olafson&amp;volume=3&amp;issue=2&amp;publication_year=2004&amp;pages=235-244&amp;pmid=15113099&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref12"><span class="mixed-citation" id="cit12">Kuhn E.; Addona T.; Keshishian H.; Burgess M.; Mani D. R.; Lee R. T.; Sabatine M. S.; Gerszten R. E.; Carr S. A.
<span class="ref-title">Developing
multiplexed assays for troponin I and interleukin-33 in plasma by
peptide immunoaffinity enrichment and targeted mass spectrometry</span>. <span class="ref-journal">Clin. Chem.</span>
2009, <span class="ref-vol">55</span><span class="ref-iss">6</span>1108&#x02013;1117. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2865473/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19372185" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Chem.&amp;title=Developing&#10;multiplexed+assays+for+troponin+I+and+interleukin-33+in+plasma+by&#10;peptide+immunoaffinity+enrichment+and+targeted+mass+spectrometry&amp;author=E.+Kuhn&amp;author=T.+Addona&amp;author=H.+Keshishian&amp;author=M.+Burgess&amp;author=D.+R.+Mani&amp;volume=55&amp;issue=6&amp;publication_year=2009&amp;pages=1108-1117&amp;pmid=19372185&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref13"><span class="mixed-citation" id="cit13">Keshishian H.; Addona T.; Burgess M.; Kuhn E.; Carr S. A.
<span class="ref-title">Quantitative,
multiplexed assays for low abundance proteins in plasma by targeted
mass spectrometry and stable isotope dilution</span>. <span class="ref-journal">Mol. Cell. Proteomics</span>
2007, <span class="ref-vol">6</span><span class="ref-iss">12</span>2212&#x02013;2229. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2435059/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17939991" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Proteomics&amp;title=Quantitative,&#10;multiplexed+assays+for+low+abundance+proteins+in+plasma+by+targeted&#10;mass+spectrometry+and+stable+isotope+dilution&amp;author=H.+Keshishian&amp;author=T.+Addona&amp;author=M.+Burgess&amp;author=E.+Kuhn&amp;author=S.+A.+Carr&amp;volume=6&amp;issue=12&amp;publication_year=2007&amp;pages=2212-2229&amp;pmid=17939991&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref14"><span class="mixed-citation" id="cit14">Razavi M.; Frick L. E.; LaMarr W. A.; Pope M. E.; Miller C. A.; Anderson N. L.; Pearson T. W.
<span class="ref-title">High-throughput SISCAPA quantitation
of peptides from human plasma digests by ultrafast, liquid chromatography-free
mass spectrometry</span>. <span class="ref-journal">J. Proteome Res.</span>
2012, <span class="ref-vol">11</span><span class="ref-iss">12</span>5642&#x02013;5649. [<a href="https://pubmed.ncbi.nlm.nih.gov/23126378" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Proteome+Res.&amp;title=High-throughput+SISCAPA+quantitation&#10;of+peptides+from+human+plasma+digests+by+ultrafast,+liquid+chromatography-free&#10;mass+spectrometry&amp;author=M.+Razavi&amp;author=L.+E.+Frick&amp;author=W.+A.+LaMarr&amp;author=M.+E.+Pope&amp;author=C.+A.+Miller&amp;volume=11&amp;issue=12&amp;publication_year=2012&amp;pages=5642-5649&amp;pmid=23126378&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref15"><span class="mixed-citation" id="cit15">Whiteaker J. R.; Lin C.; Kennedy J.; Hou L.; Trute M.; Sokal I.; Yan P.; Schoenherr R. M.; Zhao L.; Voytovich U. J.; Kelly-Spratt K. S.; Krasnoselsky A.; Gafken P. R.; Hogan J. M.; Jones L. A.; Wang P.; Amon L.; Chodosh L. A.; Nelson P. S.; McIntosh M. W.; Kemp C. J.; Paulovich A. G.
<span class="ref-title">A targeted
proteomics-based pipeline for verification of biomarkers in plasma</span>. <span class="ref-journal">Nat. Biotechnol.</span>
2011, <span class="ref-vol">29</span><span class="ref-iss">7</span>625&#x02013;634. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3232032/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21685906" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=A+targeted&#10;proteomics-based+pipeline+for+verification+of+biomarkers+in+plasma&amp;author=J.+R.+Whiteaker&amp;author=C.+Lin&amp;author=J.+Kennedy&amp;author=L.+Hou&amp;author=M.+Trute&amp;volume=29&amp;issue=7&amp;publication_year=2011&amp;pages=625-634&amp;pmid=21685906&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref16"><span class="mixed-citation" id="cit16">Addona T. A.; Shi X.; Keshishian H.; Mani D. R.; Burgess M.; Gillette M. A.; Clauser K. R.; Shen D.; Lewis G. D.; Farrell L. A.; Fifer M. A.; Sabatine M. S.; Gerszten R. E.; Carr S. A.
<span class="ref-title">A pipeline
that integrates the discovery and verification of plasma protein biomarkers
reveals candidate markers for cardiovascular disease</span>. <span class="ref-journal">Nat. Biotechnol.</span>
2011, <span class="ref-vol">29</span><span class="ref-iss">7</span>635&#x02013;643. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3366591/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21685905" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=A+pipeline&#10;that+integrates+the+discovery+and+verification+of+plasma+protein+biomarkers&#10;reveals+candidate+markers+for+cardiovascular+disease&amp;author=T.+A.+Addona&amp;author=X.+Shi&amp;author=H.+Keshishian&amp;author=D.+R.+Mani&amp;author=M.+Burgess&amp;volume=29&amp;issue=7&amp;publication_year=2011&amp;pages=635-643&amp;pmid=21685905&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref17"><span class="mixed-citation" id="cit17">Hoofnagle A. N.; Becker J. O.; Wener M. H.; Heinecke J. W.
<span class="ref-title">Quantification of
thyroglobulin, a low-abundance serum protein, by immunoaffinity peptide
enrichment and tandem mass spectrometry</span>. <span class="ref-journal">Clin.
Chem.</span>
2008, <span class="ref-vol">54</span><span class="ref-iss">11</span>1796&#x02013;104. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2739673/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18801935" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.&#10;Chem.&amp;title=Quantification+of&#10;thyroglobulin,+a+low-abundance+serum+protein,+by+immunoaffinity+peptide&#10;enrichment+and+tandem+mass+spectrometry&amp;author=A.+N.+Hoofnagle&amp;author=J.+O.+Becker&amp;author=M.+H.+Wener&amp;author=J.+W.+Heinecke&amp;volume=54&amp;issue=11&amp;publication_year=2008&amp;pages=1796-104&amp;pmid=18801935&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref18"><span class="mixed-citation" id="cit18">Berna M.; Ott L.; Engle S.; Watson D.; Solter P.; Ackermann B.
<span class="ref-title">Quantification
of NTproBNP in rat serum using immunoprecipitation and LC/MS/MS: A
biomarker of drug-induced cardiac hypertrophy</span>. <span class="ref-journal">Anal. Chem.</span>
2008, <span class="ref-vol">80</span><span class="ref-iss">3</span>561&#x02013;566. [<a href="https://pubmed.ncbi.nlm.nih.gov/18179251" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Anal.+Chem.&amp;title=Quantification&#10;of+NTproBNP+in+rat+serum+using+immunoprecipitation+and+LC/MS/MS:+A&#10;biomarker+of+drug-induced+cardiac+hypertrophy&amp;author=M.+Berna&amp;author=L.+Ott&amp;author=S.+Engle&amp;author=D.+Watson&amp;author=P.+Solter&amp;volume=80&amp;issue=3&amp;publication_year=2008&amp;pages=561-566&amp;pmid=18179251&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref19"><span class="mixed-citation" id="cit19">Berna M. J.; Zhen Y.; Watson D. E.; Hale J. E.; Ackermann B. L.
<span class="ref-title">Strategic
use of immunoprecipitation and LC/MS/MS for trace-level protein quantification:
myosin light chain 1, a biomarker of cardiac necrosis</span>. <span class="ref-journal">Anal. Chem.</span>
2007, <span class="ref-vol">79</span><span class="ref-iss">11</span>4199&#x02013;4205. [<a href="https://pubmed.ncbi.nlm.nih.gov/17447729" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Anal.+Chem.&amp;title=Strategic&#10;use+of+immunoprecipitation+and+LC/MS/MS+for+trace-level+protein+quantification:&#10;myosin+light+chain+1,+a+biomarker+of+cardiac+necrosis&amp;author=M.+J.+Berna&amp;author=Y.+Zhen&amp;author=D.+E.+Watson&amp;author=J.+E.+Hale&amp;author=B.+L.+Ackermann&amp;volume=79&amp;issue=11&amp;publication_year=2007&amp;pages=4199-4205&amp;pmid=17447729&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref20"><span class="mixed-citation" id="cit20">Nicol G. R.; Han M.; Kim J.; Birse C. E.; Brand E.; Nguyen A.; Mesri M.; FitzHugh W.; Kaminker P.; Moore P. A.; Ruben S. M.; He T.
<span class="ref-title">Use of an immunoaffinity-mass spectrometry-based
approach for the quantification of protein biomarkers from serum samples
of lung cancer patients</span>. <span class="ref-journal">Mol. Cell. Proteomics</span>
2008, <span class="ref-vol">7</span><span class="ref-iss">10</span>1974&#x02013;1982. [<a href="https://pubmed.ncbi.nlm.nih.gov/18388126" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Proteomics&amp;title=Use+of+an+immunoaffinity-mass+spectrometry-based&#10;approach+for+the+quantification+of+protein+biomarkers+from+serum+samples&#10;of+lung+cancer+patients&amp;author=G.+R.+Nicol&amp;author=M.+Han&amp;author=J.+Kim&amp;author=C.+E.+Birse&amp;author=E.+Brand&amp;volume=7&amp;issue=10&amp;publication_year=2008&amp;pages=1974-1982&amp;pmid=18388126&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref21"><span class="mixed-citation" id="cit21">Wang Q.; Chaerkady R.; Wu J.; Hwang H. J.; Papadopoulos N.; Kopelovich L.; Maitra A.; Matthaei H.; Eshleman J. R.; Hruban R. H.; Kinzler K. W.; Pandey A.; Vogelstein B.
<span class="ref-title">Mutant proteins
as cancer-specific biomarkers</span>. <span class="ref-journal">Proc. Natl. Acad.
Sci. U.S.A.</span>
2011, <span class="ref-vol">108</span><span class="ref-iss">6</span>2444&#x02013;2449. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3038743/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21248225" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.&#10;Sci.+U.S.A.&amp;title=Mutant+proteins&#10;as+cancer-specific+biomarkers&amp;author=Q.+Wang&amp;author=R.+Chaerkady&amp;author=J.+Wu&amp;author=H.+J.+Hwang&amp;author=N.+Papadopoulos&amp;volume=108&amp;issue=6&amp;publication_year=2011&amp;pages=2444-2449&amp;pmid=21248225&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref22"><span class="mixed-citation" id="cit22">MacLean B.; Tomazela D. M.; Shulman N.; Chambers M.; Finney G. L.; Frewen B.; Kern R.; Tabb D. L.; Liebler D. C.; MacCoss M. J.
<span class="ref-title">Skyline: An open source document editor for creating
and analyzing targeted proteomics experiments</span>. <span class="ref-journal">Bioinformatics</span>
2010, <span class="ref-vol">26</span><span class="ref-iss">7</span>966&#x02013;968. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2844992/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20147306" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Skyline:+An+open+source+document+editor+for+creating&#10;and+analyzing+targeted+proteomics+experiments&amp;author=B.+MacLean&amp;author=D.+M.+Tomazela&amp;author=N.+Shulman&amp;author=M.+Chambers&amp;author=G.+L.+Finney&amp;volume=26&amp;issue=7&amp;publication_year=2010&amp;pages=966-968&amp;pmid=20147306&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref23"><span class="mixed-citation" id="cit23">Lange V.; Picotti P.; Domon B.; Aebersold R.
<span class="ref-title">Selected reaction
monitoring for quantitative proteomics: a tutorial</span>. <span class="ref-journal">Mol. Syst. Biol.</span>
2008, <span class="ref-vol">4</span>, 222. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2583086/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18854821" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Syst.+Biol.&amp;title=Selected+reaction&#10;monitoring+for+quantitative+proteomics:+a+tutorial&amp;author=V.+Lange&amp;author=P.+Picotti&amp;author=B.+Domon&amp;author=R.+Aebersold&amp;volume=4&amp;publication_year=2008&amp;pages=222&amp;pmid=18854821&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref24"><span class="mixed-citation" id="cit24">Whiteaker J. R.; Zhao L.; Abbatiello S. E.; Burgess M.; Kuhn E.; Lin C.; Pope M. E.; Razavi M.; Anderson N. L.; Pearson T. W.; Carr S. A.; Paulovich A. G.
<span class="ref-title">Evaluation of large scale quantitative
proteomic assay development using peptide affinity-based mass spectrometry</span>. <span class="ref-journal">Mol. Cell. Proteomics</span>
2011, <span class="ref-vol">10</span><span class="ref-iss">4</span>M110 005645. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3069346/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21245105" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Proteomics&amp;title=Evaluation+of+large+scale+quantitative&#10;proteomic+assay+development+using+peptide+affinity-based+mass+spectrometry&amp;author=J.+R.+Whiteaker&amp;author=L.+Zhao&amp;author=S.+E.+Abbatiello&amp;author=M.+Burgess&amp;author=E.+Kuhn&amp;volume=10&amp;issue=4&amp;publication_year=2011&amp;pages=M110+005645&amp;pmid=21245105&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref25"><span class="mixed-citation" id="cit25">Fusaro V. A.; Mani D. R.; Mesirov J. P.; Carr S. A.
<span class="ref-title">Prediction of high-responding
peptides for targeted protein assays by mass spectrometry</span>. <span class="ref-journal">Nat. Biotechnol.</span>
2009, <span class="ref-vol">27</span><span class="ref-iss">2</span>190&#x02013;198. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2753399/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19169245" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=Prediction+of+high-responding&#10;peptides+for+targeted+protein+assays+by+mass+spectrometry&amp;author=V.+A.+Fusaro&amp;author=D.+R.+Mani&amp;author=J.+P.+Mesirov&amp;author=S.+A.+Carr&amp;volume=27&amp;issue=2&amp;publication_year=2009&amp;pages=190-198&amp;pmid=19169245&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref26"><span class="mixed-citation" id="cit26">Chen X.; Su Y.; Fingleton B.; Acuff H.; Matrisian L. M.; Zent R.; Pozzi A.
<span class="ref-title">Increased plasma MMP9 in integrin
&#x003b1;1-null mice enhances lung metastasis of colon carcinoma cells</span>. <span class="ref-journal">Int. J. Cancer</span>
2005, <span class="ref-vol">116</span><span class="ref-iss">1</span>52&#x02013;61. [<a href="https://pubmed.ncbi.nlm.nih.gov/15756690" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Cancer&amp;title=Increased+plasma+MMP9+in+integrin&#10;&#x003b1;1-null+mice+enhances+lung+metastasis+of+colon+carcinoma+cells&amp;author=X.+Chen&amp;author=Y.+Su&amp;author=B.+Fingleton&amp;author=H.+Acuff&amp;author=L.+M.+Matrisian&amp;volume=116&amp;issue=1&amp;publication_year=2005&amp;pages=52-61&amp;pmid=15756690&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref27"><span class="mixed-citation" id="cit27">Creaney J.; Francis R. J.; Dick I. M.; Musk A. W.; Robinson B. W.; Byrne M. J.; Nowak A. K.
<span class="ref-title">Serum soluble mesothelin concentrations
in malignant pleural mesothelioma: Relationship to tumor volume, clinical
stage and changes in tumor burden</span>. <span class="ref-journal">Clin. Cancer
Res.</span>
2011, <span class="ref-vol">17</span><span class="ref-iss">5</span>1181&#x02013;1189. [<a href="https://pubmed.ncbi.nlm.nih.gov/21177406" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Cancer&#10;Res.&amp;title=Serum+soluble+mesothelin+concentrations&#10;in+malignant+pleural+mesothelioma:+Relationship+to+tumor+volume,+clinical&#10;stage+and+changes+in+tumor+burden&amp;author=J.+Creaney&amp;author=R.+J.+Francis&amp;author=I.+M.+Dick&amp;author=A.+W.+Musk&amp;author=B.+W.+Robinson&amp;volume=17&amp;issue=5&amp;publication_year=2011&amp;pages=1181-1189&amp;pmid=21177406&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref28"><span class="mixed-citation" id="cit28">Hawinkels L. J.; Kuiper P.; Wiercinska E.; Verspaget H. W.; Liu Z.; Pardali E.; Sier C. F.; ten Dijke P.
<span class="ref-title">Matrix metalloproteinase-14
(MT1-MMP)-mediated endoglin shedding inhibits tumor angiogenesis</span>. <span class="ref-journal">Cancer Res.</span>
2010, <span class="ref-vol">70</span><span class="ref-iss">10</span>4141&#x02013;4150. [<a href="https://pubmed.ncbi.nlm.nih.gov/20424116" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res.&amp;title=Matrix+metalloproteinase-14&#10;(MT1-MMP)-mediated+endoglin+shedding+inhibits+tumor+angiogenesis&amp;author=L.+J.+Hawinkels&amp;author=P.+Kuiper&amp;author=E.+Wiercinska&amp;author=H.+W.+Verspaget&amp;author=Z.+Liu&amp;volume=70&amp;issue=10&amp;publication_year=2010&amp;pages=4141-4150&amp;pmid=20424116&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref29"><span class="mixed-citation" id="cit29">Holten-Andersen M. N.; Christensen I. J.; Nielsen H. J.; Stephens R. W.; Jensen V.; Nielsen O. H.; Sorensen S.; Overgaard J.; Lilja H.; Harris A.; Murphy G.; Brunner N.
<span class="ref-title">Total levels
of tissue inhibitor of metalloproteinases 1 in plasma yield high diagnostic
sensitivity and specificity in patients with colon cancer</span>. <span class="ref-journal">Clin. Cancer Res.</span>
2002, <span class="ref-vol">8</span><span class="ref-iss">1</span>156&#x02013;164. [<a href="https://pubmed.ncbi.nlm.nih.gov/11801553" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Cancer+Res.&amp;title=Total+levels&#10;of+tissue+inhibitor+of+metalloproteinases+1+in+plasma+yield+high+diagnostic&#10;sensitivity+and+specificity+in+patients+with+colon+cancer&amp;author=M.+N.+Holten-Andersen&amp;author=I.+J.+Christensen&amp;author=H.+J.+Nielsen&amp;author=R.+W.+Stephens&amp;author=V.+Jensen&amp;volume=8&amp;issue=1&amp;publication_year=2002&amp;pages=156-164&amp;pmid=11801553&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref30"><span class="mixed-citation" id="cit30">Jubb A. M.; Hurwitz H. I.; Bai W.; Holmgren E. B.; Tobin P.; Guerrero A. S.; Kabbinavar F.; Holden S. N.; Novotny W. F.; Frantz G. D.; Hillan K. J.; Koeppen H.
<span class="ref-title">Impact of vascular
endothelial growth factor-A expression, thrombospondin-2 expression,
and microvessel density on the treatment effect of bevacizumab in
metastatic colorectal cancer</span>. <span class="ref-journal">J. Clin. Oncol.</span>
2006, <span class="ref-vol">24</span><span class="ref-iss">2</span>217&#x02013;227. [<a href="https://pubmed.ncbi.nlm.nih.gov/16365183" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Oncol.&amp;title=Impact+of+vascular&#10;endothelial+growth+factor-A+expression,+thrombospondin-2+expression,&#10;and+microvessel+density+on+the+treatment+effect+of+bevacizumab+in&#10;metastatic+colorectal+cancer&amp;author=A.+M.+Jubb&amp;author=H.+I.+Hurwitz&amp;author=W.+Bai&amp;author=E.+B.+Holmgren&amp;author=P.+Tobin&amp;volume=24&amp;issue=2&amp;publication_year=2006&amp;pages=217-227&amp;pmid=16365183&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref31"><span class="mixed-citation" id="cit31">Klee E. W.; Bondar O. P.; Goodmanson M. K.; Dyer R. B.; Erdogan S.; Bergstralh E. J.; Bergen H. R. 3rd; Sebo T. J.; Klee G. G.
<span class="ref-title">Candidate serum biomarkers for prostate adenocarcinoma
identified by mRNA differences in prostate tissue and verified with
protein measurements in tissue and blood</span>. <span class="ref-journal">Clin.
Chem.</span>
2012, <span class="ref-vol">58</span><span class="ref-iss">3</span>599&#x02013;609. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3951013/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22247499" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.&#10;Chem.&amp;title=Candidate+serum+biomarkers+for+prostate+adenocarcinoma&#10;identified+by+mRNA+differences+in+prostate+tissue+and+verified+with&#10;protein+measurements+in+tissue+and+blood&amp;author=E.+W.+Klee&amp;author=O.+P.+Bondar&amp;author=M.+K.+Goodmanson&amp;author=R.+B.+Dyer&amp;author=S.+Erdogan&amp;volume=58&amp;issue=3&amp;publication_year=2012&amp;pages=599-609&amp;pmid=22247499&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref32"><span class="mixed-citation" id="cit32">Sorensen N. M.; Bystrom P.; Christensen I. J.; Berglund A.; Nielsen H. J.; Brunner N.; Glimelius B.
<span class="ref-title">TIMP-1 is significantly associated
with objective response and survival in metastatic colorectal cancer
patients receiving combination of irinotecan, 5-fluorouracil, and
folinic acid</span>. <span class="ref-journal">Clin. Cancer Res.</span>
2007, <span class="ref-vol">13</span><span class="ref-iss">14</span>4117&#x02013;4122. [<a href="https://pubmed.ncbi.nlm.nih.gov/17634538" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Cancer+Res.&amp;title=TIMP-1+is+significantly+associated&#10;with+objective+response+and+survival+in+metastatic+colorectal+cancer&#10;patients+receiving+combination+of+irinotecan,+5-fluorouracil,+and&#10;folinic+acid&amp;author=N.+M.+Sorensen&amp;author=P.+Bystrom&amp;author=I.+J.+Christensen&amp;author=A.+Berglund&amp;author=H.+J.+Nielsen&amp;volume=13&amp;issue=14&amp;publication_year=2007&amp;pages=4117-4122&amp;pmid=17634538&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="ref33"><span class="mixed-citation" id="cit33">Whiteaker J. R.; Zhao L.; Anderson L.; Paulovich A. G.
<span class="ref-title">An automated
and multiplexed method for high throughput peptide immunoaffinity
enrichment and multiple reaction monitoring mass spectrometry-based
quantification of protein biomarkers</span>. <span class="ref-journal">Mol. Cell.
Proteomics</span>
2010, <span class="ref-vol">9</span><span class="ref-iss">1</span>184&#x02013;196. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2808264/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19843560" ref="reftype=pubmed&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.&#10;Proteomics&amp;title=An+automated&#10;and+multiplexed+method+for+high+throughput+peptide+immunoaffinity&#10;enrichment+and+multiple+reaction+monitoring+mass+spectrometry-based&#10;quantification+of+protein+biomarkers&amp;author=J.+R.+Whiteaker&amp;author=L.+Zhao&amp;author=L.+Anderson&amp;author=A.+G.+Paulovich&amp;volume=9&amp;issue=1&amp;publication_year=2010&amp;pages=184-196&amp;pmid=19843560&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3864264&amp;issue-id=231137&amp;journal-id=822&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li></ul></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Journal of Proteome Research</span> are provided here courtesy of <strong>American Chemical Society</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC3864264/pdf/pr400877e.pdf" class="int-view">PDF (1.5M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/3864264/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/3864264/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="3864264" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3864264%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="Fu1JBMjTbWZogVYANplMZWLqx7uBwsoBH4LL1H1Ds67blK9Vj3BpeRzc8t5FF1UN">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3864264%2F&amp;text=Comparison%20of%20Protein%20Immunoprecipitation-Multiple%0AReaction%20Monitoring%20with%20ELISA%20for%20Assay%20of%20Biomarker%20Candidates%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3864264%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3864264/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC3864264/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/24224610/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC3864264/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/24224610/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC3864264/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/3864264/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC3864264/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
