
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs2.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Individual letters of the RNA polymerase II CTD code govern distinct gene expression programs in fission yeast - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="D0BD41E4AC192A45000046B1F44B95F7.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="pnas" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC3964054/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3964054/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Proceedings of the National Academy of Sciences of the United States of America" /><meta name="citation_title" content="Individual letters of the RNA polymerase II CTD code govern distinct gene expression programs in fission yeast" /><meta name="citation_author" content="Beate Schwer" /><meta name="citation_author_institution" content="aDepartment of Microbiology and Immunology, Weill Cornell Medical College, New York, NY, 10065;" /><meta name="citation_author" content="Danny Asher Bitton" /><meta name="citation_author_institution" content="bDepartment of Genetics, Evolution and Environment and UCL Cancer Institute, University College London, London WC1E 6BT, United Kingdom; and" /><meta name="citation_author" content="Ana M. Sanchez" /><meta name="citation_author_institution" content="aDepartment of Microbiology and Immunology, Weill Cornell Medical College, New York, NY, 10065;" /><meta name="citation_author" content="J&#x000fc;rg B&#x000e4;hler" /><meta name="citation_author_institution" content="bDepartment of Genetics, Evolution and Environment and UCL Cancer Institute, University College London, London WC1E 6BT, United Kingdom; and" /><meta name="citation_author" content="Stewart Shuman" /><meta name="citation_author_institution" content="cMolecular Biology Program, Sloan&#x02013;Kettering Institute, New York, NY, 10065" /><meta name="citation_publication_date" content="2014/03/03" /><meta name="citation_issue" content="11" /><meta name="citation_volume" content="111" /><meta name="citation_firstpage" content="4185" /><meta name="citation_doi" content="10.1073/pnas.1321842111" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3964054/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC3964054/" /><meta name="citation_pmid" content="24591591" /><meta name="DC.Title" content="Individual letters of the RNA polymerase II CTD code govern distinct gene expression programs in fission yeast" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="National Academy of Sciences" /><meta name="DC.Contributor" content="Beate Schwer" /><meta name="DC.Contributor" content="Danny Asher Bitton" /><meta name="DC.Contributor" content="Ana M. Sanchez" /><meta name="DC.Contributor" content="J&#x000fc;rg B&#x000e4;hler" /><meta name="DC.Contributor" content="Stewart Shuman" /><meta name="DC.Date" content="2014 Mar 18" /><meta name="DC.Identifier" content="10.1073/pnas.1321842111" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Individual letters of the RNA polymerase II CTD code govern distinct gene expression programs in fission yeast" /><meta property="og:type" content="article" /><meta property="og:description" content="The carboxyl-terminal domain (CTD) code encrypted within the YSPTSPS heptad repeats of RNA polymerase II is deeply rooted in eukaryal biology. This paper exploits fission yeast CTD mutants in which single coding cues (or combinations of cues) are subtracted ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3964054/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC3964054/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/2/">Proc Natl Acad Sci U S A</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/235471/">v.111(11); 2014 Mar 18</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC3964054
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC3964054/pdf/pnas.201321842.pdf" class="int-view">PDF (1002K)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/3964054/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/3964054/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="3964054" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3964054%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="flWqv4e8qbXhDDlM5lRZSGTBYlqq2oFFC7UG7N1lQgtXRSiz5hi9u64DZ8nZ2lt1">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3964054%2F&amp;text=Individual%20letters%20of%20the%20RNA%20polymerase%20II%20CTD%20code%20govern%20distinct%20gene%20expression%20programs%20in%20fission%20yeast" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3964054%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3964054/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC3964054/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/24591591/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC3964054/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/24591591/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC3964054/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/3964054/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/2/">Proc Natl Acad Sci U S A</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/235471/">v.111(11); 2014 Mar 18</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC3964054
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><a href="//doi.org/10.1073%2Fpnas.1321842111" title="Link to Publisher's site" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"><img src="/corehtml/pmc/pmcgifs/logo-pnas.png" alt="Logo of pnas"><!-- --></img></a></div>
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Proc Natl Acad Sci U S A.</span> 2014 Mar 18; 111(11): 4185–4190. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2014 Mar 3. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1073%2Fpnas.1321842111" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1073/pnas.1321842111</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3964054</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/24591591">24591591</a></div></div></div><h1 class="content-title">Individual letters of the RNA polymerase II CTD code govern distinct gene expression programs in fission yeast</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Schwer%20B%5BAuthor%5D" class="affpopup" co-rid="_co_idm140408615111168" co-class="co-affbox">Beate Schwer</a>,<sup>a,</sup><sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Bitton%20DA%5BAuthor%5D" class="affpopup" co-rid="_co_idm140408609124816" co-class="co-affbox">Danny Asher Bitton</a>,<sup>b</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Sanchez%20AM%5BAuthor%5D" class="affpopup" co-rid="_co_idm140408617833040" co-class="co-affbox">Ana M. Sanchez</a>,<sup>a</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=B&#x000e4;hler%20J%5BAuthor%5D" class="affpopup" co-rid="_co_idm140408617830928" co-class="co-affbox">J&#x000fc;rg B&#x000e4;hler</a>,<sup>b</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shuman%20S%5BAuthor%5D" class="affpopup" co-rid="_co_idm140408615355488" co-class="co-affbox">Stewart Shuman</a><sup>c,</sup><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140408615111168"><h3 class="no_margin">Beate Schwer</h3><p><sup>a</sup>Department of Microbiology and Immunology, Weill Cornell Medical College, New York, NY,  10065;</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Schwer%20B%5BAuthor%5D">Beate Schwer</a></div></div><div id="_co_idm140408609124816"><h3 class="no_margin">Danny Asher Bitton</h3><p><sup>b</sup>Department of Genetics, Evolution and Environment and UCL Cancer Institute, University College London, London WC1E 6BT, United Kingdom; and</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Bitton%20DA%5BAuthor%5D">Danny Asher Bitton</a></div></div><div id="_co_idm140408617833040"><h3 class="no_margin">Ana M. Sanchez</h3><p><sup>a</sup>Department of Microbiology and Immunology, Weill Cornell Medical College, New York, NY,  10065;</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Sanchez%20AM%5BAuthor%5D">Ana M. Sanchez</a></div></div><div id="_co_idm140408617830928"><h3 class="no_margin">J&#x000fc;rg B&#x000e4;hler</h3><p><sup>b</sup>Department of Genetics, Evolution and Environment and UCL Cancer Institute, University College London, London WC1E 6BT, United Kingdom; and</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=B&#x000e4;hler%20J%5BAuthor%5D">J&#x000fc;rg B&#x000e4;hler</a></div></div><div id="_co_idm140408615355488"><h3 class="no_margin">Stewart Shuman</h3><p><sup>c</sup>Molecular Biology Program, Sloan&#x02013;Kettering Institute, New York, NY,  10065</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shuman%20S%5BAuthor%5D">Stewart Shuman</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140408612866640_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140408612866640_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140408612866640_ai" style="display:none"><div class="fm-affl" id="aff1"><sup>a</sup>Department of Microbiology and Immunology, Weill Cornell Medical College, New York, NY,  10065;</div><div class="fm-affl" id="aff2"><sup>b</sup>Department of Genetics, Evolution and Environment and UCL Cancer Institute, University College London, London WC1E 6BT, United Kingdom; and</div><div class="fm-affl" id="aff3"><sup>c</sup>Molecular Biology Program, Sloan&#x02013;Kettering Institute, New York, NY,  10065</div><div id="cor1"><sup>1</sup>To whom correspondence may be addressed. E-mail: <a href="mailto:dev@null" data-email="ude.llenroc.dem@rewhcsb" class="oemail">ude.llenroc.dem@rewhcsb</a> or <a href="mailto:dev@null" data-email="gro.ccksm.iks@namuhs-s" class="oemail">gro.ccksm.iks@namuhs-s</a>.</div><div id="fn-a.i.b.f.b">Edited by Fred M. Winston, Harvard Medical School, Boston, MA, and approved February 7, 2014 (received for review November 21, 2013)</div><div><p class="p p-first-last">Author contributions: B.S., D.A.B., J.B., and S.S. designed research; B.S., D.A.B., A.M.S., and S.S. performed research; B.S., D.A.B., A.M.S., J.B., and S.S. analyzed data; and B.S., D.A.B., and S.S. wrote the paper.</p></div></div><div class="fm-article-notes half_rhythm"></div><div class="permissions half_rhythm hide" id="idm140408612866640_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">PMC Copyright notice</a> </div><div class="license half_rhythm">Freely available online through the PNAS open access option.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Supporting Information
</div>

<div><a href="/pmc/articles/PMC3964054/bin/supp_111_11_4185__index.html" data-ga-action="click_feat_suppl">supp_111_11_4185__index.html</a><span style="color:gray"> (7.2K)</span></div><div class="small guid">GUID: AA796AF1-C0D9-4B3D-8EA5-EA0E48DEE9CB</div>
<div><a href="/pmc/articles/PMC3964054/bin/1321842111_pnas.201321842SI.pdf" data-ga-action="click_feat_suppl">1321842111_pnas.201321842SI.pdf</a><span style="color:gray"> (1.5M)</span></div><div class="small guid">GUID: EC242B00-7657-4D82-B4A5-090FCF3EFBDE</div>
<div><a href="/pmc/articles/PMC3964054/bin/1321842111_sd01.xls" data-ga-action="click_feat_suppl">1321842111_sd01.xls</a><span style="color:gray"> (5.6M)</span></div><div class="small guid">GUID: 9FC5F5A1-0B70-4B3B-9DF4-B5B2A37C8610</div>
</div></dd></dl></div><div id="abstract-a.i.b.p" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.i.b.ptitle" style="text-transform: uppercase;">Significance</h2><!--article-meta--><div><p class="p p-first-last">The carboxyl-terminal domain (CTD) code encrypted within the YSPTSPS heptad repeats of RNA polymerase II is deeply rooted in eukaryal biology. This paper exploits fission yeast CTD mutants in which single coding cues (or combinations of cues) are subtracted from all consensus heptads to gauge the roles of the Tyr1, Ser2, Thr4, and Ser7 phosphoacceptors in gene expression. Transcriptome profiling revealed distinctive signatures for the CTD phosphoacceptors that were verified by cell-based activity assays. The results highlight (<em>i</em>) the role of Tyr1 and Ser2 in repressing the iron uptake regulon and (<em>ii</em>) the distinctive roles of Thr4 and Ser7 in the phosphate starvation response, whereby Thr4 facilitates and Ser7 represses the induction of Pho1 acid phosphatase.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">iron homeostasis, phosphate homeostasis, transcription profiling</span></div></div><div id="abstract-a.i.b.q" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.i.b.qtitle" style="text-transform: uppercase;">Abstract</h2><!--article-meta--><div><p class="p p-first-last">The primary structure and phosphorylation pattern of the tandem Y<sup>1</sup>S<sup>2</sup>P<sup>3</sup>T<sup>4</sup>S<sup>5</sup>P<sup>6</sup>S<sup>7</sup> repeats of the RNA polymerase II carboxyl-terminal domain (CTD) comprise an informational code that coordinates transcription, chromatin modification, and RNA processing. To gauge the contributions of individual CTD coding &#x0201c;letters&#x0201d; to gene expression, we analyzed the poly(A)<sup>+</sup> transcriptomes of fission yeast mutants that lack each of the four inessential CTD phosphoacceptors: Tyr1, Ser2, Thr4, and Ser7. There was a hierarchy of CTD mutational effects with respect to the number of dysregulated protein-coding RNAs, with <em>S2A</em> (<em>n</em> = 227) &#x0003e;&#x0003e; <em>Y1F</em> (<em>n</em> = 71) &#x0003e; <em>S7A</em> (<em>n</em> = 58) &#x0003e;&#x0003e; <em>T4A</em> (<em>n</em> = 7). The majority of the protein-coding RNAs affected in <em>Y1F</em> cells were coordinately affected by <em>S2A</em>, suggesting that Tyr1-Ser2 constitutes a two-letter code &#x0201c;word.&#x0201d; <em>Y1F</em> and <em>S2A</em> elicited increased expression of genes encoding proteins involved in iron uptake (Frp1, Fip1, Fio1, Str3, Str1, Sib1), without affecting the expression of the genes that repress the iron regulon, implying that Tyr1-Ser2 transduces a repressive signal. <em>Y1F</em> and <em>S2A</em> cells had increased levels of ferric reductase activity and were hypersensitive to phleomycin, indicative of elevated intracellular iron. The <em>T4A</em> and <em>S7A</em> mutations had opposing effects on the phosphate response pathway. <em>T4A</em> reduced the expression of two genes encoding proteins involved in phosphate acquisition (the Pho1 acid phosphatase and the phosphate transporter SPBC8E4.01c), without affecting the expression of known genes that regulate the phosphate response pathway, whereas <em>S7A</em> increased <em>pho1</em><sup><em>+</em></sup> expression. These results highlight specific cellular gene expression programs that are responsive to distinct CTD cues.</p></div></div><div id="body-a.j" class="tsec sec"><h2 class="headless nomenu"></h2><p class="p p-first">The carboxyl-terminal domain (CTD) of the Rpb1 subunit of RNA polymerase II (Pol II) consists of tandemly repeated heptapeptides of consensus sequence Y<sup>1</sup>S<sup>2</sup>P<sup>3</sup>T<sup>4</sup>S<sup>5</sup>P<sup>6</sup>S<sup>7</sup>. The inherently plastic CTD structure is modulated by phosphorylation of the Tyr1, Ser2, Thr4, Ser5, and Ser7 residues and by <em>cis</em>&#x02013;<em>trans</em> isomerization of the prolines (<a href="#r1" rid="r1" class=" bibr popnode">1</a>, <a href="#r2" rid="r2" class=" bibr popnode">2</a>). With as many as 128<sup><em>n</em></sup> potential CTD primary structures (where <em>n</em> is the number of heptads), the CTD provides information about the state of the transcription machinery&#x02014;a CTD code&#x02014;that is &#x0201c;read&#x0201d; by CTD receptor proteins that control transcription, modify chromatin structure, and catalyze or regulate mRNA capping, splicing, and polyadenylation (<a href="#r1" rid="r1" class=" bibr popnode">1</a>, <a href="#r2" rid="r2" class=" bibr popnode">2</a>).</p><p>Basic informational rules that govern the CTD code have been elucidated by genetically manipulating the composition and structure of the Rpb1 CTD in the fission yeast <em>Schizosaccharomyces pombe</em> (<a href="#r3" rid="r3" class=" bibr popnode">3</a><a href="#r4" rid="r4" class=" bibr popnode"></a><a href="#r5" rid="r5" class=" bibr popnode"></a><a href="#r6" rid="r6" class=" bibr popnode"></a>&#x02013;<a href="#r7" rid="r7" class=" bibr popnode">7</a>). By introducing alanines and conservative mutations in lieu of Tyr1, Ser2, Pro3, Thr4, Ser5, Pro6, and Ser7 of every consensus heptad of a fully functional Rpb1 CTD array (comprising 14 consensus heptad repeats linked to the body of Rpb1 by a &#x0201c;rump&#x0201d; consisting of four degenerate heptads), we determined that: (<em>i</em>) Tyr1, Pro3, Ser5, and Pro6 are essential for viability, by the criterion that alanine substitution is lethal, whereas Ser2, Thr4, and Ser7 are not essential; and (<em>ii</em>) <em>Y1F</em>, <em>S2A&#x02022;S7A</em> and <em>T4A&#x02022;S7A</em> mutants are viable, signifying that phenylalanine is functional in lieu of Tyr1 and that Ser5 is the only strictly essential phosphorylation site in fission yeast (<a href="#r6" rid="r6" class=" bibr popnode">6</a>).</p><p>Further interrogation of the grammar and punctuation of the CTD code established that: (<em>i</em>) all essential coding letters and &#x0201c;words&#x0201d; are encompassed within a diheptad repeat; (<em>ii</em>) the minimal fission yeast CTD coding unit is a decapeptide Y<sup>1</sup>S<sup>2</sup>P<sup>3</sup>T<sup>4</sup>S<sup>5</sup>P<sup>6</sup>S<sup>7</sup>Y<sup>1</sup>S<sup>2</sup>P<sup>3</sup> and the spacing between coding units is flexible; (<em>iii</em>) Tyr1 must be present in consecutive heptads and proper spacing between consecutive tyrosines is important for CTD function; (<em>iv</em>) Pro3, Ser5, and Pro6 need not be present in adjacent heptads; and (<em>v</em>) Ser5(P)-Pro6 comprises an essential two-letter code word that is read chiefly by the mRNA capping apparatus (<a href="#r7" rid="r7" class=" bibr popnode">7</a>). Querying how perturbations of CTD primary structure affect CTD serine phosphorylation patterns in vivo, as gauged by Rpb1 reactivity with phosphospecific antibodies, showed that Ser2 phosphorylation does not rely on Ser5, Pro6, Ser7, or Thr4, whereas Ser5 phosphorylation does not depend on Ser2, Thr4, or Ser7.</p><p>The fact that four of the five phosphoacceptor coding letters of the CTD heptad are not essential in fission yeast raises important questions as to whether and how these phosphate marks impact gene expression, the extreme situations being that absence of a particular CTD-PO<sub>4</sub> mark has little or no effect or that loss of a coding cue does exert significant effects, albeit on the expression of genes that are not essential under the laboratory conditions surveyed. The latter scenario applies to the Ser2 coding letter, insofar as a heterothallic haploid fission yeast <em>rpb1-S2A</em> mutant can grow vegetatively, but is unable to mate with heterothallic WT cells of either mating type, i.e., <em>S2A</em> is sterile. Sterility results from the inability of <em>S2A</em> cells to induce expression of the Ste11 transcription factor that triggers the meiotic sexual differentiation pathway (<a href="#r5" rid="r5" class=" bibr popnode">5</a>, <a href="#r6" rid="r6" class=" bibr popnode">6</a>, <a href="#r8" rid="r8" class=" bibr popnode">8</a>). Microarray analyses of the transcriptome of <em>S2A</em> cells vs. WT highlighted &#x02265;1.5-fold changes in the expression level of many meiotically regulated genes (<a href="#r5" rid="r5" class=" bibr popnode">5</a>, <a href="#r9" rid="r9" class=" bibr popnode">9</a>). The Ser2 requirement for transcription during sexual differentiation was bypassed by subtracting Ser7, suggesting that imbalance in the phosphorylation array, not absence of a CTD-PO<sub>4</sub> cue, underlies a CTD-associated pathology (<a href="#r6" rid="r6" class=" bibr popnode">6</a>).</p><p class="p">In the present study, we apply high-throughput RNA sequencing methods (RNA-seq) to gauge globally the impact of the loss of each of the four inessential CTD phosphoacceptors on gene expression. By comparing the poly(A)<sup>+</sup> transcriptomes of four fission yeast mutants, <em>rpb1</em>-<em>Y1F</em>, -<em>S2A</em>, -<em>T4A</em>, and -<em>S7A</em>, we illuminate how individual letters of the Pol II CTD code affect the expression of distinct sets of genes. Validation of selected transcriptional programs by functional tests identified Tyr1 and Ser2 as a two-letter coding word that governs an iron-uptake regulon (<a href="#r10" rid="r10" class=" bibr popnode">10</a>) and Thr4 and Ser7 as components of the phosphate response pathway that regulates Pho1 expression.</p></div><div id="s1" class="tsec sec"><h2 class="head no_bottom_margin" id="s1title" style="text-transform: uppercase;">Results</h2><div id="s2" class="sec sec-first"><p></p><h3 id="s2title" class="inline">RNA-Seq Analysis of <em>rpb1-CTD</em> Mutants. </h3><p class="p p-first">RNA-seq affords a detailed strand-specific annotated map of the protein-coding and noncoding fission yeast transcripts vs. their genomic DNA loci (<a href="#r11" rid="r11" class=" bibr popnode">11</a><a href="#r12" rid="r12" class=" bibr popnode"></a>&#x02013;<a href="#r13" rid="r13" class=" bibr popnode">13</a>). Here we performed RNA-seq on poly(A)<sup>+</sup> RNA isolated from cells expressing Rpb1 with CTDs composed of the rump plus 14 consensus YSPTSPS heptads or an identical number of Y1F, S2A, T4A, or S7A mutant heptads (<a href="#r6" rid="r6" class=" bibr popnode">6</a>). We also analyzed CTD double mutants <em>Y1F&#x02022;S7A</em>, <em>S2A&#x02022;S7A</em>, and <em>T4A&#x02022;S7A</em>. cDNAs obtained from two nonsimultaneous biological replicates (using RNA from cells grown to midlog phase at 30 &#x000b0;C) were sequenced for each strain. In all of the datasets, &#x0223c;90% of the reads were mapped to unique genomic loci (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1321842111/-/DCSupplemental/pnas.201321842SI.pdf?targetid=nameddest=SF1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S1</a>), and expression levels were highly reproducible between biological replicates (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1321842111/-/DCSupplemental/pnas.201321842SI.pdf?targetid=nameddest=SF2" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S2</a>). As an internal control, we queried the cDNA sequences encoding the Rpb1 CTD itself and found that between 99.7% and 100% of the CTD mRNA reads were correct with respect to the WT or relevant mutant CTD, with the rare exceptions corresponding to other CTD variants that were bar-coded in the pooled sequence run (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1321842111/-/DCSupplemental/pnas.201321842SI.pdf?targetid=nameddest=SF1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S1</a>). A cutoff of plus or minus twofold change in normalized transcript read level and a corrected <em>P</em> value of &#x0003c;0.05 were the criteria applied to derive lists of differentially expressed annotated loci for each <em>rpb1-CTD</em> single-mutant vs. the WT CTD strain.</p><p>The results are presented as heat maps for the ensemble of significantly changing protein-coding and noncoding RNAs (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1321842111/-/DCSupplemental/pnas.201321842SI.pdf?targetid=nameddest=SF3" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S3</a>) and as four separate Venn diagrams in <a href="/pmc/articles/PMC3964054/figure/fig01/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig01" rid-ob="ob-fig01" co-legend-rid="lgnd_fig01"><span>Fig. 1</span></a>, showing protein-coding transcripts that were down-regulated (<a href="/pmc/articles/PMC3964054/figure/fig01/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig01" rid-ob="ob-fig01" co-legend-rid="lgnd_fig01"><span>Fig. 1<em>A</em></span></a>) or up-regulated (<a href="/pmc/articles/PMC3964054/figure/fig01/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig01" rid-ob="ob-fig01" co-legend-rid="lgnd_fig01"><span>Fig. 1<em>B</em></span></a>) and noncoding transcripts that were down-regulated (<a href="/pmc/articles/PMC3964054/figure/fig01/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig01" rid-ob="ob-fig01" co-legend-rid="lgnd_fig01"><span>Fig. 1<em>C</em></span></a>) or up-regulated (<a href="/pmc/articles/PMC3964054/figure/fig01/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig01" rid-ob="ob-fig01" co-legend-rid="lgnd_fig01"><span>Fig. 1<em>D</em></span></a>) in the <em>Y1F</em>, <em>S2A</em>, <em>T4A</em>, and <em>S7A</em> strains. There was a clear hierarchy of CTD mutational effects on the number of annotated coding plus noncoding RNAs dysregulated, whereby <em>S2A</em> (<em>n</em> = 481) had greater impact than <em>Y1F</em> (<em>n</em> = 107) or <em>S7A</em> (<em>n</em> = 97) and <em>T4A</em> (<em>n</em> = 15) had only scant effects on gene expression. Of the 5,133 annotated protein-coding transcripts in fission yeast, 157 (3.1%) were down-regulated in <em>S2A</em>; 54 (1.1%) in <em>Y1F</em>; 25 (0.5%) in <em>S7A</em>; and only 4 (0.08%) in <em>T4A</em>. To the extent that the at least twofold decrement in individual transcript levels in these mutant strains reflects a contribution of the missing CTD coding letter or phosphorylation mark for achieving normal levels of transcription or processing of the affected mRNAs, these results are remarkable for how few fission yeast genes rely on these CTD cues. However, the modest effects of the <em>rpb1-CTD Y1F</em>, <em>T4A</em>, and <em>S7A</em> mutations on mRNA levels were in keeping with the fact that these strains grow as well as WT cells in YES medium at 30 &#x000b0;C (<a href="#r6" rid="r6" class=" bibr popnode">6</a>). The <em>S2A</em> strain, in which more mRNAs are down-regulated, grows slightly slower than WT at 30 &#x000b0;C; this is the case even when the length of the CTD S2A array appended to the rump is increased from 14 to 25 heptads (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1321842111/-/DCSupplemental/pnas.201321842SI.pdf?targetid=nameddest=SF4" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S4</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig01" co-legend-rid="lgnd_fig01"><a href="/pmc/articles/PMC3964054/figure/fig01/" target="figure" rid-figpopup="fig01" rid-ob="ob-fig01"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140408616723472"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3964054_pnas.1321842111fig01.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pnas.1321842111fig01.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3964054/bin/pnas.1321842111fig01.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140408616723472"><a target="object" rel="noopener" href="/pmc/articles/PMC3964054/figure/fig01/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig01"><div><a class="figpopup" href="/pmc/articles/PMC3964054/figure/fig01/" target="figure" rid-figpopup="fig01" rid-ob="ob-fig01">Fig. 1.</a></div><!--caption a7--><div class="caption"><p>Effect of CTD mutations on the fission yeast transcriptome. Four Venn diagrams specify the numbers of protein-coding (<em>A</em> and <em>B</em>) and noncoding (<em>C</em> and <em>D</em>) poly(A)<sup>+</sup> transcripts that were significantly down-regulated by at least twofold (<em>A</em> and <em>C</em>) or significantly up-regulated by at least twofold (<em>B</em> and <em>D</em>) in the indicated <em>rpb1-CTD</em> strains relative to WT strain.</p></div></div></div><p>Even fewer protein-coding transcripts were up-regulated in response to CTD mutations: 70 (1.4%) in <em>S2A</em>; 33 (0.6%) in <em>S7A</em>; 17 (0.3%) in <em>Y1F</em>; and 3 (0.06%) in <em>T4A</em>. We surmise that the pertinent CTD coding letters or phosphorylation marks negatively regulate the transcription or processing of a limited set of fission yeast mRNAs.</p><p class="p p-last">The situation was quite different with respect to the effect of CTD mutations on the noncoding poly(A)<sup>+</sup> transcriptome, whereby <em>S2A</em> elicited up-regulation of 203 (11%) of the 1,889 annotated noncoding RNAs. By contrast, 19, 11, and 0 noncoding RNAs were up-regulated in <em>Y1F</em>, <em>S7A</em>, and <em>T4A</em> cells, respectively (<a href="/pmc/articles/PMC3964054/figure/fig01/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig01" rid-ob="ob-fig01" co-legend-rid="lgnd_fig01"><span>Fig. 1<em>D</em></span></a>). These findings implicate the Ser2 hydroxyl and/or the Ser2-PO<sub>4</sub> mark as a negative regulator of noncoding RNA biogenesis. Relatively few noncoding RNAs were down-regulated in the CTD mutants: 51 (2.7%) in <em>S2A</em>; 28 (1.5%) in <em>S7A</em>; 17 (0.9%) in <em>Y1F</em>; and 8 (0.4%) in <em>T4A</em> (<a href="/pmc/articles/PMC3964054/figure/fig01/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig01" rid-ob="ob-fig01" co-legend-rid="lgnd_fig01"><span>Fig. 1<em>C</em></span></a>).</p></div><div id="s3" class="sec"><p></p><h3 id="s3title" class="inline">RNAs Dysregulated by <em>Y1F</em> Are Coordinately Dysregulated by <em>S2A</em>. </h3><p class="p p-first">From this point forward, we will focus our analysis on the extent to which the CTD mutations exert their effects on distinct vs. overlapping sets of RNAs, as a means of gleaning clues to the syntax of the CTD code. The rationale is that, if mutations in two coding letters coordinately affect the expression of a common set of genes, the two letters might well comprise a coding word read by proteins that control expression of the shared target gene set. Inspection of the Venn diagrams in <a href="/pmc/articles/PMC3964054/figure/fig01/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig01" rid-ob="ob-fig01" co-legend-rid="lgnd_fig01"><span>Fig. 1</span></a> highlights a consistent overlap pattern, whereby the majority of the RNAs that were dysregulated by <em>Y1F</em> were coordinately dysregulated by <em>S2A</em>. For example, 32 of the 54 protein-coding RNAs down-regulated in <em>Y1F</em> cells were also repressed in <em>S2A</em> cells (<em>P</em> &#x0003c; 5.8 &#x000d7; 10<sup>&#x02212;36</sup>, hypergeometric test). The shared target set for <em>Y1F</em> and <em>S2A</em> down-regulation (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1321842111/-/DCSupplemental/pnas.201321842SI.pdf?targetid=nameddest=SF5" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S5</a>, <em>Upper</em>) is rich in genes encoding enzymes of nucleotide, sugar, and amino acid metabolism. Similarly, 9 of the 17 noncoding RNAs down-regulated by <em>Y1F</em> were also down-regulated by <em>S2A</em> (<em>P</em> &#x0003c; 7.6 &#x000d7; 10<sup>&#x02212;11</sup>; <a href="/pmc/articles/PMC3964054/figure/fig01/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig01" rid-ob="ob-fig01" co-legend-rid="lgnd_fig01"><span>Fig. 1</span></a>).</p><p class="p p-last">The coregulation theme extends to the RNAs that increase in response to CTD mutations. Ten of the 17 protein-coding RNAs that were up-regulated by <em>Y1F</em> were coordinately up-regulated by <em>S2A</em> (<em>P</em> &#x0003c; 2.1 &#x000d7; 10<sup>&#x02212;15</sup>). The shared set for up-regulation (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1321842111/-/DCSupplemental/pnas.201321842SI.pdf?targetid=nameddest=SF5" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S5</a>, <em>Lower</em>) comprises mRNAs encoding proteins involved in iron uptake and/or regulated by iron (Str3, Frp1, Fip1, Fio1, Str1, Srx1) and proteins induced during meiosis (Ppk31, Mde5, Mug136, Meu14). Seventeen of the 19 noncoding RNAs up-regulated by <em>Y1F</em> were also up-regulated by <em>S2A</em> (<em>P</em> &#x0003c; 3.1 &#x000d7; 10<sup>&#x02212;17</sup>). No comparable extent of overlap was evident for any other pair of CTD mutants. These results suggest that Tyr1 and Ser2 are a two-letter CTD coding word that governs the expression of a discrete subset of fission yeast genes.</p></div><div id="s4" class="sec"><p></p><h3 id="s4title" class="inline">Tyr1 and Ser2 Transduce Repression of an Iron Uptake Regulon. </h3><p class="p p-first">Fission yeast has a finely tuned system of iron homeostasis entailing regulated expression of genes encoding proteins involved in iron uptake (<a href="#r10" rid="r10" class=" bibr popnode">10</a>). Key players are the cell surface ferric reductase Frp1 that converts extracellular ferric ion to ferrous ion, which is then transported across the cell membrane by the oxidase&#x02013;permease complex composed of Fio1 and Fip1 proteins. A parallel pathway of iron acquisition entails the action of a ferrochrome siderophore, synthesized by Sib1 and Sib2, and cell surface siderophore transporters Str1, Str2, and Str3. The mRNAs encoding Frp1, Fio1, Fip1, Str1, and Str3 were all up-regulated in the <em>Y1F</em> strain versus the isogenic WT strain, by 16-fold, 15-fold, 14-fold, 10-fold, and 98-fold, respectively (<a href="/pmc/articles/PMC3964054/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2</span></a>). These five RNAs were also up-regulated in <em>S2A</em> cells, as was the RNA encoding Sib1. Moreover, the mRNAs encoding Frp1, Fip1, Fio1, Str3, and Str1 were up-regulated in the CTD double-mutants <em>Y1F&#x02022;S7A</em> and <em>S2A&#x02022;S7A</em> (<a href="/pmc/articles/PMC3964054/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2</span></a>), fortifying the inference that CTD coding letters Tyr1 and Ser2 act to down-regulate iron uptake genes. The RNA-seq results were verified by reverse transcriptase (RT)-qPCR analyses, which showed that the levels of mRNAs encoding Fio1, Fip1, and Frp1 were up-regulated in the <em>Y1F</em>, <em>S2A</em>, <em>Y1F&#x02022;S7A</em>, and <em>S2A&#x02022;S7A</em> mutants, but not in the <em>S7A</em> mutant (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1321842111/-/DCSupplemental/pnas.201321842SI.pdf?targetid=nameddest=SF6" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S6 <em>A</em>&#x02013;<em>C</em></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig02" co-legend-rid="lgnd_fig02"><a href="/pmc/articles/PMC3964054/figure/fig02/" target="figure" rid-figpopup="fig02" rid-ob="ob-fig02"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140408616352000"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3964054_pnas.1321842111fig02.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pnas.1321842111fig02.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3964054/bin/pnas.1321842111fig02.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140408616352000"><a target="object" rel="noopener" href="/pmc/articles/PMC3964054/figure/fig02/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig02"><div><a class="figpopup" href="/pmc/articles/PMC3964054/figure/fig02/" target="figure" rid-figpopup="fig02" rid-ob="ob-fig02">Fig. 2.</a></div><!--caption a7--><div class="caption"><p>CTD mutations <em>Y1F</em> and <em>S2A</em> derepress an iron uptake regulon. The expression levels (reads per kilobase per million reads; RPKM) of genes involved in iron uptake and utilization are shown for the biological replicates of each of the strains specified. The log<sub>2</sub> fold changes vs. WT are denoted below the expression levels. Values meeting the dual cutoff criteria of at least twofold change and <em>P</em> &#x0003c; 0.05 are highlighted in brackets in plain font. Values of at least twofold change with <em>P</em> &#x0003e; 0.05 are italicized and in brackets. Other log<sub>2</sub> fold change values not meeting either criterion are in plain font. The genes derepressed by CTD mutations are listed above the spacer row shaded gray. The transcriptional regulatory genes that repress the iron regulon are listed below the spacer row.</p></div></div></div><p>The GATA-family transcription factor Fep1 represses expression of the aforementioned iron uptake genes under iron-replete growth conditions (<a href="#r10" rid="r10" class=" bibr popnode">10</a>, <a href="#r14" rid="r14" class=" bibr popnode">14</a>). Fep1 binds in an iron-dependent manner to a GATA sequence element [5&#x02032;-ATC(A/T)GATA(A/T)] in the regulatory regions upstream of the repressible iron uptake genes. Additional iron/Fep1-regulated genes identified by expression profiling with DNA microarrays include those encoding sulfiredoxin Srx1, Fe&#x02013;S cluster assembly protein Isu1, and heme utilization protein Shu1 (<a href="#r15" rid="r15" class=" bibr popnode">15</a>). The Srx1 mRNA was up-regulated in <em>Y1F</em>, <em>S2A</em>, <em>Y1F&#x02022;S7A</em>, and <em>S2A&#x02022;S7A</em> cells (<a href="/pmc/articles/PMC3964054/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2</span></a>). The Shu1 and Isu1 transcripts were up-regulated in <em>Y1F&#x02022;S7A</em> and <em>S2A&#x02022;S7A</em> cells (<a href="/pmc/articles/PMC3964054/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2</span></a>). (They were also up-regulated in <em>Y1F</em> cells, albeit not at the <em>P</em> value cutoff of 0.05). Thus, the loss of the Tyr1 and Ser2 CTD cues specifically derepresses the regulon governed by Fep1.</p><p class="p p-last">A trivial explanation for this derepression would be that <em>Y1F</em> and <em>S2A</em> down-regulate the expression of the proteins that repress the iron regulon. These include, in addition to Fep1, the corepressors Tup11 and Tup12 (<a href="#r10" rid="r10" class=" bibr popnode">10</a>, <a href="#r14" rid="r14" class=" bibr popnode">14</a>). However, this scenario was vitiated by the findings that the Fep1, Tup11, and Tup12 mRNA levels, gauged by RNA-seq, were not affected significantly by the <em>Y1F</em>, <em>S2A</em>, <em>Y1F&#x02022;S7A</em>, or <em>S2A&#x02022;S7A</em> mutations (<a href="/pmc/articles/PMC3964054/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2</span></a>). RT-qPCR analysis verified that the level of mRNA encoding the transcriptional repressor Fep1 was unaffected by these CTD mutations (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1321842111/-/DCSupplemental/pnas.201321842SI.pdf?targetid=nameddest=SF6" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S6<em>D</em></a>). Another possibility would be that <em>Y1F</em> and <em>S2A</em> prompt overexpression of a factor that derepresses the iron regulon, e.g., by negatively regulating Fep1. Grx4 is implicated as a negative regulator of Fep1 (<a href="#r10" rid="r10" class=" bibr popnode">10</a>), but Grx4 expression was not affected significantly in <em>Y1F</em>, <em>S2A</em>, <em>Y1F&#x02022;S7A</em>, or <em>S2A&#x02022;S7A</em> cells (<a href="/pmc/articles/PMC3964054/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2</span></a>). Therefore, we infer that the CTD Tyr1-Ser2 coding word is needed to transduce a repressive signal from Fep1 to the transcription apparatus.</p></div><div id="s5" class="sec"><p></p><h3 id="s5title" class="inline"><em>Y1F</em> and <em>S2A</em> Affect Iron Homeostasis. </h3><p class="p p-first">If the derepression of the iron regulon at the mRNA level is physiologically important, we would expect to see increased activity of Fep1 regulated proteins in cells bearing the <em>Y1F</em> and <em>S2A</em> CTD mutations. We assayed ferric reductase Frp1 activity by overlaying cells grown on iron-replete medium with agar containing the ferric reductase substrate triphenyltetrazolium chloride, which is reduced by cell surface-associated Frp1 to yield a red pigmented product (<a href="#r14" rid="r14" class=" bibr popnode">14</a>). This assay distinguished the pale WT, <em>S7A</em>, and <em>T4A</em> colonies from the red <em>Y1F</em>, <em>Y1F&#x02022;S7A</em>, and <em>S2A&#x02022;S7A</em> colonies (<a href="/pmc/articles/PMC3964054/figure/fig03/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig03" rid-ob="ob-fig03" co-legend-rid="lgnd_fig03"><span>Fig. 3</span></a>, <em>Right</em>). The <em>S2A</em> strain grew more slowly under these conditions, and hence there were fewer cells on the plate, but it was still apparent that the <em>S2A</em> cells generated red pigment. Thus, the effect of CTD mutations on the activity of the Frp1 enzyme in vivo correlated with the mRNA levels gauged by RNA-seq.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig03" co-legend-rid="lgnd_fig03"><a href="/pmc/articles/PMC3964054/figure/fig03/" target="figure" rid-figpopup="fig03" rid-ob="ob-fig03"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140408623075920"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3964054_pnas.1321842111fig03.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pnas.1321842111fig03.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3964054/bin/pnas.1321842111fig03.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140408623075920"><a target="object" rel="noopener" href="/pmc/articles/PMC3964054/figure/fig03/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig03"><div><a class="figpopup" href="/pmc/articles/PMC3964054/figure/fig03/" target="figure" rid-figpopup="fig03" rid-ob="ob-fig03">Fig. 3.</a></div><!--caption a7--><div class="caption"><p><em>Y1F</em> and <em>S2A</em> derepress ferric reductase and sensitize to phleomycin. (<em>Left</em>) Phleomycin sensitivity. Midlog phase cultures of the indicated strains grown in yeast extract with supplements (YES) broth at 30 &#x000b0;C were adjusted to 6 &#x000d7; 10<sup>4</sup> cells/mL and aliquots (5 &#x000b5;L) of serial twofold dilutions were spotted on YES agar medium with no addition (control) or containing 10 &#x000b5;g/mL phleomycin. The plates were incubated at 30 &#x000b0;C and photographed after 3 d (control) or 4 d (phleomycin). (<em>Right</em>) Ferric reductase activity. Cells as specified were spotted on YES agar medium containing 50 &#x000b5;g/mL FeCl<sub>3</sub> and grown for 2 d at 30 &#x000b0;C. The colonies were overlaid with 1% Bacto agar containing 0.1% TTC and 0.067 M phosphate (pH 7). The plate was photographed after incubation for 45 min at 22 &#x000b0;C.</p></div></div></div><p class="p p-last">We surveyed the effects of the CTD mutations on the sensitivity of fission yeast to phleomycin, a glycopeptide antibiotic that cleaves the phosphodiester backbone of nucleic acids in the presence of iron and oxygen. Derepression of the Fep1-regulated iron uptake regulon causes hypersensitivity to phleomycin toxicity (<a href="#r14" rid="r14" class=" bibr popnode">14</a>) by virtue of increasing the available intracellular iron. The salient findings were that <em>Y1F</em>, <em>S2A</em>, <em>Y1F&#x02022;S7A</em>, and <em>S2A&#x02022;S7A</em> strains failed to grow in the presence of 10 &#x000b5;g/mL phleomycin, a concentration of drug that did not prevent growth of the WT, <em>S7A</em>, and <em>T4A</em> strains (<a href="/pmc/articles/PMC3964054/figure/fig03/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig03" rid-ob="ob-fig03" co-legend-rid="lgnd_fig03"><span>Fig. 3</span></a>, <em>Left</em>). We conclude from these experiments that CTD Tyr1 and Ser2 are critical informational cues for proper iron homeostasis in fission yeast.</p></div><div id="s6" class="sec"><p></p><h3 id="s6title" class="inline">Thr4 and Ser7 Govern Expression of Acid Phosphatase Pho1 and Pho1 Induction by Phosphate Starvation. </h3><p class="p p-first">Although the <em>T4A</em> CTD mutation resulted in decreased expression of merely four mRNAs, two of them encode proteins involved in phosphate homeostasis: the secreted acid phosphatase Pho1 and the inorganic phosphate transporter SPBC8E4.01c. The fission yeast phosphate transporter and Pho1 are encoded by adjacent cooriented genes on chromosome II. The Pho1 and phosphate transporter mRNAs were down-regulated by 7.5-fold and fourfold, respectively, in <em>T4A</em> cells and by twofold and 2.7-fold, respectively, in the <em>S2A</em> mutant (<a href="/pmc/articles/PMC3964054/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>A</em></span></a>). By contrast, Pho1 was up-regulated twofold in response to the <em>S7A</em> mutation (<a href="/pmc/articles/PMC3964054/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>A</em></span></a>). An instructive finding was that Pho1 was down-regulated 12-fold in the <em>T4A&#x02022;S7A</em> double mutant (<a href="/pmc/articles/PMC3964054/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>A</em></span></a>), signifying that the decrement in expression caused by loss of the Thr4 cue was dominant over the gain seen with <em>S7A</em>. The phosphate transporter mRNA was also down-regulated in <em>T4A&#x02022;S7A</em> cells (<a href="/pmc/articles/PMC3964054/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>A</em></span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig04" co-legend-rid="lgnd_fig04"><a href="/pmc/articles/PMC3964054/figure/fig04/" target="figure" rid-figpopup="fig04" rid-ob="ob-fig04"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140408623083824"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3964054_pnas.1321842111fig04.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pnas.1321842111fig04.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3964054/bin/pnas.1321842111fig04.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140408623083824"><a target="object" rel="noopener" href="/pmc/articles/PMC3964054/figure/fig04/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig04"><div><a class="figpopup" href="/pmc/articles/PMC3964054/figure/fig04/" target="figure" rid-figpopup="fig04" rid-ob="ob-fig04">Fig. 4.</a></div><!--caption a7--><div class="caption"><p>CTD mutations affect expression of acid phosphatase Pho1. (<em>A</em>) Changes in transcript levels for <em>pho1</em><sup>+</sup> and SPBC8E4.01c are shown for the indicated CTD mutant compared with WT cells. (<em>B</em>) Detection of acid phosphatase activity. Exponentially growing cultures of the indicated strains were adjusted to <em>A</em><sub>600</sub> of 0.1 and aliquots (3 &#x000b5;L) of serial fivefold dilutions were spotted on YES agar or PMG agar medium. After growth for 3 d at 30 &#x000b0;C, the colonies were overlaid with 1% Bacto agar containing 0.015% &#x003b1;-naphthyl phosphate and 0.15% Fast Blue Salt in 0.1 M sodium acetate (pH 4.2). The plates were photographed after incubation for 5 min at 22 &#x000b0;C. (<em>C</em>) Induction of acid phosphatase activity after phosphate starvation. Aliquots of exponentially growing cultures in YES medium were harvested, the cells were washed in water and adjusted to <em>A</em><sub>600</sub> of &#x0223c;0.3 in PMG medium (with 5 &#x000b5;g/mL thiamine), either with 15.5 mM phosphate (+) or without phosphate (&#x02212;) to induce Pho1. After incubation for 3 h at 30 &#x000b0;C, cells were harvested, washed, and suspended in water to attain an <em>A</em><sub>600</sub> of 1.25. To quantify acid phosphatase activity, reaction mixtures (200 &#x000b5;L) containing 100 mM sodium acetate (pH 4.2), 10 mM <em>p</em>-nitrophenylphosphate, and 0.025, 0.05, or 0.1 <em>A</em><sub>600</sub> unit of cells were incubated for 5 min at 30 &#x000b0;C. The reactions were quenched by adding 1 mL of 1 M sodium carbonate, the cells were removed by centrifugation, and the absorbance of the supernatant at 410 nm was measured. Acid phosphatase activity is expressed as the ratio of <em>A</em><sub>410</sub> (<em>p</em>-nitrophenol production) to <em>A</em><sub>600</sub> (cells). Each datum in the bar graph is the average of three assays by using cells from three independent cultures &#x000b1; SEM.</p></div></div></div><p>The RNA-seq findings that <em>pho1</em> mRNA was down-regulated in <em>T4A</em> and <em>T4A&#x02022;S7A</em> cells and up-regulated in <em>S7A</em> cells were verified by two independent methods: RT-qPCR (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1321842111/-/DCSupplemental/pnas.201321842SI.pdf?targetid=nameddest=SF7" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S7<em>D</em></a>) and primer extension (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1321842111/-/DCSupplemental/pnas.201321842SI.pdf?targetid=nameddest=SF7" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S7 <em>A</em> and <em>B</em></a>). Reverse transcriptase extension of a <sup>32</sup>P-labeled DNA primer complementary to nucleotides +33 to +51 of the <em>pho1</em> ORF revealed a unique product of identical length (&#x0223c;100 nt) in each of the yeast strains, albeit differing in abundance relative to an internal <em>act1</em> primer extension control (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1321842111/-/DCSupplemental/pnas.201321842SI.pdf?targetid=nameddest=SF7" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S7<em>A</em></a>). By analyzing the <em>pho1</em> primer extension product in parallel with a chain-terminated sequencing ladder generated by DNA polymerase-catalyzed extension of the same <sup>32</sup>P-labeled primer annealed to a <em>pho1</em> DNA template (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1321842111/-/DCSupplemental/pnas.201321842SI.pdf?targetid=nameddest=SF7" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S7<em>C</em></a>), we mapped the transcription start site to the G position 51 nt upstream of the <em>pho1</em> AUG start codon.</p><p>To gauge the effect of the CTD mutations on acid phosphatase activity, we plated serial dilutions of cells on pombe minimal glutamate (PMG) agar medium (which contains 15.5 mM inorganic phosphate), allowed the colonies to attain comparable cell densities, and then overlaid the plate with phosphatase substrate &#x003b1;-naphthyl phosphate, Fast Blue Salt B stain, and 0.1 M sodium acetate (pH 4.2). The secreted acid phosphatase causes formation of a red pigment and relative color intensity provides a semiquantitative assay of Pho1 activity (<a href="#r16" rid="r16" class=" bibr popnode">16</a>). The <em>T4A</em> and <em>T4A&#x02022;S7A</em> colonies were clearly paler than the WT, whereas the <em>S7A</em> colonies were more red than the WT (<a href="/pmc/articles/PMC3964054/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>B</em></span></a>), signifying that the amount of active phosphatase was regulated down or up by CTD mutations in step with the levels of <em>pho1</em> mRNA. Similar trends in phosphatase staining were seen when the yeast colonies were grown on YES agar medium (which contains &#x0223c;2 mM phosphate), albeit with a higher basal level of phosphatase activity (<a href="/pmc/articles/PMC3964054/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>B</em></span></a>). We surmise that the variations in phosphatase activity reflect effects on Pho1 abundance, insofar as the RNA-seq analysis showed that the level of mRNA encoding a second fission yeast phosphatase Pho4 was unaffected by the CTD mutations (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1321842111/-/DCSupplemental/pnas.201321842SI.pdf?targetid=nameddest=SF8" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S8</a>).</p><p>The RNA-seq analysis was performed on yeast cells grown in the presence of phosphate. To determine the impact of the CTD mutations on the induction of Pho1 under conditions of phosphate starvation, cells grown in YES medium were washed and then incubated for 3 h in synthetic medium containing 15.5 mM phosphate, or in medium lacking exogenous phosphate to elicit the starvation response. Acid phosphatase activity was quantified in a solution assay by incubating suspensions of equal numbers of the phosphate-replete or phosphate-starved cells for 5 min with <em>p</em>-nitrophenylphosphate and scoring colorimetrically the formation of <em>p</em>-nitrophenol. The basal phosphatase activity of WT cells was increased sixfold by phosphate starvation (<a href="/pmc/articles/PMC3964054/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>C</em></span></a>). Whereas the basal phosphatase activity of <em>T4A</em> cells was fivefold lower than WT, the basal activity of <em>S7A</em> cells was ninefold higher than WT. <em>T4A</em> cells responded to phosphate starvation, but the level of induced phosphatase activity was only half that of WT cells. The constitutively high level of phosphatase activity in S7A cells was increased further during phosphate starvation, to a level twofold higher than in starved WT cells (<a href="/pmc/articles/PMC3964054/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>C</em></span></a>). The superinduction of Pho1 activity by <em>S7A</em> was effaced completely in the <em>T4A&#x02022;S7A</em> strain, which displayed the same diminished basal and induced phosphatase activities seen in the <em>T4A</em> single mutant (<a href="/pmc/articles/PMC3964054/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>C</em></span></a>). We conclude that the Thr4 mark normally exerts a positive influence on the phosphate response, whereas the Ser7 mark exerts a strong repressive effect.</p><p>Pho1 and phosphate transporter expression are induced in response to phosphate starvation via the action of the Zn-finger transcription factor Pho7 (<a href="#r17" rid="r17" class=" bibr popnode">17</a>, <a href="#r18" rid="r18" class=" bibr popnode">18</a>). The protein kinase Csk1 is a negative regulator of Pho1 and phosphate transporter expression, insofar as both are constitutively induced in <em>csk1</em>&#x02206; cells. The fact that constitutive expression of Pho1 is abolished in a <em>csk1</em>&#x02206; <em>pho7</em>&#x02206; double mutant implies that Pho7 is essential for Pho1 transcription and acts downstream of Csk1 (<a href="#r17" rid="r17" class=" bibr popnode">17</a>, <a href="#r18" rid="r18" class=" bibr popnode">18</a>). The RNA-seq data indicate that the decrements in Pho1 and phosphate transporter expression in <em>T4A</em> and <em>T4A&#x02022;S7A</em> cells are not a consequence of decreased expression of the positive regulator Pho7 or increased expression of the negative regulator Csk1, because <em>pho1</em> and <em>csk1</em> mRNA levels were not affected significantly by the CTD mutations (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1321842111/-/DCSupplemental/pnas.201321842SI.pdf?targetid=nameddest=SF8" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S8</a>).</p><p>The in vivo chromogenic colony assay indicated that <em>csk1</em>&#x02206; cells with a WT CTD elaborated increased phosphatase activity compared with <em>csk1</em><sup>+</sup> cells (<a href="/pmc/articles/PMC3964054/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>B</em></span></a>). Quantitative assay of acid phosphatase showed that basal of <em>csk1</em>&#x02206; cells was eightfold higher than <em>csk1</em><sup>+</sup> cells (<a href="/pmc/articles/PMC3964054/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>C</em></span></a>). The increase in basal Pho1 activity was attenuated in a <em>csk1&#x02206; T4A</em> strain, as was the induction by phosphate starvation (<a href="/pmc/articles/PMC3964054/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4<em>C</em></span></a>), suggesting that Csk1 acts upstream of the CTD in modulating the phosphate response.</p><p class="p p-last">It is worth noting that the <em>T4A</em> and <em>T4A&#x02022;S7A</em> mutations did not significantly affect expression of the SPBC1271.09 gene that encodes a glycerophosphodiester transporter (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1321842111/-/DCSupplemental/pnas.201321842SI.pdf?targetid=nameddest=SF8" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S8</a>), even though SPBC1271.09 is part of the phosphate response and coordinately regulated with Pho1 and the phosphate transporter by Pho7 and Csk1 (<a href="#r19" rid="r19" class=" bibr popnode">19</a>). On the contrary, SPBC1271.09 was up-regulated along with Pho1 by the <em>S7A</em> mutation (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1321842111/-/DCSupplemental/pnas.201321842SI.pdf?targetid=nameddest=SF8" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S8</a>). These findings highlight the selective impact of <em>T4A</em> and <em>T4A&#x02022;S7A</em> on expression of the tandemly arranged phosphate transporter and acid phosphatase genes.</p></div><div id="s7" class="sec sec-last"><p></p><h3 id="s7title" class="inline"><em>S7A</em> Up-Regulates the Expression of Many Meiotic Genes. </h3><p class="p p-first">The <em>S7A</em> CTD mutation resulted in the up-regulation of 33 protein-coding mRNAs; 16 of these transcripts encode proteins with known roles in fission yeast meiosis or are RNAs known to be up-regulated during meiosis (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1321842111/-/DCSupplemental/pnas.201321842SI.pdf?targetid=nameddest=SF9" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S9</a>), accounting for their naming as meiosis up-regulated genes (Mug) or Mei4-dependent (Mde) genes (<a href="#r20" rid="r20" class=" bibr popnode">20</a><a href="#r21" rid="r21" class=" bibr popnode"></a>&#x02013;<a href="#r22" rid="r22" class=" bibr popnode">22</a>). Mei4 is a meiosis-specific forkhead transcription factor that up-regulates the expression of multiple meiotic genes, either during sexual differentiation and sporulation or when Mei4 is ectopically expressed in vegetative cells (<a href="#r20" rid="r20" class=" bibr popnode">20</a><a href="#r21" rid="r21" class=" bibr popnode"></a>&#x02013;<a href="#r22" rid="r22" class=" bibr popnode">22</a>); these include the genes encoding Spo6, Spn5, Spn6, Mfr1, Ght5, Mde3, Mde7, Mde1, Mei2, Mug130, Mug136, Mug147, and SPBC725.10 that were up-regulated in vegetative <em>S7A</em> cells. Because <em>mei4</em> was itself up-regulated by <em>S7A</em>, it is likely that Mei4 drives the observed up-regulation of several of its target genes in <em>S7A</em> cells. Our results implicate the CTD Ser7 mark as a negative influence on the expression of Mei4 and other meiotic genes.</p><p class="p p-last">The up-regulation elicited by <em>S7A</em> was largely effaced in the <em>Y1F&#x02022;S7A</em> and <em>T4A&#x02022;S7A</em> double mutants (Venn diagrams in <a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1321842111/-/DCSupplemental/pnas.201321842SI.pdf?targetid=nameddest=SF10" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S10</a>). Whereas the majority of mRNAs up-regulated in <em>Y1F</em> cells remained up-regulated in <em>Y1F&#x02022;S7A</em> cells (11 of 17), this was not the case for the set of mRNAs up-regulated by <em>S7A</em>, of which only 6 of 33 were up-regulated in <em>Y1F&#x02022;S7A</em> cells. Similarly, 2 of 3 mRNAs up-regulated by <em>T4A</em> remained up-regulated in <em>T4A&#x02022;S7A</em> cells, but only 8 of 33 mRNAs up-regulated by <em>S7A</em> were up-regulated in <em>T4A&#x02022;S7A</em> cells (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1321842111/-/DCSupplemental/pnas.201321842SI.pdf?targetid=nameddest=SF10" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S10</a>). With respect to the 16 meiotic genes up-regulated by <em>S7A</em>, 11 and 12 were not up-regulated in the <em>Y1F&#x02022;S7A</em> and <em>T4A&#x02022;S7A</em> mutants, respectively. These findings suggest that the biological output of a CTD coding letter may be contingent on the state of neighboring coding cues. This point is underscored by the observations that the <em>Y1F&#x02022;S7A</em> and <em>T4A&#x02022;S7A</em> double mutations elicited up-regulation of distinctive sets of coding mRNAs that were not up-regulated by the component single mutations (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1321842111/-/DCSupplemental/pnas.201321842SI.pdf?targetid=nameddest=SF10" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S10</a>).</p></div></div><div id="s8" class="tsec sec"><h2 class="head no_bottom_margin" id="s8title" style="text-transform: uppercase;">Discussion</h2><p class="p p-first">The CTD code is deeply rooted in eukaryal biology, yet its complexity and output can vary among taxa. The trend, albeit oversimplified, is that the number of heptad repeats comprising the CTD, the minimal number of repeats required for viability, and the prevalence of nonconsensus coding letters increases as one moves up the evolutionary ladder from unicellular to multicellular eukarya, as a means to accommodate and orchestrate the actions of increasing numbers of CTD receptors that regulate gene expression (<a href="#r1" rid="r1" class=" bibr popnode">1</a>, <a href="#r2" rid="r2" class=" bibr popnode">2</a>). It is reasonable to posit a core CTD code that governs events common to most eukarya; for example, the essential function of the Ser5-PO<sub>4</sub>&#x02013;Pro6 coding word in recruiting mRNA capping enzymes to the Pol II elongation complex, which is conserved in budding yeast, fission yeast, and mammals, notwithstanding major differences in genetic and physical organization of the capping apparatus in these taxa (<a href="#r6" rid="r6" class=" bibr popnode">6</a>, <a href="#r7" rid="r7" class=" bibr popnode">7</a>, <a href="#r19" rid="r19" class=" bibr popnode">19</a>, <a href="#r23" rid="r23" class=" bibr popnode">23</a>, <a href="#r24" rid="r24" class=" bibr popnode">24</a>). The code is plastic (by virtue of the 128 potential structures each heptad can adopt), and can easily expand its functional capacity as new receptors emerge to read the code letters and words. Accordingly, the necessity for specific CTD cues and their impact on cellular gene expression may differ among taxa. The Ser2, Thr4, Ser7, and Tyr1 hydroxyl groups are inessential for vegetative growth of fission yeast (<a href="#r6" rid="r6" class=" bibr popnode">6</a>). In metazoa, Ser2, Thr4, and Ser7 are critical for cell viability and/or specific steps of transcription or RNA processing, e.g., because these coding cues are recognized by components of the processing machinery, some of which are unique to metazoa (<a href="#r25" rid="r25" class=" bibr popnode">25</a><a href="#r26" rid="r26" class=" bibr popnode"></a>&#x02013;<a href="#r27" rid="r27" class=" bibr popnode">27</a>).</p><p>The homogeneous primary structure of the fission yeast CTD and the availability of viable <em>rpb1</em> mutants in which single coding cues (or combinations of cues) are subtracted from all the consensus heptads makes for an attractive experimental system to survey the roles of the Tyr1, Ser2, Thr4, and Ser7 phosphoacceptors in gene expression. As reported here, it was remarkable how few mRNAs were significantly dysregulated at least twofold by the CTD mutations <em>S2A</em>, <em>Y1F</em>, <em>S7A</em>, and <em>T4A</em>, which affected only 4.4%, 1.4%, 1.2%, and 0.14% of the annotated fission yeast protein-coding RNAs, respectively. We note that the method of RNA-seq analysis used here, whereby equal amounts of RNA from different fission yeast strains are subjected to high-throughput sequencing, assumes that the fission yeast strains analyzed produce similar levels of RNA per cell; we cannot exclude the possibility that a CTD mutation might elicit a global increase or decrement in the steady-state levels of all, or most, RNAs per yeast cell, an issue that has been discussed in relation to gene expression analyses in metazoan cells (<a href="#r28" rid="r28" class=" bibr popnode">28</a>). Nonetheless, the analysis is highly instructive with respect to the selective effect of CTD mutations.</p><p>RNA-seq revealed distinctive gene expression signatures for the CTD phosphoacceptors that were verified by cell-based activity assays. Two examples stand out. First, our studies highlight the role of Tyr1 and Ser2 in repressing the expression of multiple unlinked genes involved in iron uptake that comprise a Fep1-repressed regulon. Although the molecular details remain to be elucidated, we speculate that Fep1 bound to DNA flanking the target gene promoters might interact, through its C-terminal repressor domain (<a href="#r10" rid="r10" class=" bibr popnode">10</a>) or corepressors bound to the repressor domain, with the Pol II CTD to exert a gene-specific repressive effect on transcription (or cotranscriptional RNA processing) and that this interaction depends on Tyr1 and Ser2.</p><p>Second, we implicated Thr4 as an activator of the expression of two adjacent genes encoding proteins involved in phosphate homeostasis: a phosphate transporter and the acid phosphatase Pho1. The CTD <em>T4A</em> mutation coordinately reduced <em>pho1</em> RNA and Pho1 activity at the cell surface. <em>T4A</em> blunted the increase in Pho1 activity that resulted from elimination of the repressive kinase Csk1, thereby placing Thr4 downstream of Csk1, perhaps acting in tandem with the transcriptional activator Pho7. However, the observation that an unlinked third Pho7-responsive gene in the phosphate pathway was not down-regulated by <em>T4A</em> raises the prospect that a particular feature of the transporter-phosphatase locus dictates its reliance on the Thr4 CTD cue. The CTD Ser7 residue was found to exert a negative effect on Pho1 expression, whereby <em>S7A</em> coordinately increased <em>pho1</em> RNA and Pho1 activity; when <em>S7A</em> was combined with <em>T4A</em>, the latter held sway and Pho1 was down-regulated.</p><p class="p p-last">In conclusion, the marriage of CTD genetics and transcriptome profiling provides unique insights to the output of the CTD code, especially by Tyr1, Thr4, and Ser7. That said, we cannot at present distinguish whether the narrow but specific gene expression dyscrasias elicited by CTD mutations <em>Y1F</em>, <em>T4A</em>, and <em>S7A</em> reflect the lack of phosphorylation of Tyr1, Thr4, or Ser7 or, rather, interference with phosphorylation-independent functions of the respective hydroxyl groups. Making this distinction will be contingent on identifying and genetically controlling the protein kinases responsible for Tyr1, Thr4, and Ser7 phosphorylation in fission yeast, an area that is presently <em>tabula rasa</em>.</p></div><div id="s9" class="tsec sec"><h2 class="head no_bottom_margin" id="s9title" style="text-transform: uppercase;">Methods</h2><p class="p p-first-last">The isogenic heterothallic <em>S. pombe rpb1-CTD</em> strains studied herein express Rpb1 variants in which 14 consensus YSPTSPS WT heptad repeats or 14 heptad repeats with single amino acid substitutions (Y1F, S2A, T4A, S7A) or double substitutions (Y1F&#x02022;S7A, S2A&#x02022;S7A, T4A&#x02022;S7A) in each of the heptads are appended to amino acid 1577 of Rpb1 (<a href="#r6" rid="r6" class=" bibr popnode">6</a>). The mating type, determined by crossing with tester strains, is h&#x02212; for all strains studied, except for the sterile <em>S2A</em> mutant, which is h+ (as determined by diagnostic PCR using mat1-P&#x02013; and mat1-M&#x02013;specific primers). RNA was isolated from yeast cells grown in liquid culture in YES medium at 30 &#x000b0;C to an <em>A</em><sub>600</sub> of 0.4&#x02013;0.7. Cells were harvested by centrifugation and total RNA was extracted by using the Qiagen RNeasy kit. The integrity and quantity of total RNA was gauged with an Agilent Technologies 2100 Bioanalyzer. Poly(A)<sup>+</sup> RNA was purified from 1 &#x000b5;g of total RNA by using the Illumina TruSeq RNA sample preparation kit. A TruSeq stranded total RNA sample preparation kit was used for subsequent steps of poly(A)<sup>+</sup> RNA fragmentation, strand-specific cDNA synthesis, indexing, and amplification. Indexed libraries were normalized and pooled (<em>n</em> = 8 samples per lane) for paired-end sequencing performed by using an Illumina HiSeq2000/1000 system. Paired-end reads of length 51 bases originating from each sample were aligned with Bowtie 0.12.7 (<a href="#r29" rid="r29" class=" bibr popnode">29</a>) to the <em>S. pombe</em> genome sequence (Ensembl <em>S. pombe</em>, Build EF1, version 13) as well as to its corresponding exon&#x02013;exon junction database (only the second part of the paired-end reads was considered). As many as three mismatches were allowed. Reads that matched multiple loci were removed from further analysis, and the resulting alignment files were processed to generate &#x0201c;pile-ups&#x0201d; against each chromosome. Differential expression between samples was determined using the DESeq Bioconductor package (<a href="#r30" rid="r30" class=" bibr popnode">30</a>). A cutoff of plus or minus twofold change and corrected <em>P</em> value &#x0003c;0.05 were applied to derive a list of differentially expressed RNAs. The data analysis reported here was limited to the 7,022 annotated protein-coding and noncoding RNAs. Full details of the RNA-seq data processing and normalization of expression levels are provided in <a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1321842111/-/DCSupplemental/pnas.201321842SI.pdf?targetid=nameddest=STXT" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Methods</em></a>.</p></div><div id="sec-a.j.i" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="sec-a.j.ititle" style="text-transform: uppercase;">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Supporting Information:</strong>

<div class="sup-box half_rhythm" id="media-a.j.i.b.b"><a href="/pmc/articles/PMC3964054/bin/supp_111_11_4185__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>


</div></div><div id="ack-a.k.a" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.k.atitle" style="text-transform: uppercase;">Acknowledgments</h2><div class="sec"><p>This research was supported by National Institutes of Health Grant GM52470 (to B.S. and S.S.) a Wellcome Trust Senior Investigator Award (to D.A.B. and J.B.), and a Royal Society Research Merit Award (to J.B.).</p></div></div><div id="fn-group-a.k.b" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.k.btitle" style="text-transform: uppercase;">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="fn-a.k.b.a"> <p class="p p-first-last">The authors declare no conflict of interest.</p></p><p class="fn sec" id="fn-a.k.b.b"><p class="p p-first-last">This article is a PNAS Direct Submission.</p></p><p class="fn sec" id="fn-a.k.b.c"><p class="p p-first-last">Data deposition: The data reported in this paper have been deposited in the Gene Expression Omnibus (GEO) database, <a href="http://www.ncbi.nlm.nih.gov/geo" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.ncbi.nlm.nih.gov/geo</a> (accession no. <a id="__tag_363202812" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE52370" ref="/geo/query/acc.cgi?acc=GSE52370"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE52370","term_id":"52370","extlink":"1"}}</span>GSE52370</a>).</p></p><p class="fn sec" id="fn-a.k.b.d"><p class="p p-first-last">This article contains supporting information online at <a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1321842111/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.pnas.org/lookup/suppl/doi:10.1073/pnas.1321842111/-/DCSupplemental</a>.</p></p></div></div><div id="ref-list-a.k.c" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.k.ctitle" style="text-transform: uppercase;">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="r1">1. <span class="element-citation">Eick D, Geyer M. The RNA polymerase II carboxy-terminal domain (CTD) code. <span><span class="ref-journal">Chem Rev. </span>2013;<span class="ref-vol">113</span>(11):8456–8490.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23952966" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem+Rev&amp;title=The+RNA+polymerase+II+carboxy-terminal+domain+(CTD)+code&amp;author=D+Eick&amp;author=M+Geyer&amp;volume=113&amp;issue=11&amp;publication_year=2013&amp;pages=8456-8490&amp;pmid=23952966&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r2">2. <span class="element-citation">Corden JL. RNA polymerase II C-terminal domain: Tethering transcription to transcript and template. <span><span class="ref-journal">Chem Rev. </span>2013;<span class="ref-vol">113</span>(11):8423–8455.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3988834/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24040939" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem+Rev&amp;title=RNA+polymerase+II+C-terminal+domain:+Tethering+transcription+to+transcript+and+template&amp;author=JL+Corden&amp;volume=113&amp;issue=11&amp;publication_year=2013&amp;pages=8423-8455&amp;pmid=24040939&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r3">3. <span class="element-citation">Karagiannis J, Balasubramanian MK. A cyclin-dependent kinase that promotes cytokinesis through modulating phosphorylation of the carboxy terminal domain of the RNA Pol II Rpb1p sub-unit. <span><span class="ref-journal">PLoS ONE. </span>2007;<span class="ref-vol">2</span>(5):e433.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1855991/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17502918" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=A+cyclin-dependent+kinase+that+promotes+cytokinesis+through+modulating+phosphorylation+of+the+carboxy+terminal+domain+of+the+RNA+Pol+II+Rpb1p+sub-unit&amp;author=J+Karagiannis&amp;author=MK+Balasubramanian&amp;volume=2&amp;issue=5&amp;publication_year=2007&amp;pages=e433&amp;pmid=17502918&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r4">4. <span class="element-citation">Schneider S, Pei Y, Shuman S, Schwer B. Separable functions of the fission yeast Spt5 carboxyl-terminal domain (CTD) in capping enzyme binding and transcription elongation overlap with those of the RNA polymerase II CTD. <span><span class="ref-journal">Mol Cell Biol. </span>2010;<span class="ref-vol">30</span>(10):2353–2364.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2863715/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20231361" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Biol&amp;title=Separable+functions+of+the+fission+yeast+Spt5+carboxyl-terminal+domain+(CTD)+in+capping+enzyme+binding+and+transcription+elongation+overlap+with+those+of+the+RNA+polymerase+II+CTD&amp;author=S+Schneider&amp;author=Y+Pei&amp;author=S+Shuman&amp;author=B+Schwer&amp;volume=30&amp;issue=10&amp;publication_year=2010&amp;pages=2353-2364&amp;pmid=20231361&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r5">5. <span class="element-citation">Coudreuse D, et al. A gene-specific requirement of RNA polymerase II CTD phosphorylation for sexual differentiation in <em>S. pombe</em>. <span><span class="ref-journal">Curr Biol. </span>2010;<span class="ref-vol">20</span>(12):1053–1064.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20605454" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Biol&amp;title=A+gene-specific+requirement+of+RNA+polymerase+II+CTD+phosphorylation+for+sexual+differentiation+in+S.+pombe&amp;author=D+Coudreuse&amp;volume=20&amp;issue=12&amp;publication_year=2010&amp;pages=1053-1064&amp;pmid=20605454&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r6">6. <span class="element-citation">Schwer B, Shuman S. Deciphering the RNA polymerase II CTD code in fission yeast. <span><span class="ref-journal">Mol Cell. </span>2011;<span class="ref-vol">43</span>(2):311–318.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3142328/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21684186" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell&amp;title=Deciphering+the+RNA+polymerase+II+CTD+code+in+fission+yeast&amp;author=B+Schwer&amp;author=S+Shuman&amp;volume=43&amp;issue=2&amp;publication_year=2011&amp;pages=311-318&amp;pmid=21684186&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r7">7. <span class="element-citation">Schwer B, Sanchez AM, Shuman S. Punctuation and syntax of the RNA polymerase II CTD code in fission yeast. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2012;<span class="ref-vol">109</span>(44):18024–18029.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3497752/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23071310" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Punctuation+and+syntax+of+the+RNA+polymerase+II+CTD+code+in+fission+yeast&amp;author=B+Schwer&amp;author=AM+Sanchez&amp;author=S+Shuman&amp;volume=109&amp;issue=44&amp;publication_year=2012&amp;pages=18024-18029&amp;pmid=23071310&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r8">8. <span class="element-citation">Sukegawa Y, Yamashita A, Yamamoto M. The fission yeast stress-responsive MAPK pathway promotes meiosis via the phosphorylation of Pol II CTD in response to environmental and feedback cues. <span><span class="ref-journal">PloS Genet. </span>2011;<span class="ref-vol">7</span>(12):e1002387.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3228818/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22144909" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PloS+Genet&amp;title=The+fission+yeast+stress-responsive+MAPK+pathway+promotes+meiosis+via+the+phosphorylation+of+Pol+II+CTD+in+response+to+environmental+and+feedback+cues&amp;author=Y+Sukegawa&amp;author=A+Yamashita&amp;author=M+Yamamoto&amp;volume=7&amp;issue=12&amp;publication_year=2011&amp;pages=e1002387&amp;pmid=22144909&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r9">9. <span class="element-citation">Saberianfar R, Cunningham-Dunlop S, Karagiannis J. Global gene expression analysis of fission yeast mutants impaired in Ser-2 phosphorylation of the RNA pol II carboxy terminal domain. <span><span class="ref-journal">PloS ONE. </span>2011;<span class="ref-vol">6</span>(9):e24694.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3171476/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21931816" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PloS+ONE&amp;title=Global+gene+expression+analysis+of+fission+yeast+mutants+impaired+in+Ser-2+phosphorylation+of+the+RNA+pol+II+carboxy+terminal+domain&amp;author=R+Saberianfar&amp;author=S+Cunningham-Dunlop&amp;author=J+Karagiannis&amp;volume=6&amp;issue=9&amp;publication_year=2011&amp;pages=e24694&amp;pmid=21931816&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r10">10. <span class="element-citation">Labb&#x000e9; S, Khan MG, Jacques JF. Iron uptake and regulation in <em>Schizosaccharomyces pombe</em>. <span><span class="ref-journal">Curr Opin Microbiol. </span>2013;<span class="ref-vol">16</span>(6):669–676.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23916750" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Microbiol&amp;title=Iron+uptake+and+regulation+in+Schizosaccharomyces+pombe&amp;author=S+Labb&#x000e9;&amp;author=MG+Khan&amp;author=JF+Jacques&amp;volume=16&amp;issue=6&amp;publication_year=2013&amp;pages=669-676&amp;pmid=23916750&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r11">11. <span class="element-citation">Wilhelm BT, et al. Dynamic repertoire of a eukaryotic transcriptome surveyed at single-nucleotide resolution. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">453</span>(7199):1239–1243.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18488015" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Dynamic+repertoire+of+a+eukaryotic+transcriptome+surveyed+at+single-nucleotide+resolution&amp;author=BT+Wilhelm&amp;volume=453&amp;issue=7199&amp;publication_year=2008&amp;pages=1239-1243&amp;pmid=18488015&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r12">12. <span class="element-citation">Rhind N, et al. Comparative functional genomics of the fission yeasts. <span><span class="ref-journal">Science. </span>2011;<span class="ref-vol">332</span>(6032):930–936.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3131103/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21511999" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Comparative+functional+genomics+of+the+fission+yeasts&amp;author=N+Rhind&amp;volume=332&amp;issue=6032&amp;publication_year=2011&amp;pages=930-936&amp;pmid=21511999&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r13">13. <span class="element-citation">Marguerat S, et al. Quantitative analysis of fission yeast transcriptomes and proteomes in proliferating and quiescent cells. <span><span class="ref-journal">Cell. </span>2012;<span class="ref-vol">151</span>(3):671–683.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3482660/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23101633" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Quantitative+analysis+of+fission+yeast+transcriptomes+and+proteomes+in+proliferating+and+quiescent+cells&amp;author=S+Marguerat&amp;volume=151&amp;issue=3&amp;publication_year=2012&amp;pages=671-683&amp;pmid=23101633&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r14">14. <span class="element-citation">Pelletier B, Beaudoin J, Mukai Y, Labb&#x000e9; S. Fep1, an iron sensor regulating iron transporter gene expression in <em>Schizosaccharomyces pombe</em>. <span><span class="ref-journal">J Biol Chem. </span>2002;<span class="ref-vol">277</span>(25):22950–22958.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11956219" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Fep1,+an+iron+sensor+regulating+iron+transporter+gene+expression+in+Schizosaccharomyces+pombe&amp;author=B+Pelletier&amp;author=J+Beaudoin&amp;author=Y+Mukai&amp;author=S+Labb&#x000e9;&amp;volume=277&amp;issue=25&amp;publication_year=2002&amp;pages=22950-22958&amp;pmid=11956219&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r15">15. <span class="element-citation">Rustici G, et al. Global transcriptional responses of fission and budding yeast to changes in copper and iron levels: a comparative study. <span><span class="ref-journal">Genome Biol. </span>2007;<span class="ref-vol">8</span>(5):R73.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1929147/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17477863" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Global+transcriptional+responses+of+fission+and+budding+yeast+to+changes+in+copper+and+iron+levels:+a+comparative+study&amp;author=G+Rustici&amp;volume=8&amp;issue=5&amp;publication_year=2007&amp;pages=R73&amp;pmid=17477863&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r16">16. <span class="element-citation">Schweingruber ME, Schweingruber AM, Sch&#x000fc;pbach ME. Isolation and characterization of acid phosphatase mutants in <em>Schizosaccharomycespombe</em>. <span><span class="ref-journal">Curr Genet. </span>1982;<span class="ref-vol">5</span>(2):109–117.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24186226" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Genet&amp;title=Isolation+and+characterization+of+acid+phosphatase+mutants+in+Schizosaccharomycespombe&amp;author=ME+Schweingruber&amp;author=AM+Schweingruber&amp;author=ME+Sch&#x000fc;pbach&amp;volume=5&amp;issue=2&amp;publication_year=1982&amp;pages=109-117&amp;pmid=24186226&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r17">17. <span class="element-citation">Henry TC, et al. Systematic screen of <em>Schizosaccharomyces pombe</em> deletion collection uncovers parallel evolution of the phosphate signal pathways in yeasts. <span><span class="ref-journal">Euk. Cell. </span>2011;<span class="ref-vol">10</span>:198–206.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3067400/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21169418" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Euk.+Cell&amp;title=Systematic+screen+of+Schizosaccharomyces+pombe+deletion+collection+uncovers+parallel+evolution+of+the+phosphate+signal+pathways+in+yeasts&amp;author=TC+Henry&amp;volume=10&amp;publication_year=2011&amp;pages=198-206&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r18">18. <span class="element-citation">Carter-O&#x02019;Connell I, Peel MT, Wykoff DD, O&#x02019;Shea EK. Genome-wide characterization of the phosphate starvation response in <em>Schizosaccharomyces pombe</em>. <span><span class="ref-journal">BMC Genomics. </span>2012;<span class="ref-vol">13</span>:697.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3556104/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23231582" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genomics&amp;title=Genome-wide+characterization+of+the+phosphate+starvation+response+in+Schizosaccharomyces+pombe&amp;author=I+Carter-O&#x02019;Connell&amp;author=MT+Peel&amp;author=DD+Wykoff&amp;author=EK+O&#x02019;Shea&amp;volume=13&amp;publication_year=2012&amp;pages=697&amp;pmid=23231582&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r19">19. <span class="element-citation">Fabrega C, Shen V, Shuman S, Lima CD. Structure of an mRNA capping enzyme bound to the phosphorylated carboxy-terminal domain of RNA polymerase II. <span><span class="ref-journal">Mol Cell. </span>2003;<span class="ref-vol">11</span>(6):1549–1561.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12820968" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell&amp;title=Structure+of+an+mRNA+capping+enzyme+bound+to+the+phosphorylated+carboxy-terminal+domain+of+RNA+polymerase+II&amp;author=C+Fabrega&amp;author=V+Shen&amp;author=S+Shuman&amp;author=CD+Lima&amp;volume=11&amp;issue=6&amp;publication_year=2003&amp;pages=1549-1561&amp;pmid=12820968&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r20">20. <span class="element-citation">Abe H, Shimoda C. Autoregulated expression of <em>Schizosaccharomyces pombe</em> meiosis-specific transcription factor Mei4 and a genome-wide search for its target genes. <span><span class="ref-journal">Genetics. </span>2000;<span class="ref-vol">154</span>(4):1497–1508.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1461037/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10747048" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Autoregulated+expression+of+Schizosaccharomyces+pombe+meiosis-specific+transcription+factor+Mei4+and+a+genome-wide+search+for+its+target+genes&amp;author=H+Abe&amp;author=C+Shimoda&amp;volume=154&amp;issue=4&amp;publication_year=2000&amp;pages=1497-1508&amp;pmid=10747048&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r21">21. <span class="element-citation">Horie S, et al. The <em>Schizosaccharomyces pombe mei4</em><sup>+</sup> gene encodes a meiosis-specific transcription factor containing a forkhead DNA-binding domain. <span><span class="ref-journal">Mol Cell Biol. </span>1998;<span class="ref-vol">18</span>(4):2118–2129.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC121445/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9528784" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Biol&amp;title=The+Schizosaccharomyces+pombe+mei4++gene+encodes+a+meiosis-specific+transcription+factor+containing+a+forkhead+DNA-binding+domain&amp;author=S+Horie&amp;volume=18&amp;issue=4&amp;publication_year=1998&amp;pages=2118-2129&amp;pmid=9528784&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r22">22. <span class="element-citation">Mata J, Wilbrey A, B&#x000e4;hler J. Transcriptional regulatory network for sexual differentiation in fission yeast. <span><span class="ref-journal">Genome Biol. </span>2007;<span class="ref-vol">8</span>(10):R217.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2246291/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17927811" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Transcriptional+regulatory+network+for+sexual+differentiation+in+fission+yeast&amp;author=J+Mata&amp;author=A+Wilbrey&amp;author=J+B&#x000e4;hler&amp;volume=8&amp;issue=10&amp;publication_year=2007&amp;pages=R217&amp;pmid=17927811&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r23">23. <span class="element-citation">Pei Y, Hausmann S, Ho CK, Schwer B, Shuman S. The length, phosphorylation state, and primary structure of the RNA polymerase II carboxyl-terminal domain dictate interactions with mRNA capping enzymes. <span><span class="ref-journal">J Biol Chem. </span>2001;<span class="ref-vol">276</span>(30):28075–28082.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11387325" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=The+length,+phosphorylation+state,+and+primary+structure+of+the+RNA+polymerase+II+carboxyl-terminal+domain+dictate+interactions+with+mRNA+capping+enzymes&amp;author=Y+Pei&amp;author=S+Hausmann&amp;author=CK+Ho&amp;author=B+Schwer&amp;author=S+Shuman&amp;volume=276&amp;issue=30&amp;publication_year=2001&amp;pages=28075-28082&amp;pmid=11387325&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r24">24. <span class="element-citation">Ghosh A, Shuman S, Lima CD. Structural insights to how mammalian capping enzyme reads the CTD code. <span><span class="ref-journal">Mol Cell. </span>2011;<span class="ref-vol">43</span>(2):299–310.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3142331/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21683636" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell&amp;title=Structural+insights+to+how+mammalian+capping+enzyme+reads+the+CTD+code&amp;author=A+Ghosh&amp;author=S+Shuman&amp;author=CD+Lima&amp;volume=43&amp;issue=2&amp;publication_year=2011&amp;pages=299-310&amp;pmid=21683636&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r25">25. <span class="element-citation">Hsin JP, Sheth A, Manley JL. RNAP II CTD phosphorylated on threonine-4 is required for histone mRNA 3&#x02032; end processing. <span><span class="ref-journal">Science. </span>2011;<span class="ref-vol">334</span>(6056):683–686.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3678764/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22053051" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=RNAP+II+CTD+phosphorylated+on+threonine-4+is+required+for+histone+mRNA+3&#x02032;+end+processing&amp;author=JP+Hsin&amp;author=A+Sheth&amp;author=JL+Manley&amp;volume=334&amp;issue=6056&amp;publication_year=2011&amp;pages=683-686&amp;pmid=22053051&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r26">26. <span class="element-citation">Hintermair C, et al. Threonine-4 of mammalian RNA polymerase II CTD is targeted by Polo-like kinase 3 and required for transcriptional elongation. <span><span class="ref-journal">EMBO J. </span>2012;<span class="ref-vol">31</span>(12):2784–2797.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3380212/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22549466" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+J&amp;title=Threonine-4+of+mammalian+RNA+polymerase+II+CTD+is+targeted+by+Polo-like+kinase+3+and+required+for+transcriptional+elongation&amp;author=C+Hintermair&amp;volume=31&amp;issue=12&amp;publication_year=2012&amp;pages=2784-2797&amp;pmid=22549466&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r27">27. <span class="element-citation">Egloff S, et al. Serine-7 of the RNA polymerase II CTD is specifically required for snRNA gene expression. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">318</span>(5857):1777–1779.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2263945/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18079403" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Serine-7+of+the+RNA+polymerase+II+CTD+is+specifically+required+for+snRNA+gene+expression&amp;author=S+Egloff&amp;volume=318&amp;issue=5857&amp;publication_year=2007&amp;pages=1777-1779&amp;pmid=18079403&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r28">28. <span class="element-citation">Lov&#x000e9;n J, et al. Revisiting global gene expression analysis. <span><span class="ref-journal">Cell. </span>2012;<span class="ref-vol">151</span>(3):476–482.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3505597/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23101621" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Revisiting+global+gene+expression+analysis&amp;author=J+Lov&#x000e9;n&amp;volume=151&amp;issue=3&amp;publication_year=2012&amp;pages=476-482&amp;pmid=23101621&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r29">29. <span class="element-citation">Langmead B, Trapnell C, Pop M, Salzberg SL. Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. <span><span class="ref-journal">Genome Biol. </span>2009;<span class="ref-vol">10</span>(3):R25.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2690996/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19261174" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Ultrafast+and+memory-efficient+alignment+of+short+DNA+sequences+to+the+human+genome&amp;author=B+Langmead&amp;author=C+Trapnell&amp;author=M+Pop&amp;author=SL+Salzberg&amp;volume=10&amp;issue=3&amp;publication_year=2009&amp;pages=R25&amp;pmid=19261174&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r30">30. <span class="element-citation">Anders S, Huber W. Differential expression analysis for sequence count data. <span><span class="ref-journal">Genome Biol. </span>2010;<span class="ref-vol">11</span>(10):R106.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3218662/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20979621" ref="reftype=pubmed&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Differential+expression+analysis+for+sequence+count+data&amp;author=S+Anders&amp;author=W+Huber&amp;volume=11&amp;issue=10&amp;publication_year=2010&amp;pages=R106&amp;pmid=20979621&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3964054&amp;issue-id=235471&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Proceedings of the National Academy of Sciences of the United States of America</span> are provided here courtesy of <strong>National Academy of Sciences</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC3964054/pdf/pnas.201321842.pdf" class="int-view">PDF (1002K)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/3964054/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/3964054/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="3964054" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3964054%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="flWqv4e8qbXhDDlM5lRZSGTBYlqq2oFFC7UG7N1lQgtXRSiz5hi9u64DZ8nZ2lt1">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3964054%2F&amp;text=Individual%20letters%20of%20the%20RNA%20polymerase%20II%20CTD%20code%20govern%20distinct%20gene%20expression%20programs%20in%20fission%20yeast" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3964054%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3964054/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC3964054/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/24591591/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC3964054/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/24591591/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC3964054/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/3964054/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC3964054/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
