
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.04fe9e55f2b2.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.aafaa535cb7f.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.ca6dfa7b90b5.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs3.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Full genome re-sequencing reveals a novel circadian clock mutation in Arabidopsis - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="D0BD4A2F1629AEB5000031A61EC53342.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="1.6.0.post2+24fead6" />

<meta name="ncbi_domain" content="genbio" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC3129678/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3129678/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Genome Biology" /><meta name="citation_title" content="Full genome re-sequencing reveals a novel circadian clock mutation in Arabidopsis" /><meta name="citation_author" content="Kevin Ashelford" /><meta name="citation_author_institution" content="School of Biological Sciences, University of Liverpool, Crown Street, Liverpool L69 7ZB, UK" /><meta name="citation_author" content="Maria E Eriksson" /><meta name="citation_author_institution" content="Department of Plant Physiology, Umea Plant Science Centre, Umea University, SE-901 87 Umea, Sweden" /><meta name="citation_author" content="Christopher M Allen" /><meta name="citation_author_institution" content="Applied Biosystems, 120 Birchwood Boulevard, Warrington WA3 7QH, UK" /><meta name="citation_author" content="Rosalinda D'Amore" /><meta name="citation_author_institution" content="School of Biological Sciences, University of Liverpool, Crown Street, Liverpool L69 7ZB, UK" /><meta name="citation_author" content="Mikael Johansson" /><meta name="citation_author_institution" content="Department of Plant Physiology, Umea Plant Science Centre, Umea University, SE-901 87 Umea, Sweden" /><meta name="citation_author" content="Peter Gould" /><meta name="citation_author_institution" content="School of Biological Sciences, University of Liverpool, Crown Street, Liverpool L69 7ZB, UK" /><meta name="citation_author" content="Suzanne Kay" /><meta name="citation_author_institution" content="School of Biological Sciences, University of Liverpool, Crown Street, Liverpool L69 7ZB, UK" /><meta name="citation_author" content="Andrew J Millar" /><meta name="citation_author_institution" content="Institute of Molecular Plant Sciences, School of Biological Sciences, University of Edinburgh, Mayfield Road, Edinburgh EH9 3JH, UK" /><meta name="citation_author" content="Neil Hall" /><meta name="citation_author_institution" content="School of Biological Sciences, University of Liverpool, Crown Street, Liverpool L69 7ZB, UK" /><meta name="citation_author" content="Anthony Hall" /><meta name="citation_author_institution" content="School of Biological Sciences, University of Liverpool, Crown Street, Liverpool L69 7ZB, UK" /><meta name="citation_publication_date" content="2011" /><meta name="citation_issue" content="3" /><meta name="citation_volume" content="12" /><meta name="citation_firstpage" content="R28" /><meta name="citation_doi" content="10.1186/gb-2011-12-3-r28" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3129678/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC3129678/" /><meta name="citation_pmid" content="21429190" /><meta name="DC.Title" content="Full genome re-sequencing reveals a novel circadian clock mutation in Arabidopsis" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="BMC" /><meta name="DC.Contributor" content="Kevin Ashelford" /><meta name="DC.Contributor" content="Maria E Eriksson" /><meta name="DC.Contributor" content="Christopher M Allen" /><meta name="DC.Contributor" content="Rosalinda D'Amore" /><meta name="DC.Contributor" content="Mikael Johansson" /><meta name="DC.Contributor" content="Peter Gould" /><meta name="DC.Contributor" content="Suzanne Kay" /><meta name="DC.Contributor" content="Andrew J Millar" /><meta name="DC.Contributor" content="Neil Hall" /><meta name="DC.Contributor" content="Anthony Hall" /><meta name="DC.Date" content="2011" /><meta name="DC.Identifier" content="10.1186/gb-2011-12-3-r28" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Full genome re-sequencing reveals a novel circadian clock mutation in Arabidopsis" /><meta property="og:type" content="article" /><meta property="og:description" content="Map based cloning in Arabidopsis thaliana can be a difficult and time-consuming process, specifically if the phenotype is subtle and scoring labour intensive. Here, we have re-sequenced the 120-Mb genome of a novel Arabidopsis clock mutant early bird ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3129678/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/7/">Genome Biol</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/198214/">v.12(3); 2011</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC3129678
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC3129678/pdf/gb-2011-12-3-r28.pdf" class="int-view">PDF (723K)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/3129678/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/3129678/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="3129678" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3129678%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="suqubEP0XvmbcNiRS8Au8DBRLLUHkXc9xRHlkvRSxHbmasXWMhAaHasdMry8RwA4">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3129678%2F&amp;text=Full%20genome%20re-sequencing%20reveals%20a%20novel%20circadian%20clock%20mutation%20in%20Arabidopsis" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3129678%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3129678/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC3129678/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/21429190/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC3129678/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/21429190/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC3129678/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/3129678/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/7/">Genome Biol</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/198214/">v.12(3); 2011</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC3129678
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-genbio.png" alt="Logo of genbio" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="https://genomebiology.biomedcentral.com/" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Genome Biol.</span> 2011; 12(3): R28. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2011 Mar 23. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1186%2Fgb-2011-12-3-r28" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1186/gb-2011-12-3-r28</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3129678</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/21429190">21429190</a></div></div></div><h1 class="content-title">Full genome re-sequencing reveals a novel circadian clock mutation in <em>Arabidopsis</em></h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ashelford%20K%5BAuthor%5D" id="A1" class="affpopup" co-rid="_co_idm139996016907184" co-class="co-affbox">Kevin Ashelford</a>,<sup>#</sup><sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Eriksson%20ME%5BAuthor%5D" id="A2" class="affpopup" co-rid="_co_idm139996019185008" co-class="co-affbox">Maria E Eriksson</a>,<sup>#</sup><sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Allen%20CM%5BAuthor%5D" id="A3" class="affpopup" co-rid="_co_idm139996019213504" co-class="co-affbox">Christopher M Allen</a>,<sup>3</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=D%27Amore%20R%5BAuthor%5D" id="A4" class="affpopup" co-rid="_co_idm139996026797792" co-class="co-affbox">Rosalinda D'Amore</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Johansson%20M%5BAuthor%5D" id="A5" class="affpopup" co-rid="_co_idm139996034455232" co-class="co-affbox">Mikael Johansson</a>,<sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gould%20P%5BAuthor%5D" id="A6" class="affpopup" co-rid="_co_idm139996029263072" co-class="co-affbox">Peter Gould</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kay%20S%5BAuthor%5D" id="A7" class="affpopup" co-rid="_co_idm139996022656000" co-class="co-affbox">Suzanne Kay</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Millar%20AJ%5BAuthor%5D" id="A8" class="affpopup" co-rid="_co_idm139996024197024" co-class="co-affbox">Andrew J Millar</a>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hall%20N%5BAuthor%5D" id="A9" class="affpopup" co-rid="_co_idm139996026237120" co-class="co-affbox">Neil Hall</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>1</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hall%20A%5BAuthor%5D" id="A10" class="affpopup" co-rid="_co_idm139996021690144" co-class="co-affbox">Anthony Hall</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139996016907184"><h3 class="no_margin">Kevin Ashelford</h3><p><sup>1</sup>School of Biological Sciences, University of Liverpool, Crown Street, Liverpool L69 7ZB, UK</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ashelford%20K%5BAuthor%5D">Kevin Ashelford</a></div></div><div id="_co_idm139996019185008"><h3 class="no_margin">Maria E Eriksson</h3><p><sup>2</sup>Department of Plant Physiology, Umea Plant Science Centre, Umea University, SE-901 87 Umea, Sweden</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Eriksson%20ME%5BAuthor%5D">Maria E Eriksson</a></div></div><div id="_co_idm139996019213504"><h3 class="no_margin">Christopher M Allen</h3><p><sup>3</sup>Applied Biosystems, 120 Birchwood Boulevard, Warrington WA3 7QH, UK</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Allen%20CM%5BAuthor%5D">Christopher M Allen</a></div></div><div id="_co_idm139996026797792"><h3 class="no_margin">Rosalinda D'Amore</h3><p><sup>1</sup>School of Biological Sciences, University of Liverpool, Crown Street, Liverpool L69 7ZB, UK</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=D%27Amore%20R%5BAuthor%5D">Rosalinda D'Amore</a></div></div><div id="_co_idm139996034455232"><h3 class="no_margin">Mikael Johansson</h3><p><sup>2</sup>Department of Plant Physiology, Umea Plant Science Centre, Umea University, SE-901 87 Umea, Sweden</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Johansson%20M%5BAuthor%5D">Mikael Johansson</a></div></div><div id="_co_idm139996029263072"><h3 class="no_margin">Peter Gould</h3><p><sup>1</sup>School of Biological Sciences, University of Liverpool, Crown Street, Liverpool L69 7ZB, UK</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gould%20P%5BAuthor%5D">Peter Gould</a></div></div><div id="_co_idm139996022656000"><h3 class="no_margin">Suzanne Kay</h3><p><sup>1</sup>School of Biological Sciences, University of Liverpool, Crown Street, Liverpool L69 7ZB, UK</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kay%20S%5BAuthor%5D">Suzanne Kay</a></div></div><div id="_co_idm139996024197024"><h3 class="no_margin">Andrew J Millar</h3><p><sup>4</sup>Institute of Molecular Plant Sciences, School of Biological Sciences, University of Edinburgh, Mayfield Road, Edinburgh EH9 3JH, UK</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Millar%20AJ%5BAuthor%5D">Andrew J Millar</a></div></div><div id="_co_idm139996026237120"><h3 class="no_margin">Neil Hall</h3><p><sup>1</sup>School of Biological Sciences, University of Liverpool, Crown Street, Liverpool L69 7ZB, UK</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hall%20N%5BAuthor%5D">Neil Hall</a></div></div><div id="_co_idm139996021690144"><h3 class="no_margin">Anthony Hall</h3><p><sup>1</sup>School of Biological Sciences, University of Liverpool, Crown Street, Liverpool L69 7ZB, UK</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hall%20A%5BAuthor%5D">Anthony Hall</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm139996016664688_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139996016664688_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139996016664688_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm139996016664688_ai" style="display:none"><div class="fm-affl" id="I1"><sup>1</sup>School of Biological Sciences, University of Liverpool, Crown Street, Liverpool L69 7ZB, UK</div><div class="fm-affl" id="I2"><sup>2</sup>Department of Plant Physiology, Umea Plant Science Centre, Umea University, SE-901 87 Umea, Sweden</div><div class="fm-affl" id="I3"><sup>3</sup>Applied Biosystems, 120 Birchwood Boulevard, Warrington WA3 7QH, UK</div><div class="fm-affl" id="I4"><sup>4</sup>Institute of Molecular Plant Sciences, School of Biological Sciences, University of Edinburgh, Mayfield Road, Edinburgh EH9 3JH, UK</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup>Corresponding author.</div><div><sup>#</sup>Contributed equally.</div><div><span class="contrib-email" id="A1">Kevin Ashelford: <a href="mailto:dev@null" data-email="ku.ca.looprevil@aekbas" class="oemail">ku.ca.looprevil@aekbas</a>; </span> <span class="contrib-email" id="A2">Maria E Eriksson: <a href="mailto:dev@null" data-email="es.umu.syhptnalp@nosskire.airam" class="oemail">es.umu.syhptnalp@nosskire.airam</a>; </span> <span class="contrib-email" id="A3">Christopher M Allen: <a href="mailto:dev@null" data-email="moc.hcetefil@nellA.rehpotsirhC" class="oemail">moc.hcetefil@nellA.rehpotsirhC</a>; </span> <span class="contrib-email" id="A4">Rosalinda D'Amore: <a href="mailto:dev@null" data-email="ku.ca.looprevil@eromad.adnil" class="oemail">ku.ca.looprevil@eromad.adnil</a>; </span> <span class="contrib-email" id="A5">Mikael Johansson: <a href="mailto:dev@null" data-email="es.umu.syhptnalp@nossnahoj.leakim" class="oemail">es.umu.syhptnalp@nossnahoj.leakim</a>; </span> <span class="contrib-email" id="A6">Peter Gould: <a href="mailto:dev@null" data-email="ku.ca.looprevil@gdpjlmom" class="oemail">ku.ca.looprevil@gdpjlmom</a>; </span> <span class="contrib-email" id="A7">Suzanne Kay: <a href="mailto:dev@null" data-email="ku.ca.looprevil@yaks" class="oemail">ku.ca.looprevil@yaks</a>; </span> <span class="contrib-email" id="A8">Andrew J Millar: <a href="mailto:dev@null" data-email="ku.ca.de@rallim.werdnanA" class="oemail">ku.ca.de@rallim.werdnanA</a>; </span> <span class="contrib-email" id="A9">Neil Hall: <a href="mailto:dev@null" data-email="ku.ca.vil@llah.lieN" class="oemail">ku.ca.vil@llah.lieN</a>; </span> <span class="contrib-email" id="A10">Anthony Hall: <a href="mailto:dev@null" data-email="ku.ca.vil@llah.ynohtna" class="oemail">ku.ca.vil@llah.ynohtna</a></span> </div></div><div class="fm-article-notes hide half_rhythm" id="idm139996016664688_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2010 Dec 30; Revised 2011 Feb 16; Accepted 2011 Mar 23.</div></div><div class="permissions half_rhythm hide" id="idm139996016664688_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  &#x000a9;2011 Ashelford et al.; licensee BioMed Central Ltd.</div><div class="license half_rhythm">This is an open access article distributed under the terms of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by/2.0" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://creativecommons.org/licenses/by/2.0</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="9" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Additional file 1 <strong>Figure S1 - plant to plant variation in clock function is greater in Col-0 than in Ws-2</strong>. Seedlings were entrained under 12-h light/12-h dark cycles for 12 days, after which they were transferred to constant light where rhythms of leaf movement were assayed. Ws-2, filled squares; Col-0, empty squares. Period estimates for individual seedlings are plotted against their relative amplitude errors (R.A.E.).</div><div><a href="/pmc/articles/PMC3129678/bin/gb-2011-12-3-r28-S1.PDF" data-ga-action="click_feat_suppl">gb-2011-12-3-r28-S1.PDF</a><span style="color:gray"> (37K)</span></div><div class="small guid">GUID: 8ED0A181-F9A7-4375-BC04-5082B85B7781</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Additional file 2 <strong>Table S1 - sequence tag counts available at various stages of the analysis, as reported by the different matching schema employed</strong>.</div><div><a href="/pmc/articles/PMC3129678/bin/gb-2011-12-3-r28-S2.PDF" data-ga-action="click_feat_suppl">gb-2011-12-3-r28-S2.PDF</a><span style="color:gray"> (52K)</span></div><div class="small guid">GUID: 438D7AFE-2DDC-4A13-A5C6-944B2B0B01F5</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Additional file 3 <strong>Table S2 - SNP counts before and after filtering as reported by the various matching schema</strong>. <sup>a </sup>Unfiltered SNPs were all those reported by the Corona lite SNP detection pipeline. <sup>b</sup>Filtering involved retaining only those SNP loci where tag coverage exceeded 5&#x000d7; in both <em>ebi-1 </em>and Ws-2, the SOLiD score was 0.7 or greater, and SNPs were homozygous. <strong>(c) </strong>'Schema screened' SNPs were those filtered SNPs reported by all five schema.</div><div><a href="/pmc/articles/PMC3129678/bin/gb-2011-12-3-r28-S3.PDF" data-ga-action="click_feat_suppl">gb-2011-12-3-r28-S3.PDF</a><span style="color:gray"> (52K)</span></div><div class="small guid">GUID: C25FBF5F-99E8-4833-B102-8DDFDFFAAE7D</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Additional file 4 <strong>Table S3 - dCAPS and CAPS marker design and use to validate SNP discovery</strong>. SNP marker denotes the chromosome position of the SNP based on the TAIR 8 <em>Arabidopsis </em>genome build. In the primer sequence the underlined base is the mismatched base in the primer sequence. ^Borderline SNP; <sup>a</sup>SNP in the clock gene <em>PRR7</em>; <sup>b</sup>SNP in At5g05660, <em>EBI</em>.</div><div><a href="/pmc/articles/PMC3129678/bin/gb-2011-12-3-r28-S4.PDF" data-ga-action="click_feat_suppl">gb-2011-12-3-r28-S4.PDF</a><span style="color:gray"> (32K)</span></div><div class="small guid">GUID: 1D3DF741-456E-419C-ACA4-DA16223FD84A</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Additional file 5 <strong>Table S4 - EMS-induced SNPs on chromosome 5</strong>.</div><div><a href="/pmc/articles/PMC3129678/bin/gb-2011-12-3-r28-S5.PDF" data-ga-action="click_feat_suppl">gb-2011-12-3-r28-S5.PDF</a><span style="color:gray"> (158K)</span></div><div class="small guid">GUID: BADBDAD3-444E-4D75-B07C-1C3C1DAA1F12</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Additional file 6 <strong>Table S5 - EMS-induced SNPs on chromosome 1</strong>.</div><div><a href="/pmc/articles/PMC3129678/bin/gb-2011-12-3-r28-S6.PDF" data-ga-action="click_feat_suppl">gb-2011-12-3-r28-S6.PDF</a><span style="color:gray"> (110K)</span></div><div class="small guid">GUID: D1E8A3D8-FAEC-45EC-AF89-FC14D536C3EA</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Additional file 7 <strong>Figure S2 - the presence or absence of EMS-induced mutations on chromosome 1 do not affect the phenotype of <em>ebi-1</em></strong>. Transgenic seedlings carrying the <em>LUC </em>reporter gene fused to the <em>CAB2 </em>promoter were entrained under 12-h light/12-h dark cycles for 7 days, after which luminescence was monitored in constant red light. WT, open squares; <em>ebi-1</em>, closed squares; <em>ebi-1 </em>with no EMS-induced SNP on chromosome 1, red triangles.</div><div><a href="/pmc/articles/PMC3129678/bin/gb-2011-12-3-r28-S7.PDF" data-ga-action="click_feat_suppl">gb-2011-12-3-r28-S7.PDF</a><span style="color:gray"> (44K)</span></div><div class="small guid">GUID: 24FFB0E8-4C44-4D2B-8301-111F5151489B</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Additional file 8 <strong>Table S6 - analysis of temporal expression patterns of non-synonymous SNPs on chromosome 5 using Diurnal to fit temporal expression data to expression pattern models consistent with circadian regulation</strong>.</div><div><a href="/pmc/articles/PMC3129678/bin/gb-2011-12-3-r28-S8.PDF" data-ga-action="click_feat_suppl">gb-2011-12-3-r28-S8.PDF</a><span style="color:gray"> (35K)</span></div><div class="small guid">GUID: E92F6863-C5F9-422F-8163-57F284C52640</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Additional file 9 <strong>Figure S3 - identification of a SNP in <em>PRR7</em></strong>. Top: A schematic representation of the PRR7 protein in <em>Arabidopsis </em>ecotype Columbia is shown in green. Gray boxes represent the two conserved region Receiver (REC) domain and CCT motif. The amino acids were aligned using the ClustalW program. Bottom: identical and similar amino acid residues are highlighted with black and gray backgrounds, respectively. The SNP leads to a change from arginine (R) to histidine (H) at position 329. The frame shows the residue in the Pseudo Response Regulator protein from <em>Arabidopsis </em>ecotype Columbia (<a class="tag_hotlink" href="/protein/BAB13742" ref="/protein/BAB13742"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"BAB13742","term_id":"10281004","term_text":"BAB13742"}}</span>BAB13742</a>, PRR7), <em>Hordeum vulgare </em>subsp. <em>vulgare </em>(<a class="tag_hotlink" href="/protein/AAY17586" ref="/protein/AAY17586"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"AAY17586","term_id":"62868803","term_text":"AAY17586"}}</span>AAY17586</a>, PRR), <em>Arabidopsis thaliana </em>(<a class="tag_hotlink" href="/protein/AAY62604" ref="/protein/AAY62604"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"AAY62604","term_id":"67005935","term_text":"AAY62604"}}</span>AAY62604</a>, PRR3), <em>Triticum aestivum </em>(<a class="tag_hotlink" href="/protein/ABL09464" ref="/protein/ABL09464"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"ABL09464","term_id":"118638616","term_text":"ABL09464"}}</span>ABL09464</a>, PRR), <em>Oryza sativa Indica </em>(<a class="tag_hotlink" href="/protein/BAD38858" ref="/protein/BAD38858"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"BAD38858","term_id":"51571883","term_text":"BAD38858"}}</span>BAD38858</a>, PRR 37), <em>Oryza sativa Indica </em>(<a class="tag_hotlink" href="/protein/BAD38859" ref="/protein/BAD38859"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"BAD38859","term_id":"51571885","term_text":"BAD38859"}}</span>BAD38859</a>, PRR73), <em>Lemna paucicostata </em>(<a class="tag_hotlink" href="/protein/BAE72697" ref="/protein/BAE72697"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"BAE72697","term_id":"84570625","term_text":"BAE72697"}}</span>BAE72697</a>, PRR37), <em>Lemna gibba </em>(<a class="tag_hotlink" href="/protein/BAE72700" ref="/protein/BAE72700"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"BAE72700","term_id":"84570631","term_text":"BAE72700"}}</span>BAE72700</a>, PRR37), obtained from NCBI database, and <em>Gossypium raimondii </em>(TC272), <em>Brassica napus </em>(TC71410), <em>Brassica napus </em>(TC78134), <em>Gossypium raimondii </em>(TC82653), and <em>Citrus clementina </em>(TC8380) obtained from TGI databases.</div><div><a href="/pmc/articles/PMC3129678/bin/gb-2011-12-3-r28-S9.PDF" data-ga-action="click_feat_suppl">gb-2011-12-3-r28-S9.PDF</a><span style="color:gray"> (330K)</span></div><div class="small guid">GUID: B623134A-3493-4176-B619-23356417018A</div></div></dd></dl></div><div id="abstract-a.t.b.t" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.t.b.ttitle">Abstract</h2><!--article-meta--><div><p class="p p-first-last">Map based cloning in <em>Arabidopsis thaliana </em>can be a difficult and time-consuming process, specifically if the phenotype is subtle and scoring labour intensive. Here, we have re-sequenced the 120-Mb genome of a novel <em>Arabidopsis </em>clock mutant <em>early bird (ebi-1</em>) in Wassilewskija (Ws-2). We demonstrate the utility of sequencing a backcrossed line in limiting the number of SNPs considered. We identify a SNP in the gene <em>AtNFXL-2 </em>as the likely cause of the <em>ebi-1 </em>phenotype.</p></div></div><div id="sec-a.u.a" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.u.atitle">Background</h2><p class="p p-first"><em>Arabidopsis </em>has a sequenced reference genome of 120 Mb from the Columbia (Col-0) accession [<a href="#B1" rid="B1" class=" bibr popnode">1</a>]. It has been used extensively as a model organism to understand plant development, physiology, and metabolism (reviewed in [<a href="#B2" rid="B2" class=" bibr popnode">2</a>]). Much of our understanding of these processes has come through the isolation and molecular characterization of chemically induced mutations in genes involved in these processes. Until recently, identifying the mutated gene required the tedious process of map-based cloning.</p><p>Map-based cloning in <em>Arabidopsis </em>involves out-crossing the mutant plant with a divergent <em>Arabidopsis </em>accession, usually Col-0 or Landsberg <em>erecta </em>(L<em>er</em>). In the F<sub>2 </sub>generation, the mutant phenotype is scored and molecular markers are then used to rough map the gene. Finally, plants with intra-chromosomal recombination events are used to narrow down the genetic interval [<a href="#B3" rid="B3" class=" bibr popnode">3</a>]. The processes can be complicated by natural variation in the phenotype being mapped between the two parental lines used to produce a mapping population [<a href="#B4" rid="B4" class=" bibr popnode">4</a>]. Also, recombination frequency has been shown to vary across the genome [<a href="#B5" rid="B5" class=" bibr popnode">5</a>,<a href="#B6" rid="B6" class=" bibr popnode">6</a>] with low recombination frequencies hindering fine mapping. Finally, the whole mapping processes can be difficult if the mutant phenotype is subtle and if assaying the phenotype is labor intensive.</p><p>The circadian clock is an endogenous 24-h timer found in most eukaryotes and photosynthetic bacteria. In plants, the clock plays a key role driving rhythms in physiology, biochemistry and metabolism [<a href="#B7" rid="B7" class=" bibr popnode">7</a>]. In <em>Arabidopsis</em>, our current model of the clock is a series of inter-locking feedback loops [<a href="#B8" rid="B8" class=" bibr popnode">8</a>]. Identification of many of the clock and clock-associated components has come through genetic screens, using the <em>CHLOROPHYLL A/B-BINDING PROTEIN2 (CAB2</em>) promoter fused to the <em>LUCIFERASE (LUC</em>) reporter gene to assay clock function [<a href="#B9" rid="B9" class=" bibr popnode">9</a>]. Through this approach mutants with long, short or arrhythmic circadian phenotypes have been identified and cloned using map-based approaches [<a href="#B10" rid="B10" class=" bibr popnode">10</a>-<a href="#B12" rid="B12" class=" bibr popnode">12</a>]. However, the phenotypic scoring of clock mutants is time consuming and natural variation in the clock phenotypes between <em>Arabidopsis </em>accessions can further slow down the mapping process.</p><p>An alternative to map-based cloning would be to directly sequence the whole genome of a mutant to uncover the mutation, potentially a SNP, that is responsible for the phenotype. Re-sequencing arrays do exist for <em>Arabidopsis</em>, although their high error rate of approximately 50% makes them unreliable for identifying single SNPs [<a href="#B13" rid="B13" class=" bibr popnode">13</a>]. Direct re-sequencing has already been successfully used to identify point mutations in the 15.4-Mb genome of the yeast <em>Pichia stipitis </em>[<a href="#B14" rid="B14" class=" bibr popnode">14</a>] and in <em>Caenorhabditis elegans </em>[<a href="#B15" rid="B15" class=" bibr popnode">15</a>]. Whole genome re-sequencing approaches like that of Sarin <em>et al. </em>[<a href="#B15" rid="B15" class=" bibr popnode">15</a>] are of limited use if, like in <em>Arabidopsis</em>, the ethyl methanesulfonate (EMS) mutation load is high. Therefore, a method of reducing the number of point mutations must be considered. One such method [<a href="#B16" rid="B16" class=" bibr popnode">16</a>,<a href="#B17" rid="B17" class=" bibr popnode">17</a>] has combined bulk segregation analysis with genome re-sequencing, thus generating both sequence and allelic frequency data. While this approach is again useful and extremely powerful, it relies on the ability to accurately score mutants in an F<sub>2 </sub>mapping cross and has all the limitations we have discussed with regards to map-based cloning.</p><p class="p p-last">Here, we re-sequence the 120-Mb genome of a novel <em>Arabidopsis </em>clock mutant <em>early bird (ebi-1</em>) and the corresponding wild type, Wassilewskija (Ws-2), using Applied Biosystems SOLiD, sequencing by ligation technology. We reduce the number of point mutations by sequencing a backcrossed line. We further narrow down the SNPs by investigating gene expression data for mutated genes. Finally, we use the new SNP data to exclude a known clock gene and identify a SNP in the gene <em>AtNFXL-2 </em>as the likely cause of the <em>ebi-1 </em>phenotype.</p></div><div id="sec-a.u.b" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.u.btitle">Results</h2><div id="sec-a.u.b.b" class="sec sec-first"><h3 id="sec-a.u.b.btitle">The isolation of the circadian clock mutant <em>early bird-1</em></h3><p class="p p-first">The <em>ebi-1 </em>mutant was identified in a screen for mutants with altered temporal expression of <em>CAB2 </em>from an EMS-mutagenized population. The M<sub>2 </sub>population was generated from the Ws-2 accession of <em>Arabidopsis </em>carrying the <em>CAB2:LUC+ </em>reporter construct (transgenic line 6A, Nottingham Arabidopsis Stock Centre (NASC) ID N9352). The screen involved growing plants in 12-h light/12-h dark cycles before screening <em>LUC </em>activity over 36 h in constant darkness [<a href="#B18" rid="B18" class=" bibr popnode">18</a>]. The <em>ebi-1 </em>mutant was isolated as a plant with a 1.5- to 2-h early peak phase of <em>CAB2 </em>expression in constant dark (Figure <a href="/pmc/articles/PMC3129678/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1a</span></span><span>1a</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3129678/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139996031251120"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gb-2011-12-3-r28-1.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is gb-2011-12-3-r28-1.jpg" src="/pmc/articles/PMC3129678/bin/gb-2011-12-3-r28-1.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm139996031251120"><a target="object" rel="noopener" href="/pmc/articles/PMC3129678/figure/F1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3129678/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p><strong><em>ebi-1 </em>causes the circadian clock to oscillate with a short period</strong>. <strong>(a,b) </strong>Transgenic seedlings carrying the <em>LUC </em>reporter gene fused to the <em>CAB2 </em>promoter were entrained under 12-h light/12-h dark cycles for 7 days, after which luminescence was monitored in either constant darkness (a) or constant red light (measured in counts/second, CPS) (b): WT, open squares; <em>ebi-1</em>, closed squares. The plots are representative of multiple experiments and are an average of between 24 and 79 individual seedlings; error bars are standard error of the mean. The inset in (b) is a mathematical analysis of the experiment represented in (b): period estimates for individual seedlings plotted against their relative amplitude errors (R.A.E.). <strong>(c) </strong>Representative leaf movement plots for WT (open squares) and <em>ebi-1 </em>(closed squares).</p></div></div></div><p>To clarify whether the early phase was the result of altered circadian clock function in the <em>ebi-1 </em>mutant, we analyzed <em>CAB2 </em>expression under constant red light. Under these conditions <em>CAB2 </em>expression in the <em>ebi-1 </em>mutant oscillated with short period (wild type (WT), 23.3 h, standard error (SE) 0.06, n = 53; <em>ebi-1</em>, 22.4 h, SE 0.05, n = 79; Figure <a href="/pmc/articles/PMC3129678/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure1b),</span></span><span>1b</span></a>), consistent with the early phase of <em>CAB2 </em>expression in the dark. To further investigate the phenotype, we assayed circadian rhythms of leaf movement under constant white light (Figure <a href="/pmc/articles/PMC3129678/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1c).</span></span><span>1c</span></a>). Similarly, the leaves in the <em>ebi-1 </em>mutant oscillated with a shorter period than the WT (WT, 24.6 h, SE 0.11, n = 12; <em>ebi-1</em>, 23.5 h, SE 0.05, n = 11). Although the phenotype is subtle, it is comparable to the 1-h period difference observed for the <em>cca1-11 </em>and <em>lhy-21 </em>mutants [<a href="#B19" rid="B19" class=" bibr popnode">19</a>]. Our data are supportive of the <em>ebi-1 </em>mutant perturbing multiple clock outputs. Furthermore, the <em>ebi-1 </em>mutation appears to affect equally the clock output in darkness (as manifested by an early phase) and light, suggesting it has a light-independent effect, and its primary defect may therefore not be in the light signaling pathway. Collectively, these results suggest that <em>ebi-1 </em>plays a role in the central circadian system of <em>Arabidopsis</em>.</p><p class="p p-last">To positional clone <em>ebi-</em>1, we took a standard approach, out crossing <em>ebi-1 </em>with Col-0, then re-isolating <em>ebi-1 </em>mutants in the F<sub>2 </sub>mapping population. This process was very difficult for two reasons: firstly, because of the subtle phenotype of the mutant and the stochastic variation in clock timing from one individual to another, the mutant and WT clock phenotypes overlapped (Figure <a href="/pmc/articles/PMC3129678/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1b,</span></span><span>1b</span></a>, inset); secondly, there is more plasticity in clock function in Col-0 compared to the mutated background Ws-2 (Additional file <a href="#S1" rid="S1" class=" supplementary-material">1</a>). Therefore, in parallel to the mapping, we sequenced the genomes of Ws-2 and <em>ebi-1 </em>in an attempt to identify candidate polymorphisms.</p></div><div id="sec-a.u.b.c" class="sec"><h3 id="sec-a.u.b.ctitle">Sequencing the genomes of WS-2 and <em>ebi-1</em></h3><p class="p p-first">The <em>ebi-1 </em>mutant was backcrossed four times with the original parent line (Ws-2 <em>CAB2:LUC+ </em>6A, used to generate the EMS population) to remove EMS-induced SNPs not associated with the phenotype. Whole genomic DNA was isolated from the original parent Ws-2 <em>CAB2:LUC+ </em>6A and the backcrossed <em>ebi-1 </em>mutant.</p><p>In total, 8 Gbp (<em>ebi-1</em>) and 8.5 Gbp (Ws-2, N9352) of raw color-space sequence data were generated for this study using the ABI SOLiD (version 2) sequencing machine. The number of uniquely mapping tags available for SNP calling after mapping to the Col-0 reference genome is summarized in Additional file <a href="#S2" rid="S2" class=" supplementary-material">2</a> and varied between 26.7 and 39.5% of the total depending on genome and schema used. Also depending on the schema used, an average of 12.9% of the genome failed to have any tags mapping to it, which likely resulted from a combination of coverage, insertions, deletions and hyper-variable regions between Ws-2 and Col-0. In this project we focused exclusively on SNPs because insertion and deletion are not associated with EMS mutagenesis.</p><p>SNP counts before and after filtering are summarized in Additional file <a href="#S3" rid="S3" class=" supplementary-material">3</a>. Filtering criteria were determined empirically; working on the assumption that all loci for both mutant and WT should be homozygous, any SNP reported as heterozygous was considered, <em>a priori</em>, to be low confidence (an assumption confirmed by the fact that the majority occurred within obvious repeat-rich regions of the reference genome). The assumption was based on the fact that we knew that the SNP responsible for the phenotype would be homozygous. On this basis, selection criteria were identified that minimize the numbers of heterozygous SNPs, whilst maximizing the number of homozygous, and thus potentially high-confidence, SNPs. Output from the corona_lite SNP-discovery pipeline (Life Technologies, Foster city, CA, USA) provided several parameters for assessing the quality of SNP calls. We found that two parameters in particular, coverage and SNP score, when applied simultaneously to both genomes, were most effective at eliminating false positive SNPs.</p><p>By ignoring loci below a threshold coverage depth on either of the genomes being compared, we could eliminate many low-confidence SNPs. It was important to consider loci with sufficiently high coverage for two reasons: to adequately distinguish real SNPs from the ubiquitous low background of false positives generated through systematic error; and to ensure loci on both genomes were sufficiently covered to allow for SNP calling (a SNP shared by <em>ebi-1 </em>and Ws-2 could be mistaken for a SNP unique to one or other of these genomes if coverage in one or the other was too low).</p><p>Secondly, we found that the SOLiD SNP score provided a robust means of filtering out low-confidence SNPs. The higher the score the greater the confidence in the SNP, the score being weighted to take into account the location of the SNP within the read. Thus, SNP calls relying on more error-prone bases towards the distal end of reads were scored lower than those supported by base calls at the proximal end. The method is schematically illustrated in Figure <a href="/pmc/articles/PMC3129678/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure2</span></span><span>2</span></a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3129678/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139996022893728"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3129678_gb-2011-12-3-r28-2.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gb-2011-12-3-r28-2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3129678/bin/gb-2011-12-3-r28-2.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139996022893728"><a target="object" rel="noopener" href="/pmc/articles/PMC3129678/figure/F2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3129678/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p><strong>Schematic representation of the analysis pathway used in this study</strong>. In this two step process, (1) a list of putative SNPs, relative to Col-0, were generated for each genome (<em>ebi-1 </em>and Ws-2) for each of the five possible matching schemas (25_2, 25_3, 35_2, 35_3, and 35_4) used by the Corona_lite software pipeline. Then (2), considering each chromosome (chr1, chr2, chr3, chr4, chr5, mitochondrial chromosome (chrM), and chloroplast (chrC)) in turn, the results of each schema were analyzed and filtered, and finally merged to form a collection of high-confidence SNPS used in the subsequent analysis (summarized in Tables 1 and 2).</p></div></div></div><p>To this end, based on an analysis of the data, only those SNPs reported where coverage exceeded 5&#x000d7; in both <em>ebi-1 </em>and Ws-2 and with a SOLiD score of 0.7 or greater were considered. We found that these cutoff values applied equally to all five of the matching schemas used.</p><p>Nevertheless, even after application of this filtering regime, examination of the remaining SNPs revealed that an unacceptably high number of low-confidence SNP calls were being reported regardless of matching schema employed (Additional file <a href="#S3" rid="S3" class=" supplementary-material">3</a>); interestingly, these were not the same low-confidence SNPs for each of the different schemas. Investigation revealed that the reason for this was that the different schema varied in their sensitivity to the various filtering strategies used. Thus, applying our filtering regime to schemas allowing the fewest mismatches (for example, 35_2) resulted in SNPs predominately being discarded due to too low coverage. Conversely, the same regime applied to higher mismatch schemas (for example, 35_4) led to more SNPs being eliminated due to a poor score.</p><p>The reason for this observation is clear: allowing for fewer mismatches resulted in fewer reads successfully mapping to the reference, leading to lower coverage overall, hence more loci being discarded because coverage was too low for one or other of the genomes. Conversely, accommodating more mismatches led to a higher depth of coverage, but also an increased number of SNPs called from the more error-prone proximal end and thus with poorer SNP scores.</p><p>We took advantage of this difference in filtering sensitivity to increase our filtering stringency: thus, cross-referencing results from all schemas, we identified SNPs that had high enough coverage in both genomes to be identified by low-mismatch schema, whilst at the same time having sufficiently high SNP scores to enable identification by the higher mismatch schema. The resulting SNPs are summarized in Tables <a href="/pmc/articles/PMC3129678/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Tables1</span></span><span>1</span></a> and <a href="/pmc/articles/PMC3129678/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and2.</span></span><span>2</span></a>. As a very conservative approach, we decided to cross-reference the results of all five of the schemas used (25_2, 25_3, 35_3, 35_4, 35_5). Whilst undoubtedly a highly conservative approach, with schema 25_2 in particular providing very strict matching criteria, we found that excluding the 25-mer schemas did not greatly increase the number of true SNPs whilst allowing more low-confidence SNPs. The limitation of this conservative strategy was that 11.5% of the genome had reads but failed to meet the filtering criteria and was therefore not interrogated for SNPs.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p>Enumeration of SNPs detected between <em>Arabidopsis </em>accessions Ws-2 and Col-0, according to chromosome</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th align="center" colspan="7" rowspan="1">Intragenic SNPs</th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th></tr><tr><th rowspan="1" colspan="1"></th><th colspan="7" rowspan="1"><hr /></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th></tr><tr><th rowspan="1" colspan="1"></th><th align="center" colspan="5" rowspan="1">Coding sequence</th><th align="center" colspan="2" rowspan="1">Non-coding sequence</th><th rowspan="1" colspan="1"></th><th align="center" colspan="2" rowspan="1">Total SNPs</th></tr><tr><th rowspan="1" colspan="1"></th><th colspan="5" rowspan="1"><hr /></th><th colspan="2" rowspan="1"><hr /></th><th rowspan="1" colspan="1"></th><th colspan="2" rowspan="1"><hr /></th></tr><tr><th align="left" rowspan="1" colspan="1">Chromosome</th><th align="center" rowspan="1" colspan="1">Synonymous</th><th align="center" rowspan="1" colspan="1">Non-synonymous</th><th align="center" rowspan="1" colspan="1">Stop created</th><th align="center" rowspan="1" colspan="1">Stop deleted</th><th align="center" rowspan="1" colspan="1">Unclassifiable</th><th align="center" rowspan="1" colspan="1">Pseudogene</th><th align="center" rowspan="1" colspan="1">Intronic</th><th align="center" rowspan="1" colspan="1">Intergenic SNPs</th><th align="center" rowspan="1" colspan="1">Apparent<sup>a</sup></th><th align="center" rowspan="1" colspan="1">Actual<sup>b</sup></th></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1">Chr1</td><td align="center" rowspan="1" colspan="1">8,559</td><td align="center" rowspan="1" colspan="1">6,608</td><td align="center" rowspan="1" colspan="1">54</td><td align="center" rowspan="1" colspan="1">19</td><td align="center" rowspan="1" colspan="1">4</td><td align="center" rowspan="1" colspan="1">25</td><td align="center" rowspan="1" colspan="1">10,144</td><td align="center" rowspan="1" colspan="1">14,292</td><td align="center" rowspan="1" colspan="1">39,705</td><td align="center" rowspan="1" colspan="1">37,381</td></tr><tr><td align="left" rowspan="1" colspan="1">Chr2</td><td align="center" rowspan="1" colspan="1">4,091</td><td align="center" rowspan="1" colspan="1">3,394</td><td align="center" rowspan="1" colspan="1">33</td><td align="center" rowspan="1" colspan="1">10</td><td align="center" rowspan="1" colspan="1">0</td><td align="center" rowspan="1" colspan="1">10</td><td align="center" rowspan="1" colspan="1">5,125</td><td align="center" rowspan="1" colspan="1">11,661</td><td align="center" rowspan="1" colspan="1">24,324</td><td align="center" rowspan="1" colspan="1">23,134</td></tr><tr><td align="left" rowspan="1" colspan="1">Chr3</td><td align="center" rowspan="1" colspan="1">6,141</td><td align="center" rowspan="1" colspan="1">4,945</td><td align="center" rowspan="1" colspan="1">36</td><td align="center" rowspan="1" colspan="1">6</td><td align="center" rowspan="1" colspan="1">7</td><td align="center" rowspan="1" colspan="1">11</td><td align="center" rowspan="1" colspan="1">7,341</td><td align="center" rowspan="1" colspan="1">13,607</td><td align="center" rowspan="1" colspan="1">32,094</td><td align="center" rowspan="1" colspan="1">30,496</td></tr><tr><td align="left" rowspan="1" colspan="1">Chr4</td><td align="center" rowspan="1" colspan="1">4,055</td><td align="center" rowspan="1" colspan="1">3,219</td><td align="center" rowspan="1" colspan="1">17</td><td align="center" rowspan="1" colspan="1">9</td><td align="center" rowspan="1" colspan="1">37</td><td align="center" rowspan="1" colspan="1">8</td><td align="center" rowspan="1" colspan="1">4,468</td><td align="center" rowspan="1" colspan="1">7,787</td><td align="center" rowspan="1" colspan="1">19,600</td><td align="center" rowspan="1" colspan="1">18,498</td></tr><tr><td align="left" rowspan="1" colspan="1">Chr5</td><td align="center" rowspan="1" colspan="1">7,810</td><td align="center" rowspan="1" colspan="1">5,924</td><td align="center" rowspan="1" colspan="1">35</td><td align="center" rowspan="1" colspan="1">15</td><td align="center" rowspan="1" colspan="1">6</td><td align="center" rowspan="1" colspan="1">18</td><td align="center" rowspan="1" colspan="1">9,062</td><td align="center" rowspan="1" colspan="1">14,309</td><td align="center" rowspan="1" colspan="1">37,179</td><td align="center" rowspan="1" colspan="1">35,278</td></tr><tr><td align="left" rowspan="1" colspan="1">Total (%)</td><td align="center" rowspan="1" colspan="1">30,656 (20.04)</td><td align="center" rowspan="1" colspan="1">24,090 (15.76)</td><td align="center" rowspan="1" colspan="1">175 (0.11)</td><td align="center" rowspan="1" colspan="1">59 (0.04)</td><td align="center" rowspan="1" colspan="1">54 (0.03)</td><td align="center" rowspan="1" colspan="1">72 (0.05)</td><td align="center" rowspan="1" colspan="1">36,140 (23.64)</td><td align="center" rowspan="1" colspan="1">61,656 (40.32)</td><td align="center" rowspan="1" colspan="1">152,902 (100.0)</td><td align="center" rowspan="1" colspan="1">144,787</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139996024184400"><a target="object" rel="noopener" href="/pmc/articles/PMC3129678/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p>Protein coding gene locations were extracted from the latest TAIR 8 genome release, with information extracted from TIGR xml formatted files cross-referenced with FASTA formatted sequence files. SNPs within coding sequence (CDS) regions were classified as either synonymous (silent) or non-synonymous (amino acid changing) mutations, or as causing the creation or deletion of stop codons. In 11 instances, across the entire genome, inconsistency in the documented CDS locations prevented unambiguous classification of SNPs falling within these CDS regions; such SNPs are recorded under the category 'unclassifiable'. Similarly, SNPs falling within transcriptional units marked as pseudogenes could not be classified. All other SNPs falling within documented transcriptional units, but outside of specified CDS regions, are marked as intronic. All SNPs located out of the documented transcriptional units are classified as intergenic. <sup>a</sup>Apparent number of SNPs based on the fact that splice variation means some SNPs will be scored twice. <sup>b</sup>Actual number of SNPs.</p></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p>Enumeration of SNPs detected between <em>Arabidopsis ebi-1 </em>and Ws-2 according to chromosome</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th align="center" colspan="4" rowspan="1">Intragenic</th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th></tr><tr><th rowspan="1" colspan="1"></th><th colspan="4" rowspan="1"><hr /></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th></tr><tr><th rowspan="1" colspan="1"></th><th align="center" colspan="3" rowspan="1">CDS</th><th align="center" rowspan="1" colspan="1">Non-CDS</th><th rowspan="1" colspan="1"></th><th align="center" colspan="2" rowspan="1">Total SNPs</th></tr><tr><th rowspan="1" colspan="1"></th><th colspan="3" rowspan="1"><hr /></th><th colspan="1" rowspan="1"><hr /></th><th rowspan="1" colspan="1"></th><th colspan="2" rowspan="1"><hr /></th></tr><tr><th rowspan="1" colspan="1"></th><th align="center" rowspan="1" colspan="1">Synonymous</th><th align="center" rowspan="1" colspan="1">Non-synonymous</th><th align="center" rowspan="1" colspan="1">Stop created</th><th align="center" rowspan="1" colspan="1">Intronic</th><th align="center" rowspan="1" colspan="1">Intergenic</th><th align="center" rowspan="1" colspan="1">Apparent</th><th align="center" rowspan="1" colspan="1">Actual</th></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1">Chr1</td><td align="center" rowspan="1" colspan="1">6</td><td align="center" rowspan="1" colspan="1">9</td><td align="center" rowspan="1" colspan="1">1</td><td align="center" rowspan="1" colspan="1">7</td><td align="center" rowspan="1" colspan="1">7</td><td align="center" rowspan="1" colspan="1">30</td><td align="center" rowspan="1" colspan="1">27</td></tr><tr><td align="left" rowspan="1" colspan="1">Chr2</td><td align="center" rowspan="1" colspan="1">0</td><td align="center" rowspan="1" colspan="1">0</td><td align="center" rowspan="1" colspan="1">0</td><td align="center" rowspan="1" colspan="1">0</td><td align="center" rowspan="1" colspan="1">0</td><td align="center" rowspan="1" colspan="1">0</td><td align="center" rowspan="1" colspan="1">0</td></tr><tr><td align="left" rowspan="1" colspan="1">Chr3</td><td align="center" rowspan="1" colspan="1">0</td><td align="center" rowspan="1" colspan="1">1</td><td align="center" rowspan="1" colspan="1">0</td><td align="center" rowspan="1" colspan="1">1</td><td align="center" rowspan="1" colspan="1">2</td><td align="center" rowspan="1" colspan="1">4</td><td align="center" rowspan="1" colspan="1">4</td></tr><tr><td align="left" rowspan="1" colspan="1">Chr4</td><td align="center" rowspan="1" colspan="1">0</td><td align="center" rowspan="1" colspan="1">2</td><td align="center" rowspan="1" colspan="1">0</td><td align="center" rowspan="1" colspan="1">1</td><td align="center" rowspan="1" colspan="1">0</td><td align="center" rowspan="1" colspan="1">3</td><td align="center" rowspan="1" colspan="1">2</td></tr><tr><td align="left" rowspan="1" colspan="1">Chr5</td><td align="center" rowspan="1" colspan="1">15</td><td align="center" rowspan="1" colspan="1">38</td><td align="center" rowspan="1" colspan="1">0</td><td align="center" rowspan="1" colspan="1">17</td><td align="center" rowspan="1" colspan="1">14</td><td align="center" rowspan="1" colspan="1">84</td><td align="center" rowspan="1" colspan="1">76</td></tr><tr><td align="left" rowspan="1" colspan="1">Total</td><td align="center" rowspan="1" colspan="1">21</td><td align="center" rowspan="1" colspan="1">50</td><td align="center" rowspan="1" colspan="1">1</td><td align="center" rowspan="1" colspan="1">26</td><td align="center" rowspan="1" colspan="1">23</td><td align="center" rowspan="1" colspan="1">121</td><td align="center" rowspan="1" colspan="1">109</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139996021932848"><a target="object" rel="noopener" href="/pmc/articles/PMC3129678/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p>CDS, coding sequence.</p></div></div><p>The accuracy of the SNP calling was validated using 454 sequencing. A single run of a 454-FLX sequencer (Roche) was carried out using Titanium&#x02122; chemistry on a whole genome shotgun library of the Ws-2 strain. This generated roughly 3&#x000d7; coverage of the genome (data not shown). SNPs were called using the Newbler read mapping software against the chromosome 5 sequence and the results compared to the SOLiD SNP calls. The software only called SNPs where there were data in the forward and reverse directions and where there were at least three reads. We only compared SNPs where the 454 phred score was &#x02265;40 and the SNP was not adjacent to a homo-polymer. The 454 data called 15,751 SNPs at this threshold on chromosome 5; this low number reflects the reduced coverage using 454 and the scoring threshold used. Of these, 15,597 were also called using SOLiD, indicating that our SNP calls were correctly identifying at least 99% of the SNPs present between the two varieties.</p><p class="p p-last">To further validate our scoring and ability to accurately predict SNPs, we tested 17 SNPs between <em>ebi-1 </em>and Ws-2 on chromosome 5 and 4 SNPs on chromosome 1 using cleaved amplified polymorphic (CAPS) and derived cleaved amplified polymorphic (dCAPS) markers [<a href="#B20" rid="B20" class=" bibr popnode">20</a>]. All 21 SNPs were validated. In addition, we considered five borderline SNPs, which had been filtered out because of low coverage either because they were below threshold scoring or they were not identified in all schemas. Of these borderline SNPs, four failed to be confirmed and one was heterozygous (Additional file <a href="#S4" rid="S4" class=" supplementary-material">4</a>). Both the 454 and the validation using CAPS/dCAPS markers together supported the accuracy of our SNP detection and our scoring and threshold setting.</p></div><div id="sec-a.u.b.d" class="sec"><h3 id="sec-a.u.b.dtitle">Variation between Ws-2 and Col-0</h3><p class="p p-first-last">Using our SOLiD data we identified 144,797 SNPs shared by Ws-2 and <em>ebi-1 </em>between Col-0. We also observed far fewer mutations leading to protein truncation (expected 5% under neutral selection, observed 0.4%) or amino acid substitutions (expected 65% under neutral selection, observed 44%) than predicted by chance, supporting natural selection against these types of mutations (Table <a href="/pmc/articles/PMC3129678/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table1).</span></span><span>1</span></a>). As the aim of this re-sequencing project was to identify EMS-induced SNPs between Ws-2 and <em>ebi-1</em>, we made no attempt to identify deletions or to <em>de novo </em>assemble sequences that failed to align with the reference. The number of SNPs we identified was far lower than that reported between Burren, Eire (Bur-0) and Col-0 (549,064) and between Tsu, Japan (Tsu-1) and Col-0 (483,352) [<a href="#B21" rid="B21" class=" bibr popnode">21</a>]. This is likely due to the relatively close geographical proximity of Col-0 (Germany) and Ws-2 (Ukraine) on the same land mass.</p></div><div id="sec-a.u.b.e" class="sec"><h3 id="sec-a.u.b.etitle">Ethyl methanesulfonate-induced SNPs in <em>ebi-1</em></h3><p class="p p-first">To identify the EMS-induced SNPs in <em>ebi-1</em>, we compared the sequence generated for both lines. While 144, 797 SNPs between Col-0 and Ws-2 were shared between Ws-2 and <em>ebi-1</em>, 109 were unique to <em>ebi-1 </em>(Table <a href="/pmc/articles/PMC3129678/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table2).</span></span><span>2</span></a>). Based on an 8.5-Mb region of chromosome 5, we would estimate a mutation rate of approximately 1 mutation per 112 kb. This is still likely to be an underestimate as we have not considered repetitive DNA within this region. The figure closely matches previous estimates from a large-scale TILLING project using a comparable EMS dose and calculated as being 1 mutation per 170 kb [<a href="#B22" rid="B22" class=" bibr popnode">22</a>]. We found that approximately 29.3% of mutations in genes were synonymous and 70.7% non-synonymous/nonsense, which reflects the rate expected under neutral selection. This is consistent with the fact that little selection had been placed on the plants other than their ability to set viable seed.</p><p class="p">The EMS-induced SNPs were not spread evenly over the genome but were grouped on the north arm of chromosome 5 (76) and to a lesser extent on chromosome 1 (27) (Figure <a href="/pmc/articles/PMC3129678/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3).</span></span><span>3</span></a>). The groupings, rather than a random distribution, were the result of backcrossing <em>ebi-1 </em>with the original parent. Rough mapping had placed the mutation on the north arm of chromosome 5 and the grouping of EMS mutations on chromosome 5 was the result of mutations 'hitchhiking' with the <em>ebi-1 </em>mutation during the backcrossing processes. All mutations were consistent with those expected from EMS G/C to A/T transitions [<a href="#B22" rid="B22" class=" bibr popnode">22</a>]. However, what we had expected was that mutation types would be random, that is, equal numbers of G to A and C to T, and this was not the case. In the clustered group of EMS mutations on chromosome 5, 96% of the mutations were C to T transitions (Additional file <a href="#S5" rid="S5" class=" supplementary-material">5</a>), whereas 100% of the mutations on chromosome 1 were G to A transitions (Additional file <a href="#S6" rid="S6" class=" supplementary-material">6</a>). This is probably because the plant had arisen from germ-line cells that inherited only a single alkylated strand of DNA for each chromosome: a daughter cell of an original mutated cell line. Thus, mutations will have occurred in only one direction. In plants, previous studies have looked at bias in populations of EMS mutant plants rather than in single plants. This is also an excellent indication of the accuracy with which we are identifying SNPs and that the thresholds we have set are unlikely to have identified false positive SNPs.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC3129678/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139996032559728"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3129678_gb-2011-12-3-r28-3.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gb-2011-12-3-r28-3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3129678/bin/gb-2011-12-3-r28-3.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139996032559728"><a target="object" rel="noopener" href="/pmc/articles/PMC3129678/figure/F3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC3129678/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p><strong>Location of <em>ebi-1 </em>SNPs relative to Ws-2</strong>. SNPs occurring in either <em>ebi-1 </em>only (blue circles) or Ws only (red squares), relative to Col-0, are plotted at their respective chromosome locations. The overall depth of coverage of unique tags is plotted in grey. Coverage depths of all data are determined from 35_4 schema results.</p></div></div></div></div><div id="sec-a.u.b.f" class="sec"><h3 id="sec-a.u.b.ftitle">A functional genomic approach to identifying the <em>ebi-1 </em>mutation</h3><p class="p p-first">Rough mapping had already confirmed that <em>ebi-1 </em>was located in the north arm of chromosome 5. Furthermore, using the EMS mutations on chromosome 1, backcrossed lines were identified that failed to have the EMS mutated region on chromosome 1. These lines still displayed an <em>ebi-1 </em>phenotype (Additional file <a href="#S7" rid="S7" class=" supplementary-material">7</a>); therefore, we focused on the chromosome 5 SNPs, where 32 of the 76 SNPs were non-synonymous. Based on the assumption that most clock components are themselves rhythmically expressed, we investigated the circadian expression pattern of the 32 non-synonymous SNP-containing genes using Diurnal [<a href="#B23" rid="B23" class=" bibr popnode">23</a>,<a href="#B24" rid="B24" class=" bibr popnode">24</a>]. We considered two transcriptomic experiments where seedlings had been entrained in 12-h light/12-h dark cycles and their gene expression then assayed in constant light [<a href="#B25" rid="B25" class=" bibr popnode">25</a>,<a href="#B26" rid="B26" class=" bibr popnode">26</a>] and a third where seedlings had been entrained in constant light with temperature cycles with their gene expression assayed upon transfer to constant dark [<a href="#B27" rid="B27" class=" bibr popnode">27</a>]. We screened the temporal expression pattern of 32 SNP-containing genes, scoring an expression profile as rhythmic if it had a correlation (&#x0003e;0.85) with an expression pattern model consistent with circadian regulation (Additional file <a href="#S8" rid="S8" class=" supplementary-material">8</a>). Only one SNP-containing gene was robustly rhythmic in all our tested conditions, <em>PSEUDO RESPONSE REGULATOR 7 (PRR7</em>, At5g02810; 0.95 correlation with a circadian time (ct) 7-h spike and 0.93 correlation with a ct 6-h spike in the constant light data sets, and a 0.87 correlation with a ct 6-h spike in the constant dark data set. A second gene, <em>AtNFXL-2 </em>(At5g05660), a zinc finger transcription factor, was not rhythmic in constant light but had a 0.91 correlation with a sine wave in constant dark and was therefore a strong potential candidate. Two other genes, At5g19850, a predicted hydrolase, and At5g12470, an organelle protein of unknown function, had good correlation with a cosine wave but only in one set of the constant light data. All other genes failed to show rhythmic patterns of expression.</p><p>The obvious strong candidate was the non-synonymous SNP in <em>PRR7</em>. Sanger sequencing and a dCAPS marker were used to validate the SNP. The gene <em>PRR7 </em>has already been shown to play a key role in the circadian clock, with the T-DNA insertion mutant <em>prr7-3 </em>causing a lengthening of the circadian period [<a href="#B28" rid="B28" class=" bibr popnode">28</a>], opposite to the affect of <em>ebi-1</em>. The point mutation in <em>PRR7 </em>in <em>ebi-1 </em>caused an R to be substituted with an H. However, the amino acid did not lie in a functional domain and was not conserved across species; in fact, in <em>Brassica napus</em>, the endogenous <em>PRR7 </em>has an H at this position (Additional file <a href="#S9" rid="S9" class=" supplementary-material">9</a>).</p><p class="p">The other strong candidate SNP, based on the circadian regulation and molecular function, was in <em>AtNFXL-2</em>. The mutation caused a C to T transition, which was confirmed by Sanger sequencing and a dCAPS marker. The AtNFXL-2 protein shares homology with the mammalian zinc finger transcription factor <em>NF-X1 </em>[<a href="#B29" rid="B29" class=" bibr popnode">29</a>]. <em>Arabidopsis </em>has two NF-X1-like genes, <em>AtNFXL-1 </em>(At1g10170) and <em>AtNFXL-2 </em>(At5g05660) [<a href="#B30" rid="B30" class=" bibr popnode">30</a>]. No previous study has suggested a role for the <em>AtNFXL </em>genes in the circadian clock. The SNP resulted in an amino acid substitution (V to I) in the gene At5g05660. The valine is relatively conserved across species and is either valine or methionine and lies within a zinc finger motif (Figure <a href="/pmc/articles/PMC3129678/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure4).</span></span><span>4</span></a>). However, in the <em>Arabidopsis </em>homolog, <em>AtNFXL-1</em>, the residue is a leucine.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC3129678/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139996022162032"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gb-2011-12-3-r28-4.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is gb-2011-12-3-r28-4.jpg" src="/pmc/articles/PMC3129678/bin/gb-2011-12-3-r28-4.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm139996022162032"><a target="object" rel="noopener" href="/pmc/articles/PMC3129678/figure/F4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC3129678/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><p><strong>Alignment of the conserved regions of NFXL proteins across plant taxa</strong>. The amino acids were aligned using the ClustalW program using the following sequences: [gi: 168037431], <em>Physcomitrella patens</em>; [gi: 218187558], <em>Oryza sativa</em>; [gi: 224028969], <em>Zea mays</em>; [gi: 242052039], <em>Sorghum bicolor</em>; [gi: 56694214], <em>Solanum lycopersicum</em>; [gi: 145357676], <em>Arabidopsis thalina</em>; [gi: 297810665], <em>Arabidopsis lyrata</em>; [gi: 157351181], <em>Vitis vinifera</em>; [gi:224112501], <em>Populus trichocarpa</em>. Identical and similar amino acid residues are highlighted with blue and light blue, respectively. The location of the V to I SNP within a zinc finger motif is highlighted in red.</p></div></div></div></div><div id="sec-a.u.b.g" class="sec sec-last"><h3 id="sec-a.u.b.gtitle">Validating the SNP in <em>AtNFXL-2 </em>as the SNP responsible for the <em>ebi-1 </em>phenotype</h3><p class="p p-first">From our functional genomics analysis two clear candidate SNPs remained. Based on the location of the SNP in a conserved domain, <em>AtNFXL-2 </em>was a strong candidate. We used SNP markers for <em>AtNFXL-2 </em>and <em>PRR7</em>, identified by our re-sequencing of <em>ebi-1</em>, to screen a backcrossed <em>ebi-1 </em>F<sub>2 </sub>population to identify recombinant individuals. To exclude the mutation in <em>PRR7</em>, we identified two lines (<em>ebi-1-clean-1 </em>and <em>ebi-1-clean-2</em>) that contained the <em>AtNFXL-2 </em>SNP but were WT for the <em>PRR7 </em>gene. We then identified a further two lines (<em>prr7-clean-1 </em>and <em>prr7-clean-2</em>) that were WT for <em>AtNFXL-2 </em>but retained the <em>PRR7 </em>SNP. We analyzed <em>CAB2 </em>expression under constant red light in all the lines. Both <em>ebi-1-clean-1 </em>and <em>ebi-1-clean-2 </em>had phenotypes identical to the original <em>ebi-1 </em>mutant while <em>prr7-clean-1 </em>and <em>prr7-clean-2 </em>had almost WT phenotypes, thus demonstrating that the mutation in <em>PRR7 </em>does not contribute significantly to the <em>ebi-1 </em>phenotype (Figure <a href="/pmc/articles/PMC3129678/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5a).</span></span><span>5a</span></a>). Furthermore, by combining new mapping data with SNP information, we were able to further narrow down the candidate SNPs to the <em>AtNFXL-2 </em>SNP, which lies between molecular markers nga158 and CIW18, thus excluding <em>PRR7</em>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC3129678/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139996024209328"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gb-2011-12-3-r28-5.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is gb-2011-12-3-r28-5.jpg" src="/pmc/articles/PMC3129678/bin/gb-2011-12-3-r28-5.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm139996024209328"><a target="object" rel="noopener" href="/pmc/articles/PMC3129678/figure/F5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC3129678/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5</a></div><!--caption a7--><div class="caption"><p><strong>A T-DNA allele of <em>ebi-2 </em>that results in a reduction in <em>EBI </em>expression and crossing out the <em>PRR7 </em>SNP result in similar clock phenotypes to <em>ebi-1</em>, supporting that the circadian phenotype of <em>ebi-1 </em>is due to a SNP in At5g05660</strong>. Transgenic seedlings carrying the <em>LUC </em>reporter gene fused to the <em>CAB2 </em>promoter were entrained under 12-h light/12-h dark cycles for 7 days, after which luminescence was monitored in either constant darkness or constant red light. <strong>(a) </strong>Analysis of <em>CAB2 </em>activity under constant red light at 22&#x000b0;C in: <em>ebi-1-clean-1</em>, the <em>ebi-1 </em>mutant with a WT <em>PRR7 </em>gene (closed triangles); the <em>ebi-1 </em>mutant (closed squares); <em>prr7-clean-1</em>, the <em>prr7 </em>mutant with WT <em>ebi-1 </em>(open triangles) and WT Ws-2 (open squares). <strong>(b) </strong>Analysis of <em>CAB2 </em>activity under constant red light at 22&#x000b0;C in <em>ebi-2 </em>(closed squares) and WT Col-0 (open squares). <strong>(c) </strong>Analysis of <em>CAB2 </em>activity under constant darkness at 22&#x000b0;C in <em>ebi-2 </em>(closed squares) and WT Col-0 (open squares). <strong>(d) </strong><em>EBI </em>expression is reduced in the <em>ebi-2 </em>mutant. RNA expression levels of <em>EBI </em>relative to &#x003b2;-tubulin were measured at either 1 h or 13 h under 12-h light/12-h dark cycles in both WT (white columns) and <em>ebi-2 </em>(gray columns).</p></div></div></div><p class="p p-last">Finally, a T-DNA insertion line was ordered, SALK_128255.54.50.n, which contains a T-DNA inserted in the promoter region of the <em>EBI </em>gene (<em>ebi-2</em>). The insertion does not stop <em>EBI </em>expression but it significantly reduces the expression level (Figure <a href="/pmc/articles/PMC3129678/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5d).</span></span><span>5d</span></a>). A homozygous T-DNA line was transformed with the <em>CAB2:LUC+ </em>reporter gene and the circadian phenotype of transformed lines analyzed. Like <em>ebi-1</em>, <em>ebi-2 </em>had a short period in constant light (WT, (Col-0) 26.74 h, SE 0.17, n = 27; T-DNA line, 25.67 h, SE 0.44, n = 28; Figure <a href="/pmc/articles/PMC3129678/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure5b)</span></span><span>5b</span></a>) and peaked early in constant dark (Figure <a href="/pmc/articles/PMC3129678/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5c</span></span><span>5c</span></a>).</p></div></div><div id="sec-a.u.c" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.u.ctitle">Discussion</h2><p class="p p-first">For many mutants, using traditional, map-based positional cloning is an extremely difficult approach for the identification of the genetic basis of some phenotypes. Here, we demonstrated the utility of massively parallel sequencing using an ABI SOLiD sequencer to spot EMS-induced mutations in a non-reference strain of <em>Arabidopsi</em>s. Using a functional genomic approach, based on the assumption that a clock component gene is likely to be rhythmically expressed, we were able to further narrow down the number of candidate SNPs. Finally, by using the SNP information we were able to exclude the previously identified clock gene <em>PRR7 </em>by generating clean backcrossed lines, identifying a SNP in the gene <em>AtNFXL-2 </em>as the likely cause of the <em>ebi-1 </em>phenotype. This was further validated by the characterization of a second allele of <em>ebi</em>, <em>ebi-2</em>. Our approach demonstrates the feasibility of next generation sequencing as a tool for positionally cloning genes in a large genome.</p><p>The gene responsible for the <em>ebi-1 </em>phenotype, <em>AtNFXL-2</em>, is a zinc finger transcription factor, a homolog of the human NF-X1 protein. In humans, NF-X1 binds to the X-box found in class II MHC genes [<a href="#B29" rid="B29" class=" bibr popnode">29</a>]. <em>Arabidopsis </em>has two NF-X1 homologs, <em>AtNFXL-1 </em>and <em>AtNFXL-2</em>, which are thought to act antagonistically to regulate genes involved in salt, osmotic and drought stress, with AtNFXL-1 activating and AtNFXL-2 repressing stress-inducing genes [<a href="#B30" rid="B30" class=" bibr popnode">30</a>]. AtNFXL-1 has also been suggested to be a negative regulator of defense-related genes [<a href="#B31" rid="B31" class=" bibr popnode">31</a>] and temperature stress [<a href="#B32" rid="B32" class=" bibr popnode">32</a>]. Thus, the clock phenotype of the <em>AtNFXL-2 </em>mutant provides an intriguing link between the clock and biotic and abiotic stress responses. This link has already been alluded to in a recent review [<a href="#B33" rid="B33" class=" bibr popnode">33</a>] and in the identification of a possible role for the clock protein GI in cold stress tolerance [<a href="#B34" rid="B34" class=" bibr popnode">34</a>].</p><p>Critical to the success of this project was to sequence the original parent from which the EMS mutant was derived. When Col-0 was recently re-sequenced using a lab strain, 1,172 SNPs were identified between the lab strain Col-0 and the original reference genome of Col-0. It is clear, therefore, that sequencing the original parent rather than relying on a previously sequenced reference is the correct approach. Secondly, the fact that we used a backcrossed line reduced the number of EMS mutations we had to consider from approximately 1,200 to 109. The large number of 'piggy-backing' SNPs also provides a stark example of just how many non-synonymous/nonsense mutations (51) are still present in what is regarded by the community as a 'clean' line.</p><p>An alternative approach to the direct sequencing method described here has been reported [<a href="#B16" rid="B16" class=" bibr popnode">16</a>,<a href="#B17" rid="B17" class=" bibr popnode">17</a>]. The technique relies on accurately scoring mutant individuals in an F<sub>2 </sub>mapping cross between divergent <em>Arabidopsis </em>accessions and then combining these individuals and sequencing the bulked DNA using next generation sequencing. The output of the sequence data provides information about the mapping position and a number of candidate SNPs. While this approach is extremely valuable, where the phenotype is subtle and there is a large amount of phenotype variation between individuals (resulting in a high number of false positives) it is unlikely to be useful. For the <em>ebi-1 </em>mutant, mapping was only possible by re-scoring potential mutants isolated in F<sub>2 </sub>again in the F<sub>3</sub>.</p><p>Our data clearly indicate strand bias in the mutagenesis process, resulting in long series of C to T or G to A transitions, rather than random mutation of either strand as expected based on previous population-level investigations [<a href="#B22" rid="B22" class=" bibr popnode">22</a>]. It has been shown that transcriptional activity affects repair efficiency [<a href="#B35" rid="B35" class=" bibr popnode">35</a>],, although this is unlikely to explain the bias, as over the long stretches of genome, both strands of the DNA are transcriptionally active. One simple explanation is that the mutagenesis event occurs and each strand of DNA is replicated and segregates to separate daughter cells. This would be sufficient to confer strand bias and thus the long stretches of identical transitions.</p><p class="p p-last">This combined approach of next generation sequencing and functional genomics can be used to identify genes previously intractable to conventional mapping approaches. The methodology is not restricted to <em>Arabidopsis </em>or to EMS-induced SNPs, but could be used to positionally clone genes in any organism with a sequenced genome. As accuracy and throughput increases, the technique should be possible in larger more complex genomes.</p></div><div id="sec-a.u.d" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.u.dtitle">Materials and methods</h2><div id="sec-a.u.d.b" class="sec sec-first"><h3 id="sec-a.u.d.btitle">Plant material</h3><p class="p p-first">Experiments were carried out with <em>ebi-1 </em>that had been backcrossed four times to the parental transgenic line 6A carrying the <em>CAB2:LUC+ </em>reporter construct (NASC ID N9352).</p><p class="p p-last">The T-DNA line SALK_128255.54.50.n was obtained from NASC and plants homozygous for the T-DNA were confirmed by PCR using primers 5'-ttgccgcagtaacaaaggtac-3', 5'-agtttatccggaagcaaatgg-3' (WT band in Col-0, no band in homozygous SALK line). The left border sequence was amplified with 5'-agtttatccggaagcaaatgg-3' and LBb primer. <em>CAB2:LUC+ </em>was introduced using <em>Agrobacterium</em>-mediated transformation and dipping protocol [<a href="#B36" rid="B36" class=" bibr popnode">36</a>].</p></div><div id="sec-a.u.d.c" class="sec"><h3 id="sec-a.u.d.ctitle">Screen for circadian clock mutants</h3><p class="p p-first-last">The mutagenesis and screening have been described in [<a href="#B18" rid="B18" class=" bibr popnode">18</a>]. Briefly, <em>Arabidopsis </em>Ws-2 transgenic seeds carrying the <em>CAB2:LUC+ </em>transgene (described above) were mutagenized by soaking in 100 mM EMS for 3 h. The resulting M<sub>1 </sub>population was sown and self-fertilized, and the M<sub>2 </sub>population was screened for seedlings with altered timing of <em>CAB2:LUC+ </em>expression in constant darkness.</p></div><div id="sec-a.u.d.d" class="sec"><h3 id="sec-a.u.d.dtitle">Analysis of circadian rhythms</h3><p class="p p-first-last">Seedlings were then sown on Murashige and Skoog medium containing 3% sucrose and 1.5% agar. They were entrained in a growth chamber in light/dark cycles at 22&#x000b0;C for 7 days before transfer to constant light and temperature. Two methods where used to measure <em>CAB2:LUC+ </em>activity. For the initial screen and preliminary characterization of the mutant in constant dark an automated luminometer was used (Topcount, Perkinelmer, Cambridge, UK)as described [<a href="#B37" rid="B37" class=" bibr popnode">37</a>]. The second method for the characterization of the mutant in constant light and subsequent characterization of backcrossed lines and T-DNA mutants was a low-light video imaging system as described in [<a href="#B37" rid="B37" class=" bibr popnode">37</a>]. The method for measuring rhythms in leaf movement used older 12-day-old seedlings and a method identical to that described in [<a href="#B38" rid="B38" class=" bibr popnode">38</a>].</p></div><div id="sec-a.u.d.e" class="sec"><h3 id="sec-a.u.d.etitle">Sequencing WS-2 and <em>ebi-1</em></h3><p class="p p-first">DNA was isolated using a plant DNeasy kit (Qiagen, Crawley, West Sussex, UK) Two read tag libraries were prepared, one for <em>ebi-1 </em>and one for Ws. Emulsion PCR using the standard SOLiD protocol was performed on each library. The libraries were deposited onto separate slides and sequenced in a single run using the SOLiD analyzer version 2 (Life Technologies).</p><p>For the 454 genome sequencing, 5 &#x003bc;g of Ws-2 DNA was fragmented by nebulization. Fragmented DNA was analyzed using a Bioanalyzer (Agilent Technologies, Wokingham, Berkshire, UK)to ensure that the majority of the fragments were between 350 and 1,000 bp. The purified fragmented DNA was processed according to the 454 FLX Titanium Library construction kit and protocol (Roche Applied Science, Burgess Hill, East Sussex, UK). Library fragments were added to emulsion PCR beads at a ratio of 1:1 to emPCR at the optimal of 1.5 DNA molecules per bead and amplified according to the manufacturer's instructions (Roche Applied Science) and a full pico-titre plate was sequenced.</p><p>The resulting 35-character color-space tags from both sequencing runs were then mapped to the 119.7 Mbp Col-0 reference sequence [<a href="#B39" rid="B39" class=" bibr popnode">39</a>] using the matching pipeline of the off-machine SOLiD data analysis package Corona Lite [<a href="#B40" rid="B40" class=" bibr popnode">40</a>] employing a range of matching schemas, based on the full-length 35-character color-space tags as well as schemas based on tags trimmed to 25 characters to remove the most error-prone positions. Putative SNPs relative to Col-0 were then called for each genome using Corona Lite's SNP detection pipeline.</p><p>The resulting SNP list for <em>ebi-1 </em>was then cross-referenced with that of Ws-2 to identify SNPs shared by both genomes, as well as SNPs occurring only in <em>ebi-1 </em>or only in Ws-2. At this stage low-confidence SNPs were filtered out by excluding all SNP loci where coverage was 5 or less, SOLiD SNP scores were less than 0.7, or the SNP was heterozygous, in either genome. To ensure only high-confidence SNPs were considered, a further screening round was undertaken in which only those reported by all matching schemas employed were considered for subsequent analysis.</p><p class="p p-last">Using current (TAIR 8) annotations [<a href="#B39" rid="B39" class=" bibr popnode">39</a>] as a guide, high-confidence SNPs were classified and enumerated. The sequence data for Ws-2 are archived at TAIR and available as a track on the <em>Arabidopsis </em>genome hosted at TAIR [SpeciesVariant:393] [<a href="#B41" rid="B41" class=" bibr popnode">41</a>].</p></div><div id="sec-a.u.d.f" class="sec"><h3 id="sec-a.u.d.ftitle">SNP validation</h3><p class="p p-first-last">To validate the SNPs between <em>ebi-1 </em>and Ws-2, we used a simple PCR-based approach of CAPS and dCAPS analysis. PCR primers for CAPS/dCAPS analysis were designed using dCAPS finder 2.0 [<a href="#B42" rid="B42" class=" bibr popnode">42</a>]. A standard PCR protocol was used to amplify products from <em>ebi-1 </em>and Ws-2, and the PCR products were digested and run on a 4% agarose gel and scored. The primers, restriction sites and product sizes are summarized in Additional file <a href="#S4" rid="S4" class=" supplementary-material">4</a>. The SNPs in <em>PRR7 </em>and <em>EBI </em>were further validated by standard sequencing methods.</p></div><div id="sec-a.u.d.g" class="sec"><h3 id="sec-a.u.d.gtitle">Quantification of RNA using real-time PCR</h3><p class="p p-first-last">Seedlings were grown under 12-h light/12-h dark cycles for 6 days. Seedlings were harvested directly into liquid nitrogen at 1 h after dawn and 1 h after dusk using a green safety light. The RNA was subsequently extracted using an RNeasy Plant Mini Kit (Qiagen, Hilden, Germany). cDNA was synthesized from 1 &#x003bc;g of total RNA using the iScript&#x02122; cDNA synthesis kit (Bio-Rad Laboratories, Inc., Hercules, CA, USA). Real-time PCR was performed with a MyIQ&#x02122;, ICycler or CFX96 Real-Time PCR Detection System (Bio-Rad Laboratories, Hempstead, Hertfordshire, UK), using iQ SYBR<sup>&#x000ae; </sup>Green Supermix (Bio-Rad Laboratories). The efficiency of amplification was assessed relative to <em>&#x003b2;-TUBULIN (&#x003b2;TUB</em>) expression. The measurements were repeated at least two times with independent biological material. Expression levels were calculated relative to the reference gene using a comparative threshold cycle method [<a href="#B43" rid="B43" class=" bibr popnode">43</a>]. The results show the mean of four biological replications, each with three technical repeats, and expressed relative to the mean of the wild-type series after standardization to <em>&#x003b2;TUB</em>. Primers for <em>&#x003b2;TUB </em>have been published previously [<a href="#B44" rid="B44" class=" bibr popnode">44</a>]. The <em>EBI</em>-specific primers were as follows: <em>EBI</em>-F, 5'-TGC GAG AAT ATG CTT AAT TGC-3'; <em>EBI</em>-R, 5'-CCA CAA CAT CAC AAG ACA AG-3'.</p></div><div id="sec-a.u.d.h" class="sec sec-last"><h3 id="sec-a.u.d.htitle">Mapping <em>ebi-1</em></h3><p class="p p-first-last">An F<sub>2 </sub>mapping population was made between <em>ebi-1 </em>and Col-0. A set of approximately 20 individuals from this population, which had their <em>ebi-1 </em>phenotype confirmed in the F<sub>3</sub>, had recombination events in chromosome 5 and placed the <em>ebi-1 </em>mutation on the north arm of chromosome 5. This mapping population was increased and with two individuals we were further able to limit the mapping interval to between CIW18 and nga158.</p></div></div><div id="sec-a.u.e" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.u.etitle">Abbreviations</h2><p class="p p-first-last">CAB2: Chlorophyll a/b-binding protein 2; CAPS: cleaved amplified polymorphic sequence; dCAPS: derived cleaved amplified polymorphic sequence; EBI: early bird mutant; EMS: ethyl methanesulfonate; LUC: luciferase; NASC: Nottingham Arabidopsis Stock Centre; SE: standard error; SNP: single nucleotide polymorphism; WT: wild type.</p></div><div id="sec-a.u.f" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.u.ftitle">Authors' contributions</h2><p class="p p-first-last">The screening and characterization of the <em>ebi </em>mutant was conceived by AH, MME and AJM and the SNP identification strategy by NH and AH, with AH responsible for overall co-ordination. SK and CA performed the SOLiD sequencing and LD performed the 454 sequencing. The characterization of <em>ebi </em>and alleles was performed by MJ, PG and MEE. The SNP validation was performed by LD. The bioinformatics was performed by KA with assistance from AH and NH, with all sequencing and sequence analysis overseen by NH. The paper was written by AH with assistance from NH and MEE. MEE was responsible for distribution of plant materials integral to the findings presented in this article and should be contacted directly. All authors read and approved the final manuscript.</p></div><div id="sec-a.u.g" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="sec-a.u.gtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="S1"><!--caption a9--><strong>Additional file 1:</strong><p><strong>Figure S1 - plant to plant variation in clock function is greater in Col-0 than in Ws-2</strong>. Seedlings were entrained under 12-h light/12-h dark cycles for 12 days, after which they were transferred to constant light where rhythms of leaf movement were assayed. Ws-2, filled squares; Col-0, empty squares. Period estimates for individual seedlings are plotted against their relative amplitude errors (R.A.E.).</p><div class="sup-box half_rhythm" id="media-a.u.g.b.b"><a href="/pmc/articles/PMC3129678/bin/gb-2011-12-3-r28-S1.PDF" data-ga-action="click_feat_suppl">Click here for file</a><sup>(37K, PDF)</sup></div></div><div class="sec suppmat" id="S2"><!--caption a9--><strong>Additional file 2:</strong><p><strong>Table S1 - sequence tag counts available at various stages of the analysis, as reported by the different matching schema employed</strong>.</p><div class="sup-box half_rhythm" id="media-a.u.g.c.b"><a href="/pmc/articles/PMC3129678/bin/gb-2011-12-3-r28-S2.PDF" data-ga-action="click_feat_suppl">Click here for file</a><sup>(52K, PDF)</sup></div></div><div class="sec suppmat" id="S3"><!--caption a9--><strong>Additional file 3:</strong><p><strong>Table S2 - SNP counts before and after filtering as reported by the various matching schema</strong>. <sup>a </sup>Unfiltered SNPs were all those reported by the Corona lite SNP detection pipeline. <sup>b</sup>Filtering involved retaining only those SNP loci where tag coverage exceeded 5&#x000d7; in both <em>ebi-1 </em>and Ws-2, the SOLiD score was 0.7 or greater, and SNPs were homozygous. <strong>(c) </strong>'Schema screened' SNPs were those filtered SNPs reported by all five schema.</p><div class="sup-box half_rhythm" id="media-a.u.g.d.b"><a href="/pmc/articles/PMC3129678/bin/gb-2011-12-3-r28-S3.PDF" data-ga-action="click_feat_suppl">Click here for file</a><sup>(52K, PDF)</sup></div></div><div class="sec suppmat" id="S4"><!--caption a9--><strong>Additional file 4:</strong><p><strong>Table S3 - dCAPS and CAPS marker design and use to validate SNP discovery</strong>. SNP marker denotes the chromosome position of the SNP based on the TAIR 8 <em>Arabidopsis </em>genome build. In the primer sequence the underlined base is the mismatched base in the primer sequence. ^Borderline SNP; <sup>a</sup>SNP in the clock gene <em>PRR7</em>; <sup>b</sup>SNP in At5g05660, <em>EBI</em>.</p><div class="sup-box half_rhythm" id="media-a.u.g.e.b"><a href="/pmc/articles/PMC3129678/bin/gb-2011-12-3-r28-S4.PDF" data-ga-action="click_feat_suppl">Click here for file</a><sup>(32K, PDF)</sup></div></div><div class="sec suppmat" id="S5"><!--caption a9--><strong>Additional file 5:</strong><p><strong>Table S4 - EMS-induced SNPs on chromosome 5</strong>.</p><div class="sup-box half_rhythm" id="media-a.u.g.f.b"><a href="/pmc/articles/PMC3129678/bin/gb-2011-12-3-r28-S5.PDF" data-ga-action="click_feat_suppl">Click here for file</a><sup>(158K, PDF)</sup></div></div><div class="sec suppmat" id="S6"><!--caption a9--><strong>Additional file 6:</strong><p><strong>Table S5 - EMS-induced SNPs on chromosome 1</strong>.</p><div class="sup-box half_rhythm" id="media-a.u.g.g.b"><a href="/pmc/articles/PMC3129678/bin/gb-2011-12-3-r28-S6.PDF" data-ga-action="click_feat_suppl">Click here for file</a><sup>(110K, PDF)</sup></div></div><div class="sec suppmat" id="S7"><!--caption a9--><strong>Additional file 7:</strong><p><strong>Figure S2 - the presence or absence of EMS-induced mutations on chromosome 1 do not affect the phenotype of <em>ebi-1</em></strong>. Transgenic seedlings carrying the <em>LUC </em>reporter gene fused to the <em>CAB2 </em>promoter were entrained under 12-h light/12-h dark cycles for 7 days, after which luminescence was monitored in constant red light. WT, open squares; <em>ebi-1</em>, closed squares; <em>ebi-1 </em>with no EMS-induced SNP on chromosome 1, red triangles.</p><div class="sup-box half_rhythm" id="media-a.u.g.h.b"><a href="/pmc/articles/PMC3129678/bin/gb-2011-12-3-r28-S7.PDF" data-ga-action="click_feat_suppl">Click here for file</a><sup>(44K, PDF)</sup></div></div><div class="sec suppmat" id="S8"><!--caption a9--><strong>Additional file 8:</strong><p><strong>Table S6 - analysis of temporal expression patterns of non-synonymous SNPs on chromosome 5 using Diurnal to fit temporal expression data to expression pattern models consistent with circadian regulation</strong>.</p><div class="sup-box half_rhythm" id="media-a.u.g.i.b"><a href="/pmc/articles/PMC3129678/bin/gb-2011-12-3-r28-S8.PDF" data-ga-action="click_feat_suppl">Click here for file</a><sup>(35K, PDF)</sup></div></div><div class="sec suppmat" id="S9"><!--caption a9--><strong>Additional file 9:</strong><p><strong>Figure S3 - identification of a SNP in <em>PRR7</em></strong>. Top: A schematic representation of the PRR7 protein in <em>Arabidopsis </em>ecotype Columbia is shown in green. Gray boxes represent the two conserved region Receiver (REC) domain and CCT motif. The amino acids were aligned using the ClustalW program. Bottom: identical and similar amino acid residues are highlighted with black and gray backgrounds, respectively. The SNP leads to a change from arginine (R) to histidine (H) at position 329. The frame shows the residue in the Pseudo Response Regulator protein from <em>Arabidopsis </em>ecotype Columbia (<a id="__tag_1135703745" class="tag_hotlink" href="/protein/BAB13742" ref="/protein/BAB13742"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"BAB13742","term_id":"10281004","term_text":"BAB13742"}}</span>BAB13742</a>, PRR7), <em>Hordeum vulgare </em>subsp. <em>vulgare </em>(<a id="__tag_1135703741" class="tag_hotlink" href="/protein/AAY17586" ref="/protein/AAY17586"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"AAY17586","term_id":"62868803","term_text":"AAY17586"}}</span>AAY17586</a>, PRR), <em>Arabidopsis thaliana </em>(<a id="__tag_1135703743" class="tag_hotlink" href="/protein/AAY62604" ref="/protein/AAY62604"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"AAY62604","term_id":"67005935","term_text":"AAY62604"}}</span>AAY62604</a>, PRR3), <em>Triticum aestivum </em>(<a id="__tag_1135703746" class="tag_hotlink" href="/protein/ABL09464" ref="/protein/ABL09464"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"ABL09464","term_id":"118638616","term_text":"ABL09464"}}</span>ABL09464</a>, PRR), <em>Oryza sativa Indica </em>(<a id="__tag_1135703748" class="tag_hotlink" href="/protein/BAD38858" ref="/protein/BAD38858"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"BAD38858","term_id":"51571883","term_text":"BAD38858"}}</span>BAD38858</a>, PRR 37), <em>Oryza sativa Indica </em>(<a id="__tag_1135703742" class="tag_hotlink" href="/protein/BAD38859" ref="/protein/BAD38859"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"BAD38859","term_id":"51571885","term_text":"BAD38859"}}</span>BAD38859</a>, PRR73), <em>Lemna paucicostata </em>(<a id="__tag_1135703744" class="tag_hotlink" href="/protein/BAE72697" ref="/protein/BAE72697"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"BAE72697","term_id":"84570625","term_text":"BAE72697"}}</span>BAE72697</a>, PRR37), <em>Lemna gibba </em>(<a id="__tag_1135703747" class="tag_hotlink" href="/protein/BAE72700" ref="/protein/BAE72700"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"BAE72700","term_id":"84570631","term_text":"BAE72700"}}</span>BAE72700</a>, PRR37), obtained from NCBI database, and <em>Gossypium raimondii </em>(TC272), <em>Brassica napus </em>(TC71410), <em>Brassica napus </em>(TC78134), <em>Gossypium raimondii </em>(TC82653), and <em>Citrus clementina </em>(TC8380) obtained from TGI databases.</p><div class="sup-box half_rhythm" id="media-a.u.g.j.b"><a href="/pmc/articles/PMC3129678/bin/gb-2011-12-3-r28-S9.PDF" data-ga-action="click_feat_suppl">Click here for file</a><sup>(330K, PDF)</sup></div></div></div><div id="sec-a.v.a" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="sec-a.v.atitle">Acknowledgements</h2><!--/article/back/sec/--><p class="p p-first-last">We would like to acknowledge funding from an EU Marie Curie Individual Fellowship QLK5-CT-2000-52165, The Swedish Research Council, The Swedish Foundation for Strategic Research, The Swedish Research Council for Environment, Agricultural Sciences and Spatial Planning (MEE) and a Marie Curie Early Stage Training project MEST-CT-2005-020526. MEE is a VINNMER Marie Curie International Qualification Fellow funded by The Swedish Governmental Agency for Innovation Systems (VINNOVA) and the European Union. We would also like to acknowledge start-up funding from the University of Liverpool (to NH) and the BBSRC research development fellowship (BB/H022333/1) awarded to AH. This work was also supported by SABR award F005237 from BBSRC and EPSRC, for the ROBuST (AH). NH is also supported by a Wolfson Merit Award from the Royal Society of Great Britain. We are grateful to Alistair Darby for his scientific contribution while car sharing.</p></div><div id="ref-list-a.v.b" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.v.btitle">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="B1"><span class="mixed-citation"><em>Arabidopsis </em>Genome Initiative. Analysis of the genome sequence of the flowering plant <em>Arabidopsis thaliana</em>. <span><span class="ref-journal">Nature. </span>2000;<span class="ref-vol">408</span>:796–815. doi: 10.1038/35048692.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11130711" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2F35048692" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Analysis+of+the+genome+sequence+of+the+flowering+plant+Arabidopsis+thaliana.&amp;volume=408&amp;publication_year=2000&amp;pages=796-815&amp;pmid=11130711&amp;doi=10.1038/35048692&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B2"><span class="mixed-citation">Somerville C, Meyerowitz E.  <span class="ref-journal">The Arabidopsis Book.</span> Rockville, MD: American Society of Plant Biologists; 2008.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=The+Arabidopsis+Book&amp;author=C+Somerville&amp;author=E+Meyerowitz&amp;publication_year=2008&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B3"><span class="mixed-citation">Lukowitz W, Gillmor CS, Scheible WR. Positional cloning in <em>Arabidopsis</em>. Why it feels good to have a genome initiative working for you. <span><span class="ref-journal">Plant Physiol. </span>2000;<span class="ref-vol">123</span>:795–805. doi: 10.1104/pp.123.3.795.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1539260/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10889228" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1104%2Fpp.123.3.795" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+Physiol&amp;title=Positional+cloning+in+Arabidopsis.+Why+it+feels+good+to+have+a+genome+initiative+working+for+you.&amp;author=W+Lukowitz&amp;author=CS+Gillmor&amp;author=WR+Scheible&amp;volume=123&amp;publication_year=2000&amp;pages=795-805&amp;pmid=10889228&amp;doi=10.1104/pp.123.3.795&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B4"><span class="mixed-citation">Alonso-Blanco C, Koornneef M. Naturally occurring variation in <em>Arabidopsis</em>: an underexploited resource for plant genetics. <span><span class="ref-journal">Trends Plant Sci. </span>2000;<span class="ref-vol">5</span>:22–29. doi: 10.1016/S1360-1385(99)01510-1.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10637658" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FS1360-1385(99)01510-1" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Plant+Sci&amp;title=Naturally+occurring+variation+in+Arabidopsis:+an+underexploited+resource+for+plant+genetics.&amp;author=C+Alonso-Blanco&amp;author=M+Koornneef&amp;volume=5&amp;publication_year=2000&amp;pages=22-29&amp;pmid=10637658&amp;doi=10.1016/S1360-1385(99)01510-1&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B5"><span class="mixed-citation">Lynn A, Koehler KE, Judis L, Chan ER, Cherry JP, Schwartz S, Seftel A, Hunt PA, Hassold TJ. Covariation of synaptonemal complex length and mammalian meiotic exchange rates. <span><span class="ref-journal">Science. </span>2002;<span class="ref-vol">296</span>:2222–2225. doi: 10.1126/science.1071220.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12052900" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.1071220" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Covariation+of+synaptonemal+complex+length+and+mammalian+meiotic+exchange+rates.&amp;author=A+Lynn&amp;author=KE+Koehler&amp;author=L+Judis&amp;author=ER+Chan&amp;author=JP+Cherry&amp;volume=296&amp;publication_year=2002&amp;pages=2222-2225&amp;pmid=12052900&amp;doi=10.1126/science.1071220&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B6"><span class="mixed-citation">Drouaud J, Camilleri C, Bourguignon PY, Canaguier A, Berard A, Vezon D, Giancola S, Brunel D, Colot V, Prum B, Quesneville H, Mezard C. Variation in crossing-over rates across chromosome 4 of <em>Arabidopsis thaliana </em>reveals the presence of meiotic recombination "hot spots". <span><span class="ref-journal">Genome Res. </span>2006;<span class="ref-vol">16</span>:106–114. doi: 10.1101/gr.4319006.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1356134/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16344568" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1101%2Fgr.4319006" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Variation+in+crossing-over+rates+across+chromosome+4+of+Arabidopsis+thaliana+reveals+the+presence+of+meiotic+recombination+&quot;hot+spots&quot;.&amp;author=J+Drouaud&amp;author=C+Camilleri&amp;author=PY+Bourguignon&amp;author=A+Canaguier&amp;author=A+Berard&amp;volume=16&amp;publication_year=2006&amp;pages=106-114&amp;pmid=16344568&amp;doi=10.1101/gr.4319006&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B7"><span class="mixed-citation">Harmer SL, Hogenesch JB, Straume M, Chang HS, Han B, Zhu T, Wang X, Kreps JA, Kay SA. Orchestrated transcription of key pathways in <em>Arabidopsis </em>by the circadian clock. <span><span class="ref-journal">Science. </span>2000;<span class="ref-vol">290</span>:2110–2113. doi: 10.1126/science.290.5499.2110.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11118138" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.290.5499.2110" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Orchestrated+transcription+of+key+pathways+in+Arabidopsis+by+the+circadian+clock.&amp;author=SL+Harmer&amp;author=JB+Hogenesch&amp;author=M+Straume&amp;author=HS+Chang&amp;author=B+Han&amp;volume=290&amp;publication_year=2000&amp;pages=2110-2113&amp;pmid=11118138&amp;doi=10.1126/science.290.5499.2110&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B8"><span class="mixed-citation">Locke JC, Kozma-Bognar L, Gould PD, Feher B, Kevei E, Nagy F, Turner MS, Hall A, Millar AJ. Experimental validation of a predicted feedback loop in the multi-oscillator clock of <em>Arabidopsis thaliana</em>. <span><span class="ref-journal">Mol Syst Biol. </span>2006;<span class="ref-vol">2</span>:59. doi: 10.1038/msb4100102.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1682024/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17102804" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fmsb4100102" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Syst+Biol&amp;title=Experimental+validation+of+a+predicted+feedback+loop+in+the+multi-oscillator+clock+of+Arabidopsis+thaliana.&amp;author=JC+Locke&amp;author=L+Kozma-Bognar&amp;author=PD+Gould&amp;author=B+Feher&amp;author=E+Kevei&amp;volume=2&amp;publication_year=2006&amp;pages=59&amp;pmid=17102804&amp;doi=10.1038/msb4100102&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B9"><span class="mixed-citation">Millar AJ, Short SR, Chua NH, Kay SA. A novel circadian phenotype based on firefly luciferase expression in transgenic plants. <span><span class="ref-journal">Plant Cell. </span>1992;<span class="ref-vol">4</span>:1075–1087. doi: 10.1105/tpc.4.9.1075.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC160198/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/1392609" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1105%2Ftpc.4.9.1075" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+Cell&amp;title=A+novel+circadian+phenotype+based+on+firefly+luciferase+expression+in+transgenic+plants.&amp;author=AJ+Millar&amp;author=SR+Short&amp;author=NH+Chua&amp;author=SA+Kay&amp;volume=4&amp;publication_year=1992&amp;pages=1075-1087&amp;pmid=1392609&amp;doi=10.1105/tpc.4.9.1075&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B10"><span class="mixed-citation">Millar AJ, Carr&#x000e9; IA, Strayer CA, Chua NH, Kay SA. Circadian clock mutants in <em>Arabidopsis </em>identified by luciferase imaging. <span><span class="ref-journal">Science. </span>1995;<span class="ref-vol">267</span>:1161–1163. doi: 10.1126/science.7855595.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7855595" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.7855595" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Circadian+clock+mutants+in+Arabidopsis+identified+by+luciferase+imaging.&amp;author=AJ+Millar&amp;author=IA+Carr&#x000e9;&amp;author=CA+Strayer&amp;author=NH+Chua&amp;author=SA+Kay&amp;volume=267&amp;publication_year=1995&amp;pages=1161-1163&amp;pmid=7855595&amp;doi=10.1126/science.7855595&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B11"><span class="mixed-citation">Somers DE, Schultz TF, Milnamow M, Kay SA. ZEITLUPE encodes a novel clock-associated PAS protein from <em>Arabidopsis</em>. <span><span class="ref-journal">Cell. </span>2000;<span class="ref-vol">101</span>:319–329. doi: 10.1016/S0092-8674(00)80841-7.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10847686" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FS0092-8674(00)80841-7" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=ZEITLUPE+encodes+a+novel+clock-associated+PAS+protein+from+Arabidopsis.&amp;author=DE+Somers&amp;author=TF+Schultz&amp;author=M+Milnamow&amp;author=SA+Kay&amp;volume=101&amp;publication_year=2000&amp;pages=319-329&amp;pmid=10847686&amp;doi=10.1016/S0092-8674(00)80841-7&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B12"><span class="mixed-citation">Hall A, Bastow RM, Davis SJ, Hanano S, McWatters HG, Hibberd V, Doyle MR, Sung S, Halliday KJ, Amasino RM, Millar AJ. The TIME FOR COFFEE gene maintains the amplitude and timing of <em>Arabidopsis </em>circadian clocks. <span><span class="ref-journal">Plant Cell. </span>2003;<span class="ref-vol">15</span>:2719–2729. doi: 10.1105/tpc.013730.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC280574/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14555691" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1105%2Ftpc.013730" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+Cell&amp;title=The+TIME+FOR+COFFEE+gene+maintains+the+amplitude+and+timing+of+Arabidopsis+circadian+clocks.&amp;author=A+Hall&amp;author=RM+Bastow&amp;author=SJ+Davis&amp;author=S+Hanano&amp;author=HG+McWatters&amp;volume=15&amp;publication_year=2003&amp;pages=2719-2729&amp;pmid=14555691&amp;doi=10.1105/tpc.013730&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B13"><span class="mixed-citation">Clark RM, Schweikert G, Toomajian C, Ossowski S, Zeller G, Shinn P, Warthmann N, Hu TT, Fu G, Hinds DA, Chen H, Frazer KA, Huson DH, Scholkopf B, Nordborg M, Ratsch G, Ecker JR, Weigel D. Common sequence polymorphisms shaping genetic diversity in <em>Arabidopsis thaliana</em>. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">317</span>:338–342. doi: 10.1126/science.1138632.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17641193" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.1138632" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Common+sequence+polymorphisms+shaping+genetic+diversity+in+Arabidopsis+thaliana.&amp;author=RM+Clark&amp;author=G+Schweikert&amp;author=C+Toomajian&amp;author=S+Ossowski&amp;author=G+Zeller&amp;volume=317&amp;publication_year=2007&amp;pages=338-342&amp;pmid=17641193&amp;doi=10.1126/science.1138632&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B14"><span class="mixed-citation">Smith DR, Quinlan AR, Peckham HE, Makowsky K, Tao W, Woolf B, Shen L, Donahue WF, Tusneem N, Stromberg MP, Stewart DA, Zhang L, Ranade SS, Warner JB, Lee CC, Coleman BE, Zhang Z, McLaughlin SF, Malek JA, Sorenson JM, Blanchard AP, Chapman J, Hillman D, Chen F, Rokhsar DS, McKernan KJ, Jeffries TW, Marth GT, Richardson PM. Rapid whole-genome mutational profiling using next-generation sequencing technologies. <span><span class="ref-journal">Genome Res. </span>2008;<span class="ref-vol">18</span>:1638–1642. doi: 10.1101/gr.077776.108.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2556265/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18775913" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1101%2Fgr.077776.108" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Rapid+whole-genome+mutational+profiling+using+next-generation+sequencing+technologies.&amp;author=DR+Smith&amp;author=AR+Quinlan&amp;author=HE+Peckham&amp;author=K+Makowsky&amp;author=W+Tao&amp;volume=18&amp;publication_year=2008&amp;pages=1638-1642&amp;pmid=18775913&amp;doi=10.1101/gr.077776.108&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B15"><span class="mixed-citation">Sarin S, Prabhu S, O'Meara MM, Pe'er I, Hobert O. <em>Caenorhabditis elegans </em>mutant allele identification by whole-genome sequencing. <span><span class="ref-journal">Nat Methods. </span>2008;<span class="ref-vol">5</span>:865–867. doi: 10.1038/nmeth.1249.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2574580/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18677319" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnmeth.1249" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=Caenorhabditis+elegans+mutant+allele+identification+by+whole-genome+sequencing.&amp;author=S+Sarin&amp;author=S+Prabhu&amp;author=MM+O'Meara&amp;author=I+Pe'er&amp;author=O+Hobert&amp;volume=5&amp;publication_year=2008&amp;pages=865-867&amp;pmid=18677319&amp;doi=10.1038/nmeth.1249&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B16"><span class="mixed-citation">Schneeberger K, Ossowski S, Lanz C, Juul T, Petersen AH, Nielsen KL, Jorgensen JE, Weigel D, Andersen SU. SHOREmap: simultaneous mapping and mutation identification by deep sequencing. <span><span class="ref-journal">Nat Methods. </span>2009;<span class="ref-vol">6</span>:550–551. doi: 10.1038/nmeth0809-550.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19644454" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnmeth0809-550" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=SHOREmap:+simultaneous+mapping+and+mutation+identification+by+deep+sequencing.&amp;author=K+Schneeberger&amp;author=S+Ossowski&amp;author=C+Lanz&amp;author=T+Juul&amp;author=AH+Petersen&amp;volume=6&amp;publication_year=2009&amp;pages=550-551&amp;pmid=19644454&amp;doi=10.1038/nmeth0809-550&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B17"><span class="mixed-citation">Cuperus JT, Montgomery TA, Fahlgren N, Burke RT, Townsend T, Sullivan CM, Carrington JC. Identification of MIR390a precursor processing-defective mutants in <em>Arabidopsis </em>by direct genome sequencing. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2010;<span class="ref-vol">107</span>:466–471. doi: 10.1073/pnas.0913203107.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2806713/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20018656" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.0913203107" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Identification+of+MIR390a+precursor+processing-defective+mutants+in+Arabidopsis+by+direct+genome+sequencing.&amp;author=JT+Cuperus&amp;author=TA+Montgomery&amp;author=N+Fahlgren&amp;author=RT+Burke&amp;author=T+Townsend&amp;volume=107&amp;publication_year=2010&amp;pages=466-471&amp;pmid=20018656&amp;doi=10.1073/pnas.0913203107&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B18"><span class="mixed-citation">Kevei E, Gyula P, Hall A, Kozma-Bognar L, Kim WY, Eriksson ME, Toth R, Hanano S, Feher B, Southern MM, Bastow RM, Viczian A, Hibberd V, Davis SJ, Somers DE, Nagy F, Millar AJ. Forward genetic analysis of the circadian clock separates the multiple functions of ZEITLUPE. <span><span class="ref-journal">Plant Physiol. </span>2006;<span class="ref-vol">140</span>:933–945. doi: 10.1104/pp.105.074864.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1400575/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16428597" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1104%2Fpp.105.074864" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+Physiol&amp;title=Forward+genetic+analysis+of+the+circadian+clock+separates+the+multiple+functions+of+ZEITLUPE.&amp;author=E+Kevei&amp;author=P+Gyula&amp;author=A+Hall&amp;author=L+Kozma-Bognar&amp;author=WY+Kim&amp;volume=140&amp;publication_year=2006&amp;pages=933-945&amp;pmid=16428597&amp;doi=10.1104/pp.105.074864&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B19"><span class="mixed-citation">Gould PD, Locke JC, Larue C, Southern MM, Davis SJ, Hanano S, Moyle R, Milich R, Putterill J, Millar AJ, Hall A. The molecular basis of temperature compensation in the <em>Arabidopsis </em>circadian clock. <span><span class="ref-journal">Plant Cell. </span>2006;<span class="ref-vol">18</span>:1177–1187. doi: 10.1105/tpc.105.039990.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1456873/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16617099" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1105%2Ftpc.105.039990" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+Cell&amp;title=The+molecular+basis+of+temperature+compensation+in+the+Arabidopsis+circadian+clock.&amp;author=PD+Gould&amp;author=JC+Locke&amp;author=C+Larue&amp;author=MM+Southern&amp;author=SJ+Davis&amp;volume=18&amp;publication_year=2006&amp;pages=1177-1187&amp;pmid=16617099&amp;doi=10.1105/tpc.105.039990&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B20"><span class="mixed-citation">Neff MM, Neff JD, Chory J, Pepper AE. dCAPS, a simple technique for the genetic analysis of single nucleotide polymorphisms: experimental applications in <em>Arabidopsis thaliana </em>genetics. <span><span class="ref-journal">Plant J. </span>1998;<span class="ref-vol">14</span>:387–392. doi: 10.1046/j.1365-313X.1998.00124.x.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9628033" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1046%2Fj.1365-313X.1998.00124.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+J&amp;title=dCAPS,+a+simple+technique+for+the+genetic+analysis+of+single+nucleotide+polymorphisms:+experimental+applications+in+Arabidopsis+thaliana+genetics.&amp;author=MM+Neff&amp;author=JD+Neff&amp;author=J+Chory&amp;author=AE+Pepper&amp;volume=14&amp;publication_year=1998&amp;pages=387-392&amp;pmid=9628033&amp;doi=10.1046/j.1365-313X.1998.00124.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B21"><span class="mixed-citation">Ossowski S, Schneeberger K, Clark RM, Lanz C, Warthmann N, Weigel D. Sequencing of natural strains of <em>Arabidopsis thaliana </em>with short reads. <span><span class="ref-journal">Genome Res. </span>2008;<span class="ref-vol">18</span>:2024–2033. doi: 10.1101/gr.080200.108.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2593571/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18818371" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1101%2Fgr.080200.108" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Sequencing+of+natural+strains+of+Arabidopsis+thaliana+with+short+reads.&amp;author=S+Ossowski&amp;author=K+Schneeberger&amp;author=RM+Clark&amp;author=C+Lanz&amp;author=N+Warthmann&amp;volume=18&amp;publication_year=2008&amp;pages=2024-2033&amp;pmid=18818371&amp;doi=10.1101/gr.080200.108&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B22"><span class="mixed-citation">Greene EA, Codomo CA, Taylor NE, Henikoff JG, Till BJ, Reynolds SH, Enns LC, Burtner C, Johnson JE, Odden AR, Comai L, Henikoff S. Spectrum of chemically induced mutations from a large-scale reverse-genetic screen in <em>Arabidopsis</em>. <span><span class="ref-journal">Genetics. </span>2003;<span class="ref-vol">164</span>:731–740.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1462604/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12807792" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Spectrum+of+chemically+induced+mutations+from+a+large-scale+reverse-genetic+screen+in+Arabidopsis.&amp;author=EA+Greene&amp;author=CA+Codomo&amp;author=NE+Taylor&amp;author=JG+Henikoff&amp;author=BJ+Till&amp;volume=164&amp;publication_year=2003&amp;pages=731-740&amp;pmid=12807792&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B23"><span class="mixed-citation">Diurnal search tool. <a href="http://diurnal.cgrb.oregonstate.edu/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://diurnal.cgrb.oregonstate.edu/</a></span></li><li id="B24"><span class="mixed-citation">Mockler TC, Michael TP, Priest HD, Shen R, Sullivan CM, Givan SA, McEntee C, Kay SA, Chory J. The DIURNAL project: DIURNAL and circadian expression profiling, model-based pattern matching, and promoter analysis. <span><span class="ref-journal">Cold Spring Harb Symp Quant Biol. </span>2007;<span class="ref-vol">72</span>:353–363. doi: 10.1101/sqb.2007.72.006.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18419293" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1101%2Fsqb.2007.72.006" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cold+Spring+Harb+Symp+Quant+Biol&amp;title=The+DIURNAL+project:+DIURNAL+and+circadian+expression+profiling,+model-based+pattern+matching,+and+promoter+analysis.&amp;author=TC+Mockler&amp;author=TP+Michael&amp;author=HD+Priest&amp;author=R+Shen&amp;author=CM+Sullivan&amp;volume=72&amp;publication_year=2007&amp;pages=353-363&amp;pmid=18419293&amp;doi=10.1101/sqb.2007.72.006&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B25"><span class="mixed-citation">Covington MF, Maloof JN, Straume M, Kay SA, Harmer SL. Global transcriptome analysis reveals circadian regulation of key pathways in plant growth and development. <span><span class="ref-journal">Genome Biol. </span>2008;<span class="ref-vol">9</span>:R130. doi: 10.1186/gb-2008-9-8-r130.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2575520/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18710561" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2Fgb-2008-9-8-r130" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Global+transcriptome+analysis+reveals+circadian+regulation+of+key+pathways+in+plant+growth+and+development.&amp;author=MF+Covington&amp;author=JN+Maloof&amp;author=M+Straume&amp;author=SA+Kay&amp;author=SL+Harmer&amp;volume=9&amp;publication_year=2008&amp;pages=R130&amp;pmid=18710561&amp;doi=10.1186/gb-2008-9-8-r130&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B26"><span class="mixed-citation">Edwards KD, Anderson PE, Hall A, Salathia NS, Locke JC, Lynn JR, Straume M, Smith JQ, Millar AJ. FLOWERING LOCUS C mediates natural variation in the high-temperature response of the <em>Arabidopsis </em>circadian clock. <span><span class="ref-journal">Plant Cell. </span>2006;<span class="ref-vol">18</span>:639–650. doi: 10.1105/tpc.105.038315.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1383639/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16473970" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1105%2Ftpc.105.038315" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+Cell&amp;title=FLOWERING+LOCUS+C+mediates+natural+variation+in+the+high-temperature+response+of+the+Arabidopsis+circadian+clock.&amp;author=KD+Edwards&amp;author=PE+Anderson&amp;author=A+Hall&amp;author=NS+Salathia&amp;author=JC+Locke&amp;volume=18&amp;publication_year=2006&amp;pages=639-650&amp;pmid=16473970&amp;doi=10.1105/tpc.105.038315&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B27"><span class="mixed-citation">Michael TP, Mockler TC, Breton G, McEntee C, Byer A, Trout JD, Hazen SP, Shen R, Priest HD, Sullivan CM, Givan SA, Yanovsky M, Hong F, Kay SA, Chory J. Network discovery pipeline elucidates conserved time-of-day-specific cis-regulatory modules. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e14. doi: 10.1371/journal.pgen.0040014.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2222925/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18248097" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pgen.0040014" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Network+discovery+pipeline+elucidates+conserved+time-of-day-specific+cis-regulatory+modules.&amp;author=TP+Michael&amp;author=TC+Mockler&amp;author=G+Breton&amp;author=C+McEntee&amp;author=A+Byer&amp;volume=4&amp;publication_year=2008&amp;pages=e14&amp;pmid=18248097&amp;doi=10.1371/journal.pgen.0040014&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B28"><span class="mixed-citation">Farre EM, Harmer SL, Harmon FG, Yanovsky MJ, Kay SA. Overlapping and distinct roles of PRR7 and PRR9 in the <em>Arabidopsis </em>circadian clock. <span><span class="ref-journal">Curr Biol. </span>2005;<span class="ref-vol">15</span>:47–54. doi: 10.1016/j.cub.2004.12.067.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15649364" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cub.2004.12.067" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Biol&amp;title=Overlapping+and+distinct+roles+of+PRR7+and+PRR9+in+the+Arabidopsis+circadian+clock.&amp;author=EM+Farre&amp;author=SL+Harmer&amp;author=FG+Harmon&amp;author=MJ+Yanovsky&amp;author=SA+Kay&amp;volume=15&amp;publication_year=2005&amp;pages=47-54&amp;pmid=15649364&amp;doi=10.1016/j.cub.2004.12.067&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B29"><span class="mixed-citation">Song Z, Krishna S, Thanos D, Strominger JL, Ono SJ. A novel cysteine-rich sequence-specific DNA-binding protein interacts with the conserved X-box motif of the human major histocompatibility complex class II genes via a repeated Cys-His domain and functions as a transcriptional repressor. <span><span class="ref-journal">J Exp Med. </span>1994;<span class="ref-vol">180</span>:1763–1774. doi: 10.1084/jem.180.5.1763.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2191754/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7964459" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1084%2Fjem.180.5.1763" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Exp+Med&amp;title=A+novel+cysteine-rich+sequence-specific+DNA-binding+protein+interacts+with+the+conserved+X-box+motif+of+the+human+major+histocompatibility+complex+class+II+genes+via+a+repeated+Cys-His+domain+and+functions+as+a+transcriptional+repressor.&amp;author=Z+Song&amp;author=S+Krishna&amp;author=D+Thanos&amp;author=JL+Strominger&amp;author=SJ+Ono&amp;volume=180&amp;publication_year=1994&amp;pages=1763-1774&amp;pmid=7964459&amp;doi=10.1084/jem.180.5.1763&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B30"><span class="mixed-citation">Lisso J, Altmann T, Mussig C. The AtNFXL1 gene encodes a NF-X1 type zinc finger protein required for growth under salt stress. <span><span class="ref-journal">FEBS Lett. </span>2006;<span class="ref-vol">580</span>:4851–4856. doi: 10.1016/j.febslet.2006.07.079.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16905136" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.febslet.2006.07.079" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEBS+Lett&amp;title=The+AtNFXL1+gene+encodes+a+NF-X1+type+zinc+finger+protein+required+for+growth+under+salt+stress.&amp;author=J+Lisso&amp;author=T+Altmann&amp;author=C+Mussig&amp;volume=580&amp;publication_year=2006&amp;pages=4851-4856&amp;pmid=16905136&amp;doi=10.1016/j.febslet.2006.07.079&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B31"><span class="mixed-citation">Asano T, Yasuda M, Nakashita H, Kimura M, Yamaguchi K, Nishiuchi T. The AtNFXL1 gene functions as a signaling component of the type A trichothecene-dependent response. <span><span class="ref-journal">Plant Signal Behav. </span>2008;<span class="ref-vol">3</span>:991–992.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2633753/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19704430" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+Signal+Behav&amp;title=The+AtNFXL1+gene+functions+as+a+signaling+component+of+the+type+A+trichothecene-dependent+response.&amp;author=T+Asano&amp;author=M+Yasuda&amp;author=H+Nakashita&amp;author=M+Kimura&amp;author=K+Yamaguchi&amp;volume=3&amp;publication_year=2008&amp;pages=991-992&amp;pmid=19704430&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B32"><span class="mixed-citation">Larkindale J, Vierling E. Core genome responses involved in acclimation to high temperature. <span><span class="ref-journal">Plant Physiol. </span>2008;<span class="ref-vol">146</span>:748–761. doi: 10.1104/pp.107.112060.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2245833/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18055584" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1104%2Fpp.107.112060" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+Physiol&amp;title=Core+genome+responses+involved+in+acclimation+to+high+temperature.&amp;author=J+Larkindale&amp;author=E+Vierling&amp;volume=146&amp;publication_year=2008&amp;pages=748-761&amp;pmid=18055584&amp;doi=10.1104/pp.107.112060&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B33"><span class="mixed-citation">Roden LC, Ingle RA. Lights, rhythms, infection: the role of light and the circadian clock in determining the outcome of plant-pathogen interactions. <span><span class="ref-journal">Plant Cell. </span>2009;<span class="ref-vol">21</span>:2546–2552. doi: 10.1105/tpc.109.069922.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2768925/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19789275" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1105%2Ftpc.109.069922" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+Cell&amp;title=Lights,+rhythms,+infection:+the+role+of+light+and+the+circadian+clock+in+determining+the+outcome+of+plant-pathogen+interactions.&amp;author=LC+Roden&amp;author=RA+Ingle&amp;volume=21&amp;publication_year=2009&amp;pages=2546-2552&amp;pmid=19789275&amp;doi=10.1105/tpc.109.069922&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B34"><span class="mixed-citation">Cao S, Ye M, Jiang S. Involvement of GIGANTEA gene in the regulation of the cold stress response in <em>Arabidopsis</em>. <span><span class="ref-journal">Plant Cell Rep. </span>2005;<span class="ref-vol">24</span>:683–690. doi: 10.1007/s00299-005-0061-x.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16231185" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs00299-005-0061-x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+Cell+Rep&amp;title=Involvement+of+GIGANTEA+gene+in+the+regulation+of+the+cold+stress+response+in+Arabidopsis.&amp;author=S+Cao&amp;author=M+Ye&amp;author=S+Jiang&amp;volume=24&amp;publication_year=2005&amp;pages=683-690&amp;pmid=16231185&amp;doi=10.1007/s00299-005-0061-x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B35"><span class="mixed-citation">Madhani HD, Bohr VA, Hanawalt PC. Differential DNA repair in transcriptionally active and inactive proto-oncogenes: c-abl and c-mos. <span><span class="ref-journal">Cell. </span>1986;<span class="ref-vol">45</span>:417–423. doi: 10.1016/0092-8674(86)90327-2.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3698104" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2F0092-8674(86)90327-2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Differential+DNA+repair+in+transcriptionally+active+and+inactive+proto-oncogenes:+c-abl+and+c-mos.&amp;author=HD+Madhani&amp;author=VA+Bohr&amp;author=PC+Hanawalt&amp;volume=45&amp;publication_year=1986&amp;pages=417-423&amp;pmid=3698104&amp;doi=10.1016/0092-8674(86)90327-2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B36"><span class="mixed-citation">Bechtold N, Ellis J, Pelletier G. In planta <em>Agrobacterium</em>-mediated gene transfer by infiltration of adult <em>Arabidopsis thaliana </em>plants. <span><span class="ref-journal">CR Acad Sci. </span>1993;<span class="ref-vol">316</span>:1194–1199.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=CR+Acad+Sci&amp;title=In+planta+Agrobacterium-mediated+gene+transfer+by+infiltration+of+adult+Arabidopsis+thaliana+plants.&amp;author=N+Bechtold&amp;author=J+Ellis&amp;author=G+Pelletier&amp;volume=316&amp;publication_year=1993&amp;pages=1194-1199&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B37"><span class="mixed-citation">Southern MM, Brown PE, Hall A. Luciferases as reporter genes. <span><span class="ref-journal">Methods Mol Biol. </span>2006;<span class="ref-vol">323</span>:293–305.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16739586" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Mol+Biol&amp;title=Luciferases+as+reporter+genes.&amp;author=MM+Southern&amp;author=PE+Brown&amp;author=A+Hall&amp;volume=323&amp;publication_year=2006&amp;pages=293-305&amp;pmid=16739586&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B38"><span class="mixed-citation">Edwards KD, Millar AJ. Analysis of circadian leaf movement rhythms in <em>Arabidopsis thaliana</em>. <span><span class="ref-journal">Methods Mol Biol. </span>2007;<span class="ref-vol">362</span>:103–113.</span> full_text. [<a href="https://pubmed.ncbi.nlm.nih.gov/17417004" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Mol+Biol&amp;title=Analysis+of+circadian+leaf+movement+rhythms+in+Arabidopsis+thaliana.&amp;author=KD+Edwards&amp;author=AJ+Millar&amp;volume=362&amp;publication_year=2007&amp;pages=103-113&amp;pmid=17417004&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B39"><span class="mixed-citation">TAIR build 8. <a href="ftp://ftp.arabidopsis.org/Sequences/whole_chromosomes/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CFTP" target="_blank">ftp://ftp.arabidopsis.org/Sequences/whole_chromosomes/</a></span></li><li id="B40"><span class="mixed-citation">SOLiD&#x02122; System Analysis Pipeline Tool (Corona Lite). <a href="http://solidsoftwaretools.com/gf/project/corona/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://solidsoftwaretools.com/gf/project/corona/</a></span></li><li id="B41"><span class="mixed-citation">TAIR <em>Arabidopsis </em>Gbrowser. <a href="http://gbrowse.arabidopsis.org/cgi-bin/gbrowse/arabidopsis/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://gbrowse.arabidopsis.org/cgi-bin/gbrowse/arabidopsis/</a></span></li><li id="B42"><span class="mixed-citation">Neff MM, Turk E, Kalishman M. Web-based primer design for single nucleotide polymorphism analysis. <span><span class="ref-journal">Trends Genet. </span>2002;<span class="ref-vol">18</span>:613–615. doi: 10.1016/S0168-9525(02)02820-2.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12446140" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FS0168-9525(02)02820-2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Genet&amp;title=Web-based+primer+design+for+single+nucleotide+polymorphism+analysis.&amp;author=MM+Neff&amp;author=E+Turk&amp;author=M+Kalishman&amp;volume=18&amp;publication_year=2002&amp;pages=613-615&amp;pmid=12446140&amp;doi=10.1016/S0168-9525(02)02820-2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B43"><span class="mixed-citation">Livak KJ, Schmittgen TD. Analysis of relative gene expression data using real-time quantitative PCR and the 2(T)(-Delta Delta C) method. <span><span class="ref-journal">Methods. </span>2001;<span class="ref-vol">25</span>:402–408. doi: 10.1006/meth.2001.1262.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11846609" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1006%2Fmeth.2001.1262" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods&amp;title=Analysis+of+relative+gene+expression+data+using+real-time+quantitative+PCR+and+the+2(T)(-Delta+Delta+C)+method.&amp;author=KJ+Livak&amp;author=TD+Schmittgen&amp;volume=25&amp;publication_year=2001&amp;pages=402-408&amp;pmid=11846609&amp;doi=10.1006/meth.2001.1262&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B44"><span class="mixed-citation">Czechowski T, Bari RP, Stitt M, Scheible WR, Udvardi MK. Real-time RT-PCR profiling of over 1400 <em>Arabidopsis </em>transcription factors: unprecedented sensitivity reveals novel root- and shoot-specific genes. <span><span class="ref-journal">Plant J. </span>2004;<span class="ref-vol">38</span>:366–379. doi: 10.1111/j.1365-313X.2004.02051.x.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15078338" ref="reftype=pubmed&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fj.1365-313X.2004.02051.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plant+J&amp;title=Real-time+RT-PCR+profiling+of+over+1400+Arabidopsis+transcription+factors:+unprecedented+sensitivity+reveals+novel+root-+and+shoot-specific+genes.&amp;author=T+Czechowski&amp;author=RP+Bari&amp;author=M+Stitt&amp;author=WR+Scheible&amp;author=MK+Udvardi&amp;volume=38&amp;publication_year=2004&amp;pages=366-379&amp;pmid=15078338&amp;doi=10.1111/j.1365-313X.2004.02051.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3129678&amp;issue-id=198214&amp;journal-id=7&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li></ul></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Genome Biology</span> are provided here courtesy of <strong>BMC</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC3129678/pdf/gb-2011-12-3-r28.pdf" class="int-view">PDF (723K)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/3129678/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/3129678/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="3129678" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3129678%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="suqubEP0XvmbcNiRS8Au8DBRLLUHkXc9xRHlkvRSxHbmasXWMhAaHasdMry8RwA4">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3129678%2F&amp;text=Full%20genome%20re-sequencing%20reveals%20a%20novel%20circadian%20clock%20mutation%20in%20Arabidopsis" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3129678%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3129678/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC3129678/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/21429190/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC3129678/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/21429190/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC3129678/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/3129678/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC3129678/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
