
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs2.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>The histone chaperones Vps75 and Nap1 form ring-like, tetrameric structures in solution - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="D0BD41E4AC192A4500005A7DE427178C.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="nar" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC4027167/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4027167/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Nucleic Acids Research" /><meta name="citation_title" content="The histone chaperones Vps75 and Nap1 form ring-like, tetrameric structures in solution" /><meta name="citation_author" content="Andrew Bowman" /><meta name="citation_author_institution" content="Centre for Gene Regulation and Expression, University of Dundee, Dundee, DD1 5EH, UK" /><meta name="citation_author" content="Colin M. Hammond" /><meta name="citation_author_institution" content="Centre for Gene Regulation and Expression, University of Dundee, Dundee, DD1 5EH, UK" /><meta name="citation_author" content="Andrew Stirling" /><meta name="citation_author_institution" content="Centre for Gene Regulation and Expression, University of Dundee, Dundee, DD1 5EH, UK" /><meta name="citation_author" content="Richard Ward" /><meta name="citation_author_institution" content="Nucleic Acids Structure Research Group, University of Dundee, Dundee, DD1 5EH, UK" /><meta name="citation_author" content="Weifeng Shang" /><meta name="citation_author_institution" content="European Molecular Biology Laboratory, Hamburg Outstation, c/o DESY, Notkestrasse 85, D-22603 Hamburg, Germany" /><meta name="citation_author" content="Hassane El-Mkami" /><meta name="citation_author_institution" content="School of Physics and Astronomy, University of St Andrews, St Andrews FE2 4KM, UK" /><meta name="citation_author" content="David A. Robinson" /><meta name="citation_author_institution" content="Division of Biological Chemistry and Drug Discovery, University of Dundee, Dundee, DD1 5EH, UK" /><meta name="citation_author" content="Dmitri I. Svergun" /><meta name="citation_author_institution" content="European Molecular Biology Laboratory, Hamburg Outstation, c/o DESY, Notkestrasse 85, D-22603 Hamburg, Germany" /><meta name="citation_author" content="David G. Norman" /><meta name="citation_author_institution" content="Nucleic Acids Structure Research Group, University of Dundee, Dundee, DD1 5EH, UK" /><meta name="citation_author" content="Tom Owen-Hughes" /><meta name="citation_author_institution" content="Centre for Gene Regulation and Expression, University of Dundee, Dundee, DD1 5EH, UK" /><meta name="citation_publication_date" content="2014/05/05" /><meta name="citation_issue" content="9" /><meta name="citation_volume" content="42" /><meta name="citation_firstpage" content="6038" /><meta name="citation_doi" content="10.1093/nar/gku232" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4027167/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC4027167/" /><meta name="citation_pmid" content="24688059" /><meta name="DC.Title" content="The histone chaperones Vps75 and Nap1 form ring-like, tetrameric structures in solution" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Oxford University Press" /><meta name="DC.Contributor" content="Andrew Bowman" /><meta name="DC.Contributor" content="Colin M. Hammond" /><meta name="DC.Contributor" content="Andrew Stirling" /><meta name="DC.Contributor" content="Richard Ward" /><meta name="DC.Contributor" content="Weifeng Shang" /><meta name="DC.Contributor" content="Hassane El-Mkami" /><meta name="DC.Contributor" content="David A. Robinson" /><meta name="DC.Contributor" content="Dmitri I. Svergun" /><meta name="DC.Contributor" content="David G. Norman" /><meta name="DC.Contributor" content="Tom Owen-Hughes" /><meta name="DC.Date" content="2014 May 1" /><meta name="DC.Identifier" content="10.1093/nar/gku232" /><meta name="DC.Language" content="en" /><meta property="og:title" content="The histone chaperones Vps75 and Nap1 form ring-like, tetrameric structures in solution" /><meta property="og:type" content="article" /><meta property="og:description" content="NAP-1 fold histone chaperones play an important role in escorting histones to and from sites of nucleosome assembly and disassembly. The two NAP-1 fold histone chaperones in budding yeast, Vps75 and Nap1, have previously been crystalized in a characteristic ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4027167/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC4027167/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/4/">Nucleic Acids Res</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/237979/">v.42(9); 2014 May 1</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC4027167
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC4027167/pdf/gku232.pdf" class="int-view">PDF (3.5M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/4027167/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/4027167/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="4027167" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4027167%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="2mIrofQJoUWQGulAwwaDYFWNSSwdPlRDpUOTBLeyeMRlUppHTCJTGW8oy2seZHkZ">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4027167%2F&amp;text=The%20histone%20chaperones%20Vps75%20and%20Nap1%20form%20ring-like%2C%20tetrameric%20structures%20in%20solution" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4027167%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4027167/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC4027167/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/24688059/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC4027167/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/24688059/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC4027167/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/4027167/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/4/">Nucleic Acids Res</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/237979/">v.42(9); 2014 May 1</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC4027167
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/narlogo.gif" alt="Logo of nar" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/nar" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Nucleic Acids Res.</span> 2014 May 1; 42(9): 6038–6051. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2014 Mar 31. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1093%2Fnar%2Fgku232" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/nar/gku232</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4027167</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/24688059">24688059</a></div></div></div><h1 class="content-title">The histone chaperones Vps75 and Nap1 form ring-like, tetrameric structures in solution</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Bowman%20A%5BAuthor%5D" class="affpopup" co-rid="_co_idm140352944074000" co-class="co-affbox">Andrew Bowman</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hammond%20CM%5BAuthor%5D" class="affpopup" co-rid="_co_idm140352980090000" co-class="co-affbox">Colin M. Hammond</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Stirling%20A%5BAuthor%5D" class="affpopup" co-rid="_co_idm140352946619696" co-class="co-affbox">Andrew Stirling</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ward%20R%5BAuthor%5D" class="affpopup" co-rid="_co_idm140352970108240" co-class="co-affbox">Richard Ward</a>,<sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shang%20W%5BAuthor%5D" class="affpopup" co-rid="_co_idm140352979526112" co-class="co-affbox">Weifeng Shang</a>,<sup>3</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=El-Mkami%20H%5BAuthor%5D" class="affpopup" co-rid="_co_idm140352946387248" co-class="co-affbox">Hassane El-Mkami</a>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Robinson%20DA%5BAuthor%5D" class="affpopup" co-rid="_co_idm140352943411152" co-class="co-affbox">David A. Robinson</a>,<sup>5</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Svergun%20DI%5BAuthor%5D" class="affpopup" co-rid="_co_idm140353013884160" co-class="co-affbox">Dmitri I. Svergun</a>,<sup>3</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Norman%20DG%5BAuthor%5D" class="affpopup" co-rid="_co_idm140352971898176" co-class="co-affbox">David G. Norman</a>,<sup>2</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Owen-Hughes%20T%5BAuthor%5D" class="affpopup" co-rid="_co_idm140353013355936" co-class="co-affbox">Tom Owen-Hughes</a><sup>1,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140352944074000"><h3 class="no_margin">Andrew Bowman</h3><p><sup>1</sup>Centre for Gene Regulation and Expression, University of Dundee, Dundee, DD1 5EH, UK</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Bowman%20A%5BAuthor%5D">Andrew Bowman</a></div></div><div id="_co_idm140352980090000"><h3 class="no_margin">Colin M. Hammond</h3><p><sup>1</sup>Centre for Gene Regulation and Expression, University of Dundee, Dundee, DD1 5EH, UK</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hammond%20CM%5BAuthor%5D">Colin M. Hammond</a></div></div><div id="_co_idm140352946619696"><h3 class="no_margin">Andrew Stirling</h3><p><sup>1</sup>Centre for Gene Regulation and Expression, University of Dundee, Dundee, DD1 5EH, UK</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Stirling%20A%5BAuthor%5D">Andrew Stirling</a></div></div><div id="_co_idm140352970108240"><h3 class="no_margin">Richard Ward</h3><p><sup>2</sup>Nucleic Acids Structure Research Group, University of Dundee, Dundee, DD1 5EH, UK</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ward%20R%5BAuthor%5D">Richard Ward</a></div></div><div id="_co_idm140352979526112"><h3 class="no_margin">Weifeng Shang</h3><p><sup>3</sup>European Molecular Biology Laboratory, Hamburg Outstation, c/o DESY, Notkestrasse 85, D-22603 Hamburg, Germany</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shang%20W%5BAuthor%5D">Weifeng Shang</a></div></div><div id="_co_idm140352946387248"><h3 class="no_margin">Hassane El-Mkami</h3><p><sup>4</sup>School of Physics and Astronomy, University of St Andrews, St Andrews FE2 4KM, UK</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=El-Mkami%20H%5BAuthor%5D">Hassane El-Mkami</a></div></div><div id="_co_idm140352943411152"><h3 class="no_margin">David A. Robinson</h3><p><sup>5</sup>Division of Biological Chemistry and Drug Discovery, University of Dundee, Dundee, DD1 5EH, UK</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Robinson%20DA%5BAuthor%5D">David A. Robinson</a></div></div><div id="_co_idm140353013884160"><h3 class="no_margin">Dmitri I. Svergun</h3><p><sup>3</sup>European Molecular Biology Laboratory, Hamburg Outstation, c/o DESY, Notkestrasse 85, D-22603 Hamburg, Germany</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Svergun%20DI%5BAuthor%5D">Dmitri I. Svergun</a></div></div><div id="_co_idm140352971898176"><h3 class="no_margin">David G. Norman</h3><p><sup>2</sup>Nucleic Acids Structure Research Group, University of Dundee, Dundee, DD1 5EH, UK</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Norman%20DG%5BAuthor%5D">David G. Norman</a></div></div><div id="_co_idm140353013355936"><h3 class="no_margin">Tom Owen-Hughes</h3><p><sup>1</sup>Centre for Gene Regulation and Expression, University of Dundee, Dundee, DD1 5EH, UK</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Owen-Hughes%20T%5BAuthor%5D">Tom Owen-Hughes</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140352946323008_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140352946323008_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140352946323008_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140352946323008_ai" style="display:none"><div class="fm-affl" id="AFF1"><sup>1</sup>Centre for Gene Regulation and Expression, University of Dundee, Dundee, DD1 5EH, UK</div><div class="fm-affl" id="AFF2"><sup>2</sup>Nucleic Acids Structure Research Group, University of Dundee, Dundee, DD1 5EH, UK</div><div class="fm-affl" id="AFF3"><sup>3</sup>European Molecular Biology Laboratory, Hamburg Outstation, c/o DESY, Notkestrasse 85, D-22603 Hamburg, Germany</div><div class="fm-affl" id="AFF4"><sup>4</sup>School of Physics and Astronomy, University of St Andrews, St Andrews FE2 4KM, UK</div><div class="fm-affl" id="AFF5"><sup>5</sup>Division of Biological Chemistry and Drug Discovery, University of Dundee, Dundee, DD1 5EH, UK</div><div id="COR1"><sup>*</sup>To whom correspondence should be addressed. Tel: +44 1382 385796; Fax: +44 1382 388702; Email: <a href="mailto:dev@null" data-email="ku.ca.eednud@sehguhnewo.a.t" class="oemail">ku.ca.eednud@sehguhnewo.a.t</a></div></div><div class="fm-article-notes hide half_rhythm" id="idm140352946323008_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2013 Aug 21; Revised 2014 Feb 14; Accepted 2014 Mar 7.</div></div><div class="permissions half_rhythm hide" id="idm140352946323008_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; The Author(s) 2014. Published by Oxford University Press on behalf of Nucleic Acids Research.</div><div class="license half_rhythm">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by/3.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
SUPPLEMENTARY DATA
</div>

<div><a href="/pmc/articles/PMC4027167/bin/supp_42_9_6038__index.html" data-ga-action="click_feat_suppl">supp_42_9_6038__index.html</a><span style="color:gray"> (977 bytes)</span></div><div class="small guid">GUID: D13BF620-EEB7-4611-BB91-4A25B154F2DF</div>
<div><a href="/pmc/articles/PMC4027167/bin/supp_gku232_nar-02436-m-2013.zip" data-ga-action="click_feat_suppl">supp_gku232_nar-02436-m-2013.zip</a><span style="color:gray"> (31M)</span></div><div class="small guid">GUID: FE5A3837-32C9-4F5E-9CD9-1BC4584E3E21</div>
</div></dd></dl></div><div id="abstract-a.l.b.p" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.l.b.ptitle" style="text-transform: uppercase;">Abstract</h2><!--article-meta--><div><p class="p p-first-last">NAP-1 fold histone chaperones play an important role in escorting histones to and from sites of nucleosome assembly and disassembly. The two NAP-1 fold histone chaperones in budding yeast, Vps75 and Nap1, have previously been crystalized in a characteristic homodimeric conformation. In this study, a combination of small angle X-ray scattering, multi angle light scattering and pulsed electron&#x02013;electron double resonance approaches were used to show that both Vps75 and Nap1 adopt ring-shaped tetrameric conformations in solution. This suggests that the formation of homotetramers is a common feature of NAP-1 fold histone chaperones. The tetramerisation of NAP-1 fold histone chaperones may act to shield acidic surfaces in the absence of histone cargo thus providing a &#x02018;self-chaperoning&#x02019; type mechanism.</p></div></div><div id="SEC1" class="tsec sec"><h2 class="head no_bottom_margin" id="SEC1title" style="text-transform: uppercase;">INTRODUCTION</h2><p class="p p-first">Histone chaperones were first characterized as factors required for correct assembly and disassembly of nucleosomes during DNA replication (<a href="#B1" rid="B1" class=" bibr popnode">1</a>). More recently, histone chaperones have been shown to be involved in many other aspects of chromatin dynamics, functioning both with adenosine triphosphate-driven molecular motors and post-translational modifiers in diverse processes such as transcriptional regulation, DNA repair, replication, homologous recombination and mRNA maturation (reviewed in (<a href="#B2" rid="B2" class=" bibr popnode">2&#x02013;4</a>)).</p><p>Vps75 was originally identified in a genetic screen to isolate factors affecting vacuolar protein sorting in budding yeast (<a href="#B5" rid="B5" class=" bibr popnode">5</a>), but was re-classified as a NAP-1 family histone chaperone based on sequence homology (<a href="#B6" rid="B6" class=" bibr popnode">6</a>). High affinity for both H3-H4 and H2A-H2B confirmed the histone chaperone function of Vps75 (<a href="#B6" rid="B6" class=" bibr popnode">6&#x02013;8</a>). Vps75 co-purifies with the histone H3 acetyltransferase Rtt109 (<a href="#B6" rid="B6" class=" bibr popnode">6</a>,<a href="#B9" rid="B9" class=" bibr popnode">9</a>) and has been reported to increase efficiency of its histone acetylation activity by over 250-fold (<a href="#B10" rid="B10" class=" bibr popnode">10</a>). The catalysis of histone acetylation by chaperone-dependent histone acetyltransferase complexes reveals interconnectivity between the regulation of chromatin organisation and nucleosome assembly/disassembly pathways.</p><p>The presence of near stoichiometric amounts of Vps75 and Rtt109 after stringent purification of the tagged complex suggests that the interaction between the two proteins is stable (<a href="#B6" rid="B6" class=" bibr popnode">6</a>,<a href="#B9" rid="B9" class=" bibr popnode">9</a>,<a href="#B11" rid="B11" class=" bibr popnode">11</a>). However, genome-wide genetic interaction analysis has revealed distinct cellular roles for Vps75 and Rtt109: Rtt109 functioning in DNA replication and repair pathways, and Vps75 having an additional function in transcriptional regulation (<a href="#B12" rid="B12" class=" bibr popnode">12</a>,<a href="#B13" rid="B13" class=" bibr popnode">13</a>). These distinct roles may explain the alternate expression patterns of Rtt109 and Vps75. Whilst Rtt109 transcripts fluctuate during the cell cycle, peaking during S-phase, Vps75 transcripts protein levels remain relatively stable (<a href="#B12" rid="B12" class=" bibr popnode">12</a>,<a href="#B14" rid="B14" class=" bibr popnode">14</a>). Thus, it has been suggested that a Rtt109-free pool of Vps75 exists outside of S-phase (<a href="#B12" rid="B12" class=" bibr popnode">12</a>).</p><p>Structural analyses of Vps75 reveal the protein to be a homodimer (<a href="#B8" rid="B8" class=" bibr popnode">8</a>,<a href="#B15" rid="B15" class=" bibr popnode">15</a>,<a href="#B16" rid="B16" class=" bibr popnode">16</a>), adopting the &#x02018;headphone&#x02019; fold common to all Nap1-like proteins (<a href="#B17" rid="B17" class=" bibr popnode">17</a>). Subsequent analyses of Vps75 bound to Rtt109, led to the discovery of two distinct structures involving the binding of either one or two Rtt109 molecules per Vps75 dimer (<a href="#B10" rid="B10" class=" bibr popnode">10</a>,<a href="#B18" rid="B18" class=" bibr popnode">18</a>,<a href="#B19" rid="B19" class=" bibr popnode">19</a>). Although the stoichiometries of the two characterized constructs alter the interaction surfaces between Rtt109 and Vps75 in both structures are similar. The physiological relevance of the two alternate structures has yet to be established (reviewed by (<a href="#B20" rid="B20" class=" bibr popnode">20</a>)).</p><p>The structurally related yeast protein Nucleosome Assembly Protein 1 (Nap1) shares many characteristics of Vps75 in that it interacts with all four core histones (<a href="#B17" rid="B17" class=" bibr popnode">17</a>,<a href="#B21" rid="B21" class=" bibr popnode">21</a>,<a href="#B22" rid="B22" class=" bibr popnode">22</a>) and has also been implicated in transcriptional regulation (<a href="#B13" rid="B13" class=" bibr popnode">13</a>,<a href="#B23" rid="B23" class=" bibr popnode">23&#x02013;25</a>). Unlike Vps75, Nap1 has been shown to form higher order assemblies dependent both on the ionic environment and its concentration. Whereas Nap1 is predominantly dimeric at high salt concentrations, independent of its concentration, at lower salt concentrations the protein self associates to form soluble aggregates in a concentration-dependent manner (<a href="#B26" rid="B26" class=" bibr popnode">26&#x02013;28</a>). However, the exact nature of these self-associated states of Nap1 is still unclear, although it has been suggested to involve a &#x003b2;-hairpin loop that protrudes from the core of the protein, and is capable of making beta sheet structures with neighbouring Nap1 molecules (<a href="#B8" rid="B8" class=" bibr popnode">8</a>).</p><p class="p p-last">In this study, we report that Vps75 adopts a homotetrameric conformation in solution at low salt concentrations. Furthermore, we derive a molecular model for the solution structure of Vps75 in its tetrameric conformation through a combination of pulsed electron-electron double resonance (PELDOR), small angle X-ray scattering (SAXS) and structure guided mutagenesis. In addition, our findings using multi angle light scattering (MALS) and PELDOR indicate that the structurally related protein Nap1 also adopts a discrete tetrameric conformation under physiological ionic conditions.</p></div><div id="SEC2" class="tsec sec"><h2 class="head no_bottom_margin" id="SEC2title" style="text-transform: uppercase;">MATERIALS AND METHODS</h2><div id="SEC2-1" class="sec sec-first"><h3 id="SEC2-1title">Recombinant protein expression and purification</h3><p class="p p-first-last">The open reading frame of Vps75 was cloned into a modified pET30a vector carrying a six-histidine N-terminal tag followed by a tobacco etch virus (TEV) protease cleavage site. The two native cysteine residues at positions 21 and 212 were mutated to alanine by polymerase chain reaction based mutagenesis. For site-directed crosslinking spin labelling, a unique cysteine was engineered at position Tyr35. For site-directed spin labelling (SDSL), unique cysteine residues were engineered at position Lys117 and Glu56. Rtt109 was expressed from a pET28a-derived vector, a kind gift from Paul Kaufman. Protein was expressed in <em>Escherichia coli</em> strain BL21 DE3 pLysS (Stratagene) and purified by a combination of cobalt affinity, ion exchange and gel filtration chromatography (<a href="#B29" rid="B29" class=" bibr popnode">29</a>). For mutagenic analysis of Vps75 tetramerisation, point mutants were created in the C21A/C212A construct and isolated using a single step cobalt affinity purification. The open reading frame of Nap1 was cloned into a pET15b vector (Novagen) and the cysteine residues 200, 249, 272 and 414 were mutated to alanine. For SDSL experiments, unique cysteine residues were engineered at E209, T251 and T307.</p></div><div id="SEC2-2" class="sec"><h3 id="SEC2-2title">Deuterated protein expression and purification</h3><p class="p p-first-last">
<em>E. coli</em> (strain BL21 DE3 pLysS (Stratagene)) carrying Vps75 Y35C, K117C or E56C in a modified pET30a expression vector or Nap1 E209C, T251C or T307C in a pET15b expression vector was grown in 20 ml of lysogeny broth (LB) until saturation. Cells were collected by centrifugation and washed twice in 1 ml of Spectra9 fully deuterated media (Cambridge Isotopes Limited) and resuspended in a final volume of 200 ml of the same media. Cells were grown to an OD<sub>600nm</sub> of 0.7 at 37&#x000b0;C (roughly <a href="#B8" rid="B8" class=" bibr popnode">8</a> h of growth), induced with 0.5 mM IPTG at 30&#x000b0;C for 18 h. Cells were collected and lysed by freeze-thaw followed by sonication in 20 mM Tris-HCL pH8.5, 150 mM NaCl supplemented with protease inhibitors. Clarified lysate was bound to 0.5 ml of HisPur cobalt affinity resin (Thermo Scientific), washed extensively and eluted in the same buffer with 200 mM imidazole added. Eluted protein was incubated overnight with TEV protease and 5 mM dithiothreitol, to remove the N-terminal six-histidine tag, and further purified by gel filtration using a Superdex S200 GL 10/300 column (GE Healthcare) equilibrated with 20 mM HEPES-KOH and 0.5 M sodium chloride. Peak fractions were pooled, concentrated using centrifugal concentrators (Amicon Ultra 4 MWCO 10 000, Millipore) and stored at &#x02212;80&#x000b0;C. The extent of deuteration was assessed by MALDI-TOF mass spectrometry and found to be virtually 100% deuterated in all cases (data not shown).</p></div><div id="SEC2-3" class="sec"><h3 id="SEC2-3title">Size exclusion chromatography and multi angle light scattering</h3><p class="p p-first">Size exclusion chromatography and multi angle light scattering (SEC&#x02013;MALS) experiments were performed on a Dionex Ultimate 3000 HPLC system with an inline Wyatt miniDAWN TREOS MALS detector and Optilab T-rEX refractive index detector. Initially SEC&#x02013;MALS experiments were performed on a MAbPac SEC&#x02013;1 (Dionex) column (Figure <a href="/pmc/articles/PMC4027167/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1)</span></span><span>1</span></a>) and due to column degradation subsequent experiments were performed on a Superdex S200 CL 10/300 (GE Healthcare) column (Figures <a href="/pmc/articles/PMC4027167/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -4em;">(Figures5</span></span><span>5</span></a> and <a href="/pmc/articles/PMC4027167/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and7).</span></span><span>7</span></a>). Buffer conditions were 20 mM HEPES-KOH pH 7.5 with sodium chloride concentrations as stated in the text. Molar masses spanning elution peaks were calculated ASTRA v6.0.0.108 (Wyatt).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4027167/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140352970292864"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gku232fig1.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is gku232fig1.jpg" src="/pmc/articles/PMC4027167/bin/gku232fig1.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140352970292864"><a target="object" rel="noopener" href="/pmc/articles/PMC4027167/figure/F1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4027167/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1.</a></div><!--caption a7--><div class="caption"><p>SEC&#x02013;MALS analysis of Vps75 suggests it forms a stable tetramer under physiological conditions. (<strong>A</strong>) SEC&#x02013;MALS elution profiles of full-length Vps75 in 150 mM (blue) and 500 mM (red) sodium chloride. The molar mass over elution peaks is shown in corresponding colours. (<strong>B</strong>) As in (A) except Vps75 lacking the unstructured C-terminal domain was analysed (residues 1&#x02013;223). The association of soluble Vps75 with Vps75 dimers immobilized to a probe via a biotin linkage was measured using bio-layer interferometry. Changes in the response units were measured in the presence of increasing concentrations of soluble Vps75 dimers (<strong>C</strong>). A fit to the binding response data was then used to calculate the dissociation constant, <em>K<sub>d</sub></em> (<strong>D</strong>).</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC4027167/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140352945737760"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gku232fig5.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is gku232fig5.jpg" src="/pmc/articles/PMC4027167/bin/gku232fig5.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140352945737760"><a target="object" rel="noopener" href="/pmc/articles/PMC4027167/figure/F5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC4027167/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5.</a></div><!--caption a7--><div class="caption"><p>Structure-based mutagenesis to disrupt the Vps75 tetramer. (<strong>A</strong>) Model of Vps75 tetramer, opposing Vps75 dimers in blue and green, with the positions of non-tetramerising mutants shown as colour-coded spheres matching SEC&#x02013;MALS elution profiles of the corresponding mutant; K169E (grey), K170E (red) and R164D (black) compared to wild-type Vps75 (blue). (<strong>B</strong>) The location of K78 (yellow spheres) that was mutated to cysteine for disulphide cross-link formation to trap Vps75 in its tetrameric conformation and SEC&#x02013;MALS analysis at 500 mM NaCl of Vps75 disulphide cross-linked at 150 mM NaCl via K78C.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F7" co-legend-rid="lgnd_F7"><a href="/pmc/articles/PMC4027167/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140352977510944"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gku232fig7.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is gku232fig7.jpg" src="/pmc/articles/PMC4027167/bin/gku232fig7.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140352977510944"><a target="object" rel="noopener" href="/pmc/articles/PMC4027167/figure/F7/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F7"><div><a class="figpopup" href="/pmc/articles/PMC4027167/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7">Figure 7.</a></div><!--caption a7--><div class="caption"><p>SEC&#x02013;MALS analysis of the structurally related protein Nap1 suggests it also adopts a stable tetramer mediated by the &#x003b2;-hairpin. (<strong>A</strong>) SEC&#x02013;MALS elution profiles of Nap1 at 150 mM (blue) and 500 mM NaCl (red). (<strong>B</strong>) SEC&#x02013;MALS analysis of the Nap1 PPP mutant. The molar mass over elution peaks is shown in corresponding colours. The association of soluble Nap1 with Nap1 dimers immobilized to a probe via a biotin linkage was measured using bio-layer interferometry. Changes in the response units were measured in the presence of increasing concentrations of soluble Nap1 dimers (<strong>C</strong>). A fit to the binding response data was then used to calculate the dissociation constant, <em>K<sub>d</sub></em> (<strong>D</strong>).</p></div></div></div></div><div id="SEC2-4" class="sec"><h3 id="SEC2-4title">Bio-layer interferometry</h3><p class="p p-first-last">Bio-layer interferometry experiments were performed using dip and read technology on an Octet&#x000ae; RED384 System (ForteBio) using Streptavidin (SA) biosenors. Following purification, Nap1 fold chaperones were biotinylated in a site-specific manner (Vps75 Y35C and Nap1 L120C) with a 5-fold excess of Maleimide-PEG2-Biotin (Thermo Scientific) overnight on ice. Unreacted Maleimide-PEG2-Biotin was removed via gel filtration on a Superdex S200 10/300 column (GE Healthcare). Biosensors were equilibrated in buffer (1 M NaCl, 20 mM HEPES.KOH pH 7.5) for 60 s and subsequently loaded with either biotinylated Nap1 or Vps75 (60 &#x003bc;g/ml) in the same buffer for 600 s. Free Streptavidin binding sites were blocked with biocytin (10 &#x003bc;g/ml) for 60 s and biosensors were washed in fresh equilibration buffer for 120 s. Subsequently, the biosensor was equilibrated in the binding buffer (150 mM NaCl, 20 mM HEPES.KOH pH 7.5 with 0.1% BSA) for 120 s prior to starting the binding assay. All subsequent measurements were performed in the aforementioned binding buffer. During the assay the association of unlabelled wild-type chaperone with its counterpart immobilized on the biosensor was measured for 120 s and the dissociation measured by dipping into fresh binding buffer for 120 s. Experiments were performed in triplicate across a 3-fold dilution concentration range of unlabelled chaperone as specified in Figures <a href="/pmc/articles/PMC4027167/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">Figures1C</span></span><span>1C</span></a> and <a href="/pmc/articles/PMC4027167/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and7C.</span></span><span>7C</span></a>. Data output by the ForteBio data analysis software v. 7.0 was plotted in SigmaPlot 12.0. Steady state analysis of the concentration versus response data was performed in SigmaPlot 12.0 by fitting to a one site saturation ligand binding curve as per inset formula in Figures <a href="/pmc/articles/PMC4027167/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">Figures1D</span></span><span>1D</span></a> and <a href="/pmc/articles/PMC4027167/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and7D</span></span><span>7D</span></a>.</p></div><div id="SEC2-5" class="sec"><h3 id="SEC2-5title">SDSL and site-directed crosslink (X) spin labelling</h3><p class="p p-first-last">Purified Vps75 and Nap1 with unique cysteine residues, as stated in the text, were reduced with 20 mM dithiothreitol for 30 min at room temperature. The dithiothreitol was removed by gel filtration on a Superdex S200 GL 10/300 column (GE Healthcare) run in 20 mM HEPES-KOH pH 7.5, 0.5 M sodium chloride. Protein containing fractions were combined and concentrated using an Amicon Ultra centrifugal concentrator MWCO 10 000 (Millipore) to 50 &#x003bc;M. For site-directed crosslink (X) spin labelling (SDXSL) of Vps75 Y35C, the bi-functional crosslinker spin label 3,4-Bis-MTSL (Toronto Research Chemicals Inc.) was dissolved in dimethyl sulfoxide and titrated against the reduced cysteine pair in 0.2 M equivalents, up to a final ratio of 1 M Vps75 to 1.2 M 3,4-bisMTSL, creating the crosslinked spin-labelled sample Vps75 Y35RX2. After each addition, the reactions were vigorously mixed and allowed to proceed at 25&#x000b0;C for 3 min before continuing with the next addition. The efficiency of labelling was assessed by separation of crosslinked from non-crosslinked species by sodium dodecyl sulphate-polyacrylamide gel electrophoresis (SDS-PAGE) and coomassie staining. A labelling efficiency of 90% was typically achieved. For SDSL of Vps75 K117C and E56C and Nap1 E209C, T251C and T307C, a 10-fold excess of MTSL was added to the pooled peak fractions and labelling left to proceed for 20 min at 25&#x000b0;C, before dialysis overnight at 4&#x000b0;C against the same buffer to remove all traces of unreacted MTSL. Labelled protein was then exchanged to buffer containing deuterium oxide (Sigma-Aldrich) in place of water, double the final sodium chloride concentration (as mentioned in the text) and 20 mM HEPES-KOH pH 7.5, using centrifugal concentrators (Millipore). After four rounds of exchange, protein was concentrated to 200&#x02013;300 &#x003bc;M dimer, mixed 1:1 with D8-glycerol (CIL), giving a final concentration of 100&#x02013;150 &#x003bc;M dimer, in a final volume of 100 &#x003bc;l and stored at &#x02212;80&#x000b0;C until PELDOR measurements were made.</p></div><div id="SEC2-6" class="sec"><h3 id="SEC2-6title">PELDOR</h3><p class="p p-first-last">PELDOR experiments were conducted as described previously (<a href="#B29" rid="B29" class=" bibr popnode">29</a>) using a Bruker ELEXSYS E580 spectrometer operating at X band with a dielectric ring resonator and a Bruker 400U second microwave source unit. Data analysis was carried out using the DeerAnalysis 2013 package (<a href="#B30" rid="B30" class=" bibr popnode">30</a>). The experimentally obtained time domain traces were first adjusted to remove background decay. Tikhonov regularisation was then used to obtain the most appropriate distance distributions from each dataset.</p></div><div id="SEC2-7" class="sec"><h3 id="SEC2-7title">Modelling the structure of the Vps75 tetramer</h3><p class="p p-first-last">Refinement of the Vps75 tetramer structure was carried out using XPLOR-NIH (<a href="#B31" rid="B31" class=" bibr popnode">31</a>). Spin labels were added either as random coordinates and then regularized using molecular dynamics with the rest of the protein held fixed, or alternatively MTSL was added using the Pymol plugin MTSSLWizard (<a href="#B32" rid="B32" class=" bibr popnode">32</a>). From both of these approaches ensembles of spin-label conformations were obtained that sampled the available space. The nitrogen atoms of each spin label in each ensemble of conformations were incorporated into the coordinate file as single residues. Distance restraints determined from the distance distributions measured by PELDOR were incorporated as Average = R-3 restraints between label ensembles and rigid body dynamics was used to minimize the distance restraint violations. Symmetry constraints were used to ensure that the overall symmetry of the Vps75 tetramer contacts was preserved. Multiple refinement runs were carried out starting from separated Vps75 dimer arrangements. Final structures with high distance restrain violations were discarded and low violation structures retained. For information on the distances restraints used to model the Vps75 and deviations observed in the final models see Supplementary Table S1.</p></div><div id="SEC2-8" class="sec sec-last"><h3 id="SEC2-8title">SAXS</h3><p class="p p-first-last">The SAXS measurements for the C-terminal truncated Vps75 and the full-length Vps75 under low (150 mM NaCl) and higher salt concentrations (500 mM NaCl) were made at the EMBL BioSAXS beamline X33 (<a href="#B33" rid="B33" class=" bibr popnode">33</a>) at the DORIS storage ring, DESY (Hamburg, Germany). A photon counting PILATUS 1M detector was used to record the scattered X-ray at a wavelength of 1.54 &#x000c5;. The sample-to-detector distance was 2.7 m, yielding a maximum recordable momentum transfer (<em>s</em> = 4&#x003c0; sin&#x003b8;/&#x003bb;) of 0.5 &#x000c5;<sup>&#x02212;1</sup>. A bovine serum albumin solution at 4.5 mg/ml in 50 mM Hepes pH 7.5 was used for calibrating the molecular mass. A series of concentrations between 0.5 and 10 mg/ml was measured for each sample at 10&#x000b0;C and no concentration dependence of the scattering profiles was observed for all samples. The standard SAXS data reduction and analysis were done using PRIMUS (<a href="#B34" rid="B34" class=" bibr popnode">34</a>). The pair distance distribution function <em>p(r)</em> and maximum size of the protein molecules <em>D<sub>max</sub></em> was calculated with the program GNOM (<a href="#B35" rid="B35" class=" bibr popnode">35</a>). The program CRYSOL (<a href="#B36" rid="B36" class=" bibr popnode">36</a>) was used to calculate the theoretical X-ray scattering profile with the high-resolution model from crystal structures. Missing termini were represented by a chain of dummy residues and added to the crystal structures using the program BUNCH (<a href="#B37" rid="B37" class=" bibr popnode">37</a>) to best fit the experimental data from full-length Vps75.</p></div></div><div id="SEC3" class="tsec sec"><h2 class="head no_bottom_margin" id="SEC3title" style="text-transform: uppercase;">RESULTS</h2><div id="SEC3-1" class="sec sec-first"><h3 id="SEC3-1title">Two dimers of Vps75 self associate to form a homotetramer</h3><p class="p p-first">Like all Nap1 related proteins characterized to date, Vps75 dimerizes, however, higher order associations of this chaperone have yet to be reported. During biochemical analysis of Vps75 by gel filtration chromatography, we discovered that its elution profile changes depending on the ionic conditions employed. At 150 mM sodium chloride, Vps75 elutes significantly earlier during size exclusion chromatography than at high salt concentrations indicative of Vps75 oligomerisation beyond the dimeric form (Figure <a href="/pmc/articles/PMC4027167/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1A).</span></span><span>1A</span></a>). To identify the oligomeric form of Vps75 present under these two conditions, SEC&#x02013;MALS was employed for molecular weight (MW) determination. Under the stringent conditions of 500 mM sodium chloride, Vps75 eluted with an average MW of 60.1 kDa (Figure <a href="/pmc/articles/PMC4027167/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1A),</span></span><span>1A</span></a>), close to the theoretical mass of 61.2 kDa calculated for a dimer. At 150 mM sodium chloride, Vps75 eluted with an average MW of 117.9 kDa, close to the theoretical MW of two dimers, 122.4 kDa. Thus, it appears that two dimers of Vps75 can associate to form a tetramer in a manner dependent on electrostatic interactions.</p><p>Vps75 contains three major domains: the structurally defined dimerisation and earmuff domains and an acidic C-terminal domain thought to be devoid of stable secondary and tertiary structure (<a href="#B8" rid="B8" class=" bibr popnode">8</a>,<a href="#B15" rid="B15" class=" bibr popnode">15</a>,<a href="#B16" rid="B16" class=" bibr popnode">16</a>). We were interested to know whether the interaction between two Vps75 dimers was mediated by the folded, globular region or the disordered C-terminal region of Vps75. To address this we created a truncated version of Vps75 lacking the C-terminal acidic region. This construct was analysed by gel filtration coupled to MALS and found to display the same characteristics of the full-length protein (Figure <a href="/pmc/articles/PMC4027167/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1B),</span></span><span>1B</span></a>), eluting with an average MW of 56.2 kDa (theoretical dimer&#x02014;54 kDa) and 110.9 kDa (theoretical tetramer&#x02014;108 kDa), in stringent and physiological conditions, respectively. We therefore concluded that interactions mediating tetramerisation must involve the structurally defined core of the protein.</p><p class="p p-last">In order to assess the concentration dependence of Vps75 self association, Vps75 Y35C dimers were specifically labelled with Maleimide-PEG2-Biotin that provided a means of attachment to Streptavidin biosensors. After attachment and washing, association of unlabelled Vps75 was measured by bio-layer interferometry and used to calculate a <em>K<sub>d</sub></em> for tetramerisation of 0.6 uM (Figure <a href="/pmc/articles/PMC4027167/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1C</span></span><span>1C</span></a> and <a href="/pmc/articles/PMC4027167/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">andD</span></span><span>D</span></a>).</p></div><div id="SEC3-2" class="sec"><h3 id="SEC3-2title">Analysis of the tetrameric structure of Vps75 in solution by SDXSL and PELDOR</h3><p class="p p-first">Vps75 has previously been crystallized as a homodimer (<a href="#B8" rid="B8" class=" bibr popnode">8</a>,<a href="#B15" rid="B15" class=" bibr popnode">15</a>,<a href="#B16" rid="B16" class=" bibr popnode">16</a>). In order to characterize the tetrameric conformation we detected in solution, we set out to obtain distance measurements within the complex using PELDOR in conjunction with SDSL. The traditional approach for SDSL utilizes a thiol reactive nitroxide compound attached to a unique cysteine residue that is engineered at a location of choice through genetic manipulation of the coding sequence. When analysing a homodimeric complex a single cysteine residue per polypeptide chain results in a doubly labelled system. With regard to homotetrameric structures, a one-chain-one-label approach results in four spins per protein complex. Such label dense systems are often difficult to deconvolute, as multiple distances are encoded within a single spectrum. To avoid this and maintain a two-spin system, we investigated the possibility of singly labelling a dimer through crosslinking of two cysteine residues using the bi-functional crosslinking spin label 3,4-Bis-(methanethiosulfonylmethyl)-2,2,5,5-tetramethyl-2,5-dihydro-1H-pyrrol-1-yloxy radical or 3,4-Bis-MTSL (Figure <a href="/pmc/articles/PMC4027167/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2A).</span></span><span>2A</span></a>). We scanned Vps75 for a residue that was both solvent accessible and came into close proximity with its partner residue in the opposing dimer. The tyrosine at position 35 fulfilled these criteria and reacted well with 3,4-Bis-MTSL, crosslinking to a high efficiency (Figure <a href="/pmc/articles/PMC4027167/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2A</span></span><span>2A</span></a> and <a href="/pmc/articles/PMC4027167/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">andB).</span></span><span>B</span></a>). Specificity of crosslinking across the dimerisation interface was assessed by gel filtration chromatography (data not shown). Whilst this research was being carried out, a similar approach was reported that used the same compound to label two adjacent cysteine residues, but on the same polypeptide chain (<a href="#B38" rid="B38" class=" bibr popnode">38</a>). To highlight the difference in crosslinking labelling across a dimeric interface as a means to singly label a protein dimer, we named this spin-labelled side chain &#x02018;RX2&#x02019;, and this alternate form of spin labelling SDXSL (Figure <a href="/pmc/articles/PMC4027167/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2A</span></span><span>2A</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4027167/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140352944499872"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gku232fig2.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is gku232fig2.jpg" src="/pmc/articles/PMC4027167/bin/gku232fig2.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140352944499872"><a target="object" rel="noopener" href="/pmc/articles/PMC4027167/figure/F2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4027167/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2.</a></div><!--caption a7--><div class="caption"><p>Probing the structure of the Vps75 tetramer through site-directed crosslinking spin labelling. (<strong>A</strong>) The position of Y35RX2 in the Vps75 dimer. The boxed expansion shows a possible conformation of the crosslinked spin-labelled residue RX2 at position Y35 determined from molecular dynamic simulations. Monomers are differentially coloured blue and green for clarity. (<strong>B</strong>) Analysis of site-directed crosslink spin labelling. The compound 3,4-Bis-MTSL was titrated against a constant concentration of Vps75. Crosslinking efficiency was determined by resolving crosslinked from uncrosslinked species through separation by SDS-PAGE and quantification of coomassie staining (gel shown in inset). An arrow head indicates a 1:1 ratio of crosslinker to Vps75 dimer. (<strong>C</strong>) Schematic representation of three possible models for the tetramerisation of Vps75. Alternative dimers are shown in green and blue. Stars represent the approximate location of the spin-labelled side chain Y35RX2. PELDOR analysis of Vps75 Y35RX2 at 150 mM NaCl, raw dipolar evolution plot with background fit (<strong>D</strong>), background corrected dipolar evolution (E) and corresponding distance distribution (F) for Y35RX2. The modal distance between Y35RX2 residues is 7.8 nm.</p></div></div></div><p>We considered three modes via which Vps75 dimers could interact to form tetramers (Figure <a href="/pmc/articles/PMC4027167/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2C).</span></span><span>2C</span></a>). Firstly, through interactions solely involving the earmuff domains in a side-by-side configuration termed &#x02018;M&#x02019;. This model seems unlikely as it does not suggest a mechanism whereby oligomerisation of Vps75 is curtailed at the tetrameric level. In a second model, tetramerisation could form a ring structure through contacts between the earmuff domains termed &#x02018;O&#x02019;. In a third model, the antiparallel helices of the dimerisation domain could form a four-helix bundle or &#x02018;X&#x02019; structure. Initial Electron Paramagnetic Resonance (EPR) measurements with Vps75 Y35RX2 under tetramerising conditions revealed the presence of a long distance in excess of 7 nm (data not shown). Such large distances are difficult to define due to the traverse relaxation of the excited state of the spin-label radical electrons by the nuclear spins of surrounding protons. This detrimentally affects the timing window over which the raw dipolar evolution function can be measured. Recently, we have shown that complete deuteration of proteins can significantly extend the relaxation time <em>T</em><sub>m</sub> allowing dipolar interactions between the spin labels to be measured over a longer time period, thereby allowing measurement of much larger distances (<a href="#B39" rid="B39" class=" bibr popnode">39</a>). Indeed, deuteration of Vps75 Y35RX2 extended the timing window over which the dipolar evolution function could be measured to 15 &#x003bc;s that was sufficient to observe the full spin echo oscillation (Figure <a href="/pmc/articles/PMC4027167/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2D).</span></span><span>2D</span></a>). Upon Tikhonov regularisation of the background corrected dipolar evolution function to a distance distribution (Figure <a href="/pmc/articles/PMC4027167/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2E),</span></span><span>2E</span></a>), a modal distance of 7.8 nm between the two cross-linked Y35RX2 residue pairs in the Vps75 tetramer was extracted (Figure <a href="/pmc/articles/PMC4027167/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2F</span></span><span>2F</span></a>).</p><p class="p p-last">Although long enough to rule out the &#x02018;X&#x02019; model, 7.8 nm between RX2 labels could still be satisfied by both the &#x02018;M&#x02019; and &#x02018;O&#x02019; models. The &#x02018;M&#x02019; model was rejected in favour of the &#x02018;O&#x02019; model that provides a more obvious mechanism of termination of Vps75 oligomerisation at the tetrameric level as observed by SEC&#x02013;MALS (see below). Initial attempts to define a structural model of the Vps75 tetramer using the crystal structure of the Vps75 dimer in a ring-like orientation revealed the extracted distance between Y35RX2 residues could be met by a limited number of sterically allowed dimer&#x02013;dimer orientations rotated around an axis drawn between the two Y35RX2 labels. Thus, further constraints were sought to confirm and refine the rotational position of the two Vps75 dimers relative to each other.</p></div><div id="SEC3-3" class="sec"><h3 id="SEC3-3title">Refinement of the Vps75 tetramer structure using a four-spin system</h3><p class="p p-first">In order to define the rotational relationship between the Vps75 dimers, labelling sites on the earmuff domains were chosen with the aim of defining the short distance between earmuffs across the tetramerisation interface. Such a labelling strategy results in a four-spin system from which it is often difficult to precisely interpret PELDOR data. A four-spin system consisting of spins a, b, c and d, with two perpendicular planes each possessing 2-fold rotational symmetry (that of the dimer and that of the tetramer), will result in distances AB, CD and AD (Figure <a href="/pmc/articles/PMC4027167/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3A).</span></span><span>3A</span></a>). The distance across the dimer interface was denoted AB (which is equivalent to the distance CD). The distances across the tetramer interface were denoted AC and AD (which are equivalent to BD and BC, respectively). From initial modelling attempts a strategy was devised to probe labelling sites on the earmuff domains of Vps75 that would most likely produce short AC distances that were resolvable from longer AB and AD distances (Figure <a href="/pmc/articles/PMC4027167/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3A).</span></span><span>3A</span></a>). Thus, whilst absent in the dimer, upon tetramerisation these shorter distances should be easily identifiable. Labelling sites at positions K117 and E56 were chosen for this purpose.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC4027167/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140352944343984"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gku232fig3.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is gku232fig3.jpg" src="/pmc/articles/PMC4027167/bin/gku232fig3.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140352944343984"><a target="object" rel="noopener" href="/pmc/articles/PMC4027167/figure/F3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC4027167/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3.</a></div><!--caption a7--><div class="caption"><p>Probing the structure of the Vps75 tetramer using quadruply labelled complex. (<strong>A</strong>) Schematic representation of the experimental approach for probing the Vps75 tetramer structure using a quadruply spin-labelled system. In the highly ionic environment of 500 mM sodium chloride the dimeric form of Vps75 is prevalent, giving a single distance AB (left). At 150 mM sodium chloride the tetrameric form is prevalent, resulting in additional distances AC and AD (right). (<strong>B</strong>) Background corrected dipolar evolution for Vps75 E56R1 (left) and Vps75 K117R1 (right) under dimeric (red) and tetrameric (blue) conditions. (<strong>C</strong>) Distance distributions for Vps75 E56R1 (left) and Vps75 K117R1 (right) under dimeric (red) and tetrameric (blue) conditions. (<strong>D</strong>) A molecular model of the Vps75 tetramer was generated using the RX2 distance, and the E56R1 AC distance. The two Vps75 dimers are differentially coloured green and blue for clarity. Left: the &#x02018;Ring&#x02019; model with Y35RX2 and K117R1 labelling sites circled (E56R1 is omitted for clarity). The RX2-RX2&#x02019; distance is depicted as a black line, the AB distance is shown as a red line and AC and AD distances are shown as blue lines. Right: side view of the Vps75 tetramer with all labelling sites circled.</p></div></div></div><p>The intradimer distance, AB, for both K117R1 and E56R1, was assigned by carrying out PELDOR under high salt conditions where Vps75 is in its dimeric form (Figure <a href="/pmc/articles/PMC4027167/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3B</span></span><span>3B</span></a> and <a href="/pmc/articles/PMC4027167/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">andC).</span></span><span>C</span></a>). Protein deuteration allowed the measurement of distances close to 8.5 nm for K117R1 and 7.3 nm for E56R1, to be extracted, as was expected from current crystal structures of the Vps75 dimer. Carrying out the same experiment under conditions that favour tetramer formation revealed distinct changes in the dipolar evolution for both sites (Figure <a href="/pmc/articles/PMC4027167/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3B</span></span><span>3B</span></a> and <a href="/pmc/articles/PMC4027167/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">andC).</span></span><span>C</span></a>). Changes in the modulation depth indicated a four-spin system (<a href="#B40" rid="B40" class=" bibr popnode">40</a>), as expected, for both labelling sites, with the oscillation becoming more convoluted compared to that of the dimer. Tikhonov regularisation revealed additional unique distances (Figure <a href="/pmc/articles/PMC4027167/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3C).</span></span><span>3C</span></a>). In both cases a well-resolved shorter distance distribution was present, which was assigned to the theoretical distance AC. The K117R1 labelling site produced an additional long distance of 9.1 nm, which was assigned the theoretical distance AD (Figure <a href="/pmc/articles/PMC4027167/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3C).</span></span><span>3C</span></a>). The distance AD was not as well resolved for E56R1, and most likely overlaps quite closely with the distance AB (Figure <a href="/pmc/articles/PMC4027167/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3C</span></span><span>3C</span></a>).</p><p class="p p-last">Using the additional distances gained from the four-spin system, a molecular model of the Vps75 tetramer was produced using the crystal structure of the dimer (PDB code: 2ZD7) as a building block. Spin labels were built onto the atomic structure of the Vps75 and molecular dynamic simulations were carried out to identify likely conformational ensembles for each nitroxide labelling site. Rigid body minimisation, using PELDOR distance measurements (Supplementary Table S1) and non-crystallographic symmetry restraints, was then used to refine the structure of the tetramer. The resulting model indicates that interactions between the earmuff domains contribute to the tetramerisation interface with a 2-fold plane of symmetry running through the earmuff&#x02013;earmuff interaction interface of alternate dimers, resulting in rotation of the dimerisation domains by &#x0223c;60&#x000b0; with respect to each other (Figure <a href="/pmc/articles/PMC4027167/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3D</span></span><span>3D</span></a>).</p></div><div id="SEC3-4" class="sec"><h3 id="SEC3-4title">SAXS analysis of the Vps75 tetramer</h3><p class="p p-first">To further validate the model of the Vps75 tetramer observed in SEC&#x02013;MALS and EPR experiments, SAXS experiments were carried out. Guinier analysis of the SAXS data for C-terminal truncated Vps75 under high salt concentrations (500 mM NaCl) yielded a radius of gyration (<em>R</em><sub>g</sub>) of 32.5 &#x000b1; 0.5 &#x000c5; and a MW of 60 &#x000b1; 5 kDa, which is approximately twice the MW estimated from the protein sequence of a Vps75 monomer, 26.5 kDa. In comparison the CRYSOL (<a href="#B36" rid="B36" class=" bibr popnode">36</a>) calculation of the corresponding atomic model of the Vps75 dimer (PDB accession code: 2ZD7) yields a consistent value of <em>R</em><sub>g</sub> = 31.8 &#x000c5;. A direct comparison of the experimental scattering profile with that calculated from the crystal structure gives a good fit with discrepancy &#x003c7; = 1.4 (data not shown). These results confirm that the C-terminal truncated Vps75 molecule forms a stable dimer in solutions containing high salt concentrations and the conformation observed in the crystal structure (PDB code: 2ZD7) is preserved.</p><p>The MWs of the full-length Vps75 assemblies estimated from SAXS analysis are 65 &#x000b1; 5 and 120 &#x000b1; 10 kDa for high and low salt concentrations, respectively. This is in close agreement with the theoretical MWs of a dimer, 61.2 kDa, and tetramer, 122.4 kDa and consistent with the SEC&#x02013;MALS analysis. As expected from the MW values, the change of ionic conditions leads to a dramatic change in structure of the Vps75 solute, evidenced by the markedly different scattering curves and pair distance distribution functions <em>p</em>(<em>r</em>) (Figure <a href="/pmc/articles/PMC4027167/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure4).</span></span><span>4</span></a>). The most probable pair distance (maximum of the <em>p</em>(<em>r</em>) function) of Vps75 increases from 41 to 52 &#x000c5; (Figure <a href="/pmc/articles/PMC4027167/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure4A)</span></span><span>4A</span></a>) upon tetramerisation and at smaller distances a close to linear increase in <em>p</em>(<em>r</em>) is observed that is typical of hollow particles. Moreover, the maximum size of the tetramer (<em>D</em><sub>max</sub> = 12 nm) increases only marginally compared to the dimer (<em>D</em><sub>max</sub> = 11 nm), and this clearly favours &#x02018;O&#x02019; model. Furthermore, bead models that fit the observed scattering were frequently observed to have a ring-like organisation (Supplementary Figure S1). Indeed, the increase of the maximum size would be larger if Vps75 were to adopt the &#x02018;M&#x02019;-like structure.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC4027167/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140352970286592"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4027167_gku232fig4.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gku232fig4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4027167/bin/gku232fig4.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140352970286592"><a target="object" rel="noopener" href="/pmc/articles/PMC4027167/figure/F4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC4027167/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4.</a></div><!--caption a7--><div class="caption"><p>SAXS analysis of Vps75 under tetrameric and dimeric conditions. (<strong>A</strong>) Normalized pair&#x000a0;distance distribution functions&#x000a0;(r) for Vps75 at 150 mM (blue, tetramer) and 500 mM NaCl (red, dimer). (<strong>B</strong>) Comparison of experimental SAXS curve (grey) with a CRYSOL calculated scattering curve (red) derived from a crystal structure of the dimer (PDB: 2ZD7), under high ionic conditions (500 mM sodium chloride). The crystal structure with unresolved residues modelled as a random coil is shown in the inset. (<strong>C</strong>) Comparison of the experimental SAXS curve (grey) under 150 mM sodium chloride, with a CRYSOL calculated curve of the Vps75 tetramer model (blue). The inset shows the modelled Vps75 tetramer with unresolved residues added in a random coil conformation. One dimer is shown in green, the other in blue. &#x003c7;<sup>2</sup> values for each fit are shown.</p></div></div></div><p class="p p-last">In order to account for the unstructured C-terminal tails in full-length Vps75 assemblies, the C-termini were modelled as chains of dummy residues added to the crystal structure of Vps75 using the program BUNCH (<a href="#B36" rid="B36" class=" bibr popnode">36</a>). The excellent agreement between the calculated scattering curve of the resultant model of the full-length Vps75 dimer and the experimental curve is illustrated in Figure <a href="/pmc/articles/PMC4027167/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure4b</span></span><span>4b</span></a> (<em>&#x003c7;</em> = 1.41). The calculated scattering curve of the Vps75 tetramer with the added C-terminal residues is also in close agreement with the experimental scattering curve (<em>&#x003c7;</em> = 2.53, Figure <a href="/pmc/articles/PMC4027167/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure4c)</span></span><span>4c</span></a>) and as such provides independent verification that the structure derived from the PELDOR distance restraints serves as a good representation of the Vps75 tetramer. The small but significant increase in the <em>R</em><sub>g</sub> value from 34.5 &#x000b1; 0.3 &#x000c5; in high salt to 38.2 &#x000b1; 0.1 &#x000c5; in low salt conditions is in line with CRYSOL predicted values of 34.6 and 40.4 &#x000c5; calculated for the model of Vps75 with tails in a dimeric and tetrameric assembly, respectively.</p></div><div id="SEC3-5" class="sec"><h3 id="SEC3-5title">Structure guided mutagenesis identifies residues at the tetramerisation interface</h3><p class="p p-first">Close inspection of the tetramer model revealed extensive charge complementarity between interacting earmuff domains. An acidic patch on one earmuff domain appeared to interact with a basic patch of another in an orthogonal fashion, providing a basis for the sensitivity of Vps75 to high ionic strengths. To further test the model, we generated 13 charge reversal point mutations within this region and tested their ability to tetramerize through gel filtration chromatography (Supplementary Table S2). A number of these mutants significantly affected the elution profile of Vps75 and were found to cluster in the eighth alpha helix of Vps75 and the preceding loop (R164D, K169E and K170E). The three mutations that most strongly affected elution were further characterized by SEC&#x02013;MALS analysis under the tetramerising conditions of 150 mM sodium chloride (Figure <a href="/pmc/articles/PMC4027167/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5A).</span></span><span>5A</span></a>). The MW of the mutants, Vps75 R164D (73.2 kDa), K169E (67.8 kDa) and K170E (71.9 kDa), suggests that the equilibrium for tetramerisation is significantly shifted towards the dimeric form. As such these mutants were termed non-tetramerising mutants. A triple mutant protein (R164D, K169E, K170E) also elutes with the mass of a dimer at under low salt conditions (Supplementary Figure S2).</p><p>Upon further inspection of the tetramer model derived from EPR measurements, the residue K78 appeared to come into close proximity with itself on the opposite dimer Vps75 within the ring-like tetramer. By mutating K78 to cysteine and using redox potential of Cu (II) phenanthroline as a catalyst to oxidize the cysteine residues, a disulphide bond was formed at 150 mM sodium chloride across the tetramerisation interface (not shown) demonstrating that indeed this residue does come into contact with itself in the opposing dimer. Furthermore, the presence of the induced disulphide effectively stabilized Vps75 tetramers that elute from a size exclusion column with the mass anticipated for a tetramer even under the high salt conditions that dissociate uncross-linked tetramers into dimers (Figure <a href="/pmc/articles/PMC4027167/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5B).</span></span><span>5B</span></a>). In addition, the efficiency of cross-linking at this site was reduced at high salt concentrations that favour the presence of dimers (Supplementary Figure S2A). This provides further validation that the tetramerisation interface has been defined correctly.</p><p class="p p-last">We also analysed the effect of adding histones to Vps75 on its ability to form cross-linked tetramers. The efficiency of Cu (II) phenanthroline cross-linking of Vps75 K78C was reduced upon titration with histones H3&#x02013;H4 (Supplementary Figure S2B). This suggests that Vps75, in the tetrameric conformation described here, does not bind histones H3&#x02013;H4. However, one cannot exclude the potential for a reconfiguration of the Vps75 tetramer upon histone binding that disfavours Cu (II) phenanthroline cross-linking at Vps75 K78C. In either case, these observations support the hypothesis that Vps75 tetramerisation restricts access to histone binding surfaces.</p></div><div id="SEC3-6" class="sec"><h3 id="SEC3-6title">Tetrameric Vps75 is incompatible with Rtt109 binding</h3><p class="p p-first-last">In addition to the histone chaperone function, Vps75 also stimulates the catalysis of histone H3 acetylation when in complex with the histone acetyltransferase Rtt109 (<a href="#B6" rid="B6" class=" bibr popnode">6</a>,<a href="#B7" rid="B7" class=" bibr popnode">7</a>,<a href="#B11" rid="B11" class=" bibr popnode">11</a>,<a href="#B12" rid="B12" class=" bibr popnode">12</a>). Structural alignment between our tetrameric model of Vps75 in solution and crystal structures of the Vps75:Rtt109 complex (<a href="#B10" rid="B10" class=" bibr popnode">10</a>,<a href="#B18" rid="B18" class=" bibr popnode">18</a>,<a href="#B19" rid="B19" class=" bibr popnode">19</a>) predicts the tetrameric form of Vps75 to be incompatible with Rtt109 binding (Figure <a href="/pmc/articles/PMC4027167/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6A).</span></span><span>6A</span></a>). We tested this incompatibility by PELDOR utilising the singly labelled Vps75 Y35RX2 dimer. If Rtt109 were to disrupt the Vps75 tetramer, we would expect the modulation depth of the dipolar coupling to decrease as a function of Rtt109 concentration. Indeed, titrating in Rtt109 reduced the echo modulation depth of the doubly labelled Vps75 tetramer until, at a 2:1 (Vps75:Rtt109) stoichiometry, the resulting spectra represented only the exponential background decay of a single spin system (Figure <a href="/pmc/articles/PMC4027167/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6B).</span></span><span>6B</span></a>). This finding demonstrates that, in agreement with predictions from the model built of the Vps75 tetramer, Rtt109 binding to Vps75 is indeed incompatible with Vps75 in a tetrameric conformation. Thus Rtt109 binding obscures the Vps75 tetramerisation interface.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F6" co-legend-rid="lgnd_F6"><a href="/pmc/articles/PMC4027167/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140352943420992"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gku232fig6.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is gku232fig6.jpg" src="/pmc/articles/PMC4027167/bin/gku232fig6.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140352943420992"><a target="object" rel="noopener" href="/pmc/articles/PMC4027167/figure/F6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F6"><div><a class="figpopup" href="/pmc/articles/PMC4027167/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6">Figure 6.</a></div><!--caption a7--><div class="caption"><p>Vps75 tetramerisation is disrupted by binding of Rtt109. (<strong>A</strong>) The tetrameric model of Vps75 in solution aligned to the crystal structure of the Vps75:Rtt109 complex (PDB: 3Q68). Vps75 dimers from the tetrameric model are represented in cartoon format and differentially coloured green and blue. The Vps75:Rtt109 crystal structure was aligned to the green Vps75 dimer of the tetrameric model. The surface of Rtt109 from the crystal structure is shown in red, whereas the surface of Vps75 from the crystal structure is omitted for clarity. (<strong>B</strong>) Background corrected dipolar evolutions from 100 &#x003bc;M Vps75 Y35RX2 dimer in the presence of Rtt109 at a ratio of 2:0, 2:0.25, 2:0.5 and 2:1, Vps75:Rtt109 as indicated. The reduction in the drop in spin echo in the presence of increasing Rtt109 indicates a transition from all spin-label-pairs to no spin-label-pairs in the sample, which is consistent with the conversion of double labelled tetrameric Vps75 to single labelled dimeric Vps75 associated with Rtt109.</p></div></div></div></div><div id="SEC3-7" class="sec"><h3 id="SEC3-7title">The structurally related protein, Nap1, also forms tetramers</h3><p class="p p-first-last">SEC&#x02013;MALS analysis of Nap1 at 150 mM sodium chloride was consistent with the dominant species in solution being tetrameric in equilibrium with a small proportion of Nap1 dimer with a weight-averaged molecular mass of 187.7 kDa (theoretical tetramer 191.5 kDa). As predicted upon raising the salt concentration to 500 mM sodium chloride, the Nap1 reverts to its dimeric form with a mass of 97.8 kDa (theoretical dimer 95.7 kDa) obtained from SEC&#x02013;MALS analysis (Figure <a href="/pmc/articles/PMC4027167/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure7A).</span></span><span>7A</span></a>). The previously reported Nap1 mutant (<a href="#B8" rid="B8" class=" bibr popnode">8</a>), in which three residues within the &#x003b2;-hairpin domain that protrude from the Nap1 earmuff domain are mutated to proline to inhibit secondary structure formation in this region, R301P, T302P, K305P, (PPP) gives a mass of 98.4 kDa at 150 mM sodium chloride, indicating a predominantly dimeric state (Figure <a href="/pmc/articles/PMC4027167/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure7B).</span></span><span>7B</span></a>). This indicates that indeed the &#x003b2;-hairpins can influence tetramerisation. The concentration dependence of Nap1 self-association was assessed by bio-layer interferometry. Nap1 L120C dimers were specifically labelled with Maleimide-PEG2-Biotin and attached to Streptavidin biosensors. After washing at high salt and equilibrating at low salt, the association of unlabelled Nap1 was measured by bio-layer interferometry and used to calculate a <em>K<sub>d</sub></em> for tetramerisation of 3.57 &#x003bc;M (Figure <a href="/pmc/articles/PMC4027167/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure7C</span></span><span>7C</span></a> and <a href="/pmc/articles/PMC4027167/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">andD</span></span><span>D</span></a>).</p></div><div id="SEC3-8" class="sec sec-last"><h3 id="SEC3-8title">Probing the Nap1 tetramer structure using a four-spin system</h3><p class="p p-first">We initially attempted to devise a RX2 labelling strategy for labelling Nap1 at L120, the equivalent position to the Vps75 RX2 labelling site, however, labelling at this site was inefficient (not shown). Upon further analysis, the Nap1 L120 site appeared to be more buried than its Vps75 counterpart, which may explain the poor cross-link labelling efficiency observed for this site. Due to the lack of a suitable RX2 labelling site a four-spin system was used to further characterize the Nap1 tetramer. As the &#x003b2;-hairpins of opposing Nap1 dimers are implicated in the tetramerisation of Nap1 (Figure <a href="/pmc/articles/PMC4027167/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure7B),</span></span><span>7B</span></a>), the sites T307 on the &#x003b2;-hairpin and E209 in close proximity to this region were chosen as labelling sites (Figure <a href="/pmc/articles/PMC4027167/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure8a).</span></span><span>8a</span></a>). In addition, the more distal site T251 was chosen to acquire additional restraints. PELDOR analysis of the three MTSL labelled cysteine mutants was performed under tetrameric (150 mM NaCl) and dimeric (500 mM NaCl) conditions. The background corrected dipolar evolution functions of all three labelling sites displayed the characteristic drop in oscillation depth associated with the increase in interacting spins in the system from two at 500 mM sodium chloride to four upon tetramerisation at 150 mM sodium chloride (Figure <a href="/pmc/articles/PMC4027167/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure8B</span></span><span>8B</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F8" co-legend-rid="lgnd_F8"><a href="/pmc/articles/PMC4027167/figure/F8/" target="figure" rid-figpopup="F8" rid-ob="ob-F8"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140352979212304"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gku232fig8.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is gku232fig8.jpg" src="/pmc/articles/PMC4027167/bin/gku232fig8.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140352979212304"><a target="object" rel="noopener" href="/pmc/articles/PMC4027167/figure/F8/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F8"><div><a class="figpopup" href="/pmc/articles/PMC4027167/figure/F8/" target="figure" rid-figpopup="F8" rid-ob="ob-F8">Figure 8.</a></div><!--caption a7--><div class="caption"><p>Probing the structure of the Nap1 tetramer using a quadruple spin system. (<strong>A</strong>) Schematic representation of the experimental approach for probing the Nap1 tetramer structure using a quadruply spin-labelled system. In 500 mM sodium chloride (red) the dimeric form of Nap1 is prevalent, giving a single distance AB (left). In 150 mM sodium chloride the tetrameric form is prevalent, resulting in additional distances AC and AD (right). (<strong>B</strong>) Corrected dipolar evolution for Nap1 E209R1 (left), Nap1 T251R1 (middle) and Nap1 T307R1 (right) under 500 mM NaCl (red) and 150mM NaCl (blue) conditions. (<strong>C</strong>) Distance distributions for Nap1 E209R1 (left), Nap1 T251R1 (middle) and Nap1 T307R1 (right) in 500 mM NaCl (red) and 150 mM NaCl (blue). Corrected dipolar evolution (<strong>D</strong>) and distance distributions (<strong>E</strong>) for Nap1 T302R1. (<strong>F</strong>)&#x000a0;If the &#x003b2;-hairpin extensions did not rearrange upon tetramerisation as indicated schematically in the top panels a long and short distance would be anticipated. Reconfiguration of the &#x003b2;-hairpin's as indicated in the bottom panels would provide a closer fit to the observed measurement's. However, distances obtained from such a complex multi spin system are not sufficiently robust to distinguish between these.</p></div></div></div><p>As expected, upon tetramerisation short AB distances across the tetramerisation interface were observed in both Nap1 E209R1 and T307R1 distance distributions (Figure <a href="/pmc/articles/PMC4027167/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure8C).</span></span><span>8C</span></a>). This is also apparent from the initial gradient of the dipolar oscillation becoming steeper at 150 mM compared to 500 mM sodium chloride (Figure <a href="/pmc/articles/PMC4027167/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure8B).</span></span><span>8B</span></a>). However in the case of Nap1 T251R1 the initial gradient in the dipolar oscillation (Figure <a href="/pmc/articles/PMC4027167/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure8B)</span></span><span>8B</span></a>) is indistinguishable at both 150 mM and 500 mM sodium chloride. This implies that the shortest distance in the Nap1 T251R1 tetramer is identical to the AB distance in the Nap1 T251R1 dimer and is consistent with the observation of two longer distances in the Nap1 T251R1 150 mM distance distribution, which were assigned as AC and AD (Figure <a href="/pmc/articles/PMC4027167/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure8C</span></span><span>8C</span></a>).</p><p>The globular domains of Nap1 are highly acidic and do not possess the charge complementarity observed in the Vps75 globular domains. As the &#x003b2;-hairpin extensions are highly basic and are required for tetramerisation, we investigated their conformation by introducing a labelling site at position T302. Consistent with the crystal structures of Nap1 dimers (<a href="#B17" rid="B17" class=" bibr popnode">17</a>), these sites are separated by 10.2 nm at 500 mM NaCl (Figure <a href="/pmc/articles/PMC4027167/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure8D</span></span><span>8D</span></a> and <a href="/pmc/articles/PMC4027167/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">andE).</span></span><span>E</span></a>). To our knowledge this represents the longest distance measured using this approach.</p><p>At 150 mM NaCl a drop in oscillation depth was observed consistent with an increase in the number of interacting spins. Regularisation indicated the presence of at least three distance distributions (Figure <a href="/pmc/articles/PMC4027167/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure8E).</span></span><span>8E</span></a>). There are several conformations the &#x003b2;-hairpin extensions could adopt to provide such a mixture of short and long distances (Figure <a href="/pmc/articles/PMC4027167/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure8F).</span></span><span>8F</span></a>). While the measurements obtained support a rearrangement of the &#x003b2;-hairpins upon tetramerisation, they are not sufficiently robust to define the details of this reconfiguration.</p><p class="p p-last">In concert these observations are consistent with Nap1 tetramers having an arrangement in which the headphone fold domains are brought into close proximity. As a result it can be proposed that Nap1 like Vps75 adopts a tetrameric configuration at low salt concentrations, making it possible that the formation of tetrameric ring-like structures is a common feature of NAP-1 fold proteins.</p></div></div><div id="SEC4" class="tsec sec"><h2 class="head no_bottom_margin" id="SEC4title" style="text-transform: uppercase;">DISCUSSION</h2><p class="p p-first">The structure and function of Vps75 has been studied using a number of biochemical and biophysical techniques (<a href="#B6" rid="B6" class=" bibr popnode">6&#x02013;8</a>,<a href="#B10" rid="B10" class=" bibr popnode">10</a>,<a href="#B15" rid="B15" class=" bibr popnode">15</a>,<a href="#B16" rid="B16" class=" bibr popnode">16</a>). Such studies have revealed the homo-dimeric nature of Vps75, but did not report the presence of tetramers. It is likely that the sensitivity of tetramerisation to ionic conditions contributes to this difference. The structural paralog of Vps75 in yeast is Nap1. Interestingly, Nap1 dimers have also been shown to self associate into higher MW complexes (<a href="#B26" rid="B26" class=" bibr popnode">26</a>,<a href="#B27" rid="B27" class=" bibr popnode">27</a>,<a href="#B41" rid="B41" class=" bibr popnode">41</a>), as have human Nap1 variants (<a href="#B42" rid="B42" class=" bibr popnode">42</a>). Nap1 has been reported to form a number of higher order states ranging from tetramers to hexadecamers (<a href="#B8" rid="B8" class=" bibr popnode">8</a>,<a href="#B26" rid="B26" class=" bibr popnode">26&#x02013;28</a>). These self-association reactions are also sensitive to ionic conditions, and our own observations suggest that under conditions that are similar to those occurring within cells, the predominant form of Nap1 is that of a tetramer. It is unclear to us why these studies do not identify a dominant tetrameric species at moderate ionic conditions. One possibility is that over the longer time courses required for analytical ultracentrifugation oxidation of cysteine residues may have stabilized larger assemblies.</p><p>In a cellular context, a proportion of Vps75 is found stably associated with the histone acetyltransferase Rtt109. Interestingly, Rtt109 has been observed to form ring-shaped complexes with Vps75 (<a href="#B10" rid="B10" class=" bibr popnode">10</a>,<a href="#B18" rid="B18" class=" bibr popnode">18</a>,<a href="#B19" rid="B19" class=" bibr popnode">19</a>). However, there appears to be some plasticity in the interface between Vps75 and Rtt109 as both 2:1 and 2:2 Vps75:Rtt109 complexes have been reported. Furthermore, at first sight both complexes share a related ring-like organisation. In both cases the head-phone fold domains involved in Vps75 tetramerisation provide the Rtt109 interaction surfaces. Nonetheless, unlike tetramerisation, the association of Rtt109 with Vps75 does not display sensitivity to high ionic conditions (data not shown) (<a href="#B19" rid="B19" class=" bibr popnode">19</a>). This is most likely due to an additional, predominantly hydrophobic, contact between a loop region of Rtt109 and the globular fold of Vps75 (<a href="#B18" rid="B18" class=" bibr popnode">18</a>,<a href="#B19" rid="B19" class=" bibr popnode">19</a>). These additional contacts likely confer a higher affinity for Rtt109 in comparison to homotetramerisation consistent with our finding that Rtt109 can disrupt Vps75 tetramers (Figure <a href="/pmc/articles/PMC4027167/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6).</span></span><span>6</span></a>). In addition, structure guided mutatgenesis indicates specificity in residues involved in the formation of homo and hetero oligomeric structures. For example, the double KK169EE mutation does not affect Rtt109 binding to Vps75 (<a href="#B19" rid="B19" class=" bibr popnode">19</a>), but the single mutation K169E strongly reduced tetramer formation (Figure <a href="/pmc/articles/PMC4027167/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5A).</span></span><span>5A</span></a>). This provides scope for the quaternary interactions of Vps75 to be regulated independently and raises the question as to what function homotetramerisation might fulfil.</p><p>Tetramerisation of Vps75 restricts access to the central cavity that includes highly charged residues likely to be involved in histone binding. Just as positively charged histones require chaperones prior to incorporation into chromatin, the acidic surfaces of the chaperones themselves may require shielding when not occupied by their cargo as they may be prone to non-specific association with basic proteins. Consistent with this concept, the majority of Vps75 purified from yeast lacks an interaction partner (<a href="#B6" rid="B6" class=" bibr popnode">6</a>). In addition, while Vps75 protein levels remain constant through the cell cycle, both histones and Rtt109 are tightly regulated with a peak just prior to S-phase (<a href="#B43" rid="B43" class=" bibr popnode">43</a>). As a result following the completion of DNA replication, the majority of Vps75 is not bound by either Rtt109 or histones. In this situation, it is attractive to speculate that tetramerisation may act to restrict access to the acidic cavity formed at the interface between Vps75 dimers.</p><p>Our attempts to characterize the interaction of histones with Vps75 have been hindered by protein aggregation. As a result it is not clear whether histone binding dissociates Vps75 tetramers or causes them to adopt an altered configuration. However, it is notable that when Nap1 is bound by histones, megadalton-sized complexes have been observed. Cryo-Electron microscopy of these assemblies reveals that they are made up of smaller particles with similar dimensions to the Vps75 tetramers we have characterized (<a href="#B28" rid="B28" class=" bibr popnode">28</a>). We note that while a recent study indicates that Nap1 can bind histones H2A and H2B in a novel tetrameric conformation, this study does not distinguish whether Nap1 is present in the form of dimers or other oligomeric assemblies (<a href="#B44" rid="B44" class=" bibr popnode">44</a>).</p><p class="p p-last">Structural information regarding the conformation of histone chaperones is crucial in deciphering the molecular mechanisms of nucleosome assembly and disassembly. In this study, we demonstrate, using a number of in-solution methods, that the histone chaperones Vps75 and Nap1 adopt a homotetrameric structure. We propose that this may provide a self-chaperoning function that serves to reduce non-specific interactions with the acidic surfaces that are encapsulated by the ring. Regulation of the oligomerisation of NAP-1 fold chaperones also potentially provides a means of regulating interactions with cargo proteins.</p></div><div id="SEC5" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="SEC5title" style="text-transform: uppercase;">SUPPLEMENTARY DATA</h2><!--/article/body/sec/--><p class="p p-first"><a href="http://nar.oxfordjournals.org/lookup/suppl/doi:10.1093/nar/gku232/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Data</a> are available at NAR Online.</p><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>SUPPLEMENTARY DATA:</strong>

<div class="sup-box half_rhythm" id="media-a.m.e.c.b"><a href="/pmc/articles/PMC4027167/bin/supp_42_9_6038__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>

</div></div><div id="ack-a.n.a" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.n.atitle" style="text-transform: uppercase;">Acknowledgments</h2><div class="sec"><p>We would like to thank Govind Malwad for assistance in the mutagenesis of Vps75 and members of the DGN and TOH labs for valuable discussion.</p></div></div><div id="ack-a.n.b" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.n.btitle" style="text-transform: uppercase;">FUNDING</h2><div class="sec"><p>MRC [G1100021]; Wellcome Senior Fellowship [095062]. Source of open access funding: The Wellcome Trust [094090].</p><p><em>Conflict of interest statement</em>. None declared.</p></div></div><div id="fn-group-a.n.c" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.n.ctitle" style="text-transform: uppercase;">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="fn1a"><p class="p p-first">Present addresses:</p><p class="p">Andrew Bowman, Department of Physiological Chemistry, Adolf-Butenandt-Institute, Butenandtstrasse 5, 81377 Munich, Germany.</p><p class="p">Weifeng Shang, Centre for Radiation Research and Instrumentation, Department of Biological, Chemical and Physical Sciences, Illinois Institute of Technology, 3101 S. Dearborn, Chicago, IL 60616, USA.</p><p class="p p-last">The authors wish it to be known that, in their opinion, the first two authors should be regarded as Joint First Authors.</p></p></div></div><div id="ref-list-a.n.d" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.n.dtitle" style="text-transform: uppercase;">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1">1. <span class="element-citation">Laskey R.A., Honda B.M., Mills A.D., Finch J.T. Nucleosomes are assembled by an acidic protein which binds histones and transfers them to DNA. <span><span class="ref-journal">Nature (London) </span>1978;<span class="ref-vol">275</span>:416–420.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/692721" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+(London)&amp;title=Nucleosomes+are+assembled+by+an+acidic+protein+which+binds+histones+and+transfers+them+to+DNA&amp;author=R.A.+Laskey&amp;author=B.M.+Honda&amp;author=A.D.+Mills&amp;author=J.T.+Finch&amp;volume=275&amp;publication_year=1978&amp;pages=416-420&amp;pmid=692721&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B2">2. <span class="element-citation">Hansen J.C., Nyborg J.K., Luger K., Stargell L.A. Histone chaperones, histone acetylation, and the fluidity of the chromogenome. <span><span class="ref-journal">J. Cell. Physiol. </span>2010;<span class="ref-vol">224</span>:289–299.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3184832/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20432449" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Cell.+Physiol.&amp;title=Histone+chaperones,+histone+acetylation,+and+the+fluidity+of+the+chromogenome&amp;author=J.C.+Hansen&amp;author=J.K.+Nyborg&amp;author=K.+Luger&amp;author=L.A.+Stargell&amp;volume=224&amp;publication_year=2010&amp;pages=289-299&amp;pmid=20432449&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B3">3. <span class="element-citation">Ransom M., Dennehey B.K., Tyler J.K. Chaperoning histones during DNA replication and repair. <span><span class="ref-journal">Cell. </span>2010;<span class="ref-vol">140</span>:183–195.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3433953/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20141833" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Chaperoning+histones+during+DNA+replication+and+repair&amp;author=M.+Ransom&amp;author=B.K.+Dennehey&amp;author=J.K.+Tyler&amp;volume=140&amp;publication_year=2010&amp;pages=183-195&amp;pmid=20141833&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B4">4. <span class="element-citation">Das C., Tyler J.K., Churchill M.E.A. The histone shuffle: histone chaperones in an energetic dance. <span><span class="ref-journal">Trends Biochem. Sci. </span>2010;<span class="ref-vol">35</span>:476–489.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4004086/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20444609" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Biochem.+Sci.&amp;title=The+histone+shuffle:+histone+chaperones+in+an+energetic+dance&amp;author=C.+Das&amp;author=J.K.+Tyler&amp;author=M.E.A.+Churchill&amp;volume=35&amp;publication_year=2010&amp;pages=476-489&amp;pmid=20444609&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B5">5. <span class="element-citation">Bonangelino C.J., Chavez E.M., Bonifacino J.S. Genomic screen for vacuolar protein sorting genes in Saccharomyces cerevisiae. <span><span class="ref-journal">Mol. Biol. Cell. </span>2002;<span class="ref-vol">13</span>:2486–2501.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC117329/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12134085" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Biol.+Cell&amp;title=Genomic+screen+for+vacuolar+protein+sorting+genes+in+Saccharomyces+cerevisiae&amp;author=C.J.+Bonangelino&amp;author=E.M.+Chavez&amp;author=J.S.+Bonifacino&amp;volume=13&amp;publication_year=2002&amp;pages=2486-2501&amp;pmid=12134085&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B6">6. <span class="element-citation">Selth L., Svejstrup J.Q. Vps75, a new yeast member of the NAP histone chaperone. <span><span class="ref-journal">J. Biol. Chem. </span>2007;<span class="ref-vol">282</span>:12358–12362.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17344218" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Vps75,+a+new+yeast+member+of+the+NAP+histone+chaperone&amp;author=L.+Selth&amp;author=J.Q.+Svejstrup&amp;volume=282&amp;publication_year=2007&amp;pages=12358-12362&amp;pmid=17344218&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B7">7. <span class="element-citation">Tsubota T., Berndsen C.E., Erkmann J.A., Smith C.L., Yang L.H., Freitas M.A., Denu J.M., Kaufman P.D. Histone H3-K56 acetylation is catalyzed by histone chaperone-dependent complexes. <span><span class="ref-journal">Mol. Cell. </span>2007;<span class="ref-vol">25</span>:703–712.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1853276/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17320445" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=Histone+H3-K56+acetylation+is+catalyzed+by+histone+chaperone-dependent+complexes&amp;author=T.+Tsubota&amp;author=C.E.+Berndsen&amp;author=J.A.+Erkmann&amp;author=C.L.+Smith&amp;author=L.H.+Yang&amp;volume=25&amp;publication_year=2007&amp;pages=703-712&amp;pmid=17320445&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B8">8. <span class="element-citation">Park Y.J., Sudhoff K.B., Andrews A.J., Stargell L.A., Luger K. Histone chaperone specificity in Rtt109 activation. <span><span class="ref-journal">Nat. Struct. Mol. Biol. </span>2008;<span class="ref-vol">15</span>:957–964.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2680711/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19172749" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Struct.+Mol.+Biol.&amp;title=Histone+chaperone+specificity+in+Rtt109+activation&amp;author=Y.J.+Park&amp;author=K.B.+Sudhoff&amp;author=A.J.+Andrews&amp;author=L.A.+Stargell&amp;author=K.+Luger&amp;volume=15&amp;publication_year=2008&amp;pages=957-964&amp;pmid=19172749&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B9">9. <span class="element-citation">Krogan N.J., Cagney G., Yu H.Y., Zhong G.Q., Guo X.H., Ignatchenko A., Li J., Pu S.Y., Datta N., Tikuisis A.P., et al. Global landscape of protein complexes in the yeast Saccharomyces cerevisiae. <span><span class="ref-journal">Nature. </span>2006;<span class="ref-vol">440</span>:637–643.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16554755" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Global+landscape+of+protein+complexes+in+the+yeast+Saccharomyces+cerevisiae&amp;author=N.J.+Krogan&amp;author=G.+Cagney&amp;author=H.Y.+Yu&amp;author=G.Q.+Zhong&amp;author=X.H.+Guo&amp;volume=440&amp;publication_year=2006&amp;pages=637-643&amp;pmid=16554755&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B10">10. <span class="element-citation">Kolonko E.M., Albaugh B.N., Lindner S.E., Chen Y., Satyshur K.A., Arnold K.M., Kaufman P.D., Keck J.L., Denu J.M. Catalytic activation of histone acetyltransferase Rtt109 by a histone chaperone. <span><span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A. </span>2010;<span class="ref-vol">107</span>:20275–20280.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2996700/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21057107" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Catalytic+activation+of+histone+acetyltransferase+Rtt109+by+a+histone+chaperone&amp;author=E.M.+Kolonko&amp;author=B.N.+Albaugh&amp;author=S.E.+Lindner&amp;author=Y.+Chen&amp;author=K.A.+Satyshur&amp;volume=107&amp;publication_year=2010&amp;pages=20275-20280&amp;pmid=21057107&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B11">11. <span class="element-citation">Fillingham J., Recht J., Silva A.C., Suter B., Emili A., Stagljar I., Krogan N.J., Allis C.D., Keogh M.C., Greenblatt J.F. Chaperone control of the activity and specificity of the histone H3 acetyltransferase Rtt109. <span><span class="ref-journal">Mol. Cell. Biol. </span>2008;<span class="ref-vol">28</span>:4342–4353.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2447148/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18458063" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Biol.&amp;title=Chaperone+control+of+the+activity+and+specificity+of+the+histone+H3+acetyltransferase+Rtt109&amp;author=J.+Fillingham&amp;author=J.+Recht&amp;author=A.C.+Silva&amp;author=B.+Suter&amp;author=A.+Emili&amp;volume=28&amp;publication_year=2008&amp;pages=4342-4353&amp;pmid=18458063&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B12">12. <span class="element-citation">Selth L.A., Lorch Y., Ocampo-Hafalla M.T., Mitter R., Shales M., Krogan N.J., Kornberg R.D., Svejstrup J.Q. An rtt109-independent role for vps75 in transcription-associated nucleosome dynamics. <span><span class="ref-journal">Mol. Cell. Biol. </span>2009;<span class="ref-vol">29</span>:4220–4234.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2715805/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19470761" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Biol.&amp;title=An+rtt109-independent+role+for+vps75+in+transcription-associated+nucleosome+dynamics&amp;author=L.A.+Selth&amp;author=Y.+Lorch&amp;author=M.T.+Ocampo-Hafalla&amp;author=R.+Mitter&amp;author=M.+Shales&amp;volume=29&amp;publication_year=2009&amp;pages=4220-4234&amp;pmid=19470761&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B13">13. <span class="element-citation">Xue Y.M., Kowalska A.K., Grabowska K., Przybyt K., Cichewicz M.A., Del Rosario B.C., Pemberton L.F. Histone chaperones Nap1 and Vps75 regulate histone acetylation during transcription elongation. <span><span class="ref-journal">Mol. Cell. Biol. </span>2013;<span class="ref-vol">33</span>:1645–1656.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3624251/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23401858" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Biol.&amp;title=Histone+chaperones+Nap1+and+Vps75+regulate+histone+acetylation+during+transcription+elongation&amp;author=Y.M.+Xue&amp;author=A.K.+Kowalska&amp;author=K.+Grabowska&amp;author=K.+Przybyt&amp;author=M.A.+Cichewicz&amp;volume=33&amp;publication_year=2013&amp;pages=1645-1656&amp;pmid=23401858&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B14">14. <span class="element-citation">Pramila T., Wu W., Miles S., Noble W.S., Breeden L.L. The Forkhead transcription factor Hcm1 regulates chromosome segregation genes and fills the S-phase gap in the transcriptional circuitry of the cell cycle. <span><span class="ref-journal">Genes Dev. </span>2006;<span class="ref-vol">20</span>:2266–2278.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1553209/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16912276" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev.&amp;title=The+Forkhead+transcription+factor+Hcm1+regulates+chromosome+segregation+genes+and+fills+the+S-phase+gap+in+the+transcriptional+circuitry+of+the+cell+cycle&amp;author=T.+Pramila&amp;author=W.+Wu&amp;author=S.+Miles&amp;author=W.S.+Noble&amp;author=L.L.+Breeden&amp;volume=20&amp;publication_year=2006&amp;pages=2266-2278&amp;pmid=16912276&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B15">15. <span class="element-citation">Berndsen C.E., Tsubota T., Lindner S.E., Lee S., Holton J.M., Kaufman P.D., Keck J.L., Denu J.M. Molecular functions of the histone acetyltransferase chaperone complex Rtt109-Vps75. <span><span class="ref-journal">Nat. Struct. Mol. Biol. </span>2008;<span class="ref-vol">15</span>:948–956.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2678805/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19172748" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Struct.+Mol.+Biol.&amp;title=Molecular+functions+of+the+histone+acetyltransferase+chaperone+complex+Rtt109-Vps75&amp;author=C.E.+Berndsen&amp;author=T.+Tsubota&amp;author=S.E.+Lindner&amp;author=S.+Lee&amp;author=J.M.+Holton&amp;volume=15&amp;publication_year=2008&amp;pages=948-956&amp;pmid=19172748&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B16">16. <span class="element-citation">Tang Y., Meeth K., Jiang E., Luo C., Marmorstein R. Structure of Vps75 and implications for histone chaperone function. <span><span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A. </span>2008;<span class="ref-vol">105</span>:12206–12211.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2527890/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18723682" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Structure+of+Vps75+and+implications+for+histone+chaperone+function&amp;author=Y.+Tang&amp;author=K.+Meeth&amp;author=E.+Jiang&amp;author=C.+Luo&amp;author=R.+Marmorstein&amp;volume=105&amp;publication_year=2008&amp;pages=12206-12211&amp;pmid=18723682&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B17">17. <span class="element-citation">Park Y.J., Luger K. The structure of nucleosome assembly protein 1. <span><span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A. </span>2006;<span class="ref-vol">103</span>:1248–1253.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1345705/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16432217" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=The+structure+of+nucleosome+assembly+protein+1&amp;author=Y.J.+Park&amp;author=K.+Luger&amp;volume=103&amp;publication_year=2006&amp;pages=1248-1253&amp;pmid=16432217&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B18">18. <span class="element-citation">Su D., Hu Q., Zhou H., Thompson J.R., Xu R.M., Zhang Z., Mer G. Structure and histone binding properties of the Vps75-Rtt109 chaperone-lysine acetyltransferase complex. <span><span class="ref-journal">J. Biol. Chem. </span>2011;<span class="ref-vol">286</span>:15625–15629.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3091171/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21454705" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Structure+and+histone+binding+properties+of+the+Vps75-Rtt109+chaperone-lysine+acetyltransferase+complex&amp;author=D.+Su&amp;author=Q.+Hu&amp;author=H.+Zhou&amp;author=J.R.+Thompson&amp;author=R.M.+Xu&amp;volume=286&amp;publication_year=2011&amp;pages=15625-15629&amp;pmid=21454705&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B19">19. <span class="element-citation">Tang Y., Holbert M.A., Delgoshaie N., Wurtele H., Guillemette B., Meeth K., Yuan H., Drogaris P., Lee E.H., Durette C., et al. Structure of the Rtt109-AcCoA/Vps75 complex and implications for chaperone-mediated histone acetylation. <span><span class="ref-journal">Structure. </span>2011;<span class="ref-vol">19</span>:221–231.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3050538/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21256037" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=Structure+of+the+Rtt109-AcCoA/Vps75+complex+and+implications+for+chaperone-mediated+histone+acetylation&amp;author=Y.+Tang&amp;author=M.A.+Holbert&amp;author=N.+Delgoshaie&amp;author=H.+Wurtele&amp;author=B.+Guillemette&amp;volume=19&amp;publication_year=2011&amp;pages=221-231&amp;pmid=21256037&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B20">20. <span class="element-citation">D&#x02019;Arcy S., Luger K. Understanding histone acetyltransferase Rtt109 structure and function: how many chaperones does it take. <span><span class="ref-journal">Curr. Opin. Struct. Biol. </span>2011;<span class="ref-vol">21</span>:728–734.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3232334/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22023828" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Struct.+Biol.&amp;title=Understanding+histone+acetyltransferase+Rtt109+structure+and+function:+how+many+chaperones+does+it+take&amp;author=S.+D&#x02019;Arcy&amp;author=K.+Luger&amp;volume=21&amp;publication_year=2011&amp;pages=728-734&amp;pmid=22023828&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B21">21. <span class="element-citation">Ishimi Y., Kikuchi A. Identification and molecular cloning of yeast homolog of nucleosome assembly protein I which facilitates nucleosome assembly in vitro. <span><span class="ref-journal">J. Biol. Chem. </span>1991;<span class="ref-vol">266</span>:7025–7029.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2016313" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Identification+and+molecular+cloning+of+yeast+homolog+of+nucleosome+assembly+protein+I+which+facilitates+nucleosome+assembly+in+vitro&amp;author=Y.+Ishimi&amp;author=A.+Kikuchi&amp;volume=266&amp;publication_year=1991&amp;pages=7025-7029&amp;pmid=2016313&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B22">22. <span class="element-citation">Andrews A.J., Downing G., Brown K., Park Y.J., Luger K. A thermodynamic model for Nap1-histone interactions. <span><span class="ref-journal">J. Biol. Chem. </span>2008;<span class="ref-vol">283</span>:32412–32418.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2583301/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18728017" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=A+thermodynamic+model+for+Nap1-histone+interactions&amp;author=A.J.+Andrews&amp;author=G.+Downing&amp;author=K.+Brown&amp;author=Y.J.+Park&amp;author=K.+Luger&amp;volume=283&amp;publication_year=2008&amp;pages=32412-32418&amp;pmid=18728017&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B23">23. <span class="element-citation">Del Rosario B.C., Pemberton L.F. Nap1 links transcription elongation, chromatin assembly, and messenger RNP complex biogenesis. <span><span class="ref-journal">Mol. Cell. Biol. </span>2008;<span class="ref-vol">28</span>:2113–2124.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2268417/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18227150" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Biol.&amp;title=Nap1+links+transcription+elongation,+chromatin+assembly,+and+messenger+RNP+complex+biogenesis&amp;author=B.C.+Del+Rosario&amp;author=L.F.+Pemberton&amp;volume=28&amp;publication_year=2008&amp;pages=2113-2124&amp;pmid=18227150&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B24">24. <span class="element-citation">Ohkuni K., Shirahige K., Kikuchi A. Genome-wide expression analysis of NAP1 in Saccharomyces cerevisiae. <span><span class="ref-journal">Biochem. Biophys. Res. Commun. </span>2003;<span class="ref-vol">306</span>:5–9.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12788058" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.+Biophys.+Res.+Commun.&amp;title=Genome-wide+expression+analysis+of+NAP1+in+Saccharomyces+cerevisiae&amp;author=K.+Ohkuni&amp;author=K.+Shirahige&amp;author=A.+Kikuchi&amp;volume=306&amp;publication_year=2003&amp;pages=5-9&amp;pmid=12788058&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B25">25. <span class="element-citation">Kuryan B.G., Kim J., Tran N.N.H., Lombardo S.R., Venkatesh S., Workman J.L., Carey M. Histone density is maintained during transcription mediated by the chromatin remodeler RSC and histone chaperone NAP1 in vitro. <span><span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A. </span>2012;<span class="ref-vol">109</span>:1931–1936.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3277555/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22308335" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Histone+density+is+maintained+during+transcription+mediated+by+the+chromatin+remodeler+RSC+and+histone+chaperone+NAP1+in+vitro&amp;author=B.G.+Kuryan&amp;author=J.+Kim&amp;author=N.N.H.+Tran&amp;author=S.R.+Lombardo&amp;author=S.+Venkatesh&amp;volume=109&amp;publication_year=2012&amp;pages=1931-1936&amp;pmid=22308335&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B26">26. <span class="element-citation">McBryant S.J., Peersen O.B. Self-association of the yeast nucleosome assembly protein 1. <span><span class="ref-journal">Biochemistry. </span>2004;<span class="ref-vol">43</span>:10592–10599.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15301556" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Self-association+of+the+yeast+nucleosome+assembly+protein+1&amp;author=S.J.+McBryant&amp;author=O.B.+Peersen&amp;volume=43&amp;publication_year=2004&amp;pages=10592-10599&amp;pmid=15301556&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B27">27. <span class="element-citation">Toth K.F., Mazurkiewicz J., Rippe K. Association states of nucleosome assembly protein 1 and its complexes with histones. <span><span class="ref-journal">J. Biol. Chem. </span>2005;<span class="ref-vol">280</span>:15690–15699.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15687486" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Association+states+of+nucleosome+assembly+protein+1+and+its+complexes+with+histones&amp;author=K.F.+Toth&amp;author=J.+Mazurkiewicz&amp;author=K.+Rippe&amp;volume=280&amp;publication_year=2005&amp;pages=15690-15699&amp;pmid=15687486&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B28">28. <span class="element-citation">Newman E.R., Kneale G.G., Ravelli R.B.G., Karuppasamy M., Nejadasl F.K., Taylor I.A., McGeehan J.E. Large multimeric assemblies of nucleosome assembly protein and histones revealed by small-angle X-ray scattering and electron microscopy. <span><span class="ref-journal">J. Biol. Chem. </span>2012;<span class="ref-vol">287</span>:26657–26665.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3411005/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22707715" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Large+multimeric+assemblies+of+nucleosome+assembly+protein+and+histones+revealed+by+small-angle+X-ray+scattering+and+electron+microscopy&amp;author=E.R.+Newman&amp;author=G.G.+Kneale&amp;author=R.B.G.+Ravelli&amp;author=M.+Karuppasamy&amp;author=F.K.+Nejadasl&amp;volume=287&amp;publication_year=2012&amp;pages=26657-26665&amp;pmid=22707715&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B29">29. <span class="element-citation">Bowman A., Ward R., Wiechens N., Singh V., El-Mkami H., Norman D.G., Owen-Hughes T. The histone chaperones Nap1 and Vps75 bind histones H3 and H4 in a tetrameric conformation. <span><span class="ref-journal">Mol. Cell. </span>2011;<span class="ref-vol">41</span>:398–408.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3093613/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21329878" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=The+histone+chaperones+Nap1+and+Vps75+bind+histones+H3+and+H4+in+a+tetrameric+conformation&amp;author=A.+Bowman&amp;author=R.+Ward&amp;author=N.+Wiechens&amp;author=V.+Singh&amp;author=H.+El-Mkami&amp;volume=41&amp;publication_year=2011&amp;pages=398-408&amp;pmid=21329878&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B30">30. <span class="element-citation">Jeschke G., Polyhach Y. Distance measurements on spin-labelled biomacromolecules by pulsed electron paramagnetic resonance. <span><span class="ref-journal">Phys. Chem. Chem. Phys. </span>2007;<span class="ref-vol">9</span>:1895–1910.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17431518" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Phys.+Chem.+Chem.+Phys.&amp;title=Distance+measurements+on+spin-labelled+biomacromolecules+by+pulsed+electron+paramagnetic+resonance&amp;author=G.+Jeschke&amp;author=Y.+Polyhach&amp;volume=9&amp;publication_year=2007&amp;pages=1895-1910&amp;pmid=17431518&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B31">31. <span class="element-citation">Schwieters C.D., Kuszewski J.J., Tjandra N., Clore G.M. The Xplor-NIH NMR molecular structure determination package. <span><span class="ref-journal">J. Magn. Reson. </span>2003;<span class="ref-vol">160</span>:65–73.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12565051" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Magn.+Reson.&amp;title=The+Xplor-NIH+NMR+molecular+structure+determination+package&amp;author=C.D.+Schwieters&amp;author=J.J.+Kuszewski&amp;author=N.+Tjandra&amp;author=G.M.+Clore&amp;volume=160&amp;publication_year=2003&amp;pages=65-73&amp;pmid=12565051&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B32">32. <span class="element-citation">Hagelueken G., Ward R., Naismith J.H., Schiemann O. MtsslWizard: In silico spin-labeling and generation of distance distributions in PyMOL. <span><span class="ref-journal">Appl. Magn. Reson. </span>2012;<span class="ref-vol">42</span>:377–391.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3296949/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22448103" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Appl.+Magn.+Reson.&amp;title=MtsslWizard:+In+silico+spin-labeling+and+generation+of+distance+distributions+in+PyMOL&amp;author=G.+Hagelueken&amp;author=R.+Ward&amp;author=J.H.+Naismith&amp;author=O.+Schiemann&amp;volume=42&amp;publication_year=2012&amp;pages=377-391&amp;pmid=22448103&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B33">33. <span class="element-citation">Roessle M.W., Klaering R., Ristau U., Robrahn B., Jahn D., Gehrmann T., Konarev P., Round A., Fiedler S., Hermes C., et al. Upgrade of the small-angle X-ray scattering beamline X33 at the European Molecular Biology Laboratory, Hamburg. <span><span class="ref-journal">J. Appl. Crystallogr. </span>2007;<span class="ref-vol">40</span>:S190–S194.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Appl.+Crystallogr.&amp;title=Upgrade+of+the+small-angle+X-ray+scattering+beamline+X33+at+the+European+Molecular+Biology+Laboratory,+Hamburg&amp;author=M.W.+Roessle&amp;author=R.+Klaering&amp;author=U.+Ristau&amp;author=B.+Robrahn&amp;author=D.+Jahn&amp;volume=40&amp;publication_year=2007&amp;pages=S190-S194&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B34">34. <span class="element-citation">Konarev P.V., Volkov V.V., Sokolova A.V., Koch M.H.J., Svergun D.I. PRIMUS: a Windows PC-based system for small-angle scattering data analysis. <span><span class="ref-journal">J. Appl. Crystallogr. </span>2003;<span class="ref-vol">36</span>:1277–1282.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Appl.+Crystallogr.&amp;title=PRIMUS:+a+Windows+PC-based+system+for+small-angle+scattering+data+analysis&amp;author=P.V.+Konarev&amp;author=V.V.+Volkov&amp;author=A.V.+Sokolova&amp;author=M.H.J.+Koch&amp;author=D.I.+Svergun&amp;volume=36&amp;publication_year=2003&amp;pages=1277-1282&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B35">35. <span class="element-citation">Svergun D.I. Determination of the regularization parameter in indirect-transform methods using perceptual criteria. <span><span class="ref-journal">J. Appl. Crystallogr. </span>1992;<span class="ref-vol">25</span>:495–503.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Appl.+Crystallogr.&amp;title=Determination+of+the+regularization+parameter+in+indirect-transform+methods+using+perceptual+criteria&amp;author=D.I.+Svergun&amp;volume=25&amp;publication_year=1992&amp;pages=495-503&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B36">36. <span class="element-citation">Svergun D., Barberato C., Koch M.H.J. CRYSOL - A program to evaluate x-ray solution scattering of biological macromolecules from atomic coordinates. <span><span class="ref-journal">J. Appl. Crystallogr. </span>1995;<span class="ref-vol">28</span>:768–773.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Appl.+Crystallogr.&amp;title=CRYSOL+-+A+program+to+evaluate+x-ray+solution+scattering+of+biological+macromolecules+from+atomic+coordinates&amp;author=D.+Svergun&amp;author=C.+Barberato&amp;author=M.H.J.+Koch&amp;volume=28&amp;publication_year=1995&amp;pages=768-773&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B37">37. <span class="element-citation">Petoukhov M.V., Svergun D.I. Global rigid body modeling of macromolecular complexes against small-angle scattering data. <span><span class="ref-journal">Biophys. J. </span>2005;<span class="ref-vol">89</span>:1237–1250.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1366608/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15923225" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biophys.+J.&amp;title=Global+rigid+body+modeling+of+macromolecular+complexes+against+small-angle+scattering+data&amp;author=M.V.+Petoukhov&amp;author=D.I.+Svergun&amp;volume=89&amp;publication_year=2005&amp;pages=1237-1250&amp;pmid=15923225&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B38">38. <span class="element-citation">Fleissner M.R., Bridges M.D., Brooks E.K., Cascio D., Kalai T., Hideg K., Hubbell W.L. Structure and dynamics of a conformationally constrained nitroxide side chain and applications in EPR spectroscopy. <span><span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A. </span>2011;<span class="ref-vol">108</span>:16241–16246.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3182725/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21911399" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Structure+and+dynamics+of+a+conformationally+constrained+nitroxide+side+chain+and+applications+in+EPR+spectroscopy&amp;author=M.R.+Fleissner&amp;author=M.D.+Bridges&amp;author=E.K.+Brooks&amp;author=D.+Cascio&amp;author=T.+Kalai&amp;volume=108&amp;publication_year=2011&amp;pages=16241-16246&amp;pmid=21911399&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B39">39. <span class="element-citation">Ward R., Bowman A., Sozudogru E., El-Mkami H., Owen-Hughes T., Norman D.G. EPR distance measurements in deuterated proteins. <span><span class="ref-journal">J. Magn. Reson. </span>2010;<span class="ref-vol">207</span>:164–167.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3516800/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20805036" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Magn.+Reson.&amp;title=EPR+distance+measurements+in+deuterated+proteins&amp;author=R.+Ward&amp;author=A.+Bowman&amp;author=E.+Sozudogru&amp;author=H.+El-Mkami&amp;author=T.+Owen-Hughes&amp;volume=207&amp;publication_year=2010&amp;pages=164-167&amp;pmid=20805036&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B40">40. <span class="element-citation">Milov A.D., Ponomarev A.B., Tsvetkov Y.D. Electron-electron double resonance in electron spin echo: Model biradical systems and the sensitized photolysis of decalin. <span><span class="ref-journal">Phys. Lett. </span>1984;<span class="ref-vol">110</span>:67–72.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Phys.+Lett.&amp;title=Electron-electron+double+resonance+in+electron+spin+echo:+Model+biradical+systems+and+the+sensitized+photolysis+of+decalin&amp;author=A.D.+Milov&amp;author=A.B.+Ponomarev&amp;author=Y.D.+Tsvetkov&amp;volume=110&amp;publication_year=1984&amp;pages=67-72&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B41">41. <span class="element-citation">McBryant S.J., Park Y.J., Abernathy S.M., Laybourn P.J., Nyborg J.K., Luger K. Preferential binding of the histone (H3-H4)2 tetramer by NAP1 is mediated by the amino-terminal histone tails. <span><span class="ref-journal">J. Biol. Chem. </span>2003;<span class="ref-vol">278</span>:44574–44583.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12928440" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Preferential+binding+of+the+histone+(H3-H4)2+tetramer+by+NAP1+is+mediated+by+the+amino-terminal+histone+tails&amp;author=S.J.+McBryant&amp;author=Y.J.+Park&amp;author=S.M.+Abernathy&amp;author=P.J.+Laybourn&amp;author=J.K.+Nyborg&amp;volume=278&amp;publication_year=2003&amp;pages=44574-44583&amp;pmid=12928440&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B42">42. <span class="element-citation">Noda M., Uchiyama S., McKay A.R., Morimoto A., Misawa S., Yoshida A., Shimahara H., Takinowaki H., Nakamura S., Kobayashi Y., et al. Assembly states of the nucleosome assembly protein 1 (NAP-1) revealed by sedimentation velocity and non-denaturing MS. <span><span class="ref-journal">Biochem. J. </span>2011;<span class="ref-vol">436</span>:101–112.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21348863" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.+J.&amp;title=Assembly+states+of+the+nucleosome+assembly+protein+1+(NAP-1)+revealed+by+sedimentation+velocity+and+non-denaturing+MS&amp;author=M.+Noda&amp;author=S.+Uchiyama&amp;author=A.R.+McKay&amp;author=A.+Morimoto&amp;author=S.+Misawa&amp;volume=436&amp;publication_year=2011&amp;pages=101-112&amp;pmid=21348863&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B43">43. <span class="element-citation">Driscoll R., Hudson A., Jackson S.P. Yeast Rtt109 promotes genome stability by acetylating histone H3 on lysine 56. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">315</span>:649–652.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3334813/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17272722" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Yeast+Rtt109+promotes+genome+stability+by+acetylating+histone+H3+on+lysine+56&amp;author=R.+Driscoll&amp;author=A.+Hudson&amp;author=S.P.+Jackson&amp;volume=315&amp;publication_year=2007&amp;pages=649-652&amp;pmid=17272722&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B44">44. <span class="element-citation">D&#x02019;Arcy S., Martin K.W., Panchenko T., Chen X., Bergeron S., Stargell L.A., Black B.E., Luger K. Chaperone Nap1 shields histone surfaces used in a nucleosome and can put H2A-H2B in an unconventional tetrameric form. <span><span class="ref-journal">Mol. Cell. </span>2013;<span class="ref-vol">51</span>:662–677.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3878309/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23973327" ref="reftype=pubmed&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=Chaperone+Nap1+shields+histone+surfaces+used+in+a+nucleosome+and+can+put+H2A-H2B+in+an+unconventional+tetrameric+form&amp;author=S.+D&#x02019;Arcy&amp;author=K.W.+Martin&amp;author=T.+Panchenko&amp;author=X.+Chen&amp;author=S.+Bergeron&amp;volume=51&amp;publication_year=2013&amp;pages=662-677&amp;pmid=23973327&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4027167&amp;issue-id=237979&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Nucleic Acids Research</span> are provided here courtesy of <strong>Oxford University Press</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC4027167/pdf/gku232.pdf" class="int-view">PDF (3.5M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/4027167/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/4027167/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="4027167" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4027167%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="2mIrofQJoUWQGulAwwaDYFWNSSwdPlRDpUOTBLeyeMRlUppHTCJTGW8oy2seZHkZ">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4027167%2F&amp;text=The%20histone%20chaperones%20Vps75%20and%20Nap1%20form%20ring-like%2C%20tetrameric%20structures%20in%20solution" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4027167%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4027167/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC4027167/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/24688059/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC4027167/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/24688059/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC4027167/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/4027167/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC4027167/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
