
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-manuscript.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Mass Spectrometric Characterization of an Acid-Labile Adduct Formed with 2-Amino-1-methyl-6-phenylimidazo[4,5-b]pyridine and Albumin in Humans - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="939BE807DB0F4BC500002A8B4F2B4B02.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="nihpa" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC5375106/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5375106/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Chemical research in toxicology" /><meta name="citation_title" content="Mass Spectrometric Characterization of an Acid-Labile Adduct Formed with 2-Amino-1-methyl-6-phenylimidazo[4,5-b]pyridine and Albumin in Humans" /><meta name="citation_author" content="Yi Wang" /><meta name="citation_author_institution" content="Masonic Cancer Center, University of Minnesota, 2231 6th Street, Minneapolis, MN 55455, USA" /><meta name="citation_author_institution" content="Department of Medicinal Chemistry, Cancer and Cardiovascular Research Building, University of Minnesota, 2231 6th Street, Minneapolis, MN 55455, USA" /><meta name="citation_author" content="Peter W. Villalta" /><meta name="citation_author_institution" content="Masonic Cancer Center, University of Minnesota, 2231 6th Street, Minneapolis, MN 55455, USA" /><meta name="citation_author" content="Lijuan Peng" /><meta name="citation_author_institution" content="School of Food Science and Engineering, Wuhan Polytechnic University, ChangQing Garden, Hankou, Wuhan 430023, P.R. China" /><meta name="citation_author" content="Karen Dingley" /><meta name="citation_author_institution" content="Biosciences and Biotechnology Division, Center for Accelerator Mass Spectrometry Lawrence Livermore National Laboratory, Livermore 94550, CA" /><meta name="citation_author" content="Michael A. Malfatti" /><meta name="citation_author_institution" content="Biosciences and Biotechnology Division, Center for Accelerator Mass Spectrometry Lawrence Livermore National Laboratory, Livermore 94550, CA" /><meta name="citation_author" content="K.W. Turteltaub" /><meta name="citation_author_institution" content="Biosciences and Biotechnology Division, Center for Accelerator Mass Spectrometry Lawrence Livermore National Laboratory, Livermore 94550, CA" /><meta name="citation_author" content="Robert J. Turesky" /><meta name="citation_author_institution" content="Masonic Cancer Center, University of Minnesota, 2231 6th Street, Minneapolis, MN 55455, USA" /><meta name="citation_author_institution" content="Department of Medicinal Chemistry, Cancer and Cardiovascular Research Building, University of Minnesota, 2231 6th Street, Minneapolis, MN 55455, USA" /><meta name="citation_publication_date" content="2017/02/02" /><meta name="citation_issue" content="2" /><meta name="citation_volume" content="30" /><meta name="citation_firstpage" content="705" /><meta name="citation_doi" content="10.1021/acs.chemrestox.6b00426" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5375106/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC5375106/" /><meta name="citation_pmid" content="27984695" /><meta name="DC.Title" content="Mass Spectrometric Characterization of an Acid-Labile Adduct Formed with 2-Amino-1-methyl-6-phenylimidazo[4,5-b]pyridine and Albumin in Humans" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="NIH Public Access" /><meta name="DC.Contributor" content="Yi Wang" /><meta name="DC.Contributor" content="Peter W. Villalta" /><meta name="DC.Contributor" content="Lijuan Peng" /><meta name="DC.Contributor" content="Karen Dingley" /><meta name="DC.Contributor" content="Michael A. Malfatti" /><meta name="DC.Contributor" content="K.W. Turteltaub" /><meta name="DC.Contributor" content="Robert J. Turesky" /><meta name="DC.Date" content="2017 Feb 20" /><meta name="DC.Identifier" content="10.1021/acs.chemrestox.6b00426" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Mass Spectrometric Characterization of an Acid-Labile Adduct Formed with 2-Amino-1-methyl-6-phenylimidazo[4,5-b]pyridine and Albumin in Humans" /><meta property="og:type" content="article" /><meta property="og:description" content="2-Amino-1-methyl-6-phenylimidazo[4,5-b]pyridine (PhIP) is a carcinogenic heterocyclic aromatic amine formed during the high temperature cooking of meats. The cytochrome P450-mediated N-hydroxylation of the exocyclic amine group of PhIP produces 2-hydroxyamino-1-methyl-6-phenylimidazo[4,5-b]pyridine ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5375106/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC5375106/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC5375106
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC5375106/pdf/nihms854476.pdf" class="int-view">PDF (672K)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/5375106/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/5375106/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="5375106" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5375106%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="tyvFndrb6MBzaTby2k960vF7rv6n4R3iyKUtCzeV4RFu04iQgK89fmPhzR1CQWE3">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5375106%2F&amp;text=Mass%20Spectrometric%20Characterization%20of%20an%20Acid-Labile%20Adduct%20Formed%20with%202-Amino-1-methyl-6-phenylimidazo%5B4%2C5-b%5Dpyridine%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5375106%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5375106/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC5375106/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/27984695/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC5375106/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/27984695/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC5375106/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/5375106/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC5375106
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-nihpa.png" alt="Logo of nihpa" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="https://www.hhs.gov/open/public-access-guiding-principles/index.html" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div><span id="pmcmata">Chem Res Toxicol.</span> Author manuscript; available in PMC 2018 Feb 20.</div><div></div><div class="final-form"><em>Published in final edited form as:</em></div><div class="final-citation"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=27984695" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Chem Res Toxicol. 2017 Feb 20; 30(2): 705–714. </a></div><span class="fm-vol-iss-date">Published online 2017 Jan 18. </span> <span class="doi"><span>doi: </span><a href="//doi.org/10.1021%2Facs.chemrestox.6b00426" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1021/acs.chemrestox.6b00426</a></span></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5375106</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS854476</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/27984695">27984695</a></div></div></div><h1 class="content-title">Mass Spectrometric Characterization of an Acid-Labile Adduct Formed with 2-Amino-1-methyl-6-phenylimidazo[4,5-<em>b</em>]pyridine and Albumin in Humans</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wang%20Y%5BAuthor%5D" class="affpopup" co-rid="_co_idm140108299402576" co-class="co-affbox">Yi Wang</a>,<sup>&#x02020;</sup><sup>&#x02016;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Villalta%20PW%5BAuthor%5D" class="affpopup" co-rid="_co_idm140108299400592" co-class="co-affbox">Peter W. Villalta</a>,<sup>&#x02020;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Peng%20L%5BAuthor%5D" class="affpopup" co-rid="_co_idm140108260992944" co-class="co-affbox">Lijuan Peng</a>,<sup>&#x02021;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Dingley%20K%5BAuthor%5D" class="affpopup" co-rid="_co_idm140108260991408" co-class="co-affbox">Karen Dingley</a>,<sup>&#x000a7;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Malfatti%20MA%5BAuthor%5D" class="affpopup" co-rid="_co_idm140108260989872" co-class="co-affbox">Michael A. Malfatti</a>,<sup>&#x000a7;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Turteltaub%20K%5BAuthor%5D" class="affpopup" co-rid="_co_idm140108260988336" co-class="co-affbox">K.W. Turteltaub</a>,<sup>&#x000a7;</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Turesky%20RJ%5BAuthor%5D" class="affpopup" co-rid="_co_idm140108260986800" co-class="co-affbox">Robert J. Turesky</a><sup>*</sup><sup>&#x02020;</sup><sup>&#x02016;</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140108299402576"><h3 class="no_margin">Yi Wang</h3><p><sup>&#x02020;</sup>Masonic Cancer Center, University of Minnesota, 2231 6th Street, Minneapolis, MN 55455, USA</p><p><sup>&#x02016;</sup>Department of Medicinal Chemistry, Cancer and Cardiovascular Research Building, University of Minnesota, 2231 6th Street, Minneapolis, MN 55455, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wang%20Y%5BAuthor%5D">Yi Wang</a></div></div><div id="_co_idm140108299400592"><h3 class="no_margin">Peter W. Villalta</h3><p><sup>&#x02020;</sup>Masonic Cancer Center, University of Minnesota, 2231 6th Street, Minneapolis, MN 55455, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Villalta%20PW%5BAuthor%5D">Peter W. Villalta</a></div></div><div id="_co_idm140108260992944"><h3 class="no_margin">Lijuan Peng</h3><p><sup>&#x02021;</sup>School of Food Science and Engineering, Wuhan Polytechnic University, ChangQing Garden, Hankou, Wuhan 430023, P.R. China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Peng%20L%5BAuthor%5D">Lijuan Peng</a></div></div><div id="_co_idm140108260991408"><h3 class="no_margin">Karen Dingley</h3><p><sup>&#x000a7;</sup>Biosciences and Biotechnology Division, Center for Accelerator Mass Spectrometry Lawrence Livermore National Laboratory, Livermore 94550, CA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Dingley%20K%5BAuthor%5D">Karen Dingley</a></div></div><div id="_co_idm140108260989872"><h3 class="no_margin">Michael A. Malfatti</h3><p><sup>&#x000a7;</sup>Biosciences and Biotechnology Division, Center for Accelerator Mass Spectrometry Lawrence Livermore National Laboratory, Livermore 94550, CA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Malfatti%20MA%5BAuthor%5D">Michael A. Malfatti</a></div></div><div id="_co_idm140108260988336"><h3 class="no_margin">K.W. Turteltaub</h3><p><sup>&#x000a7;</sup>Biosciences and Biotechnology Division, Center for Accelerator Mass Spectrometry Lawrence Livermore National Laboratory, Livermore 94550, CA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Turteltaub%20K%5BAuthor%5D">K.W. Turteltaub</a></div></div><div id="_co_idm140108260986800"><h3 class="no_margin">Robert J. Turesky</h3><p><sup>&#x02020;</sup>Masonic Cancer Center, University of Minnesota, 2231 6th Street, Minneapolis, MN 55455, USA</p><p><sup>&#x02016;</sup>Department of Medicinal Chemistry, Cancer and Cardiovascular Research Building, University of Minnesota, 2231 6th Street, Minneapolis, MN 55455, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Turesky%20RJ%5BAuthor%5D">Robert J. Turesky</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140108263590880_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140108263590880_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140108263590880_ai" style="display:none"><div class="fm-affl" id="A1"><sup>&#x02020;</sup>Masonic Cancer Center, University of Minnesota, 2231 6th Street, Minneapolis, MN 55455, USA</div><div class="fm-affl" id="A2"><sup>&#x02016;</sup>Department of Medicinal Chemistry, Cancer and Cardiovascular Research Building, University of Minnesota, 2231 6th Street, Minneapolis, MN 55455, USA</div><div class="fm-affl" id="A3"><sup>&#x02021;</sup>School of Food Science and Engineering, Wuhan Polytechnic University, ChangQing Garden, Hankou, Wuhan 430023, P.R. China</div><div class="fm-affl" id="A4"><sup>&#x000a7;</sup>Biosciences and Biotechnology Division, Center for Accelerator Mass Spectrometry Lawrence Livermore National Laboratory, Livermore 94550, CA</div><div id="FN1"><sup>*</sup><strong>Corresponding Author:</strong> Robert J. Turesky, Masonic Cancer Center and Department of Medicinal Chemistry, Cancer and Cardiology Research Building, University of Minnesota, 2231 6th Street, Minneapolis, MN 55455, USA. Tel.: +1 612-626-0141; fax: +1 612-624-3869</div></div><div class="fm-article-notes half_rhythm"></div><div class="permissions half_rhythm hide" id="idm140108263590880_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">PMC Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm" role="complementary" aria-label="Related or updated information about this article."><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=27984695" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Chem Res Toxicol</a></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supporting Info.</strong></div><div><a href="/pmc/articles/PMC5375106/bin/NIHMS854476-supplement-Supporting_Info.docx" data-ga-action="click_feat_suppl">NIHMS854476-supplement-Supporting_Info.docx</a><span style="color:gray"> (424K)</span></div><div class="small guid">GUID: 3E1782D5-67BD-4577-AACF-CF81B5974175</div></div></dd></dl></div><div id="Abs1" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="Abs1title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">2-Amino-1-methyl-6-phenylimidazo[4,5-<em>b</em>]pyridine (PhIP) is a carcinogenic heterocyclic aromatic amine formed during the high temperature cooking of meats. The cytochrome P450-mediated <em>N</em>-hydroxylation of the exocyclic amine group of PhIP produces 2-hydroxyamino-1-methyl-6-phenylimidazo[4,5-<em>b</em>]pyridine (HONH-PhIP), an electrophilic metabolite which forms adducts with DNA and proteins. Previous studies conducted by our laboratory showed that the reaction of N-oxidized-PhIP metabolites with human albumin <em>in vitro</em> occurs primarily at the Cys<sup>34</sup> residue, to produce an acid-labile linked sulfinamide adduct. Based on these findings, we developed a sensitive ultraperformance liquid chromatography-mass spectrometry (UPLC-MS) method to measure acid-labile albumin-PhIP adducts in humansvolunteers administered a dietary-relevant dose of <sup>14</sup>C-labelled-PhIP (Dingley <em>et al.</em>, Cancer Epidemiol. Biomarkers Prev. 8:507&#x02013;512). Mild acid-treatment of albumin (0.1N HCl, 37 &#x000b0;C for 1 h) or proteolytic digestion with pronase (50 mM ammonium bicarbonate buffer pH 8.5 at 37 &#x000b0;C for 18 h) released similar amounts of covalently bound PhIP, which was characterized by multistage scanning (MS<sup>3</sup>) and quantified by Orbitrap mass spectrometry. The amount of <sup>14</sup>C-PhIP recovered by acid-treatment of albumin 24 hour following dosing accounted for 7.2 to 21.3% of the <sup>14</sup>C-PhIP bound to albumin based on accelerator mass spectrometry measurements. 2-Amino-1-methyl-6-(5-hydroxy)-phenylimidazo[4,5-<em>b</em>]pyridine (5-HO-PhIP), a hydrolysis product of the Cys<sup>34</sup> S&#x02212;N linked sulfenamide adduct of PhIP, was not detected in either acid-treated or protease treated samples. These findings suggest that a portion of the PhIP bound to albumin <em>in vivo</em> probably occurs as an acid labile-sulfinamide adduct formed at the Cys<sup>34</sup> residue.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Carcinogens, heterocyclic aromatic amines, serum albumin adducts, protein adducts</span></div></div><div id="Abs2" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="Abs2title">TOC image</h2><!--article-meta--><div><p id="P2" class="p p-first-last">
</p><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140108260827056"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms854476u1.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is nihms854476u1.jpg" src="/pmc/articles/PMC5375106/bin/nihms854476u1.jpg" /></div><p></p></div></div><div id="S1" class="tsec sec"><h2 class="head no_bottom_margin" id="S1title">INTRODUCTION</h2><p id="P3" class="p p-first">Heterocyclic aromatic amines (HAA) are produced during the high-temperature cooking of meats, poultry, and fish, and some HAA are formed during the combustion of tobacco.<sup><a href="#R1" rid="R1" class=" bibr popnode">1</a>,<a href="#R2" rid="R2" class=" bibr popnode">2</a></sup> 2-Amino-1-methyl-6-phenylimidazo[4,5-<em>b</em>]pyridine (PhIP) is the most mass-abundant carcinogenic HAA formed in well done cooked red meats.<sup><a href="#R1" rid="R1" class=" bibr popnode">1</a>,<a href="#R3" rid="R3" class=" bibr popnode">3</a>&#x02013;<a href="#R5" rid="R5" class=" bibr popnode">5</a></sup> PhIP undergoes metabolic activation by cytochrome P450 (P450) to form 2-hydroxyamino-1-methyl-6-phenylimidazo[4,5-<em>b</em>]pyridine (HONH-PhIP), a genotoxic metabolite capable of reacting with DNA and proteins.<sup><a href="#R6" rid="R6" class=" bibr popnode">6</a></sup> The Working Group of the International Agency for Research on Cancer (IARC) recently classified red meat as probably carcinogenic to humans (Class 2A). The classification was based on a large body of epidemiological observations and mechanistic studies in humans and animal studies for the induction of colorectal cancer.<sup><a href="#R6" rid="R6" class=" bibr popnode">6</a>,<a href="#R7" rid="R7" class=" bibr popnode">7</a></sup> However, imprecise estimates of exposure and the lack of physicochemical evidence that mutagens formed in cooked meat induce DNA damage and mutations in target organs are critical missing links in the published literature and vital to performing a complete evaluation of the consumption of cooked meat and the risk of developing cancer.<sup><a href="#R8" rid="R8" class=" bibr popnode">8</a></sup></p><p id="P4">The biomonitoring of DNA adducts represents the most definitive biomarker to assess the biologically effective dose and genotoxic potential of a chemical.<sup><a href="#R8" rid="R8" class=" bibr popnode">8</a>,<a href="#R9" rid="R9" class=" bibr popnode">9</a></sup> However, the measurement of DNA adducts is often limited because of the lack of tissue samples. Moreover, the repair of DNA adducts can diminish the levels of DNA adducts, and the measurement of DNA adducts remains a challenging endeavor even by the most sensitive of mass spectrometry (MS) instruments.<sup><a href="#R6" rid="R6" class=" bibr popnode">6</a></sup> The use of blood protein-carcinogen adducts is an alternative approach to biomonitor exposures to hazardous chemicals.<sup><a href="#R10" rid="R10" class=" bibr popnode">10</a>&#x02013;<a href="#R12" rid="R12" class=" bibr popnode">12</a></sup> Stable blood protein adducts do not undergo repair <em>in vivo</em>, and can accumulate during chronic exposure, and facilitate detection. In the case of PhIP, the same genotoxic metabolites that adduct to albumin are involved in DNA adduct formation.<sup><a href="#R6" rid="R6" class=" bibr popnode">6</a></sup> The albumin adduct levels are thus not only a measure of exposure but also of bioactivation of PhIP.</p><p id="P5">Human albumin contains one reduced cysteine residue &#x02013; the Cys<sup>34</sup>; a strong nucleophile at physiological pH (p<em>K</em>a = ~6.5) and scavenger of a variety of electrophiles.<sup><a href="#R12" rid="R12" class=" bibr popnode">12</a>,<a href="#R13" rid="R13" class=" bibr popnode">13</a></sup> Early studies reported that the Cys<sup>34</sup> residues of mouse or rat albumins reacted with the N-oxidized metabolites of several HAA, including 2-amino-3-methylimidazo[4,5-<em>f</em>]quinoline (IQ),<sup><a href="#R14" rid="R14" class=" bibr popnode">14</a></sup> 2-amino-3,8-dimethylimidazo[4,5-<em>f</em>]quinoxaline,<sup><a href="#R15" rid="R15" class=" bibr popnode">15</a></sup> and PhIP.<sup><a href="#R16" rid="R16" class=" bibr popnode">16</a></sup> In the case of IQ, the sulfinamide adduct formed with albumin in the rat, was isolated, digested with pronase and characterized by mass spectrometry. The <sup>1</sup>H-NMR spectrum of the corresponding adduct formed <em>in vitro</em> with rat albumin was identified as a tripeptide C*PY containing <em>N</em><sup>2</sup>-cysteinylsulfinyl-IQ.<sup><a href="#R14" rid="R14" class=" bibr popnode">14</a></sup> A model peptide containing sequence homology surrounding Cys<sup>34</sup> of rat albumin was shown to react with the <em>N</em>-acetoxy derivative of PhIP to form a sulfur-nitrogen linked sulfenamide adduct, when characterized by ion trap MS.<sup><a href="#R17" rid="R17" class=" bibr popnode">17</a></sup> We employed ion trap MS to characterize human albumin adducts formed with N-oxidized metabolites of PhIP in vitro, followed by digestion of the modified protein with a mixture of trypsin and chymotrypsin.<sup><a href="#R18" rid="R18" class=" bibr popnode">18</a>&#x02013;<a href="#R21" rid="R21" class=" bibr popnode">21</a></sup> The primary adduct formed occurred at Cys<sup>34</sup> and the peptides were identified as <sup>31</sup>LQQC*PF<sup>36</sup> and the single missed-cleaved <sup>31</sup>LQQC*PFEDHVK<sup>41</sup> sulfinamide (C-[SO]-PhIP); the <sup>31</sup>LQQC*PFEDHVK<sup>41</sup> sulfonamide (C-[SO<sub>2</sub>]-PhIP) was identified as a minor adduct.<sup><a href="#R18" rid="R18" class=" bibr popnode">18</a>,<a href="#R20" rid="R20" class=" bibr popnode">20</a></sup> The hydrolysis of the albumin sulfinamide adduct of PhIP occurs under conditions sufficiently mild that the albumin is not degraded,<sup><a href="#R14" rid="R14" class=" bibr popnode">14</a>,<a href="#R20" rid="R20" class=" bibr popnode">20</a></sup> and the released PhIP can be facilely separated from the macromolecular protein and assayed by sensitive MS based methods. The hydrolysis of albumin modified with N-oxidized metabolites of PhIP, by pronase digestion or mild acid treatment, resulted in the recovery of PhIP, which was principally derived from the sulfinamide (C-[SO]-PhIP) linked adduct.<sup><a href="#R19" rid="R19" class=" bibr popnode">19</a>,<a href="#R20" rid="R20" class=" bibr popnode">20</a></sup> The ring-oxidized product 2-amino-1-methyl-6-(5-hydroxy)-phenylimidazo[4,5-<em>b</em>]pyridine (5-HO-PhIP) was also identified in the proteolytic digest of <em>N</em>-(acetyloxy)-2-amino-1-methyl-6-phenylimidazo[4,5-<em>b</em>]pyridine (<em>N</em>-acetoxy-PhIP) adducted adducted albumin, suggesting a sulfenamide adduct of PhIP had formed at the Cys<sup>34</sup> of albumin (<a href="/pmc/articles/PMC5375106/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Scheme 1</span></a>).<sup><a href="#R19" rid="R19" class=" bibr popnode">19</a>,<a href="#R21" rid="R21" class=" bibr popnode">21</a>,<a href="#R22" rid="R22" class=" bibr popnode">22</a></sup></p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F6" co-legend-rid="lgnd_F6"><a href="/pmc/articles/PMC5375106/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140108263403328"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms854476f6.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is nihms854476f6.jpg" src="/pmc/articles/PMC5375106/bin/nihms854476f6.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140108263403328"><a target="object" rel="noopener" href="/pmc/articles/PMC5375106/figure/F6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F6"><div><a class="figpopup" href="/pmc/articles/PMC5375106/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6">Scheme 1</a></div><!--caption a7--><div class="caption"><p>Formation of albumin (Alb) sulfenamide, sulfinamide and sulfonamide adducts formed with N-oxidized metabolites of PhIP, and proposed mechanism of hydrolysis of the linkages to produce PhIP and 5-HO-PhIP. NATs, N-acetyltransferases; SULTs, sulfotransferases.</p></div></div></div><p id="P6">Two studies used accelerator mass spectrometry (AMS) to measure <sup>14</sup>C-PhIP blood protein adduct formation in humans.<sup><a href="#R23" rid="R23" class=" bibr popnode">23</a>,<a href="#R24" rid="R24" class=" bibr popnode">24</a></sup> Both studies reported that PhIP bound poorly to hemoglobin, but levels approaching up to ~ one percent of the dose bound to albumin. However, the structures of the adducts were not determined. The Airoldi laboratory developed gas chromatography-mass spectrometry and liquid chromatography-tandem mass spectrometry (LC-MS/MS) methods to measure acid-labile albumin adducts of PhIP in a cohort from Italy.<sup><a href="#R25" rid="R25" class=" bibr popnode">25</a></sup> PhIP was detected in approximately 30 percent of the subjects. The mean level of albumin adducts of PhIP was significantly higher in meat consumers than in vegetarians (6.7 vs. 0.7 fmol/mg albumin). A more recent study conducted with a cohort from the United Kingdom did not reproduce the findings obtained from the Italian study. The authors concluded that acid-labile albumin adducts of PhIP or other HAA were not formed at levels sufficient to be detected by LC-MS/MS, and the proposed albumin sulfinamide adducts of HAA were unlikely to be viable biomarkers of HAA exposure.<sup><a href="#R26" rid="R26" class=" bibr popnode">26</a></sup></p><p id="P7" class="p p-last">Based on our understanding of the chemistry of reactivity of N-oxidized metabolites of PhIP with human albumin <em>in vitro</em>,<sup><a href="#R18" rid="R18" class=" bibr popnode">18</a>&#x02013;<a href="#R21" rid="R21" class=" bibr popnode">21</a></sup> we sought to determine if acid-labile albumin adducts of PhIP are formed in humans. We used our characterized <sup>31</sup>LQQC*PFEDHVK<sup>41</sup> sulfinamide (C-[SO]-PhIP) as a reference standard, and developed a sensitive UPLC-MS<sup>n</sup> method, employing high resolution accurate mass spectrometric measurement with Orbitrap MS, to characterize and measure the PhIP released from albumin purified from humans, following mild acid hydrolysis or proteolytic digestion. The samples were drawn from the Dingley study, where human volunteers were administered a single dose of [<sup>14</sup>C]-PhIP.<sup><a href="#R24" rid="R24" class=" bibr popnode">24</a></sup></p></div><div id="S2" class="tsec sec"><h2 class="head no_bottom_margin" id="S2title">MATERIALS AND METHODS</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Caution</h3><p id="P8" class="p p-first-last">PhIP is a carcinogen and should be handled in a well-ventilated fume hood with the appropriate protective clothing.</p></div><div id="S4" class="sec"><h3 id="S4title">Chemicals and Materials</h3><p id="P9" class="p p-first-last">PhIP, 2-amino-1-[<sup>2</sup>H<sub>3</sub>C]-methyl-6-phenylimidazo[4,5-<em>b</em>]pyridine ([<sup>2</sup>H<sub>3</sub>C]-PhIP, 99% isotopic purity), and [2&#x02013;<sup>14</sup>C]-PhIP (10 mCi/mmol) were purchased from Toronto Research Chemicals (Toronto, ON, Canada). Human serum albumin, pronase E, <em>meta</em>-chloroperoxybenzoic acid (<em>m</em>-CPBA), hydrochloric acid (HCl), and LC-MS grade formic acid were purchased from Sigma-Aldrich (St. Louis, MO). LC-MS grade solvents were purchased from Fisher Scientific (Pittsburgh, PA). All other chemicals were ACS grade, and purchased from Sigma-Aldrich unless stated. Isolute C18 solid-phase extraction (SPE) columns (25 mg) were from Biotage (Charlotte, NC). SOLA HRP solid-phase extraction (SPE) columns (10 mg) were from Thermo Fisher Scientific (Rockford, IL). Amicon Ultra centrifugal filter units (10,000 mw cutoff) were from Millipore (Billerica, MA). Human plasma was purchased from Bioreclamation LLC (Hicksville, NY). Reference peptide LQQCPFEDHVK was purchased from New England Peptide (Gardner, MA). N-Oxidized derivatives of PhIP were synthesized as previously reported.<sup><a href="#R20" rid="R20" class=" bibr popnode">20</a>,<a href="#R21" rid="R21" class=" bibr popnode">21</a>,<a href="#R27" rid="R27" class=" bibr popnode">27</a></sup></p></div><div id="S5" class="sec"><h3 id="S5title">Human Protocol</h3><p id="P10" class="p p-first-last">The details of this study were previously reported.<sup><a href="#R24" rid="R24" class=" bibr popnode">24</a></sup> Four volunteers were administered a dietary-equivalent of [<sup>14</sup>C]-PhIP (70 &#x02013; 84 &#x003bc;g, at a specific activity of 42 or 56 mCi/mmol) before colon cancer surgery. At various times up to 24 h after dosing, blood samples (30 mL) were collected and separated into plasma, RBCs, and buffy coat by centrifugation, and immediately stored at &#x02212;80 &#x000b0;C. Plasma samples were stored on dry-ice and transported to the University of Minnesota for chemical analyses. For reference, the dose of PhIP that the subjects in this study received was approximately equivalent to eating 175 g of very well-done chicken,<sup><a href="#R5" rid="R5" class=" bibr popnode">5</a></sup> and the radioactive dose was &#x0003c;1% of an individual&#x02019;s yearly exposure to background sources of radiation. The protocol for this study was approved by the human subject review boards at Lawrence Livermore National Laboratory, the University of Arkansas Medical School Hospital, and the J. L. McClellan Memorial Veterans Administration Medical Center in Little Rock, AR.</p></div><div id="S6" class="sec"><h3 id="S6title">AMS Analysis</h3><p id="P11" class="p p-first-last">The carbon-14:carbon-13 ratios of albumin samples were assayed previously as described.<sup><a href="#R24" rid="R24" class=" bibr popnode">24</a></sup> For subjects 1 and 2, AMS analyses were performed and reported as the mean &#x000b1; standard deviation of four separate analyses. The AMS analyses for subjects 7 and 10 were derived from single values (&#x000b1; standard deviation of the AMS measurement).</p></div><div id="S7" class="sec"><h3 id="S7title">Syntheses of the PhIP Sulfinamide and Sulfonamide Adducts of LQQC*PFEDHVK</h3><p id="P12" class="p p-first-last">The protocol for the synthesis of sulfinamide and sulfonamide adducts was described previously.<sup><a href="#R28" rid="R28" class=" bibr popnode">28</a></sup> Briefly, LQQCPFEDHVK (20 nmol, 250 <em>&#x003bc;</em>L in 10 mM potassium phosphate buffer pH 8.5) was modified with 2-nitroso-1-methyl-6-phenylimidazo[4,5-<em>b</em>]pyridine (NO-PhIP) (60 nmol, 50 <em>&#x003bc;</em>L in MeOH) at 5 &#x000b0;C for 1 h. LQQC*PFEDHVK sulfinamide (C-[SO]-PhIP) was formed by adding 250 <em>&#x003bc;</em>L 50 mM potassium phosphate buffer pH 3.6 to the reaction mixture, which agitated at 37 &#x000b0;C for 1 h. LQQC*PFEDHVK sulfonamide (C-[SO<sub>2</sub>]-PhIP) was formed by oxidization of LQQC*PFEDHVK sulfinamide (C-[SO]-PhIP) by <em>m</em>-CPBA at a molar ratio of 1:1 at 37 &#x000b0;C for 1 h. LQQC*PFEDHVK sulfinamide (C-[SO]-PhIP) and sulfonamide (C-[SO<sub>2</sub>]-PhIP) were purified by HPLC (Acquasil C18, 5 <em>&#x003bc;</em>m, 4.6 &#x000d7; 250 mm, Thermo Fisher) with a 25 min gradient from 100% A (5% ACN and 95% H<sub>2</sub>O) to 100% B (100% ACN), at a flow rate of 1 mL/min. The concentrations of LQQC*PFEDHVK sulfinamide and sulfonamide were estimated by the UV absorbance of the PhIP chromophore at 315 nm (molar extinction coefficient, 22,220 M<sup>&#x02212;1</sup> cm<sup>&#x02212;1</sup>).<sup><a href="#R29" rid="R29" class=" bibr popnode">29</a></sup> Structures of LQQC*PFEDHVK sulfinamide and sulfonamide were confirmed by sequencing and multistage scanning (MS<sup>n</sup>).<sup><a href="#R18" rid="R18" class=" bibr popnode">18</a>,<a href="#R20" rid="R20" class=" bibr popnode">20</a></sup></p></div><div id="S8" class="sec"><h3 id="S8title">Modification of Commercial Human Albumin with HONH-PhIP in vitro</h3><p id="P13" class="p p-first-last">Mixed disulfides formed at Cys<sup>34</sup> of albumin were reduced by treatment with <em>&#x003b2;</em>-mercaptoethanol.<sup><a href="#R14" rid="R14" class=" bibr popnode">14</a></sup> Albumin was modified with HONH-PhIP at a molar ratio of 1:3, at 37 &#x000b0;C for 18 h. Unbound HONH-PhIP and its decomposition products were removed by liquid extraction with 2&#x000d7; volume of ethyl acetate, twice. To fully convert the LQQC*PFEDHVK sulfinamide (C-[SO]-PhIP) to the LQQC*PFEDHVK sulfonamide (C-[SO<sub>2</sub>]-PhIP), the modified human albumin was oxidized by <em>m</em>-CPBA at a molar ratio of 1:1, at 37 &#x000b0;C for 5 h.<sup><a href="#R21" rid="R21" class=" bibr popnode">21</a></sup> The solution was subjected to a buffer exchange (450 <em>&#x003bc;</em>L of 100 mM potassium phosphate buffer, pH 7.4, 2-times) using centrifugal filters (10,000 mw cutoff).</p></div><div id="S9" class="sec"><h3 id="S9title">Purification of Albumin from Human Plasma</h3><p id="P14" class="p p-first-last">Albumin was purified from human plasma samples using the Pierce Albumin Depletion Kit. Briefly, human plasma samples (200 <em>&#x003bc;</em>L) were desalted using centrifugal filters (10,000 mw cutoff) with 25 mM potassium phosphate buffer pH 7.4, twice. Desalted human plasma was applied to affinity resins (500 <em>&#x003bc;</em>L vol), which were equilibrated in 25 mM potassium phosphate buffer pH 7.4. The resins bound with albumin were rinsed with 500 <em>&#x003bc;</em>L 25 mM potassium phosphate buffer, pH 7.4 (three times), and eluted with 1 mL of 25 mM potassium phosphate buffer, pH 7.4 containing 1.5 M potassium chloride. The elutes were desalted using centrifugal filters (10,000 mw cutoff).</p></div><div id="S10" class="sec"><h3 id="S10title">Acid Hydrolysis of Peptide Adducts of PhIP or Albumin-PhIP Adducts from Human Plasma</h3><p id="P15" class="p p-first-last">The PhIP products unbound to albumin were removed by solvent extraction with 2&#x000d7; volume of ethyl acetate, twice. Thereafter, the synthetic peptides LQQC*PFEDHVK sulfinamide (C-[SO]-PhIP) or sulfonamide (C-[SO<sub>2</sub>]-PhIP) (200 ng in 90 <em>&#x003bc;</em>L H<sub>2</sub>O), or albumin purified from human plasma containing [<sup>14</sup>C]-PhIP (10 mg albumin in 90 <em>&#x003bc;</em>L 25 mM potassium phosphate buffer, pH 7.4) was acidified with 10 <em>&#x003bc;</em>L 1 N HCl. [<sup>2</sup>H<sub>3</sub>C]-PhIP was spiked into the mixture as an internal standard at level of 500 fg [<sup>2</sup>H<sub>3</sub>C]-PhIP/mg albumin. The mixture was agitated at 37 &#x000b0;C for 1 h. Then, the mixture was made alkaline with 1N NaOH (15 <em>&#x003bc;</em>L), and the PhIP released from albumin was extracted by CH<sub>3</sub>OH or ethyl acetate. For the CH<sub>3</sub>OH extraction method, pre-chilled CH<sub>3</sub>OH (900 <em>&#x003bc;</em>L, &#x02212;80 &#x000b0;C) was added to the mixture to precipitate the proteins at &#x02212;20 &#x000b0;C for 2 h. The solution was centrifuged at 18,000 g for 20 min, and the supernatant was transferred to a 2 mL Eppendorf tube, vacuum centrifuged for 30 min to remove CH<sub>3</sub>OH, and resuspended in H<sub>2</sub>O (1 mL). For the ethyl acetate extraction method, ethyl acetate (1 mL) was added to the mixture. After vortexing, the upper layer containing the organic fraction was transferred to an Eppendorf tube, vacuum centrifuged to dryness, and resuspended in H<sub>2</sub>O (1 mL). The mixture was applied to a SOLA HRP cartridge (10 mg). The cartridge was rinsed with 10% CH<sub>3</sub>OH (2 mL), and PhIP was eluted with CH<sub>3</sub>OH (1 mL). The elute was vacuum centrifuged to dryness. The levels of adducts measured were the same for both clean-up methods (data not shown); however, the extraction with ethyl acetate provided cleaner chromatograms and was used for all in vivo studies.</p></div><div id="S11" class="sec"><h3 id="S11title">Proteolytic Digestion with Pronase E</h3><p id="P16" class="p p-first-last">Peptide or protein adducts were digested with pronase E (1:2 w/w) in 50 mM ammonium bicarbonate buffer pH 8.5 at 37 &#x000b0;C for 18 h. The mixture was applied to an Isolute C18 SPE column. Polar peptides or amino acids were removed with 10% CH<sub>3</sub>OH (2 mL) and adducts were eluted with CH<sub>3</sub>OH (1 mL), followed by concentration to dryness by vacuum centrifugation.</p></div><div id="S12" class="sec"><h3 id="S12title">Stability of PhIP-Albumin Adducts in Human Plasma <em>in vitro</em></h3><p id="P17" class="p p-first-last">Human plasma containing [<sup>14</sup>C]-PhIP (150 <em>&#x003bc;</em>L) was diluted with an equal volume of 100 mM potassium phosphate buffer (pH 7.4) and incubated at 37 &#x000b0;C for 0 h, 6 h, 1 d, 3 d or 7 d. HONH-PhIP modified commercial human albumin (150 pmol) was added to 100 mM potassium phosphate buffer (pH 7.4, 1 mL) or commercial human plasma (1 mL) and also incubated overtime. Procedures for albumin purification and ensuing acid hydrolysis of PhIP adducts are the same as those conditions reported above. The PhIP products unbound or released from albumin were removed by solvent extraction with 2&#x000d7; volume of ethyl acetate, twice, prior to acid hydrolysis of the remaining PhIP bound to albumin, followed by isolation of the released PhIP using the sample clean-up by SPE as described above.</p></div><div id="S13" class="sec"><h3 id="S13title">Measurement of PhIP and 5-HO-PhIP by Ultraperformance Liquid Chromatography-Electrospray Ionization-Multistage Mass Spectrometry (UPLC-ESI/MS<sup>n</sup>)</h3><p id="P18" class="p p-first-last">The acid hydrolysis products of albumin PhIP adducts were assayed by UPLC-ESI/MS<sup>n</sup> employing a Dionex Ultimate 3000 LC and an Orbitrap Elite&#x02122; Hybrid Ion Trap-Orbitrap mass spectrometer (Thermo Fisher Scientific, San Jose, CA). Following SPE, the extracts were resuspended in 10% DMSO:90% H<sub>2</sub>O (25 <em>&#x003bc;</em>L), and 5 <em>&#x003bc;</em>L was loaded on to a trapping column (Thermo Fisher Scientific Acclaim PepMap 100 C18, 100 A&#x000b0;, particle size 5 <em>&#x003bc;</em>m, 300 &#x003bc;m &#x000d7; 5 mm) with 99% A (1% ACN, 0.01% HCO<sub>2</sub>H) at a flow rate of 5 <em>&#x003bc;</em>L/min for 6 min. The products were back flushed onto a Michrom C18 column (0.1 &#x000d7; 150 mm, 3 <em>&#x003bc;</em>m particle), resolved with a 25 min linear gradient from 99% A (1% CH<sub>3</sub>CN, 0.01% HCO<sub>2</sub>H to 99% B (95% CH<sub>3</sub>CN, 0.01% HCO<sub>2</sub>H) at a flow rate of 1 <em>&#x003bc;</em>L/min. Chromeleon 7.2 Chromatography Data System was used for the UPLC management. The Advance CaptiveSpray (Michrom Bioresource, Auburn, CA) was employed as the ion source and MS parameters were set as follows: capillary temperature, 270 &#x000b0;C; ionization voltage, 2 kV at positive ion mode; 1 <em>&#x003bc;</em>scan; maximum injection time, 100 ms for MS and 200 ms for MS<sup>n</sup>; MS fragmentation used a normalized collision energy of 45%; no auxiliary and sheath gases were used. Resolution was set as 60,000 at <em>m/z</em> 400 for selected ion monitoring (SIM), and 15,000 at <em>m/z</em> 400 for MS<sup>2</sup> and MS<sup>3</sup>. The isolation width was set at <em>m</em>/<em>z</em> 1 for MS<sup>2</sup> and MS<sup>3</sup> scan modes, and the activation Q was set at 0.3. AGC (automated gain control) was set at 50,000 for Orbitrap (FT) MS<sup>2</sup> and MS<sup>3</sup>. The Orbitrap was routinely calibrated in positive ion mode using Pierce LTQ Velos ESI Positive Ion Calibration Solution (2 <em>&#x003bc;</em>g/mL caffeine, 1 <em>&#x003bc;</em>g/mL MRFA, 0.001% Ultramark 1621 and 0.00005% <em>n</em>-butylamine).</p></div><div id="S14" class="sec"><h3 id="S14title">Calibration Curve and Quantitation of PhIP</h3><p id="P19" class="p p-first-last">Seven calibrant points (0, 50, 100, 250, 500, 1000, 1250 fg PhIP or 5-HO-PhIP/mg albumin) were constructed using unlabeled PhIP and 5-HO-PhIP, which were added to the extract post-cleanup of acid-treated commercial albumin purified from plasma. [<sup>2</sup>H<sub>3</sub>C]-PhIP and 5-HO-[<sup>2</sup>H<sub>3</sub>C]-PhIP were added internal standards at a level of 500 fg/mg albumin. PhIP and [<sup>2</sup>H<sub>3</sub>C]-PhIP were measured at the MS<sup>3</sup> scan stage using product ions at <em>m/z</em> 225.1 or 228.1 &#x0003e; 210.1 &#x0003e; 168.0682 and 183.0791 for PhIP, and [<sup>14</sup>C]-PhIP was measured at MS<sup>3</sup> scan stage using product ions at <em>m/z</em> 227.1 &#x0003e; 212.1 &#x0003e; 168.0682, 183.0791 and 185.0823. 5-HO-PhIP, 5-HO-[<sup>2</sup>H<sub>3</sub>C]-PhIP and 5-HO-[<sup>14</sup>C]-PhIP were measured at MS<sup>3</sup> scan stage using product ions at <em>m/z</em> 241.1 &#x0003e; 223.1 &#x0003e; 196.0867, 208.0742; <em>m/z</em> 244. 1&#x0003e; 226.1 &#x0003e; 198.0902, 210.0776; and <em>m/z</em> 243.1 &#x0003e; 225.1 &#x0003e; 199.1058, 208.0742, respectively. The calibration curves were generated with unlabeled PhIP and 5-HO-PhIP because the [<sup>14</sup>C]-PhIP used for the human studies was largely carrier free (42 or 56 mCi/mmol; 67 or 90% labeled, respectively) and no longer available. Given that the signals of response of PhIP and [<sup>14</sup>C]-PhIP (10 mCi/mmol) are comparable, when the specific activity of the radioisotope is accounted for (<a href="#SD1" rid="SD1" class=" supplementary-material">Supporting information, Figs S-1A &#x02013; S-1D</a>), with little background noise in the MS<sup>3</sup> scan stage chromatograms, we used PhIP as a substitute calibrant to estimate the acid-labile [<sup>14</sup>C]-PhIP recovered from the albumin <em>in vivo</em>. This also minimized the contamination of radioactive PhIP in the mass spectrometer.</p></div><div id="S15" class="sec sec-last"><h3 id="S15title">Method Validation</h3><p id="P20" class="p p-first-last">The assay was validated by the accuracy and within-day and between-day reproducibility using human albumin modified with HONH-PhIP at a known level of acid-labile adduct and diluted with commercial human plasma to arrive at a level of adduction of 675 fg PhIP/mg albumin. This level of PhIP adduction to albumin was close to the average level [<sup>14</sup>C]-PhIP bound to albumin reported in the AMS study of Dingley.<sup><a href="#R24" rid="R24" class=" bibr popnode">24</a></sup> The reproducibility studies were based on four independent measurements conducted on three different days. The accuracy and percent coefficient of variation (%CV) were used as criteria for the performance and reproducibility of the method.</p></div></div><div id="S16" class="tsec sec"><h2 class="head no_bottom_margin" id="S16title">RESULTS</h2><p id="P21" class="p p-first">The AMS studies conducted by Dingley reported that up to ~0.2 percent of the dose of [<sup>14</sup>C]-PhIP administered to volunteers had bound to albumin; however, the structure(s) of the adduct(s) and stability towards acid (or base) were not reported.<sup><a href="#R23" rid="R23" class=" bibr popnode">23</a>,<a href="#R24" rid="R24" class=" bibr popnode">24</a></sup> The levels of [<sup>14</sup>C]-PhIP bound to albumin were estimated, by AMS, at ~ 500 pg per g albumin (~1.5 PhIP molecules bound per 10<sup><a href="#R7" rid="R7" class=" bibr popnode">7</a></sup> molecules of albumin),<sup><a href="#R24" rid="R24" class=" bibr popnode">24</a></sup> a level of modification far too low to directly measure the tryptic or chymotryptic peptide adduct. We sought to quantify the amount of [<sup>14</sup>C]-PhIP bound to albumin in humans that could be recovered as [<sup>14</sup>C]-PhIP or 5-HO-[<sup>14</sup>C]-PhIP, following mild acid treatment, as would be expected for the PhIP sulfinamide or sulfenamide linkages (Scheme l). We examined the stability of S-N linked adducts of the sulfinamide and sulfonamide adducts of with LQQC*PFEDHVK formed with HONH-PhIP following treatment with acid or pronase; however, the S-N linked adduct of the sulfenamide LQQC*PFEDHVK formed with <em>N</em>-acetoxy-PhIP was too unstable to isolate and characterize. Thereafter, we examined the recovery of acid-labile adducts of albumin modified <em>in vitro</em> with HONH-PhIP. Subsequently, we measured for [<sup>14</sup>C]-PhIP and 5-HO-[<sup>14</sup>C]-PhIP hydrolysis product(s) from <em>in vivo</em> samples of purified albumin of volunteers of the Dingley study.<sup><a href="#R24" rid="R24" class=" bibr popnode">24</a></sup></p><div id="S17" class="sec"><h3 id="S17title">Mass Spectra of PhIP, [<sup>2</sup>H<sub>3</sub>C]-PhIP and [<sup>14</sup>C]-PhIP Standards</h3><p id="P22" class="p p-first">The product ion spectra of the synthetic PhIP analogues were acquired online by UPLC/MS<sup>n</sup>. The ion chromatograms of protonated PhIP ([M+H]<sup>+</sup> at <em>m/z</em> 225.1135) and [<sup>14</sup>C]-PhIP ([M+H]<sup>+</sup> at <em>m/z</em> 227.1167) are shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Fig. S-1</a>. The ratios of peak areas of [<sup>14</sup>C]-PhIP to PhIP by SIM and MS<sup>2</sup> scan stage were 21.2 &#x000b1; 0.2% and 20.8 &#x000b1; 0.1%, respectively (<a href="#SD1" rid="SD1" class=" supplementary-material">Figs S-1A &#x02013; S-1D</a>). Since the maximum specific activity available for carrier free molecule containing a single <sup>14</sup>C label is 62.4 mCi, high resolution accurate mass measurements of the [<sup>14</sup>C]-PhIP/[<sup>12</sup>C]-PhIP ratio correspond to a specific activity of 13.2 and 13.0 mCi/mmol in SIM and MS<sup>2</sup>, respectively. The ratio of [<sup>14</sup>C]-PhIP to PhIP at the MS<sup>3</sup> scan stage was 19.1 &#x000b1; 0.4%, or 12.1 mCi/mmol (<a href="#SD1" rid="SD1" class=" supplementary-material">Fig. S-1E and S-1F</a>), which is consistent with the specific activity of [<sup>14</sup>C]-PhIP reported by the vendor (10 mCi/mmol). There was no overlap of signals of the reconstructed ion chromatograms of [<sup>2</sup>H<sub>3</sub>C]-PhIP with either PhIP or [<sup>14</sup>C]-PhIP at the MS<sup>2</sup> and MS<sup>3</sup> scan stages (<a href="#SD1" rid="SD1" class=" supplementary-material">Supporting information, Fig. S-2</a>), when employing an isolation width of 1 <em>m/z</em> for MS<sup>2</sup> and MS<sup>3</sup>, ensuring specificity of the transitions of PhIP analogues used for assay.</p><p id="P23" class="p p-last">The product ion spectra of PhIP and [<sup>2</sup>H<sub>3</sub>C]-PhIP obtained by collision induced dissociation (CID), (<a href="/pmc/articles/PMC5375106/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig 1</span></a>) show a major product ion at <em>m/z</em> 210.0897 (observed vs. calculated 210.0900, &#x02206;1.4 ppm) as the base peak, and the [<sup>14</sup>C]-PhIP undergoes CID to form a major fragment ion at <em>m/z</em> 212.0928 (observed vs. calculated 212.0932, &#x02206;1.9 ppm). These ions are attributed to the loss of CH<sub>3</sub><sup>&#x02022;</sup> for PhIP and [<sup>14</sup>C]-PhIP, and the loss of [<sup>2</sup>H<sub>3</sub>C]<sup>&#x02022;</sup> from the trideuterated internal standard (<a href="/pmc/articles/PMC5375106/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figs 2A</span></a>&#x02013;<a href="/pmc/articles/PMC5375106/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>2C</span></a>), an observation in agreement with the literature.<sup><a href="#R30" rid="R30" class=" bibr popnode">30</a></sup> A minor product ion at <em>m/z</em> 208.0874 (observed vs. calculated 208.0875, &#x02206;0.5 ppm) for PhIP, <em>m/z</em> 211.1065 (observed vs. calculated 211.1063, &#x02206;1.0 ppm) for [<sup>2</sup>H<sub>3</sub>C]-PhIP, or <em>m/z</em> 210.0906 (observed. vs. 210.0907, &#x02206;1.5 ppm) for [<sup>14</sup>C]-PhIP results from the loss of NH<sub>3</sub> (<a href="/pmc/articles/PMC5375106/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figs 2A</span></a>&#x02013;<a href="/pmc/articles/PMC5375106/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>2C</span></a>). Consecutive reaction monitoring of the radical cation at <em>m/z</em> 210.09 for PhIP and [<sup>2</sup>H<sub>3</sub>C]-PhIP at the MS<sup>3</sup> scan stage produces an ion at <em>m/z</em> 183.0787 (observed vs. calculated 183.0791, &#x02206;2.2 ppm, loss of HCN), <em>m/z</em> 168.0680 (observed vs. calculated 168.0682, &#x02206;1.1 ppm, loss of H<sub>2</sub>CN<sub>2</sub>), and <em>m/z</em> 167.0601 (observed vs. calculated 167.0604, &#x02206;1.8 ppm, loss of H<sub>3</sub>CN<sub>2</sub><sup>&#x02022;</sup>) (<a href="/pmc/articles/PMC5375106/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figs. 2D</span></a> and <a href="/pmc/articles/PMC5375106/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and2E).</span></span><span>2E</span></a>). Similarly, the product ion spectrum acquired for the <em>m/z</em> 212.0932 of [<sup>14</sup>C]-PhIP includes ions at <em>m/z</em> 168.0680, <em>m/z</em> 167.0600, <em>m/z</em> 183.0786 attributed to the loss of H<sup>14</sup>CN, and an ion at <em>m/z</em> 185.0823 (observed vs. calculated 185.0823, &#x02206;0 ppm, loss of HCN).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC5375106/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140108263427184"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms854476f1.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is nihms854476f1.jpg" src="/pmc/articles/PMC5375106/bin/nihms854476f1.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140108263427184"><a target="object" rel="noopener" href="/pmc/articles/PMC5375106/figure/F1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC5375106/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p>Product ion spectra of (A) PhIP ([M+H]<sup>+</sup>
<em>m/z</em> 225.1 &#x0003e;), (B) [<sup>2</sup>H<sub>3</sub>C]-PhIP ([M+H]<sup>+</sup>
<em>m/z</em> 228.1 &#x0003e;), and (C) [<sup>14</sup>C]-PhIP ([M+H]<sup>+</sup>
<em>m/z</em> 227.1 &#x0003e;) of [<sup>14</sup>C]-PhIP and [<sup>2</sup>H<sub>3</sub>C]-PhIP reference standards at the MS<sup>2</sup> scan stage. Product ion spectra of (D) PhIP ([M+H]<sup>+</sup>
<em>m/z</em> 225.1 &#x0003e; 210.1 &#x0003e;), (E) [<sup>2</sup>H<sub>3</sub>C]-PhIP ([M+H]<sup>+</sup>
<em>m/z</em> 228.1 &#x0003e; 210.1 &#x0003e;), and (F) [<sup>14</sup>C]-PhIP ([M+H]<sup>+</sup>
<em>m/z</em> 227.1 &#x0003e; 212.1 &#x0003e;) of [<sup>14</sup>C]-PhIP and [<sup>2</sup>H<sub>3</sub>C]-PhIP standards at MS<sup>3</sup> scan stage. The <em>m/z</em> values shown in the structure scheme are the calculated values. The <em>m/z</em> values shown in the mass spectra are the observed values.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC5375106/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140108263419168"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms854476f2.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is nihms854476f2.jpg" src="/pmc/articles/PMC5375106/bin/nihms854476f2.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140108263419168"><a target="object" rel="noopener" href="/pmc/articles/PMC5375106/figure/F2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC5375106/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p>Reconstructed ion chromatograms at MS<sup>2</sup> stage of (A, B) [<sup>14</sup>C]-PhIP ([M+H]<sup>+</sup>
<em>m/z</em> 227.1 &#x0003e; 212.0932) recovered from human plasma of volunteer 2 given a capsule containing 56 mCi/mmol [<sup>14</sup>C]-PhIP. Reconstructed ion chromatograms at MS<sup>3</sup> stage of (E, F) [<sup>14</sup>C]-PhIP ([M+H]<sup>+</sup>
<em>m/z</em> 227.1 &#x0003e; 212.0932 &#x0003e; 168.0682, 183.0791, 185.0823) recovered from in <em>in vivo</em> human plasma. (C, D, G, H) [<sup>2</sup>H<sub>3</sub>C]-PhIP were spiked to the samples as internal standards. The mass extraction window was &#x000b1; 5 ppm.</p></div></div></div></div><div id="S18" class="sec"><h3 id="S18title">Characterization of the Hydrolysis Products of LQQC*PFEDHVK Sulfinamide and Sulfonamide Adducts of PhIP</h3><p id="P24" class="p p-first">The LQQC*PFEDHVK sulfinamide (C-[SO]-PhIP) undergoes hydrolysis under mild acidic pH conditions to produce PhIP and the cysteine sulfinic acid containing peptide.<sup><a href="#R19" rid="R19" class=" bibr popnode">19</a>,<a href="#R20" rid="R20" class=" bibr popnode">20</a></sup> The LQQC*PFEDHVK sulfinamide (100 pg = 63 fmol) was subjected to acid hydrolysis in 0.1N HCl for 1 h or pronase E digestion for 18 h. [<sup>2</sup>H<sub>3</sub>C]-PhIP (200 fg) was spiked as an internal standard prior to hydrolysis. PhIP was quantitatively recovered after acid hydrolysis, whereas 51 fmol of PhIP was recovered from the LQQC*PFEDHVK sulfinamide following digestion with pronase E (<a href="/pmc/articles/PMC5375106/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). The lower recovery of PhIP by digestion of the modified albumin with pronase E was because the enzyme also produced some C* and C*PF sulfinamide adducts.<sup><a href="#R18" rid="R18" class=" bibr popnode">18</a>,<a href="#R21" rid="R21" class=" bibr popnode">21</a></sup></p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p>Amounts of PhIP recovered from LQQC*PFEDHVK sulfinamide and sulfonamide peptides after acid hydrolysis or digestion with pronase E.</p></div><div class="xtable"><table frame="hsides" rules="none" class="rendered small default_table"><thead><tr><th valign="top" align="center" rowspan="1" colspan="1"></th><th valign="top" align="center" rowspan="1" colspan="1">No treatment</th><th valign="top" align="center" rowspan="1" colspan="1">Acid Hydrolysis</th><th valign="top" align="center" rowspan="1" colspan="1">Pronase E Digestion</th></tr></thead><tbody><tr><td valign="top" align="center" rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1">fmol</td><td valign="top" align="center" rowspan="1" colspan="1">fmol</td><td valign="top" align="center" rowspan="1" colspan="1">fmol</td></tr><tr><td valign="top" align="center" rowspan="1" colspan="1">LQQC*PFEDHVK sulfinamide</td><td valign="top" align="center" rowspan="1" colspan="1">63.8 &#x000b1;15.3</td><td valign="top" align="center" rowspan="1" colspan="1">ND</td><td valign="top" align="center" rowspan="1" colspan="1">ND</td></tr><tr><td valign="top" align="center" rowspan="1" colspan="1">PhIP</td><td valign="top" align="center" rowspan="1" colspan="1">3.7 &#x000b1;0.3</td><td valign="top" align="center" rowspan="1" colspan="1">64.3 &#x000b1;0.2</td><td valign="top" align="center" rowspan="1" colspan="1">50.6 &#x000b1;0.2</td></tr><tr><td valign="top" align="center" rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1">fmol</td><td valign="top" align="center" rowspan="1" colspan="1">fmol</td><td valign="top" align="center" rowspan="1" colspan="1">fmol</td></tr><tr><td valign="top" align="center" rowspan="1" colspan="1">LQQC*PFEDHVK Sulfonamide</td><td valign="top" align="center" rowspan="1" colspan="1">38.1 &#x000b1;0.1</td><td valign="top" align="center" rowspan="1" colspan="1">31.3 &#x000b1;0.1</td><td valign="top" align="center" rowspan="1" colspan="1">ND</td></tr><tr><td valign="top" align="center" rowspan="1" colspan="1">PhIP</td><td valign="top" align="center" rowspan="1" colspan="1">ND</td><td valign="top" align="center" rowspan="1" colspan="1">ND</td><td valign="top" align="center" rowspan="1" colspan="1">ND</td></tr><tr><td valign="top" align="center" rowspan="1" colspan="1">C* sulfonamide</td><td valign="top" align="center" rowspan="1" colspan="1">ND</td><td valign="top" align="center" rowspan="1" colspan="1">ND</td><td valign="top" align="center" rowspan="1" colspan="1">19.5&#x000b1;0.1</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140108259256608"><a target="object" rel="noopener" href="/pmc/articles/PMC5375106/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>a</sup>PhIP and peptides were determined by UPLC-MS<sup>2</sup> analysis of LQQC*PFEDHVK sulfinamide ([M+3H]<sup>3+</sup>
<em>m/z</em> 527.9 &#x0003e; 225.1, 679.1), PhIP ([M+H]<sup>+</sup>
<em>m/z</em> 225.1 &#x0003e; 210.1), and C*PF sulfonamide ([M+H]<sup>+</sup>
<em>m/z</em> 620.1 &#x0003e; 225.1, 602.1), [<sup>2</sup>H<sub>3</sub>C]-PhIP ([M+H]<sup>+</sup>
<em>m/z</em> 228.1 &#x0003e; 210.1) was used the internal standard. Values are reported as the mean &#x000b1; SD (<em>n</em>=3). ND, not detected, below the limit of detection. The amount of C*sulfonamide is an estimate based on the response relative to that of [<sup>2</sup>H<sub>3</sub>C]-PhIP.</div></div></div><p id="P25" class="p p-last">The LQQC*PFEDHVK sulfinamide (C-[SO]-PhIP) linkage can be oxidized to the acid-stable sulfonamide linage (C-[SO<sub>2</sub>]-PhIP), by treatment of the adducted peptide with <em>m</em>-CPBA.<sup><a href="#R20" rid="R20" class=" bibr popnode">20</a></sup> We examined the stability of the sulfonamide linkage towards the same acid hydrolysis or proteolytic digestion conditions described above. PhIP was not recovered by either the acid or pronase treatment of LQQC*PFEDHVK sulfonamide (C-[SO<sub>2</sub>]-PhIP). The digestion with pronase produced the tripeptide adduct C*PF sulfonamide (C-[SO<sub>2</sub>]-PhIP) (<a href="/pmc/articles/PMC5375106/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). These data demonstrate that the S-N linkage of the LQQC*PFEDHVK sulfinamide (C-[SO]-PhIP) is labile, and the adduct can be quantitatively hydrolyzed to PhIP by acid treatment or by pronase E digestion. In contrast, S-N linkage of the LQQC*PFEDHVK sulfonamide (C-[SO<sub>2</sub>]-PhIP) is stable and does not undergo hydrolysis to form PhIP.</p></div><div id="S19" class="sec"><h3 id="S19title">Acid Hydrolysis and Proteolytic Digestion of Commercial Human Albumin Modified with HONH-PhIP with or without Oxidation of <em>m</em>-CPBA</h3><p id="P26" class="p p-first">LQQC*PF and LQQC*PFEDHVK sulfinamide (C-[SO]-PhIP) are major adducts identified in HONH-PhIP modified albumin following digestion with trypsin/chymotrypsin.<sup><a href="#R28" rid="R28" class=" bibr popnode">28</a></sup>
<em>m</em>-CPBA oxidizes the sulfenamide and sulfinamide-PhIP linked adducts to the stable sulfonamide (C-[SO<sub>2</sub>]-PhIP).<sup><a href="#R28" rid="R28" class=" bibr popnode">28</a></sup> Here we measured the recovery of PhIP from HONH-PhIP-modified human albumin without or with <em>m</em>-CPBA-pretreatment, followed by acid hydrolysis or proteolytic digestion with pronase E.</p><p id="P27" class="p p-last">To ensure that the PhIP recovered by hydrolysis was attributed to covalently bound PhIP-albumin adducts, a liquid extraction was performed with 2&#x000d7; volume of ethyl acetate followed by sample cleanup with filtration, which effectively removes all unbound PhIP spiked into a solution of albumin.<sup><a href="#R21" rid="R21" class=" bibr popnode">21</a></sup> In the HONH-PhIP-modified albumin, 88 &#x000b1; 0.8 and 64 &#x000b1; 14.6 (fmol PhIP/pmol albumin) PhIP was recovered after acid hydrolysis and proteolytic digestion with pronase E, respectively (<a href="/pmc/articles/PMC5375106/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). By comparison, only 1.3 &#x000b1; 0.1 and 3.3 &#x000b1; 0.3 (fmol PhIP/pmol albumin) was recovered from the <em>m</em>-CPBA oxidized albumin modified with HONH-PhIP (<a href="/pmc/articles/PMC5375106/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). These data recapitulate the findings observed with LQQC*PFEDHVK (C-[SO]-PhIP and C-[SO<sub>2</sub>]-PhIP) peptides: the sulfinamide adduct of PhIP in albumin is labile under acidic pH conditions, while sulfonamide adduct of PhIP is stable towards acid hydrolysis.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p>The measurement of PhIP recovered from <em>in vitro</em> HONH-PhIP (45 pmol) modified commercial human albumin (15 pmol), subsequently treated with or without <em>m</em>-CPBA (3&#x000d7; molar excess of albumin).<sup>a</sup></p></div><div class="xtable"><table frame="hsides" rules="none" class="rendered small default_table"><thead><tr><th valign="top" align="center" rowspan="1" colspan="1"></th><th colspan="2" valign="top" align="center" rowspan="1">HONH-PhIP modified human albumin Without oxidization by <em>m</em>-CPBA</th><th colspan="2" valign="top" align="center" rowspan="1">HONH-PhIP modified human albumin With oxidation by <em>m</em>-CPBA</th></tr></thead><tbody><tr><td valign="top" align="center" rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1">Acid hydrolysis (fmol PhIP/pmol albumin)</td><td valign="top" align="center" rowspan="1" colspan="1">Pronase E digestion (fmol PhIP/pmol albumin)</td><td valign="top" align="center" rowspan="1" colspan="1">Acid hydrolysis (fmol PhIP/pmol albumin)</td><td valign="top" align="center" rowspan="1" colspan="1">Pronase E digestion (fmol PhIP/pmol albumin)</td></tr><tr><td valign="top" align="center" rowspan="1" colspan="1">PhIP</td><td valign="top" align="center" rowspan="1" colspan="1">88&#x000b1;0.8</td><td valign="top" align="center" rowspan="1" colspan="1">64&#x000b1;14.6</td><td valign="top" align="center" rowspan="1" colspan="1">1.3&#x000b1;0.1</td><td valign="top" align="center" rowspan="1" colspan="1">3.3&#x000b1;0.3</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140108260166208"><a target="object" rel="noopener" href="/pmc/articles/PMC5375106/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN2"><sup>a</sup>The labile adduct of PhIP recovered was measured after acid hydrolysis (0.1N HCl, 37&#x000b0;C, 60 min) or digestion with pronase E for 18 h. Values are reported as the mean &#x000b1; SD (<em>n</em>=3).</div></div></div></div><div id="S20" class="sec"><h3 id="S20title">Validation of the Method</h3><p id="P28" class="p p-first">The levels of PhIP used for the calibration curve were chosen based on the level of [<sup>14</sup>C]-PhIP bound to albumin <em>in vivo</em> by AMS measurements.<sup><a href="#R24" rid="R24" class=" bibr popnode">24</a></sup> Seven calibrant points, ranging from 0, 50 up to 1250 fg PhIP/mg albumin, were constructed for the calibration curve. [<sup>2</sup>H<sub>3</sub>C]-PhIP was used as the internal standard and spiked at level of 500 fg/mg albumin. The linearity of calibration curve was confirmed by the slope (<em>s</em> = 1.03 &#x000b1; 0.0072) and linear regression value (<em>r</em><sup>2</sup> = 0.9997) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supporting information, Fig. S-3</a>). The [<sup>14</sup>C]-PhIP of high specific activity used for the human study was no longer available, therefore, the calibration curves, limit of detection (LOD), and limit of quantification (LOQ) values were determined with unlabeled PhIP. The ions at <em>m/z</em> 168.0682 and 183.0791 (and the ion at <em>m/z</em> 185.0823 for [<sup>14</sup>C]-PhIP) were used to determine the LOD and LOQ. Since no measurable ions <em>m/z</em> 168.0682, 183.0791, and 185.0823 were observed in the unspiked sample at the MS<sup>3</sup> scan stage using high-resolution Orbitrap (5 ppm mass tolerance), the LOD and LOQ were estimated by the formula 3.3<em>&#x003c3;</em>/<em>s</em> and 10<em>&#x003c3;</em>/<em>s</em>, respectively (<em>&#x003c3;</em> is the standard deviation of the slope (<em>s</em>) of calibration curve).<sup><a href="#R31" rid="R31" class=" bibr popnode">31</a>,<a href="#R32" rid="R32" class=" bibr popnode">32</a></sup> The LOD was estimated at 11 fg PhIP/mg albumin, and the LOQ was estimated at 33 fg PhIP/mg albumin. Approximately 12&#x02013;13 scans were acquired across the full width of the peak of the calibrant spiked with the lowest amount of PhIP (50 fg PhIP/mg albumin). The spiking of commercial albumin extracts with [<sup>14</sup>C]-PhIP (10 mCi/mmol) provided similar responses at the LOD and LOQ values (data not shown).</p><p id="P29" class="p p-last">The amount of plasma available from treated volunteers was limited, thus we conducted the validation of the analytical method with commercial human plasma spiked with <em>in vitro</em> HONH-PhIP modified human albumin, which was diluted with commercial plasma containing non-detectable levels of PhIP modified albumin. The level of modification was 675 fg PhIP/mg albumin, a level that is close to those levels of [<sup>14</sup>C]-PhIP bound to albumin <em>in vivo</em> by AMS measurement.<sup><a href="#R24" rid="R24" class=" bibr popnode">24</a></sup> The method of validation was done with four independent measurements of HONH-PhIP modified albumin on three different days, following acid hydrolysis (<a href="/pmc/articles/PMC5375106/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). The accuracy was 94.7 &#x000b1; 2.5%. The SD values for the within-day and between-day precision were 15% and 14%, respectively. The reproducibility studies were further conducted with samples of albumin obtained from two volunteers dosed with [<sup>14</sup>C]-PhIP and based on three independent measurements on two different days (<a href="/pmc/articles/PMC5375106/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>). The values of within-day and between-day precision (%CV) were 3.9 and 6.9% for volunteer 2, and 6.6 and 7.2% for volunteer 10, respectively.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p>Within-day and between-day measures of acid-labile PhIP (fg/mg albumin) recovered from spiked HONH-PhIP modified albumin diluted into commercial plasma.</p></div><div class="xtable"><table frame="hsides" rules="none" class="rendered small default_table"><thead><tr><th valign="top" align="center" rowspan="1" colspan="1"></th><th valign="top" align="center" rowspan="1" colspan="1">Spiked PhIP level (fg PhlP/mg albumin)</th><th valign="top" align="center" rowspan="1" colspan="1">Day 1</th><th valign="top" align="center" rowspan="1" colspan="1">Day 2</th><th valign="top" align="center" rowspan="1" colspan="1">Day 3</th><th valign="top" align="center" rowspan="1" colspan="1">CV(%) within-day<sup>a</sup></th><th valign="top" align="center" rowspan="1" colspan="1">CV(%) between-day<sup>a</sup></th></tr></thead><tbody><tr><td valign="top" align="center" rowspan="1" colspan="1">Mean</td><td valign="top" align="center" rowspan="1" colspan="1">675</td><td valign="top" align="center" rowspan="1" colspan="1">618(92%)<sup>b</sup></td><td valign="top" align="center" rowspan="1" colspan="1">642(95%)<sup>b</sup></td><td valign="top" align="center" rowspan="1" colspan="1">658(97%)<sup>b</sup></td><td valign="top" align="center" rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1"></td></tr><tr><td valign="top" align="center" rowspan="1" colspan="1">SD</td><td valign="top" align="center" rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1">127</td><td valign="top" align="center" rowspan="1" colspan="1">64</td><td valign="top" align="center" rowspan="1" colspan="1">83</td><td valign="top" align="center" rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1"></td></tr><tr><td valign="top" align="center" rowspan="1" colspan="1">RSD</td><td valign="top" align="center" rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1">21</td><td valign="top" align="center" rowspan="1" colspan="1">9</td><td valign="top" align="center" rowspan="1" colspan="1">13</td><td valign="top" align="center" rowspan="1" colspan="1">15</td><td valign="top" align="center" rowspan="1" colspan="1">14</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140108302912608"><a target="object" rel="noopener" href="/pmc/articles/PMC5375106/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN3"><sup>a</sup>Within-day and between-day estimates were conducted with four independent measurements on three different days.</div><div id="TFN4"><sup>b</sup>Values in parentheses represent accuracy of the method.</div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p>Within-day and between-day measures of [<sup>14</sup>C]-PhIP recovered from plasma of subjects 2 and 10 after acid hydrolysis.</p></div><div class="xtable"><table frame="below" rules="none" class="rendered small default_table"><thead><tr><th colspan="6" valign="bottom" align="center" rowspan="1">[<sup>14</sup>C]-PhIP (fg/mg albumin)
<hr /></th></tr><tr><th valign="top" align="center" rowspan="1" colspan="1">Subject</th><th valign="top" align="center" rowspan="1" colspan="1"></th><th valign="top" align="center" rowspan="1" colspan="1">Day 1</th><th valign="top" align="center" rowspan="1" colspan="1">Day 2</th><th valign="top" align="center" rowspan="1" colspan="1">CV (%) within-day<sup>a</sup></th><th valign="top" align="center" rowspan="1" colspan="1">CV (%) between-day<sup>a</sup></th></tr></thead><tbody><tr><td valign="top" align="center" rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1">Mean</td><td valign="top" align="center" rowspan="1" colspan="1">50</td><td valign="top" align="center" rowspan="1" colspan="1">47</td><td valign="top" align="center" rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1"></td></tr><tr><td valign="top" align="center" rowspan="1" colspan="1">2</td><td valign="top" align="center" rowspan="1" colspan="1">SD</td><td valign="top" align="center" rowspan="1" colspan="1">2</td><td valign="top" align="center" rowspan="1" colspan="1">2</td><td valign="top" align="center" rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1"></td></tr><tr><td valign="top" align="center" rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1">RSD</td><td valign="top" align="center" rowspan="1" colspan="1">5</td><td valign="top" align="center" rowspan="1" colspan="1">5</td><td valign="top" align="center" rowspan="1" colspan="1">4.6</td><td valign="top" align="center" rowspan="1" colspan="1">7.2</td></tr><tr><td valign="top" align="center" rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1">Mean</td><td valign="top" align="center" rowspan="1" colspan="1">52</td><td valign="top" align="center" rowspan="1" colspan="1">49</td><td valign="top" align="center" rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1"></td></tr><tr><td valign="top" align="center" rowspan="1" colspan="1">10</td><td valign="top" align="center" rowspan="1" colspan="1">SD</td><td valign="top" align="center" rowspan="1" colspan="1">2</td><td valign="top" align="center" rowspan="1" colspan="1">5</td><td valign="top" align="center" rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1"></td></tr><tr><td valign="top" align="center" rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1">RSD</td><td valign="top" align="center" rowspan="1" colspan="1">5</td><td valign="top" align="center" rowspan="1" colspan="1">11</td><td valign="top" align="center" rowspan="1" colspan="1">7.9</td><td valign="top" align="center" rowspan="1" colspan="1">8.4</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140108299009216"><a target="object" rel="noopener" href="/pmc/articles/PMC5375106/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN5"><sup>a</sup>Within-day and between-day measures were conducted with three independent measurements on two different days.</div></div></div></div><div id="S21" class="sec"><h3 id="S21title">Acid Hydrolysis of Albumin and Measurement of [<sup>14</sup>C]-PhIP Bound Albumin in Human Volunteers</h3><p id="P30" class="p p-first-last">We employed the UPLC-ESI/MS<sup>3</sup> method to measure [<sup>14</sup>C]-PhIP recovered from hydrolysis of albumin purified from plasma of four volunteers. The specific activity of [<sup>14</sup>C]-PhIP in capsules taken by human volunteers was 42 or 56 mCi/mmol (67% and 90% labeled, respectively).<sup><a href="#R24" rid="R24" class=" bibr popnode">24</a></sup> We monitored [<sup>14</sup>C]-PhIP by SIM (<em>m/z</em> 220 &#x02013; 230) following SPE of the acid hydrolyzed extract of albumin (0.1 N HCl, 37 &#x000b0;C for 1 h) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supporting information Fig. S4</a>). The mass chromatograms at the SIM stage of (<a href="#SD1" rid="SD1" class=" supplementary-material">Fig. S4A, C, E</a>) [<sup>14</sup>C]-PhIP ([M+H]<sup>+</sup>
<em>m/z</em> 227.1167), and (S4B, D, F) [<sup>2</sup>H<sub>3</sub>C]-PhIP ([M+H]<sup>+</sup>
<em>m/z</em> 228.1323 with mass tolerances set at 2, 5 or 10 ppm (<a href="#SD1" rid="SD1" class=" supplementary-material">Fig. S4</a>) revealed high levels of isobaric interferences, particularly in the <em>m/z</em> window for [<sup>14</sup>C]-PhIP, even at 60,000 mass resolution (<a href="#SD1" rid="SD1" class=" supplementary-material">Supporting information Fig. S4A, C, E</a>). Thus, multistage scanning (MS<sup>2</sup> and MS<sup>3</sup>) was used to characterize and identify [<sup>14</sup>C]-PhIP recovered from albumin. The reconstructed ion chromatograms of [<sup>14</sup>C]-PhIP and [<sup>2</sup>H<sub>3</sub>C]-PhIP at MS<sup>2</sup> and MS<sup>3</sup> stages from acid hydrolyzed albumin (0.1 N HCl, 37 &#x000b0;C for 1 h) obtained from commercial plasma and plasma of a volunteer who was administered with [<sup>14</sup>C]-PhIP are shown in <a href="/pmc/articles/PMC5375106/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>. Since the [<sup>14</sup>C]-PhIP used in the study was largely carrier free, unlabeled PhIP was not monitored. The product ions of [<sup>14</sup>C]-PhIP were not detected in the extract of commercial human plasma (<a href="/pmc/articles/PMC5375106/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figs 2A</span></a> and <a href="/pmc/articles/PMC5375106/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and2E),</span></span><span>2E</span></a>), but the ions of [<sup>14</sup>C]-PhIP were readily detected in plasma from subjects at 24 h following dosing (<a href="/pmc/articles/PMC5375106/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figs 2B</span></a> and <a href="/pmc/articles/PMC5375106/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and2F).</span></span><span>2F</span></a>). Some isobaric interferences are seen in the reconstructed ion chromatogram of [<sup>14</sup>C]-PhIP at the MS<sup>2</sup> scan stage (scan resolution 15,000) even when the mass tolerance is set at 2 ppm (<a href="/pmc/articles/PMC5375106/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2B</span></a>). The isobaric interference is greatly reduced in the reconstructed ion chromatogram at the MS<sup>3</sup> scan stage (<a href="/pmc/articles/PMC5375106/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2F</span></a>). The mass spectra of [<sup>14</sup>C]-PhIP and its internal standard [<sup>2</sup>H<sub>3</sub>C]-PhIP in human plasma are shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Supporting information, Fig. S5</a>. The product ion spectra at the MS<sup>2</sup> and MS<sup>3</sup> scan stages are in excellent agreement to the mass spectra of the reference standards (<a href="/pmc/articles/PMC5375106/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>).</p></div><div id="S22" class="sec"><h3 id="S22title">Kinetics and Recovery of [<sup>14</sup>C]-PhIP Bound to Albumin in Human Volunteers</h3><p id="P31" class="p p-first-last">A kinetics experiment was conducted <em>in vitro</em> to determine the effect of temperature and duration of acid hydrolysis on the recovery of [<sup>14</sup>C]-PhIP. In a plasma sample of a volunteer, which was collected 24 h after dosing, [<sup>14</sup>C]-PhIP was below the LOD prior to treatment of albumin with acid or pronase E digestion (<a href="/pmc/articles/PMC5375106/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>). The amount of acid-labile [<sup>14</sup>C]-PhIP released from albumin was estimated at a level of 16 fg/mg albumin following acid hydrolysis (0.1 N HCl) at 37 &#x000b0;C for 30 min, and the amount of the released [<sup>14</sup>C]-PhIP peaked at ~54 fg/mg albumin at 60 min (<a href="/pmc/articles/PMC5375106/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>). The level of [<sup>14</sup>C]-PhIP was not significantly different in the samples after hydrolysis at 37 &#x000b0;C for 60 and 180 min (<a href="/pmc/articles/PMC5375106/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>), or following acid hydrolysis at 65 &#x000b0;C for 180 min (data not shown, p &#x0003e; 0.05).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC5375106/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140108263414176"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms854476f3.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is nihms854476f3.jpg" src="/pmc/articles/PMC5375106/bin/nihms854476f3.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140108263414176"><a target="object" rel="noopener" href="/pmc/articles/PMC5375106/figure/F3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC5375106/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p>Kinetics study on acid hydrolysis of [<sup>14</sup>C]-PhIP-albumin obtained following purification of albumin from plasma of volunteer 10 given a capsule containing 42 mCi/mmol [<sup>14</sup>C]-PhIP. Each time point was measured in triplicate and reported as the mean &#x000b1; SD. Transitions of [<sup>14</sup>C]-PhIP were monitored ([M+H]<sup>+</sup>
<em>m/z</em> 227.1 &#x0003e; 212.0932&#x0003e; 168.0682, 183.0791, 185.0823). The mass extraction window was &#x000b1; 5 ppm.</p></div></div></div></div><div id="S23" class="sec"><h3 id="S23title">Levels of [<sup>14</sup>C]-PhIP Adduct Formation with Albumin in Humans</h3><p id="P32" class="p p-first-last">The levels of [<sup>14</sup>C]-PhIP in albumin purified from the plasma of four volunteers following acid hydrolysis or proteolytic digestion with pronase E is reported in <a href="/pmc/articles/PMC5375106/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>. The amount of acid labile [<sup>14</sup>C]-PhIP bound to albumin ranged from 28 to 57 fg/mg albumin collected 24 h after dosing (T<sub>24</sub>). These estimates were between 7.2 and 21.3% of the amount of [<sup>14</sup>C]-PhIP bound to albumin based on AMS measurements performed when the samples were originally collected. We also examined for 5-HO-[<sup>14</sup>C]-PhIP following acid or pronase treatment of albumin, using 5-HO-[<sup>2</sup>H<sub>3</sub>C]-PhIP as an internal standard. Since the same amounts of [<sup>2</sup>H<sub>3</sub>C]-PhIP and 5-HO-[<sup>2</sup>H<sub>3</sub>C]-PhIP injected for UPLC/MS analysis yield similar ion counts of peak areas at MS<sup>3</sup> scan stage (with no isobaric interferences), we estimate the values of LOD and LOQ values of 5-HO-[<sup>2</sup>H<sub>3</sub>]-PhIP to be close to that of PhIP and [<sup>14</sup>C]-PhIP. 5-HO-[<sup>14</sup>C]-PhIP was below the LOD in human plasma of these subject (&#x0003c; 11 fg/mg albumin) (<a href="#SD1" rid="SD1" class=" supplementary-material">Fig. S-6</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T5"><h3>Table 5</h3><!--caption a7--><div class="caption"><p>An estimate of [<sup>14</sup>C]-PhIP-albumin adduct level from subjects administered [<sup>14</sup>C]-PhIP in vivo<sup>a</sup><sup>,</sup><sup>b</sup></p></div><div class="xtable"><table frame="below" rules="groups" class="rendered small default_table"><thead><tr><th valign="top" align="center" rowspan="1" colspan="1"></th><th valign="top" align="center" rowspan="1" colspan="1"></th><th valign="top" align="center" rowspan="1" colspan="1">UPLC/MS<sup>3</sup></th><th valign="top" align="center" rowspan="1" colspan="1">AMS</th><th valign="top" align="center" rowspan="1" colspan="1"></th></tr><tr><th valign="top" align="center" rowspan="1" colspan="1">Subject</th><th valign="top" align="center" rowspan="1" colspan="1">Treatment</th><th valign="top" align="center" rowspan="1" colspan="1">Acid Labile ([<sup>14</sup>C]-PhlP fg/mg albumin)</th><th valign="top" align="center" rowspan="1" colspan="1">([<sup>14</sup>C]-PhlP bound fg/mg albumin)</th><th valign="top" align="center" rowspan="1" colspan="1">% [<sup>14</sup>C]-PhlP bound recovered as [<sup>14</sup>C]-PhlP</th></tr></thead><tbody><tr><td valign="top" align="center" rowspan="1" colspan="1">1, T<sub>0</sub></td><td valign="top" align="center" rowspan="1" colspan="1">Acid hydrolysis or pronase E digestion</td><td valign="top" align="center" rowspan="1" colspan="1">ND</td><td valign="top" align="center" rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1"></td></tr><tr><td valign="top" align="center" rowspan="1" colspan="1">1, T<sub>24</sub></td><td valign="top" align="center" rowspan="1" colspan="1">Acid hydrolysis</td><td valign="top" align="center" rowspan="1" colspan="1">57 &#x000b1;2</td><td valign="top" align="center" rowspan="1" colspan="1">392 &#x000b1; 4</td><td valign="top" align="center" rowspan="1" colspan="1">14.5</td></tr><tr><td valign="top" align="center" rowspan="1" colspan="1">1, T<sub>24</sub></td><td valign="top" align="center" rowspan="1" colspan="1">Pronase E digestion</td><td valign="top" align="center" rowspan="1" colspan="1">44&#x000b1;3</td><td valign="top" align="center" rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1"></td></tr><tr><td valign="top" align="center" rowspan="1" colspan="1">2, T<sub>0</sub></td><td valign="top" align="center" rowspan="1" colspan="1">Acid hydrolysis &#x00026; pronase E digestion</td><td valign="top" align="center" rowspan="1" colspan="1">ND</td><td valign="top" align="center" rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1"></td></tr><tr><td valign="top" align="center" rowspan="1" colspan="1">2, T<sub>24</sub></td><td valign="top" align="center" rowspan="1" colspan="1">Acid hydrolysis</td><td valign="top" align="center" rowspan="1" colspan="1">49 &#x000b1;3</td><td valign="top" align="center" rowspan="1" colspan="1">681 &#x000b1; 10</td><td valign="top" align="center" rowspan="1" colspan="1">7.2</td></tr><tr><td valign="top" align="center" rowspan="1" colspan="1">2, T<sub>24</sub></td><td valign="top" align="center" rowspan="1" colspan="1">Pronase E digestion</td><td valign="top" align="center" rowspan="1" colspan="1">38 &#x000b1;3</td><td valign="top" align="center" rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1"></td></tr><tr><td valign="top" align="center" rowspan="1" colspan="1">7, T<sub>0</sub></td><td valign="top" align="center" rowspan="1" colspan="1">Acid hydrolysis or pronase E digestion</td><td valign="top" align="center" rowspan="1" colspan="1">ND</td><td valign="top" align="center" rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1"></td></tr><tr><td valign="top" align="center" rowspan="1" colspan="1">7, T<sub>24</sub></td><td valign="top" align="center" rowspan="1" colspan="1">Acid hydrolysis</td><td valign="top" align="center" rowspan="1" colspan="1">28 <em>&#x000b1;</em> 2</td><td valign="top" align="center" rowspan="1" colspan="1">244 &#x000b1;3</td><td valign="top" align="center" rowspan="1" colspan="1">11.5</td></tr><tr><td valign="top" align="center" rowspan="1" colspan="1">10, T<sub>0</sub></td><td valign="top" align="center" rowspan="1" colspan="1">Acid hydrolysis</td><td valign="top" align="center" rowspan="1" colspan="1">ND</td><td valign="top" align="center" rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1"></td></tr><tr><td valign="top" align="center" rowspan="1" colspan="1">10, T<sub>24</sub></td><td valign="top" align="center" rowspan="1" colspan="1">Acid hydrolysis</td><td valign="top" align="center" rowspan="1" colspan="1">51&#x000b1;4</td><td valign="top" align="center" rowspan="1" colspan="1">240 &#x000b1;4</td><td valign="top" align="center" rowspan="1" colspan="1">21.3</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140108261368544"><a target="object" rel="noopener" href="/pmc/articles/PMC5375106/table/T5/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN6"><sup>a</sup>Albumin was purified from plasma at T<sub>0</sub> or T<sub>24</sub> h and treated with 0.1 N HCl or pronase E digestion.</div><div id="TFN7"><p class="p p-first-last">Values of acid-labile [<sup>14</sup>C]-PhIP measured by UPLC/MS<sup>3</sup> are reported as the mean &#x000b1; SD (<em>n</em>=3). For subjects 1 and 2, the AMS measurement values are the mean &#x000b1; SD of four independent analyses. The AMS analyses for subjects 7 and 10 are from single analyses (&#x000b1; SD of the AMS measurement). ND, not detected, below the LOD.</p></div><div id="TFN8"><sup>b</sup>Albumin adduct data obtained by AMS represents the [<sup>14</sup>C]-PhIP component of adduct levels and not unlabeled adducts. In the Dingley paper,<sup><a href="#R24" rid="R24" class=" bibr popnode">24</a></sup> the total adduct levels (labeled and unlabeled PhIP) were reported.</div></div></div></div><div id="S24" class="sec sec-last"><h3 id="S24title">Stability of [<sup>14</sup>C]-PhIP-Acid Labile Albumin Adducts</h3><p id="P33" class="p p-first-last">The stability of the presumed [<sup>14</sup>C]-PhIP-sulfinamide adduct formed <em>in vivo</em> was examined over time <em>in vitro</em> at 37 &#x000b0;C, following a 2-fold dilution with commercial human plasma and measurement, by UPLC-ESI/MS<sup>3</sup>, of the remaining acid-labile [<sup>14</sup>C]-PhIP bound to albumin over seven days. We acknowledge that these samples were stored for nearly 20 years at &#x02212;80 &#x000b0;C, and the stability of acid-labile adducts over this prolonged period of storage has not been determined. The level of acid-labile [<sup>14</sup>C]-PhIP bound to albumin decreased by 17% and 39% after human plasma was incubated for 6 h and 24 h, and then by 80% after 7 days, under neutral pH conditions (<a href="/pmc/articles/PMC5375106/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a>). The curve approximates a first-order rate decay, and the half-life of labile [<sup>14</sup>C]-PhIP bound to albumin is estimated at ~50 h. However, the inclusion of the last time point at 7 days may indicate that the rate of decay is biphasic. Additional time points would be required to fully characterize the decay curve of the labile [<sup>14</sup>C]-PhIP albumin adduct. We also incubated HONH-PhIP modified with albumin <em>in vitro</em> and diluted it in human plasma; the level of PhIP bound to albumin (primarily as the sulfinamide) also decreased at a similar rate to the adduct formed <em>in vivo</em> (data not shown). These data suggest that the acid-labile albumin adducts of PhIP may have a shorter half-life <em>in vivo</em> than albumin, which is approximately 20 days.<sup><a href="#R33" rid="R33" class=" bibr popnode">33</a></sup></p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC5375106/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140108263410144"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms854476f4.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is nihms854476f4.jpg" src="/pmc/articles/PMC5375106/bin/nihms854476f4.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140108263410144"><a target="object" rel="noopener" href="/pmc/articles/PMC5375106/figure/F4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC5375106/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><p>The level of labile [<sup>14</sup>C]-PhIP remaining bound to albumin of volunteer 7 given a capsule containing 42 mCi/mmol [<sup>14</sup>C]-PhIP as a function of time when the human plasma was incubated at 37 &#x000b0;C in vitro, following dilution with commercial plasma. Values represent the mean &#x000b1; SD (<em>n</em>=3).</p></div></div></div></div></div><div id="S25" class="tsec sec"><h2 class="head no_bottom_margin" id="S25title">DISCUSSION</h2><p id="P34" class="p p-first">The Cys<sup>34</sup> of rodent and human albumin reacts with N-oxidized metabolites of PhIP,<sup><a href="#R16" rid="R16" class=" bibr popnode">16</a>,<a href="#R20" rid="R20" class=" bibr popnode">20</a>,<a href="#R21" rid="R21" class=" bibr popnode">21</a></sup> and many other electrophiles to form adducts.<sup><a href="#R12" rid="R12" class=" bibr popnode">12</a></sup> UPLC-MS<sup>n</sup> characterization of proteolytic digest of human albumin modified <em>in vitro</em> with N-oxidized metabolites of PhIP revealed that the Cys<sup>34</sup> residue is the major site of adduction and forms the sulfinamide adduct with PhIP (Scheme 1).<sup><a href="#R20" rid="R20" class=" bibr popnode">20</a>,<a href="#R21" rid="R21" class=" bibr popnode">21</a></sup> We measured the presumed Cys containing sulfinamide (C-[SO]-[<sup>14</sup>C]-PhIP) formed <em>in vivo</em> indirectly, following mild acid treatment or pronase digestion of the purified albumin. Acid or pronase treatment results in hydrolysis of the sulfinamide linkage and regenerates PhIP. The C* sulfonamide (C-[SO<sub>2</sub>]-PhIP) adduct is stable toward acid or protease treatment, and does not produce PhIP. The proposed the S-N linked adduct of the PhIP sulfenamide undergoes hydrolysis by mild acid treatment or proteolysis to produce both PhIP and 5-HO-[<sup>14</sup>C]-PhIP (Scheme 1). However, we did not detect 5-HO-[<sup>14</sup>C]-PhIP in the human samples. In contrast, 5-HO-PhIP was identified as a major product recovered in pronase hydrolysates of rat albumin reacted with <em>N</em>-acetoxy-PhIP,<sup><a href="#R16" rid="R16" class=" bibr popnode">16</a></sup> Our laboratory also identified 5-HO-PhIP in the pronase digest of albumin recovered from human hepatocytes treated with PhIP,<sup><a href="#R21" rid="R21" class=" bibr popnode">21</a></sup> and from human albumin treated with <em>N</em>-acetoxy-PhIP in vitro, but not with albumin modified with HONH-PhIP.<sup><a href="#R18" rid="R18" class=" bibr popnode">18</a>,<a href="#R21" rid="R21" class=" bibr popnode">21</a></sup> Thus, we deduce that a large portion of the acid labile [<sup>14</sup>C]-PhIP bound to albumin in humans is likely present as the sulfinamide linked adduct, which is formed by reaction of nitroso-PhIP with Cys<sup>34</sup> of albumin (Scheme 1).<sup><a href="#R18" rid="R18" class=" bibr popnode">18</a>,<a href="#R21" rid="R21" class=" bibr popnode">21</a></sup> However, these plasma samples were stored at &#x02212;80&#x000b0;C ever since the study was conducted in 1997 &#x02013; 1998,<sup><a href="#R24" rid="R24" class=" bibr popnode">24</a></sup> a labile [<sup>14</sup>C]-PhIP sulfenamide may have undergone hydrolysis over this long time period.</p><p id="P35">Our studies on the characterization of the reactivity of the N-oxidized metabolites of PhIP with albumin <em>in vitro</em> revealed that &#x0003e;60% of the PhIP bound to albumin was acid-labile and recovered as PhIP (Scheme 1).<sup><a href="#R20" rid="R20" class=" bibr popnode">20</a></sup> A very high percentage of this acid labile PhIP bound to albumin occurs at Cys<sup>34</sup> since pretreatment of albumin with 4-chloromercuribenzoic acid, a selective agent that binds to Cys<sup>34</sup> of albumin,<sup><a href="#R34" rid="R34" class=" bibr popnode">34</a></sup> abolished most of the PhIP species bound as an acid labile albumin adduct. The percent of [<sup>14</sup>C]-PhIP bound to albumin in humans and recovered as PhIP, following protein purification and acid treatment, is considerably lower (7.2 &#x02013; 21.3%) than those levels observed with albumin modified in vitro with N-oxidized PhIP metabolites. We note that the stability of the sulfinamide C-[SO]-[<sup>14</sup>C]-PhIP has not been assessed under prolonged storage conditions and some portion of the adduct may have undergone hydrolysis. A portion of the sulfinamide C-[SO]-[<sup>14</sup>C]-PhIP also could have undergone oxidation to form the acid-stable sulfonamide C-[SO<sub>2</sub>]-[<sup>14</sup>C]-PhIP during prolonged storage (Scheme 1).<sup><a href="#R19" rid="R19" class=" bibr popnode">19</a></sup> Other sites of PhIP adduct formation with albumin also occur. <em>N</em>-sulfooxy-2-amino-1-methyl-6-phenylimidazo[4,5-<em>b</em>]pyridine (<em>N</em>-sulfooxy-PhIP), a likely genotoxic metabolite of PhIP formed in humans,<sup><a href="#R35" rid="R35" class=" bibr popnode">35</a></sup> reacts in vitro with the sole tryptophan (Trp<sup>214</sup>) residue of human albumin to form a minor adduct that is recovered by trypsin digestion as AW<sup>*[PhIP]</sup>AVAR.<sup><a href="#R21" rid="R21" class=" bibr popnode">21</a></sup> Another adduction product was detected at His residue(s) following pronase digestion of albumin modified with <em>N</em>-sulfooxy-PhIP.<sup><a href="#R21" rid="R21" class=" bibr popnode">21</a></sup> The proposed structures of these adducts are depicted in <a href="/pmc/articles/PMC5375106/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5</span></a>. Neither adduct undergoes proteolysis to regenerate PhIP, although the chemical stability of the adducts towards acid treatment was not investigated.<sup><a href="#R21" rid="R21" class=" bibr popnode">21</a></sup> The amounts of AW<sup>*[PhIP]</sup>AVA and His albumin adducts formed <em>in vivo</em> may be larger than those which occur <em>in vitro</em>. Because of the very large amounts of non-modified peptides in the proteolytic digest of <em>in vivo</em> samples, we do not yet have selective enrichment methods to isolate Trp and His adducts of PhIP for measurement by UPLC-ESI/MS<sup>n</sup>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC5375106/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140108263405936"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms854476f5.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is nihms854476f5.jpg" src="/pmc/articles/PMC5375106/bin/nihms854476f5.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140108263405936"><a target="object" rel="noopener" href="/pmc/articles/PMC5375106/figure/F5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC5375106/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5</a></div><!--caption a7--><div class="caption"><p>Proposed structures of PhIP adduct formed at Trp<sup>214</sup> and His residues of albumin. Mass spectral data support structures formed between the C-2 indole atom of Trp and the C-2 imidazole atom of histidine with the N-oxidized NH<sub>2</sub>-group of PhIP.<sup><a href="#R21" rid="R21" class=" bibr popnode">21</a></sup></p></div></div></div><p id="P36" class="p p-last">An LC-MS/MS method with a triple quadrupole MS was recently developed to quantify the level of HAAs released from acid treatment of albumin in humans: the LOD and LOQ values reported for PhIP were 125 and 375 fg PhIP/mg albumin, respectively (the albumin content in human plasma ~40 mg/mL).<sup><a href="#R26" rid="R26" class=" bibr popnode">26</a></sup> The authors concluded that the formation of acid-labile albumin HAA adducts was very low and would not be viable biomarkers to biomonitor HAA exposure in humans. However, the volunteers were on a free-choice diet, and the amount of PhIP ingested was not known. Our sensitive UPLC-MS<sup>n</sup> method employing the high resolution accurate mass measurement of the Orbitrap at the MS<sup>3</sup> scan stage has superior sensitivity to the triple quadrupole MS method; the LOD and LOQ values are 11 and 33 fg PhIP/mg albumin, respectively. Our kinetic studies <em>in vitro</em> with [<sup>14</sup>C]-PhIP bound to albumin in isolated plasma reveal the adduct has a half-life of about 50 h, which is considerably less than the ~20 day half-life of albumin in humans.<sup><a href="#R33" rid="R33" class=" bibr popnode">33</a></sup> A kinetics study measuring PhIP albumin adduct formation and decay over time with healthy subjects on a diet containing known amounts of PhIP is required to determine if the acid labile PhIP albumin adduct can be implemented in human biomonitoring studies.</p></div><div id="S26" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="S26title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supporting Info</h4><div class="sup-box half_rhythm" id="d36e1957"><a href="/pmc/articles/PMC5375106/bin/NIHMS854476-supplement-Supporting_Info.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(424K, docx)</sup></div></div></div><div id="S27" class="tsec sec"><h2 class="head no_bottom_margin" id="S27title">Acknowledgments</h2><div class="sec"><p><strong>Funding Sources.</strong></p><p>This research was supported by Grant 2R01CA122320 (R.J.T.). Mass spectrometry was carried out in the Analytical Biochemistry Shared Resource of the Masonic Cancer Center, University of Minnesota, funded in part by Cancer Center Support Grant CA-077598.</p></div></div><div id="GL1" class="tsec sec"><h2 class="head no_bottom_margin" id="GL1title">ABBREVIATIONS</h2><div class="bk-sec"><div><table class="default-table glossary"><tbody><tr><td id="G1">AMS</td><td>accelerator mass spectrometry</td></tr><tr><td id="G2"><em>&#x003b2;</em>ME</td><td><em>&#x003b2;</em>-mercaptoethanol</td></tr><tr><td id="G3">IQ</td><td>2-amino-3-methylimidazo[4,5-<em>f</em>]quinoline PhIP, 2-amino-1-methyl-6-phenylimidazo[4,5-b]pyridine</td></tr><tr><td id="G4">5-HO-PhIP</td><td>2-amino-1-methyl-6-(5-hydroxy)-phenylimidazo[4,5-<em>b</em>]pyridine</td></tr><tr><td id="G5"><em>N</em>-acetoxy-PhIP</td><td><em>N</em>-(acetyloxy)-2-amino-1-methyl-6-phenylimidazo[4,5-<em>b</em>]pyridine</td></tr><tr><td id="G6">HONH-PhIP</td><td>2-hydroxyamino-1-methyl-6-phenylimidazo[4,5-<em>b</em>]pyridine</td></tr><tr><td id="G7">NO-PhIP</td><td>2-nitroso-1-methyl-6-phenylimidazo[4,5-<em>b</em>]pyridine</td></tr><tr><td id="G8"><em>N</em>-sulfooxy-PhIP</td><td><em>N</em>-sulfooxy-2-amino-1-methyl-6-phenylimidazo[4,5-<em>b</em>]pyridine</td></tr><tr><td id="G9">Hb</td><td>hemoglobin</td></tr><tr><td id="G10">HAA</td><td>heterocyclic aromatic amines</td></tr><tr><td id="G11">NATs</td><td><em>N</em>-acetyltransferases</td></tr><tr><td id="G12">MS</td><td>mass spectrometry</td></tr><tr><td id="G13"><em>m</em>-CPBA</td><td><em>meta</em>-chloroperoxybenzoic acid</td></tr><tr><td id="G14">SIM</td><td>selected ion monitoring</td></tr><tr><td id="G15">SPE</td><td>solid phase extraction</td></tr><tr><td id="G16">SULTs</td><td>sulfotransferases</td></tr><tr><td id="G17">UPLC-MS<sup>n</sup></td><td>ultraperformance liquid chromatography-multistage scan mass spectrometry</td></tr></tbody></table></div></div></div><div id="fn-group-a.v.b" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.v.btitle">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"><p class="p p-first"><strong>Supporting Information</strong></p><p class="p p-last">Reconstructed ion chromatograms of PhIP, [<sup>2</sup>H<sub>3</sub>C]-PhIP, and [<sup>14</sup>C]-PhIP standards. Product ion spectra of [<sup>2</sup>H<sub>3</sub>C]-PhIP. Calibration curve of PhIP in plasma extract. Product ion spectra of [<sup>14</sup>C]-PhIP and [<sup>2</sup>H<sub>3</sub>C]-PhIP recovered from human plasma. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pubs.acs.org</a>.</p></p></div></div><div id="ref-list-a.v.d" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.v.dtitle">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Sugimura T, Wakabayashi K, Nakagama H, Nagao M. Heterocyclic amines: Mutagens/carcinogens produced during cooking of meat and fish. <span><span class="ref-journal">Cancer Sci. </span>2004;<span class="ref-vol">95</span>:290–299.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC11159436/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15072585" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Sci&amp;title=Heterocyclic+amines:+Mutagens/carcinogens+produced+during+cooking+of+meat+and+fish&amp;author=T+Sugimura&amp;author=K+Wakabayashi&amp;author=H+Nakagama&amp;author=M+Nagao&amp;volume=95&amp;publication_year=2004&amp;pages=290-299&amp;pmid=15072585&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Matsumoto Takashi, Yoshida Daisuke, Tomita Hideo. Determination of mutagens, amino-&#x003b1;-carbolines in grilled foods and cigarette smoke condensate. <span><span class="ref-journal">Cancer Letters. </span>1981;<span class="ref-vol">12</span>:105–110.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7272995" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Letters&amp;title=Determination+of+mutagens,+amino-&#x003b1;-carbolines+in+grilled+foods+and+cigarette+smoke+condensate&amp;author=Takashi+Matsumoto&amp;author=Daisuke+Yoshida&amp;author=Hideo+Tomita&amp;volume=12&amp;publication_year=1981&amp;pages=105-110&amp;pmid=7272995&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Felton JS, Jagerstad M, Knize MG, Skog K, Wakabayashi K.  Contents in foods, beverages and tobacco. In: Nagao M, Sugimura T, editors. <span class="ref-journal">Food Borne Carcinogens Heterocyclic Amines.</span> John Wiley &#x00026; Sons Ltd; Chichester, England: 2000. pp. 31–71. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Food+Borne+Carcinogens+Heterocyclic+Amines&amp;author=JS+Felton&amp;author=M+Jagerstad&amp;author=MG+Knize&amp;author=K+Skog&amp;author=K+Wakabayashi&amp;publication_year=2000&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Knize MG, Dolbeare FA, Carroll KL, Moore DH, Felton JS. Effect of cooking time and temperature on the heterocyclic amine content of fried beef patties. <span><span class="ref-journal">Food Chem Toxicol. </span>1994;<span class="ref-vol">32</span>:595–603.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8045472" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Food+Chem+Toxicol&amp;title=Effect+of+cooking+time+and+temperature+on+the+heterocyclic+amine+content+of+fried+beef+patties&amp;author=MG+Knize&amp;author=FA+Dolbeare&amp;author=KL+Carroll&amp;author=DH+Moore&amp;author=JS+Felton&amp;volume=32&amp;publication_year=1994&amp;pages=595-603&amp;pmid=8045472&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Sinha R, Rothman N, Brown ED, Salmon CP, Knize MG, Swanson CS, Rossi SC, Mark SD, Levander OA, Felton JS. High concentrations of the carcinogen 2-amino-1-methyl-6-phenylimidazo[4,5-<em>b</em>]pyridine (PhIP) occur in chicken but are dependent on the cooking method. <span><span class="ref-journal">Cancer Res. </span>1995;<span class="ref-vol">55</span>:4516–4519.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7553619" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=High+concentrations+of+the+carcinogen+2-amino-1-methyl-6-phenylimidazo[4,5-b]pyridine+(PhIP)+occur+in+chicken+but+are+dependent+on+the+cooking+method&amp;author=R+Sinha&amp;author=N+Rothman&amp;author=ED+Brown&amp;author=CP+Salmon&amp;author=MG+Knize&amp;volume=55&amp;publication_year=1995&amp;pages=4516-4519&amp;pmid=7553619&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Turesky RJ, Le Marchand L. Metabolism and biomarkers of heterocyclic aromatic amines in molecular epidemiology studies: lessons learned from aromatic amines. <span><span class="ref-journal">Chem Res Toxicol. </span>2011;<span class="ref-vol">24</span>:1169–1214.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3156293/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21688801" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem+Res+Toxicol&amp;title=Metabolism+and+biomarkers+of+heterocyclic+aromatic+amines+in+molecular+epidemiology+studies:+lessons+learned+from+aromatic+amines&amp;author=RJ+Turesky&amp;author=L+Le+Marchand&amp;volume=24&amp;publication_year=2011&amp;pages=1169-1214&amp;pmid=21688801&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Bouvard V, Loomis D, Guyton KZ, Grosse Y, Ghissassi FE, Benbrahim-Tallaa L, Guha N, Mattock H, Straif K. Carcinogenicity of consumption of red and processed meat. <span><span class="ref-journal">Lancet Oncol. </span>2015;<span class="ref-vol">16</span>:1599–1600.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26514947" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Oncol&amp;title=Carcinogenicity+of+consumption+of+red+and+processed+meat&amp;author=V+Bouvard&amp;author=D+Loomis&amp;author=KZ+Guyton&amp;author=Y+Grosse&amp;author=FE+Ghissassi&amp;volume=16&amp;publication_year=2015&amp;pages=1599-1600&amp;pmid=26514947&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Jarabek AM, Pottenger LH, Andrews LS, Casciano D, Embry MR, Kim JH, Preston RJ, Reddy MV, Schoeny R, Shuker D, Skare J, Swenberg J, Williams GM, Zeiger E. Creating context for the use of DNA adduct data in cancer risk assessment: I. Data organization. <span><span class="ref-journal">Crit Rev Toxicol. </span>2009;<span class="ref-vol">39</span>:659–678.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19743944" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Crit+Rev+Toxicol&amp;title=Creating+context+for+the+use+of+DNA+adduct+data+in+cancer+risk+assessment:+I.+Data+organization&amp;author=AM+Jarabek&amp;author=LH+Pottenger&amp;author=LS+Andrews&amp;author=D+Casciano&amp;author=MR+Embry&amp;volume=39&amp;publication_year=2009&amp;pages=659-678&amp;pmid=19743944&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Himmelstein MW, Boogaard PJ, Cadet J, Farmer PB, Kim JH, Martin EA, Persaud R, Shuker DE. Creating context for the use of DNA adduct data in cancer risk assessment: II. Overview of methods of identification and quantitation of DNA damage. <span><span class="ref-journal">Crit Rev Toxicol. </span>2009;<span class="ref-vol">39</span>:679–694.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19743945" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Crit+Rev+Toxicol&amp;title=Creating+context+for+the+use+of+DNA+adduct+data+in+cancer+risk+assessment:+II.+Overview+of+methods+of+identification+and+quantitation+of+DNA+damage&amp;author=MW+Himmelstein&amp;author=PJ+Boogaard&amp;author=J+Cadet&amp;author=PB+Farmer&amp;author=JH+Kim&amp;volume=39&amp;publication_year=2009&amp;pages=679-694&amp;pmid=19743945&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Skipper PL, Peng X, SooHoo CK, Tannenbaum SR. Protein adducts as biomarkers of human carcinogen exposure. <span><span class="ref-journal">Drug Metab Rev. </span>1994;<span class="ref-vol">26</span>:111–124.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8082561" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Drug+Metab+Rev&amp;title=Protein+adducts+as+biomarkers+of+human+carcinogen+exposure&amp;author=PL+Skipper&amp;author=X+Peng&amp;author=CK+SooHoo&amp;author=SR+Tannenbaum&amp;volume=26&amp;publication_year=1994&amp;pages=111-124&amp;pmid=8082561&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Liebler DC. Proteomic approaches to characterize protein modifications: new tools to study the effects of environmental exposures. <span><span class="ref-journal">Environ Health Perspect. </span>2002;<span class="ref-vol">110</span>(Suppl 1):3–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1241143/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11834459" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Environ+Health+Perspect&amp;title=Proteomic+approaches+to+characterize+protein+modifications:+new+tools+to+study+the+effects+of+environmental+exposures&amp;author=DC+Liebler&amp;volume=110&amp;issue=Suppl+1&amp;publication_year=2002&amp;pages=3-9&amp;pmid=11834459&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Rappaport SM, Li H, Grigoryan H, Funk WE, Williams ER. Adductomics: Characterizing exposures to reactive electrophiles. <span><span class="ref-journal">Toxicol Lett. </span>2012;<span class="ref-vol">213</span>:83–90.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4758449/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21501670" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Toxicol+Lett&amp;title=Adductomics:+Characterizing+exposures+to+reactive+electrophiles&amp;author=SM+Rappaport&amp;author=H+Li&amp;author=H+Grigoryan&amp;author=WE+Funk&amp;author=ER+Williams&amp;volume=213&amp;publication_year=2012&amp;pages=83-90&amp;pmid=21501670&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Aldini G, Regazzoni L, Orioli M, Rimoldi I, Facino RM, Carini M. A tandem MS precursor-ion scan approach to identify variable covalent modification of albumin Cys34: a new tool for studying vascular carbonylation. <span><span class="ref-journal">J Mass Spectrom. </span>2008;<span class="ref-vol">43</span>:1470–1481.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18457351" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mass+Spectrom&amp;title=A+tandem+MS+precursor-ion+scan+approach+to+identify+variable+covalent+modification+of+albumin+Cys34:+a+new+tool+for+studying+vascular+carbonylation&amp;author=G+Aldini&amp;author=L+Regazzoni&amp;author=M+Orioli&amp;author=I+Rimoldi&amp;author=RM+Facino&amp;volume=43&amp;publication_year=2008&amp;pages=1470-1481&amp;pmid=18457351&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Turesky RJ, Skipper PL, Tannenbaum SR. Binding of 2-amino-3-methylimidazo[4,5-<em>f</em>]quinoline to hemoglobin and albumin in vivo in the rat. Identification of an adduct suitable for dosimetry. <span><span class="ref-journal">Carcinogenesis. </span>1987;<span class="ref-vol">8</span>:1537–1542.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3652389" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=Binding+of+2-amino-3-methylimidazo[4,5-f]quinoline+to+hemoglobin+and+albumin+in+vivo+in+the+rat.+Identification+of+an+adduct+suitable+for+dosimetry&amp;author=RJ+Turesky&amp;author=PL+Skipper&amp;author=SR+Tannenbaum&amp;volume=8&amp;publication_year=1987&amp;pages=1537-1542&amp;pmid=3652389&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Lynch AM, Murray S, Boobis AR, Davies DS, Gooderham NJ. The measurement of MeIQx adducts with mouse haemoglobin in vitro and in vivo: implications for human dosimetry. <span><span class="ref-journal">Carcinogenesis. </span>1991;<span class="ref-vol">12</span>:1067–1072.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2044186" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=The+measurement+of+MeIQx+adducts+with+mouse+haemoglobin+in+vitro+and+in+vivo:+implications+for+human+dosimetry&amp;author=AM+Lynch&amp;author=S+Murray&amp;author=AR+Boobis&amp;author=DS+Davies&amp;author=NJ+Gooderham&amp;volume=12&amp;publication_year=1991&amp;pages=1067-1072&amp;pmid=2044186&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Reistad R, Frandsen H, Grivas S, Alexander J. In vitro formation and degradation of 2-amino-1-methyl-6-phenylimidazo[4,5-<em>b</em>]pyridine (PhIP) protein adducts. <span><span class="ref-journal">Carcinogenesis. </span>1994;<span class="ref-vol">15</span>:2547–2552.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7955104" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=In+vitro+formation+and+degradation+of+2-amino-1-methyl-6-phenylimidazo[4,5-b]pyridine+(PhIP)+protein+adducts&amp;author=R+Reistad&amp;author=H+Frandsen&amp;author=S+Grivas&amp;author=J+Alexander&amp;volume=15&amp;publication_year=1994&amp;pages=2547-2552&amp;pmid=7955104&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Chepanoske CL, Brown K, Turteltaub KW, Dingley KH. Characterization of a peptide adduct formed by N-acetoxy-2-amino-1-methyl-6-phenylimidazo[4,5-<em>b</em>]pyridine (PhIP), a reactive intermediate of the food carcinogen PhIP. <span><span class="ref-journal">Food Chem Toxicol. </span>2004;<span class="ref-vol">42</span>:1367–1372.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15207388" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Food+Chem+Toxicol&amp;title=Characterization+of+a+peptide+adduct+formed+by+N-acetoxy-2-amino-1-methyl-6-phenylimidazo[4,5-b]pyridine+(PhIP),+a+reactive+intermediate+of+the+food+carcinogen+PhIP&amp;author=CL+Chepanoske&amp;author=K+Brown&amp;author=KW+Turteltaub&amp;author=KH+Dingley&amp;volume=42&amp;publication_year=2004&amp;pages=1367-1372&amp;pmid=15207388&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Peng L, Dasari S, Tabb DL, Turesky RJ. Mapping serum albumin adducts of the food-borne carcinogen 2-amino-1-methyl-6-phenylimidazo[4,5-<em>b</em>]pyridine by data-dependent tandem mass spectrometry. <span><span class="ref-journal">Chem Res Toxicol. </span>2012;<span class="ref-vol">25</span>:2179–2193.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3544159/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22827630" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem+Res+Toxicol&amp;title=Mapping+serum+albumin+adducts+of+the+food-borne+carcinogen+2-amino-1-methyl-6-phenylimidazo[4,5-b]pyridine+by+data-dependent+tandem+mass+spectrometry&amp;author=L+Peng&amp;author=S+Dasari&amp;author=DL+Tabb&amp;author=RJ+Turesky&amp;volume=25&amp;publication_year=2012&amp;pages=2179-2193&amp;pmid=22827630&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Peng L, Turesky RJ. Capturing labile sulfenamide and sulfinamide serum albumin adducts of carcinogenic arylamines by chemical oxidation. <span><span class="ref-journal">Anal Chem. </span>2013;<span class="ref-vol">85</span>:1065–1072.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3621984/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23240913" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Anal+Chem&amp;title=Capturing+labile+sulfenamide+and+sulfinamide+serum+albumin+adducts+of+carcinogenic+arylamines+by+chemical+oxidation&amp;author=L+Peng&amp;author=RJ+Turesky&amp;volume=85&amp;publication_year=2013&amp;pages=1065-1072&amp;pmid=23240913&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Peng L, Turesky RJ. Mass spectrometric characterization of 2-amino-1-methyl-6-phenylimidazo[4,5-<em>b</em>]pyridine N-oxidized metabolites bound at Cys<sup>34</sup> of human serum albumin. <span><span class="ref-journal">Chem Res Toxicol. </span>2011;<span class="ref-vol">24</span>:2004–2017.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3221794/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21916490" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem+Res+Toxicol&amp;title=Mass+spectrometric+characterization+of+2-amino-1-methyl-6-phenylimidazo[4,5-b]pyridine+N-oxidized+metabolites+bound+at+Cys34+of+human+serum+albumin&amp;author=L+Peng&amp;author=RJ+Turesky&amp;volume=24&amp;publication_year=2011&amp;pages=2004-2017&amp;pmid=21916490&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Wang Y, Peng L, Bellamri M, Langouet S, Turesky RJ. Mass spectrometric characterization of human serum albumin adducts formed with N-oxidized metabolites of 2-amino-1-methylphenylimidazo[4,5-<em>b</em>]pyridine in human plasma and hepatocytes. <span><span class="ref-journal">Chemical research in toxicology. </span>2015;<span class="ref-vol">28</span>:1045–1059.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4437839/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25815793" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chemical+research+in+toxicology&amp;title=Mass+spectrometric+characterization+of+human+serum+albumin+adducts+formed+with+N-oxidized+metabolites+of+2-amino-1-methylphenylimidazo[4,5-b]pyridine+in+human+plasma+and+hepatocytes&amp;author=Y+Wang&amp;author=L+Peng&amp;author=M+Bellamri&amp;author=S+Langouet&amp;author=RJ+Turesky&amp;volume=28&amp;publication_year=2015&amp;pages=1045-1059&amp;pmid=25815793&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Reistad R, Rossland OJ, Latva-Kala KJ, Rasmussen T, Vikse R, Becher G, Alexander J. Heterocyclic aromatic amines in human urine following a fried meat meal. <span><span class="ref-journal">Food Chem Toxicol. </span>1997;<span class="ref-vol">35</span>:945–955.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9463528" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Food+Chem+Toxicol&amp;title=Heterocyclic+aromatic+amines+in+human+urine+following+a+fried+meat+meal&amp;author=R+Reistad&amp;author=OJ+Rossland&amp;author=KJ+Latva-Kala&amp;author=T+Rasmussen&amp;author=R+Vikse&amp;volume=35&amp;publication_year=1997&amp;pages=945-955&amp;pmid=9463528&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Garner RC, Lightfoot TJ, Cupid BC, Russell D, Coxhead JM, Kutschera W, Priller A, Rom W, Steier P, Alexander DJ, Leveson SH, Dingley KH, Mauthe RJ, Turteltaub KW. Comparative biotransformation studies of MeIQx and PhIP in animal models and humans. <span><span class="ref-journal">Cancer Lett. </span>1999;<span class="ref-vol">143</span>:161–165.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10503897" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Lett&amp;title=Comparative+biotransformation+studies+of+MeIQx+and+PhIP+in+animal+models+and+humans&amp;author=RC+Garner&amp;author=TJ+Lightfoot&amp;author=BC+Cupid&amp;author=D+Russell&amp;author=JM+Coxhead&amp;volume=143&amp;publication_year=1999&amp;pages=161-165&amp;pmid=10503897&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Dingley KH, Curtis KD, Nowell S, Felton JS, Lang NP, Turteltaub KW. DNA and protein adduct formation in the colon and blood of humans after exposure to a dietary-relevant dose of 2-amino-1-methyl-6-phenylimidazo[4,5-<em>b</em>]pyridine. <span><span class="ref-journal">Cancer Epidemiol Biomarkers Prev. </span>1999;<span class="ref-vol">8</span>:507–512.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10385140" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol+Biomarkers+Prev&amp;title=DNA+and+protein+adduct+formation+in+the+colon+and+blood+of+humans+after+exposure+to+a+dietary-relevant+dose+of+2-amino-1-methyl-6-phenylimidazo[4,5-b]pyridine&amp;author=KH+Dingley&amp;author=KD+Curtis&amp;author=S+Nowell&amp;author=JS+Felton&amp;author=NP+Lang&amp;volume=8&amp;publication_year=1999&amp;pages=507-512&amp;pmid=10385140&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Magagnotti C, Orsi F, Bagnati R, Celli N, Rotilio D, Fanelli R, Airoldi L. Effect of diet on serum albumin and hemoglobin adducts of 2-amino-1-methyl-6-phenylimidazo[4,5-<em>b</em>]pyridine (PhIP) in humans. <span><span class="ref-journal">Int J Cancer. </span>2000;<span class="ref-vol">88</span>:1–6.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10962432" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Cancer&amp;title=Effect+of+diet+on+serum+albumin+and+hemoglobin+adducts+of+2-amino-1-methyl-6-phenylimidazo[4,5-b]pyridine+(PhIP)+in+humans&amp;author=C+Magagnotti&amp;author=F+Orsi&amp;author=R+Bagnati&amp;author=N+Celli&amp;author=D+Rotilio&amp;volume=88&amp;publication_year=2000&amp;pages=1-6&amp;pmid=10962432&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Cooper KM, Brennan SF, Woodside JV, Cantwell M, Guo X, Mooney M, Elliott CT, Cuskelly GJ. Acid-labile protein-adducted heterocyclic aromatic amines in human blood are not viable biomarkers of dietary exposure: A systematic study. <span><span class="ref-journal">Food Chem Toxicol. </span>2016;<span class="ref-vol">91</span>:100–107.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26993956" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Food+Chem+Toxicol&amp;title=Acid-labile+protein-adducted+heterocyclic+aromatic+amines+in+human+blood+are+not+viable+biomarkers+of+dietary+exposure:+A+systematic+study&amp;author=KM+Cooper&amp;author=SF+Brennan&amp;author=JV+Woodside&amp;author=M+Cantwell&amp;author=X+Guo&amp;volume=91&amp;publication_year=2016&amp;pages=100-107&amp;pmid=26993956&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Turesky RJ, Lang NP, Butler MA, Teitel CH, Kadlubar FF. Metabolic activation of carcinogenic heterocyclic aromatic amines by human liver and colon. <span><span class="ref-journal">Carcinogenesis. </span>1991;<span class="ref-vol">12</span>:1839–1845.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/1934265" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=Metabolic+activation+of+carcinogenic+heterocyclic+aromatic+amines+by+human+liver+and+colon&amp;author=RJ+Turesky&amp;author=NP+Lang&amp;author=MA+Butler&amp;author=CH+Teitel&amp;author=FF+Kadlubar&amp;volume=12&amp;publication_year=1991&amp;pages=1839-1845&amp;pmid=1934265&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Peng L, Turesky RJ. Optimizing proteolytic digestion conditions for the analysis of serum albumin adducts of 2-amino-1-methyl-6-phenylimidazo[4,5-<em>b</em>]pyridine, a potential human carcinogen formed in cooked meat. <span><span class="ref-journal">J Proteomics. </span>2014;<span class="ref-vol">103</span>:267–278.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4277649/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24698664" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Proteomics&amp;title=Optimizing+proteolytic+digestion+conditions+for+the+analysis+of+serum+albumin+adducts+of+2-amino-1-methyl-6-phenylimidazo[4,5-b]pyridine,+a+potential+human+carcinogen+formed+in+cooked+meat&amp;author=L+Peng&amp;author=RJ+Turesky&amp;volume=103&amp;publication_year=2014&amp;pages=267-278&amp;pmid=24698664&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Fede JM, Thakur AP, Gooderham NJ, Turesky RJ. Biomonitoring of 2-amino-1-methyl-6-phenylimidazo[4,5-<em>b</em>]pyridine (PhIP) and its carcinogenic metabolites in urine. <span><span class="ref-journal">Chem Res Toxicol. </span>2009;<span class="ref-vol">22</span>:1096–1105.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3215276/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19441775" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem+Res+Toxicol&amp;title=Biomonitoring+of+2-amino-1-methyl-6-phenylimidazo[4,5-b]pyridine+(PhIP)+and+its+carcinogenic+metabolites+in+urine&amp;author=JM+Fede&amp;author=AP+Thakur&amp;author=NJ+Gooderham&amp;author=RJ+Turesky&amp;volume=22&amp;publication_year=2009&amp;pages=1096-1105&amp;pmid=19441775&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Toribio F, Moyano E, Puignou L, Galceran MT. Multistep mass spectrometry of heterocyclic amines in a quadrupole ion trap mass analyser. <span><span class="ref-journal">J Mass Spectrom. </span>2002;<span class="ref-vol">37</span>:812–828.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12203675" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mass+Spectrom&amp;title=Multistep+mass+spectrometry+of+heterocyclic+amines+in+a+quadrupole+ion+trap+mass+analyser&amp;author=F+Toribio&amp;author=E+Moyano&amp;author=L+Puignou&amp;author=MT+Galceran&amp;volume=37&amp;publication_year=2002&amp;pages=812-828&amp;pmid=12203675&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Guo Jingshu, Yonemori Kim, Le Marchand Loic, Turesky Robert J. A Method to Biomonitor the Cooked Meat Carcinogen 2-Amino-1-methyl-6-phenylimidazo[4,5-b]pyridine (PhIP) in Dyed Hair by Ultra Performance Liquid Chromatography-Orbitrap High Resolution Multistage Mass Spectrometry. <span><span class="ref-journal">Analytical Chemistry. </span>2015;<span class="ref-vol">87</span>:5872–5877.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4470769/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25969997" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Analytical+Chemistry&amp;title=A+Method+to+Biomonitor+the+Cooked+Meat+Carcinogen+2-Amino-1-methyl-6-phenylimidazo[4,5-b]pyridine+(PhIP)+in+Dyed+Hair+by+Ultra+Performance+Liquid+Chromatography-Orbitrap+High+Resolution+Multistage+Mass+Spectrometry&amp;author=Jingshu+Guo&amp;author=Kim+Yonemori&amp;author=Loic+Le+Marchand&amp;author=Robert+J+Turesky&amp;volume=87&amp;publication_year=2015&amp;pages=5872-5877&amp;pmid=25969997&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Validation of Analytical Procedures: Text and Methodology Q2(R1) <span><span class="ref-journal">International Conference on Harmonization. </span>2005 Nov;</span>  <a href="http://www.ich.org/products/guidelines/quality/quality-single/article/validation-of-analytical-procedures-text-and-methodology.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ich.org/products/guidelines/quality/quality-single/article/validation-of-analytical-procedures-text-and-methodology.html</a>.</span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Peters T., Jr Serum albumin. <span><span class="ref-journal">Adv Protein Chem. </span>1985;<span class="ref-vol">37</span>:161–245.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3904348" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Adv+Protein+Chem&amp;title=Serum+albumin&amp;author=T+Peters&amp;volume=37&amp;publication_year=1985&amp;pages=161-245&amp;pmid=3904348&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Boyer PD. Spectrophotometric Study of the Reaction of Protein Sulfhydryl Groups with Organic Mercurials. <span><span class="ref-journal">J Am Chem Soc. </span>1954;<span class="ref-vol">76</span>:4331–4337.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Chem+Soc&amp;title=Spectrophotometric+Study+of+the+Reaction+of+Protein+Sulfhydryl+Groups+with+Organic+Mercurials&amp;author=PD+Boyer&amp;volume=76&amp;publication_year=1954&amp;pages=4331-4337&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Nowell S, Ambrosone CB, Ozawa S, MacLeod SL, Mrackova G, Williams S, Plaxco J, Kadlubar FF, Lang NP. Relationship of phenol sulfotransferase activity (SULT1A1) genotype to sulfotransferase phenotype in platelet cytosol. <span><span class="ref-journal">Pharmacogenetics. </span>2000;<span class="ref-vol">10</span>:789–797.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11191883" ref="reftype=pubmed&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&amp;title=Relationship+of+phenol+sulfotransferase+activity+(SULT1A1)+genotype+to+sulfotransferase+phenotype+in+platelet+cytosol&amp;author=S+Nowell&amp;author=CB+Ambrosone&amp;author=S+Ozawa&amp;author=SL+MacLeod&amp;author=G+Mrackova&amp;volume=10&amp;publication_year=2000&amp;pages=789-797&amp;pmid=11191883&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5375106&amp;issue-id=290233&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC5375106/pdf/nihms854476.pdf" class="int-view">PDF (672K)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/5375106/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/5375106/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="5375106" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5375106%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="tyvFndrb6MBzaTby2k960vF7rv6n4R3iyKUtCzeV4RFu04iQgK89fmPhzR1CQWE3">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5375106%2F&amp;text=Mass%20Spectrometric%20Characterization%20of%20an%20Acid-Labile%20Adduct%20Formed%20with%202-Amino-1-methyl-6-phenylimidazo%5B4%2C5-b%5Dpyridine%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5375106%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5375106/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC5375106/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/27984695/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC5375106/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/27984695/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC5375106/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/5375106/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC5375106/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
