
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs2.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>High throughput sequencing of RNA transcriptomes in Ruditapes philippinarum identifies genes involved in osmotic stress response - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="D0BD41E4AC192A45000039B17CB05544.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="scirep" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC5504028/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5504028/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Scientific Reports" /><meta name="citation_title" content="High throughput sequencing of RNA transcriptomes in Ruditapes philippinarum identifies genes involved in osmotic stress response" /><meta name="citation_author" content="Hongtao Nie" /><meta name="citation_author_institution" content="College of Fisheries and Life Science, Dalian Ocean University, Dalian, 116023 China" /><meta name="citation_author_institution" content="Engineering Research Center of Shellfish Culture and Breeding in Liaoning Province, Dalian Ocean University, Dalian, 116023 China" /><meta name="citation_author" content="Liwen Jiang" /><meta name="citation_author_institution" content="College of Fisheries and Life Science, Dalian Ocean University, Dalian, 116023 China" /><meta name="citation_author" content="Peng Chen" /><meta name="citation_author_institution" content="College of Fisheries and Life Science, Dalian Ocean University, Dalian, 116023 China" /><meta name="citation_author" content="Zhongming Huo" /><meta name="citation_author_institution" content="College of Fisheries and Life Science, Dalian Ocean University, Dalian, 116023 China" /><meta name="citation_author_institution" content="Engineering Research Center of Shellfish Culture and Breeding in Liaoning Province, Dalian Ocean University, Dalian, 116023 China" /><meta name="citation_author" content="Feng Yang" /><meta name="citation_author_institution" content="College of Fisheries and Life Science, Dalian Ocean University, Dalian, 116023 China" /><meta name="citation_author_institution" content="Engineering Research Center of Shellfish Culture and Breeding in Liaoning Province, Dalian Ocean University, Dalian, 116023 China" /><meta name="citation_author" content="Xiwu Yan" /><meta name="citation_author_institution" content="College of Fisheries and Life Science, Dalian Ocean University, Dalian, 116023 China" /><meta name="citation_author_institution" content="Engineering Research Center of Shellfish Culture and Breeding in Liaoning Province, Dalian Ocean University, Dalian, 116023 China" /><meta name="citation_publication_date" content="2017" /><meta name="citation_volume" content="7" /><meta name="citation_doi" content="10.1038/s41598-017-05397-8" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5504028/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC5504028/" /><meta name="citation_pmid" content="28694531" /><meta name="DC.Title" content="High throughput sequencing of RNA transcriptomes in Ruditapes philippinarum identifies genes involved in osmotic stress response" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Nature Publishing Group" /><meta name="DC.Contributor" content="Hongtao Nie" /><meta name="DC.Contributor" content="Liwen Jiang" /><meta name="DC.Contributor" content="Peng Chen" /><meta name="DC.Contributor" content="Zhongming Huo" /><meta name="DC.Contributor" content="Feng Yang" /><meta name="DC.Contributor" content="Xiwu Yan" /><meta name="DC.Date" content="2017" /><meta name="DC.Identifier" content="10.1038/s41598-017-05397-8" /><meta name="DC.Language" content="en" /><meta property="og:title" content="High throughput sequencing of RNA transcriptomes in Ruditapes philippinarum identifies genes involved in osmotic stress response" /><meta property="og:type" content="article" /><meta property="og:description" content="&#10;Ruditapes philippinarum, is an economically important marine bivalve species. The ability to cope with low salinity stress is quite important for the survival of aquatic species under natural conditions. In this study, the transcriptional response of ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5504028/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC5504028/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22Sci%20Rep%22[journal]" class="navlink">Scientific Reports</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC5504028
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC5504028/pdf/41598_2017_Article_5397.pdf" class="int-view">PDF (2.1M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/5504028/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/5504028/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="5504028" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5504028%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="117PEDA5R77HH3V9jfbYwNDjQ7MX7440m17zbnxzkqDL1l91GBVL4gOiyfv41sqe">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5504028%2F&amp;text=High%20throughput%20sequencing%20of%20RNA%20transcriptomes%20in%20Ruditapes%20philippinarum%20identifies%20genes%20involved%20in%20osmotic%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5504028%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5504028/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC5504028/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/28694531/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC5504028/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/28694531/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC5504028/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/5504028/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22Sci%20Rep%22[journal]" class="navlink">Scientific Reports</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC5504028
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-scirep.jpg" alt="Logo of scirep" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="375,13,500,30" alt="About" title="About" href="http://www.nature.com/srep/about/index.html" ref="reftype=publisher&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="375,30,500,48" alt="Editorial Board" title="Editorial Board" href="http://www.nature.com/srep/eap-ebm/index.html#eb" ref="reftype=publisher&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="375,48,500,75" alt="For Authors" title="For Authors" href="http://www.nature.com/srep/authors/index.html" ref="reftype=publisher&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="0,0,375,75" alt="Scientific Reports" title="Scientific Reports" href="http://www.nature.com/srep/index.html" ref="reftype=publisher&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Sci Rep.</span> 2017; 7: 4953. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2017 Jul 10. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1038%2Fs41598-017-05397-8" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/s41598-017-05397-8</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5504028</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/28694531">28694531</a></div></div></div><h1 class="content-title">High throughput sequencing of RNA transcriptomes in <em>Ruditapes philippinarum</em> identifies genes involved in osmotic stress response</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Nie%20H%5BAuthor%5D" class="affpopup" co-rid="_co_idm139982521130272" co-class="co-affbox">Hongtao Nie</a>,<sup>1,</sup><sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Jiang%20L%5BAuthor%5D" class="affpopup" co-rid="_co_idm139982516172224" co-class="co-affbox">Liwen Jiang</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Chen%20P%5BAuthor%5D" class="affpopup" co-rid="_co_idm139982516170432" co-class="co-affbox">Peng Chen</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Huo%20Z%5BAuthor%5D" class="affpopup" co-rid="_co_idm139982517343952" co-class="co-affbox">Zhongming Huo</a>,<sup>1,</sup><sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Yang%20F%5BAuthor%5D" class="affpopup" co-rid="_co_idm139982518774576" co-class="co-affbox">Feng Yang</a>,<sup>1,</sup><sup>2</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Yan%20X%5BAuthor%5D" class="affpopup" co-rid="_co_idm139982526066544" co-class="co-affbox">Xiwu Yan</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>1,</sup><sup>2</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139982521130272"><h3 class="no_margin">Hongtao Nie</h3><p><sup>1</sup>College of Fisheries and Life Science, Dalian Ocean University, Dalian, 116023 China </p><p><sup>2</sup>Engineering Research Center of Shellfish Culture and Breeding in Liaoning Province, Dalian Ocean University, Dalian, 116023 China </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Nie%20H%5BAuthor%5D">Hongtao Nie</a></div></div><div id="_co_idm139982516172224"><h3 class="no_margin">Liwen Jiang</h3><p><sup>1</sup>College of Fisheries and Life Science, Dalian Ocean University, Dalian, 116023 China </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Jiang%20L%5BAuthor%5D">Liwen Jiang</a></div></div><div id="_co_idm139982516170432"><h3 class="no_margin">Peng Chen</h3><p><sup>1</sup>College of Fisheries and Life Science, Dalian Ocean University, Dalian, 116023 China </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Chen%20P%5BAuthor%5D">Peng Chen</a></div></div><div id="_co_idm139982517343952"><h3 class="no_margin">Zhongming Huo</h3><p><sup>1</sup>College of Fisheries and Life Science, Dalian Ocean University, Dalian, 116023 China </p><p><sup>2</sup>Engineering Research Center of Shellfish Culture and Breeding in Liaoning Province, Dalian Ocean University, Dalian, 116023 China </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Huo%20Z%5BAuthor%5D">Zhongming Huo</a></div></div><div id="_co_idm139982518774576"><h3 class="no_margin">Feng Yang</h3><p><sup>1</sup>College of Fisheries and Life Science, Dalian Ocean University, Dalian, 116023 China </p><p><sup>2</sup>Engineering Research Center of Shellfish Culture and Breeding in Liaoning Province, Dalian Ocean University, Dalian, 116023 China </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Yang%20F%5BAuthor%5D">Feng Yang</a></div></div><div id="_co_idm139982526066544"><h3 class="no_margin">Xiwu Yan</h3><p><sup>1</sup>College of Fisheries and Life Science, Dalian Ocean University, Dalian, 116023 China </p><p><sup>2</sup>Engineering Research Center of Shellfish Culture and Breeding in Liaoning Province, Dalian Ocean University, Dalian, 116023 China </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Yan%20X%5BAuthor%5D">Xiwu Yan</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm139982526050128_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139982526050128_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139982526050128_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm139982526050128_ai" style="display:none"><div class="fm-affl" id="Aff1"><sup>1</sup>College of Fisheries and Life Science, Dalian Ocean University, Dalian, 116023 China </div><div class="fm-affl" id="Aff2"><sup>2</sup>Engineering Research Center of Shellfish Culture and Breeding in Liaoning Province, Dalian Ocean University, Dalian, 116023 China </div><div><span class="fm-affl">Xiwu Yan, </span><span class="fm-affl"><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="nc.ude.uold@uwixnay" class="oemail">nc.ude.uold@uwixnay</a></span>.</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup>Corresponding author.</div></div><div class="fm-article-notes hide half_rhythm" id="idm139982526050128_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2016 Nov 24; Accepted 2017 May 30.</div></div><div class="permissions half_rhythm hide" id="idm139982526050128_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; The Author(s) 2017</div><div class="license half_rhythm">
<strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article&#x02019;s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article&#x02019;s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="Abs1" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="Abs1title">Abstract</h2><!--article-meta--><div><p id="Par1" class="p p-first-last">
<em>Ruditapes philippinarum</em>, is an economically important marine bivalve species. The ability to cope with low salinity stress is quite important for the survival of aquatic species under natural conditions. In this study, the transcriptional response of the Manila clam to low salinity stress was characterized using RNA sequencing. The transcriptomes of a low salinity-treatment group (FRp1, FRp2), which survived under low salinity stress, and control group (SRp1, SRp2), which was not subjected to low salinity stress, were sequenced with the Illumina HiSeq platform. A total of 196,578 unigenes were generated. GO and KEGG analyses revealed that signal transduction, immune response, cellular component organization or biogenesis, and energy production processes were the most highly enriched pathways among the genes that were differentially expressed under low salinity stress. All these pathways could be assigned to the following biological functions in the low salinity tolerant Manila clam: signal response to low salinity stress, antioxidant response, intracellular free amino acid transport and metabolism, energy production and conversion, cell signaling pathways, and regulation of ionic content and cell volume. In summary, this is the first study using high-throughput sequencing to identify gene targets that could explain osmotic regulation mechanisms under salinity stress in <em>R. philippinarum</em>.</p></div></div><div id="Sec1" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec1title">Introduction</h2><p id="Par2" class="p p-first">Salinity is one of the most important environmental factors. Being relatively constant in open seas, it varies considerably in intertidal zones, estuaries and other biotopes. Salinity has been long recognized as one of the fundamental factors affecting marine species distribution and influencing physiological processes of marine and estuarine organisms, such as survival, hemolymph osmolarity, and tissue water content<sup><sup><a href="#CR1" rid="CR1" class=" bibr popnode">1</a></sup>&#x02013;<sup><a href="#CR3" rid="CR3" class=" bibr popnode">3</a></sup></sup>. Organisms may be subjected to hyperosmotic or hypoosmotic stress due to heavy rains or water evaporation during low tide. Thus, the tolerance limit of mollusks is important in coping with salinity change. In recent years, due to global warming, the huge amounts of freshwater inflow disproportionately affect the seas and oceans. Salinity of the superficial water and inshore water decreased acutely in rainy season, which can incur the increasing mortality outbreaks and distribution shifts of marine species. Therefore, salinity is a limiting factor to the survival and distribution of many marine organisms, especially as it varies downward<sup><sup><a href="#CR4" rid="CR4" class=" bibr popnode">4</a></sup></sup>. Most marine invertebrates, as osmoconformers, have blood osmolarities close to that of seawater, lacking the ability to regulate the osmotic pressure of the internal medium<sup><sup><a href="#CR5" rid="CR5" class=" bibr popnode">5</a></sup></sup>.</p><p id="Par3">Many commercially important aquatic animals are sensitive to low salinity stress, and mass mortality is often caused by heavily rainfall in summer<sup><sup><a href="#CR2" rid="CR2" class=" bibr popnode">2</a></sup>, <sup><a href="#CR3" rid="CR3" class=" bibr popnode">3</a></sup></sup>. Therefore, it is very important for both scientific researchers and fisheries to investigate the mechanisms underlying the low salinity tolerance of aquaculture species. An increasing number of studies have characterized the acclimation responses of aquaculture animals to low salinity stress, and it has been shown that aquatic animals can gradually shape their salinity adaptive phenotypes with extensive biochemical, metabolic, and physiological acclimation processes, and the signaling transduction, ion transport and transcription regulation are likely involved in the adaptive process to hypo-osmotic conditions<sup><sup><a href="#CR5" rid="CR5" class=" bibr popnode">5</a></sup>, <sup><a href="#CR6" rid="CR6" class=" bibr popnode">6</a></sup></sup>.</p><p id="Par4">Researchers have characterized the transcriptional responses elicited by low salinity stress in a number of aquatic animals, including the <em>Crassostrea gigas</em>
<sup><sup><a href="#CR7" rid="CR7" class=" bibr popnode">7</a></sup>, <sup><a href="#CR8" rid="CR8" class=" bibr popnode">8</a></sup></sup>, <em>C. hongkongensis</em>
<sup><sup><a href="#CR6" rid="CR6" class=" bibr popnode">6</a></sup></sup>, <em>Penaeus monodon</em>
<sup><sup><a href="#CR9" rid="CR9" class=" bibr popnode">9</a></sup></sup>, <em>Litopenaeus vannamei</em>
<sup><sup><a href="#CR10" rid="CR10" class=" bibr popnode">10</a></sup></sup>, <em>Portunus trituberculatus</em>
<sup><sup><a href="#CR11" rid="CR11" class=" bibr popnode">11</a></sup></sup>, and <em>Eriocheir sinensis</em>
<sup><sup><a href="#CR12" rid="CR12" class=" bibr popnode">12</a></sup></sup>. These studies have identified a large number of osmoregulation-related genes involved in a variety of biological processes that are associated with acclimation to both acute and chronic salinity variations. In addition, some important salinity stress-related genes have also been identified in comparative transcriptomic analyses of <em>C. gigas</em> and <em>C. hongkongensis</em>
<sup><sup><a href="#CR6" rid="CR6" class=" bibr popnode">6</a></sup></sup>. However, there have been few transcriptomic studies of marine mollusk species to identify and characterize their salinity stress induced gene expression. The tolerance of mollusk is determined by cellular mechanisms of adaptation. Reversible changes of protein and RNA synthesis, alteration of the pattern of multiple molecular forms of different enzymes, and the regulation of ionic content and cell volume were shown to be of importance for the above mentioned mechanisms. The efficiency of resistance and tolerance adaptations to salinity changes may vary in different species and in different colour phenotypes of the same species<sup><sup><a href="#CR13" rid="CR13" class=" bibr popnode">13</a></sup></sup>.</p><p id="Par5">The Manila clam, <em>Ruditapes philippinarum</em>, is an economically important marine bivalve species of the aquaculture industry, which has a wide geographic distribution from Europe to Asia<sup><sup><a href="#CR14" rid="CR14" class=" bibr popnode">14</a></sup>, <sup><a href="#CR15" rid="CR15" class=" bibr popnode">15</a></sup></sup>. It is reported that Manila clam showed a great capacity to adapt to the new environment<sup><sup><a href="#CR16" rid="CR16" class=" bibr popnode">16</a></sup></sup>. The ability to cope with abiotic and biotic stresses is vital to survival of the clams because of their intertidal benthic inhabiting lifestyle. The Manila clam displays different shell color strains in its natural habitat, which have been selected and cultivated for several generations through clam selective breeding programs<sup><sup><a href="#CR14" rid="CR14" class=" bibr popnode">14</a></sup></sup>. Evidences indicated that zebra and orange strains had stronger resistance to environmental stressors, such as temperature and salinity<sup><sup><a href="#CR14" rid="CR14" class=" bibr popnode">14</a></sup></sup>. It has long been an interesting question how the manila clam could be survived in response to the environmental stresses involved in the fluctuation of salinity. Many studies have been performed to reveal the alterations of mRNA expression<sup><sup><a href="#CR17" rid="CR17" class=" bibr popnode">17</a></sup>&#x02013;<sup><a href="#CR20" rid="CR20" class=" bibr popnode">20</a></sup></sup> and protein expression<sup><sup><a href="#CR21" rid="CR21" class=" bibr popnode">21</a></sup>&#x02013;<sup><a href="#CR24" rid="CR24" class=" bibr popnode">24</a></sup></sup> under abiotic and biotic stresses in the oyster<sup><sup><a href="#CR25" rid="CR25" class=" bibr popnode">25</a></sup></sup>, while expression profiling of miRNAs under osmotic stress remains largely unexplored in clams.</p><p id="Par6" class="p p-last">In this study, a high throughput transcriptome sequencing (RNA-seq) approach was adopted to investigate the transcriptome profiles of the gills from <em>R. philippinarum</em> adapted to optimal and low salinity seawater. The study aimed to investigate the expression patterns of the different treatment to better understand the transcriptomic regulation in <em>R. philippinarum</em> to low salinity stress and identify genes involved in osmoregulation of the Manila clam. This study provides an important resource for future investigations on mechanism of tolerance to hypo-osmotic stress for marine invertebrates.</p></div><div id="Sec2" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec2title">Results</h2><div id="Sec3" class="sec sec-first"><h3 id="Sec3title">Sequencing and assembly</h3><p id="Par7" class="p p-first-last">The transcript length distribution and the number of transcripts are given in Fig.&#x000a0;<a href="/pmc/articles/PMC5504028/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1</span></a>. A total of 249,482,420 raw reads with an average length of 125&#x02009;bp were acquired in the four libraries FRp1, SRp1, FRp2 and SRp2. After the low-quality reads were filtered from the sequence data, 245,015,856 (98.2%) high-quality reads remained and were assembled de novo. The raw reads produced for FRp1, SRp1, FRp2 and SRp2 have been submitted to the NCBI SRA database under accession numbers SRP082480. Because no reference genome exists for <em>R. philippinarum</em>, the high-quality reads from the four libraries were combined and assembled into a reference transcriptome with the Trinity software<sup><sup><a href="#CR26" rid="CR26" class=" bibr popnode">26</a></sup></sup>. This assembly yielded 196,578 unigenes with an average length of 664&#x02009;bp, a minimum length of 201&#x02009;bp, and a maximum length of 45,933&#x02009;bp, with an N50 length of 1117&#x02009;bp (Table&#x000a0;<a href="/pmc/articles/PMC5504028/table/Tab1/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab1" rid-ob="ob-Tab1" co-legend-rid=""><span>1</span></a>). A total of 303 groups of eukaryotic BUSCO genes were searched against <em>R. philippinarum</em> assembly using the mode of trans with default settings. 293 of the 303 BUSCO genes were found to be present in <em>R. philippinarum</em>, suggesting this assembly covers most of the genes of the clam. Summarized benchmarks in BUSCO notation: C: 96% [D: 22%], F: 2.3%, M: 0.9%, n: 303. C: the percentage of complete genes [D: the percentage of duplicated genes], F: the percentage of fragmented genes, M: the percentage of missing genes, n: number of genes used.

</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig1" co-legend-rid="lgnd_Fig1"><a href="/pmc/articles/PMC5504028/figure/Fig1/" target="figure" rid-figpopup="Fig1" rid-ob="ob-Fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139982561341408"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41598_2017_5397_Fig1_HTML.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is 41598_2017_5397_Fig1_HTML.jpg" src="/pmc/articles/PMC5504028/bin/41598_2017_5397_Fig1_HTML.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm139982561341408"><a target="object" rel="noopener" href="/pmc/articles/PMC5504028/figure/Fig1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig1"><div><a class="figpopup" href="/pmc/articles/PMC5504028/figure/Fig1/" target="figure" rid-figpopup="Fig1" rid-ob="ob-Fig1">Figure 1</a></div><!--caption a7--><div class="caption"><p>Sequence length distribution of transcripts and unigenes assembled from Illumina reads of the gill transcriptome of <em>Ruditapes philippinarum</em>. The x-axis indicates the lengths of the transcripts and unigenes, and the y-axis indicates the number of transcripts and unigenes in each size category.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="Tab1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p>Summary statistics of <em>Ruditapes philippinarum</em> transcriptome assembly using Trinity software.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="left" rowspan="1" colspan="1">Unigene</th><th align="left" rowspan="1" colspan="1">Min. Length (bp)</th><th align="left" rowspan="1" colspan="1">Mean Length (bp)</th><th align="left" rowspan="1" colspan="1">Max. Length (bp)</th><th align="left" rowspan="1" colspan="1">Median Length (bp)</th><th align="left" rowspan="1" colspan="1">N50</th><th align="left" rowspan="1" colspan="1">N90</th><th align="left" rowspan="1" colspan="1">Total Length (bp)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Number</td><td rowspan="1" colspan="1">201</td><td rowspan="1" colspan="1">664</td><td rowspan="1" colspan="1">45,933</td><td rowspan="1" colspan="1">340</td><td rowspan="1" colspan="1">1117</td><td rowspan="1" colspan="1">258</td><td rowspan="1" colspan="1">130,604,832</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139982524811552"><a target="object" rel="noopener" href="/pmc/articles/PMC5504028/table/Tab1/?report=objectonly">Open in a separate window</a></div></div></div><div id="Sec4" class="sec"><h3 id="Sec4title">Annotation of Unigenes</h3><p id="Par8" class="p p-first-last">A total of 196,578 unigenes were annotated by matching them against the Nr and Swiss-Prot databases using a BLASTX search with an E value of 1.0e<sup>&#x02212;5</sup>; 33,588 unigenes (17.08% of the total) were matched to the Nr database, and 22,122 (11.25% of the total) were matched to the Swiss-Prot database. All the annotated unigenes were used to determine the GO terms and KEGG pathways (Table&#x000a0;<a href="/pmc/articles/PMC5504028/table/Tab2/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab2" rid-ob="ob-Tab2" co-legend-rid=""><span>2</span></a> and Figs&#x000a0;<a href="/pmc/articles/PMC5504028/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2</span></a>, <a href="/pmc/articles/PMC5504028/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,3).</span></span><span>3</span></a>). In the species distribution of sequences matched to the Nr database, 31.6% of the matched unigenes showed similarities to sequences of <em>Crassostrea gigas</em>, followed by <em>Lottia gigantea</em> (14.4%), <em>Perkinsus marinus</em> (9.4%), <em>Aplysia californica</em> (6.8%), <em>Saccoglossus kowalevskii</em> (4.2%), and others (33.6%) (Fig.&#x000a0;<a href="/pmc/articles/PMC5504028/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4</span></a>).



</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="Tab2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p>Summary statistics of functional annotation of <em>R. philippinarum</em> transcriptome.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="left" rowspan="1" colspan="1">Category</th><th align="left" rowspan="1" colspan="1">All sequences</th><th align="left" rowspan="1" colspan="1">300&#x02013;999&#x02009;bp</th><th align="left" rowspan="1" colspan="1">&#x0003e;=1000&#x02009;bp</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Total number of unigene</td><td rowspan="1" colspan="1">196578</td><td rowspan="1" colspan="1">83356</td><td rowspan="1" colspan="1">31158</td></tr><tr><td rowspan="1" colspan="1">Unigene matches against Nr and Swiss-Prot</td><td rowspan="1" colspan="1">22053</td><td rowspan="1" colspan="1">5579</td><td rowspan="1" colspan="1">13962</td></tr><tr><td rowspan="1" colspan="1">Unigene matches against GO</td><td rowspan="1" colspan="1">36344</td><td rowspan="1" colspan="1">12148</td><td rowspan="1" colspan="1">17228</td></tr><tr><td rowspan="1" colspan="1">Unigene matches against KEGG</td><td rowspan="1" colspan="1">10698</td><td rowspan="1" colspan="1">2782</td><td rowspan="1" colspan="1">6560</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139982521938240"><a target="object" rel="noopener" href="/pmc/articles/PMC5504028/table/Tab2/?report=objectonly">Open in a separate window</a></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig2" co-legend-rid="lgnd_Fig2"><a href="/pmc/articles/PMC5504028/figure/Fig2/" target="figure" rid-figpopup="Fig2" rid-ob="ob-Fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139982515276112"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41598_2017_5397_Fig2_HTML.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is 41598_2017_5397_Fig2_HTML.jpg" src="/pmc/articles/PMC5504028/bin/41598_2017_5397_Fig2_HTML.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm139982515276112"><a target="object" rel="noopener" href="/pmc/articles/PMC5504028/figure/Fig2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig2"><div><a class="figpopup" href="/pmc/articles/PMC5504028/figure/Fig2/" target="figure" rid-figpopup="Fig2" rid-ob="ob-Fig2">Figure 2</a></div><!--caption a7--><div class="caption"><p>GO (Gene Ontology) categorization (biological process, cellular component, and molecular function) of the unigenes in the gill transcriptome of <em>R. philippinarum</em>. Each annotated sequence is assigned at least one GO term.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig3" co-legend-rid="lgnd_Fig3"><a href="/pmc/articles/PMC5504028/figure/Fig3/" target="figure" rid-figpopup="Fig3" rid-ob="ob-Fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139982521249376"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41598_2017_5397_Fig3_HTML.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is 41598_2017_5397_Fig3_HTML.jpg" src="/pmc/articles/PMC5504028/bin/41598_2017_5397_Fig3_HTML.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm139982521249376"><a target="object" rel="noopener" href="/pmc/articles/PMC5504028/figure/Fig3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig3"><div><a class="figpopup" href="/pmc/articles/PMC5504028/figure/Fig3/" target="figure" rid-figpopup="Fig3" rid-ob="ob-Fig3">Figure 3</a></div><!--caption a7--><div class="caption"><p>KEGG (Kyoto Encyclopedia of Genes and Genomes) assignment of unigenes in the gill transcriptome of <em>R. philippinarum</em>. (<strong>A</strong>), Cellular processes; (<strong>B</strong>) Environmental information processing; (<strong>C</strong>) Genetic information processing; (<strong>D</strong>) Metabolism; (<strong>E</strong>) Organismal systems.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig4" co-legend-rid="lgnd_Fig4"><a href="/pmc/articles/PMC5504028/figure/Fig4/" target="figure" rid-figpopup="Fig4" rid-ob="ob-Fig4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139982522910048"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41598_2017_5397_Fig4_HTML.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is 41598_2017_5397_Fig4_HTML.jpg" src="/pmc/articles/PMC5504028/bin/41598_2017_5397_Fig4_HTML.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm139982522910048"><a target="object" rel="noopener" href="/pmc/articles/PMC5504028/figure/Fig4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig4"><div><a class="figpopup" href="/pmc/articles/PMC5504028/figure/Fig4/" target="figure" rid-figpopup="Fig4" rid-ob="ob-Fig4">Figure 4</a></div><!--caption a7--><div class="caption"><p>Species distribution of sequences matched to the Nr database.</p></div></div></div></div><div id="Sec5" class="sec"><h3 id="Sec5title">Differentially Expressed Genes</h3><p id="Par9" class="p p-first-last">We obtained 115,733,418 and 129,282,438 qualified Illumina read pairs from the low salinity challenged (FRp1, FRp2) and unchallenged control (SRp1, SRp2) <em>R. philippinarum</em> cDNA libraries, respectively. A comparison of this gene expression showed that 4,467 and 3,168 unigenes were differentially expressed after the low-salinity challenge (qvalue&#x02009;&#x0003c;&#x02009;0.005 and |log2(foldchange)|&#x0003e;1), including 1,846 and 1,446 up-regulated, and 2,621 and 1,722 down-regulated unigenes differentially expressed between FRp1 and SRp1, and FRp2 and SRp2, respectively (Fig.&#x000a0;<a href="/pmc/articles/PMC5504028/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5</span></a>). To validate our Illumina sequencing results, 12 differentially regulated genes were selected for RT&#x02013;qPCR analysis, in which 11 of the genes agreed well with the Illumina sequencing results. The other gene, encoding Rho guanine nucleotide exchange factor 12 (ARHGEF12), did not match the Illumina sequencing results. Therefore, the majority of low salinity responsive genes identified with the Illumina sequencing analysis were consistent with the results of quantitative real-time PCR experiments.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig5" co-legend-rid="lgnd_Fig5"><a href="/pmc/articles/PMC5504028/figure/Fig5/" target="figure" rid-figpopup="Fig5" rid-ob="ob-Fig5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139982527061744"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41598_2017_5397_Fig5_HTML.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is 41598_2017_5397_Fig5_HTML.jpg" src="/pmc/articles/PMC5504028/bin/41598_2017_5397_Fig5_HTML.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm139982527061744"><a target="object" rel="noopener" href="/pmc/articles/PMC5504028/figure/Fig5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig5"><div><a class="figpopup" href="/pmc/articles/PMC5504028/figure/Fig5/" target="figure" rid-figpopup="Fig5" rid-ob="ob-Fig5">Figure 5</a></div><!--caption a7--><div class="caption"><p>Volcano plot of differentially expressed genes (DEGs) from the transcriptomes of FRp1 vs SRp1 and FRp2 vs SRp2 in <em>R. philippinarum</em>.</p></div></div></div></div><div id="Sec6" class="sec"><h3 id="Sec6title">Gene Ontology (GO) classification</h3><p id="Par10" class="p p-first-last">We conducted a GO analysis of the Illumina sequences based on their similarities to known proteins in the Nr database. This analysis provided hierarchical relationships that provide information on the molecular functions, cellular components, and biological processes involved in the low salinity adaptation of the Manila clam (Table&#x000a0;<a href="/pmc/articles/PMC5504028/table/Tab3/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab3" rid-ob="ob-Tab3" co-legend-rid=""><span>3</span></a>). A total of 36,344 unigenes were annotated with the GO analysis, with one or more GO term. Of these, 22,287 unigenes were annotated with molecular functions, 23,039 with biological processes, and 14,647 with cellular components. For the cellular components, the major categories represented were cell (GO: 0005623) and cell part (GO: 0044464). For the biological processes, cellular process (GO: 0009987) was the most strongly represented GO term, followed by metabolic process (GO: 0008152). Genes involved in other important biological processes, such as regulation of biological process, localization, single-organism process, response to stimulus, and biological regulation were also identified. A number of unigenes were also involved in interesting categories, such as signaling, membrane, transporter activity, receptor activity, macromolecular complex, cellular component organization or biogenesis, and molecular transducer activity, which may play roles in the low-salinity resistance and adaptation of <em>R. philippinarum</em>. Genes involved in binding (GO: 0005488) and catalytic activity (GO: 0003824) were highly represented among the molecular functions.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="Tab3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p>GO classification of the differentially expressed genes from <em>R. philippinarum</em>.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Term</th><th rowspan="1" colspan="1">Comparison</th><th rowspan="1" colspan="1">Type</th><th rowspan="1" colspan="1">Gene</th><th rowspan="1" colspan="1">p-Value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">aminoglycan metabolic process (GO:0006022)</td><td rowspan="1" colspan="1">FRp1 vs SRp1</td><td rowspan="1" colspan="1">Biological process</td><td rowspan="1" colspan="1">37</td><td rowspan="1" colspan="1">4.86E-05</td></tr><tr><td rowspan="1" colspan="1">chitin binding (GO:0008061)</td><td rowspan="1" colspan="1">FRp1 vs SRp1</td><td rowspan="1" colspan="1">Molecular function</td><td rowspan="1" colspan="1">22</td><td rowspan="1" colspan="1">4.86E-05</td></tr><tr><td rowspan="1" colspan="1">extracellular region (GO:0005576)</td><td rowspan="1" colspan="1">FRp1 vs SRp1</td><td rowspan="1" colspan="1">Cellular component</td><td rowspan="1" colspan="1">220</td><td rowspan="1" colspan="1">4.86E-05</td></tr><tr><td rowspan="1" colspan="1">chitin metabolic process (GO:0006030)</td><td rowspan="1" colspan="1">FRp1 vs SRp1</td><td rowspan="1" colspan="1">Biological process</td><td rowspan="1" colspan="1">22</td><td rowspan="1" colspan="1">4.86E-05</td></tr><tr><td rowspan="1" colspan="1">glucosamine-containing compound metabolic process (GO:1901071)</td><td rowspan="1" colspan="1">FRp1 vs SRp1</td><td rowspan="1" colspan="1">Biological process</td><td rowspan="1" colspan="1">22</td><td rowspan="1" colspan="1">5.35E-05</td></tr><tr><td rowspan="1" colspan="1">amino sugar metabolic process (GO:0006040)</td><td rowspan="1" colspan="1">FRp1 vs SRp1</td><td rowspan="1" colspan="1">Biological process</td><td rowspan="1" colspan="1">25</td><td rowspan="1" colspan="1">7.53E-05</td></tr><tr><td rowspan="1" colspan="1">carbohydrate derivative binding (GO:0097367)</td><td rowspan="1" colspan="1">FRp1 vs SRp1</td><td rowspan="1" colspan="1">Molecular function</td><td rowspan="1" colspan="1">24</td><td rowspan="1" colspan="1">0.000239</td></tr><tr><td rowspan="1" colspan="1">calcium ion binding (GO:0005509)</td><td rowspan="1" colspan="1">FRp1 vs SRp1</td><td rowspan="1" colspan="1">Molecular function</td><td rowspan="1" colspan="1">110</td><td rowspan="1" colspan="1">0.00306</td></tr><tr><td rowspan="1" colspan="1">carbohydrate binding (GO:0030246)</td><td rowspan="1" colspan="1">FRp2 vs SRp2</td><td rowspan="1" colspan="1">Molecular function</td><td rowspan="1" colspan="1">62</td><td rowspan="1" colspan="1">0.002591</td></tr><tr><td rowspan="1" colspan="1">calcium ion binding (GO:0005509)</td><td rowspan="1" colspan="1">FRp2 vs SRp2</td><td rowspan="1" colspan="1">Molecular function</td><td rowspan="1" colspan="1">154</td><td rowspan="1" colspan="1">0.020968</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139982518130944"><a target="object" rel="noopener" href="/pmc/articles/PMC5504028/table/Tab3/?report=objectonly">Open in a separate window</a></div></div></div><div id="Sec7" class="sec"><h3 id="Sec7title">EuKaryotic Orthologous Groups (KOG) classification</h3><p id="Par11" class="p p-first-last">A KOG analysis was performed to provide a deeper understanding of the functions of the unigenes. About 15,716 unigenes were classified into 26 functional categories. The category &#x02018;General function prediction only&#x02019; contained the largest number of unigenes (3,599, 22.90%) (Fig.&#x000a0;<a href="/pmc/articles/PMC5504028/figure/Fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig6" rid-ob="ob-Fig6" co-legend-rid="lgnd_Fig6"><span>6</span></a>), followed by the &#x02018;signal transduction mechanisms&#x02019; cluster (3,250, 20.68%) and the &#x02018;posttranslational modification protein&#x02019; cluster (1760, 11.20%). The categories of greatest interest in this study were amino acid transport and metabolism (508, 3.23%), inorganic ion transport and metabolism (620, 3.95%), and energy production and conversion (359, 2.28%). Because the genes in these categories are probably related to intracellular free amino acid transport and metabolism, energy production and conversion, cell signaling pathways, and regulation of ionic content and cell volume, these categories should be considered when molecular markers are developed for marker assisted selection for low salinity tolerant genes in <em>R. philippinarum</em>.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig6" co-legend-rid="lgnd_Fig6"><a href="/pmc/articles/PMC5504028/figure/Fig6/" target="figure" rid-figpopup="Fig6" rid-ob="ob-Fig6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139982528313808"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41598_2017_5397_Fig6_HTML.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is 41598_2017_5397_Fig6_HTML.jpg" src="/pmc/articles/PMC5504028/bin/41598_2017_5397_Fig6_HTML.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm139982528313808"><a target="object" rel="noopener" href="/pmc/articles/PMC5504028/figure/Fig6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig6"><div><a class="figpopup" href="/pmc/articles/PMC5504028/figure/Fig6/" target="figure" rid-figpopup="Fig6" rid-ob="ob-Fig6">Figure 6</a></div><!--caption a7--><div class="caption"><p>KOG (euKaryotic Ortholog Groups) classifications of putative proteins in the gill transcriptome of <em>R. philippinarum</em>.</p></div></div></div></div><div id="Sec8" class="sec"><h3 id="Sec8title">Kyoto Encyclopedia of Genes and Genomes (KEGG) classification</h3><p id="Par12" class="p p-first-last">A KEGG pathway analysis was performed of the assembled unigenes to identify the biochemical pathways operating in <em>R. philippinarum</em>. The results classified 10,698 unigenes into 270 different pathways, including cellular processes, environmental information processing, genetic information processing, metabolism, and organismal systems. Among these, the metabolic pathways category contained the largest number of unigenes, which fell into 12 major subgroups involved in amino acid metabolism, carbohydrate metabolism, lipid metabolism, glycan biosynthesis and metabolism, energy metabolism and so on (Table&#x000a0;<a href="/pmc/articles/PMC5504028/table/Tab4/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab4" rid-ob="ob-Tab4" co-legend-rid=""><span>4</span></a>). In total, 485 sequences were classified into 15 carbohydrate metabolisms pathways, including glycolysis/gluconeogenesis, citrate cycle (TCA cycle), galactose metabolism, pyruvate metabolism, amino sugar and nucleotide sugar metabolism. A total of 446 sequences were classified into 16 lipid metabolism pathways, including arachidonic acid metabolism, fatty acid degradation, sphingolipid metabolism, glycerolipid metabolism, and glycerophospholipid metabolism. A total of 424 sequences were classified into 13 amino acid metabolism pathways, including alanine, aspartate and glutamate metabolism, cysteine and methionine metabolism, glycine, serine and threonine metabolism, tryptophan metabolism, tyrosine metabolism, arginine and proline metabolism. A total of 326 sequences were classified into 8 energy metabolism pathways, including oxidative phosphorylation, methane metabolism, nitrogen metabolism, and sulfur metabolism. Under low salinity conditions, the enzymes involved in glycolysis/gluconeogenesis, oxidative phosphorylation, and arginine and proline metabolism were enriched, indicating that shellfish might resist low salinity through energy metabolism. Another pathway of interest included the subgroups signal transduction. In the KEGG analysis, 1501 sequences were classified into 31 signal transduction pathways, including the AMPK signaling pathway, Calcium signaling pathway, Hippo signaling pathway, MAPK signaling pathway, Notch signaling pathway, PI3K-Akt signaling pathway, Rap1 signaling pathway, Ras signaling pathway, cAMP signaling pathway, Phosphatidylinositol signaling system, Wnt signaling pathway, Neuroactive ligand-receptor interaction, and cGMP - PKG signaling pathway. These pathways may play important roles in the sensing and intracellular transduction of stress signals of <em>R. philippinarum</em> under low salinity stress (Fig.&#x000a0;<a href="/pmc/articles/PMC5504028/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span>3</span></a>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="Tab4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p>Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway mapping for <em>R. philippinarum</em>.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">KEGG Pathways</th><th rowspan="1" colspan="1">Sub-Pathways</th><th rowspan="1" colspan="1">Number of Unigenes</th></tr></thead><tbody><tr><td rowspan="4" colspan="1">Cellular Processes</td><td rowspan="1" colspan="1">Cell growth and death</td><td rowspan="1" colspan="1">406</td></tr><tr><td rowspan="1" colspan="1">Cell motility</td><td rowspan="1" colspan="1">174</td></tr><tr><td rowspan="1" colspan="1">Cellular commiunity</td><td rowspan="1" colspan="1">492</td></tr><tr><td rowspan="1" colspan="1">Transport and catabolism</td><td rowspan="1" colspan="1">780</td></tr><tr><td rowspan="3" colspan="1">Environmental Information Processing</td><td rowspan="1" colspan="1">Membrane transport</td><td rowspan="1" colspan="1">90</td></tr><tr><td rowspan="1" colspan="1">Signal transduction</td><td rowspan="1" colspan="1">1501</td></tr><tr><td rowspan="1" colspan="1">Signaling molecules and interaction</td><td rowspan="1" colspan="1">372</td></tr><tr><td rowspan="4" colspan="1">Genetic Information Processing</td><td rowspan="1" colspan="1">Folding, sorting and degradation</td><td rowspan="1" colspan="1">602</td></tr><tr><td rowspan="1" colspan="1">Replication and repair</td><td rowspan="1" colspan="1">178</td></tr><tr><td rowspan="1" colspan="1">Transcription</td><td rowspan="1" colspan="1">290</td></tr><tr><td rowspan="1" colspan="1">Translation</td><td rowspan="1" colspan="1">848</td></tr><tr><td rowspan="11" colspan="1">Metabolism</td><td rowspan="1" colspan="1">Amino acid metabolism</td><td rowspan="1" colspan="1">424</td></tr><tr><td rowspan="1" colspan="1">Biosynthesis of other secondary metabolites</td><td rowspan="1" colspan="1">75</td></tr><tr><td rowspan="1" colspan="1">Carbohydrate metabolism</td><td rowspan="1" colspan="1">485</td></tr><tr><td rowspan="1" colspan="1">Energy metabolism</td><td rowspan="1" colspan="1">326</td></tr><tr><td rowspan="1" colspan="1">Glycan biosynthesis and metabolism</td><td rowspan="1" colspan="1">264</td></tr><tr><td rowspan="1" colspan="1">Lipid metabolism</td><td rowspan="1" colspan="1">446</td></tr><tr><td rowspan="1" colspan="1">Metabolism of cofactors and vitamins</td><td rowspan="1" colspan="1">238</td></tr><tr><td rowspan="1" colspan="1">Metabolism of other amino acids</td><td rowspan="1" colspan="1">191</td></tr><tr><td rowspan="1" colspan="1">Metabolism of terpenoids and polyketides</td><td rowspan="1" colspan="1">56</td></tr><tr><td rowspan="1" colspan="1">Nucleotide metabolism</td><td rowspan="1" colspan="1">303</td></tr><tr><td rowspan="1" colspan="1">Xenobiotics biodegradation and metabolism</td><td rowspan="1" colspan="1">175</td></tr><tr><td rowspan="9" colspan="1">Organismal Systems</td><td rowspan="1" colspan="1">Circulatory system</td><td rowspan="1" colspan="1">260</td></tr><tr><td rowspan="1" colspan="1">Development</td><td rowspan="1" colspan="1">241</td></tr><tr><td rowspan="1" colspan="1">Digestive system</td><td rowspan="1" colspan="1">431</td></tr><tr><td rowspan="1" colspan="1">Endocrine system</td><td rowspan="1" colspan="1">827</td></tr><tr><td rowspan="1" colspan="1">Environmental adaptation</td><td rowspan="1" colspan="1">215</td></tr><tr><td rowspan="1" colspan="1">Excretory system</td><td rowspan="1" colspan="1">160</td></tr><tr><td rowspan="1" colspan="1">Immune system</td><td rowspan="1" colspan="1">519</td></tr><tr><td rowspan="1" colspan="1">Nervous system</td><td rowspan="1" colspan="1">522</td></tr><tr><td rowspan="1" colspan="1">Sensory system</td><td rowspan="1" colspan="1">211</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139982524312112"><a target="object" rel="noopener" href="/pmc/articles/PMC5504028/table/Tab4/?report=objectonly">Open in a separate window</a></div></div></div><div id="Sec9" class="sec sec-last"><h3 id="Sec9title">Validation of RNA-Seq results with RT-qPCR</h3><p id="Par13" class="p p-first-last">To confirm the differentially expressed genes identified in the RNA-Seq expression analysis, we selected 12 genes from the gene expression heatmap for RT&#x02013;qPCR validation from those with differing expression patterns among the genes of interest based on their functions. The primers were designed according to the contig sequences (Table&#x000a0;<a href="/pmc/articles/PMC5504028/table/Tab5/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab5" rid-ob="ob-Tab5" co-legend-rid=""><span>5</span></a>). A melting-curve analysis revealed a single product for all the tested genes, indicating the high reliability of the transcriptome assembly. The fold changes detected with RT&#x02013;qPCR were compared with those detected with the RNA-Seq expression analysis (Fig.&#x000a0;<a href="/pmc/articles/PMC5504028/figure/Fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig7" rid-ob="ob-Fig7" co-legend-rid="lgnd_Fig7"><span>7</span></a>). As shown in Fig.&#x000a0;<a href="/pmc/articles/PMC5504028/figure/Fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig7" rid-ob="ob-Fig7" co-legend-rid="lgnd_Fig7"><span>7</span></a>, the RT&#x02013;qPCR results correlated significantly with the RNA-Seq results (<em>P</em>&#x02009;&#x0003c;&#x02009;0.05). In general, the RNA-Seq results were confirmed by the RT&#x02013;qPCR results, indicating the reliability and accuracy of the RNA-Seq expression analysis.

</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="Tab5"><h3>Table 5</h3><!--caption a7--><div class="caption"><p>Combination of primers used in RT-qPCR assays.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="left" rowspan="1" colspan="1">Gene</th><th align="left" rowspan="1" colspan="1">Gene ID</th><th align="left" rowspan="1" colspan="1">Oligonucleotide</th><th align="left" rowspan="1" colspan="1">Sequence</th></tr></thead><tbody><tr><td rowspan="2" colspan="1">BIRC7</td><td rowspan="2" colspan="1">c132162_g1</td><td rowspan="1" colspan="1">Forward primer</td><td rowspan="1" colspan="1">CATCTTGTTGTGCAGGGTTG</td></tr><tr><td rowspan="1" colspan="1">Reverse primer</td><td rowspan="1" colspan="1">TTCCGAGTTAAGAGCGCAAT</td></tr><tr><td rowspan="2" colspan="1">GPT2</td><td rowspan="2" colspan="1">c144470_g2</td><td rowspan="1" colspan="1">Forward primer</td><td rowspan="1" colspan="1">ATCTGGGCTGGATGGATGT</td></tr><tr><td rowspan="1" colspan="1">Reverse primer</td><td rowspan="1" colspan="1">GTGCTGGACAAAGTGTTGGTT</td></tr><tr><td rowspan="2" colspan="1">ATPGD1</td><td rowspan="2" colspan="1">c135860_g1</td><td rowspan="1" colspan="1">Forward primer</td><td rowspan="1" colspan="1">ATGTTCGGTCGACAGTTTCC</td></tr><tr><td rowspan="1" colspan="1">Reverse primer</td><td rowspan="1" colspan="1">TTAGATCCGCCGAAATCTTG</td></tr><tr><td rowspan="2" colspan="1">Cathepsin L</td><td rowspan="2" colspan="1">c154587_g1</td><td rowspan="1" colspan="1">Forward primer</td><td rowspan="1" colspan="1">TATTTCTGACCGCCCATTTC</td></tr><tr><td rowspan="1" colspan="1">Reverse primer</td><td rowspan="1" colspan="1">TCCCATAAGTCCCTGTCGTT</td></tr><tr><td rowspan="2" colspan="1">RhoJ</td><td rowspan="2" colspan="1">c132087_g1</td><td rowspan="1" colspan="1">Forward primer</td><td rowspan="1" colspan="1">TGGCTACTCAAACTGACCTGA</td></tr><tr><td rowspan="1" colspan="1">Reverse primer</td><td rowspan="1" colspan="1">ACAAATGAATCCGCACCA</td></tr><tr><td rowspan="2" colspan="1">IAP</td><td rowspan="2" colspan="1">c149735_g1</td><td rowspan="1" colspan="1">Forward primer</td><td rowspan="1" colspan="1">TTCCAACAGACGACATCCAA</td></tr><tr><td rowspan="1" colspan="1">Reverse primer</td><td rowspan="1" colspan="1">GTTTGTACTCGGCCAAGAGG</td></tr><tr><td rowspan="2" colspan="1">VWDE</td><td rowspan="2" colspan="1">c135387_g1</td><td rowspan="1" colspan="1">Forward primer</td><td rowspan="1" colspan="1">CCCGATTCTACGCATTTACC</td></tr><tr><td rowspan="1" colspan="1">Reverse primer</td><td rowspan="1" colspan="1">GACCTCCAACCAACAGTCGT</td></tr><tr><td rowspan="2" colspan="1">SLC12A9</td><td rowspan="2" colspan="1">c151354_g1</td><td rowspan="1" colspan="1">Forward primer</td><td rowspan="1" colspan="1">AACTCTGGCGGTCATTCTTG</td></tr><tr><td rowspan="1" colspan="1">Reverse primer</td><td rowspan="1" colspan="1">TTGCTTTCGGCACTTCAAC</td></tr><tr><td rowspan="2" colspan="1">GLUS</td><td rowspan="2" colspan="1">c139767_g1</td><td rowspan="1" colspan="1">Forward primer</td><td rowspan="1" colspan="1">GAGTCGGTCAATAGCCTCCT</td></tr><tr><td rowspan="1" colspan="1">Reverse primer</td><td rowspan="1" colspan="1">CAAACCAATGCCAGGTGA</td></tr><tr><td rowspan="2" colspan="1">GLUD1</td><td rowspan="2" colspan="1">c153037_g1</td><td rowspan="1" colspan="1">Forward primer</td><td rowspan="1" colspan="1">CCCAGCACAGTCAACACAG</td></tr><tr><td rowspan="1" colspan="1">Reverse primer</td><td rowspan="1" colspan="1">GTCCACAACGGCACATTTAT</td></tr><tr><td rowspan="2" colspan="1">Ubiquitin</td><td rowspan="2" colspan="1">c140036_g1</td><td rowspan="1" colspan="1">Forward primer</td><td rowspan="1" colspan="1">CTGCCTTCTGGACCTGTGAG</td></tr><tr><td rowspan="1" colspan="1">Reverse primer</td><td rowspan="1" colspan="1">CAAGTGAACGACGAAACAAATAGTC</td></tr><tr><td rowspan="2" colspan="1">ARHGEF12</td><td rowspan="2" colspan="1">c151340_g2</td><td rowspan="1" colspan="1">Forward primer</td><td rowspan="1" colspan="1">TAGCAACAGTCCGTGCGTTA</td></tr><tr><td rowspan="1" colspan="1">Reverse primer</td><td rowspan="1" colspan="1">GAGTGCCATCATTGGGTGA</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139982522413488"><a target="object" rel="noopener" href="/pmc/articles/PMC5504028/table/Tab5/?report=objectonly">Open in a separate window</a></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig7" co-legend-rid="lgnd_Fig7"><a href="/pmc/articles/PMC5504028/figure/Fig7/" target="figure" rid-figpopup="Fig7" rid-ob="ob-Fig7"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139982525057248"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 41598_2017_5397_Fig7_HTML.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is 41598_2017_5397_Fig7_HTML.jpg" src="/pmc/articles/PMC5504028/bin/41598_2017_5397_Fig7_HTML.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm139982525057248"><a target="object" rel="noopener" href="/pmc/articles/PMC5504028/figure/Fig7/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig7"><div><a class="figpopup" href="/pmc/articles/PMC5504028/figure/Fig7/" target="figure" rid-figpopup="Fig7" rid-ob="ob-Fig7">Figure 7</a></div><!--caption a7--><div class="caption"><p>Validation of RNA-Seq results using RT&#x02013;qPCR. The transcript expression levels of the selected genes were normalized to that of the &#x003b2;-actin gene.</p></div></div></div></div></div><div id="Sec10" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec10title">Discussion</h2><p id="Par14" class="p p-first">High throughput sequencing of RNA transcriptomes were conducted in <em>R. philippinarum</em> to identify genes involved in osmotic stress response. Follow-up work will be based on the results gained here as a foundation for more-targeted studies. No previous study has examined the transcriptional response to low salinity stress in the Manila clam. Future studies will use the target genes identified here to examine individuals, strains, and families. To validate the RNA-Seq results, the expression of 12 low-salinity-responsive genes that are involved in the processes of stress response, energy metabolism, substance metabolism, and immune response, was measured with RT&#x02013;qPCR. These genes may reflect different aspects of the physiological adaptation mechanism to low salinity environments in <em>R. philippinarum</em>.</p><p id="Par15">In this study, we identified a series of genes related to osmolality tolerance and low salinity stress. These data showed that the enriched genes related to signal transduction, metabolic processes, responses to stimuli, oxidation-reduction process, and so on, implicating these processes in the regulation of the response to low salinity stress in gills of <em>R. philippinarum</em>. Molecular function related to the carbohydrate binding, chitin binding, calcium ion binding were also significantly represented. The important economic traits of resistance and immunity have been the focus of much work on economically valuable marine shellfish<sup><sup><a href="#CR27" rid="CR27" class=" bibr popnode">27</a></sup>&#x02013;<sup><a href="#CR30" rid="CR30" class=" bibr popnode">30</a></sup></sup>. The unigene and annotation information from the Nr, Swiss-Prot, GO, and KEGG databases are valuable genomic resources for <em>R. philippinarum</em> and will allow further study of the molecular basis of its low salinity resistance and the energetic system.</p><p id="Par16">Osmotic stress tolerance is important for the adaptation and survival of the Manila clam under low salinity stress. Organisms exposed to low salinity experience more oxidative stress than animals living at optimum salinity<sup><sup><a href="#CR12" rid="CR12" class=" bibr popnode">12</a></sup></sup>. The expression of oxidoreductase proteins was indicated insofar as glutamate dehydrogenase 1 (GLUD1), were significantly overexpressed, consistent with the transcriptomic results presented here. In this study, the gene encoding glutamine synthetase (GLUS) and GLUD1 was significantly upregulated under low salinity stress according to the RT&#x02013;qPCR analysis (Fig.&#x000a0;<a href="/pmc/articles/PMC5504028/figure/Fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig7" rid-ob="ob-Fig7" co-legend-rid="lgnd_Fig7"><span>7</span></a>), suggesting that the osmolality tolerance of <em>R. philippinarum</em> is probably influenced by the nitrogen and glutamate metabolism and the energy homeostasis. GLUD1 is a mitochondrial matrix enzyme, with a key role in the nitrogen and glutamate metabolism and the energy homeostasis, while the GLUS gene plays an essential role in the metabolism of nitrogen by catalyzing the condensation of glutamate and ammonia to form glutamine<sup><sup><a href="#CR31" rid="CR31" class=" bibr popnode">31</a></sup></sup>.</p><p id="Par17">During cell proliferation and tissue repair, materials for cell growth must be provided. In aquatic organisms, adenosine-5&#x02032;-triphosphate (ATP), the major energy source for cells in the body, is predominantly supplied by a series of metabolic pathways, including glycolysis, the citric acid cycle, and the electron transport chain<sup><sup><a href="#CR32" rid="CR32" class=" bibr popnode">32</a></sup></sup>. Among these, glycolysis occurs (with variations) in nearly all organisms, both aerobic and anaerobic. The wide occurrence of glycolysis indicates that it is one of the most important energy metabolic pathways<sup><sup><a href="#CR31" rid="CR31" class=" bibr popnode">31</a></sup></sup>. It provides not only high-energy compounds like ATP, but also pyruvate, which can be used in the citric acid cycle to generate more ATP, NADH, and FADH<sub>2</sub>
<sup><sup><a href="#CR33" rid="CR33" class=" bibr popnode">33</a></sup>&#x02013;<sup><a href="#CR35" rid="CR35" class=" bibr popnode">35</a></sup></sup>. In this study, ATP-grasp domain-containing protein 1 (ATPGD1) was upregulated in response to low salinity stress. It has been reported that ATPGD1 is involved in metabolic process and catalyzes the degradation of beta-alanine in KEGG to maintain osmotic equilibrium under hypo-osmotic stress in the oysters<sup><sup><a href="#CR36" rid="CR36" class=" bibr popnode">36</a></sup></sup>. Its expression was found to increase significantly in the <em>C. gigas</em> on the 7th day after hypo-osmotic stress, and reached the highest level under the condition with salinity of 10<sup><sup><a href="#CR36" rid="CR36" class=" bibr popnode">36</a></sup></sup>.</p><p id="Par18">Many studies have revealed that intracellular free amino acids (FAAs) predominantly contribute to intracellular osmolality and to cell volume regulation in oysters<sup><sup><a href="#CR37" rid="CR37" class=" bibr popnode">37</a></sup></sup>. The functions of FAAs as osmolytes have been reported by previous studies<sup><sup><a href="#CR38" rid="CR38" class=" bibr popnode">38</a></sup></sup>. In this study, the FAAs metabolic pathways including glycine, alanine, aspartate, glutamate, beta-alanine, arginine, proline, and taurine metabolic pathways were found in the KEGG enrichment analysis, and especially key enzyme genes involved in these pathways, such as ATPGD, CSAD, and P5CS were activated, altering the osmotic status in the bivalves and allowing them to adapt to osmotic stress conditions<sup><sup><a href="#CR39" rid="CR39" class=" bibr popnode">39</a></sup></sup>. These pathways were observed to be expanded in intertidal animals compared with stenohaline terrestrial animals, providing one of the most important explanations for shellfish adaptation to the intertidal zone<sup><sup><a href="#CR8" rid="CR8" class=" bibr popnode">8</a></sup></sup>. Previous studies demonstrated that stress-induced immune changes have been found in many marine invertebrates, including oyster<sup><sup><a href="#CR40" rid="CR40" class=" bibr popnode">40</a></sup>, <sup><a href="#CR41" rid="CR41" class=" bibr popnode">41</a></sup></sup> and mussel<sup><sup><a href="#CR42" rid="CR42" class=" bibr popnode">42</a></sup>, <sup><a href="#CR43" rid="CR43" class=" bibr popnode">43</a></sup></sup>. The innate immunity is the only immunological defense mechanism in invertebrate metazoan<sup><sup><a href="#CR44" rid="CR44" class=" bibr popnode">44</a></sup></sup>. In addition, phagocytosis by immune cells is the predominant mechanism of marine invertebrates defense<sup><sup><a href="#CR45" rid="CR45" class=" bibr popnode">45</a></sup></sup>, which is the same to <em>R. philippinarum</em>. Up-regulated genes are all related to focal adhesion, NF-kappa B signaling pathway, apoptosis, TNF signaling pathway, and amino acid synthesis and metabolism, so we inferred that these pathways probably play significant roles in the antioxidant response and immune response associated with the low salinity stress response. This study is expect to provides useful genetic resources for further research on marine bivalves osmoregulation and their stress-induced responses to environmental changes.</p><p id="Par19" class="p p-last">In summary, we have reported a comprehensive transcript dataset for the manila clam <em>R. philippinarum</em>. The transcripts identified and annotated provide valuable genomic resources that extend our understanding of the unique biological characteristics of this economically important species, such as the low salinity tolerance and antioxidant processes of this species. This study has shown that signaling, membrane, transporter activity, receptor activity, macromolecular complex, cellular component organization or biogenesis, and molecular transducer activity are the most highly enriched pathways under hypo-osmotic stress, inferred from the genes differentially expressed at low salinity. To our knowledge, this work is the first to systematically identify the genes that respond to low salinity in <em>R. philippinarum</em>, which enriches the gene expression profile data of the species and will allow environmental, molecular, and physiological studies of bivalves to be performed.</p></div><div id="Sec11" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec11title">Materials and Methods</h2><div id="Sec12" class="sec sec-first"><h3 id="Sec12title">Clam materials and RNA extraction</h3><p id="Par20" class="p p-first">Two color strains of <em>R. philippinarum</em> (zebra and orange) were collected from the Zhangzidao aquaculture farm in Dalian, Liaoning Province, China. The clams with a similar mean shell length averaging at 19.03&#x02009;&#x000b1;&#x02009;0.93 mm, and 20.90&#x02009;&#x000b1;&#x02009;1.34 mm, respectively, were used in this study. After being transported from the field to the laboratory, the clams were cleaned to remove any fouling and were acclimated in aerated 100&#x02009;L plastic tanks, containing water at 15&#x02009;&#x000b0;C with salinity of 30 ppt. Other water quality were measured during the experiment (pH: 8.2&#x02009;&#x000b1;&#x02009;0.1, dissolved oxygen: 8.0&#x02009;&#x000b1;&#x02009;0.2&#x02009;mg/L). The water was exchanged once per day to discharge waste products from marine invertebrates. All the clams were acclimated in lab for two weeks and fed with Spirulina powder daily. The Manila clam is not an endangered or protected species, so no specific permits were required for the study. Forty clams were divided into 2 groups (10 orange and 10 zebra clams for each group). One group was kept in normal salinity water (salt 30 ppt) and used as control (SRp1, SRp2). The other group (FRp1, FRp2) was subjected to salinity change: the salinity decreased for 6 days, 5 ppt per day. After 6 days gradual hypo-osmotic stress, clams were reared in low salinity water (salt 5 ppt) for one day. The water quality did not change during the experiment (Temperature: 15&#x02009;&#x000b0;C, pH: 8.2&#x02009;&#x000b1;&#x02009;0.1, dissolved oxygen: 8.0&#x02009;&#x000b1;&#x02009;0.2&#x02009;mg/L). The vitality and motility of the clams suffered no adverse affects when the salinity changed. The gills from orange and zebra clams were collected, respectively, and then immediately frozen in liquid nitrogen and stored at &#x02212;80&#x02009;&#x000b0;C until they were processed for RNA extraction.</p><p id="Par21" class="p p-last">The total RNA was extracted from 30&#x02009;mg tissue samples from 1 individual using RNAprep pure Tissue Kit (TianGene, Beijing, China), according to the manufacturer&#x02019;s protocol. RNA purity was determined with a NanoPhotometer&#x000ae; spectrophotometer (Implen, CA, USA). The RNA concentration was measured with the Qubit&#x000ae; RNA Assay Kit in a Qubit&#x000ae; 2.0 Fluorometer (Life Technologies, CA, USA). RNA integrity was assessed with the RNA Nano 6000 Assay Kit of the Agilent Bioanalyzer 2100 system (Agilent Technologies, CA, USA).</p></div><div id="Sec13" class="sec"><h3 id="Sec13title">Library preparation for Transcriptome sequencing</h3><p id="Par22" class="p p-first-last">To assess the transcriptomes of the clams and obtain a quantitative and qualitative gene expression database for low salinity stress, four <em>R. philippinarum</em> cDNA libraries representing the control group (SRp1, SRp2) and the treated group (FRp1, FRp2) were constructed. The libraries were subjected to Illumina paired-end sequencing, identifying representative transcripts for a wide range of biological processes. A total amount of 3&#x02009;&#x003bc;g of RNA per sample was used as the input material for RNA sample preparation. The sequencing libraries were generated using the NEBNext&#x000ae; Ultra&#x02122; RNA Library Prep Kit for Illumina&#x000ae; (NEB, USA), according to the manufacturer&#x02019;s recommendations, and index codes were added to attribute the sequences to each sample. To preferentially select cDNA fragments of 150&#x02013;200&#x02009;bp, the library fragments were purified with the AMPure XP system (Beckman Coulter, Beverly, USA). The size-selected adaptor-ligated cDNA was incubated with 3&#x02009;&#x003bc;l of USER Enzyme (NEB, USA) at 37&#x02009;&#x000b0;C for 15&#x02009;min, and then at 95&#x02009;&#x000b0;C for 5&#x02009;min before PCR. The PCR was performed with Phusion High-Fidelity DNA Polymerase, universal PCR primers, and Index (X) Primer. The PCR products were purified (AMPure XP system) and the quality of the library was assessed with the Agilent Bioanalyzer 2100 system.</p></div><div id="Sec14" class="sec"><h3 id="Sec14title">Assembly and annotation</h3><p id="Par23" class="p p-first-last">The raw sequences were deposited in the National Center for Biotechnology Information (NCBI) Short Read Archive (SRA) database (<a href="http://www.ncbi.nlm.nih.gov/Traces/sra/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/Traces/sra/</a>). After the adapter sequences were removed, and the ambiguous &#x02018;N&#x02019; (unknown nucleotide)-containing sequences (N&#x0003e;10%) and low-quality sequences (quality score&#x02009;&#x0003c;&#x02009;5) were excluded, the remaining clean reads were assembled with the Trinity software, as described for de novo transcriptome assembly without a reference genome. The nonredundant sequences were checked against public databases for homology annotation, including the NCBI nonredundant protein (Nr) and nonredundant nucleotide (Nt) databases (<a href="http://www.ncbi.nlm.nih.gov/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/</a>), Swiss-Prot (<a href="http://www.ebi.ac.uk/uniprot/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ebi.ac.uk/uniprot/</a>), Gene Ontology (GO) (<a href="http://www.geneontology.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.geneontology.org/</a>), Clusters of Orthologous Groups (COG) (<a href="http://www.ncbi.nlm.nih.gov/COG/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/COG/</a>), and Kyoto Encyclopedia of Genes and Genomes (KEGG) (<a href="http://www.genome.jp/kegg/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genome.jp/kegg/</a>). If the results for the different databases were conflicting, a priority order of alignments (Nr&#x0003e;Nt&#x0003e;KEGG&#x0003e;Swiss-Prot&#x0003e;GO&#x0003e;COG databases) was followed. The sequences were compared with the Nr, Nt, and Swiss-Prot databases using the BLASTX algorithm, with an E-value cut-off of 10<sup>&#x02212;10</sup>. GO terms at the second level were used for the GO annotation. The COG and KEGG classifications were performedwith BLASTX, with an E-value cutoff of 10<sup>&#x02212;5</sup>. Benchmarking Universal Single-Copy Orthologs (BUSCO) was used for completeness score of the transcriptome assembly<sup><sup><a href="#CR46" rid="CR46" class=" bibr popnode">46</a></sup></sup>.</p></div><div id="Sec15" class="sec"><h3 id="Sec15title">Differential expression analysis</h3><p id="Par24" class="p p-first-last">Before the differential gene expression analysis, the read counts were adjusted for each sequenced library using the edgeR program package with one scaling normalization factor<sup><sup><a href="#CR47" rid="CR47" class=" bibr popnode">47</a></sup></sup>. To determine the differences in the low salinity tolerant and control <em>R. philippinarum</em> transcriptomes more accurately, we used DEGseq<sup><sup><a href="#CR48" rid="CR48" class=" bibr popnode">48</a></sup></sup> to screen the differentially expressed genes with a relation model that chose SRp1 and SRp2 as the control group and compared it with the FRp1 and FRp2 transcriptome library. <em>P</em> values&#x02009;&#x0003c;&#x02009;0.05 and |log2(fold change)|&#x0003e;1were set as the threshold criteria for significantly different expression.</p></div><div id="Sec16" class="sec sec-last"><h3 id="Sec16title">Real-time RT-PCR confirmation of Illumina sequencing data</h3><p id="Par25" class="p p-first">To validate our Illumina sequencing data, 12 differentially expressed genes were selected for quantitative RT&#x02013;PCR (RT&#x02013;qPCR) analysis, using the same FRp1, FRp2 and SRp1, SRp2 group RNA samples as were used for transcriptome profiling. The primers were designed with the Primer5 software (Premier Biosoft International). The &#x003b2;-actin gene was used as the internal control for the qPCR analysis. RNA (500 ng) from each sample was measured and treated with RQ1 RNase-Free DNase (Promega) to remove any genomic DNA. The cDNA was synthesized from the treated RNA using a reverse transcriptase reagent kit (PrimeScript&#x02122; RT reagent Kit, Takara).</p><p id="Par26" class="p p-last">The qPCR was performed with SYBR Premix Ex Taq II (Takara). The reactions were carried out in a total volume of 25&#x02009;&#x003bc;l containing 2.5&#x02009;&#x003bc;l of diluted cDNA, 2.5&#x02009;&#x003bc;l of each primer, and 12.5&#x02009;&#x003bc;l of SYBR Green PCR Master Mix, with the following cycling profile: 95&#x02009;&#x000b0;C for 15&#x02009;min for polymerase activation, followed by 40 cycles at 95&#x02009;&#x000b0;C for 15&#x02009;s, at 55&#x02009;&#x000b0;C for 30&#x02009;s, and at 70&#x02009;&#x000b0;C for 30&#x02009;s. Each sample was processed in triplicate in the Roche LightCycler 480 Real-Time PCR System (Roche). All data were analyzed using the 2<sup>&#x02212;&#x00394;&#x00394;Ct</sup> method<sup><sup><a href="#CR49" rid="CR49" class=" bibr popnode">49</a></sup></sup>.</p></div></div><div id="ack-a.u.b" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.u.btitle">Acknowledgements</h2><div class="sec"><p>The study was supported by grants from the National Natural Science Foundation of China (31302183), the Modern Agro-industry Technology Research System (CARS-48), the Program for Liaoning Innovative Research Team in University (LT2014015), the Program for Liaoning Excellent Talents in University (LJQ2014076), the Cultivation Plan for Youth Agricultural Science and Technology Innovative Talents of Liaoning Province (2014004), and the Natural Science Foundation of Dalian (2016RQ065; 2014B11NC092).</p></div></div><div id="notes-a.u.c" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.u.ctitle">Author Contributions</h2><p>H.T.N. conducted preparation of the transciptome data, bioinformatics analysis and manuscript preparation. L.W.J., P.C. and Z.M.H. were involved in one or more processes of sample collection, RNA extraction or experimental analysis. F.Y. and X.W.Y. conceived, designed and guided the research. All authors read and approved the final manuscript.</p></div><div id="notes-a.u.d" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.u.dtitle">Notes</h2><div id="FPar1" class="sec sec-first"><h2 id="FPar1title">Competing Interests</h2><p id="Par27" class="p p-first-last">The authors declare that they have no competing interests.</p></div></div><div id="fn-group-a.u.a" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.u.atitle">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="fn-a.u.a.a"><p class="p p-first-last">
<strong>Publisher's note:</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></p></div></div><div id="Bib1" class="tsec sec"><h2 class="head no_bottom_margin" id="Bib1title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="CR1">1. <span class="element-citation">De Lisle PF, Roberts MH., Jr. The effect of salinity on cadmium toxicity to the estuarine mysid <em>Mysidopsis bahia</em>: role of chemical speciation. <span><span class="ref-journal">Aquat. Toxicol. </span>1988;<span class="ref-vol">12</span>(4):357–370. doi: 10.1016/0166-445X(88)90062-8.</span> [<a href="//doi.org/10.1016%2F0166-445X(88)90062-8" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Aquat.+Toxicol.&amp;title=The+effect+of+salinity+on+cadmium+toxicity+to+the+estuarine+mysid+Mysidopsis+bahia:+role+of+chemical+speciation&amp;author=PF+De+Lisle&amp;author=MH+Roberts&amp;volume=12&amp;issue=4&amp;publication_year=1988&amp;pages=357-370&amp;doi=10.1016/0166-445X(88)90062-8&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR2">2. <span class="element-citation">Matsuda M, et al. Effects of low salinity on survival, hemolymph osmolality and tissue water content of the Manila clam <em>Ruditapes philippinarum</em>. <em>Aquaculture</em>. <span><span class="ref-journal">Science. </span>2008;<span class="ref-vol">56</span>(1):127–136.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Effects+of+low+salinity+on+survival,+hemolymph+osmolality+and+tissue+water+content+of+the+Manila+clam+Ruditapes+philippinarum.+Aquaculture&amp;author=M+Matsuda&amp;volume=56&amp;issue=1&amp;publication_year=2008&amp;pages=127-136&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR3">3. <span class="element-citation">Mc Farland K, Donaghy L, Volety AK. Effect of acute salinity changes on hemolymph osmolality and clearance rate of the non-native mussel, <em>Perna viridis</em>, and the native oyster, <em>Crassostrea virginica</em>, in Southwest Florida. <span><span class="ref-journal">Aquat. Invasions. </span>2013;<span class="ref-vol">8</span>(3):299–310. doi: 10.3391/ai.2013.8.3.06.</span> [<a href="//doi.org/10.3391%2Fai.2013.8.3.06" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Aquat.+Invasions&amp;title=Effect+of+acute+salinity+changes+on+hemolymph+osmolality+and+clearance+rate+of+the+non-native+mussel,+Perna+viridis,+and+the+native+oyster,+Crassostrea+virginica,+in+Southwest+Florida&amp;author=K+Mc+Farland&amp;author=L+Donaghy&amp;author=AK+Volety&amp;volume=8&amp;issue=3&amp;publication_year=2013&amp;pages=299-310&amp;doi=10.3391/ai.2013.8.3.06&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR4">4. <span class="element-citation">Gunter G. Some relations of estuarine organisms to salinity. <span><span class="ref-journal">Limnol. Oceanogr. </span>1961;<span class="ref-vol">6</span>:182–190. doi: 10.4319/lo.1961.6.2.0182.</span> [<a href="//doi.org/10.4319%2Flo.1961.6.2.0182" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Limnol.+Oceanogr.&amp;title=Some+relations+of+estuarine+organisms+to+salinity&amp;author=G+Gunter&amp;volume=6&amp;publication_year=1961&amp;pages=182-190&amp;doi=10.4319/lo.1961.6.2.0182&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR5">5. <span class="element-citation">Berger VJ, Kharazova AD. Mechanisms of salinity adaptations in marine molluscs. <span><span class="ref-journal">Hydrobiologia. </span>1997;<span class="ref-vol">355</span>:115–126. doi: 10.1023/A:1003023322263.</span> [<a href="//doi.org/10.1023%2FA%3A1003023322263" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hydrobiologia&amp;title=Mechanisms+of+salinity+adaptations+in+marine+molluscs&amp;author=VJ+Berger&amp;author=AD+Kharazova&amp;volume=355&amp;publication_year=1997&amp;pages=115-126&amp;doi=10.1023/A:1003023322263&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR6">6. <span class="element-citation">Zhao X, Yu H, Kong L, Liu S, Li Q. Comparative transcriptome analysis of two oysters, <em>Crassostrea gigas</em> and <em>Crassostrea hongkongensis</em> provides insights into adaptation to hypo-osmotic conditions. <span><span class="ref-journal">PLoS ONE. </span>2014;<span class="ref-vol">9</span>(11):e111915. doi: 10.1371/journal.pone.0111915.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4219811/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25369077" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pone.0111915" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Comparative+transcriptome+analysis+of+two+oysters,+Crassostrea+gigas+and+Crassostrea+hongkongensis+provides+insights+into+adaptation+to+hypo-osmotic+conditions&amp;author=X+Zhao&amp;author=H+Yu&amp;author=L+Kong&amp;author=S+Liu&amp;author=Q+Li&amp;volume=9&amp;issue=11&amp;publication_year=2014&amp;pages=e111915&amp;pmid=25369077&amp;doi=10.1371/journal.pone.0111915&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR7">7. <span class="element-citation">Zhao X, Yu H, Kong L, Li Q. Transcriptomic Responses to Salinity Stress in the Pacific Oyster <em>Crassostrea gigas</em>. <span><span class="ref-journal">PLoS ONE. </span>2012;<span class="ref-vol">7</span>(9):e46244. doi: 10.1371/journal.pone.0046244.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3459877/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23029449" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pone.0046244" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Transcriptomic+Responses+to+Salinity+Stress+in+the+Pacific+Oyster+Crassostrea+gigas&amp;author=X+Zhao&amp;author=H+Yu&amp;author=L+Kong&amp;author=Q+Li&amp;volume=7&amp;issue=9&amp;publication_year=2012&amp;pages=e46244&amp;pmid=23029449&amp;doi=10.1371/journal.pone.0046244&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR8">8. <span class="element-citation">Meng J, et al. Genome and Transcriptome Analyses Provide Insight into the Euryhaline Adaptation Mechanism of <em>Crassostrea gigas</em>. <span><span class="ref-journal">PLoS ONE. </span>2013;<span class="ref-vol">8</span>(3):e58563. doi: 10.1371/journal.pone.0058563.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3595286/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23554902" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pone.0058563" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Genome+and+Transcriptome+Analyses+Provide+Insight+into+the+Euryhaline+Adaptation+Mechanism+of+Crassostrea+gigas&amp;author=J+Meng&amp;volume=8&amp;issue=3&amp;publication_year=2013&amp;pages=e58563&amp;pmid=23554902&amp;doi=10.1371/journal.pone.0058563&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR9">9. <span class="element-citation">Shekhar MS, Kiruthika J, Ponniah AG. Identification and expression analysis of differentially expressed genes from shrimp (<em>Penaeus monodon</em>) in response to low salinity stress. <span><span class="ref-journal">Fish Shellfish Immun. </span>2013;<span class="ref-vol">35</span>:1957–1968. doi: 10.1016/j.fsi.2013.09.038.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24436977" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.fsi.2013.09.038" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Fish+Shellfish+Immun&amp;title=Identification+and+expression+analysis+of+differentially+expressed+genes+from+shrimp+(Penaeus+monodon)+in+response+to+low+salinity+stress&amp;author=MS+Shekhar&amp;author=J+Kiruthika&amp;author=AG+Ponniah&amp;volume=35&amp;publication_year=2013&amp;pages=1957-1968&amp;doi=10.1016/j.fsi.2013.09.038&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR10">10. <span class="element-citation">Chen K, et al. Transcriptome and Molecular Pathway Analysis of the Hepatopancreas in the Pacific White Shrimp <em>Litopenaeus vannamei</em> under Chronic Low-Salinity Stress. <span><span class="ref-journal">PLoS ONE. </span>2015;<span class="ref-vol">10</span>(7):e0131503. doi: 10.1371/journal.pone.0131503.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4492601/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26147449" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pone.0131503" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Transcriptome+and+Molecular+Pathway+Analysis+of+the+Hepatopancreas+in+the+Pacific+White+Shrimp+Litopenaeus+vannamei+under+Chronic+Low-Salinity+Stress&amp;author=K+Chen&amp;volume=10&amp;issue=7&amp;publication_year=2015&amp;pages=e0131503&amp;pmid=26147449&amp;doi=10.1371/journal.pone.0131503&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR11">11. <span class="element-citation">Lv J, et al. Transcriptome Analysis of <em>Portunus trituberculatus</em> in Response to Salinity Stress Provides Insights into the Molecular Basis of Osmoregulation. <span><span class="ref-journal">PLoS ONE. </span>2013;<span class="ref-vol">8</span>(12):e82155. doi: 10.1371/journal.pone.0082155.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3849447/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24312639" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pone.0082155" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Transcriptome+Analysis+of+Portunus+trituberculatus+in+Response+to+Salinity+Stress+Provides+Insights+into+the+Molecular+Basis+of+Osmoregulation&amp;author=J+Lv&amp;volume=8&amp;issue=12&amp;publication_year=2013&amp;pages=e82155&amp;pmid=24312639&amp;doi=10.1371/journal.pone.0082155&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR12">12. <span class="element-citation">Hui M, et al. Transcriptome Changes in <em>Eriocheir sinensis</em> Megalopae after Desalination Provide Insights into Osmoregulation and Stress Adaption in Larvae. <span><span class="ref-journal">PLoS ONE. </span>2014;<span class="ref-vol">9</span>(12):e114187. doi: 10.1371/journal.pone.0114187.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4254945/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25470496" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pone.0114187" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Transcriptome+Changes+in+Eriocheir+sinensis+Megalopae+after+Desalination+Provide+Insights+into+Osmoregulation+and+Stress+Adaption+in+Larvae&amp;author=M+Hui&amp;volume=9&amp;issue=12&amp;publication_year=2014&amp;pages=e114187&amp;pmid=25470496&amp;doi=10.1371/journal.pone.0114187&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR13">13. <span class="element-citation">Sokolova I, Berger VJ. Physiological variation related to shell colour polymorphism in White Sea <em>Littorina saxatilis</em>. <span><span class="ref-journal">J. Exp. Mar. Biol. Ecol. </span>2000;<span class="ref-vol">245</span>:1–23. doi: 10.1016/S0022-0981(99)00132-X.</span> [<a href="//doi.org/10.1016%2FS0022-0981(99)00132-X" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Exp.+Mar.+Biol.+Ecol.&amp;title=Physiological+variation+related+to+shell+colour+polymorphism+in+White+Sea+Littorina+saxatilis&amp;author=I+Sokolova&amp;author=VJ+Berger&amp;volume=245&amp;publication_year=2000&amp;pages=1-23&amp;doi=10.1016/S0022-0981(99)00132-X&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR14">14. <span class="mixed-citation">Zhang, G. F. &#x00026; Yan, X. W. Clam aquaculture. Science Press, Beijing, China (in Chinese) (2010).</span></div><div class="ref-cit-blk half_rhythm" id="CR15">15. <span class="element-citation">Zhang GF, Yan XW. Development of new three-phase culture methods for Manila clam, <em>Ruditapes philippinarum</em>, farming in northern China. <span><span class="ref-journal">Aquaculture. </span>2006;<span class="ref-vol">258</span>:452–261. doi: 10.1016/j.aquaculture.2006.04.046.</span> [<a href="//doi.org/10.1016%2Fj.aquaculture.2006.04.046" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Aquaculture&amp;title=Development+of+new+three-phase+culture+methods+for+Manila+clam,+Ruditapes+philippinarum,+farming+in+northern+China&amp;author=GF+Zhang&amp;author=XW+Yan&amp;volume=258&amp;publication_year=2006&amp;pages=452-261&amp;doi=10.1016/j.aquaculture.2006.04.046&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR16">16. <span class="element-citation">Solidoro C, Pastres R, Melaku Canu D, Pellizzato M, Rossi R. Modelling the growth of <em>Tapes philippinarum</em> in Northern Adriatic Lagoons. <span><span class="ref-journal">Mar. Ecol. Prog. Ser. </span>2000;<span class="ref-vol">199</span>:137–148. doi: 10.3354/meps199137.</span> [<a href="//doi.org/10.3354%2Fmeps199137" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mar.+Ecol.+Prog.+Ser.&amp;title=Modelling+the+growth+of+Tapes+philippinarum+in+Northern+Adriatic+Lagoons&amp;author=C+Solidoro&amp;author=R+Pastres&amp;author=D+Melaku+Canu&amp;author=M+Pellizzato&amp;author=R+Rossi&amp;volume=199&amp;publication_year=2000&amp;pages=137-148&amp;doi=10.3354/meps199137&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR17">17. <span class="element-citation">Eierman LE, Hare MP. Transcriptomic analysis of candidate osmoregulatory genes in the eastern oyster <em>Crassostrea virginica</em>. <span><span class="ref-journal">BMC Genomics. </span>2014;<span class="ref-vol">15</span>:503. doi: 10.1186/1471-2164-15-503.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4101419/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24950855" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2F1471-2164-15-503" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genomics&amp;title=Transcriptomic+analysis+of+candidate+osmoregulatory+genes+in+the+eastern+oyster+Crassostrea+virginica&amp;author=LE+Eierman&amp;author=MP+Hare&amp;volume=15&amp;publication_year=2014&amp;pages=503&amp;pmid=24950855&amp;doi=10.1186/1471-2164-15-503&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR18">18. <span class="element-citation">Eierman LE, Hare MP. Reef-specific patterns of gene expression plasticity in Eastern oysters (<em>Crassostrea virginica</em>) <span><span class="ref-journal">J. Hered. </span>2016;<span class="ref-vol">107</span>(1):90–100. doi: 10.1093/jhered/esv057.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26245921" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fjhered%2Fesv057" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Hered&amp;title=Reef-specific+patterns+of+gene+expression+plasticity+in+Eastern+oysters+(Crassostrea+virginica)&amp;author=LE+Eierman&amp;author=MP+Hare&amp;volume=107&amp;issue=1&amp;publication_year=2016&amp;pages=90-100&amp;pmid=26245921&amp;doi=10.1093/jhered/esv057&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR19">19. <span class="element-citation">Chaney ML, Gracey AY. Mass mortality in Pacific oysters is associated with a specific gene expression signature. <span><span class="ref-journal">Mol. Ecol. </span>2011;<span class="ref-vol">20</span>:2942–2954. doi: 10.1111/j.1365-294X.2011.05152.x.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21672066" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fj.1365-294X.2011.05152.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Ecol&amp;title=Mass+mortality+in+Pacific+oysters+is+associated+with+a+specific+gene+expression+signature&amp;author=ML+Chaney&amp;author=AY+Gracey&amp;volume=20&amp;publication_year=2011&amp;pages=2942-2954&amp;pmid=21672066&amp;doi=10.1111/j.1365-294X.2011.05152.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR20">20. <span class="element-citation">Chapman RW, et al. The transcriptomic responses of the eastern oyster, Crassostrea virginica, to environmental conditions. <span><span class="ref-journal">Mol. Ecol. </span>2011;<span class="ref-vol">20</span>:1431–1449. doi: 10.1111/j.1365-294X.2011.05018.x.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21426432" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fj.1365-294X.2011.05018.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Ecol&amp;title=The+transcriptomic+responses+of+the+eastern+oyster,+Crassostrea+virginica,+to+environmental+conditions&amp;author=RW+Chapman&amp;volume=20&amp;publication_year=2011&amp;pages=1431-1449&amp;pmid=21426432&amp;doi=10.1111/j.1365-294X.2011.05018.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR21">21. <span class="element-citation">Tomanek L, Zuzow MJ, Ivanina AV, Beniash E, Sokolova IM. Proteomic response to elevated PCO2 level in eastern oysters, <em>Crassostrea virginica</em>: evidence for oxidative stress. <span><span class="ref-journal">J. Exp. Biol. </span>2011;<span class="ref-vol">214</span>:1836–1844. doi: 10.1242/jeb.055475.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21562170" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1242%2Fjeb.055475" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Exp.+Biol.&amp;title=Proteomic+response+to+elevated+PCO2+level+in+eastern+oysters,+Crassostrea+virginica:+evidence+for+oxidative+stress&amp;author=L+Tomanek&amp;author=MJ+Zuzow&amp;author=AV+Ivanina&amp;author=E+Beniash&amp;author=IM+Sokolova&amp;volume=214&amp;publication_year=2011&amp;pages=1836-1844&amp;pmid=21562170&amp;doi=10.1242/jeb.055475&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR22">22. <span class="element-citation">Muralidharan S, Thompson E, Raftos D, Birch G, Haynes PA. Quantitative proteomics of heavy metal stress responses in Sydney rock oysters. <span><span class="ref-journal">Proteomics. </span>2012;<span class="ref-vol">12</span>:906–921. doi: 10.1002/pmic.201100417.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22539440" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fpmic.201100417" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Quantitative+proteomics+of+heavy+metal+stress+responses+in+Sydney+rock+oysters&amp;author=S+Muralidharan&amp;author=E+Thompson&amp;author=D+Raftos&amp;author=G+Birch&amp;author=PA+Haynes&amp;volume=12&amp;publication_year=2012&amp;pages=906-921&amp;pmid=22539440&amp;doi=10.1002/pmic.201100417&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR23">23. <span class="element-citation">Dineshram R, et al. Analysis of Pacific oyster larval proteome and its response to high-CO2. <span><span class="ref-journal">Mar. Pollut. Bull. </span>2012;<span class="ref-vol">64</span>:2160–2167. doi: 10.1016/j.marpolbul.2012.07.043.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22921897" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.marpolbul.2012.07.043" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mar.+Pollut.+Bull.&amp;title=Analysis+of+Pacific+oyster+larval+proteome+and+its+response+to+high-CO2&amp;author=R+Dineshram&amp;volume=64&amp;publication_year=2012&amp;pages=2160-2167&amp;pmid=22921897&amp;doi=10.1016/j.marpolbul.2012.07.043&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR24">24. <span class="element-citation">Corporeau C, Tamayo D, Pernet F, Qu&#x000e9;r&#x000e9; C, Madec S. Proteomic signatures of the oyster metabolic response to herpesvirus OsHV-1&#x02009;&#x003bc;Var infection. <span><span class="ref-journal">J. Proteomics. </span>2014;<span class="ref-vol">109</span>:176–187. doi: 10.1016/j.jprot.2014.06.030.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25009143" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.jprot.2014.06.030" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Proteomics&amp;title=Proteomic+signatures+of+the+oyster+metabolic+response+to+herpesvirus+OsHV-1&#x02009;&#x003bc;Var+infection&amp;author=C+Corporeau&amp;author=D+Tamayo&amp;author=F+Pernet&amp;author=C+Qu&#x000e9;r&#x000e9;&amp;author=S+Madec&amp;volume=109&amp;publication_year=2014&amp;pages=176-187&amp;pmid=25009143&amp;doi=10.1016/j.jprot.2014.06.030&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR25">25. <span class="element-citation">Guo X, He Y, Zhang L, Lelong C, Jouaux A. Immune and stress responses in oysters with insights on adaptation. <span><span class="ref-journal">Fish Shellfish Immun. </span>2015;<span class="ref-vol">46</span>:107–119. doi: 10.1016/j.fsi.2015.05.018.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25989624" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.fsi.2015.05.018" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Fish+Shellfish+Immun&amp;title=Immune+and+stress+responses+in+oysters+with+insights+on+adaptation&amp;author=X+Guo&amp;author=Y+He&amp;author=L+Zhang&amp;author=C+Lelong&amp;author=A+Jouaux&amp;volume=46&amp;publication_year=2015&amp;pages=107-119&amp;doi=10.1016/j.fsi.2015.05.018&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR26">26. <span class="element-citation">Li H, Durbin R. Fast and accurate short read alignment with Burrows-Wheeler transform. <span><span class="ref-journal">Bioinformatics. </span>2009;<span class="ref-vol">25</span>(14):1754–1760. doi: 10.1093/bioinformatics/btp324.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2705234/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19451168" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtp324" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Fast+and+accurate+short+read+alignment+with+Burrows-Wheeler+transform&amp;author=H+Li&amp;author=R+Durbin&amp;volume=25&amp;issue=14&amp;publication_year=2009&amp;pages=1754-1760&amp;pmid=19451168&amp;doi=10.1093/bioinformatics/btp324&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR27">27. <span class="element-citation">Milan M, et al. Transcriptome sequencing and microarray development for the Manila clam, <em>Ruditapes philippinarum</em>: genomic tools for environmental monitoring. <span><span class="ref-journal">BMC genomics. </span>2011;<span class="ref-vol">12</span>:234. doi: 10.1186/1471-2164-12-234.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3107815/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21569398" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2F1471-2164-12-234" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+genomics&amp;title=Transcriptome+sequencing+and+microarray+development+for+the+Manila+clam,+Ruditapes+philippinarum:+genomic+tools+for+environmental+monitoring&amp;author=M+Milan&amp;volume=12&amp;publication_year=2011&amp;pages=234&amp;pmid=21569398&amp;doi=10.1186/1471-2164-12-234&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR28">28. <span class="element-citation">Zhang LL, Li L, Zhu YB, Zhang GF, Guo XM. Transcriptome analysis reveals a rich gene set related to innate immunity in the eastern oyster (<em>Crassostrea virginica</em>) <span><span class="ref-journal">Mar Biotechnol. </span>2014;<span class="ref-vol">16</span>:17–33. doi: 10.1007/s10126-013-9526-z.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23907648" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs10126-013-9526-z" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mar+Biotechnol&amp;title=Transcriptome+analysis+reveals+a+rich+gene+set+related+to+innate+immunity+in+the+eastern+oyster+(Crassostrea+virginica)&amp;author=LL+Zhang&amp;author=L+Li&amp;author=YB+Zhu&amp;author=GF+Zhang&amp;author=XM+Guo&amp;volume=16&amp;publication_year=2014&amp;pages=17-33&amp;pmid=23907648&amp;doi=10.1007/s10126-013-9526-z&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR29">29. <span class="element-citation">Zhang GF, et al. The oyster genome reveals stress adaptation and complexity of shell formation. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">490</span>:49–54. doi: 10.1038/nature11413.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22992520" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnature11413" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+oyster+genome+reveals+stress+adaptation+and+complexity+of+shell+formation&amp;author=GF+Zhang&amp;volume=490&amp;publication_year=2012&amp;pages=49-54&amp;pmid=22992520&amp;doi=10.1038/nature11413&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR30">30. <span class="element-citation">Shi MJ, et al. Characterization of the Zhikong scallop (<em>Chlamys farreri</em>) mantle transcriptome and identification of biomineralization-related genes. <span><span class="ref-journal">Mar Biotechnol. </span>2013;<span class="ref-vol">15</span>:706–715. doi: 10.1007/s10126-013-9517-0.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23860577" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs10126-013-9517-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mar+Biotechnol&amp;title=Characterization+of+the+Zhikong+scallop+(Chlamys+farreri)+mantle+transcriptome+and+identification+of+biomineralization-related+genes&amp;author=MJ+Shi&amp;volume=15&amp;publication_year=2013&amp;pages=706-715&amp;pmid=23860577&amp;doi=10.1007/s10126-013-9517-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR31">31. <span class="element-citation">Romano AH, Conway T. Evolution of carbohydrate metabolic pathways. <span><span class="ref-journal">Res. Microbiol. </span>1996;<span class="ref-vol">147</span>:448–455. doi: 10.1016/0923-2508(96)83998-2.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9084754" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2F0923-2508(96)83998-2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Res.+Microbiol.&amp;title=Evolution+of+carbohydrate+metabolic+pathways&amp;author=AH+Romano&amp;author=T+Conway&amp;volume=147&amp;publication_year=1996&amp;pages=448-455&amp;pmid=9084754&amp;doi=10.1016/0923-2508(96)83998-2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR32">32. <span class="mixed-citation">Nelson, D. L. &#x00026; Cox, M. M. Lehninger Principles of Biochemistry. 5th ed.W.H. Freeman and Company, New York. (2008).</span></div><div class="ref-cit-blk half_rhythm" id="CR33">33. <span class="element-citation">Ereciska M, Wilson DF. Regulation of cellular energy metabolism. <span><span class="ref-journal">J. Membr. Biol. </span>1982;<span class="ref-vol">70</span>(1):1–14. doi: 10.1007/BF01871584.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/6226798" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2FBF01871584" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Membr.+Biol.&amp;title=Regulation+of+cellular+energy+metabolism&amp;author=M+Ereciska&amp;author=DF+Wilson&amp;volume=70&amp;issue=1&amp;publication_year=1982&amp;pages=1-14&amp;pmid=6226798&amp;doi=10.1007/BF01871584&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR34">34. <span class="element-citation">Leong HS, Brownsey RW, Kulpab JE, Allard MF. Glycolysis and pyruvate oxidation in cardiac hypertrophy&#x02013;why so unbalanced? <span><span class="ref-journal">Comp. Biochem. Physiol. A. </span>2003;<span class="ref-vol">135</span>(4):499–513. doi: 10.1016/S1095-6433(03)00007-2.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12890541" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FS1095-6433(03)00007-2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Comp.+Biochem.+Physiol.+A&amp;title=Glycolysis+and+pyruvate+oxidation+in+cardiac+hypertrophy&#x02013;why+so+unbalanced?&amp;author=HS+Leong&amp;author=RW+Brownsey&amp;author=JE+Kulpab&amp;author=MF+Allard&amp;volume=135&amp;issue=4&amp;publication_year=2003&amp;pages=499-513&amp;doi=10.1016/S1095-6433(03)00007-2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR35">35. <span class="element-citation">Moreno-S&#x000e1;nchez R, et al. Who controls the ATP supply in cancer cells? Biochemistry lessons to understand cancer energy metabolism. <span><span class="ref-journal">Int. J. Biochem. Cell Biol. </span>2014;<span class="ref-vol">50</span>:10–23. doi: 10.1016/j.biocel.2014.01.025.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24513530" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.biocel.2014.01.025" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Biochem.+Cell+Biol.&amp;title=Who+controls+the+ATP+supply+in+cancer+cells?+Biochemistry+lessons+to+understand+cancer+energy+metabolism&amp;author=R+Moreno-S&#x000e1;nchez&amp;volume=50&amp;publication_year=2014&amp;pages=10-23&amp;pmid=24513530&amp;doi=10.1016/j.biocel.2014.01.025&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR36">36. <span class="element-citation">Zhao XL, Yu H, Kong LF, Liu SK, Li Q. High throughput sequencing of small RNAs transcriptomes in two <em>Crassostrea</em> oysters identifies microRNAs involved in osmotic stress response. <span><span class="ref-journal">Sci. Rep. </span>2016;<span class="ref-vol">6</span>:22687. doi: 10.1038/srep22687.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4778033/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26940974" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fsrep22687" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Rep&amp;title=High+throughput+sequencing+of+small+RNAs+transcriptomes+in+two+Crassostrea+oysters+identifies+microRNAs+involved+in+osmotic+stress+response&amp;author=XL+Zhao&amp;author=H+Yu&amp;author=LF+Kong&amp;author=SK+Liu&amp;author=Q+Li&amp;volume=6&amp;publication_year=2016&amp;pages=22687&amp;pmid=26940974&amp;doi=10.1038/srep22687&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR37">37. <span class="element-citation">Pierce SK, Amende LM. Control Mechanisms of Amino Acid Mediated Cell Volume Regulation in Salinity Stressed Mollusks. <span><span class="ref-journal">J. Exp. Zool. </span>1981;<span class="ref-vol">215</span>:247–257. doi: 10.1002/jez.1402150304.</span> [<a href="//doi.org/10.1002%2Fjez.1402150304" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Exp.+Zool.&amp;title=Control+Mechanisms+of+Amino+Acid+Mediated+Cell+Volume+Regulation+in+Salinity+Stressed+Mollusks&amp;author=SK+Pierce&amp;author=LM+Amende&amp;volume=215&amp;publication_year=1981&amp;pages=247-257&amp;doi=10.1002/jez.1402150304&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR38">38. <span class="element-citation">Hoyaux J, Gilles R, Jeuniaux C. Osmoregulation in molluscs of the intertidal zone. <span><span class="ref-journal">Comp. Biochem. Physiol. A Comp. Physiol. </span>1976;<span class="ref-vol">53</span>:361–365. doi: 10.1016/S0300-9629(76)80157-0.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3321" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FS0300-9629(76)80157-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Comp.+Biochem.+Physiol.+A+Comp.+Physiol&amp;title=Osmoregulation+in+molluscs+of+the+intertidal+zone&amp;author=J+Hoyaux&amp;author=R+Gilles&amp;author=C+Jeuniaux&amp;volume=53&amp;publication_year=1976&amp;pages=361-365&amp;pmid=3321&amp;doi=10.1016/S0300-9629(76)80157-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR39">39. <span class="element-citation">Hosoi M, Kubota S, Toyohara M, Toyohara H, Hayashi I. Effect of salinity change on free amino acid content in Pacific oyster. <span><span class="ref-journal">Fisheries Sci. </span>2003;<span class="ref-vol">69</span>:395–400. doi: 10.1046/j.1444-2906.2003.00634.x.</span> [<a href="//doi.org/10.1046%2Fj.1444-2906.2003.00634.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Fisheries+Sci&amp;title=Effect+of+salinity+change+on+free+amino+acid+content+in+Pacific+oyster&amp;author=M+Hosoi&amp;author=S+Kubota&amp;author=M+Toyohara&amp;author=H+Toyohara&amp;author=I+Hayashi&amp;volume=69&amp;publication_year=2003&amp;pages=395-400&amp;doi=10.1046/j.1444-2906.2003.00634.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR40">40. <span class="element-citation">Lacoste A, Malham SK, G&#x000e9;l&#x000e9;bart F, Cueff A, Poulet SA. Stress-induced immune changes in the oyster <em>Crassostrea gigas</em>. <span><span class="ref-journal">Dev. Comp. Immunol. </span>2002;<span class="ref-vol">26</span>:1–9. doi: 10.1016/S0145-305X(01)00067-2.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11687258" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FS0145-305X(01)00067-2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev.+Comp.+Immunol.&amp;title=Stress-induced+immune+changes+in+the+oyster+Crassostrea+gigas&amp;author=A+Lacoste&amp;author=SK+Malham&amp;author=F+G&#x000e9;l&#x000e9;bart&amp;author=A+Cueff&amp;author=SA+Poulet&amp;volume=26&amp;publication_year=2002&amp;pages=1-9&amp;pmid=11687258&amp;doi=10.1016/S0145-305X(01)00067-2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR41">41. <span class="element-citation">Gagnaire B, Frouin H, Moreau K, Thomas-Guyon H, Renault T. Effects of temperature and salinity on haemocyte activities of the Pacific oyster, <em>Crassostrea gigas</em> (Thunberg) <span><span class="ref-journal">Fish Shellfish Immun. </span>2006;<span class="ref-vol">20</span>:536–547. doi: 10.1016/j.fsi.2005.07.003.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16182565" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.fsi.2005.07.003" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Fish+Shellfish+Immun&amp;title=Effects+of+temperature+and+salinity+on+haemocyte+activities+of+the+Pacific+oyster,+Crassostrea+gigas+(Thunberg)&amp;author=B+Gagnaire&amp;author=H+Frouin&amp;author=K+Moreau&amp;author=H+Thomas-Guyon&amp;author=T+Renault&amp;volume=20&amp;publication_year=2006&amp;pages=536-547&amp;doi=10.1016/j.fsi.2005.07.003&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR42">42. <span class="element-citation">Place SP, Menge BA, Hofmann GE. Transcriptome profiles link environmental variation and physiological response of <em>Mytilus californianus</em> between Pacific tides. <span><span class="ref-journal">Funct. Ecol. </span>2012;<span class="ref-vol">26</span>:144–155. doi: 10.1111/j.1365-2435.2011.01924.x.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3340984/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22563136" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fj.1365-2435.2011.01924.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Funct.+Ecol&amp;title=Transcriptome+profiles+link+environmental+variation+and+physiological+response+of+Mytilus+californianus+between+Pacific+tides&amp;author=SP+Place&amp;author=BA+Menge&amp;author=GE+Hofmann&amp;volume=26&amp;publication_year=2012&amp;pages=144-155&amp;pmid=22563136&amp;doi=10.1111/j.1365-2435.2011.01924.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR43">43. <span class="element-citation">Malagoli D, Casarini L, Sacchi S, Ottaviani E. Stress and immune response in the mussel <em>Mytilus galloprovincialis</em>. <span><span class="ref-journal">Fish Shellfish Immun. </span>2007;<span class="ref-vol">23</span>:171–177. doi: 10.1016/j.fsi.2006.10.004.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17132471" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.fsi.2006.10.004" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Fish+Shellfish+Immun&amp;title=Stress+and+immune+response+in+the+mussel+Mytilus+galloprovincialis&amp;author=D+Malagoli&amp;author=L+Casarini&amp;author=S+Sacchi&amp;author=E+Ottaviani&amp;volume=23&amp;publication_year=2007&amp;pages=171-177&amp;doi=10.1016/j.fsi.2006.10.004&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR44">44. <span class="element-citation">Hoffmann JA, Reichhart JM. Drosophila innate immunity: an evolutionary perspective. <span><span class="ref-journal">Nat. Immunol. </span>2002;<span class="ref-vol">3</span>:121–126. doi: 10.1038/ni0202-121.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11812988" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fni0202-121" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Immunol.&amp;title=Drosophila+innate+immunity:+an+evolutionary+perspective&amp;author=JA+Hoffmann&amp;author=JM+Reichhart&amp;volume=3&amp;publication_year=2002&amp;pages=121-126&amp;pmid=11812988&amp;doi=10.1038/ni0202-121&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR45">45. <span class="element-citation">Ellis RP, et al. Immunological function in marine invertebrates: Responses to environmental perturbation. <span><span class="ref-journal">Fish Shellfish Immun. </span>2011;<span class="ref-vol">30</span>:1209–1222. doi: 10.1016/j.fsi.2011.03.017.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21463691" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.fsi.2011.03.017" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Fish+Shellfish+Immun&amp;title=Immunological+function+in+marine+invertebrates:+Responses+to+environmental+perturbation&amp;author=RP+Ellis&amp;volume=30&amp;publication_year=2011&amp;pages=1209-1222&amp;doi=10.1016/j.fsi.2011.03.017&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR46">46. <span class="element-citation">Sim&#x000e3;o FA, Waterhouse RM, Ioannidis P, Kriventseva EV. BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs. <span><span class="ref-journal">Bioinformatics. </span>2015;<span class="ref-vol">31</span>:3210–3212. doi: 10.1093/bioinformatics/btv351.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26059717" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtv351" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=BUSCO:+assessing+genome+assembly+and+annotation+completeness+with+single-copy+orthologs&amp;author=FA+Sim&#x000e3;o&amp;author=RM+Waterhouse&amp;author=P+Ioannidis&amp;author=EV+Kriventseva&amp;volume=31&amp;publication_year=2015&amp;pages=3210-3212&amp;pmid=26059717&amp;doi=10.1093/bioinformatics/btv351&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR47">47. <span class="element-citation">Robinson MD, McCarthy DJ, Smyth GK. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>(1):139–140. doi: 10.1093/bioinformatics/btp616.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2796818/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19910308" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtp616" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=edgeR:+a+Bioconductor+package+for+differential+expression+analysis+of+digital+gene+expression+data&amp;author=MD+Robinson&amp;author=DJ+McCarthy&amp;author=GK+Smyth&amp;volume=26&amp;issue=1&amp;publication_year=2010&amp;pages=139-140&amp;pmid=19910308&amp;doi=10.1093/bioinformatics/btp616&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR48">48. <span class="element-citation">Wang L, Feng Z, Wang X, Wang X, Zhang X. DEGseq: an R package for identifying differentially expressed genes from RNA-seq data. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:136–138. doi: 10.1093/bioinformatics/btp612.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19855105" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtp612" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=DEGseq:+an+R+package+for+identifying+differentially+expressed+genes+from+RNA-seq+data&amp;author=L+Wang&amp;author=Z+Feng&amp;author=X+Wang&amp;author=X+Wang&amp;author=X+Zhang&amp;volume=26&amp;publication_year=2010&amp;pages=136-138&amp;pmid=19855105&amp;doi=10.1093/bioinformatics/btp612&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR49">49. <span class="element-citation">Schmittgen TD, Livak KJ. Analyzing real-time PCR data by the comparative C(T) method. <span><span class="ref-journal">Nat. Protoc. </span>2008;<span class="ref-vol">3</span>(6):1101–1108. doi: 10.1038/nprot.2008.73.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18546601" ref="reftype=pubmed&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnprot.2008.73" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Protoc.&amp;title=Analyzing+real-time+PCR+data+by+the+comparative+C(T)+method&amp;author=TD+Schmittgen&amp;author=KJ+Livak&amp;volume=3&amp;issue=6&amp;publication_year=2008&amp;pages=1101-1108&amp;pmid=18546601&amp;doi=10.1038/nprot.2008.73&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5504028&amp;issue-id=282893&amp;journal-id=1579&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Scientific Reports</span> are provided here courtesy of <strong>Nature Publishing Group</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC5504028/pdf/41598_2017_Article_5397.pdf" class="int-view">PDF (2.1M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/5504028/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/5504028/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="5504028" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5504028%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="117PEDA5R77HH3V9jfbYwNDjQ7MX7440m17zbnxzkqDL1l91GBVL4gOiyfv41sqe">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5504028%2F&amp;text=High%20throughput%20sequencing%20of%20RNA%20transcriptomes%20in%20Ruditapes%20philippinarum%20identifies%20genes%20involved%20in%20osmotic%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5504028%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5504028/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC5504028/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/28694531/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC5504028/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/28694531/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC5504028/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/5504028/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC5504028/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
