
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Analysis of human MutS homolog 2 missense mutations in patients with colorectal cancer - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="D0BD41E4AC192A4500005C8095E52ECE.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="onclett" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC5920917/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5920917/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Oncology Letters" /><meta name="citation_title" content="Analysis of human MutS homolog 2 missense mutations in patients with colorectal cancer" /><meta name="citation_author" content="Xiaomei Zhang" /><meta name="citation_author_institution" content="Department of Molecular Biology, Jiangsu Cancer Hospital, Jiangsu Institute of Cancer Research, Nanjing Medical University Affiliated Cancer Hospital, Nanjing, Jiangsu 210009, P.R. China" /><meta name="citation_author" content="Senqing Chen" /><meta name="citation_author_institution" content="Department of Molecular Biology, Jiangsu Cancer Hospital, Jiangsu Institute of Cancer Research, Nanjing Medical University Affiliated Cancer Hospital, Nanjing, Jiangsu 210009, P.R. China" /><meta name="citation_author" content="Jun Yu" /><meta name="citation_author_institution" content="Department of Molecular Biology, Jiangsu Cancer Hospital, Jiangsu Institute of Cancer Research, Nanjing Medical University Affiliated Cancer Hospital, Nanjing, Jiangsu 210009, P.R. China" /><meta name="citation_author" content="Yuanying Zhang" /><meta name="citation_author_institution" content="Department of Molecular Biology, Jiangsu Cancer Hospital, Jiangsu Institute of Cancer Research, Nanjing Medical University Affiliated Cancer Hospital, Nanjing, Jiangsu 210009, P.R. China" /><meta name="citation_author" content="Min Lv" /><meta name="citation_author_institution" content="Department of Molecular Biology, Jiangsu Cancer Hospital, Jiangsu Institute of Cancer Research, Nanjing Medical University Affiliated Cancer Hospital, Nanjing, Jiangsu 210009, P.R. China" /><meta name="citation_author" content="Ming Zhu" /><meta name="citation_author_institution" content="Department of Molecular Biology, Jiangsu Cancer Hospital, Jiangsu Institute of Cancer Research, Nanjing Medical University Affiliated Cancer Hospital, Nanjing, Jiangsu 210009, P.R. China" /><meta name="citation_publication_date" content="2018/05" /><meta name="citation_issue" content="5" /><meta name="citation_volume" content="15" /><meta name="citation_firstpage" content="6275" /><meta name="citation_doi" content="10.3892/ol.2018.8161" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5920917/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC5920917/" /><meta name="citation_pmid" content="29731845" /><meta name="DC.Title" content="Analysis of human MutS homolog 2 missense mutations in patients with colorectal cancer" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Spandidos Publications" /><meta name="DC.Contributor" content="Xiaomei Zhang" /><meta name="DC.Contributor" content="Senqing Chen" /><meta name="DC.Contributor" content="Jun Yu" /><meta name="DC.Contributor" content="Yuanying Zhang" /><meta name="DC.Contributor" content="Min Lv" /><meta name="DC.Contributor" content="Ming Zhu" /><meta name="DC.Date" content="2018 May" /><meta name="DC.Identifier" content="10.3892/ol.2018.8161" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Analysis of human MutS homolog 2 missense mutations in patients with colorectal cancer" /><meta property="og:type" content="article" /><meta property="og:description" content="Germline mutations of DNA mismatch repair gene human MutS homolog 2 (hMSH2) are associated with hereditary nonpolyposis colorectal cancer (HNPCC). A total of one-third of these mutations are missense mutations. Several hMSH2 missense mutations have been ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5920917/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC5920917/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/1736/">Oncol Lett</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/309448/">v.15(5); 2018 May</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC5920917
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC5920917/pdf/ol-15-05-6275.pdf" class="int-view">PDF (806K)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/5920917/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/5920917/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="5920917" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5920917%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="6ifP5MnJpQMBGowlxBG6zrVCzQz43nbmLufoJQoX2E6TZzgZfacqF3hpxflVRP1I">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5920917%2F&amp;text=Analysis%20of%20human%20MutS%20homolog%202%20missense%20mutations%20in%20patients%20with%20colorectal%20cancer" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5920917%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5920917/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC5920917/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/29731845/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC5920917/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/29731845/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC5920917/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/5920917/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/1736/">Oncol Lett</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/309448/">v.15(5); 2018 May</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC5920917
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-onclett.gif" alt="Logo of onclett" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.spandidos-publications.com/ol/archive.jsp" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Oncol Lett.</span> 2018 May; 15(5): 6275–6282. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2018 Mar 2. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.3892%2Fol.2018.8161" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.3892/ol.2018.8161</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5920917</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/29731845">29731845</a></div></div></div><h1 class="content-title">Analysis of human MutS homolog 2 missense mutations in patients with colorectal cancer</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Zhang%20X%5BAuthor%5D" class="affpopup" co-rid="_co_idm140456922762992" co-class="co-affbox">Xiaomei Zhang</a>,<sup>1,</sup><sup>*</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Chen%20S%5BAuthor%5D" class="affpopup" co-rid="_co_idm140456978362896" co-class="co-affbox">Senqing Chen</a>,<sup>1,</sup><sup>*</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Yu%20J%5BAuthor%5D" class="affpopup" co-rid="_co_idm140456914098112" co-class="co-affbox">Jun Yu</a>,<sup></sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Zhang%20Y%5BAuthor%5D" class="affpopup" co-rid="_co_idm140456940640784" co-class="co-affbox">Yuanying Zhang</a>,<sup></sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Lv%20M%5BAuthor%5D" class="affpopup" co-rid="_co_idm140456940638928" co-class="co-affbox">Min Lv</a>,<sup></sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Zhu%20M%5BAuthor%5D" class="affpopup" co-rid="_co_idm140456940637072" co-class="co-affbox">Ming Zhu</a><sup></sup><sup></sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140456922762992"><h3 class="no_margin">Xiaomei Zhang</h3><p>Department of Molecular Biology, Jiangsu Cancer Hospital, Jiangsu Institute of Cancer Research, Nanjing Medical University Affiliated Cancer Hospital, Nanjing, Jiangsu 210009, P.R. China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Zhang%20X%5BAuthor%5D">Xiaomei Zhang</a></div></div><div id="_co_idm140456978362896"><h3 class="no_margin">Senqing Chen</h3><p>Department of Molecular Biology, Jiangsu Cancer Hospital, Jiangsu Institute of Cancer Research, Nanjing Medical University Affiliated Cancer Hospital, Nanjing, Jiangsu 210009, P.R. China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Chen%20S%5BAuthor%5D">Senqing Chen</a></div></div><div id="_co_idm140456914098112"><h3 class="no_margin">Jun Yu</h3><p>Department of Molecular Biology, Jiangsu Cancer Hospital, Jiangsu Institute of Cancer Research, Nanjing Medical University Affiliated Cancer Hospital, Nanjing, Jiangsu 210009, P.R. China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Yu%20J%5BAuthor%5D">Jun Yu</a></div></div><div id="_co_idm140456940640784"><h3 class="no_margin">Yuanying Zhang</h3><p>Department of Molecular Biology, Jiangsu Cancer Hospital, Jiangsu Institute of Cancer Research, Nanjing Medical University Affiliated Cancer Hospital, Nanjing, Jiangsu 210009, P.R. China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Zhang%20Y%5BAuthor%5D">Yuanying Zhang</a></div></div><div id="_co_idm140456940638928"><h3 class="no_margin">Min Lv</h3><p>Department of Molecular Biology, Jiangsu Cancer Hospital, Jiangsu Institute of Cancer Research, Nanjing Medical University Affiliated Cancer Hospital, Nanjing, Jiangsu 210009, P.R. China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Lv%20M%5BAuthor%5D">Min Lv</a></div></div><div id="_co_idm140456940637072"><h3 class="no_margin">Ming Zhu</h3><p>Department of Molecular Biology, Jiangsu Cancer Hospital, Jiangsu Institute of Cancer Research, Nanjing Medical University Affiliated Cancer Hospital, Nanjing, Jiangsu 210009, P.R. China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Zhu%20M%5BAuthor%5D">Ming Zhu</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140456941832800_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140456941832800_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140456941832800_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140456941832800_ai" style="display:none"><div class="fm-affl" id="af1-ol-0-0-8161">Department of Molecular Biology, Jiangsu Cancer Hospital, Jiangsu Institute of Cancer Research, Nanjing Medical University Affiliated Cancer Hospital, Nanjing, Jiangsu 210009, P.R. China</div><div id="c1-ol-0-0-8161"><em>Correspondence to</em>: Dr Ming Zhu, Department of Molecular Biology, Jiangsu Cancer Hospital, Jiangsu Institute of Cancer Research, Nanjing Medical University Affiliated Cancer Hospital, 42 Baiziting Road, Nanjing, Jiangsu 210009, P.R. China, E-mail: <a href="mailto:dev@null" data-email="moc.361@1147mz" class="oemail">moc.361@1147mz</a></div><div id="fn1-ol-0-0-8161"><sup>*</sup>Contributed equally</div></div><div class="fm-article-notes hide half_rhythm" id="idm140456941832800_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2017 Jul 28; Accepted 2018 Feb 2.</div></div><div class="permissions half_rhythm hide" id="idm140456941832800_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> : &#x000a9; Zhang et al.</div><div class="license half_rhythm">This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">Creative Commons Attribution-NonCommercial-NoDerivs License</a>, which permits use and distribution in any medium, provided the original work is properly cited, the use is non-commercial and no modifications or adaptations are made.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-length="122" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first-last">The datasets used and analyzed during the current study are available from the corresponding author on reasonable request.</p></dd></dl></div><div id="abstract-a.q.b.q" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.q.b.qtitle">Abstract</h2><!--article-meta--><div><p class="p p-first-last">Germline mutations of DNA mismatch repair gene human MutS homolog 2 (<em>hMSH2</em>) are associated with hereditary nonpolyposis colorectal cancer (HNPCC). A total of one-third of these mutations are missense mutations. Several <em>hMSH2</em> missense mutations have been identified in patients in East Asia, although their function has not been evaluated. In the present study, the role of ten <em>hMSH2</em> missense mutations in the pathogenesis of colorectal cancer was examined. The hMSH2/hMSH6 protein interaction system was established using yeast two-hybrid screening. Next, the missense mutations were analyzed for their ability to affect the protein interaction of hMSH2 with its partner hMSH6. Additionally, the Sorting Intolerant from Tolerant tool was applied to predict the effects of different amino acid substitutions. The results demonstrated that certain <em>hMSH2</em> mutations (L173R and C199R) caused a significant functional change in the human hMutS&#x003b1; complex and were identified to be pathological mutations. The Y408C, D603Y, P696L and S703Y mutations partially affected interaction and partly affected the function of hMSH2. The remaining four variants, T8M, I169V, A370T and Q419K, may be non-functional polymorphisms or could affect protein function through other molecular mechanisms. The present study evaluated the functional consequences of previously unknown missense mutations in <em>hMSH2</em>, and may contribute to improved clinical diagnosis and mutation screening of HNPCC.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">human MutS homolog 2, missense mutation, yeast two-hybrid, hereditary nonpolyposis colorectal cancer</span></div></div><div id="sec-a.r.a" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.r.atitle">Introduction</h2><p class="p p-first">Hereditary nonpolyposis colorectal cancer (HNPCC) is one of the most common inherited cancer susceptibility syndromes, and is characterized by the early development of a number of types of cancer, including colorectal cancer (<a href="#b1-ol-0-0-8161" rid="b1-ol-0-0-8161" class=" bibr popnode">1</a>,<a href="#b2-ol-0-0-8161" rid="b2-ol-0-0-8161" class=" bibr popnode">2</a>). HNPCC is associated with germline mutations in one of the six DNA mismatch repair genes (MMR). These are human MutL homolog 1 (<em>hMLH1</em>), <em>hMSH2</em>, human PMS1 homolog 2 (<em>hPMS2</em>), human MutS homolog 6 (<em>hMSH6</em>), <em>hPMS1</em> and <em>hMLH3</em>. The majority of germline mutations in the HNPCC have been identified in <em>hMSH2</em> and <em>hMLH1</em> (<a href="#b3-ol-0-0-8161" rid="b3-ol-0-0-8161" class=" bibr popnode">3</a>). Genetic screening for HNPCC has primarily focused on these two genes and it was identified that ~half of the germline mutations identified in patients with HNPCC were detected in <em>hMSH2</em>. One-third of the germline mutations are missense mutations (<a href="#b4-ol-0-0-8161" rid="b4-ol-0-0-8161" class=" bibr popnode">4</a>). The etiology of missense mutations is difficult to be evaluated with respect to nonsense mutations, frameshift mutations and large genomic aberrations. Whether a missense mutation is associated with a disease can be examined by carrier family data analysis or a rigorous case-control study based on a molecular epidemiological survey (<a href="#b5-ol-0-0-8161" rid="b5-ol-0-0-8161" class=" bibr popnode">5</a>). However, there may be incomplete family data and a difficulty in obtaining epidemiological samples. Evaluation of missense mutations may lack direct experimental functional analysis, thus posing a challenge for the genetic diagnosis of hereditary colorectal cancer. Therefore, assessment of the pathogenicity of missense mutations is required if using the functional assays.</p><p>The DNA mismatch repair system aids maintenance of the stability of the genome. The human MSH2 protein is comprised of 934 amino acid residues, and the hMSH6 protein is comprised of 1360 amino acid residues. <em>In vivo</em>, hMSH2 and hMSH6 proteins constitute a heterologous heterodimer (hMutS&#x003b1;), which binds to the mismatch site and recognizes single base pair mismatches and small insertion or deletion ring structure (<a href="#b6-ol-0-0-8161" rid="b6-ol-0-0-8161" class=" bibr popnode">6</a>,<a href="#b7-ol-0-0-8161" rid="b7-ol-0-0-8161" class=" bibr popnode">7</a>). hMLH1 dimerizes with hPMS2 to form a hMutL heterodimer. Next, the hMutS&#x003b1; heterodimer and the hMutL heterodimer constitute a polymer, which recruits other proteins required for mismatch repair (<a href="#b8-ol-0-0-8161" rid="b8-ol-0-0-8161" class=" bibr popnode">8</a>). The protein-protein interaction of hMSH2 with hMSH6 is the premise of the hMutS&#x003b1; function. Inhibition of the hMSH2-hMSH6 interaction, which is the potential consequence of a missense mutation, may be a major cause of MMR defects in HNPCC (<a href="#b9-ol-0-0-8161" rid="b9-ol-0-0-8161" class=" bibr popnode">9</a>).</p><p class="p p-last">A previous study established a protein interaction system for <em>hMLH1</em> and <em>hPMS2</em> using the yeast two-hybrid system, which was applied for the functional evaluation of missense mutations in <em>hMLH1</em> genes (<a href="#b10-ol-0-0-8161" rid="b10-ol-0-0-8161" class=" bibr popnode">10</a>). In the present study, ten missense mutations of <em>hMSH2</em> that were frequently detected in patients with HNPCC in East Asia were selected. The <em>hMSH2</em> missense mutations were analyzed for their ability to affect the protein interaction of hMSH2 with its partner hMSH6 <em>in vivo</em>. Additionally, Sorting Intolerant from Tolerant tool (SIFT) tool was applied to predict the effects of amino acid substitution.</p></div><div id="sec-a.r.b" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.r.btitle">Materials and methods</h2><div id="sec-a.r.b.b" class="sec sec-first-last"><div id="sec-a.r.b.b.b" class="sec sec-first"><p></p><h4 id="sec-a.r.b.b.btitle" class="inline">Strains and plasmids </h4><p class="p p-first-last">The wild-type <em>hMSH2</em> and <em>hMSH6</em> cDNA were kindly provided by Professor Josef Jiricny (Institute of Molecular Cancer Research, University of Zurich, Zurich, Switzerland). Restriction enzymes and ligase were purchased from Takara Biotechnology Co., Ltd. (Dalian, China). DNA ladders were purchased from GenScript (Piscataway, NJ, USA). The vectors for the yeast two-hybrid system pGADT7 and pGBKT7, the reporter strain of <em>Saccharomyces cerevisiae</em> AH109 and the <em>Escherichia coli</em> strain Top10 were purchased from Clontech Laboratories, Inc. (Mountainview, CA, USA). <em>E. coli</em> strain Top10 was used for the genetic engineering of all plasmids.</p></div><div id="sec-a.r.b.b.c" class="sec"><p></p><h4 id="sec-a.r.b.b.ctitle" class="inline">Characteristics of the variants </h4><p class="p p-first-last">A total of 10 missense mutations localized to the entire coding region of <em>hMSH2</em> gene were analyzed. A frame-shift mutation, c.1664delA, was used as a positive control (<a href="#b11-ol-0-0-8161" rid="b11-ol-0-0-8161" class=" bibr popnode">11</a>&#x02013;<a href="#b18-ol-0-0-8161" rid="b18-ol-0-0-8161" class=" bibr popnode">18</a>). Mutations were identified in patients with HNPCC from East Asia (<a href="/pmc/articles/PMC5920917/table/tI-ol-0-0-8161/" target="table" class="fig-table-link figpopup" rid-figpopup="tI-ol-0-0-8161" rid-ob="ob-tI-ol-0-0-8161" co-legend-rid=""><span>Table I</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="tI-ol-0-0-8161"><h3>Table I.</h3><!--caption a7--><div class="caption"><p>Clinical and functional characteristics of the 10 <em>hMSH2</em> missense variants in the present study.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="left" valign="bottom" rowspan="1" colspan="1">Exon</th><th align="center" valign="bottom" rowspan="1" colspan="1">hMSH2 variation</th><th align="center" valign="bottom" rowspan="1" colspan="1">Nucleotide substitution</th><th align="center" valign="bottom" rowspan="1" colspan="1">Source (Ref.)</th><th align="center" valign="bottom" rowspan="1" colspan="1">Occurrence in healthy individuals</th><th align="center" valign="bottom" rowspan="1" colspan="1">SIFT score</th></tr></thead><tbody><tr><td align="left" valign="top" rowspan="1" colspan="1">1</td><td align="left" valign="top" rowspan="1" colspan="1">T8M</td><td align="left" valign="top" rowspan="1" colspan="1">c.23C&#x0003e;T</td><td align="left" valign="top" rowspan="1" colspan="1">Chinese, Japanese (<a href="#b11-ol-0-0-8161" rid="b11-ol-0-0-8161" class=" bibr popnode">11</a>,<a href="#b12-ol-0-0-8161" rid="b12-ol-0-0-8161" class=" bibr popnode">12</a>)</td><td align="center" valign="top" rowspan="1" colspan="1">0/100</td><td align="center" valign="top" rowspan="1" colspan="1">0.10</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">3</td><td align="left" valign="top" rowspan="1" colspan="1">I169V</td><td align="left" valign="top" rowspan="1" colspan="1">c.505A&#x0003e;G</td><td align="left" valign="top" rowspan="1" colspan="1">Chinese (<a href="#b13-ol-0-0-8161" rid="b13-ol-0-0-8161" class=" bibr popnode">13</a>)</td><td align="center" valign="top" rowspan="1" colspan="1">0/100</td><td align="center" valign="top" rowspan="1" colspan="1">0.30</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">3</td><td align="left" valign="top" rowspan="1" colspan="1">L173R</td><td align="left" valign="top" rowspan="1" colspan="1">c.518T&#x0003e;G</td><td align="left" valign="top" rowspan="1" colspan="1">Chinese (<a href="#b13-ol-0-0-8161" rid="b13-ol-0-0-8161" class=" bibr popnode">13</a>)</td><td align="center" valign="top" rowspan="1" colspan="1">0/100</td><td align="center" valign="top" rowspan="1" colspan="1">0.00</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">3</td><td align="left" valign="top" rowspan="1" colspan="1">C199R</td><td align="left" valign="top" rowspan="1" colspan="1">c.595T&#x0003e;C</td><td align="left" valign="top" rowspan="1" colspan="1">Chinese (<a href="#b14-ol-0-0-8161" rid="b14-ol-0-0-8161" class=" bibr popnode">14</a>)</td><td align="center" valign="top" rowspan="1" colspan="1">0/100</td><td align="center" valign="top" rowspan="1" colspan="1">0.00</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">7</td><td align="left" valign="top" rowspan="1" colspan="1">A370T</td><td align="left" valign="top" rowspan="1" colspan="1">c.1108G&#x0003e;A</td><td align="left" valign="top" rowspan="1" colspan="1">Chinese (<a href="#b15-ol-0-0-8161" rid="b15-ol-0-0-8161" class=" bibr popnode">15</a>)</td><td align="center" valign="top" rowspan="1" colspan="1">0/100</td><td align="center" valign="top" rowspan="1" colspan="1">1.00</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">7</td><td align="left" valign="top" rowspan="1" colspan="1">Y408C</td><td align="left" valign="top" rowspan="1" colspan="1">c.1223A&#x0003e;G</td><td align="left" valign="top" rowspan="1" colspan="1">Chinese (<a href="#b13-ol-0-0-8161" rid="b13-ol-0-0-8161" class=" bibr popnode">13</a>,<a href="#b16-ol-0-0-8161" rid="b16-ol-0-0-8161" class=" bibr popnode">16</a>)</td><td align="center" valign="top" rowspan="1" colspan="1">0/100</td><td align="center" valign="top" rowspan="1" colspan="1">0.00</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">7</td><td align="left" valign="top" rowspan="1" colspan="1">Q419K</td><td align="left" valign="top" rowspan="1" colspan="1">c.1255C&#x0003e;A</td><td align="left" valign="top" rowspan="1" colspan="1">Chinese (<a href="#b13-ol-0-0-8161" rid="b13-ol-0-0-8161" class=" bibr popnode">13</a>,<a href="#b16-ol-0-0-8161" rid="b16-ol-0-0-8161" class=" bibr popnode">16</a>)</td><td align="center" valign="top" rowspan="1" colspan="1">0/100</td><td align="center" valign="top" rowspan="1" colspan="1">0.72</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">11</td><td align="left" valign="top" rowspan="1" colspan="1">Frameshift</td><td align="left" valign="top" rowspan="1" colspan="1">c.1664delA</td><td align="left" valign="top" rowspan="1" colspan="1">Chinese (<a href="#b11-ol-0-0-8161" rid="b11-ol-0-0-8161" class=" bibr popnode">11</a>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">12</td><td align="left" valign="top" rowspan="1" colspan="1">D603Y</td><td align="left" valign="top" rowspan="1" colspan="1">c.1807G&#x0003e;T</td><td align="left" valign="top" rowspan="1" colspan="1">Chinese (<a href="#b17-ol-0-0-8161" rid="b17-ol-0-0-8161" class=" bibr popnode">17</a>)</td><td align="center" valign="top" rowspan="1" colspan="1">0/100</td><td align="center" valign="top" rowspan="1" colspan="1">0.02</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">13</td><td align="left" valign="top" rowspan="1" colspan="1">P696L</td><td align="left" valign="top" rowspan="1" colspan="1">c.2087C&#x0003e;T</td><td align="left" valign="top" rowspan="1" colspan="1">Chinese (<a href="#b15-ol-0-0-8161" rid="b15-ol-0-0-8161" class=" bibr popnode">15</a>)</td><td align="center" valign="top" rowspan="1" colspan="1">0/100</td><td align="center" valign="top" rowspan="1" colspan="1">0.00</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">13</td><td align="left" valign="top" rowspan="1" colspan="1">S703Y</td><td align="left" valign="top" rowspan="1" colspan="1">c.2108C&#x0003e;A</td><td align="left" valign="top" rowspan="1" colspan="1">Chinese (<a href="#b18-ol-0-0-8161" rid="b18-ol-0-0-8161" class=" bibr popnode">18</a>)</td><td align="center" valign="top" rowspan="1" colspan="1">0/100</td><td align="center" valign="top" rowspan="1" colspan="1">0.01</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140456984127296"><a target="object" rel="noopener" href="/pmc/articles/PMC5920917/table/tI-ol-0-0-8161/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tfn1-ol-0-0-8161"><p class="p p-first-last">hMSH2, human MutS homolog 2; SIFT, Sorting Intolerant from Tolerant.</p></div></div></div></div><div id="sec-a.r.b.b.d" class="sec"><p></p><h4 id="sec-a.r.b.b.dtitle" class="inline">Mutation scanning of normal individuals </h4><p class="p p-first-last">A total of 100 healthy Chinese controls, 55 males and 45 females (age range, 35&#x02013;75 years old), were randomly recruited in The Jiangsu Cancer Hospital (Nanjing, China) from October 2015 to December 2015. Peripheral blood was obtained from normal individuals. Genomic DNA was isolated from 200 &#x000b5;l peripheral blood using the QIAamp DNA Blood Mini kit (Qiagen GmbH, Hilden, Germany), according to the manufacturer's protocol. DNA was subjected to polymerase chain reaction (PCR) amplification of 16 exons of <em>hMSH2</em> gene using specific primers as listed in <a href="/pmc/articles/PMC5920917/table/tII-ol-0-0-8161/" target="table" class="fig-table-link figpopup" rid-figpopup="tII-ol-0-0-8161" rid-ob="ob-tII-ol-0-0-8161" co-legend-rid=""><span>Table II</span></a>, respectively. PCR amplification was performed in a volume of 25 &#x000b5;l that contained 50 ng genomic DNA, 0.5 pmol each primer, 2.5 &#x000b5;l 10X Taq buffer, 2.5 &#x000b5;l 25 mM MgCl<sub>2</sub>, 2 &#x000b5;l 2.5 mM dNTPs and 1 U Taq DNA Polymerase (Fermentas; Thermo Fisher Scientific, Inc., Waltham, MA, USA). A thermal cycler (Applied Biosystems Veriti Thermal Cycler; Thermo Fisher Scientific, Inc.) was used for PCR with the following cycling conditions: An initial denaturation step at 95&#x000b0;C for 2 min followed by 35 cycles of denaturation at 95&#x000b0;C for 0.5 min; annealing at 55&#x000b0;C for 0.5 min; and extension at 72&#x000b0;C for 1 min, followed by a final extension step at 72&#x000b0;C for 5 min. The PCR product was detected on denaturing high-performance liquid chromatography (WAVE System; Transgenomic, Inc., Omaha, NE, USA) to screen the germline mutations of normal individuals of the specific exons. Written informed consent was obtained from all individuals. The study was approved by the Ethics Committee of the Jiangsu Institute of Cancer Research (Nanjing, China).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="tII-ol-0-0-8161"><h3>Table II.</h3><!--caption a7--><div class="caption"><p>List of primer sequences used for polymerase chain reaction amplification of the exons of <em>hMSH2</em> gene.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="left" valign="bottom" rowspan="1" colspan="1">hMSH2 gene</th><th align="center" valign="bottom" rowspan="1" colspan="1">Forward primer sequence (5&#x02032;-3&#x02032;)</th><th align="center" valign="bottom" rowspan="1" colspan="1">Reverse primer sequence (5&#x02032;-3&#x02032;)</th></tr></thead><tbody><tr><td align="left" valign="top" rowspan="1" colspan="1">Exon 1</td><td align="left" valign="top" rowspan="1" colspan="1">GCATTTTCTTCAACCAGGAG</td><td align="left" valign="top" rowspan="1" colspan="1">GTCCCTCCCCAGCACGC</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Exon 2</td><td align="left" valign="top" rowspan="1" colspan="1">GAAGTCCAGCTAATACAGTGC</td><td align="left" valign="top" rowspan="1" colspan="1">CTTCACATTTTTATTTTTCTACTC</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Exon 3</td><td align="left" valign="top" rowspan="1" colspan="1">TTTTAAAGTATGTTCAAGAGTTTG</td><td align="left" valign="top" rowspan="1" colspan="1">ATCTCCTCTATCACTAGACTC</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Exon 4</td><td align="left" valign="top" rowspan="1" colspan="1">TTTCATTTTTGCTTTTCTTATTCC</td><td align="left" valign="top" rowspan="1" colspan="1">TGTAATTCACATTTATAATCCATG</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Exon 5</td><td align="left" valign="top" rowspan="1" colspan="1">CCAGTGGTATAGAAATCTTCG</td><td align="left" valign="top" rowspan="1" colspan="1">CCATTCAACATTTTTAACCC</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Exon 6</td><td align="left" valign="top" rowspan="1" colspan="1">GTTTTCACTAATGAGCTTGCC</td><td align="left" valign="top" rowspan="1" colspan="1">GTGGTATAATCATGTGGG</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Exon 7</td><td align="left" valign="top" rowspan="1" colspan="1">GACTTACGTGCTTAGTTG</td><td align="left" valign="top" rowspan="1" colspan="1">TATGAGGACAGCACATTGCC</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Exon 8</td><td align="left" valign="top" rowspan="1" colspan="1">GATTTGTATTCTGTAAAATGAGATC</td><td align="left" valign="top" rowspan="1" colspan="1">CTACAAACTTTCTTAAAGTGGC</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Exon 9</td><td align="left" valign="top" rowspan="1" colspan="1">TGTCTTTACCCATTATTTATAGG</td><td align="left" valign="top" rowspan="1" colspan="1">CAACCTCCAATGACCCATTC</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Exon 10</td><td align="left" valign="top" rowspan="1" colspan="1">TGGTAGTAGGTATTTATGGAATAC</td><td align="left" valign="top" rowspan="1" colspan="1">CATCATGTTAGAGCATTTAGGG</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Exon 11</td><td align="left" valign="top" rowspan="1" colspan="1">GTACACATTGCTTCTAGTACAC</td><td align="left" valign="top" rowspan="1" colspan="1">AGCCAGGTGACATTCAGAAC</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Exon 12</td><td align="left" valign="top" rowspan="1" colspan="1">ATTCAGTATTCCTGTGTAC</td><td align="left" valign="top" rowspan="1" colspan="1">CGTTACCCCCACAAAGC</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Exon 13</td><td align="left" valign="top" rowspan="1" colspan="1">CGCGATTAATCATCAGTG</td><td align="left" valign="top" rowspan="1" colspan="1">GGACAGAGACATACATTTCTATC</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Exon 14</td><td align="left" valign="top" rowspan="1" colspan="1">GTTACCACATTTTATGTGATGG</td><td align="left" valign="top" rowspan="1" colspan="1">TTCAAGGGTAGTAAGTTTCCC</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Exon 15</td><td align="left" valign="top" rowspan="1" colspan="1">CTCTTCTCATGCTGTCCC</td><td align="left" valign="top" rowspan="1" colspan="1">ATAGAGAAGCTAAGTTAAAC</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Exon 16</td><td align="left" valign="top" rowspan="1" colspan="1">TAATTACTAATGGGACATTC</td><td align="left" valign="top" rowspan="1" colspan="1">TACCTTCATTCCATTACTGG</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140456935374608"><a target="object" rel="noopener" href="/pmc/articles/PMC5920917/table/tII-ol-0-0-8161/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tfn2-ol-0-0-8161"><p class="p p-first-last">hMSH2, human MutS homolog 2.</p></div></div></div></div><div id="sec-a.r.b.b.e" class="sec"><p></p><h4 id="sec-a.r.b.b.etitle" class="inline">In silico analysis of hMSH2 variants </h4><p class="p p-first-last">The SIFT tool was used to predict whether an amino acid substitution is deleterious (<a href="#b19-ol-0-0-8161" rid="b19-ol-0-0-8161" class=" bibr popnode">19</a>,<a href="#b20-ol-0-0-8161" rid="b20-ol-0-0-8161" class=" bibr popnode">20</a>). SIFT prediction is based on a degree of conservation of amino acid residues in sequence alignments derived from closely associated sequences, obtained from PSI-BLAST (<a href="http://sift.jcvi.org/">http://sift.jcvi.org/</a>). SIFT calculates normalized probabilities for all possible substitutions from the alignment. Positions with normalized probabilities &#x0003c;0.05 are predicted to be deleterious, and those &#x02265;0.05 are predicted to be tolerated. In the preset study, 10 missense mutations in the <em>hMSH2</em> gene were analyzed to identify deleterious variants.</p></div><div id="sec-a.r.b.b.f" class="sec"><p></p><h4 id="sec-a.r.b.b.ftitle" class="inline">Plasmid construction </h4><p class="p p-first-last">Wild-type <em>hMSH2</em> cDNA was cloned into pGADT7 using the restriction enzymes <em>Sfi</em>I and <em>Xho</em>I to produce transcriptional AD-fused <em>hMSH2</em> construct pGADT7-hMSH2. Wild-type <em>hMSH6</em> cDNA was ligated into pGBKT7 using the restriction enzymes <em>Eco</em>RI and <em>Sal</em>I, to produce the transcriptional BD-fused <em>hMSH6</em> construct pGBKT7-hMSH6. Using pGBKT7-hMSH6 as the template, various domains of hMSH6 were amplified with primers containing <em>Eco</em>RI and <em>Sal</em>I restriction sites (<a href="/pmc/articles/PMC5920917/table/tIII-ol-0-0-8161/" target="table" class="fig-table-link figpopup" rid-figpopup="tIII-ol-0-0-8161" rid-ob="ob-tIII-ol-0-0-8161" co-legend-rid=""><span>Table III</span></a>). PCR amplification was performed in a volume of 25 &#x000b5;l that contained 50 ng pGBKT7-hMSH6, 0.5 pmol each primer, 2.5 &#x000b5;l 10X Taq buffer, 2.5 &#x000b5;l 25 mM MgCl<sub>2</sub>, 2 &#x000b5;l 2.5 mM dNTPs and 1 U TaKaRa LA Taq Polymerase (Takara Biotechnology Co., Ltd.). The Applied Biosystems Veriti Thermal Cycler was used for PCR with the following cycling conditions: An initial denaturation step at 95&#x000b0;C for 2 min, followed by 35 cycles of denaturation at 95&#x000b0;C for 0.5 min; annealing at 55&#x000b0;C for 0.5 min; and extension at 72&#x000b0;C for 5 min, followed by a final extension step at 72&#x000b0;C for 10 min. The PCR-amplified fragments were then cloned into pGBKT7 using restriction enzymes (<em>Eco</em>RI and <em>Sal</em>I), to produce recombinant pGBKT7 plasmids with various hMSH6 domains. Genetic sequencing was then performed using the dideoxynucleotide chain termination method with an automatic sequencer (ABI 3100; Thermo Fisher Scientific, Inc.) (<a href="#b21-ol-0-0-8161" rid="b21-ol-0-0-8161" class=" bibr popnode">21</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="tIII-ol-0-0-8161"><h3>Table III.</h3><!--caption a7--><div class="caption"><p>List of primer sequences used for polymerase chain reaction amplification of various hMSH6 domains.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="left" valign="bottom" rowspan="1" colspan="1">Domain</th><th align="center" valign="bottom" rowspan="1" colspan="1">Forward primer sequence (5&#x02032;-3&#x02032;)</th><th align="center" valign="bottom" rowspan="1" colspan="1">Reverse primer sequence (5&#x02032;-3&#x02032;)</th></tr></thead><tbody><tr><td align="left" valign="top" rowspan="1" colspan="1">MutS I&#x02013;IV</td><td align="center" valign="top" rowspan="1" colspan="1">CGGAATTCTCTGCCCCTCAAAATTCTG</td><td align="center" valign="top" rowspan="1" colspan="1">GCGTCGACTAAGGACCATCACCCCCTCGAC</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">MutS I&#x02013;V</td><td align="center" valign="top" rowspan="1" colspan="1">CGGAATTCTCTGCCCCTCAAAATTCTG</td><td align="center" valign="top" rowspan="1" colspan="1">GCGTCGACTATAATTCCTTAATCAAAGTCAG</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">MutS II&#x02013;IV</td><td align="center" valign="top" rowspan="1" colspan="1">CGGAATTCACCAAGGGTACACAGACTTAC</td><td align="center" valign="top" rowspan="1" colspan="1">GCGTCGACTAAGGACCATCACCCCCTCGAC</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">MutS II&#x02013;V</td><td align="center" valign="top" rowspan="1" colspan="1">CGGAATTCACCAAGGGTACACAGACTTAC</td><td align="center" valign="top" rowspan="1" colspan="1">GCGTCGACTATAATTCCTTAATCAAAGTCAG</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">MutS III&#x02013;IV</td><td align="center" valign="top" rowspan="1" colspan="1">CGGAATTCAGCACTACAAGATCTGGTGC</td><td align="center" valign="top" rowspan="1" colspan="1">GCGTCGACTAAGGACCATCACCCCCTCGAC</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">MutS III&#x02013;V</td><td align="center" valign="top" rowspan="1" colspan="1">CGGAATTCAGCACTACAAGATCTGGTGC</td><td align="center" valign="top" rowspan="1" colspan="1">GCGTCGACTATAATTCCTTAATCAAAGTCAG</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140456979528416"><a target="object" rel="noopener" href="/pmc/articles/PMC5920917/table/tIII-ol-0-0-8161/?report=objectonly">Open in a separate window</a></div></div></div><div id="sec-a.r.b.b.g" class="sec"><p></p><h4 id="sec-a.r.b.b.gtitle" class="inline">Site-directed mutagenesis and hMSH2 variant construction </h4><p class="p p-first-last">The nucleotide substitutions were introduced into the <em>hMSH2</em> cDNA using overlapping PCR site-directed mutagenesis, as described previously (<a href="#b22-ol-0-0-8161" rid="b22-ol-0-0-8161" class=" bibr popnode">22</a>). Upstream primers Fa, Ra and downstream primers Fb and Rb were synthesized according to the mutation site and PCR reactions were applied. In the first PCR reaction, the upstream fragment was amplified with the upstream primer Fa and the mutant primer Ra, using pGADT7-hMSH2 as template. In the second PCR reaction, the downstream fragment was amplified with the downstream primer Rb and the mutant primer Fb using pGADT7-hMSH2 as template. In the third PCR reaction, the final fragment with the target mutant was amplified with primers Fa and Rb using the previous two overlapping PCR-amplified products as the template. Specific primers were listed in <a href="/pmc/articles/PMC5920917/table/tIV-ol-0-0-8161/" target="table" class="fig-table-link figpopup" rid-figpopup="tIV-ol-0-0-8161" rid-ob="ob-tIV-ol-0-0-8161" co-legend-rid=""><span>Table IV</span></a>. PCR amplification was performed in a volume of 25 &#x000b5;l that contained 50 ng template DNA, 0.5 pmol of each primer, 2.5 &#x000b5;l 10X Taq buffer, 2.5 &#x000b5;l 25 mM MgCl<sub>2</sub>, 2 &#x000b5;l 2.5 mM dNTPs and 1 U Taq DNA Polymerase (Fermentas; Thermo Fisher Scientific, Inc.). The Applied Biosystems Veriti Thermal Cycler was used for PCR with the following cycling conditions: An initial denaturation step at 95&#x000b0;C for 2 min, followed by 35 cycles of denaturation at 95&#x000b0;C for 0.5 min; annealing at 55&#x000b0;C for 0.5 min; and extension at 72&#x000b0;C for 2 min, followed by a final extension step at 72&#x000b0;C for 5 min. The final PCR product was cut with appropriate restriction enzymes (<a href="/pmc/articles/PMC5920917/table/tIV-ol-0-0-8161/" target="table" class="fig-table-link figpopup" rid-figpopup="tIV-ol-0-0-8161" rid-ob="ob-tIV-ol-0-0-8161" co-legend-rid=""><span>Table IV</span></a>) and cloned into pGADT7-hMSH2 to replace the wild-type fragment. The recombinant plasmid pGADT7-hMSH2 with the target mutation was verified by sequencing using an automatic sequencer (ABI 3100; Thermo Fisher Scientific, Inc.).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="tIV-ol-0-0-8161"><h3>Table IV.</h3><!--caption a7--><div class="caption"><p>List of primer sequences used for polymerase chain reaction amplification of <em>hMSH2</em> variants and appropriate restriction enzymes for cloning.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="left" valign="bottom" rowspan="1" colspan="1">Mutation site</th><th align="center" valign="bottom" rowspan="1" colspan="1">Primer sequence (5&#x02032;-3&#x02032;)</th><th align="center" valign="bottom" rowspan="1" colspan="1">Restriction enzymes</th></tr></thead><tbody><tr><td align="left" valign="top" rowspan="1" colspan="1">c.23C&#x0003e;T, T8M</td><td align="left" valign="top" rowspan="1" colspan="1">Fa: CTAACGTTCATGATAACTTCATG Ra: CTCCAACTGCAGCaTCTCCTTCGGC Fb: GCCGAAGGAGAtGCTGCAGTTGGAG Rb: AATTCTGCATCTTCTACAAAAGC</td><td align="left" valign="top" rowspan="1" colspan="1"><em>Ac</em>lI <em>Hind</em>III</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">c.505A&#x0003e;G, I169V</td><td align="left" valign="top" rowspan="1" colspan="1">Fa: CGGGCCATGGAGGCCCCTGGCAATCTCTCTCAGTTTG Ra: GTTTCCTCTGTAcGGAATCCACATAC Fb: GTATGTGGATTCCgTACAGAGGAAAC Rb: AATTCTGCATCTTCTACAAAAGC</td><td align="left" valign="top" rowspan="1" colspan="1"><em>Sfi</em>I <em>Hind</em>III</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">c.518T&#x0003e;G, L173R</td><td align="left" valign="top" rowspan="1" colspan="1">Fa: CGGGCCATGGAGGCCCCTGGCAATCTCTCTCAGTTTG Ra: CACACAGTCCTcGTTTCCTCTGTATG Fb: CATACAGAGGAAACgAGGACTGTGTG Rb: AATTCTGCATCTTCTACAAAAGC</td><td align="left" valign="top" rowspan="1" colspan="1"><em>Sfi</em>I <em>Hind</em>III</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">c.595T&#x0003e;C, C199R</td><td align="left" valign="top" rowspan="1" colspan="1">Fa: CGGGCCATGGAGGCCCCTGGCAATCTCTCTCAGTTTG Ra: CGGGTAAAACACgTTCCTTTGGTCC Fb: GGACCAAAGGAAcGTGTTTTACCCG Rb: AATTCTGCATCTTCTACAAAAGC</td><td align="left" valign="top" rowspan="1" colspan="1"><em>Sfi</em>I <em>Hind</em>III</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">c.1108G&#x0003e;A, A370T</td><td align="left" valign="top" rowspan="1" colspan="1">Fa: CGGGCCATGGAGGCCCCTGGCAATCTCTCTCAGTTTG Ra: GCCTCAATTCTGtATCTTCTACAAAAG Fb: CTTTTGTAGAAGATaCAGAATTGAGGC Rb: GCCTCGAGTCACGTAGTAACTTTTATTCGTG</td><td align="left" valign="top" rowspan="1" colspan="1"><em>Sfi</em>I <em>Xho</em>I</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">c.1223A&#x0003e;G, Y408C</td><td align="left" valign="top" rowspan="1" colspan="1">Fa: GAGAGATTGAATTTAGTGGAAGC Ra: GATTTATACCCTGAcAGAGTCGGTAAC Fb: GTTACCGACTCTgTCAGGGTATAAATC Rb: GCCTCGAGTCACGTAGTAACTTTTATTCGTG</td><td align="left" valign="top" rowspan="1" colspan="1"><em>Hind</em>III <em>Xho</em>I</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">c.1255C&#x0003e;A, Q419K</td><td align="left" valign="top" rowspan="1" colspan="1">Fa: GAGAGATTGAATTTAGTGGAAGC Ra: GTTTTTCCAGAGCCTtTATAACATTAGG Fb: CCTAATGTTATAaAGGCTCTGGAAAAAC Rb: GCCTCGAGTCACGTAGTAACTTTTATTCGTG</td><td align="left" valign="top" rowspan="1" colspan="1"><em>Hind</em>III <em>Xho</em>I</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">c.1664delA, frameshift</td><td align="left" valign="top" rowspan="1" colspan="1">Fa: GAGAGATTGAATTTAGTGGAAGC Ra: CATTTAAAGAAGTCAATTGCTGTTGGTAAATTTAAC Fb: GTTAAATTTACCAACAGCAATTGACTTCTTTAAATG Rb: GCCTCGAGTCACGTAGTAACTTTTATTCGTG</td><td align="left" valign="top" rowspan="1" colspan="1"><em>Hind</em>III <em>Xho</em>I</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">c.1807G&#x0003e;T, D603Y</td><td align="left" valign="top" rowspan="1" colspan="1">Fa: GAGAGATTGAATTTAGTGGAAGC Ra: GACAACAGCATaTAGCTGAGCTAAC Fb: GTTAGCTCAGCTAtATGCTGTTGTC Rb: GCCTCGAGTCACGTAGTAACTTTTATTCGTG</td><td align="left" valign="top" rowspan="1" colspan="1"><em>Hind</em>III <em>Xho</em>I</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">c.2087C&#x0003e;T, P696L</td><td align="left" valign="top" rowspan="1" colspan="1">Fa: GAGAGATTGAATTTAGTGGAAGC Ra: GCTGACTCACATaGCACAAAACACC Fb: GGTGTTTTGTGCtATGTGAGTCAGC Rb: GCCTCGAGTCACGTAGTAACTTTTATTCGTG</td><td align="left" valign="top" rowspan="1" colspan="1"><em>Hind</em>III <em>Xho</em>I</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">c.2108C&#x0003e;A, S703Y</td><td align="left" valign="top" rowspan="1" colspan="1">Fa: GAGAGATTGAATTTAGTGGAAGC Ra: CAGTCCACAATGtACACTTCTGCTG Fb: CAGCAGAAGTGTaCATTGTGGACTG Rb: GCCTCGAGTCACGTAGTAACTTTTATTCGTG</td><td align="left" valign="top" rowspan="1" colspan="1"><em>Hind</em>III <em>Xho</em>I</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140456978856720"><a target="object" rel="noopener" href="/pmc/articles/PMC5920917/table/tIV-ol-0-0-8161/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tfn3-ol-0-0-8161"><p class="p p-first-last">Fa, forward primer sequence a; Fb, forward primer sequence b; Ra, reverse primer sequence a; Rb, reverse primer sequence b.</p></div></div></div></div><div id="sec-a.r.b.b.h" class="sec sec-last"><p></p><h4 id="sec-a.r.b.b.htitle" class="inline">Yeast two-hybrid screening </h4><p class="p p-first-last">The Matchmaker GAL4-based two-hybrid system (Clontech Laboratories, Inc.) was used in the present study. The recombinant plasmids pGBKT7-hMSH6 and pGADT7-hMSH2 (wild type or mutant) were co-transformed into <em>S. cerevisiae</em> strain AH109 and plated onto dropout medium SD/-Trp/-Leu (Clontech Laboratories, Inc.) at 30&#x000b0;C for 2&#x02013;4 days. A total of 10 clones were then selected and inoculated on the dropout medium SD/-Trp/-leu/-His (Clontech Laboratories, Inc.) at 30&#x000b0;C for 2&#x02013;6 days. The growth status of each transformant in the defect plate was then observed. The clone had normal growth when the transformant grew in two days, whereas slow growth was observed following 2&#x02013;4 days of culture. No growth was observed &#x0003e;4 days of culture.</p></div></div></div><div id="sec-a.r.c" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.r.ctitle">Results</h2><div id="sec-a.r.c.b" class="sec sec-first-last"><div id="sec-a.r.c.b.b" class="sec sec-first"><p></p><h4 id="sec-a.r.c.b.btitle" class="inline">Plasmid construction </h4><p class="p p-first-last">Gene engineering was used to construct the recombinant plasmids pGADT7-hMSH2, pGBKT7-hMSH6 from wild-type <em>hMSH2</em> and <em>hMSH6</em> cDNA (<a href="/pmc/articles/PMC5920917/figure/f1-ol-0-0-8161/" target="figure" class="fig-table-link figpopup" rid-figpopup="f1-ol-0-0-8161" rid-ob="ob-f1-ol-0-0-8161" co-legend-rid="lgnd_f1-ol-0-0-8161"><span>Fig. 1</span></a>). Overlapping PCR site-directed mutagenesis was used to construct 11 mutated pGADT7-hMSH2 plasmids. All wild-type and variant plasmids were constructed successfully and verified by sequencing. The hMSH6 protein is comprised of 1,360 amino-acid residues, which can be divided into a N-terminal domain (residues, 1&#x02013;409), and MutS I, II, III, IV and V functional domains (MutS I is the mismatch-binding domain, MutS II is the connector domain, MutS III is the core domain, MutS IV is the clamp domain and MutS V is the ATPase domain) (<a href="#b6-ol-0-0-8161" rid="b6-ol-0-0-8161" class=" bibr popnode">6</a>,<a href="#b7-ol-0-0-8161" rid="b7-ol-0-0-8161" class=" bibr popnode">7</a>). These domains all interact with the corresponding functional domains of hMSH2 protein to complete the mismatch repair function <em>in vivo</em> (<a href="#b8-ol-0-0-8161" rid="b8-ol-0-0-8161" class=" bibr popnode">8</a>) hMSH6 has a large molecular weight which may prevent proper protein folding in yeast. A total of six recombinant pGBKT7 plasmids with various hMSH6 domains were constructed, including pGBKT7-hMSH6-MutS I&#x02013;IV (residues, 341&#x02013;1,073), MutS I&#x02013;V (residues, 341&#x02013;1,360), MutS II&#x02013;IV (residues, 518&#x02013;1,073), MutS II&#x02013;V (residues, 518&#x02013;1,360), MutS III&#x02013;IV (residues, 718&#x02013;1,073) and MutS III&#x02013;V (residues, 718&#x02013;1,360) (<a href="/pmc/articles/PMC5920917/figure/f2-ol-0-0-8161/" target="figure" class="fig-table-link figpopup" rid-figpopup="f2-ol-0-0-8161" rid-ob="ob-f2-ol-0-0-8161" co-legend-rid="lgnd_f2-ol-0-0-8161"><span>Fig. 2</span></a>) (<a href="#b6-ol-0-0-8161" rid="b6-ol-0-0-8161" class=" bibr popnode">6</a>,<a href="#b7-ol-0-0-8161" rid="b7-ol-0-0-8161" class=" bibr popnode">7</a>). All of the pGBKT7 plasmids with different hMSH6 domains were constructed successfully and verified by sequencing.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f1-ol-0-0-8161" co-legend-rid="lgnd_f1-ol-0-0-8161"><a href="/pmc/articles/PMC5920917/figure/f1-ol-0-0-8161/" target="figure" rid-figpopup="f1-ol-0-0-8161" rid-ob="ob-f1-ol-0-0-8161"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140456923237968"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ol-15-05-6275-g00.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is ol-15-05-6275-g00.jpg" src="/pmc/articles/PMC5920917/bin/ol-15-05-6275-g00.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140456923237968"><a target="object" rel="noopener" href="/pmc/articles/PMC5920917/figure/f1-ol-0-0-8161/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_f1-ol-0-0-8161"><div><a class="figpopup" href="/pmc/articles/PMC5920917/figure/f1-ol-0-0-8161/" target="figure" rid-figpopup="f1-ol-0-0-8161" rid-ob="ob-f1-ol-0-0-8161">Figure 1.</a></div><!--caption a7--><div class="caption"><p>Polymerase chain reaction electrophoresis of plasmid pGBKT7-hMSH6. Lanes 1, 2 and 3, MutS I&#x02013;V fragments (3.1 kb); lanes 4 and 5, MutS III&#x02013;V fragments (1.9 kb); lane 6, DNA ladder.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f2-ol-0-0-8161" co-legend-rid="lgnd_f2-ol-0-0-8161"><a href="/pmc/articles/PMC5920917/figure/f2-ol-0-0-8161/" target="figure" rid-figpopup="f2-ol-0-0-8161" rid-ob="ob-f2-ol-0-0-8161"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140456939389600"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ol-15-05-6275-g01.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is ol-15-05-6275-g01.jpg" src="/pmc/articles/PMC5920917/bin/ol-15-05-6275-g01.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140456939389600"><a target="object" rel="noopener" href="/pmc/articles/PMC5920917/figure/f2-ol-0-0-8161/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_f2-ol-0-0-8161"><div><a class="figpopup" href="/pmc/articles/PMC5920917/figure/f2-ol-0-0-8161/" target="figure" rid-figpopup="f2-ol-0-0-8161" rid-ob="ob-f2-ol-0-0-8161">Figure 2.</a></div><!--caption a7--><div class="caption"><p>Scope of six hMSH6 domains. hMSH6, human MutS homolog 6. hMSH6 (residues 1&#x02013;1360), MutS I&#x02013;IV (residues 341&#x02013;1073), MutS I&#x02013;V (residues 341&#x02013;1360), MutS II&#x02013;IV (residues 518&#x02013;1073), MutS II&#x02013;V (residues 518&#x02013;1360), MutS III&#x02013;IV (residues 718&#x02013;1073) and MutS III&#x02013;V (residues 718&#x02013;1360).</p></div></div></div></div><div id="sec-a.r.c.b.c" class="sec"><p></p><h4 id="sec-a.r.c.b.ctitle" class="inline">Mutation identification in 100 normal individuals </h4><p class="p p-first-last">A total of 100 individuals were recruited in The Jiangsu Cancer Hospital. Denaturing high-performance liquid chromatography analysis was used to screen the germline mutations of the normal individuals by PCR of the specific exons. None of the missense mutations examined in the present study were identified in the normal controls (<a href="/pmc/articles/PMC5920917/table/tI-ol-0-0-8161/" target="table" class="fig-table-link figpopup" rid-figpopup="tI-ol-0-0-8161" rid-ob="ob-tI-ol-0-0-8161" co-legend-rid=""><span>Table I</span></a>).</p></div><div id="sec-a.r.c.b.d" class="sec"><p></p><h4 id="sec-a.r.c.b.dtitle" class="inline">Prediction of hMSH2 protein activity and structure </h4><p class="p p-first-last">The SIFT score of <em>hMSH2</em> mutations demonstrated that six of them were considered to be intolerable (L173R, C199R, Y408C, D603Y, P696L and S703Y), indicating that these amino acid substitutions may affect the function of the protein. The remaining four mutations (T8M, I169V, A370T and Q419K) were considered to be tolerable (<a href="/pmc/articles/PMC5920917/table/tI-ol-0-0-8161/" target="table" class="fig-table-link figpopup" rid-figpopup="tI-ol-0-0-8161" rid-ob="ob-tI-ol-0-0-8161" co-legend-rid=""><span>Table I</span></a>).</p></div><div id="sec-a.r.c.b.e" class="sec"><p></p><h4 id="sec-a.r.c.b.etitle" class="inline">Yeast two-hybrid assay of hMSH2 and hMSH6 domains </h4><p class="p p-first-last">Recombinant plasmids pGADT7-hMSH2 (wild type) and pGBKT7 with various hMSH6 domains were co-transformed into <em>S. cerevisiae</em> strain <em>AH109</em> and selected on nutritional defect plates. The growth status of each transformant was observed (<a href="/pmc/articles/PMC5920917/table/tV-ol-0-0-8161/" target="table" class="fig-table-link figpopup" rid-figpopup="tV-ol-0-0-8161" rid-ob="ob-tV-ol-0-0-8161" co-legend-rid=""><span>Table V</span></a>). The negative control was pGADT7/pGBKT7 and the positive control was pGADT7-hMLH1/pGBKT7-hPMS2 (<a href="#b10-ol-0-0-8161" rid="b10-ol-0-0-8161" class=" bibr popnode">10</a>). It was demonstrated that the <em>AH109</em> two-hybrid transformant pGADT7-hMSH2/pGBKT7-hMSH6-MutS II&#x02013;V and pGADT7-hMSH2/pGBKT7-hMSH6-MutS III&#x02013;V grew on the histidine-deficient medium (SD/-Trp/-leu/-His). The results demonstrated that hMSH6-MutS II&#x02013;V and MutS III&#x02013;V may interact with hMSH2 in the yeast strain AH109. Additionally, the two-hybrid transformants pGADT7/pGBKT7-hMSH6-MutS II&#x02013;V and pGADT7/pGBKT7-hMSH6-MutS III&#x02013;V did not grow on the histidine-deficient medium. These results precluded the self-activation of pGADT7/pGBKT7-hMSH6-MutS II&#x02013;V and pGADT7/pGBKT7-hMSH6-MutS III&#x02013;V for His reporter gene in yeast strain <em>AH109</em>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="tV-ol-0-0-8161"><h3>Table V.</h3><!--caption a7--><div class="caption"><p>Yeast two-hybrid assay of hMSH2 and various hMSH6 domains.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="left" valign="bottom" rowspan="1" colspan="1">Transformant</th><th align="center" valign="bottom" rowspan="1" colspan="1">Growth status</th></tr></thead><tbody><tr><td align="left" valign="top" rowspan="1" colspan="1">pGADT7/pGBKT7 (negative control)</td><td align="center" valign="top" rowspan="1" colspan="1">&#x02212;</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">pGADT7-hMSH2/pGBKT7-hMSH6</td><td align="center" valign="top" rowspan="1" colspan="1">&#x02212;</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">pGADT7-hMSH2/pGBKT7-hMSH6-MutS I&#x02013;IV</td><td align="center" valign="top" rowspan="1" colspan="1">&#x02212;</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">pGADT7-hMSH2/pGBKT7-hMSH6-MutS I&#x02013;V</td><td align="center" valign="top" rowspan="1" colspan="1">&#x02212;</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">pGADT7-hMSH2/pGBKT7-hMSH6-MutS II&#x02013;IV</td><td align="center" valign="top" rowspan="1" colspan="1">&#x02212;</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">pGADT7-hMSH2/pGBKT7-hMSH6-MutS II&#x02013;V</td><td align="center" valign="top" rowspan="1" colspan="1">+</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">pGADT7-hMSH2/pGBKT7-hMSH6-MutS III&#x02013;IV</td><td align="center" valign="top" rowspan="1" colspan="1">&#x02212;</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">pGADT7-hMSH2/pGBKT7-hMSH6-MutS III&#x02013;V</td><td align="center" valign="top" rowspan="1" colspan="1">+</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">pGADT7-hMLH1/pGBKT7-hPMS2 (positive control)</td><td align="center" valign="top" rowspan="1" colspan="1">+</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">pGADT7/pGBKT7-hMSH6-MutS II&#x02013;V</td><td align="center" valign="top" rowspan="1" colspan="1">&#x02212;</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">pGADT7/pGBKT7-hMSH6-MutS III&#x02013;V</td><td align="center" valign="top" rowspan="1" colspan="1">&#x02212;</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140456978222320"><a target="object" rel="noopener" href="/pmc/articles/PMC5920917/table/tV-ol-0-0-8161/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tfn4-ol-0-0-8161"><p class="p p-first-last">hMSH2, human MutS homolog 2; +, two-hybrid transformant grew normally in histidine-deficient medium; -, two-hybrid transformant did not grow in the histidine-deficient medium.</p></div></div></div></div><div id="sec-a.r.c.b.f" class="sec sec-last"><p></p><h4 id="sec-a.r.c.b.ftitle" class="inline">Yeast two-hybrid assay of hMSH2 variants and hMSH6-MutS II&#x02013;V </h4><p class="p p-first-last">Mutant pGADT7-hMSH2 and pGBKT7-hMSH6-MutS II&#x02013;V plasmids were co-transformed into <em>S. cerevisiae</em> strain AH109 and selected on a SD/-Leu-Trp plate. The selected clones were then inoculated on a SD/-Trp/-leu/-His plate. The growth status of each transformant was observed (<a href="/pmc/articles/PMC5920917/table/tVI-ol-0-0-8161/" target="table" class="fig-table-link figpopup" rid-figpopup="tVI-ol-0-0-8161" rid-ob="ob-tVI-ol-0-0-8161" co-legend-rid=""><span>Table VI</span></a>). The negative control was pGADT7/pGBKT7 and positive control was pGDAT7-hMSH2/pGBKT7-hMSH6-MutS II&#x02013;V. It was demonstrated that the yeast two-hybrid transformants T8M, I169V, A370T and Q419K grew normally, the transformants Y408C, D603Y, P696L and S703Y grew slowly, whereas the transformants L173R, C199R and c.1664delA did not grow on the histidine-deficient medium in yeast strain <em>AH109</em> compared with wild-type <em>hMSH2</em> (<a href="/pmc/articles/PMC5920917/figure/f3-ol-0-0-8161/" target="figure" class="fig-table-link figpopup" rid-figpopup="f3-ol-0-0-8161" rid-ob="ob-f3-ol-0-0-8161" co-legend-rid="lgnd_f3-ol-0-0-8161"><span>Fig. 3</span></a>). The Y408C, D603Y, P696L and S703Y mutations may partly affect the function of hMSH2 in yeast strain <em>AH109</em>. c.1664delA is a frameshift mutation, thus <em>hMSH2</em> c.1664delA can not bind hMSH6-MutS II&#x02013;V. The yeast two-hybrid transformant pGDAT7-hMSH2-c.1664delA/pGBKT7-hMSH6-MutS II&#x02013;V did not grow in the histidine-deficient medium in yeast strain AH109. The L173R and C199R mutants gave similar results to those of c.1664delA, indicating that the mutations prevented the interaction between hMSH2 and hMSH6.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f3-ol-0-0-8161" co-legend-rid="lgnd_f3-ol-0-0-8161"><a href="/pmc/articles/PMC5920917/figure/f3-ol-0-0-8161/" target="figure" rid-figpopup="f3-ol-0-0-8161" rid-ob="ob-f3-ol-0-0-8161"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140456939387040"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5920917_ol-15-05-6275-g02.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ol-15-05-6275-g02.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5920917/bin/ol-15-05-6275-g02.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140456939387040"><a target="object" rel="noopener" href="/pmc/articles/PMC5920917/figure/f3-ol-0-0-8161/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_f3-ol-0-0-8161"><div><a class="figpopup" href="/pmc/articles/PMC5920917/figure/f3-ol-0-0-8161/" target="figure" rid-figpopup="f3-ol-0-0-8161" rid-ob="ob-f3-ol-0-0-8161">Figure 3.</a></div><!--caption a7--><div class="caption"><p>Yeast two-hybrid system for hMSH2 variants and hMSH6-MutS II&#x02013;V domains. The two-hybrid transformant clones (A) C199R, (B) P696L (C) T8M and (D) wild-type hMSH2 were selected on SD/-Trp/-leu/-His plate, followed by incubation at 30&#x000b0;C for 4 days. hMSH2, human MutS homolog 2; SD, synthetic defined.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="tVI-ol-0-0-8161"><h3>Table VI.</h3><!--caption a7--><div class="caption"><p>Yeast two-hybrid Assay of hMSH2 variants and hMSH6-MutS II&#x02013;V.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="left" valign="bottom" rowspan="1" colspan="1"><em>hMSH2</em> variation</th><th align="center" valign="bottom" rowspan="1" colspan="1">Growth status</th></tr></thead><tbody><tr><td align="left" valign="top" rowspan="1" colspan="1">Wild-type <em>hMSH2</em></td><td align="center" valign="top" rowspan="1" colspan="1">+</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">T8M</td><td align="center" valign="top" rowspan="1" colspan="1">+</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">I169V</td><td align="center" valign="top" rowspan="1" colspan="1">+</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">L173R</td><td align="center" valign="top" rowspan="1" colspan="1">&#x02212;</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">C199R</td><td align="center" valign="top" rowspan="1" colspan="1">&#x02212;</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">A370T</td><td align="center" valign="top" rowspan="1" colspan="1">+</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Y408C</td><td align="center" valign="top" rowspan="1" colspan="1">+/&#x02212;</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Q419K</td><td align="center" valign="top" rowspan="1" colspan="1">+</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">c.1664delA</td><td align="center" valign="top" rowspan="1" colspan="1">&#x02212;</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">D603Y</td><td align="center" valign="top" rowspan="1" colspan="1">+/&#x02212;</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">P696L</td><td align="center" valign="top" rowspan="1" colspan="1">+/&#x02212;</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">S703Y</td><td align="center" valign="top" rowspan="1" colspan="1">+/&#x02212;</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140456941981104"><a target="object" rel="noopener" href="/pmc/articles/PMC5920917/table/tVI-ol-0-0-8161/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tfn5-ol-0-0-8161"><p class="p p-first-last">hMSH2, human MutS homolog 2; +, two-hybrid transformant grew normally in histidine-deficient medium; -, two-hybrid transformant did not grow in the histidine-deficient medium; +/&#x02212;, two-hybrid transformant had slow growth on the histidine-deficient medium.</p></div></div></div></div></div></div><div id="sec-a.r.d" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.r.dtitle">Discussion</h2><p class="p p-first">In the present study, 10 missense mutations to <em>hMSH2</em> were examined in patients with HNPCC from East Asia. The functional consequences of the variants were analyzed based on their genetic, epidemiological and functional characteristics.</p><p><em>In vitro</em> measurement of MMR function has been used as a direct method to evaluate the variants in the <em>hMSH2</em> gene (<a href="#b23-ol-0-0-8161" rid="b23-ol-0-0-8161" class=" bibr popnode">23</a>). It was difficult to perform this complex experiment since it required expression and purification of proteins. The yeast two-hybrid system has been used as a method to study the interaction between proteins (<a href="#b24-ol-0-0-8161" rid="b24-ol-0-0-8161" class=" bibr popnode">24</a>). A previous study established a protein interaction system for hMLH1 and hPMS2 using the yeast two-hybrid system, which was applied for the functional evaluation of missense mutations in <em>hMLH1</em> genes (<a href="#b10-ol-0-0-8161" rid="b10-ol-0-0-8161" class=" bibr popnode">10</a>). The yeast two-hybrid system has advantages for detecting protein binding: Proteins may maintain a natural conformation to mimic the physiological state <em>in vivo</em>, and this system eliminates the time-consuming steps necessary for protein purification <em>in vitro</em>. A previous study used a similar system to evaluate the partial <em>hMSH2</em> missense mutations I169V, L173R, Y408C, Q419K and S703Y (<a href="#b25-ol-0-0-8161" rid="b25-ol-0-0-8161" class=" bibr popnode">25</a>). In the present study, the yeast two-hybrid experimental method has been improved and the results have been evaluated using SIFT to evaluate the enlarged <em>hMSH2</em> missense mutations. The yeast two-hybrid system was used to analyze the ability of <em>hMSH2</em> missense mutations to affect the interaction of hMSH2 with its partner hMSH6 <em>in vivo</em>. The growth status of each transformant provided an indication of the protein interaction strength, as the enzyme activity of reporter genes was associated with the degree of protein binding.</p><p>Genetic engineering was used to construct recombinant plasmids pGADT7-hMSH2 and pGBKT7-hMSH6. Next, the recombinant plasmids pGADT7-hMSH2 and pGBKT7-hMSH6 were co-transformed into <em>S. cerevisiae</em> strain AH109. The results demonstrated that the yeast two-hybrid transformant couldn't grow in histidine-deficient medium, indicating that the interaction between hMSH2 and hMSH6 may be inhibited in yeast <em>AH109</em>. The possible reason behind this may be the inappropriate folding of hMSH6 protein when expressed in yeast strain AH109, owing to its size. Additionally, six recombinant pGBKT7 plasmids with various hMSH6 domains were constructed. The results demonstrated that the transformant pGADT7-hMSH2/pGBKT7-hMSH6-MutS II&#x02013;V and pGADT7-hMSH2/pGBKT7-hMSH6-MutS III&#x02013;V were able to grow on the histidine-deficient medium (<a href="/pmc/articles/PMC5920917/table/tV-ol-0-0-8161/" target="table" class="fig-table-link figpopup" rid-figpopup="tV-ol-0-0-8161" rid-ob="ob-tV-ol-0-0-8161" co-legend-rid=""><span>Table V</span></a>). These results indicate that hMSH6-MutS II&#x02013;V and MutS III&#x02013;V may interact with hMSH2 in yeast strain AH109. The hMSH6 MutS II&#x02013;V domain contains amino acid residues 518&#x02013;1,360 of the hMSH6 protein, which comprises the core regions of the full protein. The yeast two-hybrid transformant pGADT7-hMSH2/pGBKT7-hMSH6-MutS II&#x02013;V was used to construct the interaction system of hMSH2-hMSH6 and to evaluate <em>hMSH2</em> missense variants (<a href="/pmc/articles/PMC5920917/table/tVI-ol-0-0-8161/" target="table" class="fig-table-link figpopup" rid-figpopup="tVI-ol-0-0-8161" rid-ob="ob-tVI-ol-0-0-8161" co-legend-rid=""><span>Table VI</span></a>).</p><p>SIFT scores revealed that T8M, I169V, A370T and Q419K were tolerant substitutions. The results of the present study indicated that these variants may be nonfunctional polymorphisms or may affect the protein function through other molecular mechanisms.</p><p>SIFT scores revealed that Y408C, D603Y, P696L and S703Y were intolerable substitutions. In the Y408C mutant, tyrosine and cysteine belong to neutral amino acids, but their stereochemistry differs. For the D603Y mutant, aspartic acid is an acidic amino acid, whereas tyrosine is an aromatic amino acid, meaning their reactivities may differ. P696L was associated with a loss of hMSH2 protein expression and high level of microsatellite instability in tumors (<a href="#b15-ol-0-0-8161" rid="b15-ol-0-0-8161" class=" bibr popnode">15</a>). For S703Y, homology modeling revealed that steric hindrance was increased because of the serine-to-tyrosine mutation at residue 703. Finally, the hydrogen bonds of Ala700 and Glu706 were not formed, affecting the whole molecular structure and leading to inactivation of hMSH2 function (<a href="#b18-ol-0-0-8161" rid="b18-ol-0-0-8161" class=" bibr popnode">18</a>). The yeast two-hybrid transformant of the four variants grew slowly on the histidine-deficient medium. Therefore, these variants may affect the partial interaction and partly affect the function of hMSH2.</p><p>L173R and C199R are highly conserved from yeast to humans (<a href="#b6-ol-0-0-8161" rid="b6-ol-0-0-8161" class=" bibr popnode">6</a>). SIFT scores revealed that they were intolerable substitutions. For the L173R mutation, leucine is an uncharged hydrophobic amino acid, whereas arginine is a positively charged, weak-basic amino acid. Their functions may differ as a result of differences in solubility and ionization properties. For the C199R mutation, cysteine is an uncharged hydrophilic amino acid, whereas arginine is a positively charged, weak-basic amino acid. Their functions may also differ as a result of differences in stereochemistry and ionization properties. The yeast two-hybrid transformant of the two variants, including c.1664delA, did not grow on histidine-deficient medium. c.1664delA is a frameshift mutation, thus hMSH2 c.1664delA is not able to bind hMSH6-MutS II&#x02013;V. L173R and C199R may have a similar effect as c.1664delA. Therefore, it was considered that they these mutations may be pathological.</p><p>In the present study, segregation studies were not conducted because of the unavailability of family samples. Therefore, the synergistic effect of low-risk alleles of MMR genes, which was previously described (<a href="#b26-ol-0-0-8161" rid="b26-ol-0-0-8161" class=" bibr popnode">26</a>), was not evaluated in the present study.</p><p class="p p-last">In conclusion, the present study established the preliminary hMSH2/hMSH6 protein-interaction system using a yeast two-hybrid system. Combined with SIFT analysis and amino acid analysis, an evaluation system for missense mutations in <em>hMSH2</em> was established. Using this system, 10 missense mutations in <em>hMSH2</em> were examined. The results demonstrated that the <em>hMSH2</em> mutations L173R and C199R caused a functional change in the hMutS&#x003b1; complex and were identified as pathological mutations. The Y408C, D603Y, P696L and S703Y variants partially affected interaction, and may partly affect the function of hMSH2. The remaining four variants, T8M, I169V, A370T and Q419K, may be non-functional polymorphisms or may affect the protein function through other molecular mechanisms. The present study evaluated the functional consequences of several previously unknown missense mutations in the <em>hMSH2</em> gene, which could contribute to clinical diagnosis and mutation screening of HNPCC.</p></div><div id="ack-a.s.a" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.s.atitle">Acknowledgements</h2><div class="sec"><p>The authors thank Professor Josef Jiricny (Institute of Molecular Cancer Research, University of Zurich, Switzerland) for providing the hMSH2 and hMSH6 cDNAs.</p></div></div><div id="sec-a.s.b" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="sec-a.s.btitle">Funding</h2><!--/article/back/sec/--><p class="p p-first-last">The present study was supported by the Project of Jiangsu Provincial Commission of Health and Family Planning (grant nos. H201510 and H2017035), Jiangsu Provincial Medical Youth Talent (grant no. QNRC2016652), the Natural Science Foundation of Jiangsu Province (grant nos. BK20141490 and BK20161598) and the Research Project of Jiangsu Cancer Hospital (grant no. ZM201107).</p></div><div id="sec-a.s.c" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="sec-a.s.ctitle">Availability of data and materials</h2><!--/article/back/sec/--><p class="p p-first-last">The datasets used and analyzed during the current study are available from the corresponding author on reasonable request.</p></div><div id="sec-a.s.d" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="sec-a.s.dtitle">Authors' contributions</h2><!--/article/back/sec/--><p class="p p-first-last">MZ designed the study, performed the research and wrote the manuscript. XZ and SC constructed all the plasmids. JY analyzed the data. YZ performed the yeast two-hybrid assay. ML collected the blood samples and performed mutation scanning of normal individuals. All authors read and approved the final manuscript.</p></div><div id="sec-a.s.e" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="sec-a.s.etitle">Ethics approval and consent to participate</h2><!--/article/back/sec/--><p class="p p-first-last">This research was approved by the Ethics Committee of the Jiangsu Institute of Cancer Research (Nanjing, China) and written informed consent was obtained from all individuals.</p></div><div id="sec-a.s.f" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="sec-a.s.ftitle">Consent for publication</h2><!--/article/back/sec/--><p class="p p-first-last">Informed consent for the publication of any associated data and accompanying images was obtained from all the normal individuals.</p></div><div id="sec-a.s.g" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="sec-a.s.gtitle">Competing interests</h2><!--/article/back/sec/--><p class="p p-first-last">The authors declare that they have no competing interests.</p></div><div id="ref-list-a.s.h" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.s.htitle">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="b1-ol-0-0-8161">1. <span class="element-citation">Brosens LA, Offerhaus GJ, Giardiello FM. Hereditary colorectal cancer: Genetics and screening. <span><span class="ref-journal">Surg Clin North Am. </span>2015;<span class="ref-vol">95</span>:1067–1080. doi: 10.1016/j.suc.2015.05.004.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4555838/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26315524" ref="reftype=pubmed&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.suc.2015.05.004" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Surg+Clin+North+Am&amp;title=Hereditary+colorectal+cancer:+Genetics+and+screening&amp;author=LA+Brosens&amp;author=GJ+Offerhaus&amp;author=FM+Giardiello&amp;volume=95&amp;publication_year=2015&amp;pages=1067-1080&amp;pmid=26315524&amp;doi=10.1016/j.suc.2015.05.004&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b2-ol-0-0-8161">2. <span class="element-citation">Lynch HT, de la Chapelle A. Genetic susceptibility to non-polyposis colorectal cancer. <span><span class="ref-journal">J Med Genet. </span>1999;<span class="ref-vol">36</span>:801–818.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1734258/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10544223" ref="reftype=pubmed&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Med+Genet&amp;title=Genetic+susceptibility+to+non-polyposis+colorectal+cancer&amp;author=HT+Lynch&amp;author=A+de+la+Chapelle&amp;volume=36&amp;publication_year=1999&amp;pages=801-818&amp;pmid=10544223&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b3-ol-0-0-8161">3. <span class="element-citation"><a href="https://www.insight-group.org/variants/databases/">https://www.insight-group.org/variants/databases/</a>  [Jun 29;2017 ];<span><span class="ref-journal">In SiGHT: InSiGHT variant databases. </span></span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=In+SiGHT:+InSiGHT+variant+databases&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b4-ol-0-0-8161">4. <span class="element-citation">Woods MO, Williams P, Careen A, Edwards L, Bartlett S, McLaughlin JR, Younghusband HB. A new variant database for mismatch repair genes associated with Lynch syndrome. <span><span class="ref-journal">Hum Mutat. </span>2007;<span class="ref-vol">28</span>:669–673. doi: 10.1002/humu.20502.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17347989" ref="reftype=pubmed&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fhumu.20502" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mutat&amp;title=A+new+variant+database+for+mismatch+repair+genes+associated+with+Lynch+syndrome&amp;author=MO+Woods&amp;author=P+Williams&amp;author=A+Careen&amp;author=L+Edwards&amp;author=S+Bartlett&amp;volume=28&amp;publication_year=2007&amp;pages=669-673&amp;pmid=17347989&amp;doi=10.1002/humu.20502&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b5-ol-0-0-8161">5. <span class="element-citation">Peltomaki P, Vasen HF. Mutations predisposing to hereditary nonpolyposis colorectal cancer: Database and results of a collaborative study. The International Collaborative Group on Hereditary Nonpolyposis Colorectal Cancer. <span><span class="ref-journal">Gastroenterology. </span>1997;<span class="ref-vol">113</span>:1146–1158. doi: 10.1053/gast.1997.v113.pm9322509.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9322509" ref="reftype=pubmed&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1053%2Fgast.1997.v113.pm9322509" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Mutations+predisposing+to+hereditary+nonpolyposis+colorectal+cancer:+Database+and+results+of+a+collaborative+study.+The+International+Collaborative+Group+on+Hereditary+Nonpolyposis+Colorectal+Cancer&amp;author=P+Peltomaki&amp;author=HF+Vasen&amp;volume=113&amp;publication_year=1997&amp;pages=1146-1158&amp;pmid=9322509&amp;doi=10.1053/gast.1997.v113.pm9322509&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b6-ol-0-0-8161">6. <span class="element-citation">Obmolova G, Ban C, Hsieh P, Yang W. Crystal structures of mismatch repair protein MutS and its complex with a substrate DNA. <span><span class="ref-journal">Nature. </span>2000;<span class="ref-vol">407</span>:703–710. doi: 10.1038/35037509.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11048710" ref="reftype=pubmed&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2F35037509" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Crystal+structures+of+mismatch+repair+protein+MutS+and+its+complex+with+a+substrate+DNA&amp;author=G+Obmolova&amp;author=C+Ban&amp;author=P+Hsieh&amp;author=W+Yang&amp;volume=407&amp;publication_year=2000&amp;pages=703-710&amp;pmid=11048710&amp;doi=10.1038/35037509&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b7-ol-0-0-8161">7. <span class="element-citation">Lamers MH, Perrakis A, Enzlin JH, Winterwerp HH, de Wind N, Sixma TK. The crystal structure of DNA mismatch repair protein MutS binding to a GxT mismatch. <span><span class="ref-journal">Nature. </span>2000;<span class="ref-vol">407</span>:711–717. doi: 10.1038/35037523.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11048711" ref="reftype=pubmed&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2F35037523" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+crystal+structure+of+DNA+mismatch+repair+protein+MutS+binding+to+a+GxT+mismatch&amp;author=MH+Lamers&amp;author=A+Perrakis&amp;author=JH+Enzlin&amp;author=HH+Winterwerp&amp;author=N+de+Wind&amp;volume=407&amp;publication_year=2000&amp;pages=711-717&amp;pmid=11048711&amp;doi=10.1038/35037523&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b8-ol-0-0-8161">8. <span class="element-citation">Jiricny J. The multifaceted mismatch-repair system. <span><span class="ref-journal">Nat Rev Mol Cell Biol. </span>2006;<span class="ref-vol">7</span>:335–346. doi: 10.1038/nrm1907.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16612326" ref="reftype=pubmed&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnrm1907" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Mol+Cell+Biol&amp;title=The+multifaceted+mismatch-repair+system&amp;author=J+Jiricny&amp;volume=7&amp;publication_year=2006&amp;pages=335-346&amp;pmid=16612326&amp;doi=10.1038/nrm1907&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b9-ol-0-0-8161">9. <span class="element-citation">Geng H, Sakato M, DeRocco V, Yamane K, Du C, Erie DA, Hingorani M, Hsieh P. Biochemical analysis of the human mismatch repair proteins hMutS&#x003b1; MSH2(G674A)-MSH6 and MSH2-MSH6(T1219D) <span><span class="ref-journal">J Biol Chem. </span>2012;<span class="ref-vol">287</span>:9777–9791. doi: 10.1074/jbc.M111.316919.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3323011/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22277660" ref="reftype=pubmed&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1074%2Fjbc.M111.316919" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Biochemical+analysis+of+the+human+mismatch+repair+proteins+hMutS&#x003b1;+MSH2(G674A)-MSH6+and+MSH2-MSH6(T1219D)&amp;author=H+Geng&amp;author=M+Sakato&amp;author=V+DeRocco&amp;author=K+Yamane&amp;author=C+Du&amp;volume=287&amp;publication_year=2012&amp;pages=9777-9791&amp;pmid=22277660&amp;doi=10.1074/jbc.M111.316919&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b10-ol-0-0-8161">10. <span class="element-citation">Fan Y, Wang W, Zhu M, Zhou J, Peng J, Xu L, Hua Z, Gao X, Wang Y. Analysis of hMLH1 missense mutations in East Asian patients with suspected hereditary nonpolyposis colorectal cancer. <span><span class="ref-journal">Clin Cancer Res. </span>2007;<span class="ref-vol">13</span>:7515–7521. doi: 10.1158/1078-0432.CCR-07-1028.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18094436" ref="reftype=pubmed&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1158%2F1078-0432.CCR-07-1028" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Cancer+Res&amp;title=Analysis+of+hMLH1+missense+mutations+in+East+Asian+patients+with+suspected+hereditary+nonpolyposis+colorectal+cancer&amp;author=Y+Fan&amp;author=W+Wang&amp;author=M+Zhu&amp;author=J+Zhou&amp;author=J+Peng&amp;volume=13&amp;publication_year=2007&amp;pages=7515-7521&amp;pmid=18094436&amp;doi=10.1158/1078-0432.CCR-07-1028&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b11-ol-0-0-8161">11. <span class="element-citation">Wang XL, Yuan Y, Zhang SZ, Cai SR, Huang YQ, Jiang Q, Zheng S. Clinical and genetic characteristics of Chinese hereditary nonpolyposis colorectal cancer families. <span><span class="ref-journal">World J Gastroenterol. </span>2006;<span class="ref-vol">12</span>:4074–4077. doi: 10.3748/wjg.v12.i25.4074.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4087725/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16810763" ref="reftype=pubmed&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3748%2Fwjg.v12.i25.4074" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=World+J+Gastroenterol&amp;title=Clinical+and+genetic+characteristics+of+Chinese+hereditary+nonpolyposis+colorectal+cancer+families&amp;author=XL+Wang&amp;author=Y+Yuan&amp;author=SZ+Zhang&amp;author=SR+Cai&amp;author=YQ+Huang&amp;volume=12&amp;publication_year=2006&amp;pages=4074-4077&amp;pmid=16810763&amp;doi=10.3748/wjg.v12.i25.4074&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b12-ol-0-0-8161">12. <span class="element-citation">Nomura S, Sugano K, Kashiwabara H, Taniguchi T, Fukayama N, Fujita S, Akasu T, Moriya Y, Ohhigashi S, Kakizoe T, Sekiya T. Enhanced detection of deleterious and other germline mutations of hMSH2 and hMLH1 in Japanese hereditary nonpolyposis colorectal cancer kindreds. <span><span class="ref-journal">Biochem Biophys Res Commun. </span>2000;<span class="ref-vol">271</span>:120–129. doi: 10.1006/bbrc.2000.2547.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10777691" ref="reftype=pubmed&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1006%2Fbbrc.2000.2547" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+Biophys+Res+Commun&amp;title=Enhanced+detection+of+deleterious+and+other+germline+mutations+of+hMSH2+and+hMLH1+in+Japanese+hereditary+nonpolyposis+colorectal+cancer+kindreds&amp;author=S+Nomura&amp;author=K+Sugano&amp;author=H+Kashiwabara&amp;author=T+Taniguchi&amp;author=N+Fukayama&amp;volume=271&amp;publication_year=2000&amp;pages=120-129&amp;pmid=10777691&amp;doi=10.1006/bbrc.2000.2547&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b13-ol-0-0-8161">13. <span class="element-citation">Fan Y, Liu X, Zhang H, Dai J, Zhang X, Zhu M, Gao X, Wang Y. Variations in exon 7 of the MSH2 gene and susceptibility to gastrointestinal cancer in a Chinese population. <span><span class="ref-journal">Cancer Genet Cytogenet. </span>2006;<span class="ref-vol">170</span>:121–128. doi: 10.1016/j.cancergencyto.2006.05.010.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17011982" ref="reftype=pubmed&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cancergencyto.2006.05.010" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Genet+Cytogenet&amp;title=Variations+in+exon+7+of+the+MSH2+gene+and+susceptibility+to+gastrointestinal+cancer+in+a+Chinese+population&amp;author=Y+Fan&amp;author=X+Liu&amp;author=H+Zhang&amp;author=J+Dai&amp;author=X+Zhang&amp;volume=170&amp;publication_year=2006&amp;pages=121-128&amp;pmid=17011982&amp;doi=10.1016/j.cancergencyto.2006.05.010&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b14-ol-0-0-8161">14. <span class="element-citation">Leung SY, Chan TL, Chung LP, Chan AS, Fan YW, Hung KN, Kwong WK, Ho JW, Yuen ST. Microsatellite instability and mutation of DNA mismatch repair genes in gliomas. <span><span class="ref-journal">Am J Pathol. </span>1998;<span class="ref-vol">153</span>:1181–1188. doi: 10.1016/S0002-9440(10)65662-3.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1853047/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9777949" ref="reftype=pubmed&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FS0002-9440(10)65662-3" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Pathol&amp;title=Microsatellite+instability+and+mutation+of+DNA+mismatch+repair+genes+in+gliomas&amp;author=SY+Leung&amp;author=TL+Chan&amp;author=LP+Chung&amp;author=AS+Chan&amp;author=YW+Fan&amp;volume=153&amp;publication_year=1998&amp;pages=1181-1188&amp;pmid=9777949&amp;doi=10.1016/S0002-9440(10)65662-3&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b15-ol-0-0-8161">15. <span class="element-citation">Tang R, Hsiung C, Wang JY, Lai CH, Chien HT, Chiu LL, Liu CT, Chen HH, Wang HM, Chen SX, et al. Germ line MLH1 and MSH2 mutations in Taiwanese Lynch syndrome families: Characterization of a founder genomic mutation in the MLH1 gene. <span><span class="ref-journal">Clin Genet. </span>2009;<span class="ref-vol">75</span>:334–345. doi: 10.1111/j.1399-0004.2009.01162.x.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19419416" ref="reftype=pubmed&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fj.1399-0004.2009.01162.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Genet&amp;title=Germ+line+MLH1+and+MSH2+mutations+in+Taiwanese+Lynch+syndrome+families:+Characterization+of+a+founder+genomic+mutation+in+the+MLH1+gene&amp;author=R+Tang&amp;author=C+Hsiung&amp;author=JY+Wang&amp;author=CH+Lai&amp;author=HT+Chien&amp;volume=75&amp;publication_year=2009&amp;pages=334-345&amp;pmid=19419416&amp;doi=10.1111/j.1399-0004.2009.01162.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b16-ol-0-0-8161">16. <span class="element-citation">Zhang Y, Liu X, Fan Y, Ding J, Xu A, Zhou X, Hu X, Zhu M, Zhang X, Li S, et al. Germline mutations and polymorphic variants in MMR, E-cadherin and MYH genes associated with familial gastric cancer in Jiangsu of China. <span><span class="ref-journal">Int J Cancer. </span>2006;<span class="ref-vol">119</span>:2592–2596. doi: 10.1002/ijc.22206.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16929514" ref="reftype=pubmed&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fijc.22206" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Cancer&amp;title=Germline+mutations+and+polymorphic+variants+in+MMR,+E-cadherin+and+MYH+genes+associated+with+familial+gastric+cancer+in+Jiangsu+of+China&amp;author=Y+Zhang&amp;author=X+Liu&amp;author=Y+Fan&amp;author=J+Ding&amp;author=A+Xu&amp;volume=119&amp;publication_year=2006&amp;pages=2592-2596&amp;pmid=16929514&amp;doi=10.1002/ijc.22206&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b17-ol-0-0-8161">17. <span class="element-citation">Yuen ST, Chan TL, Ho JW, Chan AS, Chung LP, Lam PW, Tse CW, Wyllie AH, Leung SY. Germline, somatic and epigenetic events underlying mismatch repair deficiency in colorectal and HNPCC-related cancers. <span><span class="ref-journal">Oncogene. </span>2002;<span class="ref-vol">21</span>:7585–7592. doi: 10.1038/sj.onc.1205968.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12386821" ref="reftype=pubmed&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fsj.onc.1205968" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Germline,+somatic+and+epigenetic+events+underlying+mismatch+repair+deficiency+in+colorectal+and+HNPCC-related+cancers&amp;author=ST+Yuen&amp;author=TL+Chan&amp;author=JW+Ho&amp;author=AS+Chan&amp;author=LP+Chung&amp;volume=21&amp;publication_year=2002&amp;pages=7585-7592&amp;pmid=12386821&amp;doi=10.1038/sj.onc.1205968&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b18-ol-0-0-8161">18. <span class="element-citation">Jin HY, Yan HL, Song LH, Cui L, Ding YJ, Sun SH. Identification and functional analysis of a novel germ-line mutation in hMSH2 from a Chinese hereditary nonpolyposis colorectal cancer family. <span><span class="ref-journal">Acad J Sec Mil Med Univ. </span>2005;<span class="ref-vol">26</span>:888–891.</span> (In Chinese) <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acad+J+Sec+Mil+Med+Univ&amp;title=Identification+and+functional+analysis+of+a+novel+germ-line+mutation+in+hMSH2+from+a+Chinese+hereditary+nonpolyposis+colorectal+cancer+family&amp;author=HY+Jin&amp;author=HL+Yan&amp;author=LH+Song&amp;author=L+Cui&amp;author=YJ+Ding&amp;volume=26&amp;publication_year=2005&amp;pages=888-891&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b19-ol-0-0-8161">19. <span class="element-citation">Ng PC, Henikoff S. Predicting deleterious amino acid substitutions. <span><span class="ref-journal">Genome Res. </span>2001;<span class="ref-vol">11</span>:863–874. doi: 10.1101/gr.176601.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC311071/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11337480" ref="reftype=pubmed&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1101%2Fgr.176601" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Predicting+deleterious+amino+acid+substitutions&amp;author=PC+Ng&amp;author=S+Henikoff&amp;volume=11&amp;publication_year=2001&amp;pages=863-874&amp;pmid=11337480&amp;doi=10.1101/gr.176601&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b20-ol-0-0-8161">20. <span class="element-citation">Kumar P, Henikoff S, Ng PC. Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm. <span><span class="ref-journal">Nat Protoc. </span>2009;<span class="ref-vol">4</span>:1073–1081. doi: 10.1038/nprot.2009.86.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19561590" ref="reftype=pubmed&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnprot.2009.86" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Protoc&amp;title=Predicting+the+effects+of+coding+non-synonymous+variants+on+protein+function+using+the+SIFT+algorithm&amp;author=P+Kumar&amp;author=S+Henikoff&amp;author=PC+Ng&amp;volume=4&amp;publication_year=2009&amp;pages=1073-1081&amp;pmid=19561590&amp;doi=10.1038/nprot.2009.86&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b21-ol-0-0-8161">21. <span class="element-citation">Vander Molen J, Frisse LM, Fullerton SM, Qian Y, Del Bosque-Plata L, Hudson RR, Di Rienzo A. Population genetics of CAPN10 and GPR35: Implications for the evolution of type 2 diabetes variants. <span><span class="ref-journal">Am J Hum Genet. </span>2005;<span class="ref-vol">76</span>:548–560. doi: 10.1086/428784.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1199293/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15696418" ref="reftype=pubmed&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1086%2F428784" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Population+genetics+of+CAPN10+and+GPR35:+Implications+for+the+evolution+of+type+2+diabetes+variants&amp;author=J+Vander+Molen&amp;author=LM+Frisse&amp;author=SM+Fullerton&amp;author=Y+Qian&amp;author=L+Del+Bosque-Plata&amp;volume=76&amp;publication_year=2005&amp;pages=548-560&amp;pmid=15696418&amp;doi=10.1086/428784&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b22-ol-0-0-8161">22. <span class="element-citation">Nystrom-Lahti M, Perrera C, Raschle M, Panyushkina-Seiler E, Marra G, Curci A, Quaresima B, Costanzo F, D'Urso M, Venuta S, Jiricny J. Functional analysis of MLH1 mutations linked to hereditary nonpolyposis colon cancer. <span><span class="ref-journal">Genes Chromosomes Cancer. </span>2002;<span class="ref-vol">33</span>:160–167. doi: 10.1002/gcc.1225.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11793442" ref="reftype=pubmed&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fgcc.1225" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Chromosomes+Cancer&amp;title=Functional+analysis+of+MLH1+mutations+linked+to+hereditary+nonpolyposis+colon+cancer&amp;author=M+Nystrom-Lahti&amp;author=C+Perrera&amp;author=M+Raschle&amp;author=E+Panyushkina-Seiler&amp;author=G+Marra&amp;volume=33&amp;publication_year=2002&amp;pages=160-167&amp;pmid=11793442&amp;doi=10.1002/gcc.1225&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b23-ol-0-0-8161">23. <span class="element-citation">Clark AB, Cook ME, Tran HT, Gordenin DA, Resnick MA, Kunkel TA. Functional analysis of human MutSalpha and MutSbeta complexes in yeast. <span><span class="ref-journal">Nucleic Acids Res. </span>1999;<span class="ref-vol">27</span>:736–742. doi: 10.1093/nar/27.3.736.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC148241/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9889267" ref="reftype=pubmed&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2F27.3.736" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Functional+analysis+of+human+MutSalpha+and+MutSbeta+complexes+in+yeast&amp;author=AB+Clark&amp;author=ME+Cook&amp;author=HT+Tran&amp;author=DA+Gordenin&amp;author=MA+Resnick&amp;volume=27&amp;publication_year=1999&amp;pages=736-742&amp;pmid=9889267&amp;doi=10.1093/nar/27.3.736&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b24-ol-0-0-8161">24. <span class="element-citation">Rodr&#x000ed;guez-Negrete E, Bejarano ER, Castillo AG. Using the yeast two-hybrid system to identify protein-protein interactions. <span><span class="ref-journal">Methods Mol Biol. </span>2014;<span class="ref-vol">1072</span>:241–258. doi: 10.1007/978-1-62703-631-3_18.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24136527" ref="reftype=pubmed&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2F978-1-62703-631-3_18" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Mol+Biol&amp;title=Using+the+yeast+two-hybrid+system+to+identify+protein-protein+interactions&amp;author=E+Rodr&#x000ed;guez-Negrete&amp;author=ER+Bejarano&amp;author=AG+Castillo&amp;volume=1072&amp;publication_year=2014&amp;pages=241-258&amp;pmid=24136527&amp;doi=10.1007/978-1-62703-631-3_18&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b25-ol-0-0-8161">25. <span class="element-citation">Zhu M, Fan YM, Zhu YB, Wang YP. Establishment of a hMSH2/hMSH6 protein interaction system and functional evaluation of hMSH2 gene missense mutations. <span><span class="ref-journal">Zhonghua Yi Xue Yi Chuan Xue Za Zhi. </span>2013;<span class="ref-vol">30</span>:559–564.</span> (In Chinese) [<a href="https://pubmed.ncbi.nlm.nih.gov/24078570" ref="reftype=pubmed&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Zhonghua+Yi+Xue+Yi+Chuan+Xue+Za+Zhi&amp;title=Establishment+of+a+hMSH2/hMSH6+protein+interaction+system+and+functional+evaluation+of+hMSH2+gene+missense+mutations&amp;author=M+Zhu&amp;author=YM+Fan&amp;author=YB+Zhu&amp;author=YP+Wang&amp;volume=30&amp;publication_year=2013&amp;pages=559-564&amp;pmid=24078570&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b26-ol-0-0-8161">26. <span class="element-citation">Duraturo F, Liccardo R, Cavallo A, De Rosa M, Grosso M, Izzo P. Association of low-risk MSH3 and MSH2 variant alleles with Lynch syndrome: Probability of synergistic effects. <span><span class="ref-journal">Int J Cancer. </span>2011;<span class="ref-vol">129</span>:1643–1650. doi: 10.1002/ijc.25824.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21128252" ref="reftype=pubmed&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fijc.25824" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Cancer&amp;title=Association+of+low-risk+MSH3+and+MSH2+variant+alleles+with+Lynch+syndrome:+Probability+of+synergistic+effects&amp;author=F+Duraturo&amp;author=R+Liccardo&amp;author=A+Cavallo&amp;author=M+De+Rosa&amp;author=M+Grosso&amp;volume=129&amp;publication_year=2011&amp;pages=1643-1650&amp;pmid=21128252&amp;doi=10.1002/ijc.25824&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5920917&amp;issue-id=309448&amp;journal-id=1736&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Oncology Letters</span> are provided here courtesy of <strong>Spandidos Publications</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC5920917/pdf/ol-15-05-6275.pdf" class="int-view">PDF (806K)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/5920917/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/5920917/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="5920917" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5920917%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="6ifP5MnJpQMBGowlxBG6zrVCzQz43nbmLufoJQoX2E6TZzgZfacqF3hpxflVRP1I">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5920917%2F&amp;text=Analysis%20of%20human%20MutS%20homolog%202%20missense%20mutations%20in%20patients%20with%20colorectal%20cancer" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5920917%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5920917/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC5920917/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/29731845/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC5920917/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/29731845/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC5920917/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/5920917/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC5920917/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
