
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs2.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Predictive models of eukaryotic transcriptional regulation reveals changes in transcription factor roles and promoter usage between metabolic conditions - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="D0BD41E4AC192A4500004E8205403EF1.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="nar" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC6547448/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6547448/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Nucleic Acids Research" /><meta name="citation_title" content="Predictive models of eukaryotic transcriptional regulation reveals changes in transcription factor roles and promoter usage between metabolic conditions" /><meta name="citation_author" content="Petter Holland" /><meta name="citation_author_institution" content="Department of Biology and Biological Engineering, Chalmers University of Technology, Gothenburg SE-41296, Sweden" /><meta name="citation_author" content="David Bergenholm" /><meta name="citation_author_institution" content="Department of Biology and Biological Engineering, Chalmers University of Technology, Gothenburg SE-41296, Sweden" /><meta name="citation_author" content="Christoph S B&#x000f6;rlin" /><meta name="citation_author_institution" content="Department of Biology and Biological Engineering, Chalmers University of Technology, Gothenburg SE-41296, Sweden" /><meta name="citation_author" content="Guodong Liu" /><meta name="citation_author_institution" content="Department of Biology and Biological Engineering, Chalmers University of Technology, Gothenburg SE-41296, Sweden" /><meta name="citation_author" content="Jens Nielsen" /><meta name="citation_author_institution" content="Department of Biology and Biological Engineering, Chalmers University of Technology, Gothenburg SE-41296, Sweden" /><meta name="citation_author_institution" content="Novo Nordisk Foundation Center for Biosustainability, Chalmers University of Technology, Gothenburg SE-41296, Sweden" /><meta name="citation_author_institution" content="Novo Nordisk Foundation Center for Biosustainability, Technical University of Denmark, Kgs. Lyngby DK-2800, Denmark" /><meta name="citation_publication_date" content="2019/06/06" /><meta name="citation_issue" content="10" /><meta name="citation_volume" content="47" /><meta name="citation_firstpage" content="4986" /><meta name="citation_doi" content="10.1093/nar/gkz253" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC6547448/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC6547448/" /><meta name="citation_pmid" content="30976803" /><meta name="DC.Title" content="Predictive models of eukaryotic transcriptional regulation reveals changes in transcription factor roles and promoter usage between metabolic conditions" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Oxford University Press" /><meta name="DC.Contributor" content="Petter Holland" /><meta name="DC.Contributor" content="David Bergenholm" /><meta name="DC.Contributor" content="Christoph S B&#x000f6;rlin" /><meta name="DC.Contributor" content="Guodong Liu" /><meta name="DC.Contributor" content="Jens Nielsen" /><meta name="DC.Date" content="2019 Jun 4" /><meta name="DC.Identifier" content="10.1093/nar/gkz253" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Predictive models of eukaryotic transcriptional regulation reveals changes in transcription factor roles and promoter usage between metabolic conditions" /><meta property="og:type" content="article" /><meta property="og:description" content="Transcription factors (TF) are central to transcriptional regulation, but they are often studied in relative isolation and without close control of the metabolic state of the cell. Here, we describe genome-wide binding (by ChIP-exo) of 15 yeast TFs in ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6547448/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC6547448/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/4/">Nucleic Acids Res</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/335732/">v.47(10); 2019 Jun 4</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC6547448
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC6547448/pdf/gkz253.pdf" class="int-view">PDF (4.0M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/6547448/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/6547448/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="6547448" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6547448%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="9932vNij47XWW3teooJhShR4zx4xgPP0DYLKDAJZka8mBOVCcpHAXmtniC1nRcJK">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6547448%2F&amp;text=Predictive%20models%20of%20eukaryotic%20transcriptional%20regulation%20reveals%20changes%20in%20transcription%20factor%20roles%20and%20promoter%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6547448%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6547448/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC6547448/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/30976803/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC6547448/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/30976803/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC6547448/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/6547448/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/4/">Nucleic Acids Res</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/335732/">v.47(10); 2019 Jun 4</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC6547448
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/narlogo.gif" alt="Logo of nar" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/nar" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Nucleic Acids Res.</span> 2019 Jun 4; 47(10): 4986–5000. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2019 Apr 12. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1093%2Fnar%2Fgkz253" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/nar/gkz253</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC6547448</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/30976803">30976803</a></div></div></div><h1 class="content-title">Predictive models of eukaryotic transcriptional regulation reveals changes in transcription factor roles and promoter usage between metabolic conditions</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Holland%20P%5BAuthor%5D" class="affpopup" co-rid="_co_idm140438914465120" co-class="co-affbox">Petter Holland</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Bergenholm%20D%5BAuthor%5D" class="affpopup" co-rid="_co_idm140438877389120" co-class="co-affbox">David Bergenholm</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=B&#x000f6;rlin%20CS%5BAuthor%5D" class="affpopup" co-rid="_co_idm140438877704736" co-class="co-affbox">Christoph S B&#x000f6;rlin</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Liu%20G%5BAuthor%5D" class="affpopup" co-rid="_co_idm140438870573136" co-class="co-affbox">Guodong Liu</a>,<sup>1</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Nielsen%20J%5BAuthor%5D" class="affpopup" co-rid="_co_idm140438870571152" co-class="co-affbox">Jens Nielsen</a><sup>1,</sup><sup>2,</sup><sup>3</sup><sup></sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140438914465120"><h3 class="no_margin">Petter Holland</h3><p><sup>1</sup>Department of Biology and Biological Engineering, Chalmers University of Technology, Gothenburg SE-41296, Sweden</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Holland%20P%5BAuthor%5D">Petter Holland</a></div></div><div id="_co_idm140438877389120"><h3 class="no_margin">David Bergenholm</h3><p><sup>1</sup>Department of Biology and Biological Engineering, Chalmers University of Technology, Gothenburg SE-41296, Sweden</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Bergenholm%20D%5BAuthor%5D">David Bergenholm</a></div></div><div id="_co_idm140438877704736"><h3 class="no_margin">Christoph S B&#x000f6;rlin</h3><p><sup>1</sup>Department of Biology and Biological Engineering, Chalmers University of Technology, Gothenburg SE-41296, Sweden</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=B&#x000f6;rlin%20CS%5BAuthor%5D">Christoph S B&#x000f6;rlin</a></div></div><div id="_co_idm140438870573136"><h3 class="no_margin">Guodong Liu</h3><p><sup>1</sup>Department of Biology and Biological Engineering, Chalmers University of Technology, Gothenburg SE-41296, Sweden</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Liu%20G%5BAuthor%5D">Guodong Liu</a></div></div><div id="_co_idm140438870571152"><h3 class="no_margin">Jens Nielsen</h3><p><sup>1</sup>Department of Biology and Biological Engineering, Chalmers University of Technology, Gothenburg SE-41296, Sweden</p><p><sup>2</sup>Novo Nordisk Foundation Center for Biosustainability, Chalmers University of Technology, Gothenburg SE-41296, Sweden</p><p><sup>3</sup>Novo Nordisk Foundation Center for Biosustainability, Technical University of Denmark, Kgs. Lyngby DK-2800, Denmark</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Nielsen%20J%5BAuthor%5D">Jens Nielsen</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140438871067680_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140438871067680_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140438871067680_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140438871067680_ai" style="display:none"><div class="fm-affl" id="AFF1"><sup>1</sup>Department of Biology and Biological Engineering, Chalmers University of Technology, Gothenburg SE-41296, Sweden</div><div class="fm-affl" id="AFF2"><sup>2</sup>Novo Nordisk Foundation Center for Biosustainability, Chalmers University of Technology, Gothenburg SE-41296, Sweden</div><div class="fm-affl" id="AFF3"><sup>3</sup>Novo Nordisk Foundation Center for Biosustainability, Technical University of Denmark, Kgs. Lyngby DK-2800, Denmark</div><div id="COR1">To whom correspondence should be addressed. Tel: +46 31 772 3804; Fax: +46 31 772 3801; Email: <a href="mailto:dev@null" data-email="es.sremlahc@jneslein" class="oemail">es.sremlahc@jneslein</a></div></div><div class="fm-article-notes hide half_rhythm" id="idm140438871067680_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2019 Jan 11; Revised 2019 Mar 26; Accepted 2019 Apr 4.</div></div><div class="permissions half_rhythm hide" id="idm140438871067680_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; The Author(s) 2019. Published by Oxford University Press on behalf of Nucleic Acids Research.</div><div class="license half_rhythm">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>gkz253_Supplemental_Files.</strong></div><div><a href="/pmc/articles/PMC6547448/bin/gkz253_supplemental_files.zip" data-ga-action="click_feat_suppl">gkz253_supplemental_files.zip</a><span style="color:gray"> (3.7M)</span></div><div class="small guid">GUID: CDB190D7-40AF-480D-9204-A67E604C12BB</div></div></dd></dl><dl data-length="699" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first-last">ChIP-exo raw sequencing data (.fastq) can be retrieved form Arrayexpress accession E-MTAB-6673: <a href="https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6673/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6673/</a>. A processed format of the ChIP-exo data, summarized as TF reads per gene promoter is included in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Data 4</a>. RNA sequencing raw sequencing data (.fastq) can be retrieved from Arrayexpress accession E-MTAB-6657: <a href="https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6657/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6657/</a>. The number of reads annotated to each gene are found in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Data 3</a> and our processing of read counts to get to FPKM values that are used for all relevant analysis of this study can be reproduced through the R scripts included in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Data 6</a>.</p></dd></dl></div><div id="abstract-a.i.b.t" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.i.b.ttitle" style="text-transform: uppercase;">Abstract</h2><!--article-meta--><div><p class="p p-first-last">Transcription factors (TF) are central to transcriptional regulation, but they are often studied in relative isolation and without close control of the metabolic state of the cell. Here, we describe genome-wide binding (by ChIP-exo) of 15 yeast TFs in four chemostat conditions that cover a range of metabolic states. We integrate this data with transcriptomics and six additional recently mapped TFs to identify predictive models describing how TFs control gene expression in different metabolic conditions. Contributions by TFs to gene regulation are predicted to be mostly activating, additive and well approximated by assuming linear effects from TF binding signal. Notably, using TF binding peaks from peak finding algorithms gave distinctly worse predictions than simply summing the low-noise and high-resolution TF ChIP-exo reads on promoters. Finally, we discover indications of a novel functional role for three TFs; Gcn4, Ert1 and Sut1 during nitrogen limited aerobic fermentation. In only this condition, the three TFs have correlated binding to a large number of genes (enriched for glycolytic and translation processes) and a negative correlation to target gene transcript levels.</p></div></div><div id="SEC1" class="tsec sec"><h2 class="head no_bottom_margin" id="SEC1title" style="text-transform: uppercase;">INTRODUCTION</h2><p class="p p-first">The relationship between transcription factor (TF) binding to DNA and gene transcription in eukaryotes is complex. This is highlighted in several studies integrating chromatin immunoprecipitation (ChIP)-based TF binding data with transcriptomics from knockout or knockdown experiments of the TF with the goal of defining regulatory targets. Studies of transcriptional response to hormones found that in mice with the TF glucocorticoid receptor knocked out, only 11% of the differently expressed genes were targeted by the TF (<a href="#B1" rid="B1" class=" bibr popnode">1</a>) and a similar study of human estrogen receptor function found only 6% of differentially expressed genes to be targeted (<a href="#B2" rid="B2" class=" bibr popnode">2</a>). Integration of a large-scale study using microarray transcriptomics in yeast TFs deletion strains (<a href="#B3" rid="B3" class=" bibr popnode">3</a>) with previously generated ChIP-chip data (<a href="#B4" rid="B4" class=" bibr popnode">4</a>) for 188 TFs showed even less overlap with an average 3% of differentially expressed genes being targeted by the corresponding TF (<a href="#B5" rid="B5" class=" bibr popnode">5</a>). Thus, combining ChIP methods with transcriptomics to understand transcriptional regulation in eukaryotic systems gives disappointing results compared to the demonstrated successes of this approach in bacteria (<a href="#B6" rid="B6" class=" bibr popnode">6</a>).</p><p>The increased difficulty in understanding eukaryal gene regulation in comparison to bacteria may be explained by the additional levels of regulation present, such as nucleosome&#x02013;TF interactions, histone modifications and long-range effects of binding. The impressive ENCODE dataset containing TF binding, transcriptomics as well as other chromatin features (<a href="#B7" rid="B7" class=" bibr popnode">7</a>) has been used to explore the contributions of different features of human promoters to gene regulation. Using machine learning approaches, strong predictive models were created and analysis of the models suggested a highly interconnected regulatory system where TF binding has functional interactions with both nucleosome occupancy and histone modifications to regulate transcriptional outcomes (<a href="#B8" rid="B8" class=" bibr popnode">8</a>). A different approach to create predictive models of transcriptional regulation based only on TF binding was to build a model from TF-association scores that includes both the strength of the binding event and the distance from a given gene in data collected from mouse embryonic stem cells (<a href="#B9" rid="B9" class=" bibr popnode">9</a>). For 12 TFs, these scores were combined into principal components and using linear regressions on the principal components it was possible to predict an impressive 65% of the variation in gene expression genome wide (<a href="#B9" rid="B9" class=" bibr popnode">9</a>). Such complex analysis methods will be important tools to fully understand eukaryal transcriptional regulation and may allow cell engineering relying on predictably changes in gene transcription.</p><p class="p p-last">While binding has been mapped for most central yeast TFs in one of the impressive large-scale studies (<a href="#B4" rid="B4" class=" bibr popnode">4</a>,<a href="#B10" rid="B10" class=" bibr popnode">10&#x02013;12</a>), the majority of this data is captured only in a single state of the cell; exponential growth in nutrient excess. Here we performed a large-scale study of mapping TF binding of multiple yeast TFs known to be involved in metabolic regulation by ChIP-exo (chromatin immunoprecipitation with lambda exonuclease) in four distinct metabolic states of the yeast cell. We integrate TF binding data with transcriptomics of the same metabolic conditions with the goal of building predictive models using relatively simple statistical methods that allow full transparency for insights into contributions of different TFs to gene expression. Using ChIP-exo allowed us to study TF binding with high resolution and minimal background and using yeast as a model organism allowed us to study metabolic gene regulation utilizing a variety of nutrients with a constant growth rate in chemostats.</p></div><div id="SEC2" class="tsec sec"><h2 class="head no_bottom_margin" id="SEC2title" style="text-transform: uppercase;">MATERIALS AND METHODS</h2><div id="SEC2-1" class="sec sec-first"><h3 id="SEC2-1title">Yeast strains and media</h3><p class="p p-first">The host strain for all experiments contained in this study was <em>Saccharomyces cerevisiae</em> CEN.PK 113-5D (URA-). CEN.PK 113&#x02013;5D with <em>Kluyveromyces lactis</em> URA3 (KiURA3) re-integrated was used as control strain for transcriptome analysis. Strains for ChIP-exo were created by amplifying either a TAP tag or a 9xMyc tag with KiURA3 and homology arms for recombination into the C-terminal end of the TF coding sequence.</p><p class="p p-last">The components of the chemostat media that were different between the experimental conditions are as follows: Nitrogen limited media &#x02013; 1 g/l (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 5.3 g/l K<sub>2</sub>SO<sub>4</sub>, 150 ml/l glucose 40%, 12 drops Antifoam204. Ethanol limited media &#x02013; 5 g/l (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 6.67 ml/l Ethanol 96%, 12 drops Antifoam204. Respiratory glucose limited media &#x02013; 5 g/l (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 18.75 ml/l glucose 40%, 12 drops Antifoam204. Anaerobic glucose limited media &#x02013; 5 g/l (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 25 ml/l glucose 40%, 4 ml/l ergosterol in Tween80 (2.6 g/l), 16 drops Antifoam204. In addition to the previously stated components changing between the media, all media have the following: 14.4 g/l KH<sub>2</sub>PO<sub>4</sub>, 0.5 g/l MgSO<sub>4</sub>, 1 ml/l of 1000&#x000d7; vitamin and 1000&#x000d7; trace metal stock solutions. The 1000&#x000d7; stocks contains the following: Vitamins &#x02013; 0.05 g/l biotin, 0.2 g/l 4-aminobenzoic acid, 1 g/l nicotinic acid, 1 g/l calcium pantothenate, 1 g/l pyridoxine HCl, 1 g/l thiamine HCl, and 25 g/l myo-inositol. Trace metals &#x02013; 15.0 g/l EDTA-Na<sub>2</sub>, 4.5 g/l ZnSO<sub>4</sub>&#x000b7;7H<sub>2</sub>O, 0.84 g/l MnCl<sub>2</sub>&#x000b7;2H<sub>2</sub>O, 0.3 g/l CoCl<sub>2</sub>&#x000b7;6H<sub>2</sub>O, 0.3 g/l CuSO<sub>4</sub>&#x000b7;5H<sub>2</sub>O, 0.4 g/l Na<sub>2</sub>MoO<sub>4</sub>&#x000b7;2H<sub>2</sub>O, 4.5 g/l CaCl<sub>2</sub>&#x000b7;2H<sub>2</sub>O, 3 g/l FeSO<sub>4</sub>&#x000b7;7H<sub>2</sub>O, 1g/l H<sub>3</sub>BO<sub>3</sub>&#x000a0;and 0.1 g/l KI. pH of the media was adjusted by adding KOH pellets to get media pH of 6.0&#x02013;6.5 that result in a final pH of all chemostat cultures close to 5.5.</p></div><div id="SEC2-2" class="sec"><h3 id="SEC2-2title">Chemostat cultivation</h3><p class="p p-first-last">Cells were cultivated in chemostats with a dilution rate of 0.1 h<sup>&#x02212;1</sup> at 30c. Stirring and aeration was performed by either N2 (fermentative glucose metabolism) or pressurized air (for the three other conditions) supplied to the cultures (<a href="#B13" rid="B13" class=" bibr popnode">13</a>). Cultures were sampled for either ChIP-exo or transcriptomics after steady state was achieved for 48&#x02013;60 h.</p></div><div id="SEC2-3" class="sec"><h3 id="SEC2-3title">ChIP-exo</h3><p class="p p-first-last">When chemostat cultures were measured to be stable for 48&#x02013;60 h, formaldehyde with a final concentration of 1% (wt/vol) and distilled water were added to the cultures to create a final OD<sub>600</sub> of 1.0 and a total volume of 100ml. Cells were incubated in formaldehyde for 12 min at room temperature followed by quenching by addition of <span class="small-caps">l</span>-glycine to a final concentration of 125 mM. Cells were then washed twice with cold TBS and snap-frozen with liquid N<sub>2</sub>. ChIP-exo was then performed according to a protocol based on the originally established protocol (<a href="#B14" rid="B14" class=" bibr popnode">14</a>) with certain modifications, as described in (<a href="#B15" rid="B15" class=" bibr popnode">15</a>). Presentation of the ChIP-exo raw data and replicates is included in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Data&#x000a0;1</a>.</p></div><div id="SEC2-4" class="sec"><h3 id="SEC2-4title">Peak finding and target gene identification</h3><p class="p p-first-last">Peak detection was performed by GEM (<a href="#B16" rid="B16" class=" bibr popnode">16</a>) with default parameters. A peak signal threshold of &#x0003e;2-fold peak signal over the local genomic noise was applied and peaks were annotated to a gene if it was found within &#x02013;500 to +500 bp of a given genes TSS, as defined by (<a href="#B17" rid="B17" class=" bibr popnode">17</a>). The full list of peaks detected by GEM (without peak signal threshold) for each TF is included in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Data 2</a>.</p></div><div id="SEC2-5" class="sec"><h3 id="SEC2-5title">RNA sequencing</h3><p class="p p-first-last">From chemostats at steady-state, 10 OD<sub>600</sub> from three biological replicates were collected into tubes and put directly on ice. Cells were washed twice in cold TBS and snap-frozed in liquid N2. RNA extraction was performed as described in the manual for the RNeasy<sup>&#x000ae;</sup> Mini kit (QIAGEN). RNA quality was inspected by Nanodrop, Qubit and Bioanalyzer before proceeding with sample preparation for Illumina sequencing and following sequencing on the NextSeq 500 system (2 &#x000d7; 75 bp, mid-output mode; Illumina). The RNA sequencing read counts per gene in each metabolic condition is included in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Data 3</a>.</p></div><div id="SEC2-6" class="sec"><h3 id="SEC2-6title">Sequencing data processing</h3><p class="p p-first">For both the ChIP-exo data and transcriptomics, the raw sequencing output (.fastq) was mapped to a recently published CEN.PK genome (<a href="#B18" rid="B18" class=" bibr popnode">18</a>) using Bowtie2 (<a href="#B19" rid="B19" class=" bibr popnode">19</a>) with the -U parameter. Samtools (<a href="#B20" rid="B20" class=" bibr popnode">20</a>) was then used to generate sorted and indexed&#x000a0;.bam files by first creating .bam files by the &#x02018;view' command with parameters &#x02013;bS &#x02013;q 20 and further the &#x02018;sort' and &#x02018;index'&#x000a0;commands.</p><p>For the ChIP-exo data, the read count covering each nucleotide position genome-wide was determined from .bam files using the genomecov function of BEDTools (<a href="#B21" rid="B21" class=" bibr popnode">21</a>), where a read length of 10 was used for all TFs. The read counts of biological duplicate were then averaged and output as .wig files that can be found in our Mendeley Data archives as described under the Data Availability section. All further processing of ChIP-exo data is through import of .wig files into R (<a href="#B22" rid="B22" class=" bibr popnode">22</a>) and executable scripts to reproduce all figures are included in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Data 6</a>. We also supply in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Data 4</a> processed versions of the .wig data, containing only the total TF read counts for each gene promoter (TSS &#x02013;500 to TSS +500 bp) for each metabolic condition. These files can be used directly as inputs to replicate our linear regression analysis.</p><p class="p p-last">For RNA sequencing data analysis, Subread (<a href="#B23" rid="B23" class=" bibr popnode">23</a>) was used to map the aligned reads (.bam) to gene annotations. The resulting output of transcript read counts for the replicates from Subread (171206_CENPK_subOut_wt.txt) can be found in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Data 3</a>. Using R (<a href="#B22" rid="B22" class=" bibr popnode">22</a>), genes were first filtered to only include those with minimum 1 read detected in all samples and then the edgeR (<a href="#B24" rid="B24" class=" bibr popnode">24</a>) package was used to calculate FPKM values before averaging the triplicates. All subsequent analysis using the transcriptomics data in the manuscript can be reproduced using the R scripts supplied in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Data 6</a>.</p></div><div id="SEC2-7" class="sec sec-last"><h3 id="SEC2-7title">Linear regressions</h3><p class="p p-first">The mathematical expression of a simple linear regression using one explanatory variable (TF binding of a single TF in this case) to model the relationship to the dependent variable (FPKM transcript level in our case) is <em>Y<sub>i</sub></em> = &#x003b2;<sub>0</sub> + &#x003b2;<sub>1</sub><em>X</em><sub>1<em>i</em></sub> + &#x003f5;<sub><em>i</em></sub>. <em>Y</em><sub><em>i</em></sub> is the FPKM of gene <em>i</em>, &#x003b2;<sub>0</sub> is the intercept (common to all genes in the regression), <em>X</em><sub>1<em>i</em></sub> is the amount of TF binding to gene <em>i</em>, &#x003b2;<sub>1</sub> is the coefficient that is selected to give the best fit together with the intercept to the transcript levels (common to all genes in the regression) and &#x003f5;<sub><em>i</em></sub> is the error of the prediction for gene <em>i</em>.</p><p class="p p-last">In multiple linear regressions, several predictors are added together, each with their own coefficient. The equation then takes the format: <em>Y<sub>i</sub></em> = &#x003b2;<sub>0</sub> + &#x003b2;<sub>1</sub><em>X</em><sub>1<em>i</em></sub> + &#x02026; + &#x003b2;<sub><em>k</em></sub> X<sub><em>ki</em></sub> + &#x003f5;<sub><em>i</em></sub> where in our case k indicates the index of a TF. While our analysis using TF binding contains 21 TFs, we always use variable selection (TF selection) from the earth package (<a href="#B25" rid="B25" class=" bibr popnode">25</a>) in R for multiple linear regressions, in which only the most predictive set of TFs will be included and added together for predictions of transcript levels of a given set of genes. In some of our analysis we also allow the earth scripts to introduce splines (earth() parameters &#x02018;linpreds = F&#x02019; and &#x02018;endspan = 100&#x02019;), effectively allowing the algorithm to model regions where it is advantageous to have nonlinearities in the explanatory variables.</p></div></div><div id="SEC3" class="tsec sec"><h2 class="head no_bottom_margin" id="SEC3title" style="text-transform: uppercase;">RESULTS</h2><div id="SEC3-1" class="sec sec-first"><h3 id="SEC3-1title">Most yeast metabolic TFs show large changes in genes targets in different metabolic states</h3><p class="p p-first">To get information about several distinctly different states of yeast metabolism we decided to analyze gene expression regulation in chemostat cultures operated at the same specific growth rate, but still causing a range of different types of metabolism; aerobic fermentation using nitrogen limitation, respiratory glucose metabolism using glucose limitation, fermentative glucose metabolism using anaerobic conditions, and gluconeogenic respiration using ethanol limitation. These four states of metabolism should involve large changes in central carbon metabolism and hence we focused on TFs that have enriched binding (relative to all other binding targets) to central carbon metabolism enzymes. To define a list of TFs to focus on we started from the landmark dataset collected by Harbison <em>et&#x000a0;al.</em> containing TF promoter enrichment genome-wide for a majority of yeast TFs mapped by ChIP-chip in batch cultures with rich media (<a href="#B4" rid="B4" class=" bibr popnode">4</a>). All TFs that had &#x0003e;50 total targeted genes and &#x0003e;5% enrichment of central carbon metabolism genes were selected as candidates, as well as certain additional TFs suggested from other studies to be important for controlling central carbon metabolism. The criteria and process of selecting TFs for this study is described in more detail in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Data 5</a>.</p><p>To map and quantify TF binding, strains were created with TFs tagged by a C-terminal TAP or 9xMyc tag. All strains were validated for presence of the tag as well as functional binding of the tagged TF to a known target gene's promoter by ChIP-qPCR. The successfully validated strains were cultivated as biological duplicates in the four different chemostat conditions and genome-wide binding events were mapped and quantified by ChIP-exo. This method is an improvement over ChIP-seq, including exonuclease treatment of the cross-linked TF-DNA complex to increase the resolution and reduce unspecific background binding (<a href="#B14" rid="B14" class=" bibr popnode">14</a>). A demonstration of our raw data and replicates is shown for each TF in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Data 1</a>.</p><p>Peaks were identified by GEM (<a href="#B16" rid="B16" class=" bibr popnode">16</a>), the duplicates averaged, and a signal threshold of &#x0003e;2 peak signal relative to the noise of the local genomic context was applied. Comparing to what degree the targeted genes overlap between the experimental conditions (Figure <a href="/pmc/articles/PMC6547448/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1A),</span></span><span>1A</span></a>), only two of the 15 TFs first reported in this study show a relatively stable set of targets between the studied nutrient limited conditions: Cbf1 and Gcr1. For the remaining TFs there are large changes in which genes are being targeted between conditions. By analyzing the targeted genes for the most relatively enriched gene ontology (GO) term, we confirm many well-known metabolic roles for these TFs, but also find indications for new functions such as drug transport for Ert1 and carbohydrate transport for Sut1 (Figure <a href="/pmc/articles/PMC6547448/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1A</span></span><span>1A</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC6547448/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140438876236848"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6547448_gkz253fig1.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkz253fig1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6547448/bin/gkz253fig1.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140438876236848"><a target="object" rel="noopener" href="/pmc/articles/PMC6547448/figure/F1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC6547448/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1.</a></div><!--caption a7--><div class="caption"><p>(<strong>A</strong>) Genome-wide TF binding characterization for multiple conditions of 15 TFs first described in this study. The most enriched DNA sequence (defined by MEME) is indicated under the TF name. The euler diagrams indicate the number of genes targeted between the experimental conditions and how the conditions overlap. Numbers are shown for all overlaps with more than two genes. For each condition, the top significant (if there is any with <em>P</em>.adj &#x0003c; 0.05) GO category for the genes targeted that condition is shown in the same color color of the condition in the euler diagram. (<strong>B</strong>) A summary of our experimental approach.</p></div></div></div><p>For further analysis of the relationship between TF binding and transcriptional outcomes, we combined binding information of the 15 TFs reported here with data for an additional 6 TFs obtained with the same experimental conditions and using the same protocols (Ino2, Ino4, Hap1, Oaf1 and Pip2 from Bergenholm <em>et&#x000a0;al.</em> (<a href="#B26" rid="B26" class=" bibr popnode">26</a>) and Stb5 from Ouyang <em>et&#x000a0;al.</em> (<a href="#B27" rid="B27" class=" bibr popnode">27</a>) (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S1A</a>). We first explored the general distribution of peaks on promoters relative to the TSS and we found the expected strong enrichment upstream of the TSS for all metabolic conditions (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S1b</a>). Notably, when comparing the number of peaks detected for each condition, we discovered a significant decrease in count of peaks for most of the studied TFs in aerobic fermentation (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S1C</a>).</p><p>To look for an explanation for the broad changes in which genes are targeted between conditions we compared the most enriched DNA motif bound by the TFs for the different metabolic conditions. For most TFs, we found only minor differences in the enriched motif between conditions (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S2A</a>). There are a few notable situations where the most enriched motif may be different in one condition, for example Gcn4 and Rtg1 in aerobic fermentation, but we cannot conclude if these cases are from a true change of DNA preference for the TF or if it is due to noisy variation from the TF having fewer peaks in aerobic fermentation. For the lipid metabolism TFs Ino2,&#x000a0;Ino4, Hap1, Oaf1 and Pip2, this analysis was performed in Bergenholm <em>et&#x000a0;al.</em> (<a href="#B26" rid="B26" class=" bibr popnode">26</a>), with similar conclusions.</p><p>Our data and peak detection was further compared to two previously published datasets. We focused on eight TFs that are first reported in this manuscript where binding in multiple conditions was reported by Harbison <em>et&#x000a0;al.</em> (<a href="#B4" rid="B4" class=" bibr popnode">4</a>) and we also compared to the refined peak definitions reported in the MacIsaac <em>et&#x000a0;al.</em> (<a href="#B28" rid="B28" class=" bibr popnode">28</a>). We see strong overlap for many TFs and generally stronger overlap with the Harbison <em>et&#x000a0;al.</em> experiments using synthetic media (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S2B</a>), an expected observation because this media more closely resembles our experimental conditions. Two TFs show relatively poor overlap with existing datasets, Rtg1 and Rtg3, but we do see a strong enrichment of both TFs to genes involved in amino acid biosynthesis (Figure <a href="/pmc/articles/PMC6547448/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1A),</span></span><span>1A</span></a>), which is a role for these TFs that is thoroughly demonstrated (<a href="#B29" rid="B29" class=" bibr popnode">29</a>,<a href="#B30" rid="B30" class=" bibr popnode">30</a>) and makes us confident in the quality of our data also for these TFs.</p><p class="p p-last">While the selection of TFs was focused on finding TFs enriched for binding to central carbon metabolism genes, we decided to expand the gene sets for further studies of how the TFs are affecting transcriptional regulation to cover all metabolic genes. Metabolic genes were defined as being included in the latest published yeast genome-scale model, v7.6 (<a href="#B31" rid="B31" class=" bibr popnode">31</a>); in total 849 genes from the model that have a clearly defined TSS (<a href="#B17" rid="B17" class=" bibr popnode">17</a>) and where we also have robust gene expression data from transcriptomics were selected for further analysis. Using all metabolic genes was a compromise to have enough genes for strong statistical power and reliable observations from predictive models, but also retain the property of having relatively good TF-coverage of the genes. Our experimental approach is summarized in Figure <a href="/pmc/articles/PMC6547448/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure1B</span></span><span>1B</span></a>.</p></div><div id="SEC3-2" class="sec"><h3 id="SEC3-2title">Comparing predictive models of transcriptional regulation</h3><p class="p p-first">We next compared performance of different types of preprocessing of the TF binding data in predicting transcript levels (measured by RNA sequencing) using multiple linear regressions. The regressions assume a linear relationship between TF binding and effects on transcriptional regulation and we build a model where TFs binding signal is multiplied by a coefficient and added together to predict transcript levels. We first tested different signal/noise ratio (SNR) thresholds for TF peak binding signal, but found only a minimal effect on performance of the predictive models (Figure <a href="/pmc/articles/PMC6547448/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2A).</span></span><span>2A</span></a>). A different numeric representation of TF binding is to sum TF binding over an interval of DNA and we found that summing all binding -50 to +50bp around the identified peaks gave stronger predictive power to transcriptional outcomes (Figure <a href="/pmc/articles/PMC6547448/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2A).</span></span><span>2A</span></a>). We further tested an even simpler summation of the whole promoter region and found that this gave even better predictive power (Figure <a href="/pmc/articles/PMC6547448/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2A).</span></span><span>2A</span></a>). We think this improvement is most likely driven by contributions to transcriptional regulation from relatively weaker TF binding events that are not strong enough to be detected by a peak finding algorithm. The reduced background of ChIP-exo is here leveraged to be able to detect such weaker events over background noise. The promoter signal sum data format was also tested with multivariate adaptive regression splines (MARS) (<a href="#B32" rid="B32" class=" bibr popnode">32</a>). In MARS, if it is advantageous for prediction performance, the algorithm can introduce splines in the linear regressions, effectively allowing a type of peak definition where the peak threshold (spline) is introduced to create a linear relationship between TF binding and transcript levels only for a certain range of TF binding strength. We found that with MARS, the performance of the predictions further increased.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC6547448/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140438874432672"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6547448_gkz253fig2.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkz253fig2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6547448/bin/gkz253fig2.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140438874432672"><a target="object" rel="noopener" href="/pmc/articles/PMC6547448/figure/F2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC6547448/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2.</a></div><!--caption a7--><div class="caption"><p>Comparing performance of TF binding data preprocessing in linear regressions to predict transcript levels and details of multivariate adaptive regression splines (MARS) models. (<strong>A</strong>) Correlations between predicted transcript levels and real transcript levels for the different formats of TF binding data. The black line indicates the mean of the four metabolic conditions. (<strong>B</strong>&#x02013;<strong>E</strong>) MARS used to predict metabolic gene transcript levels of the different conditions from the amount of TF binding per gene promoter. The boxes shown below the predictions plots represent the different TFs that are selected by MARS to give strongest predictive performance in the conditions and how their signal is contributing to predictions in the model.</p></div></div></div><p class="p p-last">We were curious to see where in the promoter region TF binding is most strongly contributing to gene regulation. We tested the predictive power of binding in segments of the promoter using linear regressions and found that binding signal upstream of the TSS (where we also detect the majority of strong TF-binding peaks, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S1B</a>) is predicted to be most consequential for transcriptional regulation (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S2C</a>), but with a notable influence also from binding directly downstream of the TSS. Comparing the conditions, it appears that there is a relative increase in influence of TF binding directly downstream of the TSS in aerobic fermentation (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S2c</a>; highest point of red line is downstream of TSS while highest point of the other conditions is upstream of TSS). To select a region of a gene's promoter which captures as much as possible of the consequential TF binding for further analysis, we started with the assumption of a symmetrical region around the TSS (assumed based on <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S2c</a>) and tested extensions of this region in 50&#x000a0;bp increments for predicting transcript levels (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S2d</a>). The performance of predictions increase until it reaches &#x02013;500 to +500 around the TSS, after which there is no further increase, indicating that this region contains a majority of the consequential TF binding.</p></div><div id="SEC3-3" class="sec"><h3 id="SEC3-3title">MARS define a set of core TFs for different conditions and reveal general quantitative features of the relationship between TF binding and transcriptional regulation</h3><p class="p p-first">Based on the finding that MARS provided the best predictions of transcript levels from TF binding (Figure <a href="/pmc/articles/PMC6547448/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2A),</span></span><span>2A</span></a>), we explored what we could learn about the roles and functions of TFs from MARS regressions. For multiple linear regressions, the interesting parameters to describe TF function are the coefficients, which can tell us if the TF is an activator (positive correlation between binding and transcript levels) or a repressor (negative correlation between binding and transcript levels). The MARS algorithm can also introduce splines to improve prediction performance, which can be another parameter of TF function, describing the range of TF binding where there is a linear relationship with transcriptional outcomes. Finally, variable selection in MARS will select only the best combination of TFs to predict as much as possible while penalizing increasing the complexity of the model. To define a set of TFs that are most strongly predictive of transcriptional regulation for each condition, we used MARS with cross validation for the TF and spline selection to ensure that only the most robust TFs and splines were included in the final model (model building is illustrated in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S3A</a>). The resulting predictions of transcript levels from TF binding using MARS for the four metabolic conditions are shown in Figure <a href="/pmc/articles/PMC6547448/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure2B</span></span><span>2B</span></a>&#x02013;<a href="/pmc/articles/PMC6547448/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>E</span></a>. Using these conservative MARS models, we could predict 34&#x02013;43% of the variation in expression levels of metabolic genes at the four metabolic conditions we investigated. We judged this predictive power as being sufficiently good to assume that the parameters given to the TFs in these predictive models can give insights into how the TFs are contributing to gene regulation in metabolism. Typical quality control checks of the MARS models presented in Figure <a href="/pmc/articles/PMC6547448/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure2B</span></span><span>2B</span></a>&#x02013;<a href="/pmc/articles/PMC6547448/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>E</span></a>, such as the distribution of residuals and QQ plots are shown in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S3B</a>.</p><p class="p p-last">The details of how TFs are contributing at different TF binding strengths in these predictive models are shown in the lower panels of Figure <a href="/pmc/articles/PMC6547448/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure2B</span></span><span>2B</span></a>&#x02013;<a href="/pmc/articles/PMC6547448/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>E</span></a>. The coefficient of the TFs from the regressions are illustrated here by either an upward slope (activation) or a downward slope (repression). A striking observation from these models is that of the 22 TF-transcript level correlations selected by MARS over the four metabolic conditions, the linear correlations were all positive, suggesting a predominance of activation in yeast metabolic gene regulation by TFs. MARS can also create segments where there is no correlation between TF binding and transcript levels if that is advantageous for the predictions. The most common way that the MARS algorithms created splines from TF binding (15/22 cases) is to introduce a threshold after which there is a linear relationship between TF binding and transcript level. MARS also found relationships between TF binding and transcription that show a saturation effect, where more binding does not lead to higher expression (6/22 cases). Of these six cases, four of them are from the well-known interaction partners Ino2 and Ino4, suggesting that nonlinearity between binding and functional outcomes may be a general feature of this TF complex.</p></div><div id="SEC3-4" class="sec"><h3 id="SEC3-4title">Contributions from several TFs on gene regulation are generally additive, but exploring collinear TFs indicates cases of more complex functional interactions</h3><p class="p p-first">In multiple linear regressions, correlation between explanatory variables (here TFs) leads to multicollinearity, a redundancy in the information contained in explanatory variables which may complicate the interpretation of the model, In our data, collinearity between TFs could be due to TFs interacting in a common protein complex, or responding independently to the same cellular cues that regulate TF-DNA binding. A feature of the TF selection in MARS is that if there is collinearity between two TFs that are both strongly predictive of transcript levels, only the TF that gives slightly better predictions will be included while the other TF will not be visible. To look for such collinear TFs we first mapped the correlations in binding signal over metabolic genes and calculated significance of the correlations (Figure <a href="/pmc/articles/PMC6547448/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3A</span></span><span>3A</span></a>&#x02013;<a href="/pmc/articles/PMC6547448/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>D</span></a>). To find cases in the MARS models of collinearity where a TF is not included because there is a slightly better predictor selected, we tested if all included TFs could be substituted by other TFs with significantly correlated binding. Such cases are shown with black borders in Figure <a href="/pmc/articles/PMC6547448/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure3A</span></span><span>3A</span></a>&#x02013;<a href="/pmc/articles/PMC6547448/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>D</span></a> and they are TF pairs predicted to regulate similar sets of genes in similar ways, indicating overlapping functions and/or possibly more complex TF&#x02013;TF interactions. This analysis revealed several known cases of TF interactions such as Ino2&#x02013;Ino4, Gcr1-Gcr2-Tye7 and Cat8-Sip4, but also novel potential interactions such as Gcn4-Rtg1 and Ert1-Ino4.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC6547448/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140438876421216"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkz253fig3.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is gkz253fig3.jpg" src="/pmc/articles/PMC6547448/bin/gkz253fig3.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140438876421216"><a target="object" rel="noopener" href="/pmc/articles/PMC6547448/figure/F3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC6547448/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3.</a></div><!--caption a7--><div class="caption"><p>Exploring contributions of collinear TF pairs to transcriptional regulation. (<strong>A</strong>&#x02013;<strong>D</strong>) Correlation plots illustrating Pearsons correlations (in color) between TF binding in promoters of metabolic genes. Significance (Pearson's product moment correlation coefficient) is illustrated for TF pairs with <em>P</em> &#x0003c; 0.05, by one or several asterisks, as indicated. Pairs of significantly collinear TFs that are interchangeable in the MARS TF selection in Figure <a href="/pmc/articles/PMC6547448/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure2B</span></span><span>2B</span></a>&#x02013;<a href="/pmc/articles/PMC6547448/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>E</span></a> are indicated by a stronger border in (A&#x02013;D). (E&#x02013;<strong>H</strong>) Linear regressions of collinear TF pairs were tested with and without allowing a multiplication of TF signals of the two TFs. TF pairs indicated in red and with larger fonts have an <em>R</em><sup>2</sup> of the additive regression &#x0003e;0.1 and increased performance with including a multiplication of the TF pairs of at least 10%.</p></div></div></div><p class="p p-last">In the MARS models shown in Figure <a href="/pmc/articles/PMC6547448/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure2B</span></span><span>2B</span></a>&#x02013;<a href="/pmc/articles/PMC6547448/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>E</span></a>, the contribution of TFs binding to each gene is multiplied by a coefficient and then added to get the final predicted transcript level for that gene. We further looked for TF-TF interactions that contribute to transcriptional regulation in ways that are numerically more complex than simple addition. All the significantly correlated TFs were tested if the multiplication of the signal of two collinear TFs give additional predictive power compared to addition of the two TFs (Figure <a href="/pmc/articles/PMC6547448/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3E</span></span><span>3E</span></a>&#x02013;<a href="/pmc/articles/PMC6547448/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>H</span></a>). Most collinear TF pairs do not show a strong improvement in predictive power by including a multiplicative interaction term, for example the mentioned potential TF interactions of Cat8-Sip4 and Gcn4-Rtg1 during gluconeogenic respiration which only gave a 3% and 4% increase in predictive power, respectively (Figure <a href="/pmc/articles/PMC6547448/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3F,</span></span><span>3F</span></a>, percentage improvement calculated by (multiplicative R2 increase (y-axis) + additive R2 (x-axis))/additive R2 (x-axis)). The TF pair that shows the clearest indications of having a more complex functional interaction is Ino2&#x02013;Ino4, having 19%, 11%, 39% and 20% improvement (Figure <a href="/pmc/articles/PMC6547448/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3E</span></span><span>3E</span></a>&#x02013;<a href="/pmc/articles/PMC6547448/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>H</span></a>) in predictive power in the tested metabolic conditions by including a multiplication of the binding signals. TF pairs that together explain &#x0003e;10% of the metabolic gene variation using an only additive regression and also show minimum 10% improved predictive power when allowing multiplication are indicated in red in Figure <a href="/pmc/articles/PMC6547448/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure3E</span></span><span>3E</span></a>&#x02013;<a href="/pmc/articles/PMC6547448/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>H</span></a>. For Ino2&#x02013;Ino4, the strongest effect of the multiplication term is seen during fermentative glucose metabolism with 39% improved predictive power (Figure <a href="/pmc/articles/PMC6547448/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3G).</span></span><span>3G</span></a>). The plot for how the multiplied Ino2&#x02013;Ino4 signal is contributing to the regression in this condition reveal that in the genes where both TFs bind strongest together, there is a predicted reduced activation as compared to intermediate binding strengths of both TFs, and a similar trend is seen for the Ino2&#x02013;Ino4 pair for other metabolic conditions (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S3c</a>).</p></div><div id="SEC3-5" class="sec"><h3 id="SEC3-5title">Clustering metabolic genes based on their relative change in expression gives a strong enrichment of metabolic processes and improved predictive power of TF binding in linear regressions</h3><p class="p p-first">Linear regressions of metabolic genes with TF selection through MARS defined a small set of TFs that were robustly associated with transcriptional changes over all metabolic genes (Figure <a href="/pmc/articles/PMC6547448/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2B</span></span><span>2B</span></a>&#x02013;<a href="/pmc/articles/PMC6547448/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>E</span></a>), but TFs that only regulate a smaller group of genes would be unlikely to get selected by this method. We clustered genes by their sum-of-squares normalized expression between conditions to get smaller clusters of genes with a range of gene expression levels that are appropriate for predictive modeling by multiple linear regressions. The motivation for clustering genes into smaller groups is to be able to link TFs to specific patterns of gene expression changes between the tested metabolic conditions and to functionally connected groups of genes&#x02013; thus allowing more detailed predictions about the TFs&#x02019; biological roles. The optimal number of clusters to maximize the separation of the normalized expression values of metabolic genes was 16, as determined by Bayesian information criterion (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S4A</a>). Genes were sorted into 16 clusters by k-means clustering and we found that most clusters then show significant enrichment of metabolic processes, represented by GO categories (Figure <a href="/pmc/articles/PMC6547448/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure4).</span></span><span>4</span></a>). We further selected four clusters (indicated by black frames in Figure <a href="/pmc/articles/PMC6547448/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure4)</span></span><span>4</span></a>) that are both enriched for genes of central metabolic processes and have large transcriptional changes across the different metabolic conditions for further studies of how TFs are affecting gene regulation in these clusters through multiple linear regressions. While the introduction of splines was highly stable for linear regressions over all metabolic genes, we found the process of model building with MARS using splines to be less stable in smaller groups of genes (mean cluster size with 16 clusters is 55 genes). For the multiple linear regressions in the clusters, we retained TF selection (by variable selection in the MARS algorithm) to define the most important TFs, but without introduction of splines.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC6547448/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140438874844752"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6547448_gkz253fig4.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkz253fig4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6547448/bin/gkz253fig4.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140438874844752"><a target="object" rel="noopener" href="/pmc/articles/PMC6547448/figure/F4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC6547448/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4.</a></div><!--caption a7--><div class="caption"><p>Clustering genes by their relative change in expression (sum of squares normalization) over the four experimental conditions gives enrichment of functional groups of genes. For clusters which have one or several significantly (FDR-adj <em>P</em> &#x0003c; 0.01) enriched GO terms, the top GO term is indicated with p.adj-value. Clusters containing central metabolic processes selected for further analysis with linear regressions in Figure <a href="/pmc/articles/PMC6547448/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure5</span></span><span>5</span></a> are indicated by a black frame.</p></div></div></div><p>Using this framework of multiple linear regression, predictions of transcriptional regulation on the clustered genes gives an improvement in predictive power compared to predictions of all metabolic genes (Figure <a href="/pmc/articles/PMC6547448/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5E</span></span><span>5E</span></a>&#x02013;<a href="/pmc/articles/PMC6547448/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>H</span></a>, R2: 0.57&#x02013;0.68). To compare the importance of different TFs for the predictions of transcript levels in the groups over different conditions, we calculate the &#x02018;TF importance&#x02019; by multiplying R2 of the multiple linear regression predictions with the relative contribution of the TF in the linear regression (0&#x02013;1, calculated by model construction algorithm) and also a coefficient for activation or repression (+1 or &#x02013;1, respectively). Some TFs were found to regulate a certain process over several conditions, such as Hap1 for Cluster 4, enriched for ergosterol biosynthesis genes (Figure <a href="/pmc/articles/PMC6547448/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5A),</span></span><span>5A</span></a>), but Cluster 4 is generally an example of a cluster with relatively large changes in importance of different TFs for gene regulation in different conditions. To get information about the complete set of TFs regulating these clusters of genes, we also included collinear TFs that were not initially included in the variable selection, but could replace a significantly correlated TF (illustrated by a red link under the TF&#x02019;s names in the heatmaps of Figure <a href="/pmc/articles/PMC6547448/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure5).</span></span><span>5</span></a>). For Cluster 4, Oaf1 was not selected during TF selection for this cluster and was thus not used in the predictions illustrated in the prediction plot of Figure <a href="/pmc/articles/PMC6547448/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure5E,</span></span><span>5E</span></a>, but was included in the heatmap because it was correlated to the Hap1 binding and when excluding Hap1 from the TF selection, Oaf1 was included. Because the contribution of each TF is linear in these regressions, the heatmaps give a complete view of how each gene is predicted to be regulated by different TFs. For Cluster 4 in fermentative glucose metabolism, the main contributors to ergosterol genes (ERG27, ERG26, ERG11, ERG25, ERG3) are predicted to be Ert1, Hap1 and Oaf1 (Figure <a href="/pmc/articles/PMC6547448/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5E</span></span><span>5E</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC6547448/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140438878903008"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6547448_gkz253fig5.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkz253fig5.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6547448/bin/gkz253fig5.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140438878903008"><a target="object" rel="noopener" href="/pmc/articles/PMC6547448/figure/F5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC6547448/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5.</a></div><!--caption a7--><div class="caption"><p>Clustering genes by relative expression gives strong predictive models of the clustered genes. (<strong>A</strong>&#x02013;<strong>D</strong>) All significant (<em>P</em>.adj &#x0003c; 0.05) GO terms for the clustered genes and the relative importance of the TFs selected to give the strongest predictions of transcript levels for the genes in the clusters in different conditions. Linear regressions (without splines) are used and importance is calculated by R2 (of regression with selected TFs) *relative importance of each TF (0 to 1) *sign of coefficient (+1 is activation, &#x02013;1 is repression). (<strong>E</strong>&#x02013;<strong>H</strong>) Prediction plots showing the predicted transcript levels compared to the real transcript levels from using the selected TFs (written in subtitle of plots). R2 of predicted transcript levels compared to real transcript level is shown in red text. Heatmaps demonstrate the real transcript levels as well as binding signal of each TF normalized column-wise (<em>Z</em>-score). TFs linked by a red line under the heatmap have significant collinearity over the cluster genes and were demonstrated to be able replace the other(s) in the variable selection, thus having overlapping functions in regulation of genes in a given cluster.</p></div></div></div><p class="p p-last">Cluster 15 is highly enriched for glycolytic processes and across conditions we see that the TFs predicted to be most important in several conditions are the well-known glycolytic regulators Gcr1, Gcr2 and Tye7 (Figure <a href="/pmc/articles/PMC6547448/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5B).</span></span><span>5B</span></a>). In respiratory glucose metabolism, Gcr1, Hap1, Ert1 and Rtg1 are included by variable selection and together these TFs are able to explain 66% of the variation in the cluster (Figure <a href="/pmc/articles/PMC6547448/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5F).</span></span><span>5F</span></a>). Both Gcr2 and Tye7 were found to be collinear and able to replace Gcr1 if it was excluded and these three TFs together are predicted to be regulating the glycolytic genes TPI1, CDC19, TDH2, ENO2, PGK1, PGI1, FBA1, TDH3, GPM1, PFK2, TDH1 and PFK1 (Figure <a href="/pmc/articles/PMC6547448/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5F).</span></span><span>5F</span></a>). We next focused on Cluster 7, containing genes with relatively higher expression in the two respiratory conditions (seen in Figure <a href="/pmc/articles/PMC6547448/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure4)</span></span><span>4</span></a>) and most strongly enriched for genes of mitochondrial ATP biosynthesis (Figure <a href="/pmc/articles/PMC6547448/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5C).</span></span><span>5C</span></a>). The most important TF for predictions in this cluster for both the conditions with mostly respiratory metabolism is the well-known mitochondrial regulator Hap4 (Figure <a href="/pmc/articles/PMC6547448/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5C).</span></span><span>5C</span></a>). During gluconeogenic respiration, Hap4 and Hap1 are predicted to both contribute to the regulation of most electron transport chain genes such as SHD1, NDI1, COX5A, COX6, QCR6, RIP1, QCR7 and ATP14 (Figure <a href="/pmc/articles/PMC6547448/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5G).</span></span><span>5G</span></a>). Several subunits of the ATP synthase (ATP17, ATP1&#x000a0;and ATP16) as well as certain TCA cycle enzymes (KGD1, KGD2) are predicted to be regulated by Hap4 without Hap1. Oaf1 also contributes to regulation of various genes of the cluster to a lesser extent, in some cases alone, or together with Hap1 and/or Hap4. We also demonstrate the same analysis for Cluster 16, most enriched for containing genes of fatty acid beta-oxidation (Figure <a href="/pmc/articles/PMC6547448/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5D).</span></span><span>5D</span></a>). While most of the predicted effects of TF binding on transcriptional regulation explored thus far has been activation, from the importance of TFs in the different conditions of Cluster 16, three TFs showed negative correlation to transcriptional changes during aerobic fermentation (Figure <a href="/pmc/articles/PMC6547448/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5D).</span></span><span>5D</span></a>). The TF selection used three TFs &#x02013; Gcn4, Hap1 and Gcr1 to predict 60% of the variation in the cluster during aerobic fermentation (Figure <a href="/pmc/articles/PMC6547448/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5H).</span></span><span>5H</span></a>). Ert1 and Sut1 were further included in the analysis because they were found to be collinear and able to replace Gcn4 in TF selection. Exploring the contributions of the TFs to expression levels of beta-oxidation genes in the heatmap supports an inverse relationship between Sut1-Gcn4-Ert1 binding an expression levels of several beta-oxidation genes, most notably for PXA1, DCI1, CTA1, CLD1, PXA2 and IDP3 (Figure <a href="/pmc/articles/PMC6547448/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5H</span></span><span>5H</span></a>).</p></div><div id="SEC3-6" class="sec sec-last"><h3 id="SEC3-6title">The influence of TFs on gene regulation from different regions of the promoter changes between metabolic conditions</h3><p class="p p-first">In the analysis shown in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S2C</a> we noticed that there were apparent differences in importance of different regions of the promoter between the conditions, most notably a predicted shift towards more consequential TF binding downstream of the TSS during aerobic fermentation. We reasoned that this could be because TFs with more consequential binding downstream of the TSS could be more important at this condition, or it could be because TFs shift their importance from one region of the promoter to another between conditions. To distinguish these two possibilities we performed simple linear regressions using the binding signal for each TF individually in 75 bp regions of the promoter to look for potential changes in TF importance in different regions of the promoter between the metabolic conditions. Comparing the resulting profiles of the explanatory power of TF binding in the experimental conditions revealed a distinctly different profile during aerobic fermentation compared to the other three metabolic conditions (Figure <a href="/pmc/articles/PMC6547448/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6A</span></span><span>6A</span></a> compared to Figure <a href="/pmc/articles/PMC6547448/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure6B</span></span><span>6B</span></a>&#x02013;<a href="/pmc/articles/PMC6547448/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>D</span></a>). Importantly, the differences during aerobic fermentation do not appear to be driven by one or a few TFs that regulate from downstream of the TSS with stronger importance during this condition, but rather a shift of where in the promoter several TFs are most consequential to regulation (Compare Gcr1, Ino2, Ino4, Stb5, Cbf1 in Figure <a href="/pmc/articles/PMC6547448/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure6A</span></span><span>6A</span></a> to B&#x02013;D).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F6" co-legend-rid="lgnd_F6"><a href="/pmc/articles/PMC6547448/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140438873632448"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6547448_gkz253fig6.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gkz253fig6.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6547448/bin/gkz253fig6.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140438873632448"><a target="object" rel="noopener" href="/pmc/articles/PMC6547448/figure/F6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F6"><div><a class="figpopup" href="/pmc/articles/PMC6547448/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6">Figure 6.</a></div><!--caption a7--><div class="caption"><p>Comparing TF importance in different segments of the promoter shows a distinct pattern of TF importance during aerobic fermentation as compared to the other conditions. (<strong>A</strong>&#x02013;<strong>D</strong>) Linear regressions, regressing the TF binding signal of each TF individually, in the indicated segments of the promoters, against the expression levels. Lines are labeled by the TF name near the highest absolute y-axis value. (<strong>E</strong>) Two groups of genes where binding of Sut1, Ert1 and Gcn4 in &#x02212;450 to &#x02212;250 relative to the TSS is correlated to each other and oppositely correlated to transcript levels. (<strong>F</strong>) Summary of the strongest predicted links between TF function and selected metabolic processes. Based on data presented in Figures <a href="/pmc/articles/PMC6547448/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">Figures4</span></span><span>4</span></a> and&#x000a0;<a href="/pmc/articles/PMC6547448/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>5</span></a>, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S4E&#x02013;F</a> and Figure <a href="/pmc/articles/PMC6547448/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure6A</span></span><span>6A</span></a>&#x02013;<a href="/pmc/articles/PMC6547448/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>E</span></a>.</p></div></div></div><p class="p p-last">Another striking observation from the importance of TFs during aerobic fermentation was a negative correlation between transcriptional changes and binding of three TFs from an overlapping region upstream of the TSS; Sut1, Gcn4 and Ert1 (Figure <a href="/pmc/articles/PMC6547448/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6A).</span></span><span>6A</span></a>). These regressions were of all metabolic genes, suggesting that the negative correlation seen with these TFs on beta-oxidation genes, as seen in Figure <a href="/pmc/articles/PMC6547448/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure5H,</span></span><span>5H</span></a>, may be a more general phenomena of aerobic fermentation. To see if these three TFs are acting on the same genes or separately we compared the binding of the three TFs in the region 250&#x02013;450 upstream of the TSS together with expression levels over all metabolic genes to look for groups of genes where binding of Sut1, Gcn4 and Ert1 are correlated to each other and anti-correlated to expression levels (Figure <a href="/pmc/articles/PMC6547448/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6E).</span></span><span>6E</span></a>). In aerobic fermentation, we found two distinct groups of genes where all three TFs are generally correlated to each other, one with low binding of the three TFs and high expression levels and another group with relatively higher binding of all three TFs and relatively lower expression levels. Interestingly, the group of genes where the three TFs have the strongest negative correlation to transcript levels is slightly enriched for genes of translation processes (Figure <a href="/pmc/articles/PMC6547448/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6E,</span></span><span>6E</span></a>, Group 1), genes that are likely closely controlled due to the nitrogen limitation of these cultures. We also looked for similar relationships between binding of Sut1-Gcn4-Ert1 and transcript levels for the other experimental conditions (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S4B&#x02013;D</a>), but we found no coordinated changes in the other conditions, suggesting this phenomena is specific to aerobic fermentation. We summarize some of our main findings from Figures <a href="/pmc/articles/PMC6547448/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">Figures5</span></span><span>5</span></a> and&#x000a0;<a href="/pmc/articles/PMC6547448/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>6A</span></a>&#x02013;<a href="/pmc/articles/PMC6547448/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>E</span></a> regarding how TFs regulate difference metabolic processes in different conditions in Figure <a href="/pmc/articles/PMC6547448/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure6F</span></span><span>6F</span></a>.</p></div></div><div id="SEC4" class="tsec sec"><h2 class="head no_bottom_margin" id="SEC4title" style="text-transform: uppercase;">DISCUSSION</h2><p class="p p-first">The latest count of known and putative yeast TFs is 264 (<a href="#B33" rid="B33" class=" bibr popnode">33</a>). With coverage of 21 TFs we only see part of the system of gene regulation by TFs, but by selecting the TFs most enriched to binding metabolic genes and building predictive models of metabolic gene regulation we achieved good TF coverage per gene. Through various types of analysis we discovered surprisingly large changes between the studied metabolic conditions, first encountered when comparing sets of target genes for the TFs between conditions shown in Figure <a href="/pmc/articles/PMC6547448/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure1A.</span></span><span>1A</span></a>. To explain the changing sets of gene targets between conditions, we hypothesized that the TFs could have changes in preference of DNA motifs between different conditions. We cannot exclude changes in motif preference in certain cases such as Rtg1 and Gcn4 in aerobic fermentation (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S2a</a>), but in general the changes in motif preference between conditions are small and we think this is not a major determinant of changes in which genes are being targeted between conditions. We think the most likely alternative explanation is that there are changes in nucleosome occupancy or histone modifications that allow more or less binding to different sets of genes over the different conditions studied and that binding site preference is only partly driven by the recognized DNA motif. This idea is generally supported by the complex two-way interactions that have been suggested between TF binding and histones in eukaryotic gene regulation (<a href="#B8" rid="B8" class=" bibr popnode">8</a>).</p><p>Based on the success of multiple linear regressions as predictive models of gene regulation from TF binding data, we propose that a large portion of transcriptional regulation by TFs in yeast is achieved from a linear effect of TF binding on transcriptional outcome. Our predictions also suggest that a large amount of the contributions to gene regulation from several metabolic TFs are additive. However, by allowing multiplication of TF binding signal, we do detect cases that indicate more complex contributions from pairs of TFs to gene regulation than a simple addition of the two TFs binding signal can capture. Most noteworthy is the relationship between Ino2 and Ino4, where the additive contribution from both TFs seem to saturate at a certain strength of binding (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S3c</a>). The Ino2&#x02013;Ino4 relationship is best known for being required for phospholipid biosynthesis and Ino2 is described as most important for transcriptional activation of the targeted genes, but Ino2 also depends on Ino4 for translocation into the nucleus (<a href="#B34" rid="B34" class=" bibr popnode">34</a>). There is additional complexity in the regulation of this complex by involvement of the Opi1 repressor, which can bind Ino2 to inhibit activation by the complex (<a href="#B35" rid="B35" class=" bibr popnode">35</a>). We cannot conclude on why we see a saturation effect in the transcriptional activation due to increased binding of both Ino2 and Ino4, but it could be due to TF&#x02013;TF competition of the complex with other components of the transcriptional machinery, or other nonlinear relationships between binding of the TFs and transcriptional outcomes. In aerobic fermentation we detect several additional TF pairs for which there may be more complex relationships than simple addition of the contributing signal. Of these, both Ert1 and Gcn4 have relationships to Ino2 and Ino4 binding where a multiplication of the binding signal clearly improves predictive power (Figure <a href="/pmc/articles/PMC6547448/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3E).</span></span><span>3E</span></a>). It is striking that Ert1 and Gcn4 are two of only a few TFs that show this kind of complexity seen together with the negative correlation seen between transcript levels and binding of Sut1, Gcn4 and Ert1 in aerobic fermentation, but we do not know if these observations are related.</p><p>The main advantage of using linear regressions, as compared to more complex machine learning analysis, is that each TF&#x02019;s contribution in the predictions can be fully described. We highlight this feature of our analysis in Figure <a href="/pmc/articles/PMC6547448/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure5,</span></span><span>5</span></a>, where the groups of genes are biologically linked and small enough to illustrate the amount of binding from each selected TF in the individual genes of the clusters. In this analysis we confirm several previously demonstrated regulators of important cellular processes such as Hap1 activating ergosterol biosynthesis (<a href="#B36" rid="B36" class=" bibr popnode">36</a>), Gcr1-Gcr2-Tye7 activating glycolytic genes (<a href="#B37" rid="B37" class=" bibr popnode">37</a>) and Hap4 activating respiratory processes (<a href="#B38" rid="B38" class=" bibr popnode">38</a>). We also propose previously unknown contributions from other TFs to these processes such as Ert1 activating ergosterol biosynthesis at anaerobic conditions (Figure <a href="/pmc/articles/PMC6547448/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5E</span></span><span>5E</span></a>).</p><p>The minimal media used in all chemostats of this study are without added amino acids, meaning that the feedback controls activating amino acid biogenesis should generally be activated. Gcn4 is best known as an activator of amino acid biosynthesis, but also demonstrated in several studies to be a repressor of ribosomal protein expression by an unknown mechanism (<a href="#B39" rid="B39" class=" bibr popnode">39</a>,<a href="#B40" rid="B40" class=" bibr popnode">40</a>). The mechanistic details on how Gcn4 can activate some genes and repress others remain mostly unknown, but the repression has been linked to functional interactions with the repressive TF Rap1 and histone acetyltransferase Esa1 (<a href="#B41" rid="B41" class=" bibr popnode">41</a>). A recent study of the yeast strain <em>Y. lipolytica</em>, describing increased lipid accumulation of this strain during nitrogen limitation, found that beta-oxidation genes were down-regulated in this condition (<a href="#B42" rid="B42" class=" bibr popnode">42</a>). In that study, they also noted that genes with the Gcn4 motif in their promoter tended to be down-regulated during nitrogen limitation, an observation that they could not explain, but is in line with the negative correlation we observe between Gcn4 binding and transcript levels for beta oxidation genes during nitrogen limitation (Figure <a href="/pmc/articles/PMC6547448/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5H).</span></span><span>5H</span></a>). We propose that a similar mechanism as was observed in <em>Y. lipolytica</em> exists in <em>S. cerevisiae</em> and further extend it to affect a larger set of genes (Figure <a href="/pmc/articles/PMC6547448/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6E)</span></span><span>6E</span></a>) and to also involve Sut1 and Ert1. Sut1 is previously described to interact with the co-repressor complex Cyc8-Tup1 (<a href="#B43" rid="B43" class=" bibr popnode">43</a>) while Ert1 is previously described to function as both an activator and repressor with the major described functional role being regulating genes driving the diauxic shift (<a href="#B44" rid="B44" class=" bibr popnode">44</a>). Our data does not give any further insight into the mechanism of the Sut1-Ert1-Gcn4 repression, but based on the generally reduced number of peaks (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S1C</a>) and changes in importance of different regions of the promoter (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S2C</a>, Figure <a href="/pmc/articles/PMC6547448/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure6A</span></span><span>6A</span></a>&#x02013;<a href="/pmc/articles/PMC6547448/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>D</span></a>) during aerobic fermentation, we speculate that Sut1-Ert1-Gcn4 binding is correlated to changes to the nucleosomes on each side of the TSS in certain genes, for example histone modifications or changes in nucleosome occupancy. If the changes in binding of these three TFs is not driven by changes in their preference for motifs, but instead by nucleosomes, it is also not necessarily the case that the three TFs are directly repressing &#x02013; the correlation between TF binding and transcriptional regulation could potentially be a secondary effect of nucleosome changes that are also causing transcriptional changes.</p><p class="p p-last">In conclusion, our experimental design using chemostats to capture stable states of metabolism reveals large changes in functional roles of different TFs between metabolic states. The previously demonstrated difficulties in defining the regulatory targets of eukaryal TFs through transcriptomics after TF deletion could be partly explained by this highly dynamic nature of eukaryal TF function. If the deletion of the TF changes cellular conditions enough to shift the regulatory roles of a range of one or several other TFs, the following secondary transcriptional changes could be a source of significant changes in genes not targeted directly by the deleted TF. Our framework of using multiple linear regressions for full transparency of TF contributions to transcriptional regulation without relying on TF deletion will be equally applicable for future larger-scale studies as binding data for more TFs with condition-matched transcriptomics accumulate to gradually build a system-level understanding of eukaryotic transcriptional regulation.</p></div><div id="SEC5" class="tsec sec"><h2 class="head no_bottom_margin" id="SEC5title" style="text-transform: uppercase;">DATA AVAILABILITY</h2><p class="p p-first-last">ChIP-exo raw sequencing data (.fastq) can be retrieved form Arrayexpress accession E-MTAB-6673: <a href="https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6673/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6673/</a>. A processed format of the ChIP-exo data, summarized as TF reads per gene promoter is included in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Data 4</a>. RNA sequencing raw sequencing data (.fastq) can be retrieved from Arrayexpress accession E-MTAB-6657: <a href="https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6657/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6657/</a>. The number of reads annotated to each gene are found in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Data 3</a> and our processing of read counts to get to FPKM values that are used for all relevant analysis of this study can be reproduced through the R scripts included in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Data 6</a>.</p></div><div id="sec-a.j.f" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="sec-a.j.ftitle" style="text-transform: uppercase;">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="sup1"><h4>gkz253_Supplemental_Files</h4><div class="sup-box half_rhythm" id="media-a.j.f.b.b"><a href="/pmc/articles/PMC6547448/bin/gkz253_supplemental_files.zip" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(3.7M, zip)</sup></div></div></div><div id="ACK1" class="tsec sec"><h2 class="head no_bottom_margin" id="ACK1title" style="text-transform: uppercase;">ACKNOWLEDGEMENTS</h2><div class="sec"><p>
<em>Authors&#x02019; contributions:</em> Funding acquisition, J.N.; conceptualization, P.H. and J.N.; experiments, P.H., D.B., C.S.B and G.L.; analysis, P.H., D.B. and C.S.B; writing &#x02013; original draft, P.H.; writing &#x02013; review &#x00026; editing, D.B, C.S.B, G.L. and J.N.</p></div></div><div id="SEC6" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="SEC6title" style="text-transform: uppercase;">SUPPLEMENTARY DATA</h2><!--/article/back/sec/--><p class="p p-first-last">
<a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkz253#supplementary-data" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Data</a> are available at NAR Online.</p></div><div id="SEC7" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="SEC7title" style="text-transform: uppercase;">FUNDING</h2><!--/article/back/sec/--><p class="p p-first">European Union's Horizon 2020 research and innovation programme [Marie Sk&#x00142;odowska-Curie grant agreement no. 722287]; Knut and Alice Wallenberg Foundation and the Novo Nordisk Foundation [NNF10CC1016517]. Funding for open access charge:&#x000a0;Internal.</p><p class="p p-last">
<em>Conflict of interest statement</em>. None declared.</p></div><div id="REF1" class="tsec sec"><h2 class="head no_bottom_margin" id="REF1title" style="text-transform: uppercase;">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1">1. <span class="mixed-citation">
Le P.P., Friedman J.R., Schug J., Brestelli J.E., Parker J.B., Bochkis I.M., Kaestner K.H.. 
<span class="ref-title">Glucocorticoid receptor-dependent gene regulatory networks</span>. <span class="ref-journal">PLoS Genet.</span> 2005; <span class="ref-vol">1</span>:0159&#x02013;0170. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Glucocorticoid+receptor-dependent+gene+regulatory+networks&amp;author=P.P.+Le&amp;author=J.R.+Friedman&amp;author=J.+Schug&amp;author=J.E.+Brestelli&amp;author=J.B.+Parker&amp;volume=1&amp;publication_year=2005&amp;pages=0159-0170&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B2">2. <span class="mixed-citation">
Fan J.-B., Benner C., Hutt K.R., Garcia-Bassets I., Fu X.-D., Rosenfeld M.G., Bibikova M., Jin M., Kwon Y.-S., Ye Z. et al... 
<span class="ref-title">Sensitive ChIP-DSL technology reveals an extensive estrogen receptor -binding program on human gene promoters</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span> 2007; <span class="ref-vol">104</span>:4852&#x02013;4857. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1821125/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17360330" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Sensitive+ChIP-DSL+technology+reveals+an+extensive+estrogen+receptor+-binding+program+on+human+gene+promoters&amp;author=J.-B.+Fan&amp;author=C.+Benner&amp;author=K.R.+Hutt&amp;author=I.+Garcia-Bassets&amp;author=X.-D.+Fu&amp;volume=104&amp;publication_year=2007&amp;pages=4852-4857&amp;pmid=17360330&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B3">3. <span class="mixed-citation">
Hu Z., Killion P.J., Iyer V.R.. 
<span class="ref-title">Genetic reconstruction of a functional transcriptional regulatory network</span>. <span class="ref-journal">Nat. Genet.</span> 2007; <span class="ref-vol">39</span>:683&#x02013;687. [<a href="https://pubmed.ncbi.nlm.nih.gov/17417638" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genetic+reconstruction+of+a+functional+transcriptional+regulatory+network&amp;author=Z.+Hu&amp;author=P.J.+Killion&amp;author=V.R.+Iyer&amp;volume=39&amp;publication_year=2007&amp;pages=683-687&amp;pmid=17417638&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B4">4. <span class="mixed-citation">
Harbison C.T., Gordon D.B., Lee T.I., Rinaldi N.J., Macisaac K.D., Danford T.W., Hannett N.M., Tagne J.B., Reynolds D.B., Yoo J. et al... 
<span class="ref-title">Transcriptional regulatory code of a eukaryotic genome</span>. <span class="ref-journal">Nature</span>. 2004; <span class="ref-vol">431</span>:1&#x02013;5. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3006441/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15343339" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Transcriptional+regulatory+code+of+a+eukaryotic+genome&amp;author=C.T.+Harbison&amp;author=D.B.+Gordon&amp;author=T.I.+Lee&amp;author=N.J.+Rinaldi&amp;author=K.D.+Macisaac&amp;volume=431&amp;publication_year=2004&amp;pages=1-5&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B5">5. <span class="mixed-citation">
Gitter A., Siegfried Z., Klutstein M., Fornes O., Oliva B., Simon I., Bar-Joseph Z.. 
<span class="ref-title">Backup in gene regulatory networks explains differences between binding and knockout results</span>. <span class="ref-journal">Mol. Syst. Biol.</span> 2009; <span class="ref-vol">5</span>:276. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2710864/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19536199" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Syst.+Biol.&amp;title=Backup+in+gene+regulatory+networks+explains+differences+between+binding+and+knockout+results&amp;author=A.+Gitter&amp;author=Z.+Siegfried&amp;author=M.+Klutstein&amp;author=O.+Fornes&amp;author=B.+Oliva&amp;volume=5&amp;publication_year=2009&amp;pages=276&amp;pmid=19536199&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B6">6. <span class="mixed-citation">
Fang X., Sastry A., Mih N., Kim D., Tan J., Yurkovich J.T., Lloyd C.J., Gao Y., Yang L., Palsson B.O.. 
<span class="ref-title">Global transcriptional regulatory network for <em>Escherichia coli</em> robustly connects gene expression to transcription factor activities</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span> 2017; <span class="ref-vol">114</span>:10286&#x02013;10291. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5617254/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28874552" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Global+transcriptional+regulatory+network+for+Escherichia+coli+robustly+connects+gene+expression+to+transcription+factor+activities&amp;author=X.+Fang&amp;author=A.+Sastry&amp;author=N.+Mih&amp;author=D.+Kim&amp;author=J.+Tan&amp;volume=114&amp;publication_year=2017&amp;pages=10286-10291&amp;pmid=28874552&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B7">7. <span class="mixed-citation">
ENCODE consortium
<span class="ref-title">An integrated encyclopedia of DNA elements in the human genome</span>. <span class="ref-journal">Nature</span>. 2012; <span class="ref-vol">489</span>:57&#x02013;74. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3439153/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22955616" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+integrated+encyclopedia+of+DNA+elements+in+the+human+genome&amp;volume=489&amp;publication_year=2012&amp;pages=57-74&amp;pmid=22955616&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B8">8. <span class="mixed-citation">
Cheng C., Alexander R., Min R., Leng J., Yip K.Y., Rozowsky J., Yan K., Dong X., Djebali S., Ruan Y. et al... 
<span class="ref-title">Understanding transcriptional regulation by integrative analysis of transcription factor binding data</span>. <span class="ref-journal">Genome Res.</span> 2012; doi:10.1101/gr.136838.111. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3431483/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22955978" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=Understanding+transcriptional+regulation+by+integrative+analysis+of+transcription+factor+binding+data&amp;author=C.+Cheng&amp;author=R.+Alexander&amp;author=R.+Min&amp;author=J.+Leng&amp;author=K.Y.+Yip&amp;publication_year=2012&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B9">9. <span class="mixed-citation">
Ouyang Z., Zhou Q., Wong W.H.. 
<span class="ref-title">ChIP-Seq of transcription factors predicts absolute and differential gene expression in embryonic stem cells</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span> 2009; <span class="ref-vol">106</span>:21521&#x02013;21526. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2789751/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19995984" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=ChIP-Seq+of+transcription+factors+predicts+absolute+and+differential+gene+expression+in+embryonic+stem+cells&amp;author=Z.+Ouyang&amp;author=Q.+Zhou&amp;author=W.H.+Wong&amp;volume=106&amp;publication_year=2009&amp;pages=21521-21526&amp;pmid=19995984&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B10">10. <span class="mixed-citation">
Zhu C., Byers K.J.R.P., McCord R.P., Shi Z., Berger M.F., Newburger D.E., Saulrieta K., Smith Z., Shah M. V., Radhakrishnan M. et al... 
<span class="ref-title">High-resolution DNA-binding specificity analysis of yeast transcription factors</span>. <span class="ref-journal">Genome Res.</span> 2009; <span class="ref-vol">19</span>:556&#x02013;566. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2665775/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19158363" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=High-resolution+DNA-binding+specificity+analysis+of+yeast+transcription+factors&amp;author=C.+Zhu&amp;author=K.J.R.P.+Byers&amp;author=R.P.+McCord&amp;author=Z.+Shi&amp;author=M.F.+Berger&amp;volume=19&amp;publication_year=2009&amp;pages=556-566&amp;pmid=19158363&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B11">11. <span class="mixed-citation">
Badis G., Pena-Castillo L., Ansari A.Z., van&#x000a0;Bakel H., Hughes T.R., Hasinoff M.J., Terterov D., Li&#x000a0;Yeo A., Tillo D., Nislow C. et al... 
<span class="ref-title">A library of yeast transcription factor motifs reveals a widespread function for Rsc3 in targeting nucleosome exclusion at promoters</span>. <span class="ref-journal">Mol. Cell</span>. 2008; <span class="ref-vol">32</span>:878&#x02013;887. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2743730/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19111667" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=A+library+of+yeast+transcription+factor+motifs+reveals+a+widespread+function+for+Rsc3+in+targeting+nucleosome+exclusion+at+promoters&amp;author=G.+Badis&amp;author=L.+Pena-Castillo&amp;author=A.Z.+Ansari&amp;author=H.+van&#x000a0;Bakel&amp;author=T.R.+Hughes&amp;volume=32&amp;publication_year=2008&amp;pages=878-887&amp;pmid=19111667&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B12">12. <span class="mixed-citation">
Hughes T.R., de&#x000a0;Boer C.G.. 
<span class="ref-title">Mapping yeast transcriptional networks</span>. <span class="ref-journal">Genetics</span>. 2013; <span class="ref-vol">195</span>:9&#x02013;36. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3761317/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24018767" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Mapping+yeast+transcriptional+networks&amp;author=T.R.+Hughes&amp;author=C.G.+de&#x000a0;Boer&amp;volume=195&amp;publication_year=2013&amp;pages=9-36&amp;pmid=24018767&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B13">13. <span class="mixed-citation">
Bergenholm D., Liu G., Hansson D., Nielsen J.. 
<span class="ref-title">Construction of mini-chemostats for high-throughput strain characterization</span>. <span class="ref-journal">Biotechnol. Bioeng.</span> 2019; <span class="ref-vol">116</span>:1029&#x02013;1038. [<a href="https://pubmed.ncbi.nlm.nih.gov/30659597" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnol.+Bioeng.&amp;title=Construction+of+mini-chemostats+for+high-throughput+strain+characterization&amp;author=D.+Bergenholm&amp;author=G.+Liu&amp;author=D.+Hansson&amp;author=J.+Nielsen&amp;volume=116&amp;publication_year=2019&amp;pages=1029-1038&amp;pmid=30659597&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B14">14. <span class="mixed-citation">
Rhee H.S., Pugh B.F.. 
<span class="ref-title">ChiP-exo method for identifying genomic location of DNA-binding proteins with near-single-nucleotide accuracy</span>. <span class="ref-journal">Curr. Protoc. Mol. Biol.</span> 2012; <span class="ref-vol">100</span>:21.24.1&#x02013;21.24.14. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3813302/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23026909" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Protoc.+Mol.+Biol.&amp;title=ChiP-exo+method+for+identifying+genomic+location+of+DNA-binding+proteins+with+near-single-nucleotide+accuracy&amp;author=H.S.+Rhee&amp;author=B.F.+Pugh&amp;volume=100&amp;publication_year=2012&amp;pages=21.24.1-21.24.14&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B15">15. <span class="mixed-citation">
Liu G., Bergenholm D., Nielsen J.. 
<span class="ref-title">Genome-Wide mapping of binding sites reveals multiple biological functions of the transcription factor Cst6p in saccharomyces cerevisiae</span>. <span class="ref-journal">MBio</span>. 2016; <span class="ref-vol">7</span>:1&#x02013;10. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4959655/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27143390" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=MBio&amp;title=Genome-Wide+mapping+of+binding+sites+reveals+multiple+biological+functions+of+the+transcription+factor+Cst6p+in+saccharomyces+cerevisiae&amp;author=G.+Liu&amp;author=D.+Bergenholm&amp;author=J.+Nielsen&amp;volume=7&amp;publication_year=2016&amp;pages=1-10&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B16">16. <span class="mixed-citation">
Guo Y., Mahony S., Gifford D.K.. 
<span class="ref-title">High resolution genome wide binding event finding and motif discovery reveals transcription factor spatial binding constraints</span>. <span class="ref-journal">PLoS Comput. Biol.</span> 2012; <span class="ref-vol">8</span>:e1002638. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3415389/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22912568" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Comput.+Biol.&amp;title=High+resolution+genome+wide+binding+event+finding+and+motif+discovery+reveals+transcription+factor+spatial+binding+constraints&amp;author=Y.+Guo&amp;author=S.+Mahony&amp;author=D.K.+Gifford&amp;volume=8&amp;publication_year=2012&amp;pages=e1002638&amp;pmid=22912568&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B17">17. <span class="mixed-citation">
B&#x000f6;rlin C.S., Siewers V., Nielsen J., Holland P., Bergenholm D., Cvetesic N., Lenhard B.. 
<span class="ref-title">Saccharomyces cerevisiae displays a stable transcription start site landscape in multiple conditions</span>. <span class="ref-journal">FEMS Yeast Res.</span> 2018; <span class="ref-vol">19</span>:1&#x02013;10. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7612425/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30590648" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEMS+Yeast+Res.&amp;title=Saccharomyces+cerevisiae+displays+a+stable+transcription+start+site+landscape+in+multiple+conditions&amp;author=C.S.+B&#x000f6;rlin&amp;author=V.+Siewers&amp;author=J.+Nielsen&amp;author=P.+Holland&amp;author=D.+Bergenholm&amp;volume=19&amp;publication_year=2018&amp;pages=1-10&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B18">18. <span class="mixed-citation">
Salazar A.N., van&#x000a0;den&#x000a0;Broek M., Daran J.-M.G., Gorter&#x000a0;de&#x000a0;Vries A.R., Wijsman M., Brouwers N., Brickwedde A., Abeel T., de&#x000a0;la&#x000a0;Torre&#x000a0;Cort&#x000e9;s P.. 
<span class="ref-title">Nanopore sequencing enables near-complete de novo assembly of Saccharomyces cerevisiae reference strain CEN.PK113-7D</span>. <span class="ref-journal">FEMS Yeast Res.</span> 2017; <span class="ref-vol">17</span>:doi:10.1093/femsyr/fox074. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5812507/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28961779" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEMS+Yeast+Res.&amp;title=Nanopore+sequencing+enables+near-complete+de+novo+assembly+of+Saccharomyces+cerevisiae+reference+strain+CEN.PK113-7D&amp;author=A.N.+Salazar&amp;author=M.+van&#x000a0;den&#x000a0;Broek&amp;author=J.-M.G.+Daran&amp;author=A.R.+Gorter&#x000a0;de&#x000a0;Vries&amp;author=M.+Wijsman&amp;volume=17&amp;publication_year=2017&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B19">19. <span class="mixed-citation">
Langmead B., Salzberg S.L.. 
<span class="ref-title">Fast gapped-read alignment with Bowtie 2</span>. <span class="ref-journal">Nat. Methods</span>. 2012; <span class="ref-vol">9</span>:357&#x02013;359. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3322381/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22388286" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=Fast+gapped-read+alignment+with+Bowtie+2&amp;author=B.+Langmead&amp;author=S.L.+Salzberg&amp;volume=9&amp;publication_year=2012&amp;pages=357-359&amp;pmid=22388286&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B20">20. <span class="mixed-citation">
Li H., Handsaker B., Wysoker A., Fennell T., Ruan J., Homer N., Marth G., Abecasis G., Durbin R. 1000 Genome Project Data Processing Subgroup . 
<span class="ref-title">The Sequence Alignment/Map format and SAMtools</span>. <span class="ref-journal">Bioinformatics</span>. 2009; <span class="ref-vol">25</span>:2078&#x02013;2079. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2723002/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19505943" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=The+Sequence+Alignment/Map+format+and+SAMtools&amp;author=H.+Li&amp;author=B.+Handsaker&amp;author=A.+Wysoker&amp;author=T.+Fennell&amp;author=J.+Ruan&amp;volume=25&amp;publication_year=2009&amp;pages=2078-2079&amp;pmid=19505943&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B21">21. <span class="mixed-citation">
Quinlan A.R., Hall I.M.. 
<span class="ref-title">BEDTools: A flexible suite of utilities for comparing genomic features</span>. <span class="ref-journal">Bioinformatics</span>. 2010; <span class="ref-vol">26</span>:841&#x02013;842. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2832824/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20110278" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=BEDTools:+A+flexible+suite+of+utilities+for+comparing+genomic+features&amp;author=A.R.+Quinlan&amp;author=I.M.+Hall&amp;volume=26&amp;publication_year=2010&amp;pages=841-842&amp;pmid=20110278&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B22">22. <span class="mixed-citation">
R Core team
<span class="ref-title">R: A language and environment for statistical computing</span>. <span class="ref-journal">R Foundation for Statistical Computing</span>. 2016; Vienna: <a href="https://www.R-project.org/">https://www.R-project.org/</a>. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=R+Foundation+for+Statistical+Computing&amp;publication_year=2016&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B23">23. <span class="mixed-citation">
Liao Y., Smyth G.K., Shi W.. 
<span class="ref-title">featureCounts: an efficient general purpose program for assigning sequence reads to genomic features</span>. <span class="ref-journal">Bioinformatics</span>. 2014; <span class="ref-vol">30</span>:923&#x02013;930. [<a href="https://pubmed.ncbi.nlm.nih.gov/24227677" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=featureCounts:+an+efficient+general+purpose+program+for+assigning+sequence+reads+to+genomic+features&amp;author=Y.+Liao&amp;author=G.K.+Smyth&amp;author=W.+Shi&amp;volume=30&amp;publication_year=2014&amp;pages=923-930&amp;pmid=24227677&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B24">24. <span class="mixed-citation">
Robinson M.D., McCarthy D.J., Smyth G.K.. 
<span class="ref-title">edgeR: a Bioconductor package for differential expression analysis of digital gene expression data</span>. <span class="ref-journal">Bioinformatics</span>. 2010; <span class="ref-vol">26</span>:139&#x02013;140. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2796818/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19910308" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=edgeR:+a+Bioconductor+package+for+differential+expression+analysis+of+digital+gene+expression+data&amp;author=M.D.+Robinson&amp;author=D.J.+McCarthy&amp;author=G.K.+Smyth&amp;volume=26&amp;publication_year=2010&amp;pages=139-140&amp;pmid=19910308&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B25">25. <span class="mixed-citation">
Milborrow S.
<span class="ref-journal">earth: Multivariate Adaptive Regression Splines</span>. 2017; <a href="https://www.cran.r-project.org/package=earth">https://CRAN.R-project.org/package=earth</a>. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=earth:+Multivariate+Adaptive+Regression+Splines&amp;author=S.+Milborrow&amp;publication_year=2017&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B26">26. <span class="mixed-citation">
Bergenholm D., Liu G., Holland P., Nielsen J.. 
<span class="ref-title">Reconstruction of a global transcriptional regulatory network for control of lipid metabolism in yeast by using chromatin immunoprecipitation with lambda exonuclease digestion</span>. <span class="ref-journal">mSystems</span>. 2018; <span class="ref-vol">3</span>:e00215-17. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6068829/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30073202" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=mSystems&amp;title=Reconstruction+of+a+global+transcriptional+regulatory+network+for+control+of+lipid+metabolism+in+yeast+by+using+chromatin+immunoprecipitation+with+lambda+exonuclease+digestion&amp;author=D.+Bergenholm&amp;author=G.+Liu&amp;author=P.+Holland&amp;author=J.+Nielsen&amp;volume=3&amp;publication_year=2018&amp;pages=e00215-17&amp;pmid=30073202&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B27">27. <span class="mixed-citation">
Ouyang L., Holland P., Lu H., Bergenholm D., Nielsen J.. 
<span class="ref-title">Integrated analysis of the yeast NADPH-regulator Stb5 reveals distinct differences in NADPH requirements and regulation in different states of yeast metabolism</span>. <span class="ref-journal">FEMS Yeast Res.</span> 2018; <span class="ref-vol">18</span>:91. [<a href="https://pubmed.ncbi.nlm.nih.gov/30107458" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEMS+Yeast+Res.&amp;title=Integrated+analysis+of+the+yeast+NADPH-regulator+Stb5+reveals+distinct+differences+in+NADPH+requirements+and+regulation+in+different+states+of+yeast+metabolism&amp;author=L.+Ouyang&amp;author=P.+Holland&amp;author=H.+Lu&amp;author=D.+Bergenholm&amp;author=J.+Nielsen&amp;volume=18&amp;publication_year=2018&amp;pages=91&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B28">28. <span class="mixed-citation">
MacIsaac K.D., Wang T., Gordon D.B., Gifford D.K., Stormo G.D., Fraenkel E.. 
<span class="ref-title">An improved map of conserved regulatory sites for Saccharomyces cerevisiae</span>. <span class="ref-journal">BMC Bioinformatics</span>. 2006; <span class="ref-vol">7</span>:113. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1435934/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16522208" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=An+improved+map+of+conserved+regulatory+sites+for+Saccharomyces+cerevisiae&amp;author=K.D.+MacIsaac&amp;author=T.+Wang&amp;author=D.B.+Gordon&amp;author=D.K.+Gifford&amp;author=G.D.+Stormo&amp;volume=7&amp;publication_year=2006&amp;pages=113&amp;pmid=16522208&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B29">29. <span class="mixed-citation">
Hashim Z., Mukai Y., Bamba T., Fukusaki E.. 
<span class="ref-title">Metabolic profiling of retrograde pathway transcription factors Rtg1 and Rtg3 knockout yeast</span>. <span class="ref-journal">Metabolites</span>. 2014; <span class="ref-vol">4</span>:580&#x02013;598. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4192681/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25007314" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Metabolites&amp;title=Metabolic+profiling+of+retrograde+pathway+transcription+factors+Rtg1+and+Rtg3+knockout+yeast&amp;author=Z.+Hashim&amp;author=Y.+Mukai&amp;author=T.+Bamba&amp;author=E.+Fukusaki&amp;volume=4&amp;publication_year=2014&amp;pages=580-598&amp;pmid=25007314&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B30">30. <span class="mixed-citation">
Crespo L., Powers T., Fowler B., Hall M.N.. 
<span class="ref-title">The TOR-controlled transcription activators GLN3, RTG1, and RTG3 are regulated in response to intracellular levels of glutamine</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span> 2002; <span class="ref-vol">99</span>:6784&#x02013;6789. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC124480/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11997479" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=The+TOR-controlled+transcription+activators+GLN3,+RTG1,+and+RTG3+are+regulated+in+response+to+intracellular+levels+of+glutamine&amp;author=L.+Crespo&amp;author=T.+Powers&amp;author=B.+Fowler&amp;author=M.N.+Hall&amp;volume=99&amp;publication_year=2002&amp;pages=6784-6789&amp;pmid=11997479&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B31">31. <span class="mixed-citation">
S&#x000e1;nchez B., Li F., Lu H., Kerkhoven E., Nielsen J.. 
<span class="ref-title">Yeast-GEM: yeast 7.6.0</span>. 2016; doi:10.5281/zenodo.1495468.</span></div><div class="ref-cit-blk half_rhythm" id="B32">32. <span class="mixed-citation">
Friedman J.H.
<span class="ref-title">Multivariate adaptive regression splines</span>. <span class="ref-journal">Ann. Stat.</span> 2007; <span class="ref-vol">19</span>:123&#x02013;141. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Stat.&amp;title=Multivariate+adaptive+regression+splines&amp;author=J.H.+Friedman&amp;volume=19&amp;publication_year=2007&amp;pages=123-141&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B33">33. <span class="mixed-citation">
de&#x000a0;Boer C.G., Hughes T.R.. 
<span class="ref-title">YeTFaSCo: a database of evaluated yeast transcription factor sequence specificities</span>. <span class="ref-journal">Nucleic Acids Res.</span> 2012; <span class="ref-vol">40</span>:D169&#x02013;D179. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3245003/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22102575" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=YeTFaSCo:+a+database+of+evaluated+yeast+transcription+factor+sequence+specificities&amp;author=C.G.+de&#x000a0;Boer&amp;author=T.R.+Hughes&amp;volume=40&amp;publication_year=2012&amp;pages=D169-D179&amp;pmid=22102575&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B34">34. <span class="mixed-citation">
Kumme J., Dietz M., Wagner C., Sch&#x000fc;ller H.-J.. 
<span class="ref-title">Dimerization of yeast transcription factors Ino2 and Ino4 is regulated by precursors of phospholipid biosynthesis mediated by Opi1 repressor</span>. <span class="ref-journal">Curr. Genet.</span> 2008; <span class="ref-vol">54</span>:35&#x02013;45. [<a href="https://pubmed.ncbi.nlm.nih.gov/18542964" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Genet.&amp;title=Dimerization+of+yeast+transcription+factors+Ino2+and+Ino4+is+regulated+by+precursors+of+phospholipid+biosynthesis+mediated+by+Opi1+repressor&amp;author=J.+Kumme&amp;author=M.+Dietz&amp;author=C.+Wagner&amp;author=H.-J.+Sch&#x000fc;ller&amp;volume=54&amp;publication_year=2008&amp;pages=35-45&amp;pmid=18542964&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B35">35. <span class="mixed-citation">
Lai K., Mcgraws P.. 
<span class="ref-title">Dual control of inositol transport in saccharomyces cerevisiae by irreversible inactivation of permease and regulation of permease synthesis by IN02, IN04, and OPI1</span>. <span class="ref-journal">JBC</span>. 1994; <span class="ref-vol">269</span>:2246&#x02013;2251. [<a href="https://pubmed.ncbi.nlm.nih.gov/8294482" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JBC&amp;title=Dual+control+of+inositol+transport+in+saccharomyces+cerevisiae+by+irreversible+inactivation+of+permease+and+regulation+of+permease+synthesis+by+IN02,+IN04,+and+OPI1&amp;author=K.+Lai&amp;author=P.+Mcgraws&amp;volume=269&amp;publication_year=1994&amp;pages=2246-2251&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B36">36. <span class="mixed-citation">
Tamura K.I., Gu Y., Wang Q., Yamada T., Ito K., Shimoi H.. 
<span class="ref-title">A hap1 mutation in a laboratory strain of Saccharomyces cerevisiae results in decreased expression of ergosterol-related genes and cellular ergosterol content compared to sake yeast</span>. <span class="ref-journal">J. Biosci. Bioeng.</span> 2004; <span class="ref-vol">98</span>:159&#x02013;166. [<a href="https://pubmed.ncbi.nlm.nih.gov/16233684" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biosci.+Bioeng.&amp;title=A+hap1+mutation+in+a+laboratory+strain+of+Saccharomyces+cerevisiae+results+in+decreased+expression+of+ergosterol-related+genes+and+cellular+ergosterol+content+compared+to+sake+yeast&amp;author=K.I.+Tamura&amp;author=Y.+Gu&amp;author=Q.+Wang&amp;author=T.+Yamada&amp;author=K.+Ito&amp;volume=98&amp;publication_year=2004&amp;pages=159-166&amp;pmid=16233684&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B37">37. <span class="mixed-citation">
Nishi K., Park C.S., Pepper A.E., Eichinger G., Innis M.A., Holland M.J.. 
<span class="ref-title">The GCR1 requirement for yeast glycolytic gene expression is suppressed by dominant mutations in the SGC1 gene, which encodes a novel basic-helix-loop-helix protein</span>. <span class="ref-journal">Mol. Cell. Biol.</span> 1995; <span class="ref-vol">15</span>:2646&#x02013;2653. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC230494/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7739544" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Biol.&amp;title=The+GCR1+requirement+for+yeast+glycolytic+gene+expression+is+suppressed+by+dominant+mutations+in+the+SGC1+gene,+which+encodes+a+novel+basic-helix-loop-helix+protein&amp;author=K.+Nishi&amp;author=C.S.+Park&amp;author=A.E.+Pepper&amp;author=G.+Eichinger&amp;author=M.A.+Innis&amp;volume=15&amp;publication_year=1995&amp;pages=2646-2653&amp;pmid=7739544&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B38">38. <span class="mixed-citation">
Blom J., De&#x000a0;Mattos M.J.T., Grivell L.A.. 
<span class="ref-title">Redirection of the respiro-fermentative flux distribution in Saccharomyces cerevisiae by overexpression of the transcription factor Hap4P</span>. <span class="ref-journal">Appl. Environ. Microbiol.</span> 2000; <span class="ref-vol">66</span>:1970&#x02013;1973. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC101441/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10788368" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Appl.+Environ.+Microbiol.&amp;title=Redirection+of+the+respiro-fermentative+flux+distribution+in+Saccharomyces+cerevisiae+by+overexpression+of+the+transcription+factor+Hap4P&amp;author=J.+Blom&amp;author=M.J.T.+De&#x000a0;Mattos&amp;author=L.A.+Grivell&amp;volume=66&amp;publication_year=2000&amp;pages=1970-1973&amp;pmid=10788368&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B39">39. <span class="mixed-citation">
Natarajan K., Meyer M.R., Jackson B.M., Slade D., Roberts C., Hinnebusch A.G., Marton M.J.. 
<span class="ref-title">Transcriptional profiling shows that Gcn4p is a master regulator of gene expression during amino acid starvation in yeast</span>. <span class="ref-journal">Mol. Cell. Biol.</span> 2001; <span class="ref-vol">21</span>:4347&#x02013;4368. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC87095/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11390663" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Biol.&amp;title=Transcriptional+profiling+shows+that+Gcn4p+is+a+master+regulator+of+gene+expression+during+amino+acid+starvation+in+yeast&amp;author=K.+Natarajan&amp;author=M.R.+Meyer&amp;author=B.M.+Jackson&amp;author=D.+Slade&amp;author=C.+Roberts&amp;volume=21&amp;publication_year=2001&amp;pages=4347-4368&amp;pmid=11390663&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B40">40. <span class="mixed-citation">
Mittal N., Guimaraes J.C., Gross T., Schmidt A., Vina-Vilaseca A., Nedialkova D.D., Aeschimann F., Leidel S.A., Spang A., Zavolan M.. 
<span class="ref-title">The Gcn4 transcription factor reduces protein synthesis capacity and extends yeast lifespan</span>. <span class="ref-journal">Nat. Commun.</span> 2017; <span class="ref-vol">8</span>:457. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5587724/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28878244" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=The+Gcn4+transcription+factor+reduces+protein+synthesis+capacity+and+extends+yeast+lifespan&amp;author=N.+Mittal&amp;author=J.C.+Guimaraes&amp;author=T.+Gross&amp;author=A.+Schmidt&amp;author=A.+Vina-Vilaseca&amp;volume=8&amp;publication_year=2017&amp;pages=457&amp;pmid=28878244&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B41">41. <span class="mixed-citation">
Joo Y.J., Kim J.H., Kang U.B., Yu M.H., Kim J.. 
<span class="ref-title">Gcn4p-mediated transcriptional repression of ribosomal protein genes under amino-acid starvation</span>. <span class="ref-journal">EMBO J.</span> 2011; <span class="ref-vol">30</span>:859&#x02013;872. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3049204/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21183953" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+J.&amp;title=Gcn4p-mediated+transcriptional+repression+of+ribosomal+protein+genes+under+amino-acid+starvation&amp;author=Y.J.+Joo&amp;author=J.H.+Kim&amp;author=U.B.+Kang&amp;author=M.H.+Yu&amp;author=J.+Kim&amp;volume=30&amp;publication_year=2011&amp;pages=859-872&amp;pmid=21183953&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B42">42. <span class="mixed-citation">
Pomraning K.R., Kim Y.-M.M., Nicora C.D., Chu R.K., Bredeweg E.L., Purvine S.O., Hu D., Metz T.O., Baker S.E.. 
<span class="ref-title">Multi-omics analysis reveals regulators of the response to nitrogen limitation in Yarrowia lipolytica</span>. <span class="ref-journal">BMC Genomics</span>. 2016; <span class="ref-vol">17</span>:138. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4766638/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26911370" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genomics&amp;title=Multi-omics+analysis+reveals+regulators+of+the+response+to+nitrogen+limitation+in+Yarrowia+lipolytica&amp;author=K.R.+Pomraning&amp;author=Y.-M.M.+Kim&amp;author=C.D.+Nicora&amp;author=R.K.+Chu&amp;author=E.L.+Bredeweg&amp;volume=17&amp;publication_year=2016&amp;pages=138&amp;pmid=26911370&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B43">43. <span class="mixed-citation">
R&#x000e9;gnacq M., Alimardani P., El&#x000a0;Moudni B., Berg&#x000e8;s T.. 
<span class="ref-title">Sut1p interaction with Cyc8p(Ssn6p) relieves hypoxic genes from Cyc8p-Tup1p repression in Saccharomyces cerevisiae</span>. <span class="ref-journal">Mol. Microbiol.</span> 2001; <span class="ref-vol">40</span>:1085&#x02013;1096. [<a href="https://pubmed.ncbi.nlm.nih.gov/11401714" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Microbiol.&amp;title=Sut1p+interaction+with+Cyc8p(Ssn6p)+relieves+hypoxic+genes+from+Cyc8p-Tup1p+repression+in+Saccharomyces+cerevisiae&amp;author=M.+R&#x000e9;gnacq&amp;author=P.+Alimardani&amp;author=B.+El&#x000a0;Moudni&amp;author=T.+Berg&#x000e8;s&amp;volume=40&amp;publication_year=2001&amp;pages=1085-1096&amp;pmid=11401714&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B44">44. <span class="mixed-citation">
Gasmi N., Jacques P.E., Klimova N., Guo X., Ricciardi A., Robert F., Turcotte B.. 
<span class="ref-title">The switch from fermentation to respiration in Saccharomyces cerevisiae is regulated by the Ert1 transcriptional activator/repressor</span>. <span class="ref-journal">Genetics</span>. 2014; <span class="ref-vol">198</span>:547&#x02013;560. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4196612/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25123508" ref="reftype=pubmed&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=The+switch+from+fermentation+to+respiration+in+Saccharomyces+cerevisiae+is+regulated+by+the+Ert1+transcriptional+activator/repressor&amp;author=N.+Gasmi&amp;author=P.E.+Jacques&amp;author=N.+Klimova&amp;author=X.+Guo&amp;author=A.+Ricciardi&amp;volume=198&amp;publication_year=2014&amp;pages=547-560&amp;pmid=25123508&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6547448&amp;issue-id=335732&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Nucleic Acids Research</span> are provided here courtesy of <strong>Oxford University Press</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC6547448/pdf/gkz253.pdf" class="int-view">PDF (4.0M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/6547448/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/6547448/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="6547448" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6547448%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="9932vNij47XWW3teooJhShR4zx4xgPP0DYLKDAJZka8mBOVCcpHAXmtniC1nRcJK">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6547448%2F&amp;text=Predictive%20models%20of%20eukaryotic%20transcriptional%20regulation%20reveals%20changes%20in%20transcription%20factor%20roles%20and%20promoter%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6547448%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6547448/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC6547448/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/30976803/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC6547448/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/30976803/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC6547448/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/6547448/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC6547448/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
