
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>A divergent CheW confers plasticity to nucleoid-associated chemosensory arrays - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="939BE807DB0F4BC50000535B907F28E9.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="plosgen" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC6952110/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6952110/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="PLoS Genetics" /><meta name="citation_title" content="A divergent CheW confers plasticity to nucleoid-associated chemosensory arrays" /><meta name="citation_author" content="Annick Guiseppi" /><meta name="citation_author_institution" content="Laboratoire de Chimie Bact&#x000e9;rienne, Aix Marseille Univ, CNRS, Marseille, France" /><meta name="citation_author" content="Juan Jesus Vicente" /><meta name="citation_author_institution" content="Physiology &#x00026; Biophysics, University of Washington, Seattle, WA, United States of America" /><meta name="citation_author" content="Julien Herrou" /><meta name="citation_author_institution" content="Laboratoire de Chimie Bact&#x000e9;rienne, Aix Marseille Univ, CNRS, Marseille, France" /><meta name="citation_author" content="Deborah Byrne" /><meta name="citation_author_institution" content="Protein Purification Platform, Institut de Microbiologie de la M&#x000e9;diterran&#x000e9;e, CNRS, Marseille, France" /><meta name="citation_author" content="Aurelie Barneoud" /><meta name="citation_author_institution" content="Laboratoire de Chimie Bact&#x000e9;rienne, Aix Marseille Univ, CNRS, Marseille, France" /><meta name="citation_author" content="Audrey Moine" /><meta name="citation_author_institution" content="Laboratoire de Chimie Bact&#x000e9;rienne, Aix Marseille Univ, CNRS, Marseille, France" /><meta name="citation_author" content="Leon Espinosa" /><meta name="citation_author_institution" content="Laboratoire de Chimie Bact&#x000e9;rienne, Aix Marseille Univ, CNRS, Marseille, France" /><meta name="citation_author" content="Marie-Jeanne Basse" /><meta name="citation_author_institution" content="CRCM, Institute Paoli-Calmettes, CNRS, INSERM, Aix Marseille Univ, Marseille, France" /><meta name="citation_author" content="Virginie Molle" /><meta name="citation_author_institution" content="Laboratoire de Dynamique des Interactions Membranaires Normales et Pathologique, Montpellier II et I University, CNRS, France" /><meta name="citation_author" content="T&#x000e2;m Mignot" /><meta name="citation_author_institution" content="Laboratoire de Chimie Bact&#x000e9;rienne, Aix Marseille Univ, CNRS, Marseille, France" /><meta name="citation_author" content="Philippe Roche" /><meta name="citation_author_institution" content="CRCM, Institute Paoli-Calmettes, CNRS, INSERM, Aix Marseille Univ, Marseille, France" /><meta name="citation_author" content="Emilia M. F. Mauriello" /><meta name="citation_author_institution" content="Laboratoire de Chimie Bact&#x000e9;rienne, Aix Marseille Univ, CNRS, Marseille, France" /><meta name="citation_publication_date" content="2019/12" /><meta name="citation_issue" content="12" /><meta name="citation_volume" content="15" /><meta name="citation_doi" content="10.1371/journal.pgen.1008533" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC6952110/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC6952110/" /><meta name="citation_pmid" content="31860666" /><meta name="DC.Title" content="A divergent CheW confers plasticity to nucleoid-associated chemosensory arrays" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="PLOS" /><meta name="DC.Contributor" content="Annick Guiseppi" /><meta name="DC.Contributor" content="Juan Jesus Vicente" /><meta name="DC.Contributor" content="Julien Herrou" /><meta name="DC.Contributor" content="Deborah Byrne" /><meta name="DC.Contributor" content="Aurelie Barneoud" /><meta name="DC.Contributor" content="Audrey Moine" /><meta name="DC.Contributor" content="Leon Espinosa" /><meta name="DC.Contributor" content="Marie-Jeanne Basse" /><meta name="DC.Contributor" content="Virginie Molle" /><meta name="DC.Contributor" content="T&#x000e2;m Mignot" /><meta name="DC.Contributor" content="Philippe Roche" /><meta name="DC.Contributor" content="Emilia M. F. Mauriello" /><meta name="DC.Date" content="2019 Dec" /><meta name="DC.Identifier" content="10.1371/journal.pgen.1008533" /><meta name="DC.Language" content="en" /><meta property="og:title" content="A divergent CheW confers plasticity to nucleoid-associated chemosensory arrays" /><meta property="og:type" content="article" /><meta property="og:description" content="Chemosensory systems are highly organized signaling pathways that allow bacteria to adapt to environmental changes. The Frz chemosensory system from M. xanthus possesses two CheW-like proteins, FrzA (the core CheW) and FrzB. We found that FrzB does not ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6952110/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC6952110/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/331/">PLoS Genet</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/347207/">v.15(12); 2019 Dec</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC6952110
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC6952110/pdf/pgen.1008533.pdf" class="int-view">PDF (2.8M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/6952110/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/6952110/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="6952110" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6952110%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="CMhNAZd2RAAuP1uNsr8GAptaCclFRrI9GoL3bKFbJbsjZLwGJTmzmDRTROPoXAax">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6952110%2F&amp;text=A%20divergent%20CheW%20confers%20plasticity%20to%20nucleoid-associated%20chemosensory%20arrays" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6952110%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6952110/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC6952110/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/31860666/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC6952110/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/31860666/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC6952110/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/6952110/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/331/">PLoS Genet</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/347207/">v.15(12); 2019 Dec</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC6952110
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-plosgen.png" alt="Logo of plosgen" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="https://journals.plos.org/plosgenetics/" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">PLoS Genet.</span> 2019 Dec; 15(12): e1008533. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2019 Dec 20. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1371%2Fjournal.pgen.1008533" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1371/journal.pgen.1008533</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC6952110</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/31860666">31860666</a></div></div></div><h1 class="content-title">A divergent CheW confers plasticity to nucleoid-associated chemosensory arrays</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Guiseppi%20A%5BAuthor%5D" class="affpopup" co-rid="_co_idm140639603389424" co-class="co-affbox">Annick Guiseppi</a>, <span class="fm-role">Conceptualization</span>, <span class="fm-role">Investigation</span>, <span class="fm-role">Methodology</span>,<sup>#</sup><sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Vicente%20JJ%5BAuthor%5D" class="affpopup" co-rid="_co_idm140639605759312" co-class="co-affbox">Juan Jesus Vicente</a>, <span class="fm-role">Conceptualization</span>, <span class="fm-role">Investigation</span>,<sup>#</sup><sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Herrou%20J%5BAuthor%5D" class="affpopup" co-rid="_co_idm140639603370800" co-class="co-affbox">Julien Herrou</a>, <span class="fm-role">Conceptualization</span>, <span class="fm-role">Methodology</span>, <span class="fm-role">Validation</span>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Byrne%20D%5BAuthor%5D" class="affpopup" co-rid="_co_idm140639646732832" co-class="co-affbox">Deborah Byrne</a>, <span class="fm-role">Conceptualization</span>, <span class="fm-role">Data curation</span>, <span class="fm-role">Formal analysis</span>, <span class="fm-role">Investigation</span>, <span class="fm-role">Methodology</span>, <span class="fm-role">Resources</span>, <span class="fm-role">Validation</span>,<sup>3</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Barneoud%20A%5BAuthor%5D" class="affpopup" co-rid="_co_idm140639646727248" co-class="co-affbox">Aurelie Barneoud</a>, <span class="fm-role">Investigation</span>, <span class="fm-role">Methodology</span>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Moine%20A%5BAuthor%5D" class="affpopup" co-rid="_co_idm140639646724144" co-class="co-affbox">Audrey Moine</a>, <span class="fm-role">Data curation</span>, <span class="fm-role">Investigation</span>, <span class="fm-role">Methodology</span>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Espinosa%20L%5BAuthor%5D" class="affpopup" co-rid="_co_idm140639646720544" co-class="co-affbox">Leon Espinosa</a>, <span class="fm-role">Conceptualization</span>, <span class="fm-role">Data curation</span>, <span class="fm-role">Methodology</span>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Basse%20MJ%5BAuthor%5D" class="affpopup" co-rid="_co_idm140639646716208" co-class="co-affbox">Marie-Jeanne Basse</a>, <span class="fm-role">Investigation</span>, <span class="fm-role">Methodology</span>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Molle%20V%5BAuthor%5D" class="affpopup" co-rid="_co_idm140639646713104" co-class="co-affbox">Virginie Molle</a>, <span class="fm-role">Conceptualization</span>, <span class="fm-role">Data curation</span>, <span class="fm-role">Investigation</span>, <span class="fm-role">Methodology</span>, <span class="fm-role">Resources</span>, <span class="fm-role">Validation</span>,<sup>5</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Mignot%20T%5BAuthor%5D" class="affpopup" co-rid="_co_idm140639646708016" co-class="co-affbox">T&#x000e2;m Mignot</a>, <span class="fm-role">Funding acquisition</span>, <span class="fm-role">Project administration</span>, <span class="fm-role">Resources</span>, <span class="fm-role">Supervision</span>, <span class="fm-role">Validation</span>, <span class="fm-role">Writing &#x02013; review &#x00026; editing</span>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Roche%20P%5BAuthor%5D" class="affpopup" co-rid="_co_idm140639603511136" co-class="co-affbox">Philippe Roche</a>, <span class="fm-role">Conceptualization</span>, <span class="fm-role">Data curation</span>, <span class="fm-role">Investigation</span>, <span class="fm-role">Methodology</span>, <span class="fm-role">Resources</span>, <span class="fm-role">Software</span>, <span class="fm-role">Supervision</span>, <span class="fm-role">Validation</span>, <span class="fm-role">Writing &#x02013; review &#x00026; editing</span>,<sup>4</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Mauriello%20EM%5BAuthor%5D" class="affpopup" co-rid="_co_idm140639603503824" co-class="co-affbox">Emilia M. F. Mauriello</a>, <span class="fm-role">Conceptualization</span>, <span class="fm-role">Data curation</span>, <span class="fm-role">Funding acquisition</span>, <span class="fm-role">Investigation</span>, <span class="fm-role">Methodology</span>, <span class="fm-role">Project administration</span>, <span class="fm-role">Resources</span>, <span class="fm-role">Supervision</span>, <span class="fm-role">Validation</span>, <span class="fm-role">Writing &#x02013; original draft</span>, <span class="fm-role">Writing &#x02013; review &#x00026; editing</span><sup>1,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140639603389424"><h3 class="no_margin">Annick Guiseppi</h3><p><sup>1</sup>
Laboratoire de Chimie Bact&#x000e9;rienne, Aix Marseille Univ, CNRS, Marseille, France</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Guiseppi%20A%5BAuthor%5D">Annick Guiseppi</a></div></div><div id="_co_idm140639605759312"><h3 class="no_margin">Juan Jesus Vicente</h3><p><sup>2</sup>
Physiology &#x00026; Biophysics, University of Washington, Seattle, WA, United States of America</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Vicente%20JJ%5BAuthor%5D">Juan Jesus Vicente</a></div></div><div id="_co_idm140639603370800"><h3 class="no_margin">Julien Herrou</h3><p><sup>1</sup>
Laboratoire de Chimie Bact&#x000e9;rienne, Aix Marseille Univ, CNRS, Marseille, France</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Herrou%20J%5BAuthor%5D">Julien Herrou</a></div></div><div id="_co_idm140639646732832"><h3 class="no_margin">Deborah Byrne</h3><p><sup>3</sup>
Protein Purification Platform, Institut de Microbiologie de la M&#x000e9;diterran&#x000e9;e, CNRS, Marseille, France</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Byrne%20D%5BAuthor%5D">Deborah Byrne</a></div></div><div id="_co_idm140639646727248"><h3 class="no_margin">Aurelie Barneoud</h3><p><sup>1</sup>
Laboratoire de Chimie Bact&#x000e9;rienne, Aix Marseille Univ, CNRS, Marseille, France</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Barneoud%20A%5BAuthor%5D">Aurelie Barneoud</a></div></div><div id="_co_idm140639646724144"><h3 class="no_margin">Audrey Moine</h3><p><sup>1</sup>
Laboratoire de Chimie Bact&#x000e9;rienne, Aix Marseille Univ, CNRS, Marseille, France</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Moine%20A%5BAuthor%5D">Audrey Moine</a></div></div><div id="_co_idm140639646720544"><h3 class="no_margin">Leon Espinosa</h3><p><sup>1</sup>
Laboratoire de Chimie Bact&#x000e9;rienne, Aix Marseille Univ, CNRS, Marseille, France</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Espinosa%20L%5BAuthor%5D">Leon Espinosa</a></div></div><div id="_co_idm140639646716208"><h3 class="no_margin">Marie-Jeanne Basse</h3><p><sup>4</sup>
CRCM, Institute Paoli-Calmettes, CNRS, INSERM, Aix Marseille Univ, Marseille, France</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Basse%20MJ%5BAuthor%5D">Marie-Jeanne Basse</a></div></div><div id="_co_idm140639646713104"><h3 class="no_margin">Virginie Molle</h3><p><sup>5</sup>
Laboratoire de Dynamique des Interactions Membranaires Normales et Pathologique, Montpellier II et I University, CNRS, France</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Molle%20V%5BAuthor%5D">Virginie Molle</a></div></div><div id="_co_idm140639646708016"><h3 class="no_margin">T&#x000e2;m Mignot</h3><p><sup>1</sup>
Laboratoire de Chimie Bact&#x000e9;rienne, Aix Marseille Univ, CNRS, Marseille, France</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Mignot%20T%5BAuthor%5D">T&#x000e2;m Mignot</a></div></div><div id="_co_idm140639603511136"><h3 class="no_margin">Philippe Roche</h3><p><sup>4</sup>
CRCM, Institute Paoli-Calmettes, CNRS, INSERM, Aix Marseille Univ, Marseille, France</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Roche%20P%5BAuthor%5D">Philippe Roche</a></div></div><div id="_co_idm140639603503824"><h3 class="no_margin">Emilia M. F. Mauriello</h3><p><sup>1</sup>
Laboratoire de Chimie Bact&#x000e9;rienne, Aix Marseille Univ, CNRS, Marseille, France</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Mauriello%20EM%5BAuthor%5D">Emilia M. F. Mauriello</a></div></div></div></div><div class="contrib-group half_rhythm fm-editor">Ariane Briegel, <span class="fm-role">Editor</span><sup></sup></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140639604466544_ai idm140639603389552_ai idm140639603488832_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140639604466544_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140639604466544_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140639604466544_ai" style="display:none"><div class="fm-affl" id="aff001"><sup>1</sup>
Laboratoire de Chimie Bact&#x000e9;rienne, Aix Marseille Univ, CNRS, Marseille, France</div><div class="fm-affl" id="aff002"><sup>2</sup>
Physiology &#x00026; Biophysics, University of Washington, Seattle, WA, United States of America</div><div class="fm-affl" id="aff003"><sup>3</sup>
Protein Purification Platform, Institut de Microbiologie de la M&#x000e9;diterran&#x000e9;e, CNRS, Marseille, France</div><div class="fm-affl" id="aff004"><sup>4</sup>
CRCM, Institute Paoli-Calmettes, CNRS, INSERM, Aix Marseille Univ, Marseille, France</div><div class="fm-affl" id="aff005"><sup>5</sup>
Laboratoire de Dynamique des Interactions Membranaires Normales et Pathologique, Montpellier II et I University, CNRS, France</div><div class="fm-affl" id="edit1">Universiteit Leiden Instituut Biologie Leiden, NETHERLANDS</div><div><sup>#</sup>Contributed equally.</div><div id="coi001">The authors have declared that no competing interests exist.</div><div id="cor001">* E-mail: <a href="mailto:dev@null" data-email="rf.srnc.mmi@olleiruame" class="oemail">rf.srnc.mmi@olleiruame</a></div></div><div class="fm-article-notes hide half_rhythm" id="idm140639604466544_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2019 Jan 31; Accepted 2019 Nov 22.</div></div><div class="permissions half_rhythm hide" id="idm140639604466544_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; 2019 Guiseppi et al</div><div class="license half_rhythm">This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="13" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S1 Fig: </strong>(A) Motility and fruiting body formation phenotypes were photographed at 48h and 72h, respectively. (B) Box plots of reversal frequencies of single cells moving on agar pads supplemented or not with 0.15% IAA. The lower and upper boundaries of the boxes correspond to 25% and 75% percentiles, respectively. The median is shown as a line at the center of each box and whiskers represents the 10% and 90% percentiles. For the reversal frequency measurements, cells issued from two biological replicates, were used. For WT approximately 500 cells, issued from six biological replicates, were used.<p>(PDF)</p></div><div><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s001.pdf" data-ga-action="click_feat_suppl">pgen.1008533.s001.pdf</a><span style="color:gray"> (1.1M)</span></div><div class="small guid">GUID: 715BC561-A84C-48E3-B7E3-7C1D4261FEBA</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S2 Fig: </strong>Western blot with anti-FrzB antibodies on the cell extracts of the indicated <em>M</em>. <em>xanthus</em> strains. The <em>pilA</em>::<em>frzB</em> strain, used as positive control, expresses <em>frzB</em> under the control of the <em>pilA</em> promoter for high expression. The white line is used to indicate that two lanes from the same gel were separated by other lanes in the original gel.<p>(PDF)</p></div><div><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s002.pdf" data-ga-action="click_feat_suppl">pgen.1008533.s002.pdf</a><span style="color:gray"> (158K)</span></div><div class="small guid">GUID: 9F08FA6D-6EE8-484F-B75B-9D7E9510FCB0</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S3 Fig: </strong>Motility and fruiting body formation phenotypes of the indicated strains were photographed at 48h and 72h, respectively. (PDF)</div><div><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s003.pdf" data-ga-action="click_feat_suppl">pgen.1008533.s003.pdf</a><span style="color:gray"> (215K)</span></div><div class="small guid">GUID: 19A4CF32-6C69-4E04-9B0F-B94F0A8051D9</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S4 Fig: </strong>(Left) Fluorescence micrographs of the indicated <em>M</em>. <em>xanthus</em> strains carrying FrzCD-gfp fusions. The cell boundaries were drawn manually from the phase-contrast images. (Right) For each indicated strain, more than 120 cells (<em>x</em> axis) from at least two biological replicates are represented as lines and ordered according to their length (pixels) in demographs. The GFP fluorescence intensity along the cell body is represented as colored pixels at the corresponding cell position (from -1 to +1 on the <em>y</em> axis). &#x0201c;0&#x0201d; is the cell center. On the right, a scale indicates the fluorescence intensity and the corresponding colors.<p>(PDF)</p></div><div><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s004.pdf" data-ga-action="click_feat_suppl">pgen.1008533.s004.pdf</a><span style="color:gray"> (219K)</span></div><div class="small guid">GUID: F41A7F13-85B8-436D-B22A-7992BEACC5ED</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S5 Fig: </strong>Western blot with anti-FrzCD antibodies on the cell extracts of the indicated <em>M</em>. <em>xanthus</em> strains. (PDF)</div><div><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s005.pdf" data-ga-action="click_feat_suppl">pgen.1008533.s005.pdf</a><span style="color:gray"> (1.1M)</span></div><div class="small guid">GUID: 7885B754-6162-4976-8EA6-2DD3B410365A</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S6 Fig: </strong>(A) Cells of the indicated strains were incubated 30 minutes with DAPI and then imaged at the fluorescence microscope. The nucleoid and cell surfaces were measured automatically with Microbe J [<a href="#pgen.1008533.ref044" rid="pgen.1008533.ref044" class=" bibr popnode">44</a>]. (B) The ratio between the nucleoid and cell surface was then calculated for each cell and averages values plotted. The numbers of analyzed cells are indicated per each strain. Cells were issued from two independent biological replicates.<p>(PDF)</p></div><div><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s006.pdf" data-ga-action="click_feat_suppl">pgen.1008533.s006.pdf</a><span style="color:gray"> (913K)</span></div><div class="small guid">GUID: F739220B-CEA6-4592-B112-8AFBCFDD41BE</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S1 Table: </strong>List of strains used in this work. (PDF)</div><div><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s007.pdf" data-ga-action="click_feat_suppl">pgen.1008533.s007.pdf</a><span style="color:gray"> (50K)</span></div><div class="small guid">GUID: F0661132-3590-4074-8713-C13BB6CB4AE4</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S2 Table: </strong>List of plasmids used in this work. (PDF)</div><div><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s008.pdf" data-ga-action="click_feat_suppl">pgen.1008533.s008.pdf</a><span style="color:gray"> (60K)</span></div><div class="small guid">GUID: 9A76154C-6544-49A6-848C-6295D91E078C</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S3 Table: </strong>Data to generate the mean R<sup>2</sup> on <a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g004/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g004" rid-ob="ob-pgen.1008533.g004" co-legend-rid="lgnd_pgen.1008533.g004"><span>Fig 4A</span></a> to measure the colocalization of mCherry-FrzB with the nucleoid. (XLSX)</div><div><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s009.xlsx" data-ga-action="click_feat_suppl">pgen.1008533.s009.xlsx</a><span style="color:gray"> (44K)</span></div><div class="small guid">GUID: CDF3F548-98E1-4A02-A39D-DB4F46512E0D</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S4 Table: </strong>Raw data to generate the histogram on <a href="#pgen.1008533.s006" rid="pgen.1008533.s006" class=" supplementary-material">S6 Fig</a>. (CSV)</div><div><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s010.csv" data-ga-action="click_feat_suppl">pgen.1008533.s010.csv</a><span style="color:gray"> (20K)</span></div><div class="small guid">GUID: F9FC97DF-CD44-4585-B98E-CD78C4D9FF17</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S5 Table: </strong>Raw data to generate the box plots for non-reversing strains on <a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g001/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g001" rid-ob="ob-pgen.1008533.g001" co-legend-rid="lgnd_pgen.1008533.g001"><span>Fig 1B</span></a>. (CSV)</div><div><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s011.csv" data-ga-action="click_feat_suppl">pgen.1008533.s011.csv</a><span style="color:gray"> (4.4K)</span></div><div class="small guid">GUID: A5757F12-2733-4A9F-A3AD-AD8CB9392D36</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S6 Table: </strong>Raw data to generate the box plots for reversing strains on <a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g001/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g001" rid-ob="ob-pgen.1008533.g001" co-legend-rid="lgnd_pgen.1008533.g001"><span>Fig 1B</span></a>. (CSV)</div><div><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s012.csv" data-ga-action="click_feat_suppl">pgen.1008533.s012.csv</a><span style="color:gray"> (2.6M)</span></div><div class="small guid">GUID: 1C0B822D-D7E3-4044-8974-7BD7E7E52A41</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S7 Table: </strong>Raw data to generate the box plots for non-reversing strains on <a href="#pgen.1008533.s001" rid="pgen.1008533.s001" class=" supplementary-material">S1B Fig</a>. (CSV)</div><div><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s013.csv" data-ga-action="click_feat_suppl">pgen.1008533.s013.csv</a><span style="color:gray"> (13K)</span></div><div class="small guid">GUID: F1D3A0C0-C311-4A92-A723-0DBAE7AA49F5</div></div></dd></dl><dl data-length="81" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p>All relevant data are within the manuscript and its Supporting Information files.</p></dd></dl></div><div id="abstract-a.t.b.w" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.t.b.wtitle">Abstract</h2><!--article-meta--><div><p class="p p-first-last">Chemosensory systems are highly organized signaling pathways that allow bacteria to adapt to environmental changes. The Frz chemosensory system from <em>M</em>. <em>xanthus</em> possesses two CheW-like proteins, FrzA (the core CheW) and FrzB. We found that FrzB does not interact with FrzE (the cognate CheA) as it lacks the amino acid region responsible for this interaction. FrzB, instead, acts upstream of FrzCD in the regulation of <em>M</em>. <em>xanthus</em> chemotaxis behaviors and activates the Frz pathway by allowing the formation and distribution of multiple chemosensory clusters on the nucleoid. These results, together, show that the lack of the CheA-interacting region in FrzB confers new functions to this small protein.</p></div></div><div id="abstract-a.t.b.x" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.t.b.xtitle">Author summary</h2><!--article-meta--><div><p class="p p-first-last">Chemosensory systems are signaling complexes that are widespread in bacteria and allow the modulation of different cellular functions, such as taxis and development, in response to the environment. We show that the <em>Myxococcus xanthus</em> FrzB is a divergent CheW lacking the region involved in the interaction with the histidine kinase FrzE. Instead, it acts upstream of FrzCD to allow the formation of multiple distributed Frz chemosensory arrays at the nucleoid. The loss of the CheA-interacting region in FrzB might have been selected to confer plasticity to nucleoid-associated chemosensory systems. By unraveling a new accessory protein and its function, this work opens new insights into the knowledge of the regulatory potentials of bacterial chemosensory systems.</p></div></div><div id="sec001" class="tsec sec"><h2 class="head no_bottom_margin" id="sec001title">Introduction</h2><p class="p p-first">Chemosensory systems are specialized regulatory pathways that allow bacteria to perceive their external environment and respond with various cellular behaviors [<a href="#pgen.1008533.ref001" rid="pgen.1008533.ref001" class=" bibr popnode">1</a>&#x02013;<a href="#pgen.1008533.ref003" rid="pgen.1008533.ref003" class=" bibr popnode">3</a>]. In these systems, environmental signals are transduced inside the cells, initially by receptors called Methyl-accepting Chemotaxis Proteins (MCP). Most MCPs possess a transmembrane domain, but 14% are soluble proteins [<a href="#pgen.1008533.ref004" rid="pgen.1008533.ref004" class=" bibr popnode">4</a>]. Ligand-bound MCPs regulate the autophosphorylation of the histidine kinase CheA that, in turn, transfers phosphoryl groups to at least two response regulators: CheY which is the most downstream component of the pathway and CheB, which, together with CheR, constitutes the adaptation module. Che systems also include one or more versions of the docking protein CheW, which interacts with the C-terminal cytoplasmic tip of the MCP and with the P5 domain of CheA. CheW and CheA<sup>P5</sup> are paralogs and are topologically similar to SH3 domains from eukaryotic scaffold proteins that also play a role in signal transduction. Such domains are characterized by two &#x003b2;-sheets (each composed of five &#x003b2;-strands) designated subdomain 1 and 2 [<a href="#pgen.1008533.ref005" rid="pgen.1008533.ref005" class=" bibr popnode">5</a>&#x02013;<a href="#pgen.1008533.ref007" rid="pgen.1008533.ref007" class=" bibr popnode">7</a>]. MCP: CheA<sup>P5</sup>: CheW interactions all involve subdomains 1 and 2. In particular, the &#x003b2;-strands 3 and 4 of subdomain 1 of CheA<sup>P5</sup> interact with the &#x003b2;-strands 4 and 5 of subdomain 2 of CheW [<a href="#pgen.1008533.ref008" rid="pgen.1008533.ref008" class=" bibr popnode">8</a>]. The receptor cytoplasmic tip binds CheW or CheA at the hydrophobic junction between subdomain 1 and 2 [<a href="#pgen.1008533.ref008" rid="pgen.1008533.ref008" class=" bibr popnode">8</a>].</p><p>High-resolution microscopy revealed that, intracellularly, Che proteins are organized in complex arrays running parallel to the bacterial inner membrane, with the MCP layer linked to a CheA/W baseplate [<a href="#pgen.1008533.ref009" rid="pgen.1008533.ref009" class=" bibr popnode">9</a>,<a href="#pgen.1008533.ref010" rid="pgen.1008533.ref010" class=" bibr popnode">10</a>]. Transverse views of these Che arrays show ordered MCP hexagons networked by CheA-CheW rings. CheA-CheW rings contain alternating and interacting CheA<sup>P5</sup> domains and CheWs. MCPs hexagons are constituted by six MCP trimers of dimers. One MCP in the dimer is connected with either a CheA<sup>P5</sup> domain or a CheW, which allows connections between the MCP hexagons and the CheW-CheA<sup>P5</sup> rings [<a href="#pgen.1008533.ref011" rid="pgen.1008533.ref011" class=" bibr popnode">11</a>] (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g001/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g001" rid-ob="ob-pgen.1008533.g001" co-legend-rid="lgnd_pgen.1008533.g001"><span>Fig 1A</span></a>). The presence of each of these three proteins is essential for the integrity of the system [<a href="#pgen.1008533.ref012" rid="pgen.1008533.ref012" class=" bibr popnode">12</a>,<a href="#pgen.1008533.ref013" rid="pgen.1008533.ref013" class=" bibr popnode">13</a>]. This structure provides the typical signaling properties of chemotaxis systems, including signal amplification and sensitivity [<a href="#pgen.1008533.ref014" rid="pgen.1008533.ref014" class=" bibr popnode">14</a>&#x02013;<a href="#pgen.1008533.ref017" rid="pgen.1008533.ref017" class=" bibr popnode">17</a>].</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="pgen.1008533.g001" co-legend-rid="lgnd_pgen.1008533.g001"><a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g001/" target="figure" rid-figpopup="pgen.1008533.g001" rid-ob="ob-pgen.1008533.g001"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140639649406736"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6952110_pgen.1008533.g001.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pgen.1008533.g001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6952110/bin/pgen.1008533.g001.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140639649406736"><a target="object" rel="noopener" href="/pmc/articles/PMC6952110/figure/pgen.1008533.g001/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_pgen.1008533.g001"><div><a class="figpopup" href="/pmc/articles/PMC6952110/figure/pgen.1008533.g001/" target="figure" rid-figpopup="pgen.1008533.g001" rid-ob="ob-pgen.1008533.g001">Fig 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Schematic representation of the supramolecular organization of Che proteins.</strong><p>(A) MCP form trimers of dimers (each dimer is shown as a green circle), which, in turn, form hexagons connected with rings composed of the CheA<sup>P5</sup> domain (dark blue bars) and CheW (white bars). The light blue circles represent the CheA<sup>P4</sup> domain and, the red circles the interface between the &#x003b2;-strands 3 and 4 of subdomain 1 of CheA<sup>P5</sup> and the &#x003b2;-strands 4 and 5 of subdomain 2 of CheW. Rings containing six CheW proteins (shown at the center of the array) might serve to modulate the stability and activation of the system. A signaling unit is represented in the red box (Adapted from [<a href="#pgen.1008533.ref023" rid="pgen.1008533.ref023" class=" bibr popnode">23</a>]). (B) FrzCD, FrzE<sup>CheA</sup>, FrzA and FrzB proteins organization depicted by homology with Che proteins. (C) Schematic representation of the <em>frz</em> operon.</p></div></div></div><p>The gliding bacterium <em>M</em>. <em>xanthus</em> uses the cytoplasmic Frz signal transduction pathway to bias the frequency at which cells change their direction of movement in response to environmental signals [<a href="#pgen.1008533.ref018" rid="pgen.1008533.ref018" class=" bibr popnode">18</a>]. This behavior allows cells to glide towards favorable directions or away from toxic compounds. The Frz system induces cell reversal by promoting the coordinated inversion of the cell polarity of the two <em>M</em>. <em>xanthus</em> motility systems: the Social (S) motility system powered by polar Type IV Pili and the Adventurous (A) motility powered by polarly assembled Agl-Glt complexes [<a href="#pgen.1008533.ref019" rid="pgen.1008533.ref019" class=" bibr popnode">19</a>&#x02013;<a href="#pgen.1008533.ref021" rid="pgen.1008533.ref021" class=" bibr popnode">21</a>]. The regulation of the motility systems by the Frz pathway is essential for cells to enter a developmental program that leads to the formation of multicellular fruiting bodies. The Frz core is composed of a cytoplasmic MCP (FrzCD), a CheA (FrzE) and a CheW (FrzA) [<a href="#pgen.1008533.ref022" rid="pgen.1008533.ref022" class=" bibr popnode">22</a>] (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g001/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g001" rid-ob="ob-pgen.1008533.g001" co-legend-rid="lgnd_pgen.1008533.g001"><span>Fig 1B</span></a>), encoded by a single operon (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g001/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g001" rid-ob="ob-pgen.1008533.g001" co-legend-rid="lgnd_pgen.1008533.g001"><span>Fig 1C</span></a>). In the absence of any of these three proteins, cells display drastically reduced reversal frequencies and are no longer able to respond to isoamyl alcohol (IAA), a Frz activator [<a href="#pgen.1008533.ref020" rid="pgen.1008533.ref020" class=" bibr popnode">20</a>,<a href="#pgen.1008533.ref022" rid="pgen.1008533.ref022" class=" bibr popnode">22</a>]. The Frz system also includes a second CheW-like protein, FrzB, described as an accessory because while in its absence cells show phenotypes similar to those caused by the deletion of core proteins, <em>&#x00394;frzB</em> cells are still able to respond to IAA with increased reversal frequencies [<a href="#pgen.1008533.ref020" rid="pgen.1008533.ref020" class=" bibr popnode">20</a>,<a href="#pgen.1008533.ref022" rid="pgen.1008533.ref022" class=" bibr popnode">22</a>] (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g001/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g001" rid-ob="ob-pgen.1008533.g001" co-legend-rid="lgnd_pgen.1008533.g001"><span>Fig 1B</span></a>).</p><p class="p p-last">In this paper, we show that FrzB is a divergent CheW lacking the &#x003b2;-strands involved in the interaction with the histidine kinase FrzE (&#x003b2;4 and &#x003b2;5). Strikingly, the introduction of &#x003b2;4 and &#x003b2;5 from FrzA to FrzB converts the latter into a canonical CheW. Instead of mediating signal transduction from the FrzCD receptor to FrzE, FrzB acts upstream of FrzCD to allow the formation of multiple distributed Frz chemosensory clusters at the nucleoid. These clusters collapse in a unique and less active array in the absence of FrzB. Finally, the loss of the CheA-interacting region in FrzB might have been selected to confer plasticity to nucleoid-associated chemosensory systems.</p></div><div id="sec002" class="tsec sec"><h2 class="head no_bottom_margin" id="sec002title">Results</h2><div id="sec003" class="sec sec-first"><h3 id="sec003title">FrzB is upstream of the FrzCD receptor in the Frz signaling pathway</h3><p class="p p-first">The first two genes of the <em>frz</em> operon encode the CheW-like proteins, FrzA and FrzB. <em>&#x00394;frzA</em> cells show the same motility and fruiting body formation phenotypes as <em>&#x00394;frzCD</em> and <em>&#x00394;frzE</em>, suggesting that FrzA is the core CheW of the Frz pathway [<a href="#pgen.1008533.ref022" rid="pgen.1008533.ref022" class=" bibr popnode">22</a>,<a href="#pgen.1008533.ref024" rid="pgen.1008533.ref024" class=" bibr popnode">24</a>] (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g002/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g002" rid-ob="ob-pgen.1008533.g002" co-legend-rid="lgnd_pgen.1008533.g002"><span>Fig 2A</span></a>). The aberrant behaviors at the colony level are due to the inability of <em>&#x00394;frzA</em> strains to modulate single-cell reversal frequencies even in the presence of isoamyl alcohol (IAA), a known Frz activator (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g002/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g002" rid-ob="ob-pgen.1008533.g002" co-legend-rid="lgnd_pgen.1008533.g002"><span>Fig 2B</span></a>) [<a href="#pgen.1008533.ref020" rid="pgen.1008533.ref020" class=" bibr popnode">20</a>,<a href="#pgen.1008533.ref022" rid="pgen.1008533.ref022" class=" bibr popnode">22</a>].</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="pgen.1008533.g002" co-legend-rid="lgnd_pgen.1008533.g002"><a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g002/" target="figure" rid-figpopup="pgen.1008533.g002" rid-ob="ob-pgen.1008533.g002"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140639646774496"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6952110_pgen.1008533.g002.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pgen.1008533.g002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6952110/bin/pgen.1008533.g002.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140639646774496"><a target="object" rel="noopener" href="/pmc/articles/PMC6952110/figure/pgen.1008533.g002/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_pgen.1008533.g002"><div><a class="figpopup" href="/pmc/articles/PMC6952110/figure/pgen.1008533.g002/" target="figure" rid-figpopup="pgen.1008533.g002" rid-ob="ob-pgen.1008533.g002">Fig 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>The effect of FrzB on the IAA response and the location of the <em>frzB</em> gene in the Frz pathway.</strong><p>(A) Motility and fruiting body formation phenotypes were photographed at 48h and 72h, respectively. (B) Box plots of reversal frequencies of single cells moving on agar pads supplemented or not with 0.15% IAA. The lower and upper boundaries of the boxes correspond to 25th and 75th percentiles, respectively. The median is shown as a line at the center of each box, and whiskers represents the 10th and 90th percentiles. For the reversal frequency measurements, a minimum of 30 cells from three to six biological replicates, were used. <em>frzCDc</em> corresponds to the allelic variant <em>frzCD&#x00394;6&#x02013;182</em>.</p></div></div></div><p>While a <em>frzB</em> deletion causes the same defects as <em>&#x00394;frzA</em> in the absence of IAA, IAA completely rescues the reversal frequency defect of <em>&#x00394;frzB</em> cells (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g002/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g002" rid-ob="ob-pgen.1008533.g002" co-legend-rid="lgnd_pgen.1008533.g002"><span>Fig 2B</span></a>). This effect is detectable at both single-cell and colony level (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g002/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g002" rid-ob="ob-pgen.1008533.g002" co-legend-rid="lgnd_pgen.1008533.g002"><span>Fig 2</span></a>) [<a href="#pgen.1008533.ref022" rid="pgen.1008533.ref022" class=" bibr popnode">22</a>]. These results show that signals from the FrzCD receptor can still be transduced to FrzE in the absence of FrzB and, thus, FrzB is not part of the Frz core. Moreover, a <em>&#x00394;frzA &#x00394;frzB</em> double mutant phenocopies <em>&#x00394;frzA</em>, further suggesting the central role of FrzA and the accessory role of FrzB (<a href="#pgen.1008533.s001" rid="pgen.1008533.s001" class=" supplementary-material">S1 Fig</a>).</p><p class="p p-last">In order to determine the functional position of FrzB in the Frz pathway, we tested the epistasis of FrzB with respect to FrzCD. Because FrzCD is a central component of the Frz pathway, <em>&#x00394;frzB &#x00394;frzCD</em> and <em>&#x00394;frzA &#x00394;frzCD</em> double mutants showed the same aberrant motility and developmental phenotypes as <em>&#x00394;frzCD</em> (and <em>&#x00394;frzA</em>) at both single-cells and colony levels (<a href="#pgen.1008533.s001" rid="pgen.1008533.s001" class=" supplementary-material">S1 Fig</a> and <a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g002/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g002" rid-ob="ob-pgen.1008533.g002" co-legend-rid="lgnd_pgen.1008533.g002"><span>Fig 2</span></a>) [<a href="#pgen.1008533.ref022" rid="pgen.1008533.ref022" class=" bibr popnode">22</a>]. We thus took advantage of a <em>frzCD</em> allele, <em>frzCDc</em>, whose expression confers a constitutively hyper-reversing phenotype (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g002/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g002" rid-ob="ob-pgen.1008533.g002" co-legend-rid="lgnd_pgen.1008533.g002"><span>Fig 2</span></a>) (Bustamante et al., 2004) and constructed <em>&#x00394;frzA frzCDc</em> and <em>&#x00394;frzB frzCDc</em> double mutants. The <em>&#x00394;frzA frzCDc</em> reversal frequencies and colony phenotypes were indistinguishable from those of <em>&#x00394;frzA</em> cells, confirming that FrzA is located downstream of the receptor in the signaling pathway (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g002/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g002" rid-ob="ob-pgen.1008533.g002" co-legend-rid="lgnd_pgen.1008533.g002"><span>Fig 2</span></a>). On the other hand, <em>&#x00394;frzB frzCDc</em> phenotypes resembled those of <em>frzCDc</em> (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g002/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g002" rid-ob="ob-pgen.1008533.g002" co-legend-rid="lgnd_pgen.1008533.g002"><span>Fig 2</span></a>), indicating that FrzB is upstream of FrzCD in the Frz signaling pathway and that its requirement is bypassed by a constitutive <em>frzCDc</em> mutation.</p></div><div id="sec004" class="sec"><h3 id="sec004title">FrzB does not interact with the downstream histidine kinase FrzE</h3><p class="p p-first">Genetic data show that FrzB is upstream of FrzCD in the Frz regulatory pathway. We thus wanted to verify if FrzB could interact with FrzE. For this, we heterologously expressed and co-purified, using GST-affinity chromatography, GST-FrzA or GST-FrzB with either 6His-FrzCD or 6His-FrzE<sup>CheA</sup>. Both 6His-FrzCD and 6His-FrzE<sup>CheA</sup> could be co-purified with GST-FrzA, suggesting that FrzA interacts with both FrzCD and FrzE<sup>CheA</sup> (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g003/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g003" rid-ob="ob-pgen.1008533.g003" co-legend-rid="lgnd_pgen.1008533.g003"><span>Fig 3A</span></a>). On the other hand, only 6His-FrzCD, but not 6His-FrzE<sup>CheA</sup>, could be co-purified with GST-FrzB, suggesting that FrzB only interacted with FrzCD but not with FrzE<sup>CheA</sup> (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g003/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g003" rid-ob="ob-pgen.1008533.g003" co-legend-rid="lgnd_pgen.1008533.g003"><span>Fig 3A</span></a>). We then wanted to test whether, by using high amounts of GST-FrzB, we could detect an interaction with immobilized 6His-FrzE<sup>CheA</sup> in Biolayer interferometry (BLi) [<a href="#pgen.1008533.ref025" rid="pgen.1008533.ref025" class=" bibr popnode">25</a>]. While low amounts of GST-FrzB still produced an interaction with immobilized 6His-FrzCD (Kd = 0.7 &#x003bc;M), no interactions could be detected with immobilized FrzE<sup>CheA</sup>, even when we used high amounts of GST-FrzB (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g003/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g003" rid-ob="ob-pgen.1008533.g003" co-legend-rid="lgnd_pgen.1008533.g003"><span>Fig 3B</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="pgen.1008533.g003" co-legend-rid="lgnd_pgen.1008533.g003"><a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g003/" target="figure" rid-figpopup="pgen.1008533.g003" rid-ob="ob-pgen.1008533.g003"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140639605860224"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6952110_pgen.1008533.g003.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pgen.1008533.g003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6952110/bin/pgen.1008533.g003.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140639605860224"><a target="object" rel="noopener" href="/pmc/articles/PMC6952110/figure/pgen.1008533.g003/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_pgen.1008533.g003"><div><a class="figpopup" href="/pmc/articles/PMC6952110/figure/pgen.1008533.g003/" target="figure" rid-figpopup="pgen.1008533.g003" rid-ob="ob-pgen.1008533.g003">Fig 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Interaction of FrzB with FrzCD and FrzE.</strong><p>(A) GST affinity chromatography co-purification of the indicated pairs. A, B, CD and E stand for GST-FrzA, GST-FrzB, 6His-FrzCD and 6His-FrzE<sup>CheA</sup>, respectively. In the first gel, lanes are issued from the same gel and were grouped for simplicity. (B) In the left panel, BLi sensograms show the binding of different amounts of GST-FrzB with immobilized 6His-FrzCD. BlitZ ProTM was used to calculate the affinity constant (Kd = 0.7 &#x003bc;M), using a global 1:1 fit. The central panel shows the responses at 145 s (nm, x axis) (average of two biological replicates) of the binding of immobilized 6His-FrzCD with different concentrations of GST-FrzB (&#x003bc;M, y axis). Data were analyzed by GraphPad Prism 5.0, based on steady-state levels of the responses. In the right panel, BLi sensograms show the binding of different amounts of GST-FrzB with immobilized 6His-FrzCD or 6His-FrzE<sup>CheA</sup> for comparison. A reference subtraction (sensor reference and GST-FrzB at 10 &#x003bc;M) was applied to account for non-specific binding, background, and signal drift to minimize sensor variability. (C) The effect of the FrzB<sup>&#x003b2;4-&#x003b2;5</sup> chimera on the FrzE phosphorylation. The FrzE kinase domain (FrzE<sup>CheA</sup>) auto-phosphorylation was tested <em>in vitro</em> by incubation of FrzE<sup>CheA</sup> in the presence of FrzA, FrzB or FrzB<sup>&#x003b2;4-&#x003b2;5</sup>, the indicated different forms of FrzCD and ATP&#x003b3;P<sup>33</sup> as a phosphate donor. (D) 3D models of the FrzE<sup>P4-P5</sup>:FrzA and FrzE<sup>P4-P5</sup>:FrzB complexes based on the <em>T</em>. <em>maritima</em> CheA<sup>P4-P5</sup>:CheW crystal structure (PDB 2ch4). The 20 amino acid region forming the &#x003b2;-strands 4 and 5 of subdomain 2 and corresponding to the CheW region important for the CheW:CheA<sup>P5</sup> interaction are highlighted in red in the FrzA structure. These residues are missing in FrzB. (E) Protein structural alignments between the <em>T</em>. <em>maritima</em> CheW x-ray structure and <em>M</em>. <em>xanthus</em> FrzA and FrzB theoretical 3D models. CheW residues in atomic contact with CheA and MCP are shown as red stars and blue squares, respectively. Secondary structures determined from CheW x-ray structure are shown on top of the alignment.</p></div></div></div><p>If FrzB does not interact with FrzE, it should not be able to promote the phosphorylation of the FrzE kinase domain (FrzE<sup>CheA</sup>) <em>in vitro</em>. In fact, it has been previously shown that the presence of both FrzCD and FrzA is strictly required to achieve the transfer of a phosphoryl group from ATP to the Histidine at position 49 of FrzE<sup>CheA</sup>
<em>in vitro</em> [<a href="#pgen.1008533.ref026" rid="pgen.1008533.ref026" class=" bibr popnode">26</a>] (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g003/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g003" rid-ob="ob-pgen.1008533.g003" co-legend-rid="lgnd_pgen.1008533.g003"><span>Fig 3C</span></a>). As expected, when we replaced FrzA with FrzB in the mixture, no phosphorylation was detected, suggesting that FrzB cannot mediate the FrzE<sup>CheA</sup> autophosphorylation <em>in vitro</em> (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g003/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g003" rid-ob="ob-pgen.1008533.g003" co-legend-rid="lgnd_pgen.1008533.g003"><span>Fig 3C</span></a>).</p><p>Next, we asked what prevented FrzB from interacting with FrzE. To address this question, we decided to generate 3D homology models of the FrzE<sup>P4-P5</sup>:FrzA and FrzE<sup>P4-P5</sup>:FrzB complexes based on the <em>T</em>. <em>maritima</em> CheA<sup>P4-P5</sup>:CheW crystal structure (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g003/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g003" rid-ob="ob-pgen.1008533.g003" co-legend-rid="lgnd_pgen.1008533.g003"><span>Fig 3D</span></a>) [<a href="#pgen.1008533.ref008" rid="pgen.1008533.ref008" class=" bibr popnode">8</a>]. The structural models predict that FrzB cannot interact with FrzE due to the lack of a 20 amino acid region forming the &#x003b2;-strands 4 and 5 of subdomain 2. These strands correspond exactly to the CheW region important for the CheW: CheA<sup>P5</sup> interaction (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g003/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g003" rid-ob="ob-pgen.1008533.g003" co-legend-rid="lgnd_pgen.1008533.g003"><span>Fig 3D and 3E</span></a>) [<a href="#pgen.1008533.ref008" rid="pgen.1008533.ref008" class=" bibr popnode">8</a>]. On the other hand, and consistent with the co-purification results, FrzB is predicted to contain all the hydrophobic amino acids involved in its interaction with FrzCD (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g003/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g003" rid-ob="ob-pgen.1008533.g003" co-legend-rid="lgnd_pgen.1008533.g003"><span>Fig 3E</span></a>).</p><p>To verify if the lack of the FrzE-interacting region in FrzB was directly associated with the loss of the FrzB-FrzE interaction, we designed a chimeric FrzB protein carrying the &#x003b2;-strands 4 and 5 from FrzA (FrzB<sup>&#x003b2;4-&#x003b2;5</sup>). First, we tested if purified FrzB<sup>&#x003b2;4-&#x003b2;5</sup> was able to replace FrzA in our FrzE<sup>CheA</sup> phosphorylation <em>in vitro</em> assay. Strikingly, when FrzB<sup>&#x003b2;4-&#x003b2;5</sup> was the only CheW in the reaction mixture, it promoted the FrzE<sup>CheA</sup> phosphorylation similarly to FrzA, thus acting as a CheW (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g003/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g003" rid-ob="ob-pgen.1008533.g003" co-legend-rid="lgnd_pgen.1008533.g003"><span>Fig 3C</span></a>). The FrzE<sup>CheA</sup> phosphorylation levels were a direct consequence of the FrzCD activation state, because when we used the hyperactive FrzCDc allele in the place of its wild-type version, we observed a higher level of phosphorylated FrzE<sup>CheA</sup> (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g003/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g003" rid-ob="ob-pgen.1008533.g003" co-legend-rid="lgnd_pgen.1008533.g003"><span>Fig 3C</span></a>).</p><p class="p p-last">Then, to test if FrzB<sup>&#x003b2;4-&#x003b2;5</sup> could mediate FrzE autophosphorylation <em>in vivo</em> as <em>in vitro</em>, we transferred the gene coding for FrzB<sup>&#x003b2;4-&#x003b2;5</sup> into a <em>&#x00394;frzA &#x00394;frzB</em> strain (<a href="#pgen.1008533.s001" rid="pgen.1008533.s001" class=" supplementary-material">S1 Fig</a>) at the endogenous locus. FrzB<sup>&#x003b2;4-&#x003b2;5</sup> was stably expressed (<a href="#pgen.1008533.s002" rid="pgen.1008533.s002" class=" supplementary-material">S2 Fig</a>), and cells could reverse in the presence of IAA similarly to <em>&#x00394;frzB</em> (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g002/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g002" rid-ob="ob-pgen.1008533.g002" co-legend-rid="lgnd_pgen.1008533.g002"><span>Fig 2</span></a>). These results suggest that <em>in vivo</em>, like <em>in vitro</em>, FrzB<sup>&#x003b2;4-&#x003b2;5</sup> can connect FrzCD to FrzE.</p></div><div id="sec005" class="sec sec-last"><h3 id="sec005title">FrzB might be a stabilizing factor of multiple Frz signaling arrays</h3><p class="p p-first">It has been previously shown that Frz proteins form nucleoid-associated signaling complexes [<a href="#pgen.1008533.ref027" rid="pgen.1008533.ref027" class=" bibr popnode">27</a>,<a href="#pgen.1008533.ref028" rid="pgen.1008533.ref028" class=" bibr popnode">28</a>]. To verify whether FrzB was part of the Frz clusters, we constructed a <em>mCherry</em>-<em>frzB</em> fusion that replaced <em>frzB</em> at the endogenous locus and encoded a functional protein (<a href="#pgen.1008533.s002" rid="pgen.1008533.s002" class=" supplementary-material">S2</a> and <a href="#pgen.1008533.s003" rid="pgen.1008533.s003" class=" supplementary-material">S3</a> Figs). mCherry-FrzB formed multiple nucleoid-associated clusters that resembled those formed by FrzCD-GFP and FrzE-mCherry (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g004/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g004" rid-ob="ob-pgen.1008533.g004" co-legend-rid="lgnd_pgen.1008533.g004"><span>Fig 4A</span></a>) [<a href="#pgen.1008533.ref028" rid="pgen.1008533.ref028" class=" bibr popnode">28</a>].</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="pgen.1008533.g004" co-legend-rid="lgnd_pgen.1008533.g004"><a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g004/" target="figure" rid-figpopup="pgen.1008533.g004" rid-ob="ob-pgen.1008533.g004"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140639610422752"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6952110_pgen.1008533.g004.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pgen.1008533.g004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6952110/bin/pgen.1008533.g004.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140639610422752"><a target="object" rel="noopener" href="/pmc/articles/PMC6952110/figure/pgen.1008533.g004/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_pgen.1008533.g004"><div><a class="figpopup" href="/pmc/articles/PMC6952110/figure/pgen.1008533.g004/" target="figure" rid-figpopup="pgen.1008533.g004" rid-ob="ob-pgen.1008533.g004">Fig 4</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Localization of FrzB-mCherry in <em>M</em>. <em>xanthus</em> cells.</strong><p>(A) (Left) Micrographs of <em>mCherry-frzB</em> cells stained with the DNA DAPI stain. (Middle) mCherry (red) and DAPI fluorescence (blue) profiles are shown with the fluorescence intensity (arbitrary units) represented on the <em>y</em>-axis and the cell length positions with -1 and +1 indicating the poles, on the <em>x-</em>axis. The cell boundaries were drawn manually from the phase-contrast images. (Right) Plot of correlation coefficients of DAPI and mCherry localization. R<sup>2</sup> values &#x0003e; 0.5 indicate significant correlations. The table compares the R<sup>2</sup> values between the indicated fluorescent fusion and DAPI staining. (B) (Left) Fluorescence micrographs of the indicated <em>M</em>. <em>xanthus</em> strains carrying FrzCD-gfp fusions. The cell boundaries were drawn manually from the phase-contrast images. (Right) For each indicated strain, more than 120 cells (<em>x</em> axis) from at least two biological replicates are represented as lines and ordered according to their length (pixels) in demographs. The GFP fluorescence intensity along the cell body is represented as colored pixels at the corresponding cell position (from -1 to +1 on the <em>y</em> axis). &#x0201c;0&#x0201d; is the cell center. On the right, a scale indicates the fluorescence intensity and the corresponding colors.</p></div></div></div><p class="p p-last">Then, we decided to check how the absence of FrzA and FrzB affected the formation of these complexes. For that, we transferred a <em>frzCD-gfp</em> gene fusion [<a href="#pgen.1008533.ref028" rid="pgen.1008533.ref028" class=" bibr popnode">28</a>] into <em>&#x00394;frzA</em> and <em>&#x00394;frzB</em> mutants. <em>M</em>. <em>xanthus</em> cells of the resulting strains were then imaged by fluorescence microscopy. While FrzCD-GFP was completely dispersed in the cytoplasm in the absence of FrzA, this fusion protein formed a single nucleoid-associated cluster in the absence of FrzB (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g004/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g004" rid-ob="ob-pgen.1008533.g004" co-legend-rid="lgnd_pgen.1008533.g004"><span>Fig 4B</span></a>). These results suggest that, even if FrzB is dispensable for the assembly of Frz chemosensory arrays, it is important for the formation of multiple distributed clusters on the nucleoid. These clusters collapse into a single cluster in the absence of FrzB. Of 230 cells examined, 82% have a single cluster, whereas 18% have no cluster. It is possible that, in the absence of FrzB and during cell division, only one daughter cell inherits the single Frz array and the other one will synthesize a new one. This would explain the lack of clusters in a small population of cells. In the wild type, the multiple Frz clusters would be expected to be distributed to both daughter cells [<a href="#pgen.1008533.ref028" rid="pgen.1008533.ref028" class=" bibr popnode">28</a>]. Finally, we wanted to verify if FrzB<sup>&#x003b2;4-&#x003b2;5</sup> could mediate FrzCD-GFP cluster formation like FrzA. When FrzB<sup>&#x003b2;4-&#x003b2;5</sup> is the only CheW-like of the system, FrzCD-GFP could form clusters even if these clusters were less compact and bright than wild type (<a href="#pgen.1008533.s004" rid="pgen.1008533.s004" class=" supplementary-material">S4 Fig</a>). This behavior might be explained by the fact that, while FrzB<sup>&#x003b2;4-&#x003b2;5</sup> can interact with FrzE, such interactions might be not strong enough (like those of FrzA with FrzE) to allow the formation of clusters as those seen in wild type or when only FrzA is present. In the absence of both FrzA and FrzB, FrzCD-GFP was diffused in the cytoplasm, as in <em>&#x00394;frzA</em> (<a href="#pgen.1008533.s004" rid="pgen.1008533.s004" class=" supplementary-material">S4 Fig</a>). The aberrant localization patterns were not due to a change in the FrzCD-GFP protein levels nor in a change of the nucleoid structure (<a href="#pgen.1008533.s005" rid="pgen.1008533.s005" class=" supplementary-material">S5</a> and <a href="#pgen.1008533.s006" rid="pgen.1008533.s006" class=" supplementary-material">S6</a> Figs). Interestingly, and by yet unknown mechanisms, FrzA not only is essential for Frz cluster formation, but it also allows the association of FrzCD to the nucleoid (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g004/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g004" rid-ob="ob-pgen.1008533.g004" co-legend-rid="lgnd_pgen.1008533.g004"><span>Fig 4</span></a>).</p></div></div><div id="sec006" class="tsec sec"><h2 class="head no_bottom_margin" id="sec006title">Discussion</h2><p class="p p-first">Chemosensory systems are modified two-component systems in which the addition of accessory proteins confers unique properties like signal amplification and sensitivity. In this work, we found that the CheW-like protein FrzB lacks a 20 amino acid sequence corresponding to two &#x003b2;-strands that are important for CheA<sup>P5</sup>:CheW interactions [<a href="#pgen.1008533.ref008" rid="pgen.1008533.ref008" class=" bibr popnode">8</a>]. Thus, FrzB cannot interact with FrzE and mediate its autophosphorylation. The loss of this protein region might have been selected to confer to this small protein the unique function of allowing the formation and distribution of multiple nucleoid Frz chemosensory arrays. How would FrzB accomplish these tasks? It has been shown that Che arrays are constituted by hexagons of MCP trimers of dimers networked by CheA<sup>P5</sup>-CheW rings [<a href="#pgen.1008533.ref011" rid="pgen.1008533.ref011" class=" bibr popnode">11</a>]. By homology, we believe that Frz proteins can have such hexagonal organization using the nucleoid as scaffold. However, if FrzB cannot interact with FrzE<sup>P5</sup>, it cannot participate to the formation of CheA<sup>P5</sup>-CheW rings either. In addition to CheA<sup>P5</sup>-CheW rings, the formation of Che networks implies the formation of six-member CheW rings that only interact with the MCPs (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g001/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g001" rid-ob="ob-pgen.1008533.g001" co-legend-rid="lgnd_pgen.1008533.g001"><span>Fig 1</span></a>) [<a href="#pgen.1008533.ref017" rid="pgen.1008533.ref017" class=" bibr popnode">17</a>,<a href="#pgen.1008533.ref029" rid="pgen.1008533.ref029" class=" bibr popnode">29</a>,<a href="#pgen.1008533.ref030" rid="pgen.1008533.ref030" class=" bibr popnode">30</a>]. The 6-CheW rings have been visualized by cryoEM, and their suggested function is to stabilize the Che arrays by favoring allosteric interactions, like CheW-CheA rings, but without a direct effect on signaling. We can imagine that 6-FrzB rings (in addition to FrzE<sup>P5</sup>-FrzA rings and 6-FrzA rings) might further stabilize small Frz clusters, which would otherwise collapse into one single cluster (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g005/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g005" rid-ob="ob-pgen.1008533.g005" co-legend-rid="lgnd_pgen.1008533.g005"><span>Fig 5</span></a>). Alternatively FrzB allows the formation of small Frz clusters by breaking the networking of MCP trimers of dimers with Che<sup>P5</sup>-CheW rings (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g005/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g005" rid-ob="ob-pgen.1008533.g005" co-legend-rid="lgnd_pgen.1008533.g005"><span>Fig 5</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="pgen.1008533.g005" co-legend-rid="lgnd_pgen.1008533.g005"><a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g005/" target="figure" rid-figpopup="pgen.1008533.g005" rid-ob="ob-pgen.1008533.g005"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140639646666800"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6952110_pgen.1008533.g005.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pgen.1008533.g005.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6952110/bin/pgen.1008533.g005.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140639646666800"><a target="object" rel="noopener" href="/pmc/articles/PMC6952110/figure/pgen.1008533.g005/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_pgen.1008533.g005"><div><a class="figpopup" href="/pmc/articles/PMC6952110/figure/pgen.1008533.g005/" target="figure" rid-figpopup="pgen.1008533.g005" rid-ob="ob-pgen.1008533.g005">Fig 5</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Proposed models on how FrzB might confer plasticity to Frz clusters.</strong><p>(A) FrzB participates to the formation of 6-CheW rings to stabilize small Frz clusters, which would otherwise collapse in one single cluster (C). (B) Alternatively FrzB allows the formation of small Frz clusters by breaking the networking of MCP trimers of dimers with Che<sup>P5</sup>-CheW rings.</p></div></div></div><p>Interestingly, the FrzCD receptor does not contain an obvious ligand-binding domain. Instead, it contains an N-terminal region that has been recently shown to be involved in DNA binding [<a href="#pgen.1008533.ref028" rid="pgen.1008533.ref028" class=" bibr popnode">28</a>]. Thus, FrzB could also function to deliver signals to the Frz pathway. FrzB might bind directly to some cytoplasmic molecules and transduce signals to FrzCD, thus functioning as a sensor of the cellular metabolic state. However, FrzB could also connect FrzCD to another MCP or sensor kinase. <em>M</em>. <em>xanthus</em> encodes a total of 21 MCPs [<a href="#pgen.1008533.ref031" rid="pgen.1008533.ref031" class=" bibr popnode">31</a>]. Among them, only Mcp7 is cytoplasmic like FrzCD, but its localization in a single cluster randomly localized within cells, strongly excludes an association with the Frz pathway [<a href="#pgen.1008533.ref031" rid="pgen.1008533.ref031" class=" bibr popnode">31</a>]. We could imagine that FrzCD, which forms clusters on the nucleoid [<a href="#pgen.1008533.ref028" rid="pgen.1008533.ref028" class=" bibr popnode">28</a>], could associate with a transmembrane MCP or sensor kinase at regions where the nucleoid surface is particularly close to the inner membrane [<a href="#pgen.1008533.ref032" rid="pgen.1008533.ref032" class=" bibr popnode">32</a>]. In fact, previous work shows the association of FrzCD with membrane fractions [<a href="#pgen.1008533.ref033" rid="pgen.1008533.ref033" class=" bibr popnode">33</a>] as well as interactions with envelope-associated A-motility proteins [<a href="#pgen.1008533.ref034" rid="pgen.1008533.ref034" class=" bibr popnode">34</a>,<a href="#pgen.1008533.ref035" rid="pgen.1008533.ref035" class=" bibr popnode">35</a>]. The hypothesis that FrzB is a Frz input does not rule out that FrzCD itself can somehow bind signals. Indeed, FrzCD responds to IAA independently of FrzB (<a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g002/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g002" rid-ob="ob-pgen.1008533.g002" co-legend-rid="lgnd_pgen.1008533.g002"><span>Fig 2</span></a>).</p><p class="p p-last">In recent years, the discovery of different accessory Che proteins allowed the identification of new functions and degrees of complexity in the regulation of the cellular responses operated by chemosensory systems. By characterizing a new accessory protein and its function, this work opens new insights on the knowledge of the regulatory potentials of bacterial chemosensory systems.</p></div><div id="sec007" class="tsec sec"><h2 class="head no_bottom_margin" id="sec007title">Materials and methods</h2><div id="sec008" class="sec sec-first"><h3 id="sec008title">Bacterial strains, plasmids and growth</h3><p class="p p-first"><em>S</em>trains and plasmids are listed in <a href="#pgen.1008533.s007" rid="pgen.1008533.s007" class=" supplementary-material">S1</a> and <a href="#pgen.1008533.s008" rid="pgen.1008533.s008" class=" supplementary-material">S2</a> Tables, respectively. <em>M</em>. <em>xanthus</em> strains were grown at 32&#x000b0;C in CYE rich media as previously described.</p><p>Plasmids were introduced into <em>M</em>. <em>xanthus</em> cells by electroporation. Deletions and GFP fusions were inserted in frame, using the pBJ113 or pBJ114 vectors, to avoid polar effects on the downstream gene expression [<a href="#pgen.1008533.ref022" rid="pgen.1008533.ref022" class=" bibr popnode">22</a>,<a href="#pgen.1008533.ref031" rid="pgen.1008533.ref031" class=" bibr popnode">31</a>].</p><p>Strains EM736 and EM743 were obtained by deleting <em>frzA</em> from codon 15 to 145 and <em>frzB</em> from codon 11 to 92, respectively. In both strains we also deleted <em>frzCD</em> from codon 6 to 182. Strain EM742 was obtained by deleting <em>frzA</em> and <em>frzB</em> as above.</p><p>To construct plasmid pEM399 to generate EM515, a DNA fragment was generated by PCR and by overlapping a DNA region amplified with primers CCCAAGCTTCTTCAGCTCGAGGCTGGCAA and TGCCCCTTGCTCACCATCTCCTCCTCCTTCTCATGGAA and <em>mCherry</em> amplified with primers CATGAGAAGGAGGAGGAGATGGTGAGCAAGGGCGAGGA and CGGGATCCTCTAGACTTGCACAGCTCGTCCATGC from pEM147 [<a href="#pgen.1008533.ref034" rid="pgen.1008533.ref034" class=" bibr popnode">34</a>]. This fusion product was digested with HindIII and BamHI. <em>frzB</em> was amplified with primers CGGGATCCGGAGCAGGAGACCTCCTCTTCTTCGACAT and GGAATTCGCGGACGAGGACAGCCTCAA and digested with BamHI and EcoRI. The two fragments were then cloned into pBJ113 previously digested with HindIII and BamHI in a three-partner ligation reaction.</p><p>To obtain strains EM440 and EM512, plasmid pDPA20 was transferred into strains DZ4479 and DZ4478 respectively.</p><p>For the construction of <em>frzB</em><sup><em>&#x003b2;4-&#x003b2;5</em></sup> we replaced codon 27 to 36 of FrzB with codon 41 to 75 of FrzA. Plasmids pEM463 and was synthesized by Biomatik.</p><p class="p p-last"><em>Escherichia coli</em> cells were grown under standard laboratory conditions in LB broth supplemented with antibiotics if necessary.</p></div><div id="sec009" class="sec"><h3 id="sec009title">Phenotypic analysis</h3><p class="p p-first-last">Developmental assays were performed by spotting 10 &#x003bc;l of cells at 5 OD<sub>600</sub> directly onto CF 1.5% agar plates. Vegetative swarming phenotypes were analyzed by spotting the same amount of cells onto CYE plates containing a concentration of 0.5% agar. The IAA response of cell colonies was analyzed by spotting cells (10 &#x003bc;l of cells at 5 OD<sub>600</sub>) on CYE plates supplied with 0.15% of IAA and containing a concentration of 1.5% agar. After incubation at 32&#x000b0;C the swarming colonies and the fruiting bodies were photographed at an Olympus SZ61 stereoscope with a Nikon DXM1200 Digital Camera.</p></div><div id="sec010" class="sec"><h3 id="sec010title">Reversal frequencies</h3><p class="p p-first-last">5 &#x003bc;l of cells from 4 x 10<sup>8</sup> cfu ml<sup>-1</sup> vegetative CYE cultures were spotted on a thin fresh TPM agar supplied or not with 0.15% IAA. Time-lapse movies were shot for 1 hour with frames captured every 30 seconds. Movies were obtained and reversal frequencies calculated with Image J (Rasband, W.S., ImageJ, U. S. National Institutes of Health, Bethesda, Maryland, USA, <a href="http://imagej.nih.gov/ij/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://imagej.nih.gov/ij/</a>, 1997&#x02013;2012) and FIJI as previously described [<a href="#pgen.1008533.ref020" rid="pgen.1008533.ref020" class=" bibr popnode">20</a>]. For non-reversing strains, the number of reversals for each cell was plotted against time using the R software (<a href="https://www.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.r-project.org/</a>). For strains that frequently reversed, the number of periods between two reversals were plotted against time using the R software. Reversal frequencies were measured from cells issued from two to six biological replicates. Statistical significance was obtained with a Wilcox Test from R.</p></div><div id="sec011" class="sec"><h3 id="sec011title">Protein purification and co-purification</h3><p class="p p-first">BL21(DE3) [F&#x02013;ompT hsdSB(rB&#x02013;mB&#x02013;) gal dcm (DE3)] cells were grown in LB broth supplemented 100 &#x003bc;g/ml ampicillin to mid-exponential phase at 37&#x000b0;C. Overexpression was induced by adding 0,2 mM IPTG. Cells were then grown at 16&#x000b0;C over night. Cells were washed and resuspended in lysis buffer (50 mM TrisHCl, pH 8; 300 mM NaCl; 100 &#x003bc;g/ml PMSF; 30 U/mL Benzonase) and lysed at the French press. The cell lysates were centrifuged at 4&#x000b0;C for 20 min at 18000&#x000d7; rpm.</p><p>For BLi experiments, soluble 6His-tagged proteins were purified using a NiNTA resine (GE Healthcare) and imidazole was removed by dialysis into a buffer containing 50 mM TrisHCl, pH 8 and 300 mM NaCl. GST-proteins were purified by GST-affinity chromatography using the same lysing buffer supplied of 10 mM reduced glutathione to elute.</p><p>For co-purification experiments, <em>E</em>. <em>coli</em> cells expressing each of GST-FrzA, GST-FrzB, 6His-FrzCD or 6His-FrzECheA were mixed at one to one ratio to obtain the following combinations: GST-FrzA: 6His-FrzCD, GST-FrzA: 6His- FrzECheA, GST-FrzB: 6His-FrzCD, GST-FrzB: 6His- FrzECheA. Mixed cell cultures were resuspended in lysis buffer (50 mM TrisHCl, pH 8; 300 mM NaCl; 100 &#x003bc;g/ml PMSF; 30 U/mL Benzonase) and lysed at the French press. The cell lysates were centrifuged at 4&#x000b0;C for 20 min at 18000&#x000d7; rpm.</p><p class="p p-last">Proteins were, then, co-purified by GST-affinity chromatography as described above.</p></div><div id="sec012" class="sec"><h3 id="sec012title">Biolayer interpherometry</h3><p class="p p-first-last">Protein-protein interaction experiments were conducted at 25&#x000b0;C with the BLItz instrument from ForteBio (Menlo Park, CA, USA). The BLI consists in a real time optical biosensing technique exploits the interference pattern of white light reflected from two surfaces to measure biomolecular interactions [<a href="#pgen.1008533.ref025" rid="pgen.1008533.ref025" class=" bibr popnode">25</a>]. Purified 6His-FrzCD or 6His-FrzE<sup>CheA</sup> protein ligands were immobilized onto two different Ni-NTA biosensors (ForteBio) in duplicate at 4 &#x003bc;M concentrations. GST-FrzB was used as the analyte throughout the study at concentrations ranging from 0.2 to 20 &#x003bc;M. The assay was conducted in buffer containing Tris HCl 50 mM pH8 and NaCl 300 mM. The binding reactions were performed with an initial baseline during 30 seconds, an association step at 120 seconds and a dissociation step of 120 seconds with lateral shaking at 2200rpm. A double reference subtraction (sensor reference and GST-FrzB at 10 &#x003bc;M) was applied to account for non-specific binding, background, and signal drift to minimize sensor variability.</p></div><div id="sec013" class="sec"><h3 id="sec013title">Sequence alignment and homology modeling</h3><p class="p p-first-last">Predictions of secondary structures and protein sequence alignments were obtained with Jpred [<a href="#pgen.1008533.ref036" rid="pgen.1008533.ref036" class=" bibr popnode">36</a>] and Clustal Omega [<a href="#pgen.1008533.ref037" rid="pgen.1008533.ref037" class=" bibr popnode">37</a>], respectively. Homology models of FrzA (Uniprot <a id="__tag_826266571" class="tag_hotlink" href="/protein/P43498" ref="/protein/P43498"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"P43498","term_id":"1169746","term_text":"P43498"}}</span>P43498</a>) or FrzB (Uniprot <a id="__tag_826266569" class="tag_hotlink" href="/protein/P43499" ref="/protein/P43499"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"P43499","term_id":"1169747","term_text":"P43499"}}</span>P43499</a>) in the presence of FrzE (Uniprot <a id="__tag_826266570" class="tag_hotlink" href="/protein/P18769" ref="/protein/P18769"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"P18769","term_id":"120546","term_text":"P18769"}}</span>P18769</a>) were built with MODELLER 9v12 [<a href="#pgen.1008533.ref038" rid="pgen.1008533.ref038" class=" bibr popnode">38</a>]. Crystal structure of the complex between bacterial chemotaxis histidine kinase CheA<sup>P4-P5</sup> and receptor-adaptor protein CheW (PDB codes 2CH4) was used as template [<a href="#pgen.1008533.ref039" rid="pgen.1008533.ref039" class=" bibr popnode">39</a>]. The modeling was performed in a two-step manner: First, each individual domain was modeled separately using the 3D structure of its <em>T</em>. <em>maritima</em> counterpart as a template. CheW (2CH4 chain W) was used to build models of FrzA and FrzB. The N-terminal domain of CheA (2CH4 chain A resid 355&#x02013;540) and the C-terminal domain of CheA (2CH4 chain A resid 541&#x02013;671) were used to build N- and C-terminal domains of FrzE. In a second step, each model was superimposed to its counterpart in the x-ray structure of the CheW-CheA complex (chains W and A). The Frz proteins exhibit high enough identities to their Che counterparts to build reasonable 3D models. FrzB vs CheW (26.1% identity, 45% similarity), FrzA vs CheW (26.6% identity, 46.7% similarity), FrzE<sup>p4</sup> vs. CheA<sup>p4</sup> (46.5% identity, 71.7% similarity), FrzE<sup>p5</sup> vs. CheA<sup>p5</sup> (26.1% identity, 53.8% similarity). Quality of the models was assessed with Prosess server (<a href="http://www.prosess.ca/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.prosess.ca</a>). Structural alignments were performed with Protein structure comparison service PDBeFold at European Bioinformatics Institute (<a href="http://www.ebi.ac.uk/msd-srv/ssm" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ebi.ac.uk/msd-srv/ssm</a>) [<a href="#pgen.1008533.ref040" rid="pgen.1008533.ref040" class=" bibr popnode">40</a>]. <a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g003/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g003" rid-ob="ob-pgen.1008533.g003" co-legend-rid="lgnd_pgen.1008533.g003"><span>Fig 3C</span></a> showing the lowest energy model was generated with pymol (<a href="http://pymol.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pymol.org</a>). <a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g003/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g003" rid-ob="ob-pgen.1008533.g003" co-legend-rid="lgnd_pgen.1008533.g003"><span>Fig 3D</span></a> was generated with ESPript web server [<a href="#pgen.1008533.ref041" rid="pgen.1008533.ref041" class=" bibr popnode">41</a>].</p></div><div id="sec014" class="sec"><h3 id="sec014title">Fluorescence microscopy and image analysis</h3><p class="p p-first">For fluorescence microscopy analyses, 5 &#x003bc;l of cells from 4 x 10<sup>8</sup> cfu ml<sup>-1</sup> vegetative CYE cultures were spotted on a thin fresh TPM agar pad at the top a slide [<a href="#pgen.1008533.ref042" rid="pgen.1008533.ref042" class=" bibr popnode">42</a>]. A cover slip was added immediately on the top of the pad, and the obtained slide was analyzed by microscopy using a Nikon Eclipse TE2000 E PFS inverted epifluorescence microscope (100 x oil objective NA 1.3 Phase Contrast) [<a href="#pgen.1008533.ref043" rid="pgen.1008533.ref043" class=" bibr popnode">43</a>].</p><p class="p p-last">To study the co-localization with the DNA, the TPM agar pads were supplied with 1&#x003bc;g ml<sup>-1</sup> DAPI stain as shown by Moine et al. [<a href="#pgen.1008533.ref028" rid="pgen.1008533.ref028" class=" bibr popnode">28</a>]. FrzB clusters, nucleoid areas and cell areas were automatically detected and verified manually with the &#x0201c;MicrobeJ&#x0201d; Fiji/ImageJ plugin created by A. Ducret, Brun Lab (<a href="http://www.microbej.com/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.microbej.com</a>). All data plots and statistical tests were obtained with the R software (<a href="https://www.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.r-project.org/</a>).</p></div><div id="sec015" class="sec sec-last"><h3 id="sec015title"><em>In vitro</em> autophosphorylation assay</h3><p class="p p-first-last"><em>In vitro</em> phosphorylation assays were performed with <em>E</em>. <em>coli</em> purified recombinant proteins. 4 &#x003bc;g of FrzE<sup>CheA</sup> was incubated with 1 &#x003bc;g of FrzA, FrzB or FrzB<sup>&#x003b2;4-&#x003b2;5</sup> and FrzCD or FrzCD<sup>c</sup> in 25 &#x003bc;l of buffer P (50 mM Tris-HCl, pH 7.5; 1 mM DTT; 5 mM MgCl<sub>2</sub>; 50 mM KCl; 5 mM EDTA; 50 &#x003bc;M ATP, 10% (v/v) glycerol) supplemented with 200 &#x003bc;Ci ml<sup>-1</sup> (65 nM) of [&#x003b3;-33P]ATP (PerkinElmer, 3000 Ci mmol<sup>-1</sup>) for 10 minutes at room temperature in order to obtain the optimal FrzE<sup>CheA</sup> autophosphorylation activity. Each reaction mixture was stopped by addition of 5 &#x000d7; Laemmli and quickly loaded onto SDS-PAGE gel. After electrophoresis, proteins were revealed using Coomassie Brilliant Blue before gel drying. Radioactive proteins were visualized by autoradiography using direct exposure to film (Carestream).</p></div></div><div id="sec016" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="sec016title">Supporting information</h2><!--/article/body/sec/--><div class="sec suppmat" id="pgen.1008533.s001"><h4>S1 Fig</h4><p>(A) Motility and fruiting body formation phenotypes were photographed at 48h and 72h, respectively. (B) Box plots of reversal frequencies of single cells moving on agar pads supplemented or not with 0.15% IAA. The lower and upper boundaries of the boxes correspond to 25% and 75% percentiles, respectively. The median is shown as a line at the center of each box and whiskers represents the 10% and 90% percentiles. For the reversal frequency measurements, cells issued from two biological replicates, were used. For WT approximately 500 cells, issued from six biological replicates, were used.</p><p>(PDF)</p><div class="sup-box half_rhythm" id="media-a.u.e.b.c"><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s001.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(1.1M, pdf)</sup></div></div><div class="sec suppmat" id="pgen.1008533.s002"><h4>S2 Fig</h4><!--caption a9--><strong>Western blot with anti-FrzB antibodies on the cell extracts of the indicated <em>M</em>. <em>xanthus</em> strains. </strong><p>The <em>pilA</em>::<em>frzB</em> strain, used as positive control, expresses <em>frzB</em> under the control of the <em>pilA</em> promoter for high expression. The white line is used to indicate that two lanes from the same gel were separated by other lanes in the original gel.</p><p>(PDF)</p><div class="sup-box half_rhythm" id="media-a.u.e.c.c"><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s002.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(158K, pdf)</sup></div></div><div class="sec suppmat" id="pgen.1008533.s003"><h4>S3 Fig</h4><!--caption a9--><strong>Motility and fruiting body formation phenotypes of the indicated strains were photographed at 48h and 72h, respectively. </strong><p>(PDF)</p><div class="sup-box half_rhythm" id="media-a.u.e.d.c"><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s003.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(215K, pdf)</sup></div></div><div class="sec suppmat" id="pgen.1008533.s004"><h4>S4 Fig</h4><p>(Left) Fluorescence micrographs of the indicated <em>M</em>. <em>xanthus</em> strains carrying FrzCD-gfp fusions. The cell boundaries were drawn manually from the phase-contrast images. (Right) For each indicated strain, more than 120 cells (<em>x</em> axis) from at least two biological replicates are represented as lines and ordered according to their length (pixels) in demographs. The GFP fluorescence intensity along the cell body is represented as colored pixels at the corresponding cell position (from -1 to +1 on the <em>y</em> axis). &#x0201c;0&#x0201d; is the cell center. On the right, a scale indicates the fluorescence intensity and the corresponding colors.</p><p>(PDF)</p><div class="sup-box half_rhythm" id="media-a.u.e.e.c"><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s004.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(219K, pdf)</sup></div></div><div class="sec suppmat" id="pgen.1008533.s005"><h4>S5 Fig</h4><!--caption a9--><strong>Western blot with anti-FrzCD antibodies on the cell extracts of the indicated <em>M</em>. <em>xanthus</em> strains. </strong><p>(PDF)</p><div class="sup-box half_rhythm" id="media-a.u.e.f.c"><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s005.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(1.1M, pdf)</sup></div></div><div class="sec suppmat" id="pgen.1008533.s006"><h4>S6 Fig</h4><p>(A) Cells of the indicated strains were incubated 30 minutes with DAPI and then imaged at the fluorescence microscope. The nucleoid and cell surfaces were measured automatically with Microbe J [<a href="#pgen.1008533.ref044" rid="pgen.1008533.ref044" class=" bibr popnode">44</a>]. (B) The ratio between the nucleoid and cell surface was then calculated for each cell and averages values plotted. The numbers of analyzed cells are indicated per each strain. Cells were issued from two independent biological replicates.</p><p>(PDF)</p><div class="sup-box half_rhythm" id="media-a.u.e.g.c"><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s006.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(913K, pdf)</sup></div></div><div class="sec suppmat" id="pgen.1008533.s007"><h4>S1 Table</h4><!--caption a9--><strong>List of strains used in this work. </strong><p>(PDF)</p><div class="sup-box half_rhythm" id="media-a.u.e.h.c"><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s007.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(50K, pdf)</sup></div></div><div class="sec suppmat" id="pgen.1008533.s008"><h4>S2 Table</h4><!--caption a9--><strong>List of plasmids used in this work. </strong><p>(PDF)</p><div class="sup-box half_rhythm" id="media-a.u.e.i.c"><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s008.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(60K, pdf)</sup></div></div><div class="sec suppmat" id="pgen.1008533.s009"><h4>S3 Table</h4><!--caption a9--><strong>Data to generate the mean R<sup>2</sup> on <a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g004/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g004" rid-ob="ob-pgen.1008533.g004" co-legend-rid="lgnd_pgen.1008533.g004"><span>Fig 4A</span></a> to measure the colocalization of mCherry-FrzB with the nucleoid. </strong><p>(XLSX)</p><div class="sup-box half_rhythm" id="media-a.u.e.j.c"><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s009.xlsx" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(44K, xlsx)</sup></div></div><div class="sec suppmat" id="pgen.1008533.s010"><h4>S4 Table</h4><!--caption a9--><strong>Raw data to generate the histogram on <a href="#pgen.1008533.s006" rid="pgen.1008533.s006" class=" supplementary-material">S6 Fig</a>. </strong><p>(CSV)</p><div class="sup-box half_rhythm" id="media-a.u.e.k.c"><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s010.csv" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(20K, csv)</sup></div></div><div class="sec suppmat" id="pgen.1008533.s011"><h4>S5 Table</h4><!--caption a9--><strong>Raw data to generate the box plots for non-reversing strains on <a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g001/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g001" rid-ob="ob-pgen.1008533.g001" co-legend-rid="lgnd_pgen.1008533.g001"><span>Fig 1B</span></a>. </strong><p>(CSV)</p><div class="sup-box half_rhythm" id="media-a.u.e.l.c"><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s011.csv" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(4.4K, csv)</sup></div></div><div class="sec suppmat" id="pgen.1008533.s012"><h4>S6 Table</h4><!--caption a9--><strong>Raw data to generate the box plots for reversing strains on <a href="/pmc/articles/PMC6952110/figure/pgen.1008533.g001/" target="figure" class="fig-table-link figpopup" rid-figpopup="pgen.1008533.g001" rid-ob="ob-pgen.1008533.g001" co-legend-rid="lgnd_pgen.1008533.g001"><span>Fig 1B</span></a>. </strong><p>(CSV)</p><div class="sup-box half_rhythm" id="media-a.u.e.m.c"><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s012.csv" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(2.6M, csv)</sup></div></div><div class="sec suppmat" id="pgen.1008533.s013"><h4>S7 Table</h4><!--caption a9--><strong>Raw data to generate the box plots for non-reversing strains on <a href="#pgen.1008533.s001" rid="pgen.1008533.s001" class=" supplementary-material">S1B Fig</a>. </strong><p>(CSV)</p><div class="sup-box half_rhythm" id="media-a.u.e.n.c"><a href="/pmc/articles/PMC6952110/bin/pgen.1008533.s013.csv" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(13K, csv)</sup></div></div></div><div id="ack-a.v.a" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.v.atitle">Acknowledgments</h2><div class="sec"><p>We thank Romain Mercier, Doroth&#x000e9;e Murat and Laetitia My for their support and helpful discussions. We also thank Anne-Valerie Le Gall for her technical support. We thank the anonymous reviewers, whose comments helped to greatly improve the manuscript.</p></div></div><div id="funding-group-a.t.b.y" class="tsec sec"><h2 class="head no_bottom_margin" id="funding-group-a.t.b.ytitle">Funding Statement</h2><p>Research on chemotaxis in our laboratory is funded by the Agence National de la Recherche Jeune Chercheur-Jeune Chercheuse (ANR-14-CE11-0023-01) to EMFM (<a href="http://www.agence-nationale-recherche.fr/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://www.agence-nationale-recherche.fr/</a>). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></div><div id="notes-a.t.c" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.t.ctitle">Data Availability</h2><p>All relevant data are within the manuscript and its Supporting Information files.</p></div><div id="ref-list-a.v.b" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.v.btitle">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref001">1. <span class="mixed-citation">Sourjik V, Berg HC. <span class="ref-title">Binding of the Escherichia coli response regulator CheY to its target measured in vivo by fluorescence resonance energy transfer</span>. <span class="ref-journal">Proc Natl Acad Sci U S A</span>. 2002;<span class="ref-vol">99</span>: 12669&#x02013;12674. 10.1073/pnas.192463199
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC130518/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12232047" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.192463199" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Binding+of+the+Escherichia+coli+response+regulator+CheY+to+its+target+measured+in+vivo+by+fluorescence+resonance+energy+transfer&amp;author=V+Sourjik&amp;author=HC+Berg&amp;volume=99&amp;publication_year=2002&amp;pages=12669-12674&amp;pmid=12232047&amp;doi=10.1073/pnas.192463199&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref002">2. <span class="mixed-citation">Berleman JE, Bauer CE. <span class="ref-title">Involvement of a Che-like signal transduction cascade in regulating cyst cell development in Rhodospirillum centenum</span>. <span class="ref-journal">Mol Microbiol</span>. 2005;<span class="ref-vol">56</span>: 1457&#x02013;1466. 10.1111/j.1365-2958.2005.04646.x
 [<a href="https://pubmed.ncbi.nlm.nih.gov/15916598" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fj.1365-2958.2005.04646.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Microbiol&amp;title=Involvement+of+a+Che-like+signal+transduction+cascade+in+regulating+cyst+cell+development+in+Rhodospirillum+centenum&amp;author=JE+Berleman&amp;author=CE+Bauer&amp;volume=56&amp;publication_year=2005&amp;pages=1457-1466&amp;pmid=15916598&amp;doi=10.1111/j.1365-2958.2005.04646.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref003">3. <span class="mixed-citation">Bible A, Russell MH, Alexandre G. <span class="ref-title">The Azospirillum brasilense Che1 chemotaxis pathway controls swimming velocity, which affects transient cell-to-cell clumping</span>. <span class="ref-journal">J Bacteriol</span>. 2012;<span class="ref-vol">194</span>: 3343&#x02013;3355. 10.1128/JB.00310-12
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3434747/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22522896" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FJB.00310-12" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=The+Azospirillum+brasilense+Che1+chemotaxis+pathway+controls+swimming+velocity,+which+affects+transient+cell-to-cell+clumping&amp;author=A+Bible&amp;author=MH+Russell&amp;author=G+Alexandre&amp;volume=194&amp;publication_year=2012&amp;pages=3343-3355&amp;pmid=22522896&amp;doi=10.1128/JB.00310-12&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref004">4. <span class="mixed-citation">Collins KD, Lacal J, Ottemann KM. <span class="ref-title">Internal sense of direction: sensing and signaling from cytoplasmic chemoreceptors</span>. <span class="ref-journal">Microbiol Mol Biol Rev MMBR</span>. 2014;<span class="ref-vol">78</span>: 672&#x02013;684. 10.1128/MMBR.00033-14
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4248653/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25428939" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FMMBR.00033-14" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol+Mol+Biol+Rev+MMBR&amp;title=Internal+sense+of+direction:+sensing+and+signaling+from+cytoplasmic+chemoreceptors&amp;author=KD+Collins&amp;author=J+Lacal&amp;author=KM+Ottemann&amp;volume=78&amp;publication_year=2014&amp;pages=672-684&amp;pmid=25428939&amp;doi=10.1128/MMBR.00033-14&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref005">5. <span class="mixed-citation">Bilwes AM, Alex LA, Crane BR, Simon MI. <span class="ref-title">Structure of CheA, a signal-transducing histidine kinase</span>. <span class="ref-journal">Cell</span>. 1999;<span class="ref-vol">96</span>: 131&#x02013;141. 10.1016/s0092-8674(00)80966-6
 [<a href="https://pubmed.ncbi.nlm.nih.gov/9989504" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fs0092-8674(00)80966-6" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Structure+of+CheA,+a+signal-transducing+histidine+kinase&amp;author=AM+Bilwes&amp;author=LA+Alex&amp;author=BR+Crane&amp;author=MI+Simon&amp;volume=96&amp;publication_year=1999&amp;pages=131-141&amp;pmid=9989504&amp;doi=10.1016/s0092-8674(00)80966-6&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref006">6. <span class="mixed-citation">Griswold IJ, Zhou H, Matison M, Swanson RV, McIntosh LP, Simon MI, et al. 
<span class="ref-title">The solution structure and interactions of CheW from Thermotoga maritima</span>. <span class="ref-journal">Nat Struct Biol</span>. 2002;<span class="ref-vol">9</span>: 121&#x02013;125. 10.1038/nsb753
 [<a href="https://pubmed.ncbi.nlm.nih.gov/11799399" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnsb753" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Struct+Biol&amp;title=The+solution+structure+and+interactions+of+CheW+from+Thermotoga+maritima&amp;author=IJ+Griswold&amp;author=H+Zhou&amp;author=M+Matison&amp;author=RV+Swanson&amp;author=LP+McIntosh&amp;volume=9&amp;publication_year=2002&amp;pages=121-125&amp;pmid=11799399&amp;doi=10.1038/nsb753&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref007">7. <span class="mixed-citation">Reebye V, Frilling A, Hajitou A, Nicholls JP, Habib NA, Mintz PJ. <span class="ref-title">A perspective on non-catalytic Src homology (SH) adaptor signalling proteins</span>. <span class="ref-journal">Cell Signal</span>. 2012;<span class="ref-vol">24</span>: 388&#x02013;392. 10.1016/j.cellsig.2011.10.003
 [<a href="https://pubmed.ncbi.nlm.nih.gov/22024281" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cellsig.2011.10.003" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Signal&amp;title=A+perspective+on+non-catalytic+Src+homology+(SH)+adaptor+signalling+proteins&amp;author=V+Reebye&amp;author=A+Frilling&amp;author=A+Hajitou&amp;author=JP+Nicholls&amp;author=NA+Habib&amp;volume=24&amp;publication_year=2012&amp;pages=388-392&amp;pmid=22024281&amp;doi=10.1016/j.cellsig.2011.10.003&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref008">8. <span class="mixed-citation">Li X, Fleetwood AD, Bayas C, Bilwes AM, Ortega DR, Falke JJ, et al. 
<span class="ref-title">The 3.2 &#x000c5; resolution structure of a receptor: CheA:CheW signaling complex defines overlapping binding sites and key residue interactions within bacterial chemosensory arrays</span>. <span class="ref-journal">Biochemistry</span>. 2013;<span class="ref-vol">52</span>: 3852&#x02013;3865. 10.1021/bi400383e
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3694592/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23668907" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fbi400383e" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=The+3.2+&#x000c5;+resolution+structure+of+a+receptor:+CheA:CheW+signaling+complex+defines+overlapping+binding+sites+and+key+residue+interactions+within+bacterial+chemosensory+arrays&amp;author=X+Li&amp;author=AD+Fleetwood&amp;author=C+Bayas&amp;author=AM+Bilwes&amp;author=DR+Ortega&amp;volume=52&amp;publication_year=2013&amp;pages=3852-3865&amp;pmid=23668907&amp;doi=10.1021/bi400383e&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref009">9. <span class="mixed-citation">Khursigara CM, Wu X, Subramaniam S. <span class="ref-title">Chemoreceptors in Caulobacter crescentus: trimers of receptor dimers in a partially ordered hexagonally packed array</span>. <span class="ref-journal">J Bacteriol</span>. 2008;<span class="ref-vol">190</span>: 6805&#x02013;6810. 10.1128/JB.00640-08
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2566187/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18689468" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FJB.00640-08" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=Chemoreceptors+in+Caulobacter+crescentus:+trimers+of+receptor+dimers+in+a+partially+ordered+hexagonally+packed+array&amp;author=CM+Khursigara&amp;author=X+Wu&amp;author=S+Subramaniam&amp;volume=190&amp;publication_year=2008&amp;pages=6805-6810&amp;pmid=18689468&amp;doi=10.1128/JB.00640-08&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref010">10. <span class="mixed-citation">Briegel A, Ortega DR, Tocheva EI, Wuichet K, Li Z, Chen S, et al. 
<span class="ref-title">Universal architecture of bacterial chemoreceptor arrays</span>. <span class="ref-journal">Proc Natl Acad Sci U S A</span>. 2009;<span class="ref-vol">106</span>: 17181&#x02013;17186. 10.1073/pnas.0905181106
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2761316/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19805102" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.0905181106" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Universal+architecture+of+bacterial+chemoreceptor+arrays&amp;author=A+Briegel&amp;author=DR+Ortega&amp;author=EI+Tocheva&amp;author=K+Wuichet&amp;author=Z+Li&amp;volume=106&amp;publication_year=2009&amp;pages=17181-17186&amp;pmid=19805102&amp;doi=10.1073/pnas.0905181106&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref011">11. <span class="mixed-citation">Briegel A, Li X, Bilwes AM, Hughes KT, Jensen GJ, Crane BR. <span class="ref-title">Bacterial chemoreceptor arrays are hexagonally packed trimers of receptor dimers networked by rings of kinase and coupling proteins</span>. <span class="ref-journal">Proc Natl Acad Sci U S A</span>. 2012;<span class="ref-vol">109</span>: 3766&#x02013;3771. 10.1073/pnas.1115719109
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3309718/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22355139" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.1115719109" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Bacterial+chemoreceptor+arrays+are+hexagonally+packed+trimers+of+receptor+dimers+networked+by+rings+of+kinase+and+coupling+proteins&amp;author=A+Briegel&amp;author=X+Li&amp;author=AM+Bilwes&amp;author=KT+Hughes&amp;author=GJ+Jensen&amp;volume=109&amp;publication_year=2012&amp;pages=3766-3771&amp;pmid=22355139&amp;doi=10.1073/pnas.1115719109&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref012">12. <span class="mixed-citation">Sourjik V, Berg HC. <span class="ref-title">Localization of components of the chemotaxis machinery of Escherichia coli using fluorescent protein fusions</span>. <span class="ref-journal">Mol Microbiol</span>. 2000;<span class="ref-vol">37</span>: 740&#x02013;751. 10.1046/j.1365-2958.2000.02044.x
 [<a href="https://pubmed.ncbi.nlm.nih.gov/10972797" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1046%2Fj.1365-2958.2000.02044.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Microbiol&amp;title=Localization+of+components+of+the+chemotaxis+machinery+of+Escherichia+coli+using+fluorescent+protein+fusions&amp;author=V+Sourjik&amp;author=HC+Berg&amp;volume=37&amp;publication_year=2000&amp;pages=740-751&amp;pmid=10972797&amp;doi=10.1046/j.1365-2958.2000.02044.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref013">13. <span class="mixed-citation">Wadhams GH, Martin AC, Warren AV, Armitage JP. <span class="ref-title">Requirements for chemotaxis protein localization in Rhodobacter sphaeroides</span>. <span class="ref-journal">Mol Microbiol</span>. 2005;<span class="ref-vol">58</span>: 895&#x02013;902. 10.1111/j.1365-2958.2005.04880.x
 [<a href="https://pubmed.ncbi.nlm.nih.gov/16238635" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fj.1365-2958.2005.04880.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Microbiol&amp;title=Requirements+for+chemotaxis+protein+localization+in+Rhodobacter+sphaeroides&amp;author=GH+Wadhams&amp;author=AC+Martin&amp;author=AV+Warren&amp;author=JP+Armitage&amp;volume=58&amp;publication_year=2005&amp;pages=895-902&amp;pmid=16238635&amp;doi=10.1111/j.1365-2958.2005.04880.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref014">14. <span class="mixed-citation">Sourjik V, Berg HC. <span class="ref-title">Functional interactions between receptors in bacterial chemotaxis</span>. <span class="ref-journal">Nature</span>. 2004;<span class="ref-vol">428</span>: 437&#x02013;441. 10.1038/nature02406
 [<a href="https://pubmed.ncbi.nlm.nih.gov/15042093" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnature02406" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Functional+interactions+between+receptors+in+bacterial+chemotaxis&amp;author=V+Sourjik&amp;author=HC+Berg&amp;volume=428&amp;publication_year=2004&amp;pages=437-441&amp;pmid=15042093&amp;doi=10.1038/nature02406&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref015">15. <span class="mixed-citation">Ames P, Parkinson JS. <span class="ref-title">Conformational suppression of inter-receptor signaling defects</span>. <span class="ref-journal">Proc Natl Acad Sci U S A</span>. 2006;<span class="ref-vol">103</span>: 9292&#x02013;9297. 10.1073/pnas.0602135103
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1482603/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16751275" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.0602135103" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Conformational+suppression+of+inter-receptor+signaling+defects&amp;author=P+Ames&amp;author=JS+Parkinson&amp;volume=103&amp;publication_year=2006&amp;pages=9292-9297&amp;pmid=16751275&amp;doi=10.1073/pnas.0602135103&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref016">16. <span class="mixed-citation">Li M, Hazelbauer GL. <span class="ref-title">Selective allosteric coupling in core chemotaxis signaling complexes</span>. <span class="ref-journal">Proc Natl Acad Sci U S A</span>. 2014;<span class="ref-vol">111</span>: 15940&#x02013;15945. 10.1073/pnas.1415184111
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4234562/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25349385" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.1415184111" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Selective+allosteric+coupling+in+core+chemotaxis+signaling+complexes&amp;author=M+Li&amp;author=GL+Hazelbauer&amp;volume=111&amp;publication_year=2014&amp;pages=15940-15945&amp;pmid=25349385&amp;doi=10.1073/pnas.1415184111&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref017">17. <span class="mixed-citation">Pi&#x000f1;as GE, Frank V, Vaknin A, Parkinson JS. <span class="ref-title">The source of high signal cooperativity in bacterial chemosensory arrays</span>. <span class="ref-journal">Proc Natl Acad Sci U S A</span>. 2016;<span class="ref-vol">113</span>: 3335&#x02013;3340. 10.1073/pnas.1600216113
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4812747/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26951681" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.1600216113" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=The+source+of+high+signal+cooperativity+in+bacterial+chemosensory+arrays&amp;author=GE+Pi&#x000f1;as&amp;author=V+Frank&amp;author=A+Vaknin&amp;author=JS+Parkinson&amp;volume=113&amp;publication_year=2016&amp;pages=3335-3340&amp;pmid=26951681&amp;doi=10.1073/pnas.1600216113&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref018">18. <span class="mixed-citation">Blackhart BD, Zusman DR. <span class="ref-title">&#x0201c;Frizzy&#x0201d; genes of Myxococcus xanthus are involved in control of frequency of reversal of gliding motility</span>. <span class="ref-journal">Proc Natl Acad Sci U S A</span>. 1985;<span class="ref-vol">82</span>: 8767&#x02013;8770. 10.1073/pnas.82.24.8767
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC391518/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3936045" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.82.24.8767" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=&#x0201c;Frizzy&#x0201d;+genes+of+Myxococcus+xanthus+are+involved+in+control+of+frequency+of+reversal+of+gliding+motility&amp;author=BD+Blackhart&amp;author=DR+Zusman&amp;volume=82&amp;publication_year=1985&amp;pages=8767-8770&amp;pmid=3936045&amp;doi=10.1073/pnas.82.24.8767&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref019">19. <span class="mixed-citation">Treuner-Lange A, Macia E, Guzzo M, Hot E, Faure LM, Jakobczak B, et al. 
<span class="ref-title">The small G-protein MglA connects to the MreB actin cytoskeleton at bacterial focal adhesions</span>. <span class="ref-journal">J Cell Biol</span>. 2015;<span class="ref-vol">210</span>: 243&#x02013;256. 10.1083/jcb.201412047
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4508894/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26169353" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1083%2Fjcb.201412047" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Cell+Biol&amp;title=The+small+G-protein+MglA+connects+to+the+MreB+actin+cytoskeleton+at+bacterial+focal+adhesions&amp;author=A+Treuner-Lange&amp;author=E+Macia&amp;author=M+Guzzo&amp;author=E+Hot&amp;author=LM+Faure&amp;volume=210&amp;publication_year=2015&amp;pages=243-256&amp;pmid=26169353&amp;doi=10.1083/jcb.201412047&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref020">20. <span class="mixed-citation">Guzzo M, Agrebi R, Espinosa L, Baronian G, Molle V, Mauriello EMF, et al. 
<span class="ref-title">Evolution and Design Governing Signal Precision and Amplification in a Bacterial Chemosensory Pathway</span>. <span class="ref-journal">PLoS Genet</span>. 2015;<span class="ref-vol">11</span>: e1005460
 10.1371/journal.pgen.1005460
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4546325/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26291327" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pgen.1005460" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Evolution+and+Design+Governing+Signal+Precision+and+Amplification+in+a+Bacterial+Chemosensory+Pathway&amp;author=M+Guzzo&amp;author=R+Agrebi&amp;author=L+Espinosa&amp;author=G+Baronian&amp;author=V+Molle&amp;volume=11&amp;publication_year=2015&amp;pages=e1005460&amp;pmid=26291327&amp;doi=10.1371/journal.pgen.1005460&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref021">21. <span class="mixed-citation">Chang Y-W, Rettberg LA, Treuner-Lange A, Iwasa J, S&#x000f8;gaard-Andersen L, Jensen GJ. <span class="ref-title">Architecture of the type IVa pilus machine</span>. <span class="ref-journal">Science</span>. 2016;<span class="ref-vol">351</span>: aad2001
 10.1126/science.aad2001
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5929464/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26965631" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.aad2001" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Architecture+of+the+type+IVa+pilus+machine&amp;author=Y-W+Chang&amp;author=LA+Rettberg&amp;author=A+Treuner-Lange&amp;author=J+Iwasa&amp;author=L+S&#x000f8;gaard-Andersen&amp;volume=351&amp;publication_year=2016&amp;pages=aad2001&amp;pmid=26965631&amp;doi=10.1126/science.aad2001&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref022">22. <span class="mixed-citation">Bustamante VH, Martinez-Flores I, Vlamakis HC, Zusman DR. <span class="ref-title">Analysis of the Frz signal transduction system of Myxococcus xanthus shows the importance of the conserved C-terminal region of the cytoplasmic chemoreceptor FrzCD in sensing signals</span>. <span class="ref-journal">Mol Microbiol</span>. 2004;<span class="ref-vol">53</span>: 1501&#x02013;13. 10.1111/j.1365-2958.2004.04221.x
 [<a href="https://pubmed.ncbi.nlm.nih.gov/15387825" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fj.1365-2958.2004.04221.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Microbiol&amp;title=Analysis+of+the+Frz+signal+transduction+system+of+Myxococcus+xanthus+shows+the+importance+of+the+conserved+C-terminal+region+of+the+cytoplasmic+chemoreceptor+FrzCD+in+sensing+signals&amp;author=VH+Bustamante&amp;author=I+Martinez-Flores&amp;author=HC+Vlamakis&amp;author=DR+Zusman&amp;volume=53&amp;publication_year=2004&amp;pages=1501-13&amp;pmid=15387825&amp;doi=10.1111/j.1365-2958.2004.04221.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref023">23. <span class="mixed-citation">Mauriello EMF, Jones C, Moine A, Armitage JP. <span class="ref-title">Cellular targeting and segregation of bacterial chemosensory systems</span>. <span class="ref-journal">FEMS Microbiol Rev</span>. 2018;<span class="ref-vol">42</span>: 462&#x02013;476. 10.1093/femsre/fuy015
 [<a href="https://pubmed.ncbi.nlm.nih.gov/29945173" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Ffemsre%2Ffuy015" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEMS+Microbiol+Rev&amp;title=Cellular+targeting+and+segregation+of+bacterial+chemosensory+systems&amp;author=EMF+Mauriello&amp;author=C+Jones&amp;author=A+Moine&amp;author=JP+Armitage&amp;volume=42&amp;publication_year=2018&amp;pages=462-476&amp;pmid=29945173&amp;doi=10.1093/femsre/fuy015&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref024">24. <span class="mixed-citation">McBride MJ, Weinberg RA, Zusman DR. <span class="ref-title">&#x0201c;Frizzy&#x0201d; aggregation genes of the gliding bacterium Myxococcus xanthus show sequence similarities to the chemotaxis genes of enteric bacteria</span>. <span class="ref-journal">Proc Natl Acad Sci U S A</span>. 1989;<span class="ref-vol">86</span>: 424&#x02013;428. 10.1073/pnas.86.2.424
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC286482/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2492105" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.86.2.424" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=&#x0201c;Frizzy&#x0201d;+aggregation+genes+of+the+gliding+bacterium+Myxococcus+xanthus+show+sequence+similarities+to+the+chemotaxis+genes+of+enteric+bacteria&amp;author=MJ+McBride&amp;author=RA+Weinberg&amp;author=DR+Zusman&amp;volume=86&amp;publication_year=1989&amp;pages=424-428&amp;pmid=2492105&amp;doi=10.1073/pnas.86.2.424&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref025">25. <span class="mixed-citation">Concepcion J, Witte K, Wartchow C, Choo S, Yao D, Persson H, et al. 
<span class="ref-title">Label-free detection of biomolecular interactions using BioLayer interferometry for kinetic characterization</span>. <span class="ref-journal">Comb Chem High Throughput Screen</span>. 2009;<span class="ref-vol">12</span>: 791&#x02013;800. 10.2174/138620709789104915
 [<a href="https://pubmed.ncbi.nlm.nih.gov/19758119" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.2174%2F138620709789104915" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Comb+Chem+High+Throughput+Screen&amp;title=Label-free+detection+of+biomolecular+interactions+using+BioLayer+interferometry+for+kinetic+characterization&amp;author=J+Concepcion&amp;author=K+Witte&amp;author=C+Wartchow&amp;author=S+Choo&amp;author=D+Yao&amp;volume=12&amp;publication_year=2009&amp;pages=791-800&amp;pmid=19758119&amp;doi=10.2174/138620709789104915&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref026">26. <span class="mixed-citation">Incl&#x000e1;n YF, Vlamakis HC, Zusman DR. <span class="ref-title">FrzZ, a dual CheY-like response regulator, functions as an output for the Frz chemosensory pathway of Myxococcus xanthus</span>. <span class="ref-journal">Mol Microbiol</span>. 2007;<span class="ref-vol">65</span>: 90&#x02013;102. 10.1111/j.1365-2958.2007.05774.x
 [<a href="https://pubmed.ncbi.nlm.nih.gov/17581122" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fj.1365-2958.2007.05774.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Microbiol&amp;title=FrzZ,+a+dual+CheY-like+response+regulator,+functions+as+an+output+for+the+Frz+chemosensory+pathway+of+Myxococcus+xanthus&amp;author=YF+Incl&#x000e1;n&amp;author=HC+Vlamakis&amp;author=DR+Zusman&amp;volume=65&amp;publication_year=2007&amp;pages=90-102&amp;pmid=17581122&amp;doi=10.1111/j.1365-2958.2007.05774.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref027">27. <span class="mixed-citation">Kaimer C, Zusman DR. <span class="ref-title">Regulation of cell reversal frequency in Myxococcus xanthus requires the balanced activity of CheY-like domains in FrzE and FrzZ</span>. <span class="ref-journal">Mol Microbiol</span>. 2016;<span class="ref-vol">100</span>: 379&#x02013;395. 10.1111/mmi.13323
 [<a href="https://pubmed.ncbi.nlm.nih.gov/26748740" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fmmi.13323" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Microbiol&amp;title=Regulation+of+cell+reversal+frequency+in+Myxococcus+xanthus+requires+the+balanced+activity+of+CheY-like+domains+in+FrzE+and+FrzZ&amp;author=C+Kaimer&amp;author=DR+Zusman&amp;volume=100&amp;publication_year=2016&amp;pages=379-395&amp;pmid=26748740&amp;doi=10.1111/mmi.13323&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref028">28. <span class="mixed-citation">Moine A, Espinosa L, Martineau E, Yaikhomba M, Jazleena PJ, Byrne D, et al. 
<span class="ref-title">The nucleoid as a scaffold for the assembly of bacterial signaling complexes</span>. <span class="ref-journal">PLOS Genet</span>. 2017;<span class="ref-vol">13</span>: e1007103
 10.1371/journal.pgen.1007103
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5716589/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29161263" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pgen.1007103" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLOS+Genet&amp;title=The+nucleoid+as+a+scaffold+for+the+assembly+of+bacterial+signaling+complexes&amp;author=A+Moine&amp;author=L+Espinosa&amp;author=E+Martineau&amp;author=M+Yaikhomba&amp;author=PJ+Jazleena&amp;volume=13&amp;publication_year=2017&amp;pages=e1007103&amp;pmid=29161263&amp;doi=10.1371/journal.pgen.1007103&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref029">29. <span class="mixed-citation">Liu J, Hu B, Morado DR, Jani S, Manson MD, Margolin W. <span class="ref-title">Molecular architecture of chemoreceptor arrays revealed by cryoelectron tomography of Escherichia coli minicells</span>. <span class="ref-journal">Proc Natl Acad Sci U S A</span>. 2012;<span class="ref-vol">109</span>: E1481&#x02013;1488. 10.1073/pnas.1200781109
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3384206/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22556268" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.1200781109" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Molecular+architecture+of+chemoreceptor+arrays+revealed+by+cryoelectron+tomography+of+Escherichia+coli+minicells&amp;author=J+Liu&amp;author=B+Hu&amp;author=DR+Morado&amp;author=S+Jani&amp;author=MD+Manson&amp;volume=109&amp;publication_year=2012&amp;pages=E1481-1488&amp;pmid=22556268&amp;doi=10.1073/pnas.1200781109&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref030">30. <span class="mixed-citation">Cassidy CK, Himes BA, Alvarez FJ, Ma J, Zhao G, Perilla JR, et al. 
<span class="ref-title">CryoEM and computer simulations reveal a novel kinase conformational switch in bacterial chemotaxis signaling</span>. <span class="ref-journal">eLife</span>. 2015;<span class="ref-vol">4</span>
 10.7554/eLife.08419
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6746300/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26583751" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.7554%2FeLife.08419" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=CryoEM+and+computer+simulations+reveal+a+novel+kinase+conformational+switch+in+bacterial+chemotaxis+signaling&amp;author=CK+Cassidy&amp;author=BA+Himes&amp;author=FJ+Alvarez&amp;author=J+Ma&amp;author=G+Zhao&amp;volume=4&amp;publication_year=2015&amp;pmid=26583751&amp;doi=10.7554/eLife.08419&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref031">31. <span class="mixed-citation">Moine A, Agrebi R, Espinosa L, Kirby JR, Zusman DR, Mignot T, et al. 
<span class="ref-title">Functional organization of a multimodular bacterial chemosensory apparatus</span>. <span class="ref-journal">PLoS Genet</span>. 2014;<span class="ref-vol">10</span>: e1004164
 10.1371/journal.pgen.1004164
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3945109/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24603697" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pgen.1004164" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Functional+organization+of+a+multimodular+bacterial+chemosensory+apparatus&amp;author=A+Moine&amp;author=R+Agrebi&amp;author=L+Espinosa&amp;author=JR+Kirby&amp;author=DR+Zusman&amp;volume=10&amp;publication_year=2014&amp;pages=e1004164&amp;pmid=24603697&amp;doi=10.1371/journal.pgen.1004164&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref032">32. <span class="mixed-citation">Adams DW, Wu LJ, Errington J. <span class="ref-title">Nucleoid occlusion protein Noc recruits DNA to the bacterial cell membrane</span>. <span class="ref-journal">EMBO J</span>. 2015;<span class="ref-vol">34</span>: 491&#x02013;501. 10.15252/embj.201490177
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4331003/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25568309" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.15252%2Fembj.201490177" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+J&amp;title=Nucleoid+occlusion+protein+Noc+recruits+DNA+to+the+bacterial+cell+membrane&amp;author=DW+Adams&amp;author=LJ+Wu&amp;author=J+Errington&amp;volume=34&amp;publication_year=2015&amp;pages=491-501&amp;pmid=25568309&amp;doi=10.15252/embj.201490177&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref033">33. <span class="mixed-citation">McBride MJ, K&#x000f6;hler T, Zusman DR. <span class="ref-title">Methylation of FrzCD, a methyl-accepting taxis protein of Myxococcus xanthus, is correlated with factors affecting cell behavior</span>. <span class="ref-journal">J Bacteriol</span>. 1992;<span class="ref-vol">174</span>: 4246&#x02013;4257. 10.1128/jb.174.13.4246-4257.1992
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC206207/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/1624419" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2Fjb.174.13.4246-4257.1992" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=Methylation+of+FrzCD,+a+methyl-accepting+taxis+protein+of+Myxococcus+xanthus,+is+correlated+with+factors+affecting+cell+behavior&amp;author=MJ+McBride&amp;author=T+K&#x000f6;hler&amp;author=DR+Zusman&amp;volume=174&amp;publication_year=1992&amp;pages=4246-4257&amp;pmid=1624419&amp;doi=10.1128/jb.174.13.4246-4257.1992&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref034">34. <span class="mixed-citation">Mauriello EMF, Nan B, Zusman DR. <span class="ref-title">AglZ regulates adventurous (A-) motility in Myxococcus xanthus through its interaction with the cytoplasmic receptor</span>, <span class="ref-journal">FrzCD. Mol Microbiol</span>. 2009;<span class="ref-vol">72</span>: 964&#x02013;977. 10.1111/j.1365-2958.2009.06697.x
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4098657/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19400788" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fj.1365-2958.2009.06697.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FrzCD.+Mol+Microbiol&amp;title=AglZ+regulates+adventurous+(A-)+motility+in+Myxococcus+xanthus+through+its+interaction+with+the+cytoplasmic+receptor&amp;author=EMF+Mauriello&amp;author=B+Nan&amp;author=DR+Zusman&amp;volume=72&amp;publication_year=2009&amp;pages=964-977&amp;pmid=19400788&amp;doi=10.1111/j.1365-2958.2009.06697.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref035">35. <span class="mixed-citation">Nan B, Mauriello EMF, Sun I-H, Wong A, Zusman DR. <span class="ref-title">A multi-protein complex from Myxococcus xanthus required for bacterial gliding motility</span>. <span class="ref-journal">Mol Microbiol</span>. 2010;<span class="ref-vol">76</span>: 1539&#x02013;1554. 10.1111/j.1365-2958.2010.07184.x
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2908308/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20487265" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fj.1365-2958.2010.07184.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Microbiol&amp;title=A+multi-protein+complex+from+Myxococcus+xanthus+required+for+bacterial+gliding+motility&amp;author=B+Nan&amp;author=EMF+Mauriello&amp;author=I-H+Sun&amp;author=A+Wong&amp;author=DR+Zusman&amp;volume=76&amp;publication_year=2010&amp;pages=1539-1554&amp;pmid=20487265&amp;doi=10.1111/j.1365-2958.2010.07184.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref036">36. <span class="mixed-citation">Cole C, Barber JD, Barton GJ. <span class="ref-title">The Jpred 3 secondary structure prediction server</span>. <span class="ref-journal">Nucleic Acids Res</span>. 2008;<span class="ref-vol">36</span>: W197&#x02013;W201. 10.1093/nar/gkn238
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2447793/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18463136" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkn238" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=The+Jpred+3+secondary+structure+prediction+server&amp;author=C+Cole&amp;author=JD+Barber&amp;author=GJ+Barton&amp;volume=36&amp;publication_year=2008&amp;pages=W197-W201&amp;pmid=18463136&amp;doi=10.1093/nar/gkn238&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref037">37. <span class="mixed-citation">Sievers F, Higgins DG. <span class="ref-title">Clustal omega</span>. <span class="ref-journal">Curr Protoc Bioinforma Ed Board Andreas Baxevanis Al</span>. 2014;<span class="ref-vol">48</span>: 3.13.1&#x02013;16. 10.1002/0471250953.bi0313s48
 [<a href="https://pubmed.ncbi.nlm.nih.gov/25501942" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2F0471250953.bi0313s48" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Protoc+Bioinforma+Ed+Board+Andreas+Baxevanis+Al&amp;title=Clustal+omega&amp;author=F+Sievers&amp;author=DG+Higgins&amp;volume=48&amp;publication_year=2014&amp;pages=3.13.1-16&amp;pmid=25501942&amp;doi=10.1002/0471250953.bi0313s48&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref038">38. <span class="mixed-citation">Webb B, Sali A. <span class="ref-title">Comparative Protein Structure Modeling Using MODELLER</span>. <span class="ref-journal">Curr Protoc Bioinforma Ed Board Andreas Baxevanis Al</span>. 2016;<span class="ref-vol">54</span>: 5.6.1&#x02013;5.6.37. 10.1002/cpbi.3
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5031415/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27322406" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fcpbi.3" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Protoc+Bioinforma+Ed+Board+Andreas+Baxevanis+Al&amp;title=Comparative+Protein+Structure+Modeling+Using+MODELLER&amp;author=B+Webb&amp;author=A+Sali&amp;volume=54&amp;publication_year=2016&amp;pages=5.6.1-5.6.37&amp;pmid=27322406&amp;doi=10.1002/cpbi.3&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref039">39. <span class="mixed-citation">Park S-Y, Borbat PP, Gonzalez-Bonet G, Bhatnagar J, Pollard AM, Freed JH, et al. 
<span class="ref-title">Reconstruction of the chemotaxis receptor-kinase assembly</span>. <span class="ref-journal">Nat Struct Mol Biol</span>. 2006;<span class="ref-vol">13</span>: 400&#x02013;407. 10.1038/nsmb1085
 [<a href="https://pubmed.ncbi.nlm.nih.gov/16622408" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnsmb1085" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Struct+Mol+Biol&amp;title=Reconstruction+of+the+chemotaxis+receptor-kinase+assembly&amp;author=S-Y+Park&amp;author=PP+Borbat&amp;author=G+Gonzalez-Bonet&amp;author=J+Bhatnagar&amp;author=AM+Pollard&amp;volume=13&amp;publication_year=2006&amp;pages=400-407&amp;pmid=16622408&amp;doi=10.1038/nsmb1085&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref040">40. <span class="mixed-citation">Krissinel E, Henrick K. <span class="ref-title">Secondary-structure matching (SSM), a new tool for fast protein structure alignment in three dimensions</span>. <span class="ref-journal">Acta Crystallogr D Biol Crystallogr</span>. 2004;<span class="ref-vol">60</span>: 2256&#x02013;2268. 10.1107/S0907444904026460
 [<a href="https://pubmed.ncbi.nlm.nih.gov/15572779" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1107%2FS0907444904026460" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Crystallogr+D+Biol+Crystallogr&amp;title=Secondary-structure+matching+(SSM),+a+new+tool+for+fast+protein+structure+alignment+in+three+dimensions&amp;author=E+Krissinel&amp;author=K+Henrick&amp;volume=60&amp;publication_year=2004&amp;pages=2256-2268&amp;pmid=15572779&amp;doi=10.1107/S0907444904026460&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref041">41. <span class="mixed-citation">Gouet P, Robert X, Courcelle E. <span class="ref-title">ESPript/ENDscript: extracting and rendering sequence and 3D information from atomic structures of proteins</span>. <span class="ref-journal">Nucleic Acids Res</span>. 2003;<span class="ref-vol">31</span>: 3320&#x02013;3323. 10.1093/nar/gkg556
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC168963/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12824317" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkg556" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=ESPript/ENDscript:+extracting+and+rendering+sequence+and+3D+information+from+atomic+structures+of+proteins&amp;author=P+Gouet&amp;author=X+Robert&amp;author=E+Courcelle&amp;volume=31&amp;publication_year=2003&amp;pages=3320-3323&amp;pmid=12824317&amp;doi=10.1093/nar/gkg556&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref042">42. <span class="mixed-citation">Mignot T, Merlie JP, Zusman DR. <span class="ref-title">Regulated pole-to-pole oscillations of a bacterial gliding motility protein</span>. <span class="ref-journal">Science</span>. 2005;<span class="ref-vol">310</span>: 855&#x02013;7. 10.1126/science.1119052
 [<a href="https://pubmed.ncbi.nlm.nih.gov/16272122" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.1119052" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Regulated+pole-to-pole+oscillations+of+a+bacterial+gliding+motility+protein&amp;author=T+Mignot&amp;author=JP+Merlie&amp;author=DR+Zusman&amp;volume=310&amp;publication_year=2005&amp;pages=855-7&amp;pmid=16272122&amp;doi=10.1126/science.1119052&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref043">43. <span class="mixed-citation">Ducret A, Maisonneuve E, Notareschi P, Grossi A, Mignot T, Dukan S. <span class="ref-title">A microscope automated fluidic system to study bacterial processes in real time</span>. <span class="ref-journal">PloS One</span>. 2009;<span class="ref-vol">4</span>: e7282
 10.1371/journal.pone.0007282
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2748647/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19789641" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pone.0007282" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PloS+One&amp;title=A+microscope+automated+fluidic+system+to+study+bacterial+processes+in+real+time&amp;author=A+Ducret&amp;author=E+Maisonneuve&amp;author=P+Notareschi&amp;author=A+Grossi&amp;author=T+Mignot&amp;volume=4&amp;publication_year=2009&amp;pages=e7282&amp;pmid=19789641&amp;doi=10.1371/journal.pone.0007282&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pgen.1008533.ref044">44. <span class="mixed-citation">Ducret A, Quardokus EM, Brun YV. <span class="ref-title">MicrobeJ, a tool for high throughput bacterial cell detection and quantitative analysis</span>. <span class="ref-journal">Nat Microbiol</span>. 2016;<span class="ref-vol">1</span>: 16077
 10.1038/nmicrobiol.2016.77
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5010025/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27572972" ref="reftype=pubmed&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnmicrobiol.2016.77" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Microbiol&amp;title=MicrobeJ,+a+tool+for+high+throughput+bacterial+cell+detection+and+quantitative+analysis&amp;author=A+Ducret&amp;author=EM+Quardokus&amp;author=YV+Brun&amp;volume=1&amp;publication_year=2016&amp;pages=16077&amp;pmid=27572972&amp;doi=10.1038/nmicrobiol.2016.77&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6952110&amp;issue-id=347207&amp;journal-id=331&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">PLOS Genetics</span> are provided here courtesy of <strong>PLOS</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC6952110/pdf/pgen.1008533.pdf" class="int-view">PDF (2.8M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/6952110/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/6952110/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="6952110" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6952110%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="CMhNAZd2RAAuP1uNsr8GAptaCclFRrI9GoL3bKFbJbsjZLwGJTmzmDRTROPoXAax">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6952110%2F&amp;text=A%20divergent%20CheW%20confers%20plasticity%20to%20nucleoid-associated%20chemosensory%20arrays" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6952110%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6952110/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC6952110/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/31860666/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC6952110/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/31860666/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC6952110/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/6952110/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC6952110/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
