
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>An efficient transformation system for Trichoderma atroviride using the pyr4 gene as a selectable marker - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="939BE807DB0F4BC50000617CCFDF445E.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="brazjmicro" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC7688867/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7688867/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Brazilian Journal of Microbiology" /><meta name="citation_title" content="An efficient transformation system for Trichoderma atroviride using the pyr4 gene as a selectable marker" /><meta name="citation_author" content="Gabriela Calc&#x000e1;neo-Hern&#x000e1;ndez" /><meta name="citation_author_institution" content="Unit for Basic and Applied Microbiology, School of Natural Sciences, Autonomous University of Queretaro, 76140 Queretaro, Mexico" /><meta name="citation_author" content="Erick Rojas-Espinosa" /><meta name="citation_author_institution" content="Unit for Basic and Applied Microbiology, School of Natural Sciences, Autonomous University of Queretaro, 76140 Queretaro, Mexico" /><meta name="citation_author" content="Fidel Landeros-Jaime" /><meta name="citation_author_institution" content="Unit for Basic and Applied Microbiology, School of Natural Sciences, Autonomous University of Queretaro, 76140 Queretaro, Mexico" /><meta name="citation_author" content="Jos&#x000e9; Antonio Cervantes-Ch&#x000e1;vez" /><meta name="citation_author_institution" content="Unit for Basic and Applied Microbiology, School of Natural Sciences, Autonomous University of Queretaro, 76140 Queretaro, Mexico" /><meta name="citation_author" content="Edgardo Ulises Esquivel-Naranjo" /><meta name="citation_author_institution" content="Unit for Basic and Applied Microbiology, School of Natural Sciences, Autonomous University of Queretaro, 76140 Queretaro, Mexico" /><meta name="citation_publication_date" content="2020/12" /><meta name="citation_issue" content="4" /><meta name="citation_volume" content="51" /><meta name="citation_firstpage" content="1631" /><meta name="citation_doi" content="10.1007/s42770-020-00329-7" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC7688867/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC7688867/" /><meta name="citation_pmid" content="32627116" /><meta name="DC.Title" content="An efficient transformation system for Trichoderma atroviride using the pyr4 gene as a selectable marker" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Brazilian Society of Microbiology" /><meta name="DC.Contributor" content="Gabriela Calc&#x000e1;neo-Hern&#x000e1;ndez" /><meta name="DC.Contributor" content="Erick Rojas-Espinosa" /><meta name="DC.Contributor" content="Fidel Landeros-Jaime" /><meta name="DC.Contributor" content="Jos&#x000e9; Antonio Cervantes-Ch&#x000e1;vez" /><meta name="DC.Contributor" content="Edgardo Ulises Esquivel-Naranjo" /><meta name="DC.Date" content="2020 Dec" /><meta name="DC.Identifier" content="10.1007/s42770-020-00329-7" /><meta name="DC.Language" content="en" /><meta property="og:title" content="An efficient transformation system for Trichoderma atroviride using the pyr4 gene as a selectable marker" /><meta property="og:type" content="article" /><meta property="og:description" content="The development of an efficient transformation system is essential to enrich the genetic understanding of Trichoderma atroviride. To acquire an additional homologous selectable marker, uracil auxotrophic mutants were generated. First, the pyr4 gene encoding ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7688867/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC7688867/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/2094/">Braz J Microbiol</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/369536/">v.51(4); 2020 Dec</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC7688867
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC7688867/pdf/42770_2020_Article_329.pdf" class="int-view">PDF (6.3M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/7688867/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/7688867/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="7688867" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC7688867%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="Tn5rmOTKQ4jtCejjKwthtzSBFg04dODOVEfsyZ8tBQRZ6rfEdkgotJBhXBu5qH6c">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC7688867%2F&amp;text=An%20efficient%20transformation%20system%20for%20Trichoderma%20atroviride%20using%20the%20pyr4%20gene%20as%20a%20selectable%20marker" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC7688867%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7688867/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC7688867/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/32627116/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC7688867/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/32627116/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC7688867/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/7688867/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/2094/">Braz J Microbiol</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/369536/">v.51(4); 2020 Dec</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC7688867
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-brazjmicro.gif" alt="Logo of brazjmicro" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.scielo.br/scielo.php?script=sci_serial&amp;pid=1517-8382&amp;lng=en&amp;nrm=iso" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Braz J Microbiol.</span> 2020 Dec; 51(4): 1631–1643. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2020 Jul 6. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1007%2Fs42770-020-00329-7" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1007/s42770-020-00329-7</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC7688867</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/32627116">32627116</a></div></div></div><h1 class="content-title">An efficient transformation system for <em>Trichoderma atroviride</em> using the <em>pyr4</em> gene as a selectable marker</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Calc&#x000e1;neo-Hern&#x000e1;ndez%20G%5BAuthor%5D" class="affpopup" co-rid="_co_idm140069795406816" co-class="co-affbox">Gabriela Calc&#x000e1;neo-Hern&#x000e1;ndez</a>,<sup></sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Rojas-Espinosa%20E%5BAuthor%5D" class="affpopup" co-rid="_co_idm140069760831824" co-class="co-affbox">Erick Rojas-Espinosa</a>,<sup></sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Landeros-Jaime%20F%5BAuthor%5D" class="affpopup" co-rid="_co_idm140069795391408" co-class="co-affbox">Fidel Landeros-Jaime</a>,<sup></sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Cervantes-Ch&#x000e1;vez%20JA%5BAuthor%5D" class="affpopup" co-rid="_co_idm140069795389552" co-class="co-affbox">Jos&#x000e9; Antonio Cervantes-Ch&#x000e1;vez</a>,<sup></sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Esquivel-Naranjo%20EU%5BAuthor%5D" class="affpopup" co-rid="_co_idm140069795387696" co-class="co-affbox">Edgardo Ulises Esquivel-Naranjo</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup></sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140069795406816"><h3 class="no_margin">Gabriela Calc&#x000e1;neo-Hern&#x000e1;ndez</h3><p>Unit for Basic and Applied Microbiology, School of Natural Sciences, Autonomous University of Queretaro, 76140 Queretaro, Mexico </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Calc&#x000e1;neo-Hern&#x000e1;ndez%20G%5BAuthor%5D">Gabriela Calc&#x000e1;neo-Hern&#x000e1;ndez</a></div></div><div id="_co_idm140069760831824"><h3 class="no_margin">Erick Rojas-Espinosa</h3><p>Unit for Basic and Applied Microbiology, School of Natural Sciences, Autonomous University of Queretaro, 76140 Queretaro, Mexico </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Rojas-Espinosa%20E%5BAuthor%5D">Erick Rojas-Espinosa</a></div></div><div id="_co_idm140069795391408"><h3 class="no_margin">Fidel Landeros-Jaime</h3><p>Unit for Basic and Applied Microbiology, School of Natural Sciences, Autonomous University of Queretaro, 76140 Queretaro, Mexico </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Landeros-Jaime%20F%5BAuthor%5D">Fidel Landeros-Jaime</a></div></div><div id="_co_idm140069795389552"><h3 class="no_margin">Jos&#x000e9; Antonio Cervantes-Ch&#x000e1;vez</h3><p>Unit for Basic and Applied Microbiology, School of Natural Sciences, Autonomous University of Queretaro, 76140 Queretaro, Mexico </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Cervantes-Ch&#x000e1;vez%20JA%5BAuthor%5D">Jos&#x000e9; Antonio Cervantes-Ch&#x000e1;vez</a></div></div><div id="_co_idm140069795387696"><h3 class="no_margin">Edgardo Ulises Esquivel-Naranjo</h3><p>Unit for Basic and Applied Microbiology, School of Natural Sciences, Autonomous University of Queretaro, 76140 Queretaro, Mexico </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Esquivel-Naranjo%20EU%5BAuthor%5D">Edgardo Ulises Esquivel-Naranjo</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140069757395728_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140069757395728_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140069757395728_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140069757395728_ai" style="display:none"><div class="fm-affl" id="Aff1">Unit for Basic and Applied Microbiology, School of Natural Sciences, Autonomous University of Queretaro, 76140 Queretaro, Mexico </div><div><span class="fm-affl">Edgardo Ulises Esquivel-Naranjo, </span><span class="fm-affl"><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="xm.qau@leviuqse.sesilu" class="oemail">xm.qau@leviuqse.sesilu</a></span>.</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup>Corresponding author.</div><div>Communicated by Responsible Editor: Derlene Attili Agellis.</div></div><div class="fm-article-notes hide half_rhythm" id="idm140069757395728_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2019 Dec 19; Accepted 2020 Jun 25.</div></div><div class="permissions half_rhythm hide" id="idm140069757395728_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; Sociedade Brasileira de Microbiologia 2020</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>ESM 1: </strong>(DOCX 5456&#x000a0;kb).</div><div><a href="/pmc/articles/PMC7688867/bin/42770_2020_329_MOESM1_ESM.docx" data-ga-action="click_feat_suppl">42770_2020_329_MOESM1_ESM.docx</a><span style="color:gray"> (5.3M)</span></div><div class="small guid">GUID: C2D87D24-987A-4591-9E57-DF0C8B592A36</div></div></dd></dl></div><div id="Abs1" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="Abs1title">Abstract</h2><!--article-meta--><div><p id="Par1" class="p p-first">The development of an efficient transformation system is essential to enrich the genetic understanding of <em>Trichoderma atroviride</em>. To acquire an additional homologous selectable marker, uracil auxotrophic mutants were generated. First, the <em>pyr4</em> gene encoding OMP decarboxylase was replaced by the <em>hph</em> marker gene, encoding a hygromycin phosphotransferase. Then, uracil auxotrophs were employed to determine that 5&#x000a0;mM uracil restores their growth and conidia production, and 1&#x000a0;mg&#x000a0;ml<sup>&#x02212;1</sup> is the lethal dose of 5-fluoroorotic acid in <em>T. atroviride</em>. Subsequently, uracil auxotrophic strains, free of a drug-selectable marker, were selected by 5-fluoroorotic acid resistance. Two different deletions in <em>pyr4</em> were mapped in four auxotrophs, encoding a protein with frameshifts at the 310 and 335 amino acids in their COOH-terminal. Six auxotrophs did not have changes in the <em>pyr4</em> ORF even though a specific cassette to delete the <em>pyr4</em> was used, suggesting that 5-FOA could have mutagenic activity. The Ura<sup>&#x02212;</sup>1 strain was selected as a genetic background to knock out the MAPKK Pbs2, MAPK Tmk3, and the blue light receptors Blr1/Blr2, using a short version of <em>pyr4</em> as a homologous marker. The &#x02206;<em>tmk3</em> and &#x02206;<em>pbs2</em> mutants selected with <em>pyr4</em> or <em>hph</em> marker were phenotypically identical, highly sensitive to different stressors, and affected in photoconidiation. The &#x02206;<em>blr1</em> and &#x02206;<em>blr2</em> mutants were not responsive to light, and complementation of uracil biosynthesis did not interfere in the expression of <em>blu1</em>, <em>grg2</em>, <em>phr1</em>, and <em>env1</em> genes upregulated by blue light. Overall, uracil metabolism can be used as a tool for genetic manipulation in <em>T. atroviride</em>.</p><div id="sec-a.k.b.p.b" class="sec"><h3 id="sec-a.k.b.p.btitle">Electronic supplementary material</h3><p class="p p-first-last">The online version of this article (10.1007/s42770-020-00329-7) contains supplementary material, which is available to authorized users.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Auxotrophy, Selectable marker, Sporulation, Cellular stress, Light response</span></div></div><div id="Sec1" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec1title">Introduction</h2><p id="Par2" class="p p-first">Ascomycota and Basidiomycota fungi are primarily the causative agents for diseases in an extensive variety of crops, but modern crop management can control them; however, a high risk of developing resistance to the fungicides exist [<a href="#CR1" rid="CR1" class=" bibr popnode">1</a>, <a href="#CR2" rid="CR2" class=" bibr popnode">2</a>]. As an alternative, <em>Trichoderma</em> species are widely used as biofungicides and biofertilizers in agriculture; they improve crop quality and production, and the use of agrochemicals can be decreased [<a href="#CR3" rid="CR3" class=" bibr popnode">3</a>, <a href="#CR4" rid="CR4" class=" bibr popnode">4</a>]. Nevertheless, the effects of fungi as biocontrol can be slow as compared with chemical control. For this reason, a deep understanding of <em>Trichoderma</em> species&#x02019; biology would improve their biotechnological application.</p><p id="Par3"><em>Trichoderma atroviride</em> is a filamentous fungus antagonist of phytopathogenic fungi. Its mycoparasitic capacity is attributed to competition for nutrients, the production of cell wall degrading enzymes, and antibiosis [<a href="#CR4" rid="CR4" class=" bibr popnode">4</a>, <a href="#CR5" rid="CR5" class=" bibr popnode">5</a>]. In addition, it is considered an excellent model to understand asexual reproduction, as well as light and injury responses [<a href="#CR6" rid="CR6" class=" bibr popnode">6</a>&#x02013;<a href="#CR8" rid="CR8" class=" bibr popnode">8</a>]. Thus, it is necessary to develop effective strategies for gene studies in <em>T. atroviride</em> in order to expand its industrial and agricultural applications.</p><p id="Par4">The use of antibiotics, including fungicides, for selection is high priced and may interfere with the regular function of the targeted gene [<a href="#CR9" rid="CR9" class=" bibr popnode">9</a>, <a href="#CR10" rid="CR10" class=" bibr popnode">10</a>]. One way to avoid the use of antibiotics is to generate auxotrophic mutants as a genetic tool, in which the same affected gene is utilized as a selectable marker, and its manipulation is less expensive [<a href="#CR9" rid="CR9" class=" bibr popnode">9</a>]. Several auxotrophic strains have been reported in <em>Trichoderma</em> species. The <em>arg2</em> gene, which encodes the small subunit of carbamoyl phosphate synthetase, was isolated and used to complement an arginine auxotroph strain in <em>Trichoderma virens</em> [<a href="#CR11" rid="CR11" class=" bibr popnode">11</a>]. In <em>Trichoderma reesei</em>, the <em>argB</em> gene of <em>Aspergillus nidulans</em> can restore the arginine synthesis [<a href="#CR12" rid="CR12" class=" bibr popnode">12</a>]. Adenine auxotrophs of <em>T. reesei</em> lacking the <em>ade2</em> gene, encoding a phosphoribosylaminoimidazole carboxylase required for production of purines, can be easily detected by selection of red colonies [<a href="#CR13" rid="CR13" class=" bibr popnode">13</a>]. In <em>T. reesei</em> and <em>Trichoderma hypoxylon</em>, uracil auxotrophs selected by 5-fluoro-orotic acid (5-FOA) were used successfully as a genetic tool for the disruption of a single gene [<a href="#CR14" rid="CR14" class=" bibr popnode">14</a>&#x02013;<a href="#CR16" rid="CR16" class=" bibr popnode">16</a>].</p><p id="Par5">The pyrimidine analogue 5-FOA is generally used for positive selection of uracil auxotrophic mutants and is catalyzed by orotate phosphoribosyl transferase (OPRTase) and orotidine-5&#x02032;-monophosphate decarboxylase (OMP decarboxylase) to obtain 5-fluorouridine 5&#x02032;-monophosphate (5-FUMP) and 5-fluorouracil (5-FU) [<a href="#CR17" rid="CR17" class=" bibr popnode">17</a>]. The 5-FU inhibits thymidylate synthase activity, consequently leading to thymine nucleotide depletion and affecting DNA synthesis [<a href="#CR17" rid="CR17" class=" bibr popnode">17</a>&#x02013;<a href="#CR19" rid="CR19" class=" bibr popnode">19</a>]. Therefore, the toxic effect of 5-FOA on wild-type strains facilitates its selection, whereas mutants deficient in OPRTase or OMP decarboxylase, essential for uracil synthesis, can grow on 5-FOA medium supplemented with uridine/uracil [<a href="#CR20" rid="CR20" class=" bibr popnode">20</a>, <a href="#CR21" rid="CR21" class=" bibr popnode">21</a>].</p><p id="Par6" class="p p-last">In order to obtain an alternative transformation system in <em>Trichoderma atroviride</em>, we generated uracil auxotrophic strains that are unable to grow in uracil-deficient media. Physiological and molecular results indicate that the <em>pyr4</em> gene can be used as a selectable marker in a uracil auxotrophic background. Four different genes related to stress and light responses in <em>T. atroviride</em> were effectively replaced in a uracil auxotroph strain, and the mutants showed identical phenotypes in comparison with knockouts generated by resistance to hygromycin B.</p></div><div id="Sec2" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec2title">Methods</h2><div id="Sec3" class="sec sec-first"><h3 id="Sec3title">Strains and culture conditions</h3><p id="Par7" class="p p-first-last">In this study, <em>T. atroviride</em> IMI206040 was used as the wild-type strain (WT) and as the parent strain for gene replacement experiments. The &#x02206;<em>pbs2</em>&#x02013;7 and &#x02206;<em>tmk3</em>&#x02013;13 strains were previously reported [<a href="#CR8" rid="CR8" class=" bibr popnode">8</a>]. All strains were propagated on potato dextrose agar (PDA; DIFCO) at 27&#x000a0;&#x000b0;C. Transformants were selected on minimal medium (MM) containing per liter: 20&#x000a0;g glucose, 0.2&#x000a0;g MgSO4.7H<sub>2</sub>O, 0.9&#x000a0;g K<sub>2</sub>HPO<sub>4</sub>, 0.2&#x000a0;g KCl, 1&#x000a0;g NH<sub>4</sub>NO<sub>3</sub>, 2&#x000a0;mg FeSO<sub>4</sub>.7H<sub>2</sub>O, 2&#x000a0;mg ZnSO<sub>4</sub>.7H<sub>2</sub>O, 2&#x000a0;mg MnCl<sub>2</sub>.7H<sub>2</sub>O, and 15&#x000a0;g agar, and pH was adjusted to 5.5 with KOH. MM was supplemented with uracil 5&#x000a0;mM according to Gruber et al. [<a href="#CR14" rid="CR14" class=" bibr popnode">14</a>]. When necessary, 100&#x000a0;&#x003bc;g&#x000a0;ml<sup>&#x02212;1</sup> hygromycin B (Invitrogen) or 5-FOA was added to the medium. <em>Escherichia coli</em> DH5&#x003b1; strain was propagated in a Luria-Bertani (LB) medium with appropriate antibiotics for cloning and plasmid DNA isolation purposes.</p></div><div id="Sec4" class="sec"><h3 id="Sec4title">Replacement of <em>pyr4</em> by the <em>hph</em> marker</h3><p id="Par8" class="p p-first-last">To generate <em>pyr4</em> mutants, its sequence was identified in the <em>T. atroviride</em> genome database (v2.0) of the Department of Energy Joint Genome Institute through a BLASTp search against the amino acid sequence of <em>Trichoderma harzianum</em> Pyr4 (<a id="__tag_1130671769" class="tag_hotlink" href="/protein/AAA51865.1" ref="/protein/AAA51865.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"AAA51865.1","term_id":"468026"}}</span>AAA51865.1</a>), which has been previously reported [<a href="#CR22" rid="CR22" class=" bibr popnode">22</a>]. The <em>pyr4</em> open-reading frame (ORF) was replaced by a hygromycin B resistance cassette based on the double-joint PCR method [<a href="#CR23" rid="CR23" class=" bibr popnode">23</a>] with modified conditions as previously described [<a href="#CR24" rid="CR24" class=" bibr popnode">24</a>]. Primers designed to replace the <em>pyr4</em> ORF by the selectable marker are listed in Table <a href="/pmc/articles/PMC7688867/table/Tab1/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab1" rid-ob="ob-Tab1" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">Table1.</span></span><span>1</span></a>. The 5&#x02032; and 3&#x02032; flanking regions of the <em>pyr4</em> gene were amplified using the primers Ppyr4-F&#x02013;PQpyr4-R and TQpyr4-F&#x02013;Tpyr4-R, respectively. The hygromycin B resistance marker, <em>hph</em>, was amplified using the primers Hyg-F &#x02013; Hyg-R. In a second PCR round, the three fragments were joined. The complementary sequences to <em>hph</em> were added to the chimeric primers (PQpyr4-R &#x02013; TQpyr4-F). In a third PCR, the cassette was amplified using nested primers (N5pyr4-F &#x02013; N3pyr4-R). The Platinum&#x02122; Taq DNA Polymerase High Fidelity (Invitrogen) was used to construct the cassette, and the PCR conditions were as follows: first step at 94&#x000a0;&#x000b0;C for 2&#x000a0;min, 35&#x000a0;cycles at 94&#x000a0;&#x000b0;C/15&#x000a0;s, 60&#x000a0;&#x000b0;C/15&#x000a0;s, 68&#x000a0;&#x000b0;C/1&#x000a0;min per kb, and a final extension at 68&#x000a0;&#x000b0;C for 5&#x000a0;min. The final amplicon was used to transform <em>T. atroviride</em> protoplasts.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="Tab1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p>Primers used in this work</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Gene</th><th rowspan="1" colspan="1">Primer</th><th rowspan="1" colspan="1">Sequence (5&#x02032; to 3&#x02032;)</th><th rowspan="1" colspan="1">Target region</th></tr></thead><tbody><tr><td rowspan="11" colspan="1">Orotidine 5&#x02032;-phosphate decarboxylase (<em>pyr4</em>; <a id="__tag_1130671764" class="tag_hotlink" href="/protein/EHK48963.1" ref="/protein/EHK48963.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"EHK48963.1","term_id":"358399626"}}</span>EHK48963.1</a>)</td><td rowspan="1" colspan="1">Ppyr4-F</td><td rowspan="1" colspan="1">CAG GGC TTC AAC TGC CAC GG</td><td rowspan="2" colspan="1">5&#x02032; flanking region of the <em>pyr4</em> gene</td></tr><tr><td rowspan="1" colspan="1">PQpyr4-R</td><td rowspan="1" colspan="1">TGC TCC TTC AAT ATC AGT TAA CGT CGA TCA TGC CGT CTA ATG TCG GGT TG</td></tr><tr><td rowspan="1" colspan="1">TQpyr4-F</td><td rowspan="1" colspan="1">CCC AGC ACT CGT CCG AGG GCA AAG GAA TAG AGG TGG AGG CGA ACT GCA C</td><td rowspan="2" colspan="1">3&#x02032; flanking region of the <em>pyr4</em> gene</td></tr><tr><td rowspan="1" colspan="1">Tpyr4-R</td><td rowspan="1" colspan="1">GTG TCT TGG CGC TCG GTC G</td></tr><tr><td rowspan="1" colspan="1">N5pyr4-F</td><td rowspan="1" colspan="1">GGG GGA GCT ACT GAG ACT C</td><td rowspan="2" colspan="1">Nested <em>pyr4</em> primers</td></tr><tr><td rowspan="1" colspan="1">N3pyr4-R</td><td rowspan="1" colspan="1">GGA TGG GGG CGG TCC AAG</td></tr><tr><td rowspan="1" colspan="1">Q5pyr4-R</td><td rowspan="1" colspan="1">TCG CCT CCA CCT ATG CCG TCT AAT GTC GGG TTG</td><td rowspan="2" colspan="1">Chimeric <em>pyr4</em> primers with complementary sequences to the <em>pyr4</em> 5&#x02032; and 3&#x02032; regions</td></tr><tr><td rowspan="1" colspan="1">Q3pyr4-F</td><td rowspan="1" colspan="1">TAG ACG GCA TAG GTG GAG GCG AAC TGC AC</td></tr><tr><td rowspan="1" colspan="1">smPyr4-F</td><td rowspan="1" colspan="1">CTC CCT ACG CTC TCC GCC ACC ACC AAT CGA TCT TTA TCC G</td><td rowspan="3" colspan="1"><em>pyr4</em> ORF</td></tr><tr><td rowspan="1" colspan="1">smPyr4-R</td><td rowspan="1" colspan="1">CTC CAG CCT CTT CCA GCT CCG CAA CTT CTT TTT TTC TTA TCG CAG C</td></tr><tr><td rowspan="1" colspan="1">1Tpyr4-R</td><td rowspan="1" colspan="1">GAC GCA CTC AAC ATC TCG TAG C</td></tr><tr><td rowspan="2" colspan="1"><p>Hygromycin-B-phosphotransferase</p><p>(<em>hph</em>; <a id="__tag_1130671765" class="tag_hotlink" href="/protein/AEJ60084.1" ref="/protein/AEJ60084.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"AEJ60084.1","term_id":"339418417"}}</span>AEJ60084.1</a>)</p></td><td rowspan="1" colspan="1">Hyg-F</td><td rowspan="1" colspan="1">GAT CGA CGT TAA CTG ATA TTG AAG GAG CA</td><td rowspan="2" colspan="1"><em>hph</em> marker</td></tr><tr><td rowspan="1" colspan="1">Hyg-R</td><td rowspan="1" colspan="1">CTA TTC CTT TGC CCT CGG ACG AGT GCT GGG</td></tr><tr><td rowspan="8" colspan="1">Mitogen activated protein kinase Tmk3 (<em>tmk3</em>; <a id="__tag_1130671763" class="tag_hotlink" href="/protein/EHK43400.1" ref="/protein/EHK43400.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"EHK43400.1","term_id":"358393999"}}</span>EHK43400.1</a>)</td><td rowspan="1" colspan="1">Ptmk3-F</td><td rowspan="1" colspan="1">GCT GGT GCC CTC TTG TTC TC</td><td rowspan="2" colspan="1">5&#x02032; flanking region of the <em>tmk3</em> gene</td></tr><tr><td rowspan="1" colspan="1">PQtmk3-R</td><td rowspan="1" colspan="1">GGT GGT GGC GGA GAG CGT AGG GAG ATT CGC GGG GAA ATA ACG ACG</td></tr><tr><td rowspan="1" colspan="1">TQtmk3-F</td><td rowspan="1" colspan="1">GCG GAG CTG GAA GAG GCT GGA GAG AGA GTA GAG GTG CAG AAT TCG</td><td rowspan="2" colspan="1">3&#x02032; flanking region of the <em>tmk3</em> gene</td></tr><tr><td rowspan="1" colspan="1">Ttmk3-R</td><td rowspan="1" colspan="1">CTA CAG GTC GTC TCC ACA GC</td></tr><tr><td rowspan="1" colspan="1">N5tmk3-F</td><td rowspan="1" colspan="1">CTC TTG TGC TCC GAC GAA ACC</td><td rowspan="2" colspan="1">Nested <em>tmk3</em> primers</td></tr><tr><td rowspan="1" colspan="1">N3tmk3-R</td><td rowspan="1" colspan="1">CGT CCA CCT CCA AAT CAT GGC</td></tr><tr><td rowspan="1" colspan="1">tmk3-F</td><td rowspan="1" colspan="1">GTT TGG TCT TGT CTG CTC TGC G</td><td rowspan="2" colspan="1"><em>tmk3</em> ORF</td></tr><tr><td rowspan="1" colspan="1">tmk3-R</td><td rowspan="1" colspan="1">GCA GGT CGG TTC CGA GAA GC</td></tr><tr><td rowspan="8" colspan="1">MAP kinase kinase Pbs2 (<em>pbs2</em>; <a id="__tag_1130671760" class="tag_hotlink" href="/protein/EHK47459.1" ref="/protein/EHK47459.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"EHK47459.1","term_id":"358398101"}}</span>EHK47459.1</a>)</td><td rowspan="1" colspan="1">Ppbs2-F</td><td rowspan="1" colspan="1">CAG TGC GCT GGC TCC AGC</td><td rowspan="2" colspan="1">5&#x02032; flanking region of the <em>pbs2</em> gene</td></tr><tr><td rowspan="1" colspan="1">PQpbs2-R</td><td rowspan="1" colspan="1">GGT GGT GGC GGA GAG CGT AGG GAG AGG TGG GGT TTT CGG ATG CTT C</td></tr><tr><td rowspan="1" colspan="1">TQpbs2-F</td><td rowspan="1" colspan="1">GCG GAG CTG GAA GAG GCT GGA GCA GAA GAA AAG CAT AGA GCG G</td><td rowspan="2" colspan="1">3&#x02032; flanking region of the <em>pbs2</em> gene</td></tr><tr><td rowspan="1" colspan="1">Tpbs2-R</td><td rowspan="1" colspan="1">CGC GAA CCT ACC GGA AGC C</td></tr><tr><td rowspan="1" colspan="1">N5pbs2-F</td><td rowspan="1" colspan="1">TCA CGA CAG ACG AGT CTT GCC</td><td rowspan="2" colspan="1">Nested <em>pbs2</em> primers</td></tr><tr><td rowspan="1" colspan="1">N3pbs2-R</td><td rowspan="1" colspan="1">TGA AAG AGG TAA TGG CAG TAT TCG G</td></tr><tr><td rowspan="1" colspan="1">pbs2-F</td><td rowspan="1" colspan="1">GTC ACG ATG AGC GGC ACC G</td><td rowspan="2" colspan="1"><em>pbs2</em> ORF</td></tr><tr><td rowspan="1" colspan="1">pbs2-R</td><td rowspan="1" colspan="1">AAG CTT GGT GCC CCT CAC TGG CTG</td></tr><tr><td rowspan="6" colspan="1">Blue-light receptor Blr1 (<em>blr1</em>; <a id="__tag_1130671767" class="tag_hotlink" href="/protein/EHK39879.1" ref="/protein/EHK39879.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"EHK39879.1","term_id":"358390474"}}</span>EHK39879.1</a>)</td><td rowspan="1" colspan="1">Pblr1-F</td><td rowspan="1" colspan="1">GTT AGT TAG AGG ACG AAA GGC AG</td><td rowspan="2" colspan="1">5&#x02032; flanking region of the <em>blr1</em> gene</td></tr><tr><td rowspan="1" colspan="1">PQblr1-R</td><td rowspan="1" colspan="1">GGT GGT GGC GGA GAG CGT AGG GAG AGT CGG AGA GGG AGA GGG C</td></tr><tr><td rowspan="1" colspan="1">TQblr1-F</td><td rowspan="1" colspan="1">GCG GAG CTG GAA GAG GCT GGA GAC GAG AAG AGC GAG AAA GTA GC</td><td rowspan="2" colspan="1">3&#x02032; flanking region of the <em>blr1</em> gene</td></tr><tr><td rowspan="1" colspan="1">Tblr1-R</td><td rowspan="1" colspan="1">CCG AGA CGC TGC GAT GGC</td></tr><tr><td rowspan="1" colspan="1">N5blr1-F</td><td rowspan="1" colspan="1">GTA GGC CTT CAA GAC CGT GC</td><td rowspan="2" colspan="1">Nested <em>blr1</em> primers</td></tr><tr><td rowspan="1" colspan="1">N3blr1-R</td><td rowspan="1" colspan="1">TCG GTC CAT TCA CAA AAG CTC C</td></tr><tr><td rowspan="8" colspan="1">Blue-light receptor Blr2 (<em>blr2</em>; <a id="__tag_1130671761" class="tag_hotlink" href="/protein/EHK40294.1" ref="/protein/EHK40294.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"EHK40294.1","term_id":"358390889"}}</span>EHK40294.1</a>)</td><td rowspan="1" colspan="1">Pblr2-F</td><td rowspan="1" colspan="1">GAA ACG CCG GTC AGC AGG C</td><td rowspan="2" colspan="1">5&#x02032; flanking region of the <em>blr2</em> gene</td></tr><tr><td rowspan="1" colspan="1">PQblr2-R</td><td rowspan="1" colspan="1">GGT GGT GGC GGA GAG CGT AGG GAG AGC AGA CGA TGG CAC TGG AG</td></tr><tr><td rowspan="1" colspan="1">TQblr2-F</td><td rowspan="1" colspan="1">GCG GAG CTG GAA GAG GCT GGA GAC AAG GAA AGA GAG GAA GAA CGG</td><td rowspan="2" colspan="1">3&#x02032; flanking region of the <em>blr2</em> gene</td></tr><tr><td rowspan="1" colspan="1">Tblr2-R</td><td rowspan="1" colspan="1">GGA TTA TGC AGC TGA GGT TAT GG</td></tr><tr><td rowspan="1" colspan="1">N5blr2-F</td><td rowspan="1" colspan="1">GTA CCT GTT TGG CGA TGC TAC G</td><td rowspan="2" colspan="1">Nested <em>blr2</em> primers</td></tr><tr><td rowspan="1" colspan="1">N3blr2-R</td><td rowspan="1" colspan="1">GCA TCC GAG GCA AAT TGA GCG</td></tr><tr><td rowspan="1" colspan="1">blr2-F</td><td rowspan="1" colspan="1">ACC GCA GCC CTA CCC TCA TC</td><td rowspan="2" colspan="1"><em>blr2</em> ORF</td></tr><tr><td rowspan="1" colspan="1">blr2-R</td><td rowspan="1" colspan="1">CAC CGC AAC CCG CAG GC</td></tr><tr><td rowspan="2" colspan="1">Blu1 (<em>blu1</em>; <a id="__tag_1130671757" class="tag_hotlink" href="/protein/EHK44319.1" ref="/protein/EHK44319.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"EHK44319.1","term_id":"358394926"}}</span>EHK44319.1</a>)</td><td rowspan="1" colspan="1">qblu1-F</td><td rowspan="1" colspan="1">CGT TGG CTC TCG CCT GAC C</td><td rowspan="2" colspan="1"><em>blu1</em> ORF</td></tr><tr><td rowspan="1" colspan="1">blu1-R</td><td rowspan="1" colspan="1">GAA CGC CAT TGA AGG CCT CG</td></tr><tr><td rowspan="2" colspan="1">Grg2 (<em>grg2</em>; <a id="__tag_1130671766" class="tag_hotlink" href="/protein/EHK50625.1" ref="/protein/EHK50625.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"EHK50625.1","term_id":"358401319"}}</span>EHK50625.1</a>)</td><td rowspan="1" colspan="1">grg2-F</td><td rowspan="1" colspan="1">GAT TCC ATC AAG CAG GGT GCC</td><td rowspan="2" colspan="1"><em>grg2</em> ORF</td></tr><tr><td rowspan="1" colspan="1">grg2-R</td><td rowspan="1" colspan="1">GTT TAG ATA GCC TGC TTG TGG G</td></tr><tr><td rowspan="2" colspan="1">DNA photolyase Phr1 (<em>phr1</em>; <a id="__tag_1130671758" class="tag_hotlink" href="/protein/EHK39920.1" ref="/protein/EHK39920.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"EHK39920.1","term_id":"358390515"}}</span>EHK39920.1</a>)</td><td rowspan="1" colspan="1">rtPhr1-F</td><td rowspan="1" colspan="1">TCG GAC AGC CAG GGA CAG G</td><td rowspan="2" colspan="1"><em>phr1</em> ORF</td></tr><tr><td rowspan="1" colspan="1">rtPhr1-R</td><td rowspan="1" colspan="1">GTG ATT GAG CTG ACG CAT CGC</td></tr><tr><td rowspan="2" colspan="1"><p>Envoy</p><p>(<em>env1</em>; <a id="__tag_1130671755" class="tag_hotlink" href="/protein/EHK44161.1" ref="/protein/EHK44161.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"EHK44161.1","term_id":"358394768"}}</span>EHK44161.1</a>)</p></td><td rowspan="1" colspan="1">env1-F</td><td rowspan="1" colspan="1">GCC AAA ATG GTT CCT TCA GGG TC</td><td rowspan="2" colspan="1"><em>env1</em> ORF</td></tr><tr><td rowspan="1" colspan="1">env1-R</td><td rowspan="1" colspan="1">GTT TGG TCG AGA CAC AAG TCG G</td></tr><tr><td rowspan="2" colspan="1">Glyceraldehyde 3-phosphate dehydrogenase (<em>gpd</em>; <a id="__tag_1130671762" class="tag_hotlink" href="/protein/EHK49005.1" ref="/protein/EHK49005.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"EHK49005.1","term_id":"358399668"}}</span>EHK49005.1</a>)</td><td rowspan="1" colspan="1">gpd-F</td><td rowspan="1" colspan="1">GCC GAT GGT GAG CTC AAG GG</td><td rowspan="2" colspan="1"><em>gpd</em> ORF</td></tr><tr><td rowspan="1" colspan="1">gpd-R</td><td rowspan="1" colspan="1">GGT CGA GGA CAC GGC GGG A</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140069758783648"><a target="object" rel="noopener" href="/pmc/articles/PMC7688867/table/Tab1/?report=objectonly">Open in a separate window</a></div></div></div><div id="Sec5" class="sec"><h3 id="Sec5title">Protoplasts isolation</h3><p id="Par9" class="p p-first-last">Protoplasts of <em>T. atroviride</em> were prepared by modifying the method of Baek and Kenerley [<a href="#CR11" rid="CR11" class=" bibr popnode">11</a>]. A conidial suspension of the WT strain (1&#x02009;&#x000d7;&#x02009;10<sup>6</sup> conidia ml<sup>&#x02212;1</sup>) was cultivated in 100&#x000a0;ml of GYEC liquid medium (1.5% glucose, 0.3% yeast extract, 0.5% casein, and pH adjusted to 5.5 with KOH) and incubated in constant orbital agitation (160&#x000a0;rpm) at 27&#x000a0;&#x000b0;C for 18&#x000a0;h. Then, 0.2&#x000a0;g of mycelia were recovered by filtration and placed in a 50-ml centrifuge tube with a 7&#x000a0;ml of osmotic solution (50&#x000a0;mM CaCl<sub>2</sub>, 0.5&#x000a0;M mannitol, 50&#x000a0;mM MES, pH&#x000a0;5.5) and 0.1&#x000a0;g of lyophilized lysing enzymes from <em>T. harzianum</em> (Sigma-Aldrich). Mycelia were incubated in orbital agitation (120&#x000a0;rpm) at room temperature for 2&#x000a0;h for the formation of protoplasts. Protoplasts were harvested through sterile Miracloth and carefully transferred into 1.5-ml microcentrifuge tubes. After centrifuging at 8000&#x000a0;rpm for 10&#x000a0;min at 4&#x000a0;&#x000b0;C, the supernatant was discarded, and the protoplast pellet was resuspended in osmotic solution at a concentration of 1&#x02009;&#x000d7;&#x02009;10<sup>7&#x02013;8</sup>&#x000a0;ml<sup>&#x02212;1</sup>.</p></div><div id="Sec6" class="sec"><h3 id="Sec6title">Protoplast transformation by the polyethylene glycol (PEG)&#x02013;CaCl<sub>2</sub> method</h3><p id="Par10" class="p p-first-last">Transformation was conducted with 250&#x000a0;&#x003bc;l protoplasts and 100&#x000a0;&#x003bc;l DNA (50&#x000a0;&#x003bc;l PCR product plus 50&#x000a0;&#x003bc;l osmotic solution), and mixture remained on ice for 20&#x000a0;min. Afterwards, 350&#x000a0;&#x003bc;l of PEG (40% PEG 6000 dissolved in osmotic solution) at 42&#x000a0;&#x000b0;C was added and incubated at room temperature for 30&#x000a0;min. To select transformants using a <em>hph</em> marker, transformation was mixed with 7&#x02013;10-ml soft PDA (PDB, 0.7% bacteriological agar, 1&#x000a0;M sorbitol and 100&#x000a0;&#x003bc;g.ml<sup>&#x02212;1</sup> hygromycin B) and poured on selective PDA plates (PDA, 1&#x000a0;M sorbitol and 100&#x000a0;&#x003bc;g.ml<sup>&#x02212;1</sup> hygromycin B). To select transformants using the <em>pyr4</em> marker, the transformation reaction was mixed with 7&#x02013;10-ml soft MM (MM, 0.7% bacterial agar and 1&#x000a0;M sorbitol) and poured on MM plates (MM and 1&#x000a0;M sorbitol). Cultures were incubated at 27&#x000a0;&#x000b0;C for 4&#x02013;5&#x000a0;days.</p></div><div id="Sec7" class="sec"><h3 id="Sec7title">Assay to identify uracil auxotrophy</h3><p id="Par11" class="p p-first-last">Transformants resistant to hygromycin were subjected to three rounds of single-spore isolation in MM plus 5&#x000a0;mM uracil and 100&#x000a0;&#x003bc;g&#x000a0;ml<sup>&#x02212;1</sup> hygromycin B. To identify auxotrophic mutants, drops of 500 conidia (5&#x000a0;&#x003bc;l of a 1&#x000d7;10<sup>5</sup> conidia per ml suspension) of the WT and transformants were inoculated on MM plates with 0.5% Triton X-100, with or without 5&#x000a0;mM uracil, and incubated for 4&#x000a0;days at 27&#x000a0;&#x000b0;C. Transformants unable to grow in uracil-deficient medium were selected, and the <em>pyr4</em> ORF replacement was confirmed by PCR using specific primers flanking the recombined sequences. Genomic DNA of the WT and mutant strains were prepared according to the procedure described by Raeder &#x00026; Broda [<a href="#CR25" rid="CR25" class=" bibr popnode">25</a>]. The DreamTaq DNA Polymerase (Thermo Scientific) was used for PCR reactions, and the conditions were as follows: first step at 95&#x000a0;&#x000b0;C for 3&#x000a0;min, 35&#x000a0;cycles at 95&#x000a0;&#x000b0;C/30&#x000a0;s, 60&#x000a0;&#x000b0;C/30&#x000a0;s, 72&#x000a0;&#x000b0;C/1&#x000a0;min per kb, and a final extension at 72&#x000a0;&#x000b0;C for 5&#x000a0;min.</p></div><div id="Sec8" class="sec"><h3 id="Sec8title">Uracil assays to restore auxotrophy</h3><p id="Par12" class="p p-first-last">The minimal amount of uracil required for the auxotrophic mutants to restore their growth was determined using MM with different concentrations of uracil. Drops of 500 conidia of the WT and mutant strains were inoculated on MM plates plus 0.5% Triton X-100 and with different uracil concentrations (0, 1, 2.5, 5, and 10&#x000a0;mM) for 4&#x000a0;days at 27&#x000a0;&#x000b0;C. To restore growth, mycelial plugs were taken from colonies grown on MM plates with 5&#x000a0;mM uracil (0.5&#x000a0;mm of diameter of the colony front) and inoculated on MM plates supplemented with uracil (0, 1, 2.5, 5, and 10&#x000a0;mM). Cultures were incubated at 27&#x000a0;&#x000b0;C during 72&#x000a0;h, and pictures were taken for radial growth measurements with ImageJ software (<a href="https://imagej.nih.gov/ij/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://imagej.nih.gov/ij/</a>). The experiment was conducted in triplicate.</p></div><div id="Sec9" class="sec"><h3 id="Sec9title">5-FOA sensitivity assays</h3><p id="Par13" class="p p-first-last">To establish the lethal dose of 5-FOA in <em>T. atroviride</em>, drops of 500 conidia of the WT and auxotrophic mutants were inoculated on MM plates with various concentrations of 5-FOA (0.5, 1, 2, 3, and 4&#x000a0;mg&#x000a0;ml<sup>&#x02212;1</sup>), plus 5&#x000a0;mM uracil and 0.5% Triton X-100, and incubated during 4&#x000a0;days at 27&#x000a0;&#x000b0;C. The experiment was conducted in triplicate.</p></div><div id="Sec10" class="sec"><h3 id="Sec10title">Transformation based on a positive selection by use of 5-FOA</h3><p id="Par14" class="p p-first-last">To obtain uracil auxotrophic strains free of an antibiotic-resistance marker, a co-transformation was carried out using a free-marker cassette created by joining the 5&#x02032; and 3&#x02032; flanking regions of the <em>pyr4</em> gene of <em>T. atroviride</em>, 5-FOA for positive selection [<a href="#CR15" rid="CR15" class=" bibr popnode">15</a>, <a href="#CR26" rid="CR26" class=" bibr popnode">26</a>], and the plasmid pCB1004 [<a href="#CR24" rid="CR24" class=" bibr popnode">24</a>], which is autoreplicative in <em>T. atroviride</em> (Esquivel-Naranjo and Herrera-Estrella, under review). Primers listed in Table <a href="/pmc/articles/PMC7688867/table/Tab1/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab1" rid-ob="ob-Tab1" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">Table1</span></span><span>1</span></a> were designed to amplify the 5&#x02032; (Ppyr4-F &#x02013; Q5pyr4-R) and 3&#x02032; (Q3pyr4-F &#x02013; Tpyr4-R) flanking regions of <em>pyr4</em>, excluding the complete ORF. In a second PCR, 5&#x02032; and 3&#x02032; regions were joined through 22 complementary nucleotides added on the 5&#x02032;-end of the chimeric primers (Q5pyr4-R - Q3pyr4-F). In a third PCR, the cassette was amplified using nested primers N5pyr4-F &#x02013; N3pyr4-R and purified by QIAGEN columns. The Platinum&#x02122; Taq DNA Polymerase High Fidelity was used to construct the free-marker cassette, and the PCR conditions were as mentioned above. <em>T. atroviride</em> protoplasts were co-transformed with the free-marker cassette and pCB1004 in a 2:1&#x000a0;M ratio. Transformants were selected on MM supplemented with 5&#x000a0;mM uracil, 0.5&#x000a0;mg&#x000a0;ml<sup>&#x02212;1</sup> 5-FOA (sublethal dose), and 100&#x000a0;&#x003bc;g&#x000a0;ml<sup>&#x02212;1</sup> hygromycin B. Then, one round of single-spore isolation was performed in MM plus 5&#x000a0;mM uracil, 2&#x000a0;mg&#x000a0;ml<sup>&#x02212;1</sup> 5-FOA, and 100&#x000a0;&#x003bc;g&#x000a0;ml<sup>&#x02212;1</sup> hygromycin B and the next two rounds in MM plus 5&#x000a0;mM uracil, 0.5&#x000a0;mg&#x000a0;ml<sup>&#x02212;1</sup> 5-FOA, and without hygromycin B. Auxotrophy tests were carried out as previously described. The <em>pyr4</em> gene deletion was confirmed by PCR using primers flanking the coding region and sequencing. The DreamTaq DNA Polymerase was used for PCR reactions, and the conditions were as detailed above.</p></div><div id="Sec11" class="sec"><h3 id="Sec11title">The <em>pyr4</em> gene as a selectable marker for gene replacement</h3><p id="Par15" class="p p-first-last">In order to explore of uracil auxotrophic mutants as a genetic tool, deletion cassettes to replace the <em>blr1</em>, <em>blr2</em>, <em>tmk3</em>, and <em>pbs2</em> genes by the <em>pyr4</em> gene were generated. The Ura&#x02212;1 strain of <em>T. atroviride</em> was used as a parental strain. In a first PCR, the 5&#x02032; and 3&#x02032; flanking regions were amplified using primers for <em>tmk3</em> (Ptmk3-F &#x02013; pPQtmk3-R and pTQtmk3-F &#x02013; Ttmk3-R), <em>pbs2</em> (Ppbs2-F &#x02013; pPQpbs2-R and pTQpbs2-F &#x02013; Tpbs2-R), <em>blr1</em> (Pblr1-F &#x02013; pPQblr1-R and pTQblr1-F &#x02013; Tblr1-R), and <em>blr2</em> (Pblr2-F &#x02013; pPQblr2-R and pTQblr2-F &#x02013; Tblr2-R) genes, and the smPyr4-F - smPyr4-R primers were used to amplify 1247&#x000a0;bp of <em>pyr4</em> gene. In a second PCR, 5&#x02032; and 3&#x02032; flanking regions of each gene were joined to <em>pyr4</em> gene. In a third PCR, the cassettes were amplified using nested primers for <em>tmk3</em> (N5tmk3-F &#x02013; N3tmk3-R), <em>pbs2</em> (N5pbs2-F &#x02013; N3pbs2-R), <em>blr1</em> (N5blr1-F &#x02013; N3blr1-R), and <em>blr2</em> (N5blr2-F &#x02013; N3blr2-R) genes, and the transformation proceeded as described above. MM plates plus sorbitol 1&#x000a0;M were used as the selection medium. The gene replacement was determined by PCR with primer flanking the recombined 5&#x02032;- and 3&#x02032;-end sequences of <em>tmk3</em>, <em>pbs2</em>, <em>blr1</em>, and <em>blr2</em> genes. The DreamTaq DNA Polymerase was used for PCR reactions, and the conditions were as detailed above.</p></div><div id="Sec12" class="sec"><h3 id="Sec12title">Cellular stress assays</h3><p id="Par16" class="p p-first-last">Drops of 500 conidia of the WT, &#x02206;<em>pbs2</em>, and &#x02206;<em>tmk3</em> mutant strains were placed on PDA plates with 0.5% Triton X-100 plus KCl (0, 50 and 100&#x000a0;mM) for osmotic stress response; for heavy metal tolerance, cadmium (0, 400, and 800&#x000a0;mM CdCl<sub>2</sub>); for oxidative stress tolerance, menadione (0, 200, and 400&#x000a0;mM); or Congo red (0, 50, and 100&#x000a0;mM) to challenge cell wall integrity. To analyze osmotic stress tolerance in mycelia, precultures were generated on PDA plates with 2&#x000a0;&#x003bc;l of conidia of the WT, &#x02206;<em>pbs2</em>, and &#x02206;<em>tmk3</em> strains and allowed to grow for 40&#x000a0;h at 27&#x000a0;&#x000b0;C in darkness. Mycelial plugs from precultures were inoculated on PDA with NaCl (0 and 0.2&#x000a0;M) for osmotic stress. The cultures were incubated at 27&#x000a0;&#x000b0;C during 4&#x000a0;days, and pictures were taken. The experiments were carried out in triplicate.</p></div><div id="Sec13" class="sec"><h3 id="Sec13title">Analysis of conidiation induced by light and injure</h3><p id="Par17" class="p p-first-last">Light response assays were performed as previously described [<a href="#CR8" rid="CR8" class=" bibr popnode">8</a>]. Briefly, mycelial plugs from WT, &#x02206;<em>blr1</em>, and &#x02206;<em>blr2</em> precultures strains were inoculated on PDA plates and incubated for 40&#x000a0;h at 27&#x000a0;&#x000b0;C in darkness. To analyze conidiation stimulated by light, plates were exposed to blue light for 5&#x000a0;min (152.4&#x000a0;&#x003bc;mol&#x000a0;m<sup>&#x02212;2</sup>) and incubated for 48&#x000a0;h at 27&#x000a0;&#x000b0;C in darkness. To analyze conidiation stimulated by injury, colonies grown on darkness for 40&#x000a0;h were cut with a sterile scalpel to induce conidiation and incubated 48&#x000a0;h at 27&#x000a0;&#x000b0;C in darkness. The experiments were carried out in triplicate.</p></div><div id="Sec14" class="sec"><h3 id="Sec14title">Analysis of gene expression</h3><p id="Par18" class="p p-first-last">Expression of <em>blu</em>1, <em>grg</em>2, <em>env</em>1, and <em>phr</em>1 genes was determined in the WT strain, &#x02206;<em>pbs2</em>, &#x02206;<em>tmk3</em>, &#x02206;<em>blr1</em>, &#x02206;<em>blr2</em> mutants generated with the <em>pyr4</em> marker, and &#x02206;<em>pbs2</em>&#x02013;7; &#x02206;<em>tmk3</em>&#x02013;13 mutants were selected with the <em>hph</em> marker. Total RNA from mycelia exposed to blue light for 30&#x000a0;min (914.4&#x000a0;&#x003bc;mol&#x000a0;m<sup>&#x02212;2</sup>) or in darkness was extracted using TRIzol. DNase I and RNase-free (Thermo Scientific) were used to eliminate genomic DNA. Then, complementary DNA was synthesized following the manufacturer&#x02019;s recommendations (RevertAid Reverse Transcriptase; Thermo Scientific). To estimate transcript levels of genes regulated by light, 1&#x000a0;&#x003bc;l cDNA was used as template, and the DreamTaq DNA Polymerase was used for PCR reactions, following the conditions detailed above.</p></div><div id="Sec15" class="sec sec-last"><h3 id="Sec15title">Statistical analysis</h3><p id="Par19" class="p p-first-last">Statistical analysis and graphs were made with GraphPad Prism (version 5). The presented graph shows the average of three different experiments, and standard deviation is indicated. The data were analyzed through a one-way analysis of variance (ANOVA) with a Tukey-Kramer post-test.</p></div></div><div id="Sec16" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec16title">Results</h2><div id="Sec17" class="sec sec-first"><h3 id="Sec17title">Functional analysis of the <em>T. atroviride pyr4</em> gene encoding an OMP decarboxylase</h3><p id="Par20" class="p p-first">The <em>pyr4</em> gene identified in the <em>T. atroviride</em> genome encodes an OMP decarboxylase of 379 amino acids, with 91.6%, 67.3%, 56.0%, and 46.0% amino acids identities with OMP decarboxylases from <em>T. reesei</em> (<a id="__tag_1130671768" class="tag_hotlink" href="/protein/ETS06629.1" ref="/protein/ETS06629.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"ETS06629.1","term_id":"572283655"}}</span>ETS06629.1</a>), <em>Neurospora crassa</em> (<a id="__tag_1130671759" class="tag_hotlink" href="/protein/CAD21085.1" ref="/protein/CAD21085.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"CAD21085.1","term_id":"18376339"}}</span>CAD21085.1</a>), <em>A. nidulans</em> (<a id="__tag_1130671756" class="tag_hotlink" href="/protein/AAB66359.1" ref="/protein/AAB66359.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"AAB66359.1","term_id":"168076"}}</span>AAB66359.1</a>)<em>,</em> and <em>Saccharomyces cerevisiae</em> (<a id="__tag_1130671754" class="tag_hotlink" href="/protein/AAB64498.1" ref="/protein/AAB64498.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"AAB64498.1","term_id":"602388"}}</span>AAB64498.1</a>), respectively. <em>Trichoderma atroviride pyr4</em> does not contain introns similar to those of <em>T. reesei</em> and <em>N. crassa pyr4</em> genes unlike <em>pyrG</em> of <em>A. nidulans</em> and <em>Aspergillus niger</em> which contains two introns [<a href="#CR27" rid="CR27" class=" bibr popnode">27</a>], suggesting that also <em>T. atroviride</em> is more closely related to <em>N. crassa</em> than to the two <em>Aspergillus</em> species.</p><p id="Par21">In order to generate uracil auxotrophic mutants of <em>T. atroviride</em>, a knockout cassette to replace the <em>pyr4</em> gene by the <em>hph</em> selectable marker was constructed and used to transform <em>T. atroviride</em> protoplasts. Transformants were selected in MM plus 5&#x000a0;mM uracil and 100&#x000a0;&#x003bc;g&#x000a0;ml<sup>&#x02212;1</sup> hygromycin B. Eight uracil auxotrophic mutants were identified and were able to grow only on MM supplemented with uracil (Fig.&#x000a0;<a href="/pmc/articles/PMC7688867/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1a</span></a>). The gene replacement in auxotrophic strains was confirmed by PCR (Fig. <a href="#MOESM1" rid="MOESM1" class=" media">S1</a>). To evaluate the minimal amount of uracil required to supply the uracil starvation, the WT and mutant strains were cultured on MM with different uracil concentrations, and the growth rate was determined. The auxotrophic strains restored their growth with 5&#x000a0;mM uracil added to the MM (Fig.&#x000a0;<a href="/pmc/articles/PMC7688867/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1b</span></a>), and conidiation was induced by light and injury at a similar level as occurred in the WT strain (Fig. <a href="#MOESM1" rid="MOESM1" class=" media">S2</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig1" co-legend-rid="lgnd_Fig1"><a href="/pmc/articles/PMC7688867/figure/Fig1/" target="figure" rid-figpopup="Fig1" rid-ob="ob-Fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140069797935216"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 42770_2020_329_Fig1_HTML.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is 42770_2020_329_Fig1_HTML.jpg" src="/pmc/articles/PMC7688867/bin/42770_2020_329_Fig1_HTML.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140069797935216"><a target="object" rel="noopener" href="/pmc/articles/PMC7688867/figure/Fig1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig1"><div><a class="figpopup" href="/pmc/articles/PMC7688867/figure/Fig1/" target="figure" rid-figpopup="Fig1" rid-ob="ob-Fig1">Fig.&#x000a0;1</a></div><!--caption a7--><div class="caption"><p>Uracil auxotrophic strains lacking <em>pyr4</em> gene and uracil concentrations to restore auxotrophy. <strong>a</strong> Auxotrophy test for mutant identification. Drops of 500 conidia of WT, &#x02206;<em>pyr4</em>&#x02013;1, &#x02212;&#x02009;2, &#x02212;&#x02009;3, &#x02212;&#x02009;4, &#x02212;&#x02009;5, &#x02212;&#x02009;6, &#x02212;&#x02009;7, and &#x02212;&#x02009;8 strains (from top-left to bottom-right) were inoculated on MM plates with or without 5&#x000a0;mM uracil and incubated at 27&#x000a0;&#x000b0;C for 4&#x000a0;days. <strong>b</strong> Recover of uracil auxotrophy. Comparison among the mycelial growth rates of WT, &#x02206;<em>pyr4</em>&#x02013;3, &#x02212;&#x02009;4, and &#x02212;&#x02009;5 strains. Mycelial plugs were inoculated on MM plates plus different uracil concentrations. After 72&#x000a0;h, colony diameter was measured. Lines on bars represent standard errors. All experiments were carried out in triplicate</p></div></div></div><p id="Par22">5-FOA has commonly been used in fungi to select uracil auxotrophic strains without drug-resistance markers [<a href="#CR20" rid="CR20" class=" bibr popnode">20</a>, <a href="#CR21" rid="CR21" class=" bibr popnode">21</a>, <a href="#CR27" rid="CR27" class=" bibr popnode">27</a>]. In order to establish a positive selection, a conidial suspension of uracil auxotrophs and parental strain was inoculated on MM plates with 0.5% Triton X-100, 5&#x000a0;mM uracil, and different concentrations of 5-FOA (0.5, 1, 2, 3, and 4&#x000a0;mg&#x000a0;ml<sup>&#x02212;1</sup>). All auxotrophic strains were resistant to 4&#x000a0;mg&#x000a0;ml<sup>&#x02212;1</sup> 5-FOA, whereas the WT strain showed a strong reduction of growth in 0.5&#x000a0;mg&#x000a0;ml<sup>&#x02212;1</sup> of 5-FOA and was unable to grow at 1&#x000a0;mg&#x000a0;ml<sup>&#x02212;1</sup> or even higher concentrations of 5-FOA (Fig.&#x000a0;<a href="/pmc/articles/PMC7688867/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2a</span></a>). Interestingly, all auxotrophic strains were still able to produce conidia in 0.5&#x000a0;mg&#x000a0;ml<sup>&#x02212;1</sup> 5-FOA, but at higher concentrations, conidiation was severally inhibited, thus suggesting a relationship between conidiation and 5-FOA.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig2" co-legend-rid="lgnd_Fig2"><a href="/pmc/articles/PMC7688867/figure/Fig2/" target="figure" rid-figpopup="Fig2" rid-ob="ob-Fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140069797923008"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7688867_42770_2020_329_Fig2_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 42770_2020_329_Fig2_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC7688867/bin/42770_2020_329_Fig2_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140069797923008"><a target="object" rel="noopener" href="/pmc/articles/PMC7688867/figure/Fig2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig2"><div><a class="figpopup" href="/pmc/articles/PMC7688867/figure/Fig2/" target="figure" rid-figpopup="Fig2" rid-ob="ob-Fig2">Fig.&#x000a0;2</a></div><!--caption a7--><div class="caption"><p>Lethal dose of 5-FOA. <strong>a</strong> Drops of 500 conidia of WT, &#x02206;<em>pyr4</em>&#x02013;1, &#x02212;&#x02009;2, &#x02212;&#x02009;3, &#x02212;&#x02009;4, &#x02212;&#x02009;5, &#x02212;&#x02009;6, &#x02212;&#x02009;7, and &#x02212;&#x02009;8 strains (from top-left to bottom-right) were inoculated on MM plates plus 5&#x000a0;mM uracil and the indicated 5-FOA concentrations. <strong>b</strong> Effect of uracil concentrations on the sensitivity to 5-FOA. Drops of 500 conidia of strains indicated in Fig.&#x000a0;2a were inoculated on MM plates plus 2&#x000a0;mg&#x000a0;ml<sup>&#x02212;1</sup> 5-FOA and different uracil concentrations. Plates were incubated at 27&#x000a0;&#x000b0;C for 4&#x000a0;days. All experiments were performed in triplicate</p></div></div></div><p id="Par23" class="p p-last">Metabolic pathways can be subject to negative feedback inhibition, in which the final product decreases enzymatic activity or gene expression. For the above-mentioned reason, we wanted to examine if the selected uracil concentration has an effect on sensitivity to 5-FOA. A conidial suspension was inoculated on MM plus 2&#x000a0;mg&#x000a0;ml<sup>&#x02212;1</sup> 5-FOA and different uracil concentrations. Parental and auxotrophic strains showed the same phenotype, without changes in 5-FOA toxicity (Fig.&#x000a0;<a href="/pmc/articles/PMC7688867/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2b</span></a>). Altogether, 5&#x000a0;mM uracil is the optimal amount to restore uracil auxotrophy of <em>T. atroviride</em> properly, and 2&#x000a0;mg&#x000a0;ml<sup>&#x02212;1</sup> 5-FOA can be used for a positive selection approach to select uracil auxotrophic strains in this fungus.</p></div><div id="Sec18" class="sec"><h3 id="Sec18title">Generation of uracil auxotrophy by positive selection</h3><p id="Par24" class="p p-first">In fungi, uracil auxotrophy has been widely explored to use <em>ura5</em>/<em>pyr4</em>/<em>pyrG</em> or <em>ura3</em>/<em>pyr2</em> genes as selectable marker [<a href="#CR15" rid="CR15" class=" bibr popnode">15</a>, <a href="#CR26" rid="CR26" class=" bibr popnode">26</a>, <a href="#CR28" rid="CR28" class=" bibr popnode">28</a>&#x02013;<a href="#CR30" rid="CR30" class=" bibr popnode">30</a>]. In order to create uracil auxotrophic strains using positive selection with 5-FOA, a free-marker cassette was constructed to eliminate the <em>T. atroviride pyr4</em> gene. Given the fact that the plasmid pCB1004 acts as auto-replicative in <em>T. atroviride</em> (Esquivel-Naranjo and Herrera-Estrella, under review), we co-transformed protoplasts with the free-marker cassette and pCB1004 (2:1). To assure the auxotrophic mutants selection, transformants were recovered on MM plus 0.5&#x000a0;mg&#x000a0;ml<sup>&#x02212;1</sup> 5-FOA, 5&#x000a0;mM uracil, and 100&#x000a0;&#x003bc;g&#x000a0;ml<sup>&#x02212;1</sup> hygromycin B. <em>Trichoderma atroviride</em> protoplasts are multinucleated cells harboring transformed and non-transformed heterokaryotic nuclei. For this reason, we chose a 5-FOA sublethal concentration to select transformants. To obtain homokaryons, transformants were subjected to three rounds of single-spore isolation with modifications. Firstly, the selective medium comprised 5&#x000a0;mM uracil, 2&#x000a0;mg&#x000a0;ml<sup>&#x02212;1</sup> 5-FOA, and 100&#x000a0;&#x003bc;g&#x000a0;ml<sup>&#x02212;1</sup> hygromycin B. Secondly, the colonies were grown on MM supplemented with 5&#x000a0;mM uracil and 0.5&#x000a0;mg&#x000a0;ml<sup>&#x02212;1</sup> 5-FOA to produce conidia. Ten stable mutants were identified by an auxotrophy test and designated Ura<sup>&#x02212;</sup>1-10 (Fig.&#x000a0;<a href="/pmc/articles/PMC7688867/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span>3a</span></a>). All were unable to grow on uracil-deficient MM and sensitive to hygromycin B, indicating that the pCB1004 plasmid was lost in all auxotrophic strains when this antibiotic was not added to the MM. The auxotrophic strains were then analyzed in their mycelium growth, production of conidia stimulated by light, and injury on PDA plates plus 5&#x000a0;mM uracil. All uracil auxotrophic mutants were stable and showed a similar behavior to the parental strain, suggesting that these strains can be used as a reliable receiver strain for genetic manipulation (data not presented). Unexpectedly, when we amplified the coding region of the <em>pyr4</em> gene, the amplicons of six auxotrophic strains showed an electrophoretic mobility similar to that of the WT strain, whereas the amplicon of four auxotrophic strains (Ura<sup>&#x02212;</sup>1, Ura<sup>&#x02212;</sup>4, Ura<sup>&#x02212;</sup>9, and Ura<sup>&#x02212;</sup>10) was slightly shorter (Fig. <a href="#MOESM1" rid="MOESM1" class=" media">S3</a>). In order to identify mutations in the coding region of the <em>pyr4</em> gene, it was amplified from WT and Ura<sup>&#x02212;</sup>1, &#x02212;&#x02009;2, &#x02212;&#x02009;3, &#x02212;&#x02009;4, &#x02212;&#x02009;5, &#x02212;&#x02009;6, &#x02212;&#x02009;7, &#x02212;&#x02009;9, and &#x02212;&#x02009;10 strains and sequenced. As expected, the Ura<sup>&#x02212;</sup>2, &#x02212;&#x02009;3, &#x02212;&#x02009;5, &#x02212;&#x02009;6, and &#x02212;&#x02009;7 strains did not have changes in the <em>pyr4</em> nucleotide sequences and were identical to WT strain, whereas Ura<sup>&#x02212;</sup>1, &#x02212;&#x02009;4, and &#x02212;&#x02009;10 had a deletion of 139&#x000a0;bp and Ura<sup>&#x02212;</sup>9 deletion of 23&#x000a0;bp at the positions 928&#x000a0;bp and 997&#x000a0;bp, respectively (Fig. <a href="#MOESM1" rid="MOESM1" class=" media">S4</a>), considering the start codon. The deletion caused a frameshift, where the Ura<sup>&#x02212;</sup>1, &#x02212;&#x02009;4, and &#x02212;&#x02009;10 strains encoded a polypeptide of 351 amino acids of which 309 were identical, and Ura<sup>&#x02212;</sup>9 encoded 345 amino acids of which 334 were identical. Both of these were shorter than the Pyr4 protein (379 amino acids) of the WT strain (Fig.&#x000a0;<a href="/pmc/articles/PMC7688867/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span>3b&#x02013;c</span></a>). Given that, all produced auxotrophic strains showing at least two different mutations, and our results suggest that 5-FOA could be a mutagenic chemical or it could promote genomic instability, provoking mutagenesis. Furthermore, these data shows that the last 45 amino acids in the COOH-terminal of Pyr4 are essential for uracil biosynthesis.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig3" co-legend-rid="lgnd_Fig3"><a href="/pmc/articles/PMC7688867/figure/Fig3/" target="figure" rid-figpopup="Fig3" rid-ob="ob-Fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140069797887520"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7688867_42770_2020_329_Fig3_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 42770_2020_329_Fig3_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC7688867/bin/42770_2020_329_Fig3_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140069797887520"><a target="object" rel="noopener" href="/pmc/articles/PMC7688867/figure/Fig3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig3"><div><a class="figpopup" href="/pmc/articles/PMC7688867/figure/Fig3/" target="figure" rid-figpopup="Fig3" rid-ob="ob-Fig3">Fig.&#x000a0;3</a></div><!--caption a7--><div class="caption"><p>Analysis of auxotrophic mutants selected by resistance to 5-FOA. <strong>a</strong> Drops of 500 conidia of WT, Ura<sup>&#x02212;</sup>1, &#x02212;&#x02009;2, &#x02212;&#x02009;3, &#x02212;&#x02009;4, &#x02212;&#x02009;5, &#x02212;&#x02009;7, &#x02212;&#x02009;8, and &#x02212;&#x02009;9 strains (from top-left to bottom-right) were inoculated on MM plates plus different uracil concentrations and incubated at 27&#x000a0;&#x000b0;C for 4&#x000a0;days. This assay was carried out in triplicate. <strong>b</strong> Schematic representation of <em>pyr4</em> deletions. Deletions and positions were determined by alignment of the nucleotide sequence using the Clustal V method. <strong>c</strong> Protein alignment conducted among Pyr4 proteins from WT, Ura<sup>&#x02212;</sup>1, and Ura<sup>&#x02212;</sup>9 strains using the Clustal V method. Arrows indicate the end of the conserved amino acid sequence for the indicated strain</p></div></div></div><p id="Par25" class="p p-last">The Ura<sup>&#x02212;</sup>1 and Ura<sup>&#x02212;</sup>9 strains were selected for complementation tests due to their truncated <em>pyr4</em> gene. The coding region of the WT <em>pyr4</em> gene, including 1372-bp upstream and 1548-bp downstream, was amplified using the primers Ppyr4-F and Tpyr4-R (4060&#x000a0;bp), cloned into a pCR2.1-TOPO vector (Invitrogen) and used to complement uracil auxotrophic strains. In both strains, the <em>pyr4</em> gene transformed the uracil auxotrophy to prototrophy (Fig. <a href="#MOESM1" rid="MOESM1" class=" media">S5</a>), indicating that the uracil auxotrophy was due to a truncation in the <em>pyr4</em> gene, as it was demonstrated by <em>pyr4</em> sequencing analysis.</p></div><div id="Sec19" class="sec"><h3 id="Sec19title">Deletion of genes encoding the MAPK Tmk3 and MAPKK Pbs2 using <em>pyr4</em> as selectable marker</h3><p id="Par26" class="p p-first">To evaluate if the uracil auxotrophic mutants can be used as a genetic tool for gene replacement and to validate if <em>pyr4</em> restores the uracil starvation, we designed cassettes to replace the <em>tmk3</em> and <em>pbs2</em> genes with the <em>pyr4</em> gene. The <em>tmk3</em> and <em>pbs2</em> genes encode for MAPK Tmk3 and MAPKK Pbs2, respectively, related to light and stress responses [<a href="#CR8" rid="CR8" class=" bibr popnode">8</a>]. These genes were selected to compare our results with those that have been previously reported in mutants selected by <em>hph</em> gene (&#x00394;<em>tmk3</em>&#x02013;13:<em>hph</em> and &#x00394;<em>pbs2</em>&#x02013;7:<em>hph</em> mutants). To avoid gene replacement at the <em>pyr4</em> locus and favor the targeted knockouts, the Ura<sup>&#x02212;</sup>1 strain was chosen because the truncation is bigger than in Ura<sup>&#x02212;</sup>9. To construct the deletion cassettes, the <em>pyr4</em> ORF was amplified with smPyr4-F and smPyr4-R primers to generate a selectable marker of 1247&#x000a0;bp (71-bp upstream&#x02014;coding region&#x02014;36-bp downstream). In this sense, the <em>pyr4</em> marker was positioned under the control of the promoter and terminator of the deleted genes.</p><p id="Par27" class="p p-last">Transformation was carried out, and eight out of 24 transformants (33.3%) were mutants lacking the <em>tmk3</em> gene (&#x00394;<em>tmk3</em>:<em>pyr</em><em>4</em> mutants), and three out of 20 (15%) were &#x02206;<em>pbs2</em>:<em>pyr4</em> mutants (Fig. <a href="#MOESM1" rid="MOESM1" class=" media">S6&#x02013;S7</a>), all able to grow on MM without uracil. Conidiation stimulated by light in &#x02206;<em>tmk3</em>:<em>pyr4</em> and &#x02206;<em>pbs2</em>:<em>pyr4</em> strains was analyzed, and in all mutants, strains conidiation was reduced to 80%, similar to data obtained with &#x02206;<em>tmk3</em>:<em>hph</em> and &#x02206;<em>pbs2</em>:<em>hph</em> strains (Fig.&#x000a0;<a href="/pmc/articles/PMC7688867/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4a&#x02013;b</span></a>). As reported before [<a href="#CR8" rid="CR8" class=" bibr popnode">8</a>], all mutants were sensitive to osmotic stress, Congo red, CdCl<sub>2</sub>, and menadione (Fig.&#x000a0;<a href="/pmc/articles/PMC7688867/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4c&#x02013;d</span></a>). There was no difference between mutants selected by <em>hph</em> or <em>pyr4</em> markers, indicating that the Ura<sup>&#x02212;</sup>1 strain was not affected neither in conidia production stimulated by light nor cellular stress responses.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig4" co-legend-rid="lgnd_Fig4"><a href="/pmc/articles/PMC7688867/figure/Fig4/" target="figure" rid-figpopup="Fig4" rid-ob="ob-Fig4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140069761326576"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7688867_42770_2020_329_Fig4_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 42770_2020_329_Fig4_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC7688867/bin/42770_2020_329_Fig4_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140069761326576"><a target="object" rel="noopener" href="/pmc/articles/PMC7688867/figure/Fig4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig4"><div><a class="figpopup" href="/pmc/articles/PMC7688867/figure/Fig4/" target="figure" rid-figpopup="Fig4" rid-ob="ob-Fig4">Fig.&#x000a0;4</a></div><!--caption a7--><div class="caption"><p>Validation of <em>pyr4</em> gene as a selectable marker in <em>T. atroviride</em> for gene deletion. <strong>a</strong> Comparison between WT and mutant strains selected with the <em>hph</em> marker (&#x02206;<em>tmk3</em>&#x02013;13 and &#x02206;<em>pbs2</em>&#x02013;7) and <em>pyr4</em> marker (&#x02206;<em>tmk3</em>&#x02013;4 and &#x02206;<em>pbs2</em>&#x02013;5) grown on PDA plates in constant white light (0.586&#x000a0;&#x003bc;mol&#x000a0;m<sup>&#x02212;2</sup>&#x000a0;s<sup>&#x02212;1</sup>) for 7&#x000a0;days at 27&#x000a0;&#x000b0;C. <strong>b</strong> Conidial production of the WT strain and mutants selected with the <em>hph</em> marker (&#x02206;<em>tmk3</em>&#x02013;13 and &#x02206;<em>pbs2</em>&#x02013;7) and the <em>pyr4</em> marker (&#x02206;<em>pbs2</em>&#x02013;5, &#x02212;&#x02009;15, &#x02212;&#x02009;18, &#x02206;<em>tmk3</em>&#x02013;4, &#x02212;&#x02009;12, and &#x02212;&#x02009;16) grown on PDA plates in constant white light for 7&#x000a0;days at 27&#x000a0;&#x000b0;C. <strong>c</strong> Stress sensitivity in conidia of &#x02206;<em>tmk3</em> and &#x02206;<em>pbs2</em> strains. Drops of 500 conidia of &#x02206;<em>tmk3&#x02013;</em>4, &#x02212;&#x02009;12, &#x02212;&#x02009;16, &#x02212;&#x02009;13, WT, &#x02206;<em>pbs2</em>&#x02013;7, &#x02212;&#x02009;5, &#x02212;&#x02009;15, and &#x02212;&#x02009;18 strains (from top-left to bottom-right) were inoculated on PDA plates plus different concentrations of the indicated stressors. They were then, incubated in constant white light or darkness for 4&#x000a0;days at 27&#x000a0;&#x000b0;C. <strong>d</strong> Osmotic stress sensitivity in mycelia of &#x02206;<em>tmk3</em> and &#x02206;<em>pbs2</em> strains. Mycelial plugs of the indicated strains were inoculated on PDA plates with or without 200&#x000a0;mM NaCl and incubated for 4&#x000a0;days at 27&#x000a0;&#x000b0;C. All experiments were carried out in triplicate</p></div></div></div></div><div id="Sec20" class="sec"><h3 id="Sec20title">Deletion of genes encoding the blue-light receptor Blr1/2 using the <em>pyr4</em> marker</h3><p id="Par28" class="p p-first-last">To acquire more evidence on the application of Ura<sup>&#x02212;</sup>1 strain as a genetic background, cassettes were designed to replace the <em>blr1</em> and <em>blr2</em> coding regions by the <em>pyr4</em> gene marker. Eight out of 20 transformants (40%) were <em>blr1</em> mutants, and ten out of 18 transformants (55.5%) were <em>blr2</em> mutants (Fig. <a href="#MOESM1" rid="MOESM1" class=" media">S8&#x02013;S9</a>). The &#x02206;<em>blr1</em>:<em>pyr4</em> and &#x02206;<em>blr2</em>:<em>pyr4</em> mutants were exposed to a blue-light pulse (152.4&#x000a0;&#x003bc;mol&#x000a0;m<sup>&#x02212;2</sup>). As expected, the &#x02206;<em>blr1</em> and &#x02206;<em>blr2</em> strains were not photoresponsive, and the phenotype in light was the same as the control strain kept in darkness. Thus, mutant strains could not form a conidia ring in response to light on PDA medium (Fig.&#x000a0;<a href="/pmc/articles/PMC7688867/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5a&#x02013;b</span></a>), but they conidiated in response to injury as described before [<a href="#CR31" rid="CR31" class=" bibr popnode">31</a>].</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig5" co-legend-rid="lgnd_Fig5"><a href="/pmc/articles/PMC7688867/figure/Fig5/" target="figure" rid-figpopup="Fig5" rid-ob="ob-Fig5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140069761303728"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7688867_42770_2020_329_Fig5_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 42770_2020_329_Fig5_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC7688867/bin/42770_2020_329_Fig5_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140069761303728"><a target="object" rel="noopener" href="/pmc/articles/PMC7688867/figure/Fig5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig5"><div><a class="figpopup" href="/pmc/articles/PMC7688867/figure/Fig5/" target="figure" rid-figpopup="Fig5" rid-ob="ob-Fig5">Fig.&#x000a0;5</a></div><!--caption a7--><div class="caption"><p>Phenotypes of <em>T. atroviride</em> WT, &#x02206;<em>blr1</em>, and &#x02206;<em>blr2</em> strains. <strong>a</strong> Mycelial plugs of WT, &#x02206;<em>blr1</em>&#x02013;&#x000a0;14, &#x02212;&#x02009;17, &#x02212;&#x02009;18, &#x02212;&#x02009;19, and &#x02212;&#x02009;20 strains or <strong>b</strong> &#x02206;<em>blr2</em>&#x02013;5, &#x02212;&#x02009;9, &#x02212;&#x02009;10, &#x02212;&#x02009;15, and &#x02212;&#x02009;16 strains were inoculated on PDA plates and incubated for 40&#x000a0;h at 27&#x000a0;&#x000b0;C. Then, a blue-light pulse (152.4&#x000a0;&#x003bc;mol&#x000a0;m<sup>&#x02212;2</sup>) was applied for light-response assays or the mycelia were injured with a sterile scalpel for wound response. Pictures were taken 48&#x000a0;h later. Strains kept in darkness are shown as control. All experiments were performed in triplicate</p></div></div></div></div><div id="Sec21" class="sec sec-last"><h3 id="Sec21title">Selection by <em>pyr4</em> did not affect the expression of genes regulated by light</h3><p id="Par29" class="p p-first-last">All mutants selected by the <em>pyr4</em> marker showed the same phenotype in stress tolerance and photoconidiation, such as previously described using <em>hph</em> marker [<a href="#CR8" rid="CR8" class=" bibr popnode">8</a>, <a href="#CR31" rid="CR31" class=" bibr popnode">31</a>]. However, gene expression epistasis has been associated with the use of auxotrophic strains in <em>S. cerevisiae</em> [<a href="#CR32" rid="CR32" class=" bibr popnode">32</a>]. In order to discard changes in gene expression generated by the selectable marker, the expression of four light-regulated genes was examined in the &#x02206;<em>blr1</em>:<em>pyr4</em>, &#x02206;<em>blr2</em>:<em>pyr4</em>, &#x02206;<em>tmk3</em>:<em>hph</em>, &#x02206;<em>tmk3</em>:<em>pyr4</em>, &#x02206;<em>pbs2</em>:<em>hph</em>, and &#x02206;<em>pbs2</em>:<em>pyr</em>4 strains. As reported previously [<a href="#CR8" rid="CR8" class=" bibr popnode">8</a>, <a href="#CR31" rid="CR31" class=" bibr popnode">31</a>, <a href="#CR33" rid="CR33" class=" bibr popnode">33</a>], <em>blu1</em>, <em>grg2</em>, <em>phr1</em> (CPD-photolyase), and <em>env1</em> (photoreceptor for blue-light tolerance) gene expression was not detected in darkness; however, after a blue-light pulse, their transcript levels were induced in the WT strain. In the &#x02206;<em>tmk3</em> and &#x02206;<em>pbs2</em> strains, expression of <em>blu1</em> and <em>grg2</em> genes was not induced, whereas <em>phr1</em> and <em>env1</em> genes were responsive to blue light at a similar level as the WT strain. Under the same conditions, the blue-light-regulated genes presented similar patterns in the &#x02206;<em>tmk3</em> and &#x02206;<em>pbs2</em> mutants obtained with <em>pyr4</em> and <em>hph</em> markers (Fig.&#x000a0;<a href="/pmc/articles/PMC7688867/figure/Fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig6" rid-ob="ob-Fig6" co-legend-rid="lgnd_Fig6"><span>6</span></a>). In the &#x02206;<em>blr1</em> and &#x02206;<em>blr2</em> strains, <em>blu1</em>, <em>grg2</em>, <em>phr1</em>, and <em>env1</em> genes were not induced by blue-light (Fig.&#x000a0;<a href="/pmc/articles/PMC7688867/figure/Fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig6" rid-ob="ob-Fig6" co-legend-rid="lgnd_Fig6"><span>6</span></a>), indicating a preponderant function of Blr1/2 blue-light receptors on light responses in <em>T. atroviride</em>. Altogether, these data provide evidence that the <em>T. atroviride</em> Ura<sup>&#x02212;</sup>1 strain can be used as a background for genomic functional studies.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig6" co-legend-rid="lgnd_Fig6"><a href="/pmc/articles/PMC7688867/figure/Fig6/" target="figure" rid-figpopup="Fig6" rid-ob="ob-Fig6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140069761702144"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7688867_42770_2020_329_Fig6_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 42770_2020_329_Fig6_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC7688867/bin/42770_2020_329_Fig6_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140069761702144"><a target="object" rel="noopener" href="/pmc/articles/PMC7688867/figure/Fig6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig6"><div><a class="figpopup" href="/pmc/articles/PMC7688867/figure/Fig6/" target="figure" rid-figpopup="Fig6" rid-ob="ob-Fig6">Fig. 6</a></div><!--caption a7--><div class="caption"><p>Expression of light-responsive genes in &#x02206;<em>pbs2</em>, &#x02206;<em>tmk3</em>, &#x02206;<em>blr1</em>, and &#x02206;<em>blr2</em> strains. An RT-PCR was performed to compare the expression of <em>blu1</em>, <em>grg2</em>, <em>phr1</em>, and <em>env1</em> genes in WT and mutants selected with the <em>hph</em> marker (&#x02206;<em>pbs2</em>&#x02013;7 and &#x02206;<em>tmk3</em>&#x02013;13) and the <em>pyr4</em> marker (&#x02206;<em>pbs2</em>&#x02013;15, &#x02206;<em>tmk3</em>&#x02013;4, &#x02206;<em>blr1</em>&#x02013;14, and &#x02206;<em>blr2</em>&#x02013;15). The strains were kept in darkness for 48&#x000a0;h. Then, the strains were exposed to blue light for 30&#x000a0;min (914.4&#x000a0;&#x003bc;mol&#x000a0;m<sup>&#x02212;2</sup>). After the blue-light induction, total RNA was extracted with TRIzol. The <em>gpd</em> gene was used as control, and this experiment were carried out in duplicate obtaining practically the same results. Control WT: control without RT. D: darkness. L: light.</p></div></div></div></div></div><div id="Sec22" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec22title">Discussion</h2><p id="Par30" class="p p-first">The use of auxotrophic mutants as a genetic tool is an option to improve our knowledge of gene function in a low-cost way. Here we describe our transformation system in <em>T. atroviride</em> to obtain uracil auxotrophs free of drug-selectable markers, based on the 5-FOA positive selection. The <em>pyrG</em>/<em>pyr4</em> gene in <em>T. reesei</em>, <em>T. harzianum,</em> and <em>T. hypoxylon</em> encodes orotidine-5&#x02032;-monophosphate decarboxylase [<a href="#CR15" rid="CR15" class=" bibr popnode">15</a>, <a href="#CR22" rid="CR22" class=" bibr popnode">22</a>, <a href="#CR27" rid="CR27" class=" bibr popnode">27</a>]. The absence of <em>pyr4</em> in <em>T. reesei</em> can be restored by the <em>pyrG</em> gene of <em>A. niger</em> and the <em>pyr4</em> gene of <em>N. crassa</em> [<a href="#CR14" rid="CR14" class=" bibr popnode">14</a>], indicating that <em>pyr4</em> is highly conserved in fungi. Furthermore, it is possible to use uracil auxotrophs in <em>T. atroviride</em> to establish homologous and heterologous transformation system.</p><p id="Par31">To determine growth conditions, uracil auxotrophs were generated using resistance to hygromycin B. The results demonstrate that the knockout of <em>pyr4</em> in <em>T. atroviride</em> leads to a uracil-deficient strain. Uracil in the medium restores growth of mutants, and the &#x00394;<em>pyr4</em> strains were able to grow in media containing 5-FOA. This drug has already been used in <em>Trichoderma</em> species to select strains mutagenized through UV-light irradiation [<a href="#CR14" rid="CR14" class=" bibr popnode">14</a>] and knockout [<a href="#CR15" rid="CR15" class=" bibr popnode">15</a>, <a href="#CR26" rid="CR26" class=" bibr popnode">26</a>]. The 5-FOA lethal dose was determined to begin at 1&#x000a0;mg&#x000a0;ml<sup>&#x02212;1</sup>, and high uracil concentrations have no effect on 5-FOA toxicity. Together, data obtained were used to establish conditions for a markerless transformation in <em>T. atroviride</em>.</p><p id="Par32">Techniques have been developed to eliminate the <em>pyr4</em> gene without a drug-selectable marker in <em>T. reesei</em> and <em>T. hypoxylon</em> [<a href="#CR15" rid="CR15" class=" bibr popnode">15</a>, <a href="#CR26" rid="CR26" class=" bibr popnode">26</a>]. In <em>T. reesei</em>, a genetic background lacking a non-homologous end-joining (NHEJ) repair system was used to efficiently eliminate the coding region of <em>pyr4</em>, but in <em>T. hypoxylon</em>, the WT strain was employed. Although a free-marker cassette was designed as previously described [<a href="#CR15" rid="CR15" class=" bibr popnode">15</a>, <a href="#CR26" rid="CR26" class=" bibr popnode">26</a>], with the promoter and terminator fragments joined, excluding the ORF, our results showed that the mutant strains were generated by a different mechanism. Analysis of <em>pyr4</em> deletion strains by PCR and sequencing revealed that the <em>pyr4</em> ORF of four uracil auxotrophs (Ura<sup>&#x02212;</sup>1, Ura<sup>&#x02212;</sup>4, Ura<sup>&#x02212;</sup>9, and Ura<sup>&#x02212;</sup>10) was truncated by at least two independent events. Even though the <em>pyr4</em> gene was not completely deleted, the produced alleles encoded a non-functional Pyr4 protein that was unable to produce uracil, which was demonstrated by auxotrophy, and were successfully complemented with the WT <em>pyr4</em> gene. On the other hand, six auxotrophic strains showed bands with electrophoretic mobility comparable with WT <em>pyr4</em> locus, suggesting that the mutations occurred outside <em>pyr4</em> coding region as confirmed by sequencing in Ura<sup>&#x02212;</sup>2, &#x02212;&#x02009;3, &#x02212;&#x02009;5, &#x02212;&#x02009;6, and &#x02212;&#x02009;7 auxotrophic strains. The auxotrophs without changes in <em>pyr</em>4 ORF could be explained by a mutation in a cis or trans element necessary for <em>pyr4</em> expression. Another possibility that able to explain our results is that a mutation took place in another gene involved in pyrimidine biosynthesis, such as the <em>ura3</em>/<em>pyr2</em> gene encoding OPRTase. These results suggest that 5-FOA may be mutagenic or a stress condition caused by genomic instability during transformation process. In this sense, 5-FU and FdUMP, toxic molecules produced by 5-FOA metabolism, can be incorporated into RNA or DNA matching 5-FU:A or 5-FU:G and induce genomic instability [<a href="#CR18" rid="CR18" class=" bibr popnode">18</a>, <a href="#CR19" rid="CR19" class=" bibr popnode">19</a>]. <em>Saccharomyces cerevisiae</em> mutants lacking Apn1, the major abasic site endonuclease, showed a pronounced sensitivity to 5-FU, suggesting that abasic sites formed during DNA repair are more toxic, which can provoke DNA strand breaks [<a href="#CR34" rid="CR34" class=" bibr popnode">34</a>]. However, 5-FOA has not been documented to cause DNA deletions. More data supporting this hypothesis comes from experiments in <em>Candida albicans</em>. When <em>C. albicans</em> was exposed to 5-FOA, its chromosomes suffered alteration [<a href="#CR35" rid="CR35" class=" bibr popnode">35</a>, <a href="#CR36" rid="CR36" class=" bibr popnode">36</a>], presumably as an adaptive mutation under the stress conditions by 5-FOA that caused genetic changes. The above can explain the mutation that occurred in <em>T. atroviride</em>, as well as the partial loss of the <em>pyr4</em> ORF by 5-FOA added in the culture medium.</p><p id="Par33">Furthermore, we demonstrated that the Ura<sup>&#x02212;</sup>1 strain can be used as background to knockout genes in <em>T. atroviride</em>. The <em>blr1</em>, <em>blr2</em>, <em>pbs2</em>, and <em>tmk3</em> genes previously characterized [<a href="#CR8" rid="CR8" class=" bibr popnode">8</a>, <a href="#CR31" rid="CR31" class=" bibr popnode">31</a>] were successfully replaced by <em>pyr4</em> in the Ura<sup>&#x02212;</sup>1 strain. The efficiency ranged from 15 to 55% among the genes included in our analysis. Noticeably, expression of <em>blr1</em> and <em>blr2</em> genes has been described to be undetectable by Northern-blot analysis, indicating that those genes are little expressed, whereas higher transcript levels were detected in <em>tmk3</em> and <em>pbs2</em> genes [<a href="#CR31" rid="CR31" class=" bibr popnode">31</a>, <a href="#CR37" rid="CR37" class=" bibr popnode">37</a>]. Despite that the <em>pyr4</em> gene was under control of the promoter of these deleted genes, in all cases uracil auxotrophy was successfully complemented. The conidial production and gene expression in mutants obtained from the auxotroph strain were evaluated, and the results are comparable with those of mutants obtained with the <em>hph</em> marker. Although the average efficiency of gene replacement was 36%, it was similar using the <em>hph</em> marker [<a href="#CR8" rid="CR8" class=" bibr popnode">8</a>], suggesting that the knockout cassettes were targeted mainly to the corresponding locus.</p><p id="Par34">In <em>S. cerevisiae</em>, auxotrophies alter the genetic background which induces different transcriptional responses [<a href="#CR32" rid="CR32" class=" bibr popnode">32</a>]. Therefore, we evaluated the level of expression of four light-regulated genes in mutants obtained with a uracil auxotrophic strain, and the analyzed genes did not exhibited alterations in transcript levels. This suggests that the behavior of <em>T. atroviride</em> is different from that described in <em>S. cerevisiae</em>, although we cannot discard epistasis in other genes not included in this work. Our results show that the Ura<sup>&#x02212;</sup>1 strain serves as a reliable genetic background, given that transcriptional perturbations were not detected under our experimental conditions.</p><p id="Par35" class="p p-last">In conclusion, the Ura<sup>&#x02212;</sup>1 strain is an alternative tool to disrupt or complement genes in <em>T. atroviride</em>. Despite the fact that the uracil auxotroph was not obtained by knockout as initially designed, we did not observe differences in phenotype in comparison with mutants obtained with the <em>hph</em> marker. It would be interesting to know if a heterologous <em>pyr4</em> gene from other fungi can improve the gene replacement efficiency in <em>T. atroviride</em>.</p></div><div id="sec-a.l.e" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="sec-a.l.etitle">Electronic supplementary material</h2><!--/article/body/sec/--><div id="Sec23" class="sec sec-first-last"><div class="sec suppmat" id="MOESM1"><div class="sup-box half_rhythm" id="media-a.l.e.b.a.a"><a href="/pmc/articles/PMC7688867/bin/42770_2020_329_MOESM1_ESM.docx" data-ga-action="click_feat_suppl">ESM 1</a><sup>(5.3M, docx)</sup><p>(DOCX 5456&#x000a0;kb).</p></div></div></div></div><div id="notes-a.m.b" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.m.btitle">Funding information</h2><p>This work was supported by CONACYT (Consejo Nacional de Ciencia y Tecnolog&#x000ed;a, CB-2011-169,045) and FOFI-UAQ-2012 (Fondo para el fortalecimiento de la investigaci&#x000f3;n&#x02014;Universidad Aut&#x000f3;noma de Quer&#x000e9;taro, FNB-2012-10) grants to E.U. E-N. G. C-H is indebted to CONACYT for a doctoral fellowship.</p></div><div id="notes-a.m.c" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.m.ctitle">Compliance with ethical standards</h2><div>Conflict of interest<p id="Par36">The authors declare that they have no conflicts of interest.</p></div></div><div id="fn-group-a.m.a" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.m.atitle">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="fn-a.m.a.a"><p class="p p-first"><strong>Publisher&#x02019;s note</strong></p><p class="p p-last">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></p></div></div><div id="Bib1" class="tsec sec"><h2 class="head no_bottom_margin" id="Bib1title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="CR1">1. <span class="element-citation">Braun H, Woitsch L, Hetzer B, Geisen R, Zange B, Schmidt-Heydt M. <em>Trichoderma harzianum</em>: inhibition of mycotoxin producing fungi and toxin biosynthesis. <span><span class="ref-journal">Int J Food Microbiol. </span>2018;<span class="ref-vol">280</span>:10–16. doi: 10.1016/j.ijfoodmicro.2018.04.021.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29754002" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.ijfoodmicro.2018.04.021" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Food+Microbiol&amp;title=Trichoderma+harzianum:+inhibition+of+mycotoxin+producing+fungi+and+toxin+biosynthesis&amp;author=H+Braun&amp;author=L+Woitsch&amp;author=B+Hetzer&amp;author=R+Geisen&amp;author=B+Zange&amp;volume=280&amp;publication_year=2018&amp;pages=10-16&amp;pmid=29754002&amp;doi=10.1016/j.ijfoodmicro.2018.04.021&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR2">2. <span class="mixed-citation">Doehlemann G, &#x000d6;kmen B, Zhu W, Sharon A (2017) Plant pathogenic fungi. Microbiol Spectr:703&#x02013;726. 10.1128/microbiolspec.FUNK-0023-2016 [<a href="https://pubmed.ncbi.nlm.nih.gov/28155813" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="CR3">3. <span class="element-citation">Lee S, Yap M, Behringer G, Hung R, Bennett JW. Volatile organic compounds emitted by <em>Trichoderma</em> species mediate plant growth. <span><span class="ref-journal">Fungal Biol Biotechnol. </span>2016;<span class="ref-vol">3</span>(1):7. doi: 10.1186/s40694-016-0025-7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5611631/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28955466" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2Fs40694-016-0025-7" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Fungal+Biol+Biotechnol&amp;title=Volatile+organic+compounds+emitted+by+Trichoderma+species+mediate+plant+growth&amp;author=S+Lee&amp;author=M+Yap&amp;author=G+Behringer&amp;author=R+Hung&amp;author=JW+Bennett&amp;volume=3&amp;issue=1&amp;publication_year=2016&amp;pages=7&amp;pmid=28955466&amp;doi=10.1186/s40694-016-0025-7&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR4">4. <span class="element-citation">Guzm&#x000e1;n-Guzm&#x000e1;n P, Porras-Troncoso MD, Olmedo-Monfil V, Herrera-Estrella A. <em>Trichoderma</em> species: versatile plant symbionts. <span><span class="ref-journal">Phytopathology. </span>2019;<span class="ref-vol">109</span>(1):6–16. doi: 10.1094/PHYTO-07-18-0218-RVW.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30412012" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1094%2FPHYTO-07-18-0218-RVW" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Phytopathology&amp;title=Trichoderma+species:+versatile+plant+symbionts&amp;author=P+Guzm&#x000e1;n-Guzm&#x000e1;n&amp;author=MD+Porras-Troncoso&amp;author=V+Olmedo-Monfil&amp;author=A+Herrera-Estrella&amp;volume=109&amp;issue=1&amp;publication_year=2019&amp;pages=6-16&amp;pmid=30412012&amp;doi=10.1094/PHYTO-07-18-0218-RVW&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR5">5. <span class="element-citation">Brunner K, Zeilinger S, Ciliento R, Woo SL, Lorito M, Kubicek CP, Mach RL. Improvement of the fungal biocontrol agent <em>Trichoderma atroviride</em> to enhance both antagonism and induction of plant systemic disease resistance. <span><span class="ref-journal">Appl Environ Microbiol. </span>2005;<span class="ref-vol">71</span>(7):3959–3965. doi: 10.1128/AEM.71.7.3959-3965.2005.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1168994/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16000810" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FAEM.71.7.3959-3965.2005" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Appl+Environ+Microbiol&amp;title=Improvement+of+the+fungal+biocontrol+agent+Trichoderma+atroviride+to+enhance+both+antagonism+and+induction+of+plant+systemic+disease+resistance&amp;author=K+Brunner&amp;author=S+Zeilinger&amp;author=R+Ciliento&amp;author=SL+Woo&amp;author=M+Lorito&amp;volume=71&amp;issue=7&amp;publication_year=2005&amp;pages=3959-3965&amp;pmid=16000810&amp;doi=10.1128/AEM.71.7.3959-3965.2005&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR6">6. <span class="element-citation">Steyaert JM, Weld RJ, Mendoza-Mendoza A, Stewart A. Reproduction without sex: conidiation in the filamentous fungus <em>Trichoderma</em>. <span><span class="ref-journal">Microbiology. </span>2010;<span class="ref-vol">156</span>(10):2887–2900. doi: 10.1099/mic.0.041715-0.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20688823" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1099%2Fmic.0.041715-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology&amp;title=Reproduction+without+sex:+conidiation+in+the+filamentous+fungus+Trichoderma&amp;author=JM+Steyaert&amp;author=RJ+Weld&amp;author=A+Mendoza-Mendoza&amp;author=A+Stewart&amp;volume=156&amp;issue=10&amp;publication_year=2010&amp;pages=2887-2900&amp;pmid=20688823&amp;doi=10.1099/mic.0.041715-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR7">7. <span class="mixed-citation">Medina-Castellanos E, Esquivel-Naranjo EU, Heil M, Herrera-Estrella A (2014) Extracellular ATP activates MAPK and ROS signaling during injury response in the fungus <em>Trichoderma atroviride</em>. Front Plant Sci 5. 10.3389/fpls.2014.00659 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4240048/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25484887" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="CR8">8. <span class="element-citation">Esquivel-Naranjo EU, Garc&#x000ed;a-Esquivel M, Medina-Castellanos E, Correa-P&#x000e9;rez VA, Parra-Arriaga JL, Landeros-Jaime F, Cervantes-Ch&#x000e1;vez JA, Herrera-Estrella A. A <em>Trichoderma atroviride</em> stress-activated MAPK pathway integrates stress and light signals. <span><span class="ref-journal">Mol Microbiol. </span>2016;<span class="ref-vol">100</span>(5):860–876. doi: 10.1111/mmi.13355.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26878111" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fmmi.13355" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Microbiol&amp;title=A+Trichoderma+atroviride+stress-activated+MAPK+pathway+integrates+stress+and+light+signals&amp;author=EU+Esquivel-Naranjo&amp;author=M+Garc&#x000ed;a-Esquivel&amp;author=E+Medina-Castellanos&amp;author=VA+Correa-P&#x000e9;rez&amp;author=JL+Parra-Arriaga&amp;volume=100&amp;issue=5&amp;publication_year=2016&amp;pages=860-876&amp;pmid=26878111&amp;doi=10.1111/mmi.13355&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR9">9. <span class="element-citation">Pronk JT. Auxotrophic yeast strains in fundamental and applied research. <span><span class="ref-journal">Appl Environ Microbiol. </span>2002;<span class="ref-vol">68</span>(5):2095–2100. doi: 10.1128/AEM.68.5.2095-2100.2002.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC127579/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11976076" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FAEM.68.5.2095-2100.2002" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Appl+Environ+Microbiol&amp;title=Auxotrophic+yeast+strains+in+fundamental+and+applied+research&amp;author=JT+Pronk&amp;volume=68&amp;issue=5&amp;publication_year=2002&amp;pages=2095-2100&amp;pmid=11976076&amp;doi=10.1128/AEM.68.5.2095-2100.2002&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR10">10. <span class="element-citation">Narasipura SD, Ren P, Dyavaiah M, Auger I, Chaturvedi V, Chaturvedi S. An efficient method for homologous gene reconstitution in <em>Cryptococcus gattii</em> using URA5 auxotrophic marker. <span><span class="ref-journal">Mycopathologia. </span>2006;<span class="ref-vol">162</span>(6):401–409. doi: 10.1007/s11046-006-0076-z.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17146584" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs11046-006-0076-z" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mycopathologia&amp;title=An+efficient+method+for+homologous+gene+reconstitution+in+Cryptococcus+gattii+using+URA5+auxotrophic+marker&amp;author=SD+Narasipura&amp;author=P+Ren&amp;author=M+Dyavaiah&amp;author=I+Auger&amp;author=V+Chaturvedi&amp;volume=162&amp;issue=6&amp;publication_year=2006&amp;pages=401-409&amp;pmid=17146584&amp;doi=10.1007/s11046-006-0076-z&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR11">11. <span class="element-citation">Baek JM, Kenerley CM. The <em>arg</em>2 gene of <em>Trichoderma virens</em>: cloning and development of a homologous transformation system. <span><span class="ref-journal">Fungal Genet Biol. </span>1998;<span class="ref-vol">23</span>(1):34–44. doi: 10.1006/fgbi.1997.1025.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9501476" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1006%2Ffgbi.1997.1025" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Fungal+Genet+Biol&amp;title=The+arg2+gene+of+Trichoderma+virens:+cloning+and+development+of+a+homologous+transformation+system&amp;author=JM+Baek&amp;author=CM+Kenerley&amp;volume=23&amp;issue=1&amp;publication_year=1998&amp;pages=34-44&amp;pmid=9501476&amp;doi=10.1006/fgbi.1997.1025&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR12">12. <span class="element-citation">Penttil&#x000e4; M, Nevalainen H, R&#x000e4;tt&#x000f6; M, Salminen E, Knowles J. A versatile transformation system for the cellulolytic filamentous fungus <em>Trichoderma reesei</em>. <span><span class="ref-journal">Gene. </span>1987;<span class="ref-vol">61</span>(2):155–164. doi: 10.1016/0378-1119(87)90110-7.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3127274" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2F0378-1119(87)90110-7" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=A+versatile+transformation+system+for+the+cellulolytic+filamentous+fungus+Trichoderma+reesei&amp;author=M+Penttil&#x000e4;&amp;author=H+Nevalainen&amp;author=M+R&#x000e4;tt&#x000f6;&amp;author=E+Salminen&amp;author=J+Knowles&amp;volume=61&amp;issue=2&amp;publication_year=1987&amp;pages=155-164&amp;pmid=3127274&amp;doi=10.1016/0378-1119(87)90110-7&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR13">13. <span class="element-citation">J&#x000f8;rgensen M, Skovlund D, Johannesen P, Mortensen UH. A novel platform for heterologous gene expression in <em>Trichoderma reesei</em> (Teleomorph <em>Hypocrea jecorina</em>) <span><span class="ref-journal">Microb Cell Factories. </span>2014;<span class="ref-vol">13</span>(1):33. doi: 10.1186/1475-2859-13-33.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4015775/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24602241" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2F1475-2859-13-33" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microb+Cell+Factories&amp;title=A+novel+platform+for+heterologous+gene+expression+in+Trichoderma+reesei+(Teleomorph+Hypocrea+jecorina)&amp;author=M+J&#x000f8;rgensen&amp;author=D+Skovlund&amp;author=P+Johannesen&amp;author=UH+Mortensen&amp;volume=13&amp;issue=1&amp;publication_year=2014&amp;pages=33&amp;doi=10.1186/1475-2859-13-33&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR14">14. <span class="element-citation">Gruber F, Visser J, Kubicek CP, de Graaff LH. The development of a heterologous transformation system for the cellulolytic fungus <em>Trichoderma reesei</em> based on a <em>pyr</em>G-negative mutant strain. <span><span class="ref-journal">Curr Genet. </span>1990;<span class="ref-vol">18</span>(1):71–76. doi: 10.1007/BF00321118.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2245476" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2FBF00321118" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Genet&amp;title=The+development+of+a+heterologous+transformation+system+for+the+cellulolytic+fungus+Trichoderma+reesei+based+on+a+pyrG-negative+mutant+strain&amp;author=F+Gruber&amp;author=J+Visser&amp;author=CP+Kubicek&amp;author=LH+de+Graaff&amp;volume=18&amp;issue=1&amp;publication_year=1990&amp;pages=71-76&amp;pmid=2245476&amp;doi=10.1007/BF00321118&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR15">15. <span class="element-citation">Liu H, Wang G, Li W, Liu X, Li E, Yin WB. A highly efficient genetic system for the identification of a <em>harzianum</em> B biosynthetic gene cluster in <em>Trichoderma hypoxylon</em>. <span><span class="ref-journal">Microbiology. </span>2018;<span class="ref-vol">164</span>(5):769–778. doi: 10.1099/mic.0.000649.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29557773" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1099%2Fmic.0.000649" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology&amp;title=A+highly+efficient+genetic+system+for+the+identification+of+a+harzianum+B+biosynthetic+gene+cluster+in+Trichoderma+hypoxylon&amp;author=H+Liu&amp;author=G+Wang&amp;author=W+Li&amp;author=X+Liu&amp;author=E+Li&amp;volume=164&amp;issue=5&amp;publication_year=2018&amp;pages=769-778&amp;pmid=29557773&amp;doi=10.1099/mic.0.000649&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR16">16. <span class="element-citation">Schuster A, Bruno KS, Collett JR, Baker SE, Seiboth B, Kubicek CP, Schmoll M. A versatile toolkit for high throughput functional genomics with <em>Trichoderma reesei</em>. <span><span class="ref-journal">Biotechnol Biofuels. </span>2012;<span class="ref-vol">5</span>(1):1. doi: 10.1186/1754-6834-5-1.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3260098/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22212435" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2F1754-6834-5-1" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnol+Biofuels&amp;title=A+versatile+toolkit+for+high+throughput+functional+genomics+with+Trichoderma+reesei&amp;author=A+Schuster&amp;author=KS+Bruno&amp;author=JR+Collett&amp;author=SE+Baker&amp;author=B+Seiboth&amp;volume=5&amp;issue=1&amp;publication_year=2012&amp;pages=1&amp;pmid=22212435&amp;doi=10.1186/1754-6834-5-1&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR17">17. <span class="mixed-citation">Atomi H, Imanaka T, Fukui T (2012) Overview of the genetic tools in the Archaea. Front Microbiol 3. 10.3389/fmicb.2012.00337 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3462420/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23060865" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="CR18">18. <span class="element-citation">Matuo R, Sousa FG, Escargueil AE, Soares DG, Grivicich I, Saffi J, Larsen AK, Henriques JAP. DNA repair pathways involved in repair of lesions induced by 5-fluorouracil and its active metabolite FdUMP. <span><span class="ref-journal">Biochem Pharmacol. </span>2010;<span class="ref-vol">79</span>(2):147–153. doi: 10.1016/j.bcp.2009.08.016.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19712668" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.bcp.2009.08.016" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+Pharmacol&amp;title=DNA+repair+pathways+involved+in+repair+of+lesions+induced+by+5-fluorouracil+and+its+active+metabolite+FdUMP&amp;author=R+Matuo&amp;author=FG+Sousa&amp;author=AE+Escargueil&amp;author=DG+Soares&amp;author=I+Grivicich&amp;volume=79&amp;issue=2&amp;publication_year=2010&amp;pages=147-153&amp;pmid=19712668&amp;doi=10.1016/j.bcp.2009.08.016&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR19">19. <span class="element-citation">Wyatt MD, Wilson DM. Participation of DNA repair in the response to 5-fluorouracil. <span><span class="ref-journal">Cell Mol Life Sci. </span>2009;<span class="ref-vol">66</span>(5):788–799. doi: 10.1007/s00018-008-8557-5.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2649968/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18979208" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs00018-008-8557-5" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Mol+Life+Sci&amp;title=Participation+of+DNA+repair+in+the+response+to+5-fluorouracil&amp;author=MD+Wyatt&amp;author=DM+Wilson&amp;volume=66&amp;issue=5&amp;publication_year=2009&amp;pages=788-799&amp;pmid=18979208&amp;doi=10.1007/s00018-008-8557-5&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR20">20. <span class="element-citation">Boeke JD, La Croute F, Fink GR. A positive selection for mutants lacking orotidine-5&#x02032;-phosphate decarboxylase activity in yeast: 5-fluoro-orotic acid resistance. <span><span class="ref-journal">Mol Gen Genet. </span>1984;<span class="ref-vol">197</span>(2):345–346. doi: 10.1007/bf00330984.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/6394957" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fbf00330984" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Gen+Genet&amp;title=A+positive+selection+for+mutants+lacking+orotidine-5&#x02032;-phosphate+decarboxylase+activity+in+yeast:+5-fluoro-orotic+acid+resistance&amp;author=JD+Boeke&amp;author=F+La+Croute&amp;author=GR+Fink&amp;volume=197&amp;issue=2&amp;publication_year=1984&amp;pages=345-346&amp;pmid=6394957&amp;doi=10.1007/bf00330984&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR21">21. <span class="element-citation">Razanamparany V, B&#x000e9;gueret J. Positive screening and transformation of <em>ura</em>5 mutants in the fungus <em>Podospora anserina</em>: characterization of the transformants. <span><span class="ref-journal">Curr Genet. </span>1986;<span class="ref-vol">10</span>(11):811–817. doi: 10.1007/BF00418527.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3447739" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2FBF00418527" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Genet&amp;title=Positive+screening+and+transformation+of+ura5+mutants+in+the+fungus+Podospora+anserina:+characterization+of+the+transformants&amp;author=V+Razanamparany&amp;author=J+B&#x000e9;gueret&amp;volume=10&amp;issue=11&amp;publication_year=1986&amp;pages=811-817&amp;pmid=3447739&amp;doi=10.1007/BF00418527&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR22">22. <span class="element-citation">Heidenreich EJ, Kubicek CP. Sequence of the <em>pyr</em>4 gene encoding orotidine-5&#x02032;-phosphate decarboxylase from the biocontrol fungus <em>Trichoderma harzianum</em>. <span><span class="ref-journal">Gene. </span>1994;<span class="ref-vol">147</span>(1):151–152. doi: 10.1016/0378-1119(94)90057-4.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8088543" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2F0378-1119(94)90057-4" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=Sequence+of+the+pyr4+gene+encoding+orotidine-5&#x02032;-phosphate+decarboxylase+from+the+biocontrol+fungus+Trichoderma+harzianum&amp;author=EJ+Heidenreich&amp;author=CP+Kubicek&amp;volume=147&amp;issue=1&amp;publication_year=1994&amp;pages=151-152&amp;pmid=8088543&amp;doi=10.1016/0378-1119(94)90057-4&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR23">23. <span class="element-citation">Yu JH, Hamari Z, Han KH, Seo JA, Reyes-Dom&#x000ed;nguez Y, Scazzocchio C. Double-joint PCR: a PCR-based molecular tool for gene manipulations in filamentous fungi. <span><span class="ref-journal">Fungal Genet Biol. </span>2004;<span class="ref-vol">41</span>(11):973–981. doi: 10.1016/j.fgb.2004.08.001.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15465386" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.fgb.2004.08.001" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Fungal+Genet+Biol&amp;title=Double-joint+PCR:+a+PCR-based+molecular+tool+for+gene+manipulations+in+filamentous+fungi&amp;author=JH+Yu&amp;author=Z+Hamari&amp;author=KH+Han&amp;author=JA+Seo&amp;author=Y+Reyes-Dom&#x000ed;nguez&amp;volume=41&amp;issue=11&amp;publication_year=2004&amp;pages=973-981&amp;pmid=15465386&amp;doi=10.1016/j.fgb.2004.08.001&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR24">24. <span class="element-citation">Castellanos F, Schmoll M, Mart&#x000ed;nez P, Tisch D, Kubicek CP, Herrera-Estrella A, Esquivel-Naranjo EU. Crucial factors of the light perception machinery and their impact on growth and cellulase gene transcription in <em>Trichoderma reesei</em>. <span><span class="ref-journal">Fungal Genet Biol. </span>2010;<span class="ref-vol">47</span>(5):468–476. doi: 10.1016/j.fgb.2010.02.001.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20144726" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.fgb.2010.02.001" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Fungal+Genet+Biol&amp;title=Crucial+factors+of+the+light+perception+machinery+and+their+impact+on+growth+and+cellulase+gene+transcription+in+Trichoderma+reesei&amp;author=F+Castellanos&amp;author=M+Schmoll&amp;author=P+Mart&#x000ed;nez&amp;author=D+Tisch&amp;author=CP+Kubicek&amp;volume=47&amp;issue=5&amp;publication_year=2010&amp;pages=468-476&amp;pmid=20144726&amp;doi=10.1016/j.fgb.2010.02.001&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR25">25. <span class="element-citation">Raeder U, Broda P. Rapid preparation of DNA from filamentous fungi. <span><span class="ref-journal">Lett Appl Microbiol. </span>1985;<span class="ref-vol">1</span>(1):17–20. doi: 10.1111/j.1472-765X.1985.tb01479.x.</span> [<a href="//doi.org/10.1111%2Fj.1472-765X.1985.tb01479.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lett+Appl+Microbiol&amp;title=Rapid+preparation+of+DNA+from+filamentous+fungi&amp;author=U+Raeder&amp;author=P+Broda&amp;volume=1&amp;issue=1&amp;publication_year=1985&amp;pages=17-20&amp;doi=10.1111/j.1472-765X.1985.tb01479.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR26">26. <span class="element-citation">Derntl C, Kiesenhofer DP, Mach RL, Mach-Aigner AR. Novel strategies for genomic manipulation of <em>Trichoderma reesei</em> with the purpose of strain engineering. <span><span class="ref-journal">Appl Environ Microbiol. </span>2015;<span class="ref-vol">81</span>(18):6314–6323. doi: 10.1128/AEM.01545-15.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4542242/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26150462" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FAEM.01545-15" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Appl+Environ+Microbiol&amp;title=Novel+strategies+for+genomic+manipulation+of+Trichoderma+reesei+with+the+purpose+of+strain+engineering&amp;author=C+Derntl&amp;author=DP+Kiesenhofer&amp;author=RL+Mach&amp;author=AR+Mach-Aigner&amp;volume=81&amp;issue=18&amp;publication_year=2015&amp;pages=6314-6323&amp;pmid=26150462&amp;doi=10.1128/AEM.01545-15&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR27">27. <span class="element-citation">Smith JL, Bayliss FT, Ward M. Sequence of the cloned <em>pyr</em>4 gene of <em>Trichoderma reesei</em> and its use as a homologous selectable marker for transformation. <span><span class="ref-journal">Curr Genet. </span>1991;<span class="ref-vol">19</span>(1):27–33. doi: 10.1007/bf00362084.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2036683" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fbf00362084" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Genet&amp;title=Sequence+of+the+cloned+pyr4+gene+of+Trichoderma+reesei+and+its+use+as+a+homologous+selectable+marker+for+transformation&amp;author=JL+Smith&amp;author=FT+Bayliss&amp;author=M+Ward&amp;volume=19&amp;issue=1&amp;publication_year=1991&amp;pages=27-33&amp;pmid=2036683&amp;doi=10.1007/bf00362084&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR28">28. <span class="element-citation">Ballance DJ, Buxton FP, Turner G. Transformation of <em>Aspergillus nidulans</em> by the orotidine-5&#x02032;-phosphate decarboxylase gene of <em>Neurospora crassa</em>. <span><span class="ref-journal">Biochem Biophys Res Commun. </span>1983;<span class="ref-vol">112</span>(1):284–289. doi: 10.1016/0006-291x(83)91828-4.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/6220717" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2F0006-291x(83)91828-4" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+Biophys+Res+Commun&amp;title=Transformation+of+Aspergillus+nidulans+by+the+orotidine-5&#x02032;-phosphate+decarboxylase+gene+of+Neurospora+crassa&amp;author=DJ+Ballance&amp;author=FP+Buxton&amp;author=G+Turner&amp;volume=112&amp;issue=1&amp;publication_year=1983&amp;pages=284-289&amp;pmid=6220717&amp;doi=10.1016/0006-291x(83)91828-4&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR29">29. <span class="element-citation">van Hartingsveldt W, Mattern IE, van Zeijl CM, Pouwels PH, van den Hondel CA. Development of a homologous transformation system for <em>Aspergillus niger</em> based on the <em>pyr</em>G gene. <span><span class="ref-journal">Mol Gen Genet. </span>1987;<span class="ref-vol">206</span>(1):71–75. doi: 10.1007/bf00326538.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3472035" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fbf00326538" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Gen+Genet&amp;title=Development+of+a+homologous+transformation+system+for+Aspergillus+niger+based+on+the+pyrG+gene&amp;author=W+van+Hartingsveldt&amp;author=IE+Mattern&amp;author=CM+van+Zeijl&amp;author=PH+Pouwels&amp;author=CA+van+den+Hondel&amp;volume=206&amp;issue=1&amp;publication_year=1987&amp;pages=71-75&amp;pmid=3472035&amp;doi=10.1007/bf00326538&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR30">30. <span class="element-citation">Berg&#x000e8;s T, Barreau C. Isolation of uridine auxotrophs from <em>Trichoderma reesei</em> and efficient transformation with the cloned <em>ura</em>3 and <em>ura</em>5 genes. <span><span class="ref-journal">Curr Genet. </span>1991;<span class="ref-vol">19</span>(5):359–365. doi: 10.1007/bf00309596.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/1913875" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fbf00309596" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Genet&amp;title=Isolation+of+uridine+auxotrophs+from+Trichoderma+reesei+and+efficient+transformation+with+the+cloned+ura3+and+ura5+genes&amp;author=T+Berg&#x000e8;s&amp;author=C+Barreau&amp;volume=19&amp;issue=5&amp;publication_year=1991&amp;pages=359-365&amp;pmid=1913875&amp;doi=10.1007/bf00309596&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR31">31. <span class="element-citation">Casas-Flores S, Rios-Momberg M, Bibbins M, Ponce-Noyola P, Herrera-Estrella A. BLR-1 and BLR-2, key regulatory elements of photoconidiation and mycelial growth in <em>Trichoderma atroviride</em>. <span><span class="ref-journal">Microbiology. </span>2004;<span class="ref-vol">150</span>(11):3561–3569. doi: 10.1099/mic.0.27346-0.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15528646" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1099%2Fmic.0.27346-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology&amp;title=BLR-1+and+BLR-2,+key+regulatory+elements+of+photoconidiation+and+mycelial+growth+in+Trichoderma+atroviride&amp;author=S+Casas-Flores&amp;author=M+Rios-Momberg&amp;author=M+Bibbins&amp;author=P+Ponce-Noyola&amp;author=A+Herrera-Estrella&amp;volume=150&amp;issue=11&amp;publication_year=2004&amp;pages=3561-3569&amp;pmid=15528646&amp;doi=10.1099/mic.0.27346-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR32">32. <span class="element-citation">Alam MT, Zelezniak A, M&#x000fc;lleder M, Shliaha P, Schwarz R, Capuano F, Vowinckel J, Radmaneshfar E, Kr&#x000fc;ger A, Calvani E, Michel S, B&#x000f6;rno S, Christen S, Patil KR, Timmermann B, Lilley KS, Ralser M. The metabolic background is a global player in <em>Saccharomyces</em> gene expression epistasis. <span><span class="ref-journal">Nat Microbiol. </span>2016;<span class="ref-vol">1</span>(3):15030. doi: 10.1038/nmicrobiol.2015.30.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5131842/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27572163" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnmicrobiol.2015.30" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Microbiol&amp;title=The+metabolic+background+is+a+global+player+in+Saccharomyces+gene+expression+epistasis&amp;author=MT+Alam&amp;author=A+Zelezniak&amp;author=M+M&#x000fc;lleder&amp;author=P+Shliaha&amp;author=R+Schwarz&amp;volume=1&amp;issue=3&amp;publication_year=2016&amp;pages=15030&amp;pmid=27572163&amp;doi=10.1038/nmicrobiol.2015.30&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR33">33. <span class="element-citation">Rosales-Saavedra T, Esquivel-Naranjo EU, Casas-Flores S, Martinez-Hernandez P, Ibarra-Laclette E, et al. Novel light-regulated genes in <em>Trichoderma atroviride</em>: a dissection by cDNA microarrays. <span><span class="ref-journal">Microbiology. </span>2006;<span class="ref-vol">152</span>(11):3305–3317. doi: 10.1099/mic.0.29000-0.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17074901" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1099%2Fmic.0.29000-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology&amp;title=Novel+light-regulated+genes+in+Trichoderma+atroviride:+a+dissection+by+cDNA+microarrays&amp;author=T+Rosales-Saavedra&amp;author=EU+Esquivel-Naranjo&amp;author=S+Casas-Flores&amp;author=P+Martinez-Hernandez&amp;author=E+Ibarra-Laclette&amp;volume=152&amp;issue=11&amp;publication_year=2006&amp;pages=3305-3317&amp;pmid=17074901&amp;doi=10.1099/mic.0.29000-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR34">34. <span class="element-citation">Seiple L, Jaruga P, Dizdaroglu M, Stivers JT. Linking uracil base excision repair and 5-fluorouracil toxicity in yeast. <span><span class="ref-journal">Nucleic Acids Res. </span>2006;<span class="ref-vol">34</span>(1):140–151. doi: 10.1093/nar/gkj430.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1326247/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16407331" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkj430" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Linking+uracil+base+excision+repair+and+5-fluorouracil+toxicity+in+yeast&amp;author=L+Seiple&amp;author=P+Jaruga&amp;author=M+Dizdaroglu&amp;author=JT+Stivers&amp;volume=34&amp;issue=1&amp;publication_year=2006&amp;pages=140-151&amp;pmid=16407331&amp;doi=10.1093/nar/gkj430&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR35">35. <span class="element-citation">Wellington M, Rustchenko E. 5-Fluoro-orotic acid induces chromosome alterations in <em>Candida albicans</em>. <span><span class="ref-journal">Yeast. </span>2005;<span class="ref-vol">22</span>(1):57–70. doi: 10.1002/yea.1191.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15635674" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fyea.1191" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Yeast&amp;title=5-Fluoro-orotic+acid+induces+chromosome+alterations+in+Candida+albicans&amp;author=M+Wellington&amp;author=E+Rustchenko&amp;volume=22&amp;issue=1&amp;publication_year=2005&amp;pages=57-70&amp;pmid=15635674&amp;doi=10.1002/yea.1191&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR36">36. <span class="element-citation">Wellington M, Kabir MA, Rustchenko E. 5-fluoro-orotic acid induces chromosome alterations in genetically manipulated strains of <em>Candida albicans</em>. <span><span class="ref-journal">Mycologia. </span>2006;<span class="ref-vol">98</span>(3):393–398. doi: 10.3852/mycologia.98.3.393.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17040068" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3852%2Fmycologia.98.3.393" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mycologia&amp;title=5-fluoro-orotic+acid+induces+chromosome+alterations+in+genetically+manipulated+strains+of+Candida+albicans&amp;author=M+Wellington&amp;author=MA+Kabir&amp;author=E+Rustchenko&amp;volume=98&amp;issue=3&amp;publication_year=2006&amp;pages=393-398&amp;pmid=17040068&amp;doi=10.3852/mycologia.98.3.393&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR37">37. <span class="element-citation">Esquivel-Naranjo EU, Herrera-Estrella A. Enhanced responsiveness and sensitivity to blue light by <em>blr</em>-2 overexpression in <em>Trichoderma atroviride</em>. <span><span class="ref-journal">Microbiology. </span>2007;<span class="ref-vol">153</span>(11):3909–3902. doi: 10.1099/mic.0.2007/007302-0.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17975098" ref="reftype=pubmed&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1099%2Fmic.0.2007%2F007302-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology&amp;title=Enhanced+responsiveness+and+sensitivity+to+blue+light+by+blr-2+overexpression+in+Trichoderma+atroviride&amp;author=EU+Esquivel-Naranjo&amp;author=A+Herrera-Estrella&amp;volume=153&amp;issue=11&amp;publication_year=2007&amp;pages=3909-3902&amp;pmid=17975098&amp;doi=10.1099/mic.0.2007/007302-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7688867&amp;issue-id=369536&amp;journal-id=2094&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Brazilian Journal of Microbiology</span> are provided here courtesy of <strong>Brazilian Society of Microbiology</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC7688867/pdf/42770_2020_Article_329.pdf" class="int-view">PDF (6.3M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/7688867/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/7688867/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="7688867" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC7688867%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="Tn5rmOTKQ4jtCejjKwthtzSBFg04dODOVEfsyZ8tBQRZ6rfEdkgotJBhXBu5qH6c">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC7688867%2F&amp;text=An%20efficient%20transformation%20system%20for%20Trichoderma%20atroviride%20using%20the%20pyr4%20gene%20as%20a%20selectable%20marker" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC7688867%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7688867/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC7688867/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/32627116/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC7688867/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/32627116/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC7688867/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/7688867/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC7688867/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
