
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Discovery of potent and specific inhibitors targeting the active site of MMP-9 from the engineered SPINK2 library - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="939BE807DB0F4BC500003D7D054BBCF8.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="plosone" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC7771667/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7771667/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="PLoS ONE" /><meta name="citation_title" content="Discovery of potent and specific inhibitors targeting the active site of MMP-9 from the engineered SPINK2 library" /><meta name="citation_author" content="Hidenori Yano" /><meta name="citation_author_institution" content="Modality Research Laboratories, Biologics Division, Daiichi Sankyo Co., Ltd., Tokyo, Japan" /><meta name="citation_author_institution" content="Graduate School of Life and Environmental Sciences, University of Tsukuba, Ibaraki, Japan" /><meta name="citation_author" content="Daisuke Nishimiya" /><meta name="citation_author_institution" content="Modality Research Laboratories, Biologics Division, Daiichi Sankyo Co., Ltd., Tokyo, Japan" /><meta name="citation_author" content="Yoshirou Kawaguchi" /><meta name="citation_author_institution" content="Modality Research Laboratories, Biologics Division, Daiichi Sankyo Co., Ltd., Tokyo, Japan" /><meta name="citation_author" content="Masakazu Tamura" /><meta name="citation_author_institution" content="Modality Research Laboratories, Biologics Division, Daiichi Sankyo Co., Ltd., Tokyo, Japan" /><meta name="citation_author" content="Ryuji Hashimoto" /><meta name="citation_author_institution" content="Modality Research Laboratories, Biologics Division, Daiichi Sankyo Co., Ltd., Tokyo, Japan" /><meta name="citation_publication_date" content="2020" /><meta name="citation_issue" content="12" /><meta name="citation_volume" content="15" /><meta name="citation_doi" content="10.1371/journal.pone.0244656" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC7771667/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC7771667/" /><meta name="citation_pmid" content="33373399" /><meta name="DC.Title" content="Discovery of potent and specific inhibitors targeting the active site of MMP-9 from the engineered SPINK2 library" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="PLOS" /><meta name="DC.Contributor" content="Hidenori Yano" /><meta name="DC.Contributor" content="Daisuke Nishimiya" /><meta name="DC.Contributor" content="Yoshirou Kawaguchi" /><meta name="DC.Contributor" content="Masakazu Tamura" /><meta name="DC.Contributor" content="Ryuji Hashimoto" /><meta name="DC.Date" content="2020" /><meta name="DC.Identifier" content="10.1371/journal.pone.0244656" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Discovery of potent and specific inhibitors targeting the active site of MMP-9 from the engineered SPINK2 library" /><meta property="og:type" content="article" /><meta property="og:description" content="Matrix metalloproteinases (MMPs) contribute to many physiological and pathological phenomena via the proteolysis of extracellular matrix components. Specific blocking of the active site of each MMP sheds light on its particular role. However, it remains ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7771667/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC7771667/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22PLoS%20One%22[journal]" class="navlink">PLoS One</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC7771667
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC7771667/pdf/pone.0244656.pdf" class="int-view">PDF (2.2M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/7771667/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/7771667/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="7771667" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC7771667%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="9P6OqpmIJ7K1nwH2yokOWt9nndxiGcCAntLpGV4i1zZ2JgiOIq9h3j7YZ2UU6CHA">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC7771667%2F&amp;text=Discovery%20of%20potent%20and%20specific%20inhibitors%20targeting%20the%20active%20site%20of%20MMP-9%20from%20the%20engineered%20SPINK2%20library%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC7771667%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7771667/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC7771667/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/33373399/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC7771667/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/33373399/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC7771667/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/7771667/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22PLoS%20One%22[journal]" class="navlink">PLoS One</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC7771667
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-plosone.png" alt="Logo of plosone" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="https://journals.plos.org/plosone/" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">PLoS One.</span> 2020; 15(12): e0244656. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2020 Dec 29. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1371%2Fjournal.pone.0244656" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1371/journal.pone.0244656</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC7771667</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/33373399">33373399</a></div></div></div><h1 class="content-title">Discovery of potent and specific inhibitors targeting the active site of MMP-9 from the engineered SPINK2 library</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Yano%20H%5BAuthor%5D" class="affpopup" co-rid="_co_idm140659064752320" co-class="co-affbox">Hidenori Yano</a>, <span class="fm-role">Conceptualization</span>, <span class="fm-role">Data curation</span>, <span class="fm-role">Formal analysis</span>, <span class="fm-role">Investigation</span>, <span class="fm-role">Methodology</span>, <span class="fm-role">Visualization</span>, <span class="fm-role">Writing &#x02013; original draft</span>, <span class="fm-role">Writing &#x02013; review &#x00026; editing</span>,<sup>1,</sup><sup>2,</sup><sup>*</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Nishimiya%20D%5BAuthor%5D" class="affpopup" co-rid="_co_idm140659061964288" co-class="co-affbox">Daisuke Nishimiya</a>, <span class="fm-role">Conceptualization</span>, <span class="fm-role">Data curation</span>, <span class="fm-role">Formal analysis</span>, <span class="fm-role">Investigation</span>, <span class="fm-role">Methodology</span>, <span class="fm-role">Writing &#x02013; original draft</span>, <span class="fm-role">Writing &#x02013; review &#x00026; editing</span>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kawaguchi%20Y%5BAuthor%5D" class="affpopup" co-rid="_co_idm140659104803536" co-class="co-affbox">Yoshirou Kawaguchi</a>, <span class="fm-role">Formal analysis</span>, <span class="fm-role">Investigation</span>, <span class="fm-role">Visualization</span>, <span class="fm-role">Writing &#x02013; original draft</span>, <span class="fm-role">Writing &#x02013; review &#x00026; editing</span>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Tamura%20M%5BAuthor%5D" class="affpopup" co-rid="_co_idm140659104799296" co-class="co-affbox">Masakazu Tamura</a>, <span class="fm-role">Writing &#x02013; original draft</span>, <span class="fm-role">Writing &#x02013; review &#x00026; editing</span>,<sup>1</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hashimoto%20R%5BAuthor%5D" class="affpopup" co-rid="_co_idm140659101572736" co-class="co-affbox">Ryuji Hashimoto</a>, <span class="fm-role">Conceptualization</span>, <span class="fm-role">Methodology</span>, <span class="fm-role">Project administration</span>, <span class="fm-role">Supervision</span>, <span class="fm-role">Writing &#x02013; original draft</span>, <span class="fm-role">Writing &#x02013; review &#x00026; editing</span><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140659064752320"><h3 class="no_margin">Hidenori Yano</h3><p><sup>1</sup>
Modality Research Laboratories, Biologics Division, Daiichi Sankyo Co., Ltd., Tokyo, Japan</p><p><sup>2</sup>
Graduate School of Life and Environmental Sciences, University of Tsukuba, Ibaraki, Japan</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Yano%20H%5BAuthor%5D">Hidenori Yano</a></div></div><div id="_co_idm140659061964288"><h3 class="no_margin">Daisuke Nishimiya</h3><p><sup>1</sup>
Modality Research Laboratories, Biologics Division, Daiichi Sankyo Co., Ltd., Tokyo, Japan</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Nishimiya%20D%5BAuthor%5D">Daisuke Nishimiya</a></div></div><div id="_co_idm140659104803536"><h3 class="no_margin">Yoshirou Kawaguchi</h3><p><sup>1</sup>
Modality Research Laboratories, Biologics Division, Daiichi Sankyo Co., Ltd., Tokyo, Japan</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kawaguchi%20Y%5BAuthor%5D">Yoshirou Kawaguchi</a></div></div><div id="_co_idm140659104799296"><h3 class="no_margin">Masakazu Tamura</h3><p><sup>1</sup>
Modality Research Laboratories, Biologics Division, Daiichi Sankyo Co., Ltd., Tokyo, Japan</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Tamura%20M%5BAuthor%5D">Masakazu Tamura</a></div></div><div id="_co_idm140659101572736"><h3 class="no_margin">Ryuji Hashimoto</h3><p><sup>1</sup>
Modality Research Laboratories, Biologics Division, Daiichi Sankyo Co., Ltd., Tokyo, Japan</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hashimoto%20R%5BAuthor%5D">Ryuji Hashimoto</a></div></div></div></div><div class="contrib-group half_rhythm fm-editor">Yu-Hsuan Tsai, <span class="fm-role">Editor</span><sup></sup></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140659060442928_ai idm140659064752448_ai idm140659065191584_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140659060442928_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140659060442928_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140659060442928_ai" style="display:none"><div class="fm-affl" id="aff001"><sup>1</sup>
Modality Research Laboratories, Biologics Division, Daiichi Sankyo Co., Ltd., Tokyo, Japan</div><div class="fm-affl" id="aff002"><sup>2</sup>
Graduate School of Life and Environmental Sciences, University of Tsukuba, Ibaraki, Japan</div><div class="fm-affl" id="edit1">Cardiff University, UNITED KINGDOM</div><div id="coi001"><strong>Competing Interests: </strong>I have read the journal&#x02019;s policy and the authors of this manuscript have the following competing interests: All authors are employees of Daiichi Sankyo Co., Ltd. H.Y., D.N., and R.H. are the inventors of the patent application relating to the inhibitors described in this paper, and all rights of the patent have been assigned to Daiichi Sankyo Co., Ltd. This does not alter our adherence to PLOS ONE policies on sharing data and materials.</div><div id="cor001">* E-mail: <a href="mailto:dev@null" data-email="pj.oc.oyknasihciiad@bk.ironedih.onay" class="oemail">pj.oc.oyknasihciiad@bk.ironedih.onay</a></div></div><div class="fm-article-notes hide half_rhythm" id="idm140659060442928_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2020 Sep 29; Accepted 2020 Dec 14.</div></div><div class="permissions half_rhythm hide" id="idm140659060442928_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; 2020 Yano et al</div><div class="license half_rhythm">This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="15" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S1 Fig: </strong>SDS-PAGE analysis of purified MMP-9. (A) Pro-MMP-9_Cat-H6 was purified and activated by active MMP-3, as described in &#x0201c;Materials and methods.&#x0201d; (B) Pro-MMP-9_Cat_E402Q-H6 was purified and processed by active MMP-3, as described in &#x0201c;Materials and methods.&#x0201d; SDS-PAGE analysis of the purified pro-MMP-9 (1 &#x003bc;g per gel lane), active MMP-9_Cat-H6, and MMP-9_Cat_E402Q-H6 (0.5 &#x003bc;g per gel lane) was performed under reducing conditions followed by Coomassie Brilliant Blue G-250 staining.<p>(TIF)</p></div><div><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s001.tif" data-ga-action="click_feat_suppl">pone.0244656.s001.tif</a><span style="color:gray"> (359K)</span></div><div class="small guid">GUID: 3D4226E8-6D42-4D28-A3EE-5091BE18C62D</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S2 Fig: </strong>Scheme of a randomized region of the engineered SPINK2. (A) Amino acid sequence of wild-type SPINK2. Lines indicate disulfide bonds (Cys-14&#x02013;Cys-44, Cys-22&#x02013;Cys-41, Cys-30&#x02013;Cys-62). (B) Region randomized to create the engineered SPINK2 library. (C) Three-dimensional structure of wild-type SPINK2 (PDB code, 2JXD). SPINK2 is shown as a gray and red semi-transparent surface model; red indicates the randomized region. The right figure represents the left image turned 90&#x000b0; counterclockwise about the y-axis. (D) Aligned sequences of the engineered SPINK2-derived inhibitors against MMP-9.<p>(TIF)</p></div><div><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s002.tif" data-ga-action="click_feat_suppl">pone.0244656.s002.tif</a><span style="color:gray"> (1.0M)</span></div><div class="small guid">GUID: F3CA5D2C-5B53-455C-BE61-B18347F4F3AC</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S3 Fig: </strong>The binding properties of inhibitors were evaluated by ELISA. Various concentrations of pro-MMP-9_Cat-Avi or active MMP-9_Cat-Avi (0.8&#x02013;100 nM) were added to <a class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> or GS-5745 (10 &#x003bc;g/ml each)-coated plates, and then C-terminal FLAG tag of captured MMP-9 was detected by HRP-conjugated anti-FLAG tag antibody. All curves were obtained by non-linear curve fitting.<p>(TIF)</p></div><div><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s003.tif" data-ga-action="click_feat_suppl">pone.0244656.s003.tif</a><span style="color:gray"> (442K)</span></div><div class="small guid">GUID: F803CE4C-0CFC-4FD8-BC7F-FE57F6F6BE81</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S4 Fig: </strong>Sequence alignment of pre-pro-form lacking the hemopexin domain of MMP-1, -2, -8, -9, and -13. The residues are numbered according to the generic MMP-9 nomenclature. Fn-like domain of MMP-2 and -9 is represented as XXX. Symbols denote catalytic glutamate residue (<em>asterisk</em>), and residues interacting with catalytic zinc ion (Z1), structural zinc ion (Z2), and calcium ions (Ca). Conserved residues among the five MMPs are shown with a <em>gray</em> background.<p>(TIF)</p></div><div><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s004.tif" data-ga-action="click_feat_suppl">pone.0244656.s004.tif</a><span style="color:gray"> (780K)</span></div><div class="small guid">GUID: 0575B621-B6DF-4DC8-9378-4C1C030D7B06</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S5 Fig: </strong>Purification and activation of MMP-9 mutants. (A) and (B) Pro-EK-MMP-9_Cat (WT), pro-forms of the cleft mutants, and exosite mutants were purified using HisTrap excel gel and gelatin-Sepharose resin. Each pro-MMP-9 (4 &#x003bc;M) was incubated with EKMax Enterokinase (32 U/ml) for 2 h at 4&#x000b0;C. After the activation of MMP-9, EK was removed by EKapture Agarose and buffer was exchanged for PBS at 4&#x000b0;C. SDS-PAGE analysis of the purified pro-MMP-9 (A, 1 &#x003bc;g per gel lane) and activated MMP-9 (B, 0.5 &#x003bc;g per gel lane, <em>asterisk</em> indicates 1 &#x003bc;g per gel lane) was performed under reducing conditions followed by Coomassie Brilliant Blue G-250 staining. All of the pro- and active MMP-9 mutants were highly purified. (C) MMP-9 activities of each mutant were determined by enzymatic assay using peptide substrate. Active MMP-9_Cat (WT) or activated mutants (1 nM each) were incubated with peptide substrate 3226-v (10 &#x003bc;M), and then MMP-9 activities were determined by monitoring the increase of fluorescence signal of the substrate. The activities of each mutant were normalized to that of WT. Most of the activated mutants, except for the L188A mutant, showed sufficient proteolytic activity to use for enzymatic assays. Each <em>bar</em> represents the mean &#x000b1; S.D. (n = 3).<p>(TIF)</p></div><div><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s005.tif" data-ga-action="click_feat_suppl">pone.0244656.s005.tif</a><span style="color:gray"> (2.2M)</span></div><div class="small guid">GUID: 6773C3D1-B0BF-4271-B43C-7368DABE9FE9</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S6 Fig: </strong>MMP-9 inhibitory activities of sc-311438 towards the cleft mutants (A) and the exosite mutants (B). All assay conditions and data presentation are the same as in <a href="/pmc/articles/PMC7771667/figure/pone.0244656.g005/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g005" rid-ob="ob-pone.0244656.g005" co-legend-rid="lgnd_pone.0244656.g005"><span>Fig 5</span></a>, and Tables <a href="/pmc/articles/PMC7771667/table/pone.0244656.t002/" target="table" class="fig-table-link figpopup" rid-figpopup="pone.0244656.t002" rid-ob="ob-pone.0244656.t002" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Tables2</span></span><span>2</span></a> and <a href="/pmc/articles/PMC7771667/table/pone.0244656.t003/" target="table" class="fig-table-link figpopup" rid-figpopup="pone.0244656.t003" rid-ob="ob-pone.0244656.t003" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and3.</span></span><span>3</span></a>. Statistical analysis of cleft mutants <em>versus</em> WT was performed by one-way ANOVA with Dunnett&#x02019;s post tests for multiple comparisons.<p>(TIF)</p></div><div><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s006.tif" data-ga-action="click_feat_suppl">pone.0244656.s006.tif</a><span style="color:gray"> (3.1M)</span></div><div class="small guid">GUID: 9DFC3DA3-0C11-4B4C-B975-AFCB9D50354E</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S7 Fig: </strong>MMP-9 inhibitory activities of <a class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> towards three different MMP-9 constructs. For enzymatic assay with three different MMP-9 constructs, namely, full-length MMP-9, the form with HPX domain deleted (MMP-9_Cat), and the form with both Fn-like domain and HPX domain deleted (the catalytic domain), 0.4 nM active MMP-9 was incubated with threefold serially diluted inhibitors (0&#x02013;100 nM) for 1 h. Following incubation, substrate 3226-v was added to achieve a final concentration of 10 &#x003bc;M. The other assay conditions are the same as in <a href="/pmc/articles/PMC7771667/figure/pone.0244656.g001/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g001" rid-ob="ob-pone.0244656.g001" co-legend-rid="lgnd_pone.0244656.g001"><span>Fig 1C</span></a>.<p>(TIF)</p></div><div><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s007.tif" data-ga-action="click_feat_suppl">pone.0244656.s007.tif</a><span style="color:gray"> (151K)</span></div><div class="small guid">GUID: 83BAC4E2-C2B2-4C34-AACE-2D654C710F09</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S8 Fig: </strong>Expression of active-site mutants of MMP-9 (H401A, H405A, and H411A). Western blot analysis of the culture supernatants (6.5 &#x003bc;l per gel lane) of HEK293F cells transfected with each MMP-9 expression vector was performed under reducing conditions. After electrophoresis, the proteins were transferred to a PVDF membrane followed by blocking with 5% skim milk in PBS-T. After washing with PBS-T, Penta His HRP Conjugate (QIAGEN, 34460) was added (1:10,000 dilution in PBS-T with 0.5% skim milk) and incubated for 1 h at room temperature. After washing with PBS-T, the reaction was developed with ECL Prime Western Blotting Detection Reagent (GE Healthcare) at room temperature. The pre-stained visible protein markers and the chemiluminescent signals were captured using a ChemiDoc XRS+ CCD camera-based imager system (Bio-Rad). <em>Black arrowhead</em> indicates the band of full-length MMP-9 fused to a C-terminal His<sub>6</sub> tag.<p>(TIF)</p></div><div><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s008.tif" data-ga-action="click_feat_suppl">pone.0244656.s008.tif</a><span style="color:gray"> (39K)</span></div><div class="small guid">GUID: 367F250B-E403-429D-8A33-B6FDFC1BFAFF</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S9 Fig: </strong>C-terminal degradation of MMP-9 during activation reaction using trypsin. For MMP-9 activation using trypsin, the final 4 &#x003bc;M pro-MMP-9_Cat-H6 was incubated with 2.7 &#x003bc;M TPCK trypsin (Thermo Fisher Scientific, 20233) in TNC buffer at 37&#x000b0;C. Western blot analysis of the activation reaction solution (0.6 &#x003bc;g of MMP-9 per gel lane) was performed under reducing conditions. The experimental conditions after electrophoresis were carried out according to <a href="#pone.0244656.s008" rid="pone.0244656.s008" class=" supplementary-material">S8 Fig</a>. <em>Black arrowhead</em> and <em>red arrowhead</em> indicate the band of the product obtained by the activation reaction using trypsin and the band of C-terminal fragment of pro-MMP-9_Cat-H6, respectively.<p>(TIF)</p></div><div><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s009.tif" data-ga-action="click_feat_suppl">pone.0244656.s009.tif</a><span style="color:gray"> (217K)</span></div><div class="small guid">GUID: 17444582-7590-4BF4-9E9E-36E5A5CC490E</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S10 Fig: </strong>Autolysis of active MMP-9 upon long-term incubation at 25&#x000b0;C. To evaluate the stability of the bait protein, 2.4 &#x003bc;M of biotinylated active MMP-9_Cat-BAP was incubated in PBS at 25&#x000b0;C for 16 h. SDS-PAGE analysis of the biotinylated active MMP-9_Cat-BAP after incubation (0.5 &#x003bc;g per gel lane) was performed under reducing conditions followed by Coomassie Brilliant Blue G-250 staining. <em>Black arrowhead</em> and <em>red arrowhead</em> indicate the band of the biotinylated active MMP-9_Cat-BAP and the band of degradation product, respectively.<p>(TIF)</p></div><div><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s010.tif" data-ga-action="click_feat_suppl">pone.0244656.s010.tif</a><span style="color:gray"> (257K)</span></div><div class="small guid">GUID: 924428F1-AAA7-4F60-98EC-872121B6AFBD</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S1 Table: </strong>MMP inhibitory activities of inhibitors against eight MMPs. The cross-reactivities against MMP-3, -7, -10, -12, -14, -15, -16, and -17 were measured by enzymatic assay using peptide substrate. Each active MMP was incubated with inhibitors (1 &#x003bc;M), and then the peptide substrate (10 &#x003bc;M 3168-v for MMP-3, 10 &#x003bc;M 3226-v for the other MMPs) was added as described under &#x0201c;Materials and methods.&#x0201d; Enzymatic activity was determined by monitoring the hydrolysis of the peptide substrate and each remaining enzymatic activity was normalized to the activity in the absence of inhibitors. Data are shown as the mean &#x000b1; S.D. (n = 3).<p>(DOCX)</p></div><div><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s011.docx" data-ga-action="click_feat_suppl">pone.0244656.s011.docx</a><span style="color:gray"> (27K)</span></div><div class="small guid">GUID: 8F3C154A-FCE1-4FAC-9FD3-EBA1183AB91F</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S2 Table: </strong>MMP enzymatic assay conditions. (DOCX)</div><div><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s012.docx" data-ga-action="click_feat_suppl">pone.0244656.s012.docx</a><span style="color:gray"> (26K)</span></div><div class="small guid">GUID: 076C0540-3639-40A3-9275-12093FED6E76</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S3 Table: </strong>PCR primers used for site-directed mutagenesis. (DOCX)</div><div><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s013.docx" data-ga-action="click_feat_suppl">pone.0244656.s013.docx</a><span style="color:gray"> (29K)</span></div><div class="small guid">GUID: AFC90C9A-63EF-4E1E-8EF8-C1437E8476EE</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S1 Protocol: </strong>Protocol for ELISA assay using anti-MMP-9 antibody GS-5745. (DOCX)</div><div><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s014.docx" data-ga-action="click_feat_suppl">pone.0244656.s014.docx</a><span style="color:gray"> (25K)</span></div><div class="small guid">GUID: B1A5C9A0-DCFE-4531-B56F-FB052DFB5E57</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S1 Raw images: </strong>The original uncropped and unadjusted image to prepare the final figures. Images were captured using a ChemiDocXRS+ CCD camera-based imager system (Bio-Rad). The part of the blot or gel shown in the final figure is within the red box. Lanes marked with an "X" were not included in the final figure.<p>(PDF)</p></div><div><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s015.pdf" data-ga-action="click_feat_suppl">pone.0244656.s015.pdf</a><span style="color:gray"> (895K)</span></div><div class="small guid">GUID: 628978FE-FD2F-4F24-B084-E81F3ADD6D20</div></div></dd></dl><dl data-length="81" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p>All relevant data are within the manuscript and its <a href="#sec019" rid="sec019" class=" sec">Supporting information</a> files.</p></dd></dl></div><div id="abstract-a.ah.b.s" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.ah.b.stitle">Abstract</h2><!--article-meta--><div><p class="p p-first-last">Matrix metalloproteinases (MMPs) contribute to many physiological and pathological phenomena via the proteolysis of extracellular matrix components. Specific blocking of the active site of each MMP sheds light on its particular role. However, it remains difficult to acquire an active-site inhibitor with high specificity for only the target MMP due to the highly conserved structure around the active site of MMPs. Recently, we reported that potent and specific inhibitors of serine proteases were obtained from our proprietary engineered serine protease inhibitor Kazal type 2 (SPINK2) library. In this research, using this library, we succeeded in obtaining potent and specific MMP-9 inhibitors. The obtained inhibitors bound to the active site of MMP-9 and inhibited MMP-9 with low nanomolar <em>K</em><sub><em>i</em></sub> values. The inhibitors did not cross-react with other MMPs that we tested. Further analysis using MMP-9 mutants demonstrated that the inhibitors recognize not only the residues around the conserved active site of MMP-9 but also different and unique residues in exosites that are distant from each other. This unique recognition manner, which can be achieved by the large interface provided by engineered SPINK2, may contribute to the generation of specific active-site inhibitors of MMPs.</p></div></div><div id="sec001" class="tsec sec"><h2 class="head no_bottom_margin" id="sec001title">Introduction</h2><p class="p p-first">Matrix metalloproteinases (MMPs) contribute to the degradation of extracellular matrix (ECM) proteins (<em>e</em>.<em>g</em>., collagen, elastin, fibronectin, and laminin) and the processing of pro-inflammatory cytokines (<em>e</em>.<em>g</em>., IL-1&#x003b2;) and chemokines (<em>e</em>.<em>g</em>., IL-8/CXCL8). Through these proteolytic functions, MMPs play crucial roles in many physiological functions related to cell growth, differentiation, migration, and apoptosis. It has also been reported that aberrant expression or activation of MMPs may cause pathological conditions such as inflammation [<a href="#pone.0244656.ref001" rid="pone.0244656.ref001" class=" bibr popnode">1</a>] and tumor metastasis [<a href="#pone.0244656.ref002" rid="pone.0244656.ref002" class=" bibr popnode">2</a>]. Twenty-three MMP proteins have been reported in humans to date [<a href="#pone.0244656.ref003" rid="pone.0244656.ref003" class=" bibr popnode">3</a>,<a href="#pone.0244656.ref004" rid="pone.0244656.ref004" class=" bibr popnode">4</a>] and several MMPs are known to be upregulated in diseases involving abnormal degradation of the extracellular matrix, such as inflammatory bowel disease [<a href="#pone.0244656.ref003" rid="pone.0244656.ref003" class=" bibr popnode">3</a>]. To further advance MMP-related research, it is highly desirable to obtain molecules that specifically inhibit the enzymatic activity of a target MMP. Revealing the roles of individual MMPs in physiological and pathological conditions using specific inhibitors can contribute to the generation of therapeutics targeting MMPs.</p><p>A catalytic zinc ion, which is chelated by three histidine residues, and a catalytic glutamate residue are essential for the proteolytic activity of MMPs and constitute their active site. The sequence motif of this active site of MMPs, <span class="underline">HE</span>XG<span class="underline">H</span>XXGXX<span class="underline">H</span>, is highly conserved within the MMP family. MMPs are translated and secreted as the pro-form (pro-MMPs) without proteolytic activity. Both the active site and the surrounding region of the active site, the so-called active-site cleft, are completely masked by the pro-peptide in pro-MMPs, interfering with the interaction between the active site and the substrate [<a href="#pone.0244656.ref005" rid="pone.0244656.ref005" class=" bibr popnode">5</a>]. Pro-MMP is activated by enzymatic digestion (<em>e</em>.<em>g</em>., by plasmin, trypsin, and other MMPs) [<a href="#pone.0244656.ref006" rid="pone.0244656.ref006" class=" bibr popnode">6</a>,<a href="#pone.0244656.ref007" rid="pone.0244656.ref007" class=" bibr popnode">7</a>] or non-proteolytic modification [<a href="#pone.0244656.ref008" rid="pone.0244656.ref008" class=" bibr popnode">8</a>&#x02013;<a href="#pone.0244656.ref010" rid="pone.0244656.ref010" class=" bibr popnode">10</a>] that causes removal of the pro-peptide from the active-site cleft. MMPs share a consensus cleavage motif, proline at P3 and a hydrophobic residue at P1&#x02032;, which results from high sequence homology of the active-site cleft [<a href="#pone.0244656.ref011" rid="pone.0244656.ref011" class=" bibr popnode">11</a>,<a href="#pone.0244656.ref012" rid="pone.0244656.ref012" class=" bibr popnode">12</a>]. Each MMP shows some substrate specificity towards macromolecular substrates [<a href="#pone.0244656.ref004" rid="pone.0244656.ref004" class=" bibr popnode">4</a>]. This is because not only the active site but also a secondary binding site away from the active site, called an exosite, is involved in substrate recognition by MMPs. For example, some domains in MMPs are essential for the degradation of collagen: the hemopexin (HPX) domain fused to the catalytic domain by a linker in MMP-1, -8, -13, and -14; and the fibronectin (Fn)-like domain inserted in the catalytic domain in MMP-2 and -9 [<a href="#pone.0244656.ref013" rid="pone.0244656.ref013" class=" bibr popnode">13</a>&#x02013;<a href="#pone.0244656.ref015" rid="pone.0244656.ref015" class=" bibr popnode">15</a>]. MMPs utilize such domains that exhibit some diversity among MMP family members as exosites to recognize macromolecular substrates, facilitating the maintenance of an appropriate distance and direction between the cleavage site and the active site; this in turn contributes to the specific degradation of macromolecular substrates.</p><p>The proteolytic activities of MMPs are completely inhibited by molecules that bind to their active site. Many chemical compounds that interact with part of the active site have been reported as active-site inhibitors of MMPs. Most of these active-site chemical inhibitors show high potency by chelating the catalytic zinc ion; however, they also show broad inhibitory activity across the whole family of MMPs due to recognition of the highly homologous active-site cleft. Such non-specific MMP inhibitors (<em>e</em>.<em>g</em>., hydroxamate-based inhibitors) achieved promising therapeutic effects in several preclinical studies for cancer; however, in clinical trials, these inhibitors caused serious side effects such as musculoskeletal pain and inflammation due to the broad inhibition of MMPs, so all of these trials failed [<a href="#pone.0244656.ref003" rid="pone.0244656.ref003" class=" bibr popnode">3</a>]. Therefore, specific active-site inhibitors of MMPs are strongly desired from a clinical perspective. Protein-based active-site inhibitors that have the potential to recognize both the active site and the exosite in a manner similar to that of macromolecular substrates were also reported. Anti-MMP-9 antibodies, SDS3 and SDS4, showed high potency with active-site binding but also inhibited MMP-2, the sequence of which is the most similar to that of MMP-9 among the MMP family [<a href="#pone.0244656.ref016" rid="pone.0244656.ref016" class=" bibr popnode">16</a>]. Several attempts to generate specific active-site inhibitors from broadly inhibitory proteins by protein engineering were also reported. Endogenous tissue inhibitors of metalloproteinases (TIMPs) strongly but non-selectively inhibit MMPs by coordinating the catalytic zinc ion [<a href="#pone.0244656.ref017" rid="pone.0244656.ref017" class=" bibr popnode">17</a>]. Mutations to TIMP residues that interact with the exosite of MMPs improved the specificity to MMPs [<a href="#pone.0244656.ref018" rid="pone.0244656.ref018" class=" bibr popnode">18</a>&#x02013;<a href="#pone.0244656.ref020" rid="pone.0244656.ref020" class=" bibr popnode">20</a>]. However, engineered TIMPs that could inhibit only a single MMP have not been reported. Other attempts have been made to target exosite domains, such as the HPX domain, which are less conserved. Several exosite inhibitors abrogated cancer cell migration through specific prevention of the association of pro-MMP-9 with both &#x003b1;4&#x003b2;1 integrin and CD44 without modulating the catalytic activity of MMP-9 [<a href="#pone.0244656.ref021" rid="pone.0244656.ref021" class=" bibr popnode">21</a>&#x02013;<a href="#pone.0244656.ref023" rid="pone.0244656.ref023" class=" bibr popnode">23</a>]. Although exosite recognition could contribute to improving the specificity of active-site inhibitors, it remains difficult to obtain specific active-site inhibitors against a single MMP.</p><p class="p p-last">Recently, we reported that potent and specific inhibitors of serine proteases were obtained from our proprietary technology, namely, the engineered serine protease inhibitor Kazal type 2 (SPINK2) library [<a href="#pone.0244656.ref024" rid="pone.0244656.ref024" class=" bibr popnode">24</a>]. Here, using this library, we attempted to obtain a specific active-site inhibitor of MMP-9, which is involved in the degradation of ECM proteins including type IV collagen, gelatin, and elastin [<a href="#pone.0244656.ref010" rid="pone.0244656.ref010" class=" bibr popnode">10</a>]. MMP-9 breaks down the physical barrier formed by ECM, promoting cancer metastasis and inflammation [<a href="#pone.0244656.ref001" rid="pone.0244656.ref001" class=" bibr popnode">1</a>,<a href="#pone.0244656.ref025" rid="pone.0244656.ref025" class=" bibr popnode">25</a>,<a href="#pone.0244656.ref026" rid="pone.0244656.ref026" class=" bibr popnode">26</a>]. Furthermore, aberrant MMP-9 activity causes blood-brain barrier disruption which can involve in numerous neurodegenerative diseases such as multiple sclerosis, stroke, epilepsy, brain tumors, and Alzheimer&#x02019;s disease [<a href="#pone.0244656.ref027" rid="pone.0244656.ref027" class=" bibr popnode">27</a>,<a href="#pone.0244656.ref028" rid="pone.0244656.ref028" class=" bibr popnode">28</a>]. Recent studies have reported the promising efficacy of MMP-9 inhibitors in several pathological in vivo models [<a href="#pone.0244656.ref029" rid="pone.0244656.ref029" class=" bibr popnode">29</a>&#x02013;<a href="#pone.0244656.ref032" rid="pone.0244656.ref032" class=" bibr popnode">32</a>], it is also required to create MMP-9 inhibitors with higher potency and selectivity [<a href="#pone.0244656.ref026" rid="pone.0244656.ref026" class=" bibr popnode">26</a>]. We demonstrate that several inhibitors obtained from our library strongly inhibit the proteolytic activity of MMP-9 and show high specificity to only MMP-9, while not inhibiting the other 12 MMPs. Several assays using MMP-9 mutants suggested that these inhibitors recognized not only the highly conserved active site but also the specific exosite of MMP-9 to achieve the high specificity. These specific MMP-9 inhibitors discovered from the engineered SPINK2 library may contribute to clarifying the roles of MMP-9 under physiological and pathological conditions and to the development of therapeutics for MMP-9-related diseases.</p></div><div id="sec002" class="tsec sec"><h2 class="head no_bottom_margin" id="sec002title">Results</h2><div id="sec003" class="sec sec-first"><h3 id="sec003title">MMP-9 inhibitors were isolated from the engineered SPINK2 phage display library</h3><p class="p p-first">Prior to the screening of inhibitors against MMP-9 from the engineered SPINK2 phage library, we designed a recombinant pro-MMP-9 construct and examined the method of its activation to prepare highly qualified bait proteins for phage panning. First, we investigated which domains were required for the bait protein. MMP-9 has several functional domains: pro-peptide domain, catalytic domain, Fn-like domain, and HPX domain [<a href="#pone.0244656.ref001" rid="pone.0244656.ref001" class=" bibr popnode">1</a>]. As the Fn-like domain directly contacts the catalytic domain [<a href="#pone.0244656.ref005" rid="pone.0244656.ref005" class=" bibr popnode">5</a>], it is preferable to use a bait protein that retains the Fn-like domain in order to obtain inhibitors recognizing the native structure of MMP-9. On the other hand, the HPX domain is connected to the catalytic domain by a long and flexible linker region [<a href="#pone.0244656.ref033" rid="pone.0244656.ref033" class=" bibr popnode">33</a>]; thus, deletion of the HPX domain would not affect the surface exposure of the catalytic domain. Hence, we prepared a pro-MMP-9 catalytic domain with an Fn-like domain and without an HPX domain (pro-MMP-9_Cat) for subsequent experiments. In general, proteases labeled by random biotinylation using chemical reagents can be inactive due to blocking of the active-site cleft, so we prepared biotinylated pro-MMP-9_Cat with a site-specific biotin acceptor peptide (BAP) [<a href="#pone.0244656.ref034" rid="pone.0244656.ref034" class=" bibr popnode">34</a>] using biotin-protein ligase BirA. Finally, to obtain homogeneous active MMP-9, we examined several methods to activate pro-MMP-9 using MMP-3 [<a href="#pone.0244656.ref006" rid="pone.0244656.ref006" class=" bibr popnode">6</a>], trypsin [<a href="#pone.0244656.ref007" rid="pone.0244656.ref007" class=" bibr popnode">7</a>], or 4-aminophenylmercuric acetate (APMA) [<a href="#pone.0244656.ref008" rid="pone.0244656.ref008" class=" bibr popnode">8</a>,<a href="#pone.0244656.ref009" rid="pone.0244656.ref009" class=" bibr popnode">9</a>]. Trypsin could activate pro-MMP-9_Cat, but it also digested various sites in MMP-9 and cleaved off its C-terminal tag (<a href="#pone.0244656.s009" rid="pone.0244656.s009" class=" supplementary-material">S9 Fig</a>). Compared with the method using APMA, MMP-9 activated by MMP-3 showed higher proteolytic activity as previously reported [<a href="#pone.0244656.ref035" rid="pone.0244656.ref035" class=" bibr popnode">35</a>]. In addition, fewer extra bands including those representing inactivated pro-MMP-9 and degraded MMP-9 were detected in SDS-PAGE, indicating that MMP-9 activated by MMP-3 was more homogeneous. Thus, we prepared biotinylated active MMP-9_Cat-BAP as the highly qualified bait protein by the activation method using MMP-3 (<a href="#pone.0244656.s001" rid="pone.0244656.s001" class=" supplementary-material">S1A Fig</a>).</p><p class="p p-last">Then, we carried out phage panning against biotinylated active MMP-9_Cat-BAP using the engineered SPINK2 phage library [<a href="#pone.0244656.ref024" rid="pone.0244656.ref024" class=" bibr popnode">24</a>]. After three rounds of panning, we screened approximately 2,000 clones as thioredoxin tag-fused proteins expressed in <em>E</em>. <em>coli</em> by MMP-9 inhibitory assay using FRET peptide substrate. As a result, we obtained more than 380 hit clones. One hundred and thirty unique inhibitors were identified by DNA sequencing, indicating that the hit rate of unique inhibitors in this screening was 6.5%. Fourteen inhibitors with IC<sub>50</sub> &#x0003c;10 nM were obtained in the MMP-9 inhibitory assay, and we finally selected four inhibitors: M91002, <a id="__tag_904612487" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a>, <a id="__tag_904612453" class="tag_hotlink" href="/nuccore/M91011" ref="/nuccore/M91011"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91011","term_id":"337099","term_text":"M91011"}}</span>M91011</a>, and <a id="__tag_904612508" class="tag_hotlink" href="/nuccore/M91012" ref="/nuccore/M91012"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91012","term_id":"343148","term_text":"M91012"}}</span>M91012</a>. Comparing the loop sequences of their inhibitors (<a href="#pone.0244656.s002" rid="pone.0244656.s002" class=" supplementary-material">S2D Fig</a>) with previously reported MMP-9 substrate sequences [<a href="#pone.0244656.ref011" rid="pone.0244656.ref011" class=" bibr popnode">11</a>,<a href="#pone.0244656.ref036" rid="pone.0244656.ref036" class=" bibr popnode">36</a>] and the information in the MEROPS database, no substrate sequence motifs of MMP-9 were included in their inhibitors. We purified the selected inhibitors by removing the thioredoxin tag, followed by size-exclusion chromatography for further evaluation.</p></div><div id="sec004" class="sec"><h3 id="sec004title">Engineered SPINK2-derived inhibitors show potent and specific MMP-9 inhibitory activity</h3><p class="p p-first">We evaluated the binding and inhibitory activity of each inhibitor. In ELISA, all inhibitors showed strong binding activities to active MMP-9 in a concentration-dependent manner (<a href="/pmc/articles/PMC7771667/figure/pone.0244656.g001/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g001" rid-ob="ob-pone.0244656.g001" co-legend-rid="lgnd_pone.0244656.g001"><span>Fig 1A</span></a>) and their EC<sub>50</sub> values were double-digit nM (<a href="/pmc/articles/PMC7771667/table/pone.0244656.t001/" target="table" class="fig-table-link figpopup" rid-figpopup="pone.0244656.t001" rid-ob="ob-pone.0244656.t001" co-legend-rid=""><span>Table 1</span></a>). These inhibitors did not show any binding to pro-MMP-9 at concentrations up to 1 &#x003bc;M (<a href="/pmc/articles/PMC7771667/figure/pone.0244656.g001/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g001" rid-ob="ob-pone.0244656.g001" co-legend-rid="lgnd_pone.0244656.g001"><span>Fig 1B</span></a>), while anti-MMP-9 antibody GS-5745 [<a href="#pone.0244656.ref037" rid="pone.0244656.ref037" class=" bibr popnode">37</a>] bound to both active and pro-MMP-9 (<a href="#pone.0244656.s003" rid="pone.0244656.s003" class=" supplementary-material">S3 Fig</a>). In the enzyme assay using the peptide substrate, all inhibitors strongly inhibited MMP-9 activity and the <em>K</em><sub><em>i</em></sub> values of all inhibitors were low single-digit nM (<a href="/pmc/articles/PMC7771667/figure/pone.0244656.g001/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g001" rid-ob="ob-pone.0244656.g001" co-legend-rid="lgnd_pone.0244656.g001"><span>Fig 1C</span></a> and <a href="/pmc/articles/PMC7771667/table/pone.0244656.t001/" target="table" class="fig-table-link figpopup" rid-figpopup="pone.0244656.t001" rid-ob="ob-pone.0244656.t001" co-legend-rid=""><span>Table 1</span></a>). They also showed MMP-9 inhibitory activity in the enzyme assay using gelatin as a protein substrate (<a href="/pmc/articles/PMC7771667/figure/pone.0244656.g001/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g001" rid-ob="ob-pone.0244656.g001" co-legend-rid="lgnd_pone.0244656.g001"><span>Fig 1D</span></a> and <a href="/pmc/articles/PMC7771667/table/pone.0244656.t001/" target="table" class="fig-table-link figpopup" rid-figpopup="pone.0244656.t001" rid-ob="ob-pone.0244656.t001" co-legend-rid=""><span>Table 1</span></a>). These results demonstrated that the obtained MMP-9 inhibitors had potent inhibitory activity.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="pone.0244656.g001" co-legend-rid="lgnd_pone.0244656.g001"><a href="/pmc/articles/PMC7771667/figure/pone.0244656.g001/" target="figure" rid-figpopup="pone.0244656.g001" rid-ob="ob-pone.0244656.g001"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140659104119632"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7771667_pone.0244656.g001.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pone.0244656.g001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC7771667/bin/pone.0244656.g001.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140659104119632"><a target="object" rel="noopener" href="/pmc/articles/PMC7771667/figure/pone.0244656.g001/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_pone.0244656.g001"><div><a class="figpopup" href="/pmc/articles/PMC7771667/figure/pone.0244656.g001/" target="figure" rid-figpopup="pone.0244656.g001" rid-ob="ob-pone.0244656.g001">Fig 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Binding and inhibitory activity of engineered SPINK2-derived inhibitors.</strong><p>(A) The binding activity to active MMP-9 was measured by ELISA. Various concentrations of engineered SPINK2-derived inhibitors or wild-type SPINK2 (1&#x02013;1,000 nM) were added to the biotinylated active MMP-9_Cat-Avi (50 nM)-coated plate, and then inhibitors bound to active MMP-9 were detected by HRP-conjugated anti-S tag antibody. (B) The binding activities to pro- and active MMP-9 were measured by ELISA. Biotinylated pro-MMP-9_Cat-Avi or biotinylated active MMP-9_Cat-Avi (50 nM each) was coated on the plate, to which inhibitors (1 &#x003bc;M) were then added. The inhibitors bound to MMP-9 were detected by HRP-conjugated anti-S tag antibody. (C) MMP-9 inhibitory activity of <a id="__tag_904612475" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> was measured by enzymatic assay using peptide substrate. Active MMP-9_Cat-H6 (0.4 nM) was incubated with various concentrations of <a id="__tag_904612471" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> (0&#x02013;25 nM) for 1 h at 37&#x000b0;C, and then peptide substrate 3226-v (10 &#x003bc;M) was added. MMP-9 activity was determined by monitoring the hydrolysis of the peptide substrate, and the activity in the absence of <a id="__tag_904612478" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> was taken as 100%. (D) MMP-9 inhibitory activity of engineered SPINK2-derived inhibitors was measured by enzymatic assay using the macromolecular substrate. Active MMP-9_Cat-H6 (0.6 nM) was incubated with various concentrations of inhibitors (0&#x02013;100 nM) for 1 h at 37&#x000b0;C, and then DQ-gelatin (10 &#x003bc;g/ml) was added. MMP-9 activity was determined by monitoring the hydrolysis of DQ-gelatin and the activity in the absence of inhibitor was taken as 100%. Data are shown as the mean of duplicate experiments in (A) and the mean &#x000b1; S.D. (n = 3) in (B&#x02013;D). All curves were obtained by non-linear curve fitting as described in &#x0201c;Materials and methods&#x0201d;.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="pone.0244656.t001"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Binding affinity and inhibitory activity of inhibitors against active MMP-9.</strong></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="center" rowspan="2" colspan="1">Clone</th><th align="center" rowspan="2" colspan="1">ELISA EC<sub>50</sub> (nM)<sup>a</sup></th><th align="center" rowspan="1" colspan="1">Peptide</th><th align="center" colspan="2" rowspan="1">Gelatin</th></tr><tr><th align="center" rowspan="1" colspan="1"><em>K</em><sub><em>i</em></sub> (nM)<sup>b</sup></th><th align="center" rowspan="1" colspan="1">IC<sub>50</sub> (nM)<sup>c</sup></th><th align="center" rowspan="1" colspan="1">R<sup>2</sup></th></tr></thead><tbody><tr><td align="center" rowspan="1" colspan="1">M91002</td><td align="center" rowspan="1" colspan="1">47</td><td align="center" rowspan="1" colspan="1">2.3 &#x000b1; 0.6</td><td align="center" rowspan="1" colspan="1">8.5 &#x000b1; 2.1</td><td align="center" rowspan="1" colspan="1">0.984</td></tr><tr><td align="center" rowspan="1" colspan="1"><a id="__tag_904612451" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a></td><td align="center" rowspan="1" colspan="1">46</td><td align="center" rowspan="1" colspan="1">1.4 &#x000b1; 0.2</td><td align="center" rowspan="1" colspan="1">8.2 &#x000b1; 1.2</td><td align="center" rowspan="1" colspan="1">0.991</td></tr><tr><td align="center" rowspan="1" colspan="1"><a id="__tag_904612468" class="tag_hotlink" href="/nuccore/M91011" ref="/nuccore/M91011"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91011","term_id":"337099","term_text":"M91011"}}</span>M91011</a></td><td align="center" rowspan="1" colspan="1">16</td><td align="center" rowspan="1" colspan="1">1.6 &#x000b1; 0.1</td><td align="center" rowspan="1" colspan="1">5.0 &#x000b1; 0.5</td><td align="center" rowspan="1" colspan="1">0.993</td></tr><tr><td align="center" rowspan="1" colspan="1"><a id="__tag_904612497" class="tag_hotlink" href="/nuccore/M91012" ref="/nuccore/M91012"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91012","term_id":"343148","term_text":"M91012"}}</span>M91012</a></td><td align="center" rowspan="1" colspan="1">46</td><td align="center" rowspan="1" colspan="1">2.1 &#x000b1; 0.1</td><td align="center" rowspan="1" colspan="1">8.6 &#x000b1; 1.7</td><td align="center" rowspan="1" colspan="1">0.993</td></tr><tr><td align="center" rowspan="1" colspan="1">sc-311438</td><td align="center" rowspan="1" colspan="1">N/A<sup>d</sup></td><td align="center" rowspan="1" colspan="1">1.5 &#x000b1; 0.2</td><td align="center" rowspan="1" colspan="1">2.3 &#x000b1; 0.1</td><td align="center" rowspan="1" colspan="1">0.997</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140659104115072"><a target="object" rel="noopener" href="/pmc/articles/PMC7771667/table/pone.0244656.t001/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t001fn001"><p class="p p-first-last"><sup><em>a</em></sup>EC<sub>50</sub> was determined by ELISA as the mean (n = 2).</p></div><div id="t001fn002"><p class="p p-first-last"><sup><em>b</em></sup><em>K</em><sub><em>i</em></sub> was determined by the MMP-9 inhibitory assay with 10 &#x003bc;M peptide substrate 3226-v. The <em>K</em><sub><em>i</em></sub> values are shown as the mean &#x000b1; standard deviation (S.D.) (n = 3).</p></div><div id="t001fn003"><p class="p p-first-last"><sup><em>c</em></sup>IC<sub>50</sub> was determined by the MMP-9 inhibitory assay with 10 &#x003bc;g/ml DQ-gelatin substrate. The IC<sub>50</sub> values are shown as the mean &#x000b1; S.D. (n = 3).</p></div><div id="t001fn004"><p class="p p-first-last"><sup><em>d</em></sup>N/A denotes &#x0201c;not applicable&#x0201d;.</p></div></div></div><p class="p">For the cross-reactivity test, we evaluated the inhibitory activities against MMP-1, -2, -8, and -13, the primary sequence in the catalytic domain of each of which is similar to that of MMP-9 (<a href="#pone.0244656.s004" rid="pone.0244656.s004" class=" supplementary-material">S4 Fig</a>). Although the chemical active-site inhibitor sc-311438 completely inhibited all of the MMPs, our MMP-9 inhibitors did not inhibit them at a final concentration of 1 &#x003bc;M (<a href="/pmc/articles/PMC7771667/figure/pone.0244656.g002/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g002" rid-ob="ob-pone.0244656.g002" co-legend-rid="lgnd_pone.0244656.g002"><span>Fig 2</span></a>). Additionally, they did not show any inhibitory activity against the other tested MMPs (<a href="#pone.0244656.s011" rid="pone.0244656.s011" class=" supplementary-material">S1 Table</a>). Therefore, the obtained MMP-9 inhibitors showed potent inhibition with high specificity to MMP-9.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="pone.0244656.g002" co-legend-rid="lgnd_pone.0244656.g002"><a href="/pmc/articles/PMC7771667/figure/pone.0244656.g002/" target="figure" rid-figpopup="pone.0244656.g002" rid-ob="ob-pone.0244656.g002"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140659064778048"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7771667_pone.0244656.g002.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pone.0244656.g002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC7771667/bin/pone.0244656.g002.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140659064778048"><a target="object" rel="noopener" href="/pmc/articles/PMC7771667/figure/pone.0244656.g002/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_pone.0244656.g002"><div><a class="figpopup" href="/pmc/articles/PMC7771667/figure/pone.0244656.g002/" target="figure" rid-figpopup="pone.0244656.g002" rid-ob="ob-pone.0244656.g002">Fig 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Cross-reactivity of engineered SPINK2-derived inhibitors against MMP-9.</strong><p>The cross-reactivities against MMP-1, -2, -8, and -13 were measured by enzymatic assay using the peptide substrate. Each active MMP (MMP-1, 5 nM; MMP-2, 1.4 nM; MMP-8, 0.6 nM; MMP-13, 2 nM) was incubated with inhibitors (1 &#x003bc;M) for 10 min (for MMP-13) or 60 min (for MMP-1, -2, and -8) at 37&#x000b0;C, and then peptide substrate 3226-v (10 &#x003bc;M for MMP-1, -8, and -13, or 50 &#x003bc;M for MMP-2) was added. Enzymatic activity was determined by monitoring the hydrolysis of the peptide substrate, and each remaining enzymatic activity was normalized to the activity in the absence of inhibitors. Each <em>bar</em> represents the mean &#x000b1; S.D. (n = 3).</p></div></div></div></div><div id="sec005" class="sec"><h3 id="sec005title"><a id="__tag_904612443" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> interacts with catalytic domain of MMP-9</h3><p class="p p-first">The question arises as to why, despite MMPs showing high sequence homology with each other, the obtained inhibitors achieved high specificity. To address this, we investigated how the inhibitor recognizes MMP-9 using the most potent inhibitor, <a id="__tag_904612503" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a>.</p><p class="p p-last">At the beginning of this investigation, we examined which domain of MMP-9 was involved in the interaction with <a id="__tag_904612447" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a>. MMP-9 inhibitory assays were performed using full-length MMP-9, the form with the HPX domain deleted (MMP-9_Cat), and the form with both Fn-like domain and HPX domain deleted (the catalytic domain). <a id="__tag_904612496" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> inhibited all forms of active MMP-9 with similar inhibitory activities (<a href="#pone.0244656.s007" rid="pone.0244656.s007" class=" supplementary-material">S7 Fig</a>), suggesting that it interacts with the catalytic domain but not the Fn-like domain or the HPX domain. Therefore, we focused on the catalytic domain in the following analysis.</p></div><div id="sec006" class="sec"><h3 id="sec006title"><a id="__tag_904612458" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> binds to catalytic glutamate residue and active-site cleft of MMP-9</h3><p class="p p-first">We investigated which region of the catalytic domain contributes to the interaction with <a id="__tag_904612499" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a>. Given that <a id="__tag_904612484" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> bound not to pro-MMP-9 but to active MMP-9, it would be expected to bind to the region including the active site and the active-site cleft, which are hidden by the pro-peptide in pro-MMP-9.</p><p>First, to reveal whether <a id="__tag_904612481" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> recognizes the active site, we performed the assay using the active-site mutant of MMP-9. The active site of MMP-9 consists of a catalytic zinc ion, catalytic glutamate residue (Glu-402; <a href="/pmc/articles/PMC7771667/figure/pone.0244656.g003/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g003" rid-ob="ob-pone.0244656.g003" co-legend-rid="lgnd_pone.0244656.g003"><span>Fig 3</span></a>
<em>blue</em>), and three histidine residues (His-401, His-405, and His-411; <a href="/pmc/articles/PMC7771667/figure/pone.0244656.g003/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g003" rid-ob="ob-pone.0244656.g003" co-legend-rid="lgnd_pone.0244656.g003"><span>Fig 3</span></a>
<em>green</em>) chelating catalytic zinc. Four kinds of active-site mutants (H401A, E402Q, H405A, and H411A) were constructed in accordance with a previous report [<a href="#pone.0244656.ref038" rid="pone.0244656.ref038" class=" bibr popnode">38</a>]. Pro-MMP-9_Cat_E402Q-H6 was well expressed in HEK293F cells, while the other active-site mutants H401A, H405A, and H411A were not expressed (<a href="#pone.0244656.s008" rid="pone.0244656.s008" class=" supplementary-material">S8 Fig</a>). Similar to the wild-type MMP-9 (pro-MMP-9_Cat-H6), pro-MMP-9_Cat_E402Q-H6 was successfully processed by MMP-3 (<a href="#pone.0244656.s001" rid="pone.0244656.s001" class=" supplementary-material">S1B Fig</a>). Because the pro-peptide-removed E402Q mutant (MMP-9_Cat_E402Q-H6) does not show any proteolytic activity [<a href="#pone.0244656.ref039" rid="pone.0244656.ref039" class=" bibr popnode">39</a>], the interaction between <a id="__tag_904612491" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> and Glu-402 was confirmed by not the enzymatic assay but the SEC binding assay. By pre-incubation with TIMP-1 or <a id="__tag_904612473" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a>, the retention time of the peak of active MMP-9_Cat-H6 was shifted from 7.0 min to 6.4 or 6.7 min, respectively (<a href="/pmc/articles/PMC7771667/figure/pone.0244656.g004/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g004" rid-ob="ob-pone.0244656.g004" co-legend-rid="lgnd_pone.0244656.g004"><span>Fig 4A</span></a>). These findings showed that TIMP-1 and <a id="__tag_904612492" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> bound to active MMP-9_Cat-H6. In the case of MMP-9_Cat_E402Q-H6, the retention time of the peak was shifted from 7.1 min to 6.5 min by pre-incubation with TIMP-1, but not shifted by pre-incubation with <a id="__tag_904612483" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> (<a href="/pmc/articles/PMC7771667/figure/pone.0244656.g004/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g004" rid-ob="ob-pone.0244656.g004" co-legend-rid="lgnd_pone.0244656.g004"><span>Fig 4B</span></a>). <a id="__tag_904612486" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> interacted with active MMP-9_Cat-H6 but not with MMP-9_Cat_E402Q-H6, suggesting that <a id="__tag_904612449" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> directly recognized the catalytic Glu-402 residue of MMP-9.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="pone.0244656.g003" co-legend-rid="lgnd_pone.0244656.g003"><a href="/pmc/articles/PMC7771667/figure/pone.0244656.g003/" target="figure" rid-figpopup="pone.0244656.g003" rid-ob="ob-pone.0244656.g003"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140659061792384"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7771667_pone.0244656.g003.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pone.0244656.g003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC7771667/bin/pone.0244656.g003.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140659061792384"><a target="object" rel="noopener" href="/pmc/articles/PMC7771667/figure/pone.0244656.g003/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_pone.0244656.g003"><div><a class="figpopup" href="/pmc/articles/PMC7771667/figure/pone.0244656.g003/" target="figure" rid-figpopup="pone.0244656.g003" rid-ob="ob-pone.0244656.g003">Fig 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Design of MMP-9 mutants for analysis of interaction between MMP-9 and <a id="__tag_904612444" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a>.</strong><p>(Top) The sequence alignment between MMP-9 and MMP-2 in the active-site cleft and exosite of MMP-9. The residues are numbered according to the generic MMP-9 nomenclature. Catalytic Glu-402 is indicated by an <em>asterisk</em> and shown in <em>blue</em>. Three histidine residues (His-401, His-405, and His-411) that chelate catalytic zinc ion are indicated by a &#x0201c;Z&#x0201d; and shown with a <em>green</em> background. Alanine substitution sites in the active-site cleft (for cleft mutants) and MMP-9/-2 chimeric mutation sites in the exosite (for exosite mutants) are highlighted in <em>magenta</em> and <em>cyan</em>, respectively. Three residues (Ala-189, His-190, and Ala-191) in the active-site cleft, the side chain of which is not exposed to solvent, are shown with an <em>orange</em> background. (Bottom) The structure of the catalytic domain of active MMP-9, colored in <em>gray</em> (Protein Data Bank code 4H3X). Each residue is shown with <em>coloring</em> as in the <em>top panel</em>.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="pone.0244656.g004" co-legend-rid="lgnd_pone.0244656.g004"><a href="/pmc/articles/PMC7771667/figure/pone.0244656.g004/" target="figure" rid-figpopup="pone.0244656.g004" rid-ob="ob-pone.0244656.g004"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140659061784928"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7771667_pone.0244656.g004.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pone.0244656.g004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC7771667/bin/pone.0244656.g004.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140659061784928"><a target="object" rel="noopener" href="/pmc/articles/PMC7771667/figure/pone.0244656.g004/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_pone.0244656.g004"><div><a class="figpopup" href="/pmc/articles/PMC7771667/figure/pone.0244656.g004/" target="figure" rid-figpopup="pone.0244656.g004" rid-ob="ob-pone.0244656.g004">Fig 4</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong><a id="__tag_904612472" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> binds to the active site of MMP-9.</strong><p>The binding activity to the catalytic Glu-402 residue of MMP-9 was evaluated by SEC. Active MMP-9_Cat-H6 or MMP-9_Cat-E402Q-H6 (25 &#x003bc;M each) was incubated with TIMP-1 or <a id="__tag_904612470" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> (75 &#x003bc;M each) in PBS for 1 h at 4&#x000b0;C, and then 10 &#x003bc;l of the reaction mixture was analyzed by monitoring the absorbance at 280 nm in SEC. Chromatograms are shown in panels as follows: active MMP-9_Cat-H6 (A), MMP-9_Cat_E402Q-H6 (B), and no MMP-9 (C). <em>Dotted lines</em> indicate the retention time (RT) of active MMP-9_Cat-H6 (7.0 min, A) and MMP-9_Cat_E402Q-H6 (7.1 min, B). Molecular weights (158, 44, and 17 kDa) of the gel filtration standard are indicated at the top of each panel. The RT of active MMP-9_Cat-H6 but not MMP-9_Cat-E402Q-H6 was shifted to a high molecular weight by incubation with <a id="__tag_904612510" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a>. <em>mAU</em>, milliabsorbance units.</p></div></div></div><p class="p">Next, to explore the interacting residues in the region surrounding the active site, the so-called active-site cleft, we designed MMP-9 mutants with mutations in the active-site cleft and carried out enzymatic assays using them. To obtain a comprehensive understanding of the interacting residues, we selected 13 residues (Phe-110, Tyr-179, Asp-185, Gly-186, Leu-187, Leu-188, Phe-192, Tyr-393, Leu-397, Val-398, Leu-418, Pro-421, and Tyr-423; <a href="/pmc/articles/PMC7771667/figure/pone.0244656.g003/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g003" rid-ob="ob-pone.0244656.g003" co-legend-rid="lgnd_pone.0244656.g003"><span>Fig 3</span></a>
<em>magenta</em>) that met the following criteria: (i) involved in interaction with peptide substrate based on the structural information of the complex of MMP-9 and peptide substrate reported previously (Protein Data Bank code 4JIJ), (ii) identical to those of MMP-2, and (iii) having a side chain exposed to the solvent. These residues were mutated to alanine as the cleft mutants. The N-terminal residue of the cleft mutants was designed to be Phe to mimic the N-terminus after digestion by MMP-3 [<a href="#pone.0244656.ref006" rid="pone.0244656.ref006" class=" bibr popnode">6</a>]. For convenient preparation of these mutants, an enterokinase (EK) recognition site was inserted between the pro-peptide and the N-terminus of the catalytic domain (Phe-107), resulting in the mutants being uniformly processed and highly purified (<a href="#pone.0244656.s005" rid="pone.0244656.s005" class=" supplementary-material">S5A and S5B Fig</a>). Except for the L188A mutant, all of the cleft mutants with sufficient proteolytic activity could be prepared (<a href="#pone.0244656.s005" rid="pone.0244656.s005" class=" supplementary-material">S5C Fig</a>). In enzymatic assays using the peptide substrate, F110A, Y179A, L187A, F192A, Y393A, and Y423A mutants significantly weakened the inhibitory activities of <a id="__tag_904612494" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> compared with that of wild-type MMP-9 (<a href="/pmc/articles/PMC7771667/figure/pone.0244656.g005/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g005" rid-ob="ob-pone.0244656.g005" co-legend-rid="lgnd_pone.0244656.g005"><span>Fig 5A</span></a> and <a href="/pmc/articles/PMC7771667/table/pone.0244656.t002/" target="table" class="fig-table-link figpopup" rid-figpopup="pone.0244656.t002" rid-ob="ob-pone.0244656.t002" co-legend-rid=""><span>Table 2</span></a>). These results suggested that <a id="__tag_904612502" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> also interacted with Phe-110, Tyr-179, Leu-187, Phe-192, Tyr-393, and Tyr-423 residues, located in the active-site cleft of MMP-9.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="pone.0244656.g005" co-legend-rid="lgnd_pone.0244656.g005"><a href="/pmc/articles/PMC7771667/figure/pone.0244656.g005/" target="figure" rid-figpopup="pone.0244656.g005" rid-ob="ob-pone.0244656.g005"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140659099624784"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7771667_pone.0244656.g005.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pone.0244656.g005.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC7771667/bin/pone.0244656.g005.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140659099624784"><a target="object" rel="noopener" href="/pmc/articles/PMC7771667/figure/pone.0244656.g005/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_pone.0244656.g005"><div><a class="figpopup" href="/pmc/articles/PMC7771667/figure/pone.0244656.g005/" target="figure" rid-figpopup="pone.0244656.g005" rid-ob="ob-pone.0244656.g005">Fig 5</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong><a id="__tag_904612464" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> recognizes residues in the active-site cleft and the exosite of MMP-9.</strong><p>MMP-9 inhibitory activities of <a id="__tag_904612461" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> towards the cleft mutants (A) and the exosite mutants (B) were measured by enzymatic assay using peptide substrate. Active MMP-9_Cat (WT) or the MMP-9 mutants activated by EK (1 nM each) were incubated with various concentrations of <a id="__tag_904612489" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> (0&#x02013;330 nM or 0&#x02013;1,000 nM) for 1 h at 37&#x000b0;C. Enzymatic activities were determined by monitoring the degradation of peptide substrate 3226-v (10 &#x003bc;M). Each enzymatic activity was normalized to the activity of the MMP-9 mutant without <a id="__tag_904612500" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a>. Data are shown as the mean &#x000b1; S.D. (n = 3). All curves were obtained by non-linear curve fitting.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="pone.0244656.t002"><h3>Table 2</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Inhibitory activity of <a id="__tag_904612512" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> against cleft mutants.</strong></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="center" rowspan="1" colspan="1">MMP-9</th><th align="center" rowspan="1" colspan="1">IC<sub>50</sub> (nM)<sup>a</sup></th><th align="center" rowspan="1" colspan="1">R<sup>2</sup></th><th align="center" rowspan="1" colspan="1"><em>P</em> value<sup>b</sup></th></tr></thead><tbody><tr><td align="center" rowspan="1" colspan="1">WT</td><td align="center" rowspan="1" colspan="1">17 &#x000b1; 7</td><td align="center" rowspan="1" colspan="1">0.965</td><td align="center" rowspan="1" colspan="1">-</td></tr><tr><td align="center" rowspan="1" colspan="1">F110A</td><td align="center" rowspan="1" colspan="1">69 &#x000b1; 12</td><td align="center" rowspan="1" colspan="1">0.845</td><td align="center" rowspan="1" colspan="1">&#x0003c;0.001</td></tr><tr><td align="center" rowspan="1" colspan="1">Y179A</td><td align="center" rowspan="1" colspan="1">55 &#x000b1; 13</td><td align="center" rowspan="1" colspan="1">0.989</td><td align="center" rowspan="1" colspan="1">&#x0003c;0.001</td></tr><tr><td align="center" rowspan="1" colspan="1">D185A</td><td align="center" rowspan="1" colspan="1">29 &#x000b1; 5</td><td align="center" rowspan="1" colspan="1">0.990</td><td align="center" rowspan="1" colspan="1">0.69</td></tr><tr><td align="center" rowspan="1" colspan="1">G186A</td><td align="center" rowspan="1" colspan="1">15 &#x000b1; 0</td><td align="center" rowspan="1" colspan="1">0.999</td><td align="center" rowspan="1" colspan="1">0.9997</td></tr><tr><td align="center" rowspan="1" colspan="1">L187A</td><td align="center" rowspan="1" colspan="1">120 &#x000b1; 10</td><td align="center" rowspan="1" colspan="1">0.991</td><td align="center" rowspan="1" colspan="1">&#x0003c;0.001</td></tr><tr><td align="center" rowspan="1" colspan="1">F192A</td><td align="center" rowspan="1" colspan="1">&#x002c3; 1,000</td><td align="center" rowspan="1" colspan="1">0.926</td><td align="center" rowspan="1" colspan="1">&#x0003c;0.001</td></tr><tr><td align="center" rowspan="1" colspan="1">Y393A</td><td align="center" rowspan="1" colspan="1">48 &#x000b1; 14</td><td align="center" rowspan="1" colspan="1">0.985</td><td align="center" rowspan="1" colspan="1">0.0085</td></tr><tr><td align="center" rowspan="1" colspan="1">L397A</td><td align="center" rowspan="1" colspan="1">37 &#x000b1; 10</td><td align="center" rowspan="1" colspan="1">0.986</td><td align="center" rowspan="1" colspan="1">0.15</td></tr><tr><td align="center" rowspan="1" colspan="1">V398A</td><td align="center" rowspan="1" colspan="1">39 &#x000b1; 3</td><td align="center" rowspan="1" colspan="1">0.999</td><td align="center" rowspan="1" colspan="1">0.10</td></tr><tr><td align="center" rowspan="1" colspan="1">L418A</td><td align="center" rowspan="1" colspan="1">9.3 &#x000b1; 2.9</td><td align="center" rowspan="1" colspan="1">0.988</td><td align="center" rowspan="1" colspan="1">0.96</td></tr><tr><td align="center" rowspan="1" colspan="1">P421A</td><td align="center" rowspan="1" colspan="1">23 &#x000b1; 9</td><td align="center" rowspan="1" colspan="1">0.961</td><td align="center" rowspan="1" colspan="1">0.99</td></tr><tr><td align="center" rowspan="1" colspan="1">Y423A</td><td align="center" rowspan="1" colspan="1">230 &#x000b1; 20</td><td align="center" rowspan="1" colspan="1">0.980</td><td align="center" rowspan="1" colspan="1">&#x0003c;0.001</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140659099619568"><a target="object" rel="noopener" href="/pmc/articles/PMC7771667/table/pone.0244656.t002/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t002fn001"><p class="p p-first-last"><sup><em>a</em></sup>IC<sub>50</sub> was determined by the MMP-9 inhibitory assay with 10 &#x003bc;M peptide substrate 3226-v. The IC<sub>50</sub> values are shown as the mean &#x000b1; S.D. (n = 3).</p></div><div id="t002fn002"><p class="p p-first-last"><sup><em>b</em></sup>Statistical analysis of cleft mutants <em>versus</em> WT was performed by one-way ANOVA with Dunnett&#x02019;s post tests for multiple comparisons.</p></div></div></div></div><div id="sec007" class="sec sec-last"><h3 id="sec007title"><a id="__tag_904612455" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> recognizes the exosite in the catalytic domain of MMP-9</h3><p class="p p-first">Considering the high specificity of <a id="__tag_904612511" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> despite the high homology in amino acid sequences and structures of MMPs, <a id="__tag_904612445" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> should interact not only with the active site and the active-site cleft but also with an additional region in the catalytic domain that differentiates MMP-9 from the other MMPs. <a id="__tag_904612457" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> inhibited MMP-9 but not MMP-2, suggesting that <a id="__tag_904612490" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> recognizes structural differences between these two members of the MMP family in the form of an exosite. To investigate this hypothesis, we designed MMP-9/-2 chimeric mutants as the exosite mutants. Ten residues (Gln-108, Thr-109, Glu-111, Pro-193, Ile-198, Gln-199, Asp-410, Ser-413, Tyr-420, and Met-422; <a href="/pmc/articles/PMC7771667/figure/pone.0244656.g003/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g003" rid-ob="ob-pone.0244656.g003" co-legend-rid="lgnd_pone.0244656.g003"><span>Fig 3</span></a>
<em>cyan</em>) were selected based on the following criteria: (i) being near the active-site cleft and (ii) not being identical to that of MMP-2. These residues were mutated to the corresponding amino acid of MMP-2. All exosite mutants were well expressed in HEK293F cells, and then purified pro-form proteins were efficiently activated by using EK by the same method as for the cleft mutants (<a href="#pone.0244656.s005" rid="pone.0244656.s005" class=" supplementary-material">S5 Fig</a>). In enzymatic assays using the peptide substrate, Q199G, Y420A, and M422I mutants significantly weakened the inhibitory activities of <a id="__tag_904612495" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> compared with that of wild-type MMP-9 (<a href="/pmc/articles/PMC7771667/figure/pone.0244656.g005/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g005" rid-ob="ob-pone.0244656.g005" co-legend-rid="lgnd_pone.0244656.g005"><span>Fig 5B</span></a> and <a href="/pmc/articles/PMC7771667/table/pone.0244656.t003/" target="table" class="fig-table-link figpopup" rid-figpopup="pone.0244656.t003" rid-ob="ob-pone.0244656.t003" co-legend-rid=""><span>Table 3</span></a>). These results suggested that <a id="__tag_904612456" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> also interacted with Gln-199, Tyr-420, and Met-422, which were located away from the active site of MMP-9. Collectively, catalytic Glu-402 (<a href="/pmc/articles/PMC7771667/figure/pone.0244656.g006/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g006" rid-ob="ob-pone.0244656.g006" co-legend-rid="lgnd_pone.0244656.g006"><span>Fig 6</span></a>
<em>blue</em>), Phe-110, Tyr-179, Leu-187, Phe-192, Gln-199, Tyr-393, Tyr-420, Met-422, and Tyr-423 (<a href="/pmc/articles/PMC7771667/figure/pone.0244656.g006/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g006" rid-ob="ob-pone.0244656.g006" co-legend-rid="lgnd_pone.0244656.g006"><span>Fig 6</span></a>
<em>magenta</em>) were suggested to be interacted with <a id="__tag_904612448" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="pone.0244656.g006" co-legend-rid="lgnd_pone.0244656.g006"><a href="/pmc/articles/PMC7771667/figure/pone.0244656.g006/" target="figure" rid-figpopup="pone.0244656.g006" rid-ob="ob-pone.0244656.g006"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140659103261856"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7771667_pone.0244656.g006.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pone.0244656.g006.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC7771667/bin/pone.0244656.g006.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140659103261856"><a target="object" rel="noopener" href="/pmc/articles/PMC7771667/figure/pone.0244656.g006/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_pone.0244656.g006"><div><a class="figpopup" href="/pmc/articles/PMC7771667/figure/pone.0244656.g006/" target="figure" rid-figpopup="pone.0244656.g006" rid-ob="ob-pone.0244656.g006">Fig 6</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Mapping of the possible interaction sites with <a id="__tag_904612477" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> onto the structure of active MMP-9.</strong><p>In accordance with the results of binding and enzymatic assay using MMP-9 mutants, the residues identified as being involved in the interaction with <a id="__tag_904612463" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> were mapped onto the structure of the catalytic domain of active MMP-9 (<em>gray</em>, Protein Data Bank code 4H3X). The results of the binding assays (<a href="/pmc/articles/PMC7771667/figure/pone.0244656.g004/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g004" rid-ob="ob-pone.0244656.g004" co-legend-rid="lgnd_pone.0244656.g004"><span>Fig 4</span></a>) suggested that catalytic Glu-402 (<em>blue</em>) contributed to the interaction between <a id="__tag_904612505" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> and MMP-9. The results of enzyme inhibitory assays (<a href="/pmc/articles/PMC7771667/figure/pone.0244656.g005/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g005" rid-ob="ob-pone.0244656.g005" co-legend-rid="lgnd_pone.0244656.g005"><span>Fig 5</span></a>, Tables <a href="/pmc/articles/PMC7771667/table/pone.0244656.t002/" target="table" class="fig-table-link figpopup" rid-figpopup="pone.0244656.t002" rid-ob="ob-pone.0244656.t002" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Tables2</span></span><span>2</span></a> and <a href="/pmc/articles/PMC7771667/table/pone.0244656.t003/" target="table" class="fig-table-link figpopup" rid-figpopup="pone.0244656.t003" rid-ob="ob-pone.0244656.t003" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and3)</span></span><span>3</span></a>) suggested that <a id="__tag_904612465" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> recognized Phe-110, Tyr-179, Leu-187, Phe-192, Gln-199, Tyr-393, Tyr-420, Met-422, and Tyr-423 (<em>magenta</em>). The interaction with characteristic residues, Gln-199 and Met-422 (<em>black dotted circle</em>), is likely to contribute to the specific inhibition of <a id="__tag_904612482" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> toward MMP-9.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="pone.0244656.t003"><h3>Table 3</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Inhibitory activity of <a id="__tag_904612504" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> against exosite mutants.</strong></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="center" rowspan="1" colspan="1">MMP-9</th><th align="center" rowspan="1" colspan="1">IC<sub>50</sub> (nM)<sup>a</sup></th><th align="center" rowspan="1" colspan="1">R<sup>2</sup></th><th align="center" rowspan="1" colspan="1"><em>P</em> value<sup>b</sup></th></tr></thead><tbody><tr><td align="center" rowspan="1" colspan="1">WT</td><td align="center" rowspan="1" colspan="1">17 &#x000b1; 7</td><td align="center" rowspan="1" colspan="1">0.965</td><td align="center" rowspan="1" colspan="1">-</td></tr><tr><td align="center" rowspan="1" colspan="1">Q108N</td><td align="center" rowspan="1" colspan="1">20 &#x000b1; 6</td><td align="center" rowspan="1" colspan="1">0.981</td><td align="center" rowspan="1" colspan="1">0.9995</td></tr><tr><td align="center" rowspan="1" colspan="1">T109F</td><td align="center" rowspan="1" colspan="1">15 &#x000b1; 4</td><td align="center" rowspan="1" colspan="1">0.977</td><td align="center" rowspan="1" colspan="1">0.9995</td></tr><tr><td align="center" rowspan="1" colspan="1">E111P</td><td align="center" rowspan="1" colspan="1">31 &#x000b1; 9</td><td align="center" rowspan="1" colspan="1">0.983</td><td align="center" rowspan="1" colspan="1">0.50</td></tr><tr><td align="center" rowspan="1" colspan="1">P193A</td><td align="center" rowspan="1" colspan="1">13 &#x000b1; 3</td><td align="center" rowspan="1" colspan="1">0.992</td><td align="center" rowspan="1" colspan="1">0.999</td></tr><tr><td align="center" rowspan="1" colspan="1">I198V</td><td align="center" rowspan="1" colspan="1">16 &#x000b1; 4</td><td align="center" rowspan="1" colspan="1">0.989</td><td align="center" rowspan="1" colspan="1">0.9998</td></tr><tr><td align="center" rowspan="1" colspan="1">Q199G</td><td align="center" rowspan="1" colspan="1">40 &#x000b1; 16</td><td align="center" rowspan="1" colspan="1">0.967</td><td align="center" rowspan="1" colspan="1">0.043</td></tr><tr><td align="center" rowspan="1" colspan="1">D410E</td><td align="center" rowspan="1" colspan="1">22 &#x000b1; 9</td><td align="center" rowspan="1" colspan="1">0.975</td><td align="center" rowspan="1" colspan="1">0.999</td></tr><tr><td align="center" rowspan="1" colspan="1">S413Q</td><td align="center" rowspan="1" colspan="1">30 &#x000b1; 14</td><td align="center" rowspan="1" colspan="1">0.949</td><td align="center" rowspan="1" colspan="1">0.50</td></tr><tr><td align="center" rowspan="1" colspan="1">Y420A</td><td align="center" rowspan="1" colspan="1">54 &#x000b1; 8</td><td align="center" rowspan="1" colspan="1">0.995</td><td align="center" rowspan="1" colspan="1">&#x0003c;0.001</td></tr><tr><td align="center" rowspan="1" colspan="1">M422I</td><td align="center" rowspan="1" colspan="1">45 &#x000b1; 18</td><td align="center" rowspan="1" colspan="1">0.959</td><td align="center" rowspan="1" colspan="1">&#x0003c;0.001</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140659103256672"><a target="object" rel="noopener" href="/pmc/articles/PMC7771667/table/pone.0244656.t003/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t003fn001"><p class="p p-first-last"><sup><em>a</em></sup>IC<sub>50</sub> was determined by the MMP-9 inhibitory assay with 10 &#x003bc;M peptide substrate 3226-v. The IC<sub>50</sub> values are shown as the mean &#x000b1; S.D. (n = 3).</p></div><div id="t003fn002"><p class="p p-first-last"><sup><em>b</em></sup>Statistical analysis of exosite mutants <em>versus</em> WT was performed by one-way ANOVA with Dunnett&#x02019;s post tests for multiple comparisons.</p></div></div></div></div></div><div id="sec008" class="tsec sec"><h2 class="head no_bottom_margin" id="sec008title">Discussion</h2><p class="p p-first">In this study, we succeeded in obtaining MMP-9 inhibitors with high potency and specificity from our proprietary engineered SPINK2 phage display library without any affinity maturation. The potency and specificity of <a id="__tag_904612474" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> were comparable to those of the MMP-9 inhibitory antibody CALY-001 [<a href="#pone.0244656.ref040" rid="pone.0244656.ref040" class=" bibr popnode">40</a>]. CALY-001 did not bind to pro-MMP-9; however, it showed non-competitive inhibition against MMP-9, suggesting that it is not an active-site inhibitor. On the other hand, <a id="__tag_904612467" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> did not interact with the active-site mutant MMP-9_Cat_E402Q (<a href="/pmc/articles/PMC7771667/figure/pone.0244656.g004/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g004" rid-ob="ob-pone.0244656.g004" co-legend-rid="lgnd_pone.0244656.g004"><span>Fig 4B</span></a>), suggesting that it may recognize the active site. The result that <a id="__tag_904612509" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> bound to active MMP-9 and not to pro-MMP-9 (<a href="/pmc/articles/PMC7771667/figure/pone.0244656.g001/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g001" rid-ob="ob-pone.0244656.g001" co-legend-rid="lgnd_pone.0244656.g001"><span>Fig 1B</span></a>) supports the view that <a id="__tag_904612485" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> recognizes the active site. Taking these findings together, the recognition mechanism of <a id="__tag_904612506" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> differs from that of CALY-001. To our knowledge, <a id="__tag_904612466" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> is the first MMP-9-specific inhibitor, based on protein, to recognize the active site.</p><p>It was strongly suggested that <a id="__tag_904612450" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> interacted with the active site; however, it remained unclear why <a id="__tag_904612479" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> achieved specific inhibition toward MMP-9 despite the active sites being highly conserved among MMP family members. To understand the mechanism behind this high specificity, we analyzed the binding sites by assays using several MMP-9 mutants having alanine substitution in the active-site cleft (cleft mutants) and MMP-9/-2 chimeric mutation in the exosite (exosite mutants). For the enzymatic assay using MMP-9 mutants, it is critically important that they have sufficient purity and enzymatic activity. To create activated MMP-9 mutants with high stability, we found that autolysis of activated MMP-9, which occurred upon long-term incubation at 25&#x000b0;C (<a href="#pone.0244656.s010" rid="pone.0244656.s010" class=" supplementary-material">S10 Fig</a>), was prevented by activation using EK at 4&#x000b0;C. In most of the mutants, despite their good purity and processing by EK, the cleavage activities of peptide substrates were reduced compared with that of the wild-type (<a href="#pone.0244656.s005" rid="pone.0244656.s005" class=" supplementary-material">S5C Fig</a>); however, in our preliminary experiment, V398A, L418A, and P421A mutants significantly affected the inhibitory activities of sc-311438 compared with that of wild-type MMP-9 (<a href="#pone.0244656.s006" rid="pone.0244656.s006" class=" supplementary-material">S6A Fig</a>), consistent with the previous report describing that sc-311438 interacts with the S1&#x02032; pocket of MMPs [<a href="#pone.0244656.ref041" rid="pone.0244656.ref041" class=" bibr popnode">41</a>], which was formed by residues including Val-398, Leu-418, and Pro-421 in MMP-9. Therefore, we concluded that enzymatic assays using these mutants are available to explore the residues of MMP-9 interacting with the inhibitors.</p><p>The results using the cleft mutants (<a href="/pmc/articles/PMC7771667/figure/pone.0244656.g005/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g005" rid-ob="ob-pone.0244656.g005" co-legend-rid="lgnd_pone.0244656.g005"><span>Fig 5A</span></a> and <a href="/pmc/articles/PMC7771667/table/pone.0244656.t002/" target="table" class="fig-table-link figpopup" rid-figpopup="pone.0244656.t002" rid-ob="ob-pone.0244656.t002" co-legend-rid=""><span>Table 2</span></a>) strongly suggested that <a id="__tag_904612446" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> recognized Phe-192 and Tyr-423. As they are conserved in MMP-1, -2, -8, -9, and -13 (<a href="#pone.0244656.s004" rid="pone.0244656.s004" class=" supplementary-material">S4 Fig</a>), additional recognition sites are required to achieve the specific inhibition of MMP-9. For further analysis, we selected MMP-2, which has the highest homology to MMP-9 among MMP family proteins, and performed enzymatic assays using exosite mutants. The results (<a href="/pmc/articles/PMC7771667/figure/pone.0244656.g005/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g005" rid-ob="ob-pone.0244656.g005" co-legend-rid="lgnd_pone.0244656.g005"><span>Fig 5B</span></a> and <a href="/pmc/articles/PMC7771667/table/pone.0244656.t003/" target="table" class="fig-table-link figpopup" rid-figpopup="pone.0244656.t003" rid-ob="ob-pone.0244656.t003" co-legend-rid=""><span>Table 3</span></a>) suggested that <a id="__tag_904612452" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> recognized Gln-199, Tyr-420, and Met-422, which are characteristic residues of MMP-9 compared with the other MMPs. In terms of their location, Gln-199 is near Phe-192 and Met-422 is near Tyr-423 (<a href="/pmc/articles/PMC7771667/figure/pone.0244656.g006/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g006" rid-ob="ob-pone.0244656.g006" co-legend-rid="lgnd_pone.0244656.g006"><span>Fig 6</span></a>), consistent with the suggestion that <a id="__tag_904612493" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> recognizes both Phe-192 and Tyr-423. Interestingly, both Gln-199 and Met-422 are unique sequences that are found only in MMP-9 among the five MMP family proteins (<a href="#pone.0244656.s004" rid="pone.0244656.s004" class=" supplementary-material">S4 Fig</a>). It was suggested that <a id="__tag_904612507" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> recognizes not only highly homologous Phe-192 and Tyr-423 but also unique Gln-199 and Met-422 located near these residues. Therefore, it was in turn suggested that <a id="__tag_904612476" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> specifically inhibits MMP-9 by recognizing both Gln-199 and Met-422 in the exosite (<a href="/pmc/articles/PMC7771667/figure/pone.0244656.g006/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g006" rid-ob="ob-pone.0244656.g006" co-legend-rid="lgnd_pone.0244656.g006"><span>Fig 6</span></a>
<em>black dotted circle</em>).</p><p>Gln-199 and Met-422, which are distant from each other, are located on the non-prime side and prime side of the catalytic domain of MMP-9, respectively. The side chain of Gln-199 is thought to be involved in forming the S4 subsite of MMP-9 [<a href="#pone.0244656.ref042" rid="pone.0244656.ref042" class=" bibr popnode">42</a>,<a href="#pone.0244656.ref043" rid="pone.0244656.ref043" class=" bibr popnode">43</a>]. Met-422 forms a groove recognizing the region from the P2&#x02032; to the P4&#x02032; position of the substrate [<a href="#pone.0244656.ref044" rid="pone.0244656.ref044" class=" bibr popnode">44</a>]. Thus, it was suggested that <a id="__tag_904612498" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> interacts with the region including S4 and the S4&#x02032; subsite as well as the active site of MMP-9. Based on the structural information of MMP-9, the distance from C&#x003b1; of Gln-199 to that of Met-422 is over 20 &#x000c5;. This is a long distance comparable to the large interface between KLK4 and the engineered SPINK2-derived KLK4 inhibitor, K41043, which we previously demonstrated [<a href="#pone.0244656.ref024" rid="pone.0244656.ref024" class=" bibr popnode">24</a>]. K41043 recognized the large surface area containing both the S1 pocket of KLK4 and the region surrounding the pocket, and showed high specificity. Similar to K41043, the large interaction area generated by the randomized loop presented by the engineered SPINK2 may have allowed <a id="__tag_904612488" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> to recognize both active site and exosite of MMP-9. Nam <em>et al</em>. reported that the engineered antibody Fab 3A2, which accessed the active site of target MMP-14 and showed high specificity, recognized the Phe-260 residue, which forms the S1&#x02032; subsite of MMP-14 [<a href="#pone.0244656.ref045" rid="pone.0244656.ref045" class=" bibr popnode">45</a>]. Phe-260 of MMP-14 corresponds to Met-422 of MMP-9 in the sequence alignment and is a characteristic residue in the exosite of MMP-14. The importance of exosite recognition for high specificity has also been suggested in another report, supporting our finding that the binding of <a id="__tag_904612469" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> to the exosite contributed to its specificity to MMP-9 in this study.</p><p>We speculated that <a id="__tag_904612462" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> interacted with Gln-199 and Met-422 of MMP-9 by enzymatic assays using the MMP-9 mutants. ELISA or enzymatic assays using mutant enzymes provided some insights into the interacting residues of the inhibitors in previous reports [<a href="#pone.0244656.ref045" rid="pone.0244656.ref045" class=" bibr popnode">45</a>&#x02013;<a href="#pone.0244656.ref048" rid="pone.0244656.ref048" class=" bibr popnode">48</a>]. Similarly, our results using the MMP-9 mutants contributed to providing meaningful information regarding the mechanism by which <a id="__tag_904612454" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> recognizes MMP-9 in this research. Future investigation including X-ray crystallographic analysis of the complex of <a id="__tag_904612459" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> and MMP-9 may offer additional insights into the mechanism behind the potent and specific inhibition of MMP-9 by <a id="__tag_904612460" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a>.</p><p class="p p-last">Using the engineered SPINK2 library, we obtained MMP-9-specific inhibitors with high potency, and they recognized exosites distant from each other in addition to the active site, which could be achieved by the large interaction area provided by the loop of the SPINK2 scaffold. The characteristic recognition mechanism shown by engineered SPINK2-derived inhibitors may also be useful in the creation of specific active-site inhibitors of other MMPs, and the engineered SPINK2 library may contribute to the discovery of specific inhibitors useful for basic and clinical research on MMPs.</p></div><div id="sec009" class="tsec sec"><h2 class="head no_bottom_margin" id="sec009title">Materials and methods</h2><div id="sec010" class="sec sec-first"><h3 id="sec010title">Reagents</h3><p class="p p-first">Fluorescence-quenching peptide substrates, MOCAc-Arg-Pro-Lys-Pro-Val-Glu-Nva-Trp-Arg-Lys(Dnp)-NH<sub>2</sub> (3168-v, Lot No. 610613) and MOCAc-Lys-Pro-Leu-Gly-Leu-A<sub>2</sub>pr(Dnp)-Ala-Arg-NH<sub>2</sub> (3226-v, Lot No. 650414), were purchased from Peptide Institute. Dye-quenched (DQ) gelatin (D-12054, Lot No. 1694715) was obtained from Thermo Fisher Scientific. The broad MMP inhibitor <em>N</em>-hydroxy-1-(4-methoxyphenyl)sulfonyl-4-(4-biphenylcarbonyl)piperazine-2-carboxamide (sc-311438) was obtained from Santa Cruz Biotechnology.</p><p class="p p-last">The full-length active hMMP-2 and active hMMP-15 catalytic domain were purchased from EMD Millipore. The catalytic domains of active hMMP-7, active hMMP-10, and active hMMP-12 were obtained from Enzo Life Sciences. Active hMMP-14 catalytic domain was purchased from Thermo Fisher Scientific. The catalytic domains of pro-hMMP-16 and pro-hMMP-17 were purchased from R&#x00026;D Systems. Pro-MMP-16 was activated by Furin, and pro-MMP-17 was activated by 4-aminophenylmercuric acetate (APMA; Sigma-Aldrich), in accordance with each manufacturer&#x02019;s protocol.</p></div><div id="sec011" class="sec"><h3 id="sec011title">Protein expression and purification</h3><p class="p p-first">All pro-MMPs with the native leader sequences were cloned into pcDNA3.3 vector (Thermo Fisher Scientific) and their constructs were as follows. Each catalytic domain of pro-hMMP-1 (residues 20&#x02013;261), pro-hMMP-8 (residues 21&#x02013;262), or pro-hMMP-13 (residues 20&#x02013;267) was fused to the N-terminal His<sub>6</sub> tag. Pro-hMMP-9 catalytic domain with an insertion of fibronectin-like domain (residues 20&#x02013;449; pro-MMP-9_Cat) was fused to the C-terminal His<sub>6</sub> tag (pro-MMP-9_Cat-H6). For preparation of biotinylated MMP-9 protein, pro-MMP-9_Cat was fused to the N-terminal His<sub>6</sub> tag, C-terminal FLAG tag, and biotin acceptor peptide (pro-MMP-9_Cat-BAP). For activation of pro-MMP-9 using enterokinase (EK), pro-MMP-9_Cat was fused to the N-terminal His<sub>6</sub> tag, followed by insertion of the EK cleavage site (amino acid sequence: DDDDK) between Arg-106 and Phe-107 (pro-EK-MMP-9_Cat). Pro-MMP-9_Cat cleft mutants (pro-EK-MMP-9_Cat_F110A, Y179A, D185A, G186A, L187A, L188A, F192A, Y393A, L397A, V398A, L418A, P421A, and Y423A) and pro-MMP-9/MMP-2 chimeric mutants (pro-EK-MMP-9_Cat_Q108N, T109F, E111P, P193A, I198V, Q199G, D410E, S413Q, Y420A, and M422I) were constructed from pro-EK-MMP-9_Cat by single-amino-acid substitution using KOD -Plus- Mutagenesis Kit (TOYOBO). Pro-MMP-9 E402Q mutant (pro-MMP-9_Cat_E402Q-H6) was generated from pro-MMP-9_Cat-H6 using QuikChange Site-Directed Mutagenesis Kit (Agilent).</p><p>All pro-MMPs were transiently expressed in HEK293F cells (Thermo Fisher Scientific). Six days after transfection, all pro-MMP proteins were purified using HisTrap excel gel (GE Healthcare), followed by buffer exchange into PBS. Pro-EK-MMP-9_Cat and mutants were further purified using gelatin-Sepharose resin (GE Healthcare), followed by buffer exchange into PBS. Pro-MMP-9_Cat-BAP was biotinylated using biotin-protein ligase BirA (Avidity), in accordance with the manufacturer&#x02019;s protocol.</p><p class="p p-last">Human TIMP-1 (residues 1&#x02013;207) with a C-terminal FLAG tag was cloned into the pcDNA3.3 vector. TIMP-1 was transiently expressed in HEK293F cells and purified using ANTI-FLAG M2 Affinity Gel (Sigma-Aldrich), followed by buffer exchange into PBS.</p></div><div id="sec012" class="sec"><h3 id="sec012title">Activation of MMP</h3><p class="p p-first">MMP-1, -3, -8, and -13 were activated using APMA in TNC buffer (50 mM Tris-HCl, 200 mM NaCl, 2 mM CaCl<sub>2</sub>, pH 7.5) at 37&#x000b0;C as follows. Active forms of MMP-1 and MMP-8 were generated by incubation of pro-MMP-1 (15.8 &#x003bc;M) or pro-MMP-8 (15.9 &#x003bc;M) with 500 &#x003bc;M APMA for 1 h. Active MMP-3 was generated by incubation of pro-MMP-3 (9.4 &#x003bc;M) with 500 &#x003bc;M APMA for 4 h. Active MMP-13 was generated by incubation of pro-MMP-13 (15.5 &#x003bc;M) with 50 &#x003bc;M APMA for 1 h. After activation of all MMPs using APMA, buffers were exchanged into PBS at 4&#x000b0;C using Sephadex G-25 (GE Healthcare).</p><p>For MMP-9 activation using MMP-3, the final 10.2 &#x003bc;M pro-MMP-9_Cat-H6 or biotinylated pro-MMP-9_Cat-BAP was incubated with 1.1 &#x003bc;M active MMP-3 in TNC buffer for 4 h at 37&#x000b0;C. For MMP-9 activation using EK, the final 4 &#x003bc;M pro-EK-MMP-9_Cat or mutants were incubated with 32 U/ml EKMax Enterokinase (Thermo Fisher Scientific) in the cleavage buffer (20 mM Tris-HCl, 50 mM NaCl, 2 mM CaCl<sub>2</sub>, pH 7.4) for 2 h at 4&#x000b0;C. EK was removed by EKapture Agarose (EMD Millipore). After the activation of MMP-9, buffers were exchanged into PBS at 4&#x000b0;C using an Amicon-Ultra centrifugal filter (10,000 NMWL; EMD Millipore). The purity of each active MMP-9 mutant was confirmed by SDS-PAGE.</p><p class="p p-last">For the preparation of MMP-9 E402Q mutant (MMP-9_Cat_E402Q-H6) the pro-peptide of which was removed, a final concentration of 10.2 &#x003bc;M pro-MMP-9_Cat_E402Q-H6 was incubated with 3.4 &#x003bc;M active MMP-3 in TNC buffer for 4 h at 37&#x000b0;C. After incubation, buffer was exchanged into PBS at 4&#x000b0;C using Amicon-Ultra centrifugal filter (10,000 NMWL). The purity of MMP-9_Cat_E402Q-H6 was confirmed by SDS-PAGE.</p></div><div id="sec013" class="sec"><h3 id="sec013title">Phage panning</h3><p class="p p-first-last">The engineered SPINK2 phage library (functional diversity, 1.2 &#x000d7; 10<sup>10</sup>) was previously established, and phage panning was performed following methods described previously [<a href="#pone.0244656.ref024" rid="pone.0244656.ref024" class=" bibr popnode">24</a>]. Briefly, for the first round of panning, 50 nM biotinylated active MMP-9_Cat-BAP was immobilized on Dynabeads M-280 Streptavidin (Thermo Fisher Scientific) for 2 h at 4&#x000b0;C. The beads were blocked with 3% BSA in TNC-T buffer (0.05% Tween 20 in TNC buffer) for 1 h at 4&#x000b0;C. Then, the engineered SPINK2 phage library (approximately 1.8 &#x000d7; 10<sup>13</sup> phages) was incubated with active MMP-9_Cat-BAP immobilized on beads in 3% BSA in TNC-T buffer overnight at 4&#x000b0;C. The beads were washed with TNC-T buffer, and then the bound phages were eluted using AcTEV Protease (Thermo Fisher Scientific). The recovered phage repertoire was amplified in <em>Escherichia coli</em> XL1-Blue cells (Agilent Technologies), which were then subjected to the next round of panning. In the second and third rounds, the amplified phages were incubated with 5 nM biotinylated active MMP-9_Cat-BAP at 4&#x000b0;C overnight. Phages bound to biotinylated active MMP-9_Cat-BAP were collected using Dynabeads M-280 Streptavidin blocked with 3% BSA in TNC-T buffer for 30 min at 4&#x000b0;C. Bound phages were washed with TNC-T buffer, followed by elution using AcTEV Protease. During subsequent selection rounds, the number of washing steps was gradually increased.</p></div><div id="sec014" class="sec"><h3 id="sec014title">Expression and purification of inhibitors</h3><p class="p p-first-last">Engineered SPINK2-derived MMP-9 inhibitors were produced following methods described previously [<a href="#pone.0244656.ref024" rid="pone.0244656.ref024" class=" bibr popnode">24</a>]. Briefly, the enriched phagemid DNA was cloned into the EcoRI/NotI sites of the expression vector pET32a (EMD Millipore). Origami B (DE3) (EMD Millipore) was transformed with the resulting vectors. The transformed <em>E</em>. <em>coli</em> was cultured in 2-YT medium (Thermo Fisher Scientific) at 37&#x000b0;C; then, isopropyl &#x003b2;-D-1-thiogalactopyranoside (IPTG) was added to a final concentration of 1 mM for induction. After cultivation at 16&#x000b0;C overnight, the cell pellet was collected and lysed using BugBuster Master Mix (EMD Millipore). His<sub>6</sub> tag-fused inhibitor with an N-terminal thioredoxin tag was purified using TALON Metal Affinity Resin (Clontech). The purified proteins were used for the primary screening of inhibitors. For further evaluation of the inhibitors, thioredoxin and His<sub>6</sub> tag were removed using Thrombin Cleavage Capture Kit (EMD Millipore). The inhibitors were finally purified by affinity chromatography using TALON Metal Affinity Resin, followed by size-exclusion chromatography (SEC) using Superdex 75 (GE Healthcare). The purified inhibitors had an N-terminal S tag for detection.</p></div><div id="sec015" class="sec"><h3 id="sec015title">ELISA for MMP-9 binding of inhibitors</h3><p class="p p-first-last">Biotinylated pro-MMP-9_Cat-BAP or biotinylated active MMP-9_Cat-BAP (each 50 nM) in PBS was coated on a 96-well Nunc Immobilizer streptavidin clear plate (Thermo Fisher Scientific) for 2 h at 4&#x000b0;C. The plate was washed with PBS-T (0.05% Tween 20 in PBS) and blocked with 3% BSA in PBS-T for 1 h at room temperature. Then, twofold serially diluted inhibitors (1&#x02013;1,000 nM) in PBS-T were added and incubated for 90 min at room temperature. After washing with PBS-T, horseradish peroxidase (HRP)-conjugated anti-S tag antibody (Bethyl Laboratories) was added (1:10,000 dilution in PBS-T) and incubated for 1 h at room temperature. After washing with PBS-T, the reaction was developed with 2,2&#x02032;-azino-di-(3-ethylbenzthiazoline sulfonic acid) substrate (Nacalai Tesque) at room temperature. The absorbance at 405 nm was measured using an EnSpire fluorescence plate reader (PerkinElmer). The EC<sub>50</sub> values were determined by a four-parameter logistic fit of the absorbance data using GraphPad Prism version 5.0 (GraphPad Software). Results are means of at two independent experiments.</p></div><div id="sec016" class="sec"><h3 id="sec016title">Size-exclusion chromatography analysis for MMP-9 binding of inhibitors</h3><p class="p p-first-last">The purified inhibitor and active MMP-9_Cat-H6 or MMP-9_Cat_E402Q-H6 were incubated at a molar ratio of 3:1 for 1 h at 4&#x000b0;C. Then, they were analyzed by SEC using an ACQUITY UPLC BEH200 column (Waters) in PBS. Gel filtration standard (Bio-Rad) was used as a molecular weight marker.</p></div><div id="sec017" class="sec"><h3 id="sec017title">Enzymatic assays</h3><p class="p p-first">All enzymatic assays were carried out at 37&#x000b0;C in a 96-well PROTEOSAVE black plate (Sumitomo Bakelite). As the assay buffer, TC buffer (50 mM Tris-HCl, 10 mM CaCl<sub>2</sub>, pH 7.5) was used in the MMP-17 enzymatic assay, while TNC buffer was used in the enzymatic assays for the other MMPs. The increase of fluorescence signal of the substrate was measured on EnSpire. The excitation and emission wavelengths of each substrate were as follows: 328 and 393 nm for 3168-v and 3226-v; and 495 and 515 nm for DQ-gelatin. Initial reaction velocities were determined by a linear fit of raw experimental data traces (fluorescence <em>versus</em> time) as the slopes of the regression lines. The IC<sub>50</sub> values were calculated from a four-parameter logistic fit of the initial reaction velocities using GraphPad Prism.</p><p>For the primary screening of inhibitors, each clone was incubated with 0.6 nM active MMP-9_Cat-H6 for 10 min. Then, peptide substrate 3226-v was added to a final concentration of 10 &#x003bc;M. The IC<sub>50</sub> values were calculated from at least two independent experiments.</p><p>For the enzymatic assay with MMP-9 mutants activated by EK, 1 nM active MMP-9_Cat or mutants were incubated with threefold serially diluted inhibitors (0&#x02013;330 nM or 0&#x02013;1,000 nM) for 1 h. Following incubation, the substrate 3226-v was added to achieve a final concentration of 10 &#x003bc;M. The IC<sub>50</sub> values are mean &#x000b1; S.D. from three independent experiments.</p><p>For enzyme kinetic measurements of the inhibitors, 0.4 nM active MMP-9_Cat-H6 was incubated with various concentrations of inhibitors (0&#x02013;25 nM) for 1 h. The reactions were initiated by the addition of substrate 3226-v at a final concentration of 10 &#x003bc;M. The inhibition constant (<em>K</em><sub><em>i</em></sub>) values were determined by fitting the Morrison equation for tightly binding inhibitors [<a href="#pone.0244656.ref049" rid="pone.0244656.ref049" class=" bibr popnode">49</a>]. Data were plotted as relative reaction velocity <em>versus</em> inhibitor concentration and fitted by nonlinear regression to <a href="#pone.0244656.e001" rid="pone.0244656.e001" class=" disp-formula">Eq 1</a> using GraphPad Prism:
</p><div class="disp-formula" id="pone.0244656.e001"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M1"><mfrac><mrow><mi>V</mi></mrow><mrow><msub><mrow><mi>V</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow></mfrac><mo>=</mo><mn>1</mn><mo>-</mo><mfrac><mrow><msub><mrow><mo>[</mo><mrow><mi>E</mi></mrow><mo>]</mo></mrow><mrow><mi>t</mi></mrow></msub><mo>+</mo><mo>[</mo><mrow><mi>I</mi></mrow><mo>]</mo><mo>+</mo><msubsup><mrow><mi>K</mi></mrow><mrow><mi>i</mi></mrow><mrow><mtext>app</mtext></mrow></msubsup><mo>-</mo><msqrt><msup><mrow><mo>(</mo><mrow><msub><mrow><mo>[</mo><mrow><mi>E</mi></mrow><mo>]</mo></mrow><mrow><mi>t</mi></mrow></msub><mo>+</mo><mo>[</mo><mrow><mi>I</mi></mrow><mo>]</mo><mo>+</mo><msubsup><mrow><mi>K</mi></mrow><mrow><mi>i</mi></mrow><mrow><mtext>app</mtext></mrow></msubsup></mrow><mo>)</mo></mrow><mrow><mn>2</mn></mrow></msup><mo>-</mo><mn>4</mn><msub><mrow><mo>[</mo><mrow><mi>E</mi></mrow><mo>]</mo></mrow><mrow><mi>t</mi></mrow></msub><mo>[</mo><mrow><mi>I</mi></mrow><mo>]</mo></msqrt></mrow><mrow><mn>2</mn><msub><mrow><mo>[</mo><mrow><mi>E</mi></mrow><mo>]</mo></mrow><mrow><mi>t</mi></mrow></msub></mrow></mfrac></math></div><div class="l">(1)</div></div><p>
</p><div class="disp-formula" id="pone.0244656.e002"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M2"><msubsup><mrow><mi>K</mi></mrow><mrow><mi>i</mi></mrow><mrow><mtext>app</mtext></mrow></msubsup><mo>=</mo><msub><mrow><mi>K</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>(</mo><mrow><mn>1</mn><mo>+</mo><mfrac><mrow><mo>[</mo><mrow><mi>S</mi></mrow><mo>]</mo></mrow><mrow><msub><mrow><mi>K</mi></mrow><mrow><mi>m</mi></mrow></msub></mrow></mfrac></mrow><mo>)</mo></math></div><div class="l">(2)</div></div><p>
where <em>V</em> is the initial reaction velocity; <em>V</em><sub>0</sub> is the initial reaction velocity in the absence of inhibitor; [<em>E</em>]<sub><em>t</em></sub> is the total concentration of enzyme; [<em>I</em>] is the inhibitor concentration; [<em>S</em>] is the substrate concentration; <em>K</em><sub><em>m</em></sub> is the Michaelis-Menten constant; and <em>K</em><sub><em>i</em></sub><sup>app</sup> is the apparent inhibition constant given by <a href="#pone.0244656.e002" rid="pone.0244656.e002" class=" disp-formula">Eq 2</a>. The <em>K</em><sub><em>m</em></sub> values were determined by measuring the initial reaction velocities of hydrolysis of 3226-v (2.5&#x02013;10 &#x003bc;M) by active MMP-9_Cat-H6 [<a href="#pone.0244656.ref050" rid="pone.0244656.ref050" class=" bibr popnode">50</a>]. Results are mean &#x000b1; S.D. for three independent experiments.</p><p>For enzymatic assay with a macromolecular substrate, 0.6 nM active MMP-9_Cat-H6 was incubated with threefold serially diluted inhibitors (0&#x02013;100 nM) for 1 h. Following incubation, DQ-gelatin was added to achieve a final concentration of 10 &#x003bc;g/ml. The IC<sub>50</sub> values are mean &#x000b1; S.D. for three independent experiments.</p><p class="p p-last">For specificity experiments, enzymatic assays using 12 human MMPs were carried out. Each active MMP was incubated with 1 &#x003bc;M inhibitors for 10 or 60 min, followed by the addition of each fluorescence-quenching peptide substrate. Data are expressed as the percentage of uninhibited activity. Experimental conditions of each MMP are described in <a href="#pone.0244656.s012" rid="pone.0244656.s012" class=" supplementary-material">S2 Table</a>.</p></div><div id="sec018" class="sec sec-last"><h3 id="sec018title">Statistical analyses</h3><p class="p p-first-last">All statistical analyses were performed using GraphPad Prism version 5.0 (GraphPad Software). Sample size for each experimental group was reported in the figure legends. The data are presented as mean &#x000b1; S.D. One-way analysis of variance (ANOVA) followed by Dunnett&#x02019;s test was used to determine the statistical significance in the comparison of IC50 value of the inhibitor against MMP-9 mutants <em>versus</em> wild-type MMP-9. <em>P</em> values less than 0.05 were considered statistical significant.</p></div></div><div id="sec019" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="sec019title">Supporting information</h2><!--/article/body/sec/--><div class="sec suppmat" id="pone.0244656.s001"><h4>S1 Fig</h4><!--caption a9--><strong>SDS-PAGE analysis of purified MMP-9. </strong><p>(A) Pro-MMP-9_Cat-H6 was purified and activated by active MMP-3, as described in &#x0201c;Materials and methods.&#x0201d; (B) Pro-MMP-9_Cat_E402Q-H6 was purified and processed by active MMP-3, as described in &#x0201c;Materials and methods.&#x0201d; SDS-PAGE analysis of the purified pro-MMP-9 (1 &#x003bc;g per gel lane), active MMP-9_Cat-H6, and MMP-9_Cat_E402Q-H6 (0.5 &#x003bc;g per gel lane) was performed under reducing conditions followed by Coomassie Brilliant Blue G-250 staining.</p><p>(TIF)</p><div class="sup-box half_rhythm" id="media-a.ai.e.b.c"><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s001.tif" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(359K, tif)</sup></div></div><div class="sec suppmat" id="pone.0244656.s002"><h4>S2 Fig</h4><!--caption a9--><strong>Scheme of a randomized region of the engineered SPINK2. </strong><p>(A) Amino acid sequence of wild-type SPINK2. Lines indicate disulfide bonds (Cys-14&#x02013;Cys-44, Cys-22&#x02013;Cys-41, Cys-30&#x02013;Cys-62). (B) Region randomized to create the engineered SPINK2 library. (C) Three-dimensional structure of wild-type SPINK2 (PDB code, 2JXD). SPINK2 is shown as a gray and red semi-transparent surface model; red indicates the randomized region. The right figure represents the left image turned 90&#x000b0; counterclockwise about the y-axis. (D) Aligned sequences of the engineered SPINK2-derived inhibitors against MMP-9.</p><p>(TIF)</p><div class="sup-box half_rhythm" id="media-a.ai.e.c.c"><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s002.tif" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(1.0M, tif)</sup></div></div><div class="sec suppmat" id="pone.0244656.s003"><h4>S3 Fig</h4><!--caption a9--><strong>The binding properties of inhibitors were evaluated by ELISA. </strong><p>Various concentrations of pro-MMP-9_Cat-Avi or active MMP-9_Cat-Avi (0.8&#x02013;100 nM) were added to <a id="__tag_904612480" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> or GS-5745 (10 &#x003bc;g/ml each)-coated plates, and then C-terminal FLAG tag of captured MMP-9 was detected by HRP-conjugated anti-FLAG tag antibody. All curves were obtained by non-linear curve fitting.</p><p>(TIF)</p><div class="sup-box half_rhythm" id="media-a.ai.e.d.c"><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s003.tif" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(442K, tif)</sup></div></div><div class="sec suppmat" id="pone.0244656.s004"><h4>S4 Fig</h4><!--caption a9--><strong>Sequence alignment of pre-pro-form lacking the hemopexin domain of MMP-1, -2, -8, -9, and -13. </strong><p>The residues are numbered according to the generic MMP-9 nomenclature. Fn-like domain of MMP-2 and -9 is represented as XXX. Symbols denote catalytic glutamate residue (<em>asterisk</em>), and residues interacting with catalytic zinc ion (Z1), structural zinc ion (Z2), and calcium ions (Ca). Conserved residues among the five MMPs are shown with a <em>gray</em> background.</p><p>(TIF)</p><div class="sup-box half_rhythm" id="media-a.ai.e.e.c"><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s004.tif" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(780K, tif)</sup></div></div><div class="sec suppmat" id="pone.0244656.s005"><h4>S5 Fig</h4><!--caption a9--><strong>Purification and activation of MMP-9 mutants. </strong><p>(A) and (B) Pro-EK-MMP-9_Cat (WT), pro-forms of the cleft mutants, and exosite mutants were purified using HisTrap excel gel and gelatin-Sepharose resin. Each pro-MMP-9 (4 &#x003bc;M) was incubated with EKMax Enterokinase (32 U/ml) for 2 h at 4&#x000b0;C. After the activation of MMP-9, EK was removed by EKapture Agarose and buffer was exchanged for PBS at 4&#x000b0;C. SDS-PAGE analysis of the purified pro-MMP-9 (A, 1 &#x003bc;g per gel lane) and activated MMP-9 (B, 0.5 &#x003bc;g per gel lane, <em>asterisk</em> indicates 1 &#x003bc;g per gel lane) was performed under reducing conditions followed by Coomassie Brilliant Blue G-250 staining. All of the pro- and active MMP-9 mutants were highly purified. (C) MMP-9 activities of each mutant were determined by enzymatic assay using peptide substrate. Active MMP-9_Cat (WT) or activated mutants (1 nM each) were incubated with peptide substrate 3226-v (10 &#x003bc;M), and then MMP-9 activities were determined by monitoring the increase of fluorescence signal of the substrate. The activities of each mutant were normalized to that of WT. Most of the activated mutants, except for the L188A mutant, showed sufficient proteolytic activity to use for enzymatic assays. Each <em>bar</em> represents the mean &#x000b1; S.D. (n = 3).</p><p>(TIF)</p><div class="sup-box half_rhythm" id="media-a.ai.e.f.c"><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s005.tif" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(2.2M, tif)</sup></div></div><div class="sec suppmat" id="pone.0244656.s006"><h4>S6 Fig</h4><!--caption a9--><strong>MMP-9 inhibitory activities of sc-311438 towards the cleft mutants (A) and the exosite mutants (B). </strong><p>All assay conditions and data presentation are the same as in <a href="/pmc/articles/PMC7771667/figure/pone.0244656.g005/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g005" rid-ob="ob-pone.0244656.g005" co-legend-rid="lgnd_pone.0244656.g005"><span>Fig 5</span></a>, and Tables <a href="/pmc/articles/PMC7771667/table/pone.0244656.t002/" target="table" class="fig-table-link figpopup" rid-figpopup="pone.0244656.t002" rid-ob="ob-pone.0244656.t002" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Tables2</span></span><span>2</span></a> and <a href="/pmc/articles/PMC7771667/table/pone.0244656.t003/" target="table" class="fig-table-link figpopup" rid-figpopup="pone.0244656.t003" rid-ob="ob-pone.0244656.t003" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and3.</span></span><span>3</span></a>. Statistical analysis of cleft mutants <em>versus</em> WT was performed by one-way ANOVA with Dunnett&#x02019;s post tests for multiple comparisons.</p><p>(TIF)</p><div class="sup-box half_rhythm" id="media-a.ai.e.g.c"><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s006.tif" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(3.1M, tif)</sup></div></div><div class="sec suppmat" id="pone.0244656.s007"><h4>S7 Fig</h4><!--caption a9--><strong>MMP-9 inhibitory activities of <a id="__tag_904612501" class="tag_hotlink" href="/nuccore/M91005" ref="/nuccore/M91005"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"M91005","term_id":"165006","term_text":"M91005"}}</span>M91005</a> towards three different MMP-9 constructs. </strong><p>For enzymatic assay with three different MMP-9 constructs, namely, full-length MMP-9, the form with HPX domain deleted (MMP-9_Cat), and the form with both Fn-like domain and HPX domain deleted (the catalytic domain), 0.4 nM active MMP-9 was incubated with threefold serially diluted inhibitors (0&#x02013;100 nM) for 1 h. Following incubation, substrate 3226-v was added to achieve a final concentration of 10 &#x003bc;M. The other assay conditions are the same as in <a href="/pmc/articles/PMC7771667/figure/pone.0244656.g001/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone.0244656.g001" rid-ob="ob-pone.0244656.g001" co-legend-rid="lgnd_pone.0244656.g001"><span>Fig 1C</span></a>.</p><p>(TIF)</p><div class="sup-box half_rhythm" id="media-a.ai.e.h.c"><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s007.tif" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(151K, tif)</sup></div></div><div class="sec suppmat" id="pone.0244656.s008"><h4>S8 Fig</h4><!--caption a9--><strong>Expression of active-site mutants of MMP-9 (H401A, H405A, and H411A). </strong><p>Western blot analysis of the culture supernatants (6.5 &#x003bc;l per gel lane) of HEK293F cells transfected with each MMP-9 expression vector was performed under reducing conditions. After electrophoresis, the proteins were transferred to a PVDF membrane followed by blocking with 5% skim milk in PBS-T. After washing with PBS-T, Penta His HRP Conjugate (QIAGEN, 34460) was added (1:10,000 dilution in PBS-T with 0.5% skim milk) and incubated for 1 h at room temperature. After washing with PBS-T, the reaction was developed with ECL Prime Western Blotting Detection Reagent (GE Healthcare) at room temperature. The pre-stained visible protein markers and the chemiluminescent signals were captured using a ChemiDoc XRS+ CCD camera-based imager system (Bio-Rad). <em>Black arrowhead</em> indicates the band of full-length MMP-9 fused to a C-terminal His<sub>6</sub> tag.</p><p>(TIF)</p><div class="sup-box half_rhythm" id="media-a.ai.e.i.c"><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s008.tif" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(39K, tif)</sup></div></div><div class="sec suppmat" id="pone.0244656.s009"><h4>S9 Fig</h4><!--caption a9--><strong>C-terminal degradation of MMP-9 during activation reaction using trypsin. </strong><p>For MMP-9 activation using trypsin, the final 4 &#x003bc;M pro-MMP-9_Cat-H6 was incubated with 2.7 &#x003bc;M TPCK trypsin (Thermo Fisher Scientific, 20233) in TNC buffer at 37&#x000b0;C. Western blot analysis of the activation reaction solution (0.6 &#x003bc;g of MMP-9 per gel lane) was performed under reducing conditions. The experimental conditions after electrophoresis were carried out according to <a href="#pone.0244656.s008" rid="pone.0244656.s008" class=" supplementary-material">S8 Fig</a>. <em>Black arrowhead</em> and <em>red arrowhead</em> indicate the band of the product obtained by the activation reaction using trypsin and the band of C-terminal fragment of pro-MMP-9_Cat-H6, respectively.</p><p>(TIF)</p><div class="sup-box half_rhythm" id="media-a.ai.e.j.c"><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s009.tif" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(217K, tif)</sup></div></div><div class="sec suppmat" id="pone.0244656.s010"><h4>S10 Fig</h4><!--caption a9--><strong>Autolysis of active MMP-9 upon long-term incubation at 25&#x000b0;C. </strong><p>To evaluate the stability of the bait protein, 2.4 &#x003bc;M of biotinylated active MMP-9_Cat-BAP was incubated in PBS at 25&#x000b0;C for 16 h. SDS-PAGE analysis of the biotinylated active MMP-9_Cat-BAP after incubation (0.5 &#x003bc;g per gel lane) was performed under reducing conditions followed by Coomassie Brilliant Blue G-250 staining. <em>Black arrowhead</em> and <em>red arrowhead</em> indicate the band of the biotinylated active MMP-9_Cat-BAP and the band of degradation product, respectively.</p><p>(TIF)</p><div class="sup-box half_rhythm" id="media-a.ai.e.k.c"><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s010.tif" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(257K, tif)</sup></div></div><div class="sec suppmat" id="pone.0244656.s011"><h4>S1 Table</h4><!--caption a9--><strong>MMP inhibitory activities of inhibitors against eight MMPs. </strong><p>The cross-reactivities against MMP-3, -7, -10, -12, -14, -15, -16, and -17 were measured by enzymatic assay using peptide substrate. Each active MMP was incubated with inhibitors (1 &#x003bc;M), and then the peptide substrate (10 &#x003bc;M 3168-v for MMP-3, 10 &#x003bc;M 3226-v for the other MMPs) was added as described under &#x0201c;Materials and methods.&#x0201d; Enzymatic activity was determined by monitoring the hydrolysis of the peptide substrate and each remaining enzymatic activity was normalized to the activity in the absence of inhibitors. Data are shown as the mean &#x000b1; S.D. (n = 3).</p><p>(DOCX)</p><div class="sup-box half_rhythm" id="media-a.ai.e.l.c"><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s011.docx" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(27K, docx)</sup></div></div><div class="sec suppmat" id="pone.0244656.s012"><h4>S2 Table</h4><!--caption a9--><strong>MMP enzymatic assay conditions. </strong><p>(DOCX)</p><div class="sup-box half_rhythm" id="media-a.ai.e.m.c"><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s012.docx" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(26K, docx)</sup></div></div><div class="sec suppmat" id="pone.0244656.s013"><h4>S3 Table</h4><!--caption a9--><strong>PCR primers used for site-directed mutagenesis. </strong><p>(DOCX)</p><div class="sup-box half_rhythm" id="media-a.ai.e.n.c"><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s013.docx" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(29K, docx)</sup></div></div><div class="sec suppmat" id="pone.0244656.s014"><h4>S1 Protocol</h4><!--caption a9--><strong>Protocol for ELISA assay using anti-MMP-9 antibody GS-5745. </strong><p>(DOCX)</p><div class="sup-box half_rhythm" id="media-a.ai.e.o.c"><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s014.docx" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(25K, docx)</sup></div></div><div class="sec suppmat" id="pone.0244656.s015"><h4>S1 Raw images</h4><!--caption a9--><strong>The original uncropped and unadjusted image to prepare the final figures. </strong><p>Images were captured using a ChemiDocXRS+ CCD camera-based imager system (Bio-Rad). The part of the blot or gel shown in the final figure is within the red box. Lanes marked with an "X" were not included in the final figure.</p><p>(PDF)</p><div class="sup-box half_rhythm" id="media-a.ai.e.p.c"><a href="/pmc/articles/PMC7771667/bin/pone.0244656.s015.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(895K, pdf)</sup></div></div></div><div id="ack-a.aj.a" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.aj.atitle">Acknowledgments</h2><div class="sec"><p>We thank Dr. Shiho Kodama for help with calculation of the <em>Ki</em> values of the inhibitors. We also thank Aya Yamaguchi for help with preparation of the inhibitors. We are grateful to Dr. Akiyoshi Fukamizu for his thoughtful advice.</p></div></div><div id="funding-group-a.ah.b.t" class="tsec sec"><h2 class="head no_bottom_margin" id="funding-group-a.ah.b.ttitle">Funding Statement</h2><p>This work was fully supported by Daiichi Sankyo Co., Ltd. All authors are employees of Daiichi Sankyo Co., Ltd. The funder provided support in the form of salaries for authors: H.Y., D.N., Y.K., M.T., and R.H., but did not have any additional role in the study design, data collection and analysis, decision to publish, or preparation of the manuscript. The specific roles of these authors are articulated in the &#x02018;author contributions&#x02019; section.</p></div><div id="notes-a.ah.c" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.ah.ctitle">Data Availability</h2><p>All relevant data are within the manuscript and its <a href="#sec019" rid="sec019" class=" sec">Supporting information</a> files.</p></div><div id="ref-list-a.aj.b" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.aj.btitle">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref001">1. <span class="mixed-citation">Hu J, Van den Steen PE, Sang QXA, Opdenakker G. <span class="ref-title">Matrix metalloproteinase inhibitors as therapy for inflammatory and vascular diseases</span>. <span class="ref-journal">Nat Rev Drug Discov</span>. 2007;<span class="ref-vol">6</span>: 480&#x02013;498. 10.1038/nrd2308
 [<a href="https://pubmed.ncbi.nlm.nih.gov/17541420" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnrd2308" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Drug+Discov&amp;title=Matrix+metalloproteinase+inhibitors+as+therapy+for+inflammatory+and+vascular+diseases&amp;author=J+Hu&amp;author=PE+Van+den+Steen&amp;author=QXA+Sang&amp;author=G+Opdenakker&amp;volume=6&amp;publication_year=2007&amp;pages=480-498&amp;pmid=17541420&amp;doi=10.1038/nrd2308&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref002">2. <span class="mixed-citation">Gialeli C, Theocharis AD, Karamanos NK. <span class="ref-title">Roles of matrix metalloproteinases in cancer progression and their pharmacological targeting</span>. <span class="ref-journal">FEBS J</span>. 2011;<span class="ref-vol">278</span>: 16&#x02013;27. 10.1111/j.1742-4658.2010.07919.x
 [<a href="https://pubmed.ncbi.nlm.nih.gov/21087457" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fj.1742-4658.2010.07919.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEBS+J&amp;title=Roles+of+matrix+metalloproteinases+in+cancer+progression+and+their+pharmacological+targeting&amp;author=C+Gialeli&amp;author=AD+Theocharis&amp;author=NK+Karamanos&amp;volume=278&amp;publication_year=2011&amp;pages=16-27&amp;pmid=21087457&amp;doi=10.1111/j.1742-4658.2010.07919.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref003">3. <span class="mixed-citation">Vandenbroucke RE, Libert C. <span class="ref-title">Is there new hope for therapeutic matrix metalloproteinase inhibition?</span>
<span class="ref-journal">Nat Rev Drug Discov</span>. 2014;<span class="ref-vol">13</span>: 904&#x02013;927. 10.1038/nrd4390
 [<a href="https://pubmed.ncbi.nlm.nih.gov/25376097" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnrd4390" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Drug+Discov&amp;title=Is+there+new+hope+for+therapeutic+matrix+metalloproteinase+inhibition?&amp;author=RE+Vandenbroucke&amp;author=C+Libert&amp;volume=13&amp;publication_year=2014&amp;pages=904-927&amp;pmid=25376097&amp;doi=10.1038/nrd4390&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref004">4. <span class="mixed-citation">Zitka O, Kukacka J, Krizkova S, Huska D, Adam V, Masarik M, et al. 
<span class="ref-title">Matrix metalloproteinases</span>. <span class="ref-journal">Curr Med Chem</span>. 2010;<span class="ref-vol">17</span>: 3751&#x02013;3768. 10.2174/092986710793213724
 [<a href="https://pubmed.ncbi.nlm.nih.gov/20846107" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.2174%2F092986710793213724" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Med+Chem&amp;title=Matrix+metalloproteinases&amp;author=O+Zitka&amp;author=J+Kukacka&amp;author=S+Krizkova&amp;author=D+Huska&amp;author=V+Adam&amp;volume=17&amp;publication_year=2010&amp;pages=3751-3768&amp;pmid=20846107&amp;doi=10.2174/092986710793213724&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref005">5. <span class="mixed-citation">Elkins PA, Yen SH, Smith WW, Janson CA, D&#x02019;Alessio KJ, McQueney MS, et al. 
<span class="ref-title">Structure of the C-terminally truncated human ProMMP9, a gelatin-binding matrix metalloproteinase</span>. <span class="ref-journal">Acta Crystallogr Sect D Biol Crystallogr</span>. 2002;<span class="ref-vol">58</span>: 1182&#x02013;1192. 10.1107/s0907444902007849
 [<a href="https://pubmed.ncbi.nlm.nih.gov/12077439" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1107%2Fs0907444902007849" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Crystallogr+Sect+D+Biol+Crystallogr&amp;title=Structure+of+the+C-terminally+truncated+human+ProMMP9,+a+gelatin-binding+matrix+metalloproteinase&amp;author=PA+Elkins&amp;author=SH+Yen&amp;author=WW+Smith&amp;author=CA+Janson&amp;author=KJ+D&#x02019;Alessio&amp;volume=58&amp;publication_year=2002&amp;pages=1182-1192&amp;pmid=12077439&amp;doi=10.1107/s0907444902007849&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref006">6. <span class="mixed-citation">Ogata Y, Enghild JJ, Nagase H. <span class="ref-title">Matrix metalloproteinase 3 (stromelysin) activates the precursor for the human matrix metalloproteinase 9</span>. <span class="ref-journal">J Biol Chem</span>. 1992;<span class="ref-vol">267</span>: 3581&#x02013;3584.  [<a href="https://pubmed.ncbi.nlm.nih.gov/1371271" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Matrix+metalloproteinase+3+(stromelysin)+activates+the+precursor+for+the+human+matrix+metalloproteinase+9&amp;author=Y+Ogata&amp;author=JJ+Enghild&amp;author=H+Nagase&amp;volume=267&amp;publication_year=1992&amp;pages=3581-3584&amp;pmid=1371271&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref007">7. <span class="mixed-citation">Okada Y, Gonoji Y, Naka K, Tomita K, Nakanishi I, Iwata K, et al. 
<span class="ref-title">Matrix metalloproteinase 9 (92-kDa gelatinase/type IV collagenase) from HT 1080 human fibrosarcoma cells. Purification and activation of the precursor and enzymic properties</span>. <span class="ref-journal">J Biol Chem</span>. 1992;<span class="ref-vol">267</span>: 21712&#x02013;9. Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/1400481" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/pubmed/1400481</a> [<a href="https://pubmed.ncbi.nlm.nih.gov/1400481" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Matrix+metalloproteinase+9+(92-kDa+gelatinase/type+IV+collagenase)+from+HT+1080+human+fibrosarcoma+cells.+Purification+and+activation+of+the+precursor+and+enzymic+properties&amp;author=Y+Okada&amp;author=Y+Gonoji&amp;author=K+Naka&amp;author=K+Tomita&amp;author=I+Nakanishi&amp;volume=267&amp;publication_year=1992&amp;pages=21712-9&amp;pmid=1400481&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref008">8. <span class="mixed-citation">Collier IE, Wilhelm SM, Eisen AZ, Marmer BL, Grant GA, Seltzer JL, et al. 
<span class="ref-title">H-ras oncogene-transformed human bronchial epithelial cells (TBE-1) secrete a single metalloprotease capable of degrading basement membrane collagen</span>. <span class="ref-journal">J Biol Chem</span>. 1988;<span class="ref-vol">263</span>: 6579&#x02013;6587.  [<a href="https://pubmed.ncbi.nlm.nih.gov/2834383" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=H-ras+oncogene-transformed+human+bronchial+epithelial+cells+(TBE-1)+secrete+a+single+metalloprotease+capable+of+degrading+basement+membrane+collagen&amp;author=IE+Collier&amp;author=SM+Wilhelm&amp;author=AZ+Eisen&amp;author=BL+Marmer&amp;author=GA+Grant&amp;volume=263&amp;publication_year=1988&amp;pages=6579-6587&amp;pmid=2834383&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref009">9. <span class="mixed-citation">Murphy G, Ward R, Hembry RM, Reynolds JJ, Kuhnt K, Tryggvasont K. <span class="ref-title">Characterization of gelatinase from pig polymorphonuclear leucocytes</span>. <span class="ref-journal">Biochem J</span> 258, 463&#x02013;472. 1989;<span class="ref-vol">258</span>: 463&#x02013;472. 10.1042/bj2580463
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1138384/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2539808" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1042%2Fbj2580463" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+J&amp;title=Characterization+of+gelatinase+from+pig+polymorphonuclear+leucocytes&amp;author=G+Murphy&amp;author=R+Ward&amp;author=RM+Hembry&amp;author=JJ+Reynolds&amp;author=K+Kuhnt&amp;volume=258&amp;publication_year=1989&amp;pages=463-472&amp;pmid=2539808&amp;doi=10.1042/bj2580463&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref010">10. <span class="mixed-citation">Vandooren J, Van Den Steen PE, Opdenakker G. <span class="ref-title">Biochemistry and molecular biology of gelatinase B or matrix metalloproteinase-9 (MMP-9): The next decade</span>. <span class="ref-journal">Crit Rev Biochem Mol Biol</span>. 2013;<span class="ref-vol">48</span>: 222&#x02013;272. 10.3109/10409238.2013.770819
 [<a href="https://pubmed.ncbi.nlm.nih.gov/23547785" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3109%2F10409238.2013.770819" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Crit+Rev+Biochem+Mol+Biol&amp;title=Biochemistry+and+molecular+biology+of+gelatinase+B+or+matrix+metalloproteinase-9+(MMP-9):+The+next+decade&amp;author=J+Vandooren&amp;author=PE+Van+Den+Steen&amp;author=G+Opdenakker&amp;volume=48&amp;publication_year=2013&amp;pages=222-272&amp;pmid=23547785&amp;doi=10.3109/10409238.2013.770819&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref011">11. <span class="mixed-citation">Turk BE, Huang LL, Piro ET, Cantley LC. <span class="ref-title">Determination of protease cleavage site motifs using mixture-based oriented peptide libraries</span>. <span class="ref-journal">Nat Biotechnol</span>. 2001;<span class="ref-vol">19</span>: 661&#x02013;667. 10.1038/90273
 [<a href="https://pubmed.ncbi.nlm.nih.gov/11433279" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2F90273" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Biotechnol&amp;title=Determination+of+protease+cleavage+site+motifs+using+mixture-based+oriented+peptide+libraries&amp;author=BE+Turk&amp;author=LL+Huang&amp;author=ET+Piro&amp;author=LC+Cantley&amp;volume=19&amp;publication_year=2001&amp;pages=661-667&amp;pmid=11433279&amp;doi=10.1038/90273&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref012">12. <span class="mixed-citation">Maskos K. <span class="ref-title">Crystal structures of MMPs in complex with physiological and pharmacological inhibitors</span>. <span class="ref-journal">Biochimie</span>. 2005;<span class="ref-vol">87</span>: 249&#x02013;263. 10.1016/j.biochi.2004.11.019
 [<a href="https://pubmed.ncbi.nlm.nih.gov/15781312" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.biochi.2004.11.019" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochimie&amp;title=Crystal+structures+of+MMPs+in+complex+with+physiological+and+pharmacological+inhibitors&amp;author=K+Maskos&amp;volume=87&amp;publication_year=2005&amp;pages=249-263&amp;pmid=15781312&amp;doi=10.1016/j.biochi.2004.11.019&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref013">13. <span class="mixed-citation">Steffensen B, Wallon MU, Christopher OM. <span class="ref-title">Extracellular Matrix Binding Properties of Recombinant Fibronectin Type II-like Modules of Human 72-kDa Gelatinase/Type IV Collagenase</span>. <span class="ref-journal">J Biol Chem</span>. 1995;<span class="ref-vol">270</span>: 11555&#x02013;11566. 10.1074/jbc.270.19.11555
 [<a href="https://pubmed.ncbi.nlm.nih.gov/7744795" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1074%2Fjbc.270.19.11555" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Extracellular+Matrix+Binding+Properties+of+Recombinant+Fibronectin+Type+II-like+Modules+of+Human+72-kDa+Gelatinase/Type+IV+Collagenase&amp;author=B+Steffensen&amp;author=MU+Wallon&amp;author=OM+Christopher&amp;volume=270&amp;publication_year=1995&amp;pages=11555-11566&amp;pmid=7744795&amp;doi=10.1074/jbc.270.19.11555&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref014">14. <span class="mixed-citation">O&#x02019;Farrell TJ, Pourmotabbed T. <span class="ref-title">The fibronectin-like domain is required for the type V and XI collagenolytic activity of gelatinase B</span>. <span class="ref-journal">Arch Biochem Biophys</span>. 1998;<span class="ref-vol">354</span>: 24&#x02013;30. 10.1006/abbi.1998.0662
 [<a href="https://pubmed.ncbi.nlm.nih.gov/9633594" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1006%2Fabbi.1998.0662" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Biochem+Biophys&amp;title=The+fibronectin-like+domain+is+required+for+the+type+V+and+XI+collagenolytic+activity+of+gelatinase+B&amp;author=TJ+O&#x02019;Farrell&amp;author=T+Pourmotabbed&amp;volume=354&amp;publication_year=1998&amp;pages=24-30&amp;pmid=9633594&amp;doi=10.1006/abbi.1998.0662&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref015">15. <span class="mixed-citation">Xu X, Chen Z, Wang Y, Yamada Y, Steffensen B. <span class="ref-title">Functional basis for the overlap in ligand interactions and substrate specificities of matrix metalloproteinases-9 and -2</span>. <span class="ref-journal">Biochem J</span>. 2005;<span class="ref-vol">392</span>: 127&#x02013;134. 10.1042/BJ20050650
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1317671/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16008524" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1042%2FBJ20050650" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+J&amp;title=Functional+basis+for+the+overlap+in+ligand+interactions+and+substrate+specificities+of+matrix+metalloproteinases-9+and+-2&amp;author=X+Xu&amp;author=Z+Chen&amp;author=Y+Wang&amp;author=Y+Yamada&amp;author=B+Steffensen&amp;volume=392&amp;publication_year=2005&amp;pages=127-134&amp;pmid=16008524&amp;doi=10.1042/BJ20050650&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref016">16. <span class="mixed-citation">Sela-Passwell N, Kikkeri R, Dym O, Rozenberg H, Margalit R, Arad-Yellin R, et al. 
<span class="ref-title">Antibodies targeting the catalytic zinc complex of activated matrix metalloproteinases show therapeutic potential</span>. <span class="ref-journal">Nat Med</span>. 2012;<span class="ref-vol">18</span>: 143&#x02013;147. 10.1038/nm.2582
 [<a href="https://pubmed.ncbi.nlm.nih.gov/22198278" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnm.2582" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Med&amp;title=Antibodies+targeting+the+catalytic+zinc+complex+of+activated+matrix+metalloproteinases+show+therapeutic+potential&amp;author=N+Sela-Passwell&amp;author=R+Kikkeri&amp;author=O+Dym&amp;author=H+Rozenberg&amp;author=R+Margalit&amp;volume=18&amp;publication_year=2012&amp;pages=143-147&amp;pmid=22198278&amp;doi=10.1038/nm.2582&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref017">17. <span class="mixed-citation">Gomis-R&#x000fc;th FX, Maskos K, Betz M, Bergner A, Huber R, Suzuki K, et al. 
<span class="ref-title">Mechanism of inhibition of the human matrix metalloproteinase stromelysin-1 by TIMP-1</span>. <span class="ref-journal">Nature</span>. 1997;<span class="ref-vol">389</span>: 77&#x02013;81. 10.1038/37995
 [<a href="https://pubmed.ncbi.nlm.nih.gov/9288970" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2F37995" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Mechanism+of+inhibition+of+the+human+matrix+metalloproteinase+stromelysin-1+by+TIMP-1&amp;author=FX+Gomis-R&#x000fc;th&amp;author=K+Maskos&amp;author=M+Betz&amp;author=A+Bergner&amp;author=R+Huber&amp;volume=389&amp;publication_year=1997&amp;pages=77-81&amp;pmid=9288970&amp;doi=10.1038/37995&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref018">18. <span class="mixed-citation">Sharabi O, Shirian J, Grossman M, Lebendiker M, Sagi I, Shifman J. <span class="ref-title">Affinity- and specificity-enhancing mutations are frequent in multispecific interactions between TIMP2 and MMPs</span>. <span class="ref-journal">PLoS One</span>. 2014;<span class="ref-vol">9</span>
 10.1371/journal.pone.0093712
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3977929/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24710006" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pone.0093712" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Affinity-+and+specificity-enhancing+mutations+are+frequent+in+multispecific+interactions+between+TIMP2+and+MMPs&amp;author=O+Sharabi&amp;author=J+Shirian&amp;author=M+Grossman&amp;author=M+Lebendiker&amp;author=I+Sagi&amp;volume=9&amp;publication_year=2014&amp;pmid=24710006&amp;doi=10.1371/journal.pone.0093712&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref019">19. <span class="mixed-citation">Arkadash V, Yosef G, Shirian J, Cohen I, Horev Y, Grossman M, et al. 
<span class="ref-title">Development of high affinity and high specificity inhibitors of matrix metalloproteinase 14 through computational design and directed evolution</span>. <span class="ref-journal">J Biol Chem</span>. 2017;<span class="ref-vol">292</span>: 3481&#x02013;3495. 10.1074/jbc.M116.756718
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5336179/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28087697" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1074%2Fjbc.M116.756718" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Development+of+high+affinity+and+high+specificity+inhibitors+of+matrix+metalloproteinase+14+through+computational+design+and+directed+evolution&amp;author=V+Arkadash&amp;author=G+Yosef&amp;author=J+Shirian&amp;author=I+Cohen&amp;author=Y+Horev&amp;volume=292&amp;publication_year=2017&amp;pages=3481-3495&amp;pmid=28087697&amp;doi=10.1074/jbc.M116.756718&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref020">20. <span class="mixed-citation">Shirian J, Arkadash V, Cohen I, Sapir T, Radisky ES, Papo N, et al. 
<span class="ref-title">Converting a broad matrix metalloproteinase family inhibitor into a specific inhibitor of MMP-9 and MMP-14</span>. <span class="ref-journal">FEBS Lett</span>. 2018;<span class="ref-vol">592</span>: 1122&#x02013;1134. 10.1002/1873-3468.13016
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5903968/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29473954" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2F1873-3468.13016" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEBS+Lett&amp;title=Converting+a+broad+matrix+metalloproteinase+family+inhibitor+into+a+specific+inhibitor+of+MMP-9+and+MMP-14&amp;author=J+Shirian&amp;author=V+Arkadash&amp;author=I+Cohen&amp;author=T+Sapir&amp;author=ES+Radisky&amp;volume=592&amp;publication_year=2018&amp;pages=1122-1134&amp;pmid=29473954&amp;doi=10.1002/1873-3468.13016&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref021">21. <span class="mixed-citation">Dufour A, Sampson NS, Li J, Kuscu C, Rizzo RC, Deleon JL, et al. 
<span class="ref-title">Small-molecule anticancer compounds selectively target the hemopexin domain of matrix metalloproteinase-9</span>. <span class="ref-journal">Cancer Res</span>. 2011;<span class="ref-vol">71</span>: 4977&#x02013;88. 10.1158/0008-5472.CAN-10-4552
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3138841/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21646471" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1158%2F0008-5472.CAN-10-4552" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Small-molecule+anticancer+compounds+selectively+target+the+hemopexin+domain+of+matrix+metalloproteinase-9&amp;author=A+Dufour&amp;author=NS+Sampson&amp;author=J+Li&amp;author=C+Kuscu&amp;author=RC+Rizzo&amp;volume=71&amp;publication_year=2011&amp;pages=4977-88&amp;pmid=21646471&amp;doi=10.1158/0008-5472.CAN-10-4552&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref022">22. <span class="mixed-citation">Alford VM, Kamath A, Ren X, Kumar K, Gan Q, Awwa M, et al. 
<span class="ref-title">Targeting the Hemopexin-like Domain of Latent Matrix Metalloproteinase-9 (proMMP-9) with a Small Molecule Inhibitor Prevents the Formation of Focal Adhesion Junctions</span>. <span class="ref-journal">ACS Chem Biol</span>. 2017;<span class="ref-vol">12</span>: 2788&#x02013;2803. 10.1021/acschembio.7b00758
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5697452/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28945333" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Facschembio.7b00758" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=ACS+Chem+Biol&amp;title=Targeting+the+Hemopexin-like+Domain+of+Latent+Matrix+Metalloproteinase-9+(proMMP-9)+with+a+Small+Molecule+Inhibitor+Prevents+the+Formation+of+Focal+Adhesion+Junctions&amp;author=VM+Alford&amp;author=A+Kamath&amp;author=X+Ren&amp;author=K+Kumar&amp;author=Q+Gan&amp;volume=12&amp;publication_year=2017&amp;pages=2788-2803&amp;pmid=28945333&amp;doi=10.1021/acschembio.7b00758&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref023">23. <span class="mixed-citation">Hariono M, Nuwarda RF, Yusuf M, Rollando R, Jenie RI, Al-Najjar B, et al. 
<span class="ref-title">Arylamide as Potential Selective Inhibitor for Matrix Metalloproteinase 9 (MMP9): Design, Synthesis, Biological Evaluation, and Molecular Modeling</span>. <span class="ref-journal">J Chem Inf Model</span>. 2020;<span class="ref-vol">60</span>: 349&#x02013;359. 10.1021/acs.jcim.9b00630
 [<a href="https://pubmed.ncbi.nlm.nih.gov/31825614" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Facs.jcim.9b00630" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Chem+Inf+Model&amp;title=Arylamide+as+Potential+Selective+Inhibitor+for+Matrix+Metalloproteinase+9+(MMP9):+Design,+Synthesis,+Biological+Evaluation,+and+Molecular+Modeling&amp;author=M+Hariono&amp;author=RF+Nuwarda&amp;author=M+Yusuf&amp;author=R+Rollando&amp;author=RI+Jenie&amp;volume=60&amp;publication_year=2020&amp;pages=349-359&amp;pmid=31825614&amp;doi=10.1021/acs.jcim.9b00630&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref024">24. <span class="mixed-citation">Nishimiya D, Kawaguchi Y, Kodama S, Nasu H, Yano H, Yamaguchi A, et al. 
<span class="ref-title">A protein scaffold, engineered SPINK2, for generation of inhibitors with high affinity and specificity against target proteases</span>. <span class="ref-journal">Sci Rep</span>. 2019;<span class="ref-vol">9</span>: 11436
 10.1038/s41598-019-47615-5
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6686015/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31391482" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41598-019-47615-5" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci+Rep&amp;title=A+protein+scaffold,+engineered+SPINK2,+for+generation+of+inhibitors+with+high+affinity+and+specificity+against+target+proteases&amp;author=D+Nishimiya&amp;author=Y+Kawaguchi&amp;author=S+Kodama&amp;author=H+Nasu&amp;author=H+Yano&amp;volume=9&amp;publication_year=2019&amp;pages=11436&amp;pmid=31391482&amp;doi=10.1038/s41598-019-47615-5&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref025">25. <span class="mixed-citation">Itoh T, Tanioka M, Matsuda H, Nishimoto H, Yoshioka T, Suzuki R, et al. 
<span class="ref-title">Experimental metastasis is suppressed in MMP-9-deficient mice</span>. <span class="ref-journal">Clin Exp Metastasis</span>. 1999;<span class="ref-vol">17</span>: 177&#x02013;81. 10.1023/a:1006603723759
 [<a href="https://pubmed.ncbi.nlm.nih.gov/10411111" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1023%2Fa%3A1006603723759" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Exp+Metastasis&amp;title=Experimental+metastasis+is+suppressed+in+MMP-9-deficient+mice&amp;author=T+Itoh&amp;author=M+Tanioka&amp;author=H+Matsuda&amp;author=H+Nishimoto&amp;author=T+Yoshioka&amp;volume=17&amp;publication_year=1999&amp;pages=177-81&amp;pmid=10411111&amp;doi=10.1023/a:1006603723759&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref026">26. <span class="mixed-citation">Mondal S, Adhikari N, Banerjee S, Amin SA, Jha T. <span class="ref-title">Matrix metalloproteinase-9 (MMP-9) and its inhibitors in cancer: A minireview</span>. <span class="ref-journal">Eur J Med Chem</span>. 2020;<span class="ref-vol">194</span>: 112260
 10.1016/j.ejmech.2020.112260
 [<a href="https://pubmed.ncbi.nlm.nih.gov/32224379" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.ejmech.2020.112260" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Med+Chem&amp;title=Matrix+metalloproteinase-9+(MMP-9)+and+its+inhibitors+in+cancer:+A+minireview&amp;author=S+Mondal&amp;author=N+Adhikari&amp;author=S+Banerjee&amp;author=SA+Amin&amp;author=T+Jha&amp;volume=194&amp;publication_year=2020&amp;pages=112260&amp;pmid=32224379&amp;doi=10.1016/j.ejmech.2020.112260&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref027">27. <span class="mixed-citation">Rosenberg GA. <span class="ref-title">Matrix metalloproteinases in neuroinflammation</span>. <span class="ref-journal">Glia</span>. 2002;<span class="ref-vol">39</span>: 279&#x02013;91. 10.1002/glia.10108
 [<a href="https://pubmed.ncbi.nlm.nih.gov/12203394" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fglia.10108" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Glia&amp;title=Matrix+metalloproteinases+in+neuroinflammation&amp;author=GA+Rosenberg&amp;volume=39&amp;publication_year=2002&amp;pages=279-91&amp;pmid=12203394&amp;doi=10.1002/glia.10108&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref028">28. <span class="mixed-citation">Weiss N, Miller F, Cazaubon S, Couraud P-O. <span class="ref-title">The blood-brain barrier in brain homeostasis and neurological diseases</span>. <span class="ref-journal">Biochim Biophys Acta</span>. 2009;<span class="ref-vol">1788</span>: 842&#x02013;57. 10.1016/j.bbamem.2008.10.022
 [<a href="https://pubmed.ncbi.nlm.nih.gov/19061857" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.bbamem.2008.10.022" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochim+Biophys+Acta&amp;title=The+blood-brain+barrier+in+brain+homeostasis+and+neurological+diseases&amp;author=N+Weiss&amp;author=F+Miller&amp;author=S+Cazaubon&amp;author=P-O+Couraud&amp;volume=1788&amp;publication_year=2009&amp;pages=842-57&amp;pmid=19061857&amp;doi=10.1016/j.bbamem.2008.10.022&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref029">29. <span class="mixed-citation">Lin C, Wu W, Lu H, Li W, Bao Z, Wang Y, et al. 
<span class="ref-title">MMP-9 Inhibitor GM6001 Prevents the Development of ssTBI-Induced Parkinson&#x02019;s Disease via the Autophagy Pathway</span>. <span class="ref-journal">Cell Mol Neurobiol</span>. 2020. 
 10.1007/s10571-020-00933-z
 [<a href="https://pubmed.ncbi.nlm.nih.gov/32770297" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs10571-020-00933-z" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Mol+Neurobiol&amp;title=MMP-9+Inhibitor+GM6001+Prevents+the+Development+of+ssTBI-Induced+Parkinson&#x02019;s+Disease+via+the+Autophagy+Pathway&amp;author=C+Lin&amp;author=W+Wu&amp;author=H+Lu&amp;author=W+Li&amp;author=Z+Bao&amp;publication_year=2020&amp;pmid=32770297&amp;doi=10.1007/s10571-020-00933-z&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref030">30. <span class="mixed-citation">Kaplan A, Spiller KJ, Towne C, Kanning KC, Choe GT, Geber A, et al. 
<span class="ref-title">Neuronal matrix Metalloproteinase-9 is a determinant of selective Neurodegeneration</span>. <span class="ref-journal">Neuron</span>. 2014;<span class="ref-vol">81</span>: 333&#x02013;348. 10.1016/j.neuron.2013.12.009
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6015650/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24462097" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.neuron.2013.12.009" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Neuronal+matrix+Metalloproteinase-9+is+a+determinant+of+selective+Neurodegeneration&amp;author=A+Kaplan&amp;author=KJ+Spiller&amp;author=C+Towne&amp;author=KC+Kanning&amp;author=GT+Choe&amp;volume=81&amp;publication_year=2014&amp;pages=333-348&amp;pmid=24462097&amp;doi=10.1016/j.neuron.2013.12.009&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref031">31. <span class="mixed-citation">Mishiro K, Ishiguro M, Suzuki Y, Tsuruma K, Shimazawa M, Hara H. <span class="ref-title">A broad-spectrum matrix metalloproteinase inhibitor prevents hemorrhagic complications induced by tissue plasminogen activator in mice</span>. <span class="ref-journal">Neuroscience</span>. 2012;<span class="ref-vol">205</span>: 39&#x02013;48. 10.1016/j.neuroscience.2011.12.042
 [<a href="https://pubmed.ncbi.nlm.nih.gov/22244977" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.neuroscience.2011.12.042" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroscience&amp;title=A+broad-spectrum+matrix+metalloproteinase+inhibitor+prevents+hemorrhagic+complications+induced+by+tissue+plasminogen+activator+in+mice&amp;author=K+Mishiro&amp;author=M+Ishiguro&amp;author=Y+Suzuki&amp;author=K+Tsuruma&amp;author=M+Shimazawa&amp;volume=205&amp;publication_year=2012&amp;pages=39-48&amp;pmid=22244977&amp;doi=10.1016/j.neuroscience.2011.12.042&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref032">32. <span class="mixed-citation">Ishida Y, Kuninaka Y, Nosaka M, Kimura A, Taruya A, Furuta M, et al. 
<span class="ref-title">Prevention of CaCl2-induced aortic inflammation and subsequent aneurysm formation by the CCL3-CCR5 axis</span>. <span class="ref-journal">Nat Commun</span>. 2020;<span class="ref-vol">11</span>: 5994
 10.1038/s41467-020-19763-0
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7688638/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33239616" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41467-020-19763-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Commun&amp;title=Prevention+of+CaCl2-induced+aortic+inflammation+and+subsequent+aneurysm+formation+by+the+CCL3-CCR5+axis&amp;author=Y+Ishida&amp;author=Y+Kuninaka&amp;author=M+Nosaka&amp;author=A+Kimura&amp;author=A+Taruya&amp;volume=11&amp;publication_year=2020&amp;pages=5994&amp;pmid=33239616&amp;doi=10.1038/s41467-020-19763-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref033">33. <span class="mixed-citation">Rosenblum G, Van den Steen PE, Cohen SR, Grossmann JG, Frenkel J, Sertchook R, et al. 
<span class="ref-title">Insights into the Structure and Domain Flexibility of Full-Length Pro-Matrix Metalloproteinase-9/Gelatinase B</span>. <span class="ref-journal">Structure</span>. 2007;<span class="ref-vol">15</span>: 1227&#x02013;1236. 10.1016/j.str.2007.07.019
 [<a href="https://pubmed.ncbi.nlm.nih.gov/17937912" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.str.2007.07.019" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=Insights+into+the+Structure+and+Domain+Flexibility+of+Full-Length+Pro-Matrix+Metalloproteinase-9/Gelatinase+B&amp;author=G+Rosenblum&amp;author=PE+Van+den+Steen&amp;author=SR+Cohen&amp;author=JG+Grossmann&amp;author=J+Frenkel&amp;volume=15&amp;publication_year=2007&amp;pages=1227-1236&amp;pmid=17937912&amp;doi=10.1016/j.str.2007.07.019&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref034">34. <span class="mixed-citation">Beckett D, Kovaleva E, Schatz PJ. <span class="ref-title">A minimal peptide substrate in biotin holoenzyme synthetase-catalyzed biotinylation</span>. <span class="ref-journal">Protein Sci</span>. 1999;<span class="ref-vol">8</span>: 921&#x02013;929. 10.1110/ps.8.4.921
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2144313/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10211839" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1110%2Fps.8.4.921" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Sci&amp;title=A+minimal+peptide+substrate+in+biotin+holoenzyme+synthetase-catalyzed+biotinylation&amp;author=D+Beckett&amp;author=E+Kovaleva&amp;author=PJ+Schatz&amp;volume=8&amp;publication_year=1999&amp;pages=921-929&amp;pmid=10211839&amp;doi=10.1110/ps.8.4.921&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref035">35. <span class="mixed-citation">O&#x02019;Connell JP, Willenbrock F, Docherty AJ, Eaton D, Murphy G. <span class="ref-title">Analysis of the role of the COOH-terminal domain in the activation, proteolytic activity, and tissue inhibitor of metalloproteinase interactions of gelatinase B</span>. <span class="ref-journal">J Biol Chem</span>. 1994;<span class="ref-vol">269</span>: 14967&#x02013;73. Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/8195131" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/pubmed/8195131</a> [<a href="https://pubmed.ncbi.nlm.nih.gov/8195131" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Analysis+of+the+role+of+the+COOH-terminal+domain+in+the+activation,+proteolytic+activity,+and+tissue+inhibitor+of+metalloproteinase+interactions+of+gelatinase+B&amp;author=JP+O&#x02019;Connell&amp;author=F+Willenbrock&amp;author=AJ+Docherty&amp;author=D+Eaton&amp;author=G+Murphy&amp;volume=269&amp;publication_year=1994&amp;pages=14967-73&amp;pmid=8195131&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref036">36. <span class="mixed-citation">Kridel SJ, Chen E, Kotra LP, Howard EW, Mobashery S, Smith JW. <span class="ref-title">Substrate hydrolysis by matrix metalloproteinase-9</span>. <span class="ref-journal">J Biol Chem</span>. 2001;<span class="ref-vol">276</span>: 20572&#x02013;8. 10.1074/jbc.M100900200
 [<a href="https://pubmed.ncbi.nlm.nih.gov/11279151" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1074%2Fjbc.M100900200" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Substrate+hydrolysis+by+matrix+metalloproteinase-9&amp;author=SJ+Kridel&amp;author=E+Chen&amp;author=LP+Kotra&amp;author=EW+Howard&amp;author=S+Mobashery&amp;volume=276&amp;publication_year=2001&amp;pages=20572-8&amp;pmid=11279151&amp;doi=10.1074/jbc.M100900200&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref037">37. <span class="mixed-citation">Appleby TC, Greenstein AE, Hung M, Liclican A, Velasquez M, Villasenor AG, et al. 
<span class="ref-title">Biochemical characterization and structure determination of a potent, selective antibody inhibitor of human MMP9</span>. <span class="ref-journal">J Biol Chem</span>. 2017;<span class="ref-vol">292</span>: 6810&#x02013;6820. 10.1074/jbc.M116.760579
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5399127/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28235803" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1074%2Fjbc.M116.760579" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Biochemical+characterization+and+structure+determination+of+a+potent,+selective+antibody+inhibitor+of+human+MMP9&amp;author=TC+Appleby&amp;author=AE+Greenstein&amp;author=M+Hung&amp;author=A+Liclican&amp;author=M+Velasquez&amp;volume=292&amp;publication_year=2017&amp;pages=6810-6820&amp;pmid=28235803&amp;doi=10.1074/jbc.M116.760579&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref038">38. <span class="mixed-citation">Roeb E, Behrmann I, Gr&#x000f6;tzinger J, Breuer B, Matern S. <span class="ref-title">An MMP-9 mutant without gelatinolytic activity as a novel TIMP-1-antagonist</span>. <span class="ref-journal">FASEB J</span>. 2000;<span class="ref-vol">14</span>: 1671&#x02013;1673. 10.1096/fj.99-0947fje
 [<a href="https://pubmed.ncbi.nlm.nih.gov/10973913" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1096%2Ffj.99-0947fje" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FASEB+J&amp;title=An+MMP-9+mutant+without+gelatinolytic+activity+as+a+novel+TIMP-1-antagonist&amp;author=E+Roeb&amp;author=I+Behrmann&amp;author=J+Gr&#x000f6;tzinger&amp;author=B+Breuer&amp;author=S+Matern&amp;volume=14&amp;publication_year=2000&amp;pages=1671-1673&amp;pmid=10973913&amp;doi=10.1096/fj.99-0947fje&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref039">39. <span class="mixed-citation">Roderfeld M, Weiskirchen R, Wagner S, Berres M-L, Henkel C, Gr&#x000f6;tzinger J, et al. 
<span class="ref-title">Inhibition of hepatic fibrogenesis by matrix metalloproteinase-9 mutants in mice</span>. <span class="ref-journal">FASEB J</span>. 2006;<span class="ref-vol">20</span>: 444&#x02013;454. 10.1096/fj.05-4828com
 [<a href="https://pubmed.ncbi.nlm.nih.gov/16507762" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1096%2Ffj.05-4828com" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FASEB+J&amp;title=Inhibition+of+hepatic+fibrogenesis+by+matrix+metalloproteinase-9+mutants+in+mice&amp;author=M+Roderfeld&amp;author=R+Weiskirchen&amp;author=S+Wagner&amp;author=M-L+Berres&amp;author=C+Henkel&amp;volume=20&amp;publication_year=2006&amp;pages=444-454&amp;pmid=16507762&amp;doi=10.1096/fj.05-4828com&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref040">40. <span class="mixed-citation">Goffin L, Fagagnini S, Vicari A, Mamie C, Melhem H, Weder B, et al. 
<span class="ref-title">Anti-MMP-9 Antibody: A Promising Therapeutic Strategy for Treatment of Inflammatory Bowel Disease Complications with Fibrosis</span>. <span class="ref-journal">Inflamm Bowel Dis</span>. 2016;<span class="ref-vol">22</span>: 2041&#x02013;2057. 10.1097/MIB.0000000000000863
 [<a href="https://pubmed.ncbi.nlm.nih.gov/27542125" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1097%2FMIB.0000000000000863" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Inflamm+Bowel+Dis&amp;title=Anti-MMP-9+Antibody:+A+Promising+Therapeutic+Strategy+for+Treatment+of+Inflammatory+Bowel+Disease+Complications+with+Fibrosis&amp;author=L+Goffin&amp;author=S+Fagagnini&amp;author=A+Vicari&amp;author=C+Mamie&amp;author=H+Melhem&amp;volume=22&amp;publication_year=2016&amp;pages=2041-2057&amp;pmid=27542125&amp;doi=10.1097/MIB.0000000000000863&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref041">41. <span class="mixed-citation">Cheng M, De B, Pikul S, Almstead NG, Natchus MG, Anastasio MV, et al. 
<span class="ref-title">Design and synthesis of piperazine-based matrix metalloproteinase inhibitors</span>. <span class="ref-journal">J Med Chem</span>. 2000;<span class="ref-vol">43</span>: 369&#x02013;80. 10.1021/jm990366q
 [<a href="https://pubmed.ncbi.nlm.nih.gov/10669564" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fjm990366q" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Med+Chem&amp;title=Design+and+synthesis+of+piperazine-based+matrix+metalloproteinase+inhibitors&amp;author=M+Cheng&amp;author=B+De&amp;author=S+Pikul&amp;author=NG+Almstead&amp;author=MG+Natchus&amp;volume=43&amp;publication_year=2000&amp;pages=369-80&amp;pmid=10669564&amp;doi=10.1021/jm990366q&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref042">42. <span class="mixed-citation">Higashi S, Miyazaki K. <span class="ref-title">Identification of amino acid residues of the matrix metalloproteinase-2 essential for its selective inhibition by &#x003b2;-amyloid precursor protein-derived inhibitor</span>. <span class="ref-journal">J Biol Chem</span>. 2008;<span class="ref-vol">283</span>: 10068&#x02013;10078. 10.1074/jbc.M709509200
 [<a href="https://pubmed.ncbi.nlm.nih.gov/18238779" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1074%2Fjbc.M709509200" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Identification+of+amino+acid+residues+of+the+matrix+metalloproteinase-2+essential+for+its+selective+inhibition+by+&#x003b2;-amyloid+precursor+protein-derived+inhibitor&amp;author=S+Higashi&amp;author=K+Miyazaki&amp;volume=283&amp;publication_year=2008&amp;pages=10068-10078&amp;pmid=18238779&amp;doi=10.1074/jbc.M709509200&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref043">43. <span class="mixed-citation">Hashimoto H, Takeuchi T, Komatsu K, Miyazaki K, Sato M, Higashi S. <span class="ref-title">Structural basis for matrix metalloproteinase-2 (MMP-2)-selective inhibitory action of &#x003b2;-amyloid precursor protein-derived inhibitor</span>. <span class="ref-journal">J Biol Chem</span>. 2011;<span class="ref-vol">286</span>: 33236&#x02013;33243. 10.1074/jbc.M111.264176
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3190879/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21813640" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1074%2Fjbc.M111.264176" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Structural+basis+for+matrix+metalloproteinase-2+(MMP-2)-selective+inhibitory+action+of+&#x003b2;-amyloid+precursor+protein-derived+inhibitor&amp;author=H+Hashimoto&amp;author=T+Takeuchi&amp;author=K+Komatsu&amp;author=K+Miyazaki&amp;author=M+Sato&amp;volume=286&amp;publication_year=2011&amp;pages=33236-33243&amp;pmid=21813640&amp;doi=10.1074/jbc.M111.264176&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref044">44. <span class="mixed-citation">Tranchant I, Vera L, Czarny B, Amoura M, Cassar E, Beau F, et al. 
<span class="ref-title">Halogen bonding controls selectivity of FRET substrate probes for MMP-9</span>. <span class="ref-journal">Chem Biol</span>. 2014;<span class="ref-vol">21</span>: 408&#x02013;413. 10.1016/j.chembiol.2014.01.008
 [<a href="https://pubmed.ncbi.nlm.nih.gov/24583051" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.chembiol.2014.01.008" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem+Biol&amp;title=Halogen+bonding+controls+selectivity+of+FRET+substrate+probes+for+MMP-9&amp;author=I+Tranchant&amp;author=L+Vera&amp;author=B+Czarny&amp;author=M+Amoura&amp;author=E+Cassar&amp;volume=21&amp;publication_year=2014&amp;pages=408-413&amp;pmid=24583051&amp;doi=10.1016/j.chembiol.2014.01.008&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref045">45. <span class="mixed-citation">Nam DH, Rodriguez C, Remacle AG, Strongin AY, Ge X. <span class="ref-title">Active-site MMP-selective antibody inhibitors discovered from convex paratope synthetic libraries</span>. <span class="ref-journal">Proc Natl Acad Sci U S A</span>. 2016;<span class="ref-vol">113</span>: 14970&#x02013;14975. 10.1073/pnas.1609375114
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5206513/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27965386" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.1609375114" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Active-site+MMP-selective+antibody+inhibitors+discovered+from+convex+paratope+synthetic+libraries&amp;author=DH+Nam&amp;author=C+Rodriguez&amp;author=AG+Remacle&amp;author=AY+Strongin&amp;author=X+Ge&amp;volume=113&amp;publication_year=2016&amp;pages=14970-14975&amp;pmid=27965386&amp;doi=10.1073/pnas.1609375114&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref046">46. <span class="mixed-citation">Marshall DC, Lyman SK, McCauley S, Kovalenko M, Spangler R, Liu C, et al. 
<span class="ref-title">Selective allosteric inhibition of MMP9 is efficacious in preclinical models of ulcerative colitis and colorectal cancer</span>. <span class="ref-journal">PLoS One</span>. 2015;10
 10.1371/journal.pone.0127063
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4427291/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25961845" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pone.0127063" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Selective+allosteric+inhibition+of+MMP9+is+efficacious+in+preclinical+models+of+ulcerative+colitis+and+colorectal+cancer&amp;author=DC+Marshall&amp;author=SK+Lyman&amp;author=S+McCauley&amp;author=M+Kovalenko&amp;author=R+Spangler&amp;publication_year=2015&amp;pages=10&amp;pmid=25961845&amp;doi=10.1371/journal.pone.0127063&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref047">47. <span class="mixed-citation">Ciferri C, Lipari MT, Liang W-C, Estevez A, Hang J, Stawicki S, et al. 
<span class="ref-title">The trimeric serine protease HtrA1 forms a cage-like inhibition complex with an anti-HtrA1 antibody</span>. <span class="ref-journal">Biochem J</span>. 2015;<span class="ref-vol">472</span>: 169&#x02013;181. 10.1042/BJ20150601
 [<a href="https://pubmed.ncbi.nlm.nih.gov/26385991" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1042%2FBJ20150601" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+J&amp;title=The+trimeric+serine+protease+HtrA1+forms+a+cage-like+inhibition+complex+with+an+anti-HtrA1+antibody&amp;author=C+Ciferri&amp;author=MT+Lipari&amp;author=W-C+Liang&amp;author=A+Estevez&amp;author=J+Hang&amp;volume=472&amp;publication_year=2015&amp;pages=169-181&amp;pmid=26385991&amp;doi=10.1042/BJ20150601&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref048">48. <span class="mixed-citation">Scannevin RH, Alexander R, Haarlander TM, Burke SL, Singer M, Huo C, et al. 
<span class="ref-title">Discovery of a highly selective chemical inhibitor of matrix metalloproteinase-9 (MMP-9) that allosterically inhibits zymogen activation</span>. <span class="ref-journal">J Biol Chem</span>. 2017;<span class="ref-vol">292</span>: 17963&#x02013;17974. 10.1074/jbc.M117.806075
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5663893/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28860188" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1074%2Fjbc.M117.806075" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Discovery+of+a+highly+selective+chemical+inhibitor+of+matrix+metalloproteinase-9+(MMP-9)+that+allosterically+inhibits+zymogen+activation&amp;author=RH+Scannevin&amp;author=R+Alexander&amp;author=TM+Haarlander&amp;author=SL+Burke&amp;author=M+Singer&amp;volume=292&amp;publication_year=2017&amp;pages=17963-17974&amp;pmid=28860188&amp;doi=10.1074/jbc.M117.806075&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref049">49. <span class="mixed-citation">Morrison JF. <span class="ref-title">Kinetics of the reversible inhibition of enzyme-catalysed reactions by tight-binding inhibitors</span>. <span class="ref-journal">Biochim Biophys Acta</span>. 1969;<span class="ref-vol">185</span>: 269&#x02013;286. 10.1016/0005-2744(69)90420-3
 [<a href="https://pubmed.ncbi.nlm.nih.gov/4980133" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2F0005-2744(69)90420-3" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochim+Biophys+Acta&amp;title=Kinetics+of+the+reversible+inhibition+of+enzyme-catalysed+reactions+by+tight-binding+inhibitors&amp;author=JF+Morrison&amp;volume=185&amp;publication_year=1969&amp;pages=269-286&amp;pmid=4980133&amp;doi=10.1016/0005-2744(69)90420-3&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0244656.ref050">50. <span class="mixed-citation">Johnson K, Goody R. <span class="ref-title">The Original Michaelis Constant</span>. <span class="ref-journal">Biochemistry</span>. 2012;<span class="ref-vol">50</span>: 8264&#x02013;8269. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3381512/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21888353" ref="reftype=pubmed&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=The+Original+Michaelis+Constant&amp;author=K+Johnson&amp;author=R+Goody&amp;volume=50&amp;publication_year=2012&amp;pages=8264-8269&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7771667&amp;issue-id=369896&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">PLOS ONE</span> are provided here courtesy of <strong>PLOS</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC7771667/pdf/pone.0244656.pdf" class="int-view">PDF (2.2M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/7771667/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/7771667/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="7771667" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC7771667%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="9P6OqpmIJ7K1nwH2yokOWt9nndxiGcCAntLpGV4i1zZ2JgiOIq9h3j7YZ2UU6CHA">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC7771667%2F&amp;text=Discovery%20of%20potent%20and%20specific%20inhibitors%20targeting%20the%20active%20site%20of%20MMP-9%20from%20the%20engineered%20SPINK2%20library%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC7771667%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7771667/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC7771667/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/33373399/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC7771667/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/33373399/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC7771667/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/7771667/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript" src="/core/mathjax/2.7.9/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC7771667/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
