
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Epigenetic reader BRD4 supports mycobacterial pathogenesis by co-modulating host lipophagy and angiogenesis - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="322C300B24F39205000057AF09048B05.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="autoph" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC8942508/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8942508/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Autophagy" /><meta name="citation_title" content="Epigenetic reader BRD4 supports mycobacterial pathogenesis by co-modulating host lipophagy and angiogenesis" /><meta name="citation_author" content="Tanushree Mukherjee" /><meta name="citation_author_institution" content="Department of Microbiology and Cell Biology, Indian Institute of Science, Bengaluru, India" /><meta name="citation_author" content="Bharat Bhatt" /><meta name="citation_author_institution" content="Department of Microbiology and Cell Biology, Indian Institute of Science, Bengaluru, India" /><meta name="citation_author" content="Praveen Prakhar" /><meta name="citation_author_institution" content="Department of Microbiology and Cell Biology, Indian Institute of Science, Bengaluru, India" /><meta name="citation_author" content="Gaurav Kumar Lohia" /><meta name="citation_author_institution" content="Department of Microbiology and Cell Biology, Indian Institute of Science, Bengaluru, India" /><meta name="citation_author" content="R.S. Rajmani" /><meta name="citation_author_institution" content="Centre for Infectious Disease Research, Indian Institute of Science, Bengaluru, India" /><meta name="citation_author" content="Kithiganahalli Narayanaswamy Balaji" /><meta name="citation_author_institution" content="Department of Microbiology and Cell Biology, Indian Institute of Science, Bengaluru, India" /><meta name="citation_publication_date" content="2022" /><meta name="citation_issue" content="2" /><meta name="citation_volume" content="18" /><meta name="citation_firstpage" content="391" /><meta name="citation_doi" content="10.1080/15548627.2021.1936355" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC8942508/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC8942508/" /><meta name="citation_pmid" content="34074211" /><meta name="DC.Title" content="Epigenetic reader BRD4 supports mycobacterial pathogenesis by co-modulating host lipophagy and angiogenesis" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Taylor &#x0026; Francis" /><meta name="DC.Contributor" content="Tanushree Mukherjee" /><meta name="DC.Contributor" content="Bharat Bhatt" /><meta name="DC.Contributor" content="Praveen Prakhar" /><meta name="DC.Contributor" content="Gaurav Kumar Lohia" /><meta name="DC.Contributor" content="R.S. Rajmani" /><meta name="DC.Contributor" content="Kithiganahalli Narayanaswamy Balaji" /><meta name="DC.Date" content="2022" /><meta name="DC.Identifier" content="10.1080/15548627.2021.1936355" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Epigenetic reader BRD4 supports mycobacterial pathogenesis by co-modulating host lipophagy and angiogenesis" /><meta property="og:type" content="article" /><meta property="og:description" content="Mycobacterium tuberculosis (Mtb)-driven lipid accumulation is intricately associated with the progression of tuberculosis (TB) disease. Although several studies elucidating the mechanisms for lipid droplet (LD) biosynthesis exist, we provide evidence ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8942508/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC8942508/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/1359/">Autophagy</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/403716/">v.18(2); 2022</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC8942508
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC8942508/pdf/KAUP_18_1936355.pdf" class="int-view">PDF (8.4M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/8942508/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/8942508/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="8942508" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC8942508%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="Xpgq7VdB92E50VkXWmvWp4K60dB04d2ZvXLXexrEtTH1SucXWS4XD4lDJy7S7XHD">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC8942508%2F&amp;text=Epigenetic%20reader%20BRD4%20supports%20mycobacterial%20pathogenesis%20by%20co-modulating%20host%20lipophagy%20and%20angiogenesis" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC8942508%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8942508/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC8942508/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/34074211/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC8942508/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/34074211/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC8942508/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/8942508/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/1359/">Autophagy</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/403716/">v.18(2); 2022</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC8942508
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-autoph.gif" alt="Logo of autoph" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.tandfonline.com/loi/kaup20" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Autophagy.</span> 2022; 18(2): 391–408. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2021 Jun 28. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1080%2F15548627.2021.1936355" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1080/15548627.2021.1936355</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC8942508</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/34074211">34074211</a></div></div></div><h1 class="content-title">Epigenetic reader BRD4 supports mycobacterial pathogenesis by co-modulating host lipophagy and angiogenesis</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Mukherjee%20T%5BAuthor%5D" class="affpopup" co-rid="_co_idm140714695571376" co-class="co-affbox">Tanushree Mukherjee</a>,<sup>
a
,</sup><sup>
*
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Bhatt%20B%5BAuthor%5D" class="affpopup" co-rid="_co_idm140714695568352" co-class="co-affbox">Bharat Bhatt</a>,<sup>
a
,</sup><sup>
*
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Prakhar%20P%5BAuthor%5D" class="affpopup" co-rid="_co_idm140714696073728" co-class="co-affbox">Praveen Prakhar</a>,<sup>
a
,</sup><sup>
*
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Lohia%20GK%5BAuthor%5D" class="affpopup" co-rid="_co_idm140714696070912" co-class="co-affbox">Gaurav Kumar Lohia</a>,<sup>
a
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Rajmani%20R%5BAuthor%5D" class="affpopup" co-rid="_co_idm140714696068480" co-class="co-affbox">R.S. Rajmani</a>,<sup>
b
</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Balaji%20KN%5BAuthor%5D" class="affpopup" co-rid="_co_idm140714696066112" co-class="co-affbox">Kithiganahalli Narayanaswamy Balaji</a><sup>
a
</sup><sup></sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140714695571376"><h3 class="no_margin">Tanushree Mukherjee</h3><p><sup>a</sup>Department of Microbiology and Cell Biology, Indian Institute of Science, Bengaluru, India</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Mukherjee%20T%5BAuthor%5D">Tanushree Mukherjee</a></div></div><div id="_co_idm140714695568352"><h3 class="no_margin">Bharat Bhatt</h3><p><sup>a</sup>Department of Microbiology and Cell Biology, Indian Institute of Science, Bengaluru, India</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Bhatt%20B%5BAuthor%5D">Bharat Bhatt</a></div></div><div id="_co_idm140714696073728"><h3 class="no_margin">Praveen Prakhar</h3><p><sup>a</sup>Department of Microbiology and Cell Biology, Indian Institute of Science, Bengaluru, India</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Prakhar%20P%5BAuthor%5D">Praveen Prakhar</a></div></div><div id="_co_idm140714696070912"><h3 class="no_margin">Gaurav Kumar Lohia</h3><p><sup>a</sup>Department of Microbiology and Cell Biology, Indian Institute of Science, Bengaluru, India</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Lohia%20GK%5BAuthor%5D">Gaurav Kumar Lohia</a></div></div><div id="_co_idm140714696068480"><h3 class="no_margin">R.S. Rajmani</h3><p><sup>b</sup>Centre for Infectious Disease Research, Indian Institute of Science, Bengaluru, India</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Rajmani%20R%5BAuthor%5D">R.S. Rajmani</a></div></div><div id="_co_idm140714696066112"><h3 class="no_margin">Kithiganahalli Narayanaswamy Balaji</h3><p><sup>a</sup>Department of Microbiology and Cell Biology, Indian Institute of Science, Bengaluru, India</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Balaji%20KN%5BAuthor%5D">Kithiganahalli Narayanaswamy Balaji</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140714698025168_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140714698025168_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140714698025168_ai" style="display:none"><div class="fm-affl" id="aff0001"><sup>a</sup>Department of Microbiology and Cell Biology, Indian Institute of Science, Bengaluru, India</div><div class="fm-affl" id="aff0002"><sup>b</sup>Centre for Infectious Disease Research, Indian Institute of Science, Bengaluru, India</div><div id="an0001">CONTACT Kithiganahalli Narayanaswamy Balaji <a href="mailto:dev@null" data-email="ni.ca.csii@ijalab" class="oemail">ni.ca.csii@ijalab</a> Department of Microbiology and Cell Biology, Indian Institute of Science, Bengaluru, Karnataka
560012, India</div><div id="an0002"><sup>*</sup>These authors contributed equally to this work</div></div><div class="fm-article-notes half_rhythm"></div><div class="permissions half_rhythm hide" id="idm140714698025168_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; 2021 Informa UK Limited, trading as Taylor &#x00026; Francis Group</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplemental Material</div><div><a href="/pmc/articles/PMC8942508/bin/KAUP_A_1936355_SM1284.zip" data-ga-action="click_feat_suppl">KAUP_A_1936355_SM1284.zip</a><span style="color:gray"> (219M)</span></div><div class="small guid">GUID: 4916FF1A-9926-4ADA-B24A-B222DD4097A1</div></div></dd></dl></div><div id="abstract-a.u.b.r" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.u.b.rtitle">ABSTRACT</h2><!--article-meta--><div><p class="p p-first"><em>Mycobacterium tuberculosis</em> (Mtb)-driven lipid accumulation is intricately associated with the progression of tuberculosis (TB) disease. Although several studies elucidating the mechanisms for lipid droplet (LD) biosynthesis exist, we provide evidence for the significance of their regulated turnover via macroautophagy/autophagy during Mtb infection. We demonstrate that Mtb utilizes EGFR (epidermal growth factor receptor) signaling to induce the expression of the histone acetylation reader, BRD4 (bromodomain containing 4). The EGFR-BRD4 axis suppresses lipid-specific autophagy, and hence favors cellular lipid accumulation. Specifically, we found that pharmacological inhibition or knockdown of <em>Egfr</em> or <em>Brd4</em> enhances autophagic flux and concomitantly decreases cellular LDs that is otherwise maintained at a significant level in chloroquine-treated or <em>Atg5</em> knocked down autophagy-compromised host cells. In line with the enhanced lipophagy, we found that loss of EGFR or BRD4 function restricts mycobacterial burden that is rescued by external replenishment with oleic acid. We also report that the EGFR-BRD4 axis exerts additional effects by modulating pro-angiogenic gene expression and consequently aberrant angiogenesis during mycobacterial infection. This is important in the context of systemic Mtb dissemination as well as for the efficient delivery of anti-mycobacterial therapeutics to the Mtb-rich core of TB granuloma. Finally, utilizing an <em>in vivo</em> mouse model of TB, we show that pharmacological inhibition of EGFR and BRD4 compromises LD buildup via enhanced lipophagy and normalizes angiogenesis, thereby restricting Mtb burden and rescuing mice from severe TB-like pathology. These findings shed light on the novel roles of BRD4 during Mtb infection, and its possible implication in potentiating anti-TB responses.</p><p class="p p-last"><strong>Abbreviations</strong>: ATG5: autophagy related 5; BRDs: bromodomain containing; COL18A1: collagen type XVIII alpha 1 chain; EGFR: epidermal growth factor receptor; EP300: E1A binding protein p300; KDR: kinase insert domain receptor; KLF5: Kruppel like factor 5; LDs: lipid droplets; MAP1LC3B: microtubule associated protein 1 light chain 3 beta; Mtb: <em>Mycobacterium tuberculosis</em>; PECAM1: platelet and endothelial cell adhesion molecule 1; SQSTM1/p62: sequestosome 1; TB: tuberculosis; THBS1: thrombospondin 1; VEGF: vascular endothelial growth factor</p></div><div class="sec"><strong class="kwd-title">KEYWORDS: </strong><span class="kwd-text">Angiogenesis, bromodomain-containing protein BRD4, EGFR, lipid droplets, lipophagy, <em>Mycobacterium tuberculosis</em></span></div></div><div id="s0001" class="tsec sec"><h2 class="head no_bottom_margin" id="s0001title">Introduction</h2><p class="p p-first"><em>Mycobacterium tuberculosis</em> (Mtb) reprograms host cellular processes to the benefit of its pathogenesis. Among the various strategies, accumulation of lipids in the form of lipid droplets (LDs) forms an essential determinant of Mtb survival. The neutral lipids and cholesterol esters harbored in the LDs provide a source of nutrition, regulate immune responses, and define critical aspects such as Mtb dormancy and reactivation [<sup><a href="#cit0001" rid="cit0001" class=" bibr popnode">1&#x02013;3</a></sup>]. Therefore, it is imperative to understand the mechanisms underlying the formation of such lipid-laden foam cells, ranging from the transcriptional control of LD genes to processes that regulate lipid turnover such as autophagy.</p><p>Macroautophagy/autophagy is a highly regulated cell homeostatic process that has been shown to be co-opted by several pathogens, including Mtb, for their survival. A wide array of factors have been associated with the inhibition of this pathogen-clearance mechanism, including signaling pathways and miRNAs [<sup><a href="#cit0004" rid="cit0004" class=" bibr popnode">4&#x02013;7</a></sup>]. Emerging studies offer strong evidence for the implication of distinct epigenetic factors in the realm of tuberculosis (TB) infection. Histone methyl transferases (such as EZH2, KMT5A [<sup><a href="#cit0008" rid="cit0008" class=" bibr popnode">8</a></sup><sup>,</sup><sup><a href="#cit0009" rid="cit0009" class=" bibr popnode">9</a></sup>]), histone demethylases (KDM6B [<sup><a href="#cit0010" rid="cit0010" class=" bibr popnode">10</a></sup>]), histone acetyl transferases (such as KAT5 [<sup><a href="#cit0011" rid="cit0011" class=" bibr popnode">11</a></sup>]) and histone deacetylases (SIRT1 [<sup><a href="#cit0012" rid="cit0012" class=" bibr popnode">12</a></sup>]) have been associated with the modulation of discrete cellular phenotypes that dictate Mtb survival. However, the identification of the readers of such epigenetic marks that may contribute to pivotal events in the pathogenesis of Mtb has been obscure. In this perspective, BRD (bromodomain containing) proteins, recognizing specific histone acetylation marks to govern gene expression, have gained relevance in several physiological and pathological processes. Among the different BRDs (BRD2, BRD3, BRD4 and BRDT), rapidly accumulating evidence have shown the implication of BRD4 in modulating innate immune responses during infections with <em>L. monocytogenes</em>, herpes simplex virus, human papillomavirus as well as in murine models of induced inflammation and endotoxin shock [<sup><a href="#cit0013" rid="cit0013" class=" bibr popnode">13&#x02013;15</a></sup>]. In principle, the two bromodomains of BRD4 bind to two acetylated moieties on histone and/or non-histone proteins, including transcription factors, and recruit the necessary transcription machinery for modulating gene expression [<sup><a href="#cit0016" rid="cit0016" class=" bibr popnode">16</a></sup>]. To our interest, BRD4 has been reported to directly regulate lipogenesis [<sup><a href="#cit0017" rid="cit0017" class=" bibr popnode">17</a></sup>] and is also known to suppress autophagy [<sup><a href="#cit0018" rid="cit0018" class=" bibr popnode">18</a></sup>]. With this premise, we focused on elucidating the contribution of BRD4 in effectuating lipid accumulation during Mtb infection and its subsequent impact on TB pathogenesis.</p><p>At the systemic level, the granuloma forms as a result of structured congregation of immune cells at the site of infection that depicts a characteristic TB lesion [<sup><a href="#cit0019" rid="cit0019" class=" bibr popnode">19</a></sup>]. Interestingly, the core of the granuloma is generally abundant in LDs and displays a hypoxic environment [<sup><a href="#cit0020" rid="cit0020" class=" bibr popnode">20</a></sup>]. This hypoxia is known to drive angiogenic mechanisms in TB granulomas [<sup><a href="#cit0021" rid="cit0021" class=" bibr popnode">21</a></sup>]. Numerous studies pertaining to transcriptomics and serum analyses from Mtb-infected sources have identified the capacity of mycobacterial infection to induce angiogenic factors such as VEGFA (vascular endothelial growth factor A) and mediate blood vessel formation [<sup><a href="#cit0022" rid="cit0022" class=" bibr popnode">22</a></sup>]. However, a recent report demonstrated that TB-associated angiogenesis leads to the formation of inefficient blood vessels that strategically limit the access of immune cells and anti-TB therapeutics to the Mtb-harboring granuloma core [<sup><a href="#cit0023" rid="cit0023" class=" bibr popnode">23</a></sup>]. In this regard, the potential of anti-VEGF therapy and matrix remodelers have been harnessed for normalizing vasculature to enhance the delivery of small molecules into TB lesions [<sup><a href="#cit0024" rid="cit0024" class=" bibr popnode">24</a></sup><sup>,</sup><sup><a href="#cit0025" rid="cit0025" class=" bibr popnode">25</a></sup>]. These evidences emphasize on the importance of unraveling the intricate mechanisms and factors that may govern the progression of TB-associated angiogenesis.</p><p class="p p-last">We found that the epigenetic reader BRD4 is induced by Mtb in an EGFR (epidermal growth factor receptor) signaling-dependent manner, and it differentially regulates the expression of angiogenic genes, leading to ineffective angiogenesis. We also demonstrate the implication of EGFR and BRD4 in Mtb-driven LD generation and their sustenance in host cells via inhibition of lipophagy. Together, we report for the first time that Mtb utilizes EGFR-induced BRD4 to co-opt host cellular and immunological processes for successful pathogenesis. In corroboration, loss of function of EGFR or BRD4 was found to restore vasculature in the lung tissue while augmenting lipophagy and hence inhibiting LD accumulation at the cellular level. These outcomes directly assist in restricting Mtb burden <em>in vitro</em> and <em>in vivo</em>; thereby protecting mice from developing severe TB pathology in an <em>in vivo</em> therapeutic mouse model of TB. Besides, the current study highlights the pertinence of specific inhibitors of EGFR and BRD4 as probable adjuvants for anti-TB therapy.</p></div><div id="s0002" class="tsec sec"><h2 class="head no_bottom_margin" id="s0002title">Results</h2><div id="s0002-s2001" class="sec sec-first"><h3 id="s0002-s2001title">Mycobacteria-induced BRD4 facilitates TB pathogenesis <em>in vivo</em></h3><p class="p p-first-last">A therapeutic model of TB infection was utilized to study the effect of EGFR-BRD4 axis during Mtb pathogenesis in mice (<a href="/pmc/articles/PMC8942508/figure/f0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0001" rid-ob="ob-f0001" co-legend-rid="lgnd_f0001"><span>Figure 1A</span></a>). The transcript level of the BRD family members was assessed in the lungs of mice aerosol infected with Mtb H37Rv for 44 d, and among the tested, a significantly enhanced expression of BRD4 was observed (<a href="/pmc/articles/PMC8942508/figure/f0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0001" rid-ob="ob-f0001" co-legend-rid="lgnd_f0001"><span>Figure 1B</span></a>). In line, examination of the protein level of BRD4 by immunoblotting and immunofluorescence imaging in mouse peritoneal macrophages infected <em>in vitro</em> with Mtb H37Rv displayed its higher expression in infected samples compared to the uninfected controls (Fig. S1A and S1B). Corollary to this, the lung homogenates obtained from mice infected <em>in vivo</em> with Mtb H37Rv expressed BRD4 at an elevated level (<a href="/pmc/articles/PMC8942508/figure/f0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0001" rid-ob="ob-f0001" co-legend-rid="lgnd_f0001"><span>Figure 1C</span></a>), as was also confirmed in infected lung tissues by immunohistochemistry and immunofluorescence (<a href="/pmc/articles/PMC8942508/figure/f0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0001" rid-ob="ob-f0001" co-legend-rid="lgnd_f0001"><span>Figure 1D</span></a>,E). We next aimed to delineate the mechanism by which Mtb drives the expression of BRD4. Evidence from literature support the intricate correlation of BRD4 with EGFR in cellular processes relating to glioblastoma and ovarian cancers [<sup><a href="#cit0026" rid="cit0026" class=" bibr popnode">26</a></sup><sup>,</sup><sup><a href="#cit0027" rid="cit0027" class=" bibr popnode">27</a></sup>]. A study also identifies EGFR as a potential therapeutic target for TB [<sup><a href="#cit0027" rid="cit0027" class=" bibr popnode">27</a></sup>]. Therefore, we hypothesized if EGFR utilizes BRD4 in the realm of Mtb pathogenesis. To this end, first, we demonstrated that EGFR pathway is activated by Mtb H37Rv <em>in vitro</em> and <em>in vivo</em> (<a href="/pmc/articles/PMC8942508/figure/f0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0001" rid-ob="ob-f0001" co-legend-rid="lgnd_f0001"><span>Figure 1F</span></a>,G and S1C-E). siRNA-mediated knockdown of <em>Egfr</em> as well as its pharmacological inhibition using gefitinib compromised Mtb H37Rv-induced BRD4 expression, suggesting the possible implication of EGFR in driving BRD4-modulated immune functions during Mtb infection (<a href="/pmc/articles/PMC8942508/figure/f0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0001" rid-ob="ob-f0001" co-legend-rid="lgnd_f0001"><span>Figure 1H</span></a>, S1F and S1G). These findings prompted us to assess the relevance of Mtb-dependent epigenetic reader BRD4 and the receptor tyrosine kinase EGFR in an <em>in vivo</em> mouse model of tuberculosis (TB) (as depicted in <a href="/pmc/articles/PMC8942508/figure/f0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0001" rid-ob="ob-f0001" co-legend-rid="lgnd_f0001"><span>Figure 1A</span></a>). Utilizing respective pharmacological inhibitors gefitinib and JQ1, we found that perturbation of EGFR or BRD4 significantly compromises the formation of hallmark TB granulomatous lesions in the lungs of treated mice, as indicated by granuloma fraction and total granuloma score (<a href="/pmc/articles/PMC8942508/figure/f0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0001" rid-ob="ob-f0001" co-legend-rid="lgnd_f0001"><span>Figure 1I</span></a>,J and S1H: arrows indicate granulomatous lesions). With these lines of evidence, we first report the EGFR-dependent expression of BRD4 during Mtb infection and highlight their crucial contribution in the pathogenesis of TB disease.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f0001" co-legend-rid="lgnd_f0001"><a href="/pmc/articles/PMC8942508/figure/f0001/" target="figure" rid-figpopup="f0001" rid-ob="ob-f0001"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140714700377088"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is KAUP_A_1936355_F0001_OC.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is KAUP_A_1936355_F0001_OC.jpg" src="/pmc/articles/PMC8942508/bin/KAUP_A_1936355_F0001_OC.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140714700377088"><a target="object" rel="noopener" href="/pmc/articles/PMC8942508/figure/f0001/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_f0001"><div><a class="figpopup" href="/pmc/articles/PMC8942508/figure/f0001/" target="figure" rid-figpopup="f0001" rid-ob="ob-f0001">Figure 1.</a></div><!--caption a7--><div class="caption"><p>Epigenetic reader, BRD4, supports TB pathogenesis <em>in vivo</em>. (A) Schematic for the <em>in vivo</em> mouse TB model and therapeutic treatment regimen. (B) Lung homogenates from mice aerosol infected with Mtb H37Rv for 44 d were assessed for the expression of BRD family genes by qRT-PCR. (C-E) Lungs of mice infected with Mtb H37Rv for 44 d were assessed for the protein level expression of BRD4 by (C) immunoblotting in lung homogenates; (D) immunohistochemistry in infected lung sections and (E) by immunofluorescence imaging on infected lung cryosections. (F and G) The activation of EGFR was assessed (F) in the lung homogenates of 44 d-Mtb H37Rv-infected mice by immunoblotting, and (G) in the infected lung cryosections by immunofluorescence microscopy. (H) Mouse peritoneal macrophages were transfected with NT or <em>Egfr</em> siRNA and infected with Mtb H37Rv for 12&#x000a0;h. Whole cell lysates were assessed for BRD4 expression by immunoblotting. (I and J) H and E staining of lung sections of uninfected/infected/and inhibitor-treated mice (representative image, I; 40X; scale bar: 200&#x000a0;&#x003bc;m; G =&#x000a0;granulomatous lesion; AS&#x000a0;=&#x000a0;alveolar space) and the respective quantification of granuloma score (J, left panel) and granuloma fraction (J, right panel). All qRT-PCR data represents mean&#x000b1;S.E.M. and immunoblotting data is representative of three independent experiments. At least 3 mice were utilized for each analysis derived from lung tissues obtained following therapeutic TB model. Granuloma score and fraction were evaluated in total 8 mice from two independent experiments. Med, medium; *** p &#x0003c;&#x000a0;0.001; ****, p &#x0003c;&#x000a0;0.0001 (Student&#x02019;s t-test in B; One-way ANOVA in J; GraphPad Prism 8.0); scale bar for IF: 10&#x000a0;&#x003bc;m; Immunohistochemistry (IHC) magnification, 20X (upper panel; scale bar: 300&#x000a0;&#x003bc;m) and 40X (lower panel). ACTB was utilized as loading control.</p></div></div></div></div><div id="s0002-s2002" class="sec"><h3 id="s0002-s2002title">EGFR-BRD4 axis regulates autophagy during mycobacterial infection</h3><p class="p p-first-last">To elucidate the mechanism that enables EGFR-driven BRD4 to support Mtb infection, we initiated by assessing their contribution to autophagy as both have been independently reported as inhibitors of the process [<sup><a href="#cit0028" rid="cit0028" class=" bibr popnode">28</a></sup><sup>,</sup><sup><a href="#cit0029" rid="cit0029" class=" bibr popnode">29</a></sup>], and Mtb is also distinctly known to modulate its host to achieve the same [<sup><a href="#cit0030" rid="cit0030" class=" bibr popnode">30</a></sup><sup>,</sup><sup><a href="#cit0031" rid="cit0031" class=" bibr popnode">31</a></sup>]. Firstly, we analyzed a panel of autophagy-related genes for their transcriptional dependency on EGFR and BRD4 during Mtb H37Rv infection, without any significant correlation being found (Fig. S2A and S2B). However, we observed that pre-treatment of peritoneal macrophages with gefitinib or JQ1 augmented the autophagic flux, as may be appreciated with the accumulation of cleaved MAP1LC3B-II (microtubule associated protein 1 light chain 3 beta) and SQSTM1/p62 (sequestosome 1) in the presence of chloroquine (inhibitor of lysosomal acidification and cargo clearance) (<a href="/pmc/articles/PMC8942508/figure/f0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0002" rid-ob="ob-f0002" co-legend-rid="lgnd_f0002"><span>Figure 2A</span></a>,B). Corollary results were obtained <em>in vivo</em> from the lung homogenates of infected mice treated with respective EGFR and BRD4 inhibitors. Loss of SQSTM1 and accumulation of BECN1 and MAP1LC3B-II from gefitinib- and JQ1-treated lungs indicated an active autophagy (<a href="/pmc/articles/PMC8942508/figure/f0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0002" rid-ob="ob-f0002" co-legend-rid="lgnd_f0002"><span>Figure 2C</span></a>). Consequent to the augmented autophagy, we observed the mycobacterial burden to reduce in the presence of EGFR and BRD4 inhibitors (<a href="/pmc/articles/PMC8942508/figure/f0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0002" rid-ob="ob-f0002" co-legend-rid="lgnd_f0002"><span>Figure 2D</span></a>). Further, JQ1-dependent mycobacterial clearance could be reverted by blocking autophagy via <em>Atg5</em> (autophagy related 5) knockdown (<a href="/pmc/articles/PMC8942508/figure/f0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0002" rid-ob="ob-f0002" co-legend-rid="lgnd_f0002"><span>Figure 2E</span></a>). We also confirmed that JQ1-mediated restriction of Mtb H37Rv burden in macrophages does not arise from any direct anti-mycobacterial activity of JQ1 by monitoring the growth kinetics and CFU of Mtb H37Rv cultured in the presence of JQ1 (<a href="/pmc/articles/PMC8942508/figure/f0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0002" rid-ob="ob-f0002" co-legend-rid="lgnd_f0002"><span>Figure 2E</span></a> and S2C). These results indicate the possible role for host EGFR-BRD4 axis in favoring mycobacterial survival by suppressing the host cell function of autophagy.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f0002" co-legend-rid="lgnd_f0002"><a href="/pmc/articles/PMC8942508/figure/f0002/" target="figure" rid-figpopup="f0002" rid-ob="ob-f0002"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140714698109712"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is KAUP_A_1936355_F0002_B.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is KAUP_A_1936355_F0002_B.jpg" src="/pmc/articles/PMC8942508/bin/KAUP_A_1936355_F0002_B.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140714698109712"><a target="object" rel="noopener" href="/pmc/articles/PMC8942508/figure/f0002/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_f0002"><div><a class="figpopup" href="/pmc/articles/PMC8942508/figure/f0002/" target="figure" rid-figpopup="f0002" rid-ob="ob-f0002">Figure 2.</a></div><!--caption a7--><div class="caption"><p>EGFR-BRD4 axis suppresses autophagic flux. (A and B) Mouse peritoneal macrophages were infected Mtb H37Rv. Following 48&#x000a0;h, infected macrophages were treated with (A) gefitinib or (B) JQ1 for 12&#x000a0;h. Cells were then treated with chloroquine for 4&#x000a0;h where indicated, and whole cell lysates were analyzed for autophagy flux by immunoblotting. (C) The status of the indicated autophagy markers was assessed in the lung homogenates from the concerned sets of infected and treated mice (n&#x000a0;=&#x000a0;3 mice in each group). (D) Murine peritoneal macrophages were infected with Mtb H37Rv for 4&#x000a0;h, extracellular Mtb H37Rv were washed and cells were incubated with the indicated inhibitors for 48&#x000a0;h and 60&#x000a0;h. Cells were lysed and plated on 7H11 to estimate intracellular Mtb H37Rv burden. (E) Mid log phase Mtb H37Rv cultures were sub-cultured in the presence or absence of JQ1 and plated on 7H11 medium to enumerate corresponding CFU. (F) Mouse peritoneal macrophages were transfected with NT or <em>Atg5</em> siRNA as indicated. Transfected cells were infected with Mtb H37Rv for 60&#x000a0;h in the presence or absence of JQ1 and assessed for intracellular mycobacterial burden. NT, non-targeting; CFU, colony forming units; *, p &#x0003c;&#x000a0;0.05; **, p &#x0003c;&#x000a0;0.01; *** p &#x0003c;&#x000a0;0.001; ****, p &#x0003c;&#x000a0;0.0001; ns, not significant (Student&#x02019;s t-test in D, One-way ANOVA in E; GraphPad Prism 5.0, 9.0). ACTB was utilized as loading control.</p></div></div></div></div><div id="s0002-s2003" class="sec"><h3 id="s0002-s2003title">EGFR-dependent BRD4 couples Mtb-induced lipid accumulation and autophagy</h3><p class="p p-first-last">Literature evidence shows that BRD4 inhibition-induced autophagy does not mediate pathogen clearance, thereby suggesting the xenophagy arm to remain unaffected [<sup><a href="#cit0018" rid="cit0018" class=" bibr popnode">18</a></sup>]. However, our observation clearly demonstrates a compromised Mtb burden in the presence of JQ1, that is lost upon <em>Atg5</em> knockdown (<a href="/pmc/articles/PMC8942508/figure/f0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0002" rid-ob="ob-f0002" co-legend-rid="lgnd_f0002"><span>Figure 2F</span></a>), indicating toward the possible loss of survival factors for Mtb via autophagy. One of the key components of Mtb pathogenesis is the formation of LDs that provide a source of nutrition and a secure niche for the bacterium [<sup><a href="#cit0001" rid="cit0001" class=" bibr popnode">1</a></sup><sup>,</sup><sup><a href="#cit0002" rid="cit0002" class=" bibr popnode">2</a></sup>]. Since lipids are among the various specified cargoes for autophagy [<sup><a href="#cit0032" rid="cit0032" class=" bibr popnode">32</a></sup>], we assessed the status of LDs in the presence of EGFR and BRD4 inhibitors. We found that Mtb-driven LD accumulation (BODIPY 493/503) is compromised in the lung sections of mice treated with inhibitors of EGFR and BRD4 (<a href="/pmc/articles/PMC8942508/figure/f0003/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0003" rid-ob="ob-f0003" co-legend-rid="lgnd_f0003"><span>Figure 3A</span></a>). Further, the premise that LD accumulation and autophagy are highly interdependent processes led us to speculate the prowess of Mtb to utilize EGFR-dependent BRD4 in regulating autophagy, and in turn LD accumulation. <em>In vitro</em> analysis of the possible interaction by co-staining for LDs (using BODIPY 493/503) and autophagy (using MAP1LC3B immunostaining) yielded an inverse correlation between the two processes (<a href="/pmc/articles/PMC8942508/figure/f0003/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0003" rid-ob="ob-f0003" co-legend-rid="lgnd_f0003"><span>Figure 3B</span></a>,C). Further investigation revealed that the LDs associate with MAP1LC3B (autophagy marker) as early as 4&#x000a0;h of JQ1 treatment, suggesting the utilization of autophagy as a readily available machinery for the turnover of lipids (<a href="/pmc/articles/PMC8942508/figure/f0003/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0003" rid-ob="ob-f0003" co-legend-rid="lgnd_f0003"><span>Figure 3D</span></a>,E). Importantly, a screen for various lipid biosynthesis (<em>Dgat1, Dgat2, Fasn</em>), uptake (<em>Cd36</em>), LD coating (<em>Plin2</em>) and degradation (<em>Lpl, Lipa</em>) markers highlighted critical LD accumulation genes (<em>Cd36</em> and <em>Plin2</em>) to be expressed in EGFR-BRD4-dependent manner (Fig. S2D), indicating their sustained inhibition via transcriptional downregulation. This set of results demonstrate the EGFR-BRD4-dependent interaction of the processes of lipid accretion and autophagy.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f0003" co-legend-rid="lgnd_f0003"><a href="/pmc/articles/PMC8942508/figure/f0003/" target="figure" rid-figpopup="f0003" rid-ob="ob-f0003"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140714739910080"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is KAUP_A_1936355_F0003_OC.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is KAUP_A_1936355_F0003_OC.jpg" src="/pmc/articles/PMC8942508/bin/KAUP_A_1936355_F0003_OC.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140714739910080"><a target="object" rel="noopener" href="/pmc/articles/PMC8942508/figure/f0003/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_f0003"><div><a class="figpopup" href="/pmc/articles/PMC8942508/figure/f0003/" target="figure" rid-figpopup="f0003" rid-ob="ob-f0003">Figure 3.</a></div><!--caption a7--><div class="caption"><p>EGFR-dependent BRD4 couples Mtb-induced lipid accumulation and autophagy. (A) Lipid accumulation was observed in the lung cryosections of mice infected with Mtb H37Rv, and treated with gefitinib or JQ1 as indicated, using BODIPY 493/503 staining (n&#x000a0;=&#x000a0;3 mice in each group). (B and C) Murine peritoneal macrophages were pre-treated with gefitinib and JQ1 followed by 48&#x000a0;h infection with Mtb H37Rv. Lipid droplets were stained with BODIPY 493/503 and MAP1LC3B was immunostained and observed by confocal microscopy; (B) representative images and (C) the respective quantification. (D and E) Murine peritoneal macrophages were infected with tdTomato Mtb H37Rv for a duration of 60&#x000a0;h, with JQ1 treatment for 4&#x000a0;h, 6&#x000a0;h or 12&#x000a0;h prior to harvest, as indicated. Lipid accumulation and autophagy were assessed using BODIPY 493/503 and MAP1LC3B immunofluorescence, respectively; (D) representative image and (E) quantification. Med, medium; CTCF, corrected total cell fluorescence; *, p &#x0003c;&#x000a0;0.05; *** p &#x0003c;&#x000a0;0.001, ****, p &#x0003c;&#x000a0;0.0001; ns, not significant (one-way ANOVA in C; GraphPad Prism 5.0 &#x00026; two-way ANOVA in E; GraphPad Prism 8.0); scale bar: 5&#x000a0;&#x003bc;m.</p></div></div></div></div><div id="s0002-s2004" class="sec"><h3 id="s0002-s2004title">Lipophagy is subdued along EGFR-BRD4 axis during Mtb infection</h3><p class="p p-first-last">With our observations on inversely correlated LDs and autophagy and the colocalization of LDs with the autophagy marker MAP1LC3B, we investigated the possible clearance of lipids via the autophagic system upon gefitinib and JQ1 treatment by blocking autophagy with chloroquine (Fig. S3A and S3B) or by knocking down the key autophagy player <em>Atg5</em> (<a href="/pmc/articles/PMC8942508/figure/f0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0004" rid-ob="ob-f0004" co-legend-rid="lgnd_f0004"><span>Figure 4A</span></a> and S3C) [<sup><a href="#cit0033" rid="cit0033" class=" bibr popnode">33</a></sup>]. We found that compromising autophagic machinery reverses gefitinib- and JQ1-mediated reduction in LDs during Mtb infection, indicating toward lipophagy. The same was also observed in cells wherein <em>Egfr</em> and <em>Brd4</em> were knocked down alongside <em>Atg5</em> (<a href="/pmc/articles/PMC8942508/figure/f0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0004" rid-ob="ob-f0004" co-legend-rid="lgnd_f0004"><span>Figure 4B</span></a> and S3D). This was further validated using PLIN2-traffic light construct (mRFP1-EGFP-PLIN2) that forms a direct determinant of the lipophagy flux. We observed an accumulation of mRFP1-EGFP-PLIN2 in Mtb-infected cells, depicting LD formation. However, treatment with gefitinib and JQ1 showed only an increased RFP signal, thereby confirming an enhanced flux of the traffic light construct through the autophagy system leading to quenched acidic pH sensitive-GFP fluorescence (<a href="/pmc/articles/PMC8942508/figure/f0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0005" rid-ob="ob-f0005" co-legend-rid="lgnd_f0005"><span>Figure 5A</span></a>,B). Since lipids form an essential entity for the survival of Mtb [<sup><a href="#cit0001" rid="cit0001" class=" bibr popnode">1</a></sup><sup>,</sup><sup><a href="#cit0002" rid="cit0002" class=" bibr popnode">2</a></sup>], we propose that the observed restriction in Mtb burden in the presence of gefitinib and JQ1 (<a href="/pmc/articles/PMC8942508/figure/f0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0002" rid-ob="ob-f0002" co-legend-rid="lgnd_f0002"><span>Figure 2D</span></a>) may correspond to the lipophagy-active state of the host cells. To this end, we assessed mycobacterial burden in <em>Egfr-</em> and <em>Brd4</em>-knocked down macrophages, supplemented with an external source of lipids &#x000a0;(oleic acid). We found that providing external lipids partially rescued Mtb burden (<a href="/pmc/articles/PMC8942508/figure/f0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0005" rid-ob="ob-f0005" co-legend-rid="lgnd_f0005"><span>Figure 5C</span></a>), suggesting lipophagy to be a possible mechanism by which inhibition of EGFR-BRD4 axis limits mycobacterial burden. Together, these results demonstrate the ability of Mtb to repurpose EGFR and BRD4 to its survival benefit by inducing LD genes as well as by subduing lipophagy. Next, we aimed to determine the effect of these signaling components on Mtb pathogenesis <em>in vivo</em>. Lipid accumulation deregulates various homeostatic processes and is often associated with hypoxic microenvironments and the production of pro-angiogenic factors that eventually lead to aberrant blood vessel formation [<sup><a href="#cit0034" rid="cit0034" class=" bibr popnode">34</a></sup>], a feature critical for defining the progression of TB disease and for the dissemination of Mtb [<sup><a href="#cit0035" rid="cit0035" class=" bibr popnode">35</a></sup>]. In line, during Mtb infection, LD-rich foam cells majorly localize in the hypoxic core of the granuloma [<sup><a href="#cit0036" rid="cit0036" class=" bibr popnode">36</a></sup><sup>,</sup><sup><a href="#cit0037" rid="cit0037" class=" bibr popnode">37</a></sup>], thus offering a crucial link between LDs and blood vessel formation, and thereby prompting us to evaluate the contribution of EGFR-BRD4 pathway in TB-associated angiogenesis.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f0004" co-legend-rid="lgnd_f0004"><a href="/pmc/articles/PMC8942508/figure/f0004/" target="figure" rid-figpopup="f0004" rid-ob="ob-f0004"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140714701332944"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is KAUP_A_1936355_F0004_OC.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is KAUP_A_1936355_F0004_OC.jpg" src="/pmc/articles/PMC8942508/bin/KAUP_A_1936355_F0004_OC.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140714701332944"><a target="object" rel="noopener" href="/pmc/articles/PMC8942508/figure/f0004/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_f0004"><div><a class="figpopup" href="/pmc/articles/PMC8942508/figure/f0004/" target="figure" rid-figpopup="f0004" rid-ob="ob-f0004">Figure 4.</a></div><!--caption a7--><div class="caption"><p>Inhibition of EGFR-BRD4 induces lipophagy. (A) Murine peritoneal macrophages were transfected with NT or <em>Atg5</em> siRNA. The transfected cells were infected for 48&#x000a0;h with tdTomato Mtb H37Rv followed by 12&#x000a0;h treatment with gefitinib or JQ1 and assessed for lipophagy by staining for LDs with BODIPY 493/503. (B) Mouse peritoneal macrophages were transfected with NT, <em>Egfr</em> or <em>Brd4</em> siRNAs in the presence or absence of <em>Atg5</em> siRNA. Transfected cells were infected with Mtb H37Rv for 60&#x000a0;h and assessed for lipid droplet accumulation (BODIPY 493/503) and autophagy (MAP1LC3B) by immunofluorescence. Med, Medium; NT, non-targeting; Scale bar: 5&#x000a0;&#x003bc;m.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f0005" co-legend-rid="lgnd_f0005"><a href="/pmc/articles/PMC8942508/figure/f0005/" target="figure" rid-figpopup="f0005" rid-ob="ob-f0005"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140714739410704"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is KAUP_A_1936355_F0005_OC.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is KAUP_A_1936355_F0005_OC.jpg" src="/pmc/articles/PMC8942508/bin/KAUP_A_1936355_F0005_OC.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140714739410704"><a target="object" rel="noopener" href="/pmc/articles/PMC8942508/figure/f0005/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_f0005"><div><a class="figpopup" href="/pmc/articles/PMC8942508/figure/f0005/" target="figure" rid-figpopup="f0005" rid-ob="ob-f0005">Figure 5.</a></div><!--caption a7--><div class="caption"><p>Inhibition of EGFR-BRD4 leads to mycobacterial clearance via LD depletion. (A and B) RAW 264.7 macrophages were transfected with mRFP1-EGFP-PLIN2 construct using Lipofectamine 3000 reagent, along with NT or <em>Atg5</em> siRNA. The transfected cells were infected for 48&#x000a0;h with Mtb H37Rv followed by 12&#x000a0;h treatment with gefitinib or JQ1 and assessed for lipophagy by analyzing the colocalization of RFP and GFP signals; (A) representative images; (B) respective quantification. (C) Mouse peritoneal macrophages were transfected with NT or <em>Egfr</em> or <em>Brd4</em> siRNA, in the presence or absence of oleic acid, and subsequently infected with Mtb H37Rv for 60&#x000a0;. Cells were lysed and plated on 7H11 medium to assess mycobacterial burden. Med, Medium; NT, non-targeting; *, p &#x0003c;&#x000a0;0.05; **, p &#x0003c;&#x000a0;0.01; ***, p &#x0003c;&#x000a0;0.001; ****, p &#x0003c;&#x000a0;0.0001 (one-way ANOVA, GraphPad PRISM 9.0).</p></div></div></div></div><div id="s0002-s2005" class="sec"><h3 id="s0002-s2005title">BRD4 effectuates Mtb-driven angiogenesis</h3><p class="p p-first-last">The formation of blood vessels occurs from the concerted action of several pro- and anti-angiogenic genes. Firstly, we inspected a panel of such genes for their differential regulation during Mtb infection <em>in vivo</em>. We found a significantly enhanced expression of pro-angiogenic (<em>Vegfa, Kdr, Tie1, Tek, Mmp2</em>) and anti-angiogenic makers in the lungs of mice infected <em>in vivo</em> with Mtb H37Rv (<a href="/pmc/articles/PMC8942508/figure/f0006/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0006" rid-ob="ob-f0006" co-legend-rid="lgnd_f0006"><span>Figure 6A</span></a>). Substantiating the same, an elevated protein level of the pro-angiogenic factor VEGFA was observed in the lung homogenates of infected mice (<a href="/pmc/articles/PMC8942508/figure/f0006/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0006" rid-ob="ob-f0006" co-legend-rid="lgnd_f0006"><span>Figure 6B</span></a>), with concomitantly augmented PECAM1 (platelet and endothelial cell adhesion molecule 1) staining in lung tissues demarcating endothelial cells constituting the blood vessels (<a href="/pmc/articles/PMC8942508/figure/f0006/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0006" rid-ob="ob-f0006" co-legend-rid="lgnd_f0006"><span>Figure 6C</span></a>). With this premise, we utilized the pharmacological inhibitor of BRD4 (JQ1) and <em>Brd4</em> siRNA and found that the expression of pro-angiogenic genes (such as <em>Vegfa</em>) was strongly dependent on BRD4 function (Fig. S4A-C); with no evident effect on the anti-angiogenic markers. In line with these findings, a significant upregulation of <em>Vegfa</em> and <em>Kdr</em> transcripts was observed in RAW 264.7 macrophages overexpressing BRD4 WT construct, without any effect on the levels of the anti-angiogenic markers <em>Thbs1</em> (thrombospondin 1) and <em>Col18a1</em> (collagen type XVIII alpha 1 chain) (Fig. S4D). Further, we found EGFR-BRD4 axis to regulate the expression of pro-angiogenic genes in the lung homogenates and tissue sections obtained from mice subjected to our <em>in vivo</em> therapeutic model of TB (<a href="/pmc/articles/PMC8942508/figure/f0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0001" rid-ob="ob-f0001" co-legend-rid="lgnd_f0001"><span>Figure 1A</span></a>) by qRT-PCR, ELISA and immunofluorescence assays (<a href="/pmc/articles/PMC8942508/figure/f0006/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0006" rid-ob="ob-f0006" co-legend-rid="lgnd_f0006"><span>Figure 6D</span></a>&#x02013;F). These results indicate an intricate function of BRD4 in differentially regulating angiogenic genes during Mtb infection. Importantly, this aligns with studies that correlate lipid accumulation with an increased expression of pro-angiogenic factors such as VEGF and HGF (hepatocyte growth factor) [<sup><a href="#cit0038" rid="cit0038" class=" bibr popnode">38</a></sup>].</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f0006" co-legend-rid="lgnd_f0006"><a href="/pmc/articles/PMC8942508/figure/f0006/" target="figure" rid-figpopup="f0006" rid-ob="ob-f0006"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140714695537440"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is KAUP_A_1936355_F0006_OC.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is KAUP_A_1936355_F0006_OC.jpg" src="/pmc/articles/PMC8942508/bin/KAUP_A_1936355_F0006_OC.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140714695537440"><a target="object" rel="noopener" href="/pmc/articles/PMC8942508/figure/f0006/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_f0006"><div><a class="figpopup" href="/pmc/articles/PMC8942508/figure/f0006/" target="figure" rid-figpopup="f0006" rid-ob="ob-f0006">Figure 6.</a></div><!--caption a7--><div class="caption"><p>BRD4 modulates angiogenic genes during mycobacterial infection. (A) The transcript levels of the indicated angiogenic genes were assessed in the lung homogenates of mice infected for 44 d with Mtb H37Rv. (B) The protein level of VEGFA was analyzed in Mtb H37Rv-infected lung homogenates by ELISA. (C) The expression of PECAM1 was ascertained by immunofluorescence imaging of infected lung sections. (D) Lung homogenates of infected mice treated with gefitinib or JQ1 was analyzed for the expression of the indicated angiogenic markers by qRT-PCR. (E) The protein level of VEGFA was assessed in the lung homogenates of the indicated groups of mice by ELISA. (F) The expression of PECAM1 was assessed in the lung cryosections of Mtb H37Rv-infected and gefitinib/JQ1-treated mice. At least 3 sets of mice were utilized for each analysis derived from lung tissues obtained following therapeutic TB model and qRT-PCR data represents mean&#x000b1;S.E.M. from five sets of mice. Med, medium; *, p &#x0003c;&#x000a0;0.05; **, p &#x0003c;&#x000a0;0.005; ***, p &#x0003c;&#x000a0;0.001; ****, p &#x0003c;&#x000a0;0.0001 (Students t-test in A, B and one-way ANOVA in D, E; GraphPad Prism 5.0 and 8.0); ns, not significant; scale bar: 10&#x000a0;&#x003bc;m.</p></div></div></div></div><div id="s0002-s2006" class="sec"><h3 id="s0002-s2006title">KLF5 mediates BRD4-dependent modulation of angiogenic genes</h3><p class="p p-first-last">The transcriptional regulation of Mtb-induced <em>Vegfa</em> and <em>Kdr</em> by the epigenetic mark reader BRD4 requires BRD4-driven assembly of transcription machinery at the concerned promoters. As introduced, BRD4 influences gene expression by interacting with acetylated moieties of histones or transcription factors. Among several possible targets, certain pieces of evidence suggest that KLF5 (Kruppel like factor 5) is activated by EP300 (E1A binding protein p300)-mediated acetylation [<sup><a href="#cit0039" rid="cit0039" class=" bibr popnode">39</a></sup>]; and is involved in the modulation of <em>Vegfa</em> expression and angiogenesis [<sup><a href="#cit0040" rid="cit0040" class=" bibr popnode">40</a></sup>]. We found that KLF5 is expressed in the lungs of mice infected with Mtb H37Rv (<a href="/pmc/articles/PMC8942508/figure/f0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0007" rid-ob="ob-f0007" co-legend-rid="lgnd_f0007"><span>Figure 7A</span></a>,B), and <em>in vitro</em> analysis indicated its expression to depend on Mtb-activated EGFR signaling (<a href="/pmc/articles/PMC8942508/figure/f0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0007" rid-ob="ob-f0007" co-legend-rid="lgnd_f0007"><span>Figure 7C</span></a>,D). Loss of KLF5 by siRNA-mediated knockdown compromised the transcript-level expression of the concerned pro-angiogenic markers, <em>Vegfa</em> and <em>kdr</em> (<a href="/pmc/articles/PMC8942508/figure/f0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0007" rid-ob="ob-f0007" co-legend-rid="lgnd_f0007"><span>Figure 7E</span></a>). Akin to BRD4, KLF5 also showed no impact on the expression of anti-angiogenic genes <em>Thbs1</em> and <em>Col18a1</em>. With these observations, we surmised the intersection of KLF5 and BRD4 to shape Mtb-induced angiogenic dysregulation. In this context, KLF5 was found to interact with EP300 and undergo acetylation during Mtb infection. Further, it was observed that Mtb favored the interaction of BRD4 and acetylated KLF5 (<a href="/pmc/articles/PMC8942508/figure/f0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0007" rid-ob="ob-f0007" co-legend-rid="lgnd_f0007"><span>Figure 7F</span></a>). Congruently, we could observe an enhanced recruitment of BRD4 and associated H3K27 acetylation mark at KLF5-binding regions on the promoters of the indicated pro-angiogenic markers, <em>Vegfa</em> and <em>Kdr</em> (<a href="/pmc/articles/PMC8942508/figure/f0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0007" rid-ob="ob-f0007" co-legend-rid="lgnd_f0007"><span>Figure 7G</span></a>). Sequential ChIP showing high co-occupancy of the promoters of <em>Vegfa</em> and <em>Kdr</em> with BRD4 and KLF5 further substantiated our findings. Such co-occupancy was partly compromised on the promoters of anti-angiogenic genes, <em>Col18a1</em> and <em>Thbs1</em> (<a href="/pmc/articles/PMC8942508/figure/f0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0007" rid-ob="ob-f0007" co-legend-rid="lgnd_f0007"><span>Figure 7H</span></a>). These results indicate the coordinated action of KLF5 and BRD4 in differentially regulating Mtb-induced angiogenic progression. Alongside, we assessed the possible contribution of KLF5 to lipophagy. We found that the loss of KLF5 function disrupts lipid droplet accumulation, however, majorly independent of autophagic flux. This is because knocking down <em>Atg5</em> did not restore <em>Klf5</em> knockdown-dependent reduction in LDs (Fig. S5A and S5B), indicating possible alternate mechanisms of lipid accumulation via KLF5 and the specificity of EGFR-BRD4 axis in subduing lipophagy in this context.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f0007" co-legend-rid="lgnd_f0007"><a href="/pmc/articles/PMC8942508/figure/f0007/" target="figure" rid-figpopup="f0007" rid-ob="ob-f0007"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140714696001728"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is KAUP_A_1936355_F0007_OC.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is KAUP_A_1936355_F0007_OC.jpg" src="/pmc/articles/PMC8942508/bin/KAUP_A_1936355_F0007_OC.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140714696001728"><a target="object" rel="noopener" href="/pmc/articles/PMC8942508/figure/f0007/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_f0007"><div><a class="figpopup" href="/pmc/articles/PMC8942508/figure/f0007/" target="figure" rid-figpopup="f0007" rid-ob="ob-f0007">Figure 7.</a></div><!--caption a7--><div class="caption"><p>Mtb-dependent KLF5 assists BRD4-mediated regulation of angiogenic markers. (A and B) KLF5 protein expression was assessed (A) in the lung homogenates of 44 d-Mtb H37Rv-infected mice by immunoblotting and (B) in the lung cryosections of infected mice by immunofluorescence imaging (n&#x000a0;=&#x000a0;3 mice in each group). (C) Murine peritoneal macrophages were pre-treated with EGFR inhibitor gefitinib for 1&#x000a0;h, followed by 12&#x000a0;h infection with Mtb H37Rv; and whole cell lysates were assessed for the expression of KLF5 by immunoblotting. (D) Mouse peritoneal macrophages were transfected with NT or <em>Egfr</em> siRNA and infected with Mtb H37Rv for 12&#x000a0;h. Whole cell lysates were assessed for KLF5 expression by immunoblotting. (E) Murine peritoneal macrophages were transfected with <em>Klf5</em> siRNA or NT siRNA for 24&#x000a0;h. Transfected cells were infected with Mtb H37Rv for 12&#x000a0;h and assessed for the expression of the indicated angiogenic genes. (F) Murine peritoneal macrophages were infected with Mtb H37Rv for 12&#x000a0;h and assessed for the interaction of BRD4, EP300 and KLF5 and KLF5 acetylation by immunoprecipitation assay. (G) The recruitment of BRD4 and corresponding H3K27Ac marks were analyzed on the promoters of the indicated genes (at KLF5 binding regions) by ChIP. (H) Sequential ChIP was performed to access the co-occupancy of the indicated promoters by KLF5 and BRD4 in uninfected and Mtb H37Rv-infected peritoneal macrophages. All qRT-PCR data represents mean&#x000b1;S.E.M. and immunoblotting data is representative of three independent experiments. Med, medium; *, p &#x0003c;&#x000a0;0.05; **, p &#x0003c;&#x000a0;0.005; *** p &#x0003c;&#x000a0;0.001 (Students t-test in H and one-way ANOVA in E, G; GraphPad Prism 5.0); ns, not significant; scale bar: 5&#x000a0;&#x003bc;m. ACTB was utilized as loading control.</p></div></div></div></div><div id="s0002-s2007" class="sec"><h3 id="s0002-s2007title">EGFR-PI3K-HIPPO axis drives BRD4-dependent angiogenic deregulation during Mtb infection</h3><p class="p p-first">Having observed the interaction of BRD4 and KLF5 to modulate angiogenic genes, we delineated the mechanism adopted by Mtb-driven EGFR pathway to regulate the expression of the aforesaid factors. EGFR utilizes several adaptors and signal transducers to alter cellular gene expression [<sup><a href="#cit0041" rid="cit0041" class=" bibr popnode">41</a></sup><sup>,</sup><sup><a href="#cit0042" rid="cit0042" class=" bibr popnode">42</a></sup>]. In an attempt to identify novel intermediates that lead to the expression of BRD4, we speculated the implication for HIPPO pathway as HIPPO is reported to be repurposed by Mtb for the expression of chemokines and inflammatory cytokines [<sup><a href="#cit0043" rid="cit0043" class=" bibr popnode">43</a></sup>]; and has also been independently reported as a regulator of angiogenesis [<sup><a href="#cit0044" rid="cit0044" class=" bibr popnode">44</a></sup><sup>,</sup><sup><a href="#cit0045" rid="cit0045" class=" bibr popnode">45</a></sup>]. In this context, we found that Mtb-induced EGFR signaling regulated HIPPO pathway, as perturbation of EGFR through pharmacological inhibitors compromised the ability of Mtb to effectuate HIPPO signaling activation in Mtb H37Rv-infected macrophages (<a href="/pmc/articles/PMC8942508/figure/f0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0008" rid-ob="ob-f0008" co-legend-rid="lgnd_f0008"><span>Figure 8A</span></a>). Consisting of a series of serine threonine kinases, STK4 (serine/threonine kinase 4)-STK3 cannot be directly activated by the receptor tyrosine kinase, EGFR. Canonical downstream EGFR signaling adaptors PI3K and MTOR [<sup><a href="#cit0046" rid="cit0046" class=" bibr popnode">46</a></sup>] were found to regulate HIPPO; as Mtb failed to activate HIPPO pathway components STK4-STK3 and LATS1 (large tumor suppressor kinase 1) in the presence of respective pharmacological inhibitors LY294002 (a PI3K and PtdIns3K inhibitor) and rapamycin (MTOR inhibitor) (<a href="/pmc/articles/PMC8942508/figure/f0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0008" rid-ob="ob-f0008" co-legend-rid="lgnd_f0008"><span>Figure 8B</span></a>). Further, we found BRD4 and KLF5 to be expressed along the PI3K-MTOR-HIPPO signaling axis; as perturbation of either component led to a compromised expression of BRD4 and KLF5 in the presence of Mtb H37Rv infection (<a href="/pmc/articles/PMC8942508/figure/f0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0008" rid-ob="ob-f0008" co-legend-rid="lgnd_f0008"><span>Figure 8C</span></a>&#x02013;E).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f0008" co-legend-rid="lgnd_f0008"><a href="/pmc/articles/PMC8942508/figure/f0008/" target="figure" rid-figpopup="f0008" rid-ob="ob-f0008"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140714697227408"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is KAUP_A_1936355_F0008_OC.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is KAUP_A_1936355_F0008_OC.jpg" src="/pmc/articles/PMC8942508/bin/KAUP_A_1936355_F0008_OC.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140714697227408"><a target="object" rel="noopener" href="/pmc/articles/PMC8942508/figure/f0008/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_f0008"><div><a class="figpopup" href="/pmc/articles/PMC8942508/figure/f0008/" target="figure" rid-figpopup="f0008" rid-ob="ob-f0008">Figure 8.</a></div><!--caption a7--><div class="caption"><p>EGFR-PI3K-HIPPO axis regulates angiogenesis markers during mycobacterial infection. (A) Murine peritoneal macrophages were pre-treated with gefitinib and AG1478, followed by 1&#x000a0;h infection with Mtb H37Rv. Whole cell lysates were assessed for the activation of the indicated HIPPO pathway components by immunoblotting. (B) Peritoneal macrophages were pre-treated with LY294002 (PI3K and PtdIns3K inhibitor) or rapamycin (MTOR inhibitor), followed by 1&#x000a0;h infection with Mtb H37Rv and assessed for the activation of Hippo pathway by immunoblotting. (C) Peritoneal macrophages were pre-treated with LY294002 or rapamycin, followed by 12&#x000a0;h infection with Mtb H37Rv and assessed for the expression of KLF5 and BRD4 by immunoblotting. (D) RAW 264.7 macrophages were transfected with the indicated constructs (PIK3R/p85 DN, PIK3C/p110 OE) and transfected cells were infected with Mtb H37Rv for 12&#x000a0;h. Whole cell lysates were assessed for the expression of KLF5 and BRD4 by immunoblotting. (E) Peritoneal macrophages were transfected with <em>Stk4</em> and <em>Stk3</em> siRNA, <em>Lats1</em> siRNAs&#x000a0;or NT siRNA for 24&#x000a0;h followed by 12&#x000a0;h infection with Mtb H37Rv; and assessed for (E, left panel) the protein levels of KLF5, BRD4; and (E, right panel) knockdown of <em>Stk4, Stk3</em> and <em>Lats1</em>, by immunoblotting. (F) The expression of the indicated angiogenic genes was analyzed in the murine peritoneal macrophages pre-treated with gefitinib for 1&#x000a0;h and followed by 12&#x000a0;h infection with Mtb H37Rv. (G) Peritoneal macrophages were pre-treated with LY294002 or rapamycin, followed by 12&#x000a0;h infection with Mtb H37Rv and assessed for the expression of the indicated angiogenesis genes by qRT-PCR. (H) Peritoneal macrophages were transfected with <em>Stk4</em> and <em>Stk3</em> siRNA, <em>Lats1</em> siRNA or NT siRNA for 24&#x000a0;h followed by 12&#x000a0;h infection with Mtb H37Rv; and assessed for the expression of angiogenesis genes by qRT-PCR. All qRT-PCR data represents mean&#x000b1;S.E.M. and immunoblotting data is representative of three independent experiments. NT, non-targeting; Med, medium; DN, dominant-negative; OE, overexpression; *, p &#x0003c;&#x000a0;0.05; **, p &#x0003c;&#x000a0;0.005; *** p &#x0003c;&#x000a0;0.001; ns, not significant (one-way ANOVA in F-H; GraphPad Prism 5.0); ACTB was utilized as loading control.</p></div></div></div><p class="p p-last">Having the premise that EGFR integrates the activation of HIPPO signaling; its further role in Mtb-induced angiogenic deregulation was surmised. It was found that EGFR-PI3K-MTOR-HIPPO axis plays a critical role in defining angiogenic gene expression during Mtb H37Rv infection. Abrogation of EGFR-PI3K-MTOR using specific inhibitors; and the HIPPO pathway using siRNA-mediated knockdown; suppressed the ability of Mtb to drive <em>Vegfa</em> and <em>Kdr</em> expression (<a href="/pmc/articles/PMC8942508/figure/f0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0008" rid-ob="ob-f0008" co-legend-rid="lgnd_f0008"><span>Figure 8F</span></a>&#x02013;H). Thus, the process of Mtb granuloma-associated angiogenesis was found to be finely regulated by BRD4 and KLF5 along the EGFR-PI3K-MTOR-HIPPO signaling axis.</p></div><div id="s0002-s2008" class="sec sec-last"><h3 id="s0002-s2008title">Perturbation of EGFR and BRD4 restricts mycobacterial survival <em>in vivo</em></h3><p class="p p-first-last">The observation that EGFR-dependent BRD4 governs the expression of angiogenic markers, LD accumulation and autophagy <em>in vitro</em> and <em>in vivo</em>; led us to identify their role in Mtb survival within the host by utilizing mouse model of TB infection (depicted in <a href="/pmc/articles/PMC8942508/figure/f0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0001" rid-ob="ob-f0001" co-legend-rid="lgnd_f0001"><span>Figure 1A</span></a>). Firstly, we assessed the formation of PECAM1-lined normal blood vessels during Mtb infection. This is because excessive VEGFA production is associated with pathological angiogenesis characterized by constricted and blocked blood vessels. This feature is considered particularly beneficial for Mtb as inefficient blood flow through such aberrant vessels would keep immune cells and anti-mycobacterial therapeutics from reaching the Mtb-containing core of the granuloma. It was observed that PECAM1- stained endothelial cells that demarcate the lining of blood vessels displayed empty or blood-filled spaces in uninfected and inhibitor-treated mice, as indicated by the red arrows (<a href="/pmc/articles/PMC8942508/figure/f0009/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0009" rid-ob="ob-f0009" co-legend-rid="lgnd_f0009"><span>Figure 9A</span></a>). However, infection of mice with Mtb H37Rv led to concentrated PECAM1 staining, without the presence of any noticeable lumen, depicting abnormal and inefficient vasculature. Thus, treatment of mice with gefitinib or JQ1 restored blood vessel architecture as quantified in <a href="/pmc/articles/PMC8942508/figure/f0009/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0009" rid-ob="ob-f0009" co-legend-rid="lgnd_f0009"><span>Figure 9B</span></a>. Also, as speculated, inhibition of EGFR and BRD4 inhibited LD accumulation and augmented autophagy in the lung sections of infected mice (<a href="/pmc/articles/PMC8942508/figure/f0009/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0009" rid-ob="ob-f0009" co-legend-rid="lgnd_f0009"><span>Figure 9C</span></a>). With these findings, the overall relevance of the orchestrated induction of EGFR-BRD4 axis-defined angiogenesis, autophagy and LDs was assessed by analyzing the mycobacterial burden. Congruent to the resolution of hallmark TB granulomatous lesions observed in <a href="/pmc/articles/PMC8942508/figure/f0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0001" rid-ob="ob-f0001" co-legend-rid="lgnd_f0001"><span>Figure 1I</span></a>,J, a reduction in mycobacterial burden was found in the lungs (<a href="/pmc/articles/PMC8942508/figure/f0009/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0009" rid-ob="ob-f0009" co-legend-rid="lgnd_f0009"><span>Figure 9D</span></a>, upper panel) and spleen (<a href="/pmc/articles/PMC8942508/figure/f0009/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0009" rid-ob="ob-f0009" co-legend-rid="lgnd_f0009"><span>Figure 9D</span></a>, lower panel) of infected mice that were treated with gefitinib or JQ1. This set of <em>in vivo</em> experiments indicates that EGFR-BRD4 axis plays a crucial role in regulating TB-associated angiogenesis and lipophagy, and subsequently Mtb survival within the host. Together, we propose EGFR and BRD4 inhibitors as potent anti-mycobacterial therapeutic adjuvants for their capacity to target multiple strategies adopted by Mtb for its persistence.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f0009" co-legend-rid="lgnd_f0009"><a href="/pmc/articles/PMC8942508/figure/f0009/" target="figure" rid-figpopup="f0009" rid-ob="ob-f0009"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140714701647968"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is KAUP_A_1936355_F0009_OC.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is KAUP_A_1936355_F0009_OC.jpg" src="/pmc/articles/PMC8942508/bin/KAUP_A_1936355_F0009_OC.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140714701647968"><a target="object" rel="noopener" href="/pmc/articles/PMC8942508/figure/f0009/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_f0009"><div><a class="figpopup" href="/pmc/articles/PMC8942508/figure/f0009/" target="figure" rid-figpopup="f0009" rid-ob="ob-f0009">Figure 9.</a></div><!--caption a7--><div class="caption"><p>EGFR-BRD4 aids in mycobacterial survival. (A and B) Immunohistochemical analysis of PECAM1 expression in the lung sections of the indicated groups of mice and the respective quantification. PECAM1-lined blood vessels with blood-filled lumen are close representatives of normal vasculature (n&#x000a0;=&#x000a0;3 mice in each group). (C) Lung cryosections from the indicated groups of mice were analyzed for autophagy and LD accumulation by MAP1LC3B immunostaining and BODIPY 493/503 staining, respectively (n&#x000a0;=&#x000a0;3 mice in each group). (D) The indicated groups of mice were assessed for Mtb H37Rv CFU in the lungs (D, top panel) and spleen (D, bottom&#x000a0;panel) (n&#x000a0;=&#x000a0;8 mice in each group). (E) Model: Mycobacteria coopts EGFR-BRD4 axis to promote aberrant angiogenesis and suppress lipophagy to thrive within the host milieu. **, p &#x0003c;&#x000a0;0.01; *** p &#x0003c;&#x000a0;0.001; **** p &#x0003c;&#x000a0;0.0001 (one-way ANOVA in B, D; GraphPad Prism 9.0).</p></div></div></div></div></div><div id="s0003" class="tsec sec"><h2 class="head no_bottom_margin" id="s0003title">Discussion</h2><p class="p p-first">In the current study, we showed that EGFR signaling-dependent BRD4 transcriptionally regulates the genes responsible for lipid accumulation and aids in LD buildup by actively suppressing autophagy. Although, the significance of autophagy and LDs have been individually detailed in various studies, their correlation during Mtb infection has remained understudied. One of the existing reports shows an inverse correlation of LDs and autophagy through the action of <em>Mir33</em> and <em>Mir33</em>* on the components of the two pathways. However, it does not clearly demonstrate the intersection of these two processes [<sup><a href="#cit0047" rid="cit0047" class=" bibr popnode">47</a></sup>]. We provide the first evidence for the suppression of LD turnover via autophagy in Mtb-infected cells, and that this restriction requires the participation of EGFR and BRD4. Further, we believe that BRD4 might act in concert with the H3K27 demethylase KDM6B to effectuate the transcriptional control of LD genes by recognizing active acetylation marks conferred on H3K27 sites upon the already known removal of H3K27me3 by KDM6B from specific LD gene promoters [<sup><a href="#cit0009" rid="cit0009" class=" bibr popnode">9</a></sup>]. Importantly, we support the use of BRD4 inhibitors as anti-mycobacterial agents by virtue of their potential to limit pro-survival factors for Mtb, despite their reported non-association with pathogen-specific xenophagy. We demonstrated that the loss-of-function of EGFR and BRD4 induces lipophagy that depletes the host cells of the LDs essential for the persistence of Mtb. Consequently, supplementation with external lipid sources, such as oleic acid, partially restored mycobacterial burden. With these observations, we also find it imperative to emphasize that multiple parallel cellular pathways exist for lipid breakdown, including autophagy (lysosomal) and cytosolic enzyme-mediated degradation; each of which would contribute to lipid availability for Mtb survival. Thus, the overall impact and the segregation or inter-regulation of such cellular processes on mycobacterial pathogenesis would require further elucidation.</p><p>The central role of lipids does not limit to survival, dormancy and reactivation of Mtb. With the availability of literature evidence demonstrating the correlation of the deposition of foam cells and formation of cholesterol crystals with neo-angiogenesis in clinical manifestations such as atherosclerosis [<sup><a href="#cit0048" rid="cit0048" class=" bibr popnode">48</a></sup>], we sought to assess the molecular regulation of TB-associated blood vessel formation along the EGFR-BRD4 axis. We showed that the expression of pro-angiogenic markers, majorly <em>Vegfa</em> and <em>Kdr</em>, is dependent on EGFR and BRD4. Inhibition of this pathway compromises the expression of the concerned angiogenic genes and normalizes blood vessel formation. As as been reported that normalized vasculature leads to a better prognosis for TB, we proposed that specific inhibitors for EGFR and BRD4 may serve as potential adjuvants in amplifying the effectiveness of anti-TB therapies. This study provides mechanistic insights into the hitherto unknown regulatory circuits operating in host cells for effectuating TB-associated angiogenesis, thus widening our arsenal that may be targeted for therapeutic purposes..</p><p class="p p-last">Together, we first report that the epigenetic mark reader, BRD4, is exploited by Mtb to execute multifaceted immune evasion strategies. We presented the molecular mechanism of transcriptional regulation of angiogenesis; and the inter-dependency of LDs and autophagy during TB infection (<a href="/pmc/articles/PMC8942508/figure/f0009/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0009" rid-ob="ob-f0009" co-legend-rid="lgnd_f0009"><span>Figure 9E</span></a>). We found a distinctive impact of EGFR-BRD4 inhibition on Mtb burden and TB pathogenesis. Moreover, we did not rule out the implication of this reader in the modulation of any other Mtb-driven immune subversion strategies. For instance, BRD4 is well known for its ability to mediate nuclear factor kappa B (NFKB)-driven inflammatory responses in the context of several viral infections and septic shock. With the ability of mycobacteria to intercept a plethora of host immune pathways, including NFKB-mediated events, we believe that EGFR-BRD4 axis may contribute to additional immune cross-talks. Interestingly, genome-wide association studies have now started to identify the correlation of single nucleotide polymorphisms (SNPs) in the enhancer regions of gens with specific disease states. Such observations highlight the relevance of targeting epigenetic mechanisms, including BRD4 as one of the readers of such SNPs for the treatment of diseases [<sup><a href="#cit0049" rid="cit0049" class=" bibr popnode">49</a></sup>]. Alongside, host-directed therapeutics (HDTs) is garnering interest amid the dwindling development of novel antibiotics and rapid emergence of antibiotic resistance. In this light, we believe that strategies targeting these factors would provide rational and clinically relevant adjuncts for TB treatment.</p></div><div id="s0004" class="tsec sec"><h2 class="head no_bottom_margin" id="s0004title">Materials and methods</h2><div id="s0004-s2001" class="sec sec-first"><h3 id="s0004-s2001title">Cells and mice</h3><p class="p p-first-last">Four- to six-weeks-old male and female BALB/c mice were utilized for all experiments. Mice were procured from The Jackson Laboratory (stock number 000651) and maintained at the Central Animal Facility (CAF) in Indian Institute of Science (IISc) under 12-h light and dark cycle. For <em>in vitro</em> experiments, mouse peritoneal macrophages were utilized. Briefly, mice were injected intraperitoneally with 8% Brewer&#x02019;s thioglycolate (HiMedia, M019) and peritoneal exudates were harvested in ice cold PBS (HiMedia, M1866TL1006) after 4 d, seeded in tissue culture dishes. Adherent cells were utilized as peritoneal macrophages. RAW 264.7 mouse monocyte-like cell line was procured from the National Centre for Cell Science (NCCS; ATCC TIB-71). All cells were cultured in Dulbecco&#x02019;s Modified Eagle Medium (DMEM; Gibco, Thermo Fisher Scientific, 12100061) supplemented with 10% heat-inactivated fetal bovine serum (FBS; Gibco, Thermo Fisher Scientific, 10270106) and maintained at 37&#x000b0;C in 5% CO<sub>2</sub> incubator.</p></div><div id="s0004-s2002" class="sec"><h3 id="s0004-s2002title">Ethics statement</h3><p class="p p-first-last">Experiments involving mice were carried out after the approval from Institutional Ethics Committee for animal experimentation. The animal care and use protocol adhered to were approved by national guidelines of the Committee for the Purpose of Control and Supervision of Experiments on Animals (CPCSEA), Government of India. Experiments with virulent mycobacteria (Mtb H37Rv) were approved by the Institutional Biosafety Committee.</p></div><div id="s0004-s2003" class="sec"><h3 id="s0004-s2003title">Bacteria</h3><p class="p p-first-last">Virulent strain of Mtb (Mtb H37Rv) was a kind research gift from Prof. Kanury Venkata Subba Rao (THSTI, India). Mycobacteria were cultured in Middlebrook 7&#x000a0;H9 medium (Difco, DF0713-17-9) supplemented with 10% OADC (oleic acid [Sigma-Aldrich, O1008], ALB [albumin; HiMedia, MB083], dextrose [HiMedia GRM077], CAT [catalase; HiMedia, RM446]). Single-cell suspensions of mycobacteria were obtained by passing mid log phase culture through 23-, 28- and 30-gauge needle 10 times each and used for infecting primary cells or RAW 264.7 macrophages at multiplicity of infection 10 (macrophage:mycobacteria, 1:10). The studies involving virulent mycobacterial strains were carried out at the biosafety level 3 (BSL-3) facility at Center for Infectious Disease Research (CIDR), IISc.</p></div><div id="s0004-s2004" class="sec"><h3 id="s0004-s2004title">Reagents and antibodies</h3><p class="p p-first-last">All general chemicals and reagents were procured from Sigma-Aldrich/Merck Millipore, HiMedia and Promega. Tissue culture plasticware was purchased from Jet Biofil, Tarsons India Pvt. Ltd. and Corning Inc. siRNAs were obtained from Dharmacon as siGENOME SMART-pool reagents against <em>Klf5, Stk4, Stk3, Lats1, Atg5. Egfr</em> and <em>Brd4</em> siRNAs were purchased from Eurogentec. Oleic acid, HRP-tagged anti-ACTB (A3854), 4&#x02032;,6-diamidino-2-phenylindole dihydrochloride (DAPI, 10236276001), 3,3&#x02032;-diaminobenzidine tetrahydrochloride hydrate (DAB, D5637), and anti-MAP1LC3B (L7543) were procured from Sigma-Aldrich. Anti-phospho-STK4-STK3 (3681), anti-STK4 (14946), anti-STK3 (3952), anti-phospho-LATS1 (8654), anti-LATS1 (3477), anti-BECN1 (3738), anti-SQSTM1 (5114), anti-phospho-EGFR (2231), anti-EGFR (2232), anti-EP300 (59240), anti-acetylated lysine (9441) and anti-acetyl H3K27 (4353) antibodies were purchased from Cell Signaling Technology. Anti-KLF5 antibody (PA5-27876) was purchased from Thermo Fisher Scientific, anti-PECAM1 (ab288364) and anti-BRD4 (ab75898) antibodies were procured from Abcam. VEGFA ELISA kit (900-K99) was obtained from PeproTech, USA. Lipofectamine 3000 (L3000015) was purchased from Thermo Fisher Scientific.</p></div><div id="s0004-s2005" class="sec"><h3 id="s0004-s2005title">Treatment with pharmacological reagents</h3><p class="p p-first-last">All the reagents were procured from Calbiochem, Sigma-Aldrich, PeproTech and Cayman Chemicals. Primary macrophages treated with the following reagents 1&#x000a0;h prior to infection with Mtb H37Rv &#x000a0;unless otherwise stated: Gefitinib (Cayman Chemicals, 13166; 20&#x000a0;&#x003bc;M); JQ1 (Cayman Chemicals, 11187; 400&#x000a0;nM); LY294002 (Calbiochem, 440202; 50&#x000a0;&#x003bc;M), rapamycin (Calbiochem, 553210; 100&#x000a0;nM). Chloroquine (Sigma-Aldrich, C6628; 10&#x000a0;&#x003bc;M) was added 6&#x000a0;h prior to harvest.</p></div><div id="s0004-s2006" class="sec"><h3 id="s0004-s2006title">Transient transfection studies</h3><p class="p p-first-last">RAW 264.7 macrophages were transfected with the indicated constructs (PI3K overexpression [PIK3C/p110 OE], PI3K dominant-negative [PIK3R/p85 DN], BRD4 WT); or primary macrophages were transfected with 100&#x000a0;nM each of siGLO <em>Lmna</em> (lamin A/C; Horizon, D-001620-01), non-targeting siRNA or specific siRNAs with the help of polyethyleneimine (PEI; Sigma-Aldrich, 764,604) or Lipofectamine 3000 for 8&#x000a0;h; followed by 24&#x000a0;h recovery. 70&#x02013;80% transfection efficiency was observed by counting the number of siGLO <em>Lmna</em>-positive cells in a microscopy field using fluorescence microscopy. Transfected cells were subjected to the required infections/treatments for the indicated time points and processed for analyses.</p></div><div id="s0004-s2007" class="sec"><h3 id="s0004-s2007title">RNA isolation and quantitative real time PCR (qRT-PCR)</h3><p class="p p-first-last">Treated samples were harvested in TRIzol (Sigma-Aldrich, T9424) and incubated with chloroform for phase separation. Total RNA was precipitated from the aqueous layer. Equal amount of RNA was converted into cDNA using First Strand cDNA synthesis kit (Applied Biological Materials Inc., G236). The cDNA thus obtained was used for SYBR Green (Thermo Fisher Scientific, F416) based quantitative real time PCR analysis for the concerned genes. <em>Gapdh</em> was used as internal control gene. Primer pairs used for expression analyses are detailed in Table S1.</p></div><div id="s0004-s2008" class="sec"><h3 id="s0004-s2008title">Immunoblotting</h3><p class="p p-first-last">Cells post treatment and/or infection were washed with PBS. Whole cell lysate was prepared by lysing in RIPA buffer (50&#x000a0;mM Tris-HCl, pH 7.4, 1% NP40 [Sigma-Aldrich, 74,385], 0.25% sodium deoxycholate [Sigma-Aldrich, D6750], 150&#x000a0;mM NaCl, 1&#x000a0;mM EDTA, 1&#x000a0;mM PMSF [Sigma-Aldrich, 78,830], 1&#x000a0;&#x003bc;g/ml each of aprotinin [Sigma-Aldrich, A4529], leupeptin [Sigma-Aldrich, L2884], pepstatin [Sigma-Aldrich, P4265], 1&#x000a0;mM Na<sub>3</sub>VO<sub>4</sub> [Sigma-Aldrich, S6508], 1&#x000a0;mM NaF [Sigma-Aldrich, S7920]) on ice for 30&#x000a0;min. Total protein from whole cell lysates was estimated by Bradford reagent (Sigma-Aldrich, B6916). Equal amount of protein was resolved using 12% SDS-PAGE and transferred onto PVDF membranes (Millipore, IPVH00010) by the semi-dry immunoblotting method (Bio-Rad). Nonfat dry milk powder (HiMedia, GRM1254) (5%) in TBST (20&#x000a0;mM Tris-HCl [Sigma-Aldrich, 10812846001], pH 7.4, 137&#x000a0;mM NaCl [Sigma-Aldrich, 746398, 0.1% Tween 20 [Sigma-Aldrich, P9416]) was used for blocking nonspecific binding for 60&#x000a0;min. After washing with TBST, the blots were incubated overnight at 4&#x000b0;C with primary antibody diluted in TBST with 5% BSA (HiMedia, MB083). After washing with TBST, blots were incubated with anti-rabbit IgG secondary antibody conjugated to HRP antibody (Jackson ImmunoResearch, 111&#x02013;035-045) for 4&#x000a0;h at 4&#x000b0;C. The immunoblots were developed with enhanced chemiluminescence detection system (Perkin Elmer, NEL105001EA) as per the manufacturer&#x02019;s instructions. For developing more than one protein at a particular molecular weight range, the blots were stripped off the first antibody at 60&#x000b0;C for 5&#x000a0;min using stripping buffer (62.5&#x000a0;mM Tris-HCl, pH 6.8, with 2% SDS [Sigma-Aldrich, L3771], 100&#x000a0;mM 2-mercaptoethanol [Sigma-Aldrich, M314A]), washed with TBST, blocked; followed by probing with the subsequent antibody following the described procedure. ACTB was used as loading control.</p></div><div id="s0004-s2009" class="sec"><h3 id="s0004-s2009title">Immunoprecipitation assay</h3><p class="p p-first-last">Immunoprecipitation assays were carried out following a modified version of the protocol provided by Millipore, USA. Treated samples were washed in ice-cold PBS and gently lysed in RIPA buffer. The cell lysates obtained were subjected to pre-clearing with BSA-blocked protein A beads (Bangalore Genei, LIA1M) for 30&#x000a0;min at 4&#x000b0;C and slow rotation. The amount of protein in the supernatant was quantified and equal amount of protein was used for pull down from each treatment condition; using Protein A beads pre-conjugated with the antibody of interest or isotype control IgG antibody. After incubation of the whole cell lysates with the antibody-complexed beads for 4&#x000a0;h at 4&#x000b0;C on slow rotation, the pellet containing the bead-bound immune complexes were washed with RIPA buffer twice. The complexes were eluted by boiling the beads in Laemmli buffer for 10&#x000a0;min. Bead-free samples were resolved by SDS-PAGE and the target interacting partners were identified by immunoblotting. Clean-Blot IP Detection Reagent (Thermo Fisher Scientific, 21230) was used for immunoblotting of the specific proteins.</p></div><div id="s0004-s2010" class="sec"><h3 id="s0004-s2010title">Chromatin immunoprecipitation (ChIP) assay</h3><p class="p p-first-last">ChIP assays were carried out using a protocol provided by Upstate Biotechnology and Sigma-Aldrich with certain modifications. Briefly, treated samples were washed with ice-cold PBS and fixed with 3.6% formaldehyde for 15&#x000a0;min at room temperature followed by inactivation of formaldehyde with 125&#x000a0;mM glycine. Nuclei were lysed in 0.1% SDS lysis buffer (50&#x000a0;mM Tris-HCl, pH 8.0, 200&#x000a0;mM NaCl, 10&#x000a0;mM HEPES, pH 6.5, 0.1% SDS, 10&#x000a0;mM EDTA, 0.5&#x000a0;mM EGTA, 1&#x000a0;mM PMSF, 1&#x000a0;&#x003bc;g/ml of each aprotinin, leupeptin, pepstatin, 1&#x000a0;mM Na<sub>3</sub>VO<sub>4</sub> and 1&#x000a0;mM NaF). Chromatin was sheared using Bioruptor Plus (Diagenode, Belgium) at high power for 70 rounds of 30 s pulse ON and 45 s pulse OFF. Chromatin extracts containing DNA fragments with an average size of 500 bp were immunoprecipitated with KLF5 or BRD4 or H3K27Ac or rabbit preimmune sera complexed with protein A agarose beads (Bangalore Genei, LIA1M). Immunoprecipitated complexes were sequentially washed twice with Wash Buffer A, B and TE (Wash Buffer A: 50&#x000a0;mM Tris-HCl, pH 8.0, 500&#x000a0;mM NaCl, 1&#x000a0;mM EDTA, 1% Triton X-100 [Sigma-Aldrich, T8787], 0.1% sodium deoxycholate, 0.1% SDS and protease/phosphatase inhibitors [1&#x000a0;&#x003bc;g/ml each of aprotinin [Sigma-Aldrich, A4529], leupeptin [Sigma-Aldrich, L2884], pepstatin [Sigma-Aldrich, P4265], 1&#x000a0;mM Na<sub>3</sub>VO<sub>4</sub> [Sigma-Aldrich, S6508], 1&#x000a0;mM NaF [Sigma-Aldrich, S7920]; Wash Buffer B: 50&#x000a0;mM Tris-HCl, pH 8.0, 1&#x000a0;mM EDTA, 250&#x000a0;mM LiCl, 0.5% NP40, 0.5% sodium deoxycholate and protease/phosphatase inhibitors; TE: 10&#x000a0;mM Tris-HCl, pH 8.0, 1&#x000a0;mM EDTA) and eluted in elution buffer (1% SDS, 0.1&#x000a0;M NaHCO<sub>3</sub>). After treating the eluted samples with RNase A (Thermo Fisher Scientific, EN0531) and Proteinase K (Thermo Fisher Scientific, 25,530,049), DNA was purified and precipitated using phenol-chloroform-ethanol method. Purified DNA was analyzed by quantitative real time RT-PCR. All values in the test samples were normalized to amplification of the specific gene in Input and IgG pull down and represented as fold-change in modification or enrichment. All ChIP experiments were repeated at least three times. The list of primers is given in Table S2.</p></div><div id="s0004-s2011" class="sec"><h3 id="s0004-s2011title">Sequential ChIP assay</h3><p class="p p-first-last">The protocol for sequential ChIP was adopted from [<sup><a href="#cit0050" rid="cit0050" class=" bibr popnode">50</a></sup><sup>,</sup><sup><a href="#cit0051" rid="cit0051" class=" bibr popnode">51</a></sup>]. Briefly, the DNA fragments obtained from fixed cells following sonication (in lysis buffer; 1% SDS, 10&#x000a0;mM EDTA, 50&#x000a0;mM Tris-HCl, pH 8.0) were immunoprecipitated with BRD4-complexed protein A beads. After first pull down, beads were washed with Re-ChIP Buffer (2&#x000a0;mM EDTA, 500&#x000a0;mM NaCl, 0.1% SDS, 1% NP40), followed by elution of DNA in Re-ChIP elution buffer (2% SDS, 15&#x000a0;mM DTT in TE) at 37&#x000b0;C for 30&#x000a0;min. The eluted DNA was subjected to subsequent round to immunoprecipitation with Protein-A beads pre-complexed with KLF5 or rabbit pre-immune sera. Immunoprecipitated complexes were sequentially washed with Wash Buffer A, B and TE (Wash Buffer A: 20&#x000a0;mM Tris-HCl, pH 8.0, 150&#x000a0;mM NaCl, 2&#x000a0;mM EDTA, 1% Triton X-100, 0.1% SDS and protease/phosphatase inhibitors; Wash Buffer B: 20&#x000a0;mM Tris-HCl, pH 8.0, 2&#x000a0;mM EDTA, 500&#x000a0;mM NaCl, 1% Triton X-100, 0.1% SDS and protease/phosphatase inhibitors; Wash Buffer C: 10&#x000a0;mM Tris-HCl, pH 8.0, 1&#x000a0;mM EDTA, 1% sodium deoxycholate, 1% NP40, 250&#x000a0;mM LiCl and protease/phosphatase inhibitors; TE: 10&#x000a0;mM Tris-HCl, pH 8.0, 1&#x000a0;mM EDTA and protease/phosphatase inhibitors) and eluted (0.1&#x000a0;M NaHCO<sub>3</sub>, 1% SDS), purified and subjected to qRT-PCR (as described previously). The fold-change of BRD4-KLF5 versus BRD4-IgG upon infection signified the co-occupancy of the two factors at concerned promoters. The list of primers is given in Table S2.</p></div><div id="s0004-s2012" class="sec"><h3 id="s0004-s2012title"><em>In vitro</em> CFU analysis</h3><p class="p p-first-last">Primary mouse macrophages were infected with Mtb H37Rv at MOI 1 for 4&#x000a0;h. Post 4&#x000a0;h, the cells were thoroughly washed with PBS to remove any surface adhered bacteria and medium containing amikacin (HiMedia, CMS644) (0.2 mg/ml) was added for 2&#x000a0;h to deplete any extracellular mycobacteria. After amikacin treatment, the cells thoroughly washed with PBS were taken for 0&#x000a0;h time point. A duplicate set was maintained in antibiotic-free medium for the indicated time points along with respective inhibitors, gefitinib and JQ1. Similar protocol was followed for macrophages knocked down for <em>Atg5</em>. Intracellular mycobacterial burden was enumerated by lysing macrophages with 0.06% SDS in 7H9 Middlebrook medium. Appropriate dilutions were plated on Middlebrook 7H11 agar plates supplemented with OADC. Total colony-forming units (CFUs) were counted after 21 d of plating.</p></div><div id="s0004-s2013" class="sec"><h3 id="s0004-s2013title">Mtb growth kinetics in the presence of JQ1</h3><p class="p p-first-last">Mtb H37Rv was grown to exponential phase (O.D. 0.6), then diluted to 0.1 O.D. and cultured in the presence of JQ1 (400&#x000a0;nM) for the indicated time points. Growth kinetics was measured as a function of absorbance read at 600&#x000a0;nm using a spectrophotometer and cultures were also plated on 7H11 medium to enumerate corresponding CFUs.</p></div><div id="s0004-s2014" class="sec"><h3 id="s0004-s2014title"><em>In vivo</em> mouse model for TB and treatment with pharmacological inhibitors</h3><p class="p p-first-last">BALB/c mice (n&#x000a0;=&#x000a0;24) were infected with mid-log phase Mtb H37Rv, using a Madison chamber aerosol generation instrument calibrated to 200 CFU/animal. Aerosolized animals were maintained in securely commissioned BSL3 facility. Post 28 d of established infection, mice were administered eight intra-peritoneal doses of gefitinib (50 mg/kg) [<sup><a href="#cit0027" rid="cit0027" class=" bibr popnode">27</a></sup>] or JQ1 (6.125 mg/kg) (modified from [<sup><a href="#cit0012" rid="cit0012" class=" bibr popnode">12</a></sup>]) every alternate day over 16 d. On 44<sup>th</sup> day post inhibitor treatment, mice were sacrificed, spleen and left lung lobe were homogenized in sterile PBS, serially diluted and plated on 7H11 agar containing OADC to quantify CFU. Upper right lung lobes were fixed in formalin, and processed for hematoxylin and eosin staining, or immunohistochemistry, and immunofluorescence analyses.</p></div><div id="s0004-s2015" class="sec"><h3 id="s0004-s2015title">Hematoxylin and eosin staining</h3><p class="p p-first-last">Microtome sections (5&#x000a0;&#x003bc;m) were obtained from formalin-fixed, paraffin-embedded mouse lung tissue samples using Leica RM2245 microtome. Deparaffinized and rehydrated sections were subjected to hematoxylin staining followed by eosin staining as per manufacturer instructions. After dehydrating, sections were mounted using DPX (Fisher Scientific, <a id="__tag_1054619745" class="tag_hotlink" href="/protein/Q18404" ref="/protein/Q18404"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q18404","term_id":"74962911","term_text":"Q18404"}}</span>Q18404</a>). Sections were kept for drying overnight and handed over to consultant pathologist for blinded analyses.</p></div><div id="s0004-s2016" class="sec"><h3 id="s0004-s2016title">Immunohistochemistry (IHC)</h3><p class="p p-first-last">3.6% formaldehyde-fixed, decalcified tissues were embedded in paraffin and microtome sections (5&#x000a0;&#x000b5;m) were obtained using Leica RM2245 microtome (Leica, Germany). Deparaffinized sections were subjected to antigen retrieval by boiling in 10&#x000a0;mM citrate buffer (pH 6.0) for 15&#x000a0;min; treated with 1% H<sub>2</sub>O<sub>2</sub> for 10&#x000a0;min in dark and blocked with 5% BSA in PBST for 1&#x000a0;h at room temperature. Post washing with PBST, the tissue sections were further incubated with primary antibodies in 5% BSA in PBST overnight. After incubation, the sections were washed with PBST and incubated with anti-rabbit HRP-conjugated secondary antibody for 90&#x000a0;min. The sections were stained using 0.05% diaminobenzidine in 0.03% H<sub>2</sub>O<sub>2</sub> solution. Finally, it was counterstained with hematoxylin, dehydrated and mounted in DPX (Fisher Scientific, <a id="__tag_1054619744" class="tag_hotlink" href="/protein/Q18404" ref="/protein/Q18404"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q18404","term_id":"74962911","term_text":"Q18404"}}</span>Q18404</a>). Axio Scope.A1 microscope (Zeiss, Germany) was used to image the stained tissue sections at the indicated magnification. All experiments were performed with appropriate isotype-matched control antibodies.</p></div><div id="s0004-s2017" class="sec"><h3 id="s0004-s2017title">Cryosection preparation and immunofluorescence (IF)</h3><p class="p p-first-last">The fixed lung pieces were placed in the optimal cutting temperature (OCT) media (Jung, Leica, 14020108926). Cryosections of 10&#x000a0;&#x000b5;m were prepared using Leica CM 1510 S or Leica CM 3050 S cryostat and then stored at &#x02212;80&#x000b0;C. For <em>in vitro</em> experiments, cells were fixed with 3.6% formaldehyde for 30&#x000a0;min at room temperature. Fixed samples from both <em>in vitro</em> and <em>in vivo</em> sources were blocked with 2% BSA in PBST (containing 0.02% saponin [Sigma-Aldrich, S4521]) for 1&#x000a0;h. After blocking, samples were stained with the indicated antibodies at 4&#x000b0;C overnight, followed by incubation with DyLight 488- (Thermo Fisher Scientific, 35502), Alexa Fluor 555 (Cell Signaling Technology, 4413) or Alexa Fluor 647 (BioLegend, 406414)-conjugated secondary antibodies for 2&#x000a0;h and nuclei were stained with DAPI. For EGFR and p-EGFR staining, following infection, cells were washed with PBS thrice and incubated with anti-EGFR or anti-p-EGFR antibody (in 2% BSA in PBS) at room temperature for 30&#x000a0;min. Cells were washed with PBS and incubated with anti-Rabbit Alexa-555 antibody for 30&#x000a0;min at room temperature. Post this, cells were washed in PBS and fixed with 3.7% PFA for 30&#x000a0;min, washed and stained with DAPI. All the samples were mounted on glycerol (Sigma-Aldrich, G5516) medium. Confocal images were taken with Zeiss LSM 710 Meta confocal laser scanning microscope (Carl Zeiss AG, Germany) using a plan-Apochromat 63X/1.4 Oil DIC objective (Carl Zeiss AG, Germany) and images were analyzed using ZEN 2009 software.</p></div><div id="s0004-s2018" class="sec"><h3 id="s0004-s2018title">Lipid droplet staining</h3><p class="p p-first-last">Lipid droplets were stained using neutral lipid dye BODIPY 493/503 (Invitrogen, D3922). Formaldehyde-fixed cells or tissue sections were stained with BODIPY (10&#x000a0;&#x003bc;g/ml) for 30&#x000a0;min at room temperature. Cells/tissues were washed, and nuclei were stained with DAPI. After washing with PBS, samples were mounted on glycerol and visualized and analyzed by confocal microscopy.</p></div><div id="s0004-s2019" class="sec"><h3 id="s0004-s2019title">Enzyme linked immunosorbent assay (ELISA)</h3><p class="p p-first-last">Cell-free lung homogenates were used for performing ELISA for VEGFA as per the manufacturer&#x02019;s instructions. Briefly, 96-well flat bottom plates (Nunc MaxiSorp; Thermo Scientific, 44&#x02013;2404-21) were coated with specific capture antibodies overnight at 4&#x000b0;C followed by three washes with PBST (PBS with 0.05% Tween 20). After blocking with 1% BSA for 1&#x000a0;h at room temperature, wells were washed and incubated with lung homogenates for 2&#x000a0;h. Post three washes with PBST, wells were incubated with respective detection antibodies for 2&#x000a0;h at room temperature. The wells were washed and incubated with streptavidin-HRP antibody (Peprotech, 900-K99) for 30&#x000a0;min at room temperature. The reactions were developed with 3,3&#x002b9;,5,5&#x002b9;-tetramethylbenzidine (TMB; Sigma-Aldrich, T8768) followed by arrest using 7% H<sub>2</sub>SO<sub>4</sub> and the absorbance was measured at 450&#x000a0;nm using an ELISA reader (Tecan, Switzerland).</p></div><div id="s0004-s2020" class="sec sec-last"><h3 id="s0004-s2020title">Statistical analysis</h3><p class="p p-first-last">Levels of significance for comparison between samples were determined by the Student&#x02019;s t-test, one-way ANOVA&#x000a0;and two-way ANOVA followed by Bonferroni&#x02019;s multiple-comparisons. The data in the graphs are expressed as the mean &#x000b1; S.E. for the values from at least 3 or more independent experiments and P values &#x0003c; 0.05 were defined as significant. GraphPad Prism software (5.0, 8.0 and 9.0 versions, GraphPad Software, USA) was used for all the statistical analyses.</p></div></div><div id="sec-a.v.e" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="sec-a.v.etitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SM1284"><!--caption a9--><strong>Supplemental Material:</strong><div class="sup-box half_rhythm" id="media-a.v.e.b.b"><a href="/pmc/articles/PMC8942508/bin/KAUP_A_1936355_SM1284.zip" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(219M, zip)</sup></div></div></div><div id="ack-a.w.a" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.w.atitle">Acknowledgments</h2><div class="sec"><p>We thank CAF, IISc for maintaining and providing mice for experimentation. BRD4 WT construct was provided by Dr. Rab Prinjha and Dr. Neil Garton from Glaxo Smith Kline (Deposited at GSK by Dr. Angela Bridges, BRAD Biochemistry, UK). PI3K dominant negative (PIK3R/p85 DN) construct was a kind research gift from Dr. Kumaravel Somasundaram, Department of Microbiology and Cell Biology, IISc. mRFP1-EGFP-PLIN2 construct was kindly provided by Dr. Mark A. McNiven, Mayo Clinic, Rochester, USA. We acknowledge BSL-3 facility at Center for Infectious Disease Research (CIDR) for permitting our <em>in vitro</em> and <em>in vivo</em> experiments with Mtb H37Rv. We thank Dr. Ravi Manjithaya, JNCASR, Bengaluru, India for kind suggestions.</p></div></div><div id="funding-group-a.u.b.t.a" class="tsec sec"><h2 class="head no_bottom_margin" id="funding-group-a.u.b.t.atitle">Funding Statement</h2><p>This work was supported by funds from the Department of Biotechnology (DBT, No. BT/PR13522/COE/34/27/2015 dt 22/8/2017 and BT/PR27352/BRB/10/1639/2017, dt 30/8/2018 to K.N.B) and the Department of Science and Technology (DST, EMR/2014/000875 dt 4.12.15 to K.N.B.), New Delhi, India. K.N.B. thanks Science and Engineering Research Board (SERB), DST for the award of J. C. Bose National Fellowship (No. SB/S2/JCB-025/2016 dt 25/7/15) and core research grant (CRG/2019/002062); and for the funding (SP/DSTO-19-0176 dt 06/02/2020). The authors thank DST-FIST, UGC Centre for Advanced Study and DBT-IISc Partnership Program (Phase-II at IISc, BT/PR27952/INF/22/212/2018) for the funding and infrastructure support. Fellowships were received from IISc (TM, PP, GKL) and UGC (BB). The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript.</p></div><div id="s0005" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="s0005title">Disclosure statement</h2><!--/article/back/sec/--><p class="p p-first-last">No potential conflict of interest was reported by the author(s).</p></div><div id="s0006" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="s0006title">Supplementary material</h2><!--/article/back/sec/--><p class="p p-first-last">Supplemental data for this article can be accessed <a href="https://doi.org/10.1080/15548627.2021.1936355" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">here</a></p></div><div id="ref-list-a.w.d" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.w.dtitle">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="cit0001">[1] <span class="mixed-citation">Peyron P, Vaubourgeix P, Poquet Y, et al. <span class="ref-title">Foamy macrophages from tuberculous patients&#x02019; granulomas constitute a nutrient-rich reservoir for M. tuberculosis persistence</span>. <span class="ref-journal">PLoS Pathog</span>. 2008;<span class="ref-vol">4</span>:e1000204. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2575403/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19002241" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Pathog&amp;title=Foamy+macrophages+from+tuberculous+patients&#x02019;+granulomas+constitute+a+nutrient-rich+reservoir+for+M.+tuberculosis+persistence&amp;volume=4&amp;publication_year=2008&amp;pages=e1000204&amp;pmid=19002241&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0002">[2] <span class="mixed-citation">Daniel J, Maamar H, Deb C, et al. <span class="ref-title">Mycobacterium tuberculosis uses host triacylglycerol to accumulate lipid droplets and acquires a dormancy-like phenotype in lipid-loaded macrophages</span>. <span class="ref-journal">PLoS Pathog</span>. 2011;<span class="ref-vol">7</span>:e1002093. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3121879/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21731490" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Pathog&amp;title=Mycobacterium+tuberculosis+uses+host+triacylglycerol+to+accumulate+lipid+droplets+and+acquires+a+dormancy-like+phenotype+in+lipid-loaded+macrophages&amp;volume=7&amp;publication_year=2011&amp;pages=e1002093&amp;pmid=21731490&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0003">[3] <span class="mixed-citation">Knight M, Braverman J, Asfaha K, et al. <span class="ref-title">Lipid droplet formation in Mycobacterium tuberculosis infected macrophages requires IFN-?/HIF-1a signaling and supports host defense</span>. <span class="ref-journal">PLoS Pathog</span>. 2018;<span class="ref-vol">14</span>(<span class="ref-iss">1</span>):e1006874. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5800697/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29370315" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Pathog&amp;title=Lipid+droplet+formation+in+Mycobacterium+tuberculosis+infected+macrophages+requires+IFN-?/HIF-1a+signaling+and+supports+host+defense&amp;volume=14&amp;issue=1&amp;publication_year=2018&amp;pages=e1006874&amp;pmid=29370315&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0004">[4] <span class="mixed-citation">Deretic V, Saitoh T, Akira S.. 
<span class="ref-title">Autophagy in infection, inflammation, and immunity</span>. <span class="ref-journal">Nat Rev Immunol</span>. 2013;<span class="ref-vol">13</span>(<span class="ref-iss">10</span>):722&#x02013;737. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5340150/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24064518" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Immunol&amp;title=Autophagy+in+infection,+inflammation,+and+immunity&amp;volume=13&amp;issue=10&amp;publication_year=2013&amp;pages=722-737&amp;pmid=24064518&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0005">[5] <span class="mixed-citation">Kim JK, Kim TS, Basu J, et al. <span class="ref-title">MicroRNA in innate immunity and autophagy during mycobacterial infection</span>. <span class="ref-journal">Cell Microbiol</span>. 2017;<span class="ref-vol">19</span>(<span class="ref-iss">1</span>):e12687. [<a href="https://pubmed.ncbi.nlm.nih.gov/27794209" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Microbiol&amp;title=MicroRNA+in+innate+immunity+and+autophagy+during+mycobacterial+infection&amp;volume=19&amp;issue=1&amp;publication_year=2017&amp;pages=e12687&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0006">[6] <span class="mixed-citation">Holla S, Kurowska-Stolarska M, Bayry J, et al. <span class="ref-title">Selective inhibition of IFNG-induced autophagy by miR155- and miR31-responsive WNT5A and SHH signaling</span>. <span class="ref-journal">Autophagy</span>. 2014;<span class="ref-vol">10</span>(<span class="ref-iss">2</span>):311&#x02013;330. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5396086/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24343269" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Autophagy&amp;title=Selective+inhibition+of+IFNG-induced+autophagy+by+miR155-+and+miR31-responsive+WNT5A+and+SHH+signaling&amp;volume=10&amp;issue=2&amp;publication_year=2014&amp;pages=311-330&amp;pmid=24343269&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0007">[7] <span class="mixed-citation">Kumar D, Nath L, Kamal AM, et al. <span class="ref-title">Genome-wide analysis of the host intracellular network that regulates survival of Mycobacterium tuberculosis</span>. <span class="ref-journal">Cell</span>. 2010;<span class="ref-vol">140</span>:731&#x02013;743. [<a href="https://pubmed.ncbi.nlm.nih.gov/20211141" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Genome-wide+analysis+of+the+host+intracellular+network+that+regulates+survival+of+Mycobacterium+tuberculosis&amp;volume=140&amp;publication_year=2010&amp;pages=731-743&amp;pmid=20211141&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0008">[8] <span class="mixed-citation">Ghorpade DS, Holla S, Sinha AY, et al. <span class="ref-title">Nitric oxide and KLF4 protein epigenetically modify class II transactivator to repress major histocompatibility complex II expression during Mycobacterium bovis bacillus Calmette-Guerin infection</span>. <span class="ref-journal">J Biol Chem</span>. 2013;<span class="ref-vol">288</span>:20592&#x02013;20606. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3711323/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23733190" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Nitric+oxide+and+KLF4+protein+epigenetically+modify+class+II+transactivator+to+repress+major+histocompatibility+complex+II+expression+during+Mycobacterium+bovis+bacillus+Calmette-Guerin+infection&amp;volume=288&amp;publication_year=2013&amp;pages=20592-20606&amp;pmid=23733190&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0009">[9] <span class="mixed-citation">Singh V, Prakhar P, Rajmani RS, et al. <span class="ref-title">Histone methyltransferase SET8 epigenetically reprograms host immune responses to assist mycobacterial survival</span>. <span class="ref-journal">J Infect Dis</span>. 2017;<span class="ref-vol">216:</span>:477&#x02013;488. [<a href="https://pubmed.ncbi.nlm.nih.gov/28931237" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Infect+Dis&amp;title=Histone+methyltransferase+SET8+epigenetically+reprograms+host+immune+responses+to+assist+mycobacterial+survival&amp;volume=216:&amp;publication_year=2017&amp;pages=477-488&amp;pmid=28931237&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0010">[10] <span class="mixed-citation">Holla S, Prakhar P, Singh V, et al. <span class="ref-title">MUSASHI-mediated expression of JMJD3, a H3K27me3 demethylase, is involved in foamy macrophage generation during mycobacterial infection</span>. <span class="ref-journal">PLoS Pathog</span>. 2016;<span class="ref-vol">12</span>:e1005814. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4988650/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27532872" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Pathog&amp;title=MUSASHI-mediated+expression+of+JMJD3,+a+H3K27me3+demethylase,+is+involved+in+foamy+macrophage+generation+during+mycobacterial+infection&amp;volume=12&amp;publication_year=2016&amp;pages=e1005814&amp;pmid=27532872&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0011">[11] <span class="mixed-citation">Mahadik K, Prakhar P, Rajmani RS, et al. <span class="ref-title">c-Abl-TWIST1 epigenetically dysregulate inflammatory responses during mycobacterial infection by co-regulating Bone Morphogenesis Protein and miR27a</span>. <span class="ref-journal">Front Immunol</span>. 2018;<span class="ref-vol">9</span>:85. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5799226/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29449840" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front+Immunol&amp;title=c-Abl-TWIST1+epigenetically+dysregulate+inflammatory+responses+during+mycobacterial+infection+by+co-regulating+Bone+Morphogenesis+Protein+and+miR27a&amp;volume=9&amp;publication_year=2018&amp;pages=85&amp;pmid=29449840&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0012">[12] <span class="mixed-citation">Cheng CY, Gutierrez NM, Marzuki MB, et al. <span class="ref-title">Host sirtuin 1 regulates mycobacterial immunopathogenesis and represents a therapeutic target against tuberculosis</span>. <span class="ref-journal">Sci Immunol</span>. 2017;<span class="ref-vol">2</span>(<span class="ref-iss">9</span>):eaaj1789. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5505666/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28707004" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci+Immunol&amp;title=Host+sirtuin+1+regulates+mycobacterial+immunopathogenesis+and+represents+a+therapeutic+target+against+tuberculosis&amp;volume=2&amp;issue=9&amp;publication_year=2017&amp;pages=eaaj1789&amp;pmid=28707004&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0013">[13] <span class="mixed-citation">Belkina AC, Nikolajczyk BS, Denis GV.. 
<span class="ref-title">BET protein function is required for inflammation: brd2 genetic disruption and BET inhibitor JQ1 impair mouse macrophage inflammatory responses</span>. <span class="ref-journal">J Immunol</span>. 2013;<span class="ref-vol">190</span>:3670&#x02013;3678. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3608815/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23420887" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Immunol&amp;title=BET+protein+function+is+required+for+inflammation:+brd2+genetic+disruption+and+BET+inhibitor+JQ1+impair+mouse+macrophage+inflammatory+responses&amp;volume=190&amp;publication_year=2013&amp;pages=3670-3678&amp;pmid=23420887&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0014">[14] <span class="mixed-citation">McKinney CC, Kim MJ, Chen D, et al. <span class="ref-title">Brd4 activates early viral transcription upon Human Papillomavirus 18 infection of primary keratinocytes</span>. <span class="ref-journal">MBio</span>. 2016;<span class="ref-vol">7</span>(<span class="ref-iss">6</span>):e01644&#x02013;16. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5120138/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27879331" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=MBio&amp;title=Brd4+activates+early+viral+transcription+upon+Human+Papillomavirus+18+infection+of+primary+keratinocytes&amp;volume=7&amp;issue=6&amp;publication_year=2016&amp;pages=e01644-16&amp;pmid=27879331&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0015">[15] <span class="mixed-citation">Ren K, Zhang W, Chen X, et al. <span class="ref-title">An epigenetic compound library screen identifies BET inhibitors that promote HSV-1 and &#x02212;2 replication by bridging P-TEFb to viral gene promoters through BRD4</span>. <span class="ref-journal">PLoS Pathog</span>. 2016;<span class="ref-vol">12</span>:e1005950. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5072739/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27764245" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Pathog&amp;title=An+epigenetic+compound+library+screen+identifies+BET+inhibitors+that+promote+HSV-1+and+&#x02212;2+replication+by+bridging+P-TEFb+to+viral+gene+promoters+through+BRD4&amp;volume=12&amp;publication_year=2016&amp;pages=e1005950&amp;pmid=27764245&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0016">[16] <span class="mixed-citation">Shi J, Vakoc CR. <span class="ref-title">The mechanisms behind the therapeutic activity of BET bromodomain inhibition</span>. <span class="ref-journal">Mol Cell</span>. 2014;<span class="ref-vol">54</span>:728&#x02013;736. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4236231/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24905006" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell&amp;title=The+mechanisms+behind+the+therapeutic+activity+of+BET+bromodomain+inhibition&amp;volume=54&amp;publication_year=2014&amp;pages=728-736&amp;pmid=24905006&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0017">[17] <span class="mixed-citation">Sakurai N, Inamochi Y, Inoue T, et al. <span class="ref-title">BRD4 regulates adiponectin gene induction by recruiting the P-TEFb complex to the transcribed region of the gene</span>. <span class="ref-journal">Sci Rep</span>. 2017;<span class="ref-vol">7</span>:11962. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5607256/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28931940" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci+Rep&amp;title=BRD4+regulates+adiponectin+gene+induction+by+recruiting+the+P-TEFb+complex+to+the+transcribed+region+of+the+gene&amp;volume=7&amp;publication_year=2017&amp;pages=11962&amp;pmid=28931940&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0018">[18] <span class="mixed-citation">Sakamaki JI, Wilkinson S, Hahn M, et al. <span class="ref-title">Bromodomain Protein BRD4 is a transcriptional repressor of autophagy and lysosomal function</span>. <span class="ref-journal">Mol Cell</span>. 2017;<span class="ref-vol">66</span>:517&#x02013;532 e519. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5446411/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28525743" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell&amp;title=Bromodomain+Protein+BRD4+is+a+transcriptional+repressor+of+autophagy+and+lysosomal+function&amp;volume=66&amp;publication_year=2017&amp;pages=517-532+e519&amp;pmid=28525743&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0019">[19] <span class="mixed-citation">Ramakrishnan L. <span class="ref-title">Revisiting the role of the granuloma in tuberculosis</span>. <span class="ref-journal">Nat Rev Immunol</span>. 2012;<span class="ref-vol">12</span>:352&#x02013;366. [<a href="https://pubmed.ncbi.nlm.nih.gov/22517424" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Immunol&amp;title=Revisiting+the+role+of+the+granuloma+in+tuberculosis&amp;volume=12&amp;publication_year=2012&amp;pages=352-366&amp;pmid=22517424&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0020">[20] <span class="mixed-citation">Kim M, Wainwright HC, Locketz M, et al. <span class="ref-title">Caseation of human tuberculosis granulomas correlates with elevated host lipid metabolism</span>. <span class="ref-journal">EMBO Mol Med</span>. 2010;<span class="ref-vol">2</span>(<span class="ref-iss">7</span>):258&#x02013;274. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2913288/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20597103" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+Mol+Med&amp;title=Caseation+of+human+tuberculosis+granulomas+correlates+with+elevated+host+lipid+metabolism&amp;volume=2&amp;issue=7&amp;publication_year=2010&amp;pages=258-274&amp;pmid=20597103&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0021">[21] <span class="mixed-citation">Osherov N, Ben-Ami R. <span class="ref-title">Modulation of host angiogenesis as a microbial survival strategy and therapeutic target</span>. <span class="ref-journal">PLoS Pathog</span>. 2016;<span class="ref-vol">12</span>:e1005479. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4831739/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27078259" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Pathog&amp;title=Modulation+of+host+angiogenesis+as+a+microbial+survival+strategy+and+therapeutic+target&amp;volume=12&amp;publication_year=2016&amp;pages=e1005479&amp;pmid=27078259&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0022">[22] <span class="mixed-citation">Zielonka TM, Demkow U, Michalowska-Mitczuk D, et al. <span class="ref-title">Angiogenic activity of sera from pulmonary tuberculosis patients in relation to IL-12p40 and TNFalpha serum levels</span>. <span class="ref-journal">Lung</span>. 2011;<span class="ref-vol">189</span>:351&#x02013;357. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3140944/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21603999" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lung&amp;title=Angiogenic+activity+of+sera+from+pulmonary+tuberculosis+patients+in+relation+to+IL-12p40+and+TNFalpha+serum+levels&amp;volume=189&amp;publication_year=2011&amp;pages=351-357&amp;pmid=21603999&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0023">[23] <span class="mixed-citation">Polena H, Boudou F, Tilleul S, et al. <span class="ref-title">Mycobacterium tuberculosis exploits the formation of new blood vessels for its dissemination</span>. <span class="ref-journal">Sci Rep</span>. 2016;<span class="ref-vol">6</span>:33162. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5018821/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27616470" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci+Rep&amp;title=Mycobacterium+tuberculosis+exploits+the+formation+of+new+blood+vessels+for+its+dissemination&amp;volume=6&amp;publication_year=2016&amp;pages=33162&amp;pmid=27616470&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0024">[24] <span class="mixed-citation">Datta M, Via LE, Kamoun WS, et al. <span class="ref-title">Anti-vascular endothelial growth factor treatment normalizes tuberculosis granuloma vasculature and improves small molecule delivery</span>. <span class="ref-journal">Proc Natl Acad Sci U S A</span>. 2015;<span class="ref-vol">112</span>:1827&#x02013;1832. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4330784/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25624495" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Anti-vascular+endothelial+growth+factor+treatment+normalizes+tuberculosis+granuloma+vasculature+and+improves+small+molecule+delivery&amp;volume=112&amp;publication_year=2015&amp;pages=1827-1832&amp;pmid=25624495&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0025">[25] <span class="mixed-citation">Xu Y, Wang L, Zimmerman MD, et al. <span class="ref-title">Matrix metalloproteinase inhibitors enhance the efficacy of frontline drugs against Mycobacterium tuberculosis</span>. <span class="ref-journal">PLoS Pathog</span>. 2018;<span class="ref-vol">14</span>:e1006974. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5919409/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29698476" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Pathog&amp;title=Matrix+metalloproteinase+inhibitors+enhance+the+efficacy+of+frontline+drugs+against+Mycobacterium+tuberculosis&amp;volume=14&amp;publication_year=2018&amp;pages=e1006974&amp;pmid=29698476&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0026">[26] <span class="mixed-citation">Kurimchak AM, Shelton C, Duncan KE, et al. <span class="ref-title">Resistance to BET bromodomain inhibitors is mediated by kinome reprogramming in ovarian cancer</span>. <span class="ref-journal">Cell Rep</span>. 2016;<span class="ref-vol">16</span>:1273&#x02013;1286. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4972668/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27452461" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Rep&amp;title=Resistance+to+BET+bromodomain+inhibitors+is+mediated+by+kinome+reprogramming+in+ovarian+cancer&amp;volume=16&amp;publication_year=2016&amp;pages=1273-1286&amp;pmid=27452461&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0027">[27] <span class="mixed-citation">Liu F, Hon GC, Villa GR, et al. <span class="ref-title">EGFR mutation promotes glioblastoma through epigenome and transcription factor network remodeling</span>. <span class="ref-journal">Mol Cell</span>. 2015;<span class="ref-vol">60</span>:307&#x02013;318. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4609298/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26455392" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell&amp;title=EGFR+mutation+promotes+glioblastoma+through+epigenome+and+transcription+factor+network+remodeling&amp;volume=60&amp;publication_year=2015&amp;pages=307-318&amp;pmid=26455392&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0028">[28] <span class="mixed-citation">Stanley SA, Barczak AK, Silvis MR, et al. <span class="ref-title">Identification of host-targeted small molecules that restrict intracellular Mycobacterium tuberculosis growth</span>. <span class="ref-journal">PLoS Pathog</span>. 2014;<span class="ref-vol">10</span>:e1003946. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3930586/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24586159" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Pathog&amp;title=Identification+of+host-targeted+small+molecules+that+restrict+intracellular+Mycobacterium+tuberculosis+growth&amp;volume=10&amp;publication_year=2014&amp;pages=e1003946&amp;pmid=24586159&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0029">[29] <span class="mixed-citation">Wen X, Klionsky DJ. <span class="ref-title">BRD4 is a newly characterized transcriptional regulator that represses autophagy and lysosomal function</span>. <span class="ref-journal">Autophagy</span>. 2017;<span class="ref-vol">13</span>(<span class="ref-iss">11</span>):1801&#x02013;1803. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5788468/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28837371" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Autophagy&amp;title=BRD4+is+a+newly+characterized+transcriptional+regulator+that+represses+autophagy+and+lysosomal+function&amp;volume=13&amp;issue=11&amp;publication_year=2017&amp;pages=1801-1803&amp;pmid=28837371&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0030">[30] <span class="mixed-citation">Feng L, Hu J, Zhang W, et al. <span class="ref-title">RELL1 inhibits autophagy pathway and regulates Mycobacterium tuberculosis survival in macrophages</span>. <span class="ref-journal">Tuberculosis (Ednib)</span>. 2020;<span class="ref-vol">120</span>:101900. [<a href="https://pubmed.ncbi.nlm.nih.gov/32090861" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Tuberculosis+(Ednib)&amp;title=RELL1+inhibits+autophagy+pathway+and+regulates+Mycobacterium+tuberculosis+survival+in+macrophages&amp;volume=120&amp;publication_year=2020&amp;pages=101900&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0031">[31] <span class="mixed-citation">da Motta LL, Ledaki I, Purshouse K, et al. <span class="ref-title">The BET inhibitor JQ1 selectively impairs tumour response to hypoxia and downregulates CA9 and angiogenesis in triple negative breast cancer</span>. <span class="ref-journal">Oncogene</span>. 2017;<span class="ref-vol">36</span>:122&#x02013;132. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5061082/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27292261" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=The+BET+inhibitor+JQ1+selectively+impairs+tumour+response+to+hypoxia+and+downregulates+CA9+and+angiogenesis+in+triple+negative+breast+cancer&amp;volume=36&amp;publication_year=2017&amp;pages=122-132&amp;pmid=27292261&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0032">[32] <span class="mixed-citation">Singh R, Kaushik S, Wang Y, et al. <span class="ref-title">Autophagy regulates lipid metabolism</span>. <span class="ref-journal">Nature</span>. 2009;<span class="ref-vol">458</span>:1131&#x02013;1135. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2676208/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19339967" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Autophagy+regulates+lipid+metabolism&amp;volume=458&amp;publication_year=2009&amp;pages=1131-1135&amp;pmid=19339967&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0033">[33] <span class="mixed-citation">Ding S, Qu Y, Yang S, et al. <span class="ref-title">Novel miR-1958 promotes Mycobacterium tuberculosis survival in RAW 264.7 cells by inhibiting autophagy via Atg5</span>. <span class="ref-journal">J Microbiol Biotechnol</span>. 2019;<span class="ref-vol">29</span>(<span class="ref-iss">6</span>):989&#x02013;998. [<a href="https://pubmed.ncbi.nlm.nih.gov/31154748" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Microbiol+Biotechnol&amp;title=Novel+miR-1958+promotes+Mycobacterium+tuberculosis+survival+in+RAW+264.7+cells+by+inhibiting+autophagy+via+Atg5&amp;volume=29&amp;issue=6&amp;publication_year=2019&amp;pages=989-998&amp;pmid=31154748&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0034">[34] <span class="mixed-citation">Camare C, Pucelle M, Negre-Salvayre A, et al. <span class="ref-title">Angiogenesis in the atherosclerotic plaque</span>. <span class="ref-journal">Redox Biol</span>. 2017;<span class="ref-vol">12</span>:18&#x02013;34. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5312547/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28212521" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Redox+Biol&amp;title=Angiogenesis+in+the+atherosclerotic+plaque&amp;volume=12&amp;publication_year=2017&amp;pages=18-34&amp;pmid=28212521&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0035">[35] <span class="mixed-citation">Alatas F, Alatas O, Metintas M, et al. <span class="ref-title">Vascular endothelial growth factor levels in active pulmonary tuberculosis</span>. <span class="ref-journal">Chest</span>. 2004;<span class="ref-vol">125</span>:2156&#x02013;2159. [<a href="https://pubmed.ncbi.nlm.nih.gov/15189936" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chest&amp;title=Vascular+endothelial+growth+factor+levels+in+active+pulmonary+tuberculosis&amp;volume=125&amp;publication_year=2004&amp;pages=2156-2159&amp;pmid=15189936&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0036">[36] <span class="mixed-citation">Jaisinghani N, Dawa S, Singh K, et al. <span class="ref-title">Necrosis driven triglyceride synthesis primes macrophages for inflammation during Mycobacterium tuberculosis infection</span>. <span class="ref-journal">Front Immunol</span>. 2018;<span class="ref-vol">9</span>:1490. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6037689/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30018616" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front+Immunol&amp;title=Necrosis+driven+triglyceride+synthesis+primes+macrophages+for+inflammation+during+Mycobacterium+tuberculosis+infection&amp;volume=9&amp;publication_year=2018&amp;pages=1490&amp;pmid=30018616&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0037">[37] <span class="mixed-citation">Wong KW, Jacobs WR Jr.. 
<span class="ref-title">Postprimary tuberculosis and macrophage necrosis: is there a big ConNECtion?</span>
<span class="ref-journal">MBio</span>. 2016;<span class="ref-vol">7</span>:e01589&#x02013;01515. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4724999/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26758178" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=MBio&amp;title=Postprimary+tuberculosis+and+macrophage+necrosis:+is+there+a+big+ConNECtion?&amp;volume=7&amp;publication_year=2016&amp;pages=e01589-01515&amp;pmid=26758178&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0038">[38] <span class="mixed-citation">Offer S, Menard JA, Perez JE, et al. <span class="ref-title">Extracellular lipid loading augments hypoxic paracrine signaling and promotes glioma angiogenesis and macrophage infiltration</span>. <span class="ref-journal">J Exp Clin Cancer Res</span>. 2019;<span class="ref-vol">38</span>:241. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6556032/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31174567" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Exp+Clin+Cancer+Res&amp;title=Extracellular+lipid+loading+augments+hypoxic+paracrine+signaling+and+promotes+glioma+angiogenesis+and+macrophage+infiltration&amp;volume=38&amp;publication_year=2019&amp;pages=241&amp;pmid=31174567&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0039">[39] <span class="mixed-citation">Miyamoto S, Suzuki T, Muto S, et al. <span class="ref-title">Positive and negative regulation of the cardiovascular transcription factor KLF5 by p300 and the oncogenic regulator SET through interaction and acetylation on the DNA-binding domain</span>. <span class="ref-journal">Mol Cell Biol</span>. 2003;<span class="ref-vol">23</span>:8528&#x02013;8541. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC262669/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14612398" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Biol&amp;title=Positive+and+negative+regulation+of+the+cardiovascular+transcription+factor+KLF5+by+p300+and+the+oncogenic+regulator+SET+through+interaction+and+acetylation+on+the+DNA-binding+domain&amp;volume=23&amp;publication_year=2003&amp;pages=8528-8541&amp;pmid=14612398&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0040">[40] <span class="mixed-citation">Gao Y, Wu K, Chen Y, et al. <span class="ref-title">Beyond proliferation: KLF5 promotes angiogenesis of bladder cancer through directly regulating VEGFA transcription</span>. <span class="ref-journal">Oncotarget</span>. 2015;<span class="ref-vol">6</span>:43791&#x02013;43805. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4791267/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26544730" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=Beyond+proliferation:+KLF5+promotes+angiogenesis+of+bladder+cancer+through+directly+regulating+VEGFA+transcription&amp;volume=6&amp;publication_year=2015&amp;pages=43791-43805&amp;pmid=26544730&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0041">[41] <span class="mixed-citation">Holbro T, Hynes NE. <span class="ref-title">ErbB receptors: directing key signaling networks throughout life</span>. <span class="ref-journal">Annu Rev Pharmacol Toxicol</span>. 2004;<span class="ref-vol">44</span>:195&#x02013;217. [<a href="https://pubmed.ncbi.nlm.nih.gov/14744244" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Pharmacol+Toxicol&amp;title=ErbB+receptors:+directing+key+signaling+networks+throughout+life&amp;volume=44&amp;publication_year=2004&amp;pages=195-217&amp;pmid=14744244&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0042">[42] <span class="mixed-citation">Lindsey S, Langhans SA. <span class="ref-title">Crosstalk of oncogenic signaling pathways during epithelial-mesenchymal transition</span>. <span class="ref-journal">Front Oncol</span>. 2014;<span class="ref-vol">4</span>:358. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4263086/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25566498" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front+Oncol&amp;title=Crosstalk+of+oncogenic+signaling+pathways+during+epithelial-mesenchymal+transition&amp;volume=4&amp;publication_year=2014&amp;pages=358&amp;pmid=25566498&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0043">[43] <span class="mixed-citation">Boro M, Singh V, Balaji KN. <span class="ref-title">Mycobacterium tuberculosis-triggered Hippo pathway orchestrates CXCL1/2 expression to modulate host immune responses</span>. <span class="ref-journal">Sci Rep</span>. 2016;<span class="ref-vol">6</span>:37695. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5121601/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27883091" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci+Rep&amp;title=Mycobacterium+tuberculosis-triggered+Hippo+pathway+orchestrates+CXCL1/2+expression+to+modulate+host+immune+responses&amp;volume=6&amp;publication_year=2016&amp;pages=37695&amp;pmid=27883091&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0044">[44] <span class="mixed-citation">Dai X, She P, Chi F, et al. <span class="ref-title">Phosphorylation of angiomotin by Lats1/2 kinases inhibits F-actin binding, cell migration, and angiogenesis</span>. <span class="ref-journal">J Biol Chem</span>. 2013;<span class="ref-vol">288</span>:34041&#x02013;34051. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3837143/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24106267" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Phosphorylation+of+angiomotin+by+Lats1/2+kinases+inhibits+F-actin+binding,+cell+migration,+and+angiogenesis&amp;volume=288&amp;publication_year=2013&amp;pages=34041-34051&amp;pmid=24106267&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0045">[45] <span class="mixed-citation">Kim J, Kim YH, Park DY, et al. <span class="ref-title">YAP/TAZ regulates sprouting angiogenesis and vascular barrier maturation</span>. <span class="ref-journal">J Clin Invest</span>. 2017;<span class="ref-vol">127</span>:3441&#x02013;3461. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5669570/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28805663" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Invest&amp;title=YAP/TAZ+regulates+sprouting+angiogenesis+and+vascular+barrier+maturation&amp;volume=127&amp;publication_year=2017&amp;pages=3441-3461&amp;pmid=28805663&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0046">[46] <span class="mixed-citation">Reddy RJ, Gajadhar AS, Swenson EJ, et al. <span class="ref-title">Early signaling dynamics of the epidermal growth factor receptor</span>. <span class="ref-journal">Proc Natl Acad Sci U S A</span>. 2016;<span class="ref-vol">113</span>:3114&#x02013;3119. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4801278/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26929352" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Early+signaling+dynamics+of+the+epidermal+growth+factor+receptor&amp;volume=113&amp;publication_year=2016&amp;pages=3114-3119&amp;pmid=26929352&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0047">[47] <span class="mixed-citation">Ouimet M, Koster S, Sakowski E, et al. <span class="ref-title">Mycobacterium tuberculosis induces the miR-33 locus to reprogram autophagy and host lipid metabolism</span>. <span class="ref-journal">Nat Immunol</span>. 2016;<span class="ref-vol">17</span>(<span class="ref-iss">6</span>):677&#x02013;686. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4873392/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27089382" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Immunol&amp;title=Mycobacterium+tuberculosis+induces+the+miR-33+locus+to+reprogram+autophagy+and+host+lipid+metabolism&amp;volume=17&amp;issue=6&amp;publication_year=2016&amp;pages=677-686&amp;pmid=27089382&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0048">[48] <span class="mixed-citation">Kolodgie FD, Gold HK, Burke AP, et al. <span class="ref-title">Intraplaque hemorrhage and progression of coronary atheroma</span>. <span class="ref-journal">N Engl J Med</span>. 2003;<span class="ref-vol">349</span>:2316&#x02013;2325. [<a href="https://pubmed.ncbi.nlm.nih.gov/14668457" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Intraplaque+hemorrhage+and+progression+of+coronary+atheroma&amp;volume=349&amp;publication_year=2003&amp;pages=2316-2325&amp;pmid=14668457&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0049">[49] <span class="mixed-citation">Tough DF, Prinjha RK. <span class="ref-title">Immune disease-associated variants in gene enhancers point to BET epigenetic mechanisms for therapeutic intervention</span>. <span class="ref-journal">Epigenomics</span>. 2016;<span class="ref-vol">9</span>:573&#x02013;584. [<a href="https://pubmed.ncbi.nlm.nih.gov/27925476" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Epigenomics&amp;title=Immune+disease-associated+variants+in+gene+enhancers+point+to+BET+epigenetic+mechanisms+for+therapeutic+intervention&amp;volume=9&amp;publication_year=2016&amp;pages=573-584&amp;pmid=27925476&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0050">[50] <span class="mixed-citation">de Medeiros RB. <span class="ref-title">Sequential chromatin immunoprecipitation assay and analysis</span>. <span class="ref-journal">Methods Mol Biol</span>. 2011;<span class="ref-vol">791</span>:225&#x02013;237. [<a href="https://pubmed.ncbi.nlm.nih.gov/21913083" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Mol+Biol&amp;title=Sequential+chromatin+immunoprecipitation+assay+and+analysis&amp;volume=791&amp;publication_year=2011&amp;pages=225-237&amp;pmid=21913083&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="cit0051">[51] <span class="mixed-citation">Truax AD, Greer SF. <span class="ref-title">ChIP and Re-ChIP assays: investigating interactions between regulatory proteins, histone modifications, and the DNA sequences to which they bind</span>. <span class="ref-journal">Methods Mol Biol</span>. 2012;<span class="ref-vol">809</span>:175&#x02013;188. [<a href="https://pubmed.ncbi.nlm.nih.gov/22113276" ref="reftype=pubmed&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Mol+Biol&amp;title=ChIP+and+Re-ChIP+assays:+investigating+interactions+between+regulatory+proteins,+histone+modifications,+and+the+DNA+sequences+to+which+they+bind&amp;volume=809&amp;publication_year=2012&amp;pages=175-188&amp;pmid=22113276&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8942508&amp;issue-id=403716&amp;journal-id=1359&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Autophagy</span> are provided here courtesy of <strong>Taylor &#x0026; Francis</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC8942508/pdf/KAUP_18_1936355.pdf" class="int-view">PDF (8.4M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/8942508/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/8942508/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="8942508" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC8942508%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="Xpgq7VdB92E50VkXWmvWp4K60dB04d2ZvXLXexrEtTH1SucXWS4XD4lDJy7S7XHD">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC8942508%2F&amp;text=Epigenetic%20reader%20BRD4%20supports%20mycobacterial%20pathogenesis%20by%20co-modulating%20host%20lipophagy%20and%20angiogenesis" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC8942508%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8942508/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC8942508/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/34074211/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC8942508/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/34074211/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC8942508/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/8942508/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC8942508/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
