
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-manuscript.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Proteome-wide screening for mitogen-activated protein kinase docking motifs and interactors - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="D0BD41E4AC192A4500003A9285098B9C.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="nihpa" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC9995140/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9995140/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Science signaling" /><meta name="citation_title" content="Proteome-wide screening for mitogen-activated protein kinase docking motifs and interactors" /><meta name="citation_author" content="Guangda Shi" /><meta name="citation_author_institution" content="Department of Pharmacology, Yale University School of Medicine, New Haven, CT 06520, USA" /><meta name="citation_author" content="Claire Song" /><meta name="citation_author_institution" content="Department of Pharmacology, Yale University School of Medicine, New Haven, CT 06520, USA" /><meta name="citation_author" content="Jaylissa Torres Robles" /><meta name="citation_author_institution" content="Department of Pharmacology, Yale University School of Medicine, New Haven, CT 06520, USA" /><meta name="citation_author_institution" content="Department of Chemistry, Yale University, New Haven, CT 06520, USA" /><meta name="citation_author" content="Leonidas Salichos" /><meta name="citation_author_institution" content="Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, CT 06510, USA" /><meta name="citation_author" content="Hua Jane Lou" /><meta name="citation_author_institution" content="Department of Pharmacology, Yale University School of Medicine, New Haven, CT 06520, USA" /><meta name="citation_author" content="TuKiet T. Lam" /><meta name="citation_author_institution" content="Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, CT 06510, USA" /><meta name="citation_author_institution" content="Keck MS and Proteomics Resource, Yale University School of Medicine, New Haven, CT 06510, USA" /><meta name="citation_author" content="Mark Gerstein" /><meta name="citation_author_institution" content="Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, CT 06510, USA" /><meta name="citation_author" content="Benjamin E. Turk" /><meta name="citation_author_institution" content="Department of Pharmacology, Yale University School of Medicine, New Haven, CT 06520, USA" /><meta name="citation_publication_date" content="2023/01/01" /><meta name="citation_issue" content="767" /><meta name="citation_volume" content="16" /><meta name="citation_firstpage" content="eabm5518" /><meta name="citation_doi" content="10.1126/scisignal.abm5518" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC9995140/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC9995140/" /><meta name="citation_pmid" content="36626580" /><meta name="DC.Title" content="Proteome-wide screening for mitogen-activated protein kinase docking motifs and interactors" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="NIH Public Access" /><meta name="DC.Contributor" content="Guangda Shi" /><meta name="DC.Contributor" content="Claire Song" /><meta name="DC.Contributor" content="Jaylissa Torres Robles" /><meta name="DC.Contributor" content="Leonidas Salichos" /><meta name="DC.Contributor" content="Hua Jane Lou" /><meta name="DC.Contributor" content="TuKiet T. Lam" /><meta name="DC.Contributor" content="Mark Gerstein" /><meta name="DC.Contributor" content="Benjamin E. Turk" /><meta name="DC.Date" content="2023 Jan 10" /><meta name="DC.Identifier" content="10.1126/scisignal.abm5518" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Proteome-wide screening for mitogen-activated protein kinase docking motifs and interactors" /><meta property="og:type" content="article" /><meta property="og:description" content="Essential functions of mitogen-activated protein kinases (MAPKs) depend on their capacity to selectively phosphorylate a limited repertoire of substrates. MAPKs harbor a conserved groove located outside of the catalytic cleft that binds to short linear ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9995140/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means itâ€™s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure youâ€™re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC9995140/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC9995140
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC9995140/pdf/nihms-1871559.pdf" class="int-view">PDF (1.8M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/9995140/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/9995140/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="9995140" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC9995140%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="8QXVS6dRspT2mL4NFRZ0RcAWBG8xgZKxeCIkE8Me3owvbOvK16h4qiY2LxiXBQ45">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC9995140%2F&amp;text=Proteome-wide%20screening%20for%20mitogen-activated%20protein%20kinase%20docking%20motifs%20and%20interactors" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC9995140%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9995140/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC9995140/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/36626580/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC9995140/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/36626580/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC9995140/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/9995140/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC9995140
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-nihpa.png" alt="Logo of nihpa" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="https://www.hhs.gov/open/public-access-guiding-principles/index.html" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div><span id="pmcmata">Sci Signal.</span> Author manuscript; available in PMC 2023 Jul 10.</div><div></div><div class="final-form"><em>Published in final edited form as:</em></div><div class="final-citation"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=36626580" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Sci Signal. 2023 Jan 10; 16(767): eabm5518. </a></div><span class="fm-vol-iss-date">Published online 2023 Jan 10. </span> <span class="doi"><span>doi:Â </span><a href="//doi.org/10.1126%2Fscisignal.abm5518" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1126/scisignal.abm5518</a></span></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC9995140</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS1871559</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/36626580">36626580</a></div></div></div><h1 class="content-title">Proteome-wide screening for mitogen-activated protein kinase docking motifs and interactors</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shi%20G%5BAuthor%5D" class="affpopup" co-rid="_co_idm140171986788400" co-class="co-affbox">Guangda Shi</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Song%20C%5BAuthor%5D" class="affpopup" co-rid="_co_idm140172028702144" co-class="co-affbox">Claire Song</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Torres%20Robles%20J%5BAuthor%5D" class="affpopup" co-rid="_co_idm140172028700608" co-class="co-affbox">Jaylissa Torres Robles</a>,<sup>1,</sup><sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Salichos%20L%5BAuthor%5D" class="affpopup" co-rid="_co_idm140171974631568" co-class="co-affbox">Leonidas Salichos</a>,<sup>3</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Jane%20Lou%20H%5BAuthor%5D" class="affpopup" co-rid="_co_idm140171974629696" co-class="co-affbox">Hua Jane Lou</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Lam%20TT%5BAuthor%5D" class="affpopup" co-rid="_co_idm140171988614592" co-class="co-affbox">TuKiet T. Lam</a>,<sup>3,</sup><sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gerstein%20M%5BAuthor%5D" class="affpopup" co-rid="_co_idm140171988611872" co-class="co-affbox">Mark Gerstein</a>,<sup>3</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Turk%20BE%5BAuthor%5D" class="affpopup" co-rid="_co_idm140171992670512" co-class="co-affbox">Benjamin E. Turk</a><sup>1,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140171986788400"><h3 class="no_margin">Guangda Shi</h3><p><sup>1</sup>Department of Pharmacology, Yale University School of Medicine, New Haven, CT 06520, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shi%20G%5BAuthor%5D">Guangda Shi</a></div></div><div id="_co_idm140172028702144"><h3 class="no_margin">Claire Song</h3><p><sup>1</sup>Department of Pharmacology, Yale University School of Medicine, New Haven, CT 06520, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Song%20C%5BAuthor%5D">Claire Song</a></div></div><div id="_co_idm140172028700608"><h3 class="no_margin">Jaylissa Torres Robles</h3><p><sup>1</sup>Department of Pharmacology, Yale University School of Medicine, New Haven, CT 06520, USA</p><p><sup>2</sup>Department of Chemistry, Yale University, New Haven, CT 06520, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Torres%20Robles%20J%5BAuthor%5D">Jaylissa Torres Robles</a></div></div><div id="_co_idm140171974631568"><h3 class="no_margin">Leonidas Salichos</h3><p><sup>3</sup>Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, CT 06510, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Salichos%20L%5BAuthor%5D">Leonidas Salichos</a></div></div><div id="_co_idm140171974629696"><h3 class="no_margin">Hua Jane Lou</h3><p><sup>1</sup>Department of Pharmacology, Yale University School of Medicine, New Haven, CT 06520, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Jane%20Lou%20H%5BAuthor%5D">Hua Jane Lou</a></div></div><div id="_co_idm140171988614592"><h3 class="no_margin">TuKiet T. Lam</h3><p><sup>3</sup>Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, CT 06510, USA</p><p><sup>4</sup>Keck MS and Proteomics Resource, Yale University School of Medicine, New Haven, CT 06510, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Lam%20TT%5BAuthor%5D">TuKiet T. Lam</a></div></div><div id="_co_idm140171988611872"><h3 class="no_margin">Mark Gerstein</h3><p><sup>3</sup>Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, CT 06510, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gerstein%20M%5BAuthor%5D">Mark Gerstein</a></div></div><div id="_co_idm140171992670512"><h3 class="no_margin">Benjamin E. Turk</h3><p><sup>1</sup>Department of Pharmacology, Yale University School of Medicine, New Haven, CT 06520, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Turk%20BE%5BAuthor%5D">Benjamin E. Turk</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140172029319360_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140172029319360_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140172029319360_ai" style="display:none"><div class="fm-affl" id="A1"><sup>1</sup>Department of Pharmacology, Yale University School of Medicine, New Haven, CT 06520, USA</div><div class="fm-affl" id="A2"><sup>2</sup>Department of Chemistry, Yale University, New Haven, CT 06520, USA</div><div class="fm-affl" id="A3"><sup>3</sup>Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, CT 06510, USA</div><div class="fm-affl" id="A4"><sup>4</sup>Keck MS and Proteomics Resource, Yale University School of Medicine, New Haven, CT 06510, USA</div><div id="CR1"><sup>*</sup>Corresponding author. <a href="mailto:dev@null" data-email="ude.elay@krut.neb" class="oemail">ude.elay@krut.neb</a></div><div><p id="P1" class="p p-first-last"><strong>Author contributions:</strong> G.S, J.T.R., C.S., and B.E.T. designed experiments. G.S. conducted all experiments except for analysis of protein phosphorylation in cells, which was conducted by C.S., and generation of inducible cell lines, which was performed by J.T.R. T.T.L carried out the LC-MS/MS data analysis with downstream quantitative analyses refined by C.S and B.E.T. Some expression constructs and purified proteins were generated by H.J.L. G.S., L.S., M.G. and B.E.T. designed the proteomic library. G.S. and B.E.T. drafted the manuscript, with additional editing from J.T.R., C.S. and L.S.</p></div></div><div class="fm-article-notes half_rhythm"></div><div class="permissions half_rhythm hide" id="idm140172029319360_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">PMC Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm" role="complementary" aria-label="Related or updated information about this article."><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=36626580" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Sci Signal</a></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="11" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SM Excel data file 1.</strong></div><div><a href="/pmc/articles/PMC9995140/bin/NIHMS1871559-supplement-SM_Excel_data_file_1.xlsx" data-ga-action="click_feat_suppl">NIHMS1871559-supplement-SM_Excel_data_file_1.xlsx</a><span style="color:gray"> (556K)</span></div><div class="small guid">GUID:Â 90AF42BD-4F84-4EDE-9130-D6E91D846832</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SM Excel data file 2.</strong></div><div><a href="/pmc/articles/PMC9995140/bin/NIHMS1871559-supplement-SM_Excel_data_file_2.xlsx" data-ga-action="click_feat_suppl">NIHMS1871559-supplement-SM_Excel_data_file_2.xlsx</a><span style="color:gray"> (303K)</span></div><div class="small guid">GUID:Â 90D06D97-5D39-465D-9493-A95F9C4CE866</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SM Excel data file 3.</strong></div><div><a href="/pmc/articles/PMC9995140/bin/NIHMS1871559-supplement-SM_Excel_data_file_3.xlsx" data-ga-action="click_feat_suppl">NIHMS1871559-supplement-SM_Excel_data_file_3.xlsx</a><span style="color:gray"> (78K)</span></div><div class="small guid">GUID:Â 625D17BE-1322-4664-A1F8-9EC02F31D3FC</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SM Excel data file 4.</strong></div><div><a href="/pmc/articles/PMC9995140/bin/NIHMS1871559-supplement-SM_Excel_data_file_4.xlsx" data-ga-action="click_feat_suppl">NIHMS1871559-supplement-SM_Excel_data_file_4.xlsx</a><span style="color:gray"> (8.9M)</span></div><div class="small guid">GUID:Â 57D3EE78-9C12-4E89-85D2-1998534CFCC0</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SM data file 5.</strong></div><div><a href="/pmc/articles/PMC9995140/bin/NIHMS1871559-supplement-SM_data_file_5.xlsx" data-ga-action="click_feat_suppl">NIHMS1871559-supplement-SM_data_file_5.xlsx</a><span style="color:gray"> (2.6M)</span></div><div class="small guid">GUID:Â 56740AA4-D1D6-42A0-9BFC-AAB79F67F75F</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SM data file 6.</strong></div><div><a href="/pmc/articles/PMC9995140/bin/NIHMS1871559-supplement-SM_data_file_6.xlsx" data-ga-action="click_feat_suppl">NIHMS1871559-supplement-SM_data_file_6.xlsx</a><span style="color:gray"> (634K)</span></div><div class="small guid">GUID:Â 000ADE21-22FF-496C-A433-2C983E0289C0</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SM data file 7.</strong></div><div><a href="/pmc/articles/PMC9995140/bin/NIHMS1871559-supplement-SM_data_file_7.xlsx" data-ga-action="click_feat_suppl">NIHMS1871559-supplement-SM_data_file_7.xlsx</a><span style="color:gray"> (228K)</span></div><div class="small guid">GUID:Â A69BB4C1-9367-4ADB-8CE1-06F57AD766AB</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SM data file 8.</strong></div><div><a href="/pmc/articles/PMC9995140/bin/NIHMS1871559-supplement-SM_data_file_8.xlsx" data-ga-action="click_feat_suppl">NIHMS1871559-supplement-SM_data_file_8.xlsx</a><span style="color:gray"> (192K)</span></div><div class="small guid">GUID:Â C1AF2A82-3C2F-4803-A575-B70F0202F49E</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SM data file 9.</strong></div><div><a href="/pmc/articles/PMC9995140/bin/NIHMS1871559-supplement-SM_data_file_9.xlsx" data-ga-action="click_feat_suppl">NIHMS1871559-supplement-SM_data_file_9.xlsx</a><span style="color:gray"> (14K)</span></div><div class="small guid">GUID:Â 98929D83-9682-4D25-82F9-B34027303047</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SM data file 10.</strong></div><div><a href="/pmc/articles/PMC9995140/bin/NIHMS1871559-supplement-SM_data_file_10.xlsx" data-ga-action="click_feat_suppl">NIHMS1871559-supplement-SM_data_file_10.xlsx</a><span style="color:gray"> (12M)</span></div><div class="small guid">GUID:Â AF12CC61-94EE-4F88-982D-26CC68F063A7</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplemental File.</strong></div><div><a href="/pmc/articles/PMC9995140/bin/NIHMS1871559-supplement-Supplemental_File.docx" data-ga-action="click_feat_suppl">NIHMS1871559-supplement-Supplemental_File.docx</a><span style="color:gray"> (3.1M)</span></div><div class="small guid">GUID:Â D2A05502-73B6-4095-B190-B8EBAD3BF5BE</div></div></dd></dl></div><div id="ABS1" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="ABS1title">Abstract</h2><!--article-meta--><div><p id="P2" class="p p-first-last">Essential functions of mitogen-activated protein kinases (MAPKs) depend on their capacity to selectively phosphorylate a limited repertoire of substrates. MAPKs harbor a conserved groove located outside of the catalytic cleft that binds to short linear sequence motifs found in substrates and regulators. However, the weak and transient nature of these &#x0201c;docking&#x0201d; interactions poses a challenge to defining MAPK interactomes and associated sequence motifs. Here, we describe a yeast-based genetic screening pipeline to evaluate large collections of MAPK docking sequences in parallel. Using this platform, we analyzed a combinatorial library based on the docking sequences from the MAPK kinases MKK6 and MKK7, defining features critical for binding to the stress-activated MAPKs JNK1 and p38&#x003b1;. Our screen of a library consisting of ~12,000 sequences from the human proteome revealed multiple MAPK-selective interactors, including many that did not conform to previously defined docking motifs. Analysis of p38&#x003b1;/JNK1 exchange mutants identified specific docking groove residues that mediate selective binding. Finally, we verified that docking sequences identified in the screen functioned in substrate recruitment in vitro and in cultured cells. Together, these studies establish an approach to characterize MAPK docking sequences and provide a resource for future investigation of signaling downstream of p38 and JNK.</p></div></div><div id="S1" class="tsec sec"><h2 class="head no_bottom_margin" id="S1title">INTRODUCTION</h2><p id="P3" class="p p-first">Protein kinases function by phosphorylating a limited number of effector substrates (<a href="#R1" rid="R1" class=" bibr popnode">1</a>). Kinases achieve specificity in part through complementarity between the catalytic cleft and residues surrounding the site of phosphorylation. However, closely related kinases with identical phosphorylation site specificity can nonetheless target distinct sets of substrates. A key example is the mitogen-activated protein kinase (MAPK) family, a group of Ser-Thr kinases conserved widely in eukaryotes (<a href="#R2" rid="R2" class=" bibr popnode">2</a>). Canonical animal MAPKs, including the extracellular signal-regulated kinases (ERKs), c-Jun N-terminal kinases (JNKs) and p38 MAPKs (hereafter, p38), are positioned at the bottom of three-tiered kinase cascades activated in response to diverse cellular stimuli. The different MAPK subfamilies phosphorylate largely unique sets of substrates to elicit distinct cellular responses, yet each targets a minimal Ser/Thr-Pro consensus sequence. MAPK specificity is at least in part driven by docking interactions, in which regions of the kinase outside of the catalytic cleft recruit substrates through binding sites distal from their sites of phosphorylation.</p><p id="P4">A major hub for MAPK docking interactions is the D-recruitment site (DRS), a conserved region of the catalytic domain that binds to substrates, scaffold proteins, MAPK kinases (MKKs) and MAPK phosphatases (MKPs) (<a href="#R3" rid="R3" class=" bibr popnode">3</a>-<a href="#R5" rid="R5" class=" bibr popnode">5</a>). Several highly conserved binding partners bind to the DRS through domains with intrinsic tertiary structure (<a href="#R6" rid="R6" class=" bibr popnode">6</a>). However, the DRS more generally recognizes short linear motifs (SLiMs) termed D-sites found in unstructured regions of MAPK interactors (<a href="#R7" rid="R7" class=" bibr popnode">7</a>-<a href="#R10" rid="R10" class=" bibr popnode">10</a>). D-sites bind to MAPKs with moderate affinity (~100 nM &#x02013; 30 &#x003bc;M), promoting transient kinase-substrate interactions and dynamic remodeling of signaling networks in response to stimuli (<a href="#R7" rid="R7" class=" bibr popnode">7</a>). DRS engagement can also impact the conformation and dynamics of the MAPK catalytic domain, which may be important in promoting activation by MKKs and inactivation by MKPs (<a href="#R11" rid="R11" class=" bibr popnode">11</a>-<a href="#R13" rid="R13" class=" bibr popnode">13</a>).</p><p id="P5">The MAPK DRS forms a groove consisting of three adjacent hydrophobic pockets (designated the &#x003a6;<sub>L</sub>, &#x003a6;<sub>A</sub> and &#x003a6;<sub>B</sub> pockets) and a proximal negatively charged &#x0201c;common docking&#x0201d; (CD) site (<a href="#R7" rid="R7" class=" bibr popnode">7</a>, <a href="#R8" rid="R8" class=" bibr popnode">8</a>, <a href="#R14" rid="R14" class=" bibr popnode">14</a>, <a href="#R15" rid="R15" class=" bibr popnode">15</a>). D-site sequences include a cluster of basic residues complementary to the CD site, a variable linker, and two or three hydrophobic (&#x003d5;) residues (in a &#x003d5;<sub>L</sub>-x-x-&#x003d5;<sub>A</sub>-x-&#x003d5;<sub>B</sub>, &#x003d5;<sub>L</sub>-x-&#x003d5;<sub>A</sub>-x-&#x003d5;<sub>B</sub>, or &#x003d5;<sub>A</sub>-x-&#x003d5;<sub>B</sub> arrangement) that engage the corresponding &#x003a6; pockets (<a href="#R10" rid="R10" class=" bibr popnode">10</a>, <a href="#R15" rid="R15" class=" bibr popnode">15</a>). Additional sequence features within the context of this general motif appear to confer selectivity for particular MAPK subfamilies (<a href="#R7" rid="R7" class=" bibr popnode">7</a>, <a href="#R9" rid="R9" class=" bibr popnode">9</a>, <a href="#R16" rid="R16" class=" bibr popnode">16</a>, <a href="#R17" rid="R17" class=" bibr popnode">17</a>). Structural studies of MAPK&#x02022;D-site complexes have revealed distinct binding modes associated with these subfamily-selective motifs, driven by arrangement of the hydrophobic residues as well as the sequence composition and bound conformation of the D-site linker sequence (<a href="#R7" rid="R7" class=" bibr popnode">7</a>, <a href="#R8" rid="R8" class=" bibr popnode">8</a>, <a href="#R11" rid="R11" class=" bibr popnode">11</a>, <a href="#R18" rid="R18" class=" bibr popnode">18</a>-<a href="#R20" rid="R20" class=" bibr popnode">20</a>).</p><p id="P6" class="p p-last">Knowledge of these MAPK-selective D-site motifs has driven computational approaches to identify previously unknown MAPK substrates by scanning protein databases for sequences similar to previously established interactors (<a href="#R9" rid="R9" class=" bibr popnode">9</a>, <a href="#R21" rid="R21" class=" bibr popnode">21</a>). However, experimental approaches for the discovery of additional MAPK-selective sequence motifs are needed to better define MAPK interactomes and to understand how they assemble. Here, we describe a genetically encoded library screening platform to identify new MAPK-interacting D-sites that exploits the conservation of the core cascade from humans to budding yeast. We used this strategy to analyze a saturation mutagenesis library of MKK-derived D-sites interacting with the MAPKs JNK1 and p38&#x003b1; and discovered previously unknown features of their corresponding motifs. We subsequently screened these MAPKs against a large library of human proteomic sequences, which allowed us to further refine the motifs and to identify new direct interaction partners. The screen differentiated between known p38&#x003b1; and JNK1 interactors and identified several binding partners that are selective for either kinase. The dataset therefore provides an unbiased resource to probe selective MAPK signaling pathways and will be an important resource for future investigation into MAPK biology.</p></div><div id="S2" class="tsec sec"><h2 class="head no_bottom_margin" id="S2title">RESULTS</h2><div id="S3" class="sec sec-first"><h3 id="S3title">A yeast-based system for evaluation of MAPK docking sequences</h3><p id="P7" class="p p-first">We exploited the evolutionary conservation of core MAPK cascades to enable screening complex D-site sequence libraries in budding yeast. Yeast require the p38/JNK ortholog Hog1 to adapt to osmotic stress (<a href="#R22" rid="R22" class=" bibr popnode">22</a>). Normal activation of the Hog1 cascade by hyperosmolarity induces transient cell cycle arrest, but constitutive signaling causes stable growth suppression (<a href="#R23" rid="R23" class=" bibr popnode">23</a>) (<a href="/pmc/articles/PMC9995140/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1A</span></a>). Presumably because it is deregulated in yeast, we found that co-expression of mammalian MKK6 with p38&#x003b1; was toxic to a strain lacking Hog1 and its cognate MKK Pbs2 (<a href="#SD11" rid="SD11" class=" supplementary-material">fig. S1</a>). MKK6 harbors a D-site located upstream of the catalytic domain (<a href="/pmc/articles/PMC9995140/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1B</span></a>) that promotes its phosphorylation of p38&#x003b1; (<a href="#R24" rid="R24" class=" bibr popnode">24</a>) . Because growth suppression in the context of wild-type (WT) p38&#x003b1;, which partially autophosphorylates in yeast, occurred independently of the MKK6 D-site (<a href="#SD11" rid="SD11" class=" supplementary-material">fig. S1</a>), we examined a fully MKK-dependent p38&#x003b1; mutant (L195A) (<a href="#R25" rid="R25" class=" bibr popnode">25</a>). Growth impairment by p38&#x003b1;<sup>L195A</sup> was less severe than that by WT p38&#x003b1;, depended on an intact MKK6 D-site, and was partly alleviated by exchanging the MKK6 D-site with the highest affinity D-site from MKK7 (MKK6<sup>D7</sup>) (<a href="#R26" rid="R26" class=" bibr popnode">26</a>) (<a href="/pmc/articles/PMC9995140/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1C</span></a>). Furthermore, MKK6<sup>D7</sup>, but not WT MKK6 or MKK6<sup>&#x00394;D</sup>, partly suppressed cell growth when co-expressed with the analogous JNK1<sup>L198A</sup> mutant (<a href="/pmc/articles/PMC9995140/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1D</span></a>). These results provide a system in which yeast growth is inhibited by a D-site interaction with either p38&#x003b1; or JNK1.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC9995140/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140171928554576"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9995140_nihms-1871559-f0001.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-1871559-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC9995140/bin/nihms-1871559-f0001.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140171928554576"><a target="object" rel="noopener" href="/pmc/articles/PMC9995140/figure/F1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC9995140/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>A system coupling MAPK D-site interactions to yeast cell growth.</strong><p id="P49">(A) Scheme showing the impact of replacing yeast MAPK pathway components with human homologs on cell growth. (B) Domain structure of MKK6 and D-site variants. MKK6<sup>D7</sup> replaces the native p38-selective D-site with that from MKK7, which binds only to JNK. (C) Growth assay for a <em>hog1&#x00394; pbs2&#x00394;</em> strain co-expressing p38&#x003b1; and the indicated MKK6 variants. Cells were grown in liquid culture, derepressed in raffinose media, and then spotted in 5-fold serial dilutions on solid media containing either galactose (to induce p38 expression) or glucose. Representative of 2 independent experiments. (D) Growth assay of a <em>hog1&#x00394; pbs2&#x00394;</em> strain co-expressing JNK1 and MKK6 variants conducted as in (C). Representative of 2 independent experiments. (E) Scheme showing potential mechanisms for growth rescue resulting from the expression of GST-D-site fusion proteins. (F) Effect of expressing cognate (MKK6, MEF2A) or non-cognate (MKK7) D-sites fused to GST on growth arrest mediated by p38&#x003b1;-MKK6 co-expression in a <em>hog1&#x00394; pbs2&#x00394;</em> strain. Cells were grown and plated as in (C). Representative of 2 independent experiments. (G) Effect of expressing cognate (MKK7, NFAT4) or non-cognate (MKK6) D-sites fused to GST on growth arrest mediated by JNK1-MKK6<sup>D7</sup> co-expression in a <em>hog1&#x00394; pbs2&#x00394;</em> strain. Representative of 2 independent experiments.</p></div></div></div><p id="P8" class="p p-last">To evaluate D-site sequences for their MAPK binding capability, we introduced a third component into this system by ectopically expressing a D-site peptide. We reasoned that engagement of the MAPK DRS by the D-site would rescue growth inhibition by blocking either MAPK-MKK or MAPK-substrate binding (<a href="/pmc/articles/PMC9995140/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1E</span></a>). We first engineered <em>hog1&#x00394; pbs2&#x00394;</em> strains harboring chromosomally integrated cassettes for constitutive expression of the MKK (WT MKK6 or MKK6<sup>D7</sup>) and for inducible expression of the MAPK (p38&#x003b1;<sup>L195A</sup> or JNK1<sup>L198A</sup>). We next introduced plasmids inducibly expressing D-site peptides fused to glutathione S-transferase (GST) into these strains and assessed their growth under inducing or non-inducing conditions. Expression of p38&#x003b1;-binding D-site peptides derived from MKK6 or MEF2A both substantially reversed growth impairment associated with co-expression of MKK6 and p38&#x003b1;<sup>L195A</sup>, whereas the MKK7 D-peptide improved growth to a lesser extent (<a href="/pmc/articles/PMC9995140/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1F</span></a>). Likewise, D-site peptides from MKK7 and the JNK substrate NFAT4, but not from MKK6, rescued growth of the strain expressing MKK6<sup>D7</sup> and JNK1<sup>L198A</sup> (<a href="/pmc/articles/PMC9995140/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1G</span></a>). These experiments establish a system in which expression of a MAPK-binding D-site is coupled to cell growth, providing the basis for screens to identify sequences engaging the MAPK DRS.</p></div><div id="S4" class="sec"><h3 id="S4title">Positional scanning mutagenesis of MAPK docking sites</h3><p id="P9" class="p p-first">The importance of specific D-site residues in mediating selective interaction with MAPKs has been established by analysis of mutant proteins and synthetic peptides (<a href="#R16" rid="R16" class=" bibr popnode">16</a>, <a href="#R17" rid="R17" class=" bibr popnode">17</a>, <a href="#R27" rid="R27" class=" bibr popnode">27</a>, <a href="#R28" rid="R28" class=" bibr popnode">28</a>), but low-throughput approaches can only analyze a limited number of variants. To analyze docking specificity more comprehensively using our yeast system, we designed a library of 715 unique sequences that included all possible single amino acid substitutions, as well as double Ala substitutions in all pairwise combinations, to D-sites from MKK6 and MKK7 (<a href="/pmc/articles/PMC9995140/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2A</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Data File S1</a>). The MKK7 library was based on the second of its three D-sites, which has the highest affinity and has been co-crystallized with JNK1 (<a href="#R13" rid="R13" class=" bibr popnode">13</a>, <a href="#R26" rid="R26" class=" bibr popnode">26</a>). Oligonucleotides encoding these sequences were custom synthesized and cloned as a pool into the yeast GST fusion vector (<a href="/pmc/articles/PMC9995140/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2B</span></a>). Plasmid pools were introduced into MAPK/MKK-expressing yeast strains, and liquid cultures were expanded in derepressing (raffinose) media. A portion of the culture was reserved for sequencing, and the remainder was split and propagated under either inducing (raffinose + galactose) or non-inducing (glucose) conditions. At various times, cultures were sampled, plasmids were extracted from cells, and the D-site regions were PCR amplified with barcoding primers. PCR samples were pooled and analyzed by Illumina sequencing, providing the relative abundance of each component of the library at each time point. Although no sequences changed substantially (&#x0003e;10%) in relative abundance under non-inducing conditions, library representation became skewed upon induction of either MAPK (<a href="/pmc/articles/PMC9995140/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2C</span></a>,<a href="/pmc/articles/PMC9995140/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,D,</span></span><span>D</span></a>, <a href="#SD11" rid="SD11" class=" supplementary-material">fig. S2A</a>,<a href="#SD11" rid="SD11" class=" supplementary-material">B</a>, <a href="#SD2" rid="SD2" class=" supplementary-material">Data File S2</a>). The change of representation of each variant over time was fit to an exponential function to provide a measure of its relative depletion or enrichment within the screen (<a href="/pmc/articles/PMC9995140/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2C</span></a>,<a href="/pmc/articles/PMC9995140/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,D,</span></span><span>D</span></a>, <a href="#SD2" rid="SD2" class=" supplementary-material">Data Files S2</a> and <a href="#SD3" rid="SD3" class=" supplementary-material">S3</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC9995140/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140172028280864"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9995140_nihms-1871559-f0002.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-1871559-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC9995140/bin/nihms-1871559-f0002.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140172028280864"><a target="object" rel="noopener" href="/pmc/articles/PMC9995140/figure/F2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC9995140/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Combinatorial library screens.</strong><p id="P50">(A) MKK6 and MKK7 sequences with the core elements of the D-site motif highlighted. (B) A schematic showing the workflow of the screen. (C) Waterfall plots showing the average Z-score for the enrichment/depletion rate of each D-site sequence variant from two independent screens against JNK1 (left) or p38&#x003b1; (right). WT MKK6 and MKK7 D-sites are highlighted. Read counts from Illumina sequencing runs are provided in <a href="#SD2" rid="SD2" class=" supplementary-material">Data File S2</a>. (D) Scatter plot showing correlation of Z-scores between two replicate screens for JNK (left) and p38&#x003b1; (right). Blue, MKK7 variants; red, MKK6 variants. (E) Heatmaps showing the effect of each amino acid substitution to the MKK7 D-site in the JNK screen (left) and MKK6 D-site in the p38&#x003b1; screen (right). Values are normalized to the respective WT sequence (white), with red indicating enrichment and blue indicating depletion of the sequence from the population. Crossmarked boxes indicate sequences missing from the screening library. (F and G) Heat maps showing mean enrichment scores of double alanine substitutions at each position in the MKK7 D-site sequence in the JNK1 screen (F) or the MKK6 D-site sequence in the p38&#x003b1; screen (G). In the color scheme, white indicates the enrichment score for the WT sequence. N=2 independent experiments. (H and I) Crystal structure of JNK1 (H) or p38&#x003b1; (I) (tan) in complex with MKK7 (H) or MKK6 (I) D-site peptides (cyan) with the CD region and three hydrophobic pockets indicated (PDB entries 4UX9 and 5ETF) (<a href="#R13" rid="R13" class=" bibr popnode">13</a>) (<a href="#R29" rid="R29" class=" bibr popnode">29</a>).</p></div></div></div><p id="P10" class="p p-last">As anticipated, variants derived from cognate D-sites (MKK6 for p38&#x003b1;; MKK7 for JNK1) were generally enriched during selection in two separate experiments, indicative of a successful interaction, whereas with few exceptions non-cognate D-sites were depleted and presumably did not bind to the MAPK (<a href="/pmc/articles/PMC9995140/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig 2D</span></a>). Most substitutions to cognate D-sites resulted in slower growth rates and thus reduced binding affinities, suggesting that the WT sequences are largely optimal, but some variants displayed improved binding to either p38&#x003b1; or JNK1 (<a href="/pmc/articles/PMC9995140/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2E</span></a>,<a href="/pmc/articles/PMC9995140/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,F</span></span><span>F</span></a>,<a href="/pmc/articles/PMC9995140/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,G).</span></span><span>G</span></a>). In the MKK7 D-site, Leu<sup>8</sup> and Leu<sup>10</sup>, which engage the &#x003a6;<sub>L</sub> and &#x003a6;<sub>A</sub> pockets of JNK (<a href="#R13" rid="R13" class=" bibr popnode">13</a>), appeared to be the most critical for binding, whereas Leu<sup>12</sup>, which binds the &#x003a6;<sub>B</sub> pocket, was more tolerant to substitution (<a href="/pmc/articles/PMC9995140/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig 2E</span></a>,<a href="/pmc/articles/PMC9995140/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,F</span></span><span>F</span></a>,<a href="/pmc/articles/PMC9995140/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,H).</span></span><span>H</span></a>). As anticipated, substitutions to the N-terminally positioned basic residues Arg<sup>3</sup> and Arg<sup>5</sup> of the MKK7 D-site (<a href="/pmc/articles/PMC9995140/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2A</span></a>), which bind to the CD region, were also deleterious. In the JNK1 screen, the largest increase in growth was observed by aromatic substitutions to Gln<sup>9</sup> of the MKK7 D-site. A Tyr residue located at the analogous position in the NFAT4 D-site appears to provide intra- and intermolecular van der Waals contacts in the co-crystal structure with JNK1 (<a href="#SD11" rid="SD11" class=" supplementary-material">fig. S3A</a>) (<a href="#R7" rid="R7" class=" bibr popnode">7</a>). In the p38&#x003b1; screen, the most deleterious single (<a href="/pmc/articles/PMC9995140/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2E</span></a>) and double (<a href="/pmc/articles/PMC9995140/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2G</span></a>) amino acid substitutions were to Leu<sup>10</sup> and Ile<sup>12</sup> in the MKK6 D-site, which engage the &#x003a6;<sub>A</sub> and &#x003a6;<sub>B</sub> pockets in the p38 hydrophobic groove (<a href="/pmc/articles/PMC9995140/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2I</span></a>) (<a href="#R7" rid="R7" class=" bibr popnode">7</a>, <a href="#R29" rid="R29" class=" bibr popnode">29</a>). As anticipated, substitutions to basic residues in the N-terminal cluster of the D-site (<a href="/pmc/articles/PMC9995140/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2A</span></a>) were also disfavored, and incorporation of additional Lys or Arg residues in this region appeared to promote binding. Surprisingly, mutating either Lys<sup>11</sup> or Lys<sup>14</sup> near the D-site C-terminus, not previously reported as essential for interaction with p38&#x003b1;, also reduced binding. Compared to JNK1, p38&#x003b1; was generally less tolerant of acidic residues in the D-site sequence, particularly in the C-terminal region (positions 11 &#x02013; 14, <a href="/pmc/articles/PMC9995140/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2A</span></a>). These observations suggest that p38&#x003b1; D-site binding may be in part driven by bulk electrostatics, rather than position-specific ionic interactions. At two sites near the D-site N-terminus, binding was apparently improved by substitution with hydrophobic residues that may access an additional hydrophobic pocket (termed the &#x003a6;<sub>U</sub> pocket) previously observed in some p38 and ERK D-site complexes (<a href="#SD11" rid="SD11" class=" supplementary-material">fig. S3B</a>) (<a href="#R11" rid="R11" class=" bibr popnode">11</a>, <a href="#R29" rid="R29" class=" bibr popnode">29</a>). Only three residues within the MKK6 D-site sequence appeared to be suboptimal for binding, including a Pro residue immediately downstream of the L-x-I motif. Overall, these positional scanning screens both confirmed elements of the MKK6 and MKK7 D-sites known to be critical for MAPK binding, and also identified additional features that favor or disfavor DRS interactions.</p></div><div id="S5" class="sec"><h3 id="S5title">Screening the human proteome for MAPK-interacting D-sites</h3><p id="P11" class="p p-first">The screens described above allowed us to define key features of the MKK6 and MKK7 D-sites. However, different D-site peptides can assume distinct binding modes at the DRS, each associated with its own conformation and associated sequence motif (<a href="#R7" rid="R7" class=" bibr popnode">7</a>, <a href="#R9" rid="R9" class=" bibr popnode">9</a>, <a href="#R11" rid="R11" class=" bibr popnode">11</a>, <a href="#R13" rid="R13" class=" bibr popnode">13</a>, <a href="#R19" rid="R19" class=" bibr popnode">19</a>). To enable discovery of multiple MAPK-targeting motifs and to discover new p38 or JNK interactors, we designed a large library consisting of candidate D-site sequences derived from the human proteome following similar criteria to those used previously for in silico screens (<a href="#R9" rid="R9" class=" bibr popnode">9</a>, <a href="#R21" rid="R21" class=" bibr popnode">21</a>) (<a href="/pmc/articles/PMC9995140/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3A</span></a>). We identified ~50,000 occurrences of the general D-site motif (defined as [RK]-x<sub>0-2</sub>-[RK]-x<sub>3-5</sub>-[ILV]-x-[FILMV]) in the human proteome. To increase the likelihood that sites would be accessible to bind MAPKs, we excluded sequences that fell within folded domains and those annotated to be extracellular or located within the endoplasmic reticulum or the Golgi apparatus. The remaining sequences were incorporated into a final library of 11,756 sequences from 5426 proteins, including most established JNK- and p38-interacting D-sites (<a href="#SD1" rid="SD1" class=" supplementary-material">Data File S1</a>, <a href="#SD11" rid="SD11" class=" supplementary-material">table S1</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC9995140/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140171993723792"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9995140_nihms-1871559-f0003.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-1871559-f0003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC9995140/bin/nihms-1871559-f0003.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140171993723792"><a target="object" rel="noopener" href="/pmc/articles/PMC9995140/figure/F3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC9995140/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Proteomic library screens.</strong><p id="P51">(A) Schematic depicting the selection of candidate D-site sequences from the human proteome. (B) and (C) Waterfall plots for the JNK1 (B) and p38&#x003b1; (C) screens, respectively, ordered by decreasing average Z-score (N=3 independent experiments) for each sequence. The insets show the 500 most highly enriched sequences, with the dashed line indicating the Z=2 hit cutoff. Established functional JNK- and p38-interacting D-sites are indicated as colored bars. Proteins corresponding to the 10 most enriched sequences are shown at right with previously known JNK interactors highlighted. (D) Venn diagram showing the overlap of hits from the two screens.</p></div></div></div><p id="P12">Coding sequences for all components of the library were introduced as a pool into the yeast GST fusion vector and screened for interaction with p38&#x003b1; and JNK1 as described above for the positional scanning libraries (<a href="#SD4" rid="SD4" class=" supplementary-material">Data File S4</a>, <a href="#SD11" rid="SD11" class=" supplementary-material">fig. S2C</a>,<a href="#SD11" rid="SD11" class=" supplementary-material">D</a>). Sequences were ranked by the average Z-score from three replicate screens (<a href="#SD5" rid="SD5" class=" supplementary-material">Data File S5</a>). We observed established D-sites to be strongly enriched in screens for their respective MAPK (<a href="/pmc/articles/PMC9995140/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3B</span></a>,<a href="/pmc/articles/PMC9995140/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,C,</span></span><span>C</span></a>, <a href="#SD5" rid="SD5" class=" supplementary-material">Data File S5</a>). For example four of the five most highly ranked JNK1-interacting sequences (corresponding to JIP1, NFAT4, MKK4 and BMPR2) were previously reported docking sites (<a href="/pmc/articles/PMC9995140/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3B</span></a>). We defined &#x0201c;hits&#x0201d; as sequences enriched in all replicates with an average Z score &#x02265; 2. By these criteria, all save one of the known JNK-interacting D-sites present in our library scored among the 137 hits in the JNK1 screen (<a href="/pmc/articles/PMC9995140/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3B</span></a>, <a href="#SD11" rid="SD11" class=" supplementary-material">table S2</a>). The 80 p38&#x003b1; hit sequences (<a href="/pmc/articles/PMC9995140/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3C</span></a>, <a href="#SD11" rid="SD11" class=" supplementary-material">table S3</a>) included five of seven known interacting D-sites (MKK3, MKK4, MKK6, MEF2A and MEF2C). We note that the remaining D-sites (PTPN5 and PTPN7) were also enriched but did not meet our hit threshold. We observed little overlap among hits for p38&#x003b1; and JNK1, with only seven sequences on both lists (<a href="/pmc/articles/PMC9995140/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3D</span></a>), and no non-cognate D-sites scored as hits for either MAPK. The strong enrichment of previously known D-sites suggests that other hits are likely to include authentic MAPK-interacting sequences.</p><p id="P13">To validate results from our screens, we examined the capacity of individual enriched sequences to selectively bind p38&#x003b1; and JNK1. To assess MAPK binding, we examined that ability of synthetic peptides to inhibit JNK1 and p38&#x003b1; activity toward a fluorogenic substrate incorporating a requisite D-site (<a href="#R30" rid="R30" class=" bibr popnode">30</a>). We chose 24 hit sequences that had not previously been established to bind the corresponding MAPKs. As a measure of relative binding affinity, we determined IC<sub>50</sub> values for inhibition of both kinases from assays conducted at a range of competitor peptide concentrations (<a href="/pmc/articles/PMC9995140/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4A</span></a>, <a href="#SD11" rid="SD11" class=" supplementary-material">table S4</a>). We found that all sequences scoring as hits in the JNK1 screen indeed bound JNK1 with higher affinity than to p38&#x003b1; (<a href="/pmc/articles/PMC9995140/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4A</span></a>,<a href="/pmc/articles/PMC9995140/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,B).</span></span><span>B</span></a>). Although p38&#x003b1; was favored by most of its hit sequences, two of them were non-selective, and one slightly favored JNK1. We noted a tendency for JNK1 to bind D-sites more tightly than p38&#x003b1;, with three having sub-&#x003bc;M IC<sub>50</sub> values (<a href="/pmc/articles/PMC9995140/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4A</span></a>, <a href="#SD11" rid="SD11" class=" supplementary-material">table S4</a>). This phenomenon may reflect an intrinsic property of the JNK1 DRS or a more stringent affinity cutoff for enrichment in the yeast-based screen. We found a relatively weak correlation between IC<sub>50</sub> value and Z-score among high affinity sites, particularly for JNK1-interacting sequences (<a href="#SD11" rid="SD11" class=" supplementary-material">fig. S4</a>). This phenomenon could be due to variability in expression levels among D-site sequences, or alternatively that sequences above a given affinity threshold do not impart faster growth rates in the proteomic screen. Overall, these experiments confirm that most sequences identified in our screen bind selectively to p38&#x003b1; and JNK1 with affinities comparable to those of established D-sites.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC9995140/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140171946833952"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9995140_nihms-1871559-f0004.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-1871559-f0004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC9995140/bin/nihms-1871559-f0004.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140171946833952"><a target="object" rel="noopener" href="/pmc/articles/PMC9995140/figure/F4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC9995140/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Hit validation.</strong><p id="P52">(A) Plot showing mean IC<sub>50</sub> values (N=3 independent experiments) for inhibition of the indicated MAPK by each of 24 synthetic peptides. IC<sub>50</sub> values too high to be confidently determined are indicated as being &#x0003e;100 &#x003bc;M and placed at the top or right edge. JNK1- and p38&#x003b1;-specific hits from the yeast screens are red squares and yellow diamonds, respectively. JNK1- and p38&#x003b1;-selective peptides fall respectively above and below the dotted line. (B) Graph showing the ratios of JNK1 to p38&#x003b1; IC<sub>50</sub> values for the indicated peptides, ordered from most JNK1-selective at top to most p38&#x003b1;-selective at bottom. Peptides binding weakly to either p38&#x003b1; or JNK1 are indicated with an asterisk, and ratios were calculated using an IC<sub>50</sub> value of 100 &#x003bc;M. Error bars show 95% confidence intervals. (C) Graphs showing the relative level of phosphorylation by JNK1 (left) and p38&#x003b1; (right) of chimeric NFAT4 constructs in which its D-site was replaced with the indicated hit peptides. Phosphorylation was assessed by immunoblotting with NFAT4 anti-phosphoSer<sup>165</sup> antibody. Error bars indicate SD (N = 3 independent experiments). Values significantly different from the WT construct (<em>p</em> = 0.05 by Mann-Whitney non-parametric test) are indicated (*).</p></div></div></div><p id="P14" class="p p-last">Because D-sites are often found in MAPK substrates, we further examined the capacity for selected p38 and JNK-targeting sequences to function in substrate recruitment. For these experiments, we used an N-terminal fragment of the JNK substrate NFAT4, which harbors a D-site positioned upstream of established phosphorylation sites, including Ser<sup>165</sup>. We generated constructs in which the native NFAT4 D-site was either eliminated or substituted with hit sequences from the yeast screens. As anticipated, the WT fragment was robustly phosphorylated at Ser<sup>165</sup> by JNK1 in vitro, and this was significantly reduced by mutation of the docking site (<a href="/pmc/articles/PMC9995140/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4C</span></a>). Furthermore, substitution with four out of five JNK1 hit sequences tested restored Ser<sup>165</sup> phosphorylation to that of the WT construct or higher, whereas incorporation of p38&#x003b1;-targeting sites resulted in significantly lower phosphorylation. By contrast, p38&#x003b1; poorly phosphorylated the WT fragment, and incorporating either of two of its hit sequences (L3MBTL3 and TRERF1) converted NFAT4 to a p38&#x003b1; substrate. However, the construct harboring one other hit (SETD1B) was not detectably phosphorylated, even though the corresponding peptide did bind to p38&#x003b1; in competitive kinase assays (<a href="#SD11" rid="SD11" class=" supplementary-material">table S4</a>). This observation suggests that in some cases, D-site binding is insufficient to promote phosphorylation of an associated protein. D-site engagement can impact MAPK conformational dynamics to promote kinase activity (<a href="#R31" rid="R31" class=" bibr popnode">31</a>), and our results suggest that such effects may occur in a sequence-specific manner. We also note that one D-site that was exclusively a JNK1 hit (SYDE1) promoted phosphorylation by both MAPKs and was presumably a false negative in the p38&#x003b1; screen.</p></div><div id="S6" class="sec"><h3 id="S6title">D-site sequence motifs conferring MAPK selective binding</h3><p id="P15" class="p p-first">Previous structural, biochemical, and computational analyses have identified multiple distinct sequence motifs selectively targeted by JNK and p38 MAPKs. Analysis of aligned hit sequences revealed that for both MAPKs, specific amino acids were substantially overrepresented at multiple positions in comparison to the full set of sequences in the library (<a href="/pmc/articles/PMC9995140/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5A</span></a> to <a href="/pmc/articles/PMC9995140/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1em;">toE).</span></span><span>E</span></a>). We expected that hit sequences for a given MAPK would conform to multiple distinct motifs that would not be evident from alignment of all hits. To deconvolute distinct motifs from the full dataset, we examined subsets of sequences in which a single overrepresented residue was fixed at one position. This analysis revealed two previously defined signatures within the JNK1 dataset: the &#x0201c;JIP class&#x0201d; motif (R-P-x-x-L-x-L, <a href="/pmc/articles/PMC9995140/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5A</span></a>) and the &#x0201c;NFAT4 class&#x0201d; motif (L-x-L-x-L/F, <a href="/pmc/articles/PMC9995140/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5B</span></a>) (<a href="#R9" rid="R9" class=" bibr popnode">9</a>). Among these core motifs, we observed specific residues to be enriched at intervening positions, such as hydrophilic residues downstream of the conserved Pro in the JIP class motif and acidic/amidic residues intervening the &#x003d5;<sub>A</sub> and &#x003d5;<sub>B</sub> residues of both motifs. These two motif classes account for 85% of JNK1 hits, with the remaining sequences being enriched for a Leu residue in position 7 (producing a L-x-x-L-x-L motif, <a href="/pmc/articles/PMC9995140/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5C</span></a>) that likely occupies the &#x003a6;<sub>L</sub> pocket similar to the Pro residue in the JIP class motif. Hit sequences for p38&#x003b1; were most strongly enriched for the previously defined &#x0201c;MEF2A class&#x0201d; motif (P/L-x-L-x-I/L-P) (<a href="#R9" rid="R9" class=" bibr popnode">9</a>), with 62.5% harboring a Pro or aliphatic residue in position 8 (&#x003d5;<sub>L</sub>) (<a href="/pmc/articles/PMC9995140/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5D</span></a>). Of the other sequences, 40% had an aliphatic residue in position 7, with the remainder were characterized by one or more basic residues upstream of the &#x003d5;<sub>A</sub> (position 10) Leu residue (<a href="/pmc/articles/PMC9995140/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5E</span></a>). These residual sequences generally lacked an Ile residue at the &#x003d5;<sub>B</sub> position or a Pro residue at position 13, suggestive of a distinct binding mode. In keeping with the results from the positional scanning library above, proteomic sequences selected by p38&#x003b1; were generally enriched for basic residues in positions proximal to the hydrophobic residues, a feature not generally considered as part of known interaction motifs. We note that Lys or Arg residues near the D-site N-terminus were not notably selected by either MAPK (<a href="/pmc/articles/PMC9995140/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5A</span></a> to <a href="/pmc/articles/PMC9995140/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1em;">toE).</span></span><span>E</span></a>). However, because our library design included at least two basic residues in all sequences, they may still promote binding in a manner independently of their precise position.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC9995140/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140171921789328"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9995140_nihms-1871559-f0005.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-1871559-f0005.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC9995140/bin/nihms-1871559-f0005.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140171921789328"><a target="object" rel="noopener" href="/pmc/articles/PMC9995140/figure/F5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC9995140/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>MAPK-interacting sequence motifs.</strong><p id="P53">(A) The subset of JNK1 hit sequences having a Pro residue at position 7 corresponding to the JIP class motif, represented by pLogo (<a href="#R65" rid="R65" class=" bibr popnode">65</a>). (B) JNK1 hits with a Leu residue at position 8 comprising the NFAT4 class motif. (C) Sequences lacking an R-P-x-x-L or &#x003d5;-x-L-x-L/F motif in any register. (D) pLogos corresponding to p38&#x003b1; MEF2A motif class hit sequences, defined as those with an Ile, Val, Leu, or Pro residue at position 8 or a Pro residue at position 13. (E) All remaining p38&#x003b1; hits. (F) Graph depicting ratios of the IC<sub>50</sub> values for the indicated point-substituted KMT2D synthetic D-peptide to those of the WT peptide in competitive inhibition JNK1 or p38&#x003b1; in vitro kinase assays. Error bars show 95% confidence intervals from 3 independent experiments. IC<sub>50</sub> values are provided in <a href="#SD11" rid="SD11" class=" supplementary-material">table S4</a>. (G) Two patches of surface residues in the docking groove of JNK, the CD (red) and ED (blue) regions, are shown mapped on the crystal structure of JNK1 in complex with the JIP1 peptide (PDB code 1UKI) (<a href="#R18" rid="R18" class=" bibr popnode">18</a>).</p></div></div></div><p id="P16">We probed the importance of key elements of these motifs by substitution analysis of a p38&#x003b1; hit peptide derived from KMT2D (RKTDRPALHLRIPP) in kinase inhibition assays as described above (<a href="/pmc/articles/PMC9995140/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5F</span></a>, <a href="#SD11" rid="SD11" class=" supplementary-material">table S4</a>). Consistent with our screens and previous reports that the identity of hydrophobic residues can drive MAPK specificity (<a href="#R16" rid="R16" class=" bibr popnode">16</a>), we found that substitution of the &#x003d5;<sub>B</sub> Ile residue with Phe led to a large decrease in p38&#x003b1; binding but did not impact binding to JNK1. Combined substitution of the Ile<sup>12</sup>-Pro<sup>13</sup> sequence with Leu-Ala also made the peptide more JNK1 selective, suggesting a switch to an NFAT4 motif class. Substitution of Arg<sup>12</sup> with Glu had a larger impact on binding to p38&#x003b1; compared to binding to JNK, consistent with results from both the proteomic and positional scanning screens. Replacement of Asp<sup>4</sup> with Arg improved binding to both MAPKs, confirming the importance of basic residues in that region. Collectively, these assays verify key sequence features selected by the two MAPKs in our screens.</p><p id="P17" class="p p-last">Knowledge of a protein interaction motif can be used to discover new interacting proteins by searching databases for matching sequences (<a href="#R32" rid="R32" class=" bibr popnode">32</a>). Although our screening approach identified MAPK interacting sequences present in the human proteome, authentic MAPK D-sites that do not conform strictly to the criteria used to build the library would be excluded. To identify D-sites not present within our screening library, we scanned the human proteome for sequences similar to our hits using PSSMsearch (<a href="#R33" rid="R33" class=" bibr popnode">33</a>). We performed four separate searches using hit sequences corresponding to the p38&#x003b1; (MEF2A class and non-MEF2A class) and JNK1 (JIP1 class and NFAT4 class) targeting motifs (<a href="#SD6" rid="SD6" class=" supplementary-material">Data File S6</a>). As anticipated, these searches returned sequences from the yeast screens that were used to build the position-specific scoring matrix (PSSM), as well as sequences that were absent from the library. We chose 8 peptides excluded from the original library and previous studies, and we evaluated their affinities for p38&#x003b1; and JNK1 in the competitive kinase assay (<a href="/pmc/articles/PMC9995140/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). With one exception, peptides bound to their predicted kinases with affinities comparable to those from the yeast screens (IC<sub>50</sub> range 0.3 - 5.6 &#x003bc;M). These results confirm our ability to computationally identify new docking sequences, providing additional validation for our experimentally determined motifs.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T1"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p id="P55">Inhibitory potency of peptide sequences identified by PSSMsearch.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="left" valign="bottom" rowspan="1" colspan="1">Protein</th><th align="left" valign="bottom" rowspan="1" colspan="1">Sequence</th><th align="left" valign="bottom" rowspan="1" colspan="1">JNK1<br />search<br />rank</th><th align="left" valign="bottom" rowspan="1" colspan="1">JNK1<br />PSSM</th><th align="left" valign="bottom" rowspan="1" colspan="1">p38&#x003b1;<br />search<br />rank</th><th align="left" valign="bottom" rowspan="1" colspan="1">p38&#x003b1;<br />PSSM</th><th align="left" valign="bottom" rowspan="1" colspan="1">JNK1 IC<sub>50</sub>,<br />&#x003bc;M (95% CI)</th><th align="left" valign="bottom" rowspan="1" colspan="1">p38&#x003b1; IC<sub>50</sub>, &#x003bc;M<br />(95% CI)</th></tr></thead><tbody><tr><td align="left" valign="top" rowspan="1" colspan="1">AFAP1L2</td><td align="left" valign="top" rowspan="1" colspan="1">KQVRKKEHKLKITP</td><td align="left" valign="top" rowspan="1" colspan="1">-</td><td align="left" valign="top" rowspan="1" colspan="1">-</td><td align="left" valign="top" rowspan="1" colspan="1">4</td><td align="left" valign="top" rowspan="1" colspan="1">Other</td><td align="left" valign="top" rowspan="1" colspan="1">&#x0003e;100</td><td align="left" valign="top" rowspan="1" colspan="1">40 (34 - 47)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">RASGRF1</td><td align="left" valign="top" rowspan="1" colspan="1">SPSRRRKLSLNIPI</td><td align="left" valign="top" rowspan="1" colspan="1">74</td><td align="left" valign="top" rowspan="1" colspan="1">NFAT</td><td align="left" valign="top" rowspan="1" colspan="1">2</td><td align="left" valign="top" rowspan="1" colspan="1">MEF2A</td><td align="left" valign="top" rowspan="1" colspan="1">1.1 (0.89 - 1.3)</td><td align="left" valign="top" rowspan="1" colspan="1">2.4 (2.0 - 2.9)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">WDR3</td><td align="left" valign="top" rowspan="1" colspan="1">KRKRKKREKLILTL</td><td align="left" valign="top" rowspan="1" colspan="1">25</td><td align="left" valign="top" rowspan="1" colspan="1">NFAT</td><td align="left" valign="top" rowspan="1" colspan="1">7</td><td align="left" valign="top" rowspan="1" colspan="1">Other</td><td align="left" valign="top" rowspan="1" colspan="1">0.90 (0.81 - 1.0)</td><td align="left" valign="top" rowspan="1" colspan="1">0.30 (0.24 - 0.36)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">TTLL13P</td><td align="left" valign="top" rowspan="1" colspan="1">RRRKRRSLAINLTN</td><td align="left" valign="top" rowspan="1" colspan="1">10</td><td align="left" valign="top" rowspan="1" colspan="1">NFAT</td><td align="left" valign="top" rowspan="1" colspan="1">48</td><td align="left" valign="top" rowspan="1" colspan="1">MEF2A</td><td align="left" valign="top" rowspan="1" colspan="1">1.8 (1.7 - 2.0)</td><td align="left" valign="top" rowspan="1" colspan="1">0.76 (0.64 - 0.90)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">MXD4</td><td align="left" valign="top" rowspan="1" colspan="1">EKHRRAKLRLYLEQ</td><td align="left" valign="top" rowspan="1" colspan="1">1</td><td align="left" valign="top" rowspan="1" colspan="1">NFAT</td><td align="left" valign="top" rowspan="1" colspan="1">34</td><td align="left" valign="top" rowspan="1" colspan="1">MEF2A</td><td align="left" valign="top" rowspan="1" colspan="1">1.1 (0.95 - 1.3)</td><td align="left" valign="top" rowspan="1" colspan="1">5.2 (3.5 - 7.8)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">MYH14</td><td align="left" valign="top" rowspan="1" colspan="1">GEQRRRRLELQLQE</td><td align="left" valign="top" rowspan="1" colspan="1">8</td><td align="left" valign="top" rowspan="1" colspan="1">NFAT</td><td align="left" valign="top" rowspan="1" colspan="1">-</td><td align="left" valign="top" rowspan="1" colspan="1">-</td><td align="left" valign="top" rowspan="1" colspan="1">5.6 (4.6 - 6.9)</td><td align="left" valign="top" rowspan="1" colspan="1">35 (26 - 49)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">CDX1</td><td align="left" valign="top" rowspan="1" colspan="1">YPGPARPASLGLGP</td><td align="left" valign="top" rowspan="1" colspan="1">36</td><td align="left" valign="top" rowspan="1" colspan="1">JIP</td><td align="left" valign="top" rowspan="1" colspan="1">-</td><td align="left" valign="top" rowspan="1" colspan="1">-</td><td align="left" valign="top" rowspan="1" colspan="1">2.7 (2.4 - 3.0)</td><td align="left" valign="top" rowspan="1" colspan="1">&#x0003e;100</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140171934571232"><a target="object" rel="noopener" href="/pmc/articles/PMC9995140/table/T1/?report=objectonly">Open in a separate window</a></div></div></div><div id="S7" class="sec"><h3 id="S7title">Determinants of MAPK-selective D-site interactions</h3><p id="P18" class="p p-first-last">We next considered features of the p38 and JNK DRS that might mediate selective targeting of distinct motifs. As noted above, p38&#x003b1; was more generally selective for basic residues throughout the D-site sequence. Notably, the p38&#x003b1; DRS is more negatively charged than that of JNK1, having nine acidic residues in comparison to four for JNK1. Unique acidic residues in p38&#x003b1; cluster at two sites: the so-called Glu-Asp (ED) region proximal to the &#x003a6;<sub>L</sub> and &#x003a6;<sub>A</sub> pockets (<a href="#R4" rid="R4" class=" bibr popnode">4</a>) and the common docking (CD) region (<a href="/pmc/articles/PMC9995140/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5G</span></a>). To examine the importance of these two regions to D-site specificity, we generated point mutants exchanging residues between p38&#x003b1; and JNK1 and examined binding of peptides corresponding to MAPK-selective motifs by competitive kinase assay (<a href="/pmc/articles/PMC9995140/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Mutating either the ED or CD regions of JNK1 to the corresponding residues in p38&#x003b1; reduced binding of the JNK-selective NFAT4 and JIP1 peptides by an order of magnitude. Although the JNK1 CD mutant had modestly increased affinity for the p38-selective MKK6 (MEF2A motif) and SETD1B (non-MEF2A motif) peptides, a larger effect was seen with the ED mutant, which had affinities similar to that of WT p38&#x003b1; for the two peptides. Conversely, the p38&#x003b1; ED mutant decreased the affinity of both cognate peptides to levels comparable to those seen for JNK1, whereas the CD mutant did not have a substantial effect. None of the p38&#x003b1; mutants conferred detectable binding to the JNK1 cognate NFAT4 and JIP1 peptides. These experiments substantiate a role for a specific cluster of residues in mediating selective binding of D-sites to p38&#x003b1;, either through specific side chain interactions or though bulk electrostatic effects.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T2"><h3>Table 2.</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Impact of p38&#x003b1;/JNK1 exchange mutagenesis on peptide binding affinities.</strong><p id="P56">JNK1 ED mutant, K160N/S161E/T164E; JNK1 CD mutant, S328D/E329D/A330E; p38&#x003b1; ED mutant, N159K/E160S/E163T; p38&#x003b1; CD mutant, D315S/D316E/E317A. Values are the average of 3 independent replicate experiments.</p></div><div class="xtable"><table frame="below" rules="groups" class="rendered small default_table"><thead><tr><th align="left" valign="top" rowspan="1" colspan="1"></th><th align="left" valign="top" rowspan="1" colspan="1"></th><th colspan="3" align="center" valign="top" rowspan="1">JNK1 IC<sub>50</sub>, &#x003bc;M (95% CI)<hr /></th><th colspan="3" align="center" valign="top" rowspan="1">p38&#x003b1; IC<sub>50</sub>, &#x003bc;M (95% CI)<hr /></th></tr><tr><th align="left" valign="top" rowspan="1" colspan="1">Peptide</th><th align="left" valign="top" rowspan="1" colspan="1">Sequence</th><th align="left" valign="top" rowspan="1" colspan="1">WT</th><th align="left" valign="top" rowspan="1" colspan="1">ED mutant</th><th align="left" valign="top" rowspan="1" colspan="1">CD mutant</th><th align="left" valign="top" rowspan="1" colspan="1">WT</th><th align="left" valign="top" rowspan="1" colspan="1">ED mutant</th><th align="left" valign="top" rowspan="1" colspan="1">CD mutant</th></tr></thead><tbody><tr><td align="left" valign="top" rowspan="1" colspan="1">NFAT4</td><td align="left" valign="top" rowspan="1" colspan="1">ERPSRDHLYLPLEP</td><td align="left" valign="top" rowspan="1" colspan="1">0.89 (0.76 - 1.0)</td><td align="left" valign="top" rowspan="1" colspan="1">9.5 (8.6 - 10)</td><td align="left" valign="top" rowspan="1" colspan="1">6.3 (5.0 - 7.9)</td><td align="left" valign="top" rowspan="1" colspan="1">&#x0003e;100</td><td align="left" valign="top" rowspan="1" colspan="1">&#x0003e;100</td><td align="left" valign="top" rowspan="1" colspan="1">&#x0003e;100</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">JIP3</td><td align="left" valign="top" rowspan="1" colspan="1">GRRKERPTSLNVFP</td><td align="left" valign="top" rowspan="1" colspan="1">0.36 (0.31 - 0.43)</td><td align="left" valign="top" rowspan="1" colspan="1">2.5 (2.2 - 2.8)</td><td align="left" valign="top" rowspan="1" colspan="1">2.9 (2.6 - 3.1)</td><td align="left" valign="top" rowspan="1" colspan="1">&#x0003e;100</td><td align="left" valign="top" rowspan="1" colspan="1">&#x0003e;100</td><td align="left" valign="top" rowspan="1" colspan="1">&#x0003e;100</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">MKK6</td><td align="left" valign="top" rowspan="1" colspan="1">SKGKKRNPGLKIPK</td><td align="left" valign="top" rowspan="1" colspan="1">95 (78 - 120)</td><td align="left" valign="top" rowspan="1" colspan="1">24 (21 - 27)</td><td align="left" valign="top" rowspan="1" colspan="1">75 (53 - 110)</td><td align="left" valign="top" rowspan="1" colspan="1">16 (14 - 19)</td><td align="left" valign="top" rowspan="1" colspan="1">65 (57 - 76)</td><td align="left" valign="top" rowspan="1" colspan="1">16 (14 - 18)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">SETD1B</td><td align="left" valign="top" rowspan="1" colspan="1">NQLKFRKKKLKFCK</td><td align="left" valign="top" rowspan="1" colspan="1">28 (25 - 33)</td><td align="left" valign="top" rowspan="1" colspan="1">2.0 (1.6 - 2.4)</td><td align="left" valign="top" rowspan="1" colspan="1">14 (7.4 - 28)</td><td align="left" valign="top" rowspan="1" colspan="1">7.8 (6.3 - 9.7)</td><td align="left" valign="top" rowspan="1" colspan="1">30 (25 - 37)</td><td align="left" valign="top" rowspan="1" colspan="1">8.0 (6.4 - 10)</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140171991359840"><a target="object" rel="noopener" href="/pmc/articles/PMC9995140/table/T2/?report=objectonly">Open in a separate window</a></div></div></div><div id="S8" class="sec sec-last"><h3 id="S8title">Discovery of docking-dependent MAPK substrates</h3><p id="P19" class="p p-first">To explore how our D-site screens might contribute to our understanding of MAPK function, we first performed gene set enrichment analysis (<a href="#R34" rid="R34" class=" bibr popnode">34</a>) to identify potential cellular processes invovling p38 and JNK (<a href="/pmc/articles/PMC9995140/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). Hits for p38&#x003b1; were enriched for proteins involved in regulation of chromatin organization and gene transcription, with most (65%) localizing to the nucleus. These observations are consistent with nuclear translocalization of activated p38 and with its established roles in transcriptional regulation (<a href="#R35" rid="R35" class=" bibr popnode">35</a>), and suggest broader control of the process than previously appreciated. Although the observed enrichment of JNK1 hits for proteins involved in the JNK MAPK cascade was expected due to the abundance of known interactors, this category included four upstream regulators not previously known to interact with JNK (TNIK, NCOR1, MAGI3, CARD9) that could constitute points for feedback regulation. In addition, JNK1 interactors were significantly enriched for the gene ontology (GO) biological process category &#x0201c;cytoskeleton organization&#x0201d; and for multiple categories related to GTPase signaling. Notably, in comparison to the full set of sequences in the library, both p38&#x003b1; and JNK1 hit sequences tended to be more conserved across species (<a href="#SD11" rid="SD11" class=" supplementary-material">fig. S5</a>), suggesting evolutionary pressure to maintain functional sequences. We do note that the degree of conservation varied among hit sequences, consistent with prior analyses of D-site evolution (<a href="#R9" rid="R9" class=" bibr popnode">9</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T3"><h3>Table 3.</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Gene set enrichment analysis of p38&#x003b1; and JNK1 hits.</strong><p id="P57">FDR, false discovery rate.</p></div><div class="xtable"><table frame="void" rules="none" class="rendered small default_table"><thead><tr><th colspan="2" align="center" valign="middle" rowspan="1">JNK1 screen<hr /></th><th colspan="2" align="center" valign="middle" rowspan="1">p38&#x003b1; screen<hr /></th></tr><tr><th align="left" valign="middle" rowspan="1" colspan="1">GO category</th><th align="left" valign="middle" rowspan="1" colspan="1">FDR q-value</th><th align="left" valign="middle" rowspan="1" colspan="1">GO category</th><th align="left" valign="middle" rowspan="1" colspan="1">FDR q-value</th></tr><tr><th colspan="2" align="center" valign="middle" rowspan="1">
<hr />
</th><th colspan="2" align="center" valign="middle" rowspan="1">
<hr />
</th></tr></thead><tbody><tr><td align="left" valign="middle" rowspan="1" colspan="1">Regulation of Intracellular Signal Transduction</td><td align="left" valign="middle" rowspan="1" colspan="1">1.44E-10</td><td align="left" valign="middle" rowspan="1" colspan="1">Chromatin Organization</td><td align="left" valign="middle" rowspan="1" colspan="1">9.80E-09</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Nucleoside Triphosphatase Regulator Activity</td><td align="left" valign="middle" rowspan="1" colspan="1">5.82E-10</td><td align="left" valign="middle" rowspan="1" colspan="1">Chromosome Organization</td><td align="left" valign="middle" rowspan="1" colspan="1">2.73E-08</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Cytoskeleton Organization</td><td align="left" valign="middle" rowspan="1" colspan="1">1.65E-09</td><td align="left" valign="middle" rowspan="1" colspan="1">Catalytic Complex</td><td align="left" valign="middle" rowspan="1" colspan="1">1.19E-07</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Small GTPase Mediated Signal Transduction</td><td align="left" valign="middle" rowspan="1" colspan="1">2.85E-09</td><td align="left" valign="middle" rowspan="1" colspan="1">Positive Regulation of Biosynthetic Process</td><td align="left" valign="middle" rowspan="1" colspan="1">3.54E-07</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Guanyl Nucleotide Exchange Factor Activity</td><td align="left" valign="middle" rowspan="1" colspan="1">7.90E-09</td><td align="left" valign="middle" rowspan="1" colspan="1">Positive Regulation of Nucleobase Containing Compound Metabolic Process</td><td align="left" valign="middle" rowspan="1" colspan="1">8.98E-07</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Positive Regulation of Catalytic Activity</td><td align="left" valign="middle" rowspan="1" colspan="1">8.41E-09</td><td align="left" valign="middle" rowspan="1" colspan="1">Nuclear Protein Containing Complex</td><td align="left" valign="middle" rowspan="1" colspan="1">8.98E-07</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Enzyme Regulator Activity</td><td align="left" valign="middle" rowspan="1" colspan="1">1.94E-08</td><td align="left" valign="middle" rowspan="1" colspan="1">Chromosome</td><td align="left" valign="middle" rowspan="1" colspan="1">2.65E-06</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Positive Regulation of Molecular Function</td><td align="left" valign="middle" rowspan="1" colspan="1">1.94E-08</td><td align="left" valign="middle" rowspan="1" colspan="1">Positive Regulation of Transcription by RNA Pol II</td><td align="left" valign="middle" rowspan="1" colspan="1">3.04E-06</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">MAPK cascade</td><td align="left" valign="middle" rowspan="1" colspan="1">2.92E-08</td><td align="left" valign="middle" rowspan="1" colspan="1">Negative Regulation of Nucleobase Containing Compound Metabolic Process</td><td align="left" valign="middle" rowspan="1" colspan="1">1.04E-05</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">JNK cascade</td><td align="left" valign="middle" rowspan="1" colspan="1">5.55E-08</td><td align="left" valign="middle" rowspan="1" colspan="1">Covalent Chromatin Modification</td><td align="left" valign="middle" rowspan="1" colspan="1">1.28E-05</td></tr><tr><td colspan="2" align="center" valign="middle" rowspan="1">
<hr />
</td><td colspan="2" align="center" valign="middle" rowspan="1">
<hr />
</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140172031686416"><a target="object" rel="noopener" href="/pmc/articles/PMC9995140/table/T3/?report=objectonly">Open in a separate window</a></div></div><p id="P20">Because D-sites can play a role in MAPK substrate recruitment, we reasoned that hits from our yeast screens include previously undescribed substrates. To verify this notion, we examined a set of cytoskeletal and GTPase regulatory proteins as candidate substrates from our JNK1 screens. We found that JNK1 phosphorylated two of these proteins, the RHO family GTPase activating proteins (GAPs) SYDE1 and SYDE2 (<a href="#R36" rid="R36" class=" bibr popnode">36</a>, <a href="#R37" rid="R37" class=" bibr popnode">37</a>), in kinase assays performed in vitro (<a href="/pmc/articles/PMC9995140/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6A</span></a>,<a href="/pmc/articles/PMC9995140/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,B).</span></span><span>B</span></a>). Addition of a peptide corresponding to the NFAT4 D-site inhibited JNK1 activity toward both proteins, suggesting that phosphorylation depended on an interaction with the DRS. We next examined whether SYDE1, SYDE2, and two other hit proteins (TAGAP and FMN1) could be phosphorylated by JNK in cells. We treated HEK293T cells expressing the proteins with the protein synthesis inhibitor anisomycin to activate the JNK pathway in the presence or absence of the selective JNK inhibitor JNK-IN-8. We found that anisomycin induced phosphorylation of all four proteins as judged by partial or complete electrophoretic mobility shift on Phos-tag sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE), which was reversed by treatment with the JNK inhibitor (<a href="/pmc/articles/PMC9995140/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6C</span></a>-<a href="/pmc/articles/PMC9995140/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">-F).</span></span><span>F</span></a>). Analysis of SYDE2 by liquid chromatography-tandem mass spectrometry (LC-MS/MS) revealed that a phosphorylation at a single residue (Ser<sup>935</sup>) increased with anisomycin treatment and decreased with JNK-IN-8 treatment (<a href="/pmc/articles/PMC9995140/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6G</span></a>, <a href="#SD7" rid="SD7" class=" supplementary-material">Data File S7</a>). Similarly, we found that anisomycin-induced phosphorylation of multiple sites in a Ser-Pro consensus in FMN1 was also blocked by JNK inhibition (<a href="/pmc/articles/PMC9995140/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6H</span></a>, <a href="#SD11" rid="SD11" class=" supplementary-material">fig. S6</a>, and <a href="#SD8" rid="SD8" class=" supplementary-material">Data File S8</a>). Overall, these observations verify that our ability to identify new MAPK substrates from our D-site screens.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F6" co-legend-rid="lgnd_F6"><a href="/pmc/articles/PMC9995140/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140171934574096"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9995140_nihms-1871559-f0006.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-1871559-f0006.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC9995140/bin/nihms-1871559-f0006.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140171934574096"><a target="object" rel="noopener" href="/pmc/articles/PMC9995140/figure/F6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F6"><div><a class="figpopup" href="/pmc/articles/PMC9995140/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6">Figure 6.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>JNK substrate discovery.</strong><p id="P54">(A) In vitro kinase assay measuring JNK1-mediated phosphorylation of full length SYDE1. Reactions were performed with increasing concentrations of the NFAT4 D-site competitor peptide (pep-NFAT4). Representative images from 3 separate experiments are shown. (B) In vitro kinase assay measuring JNK1-mediated phosphorylation of full length SYDE2, performed as in (A). Representative images from 3 separate experiments are shown. (C-F) Immunoblotting analysis with the indicated antibodies of HEK293T cells expressing FLAG-SYDE1, FLAG-SYDE2, FLAG-TAGAP, or GFP-FMN1 and treated with anisomycin following preincubation with or without the covalent JNK inhibitor JNK-IN-8. Lysates were subjected to either Phos-tag (top image) or standard (all others) SDS-PAGE. A representative immunoblot is shown for 2 (SYDE1), 3 (SYDE2), 5 (TAGAP) and 5 (FMN1) independent experiments. (G) LC-MS/MS analysis of tryptic peptides derived from immunopurified FLAG-SYDE2 transiently expressed in HEK293T cells treated as indicated. The graph shows the ratio of the Ser<sup>935</sup>-phosphorylated peptide to the corresponding unphosphorylated peptide. Data are normalized to the anisomycin treated sample. N=3 independent experiments. Significance was calculated using a one-tailed Mann-Whitney non-parametric test. (H) LC-MS/MS analysis of singly or doubly phosphorylated GFP-FMN1-derived tryptic phosphopeptides spanning Ser<sup>520</sup> and Ser<sup>524</sup> showing the ratio of phosphorylated to unphosphorylated peptides, normalized to the anisomycin treated sample. N=3 independent experiments. Significance was calculated using a one-tailed Mann-Whitney non-parametric test. (I) Immunoblotting analysis with the indicated antibodies of lysates from HEK293T cells expressing either WT SYDE2 or its D-site mutant (R1179Q/P1180Q/L1183A/L1185A) and treated with anisomycin following preincubation with or without the covalent JNK inhibitor JNK-IN-8. N=3 independent experiments. (J) Immunoblots of lysates from HEK293T cells harboring an episomal Dox-inducible MLK3-MKK7 expression plasmid and transfected with FLAG-SYDE2 variants in which the D-site was substituted with the sequence shown at top. Cells were incubated for 6 hr in the presence or absence of Dox prior to lysis. N=2 independent experiments. (K) Immunoblots of HEK293T cell lysates transfected with plasmids expressing either WT MEF2A or a mutant in which the D-site was exchanged for an optimal JNK-binding sequence shown at top (JNK-D). Cells were treated with anisomycin in the presence or absence of JNK-IN-8 or the p38&#x003b1;/&#x003b2; inhibitor SB203580 prior to lysis. Asterisk (*) indicates non-specific bands present in the untransfected control samples. N=2 independent experiments.</p></div></div></div><p id="P21" class="p p-last">To determine the importance of specific features of docking motifs for phosphorylation of MAPK substrates in cells, we examined a series of SYDE2 D-site mutations. Mutation of core residues within the SYDE2 &#x0201c;JIP class&#x0201d; D-site motif (R1179Q/P1180Q/L1183A/L1185A) prevented its anisomycin-induced Phos-tag PAGE mobility shift, confirming that the docking interaction is indeed essential for phosphorylation by JNK in cells (<a href="/pmc/articles/PMC9995140/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6I</span></a>). We noted that in addition to this core R-P-x-x-L-x-L motif, the SYDE2 sequence harbored residues at other positions that were positively selected in the proteomic screen. To determine the importance of these &#x0201c;secondary&#x0201d; residues, we generated an SYDE2 mutant in which they were substituted (E1178S/C1181E/N1184G) but maintained the core motif. We expressed this mutant in cells inducibly expressing constitutively active forms of either MKK7 or MKK6 to specifically activate JNK and p38, respectively. We found that this mutation reduced, but did not eliminate, phosphorylation by JNK, confirming that residues outside of the core motif do contribute to optimal phosphorylation (<a href="/pmc/articles/PMC9995140/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6J</span></a>). We similarly exchanged the SYDE2 D-site for a sequence optimized to bind to p38&#x003b1;. This mutation eliminated JNK-dependent phosphorylation, despite the presence of core basic and hydrophobic residues recognized by both kinases (<a href="/pmc/articles/PMC9995140/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6J</span></a>). This substitution did not lead to detectable phosphorylation of SYDE2 by p38 in cells (<a href="#SD11" rid="SD11" class=" supplementary-material">fig. S7</a>), suggesting that in additional to the essential D-site interaction, other regions of SYDE2 may contribute to its phosphorylation by MAPKs. To examine whether a JNK-selective motif could be sufficient to effect substrate recruitment in a different context, we examined the impact of exchanging the native D-site of the established p38 substrate MEF2A for a predicted JNK-targeting sequence. We were unable to observe anisomycin-induced phosphorylation of full length WT MEF2A at an ascribed p38 site (Thr<sup>319</sup>) when ectopically expressed in HEK293T cells, presumably because it was below the limit of detection (<a href="/pmc/articles/PMC9995140/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6K</span></a>). However, introduction of an optimized JNK-binding D-site led to robust anisomycin-induced phosphorylation that was eliminated by treatment with JNK-IN-8 but not affected by the p38&#x003b1;/&#x003b2;-selective inhibitor SB203580. Collectively, these results confirm the ability of specific D-site sequences to effect selective JNK recruitment in the context of full length proteins.</p></div></div><div id="S9" class="tsec sec"><h2 class="head no_bottom_margin" id="S9title">DISCUSSION</h2><p id="P22" class="p p-first">Here, we have described a genetic screening approach to the identification of kinase docking motifs and interacting sequences. Most prior genetic screens for docking sequences involved in substrate recruitment have focused on protein phosphatases, which are generally described as lacking dephosphorylation site specificity and hence depend on non-catalytic interactions (<a href="#R38" rid="R38" class=" bibr popnode">38</a>-<a href="#R41" rid="R41" class=" bibr popnode">41</a>). However, there has been growing appreciation for the importance of non-catalytic SLiM-mediated interactions to kinase substrate targeting. For MAPKs, docking interactions can enforce selective targeting to individual subfamilies and can restrict the kinase to phosphorylating specific Ser or Thr residues in a given substrate (<a href="#R9" rid="R9" class=" bibr popnode">9</a>, <a href="#R10" rid="R10" class=" bibr popnode">10</a>, <a href="#R42" rid="R42" class=" bibr popnode">42</a>). In other systems, docking is not absolutely required for phosphorylation in cells, yet tuning the strength of such interactions can reportedly set phosphorylation rate. For example, SLiM-mediated recruitment of substrates to cyclin-dependent kinases through the cyclin subunit controls the timing of phosphorylation within the cell division cycle (<a href="#R43" rid="R43" class=" bibr popnode">43</a>, <a href="#R44" rid="R44" class=" bibr popnode">44</a>). For the yeast LATS/NDR kinase Cbk1, an optimal docking sequence is not absolutely required for substrate phosphorylation but confers robustness to perturbations that attenuate kinase activity (<a href="#R45" rid="R45" class=" bibr popnode">45</a>).</p><p id="P23">Previous approaches to identify SLiMs mediating protein-protein or protein-enzyme interactions have used libraries of synthetic peptides or genetically-encoded phage display and cell surface display libraries (<a href="#R40" rid="R40" class=" bibr popnode">40</a>, <a href="#R46" rid="R46" class=" bibr popnode">46</a>-<a href="#R48" rid="R48" class=" bibr popnode">48</a>). Our method involving reconstitution of signaling pathways in yeast involves tunable pathway inhibition through a competitive interaction between kinases in the MAPK cascade or with downstream effectors. This approach may be advantageous in that interactions occur within a eukaryotic cell while maintaining sufficiently high throughput to enable extraction of binding motifs. It also enables discovery of interaction partners that might escape detection in MS-based proteomics experiments due to low abundance or restricted distribution patterns. The expression level of D-site fusion proteins in yeast sets a relatively stringent affinity threshold providing a low false positive rate, but we may consequently fail to identify low affinity interactions. For example, although our approach selected almost all known JNK interactors in our library, it failed to identify the established D-site in the transcription factor ATF2. This is consistent with regions of ATF2 outside of its SLiM reportedly making an additional contact with MAPKs to increase binding affinity (<a href="#R49" rid="R49" class=" bibr popnode">49</a>). Likewise, the full p38&#x003b1;-interacting region of the phosphatase PTPN7 (also known as HePTP) includes additional sequence flanking the canonical D-site (<a href="#R50" rid="R50" class=" bibr popnode">50</a>, <a href="#R51" rid="R51" class=" bibr popnode">51</a>), and although enriched in our screen, it fell below our hit threshold.</p><p id="P24">In this study, we performed both a comprehensive mutagenesis screen of known docking sites as well as a screen of proteome-derived sequences. Screening positional scanning libraries has facilitated discovery of new kinase substrates conforming closely to the resulting motifs (<a href="#R52" rid="R52" class=" bibr popnode">52</a>). Selection of proteomic libraries has an advantage in that it directly nominates candidate interaction partners and can discover high affinity sequences that might appear suboptimal. Furthermore, a sufficiently large set of interacting sequences will also provide key sequence features constituting the interaction motifs. Indeed, using PSSMs derived from our experimental hits, we were able to identify additional high affinity JNK1 and p38&#x003b1;-binding sequences absent from our screening library. We do note that the PSSMsearch scores correlated poorly with peptide binding affinity as measured in competitive kinase assays. Possible explanations for this phenomenon include compositional bias among hit sequences for reasons unrelated to binding affinity (for example to promote disorder) or that residues at distinct positions make non-additive or context-dependent contributions to binding affinity. Our PSSMs were not corrected for the background amino acid frequency of library sequences, which could have overemphasized the contribution of N-terminal basic residues to binding affinity at the expense of other residues. Determining the binding affinity of a larger set of training peptides could be used to refine input PSSMs to allow for more accurate predictions. In addition, computational approaches that can account for non-additivity, such as the use of artificial neural networks, would be expected to improve the correlations between search score and binding affinity.</p><p id="P25">Several previous studies have used computational approaches to scan proteomes for MAPK-interacting D-sites based on consensus motifs defined by alignment of known interactors and by integrating structural constraints based on crystallographic studies of MAPK-D-site complexes (<a href="#R9" rid="R9" class=" bibr popnode">9</a>, <a href="#R21" rid="R21" class=" bibr popnode">21</a>). Although we did observe substantial overlap in our datasets, most of our hit sequences were not previously predicted (<a href="#SD11" rid="SD11" class=" supplementary-material">tables S2</a> and <a href="#SD11" rid="SD11" class=" supplementary-material">S3</a>). For example, 38% of JNK1 and 33% of our p38&#x003b1; hits were also discovered in the structure-guided in silico screens conducted by Zeke <em>et al</em>. (<a href="#R9" rid="R9" class=" bibr popnode">9</a>) , and 15% of JNK1 hits had been identified by Whisenant <em>et al</em>. through database searches for sequences similar to known D-sites (<a href="#R21" rid="R21" class=" bibr popnode">21</a>). We found comparable overlap between these prior screens and our own in silico PSSM-based searches (45, 24, and 18 sequences from Zeke <em>et al</em>. were among our top 100 scoring sequences in the JIP, NFAT and MEF2A searches, respectively). We note that hits unique to our dataset were often dissimilar in sequence to those for which structural information is available. In particular, over 90% of our p38&#x003b1; hits lacking an &#x003d5;<sub>L</sub> residue had not been previously predicted. These observations underscore the value of unbiased screens of proteome-derived libraries in the discovery of new interactors.</p><p id="P26" class="p p-last">JNK and p38 MAPKs were originally identified as stress-activated kinases and are now known to have diverse roles in normal and disease physiology (<a href="#R35" rid="R35" class=" bibr popnode">35</a>, <a href="#R53" rid="R53" class=" bibr popnode">53</a>, <a href="#R54" rid="R54" class=" bibr popnode">54</a>). We found substantial enrichment for specific processes associated with proteins harboring MAPK-interacting D-sites uncovered in our studies, suggestive of expanded roles for JNK and p38. For example, p38 regulates gene expression by directly phosphorylating more than a dozen sequence-specific transcription factors (<a href="#R35" rid="R35" class=" bibr popnode">35</a>). In addition to several previously unidentified transcription factor targets, approximately 25% of our hits were derived from chromatin-associated proteins, including multiple chromatin remodeling factors, components of lysine modification complexes, and methyllysine readers. These results suggest previously unappreciated mechanisms by which p38 may impact transcription. Furthermore, enrichment of GTPase regulators among JNK1 interactors is interesting in light of the capacity of RHO family GTPases to activate the JNK cascade (<a href="#R55" rid="R55" class=" bibr popnode">55</a>, <a href="#R56" rid="R56" class=" bibr popnode">56</a>), and we verified the RHO GAPs SYDE1, SYDE2 and TAGAP to be cellular JNK substrates. JNK isoforms may therefore have general roles in crosstalk or feedback regulation between GTPase signaling pathways. Overall, these studies provide a resource for further investigation into regulation of basic cellular processes by the MAPKs p38 and JNK.</p></div><div id="S10" class="tsec sec"><h2 class="head no_bottom_margin" id="S10title">MATERIALS AND METHODS</h2><div id="S11" class="sec sec-first"><h3 id="S11title">Plasmids</h3><p id="P27" class="p p-first">The plasmid for constitutive expression of N-terminally HisMax-tagged human MKK6 in yeast was generated by insertion of the full coding sequence (polymerase chain reaction [PCR] amplified from pcDNA3-HisMax-MKK6) (<a href="#R57" rid="R57" class=" bibr popnode">57</a>) downstream of the <em>ACT1</em> promoter (PCR amplified from pGS62, a gift from Gavin Sherlock, Stanford University) in pRS416. MKK6<sup>&#x00394;D</sup> and D-site substitution mutants on this background were generated by overlap extension PCR using oligonucleotides deleting residues Ser<sup>4</sup> &#x02013; Lys<sup>17</sup> (SKGKKRNPGLKI) or replacing them with D-sites from MKK7 (PQRPRPTLQLPLAN), MEF2A (SRKPDLRVVIPPS) or NFAT4 (LERPSRDHLYLPLE). To generate integrative expression vectors for WT MKK6 and MKK6<sup>D7</sup>, coding sequences were subcloned into pRS416-GPD downstream of the yeast <em>TDH3</em> (GPD) promoter, and the entire expression cassette was PCR amplified and cloned into the PacI and BglII sites of the plasmid HO-hisG-URA3-hisG-poly-HO (Addgene plasmid #51661). The integrating inducible yeast expression vectors for N-terminally FLAG-tagged human JNK1 (isoform &#x003b1;<sub>1</sub>) and rat p38&#x003b1; were generated by inserting the full-length coding sequences into pRS403-GAL1. The inducible yeast GST expression vector was generated by PCR amplifying the GST coding sequence from pGEX-4T1 and cloning into the SacI and SpeI sites of pRS416-GAL1. Coding sequences for individual D-sites together with the HisMax tag were PCR amplified from the corresponding pRS416-MKK6 plasmid and inserted into the SpeI and ClaI sites downstream of GST.</p><p id="P28">Bacterial expression vectors for GST-JNK1 (pGEX4T1-3xFLAG-JNK1, Addgene #47574), GST-p38&#x003b1;, His<sub>6</sub>-MKK4, His<sub>6</sub>-MKK6<sup>S207E/T211E</sup> (MKK6-EE), and constitutively active MEKK1 (MEKK-C) were previously reported (<a href="#R58" rid="R58" class=" bibr popnode">58</a>, <a href="#R59" rid="R59" class=" bibr popnode">59</a>). Expression vectors for NFAT4 D-site variants were prepared by subcloning residues 3 &#x02013; 407 of WT NFAT4 from the corresponding mammalian expression vector (Addgene #21664) into pGEX4T1, introducing ClaI and HindIII restriction sites flanking the D-site by site-directed mutagenesis, excising the native D-site coding sequence, and replacing it with synthetic oligonucleotide pairs harboring compatible ends.</p><p id="P29">The mammalian expression vectors for human SYDE1 (UniProt isoform 2, Q6ZW31-2) and human TAGAP (UniProt isoform 1, Q8N103-1) were generated by Gateway recombination from pDONR223-SYDE1 (human ORFeome collection) into the C-terminal 3xFLAG epitope tagged plasmid pV1900. The expression vector for N-terminally FLAG-tagged mouse SYDE2 was generated by PCR amplification of the coding sequence from pNICE HA-mSYD1B (<a href="#R36" rid="R36" class=" bibr popnode">36</a>) (Addgene #59362) and insertion into pcDNA3-FLAG by Gibson assembly. The plasmid expressing Myc epitope and 6xHis-tagged MEF2A in the pcDNA3.1 vector (<a href="#R60" rid="R60" class=" bibr popnode">60</a>) and N-terminally GFP-tagged mouse FMN1 isoform 2 were obtained from Addgene (#118354 and #19320).</p><p id="P30" class="p p-last">Point mutations in all plasmids were introduced by QuikChange site-directed mutagenesis. Sequences of all primers used for cloning, mutagenesis and Illumina sequencing are provided in <a href="#SD9" rid="SD9" class=" supplementary-material">Data File S9</a>.</p></div><div id="S12" class="sec"><h3 id="S12title">Design and generation of D-site libraries</h3><p id="P31" class="p p-first-last">The positional scanning libraries consisted of all possible single amino acid substitutions and all double Ala mutations to the MKK6 (SKGKKRNPGLKIPK) and second MKK7 (PQRPRPTLQLPLAN) D-site. To design the proteomic library, we identified all sequences matching the regular expression [RK]-x<sub>0-2</sub>-[RK]-x<sub>3-5</sub>-[ILV]-x-[FILMV] in human proteins in the UniProt database. Sequences were extended at both termini to include two residues downstream of the motif and to bring the total length to 14 residues, and overlapping sequences were removed. With R and custom Perl scripts, we parsed the reviewed UniProt KnowledgeBase to filter sequences based on localization and to exclude motifs falling within annotated domains. Only proteins labeled as &#x0201c;cytoplasm&#x0201d; or &#x0201c;nucleus&#x0201d; were included, and non-cytosolic sequences in transmembrane proteins were excluded. Sequences were reverse translated in silico to yeast optimized codons, and silent mutations were introduced to remove restriction sites used for cloning. Common flanking sequences were added for separate PCR amplification of the positional scanning (5&#x02019;: GCTTCAGGTGGACAACAATCACAA, 3&#x02019;: GAAGCTTCACTCTGTGTTGAAGTTCCGTCAG) and proteomic (5&#x02019;: GGTCGCGGATCTATGTCTCAG, 3&#x02019;: GAAGCTTTTGAACAACCTCAGCAC) libraries. Core DNA and protein sequences are provided in <a href="#SD1" rid="SD1" class=" supplementary-material">Data File S1</a>. Oligonucleotides were commercially synthesized as a pool (CustomArray), PCR-amplified, and restriction enzyme cloned into the NheI and HindIII sites downstream of the GST coding sequence of the pRS416-GAL1-GST plasmid (<a href="#SD11" rid="SD11" class=" supplementary-material">fig. S8</a>). DH10B cells (Invitrogen ElectroMAX) were transformed by electroporation with ligation products to produce at least 1000 transformants per library variant, and plasmid library DNA was prepared from pooled colonies. To ensure representation of all components of the library, the variable region was PCR-amplified and sequenced on an Illumina HiSeq 4000 instrument. Read counts for the proteomic library are provided in <a href="#SD10" rid="SD10" class=" supplementary-material">Data File S10</a>.</p></div><div id="S13" class="sec"><h3 id="S13title">Yeast growth assays</h3><p id="P32" class="p p-first-last">Liquid cultures of the indicated strains transformed with the indicated plasmids were grown to mid-logarithmic phase in the appropriate selective media containing 2% raffinose. Aliquots of five-fold dilution series were spotted onto agar plates containing either 2% glucose or 2% raffinose + 1% galactose as indicated. Plates were incubated at 30&#x000b0;C for 48 &#x02013; 96 hours.</p></div><div id="S14" class="sec"><h3 id="S14title">Yeast-based screens</h3><p id="P33" class="p p-first">The <em>S. cerevisiae hog1&#x00394; pbs2&#x00394;</em> strain was generated by PCR-based replacement of the <em>HOG1</em> open reading frame with the <em>LEU2</em> marker in a <em>pbs2&#x00394;</em>::KanMX strain from the yeast knockout collection (BY4741 strain background, Open Biosystems). The genotype was confirmed by diagnostic PCR of both deletion arms from genomic DNA. Strains used for screening were generated by subsequent integration of cassettes for galactose-inducible expression of p38&#x003b1;<sup>L195A</sup> or JNK1<sup>L198A</sup> (at the <em>HIS3</em> locus) and constitutive GPD promoter-driven expression of His-tagged WT MKK6 or MKK6<sup>D7</sup> (at the <em>HO</em> locus). Expression of MAPK and MKK6 alleles were confirmed by immunoblotting lysates from galactose-treated cells with antibodies to the FLAG and His<sub>6</sub> tags, respectively.</p><p id="P34" class="p p-last">Libraries of plasmids expressing D-site GST fusion proteins were introduced into yeast by LiOAc high-efficiency transformation (<a href="#R61" rid="R61" class=" bibr popnode">61</a>) and selection on synthetic complete (SC)-Ura agar plates to produce at least 200 transformants per component. Transformed yeast were scraped from plates, pooled, diluted to an OD<sub>600</sub> of 0.1 in SC-Ura liquid media with 2% glucose, and grown to saturation at 30&#x000b0;C. Cells were diluted into SC-Ura with 2% raffinose and grown for 6 hours to derepress the <em>GAL1</em> promoter. A starting time (T<sub>0</sub>) sample (20 OD<sub>600</sub> units) was reserved, and the remaining cells were split and diluted to an OD<sub>600</sub> of 0.1 in either SC-Ura + 2% raffinose + 1% galactose (inducing conditions) or SC-Ura + 2% glucose (control conditions). Cultures were subjected to four growth and dilution cycles in which they were propagated until the induced culture reached an OD<sub>600</sub> of ~1.5, a portion (20 OD<sub>600</sub> units) reserved, and remaining cells diluted in fresh pre-warmed media to an OD<sub>600</sub> of 0.1. Reserved cells were pelleted, washed once with sterile dH<sub>2</sub>O, snap-frozen on dry ice/EtOH and stored at &#x02212;80&#x000b0;C. Plasmids were extracted from each cell pellet and the D-site regions were PCR amplified, incorporating barcodes specific to each condition and time point and adaptors for sequencing. PCR products were agarose gel-purified, pooled, and subjected to Illumina sequencing (HiSeq 4000). The positional scanning library was screened twice, and the human proteomic library was screened three times against each MAPK. Data for each sequence were fit to an exponential function in Microsoft Excel, in which the inverse time constant &#x003bb; was calculated as the slope of the line of the log<sub>2</sub> transformed fold-change in normalized read counts as a function of time, with the y-intercept set to zero. Z-scores for each sequence within an individual screen were calculated from (&#x003bb;<sub>sequence</sub> - &#x003bb;<sub>mean</sub>)/SD. In each of the proteomic screens, a small number of components were not detected in the time zero sample. To allow comparisons across all screens, we performed Z-score analyses for only those components present in the starting time point samples from every replicate. In total, 53 sequences (0.45% of the library) were excluded from the analysis.</p></div><div id="S15" class="sec"><h3 id="S15title">Database searches</h3><p id="P35" class="p p-first-last">For PSSM searching, hit sequences were binned into four categories, accounting for sites to occur in a different register from our original definition. The JIP class comprised JNK1 hits containing a R-P-x-x-&#x003d5;-x-&#x003d5; sequence starting at either position 4 or position 6. The NFAT4 class comprised JNK1 hits with an &#x003d5;-x-&#x003d5;-x-&#x003d5; sequence starting at either position 8 or position 10 that were not included in the JIP class. The MEF2 class comprised p38&#x003b1; hit sequences with either a Leu or Pro residue at position 8 or a Pro residue at position 13. The &#x0201c;other p38&#x003b1;&#x0201d; class comprised p38&#x003b1; hit sequences with a Lys or Arg residue at either position 8 or 9, excluding sequences defined in the MEF2 motif. Sequences belonging to each motif class were entered into the program PSSMsearch (<a href="http://slim.icr.ac.uk/pssmsearch/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://slim.icr.ac.uk/pssmsearch/</a>) (<a href="#R33" rid="R33" class=" bibr popnode">33</a>) and the resulting PSSM used to searched the human proteome with default settings (disorder cutoff = 0.4, p-value cutoff = 0.001). Search results were ranked based on the PWM score, and the top 1000 sequences are shown in <a href="#SD6" rid="SD6" class=" supplementary-material">Data File S6</a>. Gene set enrichment analysis for GO categories associated with hit sequences was performed using the Broad Institute web interface (<a href="https://www.gsea-msigdb.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.gsea-msigdb.org/</a>) (<a href="#R34" rid="R34" class=" bibr popnode">34</a>).</p></div><div id="S16" class="sec"><h3 id="S16title">Protein Expression and Purification</h3><p id="P36" class="p p-first">GST-tagged (JNK1, p38&#x003b1;, and NFAT4<sup>3-407</sup> variants) and His<sub>6</sub>-tagged (MKK6-EE and active MKK4 prepared by co-expression with MEKK-C) were expressed in BL21(DE3) <em>E. coli</em> and purified as described (<a href="#R58" rid="R58" class=" bibr popnode">58</a>, <a href="#R59" rid="R59" class=" bibr popnode">59</a>). FLAG epitope-tagged SYDE1 and SYDE2 were expressed in and purified from polyethyleneimine-transfected HEK293T cells (ATCC #CRL-3216) as previously described (<a href="#R62" rid="R62" class=" bibr popnode">62</a>). The concentration and purity of protein preparations were assessed by SDS-PAGE and Coomassie Brilliant Blue R250 staining alongside bovine serum albumin (BSA) standards.</p><p id="P37" class="p p-last">GST-p38&#x003b1; and GST-JNK1 (50 &#x003bc;M) were activated in vitro by incubation with 500 nM His<sub>6</sub>-MKK6-EE or 50 nM active His<sub>6</sub>-MKK4, respectively, in reaction buffer (50 mM Tris [pH 8.0], 100 mM NaCl, 10 mM MgCl<sub>2</sub>, 1mM DTT, 0.012% Brij-35, 300 &#x003bc;M ATP) at 30&#x000b0;C for 1.5 hours. Phosphorylation was confirmed by immunoblotting with anti-p38 pThr<sup>180</sup>/pTyr<sup>182</sup> and JNK pThr<sup>183</sup>/pTyr<sup>185</sup> antibodies as appropriate.</p></div><div id="S17" class="sec"><h3 id="S17title">Immunoblotting</h3><p id="P38" class="p p-first-last">The following primary antibodies used for immunoblotting were obtained from Cell Signaling Technology: p38 pThr<sup>180</sup>/pTyr<sup>182</sup> (#9211), JNK pThr<sup>183</sup>/pTyr<sup>185</sup> (#9251 and #9255), JNK (#3708), c-JUN (#2315), c-JUN pSer<sup>63</sup> (#9261), MK2 (#3042), GST (#2624), and Myc tag (#2276). Other antibodies used were: NFAT4 pSer<sup>165</sup> (Sigma-Aldrich SAB4503947), FLAG M2 (Sigma-Aldrich F3165), vinculin (Sigma-Aldrich V9131), penta-His (Qiagen), MEF2A pThr<sup>319</sup> (Origene #TA325686). Fluorophore-conjugated secondary antibodies (Invitrogen #A21109 goat anti-rabbit and Li-Cor #926-32210 goat anti-mouse) were diluted 1:20,000 in TBS-T and 5% BSA. Blotting membranes were analyzed using an Odyssey CLx imaging system (LI-COR Biosciences) and quantified using Image Studio Lite software.</p></div><div id="S18" class="sec"><h3 id="S18title">MAPK D-site peptide inhibition assays</h3><p id="P39" class="p p-first-last">D-site peptides were commercially synthesized (GenScript) incorporating a fixed Tyr-Ala sequence upstream of 14 residues corresponding to the yeast library sequence. Peptides were dissolved in dimethyl sulfoxide (DMSO) to 10 mM and stored at &#x02212;20&#x000b0;C. MAPK assays were performed with a sulfonamido-oxine (SOX) containing substrate peptide (AssayQuant AQT0376). Kinase assays were performed in technical duplicate in black 384 well plates in reactions containing 50 mM HEPES (pH 7.5), 10 mM MgCl<sub>2</sub>, 0.012% Brij-35, 1% glycerol, 0.2 mg/mL BSA, 1 mM ATP, 1.2 mM DTT and 4 &#x003bc;M SOX peptide substrate. Competitor D-site peptides were titrated in two-fold increments over a range from 31 nM &#x02013; 64 &#x003bc;M. Reactions were initiated by adding activated p38&#x003b1; or JNK1 to final concentrations of 3 nM and 60 nM, respectively, and fluorescence (excitation 360 nm, emission 485 nm) was read every min over 1 hour in a Molecular Devices SpectraMax M5 plate reader. Initial velocities were calculated from the linear portions of the reaction progress curves. IC<sub>50</sub> values and 95% confidence intervals were calculated by fitting data collected from three biological replicates to a sigmoidal dose-response curve using Prism 8.2.0 (GraphPad).</p></div><div id="S19" class="sec"><h3 id="S19title">Protein kinase assays</h3><p id="P40" class="p p-first">GST-NFAT4<sup>3-407</sup> and its variants (0.5 &#x003bc;M) were incubated with p38&#x003b1; (14 nM) or JNK1 (7 nM) in reaction buffer (50mM HEPES [pH 7.4], 100mM NaCl, 10mM MgCl<sub>2</sub>, 0.012% Brij-35, 1mM DTT, 1mM Na<sub>3</sub>VO<sub>4</sub>, 5mM &#x003b2;-glycerophosphate, 100 &#x003bc;M ATP) at 30&#x000b0;C for 20 min. Reactions were quenched by adding SDS-PAGE loading buffer and analyzed by immunoblotting with antibodies to GST and NFAT4 pSer<sup>165</sup>.</p><p id="P41" class="p p-last">Purified SYDE1 (100 nM) or SYDE2 (120 nM) was incubated with or without active JNK1 (70 nM) in kinase reaction buffer containing 50 mM HEPES (pH 7.4), 100 mM NaCl, 10 mM MgCl<sub>2</sub>, 0.012% Brij-35, 1 mM DTT, 1 mM Na<sub>3</sub>VO<sub>4</sub>, 1 mM &#x003b2;-glycerophosphate and 50 nM staurosporine (to suppress background phosphorylation in the control reaction). Reactions were initiated by adding [&#x003b3;-<sup>32</sup>P]ATP to a final concentration of 20 &#x003bc;M at 0.1 &#x003bc;Ci/&#x003bc;L. Reactions were incubated at 30&#x000b0;C for 20 min and then quenched with the addition of 5 &#x003bc;L 4x SDS-PAGE loading buffer. Samples were separated by SDS-PAGE (10% acrylamide) and gels were stained with Coomassie, destained, and exposed to a phosphor screen. Exposures were analyzed by phosphorimager and quantified using QuantityOne software (BioRad). Experiments were performed at least three times.</p></div><div id="S20" class="sec sec-last"><h3 id="S20title">Analysis of protein phosphorylation in cultured cells</h3><p id="P42" class="p p-first">HEK293T cells were transiently transfected with the indicated expression plasmids using polyethyleneimine. After 48 hr, cells were treated with either 5 &#x003bc;M JNK-IN-8 (SelleckChem, S4901), 50 &#x003bc;M SB203580, or vehicle (0.1% DMSO) for 1 hour followed by either 10 &#x003bc;g/mL anisomycin or vehicle (0.1% DMSO) for an additional hour at 37&#x000b0;C. Lysates were prepared as described (<a href="#R62" rid="R62" class=" bibr popnode">62</a>), and a portion was subjected to either standard or Phos-tag SDS-PAGE (<a href="#R63" rid="R63" class=" bibr popnode">63</a>) and immunoblotting with the indicated antibodies (all used at 1:1000 dilution). Phos-tag gels included 4% or 7.5% acrylamide, 50 &#x003bc;M Phos-tag reagent (Nard Institute AAL-107), and 100 &#x003bc;M MnCl<sub>2</sub>.</p><p id="P43">Phosphorylation of SYDE2 D-site mutants was analyzed in cells inducibly expressing constitutively active forms of MKK7 (MLK3-MKK7 fusion protein) or MKK6 (MKK6-EE) to activate the JNK and p38 pathways, respectively (<a href="#R64" rid="R64" class=" bibr popnode">64</a>). HEK293T cells were stably transfected with episomal plasmids expressing active MKKs from a doxycycline (Dox)-inducible promoter and selected for at least one week in the appropriate antibiotic (5 &#x003bc;g/mL puromycin and 100 &#x003bc;g/mL hygromycin for MLK3-MKK7 and MKK6-EE, respectively). Cells were transfected as above, and two days later were treated for 6 hr with 2 &#x003bc;g/mL Dox before lysis and immunoblotting as described above.</p><p id="P44" class="p p-last">For MS analysis, FLAG-SYDE2 was isolated from 10 cm plates as described above, and GFP-FMN1 was isolated similarly using GFP nanotrap beads. Proteins were fractionated by SDS-PAGE, and gels were stained briefly with Coomassie Brilliant Blue and de-stained. Protein bands were excised and in-gel digested with trypsin (MS grade, Promega) overnight at 37&#x000b0;C overnight, and peptides extracted with 80% acetonitrile/0.1% formic acid. Peptides were desalted with C18 spin columns (Nest Group, Inc.) and dried. For SYDE2 only, phosphopeptides were enriched by dissolving in 70 mM glutamic acid in 65% acetonitrile with 2% TFA, and subjected to TiO<sub>2</sub> phosphopeptide enrichment; the flowthrough fraction was reserved, and the enriched phosphopeptide fraction was eluted with 28% NH<sub>4</sub>OH and dried. Peptides were dissolved in 10 &#x003bc;L 70% formic acid plus 30 &#x003bc;L 50 mM sodium phosphate. Flowthrough and enriched SYDE2 samples and total FMN1 samples (all 0.3 &#x003bc;g in 5 &#x003bc;L) were subjected to LC-MS/MS analysis on a Q-Exactive Plus mass spectrometer (ThermoFisher Scientific) equipped with a Waters nanoAcquity ultra-performance liquid chromatography (UPLC) system (Waters) with a C18 (180 &#x003bc;m x 20 mm) trap column and a 1.7 &#x003bc;m, 75 &#x003bc;m x 250 mm nanoAcquity UPLC column. Trapping was done using 99% Buffer A (100% water, 0.1% formic acid) and peptide separation was undertaken using a linear gradient of solvents A (0.1% formic acid in water) and B (0.075% formic acid in acetonitrile) over 90 minutes, at a flow rate of 300 nL/min. Raw LC-MS/MS data files were processed with Progenesis QI Software (Waters, Inc.) with exported .mgf files searched against a custom database for the SYDE2 construct with MASCOT 2.7 (Matrix Science). Sites of phosphorylation were validated manually. Search results were imported into Progenesis QI and matched with the spectral features, and where relevant normalized enriched and flowthrough data were merged. The exported peptide lists with normalized abundances are provided in <a href="#SD7" rid="SD7" class=" supplementary-material">Data Files S7</a> and <a href="#SD8" rid="SD8" class=" supplementary-material">S8</a>. Data are available through ProteomeXchange with identifier PXD028694.</p></div></div><div id="S21" class="tsec sec"><h2 class="head no_bottom_margin" id="S21title">Data and materials availability</h2><p id="P45" class="p p-first-last">The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium through the PRIDE partner repository with the dataset identifier PXD028694. All other data needed to evaluate the conclusions in the paper are present in the paper or the <a href="#SD11" rid="SD11" class=" supplementary-material">Supplementary Materials</a>. Plasmids and yeast strains generated for this study are available through an MTA with Yale University.</p></div><div id="SM1" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="SM1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>SM Excel data file 1</h4><div class="sup-box half_rhythm" id="d64e1216"><a href="/pmc/articles/PMC9995140/bin/NIHMS1871559-supplement-SM_Excel_data_file_1.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(556K, xlsx)</sup></div></div><div class="sec suppmat" id="SD2"><h4>SM Excel data file 2</h4><div class="sup-box half_rhythm" id="d64e1219"><a href="/pmc/articles/PMC9995140/bin/NIHMS1871559-supplement-SM_Excel_data_file_2.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(303K, xlsx)</sup></div></div><div class="sec suppmat" id="SD3"><h4>SM Excel data file 3</h4><div class="sup-box half_rhythm" id="d64e1222"><a href="/pmc/articles/PMC9995140/bin/NIHMS1871559-supplement-SM_Excel_data_file_3.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(78K, xlsx)</sup></div></div><div class="sec suppmat" id="SD4"><h4>SM Excel data file 4</h4><div class="sup-box half_rhythm" id="d64e1225"><a href="/pmc/articles/PMC9995140/bin/NIHMS1871559-supplement-SM_Excel_data_file_4.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(8.9M, xlsx)</sup></div></div><div class="sec suppmat" id="SD5"><h4>SM data file 5</h4><div class="sup-box half_rhythm" id="d64e1228"><a href="/pmc/articles/PMC9995140/bin/NIHMS1871559-supplement-SM_data_file_5.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(2.6M, xlsx)</sup></div></div><div class="sec suppmat" id="SD6"><h4>SM data file 6</h4><div class="sup-box half_rhythm" id="d64e1231"><a href="/pmc/articles/PMC9995140/bin/NIHMS1871559-supplement-SM_data_file_6.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(634K, xlsx)</sup></div></div><div class="sec suppmat" id="SD7"><h4>SM data file 7</h4><div class="sup-box half_rhythm" id="d64e1234"><a href="/pmc/articles/PMC9995140/bin/NIHMS1871559-supplement-SM_data_file_7.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(228K, xlsx)</sup></div></div><div class="sec suppmat" id="SD8"><h4>SM data file 8</h4><div class="sup-box half_rhythm" id="d64e1237"><a href="/pmc/articles/PMC9995140/bin/NIHMS1871559-supplement-SM_data_file_8.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(192K, xlsx)</sup></div></div><div class="sec suppmat" id="SD9"><h4>SM data file 9</h4><div class="sup-box half_rhythm" id="d64e1240"><a href="/pmc/articles/PMC9995140/bin/NIHMS1871559-supplement-SM_data_file_9.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(14K, xlsx)</sup></div></div><div class="sec suppmat" id="SD10"><h4>SM data file 10</h4><div class="sup-box half_rhythm" id="d64e1243"><a href="/pmc/articles/PMC9995140/bin/NIHMS1871559-supplement-SM_data_file_10.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(12M, xlsx)</sup></div></div><div class="sec suppmat" id="SD11"><h4>Supplemental File</h4><div class="sup-box half_rhythm" id="d64e1247"><a href="/pmc/articles/PMC9995140/bin/NIHMS1871559-supplement-Supplemental_File.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(3.1M, docx)</sup></div></div></div><div id="S22" class="tsec sec"><h2 class="head no_bottom_margin" id="S22title">ACKNOWLEDGMENTS</h2><div class="sec"><p id="P46">We thank Elias Lolis, Anton Bennett and Joel Sexton (Yale University), and Erik Schaefer (AssayQuant), for advice and suggestions regarding this work, and we thank Titus Boggon (Yale University) for advice and feedback on the manuscript. We thank Karl Barber (Yale University) for assistance with oligonucleotide library design, and Florine Collin and Jean Kanyo (Yale Keck Mass Spectrometry and Proteomics Resource) for assistance with proteomics sample preparation and LC MS/MS data collection. We thank Ana Thevenin (Moravian College), Gavin Sherlock (Stanford University) and Attila Rem&#x000e9;nyi (Research Center for Natural Sciences, Budapest) for providing plasmids, and the following investigators from whom plasmids were obtained through Addgene: Anjana Rao, Peter Scheiffele, David Stillman, Lea Sistonen, Philip Leder, and Kevin Janes.</p><div id="S23" class="sec"><h3 id="S23title">Funding</h3><p id="P47" class="p p-first-last">This work was supported by National Institutes of Health grant R01 GM135331 to B.E.T. Additional support was provided by the China Scholarship Council to G.S., a National Science Foundation Graduate Research Fellowship to J.T.R. and NIH T32 GM007324 to C.S. The Q-Exactive Plus mass spectrometer located at the Keck MS &#x00026; Proteomics Resource was supported in part by NIH SIG grant S10 OD018034 and Yale School of Medicine.</p></div></div></div><div id="fn-group-a.ab.b" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.ab.btitle">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"><p id="P48" class="p p-first-last"><strong>Competing interests:</strong> The authors declare that they have no competing interests</p></p></div></div><div id="ref-list-a.ab.c" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.ab.ctitle">REFERENCES AND NOTES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="mixed-citation">Miller CJ, Turk BE, <span class="ref-title">Homing in: Mechanisms of Substrate Targeting by Protein Kinases</span>. <span class="ref-journal">Trends Biochem. Sci</span>
<span class="ref-vol">43</span>, 380&#x02013;394 (2018). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5923429/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29544874" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Biochem.+Sci&amp;title=Homing+in:+Mechanisms+of+Substrate+Targeting+by+Protein+Kinases&amp;author=CJ+Miller&amp;author=BE+Turk&amp;volume=43&amp;publication_year=2018&amp;pages=380-394&amp;pmid=29544874&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="mixed-citation">Raman M, Chen W, Cobb MH, <span class="ref-title">Differential regulation and properties of MAPKs</span>. <span class="ref-journal">Oncogene</span>
<span class="ref-vol">26</span>, 3100&#x02013;3112 (2007). [<a href="https://pubmed.ncbi.nlm.nih.gov/17496909" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Differential+regulation+and+properties+of+MAPKs&amp;author=M+Raman&amp;author=W+Chen&amp;author=MH+Cobb&amp;volume=26&amp;publication_year=2007&amp;pages=3100-3112&amp;pmid=17496909&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="mixed-citation">Peti W, Page R, <span class="ref-title">Molecular basis of MAP kinase regulation</span>. <span class="ref-journal">Protein. Sci</span>
<span class="ref-vol">22</span>, 1698&#x02013;1710 (2013). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3843625/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24115095" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein.+Sci&amp;title=Molecular+basis+of+MAP+kinase+regulation&amp;author=W+Peti&amp;author=R+Page&amp;volume=22&amp;publication_year=2013&amp;pages=1698-1710&amp;pmid=24115095&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="mixed-citation">Tanoue T, Maeda R, Adachi M, Nishida E, <span class="ref-title">Identification of a docking groove on ERK and p38 MAP kinases that regulates the specificity of docking interactions</span>. <span class="ref-journal">EMBO J</span>. <span class="ref-vol">20</span>, 466&#x02013;479 (2001). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC133461/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11157753" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+J&amp;title=Identification+of+a+docking+groove+on+ERK+and+p38+MAP+kinases+that+regulates+the+specificity+of+docking+interactions&amp;author=T+Tanoue&amp;author=R+Maeda&amp;author=M+Adachi&amp;author=E+Nishida&amp;volume=20&amp;publication_year=2001&amp;pages=466-479&amp;pmid=11157753&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="mixed-citation">Goldsmith EJ, Akella R, Min X, Zhou T, Humphreys JM, <span class="ref-title">Substrate and docking interactions in serine/threonine protein kinases</span>. <span class="ref-journal">Chem. Rev</span>
<span class="ref-vol">107</span>, 5065&#x02013;5081 (2007). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4012561/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17949044" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem.+Rev&amp;title=Substrate+and+docking+interactions+in+serine/threonine+protein+kinases&amp;author=EJ+Goldsmith&amp;author=R+Akella&amp;author=X+Min&amp;author=T+Zhou&amp;author=JM+Humphreys&amp;volume=107&amp;publication_year=2007&amp;pages=5065-5081&amp;pmid=17949044&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="mixed-citation">Zhang YY, Wu JW, Wang ZX, <span class="ref-title">A distinct interaction mode revealed by the crystal structure of the kinase p38&#x003b1; with the MAPK binding domain of the phosphatase MKP5</span>. <span class="ref-journal">Sci. Signal</span>
<span class="ref-vol">4</span>, ra88 (2011). [<a href="https://pubmed.ncbi.nlm.nih.gov/22375048" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Signal&amp;title=A+distinct+interaction+mode+revealed+by+the+crystal+structure+of+the+kinase+p38&#x003b1;+with+the+MAPK+binding+domain+of+the+phosphatase+MKP5&amp;author=YY+Zhang&amp;author=JW+Wu&amp;author=ZX+Wang&amp;volume=4&amp;publication_year=2011&amp;pages=ra88&amp;pmid=22375048&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="mixed-citation">Garai A, Zeke A, Gogl G, Toro I, Fordos F, Blankenburg H, Barkai T, Varga J, Alexa A, Emig D, Albrecht M, Remenyi A, <span class="ref-title">Specificity of linear motifs that bind to a common mitogen-activated protein kinase docking groove</span>. <span class="ref-journal">Sci. Signal</span>
<span class="ref-vol">5</span>, ra74 (2012). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3500698/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23047924" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Signal&amp;title=Specificity+of+linear+motifs+that+bind+to+a+common+mitogen-activated+protein+kinase+docking+groove&amp;author=A+Garai&amp;author=A+Zeke&amp;author=G+Gogl&amp;author=I+Toro&amp;author=F+Fordos&amp;volume=5&amp;publication_year=2012&amp;pages=ra74&amp;pmid=23047924&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="mixed-citation">Chang CI, Xu BE, Akella R, Cobb MH, Goldsmith EJ, <span class="ref-title">Crystal structures of MAP kinase p38 complexed to the docking sites on its nuclear substrate MEF2A and activator MKK3b</span>. <span class="ref-journal">Mol. Cell</span>
<span class="ref-vol">9</span>, 1241&#x02013;1249 (2002). [<a href="https://pubmed.ncbi.nlm.nih.gov/12086621" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=Crystal+structures+of+MAP+kinase+p38+complexed+to+the+docking+sites+on+its+nuclear+substrate+MEF2A+and+activator+MKK3b&amp;author=CI+Chang&amp;author=BE+Xu&amp;author=R+Akella&amp;author=MH+Cobb&amp;author=EJ+Goldsmith&amp;volume=9&amp;publication_year=2002&amp;pages=1241-1249&amp;pmid=12086621&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="mixed-citation">Zeke A, Bastys T, Alexa A, Garai A, Meszaros B, Kirsch K, Dosztanyi Z, Kalinina OV, Remenyi A, <span class="ref-title">Systematic discovery of linear binding motifs targeting an ancient protein interaction surface on MAP kinases</span>. <span class="ref-journal">Mol. Syst. Biol</span>
<span class="ref-vol">11</span>, 837 (2015). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4670726/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26538579" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Syst.+Biol&amp;title=Systematic+discovery+of+linear+binding+motifs+targeting+an+ancient+protein+interaction+surface+on+MAP+kinases&amp;author=A+Zeke&amp;author=T+Bastys&amp;author=A+Alexa&amp;author=A+Garai&amp;author=B+Meszaros&amp;volume=11&amp;publication_year=2015&amp;pages=837&amp;pmid=26538579&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="mixed-citation">Sharrocks AD, Yang SH, Galanis A, <span class="ref-title">Docking domains and substrate-specificity determination for MAP kinases</span>. <span class="ref-journal">Trends Biochem. Sci</span>
<span class="ref-vol">25</span>, 448&#x02013;453 (2000). [<a href="https://pubmed.ncbi.nlm.nih.gov/10973059" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Biochem.+Sci&amp;title=Docking+domains+and+substrate-specificity+determination+for+MAP+kinases&amp;author=AD+Sharrocks&amp;author=SH+Yang&amp;author=A+Galanis&amp;volume=25&amp;publication_year=2000&amp;pages=448-453&amp;pmid=10973059&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="mixed-citation">Zhou T, Sun L, Humphreys J, Goldsmith EJ, <span class="ref-title">Docking interactions induce exposure of activation loop in the MAP kinase ERK2</span>. <span class="ref-journal">Structure</span>
<span class="ref-vol">14</span>, 1011&#x02013;1019 (2006). [<a href="https://pubmed.ncbi.nlm.nih.gov/16765894" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=Docking+interactions+induce+exposure+of+activation+loop+in+the+MAP+kinase+ERK2&amp;author=T+Zhou&amp;author=L+Sun&amp;author=J+Humphreys&amp;author=EJ+Goldsmith&amp;volume=14&amp;publication_year=2006&amp;pages=1011-1019&amp;pmid=16765894&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="mixed-citation">Kumar GS, Zettl H, Page R, Peti W, <span class="ref-title">Structural basis for the regulation of the mitogen-activated protein (MAP) kinase p38&#x003b1; by the dual specificity phosphatase 16 MAP kinase binding domain in solution</span>. <span class="ref-journal">J. Biol. Chem</span>
<span class="ref-vol">288</span>, 28347&#x02013;28356 (2013). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3784751/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23926106" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=Structural+basis+for+the+regulation+of+the+mitogen-activated+protein+(MAP)+kinase+p38&#x003b1;+by+the+dual+specificity+phosphatase+16+MAP+kinase+binding+domain+in+solution&amp;author=GS+Kumar&amp;author=H+Zettl&amp;author=R+Page&amp;author=W+Peti&amp;volume=288&amp;publication_year=2013&amp;pages=28347-28356&amp;pmid=23926106&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="mixed-citation">Kragelj J, Palencia A, Nanao MH, Maurin D, Bouvignies G, Blackledge M, Jensen MR, <span class="ref-title">Structure and dynamics of the MKK7-JNK signaling complex</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U. S. A</span>
<span class="ref-vol">112</span>, 3409&#x02013;3414 (2015). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4371970/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25737554" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.+S.+A&amp;title=Structure+and+dynamics+of+the+MKK7-JNK+signaling+complex&amp;author=J+Kragelj&amp;author=A+Palencia&amp;author=MH+Nanao&amp;author=D+Maurin&amp;author=G+Bouvignies&amp;volume=112&amp;publication_year=2015&amp;pages=3409-3414&amp;pmid=25737554&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="mixed-citation">Xu B, Stippec S, Robinson FL, Cobb MH, <span class="ref-title">Hydrophobic as well as charged residues in both MEK1 and ERK2 are important for their proper docking</span>. <span class="ref-journal">J. Biol. Chem</span>
<span class="ref-vol">276</span>, 26509&#x02013;26515 (2001). [<a href="https://pubmed.ncbi.nlm.nih.gov/11352917" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=Hydrophobic+as+well+as+charged+residues+in+both+MEK1+and+ERK2+are+important+for+their+proper+docking&amp;author=B+Xu&amp;author=S+Stippec&amp;author=FL+Robinson&amp;author=MH+Cobb&amp;volume=276&amp;publication_year=2001&amp;pages=26509-26515&amp;pmid=11352917&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="mixed-citation">Tanoue T, Adachi M, Moriguchi T, Nishida E, <span class="ref-title">A conserved docking motif in MAP kinases common to substrates, activators and regulators</span>. <span class="ref-journal">Nat. Cell Biol</span>
<span class="ref-vol">2</span>, 110&#x02013;116 (2000). [<a href="https://pubmed.ncbi.nlm.nih.gov/10655591" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Cell+Biol&amp;title=A+conserved+docking+motif+in+MAP+kinases+common+to+substrates,+activators+and+regulators&amp;author=T+Tanoue&amp;author=M+Adachi&amp;author=T+Moriguchi&amp;author=E+Nishida&amp;volume=2&amp;publication_year=2000&amp;pages=110-116&amp;pmid=10655591&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="mixed-citation">Bardwell AJ, Bardwell L, <span class="ref-title">Two hydrophobic residues can determine the specificity of mitogen-activated protein kinase docking interactions</span>. <span class="ref-journal">J. Biol. Chem</span>
<span class="ref-vol">290</span>, 26661&#x02013;26674 (2015). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4646321/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26370088" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=Two+hydrophobic+residues+can+determine+the+specificity+of+mitogen-activated+protein+kinase+docking+interactions&amp;author=AJ+Bardwell&amp;author=L+Bardwell&amp;volume=290&amp;publication_year=2015&amp;pages=26661-26674&amp;pmid=26370088&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="mixed-citation">Barsyte-Lovejoy D, Galanis A, Sharrocks AD, <span class="ref-title">Specificity determinants in MAPK signaling to transcription factors</span>. <span class="ref-journal">J. Biol. Chem</span>
<span class="ref-vol">277</span>, 9896&#x02013;9903 (2002). [<a href="https://pubmed.ncbi.nlm.nih.gov/11786537" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=Specificity+determinants+in+MAPK+signaling+to+transcription+factors&amp;author=D+Barsyte-Lovejoy&amp;author=A+Galanis&amp;author=AD+Sharrocks&amp;volume=277&amp;publication_year=2002&amp;pages=9896-9903&amp;pmid=11786537&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="mixed-citation">Heo YS, Kim SK, Seo CI, Kim YK, Sung BJ, Lee HS, Lee JI, Park SY, Kim JH, Hwang KY, Hyun YL, Jeon YH, Ro S, Cho JM, Lee TG, Yang CH, <span class="ref-title">Structural basis for the selective inhibition of JNK1 by the scaffolding protein JIP1 and SP600125</span>. <span class="ref-journal">EMBO J</span>. <span class="ref-vol">23</span>, 2185&#x02013;2195 (2004). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC419904/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15141161" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+J&amp;title=Structural+basis+for+the+selective+inhibition+of+JNK1+by+the+scaffolding+protein+JIP1+and+SP600125&amp;author=YS+Heo&amp;author=SK+Kim&amp;author=CI+Seo&amp;author=YK+Kim&amp;author=BJ+Sung&amp;volume=23&amp;publication_year=2004&amp;pages=2185-2195&amp;pmid=15141161&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="mixed-citation">Haar ET, Prabakhar P, Liu X, Lepre C, <span class="ref-title">Crystal structure of the p38 alpha-MAPKAP kinase 2 heterodimer</span>. <span class="ref-journal">J. Biol. Chem</span>
<span class="ref-vol">282</span>, 9733&#x02013;9739 (2007). [<a href="https://pubmed.ncbi.nlm.nih.gov/17255097" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=Crystal+structure+of+the+p38+alpha-MAPKAP+kinase+2+heterodimer&amp;author=ET+Haar&amp;author=P+Prabakhar&amp;author=X+Liu&amp;author=C+Lepre&amp;volume=282&amp;publication_year=2007&amp;pages=9733-9739&amp;pmid=17255097&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="mixed-citation">Xin F, Wu J, <span class="ref-title">Crystal structure of the p38&#x003b1; MAP kinase in complex with a docking peptide from TAB1</span>. <span class="ref-journal">Sci. China Life Sci</span>
<span class="ref-vol">56</span>, 653&#x02013;660 (2013). [<a href="https://pubmed.ncbi.nlm.nih.gov/23722236" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+China+Life+Sci&amp;title=Crystal+structure+of+the+p38&#x003b1;+MAP+kinase+in+complex+with+a+docking+peptide+from+TAB1&amp;author=F+Xin&amp;author=J+Wu&amp;volume=56&amp;publication_year=2013&amp;pages=653-660&amp;pmid=23722236&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="mixed-citation">Whisenant TC, Ho DT, Benz RW, Rogers JS, Kaake RM, Gordon EA, Huang L, Baldi P, Bardwell L, <span class="ref-title">Computational prediction and experimental verification of new MAP kinase docking sites and substrates including Gli transcription factors</span>. <span class="ref-journal">PLoS Comput. Biol</span>
<span class="ref-vol">6</span>, e1000908 (2010). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2928751/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20865152" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Comput.+Biol&amp;title=Computational+prediction+and+experimental+verification+of+new+MAP+kinase+docking+sites+and+substrates+including+Gli+transcription+factors&amp;author=TC+Whisenant&amp;author=DT+Ho&amp;author=RW+Benz&amp;author=JS+Rogers&amp;author=RM+Kaake&amp;volume=6&amp;publication_year=2010&amp;pages=e1000908&amp;pmid=20865152&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="mixed-citation">Saito H, Posas F, <span class="ref-title">Response to hyperosmotic stress</span>. <span class="ref-journal">Genetics</span>
<span class="ref-vol">192</span>, 289&#x02013;318 (2012). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3454867/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23028184" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Response+to+hyperosmotic+stress&amp;author=H+Saito&amp;author=F+Posas&amp;volume=192&amp;publication_year=2012&amp;pages=289-318&amp;pmid=23028184&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="mixed-citation">Wurgler-Murphy SM, Maeda T, Witten EA, Saito H, <span class="ref-title">Regulation of the Saccharomyces cerevisiae HOG1 mitogen-activated protein kinase by the PTP2 and PTP3 protein tyrosine phosphatases</span>. <span class="ref-journal">Mol. Cell. Biol</span>
<span class="ref-vol">17</span>, 1289&#x02013;1297 (1997). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC231854/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9032256" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Biol&amp;title=Regulation+of+the+Saccharomyces+cerevisiae+HOG1+mitogen-activated+protein+kinase+by+the+PTP2+and+PTP3+protein+tyrosine+phosphatases&amp;author=SM+Wurgler-Murphy&amp;author=T+Maeda&amp;author=EA+Witten&amp;author=H+Saito&amp;volume=17&amp;publication_year=1997&amp;pages=1289-1297&amp;pmid=9032256&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="mixed-citation">Grewal S, Molina DM, Bardwell L, <span class="ref-title">Mitogen-activated protein kinase (MAPK)-docking sites in MAPK kinases function as tethers that are crucial for MAPK regulation in vivo</span>. <span class="ref-journal">Cell. Signal</span>
<span class="ref-vol">18</span>, 123&#x02013;134 (2006). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3017502/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15979847" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell.+Signal&amp;title=Mitogen-activated+protein+kinase+(MAPK)-docking+sites+in+MAPK+kinases+function+as+tethers+that+are+crucial+for+MAPK+regulation+in+vivo&amp;author=S+Grewal&amp;author=DM+Molina&amp;author=L+Bardwell&amp;volume=18&amp;publication_year=2006&amp;pages=123-134&amp;pmid=15979847&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="mixed-citation">Tzarum N, Komornik N, Ben Chetrit D, Engelberg D, Livnah O, <span class="ref-title">DEF pocket in p38alpha facilitates substrate selectivity and mediates autophosphorylation</span>. <span class="ref-journal">J. Biol. Chem</span>
<span class="ref-vol">288</span>, 19537&#x02013;19547 (2013). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3707655/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23671282" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=DEF+pocket+in+p38alpha+facilitates+substrate+selectivity+and+mediates+autophosphorylation&amp;author=N+Tzarum&amp;author=N+Komornik&amp;author=D+Ben+Chetrit&amp;author=D+Engelberg&amp;author=O+Livnah&amp;volume=288&amp;publication_year=2013&amp;pages=19537-19547&amp;pmid=23671282&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="mixed-citation">Ho DT, Bardwell AJ, Grewal S, Iverson C, Bardwell L, <span class="ref-title">Interacting JNK-docking sites in MKK7 promote binding and activation of JNK MAP kinases</span>. <span class="ref-journal">J. Biol. Chem</span>
<span class="ref-vol">281</span>, 13169&#x02013;13179 (2006). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3017509/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16533805" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=Interacting+JNK-docking+sites+in+MKK7+promote+binding+and+activation+of+JNK+MAP+kinases&amp;author=DT+Ho&amp;author=AJ+Bardwell&amp;author=S+Grewal&amp;author=C+Iverson&amp;author=L+Bardwell&amp;volume=281&amp;publication_year=2006&amp;pages=13169-13179&amp;pmid=16533805&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="mixed-citation">Bardwell AJ, Frankson E, Bardwell L, <span class="ref-title">Selectivity of docking sites in MAPK kinases</span>. <span class="ref-journal">J. Biol. Chem</span>
<span class="ref-vol">284</span>, 13165&#x02013;13173 (2009). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2676048/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19196711" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=Selectivity+of+docking+sites+in+MAPK+kinases&amp;author=AJ+Bardwell&amp;author=E+Frankson&amp;author=L+Bardwell&amp;volume=284&amp;publication_year=2009&amp;pages=13165-13173&amp;pmid=19196711&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="mixed-citation">Ho DT, Bardwell AJ, Abdollahi M, Bardwell L, <span class="ref-title">A docking site in MKK4 mediates high affinity binding to JNK MAPKs and competes with similar docking sites in JNK substrates</span>. <span class="ref-journal">J. Biol. Chem</span>
<span class="ref-vol">278</span>, 32662&#x02013;32672 (2003). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3017503/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12788955" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=A+docking+site+in+MKK4+mediates+high+affinity+binding+to+JNK+MAPKs+and+competes+with+similar+docking+sites+in+JNK+substrates&amp;author=DT+Ho&amp;author=AJ+Bardwell&amp;author=M+Abdollahi&amp;author=L+Bardwell&amp;volume=278&amp;publication_year=2003&amp;pages=32662-32672&amp;pmid=12788955&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="mixed-citation">Pellegrini E, Palencia A, Braun L, Kapp U, Bougdour A, Belrhali H, Bowler MW, Hakimi MA, <span class="ref-title">Structural basis for the subversion of MAP kinase signaling by an intrinsically disordered parasite secreted agonist</span>. <span class="ref-journal">Structure</span>
<span class="ref-vol">25</span>, 16&#x02013;26 (2017). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5222587/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27889209" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=Structural+basis+for+the+subversion+of+MAP+kinase+signaling+by+an+intrinsically+disordered+parasite+secreted+agonist&amp;author=E+Pellegrini&amp;author=A+Palencia&amp;author=L+Braun&amp;author=U+Kapp&amp;author=A+Bougdour&amp;volume=25&amp;publication_year=2017&amp;pages=16-26&amp;pmid=27889209&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="mixed-citation">Peterson LB, Yaffe MB, Imperiali B, <span class="ref-title">Selective mitogen activated protein kinase activity sensors through the application of directionally programmable D domain motifs</span>. <span class="ref-journal">Biochemistry</span>
<span class="ref-vol">53</span>, 5771&#x02013;5778 (2014). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4165445/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25153342" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Selective+mitogen+activated+protein+kinase+activity+sensors+through+the+application+of+directionally+programmable+D+domain+motifs&amp;author=LB+Peterson&amp;author=MB+Yaffe&amp;author=B+Imperiali&amp;volume=53&amp;publication_year=2014&amp;pages=5771-5778&amp;pmid=25153342&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="mixed-citation">Tokunaga Y, Takeuchi K, Takahashi H, Shimada I, <span class="ref-title">Allosteric enhancement of MAP kinase p38&#x003b1;'s activity and substrate selectivity by docking interactions</span>. <span class="ref-journal">Nat. Struct. Mol. Biol</span>
<span class="ref-vol">21</span>, 704&#x02013;711 (2014). [<a href="https://pubmed.ncbi.nlm.nih.gov/25038803" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Struct.+Mol.+Biol&amp;title=Allosteric+enhancement+of+MAP+kinase+p38&#x003b1;'s+activity+and+substrate+selectivity+by+docking+interactions&amp;author=Y+Tokunaga&amp;author=K+Takeuchi&amp;author=H+Takahashi&amp;author=I+Shimada&amp;volume=21&amp;publication_year=2014&amp;pages=704-711&amp;pmid=25038803&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="mixed-citation">Ehrenberger T, Cantley LC, Yaffe MB, <span class="ref-title">Computational prediction of protein-protein interactions</span>. <span class="ref-journal">Methods Mol. Biol</span>
<span class="ref-vol">1278</span>, 57&#x02013;75 (2015). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4435844/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25859943" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Mol.+Biol&amp;title=Computational+prediction+of+protein-protein+interactions&amp;author=T+Ehrenberger&amp;author=LC+Cantley&amp;author=MB+Yaffe&amp;volume=1278&amp;publication_year=2015&amp;pages=57-75&amp;pmid=25859943&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="mixed-citation">Krystkowiak I, Manguy J, Davey NE, <span class="ref-title">PSSMSearch: a server for modeling, visualization, proteome-wide discovery and annotation of protein motif specificity determinants</span>. <span class="ref-journal">Nucleic Acids Res</span>. <span class="ref-vol">46</span>, W235&#x02013;W241 (2018). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6030969/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29873773" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=PSSMSearch:+a+server+for+modeling,+visualization,+proteome-wide+discovery+and+annotation+of+protein+motif+specificity+determinants&amp;author=I+Krystkowiak&amp;author=J+Manguy&amp;author=NE+Davey&amp;volume=46&amp;publication_year=2018&amp;pages=W235-W241&amp;pmid=29873773&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="mixed-citation">Subramanian A, Tamayo P, Mootha VK, Mukherjee S, Ebert BL, Gillette MA, Paulovich A, Pomeroy SL, Golub TR, Lander ES, Mesirov JP, <span class="ref-title">Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U. S. A</span>
<span class="ref-vol">102</span>, 15545&#x02013;15550 (2005). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1239896/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16199517" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.+S.+A&amp;title=Gene+set+enrichment+analysis:+a+knowledge-based+approach+for+interpreting+genome-wide+expression+profiles&amp;author=A+Subramanian&amp;author=P+Tamayo&amp;author=VK+Mootha&amp;author=S+Mukherjee&amp;author=BL+Ebert&amp;volume=102&amp;publication_year=2005&amp;pages=15545-15550&amp;pmid=16199517&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="mixed-citation">Canovas B, Nebreda AR, <span class="ref-title">Diversity and versatility of p38 kinase signalling in health and disease</span>. <span class="ref-journal">Nat. Rev. Mol. Cell Biol</span>
<span class="ref-vol">22</span>, 346&#x02013;366 (2021). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7838852/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33504982" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Mol.+Cell+Biol&amp;title=Diversity+and+versatility+of+p38+kinase+signalling+in+health+and+disease&amp;author=B+Canovas&amp;author=AR+Nebreda&amp;volume=22&amp;publication_year=2021&amp;pages=346-366&amp;pmid=33504982&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="mixed-citation">Wentzel C, Sommer JE, Nair R, Stiefvater A, Sibarita JB, Scheiffele P, <span class="ref-title">mSYD1A, a mammalian synapse-defective-1 protein, regulates synaptogenic signaling and vesicle docking</span>. <span class="ref-journal">Neuron</span>
<span class="ref-vol">78</span>, 1012&#x02013;1023 (2013). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3719036/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23791195" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=mSYD1A,+a+mammalian+synapse-defective-1+protein,+regulates+synaptogenic+signaling+and+vesicle+docking&amp;author=C+Wentzel&amp;author=JE+Sommer&amp;author=R+Nair&amp;author=A+Stiefvater&amp;author=JB+Sibarita&amp;volume=78&amp;publication_year=2013&amp;pages=1012-1023&amp;pmid=23791195&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="mixed-citation">Lo HF, Tsai CY, Chen CP, Wang LJ, Lee YS, Chen CY, Liang CT, Cheong ML, Chen H, <span class="ref-title">Association of dysfunctional synapse defective 1 (SYDE1) with restricted fetal growth - SYDE1 regulates placental cell migration and invasion</span>. <span class="ref-journal">J Pathol</span>. <span class="ref-vol">241</span>, 324&#x02013;336 (2017). [<a href="https://pubmed.ncbi.nlm.nih.gov/27917469" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Pathol&amp;title=Association+of+dysfunctional+synapse+defective+1+(SYDE1)+with+restricted+fetal+growth+-+SYDE1+regulates+placental+cell+migration+and+invasion&amp;author=HF+Lo&amp;author=CY+Tsai&amp;author=CP+Chen&amp;author=LJ+Wang&amp;author=YS+Lee&amp;volume=241&amp;publication_year=2017&amp;pages=324-336&amp;pmid=27917469&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="mixed-citation">Hertz EPT, Kruse T, Davey NE, Lopez-Mendez B, Sigurethsson JO, Montoya G, Olsen JV, Nilsson J, <span class="ref-title">A conserved motif provides binding specificity to the PP2A-B56 phosphatase</span>. <span class="ref-journal">Mol. Cell</span>
<span class="ref-vol">63</span>, 686&#x02013;695 (2016). [<a href="https://pubmed.ncbi.nlm.nih.gov/27453045" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=A+conserved+motif+provides+binding+specificity+to+the+PP2A-B56+phosphatase&amp;author=EPT+Hertz&amp;author=T+Kruse&amp;author=NE+Davey&amp;author=B+Lopez-Mendez&amp;author=JO+Sigurethsson&amp;volume=63&amp;publication_year=2016&amp;pages=686-695&amp;pmid=27453045&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="mixed-citation">Ueki Y, Kruse T, Weisser MB, Sundell GN, Larsen MSY, Mendez BL, Jenkins NP, Garvanska DH, Cressey L, Zhang G, Davey N, Montoya G, Ivarsson Y, Kettenbach AN, Nilsson J, <span class="ref-title">A consensus binding motif for the PP4 protein phosphatase</span>. <span class="ref-journal">Mol. Cell</span>
<span class="ref-vol">76</span>, 953&#x02013;964 e956 (2019). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6981294/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31585692" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=A+consensus+binding+motif+for+the+PP4+protein+phosphatase&amp;author=Y+Ueki&amp;author=T+Kruse&amp;author=MB+Weisser&amp;author=GN+Sundell&amp;author=MSY+Larsen&amp;volume=76&amp;publication_year=2019&amp;pages=953-964+e956&amp;pmid=31585692&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="mixed-citation">Wigington CP, Roy J, Damle NP, Yadav VK, Blikstad C, Resch E, Wong CJ, Mackay DR, Wang JT, Krystkowiak I, Bradburn DA, Tsekitsidou E, Hong SH, Kaderali MA, Xu SL, Stearns T, Gingras AC, Ullman KS, Ivarsson Y, Davey NE, Cyert MS, <span class="ref-title">Systematic discovery of short linear motifs decodes calcineurin phosphatase signaling</span>. <span class="ref-journal">Mol. Cell</span>
<span class="ref-vol">79</span>, 342&#x02013;358 e312 (2020). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7416808/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32645368" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;title=Systematic+discovery+of+short+linear+motifs+decodes+calcineurin+phosphatase+signaling&amp;author=CP+Wigington&amp;author=J+Roy&amp;author=NP+Damle&amp;author=VK+Yadav&amp;author=C+Blikstad&amp;volume=79&amp;publication_year=2020&amp;pages=342-358+e312&amp;pmid=32645368&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="mixed-citation">Kataria M, Mouilleron S, Seo MH, Corbi-Verge C, Kim PM, Uhlmann F, <span class="ref-title">A PxL motif promotes timely cell cycle substrate dephosphorylation by the Cdc14 phosphatase</span>. <span class="ref-journal">Nat. Struct. Mol. Biol</span>
<span class="ref-vol">25</span>, 1093&#x02013;1102 (2018). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6292506/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30455435" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Struct.+Mol.+Biol&amp;title=A+PxL+motif+promotes+timely+cell+cycle+substrate+dephosphorylation+by+the+Cdc14+phosphatase&amp;author=M+Kataria&amp;author=S+Mouilleron&amp;author=MH+Seo&amp;author=C+Corbi-Verge&amp;author=PM+Kim&amp;volume=25&amp;publication_year=2018&amp;pages=1093-1102&amp;pmid=30455435&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="mixed-citation">Fantz DA, Jacobs D, Glossip D, Kornfeld K, <span class="ref-title">Docking sites on substrate proteins direct extracellular signal-regulated kinase to phosphorylate specific residues</span>. <span class="ref-journal">J. Biol. Chem</span>
<span class="ref-vol">276</span>, 27256&#x02013;27265 (2001). [<a href="https://pubmed.ncbi.nlm.nih.gov/11371562" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=Docking+sites+on+substrate+proteins+direct+extracellular+signal-regulated+kinase+to+phosphorylate+specific+residues&amp;author=DA+Fantz&amp;author=D+Jacobs&amp;author=D+Glossip&amp;author=K+Kornfeld&amp;volume=276&amp;publication_year=2001&amp;pages=27256-27265&amp;pmid=11371562&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="mixed-citation">Faustova I, Bulatovic L, Matiyevskaya F, Valk E, &#x000d6;rd M, Loog M, <span class="ref-title">A new linear cyclin docking motif that mediates exclusively S-phase CDK-specific signaling</span>. <span class="ref-journal">EMBO J</span>. <span class="ref-vol">40</span>, e105839 (2021). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7809796/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33210757" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+J&amp;title=A+new+linear+cyclin+docking+motif+that+mediates+exclusively+S-phase+CDK-specific+signaling&amp;author=I+Faustova&amp;author=L+Bulatovic&amp;author=F+Matiyevskaya&amp;author=E+Valk&amp;author=M+&#x000d6;rd&amp;volume=40&amp;publication_year=2021&amp;pages=e105839&amp;pmid=33210757&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="mixed-citation">Bandyopadhyay S, Bhaduri S, Ord M, Davey NE, Loog M, Pryciak PM, <span class="ref-title">Comprehensive analysis of G1 cyclin docking motif sequences that control CDK regulatory potency in vivo</span>. <span class="ref-journal">Curr. Biol</span>
<span class="ref-vol">30</span>, 4454&#x02013;4466 (2020). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8009629/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32976810" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Biol&amp;title=Comprehensive+analysis+of+G1+cyclin+docking+motif+sequences+that+control+CDK+regulatory+potency+in+vivo&amp;author=S+Bandyopadhyay&amp;author=S+Bhaduri&amp;author=M+Ord&amp;author=NE+Davey&amp;author=M+Loog&amp;volume=30&amp;publication_year=2020&amp;pages=4454-4466&amp;pmid=32976810&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="mixed-citation">G&#x000f3;gl G, Schneider KD, Yeh BJ, Alam N, Nguyen Ba AN, Moses AM, Het&#x000e9;nyi C, Rem&#x000e9;nyi A, Weiss EL, <span class="ref-title">The structure of an NDR/LATS kinase-Mob complex reveals a novel kinase-coactivator system and substrate docking mechanism</span>. <span class="ref-journal">PLoS Biol</span>. <span class="ref-vol">13</span>, e1002146 (2015). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4428629/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25966461" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol&amp;title=The+structure+of+an+NDR/LATS+kinase-Mob+complex+reveals+a+novel+kinase-coactivator+system+and+substrate+docking+mechanism&amp;author=G+G&#x000f3;gl&amp;author=KD+Schneider&amp;author=BJ+Yeh&amp;author=N+Alam&amp;author=AN+Nguyen+Ba&amp;volume=13&amp;publication_year=2015&amp;pages=e1002146&amp;pmid=25966461&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="mixed-citation">Davey NE, Seo MH, Yadav VK, Jeon J, Nim S, Krystkowiak I, Blikstad C, Dong D, Markova N, Kim PM, Ivarsson Y, <span class="ref-title">Discovery of short linear motif-mediated interactions through phage display of intrinsically disordered regions of the human proteome</span>. <span class="ref-journal">FEBS J</span>. <span class="ref-vol">284</span>, 485&#x02013;498 (2017). [<a href="https://pubmed.ncbi.nlm.nih.gov/28002650" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEBS+J&amp;title=Discovery+of+short+linear+motif-mediated+interactions+through+phage+display+of+intrinsically+disordered+regions+of+the+human+proteome&amp;author=NE+Davey&amp;author=MH+Seo&amp;author=VK+Yadav&amp;author=J+Jeon&amp;author=S+Nim&amp;volume=284&amp;publication_year=2017&amp;pages=485-498&amp;pmid=28002650&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="mixed-citation">Ivarsson Y, Arnold R, McLaughlin M, Nim S, Joshi R, Ray D, Liu B, Teyra J, Pawson T, Moffat J, Li SS, Sidhu SS, Kim PM, <span class="ref-title">Large-scale interaction profiling of PDZ domains through proteomic peptide-phage display using human and viral phage peptidomes</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U. S. A</span>
<span class="ref-vol">111</span>, 2542&#x02013;2547 (2014). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3932933/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24550280" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.+S.+A&amp;title=Large-scale+interaction+profiling+of+PDZ+domains+through+proteomic+peptide-phage+display+using+human+and+viral+phage+peptidomes&amp;author=Y+Ivarsson&amp;author=R+Arnold&amp;author=M+McLaughlin&amp;author=S+Nim&amp;author=R+Joshi&amp;volume=111&amp;publication_year=2014&amp;pages=2542-2547&amp;pmid=24550280&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="mixed-citation">Nguyen HQ, Roy J, Harink B, Damle NP, Latorraca NR, Baxter BC, Brower K, Longwell SA, Kortemme T, Thorn KS, Cyert MS, Fordyce PM, <span class="ref-title">Quantitative mapping of protein-peptide affinity landscapes using spectrally encoded beads</span>. <span class="ref-journal">eLife</span>
<span class="ref-vol">8</span>, e40499 (2019). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6728138/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31282865" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=Quantitative+mapping+of+protein-peptide+affinity+landscapes+using+spectrally+encoded+beads&amp;author=HQ+Nguyen&amp;author=J+Roy&amp;author=B+Harink&amp;author=NP+Damle&amp;author=NR+Latorraca&amp;volume=8&amp;publication_year=2019&amp;pages=e40499&amp;pmid=31282865&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="mixed-citation">Kirsch K, Zeke A, T&#x00151;ke O, Sok P, Sethi A, Seb&#x00151; A, Kumar GS, Egri P, P&#x000f3;ti &#x000c1; L, Gooley P, Peti W, Bento I, Alexa A, Rem&#x000e9;nyi A, <span class="ref-title">Co-regulation of the transcription controlling ATF2 phosphoswitch by JNK and p38</span>. <span class="ref-journal">Nat. Commun</span>
<span class="ref-vol">11</span>, 5769 (2020). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7666158/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33188182" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun&amp;title=Co-regulation+of+the+transcription+controlling+ATF2+phosphoswitch+by+JNK+and+p38&amp;author=K+Kirsch&amp;author=A+Zeke&amp;author=O+T&#x00151;ke&amp;author=P+Sok&amp;author=A+Sethi&amp;volume=11&amp;publication_year=2020&amp;pages=5769&amp;pmid=33188182&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="mixed-citation">Francis DM, R&#x000f3;&#x0017c;ycki B, Koveal D, Hummer G, Page R, Peti W, <span class="ref-title">Structural basis of p38&#x003b1; regulation by hematopoietic tyrosine phosphatase</span>. <span class="ref-journal">Nat. Chem. Biol</span>
<span class="ref-vol">7</span>, 916&#x02013;924 (2011). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3657131/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22057126" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Chem.+Biol&amp;title=Structural+basis+of+p38&#x003b1;+regulation+by+hematopoietic+tyrosine+phosphatase&amp;author=DM+Francis&amp;author=B+R&#x000f3;&#x0017c;ycki&amp;author=D+Koveal&amp;author=G+Hummer&amp;author=R+Page&amp;volume=7&amp;publication_year=2011&amp;pages=916-924&amp;pmid=22057126&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="mixed-citation">Piserchio A, Francis DM, Koveal D, Dalby KN, Page R, Peti W, Ghose R, <span class="ref-title">Docking interactions of hematopoietic tyrosine phosphatase with MAP kinases ERK2 and p38&#x003b1;</span>. <span class="ref-journal">Biochemistry</span>
<span class="ref-vol">51</span>, 8047&#x02013;8049 (2012). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3548032/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23030599" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Docking+interactions+of+hematopoietic+tyrosine+phosphatase+with+MAP+kinases+ERK2+and+p38&#x003b1;&amp;author=A+Piserchio&amp;author=DM+Francis&amp;author=D+Koveal&amp;author=KN+Dalby&amp;author=R+Page&amp;volume=51&amp;publication_year=2012&amp;pages=8047-8049&amp;pmid=23030599&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R52">52. <span class="mixed-citation">Mok J, Kim PM, Lam HY, Piccirillo S, Zhou X, Jeschke GR, Sheridan DL, Parker SA, Desai V, Jwa M, Cameroni E, Niu H, Good M, Remenyi A, Ma JL, Sheu YJ, Sassi HE, Sopko R, Chan CS, De Virgilio C, Hollingsworth NM, Lim WA, Stern DF, Stillman B, Andrews BJ, Gerstein MB, Snyder M, Turk BE, <span class="ref-title">Deciphering protein kinase specificity through large-scale analysis of yeast phosphorylation site motifs</span>. <span class="ref-journal">Sci. Signal</span>
<span class="ref-vol">3</span>, ra12 (2010). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2846625/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20159853" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Signal&amp;title=Deciphering+protein+kinase+specificity+through+large-scale+analysis+of+yeast+phosphorylation+site+motifs&amp;author=J+Mok&amp;author=PM+Kim&amp;author=HY+Lam&amp;author=S+Piccirillo&amp;author=X+Zhou&amp;volume=3&amp;publication_year=2010&amp;pages=ra12&amp;pmid=20159853&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R53">53. <span class="mixed-citation">Zeke A, Misheva M, Rem&#x000e9;nyi A, Bogoyevitch MA, <span class="ref-title">JNK signaling: Regulation and functions based on complex protein-protein partnerships</span>. <span class="ref-journal">Microbiol. Mol. Biol. Rev</span>
<span class="ref-vol">80</span>, 793&#x02013;835 (2016). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4981676/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27466283" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol.+Mol.+Biol.+Rev&amp;title=JNK+signaling:+Regulation+and+functions+based+on+complex+protein-protein+partnerships&amp;author=A+Zeke&amp;author=M+Misheva&amp;author=A+Rem&#x000e9;nyi&amp;author=MA+Bogoyevitch&amp;volume=80&amp;publication_year=2016&amp;pages=793-835&amp;pmid=27466283&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R54">54. <span class="mixed-citation">Kyriakis JM, Avruch J, <span class="ref-title">Mammalian MAPK signal transduction pathways activated by stress and inflammation: a 10-year update</span>. <span class="ref-journal">Physiol. Rev</span>
<span class="ref-vol">92</span>, 689&#x02013;737 (2012). [<a href="https://pubmed.ncbi.nlm.nih.gov/22535895" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Physiol.+Rev&amp;title=Mammalian+MAPK+signal+transduction+pathways+activated+by+stress+and+inflammation:+a+10-year+update&amp;author=JM+Kyriakis&amp;author=J+Avruch&amp;volume=92&amp;publication_year=2012&amp;pages=689-737&amp;pmid=22535895&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R55">55. <span class="mixed-citation">Minden A, Lin A, Claret FX, Abo A, Karin M, <span class="ref-title">Selective activation of the JNK signaling cascade and c-Jun transcriptional activity by the small GTPases Rac and Cdc42Hs</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">81</span>, 1147&#x02013;1157 (1995). [<a href="https://pubmed.ncbi.nlm.nih.gov/7600582" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Selective+activation+of+the+JNK+signaling+cascade+and+c-Jun+transcriptional+activity+by+the+small+GTPases+Rac+and+Cdc42Hs&amp;author=A+Minden&amp;author=A+Lin&amp;author=FX+Claret&amp;author=A+Abo&amp;author=M+Karin&amp;volume=81&amp;publication_year=1995&amp;pages=1147-1157&amp;pmid=7600582&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R56">56. <span class="mixed-citation">Coso OA, Chiariello M, Yu JC, Teramoto H, Crespo P, Xu N, Miki T, Gutkind JS, <span class="ref-title">The small GTP-binding proteins Rac1 and Cdc42 regulate the activity of the JNK/SAPK signaling pathway</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">81</span>, 1137&#x02013;1146 (1995). [<a href="https://pubmed.ncbi.nlm.nih.gov/7600581" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=The+small+GTP-binding+proteins+Rac1+and+Cdc42+regulate+the+activity+of+the+JNK/SAPK+signaling+pathway&amp;author=OA+Coso&amp;author=M+Chiariello&amp;author=JC+Yu&amp;author=H+Teramoto&amp;author=P+Crespo&amp;volume=81&amp;publication_year=1995&amp;pages=1137-1146&amp;pmid=7600581&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R57">57. <span class="mixed-citation">Goldberg AB, Cho E, Miller CJ, Lou HJ, Turk BE, <span class="ref-title">Identification of a Substrate-selective Exosite within the Metalloproteinase Anthrax Lethal Factor</span>. <span class="ref-journal">J. Biol. Chem</span>
<span class="ref-vol">292</span>, 814&#x02013;825 (2017). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5247655/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27909054" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=Identification+of+a+Substrate-selective+Exosite+within+the+Metalloproteinase+Anthrax+Lethal+Factor&amp;author=AB+Goldberg&amp;author=E+Cho&amp;author=CJ+Miller&amp;author=HJ+Lou&amp;author=BE+Turk&amp;volume=292&amp;publication_year=2017&amp;pages=814-825&amp;pmid=27909054&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R58">58. <span class="mixed-citation">Sheridan DL, Kong Y, Parker SA, Dalby KN, Turk BE, <span class="ref-title">Substrate discrimination among mitogen-activated protein kinases through distinct docking sequence motifs</span>. <span class="ref-journal">J. Biol. Chem</span>
<span class="ref-vol">283</span>, 19511&#x02013;19520 (2008). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2443660/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18482985" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=Substrate+discrimination+among+mitogen-activated+protein+kinases+through+distinct+docking+sequence+motifs&amp;author=DL+Sheridan&amp;author=Y+Kong&amp;author=SA+Parker&amp;author=KN+Dalby&amp;author=BE+Turk&amp;volume=283&amp;publication_year=2008&amp;pages=19511-19520&amp;pmid=18482985&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R59">59. <span class="mixed-citation">Th&#x000e9;venin AF, Zony CL, Bahnson BJ, Colman RF, <span class="ref-title">Activation by phosphorylation and purification of human c-Jun N-terminal kinase (JNK) isoforms in milligram amounts</span>. <span class="ref-journal">Protein Expr. Purif</span>
<span class="ref-vol">75</span>, 138&#x02013;146 (2011). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2991412/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20709173" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Expr.+Purif&amp;title=Activation+by+phosphorylation+and+purification+of+human+c-Jun+N-terminal+kinase+(JNK)+isoforms+in+milligram+amounts&amp;author=AF+Th&#x000e9;venin&amp;author=CL+Zony&amp;author=BJ+Bahnson&amp;author=RF+Colman&amp;volume=75&amp;publication_year=2011&amp;pages=138-146&amp;pmid=20709173&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R60">60. <span class="mixed-citation">Hietakangas V, Anckar J, Blomster HA, Fujimoto M, Palvimo JJ, Nakai A, Sistonen L, <span class="ref-title">PDSM, a motif for phosphorylation-dependent SUMO modification</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U. S. A</span>
<span class="ref-vol">103</span>, 45&#x02013;50 (2006). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1324973/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16371476" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.+S.+A&amp;title=PDSM,+a+motif+for+phosphorylation-dependent+SUMO+modification&amp;author=V+Hietakangas&amp;author=J+Anckar&amp;author=HA+Blomster&amp;author=M+Fujimoto&amp;author=JJ+Palvimo&amp;volume=103&amp;publication_year=2006&amp;pages=45-50&amp;pmid=16371476&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R61">61. <span class="mixed-citation">Gietz RD, Schiestl RH, <span class="ref-title">High-efficiency yeast transformation using the LiAc/SS carrier DNA/PEG method</span>. <span class="ref-journal">Nat. Protoc</span>
<span class="ref-vol">2</span>, 31&#x02013;34 (2007). [<a href="https://pubmed.ncbi.nlm.nih.gov/17401334" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Protoc&amp;title=High-efficiency+yeast+transformation+using+the+LiAc/SS+carrier+DNA/PEG+method&amp;author=RD+Gietz&amp;author=RH+Schiestl&amp;volume=2&amp;publication_year=2007&amp;pages=31-34&amp;pmid=17401334&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R62">62. <span class="mixed-citation">Miller CJ, Lou HJ, Simpson C, van de Kooij B, Ha BH, Fisher OS, Pirman NL, Boggon TJ, Rinehart J, Yaffe MB, Linding R, Turk BE, <span class="ref-title">Comprehensive profiling of the STE20 kinase family defines features essential for selective substrate targeting and signaling output</span>. <span class="ref-journal">PLoS Biol</span>. <span class="ref-vol">17</span>, e2006540 (2019). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6445471/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30897078" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol&amp;title=Comprehensive+profiling+of+the+STE20+kinase+family+defines+features+essential+for+selective+substrate+targeting+and+signaling+output&amp;author=CJ+Miller&amp;author=HJ+Lou&amp;author=C+Simpson&amp;author=B+van+de+Kooij&amp;author=BH+Ha&amp;volume=17&amp;publication_year=2019&amp;pages=e2006540&amp;pmid=30897078&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R63">63. <span class="mixed-citation">Kinoshita-Kikuta E, Aoki Y, Kinoshita E, Koike T, <span class="ref-title">Label-free kinase profiling using phosphate affinity polyacrylamide gel electrophoresis</span>. <span class="ref-journal">Mol. Cell. Proteomics</span>
<span class="ref-vol">6</span>, 356&#x02013;366 (2007). [<a href="https://pubmed.ncbi.nlm.nih.gov/17088264" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Proteomics&amp;title=Label-free+kinase+profiling+using+phosphate+affinity+polyacrylamide+gel+electrophoresis&amp;author=E+Kinoshita-Kikuta&amp;author=Y+Aoki&amp;author=E+Kinoshita&amp;author=T+Koike&amp;volume=6&amp;publication_year=2007&amp;pages=356-366&amp;pmid=17088264&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R64">64. <span class="mixed-citation">Kirsch K, Zeke A, Toke O, Sok P, Sethi A, Sebo A, Kumar GS, Egri P, Poti AL, Gooley P, Peti W, Bento I, Alexa A, Remenyi A, <span class="ref-title">Co-regulation of the transcription controlling ATF2 phosphoswitch by JNK and p38</span>. <span class="ref-journal">Nat. Commun</span>
<span class="ref-vol">11</span>, 5769 (2020). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7666158/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33188182" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun&amp;title=Co-regulation+of+the+transcription+controlling+ATF2+phosphoswitch+by+JNK+and+p38&amp;author=K+Kirsch&amp;author=A+Zeke&amp;author=O+Toke&amp;author=P+Sok&amp;author=A+Sethi&amp;volume=11&amp;publication_year=2020&amp;pages=5769&amp;pmid=33188182&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R65">65. <span class="mixed-citation">O'Shea JP, Chou MF, Quader SA, Ryan JK, Church GM, Schwartz D, <span class="ref-title">pLogo: a probabilistic approach to visualizing sequence motifs</span>. <span class="ref-journal">Nat. Methods</span>
<span class="ref-vol">10</span>, 1211&#x02013;1212 (2013). [<a href="https://pubmed.ncbi.nlm.nih.gov/24097270" ref="reftype=pubmed&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=pLogo:+a+probabilistic+approach+to+visualizing+sequence+motifs&amp;author=JP+O'Shea&amp;author=MF+Chou&amp;author=SA+Quader&amp;author=JK+Ryan&amp;author=GM+Church&amp;volume=10&amp;publication_year=2013&amp;pages=1211-1212&amp;pmid=24097270&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9995140&amp;issue-id=430691&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC9995140/pdf/nihms-1871559.pdf" class="int-view">PDF (1.8M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/9995140/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/9995140/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="9995140" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC9995140%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="8QXVS6dRspT2mL4NFRZ0RcAWBG8xgZKxeCIkE8Me3owvbOvK16h4qiY2LxiXBQ45">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC9995140%2F&amp;text=Proteome-wide%20screening%20for%20mitogen-activated%20protein%20kinase%20docking%20motifs%20and%20interactors" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC9995140%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9995140/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC9995140/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/36626580/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC9995140/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/36626580/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC9995140/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/9995140/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC9995140/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
