
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.04fe9e55f2b2.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.aafaa535cb7f.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.ca6dfa7b90b5.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Integrative RNA profiling of TBEV-infected neurons and astrocytes reveals potential pathogenic effectors - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="322C636756E6DED5000061DC4B00494D.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="1.6.0.post2+24fead6" />

<meta name="ncbi_domain" content="csbj" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC9167876/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9167876/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Computational and Structural Biotechnology Journal" /><meta name="citation_title" content="Integrative RNA profiling of TBEV-infected neurons and astrocytes reveals potential pathogenic effectors" /><meta name="citation_author" content="Martin Selinger" /><meta name="citation_author_institution" content="Faculty of Science, University of South Bohemia in &#x0010c;esk&#x000e9; Bud&#x0011b;jovice, Brani&#x00161;ovsk&#x000e1; 1760, &#x0010c;esk&#x000e9; Bud&#x0011b;jovice 370 05, Czech Republic" /><meta name="citation_author_institution" content="Institute of Parasitology, Biology Centre of the Academy of Sciences of the Czech Republic, Brani&#x00161;ovsk&#x000e1; 31, &#x0010c;esk&#x000e9; Bud&#x0011b;jovice 370 05, Czech Republic" /><meta name="citation_author" content="Pavl&#x000ed;na V&#x0011b;chtov&#x000e1;" /><meta name="citation_author_institution" content="Faculty of Science, University of South Bohemia in &#x0010c;esk&#x000e9; Bud&#x0011b;jovice, Brani&#x00161;ovsk&#x000e1; 1760, &#x0010c;esk&#x000e9; Bud&#x0011b;jovice 370 05, Czech Republic" /><meta name="citation_author" content="Hana Tykalov&#x000e1;" /><meta name="citation_author_institution" content="Faculty of Science, University of South Bohemia in &#x0010c;esk&#x000e9; Bud&#x0011b;jovice, Brani&#x00161;ovsk&#x000e1; 1760, &#x0010c;esk&#x000e9; Bud&#x0011b;jovice 370 05, Czech Republic" /><meta name="citation_author_institution" content="Institute of Parasitology, Biology Centre of the Academy of Sciences of the Czech Republic, Brani&#x00161;ovsk&#x000e1; 31, &#x0010c;esk&#x000e9; Bud&#x0011b;jovice 370 05, Czech Republic" /><meta name="citation_author" content="Petra O&#x00161;lej&#x00161;kov&#x000e1;" /><meta name="citation_author_institution" content="Faculty of Science, University of South Bohemia in &#x0010c;esk&#x000e9; Bud&#x0011b;jovice, Brani&#x00161;ovsk&#x000e1; 1760, &#x0010c;esk&#x000e9; Bud&#x0011b;jovice 370 05, Czech Republic" /><meta name="citation_author" content="Michaela Rumlov&#x000e1;" /><meta name="citation_author_institution" content="Department of Biotechnology, University of Chemistry and Technology, Prague 166 28, Czech Republic" /><meta name="citation_author" content="J&#x000e1;n &#x00160;t&#x0011b;rba" /><meta name="citation_author_institution" content="Faculty of Science, University of South Bohemia in &#x0010c;esk&#x000e9; Bud&#x0011b;jovice, Brani&#x00161;ovsk&#x000e1; 1760, &#x0010c;esk&#x000e9; Bud&#x0011b;jovice 370 05, Czech Republic" /><meta name="citation_author" content="Libor Grubhoffer" /><meta name="citation_author_institution" content="Faculty of Science, University of South Bohemia in &#x0010c;esk&#x000e9; Bud&#x0011b;jovice, Brani&#x00161;ovsk&#x000e1; 1760, &#x0010c;esk&#x000e9; Bud&#x0011b;jovice 370 05, Czech Republic" /><meta name="citation_author_institution" content="Institute of Parasitology, Biology Centre of the Academy of Sciences of the Czech Republic, Brani&#x00161;ovsk&#x000e1; 31, &#x0010c;esk&#x000e9; Bud&#x0011b;jovice 370 05, Czech Republic" /><meta name="citation_publication_date" content="2022" /><meta name="citation_volume" content="20" /><meta name="citation_firstpage" content="2759" /><meta name="citation_doi" content="10.1016/j.csbj.2022.05.052" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC9167876/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC9167876/" /><meta name="citation_pmid" content="35685361" /><meta name="DC.Title" content="Integrative RNA profiling of TBEV-infected neurons and astrocytes reveals potential pathogenic effectors" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Research Network of Computational and Structural Biotechnology" /><meta name="DC.Contributor" content="Martin Selinger" /><meta name="DC.Contributor" content="Pavl&#x000ed;na V&#x0011b;chtov&#x000e1;" /><meta name="DC.Contributor" content="Hana Tykalov&#x000e1;" /><meta name="DC.Contributor" content="Petra O&#x00161;lej&#x00161;kov&#x000e1;" /><meta name="DC.Contributor" content="Michaela Rumlov&#x000e1;" /><meta name="DC.Contributor" content="J&#x000e1;n &#x00160;t&#x0011b;rba" /><meta name="DC.Contributor" content="Libor Grubhoffer" /><meta name="DC.Date" content="2022" /><meta name="DC.Identifier" content="10.1016/j.csbj.2022.05.052" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Integrative RNA profiling of TBEV-infected neurons and astrocytes reveals potential pathogenic effectors" /><meta property="og:type" content="article" /><meta property="og:description" content="Tick-borne encephalitis virus (TBEV), the most medically relevant tick-transmitted flavivirus in Eurasia, targets the host central nervous system and frequently causes severe encephalitis. The severity of TBEV-induced neuropathogenesis is highly cell-type ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9167876/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/2373/">Comput Struct Biotechnol J</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/395563/">v.20; 2022</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC9167876
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC9167876/pdf/main.pdf" class="int-view">PDF (4.1M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/9167876/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/9167876/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="9167876" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC9167876%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="cZ6wqT6S8LMNFmC5BSaseL5QJbXWrR8dKDrAvO7BseOunbMdEuove2zm1YkeLtLU">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC9167876%2F&amp;text=Integrative%20RNA%20profiling%20of%20TBEV-infected%20neurons%20and%20astrocytes%20reveals%20potential%20pathogenic%20effectors" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC9167876%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9167876/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC9167876/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/35685361/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC9167876/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/35685361/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC9167876/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/9167876/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/2373/">Comput Struct Biotechnol J</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/395563/">v.20; 2022</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC9167876
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-csbj.gif" alt="Logo of csbj" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="0,30,118,50" alt="Guide for Authors" title="Guide for Authors" href="http://www.elsevier.com/journals/Computational-and-Structural-Biotechnology-Journal/2001-0370/guide-for-authors" ref="reftype=publisher&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="118,30,165,50" alt="About this journal" title="About this journal" href="http://www.journals.elsevier.com/Computational-and-Structural-Biotechnology-Journal" ref="reftype=publisher&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="165,30,286,50" alt="Explore this journal" title="Explore this journal" href="http://www.sciencedirect.com/science/journal/20010370" ref="reftype=publisher&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="0,0,499,74" alt="Computational and Structural Biotechnology Journal" title="Computational and Structural Biotechnology Journal" href="http://csbj.org/" ref="reftype=publisher&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Comput Struct Biotechnol J.</span> 2022; 20: 2759–2777. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2022 May 30. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1016%2Fj.csbj.2022.05.052" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.csbj.2022.05.052</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC9167876</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/35685361">35685361</a></div></div></div><h1 class="content-title">Integrative RNA profiling of TBEV-infected neurons and astrocytes reveals potential pathogenic effectors</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Selinger%20M%5BAuthor%5D" id="au005" class="affpopup" co-rid="_co_idm139861396735840" co-class="co-affbox">Martin Selinger</a>,<sup>a,</sup><sup>b,</sup><sup>&#x0204e;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=V&#x0011b;chtov&#x000e1;%20P%5BAuthor%5D" id="au010" class="affpopup" co-rid="_co_idm139861396708528" co-class="co-affbox">Pavl&#x000ed;na V&#x0011b;chtov&#x000e1;</a>,<sup>a</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Tykalov&#x000e1;%20H%5BAuthor%5D" id="au015" class="affpopup" co-rid="_co_idm139861396706304" co-class="co-affbox">Hana Tykalov&#x000e1;</a>,<sup>a,</sup><sup>b</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=O&#x00161;lej&#x00161;kov&#x000e1;%20P%5BAuthor%5D" id="au020" class="affpopup" co-rid="_co_idm139861396703344" co-class="co-affbox">Petra O&#x00161;lej&#x00161;kov&#x000e1;</a>,<sup>a</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Rumlov&#x000e1;%20M%5BAuthor%5D" id="au025" class="affpopup" co-rid="_co_idm139861396701120" co-class="co-affbox">Michaela Rumlov&#x000e1;</a>,<sup>c</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=&#x00160;t&#x0011b;rba%20J%5BAuthor%5D" id="au030" class="affpopup" co-rid="_co_idm139861396698896" co-class="co-affbox">J&#x000e1;n &#x00160;t&#x0011b;rba</a>,<sup>a</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Grubhoffer%20L%5BAuthor%5D" id="au035" class="affpopup" co-rid="_co_idm139861396696672" co-class="co-affbox">Libor Grubhoffer</a><sup>a,</sup><sup>b</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139861396735840"><h3 class="no_margin">Martin Selinger</h3><p><sup>a</sup>Faculty of Science, University of South Bohemia in &#x0010c;esk&#x000e9; Bud&#x0011b;jovice, Brani&#x00161;ovsk&#x000e1; 1760, &#x0010c;esk&#x000e9; Bud&#x0011b;jovice 370 05, Czech Republic</p><p><sup>b</sup>Institute of Parasitology, Biology Centre of the Academy of Sciences of the Czech Republic, Brani&#x00161;ovsk&#x000e1; 31, &#x0010c;esk&#x000e9; Bud&#x0011b;jovice 370 05, Czech Republic</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Selinger%20M%5BAuthor%5D">Martin Selinger</a></div></div><div id="_co_idm139861396708528"><h3 class="no_margin">Pavl&#x000ed;na V&#x0011b;chtov&#x000e1;</h3><p><sup>a</sup>Faculty of Science, University of South Bohemia in &#x0010c;esk&#x000e9; Bud&#x0011b;jovice, Brani&#x00161;ovsk&#x000e1; 1760, &#x0010c;esk&#x000e9; Bud&#x0011b;jovice 370 05, Czech Republic</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=V&#x0011b;chtov&#x000e1;%20P%5BAuthor%5D">Pavl&#x000ed;na V&#x0011b;chtov&#x000e1;</a></div></div><div id="_co_idm139861396706304"><h3 class="no_margin">Hana Tykalov&#x000e1;</h3><p><sup>a</sup>Faculty of Science, University of South Bohemia in &#x0010c;esk&#x000e9; Bud&#x0011b;jovice, Brani&#x00161;ovsk&#x000e1; 1760, &#x0010c;esk&#x000e9; Bud&#x0011b;jovice 370 05, Czech Republic</p><p><sup>b</sup>Institute of Parasitology, Biology Centre of the Academy of Sciences of the Czech Republic, Brani&#x00161;ovsk&#x000e1; 31, &#x0010c;esk&#x000e9; Bud&#x0011b;jovice 370 05, Czech Republic</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Tykalov&#x000e1;%20H%5BAuthor%5D">Hana Tykalov&#x000e1;</a></div></div><div id="_co_idm139861396703344"><h3 class="no_margin">Petra O&#x00161;lej&#x00161;kov&#x000e1;</h3><p><sup>a</sup>Faculty of Science, University of South Bohemia in &#x0010c;esk&#x000e9; Bud&#x0011b;jovice, Brani&#x00161;ovsk&#x000e1; 1760, &#x0010c;esk&#x000e9; Bud&#x0011b;jovice 370 05, Czech Republic</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=O&#x00161;lej&#x00161;kov&#x000e1;%20P%5BAuthor%5D">Petra O&#x00161;lej&#x00161;kov&#x000e1;</a></div></div><div id="_co_idm139861396701120"><h3 class="no_margin">Michaela Rumlov&#x000e1;</h3><p><sup>c</sup>Department of Biotechnology, University of Chemistry and Technology, Prague 166 28, Czech Republic</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Rumlov&#x000e1;%20M%5BAuthor%5D">Michaela Rumlov&#x000e1;</a></div></div><div id="_co_idm139861396698896"><h3 class="no_margin">J&#x000e1;n &#x00160;t&#x0011b;rba</h3><p><sup>a</sup>Faculty of Science, University of South Bohemia in &#x0010c;esk&#x000e9; Bud&#x0011b;jovice, Brani&#x00161;ovsk&#x000e1; 1760, &#x0010c;esk&#x000e9; Bud&#x0011b;jovice 370 05, Czech Republic</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=&#x00160;t&#x0011b;rba%20J%5BAuthor%5D">J&#x000e1;n &#x00160;t&#x0011b;rba</a></div></div><div id="_co_idm139861396696672"><h3 class="no_margin">Libor Grubhoffer</h3><p><sup>a</sup>Faculty of Science, University of South Bohemia in &#x0010c;esk&#x000e9; Bud&#x0011b;jovice, Brani&#x00161;ovsk&#x000e1; 1760, &#x0010c;esk&#x000e9; Bud&#x0011b;jovice 370 05, Czech Republic</p><p><sup>b</sup>Institute of Parasitology, Biology Centre of the Academy of Sciences of the Czech Republic, Brani&#x00161;ovsk&#x000e1; 31, &#x0010c;esk&#x000e9; Bud&#x0011b;jovice 370 05, Czech Republic</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Grubhoffer%20L%5BAuthor%5D">Libor Grubhoffer</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm139861434671520_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139861434671520_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139861434671520_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm139861434671520_ai" style="display:none"><div class="fm-affl" id="af005"><sup>a</sup>Faculty of Science, University of South Bohemia in &#x0010c;esk&#x000e9; Bud&#x0011b;jovice, Brani&#x00161;ovsk&#x000e1; 1760, &#x0010c;esk&#x000e9; Bud&#x0011b;jovice 370 05, Czech Republic</div><div class="fm-affl" id="af010"><sup>b</sup>Institute of Parasitology, Biology Centre of the Academy of Sciences of the Czech Republic, Brani&#x00161;ovsk&#x000e1; 31, &#x0010c;esk&#x000e9; Bud&#x0011b;jovice 370 05, Czech Republic</div><div class="fm-affl" id="af015"><sup>c</sup>Department of Biotechnology, University of Chemistry and Technology, Prague 166 28, Czech Republic</div><div><span class="contrib-email" id="au005">Martin Selinger: <a href="mailto:dev@null" data-email="zc.ucj.frp@10mniles" class="oemail">zc.ucj.frp@10mniles</a></span> </div><div id="cor1"><sup>&#x0204e;</sup>Corresponding author at: Faculty of Science, University of South Bohemia in &#x0010c;esk&#x000e9; Bud&#x0011b;jovice, Brani&#x00161;ovsk&#x000e1; 1760, &#x0010c;esk&#x000e9; Bud&#x0011b;jovice 370 05, Czech Republic. <a href="mailto:dev@null" data-email="zc.ucj.frp@10mniles" class="oemail">zc.ucj.frp@10mniles</a></div></div><div class="fm-article-notes hide half_rhythm" id="idm139861434671520_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2022 Feb 13; Revised 2022 May 27; Accepted 2022 May 27.</div></div><div class="permissions half_rhythm hide" id="idm139861434671520_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; 2022 The Authors</div><div class="license half_rhythm">This is an open access article under the CC BY-NC-ND license (http://creativecommons.org/licenses/by-nc-nd/4.0/).</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="10" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary data 1</div><div><a href="/pmc/articles/PMC9167876/bin/mmc1.pdf" data-ga-action="click_feat_suppl">mmc1.pdf</a><span style="color:gray"> (225K)</span></div><div class="small guid">GUID: 855C3797-CFA0-4810-B746-4EA07B821654</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary data 2</div><div><a href="/pmc/articles/PMC9167876/bin/mmc2.xlsx" data-ga-action="click_feat_suppl">mmc2.xlsx</a><span style="color:gray"> (11K)</span></div><div class="small guid">GUID: DDAA8EF3-67AC-40C7-9080-C4518E1EB13C</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary data 3</div><div><a href="/pmc/articles/PMC9167876/bin/mmc3.xlsx" data-ga-action="click_feat_suppl">mmc3.xlsx</a><span style="color:gray"> (292K)</span></div><div class="small guid">GUID: 17B46CDF-C5E5-4E26-A7A9-4A041ABB2D6E</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary data 4</div><div><a href="/pmc/articles/PMC9167876/bin/mmc4.xlsx" data-ga-action="click_feat_suppl">mmc4.xlsx</a><span style="color:gray"> (9.1K)</span></div><div class="small guid">GUID: 6C69097D-F411-4715-BF41-CE5513CD06A9</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary data 5</div><div><a href="/pmc/articles/PMC9167876/bin/mmc5.xlsx" data-ga-action="click_feat_suppl">mmc5.xlsx</a><span style="color:gray"> (18K)</span></div><div class="small guid">GUID: C4A54E1F-8BB0-490B-A730-EC054EC96CAE</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary data 6</div><div><a href="/pmc/articles/PMC9167876/bin/mmc6.xlsx" data-ga-action="click_feat_suppl">mmc6.xlsx</a><span style="color:gray"> (12K)</span></div><div class="small guid">GUID: 0831F665-53F8-48BB-A09E-F8C99C55C56C</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary data 7</div><div><a href="/pmc/articles/PMC9167876/bin/mmc7.xlsx" data-ga-action="click_feat_suppl">mmc7.xlsx</a><span style="color:gray"> (11K)</span></div><div class="small guid">GUID: 35D56422-5DE3-4699-919A-78D7D3DD3576</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary data 8</div><div><a href="/pmc/articles/PMC9167876/bin/mmc8.xlsx" data-ga-action="click_feat_suppl">mmc8.xlsx</a><span style="color:gray"> (13K)</span></div><div class="small guid">GUID: 9380D5C3-57D1-4A2F-8159-FDB86BB921BD</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary data 9</div><div><a href="/pmc/articles/PMC9167876/bin/mmc9.xlsx" data-ga-action="click_feat_suppl">mmc9.xlsx</a><span style="color:gray"> (11K)</span></div><div class="small guid">GUID: C804D7E5-F212-4160-A078-CA664F863FB4</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary data 10</div><div><a href="/pmc/articles/PMC9167876/bin/mmc10.xlsx" data-ga-action="click_feat_suppl">mmc10.xlsx</a><span style="color:gray"> (1.0M)</span></div><div class="small guid">GUID: 35BC5DDA-838E-46C6-A379-7926EAB11829</div></div></dd></dl></div><div id="ab005" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="ab005title">Graphical abstract</h2><!--article-meta--><div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig graphic_only iconblock anchored whole_rhythm" id="f0040"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139861398615360"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9167876_ga1.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ga1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC9167876/bin/ga1.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139861398615360"><a target="object" rel="noopener" href="/pmc/articles/PMC9167876/figure/f0040/?report=objectonly">Open in a separate window</a></div></div></div><div class="sec"><strong class="kwd-title">Abbreviations: </strong><span class="kwd-text">A3SS, alternative 3&#x02032; splice site, A5SS, alternative 5&#x02032; splice site, ACACA, Acetyl-CoA Carboxylase Alpha, AKR1C2, Aldo-Keto Reductase Family 1 Member C2, ANKS1A, Ankyrin Repeat And Sterile Alpha Motif Domain Containing 1A, ANOS1, Anosmin 1, AOX1, Aldehyde Oxidase 1, APOBEC3G, Apolipoprotein B MRNA Editing Enzyme Catalytic Subunit 3G, APOL1/6, Apolipoprotein L1/6, ARID2, AT-Rich Interaction Domain 2, AUTS2, Activator Of Transcription And Developmental Regulator AUTS2, BCL11B, BAF Chromatin Remodeling Complex Subunit BCL11B, BCL9L, BCL9 Transcription Coactivator-like, BDKRB2, Bradykinin Receptor B2, BDNF, Brain Derived Neurotrophic Factor, BEND3, BEN Domain Containing 3, BSA, bovine serum albumin, BST2, Bone Marrow Stromal Cell Antigen 2, CALB1, Calbindin 1, CAMK2A, Calcium/Calmodulin Dependent Protein Kinase II Alpha, CD, complement determinant, CDKN1C, Cyclin Dependent Kinase Inhibitor 1C, CFAP61, Cilia And Flagella Associated Protein 61, CHRNA3, Cholinergic Receptor Nicotinic Alpha 3 Subunit, CHRNB4, Cholinergic Receptor Nicotinic Beta 4 Subunit, CLIC5, Chloride Intracellular Channel 5, CMPK2, Cytidine/Uridine Monophosphate Kinase 2, CNS, central nervous system, CNTN2, Contactin 2, CREG2, Cellular Repressor Of E1A Stimulated Genes 2, CXADR, Coxsackievirus B-Adenovirus Receptor, CYYR1, Cysteine And Tyrosine Rich 1, DACH1, Dachshund Family Transcription Factor 1, DAPI, diamidino-2-phenylindole, DCC, Netrin 1 Receptor, DCX, Doublecortin, DDX60, DExD/H-Box Helicase 60, DDX60L, DExD/H-Box 60 Like, DE, differentially expressed, DENV, Dengue virus, DIRAS2, DIRAS Family GTPase 2, DLX1/5/6, Distal-Less Homeobox 1/5/6, DNMT3B, DNA Methyltransferase 3 Beta, DPYSL2, Dihydropyrimidinase Like 2, EBF1, EBF Transcription Factor 1, EGF, Epidermal Growth Factor, ELAVL2/4, ELAV Like RNA Binding Protein 2/4, EPHB1, EPH Receptor B1, EPSTI1, Epithelial Stromal Interaction 1, ERBB4, Erb-B2 Receptor Tyrosine Kinase 4, ES, exon skipping, ESRRG, Estrogen Related Receptor Gamma, FGFb, Fibroblast Growth Factor 2, FPKM, Fragments Per Kilobase of transcript per Million mapped reads, FUT9, Fucosyltransferase 9, G2E3, G2/M&#x02212;Phase Specific E3 Ubiquitin Protein Ligase, GABRG2, Gamma-Aminobutyric Acid Type A Receptor Subunit Gamma 2, GAPDH, Glyceraldehyde-3-Phosphate Dehydrogenase, GAS2L3, Growth Arrest Specific 2 Like 3, GAS7, Growth Arrest Specific 7, GATAD2B, GATA Zinc Finger Domain Containing 2B, GFAP, Glial Fibrillary Acidic Protein, GIPC2, GIPC PDZ Domain Containing Family Member 2, GLRA2, Glycine Receptor Alpha 2, GNG2, G Protein Subunit Gamma 2, GO, gene ontology, GOLGA4, Golgin A4, GRIN2A, Glutamate Ionotropic Receptor NMDA Type Subunit 2A, gRNA, genomic TBEV RNA, GSEA, gene set enrichment analysis, HERC5/6, HECT And RLD Domain Containing E3 Ubiquitin Protein Ligase 5/6, HEYL, Hes Related Family BHLH Transcription Factor With YRPW Motif Like, hNSC, human neural stem cells, HPRT1, Hypoxanthine Phosphoribosyltransferase 1, HS, hot-spot, HSPA6, Heat Shock Protein Family A (Hsp70) Member 6, HUDD (ELAV4), Hu-Antigen D/ELAV Like Neuron-Specific RNA Binding Protein 4, IFI6, Interferon Alpha Inducible Protein 6, IFIH1 (MDA5), Interferon Induced With Helicase C Domain 1/Melanoma Differentiation-Associated Protein 5, IFIT1-3, Interferon Induced Protein With Tetratricopeptide Repeats 1&#x02013;3, IFITM1/2, Interferon Induced Transmembrane Protein 1/2, IFN, interferon, IGB, Integrated Genome Browser, IL6, Interleukin 6, IR, intron retention, ISG20, Interferon Stimulated Exonuclease Gene 20, ISGF3, Interferon-Stimulated Gene Factor 3 Gamma, ISGs, interferon-stimulated genes, JEV, Japanese encephalitis virus, KCND2, Potassium Voltage-Gated Channel Subfamily D Member 2, KCNK10, Potassium Two Pore Domain Channel Subfamily K Member 10, KCNS2, Potassium Voltage-Gated Channel Modifier Subfamily S Member 2, KIT, KIT Proto-Oncogene, Receptor Tyrosine Kinase, KLHDC8A, Kelch Domain Containing 8A, KLHL13, Kelch Like Family Member 13, KRR1, KRR1 Small Subunit Processome Component Homolog, LCOR, Ligand Dependent Nuclear Receptor Corepressor, LEKR1, Leucine, Glutamate And Lysine Rich 1, LGI1, Leucine Rich Glioma Inactivated 1, lncRNA, long non-coding RNA, LRRTM3, Leucine Rich Repeat Transmembrane Neuronal 3, LSV, local splicing variation, LUZP2, Leucine Zipper Protein 2, MAN1A1, Mannosidase Alpha Class 1A Member 1, MAP2, Microtubule Associated Protein 2, MBNL2, Muscleblind Like Splicing Regulator 2, MCTP1, Multiple C2 And Transmembrane Domain Containing 1, miRNA, micro RNA, MMP13, Matrix Metallopeptidase 13, MN1, MN1 Proto-Oncogene, Transcriptional Regulator, MOI, multiplicity of infection, mRNA, messenger RNA, MTUS2, Microtubule Associated Scaffold Protein 2, MX2, MX Dynamin Like GTPase 2, MYCN, MYCN Proto-Oncogene, BHLH Transcription Factor, NAV1, Neuron Navigator 1, NCAM1, Neural Cell Adhesion Molecule 1, ncRNA, non-coding RNA, NDRG4, N-Myc Downstream-Regulated Gene 4 Protein, NEK7, NIMA Related Kinase 7, NFASC, Neurofascin, NKAIN1, Sodium/Potassium Transporting ATPase Interacting 1, NMI, N-Myc And STAT Interactor 2, NRAP, Nebulin Related Anchoring Protein, NRARP, NOTCH Regulated Ankyrin Repeat Protein, NREP, Neuronal Regeneration Related Protein, NRN1, Neuritin 1, NS3, flaviviral non-structural protein 3, NXPH2, Neurexophilin 2, NYNRIN, NYN Domain And Retroviral Integrase Containing, OAS, 2&#x02032;-5&#x02032;-Oligoadenylate Synthetase, OASL, 2&#x02032;-5&#x02032;-Oligoadenylate Synthetase Like, ONECUT2, ONECUT-2 Homeodomain Transcription Factor, OPCML, Opioid Binding Protein/Cell Adhesion Molecule Like, OTX2, Orthodenticle Homeobox 2, PBS, phosphate buffer saline, PBX1, Pre-B-Cell Leukemia Transcription Factor 1, PCDH18/20, Protocadherin 18/20, pc-mRNA, protein-coding mRNA, PFKFB3, 6-Phosphofructo-2-Kinase/Fructose-2,6-Biphosphatase 3, PIK3C2B, Phosphatidylinositol-4-Phosphate 3-Kinase Catalytic Subunit Type 2 Beta, PIP4P2, Phosphatidylinositol-4,5-Bisphosphate 4-Phosphatase 2, PLCH1, Phospholipase C Eta 1, POU3F4, Brain-Specific Homeobox/POU Domain Protein 4, PPM1L, Protein Phosphatase, Mg2+/Mn2+ Dependent 1L, PPP1R17, Protein Phosphatase 1 Regulatory Subunit 17, PRDM12, PR Domain Zinc Finger Protein 12, PSI, percent selective index, PSRC1, Proline And Serine Rich Coiled-Coil 1, PTPN5, Protein Tyrosine Phosphatase Non-Receptor Type 5, PTPRH, Protein Tyrosine Phosphatase Receptor Type H, qRT-PCR, quantitative reverse transcription real-time PCR, RAPGEF5, Rap Guanine Nucleotide Exchange Factor 5, RBFOX1, RNA Binding Fox-1 Homolog 1, RIG-I (DDX58), Retinoic Acid-Inducible Gene 1 Protein, RNF212, Ring Finger Protein 212, RNVU1, RNA, Variant U1 Small Nuclear, RSAD2, Radical S-Adenosyl Methionine Domain Containing 2, RTL8B, Retrotransposon Gag Like 8B, SAMD9, Sterile Alpha Motif Domain Containing 9, SEMA3E, Semaphorin 3E, SH3TC2, SH3 Domain And Tetratricopeptide Repeats 2, SHF, Src Homology 2 Domain Containing F, SHISAL1, Shisa Like 1, SIAH3, Siah E3 Ubiquitin Protein Ligase Family Member 3, SIRPA, Signal Regulatory Protein Alpha, SLITRK5, SLIT And NTRK Like Family Member 5, SNP, single-nucleotide polymorphism, snRNP, small nuclear ribonucleoproteins, SOGA1, Suppressor Of Glucose, Autophagy Associated 1, SPSB4, SplA/Ryanodine Receptor Domain And SOCS Box Containing 4, ST6GAL1, ST6 Beta-Galactoside Alpha-2,6-Sialyltransferase 1, TBC1D30, TBC1 Domain Family Member 30, TBEV, Tick-borne encephalitis virus, TFAP2A, Transcription Factor AP-2 Alpha, TFAP2B, Transcription Factor AP-2 Beta, THSD7A, Thrombospondin Type 1 Domain Containing 7A, THUMPD2, THUMP Domain-Containing Protein 2/SAM-Dependent Methyltransferase, TIPARP, TCDD Inducible Poly(ADP-Ribose) Polymerase, TM4SF18, Transmembrane 4 L Six Family Member 18, TMC8, Transmembrane Channel Like 6, TMEM229B, Transmembrane Protein 229B, TMTC1, Transmembrane O-Mannosyltransferase Targeting Cadherins 1, TNFSF10, TNF Superfamily Member 10, TRHDE, Thyrotropin Releasing Hormone Degrading Enzyme, TRIM38, Tripartite Motif Containing 38, TSHZ1, Teashirt Zinc Finger Homeobox 1, USP18, Ubiquitin Specific Peptidase 18/ISG15-Specific-Processing Protease, UTR, untranslated region, UTS2R, Urotensin 2 Receptor, vd-sRNA, virus-derived small RNA, WNV, West Nile virus, XAF1, XIAP Associated Factor 1, XRN1, 5&#x02032;-3&#x02032; Exoribonuclease 1, ZIKV, Zika virus, ZMAT3, Zinc Finger Matrin-Type 3, ZMYM5, Zinc Finger MYM-Type Containing 5, ZNF124, Zinc Finger Protein 124, ZNF730, Zinc Finger Protein 730</span></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Alternative splicing, Astrocytes, miRNA, Neurons, Response to infection, Neuropathogenesis, Tick-borne encephalitis virus, Transcriptomics</span></div></div><div id="ab010" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="ab010title">Abstract</h2><!--article-meta--><div><p class="p p-first-last">Tick-borne encephalitis virus (TBEV), the most medically relevant tick-transmitted flavivirus in Eurasia, targets the host central nervous system and frequently causes severe encephalitis. The severity of TBEV-induced neuropathogenesis is highly cell-type specific and the exact mechanism responsible for such differences has not been fully described yet. Thus, we performed a comprehensive analysis of alterations in host poly-(A)/miRNA/lncRNA expression upon TBEV infection <em>in vitro</em> in human primary neurons (high cytopathic effect) and astrocytes (low cytopathic effect). Infection with severe but not mild TBEV strain resulted in a high neuronal death rate. In comparison, infection with either of TBEV strains in human astrocytes did not. Differential expression and splicing analyses with an <em>in silico</em> prediction of miRNA/mRNA/lncRNA/vd-sRNA networks found significant changes in inflammatory and immune response pathways, nervous system development and regulation of mitosis in TBEV Hypr-infected neurons. Candidate mechanisms responsible for the aforementioned phenomena include specific regulation of host mRNA levels via differentially expressed miRNAs/lncRNAs or vd-sRNAs mimicking endogenous miRNAs and virus-driven modulation of host pre-mRNA splicing. We suggest that these factors are responsible for the observed differences in the virulence manifestation of both TBEV strains in different cell lines. This work brings the first complex overview of alterations in the transcriptome of human astrocytes and neurons during the infection by two TBEV strains of different virulence. The resulting data could serve as a starting point for further studies dealing with the mechanism of TBEV-host interactions and the related processes of TBEV pathogenesis.</p></div></div><div id="s0005" class="tsec sec"><h2 class="head no_bottom_margin" id="s0005title">1. Introduction</h2><p id="p0005" class="p p-first">Tick-borne encephalitis virus (TBEV; genus <em>Flavivirus</em>, family <em>Flaviviridae</em>) is the most medically important tick-transmitted virus in Eurasia, affecting the lives of 10 000&#x02013;12 000 diagnosed patients annually <a href="#b0005" rid="b0005" class=" bibr popnode">[1]</a>. The virus is also gaining attention because of its recent emergence in new localities such as the Netherlands <a href="#b0010" rid="b0010" class=" bibr popnode">[2]</a>. TBEV, like other flaviviruses, forms spherical virions (50&#x000a0;nm in diameter), and its monopartite genome comprises a&#x000a0;&#x0223c;&#x000a0;11 kilobases-long single-stranded RNA of positive polarity. The genomic RNA contains one ORF coding for 10 proteins, which is flanked by 5&#x02032; and 3&#x02032; untranslated regions (UTRs) <a href="#b0015" rid="b0015" class=" bibr popnode">[3]</a>.</p><p id="p0010">Infection dissemination into the central nervous system (CNS) is a final stage in the process of TBEV pathogenesis, where neurons from different brain areas are the predominantly infected cell type and show a high level of death rate <a href="#b0020" rid="b0020" class=" bibr popnode">[4]</a>, <a href="#b0025" rid="b0025" class=" bibr popnode">[5]</a>. TBEV also successfully replicates in astrocytes <a href="#b0030" rid="b0030" class=" bibr popnode">[6]</a>, <a href="#b0035" rid="b0035" class=" bibr popnode">[7]</a>, the neuroglial cells that provide all the necessary support for the proper neuronal function. Such as they maintain homeostasis, perform energy metabolism, regulate blood flow, support the synaptic function, and protect neurons from the infection <a href="#b0040" rid="b0040" class=" bibr popnode">[8]</a>. Unlike in neurons, viability of TBEV-infected astrocytes is not negatively affected, even though similarly high viral titres in both cell types have been described <a href="#b0030" rid="b0030" class=" bibr popnode">[6]</a>, <a href="#b0035" rid="b0035" class=" bibr popnode">[7]</a>. An identical pattern of distinctive pathogenicity in neurons and astrocytes was observed in the case of West Nile virus (WNV) <a href="#b0045" rid="b0045" class=" bibr popnode">[9]</a>, <a href="#b0050" rid="b0050" class=" bibr popnode">[10]</a>, but not in Zika virus (ZIKV) <a href="#b0055" rid="b0055" class=" bibr popnode">[11]</a>.</p><p id="p0015">The exact factors responsible for the contrasting outcome of TBEV infection in neurons and astrocytes have not been identified yet; however, it is believed that cell type-specific response on the level of gene expression is one of the key factors involved. Indeed, a recent study by Fares <em>et al</em>. have described the neuron/astrocyte-specific response via a distinctive expression profile of specific innate immune response genes <a href="#b0060" rid="b0060" class=" bibr popnode">[12]</a>. The phenomenon of neuron/astrocyte-specific immune response to TBEV was confirmed also on the level of cytokine and chemokine production <a href="#b0065" rid="b0065" class=" bibr popnode">[13]</a>. Both studies point towards the TBEV-induced dysregulation of host gene expression with an outcome strongly dependent on the cell type-specific background.</p><p id="p0020">The regulatory network of non-coding RNAs (ncRNAs) represents one of the most important players in the cell type-specific changes of gene expression upon infection. Among these, expression profiles of specific microRNAs (miRNAs) and long non-coding RNAs (lncRNAs) were shown to be affected by flaviviral infection. miRNAs are a class of short (21&#x02013;25 nucleotides) ncRNAs regulating gene expression at the post-transcriptional level by sequence-specific binding to mRNA <a href="#b0070" rid="b0070" class=" bibr popnode">[14]</a>. Several flaviviruses, such as Japanese encephalitis virus (JEV), WNV, and ZIKV, were shown to dysregulate the expression profile of host miRNAs <a href="#b0075" rid="b0075" class=" bibr popnode">[15]</a>, <a href="#b0080" rid="b0080" class=" bibr popnode">[16]</a>, <a href="#b0085" rid="b0085" class=" bibr popnode">[17]</a>, <a href="#b0090" rid="b0090" class=" bibr popnode">[18]</a>, <a href="#b0095" rid="b0095" class=" bibr popnode">[19]</a>, of which some were identified to act in a proviral <a href="#b0100" rid="b0100" class=" bibr popnode">[20]</a> or antiviral manner <a href="#b0080" rid="b0080" class=" bibr popnode">[16]</a>, <a href="#b0105" rid="b0105" class=" bibr popnode">[21]</a>, <a href="#b0110" rid="b0110" class=" bibr popnode">[22]</a>. No miRNA-related data are currently available for TBEV <a href="#b0115" rid="b0115" class=" bibr popnode">[23]</a>. lncRNAs represent another group of ncRNAs that are longer than 200 nucleotides and also contribute to gene expression regulation. In comparison to miRNAs, lncRNAs modulate the gene expression at numerous levels, including chromatin remodelling, cis-/<em>trans</em>-regulation of gene transcription, mRNA splicing, and translation <a href="#b0120" rid="b0120" class=" bibr popnode">[24]</a>. Similarly to miRNAs, the expression pattern of lncRNAs alters upon flaviviral infection <a href="#b0125" rid="b0125" class=" bibr popnode">[25]</a>, <a href="#b0130" rid="b0130" class=" bibr popnode">[26]</a>, <a href="#b0135" rid="b0135" class=" bibr popnode">[27]</a>, <a href="#b0140" rid="b0140" class=" bibr popnode">[28]</a>, <a href="#b0145" rid="b0145" class=" bibr popnode">[29]</a> and particular lncRNAs were identified as proviral <a href="#b0150" rid="b0150" class=" bibr popnode">[30]</a>, <a href="#b0155" rid="b0155" class=" bibr popnode">[31]</a>, <a href="#b0160" rid="b0160" class=" bibr popnode">[32]</a> or antiviral <a href="#b0165" rid="b0165" class=" bibr popnode">[33]</a> factors. Except for the host-derived ncRNAs, flavivirus-derived small RNAs (vd-sRNAs; 13&#x02013;36 nt), were identified in mosquito/tick and mammalian cells <a href="#b0170" rid="b0170" class=" bibr popnode">[34]</a>, <a href="#b0175" rid="b0175" class=" bibr popnode">[35]</a>. In arthropod vectors, the generation of vd-sRNAs results from the RNA interference process (RNAi), where vd-sRNAs are used by the host cell to target and cleave viral genomic RNA <a href="#b0180" rid="b0180" class=" bibr popnode">[36]</a>. However, in the mammalian host, vd-sRNAs may play a different role by mimicking endogenous miRNA and target specific host mRNAs, thus substantially contribute to viral replication and pathogenesis as in the case of influenza A virus and human immunodeficiency virus <a href="#b0185" rid="b0185" class=" bibr popnode">[37]</a>, <a href="#b0190" rid="b0190" class=" bibr popnode">[38]</a>.</p><p id="p0025" class="p p-last">Despite numerous studies describing TBEV-induced changes in the expression profile of selected genes, no study so far has described a comprehensive transcriptomic analysis of both, ncRNAs and mRNAs in the most affected tissues of CNS. Data from such analysis would extensively broaden our understanding of TBEV-induced pathogenesis mechanism and may identify new target pathways for antiviral drug design. To fill that gap, we performed an integrative analysis describing the changes of host transcriptome on mRNA, miRNA, and lncRNA levels, including the pre-mRNA splicing evaluation, in combination with TBEV-derived vd-sRNA profiling. An <em>in vitro</em> model of human primary neurons and astrocytes infected by two TBEV strains of different virulence was utilised to elucidate the key players involved in TBEV-induced neuronal pathogenesis.</p></div><div id="s0010" class="tsec sec"><h2 class="head no_bottom_margin" id="s0010title">2. Methods</h2><div id="s0015" class="sec sec-first"><h3 id="s0015title">2.1. Primary cells cultivation and differentiation</h3><p id="p0030" class="p p-first">Neural progenitor cells of human origin &#x02013; Human Neural Stem Cells (hNSCs) purchased from Alstem (#hNSC11, Richmond, USA) were maintained in KnockOut DMEM/F-12 culture medium (#1260012, Gibco), supplemented with FGFb 20&#x000a0;ng/ml (#PHG0021, Thermo Fisher Scientific), EGF 20&#x000a0;ng/ml (#PHG0314, Thermo Fisher Scientific), 2 % StemPro Neural Supplement (#A10508-1, Thermo Fisher Scientific) and 1 % GlutaMAX-1 (2&#x000a0;mM) (#35050061, Gibco, Thermo Fisher Scientific) and grown in a 6-well plate at the basement membrane of Geltrex&#x02122; solution (1 % Geltrex&#x02122;; #A1413302, Thermo Fisher Scientific; in KnockOut DMEM/F-12 culture medium). hNSCs were split after PBS wash by treatment with StemPro Acutase (#A11105-1, Gibco, Thermo Fisher Scientific) when reaching the 80&#x02013;90 % confluency in a 1:4&#x02013;1:6 ratio. Cells were cultivated in 5 % CO<sub>2</sub> humidified atmosphere at 37&#x000a0;&#x000b0;C.</p><p id="p0035" class="p p-last">hNSCs were seeded at desired density according to the parameters of the specific experiments. Details are summarized in <a href="#s0170" rid="s0170" class=" sec">Table S1</a>. After 24&#x000a0;h, differentiation was initiated by a transition to either Astrocyte Medium (#1801; ScienCell Research Labs, Carlsbad, CA, USA) or Neurobasal Plus Medium (#A35829-01, Gibco, Thermo Fisher Scientific), supplemented with 2 % B-27<sup>TM</sup> supplement (#17504044, Gibco, Thermo Fisher Scientific) and 2&#x000a0;mM GlutaMAX-1 (#35050061, Gibco, Thermo Fisher Scientific) to derive astrocytes or neurons respectively. During the differentiation process, the media were changed twice a week and the differentiation of astrocytes took 21&#x02013;22&#x000a0;days and neurons 14&#x02013;18&#x000a0;days before the target experiment was undertaken. For seeding of differentiated or partially differentiated cells, astrocytes or neurons were washed with PBS and detached by CTS&#x02122; TrypLE&#x02122; Select Enzyme (#A1285901, Thermo Fisher Scientific).</p></div><div id="s0020" class="sec"><h3 id="s0020title">2.2. Viruses and infection</h3><p id="p0040" class="p p-first-last">We used two strains of the European TBEV subtype with differing pathogenicity for cell infections. The prototype TBEV strain Neudoerfl originating from infected tick (4th passage in suckling mice brains; GenBank accession no. <a id="__tag_1132452540" class="tag_hotlink" href="/nuccore/U27495" ref="/nuccore/U27495"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"U27495","term_id":"975237","term_text":"U27495"}}</span>U27495</a>), was provided by Prof. F.X. Heinz (Medical University of Vienna, Austria) <a href="#b0195" rid="b0195" class=" bibr popnode">[39]</a>. The highly virulent TBEV strain Hypr (4th passage in suckling mice brains; GenBank accession no. <a id="__tag_1132452543" class="tag_hotlink" href="/nuccore/U39292" ref="/nuccore/U39292"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"U39292","term_id":"1066074","term_text":"U39292"}}</span>U39292</a>), isolated from a 5-year old child with a multi-tick bite history and suspect tick-borne encephalitis <a href="#b0200" rid="b0200" class=" bibr popnode">[40]</a>, is available at the Institute of Parasitology, Biology Centre of Czech Academy of Sciences, &#x0010c;esk&#x000e9; Bud&#x0011b;jovice, Czech Republic. Mock-inoculated brain suspensions of suckling mice brains were used as a control in all experiments. Differentiated astrocytes and neurons were infected with both TBEV strains at the multiplicity of infection of 5 (MOI) for two hours in half volume of cultivation media to favour virus adsorption. Then, the inoculum was removed, and fresh cultivation media was replenished to the normal volume. To determine the appropriate MOI for infection, parallelly differentiated cells were counted prior to the infection in order to determine the cell numbers after differentiation.</p></div><div id="s0025" class="sec"><h3 id="s0025title">2.3. Immunofluorescence assay</h3><p id="p0045" class="p p-first-last">The presence of characteristic markers denoting the state of differentiation and development of infection in astrocytes and neurons was analysed by immunofluorescence assay. Cells were seeded on coated chamber slides at concentrations detailed in <a href="#s0170" rid="s0170" class=" sec">Table S1</a> and, when applicable, infected or mock-infected. At 24/72&#x000a0;h post-infection (p.i.), the presence of differentiation and infection markers was assayed. During sample processing, cells were fixed with 4 % paraformaldehyde for 15&#x000a0;min. Cells were washed in PBS and permeabilized by 0.1 % Triton X-100 in PBS, and formaldehyde auto-fluorescence was quenched by 50&#x000a0;mM NH<sub>4</sub>Cl in 1 % bovine serum albumin (BSA) in PBS for 10&#x000a0;min twice. After PBS washing, blocking in 3 % BSA in PBS was undertaken for 1&#x000a0;h at room temperature. Target antigens were labelled for 1&#x000a0;h at room temperature or overnight at 4&#x000a0;&#x000b0;C by the following primary and secondary antibodies: dsRNA mAb (#10010200, SCICONS J2 from Scicons, Biocompare), GFAP (GA5) Mouse mAb (#3670S, Cell Signaling Technology), HuC/HuD Monoclonal Antibody (16A11) (#A-21271 Invitrogen, Thermo), MAP2 (#4542S, Cell Signaling Technology), S100B Polyclonal antibody (#bs-2015R Bioss Antibodies), TBEV NS3 Langat Chicken IgY (NS3 antibodies to closely related Langat virus NS3 were kindly provided by Dr M. Bloom, 94 % homology with TBEV), Goat Chicken IgY H&#x00026;L (DyLight&#x000ae; 488) (#ab96947, Abcam), Goat Guinea pig IgG H&#x00026;L (Alexa Fluor&#x000ae; 594) (#ab150188, Abcam), Goat Mouse IgG H&#x00026;L (DyLight&#x000ae; 594) preadsorbed (#ab96881 Abcam), Goat Rabbit IgG H&#x00026;L (Alexa Fluor&#x000ae; 488) (#ab181448 Abcam). Concurrent nuclei staining and sample mounting were done with VECTASHIELD&#x000ae; Antifade Mounting Medium with DAPI (#H-1200-10, Vector Laboratories). Images were taken with the Olympus Fluoview FV10i confocal microscope equipped with FV10-ASW software (v.1.7).</p></div><div id="s0030" class="sec"><h3 id="s0030title">2.4. Growth curve</h3><p id="p0050" class="p p-first-last">To assess the replication rate of TBEV in astrocytes and neurons, cells were either mock-treated or infected with the TBEV strains Hypr and Neudoerfl at the MOI of 5 (see details of differentiation in <a href="#s0170" rid="s0170" class=" sec">Table S1</a>). At 2, 12, 24, 48, 72 for both cell lines and at 120&#x000a0;h p.i. (astrocytes only) culture supernatant was sampled and clarified by centrifugation for TBEV titre assessment. TBEV titres were plaque assayed on the human lung adenocarcinoma monolayers (A549; kindly gifted by R. Randall, University of St. Andrews, UK) according to the modified protocol of de Madrid <em>et al</em>. <a href="#b0205" rid="b0205" class=" bibr popnode">[41]</a>. A549 were maintained in low glucose DMEM cultivation medium (#L0064-500, Biowest, VWR) supplemented with 10 % foetal bovine serum (FBS; #S1810-500, Biowest, VWR), 1 % antibiotics (penicillin G 100 units/ml, streptomycin 100 &#x003bc;g/ml; #L0022-020, Biowest), and 1 % L-alanyl-L-glutamine (#X0551-100, VWR) in 5 % CO<sub>2</sub> humidified atmosphere at 37&#x000a0;&#x000b0;C. Briefly, ten-fold dilutions of virus samples were mixed with the A549 cell suspension (1.5&#x000d7;10<sup>5</sup> cells/well of 24-well plate), cells were let to adhere, and virus to adsorb. After 4&#x000a0;h, an overlay mixture (1:1 v/v of carboxymethyl cellulose and 2&#x000d7;&#x000a0;concentrated DMEM cultivation medium) was applied dropwise to the cells. After 5&#x000a0;days, cell monolayers were washed with physiologic solution (0.9 % NaCl), and plaques were fixed and stained with naphthalene black solution (0.1 % naphthalene black in 6 % acetic acid solution) for 45&#x000a0;min.</p></div><div id="s0035" class="sec"><h3 id="s0035title">2.5. Viability assay</h3><p id="p0055" class="p p-first-last">To detect the viability rates, neurons and astrocytes were differentiated as specified in <a href="#s0170" rid="s0170" class=" sec">Table S1</a> and infected with 5 MOI of Hypr and Neudoerfl TBEV strain. 2&#x02013;3&#x000a0;h prior to the sampling interval (12, 24, 48, 72, 120, 168&#x000a0;h p.i.) incubation with AlamarBlue reagent in the cultivation media (1:10 (v/v); #DAL1025, Thermo Fisher Scientific) was performed in a dark. Viability for each cell type was assessed by fluorescence measurement (&#x003bb;<sub>Ex</sub>&#x000a0;=&#x000a0;550&#x000a0;nm, &#x003bb;<sub>Em</sub>&#x000a0;=&#x000a0;590&#x000a0;nm) using Tecan infinite 2000Pro, (Tecan i-control, 1.11.1.0) from four biological and two technical replicates and the viability value of the mock-treated cells was set at 1 (100%).</p></div><div id="s0040" class="sec"><h3 id="s0040title">2.6. qRT-PCR</h3><p id="p0060" class="p p-first">For the analysis of TBEV replication in infected cells, viral RNA was quantified by an assay designed by Achazi et al. <a href="#b0210" rid="b0210" class=" bibr popnode">[42]</a>. Total RNA (80&#x000a0;ng/reaction) was used for TBEV gRNA quantification with the KAPA PROBE FAST Universal One-Step qRT-PCR Master Mix (2X) (#KK4752, Sigma-Aldrich, MERCK) and relative fold induction of TBEV RNA amount was determined using the delta-delta ct (&#x00394;&#x00394;-ct) with the comparison to mock. For the verification of poly-(A) transcriptomic data, samples were treated with dsDNase (#EN0771, Thermo Fisher Scientific) and gene expression was analysed with the KAPA SYBR FAST Universal One-Step qRT-PCR Kit (#KK4652, Sigma-Aldrich) according to the manufacturer&#x02019;s protocol. Relative expressions of HSPA-6, OASL, RNVU1, and RSAD2 (viperin) genes were processed via the &#x00394;&#x00394;-ct method with HPRT1 as a reference gene. All samples were analysed in biological and technical triplicates. A list of primers and probes used can be found in <a href="#s0170" rid="s0170" class=" sec">Table S1</a>.</p><p id="p0065" class="p p-last">Verification of small RNA transcriptomic data was performed using miRCURY LNA miRNA PCR Assay Kit (#339306, Qiagen) according to the manufacturer&#x02019;s instructions. 200&#x000a0;ng of total RNA was used as an input with subsequent quantification of <em>hsa-miR-1248</em> (#YP00204253) and <em>hsa-miR-145-5p</em> (#YP00204483) in all samples. Data obtained were processed via relative quantification using the delta-delta c<sub>t</sub> (&#x00394;&#x00394;-c<sub>t</sub>) method with <em>hsa-miR-103a</em> as a reference miRNA and Sp6 as an internal spike-in control. All samples were analysed in biological and technical triplicates.</p></div><div id="s0045" class="sec"><h3 id="s0045title">2.7. Transcriptomic analysis</h3><div id="s0050" class="sec sec-first"><p></p><h4 id="s0050title" class="inline">2.7.1.  Sample preparation </h4><p id="p0070" class="p p-first-last">Each sample was prepared and analysed in biological triplicates. Total RNA from TBEV-infected human neurons and astrocytes was isolated at 24 and 72&#x000a0;h p.i. using RNA Blue reagent (Top-Bio, Vestec, Czech Republic) according to the manufacturer&#x02019;s instructions. RNA concentration was measured using NanoPhotometer Pearl (Implen, M&#x000fc;nchen, Germany) and the quality of RNA was determined using 2100 Bioanalyzer with RNA 6000 Nano kit (Agilent Technologies, Santa Clara, CA, USA).</p></div><div id="s0055" class="sec"><p></p><h4 id="s0055title" class="inline">2.7.2.  Poly-(A)-enriched RNAs </h4><p id="p0075" class="p p-first-last">Sequencing library construction, sequencing, raw read quality check and adapter trimming were done by Novogene Co., Ltd (Beijing, China). The 150 PE library was sequenced in HiSeq 4000 sequencer (Illumina, San Diego, CA, USA). Quality filtering was performed using Cutadapt v1.15 <a href="#b0215" rid="b0215" class=" bibr popnode">[43]</a> and the subsequent clean read quality check was done using FastQC v0.11.5. <a href="#b0220" rid="b0220" class=" bibr popnode">[44]</a>. The mapping (GRCh38.p13 reference genome), assembly and differential expression analysis were performed using the Tuxedo suite <a href="#b0225" rid="b0225" class=" bibr popnode">[45]</a>.</p></div><div id="s0060" class="sec"><p></p><h4 id="s0060title" class="inline">2.7.3.  Small RNAs </h4><p id="p0080" class="p p-first-last">Sequencing library construction, sequencing, raw read quality check and adapter trimming were done by Novogene Co., Ltd (Beijing, China). The 50 SE library was sequenced in HiSeq 2500 sequencer (Illumina, San Diego, CA, USA). The clean read quality check was done using FastQC v0.11.5 <a href="#b0220" rid="b0220" class=" bibr popnode">[44]</a>. Adapter contamination, short and low quality reads were removed using Cutadapt v1.15 <a href="#b0215" rid="b0215" class=" bibr popnode">[43]</a>. The clean reads were mapped to the reference genome (GRCh38.p13) using miRDeep 2.0.1.2 <a href="#b0230" rid="b0230" class=" bibr popnode">[46]</a>. The -q option for the mapper.pl script was used to allow 1 mismatch for mapping. The miRDeep2.pl module was run using the human miRBase database (miRBase v.21) <a href="#b0235" rid="b0235" class=" bibr popnode">[47]</a>. Identification of differentially expressed (DE) miRNAs was based on Benjamini-Hochberg P-value&#x000a0;&#x0003c;&#x000a0;0.05 (unpaired Student&#x02019;s <em>t</em>-test) and log<sub>2</sub> fold change&#x000a0;&#x0003e;1.5 or&#x000a0;&#x0003c;&#x02212;1.5 using the normalized fragments per kilobase of transcript per million mapped reads (FPKMs; cut-off 10 FPKM per miRNA species) ratios between the respective TBEV-infected sample and mock control.</p></div><div id="s0065" class="sec sec-last"><p></p><h4 id="s0065title" class="inline">2.7.4.  Virus-derived small RNAs </h4><p id="p0085" class="p p-first-last">The raw reads from small RNA library were also mapped to the genome of both TBEV strains, Hypr (GenBank accession number <a id="__tag_1132452545" class="tag_hotlink" href="/nuccore/U39292" ref="/nuccore/U39292"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"U39292","term_id":"1066074","term_text":"U39292"}}</span>U39292</a>) and Neudoerfl (GenBank accession number <a id="__tag_1132452542" class="tag_hotlink" href="/nuccore/U27495" ref="/nuccore/U27495"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"U27495","term_id":"975237","term_text":"U27495"}}</span>U27495</a>), using Bowtie <a href="#b0240" rid="b0240" class=" bibr popnode">[48]</a>. The read depth counts of each sample were retrieved from the corresponding bedgraphs produced using Bedtools v.2.27.1 <a href="#b0245" rid="b0245" class=" bibr popnode">[49]</a>. Read distribution along the TBEV genome was evaluated and visualised using Geneious Prime v2020.0.5. (Biomatters, Ltd, Auckland, New Zealand). The sense and antisense reads were additionally discriminated by extracting individual read types from their respective bam files using Samtools v1.10 <a href="#b0250" rid="b0250" class=" bibr popnode">[50]</a> and their counts were compared for each sample.</p></div></div><div id="s0070" class="sec"><h3 id="s0070title">2.8. <em>In silico</em> miRNA/vd-sRNA target prediction</h3><p id="p0090" class="p p-first-last">miRNA <em>trans</em> target prediction was performed using miRWalk <a href="#b0255" rid="b0255" class=" bibr popnode">[51]</a> and LncBase v.2 <a href="#b0260" rid="b0260" class=" bibr popnode">[52]</a> toolkits. Briefly, DE miRNAs were divided into two groups: (1) up-regulated and (2) down-regulated; these served as an input for the respective prediction toolkit. Generated lists of predicted miRNA targets were subsequently used to identify these targets in DE protein-coding mRNAs (pc-mRNAs)/lncRNA datasets. For up-regulated miRNAs, the datasets of down-regulated pc-mRNAs/lncRNAs were used, and <em>vice versa</em>, datasets of up-regulated pc-mRNAs/lncRNAs were used in the case of down-regulated miRNAs. Default settings were used for miRWalk (P-value&#x000a0;&#x0003c;&#x000a0;0.05, miRNAs targeting 3&#x02032;UTR) in combination with the miRDB prediction tool. In the case of LncBase, the prediction module with default settings was used (cut-off score&#x000a0;&#x0003e;&#x000a0;0.90). Identification of human miRNAs targeting TBEV Hypr/Neudoerfl genomic RNA or human mRNAs being targeted by 21&#x02013;23 nt long vd-sRNAs was performed using the miRDB tool with a cut-off score&#x000a0;&#x0003e;&#x000a0;0.70 <a href="#b0265" rid="b0265" class=" bibr popnode">[53]</a>.</p></div><div id="s0075" class="sec"><h3 id="s0075title">2.9. Differential splicing analysis</h3><p id="p0095" class="p p-first">The identification and quantification of differential splicing were computed using MAJIQ v2.2 <a href="#b0270" rid="b0270" class=" bibr popnode">[54]</a>, which employs Local splicing variations (LSV) derived from the provided transcriptome annotation file and RNA-seq data. The differential splicing was considered significant for LSVs with MAJIQ default cut-off values for deltaPSI (|&#x00394;&#x003a8;|&#x02265;0.2) supported by P(|&#x00394;&#x003a8;]|&#x02265;0.2) with a 0.05 cut-off (Bonferroni correction).</p><p id="p0100" class="p p-last">The raw reads of each sample represented by biological triplicates were mapped to the reference genome (GRCh38.p13) with STAR 2.7.7a <a href="#b0275" rid="b0275" class=" bibr popnode">[55]</a>. The resulting bam files and an annotation file of the respective human genome were supplied to a MAJIQ builder for splice graph construction using default parameters. The relative abundance of LSVs was calculated using MAJIQ Quantifier with default parameters. The quantifier module computes the abundance of each LSV using marginal percent spliced index (PSI, denoted &#x003a8;). The PSI is calculated for each splice junction (SJ) and expresses the probability of splicing compared to other SJs in a given splicing event. The differential splicing is inferred from relative changes of LSVs among different conditions using delta PSI (dPSI, denoted &#x00394;&#x003a8;). The summary and visualization of differential splicing were done for each sample with the MAJIQ Voila package.</p></div><div id="s0080" class="sec"><h3 id="s0080title">2.10. Immunoblotting</h3><p id="p0105" class="p p-first-last">Isolation of proteins was performed from the samples used for RNA isolation using RNA Blue according to the manufacturer&#x02019;s instructions. The resulting protein isolates were separated on 12 % polyacrylamide gels and subsequent immunoblotting detection was performed as described previously <a href="#b0280" rid="b0280" class=" bibr popnode">[56]</a>. The following antibodies were used: guinea-pig polyclonal serum against TBEV capsid protein (C) (produced in-house), anti-GAPDH antibody [<a id="__tag_1132452537" class="tag_hotlink" href="/protein/EPR16891" ref="/protein/EPR16891"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"EPR16891","term_id":"523382963","term_text":"EPR16891"}}</span>EPR16891</a>] (Abcam; #ab181602), HRP goat anti-guinea pig (Novex; #A18769), HRP goat anti-rabbit IgG antibody (Vector Laboratories; #PI-1000). Sample inputs were standardised by equal protein amounts loaded into each well (10&#x000a0;&#x000b5;g).</p></div><div id="s0085" class="sec sec-last"><h3 id="s0085title">2.11. Statistical analyses</h3><p id="p0110" class="p p-first-last">If not mentioned otherwise default settings for built-in statistics were used in employed computational packages/tools. Statistical data analysis in the case of TBEV infection dynamics (<a href="/pmc/articles/PMC9167876/figure/f0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0005" rid-ob="ob-f0005" co-legend-rid="lgnd_f0005"><span>Fig. 1</span></a>) was performed in the GraphPad Prism software (version 8.3.0, GraphPad Software, San Diego, CA, USA): cell viability was analysed by Welc<math><mover accent="true"><mi>&#x068;</mi><mo>&#x0301;</mo></mover></math>s <em>t</em>-test with multiple testing correction by FDR (Benjamini, Krieger, and Yekutieli, q&#x000a0;&#x0003c;&#x000a0;0.05) and TBEV titre differences were analysed by unpaired Student&#x02019;s <em>t</em>-test (&#x003b1;&#x000a0;=&#x000a0;0.05). Conformity of sequencing and qRT-PCR data was analysed with the Spearman correlation analysis (&#x003b1;&#x000a0;=&#x000a0;0.05).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f0005" co-legend-rid="lgnd_f0005"><a href="/pmc/articles/PMC9167876/figure/f0005/" target="figure" rid-figpopup="f0005" rid-ob="ob-f0005"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139861394153344"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9167876_gr1.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gr1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC9167876/bin/gr1.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139861394153344"><a target="object" rel="noopener" href="/pmc/articles/PMC9167876/figure/f0005/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_f0005"><div><a class="figpopup" href="/pmc/articles/PMC9167876/figure/f0005/" target="figure" rid-figpopup="f0005" rid-ob="ob-f0005">Fig. 1</a></div><!--caption a7--><div class="caption"><p><strong>TBEV infection dynamics assessment in <em>in vitro</em> differentiated human neurons and astrocytes reveals higher neuronal susceptibility.</strong> Astrocytes were differentiated for 22&#x000a0;days, neurons for 14&#x02013;17&#x000a0;days prior to infection with TBEV strains Hypr and Neudoerfl at MOI of 5. <strong>(A)</strong> Immunofluorescent labelling of selected differentiation markers in astrocytes (GFAP &#x02013; red and S100B &#x02013; green) &#x02013; left and neurons (HuD &#x02013; red and MAP2 -green) &#x02013; right. Nuclei were stained with DAPI. Controls are labelled with secondary antibodies only. The scale bar represents 20&#x000a0;&#x000b5;m. Representative images from two independent experiments are shown. <strong>(B)</strong> TBEV growth curve of strains Hypr and Neudoerfl, virus shedding into cultivation media was quantified by plaque assay using A549 cells. Data summarise three independent experiments and values in graphs are expressed as mean&#x000a0;&#x000b1;&#x000a0;SD. Significant difference from control was calculated by unpaired Student&#x02019;s <em>t</em>-test (p&#x000a0;<em>&#x0003c;</em>&#x000a0;0.05 (*), p&#x000a0;<em>&#x0003c;</em>&#x000a0;0.01(**)). <strong>(C)</strong> Relative viability of neurons and astrocytes upon TBEV infection measured by metabolic conversion of alamarBlue and related to the viability of mock-treated cells (1.0&#x000a0;=&#x000a0;100 %, dotted line). Data summarise four biological and two technical replicates experiments, and values in graphs are expressed as mean&#x000a0;&#x000b1;&#x000a0;SD. Welc<math><mover accent="true"><mi>&#x068;</mi><mo>&#x0301;</mo></mover></math>s <em>t</em>-test showed a significant difference from control with the multiple testing correction by FDR (Benjamini, Krieger, and Yekutieli, q&#x000a0;&#x02264;&#x000a0;0.05), p&#x000a0;&#x02264;&#x000a0;0.05 (*), p&#x000a0;&#x02264;&#x000a0;0.01 (**), p&#x000a0;&#x02264;&#x000a0;0.001 (***). <strong>(D)</strong> TBEV replication in astrocytes and neurons was assessed by the amount of genomic RNA quantified by qRT-PCR and related to mock with the &#x00394;&#x00394;-ct method. Results are represented as means&#x000a0;&#x000b1;&#x000a0;SD of three biological and technical replicates. <strong>(E)</strong> TBEV C (capsid protein) and GAPDH (loading control) levels in infected astrocytes and neurons at 24 and 72&#x000a0;h p.i. were determined by immunoblotting. Representative data from three independent experiments are shown. N &#x02013; Neudoerfl strain, H &#x02013; Hypr strain. (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this article.)</p></div></div></div></div></div><div id="s0090" class="tsec sec"><h2 class="head no_bottom_margin" id="s0090title">3. Results</h2><div id="s0095" class="sec sec-first"><h3 id="s0095title">3.1. Infection of <em>in vitro</em> differentiated human neurons and astrocytes by TBEV Hypr or Neudoerfl resulted in a different pathogenic pattern</h3><p id="p0115" class="p p-first">Initially, we focused on deriving a suitable experimental model to study TBEV infection in the target cells, human neurons and astrocytes. Primary human neurons and astrocytes were derived from neural progenitor cells of human origin &#x02013; Human Neural Stem Cells (hNSCs). Following the differentiation, the characteristic appearance of astrocyte and neuron morphology was observed and further, the expression of cell-type-specific markers was verified using an immunofluorescence assay. Neuron identity was verified by the presence of markers HUDD and MAP2, and astrocytes identity was assayed by the expression of S100B and GFAP (<a href="/pmc/articles/PMC9167876/figure/f0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0005" rid-ob="ob-f0005" co-legend-rid="lgnd_f0005"><span>Fig. 1</span></a>A). In each cell type, at least 90 % of cells were labelled positively for at least one of the differentiation markers (data not shown).</p><p id="p0120">Further, we intended to characterize the development of infection in neurons and astrocytes. Therefore, both cell types were infected with two European subtype strains of TBEV, the highly pathogenic Hypr and mild Neudoerfl, at the multiplicity of infection (MOI) of 5. The production of the virus in both cell lines was analysed with plaque assay on A549 cells (<a href="/pmc/articles/PMC9167876/figure/f0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0005" rid-ob="ob-f0005" co-legend-rid="lgnd_f0005"><span>Fig. 1</span></a>B). In addition to TBEV titres, the relative levels of TBEV genomic RNA (gRNA) were determined in the course of infection as well (<a href="/pmc/articles/PMC9167876/figure/f0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0005" rid-ob="ob-f0005" co-legend-rid="lgnd_f0005"><span>Fig. 1</span></a>D).</p><p id="p0125">Infection of both cell types with TBEV resulted in the production of relatively high viral titres. In neurons, peak production of the virus was reached at 48&#x000a0;h p.i. (12.96&#x000d7;10<sup>7</sup>&#x000a0;&#x000b1;&#x000a0;3.21&#x000d7;10<sup>7</sup> and 5.93&#x000d7;10<sup>7</sup>&#x000a0;&#x000b1;&#x000a0;1.16&#x000d7;10<sup>7</sup> for Hypr and Neudoerfl, respectively) and remained steady further on. In astrocytes, the peak production of the virus was reached at 48&#x000a0;h p.i. (1.53&#x000d7;10<sup>7</sup>&#x000a0;&#x000b1;&#x000a0;0.46&#x000d7;10<sup>7</sup> and 0.35&#x000d7;10<sup>7</sup>&#x000a0;&#x000b1;&#x000a0;0.09&#x000d7;10<sup>7</sup> for Hypr and Neudoerfl, respectively), however, a decline in the viral titres was observed hereafter (<a href="/pmc/articles/PMC9167876/figure/f0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0005" rid-ob="ob-f0005" co-legend-rid="lgnd_f0005"><span>Fig. 1</span></a>B).</p><p id="p0130">The viability of infected astrocytes was not significantly affected by TBEV in most of the time periods tested (12&#x02013;168&#x000a0;h p.i.) regardless of the strain of TBEV used. The only significant reduction in viability by 10 % was seen in astrocytes infected with Neudoerfl after 5&#x000a0;days p.i. Dissimilarly, TBEV Hypr reduced the viability of infected neurons by 40 % at 72&#x000a0;h p.i. and &#x0003c;20 % of cells survived in the later time points. Less virulent strain Neudoerfl affected cell viability the most at 120&#x000a0;h p.i. by killing approximately 30 % of the neurons (<a href="/pmc/articles/PMC9167876/figure/f0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0005" rid-ob="ob-f0005" co-legend-rid="lgnd_f0005"><span>Fig. 1</span></a>C).</p><p id="p0135">To illustrate the infection process entirely, the amount of viral genomic RNA was measured by qRT-PCR (<a href="/pmc/articles/PMC9167876/figure/f0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0005" rid-ob="ob-f0005" co-legend-rid="lgnd_f0005"><span>Fig. 1</span></a>D), and the amount of viral capsid protein reflecting the proteosynthesis was determined by immunoblotting at 24 and 72&#x000a0;h p.i. (<a href="/pmc/articles/PMC9167876/figure/f0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0005" rid-ob="ob-f0005" co-legend-rid="lgnd_f0005"><span>Fig. 1</span></a>E). Alike the progeny production, the amount of viral RNA in neurons exceeded the amount in astrocytes by &#x0003e;10-fold in both intervals examined. The more virulent TBEV strain Hypr exhibited a slightly higher replication rate than the Neudoerfl strain (<a href="/pmc/articles/PMC9167876/figure/f0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0005" rid-ob="ob-f0005" co-legend-rid="lgnd_f0005"><span>Fig. 1</span></a>D). A similar trend of more pronounced viral protein production in the case of TBEV Hypr infection was apparent also in the viral protein production (<a href="/pmc/articles/PMC9167876/figure/f0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0005" rid-ob="ob-f0005" co-legend-rid="lgnd_f0005"><span>Fig. 1</span></a>E).</p><p id="p0140"><em>In situ</em> labelling of viral proteins and dsRNA showed that TBEV replication was concentrated in discrete regions adjacent to nuclei, whereas TBEV non-structural protein NS3 exhibited a more diffuse, whole cytoplasmic pattern of distribution (<a href="#s0170" rid="s0170" class=" sec">Fig. S1</a>).</p><p id="p0145" class="p p-last">To sum up, both cell types were highly susceptible to TBEV infection, supporting virus replication and progeny production. However, neural cell types differed in resilience, with astrocytes able to withstand the infection. We wanted to examine further the factors that underlay the difference in response between neurons and astrocytes.</p></div><div id="s0100" class="sec"><h3 id="s0100title">3.2. Differential expression analysis of TBEV-infected human neurons and astrocytes reveals significant differences linked to the cell origin and virus virulence</h3><p id="p0150" class="p p-first">Higher susceptibility of neurons to TBEV infection accompanied by higher pathogenicity than in astrocytes is a previously described phenomenon <a href="#b0025" rid="b0025" class=" bibr popnode">[5]</a>, <a href="#b0035" rid="b0035" class=" bibr popnode">[7]</a>, <a href="#b0060" rid="b0060" class=" bibr popnode">[12]</a>. However, no comprehensive study combines the description of the host response on the mRNA, miRNA, and lncRNA levels. Thus, we performed an integrated differential expression analysis in both human neurons and astrocytes challenged by either TBEV Hypr or Neudoerfl infection. <a href="/pmc/articles/PMC9167876/figure/f0010/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0010" rid-ob="ob-f0010" co-legend-rid="lgnd_f0010"><span>Fig. 2</span></a> summarizes the experimental outline for RNA-seq sample preparation (<a href="/pmc/articles/PMC9167876/figure/f0010/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0010" rid-ob="ob-f0010" co-legend-rid="lgnd_f0010"><span>Fig. 2</span></a>A) and subsequent <em>in silico</em> analyses (<a href="/pmc/articles/PMC9167876/figure/f0010/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0010" rid-ob="ob-f0010" co-legend-rid="lgnd_f0010"><span>Fig. 2</span></a>B). Differentiated cells were infected at MOI of 5 and total RNA was isolated at 24 and 72&#x000a0;h p.i. to describe the transcription dynamics of the host response. In total, 36 samples were prepared; this number includes three biological replicates for each variant (<a href="/pmc/articles/PMC9167876/table/t0005/" target="table" class="fig-table-link figpopup" rid-figpopup="t0005" rid-ob="ob-t0005" co-legend-rid=""><span>Table 1</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f0010" co-legend-rid="lgnd_f0010"><a href="/pmc/articles/PMC9167876/figure/f0010/" target="figure" rid-figpopup="f0010" rid-ob="ob-f0010"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139861394117760"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9167876_gr2.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gr2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC9167876/bin/gr2.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139861394117760"><a target="object" rel="noopener" href="/pmc/articles/PMC9167876/figure/f0010/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_f0010"><div><a class="figpopup" href="/pmc/articles/PMC9167876/figure/f0010/" target="figure" rid-figpopup="f0010" rid-ob="ob-f0010">Fig. 2</a></div><!--caption a7--><div class="caption"><p><strong>Schematic representation of an experimental design performed in this study. (A)</strong> hNSCs were differentiated into astrocytes and neurons, infected with TBEV strains Hypr and Neudoerfl at the MOI of 5 and total RNA was isolated at 24 and 72&#x000a0;h p.i. with RNA Blue reagent. The quality of RNA was verified using Bioanalyzer 2100 and <strong>(B)</strong> Diagram describing the generation of miRNA/mRNA/lncRNA/vd-sRNA networks. (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this article.)</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="t0005"><h3>Table 1</h3><!--caption a7--><div class="caption"><p>An overview of samples subjected to RNA-seq analysis.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1">Cell type</th><th rowspan="1" colspan="1">TBEV strain</th><th rowspan="1" colspan="1">Hours p.i.</th><th rowspan="1" colspan="1">Number of samples</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">A24_H</td><td rowspan="1" colspan="1">astrocytes</td><td rowspan="1" colspan="1">Hypr</td><td rowspan="1" colspan="1">24</td><td rowspan="1" colspan="1">3</td></tr><tr><td rowspan="1" colspan="1">A24_N</td><td rowspan="1" colspan="1">astrocytes</td><td rowspan="1" colspan="1">Neudoerfl</td><td rowspan="1" colspan="1">24</td><td rowspan="1" colspan="1">3</td></tr><tr><td rowspan="1" colspan="1">A24_M</td><td rowspan="1" colspan="1">astrocytes</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">24</td><td rowspan="1" colspan="1">3</td></tr><tr><td rowspan="1" colspan="1">A72_H</td><td rowspan="1" colspan="1">astrocytes</td><td rowspan="1" colspan="1">Neudoerfl</td><td rowspan="1" colspan="1">72</td><td rowspan="1" colspan="1">3</td></tr><tr><td rowspan="1" colspan="1">A72_N</td><td rowspan="1" colspan="1">astrocytes</td><td rowspan="1" colspan="1">Hypr</td><td rowspan="1" colspan="1">72</td><td rowspan="1" colspan="1">3</td></tr><tr><td rowspan="1" colspan="1">A72_M</td><td rowspan="1" colspan="1">astrocytes</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">72</td><td rowspan="1" colspan="1">3</td></tr><tr><td rowspan="1" colspan="1">N24_H</td><td rowspan="1" colspan="1">neurons</td><td rowspan="1" colspan="1">Neudoerfl</td><td rowspan="1" colspan="1">24</td><td rowspan="1" colspan="1">3</td></tr><tr><td rowspan="1" colspan="1">N24_N</td><td rowspan="1" colspan="1">neurons</td><td rowspan="1" colspan="1">Hypr</td><td rowspan="1" colspan="1">24</td><td rowspan="1" colspan="1">3</td></tr><tr><td rowspan="1" colspan="1">N24_M</td><td rowspan="1" colspan="1">neurons</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">24</td><td rowspan="1" colspan="1">3</td></tr><tr><td rowspan="1" colspan="1">N72_H</td><td rowspan="1" colspan="1">neurons</td><td rowspan="1" colspan="1">Neudoerfl</td><td rowspan="1" colspan="1">72</td><td rowspan="1" colspan="1">3</td></tr><tr><td rowspan="1" colspan="1">N72_N</td><td rowspan="1" colspan="1">neurons</td><td rowspan="1" colspan="1">Hypr</td><td rowspan="1" colspan="1">72</td><td rowspan="1" colspan="1">3</td></tr><tr><td rowspan="1" colspan="1">N72_M</td><td rowspan="1" colspan="1">neurons</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">72</td><td rowspan="1" colspan="1">3</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139861394115376"><a target="object" rel="noopener" href="/pmc/articles/PMC9167876/table/t0005/?report=objectonly">Open in a separate window</a></div></div><div id="s0105" class="sec"><p></p><h4 id="s0105title" class="inline">3.2.1.  Differential expression analysis of poly-(A) RNAs </h4><p id="p0155" class="p p-first">Sequencing of poly-(A) selected library of all samples yielded on average in 80&#x000a0;M 150 PE reads. Quality filtering using Cutadapt dropped on average 1.8 % low-quality and/or short reads. The remaining reads were mapped to the human genome assembly (GRCh38.p13) using TopHat with 78.1 % average mapping efficiency <a href="#b0285" rid="b0285" class=" bibr popnode">[57]</a>. The details of sequencing output, quality filtering, and mapping statistics per sample are given in <a href="#s0170" rid="s0170" class=" sec">Table S2</a>.</p><p id="p0160">For poly-(A)-enriched datasets, a total number of 4288 differentially expressed genes (DE genes, Benjamini-Hochberg P-value&#x000a0;&#x0003c;&#x000a0;0.05 and log<sub>2</sub> fold change&#x000a0;&#x0003e;1.5 or&#x000a0;&#x0003c;&#x02212;1.5) was identified in TBEV-infected cells when compared to mock control (<a href="#s0170" rid="s0170" class=" sec">Table S3</a>). Of these, 2975 genes (69.4 %) represented protein-coding mRNAs (pc-mRNAs) and 923 (21.5 %) ncRNAs (<a href="/pmc/articles/PMC9167876/figure/f0015/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0015" rid-ob="ob-f0015" co-legend-rid="lgnd_f0015"><span>Fig. 3</span></a>A); pseudogenes and unclassified genes (9.1 %) were excluded from further analyses. Infection with TBEV Hypr resulted in a considerably higher number of DE genes in both, astrocytes and neurons, in comparison to TBEV Neudoerfl (<a href="/pmc/articles/PMC9167876/figure/f0015/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0015" rid-ob="ob-f0015" co-legend-rid="lgnd_f0015"><span>Fig. 3</span></a>B). Moreover, based on the expression profiles of pc-mRNAs and ncRNAs (<a href="/pmc/articles/PMC9167876/figure/f0015/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0015" rid-ob="ob-f0015" co-legend-rid="lgnd_f0015"><span>Fig. 3</span></a>C and D), a strong cell- and strain-specific response was observed, when only 1287 (33.1 %) and 591 (15.2 %) DE genes were identified in both cell types or both TBEV strains, respectively (<a href="#s0170" rid="s0170" class=" sec">Fig. S2</a>). From those, a conserved group of 32 DE pc-mRNAs up-regulated in TBEV-infected datasets was identified (<a href="/pmc/articles/PMC9167876/table/t0010/" target="table" class="fig-table-link figpopup" rid-figpopup="t0010" rid-ob="ob-t0010" co-legend-rid=""><span>Table 2</span></a>), which included genes coding for effectors of the innate immune response, such as receptors sensing viral infection RIG-I and MDA5, or proteins with direct antiviral effect (e.g. BST2, IFIT1-3, RSAD2, MX1, OAS1-3, OASL).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f0015" co-legend-rid="lgnd_f0015"><a href="/pmc/articles/PMC9167876/figure/f0015/" target="figure" rid-figpopup="f0015" rid-ob="ob-f0015"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139861393230416"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9167876_gr3.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gr3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC9167876/bin/gr3.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139861393230416"><a target="object" rel="noopener" href="/pmc/articles/PMC9167876/figure/f0015/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_f0015"><div><a class="figpopup" href="/pmc/articles/PMC9167876/figure/f0015/" target="figure" rid-figpopup="f0015" rid-ob="ob-f0015">Fig. 3</a></div><!--caption a7--><div class="caption"><p>Differential expression analysis of poly-(A) enriched RNAs in TBEV-infected neurons and astrocytes reveals strain-specific and cell-specific responses. <strong>(A)</strong> Graphic summary of poly-(A) classes identified as differentially expressed upon TBEV infection. <strong>(B)</strong> Graphic summary of DE genes (protein-coding mRNAs and non-coding RNAs) in the respective dataset. <strong>(C)</strong> Heatmap visualizing the overall expression pattern of protein-coding mRNAs in analysed samples. <strong>(D)</strong> Heatmap visualizing the overall expression pattern of non-coding RNAs in analysed samples. <strong>(E)</strong> Verification of RNA-seq data by qRT-PCR. Relative expressions of HSPA-6, OASL, RNVU1, and RSAD2 (viperin) genes were processed via the &#x00394;&#x00394;-ct method with HPRT1 as a reference gene. All samples were analysed in biological and technical triplicates. Spearman correlation was evaluated with GraphPad Prism software. <strong>(F)</strong> Gene set enrichment analysis (DAVID tool; GO Biological processes; p&#x000a0;&#x0003c;&#x000a0;0.001) of significantly up-regulated genes in analysed samples performed by DAVID tool. <strong>(G)</strong> Gene set enrichment analysis (DAVID tool; GO Biological processes; p&#x000a0;&#x0003c;&#x000a0;0.001) of significantly down-regulated genes in analysed samples performed by DAVID tool.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="t0010"><h3>Table 2</h3><!--caption a7--><div class="caption"><p>Identification of a common set of genes with TBEV-induced expression in human neurons and astrocytes.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">gene_id</th><th rowspan="1" colspan="1">gene_name</th><th rowspan="1" colspan="1">A24_H</th><th rowspan="1" colspan="1">A24_N</th><th rowspan="1" colspan="1">A72_H</th><th rowspan="1" colspan="1">A72_N</th><th rowspan="1" colspan="1">N24_H</th><th rowspan="1" colspan="1">N24_N</th><th rowspan="1" colspan="1">N72_H</th><th rowspan="1" colspan="1">N72_N</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">ENSG00000239713</td><td rowspan="1" colspan="1">APOBEC3G</td><td align="center" rowspan="1" colspan="1">2.39</td><td align="center" rowspan="1" colspan="1">1.97</td><td align="center" rowspan="1" colspan="1">2.87</td><td align="center" rowspan="1" colspan="1">1.85</td><td align="center" rowspan="1" colspan="1">1.62</td><td align="center" rowspan="1" colspan="1">1.54</td><td align="center" rowspan="1" colspan="1">1.70</td><td align="center" rowspan="1" colspan="1">1.72</td></tr><tr><td rowspan="1" colspan="1">ENSG00000100342</td><td rowspan="1" colspan="1">APOL1</td><td align="center" rowspan="1" colspan="1">2.90</td><td align="center" rowspan="1" colspan="1">2.12</td><td align="center" rowspan="1" colspan="1">2.98</td><td align="center" rowspan="1" colspan="1">1.90</td><td align="center" rowspan="1" colspan="1">3.83</td><td align="center" rowspan="1" colspan="1">3.32</td><td align="center" rowspan="1" colspan="1">2.17</td><td align="center" rowspan="1" colspan="1">2.14</td></tr><tr><td rowspan="1" colspan="1">ENSG00000221963</td><td rowspan="1" colspan="1">APOL6</td><td align="center" rowspan="1" colspan="1">3.29</td><td align="center" rowspan="1" colspan="1">3.03</td><td align="center" rowspan="1" colspan="1">3.02</td><td align="center" rowspan="1" colspan="1">1.99</td><td align="center" rowspan="1" colspan="1">3.02</td><td align="center" rowspan="1" colspan="1">2.76</td><td align="center" rowspan="1" colspan="1">2.77</td><td align="center" rowspan="1" colspan="1">2.42</td></tr><tr><td rowspan="1" colspan="1">ENSG00000130303</td><td rowspan="1" colspan="1">BST2</td><td align="center" rowspan="1" colspan="1">7.06</td><td align="center" rowspan="1" colspan="1">6.40</td><td align="center" rowspan="1" colspan="1">9.80</td><td align="center" rowspan="1" colspan="1">8.65</td><td align="center" rowspan="1" colspan="1">5.46</td><td align="center" rowspan="1" colspan="1">6.16</td><td align="center" rowspan="1" colspan="1">7.50</td><td align="center" rowspan="1" colspan="1">7.60</td></tr><tr><td rowspan="1" colspan="1">ENSG00000134326</td><td rowspan="1" colspan="1">CMPK2</td><td align="center" rowspan="1" colspan="1">3.75</td><td align="center" rowspan="1" colspan="1">3.51</td><td align="center" rowspan="1" colspan="1">4.46</td><td align="center" rowspan="1" colspan="1">3.58</td><td align="center" rowspan="1" colspan="1">1.86</td><td align="center" rowspan="1" colspan="1">2.23</td><td align="center" rowspan="1" colspan="1">2.94</td><td align="center" rowspan="1" colspan="1">2.89</td></tr><tr><td rowspan="1" colspan="1">ENSG00000107201</td><td rowspan="1" colspan="1">DDX58</td><td align="center" rowspan="1" colspan="1">3.41</td><td align="center" rowspan="1" colspan="1">3.23</td><td align="center" rowspan="1" colspan="1">3.94</td><td align="center" rowspan="1" colspan="1">2.99</td><td align="center" rowspan="1" colspan="1">3.23</td><td align="center" rowspan="1" colspan="1">2.93</td><td align="center" rowspan="1" colspan="1">4.05</td><td align="center" rowspan="1" colspan="1">3.05</td></tr><tr><td rowspan="1" colspan="1">ENSG00000137628</td><td rowspan="1" colspan="1">DDX60</td><td align="center" rowspan="1" colspan="1">2.60</td><td align="center" rowspan="1" colspan="1">2.45</td><td align="center" rowspan="1" colspan="1">3.12</td><td align="center" rowspan="1" colspan="1">2.45</td><td align="center" rowspan="1" colspan="1">2.46</td><td align="center" rowspan="1" colspan="1">2.31</td><td align="center" rowspan="1" colspan="1">3.23</td><td align="center" rowspan="1" colspan="1">2.03</td></tr><tr><td rowspan="1" colspan="1">ENSG00000181381</td><td rowspan="1" colspan="1">DDX60L</td><td align="center" rowspan="1" colspan="1">2.60</td><td align="center" rowspan="1" colspan="1">2.45</td><td align="center" rowspan="1" colspan="1">3.12</td><td align="center" rowspan="1" colspan="1">2.45</td><td align="center" rowspan="1" colspan="1">2.46</td><td align="center" rowspan="1" colspan="1">2.31</td><td align="center" rowspan="1" colspan="1">3.23</td><td align="center" rowspan="1" colspan="1">2.03</td></tr><tr><td rowspan="1" colspan="1">ENSG00000108771</td><td rowspan="1" colspan="1">DHX58</td><td align="center" rowspan="1" colspan="1">5.10</td><td align="center" rowspan="1" colspan="1">4.27</td><td align="center" rowspan="1" colspan="1">5.57</td><td align="center" rowspan="1" colspan="1">3.83</td><td align="center" rowspan="1" colspan="1">4.49</td><td align="center" rowspan="1" colspan="1">4.00</td><td align="center" rowspan="1" colspan="1">6.17</td><td align="center" rowspan="1" colspan="1">4.58</td></tr><tr><td rowspan="1" colspan="1">ENSG00000133106</td><td rowspan="1" colspan="1">EPSTI1</td><td align="center" rowspan="1" colspan="1">3.80</td><td align="center" rowspan="1" colspan="1">3.36</td><td align="center" rowspan="1" colspan="1">4.25</td><td align="center" rowspan="1" colspan="1">3.48</td><td align="center" rowspan="1" colspan="1">3.27</td><td align="center" rowspan="1" colspan="1">3.18</td><td align="center" rowspan="1" colspan="1">3.89</td><td align="center" rowspan="1" colspan="1">3.46</td></tr><tr><td rowspan="1" colspan="1">ENSG00000138646</td><td rowspan="1" colspan="1">HERC5</td><td align="center" rowspan="1" colspan="1">5.70</td><td align="center" rowspan="1" colspan="1">4.76</td><td align="center" rowspan="1" colspan="1">6.51</td><td align="center" rowspan="1" colspan="1">5.05</td><td align="center" rowspan="1" colspan="1">5.24</td><td align="center" rowspan="1" colspan="1">3.95</td><td align="center" rowspan="1" colspan="1">7.76</td><td align="center" rowspan="1" colspan="1">5.55</td></tr><tr><td rowspan="1" colspan="1">ENSG00000138642</td><td rowspan="1" colspan="1">HERC6</td><td align="center" rowspan="1" colspan="1">4.50</td><td align="center" rowspan="1" colspan="1">4.08</td><td align="center" rowspan="1" colspan="1">4.72</td><td align="center" rowspan="1" colspan="1">3.90</td><td align="center" rowspan="1" colspan="1">3.34</td><td align="center" rowspan="1" colspan="1">3.71</td><td align="center" rowspan="1" colspan="1">5.00</td><td align="center" rowspan="1" colspan="1">4.14</td></tr><tr><td rowspan="1" colspan="1">ENSG00000068079</td><td rowspan="1" colspan="1">IFI35</td><td align="center" rowspan="1" colspan="1">3.95</td><td align="center" rowspan="1" colspan="1">3.13</td><td align="center" rowspan="1" colspan="1">4.64</td><td align="center" rowspan="1" colspan="1">3.47</td><td align="center" rowspan="1" colspan="1">1.98</td><td align="center" rowspan="1" colspan="1">2.76</td><td align="center" rowspan="1" colspan="1">3.44</td><td align="center" rowspan="1" colspan="1">3.77</td></tr><tr><td rowspan="1" colspan="1">ENSG00000137965</td><td rowspan="1" colspan="1">IFI44</td><td align="center" rowspan="1" colspan="1">2.51</td><td align="center" rowspan="1" colspan="1">2.52</td><td align="center" rowspan="1" colspan="1">3.53</td><td align="center" rowspan="1" colspan="1">2.83</td><td align="center" rowspan="1" colspan="1">2.01</td><td align="center" rowspan="1" colspan="1">2.72</td><td align="center" rowspan="1" colspan="1">3.79</td><td align="center" rowspan="1" colspan="1">3.15</td></tr><tr><td rowspan="1" colspan="1">ENSG00000137959</td><td rowspan="1" colspan="1">IFI44L</td><td align="center" rowspan="1" colspan="1">2.51</td><td align="center" rowspan="1" colspan="1">2.52</td><td align="center" rowspan="1" colspan="1">3.53</td><td align="center" rowspan="1" colspan="1">2.83</td><td align="center" rowspan="1" colspan="1">2.01</td><td align="center" rowspan="1" colspan="1">2.72</td><td align="center" rowspan="1" colspan="1">3.79</td><td align="center" rowspan="1" colspan="1">3.15</td></tr><tr><td rowspan="1" colspan="1">ENSG00000126709</td><td rowspan="1" colspan="1">IFI6</td><td align="center" rowspan="1" colspan="1">3.84</td><td align="center" rowspan="1" colspan="1">3.11</td><td align="center" rowspan="1" colspan="1">5.45</td><td align="center" rowspan="1" colspan="1">4.50</td><td align="center" rowspan="1" colspan="1">1.95</td><td align="center" rowspan="1" colspan="1">2.93</td><td align="center" rowspan="1" colspan="1">4.06</td><td align="center" rowspan="1" colspan="1">4.44</td></tr><tr><td rowspan="1" colspan="1">ENSG00000115267</td><td rowspan="1" colspan="1">IFIH1</td><td align="center" rowspan="1" colspan="1">5.47</td><td align="center" rowspan="1" colspan="1">4.97</td><td align="center" rowspan="1" colspan="1">6.24</td><td align="center" rowspan="1" colspan="1">5.02</td><td align="center" rowspan="1" colspan="1">5.58</td><td align="center" rowspan="1" colspan="1">4.56</td><td align="center" rowspan="1" colspan="1">7.48</td><td align="center" rowspan="1" colspan="1">5.99</td></tr><tr><td rowspan="1" colspan="1">ENSG00000185745</td><td rowspan="1" colspan="1">IFIT1</td><td align="center" rowspan="1" colspan="1">6.47</td><td align="center" rowspan="1" colspan="1">6.04</td><td align="center" rowspan="1" colspan="1">7.29</td><td align="center" rowspan="1" colspan="1">6.30</td><td align="center" rowspan="1" colspan="1">5.84</td><td align="center" rowspan="1" colspan="1">5.72</td><td align="center" rowspan="1" colspan="1">7.84</td><td align="center" rowspan="1" colspan="1">6.79</td></tr><tr><td rowspan="1" colspan="1">ENSG00000119922</td><td rowspan="1" colspan="1">IFIT2</td><td align="center" rowspan="1" colspan="1">6.28</td><td align="center" rowspan="1" colspan="1">5.72</td><td align="center" rowspan="1" colspan="1">6.98</td><td align="center" rowspan="1" colspan="1">5.72</td><td align="center" rowspan="1" colspan="1">6.74</td><td align="center" rowspan="1" colspan="1">5.23</td><td align="center" rowspan="1" colspan="1">8.12</td><td align="center" rowspan="1" colspan="1">5.57</td></tr><tr><td rowspan="1" colspan="1">ENSG00000119917</td><td rowspan="1" colspan="1">IFIT3</td><td align="center" rowspan="1" colspan="1">5.51</td><td align="center" rowspan="1" colspan="1">5.14</td><td align="center" rowspan="1" colspan="1">6.71</td><td align="center" rowspan="1" colspan="1">5.65</td><td align="center" rowspan="1" colspan="1">6.09</td><td align="center" rowspan="1" colspan="1">5.24</td><td align="center" rowspan="1" colspan="1">7.61</td><td align="center" rowspan="1" colspan="1">6.21</td></tr><tr><td rowspan="1" colspan="1">ENSG00000187608</td><td rowspan="1" colspan="1">ISG15</td><td align="center" rowspan="1" colspan="1">5.81</td><td align="center" rowspan="1" colspan="1">4.84</td><td align="center" rowspan="1" colspan="1">6.83</td><td align="center" rowspan="1" colspan="1">5.46</td><td align="center" rowspan="1" colspan="1">4.25</td><td align="center" rowspan="1" colspan="1">5.14</td><td align="center" rowspan="1" colspan="1">6.64</td><td align="center" rowspan="1" colspan="1">5.99</td></tr><tr><td rowspan="1" colspan="1">ENSG00000157601</td><td rowspan="1" colspan="1">MX1</td><td align="center" rowspan="1" colspan="1">7.16</td><td align="center" rowspan="1" colspan="1">6.37</td><td align="center" rowspan="1" colspan="1">7.12</td><td align="center" rowspan="1" colspan="1">6.11</td><td align="center" rowspan="1" colspan="1">5.11</td><td align="center" rowspan="1" colspan="1">6.60</td><td align="center" rowspan="1" colspan="1">6.93</td><td align="center" rowspan="1" colspan="1">7.45</td></tr><tr><td rowspan="1" colspan="1">ENSG00000123609</td><td rowspan="1" colspan="1">NMI</td><td align="center" rowspan="1" colspan="1">2.85</td><td align="center" rowspan="1" colspan="1">2.54</td><td align="center" rowspan="1" colspan="1">3.87</td><td align="center" rowspan="1" colspan="1">2.95</td><td align="center" rowspan="1" colspan="1">2.39</td><td align="center" rowspan="1" colspan="1">1.81</td><td align="center" rowspan="1" colspan="1">3.14</td><td align="center" rowspan="1" colspan="1">2.90</td></tr><tr><td rowspan="1" colspan="1">ENSG00000089127</td><td rowspan="1" colspan="1">OAS1</td><td align="center" rowspan="1" colspan="1">6.63</td><td align="center" rowspan="1" colspan="1">5.84</td><td align="center" rowspan="1" colspan="1">8.54</td><td align="center" rowspan="1" colspan="1">7.47</td><td align="center" rowspan="1" colspan="1">3.87</td><td align="center" rowspan="1" colspan="1">4.44</td><td align="center" rowspan="1" colspan="1">5.88</td><td align="center" rowspan="1" colspan="1">6.44</td></tr><tr><td rowspan="1" colspan="1">ENSG00000111335</td><td rowspan="1" colspan="1">OAS2</td><td align="center" rowspan="1" colspan="1">9.00</td><td align="center" rowspan="1" colspan="1">8.05</td><td align="center" rowspan="1" colspan="1">12.07</td><td align="center" rowspan="1" colspan="1">11.03</td><td align="center" rowspan="1" colspan="1">9.10</td><td align="center" rowspan="1" colspan="1">9.32</td><td align="center" rowspan="1" colspan="1">5.68</td><td align="center" rowspan="1" colspan="1">5.91</td></tr><tr><td rowspan="1" colspan="1">ENSG00000111331</td><td rowspan="1" colspan="1">OAS3</td><td align="center" rowspan="1" colspan="1">3.99</td><td align="center" rowspan="1" colspan="1">3.37</td><td align="center" rowspan="1" colspan="1">5.85</td><td align="center" rowspan="1" colspan="1">4.68</td><td align="center" rowspan="1" colspan="1">2.83</td><td align="center" rowspan="1" colspan="1">3.31</td><td align="center" rowspan="1" colspan="1">4.18</td><td align="center" rowspan="1" colspan="1">4.01</td></tr><tr><td rowspan="1" colspan="1">ENSG00000135114</td><td rowspan="1" colspan="1">OASL</td><td align="center" rowspan="1" colspan="1">8.49</td><td align="center" rowspan="1" colspan="1">7.18</td><td align="center" rowspan="1" colspan="1">11.16</td><td align="center" rowspan="1" colspan="1">9.47</td><td align="center" rowspan="1" colspan="1">8.55</td><td align="center" rowspan="1" colspan="1">7.59</td><td align="center" rowspan="1" colspan="1">9.89</td><td align="center" rowspan="1" colspan="1">8.58</td></tr><tr><td rowspan="1" colspan="1">ENSG00000173193</td><td rowspan="1" colspan="1">PARP14</td><td align="center" rowspan="1" colspan="1">2.37</td><td align="center" rowspan="1" colspan="1">2.51</td><td align="center" rowspan="1" colspan="1">2.85</td><td align="center" rowspan="1" colspan="1">2.27</td><td align="center" rowspan="1" colspan="1">1.73</td><td align="center" rowspan="1" colspan="1">1.99</td><td align="center" rowspan="1" colspan="1">2.26</td><td align="center" rowspan="1" colspan="1">2.01</td></tr><tr><td rowspan="1" colspan="1">ENSG00000134321</td><td rowspan="1" colspan="1">RSAD2</td><td align="center" rowspan="1" colspan="1">6.46</td><td align="center" rowspan="1" colspan="1">6.01</td><td align="center" rowspan="1" colspan="1">8.50</td><td align="center" rowspan="1" colspan="1">7.41</td><td align="center" rowspan="1" colspan="1">4.19</td><td align="center" rowspan="1" colspan="1">3.87</td><td align="center" rowspan="1" colspan="1">6.37</td><td align="center" rowspan="1" colspan="1">6.09</td></tr><tr><td rowspan="1" colspan="1">ENSG00000205413</td><td rowspan="1" colspan="1">SAMD9</td><td align="center" rowspan="1" colspan="1">4.06</td><td align="center" rowspan="1" colspan="1">3.99</td><td align="center" rowspan="1" colspan="1">5.06</td><td align="center" rowspan="1" colspan="1">4.10</td><td align="center" rowspan="1" colspan="1">3.67</td><td align="center" rowspan="1" colspan="1">3.84</td><td align="center" rowspan="1" colspan="1">5.47</td><td align="center" rowspan="1" colspan="1">4.94</td></tr><tr><td rowspan="1" colspan="1">ENSG00000184979</td><td rowspan="1" colspan="1">USP18</td><td align="center" rowspan="1" colspan="1">4.11</td><td align="center" rowspan="1" colspan="1">3.65</td><td align="center" rowspan="1" colspan="1">4.87</td><td align="center" rowspan="1" colspan="1">3.77</td><td align="center" rowspan="1" colspan="1">1.79</td><td align="center" rowspan="1" colspan="1">2.55</td><td align="center" rowspan="1" colspan="1">3.26</td><td align="center" rowspan="1" colspan="1">3.19</td></tr><tr><td rowspan="1" colspan="1">ENSG00000132530</td><td rowspan="1" colspan="1">XAF1</td><td align="center" rowspan="1" colspan="1">2.27</td><td align="center" rowspan="1" colspan="1">2.20</td><td align="center" rowspan="1" colspan="1">2.81</td><td align="center" rowspan="1" colspan="1">2.32</td><td align="center" rowspan="1" colspan="1">2.95</td><td align="center" rowspan="1" colspan="1">2.69</td><td align="center" rowspan="1" colspan="1">3.07</td><td align="center" rowspan="1" colspan="1">2.98</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139861393227968"><a target="object" rel="noopener" href="/pmc/articles/PMC9167876/table/t0010/?report=objectonly">Open in a separate window</a></div></div><p id="p0165">The differential expression analysis results were verified using real-time PCR quantification of mRNA levels for selected DE genes (RSAD2, RNVU1, HSPA6, OASL). The correlation between qPCR and NGS data is statistically significant (Spearman correlation r&#x000a0;=&#x000a0;0.81, p&#x000a0;&#x0003c;&#x000a0;0.001), therefore, the RNA-seq data were successfully validated (<a href="/pmc/articles/PMC9167876/figure/f0015/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0015" rid-ob="ob-f0015" co-legend-rid="lgnd_f0015"><span>Fig. 3</span></a>E).</p><p id="p0170" class="p p-last">To describe the cellular processes affected by the dysregulation of identified DE pc-mRNAs, we performed a gene set enrichment analysis (GSEA) using the DAVID toolkit <a href="#b0290" rid="b0290" class=" bibr popnode">[58]</a>. A vast majority of significantly enriched GO classes (Benjamini-Hochberg P-value&#x000a0;&#x0003c;&#x000a0;0.05) were related to the antiviral immune response (<a href="/pmc/articles/PMC9167876/figure/f0015/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0015" rid-ob="ob-f0015" co-legend-rid="lgnd_f0015"><span>Fig. 3</span></a>G). In addition, the same analyses of down-regulated gene sets uncovered an interesting phenomenon, where the proper neural development and assembly of the extracellular matrix are negatively affected by TBEV infection (<a href="/pmc/articles/PMC9167876/figure/f0015/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0015" rid-ob="ob-f0015" co-legend-rid="lgnd_f0015"><span>Fig. 3</span></a>G).</p></div><div id="s0110" class="sec sec-last"><p></p><h4 id="s0110title" class="inline">3.2.2.  Differential expression analysis of small RNAs </h4><p id="p0180" class="p p-first">Sequencing the small RNA selected library of all samples yielded an average of 41&#x000a0;M 50 SE reads. Quality filtering using Cutadapt dropped on average 1.8 % low-quality and/or short reads. The remaining reads were mapped to the human genome assembly (GRCh38.p13) using miRDeep2 with 58.2 % average mapping efficiency. The details of sequencing output, quality filtering and mapping statistics per sample are given in <a href="#s0170" rid="s0170" class=" sec">Table S4</a>.</p><p id="p0185">In small RNA datasets, a total of 145 miRNA species were described as significantly dysregulated upon TBEV infection (Benjamini-Hochberg P-value&#x000a0;&#x0003c;&#x000a0;0.05 and log<sub>2</sub> fold change&#x000a0;&#x0003e;1.5 or&#x000a0;&#x0003c;&#x02212;1.5) (<a href="/pmc/articles/PMC9167876/figure/f0020/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0020" rid-ob="ob-f0020" co-legend-rid="lgnd_f0020"><span>Fig. 4</span></a>; <a href="#s0170" rid="s0170" class=" sec">Table S5</a>). Interestingly, while infection with severe Hypr strain resulted in higher numbers of dysregulated pc-mRNAs and ncRNAs when compared to mild Neudoerfl strain, the DE miRNA pattern showed an opposite trend (<a href="/pmc/articles/PMC9167876/figure/f0020/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0020" rid-ob="ob-f0020" co-legend-rid="lgnd_f0020"><span>Fig. 4</span></a>A), that is distinct reduction of significantly affected miRNAs by more virulent TBEV strain. Furthermore, the expression profiles document a surprisingly high cell-specific response as only one miRNA species (<em>hsa-miR-592</em>) was found to be up-regulated in both cell types, astrocytes (A72_H) and neurons (N72_N) (<a href="/pmc/articles/PMC9167876/figure/f0020/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0020" rid-ob="ob-f0020" co-legend-rid="lgnd_f0020"><span>Fig. 4</span></a>B and D). The phenomenon of high specificity was also observed between TBEV strains, where only 14 (9.7 %) miRNA species followed the same pattern of dysregulation in the case of both, Hypr and Neudoerfl (<a href="/pmc/articles/PMC9167876/figure/f0020/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0020" rid-ob="ob-f0020" co-legend-rid="lgnd_f0020"><span>Fig. 4</span></a>C and D). The verification of miRNA-seq data was performed using qPCR quantification of <em>hsa-miR-1248</em> and <em>hsa-miR-145-5p; the</em> positive correlation between both methods was confirmed to be statistically significant (<a href="/pmc/articles/PMC9167876/figure/f0020/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0020" rid-ob="ob-f0020" co-legend-rid="lgnd_f0020"><span>Fig. 4</span></a>E; Spearman correlation r&#x000a0;=&#x000a0;0.64, p&#x000a0;=&#x000a0;0.009).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f0020" co-legend-rid="lgnd_f0020"><a href="/pmc/articles/PMC9167876/figure/f0020/" target="figure" rid-figpopup="f0020" rid-ob="ob-f0020"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139861395641088"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9167876_gr4.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gr4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC9167876/bin/gr4.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139861395641088"><a target="object" rel="noopener" href="/pmc/articles/PMC9167876/figure/f0020/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_f0020"><div><a class="figpopup" href="/pmc/articles/PMC9167876/figure/f0020/" target="figure" rid-figpopup="f0020" rid-ob="ob-f0020">Fig. 4</a></div><!--caption a7--><div class="caption"><p><strong>Differential expression analysis of miRNAs in TBEV-infected neurons and astrocytes revealed strain-specific and cell-specific responses. (A)</strong> Graphic summary of DE miRNAs in analysed samples. (&#x02191;) up-regulated miRNAs, (&#x02193;) down-regulated miRNAs. <strong>(B)</strong> Venn diagram describing the cell specificity of miRNA expression upon TBEV infection. (&#x02191;) up-regulated miRNAs, (&#x02193;) down-regulated miRNAs. <strong>(C)</strong> Venn diagram describing the strain specificity of miRNA expression upon TBEV infection. (H) &#x02013; TBEV Hypr strain, (N) &#x02013; TBEV Neudoerfl strain. <strong>(D)</strong> Heatmap visualizing the overall expression pattern of mi RNAs in analysed samples. <strong>(E)</strong> Verification of RNA-seq data by qRT-PCR. Relative expression of <em>hsa-miR-1248</em> and <em>hsa-miR-145-5p</em> was calculated using the &#x00394;&#x00394;-ct method with <em>hsa-miR-103a</em> as a reference gene and Sp6 as an internal control. All samples were analysed in biological and technical triplicates. Spearman correlation was evaluated with GraphPad Prism software. <strong>(F)</strong> Frequency distribution of 21&#x02013;23 nt vd-sRNAs mapped to TBEV Hypr genome in N72_H (black) and A72_H (red) samples. HS &#x02013; hot-spot. (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this article.)</p></div></div></div><p id="p0190" class="p p-last">Several studies dealing with small RNA profiling in flavivirus-infected cells have reported the presence of vd-sRNAs <a href="#b0170" rid="b0170" class=" bibr popnode">[34]</a>, <a href="#b0175" rid="b0175" class=" bibr popnode">[35]</a>. We, therefore, assessed whether TBEV infection also resulted in the generation of vd-sRNAs by mapping the small RNA datasets from RNA-seq to TBEV Hypr genomic sequence (GenBank accession no. <a id="__tag_1132452544" class="tag_hotlink" href="/nuccore/U39292" ref="/nuccore/U39292"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"U39292","term_id":"1066074","term_text":"U39292"}}</span>U39292</a>). The production of vd-sRNAs was confirmed in all TBEV Hypr-infected samples with small RNA species ranging from 16 to 50 nucleotides in size. As expected, the amounts of mapped reads increased with the infection progress in all specimens. Furthermore, vd-sRNAs were predominantly derived from the sense strand of the TBEV genome (<a href="#s0170" rid="s0170" class=" sec">Fig. S3</a>) and reached higher, but not significantly different (unpaired Student&#x02019;s <em>t</em>-test; p&#x000a0;&#x0003c;&#x000a0;0.05), levels in neurons (6.24&#x000d7;10<sup>5</sup>&#x000a0;&#x000b1;&#x000a0;5.74&#x000d7;10<sup>4</sup>) than in astrocytes (4.87&#x000d7;10<sup>5</sup>&#x000a0;&#x000b1;&#x000a0;6.30&#x000d7;10<sup>4</sup>) (<a href="/pmc/articles/PMC9167876/figure/f0020/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0020" rid-ob="ob-f0020" co-legend-rid="lgnd_f0020"><span>Fig. 4</span></a>E). These findings positively correlate with higher viral titres and gRNA observed in neurons (<a href="/pmc/articles/PMC9167876/figure/f0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0005" rid-ob="ob-f0005" co-legend-rid="lgnd_f0005"><span>Fig. 1</span></a>). The pattern of reads distribution throughout the viral genome appears rather mosaic with several specific hot-spots. Besides, the hot-spots identified in all of the analysed samples are strikingly conserved and the main difference was observed on the level of read counts (<a href="#s0170" rid="s0170" class=" sec">Fig. S4</a>). The most universal/abundant peak was identified at the end of the 3&#x02032; UTR.</p></div></div><div id="s0115" class="sec"><h3 id="s0115title">3.3. Candidate gene networks responsible for a different outcome of TBEV Hypr infection in neurons vs. Astrocytes</h3><p id="p0195" class="p p-first">As TBEV Hypr dramatically decreased the number of living neurons but not astrocytes (<a href="/pmc/articles/PMC9167876/figure/f0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0005" rid-ob="ob-f0005" co-legend-rid="lgnd_f0005"><span>Fig. 1</span></a>C), we sought to characterize the differences between N72_H and A72_H datasets to identify potential key factors involved in higher neuronal susceptibility to TBEV Hypr infection. First, we selected DE pc-mRNAs, which were either uniquely up-/down-regulated in N72_H/A72_H datasets or the difference in up-/down-regulation between N72_H/A72_H datasets was higher than 1.5 log<sub>2</sub>. A total of 1849 DE pc-mRNAs were identified and further divided into four groups according to their expression profile (unique expression in neurons/astrocytes or higher expression in neurons/astrocytes). Subsequently, the GSEA analysis of all groups revealed that the main difference is a more robust immune response elicited in infected astrocytes (<a href="/pmc/articles/PMC9167876/figure/f0025/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0025" rid-ob="ob-f0025" co-legend-rid="lgnd_f0025"><span>Fig. 5</span></a>A). In more detail, either a triggering of astrocyte-specific antiviral response genes (e.g., <em>TNFSF10</em>, <em>MX2</em>, <em>MMP13</em>, <em>IFITM1</em>, or <em>IFITM2</em>) or higher up-regulation of immune response genes (e.g., <em>OAS1-3</em>, <em>RSAD2</em>, <em>BST2</em>, <em>IL6</em>, or <em>ISG20</em>) was observed (<a href="/pmc/articles/PMC9167876/table/t0010/" target="table" class="fig-table-link figpopup" rid-figpopup="t0010" rid-ob="ob-t0010" co-legend-rid=""><span>Table 2</span></a>; <a href="#s0170" rid="s0170" class=" sec">Fig. S5</a>). These findings correlate with qPCR data, where <em>RSAD2</em> (viperin) and <em>OASL</em> were significantly more up-regulated in astrocytes than in neurons (<a href="/pmc/articles/PMC9167876/figure/f0025/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0025" rid-ob="ob-f0025" co-legend-rid="lgnd_f0025"><span>Fig. 5</span></a>B). A specific expression pattern was observed for <em>IFN-&#x003b2;</em> and <em>IFN-&#x003bb;</em> genes, when <em>IFN-&#x003b2;</em> seems to be responsible for an early immune response to TBEV followed by the expression of <em>IFN-&#x003bb;</em> (<a href="/pmc/articles/PMC9167876/figure/f0025/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0025" rid-ob="ob-f0025" co-legend-rid="lgnd_f0025"><span>Fig. 5</span></a>C). Interestingly, the late expression of <em>IFN-&#x003bb;</em> (72&#x000a0;h p.i.) appeared in astrocytes only, suggesting its role as a factor associated with the neuronal susceptibility to TBEV infection. Additionally, to put our data in context with previous research, we compared our comprehensive list of DE immune response-related genes with altered protein levels of selected cytokines <a href="#b0065" rid="b0065" class=" bibr popnode">[13]</a> and mRNAs <a href="#b0060" rid="b0060" class=" bibr popnode">[12]</a> retrieved for TBEV by others (<a href="/pmc/articles/PMC9167876/figure/f0025/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0025" rid-ob="ob-f0025" co-legend-rid="lgnd_f0025"><span>Fig. 5</span></a>D). Our data showed a high level of coherence with these previously published expression patterns and considerably extended the previous ones.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f0025" co-legend-rid="lgnd_f0025"><a href="/pmc/articles/PMC9167876/figure/f0025/" target="figure" rid-figpopup="f0025" rid-ob="ob-f0025"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139861395327760"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9167876_gr5.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gr5.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC9167876/bin/gr5.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139861395327760"><a target="object" rel="noopener" href="/pmc/articles/PMC9167876/figure/f0025/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_f0025"><div><a class="figpopup" href="/pmc/articles/PMC9167876/figure/f0025/" target="figure" rid-figpopup="f0025" rid-ob="ob-f0025">Fig. 5</a></div><!--caption a7--><div class="caption"><p><strong>TBEV Hypr infection in astrocytes results in a stronger innate immune response than in neurons. (A)</strong> Heatmap summarizing results from gene set enrichment analysis (DAVID tool; GO Biological processes; p&#x000a0;&#x0003c;&#x000a0;0.05) of DE genes 1) uniquely expressed in either A72_H or N72_H, and 2) genes expressed with &#x0003e;1.5 log<sub>2</sub> difference in one of the samples. <strong>(B)</strong> Relative expression of OASL and RSAD2 (viperin) genes in TBEV infected cells. Calculations were performed via the &#x00394;&#x00394;-ct method with HPRT1 as a reference gene. All samples were analysed in biologcal and technical triplicates. <strong>(C)</strong> Expression pattern of <em>IFN-&#x003b2;</em> and <em>IFN-&#x003bb;</em> in TBEV-infected cells based on RNA-seq data. <strong>(D)</strong> Overview and comparison of the expression pattern of selected immune response-related genes from this study and works of Fares <em>et al.</em> [<a href="#b0060" rid="b0060" class=" bibr popnode">12</a>] and Pokorna <em>et al.</em> [<a href="#b0065" rid="b0065" class=" bibr popnode">13</a>] <strong>(E)</strong> Number of mapped reads for vd-sRNAs (16&#x02013;50 nt) in the respective sample. Data summarise three independent experiments, and values in graphs are expressed as mean&#x000a0;&#x000b1;&#x000a0;SD.</p></div></div></div><p id="p0200">The DE miRNAs and lncRNAs were selected based on the same unique or more pronounced response criteria used to evaluate pc-mRNAs. This way, the miRNA <em>trans</em> target prediction was performed for the selected datasets of pc-mRNAs (miRWalk; <a href="#b0255" rid="b0255" class=" bibr popnode">[51]</a>) and lncRNAs (LncBase v.2; <a href="#b0260" rid="b0260" class=" bibr popnode">[52]</a>). Prediction results summarized in <a href="#s0170" rid="s0170" class=" sec">Table S6</a> show that 173 genes were identified as possible <em>trans</em> targets (133 pc-mRNAs and 40 lncRNAs), including genes involved in immune response (e.g., <em>OAS1</em>, <em>TRIM38</em>, <em>SPSB4</em>), proper neuronal development and function (e.g., <em>NRARP</em>, <em>KCNK10</em>, <em>SIRPA</em>, <em>CLIC5</em>, <em>TMC8</em>, <em>KIT</em>, <em>CNTN2</em>, <em>DCC</em>) as well as transcription factors (e.g., <em>TFAP2B</em>, <em>HEYL</em>, <em>GATAD2B</em>, <em>BEND3</em>). In order to get a complete overview of the possible interactions, potential <em>cis</em> targets (100&#x000a0;kb upstream and downstream) were predicted for DE lncRNAs with log<sub>2</sub>-fold change&#x000a0;&#x0003e;4 or&#x000a0;&#x0003c;&#x02212;4 as well as for lncRNAs identified as potential targets of DE miRNAs (<a href="#s0170" rid="s0170" class=" sec">Table S7</a>). In total, 45 <em>cis</em> targets were predicted, including immune response-related genes (loci <em>RSAD2</em>/<em>CMPK2</em> and IFNL1-4; <em>IFI6</em>), receptors (e.g., <em>DCC</em>, <em>UTS2R</em>), and transcription factors (e.g., <em>DLX5</em>, <em>OTX2</em>).</p><p id="p0205" class="p p-last">The differences in the vd-sRNA mapping profile between the two cell types may substantially influence the enhanced neuronal sensitivity to Hypr infection. One of the possible scenarios suggests that vd-sRNAs could mimic host miRNAs, leading to a lowered expression of a target gene <a href="#b0185" rid="b0185" class=" bibr popnode">[37]</a>. Therefore, we extracted the 21&#x02013;23 nt long vd-sRNAs and selected hotspots HS1-HS12 which were present uniquely or in higher/lower numbers in N72_H and A72_H samples (<a href="/pmc/articles/PMC9167876/table/t0015/" target="table" class="fig-table-link figpopup" rid-figpopup="t0015" rid-ob="ob-t0015" co-legend-rid=""><span>Table 3</span></a>; <a href="/pmc/articles/PMC9167876/figure/f0025/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0025" rid-ob="ob-f0025" co-legend-rid="lgnd_f0025"><span>Fig. 5</span></a>E). Surprisingly, HS3, HS7, HS9, HS10, and HS12 had higher read counts in Hypr-infected astrocytes, and only HS11 had a higher read count in Hypr-infected neurons. Subsequently, the prediction of vd-sRNA targets was performed using the miRDB tool in combination with DE pc-mRNA lists. In total, 51 host DE pc-mRNAs were identified (<a href="/pmc/articles/PMC9167876/table/t0020/" target="table" class="fig-table-link figpopup" rid-figpopup="t0020" rid-ob="ob-t0020" co-legend-rid=""><span>Table 4</span></a>). Interestingly, vd-sRNAs derived from HS10 and HS12, the two hot-spots with the highest read count in astrocytes, were both predicted to target RNA splicing factors MBNL2 and RBFOX1, whose mRNA levels were significantly lower in A72_H in comparison to N72_H (<a href="#s0170" rid="s0170" class=" sec">Table S3</a>). For HS11-derived vd-sRNA, nine mRNA targets with the corresponding expression pattern (N72_H&#x000a0;&#x0003c;&#x000a0;A72_H) were predicted (<em>LEKR1</em>, <em>DLX1</em>, <em>AOX1</em>, <em>MYCN</em>, <em>PCDH18</em>, <em>DPYSL2</em>, <em>TSHZ1</em>, <em>PFKFB3</em>, <em>BCL9L</em>). Notably, four out of nine identified target genes were characterized as transcription factors (<em>DLX1</em>, <em>MYCN</em>, <em>TSHZ1</em>, <em>BCL9L</em>), with DLX1 and DPYSL2 further considered as factors contributing to proper neural development <a href="#b0295" rid="b0295" class=" bibr popnode">[59]</a>, <a href="#b0300" rid="b0300" class=" bibr popnode">[60]</a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="t0015"><h3>Table 3</h3><!--caption a7--><div class="caption"><p><strong>Identification of hot-spots for 21</strong>&#x02013;<strong>23 nucleotides long vd-sRNAs in TBEV Neudoerfl and Hypr genome.</strong> SNPs characteristic for either Hypr or Neudoerfl strains are marked in bold.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">vd-sRNA</th><th rowspan="1" colspan="1">sequence (5&#x02032;-3&#x02032;)</th><th rowspan="1" colspan="1">start</th><th rowspan="1" colspan="1">end</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">HS1</td><td rowspan="1" colspan="1">TGCTTCGGACAGCATTAGCAGC</td><td rowspan="1" colspan="1">26</td><td rowspan="1" colspan="1">47</td></tr><tr><td rowspan="1" colspan="1">HS2</td><td rowspan="1" colspan="1">AAGGCGTTCTGGAACTCAGTCCC</td><td rowspan="1" colspan="1">310</td><td rowspan="1" colspan="1">332</td></tr><tr><td rowspan="1" colspan="1">HS3</td><td rowspan="1" colspan="1">AGGAGAAGAGCCTGTTGACGTG</td><td rowspan="1" colspan="1">645</td><td rowspan="1" colspan="1">666</td></tr><tr><td rowspan="1" colspan="1">HS4</td><td rowspan="1" colspan="1">ATCTCCAGATGTGAACGTGGCC</td><td rowspan="1" colspan="1">2016</td><td rowspan="1" colspan="1">2037</td></tr><tr><td rowspan="1" colspan="1">HS5</td><td rowspan="1" colspan="1">TAAGGACGGTGTCTACAGGATT</td><td rowspan="1" colspan="1">4659</td><td rowspan="1" colspan="1">4680</td></tr><tr><td rowspan="2" colspan="1">HS6</td><td rowspan="1" colspan="1">AAAGTGTGAT<strong>C</strong>TGTTTGAACAG (Hypr)</td><td rowspan="2" colspan="1">5756</td><td rowspan="2" colspan="1">5777</td></tr><tr><td rowspan="1" colspan="1">AAAGTGTGAT<strong>T</strong>TGTTTGAACAG (Neu)</td></tr><tr><td rowspan="2" colspan="1">HS7</td><td rowspan="1" colspan="1">TGAAAAGGACTACTC<strong>A</strong>AGAGT (Hypr)</td><td rowspan="2" colspan="1">5787</td><td rowspan="2" colspan="1">5807</td></tr><tr><td rowspan="1" colspan="1">TGAAAAGGACTACTC<strong>C</strong>AGAGT (Neu)</td></tr><tr><td rowspan="1" colspan="1">HS8-1</td><td rowspan="1" colspan="1">TGTGGTGACGACTGATATCTC</td><td rowspan="1" colspan="1">5829</td><td rowspan="1" colspan="1">5849</td></tr><tr><td rowspan="1" colspan="1">HS8-2</td><td rowspan="1" colspan="1">TGACGACTGATATCTCGGAGATG</td><td rowspan="1" colspan="1">5834</td><td rowspan="1" colspan="1">5856</td></tr><tr><td rowspan="1" colspan="1">HS9</td><td rowspan="1" colspan="1">TGGACAGTGTGATGATGATGAC</td><td rowspan="1" colspan="1">6030</td><td rowspan="1" colspan="1">6051</td></tr><tr><td rowspan="2" colspan="1">HS10</td><td rowspan="1" colspan="1">ATTT<strong>A</strong>GATCAGGAATGGACGTG (Hypr)</td><td rowspan="2" colspan="1">8040</td><td rowspan="2" colspan="1">8061</td></tr><tr><td rowspan="1" colspan="1">ATTT<strong>C</strong>GATCAGGAATGGACGTG (Neu)</td></tr><tr><td rowspan="2" colspan="1">HS11</td><td rowspan="1" colspan="1">GACACAG<strong>A</strong>TAGTCTGACAAGGA (Hypr)</td><td rowspan="2" colspan="1">10,784 (Hypr), 11,090 (Neu)</td><td rowspan="2" colspan="1">10,805 (Hypr), 11,111 (Neu)</td></tr><tr><td rowspan="1" colspan="1">GACACAG<strong>G</strong>TAGTCTGACAAGGA (Neu)</td></tr><tr><td rowspan="1" colspan="1">HS12</td><td rowspan="1" colspan="1">TGATGTGTGACTCGGAAAAAC</td><td rowspan="1" colspan="1">10,808 (Hypr), 11,114 (Neu)</td><td rowspan="1" colspan="1">10,828 (Hypr), 11,134 (Neu)</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139861395281520"><a target="object" rel="noopener" href="/pmc/articles/PMC9167876/table/t0015/?report=objectonly">Open in a separate window</a></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="t0020"><h3>Table 4</h3><!--caption a7--><div class="caption"><p><strong>List of miRDB tool-predicted host targets of vd-sRNAs.</strong> A/N ratio &#x02013; mapped read count ratio between astrocytes and neurons.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1">vd-sRNA</th><th colspan="2" rowspan="1">&#x003a3; read count<hr /></th><th rowspan="2" colspan="1">A/N ratio</th><th rowspan="2" colspan="1">predicted gene targets (miRDB)</th></tr><tr><th rowspan="1" colspan="1">A72_H</th><th rowspan="1" colspan="1">N72_H</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">HS1</td><td rowspan="1" colspan="1">279 497</td><td rowspan="1" colspan="1">243 019</td><td rowspan="1" colspan="1">1.15</td><td rowspan="1" colspan="1">N/A</td></tr><tr><td rowspan="1" colspan="1">HS2</td><td rowspan="1" colspan="1">171 548</td><td rowspan="1" colspan="1">169 885</td><td rowspan="1" colspan="1">1.01</td><td rowspan="1" colspan="1">N/A</td></tr><tr><td rowspan="1" colspan="1">HS3</td><td rowspan="1" colspan="1">373 158</td><td rowspan="1" colspan="1">159 523</td><td rowspan="1" colspan="1">2.34</td><td rowspan="1" colspan="1">OPCML, THUMPD2, LPXN, GAS7, CDH7, ERBB4, ZMAT3, ANKS1A, GIPC2</td></tr><tr><td rowspan="1" colspan="1">HS4</td><td rowspan="1" colspan="1">92 596</td><td rowspan="1" colspan="1">96 652</td><td rowspan="1" colspan="1">0.96</td><td rowspan="1" colspan="1">N/A</td></tr><tr><td rowspan="1" colspan="1">HS5</td><td rowspan="1" colspan="1">108 436</td><td rowspan="1" colspan="1">50 154</td><td rowspan="1" colspan="1">2.16</td><td rowspan="1" colspan="1">no targets</td></tr><tr><td rowspan="1" colspan="1">HS6</td><td rowspan="1" colspan="1">246 963</td><td rowspan="1" colspan="1">193 317</td><td rowspan="1" colspan="1">1.28</td><td rowspan="1" colspan="1">N/A</td></tr><tr><td rowspan="1" colspan="1">HS7</td><td rowspan="1" colspan="1">193 436</td><td rowspan="1" colspan="1">120 863</td><td rowspan="1" colspan="1">1.60</td><td rowspan="1" colspan="1">TBC1D30, PLCH1, G2E3, PBX1, ESRRG, TIPARP, MAN1A1</td></tr><tr><td rowspan="1" colspan="1">HS8-1</td><td rowspan="1" colspan="1">91 952</td><td rowspan="1" colspan="1">72 746</td><td rowspan="1" colspan="1">1.26</td><td rowspan="1" colspan="1">N/A</td></tr><tr><td rowspan="1" colspan="1">HS8-2</td><td rowspan="1" colspan="1">92 748</td><td rowspan="1" colspan="1">81 201</td><td rowspan="1" colspan="1">1.14</td><td rowspan="1" colspan="1">N/A</td></tr><tr><td rowspan="1" colspan="1">HS9</td><td rowspan="1" colspan="1">309 580</td><td rowspan="1" colspan="1">125 640</td><td rowspan="1" colspan="1">2.46</td><td rowspan="1" colspan="1">BDNF, KCNS2, SHISAL1, GOLGA4, LGI1</td></tr><tr><td rowspan="1" colspan="1">HS10</td><td rowspan="1" colspan="1">751 154</td><td rowspan="1" colspan="1">150 617</td><td rowspan="1" colspan="1">4.99</td><td rowspan="1" colspan="1">PCHD20, TM4SF18, TMTC1, MCTP1, RBFOX1, TRHDE, ZMYM5, SEMA3E, MBNL2, LCOR, CXADR, CALB1, ARID2, GABRG2, DIRAS2, NEK7, FUT9</td></tr><tr><td rowspan="1" colspan="1">HS11</td><td rowspan="1" colspan="1">124 140</td><td rowspan="1" colspan="1">202 003</td><td rowspan="1" colspan="1">0.61</td><td rowspan="1" colspan="1">LEKR1, DLX1, AOX1, MYCN, PCDH18, DPYSL2, TSHZ1, PFKFB3, BCL9L</td></tr><tr><td rowspan="1" colspan="1">HS12</td><td rowspan="1" colspan="1">597 869</td><td rowspan="1" colspan="1">338 876</td><td rowspan="1" colspan="1">1.76</td><td rowspan="1" colspan="1">RBFOX1, RNF212, MBNL2, GRIN2A, CREG2, KRR1</td></tr><tr><td colspan="5" rowspan="1"><br /><br /></td></tr><tr><td rowspan="2" colspan="1">vd-sRNA</td><td colspan="2" rowspan="1">&#x003a3; read count</td><td rowspan="2" colspan="1">N/H ratio</td><td rowspan="2" colspan="1">predicted gene targets (miRDB)</td></tr><tr><td rowspan="1" colspan="1">N72_N</td><td rowspan="1" colspan="1">N72_H</td></tr><tr><td rowspan="1" colspan="1">HS1</td><td rowspan="1" colspan="1">5 594</td><td rowspan="1" colspan="1">243 019</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">no targets</td></tr><tr><td rowspan="1" colspan="1">HS2</td><td rowspan="1" colspan="1">26 642</td><td rowspan="1" colspan="1">169 885</td><td rowspan="1" colspan="1">0.16</td><td rowspan="1" colspan="1">no targets</td></tr><tr><td rowspan="1" colspan="1">HS3</td><td rowspan="1" colspan="1">39 241</td><td rowspan="1" colspan="1">159 523</td><td rowspan="1" colspan="1">0.25</td><td rowspan="1" colspan="1">SIAH3, NKAIN1, EBF1, KLHDC8A, ST6GAL1, DNMT3B, SOGA1, SHF, KLHL13</td></tr><tr><td rowspan="1" colspan="1">HS4</td><td rowspan="1" colspan="1">25 886</td><td rowspan="1" colspan="1">96 652</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1">PIK3C2B, EPHB1, NYNRIN, ELAVL2, SH3TC2, ZNF124, SLITRK5, SIRPA, ONECUT2, BDKRB2, ACACA, RTL8B, ZNF730</td></tr><tr><td rowspan="1" colspan="1">HS5</td><td rowspan="1" colspan="1">11 486</td><td rowspan="1" colspan="1">50 154</td><td rowspan="1" colspan="1">0.23</td><td rowspan="1" colspan="1">no targets</td></tr><tr><td rowspan="1" colspan="1">HS6</td><td rowspan="1" colspan="1">27 970</td><td rowspan="1" colspan="1">193 317</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">AKR1C2, NCAM1</td></tr><tr><td rowspan="1" colspan="1">HS7</td><td rowspan="1" colspan="1">22 292</td><td rowspan="1" colspan="1">120 863</td><td rowspan="1" colspan="1">0.18</td><td rowspan="1" colspan="1">TFAP2A, ELAVL4, BCL11B, POU3F4, AUTS2, MN1, SH3TC2, DACH1, ONECUT2, PIP4P2, TMEM229B</td></tr><tr><td rowspan="1" colspan="1">HS8-1</td><td rowspan="1" colspan="1">20 918</td><td rowspan="1" colspan="1">72 746</td><td rowspan="1" colspan="1">0.29</td><td rowspan="1" colspan="1">NFASC, NCAM1</td></tr><tr><td rowspan="1" colspan="1">HS8-2</td><td rowspan="1" colspan="1">22 262</td><td rowspan="1" colspan="1">81 201</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1">no targets</td></tr><tr><td rowspan="1" colspan="1">HS9</td><td rowspan="1" colspan="1">34 893</td><td rowspan="1" colspan="1">125 640</td><td rowspan="1" colspan="1">0.28</td><td rowspan="1" colspan="1">NAV1, GLRA2, GAS2L3</td></tr><tr><td rowspan="1" colspan="1">HS10</td><td rowspan="1" colspan="1">52 035</td><td rowspan="1" colspan="1">150 617</td><td rowspan="1" colspan="1">0.35</td><td rowspan="1" colspan="1">PRDM12, NRN1, ELAVL4, CDKN1C, ANOS1, NXPH2, PSRC1, PPM1L, CYYR1, LUZP2, LRRTM3</td></tr><tr><td rowspan="1" colspan="1">HS11</td><td rowspan="1" colspan="1">19 224</td><td rowspan="1" colspan="1">202 003</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">PTPN5, DCX, MYCN, DLX1, NREP, THSD7A, NXPH2, PIK3C2B, CAMK2A, EPHB1, RAPGEF5, PCDH18, DPYSL2, KCND2, GNG2, TSHZ1, KLHL13, ONECUT2, ISGF3, PFKFB3, BCL9L, PPP1R17</td></tr><tr><td rowspan="1" colspan="1">HS12</td><td rowspan="1" colspan="1">46 500</td><td rowspan="1" colspan="1">338 876</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">DACH1, NDRG4</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139861395223376"><a target="object" rel="noopener" href="/pmc/articles/PMC9167876/table/t0020/?report=objectonly">Open in a separate window</a></div></div></div><div id="s0120" class="sec"><h3 id="s0120title">3.4. Candidate gene networks responsible for distinct TBEV infection outcomes in neurons during infection with strains of varying severity</h3><p id="p0210" class="p p-first">The comparison of infection dynamics and cell death rate between severe Hypr and mild Neudoerfl strains in infected neurons confirmed a higher pathogenic effect for Hypr (<a href="/pmc/articles/PMC9167876/figure/f0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0005" rid-ob="ob-f0005" co-legend-rid="lgnd_f0005"><span>Fig. 1</span></a>). Therefore, observed differences between Hypr and Neudoerfl infection manifestation in human neuronal cells drove us to search for differences between N72_H and N72_N datasets on the pc-mRNA/miRNA/ncRNA level. Using an <em>in silico</em> approach, we employed an analogous pipeline to A72_H and N72_H datasets.</p><p id="p0215">We identified 1935 DE pc-mRNAs either uniquely up-/down-regulated in N72_H/N72_N datasets or differentially expressed (both up and down-regulated) between N72_H/N72_N datasets with a fold change higher than 1.5 log<sub>2</sub>. The GSEA analysis found significant differences in infection between the two TBEV strains in (i) the extent of the negative impact of Hypr infection on neuronal development, (ii) the level of Hypr-induced dysregulated activity of RNA polymerase II promoters, and (iii) higher activation of host immune response to Hypr infection (<a href="/pmc/articles/PMC9167876/figure/f0030/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0030" rid-ob="ob-f0030" co-legend-rid="lgnd_f0030"><span>Fig. 6</span></a>A).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f0030" co-legend-rid="lgnd_f0030"><a href="/pmc/articles/PMC9167876/figure/f0030/" target="figure" rid-figpopup="f0030" rid-ob="ob-f0030"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139861395459728"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9167876_gr6.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gr6.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC9167876/bin/gr6.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139861395459728"><a target="object" rel="noopener" href="/pmc/articles/PMC9167876/figure/f0030/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_f0030"><div><a class="figpopup" href="/pmc/articles/PMC9167876/figure/f0030/" target="figure" rid-figpopup="f0030" rid-ob="ob-f0030">Fig. 6</a></div><!--caption a7--><div class="caption"><p><strong>Comparison of TBEV Hypr/Neudoerfl infection in neurons reveals disruption of proper neural development. (A)</strong> Heatmap summarizing results from gene set enrichment analysis (DAVID tool; GO Biological processes; p&#x000a0;&#x0003c;&#x000a0;0.05) of DE genes 1) uniquely expressed in either N72_H or N72_N, and 2) genes expressed with &#x0003e;1.5 log<sub>2</sub> difference in one of the samples. <strong>(B)</strong> Number of mapped reads for vd-sRNAs (16&#x02013;50 nt) in the respective sample. Data summarise three independent experiments and values in graphs are expressed as mean&#x000a0;&#x000b1;&#x000a0;SD. <strong>(C)</strong> Frequency distribution of 21&#x02013;23 nt vd-sRNAs mapped to TBEV Hypr/Neudoerfl genome in N72_H (black) and N72_N (red) samples. HS &#x02013; hot-spot. (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this article.)</p></div></div></div><p id="p0220">We identified 1935 DE pc-mRNAs either uniquely up-/down-regulated in N72_H/N72_N datasets or differentially expressed (both up and down-regulated) between N72_H/N72_N datasets with a fold change higher than 1.5 log<sub>2</sub>. The GSEA analysis found significant differences in infection between the two TBEV strains in (i) the extent of the negative impact of Hypr infection on neuronal development, (ii) the level of Hypr-induced dysregulated activity of RNA polymerase II promoters, and (iii) higher activation of host immune response to Hypr infection (<a href="/pmc/articles/PMC9167876/figure/f0030/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0030" rid-ob="ob-f0030" co-legend-rid="lgnd_f0030"><span>Fig. 6</span></a>A).</p><p id="p0225">Identifying the complex regulatory network of small regulatory RNAs also requires the identification of their target sites. Thus, we selected DE expressed miRNAs and lncRNAs under the same conditions as the pc-mRNAs selection. Using the miRWalk and lncBase v.2 databases, the miRNA targets were identified for pc-mRNAs and lncRNAs, respectively. The predicted 268 <em>trans</em> targets for both small RNA classes (207 pc-mRNAs and 61 lncRNAs) are listed in <a href="#s0170" rid="s0170" class=" sec">Table S8</a>. The complete overview of the possible interactions was predicted by the identification of potential <em>cis</em> targets (100&#x000a0;kb upstream and downstream) for DE lncRNAs with log<sub>2</sub>-fold change&#x000a0;&#x0003e;4 or&#x000a0;&#x0003c;&#x02212;4 and lncRNAs identified as potential targets of DE miRNAs (<a href="#s0170" rid="s0170" class=" sec">Table S9</a>). In total, 47 cis targets were predicted, including transcription factors (e.g. <em>DLX5</em>, <em>DLX6</em>, <em>TFAP2A</em>) and various receptors (e.g. <em>CHRNA3</em>, <em>CHRNB4</em>, <em>DCC</em>, <em>PTPRH</em>).</p><p id="p0230">The assessment of vd-sRNA presence in TBEV Neudoerfl-infected cells revealed a similar pattern to Hypr vd-sRNA data &#x02013; increasing number of vd-sRNAs through the infection and higher numbers of sense reads (<a href="#s0170" rid="s0170" class=" sec">Fig. S4</a>). Comparison between N72_N and N72_H datasets further documented higher numbers of mapped reads during Hypr infection compared to Neudoerfl (<a href="/pmc/articles/PMC9167876/figure/f0030/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0030" rid-ob="ob-f0030" co-legend-rid="lgnd_f0030"><span>Fig. 6</span></a>B and C), which is in agreement with higher titres and viral gRNA levels (<a href="/pmc/articles/PMC9167876/figure/f0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0005" rid-ob="ob-f0005" co-legend-rid="lgnd_f0005"><span>Fig. 1</span></a>). The distribution pattern of 21&#x02013;23 nt long vd-sRNAs was identical for 12 hot-spots with higher abundance in N72_H samples (<a href="/pmc/articles/PMC9167876/table/t0015/" target="table" class="fig-table-link figpopup" rid-figpopup="t0015" rid-ob="ob-t0015" co-legend-rid=""><span>Table 3</span></a>; <a href="/pmc/articles/PMC9167876/figure/f0030/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0030" rid-ob="ob-f0030" co-legend-rid="lgnd_f0030"><span>Fig. 6</span></a>D). Prediction of host mRNA targets was performed using the miRDB tool in combination with DE pc-mRNA datasets with 66 identified host mRNA targets (<a href="/pmc/articles/PMC9167876/table/t0020/" target="table" class="fig-table-link figpopup" rid-figpopup="t0020" rid-ob="ob-t0020" co-legend-rid=""><span>Table 4</span></a>).</p><p id="p0235" class="p p-last">We further focused on HS11, the hot-spot with the highest number of mapped 21&#x02013;23 nt reads in N72_H samples. The vd-sRNA derived from HS11 was predicted to target 22 mRNAs with the corresponding expression pattern (N72_H&#x000a0;&#x0003c;&#x000a0;N72_N), out of which seven pc-mRNAs (<em>DLX1</em>, <em>MYCN</em>, <em>PCDH18</em>, <em>DPYSL2</em>, <em>TSHZ1</em>, <em>PFKFB3</em>, <em>BCL9L</em>) were also identified as less expressed in A72_H samples compared to N72_H. We suggest that these genes represent a candidate set of markers of increased pathogenicity that Hypr strain confers towards neurons.</p></div><div id="s0125" class="sec sec-last"><h3 id="s0125title">3.5. TBEV infection results in altered pre-mRNA splicing</h3><p id="p0240" class="p p-first-last">Both, neuron/astrocyte-specific response to infection and pathogenicity-related response to infection, led us to examine the potential impairment of host pre-mRNA splicing events since several lncRNAs and vd-sRNAs were predicted to interact with partners involved in this pathway. The idea was also supported by previous studies describing DENV- and ZIKV-induced alterations in host pre-mRNA splicing <a href="#b0135" rid="b0135" class=" bibr popnode">[27]</a>, <a href="#b0305" rid="b0305" class=" bibr popnode">[61]</a>, <a href="#b0310" rid="b0310" class=" bibr popnode">[62]</a>. Thus, RNA-seq data of astrocytes and neurons infected with TBEV were submitted to a differential splicing analysis using MAJIQ v2.2 <a href="#b0270" rid="b0270" class=" bibr popnode">[54]</a>. The corresponding mock controls were used as a reference for differential splicing calculation. Identified LSVs were further classified by MAJIQ to binary and complex events. Binary LSVs include exon skipping (ES), intron retention (IR), alternative 3&#x02032;/5&#x02032; splice site (A3SS/A5SS) and involve only two exons or two splice sites in the same exon. Complex LSVs combine several binary events originating from or targeting the same site. In total, differential splicing analysis revealed 4009 LSVs in all datasets (<a href="#s0170" rid="s0170" class=" sec">Table S10</a>) and the vast majority of detected LSVs (3639; 90.7 %) were documented for N72_H (2390; 59.6 %) and A72_H (1249; 31.2 %) datasets; interestingly, infection with Neudoerfl strain resulted in a total of only 94 (2.3 %) identified LSVs. The detailed characterization of identified LSVs revealed that exon skipping was the most frequent LSV type (<a href="/pmc/articles/PMC9167876/figure/f0035/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0035" rid-ob="ob-f0035" co-legend-rid="lgnd_f0035"><span>Fig. 7</span></a>B). The biological pathways affected by TBEV-induced differential splicing were determined using a GSEA analysis for A72_H, N72_H, and N72_N samples using the list of differentially spliced genes. Significantly enriched GO terms (biological process; Benjamini-Hochberg P-value&#x000a0;&#x0003c;&#x000a0;0.05) were identified only in the case of A72_H and N72_H samples (<a href="/pmc/articles/PMC9167876/figure/f0035/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0035" rid-ob="ob-f0035" co-legend-rid="lgnd_f0035"><span>Fig. 7</span></a>C). Only a single GO term &#x0201c;mRNA splicing process&#x0201d; was significantly enriched in both samples. The remaining biological processes were significantly enriched only in the case of N72_H. The most highly ranking terms were found among DNA repair, G2/M transition of mitotic cell cycle, protein transport, and membrane fusion biological processes. <em>MTUS2</em> and <em>CFAP61</em> were the top-ranked genes recognized in the list of TBEV-induced alterations in host pre-mRNA splicing in A72_H and N72_H samples, respectively. Therefore, we prepared sashimi plots of LSVs identified as significantly spliced in these two genes using the MAJIQ Voila tool (<a href="/pmc/articles/PMC9167876/figure/f0035/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0035" rid-ob="ob-f0035" co-legend-rid="lgnd_f0035"><span>Fig. 7</span></a>D). As a control, the visualization of bam files used for LSV identification in <em>MTUS2</em> and <em>CFAP61</em> genes was done using Integrated Genome Browser (IGB) v9.1.8 <a href="#b0355" rid="b0355" class=" bibr popnode">[71]</a> to support the MAJIQ LSV detection and quantification capacity. Both IGB graphs also support the presence of newly included exons in LSV detection by MAJIQ in these two genes in Hypr-infected cells (<a href="/pmc/articles/PMC9167876/figure/f0035/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0035" rid-ob="ob-f0035" co-legend-rid="lgnd_f0035"><span>Fig. 7</span></a>E).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f0035" co-legend-rid="lgnd_f0035"><a href="/pmc/articles/PMC9167876/figure/f0035/" target="figure" rid-figpopup="f0035" rid-ob="ob-f0035"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139861395406896"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9167876_gr7.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is gr7.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC9167876/bin/gr7.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139861395406896"><a target="object" rel="noopener" href="/pmc/articles/PMC9167876/figure/f0035/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_f0035"><div><a class="figpopup" href="/pmc/articles/PMC9167876/figure/f0035/" target="figure" rid-figpopup="f0035" rid-ob="ob-f0035">Fig. 7</a></div><!--caption a7--><div class="caption"><p><strong>TBEV infection induces changes in the splicing of host pre-mRNAs. (A)</strong> Graphical summary of identified LSVs (local splicing variants) in RNA-seq data using MAJIQ tool. Binary LSVs include exon skipping (ES), intron retention (IR), alternative 3&#x02032;/5&#x02032; splice site (A3SS/A5SS) and involve only two exons or two splice sites in the same exon. Complex LSVs combine several binary events originating from or targeting the same site. <strong>(B)</strong> Classification of binary and complex LSVs identified in TBEV-infected astrocytes and neurons (combined). <strong>(C)</strong> Heatmap summarizing results from gene set enrichment analysis (DAVID tool; GO Biological processes; p&#x000a0;&#x0003c;&#x000a0;0.05) of genes in which&#x000a0;&#x0003e;&#x000a0;1 LSV was identified (for samples A72_H, N72_H, and N72_N). <strong>(D)</strong> MAJIQ-generated Sashimi plots of top-ranked genes in N72_H (<em>CFAP61</em>) and A72_H (<em>MTUS2</em>) samples. Numbers above splicing event arcs represent the number of mapped junction reads. For the sake of readability, cropped areas with identified LSVs are shown. <strong>(E)</strong> Analysis of read depth distribution across <em>CFAP61</em> and <em>MTUS2</em> genes (areas with identified LSVs) in N72_H/M and A72_H/M samples, respectively, using the integrated genome browser.</p></div></div></div></div></div><div id="s0130" class="tsec sec"><h2 class="head no_bottom_margin" id="s0130title">4. Discussion</h2><p id="p0245" class="p p-first">TBEV, a neurotropic flavivirus, targets the host CNS and frequently causes severe encephalitis in infected patients. The exact mechanism responsible for TBEV-induced neuropathogenesis has not been fully understood to date, although several studies suggest the combination of virus-induced neuronal cell death and the immunopathogenic effect of activated host immune response [<a href="#b0050" rid="b0050" class=" bibr popnode">[10]</a>, <a href="#b0060" rid="b0060" class=" bibr popnode">[12]</a>]. The complexity of the whole process is further supported by the different outcomes of TBEV infection in neurons and astrocytes <a href="#b0025" rid="b0025" class=" bibr popnode">[5]</a>, <a href="#b0030" rid="b0030" class=" bibr popnode">[6]</a>, <a href="#b0035" rid="b0035" class=" bibr popnode">[7]</a>, <a href="#b0060" rid="b0060" class=" bibr popnode">[12]</a>. One of the suggested mechanisms underlying the distinct susceptibility of both cell types to TBEV infection is the neuron-/astrocyte-specific expression pattern of immune response-related genes <a href="#b0060" rid="b0060" class=" bibr popnode">[12]</a>, <a href="#b0065" rid="b0065" class=" bibr popnode">[13]</a>. However, both studies assayed only a limited number of pre-selected genes in a microarray-based analysis. Here, we present a thorough transcriptomic study describing differential expression of both small RNAs as well as poly-(A) RNAs in human neurons and astrocytes infected by mild and severe strains of TBEV in an early and late stage infection. Thus, the combination of all factors (infection time, cell type, TBEV strain) gives a complex overview of host-cell response at the RNA level and provides new significant data for a better understanding of TBEV neuropathogenesis.</p><div id="s0135" class="sec"><h3 id="s0135title">4.1. Cell-type specific response in neurons and astrocytes challenged by TBEV</h3><p id="p0250" class="p p-first"><em>In silico</em> pc-mRNA-miRNA-lncRNA interactome analysis identified major differences in response between TBEV challenged astrocytes and neurons. The key feature was the expression pattern of immune response-related genes (<a href="/pmc/articles/PMC9167876/figure/f0025/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0025" rid-ob="ob-f0025" co-legend-rid="lgnd_f0025"><span>Fig. 5</span></a>; <a href="#s0170" rid="s0170" class=" sec">Table S3</a>; <a href="#s0170" rid="s0170" class=" sec">Fig. S3</a>), which is in concordance with studies of Fares <em>et al</em>. and Pokorna <em>et al</em>. <a href="#b0060" rid="b0060" class=" bibr popnode">[12]</a>, <a href="#b0065" rid="b0065" class=" bibr popnode">[13]</a>. The phenomenon of cell type-specific immune response to flaviviral infection in CNS was also described for WNV <a href="#b0315" rid="b0315" class=" bibr popnode">[63]</a>, which further highlights its relevance for the flaviviral pathogenic effect. Besides, Hypr infection negatively affects extracellular matrix organization and RNAPII promoter activity pathways in infected neurons, which may also play a substantial role in the impairment of the neuronal tissue.</p><p id="p0255">The altered expression pattern of pc-mRNAs, which belong to the aforementioned pathways, could result from the miRNA/lncRNA-driven regulation process. Thus, we further focused on the linkage of the DE miRNAs/lncRNAs to their possible targets in the DE pc-mRNA datasets. The <em>in silico</em> prediction revealed several networks possibly involved in the differential expression pattern documented in astrocytes and neurons.</p><p id="p0260">Interferon signalling is considered a key regulatory cascade of the innate immune system triggering the expression of a wide panel of ISGs with a direct or indirect antiviral effect <a href="#b0320" rid="b0320" class=" bibr popnode">[64]</a>. Our data revealed that in addition to IFN-&#x003b2;, IFN-&#x003bb;1/2 expression is induced in both cell types in an early TBEV infection (24&#x000a0;h p.i.). However, only astrocytes maintained the IFN-&#x003bb;1/2 expression later during the infection (72&#x000a0;h p.i.). Interestingly, the IFN-&#x003bb; cluster was predicted to be a <em>cis</em> target regulated by <a id="__tag_1132452539" class="tag_hotlink" href="/nuccore/AC011445.2" ref="/nuccore/AC011445.2"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AC011445.2","term_id":"6604392","term_text":"AC011445.2"}}</span>AC011445.2</a> lncRNA (ENSG00000269246), which was highly up-regulated in A72_H, but not in N72_H samples (<a href="/pmc/articles/PMC9167876/figure/f0025/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0025" rid-ob="ob-f0025" co-legend-rid="lgnd_f0025"><span>Fig. 5</span></a>, <a href="#s0170" rid="s0170" class=" sec">Table S3</a>). Our previous study demonstrates that IFN-&#x003bb;1 is the predominantly expressed interferon in response to TBEV infection in cells of neuronal origin <a href="#b0140" rid="b0140" class=" bibr popnode">[28]</a>, and its significance was also demonstrated in the case of WNV infection <a href="#b0325" rid="b0325" class=" bibr popnode">[65]</a>. Thus, the deficiency of IFN-&#x003bb;1/2 production in neurons is a potential contributor to the decreased expression of specific ISGs, which are necessary for a successful antiviral response to TBEV in astrocytes. Similarly, astrocyte-specific up-regulation of <a id="__tag_1132452541" class="tag_hotlink" href="/nuccore/AL445490.1" ref="/nuccore/AL445490.1"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AL445490.1","term_id":"10803122","term_text":"AL445490.1"}}</span>AL445490.1</a> lncRNA (ENSG00000225886) may contribute to the elevated expression of <em>IFI6</em>, a predicted <em>cis</em>-target of this lncRNA (<a href="#s0170" rid="s0170" class=" sec">Table S7</a>).</p><p id="p0265">When we focused on the neural cell-type-specific response in the case of host miRNAs, the <em>hsa-miR-1298</em> is the only miRNA species which was significantly up-regulated in Hypr-infected neurons, while significantly down-regulated in Hypr-infected astrocytes (<a href="/pmc/articles/PMC9167876/figure/f0020/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0020" rid-ob="ob-f0020" co-legend-rid="lgnd_f0020"><span>Fig. 4</span></a>D). Among the predicted <em>hsa-miR-1298</em> mRNA targets with the corresponding DE profile (<em>LUZP2</em>, <em>SPSB4</em>, <em>VCAN</em>, <em>TFAP2B</em>, <em>MAST3</em>, <em>RSAD2</em>, <em>CMPK2</em>, and <em>DLX5</em>), the <em>RSAD2</em> and <em>CMPK2</em> we found the most important since both were documented to interfere with flavivirus infection <a href="#b0330" rid="b0330" class=" bibr popnode">[66]</a>. Additionally, <em>hsa-miR-7974</em>, the second most up-regulated miRNA in Hypr-infected neurons, was predicted to target OAS1, whose DE profile also showed a decreasing induction rate in Hypr-infected neurons and which has already been proven to possess strong antiviral activity against flaviviruses <a href="#b0335" rid="b0335" class=" bibr popnode">[67]</a>. Interestingly, a post mortem analysis of brain tissue from children with congenital ZIKV syndrome revealed elevated levels of <em>hsa-miR-145</em> and <em>hsa-miR-148a</em>
<a href="#b0340" rid="b0340" class=" bibr popnode">[68]</a>. Similarly, here we documented that <em>hsa-miR-145-5p</em> was up-regulated exclusively in Hypr- and Neudoerfl-infected neurons, whereas <em>hsa-miR-148a-5p</em> was up-regulated exclusively in Hypr- and Neudoerfl-infected astrocytes (<a href="/pmc/articles/PMC9167876/figure/f0020/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0020" rid-ob="ob-f0020" co-legend-rid="lgnd_f0020"><span>Fig. 4</span></a>).</p><p id="p0270" class="p p-last">Besides dysregulated host miRNAs, small RNAs derived from the viral genome may also interfere with host mRNA levels <a href="#b0185" rid="b0185" class=" bibr popnode">[37]</a>, <a href="#b0190" rid="b0190" class=" bibr popnode">[38]</a>. Our data document an accumulation of TBEV vd-sRNAs in infected neurons and astrocytes; the distribution pattern of vd-sRNAs across the TBEV genome was remarkably similar among the samples, with coverage rate as the only variable factor. Interestingly, in the case of TBEV Hypr-infected astrocytes, vd-sRNAs (21&#x02013;23 nt long) derived from the two hot-spots with the highest coverage (HS10, HS12) were both predicted to target <em>MBNL2</em> and <em>RBFOX1</em>. These genes are involved in pre-mRNA splicing regulation necessary for proper neural development <a href="#b0345" rid="b0345" class=" bibr popnode">[69]</a>, <a href="#b0350" rid="b0350" class=" bibr popnode">[70]</a>, <a href="#b0355" rid="b0355" class=" bibr popnode">[71]</a>, <a href="#b0360" rid="b0360" class=" bibr popnode">[72]</a>, and their depletion may thus contribute to the observed dysregulation of the splicing process in astrocytes. This hypothesis is supported by a finding that one of the <em>RBFOX1</em> targets, <em>SNAP25</em>
<a href="#b0345" rid="b0345" class=" bibr popnode">[69]</a>, was identified among differently spliced mRNAs in TBEV Hypr-infected astrocytes. Analogously, vd-sRNA derived from HS11 (hot-spot with the highest coverage in TBEV Hypr-infected neurons) was predicted to target 22 genes in total. However, only seven genes (<em>DLX1</em>, <em>MYCN</em>, <em>PCDH18</em>, <em>DPYSL2</em>, <em>TSHZ1</em>, <em>PFKFB3</em>, <em>BCL9L</em>) met the criterion of being expressed at a lower rate in N72_H in comparison to A72_H and N72_N samples. The dysregulation of mRNA levels through HS11-derived vd-sRNA for these genes could be thus responsible for the severe pathogenic effect in Hypr-infected neurons when compared to Hypr-infected astrocytes and Neudoerfl-infected neurons.</p></div><div id="s0140" class="sec"><h3 id="s0140title">4.2. Alternative splicing as an important co-factor of high TBEV Hypr virulence in neurons</h3><p id="p0275" class="p p-first">Based on the previous studies describing flavivirus-induced changes in the host pre-mRNA splicing process <a href="#b0305" rid="b0305" class=" bibr popnode">[61]</a>, <a href="#b0310" rid="b0310" class=" bibr popnode">[62]</a> in combination with observed dysregulation of various splicing factors upon TBEV infection (e.g., MBNL2, RBFOX1, or CELF3/4/5; <a href="#s0170" rid="s0170" class=" sec">Table S3</a>), we hypothesized that TBEV infection alters the splicing of host mRNAs. Indeed, the differential splicing analysis revealed significant alterations in TBEV-infected cells. Since more than half of differently spliced LSVs were identified in Hypr-infected neurons (64.5 %), we contemplate that this phenomenon may also contribute substantially to the high pathogenic effect of Hypr strain in infected neurons. This assumption is further corroborated by the list of affected significantly enriched biological pathways, including DNA repair, G2/M transition during the mitotic cycle, protein transport, and membrane fusion. Malfunction of these processes, especially DNA repair and G2/M checkpoint, results in an elevated apoptotic rate and neuronal dysfunction <a href="#b0365" rid="b0365" class=" bibr popnode">[73]</a>, <a href="#b0370" rid="b0370" class=" bibr popnode">[74]</a>.</p><p id="p0280" class="p p-last">Earlier, we reported a diverse rate of pathogenesis between TBEV Hypr and Neudoerfl strains in a human cell line of neuronal origin <a href="#b0280" rid="b0280" class=" bibr popnode">[56]</a>. Similar findings were documented in the human primary neurons in this work, where Hypr infection resulted in a substantially more severe pathogenic effect (<a href="/pmc/articles/PMC9167876/figure/f0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0005" rid-ob="ob-f0005" co-legend-rid="lgnd_f0005"><span>Fig. 1</span></a>C). Therefore, we tried to extrapolate the link between the intensity of pathogenesis, replication rate, and host transcriptional response. Even though both strains reached high titres in neurons, Hypr strain replication and progeny production were more efficient, generating almost 10-times more mature virions (<a href="/pmc/articles/PMC9167876/figure/f0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0005" rid-ob="ob-f0005" co-legend-rid="lgnd_f0005"><span>Fig. 1</span></a>B), gRNA (<a href="/pmc/articles/PMC9167876/figure/f0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0005" rid-ob="ob-f0005" co-legend-rid="lgnd_f0005"><span>Fig. 1</span></a>D), and vd-sRNAs (<a href="/pmc/articles/PMC9167876/figure/f0030/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0030" rid-ob="ob-f0030" co-legend-rid="lgnd_f0030"><span>Fig. 6</span></a>B-D). Moreover, a lower replication rate of the Neudoerfl strain correlates with low numbers of identified DE poly-(A) RNAs (<a href="/pmc/articles/PMC9167876/figure/f0015/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0015" rid-ob="ob-f0015" co-legend-rid="lgnd_f0015"><span>Fig. 3</span></a>B) and altered splicing events (<a href="/pmc/articles/PMC9167876/figure/f0035/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0035" rid-ob="ob-f0035" co-legend-rid="lgnd_f0035"><span>Fig. 7</span></a>A), which suggests a direct dependency on virus replication rate and intensity of the host response. Almost identical findings were described in the study of &#x000d6;verby <em>et al</em>., where the Hypr strain replicated with higher efficiency and induced a stronger immune response than the Neudoerfl strain <a href="#b0375" rid="b0375" class=" bibr popnode">[75]</a>. On the contrary, the study of Sessions <em>et al</em>. described an inverse trend, where an attenuated DENV1 strain caused three-fold more changes in the host transcriptome than the wild-type DENV1 <a href="#b0305" rid="b0305" class=" bibr popnode">[61]</a>.</p></div><div id="s0145" class="sec sec-last"><h3 id="s0145title">4.3. Diverse strain virulence in neurons</h3><p id="p0285" class="p p-first-last">The lower replication and pathogenesis rates observed for the mild Neudoerfl strain can also be perceived as a consequence of a more successful host response. However, Neudoerfl-infected cells showed a lower rate of immune response activation in the case of pc-mRNAs (<a href="#s0170" rid="s0170" class=" sec">Table S3</a> and <a href="#s0170" rid="s0170" class=" sec">Fig. S3</a>). Thus, the dysregulation of miRNA with proviral or antiviral properties may be the factor involved. We, therefore, compared the list of DE miRNAs in N72_H and N72_N samples with the list of experimentally verified proviral and antiviral miRNAs in the case of ZIKV <a href="#b0380" rid="b0380" class=" bibr popnode">[76]</a> and DENV <a href="#b0385" rid="b0385" class=" bibr popnode">[77]</a>. However, no corresponding matches were found. We, therefore, searched for novel antiviral miRNAs targeting Hypr or Neudoerfl gRNA using the miRDB tool. The <em>hsa-miR-592</em> was identified as the only miRNA species targeting Hypr and Neudoerfl gRNA. At the same time, this miRNA was up-regulated in samples with lower TBEV titres (A72_H and N72_N), but not in samples with higher TBEV titres (N72_H) (<a href="/pmc/articles/PMC9167876/figure/f0020/" target="figure" class="fig-table-link figpopup" rid-figpopup="f0020" rid-ob="ob-f0020" co-legend-rid="lgnd_f0020"><span>Fig. 4</span></a>). These correlations thus suppose that <em>hsa-miR-592</em> may negatively regulate TBEV replication in human neurons and astrocytes. Except for the direct interaction with TBEV gRNA, <em>hsa-miR-592</em> may indirectly affect TBEV replication as well. He <em>et al</em>. <a href="#b0390" rid="b0390" class=" bibr popnode">[78]</a> experimentally proved that <em>hsa-miR-592</em> targets SPRY2 mRNA, which is a negative regulator of interferon signalling <a href="#b0395" rid="b0395" class=" bibr popnode">[79]</a>, and in our study was shown to be up-regulated only in N72_H samples. We thus identified a new potential circuit of host cell countermeasure against TBEV involving <em>hsa-miR-592</em> unique among flaviviruses.</p></div></div><div id="s0150" class="tsec sec"><h2 class="head no_bottom_margin" id="s0150title">5. Conclusions</h2><p id="p0290" class="p p-first-last">Altogether, we have characterized the alterations in poly-(A) RNA and small RNA expression profile of human neurons and astrocytes upon TBEV infection using two TBEV strains of distinctive virulence (mild Neudoerfl and severe Hypr). Subsequent integrative <em>in silico</em> analysis of miRNA/mRNA/lncRNA/vd-sRNA networks and pre-mRNA splicing found significant changes in inflammatory and immune response pathways, nervous system development and regulation of mitosis in TBEV Hypr-infected neurons. Candidate mechanisms include specific regulation of host mRNA levels via differentially expressed miRNAs/lncRNAs or vd-sRNAs mimicking endogenous miRNAs and virus-driven modulation of host pre-mRNA splicing. Thus, our data provide a valuable source of information for any research that aims to investigate and further characterise the mechanism of TBEV-host interactions and the related process of TBEV pathogenesis. Moreover, strain-specific expression pattern of selected host genes also represents a list of potential biomarkers, which can be used for improved TBEV diagnostics.</p></div><div id="sec-a.ai.f" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.ai.ftitle">Declaration of Competing Interest</h2><p id="p0295" class="p p-first-last">The authors declare that they have no known competing financial interests or personal relationships that could have appeared to influence the work reported in this paper.</p></div><div id="ak005" class="tsec sec"><h2 class="head no_bottom_margin" id="ak005title">Acknowledgements</h2><div class="sec"><div id="s0155" class="sec sec-first"><h3 id="s0155title"></h3><p id="p0300" class="p p-first-last">We thank dr. R. Randall (University of St. Andrews, UK) for providing us with A549 cell line and dr. M. Bloom (National Institute of Allergy and Infectious Diseases, USA) for NS3 antibodies.</p></div><div id="s0160" class="sec sec-last"><h3 id="s0160title">Funding</h3><p id="p0305" class="p p-first-last">This research was supported by the Ministry of Education, Youth, and Sports of the Czech Republic INTER-ACTION (projects LTARF 18021 and LTAUSA 18040), and the Czech Science Foundation (GA18-27204S).</p></div></div></div><div id="fn-group-a.aj.d" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.aj.dtitle">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="s0165"><sup>Appendix A</sup>Supplementary data to this article can be found online at <a href="https://doi.org/10.1016/j.csbj.2022.05.052" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://doi.org/10.1016/j.csbj.2022.05.052</a>.</p></div></div><div id="s0170" class="tsec bk-sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="s0170title">Appendix A. Supplementary data</h2><!--/article/back/sec/--><p id="p0315" class="p p-first">The following are the Supplementary data to this article:<!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--></p><div class="fig  iconblock anchored whole_rhythm" id="f0045"><h3>Supplementary figure 1</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139861397059936"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9167876_fx1.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fx1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC9167876/bin/fx1.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139861397059936"><a target="object" rel="noopener" href="/pmc/articles/PMC9167876/figure/f0045/?report=objectonly">Open in a separate window</a></div></div><p></p><p id="p0320">
<!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--></p><div class="fig  iconblock anchored whole_rhythm" id="f0050"><h3>Supplementary figure 2</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139861397057104"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9167876_fx2.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fx2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC9167876/bin/fx2.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139861397057104"><a target="object" rel="noopener" href="/pmc/articles/PMC9167876/figure/f0050/?report=objectonly">Open in a separate window</a></div></div><p>
</p><p id="p0325">
<!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--></p><div class="fig  iconblock anchored whole_rhythm" id="f0055"><h3>Supplementary figure 3</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139861397054144"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9167876_fx3.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fx3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC9167876/bin/fx3.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139861397054144"><a target="object" rel="noopener" href="/pmc/articles/PMC9167876/figure/f0055/?report=objectonly">Open in a separate window</a></div></div><p>
</p><p id="p0330">
<!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--></p><div class="fig  iconblock anchored whole_rhythm" id="f0060"><h3>Supplementary figure 4</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139861397051280"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9167876_fx4.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fx4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC9167876/bin/fx4.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139861397051280"><a target="object" rel="noopener" href="/pmc/articles/PMC9167876/figure/f0060/?report=objectonly">Open in a separate window</a></div></div><p>
</p><p id="p0335">
<!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--></p><div class="fig  iconblock anchored whole_rhythm" id="f0065"><h3>Supplementary figure 5</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139861397048416"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9167876_fx5.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fx5.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC9167876/bin/fx5.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139861397048416"><a target="object" rel="noopener" href="/pmc/articles/PMC9167876/figure/f0065/?report=objectonly">Open in a separate window</a></div></div><p>
</p><p id="p0340">
</p><div class="sec suppmat" id="m0050"><!--caption a9--><strong>Supplementary data 1:</strong><div class="sup-box half_rhythm" id="media-a.aj.b.h.a.b"><a href="/pmc/articles/PMC9167876/bin/mmc1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(225K, pdf)</sup></div></div><p>
</p><p id="p0345">
</p><div class="sec suppmat" id="m0045"><!--caption a9--><strong>Supplementary data 2:</strong><div class="sup-box half_rhythm" id="media-a.aj.b.i.a.b"><a href="/pmc/articles/PMC9167876/bin/mmc2.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(11K, xlsx)</sup></div></div><p>
</p><p id="p0350">
</p><div class="sec suppmat" id="m0040"><!--caption a9--><strong>Supplementary data 3:</strong><div class="sup-box half_rhythm" id="media-a.aj.b.j.a.b"><a href="/pmc/articles/PMC9167876/bin/mmc3.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(292K, xlsx)</sup></div></div><p>
</p><p id="p0355">
</p><div class="sec suppmat" id="m0035"><!--caption a9--><strong>Supplementary data 4:</strong><div class="sup-box half_rhythm" id="media-a.aj.b.k.a.b"><a href="/pmc/articles/PMC9167876/bin/mmc4.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(9.1K, xlsx)</sup></div></div><p>
</p><p id="p0360">
</p><div class="sec suppmat" id="m0030"><!--caption a9--><strong>Supplementary data 5:</strong><div class="sup-box half_rhythm" id="media-a.aj.b.l.a.b"><a href="/pmc/articles/PMC9167876/bin/mmc5.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(18K, xlsx)</sup></div></div><p>
</p><p id="p0365">
</p><div class="sec suppmat" id="m0025"><!--caption a9--><strong>Supplementary data 6:</strong><div class="sup-box half_rhythm" id="media-a.aj.b.m.a.b"><a href="/pmc/articles/PMC9167876/bin/mmc6.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(12K, xlsx)</sup></div></div><p>
</p><p id="p0370">
</p><div class="sec suppmat" id="m0020"><!--caption a9--><strong>Supplementary data 7:</strong><div class="sup-box half_rhythm" id="media-a.aj.b.n.a.b"><a href="/pmc/articles/PMC9167876/bin/mmc7.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(11K, xlsx)</sup></div></div><p>
</p><p id="p0375">
</p><div class="sec suppmat" id="m0015"><!--caption a9--><strong>Supplementary data 8:</strong><div class="sup-box half_rhythm" id="media-a.aj.b.o.a.b"><a href="/pmc/articles/PMC9167876/bin/mmc8.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(13K, xlsx)</sup></div></div><p>
</p><p id="p0380">
</p><div class="sec suppmat" id="m0010"><!--caption a9--><strong>Supplementary data 9:</strong><div class="sup-box half_rhythm" id="media-a.aj.b.p.a.b"><a href="/pmc/articles/PMC9167876/bin/mmc9.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(11K, xlsx)</sup></div></div><p>
</p><p id="p0385" class="p p-last">
</p><div class="sec suppmat" id="m0005"><!--caption a9--><strong>Supplementary data 10:</strong><div class="sup-box half_rhythm" id="media-a.aj.b.q.a.b"><a href="/pmc/articles/PMC9167876/bin/mmc10.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.0M, xlsx)</sup></div></div><p>
</p></div><div id="bi005" class="tsec sec"><h2 class="head no_bottom_margin" id="bi005title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="b0005">1. <span class="mixed-citation" id="h0005">Gritsun TS, Lashkevich VA, Gould EA. Tick-borne encephalitis. Antiviral research. 2003;57(1-2):129-46. Epub 2003/03/05. doi: 10.1016/s0166-3542(02)00206-PubMed PMID: 12615309. [<a href="https://pubmed.ncbi.nlm.nih.gov/12615309" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0010">2. <span class="mixed-citation" id="h0010">de Graaf JA, Reimerink JH, Voorn GP, Bij de Vaate EA, de Vries A, Rockx B, et al. First human case of tick-borne encephalitis virus infection acquired in the Netherlands, July 2016. Euro surveillance : bulletin Europeen sur les maladies transmissibles = European communicable disease bulletin. 2016;21(33). Epub 2016/08/27. doi: 10.2807/1560-7917.Es .2016.21.33.30318. PubMed PMID: 27562931; PubMed Central PMCID: PMCPMC4998423. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4998423/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27562931" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0015">3. <span class="mixed-citation" id="h0015">Neufeldt CJ, Cortese M, Acosta EG, Bartenschlager R. Rewiring cellular networks by members of the Flaviviridae family. Nature reviews Microbiology. 2018;16(3):125-42. Epub 2018/02/1doi: 10.1038/nrmicro.2017.170. PubMed PMID: 29430005; PubMed Central PMCID: PMCPMC7097628. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7097628/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29430005" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0020">4. <span class="mixed-citation" id="h0020">Gelpi E, Preusser M, Garzuly F, Holzmann H, Heinz FX, Budka H. Visualization of Central European tick-borne encephalitis infection in fatal human cases. Journal of neuropathology and experimental neurology. 2005;64(6):506-12. Epub 2005/06/28. doi: 10.1093/jnen/64.6.506. PubMed PMID: 15977642. [<a href="https://pubmed.ncbi.nlm.nih.gov/15977642" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0025">5. <span class="mixed-citation" id="h0025">B&#x000ed;l&#x000fd; T, Palus M, Eyer L, Elsterov&#x000e1; J, Vancov&#x000e1; M, R&#x0016f;&#x0017e;ek D. Electron Tomography Analysis of Tick-Borne Encephalitis Virus Infection in Human Neurons. Scientific reports. 2015;5:1074Epub 2015/06/16. doi: 10.1038/srep1074PubMed PMID: 26073783; PubMed Central PMCID: PMCPMC4466586. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4466586/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26073783" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0030">6. <span class="mixed-citation" id="h0030">Potokar M, Korva M, Jorga&#x0010d;evski J, Av&#x00161;i&#x0010d;-&#x0017d;upanc T, Zorec R. Tick-borne encephalitis virus infects rat astrocytes but does not affect their viability. PloS one. 2014;9(1):e86219. Epub 2014/01/28. doi: 10.1371/journal.pone.0086219. PubMed PMID: 24465969; PubMed Central PMCID: PMCPMC3896472. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3896472/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24465969" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0035">7. <span class="mixed-citation" id="h0035">Palus M, B&#x000ed;l&#x000fd; T, Elsterov&#x000e1; J, Langhansov&#x000e1; H, Sal&#x000e1;t J, Vancov&#x000e1; M, et al. Infection and injury of human astrocytes by tick-borne encephalitis virus. The Journal of general virology. 2014;95(Pt 11):2411-26. Epub 2014/07/09. doi: 10.1099/vir.0.068411-0. PubMed PMID: 25000960. [<a href="https://pubmed.ncbi.nlm.nih.gov/25000960" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0040">8. <span class="mixed-citation" id="h0040">Sofroniew MV, Vinters HV. Astrocytes: biology and pathology. Acta neuropathologica. 2010;119(1):7-35. Epub 2009/12/17. doi: 10.1007/s00401-009-0619-PubMed PMID: 20012068; PubMed Central PMCID: PMCPMC2799634. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2799634/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20012068" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0045">9. <span class="element-citation" id="h0045">Diniz J.A., Da Rosa A.P., Guzman H., Xu F., Xiao S.Y., Popov V.L., et al. West Nile virus infection of primary mouse neuronal and neuroglial cells: the role of astrocytes in chronic infection. <span><span class="ref-journal">The American journal of tropical medicine and hygiene. </span>2006;<span class="ref-vol">75</span>(4):691–696.</span> Epub 2006/10/14 PubMed PMID: 17038696. [<a href="https://pubmed.ncbi.nlm.nih.gov/17038696" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+American+journal+of+tropical+medicine+and+hygiene&amp;title=West+Nile+virus+infection+of+primary+mouse+neuronal+and+neuroglial+cells:+the+role+of+astrocytes+in+chronic+infection&amp;author=J.A.+Diniz&amp;author=A.P.+Da+Rosa&amp;author=H.+Guzman&amp;author=F.+Xu&amp;author=S.Y.+Xiao&amp;volume=75&amp;issue=4&amp;publication_year=2006&amp;pages=691-696&amp;pmid=17038696&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b0050">10. <span class="mixed-citation" id="h0050">van Marle G, Antony J, Ostermann H, Dunham C, Hunt T, Halliday W, et al. West Nile virus-induced neuroinflammation: glial infection and capsid protein-mediated neurovirulence. Journal of virology. 2007;81(20):10933-49. Epub 2007/08/03. doi: 10.1128/jvi.02422-06. PubMed PMID: 17670819; PubMed Central PMCID: PMCPMC2045515. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2045515/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17670819" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0055">11. <span class="mixed-citation" id="h0055">Jorga&#x0010d;evski J, Korva M, Potokar M, Lisjak M, Av&#x00161;i&#x0010d;-&#x0017d;upanc T, Zorec R. ZIKV Strains Differentially Affect Survival of Human Fetal Astrocytes versus Neurons and Traffic of ZIKV-Laden Endocytotic Compartments. Scientific reports. 2019;9(1):8069. Epub 2019/05/31. doi: 10.1038/s41598-019-44559-8. PubMed PMID: 31147629; PubMed Central PMCID: PMCPMC6542792. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6542792/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31147629" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0060">12. <span class="mixed-citation" id="h0060">Fares M, Cochet-Bernoin M, Gonzalez G, Montero-Menei CN, Blanchet O, Benchoua A, et al. Pathological modeling of TBEV infection reveals differential innate immune responses in human neurons and astrocytes that correlate with their susceptibility to infection. Journal of neuroinflammation. 2020;17(1):76. Epub 2020/03/05. doi: 10.1186/s12974-020-01756-x. PubMed PMID: 32127025; PubMed Central PMCID: PMCPMC7053149. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7053149/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32127025" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0065">13. <span class="mixed-citation" id="h0065">Pokorna Formanova P, Palus M, Salat J, H&#x000f6;nig V, Stefanik M, Svoboda P, et al. Changes in cytokine and chemokine profiles in mouse serum and brain, and in human neural cells, upon tick-borne encephalitis virus infection. Journal of neuroinflammation. 2019;16(1):205. Epub 2019/11/09. doi: 10.1186/s12974-019-1596-z. PubMed PMID: 31699097; PubMed Central PMCID: PMCPMC6839073. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6839073/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31699097" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0070">14. <span class="mixed-citation" id="h0070">O'Brien J, Hayder H, Zayed Y, Peng C. Overview of MicroRNA Biogenesis, Mechanisms of Actions, and Circulation. Frontiers in endocrinology. 2018;9:402. Epub 2018/08/21. doi: 10.3389/fendo.2018.00402. PubMed PMID: 30123182; PubMed Central PMCID: PMCPMC6085463. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6085463/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30123182" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0075">15. <span class="mixed-citation" id="h0075">Kumar M, Nerurkar VR. Integrated analysis of microRNAs and their disease related targets in the brain of mice infected with West Nile virus. Virology. 2014;452-453:143-51. Epub 2014/03/13. doi: 10.1016/j.virol.2014.01.004. PubMed PMID: 24606691; PubMed Central PMCID: PMCPMC3959158. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3959158/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24606691" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0080">16. <span class="mixed-citation" id="h0080">Slonchak A, Shannon RP, Pali G, Khromykh AA. Human MicroRNA miR-532-5p Exhibits Antiviral Activity against West Nile Virus via Suppression of Host Genes SESTD1 and TAB3 Required for Virus Replication. Journal of virology. 2015;90(5):2388-402. Epub 2015/12/18. doi: 10.1128/jvi.02608-15. PubMed PMID: 26676784; PubMed Central PMCID: PMCPMC4810706. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4810706/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26676784" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0085">17. <span class="mixed-citation" id="h0085">Rastogi M, Srivastava N, Singh SK. Exploitation of microRNAs by Japanese Encephalitis virus in human microglial cells. Journal of medical virology. 2018;90(4):648-54. Epub 2017/11/18. doi: 10.1002/jmv.24995. PubMed PMID: 29149532. [<a href="https://pubmed.ncbi.nlm.nih.gov/29149532" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0090">18. <span class="mixed-citation" id="h0090">Dang JW, Tiwari SK, Qin Y, Rana TM. Genome-wide Integrative Analysis of Zika-Virus-Infected Neuronal Stem Cells Reveals Roles for MicroRNAs in Cell Cycle and Stemness. Cell reports. 2019;27(12):3618-28.e5. Epub 2019/06/20. doi: 10.1016/j.celrep.2019.05.059. PubMed PMID: 31216479; PubMed Central PMCID: PMCPMC6687627. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6687627/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31216479" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0095">19. <span class="mixed-citation" id="h0095">Seong RK, Lee JK, Cho GJ, Kumar M, Shin OS. mRNA and miRNA profiling of Zika virus-infected human umbilical cord mesenchymal stem cells identifies miR-142-5p as an antiviral factor. Emerging microbes &#x00026; infections. 2020;9(1):2061-75. Epub 2020/09/10. doi: 10.1080/22221751.2020.1821581. PubMed PMID: 32902370; PubMed Central PMCID: PMCPMC7534337. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7534337/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32902370" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0100">20. <span class="element-citation" id="h0100">Hazra B., Kumawat K.L., Basu A. The host microRNA miR-301a blocks the IRF1-mediated neuronal innate immune response to Japanese encephalitis virus infection. <span><span class="ref-journal">Sci Signaling. </span>2017;10(466):eaaf5185.;<span class="ref-vol">Epub 2017/02/16</span> doi: 10.1126/scisignal.aaf5185.</span> PubMed PMID: 28196914. [<a href="https://pubmed.ncbi.nlm.nih.gov/28196914" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscisignal.aaf5185" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci+Signaling&amp;title=The+host+microRNA+miR-301a+blocks+the+IRF1-mediated+neuronal+innate+immune+response+to+Japanese+encephalitis+virus+infection&amp;author=B.+Hazra&amp;author=K.L.+Kumawat&amp;author=A.+Basu&amp;volume=Epub+2017/02/16&amp;publication_year=2017;10(466):eaaf5185.&amp;doi=10.1126/scisignal.aaf5185&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b0105">21. <span class="mixed-citation" id="h0105">Wu N, Gao N, Fan D, Wei J, Zhang J, An J. miR-223 inhibits dengue virus replication by negatively regulating the microtubule-destabilizing protein STMN1 in EAhy926&#x000a0;cells. Microbes and infection. 2014;16(11):911-22. Epub 2014/09/03. doi: 10.1016/j.micinf.2014.08.011. PubMed PMID: 25181337; PubMed Central PMCID: PMCPMC7110837. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7110837/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25181337" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0110">22. <span class="mixed-citation" id="h0110">Diosa-Toro M, Echavarr&#x000ed;a-Consuegra L, Flipse J, Fern&#x000e1;ndez GJ, Kluiver J, van den Berg A, et al. MicroRNA profiling of human primary macrophages exposed to dengue virus identifies miRNA-3614-5p as antiviral and regulator of ADAR1 expression. PLoS neglected tropical diseases. 2017;11(10):e0005981. Epub 2017/10/19. doi: 10.1371/journal.pntd.0005981. PubMed PMID: 29045406; PubMed Central PMCID: PMCPMC5662241. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5662241/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29045406" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0115">23. <span class="mixed-citation" id="h0115">Cai W, Pan Y, Cheng A, Wang M, Yin Z, Jia R. Regulatory Role of Host MicroRNAs in Flaviviruses Infection. Frontiers in microbiology. 2022;13:869441. Epub 2022/04/29. doi: 10.3389/fmicb.2022.869441. PubMed PMID: 35479613; PubMed Central PMCID: PMCPMC9036177. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9036177/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35479613" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0120">24. <span class="mixed-citation" id="h0120">Statello L, Guo CJ, Chen LL, Huarte M. Gene regulation by long non-coding RNAs and its biological functions. Nature reviews Molecular cell biology. 2021;22(2):96-118. Epub 2020/12/doi: 10.1038/s41580-020-00315-9. PubMed PMID: 33353982; PubMed Central PMCID: PMCPMC7754182. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7754182/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33353982" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0125">25. <span class="mixed-citation" id="h0125">Bhattacharyya S, Vrati S. The Malat1 long non-coding RNA is upregulated by signalling through the PERK axis of unfolded protein response during flavivirus infection. Scientific reports. 2015;5:17794. Epub 2015/12/05. doi: 10.1038/srep17794. PubMed PMID: 26634309; PubMed Central PMCID: PMCPMC4669524. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4669524/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26634309" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0130">26. <span class="mixed-citation" id="h0130">Wang XJ, Jiang SC, Wei HX, Deng SQ, He C, Peng HJ. The Differential Expression and Possible Function of Long Noncoding RNAs in Liver Cells Infected by Dengue Virus. The American journal of tropical medicine and hygiene. 2017;97(6):1904-12. Epub 2017/10/11. doi: 10.4269/ajtmh.17-0307. PubMed PMID: 29016307; PubMed Central PMCID: PMCPMC5805055. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5805055/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29016307" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0135">27. <span class="mixed-citation" id="h0135">Hu B, Huo Y, Yang L, Chen G, Luo M, Yang J, et al. ZIKV infection effects changes in gene splicing, isoform composition and lncRNA expression in human neural progenitor cells. Virology journal. 2017;14(1):217. Epub 2017/11/09. doi: 10.1186/s12985-017-0882-6. PubMed PMID: 29116029; PubMed Central PMCID: PMCPMC5688814. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5688814/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29116029" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0140">28. <span class="mixed-citation" id="h0140">Selinger M, Wilkie GS, Tong L, Gu Q, Schnettler E, Grubhoffer L, et al. Analysis of tick-borne encephalitis virus-induced host responses in human cells of neuronal origin and interferon-mediated protection. The Journal of general virology. 2017;98(8):2043-60. Epub 2017/08/09. doi: 10.1099/jgv.0.000853. PubMed PMID: 28786780; PubMed Central PMCID: PMCPMC5817271. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5817271/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28786780" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0145">29. <span class="mixed-citation" id="h0145">Zhong XL, Liao XM, Shen F, Yu HJ, Yan WS, Zhang YF, et al. Genome-wide profiling of mRNA and lncRNA expression in dengue fever and dengue hemorrhagic fever. FEBS open bio. 2019;9(3):468-77. Epub 2019/03/15. doi: 10.1002/2211-5463.12576. PubMed PMID: 30868055; PubMed Central PMCID: PMCPMC6396354. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6396354/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30868055" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0150">30. <span class="mixed-citation" id="h0150">Wang P, Xu J, Wang Y, Cao X. An interferon-independent lncRNA promotes viral replication by modulating cellular metabolism. Science (New York, NY). 2017;358(6366):1051-5. Epub 2017/10/28. doi: 10.1126/science.aao0409. PubMed PMID: 29074580. [<a href="https://pubmed.ncbi.nlm.nih.gov/29074580" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0155">31. <span class="mixed-citation" id="h0155">Barriocanal M, Carnero E, Segura V, Fortes P. Long Non-Coding RNA BST2/BISPR is Induced by IFN and Regulates the Expression of the Antiviral Factor Tetherin. Frontiers in immunology. 2014;5:655. Epub 2015/01/27. doi: 10.3389/fimmu.2014.00655. PubMed PMID: 25620967; PubMed Central PMCID: PMCPMC4288319. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4288319/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25620967" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0160">32. <span class="mixed-citation" id="h0160">Kambara H, Niazi F, Kostadinova L, Moonka DK, Siegel CT, Post AB, et al. Negative regulation of the interferon response by an interferon-induced long non-coding RNA. Nucleic acids research. 2014;42(16):10668-80. Epub 2014/08/15. doi: 10.1093/nar/gku713. PubMed PMID: 25122750; PubMed Central PMCID: PMCPMC4176326. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4176326/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25122750" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0165">33. <span class="element-citation" id="h0165">Qian X., Xu C., Zhao P., Qi Z. Long non-coding RNA GAS5 inhibited hepatitis C virus replication by binding viral NS3 protein. <span><span class="ref-journal">Virology. </span>2016;<span class="ref-vol">Epub 2016/03/08</span>(492):155–165. doi: 10.1016/j.virol.2016.02.020.</span> PubMed PMID: 26945984. [<a href="https://pubmed.ncbi.nlm.nih.gov/26945984" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.virol.2016.02.020" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Virology&amp;title=Long+non-coding+RNA+GAS5+inhibited+hepatitis+C+virus+replication+by+binding+viral+NS3+protein&amp;author=X.+Qian&amp;author=C.+Xu&amp;author=P.+Zhao&amp;author=Z.+Qi&amp;volume=Epub+2016/03/08&amp;issue=492&amp;publication_year=2016&amp;pages=155-165&amp;doi=10.1016/j.virol.2016.02.020&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b0170">34. <span class="mixed-citation" id="h0170">Schnettler E, Tykalov&#x000e1; H, Watson M, Sharma M, Sterken MG, Obbard DJ, et al. Induction and suppression of tick cell antiviral RNAi responses by tick-borne flaviviruses. Nucleic acids research. 2014;42(14):9436-46. Epub 2014/07/24. doi: 10.1093/nar/gku657. PubMed PMID: 25053841; PubMed Central PMCID: PMCPMC4132761. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4132761/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25053841" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0175">35. <span class="mixed-citation" id="h0175">Schirtzinger EE, Andrade CC, Devitt N, Ramaraj T, Jacobi JL, Schilkey F, et al. Repertoire of virus-derived small RNAs produced by mosquito and mammalian cells in response to dengue virus infection. Virology. 2015;476:54-60. Epub 2014/12/22. doi: 10.1016/j.virol.2014.11.019. PubMed PMID: 25528416; PubMed Central PMCID: PMCPMC4323773. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4323773/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25528416" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0180">36. <span class="mixed-citation" id="h0180">Schuster S, Miesen P, van Rij RP. Antiviral RNAi in Insects and Mammals: Parallels and Differences. Viruses. 2019;11(5). Epub 2019/05/19. doi: 10.3390/v11050448. PubMed PMID: 31100912; PubMed Central PMCID: PMCPMC6563508. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6563508/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31100912" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0185">37. <span class="mixed-citation" id="h0185">Li X, Fu Z, Liang H, Wang Y, Qi X, Ding M, et al. H5N1 influenza virus-specific miRNA-like small RNA increases cytokine production and mouse mortality via targeting poly(rC)-binding protein 2. Cell research. 2018;28(2):157-71. Epub 2018/01/13. doi: 10.1038/cr.2018.3. PubMed PMID: 29327729; PubMed Central PMCID: PMCPMC5799819. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5799819/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29327729" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0190">38. <span class="mixed-citation" id="h0190">Ouellet DL, Vigneault-Edwards J, L&#x000e9;tourneau K, Gobeil LA, Plante I, Burnett JC, et al. Regulation of host gene expression by HIV-1 TAR microRNAs. Retrovirology. 2013;10:86. Epub 2013/08/14. doi: 10.1186/1742-4690-10-86. PubMed PMID: 23938024; PubMed Central PMCID: PMCPMC3751525. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3751525/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23938024" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0195">39. <span class="mixed-citation" id="h0195">Heinz FX, Kunz C. Homogeneity of the structural glycoprotein from European isolates of tick-borne encephalitis virus: comparison with other flaviviruses. The Journal of general virology. 1981;57(Pt 2):263-74. Epub 1981/12/01. doi: 10.1099/0022-1317-57-2-263. PubMed PMID: 6172553. [<a href="https://pubmed.ncbi.nlm.nih.gov/6172553" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0200">40. <span class="element-citation" id="h0200">Pospisil L., Jandasek L., Pesek J. Isolation of new strains of meningoencephalitis virus in the Brno region during the summer of 1953. <span><span class="ref-journal">Lekarske listy. </span>1954;<span class="ref-vol">9</span>(1):3–5.</span> Epub 1954/01/01 PubMed PMID: 13131921. [<a href="https://pubmed.ncbi.nlm.nih.gov/13131921" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lekarske+listy&amp;title=Isolation+of+new+strains+of+meningoencephalitis+virus+in+the+Brno+region+during+the+summer+of+1953&amp;author=L.+Pospisil&amp;author=L.+Jandasek&amp;author=J.+Pesek&amp;volume=9&amp;issue=1&amp;publication_year=1954&amp;pages=3-5&amp;pmid=13131921&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b0205">41. <span class="mixed-citation" id="h0205">De Madrid AT, Porterfield JS. A simple micro-culture method for the study of group B arboviruses. Bulletin of the World Health Organization. 1969;40(1):113-21. Epub 1969/01/01. PubMed PMID: 4183812; PubMed Central PMCID: PMCPMC2554446. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2554446/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/4183812" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0210">42. <span class="mixed-citation" id="h0210">Achazi K, Nitsche A, Patel P, Radoni&#x00107; A, Donoso Mantke O, Niedrig M. Detection and differentiation of tick-borne encephalitis virus subtypes by a reverse transcription quantitative real-time PCR and pyrosequencing. Journal of virological methods. 2011;171(1):34-9. Epub 2010/10/12. doi: 10.1016/j.jviromet.2010.09.026. PubMed PMID: 20933016. [<a href="https://pubmed.ncbi.nlm.nih.gov/20933016" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0215">43. <span class="mixed-citation" id="h0215">Martin M. Cutadapt removes adapter sequences from high-throughput sequencing reads. 2011. 2011;17(1):3. Epub 2011-08-02. doi: 10.14806/ej.17.1.200.</span></div><div class="ref-cit-blk half_rhythm" id="b0220">44. <span class="mixed-citation" id="h0220">Andrews S. FastQC: a quality control tool for high throughput sequence data. 2010;Available online at: http://www.bioinformatics.babraham.ac.uk/projects/fastqc.</span></div><div class="ref-cit-blk half_rhythm" id="b0225">45. <span class="mixed-citation" id="h0225">Ghosh S, Chan CK. Analysis of RNA-Seq Data Using TopHat and Cufflinks. Methods in molecular biology (Clifton, NJ). 2016;1374:339-61. Epub 2015/11/01. doi: 10.1007/978-1-4939-3167-5_18. PubMed PMID: 26519415. [<a href="https://pubmed.ncbi.nlm.nih.gov/26519415" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0230">46. <span class="mixed-citation" id="h0230">Friedl&#x000e4;nder MR, Mackowiak SD, Li N, Chen W, Rajewsky N. miRDeep2 accurately identifies known and hundreds of novel microRNA genes in seven animal clades. Nucleic acids research. 2012;40(1):37-52. Epub 2011/09/14. doi: 10.1093/nar/gkr688. PubMed PMID: 21911355; PubMed Central PMCID: PMCPMC3245920. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3245920/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21911355" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0235">47. <span class="mixed-citation" id="h0235">Griffiths-Jones S, Grocock RJ, van Dongen S, Bateman A, Enright AJ. miRBase: microRNA sequences, targets and gene nomenclature. Nucleic acids research. 2006;34(Database issue):D140-4. Epub 2005/12/31. doi: 10.1093/nar/gkj112. PubMed PMID: 16381832; PubMed Central PMCID: PMCPMC1347474. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1347474/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16381832" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0240">48. <span class="mixed-citation" id="h0240">Langmead B, Trapnell C, Pop M, Salzberg SL. Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. Genome biology. 2009;10(3):R25. Epub 2009/03/06. doi: 10.1186/gb-2009-10-3-r25. PubMed PMID: 19261174; PubMed Central PMCID: PMCPMC2690996. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2690996/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19261174" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0245">49. <span class="mixed-citation" id="h0245">Quinlan AR, Hall IM. BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics (Oxford, England). 2010;26(6):841-2. Epub 2010/01/30. doi: 10.1093/bioinformatics/btq033. PubMed PMID: 20110278; PubMed Central PMCID: PMCPMC2832824. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2832824/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20110278" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0250">50. <span class="mixed-citation" id="h0250">Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, et al. The Sequence Alignment/Map format and SAMtools. Bioinformatics (Oxford, England). 2009;25(16):2078-9. Epub 2009/06/10. doi: 10.1093/bioinformatics/btp352. PubMed PMID: 19505943; PubMed Central PMCID: PMCPMC2723002. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2723002/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19505943" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0255">51. <span class="mixed-citation" id="h0255">Sticht C, De La Torre C, Parveen A, Gretz N. miRWalk: An online resource for prediction of microRNA binding sites. PloS one. 2018;13(10):e0206239. Epub 2018/10/20. doi: 10.1371/journal.pone.0206239. PubMed PMID: 30335862; PubMed Central PMCID: PMCPMC6193719. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6193719/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30335862" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0260">52. <span class="mixed-citation" id="h0260">Paraskevopoulou MD, Vlachos IS, Karagkouni D, Georgakilas G, Kanellos I, Vergoulis T, et al. DIANA-LncBase v2: indexing microRNA targets on non-coding transcripts. Nucleic acids research. 2016;44(D1):D231-8. Epub 2015/11/28. doi: 10.1093/nar/gkv1270. PubMed PMID: 26612864; PubMed Central PMCID: PMCPMC4702897. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4702897/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26612864" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0265">53. <span class="mixed-citation" id="h0265">Wong N, Wang X. miRDB: an online resource for microRNA target prediction and functional annotations. Nucleic acids research. 2015;43(Database issue):D146-52. Epub 2014/11/08. doi: 10.1093/nar/gku1104. PubMed PMID: 25378301; PubMed Central PMCID: PMCPMC4383922. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4383922/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25378301" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0270">54. <span class="mixed-citation" id="h0270">Vaquero-Garcia J, Barrera A, Gazzara MR, Gonz&#x000e1;lez-Vallinas J, Lahens NF, Hogenesch JB, et al. A new view of transcriptome complexity and regulation through the lens of local splicing variations. eLife. 2016;5:e11752. Epub 2016/02/02. doi: 10.7554/eLife.11752. PubMed PMID: 26829591; PubMed Central PMCID: PMCPMC4801060. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4801060/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26829591" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0275">55. <span class="mixed-citation" id="h0275">Dobin A, Davis CA, Schlesinger F, Drenkow J, Zaleski C, Jha S, et al. STAR: ultrafast universal RNA-seq aligner. Bioinformatics (Oxford, England). 2013;29(1):15-21. Epub 2012/10/30. doi: 10.1093/bioinformatics/bts635. PubMed PMID: 23104886; PubMed Central PMCID: PMCPMC3530905. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3530905/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23104886" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0280">56. <span class="mixed-citation" id="h0280">Selinger M, Tykalov&#x000e1; H, &#x00160;t&#x0011b;rba J, V&#x0011b;chtov&#x000e1; P, Vavru&#x00161;kov&#x000e1; Z, Lieskovsk&#x000e1; J, et al. Tick-borne encephalitis virus inhibits rRNA synthesis and host protein production in human cells of neural origin. PLoS neglected tropical diseases. 2019;13(9):e0007745. Epub 2019/09/29. doi: 10.1371/journal.pntd.0007745. PubMed PMID: 31560682; PubMed Central PMCID: PMCPMC6785130. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6785130/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31560682" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0285">57. <span class="mixed-citation" id="h0285">Trapnell C, Roberts A, Goff L, Pertea G, Kim D, Kelley DR, et al. Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks. Nature protocols. 2012;7(3):562-78. Epub 2012/03/03. doi: 10.1038/nprot.2012.016. PubMed PMID: 22383036; PubMed Central PMCID: PMCPMC3334321. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3334321/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22383036" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0290">58. <span class="mixed-citation" id="h0290">Huang da W, Sherman BT, Lempicki RA. Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources. Nature protocols. 2009;4(1):44-57. Epub 2009/01/10. doi: 10.1038/nprot.2008.211. PubMed PMID: 19131956. [<a href="https://pubmed.ncbi.nlm.nih.gov/19131956" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0295">59. <span class="mixed-citation" id="h0295">Pla R, Stanco A, Howard MA, Rubin AN, Vogt D, Mortimer N, et al. Dlx1 and Dlx2 Promote Interneuron GABA Synthesis, Synaptogenesis, and Dendritogenesis. Cerebral cortex (New York, NY : 1991). 2018;28(11):3797-815. Epub 2017/10/14. doi: 10.1093/cercor/bhx241. PubMed PMID: 29028947; PubMed Central PMCID: PMCPMC6188538. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6188538/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29028947" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0300">60. <span class="mixed-citation" id="h0300">Ip JP, Fu AK, Ip NY. CRMP2: functional roles in neural development and therapeutic potential in neurological diseases. The Neuroscientist : a review journal bringing neurobiology, neurology and psychiatry. 2014;20(6):589-98. Epub 2014/01/10. doi: 10.1177/1073858413514278. PubMed PMID: 24402611. [<a href="https://pubmed.ncbi.nlm.nih.gov/24402611" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0305">61. <span class="mixed-citation" id="h0305">Sessions OM, Tan Y, Goh KC, Liu Y, Tan P, Rozen S, et al. Host cell transcriptome profile during wild-type and attenuated dengue virus infection. PLoS neglected tropical diseases. 2013;7(3):e2107. Epub 2013/03/22. doi: 10.1371/journal.pntd.0002107. PubMed PMID: 23516652; PubMed Central PMCID: PMCPMC3597485. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3597485/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23516652" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0310">62. <span class="mixed-citation" id="h0310">De Maio FA, Risso G, Iglesias NG, Shah P, Pozzi B, Gebhard LG, et al. The Dengue Virus NS5 Protein Intrudes in the Cellular Spliceosome and Modulates Splicing. PLoS pathogens. 2016;12(8):e1005841. Epub 2016/08/31. doi: 10.1371/journal.ppat.1005841. PubMed PMID: 27575636; PubMed Central PMCID: PMCPMC5004807. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5004807/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27575636" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0315">63. <span class="mixed-citation" id="h0315">Cho H, Proll SC, Szretter KJ, Katze MG, Gale M, Jr., Diamond MS. Differential innate immune response programs in neuronal subtypes determine susceptibility to infection in the brain by positive-stranded RNA viruses. Nature medicine. 2013;19(4):458-64. Epub 2013/03/05. doi: 10.1038/nm.3108. PubMed PMID: 23455712; PubMed Central PMCID: PMCPMC3618596. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3618596/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23455712" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0320">64. <span class="mixed-citation" id="h0320">Schneider WM, Chevillotte MD, Rice CM. Interferon-stimulated genes: a complex web of host defenses. Annual review of immunology. 2014;32:513-45. Epub 2014/02/22. doi: 10.1146/annurev-immunol-032713-120231. PubMed PMID: 24555472; PubMed Central PMCID: PMCPMC4313732. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4313732/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24555472" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0325">65. <span class="mixed-citation" id="h0325">Lazear HM, Daniels BP, Pinto AK, Huang AC, Vick SC, Doyle SE, et al. Interferon-lambda restricts West Nile virus neuroinvasion by tightening the blood-brain barrier. Science translational medicine. 2015;7(284):284ra59. Epub 2015/04/24. doi: 10.1126/scitranslmed.aaa4304. PubMed PMID: 25904743; PubMed Central PMCID: PMCPMC4435724. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4435724/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25904743" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0330">66. <span class="mixed-citation" id="h0330">Rivera-Serrano EE, Gizzi AS, Arnold JJ, Grove TL, Almo SC, Cameron CE. Viperin Reveals Its True Function. Annual review of virology. 2020;7(1):421-46. Epub 2020/07/01. doi: 10.1146/annurev-virology-011720-095930. PubMed PMID: 32603630; PubMed Central PMCID: PMCPMC8191541. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8191541/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32603630" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0335">67. <span class="mixed-citation" id="h0335">Soveg FW, Schwerk J, Gokhale NS, Cerosaletti K, Smith JR, Pairo-Castineira E, et al. Endomembrane targeting of human OAS1 p46 augments antiviral activity. eLife. 2021;10. Epub 2021/08/04. doi: 10.7554/eLife.71047. PubMed PMID: 34342578; PubMed Central PMCID: PMCPMC8357416. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8357416/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34342578" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0340">68. <span class="mixed-citation" id="h0340">Castro FL, Geddes VEV, Monteiro FLL, Gon&#x000e7;alves R, Campanati L, Pezzuto P, et al. MicroRNAs 145 and 148a Are Upregulated During Congenital Zika Virus Infection. ASN neuro. 2019;11:1759091419850983. Epub 2019/06/20. doi: 10.1177/1759091419850983. PubMed PMID: 31213064; PubMed Central PMCID: PMCPMC6585135. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6585135/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31213064" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0345">69. <span class="mixed-citation" id="h0345">Gehman LT, Stoilov P, Maguire J, Damianov A, Lin CH, Shiue L, et al. The splicing regulator Rbfox1 (A2BP1) controls neuronal excitation in the mammalian brain. Nature genetics. 2011;43(7):706-11. Epub 2011/05/31. doi: 10.1038/ng.841. PubMed PMID: 21623373; PubMed Central PMCID: PMCPMC3125461. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3125461/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21623373" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0350">70. <span class="mixed-citation" id="h0350">Lovci MT, Ghanem D, Marr H, Arnold J, Gee S, Parra M, et al. Rbfox proteins regulate alternative mRNA splicing through evolutionarily conserved RNA bridges. Nature structural &#x00026; molecular biology. 2013;20(12):1434-42. Epub 2013/11/12. doi: 10.1038/nsmb.2699. PubMed PMID: 24213538; PubMed Central PMCID: PMCPMC3918504. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3918504/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24213538" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0355">71. <span class="mixed-citation" id="h0355">Wang ET, Ward AJ, Cherone JM, Giudice J, Wang TT, Treacy DJ, et al. Antagonistic regulation of mRNA expression and splicing by CELF and MBNL proteins. Genome research. 2015;25(6):858-Epub 2015/04/18. doi: 10.1101/gr.184390.114. PubMed PMID: 25883322; PubMed Central PMCID: PMCPMC4448682. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4448682/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25883322" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0360">72. <span class="mixed-citation" id="h0360">Su CH, D D, Tarn WY. Alternative Splicing in Neurogenesis and Brain Development. Frontiers in molecular biosciences. 2018;5:12. Epub 2018/02/28. doi: 10.3389/fmolb.2018.00012. PubMed PMID: 29484299; PubMed Central PMCID: PMCPMC5816070. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5816070/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29484299" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0365">73. <span class="mixed-citation" id="h0365">Wang Y, Ji P, Liu J, Broaddus RR, Xue F, Zhang W. Centrosome-associated regulators of the G(2)/M checkpoint as targets for cancer therapy. Molecular cancer. 2009;8:8. Epub 2009/02/17. doi: 10.1186/1476-4598-8-8. PubMed PMID: 19216791; PubMed Central PMCID: PMCPMC2657106. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2657106/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19216791" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0370">74. <span class="mixed-citation" id="h0370">Ciccia A, Elledge SJ. The DNA damage response: making it safe to play with knives. Molecular cell. 2010;40(2):179-204. Epub 2010/10/23. doi: 10.1016/j.molcel.2010.09.019. PubMed PMID: 20965415; PubMed Central PMCID: PMCPMC2988877. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2988877/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20965415" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0375">75. <span class="mixed-citation" id="h0375">Overby AK, Popov VL, Niedrig M, Weber F. Tick-borne encephalitis virus delays interferon induction and hides its double-stranded RNA in intracellular membrane vesicles. Journal of virology. 2010;84(17):8470-83. Epub 2010/06/18. doi: 10.1128/jvi.00176-10. PubMed PMID: 20554782; PubMed Central PMCID: PMCPMC2919015. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2919015/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20554782" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0380">76. <span class="mixed-citation" id="h0380">Polonio CM, Peron JPS. ZIKV Infection and miRNA Network in Pathogenesis and Immune Response. Viruses. 2021;13(10). Epub 2021/10/27. doi: 10.3390/v13101992. PubMed PMID: 34696422; PubMed Central PMCID: PMCPMC8541119. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8541119/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34696422" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0385">77. <span class="mixed-citation" id="h0385">Wong RR, Abd-Aziz N, Affendi S, Poh CL. Role of microRNAs in antiviral responses to dengue infection. Journal of biomedical science. 2020;27(1):4. Epub 2020/01/04. doi: 10.1186/s12929-019-0614-x. PubMed PMID: 31898495; PubMed Central PMCID: PMCPMC6941309. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6941309/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31898495" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0390">78. <span class="mixed-citation" id="h0390">He Y, Ge Y, Jiang M, Zhou J, Luo D, Fan H, et al. MiR-592 Promotes Gastric Cancer Proliferation, Migration, and Invasion Through the PI3K/AKT and MAPK/ERK Signaling Pathways by Targeting Spry2. Cellular physiology and biochemistry : international journal of experimental cellular physiology, biochemistry, and pharmacology. 2018;47(4):1465-81. Epub 2018/06/28. doi: 10.1159/000490839. PubMed PMID: 29949784. [<a href="https://pubmed.ncbi.nlm.nih.gov/29949784" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="b0395">79. <span class="mixed-citation" id="h0395">Sharma B, Joshi S, Sassano A, Majchrzak B, Kaur S, Aggarwal P, et al. Sprouty proteins are negative regulators of interferon (IFN) signaling and IFN-inducible biological responses. The Journal of biological chemistry. 2012;287(50):42352-60. Epub 2012/10/18. doi: 10.1074/jbc.M112.400721. PubMed PMID: 23074222; PubMed Central PMCID: PMCPMC3516778. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3516778/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23074222" ref="reftype=pubmed&amp;article-id=9167876&amp;issue-id=395563&amp;journal-id=2373&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Computational and Structural Biotechnology Journal</span> are provided here courtesy of <strong>Research Network of Computational and Structural Biotechnology</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC9167876/pdf/main.pdf" class="int-view">PDF (4.1M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/9167876/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/9167876/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="9167876" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC9167876%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="cZ6wqT6S8LMNFmC5BSaseL5QJbXWrR8dKDrAvO7BseOunbMdEuove2zm1YkeLtLU">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC9167876%2F&amp;text=Integrative%20RNA%20profiling%20of%20TBEV-infected%20neurons%20and%20astrocytes%20reveals%20potential%20pathogenic%20effectors" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC9167876%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9167876/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC9167876/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/35685361/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC9167876/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/35685361/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC9167876/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/9167876/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC9167876/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
