
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-manuscript.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Discovery and Structure Determination of the Orphan Enzyme Isoxanthopterin Deaminase - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="322C300B24F3920500002A8E895EC7CB.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="nihpa" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC2892419/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2892419/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Biochemistry" /><meta name="citation_title" content="Discovery and Structure Determination of the Orphan Enzyme Isoxanthopterin Deaminase" /><meta name="citation_author" content="Richard S. Hall" /><meta name="citation_author_institution" content="Department of Chemistry, P.O. Box 30012, Texas A&#x00026;M University, College Station, Texas 77842-3012" /><meta name="citation_author" content="Rakhi Agarwal" /><meta name="citation_author_institution" content="Biology Department, Brookhaven National Laboratory, Upton, New York 11973" /><meta name="citation_author" content="Daniel Hitchcock" /><meta name="citation_author_institution" content="Department of Biochemistry &#x00026; Biophysics, Texas A&#x00026;M University, College Station, Texas 77843" /><meta name="citation_author" content="J. Michael Sauder" /><meta name="citation_author_institution" content="Lilly Biotechnology Center, 10300 Campus Point Dr., San Diego, California 92121" /><meta name="citation_author" content="Stephen K. Burley" /><meta name="citation_author_institution" content="Lilly Biotechnology Center, 10300 Campus Point Dr., San Diego, California 92121" /><meta name="citation_author" content="Subramanyam Swaminathan" /><meta name="citation_author_institution" content="Biology Department, Brookhaven National Laboratory, Upton, New York 11973" /><meta name="citation_author" content="Frank M. Raushel" /><meta name="citation_author_institution" content="Department of Chemistry, P.O. Box 30012, Texas A&#x00026;M University, College Station, Texas 77842-3012" /><meta name="citation_author_institution" content="Department of Biochemistry &#x00026; Biophysics, Texas A&#x00026;M University, College Station, Texas 77843" /><meta name="citation_publication_date" content="2010/05/05" /><meta name="citation_issue" content="20" /><meta name="citation_volume" content="49" /><meta name="citation_firstpage" content="4374" /><meta name="citation_doi" content="10.1021/bi100252s" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2892419/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC2892419/" /><meta name="citation_pmid" content="20415463" /><meta name="DC.Title" content="Discovery and Structure Determination of the Orphan Enzyme Isoxanthopterin Deaminase" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="NIH Public Access" /><meta name="DC.Contributor" content="Richard S. Hall" /><meta name="DC.Contributor" content="Rakhi Agarwal" /><meta name="DC.Contributor" content="Daniel Hitchcock" /><meta name="DC.Contributor" content="J. Michael Sauder" /><meta name="DC.Contributor" content="Stephen K. Burley" /><meta name="DC.Contributor" content="Subramanyam Swaminathan" /><meta name="DC.Contributor" content="Frank M. Raushel" /><meta name="DC.Date" content="2010 May 25" /><meta name="DC.Identifier" content="10.1021/bi100252s" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Discovery and Structure Determination of the Orphan Enzyme Isoxanthopterin Deaminase" /><meta property="og:type" content="article" /><meta property="og:description" content="Two previously uncharacterized proteins have been identified that efficiently catalyze the deamination of isoxanthopterin and pterin-6-carboxylate. The genes encoding these two enzymes, NYSGXRC-9339a (gi|44585104) and NYSGXRC-9236b (gi|44611670), were ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2892419/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC2892419/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC2892419
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC2892419/pdf/nihms-199947.pdf" class="int-view">PDF (3.6M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/2892419/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/2892419/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="2892419" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2892419%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="jFo4CpdHDj1AtQyBEtZRdpp55tOQVH8d1eLuRUHbIEXlVJU6uREWmomFwpOhuKph">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2892419%2F&amp;text=Discovery%20and%20Structure%20Determination%20of%20the%20Orphan%20Enzyme%20Isoxanthopterin%20Deaminase" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2892419%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2892419/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC2892419/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/20415463/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC2892419/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/20415463/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC2892419/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/2892419/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC2892419
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-nihpa.png" alt="Logo of nihpa" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="https://www.hhs.gov/open/public-access-guiding-principles/index.html" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div><span id="pmcmata">Biochemistry.</span> Author manuscript; available in PMC 2011 May 25.</div><div></div><div class="final-form"><em>Published in final edited form as:</em></div><div class="final-citation"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20415463" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Biochemistry. 2010 May 25; 49(20): 4374–4382. </a></div> <span class="doi"><span>doi: </span><a href="//doi.org/10.1021%2Fbi100252s" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1021/bi100252s</a></span></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2892419</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS199947</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/20415463">20415463</a></div></div></div><h1 class="content-title">Discovery and Structure Determination of the Orphan Enzyme Isoxanthopterin Deaminase<sup><a href="#FN1" rid="FN1" class=" fn">&#x02020;</a></sup></h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hall%20RS%5BAuthor%5D" class="affpopup" co-rid="_co_idm139871428085056" co-class="co-affbox">Richard S. Hall</a>,<sup>&#x02021;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Agarwal%20R%5BAuthor%5D" class="affpopup" co-rid="_co_idm139871464858896" co-class="co-affbox">Rakhi Agarwal</a>,<sup>&#x000a7;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hitchcock%20D%5BAuthor%5D" class="affpopup" co-rid="_co_idm139871464856624" co-class="co-affbox">Daniel Hitchcock</a>,<sup>&#x003b6;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Sauder%20JM%5BAuthor%5D" class="affpopup" co-rid="_co_idm139871421665280" co-class="co-affbox">J. Michael Sauder</a>,<sup>&#x003c8;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Burley%20SK%5BAuthor%5D" class="affpopup" co-rid="_co_idm139871421662960" co-class="co-affbox">Stephen K. Burley</a>,<sup>&#x003c8;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Swaminathan%20S%5BAuthor%5D" class="affpopup" co-rid="_co_idm139871421660688" co-class="co-affbox">Subramanyam Swaminathan</a>,<sup>*</sup><sup>&#x000a7;</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Raushel%20FM%5BAuthor%5D" class="affpopup" co-rid="_co_idm139871421657584" co-class="co-affbox">Frank M. Raushel</a><sup>&#x02021;</sup><sup>&#x003b6;</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139871428085056"><h3 class="no_margin">Richard S. Hall</h3><p><sup>&#x02021;</sup>Department of Chemistry, P.O. Box 30012, Texas A&#x00026;M University, College Station, Texas 77842-3012</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hall%20RS%5BAuthor%5D">Richard S. Hall</a></div></div><div id="_co_idm139871464858896"><h3 class="no_margin">Rakhi Agarwal</h3><p><sup>&#x000a7;</sup>Biology Department, Brookhaven National Laboratory, Upton, New York 11973</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Agarwal%20R%5BAuthor%5D">Rakhi Agarwal</a></div></div><div id="_co_idm139871464856624"><h3 class="no_margin">Daniel Hitchcock</h3><p><sup>&#x003b6;</sup>Department of Biochemistry &#x00026; Biophysics, Texas A&#x00026;M University, College Station, Texas 77843</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hitchcock%20D%5BAuthor%5D">Daniel Hitchcock</a></div></div><div id="_co_idm139871421665280"><h3 class="no_margin">J. Michael Sauder</h3><p><sup>&#x003c8;</sup>Lilly Biotechnology Center, 10300 Campus Point Dr., San Diego, California 92121</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Sauder%20JM%5BAuthor%5D">J. Michael Sauder</a></div></div><div id="_co_idm139871421662960"><h3 class="no_margin">Stephen K. Burley</h3><p><sup>&#x003c8;</sup>Lilly Biotechnology Center, 10300 Campus Point Dr., San Diego, California 92121</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Burley%20SK%5BAuthor%5D">Stephen K. Burley</a></div></div><div id="_co_idm139871421660688"><h3 class="no_margin">Subramanyam Swaminathan</h3><p><sup>&#x000a7;</sup>Biology Department, Brookhaven National Laboratory, Upton, New York 11973</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Swaminathan%20S%5BAuthor%5D">Subramanyam Swaminathan</a></div></div><div id="_co_idm139871421657584"><h3 class="no_margin">Frank M. Raushel</h3><p><sup>&#x02021;</sup>Department of Chemistry, P.O. Box 30012, Texas A&#x00026;M University, College Station, Texas 77842-3012</p><p><sup>&#x003b6;</sup>Department of Biochemistry &#x00026; Biophysics, Texas A&#x00026;M University, College Station, Texas 77843</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Raushel%20FM%5BAuthor%5D">Frank M. Raushel</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm139871426579920_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139871426579920_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm139871426579920_ai" style="display:none"><div class="fm-affl" id="A1"><sup>&#x02021;</sup>Department of Chemistry, P.O. Box 30012, Texas A&#x00026;M University, College Station, Texas 77842-3012</div><div class="fm-affl" id="A2"><sup>&#x003b6;</sup>Department of Biochemistry &#x00026; Biophysics, Texas A&#x00026;M University, College Station, Texas 77843</div><div class="fm-affl" id="A3"><sup>&#x000a7;</sup>Biology Department, Brookhaven National Laboratory, Upton, New York 11973</div><div class="fm-affl" id="A4"><sup>&#x003c8;</sup>Lilly Biotechnology Center, 10300 Campus Point Dr., San Diego, California 92121</div><div id="CR1"><sup>*</sup>To whom correspondence may be sent: (FMR) telephone: (979)-845-3373; fax: (979)-845-9452; <a href="mailto:dev@null" data-email="ude.umat@lehsuar" class="oemail">ude.umat@lehsuar</a> (SS) telephone: (631)-344-3187; fax (631)-344-3407; <a href="mailto:dev@null" data-email="vog.lnb@imaws" class="oemail">vog.lnb@imaws</a></div></div><div class="fm-article-notes half_rhythm"></div><div class="permissions half_rhythm hide" id="idm139871426579920_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">PMC Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm" role="complementary" aria-label="Related or updated information about this article."><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20415463" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Biochemistry</a></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>1_si_001.</strong></div><div><a href="/pmc/articles/PMC2892419/bin/NIHMS199947-supplement-1_si_001.pdf" data-ga-action="click_feat_suppl">NIHMS199947-supplement-1_si_001.pdf</a><span style="color:gray"> (23K)</span></div><div class="small guid">GUID: 314429E9-7DD7-4781-939C-54E06B86278B</div></div></dd></dl></div><div id="abstract-a.r.b.s" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.r.b.stitle">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">Two previously uncharacterized proteins have been identified that efficiently catalyze the deamination of isoxanthopterin and pterin-6-carboxylate. The genes encoding these two enzymes, NYSGXRC-9339a (gi|44585104) and NYSGXRC-9236b (gi|44611670), were first identified from DNA isolated from the Sargasso Sea as part of the Global Ocean Sampling Project. The genes were synthesized, and the proteins were subsequently expressed and purified. The X-ray structure of Sgx9339a was determined at 2.7 &#x000c5; resolution (PDB code: 2PAJ). This protein folds as a distorted (&#x003b2;/&#x003b1;)<sub>8</sub>-barrel and contains a single zinc ion in the active site. These enzymes are members of the amidohydrolase superfamily and belong to cog0402 within the clusters of orthologous groups (COG). Enzymes in cog0402 have previously been shown to catalyze the deamination of guanine, cytosine, S-adenosyl homocysteine, and 8-oxoguanine. A small compound library of pteridines, purines, and pyrimidines was used to probe catalytic activity. The only substrates identified in this search were isoxanthopterin and pterin-6-carboxylate. The kinetic constants for the deamination of isoxanthopterin with Sgx9339a were determined to be 1.0 s<sup>&#x02212;1</sup>, 8.0 &#x003bc;M, and 1.3 &#x000d7; 10<sup>5</sup> M<sup>&#x02212;1</sup> s<sup>&#x02212;1</sup> for <em>k</em><sub>cat</sub>, <em>K</em><sub>m</sub>, and <em>k</em><sub>cat</sub>/<em>K</em><sub>m</sub>, respectively. The active site of Sgx9339a most closely resembles the active site for 8-oxoguanine deaminase (PDB code: 2UZ9). A model for substrate recognition of isoxanthopterin by Sgx9339a was proposed based upon the binding of guanine and xanthine in the active site of guanine deaminase. Residues critical for substrate binding appear to be conserved glutamine and tyrosine residues that hydrogen bond with the carbonyl oxygen at C4, a conserved threonine residue that hydrogen bonds with N5, and another conserved threonine residue that hydrogen bonds with the carbonyl group at C7. These conserved active site residues were used to identify 24 other genes which are predicted to deaminate isoxanthopterin.</p></div></div><div id="body-a.s" class="tsec sec"><h2 class="headless nomenu"></h2><p id="P2" class="p p-first">High throughput DNA sequencing efforts have substantially increased the number of fully sequenced bacterial genomes and our understanding of the enzymes contained within these organisms. A critical assessment of the corresponding protein sequences reveals that a significant fraction of these enzymes has not been functionally annotated or are incorrectly annotated as catalyzing the wrong reactions. Misannotated functional assignments are likely due to an overreliance on automated homology-based sequence comparisons that are not substantiated by active site analysis and experimental verification. The ever increasing number of putative enzymes for which the function(s) are unknown further suggests that many metabolic and catabolic pathways remain to be discovered. Our efforts to develop comprehensive strategies for annotating enzymes of unknown function have focused on the amidohydrolase superfamily (<a href="#R1" rid="R1" class=" bibr popnode">1</a>-<a href="#R10" rid="R10" class=" bibr popnode">10</a>).</p><p id="P3">The amidohydrolase superfamily (AHS<sup><a href="#FN2" rid="FN2" class=" fn">1</a></sup>) was first recognized by Sander and Holm based upon the structural similarities of urease, phosphotriesterase, and adenosine deaminase (<a href="#R11" rid="R11" class=" bibr popnode">11</a>). Enzymes within this superfamily catalyze hydrolytic reactions of amide and ester bonds contained within carbohydrates, peptides, and nucleic acids (<a href="#R12" rid="R12" class=" bibr popnode">12</a>). Less common amidohydrolasecatalyzed reactions include decarboxylations, hydrations, and isomerizations (<a href="#R3" rid="R3" class=" bibr popnode">3</a>, <a href="#R13" rid="R13" class=" bibr popnode">13</a>-<a href="#R17" rid="R17" class=" bibr popnode">17</a>). Active sites of enzymes within the AHS contain a binuclear or mononuclear metal center that activates a hydrolytic water molecule and/or the substrate for nucleophilic attack. Over 12,000 unique protein sequences have been identified as members of the AHS within the genomes of the first 1,000 completely sequenced bacteria deposited in the NCBI. Contained within the AHS are 24 clusters of orthologous groups (COG).</p><p id="P4">One of the largest of these clusters with more than 1,000 members is the group of proteins within cog0402. A network representation of the sequence relationships for cog0402 at a BLAST e-value cutoff of 10<sup>&#x02212;70</sup> is presented in <a href="/pmc/articles/PMC2892419/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 1</span></a> (<a href="#R18" rid="R18" class=" bibr popnode">18</a>). All of the functionally annotated enzymes within this COG catalyze deamination reactions including guanine deaminase (subcluster <strong>2</strong>), cytosine deaminase (subcluster <strong>6</strong>), <em>S</em>-adenosyl homocysteine/thiomethyl adenosine deaminase (subcluster <strong>1</strong>), and <em>N</em>-formimino-L-glutamate deiminase (subcluster <strong>8</strong>). In addition, we have recently demonstrated that the majority of enzymes found within subcluster <strong>4</strong> catalyze the deamination of 8-oxoguanine to urate (<a href="#R19" rid="R19" class=" bibr popnode">19</a>). The catalytic functions for the remaining subclusters within cog0402 remain uncharacterized.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2892419/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139871428655472"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2892419_nihms-199947-f0003.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-199947-f0003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2892419/bin/nihms-199947-f0003.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139871428655472"><a target="object" rel="noopener" href="/pmc/articles/PMC2892419/figure/F3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2892419/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 1</a></div><!--caption a7--><div class="caption"><p>Cytoscape network representation (<a href="http://www.cytoscape.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.cytoscape.org</a>) of the sequence relationships in cog0402 from the amidohydrolase superfamily. Each node in the network represents a single sequence and each edge (depicted as lines) represents the pairwise connection between two sequences with the most significant BLAST E-value (better than 1 &#x000d7; 10<sup>&#x02212;70</sup>). Lengths of edges are not meaningful except that sequences in tightly clustered groups are relatively more similar to each other than sequences with few connections. In subcluster <strong>4</strong> the green-colored nodes catalyze the deamination of 8-oxoguanine and possess a conserved Cys-Ser dyad C-terminal to &#x003b2;-strand 4. In subcluster <strong>4</strong> the blue-colored nodes possess a Thr-Thr dyad and catalyze the deamination of isoxanthopterin. In subcluster <strong>4</strong> the red-colored nodes have a Thr-Ser dyad. Additional information is presented in the text.</p></div></div></div><p id="P5">The structural determinants of 8-oxoguanine deaminase in subcluster <strong>4</strong> are a conserved glutamine and tyrosine which occur in the loops at the C-termini of &#x003b2;-strands 1 and 2, respectively, and a conserved Cys-Ser dyad found at the C-terminus of &#x003b2;-strand 4 within a (&#x003b2;/&#x003b1;)<sub>8</sub>-barrel structural fold. In the subcluster of enzymes that catalyze the deamination of guanine, the Cys-Ser dyad is replaced with an Arg-Phe pair. Curiously, at the periphery of subcluster <strong>4</strong> is a group of sequences (colored blue in <a href="/pmc/articles/PMC2892419/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 1</span></a>) with a Thr-Thr dyad. This paper describes the discovery of function for this group of enzymes.</p><p id="P6" class="p">Two enzymes interrogated for this investigation were prepared for crystallization as part of the NIH-sponsored Protein Structure Initiative (PSI) by the New York SGX Research Center for Structural Genomics (NYSGXRC). The two targets, Sgx9236b and Sgx9339a, were derived from environmental DNA sequences isolated from bacteria in the Sargasso Sea under the Global Ocean Sampling Project. The closest homologue to Sgx9339a (88% identical) is Bxe_A2016 from <em>Burkholderia xenovorans</em> LB400 and the closest homologue of Sgx9236b (95% identical) is Bphyt_7136 from <em>Burkholderia phytofirmans</em> PsJN. Both of these proteins are currently annotated by NCBI as ATZ/TRZ-like amidohydrolases. The three-dimensional structure of Sgx9339a was determined at 2.7 &#x000c5; resolution. Using a combination of compound library screening and structural comparisons, we have determined that both of these enzymes catalyze the deamination of isoxanthopterin to 7-oxylumazine.</p></div><div id="S1" class="tsec sec"><h2 class="head no_bottom_margin" id="S1title">Materials and Methods</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Materials</h3><p id="P7" class="p p-first-last">All chemicals and buffers were obtained from Sigma-Aldrich unless specified otherwise. 8-Oxoguanine was purchased from Toronto Research Chemicals Inc. Kinetic assays were performed in 96-well microtiter plates with a SPECTRAmax 384 Plus spectrophotometer from Molecular Devices. Compounds tested for enzymatic activity with Sgx9339a and Sgx9236b included isoxanthopterin, pterin-6-carboxylate, D-erythro-neopterin, 6-biopterin, 6,7-dimethylpterin, pterin, 8-oxoguanine, guanine, acycloguanosine, isocytosine, cytosine, folate, 2,4-diamino-6-hydroxymethylpteridine, GTP, GDP, GMP, guanosine, 3&#x02032;,5&#x02032;-cGMP, 2&#x02032;-dGTP, 7-methylguanosine, 7-methylguanine, 2&#x02032;-deoxyguanosine, melamine, ammeline, 5-azacytosine, isoguanine, creatinine, thiamine, cytidine, 2-deoxycytidine, 5-fluorocytosine, 5-methylcytosine, 3-oxauracil, CTP, CDP, CMP, cytosine arabinoside, cytidine-5&#x02032;-diphosphocholine, dCTP, dCTP, dCMP, 3&#x02032;,5&#x02032;-cCMP, 3&#x02032;-CMP, 2&#x02032;,3&#x02032;-cCMP, 5-methyl-2&#x02032;-deoxycytidine, adenine, 7-methyladenine, 2&#x02032;-deoxyadenosine, ATP, ADP, AMP, 3&#x02032;,5&#x02032;-cAMP, and thiomethyl adenosine.</p></div><div id="S3" class="sec"><h3 id="S3title">Protein Expression and Purification</h3><p id="P8" class="p p-first">The sequences of Sgx9339a (gi|44585104) and Sgx9236b (gi|44611670) were obtained from the Global Ocean Sampling Project. The target selection processes for structure determination are described in Pieper <em>et al.</em> (<a href="#R6" rid="R6" class=" bibr popnode">6</a>). Codon-optimized synthetic DNA (Codon Devices, Inc.) was used as a template and cloned into a custom TOPO-isomerase vector, pSGX3(BC), supplied by Invitrogen. Forward and reverse primers for Sgx9339a were ACCACGTATGATACCCAGCC and CTACCACAACTTCGCGCAAGAGC, and for Sgx9236b were CAGACCGAAGCGCTGTTAATTA and CATCCGCTAACAACTGTTTAACCC, respectively. The clone encodes Met-Ser-Leu followed by the PCR product and then Glu-Gly-His<sub>6</sub>. Miniprep DNA was transformed into BL21(DE3)-Codon+RIL expression cells (Stratagene), expressed, and made into a 30% glycerol stock for large scale fermentation.</p><p id="P9">Expression clones of Sgx9339a and Sgx9236b were cultured using ZYP-5052 auto-induction media (William Studier, Brookhaven National Laboratory). Overnight cultures were prepared from glycerol stocks that were derived from fresh transformants in LB media. The following morning, overnight cultures were used to inoculate ZYP-5052 auto-induction media. A 5-10% inoculum was achieved by adding 12-25 mL of overnight culture to 250 mL ZYP-5052 media containing 90 &#x003b3;g/mL kanamycin and 30 &#x003b3;g/mL chloramphenicol in a 2 L baffled shake flask. Cultures were grown at 37 &#x000b0;C for 5 hours at 225 rpm agitation. The temperature was then switched to 22 &#x000b0;C and the cells grown overnight for 18 hours to allow for induction of the intended protein. Selenomethionine-labeled protein was produced for Sgx9339a and Sgx9236b using High Yield SeMet media (Orion Enterprises, Inc, Northbrook, IL). Cells were harvested using standard centrifugation for 10 minutes at 6000 rpm and frozen at &#x02212;80 &#x000b0;C.</p><p id="P10" class="p p-last">Cells were lysed in 20 mM TRIS pH 8.0, 0.5 M NaCl, 25 mM imidazole, and 0.1% Tween 20 by sonication. The cellular debris was removed by centrifugation for 30 minutes (39,800 &#x003bc; <em>g</em>). The supernatant was collected and incubated with 10 mL of a 50% slurry of NiNTA Agarose (Qiagen) for 30 minutes with gentle stirring. The sample was then poured into a drip column and washed with 50 mL of wash buffer (20 mM Tris-HCl pH 8.0, 500 mM NaCl, 10% glycerol, and 25 mM imidazole) to remove unbound proteins. The protein of interest was eluted using 25 mL of elution buffer (wash buffer with 500 mM imidazole). Fractions containing the protein were pooled and further purified by gel filtration chromatography on a GE Healthcare HiLoad 16/60 Superdex 200 prep grade column pre-equilibrated with gel filtration buffer (10 mM HEPES, pH 7.5, 150 mM NaCl, 10% glycerol and 5 mM DTT). Fractions containing the protein of interest were combined and concentrated to 10 mg/mL by centrifugation in an Amicon Ultra-15 10,000 Da MWCO centrifugal filter unit. Electrospray mass spectrometry was used to obtain an accurate mass of the protein purified to confirm its identity. The expression plasmids (NYSGXRC clone ID 9339a2BCt7p1 and 9236b2BCt11p1) are available through the PSI Material Repository (<a href="http://psimr.asu.edu/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://psimr.asu.edu/</a>), and the DNA sequences and experimental details are available in the Protein Expression Purification and Crystallization Database (PepcDB) as TargetID &#x0201c;NYSGXRC-9339a&#x0201d; and &#x0201c;NYSGXRC-9236b&#x0201d;.</p></div><div id="S4" class="sec"><h3 id="S4title">Crystallization, Data Collection and Structure Determination</h3><p id="P11" class="p p-first">Diffraction quality crystals of the seleno-methionine form of Sgx9339a were obtained in the trigonal space group P3<sub>1</sub>21 at room temperature via vapor diffusion against a reservoir solution containing 0.1 M MES, pH 6.5, 12% (w/v) PEG 20000, (1 &#x003bc;L of reservoir solution plus 1 &#x003bc;L of protein at 20 mg/mL). Crystals were frozen by direct immersion in liquid nitrogen using the mother liquor with 15% (v/v) glycerol added. X-ray diffraction data were collected under standard cryogenic conditions at Beamline X29 (National Synchrotron Light Source, Brookhaven National Laboratory) and were reduced, merged, and scaled using HKL2000 (<a href="#R20" rid="R20" class=" bibr popnode">20</a>). The Matthews coefficient was calculated to be 3.45 &#x000c5;<sup>3</sup>/Da assuming one molecule/asymmetric unit. Data collection and refinement statistics are provided in <a href="/pmc/articles/PMC2892419/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. The presence of zinc was confirmed by absorption edge scanning at the NSLS beamline.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p>Data collection and refinement statistics.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><tbody><tr><td align="left" valign="top" rowspan="1" colspan="1">
<strong>Data collection statistics:</strong>
</td><td align="left" valign="top" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Wavelength (&#x000c5;)</td><td align="left" valign="top" rowspan="1" colspan="1">0.9792</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Resolution (&#x000c5;)</td><td align="left" valign="top" rowspan="1" colspan="1">50-2.7</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Space group</td><td align="left" valign="top" rowspan="1" colspan="1">P3<sub>1</sub>21</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Unit Cell dimensions</td><td align="left" valign="top" rowspan="1" colspan="1">a = 88.9, b = 88.9, c = 161.9, &#x003b1; = &#x003b2; = 90&#x000b0;, &#x003b3; = 120&#x000b0;</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Diffraction protocol</td><td align="left" valign="top" rowspan="1" colspan="1">Single wavelength anomalous dispersion</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Molecules /asymmetric unit</td><td align="left" valign="top" rowspan="1" colspan="1">1</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Redundancy (overall/outermost shell)</td><td align="left" valign="top" rowspan="1" colspan="1">19 (10.5)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1"><em>I</em>/&#x003c3; (<em>I</em>)</td><td align="left" valign="top" rowspan="1" colspan="1">11.1(2.5)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">outermost shell (&#x000c5;)</td><td align="left" valign="top" rowspan="1" colspan="1">2.8 &#x02013; 2.7</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">R<sub><em>merge</em></sub> (overall/outmost shell)</td><td align="left" valign="top" rowspan="1" colspan="1">0.07 (0.30)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Completeness (%) (overall/outmost shell)</td><td align="left" valign="top" rowspan="1" colspan="1">94.4 (63)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Number of Reflections</td><td align="left" valign="top" rowspan="1" colspan="1">19968</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">
<strong>Refinement statistics:</strong>
</td><td align="left" valign="top" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Resolution range (&#x000c5;)</td><td align="left" valign="top" rowspan="1" colspan="1">50-2.7</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Number of reflections</td><td align="left" valign="top" rowspan="1" colspan="1">19173</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Completeness (working + test) (%)</td><td align="left" valign="top" rowspan="1" colspan="1">91.6</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">R<sub><em>factor</em></sub></td><td align="left" valign="top" rowspan="1" colspan="1">0.23</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">R<sub><em>free</em></sub></td><td align="left" valign="top" rowspan="1" colspan="1">0.28</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Number of protein atoms</td><td align="left" valign="top" rowspan="1" colspan="1">3194</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Number of zinc ions</td><td align="left" valign="top" rowspan="1" colspan="1">1</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Number of water molecules</td><td align="left" valign="top" rowspan="1" colspan="1">85</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">B Values</td><td align="left" valign="top" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">&#x02003;from Wilson plot (&#x000c5;<sup>2</sup>)</td><td align="left" valign="top" rowspan="1" colspan="1">39.2</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">&#x02003;mean B value (overall, &#x000c5;<sup>2</sup>)</td><td align="left" valign="top" rowspan="1" colspan="1">58.4</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">r.m.s.d. bonds (&#x000c5;)</td><td align="left" valign="top" rowspan="1" colspan="1">0.009</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">r.m.s.d. angles (&#x000b0;)</td><td align="left" valign="top" rowspan="1" colspan="1">1.6</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Ramachandran plot analysis:</td><td align="left" valign="top" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Most favored region (additionally allowed) (%)</td><td align="left" valign="top" rowspan="1" colspan="1">82.5(16.1)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Generously allowed region (disallowed) (%)</td><td align="left" valign="top" rowspan="1" colspan="1">1.1(0.3)</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139871430235616"><a target="object" rel="noopener" href="/pmc/articles/PMC2892419/table/T1/?report=objectonly">Open in a separate window</a></div></div><p id="P12" class="p p-last">All 13 internal (excluding N-terminal) selenium atoms were located using ShelxD (<a href="#R21" rid="R21" class=" bibr popnode">21</a>), and used for phase refinement with SHARP (<a href="#R22" rid="R22" class=" bibr popnode">22</a>) and density modification with SOLOMON (<a href="#R23" rid="R23" class=" bibr popnode">23</a>). ARP/wARP (<a href="#R24" rid="R24" class=" bibr popnode">24</a>) automatically built ~50% of the protein atomic model and the remainder of the structure was built manually with O (<a href="#R25" rid="R25" class=" bibr popnode">25</a>) and subsequently refined with CNS (<a href="#R26" rid="R26" class=" bibr popnode">26</a>) at 2.7 &#x000c5; resolution.</p></div><div id="S5" class="sec"><h3 id="S5title">Measurement of Enzymatic Activity</h3><p id="P13" class="p p-first-last">The assays for the detection of ammonia for the enzymatic deamination of pteridines, purines and pyrimidines were conducted with a 96-well UV/vis microplate spectrophotometer at 30 &#x000b0;C. Compounds in this putative substrate library were tested at a concentration of 1.0 mM, except for the pteridine derivatives and 8-oxoguanine, which were assayed at 0.1 mM owing to solubility limitations. The coupling system contained 7.4 mM &#x003b1;-ketoglutarate, 0.4 mM NADH, 6 units glutamate dehydrogenase, and 100 mM HEPES, pH 8.5, in a final volume of 0.25 mL. For direct product assays, extinction coefficients for the deamination of isoxanthopterin (&#x00394;&#x003b5;<sub>350 nm</sub> = 7654 M<sup>&#x02212;1</sup> cm<sup>&#x02212;1</sup>) and pterin-6-carboxylate (&#x00394;&#x003b5;<sub>264 nm</sub> = 8080 M<sup>&#x02212;1</sup> cm<sup>&#x02212;1</sup>) were determined from changes in the UV/vis absorption spectra after the deamination reaction was complete using substrate concentrations from 0.004-0.116 mM. Mass spectrometric analysis of product formation was performed by the Laboratory for Biological Mass Spectrometry (LBMS) in the Department of Chemistry at Texas A&#x00026;M University.</p></div><div id="S6" class="sec sec-last"><h3 id="S6title">Data Analysis</h3><p id="P14" class="p p-first">Sequence alignments were created using the ClustalW program at <a href="http://align.genome.jp/sit-bin/clustalw" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://align.genome.jp/sit-bin/clustalw</a>. Alignments were formatted using the Multiple Align Show program at <a href="http://www.bioinformatics.org/sms/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.bioinformatics.org/sms/</a>. Steady-state kinetic data were analyzed using the program Softmax Pro version 4.7.1. For the determination of the kinetic parameters (<em>k</em><sub>cat</sub>, <em>K</em><sub>m</sub> and <em>k</em><sub>cat</sub>/<em>K</em><sub>m</sub>) the data were fit to <a href="#FD1" rid="FD1" class=" disp-formula">equation 1</a> using the non-linear least squares fitting program SigmaPlot 10 where A is the substrate concentration, <em>v</em> is the velocity of the reaction, <em>k</em><sub>cat</sub> is the turnover number and <em>K</em><sub>m</sub> is the Michaelis constant.</p><div class="disp-formula" id="FD1"><div class="f"><em>v</em> ∕ <em>E</em><sub>t</sub> = <em>k</em><sub>cat</sub>A ∕ (<em>K</em><sub>m</sub> + A)</div><div class="l">(1)</div></div></div></div><div id="S7" class="tsec sec"><h2 class="head no_bottom_margin" id="S7title">Results</h2><div id="S8" class="sec sec-first"><h3 id="S8title">Three-Dimensional Structure of Sgx9339a</h3><p id="P15" class="p p-first-last">The X-ray structure of Sgx9339a was determined to a resolution of 2.7 &#x000c5; and a ribbon representation of an individual subunit is presented in <a href="/pmc/articles/PMC2892419/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 2</span></a>. The protein folds into two distinct domains. The larger of the two domains consists of a distorted (&#x003b1;/&#x003b2;)<sub>8</sub>-TIM barrel whereas the smaller domain is made up of the N-terminal residues 10-70 and residues 413-456, together making a &#x003b2;-sheet sandwich as illustrated in <a href="/pmc/articles/PMC2892419/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 3</span></a>. Interestingly, this amidohydrolase is higher in molecular weight than many of the other structurally characterized members of this superfamily and has an extra helical tail that is preceded by a loop comprised of residues 457-463. The helix (residues 464-484) is approximated to the large domain by salt bridge interactions between glutamate and arginine residues and hydophobic interactions at the interface. The opposite surface of the helix is positively charged, displaying five arginines and one lysine. The final refined model consists of residues 10-484, one zinc atom and 85 water molecules. Loop flexibility may have caused poor or no electron density for the loops consisting of residues 89-105, 251-263, 315-319 and 362-380. These residues were not modeled in the final structure.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC2892419/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139871428652768"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2892419_nihms-199947-f0004.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-199947-f0004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2892419/bin/nihms-199947-f0004.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139871428652768"><a target="object" rel="noopener" href="/pmc/articles/PMC2892419/figure/F4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC2892419/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 2</a></div><!--caption a7--><div class="caption"><p>Ribbon representation of the overall structure of Sgx9339a (PDB code: 2PAJ). The &#x003b1;-helices, &#x003b2;-strands and random coil loops are colored cyan, violet and yellow, respectively. The zinc ion is colored in pink. The small domain is made up of the N-terminal residues 10-70 and of residues 413-456. The ribbon diagram was made using PYMOL.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC2892419/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139871426199728"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2892419_nihms-199947-f0005.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-199947-f0005.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2892419/bin/nihms-199947-f0005.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139871426199728"><a target="object" rel="noopener" href="/pmc/articles/PMC2892419/figure/F5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC2892419/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 3</a></div><!--caption a7--><div class="caption"><p>Ribbon representation of the small domain that extends from residues 10-70 and from 413-456</p></div></div></div></div><div id="S9" class="sec"><h3 id="S9title">The Active Site</h3><p id="P16" class="p p-first-last">The active center is located in the middle of the (&#x003b2;/&#x003b1;)<sub>8</sub>-barrel and is exposed to solvent as illustrated in <a href="/pmc/articles/PMC2892419/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Figure 4</span></a>. There is a single zinc ion in the active site. This metal ion is coordinated to two histidine residues (His-76 and His-78) at the C-terminus of &#x003b2;-strand 1 and another histidine at the C-terminus of &#x003b2;-strand 5 (His-248). The remaining protein ligand to the metal center is an aspartate that follows &#x003b2;-strand 8 (Asp-336). The water molecule responsible for nucleophilic attack of the substrate occupies the final coordination site. This water molecule is hydrogen bonded to the side chain of His-285 (2.9 &#x000c5;) positioned near the C-terminus of &#x003b2;-strand 6. The geometry of the mononuclear metal center is identical to that previously observed with guanine deaminase (PDB code: 2UZ9), cytosine deaminase (<a href="#R27" rid="R27" class=" bibr popnode">27</a>) and adenosine deaminase (<a href="#R28" rid="R28" class=" bibr popnode">28</a>). However, the catalytically important glutamate (Glu-251) found within the HxxE motif at the C-terminus of &#x003b2;-strand 5 is not observed, presumably due to conformational flexibility within the crystal.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F6" co-legend-rid="lgnd_F6"><a href="/pmc/articles/PMC2892419/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139871426197296"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2892419_nihms-199947-f0006.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-199947-f0006.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2892419/bin/nihms-199947-f0006.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139871426197296"><a target="object" rel="noopener" href="/pmc/articles/PMC2892419/figure/F6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F6"><div><a class="figpopup" href="/pmc/articles/PMC2892419/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6">Figure 4</a></div><!--caption a7--><div class="caption"><p>Active site interactions of Sgx9339a with the bound zinc and water molecule.</p></div></div></div></div><div id="S10" class="sec sec-last"><h3 id="S10title">Substrate Specificity</h3><p id="P17" class="p p-first">Sgx9339a and Sgx9236b were assayed with a small compound library of pteridines, purines and pyrimidines for deaminase activity. At a fixed enzyme concentration of ~100 nM, significant formation of ammonia could only be detected with isoxanthopterin (<strong>I</strong>) and pterin-6-carboxylate (<strong>II</strong>) as substrates. The reactions catalyzed by these enzymes are shown in <a href="/pmc/articles/PMC2892419/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Scheme 1</span></a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2892419/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139871428663184"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2892419_nihms-199947-f0001.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-199947-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2892419/bin/nihms-199947-f0001.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139871428663184"><a target="object" rel="noopener" href="/pmc/articles/PMC2892419/figure/F1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2892419/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Scheme 1</a></div></div></div><p id="P18">The reaction products for the deamination of isoxanthopterin and pterin-6-carboxylate were verified by UV/vis spectroscopy and mass spectrometry. The absorbance spectrum of isoxanthopterin (<strong>I</strong>) is presented in <a href="/pmc/articles/PMC2892419/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Figure 5</span></a>. At pH 8.5, there are absorbance maxima at 281 and 333 nm (<a href="#R29" rid="R29" class=" bibr popnode">29</a>). The spectrum changes after the addition of Sgx9339a and the absorbance maxima shift to 274 and 326 nm, which is consistent with the formation of 7-oxylumazine (<a href="#R29" rid="R29" class=" bibr popnode">29</a>). The ESI negative ion mass spectrum for M-H of m/z = 178.04 from isoxanthopterin changed to m/z = 179.02 for 7-oxylumazine upon enzyme addition. The maximum absorbance difference between isoxanthopterin and 7-oxylumazine was determined to be at 350 nm with a &#x00394;&#x003b5; = 7650 M<sup>&#x02212;1</sup> cm<sup>&#x02212;1</sup>. The kinetic constants for the deamination of isoxanthopterin with Sxg9339a and Sgx9236b were determined and the results are presented in <a href="/pmc/articles/PMC2892419/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F7" co-legend-rid="lgnd_F7"><a href="/pmc/articles/PMC2892419/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139871426194592"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2892419_nihms-199947-f0007.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-199947-f0007.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2892419/bin/nihms-199947-f0007.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139871426194592"><a target="object" rel="noopener" href="/pmc/articles/PMC2892419/figure/F7/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F7"><div><a class="figpopup" href="/pmc/articles/PMC2892419/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7">Figure 5</a></div><!--caption a7--><div class="caption"><p>UV/vis absorption spectra of isoxanthopterin and pterin-6-carboxylate before and after the addition of Sgx9339a. Upper spectra: Isoxanthopterin, before (black) and after (red) the addition of Sgx9339a at pH 8.5. Lower spectra: Pterin-6-carboxylate before (black) and after (red) the addition of Sgx9339a. Additional information is provided in the text.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p>Kinetic Parameters for Isoxanthopterin and Pterin-6-carboxylate Deamination with Sgx9339a and Sgx9236b.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="left" valign="top" rowspan="1" colspan="1">enzyme</th><th align="left" valign="top" rowspan="1" colspan="1">substrate</th><th align="center" valign="top" rowspan="1" colspan="1"><em>k</em><sub>cat</sub> (s<sup>&#x02212;1</sup>)</th><th align="left" valign="top" rowspan="1" colspan="1"><em>K</em><sub>m</sub> (mM)</th><th align="center" valign="top" rowspan="1" colspan="1"><em>k</em><sub>cat</sub>/<em>K</em><sub>m</sub> (M<sup>&#x02212;1</sup> s<sup>&#x02212;1</sup>)</th></tr></thead><tbody><tr><td align="left" valign="top" rowspan="1" colspan="1">Sgx9339a</td><td align="left" valign="top" rowspan="1" colspan="1">isoxanthopterin</td><td align="center" valign="top" rowspan="1" colspan="1">1.05 &#x000b1; 0.03</td><td align="left" valign="top" rowspan="1" colspan="1">0.008 &#x000b1; 0.001</td><td align="center" valign="top" rowspan="1" colspan="1">(1.3 &#x000b1; 0.2) &#x000d7; 105</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1"></td><td align="left" valign="top" rowspan="1" colspan="1">pterin-6-carboxylate</td><td align="center" valign="top" rowspan="1" colspan="1">1.01 &#x000b1; 0.10</td><td align="left" valign="top" rowspan="1" colspan="1">0.14 &#x000b1; 0.02</td><td align="center" valign="top" rowspan="1" colspan="1">(7.2 &#x000b1; 0.2) &#x000d7; 10<sup>3</sup></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Sgx9236b</td><td align="left" valign="top" rowspan="1" colspan="1">isoxanthopterin</td><td align="center" valign="top" rowspan="1" colspan="1">0.91 &#x000b1; 0.06</td><td align="left" valign="top" rowspan="1" colspan="1">0.05 &#x000b1; 0.01</td><td align="center" valign="top" rowspan="1" colspan="1">(1.8 &#x000b1; 0.2) &#x000d7; 10<sup>4</sup></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1"></td><td align="left" valign="top" rowspan="1" colspan="1">pterin-6-carboxylate</td><td align="center" valign="top" rowspan="1" colspan="1">0.28 &#x000b1; 0.05</td><td align="left" valign="top" rowspan="1" colspan="1">0.09 &#x000b1; 0.03</td><td align="center" valign="top" rowspan="1" colspan="1">(3.1 &#x000b1; 0.4) &#x000d7; 10<sup>3</sup></td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139871430112832"><a target="object" rel="noopener" href="/pmc/articles/PMC2892419/table/T2/?report=objectonly">Open in a separate window</a></div></div><p id="P19" class="p p-last">The absorbance spectrum for pterin-6-carboxylate at pH 8.5 is presented in <a href="/pmc/articles/PMC2892419/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Figure 5</span></a>. In this spectrum there are absorbance maxima at 264 and 361 nm. After the addition of Sgx9339a the spectrum changes to one with absorbance maxima at 287 and 347 nm. The ESI negative ion mass spectrum for M-H of m/z = 206.03 from pterin-6-carboxylate changed to m/z = 207.02 for 2,4-dioxo-pteridine-6-carboxylate after enzyme addition. Direct kinetic assays were conducted at a wavelength of 264 nm with a &#x00394;&#x003b5; of 8080 M<sup>&#x02212;1</sup> cm<sup>&#x02212;1</sup>. The kinetic constants for the deamination of pterin-6-carboxylate with Sgx9339a and Sgx9236b are presented in <a href="/pmc/articles/PMC2892419/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>. For each enzyme the values of <em>k</em><sub>cat</sub>/<em>K</em><sub>m</sub> for the deamination of isoxanthopterin are about an order of magnitude greater than for pterin-6-carboxylate.</p></div></div><div id="S11" class="tsec sec"><h2 class="head no_bottom_margin" id="S11title">Discussion</h2><div id="S12" class="sec sec-first"><h3 id="S12title">Structural Neighbors to Sgx9339a</h3><p id="P20" class="p p-first">The vector alignment search tool (VAST) from NCBI identified two close structural hits to Sgx9339a. The most similar protein (PDB code: 3HPA) catalyzes the deamination of 8-oxoguanine to urate (<a href="#R19" rid="R19" class=" bibr popnode">19</a>) and the other protein (PDB code: 1P1M) catalyzes the deamination of S-adenosyl homocysteine (<a href="#R5" rid="R5" class=" bibr popnode">5</a>). These two proteins superimpose with 2PAJ with their C&#x003b1; atoms of 389 and 374 residues with a Z-score: 49.6 and 43.5; rmsd: 1.8 and 2.5 &#x000c5;; and 35 and 25% sequence identity, respectively. A stereoview of the structural alignments for these three proteins is presented in <a href="/pmc/articles/PMC2892419/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span>Figure 6</span></a>. As expected, 2PAJ is structurally more similar to 3HPA in terms of loop size and arrangement, protein length, active site orientation, mononuclear metal center and the presence of the C-terminal helix. The primary difference in the active site structures is the presence of a threonine (Thr-220) in 2PAJ instead of the cysteine (Cys-232) in 3HPA. The structural alignment comparing the active sites contained within 2PAJ and 3HPA is presented in <a href="/pmc/articles/PMC2892419/figure/F9/" target="figure" class="fig-table-link figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span>Figure 7</span></a>. The helix-turn-helix region of 3HPA (residues 105-123), which acts as a lid for the active site, is disordered in 2PAJ (residues 89-105).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F8" co-legend-rid="lgnd_F8"><a href="/pmc/articles/PMC2892419/figure/F8/" target="figure" rid-figpopup="F8" rid-ob="ob-F8"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139871426192000"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2892419_nihms-199947-f0008.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-199947-f0008.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2892419/bin/nihms-199947-f0008.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139871426192000"><a target="object" rel="noopener" href="/pmc/articles/PMC2892419/figure/F8/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F8"><div><a class="figpopup" href="/pmc/articles/PMC2892419/figure/F8/" target="figure" rid-figpopup="F8" rid-ob="ob-F8">Figure 6</a></div><!--caption a7--><div class="caption"><p>Stereoview of the structural alignment of Sgx9339a (cyan, PDB code: 2PAJ) with the structural neighbors 8-oxoguanine deaminase (green, PDB code: 3HPA) and S-adenosyl homocysteine deaminase (pink, PDB code: 1P1M) and zinc ions (pink).</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F9" co-legend-rid="lgnd_F9"><a href="/pmc/articles/PMC2892419/figure/F9/" target="figure" rid-figpopup="F9" rid-ob="ob-F9"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139871426189504"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2892419_nihms-199947-f0009.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-199947-f0009.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2892419/bin/nihms-199947-f0009.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139871426189504"><a target="object" rel="noopener" href="/pmc/articles/PMC2892419/figure/F9/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F9"><div><a class="figpopup" href="/pmc/articles/PMC2892419/figure/F9/" target="figure" rid-figpopup="F9" rid-ob="ob-F9">Figure 7</a></div><!--caption a7--><div class="caption"><p>Active site overlay of Sgx9339a (PDB code: 2PAJ) shown with green carbon atoms and guanine deaminase (PDB code: 2UZ9) shown with grey carbons.</p></div></div></div><p id="P21">It has been shown that Sgx9339a preferentially catalyzes the deamination of isoxanthopterin to 7-oxylumazine with a <em>k</em><sub>cat</sub>/<em>K</em><sub>m</sub> value that exceeds 10<sup>5</sup> M<sup>&#x02212;1</sup> s<sup>&#x02212;1</sup>. Although the structure of this enzyme has been determined it has thus far not been possible to obtain co-crystals that contain the substrate or product bound in the active site. How then is the substrate accommodated within the active site? One of the closest structural homologues to Sgx9339a in which a product or substrate is bound in the active site is human guanine deaminase complexed with the product xanthine (PDB code: 2UZ9). In that structure the carbonyl group at C6 of xanthine hydrogen bonds with Gln-87 and Arg-213. Arg-213 also interacts with N7. In addition to these recognition elements, Asp-330 and His-279 function together with the metal center to activate the hydrolytic water molecule, and Glu-243 serves as the ultimate proton donor to N1 during the transition from substrate to product. Shown in <a href="/pmc/articles/PMC2892419/figure/F10/" target="figure" class="fig-table-link figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Figure 8a</span></a> is a model of how guanine binds in the active site of human guanine deaminase.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F10" co-legend-rid="lgnd_F10"><a href="/pmc/articles/PMC2892419/figure/F10/" target="figure" rid-figpopup="F10" rid-ob="ob-F10"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139871430241792"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2892419_nihms-199947-f0010.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-199947-f0010.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2892419/bin/nihms-199947-f0010.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139871430241792"><a target="object" rel="noopener" href="/pmc/articles/PMC2892419/figure/F10/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F10"><div><a class="figpopup" href="/pmc/articles/PMC2892419/figure/F10/" target="figure" rid-figpopup="F10" rid-ob="ob-F10">Figure 8</a></div><!--caption a7--><div class="caption"><p>Models for substrate binding in the active sites of guanine deaminase, 8-oxoguanine deaminase and isoxanthopterin. (A) Guanine was placed in the active site of human guanine deaminase (PDB code: 2UZ9) via a simple substitution of this compound for the product xanthine. (B) 8-Oxoguanine was positioned in the active site of Sgx9236e (PDB code: 3HPA) using a structural overlay of this protein with the active site of guanine deaminase (PDB code: 2UZ9) while overlapping the common atoms of 8-oxoguanine and xanthine. (C) Isoxanthopterin was positioned in the active site of Sgx9339a using a structural overlay of this protein with the active site of guanine deaminase while overlapping the common atoms of isoxanthopterin and xanthine. Since residues 251-263 are disordered in 2PAJ, the important glutamate (Glu-251) from the HxxE motif at the C-terminal end of &#x003b2;-strand 5 was inserted into the active site based upon the orientation of Glu-243 in guanine deaminase (PDB code: 2UZ9).</p></div></div></div><p id="P22">The closest structural homologue to Sgx9339a is Sgx9236e (PDB code: 3HPA). This protein catalyzes the deamination of 8-oxoguanine (<a href="#R19" rid="R19" class=" bibr popnode">19</a>). A ligand-bound structure has not been determined but a model for how the substrate for this enzyme binds in the active site has been developed based upon computational docking of potential high energy intermediates in the active site and direct structural comparisons with guanine deaminase (<a href="#R19" rid="R19" class=" bibr popnode">19</a>). A model for the interaction of 8-oxoguanine in the active site of Sgx9236e is presented in <a href="/pmc/articles/PMC2892419/figure/F10/" target="figure" class="fig-table-link figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Figure 8b</span></a>. In this model a glutamine residue (Gln-98), homologous to Gln-87 in guanine deaminase, hydrogen bonds with the carbonyl group at C6, Tyr-155 also hydrogen bonds with the carbonyl group at C6, while Cys-232 and Ser-233 interact with the carbonyl group at C8. Asp-347, His-296, and Glu-262 function in concert to activate the hydrolytic water molecule and facilitate the various proton transfer reactions.</p><p id="P23" class="p p-last">The structures of guanine deaminase (PDB code: 2UZ9) and 8-oxoguanine deaminase (PDB code: 3HPA) were used to rationalize the binding of isoxanthopterin in the active site of Sgx9339a (PDB code: 2PAJ). In this model the conserved glutamine (Gln-81) and tyrosine (Tyr-139) hydrogen bond with the carbonyl group at C4. One of the two conserved threonine residues (Thr-220) hydrogen bonds with N5 and the other conserved threonine (Thr-221) hydrogen bonds with the carbonyl group at C7. The catalytic triad, consisting of the conserved aspartate (Asp-336), histidine (His-285), and glutamate (Glu-251) are poised to facilitate the deamination reaction in the same fashion as the other functionally characterized enzymes in cog0402. However, Glu-251 cannot be positioned precisely because of conformational flexibility. In this model we have orientated this residue based upon the observed position of the homologous residue in guanine deaminase. The proposed interactions for the binding of isoxanthopterin in the active site of Sgx9339a are presented in <a href="/pmc/articles/PMC2892419/figure/F10/" target="figure" class="fig-table-link figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Figure 8c</span></a>. The numbering for the atoms contained within guanine, 8-oxoguanine, and isoxanthopterin are presented in <a href="/pmc/articles/PMC2892419/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Scheme 2</span></a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2892419/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139871428661344"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2892419_nihms-199947-f0002.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-199947-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2892419/bin/nihms-199947-f0002.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139871428661344"><a target="object" rel="noopener" href="/pmc/articles/PMC2892419/figure/F2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2892419/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Scheme 2</a></div></div></div></div><div id="S13" class="sec sec-last"><h3 id="S13title">Amino Acid Sequence Analysis</h3><p id="P24" class="p p-first">The amino acid sequences of the two isoxanthopterin deaminases, Sgx9339a and Sgx9236b, were aligned with two of the newly discovered enzymes that deaminate 8-oxoguanine (Pa0142 from <em>Pseudomonas aeruginosa</em> and Sgx9236e) and guanine deaminases from <em>Homo sapiens</em> and <em>E. coli</em>.<sup><a href="#FN3" rid="FN3" class=" fn">2</a></sup> The sequence alignment is shown in <a href="/pmc/articles/PMC2892419/figure/F11/" target="figure" class="fig-table-link figpopup" rid-figpopup="F11" rid-ob="ob-F11" co-legend-rid="lgnd_F11"><span>Figure 9</span></a>. The eight &#x003b2;-strands from the (&#x003b2;/&#x003b1;)<sub>8</sub>-barrel structural fold in Sgx9339a are highlighted by the black arrows. The fully conserved metal binding and catalytic residues are highlighted in red. The metal binding residues include the two histidine residues from the C-terminus of &#x003b2;-strand 1, a histidine residue from the C-terminus of &#x003b2;-strand 5 and an aspartate from the C-terminus of &#x003b2;-strand 8. The two additional residues that are involved in the facilitation of the proton transfers and activation of the nucleophilic water molecule include a glutamate at the C-terminus of &#x003b2;-strand 5 (HxxE motif) and a histidine at the C-terminus of &#x003b2;-strand 6. The conserved glutamine residue in all six proteins that is positioned after the HxH motif at the C-terminal end of &#x003b2;-strand 1 is also highlighted in red. Conserved substrate binding residues found in all guanine deaminases are highlighted in green. These residues include the arginine and phenylalanine at the C-terminal end of &#x003b2;-strand 4 and the threonine at the C-terminal end of &#x003b2;-strand 2. For the two 8-oxoguanine deaminases (Pa0142 and Sgx9236e), these residues (in addition to a conserved tyrosine found at the C-terminal end of &#x003b2;-strand 2) are highlighted in light blue and the proposed substrate binding residues specific for the recognition of isoxanthopterin are highlighted in yellow. The remaining residues that are common to all three sets of enzymes are highlighted in dark blue</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F11" co-legend-rid="lgnd_F11"><a href="/pmc/articles/PMC2892419/figure/F11/" target="figure" rid-figpopup="F11" rid-ob="ob-F11"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139871430237952"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2892419_nihms-199947-f0011.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-199947-f0011.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2892419/bin/nihms-199947-f0011.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139871430237952"><a target="object" rel="noopener" href="/pmc/articles/PMC2892419/figure/F11/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F11"><div><a class="figpopup" href="/pmc/articles/PMC2892419/figure/F11/" target="figure" rid-figpopup="F11" rid-ob="ob-F11">Figure 9</a></div><!--caption a7--><div class="caption"><p>Multiple sequence alignment of two isoxanthopterin deaminases (Sgx9339a and Sgx9236b), two 8-oxoguanine deaminases (Pa0142 and Sgx9236e) and two guanine deaminases (<em>E. coli</em> guanine deaminase and guanine deaminase from humans). The 8 beta strands of the (&#x003b2;/&#x003b1;)<sub>8</sub>-barrel in Sgx9339a are denoted by the black arrows. The conserved metal binding residues and catalytic residues in all six proteins are highlighted in red, The conserved ligand binding residues in isoxanthopterin deaminase, 8-oxoguanine deaminase, and guanine deaminase are highlighted in yellow, light blue, and green, respectively. The remaining fully conserved residues in all three proteins are highlighted in dark blue.</p></div></div></div><p id="P25">Sequence comparisons with guanine deaminase and 8-oxoguanine deaminase were subsequently used to differentiate a small group of enzymes, which are now predicted to catalyze the deamination of isoxanthopterin. Guanine deaminases utilize an Arg-Phe dyad for the recognition of guanine, whereas in 8-oxoguanine deaminase this dyad is changed to Cys-Ser. In the isoxanthopterin deaminases this dyad changes to Thr-Thr. It is remarkable that the glutamine, which hydrogen bonds with the carbonyl group of the ring to be deaminated, occurs in all three enzymes and the tyrosine, which interacts with the carbonyl group or ring nitrogen of the adjacent ring, occurs in 8-oxoguanine deaminase and isoxanthopterin deaminase (see <a href="/pmc/articles/PMC2892419/figure/F10/" target="figure" class="fig-table-link figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Figure 8</span></a>).</p><p id="P26">A BLAST search of protein sequences in the NCBI database was used to identify the organisms containing genes anticipated to catalyze the deamination of isoxanthopterin. From this search, 23 organisms were found to share this gene (as evidenced by conservation of the Thr-Thr motif from the C-terminus of &#x003b2;-strand 4, the glutamine from the C-terminus of &#x003b2;-strand 1, and the tyrosine from the C-terminus of &#x003b2;-strand 2). A complete list of genes and organisms is provided in <a href="#SD1" rid="SD1" class=" supplementary-material">Table S1</a>. The novel isoxanthopterin deaminases share a sequence identity of greater than 41% and are predicted to participate in the deamination of isoxanthopterin within these organisms. Included with these 23 genes are four outliers (represented as red circles in subcluster 4 from <a href="/pmc/articles/PMC2892419/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 1</span></a>) which share a threonine-serine dyad at this position. These genes are: Mmwyl1_4101 (gi|152998097) from <em>Marinomonas sp. MWYL1</em>; HMPREF0023_2011 (gi|226953181) from <em>Acinetobacter sp. ATCC 27244</em>; PputW619_2587 (gi|170721762) from <em>Pseudomonas putida W619</em>; and PP_3209 (gi| 26989928) from <em>Pseudomonas putida KT2440</em>. It appears that the Thr-Ser dyad may be an evolutionary intermediate between the Cys-Ser dyad of the 8-oxoguanine deaminases and the Thr-Thr dyad of the isoxanthopterin deaminases. It has not been determined how this small difference in amino acid sequence will affect substrate specificity.</p><p id="P27" class="p p-last">Isoxanthopterin has been isolated from multiple biological sources such as zebrafish embryos (<a href="#R30" rid="R30" class=" bibr popnode">30</a>), the firebug <em>Pyrrhocoris apterus</em> (<a href="#R31" rid="R31" class=" bibr popnode">31</a>), caviar (<a href="#R32" rid="R32" class=" bibr popnode">32</a>) and in silkworm where isoxanthopterin deaminase activity was measured (<a href="#R33" rid="R33" class=" bibr popnode">33</a>). Isoxanthopterin deaminase activity has previously been observed in extracts of <em>Alcaligenes faecalis</em> (<a href="#R34" rid="R34" class=" bibr popnode">34</a>). A search of that genome was conducted in an attempt to identify the specific deaminase that may be similar to Sgx9339a and Sgx9236b but, unfortunately, the complete sequence of <em>A. faecalis</em> is not yet publicly available. Sgx9339a and Sgx9236b are the first proteins of known sequence found to possess significant deaminase activities against the pteridine compounds, isoxanthopterin and pterin-6-carboxylate.</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>1_si_001</h4><div class="sup-box half_rhythm" id="N0x20daef0N0x2d75a50"><a href="/pmc/articles/PMC2892419/bin/NIHMS199947-supplement-1_si_001.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(23K, pdf)</sup></div></div></div><div id="S14" class="tsec sec"><h2 class="head no_bottom_margin" id="S14title">Acknowledgment</h2><div class="sec"><p>We thank Abby Sisco for help in the construction of <a href="/pmc/articles/PMC2892419/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 1</span></a> and sequence analyses within cog0402. We thank the NYSGXRC protein production team for the preparation of Sgx9339a and Sgx9236b. We gratefully acknowledge data collection support for beamline X29 (NSLS).</p></div></div><div id="fn-group-a.t.b" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.t.btitle">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"><sup>&#x02020;</sup>This work was supported in part by the National Institutes of Health (GM 71790 and GM 74945). The X-ray coordinates and structure factors for Sgx9339a have been deposited in the Protein Data Bank (PDB accession code: 2PAJ).</p><p class="fn sec" id="FN2"><sup>1</sup>Abbreviations: AHS, amidohydrolase superfamily COG, cluster of orthologous groups, NYSGXRC, New York SGX Research Center for Structural Genomics VAST, vector alignment search tool.</p><p class="fn sec" id="FN3"><sup>2</sup>The residue numbering for Sgx9339a that appears in <a href="/pmc/articles/PMC2892419/figure/F10/" target="figure" class="fig-table-link figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Figure 8</span></a> corresponds to the numbering system and amino acid sequence used in the PDB file for 2PAJ.</p><p class="fn sec" id="FN4"><p id="P31" class="p p-first">SUPPORTING INFORMATION AVAILABLE</p><p id="P32" class="p p-last">A supplementary table (S1) that identifies enzymes predicted to catalyze the deamination of isoxanthopterin is provided. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pubs.acs.org</a>.</p></p></div></div><div id="ref-list-a.t.c" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.t.ctitle">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">Cummings JA, Fedorov AA, Xu C, Brown S, Fedorov E, Babbitt PC, Almo SC, Raushel FM. Annotating enzymes of uncertain function: the deacylation of D-amino acids by members of the amidohydrolase superfamily. <span><span class="ref-journal">Biochemistry. </span>2009;<span class="ref-vol">48</span>:6469–6481.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2748305/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19518059" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Annotating+enzymes+of+uncertain+function:+the+deacylation+of+D-amino+acids+by+members+of+the+amidohydrolase+superfamily&amp;author=JA+Cummings&amp;author=AA+Fedorov&amp;author=C+Xu&amp;author=S+Brown&amp;author=E+Fedorov&amp;volume=48&amp;publication_year=2009&amp;pages=6469-6481&amp;pmid=19518059&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Xiang DF, Patskovsky Y, Xu C, Meyer AJ, Sauder JM, Burley SK, Almo SC, Raushel FM. Functional identification of incorrectly annotated prolidases from the amidohydrolase superfamily of enzymes. <span><span class="ref-journal">Biochemistry. </span>2009;<span class="ref-vol">48</span>:3730–3742.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2683473/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19281183" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Functional+identification+of+incorrectly+annotated+prolidases+from+the+amidohydrolase+superfamily+of+enzymes&amp;author=DF+Xiang&amp;author=Y+Patskovsky&amp;author=C+Xu&amp;author=AJ+Meyer&amp;author=JM+Sauder&amp;volume=48&amp;publication_year=2009&amp;pages=3730-3742&amp;pmid=19281183&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Nguyen TT, Brown S, Fedorov AA, Fedorov EV, Babbitt PC, Almo SC, Raushel FM. At the periphery of the amidohydrolase superfamily: Bh0493 from <em>Bacillus halodurans</em> catalyzes the isomerization of <span class="small-caps">d</span>-galacturonate to <span class="small-caps">d</span>-tagaturonate. <span><span class="ref-journal">Biochemistry. </span>2008;<span class="ref-vol">47</span>:1194–1206.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18171028" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=At+the+periphery+of+the+amidohydrolase+superfamily:+Bh0493+from+Bacillus+halodurans+catalyzes+the+isomerization+of+d-galacturonate+to+d-tagaturonate&amp;author=TT+Nguyen&amp;author=S+Brown&amp;author=AA+Fedorov&amp;author=EV+Fedorov&amp;author=PC+Babbitt&amp;volume=47&amp;publication_year=2008&amp;pages=1194-1206&amp;pmid=18171028&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Marti-Arbona R, Xu C, Steele S, Weeks A, Kuty GF, Seibert CM, Raushel FM. Annotating enzymes of unknown function: <em>N</em>-formimino-<span class="small-caps">l</span>-glutamate deiminase is a member of the amidohydrolase superfamily. <span><span class="ref-journal">Biochemistry. </span>2006;<span class="ref-vol">45</span>:1997–2005.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16475788" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Annotating+enzymes+of+unknown+function:+N-formimino-l-glutamate+deiminase+is+a+member+of+the+amidohydrolase+superfamily&amp;author=R+Marti-Arbona&amp;author=C+Xu&amp;author=S+Steele&amp;author=A+Weeks&amp;author=GF+Kuty&amp;volume=45&amp;publication_year=2006&amp;pages=1997-2005&amp;pmid=16475788&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">Hermann JC, Marti-Arbona R, Fedorov AA, Fedorov E, Almo SC, Shoichet BK, Raushel FM. Structure-based activity prediction for an enzyme of unknown function. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">448</span>:775–779.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2254328/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17603473" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structure-based+activity+prediction+for+an+enzyme+of+unknown+function&amp;author=JC+Hermann&amp;author=R+Marti-Arbona&amp;author=AA+Fedorov&amp;author=E+Fedorov&amp;author=SC+Almo&amp;volume=448&amp;publication_year=2007&amp;pages=775-779&amp;pmid=17603473&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">Pieper U, Chiang R, Seffernick JJ, Brown SD, Glasner ME, Kelly L, Eswar N, Sauder JM, Bonanno JB, Swaminathan S, Burley SK, Zheng X, Chance MR, Almo SC, Gerlt JA, Raushel FM, Jacobson MP, Babbitt PC, Sali A. Target selection and annotation for the structural genomics of the amidohydrolase and enolase superfamilies. <span><span class="ref-journal">J. Struct. Funct. Genomics. </span>2009;<span class="ref-vol">10</span>:107–125.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2693957/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19219566" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Struct.+Funct.+Genomics&amp;title=Target+selection+and+annotation+for+the+structural+genomics+of+the+amidohydrolase+and+enolase+superfamilies&amp;author=U+Pieper&amp;author=R+Chiang&amp;author=JJ+Seffernick&amp;author=SD+Brown&amp;author=ME+Glasner&amp;volume=10&amp;publication_year=2009&amp;pages=107-125&amp;pmid=19219566&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Hermann JC, Ghanem E, Li Y, Raushel FM, Irwin JJ, Shoichet BK. Predicting substrates by docking high-energy intermediates to enzyme structures. <span><span class="ref-journal">J. Am. Chem. Soc. </span>2006;<span class="ref-vol">128</span>:15882–15891.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17147401" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Am.+Chem.+Soc&amp;title=Predicting+substrates+by+docking+high-energy+intermediates+to+enzyme+structures&amp;author=JC+Hermann&amp;author=E+Ghanem&amp;author=Y+Li&amp;author=FM+Raushel&amp;author=JJ+Irwin&amp;volume=128&amp;publication_year=2006&amp;pages=15882-15891&amp;pmid=17147401&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Irwin JJ, Raushel FM, Shoichet BK. Virtual screening against metalloenzymes for inhibitors and substrates. <span><span class="ref-journal">Biochemistry. </span>2005;<span class="ref-vol">44</span>:12316–12328.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16156645" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Virtual+screening+against+metalloenzymes+for+inhibitors+and+substrates&amp;author=JJ+Irwin&amp;author=FM+Raushel&amp;author=BK+Shoichet&amp;volume=44&amp;publication_year=2005&amp;pages=12316-12328&amp;pmid=16156645&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Xiang DF, Xu C, Kumaran D, Brown AC, Sauder JM, Burley SK, Swaminathan S, Raushel FM. Functional annotation of two new carboxypeptidases from the amidohydrolase superfamily of enzymes. <span><span class="ref-journal">Biochemistry. </span>2009;<span class="ref-vol">48</span>:4567–4576.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2748308/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19358546" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Functional+annotation+of+two+new+carboxypeptidases+from+the+amidohydrolase+superfamily+of+enzymes&amp;author=DF+Xiang&amp;author=C+Xu&amp;author=D+Kumaran&amp;author=AC+Brown&amp;author=JM+Sauder&amp;volume=48&amp;publication_year=2009&amp;pages=4567-4576&amp;pmid=19358546&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Xiang DF, Kolb P, Fedorov AA, Meier MM, Fedorov LV, Nguyen TT, Sterner R, Almo SC, Shoichet BK, Raushel FM. Functional annotation and three-dimensional structure of Dr0930 from <em>Deinococcus radiodurans</em>, a close relative of phosphotriesterase in the amidohydrolase superfamily. <span><span class="ref-journal">Biochemistry. </span>2009;<span class="ref-vol">48</span>:2237–2247.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3176505/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19159332" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Functional+annotation+and+three-dimensional+structure+of+Dr0930+from+Deinococcus+radiodurans,+a+close+relative+of+phosphotriesterase+in+the+amidohydrolase+superfamily&amp;author=DF+Xiang&amp;author=P+Kolb&amp;author=AA+Fedorov&amp;author=MM+Meier&amp;author=LV+Fedorov&amp;volume=48&amp;publication_year=2009&amp;pages=2237-2247&amp;pmid=19159332&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">Holm L, Sander C. An evolutionary treasure: unification of a broad set of amidohydrolases related to urease. <span><span class="ref-journal">Proteins. </span>1997;<span class="ref-vol">28</span>:72–82.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9144792" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=An+evolutionary+treasure:+unification+of+a+broad+set+of+amidohydrolases+related+to+urease&amp;author=L+Holm&amp;author=C+Sander&amp;volume=28&amp;publication_year=1997&amp;pages=72-82&amp;pmid=9144792&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Seibert CM, Raushel FM. Structural and catalytic diversity within the amidohydrolase superfamily. <span><span class="ref-journal">Biochemistry. </span>2005;<span class="ref-vol">44</span>:6383–6391.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15850372" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Structural+and+catalytic+diversity+within+the+amidohydrolase+superfamily&amp;author=CM+Seibert&amp;author=FM+Raushel&amp;volume=44&amp;publication_year=2005&amp;pages=6383-6391&amp;pmid=15850372&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">Martynowski D, Eyobo Y, Li T, Yang K, Liu A, Zhang H. Crystal structure of alpha-amino-beta-carboxymuconate-epsilon-semialdehyde decarboxylase: insight into the active site and catalytic mechanism of a novel decarboxylation reaction. <span><span class="ref-journal">Biochemistry. </span>2006;<span class="ref-vol">45</span>:10412–10421.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16939194" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Crystal+structure+of+alpha-amino-beta-carboxymuconate-epsilon-semialdehyde+decarboxylase:+insight+into+the+active+site+and+catalytic+mechanism+of+a+novel+decarboxylation+reaction&amp;author=D+Martynowski&amp;author=Y+Eyobo&amp;author=T+Li&amp;author=K+Yang&amp;author=A+Liu&amp;volume=45&amp;publication_year=2006&amp;pages=10412-10421&amp;pmid=16939194&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">Liu A, Zhang H. Transition metal-catalyzed nonoxidative decarboxylation reactions. <span><span class="ref-journal">Biochemistry. </span>2006;<span class="ref-vol">45</span>:10407–10411.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16939193" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Transition+metal-catalyzed+nonoxidative+decarboxylation+reactions&amp;author=A+Liu&amp;author=H+Zhang&amp;volume=45&amp;publication_year=2006&amp;pages=10407-10411&amp;pmid=16939193&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Hara H, Masai E, Katayama Y, Fukuda M. The 4-oxalomesaconate hydratase gene, involved in the protocatechuate 4,5-cleavage pathway, is essential to vanillate and syringate degradation in <em>Sphingomonas paucimobilis SYK-6</em>. <span><span class="ref-journal">J. Bacteriol. </span>2000;<span class="ref-vol">182</span>:6950–6957.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC94820/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11092855" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Bacteriol&amp;title=The+4-oxalomesaconate+hydratase+gene,+involved+in+the+protocatechuate+4,5-cleavage+pathway,+is+essential+to+vanillate+and+syringate+degradation+in+Sphingomonas+paucimobilis+SYK-6&amp;author=H+Hara&amp;author=E+Masai&amp;author=Y+Katayama&amp;author=M+Fukuda&amp;volume=182&amp;publication_year=2000&amp;pages=6950-6957&amp;pmid=11092855&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Williams L, Nguyen T, Li Y, Porter TN, Raushel FM. Uronate isomerase: a nonhydrolytic member of the amidohydrolase superfamily with an ambivalent requirement for a divalent metal ion. <span><span class="ref-journal">Biochemistry. </span>2006;<span class="ref-vol">45</span>:7453–7462.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2505117/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16768441" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Uronate+isomerase:+a+nonhydrolytic+member+of+the+amidohydrolase+superfamily+with+an+ambivalent+requirement+for+a+divalent+metal+ion&amp;author=L+Williams&amp;author=T+Nguyen&amp;author=Y+Li&amp;author=TN+Porter&amp;author=FM+Raushel&amp;volume=45&amp;publication_year=2006&amp;pages=7453-7462&amp;pmid=16768441&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="citation">Nguyen TT, Fedorov AA, Williams L, Fedorov EV, Li Y, Xu C, Almo SC, Raushel FM. The mechanism of the reaction catalyzed by uronate isomerase illustrates how an isomerase may have evolved from a hydrolase within the amidohydrolase superfamily. <span><span class="ref-journal">Biochemistry. </span>2009;<span class="ref-vol">48</span>:8879–8890.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2773443/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19678710" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=The+mechanism+of+the+reaction+catalyzed+by+uronate+isomerase+illustrates+how+an+isomerase+may+have+evolved+from+a+hydrolase+within+the+amidohydrolase+superfamily&amp;author=TT+Nguyen&amp;author=AA+Fedorov&amp;author=L+Williams&amp;author=EV+Fedorov&amp;author=Y+Li&amp;volume=48&amp;publication_year=2009&amp;pages=8879-8890&amp;pmid=19678710&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="citation">Atkinson HJ, Morris JH, Ferrin TE, Babbitt PC. Using Sequence Similarity Networks for Visualization of Relationships Across Diverse Protein Superfamilies. <span><span class="ref-journal">PLoS ONE. </span>2009;<span class="ref-vol">4</span>:e4345.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2631154/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19190775" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Using+Sequence+Similarity+Networks+for+Visualization+of+Relationships+Across+Diverse+Protein+Superfamilies&amp;author=HJ+Atkinson&amp;author=JH+Morris&amp;author=TE+Ferrin&amp;author=PC+Babbitt&amp;volume=4&amp;publication_year=2009&amp;pages=e4345&amp;pmid=19190775&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="citation">Hall RS, Fedorov AA, Marti-Arbona R, Fedorov EV, Kolb P, Sauder JM, Burley SK, Shoichet BK, Almo SC, Raushel FM. The Hunt for 8-Oxoguanine Deaminase. <span><span class="ref-journal">J. Am. Chem. Soc. </span>2010;<span class="ref-vol">132</span>:1762–1763.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2820149/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20088583" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Am.+Chem.+Soc&amp;title=The+Hunt+for+8-Oxoguanine+Deaminase&amp;author=RS+Hall&amp;author=AA+Fedorov&amp;author=R+Marti-Arbona&amp;author=EV+Fedorov&amp;author=P+Kolb&amp;volume=132&amp;publication_year=2010&amp;pages=1762-1763&amp;pmid=20088583&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="citation">Otwinowski Z, Minor W. Processing of X-ray diffraction data collected in oscillation mode. <span><span class="ref-journal">Methods Enzymol. </span>1997;<span class="ref-vol">276</span>:307–326.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27754618" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Enzymol&amp;title=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode&amp;author=Z+Otwinowski&amp;author=W+Minor&amp;volume=276&amp;publication_year=1997&amp;pages=307-326&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="citation">Schneider TR, Sheldrick GM. Substructure solution with SHELXD. <span><span class="ref-journal">Acta Crystallogr. </span>2002;<span class="ref-vol">D 58</span>:1772–1779.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12351820" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Crystallogr&amp;title=Substructure+solution+with+SHELXD&amp;author=TR+Schneider&amp;author=GM+Sheldrick&amp;volume=D+58&amp;publication_year=2002&amp;pages=1772-1779&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="citation">De-La-Fortelle E, Bricogne G. Maximum-likelihood heavy atom parameter refinement in the MIR and MAD methods. <span><span class="ref-journal">Methods Enzymol. </span>1997;<span class="ref-vol">276</span>:472–493.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27799110" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Enzymol&amp;title=Maximum-likelihood+heavy+atom+parameter+refinement+in+the+MIR+and+MAD+methods&amp;author=E+De-La-Fortelle&amp;author=G+Bricogne&amp;volume=276&amp;publication_year=1997&amp;pages=472-493&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="citation">The CCP4 suite: programs for protein crystallography. <span><span class="ref-journal">Acta Crystallogr. D Biol. Crystallogr. </span>1994;<span class="ref-vol">50</span>:760–763.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15299374" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Crystallogr.+D+Biol.+Crystallogr&amp;title=The+CCP4+suite:+programs+for+protein+crystallography&amp;volume=50&amp;publication_year=1994&amp;pages=760-763&amp;pmid=15299374&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="citation">Perrakis A, Morris R, Lamzin VS. Automated protein model building combined with iterative structure refinement. <span><span class="ref-journal">Nature Struct. Biol. </span>1999;<span class="ref-vol">6</span>:458–463.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10331874" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Struct.+Biol&amp;title=Automated+protein+model+building+combined+with+iterative+structure+refinement&amp;author=A+Perrakis&amp;author=R+Morris&amp;author=VS+Lamzin&amp;volume=6&amp;publication_year=1999&amp;pages=458-463&amp;pmid=10331874&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="citation">Jones TA, Zou J-Y, Cowan SW, Kjeldgaard M. Improved methods in building protein models in electron density map and the location of errors in these models. <span><span class="ref-journal">Acta Crystallogr. </span>1991;<span class="ref-vol">A 47</span>:110–119.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2025413" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Crystallogr&amp;title=Improved+methods+in+building+protein+models+in+electron+density+map+and+the+location+of+errors+in+these+models&amp;author=TA+Jones&amp;author=J-Y+Zou&amp;author=SW+Cowan&amp;author=M+Kjeldgaard&amp;volume=A+47&amp;publication_year=1991&amp;pages=110-119&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="citation">Brunger AT, Adams PD, Clore GM, Delano WL, Gros P, Grosse-Kunstleve RW, Jiang JS, Kuszwewski J, Nilges M, Pannu NS, Read RJ, Rice LM, Somonsom T, Warren GL. Crystallography &#x00026; NMR system: a new software suite for macromolecular structure determination. <span><span class="ref-journal">Acta Crystallogr. </span>1998;<span class="ref-vol">D 54</span>:905–921.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9757107" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Crystallogr&amp;title=Crystallography+&#x00026;+NMR+system:+a+new+software+suite+for+macromolecular+structure+determination&amp;author=AT+Brunger&amp;author=PD+Adams&amp;author=GM+Clore&amp;author=WL+Delano&amp;author=P+Gros&amp;volume=D+54&amp;publication_year=1998&amp;pages=905-921&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="citation">Ireton GC, McDermott G, Black ME, Stoddard BL. The structure of <em>Escherichia coli</em> cytosine deaminase. <span><span class="ref-journal">J. Mol. Biol. </span>2002;<span class="ref-vol">315</span>:687–697.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11812140" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Mol.+Biol&amp;title=The+structure+of+Escherichia+coli+cytosine+deaminase&amp;author=GC+Ireton&amp;author=G+McDermott&amp;author=ME+Black&amp;author=BL+Stoddard&amp;volume=315&amp;publication_year=2002&amp;pages=687-697&amp;pmid=11812140&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="citation">Wilson DK, Rudolph FB, Quiocho FA. Atomic structure of adenosine deaminase complexed with a transition-state analog: understanding catalysis and immunodeficiency mutations. <span><span class="ref-journal">Science. </span>1991;<span class="ref-vol">252</span>:1278–1284.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/1925539" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Atomic+structure+of+adenosine+deaminase+complexed+with+a+transition-state+analog:+understanding+catalysis+and+immunodeficiency+mutations&amp;author=DK+Wilson&amp;author=FB+Rudolph&amp;author=FA+Quiocho&amp;volume=252&amp;publication_year=1991&amp;pages=1278-1284&amp;pmid=1925539&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="citation">Levy CC, Mc NW. The biological transformation of xanthopterin by a bacterium isolated from soil. <span><span class="ref-journal">Biochemistry. </span>1962;<span class="ref-vol">1</span>:1161–1170.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/13930253" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=The+biological+transformation+of+xanthopterin+by+a+bacterium+isolated+from+soil&amp;author=CC+Levy&amp;author=NW+Mc&amp;volume=1&amp;publication_year=1962&amp;pages=1161-1170&amp;pmid=13930253&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="citation">Ziegler I, McDonald T, Hesslinger C, Pelletier I, Boyle P. Development of the pteridine pathway in the zebrafish. <span><span class="ref-journal">J. Biol. Chem. </span>2000;<span class="ref-vol">275</span>:18926–18932.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10770954" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=Development+of+the+pteridine+pathway+in+the+zebrafish&amp;author=I+Ziegler&amp;author=T+McDonald&amp;author=C+Hesslinger&amp;author=I+Pelletier&amp;author=P+Boyle&amp;volume=275&amp;publication_year=2000&amp;pages=18926-18932&amp;pmid=10770954&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="citation">Bel Y, Porcar M, Socha R, Nemec V, Ferre J. Analysis of pteridines in <em>Pyrrhocoris apterus</em> (L.) (Heteroptera, Pyrrhocoridae) during development and in body-color mutants. <span><span class="ref-journal">Arch. Insect Biochem. Physiol. </span>1997;<span class="ref-vol">34</span>:83–98.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Insect+Biochem.+Physiol&amp;title=Analysis+of+pteridines+in+Pyrrhocoris+apterus+(L.)+(Heteroptera,+Pyrrhocoridae)+during+development+and+in+body-color+mutants&amp;author=Y+Bel&amp;author=M+Porcar&amp;author=R+Socha&amp;author=V+Nemec&amp;author=J+Ferre&amp;volume=34&amp;publication_year=1997&amp;pages=83-98&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="citation">Andondonskaja-Renz B, Zeitler HJ. Determination of pteridines in royal jelly and caviar by reverse phase high-performance liquid chromatography. <span><span class="ref-journal">J. Micronutr. Anal. </span>1989;<span class="ref-vol">5</span>:83–90.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Micronutr.+Anal&amp;title=Determination+of+pteridines+in+royal+jelly+and+caviar+by+reverse+phase+high-performance+liquid+chromatography&amp;author=B+Andondonskaja-Renz&amp;author=HJ+Zeitler&amp;volume=5&amp;publication_year=1989&amp;pages=83-90&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="citation">Gyure WL. Catabolism of isoxanthopterin during the development of the silkworm, <em>Bombyx mori</em>. <span><span class="ref-journal">Insect Biochem. </span>1974;<span class="ref-vol">4</span>:303–312.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Insect+Biochem&amp;title=Catabolism+of+isoxanthopterin+during+the+development+of+the+silkworm,+Bombyx+mori&amp;author=WL+Gyure&amp;volume=4&amp;publication_year=1974&amp;pages=303-312&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="citation">McNutt WS., Jr. The metabolism of isoxanthopterin by <em>Alcaligenes faecalis</em>. <span><span class="ref-journal">J. Biol. Chem. </span>1963;<span class="ref-vol">238</span>:1116–1121.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/13932252" ref="reftype=pubmed&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=The+metabolism+of+isoxanthopterin+by+Alcaligenes+faecalis&amp;author=WS+McNutt&amp;volume=238&amp;publication_year=1963&amp;pages=1116-1121&amp;pmid=13932252&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2892419&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC2892419/pdf/nihms-199947.pdf" class="int-view">PDF (3.6M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/2892419/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/2892419/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="2892419" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2892419%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="jFo4CpdHDj1AtQyBEtZRdpp55tOQVH8d1eLuRUHbIEXlVJU6uREWmomFwpOhuKph">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2892419%2F&amp;text=Discovery%20and%20Structure%20Determination%20of%20the%20Orphan%20Enzyme%20Isoxanthopterin%20Deaminase" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2892419%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2892419/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC2892419/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/20415463/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC2892419/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/20415463/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC2892419/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/2892419/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript" src="/core/mathjax/2.7.9/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC2892419/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
