
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Quest for Novel Preventive and Therapeutic Options Against Multidrug-Resistant Pseudomonas aeruginosa - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="939BE807DB0F4BC500004F8D6DFFE76A.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="phenaturepg" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC8351238/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8351238/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="International Journal of Peptide Research and Therapeutics" /><meta name="citation_title" content="Quest for Novel Preventive and Therapeutic Options Against Multidrug-Resistant Pseudomonas aeruginosa" /><meta name="citation_author" content="Sidra Irum" /><meta name="citation_author_institution" content="Department of Industrial Biotechnology, Atta-ur-Rahman School of Applied Biosciences (ASAB), National University of Science &#x00026; Technology (NUST), Sector H-12, Islamabad, 44000 Pakistan" /><meta name="citation_author" content="Saadia Andleeb" /><meta name="citation_author_institution" content="Department of Industrial Biotechnology, Atta-ur-Rahman School of Applied Biosciences (ASAB), National University of Science &#x00026; Technology (NUST), Sector H-12, Islamabad, 44000 Pakistan" /><meta name="citation_author" content="Amjad Ali" /><meta name="citation_author_institution" content="Department of Industrial Biotechnology, Atta-ur-Rahman School of Applied Biosciences (ASAB), National University of Science &#x00026; Technology (NUST), Sector H-12, Islamabad, 44000 Pakistan" /><meta name="citation_author" content="Muhammad Ibrahim Rashid" /><meta name="citation_author_institution" content="Department of Industrial Biotechnology, Atta-ur-Rahman School of Applied Biosciences (ASAB), National University of Science &#x00026; Technology (NUST), Sector H-12, Islamabad, 44000 Pakistan" /><meta name="citation_author_institution" content="Institute of Basic Medical Sciences, Khyber Medical University, Peshawar, Pakistan" /><meta name="citation_author" content="Mahnoor Majid" /><meta name="citation_author_institution" content="Department of Industrial Biotechnology, Atta-ur-Rahman School of Applied Biosciences (ASAB), National University of Science &#x00026; Technology (NUST), Sector H-12, Islamabad, 44000 Pakistan" /><meta name="citation_publication_date" content="2021" /><meta name="citation_issue" content="4" /><meta name="citation_volume" content="27" /><meta name="citation_firstpage" content="2313" /><meta name="citation_doi" content="10.1007/s10989-021-10255-3" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC8351238/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC8351238/" /><meta name="citation_pmid" content="34393689" /><meta name="DC.Title" content="Quest for Novel Preventive and Therapeutic Options Against Multidrug-Resistant Pseudomonas aeruginosa" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Nature Publishing Group" /><meta name="DC.Contributor" content="Sidra Irum" /><meta name="DC.Contributor" content="Saadia Andleeb" /><meta name="DC.Contributor" content="Amjad Ali" /><meta name="DC.Contributor" content="Muhammad Ibrahim Rashid" /><meta name="DC.Contributor" content="Mahnoor Majid" /><meta name="DC.Date" content="2021" /><meta name="DC.Identifier" content="10.1007/s10989-021-10255-3" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Quest for Novel Preventive and Therapeutic Options Against Multidrug-Resistant Pseudomonas aeruginosa" /><meta property="og:type" content="article" /><meta property="og:description" content="Pseudomonas aeruginosa (P. aeruginosa) is a critical healthcare challenge due to its ability to cause persistent infections and the acquisition of antibiotic resistance mechanisms. Lack of preventive vaccines and rampant drug resistance phenomenon has ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8351238/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC8351238/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=Springer%20Nature%20PMC%20COVID-19%20Collection[filter]" class="navlink">Springer Nature - PMC COVID-19 Collection</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC8351238
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC8351238/pdf/10989_2021_Article_10255.pdf" class="int-view">PDF (2.7M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/8351238/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/8351238/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="8351238" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC8351238%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="VbZLQsINPHKT9l1LQrp7qBO29gwHEhRkDQIFKaa1YnCLZTfwJ96DmOTX6sYZ6rfh">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC8351238%2F&amp;text=Quest%20for%20Novel%20Preventive%20and%20Therapeutic%20Options%20Against%20Multidrug-Resistant%20Pseudomonas%20aeruginosa" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC8351238%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8351238/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC8351238/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/34393689/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC8351238/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/34393689/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC8351238/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/8351238/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=Springer%20Nature%20PMC%20COVID-19%20Collection[filter]" class="navlink">Springer Nature - PMC COVID-19 Collection</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC8351238
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-phenaturepg.png" alt="Logo of phenaturepg" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="https://www.ncbi.nlm.nih.gov/pmc/about/covid-19/" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Int J Pept Res Ther.</span> 2021; 27(4): 2313–2331. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2021 Aug 9. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1007%2Fs10989-021-10255-3" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1007/s10989-021-10255-3</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC8351238</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/34393689">34393689</a></div></div></div><h1 class="content-title">Quest for Novel Preventive and Therapeutic Options Against Multidrug-Resistant <em>Pseudomonas aeruginosa</em></h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Irum%20S%5BAuthor%5D" class="affpopup" co-rid="_co_idm139948924111728" co-class="co-affbox">Sidra Irum</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Andleeb%20S%5BAuthor%5D" class="affpopup" co-rid="_co_idm139948924097248" co-class="co-affbox">Saadia Andleeb</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ali%20A%5BAuthor%5D" class="affpopup" co-rid="_co_idm139948924093440" co-class="co-affbox">Amjad Ali</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Rashid%20MI%5BAuthor%5D" class="affpopup" co-rid="_co_idm139948916466960" co-class="co-affbox">Muhammad Ibrahim Rashid</a>,<sup>1,</sup><sup>2</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Majid%20M%5BAuthor%5D" class="affpopup" co-rid="_co_idm139948916463856" co-class="co-affbox">Mahnoor Majid</a><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139948924111728"><h3 class="no_margin">Sidra Irum</h3><p><sup>1</sup>Department of Industrial Biotechnology, Atta-ur-Rahman School of Applied Biosciences (ASAB), National University of Science &#x00026; Technology (NUST), Sector H-12, Islamabad, 44000 Pakistan </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Irum%20S%5BAuthor%5D">Sidra Irum</a></div></div><div id="_co_idm139948924097248"><h3 class="no_margin">Saadia Andleeb</h3><p><sup>1</sup>Department of Industrial Biotechnology, Atta-ur-Rahman School of Applied Biosciences (ASAB), National University of Science &#x00026; Technology (NUST), Sector H-12, Islamabad, 44000 Pakistan </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Andleeb%20S%5BAuthor%5D">Saadia Andleeb</a></div></div><div id="_co_idm139948924093440"><h3 class="no_margin">Amjad Ali</h3><p><sup>1</sup>Department of Industrial Biotechnology, Atta-ur-Rahman School of Applied Biosciences (ASAB), National University of Science &#x00026; Technology (NUST), Sector H-12, Islamabad, 44000 Pakistan </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ali%20A%5BAuthor%5D">Amjad Ali</a></div></div><div id="_co_idm139948916466960"><h3 class="no_margin">Muhammad Ibrahim Rashid</h3><p><sup>1</sup>Department of Industrial Biotechnology, Atta-ur-Rahman School of Applied Biosciences (ASAB), National University of Science &#x00026; Technology (NUST), Sector H-12, Islamabad, 44000 Pakistan </p><p><sup>2</sup>Institute of Basic Medical Sciences, Khyber Medical University, Peshawar, Pakistan </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Rashid%20MI%5BAuthor%5D">Muhammad Ibrahim Rashid</a></div></div><div id="_co_idm139948916463856"><h3 class="no_margin">Mahnoor Majid</h3><p><sup>1</sup>Department of Industrial Biotechnology, Atta-ur-Rahman School of Applied Biosciences (ASAB), National University of Science &#x00026; Technology (NUST), Sector H-12, Islamabad, 44000 Pakistan </p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Majid%20M%5BAuthor%5D">Mahnoor Majid</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm139948921611824_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139948921611824_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139948921611824_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm139948921611824_ai" style="display:none"><div class="fm-affl" id="Aff1"><sup>1</sup>Department of Industrial Biotechnology, Atta-ur-Rahman School of Applied Biosciences (ASAB), National University of Science &#x00026; Technology (NUST), Sector H-12, Islamabad, 44000 Pakistan </div><div class="fm-affl" id="Aff2"><sup>2</sup>Institute of Basic Medical Sciences, Khyber Medical University, Peshawar, Pakistan </div><div><span class="fm-affl">Saadia Andleeb, </span><span class="fm-affl"><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="kp.ude.tsun.basa@beeldna.aidaas" class="oemail">kp.ude.tsun.basa@beeldna.aidaas</a><span style="unicode-bidi:bidi-override">, </span><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="moc.oohay@tawramaidaaS" class="oemail">moc.oohay@tawramaidaaS</a></span>.</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup>Corresponding author.</div></div><div class="fm-article-notes hide half_rhythm" id="idm139948921611824_an" style="display:none"><div class="fm-pubdate half_rhythm">Accepted 2021 Jul 9.</div></div><div class="permissions half_rhythm hide" id="idm139948921611824_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; The Author(s), under exclusive licence to Springer Nature B.V. 2021</div><div class="license half_rhythm">This article is made available via the PMC Open Access Subset for unrestricted research re-use and secondary analysis in any form or by any means with acknowledgement of the original source. These permissions are granted for the duration of the World Health Organization (WHO) declaration of COVID-19 as a global pandemic.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="7" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary file1 (JPG 345 kb)</div><div><a href="/pmc/articles/PMC8351238/bin/10989_2021_10255_MOESM1_ESM.jpg" data-ga-action="click_feat_suppl">10989_2021_10255_MOESM1_ESM.jpg</a><span style="color:gray"> (345K)</span></div><div class="small guid">GUID: 18B69508-B872-40F5-B396-6737B0BBC550</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary file2 (JPG 154 kb)</div><div><a href="/pmc/articles/PMC8351238/bin/10989_2021_10255_MOESM2_ESM.jpg" data-ga-action="click_feat_suppl">10989_2021_10255_MOESM2_ESM.jpg</a><span style="color:gray"> (154K)</span></div><div class="small guid">GUID: 86865A3D-6846-4396-BFFB-9DF50EE82914</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary file3 (JPG 545 kb)</div><div><a href="/pmc/articles/PMC8351238/bin/10989_2021_10255_MOESM3_ESM.jpg" data-ga-action="click_feat_suppl">10989_2021_10255_MOESM3_ESM.jpg</a><span style="color:gray"> (545K)</span></div><div class="small guid">GUID: 860EE392-D800-4CAE-AA91-9DF1110B9069</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary file4 (JPG 416 kb)</div><div><a href="/pmc/articles/PMC8351238/bin/10989_2021_10255_MOESM4_ESM.jpg" data-ga-action="click_feat_suppl">10989_2021_10255_MOESM4_ESM.jpg</a><span style="color:gray"> (416K)</span></div><div class="small guid">GUID: DC6CA2E7-AF98-4BE0-BA34-129A8AD8DCAB</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary file5 (XLSX 40066 kb)</div><div><a href="/pmc/articles/PMC8351238/bin/10989_2021_10255_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl">10989_2021_10255_MOESM5_ESM.xlsx</a><span style="color:gray"> (39M)</span></div><div class="small guid">GUID: CC1835D5-667F-42DA-B4CD-C00C8A907EB0</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary file6 (DOCX 14 kb)</div><div><a href="/pmc/articles/PMC8351238/bin/10989_2021_10255_MOESM6_ESM.docx" data-ga-action="click_feat_suppl">10989_2021_10255_MOESM6_ESM.docx</a><span style="color:gray"> (15K)</span></div><div class="small guid">GUID: 873A197A-1C4C-4B9C-9336-86D055FAE075</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary file7 (DOCX 13 kb)</div><div><a href="/pmc/articles/PMC8351238/bin/10989_2021_10255_MOESM7_ESM.docx" data-ga-action="click_feat_suppl">10989_2021_10255_MOESM7_ESM.docx</a><span style="color:gray"> (13K)</span></div><div class="small guid">GUID: 060A1A18-14EE-43F7-B675-FE30E3772458</div></div></dd></dl><dl data-length="211" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p>All data generated or analyzed during the study are included in the submitted manuscript. The sequences of the protein analyzed can be retrieved from UniProt database (uniport.org) using their accession numbers.</p></dd></dl></div><div id="Abs1" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="Abs1title">Abstract</h2><!--article-meta--><div><p id="Par1" class="p p-first"><em>Pseudomonas aeruginosa</em> (<em>P. aeruginosa</em>) is a critical healthcare challenge due to its ability to cause persistent infections and the acquisition of antibiotic resistance mechanisms. Lack of preventive vaccines and rampant drug resistance phenomenon has rendered patients vulnerable. As new antimicrobials are in the preclinical stages of development, mining for the unexploited drug targets is also crucial. In the present study, we designed a B- and T-cell multi-epitope vaccine against <em>P. aeruginosa</em> using a subtractive proteomics and immunoinformatics approach. A total of five proteins were shortlisted based on essentiality, extracellular localization, virulence, antigenicity, pathway association, hydrophilicity, and low molecular weight. These include two outer membrane porins; <em>OprF</em> (<a id="__tag_1133178643" class="tag_hotlink" href="/protein/P13794" ref="/protein/P13794"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"P13794","term_id":"130680","term_text":"P13794"}}</span>P13794</a>) and <em>OprD</em> (<a id="__tag_1133178586" class="tag_hotlink" href="/protein/P32722" ref="/protein/P32722"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"P32722","term_id":"417518","term_text":"P32722"}}</span>P32722</a>), a protein activator precursor <em>pra</em> (G3XDA9), a probable outer membrane protein precursor PA1288 (<a id="__tag_1133178639" class="tag_hotlink" href="/protein/Q9I456" ref="/protein/Q9I456"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9I456","term_id":"81541503","term_text":"Q9I456"}}</span>Q9I456</a>), and a conserved hypothetical protein PA4874 (<a id="__tag_1133178583" class="tag_hotlink" href="/protein/Q9HUT9" ref="/protein/Q9HUT9"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9HUT9","term_id":"81539695","term_text":"Q9HUT9"}}</span>Q9HUT9</a>). These shortlisted proteins were further analyzed to identify immunogenic and antigenic B- and T-cell epitopes. The best scoring epitopes were then further subjected to the construction of a polypeptide multi-epitope vaccine and joined with cholera toxin B subunit adjuvant. The final chimeric construct was docked with TLR4 and confirmed by normal mode simulation studies. The designed B- and T-cell multi-epitope vaccine candidate is predicted immunogenic in nature and has shown strong interactions with TLR-4. Immune simulation predicted high-level production of B- and T-cell population and maximal expression was ensured in <em>E. coli</em> strain K12. The identified drug targets qualifying the screening criteria were: UDP-2-acetamido-2-deoxy-d-glucuronic acid 3-dehydrogenase WbpB (<a id="__tag_1133178632" class="tag_hotlink" href="/protein/G3XD23" ref="/protein/G3XD23"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"G3XD23","term_id":"408387591","term_text":"G3XD23"}}</span>G3XD23</a>), aspartate semialdehyde dehydrogenase (<a id="__tag_1133178604" class="tag_hotlink" href="/protein/Q51344" ref="/protein/Q51344"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q51344","term_id":"12230918","term_text":"Q51344"}}</span>Q51344</a>), 2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase (<a id="__tag_1133178594" class="tag_hotlink" href="/protein/Q9HV71" ref="/protein/Q9HV71"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9HV71","term_id":"81539759","term_text":"Q9HV71"}}</span>Q9HV71</a>), 3-deoxy-D-manno-octulosonic-acid transferase (<a id="__tag_1133178612" class="tag_hotlink" href="/protein/Q9HUH7" ref="/protein/Q9HUH7"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9HUH7","term_id":"81539636","term_text":"Q9HUH7"}}</span>Q9HUH7</a>), glycyl-tRNA synthetase alpha chain (<a id="__tag_1133178617" class="tag_hotlink" href="/protein/Q9I7B7" ref="/protein/Q9I7B7"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9I7B7","term_id":"25009323","term_text":"Q9I7B7"}}</span>Q9I7B7</a>), riboflavin kinase/FAD synthase (<a id="__tag_1133178641" class="tag_hotlink" href="/protein/Q9HVM3" ref="/protein/Q9HVM3"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9HVM3","term_id":"81539848","term_text":"Q9HVM3"}}</span>Q9HVM3</a>), aconitate hydratase 2 (<a id="__tag_1133178628" class="tag_hotlink" href="/protein/Q9I2V5" ref="/protein/Q9I2V5"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9I2V5","term_id":"81541220","term_text":"Q9I2V5"}}</span>Q9I2V5</a>), probable glycosyltransferase WbpH (G3XD85) and UDP-3-O-[3-hydroxylauroyl] glucosamine N-acyltransferase (<a id="__tag_1133178637" class="tag_hotlink" href="/protein/Q9HXY6" ref="/protein/Q9HXY6"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9HXY6","term_id":"20138743","term_text":"Q9HXY6"}}</span>Q9HXY6</a>). For druggability and pocketome analysis crystal and homology structures of these proteins were retrieved and developed. A sequence-based search was performed in different databases (ChEMBL, Drug Bank, PubChem and Pseudomonas database) for the availability of reported ligands and tested drugs for the screened targets. These predicted targets may provide a basis for the development of reliable antibacterial preventive and therapeutic options against <em>P. aeruginosa</em>.</p><div id="sec-a.y.b.o.b" class="sec"><h3 id="sec-a.y.b.o.btitle">Supplementary Information</h3><p class="p p-first-last">The online version contains supplementary material available at 10.1007/s10989-021-10255-3.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text"><em>Pseudomonas aeruginosa</em>, Reverse vaccinology, Subtractive proteomics, Vaccine candidates, Chimeric vaccine, Druggable targets</span></div></div><div id="Sec1" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec1title">Introduction</h2><p id="Par2" class="p p-first">World Health Organization has classified <em>P. aeruginosa</em>, as a top priority critical pathogen against which there is a dire need to develop novel antimicrobial therapeutics (WHO <a href="#CR27" rid="CR27" class=" bibr popnode">2021</a>). The wide range of hosts (i.e. humans, plants and animals), ubiquitous environmental distribution, complex genome (carrying an enormous arsenal of virulence and resistance genes), and intrinsic antibiotic resistance potential make it one of the most unique and widespread pathogen worldwide (Wang <a href="#CR80" rid="CR80" class=" bibr popnode">2017</a>; Lister et al. <a href="#CR44" rid="CR44" class=" bibr popnode">2009</a>). <em>P. aeruginosa</em> is capable of causing both acute as well as chronic infections and is reported to harbor resistance to almost all available classes of antibiotics through chromosomal mutations and horizontal gene transfer (Rizvi <a href="#CR68" rid="CR68" class=" bibr popnode">2015</a>).</p><p id="Par3">The arsenal of virulence factors comes in handy at the sites of infection where it inflicts serious injuries to the host causing different types of cytotoxic effects leading to apoptosis, necrosis, ultimately resulting in immune evasion and immune modulation, hence paving the way for infection establishment (Rashid <a href="#CR67" rid="CR67" class=" bibr popnode">2017</a>; Gellatly and Hancock <a href="#CR25" rid="CR25" class=" bibr popnode">2013</a>; Yang <a href="#CR87" rid="CR87" class=" bibr popnode">2017</a>). Hence, the infections caused by this notorious pathogen are often difficult to eradicate despite intense antibiotic combination treatments (Lister et al. <a href="#CR44" rid="CR44" class=" bibr popnode">2009</a>).</p><p id="Par4">Lack of effective treatment options has fueled many studies for exploring preventive approaches. Enormous efforts have been put in the past few decades to develop effective vaccine candidates against <em>P. aeruginosa</em> (Priebe and Goldberg <a href="#CR61" rid="CR61" class=" bibr popnode">2014</a>). Several vaccine candidates have been tested (Fran&#x000e7;ois <a href="#CR21" rid="CR21" class=" bibr popnode">2012</a>; Westritschnig <a href="#CR82" rid="CR82" class=" bibr popnode">2014</a>), however, no candidate has been licensed yet for clinical trials (Vincent <a href="#CR78" rid="CR78" class=" bibr popnode">2014</a>). <em>P. aeruginosa</em> possesses abundant outer membrane proteins of which the function and expression of a large number is still unknown (Montor <a href="#CR51" rid="CR51" class=" bibr popnode">2009</a>). The plasticity of <em>P. aeruginosa</em> genome is another hurdle in the selection of conserved antigenic vaccine candidates (Mosquera-Rend&#x000f3;n <a href="#CR53" rid="CR53" class=" bibr popnode">2016</a>; Pennington <a href="#CR57" rid="CR57" class=" bibr popnode">1975</a>). The identification of conserved novel immunogenic candidates can potentially contribute to the development of effective vaccines.</p><p id="Par5">&#x0201c;Reverse vaccinology (RV)&#x0201d; is a much better option to prepare successful vaccines. This approach allows the identification of complete potential protein&#x02019;s repertoire of an organism using genomic and proteomic information (Delany et al. <a href="#CR16" rid="CR16" class=" bibr popnode">2013</a>). RV has been found cost-effective with high accuracy compared to the traditional vaccinology approaches and has been successfully applied in designing vaccines for serogroup <em>B. meningococcal</em> infection (Giuliani <a href="#CR26" rid="CR26" class=" bibr popnode">2006</a>). This strategy includes a detailed analysis of the genome/proteome of pathogen i.e. essential and regulatory genes of the targeted pathogen, Screening of virulence-associated proteins involved in establishment and progression of infections, and prioritized protein&#x02019;s interactions with host proteome (Rashid <a href="#CR67" rid="CR67" class=" bibr popnode">2017</a>). The prospective candidate proteins are screened for surface exposure, immunogenic potential, abundance, number of transmembrane helices and involvement in virulence (Brunham et al. <a href="#CR9" rid="CR9" class=" bibr popnode">1993</a>). <em>P. aeruginosa</em> possesses abundant outer membrane proteins (Montor <a href="#CR51" rid="CR51" class=" bibr popnode">2009</a>), of which the function and expression of a large number are still unknown.</p><p id="Par6">The basic theme of our study was the rigorous screening of <em>P. aeruginosa</em> reference strain PAO1 proteome for the identification of non-human homologous exoproteins which were subsequently screened for highly conserved novel epitopes with the ability to bind with maximum number of MHC-I and MHC-II class molecules. The epitopes were then joined together, and designed chimera was further tested for stability and ability to bind to TLR4. Finally immune simulation was performed and the multi-epitope vaccine coding gene was in silico cloned in pET28a vector for expression and purification of the designed vaccine for in vivo studies.</p><p id="Par7" class="p p-last">Genome projects and the development of bioinformatics tools have resulted in a paradigm shift in drug development research (Hopkins and Groom <a href="#CR34" rid="CR34" class=" bibr popnode">2002</a>). Conventional drug development strategies based on hunches and trial and error approaches are costly time consuming and inefficient (Adams and Brantner <a href="#CR1" rid="CR1" class=" bibr popnode">2006</a>; Craik <a href="#CR13" rid="CR13" class=" bibr popnode">2013</a>; Hedgecoe and Martin <a href="#CR32" rid="CR32" class=" bibr popnode">2003</a>). Drug Development in the age of genomics is more focused on the assessment of genes and their functions concerning particular diseases or conditions of interest. This approach works more efficiently and in a more guided manner limiting the scope of hunches and trials. The availability of a huge amount of genomic data has provided an opportunity to study these genes of interest and to use computational tools for determining their therapeutic potential before going into the lab (Lenz et al. <a href="#CR41" rid="CR41" class=" bibr popnode">2000</a>; Roses <a href="#CR69" rid="CR69" class=" bibr popnode">2000</a>). The &#x0201c;druggable genome&#x0201d; indicates the genes which hold therapeutic potential and can serve as targets for drugs (Hopkins and Groom <a href="#CR34" rid="CR34" class=" bibr popnode">2002</a>). These drug targets are involved in metabolic pathways of the organism, and one way or the other may contribute to pathophysiology. While druggability of a gene/protein is a more topological phenomenon that denotes the ability of the protein to have binding pockets for small drug-like molecules in its three-dimensional structure (Keller et al. <a href="#CR39" rid="CR39" class=" bibr popnode">2006</a>). These binding pockets provide the drug molecules with a place to bind and influence the activity of the target. A schematic overview of our adopted strategy is represented in Fig.&#x000a0;<a href="/pmc/articles/PMC8351238/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1</span></a><strong>.</strong></p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig1" co-legend-rid="lgnd_Fig1"><a href="/pmc/articles/PMC8351238/figure/Fig1/" target="figure" rid-figpopup="Fig1" rid-ob="ob-Fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139948957077584"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8351238_10989_2021_10255_Fig1_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 10989_2021_10255_Fig1_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC8351238/bin/10989_2021_10255_Fig1_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139948957077584"><a target="object" rel="noopener" href="/pmc/articles/PMC8351238/figure/Fig1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig1"><div><a class="figpopup" href="/pmc/articles/PMC8351238/figure/Fig1/" target="figure" rid-figpopup="Fig1" rid-ob="ob-Fig1">Fig. 1</a></div><!--caption a7--><div class="caption"><p>The schematic workflow overview: The <em>Pseudomonas aeruginosa</em> reference strain PAO1 was subjected to reductive genome screening for the prediction of potential vaccine candidates and identification of novel drug targets</p></div></div></div></div><div id="Sec2" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec2title">Materials and Methods</h2><div id="Sec3" class="sec sec-first"><h3 id="Sec3title">Retrieval of <em>P. aeruginosa</em> Proteome and Removal of Paralogous Sequences</h3><p id="Par8" class="p p-first-last">Primary genomic and proteomic data of <em>P. aeruginosa</em> reference strain PAO1 (RefSeq <a id="__tag_1041050567" class="tag_hotlink" href="/nuccore/NC_002516.2" ref="/nuccore/NC_002516.2"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NC_002516.2","term_id":"110645304","term_text":"NC_002516.2"}}</span>NC_002516.2</a>, Proteome; UP000002438) was retrieved from NCBI GenBank (Bilofsky and Christian <a href="#CR8" rid="CR8" class=" bibr popnode">1988</a>) and UniProt database (Bairoch and Apweiler <a href="#CR4" rid="CR4" class=" bibr popnode">2000</a>). The pan-proteome of PAO1 was subjected to a stepwise identification of vital virulent proteins (Rashid <a href="#CR67" rid="CR67" class=" bibr popnode">2017</a>). The complete proteome was checked for redundancy through CD-HIT (Li and Godzik <a href="#CR42" rid="CR42" class=" bibr popnode">2006</a>). A schematic overview of our adopted strategy is represented in Fig.&#x000a0;<a href="/pmc/articles/PMC8351238/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1</span></a>.</p></div><div id="Sec4" class="sec"><h3 id="Sec4title">Subcellular Localization and Host Non-homologous Proteins Prediction by Vaxign Pipeline</h3><p id="Par9" class="p p-first">Subcellular localization prediction is an important step for the selection of suitable vaccine and drug target candidates out of an infectious agent proteome. Proteome data was screened using the Vaxign pipeline (<a href="http://www.violinet.org/vaxign/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.violinet.org/vaxign/</a>) (Xiang and He <a href="#CR84" rid="CR84" class=" bibr popnode">2009</a>). Dynamic Vaxign analysis was performed instead of using the Vaxign query program due to the unavailability of <em>P. aeruginosa</em> genome in the list of pre-computed genome groups. Vaxign includes PSORTb for precise bacterial protein localization prediction. Due to the limitations of PSORTb in sorting proteins of multiple localization, CELLO v2.5 was also used in parallel (Yu et al. <a href="#CR88" rid="CR88" class=" bibr popnode">2004</a>).</p><p id="Par10">The topology of secreted and surface-exposed proteins was verified using TMHMM v2.0 (Tusnady and Simon <a href="#CR77" rid="CR77" class=" bibr popnode">2001</a>). Proteins with more than one transmembrane helices were removed from the shortlisted pool. The Vaxign pipeline calculated adhesins using SPAAN (Sachdeva <a href="#CR70" rid="CR70" class=" bibr popnode">2004</a>) with up to 89% sensitivity and 100% specificity. These cell-surface components are also considered good vaccine targets.</p><p id="Par11" class="p p-last">Host homologous proteins could elicit autoimmune responses and thus removal of such homologs is essential. From a pharmacokinetic perspective, the ideal vaccine or drug target must be host non-homolog. Host homology/similarity analysis was done via Vaxign customized OrthoMCL (Li et al. <a href="#CR43" rid="CR43" class=" bibr popnode">2003</a>) to calculate homology between screened/filtered proteins and protein of specific host (e.g., human). In addition to this, the Basic Local Alignment Search Tool for proteins (BLASTp) available at NCBI server (Altschul <a href="#CR2" rid="CR2" class=" bibr popnode">1990</a>) was used to filter out proteins with similarity to human proteins. Human homologs were removed from the prioritized protein pool. The cut-off criteria were set at 0.005 E-value and&#x02009;&#x0003c;&#x02009;40% sequence identity with at least 50% query coverage. Proteins having sequence similarity of&#x02009;&#x02265;&#x02009;40% were removed from the data set (Calabriso <a href="#CR11" rid="CR11" class=" bibr popnode">2016</a>). Apart from the human homologs, orthologs search across Pseudomonas genus was conducted using organism-specific BLASTp search and &#x0201c;The Pseudomonas genome Database&#x0201d; (Winsor et al. <a href="#CR83" rid="CR83" class=" bibr popnode">2011</a>).</p></div><div id="Sec5" class="sec"><h3 id="Sec5title">Screening of Pathogen Essential, Virulent and Antigenic Proteins</h3><p id="Par12" class="p p-first">Database of Essential Genes (DEG) version 10.4 (Luo <a href="#CR47" rid="CR47" class=" bibr popnode">2014</a>) was used to get information about the essentiality status of the shortlisted proteins as a potential vaccine candidate. Selected protein sequences were subjected to DEG limited BLAST run.</p><p id="Par13">Proteins involved in metabolism can also serve as ideal drug targets as inhibiting such proteins could greatly hinder the pathogen&#x02019;s survival. For this purpose, a separate DEG run was employed for screening essential proteins as a potential druggable target. The screened pool was subjected to KEGG pathway analysis for shortlisting proteins involved in the metabolism (Weigt <a href="#CR81" rid="CR81" class=" bibr popnode">2018</a>). Enzymes were selected from the pool and were again subjected to host homology evaluation.</p><p id="Par14" class="p p-last">The pool of essential exoproteome and secretome was evaluated for their potential role in virulence by using the Virulence Factor Database (VFDB) (Chen et al. <a href="#CR12" rid="CR12" class=" bibr popnode">2012</a>) and SVM-based tool, Virulentpred (Garg and Gupta <a href="#CR23" rid="CR23" class=" bibr popnode">2008</a>). Antigenicity scores of the filtered proteins were estimated by Vaxijen v2.0 (Doytchinova and Flower <a href="#CR18" rid="CR18" class=" bibr popnode">2007</a>) (threshold&#x02009;=&#x02009;0.4). Proteins with high antigenic scores were prioritized as good vaccine candidates.</p></div><div id="Sec6" class="sec"><h3 id="Sec6title">Molecular Weight Estimation of Selected Vaccine Candidate Proteins</h3><p id="Par15" class="p p-first-last">The ease with which a protein could be expressed, extracted and purified is important in the selection of potential proteins to be used for the development of peptide vaccines. For the analysis of the molecular weights of the proteins, their accession numbers were given to ExPASy (Bioinformatics resource portal) compute pI/Mw online tool (Gasteiger <a href="#CR24" rid="CR24" class=" bibr popnode">2005</a>). Proteins having a molecular weight of&#x02009;&#x02264;&#x02009;110&#x000a0;kDa were selected while the rest were excluded <strong>(</strong>Table <a href="/pmc/articles/PMC8351238/table/Tab1/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab1" rid-ob="ob-Tab1" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">Table1</span></span><span>1</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="Tab1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p>Characteristics of finalized vaccine candidate proteins</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="left" rowspan="2" colspan="1">Protein accession</th><th align="left" rowspan="2" colspan="1">Gene name</th><th align="left" rowspan="2" colspan="1">Protein product</th><th align="left" colspan="2" rowspan="1">Localization</th><th align="left" rowspan="2" colspan="1">Adhesins probability</th><th align="left" rowspan="2" colspan="1">Antigenicity score</th><th align="left" rowspan="2" colspan="1">Essentiality</th><th align="left" rowspan="2" colspan="1">Virulence</th></tr><tr><th align="left" rowspan="1" colspan="1">PSORTb</th><th align="left" rowspan="1" colspan="1">CELLO</th></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178638" class="tag_hotlink" href="/protein/P13794" ref="/protein/P13794"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"P13794","term_id":"130680","term_text":"P13794"}}</span>P13794</a></td><td align="left" rowspan="1" colspan="1">oprF</td><td align="left" rowspan="1" colspan="1">Outer membrane porin F</td><td align="left" rowspan="1" colspan="1">Outer Membrane</td><td align="left" rowspan="1" colspan="1">Outer Membrane</td><td align="center" rowspan="1" colspan="1">0.553</td><td align="center" rowspan="1" colspan="1">0.8306</td><td align="left" rowspan="1" colspan="1">&#x02713;</td><td align="left" rowspan="1" colspan="1">&#x02713;</td></tr><tr><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178610" class="tag_hotlink" href="/protein/P32722" ref="/protein/P32722"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"P32722","term_id":"417518","term_text":"P32722"}}</span>P32722</a></td><td align="left" rowspan="1" colspan="1">oprD</td><td align="left" rowspan="1" colspan="1">Porin D</td><td align="left" rowspan="1" colspan="1">Outer Membrane</td><td align="left" rowspan="1" colspan="1">Outer Membrane</td><td align="center" rowspan="1" colspan="1">0.564</td><td align="center" rowspan="1" colspan="1">0.8658</td><td align="left" rowspan="1" colspan="1">&#x02713;</td><td align="left" rowspan="1" colspan="1">&#x02713;</td></tr><tr><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178634" class="tag_hotlink" href="/protein/Q9I456" ref="/protein/Q9I456"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9I456","term_id":"81541503","term_text":"Q9I456"}}</span>Q9I456</a></td><td align="left" rowspan="1" colspan="1">PA1288</td><td align="left" rowspan="1" colspan="1">Probable outer membrane protein</td><td align="left" rowspan="1" colspan="1">Outer Membrane</td><td align="left" rowspan="1" colspan="1">Outer Membrane</td><td align="center" rowspan="1" colspan="1">0.641</td><td align="center" rowspan="1" colspan="1">0.8067</td><td align="left" rowspan="1" colspan="1">&#x02713;</td><td align="left" rowspan="1" colspan="1">&#x02713;</td></tr><tr><td align="left" rowspan="1" colspan="1">G3XDA9</td><td align="left" rowspan="1" colspan="1">pra</td><td align="left" rowspan="1" colspan="1">Protein activator</td><td align="left" rowspan="1" colspan="1">Extracellular</td><td align="left" rowspan="1" colspan="1">Extracellular</td><td align="center" rowspan="1" colspan="1">0.692</td><td align="center" rowspan="1" colspan="1">0.8073</td><td align="left" rowspan="1" colspan="1">&#x02713;</td><td align="left" rowspan="1" colspan="1">&#x02713;</td></tr><tr><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178622" class="tag_hotlink" href="/protein/Q9HUT9" ref="/protein/Q9HUT9"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9HUT9","term_id":"81539695","term_text":"Q9HUT9"}}</span>Q9HUT9</a></td><td align="left" rowspan="1" colspan="1">PA4874</td><td align="left" rowspan="1" colspan="1">Uncharacterized protein</td><td align="left" rowspan="1" colspan="1">Periplasmic</td><td align="left" rowspan="1" colspan="1">Periplasmic</td><td align="center" rowspan="1" colspan="1">0.636</td><td align="center" rowspan="1" colspan="1">0.858</td><td align="left" rowspan="1" colspan="1">&#x02713;</td><td align="left" rowspan="1" colspan="1">&#x02713;</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139948957065360"><a target="object" rel="noopener" href="/pmc/articles/PMC8351238/table/Tab1/?report=objectonly">Open in a separate window</a></div></div></div><div id="Sec7" class="sec"><h3 id="Sec7title">Prioritized Protein&#x02019;s 3-D Model Building and Druggability Assessment</h3><p id="Par16" class="p p-first">The presence of a 3D protein structure was checked in Protein Data Bank (PDB) (Berman et al. <a href="#CR7" rid="CR7" class=" bibr popnode">2006</a>) for all proteins. SWISS-MODEL (Arnold <a href="#CR3" rid="CR3" class=" bibr popnode">2006</a>) and RaptorX (K&#x000e4;llberg <a href="#CR37" rid="CR37" class=" bibr popnode">2012</a>) were used for homology modeling of proteins lacking crystal structures. For each protein, a suitable structural template was selected and 3-D models were generated.</p><p id="Par17" class="p p-last">Druggability was analyzed for each of the selected proteins for predicting the quality and number of drug binding pockets using proteins&#x02019; tertiary structures. DoGSiteScorer online server (<a href="https://proteins.plus/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://proteins.plus/</a>) was used for this purpose (Volkamer <a href="#CR79" rid="CR79" class=" bibr popnode">2012</a>). DoGSiteScorer uses a grid-based method with a Gaussian filter for the estimation of prospective binding pockets. This server uses descriptive values of volume, hydrophobicity and shape (enclosure) for determining the score ranged between 0 and 1 (1 as highest druggability potential) for the respective binding pocket. The retrieved crystal structures and predicted homology structures were submitted to the server and resultant scores and details of the predictions were saved. Drug binding pockets with estimated drug score&#x02009;&#x0003e;&#x02009;0.50 were selected. In case of multiple pockets having a higher score than the cut off top five pockets were selected.</p></div><div id="Sec8" class="sec"><h3 id="Sec8title">Protein&#x02013;Protein Interactions (PPIs) and Clusters of Orthologous Group of Proteins (COGs) Analysis</h3><p id="Par18" class="p p-first-last">Proteins interact with each other and constitute a complex network, where different proteins contribute synergistically as well as antagonistically towards common functions/pathways. STRING (Search Tool for the Retrieval of Interacting Genes/Proteins) database (Szklarczyk <a href="#CR76" rid="CR76" class=" bibr popnode">2011</a>) was used which provides information about <em>PPIs</em> and <em>COGs</em> based on sequence identity. A chimeric vaccine has the potential to affect the expression of all other interacting proteins in the bacterial cell and thus reduces the pathogenesis. The intra specie protein&#x02013;protein interactomes were compiled in a datasheet for analysis.</p></div><div id="Sec9" class="sec"><h3 id="Sec9title">Antigenic Epitopes Prediction</h3><p id="Par19" class="p p-first-last">Vaxitop online server was used to predict continuous B-cell Epitopes from selected proteins using default threshold values (He et al. <a href="#CR30" rid="CR30" class=" bibr popnode">2010</a>). The predicted peptides were reexamined for topology via TMHMM v2.0 (Tusnady and Simon <a href="#CR77" rid="CR77" class=" bibr popnode">2001</a>), and antigenicity scores using Vaxijen (Doytchinova and Flower <a href="#CR18" rid="CR18" class=" bibr popnode">2007</a>) (threshold value&#x02009;&#x0003e;&#x02009;0.4). The peptides with antigenicity scores&#x02009;&#x02265;&#x02009;0.8 were considered more antigenic and thus selected. The screening of epitopes was performed based on the least P-value and highest antigenicity score. The surface-exposed B-cell epitopes with the best antigenicity scores were evaluated for their affinity for T-cells using Tepitool server (Paul et al. <a href="#CR56" rid="CR56" class=" bibr popnode">2016</a>). Epitopes were studied for their interactions with MHC class molecules including DRB1*0101 as it is the commonest bound allele. MHCPred was used for calculation of antigenicity and IC<sub>50</sub> value (Guan <a href="#CR28" rid="CR28" class=" bibr popnode">2003</a>). Extracted prioritized peptide out of respective virulent protein candidates was again subjected to VirulentPred (Garg and Gupta <a href="#CR23" rid="CR23" class=" bibr popnode">2008</a>) (threshold value&#x02009;&#x02265;&#x02009;0.5). The topology of the selected epitopes was visualized using Pepitope server (Mayrose <a href="#CR49" rid="CR49" class=" bibr popnode">2007</a>).</p></div><div id="Sec10" class="sec"><h3 id="Sec10title">Evaluation of Genetic Diversity</h3><p id="Par20" class="p p-first-last">Epitopic sequences were checked for conservation among <em>P. aeruginosa</em> taxonomic group (Taxid: 136841) and other Pseudomonas spp. via sequence alignment. Multiple sequence alignment was performed using CLC workbench V7 and consensus sequences were obtained in case of sequence variation for each of the respective finalized epitopes. The immunogenic potential of both 100% conserved and derived consensus peptide sequence was again determined using Vaxijen server (Doytchinova and Flower <a href="#CR18" rid="CR18" class=" bibr popnode">2007</a>).</p></div><div id="Sec11" class="sec sec-last"><h3 id="Sec11title">Designing of Polypeptide Vaccine Construct</h3><div id="Sec12" class="sec sec-first"><p></p><h4 id="Sec12title" class="inline">Assembly of the B-Cell Epitopes for Designing of the Vaccine </h4><p id="Par21" class="p p-first-last">The vaccine sequence was built by adding the adjuvant at the amino-terminus followed by the top B cell epitopes for the selected proteins. The adjuvant selected was the Cholera toxin subunit B (CTB) linked with the first B cell epitope via an EAAAK linker while the rest of the epitopes were linked together by incorporating the GPGPG linkers. At the carboxy-terminal, a six His Tag was attached for protein identification and purification in later stages (Fig.&#x000a0;<a href="/pmc/articles/PMC8351238/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig2" co-legend-rid="lgnd_Fig2"><a href="/pmc/articles/PMC8351238/figure/Fig2/" target="figure" rid-figpopup="Fig2" rid-ob="ob-Fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139948914979440"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 10989_2021_10255_Fig2_HTML.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is 10989_2021_10255_Fig2_HTML.jpg" src="/pmc/articles/PMC8351238/bin/10989_2021_10255_Fig2_HTML.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm139948914979440"><a target="object" rel="noopener" href="/pmc/articles/PMC8351238/figure/Fig2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig2"><div><a class="figpopup" href="/pmc/articles/PMC8351238/figure/Fig2/" target="figure" rid-figpopup="Fig2" rid-ob="ob-Fig2">Fig. 2</a></div><!--caption a7--><div class="caption"><p>Representation of the designed vaccine model. The amino terminus starts with the adjuvant attached to the first B-cell epitope with an EAAAK linker. The rest of the epitopes are conjoined together via GPGPG linkers. At the c-terminus, a 6&#x02009;&#x000d7;&#x02009;His Tag is attached for identification purpose</p></div></div></div></div><div id="Sec13" class="sec"><p></p><h4 id="Sec13title" class="inline">Prediction of the Allergenicity and Solubility </h4><p id="Par22" class="p p-first-last">To predict the allergenicity of the vaccine sequence, AlgPred v.2.0 (Saha and Raghava <a href="#CR71" rid="CR71" class=" bibr popnode">2006</a>) and AllerTOP v.2.0 (Dimitrov <a href="#CR17" rid="CR17" class=" bibr popnode">2014</a>) servers were utilized. These servers work on specific algorithms to forecast the allergenicity of the amino acid sequence. VaxiJen v2.0 server was employed for the prediction of the antigenicity of the vaccine protein whereas, the solubility was assessed through the Proso II (Smialowski et al. <a href="#CR74" rid="CR74" class=" bibr popnode">2012</a>) and Protein-sol servers (Hebditch <a href="#CR31" rid="CR31" class=" bibr popnode">2017</a>).</p></div><div id="Sec14" class="sec"><p></p><h4 id="Sec14title" class="inline">Evaluation of the Physiochemical Properties </h4><p id="Par23" class="p p-first-last">For the functional characterization of the protein, ExPASy ProtParam (Gasteiger <a href="#CR24" rid="CR24" class=" bibr popnode">2005</a>) tool was used which explored the vaccine sequence on a number of physiochemical parameters such as its molecular weight, pI, aliphatic index, GRAVY value, instability index and estimated half-lives. The PSIPRED v3.3 was utilized for the investigation of the alpha helix, beta-sheet and coil structure of the vaccine (Buchan and Jones <a href="#CR10" rid="CR10" class=" bibr popnode">2019</a>).</p></div><div id="Sec15" class="sec"><p></p><h4 id="Sec15title" class="inline">Tertiary Structure Prediction and Validation </h4><p id="Par24" class="p p-first-last">For the 3D modeling of the vaccine protein, the amino acid sequence was subjected to I-TASSER server that builds a 3D structure on the basis of the degree of similarity between the target protein and the available template structures from the Protein Data Bank (Yang and Zhang <a href="#CR86" rid="CR86" class=" bibr popnode">2015</a>). To validate the generated 3D structure, its refinement was carried out to enhance the stability of the predicted model using GalaxyRefine (Heo et al. <a href="#CR33" rid="CR33" class=" bibr popnode">2013</a>). The refined protein was then validated through the Ramachandran plot assessment via RAMPAGE server (Lovell <a href="#CR45" rid="CR45" class=" bibr popnode">2003</a>).</p></div><div id="Sec16" class="sec"><p></p><h4 id="Sec16title" class="inline">Molecular Docking of the Vaccine Protein with TLR4 </h4><p id="Par25" class="p p-first-last">HADDOCK 2.2 web server was employed for predicting the binding affinity and favorable interactions between the designed vaccine and the human TLR4 (Vries et al. <a href="#CR15" rid="CR15" class=" bibr popnode">2010</a>). For the docking analysis, TLR4 3D model was retrieved form the PDB and active and passive residues were predicted for both the complexes via CPORT<sup>94</sup>. iMODs<sup>95</sup> server was used for the prediction of the normal mode analysis. This server is a dynamic tool that simulates the protein model with reference to its internal normal modes and aids in determining the flexibility and mobility of the complexes via a number of factors including deformability, eigenvalues, B-factors, and covariance. The structural dynamics of the protein&#x02013;protein complex were thus investigated.</p></div><div id="Sec17" class="sec"><p></p><h4 id="Sec17title" class="inline">Codon Optimization </h4><p id="Par26" class="p p-first-last">JCat<sup>96</sup> server is a codon optimization tool that was utilized for the reverse translation of the vaccine sequence. E. coli (K12) was selected as the appropriate host for the codon optimization as it differs significantly from the human in terms of codon usage. Parameters such as Rho-independent transcription termination, prokaryote ribosome binding site and cleavage sites of restriction enzyme BglII and Apa1 were not selected for the final optimization of the vaccine sequence.</p></div><div id="Sec18" class="sec sec-last"><p></p><h4 id="Sec18title" class="inline">Analysis of the Immune Capacity of the Designed Vaccine </h4><p id="Par27" class="p p-first-last">The peptide vaccine was subjected to C-ImmSim web server for the simulations of immune responses. The server incorporates the position-specific matrix for the generation of immune connections. The default parameters were employed for this procedure with three subsequent doses of injections<sup>97</sup>.</p></div></div></div><div id="Sec19" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec19title">Results</h2><div id="Sec20" class="sec sec-first"><h3 id="Sec20title">Proteome Subtraction</h3><p id="Par28" class="p p-first">PAO1 proteome comprises of a total 5562 proteins. Initial filtration using the CD-HIT online server resulted in 5548 proteins with the removal of 14 paralog sequences from the pan-proteome. Elimination of paralogous sequences resulted in a non-redundant set of proteins which was used for further analysis. The resultant proteome was subjected to dynamic Vaxign analysis which estimated 169 periplasmic, 169 outer-membranous, 65 extracellular and 1277 proteins localized on the cytoplasmic membrane. A total of 4628 proteins were having&#x02009;&#x0003c;&#x02009;2 number of helices (Supplementary Table S1). Proteins with more number of helices get anchored onto the cell surface and hence are not accessible to the host immune system. Moreover, 1863 proteins were found essential and 1032 as virulent. The decision tree (DT) method was used to filter the proteome pool based on the predefined threshold values thus excluding the undesired proteins.</p><p id="Par29">For druggable protein target identification, 2623 cytoplasmic proteins were selected. These selected proteins were subjected to essentiality checks via DEG and the number of selected proteins was reduced to 209. Using KEGG output 148 essential cytoplasmic enzymes were obtained. The essential and virulent exoproteins were further used for analysis.</p><p id="Par30">Following localization and virulence prediction for selection of putative vaccine candidates&#x02019; rigorous screening of the resultant protein pool involved; removal of 1224 human homologous proteins (identified by Vaxign pipeline), selection of surface-exposed proteins of&#x02009;&#x0003c;&#x02009;2 transmembrane helices and&#x02009;&#x0003e;&#x02009;0.5 adhesins probability. This yielded a hand full of 68 initially prioritized proteins (Supplementary Table S2, Fig. S1 (a, b). Based on prioritizing parameters particularly antigenicity scores estimation by Vaxijen v2.0 online server (threshold value&#x02009;=&#x02009;0.8), 14 proteins were selected for further analysis (Supplementary Table S3). The relative position of the genes encoding the 14 selected proteins on the PAO1 circular genome is shown in Supplementary Fig. S2.</p><p id="Par31">In addition to this, organism-specific (Homo sapiens) BLASTp search of selected proteins revealed 10 proteins with no significant homology with human proteins (Supplementary Table S4). OprF (<a id="__tag_1133178595" class="tag_hotlink" href="/protein/P13794" ref="/protein/P13794"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"P13794","term_id":"130680","term_text":"P13794"}}</span>P13794</a>) showed a 35% identity with only 20% query coverage predicting it non-homologous to humans. Three proteins PA0833 (<a id="__tag_1133178636" class="tag_hotlink" href="/protein/Q9I5A7" ref="/protein/Q9I5A7"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9I5A7","term_id":"81622535","term_text":"Q9I5A7"}}</span>Q9I5A7</a>), PA1471 (<a id="__tag_1133178598" class="tag_hotlink" href="/protein/Q9I3P1" ref="/protein/Q9I3P1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9I3P1","term_id":"81541397","term_text":"Q9I3P1"}}</span>Q9I3P1</a>) and OprG (<a id="__tag_1133178626" class="tag_hotlink" href="/protein/Q9HWW1" ref="/protein/Q9HWW1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9HWW1","term_id":"81540071","term_text":"Q9HWW1"}}</span>Q9HWW1</a>) were removed from the pool because of their higher identity, query coverage scores and probability of evoking autoimmune responses in the host.</p><p id="Par32">Apart from homologs search, orthologous sequence search using BLASTp and PATRIC v3.5.23 identified putative conserved domains for all proteins in other genera except for proteins PA1956 (<a id="__tag_1133178611" class="tag_hotlink" href="/protein/Q9I2E8" ref="/protein/Q9I2E8"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9I2E8","term_id":"81541127","term_text":"Q9I2E8"}}</span>Q9I2E8</a>), pra (G3XDA9) and PA0724 (<a id="__tag_1133178616" class="tag_hotlink" href="/protein/Q9I5K4" ref="/protein/Q9I5K4"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9I5K4","term_id":"81541788","term_text":"Q9I5K4"}}</span>Q9I5K4</a>) (Supplementary Tables S5, S6).</p><p id="Par33" class="p p-last">For druggable targets&#x02019; search, the obtained 148 (KEGG output) essential cytoplasmic enzymes were subjected to host (human) homolog exclusion using BLAST against Homo sapiens proteomic data. Host homolog exclusion reduced the selection pool to 9 proteins (Supplementary Table S7).</p></div><div id="Sec21" class="sec"><h3 id="Sec21title">Molecular Weight Estimation</h3><p id="Par34" class="p p-first">Identified putative vaccine candidate proteins were further analyzed for their respective molecular weights. Only one protein PA4541 (<a id="__tag_1133178624" class="tag_hotlink" href="/protein/Q9HVN6" ref="/protein/Q9HVN6"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9HVN6","term_id":"81539854","term_text":"Q9HVN6"}}</span>Q9HVN6</a>) had a molecular weight of 139 KDa, higher than the cutoff&#x02009;&#x02264;&#x02009;110&#x000a0;kDa and hence was removed from the list of prioritized proteins (Supplementary Table S8). No such exclusion was performed for potential druggable targets.</p><p id="Par35" class="p p-last">Based on antigenicity scores estimation by VaxiJen v2.0 online server (threshold value&#x02009;=&#x02009;0.8) and molecular weights (cutoff&#x02009;&#x02264;&#x02009;110&#x000a0;kDa), 5 proteins were shortlisted as putative vaccine candidates suitable for screening and prediction of best immunogenic epitopes (Table <a href="/pmc/articles/PMC8351238/table/Tab1/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab1" rid-ob="ob-Tab1" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table1</span></span><span>1</span></a>).</p></div><div id="Sec22" class="sec"><h3 id="Sec22title">Three-D Structure Prediction and Refinement</h3><p id="Par36" class="p p-first-last">Prioritized proteins were checked for crystalline structure availability in Protein Data Bank. Only two potential vaccine candidate proteins (OprF, and OprD) have had available structures in both organism-specific and general PDB search. Remaining three proteins, PA1288, pra and PA4874 had partially resolved structures depending on the homology search in PDB. Structures of these proteins were modeled using SWISS-MODEL and RaptorX. Only the RaptorX models were used for the antigenic region&#x02019;s visualization (Fig.&#x000a0;<a href="/pmc/articles/PMC8351238/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span>3</span></a>a). For putative druggable targets, the initial organism-specific search (limited to <em>P. aeruginosa</em>) yielded two crystal structures for the proteins with 100% sequence identity and (PDB ID: 5BNT_C) and lpxD (PDB ID: 3PMO_A). Homology structures were predicted for proteins wbpB, folk, glyQ, ribF, acnB using the Swiss-Model server. Two proteins waaA, wbpH did not has any crystalline structure and also lacked any suitable template for homology prediction thence were not included in the druggability assessment study. The 3D structures presented an improved prospect for comprehension of the relationship of the studied proteins within the biological system (Fig.&#x000a0;<a href="/pmc/articles/PMC8351238/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span>3</span></a>b).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig3" co-legend-rid="lgnd_Fig3"><a href="/pmc/articles/PMC8351238/figure/Fig3/" target="figure" rid-figpopup="Fig3" rid-ob="ob-Fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139948919396496"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8351238_10989_2021_10255_Fig3_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 10989_2021_10255_Fig3_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC8351238/bin/10989_2021_10255_Fig3_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139948919396496"><a target="object" rel="noopener" href="/pmc/articles/PMC8351238/figure/Fig3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig3"><div><a class="figpopup" href="/pmc/articles/PMC8351238/figure/Fig3/" target="figure" rid-figpopup="Fig3" rid-ob="ob-Fig3">Fig. 3</a></div><!--caption a7--><div class="caption"><p>Three-dimensional structure of the predicted vaccine candidate proteins along with the topological view of corresponding antigenic peptides (on top) and druggable target proteins. <strong>a</strong> illustrates the RaptorX modeled three-dimensional structures of proteins with their corresponding surface-exposed antigenic epitopes. Grey color is indicating the protein structure while the respective epitopes are shown in cartoon form. Proteins with more than one antigenic epitope are shown in pink and blue colors. The images were visualized by using the Pepitope server (URL: <a href="http://pepitope.tau.ac.il/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pepitope.tau.ac.il/</a>). Best scoring drug-binding pockets <strong>b</strong>: Pockets with a drug score of&#x02009;&#x0003e;&#x02009;0.5 were selected for each of the screened druggable proteins</p></div></div></div></div><div id="Sec23" class="sec"><h3 id="Sec23title">PPIs and COGs Analysis</h3><p id="Par37" class="p p-first">STRING network prediction of five screened proteins revealed intra-species interaction between different proteins. <strong>OprF (OMP)</strong> is a member of the cell outer-membrane complex with involvement in several biological processes, transport of organic substances, protein and ion transport. Being a member of OM-channel superfamily other predicted functional porin partners include OprD, OprP, OprB, OprE, OprG, OprI, OprH and tolB. There is also a functional link of OprF protein with RNA polymerase sigma factor SigX.</p><p id="Par38">Direct interaction of many multidrug efflux outer membrane porins, transcriptional regulators and operon repressors is also observed with <strong>OprD protein</strong> (Porins with serine protease activity and specificity for basic amino acids). This includes OprJ; channel-forming component of multidrug efflux pump, OprN, OprM; multidrug ABC transporter and component of MexAB-OprM efflux system, nfxB; confer resistance to quinolones, may negatively regulate genes associated with cell permeability to drugs and mexR; MDR operon repressor. OprD also has involvement in KEGG pathways beta-lactam resistance, transmembrane transport activity and lipid binding. No significant functional enrichment was found for <strong>pra</strong> (protein activator of <em>P. aeruginosa</em>) although it was found to have indirect interaction with a range of virulence factors of <em>P. aeruginosa</em> including phzH, lasB, lap, and napD. <strong>PA1288</strong> and <strong>PA4874</strong>, being probable outer membrane protein precursors and hypothetical proteins respectively do not have any significant functional role. Individual interaction analysis of these proteins revealed interaction of PA1288 with a long chain fatty-acid-CoA ligase fadD2 protein, eraR protein, acyl-CoA dehydrogenase PA0508 and various hypothetical proteins of unknown functions. The predicted functional partners of PA4874 include sporulation protein SpoVR which is directly involved in spore formation, flagellar hook protein flgK, a unique metal-binding pterin Molybdopterin-binding protein PA3441, and a few hypothetical proteins.</p><p id="Par39">A chimeric vaccine against these five proteins might affect all interacting proteins which will further enhance the efficacy of the vaccine by resulting in the reduction in the pathogenesis of <em>P. aeruginosa</em> infections. All protein network interactions are shown in Supplementary Fig. S3(a).</p><p id="Par40">During COG analysis the protein OprF was sorted to COG2885&#x02014;outer membrane protein and related peptidoglycan associated proteins whereas PA1288 lies in COG2067&#x02014;predicted long-chain fatty acid transport protein. OprD and PA4874 were sorted into a non-supervised orthologous group. Outer membrane porin precursor OprD was sorted to NOG08180 and conserved hypothetical protein PA4874 to NOG79339. PA4874 has a probable link with phosphate starvation-inducible protein PsiF. Protein activator pra could not be sorted to any orthologous group.</p><p id="Par41" class="p p-last">The generated interaction and network map for putative druggable targets had about 19 nodes, 39 edges, with average node degree about 4.11 Supplementary Fig. S3 (b). The average clustering coefficient was 0.677 and PPI enrichment p-value was 3.55&#x02009;&#x000d7;&#x02009;10<sup>&#x02013;08</sup>. Proteins wbpB, and, folK, waaA, glyQ, ribF, acnB, wbpH, and lpxD are involved in multiple pathways and perform biological functions ranging from amino acids, lipid and carbohydrate biosynthesis, LPS biosynthesis, protein translation, cell wall organization and energy generation. Protein <strong>WbpB</strong> and <strong>WbpH</strong> are involved in Biological Processes (GO): O antigen biosynthetic process (GO: 0009243), cell wall organization (GO: 0071555), lipopolysaccharide biosynthetic process (GO: 0009103), protein oligomerization (GO: 0051259) and protein complex biogenesis (GO: 0070271). Protein <strong>asd</strong> is primarily involved in amino acid biosynthesis as threonine biosynthetic process (GO: 0009088), aspartate family amino acid biosynthetic process (GO: 0009067), alpha-amino acid biosynthetic process (GO: 1901607), lysine biosynthetic process via diaminopimelate (GO: 0009089), and branched-chain amino acid biosynthetic process (GO: 0009082). <strong>folK</strong> plays its role in tetrahydrofolate biosynthetic process (GO: 0046654), cellular modified amino acid biosynthetic process (GO: 0042398), and coenzyme biosynthetic process (GO: 0009108). Enzymes <strong>waaA</strong> and <strong>lpxD</strong> both are collectively involved in lipid A biosynthetic process (GO: 0009245), liposaccharide metabolic process (GO: 1903509), carbohydrate biosynthetic process (GO: 0016051), lipid biosynthetic process (GO: 0008610), cellular lipid metabolic process (GO: 0044255), carbohydrate derivative biosynthetic process (GO: 1901137). <strong>qlyQ</strong> is involved processes; nucleobase-containing compound metabolic process (GO: 0006139), tRNA aminoacylation for protein translation (GO: 0006418), and had aminoacyl-tRNA ligase activity (GO: 0004812). <strong>ribF</strong> is involved in Riboflavin metabolism (00740). <strong>acnB</strong> is involved in tricarboxylic acid cycle (GO: 0006099), aerobic respiration (GO: 0009060), propionate metabolic process (GO: 0019541) along with mRNA 3 -UTR binding potential (GO: 0003730), and aconitate hydratase activity (GO: 0003994).</p></div><div id="Sec24" class="sec"><h3 id="Sec24title">Screening of B-Cell and T-Cell Epitopes</h3><p id="Par42" class="p p-first">Vaxitop online server predicted enormous B-cell epitopes from selected proteins out of which a total of 54 best scoring B-cell epitopes based on least p-value, highest antigenicity score and topologically exposed on the surface of protein were shortlisted (Supplementary Table S9). A list of tested MHC-I and MHC-II alleles are provided in Supplementary Table S10. Selected peptides were subjected to IC<sub>50</sub> value analysis for DRB1*0101 allele, several bound alleles by epitopes and T-cell binding affinity (prediction by tepitool server, IC50 value cut off&#x02009;&#x0003c;&#x02009;100&#x000a0;nM). Virulence prediction of screened peptides revealed all peptides as virulent with good scores. Principally the targeted peptides should elicit both B-cell and T-cell mediated immunity. Hence selection of epitopes was performed based on the highest binding affinity and recognition by both B-cell and T-cell. Finally, 8 peptides with the lowest IC<sub>50</sub> values (cut off&#x02009;&#x0003c;&#x02009;100&#x000a0;nM), highest antigenicity scores, having a binding affinity with a maximum number of alleles of both classes were selected as shown in Table <a href="/pmc/articles/PMC8351238/table/Tab2/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab2" rid-ob="ob-Tab2" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">Table2</span></span><span>2</span></a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="Tab2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p>List of prioritized antigenic and virulent epitopes in potential vaccine candidate proteins</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="left" rowspan="1" colspan="1">Protein accession</th><th align="left" rowspan="1" colspan="1">Gene name</th><th align="left" rowspan="1" colspan="1">Protein length (aa)</th><th align="left" rowspan="1" colspan="1">Localization</th><th align="left" rowspan="1" colspan="1">B-cell epitope</th><th align="left" rowspan="1" colspan="1">IC50 value (nm)</th><th align="left" rowspan="1" colspan="1">MHC I allele count</th><th align="left" rowspan="1" colspan="1">MHC II allele count</th><th align="left" rowspan="1" colspan="1">P value</th><th align="left" rowspan="1" colspan="1">Antigenicity score</th></tr></thead><tbody><tr><td align="left" rowspan="2" colspan="1"><a id="__tag_1133178582" class="tag_hotlink" href="/protein/P13794" ref="/protein/P13794"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"P13794","term_id":"130680","term_text":"P13794"}}</span>P13794</a></td><td align="left" rowspan="2" colspan="1"><em>opr</em>F</td><td align="left" rowspan="1" colspan="1">350</td><td align="left" rowspan="1" colspan="1">Outer membrane</td><td align="left" rowspan="1" colspan="1">HTDSVGTDAY</td><td align="center" rowspan="1" colspan="1">65.53</td><td align="left" rowspan="1" colspan="1">4</td><td align="left" rowspan="1" colspan="1">0</td><td align="center" rowspan="1" colspan="1">0.000385</td><td align="center" rowspan="1" colspan="1">1.9872</td></tr><tr><td align="left" rowspan="1" colspan="1">350</td><td align="left" rowspan="1" colspan="1">Outer membrane</td><td align="left" rowspan="1" colspan="1">FLTDDVELAL</td><td align="center" rowspan="1" colspan="1">2.09</td><td align="left" rowspan="1" colspan="1">18</td><td align="left" rowspan="1" colspan="1">0</td><td align="center" rowspan="1" colspan="1">0.0042</td><td align="center" rowspan="1" colspan="1">0.4731</td></tr><tr><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178625" class="tag_hotlink" href="/protein/P32722" ref="/protein/P32722"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"P32722","term_id":"417518","term_text":"P32722"}}</span>P32722</a></td><td align="left" rowspan="1" colspan="1"><em>opr</em>D</td><td align="left" rowspan="1" colspan="1">443</td><td align="left" rowspan="1" colspan="1">Outer membrane</td><td align="left" rowspan="1" colspan="1">ITDNLSASLY</td><td align="center" rowspan="1" colspan="1">4.73</td><td align="left" rowspan="1" colspan="1">15</td><td align="left" rowspan="1" colspan="1">0</td><td align="center" rowspan="1" colspan="1">0.00000231</td><td align="center" rowspan="1" colspan="1">0.7422</td></tr><tr><td align="left" rowspan="2" colspan="1"><a id="__tag_1133178596" class="tag_hotlink" href="/protein/Q9I456" ref="/protein/Q9I456"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9I456","term_id":"81541503","term_text":"Q9I456"}}</span>Q9I456</a></td><td align="left" rowspan="2" colspan="1">PA1288</td><td align="left" rowspan="1" colspan="1">424</td><td align="left" rowspan="1" colspan="1">Outer membrane</td><td align="left" rowspan="1" colspan="1">KLDITLPESV</td><td align="center" rowspan="1" colspan="1">5.28</td><td align="left" rowspan="1" colspan="1">6</td><td align="left" rowspan="1" colspan="1">3</td><td align="center" rowspan="1" colspan="1">0.00166</td><td align="center" rowspan="1" colspan="1">1.2296</td></tr><tr><td align="left" rowspan="1" colspan="1">424</td><td align="left" rowspan="1" colspan="1">Outer membrane</td><td align="left" rowspan="1" colspan="1">FTFGLGIYV</td><td align="center" rowspan="1" colspan="1">55.34</td><td align="left" rowspan="1" colspan="1">27</td><td align="left" rowspan="1" colspan="1">1</td><td align="center" rowspan="1" colspan="1">0.00277</td><td align="center" rowspan="1" colspan="1">2.1776</td></tr><tr><td align="left" rowspan="2" colspan="1">G3XDA9</td><td align="left" rowspan="2" colspan="1"><em>pra</em></td><td align="left" rowspan="1" colspan="1">162</td><td align="left" rowspan="1" colspan="1">Extracellular</td><td align="left" rowspan="1" colspan="1">QMTGLPWKL</td><td align="center" rowspan="1" colspan="1">9.23</td><td align="left" rowspan="1" colspan="1">12</td><td align="left" rowspan="1" colspan="1">0</td><td align="center" rowspan="1" colspan="1">0.0125</td><td align="center" rowspan="1" colspan="1">2.2073</td></tr><tr><td align="left" rowspan="1" colspan="1">162</td><td align="left" rowspan="1" colspan="1">Extracellular</td><td align="left" rowspan="1" colspan="1">SLAALALCL</td><td align="center" rowspan="1" colspan="1">3.85</td><td align="left" rowspan="1" colspan="1">12</td><td align="left" rowspan="1" colspan="1">0</td><td align="center" rowspan="1" colspan="1">0.0149</td><td align="center" rowspan="1" colspan="1">0.9236</td></tr><tr><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178588" class="tag_hotlink" href="/protein/Q9HUT9" ref="/protein/Q9HUT9"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9HUT9","term_id":"81539695","term_text":"Q9HUT9"}}</span>Q9HUT9</a></td><td align="left" rowspan="1" colspan="1">PA4874</td><td align="left" rowspan="1" colspan="1">138</td><td align="left" rowspan="1" colspan="1">Periplasmic</td><td align="left" rowspan="1" colspan="1">IPMFVLAMAV</td><td align="center" rowspan="1" colspan="1">5.06</td><td align="left" rowspan="1" colspan="1">14</td><td align="left" rowspan="1" colspan="1">23</td><td align="center" rowspan="1" colspan="1">0.000385</td><td align="center" rowspan="1" colspan="1">1.6836</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139948919372080"><a target="object" rel="noopener" href="/pmc/articles/PMC8351238/table/Tab2/?report=objectonly">Open in a separate window</a></div></div><p id="Par43" class="p p-last">These predicted epitopes fulfill all attributes of a potent peptide capable of provoking an immune response. To check the topology of all epitopes as surface-exposed/outside and not folded/hidden within the globular proteins these were visualized on their respective proteins using Pepitope online server (Fig.&#x000a0;<a href="/pmc/articles/PMC8351238/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span>3</span></a>a).</p></div><div id="Sec25" class="sec"><h3 id="Sec25title">Genetic Diversity Analysis</h3><p id="Par44" class="p p-first-last">Potential sequence variations in other strains are a serious issue as these variations can potentially result in a lack of immunogenicity of the predicted vaccine candidate. For evaluating such probable sequence divergence, sequences of potential epitopes were aligned with members of <em>Pseudomonas aeruginosa</em> taxonomic group (Taxid: 136841). Multiple sequence alignment of screened epitopes revealed the requirement of only one consensus sequence for a 9-mer pra antigenic epitope QMTGLPWKL at position number 8 from lysine to threonine (Supplementary Fig. S4). Immunogenicity potential of both predicted epitopes and sequence alignment derived consensus peptides was determined using Vaxijen v2.0 and revealed no significant difference between epitope predicted by pipeline and derived consensus peptide.</p></div><div id="Sec26" class="sec"><h3 id="Sec26title">Alternate Drug Targets Assessment</h3><div id="Sec27" class="sec sec-first"><p></p><h4 id="Sec27title" class="inline">Structure-Based Druggability Assessment </h4><p id="Par45" class="p p-first-last">Out of 9 screened potential targets, 7 having the 3d structure were subjected to druggability assessment using DoGSiteScorer. The online server revealed druggable pockets for each target. The pockets with drug scores&#x02009;&#x0003e;&#x02009;0.50 were selected as shown in Fig.&#x000a0;<a href="/pmc/articles/PMC8351238/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span>3</span></a>b. Best pockets were selected based on the generated drug score by the server to be&#x02009;&#x02265;&#x02009;0.80. The list of best pockets is shown in Table <a href="/pmc/articles/PMC8351238/table/Tab3/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab3" rid-ob="ob-Tab3" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">Table3.</span></span><span>3</span></a>. The highest scoring pocket was P_2 detected in acnB (<a id="__tag_1133178640" class="tag_hotlink" href="/protein/Q9I2V5" ref="/protein/Q9I2V5"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9I2V5","term_id":"81541220","term_text":"Q9I2V5"}}</span>Q9I2V5</a>) with a drug score of 0.87. While the second and third high scoring pockets were P_1 glyQ (<a id="__tag_1133178627" class="tag_hotlink" href="/protein/Q9I7B7" ref="/protein/Q9I7B7"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9I7B7","term_id":"25009323","term_text":"Q9I7B7"}}</span>Q9I7B7</a>) and P_3 wbpB (<a id="__tag_1133178613" class="tag_hotlink" href="/protein/G3XD23" ref="/protein/G3XD23"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"G3XD23","term_id":"408387591","term_text":"G3XD23"}}</span>G3XD23</a>) with drug scores about 0.85 and 0.84 respectively.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="Tab3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p>Best scoring drug binding pockets</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="left" rowspan="1" colspan="1">Protein</th><th align="left" rowspan="1" colspan="1">Name</th><th align="left" rowspan="1" colspan="1">Volume (&#x000c5;<sup>3</sup>)</th><th align="left" rowspan="1" colspan="1">Surface (&#x000c5;<sup>2</sup>)</th><th align="left" rowspan="1" colspan="1">Drug score</th><th align="left" rowspan="1" colspan="1">Simple score</th></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178609" class="tag_hotlink" href="/protein/Q9I2V5" ref="/protein/Q9I2V5"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9I2V5","term_id":"81541220","term_text":"Q9I2V5"}}</span>Q9I2V5</a></td><td align="left" rowspan="1" colspan="1">P_2</td><td align="center" rowspan="1" colspan="1">693.74</td><td align="center" rowspan="1" colspan="1">796.85</td><td align="center" rowspan="1" colspan="1">0.87</td><td align="center" rowspan="1" colspan="1">0.4</td></tr><tr><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178581" class="tag_hotlink" href="/protein/Q9I7B7" ref="/protein/Q9I7B7"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9I7B7","term_id":"25009323","term_text":"Q9I7B7"}}</span>Q9I7B7</a></td><td align="left" rowspan="1" colspan="1">P_1</td><td align="center" rowspan="1" colspan="1">669.89</td><td align="center" rowspan="1" colspan="1">690.94</td><td align="center" rowspan="1" colspan="1">0.85</td><td align="center" rowspan="1" colspan="1">0.4</td></tr><tr><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178589" class="tag_hotlink" href="/protein/G3XD23" ref="/protein/G3XD23"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"G3XD23","term_id":"408387591","term_text":"G3XD23"}}</span>G3XD23</a></td><td align="left" rowspan="1" colspan="1">P_3</td><td align="center" rowspan="1" colspan="1">706.51</td><td align="center" rowspan="1" colspan="1">884.98</td><td align="center" rowspan="1" colspan="1">0.84</td><td align="center" rowspan="1" colspan="1">0.45</td></tr><tr><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178644" class="tag_hotlink" href="/protein/Q9I2V5" ref="/protein/Q9I2V5"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9I2V5","term_id":"81541220","term_text":"Q9I2V5"}}</span>Q9I2V5</a></td><td align="left" rowspan="1" colspan="1">P_4</td><td align="center" rowspan="1" colspan="1">498.1</td><td align="center" rowspan="1" colspan="1">578.96</td><td align="center" rowspan="1" colspan="1">0.84</td><td align="center" rowspan="1" colspan="1">0.24</td></tr><tr><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178614" class="tag_hotlink" href="/protein/Q9I7B7" ref="/protein/Q9I7B7"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9I7B7","term_id":"25009323","term_text":"Q9I7B7"}}</span>Q9I7B7</a></td><td align="left" rowspan="1" colspan="1">P_2</td><td align="center" rowspan="1" colspan="1">614.78</td><td align="center" rowspan="1" colspan="1">638.43</td><td align="center" rowspan="1" colspan="1">0.84</td><td align="center" rowspan="1" colspan="1">0.35</td></tr><tr><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178629" class="tag_hotlink" href="/protein/G3XD23" ref="/protein/G3XD23"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"G3XD23","term_id":"408387591","term_text":"G3XD23"}}</span>G3XD23</a></td><td align="left" rowspan="1" colspan="1">P_2</td><td align="center" rowspan="1" colspan="1">711.81</td><td align="center" rowspan="1" colspan="1">810.21</td><td align="center" rowspan="1" colspan="1">0.83</td><td align="center" rowspan="1" colspan="1">0.45</td></tr><tr><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178597" class="tag_hotlink" href="/protein/Q9HVM3" ref="/protein/Q9HVM3"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9HVM3","term_id":"81539848","term_text":"Q9HVM3"}}</span>Q9HVM3</a></td><td align="left" rowspan="1" colspan="1">P_3</td><td align="center" rowspan="1" colspan="1">343.36</td><td align="center" rowspan="1" colspan="1">732.47</td><td align="center" rowspan="1" colspan="1">0.82</td><td align="center" rowspan="1" colspan="1">0.21</td></tr><tr><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178608" class="tag_hotlink" href="/protein/G3XD23" ref="/protein/G3XD23"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"G3XD23","term_id":"408387591","term_text":"G3XD23"}}</span>G3XD23</a></td><td align="left" rowspan="1" colspan="1">P_0</td><td align="center" rowspan="1" colspan="1">3435.73</td><td align="center" rowspan="1" colspan="1">3823.06</td><td align="center" rowspan="1" colspan="1">0.81</td><td align="center" rowspan="1" colspan="1">0.63</td></tr><tr><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178591" class="tag_hotlink" href="/protein/Q9HVM3" ref="/protein/Q9HVM3"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9HVM3","term_id":"81539848","term_text":"Q9HVM3"}}</span>Q9HVM3</a></td><td align="left" rowspan="1" colspan="1">P_0</td><td align="center" rowspan="1" colspan="1">1271.36</td><td align="center" rowspan="1" colspan="1">1712.96</td><td align="center" rowspan="1" colspan="1">0.81</td><td align="center" rowspan="1" colspan="1">0.65</td></tr><tr><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178642" class="tag_hotlink" href="/protein/Q9I2V5" ref="/protein/Q9I2V5"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9I2V5","term_id":"81541220","term_text":"Q9I2V5"}}</span>Q9I2V5</a></td><td align="left" rowspan="1" colspan="1">P_0</td><td align="center" rowspan="1" colspan="1">1923.9</td><td align="center" rowspan="1" colspan="1">1907.15</td><td align="center" rowspan="1" colspan="1">0.81</td><td align="center" rowspan="1" colspan="1">0.56</td></tr><tr><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178623" class="tag_hotlink" href="/protein/Q9I7B7" ref="/protein/Q9I7B7"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9I7B7","term_id":"25009323","term_text":"Q9I7B7"}}</span>Q9I7B7</a></td><td align="left" rowspan="1" colspan="1">P_0</td><td align="center" rowspan="1" colspan="1">3312.32</td><td align="center" rowspan="1" colspan="1">2928.38</td><td align="center" rowspan="1" colspan="1">0.81</td><td align="center" rowspan="1" colspan="1">0.59</td></tr><tr><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178580" class="tag_hotlink" href="/protein/G3XD23" ref="/protein/G3XD23"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"G3XD23","term_id":"408387591","term_text":"G3XD23"}}</span>G3XD23</a></td><td align="left" rowspan="1" colspan="1">P_1</td><td align="center" rowspan="1" colspan="1">1954.12</td><td align="center" rowspan="1" colspan="1">1955.06</td><td align="center" rowspan="1" colspan="1">0.8</td><td align="center" rowspan="1" colspan="1">0.63</td></tr><tr><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178600" class="tag_hotlink" href="/protein/Q9I2V5" ref="/protein/Q9I2V5"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9I2V5","term_id":"81541220","term_text":"Q9I2V5"}}</span>Q9I2V5</a></td><td align="left" rowspan="1" colspan="1">P_1</td><td align="center" rowspan="1" colspan="1">1193.96</td><td align="center" rowspan="1" colspan="1">1104.93</td><td align="center" rowspan="1" colspan="1">0.8</td><td align="center" rowspan="1" colspan="1">0.55</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139948919259712"><a target="object" rel="noopener" href="/pmc/articles/PMC8351238/table/Tab3/?report=objectonly">Open in a separate window</a></div></div></div><div id="Sec28" class="sec sec-last"><p></p><h4 id="Sec28title" class="inline">Sequence-Based Database Mining for Reported Inhibitors Against Selected Targets </h4><p id="Par46" class="p p-first-last">Chemical and drug databases i.e. PubChem, ChEMBL, DrugBank were searched using keyword and sequences (as database allowed) for identification of any potentially reported inhibitor for our screened targets. PubChem and DrugBank search failed to reveal any similar target while ChEMBL revealed 43 different targets with about 1708 inhibitors for our 9 selected screened targets with a variable degree of % identity ranging from 22.55 to 51.3%. The selection was made by excluding all the inhibitors with % identity&#x02009;&#x0003c;&#x02009;40%. Any similar targets originating in Homo sapiens were also excluded for avoiding any potential side effects. The most identical one was against folK (<a id="__tag_1133178631" class="tag_hotlink" href="/protein/Q9HV71" ref="/protein/Q9HV71"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9HV71","term_id":"81539759","term_text":"Q9HV71"}}</span>Q9HV71</a>) reported in <em>E. coli</em> (CHEMBL3217379) 2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase (<a id="__tag_1133178615" class="tag_hotlink" href="/protein/P26281" ref="/protein/P26281"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"P26281","term_id":"123493","term_text":"P26281"}}</span>P26281</a>) having 74 reported compounds and 51.3% identity. The most similar reported targets are described in Table <a href="/pmc/articles/PMC8351238/table/Tab4/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab4" rid-ob="ob-Tab4" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">Table4</span></span><span>4</span></a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="Tab4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p>List of reported inhibitors available for similar targets searched based on sequence homology</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="left" rowspan="1" colspan="1">Query Sequence</th><th align="left" rowspan="1" colspan="1">ChEMBL ID</th><th align="left" rowspan="1" colspan="1">Preferred name</th><th align="left" rowspan="1" colspan="1">UniProt accession</th><th align="left" rowspan="1" colspan="1">Target type</th><th align="left" rowspan="1" colspan="1">Organism</th><th align="left" rowspan="1" colspan="1">Compounds</th><th align="left" rowspan="1" colspan="1">% Identity</th><th align="left" rowspan="1" colspan="1">BLAST score</th><th align="left" rowspan="1" colspan="1">E-value</th></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178599" class="tag_hotlink" href="/protein/Q9HV71" ref="/protein/Q9HV71"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9HV71","term_id":"81539759","term_text":"Q9HV71"}}</span>Q9HV71</a></td><td align="left" rowspan="1" colspan="1">CHEMBL3217379</td><td align="left" rowspan="1" colspan="1">2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase</td><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178605" class="tag_hotlink" href="/protein/P26281" ref="/protein/P26281"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"P26281","term_id":"123493","term_text":"P26281"}}</span>P26281</a></td><td align="left" rowspan="1" colspan="1">Single protein</td><td align="left" rowspan="1" colspan="1"><em>Escherichia coli (strain K12)</em></td><td align="left" rowspan="1" colspan="1">74</td><td align="center" rowspan="1" colspan="1">51.3</td><td align="left" rowspan="1" colspan="1">164</td><td align="left" rowspan="1" colspan="1">3.00E&#x02212;41</td></tr><tr><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178620" class="tag_hotlink" href="/protein/Q9I7B7" ref="/protein/Q9I7B7"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9I7B7","term_id":"25009323","term_text":"Q9I7B7"}}</span>Q9I7B7</a></td><td align="left" rowspan="1" colspan="1">CHEMBL3086</td><td align="left" rowspan="1" colspan="1">Acetyl-Coenzyme A carboxylase</td><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178618" class="tag_hotlink" href="/protein/Q5SWU9" ref="/protein/Q5SWU9"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q5SWU9","term_id":"81862571","term_text":"Q5SWU9"}}</span>Q5SWU9</a></td><td align="left" rowspan="1" colspan="1">Single protein</td><td align="left" rowspan="1" colspan="1"><em>Mus musculus</em></td><td align="left" rowspan="1" colspan="1">2</td><td align="center" rowspan="1" colspan="1">42.5</td><td align="left" rowspan="1" colspan="1">28.9</td><td align="left" rowspan="1" colspan="1">1.80E+00</td></tr><tr><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178590" class="tag_hotlink" href="/protein/Q9I7B7" ref="/protein/Q9I7B7"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9I7B7","term_id":"25009323","term_text":"Q9I7B7"}}</span>Q9I7B7</a></td><td align="left" rowspan="1" colspan="1">CHEMBL2397</td><td align="left" rowspan="1" colspan="1">Acetyl-CoA carboxylase 1</td><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178630" class="tag_hotlink" href="/protein/P11497" ref="/protein/P11497"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"P11497","term_id":"116670","term_text":"P11497"}}</span>P11497</a></td><td align="left" rowspan="1" colspan="1">Single protein</td><td align="left" rowspan="1" colspan="1"><em>Rattus norvegicus</em></td><td align="left" rowspan="1" colspan="1">150</td><td align="center" rowspan="1" colspan="1">42.5</td><td align="left" rowspan="1" colspan="1">28.9</td><td align="left" rowspan="1" colspan="1">1.90E+00</td></tr><tr><td align="left" rowspan="1" colspan="1">G3XD85</td><td align="left" rowspan="1" colspan="1">CHEMBL3774301</td><td align="left" rowspan="1" colspan="1">Cysteine proteinase 1</td><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178593" class="tag_hotlink" href="/protein/Q01957" ref="/protein/Q01957"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q01957","term_id":"2450958235","term_text":"Q01957"}}</span>Q01957</a></td><td align="left" rowspan="1" colspan="1">Single protein</td><td align="left" rowspan="1" colspan="1"><em>Entamoeba histolytica</em></td><td align="left" rowspan="1" colspan="1">3</td><td align="center" rowspan="1" colspan="1">42.11</td><td align="left" rowspan="1" colspan="1">28.9</td><td align="left" rowspan="1" colspan="1">2.20E+00</td></tr><tr><td align="left" rowspan="1" colspan="1">G3XD85</td><td align="left" rowspan="1" colspan="1">CHEMBL3774301</td><td align="left" rowspan="1" colspan="1">Cysteine proteinase 1</td><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178587" class="tag_hotlink" href="/protein/Q01957" ref="/protein/Q01957"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q01957","term_id":"2450958235","term_text":"Q01957"}}</span>Q01957</a></td><td align="left" rowspan="1" colspan="1">SINGLE PROTEIN</td><td align="left" rowspan="1" colspan="1"><em>Entamoeba histolytica</em></td><td align="left" rowspan="1" colspan="1">3</td><td align="center" rowspan="1" colspan="1">42.11</td><td align="left" rowspan="1" colspan="1">28.9</td><td align="left" rowspan="1" colspan="1">2.20E+00</td></tr><tr><td align="left" rowspan="1" colspan="1"><a id="__tag_1133178602" class="tag_hotlink" href="/protein/Q9HV71" ref="/protein/Q9HV71"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9HV71","term_id":"81539759","term_text":"Q9HV71"}}</span>Q9HV71</a></td><td align="left" rowspan="1" colspan="1">CHEMBL3559681</td><td align="left" rowspan="1" colspan="1">2-amino-4-hydroxy-6-hydroxymethyldihydropteridin e pyrophosphokinase</td><td align="left" rowspan="1" colspan="1">X5EH84</td><td align="left" rowspan="1" colspan="1">SINGLE PROTEIN</td><td align="left" rowspan="1" colspan="1"><em>Staphylococcus aureus</em></td><td align="left" rowspan="1" colspan="1">60</td><td align="center" rowspan="1" colspan="1">41.55</td><td align="left" rowspan="1" colspan="1">117</td><td align="left" rowspan="1" colspan="1">5.00E&#x02212;27</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139948919025328"><a target="object" rel="noopener" href="/pmc/articles/PMC8351238/table/Tab4/?report=objectonly">Open in a separate window</a></div></div></div></div><div id="Sec29" class="sec sec-last"><h3 id="Sec29title">Designing of Polypeptide Vaccine Construct</h3><div id="Sec30" class="sec sec-first"><p></p><h4 id="Sec30title" class="inline">Prediction of the Allergenicity and Solubility </h4><p id="Par47" class="p p-first-last">The protein sequence of the vaccine was found to be non-allergenic in nature as predicted by the AlgPred server. The antigenicity of the designed vaccine was estimated to be 0.9193 exhibiting a potential immunogenic response. Upon solubility analysis, Proso II indicated the construct as soluble molecule. Moreover, the designed vaccine is soluble as per its charge distribution, hydrophobicity and stability.</p></div><div id="Sec31" class="sec"><p></p><h4 id="Sec31title" class="inline">Physiochemical Properties of the Designed Vaccine </h4><p id="Par48" class="p p-first-last">ExPASy server was employed for determining various physiochemical properties of the designed vaccine. The molecular weight of the designed sequence is 26.5&#x000a0;kDa. The pI of the vaccine was predicted to be 6.35 which indicates its acidic nature. The aliphatic index was recorded to be 83.98 along with the GRAVY value of &#x02212;&#x000a0;0.092. The vaccine was forecasted to have an estimated half-life of one hour within the mammalian reticulocytes in vitro and greater than ten hours in <em>E. coli </em>in vivo. All these parameters show the thermostable nature of the designed vaccine.</p></div><div id="Sec32" class="sec"><p></p><h4 id="Sec32title" class="inline">Multi-epitope Vaccine Structure Prediction and Validation </h4><p id="Par49" class="p p-first-last">PSIPRED v3.3 categorized the secondary structure of the designed vaccine as; 28% alpha-helix, 18% &#x003b2; sheets and, 52% random coil (Fig.&#x000a0;<a href="/pmc/articles/PMC8351238/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4</span></a>a). I-TASSER was used for predicting the tertiary structure and first phase of refinement was carried out by the ModRefiner which was preceded by the GalaxyRefine treatment for the effective enhancement of the tertiary structure of the vaccine (Fig.&#x000a0;<a href="/pmc/articles/PMC8351238/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4</span></a>b). The Ramachandran plot predicted 86.7% residues of the protein in the favored region whereas 8.4% residues in the allowed and 4.8% residues in the outlier area (Fig.&#x000a0;<a href="/pmc/articles/PMC8351238/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4</span></a>c). The quality of the predicted model was assessed via ProSA-Web server, which reported the Z-score of &#x02212;&#x000a0;4.13 (Fig.&#x000a0;<a href="/pmc/articles/PMC8351238/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4</span></a>d).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig4" co-legend-rid="lgnd_Fig4"><a href="/pmc/articles/PMC8351238/figure/Fig4/" target="figure" rid-figpopup="Fig4" rid-ob="ob-Fig4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139948960222064"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8351238_10989_2021_10255_Fig4_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 10989_2021_10255_Fig4_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC8351238/bin/10989_2021_10255_Fig4_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139948960222064"><a target="object" rel="noopener" href="/pmc/articles/PMC8351238/figure/Fig4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig4"><div><a class="figpopup" href="/pmc/articles/PMC8351238/figure/Fig4/" target="figure" rid-figpopup="Fig4" rid-ob="ob-Fig4">Fig. 4</a></div><!--caption a7--><div class="caption"><p>Proposed multi-epitope vaccine structure and its validation, <strong>a</strong> elements of secondary structure <strong>b</strong> the 3D model of the designed vaccine obtained through I-TASSER based on homology modelling and refined structure of vaccine protein obtained via Galaxy Refine after its refinement on ModRefiner <strong>c</strong> Ramachandran plot showing 86.7% residues (inside blue circle) of the protein in the favored region <strong>d</strong> ProSA-Web analysis showing the z-score value of&#x02009;&#x02212;&#x02009;4.13</p></div></div></div></div><div id="Sec33" class="sec"><p></p><h4 id="Sec33title" class="inline">Molecular Docking and Normal Mode Analysis </h4><p id="Par50" class="p p-first-last">To analyze the binding affinity and pattern of the vaccine with the host TLR4, molecular docking was carried out by using the HADDOCK 2.2 server (Fig.&#x000a0;<a href="/pmc/articles/PMC8351238/figure/Fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig5" rid-ob="ob-Fig5" co-legend-rid="lgnd_Fig5"><span>5</span></a>). For the analysis of the flexibility of the docked vaccine with TLR4, normal mode simulation gave suitable results. iMODs predicted the mobility of the complex by the orientation of arrows that were directed towards each other. The B-factor value was equivalent to RMS. The Eigenvalue of the complex was estimated as 7.045871e<sup>&#x02212;05</sup>. The covariance of the vaccine-TLR4 complex was exhibited by different colors mainly red, white and blue, whereas the variance was found to be inversely linked with the assessed Eigenvalue. The elasticity of the vaccine was analyzed through the generation of an elastic network model where the light and dark shades of each dot represented the varying stiffness of the model.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig5" co-legend-rid="lgnd_Fig5"><a href="/pmc/articles/PMC8351238/figure/Fig5/" target="figure" rid-figpopup="Fig5" rid-ob="ob-Fig5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139948923831872"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8351238_10989_2021_10255_Fig5_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 10989_2021_10255_Fig5_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC8351238/bin/10989_2021_10255_Fig5_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139948923831872"><a target="object" rel="noopener" href="/pmc/articles/PMC8351238/figure/Fig5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig5"><div><a class="figpopup" href="/pmc/articles/PMC8351238/figure/Fig5/" target="figure" rid-figpopup="Fig5" rid-ob="ob-Fig5">Fig. 5</a></div><!--caption a7--><div class="caption"><p>Molecular docking of designed vaccine with TLR4 and Normal mode analysis. <strong>a</strong> The docking complex shows the binding of the protein with the immune receptor TLR4. The grey molecule represents the vaccine with its active amino acid residues colored as blue whereas the pink molecule represents the TLR4 with green colored active amino acid residues. <strong>b</strong> The B-factor value of the protein model generated. <strong>c</strong> The eigenvalue of the protein model. <strong>d</strong> The co-variance of the model generated depicted via three colors: blue, red and white. <strong>e</strong> The elasticity model of the designed vaccine representing its flexibility</p></div></div></div></div><div id="Sec34" class="sec"><p></p><h4 id="Sec34title" class="inline">Codon Adaptation </h4><p id="Par51" class="p p-first-last">For the expression of the designed vaccine in an appropriate host, reverse translation of its sequence was performed. JCat server predicted the codon adaptation index of 0.933 value along with a GC content of 55.77% (Fig.&#x000a0;<a href="/pmc/articles/PMC8351238/figure/Fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig6" rid-ob="ob-Fig6" co-legend-rid="lgnd_Fig6"><span>6</span></a>). The construct does not carry sites for BglII and ApaI that indicates its safety for cloning.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig6" co-legend-rid="lgnd_Fig6"><a href="/pmc/articles/PMC8351238/figure/Fig6/" target="figure" rid-figpopup="Fig6" rid-ob="ob-Fig6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139948923826112"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8351238_10989_2021_10255_Fig6_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 10989_2021_10255_Fig6_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC8351238/bin/10989_2021_10255_Fig6_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139948923826112"><a target="object" rel="noopener" href="/pmc/articles/PMC8351238/figure/Fig6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig6"><div><a class="figpopup" href="/pmc/articles/PMC8351238/figure/Fig6/" target="figure" rid-figpopup="Fig6" rid-ob="ob-Fig6">Fig. 6</a></div><!--caption a7--><div class="caption"><p>The codon adaptation index (CAI). The CAI graph indicates that codon optimization of the vaccine with 0.933% and the GC content of the optimized codons (55.77%)</p></div></div></div></div><div id="Sec35" class="sec sec-last"><p></p><h4 id="Sec35title" class="inline">Immune Response Against Designed Vaccine </h4><p id="Par52" class="p p-first-last">C-ImmSim was employed for the simulation of immune responses of the finalized vaccine construct. The results indicated a consistent rise in the production of primary and secondary immune responses. The antigens along with the IgG antibodies were also found to increase in concentration (Fig.&#x000a0;<a href="/pmc/articles/PMC8351238/figure/Fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig7" rid-ob="ob-Fig7" co-legend-rid="lgnd_Fig7"><span>7</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="Fig7" co-legend-rid="lgnd_Fig7"><a href="/pmc/articles/PMC8351238/figure/Fig7/" target="figure" rid-figpopup="Fig7" rid-ob="ob-Fig7"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139948923818848"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8351238_10989_2021_10255_Fig7_HTML.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 10989_2021_10255_Fig7_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC8351238/bin/10989_2021_10255_Fig7_HTML.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139948923818848"><a target="object" rel="noopener" href="/pmc/articles/PMC8351238/figure/Fig7/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_Fig7"><div><a class="figpopup" href="/pmc/articles/PMC8351238/figure/Fig7/" target="figure" rid-figpopup="Fig7" rid-ob="ob-Fig7">Fig. 7</a></div><!--caption a7--><div class="caption"><p>The in silico immune simulation of the proposed vaccine. <strong>a</strong> Antibodies production, <strong>b</strong> B-cell production, <strong>c</strong> Th cell production, and <strong>d</strong> T-cell production</p></div></div></div></div></div></div><div id="Sec36" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec36title">Discussion</h2><p id="Par53" class="p p-first">Being one of the top three globally most prevalent nosocomial infectious agents and equipped with virulence and resistance, <em>P. aeruginosa&#x02019;s</em> infections require additional resources and time for patients to recover (Pires, et al. <a href="#CR60" rid="CR60" class=" bibr popnode">2018</a>; Moradali et al. <a href="#CR52" rid="CR52" class=" bibr popnode">2017</a>; Juan et al. <a href="#CR35" rid="CR35" class=" bibr popnode">2017</a>). Countless studies have reported the extent of antibiotic resistance due to multiple factors ranging from over-prescription, self-medication, and abuse of antibiotics on farms, all ultimately affecting the patients&#x02019; survival and recovery (Xipell <a href="#CR85" rid="CR85" class=" bibr popnode">2017</a>; McLean <a href="#CR50" rid="CR50" class=" bibr popnode">2019</a>; Pang <a href="#CR55" rid="CR55" class=" bibr popnode">2019</a>; Frieri et al. <a href="#CR22" rid="CR22" class=" bibr popnode">2017</a>; Qiao <a href="#CR63" rid="CR63" class=" bibr popnode">2018</a>; Karkman <a href="#CR38" rid="CR38" class=" bibr popnode">2018</a>). While the lack of vaccines for superbugs was not even an area of interest for academia as well as an industry until recently (Rappuoli et al. <a href="#CR65" rid="CR65" class=" bibr popnode">2017</a>). The fact that no preventive vaccine is available for protecting the susceptible patients was the main reason for us to look into the pathogen&#x02019;s genome/proteome for finding solutions to this challenge (Rashid <a href="#CR66" rid="CR66" class=" bibr popnode">2020</a>; Baker et al. <a href="#CR5" rid="CR5" class=" bibr popnode">2019</a>; Priebe <a href="#CR62" rid="CR62" class=" bibr popnode">2003</a>). In this study, we attempted to answer the two key questions associated with any nosocomial infection; prevention and cure in the face of superbugs rampant in healthcare setups. To meet our two objectives effective prevention and cure, we devised an in silico subtractive genomics strategy intending to identify potential vaccine candidates and alternative drug targets against <em>P. aeruginosa</em>.</p><p id="Par54">Since the dawn of genomics and the availability of publically accessible genomic data and analysis tools, many different studies have been done to expedite the vaccine and drug development against noticeable pathogens (Sette and Rappuoli <a href="#CR73" rid="CR73" class=" bibr popnode">2010</a>). The most prominent one was the case of the meningococcal group B vaccine which is currently available in markets (Giuliani <a href="#CR26" rid="CR26" class=" bibr popnode">2006</a>). Unfortunately, the situation is quite adverse for <em>P. aeruginosa</em> in terms of both prevention and treatment (Priebe <a href="#CR62" rid="CR62" class=" bibr popnode">2003</a>).</p><p id="Par55">In our previous study, we have presented an overview of all the vaccine development studies and reports (Rashid <a href="#CR67" rid="CR67" class=" bibr popnode">2017</a>). In that study, we primarily focused on the essential proteins involved in the virulence and pathogenicity against the host (i.e. a component of lipopolysaccharide, T3SS, proteins involved in adhesion, mobility, and penetration) as it has been done for almost a century now, yet with an updated methodology. In the current study, we have taken into consideration the resistance phenomenon and endeavored to not only focus on the essential proteins that are involved in virulence but also play their due role in antibiotic resistance as well. Two sets of essential proteins were scrutinized based on their computational analysis for preventive and therapeutic potentials.</p><p id="Par56">Dalsass et al. have recently reported a comparative analysis of reverse vaccinology tools and have discussed the advantages and disadvantages of the two methodologies (Dalsass <a href="#CR14" rid="CR14" class=" bibr popnode">2019</a>). Reductive screening (about genomic/proteomic data) can be performed using two different algorithms, Decision Tree (DT) or filtering method and Support Vector Machine (SVM) or classification method. In the DT method the proteome is filtered based on the predetermined threshold values and proteins are excluded from the pool thus enriching the desirable proteins, while in SVM, all the desired parameters have assigned quantitative values and based on the accumulative score, proteins are selected with the most desirable features.</p><p id="Par57">We adopted a comprehensive screening strategy employing both SVM and DT algorithms. These two algorithms have their respective benefits and shortcomings hence it was difficult to rely on one screening strategy while overlooking the advantages of the other. To minimize the probability of off-target interactions for our prospective vaccine and drug candidates we adopted a more cautious DT approach for primary screening where the essentiality, localization, functionality and host homology were assessed, and only proteins with desirable qualities were selected. The obtained proteins were essential in their respective metabolic or physiological pathways of the pathogen and lacked any homology with humans. The screened prospective vaccine candidates were either secreted or surface-exposed, involved in pathogenicity and resistance while the potential drug target proteins were enzymes localized within the cytoplasm and involved in essential metabolic pathways. Secretome and exoproteome are considered as appealing vaccine candidates because of their potential role in virulence, adherence, invasion, survival, and proliferation into the host cell (Ranganathan and Garg <a href="#CR64" rid="CR64" class=" bibr popnode">2009</a>). In the first round, the Vaxign pipeline screened PAO1 proteome for the identification of the best vaccine candidate proteins. PSORTb is an integral part of the Vaxign pipeline for subcellular localization prediction of proteins which emphasize specificity over sensitivity and do not detect lipoprotein motifs. Hence to overcome such limitations another subcellular localization prediction tool, CELLO v2.5 was synergistically used. Inclusion of complementing results provided by both servers resulted in 7.2% of PAO1 exoproteome. This initially yielded a total of 68 proteins for further analysis.</p><p id="Par58">Although the screened putative protein does not always have to be a virulence factor, selecting proteins with a crucial role in bacterial survival and pathogenesis is extremely useful, because antibodies elicited with such antigens may hinder their functions and neutralize conditions before infections, i.e. use of adhesins as protective antigens may be of great significance through reduction of attachment and colonization by pathogen thus providing herd immunity (Delany et al. <a href="#CR16" rid="CR16" class=" bibr popnode">2013</a>). For this reason, proteins directly or indirectly involved in virulence were prioritized. Similarly, the selection of non-human homologs is a prerequisite for therapeutic target identification to avoid any kind of autoimmune response.</p><p id="Par59">Selection of essential, virulent and antigenic exoproteins revealed five outer membrane porin proteins (OprF, OprD, OprG, OprO, and OprE), six uncharacterized/hypothetical outer-membrane proteins, a protein activator (pra) of <em>P. aeruginosa</em>, an adhesion protein (CupB<sub>5</sub>) and a probable coat protein A of bacteriophage Pf1 (PA0724). The majority of essential genes and respective proteins play a fundamental role in cellular functions but interestingly a bulk of hypothetical genes of unknown functions are also considered essential and their essentiality is growth-condition dependent (Juhas <a href="#CR36" rid="CR36" class=" bibr popnode">2015</a>). Multiple B-cell and T-cell epitopes prediction servers were used to get the best immunogenic epitopes. The selected potential peptide must be capable of binding efficiently with a higher number of MHC class I and class II alleles and thus ensure to cover a major population (Foged et al. <a href="#CR20" rid="CR20" class=" bibr popnode">2012</a>). After rigorous screening, our proposed pipeline finally yielded 8 surface-exposed epitopes belonging to five proteins. These prioritized epitopes include components of outer membrane porin proteins (OprF and OprD), protein activator of <em>P. aeruginosa</em> (pra), a probable extracellular/outer-membrane protein (PA1288) and uncharacterized/hypothetical secretory protein components (PA4874). Although the remaining five proteins yielded conserved antigenic B-cell epitopes those were not capable of efficient recognition and binding to cytotoxic T-cell at IC<sub>50</sub>&#x02009;&#x0003c;&#x02009;100&#x000a0;nM, hence were removed from the list of best epitopes. The best target proteins and their respective epitopic components were reviewed in the literature to get further insight into the prioritized candidates.</p><p id="Par60">Outer membrane proteins (Opr&#x02019;s) of <em>P. aeruginosa</em> form porins which are essential structural components of the bacterial cell surface. Components of these porins are attractive for peptide vaccines due to their presence on the cell surface and conserved antigenic domains across wild-type strains of <em>P. aeruginosa</em> (Sorichter <a href="#CR75" rid="CR75" class=" bibr popnode">2009</a>). In a small scale, clinical trial these candidates have been found promising in eliciting mucosal immunity in the respiratory tract of cystic fibrosis (CF) patients (Fito-Boncompte <a href="#CR19" rid="CR19" class=" bibr popnode">2011</a>). Our pipeline screened and prioritized two outer membrane porin proteins (OprF and OprD) as putative candidates for developing efficacious vaccines. OprF porins are small water-filled channels and have a structural role in maintaining cell shape and growth in a low osmolarity medium. Fito-Boncompte et al., have also reported that OprF is a significant virulence factor of <em>P. aeruginosa</em> and is involved in biofilm formation in oxygen-deprived conditions. Interaction of OprF with gamma interferons has also resulted in the production of phenazine pyocyanin and lectin PA-1L, both of which are important virulence factors (L&#x000fc;cking <a href="#CR46" rid="CR46" class=" bibr popnode">1997</a>).</p><p id="Par61">Several studies have reported elicitation of opsonizing, cross-reactive and protective antibodies in response to immunization with epitopes obtained from OprF and OprI in human and animal models (Knapp <a href="#CR40" rid="CR40" class=" bibr popnode">1999</a>). In another study to check the synergistic immunogenicity of OprF and OprI epitopes, the recombinant hybrid vaccines were found protective against <em>P. aeruginosa</em> systemic infections in animal models using both active and passive immunization strategies. Mansouri et al. conducted a phase I/II human clinical trial study in 2003 for OprF-OprI epitopes based vaccines (Mansouri <a href="#CR48" rid="CR48" class=" bibr popnode">2003</a>).</p><p id="Par62">The loss or reduced OprD porins and decrease in the expression level of these porins in the outer membrane is associated with resistance to carbapenem antibiotics in <em>P. aeruginosa</em> infections. Reverse vaccinology based study by Zhaohui et al., on vaccine candidates identification for <em>Acinetobacter baumannii</em>, also found OprD superfamily outer membrane porin proteins: <a id="__tag_1133178606" class="tag_hotlink" href="/protein/CAM88440.1" ref="/protein/CAM88440.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"CAM88440.1","term_id":"169150532","term_text":"CAM88440.1"}}</span>CAM88440.1</a> in AYE and <a id="__tag_1133178592" class="tag_hotlink" href="/protein/CAM86576.1" ref="/protein/CAM86576.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"CAM86576.1","term_id":"169148710","term_text":"CAM86576.1"}}</span>CAM86576.1</a> in <a id="__tag_1133178633" class="tag_hotlink" href="/protein/WP_000018327.1" ref="/protein/WP_000018327.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"WP_000018327.1","term_id":"445940472","term_text":"WP_000018327.1"}}</span>WP_000018327.1</a> as a potent vaccine target (Ni <a href="#CR54" rid="CR54" class=" bibr popnode">2017</a>). More analysis and investigation are required to know the potential of OprD porins as vaccine candidates. Not much is known about the exact functioning of protein activator (Pra) precursor yet. The PPIs revealed interactions with phenazine modifying protein phzH for the biosynthesis of pyocyanin (virulence factor of <em>P. aeruginosa</em>), lap and lasB which are secreted aminopeptidase and assists pathogen in cleaving host elastin, IgG, collagen and several components of complement system proteins. napD protein of periplasmic nitrate reductase (NAP) complex also interacts directly with pra protein activator of both pathogenic and non-pathogenic strains. Hence based on the indirect involvement in virulence and high antigenic potential of pra protein, two epitopic regions QMTGLPWKL at amino acid position number 88&#x02013;96 and SLAALALCL at position 9&#x02013;17 have been identified as the best vaccine candidates. Multiple sequence alignment revealed only one residue substitution at position number 8 of the epitope (QMTGLPW<span class="underline">K</span>L) from lysine, a basic and polar amino acid with a positive side chain to (QMTGLPW<span class="underline">T</span>L) threonine, a hydroxyl-containing polar amino acid with a neutral side chain which did not significantly affect the antigenicity of peptide and thus could be considered as a potent vaccine candidate.</p><p id="Par63">The proposed vaccine candidate&#x02019;s prediction pipeline also predicted two uncharacterized proteins (PA1288 and PA4874). These hypothetical proteins were selected based on their sequence profile, outer membrane localization, immunogenic potency and interactions with different proteins of significant functional assignments. Regardless of lack of functional information expression profiling by microarray of both proteins in response to lung surfactant in CF patients has been reported (Harmer et al. <a href="#CR29" rid="CR29" class=" bibr popnode">2013</a>). To the best of our knowledge, the immunogenic potential of these hypothetical proteins has not been explored yet. A few immunogenic hypothetical proteins have also been identified in one of our previous studies.</p><p id="Par64">The best scoring epitopes were then further subjected to the construction of a polypeptide multi-epitope vaccine construct with cholera toxin B (CtxB) subunit as an adjuvant for oral administration. The selection of a suitable adjuvant is a sensitive decision that has a greater impact on the efficacy of any vaccine. Hence we selected CtxB for its stability and efficacy as demonstrated in many recombinant vaccine strategies (Bavandpour et al. <a href="#CR6" rid="CR6" class=" bibr popnode">2020</a>). The predicted epitopes were joined together through the use of two types of linkers i.e. EAAAK and GPGPG. These linkers were employed in the construct owing to their potential role in avoiding the immunogenicity that might arise due to the juxtaposition of the epitopes. Moreover, several studies have reported them to be efficient in designing a rational vaccine as they are flexible and frequently utilized (Majid and Andleeb <a href="#CR92" rid="CR92" class=" bibr popnode">2019</a>). The designed vaccine construct was evaluated for its chemical and structural stability, solubility and ability to interact effectively with TLR receptors, to ensure its efficacy. Based on the results, the vaccine was found to be antigenic, non-allergenic and thermostable. The molecular weight was calculated to be 26.5&#x000a0;kDa which is indicative of its stability. According to these parameters, the designed vaccine is predicted to be a putative vaccine model that can be further validated in-vivo. To analyze the interactions of the vaccine with the immune receptors, molecular docking was performed which concluded favorable interactions. The immune simulations generated a consistent increase in the number of antigens and antibodies which depicts a harmonious increment in immune responses. Further, the immunoreactivity of the vaccine model was evaluated by its expression in E. coli (K12) (Ahmad et al. <a href="#CR93" rid="CR93" class=" bibr popnode">2021</a>), an appropriate host expression system. The optimization of the codon generated relevant CAI index (0.993) and GC content (55.7%) which are representatives of the suitable expression of the designed protein in the microorganism.</p><p id="Par65" class="p p-last">Antigenicity and druggability potentials of the selected proteins were evaluated in the second round of screening using the SVM approach. Our adopted strategy focused on essential enzymes involved in <em>P. aeruginosa</em> metabolism with the least homology in the host (human) genome. The proteome was screened and out of&#x02009;~&#x02009;6000 genes, initially, 209 cytoplasmic essential genes were obtained. Among those 148 were enzymes that catalyze essential metabolic pathways. Host homologs exclusion reduced the number to 9. These 9 proteins were lpxD, asd, wbpB, FolK, glyQ, ribF, acnB, WaaA and wbpH. The earliest genome screening study reported about 300 essential protein targets for <em>P. aeruginosa</em> as short communication (Sakharkar et al. <a href="#CR72" rid="CR72" class=" bibr popnode">2004</a>). This approach adopted reductive screening of <em>P. aeruginosa</em> genome and focused on the essential protein which could be exploited as therapeutic targets. Latter these authors published their finding in 2007 (Perumal, et al. <a href="#CR58" rid="CR58" class=" bibr popnode">2007</a>). Our predicted 4 out of 9 potential drug targets were also reported in their findings. Our reported targets were lpxD, WaaA, glyQ and folK, while similar proteins were reported as lpxA, lpxB, lpxC, lpxK, WaaP, glyS and folK. Another study by the same group focused on Constraint-based flux balance analysis for predicting novel targets for <em>P. aeruginosa</em> (Perumal <a href="#CR59" rid="CR59" class=" bibr popnode">2011</a>). The authors developed a genome-scale metabolic network by assessing the gene deletion effects on flux balance and screened 41 unique targets. When compared with our predicted drug targets again 4 out of 9 were similar proteins lpxD, WaaA, ribF and folk, while similar proteins were reported as lpxA, lpxB, lpxC, and lpxK, WaaP, WaaG, ribC, ribD, ribH, folp and folB. Unlike previous studies, the target druggability was also evaluated dually. The binding pockets were predicted and best scoring pockets are presented. While database search revealed similar targets in other bacterial pathogen providing confidence to the predicted drug targets reported. Overlapping predicted protein candidates and the ones in the discussed vaccination studies demonstrate the validity of our proposed pipeline. Hence this pipeline could also be employed for other pathogenic bacteria.</p></div><div id="Sec37" class="tsec sec"><h2 class="head no_bottom_margin" id="Sec37title">Conclusions</h2><p id="Par66" class="p p-first-last">The availability of complete genomes/proteomes have facilitated many computational approaches. In this study, we report novel vaccine candidate proteins, stepwise prioritized using Reverse Vaccinology approach for rapid reduction of unwanted number of <em>P. aeruginosa</em> proteins. The targeted proteins play a fundamental role in bacterial survival, pathogenesis, and progression of the infection. Moreover, 8 antigenic, non-toxic and non-allergenic MHC-I, MHC-II and B-cell epitopes were selected and merged with Cholera toxin subunit B adjuvant to enhance the immune response of the final chimeric vaccine construct to improve its immunogenic efficacy. The stability of multi-epitope vaccine construct was assessed using normal mode simulation and molecular docking analysis that could be further tested <em>in-vivo</em>. In addition, the designed vaccine need to be experimentally validated to ensure its efficacy against multiple <em>P. aeruginosa</em> infections. In the second round, we found 9 essential enzymes with the best druggability score. These identified drug targets were: UDP-2-acetamido-2-deoxy-d-glucuronic acid 3-dehydrogenase WbpB (<a id="__tag_1133178601" class="tag_hotlink" href="/protein/G3XD23" ref="/protein/G3XD23"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"G3XD23","term_id":"408387591","term_text":"G3XD23"}}</span>G3XD23</a>), aspartate semialdehyde dehydrogenase (<a id="__tag_1133178619" class="tag_hotlink" href="/protein/Q51344" ref="/protein/Q51344"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q51344","term_id":"12230918","term_text":"Q51344"}}</span>Q51344</a>), 2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase (<a id="__tag_1133178585" class="tag_hotlink" href="/protein/Q9HV71" ref="/protein/Q9HV71"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9HV71","term_id":"81539759","term_text":"Q9HV71"}}</span>Q9HV71</a>), 3-deoxy-D-manno-octulosonic-acid transferase (<a id="__tag_1133178603" class="tag_hotlink" href="/protein/Q9HUH7" ref="/protein/Q9HUH7"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9HUH7","term_id":"81539636","term_text":"Q9HUH7"}}</span>Q9HUH7</a>), glycyl-tRNA synthetase alpha chain (<a id="__tag_1133178635" class="tag_hotlink" href="/protein/Q9I7B7" ref="/protein/Q9I7B7"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9I7B7","term_id":"25009323","term_text":"Q9I7B7"}}</span>Q9I7B7</a>), riboflavin kinase/FAD synthase (<a id="__tag_1133178584" class="tag_hotlink" href="/protein/Q9HVM3" ref="/protein/Q9HVM3"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9HVM3","term_id":"81539848","term_text":"Q9HVM3"}}</span>Q9HVM3</a>), aconitate hydratase 2 (<a id="__tag_1133178607" class="tag_hotlink" href="/protein/Q9I2V5" ref="/protein/Q9I2V5"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9I2V5","term_id":"81541220","term_text":"Q9I2V5"}}</span>Q9I2V5</a>), probable glycosyltransferase WbpH (G3XD85) and UDP-3-O-[3-hydroxylauroyl] glucosamine N-acyltransferase (<a id="__tag_1133178621" class="tag_hotlink" href="/protein/Q9HXY6" ref="/protein/Q9HXY6"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9HXY6","term_id":"20138743","term_text":"Q9HXY6"}}</span>Q9HXY6</a>). Based on in-<em>silico</em> analyses, these reported vaccine and drug targets can help manage the antibiotic resistance phenomenon with special reference to hospital-acquired infections.</p></div><div id="sec-a.z.f" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="sec-a.z.ftitle">Supplementary Information</h2><!--/article/body/sec/--><div id="Sec38" class="sec sec-first-last"><p class="p p-first-last">Below is the link to the electronic supplementary material.</p><div class="sec suppmat" id="MOESM1"><div class="sup-box half_rhythm" id="media-a.z.f.b.a.a.a"><a href="/pmc/articles/PMC8351238/bin/10989_2021_10255_MOESM1_ESM.jpg" data-ga-action="click_feat_suppl">Supplementary file1 (JPG 345 kb)</a><sup>(345K, jpg)</sup></div></div><div class="sec suppmat" id="MOESM2"><div class="sup-box half_rhythm" id="media-a.z.f.b.a.b.a"><a href="/pmc/articles/PMC8351238/bin/10989_2021_10255_MOESM2_ESM.jpg" data-ga-action="click_feat_suppl">Supplementary file2 (JPG 154 kb)</a><sup>(154K, jpg)</sup></div></div><div class="sec suppmat" id="MOESM3"><div class="sup-box half_rhythm" id="media-a.z.f.b.a.c.a"><a href="/pmc/articles/PMC8351238/bin/10989_2021_10255_MOESM3_ESM.jpg" data-ga-action="click_feat_suppl">Supplementary file3 (JPG 545 kb)</a><sup>(545K, jpg)</sup></div></div><div class="sec suppmat" id="MOESM4"><div class="sup-box half_rhythm" id="media-a.z.f.b.a.d.a"><a href="/pmc/articles/PMC8351238/bin/10989_2021_10255_MOESM4_ESM.jpg" data-ga-action="click_feat_suppl">Supplementary file4 (JPG 416 kb)</a><sup>(416K, jpg)</sup></div></div><div class="sec suppmat" id="MOESM5"><div class="sup-box half_rhythm" id="media-a.z.f.b.a.e.a"><a href="/pmc/articles/PMC8351238/bin/10989_2021_10255_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl">Supplementary file5 (XLSX 40066 kb)</a><sup>(39M, xlsx)</sup></div></div><div class="sec suppmat" id="MOESM6"><div class="sup-box half_rhythm" id="media-a.z.f.b.a.f.a"><a href="/pmc/articles/PMC8351238/bin/10989_2021_10255_MOESM6_ESM.docx" data-ga-action="click_feat_suppl">Supplementary file6 (DOCX 14 kb)</a><sup>(15K, docx)</sup></div></div><div class="sec suppmat" id="MOESM7"><div class="sup-box half_rhythm" id="media-a.z.f.b.a.g.a"><a href="/pmc/articles/PMC8351238/bin/10989_2021_10255_MOESM7_ESM.docx" data-ga-action="click_feat_suppl">Supplementary file7 (DOCX 13 kb)</a><sup>(13K, docx)</sup></div></div><p></p></div></div><div id="notes-a.aa.b" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.aa.btitle">Author Contribution</h2><p>SI and SA designed the study and wrote the draft. MIR and MM performed analysis and helped in making figures. AA edited and revised the draft. SA reviewed and approved the final draft.</p></div><div id="notes-a.aa.c" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.aa.ctitle">Funding</h2><p>The study is based on computational analysis for prediction of vaccine and drug candidates in PAO1 genome. Therefore, there is no funding or support involved from any company or organization.</p></div><div id="notes-a.aa.d" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.aa.dtitle">Data Availability</h2><p>All data generated or analyzed during the study are included in the submitted manuscript. The sequences of the protein analyzed can be retrieved from UniProt database (uniport.org) using their accession numbers.</p></div><div id="notes-a.aa.e" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.aa.etitle">Declarations</h2><div>Conflict of interest<p id="Par67">There is no conflict of interest among the authors as well as with any organization in the world.</p></div><div>Research Involving Human and Animal Rights<p id="Par68">There is no involvement of human and/or animals trials in this study.</p></div></div><div id="fn-group-a.aa.a" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.aa.atitle">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="fn-a.aa.a.a"><p class="p p-first"><strong>Publisher's Note</strong></p><p class="p p-last">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></p></div></div><div id="Bib1" class="tsec sec"><h2 class="head no_bottom_margin" id="Bib1title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="CR1"><span class="element-citation">Adams CP, Brantner VV. Estimating the cost of new drug development: is it really $802 million? <span><span class="ref-journal">Health Aff. </span>2006;<span class="ref-vol">25</span>(2):420–428. doi: 10.1377/hlthaff.25.2.420.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16522582" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1377%2Fhlthaff.25.2.420" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Health+Aff&amp;title=Estimating+the+cost+of+new+drug+development:+is+it+really+$802+million?&amp;author=CP+Adams&amp;author=VV+Brantner&amp;volume=25&amp;issue=2&amp;publication_year=2006&amp;pages=420-428&amp;doi=10.1377/hlthaff.25.2.420&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR93"><span class="element-citation">Ahmad S, et al. Immuno-informatics analysis of pakistan-based HCV subtype-3a for chimeric polypeptide vaccine design. <span><span class="ref-journal">Vaccines. </span>2021;<span class="ref-vol">9</span>(3):293. doi: 10.3390/vaccines9030293.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8004085/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33801143" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fvaccines9030293" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Vaccines&amp;title=Immuno-informatics+analysis+of+pakistan-based+HCV+subtype-3a+for+chimeric+polypeptide+vaccine+design&amp;author=S+Ahmad&amp;volume=9&amp;issue=3&amp;publication_year=2021&amp;pages=293&amp;pmid=33801143&amp;doi=10.3390/vaccines9030293&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR2"><span class="element-citation">Altschul SF, et al. Basic local alignment search tool. <span><span class="ref-journal">J Mol Biol. </span>1990;<span class="ref-vol">215</span>(3):403–410. doi: 10.1016/S0022-2836(05)80360-2.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2231712" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FS0022-2836(05)80360-2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Basic+local+alignment+search+tool&amp;author=SF+Altschul&amp;volume=215&amp;issue=3&amp;publication_year=1990&amp;pages=403-410&amp;pmid=2231712&amp;doi=10.1016/S0022-2836(05)80360-2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR3"><span class="element-citation">Arnold K, et al. The SWISS-MODEL workspace: a web-based environment for protein structure homology modelling. <span><span class="ref-journal">Bioinformatics (oxford, England) </span>2006;<span class="ref-vol">22</span>(2):195–201. doi: 10.1093/bioinformatics/bti770.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16301204" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbti770" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics+(oxford,+England)&amp;title=The+SWISS-MODEL+workspace:+a+web-based+environment+for+protein+structure+homology+modelling&amp;author=K+Arnold&amp;volume=22&amp;issue=2&amp;publication_year=2006&amp;pages=195-201&amp;doi=10.1093/bioinformatics/bti770&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR4"><span class="element-citation">Bairoch A, Apweiler R. The SWISS-PROT protein sequence database and its supplement TrEMBL in 2000. <span><span class="ref-journal">Nucleic Acids Res. </span>2000;<span class="ref-vol">28</span>(1):45–48. doi: 10.1093/nar/28.1.45.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC102476/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10592178" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2F28.1.45" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=The+SWISS-PROT+protein+sequence+database+and+its+supplement+TrEMBL+in+2000&amp;author=A+Bairoch&amp;author=R+Apweiler&amp;volume=28&amp;issue=1&amp;publication_year=2000&amp;pages=45-48&amp;pmid=10592178&amp;doi=10.1093/nar/28.1.45&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR5"><span class="element-citation">Baker SM, McLachlan JB, Morici LA. Immunological considerations in the development of <em>Pseudomonas aeruginosa</em> vaccines. <span><span class="ref-journal">Hum Vaccin Immunother. </span>2019 doi: 10.1080/21645515.2019.1650999.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7062425/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31368828" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1080%2F21645515.2019.1650999" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Vaccin+Immunother&amp;title=Immunological+considerations+in+the+development+of+Pseudomonas+aeruginosa+vaccines&amp;author=SM+Baker&amp;author=JB+McLachlan&amp;author=LA+Morici&amp;publication_year=2019&amp;pmid=31368828&amp;doi=10.1080/21645515.2019.1650999&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR6"><span class="element-citation">Bavandpour AK, Bakhshi B, Najar-Peerayeh S. The roles of mesoporous silica and carbon nanoparticles in antigen stability and intensity of immune response against recombinant subunit B of cholera toxin in a rabbit animal model. <span><span class="ref-journal">Int J Pharm. </span>2020;<span class="ref-vol">573</span>:118868. doi: 10.1016/j.ijpharm.2019.118868.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31765785" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.ijpharm.2019.118868" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Pharm&amp;title=The+roles+of+mesoporous+silica+and+carbon+nanoparticles+in+antigen+stability+and+intensity+of+immune+response+against+recombinant+subunit+B+of+cholera+toxin+in+a+rabbit+animal+model&amp;author=AK+Bavandpour&amp;author=B+Bakhshi&amp;author=S+Najar-Peerayeh&amp;volume=573&amp;publication_year=2020&amp;pages=118868&amp;pmid=31765785&amp;doi=10.1016/j.ijpharm.2019.118868&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR7"><span class="element-citation">Berman H, et al. The worldwide Protein Data Bank (wwPDB): ensuring a single, uniform archive of PDB data. <span><span class="ref-journal">Nucleic Acids Res. </span>2006;<span class="ref-vol">35</span>:D301–D303. doi: 10.1093/nar/gkl971.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1669775/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17142228" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkl971" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=The+worldwide+Protein+Data+Bank+(wwPDB):+ensuring+a+single,+uniform+archive+of+PDB+data&amp;author=H+Berman&amp;volume=35&amp;publication_year=2006&amp;pages=D301-D303&amp;pmid=17142228&amp;doi=10.1093/nar/gkl971&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR8"><span class="element-citation">Bilofsky HS, Christian B. The GenBank&#x000ae; genetic sequence data bank. <span><span class="ref-journal">Nucleic Acids Res. </span>1988;<span class="ref-vol">16</span>(5):1861–1863. doi: 10.1093/nar/16.5.1861.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC338181/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3353225" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2F16.5.1861" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=The+GenBank&#x000ae;+genetic+sequence+data+bank&amp;author=HS+Bilofsky&amp;author=B+Christian&amp;volume=16&amp;issue=5&amp;publication_year=1988&amp;pages=1861-1863&amp;pmid=3353225&amp;doi=10.1093/nar/16.5.1861&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR9"><span class="element-citation">Brunham RC, Plummer FA, Stephens RS. Bacterial antigenic variation, host immune response, and pathogen-host coevolution. <span><span class="ref-journal">Infect Immun. </span>1993;<span class="ref-vol">61</span>(6):2273. doi: 10.1128/iai.61.6.2273-2276.1993.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC280844/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8500868" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2Fiai.61.6.2273-2276.1993" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Infect+Immun&amp;title=Bacterial+antigenic+variation,+host+immune+response,+and+pathogen-host+coevolution&amp;author=RC+Brunham&amp;author=FA+Plummer&amp;author=RS+Stephens&amp;volume=61&amp;issue=6&amp;publication_year=1993&amp;pages=2273&amp;pmid=8500868&amp;doi=10.1128/iai.61.6.2273-2276.1993&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR10"><span class="element-citation">Buchan DWA, Jones DT. The PSIPRED protein analysis workbench: 20 years on. <span><span class="ref-journal">Nucleic Acids Res. </span>2019;<span class="ref-vol">47</span>(W1):W402–W407. doi: 10.1093/nar/gkz297.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6602445/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31251384" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkz297" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=The+PSIPRED+protein+analysis+workbench:+20+years+on&amp;author=DWA+Buchan&amp;author=DT+Jones&amp;volume=47&amp;issue=W1&amp;publication_year=2019&amp;pages=W402-W407&amp;pmid=31251384&amp;doi=10.1093/nar/gkz297&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR11"><span class="element-citation">Calabriso N, et al. Multiple anti-inflammatory and anti-atherosclerotic properties of red wine polyphenolic extracts: differential role of hydroxycinnamic acids, flavonols and stilbenes on endothelial inflammatory gene expression. <span><span class="ref-journal">Eur J Nutr. </span>2016;<span class="ref-vol">55</span>(2):477–489. doi: 10.1007/s00394-015-0865-6.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25724173" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs00394-015-0865-6" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Nutr&amp;title=Multiple+anti-inflammatory+and+anti-atherosclerotic+properties+of+red+wine+polyphenolic+extracts:+differential+role+of+hydroxycinnamic+acids,+flavonols+and+stilbenes+on+endothelial+inflammatory+gene+expression&amp;author=N+Calabriso&amp;volume=55&amp;issue=2&amp;publication_year=2016&amp;pages=477-489&amp;pmid=25724173&amp;doi=10.1007/s00394-015-0865-6&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR12"><span class="element-citation">Chen L, et al. VFDB 2012 update: toward the genetic diversity and molecular evolution of bacterial virulence factors. <span><span class="ref-journal">Nucleic Acids Res. </span>2011 doi: 10.1093/nar/gkr989.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3245122/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22067448" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkr989" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=VFDB+2012+update:+toward+the+genetic+diversity+and+molecular+evolution+of+bacterial+virulence+factors&amp;author=L+Chen&amp;publication_year=2011&amp;pmid=22199257&amp;doi=10.1093/nar/gkr989&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR13"><span class="element-citation">Craik DJ, et al. The future of peptide-based drugs. <span><span class="ref-journal">Chem Biol Drug Des. </span>2013;<span class="ref-vol">81</span>(1):136–147. doi: 10.1111/cbdd.12055.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23253135" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fcbdd.12055" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem+Biol+Drug+Des&amp;title=The+future+of+peptide-based+drugs&amp;author=DJ+Craik&amp;volume=81&amp;issue=1&amp;publication_year=2013&amp;pages=136-147&amp;pmid=23253135&amp;doi=10.1111/cbdd.12055&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR14"><span class="element-citation">Dalsass M, et al. Comparison of open-source reverse vaccinology programs for bacterial vaccine antigen discovery. <span><span class="ref-journal">Front Immunol. </span>2019;<span class="ref-vol">10</span>:113. doi: 10.3389/fimmu.2019.00113.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6382693/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30837982" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3389%2Ffimmu.2019.00113" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front+Immunol&amp;title=Comparison+of+open-source+reverse+vaccinology+programs+for+bacterial+vaccine+antigen+discovery&amp;author=M+Dalsass&amp;volume=10&amp;publication_year=2019&amp;pages=113&amp;pmid=30837982&amp;doi=10.3389/fimmu.2019.00113&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR15"><span class="element-citation">de Vries SJ, van Dijk M, Bonvin AM. The HADDOCK web server for data-driven biomolecular docking. <span><span class="ref-journal">Nat Protoc. </span>2010;<span class="ref-vol">5</span>(5):883–897. doi: 10.1038/nprot.2010.32.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20431534" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnprot.2010.32" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Protoc&amp;title=The+HADDOCK+web+server+for+data-driven+biomolecular+docking&amp;author=SJ+de+Vries&amp;author=M+van+Dijk&amp;author=AM+Bonvin&amp;volume=5&amp;issue=5&amp;publication_year=2010&amp;pages=883-897&amp;pmid=20431534&amp;doi=10.1038/nprot.2010.32&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR16"><span class="element-citation">Delany I, Rappuoli R, Seib KL. Vaccines, reverse vaccinology, and bacterial pathogenesis. <span><span class="ref-journal">Cold Spring Harb Perspect Med. </span>2013;<span class="ref-vol">3</span>(5):a012476. doi: 10.1101/cshperspect.a012476.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3633180/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23637311" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1101%2Fcshperspect.a012476" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cold+Spring+Harb+Perspect+Med&amp;title=Vaccines,+reverse+vaccinology,+and+bacterial+pathogenesis&amp;author=I+Delany&amp;author=R+Rappuoli&amp;author=KL+Seib&amp;volume=3&amp;issue=5&amp;publication_year=2013&amp;pages=a012476&amp;pmid=23637311&amp;doi=10.1101/cshperspect.a012476&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR17"><span class="element-citation">Dimitrov I, et al. AllerTOP vol 2&#x02013;a server for in silico prediction of allergens. <span><span class="ref-journal">J Mol Model. </span>2014;<span class="ref-vol">20</span>(6):2278. doi: 10.1007/s00894-014-2278-5.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24878803" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs00894-014-2278-5" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Model&amp;title=AllerTOP+vol+2&#x02013;a+server+for+in+silico+prediction+of+allergens&amp;author=I+Dimitrov&amp;volume=20&amp;issue=6&amp;publication_year=2014&amp;pages=2278&amp;pmid=24878803&amp;doi=10.1007/s00894-014-2278-5&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR18"><span class="element-citation">Doytchinova IA, Flower DR. VaxiJen: a server for prediction of protective antigens, tumour antigens and subunit vaccines. <span><span class="ref-journal">BMC Bioinf. </span>2007;<span class="ref-vol">8</span>(1):4. doi: 10.1186/1471-2105-8-4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1780059/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17207271" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2F1471-2105-8-4" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinf&amp;title=VaxiJen:+a+server+for+prediction+of+protective+antigens,+tumour+antigens+and+subunit+vaccines&amp;author=IA+Doytchinova&amp;author=DR+Flower&amp;volume=8&amp;issue=1&amp;publication_year=2007&amp;pages=4&amp;doi=10.1186/1471-2105-8-4&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR19"><span class="element-citation">Fito-Boncompte L, et al. Full virulence of <em>Pseudomonas aeruginosa</em> requires OprF. <span><span class="ref-journal">Infect Immun. </span>2011;<span class="ref-vol">79</span>(3):1176–1186. doi: 10.1128/IAI.00850-10.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3067511/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21189321" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FIAI.00850-10" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Infect+Immun&amp;title=Full+virulence+of+Pseudomonas+aeruginosa+requires+OprF&amp;author=L+Fito-Boncompte&amp;volume=79&amp;issue=3&amp;publication_year=2011&amp;pages=1176-1186&amp;pmid=21189321&amp;doi=10.1128/IAI.00850-10&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR20"><span class="element-citation">Foged C, Hansen J, Agger EM. License to kill: formulation requirements for optimal priming of CD8+ CTL responses with particulate vaccine delivery systems. <span><span class="ref-journal">Eur J Pharm Sci. </span>2012;<span class="ref-vol">45</span>(4):482–491. doi: 10.1016/j.ejps.2011.08.016.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21888971" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.ejps.2011.08.016" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Pharm+Sci&amp;title=License+to+kill:+formulation+requirements+for+optimal+priming+of+CD8++CTL+responses+with+particulate+vaccine+delivery+systems&amp;author=C+Foged&amp;author=J+Hansen&amp;author=EM+Agger&amp;volume=45&amp;issue=4&amp;publication_year=2012&amp;pages=482-491&amp;pmid=21888971&amp;doi=10.1016/j.ejps.2011.08.016&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR21"><span class="element-citation">Fran&#x000e7;ois B, et al. Safety and pharmacokinetics of an anti-PcrV PEGylated monoclonal antibody fragment in mechanically ventilated patients colonized with <em>Pseudomonas aeruginosa</em>: a randomized, double-blind, placebo-controlled trial. <span><span class="ref-journal">Crit Care Med. </span>2012;<span class="ref-vol">40</span>(8):2320–2326. doi: 10.1097/CCM.0b013e31825334f6.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22622405" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1097%2FCCM.0b013e31825334f6" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Crit+Care+Med&amp;title=Safety+and+pharmacokinetics+of+an+anti-PcrV+PEGylated+monoclonal+antibody+fragment+in+mechanically+ventilated+patients+colonized+with+Pseudomonas+aeruginosa:+a+randomized,+double-blind,+placebo-controlled+trial&amp;author=B+Fran&#x000e7;ois&amp;volume=40&amp;issue=8&amp;publication_year=2012&amp;pages=2320-2326&amp;pmid=22622405&amp;doi=10.1097/CCM.0b013e31825334f6&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR22"><span class="element-citation">Frieri M, Kumar K, Boutin A. Antibiotic resistance. <span><span class="ref-journal">J Infect Public Health. </span>2017;<span class="ref-vol">10</span>(4):369–378. doi: 10.1016/j.jiph.2016.08.007.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27616769" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.jiph.2016.08.007" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Infect+Public+Health&amp;title=Antibiotic+resistance&amp;author=M+Frieri&amp;author=K+Kumar&amp;author=A+Boutin&amp;volume=10&amp;issue=4&amp;publication_year=2017&amp;pages=369-378&amp;pmid=27616769&amp;doi=10.1016/j.jiph.2016.08.007&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR23"><span class="element-citation">Garg A, Gupta D. VirulentPred: a SVM based prediction method for virulent proteins in bacterial pathogens. <span><span class="ref-journal">BMC Bioinf. </span>2008;<span class="ref-vol">9</span>(1):62. doi: 10.1186/1471-2105-9-62.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2254373/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18226234" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2F1471-2105-9-62" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinf&amp;title=VirulentPred:+a+SVM+based+prediction+method+for+virulent+proteins+in+bacterial+pathogens&amp;author=A+Garg&amp;author=D+Gupta&amp;volume=9&amp;issue=1&amp;publication_year=2008&amp;pages=62&amp;doi=10.1186/1471-2105-9-62&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR24"><span class="element-citation">Gasteiger E, et al.  <span class="ref-journal">Protein identification and analysis tools on the ExPASy server.</span> Berlin: Springer; 2005.  [<a href="https://pubmed.ncbi.nlm.nih.gov/10027275" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Protein+identification+and+analysis+tools+on+the+ExPASy+server&amp;author=E+Gasteiger&amp;publication_year=2005&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR25"><span class="element-citation">Gellatly SL, Hancock RE. <em>Pseudomonas aeruginosa</em>: new insights into pathogenesis and host defenses. <span><span class="ref-journal">Pathogens and Disease. </span>2013;<span class="ref-vol">67</span>(3):159–173. doi: 10.1111/2049-632X.12033.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23620179" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2F2049-632X.12033" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pathogens+and+Disease&amp;title=Pseudomonas+aeruginosa:+new+insights+into+pathogenesis+and+host+defenses&amp;author=SL+Gellatly&amp;author=RE+Hancock&amp;volume=67&amp;issue=3&amp;publication_year=2013&amp;pages=159-173&amp;pmid=23620179&amp;doi=10.1111/2049-632X.12033&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR26"><span class="element-citation">Giuliani MM, et al. A universal vaccine for serogroup B meningococcus. <span><span class="ref-journal">Proc Natl Acad Sci. </span>2006;<span class="ref-vol">103</span>(29):10834–10839. doi: 10.1073/pnas.0603940103.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2047628/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16825336" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.0603940103" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci&amp;title=A+universal+vaccine+for+serogroup+B+meningococcus&amp;author=MM+Giuliani&amp;volume=103&amp;issue=29&amp;publication_year=2006&amp;pages=10834-10839&amp;pmid=16825336&amp;doi=10.1073/pnas.0603940103&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR28"><span class="element-citation">Guan P, et al. MHCPred: a server for quantitative prediction of peptide&#x02013;MHC binding. <span><span class="ref-journal">Nucleic Acids Res. </span>2003;<span class="ref-vol">31</span>(13):3621–3624. doi: 10.1093/nar/gkg510.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC168917/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12824380" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkg510" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=MHCPred:+a+server+for+quantitative+prediction+of+peptide&#x02013;MHC+binding&amp;author=P+Guan&amp;volume=31&amp;issue=13&amp;publication_year=2003&amp;pages=3621-3624&amp;pmid=12824380&amp;doi=10.1093/nar/gkg510&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR29"><span class="element-citation">Harmer C, et al. Modulation of gene expression by <em>Pseudomonas aeruginosa</em> during chronic infection in the adult cystic fibrosis lung. <span><span class="ref-journal">Microbiology. </span>2013;<span class="ref-vol">159</span>:2354–2363. doi: 10.1099/mic.0.066985-0.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24014664" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1099%2Fmic.0.066985-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology&amp;title=Modulation+of+gene+expression+by+Pseudomonas+aeruginosa+during+chronic+infection+in+the+adult+cystic+fibrosis+lung&amp;author=C+Harmer&amp;volume=159&amp;publication_year=2013&amp;pages=2354-2363&amp;pmid=24014664&amp;doi=10.1099/mic.0.066985-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR30"><span class="element-citation">He Y, Xiang Z, Mobley HL. Vaxign: the first web-based vaccine design program for reverse vaccinology and applications for vaccine development. <span><span class="ref-journal">J Biomed Biotechnol. </span>2010;<span class="ref-vol">2010</span>:297505. doi: 10.1155/2010/297505.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2910479/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20671958" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1155%2F2010%2F297505" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biomed+Biotechnol&amp;title=Vaxign:+the+first+web-based+vaccine+design+program+for+reverse+vaccinology+and+applications+for+vaccine+development&amp;author=Y+He&amp;author=Z+Xiang&amp;author=HL+Mobley&amp;volume=2010&amp;publication_year=2010&amp;pages=297505&amp;pmid=20671958&amp;doi=10.1155/2010/297505&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR31"><span class="element-citation">Hebditch M, et al. Protein-Sol: a web tool for predicting protein solubility from sequence. <span><span class="ref-journal">Bioinformatics. </span>2017;<span class="ref-vol">33</span>(19):3098–3100. doi: 10.1093/bioinformatics/btx345.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5870856/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28575391" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtx345" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Protein-Sol:+a+web+tool+for+predicting+protein+solubility+from+sequence&amp;author=M+Hebditch&amp;volume=33&amp;issue=19&amp;publication_year=2017&amp;pages=3098-3100&amp;pmid=28575391&amp;doi=10.1093/bioinformatics/btx345&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR32"><span class="element-citation">Hedgecoe A, Martin P. The drugs don't work: expectations and the shaping of pharmacogenetics. <span><span class="ref-journal">Soc Stud Sci. </span>2003;<span class="ref-vol">33</span>(3):327–364. doi: 10.1177/03063127030333002.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14621671" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1177%2F03063127030333002" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Soc+Stud+Sci&amp;title=The+drugs+don't+work:+expectations+and+the+shaping+of+pharmacogenetics&amp;author=A+Hedgecoe&amp;author=P+Martin&amp;volume=33&amp;issue=3&amp;publication_year=2003&amp;pages=327-364&amp;pmid=14621671&amp;doi=10.1177/03063127030333002&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR33"><span class="element-citation">Heo L, Park H, Seok C. GalaxyRefine: protein structure refinement driven by side-chain repacking. <span><span class="ref-journal">Nucleic Acids Res. </span>2013;<span class="ref-vol">41</span>(W1):W384–W388. doi: 10.1093/nar/gkt458.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3692086/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23737448" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkt458" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=GalaxyRefine:+protein+structure+refinement+driven+by+side-chain+repacking&amp;author=L+Heo&amp;author=H+Park&amp;author=C+Seok&amp;volume=41&amp;issue=W1&amp;publication_year=2013&amp;pages=W384-W388&amp;pmid=23737448&amp;doi=10.1093/nar/gkt458&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR34"><span class="element-citation">Hopkins AL, Groom CR. The druggable genome. <span><span class="ref-journal">Nat Rev Drug Discovery. </span>2002;<span class="ref-vol">1</span>(9):727. doi: 10.1038/nrd892.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12209152" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnrd892" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Drug+Discovery&amp;title=The+druggable+genome&amp;author=AL+Hopkins&amp;author=CR+Groom&amp;volume=1&amp;issue=9&amp;publication_year=2002&amp;pages=727&amp;pmid=12209152&amp;doi=10.1038/nrd892&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR35"><span class="element-citation">Juan C, Pe&#x000f1;a C, Oliver A. Host and pathogen biomarkers for severe <em>Pseudomonas aeruginosa</em> infections. <span><span class="ref-journal">J Infect Dis. </span>2017;<span class="ref-vol">215</span>:S44–S51. doi: 10.1093/infdis/jiw299.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28375513" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Finfdis%2Fjiw299" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Infect+Dis&amp;title=Host+and+pathogen+biomarkers+for+severe+Pseudomonas+aeruginosa+infections&amp;author=C+Juan&amp;author=C+Pe&#x000f1;a&amp;author=A+Oliver&amp;volume=215&amp;publication_year=2017&amp;pages=S44-S51&amp;pmid=28375513&amp;doi=10.1093/infdis/jiw299&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR36"><span class="element-citation">Juhas M. <em>Pseudomonas aeruginosa</em> essentials: an update on investigation of essential genes. <span><span class="ref-journal">Microbiology. </span>2015;<span class="ref-vol">161</span>(11):2053–2060. doi: 10.1099/mic.0.000161.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26311069" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1099%2Fmic.0.000161" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology&amp;title=Pseudomonas+aeruginosa+essentials:+an+update+on+investigation+of+essential+genes&amp;author=M+Juhas&amp;volume=161&amp;issue=11&amp;publication_year=2015&amp;pages=2053-2060&amp;pmid=26311069&amp;doi=10.1099/mic.0.000161&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR37"><span class="element-citation">K&#x000e4;llberg M, et al. Template-based protein structure modeling using the RaptorX web server. <span><span class="ref-journal">Nat Protoc. </span>2012;<span class="ref-vol">7</span>(8):1511. doi: 10.1038/nprot.2012.085.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4730388/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22814390" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnprot.2012.085" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Protoc&amp;title=Template-based+protein+structure+modeling+using+the+RaptorX+web+server&amp;author=M+K&#x000e4;llberg&amp;volume=7&amp;issue=8&amp;publication_year=2012&amp;pages=1511&amp;pmid=22814390&amp;doi=10.1038/nprot.2012.085&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR38"><span class="element-citation">Karkman A, et al. Antibiotic-resistance genes in waste water. <span><span class="ref-journal">Trends Microbiol. </span>2018;<span class="ref-vol">26</span>(3):220–228. doi: 10.1016/j.tim.2017.09.005.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29033338" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.tim.2017.09.005" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Microbiol&amp;title=Antibiotic-resistance+genes+in+waste+water&amp;author=A+Karkman&amp;volume=26&amp;issue=3&amp;publication_year=2018&amp;pages=220-228&amp;pmid=29033338&amp;doi=10.1016/j.tim.2017.09.005&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR39"><span class="element-citation">Keller TH, Pichota A, Yin Z. A practical view of &#x02018;druggability&#x02019; <span><span class="ref-journal">Curr Opin Chem Biol. </span>2006;<span class="ref-vol">10</span>(4):357–361. doi: 10.1016/j.cbpa.2006.06.014.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16814592" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cbpa.2006.06.014" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Chem+Biol&amp;title=A+practical+view+of+&#x02018;druggability&#x02019;&amp;author=TH+Keller&amp;author=A+Pichota&amp;author=Z+Yin&amp;volume=10&amp;issue=4&amp;publication_year=2006&amp;pages=357-361&amp;pmid=16814592&amp;doi=10.1016/j.cbpa.2006.06.014&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR40"><span class="element-citation">Knapp B, et al. A recombinant hybrid outer membrane protein for vaccination against <em>Pseudomonas aeruginosa</em>. <span><span class="ref-journal">Vaccine. </span>1999;<span class="ref-vol">17</span>(13&#x02013;14):1663–1666. doi: 10.1016/s0264-410x(98)00420-4.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10194820" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fs0264-410x(98)00420-4" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Vaccine&amp;title=A+recombinant+hybrid+outer+membrane+protein+for+vaccination+against+Pseudomonas+aeruginosa&amp;author=B+Knapp&amp;volume=17&amp;issue=13&#x02013;14&amp;publication_year=1999&amp;pages=1663-1666&amp;pmid=10194820&amp;doi=10.1016/s0264-410x(98)00420-4&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR41"><span class="element-citation">Lenz GR, Nash HM, Jindal S. Chemical ligands, genomics and drug discovery. <span><span class="ref-journal">Drug Discovery Today. </span>2000;<span class="ref-vol">5</span>(4):145–156. doi: 10.1016/s1359-6446(00)01468-9.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10729820" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fs1359-6446(00)01468-9" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Drug+Discovery+Today&amp;title=Chemical+ligands,+genomics+and+drug+discovery&amp;author=GR+Lenz&amp;author=HM+Nash&amp;author=S+Jindal&amp;volume=5&amp;issue=4&amp;publication_year=2000&amp;pages=145-156&amp;pmid=10729820&amp;doi=10.1016/s1359-6446(00)01468-9&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR42"><span class="element-citation">Li W, Godzik A. Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences. <span><span class="ref-journal">Bioinformatics. </span>2006;<span class="ref-vol">22</span>(13):1658–1659. doi: 10.1093/bioinformatics/btl158.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16731699" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtl158" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Cd-hit:+a+fast+program+for+clustering+and+comparing+large+sets+of+protein+or+nucleotide+sequences&amp;author=W+Li&amp;author=A+Godzik&amp;volume=22&amp;issue=13&amp;publication_year=2006&amp;pages=1658-1659&amp;pmid=16731699&amp;doi=10.1093/bioinformatics/btl158&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR43"><span class="element-citation">Li L, Stoeckert CJ, Roos DS. OrthoMCL: identification of ortholog groups for eukaryotic genomes. <span><span class="ref-journal">Genome Res. </span>2003;<span class="ref-vol">13</span>(9):2178–2189. doi: 10.1101/gr.1224503.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC403725/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12952885" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1101%2Fgr.1224503" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=OrthoMCL:+identification+of+ortholog+groups+for+eukaryotic+genomes&amp;author=L+Li&amp;author=CJ+Stoeckert&amp;author=DS+Roos&amp;volume=13&amp;issue=9&amp;publication_year=2003&amp;pages=2178-2189&amp;pmid=12952885&amp;doi=10.1101/gr.1224503&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR44"><span class="element-citation">Lister PD, Wolter DJ, Hanson ND. Antibacterial-resistant <em>Pseudomonas aeruginosa</em>: clinical impact and complex regulation of chromosomally encoded resistance mechanisms. <span><span class="ref-journal">Clin Microbiol Rev. </span>2009;<span class="ref-vol">22</span>(4):582–610. doi: 10.1128/CMR.00040-09.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2772362/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19822890" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FCMR.00040-09" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Microbiol+Rev&amp;title=Antibacterial-resistant+Pseudomonas+aeruginosa:+clinical+impact+and+complex+regulation+of+chromosomally+encoded+resistance+mechanisms&amp;author=PD+Lister&amp;author=DJ+Wolter&amp;author=ND+Hanson&amp;volume=22&amp;issue=4&amp;publication_year=2009&amp;pages=582-610&amp;pmid=19822890&amp;doi=10.1128/CMR.00040-09&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR45"><span class="element-citation">Lovell SC, et al. Structure validation by Calpha geometry: phi, psi and Cbeta deviation. <span><span class="ref-journal">Proteins. </span>2003;<span class="ref-vol">50</span>(3):437–450. doi: 10.1002/prot.10286.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12557186" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fprot.10286" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Structure+validation+by+Calpha+geometry:+phi,+psi+and+Cbeta+deviation&amp;author=SC+Lovell&amp;volume=50&amp;issue=3&amp;publication_year=2003&amp;pages=437-450&amp;pmid=12557186&amp;doi=10.1002/prot.10286&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR46"><span class="element-citation">L&#x000fc;cking H, et al. The <em>Pseudomonas aeruginosa</em> outer membrane protein I vaccine: immunogenicity and safe administration in man. <span><span class="ref-journal">Behring Inst Mitt. </span>1997;<span class="ref-vol">98</span>:326–337.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9382756" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Behring+Inst+Mitt&amp;title=The+Pseudomonas+aeruginosa+outer+membrane+protein+I+vaccine:+immunogenicity+and+safe+administration+in+man&amp;author=H+L&#x000fc;cking&amp;volume=98&amp;publication_year=1997&amp;pages=326-337&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR47"><span class="element-citation">Luo H, et al. DEG 10, an update of the database of essential genes that includes both protein-coding genes and noncoding genomic elements. <span><span class="ref-journal">Nucleic Acids Res. </span>2014;<span class="ref-vol">42</span>(D1):D574–D580. doi: 10.1093/nar/gkt1131.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3965060/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24243843" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkt1131" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=DEG+10,+an+update+of+the+database+of+essential+genes+that+includes+both+protein-coding+genes+and+noncoding+genomic+elements&amp;author=H+Luo&amp;volume=42&amp;issue=D1&amp;publication_year=2014&amp;pages=D574-D580&amp;pmid=24243843&amp;doi=10.1093/nar/gkt1131&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR92"><span class="element-citation">Majid M, Andleeb S. Designing a multi-epitopic vaccine against the enterotoxigenic Bacteroides fragilis based on immunoinformatics approach. <span><span class="ref-journal">Sci Rep. </span>2019;<span class="ref-vol">9</span>(1):1–15. doi: 10.1038/s41598-019-55613-w.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6930219/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31874963" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41598-019-55613-w" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci+Rep&amp;title=Designing+a+multi-epitopic+vaccine+against+the+enterotoxigenic+Bacteroides+fragilis+based+on+immunoinformatics+approach&amp;author=M+Majid&amp;author=S+Andleeb&amp;volume=9&amp;issue=1&amp;publication_year=2019&amp;pages=1-15&amp;pmid=30626917&amp;doi=10.1038/s41598-019-55613-w&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR48"><span class="element-citation">Mansouri E, et al. Clinical study to assess the immunogenicity and safety of a recombinant <em>Pseudomonas aeruginosa</em> OprF-OprI vaccine in burn patients. <span><span class="ref-journal">FEMS Immunol Med Microbiol. </span>2003;<span class="ref-vol">37</span>(2&#x02013;3):161–166. doi: 10.1016/S0928-8244(03)00072-5.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12832120" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FS0928-8244(03)00072-5" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEMS+Immunol+Med+Microbiol&amp;title=Clinical+study+to+assess+the+immunogenicity+and+safety+of+a+recombinant+Pseudomonas+aeruginosa+OprF-OprI+vaccine+in+burn+patients&amp;author=E+Mansouri&amp;volume=37&amp;issue=2&#x02013;3&amp;publication_year=2003&amp;pages=161-166&amp;pmid=12832120&amp;doi=10.1016/S0928-8244(03)00072-5&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR49"><span class="element-citation">Mayrose I, et al. Pepitope: epitope mapping from affinity-selected peptides. <span><span class="ref-journal">Bioinformatics. </span>2007;<span class="ref-vol">23</span>(23):3244–3246. doi: 10.1093/bioinformatics/btm493.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7110138/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17977889" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtm493" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Pepitope:+epitope+mapping+from+affinity-selected+peptides&amp;author=I+Mayrose&amp;volume=23&amp;issue=23&amp;publication_year=2007&amp;pages=3244-3246&amp;pmid=17977889&amp;doi=10.1093/bioinformatics/btm493&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR50"><span class="element-citation">McLean K, et al. Genomic analysis identifies novel <em>Pseudomonas aeruginosa</em> resistance genes under selection during inhaled aztreonam therapy in vivo. <span><span class="ref-journal">Antimicrob Agents Chemother. </span>2019;<span class="ref-vol">63</span>(9):e00866–e919. doi: 10.1128/AAC.00866-19.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6709462/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31285231" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FAAC.00866-19" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Antimicrob+Agents+Chemother&amp;title=Genomic+analysis+identifies+novel+Pseudomonas+aeruginosa+resistance+genes+under+selection+during+inhaled+aztreonam+therapy+in+vivo&amp;author=K+McLean&amp;volume=63&amp;issue=9&amp;publication_year=2019&amp;pages=e00866-e919&amp;pmid=31285231&amp;doi=10.1128/AAC.00866-19&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR51"><span class="element-citation">Montor WR, et al. Genome-wide study of <em>Pseudomonas aeruginosa</em> outer membrane protein immunogenicity using self-assembling protein microarrays. <span><span class="ref-journal">Infect Immun. </span>2009;<span class="ref-vol">77</span>(11):4877–4886. doi: 10.1128/IAI.00698-09.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2772540/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19737893" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FIAI.00698-09" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Infect+Immun&amp;title=Genome-wide+study+of+Pseudomonas+aeruginosa+outer+membrane+protein+immunogenicity+using+self-assembling+protein+microarrays&amp;author=WR+Montor&amp;volume=77&amp;issue=11&amp;publication_year=2009&amp;pages=4877-4886&amp;pmid=19737893&amp;doi=10.1128/IAI.00698-09&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR52"><span class="element-citation">Moradali MF, Ghods S, Rehm BH. Pseudomonas aeruginosa lifestyle: a paradigm for adaptation, survival, and persistence. <span><span class="ref-journal">Front Cell Infect Microbiol. </span>2017;<span class="ref-vol">7</span>:39. doi: 10.3389/fcimb.2017.00039.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5310132/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28261568" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3389%2Ffcimb.2017.00039" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front+Cell+Infect+Microbiol&amp;title=Pseudomonas+aeruginosa+lifestyle:+a+paradigm+for+adaptation,+survival,+and+persistence&amp;author=MF+Moradali&amp;author=S+Ghods&amp;author=BH+Rehm&amp;volume=7&amp;publication_year=2017&amp;pages=39&amp;pmid=28261568&amp;doi=10.3389/fcimb.2017.00039&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR53"><span class="element-citation">Mosquera-Rend&#x000f3;n J, et al. Pangenome-wide and molecular evolution analyses of the <em>Pseudomonas aeruginosa</em> species. <span><span class="ref-journal">BMC Genomics. </span>2016;<span class="ref-vol">17</span>(1):45. doi: 10.1186/s12864-016-2364-4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4710005/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26754847" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2Fs12864-016-2364-4" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genomics&amp;title=Pangenome-wide+and+molecular+evolution+analyses+of+the+Pseudomonas+aeruginosa+species&amp;author=J+Mosquera-Rend&#x000f3;n&amp;volume=17&amp;issue=1&amp;publication_year=2016&amp;pages=45&amp;pmid=26754847&amp;doi=10.1186/s12864-016-2364-4&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR54"><span class="element-citation">Ni Z, et al. Antibiotic resistance determinant-focused <em>Acinetobacter baumannii</em> vaccine designed using reverse vaccinology. <span><span class="ref-journal">Int J Mol Sci. </span>2017;<span class="ref-vol">18</span>(2):458. doi: 10.3390/ijms18020458.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5343991/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28230771" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fijms18020458" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Mol+Sci&amp;title=Antibiotic+resistance+determinant-focused+Acinetobacter+baumannii+vaccine+designed+using+reverse+vaccinology&amp;author=Z+Ni&amp;volume=18&amp;issue=2&amp;publication_year=2017&amp;pages=458&amp;doi=10.3390/ijms18020458&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR55"><span class="element-citation">Pang Z, et al. Antibiotic resistance in <em>Pseudomonas aeruginosa</em>: mechanisms and alternative therapeutic strategies. <span><span class="ref-journal">Biotechnol Adv. </span>2019;<span class="ref-vol">37</span>(1):177–192. doi: 10.1016/j.biotechadv.2018.11.013.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30500353" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.biotechadv.2018.11.013" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnol+Adv&amp;title=Antibiotic+resistance+in+Pseudomonas+aeruginosa:+mechanisms+and+alternative+therapeutic+strategies&amp;author=Z+Pang&amp;volume=37&amp;issue=1&amp;publication_year=2019&amp;pages=177-192&amp;pmid=30500353&amp;doi=10.1016/j.biotechadv.2018.11.013&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR56"><span class="element-citation">Paul S, et al. TepiTool: a pipeline for computational prediction of T cell epitope candidates. <span><span class="ref-journal">Curr Protoc Immunol. </span>2016;<span class="ref-vol">114</span>:18 19 1–18 19 24. doi: 10.1002/cpim.12.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4981331/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27479659" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fcpim.12" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Protoc+Immunol&amp;title=TepiTool:+a+pipeline+for+computational+prediction+of+T+cell+epitope+candidates&amp;author=S+Paul&amp;volume=114&amp;publication_year=2016&amp;pages=18+19+1-18+19+24&amp;pmid=27479659&amp;doi=10.1002/cpim.12&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR57"><span class="element-citation">Pennington JE, et al. Use of a <em>Pseudomonas aeruginosa</em> vaccine in patients with acute leukemia and cystic fibrosis. <span><span class="ref-journal">Am J Med. </span>1975;<span class="ref-vol">58</span>(5):629–636. doi: 10.1016/0002-9343(75)90498-2.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/805535" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2F0002-9343(75)90498-2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med&amp;title=Use+of+a+Pseudomonas+aeruginosa+vaccine+in+patients+with+acute+leukemia+and+cystic+fibrosis&amp;author=JE+Pennington&amp;volume=58&amp;issue=5&amp;publication_year=1975&amp;pages=629-636&amp;pmid=805535&amp;doi=10.1016/0002-9343(75)90498-2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR58"><span class="element-citation">Perumal D, et al. Differential genome analyses of metabolic enzymes in <em>Pseudomonas aeruginosa</em> for drug target identification. <span><span class="ref-journal">In Silico Biol. </span>2007;<span class="ref-vol">7</span>:453–465.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18391237" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=In+Silico+Biol&amp;title=Differential+genome+analyses+of+metabolic+enzymes+in+Pseudomonas+aeruginosa+for+drug+target+identification&amp;author=D+Perumal&amp;volume=7&amp;publication_year=2007&amp;pages=453-465&amp;pmid=18391237&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR59"><span class="element-citation">Perumal D, et al. Targeting multiple targets in <em>Pseudomonas aeruginosa</em> PAO1 using flux balance analysis of a reconstructed genome-scale metabolic network. <span><span class="ref-journal">J Drug Target. </span>2011;<span class="ref-vol">19</span>(1):1–13. doi: 10.3109/10611861003649753.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20233082" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3109%2F10611861003649753" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Drug+Target&amp;title=Targeting+multiple+targets+in+Pseudomonas+aeruginosa+PAO1+using+flux+balance+analysis+of+a+reconstructed+genome-scale+metabolic+network&amp;author=D+Perumal&amp;volume=19&amp;issue=1&amp;publication_year=2011&amp;pages=1-13&amp;pmid=20233082&amp;doi=10.3109/10611861003649753&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR60"><span class="mixed-citation">Pires DPP, et al (2018) Bacteriophage interaction with <em>P. aeruginosa</em> and <em>A. baumannii</em> biofilms and colonizing human airway epithelium<em>.</em>10.1186/s12941-020-00389-5</span></li><li id="CR61"><span class="element-citation">Priebe GP, Goldberg JB. Vaccines for <em>Pseudomonas aeruginosa</em>: a long and winding road. <span><span class="ref-journal">Expert Rev Vaccines. </span>2014;<span class="ref-vol">13</span>(4):507–519. doi: 10.1586/14760584.2014.890053.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4521563/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24575895" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1586%2F14760584.2014.890053" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Expert+Rev+Vaccines&amp;title=Vaccines+for+Pseudomonas+aeruginosa:+a+long+and+winding+road&amp;author=GP+Priebe&amp;author=JB+Goldberg&amp;volume=13&amp;issue=4&amp;publication_year=2014&amp;pages=507-519&amp;pmid=24575895&amp;doi=10.1586/14760584.2014.890053&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR62"><span class="element-citation">Priebe GP, et al. Protection against fatal <em>Pseudomonas aeruginosa</em> pneumonia in mice after nasal immunization with a live, attenuated aroA deletion mutant. <span><span class="ref-journal">Infect Immun. </span>2003;<span class="ref-vol">71</span>(3):1453–1461. doi: 10.1128/IAI.71.3.1453-1461.2003.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC148856/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12595463" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FIAI.71.3.1453-1461.2003" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Infect+Immun&amp;title=Protection+against+fatal+Pseudomonas+aeruginosa+pneumonia+in+mice+after+nasal+immunization+with+a+live,+attenuated+aroA+deletion+mutant&amp;author=GP+Priebe&amp;volume=71&amp;issue=3&amp;publication_year=2003&amp;pages=1453-1461&amp;pmid=12595463&amp;doi=10.1128/IAI.71.3.1453-1461.2003&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR63"><span class="element-citation">Qiao M, et al. Review of antibiotic resistance in China and its environment. <span><span class="ref-journal">Environ Int. </span>2018;<span class="ref-vol">110</span>:160–172. doi: 10.1016/j.envint.2017.10.016.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29107352" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.envint.2017.10.016" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Environ+Int&amp;title=Review+of+antibiotic+resistance+in+China+and+its+environment&amp;author=M+Qiao&amp;volume=110&amp;publication_year=2018&amp;pages=160-172&amp;pmid=29107352&amp;doi=10.1016/j.envint.2017.10.016&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR64"><span class="element-citation">Ranganathan S, Garg G. Secretome: clues into pathogen infection and clinical applications. <span><span class="ref-journal">Genome Med. </span>2009;<span class="ref-vol">1</span>(11):113. doi: 10.1186/gm113.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2808748/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19951402" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2Fgm113" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Med&amp;title=Secretome:+clues+into+pathogen+infection+and+clinical+applications&amp;author=S+Ranganathan&amp;author=G+Garg&amp;volume=1&amp;issue=11&amp;publication_year=2009&amp;pages=113&amp;pmid=19951402&amp;doi=10.1186/gm113&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR65"><span class="element-citation">Rappuoli R, Bloom DE, Black S. Deploy vaccines to fight superbugs. <span><span class="ref-journal">Nature. </span>2017 doi: 10.1038/d41586-017-08323-0.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29239371" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fd41586-017-08323-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Deploy+vaccines+to+fight+superbugs&amp;author=R+Rappuoli&amp;author=DE+Bloom&amp;author=S+Black&amp;publication_year=2017&amp;pmid=29239371&amp;doi=10.1038/d41586-017-08323-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR66"><span class="element-citation">Rashid MI. Study of pyocyanin induced virulence &#x00026; pathogenicity of nosocomial infectious agent <em>Pseudomonas aeruginosa</em> and its prevention. <span><span class="ref-journal">Pak J Pharmaceu Sci. </span>2020;<span class="ref-vol">33</span>(3):915–922.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33191213" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pak+J+Pharmaceu+Sci&amp;title=Study+of+pyocyanin+induced+virulence+&#x00026;+pathogenicity+of+nosocomial+infectious+agent+Pseudomonas+aeruginosa+and+its+prevention&amp;author=MI+Rashid&amp;volume=33&amp;issue=3&amp;publication_year=2020&amp;pages=915-922&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR67"><span class="element-citation">Rashid MI, et al. Prediction of vaccine candidates against <em>Pseudomonas aeruginosa</em>: an integrated genomics and proteomics approach. <span><span class="ref-journal">Genomics. </span>2017;<span class="ref-vol">109</span>(3):274–283. doi: 10.1016/j.ygeno.2017.05.001.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28487172" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.ygeno.2017.05.001" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genomics&amp;title=Prediction+of+vaccine+candidates+against+Pseudomonas+aeruginosa:+an+integrated+genomics+and+proteomics+approach&amp;author=MI+Rashid&amp;volume=109&amp;issue=3&amp;publication_year=2017&amp;pages=274-283&amp;pmid=28487172&amp;doi=10.1016/j.ygeno.2017.05.001&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR68"><span class="element-citation">Rizvi M, et al. Synergy of drug combinations in treating multidrug-resistant <em>Pseudomonas aeruginosa</em>. <span><span class="ref-journal">Australas Med J. </span>2015;<span class="ref-vol">8</span>(1):1. doi: 10.4066/AMJ.2015.2096.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4321197/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25848402" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.4066%2FAMJ.2015.2096" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Australas+Med+J&amp;title=Synergy+of+drug+combinations+in+treating+multidrug-resistant+Pseudomonas+aeruginosa&amp;author=M+Rizvi&amp;volume=8&amp;issue=1&amp;publication_year=2015&amp;pages=1&amp;pmid=25848402&amp;doi=10.4066/AMJ.2015.2096&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR69"><span class="element-citation">Roses AD. Pharmacogenetics and future drug development and delivery. <span><span class="ref-journal">Lancet. </span>2000;<span class="ref-vol">355</span>(9212):1358–1361. doi: 10.1016/S0140-6736(00)02126-7.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10776762" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FS0140-6736(00)02126-7" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Pharmacogenetics+and+future+drug+development+and+delivery&amp;author=AD+Roses&amp;volume=355&amp;issue=9212&amp;publication_year=2000&amp;pages=1358-1361&amp;pmid=10776762&amp;doi=10.1016/S0140-6736(00)02126-7&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR70"><span class="element-citation">Sachdeva G, et al. SPAAN: a software program for prediction of adhesins and adhesin-like proteins using neural networks. <span><span class="ref-journal">Bioinformatics. </span>2004;<span class="ref-vol">21</span>(4):483–491. doi: 10.1093/bioinformatics/bti028.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7109999/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15374866" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbti028" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=SPAAN:+a+software+program+for+prediction+of+adhesins+and+adhesin-like+proteins+using+neural+networks&amp;author=G+Sachdeva&amp;volume=21&amp;issue=4&amp;publication_year=2004&amp;pages=483-491&amp;pmid=15374866&amp;doi=10.1093/bioinformatics/bti028&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR71"><span class="element-citation">Saha S, Raghava GP. AlgPred: prediction of allergenic proteins and mapping of IgE epitopes. <span><span class="ref-journal">Nucleic Acids Res. </span>2006;<span class="ref-vol">34</span>:W202–W209. doi: 10.1093/nar/gkl343.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1538830/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16844994" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkl343" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=AlgPred:+prediction+of+allergenic+proteins+and+mapping+of+IgE+epitopes&amp;author=S+Saha&amp;author=GP+Raghava&amp;volume=34&amp;publication_year=2006&amp;pages=W202-W209&amp;pmid=16844994&amp;doi=10.1093/nar/gkl343&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR72"><span class="element-citation">Sakharkar KR, Sakharkar MK, Chow VT. A novel genomics approach for the identification of drug targets in pathogens, with special reference to <em>Pseudomonas aeruginosa</em>. <span><span class="ref-journal">Silico Biology. </span>2004;<span class="ref-vol">4</span>(3):355–360.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15724285" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Silico+Biology&amp;title=A+novel+genomics+approach+for+the+identification+of+drug+targets+in+pathogens,+with+special+reference+to+Pseudomonas+aeruginosa&amp;author=KR+Sakharkar&amp;author=MK+Sakharkar&amp;author=VT+Chow&amp;volume=4&amp;issue=3&amp;publication_year=2004&amp;pages=355-360&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR73"><span class="element-citation">Sette A, Rappuoli R. Reverse vaccinology: developing vaccines in the era of genomics. <span><span class="ref-journal">Immunity. </span>2010;<span class="ref-vol">33</span>(4):530–541. doi: 10.1016/j.immuni.2010.09.017.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3320742/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21029963" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.immuni.2010.09.017" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Reverse+vaccinology:+developing+vaccines+in+the+era+of+genomics&amp;author=A+Sette&amp;author=R+Rappuoli&amp;volume=33&amp;issue=4&amp;publication_year=2010&amp;pages=530-541&amp;pmid=21029963&amp;doi=10.1016/j.immuni.2010.09.017&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR74"><span class="element-citation">Smialowski P, et al. PROSO II&#x02013;a new method for protein solubility prediction. <span><span class="ref-journal">FEBS J. </span>2012;<span class="ref-vol">279</span>(12):2192–2200. doi: 10.1111/j.1742-4658.2012.08603.x.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22536855" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fj.1742-4658.2012.08603.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEBS+J&amp;title=PROSO+II&#x02013;a+new+method+for+protein+solubility+prediction&amp;author=P+Smialowski&amp;volume=279&amp;issue=12&amp;publication_year=2012&amp;pages=2192-2200&amp;pmid=22536855&amp;doi=10.1111/j.1742-4658.2012.08603.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR75"><span class="element-citation">Sorichter S, et al. Immune responses in the airways by nasal vaccination with systemic boosting against <em>Pseudomonas aeruginosa</em> in chronic lung disease. <span><span class="ref-journal">Vaccine. </span>2009;<span class="ref-vol">27</span>(21):2755–2759. doi: 10.1016/j.vaccine.2009.03.010.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19366571" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.vaccine.2009.03.010" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Vaccine&amp;title=Immune+responses+in+the+airways+by+nasal+vaccination+with+systemic+boosting+against+Pseudomonas+aeruginosa+in+chronic+lung+disease&amp;author=S+Sorichter&amp;volume=27&amp;issue=21&amp;publication_year=2009&amp;pages=2755-2759&amp;pmid=19366571&amp;doi=10.1016/j.vaccine.2009.03.010&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR76"><span class="element-citation">Szklarczyk D, et al. The STRING database in 2011: functional interaction networks of proteins, globally integrated and scored. <span><span class="ref-journal">Nucleic Acids Res. </span>2011;<span class="ref-vol">39</span>(suppl 1):D561–D568. doi: 10.1093/nar/gkq973.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3013807/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21045058" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkq973" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=The+STRING+database+in+2011:+functional+interaction+networks+of+proteins,+globally+integrated+and+scored&amp;author=D+Szklarczyk&amp;volume=39&amp;issue=suppl+1&amp;publication_year=2011&amp;pages=D561-D568&amp;pmid=21045058&amp;doi=10.1093/nar/gkq973&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR77"><span class="element-citation">Tusnady GE, Simon I. The HMMTOP transmembrane topology prediction server. <span><span class="ref-journal">Bioinformatics. </span>2001;<span class="ref-vol">17</span>(9):849–850. doi: 10.1093/bioinformatics/17.9.849.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11590105" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2F17.9.849" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=The+HMMTOP+transmembrane+topology+prediction+server&amp;author=GE+Tusnady&amp;author=I+Simon&amp;volume=17&amp;issue=9&amp;publication_year=2001&amp;pages=849-850&amp;pmid=11590105&amp;doi=10.1093/bioinformatics/17.9.849&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR78"><span class="element-citation">Vincent J-L. Vaccine development and passive immunization for <em>Pseudomonas aeruginosa</em> in critically ill patients: a clinical update. <span><span class="ref-journal">Future Microbiol. </span>2014;<span class="ref-vol">9</span>(4):457–463. doi: 10.2217/fmb.14.10.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24810345" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.2217%2Ffmb.14.10" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Future+Microbiol&amp;title=Vaccine+development+and+passive+immunization+for+Pseudomonas+aeruginosa+in+critically+ill+patients:+a+clinical+update&amp;author=J-L+Vincent&amp;volume=9&amp;issue=4&amp;publication_year=2014&amp;pages=457-463&amp;pmid=24810345&amp;doi=10.2217/fmb.14.10&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR79"><span class="element-citation">Volkamer A, et al. DoGSiteScorer: a web server for automatic binding site prediction, analysis and druggability assessment. <span><span class="ref-journal">Bioinformatics. </span>2012;<span class="ref-vol">28</span>(15):2074–2075. doi: 10.1093/bioinformatics/bts310.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22628523" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbts310" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=DoGSiteScorer:+a+web+server+for+automatic+binding+site+prediction,+analysis+and+druggability+assessment&amp;author=A+Volkamer&amp;volume=28&amp;issue=15&amp;publication_year=2012&amp;pages=2074-2075&amp;pmid=22628523&amp;doi=10.1093/bioinformatics/bts310&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR80"><span class="element-citation">Wang K, et al. The rapid in vivo evolution of <em>Pseudomonas aeruginosa</em> in ventilator-associated pneumonia patients leads to attenuated virulence. <span><span class="ref-journal">Open Biol. </span>2017;<span class="ref-vol">7</span>(9):170029. doi: 10.1098/rsob.170029.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5627047/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28878043" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1098%2Frsob.170029" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Open+Biol&amp;title=The+rapid+in+vivo+evolution+of+Pseudomonas+aeruginosa+in+ventilator-associated+pneumonia+patients+leads+to+attenuated+virulence&amp;author=K+Wang&amp;volume=7&amp;issue=9&amp;publication_year=2017&amp;pages=170029&amp;pmid=28878043&amp;doi=10.1098/rsob.170029&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR27"><span class="mixed-citation">WHO (2021) Global priority list of antibiotic research bacteria to guide research, discovery, and development of new antibiotics. World Health Organization. <a href="https://www.combatamr.org.au/news-events/who-global-priority-pathogens-list-of-antibiotic-resistant-bacteria" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">https://www.combatamr.org.au/news-events/who-global-priority-pathogens-list-of-antibiotic-resistant-bacteria</a></span></li><li id="CR81"><span class="element-citation">Weigt S, et al. Gene expression profiling of bronchioalveolar lavage cells during aspergillus colonization of the lung allograft. <span><span class="ref-journal">Transpl Clin Stud. </span>2018;<span class="ref-vol">102</span>(6):986–993. doi: 10.1097/TP.0000000000002058.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5962368/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29256975" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1097%2FTP.0000000000002058" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transpl+Clin+Stud&amp;title=Gene+expression+profiling+of+bronchioalveolar+lavage+cells+during+aspergillus+colonization+of+the+lung+allograft&amp;author=S+Weigt&amp;volume=102&amp;issue=6&amp;publication_year=2018&amp;pages=986-993&amp;doi=10.1097/TP.0000000000002058&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR82"><span class="element-citation">Westritschnig K, et al. A randomized, placebo-controlled phase I study assessing the safety and immunogenicity of a <em>Pseudomonas aeruginosa</em> hybrid outer membrane protein OprF/I vaccine (IC43) in healthy volunteers. <span><span class="ref-journal">Hum Vaccin Immunother. </span>2014;<span class="ref-vol">10</span>(1):170–183. doi: 10.4161/hv.26565.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4181012/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24064511" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.4161%2Fhv.26565" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Vaccin+Immunother&amp;title=A+randomized,+placebo-controlled+phase+I+study+assessing+the+safety+and+immunogenicity+of+a+Pseudomonas+aeruginosa+hybrid+outer+membrane+protein+OprF/I+vaccine+(IC43)+in+healthy+volunteers&amp;author=K+Westritschnig&amp;volume=10&amp;issue=1&amp;publication_year=2014&amp;pages=170-183&amp;pmid=24064511&amp;doi=10.4161/hv.26565&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR83"><span class="element-citation">Winsor GL, et al. Pseudomonas Genome Database: improved comparative analysis and population genomics capability for <em>Pseudomonas genomes</em>. <span><span class="ref-journal">Nucleic Acids Res. </span>2011;<span class="ref-vol">39</span>:D596–D600. doi: 10.1093/nar/gkq869.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3013766/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20929876" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkq869" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Pseudomonas+Genome+Database:+improved+comparative+analysis+and+population+genomics+capability+for+Pseudomonas+genomes&amp;author=GL+Winsor&amp;volume=39&amp;publication_year=2011&amp;pages=D596-D600&amp;pmid=20929876&amp;doi=10.1093/nar/gkq869&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR84"><span class="element-citation">Xiang Z, He Y. Vaxign: a web-based vaccine target design program for reverse vaccinology. <span><span class="ref-journal">Proc Vaccinol. </span>2009;<span class="ref-vol">1</span>(1):23–29. doi: 10.1016/j.provac.2009.07.005.</span> [<a href="//doi.org/10.1016%2Fj.provac.2009.07.005" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Vaccinol&amp;title=Vaxign:+a+web-based+vaccine+target+design+program+for+reverse+vaccinology&amp;author=Z+Xiang&amp;author=Y+He&amp;volume=1&amp;issue=1&amp;publication_year=2009&amp;pages=23-29&amp;doi=10.1016/j.provac.2009.07.005&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR85"><span class="element-citation">Xipell M, et al. Successful treatment of three severe MDR or XDR <em>Pseudomonas aeruginosa</em> infections with ceftolozane/tazobactam. <span><span class="ref-journal">Future Microbiol. </span>2017;<span class="ref-vol">12</span>(14):1323–1326. doi: 10.2217/fmb-2017-0018.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28980834" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.2217%2Ffmb-2017-0018" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Future+Microbiol&amp;title=Successful+treatment+of+three+severe+MDR+or+XDR+Pseudomonas+aeruginosa+infections+with+ceftolozane/tazobactam&amp;author=M+Xipell&amp;volume=12&amp;issue=14&amp;publication_year=2017&amp;pages=1323-1326&amp;pmid=28980834&amp;doi=10.2217/fmb-2017-0018&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR86"><span class="element-citation">Yang J, Zhang Y. I-TASSER server: new development for protein structure and function predictions. <span><span class="ref-journal">Nucleic Acids Res. </span>2015;<span class="ref-vol">43</span>(W1):W174–W181. doi: 10.1093/nar/gkv342.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4489253/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25883148" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkv342" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=I-TASSER+server:+new+development+for+protein+structure+and+function+predictions&amp;author=J+Yang&amp;author=Y+Zhang&amp;volume=43&amp;issue=W1&amp;publication_year=2015&amp;pages=W174-W181&amp;pmid=25883148&amp;doi=10.1093/nar/gkv342&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR87"><span class="element-citation">Yang F, et al. Protective efficacy of the trivalent <em>Pseudomonas aeruginosa</em> vaccine candidate PcrV-OprI-Hcp1 in murine pneumonia and burn models. <span><span class="ref-journal">Sci Rep. </span>2017;<span class="ref-vol">7</span>(1):3957. doi: 10.1038/s41598-017-04029-5.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5479855/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28638106" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41598-017-04029-5" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci+Rep&amp;title=Protective+efficacy+of+the+trivalent+Pseudomonas+aeruginosa+vaccine+candidate+PcrV-OprI-Hcp1+in+murine+pneumonia+and+burn+models&amp;author=F+Yang&amp;volume=7&amp;issue=1&amp;publication_year=2017&amp;pages=3957&amp;pmid=28638106&amp;doi=10.1038/s41598-017-04029-5&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="CR88"><span class="element-citation">Yu CS, Lin CJ, Hwang JK. Predicting subcellular localization of proteins for Gram-negative bacteria by support vector machines based on n-peptide compositions. <span><span class="ref-journal">Protein Sci. </span>2004;<span class="ref-vol">13</span>(5):1402–1406. doi: 10.1110/ps.03479604.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2286765/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15096640" ref="reftype=pubmed&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1110%2Fps.03479604" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Sci&amp;title=Predicting+subcellular+localization+of+proteins+for+Gram-negative+bacteria+by+support+vector+machines+based+on+n-peptide+compositions&amp;author=CS+Yu&amp;author=CJ+Lin&amp;author=JK+Hwang&amp;volume=13&amp;issue=5&amp;publication_year=2004&amp;pages=1402-1406&amp;pmid=15096640&amp;doi=10.1110/ps.03479604&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8351238&amp;issue-id=387595&amp;journal-id=3814&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li></ul></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">International Journal of Peptide Research and Therapeutics</span> are provided here courtesy of <strong>Nature Publishing Group</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC8351238/pdf/10989_2021_Article_10255.pdf" class="int-view">PDF (2.7M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/8351238/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/8351238/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="8351238" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC8351238%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="VbZLQsINPHKT9l1LQrp7qBO29gwHEhRkDQIFKaa1YnCLZTfwJ96DmOTX6sYZ6rfh">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC8351238%2F&amp;text=Quest%20for%20Novel%20Preventive%20and%20Therapeutic%20Options%20Against%20Multidrug-Resistant%20Pseudomonas%20aeruginosa" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC8351238%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8351238/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC8351238/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/34393689/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC8351238/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/34393689/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC8351238/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/8351238/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC8351238/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
