
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Drug Discovery Targeting the Disorder-To-Order Transition Regions through the Conformational Diversity Mimicking and Statistical Analysis - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="D0BD41E4AC192A45000035940452B817.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="ijms" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC7432763/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7432763/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="International Journal of Molecular Sciences" /><meta name="citation_title" content="Drug Discovery Targeting the Disorder-To-Order Transition Regions through the Conformational Diversity Mimicking and Statistical Analysis" /><meta name="citation_author" content="Insung Na" /><meta name="citation_author_institution" content="Department of Life Science and Research Institute for Natural Sciences, Hanyang University, Seoul 04763, Korea; rk.ca.gnaynah@gnusnian (I.N.); moc.revan@aywsc (S.C.); moc.revan@21kognasmi (S.H.S.)" /><meta name="citation_author" content="Sungwoo Choi" /><meta name="citation_author_institution" content="Department of Life Science and Research Institute for Natural Sciences, Hanyang University, Seoul 04763, Korea; rk.ca.gnaynah@gnusnian (I.N.); moc.revan@aywsc (S.C.); moc.revan@21kognasmi (S.H.S.)" /><meta name="citation_author" content="Seung Han Son" /><meta name="citation_author_institution" content="Department of Life Science and Research Institute for Natural Sciences, Hanyang University, Seoul 04763, Korea; rk.ca.gnaynah@gnusnian (I.N.); moc.revan@aywsc (S.C.); moc.revan@21kognasmi (S.H.S.)" /><meta name="citation_author" content="Vladimir N. Uversky" /><meta name="citation_author_institution" content="Department of Molecular Medicine, Morsani College of Medicine, University of South Florida, Tampa, FL 33612, USA" /><meta name="citation_author_institution" content="Institute for Biological Instrumentation of the Russian Academy of Sciences, 142290 Pushchino, Russia" /><meta name="citation_author" content="Chul Geun Kim" /><meta name="citation_author_institution" content="Department of Life Science and Research Institute for Natural Sciences, Hanyang University, Seoul 04763, Korea; rk.ca.gnaynah@gnusnian (I.N.); moc.revan@aywsc (S.C.); moc.revan@21kognasmi (S.H.S.)" /><meta name="citation_author_institution" content="CGK Biopharma Co. Ltd., 222 Wangshipri-ro, Sungdong-gu, Seoul 04763, Korea" /><meta name="citation_publication_date" content="2020/08" /><meta name="citation_issue" content="15" /><meta name="citation_volume" content="21" /><meta name="citation_doi" content="10.3390/ijms21155248" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC7432763/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC7432763/" /><meta name="citation_pmid" content="32722024" /><meta name="DC.Title" content="Drug Discovery Targeting the Disorder-To-Order Transition Regions through the Conformational Diversity Mimicking and Statistical Analysis" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Multidisciplinary Digital Publishing Institute  (MDPI)" /><meta name="DC.Contributor" content="Insung Na" /><meta name="DC.Contributor" content="Sungwoo Choi" /><meta name="DC.Contributor" content="Seung Han Son" /><meta name="DC.Contributor" content="Vladimir N. Uversky" /><meta name="DC.Contributor" content="Chul Geun Kim" /><meta name="DC.Date" content="2020 Aug" /><meta name="DC.Identifier" content="10.3390/ijms21155248" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Drug Discovery Targeting the Disorder-To-Order Transition Regions through the Conformational Diversity Mimicking and Statistical Analysis" /><meta property="og:type" content="article" /><meta property="og:description" content="Intrinsically disordered proteins exist as highly dynamic conformational ensembles of diverse forms. However, the majority of virtual screening only focuses on proteins with defined structures. This means that computer-aided drug discovery is restricted. ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7432763/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC7432763/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22Int%20J%20Mol%20Sci%22[journal]" class="navlink">Int J Mol Sci</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC7432763
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC7432763/pdf/ijms-21-05248.pdf" class="int-view">PDF (5.3M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/7432763/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/7432763/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="7432763" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC7432763%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="mAqJqk1p3aIlKLk1W2AKVRv1Gurt6d8hsaShjunWFQ0isOvhFzaz5bXTCR3WK01B">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC7432763%2F&amp;text=Drug%20Discovery%20Targeting%20the%20Disorder-To-Order%20Transition%20Regions%20through%20the%20Conformational%20Diversity%20Mimicking%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC7432763%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7432763/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC7432763/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/32722024/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC7432763/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/32722024/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC7432763/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/7432763/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22Int%20J%20Mol%20Sci%22[journal]" class="navlink">Int J Mol Sci</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC7432763
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-ijms.png" alt="Logo of ijms" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.mdpi.com/journal/ijms/" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Int J Mol Sci.</span> 2020 Aug; 21(15): 5248. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2020 Jul 24. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.3390%2Fijms21155248" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.3390/ijms21155248</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC7432763</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/32722024">32722024</a></div></div></div><h1 class="content-title">Drug Discovery Targeting the Disorder-To-Order Transition Regions through the Conformational Diversity Mimicking and Statistical Analysis</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Na%20I%5BAuthor%5D" class="affpopup" co-rid="_co_idm140371325819952" co-class="co-affbox">Insung Na</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Choi%20S%5BAuthor%5D" class="affpopup" co-rid="_co_idm140371336946544" co-class="co-affbox">Sungwoo Choi</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Son%20SH%5BAuthor%5D" class="affpopup" co-rid="_co_idm140371373968016" co-class="co-affbox">Seung Han Son</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Uversky%20VN%5BAuthor%5D" class="affpopup" co-rid="_co_idm140371373966144" co-class="co-affbox">Vladimir N. Uversky</a>,<sup>2,</sup><sup>3,</sup><sup>*</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kim%20CG%5BAuthor%5D" class="affpopup" co-rid="_co_idm140371378017920" co-class="co-affbox">Chul Geun Kim</a><sup>1,</sup><sup>4,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140371325819952"><h3 class="no_margin">Insung Na</h3><p><sup>1</sup>Department of Life Science and Research Institute for Natural Sciences, Hanyang University, Seoul 04763, Korea; <a href="mailto:dev@null" data-email="rk.ca.gnaynah@gnusnian" class="oemail">rk.ca.gnaynah@gnusnian</a> (I.N.); <a href="mailto:dev@null" data-email="moc.revan@aywsc" class="oemail">moc.revan@aywsc</a> (S.C.); <a href="mailto:dev@null" data-email="moc.revan@21kognasmi" class="oemail">moc.revan@21kognasmi</a> (S.H.S.)</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Na%20I%5BAuthor%5D">Insung Na</a></div></div><div id="_co_idm140371336946544"><h3 class="no_margin">Sungwoo Choi</h3><p><sup>1</sup>Department of Life Science and Research Institute for Natural Sciences, Hanyang University, Seoul 04763, Korea; <a href="mailto:dev@null" data-email="rk.ca.gnaynah@gnusnian" class="oemail">rk.ca.gnaynah@gnusnian</a> (I.N.); <a href="mailto:dev@null" data-email="moc.revan@aywsc" class="oemail">moc.revan@aywsc</a> (S.C.); <a href="mailto:dev@null" data-email="moc.revan@21kognasmi" class="oemail">moc.revan@21kognasmi</a> (S.H.S.)</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Choi%20S%5BAuthor%5D">Sungwoo Choi</a></div></div><div id="_co_idm140371373968016"><h3 class="no_margin">Seung Han Son</h3><p><sup>1</sup>Department of Life Science and Research Institute for Natural Sciences, Hanyang University, Seoul 04763, Korea; <a href="mailto:dev@null" data-email="rk.ca.gnaynah@gnusnian" class="oemail">rk.ca.gnaynah@gnusnian</a> (I.N.); <a href="mailto:dev@null" data-email="moc.revan@aywsc" class="oemail">moc.revan@aywsc</a> (S.C.); <a href="mailto:dev@null" data-email="moc.revan@21kognasmi" class="oemail">moc.revan@21kognasmi</a> (S.H.S.)</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Son%20SH%5BAuthor%5D">Seung Han Son</a></div></div><div id="_co_idm140371373966144"><h3 class="no_margin">Vladimir N. Uversky</h3><p><sup>2</sup>Department of Molecular Medicine, Morsani College of Medicine, University of South Florida, Tampa, FL 33612, USA</p><p><sup>3</sup>Institute for Biological Instrumentation of the Russian Academy of Sciences, 142290 Pushchino, Russia</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Uversky%20VN%5BAuthor%5D">Vladimir N. Uversky</a></div></div><div id="_co_idm140371378017920"><h3 class="no_margin">Chul Geun Kim</h3><p><sup>1</sup>Department of Life Science and Research Institute for Natural Sciences, Hanyang University, Seoul 04763, Korea; <a href="mailto:dev@null" data-email="rk.ca.gnaynah@gnusnian" class="oemail">rk.ca.gnaynah@gnusnian</a> (I.N.); <a href="mailto:dev@null" data-email="moc.revan@aywsc" class="oemail">moc.revan@aywsc</a> (S.C.); <a href="mailto:dev@null" data-email="moc.revan@21kognasmi" class="oemail">moc.revan@21kognasmi</a> (S.H.S.)</p><p><sup>4</sup>CGK Biopharma Co. Ltd., 222 Wangshipri-ro, Sungdong-gu, Seoul 04763, Korea</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kim%20CG%5BAuthor%5D">Chul Geun Kim</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140371328799696_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140371328799696_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140371328799696_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140371328799696_ai" style="display:none"><div class="fm-affl" id="af1-ijms-21-05248"><sup>1</sup>Department of Life Science and Research Institute for Natural Sciences, Hanyang University, Seoul 04763, Korea; <a href="mailto:dev@null" data-email="rk.ca.gnaynah@gnusnian" class="oemail">rk.ca.gnaynah@gnusnian</a> (I.N.); <a href="mailto:dev@null" data-email="moc.revan@aywsc" class="oemail">moc.revan@aywsc</a> (S.C.); <a href="mailto:dev@null" data-email="moc.revan@21kognasmi" class="oemail">moc.revan@21kognasmi</a> (S.H.S.)</div><div class="fm-affl" id="af2-ijms-21-05248"><sup>2</sup>Department of Molecular Medicine, Morsani College of Medicine, University of South Florida, Tampa, FL 33612, USA</div><div class="fm-affl" id="af3-ijms-21-05248"><sup>3</sup>Institute for Biological Instrumentation of the Russian Academy of Sciences, 142290 Pushchino, Russia</div><div class="fm-affl" id="af4-ijms-21-05248"><sup>4</sup>CGK Biopharma Co. Ltd., 222 Wangshipri-ro, Sungdong-gu, Seoul 04763, Korea</div><div id="c1-ijms-21-05248"><sup>*</sup>Correspondence: <a href="mailto:dev@null" data-email="ude.fsu@yksrevuv" class="oemail">ude.fsu@yksrevuv</a> (V.N.U.); <a href="mailto:dev@null" data-email="rk.ca.gnaynah@mikgc" class="oemail">rk.ca.gnaynah@mikgc</a> (C.G.K.)</div></div><div class="fm-article-notes hide half_rhythm" id="idm140371328799696_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2020 Jun 9; Accepted 2020 Jul 21.</div></div><div class="permissions half_rhythm hide" id="idm140371328799696_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; 2020 by the authors.</div><div class="license half_rhythm">Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (<a href="http://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>).</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div><a href="/pmc/articles/PMC7432763/bin/ijms-21-05248-s001.pdf" data-ga-action="click_feat_suppl">ijms-21-05248-s001.pdf</a><span style="color:gray"> (1.5M)</span></div><div class="small guid">GUID: 2FB52539-3245-43EF-9B79-9FB9E6E16117</div></div></dd></dl></div><div id="abstract-a.l.b.r" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.l.b.rtitle">Abstract</h2><!--article-meta--><div><p class="p p-first-last">Intrinsically disordered proteins exist as highly dynamic conformational ensembles of diverse forms. However, the majority of virtual screening only focuses on proteins with defined structures. This means that computer-aided drug discovery is restricted. As a breakthrough, understanding the structural characteristics of intrinsically disordered proteins and its application can open the gate for unrestricted drug discovery. First, we segmented the target disorder-to-order transition region into a series of overlapping 20-amino-acid-long peptides. Folding prediction generated diverse conformations of these peptides. Next, we applied molecular docking, new evaluation score function, and statistical analysis. This approach successfully distinguished known compounds and their corresponding binding regions. Especially, Myc proto-oncogene protein (MYC) inhibitor 10058F4 was well distinguished from others of the chemical compound library. We also studied differences between the two Methyl-CpG-binding domain protein 2 (MBD2) inhibitors (ABA (2-amino-N-[[(3S)-2,3-dihydro-1,4-benzodioxin-3-yl]methyl]-acetamide) and APC ((R)-(3-(2-Amino-acetylamino)-pyrrolidine-1-carboxylic acid tert-butyl ester))). Both compounds bind MBD2 through electrostatic interaction behind its p66&#x003b1;-binding site. ABA is also able to bind p66&#x003b1; through electrostatic interaction behind its MBD2-binding site while APC-p66&#x003b1; binding was nonspecific. Therefore, structural heterogeneity mimicking of the disorder-to-order transition region at the peptide level and utilization of the new docking score function represent a useful approach that can efficiently discriminate compounds for expanded virtual screening toward intrinsically disordered proteins.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">disorder-to-order transition, computer-aided drug discovery, protein&#x02013;protein interaction inhibitor, conformational diversity mimicking, peptide docking</span></div></div><div id="sec1-ijms-21-05248" class="tsec sec"><h2 class="head no_bottom_margin" id="sec1-ijms-21-05248title">1. Introduction</h2><p class="p p-first">Currently, computer-aided drug discovery represents a usual initiation of the drug development pathway. Among the computer-aided drug discovery approaches, computational hit screening is known as &#x0201c;virtual screening&#x0201d;, which uses ligand- or structure-based strategies and which is complementary to the traditional &#x0201c;high-throughput screening&#x0201d; methods utilized in wet-lab settings. Molecular docking is one of the representative techniques for virtual screening, and machine learning and deep learning techniques have become powerful tools nowadays [<a href="#B1-ijms-21-05248" rid="B1-ijms-21-05248" class=" bibr popnode">1</a>]. Virtual screening depends on the knowledge of the structure of target proteins determined by X-ray crystallography, solution or solid-state NMR (Nuclear Magnetic Resonance), cryo-EM (Electron Microscopy), and sometimes homology modeling. This means that virtual screening is limited to the set of proteins with known structures, although disease does not always occur due to some problems pertaining to the ordered proteins. Furthermore, blocking such proteins is not always a perfect solution [<a href="#B2-ijms-21-05248" rid="B2-ijms-21-05248" class=" bibr popnode">2</a>,<a href="#B3-ijms-21-05248" rid="B3-ijms-21-05248" class=" bibr popnode">3</a>]. From this viewpoint, the major field of current computer-aided drug discovery process, virtual screening, is restricted to searches within a very limited space at the bottom of the funnel.</p><p>Intrinsically disordered proteins (IDPs) do not have defined structures; instead, they exist as highly dynamic conformational ensembles that rapidly change in response to subtle changes in the environment [<a href="#B4-ijms-21-05248" rid="B4-ijms-21-05248" class=" bibr popnode">4</a>,<a href="#B5-ijms-21-05248" rid="B5-ijms-21-05248" class=" bibr popnode">5</a>]. Not only can entire proteins be disordered but also many proteins represent hybrid entities containing ordered domains and intrinsically disordered protein regions (IDPRs) [<a href="#B6-ijms-21-05248" rid="B6-ijms-21-05248" class=" bibr popnode">6</a>]. Some IDPRs function as binding sites for their corresponding partners through disorder-to-order transition, i.e., they utilize the binding-induced folding mechanism [<a href="#B7-ijms-21-05248" rid="B7-ijms-21-05248" class=" bibr popnode">7</a>]. Such binding-induced disorder-to-order transition is characterized by structural transformation of a mostly unfolded unbound state into a specifically folded bound form. Such regions are commonly involved in molecular recognition, and therefore, they are known as molecular recognition features (MoRFs) [<a href="#B8-ijms-21-05248" rid="B8-ijms-21-05248" class=" bibr popnode">8</a>,<a href="#B9-ijms-21-05248" rid="B9-ijms-21-05248" class=" bibr popnode">9</a>]. Although such foldable intrinsically disordered recognition regions represent an attractive target for the discovery of protein&#x02013;protein interaction inhibitors, for a long time, there were no drugs targeting disorder-based binding sites and there were reliable approaches for the utilization of these specific characteristics (binding-induced folding) in the drug discovery. However, the potential of this shift from ordered to disordered binding sites for the selection of promising drug targets was recently demonstrated by the traditional high-throughput screening-based discovery of the Myc proto-oncogene protein (MYC) inhibitors, including 10058F4 [<a href="#B10-ijms-21-05248" rid="B10-ijms-21-05248" class=" bibr popnode">10</a>,<a href="#B11-ijms-21-05248" rid="B11-ijms-21-05248" class=" bibr popnode">11</a>]. Furthermore, we recently elaborated an approach for the rational discovery of the inhibitors targeting such disorder-to-order transition regions and used this approach to find inhibitors targeting Methyl-CpG-binding domain protein 2 (MBD2) [<a href="#B12-ijms-21-05248" rid="B12-ijms-21-05248" class=" bibr popnode">12</a>]. </p><p>Earlier, inhibitors targeting disorder-to-order transition region of MYC involved in interaction with Protein Max (MAX) to form a tweezer-like coiled-coil dimer interacting with DNA were discovered using the yeast two-hybrid experiments [<a href="#B10-ijms-21-05248" rid="B10-ijms-21-05248" class=" bibr popnode">10</a>]. Some compounds identified in this study were shown to possess antitumor efficacy <em>in vitro</em> and <em>in vivo</em> [<a href="#B10-ijms-21-05248" rid="B10-ijms-21-05248" class=" bibr popnode">10</a>]. Later, two members of the library of these compounds, 10058F4 and 10074G5, targeting the MYC binding region were studied by a set of biophysical approaches [<a href="#B11-ijms-21-05248" rid="B11-ijms-21-05248" class=" bibr popnode">11</a>]. The experiment employed mutagenesis and truncations of the MYC disorder-to-order transition region of MYC involved in interaction with MAX [<a href="#B13-ijms-21-05248" rid="B13-ijms-21-05248" class=" bibr popnode">13</a>] and determined that 10058F4 binds to the region 402&#x02013;412 of MYC whereas residues 363&#x02013;381 were described as the 10074G5-binding region [<a href="#B11-ijms-21-05248" rid="B11-ijms-21-05248" class=" bibr popnode">11</a>]. A recent comprehensive NMR analysis revealed the presence of subregions of a residual secondary structure within the MYC disorder-to-order transition region and structurally characterized regions affected by the 10058F4 binding [<a href="#B14-ijms-21-05248" rid="B14-ijms-21-05248" class=" bibr popnode">14</a>]. This analysis revealed that the MYC 402&#x02013;412 region serves as the 10058F4 target site and is embedded within the MYC disorder-to-order transition region responsible for MAX binding [<a href="#B14-ijms-21-05248" rid="B14-ijms-21-05248" class=" bibr popnode">14</a>]. Curiously, according to our previous study, in the MYC per-residue intrinsic disorder profile, this 402&#x02013;412 segment corresponds to a region with a high positive slope [<a href="#B12-ijms-21-05248" rid="B12-ijms-21-05248" class=" bibr popnode">12</a>].</p><p>We also reported the discovery of two inhibitors targeting the disorder-to-order transition regions of MBD2 involved in an interaction with p66&#x003b1; [<a href="#B12-ijms-21-05248" rid="B12-ijms-21-05248" class=" bibr popnode">12</a>]. We showed that the MBD2 segment that undergoes binding-induced disorder-to-order transition and forms coiled-coil dimer with its partner, p66&#x003b1; [<a href="#B15-ijms-21-05248" rid="B15-ijms-21-05248" class=" bibr popnode">15</a>] (i.e., acts similarly to the MYC interacting with MAX [<a href="#B13-ijms-21-05248" rid="B13-ijms-21-05248" class=" bibr popnode">13</a>]), is located within the region with a positive slope in the per-residue disorder profile of MBD2 [<a href="#B12-ijms-21-05248" rid="B12-ijms-21-05248" class=" bibr popnode">12</a>]. By molecular docking utilized for virtual screening toward the coiled-coil disorder-to-order transition region of MBD2, we obtained two positive hits, ABA (2-amino-N-[[(3S)-2,3-dihydro-1,4-benzodioxin-3-yl]methyl]-acetamide) and APC (R)-(3-(2-Amino-acetylamino)-pyrrolidine-1-carboxylic acid tert-butyl ester). These compounds were shown to possess noticeable anti-metastasis activity due to the inhibition of the MBD2&#x02013;p66&#x003b1; complex formation [<a href="#B12-ijms-21-05248" rid="B12-ijms-21-05248" class=" bibr popnode">12</a>]. However, we did not analyze whether these two compounds can distinguish MBD2 and p66&#x003b1; (which has a disordered region that undergoes disorder-to-order transition while forming the coiled-coil heterodimer with MBD2).</p><p class="p p-last">Here, we developed a new method for escaping the bottom of the funnel that limits the structure-defined protein virtual screening preferentially to ordered proteins. We overcame this limit through segmentation of the target disorder-to-order transition region to a set of overlapping peptides, analysis of the structural preferences of these peptides, and elaboration of a new function for evaluation of the molecular docking score. We used this approach to study the binding mechanism of the MYC inhibitor (10058F4) toward its binding site in MYC. Besides, we also proved that the MBD2 inhibitors (ABA and APC) have different binding preferences toward MBD2 and p66&#x003b1;. We think that our novel method represents an important breakthrough that will expand the computer-aided drug discovery field, making possible the identification of inhibitors for proteins with unknown structures.</p></div><div id="sec2-ijms-21-05248" class="tsec sec"><h2 class="head no_bottom_margin" id="sec2-ijms-21-05248title">2. Results and Discussion</h2><div id="sec2dot1-ijms-21-05248" class="sec sec-first"><h3 id="sec2dot1-ijms-21-05248title">2.1. Computational Reevaluation of the 10058F4 Binding to the MYC Disorder-To-Order Transition Region</h3><p class="p p-first">Earlier, we discovered the anti-metastasis activities of two compounds, ABA and APC (see the <a href="#sec3-ijms-21-05248" rid="sec3-ijms-21-05248" class=" sec">Section 3</a> Materials and Methods) by targeting MBD2 in molecular docking environment [<a href="#B12-ijms-21-05248" rid="B12-ijms-21-05248" class=" bibr popnode">12</a>]. In the field of rational drug design, it is a commonly accepted practice to apply molecular docking toward target protein structures determined by X-ray crystallography, NMR, and cryo-EM. However, application of this powerful approach to intrinsically disordered proteins (IDPs) or intrinsically disordered protein regions (IDPRs) is challenging. In our previous study, we used a modified rational drug design approach for the discovery of drug leads based on molecular docking and molecular dynamics (MD) simulations of the IDPRs of target proteins capable of the disorder-to-order transitions (DOTs) [<a href="#B12-ijms-21-05248" rid="B12-ijms-21-05248" class=" bibr popnode">12</a>]. The process started with the analysis of the intrinsic disorder predisposition of a drug target protein of interest followed by the prediction of the presence of potential disorder-based binding regions that can undergo DOTs. In parallel, a Protein Data Bank (PDB) search was conducted to find potential drug-target sites (DOT-based protein&#x02013;protein interaction regions), and the corresponding structures were retrieved and used for molecular docking. This was combined with the evaluation of the off-target probabilities for the selection of lead compounds from the molecular-docked hit compounds. Finally, MD simulations [<a href="#B12-ijms-21-05248" rid="B12-ijms-21-05248" class=" bibr popnode">12</a>] were used to evaluate the mode and efficiency of the prospected candidate compound binding. Although this approach targeted IDPRs (more precisely IDPRs capable of DOTs), by virtue of utilizing known structure of a target region in its bound state as a starting point for drug discovery, it used logistics of a conventional rational drug design.</p><p>In this study, we further developed this approach and moved it closer to the real-life settings by mimicking disorder-to-order transitions. Instead of considering a bound form of a long IDPR that underwent binding-induced folding and gained a specific structure, we focused on the intrinsic disorder nature of this region. To this end, we split each long IDPR of interest to a set of overlapping short peptides, used folding prediction algorithm to generate diverse conformational forms of these peptides, and then subjected these diverse conformations of model peptides to the molecular docking.</p><p>In more detail, the target regions used in this study were determined based on the structure for the MYC&#x02013;MAX (PDB ID 1NKP) [<a href="#B13-ijms-21-05248" rid="B13-ijms-21-05248" class=" bibr popnode">13</a>] complex by the X-ray crystallography. We considered the target DOT of MYC (residues 350&#x02013;439) and segmented it into the sets of the overlapping 20-amino-acid-long sequences (<a href="/pmc/articles/PMC7432763/table/ijms-21-05248-t001/" target="table" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-t001" rid-ob="ob-ijms-21-05248-t001" co-legend-rid=""><span>Table 1</span></a>). Next, each of the peptides was subjected to structure prediction using the PEPFOLD ver 3.5 web server (<a href="https://mobyle.rpbs.univ-paris-diderot.fr/cgi-bin/portal.py#forms::PEP-FOLD3">https://mobyle.rpbs.univ-paris-diderot.fr/cgi-bin/portal.py#forms::PEP-FOLD3</a>) to predict its potential structures [<a href="#B16-ijms-21-05248" rid="B16-ijms-21-05248" class=" bibr popnode">16</a>]. This stage generated a multitude of probable structures, which are exemplified by a set of conformations predicted for the 15 peptides overlapping derived from the MYC 350&#x02013;439 region. <a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f001/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f001" rid-ob="ob-ijms-21-05248-f001" co-legend-rid="lgnd_ijms-21-05248-f001"><span>Figure 1</span></a> clearly shows that different peptides possess very different folding potentials.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="ijms-21-05248-f001" co-legend-rid="lgnd_ijms-21-05248-f001"><a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f001/" target="figure" rid-figpopup="ijms-21-05248-f001" rid-ob="ob-ijms-21-05248-f001"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140371339117392"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7432763_ijms-21-05248-g001.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ijms-21-05248-g001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC7432763/bin/ijms-21-05248-g001.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140371339117392"><a target="object" rel="noopener" href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f001/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_ijms-21-05248-f001"><div><a class="figpopup" href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f001/" target="figure" rid-figpopup="ijms-21-05248-f001" rid-ob="ob-ijms-21-05248-f001">Figure 1</a></div><!--caption a7--><div class="caption"><p>Myc proto-oncogene protein (MYC) 350&#x02013;439 region segmented peptides: PEPFOLD ver 3.5 prediction result (see the <a href="#sec3-ijms-21-05248" rid="sec3-ijms-21-05248" class=" sec">Section 3</a> Materials and Methods) of peptide structures based on corresponding sequences (please look at the <a href="/pmc/articles/PMC7432763/table/ijms-21-05248-t001/" target="table" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-t001" rid-ob="ob-ijms-21-05248-t001" co-legend-rid=""><span>Table 1</span></a>). (<strong>A</strong>) MYC_350&#x02013;369 (10 models), (<strong>B</strong>) MYC_355&#x02013;374 (2 models), (<strong>C</strong>) MYC_360&#x02013;379 (2 models), (<strong>D</strong>) MYC_365&#x02013;384 (5 models), (<strong>E</strong>) MYC_370&#x02013;389 (27 models), (<strong>F</strong>) MYC_375&#x02013;394 (61 models), (<strong>G</strong>) MYC_380&#x02013;399 (56 models), (<strong>H</strong>) MYC_385&#x02013;404 (43 models), (<strong>I</strong>) MYC_390&#x02013;409 (22 models), (<strong>J</strong>) MYC_395&#x02013;414 (4 models), (<strong>K</strong>) MYC_400&#x02013;419 (28 models), (<strong>L</strong>) MYC_405&#x02013;424 (20 models), (<strong>M</strong>) MYC_410&#x02013;429 (6 models), (<strong>N</strong>) MYC_415&#x02013;434 (18 models), (<strong>O</strong>) MYC_420&#x02013;439 (2 models).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="ijms-21-05248-t001"><h3>Table 1</h3><!--caption a7--><div class="caption"><p>Protein target regions and segmented peptides.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Protein</th><th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Target Region</th><th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Peptides</th><th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Sequence (N&#x02019;&#x02013;C&#x02019;)</th></tr></thead><tbody><tr><td rowspan="15" align="center" valign="middle" style="border-bottom:solid thin" colspan="1">MYC</td><td rowspan="15" align="center" valign="middle" style="border-bottom:solid thin" colspan="1">350&#x02013;439</td><td align="center" valign="middle" rowspan="1" colspan="1">MYC_350&#x02013;369</td><td align="center" valign="middle" rowspan="1" colspan="1">TEENVKRRTHNVLERQRRNE</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">MYC_355&#x02013;374</td><td align="center" valign="middle" rowspan="1" colspan="1">KRRTHNVLERQRRNELKRSF</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">MYC_360&#x02013;379</td><td align="center" valign="middle" rowspan="1" colspan="1">NVLERQRRNELKRSFFALRD</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">MYC_365&#x02013;384</td><td align="center" valign="middle" rowspan="1" colspan="1">QRRNELKRSFFALRDQIPEL</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">MYC_370&#x02013;389</td><td align="center" valign="middle" rowspan="1" colspan="1">LKRSFFALRDQIPELENNEK</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">MYC_375&#x02013;394</td><td align="center" valign="middle" rowspan="1" colspan="1">FALRDQIPELENNEKAPKVV</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">MYC_380&#x02013;399</td><td align="center" valign="middle" rowspan="1" colspan="1">QIPELENNEKAPKVVILKKA</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">MYC_385&#x02013;404</td><td align="center" valign="middle" rowspan="1" colspan="1">ENNEKAPKVVILKKATAYIL</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">MYC_390&#x02013;409</td><td align="center" valign="middle" rowspan="1" colspan="1">APKVVILKKATAYILSVQAE</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">MYC_395&#x02013;414</td><td align="center" valign="middle" rowspan="1" colspan="1">ILKKATAYILSVQAEEQKLI</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">MYC_400&#x02013;419</td><td align="center" valign="middle" rowspan="1" colspan="1">TAYILSVQAEEQKLISEEDL</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">MYC_405&#x02013;424</td><td align="center" valign="middle" rowspan="1" colspan="1">SVQAEEQKLISEEDLLRKRR</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">MYC_410&#x02013;429</td><td align="center" valign="middle" rowspan="1" colspan="1">EQKLISEEDLLRKRREQLKH</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">MYC_415&#x02013;434</td><td align="center" valign="middle" rowspan="1" colspan="1">SEEDLLRKRREQLKHKLEQL</td></tr><tr><td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">MYC_420&#x02013;439</td><td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">LRKRREQLKHKLEQLRNSCA</td></tr><tr><td rowspan="4" align="center" valign="middle" style="border-bottom:solid thin" colspan="1">MBD2</td><td rowspan="4" align="center" valign="middle" style="border-bottom:solid thin" colspan="1">359&#x02013;393</td><td align="center" valign="middle" rowspan="1" colspan="1">MBD2_359&#x02013;378</td><td align="center" valign="middle" rowspan="1" colspan="1">CKAFIVTDEDIRKQEERVQQ</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">MBD2_364&#x02013;383</td><td align="center" valign="middle" rowspan="1" colspan="1">VTDEDIRKQEERVQQVRKKL</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">MBD2_369&#x02013;388</td><td align="center" valign="middle" rowspan="1" colspan="1">IRKQEERVQQVRKKLEEALM</td></tr><tr><td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">MBD2_374&#x02013;393</td><td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ERVQQVRKKLEEALMADILS</td></tr><tr><td rowspan="5" align="center" valign="middle" style="border-bottom:solid thin" colspan="1">p66&#x003b1;</td><td rowspan="5" align="center" valign="middle" style="border-bottom:solid thin" colspan="1">136&#x02013;175</td><td align="center" valign="middle" rowspan="1" colspan="1">p66&#x003b1;_136&#x02013;155</td><td align="center" valign="middle" rowspan="1" colspan="1">SSPEERERMIKQLKEELRLE</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">p66&#x003b1;_141&#x02013;160</td><td align="center" valign="middle" rowspan="1" colspan="1">RERMIKQLKEELRLEEAKLV</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">p66&#x003b1;_146&#x02013;165</td><td align="center" valign="middle" rowspan="1" colspan="1">KQLKEELRLEEAKLVLLKKL</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">p66&#x003b1;_151&#x02013;170</td><td align="center" valign="middle" rowspan="1" colspan="1">ELRLEEAKLVLLKKLRQSQI</td></tr><tr><td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">p66&#x003b1;_156&#x02013;175</td><td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">EAKLVLLKKLRQSQIQKEAT</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140371327754992"><a target="object" rel="noopener" href="/pmc/articles/PMC7432763/table/ijms-21-05248-t001/?report=objectonly">Open in a separate window</a></div></div><p>Importantly, we designed a new evaluation score function focusing on the disorder-to-order transition region as described in the <a href="#sec3-ijms-21-05248" rid="sec3-ijms-21-05248" class=" sec">Section 3</a> Materials and Methods. The procedure of using such a segmented peptide for docking is summarized in <a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f002/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f002" rid-ob="ob-ijms-21-05248-f002" co-legend-rid="lgnd_ijms-21-05248-f002"><span>Figure 2</span></a>A,B.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="ijms-21-05248-f002" co-legend-rid="lgnd_ijms-21-05248-f002"><a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f002/" target="figure" rid-figpopup="ijms-21-05248-f002" rid-ob="ob-ijms-21-05248-f002"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140371326936384"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7432763_ijms-21-05248-g002.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ijms-21-05248-g002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC7432763/bin/ijms-21-05248-g002.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140371326936384"><a target="object" rel="noopener" href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f002/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_ijms-21-05248-f002"><div><a class="figpopup" href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f002/" target="figure" rid-figpopup="ijms-21-05248-f002" rid-ob="ob-ijms-21-05248-f002">Figure 2</a></div><!--caption a7--><div class="caption"><p>Peptide molecular docking procedure and its application to 10058F4 and other disorder-to-order transition region inhibitors (ABA and APC targeting MYC 350&#x02013;439 region 15 peptide groups (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f001/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f001" rid-ob="ob-ijms-21-05248-f001" co-legend-rid="lgnd_ijms-21-05248-f001"><span>Figure 1</span></a>). (<strong>A</strong>) The procedure of target region segmentation and molecular docking toward generated peptides. (<strong>B</strong>) A workflow diagram following the procedure described in plot A. (<strong>C</strong>) Application of the method in the analysis of 10058F4, ABA, and APC interactions with 15 peptide groups derived from the MYC 350&#x02013;439 region and pie charts showing distribution of the 10 best docking results for each peptide group.</p></div></div></div><p>To validate the applicability of this new approach for accurate discovery of small molecules inhibiting DOT-based protein&#x02013;protein interactions, we aimed to distinguish a MYC disorder-to-order transition coiled-coil region inhibitor (10058F4) from other inhibitors (ABA and APC). Recent an NMR study described secondary structure regions in MYC and annotated structural changes induced by 10058F4 in this protein [<a href="#B14-ijms-21-05248" rid="B14-ijms-21-05248" class=" bibr popnode">14</a>]. We expected that our approach would show results that would be similar or comparable with the outputs of this recent NMR study, especially for the higher rank docking results. After molecular docking of 10058F4 to 15 peptide groups, we discovered that the best 10 molecular docking outputs (see the <a href="#sec3-ijms-21-05248" rid="sec3-ijms-21-05248" class=" sec">Section 3</a> Materials and Methods) from each of the 15 groups showed two distinguishable interaction parts separated by the 380&#x02013;399 region (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f002/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f002" rid-ob="ob-ijms-21-05248-f002" co-legend-rid="lgnd_ijms-21-05248-f002"><span>Figure 2</span></a>C). These observations follow the results of a recent NMR study [<a href="#B14-ijms-21-05248" rid="B14-ijms-21-05248" class=" bibr popnode">14</a>].</p><p>According to the previous research on MYC inhibitors, the MYC 363&#x02013;381 region is involved in 10074G5 binding and the 402&#x02013;412 region plays a role in 10058F4 binding [<a href="#B11-ijms-21-05248" rid="B11-ijms-21-05248" class=" bibr popnode">11</a>]. Our peptide docking study revealed that the 10 best positions from the MYC 390&#x02013;409 region interact only with 10058F4. Because our peptide segmentation slides 5 amino acids, surrounding peptide docking results also showed higher 10058F4 occupancy in the 10 best docking results (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f002/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f002" rid-ob="ob-ijms-21-05248-f002" co-legend-rid="lgnd_ijms-21-05248-f002"><span>Figure 2</span></a>C). This region overlaps with the already known 10058F4 binding site (residues 402&#x02013;412). Especially Y402 was discovered as the major binding residue from the previous circular dichroism and NMR experiment, which revealed Y402 activity in the MYC 402&#x02013;412 under 10058F4 treatment [<a href="#B11-ijms-21-05248" rid="B11-ijms-21-05248" class=" bibr popnode">11</a>], and two molecular simulation studies confirmed the role of Y402 in binding [<a href="#B17-ijms-21-05248" rid="B17-ijms-21-05248" class=" bibr popnode">17</a>,<a href="#B18-ijms-21-05248" rid="B18-ijms-21-05248" class=" bibr popnode">18</a>]. These previous studies support our discovery of the regions of major occupancy of 10058F4 in MYC 385&#x02013;424 region peptides.</p><p class="p p-last">In summary, molecular docking traditionally relied on already determined target protein structures. As a result, it was hard to apply the technique toward the disorder-to-order transition region in noncomplex forms. Another problem is that intact protein corresponding binding site in MYC is so long to study disorder-to-order transition. To overcome the problems and to follow disorder-to-order transition definition, it was necessary to predict possible conformations in shorter length peptides derived from the binding site. Thus, we utilized web algorithm to predict diverse forms of binding site originated peptides. Molecular docking was applied toward peptides in diverse conformation. Finally, we proved well-known MYC inhibitor (10058F4) bindings toward corresponding disorder-to-order transition region.</p></div><div id="sec2dot2-ijms-21-05248" class="sec"><h3 id="sec2dot2-ijms-21-05248title">2.2. MYC and MDM2 Inhibitors Docking Evaluation Shows Each Compound Binds to Peptide Corresponding Regions, and 10058F4 Interacts with MYC Y402</h3><p class="p p-first">In the previous study, we focused on the similarity of the intrinsic disorder score pattern (positive slope) within the disorder-to-order transition coiled-coil regions of MYC and MBD2 [<a href="#B12-ijms-21-05248" rid="B12-ijms-21-05248" class=" bibr popnode">12</a>]. Interestingly, the higher occupancy rank of 10058F4 was discovered for the MYC 360&#x02013;394 region (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f002/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f002" rid-ob="ob-ijms-21-05248-f002" co-legend-rid="lgnd_ijms-21-05248-f002"><span>Figure 2</span></a>C), which is known as the 10074G5 binding site [<a href="#B11-ijms-21-05248" rid="B11-ijms-21-05248" class=" bibr popnode">11</a>]. The higher occupancy of 10058F4 among the MYC-derived peptides could be due to their high protein-ligand activity, as a recent NMR discovered [<a href="#B14-ijms-21-05248" rid="B14-ijms-21-05248" class=" bibr popnode">14</a>]. To compare with its original binding partner, we tested the interaction of 10074G5 with these peptides. Furthermore, we further tested the relevance using another well-known protein intrinsic disorder advantage for drug discovery, RG7112, an MDM2 inhibitor [<a href="#B19-ijms-21-05248" rid="B19-ijms-21-05248" class=" bibr popnode">19</a>,<a href="#B20-ijms-21-05248" rid="B20-ijms-21-05248" class=" bibr popnode">20</a>]. We determined the RG7112 binding site according to the structure of the humanized <em>Xenopus</em> MDM2, which showed that the corresponding human MDM2 binding site is located at the 50&#x02013;99 region [<a href="#B21-ijms-21-05248" rid="B21-ijms-21-05248" class=" bibr popnode">21</a>].</p><p>We applied the same molecular docking environment to 10074G5 and RG7112 (see the <a href="#sec3-ijms-21-05248" rid="sec3-ijms-21-05248" class=" sec">Section 3</a> Materials and Methods) with corresponding targets (10074G5&#x02013;MYC and RG7112&#x02013;MDM2). From each peptide docking result, we selected the 10 best positions (samples) and obtained T scores for them. We considered each compound docking result as a population for statistic calculation.</p><p>Importantly, our segmented peptide docking and evaluation of newly derived T scores showed distinguishable patterns between 10058F4 in MYC 390&#x02013;409 and 10074G5 in MYC 365&#x02013;384 (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f003/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f003" rid-ob="ob-ijms-21-05248-f003" co-legend-rid="lgnd_ijms-21-05248-f003"><span>Figure 3</span></a>A), which is in the agreement with the results of previous CD and NMR studies [<a href="#B14-ijms-21-05248" rid="B14-ijms-21-05248" class=" bibr popnode">14</a>]. Besides, RG7112 binding site at MDM2 50&#x02013;69 showed the highest T score among segments derived from the 50&#x02013;99 region (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f003/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f003" rid-ob="ob-ijms-21-05248-f003" co-legend-rid="lgnd_ijms-21-05248-f003"><span>Figure 3</span></a>B). The MDM2 50&#x02013;69 region contains a &#x003b1;-helix between residues 50 and 64, whereas residues 65&#x02013;69 form a loop. This region directly contacts RG7112 [<a href="#B21-ijms-21-05248" rid="B21-ijms-21-05248" class=" bibr popnode">21</a>]. The majority of the remaining regions in the 70&#x02013;90 segments go away from the RG7112. However, a short loop (93&#x02013;96) comes back to the binding site and contacts RG7112 [<a href="#B21-ijms-21-05248" rid="B21-ijms-21-05248" class=" bibr popnode">21</a>]. Therefore, the already known structural biology study results of the active compound to the intrinsically disordered region are well coordinated with our observations.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="ijms-21-05248-f003" co-legend-rid="lgnd_ijms-21-05248-f003"><a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f003/" target="figure" rid-figpopup="ijms-21-05248-f003" rid-ob="ob-ijms-21-05248-f003"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140371325729328"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7432763_ijms-21-05248-g003.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ijms-21-05248-g003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC7432763/bin/ijms-21-05248-g003.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140371325729328"><a target="object" rel="noopener" href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f003/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_ijms-21-05248-f003"><div><a class="figpopup" href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f003/" target="figure" rid-figpopup="ijms-21-05248-f003" rid-ob="ob-ijms-21-05248-f003">Figure 3</a></div><!--caption a7--><div class="caption"><p>A protein intrinsic disorder score plot of peptide docking region and the corresponding region docking T score heatmap (normalized T scores): (<strong>A</strong>) MYC protein intrinsic disorder score and corresponding region T score heatmap of 10058F4 and 10074G5. (<strong>B</strong>) MDM2 protein intrinsic disorder score and corresponding region T score heatmap of RG7112. (<strong>C</strong>) MYC contact residues shown on MYC&#x02013;Protein max (MAX) complex structure (PDB ID 1NKP) [<a href="#B13-ijms-21-05248" rid="B13-ijms-21-05248" class=" bibr popnode">13</a>]. (<strong>D</strong>) <em>T</em>-test on the new docking evaluation score (<em>y</em>-axis) between contact mapping cases and others. * <em>p</em>-value &#x0003c; 0.001.</p></div></div></div><p>This evidence supports our method validity for IDPR binding site inhibition, not limited to direct IDPR inhibition. Another merit of our method is that it does not depend on binding site structure. Although we did not apply our method for fuzzy complex inhibition, we propose that our method is applicable to block such fuzzy complex. NUPR1-MSL1 forms a fuzzy complex, and a chemical compound inhibitor (ZZW-115) was discovered for blocking their complex formation [<a href="#B22-ijms-21-05248" rid="B22-ijms-21-05248" class=" bibr popnode">22</a>,<a href="#B23-ijms-21-05248" rid="B23-ijms-21-05248" class=" bibr popnode">23</a>]. We suggest that our next goal is to validate this compound binding toward the corresponding target regions in NUPR1 [<a href="#B23-ijms-21-05248" rid="B23-ijms-21-05248" class=" bibr popnode">23</a>].</p><p>To study binding properties of compounds, we analyzed peptide contacts from molecular docking and compared those contact sites with sites involved in interaction with their target protein structures. To do this from molecular docking result, we evaluated the number of contacts between the peptides and compounds. For double evaluation, we used two contact distance cutoffs (4 &#x0212b; and 3.6 &#x0212b;) for contact decision. After mapping those contact residues, we compared the residue positions with the corresponding structures of MYC (PDB ID: 1NKP chain A) in its complex with MAX (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f003/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f003" rid-ob="ob-ijms-21-05248-f003" co-legend-rid="lgnd_ijms-21-05248-f003"><span>Figure 3</span></a>C). Newly evaluated contact scores from the analyzed docking cases showed significantly higher values than the results of other molecular docking experiments (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f003/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f003" rid-ob="ob-ijms-21-05248-f003" co-legend-rid="lgnd_ijms-21-05248-f003"><span>Figure 3</span></a>D). </p><p>Our structural comparison study of 10058F4 and MYC confirmed that Y402 serves as an important contact point (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f003/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f003" rid-ob="ob-ijms-21-05248-f003" co-legend-rid="lgnd_ijms-21-05248-f003"><span>Figure 3</span></a>C, <a href="#app1-ijms-21-05248" rid="app1-ijms-21-05248" class=" app">Supplementary Figure S1</a>). According to the previous studies on the 10058F4 and MYC, Y402 was considered as the main contact residue of 10058F4 [<a href="#B11-ijms-21-05248" rid="B11-ijms-21-05248" class=" bibr popnode">11</a>,<a href="#B17-ijms-21-05248" rid="B17-ijms-21-05248" class=" bibr popnode">17</a>,<a href="#B18-ijms-21-05248" rid="B18-ijms-21-05248" class=" bibr popnode">18</a>]. In corresponding analyses, two studies focused on the 402&#x02013;412 region and used it in the molecular dynamics simulation [<a href="#B17-ijms-21-05248" rid="B17-ijms-21-05248" class=" bibr popnode">17</a>,<a href="#B18-ijms-21-05248" rid="B18-ijms-21-05248" class=" bibr popnode">18</a>]. Because we segmented the 385&#x02013;424 region, K398 also appeared as the contact residue near Y402. Furthermore, Y402 is embedded within the T400&#x02013;S405 region between two polar amino acid residues. Therefore, our study suggests that the major MYC-10058F4 binding mechanism is centered at Y402 and the polarity of its environment.</p><p class="p p-last">Although they were less frequent than 10058F4&#x02013;MYC interactions, we also found contacts between ABA and the MYC A401&#x02013;Y402 region (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f003/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f003" rid-ob="ob-ijms-21-05248-f003" co-legend-rid="lgnd_ijms-21-05248-f003"><span>Figure 3</span></a>C, <a href="#app1-ijms-21-05248" rid="app1-ijms-21-05248" class=" app">Supplementary Figure S1</a>). In our previous study, we applied the Similarity Ensemble Approach (SEA) database to search for the off-targets [<a href="#B24-ijms-21-05248" rid="B24-ijms-21-05248" class=" bibr popnode">24</a>]. Our novel method showed distinguishable contacts for 10058F4, 10074G5, and RG7112 inside their known binding regions as well as the direct/indirect contacts in the disorder-to-order transition region (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f003/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f003" rid-ob="ob-ijms-21-05248-f003" co-legend-rid="lgnd_ijms-21-05248-f003"><span>Figure 3</span></a>A,B). Although SEA was focusing on structural data, our new method considers diverse forms of disorder-to-order transition regions. MYC A401&#x02013;Y402 contacts of ABA may provide an evidence of MYC inhibition, which constitutes a new discovery.</p></div><div id="sec2dot3-ijms-21-05248" class="sec"><h3 id="sec2dot3-ijms-21-05248title">2.3. Chemical Compound Library Docking to MYC 390&#x02013;409 Shows Better 10058F4 Scores</h3><p class="p p-first">Generally, molecular docking aims to find hits from the chemical compound libraries, and this is the reason why the procedure is called &#x0201c;virtual screening&#x0201d;. Instead of virtual screening, we studied the statistical significance of 10058F4 docking through comparison with the results of docking of the members of a valid chemical compound library (Diversity set V of National Cancer Institute). Besides, because of utilization of our new evaluation score function characteristics (<a href="#sec3-ijms-21-05248" rid="sec3-ijms-21-05248" class=" sec">Section 3</a> Materials and Methods), we hypothesized that there is a positive correlation between the docking-involved atom number and a new evaluation score. From the compounds (10058F4, ABA, and APC) and MYC peptide dockings, we discovered that the 10 best results of the MYC 390&#x02013;409 docking contain only 10058F4 (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f002/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f002" rid-ob="ob-ijms-21-05248-f002" co-legend-rid="lgnd_ijms-21-05248-f002"><span>Figure 2</span></a>C). Therefore, we applied molecular docking focused on the MYC 390&#x02013;409 peptide as described in the <a href="#sec3-ijms-21-05248" rid="sec3-ijms-21-05248" class=" sec">Section 3</a> Materials and Methods.</p><p>To check the correlation between the docking-involved atom numbers and the new evaluation scores, we generated linear regression equation from each population (sample 1, sample 5, and sample 10) (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f004/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f004" rid-ob="ob-ijms-21-05248-f004" co-legend-rid="lgnd_ijms-21-05248-f004"><span>Figure 4</span></a>A,B). The 22 models of MYC 390&#x02013;409 peptide docking with 10058F4 (220 results per sample) or without 10058F4 did not show meaningful differences in linear regression. T scores of the 10 best 10058F4 molecular docking results from 22 models, considering each sample as population, were like each other among different populations (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f004/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f004" rid-ob="ob-ijms-21-05248-f004" co-legend-rid="lgnd_ijms-21-05248-f004"><span>Figure 4</span></a>C). The other remaining 10058F4 docking results (210 results) did not cause significant differences in the T score distribution (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f004/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f004" rid-ob="ob-ijms-21-05248-f004" co-legend-rid="lgnd_ijms-21-05248-f004"><span>Figure 4</span></a>C).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="ijms-21-05248-f004" co-legend-rid="lgnd_ijms-21-05248-f004"><a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f004/" target="figure" rid-figpopup="ijms-21-05248-f004" rid-ob="ob-ijms-21-05248-f004"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140371328002880"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7432763_ijms-21-05248-g004.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ijms-21-05248-g004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC7432763/bin/ijms-21-05248-g004.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140371328002880"><a target="object" rel="noopener" href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f004/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_ijms-21-05248-f004"><div><a class="figpopup" href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f004/" target="figure" rid-figpopup="ijms-21-05248-f004" rid-ob="ob-ijms-21-05248-f004">Figure 4</a></div><!--caption a7--><div class="caption"><p>Statistical analysis of the new docking evaluation score focusing on 10058F4&#x02013;MYC 390&#x02013;409 region peptides: (<strong>A</strong>,<strong>B</strong>) Linear regression between docking-involved atom number (<em>x</em>-axis) and the new docking evaluation score (<em>y</em>-axis) with 10058F4 docking results in the population (<strong>A</strong>) and without 10058F4 docking results in the population (<strong>B</strong>). (<strong>C</strong>) The 10 best 10058F4 docking result T scores in 10 random samples (considering each sample as a population) with/without 10058F4 docking results. (<strong>D</strong>) The 10 best 10058F4 docking result T scores in three different docking-involved atom number groups (15&#x02013;17, 16&#x02013;18, and 17&#x02013;19) with/without 10058 docking results from all 10 random samples. (<strong>E</strong>) <em>T</em>-test between the 10 best 10058F4 docking results and each of the docking-involved atom number groups (14&#x02013;20) from all 10 random samples. * <em>p</em>-value &#x0003c; 0.001.</p></div></div></div><p>Because of a new docking evaluation score function with docking-involved atom number normalization, we also grouped all samples (sample 1&#x02013;10) according to each compound docking-involved atom number. Since the 10058F4 docking-involved atom number was 17, we selected three groups, 15&#x02013;17, 16&#x02013;18, and 17&#x02013;19, for the analysis. As shown in <a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f004/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f004" rid-ob="ob-ijms-21-05248-f004" co-legend-rid="lgnd_ijms-21-05248-f004"><span>Figure 4</span></a>D, the population of the 10058F4 docking results did not cause differences in trend. However, the grouped docking-involved atom number increase caused a decrease in the T score of the 10 best 10058F4 docking results. This means that the significance of the score of the 10 best 10058F4 docking results is reduced along the docking-involved atom number increase.</p><p class="p p-last">Nevertheless, the 10 best 10058F4 docking results which had many contacts with Y402 (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f003/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f003" rid-ob="ob-ijms-21-05248-f003" co-legend-rid="lgnd_ijms-21-05248-f003"><span>Figure 3</span></a>C, <a href="#app1-ijms-21-05248" rid="app1-ijms-21-05248" class=" app">Supplementary Figure S1</a>) as well as the majority of binding with MYC 390&#x02013;409 (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f002/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f002" rid-ob="ob-ijms-21-05248-f002" co-legend-rid="lgnd_ijms-21-05248-f002"><span>Figure 2</span></a>C and <a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f003/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f003" rid-ob="ob-ijms-21-05248-f003" co-legend-rid="lgnd_ijms-21-05248-f003"><span>Figure 3</span></a>A) showed greater new evaluation scores than each of the docking-involved atom number 14&#x02013;20 group average scores (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f004/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f004" rid-ob="ob-ijms-21-05248-f004" co-legend-rid="lgnd_ijms-21-05248-f004"><span>Figure 4</span></a>E). In the study describing the 10058F4 discovery, the compound was found using the yeast two-hybrid assay, applying 10,000 drug-like small molecules [<a href="#B10-ijms-21-05248" rid="B10-ijms-21-05248" class=" bibr popnode">10</a>]. The following study revealed that the specific 10058F4 binding region in MYC is the 402&#x02013;412 region [<a href="#B11-ijms-21-05248" rid="B11-ijms-21-05248" class=" bibr popnode">11</a>]. We showed that the 10 best docking results of 10058F4 toward MYC 390&#x02013;409 have higher scores than other compounds with similar docking-involved atom numbers. Although the binding mechanism between 10058F4 and MYC 402&#x02013;412 is unclear, we confirmed the previous discovery of specific binding of 10058F4 to Y402 [<a href="#B11-ijms-21-05248" rid="B11-ijms-21-05248" class=" bibr popnode">11</a>,<a href="#B17-ijms-21-05248" rid="B17-ijms-21-05248" class=" bibr popnode">17</a>,<a href="#B18-ijms-21-05248" rid="B18-ijms-21-05248" class=" bibr popnode">18</a>], applying the original definition of the disorder-to-order transition.</p></div><div id="sec2dot4-ijms-21-05248" class="sec sec-last"><h3 id="sec2dot4-ijms-21-05248title">2.4. ABA and APC Both Have Electrostatic Interaction with MBD2, but Only ABA Shows Specific Electrostatic Interaction with p66&#x003b1;</h3><p class="p p-first">We studied ABA and APC interactions with MBD2 and p66&#x003b1;, applying the same procedure used in the 10058F4 analysis. Although we discovered these two compounds from molecular docking to MBD2 and both compounds were shown to efficiently block the MBD2&#x02013;p66&#x003b1; complex formation [<a href="#B12-ijms-21-05248" rid="B12-ijms-21-05248" class=" bibr popnode">12</a>], it was unclear if there is a binding difference between those proteins. To see their difference using our novel method, the MBD2&#x02013;p66&#x003b1; (PDB ID 2L2L) [<a href="#B15-ijms-21-05248" rid="B15-ijms-21-05248" class=" bibr popnode">15</a>] complex structure was referred to determine the ABA and APC binding region as a disorder-to-order transition region. Then, we segmented MBD2 (residues 359&#x02013;393) and p66&#x003b1; (residues 136&#x02013;175) into the sets of the overlapping 20-amino-acid-long sequences (<a href="/pmc/articles/PMC7432763/table/ijms-21-05248-t001/" target="table" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-t001" rid-ob="ob-ijms-21-05248-t001" co-legend-rid=""><span>Table 1</span></a>) and obtained diverse structures of each peptide region. From diverse forms of peptides (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f005/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f005" rid-ob="ob-ijms-21-05248-f005" co-legend-rid="lgnd_ijms-21-05248-f005"><span>Figure 5</span></a>), we applied molecular docking and analyzed the result as described in the Materials and Methods section.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="ijms-21-05248-f005" co-legend-rid="lgnd_ijms-21-05248-f005"><a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f005/" target="figure" rid-figpopup="ijms-21-05248-f005" rid-ob="ob-ijms-21-05248-f005"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140371327997040"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7432763_ijms-21-05248-g005.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ijms-21-05248-g005.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC7432763/bin/ijms-21-05248-g005.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140371327997040"><a target="object" rel="noopener" href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f005/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_ijms-21-05248-f005"><div><a class="figpopup" href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f005/" target="figure" rid-figpopup="ijms-21-05248-f005" rid-ob="ob-ijms-21-05248-f005">Figure 5</a></div><!--caption a7--><div class="caption"><p>Methyl-CpG-binding domain protein 2 (MBD2) 359&#x02013;393 region (<strong>A</strong>&#x02013;<strong>D</strong>) and p66&#x003b1; 136&#x02013;175 region (<strong>E</strong>&#x02013;<strong>I</strong>) segmented peptides: PEPFOLD ver 3.5 prediction result of peptide structures based on corresponding sequences (please look at the <a href="/pmc/articles/PMC7432763/table/ijms-21-05248-t001/" target="table" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-t001" rid-ob="ob-ijms-21-05248-t001" co-legend-rid=""><span>Table 1</span></a>). (A) MBD2_359&#x02013;378 (18 models), (B) MBD2_364&#x02013;383 (2 models), (C) MBD2_369&#x02013;388 (14 models), (D) MBD2_374&#x02013;393 (16 models), (E) p66&#x003b1;_136&#x02013;155 (6 models), (F) p66&#x003b1;_141&#x02013;160 (22 models), (G) p66&#x003b1;_146&#x02013;165 (3 models), (H) p66&#x003b1;_151&#x02013;170 (5 models), (I) p66&#x003b1;_156&#x02013;175 (8 models).</p></div></div></div><p>ABA appears to have high T score for regions 359&#x02013;378/369&#x02013;388 in MBD2 and for the 136&#x02013;155 region in p66&#x003b1; (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f006/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f006" rid-ob="ob-ijms-21-05248-f006" co-legend-rid="lgnd_ijms-21-05248-f006"><span>Figure 6</span></a>A,B). Protein intrinsic disorder score patterns did not show meaningful correlation with the T scores as it was in the 10058F4 result (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f003/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f003" rid-ob="ob-ijms-21-05248-f003" co-legend-rid="lgnd_ijms-21-05248-f003"><span>Figure 3</span></a>A,B). ABA contact mapping revealed residues from the MBD2 backside. These mainly include charged and polar residues, such as R (Arg, +1), K (Lys, +1), E (Glu, &#x02212;1), and Q (Gln) (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f006/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f006" rid-ob="ob-ijms-21-05248-f006" co-legend-rid="lgnd_ijms-21-05248-f006"><span>Figure 6</span></a>C, <a href="#app1-ijms-21-05248" rid="app1-ijms-21-05248" class=" app">Supplementary Figure S2</a>). These results suggested that electrostatic interactions constitute a main binding mechanism that defines the MBD2&#x02013;ABA interaction. The p66&#x003b1; structural comparison with the contact mapping also revealed the presence of charged and polar residues, R(Arg, +1), K(Lys, +1), E(Glu, &#x02212;1), and Q (Gln) from behind the MBD2 binding site (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f006/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f006" rid-ob="ob-ijms-21-05248-f006" co-legend-rid="lgnd_ijms-21-05248-f006"><span>Figure 6</span></a>D, <a href="#app1-ijms-21-05248" rid="app1-ijms-21-05248" class=" app">Supplementary Figure S3</a>). There was no overlap with MBD2&#x02013;p66&#x003b1; binding site residues [<a href="#B15-ijms-21-05248" rid="B15-ijms-21-05248" class=" bibr popnode">15</a>]. Although we discovered ABA and APC from targeting p66-binding residues of MBD2 [<a href="#B12-ijms-21-05248" rid="B12-ijms-21-05248" class=" bibr popnode">12</a>], the majority of contacts is located behind the binding site. It suggests that ABA interacts with MBD2 behind the p66&#x003b1; binding site and that this interaction prevents MBD2 from p66&#x003b1; binding, likely by inducing the formation of a conformation incapable of MBD2 binding. In p66&#x003b1; as well, ABA binds p66&#x003b1; behind its MBD2-binding site. Therefore, it may also cause the structural changes of p66&#x003b1;, which do not allow MBD2 binding.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="ijms-21-05248-f006" co-legend-rid="lgnd_ijms-21-05248-f006"><a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f006/" target="figure" rid-figpopup="ijms-21-05248-f006" rid-ob="ob-ijms-21-05248-f006"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140371331610240"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7432763_ijms-21-05248-g006.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ijms-21-05248-g006.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC7432763/bin/ijms-21-05248-g006.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140371331610240"><a target="object" rel="noopener" href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f006/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_ijms-21-05248-f006"><div><a class="figpopup" href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f006/" target="figure" rid-figpopup="ijms-21-05248-f006" rid-ob="ob-ijms-21-05248-f006">Figure 6</a></div><!--caption a7--><div class="caption"><p>ABA molecular docking toward MBD2 and p66&#x003b1;: (<strong>A</strong>,<strong>B</strong>) Protein intrinsic disorder score plot and the 10 best target region segment peptides ABA docking result T score heatmap of MBD2 (A) and p66&#x003b1; (B). (<strong>C</strong>) MBD2 (cyan) contact residues shown on the MBD2&#x02013;p66&#x003b1; complex (PDB ID 2L2L) [<a href="#B15-ijms-21-05248" rid="B15-ijms-21-05248" class=" bibr popnode">15</a>] and the corresponding new docking evaluation score (<em>y</em>-axis) T-test between contact cases and others. (<strong>D</strong>) p66&#x003b1; (green) contact residues shown on the MBD2&#x02013;p66&#x0237a; complex (PDB ID 2L2L) [<a href="#B15-ijms-21-05248" rid="B15-ijms-21-05248" class=" bibr popnode">15</a>] and the corresponding new docking evaluation score (<em>y</em>-axis) <em>T</em>-test between contact cases and others. * <em>p</em>-value &#x0003c; 0.001.</p></div></div></div><p>To compare ABA and APC binding, we further analyzed the 10 best docking results of the MBD2 359&#x02013;393 region and p66&#x003b1; 136&#x02013;175 region in comparison with 10058F4. It turned out that ABA occupies the majority of the peptides derived from the MBD2 364&#x02013;393 region (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f007/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f007" rid-ob="ob-ijms-21-05248-f007" co-legend-rid="lgnd_ijms-21-05248-f007"><span>Figure 7</span></a>A) and the p66&#x003b1; 136&#x02013;175 region (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f007/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f007" rid-ob="ob-ijms-21-05248-f007" co-legend-rid="lgnd_ijms-21-05248-f007"><span>Figure 7</span></a>B). APC was not ranked among the 10 best docking results for any of the MBD2 359&#x02013;393 peptides (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f007/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f007" rid-ob="ob-ijms-21-05248-f007" co-legend-rid="lgnd_ijms-21-05248-f007"><span>Figure 7</span></a>A), but it ranked in the 10 best docking results for the majority of the p66&#x003b1; 136&#x02013;175 peptides except for the p66&#x003b1; 146&#x02013;165 peptide (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f007/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f007" rid-ob="ob-ijms-21-05248-f007" co-legend-rid="lgnd_ijms-21-05248-f007"><span>Figure 7</span></a>B).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="ijms-21-05248-f007" co-legend-rid="lgnd_ijms-21-05248-f007"><a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f007/" target="figure" rid-figpopup="ijms-21-05248-f007" rid-ob="ob-ijms-21-05248-f007"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140371327758080"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7432763_ijms-21-05248-g007.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ijms-21-05248-g007.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC7432763/bin/ijms-21-05248-g007.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140371327758080"><a target="object" rel="noopener" href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f007/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_ijms-21-05248-f007"><div><a class="figpopup" href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f007/" target="figure" rid-figpopup="ijms-21-05248-f007" rid-ob="ob-ijms-21-05248-f007">Figure 7</a></div><!--caption a7--><div class="caption"><p>Molecular docking toward MBD2 and p66&#x003b1;, and APC focused docking result analysis: (<strong>A</strong>,<strong>B</strong>) 10058F4, ABA, and APC molecular docking toward MBD2 (A) and p66&#x003b1; (B) and the corresponding 10 best result distribution pie charts. (<strong>C</strong>,<strong>D</strong>) APC-focused docking result T score heatmap from MBD2 docking (C) and p66&#x003b1; docking (D). (<strong>E</strong>) MBD2 (cyan) contact residues shown on the MBD2&#x02013;p66&#x003b1; complex (PDB ID 2L2L) [<a href="#B15-ijms-21-05248" rid="B15-ijms-21-05248" class=" bibr popnode">15</a>] and the corresponding new docking evaluation score (<em>y</em>-axis) <em>t</em>-test between contact cases and others. (<strong>F</strong>) p66&#x003b1; (green) contact residues shown on the MBD2&#x02013;p66&#x003b1; complex (PDB ID 2L2L) [<a href="#B15-ijms-21-05248" rid="B15-ijms-21-05248" class=" bibr popnode">15</a>] and the corresponding new docking evaluation score (<em>y</em>-axis) <em>t</em>-test between contact cases and others. * <em>p</em>-value &#x0003c; 0.001.</p></div></div></div><p>Because APC docking scores were lower than those of ABA as shown in <a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f007/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f007" rid-ob="ob-ijms-21-05248-f007" co-legend-rid="lgnd_ijms-21-05248-f007"><span>Figure 7</span></a>A,B, we collected only APC docking results and further analyzed them. The heatmap derived using the APC binding T score matrix proved APC preference for the MBD2 374&#x02013;393 (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f007/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f007" rid-ob="ob-ijms-21-05248-f007" co-legend-rid="lgnd_ijms-21-05248-f007"><span>Figure 7</span></a>C). However, there was no APC among the 10 best results of the MBD2 374&#x02013;393 peptide docking. In the case of p66&#x003b1; binding, T score was relatively higher in p66&#x003b1; 136&#x02013;155 and 141&#x02013;160 peptides, and APC occupied 25% of the 10 best positions in the p66&#x003b1; 136&#x02013;155 and 141&#x02013;160 regions (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f007/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f007" rid-ob="ob-ijms-21-05248-f007" co-legend-rid="lgnd_ijms-21-05248-f007"><span>Figure 7</span></a>B,D).</p><p class="p p-last">MBD2&#x02013;APC contact mapping (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f007/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f007" rid-ob="ob-ijms-21-05248-f007" co-legend-rid="lgnd_ijms-21-05248-f007"><span>Figure 7</span></a>E, <a href="#app1-ijms-21-05248" rid="app1-ijms-21-05248" class=" app">Supplementary Figure S4</a>) revealed results like the outputs of MBD2&#x02013;ABA contact mapping (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f006/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f006" rid-ob="ob-ijms-21-05248-f006" co-legend-rid="lgnd_ijms-21-05248-f006"><span>Figure 6</span></a>C). Contact residues appeared from the MBD2 backside (opposite from p66&#x003b1;-facing side) and were mostly charged and polar ones, R (Arg, +1), K (Lys, +1), E (Glu, &#x02212;1), and Q (Gln). However, p66&#x003b1;&#x02013;APC contact mapping (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f007/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f007" rid-ob="ob-ijms-21-05248-f007" co-legend-rid="lgnd_ijms-21-05248-f007"><span>Figure 7</span></a>F, <a href="#app1-ijms-21-05248" rid="app1-ijms-21-05248" class=" app">Supplementary Figure S5</a>) was different from that of p66&#x003b1;&#x02013;ABA (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f006/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f006" rid-ob="ob-ijms-21-05248-f006" co-legend-rid="lgnd_ijms-21-05248-f006"><span>Figure 6</span></a>D), and in addition to charged and polar residues R (Arg, +1), K (Lys, +1), E (Glu, &#x02212;1), and Q (Gln), hydrophobic residue L (Leu) and polar residue S (Ser) were found in the analysis. Despite the fact that no specific interaction mechanisms were identified and no inside&#x02013;backside distinctions were found, L152 and R166 contact was discovered. Furthermore, the 10 best molecular docking results of 10058F4 and ABA with MBD2 (4 groups of peptides) and p66&#x003b1; (5 groups of peptides) without APC showed greater score than the APC docking results discovered based on the significant contacts of this compound with MBD2 and p66&#x003b1; (<a href="#app1-ijms-21-05248" rid="app1-ijms-21-05248" class=" app">Supplementary Figure S6</a>).</p></div></div><div id="sec3-ijms-21-05248" class="tsec sec"><h2 class="head no_bottom_margin" id="sec3-ijms-21-05248title">3. Materials and Methods</h2><div id="sec3dot1-ijms-21-05248" class="sec sec-first"><h3 id="sec3dot1-ijms-21-05248title">3.1. Segmented Peptide Preparation, Molecular Docking, and Evaluation Score Update</h3><p class="p p-first-last">For mimicking characteristics of intrinsically disordered region, we obtained diverse structures of target disorder-to-order transition region after its segmentation into a set of overlapping 20-amino-acid-long peptides. This length of segmentation was used because a 20-amino-acid-long peptide is optimal for the unfolding and folding studies [<a href="#B25-ijms-21-05248" rid="B25-ijms-21-05248" class=" bibr popnode">25</a>]. We segmented protein sequences into subsets (<a href="/pmc/articles/PMC7432763/table/ijms-21-05248-t001/" target="table" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-t001" rid-ob="ob-ijms-21-05248-t001" co-legend-rid=""><span>Table 1</span></a>) and then uploaded them on PEPFOLD v3.5 web server (<a href="https://mobyle.rpbs.univ-paris-diderot.fr/cgi-bin/portal.py#forms::PEP-FOLD3">https://mobyle.rpbs.univ-paris-diderot.fr/cgi-bin/portal.py#forms::PEP-FOLD3</a>) to predict peptide structures [<a href="#B16-ijms-21-05248" rid="B16-ijms-21-05248" class=" bibr popnode">16</a>]. From the corresponding results (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f001/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f001" rid-ob="ob-ijms-21-05248-f001" co-legend-rid="lgnd_ijms-21-05248-f001"><span>Figure 1</span></a> and <a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f005/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f005" rid-ob="ob-ijms-21-05248-f005" co-legend-rid="lgnd_ijms-21-05248-f005"><span>Figure 5</span></a>), we obtained representative structures of clusters and applied molecular docking with custom function (smina) of Autodock vina (The Scripps Research Institute, Molecular Biology, La Jolla, CA 92037, USA), targeting the central residue (every peptide tenth residue) C&#x003b1; coordinates [<a href="#B26-ijms-21-05248" rid="B26-ijms-21-05248" class=" bibr popnode">26</a>,<a href="#B27-ijms-21-05248" rid="B27-ijms-21-05248" class=" bibr popnode">27</a>]. Chemical compound structures in 3-dimensional SDF formats were obtained from the PubChem database (<a href="https://pubchem.ncbi.nlm.nih.gov/">https://pubchem.ncbi.nlm.nih.gov/</a>). Chemical compounds used in this study are 10058F4 (5-[(4-Ethylphenyl)methylene]-2-thioxo-4-thiazolidinone, ZINC12406714, PubChem CID: 1271002), 10074G5 (4-nitro-N-(2-phenylphenyl)-2,1,3-benzoxadiazol-7-amine, ZINC3879010, PubChem CID: 2836600), RG7112 ([(4S,5R)-2-(4-tert-butyl-2-ethoxyphenyl)-4,5-bis(4-chlorophenyl)-4,5-dimethylimidazol-1-yl]-[4-(3-methylsulfonyl propyl)piperazin-1-yl]methanone, ZINC96270381, PubChem CID: 57406853), ABA (2-Amino-N-[[(3S)-2,3-dihydro-1,4-benzodioxin-3-yl]methyl]acetamide, ZINC40430779, PubChem CID: 93602182), and APC ((R)-3-(2-Amino-acetylamino)-pyrrolidine-1-carboxylic acid tert-butyl ester, ZINC60177071, PubChem CID: 66563909). Recent reverse docking study revealed that Autodock vina is biased (shows negative correlation) due to the contact surface area [<a href="#B28-ijms-21-05248" rid="B28-ijms-21-05248" class=" bibr popnode">28</a>]. Therefore, we measured the protein&#x02013;compound contact surface area [<a href="#B29-ijms-21-05248" rid="B29-ijms-21-05248" class=" bibr popnode">29</a>] and normalized for the number of the molecular docking-involved atoms. We also evaluated the peptide model quality applying DOPE (discrete optimized protein energy) [<a href="#B30-ijms-21-05248" rid="B30-ijms-21-05248" class=" bibr popnode">30</a>] because we obtained structures from the modeling web server. Finally, we designed a new custom score for intrinsically disordered region direct targeting drug discovery evaluation, combining all the aforementioned scores.
</p><div class="disp-formula" id="FD1-ijms-21-05248"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="mm1"><mrow><mrow><mi>S</mi><mi>C</mi><mi>O</mi><mi>R</mi><mi>E</mi><mo>=</mo><munder><mstyle mathsize="140%" displaystyle="true"><mo>&#x02211;</mo></mstyle><mrow><mi>i</mi><mo>&#x0003c;</mo><mi>j</mi></mrow></munder><msub><mi>f</mi><mrow><msub><mi>t</mi><mi>i</mi></msub><msub><mi>t</mi><mi>j</mi></msub><mo>&#x000a0;</mo></mrow></msub><mrow><mo stretchy="false">(</mo><mrow><msub><mi>r</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow><mo stretchy="false">)</mo></mrow><mo>&#x000d7;</mo><mo>&#x02212;</mo><msub><mi>k</mi><mi>B</mi></msub><mi>T</mi><mo>&#x000a0;</mo><mi>ln</mi><msup><mi>g</mi><mrow><mrow><mo stretchy="false">(</mo><mi>n</mi><mo stretchy="false">)</mo></mrow></mrow></msup><mrow><mo stretchy="false">(</mo><mrow><mover accent="true"><mrow><msub><mi>x</mi><mn>1</mn></msub></mrow><mo stretchy="true">&#x02192;</mo></mover><mo>,</mo><mover accent="true"><mrow><msub><mi>x</mi><mn>2</mn></msub></mrow><mo stretchy="true">&#x02192;</mo></mover><mo>,</mo><mo>&#x000a0;</mo><mo>&#x02026;</mo><mo>,</mo><mo>&#x000a0;</mo><mover accent="true"><mrow><msub><mi>x</mi><mi>n</mi></msub></mrow><mo stretchy="true">&#x02192;</mo></mover></mrow><mo stretchy="false">)</mo></mrow><mo>&#x000a0;</mo><mo>/</mo><mo>&#x000a0;</mo><mrow><mo stretchy="false">(</mo><mrow><munderover><mstyle mathsize="140%" displaystyle="true"><mo>&#x02211;</mo></mstyle><mrow><mi>i</mi><mo>=</mo><mn>0</mn></mrow><mi>n</mi></munderover><mi>C</mi><mi>S</mi><msub><mi>A</mi><mi>i</mi></msub><mo>&#x000a0;</mo><mo>&#x000f7;</mo><mi>n</mi></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></div></div><p>
where <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="mm2"><mrow><mrow><mstyle displaystyle="true"><munder><mstyle displaystyle="true"><mo>&#x02211;</mo></mstyle><mrow><mi>i</mi><mo>&#x0003c;</mo><mi>j</mi></mrow></munder></mstyle><msub><mi>f</mi><mrow><msub><mi>t</mi><mi>i</mi></msub><msub><mi>t</mi><mi>j</mi></msub><mo>&#x000a0;</mo></mrow></msub><mrow><mo stretchy="false">(</mo><mrow><msub><mi>r</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math> is the smina (Autodock vina) score function [<a href="#B28-ijms-21-05248" rid="B28-ijms-21-05248" class=" bibr popnode">28</a>], <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="mm3"><mrow><mrow><mo>&#x02212;</mo><msub><mi>k</mi><mi>B</mi></msub><mi>T</mi><mo>&#x000a0;</mo><mi>ln</mi><msup><mi>g</mi><mrow><mrow><mo stretchy="false">(</mo><mi>n</mi><mo stretchy="false">)</mo></mrow></mrow></msup><mrow><mo stretchy="false">(</mo><mrow><mover accent="true"><mrow><msub><mi>x</mi><mn>1</mn></msub></mrow><mo stretchy="true">&#x02192;</mo></mover><mo>,</mo><mover accent="true"><mrow><msub><mi>x</mi><mn>2</mn></msub></mrow><mo stretchy="true">&#x02192;</mo></mover><mo>,</mo><mo>&#x000a0;</mo><mo>&#x02026;</mo><mo>,</mo><mo>&#x000a0;</mo><mover accent="true"><mrow><msub><mi>x</mi><mi>n</mi></msub></mrow><mo stretchy="true">&#x02192;</mo></mover></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math> is the DOPE score function [<a href="#B30-ijms-21-05248" rid="B30-ijms-21-05248" class=" bibr popnode">30</a>], and <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="mm4"><mrow><mrow><mstyle displaystyle="true"><munderover><mstyle displaystyle="true"><mo>&#x02211;</mo></mstyle><mrow><mi>i</mi><mo>=</mo><mn>0</mn></mrow><mi>n</mi></munderover></mstyle><mi>C</mi><mi>S</mi><msub><mi>A</mi><mi>i</mi></msub></mrow></mrow></math> is protein&#x02013;compound contact surface area calculation algorithm (dr_sasa) score [<a href="#B29-ijms-21-05248" rid="B29-ijms-21-05248" class=" bibr popnode">29</a>]. Normalization of the dr_sasa score with the molecular docking-involved atom number (<em>n</em>) was empirical adjustment to calculate atomic contribution.</p></div><div id="sec3dot2-ijms-21-05248" class="sec"><h3 id="sec3dot2-ijms-21-05248title">3.2. MYC, MDM2, MBD2, and p66&#x003b1; T Score Heatmap</h3><p class="p p-first-last">The new function score was employed for molecular docking evaluation. Each of the 10 best scores of the compound&#x02013;peptide pair was collected, and we considered each compound docking score from all peptide dockings as a population. Then, we obtained T scores of each compound&#x02013;peptide pair for the 10 best scores (the 10 best results for each compound) according to the equation below.
</p><div class="disp-formula" id="FD2-ijms-21-05248"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="mm5"><mrow><mrow><mi>T</mi><mo>=</mo><mfrac><mrow><mover accent="true"><mi>x</mi><mo>&#x000af;</mo></mover><mo>&#x02212;</mo><mi>&#x003bc;</mi></mrow><mrow><mi>S</mi><mo>&#x000a0;</mo><mo>/</mo><mo>&#x000a0;</mo><msqrt><mi>n</mi></msqrt></mrow></mfrac></mrow></mrow></math></div><div class="l">(1)</div></div><p>
where <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="mm6"><mrow><mover accent="true"><mi>x</mi><mo>&#x000af;</mo></mover></mrow></math> is each compound&#x02019;s 10 best docking results mean score, <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="mm7"><mrow><mi>&#x003bc;</mi></mrow></math> is the population mean score, <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="mm8"><mrow><mi>S</mi></mrow></math> is the standard deviation of each compound&#x02019;s 10 best docking results, and <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="mm9"><mrow><mrow><msqrt><mi>n</mi></msqrt></mrow></mrow></math> is the square root of the number of best 10 docking results (<math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="mm10"><mrow><mrow><msqrt><mrow><mn>10</mn></mrow></msqrt></mrow></mrow></math>). After normalization of each compound T score divided by the corresponding compound of all peptide docking T scores sum, we generated heatmaps using Python 3.7 (Python Software Foundation, Wilmington, DE 19801, USA) and matplotlib 3.2 (Plotly, Montreal, QC H2T 2A3, Canada).</p></div><div id="sec3dot3-ijms-21-05248" class="sec"><h3 id="sec3dot3-ijms-21-05248title">3.3. MYC, MBD2, and p66&#x003b1; Docking Contact Residue Mapping</h3><p class="p p-first-last">Each peptide&#x02019;s 10 best docking results from MYC 385&#x02013;424 (5 peptide groups), MBD2 359&#x02013;393 (4 peptide groups), and p66&#x003b1; 136&#x02013;175 (5 peptide groups) were collected. Next, we analyzed the contacts between the peptide residue atoms and the compound docking-involved atoms. We applied two distance cutoffs (4 &#x0212b; and 3.6 &#x0212b;) for double evaluation (<a href="#app1-ijms-21-05248" rid="app1-ijms-21-05248" class=" app">Supplementary Figures S1&#x02013;S5</a>). If the atom pair exists, we determined it the contact of the corresponding residue. After locating all residues from highest rank (1) to lowest rank (10), bottom-up order, we counted all contacts at a distance &#x02264;4 &#x0212b; from segmented peptides and summed up the counts. If the summed-up count of a residue was &#x02265;5, the corresponding residue was chosen. The chosen residue contact count sum were placed along the protein target region and compared with the X-ray crystallography determined structure of MYC&#x02013;MAX (PDB ID 1NKP) [<a href="#B13-ijms-21-05248" rid="B13-ijms-21-05248" class=" bibr popnode">13</a>] and with the NMR structure of MBD2&#x02013;p66&#x003b1; (PDB ID 2L2L) [<a href="#B15-ijms-21-05248" rid="B15-ijms-21-05248" class=" bibr popnode">15</a>].</p></div><div id="sec3dot4-ijms-21-05248" class="sec sec-last"><h3 id="sec3dot4-ijms-21-05248title">3.4. MYC 390&#x02013;409 Peptide Molecular Docking with 10058F4 and Chemical Compound Library (NCI Diversity Set V)</h3><p class="p p-first">We utilized National Cancer Institute chemical compound library Diversity set V (<a href="https://dtp.cancer.gov/organization/dscb/obtaining/available_plates.htm">https://dtp.cancer.gov/organization/dscb/obtaining/available_plates.htm</a>) for molecular docking toward MYC 390&#x02013;409 peptide 22 models (<a href="/pmc/articles/PMC7432763/figure/ijms-21-05248-f001/" target="figure" class="fig-table-link figpopup" rid-figpopup="ijms-21-05248-f001" rid-ob="ob-ijms-21-05248-f001" co-legend-rid="lgnd_ijms-21-05248-f001"><span>Figure 1</span></a>I) to compare with 10058F4. It generated about 350,000 docking results, and we obtained 10 groups of each 3000 docking results (sample size) from random sampling and, then, considered each group as a population for the statistic (T score) calculation based on the new docking evaluation scores.</p><p>To see the new score docking-involved atom number correlation, we employed linear regression toward three random samples (sample 1, sample 5, and sample 10) using Microsoft Excel 16.37. For T score calculation, we collected the 10 best 10058F4 docking scores from each random sample (1&#x02013;10), and considered each random sample with/without 10085F4 docking results as the population. Docking-involved atom number groupings into 15&#x02013;17, 16&#x02013;18, and 17&#x02013;19 were pursued, and we collected all docking results from 10 random samples. The 10058F4 docking-involved atom number is 17; thus, we selected the group range in this manner. In this case, each group with/without 10085F4 docking results was considered as a population. Then, we obtained the T scores of the 10 best 10058F4 docking results according to the equation below.
</p><div class="disp-formula" id="FD3-ijms-21-05248"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="mm11"><mrow><mrow><mi>T</mi><mo>=</mo><mfrac><mrow><mover accent="true"><mi>x</mi><mo>&#x000af;</mo></mover><mo>&#x02212;</mo><mi>&#x003bc;</mi></mrow><mrow><mi>S</mi><mo>/</mo><msqrt><mi>n</mi></msqrt></mrow></mfrac></mrow></mrow></math></div><div class="l">(2)</div></div><p>
where <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="mm12"><mrow><mover accent="true"><mi>x</mi><mo>&#x000af;</mo></mover></mrow></math> is the 10 best 10058F4 docking results mean score, <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="mm13"><mrow><mi>&#x003bc;</mi></mrow></math> is the population mean score, <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="mm14"><mrow><mi>S</mi></mrow></math> is the standard deviation of the 10 best 10058F4 docking results, and <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="mm15"><mrow><mrow><msqrt><mi>n</mi></msqrt></mrow></mrow></math> is the square root of the number of 10 best 10058F4 docking results (<math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="mm16"><mrow><mrow><msqrt><mrow><mn>10</mn></mrow></msqrt></mrow></mrow></math>).</p><p class="p p-last">Finally, we compared the 10 best 10058F4 docking results with docking-involved atom number groups (14, 15, 16, 17 without 10058F4 results, 18, 19, and 20) using <em>t</em>-test. For the analysis, again, we collected docking results according to the docking-involved atom number from all of random samples (sample 1&#x02013;10).</p></div></div><div id="sec4-ijms-21-05248" class="tsec sec"><h2 class="head no_bottom_margin" id="sec4-ijms-21-05248title">4. Conclusions</h2><p class="p p-first-last">The target binding region segmentation generated sets of the overlapping 20-amino-acid-long peptides, and diverse structural forms of these peptides were predicted. We used this step of structural diversity mimicking to study the disordered binding sites that undergo binding-induced folding (i.e., the disorder-to-order transition regions). Molecular docking to these peptides and the new evaluation score successfully distinguished already known compounds and their specific binding sites. Furthermore, we discovered that 10058F4 binds specifically to MYC Y402 and polar residues near this key residue. These observations suggest that the MYC&#x02013;10058F4 interaction is mainly driven by polarity. Statistically, the new docking score is correlated with the number of contact atoms due to the normalization inside the new docking evaluation score function. Nevertheless, 10058F4 is distinguished rather well from compounds with similar number of atoms randomly chosen from a chemical compound library, NCI Diversity set V. We applied the same procedure for the analysis of other inhibitors targeting disorder-to-order transition regions, ABA and APC. ABA showed contacts with charged residues of MBD2 and p66&#x003b1; located behind each protein partner-binding site. APC also showed contacts with charged residues behind the p66&#x003b1;-binding site of MBD2, but no specific binding mechanism was discovered from the analysis of the p66&#x003b1; docking. In summary, we mimicked structural heterogeneity of the disorder-to-order transition regions based on their native characteristics and evaluated compound binding status with a new score function. We suggest this method as a novel protein intrinsic disorder utilizing a drug discovery platform and hope that it will expand the field of the computer-aided drug discovery.</p></div><div id="glossary-a.n.e" class="tsec sec"><h2 class="head no_bottom_margin" id="glossary-a.n.etitle">Abbreviations</h2><div class="bk-sec"><table class="array small"><tr><td align="left" valign="middle" rowspan="1" colspan="1">10058F4</td><td align="left" valign="middle" rowspan="1" colspan="1">10058-F4: 5-[(4-Ethylphenyl)methylene]-2-thioxo-4-thiazolidinone, ZINC12406714, PubChem CID: 1271002</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">10074G5</td><td align="left" valign="middle" rowspan="1" colspan="1">10074-G5: 4-nitro-N-(2-phenylphenyl)-2,1,3-benzoxadiazol-7-amine, ZINC3879010, PubChem CID: 2836600</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">ABA</td><td align="left" valign="middle" rowspan="1" colspan="1">2-Amino-N-[[(3S)-2,3-dihydro-1,4-benzodioxin-3-yl]methyl]acetamide, ZINC40430779, PubChem CID: 93602182</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">APC</td><td align="left" valign="bottom" rowspan="1" colspan="1">(R)-3-(2-Amino-acetylamino)-pyrrolidine-1-carboxylic acid tert-butyl ester, ZINC60177071, PubChem CID: 66563909</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">CD</td><td align="left" valign="middle" rowspan="1" colspan="1">Circular dichroism</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">CSA</td><td align="left" valign="middle" rowspan="1" colspan="1">Contact surface area</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">DOPE</td><td align="left" valign="middle" rowspan="1" colspan="1">Discrete optimized protein energy</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">DOT</td><td align="left" valign="middle" rowspan="1" colspan="1">Disorder-to-order transition</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">EM</td><td align="left" valign="bottom" rowspan="1" colspan="1">Electron microscopy</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">IDP</td><td align="left" valign="bottom" rowspan="1" colspan="1">Intrinsically disordered protein</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">IDPR</td><td align="left" valign="middle" rowspan="1" colspan="1">Intrinsically disordered protein region</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">MAX</td><td align="left" valign="middle" rowspan="1" colspan="1">Protein Max</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">MBD2</td><td align="left" valign="middle" rowspan="1" colspan="1">Methyl-CpG-binding domain protein 2</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">MDM2</td><td align="left" valign="middle" rowspan="1" colspan="1">E3 ubiquitin-protein ligase Mdm2</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">MoRF</td><td align="left" valign="middle" rowspan="1" colspan="1">Molecular recognition feature</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">MYC</td><td align="left" valign="middle" rowspan="1" colspan="1">Myc proto-oncogene protein</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">NCI</td><td align="left" valign="middle" rowspan="1" colspan="1">National Cancer Institute</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">NMR</td><td align="left" valign="middle" rowspan="1" colspan="1">Nuclear magnetic resonance</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">p66&#x003b1;</td><td align="left" valign="middle" rowspan="1" colspan="1">Transcriptional repressor p66-alpha</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">RG7112</td><td align="left" valign="middle" rowspan="1" colspan="1">RG-7112: [(4S,5R)-2-(4-tert-butyl-2-ethoxyphenyl)-4,5-bis(4-chlorophenyl)-4,5-dimethylimidazol-1-yl]-[4-<br />(3-methylsulfonyl propyl)piperazin-1-yl]methanone, ZINC96270381, PubChem CID: 57406853</td></tr></table></div></div><div id="app1-ijms-21-05248" class="tsec sec"><h2 class="head no_bottom_margin" id="app1-ijms-21-05248title">Supplementary Materials</h2><!--/article/back/app-group/app/--><p>Supplementary materials can be found at <a href="https://www.mdpi.com/1422-0067/21/15/5248/s1">https://www.mdpi.com/1422-0067/21/15/5248/s1</a>. Figure S1. MYC binding region peptide docking result contact mapping, Figure S2. MBD2 binding region peptide docking result contact mapping, Figure S3. p66&#x003b1; binding region peptide docking result contact mapping, Figure S4. APC-focused MBD2 binding region peptide docking result contact mapping, Figure S5. APC-focused p66&#x003b1; binding region peptide docking result contact mapping, Figure S6. APC-focused docking score comparison between APC contact residue mapping result and the other 10 best peptide docking results (MBD2, 4 peptides; p66&#x003b1;, 5 peptides) without APC (wo APC).</p><div class="sec suppmat" id="ijms-21-05248-s001"><div class="sup-box half_rhythm" id="media-a.n.a.a.c.a"><a href="/pmc/articles/PMC7432763/bin/ijms-21-05248-s001.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(1.5M, pdf)</sup></div></div></div><div id="notes-a.n.b" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.n.btitle">Author Contributions</h2><p>Conceptualization, I.N. and C.G.K.; methodology, I.N., S.C., and S.H.S.; validation, I.N. and S.C.; formal analysis, I.N.; investigation, I.N.; resources, C.G.K.; data curation, I.N.; writing&#x02014;original draft preparation, I.N.; writing&#x02014;review and editing, I.N., V.N.U., and C.G.K.; visualization, I.N.; supervision, V.N.U. and C.G.K.; project administration, C.G.K.; funding acquisition, C.G.K. All authors have read and agreed to the published version of the manuscript.</p></div><div id="notes-a.n.c" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.n.ctitle">Funding</h2><p>This work was supported by the Basic Science Research Program (NRF-2020R1A2C2009112 to C.G.K.) and the Bio &#x00026; Medical Technology Development program (NRF-2017M3A9C8027975 and NRF-2020M3A9G4021041 to C.G.K), National Research Foundation (NRF), Republic of Korea.</p></div><div id="notes-a.n.d" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.n.dtitle">Conflicts of Interest</h2><p>The authors declare no conflict of interest.</p></div><div id="ref-list-a.n.f" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.n.ftitle">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1-ijms-21-05248">1. <span class="element-citation">Lin X., Li X., Lin X. A Review on applications of computational methods in drug screening and design. <span><span class="ref-journal">Molecules. </span>2020;<span class="ref-vol">25</span>:1375.  doi: 10.3390/molecules25061375.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7144386/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32197324" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fmolecules25061375" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=A+Review+on+applications+of+computational+methods+in+drug+screening+and+design&amp;author=X.+Lin&amp;author=X.+Li&amp;author=X.+Lin&amp;volume=25&amp;publication_year=2020&amp;pages=1375&amp;pmid=32197324&amp;doi=10.3390/molecules25061375&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B2-ijms-21-05248">2. <span class="element-citation">Uversky V.N., Oldfield C.J., Dunker A.K. Intrinsically disordered proteins in human diseases: Introducing the D2 concept. <span><span class="ref-journal">Annu. Rev. Biophys. </span>2008;<span class="ref-vol">37</span>:215–246. doi: 10.1146/annurev.biophys.37.032807.125924.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18573080" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1146%2Fannurev.biophys.37.032807.125924" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Biophys.&amp;title=Intrinsically+disordered+proteins+in+human+diseases:+Introducing+the+D2+concept&amp;author=V.N.+Uversky&amp;author=C.J.+Oldfield&amp;author=A.K.+Dunker&amp;volume=37&amp;publication_year=2008&amp;pages=215-246&amp;pmid=18573080&amp;doi=10.1146/annurev.biophys.37.032807.125924&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B3-ijms-21-05248">3. <span class="element-citation">Babu M.M., Van Der Lee R., De Groot N.S., Gsponer J. Intrinsically disordered proteins: Regulation and disease. <span><span class="ref-journal">Curr. Opin. Struct. Boil. </span>2011;<span class="ref-vol">21</span>:432–440. doi: 10.1016/j.sbi.2011.03.011.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21514144" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.sbi.2011.03.011" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Struct.+Boil.&amp;title=Intrinsically+disordered+proteins:+Regulation+and+disease&amp;author=M.M.+Babu&amp;author=R.+Van+Der+Lee&amp;author=N.S.+De+Groot&amp;author=J.+Gsponer&amp;volume=21&amp;publication_year=2011&amp;pages=432-440&amp;doi=10.1016/j.sbi.2011.03.011&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B4-ijms-21-05248">4. <span class="element-citation">Uversky V. p53 Proteoforms and Intrinsic Disorder: An Illustration of the Protein Structure&#x02013;Function Continuum Concept. <span><span class="ref-journal">Int. J. Mol. Sci. </span>2016;<span class="ref-vol">17</span>:1874.  doi: 10.3390/ijms17111874.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5133874/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27834926" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fijms17111874" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Mol.+Sci.&amp;title=p53+Proteoforms+and+Intrinsic+Disorder:+An+Illustration+of+the+Protein+Structure&#x02013;Function+Continuum+Concept&amp;author=V.+Uversky&amp;volume=17&amp;publication_year=2016&amp;pages=1874&amp;pmid=27834926&amp;doi=10.3390/ijms17111874&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B5-ijms-21-05248">5. <span class="element-citation">Uversky V.N. Intrinsically Disordered Proteins and Their &#x0201c;Mysterious&#x0201d; (Meta)Physics. <span><span class="ref-journal">Front. Phys. </span>2019;<span class="ref-vol">7</span>:10. doi: 10.3389/fphy.2019.00010.</span> [<a href="//doi.org/10.3389%2Ffphy.2019.00010" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Phys.&amp;title=Intrinsically+Disordered+Proteins+and+Their+&#x0201c;Mysterious&#x0201d;+(Meta)Physics&amp;author=V.N.+Uversky&amp;volume=7&amp;publication_year=2019&amp;pages=10&amp;doi=10.3389/fphy.2019.00010&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B6-ijms-21-05248">6. <span class="element-citation">Romero P., Obradovic Z., Kissinger C.R., E Villafranca J., Garner E., Guilliot S., Dunker A. Thousands of proteins likely to have long disordered regions. <span><span class="ref-journal">Pac. Symp. Biocomput. Pac. Symp. Biocomput. </span>1998:437–448.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9697202" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pac.+Symp.+Biocomput.+Pac.+Symp.+Biocomput.&amp;title=Thousands+of+proteins+likely+to+have+long+disordered+regions&amp;author=P.+Romero&amp;author=Z.+Obradovic&amp;author=C.R.+Kissinger&amp;author=J.+E+Villafranca&amp;author=E.+Garner&amp;publication_year=1998&amp;pages=437-448&amp;pmid=9697202&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B7-ijms-21-05248">7. <span class="element-citation">Garner E., Romero P., Dunker A.K., Brown C., Obradovic Z. Predicting Binding Regions within Disordered Proteins. <span><span class="ref-journal">Genome Inform. Ser. Workshop Genome Inform. </span>1999;<span class="ref-vol">10</span>:41–50.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11072341" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Inform.+Ser.+Workshop+Genome+Inform.&amp;title=Predicting+Binding+Regions+within+Disordered+Proteins&amp;author=E.+Garner&amp;author=P.+Romero&amp;author=A.K.+Dunker&amp;author=C.+Brown&amp;author=Z.+Obradovic&amp;volume=10&amp;publication_year=1999&amp;pages=41-50&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B8-ijms-21-05248">8. <span class="element-citation">Oldfield C.J., Cheng Y., Cortese M.S., Romero P., Uversky V.N., Dunker A.K. Coupled folding and binding with &#x003b1;-helix-forming molecular recognition elements. <span><span class="ref-journal">Biochemistry. </span>2005;<span class="ref-vol">44</span>:12454–12470. doi: 10.1021/bi050736e.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16156658" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fbi050736e" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Coupled+folding+and+binding+with+&#x003b1;-helix-forming+molecular+recognition+elements&amp;author=C.J.+Oldfield&amp;author=Y.+Cheng&amp;author=M.S.+Cortese&amp;author=P.+Romero&amp;author=V.N.+Uversky&amp;volume=44&amp;publication_year=2005&amp;pages=12454-12470&amp;pmid=16156658&amp;doi=10.1021/bi050736e&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B9-ijms-21-05248">9. <span class="element-citation">Mohan A., Oldfield C.J., Radivojac P., Vacic V., Cortese M.S., Dunker A.K., Uversky V.N. Analysis of Molecular Recognition Features (MoRFs) <span><span class="ref-journal">J. Mol. Boil. </span>2006;<span class="ref-vol">362</span>:1043–1059. doi: 10.1016/j.jmb.2006.07.087.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16935303" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.jmb.2006.07.087" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Mol.+Boil.&amp;title=Analysis+of+Molecular+Recognition+Features+(MoRFs)&amp;author=A.+Mohan&amp;author=C.J.+Oldfield&amp;author=P.+Radivojac&amp;author=V.+Vacic&amp;author=M.S.+Cortese&amp;volume=362&amp;publication_year=2006&amp;pages=1043-1059&amp;doi=10.1016/j.jmb.2006.07.087&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B10-ijms-21-05248">10. <span class="element-citation">Yin X., Giap C., Lazo J.S., Prochownik E.V. Low molecular weight inhibitors of Myc&#x02013;Max interaction and function. <span><span class="ref-journal">Oncogene. </span>2003;<span class="ref-vol">22</span>:6151–6159. doi: 10.1038/sj.onc.1206641.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/13679853" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fsj.onc.1206641" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Low+molecular+weight+inhibitors+of+Myc&#x02013;Max+interaction+and+function&amp;author=X.+Yin&amp;author=C.+Giap&amp;author=J.S.+Lazo&amp;author=E.V.+Prochownik&amp;volume=22&amp;publication_year=2003&amp;pages=6151-6159&amp;pmid=13679853&amp;doi=10.1038/sj.onc.1206641&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B11-ijms-21-05248">11. <span class="element-citation">Follis A.V., Hammoudeh D.I., Wang H., Prochownik E.V., Metallo S.J. structural rationale for the coupled binding and unfolding of the c-Myc oncoprotein by small molecules. <span><span class="ref-journal">Chem. Boil. </span>2008;<span class="ref-vol">15</span>:1149–1155. doi: 10.1016/j.chembiol.2008.09.011.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19022175" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.chembiol.2008.09.011" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem.+Boil.&amp;title=structural+rationale+for+the+coupled+binding+and+unfolding+of+the+c-Myc+oncoprotein+by+small+molecules&amp;author=A.V.+Follis&amp;author=D.I.+Hammoudeh&amp;author=H.+Wang&amp;author=E.V.+Prochownik&amp;author=S.J.+Metallo&amp;volume=15&amp;publication_year=2008&amp;pages=1149-1155&amp;pmid=19022175&amp;doi=10.1016/j.chembiol.2008.09.011&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B12-ijms-21-05248">12. <span class="element-citation">Kim M.Y., Na I., Kim J.S., Son S.H., Choi S., Lee S.E., Kim J.-H., Jang K., Alterovitz G., Chen Y., et al. Rational discovery of antimetastatic agents targeting the intrinsically disordered region of MBD2. <span><span class="ref-journal">Sci. Adv. </span>2019;<span class="ref-vol">5</span>:eaav9810. doi: 10.1126/sciadv.aav9810.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6867884/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31799386" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fsciadv.aav9810" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Adv.&amp;title=Rational+discovery+of+antimetastatic+agents+targeting+the+intrinsically+disordered+region+of+MBD2&amp;author=M.Y.+Kim&amp;author=I.+Na&amp;author=J.S.+Kim&amp;author=S.H.+Son&amp;author=S.+Choi&amp;volume=5&amp;publication_year=2019&amp;pages=eaav9810&amp;pmid=31799386&amp;doi=10.1126/sciadv.aav9810&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B13-ijms-21-05248">13. <span class="element-citation">Nair S.K., Burley S.K. X-ray structures of Myc-Max and Mad-Max recognizing DNA. Molecular bases of regulation by proto-oncogenic transcription factors. <span><span class="ref-journal">Cell. </span>2003;<span class="ref-vol">112</span>:193–205. doi: 10.1016/S0092-8674(02)01284-9.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12553908" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FS0092-8674(02)01284-9" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=X-ray+structures+of+Myc-Max+and+Mad-Max+recognizing+DNA.+Molecular+bases+of+regulation+by+proto-oncogenic+transcription+factors&amp;author=S.K.+Nair&amp;author=S.K.+Burley&amp;volume=112&amp;publication_year=2003&amp;pages=193-205&amp;pmid=12553908&amp;doi=10.1016/S0092-8674(02)01284-9&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B14-ijms-21-05248">14. <span class="element-citation">Panova S., Cliff M., Macek P., Blackledge M., Jensen M.R., Nissink J.W.M., Embrey K.J., Davies R., Waltho J.P. Mapping hidden residual structure within the Myc bHLH-LZ domain using chemical denaturant titration. <span><span class="ref-journal">Structure. </span>2019;<span class="ref-vol">27</span>:1537–1546. doi: 10.1016/j.str.2019.07.006.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31402220" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.str.2019.07.006" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=Mapping+hidden+residual+structure+within+the+Myc+bHLH-LZ+domain+using+chemical+denaturant+titration&amp;author=S.+Panova&amp;author=M.+Cliff&amp;author=P.+Macek&amp;author=M.+Blackledge&amp;author=M.R.+Jensen&amp;volume=27&amp;publication_year=2019&amp;pages=1537-1546&amp;pmid=31402220&amp;doi=10.1016/j.str.2019.07.006&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B15-ijms-21-05248">15. <span class="element-citation">Gnanapragasam M.N., Scarsdale J.N., Amaya M.L., Webb H.D., Desai M.A., Walavalkar N.M., Wang S.Z., Zu Zhu S., Ginder G.D., Williams J.D.C. p66 -MBD2 coiled-coil interaction and recruitment of Mi-2 are critical for globin gene silencing by the MBD2-NuRD complex. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>2011;<span class="ref-vol">108</span>:7487–7492. doi: 10.1073/pnas.1015341108.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3088629/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21490301" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.1015341108" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=p66+-MBD2+coiled-coil+interaction+and+recruitment+of+Mi-2+are+critical+for+globin+gene+silencing+by+the+MBD2-NuRD+complex&amp;author=M.N.+Gnanapragasam&amp;author=J.N.+Scarsdale&amp;author=M.L.+Amaya&amp;author=H.D.+Webb&amp;author=M.A.+Desai&amp;volume=108&amp;publication_year=2011&amp;pages=7487-7492&amp;pmid=21490301&amp;doi=10.1073/pnas.1015341108&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B16-ijms-21-05248">16. <span class="element-citation">Lamiable A., Thevenet P., Rey J., Vavrusa M., Derreumaux P., Tuff&#x000e9;ry P. PEP-FOLD3: Faster de novo structure prediction for linear peptides in solution and in complex. <span><span class="ref-journal">Nucleic Acids Res. </span>2016;<span class="ref-vol">44</span>:W449–W454. doi: 10.1093/nar/gkw329.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4987898/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27131374" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkw329" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=PEP-FOLD3:+Faster+de+novo+structure+prediction+for+linear+peptides+in+solution+and+in+complex&amp;author=A.+Lamiable&amp;author=P.+Thevenet&amp;author=J.+Rey&amp;author=M.+Vavrusa&amp;author=P.+Derreumaux&amp;volume=44&amp;publication_year=2016&amp;pages=W449-W454&amp;pmid=27131374&amp;doi=10.1093/nar/gkw329&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B17-ijms-21-05248">17. <span class="element-citation">Michel J., Cuchillo R. The impact of small molecule binding on the energy landscape of the intrinsically disordered protein C-Myc. <span><span class="ref-journal">PLoS ONE. </span>2012;<span class="ref-vol">7</span>:e41070.  doi: 10.1371/journal.pone.0041070.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3397933/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22815918" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pone.0041070" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=The+impact+of+small+molecule+binding+on+the+energy+landscape+of+the+intrinsically+disordered+protein+C-Myc&amp;author=J.+Michel&amp;author=R.+Cuchillo&amp;volume=7&amp;publication_year=2012&amp;pages=e41070&amp;pmid=22815918&amp;doi=10.1371/journal.pone.0041070&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B18-ijms-21-05248">18. <span class="element-citation">Heller G.T., Aprile F.A., Bonomi M., Camilloni C., De Simone A., Vendruscolo M. sequence specificity in the entropy-driven binding of a small molecule and a disordered peptide. <span><span class="ref-journal">J. Mol. Boil. </span>2017;<span class="ref-vol">429</span>:2772–2779. doi: 10.1016/j.jmb.2017.07.016.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28743590" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.jmb.2017.07.016" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Mol.+Boil.&amp;title=sequence+specificity+in+the+entropy-driven+binding+of+a+small+molecule+and+a+disordered+peptide&amp;author=G.T.+Heller&amp;author=F.A.+Aprile&amp;author=M.+Bonomi&amp;author=C.+Camilloni&amp;author=A.+De+Simone&amp;volume=429&amp;publication_year=2017&amp;pages=2772-2779&amp;doi=10.1016/j.jmb.2017.07.016&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B19-ijms-21-05248">19. <span class="element-citation">Vassilev L.T., Vu B., Graves B., Carvajal D., Podlaski F., Filipovic Z., Kong N., Kammlott U., Lukacs C., Klein C., et al. <em>In vivo</em> activation of the p53 pathway by small-molecule antagonists of MDM2. <span><span class="ref-journal">Science. </span>2004;<span class="ref-vol">303</span>:844–848. doi: 10.1126/science.1092472.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14704432" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.1092472" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=In+vivo+activation+of+the+p53+pathway+by+small-molecule+antagonists+of+MDM2&amp;author=L.T.+Vassilev&amp;author=B.+Vu&amp;author=B.+Graves&amp;author=D.+Carvajal&amp;author=F.+Podlaski&amp;volume=303&amp;publication_year=2004&amp;pages=844-848&amp;pmid=14704432&amp;doi=10.1126/science.1092472&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B20-ijms-21-05248">20. <span class="element-citation">Vu B., Wovkulich P., Pizzolato G., Lovey A., Ding Q., Jiang N., Liu J.-J., Zhao C., Glenn K., Wen Y., et al. discovery of RG7112: A small-molecule MDM2 inhibitor in clinical development. <span><span class="ref-journal">ACS Med. Chem. Lett. </span>2013;<span class="ref-vol">4</span>:466–469. doi: 10.1021/ml4000657.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4027145/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24900694" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fml4000657" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=ACS+Med.+Chem.+Lett.&amp;title=discovery+of+RG7112:+A+small-molecule+MDM2+inhibitor+in+clinical+development&amp;author=B.+Vu&amp;author=P.+Wovkulich&amp;author=G.+Pizzolato&amp;author=A.+Lovey&amp;author=Q.+Ding&amp;volume=4&amp;publication_year=2013&amp;pages=466-469&amp;pmid=24900694&amp;doi=10.1021/ml4000657&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B21-ijms-21-05248">21. <span class="element-citation">Tovar C., Graves B., Packman K., Filipovic Z., Xia B.H.M., Tardell C., Garrido R., Lee E., Kolinsky K., To K.-H., et al. MDM2 small-molecule antagonist RG7112 activates p53 signaling and regresses human tumors in preclinical cancer models. <span><span class="ref-journal">Cancer Res. </span>2013;<span class="ref-vol">73</span>:2587–2597. doi: 10.1158/0008-5472.CAN-12-2807.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23400593" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1158%2F0008-5472.CAN-12-2807" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res.&amp;title=MDM2+small-molecule+antagonist+RG7112+activates+p53+signaling+and+regresses+human+tumors+in+preclinical+cancer+models&amp;author=C.+Tovar&amp;author=B.+Graves&amp;author=K.+Packman&amp;author=Z.+Filipovic&amp;author=B.H.M.+Xia&amp;volume=73&amp;publication_year=2013&amp;pages=2587-2597&amp;pmid=23400593&amp;doi=10.1158/0008-5472.CAN-12-2807&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B22-ijms-21-05248">22. <span class="element-citation">Neira J.L., Bintz J., Arruebo M., Rizzuti B., Bonacci T., Vega S., Lanas A., Velazquez-Campoy A., Iovanna J., Abi&#x000e1;n O. Identification of a drug targeting an intrinsically disordered protein involved in pancreatic adenocarcinoma. <span><span class="ref-journal">Sci. Rep. </span>2017;<span class="ref-vol">7</span>:39732. doi: 10.1038/srep39732.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5213423/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28054562" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fsrep39732" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Rep.&amp;title=Identification+of+a+drug+targeting+an+intrinsically+disordered+protein+involved+in+pancreatic+adenocarcinoma&amp;author=J.L.+Neira&amp;author=J.+Bintz&amp;author=M.+Arruebo&amp;author=B.+Rizzuti&amp;author=T.+Bonacci&amp;volume=7&amp;publication_year=2017&amp;pages=39732&amp;pmid=28054562&amp;doi=10.1038/srep39732&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B23-ijms-21-05248">23. <span class="element-citation">Santofimia-Casta&#x000f1;o P., Rizzuti B., Xia Y., Abian O., Peng L., Vel&#x000e1;zquez-Campoy A., Neira J.L., Iovanna J. Targeting intrinsically disordered proteins involved in cancer. <span><span class="ref-journal">Cell. Mol. Life Sci. </span>2019;<span class="ref-vol">77</span>:1695–1707. doi: 10.1007/s00018-019-03347-3.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7190594/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31667555" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs00018-019-03347-3" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell.+Mol.+Life+Sci.&amp;title=Targeting+intrinsically+disordered+proteins+involved+in+cancer&amp;author=P.+Santofimia-Casta&#x000f1;o&amp;author=B.+Rizzuti&amp;author=Y.+Xia&amp;author=O.+Abian&amp;author=L.+Peng&amp;volume=77&amp;publication_year=2019&amp;pages=1695-1707&amp;pmid=31667555&amp;doi=10.1007/s00018-019-03347-3&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B24-ijms-21-05248">24. <span class="element-citation">Keiser M.J., Setola V., Irwin J.J., Laggner C., Abbas A.I., Hufeisen S.J., Jensen N.H., Kuijer M.B., Matos R.C., Tran T.B., et al. Predicting new molecular targets for known drugs. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">462</span>:175–181. doi: 10.1038/nature08506.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2784146/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19881490" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnature08506" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Predicting+new+molecular+targets+for+known+drugs&amp;author=M.J.+Keiser&amp;author=V.+Setola&amp;author=J.J.+Irwin&amp;author=C.+Laggner&amp;author=A.I.+Abbas&amp;volume=462&amp;publication_year=2009&amp;pages=175-181&amp;pmid=19881490&amp;doi=10.1038/nature08506&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B25-ijms-21-05248">25. <span class="element-citation">Sluysmans D., Willet N., Th&#x000e9;venot J., Lecommandoux S., Duwez A.-S. Single-molecule mechanical unfolding experiments reveal a critical length for the formation of &#x003b1;-helices in peptides. <span><span class="ref-journal">Nanoscale Horizons. </span>2020;<span class="ref-vol">5</span>:671–678. doi: 10.1039/D0NH00036A.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32226978" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1039%2FD0NH00036A" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nanoscale+Horizons&amp;title=Single-molecule+mechanical+unfolding+experiments+reveal+a+critical+length+for+the+formation+of+&#x003b1;-helices+in+peptides&amp;author=D.+Sluysmans&amp;author=N.+Willet&amp;author=J.+Th&#x000e9;venot&amp;author=S.+Lecommandoux&amp;author=A.-S.+Duwez&amp;volume=5&amp;publication_year=2020&amp;pages=671-678&amp;pmid=32226978&amp;doi=10.1039/D0NH00036A&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B26-ijms-21-05248">26. <span class="element-citation">Trott O., Olson A.J. AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading. <span><span class="ref-journal">J. Comput. Chem. </span>2009;<span class="ref-vol">31</span>:455–461. doi: 10.1002/jcc.21334.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3041641/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19499576" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fjcc.21334" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Comput.+Chem.&amp;title=AutoDock+Vina:+Improving+the+speed+and+accuracy+of+docking+with+a+new+scoring+function,+efficient+optimization,+and+multithreading&amp;author=O.+Trott&amp;author=A.J.+Olson&amp;volume=31&amp;publication_year=2009&amp;pages=455-461&amp;doi=10.1002/jcc.21334&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B27-ijms-21-05248">27. <span class="element-citation">Koes D.R., Baumgartner M.P., Camacho C.J. Lessons learned in empirical scoring with smina from the CSAR 2011 Benchmarking exercise. <span><span class="ref-journal">J. Chem. Inf. Model. </span>2013;<span class="ref-vol">53</span>:1893–1904. doi: 10.1021/ci300604z.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3726561/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23379370" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fci300604z" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Chem.+Inf.+Model.&amp;title=Lessons+learned+in+empirical+scoring+with+smina+from+the+CSAR+2011+Benchmarking+exercise&amp;author=D.R.+Koes&amp;author=M.P.+Baumgartner&amp;author=C.J.+Camacho&amp;volume=53&amp;publication_year=2013&amp;pages=1893-1904&amp;pmid=23379370&amp;doi=10.1021/ci300604z&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B28-ijms-21-05248">28. <span class="element-citation">Luo Q., Zhao L., Hu J., Jin H., Liu Z., Zhang L. The scoring bias in reverse docking and the score normalization strategy to improve success rate of target fishing. <span><span class="ref-journal">PLoS ONE. </span>2017;<span class="ref-vol">12</span>:e0171433.  doi: 10.1371/journal.pone.0171433.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5308821/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28196116" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pone.0171433" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=The+scoring+bias+in+reverse+docking+and+the+score+normalization+strategy+to+improve+success+rate+of+target+fishing&amp;author=Q.+Luo&amp;author=L.+Zhao&amp;author=J.+Hu&amp;author=H.+Jin&amp;author=Z.+Liu&amp;volume=12&amp;publication_year=2017&amp;pages=e0171433&amp;pmid=28196116&amp;doi=10.1371/journal.pone.0171433&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B29-ijms-21-05248">29. <span class="element-citation">Ribeiro J., R&#x000ed;os-Vera C., Melo F., Sch&#x000fc;ller A. Calculation of accurate interatomic contact surface areas for the quantitative analysis of non-bonded molecular interactions. <span><span class="ref-journal">Bioinformatics. </span>2019;<span class="ref-vol">35</span>:3499–3501. doi: 10.1093/bioinformatics/btz062.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6748739/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30698657" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtz062" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Calculation+of+accurate+interatomic+contact+surface+areas+for+the+quantitative+analysis+of+non-bonded+molecular+interactions&amp;author=J.+Ribeiro&amp;author=C.+R&#x000ed;os-Vera&amp;author=F.+Melo&amp;author=A.+Sch&#x000fc;ller&amp;volume=35&amp;publication_year=2019&amp;pages=3499-3501&amp;pmid=30698657&amp;doi=10.1093/bioinformatics/btz062&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B30-ijms-21-05248">30. <span class="element-citation">Shen M.-Y., Sali A. Statistical potential for assessment and prediction of protein structures. <span><span class="ref-journal">Protein Sci. </span>2006;<span class="ref-vol">15</span>:2507–2524. doi: 10.1110/ps.062416606.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2242414/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17075131" ref="reftype=pubmed&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1110%2Fps.062416606" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Sci.&amp;title=Statistical+potential+for+assessment+and+prediction+of+protein+structures&amp;author=M.-Y.+Shen&amp;author=A.+Sali&amp;volume=15&amp;publication_year=2006&amp;pages=2507-2524&amp;pmid=17075131&amp;doi=10.1110/ps.062416606&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=7432763&amp;issue-id=363503&amp;journal-id=808&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">International Journal of Molecular Sciences</span> are provided here courtesy of <strong>Multidisciplinary Digital Publishing Institute  (MDPI)</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC7432763/pdf/ijms-21-05248.pdf" class="int-view">PDF (5.3M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/7432763/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/7432763/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="7432763" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC7432763%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="mAqJqk1p3aIlKLk1W2AKVRv1Gurt6d8hsaShjunWFQ0isOvhFzaz5bXTCR3WK01B">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC7432763%2F&amp;text=Drug%20Discovery%20Targeting%20the%20Disorder-To-Order%20Transition%20Regions%20through%20the%20Conformational%20Diversity%20Mimicking%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC7432763%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7432763/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC7432763/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/32722024/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC7432763/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/32722024/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC7432763/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/7432763/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript" src="/core/mathjax/2.7.9/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC7432763/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
