
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Programming a Ferroptosis&#8208;to&#8208;Apoptosis Transition Landscape Revealed Ferroptosis Biomarkers and Repressors for Cancer Therapy - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="D0BD41E4AC192A45000059A94ACB2551.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="advsci" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC11077643/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11077643/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Advanced Science" /><meta name="citation_title" content="Programming a Ferroptosis&#x02010;to&#x02010;Apoptosis Transition Landscape Revealed Ferroptosis Biomarkers and Repressors for Cancer Therapy" /><meta name="citation_author" content="Yaron Vinik" /><meta name="citation_author_institution" content="Molecular Cell Biology Department, Weizmann Institute of Science, Rehovot 76100 Israel" /><meta name="citation_author" content="Avi Maimon" /><meta name="citation_author_institution" content="Molecular Cell Biology Department, Weizmann Institute of Science, Rehovot 76100 Israel" /><meta name="citation_author" content="Vinay Dubey" /><meta name="citation_author_institution" content="Molecular Cell Biology Department, Weizmann Institute of Science, Rehovot 76100 Israel" /><meta name="citation_author" content="Harsha Raj" /><meta name="citation_author_institution" content="Molecular Cell Biology Department, Weizmann Institute of Science, Rehovot 76100 Israel" /><meta name="citation_author" content="Ifat Abramovitch" /><meta name="citation_author_institution" content="The Ruth and Bruce Rappaport, Faculty of Medicine, Technion&#x02013;Israel Institute of Technology, Haifa 3525433 Israel" /><meta name="citation_author" content="Sergey Malitsky" /><meta name="citation_author_institution" content="Metabolic Profiling Unit, Weizmann Institute of Science, Rehovot 76100 Israel" /><meta name="citation_author" content="Maxim Itkin" /><meta name="citation_author_institution" content="Metabolic Profiling Unit, Weizmann Institute of Science, Rehovot 76100 Israel" /><meta name="citation_author" content="Avi Ma'ayan" /><meta name="citation_author_institution" content="Department of Pharmacological Sciences, Mount Sinai Center for Bioinformatics, Icahn School of Medicine at Mount Sinai, New York NY, 10029 USA" /><meta name="citation_author" content="Frank Westermann" /><meta name="citation_author_institution" content="Neuroblastoma Genomics, German Cancer Research Center (DKFZ), 69120 Heidelberg Germany" /><meta name="citation_author" content="Eyal Gottlieb" /><meta name="citation_author_institution" content="The Ruth and Bruce Rappaport, Faculty of Medicine, Technion&#x02013;Israel Institute of Technology, Haifa 3525433 Israel" /><meta name="citation_author" content="Eytan Ruppin" /><meta name="citation_author_institution" content="Cancer Data Science Laboratory, National Cancer Institute, National Institutes of Health, Bethesda MD, 20892 USA" /><meta name="citation_author" content="Sima Lev" /><meta name="citation_author_institution" content="Molecular Cell Biology Department, Weizmann Institute of Science, Rehovot 76100 Israel" /><meta name="citation_publication_date" content="2024/05" /><meta name="citation_issue" content="17" /><meta name="citation_volume" content="11" /><meta name="citation_doi" content="10.1002/advs.202307263" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC11077643/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC11077643/" /><meta name="citation_pmid" content="38441406" /><meta name="DC.Title" content="Programming a Ferroptosis&#x02010;to&#x02010;Apoptosis Transition Landscape Revealed Ferroptosis Biomarkers and Repressors for Cancer Therapy" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Wiley" /><meta name="DC.Contributor" content="Yaron Vinik" /><meta name="DC.Contributor" content="Avi Maimon" /><meta name="DC.Contributor" content="Vinay Dubey" /><meta name="DC.Contributor" content="Harsha Raj" /><meta name="DC.Contributor" content="Ifat Abramovitch" /><meta name="DC.Contributor" content="Sergey Malitsky" /><meta name="DC.Contributor" content="Maxim Itkin" /><meta name="DC.Contributor" content="Avi Ma'ayan" /><meta name="DC.Contributor" content="Frank Westermann" /><meta name="DC.Contributor" content="Eyal Gottlieb" /><meta name="DC.Contributor" content="Eytan Ruppin" /><meta name="DC.Contributor" content="Sima Lev" /><meta name="DC.Date" content="2024 May" /><meta name="DC.Identifier" content="10.1002/advs.202307263" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Programming a Ferroptosis&#x02010;to&#x02010;Apoptosis Transition Landscape Revealed Ferroptosis Biomarkers and Repressors for Cancer Therapy" /><meta property="og:type" content="article" /><meta property="og:description" content="Ferroptosis and apoptosis are key cell&#x02010;death pathways implicated in several human diseases including cancer. Ferroptosis is driven by iron&#x02010;dependent lipid peroxidation and currently has no characteristic biomarkers or gene signatures. ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11077643/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11077643/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/2933/">Adv Sci (Weinh)</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/461958/">v.11(17); 2024 May</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11077643
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11077643/pdf/ADVS-11-2307263.pdf" class="int-view">PDF (18M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11077643/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11077643/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11077643" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11077643%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="hWjfpGKw7y1a1EAJhL71lTWqVnMggibZIp0GAw48s6ODPOy3cfeciku6sONzVUcq">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11077643%2F&amp;text=Programming%20a%20Ferroptosis%02010to%02010Apoptosis%20Transition%20Landscape%20Revealed%20Ferroptosis%20Biomarkers%20and%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11077643%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11077643/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11077643/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38441406/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11077643/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38441406/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11077643/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11077643/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/2933/">Adv Sci (Weinh)</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/461958/">v.11(17); 2024 May</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11077643
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-advsci.gif" alt="Logo of advsci" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://onlinelibrary.wiley.com/journal/10.1002/%28ISSN%292198-3844" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Adv Sci (Weinh).</span> 2024 May; 11(17): 2307263. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2024 Mar 5. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1002%2Fadvs.202307263" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1002/advs.202307263</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC11077643</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38441406">38441406</a></div></div></div><h1 class="content-title">Programming a Ferroptosis&#x02010;to&#x02010;Apoptosis Transition Landscape Revealed Ferroptosis Biomarkers and Repressors for Cancer Therapy</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Vinik%20Y%5BAuthor%5D" id="advs7707-cr-0001" class="affpopup" co-rid="_co_idm140165947014688" co-class="co-affbox">Yaron Vinik</a>,<sup>
1
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Maimon%20A%5BAuthor%5D" id="advs7707-cr-0002" class="affpopup" co-rid="_co_idm140165947012080" co-class="co-affbox">Avi Maimon</a>,<sup>
1
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Dubey%20V%5BAuthor%5D" id="advs7707-cr-0003" class="affpopup" co-rid="_co_idm140165947009472" co-class="co-affbox">Vinay Dubey</a>,<sup>
1
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Raj%20H%5BAuthor%5D" id="advs7707-cr-0004" class="affpopup" co-rid="_co_idm140165947006864" co-class="co-affbox">Harsha Raj</a>,<sup>
1
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Abramovitch%20I%5BAuthor%5D" id="advs7707-cr-0005" class="affpopup" co-rid="_co_idm140165947004256" co-class="co-affbox">Ifat Abramovitch</a>,<sup>
2
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Malitsky%20S%5BAuthor%5D" id="advs7707-cr-0006" class="affpopup" co-rid="_co_idm140165947001648" co-class="co-affbox">Sergey Malitsky</a>,<sup>
3
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Itkin%20M%5BAuthor%5D" id="advs7707-cr-0007" class="affpopup" co-rid="_co_idm140165990862736" co-class="co-affbox">Maxim Itkin</a>,<sup>
3
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ma%27ayan%20A%5BAuthor%5D" id="advs7707-cr-0008" class="affpopup" co-rid="_co_idm140165990860128" co-class="co-affbox">Avi Ma'ayan</a>,<sup>
4
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Westermann%20F%5BAuthor%5D" id="advs7707-cr-0009" class="affpopup" co-rid="_co_idm140165990857520" co-class="co-affbox">Frank Westermann</a>,<sup>
5
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gottlieb%20E%5BAuthor%5D" id="advs7707-cr-0010" class="affpopup" co-rid="_co_idm140165990854912" co-class="co-affbox">Eyal Gottlieb</a>,<sup>
2
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ruppin%20E%5BAuthor%5D" id="advs7707-cr-0011" class="affpopup" co-rid="_co_idm140165990852304" co-class="co-affbox">Eytan Ruppin</a>,<sup>
6
</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Lev%20S%5BAuthor%5D" id="advs7707-cr-0012" class="affpopup" co-rid="_co_idm140165990849696" co-class="co-affbox">Sima Lev</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup>
1
</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140165947014688"><h3 class="no_margin">Yaron Vinik</h3><p>
<sup>
1
</sup>
Molecular Cell Biology Department, 
Weizmann Institute of Science, 
Rehovot
76100
Israel
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Vinik%20Y%5BAuthor%5D">Yaron Vinik</a></div></div><div id="_co_idm140165947012080"><h3 class="no_margin">Avi Maimon</h3><p>
<sup>
1
</sup>
Molecular Cell Biology Department, 
Weizmann Institute of Science, 
Rehovot
76100
Israel
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Maimon%20A%5BAuthor%5D">Avi Maimon</a></div></div><div id="_co_idm140165947009472"><h3 class="no_margin">Vinay Dubey</h3><p>
<sup>
1
</sup>
Molecular Cell Biology Department, 
Weizmann Institute of Science, 
Rehovot
76100
Israel
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Dubey%20V%5BAuthor%5D">Vinay Dubey</a></div></div><div id="_co_idm140165947006864"><h3 class="no_margin">Harsha Raj</h3><p>
<sup>
1
</sup>
Molecular Cell Biology Department, 
Weizmann Institute of Science, 
Rehovot
76100
Israel
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Raj%20H%5BAuthor%5D">Harsha Raj</a></div></div><div id="_co_idm140165947004256"><h3 class="no_margin">Ifat Abramovitch</h3><p>
<sup>
2
</sup>
The Ruth and Bruce Rappaport, 
Faculty of Medicine, 
Technion&#x02013;Israel Institute of Technology, 
Haifa
3525433
Israel
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Abramovitch%20I%5BAuthor%5D">Ifat Abramovitch</a></div></div><div id="_co_idm140165947001648"><h3 class="no_margin">Sergey Malitsky</h3><p>
<sup>
3
</sup>
Metabolic Profiling Unit, 
Weizmann Institute of Science, 
Rehovot
76100
Israel
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Malitsky%20S%5BAuthor%5D">Sergey Malitsky</a></div></div><div id="_co_idm140165990862736"><h3 class="no_margin">Maxim Itkin</h3><p>
<sup>
3
</sup>
Metabolic Profiling Unit, 
Weizmann Institute of Science, 
Rehovot
76100
Israel
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Itkin%20M%5BAuthor%5D">Maxim Itkin</a></div></div><div id="_co_idm140165990860128"><h3 class="no_margin">Avi Ma'ayan</h3><p>
<sup>
4
</sup>
Department of Pharmacological Sciences, 
Mount Sinai Center for Bioinformatics, 
Icahn School of Medicine at Mount Sinai, 
New York
NY, 
10029
USA
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ma%27ayan%20A%5BAuthor%5D">Avi Ma'ayan</a></div></div><div id="_co_idm140165990857520"><h3 class="no_margin">Frank Westermann</h3><p>
<sup>
5
</sup>
Neuroblastoma Genomics, 
German Cancer Research Center (DKFZ), 
69120
Heidelberg
Germany
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Westermann%20F%5BAuthor%5D">Frank Westermann</a></div></div><div id="_co_idm140165990854912"><h3 class="no_margin">Eyal Gottlieb</h3><p>
<sup>
2
</sup>
The Ruth and Bruce Rappaport, 
Faculty of Medicine, 
Technion&#x02013;Israel Institute of Technology, 
Haifa
3525433
Israel
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gottlieb%20E%5BAuthor%5D">Eyal Gottlieb</a></div></div><div id="_co_idm140165990852304"><h3 class="no_margin">Eytan Ruppin</h3><p>
<sup>
6
</sup>
Cancer Data Science Laboratory, 
National Cancer Institute, 
National Institutes of Health, 
Bethesda
MD, 
20892
USA
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ruppin%20E%5BAuthor%5D">Eytan Ruppin</a></div></div><div id="_co_idm140165990849696"><h3 class="no_margin">Sima Lev</h3><p>
<sup>
1
</sup>
Molecular Cell Biology Department, 
Weizmann Institute of Science, 
Rehovot
76100
Israel
</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Lev%20S%5BAuthor%5D">Sima Lev</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140165998501472_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140165998501472_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140165998501472_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140165998501472_ai" style="display:none"><div class="fm-affl" id="advs7707-aff-0001">
<sup>
1
</sup>
Molecular Cell Biology Department, 
Weizmann Institute of Science, 
Rehovot
76100
Israel
</div><div class="fm-affl" id="advs7707-aff-0002">
<sup>
2
</sup>
The Ruth and Bruce Rappaport, 
Faculty of Medicine, 
Technion&#x02013;Israel Institute of Technology, 
Haifa
3525433
Israel
</div><div class="fm-affl" id="advs7707-aff-0003">
<sup>
3
</sup>
Metabolic Profiling Unit, 
Weizmann Institute of Science, 
Rehovot
76100
Israel
</div><div class="fm-affl" id="advs7707-aff-0004">
<sup>
4
</sup>
Department of Pharmacological Sciences, 
Mount Sinai Center for Bioinformatics, 
Icahn School of Medicine at Mount Sinai, 
New York
NY, 
10029
USA
</div><div class="fm-affl" id="advs7707-aff-0005">
<sup>
5
</sup>
Neuroblastoma Genomics, 
German Cancer Research Center (DKFZ), 
69120
Heidelberg
Germany
</div><div class="fm-affl" id="advs7707-aff-0006">
<sup>
6
</sup>
Cancer Data Science Laboratory, 
National Cancer Institute, 
National Institutes of Health, 
Bethesda
MD, 
20892
USA
</div><div><span class="fm-affl">Sima Lev, </span><span class="fm-affl"><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="li.ca.nnamziew@veL.amiS" class="oemail">li.ca.nnamziew@veL.amiS</a></span>.</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup>Corresponding author.</div><div id="correspondenceTo"><sup>*</sup>
E&#x02010;mail: <a href="mailto:dev@null" data-email="li.ca.nnamziew@veL.amiS" class="oemail">li.ca.nnamziew@veL.amiS</a>
<br />
</div></div><div class="fm-article-notes hide half_rhythm" id="idm140165998501472_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2023 Oct 5; Revised 2024 Feb 11</div></div><div class="permissions half_rhythm hide" id="idm140165998501472_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; 2024 The Authors. Advanced Science published by Wiley&#x02010;VCH GmbH</div><div class="license half_rhythm">This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supporting Information</div><div><a href="/pmc/articles/PMC11077643/bin/ADVS-11-2307263-s001.pdf" data-ga-action="click_feat_suppl">ADVS-11-2307263-s001.pdf</a><span style="color:gray"> (2.4M)</span></div><div class="small guid">GUID: 487144AE-6E18-459B-8F0E-89FB7A4B5FBA</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplemental Table 1&#x02010;6</div><div><a href="/pmc/articles/PMC11077643/bin/ADVS-11-2307263-s002.xlsx" data-ga-action="click_feat_suppl">ADVS-11-2307263-s002.xlsx</a><span style="color:gray"> (194K)</span></div><div class="small guid">GUID: 3B27BE29-4313-4ADA-8094-70C05C36ED91</div></div></dd></dl><dl data-length="522" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first-last">RNA&#x02010;seq data is publicly available in the Gene Expression Omnibus (GEO) repository (<a href="https://www.ncbi.nlm.nih.gov/geo/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.ncbi.nlm.nih.gov/geo/</a>), accession number <a class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE235201" ref="/geo/query/acc.cgi?acc=GSE235201"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE235201","term_id":"235201"}}</span>GSE235201</a>. The code used to generate the computational analysis is available through GitHub (<a href="https://github.com/SimaLevLab/A-continuous-ferroptosis-to-apoptosis-landscape" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/SimaLevLab/A&#x02010;continuous&#x02010;ferroptosis&#x02010;to&#x02010;apoptosis&#x02010;landscape</a>). All other data generated in this study is available upon request from the corresponding author.</p></dd></dl></div><div id="abstract-a.n.b.w" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.n.b.wtitle">Abstract</h2><!--article-meta--><div><p class="p p-first-last">Ferroptosis and apoptosis are key cell&#x02010;death pathways implicated in several human diseases including cancer. Ferroptosis is driven by iron&#x02010;dependent lipid peroxidation and currently has no characteristic biomarkers or gene signatures. Here a continuous phenotypic gradient between ferroptosis and apoptosis coupled to transcriptomic and metabolomic landscapes is established. The gradual ferroptosis&#x02010;to&#x02010;apoptosis transcriptomic landscape is used to generate a unique, unbiased transcriptomic predictor, the Gradient Gene Set (GGS), which classified ferroptosis and apoptosis with high accuracy. Further GGS optimization using multiple ferroptotic and apoptotic datasets revealed highly specific ferroptosis biomarkers, which are robustly validated in vitro and in vivo. A subset of the GGS is associated with poor prognosis in breast cancer patients and PDXs and contains different ferroptosis repressors. Depletion of one representative, PDGFA&#x02010;assaociated protein 1(PDAP1), is found to suppress basal&#x02010;like breast tumor growth in a mouse model. Omics and mechanistic studies revealed that ferroptosis is associated with enhanced lysosomal function, glutaminolysis, and the tricarboxylic acid (TCA) cycle, while its transition into apoptosis is attributed to enhanced endoplasmic reticulum(ER)&#x02010;stress and phosphatidylethanolamine (PE)&#x02010;to&#x02010;phosphatidylcholine (PC)&#x000a0;metabolic shift. Collectively, this study highlights molecular mechanisms underlying ferroptosis execution, identified a highly predictive ferroptosis gene signature with prognostic value, ferroptosis versus apoptosis biomarkers, and ferroptosis repressors for breast cancer therapy.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">apoptosis, biomarkers, breast cancer, classification signature, cancer therapy, ferroptosis, TNBC</span></div></div><div id="advs7707-abs-0001" lang="en" class="tsec sec"><h2 class="headless nomenu"></h2><!--article-meta--><div><p class="p p-first-last">Transcriptomic, metabolomics, mechanistic, and computational analyses of programmed ferroptosis&#x02010;to&#x02010;apoptosis transition landscape identified ferroptosis selective biomarkers, revealed ferroptosis repressors as potential targets for breast cancer therapy, and highlighted ferroptosis&#x02010;toapoptosis molecular switches, including the PE/PC lipidomic shift.
</p><div class="boxed-text-box whole_rhythm hide-overflow"><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140165990804528"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ADVS-11-2307263-g001.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is ADVS-11-2307263-g001.jpg" src="/pmc/articles/PMC11077643/bin/ADVS-11-2307263-g001.jpg" /></div></div><p>
</p></div></div><div id="advs7707-sec-0010" class="tsec sec"><h2 class="head no_bottom_margin" id="advs7707-sec-0010title">1. Introduction</h2><p class="p p-first">Regulated cell death (RCD) pathways can be classified into apoptotic and non&#x02010;apoptotic death, such as necroptosis, autophagy, pyroptosis, and ferroptosis.<sup>[</sup>
<sup><a href="#advs7707-bib-0001" rid="advs7707-bib-0001" class=" bibr popnode">
1
</a></sup>
<sup>]</sup> Although different RCDs are mediated by distinct molecular mechanisms and characterized by unique morphological changes, increasing evidence suggests that RCDs can be connected through crosstalk and integrated signaling networks.<sup>[</sup>
<sup><a href="#advs7707-bib-0002" rid="advs7707-bib-0002" class=" bibr popnode">
2
</a></sup>
<sup>]</sup> Furthermore, different RCDs can be triggered by similar stress inducers, such as oxidative or ER stress, and recent studies suggest that dying cells may display molecular characteristics of more than one type of RCD, implying that &#x0201c;pure&#x0201d; selective or &#x0201c;mixed&#x0201d; RCD may exist.<sup>[</sup>
<sup><a href="#advs7707-bib-0003" rid="advs7707-bib-0003" class=" bibr popnode">
3
</a></sup>
<sup>]</sup> These observations highlight the need for a selective response signature for each RCD and specific biomarkers that can distinguish between different death modules, such as ferroptosis and apoptosis.<sup>[</sup>
<sup><a href="#advs7707-bib-0004" rid="advs7707-bib-0004" class=" bibr popnode">
4
</a></sup>
<sup>]</sup>
</p><p>Apoptosis, the most extensively studied RCD, has long been considered a promising target for cancer therapy, particularly since the discovery of B&#x02010;cell lymphoma 2 (BCL2) as an anti&#x02010;apoptotic oncogene.<sup>[</sup>
<sup><a href="#advs7707-bib-0005" rid="advs7707-bib-0005" class=" bibr popnode">
5
</a></sup>
<sup>]</sup> However, many cancer cells can escape apoptotic drugs and develop drug resistance.<sup>[</sup>
<sup><a href="#advs7707-bib-0006" rid="advs7707-bib-0006" class=" bibr popnode">
6
</a></sup>
<sup>]</sup> Increasing evidence suggests that ferroptosis could be a powerful therapeutic strategy for certain human cancers,<sup>[</sup>
<sup><a href="#advs7707-bib-0007" rid="advs7707-bib-0007" class=" bibr popnode">
7
</a></sup>
<sup>]</sup> and we and others have shown that among the different breast cancer subtypes, triple&#x02010;negative breast cancer (TNBC) is particularly susceptible to ferroptosis,<sup>[</sup>
<sup><a href="#advs7707-bib-0008" rid="advs7707-bib-0008" class=" bibr popnode">
8
</a></sup>
<sup>]</sup> and tumors from TNBC patients are enriched in ferroptosis&#x02010;related genes.<sup>[</sup>
<sup><a href="#advs7707-bib-0008" rid="advs7707-bib-0008" class=" bibr popnode">
8a
</a></sup>
<sup>]</sup>
</p><p>Ferroptosis is a unique RCD pathway driven by iron&#x02010;dependent lipid peroxidation and severe damage to cellular membranes.<sup>[</sup>
<sup><a href="#advs7707-bib-0009" rid="advs7707-bib-0009" class=" bibr popnode">
9
</a></sup>
<sup>]</sup> Labile iron (Fe2+) pool (LIP), polyunsaturated fatty acids (PUFAs), and reactive oxygen species (ROS) are fundamental for lipid peroxidation and ferroptosis execution.<sup>[</sup>
<sup><a href="#advs7707-bib-0010" rid="advs7707-bib-0010" class=" bibr popnode">
10
</a></sup>
<sup>]</sup> Unlike apoptosis, which is characterized by cell shrinkage, chromatin condensation, and membrane blebbing,<sup>[</sup>
<sup><a href="#advs7707-bib-0001" rid="advs7707-bib-0001" class=" bibr popnode">
1
</a></sup>
<sup>]</sup> ferroptotic cells mainly exhibit morphological changes in the mitochondria.<sup>[</sup>
<sup><a href="#advs7707-bib-0011" rid="advs7707-bib-0011" class=" bibr popnode">
11
</a></sup>
<sup>]</sup>
</p><p>Ferroptosis can be triggered by different mechanisms that modulate intracellular glutathione (GSH) pool, labile iron levels, PUFAs, and the activity of glutathione peroxidase 4 (GPX4), a selenoprotein that reduces phospholipid hydroperoxides (PLOOH) by glutathione.<sup>[</sup>
<sup><a href="#advs7707-bib-0012" rid="advs7707-bib-0012" class=" bibr popnode">
12
</a></sup>
<sup>]</sup> Since the initial discovery of erastin as an inhibitor of the cystine/glutamate antiporter (System x<sub>c</sub>
<sup>&#x02212;</sup>) and a ferroptosis inducer,<sup>[</sup>
<sup><a href="#advs7707-bib-0009" rid="advs7707-bib-0009" class=" bibr popnode">
9
</a></sup>
<sup>]</sup> many different ferroptosis inducers (FINs) and inhibitors were identified.<sup>[</sup>
<sup><a href="#advs7707-bib-0013" rid="advs7707-bib-0013" class=" bibr popnode">
13
</a></sup>
<sup>]</sup> Canonical FINs trigger ferroptosis through different mechanisms, including GSH depletion (erastin, artesunate), GPX4 inactivation (RSL3, ML210, ML162), or coenzyme Q10 (CoQ10) depletion (FIN56, CIL56).<sup>[</sup>
<sup><a href="#advs7707-bib-0014" rid="advs7707-bib-0014" class=" bibr popnode">
14
</a></sup>
<sup>]</sup> However, most of these FINs are effective in vitro but have low efficacy and/or high toxicity in vivo.<sup>[</sup>
<sup><a href="#advs7707-bib-0015" rid="advs7707-bib-0015" class=" bibr popnode">
15
</a></sup>
<sup>]</sup> We have recently identified a synthetic lethal drug combination targeting bromodomain&#x02010;containing protein 4 &#x000a0;(BRD4) and the proteasome, which could also trigger ferroptosis in multiple TNBC cell lines and effectively inhibit tumor growth in xenograft and allograft mice models of TNBC.<sup>[</sup>
<sup><a href="#advs7707-bib-0008" rid="advs7707-bib-0008" class=" bibr popnode">
8a
</a></sup>
<sup>]</sup>
</p><p class="p p-last">In this study, we exploited the unique synthetic lethal drug combination targeting BRD4 (by JQ1) and the proteasome (by bortezomib) to induce ferroptosis in TNBC, and concurrently used a drug dose titration to program a continuous phenotypic gradient between ferroptosis and apoptosis. We showed that very low doses of JQ1 and bortezomib (BTZ) induced ferroptosis in basal&#x02010;like breast cancer through integrated responses mediated by enhanced lysosomal function, increased ferritinophagy, glutaminolysis, TCA cycle, oxidative phosphorylation, and activation of activating transcription factor 4 (ATF4), a master regulator of the integrated stress response (IRS) and amino acid metabolism.<sup>[</sup>
<sup><a href="#advs7707-bib-0016" rid="advs7707-bib-0016" class=" bibr popnode">
16
</a></sup>
<sup>]</sup> Ferroptosis&#x02010;to&#x02010;apoptosis transition was coupled to enhanced ER stress, XBP1, and ATF4/CHOP activation concomitant with a PE/PC&#x02010; metabolic shift. Analysis of the gradual ferroptosis&#x02010;apoptosis transcriptomic landscape together with machine learning, public datasets of ferroptosis and apoptotic responses, as well as transcriptomic data from cancer patients and Patient&#x02010;derived xenografts (PDXs), led us to identify a unique Gradient Gene Set (GGS), which classified ferroptosis from apoptosis with high accuracy and could stratify breast cancer patients with poor prognosis. We also identified and validated a set of 24 selective ferroptosis versus apoptosis biomarkers and showed that depletion of PDAP1, a representative gene of the GGS, induced ferroptotic cell death and inhibited tumor growth in the xenograft model of basal&#x02010;like breast cancer.</p></div><div id="advs7707-sec-0020" class="tsec sec"><h2 class="head no_bottom_margin" id="advs7707-sec-0020title">2. Results</h2><div id="advs7707-sec-0030" class="sec sec-first"><h3 id="advs7707-sec-0030title">2.1. Programming a Ferroptosis to Apoptosis Transition Landscape</h3><p class="p p-first">We previously showed that very low doses of JQ1 and BTZ induced ferroptosis across multiple TNBC cell lines.<sup>[</sup>
<sup><a href="#advs7707-bib-0008" rid="advs7707-bib-0008" class=" bibr popnode">
8a
</a></sup>
<sup>]</sup> To further assess the effects of JQ1 and BTZ on ferroptotic cell death, we slightly increased the levels of JQ1 (50&#x02013;200&#x000a0;nM) and BTZ (2&#x02013;6&#x000a0;nM), and examined cell viability (<strong>Figure</strong>&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0001" rid-ob="ob-advs7707-fig-0001" co-legend-rid="lgnd_advs7707-fig-0001"><span>
<strong>1</strong>A,B</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S1A,B,D</a>, Supporting Information) and cell death (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0001" rid-ob="ob-advs7707-fig-0001" co-legend-rid="lgnd_advs7707-fig-0001"><span>1C</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S1C</a>, Suppoting Information), in the absence or presence of either ferroptosis inhibitors (liproxstatin&#x02010;1, ferrostatin&#x02010;1), an apoptotic inhibitor (Z&#x02010;VAD) or a necrosis inhibitor (Necrostatin&#x02010;1) in two basal&#x02010;like breast cancer cell lines; MDA&#x02010;MB&#x02010;468 (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0001" rid-ob="ob-advs7707-fig-0001" co-legend-rid="lgnd_advs7707-fig-0001"><span>1A&#x02013;C</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S1D</a>, Supporting Information) and HCC70 (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S1A&#x02013;C</a>, Supporting Information). As shown, the effect of JQ1 (50&#x02013;200&#x000a0;nM) together with 2&#x000a0;nM BTZ was predominantly rescued by the ferroptosis inhibitors, while JQ1 (50&#x02013;200&#x000a0;nM) with 3&#x000a0;nM BTZ was partially rescued by both liproxstatin&#x02010;1 and Z&#x02010;VAD, and JQ1 (50&#x02013;200&#x000a0;nM) with 6&#x000a0;nM BTZ was mainly rescued by Z&#x02010;VAD. These results indicate that a minor gradual increase of BTZ from 2&#x02010;to&#x02010;3&#x02010;to&#x02010;6&#x000a0;nM could program a transition from ferroptosis to apoptosis. Accordingly, we proceeded with combinations of fixed, low JQ1 dose (100&#x000a0;nM) and increased doses of BTZ from 2&#x000a0;nM (JB2) to induce ferroptosis, 3&#x000a0;nM BTZ (JB3) to induce an intermediate &#x0201c;ferroapoptosis&#x0201d; state, and 6&#x000a0;nM (JB6) to induce apoptosis (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0001" rid-ob="ob-advs7707-fig-0001" co-legend-rid="lgnd_advs7707-fig-0001"><span>1D</span></a>). By using death pathway characteristic assays (Experimental Section),<sup>[</sup>
<sup><a href="#advs7707-bib-0017" rid="advs7707-bib-0017" class=" bibr popnode">
17
</a></sup>
<sup>]</sup> we showed that JB2 strongly induced lipid peroxidation, a hallmark of ferroptosis<sup>[</sup>
<sup><a href="#advs7707-bib-0018" rid="advs7707-bib-0018" class=" bibr popnode">
18
</a></sup>
<sup>]</sup> (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0001" rid-ob="ob-advs7707-fig-0001" co-legend-rid="lgnd_advs7707-fig-0001"><span>1E</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S1E</a>, Supporting Information), JB6 induced strong Annexin V staining (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0001" rid-ob="ob-advs7707-fig-0001" co-legend-rid="lgnd_advs7707-fig-0001"><span>1F,G</span></a>, Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S1G</a>, Supporting Information), upregulation of BIM level (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0001" rid-ob="ob-advs7707-fig-0001" co-legend-rid="lgnd_advs7707-fig-0001"><span>1I</span></a>) and cleavage of Caspase 3 and PARP (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0001" rid-ob="ob-advs7707-fig-0001" co-legend-rid="lgnd_advs7707-fig-0001"><span>1H,I</span></a>, Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S1H</a>, Supporting Information), all are hallmarks of apoptosis, while JB3 exhibit an intermediate phenotype. We also observed reduced levels of GPX4 and FSP1 proteins in JB2 (Figure <a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S1I</a>,Supporing Information), and a gradual inhibition in the proteasomal activity from JB2&#x02010;to&#x02010;JB6 (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0001" rid-ob="ob-advs7707-fig-0001" co-legend-rid="lgnd_advs7707-fig-0001"><span>1J</span></a>). Notably, Necrostatin&#x02010;1 could not rescue the effects of either JB6 or JB2 on cell viability (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S1D</a>, Supporting Information), consistent with our previous report on JB2 and its specific effects on ferroptotic death across multiple TNBC cell lines, but not on normal&#x02010;like mammary cells (MCF10A) or luminal breast cancer cell line (MCF7).<sup>[</sup>
<sup><a href="#advs7707-bib-0008" rid="advs7707-bib-0008" class=" bibr popnode">
8a
</a></sup>
<sup>]</sup> Interestingly, JB6 also had a weaker effect in MCF7 compared to basal&#x02010;like cells (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S1D</a>, Supporting Information) and didn't influence the sensitivity to ferroptosis (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S1J</a>, Supporting Information).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="advs7707-fig-0001" co-legend-rid="lgnd_advs7707-fig-0001"><a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0001/" target="figure" rid-figpopup="advs7707-fig-0001" rid-ob="ob-advs7707-fig-0001"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140165990684128"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ADVS-11-2307263-g008.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is ADVS-11-2307263-g008.jpg" src="/pmc/articles/PMC11077643/bin/ADVS-11-2307263-g008.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140165990684128"><a target="object" rel="noopener" href="/pmc/articles/PMC11077643/figure/advs7707-fig-0001/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_advs7707-fig-0001"><div><a class="figpopup" href="/pmc/articles/PMC11077643/figure/advs7707-fig-0001/" target="figure" rid-figpopup="advs7707-fig-0001" rid-ob="ob-advs7707-fig-0001">Figure 1</a></div><!--caption a7--><div class="caption"><p>Programming of ferroptosis&#x02010;to&#x02010;apoptosis transition by JQ1 and bortezomib combination. (A) MDA&#x02010;MB&#x02010;468 basal&#x02010;like breast cancer cells were treated with the indicated concentration of JQ1 and bortezomib (BTZ) in the absence or presence of liproxstatin&#x02010;1 (5&#x000a0;&#x000b5;M) or z&#x02010;VAD&#x02010;FMK (10&#x000a0;&#x000b5;M) and cell viability was measured 72&#x000a0;h later by MTT. Percentageage (%) of cell viability was calculated compared to the control. (B,C) For JB combinations, JQ1 was applied at 100&#x000a0;nM while BTZ was at either 2&#x000a0;nM (JB2), 3&#x000a0;nM (JB3), or 6&#x000a0;nM (JB6). Cell viability (B) was measured by MTT, and cell death C) was measured by CellTox green. Shown are the mean values &#x000b1; SD from at least 2 independent experiments (except for singles rescue). P&#x02010;values were measured by t&#x02010;test. (D) JB2, JB3, and JB6 are defined as the combinations of 100&#x000a0;nM JQ1 with 2&#x000a0;nM, 3&#x000a0;nM, and 6&#x000a0;nM BTZ, respectively. (E,F) Representative confocal images of MDA&#x02010;MB&#x02010;468 cells treated with the JB drug combinations and stained with C11&#x02010;BODIPY (E) or FITC&#x02010;Annexin V (F) Cumene hydroperoxide (CH; 75&#x000a0;&#x000b5;M) (D) or staurosporine (25&#x000a0;nM) (F) were used as positive controls. Scale bar, 10&#x000a0;&#x000b5;m (in E), 20&#x000a0;&#x000b5;m (in F). (G) FACS analysis of FITC&#x02010;Annexin&#x02010;V labeled MDA&#x02010;MB&#x02010;468 cells 24&#x000a0;h post&#x02010;treatment with the indicated drugs. (H&#x02013;I) Cleavage of caspase&#x02010;3 (H), cleaved PARP, and BIM level (I) in response to drug treatment for 24&#x000a0;h (H) or for the indicated time points (in hours) (I) in MDA&#x02010;MB&#x02010;468 were assessed by Western Blotting (WB). Representative confocal images of cleaved caspase&#x02010;3 are shown in H (bottom). Scale bar, 10&#x000a0;&#x000b5;m. (J) Proteasome activity was assessed in MDA&#x02010;MB&#x02010;468 and HCC70 cells in response to the indicated drug treatment for 24&#x000a0;h. Cell lysates were subjected to proteasome activity using LLVY peptide reporter assay. Results are presented as % activity relative to control (DMSO). Shown are the mean values &#x000b1; SD from 2 (MDA&#x02010;MB&#x02010;468) and 3 (HCC70) independent experiments done in duplicates. P&#x02010;values were measured by t&#x02010;test. (K&#x02013;M) RNAseq was performed for the combinations JB2, JB3, and JB6, for single drug treatments (B2, B3, B6, J) and for DMSO (`D`) as control. (K) Principal component analysis of all treatments in the RNAseq was performed. Shown are the results for MDA&#x02010;MB&#x02010;468 cells. (L) All genes in each treatment were ranked based on the t&#x02010;statistics calculated for the fold change of gene expression in the treatment versus DMSO. GSVA was used to calculate the enrichment of a ferroptosis gene signature (x&#x02010;axis) and apoptosis gene signature (y&#x02010;axis) in JB2, JB3, and JB6 based on the t&#x02010;statistics ranks. The ferroptosis and apoptosis signatures were built for TNBC based on public datasets, using 5FU as a representative of apoptosis inducer, and erastin and ML162 as ferroptosis inducers. (M) Venn diagram of significantly upregulated genes (FDR &#x0003c; 0.05) in MDA&#x02010;MB&#x02010;468 cells in response to JB2, JB3, and JB6 treatments versus DMSO. *<em>p</em>&#x02010;value&#x000a0;&#x0003c;&#x000a0;0.05, **<em>p</em>&#x02010;value&#x000a0;&#x0003c;&#x000a0;0.01, ***<em>p</em>&#x02010;value&#x000a0;&#x0003c;&#x000a0;0.001, ****<em>p</em>&#x02010;value&#x000a0;&#x0003c;&#x000a0;1&#x000a0;&#x000d7;&#x000a0;10<sup>&#x02212;4</sup>.</p></div></div></div><p class="p p-last">We next coupled the gradual phenotypic transition of ferroptosis&#x02010;to&#x02010;apoptosis (JB2&#x02010;JB3&#x02010;JB6) to transcriptomic profiling. We performed RNAseq for MDA&#x02010;MB&#x02010;468 and HCC70 cells treated with individual drugs (100&#x000a0;nM JQ1 or 2, 3 or 6&#x000a0;nM BTZ; B2, B3, B6, respectively) and drug combinations (JB2, JB3, JB6) for 6 and 24&#x000a0;h (before cell death onset, Experimental Section). To visualize the transcriptomic changes induced by the different drug treatments, we performed principal component analysis (PCA) (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0001" rid-ob="ob-advs7707-fig-0001" co-legend-rid="lgnd_advs7707-fig-0001"><span>1K</span></a>, Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S1K</a>, Supporting Information). While JB2, JB3, and JB6 clustered together, they were distant from the control (DMSO), suggesting a large transcriptomic change in all three. The JB2&#x02010;JB3&#x02010;JB6 transcriptomic transition was further visualized on a plot whose axes represent gene set enrichment scores of ferroptosis<sup>[</sup>
<sup><a href="#advs7707-bib-0019" rid="advs7707-bib-0019" class=" bibr popnode">
19
</a></sup>
<sup>]</sup> versus apoptosis<sup>[</sup>
<sup><a href="#advs7707-bib-0020" rid="advs7707-bib-0020" class=" bibr popnode">
20
</a></sup>
<sup>]</sup> responses in TNBC cell lines of publicly available data. Enrichment analysis of the RNAseq data was measured by Gene Set Variation Analysis (GSVA) (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0001" rid-ob="ob-advs7707-fig-0001" co-legend-rid="lgnd_advs7707-fig-0001"><span>1L</span></a>). As shown, the three JB combinations were found in close proximity, with a gradual transcriptomic transition from ferroptosis (JB2) through the interphase (JB3) and into apoptosis (JB6), consistent with the parallel phenotypic transition. Finally, analysis of significant differentially expressed genes (DEGs) compared to DMSO, demonstrates that JB2, JB3, and JB6 share the vast majority (&#x02248;80%) of significant DEGs (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0001" rid-ob="ob-advs7707-fig-0001" co-legend-rid="lgnd_advs7707-fig-0001"><span>1M</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S1L</a>, Supporting Information), which represent a general stress response to the applied drug combinations regardless of the specific cell death module.</p></div><div id="advs7707-sec-0040" class="sec"><h3 id="advs7707-sec-0040title">2.2. Identification of a Ferroptosis Versus Apoptosis Gene Signature</h3><p class="p p-first">The transcriptomic landscapes induced by JB2 and JB6 revealed that only &#x02248;20% of the DEGs can be specifically attributed to ferroptosis or apoptosis (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0001" rid-ob="ob-advs7707-fig-0001" co-legend-rid="lgnd_advs7707-fig-0001"><span>1M</span></a>), suggesting that these DEGs may segregate the transcriptomic response of &#x02010;FINs&#x02010; from apoptosis inducers (AINs).</p><p>To explore this possibility, we used the RNAseq data of JB2 (ferroptosis) and JB6 (apoptosis) at 24&#x000a0;h to generate four gene sets, each containing genes that were significantly up or down&#x02010;regulated in either JB2 or JB6 but not in both. For example, the &#x0201c;JB2 Up&#x0201d; set included genes that were significantly upregulated (FDR &#x0003c; 0.05) in JB2 versus DMSO but not significant in JB6 versus DMSO. This gene set was then filtered to include only the genes with the highest significant difference (FDR &#x0003c; 0.05) between JB2 and JB6 (<strong>Figure</strong>&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>
<strong>2</strong>A,B</span></a>, Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S2A</a>, Supporting Information). Notably, this gene set included genes associated with GSH metabolism, TCA cycle, and lysosome (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S2B</a>, Supporting Information), among others.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140165987970464"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ADVS-11-2307263-g003.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is ADVS-11-2307263-g003.jpg" src="/pmc/articles/PMC11077643/bin/ADVS-11-2307263-g003.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140165987970464"><a target="object" rel="noopener" href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_advs7707-fig-0002"><div><a class="figpopup" href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002">Figure 2</a></div><!--caption a7--><div class="caption"><p>Establishment of ferroptosis versus apoptosis gene sets and biomarkers. (A) Average log2 fold change of JB2/3/6&#x000a0;versus DMSO for all the genes included in each of the GGS subsets. (B) Volcano plot highlighting the four subsets of the GGS. Data is based on RNAseq performed on MDA&#x02010;MB&#x02010;468 at 24&#x000a0;h. The insert table shows the number of genes in each subset (total&#x000a0;=&#x000a0;306 genes). (C) UMAP projection of 19 ferroptosis inducers (green), 26 apoptosis inducers (blue), and 5 FIN validation sets (red) using the 306 genes of the gradient gene set (GGS). Shown is the AUC of classification between the ferroptosis and apoptosis inducers, calculated using k&#x02010;Nearest Neighbors. (D) Enrichment scores for the 15 top genes in the &#x0201c;Datasets&#x02010;derived&#x0201d; and the &#x0201c;Gradient&#x02010;derived&#x0201d; sets as well as the published response signature to erastin in HT1080 (<em>n</em>&#x000a0;=&#x000a0;33 genes), and the hallmark apoptosis set from MsigDB (<em>n</em>&#x000a0;=&#x000a0;154 genes). Scoring was determined by ssGSEA, in the public 19 ferroptosis and 26 apoptosis datasets (Table <a href="#advs7707-supitem-0002" rid="advs7707-supitem-0002" class=" supplementary-material">S1</a>, Supporting Information), as well as in the validation sets of 5 FINs. ROC&#x02010;AUCs and <em>p</em>&#x02010;values of AUCs (at the bottom of each panel) were calculated for the FINs versus AINs classification. The Center line indicates the median of each group. `*` denotes p&#x02010;values measured by t&#x02010;test, between the FINs and AINs scores. `#` denotes non&#x02010;significant p&#x02010;values measured by a one&#x02010;sample t&#x02010;test, comparing the scores to zero (suggesting the genes are not significantly enriched in those sets). (E) Venn diagram showing the genes in the different gene sets. (F&#x02010;G) Shown are the selected 24 genes from the Datasets&#x02010;derived (DDB) and the Gradient&#x02010;derived (GDB) biomarkers that were used for validation by qRT&#x02010;PCR. For in&#x02010;vitro validation (F), MDA&#x02010;MB&#x02010;468 and HCC70 were treated with the indicated FINs (Erastin, FIN56, RSL3) and AINs (Paxlitaxel, Staurosporine), for the indicated time using IC<sub>50</sub> concentrations. For in&#x02010;vivo validation (G), mice harboring MDA&#x02010;MB&#x02010;468 xenograft tumors were treated with IKE and paclitaxel as described in the experimental section. RNA was extracted and gene expression was examined by qPCR. Numbers are mean fold&#x02010;changes in gene expression versus DMSO of three independent repeats (in F), or three tumors per group (in G). Actin was used as a housekeeping gene for normalization. The genes are sorted by average fold changes in all FIN treatments. Points on the left mark the source of the biomarkers; Dataset&#x02010;derived (DDB) or Gradient&#x02010;derived (GDB) sets. (H) Expression of the 24 selected biomarkers in the RNAseq datasets used in this study. Left: the mean fold&#x02010;changes in the 19 FINs and 26 AINs public datasets. Right: Fold change in the 5 FIN validation datasets (averages of two repeats per set). (I) Integrated qRT&#x02010;PCR results of F&#x02010;G were performed by converting the fold changes values (vs DMSO) of the 24 genes into z&#x02010;scores (to normalize all the genes to the same scale), and then average the z&#x02010;scores of all 24 genes. Z&#x02010;scores were calculated by centering the fold&#x02010;changes around 1, with a standard deviation of 1. Shown are the means &#x000b1; SD of three repeats or mice. <em>p</em>&#x02010;values determined by t&#x02010;test. (J) ssGSEA enrichment scores for the 24 biomarkers in the public datasets (averages) and the 5 FIN validation sets.</p></div></div></div><p class="p p-last">We then combined the four gene sets (JB2 Up/Down; 146 genes, JB6 Up/Down; 160 genes, a total 306 genes) to form the &#x0201c;Gradient Gene Set&#x0201d; (GGS), a unique gene set of the most differential transcriptomic response between JB2 (ferroptosis) and JB6 (apoptosis) (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2B</span></a>, Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S2A</a>, Supporting Information). We hypothesized that the GGS can classify the transcriptomic response of ferroptosis distinctly from apoptosis. To test this hypothesis, we collected transcriptomic data from 19 FINs and 26 AINs&#x02010;from the GEO datasets (Table&#x000a0;<a href="#advs7707-supitem-0002" rid="advs7707-supitem-0002" class=" supplementary-material">S1</a>, Supporting Information). In addition, we performed RNAseq analysis of classical FINs (erastin and RSL3) responses in basal&#x02010;like breast cancer cell lines (MDA&#x02010;MB&#x02010;468, HCC70, and HCC38), and combined these five datasets (validation group) to the 45 (19 FINs, 26 AINs) public datasets. UMAP was used to visualize the expression of the 306 GGS genes in these 50 datasets (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2C</span></a>). This analysis generated a map with distinct ferroptosis and apoptosis regions and a clear decision boundary in between. Importantly, the five validation datasets of FINs responses were mapped in the ferroptosis area (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2C</span></a>, red dots). Furthermore, a K&#x02010;nearest neighbors analysis showed the high classification power of the GGS, which accurately classified the ferroptosis and apoptosis datasets (ROC&#x02010;AUC = 0.90) (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2C</span></a>).</p></div><div id="advs7707-sec-0050" class="sec"><h3 id="advs7707-sec-0050title">2.3. Optimization of Ferroptosis Versus Apoptosis Biomarkers</h3><p class="p p-first">Although the GGS could classify ferroptosis from apoptosis with high accuracy, it consists of a relatively high number of genes and thus can be useful to classify RNAseq data. To define a smaller subset of genes that can be used as ferroptosis versus apoptosis biomarkers, we selected various gene sets of different sizes according to their differential expression (fold change) between JB2 (ferroptosis) and JB6 (apoptosis) at two&#x02010;time points (6&#x000a0;h, 24&#x000a0;h). We then measured the enrichment of each gene set in the 19 FINs and 26 AINs datasets by ssGSEA and scored their classification accuracy by ROC&#x02010;AUC.</p><p>We found that the best AUC is given by a set of genes whose transcriptomic gradient shifts from 6 to 24&#x000a0;h: their expression was significantly lower in JB2 versus JB6 at 6&#x000a0;h but significantly higher in JB2 versus JB6 at 24&#x000a0;h (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S2C</a>, Table&#x000a0;<a href="#advs7707-supitem-0002" rid="advs7707-supitem-0002" class=" supplementary-material">S2</a>, Supporting Information). This set consists of 26 genes and was termed the &#x0201c;Gradient derived&#x0201d; genes set. We assume that the decreased expression at an early time point may induce the onset of ferroptosis, while the increase at a later time may represent an attempt to compensate and protect from ferroptosis. The &#x0201c;Gradient derived&#x0201d; genes set had a strong and significant classification accuracy between the 19 FINs and 26 AINs datasets (AUC&#x000a0;=&#x000a0;0.88, Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2D</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S2E</a>, Supporting Information). It was also highly enriched in the validation group of the 5 FINs response datasets of basal&#x02010;like TNBC cell lines (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2D</span></a>) and thus can be considered as a powerful ferroptosis to apoptosis classifier.</p><p>To further corroborate the &#x0201c;Gradient derived&#x0201d; classification capacity, we compared it to the best possible classifier between the 19 FINs and 26 AINs datasets. To this end, we generated a set of 15 genes, termed the &#x0201c;Datasets derived` genes set, by ranking all the genes in the datasets according to their relative expression in the FINs versus the AINs datasets, using a bootstrapped aggregation method (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S2D</a>, Supporting Information). This analysis revealed that the classification accuracy of the top&#x02010;ranked 15 genes was very similar (AUC&#x000a0;=&#x000a0;0.9) to the top 15 genes of the &#x02018;Gradient derived&#x0201d; set (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2D</span></a>). Nevertheless, increasing the number of the &#x0201c;Datasets derived&#x0201d; genes above 15 (Table&#x000a0;<a href="#advs7707-supitem-0002" rid="advs7707-supitem-0002" class=" supplementary-material">S2</a>, Supporting Information), concurrently increased the classification accuracy (AUC) (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S2E</a>, Supporting Information) and, thus could also be useful when full transcriptomic data is available.</p><p>Collectively, both the &#x0201c;Gradient derived&#x0201d; and the &#x0201c;Datasets derived&#x0201d; ferroptosis versus apoptosis biomarkers display high classification accuracy between the FINs and AINs datasets (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S2F</a>, Supporting Information), and could accurately classify the JB2, JB3, and JB6 transcriptomic landscape (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S2G</a>, Supporting Information). Furthermore, their enrichment scores in the AINs datasets were negligible (almost zero) (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2D</span></a>), highlighting their specificity to ferroptosis. Of note, a set of 33 genes (Table&#x000a0;<a href="#advs7707-supitem-0002" rid="advs7707-supitem-0002" class=" supplementary-material">S2</a>, Supporting Information), which was generated in response to erastin in HT&#x02010;1080 (&#x0201c;Erastin/HT1080 derived&#x0201d;),<sup>[</sup>
<sup><a href="#advs7707-bib-0021" rid="advs7707-bib-0021" class=" bibr popnode">
21
</a></sup>
<sup>]</sup> and is often considered a ferroptotic biomarkers set,<sup>[</sup>
<sup><a href="#advs7707-bib-0022" rid="advs7707-bib-0022" class=" bibr popnode">
22
</a></sup>
<sup>]</sup> was found to be highly enriched in the FINs datasets, as expected, but also in several AINs datasets, exhibiting a lower AUC between the FINs and AINs datasets (AUC&#x000a0;=&#x000a0;0.72, Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2D</span></a>, third panel). Similarly, the commonly used &#x0201c;hallmark_apoptosis&#x0201d;, a set of 154 genes (MsigDB), was enriched both in the AINs and the FINs datasets, suggesting a weak specificity (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2D</span></a>, fourth panel).</p><p>The high accuracy and specificity of the &#x0201c;Gradient derived&#x0201d; and the &#x0201c;Datasets derived&#x0201d; ferroptosis versus apoptosis biomarkers (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2D</span></a>), which share only two genes (CHAC1, DDIT4) (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2E</span></a>; Table&#x000a0;<a href="#advs7707-supitem-0002" rid="advs7707-supitem-0002" class=" supplementary-material">S2</a>, Supporting Information), highlight their strength as reliable biomarkers. For validation, we selected the 15 and 12 top&#x02010;ranked genes of the &#x0201c;Datasets derived&#x0201d; and &#x0201c;Gradient derived&#x0201d; gene sets and composed a set of 24 genes (2 of the 27 are common, and one gene, SACS, was removed due to high variation in the qRT&#x02010;PCR results) with the best ferroptosis versus apoptosis prediction. We then experimentally validated the accuracy of this ferroptosis&#x02010;optimized gene set by qPCR (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2F,G</span></a>). For in vitro validation studies, we used mRNA of MDA&#x02010;MB&#x02010;468 or HCC70 cells treated with either classical FINs; erastin, FIN56, and RSL3, or with known apoptosis inducers; paclitaxel and staurosporine, as indicated (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2F</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S3A</a>, Supporting Information). For the in vivo validation, we established orthotopic xenograft&#x000a0;mouse models by implanting MDA&#x02010;MB&#x02010;468 in the mammary fat pad of nude mice. When tumors reached &#x02248;50 mm<sup>3</sup>, the mice were treated with either imidazole ketone erastin (IKE) or with paclitaxel (PTX) as described in the experimental section, to induce ferroptosis or apoptosis, respectively (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S3B</a>, Supporting Information), and the expression levels of the 24 biomarkers were assessed by qRT&#x02010;PCR (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2G</span></a>). We then calculated the average fold changes for each in vitro and in vivo treatment. As seen in Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2I</span></a>, the expression of the 24 biomarkers was significantly and specifically increased in response to ferroptosis inducers but not in response to the apoptotic inducers both in vitro and in vivo (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2F,G,I</span></a>).</p><p>These results were further corroborated by the high expression levels and ssGSEA enrichment scores of the 24 biomarkers within the 19 FIN datasets and five validation RNAseq datasets of FINs (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2H,J</span></a>). Overall, our comprehensive validation demonstrates the power, specificity, and predictive value of this optimized list of 24 ferroptosis versus apoptosis biomarkers, which can be used as reliable, universal ferroptosis selective biomarkers and can be tested by several qPCRs.</p><p class="p p-last">Importantly, we propose to validate a few biomarkers instead of individual genes due to different modes of action of various chemical FINs, each displaying slightly different kinetics (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S3C</a>, Supporting Information) and intensity (fold change) in their transcriptomic response of these biomarkers as well as different kinetics in lipid peroxidation (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S3D,E</a>, Supporting Information). Furthermore, ferroptosis induction by a nonchemical genetic mode (shRNA, gene knockdown, siRNA), might induce lipid peroxidation at different time windows due to adaptation responses. This could influence their transcriptomic response and the enrichment of the 24 biomarkers as shown for the GPX4 knockdown dataset (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S2F</a>, Supporting Information, point #F11).</p></div><div id="advs7707-sec-0060" class="sec"><h3 id="advs7707-sec-0060title">2.4. A GGS Subset Exhibits a Prognostic Value</h3><p class="p p-first">We next examined the enrichment of the GGS signature (by ssGSEA) in TNBC patients from the METABRIC cohort (<em>n</em>&#x000a0;=&#x000a0;299). UMAP analysis of the enrichment scores revealed two major clusters of patients (<strong>Figure</strong>&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0003/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0003" rid-ob="ob-advs7707-fig-0003" co-legend-rid="lgnd_advs7707-fig-0003"><span>
<strong>3</strong>A</span></a>), which displayed significant differences in the enrichment of the &#x0201c;JB2 Up&#x0201d;&#x000a0;subset (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S4A</a>, Supporting Information) and concomitantly in their survival as determined by Kaplan&#x02010;Meier analysis (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0003/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0003" rid-ob="ob-advs7707-fig-0003" co-legend-rid="lgnd_advs7707-fig-0003"><span>3B</span></a>). This was further corroborated by univariate (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0003/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0003" rid-ob="ob-advs7707-fig-0003" co-legend-rid="lgnd_advs7707-fig-0003"><span>3C</span></a>) and multivariate (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S4B</a>, Supporting Information) Cox proportional hazard models, which revealed that the &#x0201c;JB2 Up&#x0201d;&#x000a0;gene set is highly correlated with poor prognosis in breast cancer patients (TNBC and non&#x02010;TNBC). These findings suggest that tumors with high enrichment of the &#x0201c;JB2 Up&#x0201d;&#x000a0;genes are more aggressive. Indeed, high enrichment of &#x0201c;JB2 Up&#x0201d;&#x000a0;gene set was obtained in chemoresistant breast cancer patients (<em>n</em>&#x000a0;=&#x000a0;257)<sup>[</sup>
<sup><a href="#advs7707-bib-0023" rid="advs7707-bib-0023" class=" bibr popnode">
23
</a></sup>
<sup>]</sup> (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0003/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0003" rid-ob="ob-advs7707-fig-0003" co-legend-rid="lgnd_advs7707-fig-0003"><span>3D</span></a>) and was further corroborated in 40 PDX models of breast cancer, with known transcriptomic data and response to chemotherapy.<sup>[</sup>
<sup><a href="#advs7707-bib-0024" rid="advs7707-bib-0024" class=" bibr popnode">
24
</a></sup>
<sup>]</sup> As shown, we observed a high correlation between the enrichment of &#x0201c;JB2 Up&#x0201d; and the response of the PDXs to docetaxel and carboplatin treatments (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0003/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0003" rid-ob="ob-advs7707-fig-0003" co-legend-rid="lgnd_advs7707-fig-0003"><span>3E</span></a>). Taken together, we identified a subset of GGS, the &#x0201c;JB2 Up&#x0201d; genes, as a potential prognostic gene set for breast cancer aggressiveness and prediction of chemotherapy response.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="advs7707-fig-0003" co-legend-rid="lgnd_advs7707-fig-0003"><a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0003/" target="figure" rid-figpopup="advs7707-fig-0003" rid-ob="ob-advs7707-fig-0003"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140165987896112"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ADVS-11-2307263-g002.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is ADVS-11-2307263-g002.jpg" src="/pmc/articles/PMC11077643/bin/ADVS-11-2307263-g002.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140165987896112"><a target="object" rel="noopener" href="/pmc/articles/PMC11077643/figure/advs7707-fig-0003/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_advs7707-fig-0003"><div><a class="figpopup" href="/pmc/articles/PMC11077643/figure/advs7707-fig-0003/" target="figure" rid-figpopup="advs7707-fig-0003" rid-ob="ob-advs7707-fig-0003">Figure 3</a></div><!--caption a7--><div class="caption"><p>A GGS subset exhibits a prognostic value. A,B) Enrichment of the four GGS subsets (JB2/6 Up/Down) was scored in human TNBC patients (<em>n</em>&#x000a0;=&#x000a0;299, METABRIC) by ssGSEA. The scores are visualized by UMAP (A). Kmeans clustering was used to cluster the patients into two clusters. Kaplan&#x02010;Meier analysis was used to show the survival probabilities of patients in both clusters (B). p&#x02010;value in B was measured by a log&#x02010;rank test. C) Hazard ratios (HR) were calculated for the GGS subsets in breast cancer patients (<em>n</em>&#x000a0;=&#x000a0;1904 patients) and only in TNBC patients (<em>n</em>&#x000a0;=&#x000a0;299 patients; METABRIC dataset). Quantification was done using Cox proportional hazards model. Signature scoring in each patient was done using ssGSEA. Bar graphs show the p&#x02010;values of the HR; the numbers in each bar are the HR values. (D) ssGSEA was used to quantify the enrichment of the 75 `JB2 Up` genes in a dataset containing <em>n</em>&#x000a0;=&#x000a0;279 patients treated with ixabepilone or paclitaxel (data taken from <a id="__tag_1148364247" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE41998" ref="/geo/query/acc.cgi?acc=GSE41998"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE41998","term_id":"41998"}}</span>GSE41998</a>). Patients were grouped based on the therapy outcome: CR&#x02010;complete response; PR&#x02010;partial response; and SD&#x02010;stable disease. <em>p</em>&#x02010;values were determined by t&#x02010;test. (E) ssGSEA was used to quantify the enrichment of the 75 `JB2 Up` genes in a dataset containing 40 PDX models. PDX models were grouped based on the therapy outcome: CR&#x02010;complete response; PR&#x02010;partial response; SD&#x02010;stable disease, PD&#x02010;progressive disease. Pearson's correlations are shown with their p&#x02010;values. *<em>p</em>&#x02010;value&#x000a0;&#x0003c;&#x000a0;0.05, **<em>p</em>&#x02010;value&#x000a0;&#x0003c;&#x000a0;0.01, ***<em>p</em>&#x02010;value&#x000a0;&#x0003c;&#x000a0;0.001.</p></div></div></div></div><div id="advs7707-sec-0070" class="sec"><h3 id="advs7707-sec-0070title">2.5. A GGS Subset Contains Potential Ferroptosis Repressors For Breast Cancer Therapy</h3><p class="p p-first">The finding that the &#x0201c;JB2 Up&#x0201d; set of 75 ferroptosis upregulated genes has prognostic value (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0003/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0003" rid-ob="ob-advs7707-fig-0003" co-legend-rid="lgnd_advs7707-fig-0003"><span>3</span></a>) suggests that it may contain ferroptosis repressors. Indeed, a few genes of this subset have already been shown to suppress ferroptosis, such as RRM2,<sup>[</sup>
<sup><a href="#advs7707-bib-0025" rid="advs7707-bib-0025" class=" bibr popnode">
25
</a></sup>
<sup>]</sup> DNM1<sup>[</sup>
<sup><a href="#advs7707-bib-0026" rid="advs7707-bib-0026" class=" bibr popnode">
26
</a></sup>
<sup>]</sup> and PCK2.<sup>[</sup>
<sup><a href="#advs7707-bib-0027" rid="advs7707-bib-0027" class=" bibr popnode">
27
</a></sup>
<sup>]</sup> As a proof of concept, we validated one of these genes, PDAP1 &#x02010;. Previos studies showed that PDAP1 is overexpressed and/or involved in the progression of several cancers including gastric cancer, rectal carcinoma, glioma, and colorectal cancer.<sup>[</sup>
<sup><a href="#advs7707-bib-0028" rid="advs7707-bib-0028" class=" bibr popnode">
28
</a></sup>
<sup>]</sup> We selected PDAP1 since it is significantly highly expressed in breast cancer patients, particularly in TNBC (<strong>Figure</strong>&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0004" rid-ob="ob-advs7707-fig-0004" co-legend-rid="lgnd_advs7707-fig-0004"><span>
<strong>4</strong>A</span></a>). Furthermore, we found that its knockdown (KD) induced a very similar transcriptomic response of GPX4 KD based on the Connectivity Map (Broad,<sup>[</sup>
<sup><a href="#advs7707-bib-0029" rid="advs7707-bib-0029" class=" bibr popnode">
29
</a></sup>
<sup>]</sup> Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S5A</a>, Supporting Information), and thus, could be a promising candidate for further analysis.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="advs7707-fig-0004" co-legend-rid="lgnd_advs7707-fig-0004"><a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/" target="figure" rid-figpopup="advs7707-fig-0004" rid-ob="ob-advs7707-fig-0004"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140165987860976"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ADVS-11-2307263-g007.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is ADVS-11-2307263-g007.jpg" src="/pmc/articles/PMC11077643/bin/ADVS-11-2307263-g007.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140165987860976"><a target="object" rel="noopener" href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_advs7707-fig-0004"><div><a class="figpopup" href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/" target="figure" rid-figpopup="advs7707-fig-0004" rid-ob="ob-advs7707-fig-0004">Figure 4</a></div><!--caption a7--><div class="caption"><p>PDAP1 depletion induced ferroptosis and inhibited breast tumor growth in mice. (A) PDAP1 expression in breast cancer patients. Data was taken from TCGA (<em>n</em>&#x000a0;=&#x000a0;114 normal, 879 non&#x02010;TNBC, 180 TNBC). <em>p</em>&#x02010;values were calculated by t&#x02010;test. (B) Knockdown of PDAP1 reduced cell viability. MDA&#x02010;MB&#x02010;468 and HCC70 cells depleted of PDAP1 (shPDAP1) were grown in the absence or presence of liproxstatin&#x02010;1 and cell viability was assessed 72&#x000a0;h later. Cell viability was calculated relative to control cells (pLKO) (%) and mean values &#x000b1; SD from two independent experiments are shown. <em>p</em>&#x02010;values were measured by t&#x02010;test. (C) Control MDA&#x02010;MB&#x02010;468 cells (pLKO) and PDAP1 or GPX4 knockdown (KD) cells were analyzed for 4&#x02010;NHE&#x02010;protein adducts by WB with &#x000df;&#x02010;tubulin as a loading control. (D&#x02010;F) MDA&#x02010;MB&#x02010;468 cells expressing an inducible shPDAP1 (Tet&#x02010;pLKO&#x02010;shPDAP1) were treated with doxycycline (DOX) as indicated and assessed for either cell viability 96&#x000a0;h later (D) or for lipid&#x02010;ROS 48&#x000a0;h later (E, F). Cell viability and fluorometric quantification of lipid&#x02010;ROS were calculated relative to DOX untreated (DMSO) control and the mean values &#x000b1; SD from two independent experiments are shown. Representative confocal images of BODIPY&#x02010;C11 fluorescence are shown (E). Scale bar, 10&#x000a0;&#x000b5;m. (G) Western blot showing the levels of ferroptosis&#x02010;related proteins in control and PDAP1 depleted cells (&#x000b1; DOX). (H) GSH levels were quantified in PDAP1 KD MDA&#x02010;MB&#x02010;468 cells. Shown are means (relative to pLKO) &#x000b1; SD of 3 experiments done in duplicates. <em>p</em>&#x02010;values were measured by t&#x02010;test. (I) Intracellular iron levels were assessed by colorimetric assay in control and PDAP1&#x02010;KD. Shown are mean values &#x000b1; SD of 2 independent repeats. p&#x02010;values were measured by t&#x02010;test. (J, K) Representative confocal images of control (DMSO) and PDAP1 depleted (DOX) MDA&#x02010;MB&#x02010;468 cells stained with the mitochondrial lipid peroxidation probe MitoPeDPP (J) or with lysotracker (K). Scale bar, 5&#x000a0;&#x000b5;m. (L) Control (pLKO) or PDAP1 KD MDA&#x02010;MB&#x02010;468 cells were analyzed for LC3B expression by immunofluorescence. Scale bars 10&#x000a0;&#x000b5;m. (M) Western blot analysis of FTH level in control (&#x02010;DOX) and PDAP1 depleted MDA&#x02010;MB&#x02010;468 cells (+ DOX). (N, O) Western blot analysis of the indicated signaling components in control (pLKO/&#x02010;DOX) and PDAP1 KD (shPDAP1/+DOX) MDA&#x02010;MB&#x02010;468 and/or HCC70 cells. (P) Transcripts of SREBP and SCD1 were assessed by qPCR. Mean values &#x000b1; SD of two independent experiments are shown. p&#x02010;values were calculated by t&#x02010;test. (Q&#x02010;S) Depletion of PDAP1 reduced tumor burden and increased lipid&#x02010;ROS in vivo. Luciferase&#x02010;labeled MDA&#x02010;MB&#x02010;468 cells expressing the Tet&#x02010;pLKO&#x02010;shPDAP1 were implanted bilaterally into the mammary fat pad of female nude mice. When tumors reached &#x02248;50&#x000a0;mm<sup>3</sup>, mice were randomized into two groups; shPDAP1 (+DOX) and non&#x02010;induced (&#x02010;DOX), <em>n</em>&#x000a0;=&#x000a0;10 tumors/group (Q). Parallel experiment (S) was performed with scramble shRNA instead of PDAP1 shRNA (shScramble) as control (<em>n</em>&#x000a0;=&#x000a0;6 tumors/group). (Q) The left graph shows the mean tumor volume &#x000b1; SEM in the indicated time points from non&#x02010;induced (&#x02212;DOX) and induced (+DOX) shPDAP1 tumors. Statistical significance was determined by 2&#x02010;way ANOVA (*<em>p</em>&#x02010;value &#x0003c;&#x000a0;0.05). The right Boxplot shows the tumor volumes at the experimental endpoint (24 days post&#x02010;induction with DOX). <em>p</em>&#x02010;values were measured by t&#x02010;test. (R) Representative bioluminescence imaging of xenograft tumors at day 24 after DOX induction (left), along with the excised tumors (right) showing the reduced size of PDAP1 KD tumors. (S) Weights of PDAP1 depleted (+DOX) or not&#x02010;depleted (&#x02010;DOX) tumors (day 24), and of Control (scrambled shRNA) tumors &#x000b1; DOX. Shown are the mean mass of excised tumors &#x000b1;SEM. T&#x02010;test was used to measure statistical significance. Right: Excised tumors from mice harboring the control (scrambled) shRNA treated with or without DOX. (T) Western blot analysis of tumors expressing the inducible shPDAP1 in the presence (+) or absence (&#x02212;) of DOX (<em>n</em>&#x000a0;=&#x000a0;7). Tumors were homogenized, lysed and analyzed for protein levels of PDAP1, 4HNE&#x02010;protein adducts, and cleaved PARP. &#x003b2;&#x02010;Tubulin was used as a loading control. (U) RNA was extracted from the tumors, and qRT&#x02010;PCR was used to quantify the expression levels of 21 out of the 24 ferroptosis versus apoptosis biomarkers (shown in Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2F</span></a>). Plot shows the average z&#x02010;scores (fold changes centered &#x02248;1) of the 21 genes (actual fold changes are shown in Figure <a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S5J</a>). t&#x02010;test was used to measure p&#x02010;value. *<em>p</em>&#x02010;value&#x000a0;&#x0003c;&#x000a0;0.05, **<em>p</em>&#x02010;value&#x000a0;&#x0003c;&#x000a0;0.01, ***<em>p</em>&#x02010;value&#x000a0;&#x0003c;&#x000a0;0.001, ****<em>p</em>&#x02010;value&#x000a0;&#x0003c;&#x000a0;1&#x000d7;10<sup>&#x02212;4</sup>.</p></div></div></div><p>Indeed, depletion of PDAP1 expression by shRNA in basal&#x02010;like TNBC cell lines MDA&#x02010;MB&#x02010;468 and HCC70 (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S5B</a>, Supporting Information) markedly reduced cell survival, which could be rescued by liproxstatin&#x02010;1 (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0004" rid-ob="ob-advs7707-fig-0004" co-legend-rid="lgnd_advs7707-fig-0004"><span>4B</span></a>) or ferrostatin&#x02010;1 (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S5C</a>, Supporting Information). PDAP1 depletion increased the levels of 4&#x02010;hydroxynonenal (4&#x02010;HNE) modified proteins, comparable to shGPX4 (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0004" rid-ob="ob-advs7707-fig-0004" co-legend-rid="lgnd_advs7707-fig-0004"><span>4C</span></a>) and of oxidized BODIPY&#x02010;C11 (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S5D</a>, Supporting Information). Similar effects were observed with the doxycycline (DOX) inducible PDAP1 KD (Tet&#x02010;pLKO&#x02010;shPDAP1) (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0004" rid-ob="ob-advs7707-fig-0004" co-legend-rid="lgnd_advs7707-fig-0004"><span>4D&#x02013;F</span></a>). The increased lipid peroxidation in PDAP1&#x02010;depleted cells was accompanied by reduced levels of GPX4 and FSP1 proteins (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0004" rid-ob="ob-advs7707-fig-0004" co-legend-rid="lgnd_advs7707-fig-0004"><span>4G</span></a>), reduced levels of intracellular GSH (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0004" rid-ob="ob-advs7707-fig-0004" co-legend-rid="lgnd_advs7707-fig-0004"><span>4H</span></a>), and elevated levels of iron (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0004" rid-ob="ob-advs7707-fig-0004" co-legend-rid="lgnd_advs7707-fig-0004"><span>4I</span></a>), TFRC (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0004" rid-ob="ob-advs7707-fig-0004" co-legend-rid="lgnd_advs7707-fig-0004"><span>4G</span></a>), and lipid ROS in the mitochondria (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0004" rid-ob="ob-advs7707-fig-0004" co-legend-rid="lgnd_advs7707-fig-0004"><span>4J</span></a>), which all can contribute to ferroptosis cell death.</p><p>We further found that exogenous GSH suppressed lipid peroxidation in PDAP1&#x02010;KD cells (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S5E</a>, Supporting Information) and almost restored their cell viability (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S5F</a>, Supporting Information), similar to the effect of the iron chelator 2,2&#x02032;&#x02010;bipyridine (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S5G</a>, Supporting Information). The elevated cellular iron levels in PDAP1&#x02010;depleted cells were associated with increased intensity of lysosomal staining (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0004" rid-ob="ob-advs7707-fig-0004" co-legend-rid="lgnd_advs7707-fig-0004"><span>4K</span></a>) and LC3B (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0004" rid-ob="ob-advs7707-fig-0004" co-legend-rid="lgnd_advs7707-fig-0004"><span>4L</span></a>), an autophagic activity marker, reduced level of ferritin (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0004" rid-ob="ob-advs7707-fig-0004" co-legend-rid="lgnd_advs7707-fig-0004"><span>4M</span></a>) and increased level of lysosomal iron, as shown by ferro&#x02010;orange staining (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S5H</a>, Supporting Information). These results suggest that PDAP1 depletion enhanced ferretinophagy and consequently increased the labile iron pool.</p><p>It is well known that autophagy is negatively regulated by mTOR,<sup>[</sup>
<sup><a href="#advs7707-bib-0030" rid="advs7707-bib-0030" class=" bibr popnode">
30
</a></sup>
<sup>]</sup> and previous studies suggest that PDAP1 is an upstream regulator of AKT,<sup>[</sup>
<sup><a href="#advs7707-bib-0031" rid="advs7707-bib-0031" class=" bibr popnode">
31
</a></sup>
<sup>]</sup> implying that PDAP1 regulates the AKT&#x02010;mTOR pathway. Indeed, we found that PDAP1 KD reduced the phosphorylation of AKT1 (S473), of 4EBP and S6 kinase, two mTORC1 downstream targets (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0004" rid-ob="ob-advs7707-fig-0004" co-legend-rid="lgnd_advs7707-fig-0004"><span>4N</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S5I</a>, Supporting Information). We also observed a reduced level of stearoyl&#x02010;CoA desaturase&#x02010;1&#x02010;(SCD1) protein, the rate&#x02010;limiting enzyme of monounsaturated fatty acids (MUFAs) biosynthesis, and of SCD1 transcript (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0004" rid-ob="ob-advs7707-fig-0004" co-legend-rid="lgnd_advs7707-fig-0004"><span>4N&#x02013;P</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S5I</a>, Supporting Information), which is regulated by the &#x02010;Sterol regulatory element binding protein 1 (SREBP1) transcription factor, a downstream effector of AKT&#x02010;mTOR pathway.<sup>[</sup>
<sup><a href="#advs7707-bib-0032" rid="advs7707-bib-0032" class=" bibr popnode">
32
</a></sup>
<sup>]</sup> Importantly, both the protein and mRNA levels of SREBP1 were also reduced in PDAP1&#x02010;depleted MDA&#x02010;MB&#x02010;468 or HCC70 cells (Figure 4O,&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0004" rid-ob="ob-advs7707-fig-0004" co-legend-rid="lgnd_advs7707-fig-0004"><span>4P</span></a>). Collectively, these results suggest that PDAP1 can protect from ferroptotic death through the AKT&#x02010;mTOR&#x02010;SREBP1&#x02010;SCD1 signaling axis. Its depletion increased lysosomal activity, autophagy, and ferritinophagy, thereby elevating labile iron. Concurrently, PDAP1 depletion decreased SREB1 and SCD1 to possibly attenuate the mevalonate pathway and to&#x000a0;elevate PUFAs. These effects together with reduced GSH levels, GPX4 and FSP1 can contribute to ferroptosis execution in PDAP1&#x02010;KD cells.</p><p>We then examined the impact of PDAP1 depletion on tumorigenesis in a xenograft model of MDA&#x02010;MB&#x02010;468 cells. MDA&#x02010;MB&#x02010;468 cells harboring the Tet&#x02010;pLKO&#x02010;shPDAP1 or shScrambled (control) were orthotopically implanted into the mammary fat pad of athymic&#x000a0;nude mice (<em>n</em>&#x000a0;=&#x000a0;10 tumors), and tumor growth was monitored over time by caliper as described in methods. When tumors reached &#x02248;50 mm<sup>3</sup> (&#x02248;3&#x02013;4 weeks post&#x02010;implantation), the mice were randomized into control and doxycycline (DOX) untreated and treated groups to induce PDAP1 KD. As shown, DOX&#x02010;treated mice exhibited a significantly reduced tumor growth (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0004" rid-ob="ob-advs7707-fig-0004" co-legend-rid="lgnd_advs7707-fig-0004"><span>4Q</span></a>). Tumor volume and tumor mass were significantly smaller in the DOX&#x02010;treated mice expressing the shPDAP1 compared to control groups (&#x02010;DOX, shScrambled) at the endpoint (24 days post treatment) (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0004" rid-ob="ob-advs7707-fig-0004" co-legend-rid="lgnd_advs7707-fig-0004"><span>4Q&#x02013;S</span></a>).</p><p class="p p-last">
<em>Ex vivo</em> analysis of xenograft tumors (<em>n</em>&#x000a0;=&#x000a0;7/group) revealed enhanced lipid&#x02010;ROS in PDAP1&#x02010;depleted tumors as determined by 4HNE protein adducts, but not of cleaved PARP (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0004" rid-ob="ob-advs7707-fig-0004" co-legend-rid="lgnd_advs7707-fig-0004"><span>4T</span></a>), suggesting that KD of PDAP1&#x02010;induced ferroptosis not only in vitro but also in vivo. Importantly, the PDPA1&#x02010;KD tumors were enriched in the optimized ferroptosis versus apoptosis biomarkers that we discovered (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2F</span></a>) as determined by qPCR (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0004" rid-ob="ob-advs7707-fig-0004" co-legend-rid="lgnd_advs7707-fig-0004"><span>4U</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S5J</a>, Supporting Information), thus further demonstrating the robustness and reliability of the identified biomarkers. Collectively, these results suggest that PDAP1, which belongs to the &#x0201c;JB2 Up&#x0201d; subset, is a ferroptosis suppressor in basal&#x02010;like breast cancer and a potential therapeutic target.</p></div><div id="advs7707-sec-0080" class="sec"><h3 id="advs7707-sec-0080title">2.6. Transcriptomic Analysis of JB2&#x02010; and JB6&#x02010;Induced Ferroptotic and Apoptotic Responses</h3><p class="p p-first">The established JB2 (ferroptosis)&#x02010;to&#x02010;JB6 (apoptosis) transcriptomic landscape was used to gain mechanistic insights into each death module. To that end, we compared the DEGs induced by JB2, JB3, and JB6 versus DMSO control. As shown in Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0001" rid-ob="ob-advs7707-fig-0001" co-legend-rid="lgnd_advs7707-fig-0001"><span>1M</span></a>, &#x02248;80% of the DEGs are common to the three JB&#x02010;combinations induced death states, and pathway enrichment analysis (KEGG) of these common genes revealed enrichment of ER stress and the proteasomal pathways in both MDA&#x02010;MB&#x02010;468 and HCC70 cells (<strong>Figure</strong>&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0005" rid-ob="ob-advs7707-fig-0005" co-legend-rid="lgnd_advs7707-fig-0005"><span>
<strong>5</strong>A</span></a>). Enrichment analysis of the upregulated genes in ferroptosis versus apoptosis (JB2 vs JB6) revealed high enrichment of glutathione metabolism and the TCA cycle in MDA&#x02010;MB&#x02010;468 (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0005" rid-ob="ob-advs7707-fig-0005" co-legend-rid="lgnd_advs7707-fig-0005"><span>5B</span></a>) and of oxidative phosphorylation in HCC70 (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S6A</a>, Supporting Information). Protein processing in the ER, and ribosome&#x02010;related genes were upregulated in JB6 versus JB2 (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0005" rid-ob="ob-advs7707-fig-0005" co-legend-rid="lgnd_advs7707-fig-0005"><span>5C</span></a>, Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S6A</a>, Supporting Information) in both cell lines.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="advs7707-fig-0005" co-legend-rid="lgnd_advs7707-fig-0005"><a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0005/" target="figure" rid-figpopup="advs7707-fig-0005" rid-ob="ob-advs7707-fig-0005"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140165987806624"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ADVS-11-2307263-g006.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is ADVS-11-2307263-g006.jpg" src="/pmc/articles/PMC11077643/bin/ADVS-11-2307263-g006.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140165987806624"><a target="object" rel="noopener" href="/pmc/articles/PMC11077643/figure/advs7707-fig-0005/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_advs7707-fig-0005"><div><a class="figpopup" href="/pmc/articles/PMC11077643/figure/advs7707-fig-0005/" target="figure" rid-figpopup="advs7707-fig-0005" rid-ob="ob-advs7707-fig-0005">Figure 5</a></div><!--caption a7--><div class="caption"><p>Transcriptomic analysis of the ferroptosis (JB2)&#x02010;to&#x02010;apoptosis (JB6) transition landscape. (A) KEGG pathways enrichment for upregulated genes by all three JB combinations versus DMSO in both MDA&#x02010;MB&#x02010;468 and HCC70. (B,C) KEGG pathways enrichment for genes upregulated in JB2 versus JB6 (B) or JB6 versus JB2 (C) in MDA&#x02010;MB&#x02010;468. (D,F,G) Venn diagram of significantly upregulated genes in MDA&#x02010;MB&#x02010;468 cells in response to JB2 (D), JB3 (F), or JB6 (G) and their single components versus DMSO. (E) KEGG pathways enrichment for the genes significantly upregulated only in JB2, and not in JQ1 or in B2, versus DMSO in both MDA&#x02010;MB&#x02010;468 and HCC70. (H) Fold change in gene expression (log2) between JB2 and DMSO in MDA&#x02010;MB&#x02010;468 in 6 and 24&#x000a0;h. Shown are JB2 exclusive genes relevant to ferroptosis or lipid metabolism. &#x0201c;*&#x0201d; marks statistical significance (FDR &#x0003c; 0.05). Red, blue, and green dots indicate known ferroptosis inducers, suppressors, and markers, respectively, taken from FerrDB.</p></div></div></div><p class="p p-last">Further analysis of the DEGs of each combination (JB2, JB3, JB6) compared to those of the single drugs revealed robust transcriptomic changes of JB2 in both MDA&#x02010;MB&#x02010;468 and HCC70 cells (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0005" rid-ob="ob-advs7707-fig-0005" co-legend-rid="lgnd_advs7707-fig-0005"><span>5D&#x02013;G</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S6B&#x02013;D</a>, Supporting Information), consistent with their synergy.<sup>[</sup>
<sup><a href="#advs7707-bib-0008" rid="advs7707-bib-0008" class=" bibr popnode">
8a
</a></sup>
<sup>]</sup> The JB2 upregulated genes versus single drugs were enriched for lysosomal components, autophagy, and ribosome (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0005" rid-ob="ob-advs7707-fig-0005" co-legend-rid="lgnd_advs7707-fig-0005"><span>5E</span></a>). In total, 1872 DEGs were obtained in both cell lines in response to the JB2 combination (vs JQ1 and 2&#x000a0;nM BTZ) at 6 or 24&#x000a0;h post&#x02010;treatment (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S6D</a>, Supporting Information). Many of these genes were associated with the ferroptotic response of JB2 and included ferroptosis inducers, suppressors, and markers,<sup>[</sup>
<sup><a href="#advs7707-bib-0033" rid="advs7707-bib-0033" class=" bibr popnode">
33
</a></sup>
<sup>]</sup> as well as genes known to be involved in ferroptosis&#x02010;related processes, such as iron metabolism, TCA cycle, lipid metabolism, and lysosomal function (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0005" rid-ob="ob-advs7707-fig-0005" co-legend-rid="lgnd_advs7707-fig-0005"><span>5H</span></a>).</p></div><div id="advs7707-sec-0090" class="sec"><h3 id="advs7707-sec-0090title">2.7. Lysosomal Function is Associated with Ferroptosis</h3><p class="p p-first">The significant enrichment for lysosomal genes in JB2&#x02010;treated cells (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0005" rid-ob="ob-advs7707-fig-0005" co-legend-rid="lgnd_advs7707-fig-0005"><span>5E</span></a>) was associated with an increased level of lysosomal characteristic markers and staining, including the &#x02010;lysosomal&#x02010;associated membrane protein 1&#x02010;(LAMP1) (<strong>Figure</strong>&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0006/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0006" rid-ob="ob-advs7707-fig-0006" co-legend-rid="lgnd_advs7707-fig-0006"><span>
<strong>6</strong>A,B</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S7A</a>, Supporting Information), enhanced LysoTracker staining (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0006/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0006" rid-ob="ob-advs7707-fig-0006" co-legend-rid="lgnd_advs7707-fig-0006"><span>6C</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S7B</a>, Supporting Information), increase in acridine orange (AO) acidification as reflected by the red perinuclear spotted signals (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0006/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0006" rid-ob="ob-advs7707-fig-0006" co-legend-rid="lgnd_advs7707-fig-0006"><span>6D</span></a>),<sup>[</sup>
<sup><a href="#advs7707-bib-0034" rid="advs7707-bib-0034" class=" bibr popnode">
34
</a></sup>
<sup>]</sup> increased intensity of perinuclear punctate structures labeled by fluorescent dextran (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0006/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0006" rid-ob="ob-advs7707-fig-0006" co-legend-rid="lgnd_advs7707-fig-0006"><span>6E</span></a>), and increase of &#x003b2;&#x02010;hexosaminidase activity, a lysosomal enzyme that catalyzes the hydrolysis of ganglioside monosialic 2 (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0006/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0006" rid-ob="ob-advs7707-fig-0006" co-legend-rid="lgnd_advs7707-fig-0006"><span>6F</span></a>).<sup>[</sup>
<sup><a href="#advs7707-bib-0035" rid="advs7707-bib-0035" class=" bibr popnode">
35
</a></sup>
<sup>]</sup> Collectively, these results indicate that JB2 enhances lysosomal biogenesis and activity.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="advs7707-fig-0006" co-legend-rid="lgnd_advs7707-fig-0006"><a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0006/" target="figure" rid-figpopup="advs7707-fig-0006" rid-ob="ob-advs7707-fig-0006"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140165987773600"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ADVS-11-2307263-g004.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is ADVS-11-2307263-g004.jpg" src="/pmc/articles/PMC11077643/bin/ADVS-11-2307263-g004.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140165987773600"><a target="object" rel="noopener" href="/pmc/articles/PMC11077643/figure/advs7707-fig-0006/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_advs7707-fig-0006"><div><a class="figpopup" href="/pmc/articles/PMC11077643/figure/advs7707-fig-0006/" target="figure" rid-figpopup="advs7707-fig-0006" rid-ob="ob-advs7707-fig-0006">Figure 6</a></div><!--caption a7--><div class="caption"><p>Ferroptotic response is associated with enhanced lysosomal activity and ferritinophagy. (A,B) The level of LAMP1 protein in MDA&#x02010;MB&#x02010;468 cells treated with the indicated drug combinations for 24&#x000a0;h was assessed by immunofluorescence (IF), Scale bar, 5&#x000a0;&#x000b5;m (A), and by WB (B). (C&#x02013;E) MDA&#x02010;MB&#x02010;468 cells were treated with the indicated drug combinations and stained with lysotracker (C) acridine orange (AO) (D) or FITC&#x02010;dextran (1&#x000a0;mg ml<sup>&#x02212;1</sup>)(E). Shown are representative confocal images. Scale bar, 5&#x000a0;&#x000b5;m (D,E), 10&#x000a0;&#x000b5;m (C). (F) MDA&#x02010;MB&#x02010;468 and HCC70 cells were treated with the indicated drug combinations in the absence or presence of bafilomycin A1 (Baf&#x02010;1a, 120&#x000a0;nM) for 24&#x000a0;h. Lysosomal activity was assessed by &#x003b2;&#x02010;hexosaminidase activity in cell lysates. Relative activity in treated versus untreated samples was measured and mean values &#x000b1; SD from two independent experiments are shown. (G) Fold change in gene expression (log2) of lysosome related genes in JB2 (6 and 24&#x000a0;h) versus DMSO in MDA&#x02010;MB&#x02010;468 cells, measured by RNAseq. `*` indicates significantly regulated genes in JB2 versus DMSO, evaluated by Limma. (H&#x02010;J) MITF targets over&#x02010;representation in the RNAseq was determined by ChEA3, using the ReMap dataset. Shown is the p&#x02010;value of the over&#x02010;representation, adjusted for multiple testing (H, left). The effect of JB2 and JB6 on MITF mRNA level was measured by qPCR, 6&#x000a0;h post&#x02010;treatment (4 repeats, <em>p</em>&#x02010;values evaluated by one&#x02010;sample t&#x02010;test) (H, right). Nuclear localization of MITF or TFEB was assessed 24&#x000a0;h post JB2 or JB6 treatment by either IF analysis, Scale bar, 10&#x000a0;&#x000b5;m, (I), or by cell fractionation (J). Histone 3 (H3) and GAPDH were used as loading control for nuclear (Nuc) or cytosolic (Cyt) fractions (J). (K) Accumulation of ferritin in the lysosome of JB2&#x02010;treated MDA&#x02010;MB&#x02010;468 cells at 24&#x000a0;h as shown by co&#x02010;staining of ferratin and lysotracker. Scale bar, 5&#x000a0;&#x000b5;m. (L) Blocking the lysosomal activity by Baf&#x02010;1 (100&#x000a0;nM) suppressed the increase in iron levels in JB2&#x02010;treated MDA&#x02010;MB&#x02010;468 cells. Baf&#x02010;1 was incubated with the indicated drugs for 24&#x000a0;h and intracellular iron levels were measured by colorimetric assay. Two independent experiments were performed; Shown are mean values &#x000b1; SD relative to the control of a representative experiment (done in duplicates). <em>p</em>&#x02010;values are evaluated by t&#x02010;test. **<em>p</em>&#x02010;value&#x000a0;&#x0003c;&#x000a0;0.01, ***<em>p</em>&#x02010;value&#x000a0;&#x0003c;&#x000a0;0.001.</p></div></div></div><p class="p p-last">Among the 96 lysosomal upregulated genes in JB2, 51 are MITF targets and 19 are TFEB targets (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0006/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0006" rid-ob="ob-advs7707-fig-0006" co-legend-rid="lgnd_advs7707-fig-0006"><span>6G</span></a>). MITF, TFEB, TFEC, and TFE3 belong to the microphthalmia&#x02010;associated family of basic helix loop helix (b&#x02010;HLH) leucine zipper transcription factors (MiT/TFE) which regulate the expression of lysosomal and autophagy genes and modulate lysosomal response.<sup>[</sup>
<sup><a href="#advs7707-bib-0036" rid="advs7707-bib-0036" class=" bibr popnode">
36
</a></sup>
<sup>]</sup> The upregulation of MITF/TFEB&#x02010;target genes in JB2 and their enrichment compared to JB6 (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0006/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0006" rid-ob="ob-advs7707-fig-0006" co-legend-rid="lgnd_advs7707-fig-0006"><span>6H</span></a>, left) was associated with upregulation of MITF expression (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0006/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0006" rid-ob="ob-advs7707-fig-0006" co-legend-rid="lgnd_advs7707-fig-0006"><span>6H</span></a>, right), and with MITF/TFEB translocation into the nucleus of JB2&#x02010;treated cells (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0006/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0006" rid-ob="ob-advs7707-fig-0006" co-legend-rid="lgnd_advs7707-fig-0006"><span>6I,J</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S7C</a>, Supporting Information). The enhanced lysosomal activity was accompanied by a robust accumulation of ferritin in the lysosome (Figure&#x000a0; <a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0006/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0006" rid-ob="ob-advs7707-fig-0006" co-legend-rid="lgnd_advs7707-fig-0006"><span>6K</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S7B</a>, Supporting Information). This accumulation implies that ferritin is degraded in the lysosome of JB2&#x02010;treated cells through ferritinophagy<sup>[</sup>
<sup><a href="#advs7707-bib-0037" rid="advs7707-bib-0037" class=" bibr popnode">
37
</a></sup>
<sup>]</sup> to increase the labile iron pool. Indeed, we observed a strong increase in lysosomal iron, as detected by Ferro&#x02010;orange staining (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S7D,E</a>, Supporting Information) as well as an increase in total iron levels in JB2&#x02010;treated cells which was markedly reduced by Bafilomycin A1 treatment (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0006/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0006" rid-ob="ob-advs7707-fig-0006" co-legend-rid="lgnd_advs7707-fig-0006"><span>6L</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S7F</a>, Supporting Information), a drug that prevents lysosomal acidification and ferritinophagy. Bafilomycin A1, as well as the lysosomal inhibitor chloroquine, could partially rescue cell viability of JB2&#x02010; but not of JB6&#x02010;treated cells (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S7G</a>, Supporting Information), highlighting the impact of the lysosome on JB2 response.</p></div><div id="advs7707-sec-0100" class="sec"><h3 id="advs7707-sec-0100title">2.8. Ferroptosis&#x02010;Apoptosis Transition is Associated with XBP1, JNK and ATF4/CHOP Activation</h3><p class="p p-first">The enrichment of ER&#x02010;associated pathways was obtained in response to JB2, JB3, or JB6 treatments (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0005" rid-ob="ob-advs7707-fig-0005" co-legend-rid="lgnd_advs7707-fig-0005"><span>5A</span></a>), most strongly to JB6 (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0005" rid-ob="ob-advs7707-fig-0005" co-legend-rid="lgnd_advs7707-fig-0005"><span>5C</span></a>). ER&#x02010;stress response was reported to be associated with apoptosis, ferroptosis, and their crosstalk,<sup>[</sup>
<sup><a href="#advs7707-bib-0004" rid="advs7707-bib-0004" class=" bibr popnode">
4
</a></sup><sup>, </sup><sup><a href="#advs7707-bib-0038" rid="advs7707-bib-0038" class=" bibr popnode">
38
</a></sup>
<sup>]</sup> and is known to be induced by proteasomal inhibition. Bortezomib was reported to induce accumulation of misfolded proteins in the ER,<sup>[</sup>
<sup><a href="#advs7707-bib-0039" rid="advs7707-bib-0039" class=" bibr popnode">
39
</a></sup>
<sup>]</sup> depletion of intracellular amino acids, and activation of GCN2 (general control nonderepressible 2), which in turn phosphorylates the eukaryotic translation initiation factor 2 (eIF2&#x003b1;) to increase ATF4 translation<sup>[</sup>
<sup><a href="#advs7707-bib-0040" rid="advs7707-bib-0040" class=" bibr popnode">
40
</a></sup>
<sup>]</sup> (<strong>Figure</strong>&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007" co-legend-rid="lgnd_advs7707-fig-0007"><span>
<strong>7</strong>A</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="advs7707-fig-0007" co-legend-rid="lgnd_advs7707-fig-0007"><a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140165988596032"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ADVS-11-2307263-g009.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is ADVS-11-2307263-g009.jpg" src="/pmc/articles/PMC11077643/bin/ADVS-11-2307263-g009.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140165988596032"><a target="object" rel="noopener" href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_advs7707-fig-0007"><div><a class="figpopup" href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007">Figure 7</a></div><!--caption a7--><div class="caption"><p>Ferroptosis (JB2)&#x02010;to apoptosis (JB6) transition is associated with XBP1, JNK, and ATF4/CHOP activation. (A) Schematic diagram of key pathways in ER stress response. ER stress and/or amino acid deprivation induces the unfolded protein response (UPR) and consequently IRE1&#x003b1; activation. Activated IRE1&#x003b1; cleaves XBP1 mRNA to produce an active XBP1s transcription factor and indirectly activates the ASK1&#x02010;JNK axis and apoptotic response. PERK phosphorylates eIF2&#x003b1; to inhibit protein translation, of which ATF4 escapes. ATF4 induces the expression of CHOP and GADD34, a phosphatase that dephosphorylates eIF2&#x003b1; to resume protein translation. ATF4 regulates chaperons, ER&#x02010;associated degradation (ERAD), and antioxidant gene transcription. ATF4 also cooperates with CHOP, which plays a role in ER&#x02010;stress&#x02010;induced apoptosis. (B,C) Activation of IRE1&#x003b1; in response to JB6, B6, or tunicamycin (Tm; 1&#x000a0;&#x000b5;g&#x000a0;mL<sup>&#x02212;1</sup>, a positive control), but not to JB2 treatment, for 24&#x000a0;h in MDA&#x02010;MB&#x02010;468 cell. (B) XBP1 splicing was detected by RT&#x02010;PCR. The unspliced (Un.) and spliced (Sp.) isoforms are shown. (C) WB analysis demonstrates JNK phosphorylation (pJNK) as well as total JNK, IRE1&#x003b1;, and PERK. GAPDH was used for loading control. (D) Inhibition of JNK (by SP600125, 5&#x000a0;&#x000b5;m) partially restored cell viability of B6&#x02010; or JB6&#x02010;treated cells. Cell viability was measured by MTT 60&#x000a0;h post&#x02010;treatment. Shown are means &#x000b1; SD of an experiment done in triplicates. P&#x02010;values were measured by t&#x02010;test. (E,F) Translocation of ATF4 (E) or ATF4/CHOP (F) into the nucleus of HCC70 and MDA&#x02010;MB&#x02010;468 cells at 24&#x000a0;h post&#x02010;treatment with the indicated drugs. Scale bar, 10&#x000a0;&#x000b5;m (for E), 5&#x000a0;&#x000b5;m (for E). (G) PPP1R15A expression levels in response to the applied drugs for 24&#x000a0;h was determined by qPCR analysis and shown as fold of untreated control. The mean values of two independent experiments are shown. (H) Mean fold change of gene expression in JB2/JB6 (24&#x000a0;h) versus DMSO in MDA&#x02010;MB&#x02010;468 and HCC70 was calculated. Enrichment scores of ATF4 and CHOP targets were measured by ssGSEA. (I) Fold change in gene expression (log2) of ATF4&#x02010;only targets in JB2 (24&#x000a0;h) versus DMSO in MDA&#x02010;MB&#x02010;468 and HCC70 cells, as measured by RNAseq. `*` indicates significantly (FDR &#x0003c; 0.05) regulated genes in JB2 versus DMSO, evaluated by Limma.( J) Heatmap depicts fold change (log2) in gene expression of JB2, JB3, and JB6 versus DMSO, taken from the RNAseq analysis. Top: genes of the aminoacyl&#x02010;tRNA synthetases, among the highly enriched pathway in response to CHOP. Bottom: Apoptosis&#x02010;related genes. Black dots above heatmaps indicate known CHOP targets. Scale bars indicate log2 fold change versus DMSO. (K) CHOP (DDIT3) was knocked down in MDA&#x02010;MB&#x02010;468 and HCC70 using siRNA and 24&#x000a0;h later the cells were treated with the indicated drug combinations. Cell viability was measured by MTT 60&#x000a0;h later. Shown are means &#x000b1; SD of two independent experiments done in duplicates. <em>p</em>&#x02010;values were measured by t&#x02010;test. *<em>p</em>&#x02010;value&#x000a0;&#x0003c;&#x000a0;0.05, ***<em>p</em>&#x02010;value&#x000a0;&#x0003c;&#x000a0;0.001, ****<em>p</em>&#x02010;value&#x000a0;&#x0003c;&#x000a0;1&#x000d7;10<sup>&#x02212;4</sup>.</p></div></div></div><p>ER stress commonly triggers the unfolded protein responses (UPR) through three main pathways; the IRE1&#x003b1; (inositol&#x02010;requiring protein 1&#x003b1;)&#x02010;XBP1 (X box binding protein 1) pathway, PERK (PKR&#x02010;like ER kinase)&#x02010;ATF4, and the ATF6 (activating transcription factor 6) pathway<sup>[</sup>
<sup><a href="#advs7707-bib-0041" rid="advs7707-bib-0041" class=" bibr popnode">
41
</a></sup>
<sup>]</sup> (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007" co-legend-rid="lgnd_advs7707-fig-0007"><span>7A</span></a>). To evaluate the role of ER&#x02010;stress in ferroptotic/apoptotic (JB2/JB6)&#x02010;death responses, we examined different arms of the UPR. As shown, JB6 and B6 (6&#x000a0;nM, BTZ), but not JB2, enhanced the RNase activity of IRE1&#x003b1; as indicated by mRNA processing of the unspliced XBP1 into an active spliced isoform (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007" co-legend-rid="lgnd_advs7707-fig-0007"><span>7B</span></a>).<sup>[</sup>
<sup><a href="#advs7707-bib-0042" rid="advs7707-bib-0042" class=" bibr popnode">
42
</a></sup>
<sup>]</sup> Likewise, JNK phosphorylation (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007" co-legend-rid="lgnd_advs7707-fig-0007"><span>7C</span></a>), which is implicated in apoptosis execution downstream of IRE1&#x003b1;,<sup>[</sup>
<sup><a href="#advs7707-bib-0042" rid="advs7707-bib-0042" class=" bibr popnode">
42
</a></sup>
<sup>]</sup> was also increased in response to JB6/B6 treatment concomitant with an increase in IRE1&#x003b1; and PERK levels (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007" co-legend-rid="lgnd_advs7707-fig-0007"><span>7C</span></a>), while JNK inhibitor (SP600125) substantially restored cell viability of B6/JB6&#x02010; but not of JB2&#x02010;treated cells (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007" co-legend-rid="lgnd_advs7707-fig-0007"><span>7D</span></a>).</p><p>ATF4, a master regulator of stress response, is activated downstream of PERK or GCN2 kinase, which both phosphorylate eIF2&#x003b1; to repress global protein translation and concomitantly enhance translation of a subset of stress&#x02010;related mRNAs, including <em>ATF4</em> (<em>CREB2</em>).<sup>[</sup>
<sup><a href="#advs7707-bib-0043" rid="advs7707-bib-0043" class=" bibr popnode">
43
</a></sup>
<sup>]</sup> Cellular fractionation showed enhanced translocation of ATF4 into the nucleus of both JB2&#x02010; and JB6&#x02010;treated cells (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S8A,B</a>, Supporting Information). The nuclear localization of ATF4 was detected earlier in JB6&#x02010;treated cells (at 6&#x000a0;h) (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S8C</a>, Supporting Information) and later in JB2&#x02010;treated cells (24&#x000a0;h) (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007" co-legend-rid="lgnd_advs7707-fig-0007"><span>7E</span></a>), suggesting that both ferroptotic (JB2) and apoptotic (JB6) responses are associated with ATF4 activation.</p><p>ATF4 induces transcription of genes involved in amino acid metabolism, glutaminolysis, autophagy, and oxidative stress, and it also regulates the expression of the proapoptotic transcription factor CHOP (GADD153/DDIT3).<sup>[</sup>
<sup><a href="#advs7707-bib-0044" rid="advs7707-bib-0044" class=" bibr popnode">
44
</a></sup>
<sup>]</sup> Importantly, CHOP was strongly detected in the nucleus of JB6&#x02010; but not of JB2&#x02010;treated cells (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007" co-legend-rid="lgnd_advs7707-fig-0007"><span>7F</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S8D</a>, Supporting Information), consistent with their apoptotic response.</p><p>ATF4 and CHOP cooperate to induce the expression of GADD34 (PPP1R15A) and upregulation of GADD34 (growth arrest and DNA damage&#x02010;inducible 34). GADD34 dephosphorylates eIF2&#x003b1; to increase protein synthesis which leads to the overloading of unfolded proteins within the ER and apoptotic cell death.<sup>[</sup>
<sup><a href="#advs7707-bib-0045" rid="advs7707-bib-0045" class=" bibr popnode">
45
</a></sup>
<sup>]</sup> We observed strong induction of PPP1R15A expression in JB6&#x02010; versus JB2&#x02010;treated cells (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007" co-legend-rid="lgnd_advs7707-fig-0007"><span>7G</span></a>), concomitant with a decrease in eIF2&#x003b1; phosphorylation (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S8E</a>, Supporting Information). Single&#x02010;sample GSEA (ssGSEA) analysis of the combined DEGs in MDA&#x02010;MB&#x02010;468 and HCC70 revealed a slightly higher enrichment of ATF4&#x02010;only target genes<sup>[</sup>
<sup><a href="#advs7707-bib-0045" rid="advs7707-bib-0045" class=" bibr popnode">
45
</a></sup>
<sup>]</sup> in JB2, but much stronger enrichment of CHOP&#x02010;target genes in JB6 (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007" co-legend-rid="lgnd_advs7707-fig-0007"><span>7H</span></a>), implying that ATF4/CHOP activation is associated with ER&#x02010;stress induced apoptosis, consistent with the nuclear translocation of CHOP (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007" co-legend-rid="lgnd_advs7707-fig-0007"><span>7F</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S8D</a>, Supporting Information). Indeed, the expression levels of many ferroptosis regulators, which are ATF4 targets, were increased by JB2 (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007" co-legend-rid="lgnd_advs7707-fig-0007"><span>7I</span></a>), while translation&#x02010;related genes, mainly those involved in tRNA synthesis, which are known ATF4/CHOP targets,<sup>[</sup>
<sup><a href="#advs7707-bib-0045" rid="advs7707-bib-0045" class=" bibr popnode">
45
</a></sup>
<sup>]</sup> were mainly increased in JB6 (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007" co-legend-rid="lgnd_advs7707-fig-0007"><span>7J</span></a>). Furthermore, known pro&#x02010;apoptotic CHOP targets (<em>BCL2L11</em> [BIM], <em>BBC3</em> [PUMA], <em>PMAIP1</em> [NOXA], <em>TRIB3</em>)<sup>[</sup>
<sup><a href="#advs7707-bib-0046" rid="advs7707-bib-0046" class=" bibr popnode">
46
</a></sup>
<sup>]</sup> were also upregulated in JB6&#x02010;treated cells (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007" co-legend-rid="lgnd_advs7707-fig-0007"><span>7J</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S8F</a>, Supporting Information), highlighting the impact of CHOP on the apoptotic response. Indeed, CHOP knockdown by siRNA (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S8G</a>, Supporting Information) could partially restore cell viability following treatment by JB6/B6, but not JB2 (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007" co-legend-rid="lgnd_advs7707-fig-0007"><span>7K</span></a>). Likewise, Pioglitazone, which inhibits ER stress ,<sup>[</sup>
<sup><a href="#advs7707-bib-0047" rid="advs7707-bib-0047" class=" bibr popnode">
47
</a></sup>
<sup>]</sup> partially rescued the viability of JB6&#x02010; but not of JB2&#x02010;treated cells (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S8H</a>, Supporting Information) and concomitantly attenuated translocation of CHOP to the nucleus (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S8I</a>, Supporting Information).</p><p class="p p-last">Collectively, we found that ATF4 is activated both in ferroptosis and apoptosis, possibly in different kinetics, but the concomitant activation of XBP1, JNK, and CHOP switches to chronic ER stress and apoptotic response.</p></div><div id="advs7707-sec-0110" class="sec"><h3 id="advs7707-sec-0110title">2.9. Ferroptotic Response is Associated with Enhanced Glutaminolysis and TCA Cycle</h3><p class="p p-first">Several ATF4 target genes, which were upregulated in JB2, regulate glutamine transport or metabolism (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007" co-legend-rid="lgnd_advs7707-fig-0007"><span>7I</span></a>), including the two glutamine transporters SLC38A2 and SLC1A5 and the glutamate transporter SLC1A4.<sup>[</sup>
<sup><a href="#advs7707-bib-0048" rid="advs7707-bib-0048" class=" bibr popnode">
48
</a></sup>
<sup>]</sup> Importantly, glutaminase 1 (GLS), which converts glutamine to glutamate was also upregulated in JB2&#x02010;treated cells (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007" co-legend-rid="lgnd_advs7707-fig-0007"><span>7I</span></a>), implying that glutamine metabolism plays an important role in the JB2 response.</p><p>To assess the impact of glutamine on death response, we incubated MDA&#x02010;MB&#x02010;468 (<strong>Figure</strong>&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0008" rid-ob="ob-advs7707-fig-0008" co-legend-rid="lgnd_advs7707-fig-0008"><span>
<strong>8</strong>A</span></a>) or HCC70 cells (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S9A</a>, Supporting Information) in glutamine (Gln)&#x02010;free media and assessed cell viability in response to JB2 or JB6 treatment. We found that Gln&#x02010;depleted media specifically abrogated JB2&#x02010;induced death, while the addition of &#x003b1;&#x02010;Ketoglutarate (&#x003b1;&#x02010;KG) restored cell death in Gln&#x02010;free media (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0008" rid-ob="ob-advs7707-fig-0008" co-legend-rid="lgnd_advs7707-fig-0008"><span>8A,B</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S9A</a>, Supporting Information). These results suggest that &#x003b1;&#x02010;KG, a TCA cycle intermediate, which can be produced from glutamate through glutaminolysis,<sup>[</sup>
<sup><a href="#advs7707-bib-0049" rid="advs7707-bib-0049" class=" bibr popnode">
49
</a></sup>
<sup>]</sup> and supports oxidative phosphorylation (OXPHOS), plays a role in the JB2&#x02010;ferroptotic response. Indeed, inhibition of &#x003b1;&#x02010;KG production from glutamate using the glutamate transaminase inhibitor, aminooxyacetate (AOA) (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0008" rid-ob="ob-advs7707-fig-0008" co-legend-rid="lgnd_advs7707-fig-0008"><span>8C,D</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S9B</a>, Supporting Information)<sup>[</sup>
<sup><a href="#advs7707-bib-0050" rid="advs7707-bib-0050" class=" bibr popnode">
50
</a></sup>
<sup>]</sup> fully suppressed cell death in response to JB2, but not to JB6.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="advs7707-fig-0008" co-legend-rid="lgnd_advs7707-fig-0008"><a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/" target="figure" rid-figpopup="advs7707-fig-0008" rid-ob="ob-advs7707-fig-0008"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140165988509552"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is ADVS-11-2307263-g005.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is ADVS-11-2307263-g005.jpg" src="/pmc/articles/PMC11077643/bin/ADVS-11-2307263-g005.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140165988509552"><a target="object" rel="noopener" href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_advs7707-fig-0008"><div><a class="figpopup" href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/" target="figure" rid-figpopup="advs7707-fig-0008" rid-ob="ob-advs7707-fig-0008">Figure 8</a></div><!--caption a7--><div class="caption"><p>Ferroptotic response of JB2 is associated with metabolic shifts. (A&#x02013;D) Glutaminolysis is necessary for JB2 mediated ferroptosis. (A) MDA&#x02010;MB&#x02010;468 cells were treated with the indicated drugs in either normal (Mock), glutamine&#x02010;free media (&#x02010;Gln), or with glutamine&#x02010;free media plus 5&#x000a0;mM &#x003b1;&#x02010;ketoglutarate (&#x02010;Gln+ &#x003b1;KG). Cell viability was measured after 72&#x000a0;h by MTT and is shown relative to control (%). Mean values &#x000b1; SD from two independent experiments done in duplicates. P&#x02010;values measured by t&#x02010;test. (B) Representative images of cells described in A, 72&#x000a0;h post&#x02010;treatment. Scale bar, 50&#x000a0;&#x000b5;m. (C) Chart pathway showing the TCA cycle together with the anaplerotic reaction mediated by transaminases such as GOT2 and GPT2, which convert glutamate to &#x003b1;KG using oxaloacetate/pyruvate, and can be blocked by aminooxyacetic acid (AOA). Acytel&#x02010;coA is also fueling the TCA cycle through glycolysis and fatty acid oxidation. (D) Cells described in A were treated with the indicated drugs in the absence or presence of AOA inhibitor (0.25&#x000a0;mM). Cell viability was measured 72&#x000a0;h later by MTT and is shown relative to control (%). Mean values &#x000b1; SD from two independent experiments done in duplicates. P&#x02010;values measured by t&#x02010;test. (E,F) Blocking the mitochondria ETC rescued JB2&#x02010;induced ferroptosis. Representative confocal images of MDA&#x02010;MB&#x02010;468 or HCC70 cells treated with the indicated drugs with or without rotenone for 16&#x000a0;h. Cells were stained with Mitotracker (red) and C11&#x02010;BODIPY (green oxidized). Scale bar, 5&#x000a0;&#x000b5;m. (G,H) Cells described in A were treated with the indicated drugs in the absence or presence of rotenone or CCCP in MDA&#x02010;MB&#x02010;468 (G) and HCC70 (H). Cell death was measured after 60&#x000a0;h by trypan blue exclusion. Mean values &#x000b1; SD of % rescue of cell death induced by the JB combinations, from two independent experiments are shown. <em>p</em>&#x02010;values measured by t&#x02010;test. (I) Enhanced mitochondria lipid peroxidation in JB2 cells. MDA&#x02010;MB&#x02010;468 cells treated with drugs as indicated in the presence or absence of 0.5&#x000a0;mM AOA for 16&#x02013;18&#x000a0;h, stained with MitoPeDPP probe to detect lipid peroxidation within mitochondria membrane. Shown are representative confocal images. Scale bar, 5&#x000a0;&#x000b5;m. (J,K) JB2 treatment decreased the intracellular level of GSH (J) and the protein level of the xCT subunit SLC7A11 (K) in MDA&#x02010;MB&#x02010;468 (J,K) or HCC70 (J). Cells were treated with the indicated drugs and 24&#x000a0;h later the level of SLC7A11was assessed by WB (K), while GSH levels were measured by colorimetric assay and are shown relative to control (J). Shown are mean values &#x000b1; SD from two independent experiments. P&#x02010;values measured by t&#x02010;test. (L) Gluthamine&#x02010;C<sup>13</sup> tracing in GSH and TCA cycle metabolites. Shown is the total carbon intensity in JB2 versus DMSO of all +2 traces and above. The sum of intensities is weighted (each intensity is multiplied by the number of marked carbons). Shown is the mean &#x000b1; SD of 4&#x02013;5 repeats. P&#x02010;values were determined by t&#x02010;test, comparing JB2 (5 repeats) to DMSO (4 repeats). (M) A polar metabolite screen was performed in JB2 and DMSO. KEGG pathways enrichment for metabolites was performed using Metaboanalyst.Ca. (N,O) Plots show the fold change in normalized intensities of metabolites in JB2 versus DMSO (3&#x02013;5 individual repeats). <em>p</em>&#x02010;values were determined by t&#x02010;test, comparing JB2 to DMSO. P,Q) Lipidomics profile was performed for JB2, JB6, and DMSO control&#x02010;treated MDA&#x02010;MB&#x02010;468 cells (<em>n</em>&#x000a0;=&#x000a0;3 independent repeats). (P) Heatmap represents the ferroprosis&#x02010;related lipids, 24&#x000a0;h post&#x02010;treatment (scaled lipid levels). A T&#x02010;test was used to evaluate p&#x02010;values. (Q) Fold change in phosphatidylethanolamine (PE, blue dots) and phosphatidylcholine (PC, red dots) in JB6 versus JB2 and DMSO, 24&#x000a0;h post&#x02010;treatment. Labeled are lipids relevant to ferroptosis. *<em>p</em>&#x02010;value&#x000a0;&#x0003c;&#x000a0;0.05,**<em>p</em>&#x02010;value&#x000a0;&#x0003c;&#x000a0;0.01, ***<em>p</em>&#x02010;value&#x000a0;&#x0003c;&#x000a0;0.001, ****<em>p</em>&#x02010;value&#x000a0;&#x0003c;&#x000a0;1&#x000d7;10<sup>&#x02212;4</sup>.</p></div></div></div><p>&#x003b1;&#x02010;KG participates in the TCA cycle, which is highly enriched in the JB2 versus JB6 transcriptomic response (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0005" rid-ob="ob-advs7707-fig-0005" co-legend-rid="lgnd_advs7707-fig-0005"><span>5B</span></a>) and is also required for electron transport through the mitochondrial electron transfer chain (ETC). We, therefore, evaluated the effects of the ETC inhibitors, Rotenone, a mitochondrial complex I inhibitor, or the mitochondrial uncoupler CCCP (carbonyl cyanide m&#x02010;chlorophenyl hydrazine) (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0008" rid-ob="ob-advs7707-fig-0008" co-legend-rid="lgnd_advs7707-fig-0008"><span>8F&#x02013;H</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S9C</a>, Supporting Information),<sup>[</sup>
<sup><a href="#advs7707-bib-0051" rid="advs7707-bib-0051" class=" bibr popnode">
51
</a></sup>
<sup>]</sup> on JB2&#x02010;death response. We found that both ETC inhibitors attenuated the death response of JB2 but not of JB6 or B6. Furthermore, we observed extensive colocalization between oxidized BODIPY&#x02010;C11 and MitoTracker in JB2&#x02010; but not in JB6&#x02010;treated cells (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0008" rid-ob="ob-advs7707-fig-0008" co-legend-rid="lgnd_advs7707-fig-0008"><span>8E</span></a>), which was completely abolished in the presence of rotenone (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0008" rid-ob="ob-advs7707-fig-0008" co-legend-rid="lgnd_advs7707-fig-0008"><span>8F</span></a>). AOA treatment also reduced the level of mitochondrial lipid ROS in JB2&#x02010;treated cells (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0008" rid-ob="ob-advs7707-fig-0008" co-legend-rid="lgnd_advs7707-fig-0008"><span>8I</span></a>). Importantly, MitoTracker staining revealed a unique morphology of swollen mitochondria in JB2&#x02010;treated cells (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S9D</a>, Supporting Information), a characteristic feature of ferroptotic cells.<sup>[</sup>
<sup><a href="#advs7707-bib-0011" rid="advs7707-bib-0011" class=" bibr popnode">
11
</a></sup>
<sup>]</sup> Collectively these results suggest that JB2&#x02010;cell death is mediated by extensive accumulation of lipid ROS in the mitochondria due to a metabolic switch associated with glutamine&#x02010;driven OXPHOS. These results are consistent with previous reports demonstrating the role of glutamine on ferroptosis through the TCA cycle.<sup>[</sup>
<sup><a href="#advs7707-bib-0051" rid="advs7707-bib-0051" class=" bibr popnode">
51
</a></sup>
<sup>]</sup>
</p><p>In addition to the anaplerotic role of glutamine in the TCA cycle, glutamine/glutamate are required for biosynthesis of glutathione, for cystine import via the System xc<sup>&#x02212;</sup>, for synthesis of several nonessential amino acids (alanine, proline, aspartate, asparagine, and arginine), as well as of pyrimidines, purines, hexosamines, and NAD.<sup>[</sup>
<sup><a href="#advs7707-bib-0052" rid="advs7707-bib-0052" class=" bibr popnode">
52
</a></sup>
<sup>]</sup> Importantly, we observed significant decreased levels of GSH in JB2&#x02010;treated MDA&#x02010;MB&#x02010;468/HCC70 cells (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0008" rid-ob="ob-advs7707-fig-0008" co-legend-rid="lgnd_advs7707-fig-0008"><span>8J</span></a>) concomitant with a substantial decrease in SLC7A11 protein level (a subunit of System xc<sup>&#x02212;</sup>) (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0008" rid-ob="ob-advs7707-fig-0008" co-legend-rid="lgnd_advs7707-fig-0008"><span>8K</span></a>), which both can contribute to ferroptosis execution.</p><p class="p p-last">The impact of glutamine on the TCA cycle and GSH biosynthesis was further corroborated by <sup>13</sup>C&#x02010;glutamine tracing experiment. MDA&#x02010;MB&#x02010;468 were incubated with <sup>13</sup>C&#x02010;glutamine as a tracer and the levels of <sup>13</sup>C&#x02010;labeled TCA intermediates were examined 16&#x000a0;h post JB2 treatment. We observed significant <sup>13</sup>C&#x02010;labeling of the TCA cycle intermediates succinate, malate, and fumarate (M4) as well as citrate and cis&#x02010;aconitate (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0008" rid-ob="ob-advs7707-fig-0008" co-legend-rid="lgnd_advs7707-fig-0008"><span>8L</span></a>, <a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S9E</a>). We also observed the labeling of glutathione (GSH) and gamma&#x02010;glutamylcysteine (GCC; a GSH biosynthesis intermediate) (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0008" rid-ob="ob-advs7707-fig-0008" co-legend-rid="lgnd_advs7707-fig-0008"><span>8L</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S9E</a>, Supporting Information). Collectively, these results demonstrate the critical role of glutamine in JB2&#x02010;induced ferroptosis.</p></div><div id="advs7707-sec-0120" class="sec sec-last"><h3 id="advs7707-sec-0120title">2.10. Metabolic State of Ferroptosis and its Transition into Apoptosis is Associated with PE/PC Switch</h3><p class="p p-first">To characterize the effects of JB2 on cellular metabolism, we performed metabolomics and lipidomic profiling of MDA&#x02010;MB&#x02010;468 cells treated with JB2 or JB6 for 16 and 24&#x000a0;h (Tables&#x000a0;<a href="#advs7707-supitem-0002" rid="advs7707-supitem-0002" class=" supplementary-material">S3,S4</a>, Supporting Information). We observed significant upregulation of purine and pyrimidine metabolites and the TCA&#x02010;cycle intermediates in response to JB2 (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0008" rid-ob="ob-advs7707-fig-0008" co-legend-rid="lgnd_advs7707-fig-0008"><span>8M,O</span></a>), concomitant with a remarkable downregulation of amino&#x02010;acids and derivates at both time points, and further robust reduction in glutamine, argininosuccinate, cystathionine, taurine and hypotaurine at 24&#x000a0;h (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0008" rid-ob="ob-advs7707-fig-0008" co-legend-rid="lgnd_advs7707-fig-0008"><span>8N</span></a>). Hence, the ferroptotic (JB2)&#x02010;death response was associated with global amino&#x02010;acids deprivation, and predominantly affected cysteine, glutamine, and arginine metabolism. Although cysteine levels were not measured, cystathionine, an intermediate of cysteine biosynthesis from homocysteine and serine through the transsulfuration pathway<sup>[</sup>
<sup><a href="#advs7707-bib-0053" rid="advs7707-bib-0053" class=" bibr popnode">
53
</a></sup>
<sup>]</sup> as well as hypotaurine and taurine, which are generated by the cysteine dioxygenase pathway,<sup>[</sup>
<sup><a href="#advs7707-bib-0054" rid="advs7707-bib-0054" class=" bibr popnode">
54
</a></sup>
<sup>]</sup> were markedly reduced. Argininosuccinate, an intermediate in arginine biosynthesis from citrulline and aspartate in the urea cycle,<sup>[</sup>
<sup><a href="#advs7707-bib-0055" rid="advs7707-bib-0055" class=" bibr popnode">
55
</a></sup>
<sup>]</sup> was also reduced (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0008" rid-ob="ob-advs7707-fig-0008" co-legend-rid="lgnd_advs7707-fig-0008"><span>8N</span></a>).</p><p>Similar to JB2&#x02010;treated cells, we also observed a global reduction in amino acid levels in JB6&#x02010;treated cells (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S9F</a>, Supporting Information) possibly due to the strong inhibition of the proteasome (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0001" rid-ob="ob-advs7707-fig-0001" co-legend-rid="lgnd_advs7707-fig-0001"><span>1J</span></a>). However, unlike JB6, upregulation of alanine was detected only in JB2&#x02010;treated cells, consistent with the enhanced glutaminolysis and glutamate transamination, which can produce alanine from pyruvate by glutamate&#x000a0;pyruvate transaminase (GPT).<sup>[</sup>
<sup><a href="#advs7707-bib-0056" rid="advs7707-bib-0056" class=" bibr popnode">
56
</a></sup>
<sup>]</sup>
</p><p class="p p-last">As ferroptosis is mediated by lipid peroxidation of PUFAs,<sup>[</sup>
<sup><a href="#advs7707-bib-0057" rid="advs7707-bib-0057" class=" bibr popnode">
57
</a></sup>
<sup>]</sup> and JB2 modulates the expression of lipid metabolic genes (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0005" rid-ob="ob-advs7707-fig-0005" co-legend-rid="lgnd_advs7707-fig-0005"><span>5H</span></a>), we performed lipidomic profiling of JB2&#x02010; and JB6&#x02010;treated cells (Table&#x000a0;<a href="#advs7707-supitem-0002" rid="advs7707-supitem-0002" class=" supplementary-material">S4</a>, Supporting Information). As seen in Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0008" rid-ob="ob-advs7707-fig-0008" co-legend-rid="lgnd_advs7707-fig-0008"><span>8P</span></a>, JB2 treatment increased the level of key lipid species associated with ferroptosis execution, including PE&#x02010;containing arachidonic acid (C20:4), linoleic acid (C18:2), and adrenic acid (C22:4),<sup>[</sup>
<sup><a href="#advs7707-bib-0008" rid="advs7707-bib-0008" class=" bibr popnode">
8b
</a></sup>
<sup>]</sup> whereas none of these lipids were increased in JB6. The level of PE (18:0/20:4), which is highly susceptible to ferroptosis&#x02010;driven oxidation,<sup>[</sup>
<sup><a href="#advs7707-bib-0058" rid="advs7707-bib-0058" class=" bibr popnode">
58
</a></sup>
<sup>]</sup> was significantly increased in JB2&#x02010;treated cells (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0008" rid-ob="ob-advs7707-fig-0008" co-legend-rid="lgnd_advs7707-fig-0008"><span>8P</span></a>). Moreover, an assessment of PE/PC species indicated that the ferroptosis (JB2)&#x02010;to&#x02010; &#x02010;apoptosis (JB6) transition was coupled to a global shift from PE to PC (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0008" rid-ob="ob-advs7707-fig-0008" co-legend-rid="lgnd_advs7707-fig-0008"><span>8Q</span></a>). Together these observations highlight the robust impact of JB2 on metabolic hallmarks of ferroptosis and demonstrate the distinct metabolic switch associated with JB2 compared to JB6.</p></div></div><div id="advs7707-sec-0130" class="tsec sec"><h2 class="head no_bottom_margin" id="advs7707-sec-0130title">3. Conclusion</h2><p class="p p-first">In this study, we used a unique synthetic lethal drug combination targeting BRD4 and the proteasome to induce ferroptosis in basal&#x02010;like breast cancer. Subsequently, we established a concentration gradient and systematically reprogrammed a gradual phenotypic, transcriptomic, and metabolomic transition between ferroptosis and apoptosis (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0001" rid-ob="ob-advs7707-fig-0001" co-legend-rid="lgnd_advs7707-fig-0001"><span>1</span></a>).</p><p>The integrated omics approach together with mechanistic studies highlighted several key principles related to the three death modules, ferroptosis (JB2), apoptosis (JB6), and the intermediate &#x0201c;ferroapoptosis&#x0201d; (JB3). First, the transcriptomic profile of these JB2&#x02010;, JB3&#x02010;, and JB6&#x02010;induced death states is largely overlapping (&#x02248;80%), suggesting that they share a common stress response. Nevertheless, each manifests unique transcriptomic changes to induce a selective response (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0005" rid-ob="ob-advs7707-fig-0005" co-legend-rid="lgnd_advs7707-fig-0005"><span>5A&#x02013;C</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S6A</a>, Supporting Information). Second, the synergistic effect of drug combination is associated with robust transcriptomic landscape reprogramming as obtained for JB2 (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0005" rid-ob="ob-advs7707-fig-0005" co-legend-rid="lgnd_advs7707-fig-0005"><span>5D</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S6B,D</a>, Supporting Information). Third, the different death pathways are associated with differential activities in three cellular organelles, the ER, the lysosome, and the mitochondria (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0006/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0006" rid-ob="ob-advs7707-fig-0006" co-legend-rid="lgnd_advs7707-fig-0006"><span>6</span></a>, <a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007" co-legend-rid="lgnd_advs7707-fig-0007"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,7,</span></span><span>7</span></a>, <a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0008" rid-ob="ob-advs7707-fig-0008" co-legend-rid="lgnd_advs7707-fig-0008"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,8).</span></span><span>8</span></a>). Fourth, the transition between death pathways can be mediated by specific transcription factors and is associated with metabolic switch (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0006/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0006" rid-ob="ob-advs7707-fig-0006" co-legend-rid="lgnd_advs7707-fig-0006"><span>6H</span></a> and Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007" co-legend-rid="lgnd_advs7707-fig-0007"><span>7H</span></a>, Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0008" rid-ob="ob-advs7707-fig-0008" co-legend-rid="lgnd_advs7707-fig-0008"><span>8Q</span></a>).</p><p>The GGS, which was generated from the DEGs in the continuous JB2 (ferroptosis)&#x02010;to&#x02010;JB6 (apoptosis) transcriptomic landscape, is unique, unbiased, and robust. First, it accurately classifies ferroptosis from apoptosis responses of various FINs and AINs across multiple lineages (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2C</span></a>), despite the specific ferroptotic response of JB2 in TNBC,<sup>[</sup>
<sup><a href="#advs7707-bib-0008" rid="advs7707-bib-0008" class=" bibr popnode">
8a
</a></sup>
<sup>]</sup> suggesting that it represents the core gene set downstream of a convergence point of multiple FINs. Second, it was used to optimize a set of selective ferroptosis biomarkers to distinguish ferroptosis from apoptosis (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2D,E</span></a>). Third, a subset of GGS, the (&#x0201c;JB2 Up&#x0201d;) is associated with poor prognosis and chemotherapy resistance in breast cancer, particularly in TNBC (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0003/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0003" rid-ob="ob-advs7707-fig-0003" co-legend-rid="lgnd_advs7707-fig-0003"><span>3</span></a>), and thus can be used for patients&#x02019; stratification and treatment decisions.</p><p>The optimized biomarkers, the 26 &#x0201c;Gradient derived&#x0201d; and 15 &#x0201c;Dataset derived&#x0201d; displayed very similar classification accuracy of ferroptosis and apoptosis, despite their different source and biased/unbiased nature (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2D</span></a>). These biomarkers were combined into a list of 24 top&#x02010;ranked genes which were robustly validated in vitro and in vivo by qPCR, and further assessed in the 45 public datasets and the 5&#x02010;validation datasets of FINs in basal&#x02010;like breast cancer cells (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2F&#x02013;H</span></a>). This rigorous validation convincingly highlights the strength of these biomarkers as reliable transcriptomic tools to distinguish between ferroptosis to apoptosis death modules. These robust transcriptomic biomarkers might be used together with TfR1 immunostaining which was shown as a powerful approach to distinguish ferroptosis from apoptosis.<sup>[</sup>
<sup><a href="#advs7707-bib-0059" rid="advs7707-bib-0059" class=" bibr popnode">
59
</a></sup>
<sup>]</sup> Nevertheless, considering the different kinetic responses of various ferroptosis inducers and the multiple mechanisms to induce ferroptosis (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S3C</a>, Supporting Information), we expect a variable transcriptomic response for each individual biomarker, but concurrently, a significant enrichment of the 24 biomarkers as an entire set in a given time window (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2I</span></a>).</p><p>We found that the lysosomal pathways are specifically enriched in JB2 (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0006/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0006" rid-ob="ob-advs7707-fig-0006" co-legend-rid="lgnd_advs7707-fig-0006"><span>6</span></a>), and that MITF and TFEB, which are involved in lysosomal biogenesis were upregulated by JB2. Concomitantly, our RNAseq data indicate that <em>c&#x02010;MYC</em> was downregulated, and previous reports suggest that MYC competes with MITF/TFEB and suppresses lysosomal biogenesis and autophagy,<sup>[</sup>
<sup><a href="#advs7707-bib-0060" rid="advs7707-bib-0060" class=" bibr popnode">
60
</a></sup>
<sup>]</sup> suggesting that the MYC/MITF switch may attribute to lysosomal activation in JB2.</p><p>The ER processing/stress pathway was found to be commonly enriched by JB2&#x02010;induced ferroptosis and JB6&#x02010;induced apoptosis, albeit more strongly in apoptosis (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0005" rid-ob="ob-advs7707-fig-0005" co-legend-rid="lgnd_advs7707-fig-0005"><span>5A,C</span></a>, Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007" co-legend-rid="lgnd_advs7707-fig-0007"><span>7</span></a>). Activation of ATF4 (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007" co-legend-rid="lgnd_advs7707-fig-0007"><span>7E</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S8A&#x02013;C</a>, Supporting Information), a major transcription factor of the integrated stress response,<sup>[</sup>
<sup><a href="#advs7707-bib-0016" rid="advs7707-bib-0016" class=" bibr popnode">
16
</a></sup>
<sup>]</sup> was also observed by the two death modules. Nevertheless, the JB2 (ferroptosis)&#x02010;to&#x02010;JB6 (apoptosis) transition was coupled to XBP1, JNK, and ATF4/CHOP activation (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007" co-legend-rid="lgnd_advs7707-fig-0007"><span>7B&#x02013;F</span></a>). Activation of CHOP by ER stress and its further involvement in ER oxidation and apoptotic response was demonstrated by previous studies.<sup>[</sup>
<sup><a href="#advs7707-bib-0061" rid="advs7707-bib-0061" class=" bibr popnode">
61
</a></sup>
<sup>]</sup> ATF4&#x02010;CHOP overexpression was proposed to increase protein translation by upregulating ribosomal protein expression, which consequently caused to increase in protein synthesis, ATP depletion, oxidative stress, and apoptotic cell death.<sup>[</sup>
<sup><a href="#advs7707-bib-0045" rid="advs7707-bib-0045" class=" bibr popnode">
45
</a></sup>
<sup>]</sup> CHOP was also proposed to increase oxidative damage within the ER through upregulation of the ER oxidase 1 (ERO1), which activates the inositol 1,4,5&#x02010;triphosphate receptor (IP3R) to release calcium and consequently induce apoptotic cell death.<sup>[</sup>
<sup><a href="#advs7707-bib-0061" rid="advs7707-bib-0061" class=" bibr popnode">
61a
</a></sup>
<sup>]</sup>
</p><p>The strong activation of ATF4 by both JB6&#x02010; and JB2&#x02010;induced apoptosis and ferroptosis was associated with global amino acids deprivation in response to both death pathways (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0008" rid-ob="ob-advs7707-fig-0008" co-legend-rid="lgnd_advs7707-fig-0008"><span>8N</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S9F</a>, Supporting Information). However, several ATF4 target genes that were strongly changed in ferroptosis play roles in glutamine/glutamate transport. Indeed, we found enhanced glutaminolysis and upregulation in TCA cycle metabolites and genes (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0005" rid-ob="ob-advs7707-fig-0005" co-legend-rid="lgnd_advs7707-fig-0005"><span>5B</span></a>, Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0008" rid-ob="ob-advs7707-fig-0008" co-legend-rid="lgnd_advs7707-fig-0008"><span>8A,B,L,O</span></a>, Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S9A,B,E</a>, Supporting Information). We also found that inhibition of the anapleurotic reactions feeding the TCA cycle or the ETC was sufficient to rescue the cells from ferroptosis (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0008" rid-ob="ob-advs7707-fig-0008" co-legend-rid="lgnd_advs7707-fig-0008"><span>8A&#x02013;H</span></a>), similar to their effects on cysteine deprivation&#x02010;induced ferroptosis.<sup>[</sup>
<sup><a href="#advs7707-bib-0051" rid="advs7707-bib-0051" class=" bibr popnode">
51
</a></sup>
<sup>]</sup>
</p><p>Our finding that ER stress is associated with both JB2/JB6&#x02010;induced ferroptosis and apoptosis, but more prominent in apoptosis, is possibly related to the gradual inactivation of the proteasome activity over the concentration gradient. While high BTZ (6&#x000a0;nM) concentration strongly inhibited the proteasomal activity (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0001" rid-ob="ob-advs7707-fig-0001" co-legend-rid="lgnd_advs7707-fig-0001"><span>1J</span></a>) and consequently induced strong ER&#x02010;stress response and UPR (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007" co-legend-rid="lgnd_advs7707-fig-0007"><span>7B,C,E</span></a>), low BTZ concentration (B2) had significantly weaker effects on the proteasome activity (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0001" rid-ob="ob-advs7707-fig-0001" co-legend-rid="lgnd_advs7707-fig-0001"><span>1J</span></a>), ER stress and the overall transcriptome. Nevertheless, in the presence of a low JQ1 dose, both combinations of JB2 and JB6 induced strong stress response reflected by ATF4 activation (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0007" rid-ob="ob-advs7707-fig-0007" co-legend-rid="lgnd_advs7707-fig-0007"><span>7E</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S8A,B,C</a>, Supporting Information).</p><p>Importantly, depletion of PDAP1 in B cells also caused a sustained expression of ATF4 and induction of ATF4 stress response transcriptional program,<sup>[</sup>
<sup><a href="#advs7707-bib-0062" rid="advs7707-bib-0062" class=" bibr popnode">
62
</a></sup>
<sup>]</sup> and our data showed that PDAP1 depletion in basal&#x02010;like breast cancer induced ferroptotic cell death and significantly reduced tumor growth in xenograft model (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0004" rid-ob="ob-advs7707-fig-0004" co-legend-rid="lgnd_advs7707-fig-0004"><span>4</span></a>). Knockdown of PDAP1 in colorectal cancer also inhibited tumor growth in PDX model but had a minor effect, if at all, on apoptosis in cultured cells.<sup>[</sup>
<sup><a href="#advs7707-bib-0028" rid="advs7707-bib-0028" class=" bibr popnode">
28a
</a></sup>
<sup>]</sup> Likewise, PDAP1 depletion in breast tumors was associated with increased 4HNE staining, but not with cleaved PARP (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0004" rid-ob="ob-advs7707-fig-0004" co-legend-rid="lgnd_advs7707-fig-0004"><span>4T</span></a>), despite that the viability of PDAP1&#x02010;depleted MDA&#x02010;MB&#x02010;468 cells in vitro was partially restored by ZVAD (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S5C</a>, Supporting Information), implying a possible crosstalk between the two death pathways upon PDAP1 knockdown.<sup>[</sup>
<sup><a href="#advs7707-bib-0063" rid="advs7707-bib-0063" class=" bibr popnode">
63
</a></sup>
<sup>]</sup> Although the death module imposed by PDAP1 depletion could be cell type&#x02010;specific, we found that PDAP1 knockdown in basal&#x02010;like breast cancer cells induced canonical hallmarks of ferroptosis.<sup>[</sup>
<sup><a href="#advs7707-bib-0018" rid="advs7707-bib-0018" class=" bibr popnode">
18
</a></sup>
<sup>]</sup> Our mechanistic studies suggest that PDAP1 protects from ferroptosis by positively regulating the AKT&#x02010;mTOR&#x02010;SRBP1&#x02010;SCD1 signaling axis (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0004" rid-ob="ob-advs7707-fig-0004" co-legend-rid="lgnd_advs7707-fig-0004"><span>4N&#x02013;P</span></a>; Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S5I</a>, Supporting Information). This protective role is associated with the related transcriptomic response of PDAP1 knockdown and GPX4 depletion, as determined by CMAP analysis (Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S5A</a>, Supporting Information), and thus introduced PDAP1 as a novel ferroptosis regulator. Indeed, PDAP1 is one of the GGS genes and belongs to the &#x0201c;JB2 Up&#x0201d; subset consisting of potential ferroptosis repressors.</p><p class="p p-last">In summary, by programming a unique continuous ferroptosis (JB2)&#x02010;to&#x02010;apoptosis (JB6) landscape, we identified critical molecular switches between these death pathways, and defined a gradient gene set, the GGS, as an excellent predictor for ferroptosis and apoptosis classification. The GGS was also used to optimize a list of reliable, validated ferroptosis biomarkers and to predict chemoresistant tumors. Furthermore, the GGS is a unique source of novel ferroptosis regulators, such as PDAP1, which might be used as targets for cancer therapy.</p></div><div id="advs7707-sec-0140" class="tsec sec"><h2 class="head no_bottom_margin" id="advs7707-sec-0140title">4. Experimental Section</h2><div id="advs7707-sec-0170" class="sec sec-first"><h3 id="advs7707-sec-0170title">Drugs and Chemicals</h3><p class="p p-first-last">The following drugs and chemicals were used in this study: JQ1 (11187), Bortezomib (BTZ, 10008822), Liproxstatin&#x02010;1 (17730), rotenone (13995) and carbonyl cyanide m&#x02010;chlorophenyl&#x000a0;hydrazine (CCCP, 25458) were purchased from Cayman Chemicals. SP600125 (1496) was from Tocris Bioscience. Imidazole ketone erastin (IKE) was from Taizhou Crene Biotechnology, China. The following were purchased from Sigma: Ferrostatin&#x02010;1 (SML0583), zVAD&#x02010;FMK (V116), Necrostatin&#x02010;1 (N9037),&#x000a0;reduced&#x000a0;l&#x02010;GSH (G6013), 2,2&#x02032;&#x02010;bipyridyl (D216305), staurosporine (S5921), Doxicycline (D9891, D3072) bafilomycin&#x02010;A1 (B1793), &#x003b1;&#x02010;ketoglutarate (K1128), erastin (E7781), RSL3 (SML2234), FIN56 (SML1740), aminooxyacetic acid (AOA, <a id="__tag_1148364246" class="tag_hotlink" href="/nuccore/C13408" ref="/nuccore/C13408"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"C13408","term_id":"1560961","term_text":"C13408"}}</span>C13408</a>), pioglitazone (CDS021593), tunicamycin (T7765), paclitaxel (T7402) and chloroquine (C6628).</p></div><div id="advs7707-sec-0180" class="sec"><h3 id="advs7707-sec-0180title">Antibodies</h3><p class="p p-first-last">The antibodies list is provided in Supplementary Table&#x000a0;<a href="#advs7707-supitem-0002" rid="advs7707-supitem-0002" class=" supplementary-material">S5</a> (Supporting Information).</p></div><div id="advs7707-sec-0190" class="sec"><h3 id="advs7707-sec-0190title">Cell Culture</h3><p class="p p-first-last">The basal&#x02010;like breast cancer cell lines MDA&#x02010;MB&#x02010;468, HCC70, HCC38, the luminal breast cancer line MCF7, and the human embryonic kidney (HEK) 293T cells were originally obtained from the American Type Culture Collection (USA). MDA&#x02010;MB&#x02010;468, HCC70, HCC38, and MCF7 were grown in RPMI, while HEK293T cells were grown in Dulbecco's modified Eagle medium (DMEM) (Gibco BRL, USA). Unless otherwise indicated, all cell lines were cultured in a medium containing 10% fetal bovine serum (Gibco BRL, USA), L&#x02010;Glutamine (2&#x000a0;mM), and penicillin/streptomycin.&#x000a0;Cells were cultured at 37&#x000a0;&#x000b0;C in a humidified incubator of 5% CO<sub>2</sub>. Cell lines were routinely (once a month) checked for mycoplasma using an RT&#x02010;PCR.</p></div><div id="advs7707-sec-0200" class="sec"><h3 id="advs7707-sec-0200title">Drugs Treatments</h3><p class="p p-first-last">JQ1 and Bortezomib (BTZ) were dissolved in DMSO (20&#x000a0;mM stock for JQ1, and 10&#x000a0;mM for BTZ), aliquoted, and stored at &#x02212;20&#x000a0;&#x000b0;C protected from light. Drugs were routinely applied at a cell density of &#x02248;50% confluency, &#x02248;24&#x000a0;h after cell seeding. BTZ was diluted to 20&#x000a0;&#x000b5;M in DMSO and JQ1 was diluted to 200&#x000a0;&#x000b5;M in RPMI media. Drug combinations were freshly prepared at a final concentration (0.1&#x000a0;&#x000b5;M for JQ1 and 2&#x02013;6&#x000a0;nM for BTZ) in a growth medium before use, by vigorous vortex (3&#x000a0;min). DMSO concentration was maintained below 0.1%. Importantly, batch effects of JQ1 and BTZ might influence the final concentration for each JB combinations and should be carefully calibrated due to the delicate titration and death response. Ferrostatin&#x02010;1 (100&#x000a0;mM stock in DMSO), liproxstatin&#x02010;1 (25&#x000a0;mM in DMSO), and zVAD&#x02010;FMK (20&#x000a0;mM stock in DMSO) were applied 2&#x000a0;h before and during the drugs treatment at a final concentration of 5&#x02013;10&#x000a0;&#x000b5;M.</p></div><div id="advs7707-sec-0210" class="sec"><h3 id="advs7707-sec-0210title">Drug Response Assays</h3><p class="p p-first-last">Various assays were applied in response to drug treatments (cell viability, cell death, lipid peroxidation, qPCR, RNAseq, iron measurements, etc), and were performed at different time points as indicated for each assay and in the corresponding legends. The different time points related to the different kinetics of the various responses, which were calibrated in preliminary experiments. Usually, lipid peroxidation (BODIPY, 4HNE), as well as its effect on other cellular proteins, was measured 16&#x02013;24&#x000a0;h post&#x02010;treatment, while cell death and viability are measured at longer time points, usually 72&#x000a0;h post&#x02010;drug treatment.</p></div><div id="advs7707-sec-0220" class="sec"><h3 id="advs7707-sec-0220title">Cell Viability and Death Assays</h3><p class="p p-first-last">For MTT assay, 5000 cells were seeded in a 96&#x02010;well plate and 24&#x000a0;h later treated with the indicated drugs. Cell viability was measured 72&#x000a0;h later using the MTT (M2128, SIGMA) assay as described previously.<sup>[</sup>
<sup><a href="#advs7707-bib-0064" rid="advs7707-bib-0064" class=" bibr popnode">
64
</a></sup>
<sup>]</sup> Results were presented as % cell viability of control untreated (DMSO) cells. For the trypan blue exclusion assay, the cell culture media was collected following drug treatment, combined with the trypsinized cells that remained on the plate, and spun down at 1&#x000a0;200&#x000a0;rpm for 5&#x000a0;min. Pellets were resuspended in PBS, mixed 1:1 with trypan blue, and counted by an automated cell counter (Countess&#x02010;II, Invitrogen). A fraction of trypan blue negative cells was used to calculate cell viability compared to the control (mock&#x02010;treated cells). Cell death was measured by the CellTox&#x02010;green Cytotoxicity assay (G8741, Promega) according to the manufacturer instructions. In brief, 5000&#x000a0;cells/well in 96 wells black plate (Greiner, 655&#x000a0;090) were treated with the indicated drugs in the absence or presence of either liproxstatin&#x02010;1 or ZVAD for 72&#x000a0;h. CellTox green reagent was diluted 1:1000 in assay buffer and incubated with the cells (100&#x000a0;&#x000b5;l&#x000a0;well<sup>&#x02212;1</sup>). Plates were shaken for 1&#x000a0;min using an orbital shaker and then incubated in the dark at room temperature for 10&#x02013;15&#x000a0;min. The fluorescence signal was measured by the Infinite 200 PRO Tecan fluorometric microplate reader at 490/525&#x000a0;nm (excitation/emission).</p></div><div id="advs7707-sec-0230" class="sec"><h3 id="advs7707-sec-0230title">RNA Extraction and Real&#x02010;Time PCR</h3><p class="p p-first-last">Total RNA from cell lines was extracted and purified using TRI Reagent (Sigma&#x02010;Aldrich). RNA was reverse&#x02010;transcribed into complementary DNA (cDNA) using the High&#x02010;Capacity cDNA Reverse Transcription Kit (Applied Biosystems; Cat. No. 4368814) with random primers according to the manufacturer's instructions. Real&#x02010;time PCR analysis was performed in QuantStudio&#x02010;3 Real&#x02010;Time PCR system (Applied Biosystems, Thermo Fisher Scientific) using SYBR Green Master Mix reagents (Roche) according to the manufacturer's guidelines. House&#x02010;keeping gene &#x003b2;&#x02010;acting was used for normalization. The relative levels of mRNA were calculated using the &#x00394;&#x00394;<em>C</em>
<sub>T</sub>&#x000a0;method. The primers list is provided in Table&#x000a0;<a href="#advs7707-supitem-0002" rid="advs7707-supitem-0002" class=" supplementary-material">S6</a>, Supporting Information.</p></div><div id="advs7707-sec-0240" class="sec"><h3 id="advs7707-sec-0240title">RNA Sequencing</h3><p class="p p-first-last">RNAseq was performed for control (DMSO) basal&#x02010;like breast cancer cell lines, for cells treated with JB2, JB3, JB6, or single treatments (B2, B3, B6, JQ1), as well as with ferroptosis inducers (erastin, RSL3). DMSO concentration was similar in all samples. Two&#x02010;time points were selected for the RNAseq of JQ1/BTZ/JB2/JB3/JB6 treatments, 6 and 24&#x000a0;h, based on qPCR analysis of well&#x02010;known ferroptotic upregulated genes (ASNS, CHAC1, DDIT4)<sup>[</sup>
<sup><a href="#advs7707-bib-0021" rid="advs7707-bib-0021" class=" bibr popnode">
21
</a></sup>
<sup>]</sup> concomitantly with cell viability assay. At 24&#x000a0;h no effect on cell viability was observed, but the indicated genes were significantly upregulated, while the earlier time point (6&#x000a0;h) represents the initial transcriptomic response to the treatments. All RNAseq samples were done in independent duplicates. Total RNA was extracted as described above. RNA quality was assessed using the Agilent 4200 TapeStation System (Agilent Technologies, Santa Clara, CA). RNA&#x02010;seq libraries were generated by applying a bulk adaptation of the MARS&#x02010;seq protocol, as previously described.<sup>[</sup>
<sup><a href="#advs7707-bib-0065" rid="advs7707-bib-0065" class=" bibr popnode">
65
</a></sup>
<sup>]</sup>&#x000a0;Libraries were sequenced by the Illumina Novaseq 6000 using SP mode 100 cycles kit (Illumina). The mapping of sequences to the genome and generation of the count matrix was performed by the UTAP pipeline (Weizmann Institute). Libraries normalization, filtration of low count genes, and discovery of differentially expressed genes were performed using the edgeR and Limma packages in R. PCA plots and a Venn diagram was generated using the factoextra and eulerr packages in R, respectively. KEGG enrichment analysis was performed using the enrichR package; p&#x02010;values for KEGG enrichment analysis were determined by Fisher exact test and adjusted for multiple hypotheses testing by the Benjamini&#x02010;Hochberg method. Gene set enrichment analysis (single sample) was performed using the GSVA package. Over&#x02010;representation of transcription factor targets was performed using ChEA3.<sup>[</sup>
<sup><a href="#advs7707-bib-0066" rid="advs7707-bib-0066" class=" bibr popnode">
66
</a></sup>
<sup>]</sup>
</p></div><div id="advs7707-sec-0250" class="sec"><h3 id="advs7707-sec-0250title">Gene Signatures and Biomarkers</h3><p class="p p-first-last">Computational analysis of the RNAseq to produce the gradient gene set, the specific ferroptosis to apoptosis biomarkers, and their validations using UMAP and ssGSEA were all done in R. The computational methods are detailed in the supplementary text.</p></div><div id="advs7707-sec-0260" class="sec"><h3 id="advs7707-sec-0260title">Cell lysates and Western Blotting</h3><p class="p p-first-last">Cell were lysed in lysis buffer containing 0.5% Triton X&#x02010;100, 50&#x000a0;mM Hepes (pH 7.5), 100&#x000a0;mM NaCl, 1&#x000a0;mM MgCl<sub>2</sub>, 50&#x000a0;mM NaF, 0.5&#x000a0;mM NaVO<sub>3</sub>, 20&#x000a0;mM &#x003b2;&#x02010;glycerophosphate, 1&#x000a0;mM phenylmethylsulfonyl fluoride, 10&#x000a0;&#x000b5;g ml<sup>&#x02212;1</sup> leupeptin, and 10&#x000a0;&#x000b5;g&#x000a0;ml<sup>&#x02212;1</sup> aprotinin. Freshly isolated tumors were manually chopped with a sterile blade, and 50&#x000a0;mg tissue was homogenized in 400&#x000a0;&#x000b5;l lysis buffer using 4&#x000a0;mm stainless steel beads in a homogenizer device (TissueLyser&#x02010;LT, QIAGEN) for 4&#x000a0;min (or until a homogenous solution was obtained). Cell lysates were centrifuged at 14,000&#x000a0;rpm for 15&#x000a0;min at 4&#x000a0;&#x000b0;C, protein concentration of the supernatants was measured by Bradford assay (Bio&#x02010;Rad, Hercules, CA). Equal amounts of total protein (30&#x02013;50&#x000a0;&#x000b5;g per sample) were analyzed by SDS&#x02013;PAGE (polyacrylamide gel electrophoresis) and Western Blotting (WB) was performed as previously described.<sup>[</sup>
<sup><a href="#advs7707-bib-0067" rid="advs7707-bib-0067" class=" bibr popnode">
67
</a></sup>
<sup>]</sup>
</p></div><div id="advs7707-sec-0270" class="sec"><h3 id="advs7707-sec-0270title">Cell Fractionation</h3><p class="p p-first-last">Nuclear and cytoplasmic extracts were prepared as follows: 1&#x000a0;&#x000d7;&#x000a0;10<sup>6</sup> cells were seeded in 10&#x000a0;cm plates for &#x02248;24&#x000a0;h, then treated with drugs for the indicated time periods, washed twice with cold PBS, and lysed with 250&#x000a0;&#x000b5;l of cytoplasmic extract (CE) buffer (0.075% (v/v) NP40, 10&#x000a0;mM HEPES, 60&#x000a0;mM KCl, 1&#x000a0;mM EDTA, 1&#x000a0;mM DTT, 1&#x000a0;mM PMSF, 0.5&#x000a0;mM NaVO<sub>3</sub>, 10&#x000a0;&#x000b5;g&#x000a0;ml<sup>&#x02212;1</sup> leupeptin, and 10&#x000a0;&#x000b5;g&#x000a0;mL<sup>&#x02212;1</sup> aprotinin, adjusted to pH 7.6). Samples were centrifuged at 1,000&#x000a0;rpm for 5&#x000a0;min at 4&#x000a0;&#x000b0;C and cytoplasmic fractions were collected from the supernatant. Pellets were washed gently with 250&#x000a0;&#x000b5;l of CE buffer without NP40 and spun again. Nuclear fractions were extracted by nuclear extract (NE) buffer (20&#x000a0;mM Tris HCl, 420&#x000a0;mM NaCl, 1.5&#x000a0;mM MgCl<sub>2</sub>, 0.2&#x000a0;mM EDTA, 1&#x000a0;mM PMSF, 25% (v/v) glycerol, 0.5&#x000a0;mM NaVO<sub>3</sub>, 10&#x000a0;&#x000b5;g&#x000a0;mL<sup>&#x02212;1</sup> leupeptin, 10&#x000a0;&#x000b5;g&#x000a0;mL<sup>&#x02212;1</sup> aprotinin, adjusted to pH 8.0). Pellets were resuspended in 100&#x000a0;&#x000b5;L NE buffer containing 400&#x000a0;mM NaCl, vortexed, and incubated on ice for 10&#x000a0;min. The cytoplasmic and nuclear fractions were centrifuged at 14,000&#x000a0;rpm for 15&#x000a0;min at 4&#x000a0;&#x000b0;C. Supernatant from the CE and NE was collected. Protein content was measured by Bradford and samples were analyzed by SDS&#x02010;PAGE and WB.</p></div><div id="advs7707-sec-0280" class="sec"><h3 id="advs7707-sec-0280title">Fluorescence Staining</h3><p class="p p-first">Cells grown on coverslips in a 24&#x02010;wells plate were treated with drugs as indicated, washed twice with PBS, fixed in 4% paraformaldehyde (PFA), and processed for immunofluorescence (IF) staining essentially as described previously<sup>[</sup>
<sup><a href="#advs7707-bib-0068" rid="advs7707-bib-0068" class=" bibr popnode">
68
</a></sup>
<sup>]</sup> using the following IF blocking buffer (10&#x000a0;mM Tris&#x02010;Hcl pH 7.5, 150&#x000a0;mM NaCl, 2% bovine serum albumin (BSA), 10% goat serum, 1% Glycine, 0.05%&#x02013;0.1% Triton&#x02010;X&#x02010;100).</p><p class="p p-last">Live cell imaging was performed with MitoTracker (Red&#x02010;CMXRos, Invitrogen) or LysoTracker (Red DND&#x02010;99 or Green DND&#x02010;26, Invitrogen). Cells grown in a 96&#x02010;well glass bottom plate (8000 cells/well) were treated with drugs as indicated, washed with PBS, and incubated in either serum&#x02010;free RPMI media containing 70&#x000a0;nM of MitoTracker or in full RPMI (10% FBS) media with 70&#x000a0;nM of LysoTracker. Hoechst (1&#x000a0;mM) was also added to the media and the cells were incubated for 30 or 45&#x000a0;min, respectively, in a tissue culture incubator in the dark. The cells were then washed gently in PBS and incubated with 100&#x000a0;&#x000b5;L of live cell imaging solution (Invitrogen) for analysis by confocal microscopy (LSM800, Zeiss). Co&#x02010;staining with LysoTracker of fixed cells was performed by staining with LysoTracker (80&#x000a0;nM) as described above, followed by fixation in 4% PFA and blocking in IF blocking buffer containing 0.05% Triton&#x02010;X&#x02010;100 as described above.</p></div><div id="advs7707-sec-0290" class="sec"><h3 id="advs7707-sec-0290title">Acridine Orange</h3><p class="p p-first-last">Cells were incubated in serum&#x02010;free RPMI media with 1&#x000a0;&#x000b5;g&#x000a0;mL<sup>&#x02212;1</sup> Acridine Orange (AO) (ImmunoChemistry Technologies) and 1&#x000a0;mM Hoechst 33&#x000a0;342 (Sigma&#x02010;Aldrich) for 30&#x000a0;min. Cells were then washed with PBS incubated with live cell imaging solution and processed by confocal microscope.</p></div><div id="advs7707-sec-0300" class="sec"><h3 id="advs7707-sec-0300title">FITC&#x02010;Dextran</h3><p class="p p-first-last">The polysaccharides dextran conjugated to FITC (FD10S, 10&#x000a0;KDa, SIGMA), which accumulates in the lysosome,<sup>[</sup>
<sup><a href="#advs7707-bib-0069" rid="advs7707-bib-0069" class=" bibr popnode">
69
</a></sup>
<sup>]</sup> was incubated (1&#x000a0;mg&#x000a0;mL<sup>&#x02212;1</sup>) with the cells grown on coverslips in complete RPMI media for 2&#x000a0;h. The cells were washed in PBS and incubated in complete RPMI media for an additional 2&#x02013;3&#x000a0;h, and then treated with drugs as indicated for 24&#x000a0;h. Cells were washed with PBS, fixed with 2% formaldehyde for 25&#x000a0;min at RT, and stained with 1&#x000a0;mM Hoechst in PBS for 5&#x000a0;min. Images were processed by confocal microscope (LSM800, Zeiss).</p></div><div id="advs7707-sec-0310" class="sec"><h3 id="advs7707-sec-0310title">Animal Studies</h3><p class="p p-first">All animal studies were performed according to protocols approved by the Weizmann Institutional Animal Care and Use Committee (IACUC approval numbers 00350122&#x02010;2 for the PDAP1 in vivo experiment, and 08311121&#x02013;2 for the in vivo validation of the biomarkers).</p><p>For PDAP1 mouse xenograft models, GFP&#x02010;luciferase expressing MDA&#x02010;MB&#x02010;468 cells were infected with Tet&#x02010;pLKO&#x02010;shPDAP1 lentivirus, selected with puromycin (1&#x000a0;&#x000b5;g ml<sup>&#x02212;1</sup>) and then implanted bilaterally into the mammary fat pads (2&#x000a0;&#x000d7;&#x000a0;10<sup>6</sup>&#x000a0;per gland) of the fourth inguinal gland of 5&#x02013;6 weeks old female athymic nude&#x02010;<em>Foxn1nu</em>&#x000a0;mice. Tet&#x02010;pLKO harboring scrambled sequence was used as control. When tumor size reached an average of &#x02248;50&#x000a0;mm<sup>3</sup>, mice were randomized into two groups (<em>n</em>&#x000a0;=&#x000a0;10 mice per group for shPDAP1, <em>n</em>&#x000a0;=&#x000a0;5 for scrambled shRNA); control (regular water) and DOX experimental group (2&#x000a0;mg&#x000a0;mL<sup>&#x02212;1</sup> of doxycycline (D9891, SIGMA) in drinking water, DOX was replaced every 4&#x02013;5 days). Mouse body weights and tumor size were measured every 3&#x02013;4 days. Tumor volume was measured by a digital Vernier caliper and calculated according to the (width<sup>2</sup>&#x000a0;&#x000d7;&#x000a0;length)/2 formula. Tumor imaging was monitored using Xenogen IVIS Spectrum in vivo bioluminescence. Mice were anesthetized by isoflurane inhalation, and then&#x000a0;D&#x02010;luciferin (100&#x000a0;&#x000b5;L; 5&#x000a0;mg&#x000a0;mL<sup>&#x02212;1</sup>; Regis Technologies) was intraperitoneally injected. Bioluminescence images were acquired within 10&#x000a0;min after injection by the charge&#x02010;coupled device camera of the IVIS instrument with the Living Image 3.0 software (Xenogen Caliper Life Sciences). At the end of the study, mice were sacrificed, and tumors were excised, weighed, and documented using a digital camera.</p><p class="p p-last">For <em>invivo</em> validation of the selective ferroptosis biomarkers, MDA&#x02010;MB&#x02010;468 cells were orthotopically implanted into mammary fat pads as described above. When tumor size reached an average of &#x02248;50&#x000a0;mm<sup>3</sup>, mice were randomized into three groups (<em>n</em>&#x000a0;=&#x000a0;3 mice per group): (1) IKE&#x02010; imidazole ketone erastin (Taizhou Crene Biotechnology, China), 3 days 40&#x000a0;mg&#x000a0;kg<sup>&#x02212;1</sup>, and 80&#x000a0;mg&#x000a0;kg<sup>&#x02212;1</sup> on the 4th day (2) PTX&#x02010;paclitaxel (#1567&#x02010;125, Biovision) 20&#x000a0;mg&#x000a0;kg<sup>&#x02212;1</sup>, and (3) Ctrl &#x02010;control. IKE was dissolved in HBSS buffer containing 5% DMSO at pH&#x02010;4, and paclitaxel was dissolved in HBSS. Drugs were administered intraperitoneal in a dose of 100&#x000a0;&#x000b5;L per mouse (ctrl group received only HBSS/5% DMSO, pH&#x02010;4) and 96&#x000a0;h later mice were anesthetized by intraperitoneal injection with a lethal dose of pental (300&#x000a0;&#x000b5;L, 4&#x000a0;ml&#x000a0;kg<sup>&#x02212;1</sup>). Tumors were excised and immediately processed for RNA purification or WB analysis.</p></div><div id="advs7707-sec-0320" class="sec"><h3 id="advs7707-sec-0320title">GSH, Lipid Ros, and Iron measurements</h3><p class="p p-first-last">Methods have been previously described,<sup>[</sup>
<sup><a href="#advs7707-bib-0008" rid="advs7707-bib-0008" class=" bibr popnode">
8a
</a></sup>
<sup>]</sup> and are included in the Supplementary Methods</p></div><div id="advs7707-sec-0330" class="sec"><h3 id="advs7707-sec-0330title">Supplementary Methods</h3><p class="p p-first-last">Gene knockdown, Mitochondrial, ER, and lysosomal manipulations, GSH measurements, Lipid ROS measurements, FACS analysis, iron measurements, proteasomal activity assay, iron staining, &#x003b2;&#x02010;hexaminidase activity assay, metabolomics, and glutamine tracing are detailed in the supplementary methods section.</p></div><div id="advs7707-sec-0340" class="sec sec-last"><h3 id="advs7707-sec-0340title">Statistical Analysis</h3><p class="p p-first-last">The statistical test used to determine the significance levels for each experiment was done in R and is described in the figure legends. In most figures (unless otherwise mentioned in the figure legends), the means &#x000b1; sd of the repeats are shown, and a two&#x02010;sided t&#x02010;test was used to measure statistical significance. The sample size for each experiment is described in the figure legends. <em>p</em>&#x02010;values lower than 0.05 were considered statistically significant.</p></div></div><div id="advs7707-sec-0360" class="tsec sec"><h2 class="head no_bottom_margin" id="advs7707-sec-0360title">Conflict of Interest</h2><p class="p p-first-last">The authors declare no conflict of interest.</p></div><div id="advs7707-sec-0370" class="tsec sec"><h2 class="head no_bottom_margin" id="advs7707-sec-0370title">Author Contributions</h2><p class="p p-first-last">Y.V. and A.M. contributed equally to this work. Y.V. designed and performed the computational analysis (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0001" rid-ob="ob-advs7707-fig-0001" co-legend-rid="lgnd_advs7707-fig-0001"><span>1K&#x02013;M</span></a>, Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0002" rid-ob="ob-advs7707-fig-0002" co-legend-rid="lgnd_advs7707-fig-0002"><span>2</span></a>, Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0003/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0003" rid-ob="ob-advs7707-fig-0003" co-legend-rid="lgnd_advs7707-fig-0003"><span>3U</span></a>, Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0004" rid-ob="ob-advs7707-fig-0004" co-legend-rid="lgnd_advs7707-fig-0004"><span>4</span></a>, Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0005" rid-ob="ob-advs7707-fig-0005" co-legend-rid="lgnd_advs7707-fig-0005"><span>5G,H</span></a>, Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0006/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0006" rid-ob="ob-advs7707-fig-0006" co-legend-rid="lgnd_advs7707-fig-0006"><span>6I,J</span></a>, Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S1K,L,S2,S4,S5A,S6</a>, Supporting Information), assisted in writing the manuscript and checked the raw data. A. Maimon designed and performed all the experiments except those mentioned below. V.D. performed the experiments shown in Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0001" rid-ob="ob-advs7707-fig-0001" co-legend-rid="lgnd_advs7707-fig-0001"><span>1A,B</span></a>, Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S1A,B</a>, Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0006/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0006" rid-ob="ob-advs7707-fig-0006" co-legend-rid="lgnd_advs7707-fig-0006"><span>6A,D,F,K,I</span></a>, Figure&#x000a0;<a href="#advs7707-supitem-0001" rid="advs7707-supitem-0001" class=" supplementary-material">S7A,B</a>, (Supporting Information) and prepared the RNA for RNAseq. H.R assisted with the immunofluorescence experiments shown in Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0001" rid-ob="ob-advs7707-fig-0001" co-legend-rid="lgnd_advs7707-fig-0001"><span>1E</span></a>, Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0006/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0006" rid-ob="ob-advs7707-fig-0006" co-legend-rid="lgnd_advs7707-fig-0006"><span>6I</span></a>.. I.A. performed the glutamine tracing experiment (Figure&#x000a0;<a href="/pmc/articles/PMC11077643/figure/advs7707-fig-0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="advs7707-fig-0008" rid-ob="ob-advs7707-fig-0008" co-legend-rid="lgnd_advs7707-fig-0008"><span>8L</span></a>). S.M. and M.I. performed the polar metabolites and lipidomics screen. A. Ma'ayan contributed to the discussion and reviewed the paper. F.W. contributed to the discussion and reviewed the paper., E.G., supervised the glutamine tracing experiment. E.R. provided valuable suggestions for the computational studies, and reviewed and edited the manuscript. S.L. designed and supervised the study, contributed to the analysis and interpretation of the data, ensured financial support, was involved in figure preparation, and wrote the manuscript.</p></div><div id="sec-a.o.g" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="sec-a.o.gtitle">Supporting information</h2><!--/article/body/sec/--><div class="sec suppmat" id="advs7707-supitem-0001"><p>Supporting Information</p><div class="sup-box half_rhythm" id="media-a.o.g.b.b"><a href="/pmc/articles/PMC11077643/bin/ADVS-11-2307263-s001.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(2.4M, pdf)</sup></div></div><div class="sec suppmat" id="advs7707-supitem-0002"><p>Supplemental Table 1&#x02010;6</p><div class="sup-box half_rhythm" id="media-a.o.g.c.b"><a href="/pmc/articles/PMC11077643/bin/ADVS-11-2307263-s002.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(194K, xlsx)</sup></div></div></div><div id="advs7707-sec-0350" class="tsec sec"><h2 class="head no_bottom_margin" id="advs7707-sec-0350title">Acknowledgements</h2><div class="sec"><p>S.L. is the incumbent of the Joyce and Ben B. Eisenberg Chair of Molecular Endocrinology and Cancer Research. The authors thank Michael T. Lewis and Lacey E. Dobrolecki, directors of the Patient&#x02010;derived Xenograft and Advanced In Vivo Models (Baylor collage, TX), for info regarding PDX models. The authors thank Shubhankar Das for his assistance with the in vivo experiment, forPier Giorgio Puzzovio for his assistance during the revision, and for Yossi Yarden and his team members for productive discussions and suggestions. This study was supported by the Cooperation Program in Cancer Research of the Deutsches Krebsforschungszentrum (DKFZ) and Israel's Ministry of Innovation, Science and Technology (MOST), the Israel Science Foundation (Grant No. 1564/23), Merck KGaA, Darmstadt, Germany, by the Weizmann&#x02010;Pasteur collaborative grant, and by a research grant from the Erica Drake Fund.</p></div></div><div id="notes-a.n.c" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.n.ctitle">Notes</h2><p>
<span class="mixed-citation" id="advs7707-cit-0080">
Vinik Y., Maimon A., Dubey V., Raj H., Abramovitch I., Malitsky S., Itkin M., Ma'ayan A., Westermann F., Gottlieb E., Ruppin E., Lev S., <span class="ref-title">Programming a Ferroptosis&#x02010;to&#x02010;Apoptosis Transition Landscape Revealed Ferroptosis Biomarkers and Repressors for Cancer Therapy</span>. <span class="ref-journal">Adv. Sci.</span>
2024, <span class="ref-vol">11</span>, 2307263. 10.1002/advs.202307263
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC11077643/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/38441406" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fadvs.202307263" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Adv.+Sci.&amp;title=Programming+a+Ferroptosis&#x02010;to&#x02010;Apoptosis+Transition+Landscape+Revealed+Ferroptosis+Biomarkers+and+Repressors+for+Cancer+Therapy&amp;volume=11&amp;publication_year=2024&amp;pages=2307263&amp;doi=10.1002/advs.202307263&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
</p></div><div id="advs7707-sec-0390" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="advs7707-sec-0390title">Data Availability Statement</h2><!--/article/back/sec/--><p class="p p-first-last">RNA&#x02010;seq data is publicly available in the Gene Expression Omnibus (GEO) repository (<a href="https://www.ncbi.nlm.nih.gov/geo/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.ncbi.nlm.nih.gov/geo/</a>), accession number <a id="__tag_1148364248" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE235201" ref="/geo/query/acc.cgi?acc=GSE235201"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE235201","term_id":"235201"}}</span>GSE235201</a>. The code used to generate the computational analysis is available through GitHub (<a href="https://github.com/SimaLevLab/A-continuous-ferroptosis-to-apoptosis-landscape" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/SimaLevLab/A&#x02010;continuous&#x02010;ferroptosis&#x02010;to&#x02010;apoptosis&#x02010;landscape</a>). All other data generated in this study is available upon request from the corresponding author.</p></div><div id="advs7707-bibl-0001" class="tsec sec"><h2 class="head no_bottom_margin" id="advs7707-bibl-0001title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0001">1. <span class="mixed-citation" id="advs7707-cit-0001">
Tang D., Kang R., Berghe T. V., Vandenabeele P., Kroemer G., <span class="ref-journal">Cell Res.</span>
2019, <span class="ref-vol">29</span>, 347.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6796845/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30948788" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Res.&amp;volume=29&amp;publication_year=2019&amp;pages=347&amp;pmid=30948788&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0002">2. <span class="mixed-citation" id="advs7707-cit-0002">
Ouyang L., Shi Z., Zhao S., Wang F. T., Zhou T. T., Liu B., Bao J. K., <span class="ref-journal">Cell Prolif</span>
2012, <span class="ref-vol">45</span>, 487.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6496669/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23030059" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Prolif&amp;volume=45&amp;publication_year=2012&amp;pages=487&amp;pmid=23030059&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0003">3. <span class="mixed-citation" id="advs7707-cit-0003">
Yan G., Elbadwi M., Efferth T., <span class="ref-journal">World Acad. Sci. J.</span>
2020, <span class="ref-vol">2</span>, 39. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=World+Acad.+Sci.+J.&amp;volume=2&amp;publication_year=2020&amp;pages=39&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0004">4. <span class="mixed-citation" id="advs7707-cit-0004">
Hong S. H., Lee D. H., Lee Y. S., Jo M. J., Jeong Y. A., Kwon W. T., Choudry H. A., Bartlett D. L., Lee Y. J., <span class="ref-journal">Oncotarget</span>
2017, <span class="ref-vol">8</span>, 115164.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5777762/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29383150" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;volume=8&amp;publication_year=2017&amp;pages=115164&amp;pmid=29383150&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0005">5. <span class="mixed-citation" id="advs7707-cit-0005">
Carneiro B. A., El&#x02010;Deiry W. S., <span class="ref-journal">Nat. Rev. Clin. Oncol.</span>
2020, <span class="ref-vol">17</span>, 395.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8211386/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32203277" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Clin.+Oncol.&amp;volume=17&amp;publication_year=2020&amp;pages=395&amp;pmid=32203277&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0006">6. <span class="mixed-citation" id="advs7707-cit-0006">
Strasser A., Vaux D. L., <span class="ref-journal">Mol. Cell</span>
2020, <span class="ref-vol">78</span>, 1045.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/32516599" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;volume=78&amp;publication_year=2020&amp;pages=1045&amp;pmid=32516599&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0007">7. <span class="mixed-citation" id="advs7707-cit-0007">
Lu B., Chen X. B., Ying M. D., He Q. J., Cao J., Yang B., <span class="ref-journal">Front. Pharmacol.</span>
2017, <span class="ref-vol">8</span>, 992.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5770584/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29375387" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Pharmacol.&amp;volume=8&amp;publication_year=2017&amp;pages=992&amp;pmid=29375387&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0008">8. <span class="mixed-citation" id="advs7707-cit-0008">a) Verma N., Vinik Y., Saroha A., Nair N. U., Ruppin E., Mills G., Karn T., Dubey V., Khera L., Raj H., Maina F., Lev S., <span class="ref-journal">Sci. Adv.</span>
2020, <span class="ref-vol">6</span>; <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7442484/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32937365" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Adv.&amp;volume=6&amp;publication_year=2020&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span><span class="mixed-citation" id="advs7707-cit-0009">b) Doll S., Proneth B., Tyurina Y. Y., Panzilius E., Kobayashi S., Ingold I., Irmler M., Beckers J., Aichler M., Walch A., Prokisch H., Trumbach D., Mao G., Qu F., Bayir H., Fullekrug J., Scheel C. H., Wurst W., Schick J. A., Kagan V. E., Angeli J. P., Conrad M., <span class="ref-journal">Nat. Chem. Biol.</span>
2017, <span class="ref-vol">13</span>, 91.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5610546/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27842070" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Chem.+Biol.&amp;volume=13&amp;publication_year=2017&amp;pages=91&amp;pmid=27842070&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0009">9. <span class="mixed-citation" id="advs7707-cit-0010">
Dixon S. J., Lemberg K. M., Lamprecht M. R., Skouta R., Zaitsev E. M., Gleason C. E., Patel D. N., Bauer A. J., Cantley A. M., Yang W. S., Morrison B., 
3rd, B. R. S
, <span class="ref-journal">Cell</span>
2012, <span class="ref-vol">149</span>, 1060.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3367386/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22632970" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;volume=149&amp;publication_year=2012&amp;pages=1060&amp;pmid=22632970&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0010">10. <span class="mixed-citation" id="advs7707-cit-0011">a) Conrad M., Pratt D. A., <span class="ref-journal">Nat. Chem. Biol.</span>
2019, <span class="ref-vol">15</span>, 1137;
 [<a href="https://pubmed.ncbi.nlm.nih.gov/31740834" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Chem.+Biol.&amp;volume=15&amp;publication_year=2019&amp;pages=1137&amp;pmid=31740834&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span><span class="mixed-citation" id="advs7707-cit-0012">b) Stockwell B. R., Friedmann Angeli J. P., Bayir H., Bush A. I., Conrad M., Dixon S. J., Fulda S., Gascon S., Hatzios S. K., Kagan V. E., Noel K., Jiang X., Linkermann A., Murphy M. E., Overholtzer M., Oyagi A., Pagnussat G. C., Park J., Ran Q., Rosenfeld C. S., Salnikow K., Tang D., Torti F. M., Torti S. V., Toyokuni S., Woerpel K. A., Zhang D. D., <span class="ref-journal">Cell</span>
2017, <span class="ref-vol">171</span>, 273.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5685180/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28985560" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;volume=171&amp;publication_year=2017&amp;pages=273&amp;pmid=28985560&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0011">11. <span class="mixed-citation" id="advs7707-cit-0013">
Wu H., Wang F., Ta N., Zhang T., Gao W., <span class="ref-journal">Life (Basel)</span>
2021, <span class="ref-vol">11</span>, 222.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8001967/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33801920" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Life+(Basel)&amp;volume=11&amp;publication_year=2021&amp;pages=222&amp;pmid=33801920&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0012">12. <span class="mixed-citation" id="advs7707-cit-0014">
Forcina G. C., Dixon S. J., <span class="ref-journal">Proteomics</span>
2019, <span class="ref-vol">19</span>, e1800311.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/30888116" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;volume=19&amp;publication_year=2019&amp;pages=e1800311&amp;pmid=30888116&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0013">13. <span class="mixed-citation" id="advs7707-cit-0015">
Hassannia B., Vandenabeele P., Vanden Berghe T., <span class="ref-journal">Cancer Cell</span>
2019, <span class="ref-vol">35</span>, 830.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/31105042" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Cell&amp;volume=35&amp;publication_year=2019&amp;pages=830&amp;pmid=31105042&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0014">14. <span class="mixed-citation" id="advs7707-cit-0016">
Yang W. S., Stockwell B. R., <span class="ref-journal">Trends Cell Biol.</span>
2016, <span class="ref-vol">26</span>, 165.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4764384/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26653790" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Cell+Biol.&amp;volume=26&amp;publication_year=2016&amp;pages=165&amp;pmid=26653790&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0015">15. <span class="mixed-citation" id="advs7707-cit-0017">
Zhang Y., Tan H., Daniels J. D., Zandkarimi F., Liu H., Brown L. M., Uchida K., O'Connor O. A., Stockwell B. R., <span class="ref-journal">Cell Chem. Biol.</span>
2019, <span class="ref-vol">26</span>, 623.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6525071/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30799221" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Chem.+Biol.&amp;volume=26&amp;publication_year=2019&amp;pages=623&amp;pmid=30799221&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0016">16. <span class="mixed-citation" id="advs7707-cit-0018">
Harding H. P., Zhang Y., Zeng H., Novoa I., Lu P. D., Calfon M., Sadri N., Yun C., Popko B., Paules R., Stojdl D. F., Bell J. C., Hettmann T., Leiden J. M., Ron D., <span class="ref-journal">Mol. Cell</span>
2003, <span class="ref-vol">11</span>, 619.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/12667446" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;volume=11&amp;publication_year=2003&amp;pages=619&amp;pmid=12667446&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0017">17. <span class="mixed-citation" id="advs7707-cit-0019">
Murray M. B., Leak L. B., Lee W. C., Dixon S. J., <span class="ref-journal">STAR Protoc</span>
2023, <span class="ref-vol">4</span>, 102457.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10432795/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37556320" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=STAR+Protoc&amp;volume=4&amp;publication_year=2023&amp;pages=102457&amp;pmid=37556320&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0018">18. <span class="mixed-citation" id="advs7707-cit-0020">
Dixon S. J., Stockwell B. R., <span class="ref-journal">Annu. Rev. Cancer Biol.</span>
2019, <span class="ref-vol">3</span>, 35. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Cancer+Biol.&amp;volume=3&amp;publication_year=2019&amp;pages=35&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0019">19. <span class="mixed-citation" id="advs7707-cit-0021">a) Alothaim T., Charbonneau M., Tang X., <span class="ref-journal">Sci. Rep.</span>
2021, <span class="ref-vol">11</span>, 10956;
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8155140/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34040090" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Rep.&amp;volume=11&amp;publication_year=2021&amp;pages=10956&amp;pmid=34040090&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span><span class="mixed-citation" id="advs7707-cit-0022">b) Beatty A., Singh T., Tyurina Y. Y., Tyurin V. A., Samovich S., Nicolas E., Maslar K., Zhou Y., Cai K. Q., Tan Y., Doll S., Conrad M., Subramanian A., Bayir H., Kagan V. E., Rennefahrt U., Peterson J. R., <span class="ref-journal">Nat. Commun.</span>
2021, <span class="ref-vol">12</span>, 2244.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8046803/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33854057" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;volume=12&amp;publication_year=2021&amp;pages=2244&amp;pmid=33854057&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0020">20. <span class="mixed-citation" id="advs7707-cit-0023">
Cui J., Qu Z., Harata&#x02010;Lee Y., Shen H., Aung T. N., Wang W., Kortschak R. D., Adelson D. L., <span class="ref-journal">PLoS One</span>
2020, <span class="ref-vol">15</span>, e0236395.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7392229/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32730293" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;volume=15&amp;publication_year=2020&amp;pages=e0236395&amp;pmid=32730293&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0021">21. <span class="mixed-citation" id="advs7707-cit-0024">
Dixon S. J., Patel D. N., Welsch M., Skouta R., Lee E. D., Hayano M., Thomas A. G., Gleason C. E., Tatonetti N. P., Slusher B. S., Stockwell B. R., <span class="ref-journal">Elife</span>
2014, <span class="ref-vol">3</span>, e02523.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4054777/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24844246" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Elife&amp;volume=3&amp;publication_year=2014&amp;pages=e02523&amp;pmid=24844246&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0022">22. <span class="mixed-citation" id="advs7707-cit-0025">
Badgley M. A., Kremer D. M., Maurer H. C., DelGiorno K. E., Lee H. J., Purohit V., Sagalovskiy I. R., Ma A., Kapilian J., Firl C. E. M., Decker A. R., Sastra S. A., Palermo C. F., Andrade L. R., Sajjakulnukit P., Zhang L., Tolstyka Z. P., Hirschhorn T., Lamb C., Liu T., Gu W., Seeley E. S., Stone E., Georgiou G., Manor U., Iuga A., Wahl G. M., Stockwell B. R., Lyssiotis C. A., Olive K. P., <span class="ref-journal">Science</span>
2020, <span class="ref-vol">368</span>, 85.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7681911/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32241947" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;volume=368&amp;publication_year=2020&amp;pages=85&amp;pmid=32241947&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0023">23. <span class="mixed-citation" id="advs7707-cit-0026">
Horak C. E., Pusztai L., Xing G., Trifan O. C., Saura C., Tseng L. M., Chan S., Welcher R., Liu D., <span class="ref-journal">Clin. Cancer Res.</span>
2013, <span class="ref-vol">19</span>, 1587.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/23340299" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Cancer+Res.&amp;volume=19&amp;publication_year=2013&amp;pages=1587&amp;pmid=23340299&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0024">24. <span class="mixed-citation" id="advs7707-cit-0027">
Petrosyan V., Dobrolecki L. E., Thistlethwaite L., Lewis A. N., Sallas C., Srinivasan R. R., Lei J. T., Kovacevic V., Obradovic P., Ellis M. J., Osborne C. K., Rimawi M. F., Pavlick A., Shafaee M. N., Dowst H., Jain A., Saltzman A. B., Malovannaya A., Marangoni E., Welm A. L., Welm B. E., Li S., Wulf G. M., Sonzogni O., Huang C., Vasaikar S., Hilsenbeck S. G., Zhang B., Milosavljevic A., Lewis M. T., <span class="ref-journal">iScience</span>
2023, <span class="ref-vol">26</span>, 105799.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9813793/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36619972" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=iScience&amp;volume=26&amp;publication_year=2023&amp;pages=105799&amp;pmid=36619972&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0025">25. <span class="mixed-citation" id="advs7707-cit-0028">
Yang Y., Lin J., Guo S., Xue X., Wang Y., Qiu S., Cui J., Ma L., Zhang X., Wang J., <span class="ref-journal">Cancer Cell Int.</span>
2020, <span class="ref-vol">20</span>, 587.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7720568/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33372599" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Cell+Int.&amp;volume=20&amp;publication_year=2020&amp;pages=587&amp;pmid=33372599&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0026">26. <span class="mixed-citation" id="advs7707-cit-0029">
Clemente L. P., Rabenau M., Tang S., Stanka J., Cors E., Stroh J., Culmsee C., von Karstedt S., <span class="ref-journal">Cells</span>
2020, <span class="ref-vol">9</span>, 2259.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7650611/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33050207" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cells&amp;volume=9&amp;publication_year=2020&amp;pages=2259&amp;pmid=33050207&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0027">27. <span class="mixed-citation" id="advs7707-cit-0030">
Bluemel G., Planque M., Madreiter&#x02010;Sokolowski C. T., Haitzmann T., Hrzenjak A., Graier W. F., Fendt S. M., Olschewski H., Leithner K., <span class="ref-journal">Free Radic. Biol. Med.</span>
2021, <span class="ref-vol">176</span>, 34.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/34520823" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Free+Radic.+Biol.+Med.&amp;volume=176&amp;publication_year=2021&amp;pages=34&amp;pmid=34520823&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0028">28. <span class="mixed-citation" id="advs7707-cit-0031">a) Cui H. Y., Wei W., Qian M. R., Tian R. F., Fu X., Li H. W., Nan G., Yang T., Lin P., Chen X., Zhu Y. M., Wang B., Sun X. X., Dou J. H., Jiang J. L., Li L., Wang S. J., Chen Z. N., <span class="ref-journal">Cancer Commun.</span>
2022, 750, 10.1002/cac2.12322; <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9395323/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35716012" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fcac2.12322" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Commun.&amp;publication_year=2022&amp;pages=750&amp;doi=10.1002/cac2.12322&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span><span class="mixed-citation" id="advs7707-cit-0032">b) Calzolari F., Appolloni I., Tutucci E., Caviglia S., Terrile M., Corte G., Malatesta P., <span class="ref-journal">Neoplasia</span>
2008, <span class="ref-vol">10</span>, 1373.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2586688/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19048116" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neoplasia&amp;volume=10&amp;publication_year=2008&amp;pages=1373&amp;pmid=19048116&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0029">29. <span class="mixed-citation" id="advs7707-cit-0033">
Lamb J., Crawford E. D., Peck D., Modell J. W., Blat I. C., Wrobel M. J., Lerner J., Brunet J. P., Subramanian A., Ross K. N., Reich M., Hieronymus H., Wei G., Armstrong S. A., Haggarty S. J., Clemons P. A., Wei R., Carr S. A., Lander E. S., Golub T. R., <span class="ref-journal">Science</span>
2006, <span class="ref-vol">313</span>, 1929.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/17008526" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;volume=313&amp;publication_year=2006&amp;pages=1929&amp;pmid=17008526&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0030">30. <span class="mixed-citation" id="advs7707-cit-0034">
Chang Y. Y., Juhasz G., Goraksha&#x02010;Hicks P., Arsham A. M., Mallin D. R., Muller L. K., Neufeld T. P., <span class="ref-journal">Biochem. Soc. Trans.</span>
2009, <span class="ref-vol">37</span>, 232.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/19143638" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.+Soc.+Trans.&amp;volume=37&amp;publication_year=2009&amp;pages=232&amp;pmid=19143638&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0031">31. <span class="mixed-citation" id="advs7707-cit-0035">
Sharma V. K., Singh A., Srivastava S. K., Kumar V., Gardi N. L., Nalwa A., Dinda A. K., Chattopadhyay P., Yadav S., <span class="ref-journal">Int. J. Biochem. Cell Biol.</span>
2016, <span class="ref-vol">78</span>, 194.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/27448842" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Biochem.+Cell+Biol.&amp;volume=78&amp;publication_year=2016&amp;pages=194&amp;pmid=27448842&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0032">32. <span class="mixed-citation" id="advs7707-cit-0036">
Yi J., Zhu J., Wu J., Thompson C. B., Jiang X., <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span>
2020, <span class="ref-vol">117</span>, 31189.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7733797/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33229547" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;volume=117&amp;publication_year=2020&amp;pages=31189&amp;pmid=33229547&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0033">33. <span class="mixed-citation" id="advs7707-cit-0037">
Zhou N., Bao J., <span class="ref-journal">Database</span>
2020, <span class="ref-vol">2020</span>, 10.1093/database/baaa021. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7100629/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32219413" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fdatabase%2Fbaaa021" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Database&amp;volume=2020&amp;publication_year=2020&amp;doi=10.1093/database/baaa021&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0034">34. <span class="mixed-citation" id="advs7707-cit-0038">
Llanos S., Megias D., Blanco&#x02010;Aparicio C., Hernandez&#x02010;Encinas E., Rovira M., Pietrocola F., Serrano M., <span class="ref-journal">Oncogene</span>
2019, <span class="ref-vol">38</span>, 3886.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6756094/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30692638" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;volume=38&amp;publication_year=2019&amp;pages=3886&amp;pmid=30692638&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0035">35. <span class="mixed-citation" id="advs7707-cit-0039">
Sakamaki J. I., Wilkinson S., Hahn M., Tasdemir N., O'Prey J., Clark W., Hedley A., Nixon C., Long J. S., New M., Van Acker T., Tooze S. A., Lowe S. W., Dikic I., Ryan K. M., <span class="ref-journal">Mol. Cell</span>
2017, <span class="ref-vol">66</span>, 517.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5446411/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28525743" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;volume=66&amp;publication_year=2017&amp;pages=517&amp;pmid=28525743&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0036">36. <span class="mixed-citation" id="advs7707-cit-0040">a) Slade L., Pulinilkunnil T., <span class="ref-journal">Mol. Cancer Res.</span>
2017, <span class="ref-vol">15</span>, 1637;
 [<a href="https://pubmed.ncbi.nlm.nih.gov/28851811" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cancer+Res.&amp;volume=15&amp;publication_year=2017&amp;pages=1637&amp;pmid=28851811&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span><span class="mixed-citation" id="advs7707-cit-0041">b) Sardiello M., Palmieri M., di Ronza A., Medina D. L., Valenza M., Gennarino V. A., Di Malta C., Donaudy F., Embrione V., Polishchuk R. S., Banfi S., Parenti G., Cattaneo E., Ballabio A., <span class="ref-journal">Science</span>
2009, <span class="ref-vol">325</span>, 473.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/19556463" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;volume=325&amp;publication_year=2009&amp;pages=473&amp;pmid=19556463&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0037">37. <span class="mixed-citation" id="advs7707-cit-0042">
Mancias J. D., Wang X., Gygi S. P., Harper J. W., Kimmelman A. C., <span class="ref-journal">Nature</span>
2014, <span class="ref-vol">509</span>, 105.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4180099/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24695223" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;volume=509&amp;publication_year=2014&amp;pages=105&amp;pmid=24695223&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0038">38. <span class="mixed-citation" id="advs7707-cit-0043">
Lee Y. S., Lee D. H., Choudry H. A., Bartlett D. L., Lee Y. J., <span class="ref-journal">Mol. Cancer Res.</span>
2018, <span class="ref-vol">16</span>, 1073.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6030493/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29592897" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cancer+Res.&amp;volume=16&amp;publication_year=2018&amp;pages=1073&amp;pmid=29592897&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0039">39. <span class="mixed-citation" id="advs7707-cit-0044">
Ri M., <span class="ref-journal">Int J Hematol</span>
2016, <span class="ref-vol">104</span>, 273.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/27169614" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Hematol&amp;volume=104&amp;publication_year=2016&amp;pages=273&amp;pmid=27169614&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0040">40. <span class="mixed-citation" id="advs7707-cit-0045">
Pakos&#x02010;Zebrucka K., Koryga I., Mnich K., Ljujic M., Samali A., Gorman A. M., <span class="ref-journal">EMBO Rep.</span>
2016, <span class="ref-vol">17</span>, 1374.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5048378/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27629041" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+Rep.&amp;volume=17&amp;publication_year=2016&amp;pages=1374&amp;pmid=27629041&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0041">41. <span class="mixed-citation" id="advs7707-cit-0046">
Ron D., Walter P., <span class="ref-journal">Nat. Rev. Mol. Cell Biol.</span>
2007, <span class="ref-vol">8</span>, 519.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/17565364" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Mol.+Cell+Biol.&amp;volume=8&amp;publication_year=2007&amp;pages=519&amp;pmid=17565364&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0042">42. <span class="mixed-citation" id="advs7707-cit-0047">
Hetz C., <span class="ref-journal">Nat. Rev. Mol. Cell Biol.</span>
2012, <span class="ref-vol">13</span>, 89.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/22251901" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Mol.+Cell+Biol.&amp;volume=13&amp;publication_year=2012&amp;pages=89&amp;pmid=22251901&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0043">43. <span class="mixed-citation" id="advs7707-cit-0048">
Dey S., Baird T. D., Zhou D., Palam L. R., Spandau D. F., Wek R. C., <span class="ref-journal">J. Biol. Chem.</span>
2010, <span class="ref-vol">285</span>, 33165.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2963398/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20732869" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;volume=285&amp;publication_year=2010&amp;pages=33165&amp;pmid=20732869&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0044">44. <span class="mixed-citation" id="advs7707-cit-0049">
Fusakio M. E., Willy J. A., Wang Y., Mirek E. T., Al Baghdadi R. J., Adams C. M., Anthony T. G., Wek R. C., <span class="ref-journal">Mol. Biol. Cell</span>
2016, <span class="ref-vol">27</span>, 1536.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4850040/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26960794" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Biol.+Cell&amp;volume=27&amp;publication_year=2016&amp;pages=1536&amp;pmid=26960794&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0045">45. <span class="mixed-citation" id="advs7707-cit-0050">
Han J., Back S. H., Hur J., Lin Y. H., Gildersleeve R., Shan J., Yuan C. L., Krokowski D., Wang S., Hatzoglou M., Kilberg M. S., Sartor M. A., Kaufman R. J., <span class="ref-journal">Nat. Cell Biol.</span>
2013, <span class="ref-vol">15</span>, 481.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3692270/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23624402" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Cell+Biol.&amp;volume=15&amp;publication_year=2013&amp;pages=481&amp;pmid=23624402&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0046">46. <span class="mixed-citation" id="advs7707-cit-0051">a) Hetz C., Papa F. R., <span class="ref-journal">Mol. Cell</span>
2018, <span class="ref-vol">69</span>, 169;
 [<a href="https://pubmed.ncbi.nlm.nih.gov/29107536" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;volume=69&amp;publication_year=2018&amp;pages=169&amp;pmid=29107536&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span><span class="mixed-citation" id="advs7707-cit-0052">b) Ghosh A. P., Klocke B. J., Ballestas M. E., Roth K. A., <span class="ref-journal">PLoS One</span>
2012, <span class="ref-vol">7</span>, e39586.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3386252/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22761832" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;volume=7&amp;publication_year=2012&amp;pages=e39586&amp;pmid=22761832&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0047">47. <span class="mixed-citation" id="advs7707-cit-0053">
Yoshiuchi K., Kaneto H., Matsuoka T. A., Kasami R., Kohno K., Iwawaki T., Nakatani Y., Yamasaki Y., Shimomura I., Matsuhisa M., <span class="ref-journal">Endocr. J.</span>
2009, <span class="ref-vol">56</span>, 1103.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/19789420" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Endocr.+J.&amp;volume=56&amp;publication_year=2009&amp;pages=1103&amp;pmid=19789420&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0048">48. <span class="mixed-citation" id="advs7707-cit-0054">
Renga B., Mencarelli A., Cipriani S., D'Amore C., Zampella A., Monti M. C., Distrutti E., Fiorucci S., <span class="ref-journal">Biochim. Biophys. Acta</span>
2011, <span class="ref-vol">1812</span>, 1522.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/21757002" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.+Biophys.+Acta&amp;volume=1812&amp;publication_year=2011&amp;pages=1522&amp;pmid=21757002&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0049">49. <span class="mixed-citation" id="advs7707-cit-0055">a) Gao M., Monian P., Quadri N., Ramasamy R., Jiang X., <span class="ref-journal">Mol. Cell</span>
2015, <span class="ref-vol">59</span>, 298;
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4506736/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26166707" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;volume=59&amp;publication_year=2015&amp;pages=298&amp;pmid=26166707&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span><span class="mixed-citation" id="advs7707-cit-0056">b) Gao M., Monian P., Jiang X., <span class="ref-journal">Oncotarget</span>
2015, <span class="ref-vol">6</span>, 35145.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4742090/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26387139" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;volume=6&amp;publication_year=2015&amp;pages=35145&amp;pmid=26387139&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0050">50. <span class="mixed-citation" id="advs7707-cit-0057">
Kim M., Gwak J., Hwang S., Yang S., Jeong S. M., <span class="ref-journal">Oncogene</span>
2019, <span class="ref-vol">38</span>, 4729.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/30765862" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;volume=38&amp;publication_year=2019&amp;pages=4729&amp;pmid=30765862&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0051">51. <span class="mixed-citation" id="advs7707-cit-0058">
Gao M., Yi J., Zhu J., Minikes A. M., Monian P., Thompson C. B., Jiang X., <span class="ref-journal">Mol. Cell</span>
2019, <span class="ref-vol">73</span>, 354.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6338496/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30581146" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell&amp;volume=73&amp;publication_year=2019&amp;pages=354&amp;pmid=30581146&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0052">52. <span class="mixed-citation" id="advs7707-cit-0059">
Kurmi K., Haigis M. C., <span class="ref-journal">Trends Cell Biol.</span>
2020, <span class="ref-vol">30</span>, 408.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7386658/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32302552" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Cell+Biol.&amp;volume=30&amp;publication_year=2020&amp;pages=408&amp;pmid=32302552&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0053">53. <span class="mixed-citation" id="advs7707-cit-0060">
Bonifacio V. D. B., Pereira S. A., Serpa J., Vicente J. B., <span class="ref-journal">Br. J. Cancer</span>
2021, <span class="ref-vol">124</span>, 862.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7921671/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33223534" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Cancer&amp;volume=124&amp;publication_year=2021&amp;pages=862&amp;pmid=33223534&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0054">54. <span class="mixed-citation" id="advs7707-cit-0061">
Stipanuk M. H., Ueki I., Dominy J. E. Jr., Simmons C. R., Hirschberger L. L., <span class="ref-journal">Amino Acids</span>
2009, <span class="ref-vol">37</span>, 55.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2736881/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19011731" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Amino+Acids&amp;volume=37&amp;publication_year=2009&amp;pages=55&amp;pmid=19011731&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0055">55. <span class="mixed-citation" id="advs7707-cit-0062">
Chen C. L., Hsu S. C., Ann D. K., Yen Y., Kung H. J., <span class="ref-journal">Cancers</span>
2021, <span class="ref-vol">13</span>, 3541.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8306961/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34298755" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancers&amp;volume=13&amp;publication_year=2021&amp;pages=3541&amp;pmid=34298755&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0056">56. <span class="mixed-citation" id="advs7707-cit-0063">
Elia I., Rossi M., Stegen S., Broekaert D., Doglioni G., van Gorsel M., Boon R., Escalona&#x02010;Noguero C., Torrekens S., Verfaillie C., Verbeken E., Carmeliet G., Fendt S. M., <span class="ref-journal">Nature</span>
2019, <span class="ref-vol">568</span>, 117.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6451642/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30814728" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;volume=568&amp;publication_year=2019&amp;pages=117&amp;pmid=30814728&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0057">57. <span class="mixed-citation" id="advs7707-cit-0064">
Das U. N., <span class="ref-journal">Cell Chem. Biol.</span>
2019, <span class="ref-vol">26</span>, 309.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/30901556" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Chem.+Biol.&amp;volume=26&amp;publication_year=2019&amp;pages=309&amp;pmid=30901556&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0058">58. <span class="mixed-citation" id="advs7707-cit-0065">
Lee J. Y., Nam M., Son H. Y., Hyun K., Jang S. Y., Kim J. W., Kim M. W., Jung Y., Jang E., Yoon S. J., Kim J., Kim J., Seo J., Min J. K., Oh K. J., Han B. S., Kim W. K., Bae K. H., Song J., Kim J., Huh Y. M., Hwang G. S., Lee E. W., Lee S. C., <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span>
2020, <span class="ref-vol">117</span>, 32433.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7768719/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33288688" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;volume=117&amp;publication_year=2020&amp;pages=32433&amp;pmid=33288688&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0059">59. <span class="mixed-citation" id="advs7707-cit-0066">
Jin J., Schorpp K., Samaga D., Unger K., Hadian K., Stockwell B. R., <span class="ref-journal">ACS Chem. Biol.</span>
2022, <span class="ref-vol">17</span>, 654.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8938922/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35230809" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=ACS+Chem.+Biol.&amp;volume=17&amp;publication_year=2022&amp;pages=654&amp;pmid=35230809&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0060">60. <span class="mixed-citation" id="advs7707-cit-0067">
d&#x02019;Azzo A., Annunziata I., <span class="ref-journal">Mol. Cell Oncol.</span>
2020, <span class="ref-vol">7</span>, 1685840.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7051135/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32158913" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell+Oncol.&amp;volume=7&amp;publication_year=2020&amp;pages=1685840&amp;pmid=32158913&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0061">61. <span class="mixed-citation" id="advs7707-cit-0068">a) Li G., Mongillo M., Chin K. T., Harding H., Ron D., Marks A. R., Tabas I., <span class="ref-journal">J. Cell Biol.</span>
2009, <span class="ref-vol">186</span>, 783;
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2753154/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19752026" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Cell+Biol.&amp;volume=186&amp;publication_year=2009&amp;pages=783&amp;pmid=19752026&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span><span class="mixed-citation" id="advs7707-cit-0069">b) Marciniak S. J., Yun C. Y., Oyadomari S., Novoa I., Zhang Y., Jungreis R., Nagata K., Harding H. P., Ron D., <span class="ref-journal">Genes Dev.</span>
2004, <span class="ref-vol">18</span>, 3066.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC535917/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15601821" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev.&amp;volume=18&amp;publication_year=2004&amp;pages=3066&amp;pmid=15601821&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0062">62. <span class="mixed-citation" id="advs7707-cit-0070">
Delgado&#x02010;Benito V., Berruezo&#x02010;Llacuna M., Altwasser R., Winkler W., Sundaravinayagam D., Balasubramanian S., Caganova M., Graf R., Rahjouei A., Henke M. T., Driesner M., Keller L., Prigione A., Janz M., Akalin A., Di Virgilio M., <span class="ref-journal">J. Exp. Med.</span>
2020, <span class="ref-vol">217</span>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7537392/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32609329" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Exp.+Med.&amp;volume=217&amp;publication_year=2020&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0063">63. <span class="mixed-citation" id="advs7707-cit-0071">
Wu P., Zhang X., Duan D., Zhao L., <span class="ref-journal">Oxid. Med. Cell Longev.</span>
2023, <span class="ref-vol">2023</span>, 1. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9836800/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36644574" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oxid.+Med.+Cell+Longev.&amp;volume=2023&amp;publication_year=2023&amp;pages=1&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0064">64. <span class="mixed-citation" id="advs7707-cit-0072">a) Verma N., Muller A. K., Kothari C., Panayotopoulou E., Kedan A., Selitrennik M., Mills G. B., Nguyen L. K., Shin S., Karn T., Holtrich U., Lev S., <span class="ref-journal">Cancer Res.</span>
2017, <span class="ref-vol">77</span>, 86;
 [<a href="https://pubmed.ncbi.nlm.nih.gov/27793840" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res.&amp;volume=77&amp;publication_year=2017&amp;pages=86&amp;pmid=27793840&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span><span class="mixed-citation" id="advs7707-cit-0073">b) Dixon S. J., Lee M. J., <span class="ref-journal">Nat. Cell Biol.</span>
2023, <span class="ref-vol">25</span>, 1720.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/37985871" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Cell+Biol.&amp;volume=25&amp;publication_year=2023&amp;pages=1720&amp;pmid=37985871&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0065">65. <span class="mixed-citation" id="advs7707-cit-0074">
Jaitin D. A., Kenigsberg E., Keren&#x02010;Shaul H., Elefant N., Paul F., Zaretsky I., Mildner A., Cohen N., Jung S., Tanay A., Amit I., <span class="ref-journal">Science</span>
2014, <span class="ref-vol">343</span>, 776.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4412462/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24531970" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;volume=343&amp;publication_year=2014&amp;pages=776&amp;pmid=24531970&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0066">66. <span class="mixed-citation" id="advs7707-cit-0075">
Keenan A. B., Torre D., Lachmann A., Leong A. K., Wojciechowicz M. L., Utti V., Jagodnik K. M., Kropiwnicki E., Wang Z., Ma'ayan A., <span class="ref-journal">Nucleic Acids Res.</span>
2019, <span class="ref-vol">47</span>, W212.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6602523/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31114921" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;volume=47&amp;publication_year=2019&amp;pages=W212&amp;pmid=31114921&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0067">67. <span class="mixed-citation" id="advs7707-cit-0076">
Kim S., Kedan A., Marom M., Gavert N., Keinan O., Selitrennik M., Laufman O., Lev S., <span class="ref-journal">EMBO Rep.</span>
2013, <span class="ref-vol">14</span>, 891.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3807235/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23897088" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+Rep.&amp;volume=14&amp;publication_year=2013&amp;pages=891&amp;pmid=23897088&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0068">68. <span class="mixed-citation" id="advs7707-cit-0077">a) Muller A. K., Kohler U. A., Trzebanski S., Vinik Y., Raj H. M., Girault J. A., Ben&#x02010;Chetrit N., Maraver A., Jung S., Lev S., <span class="ref-journal">Adv. Sci.</span>
2022, <span class="ref-vol">9</span>, e2105696; <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8948556/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35092356" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Adv.+Sci.&amp;volume=9&amp;publication_year=2022&amp;pages=e2105696&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span><span class="mixed-citation" id="advs7707-cit-0078">b) Verma N., Keinan O., Selitrennik M., Karn T., Filipits M., Lev S., <span class="ref-journal">Nat. Commun.</span>
2015, <span class="ref-vol">6</span>, 6064.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/25648557" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;volume=6&amp;publication_year=2015&amp;pages=6064&amp;pmid=25648557&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="advs7707-bib-0069">69. <span class="mixed-citation" id="advs7707-cit-0079">
Mai T. T., Hamai A., Hienzsch A., Caneque T., Muller S., Wicinski J., Cabaud O., Leroy C., David A., Acevedo V., Ryo A., Ginestier C., Birnbaum D., Charafe&#x02010;Jauffret E., Codogno P., Mehrpour M., Rodriguez R., <span class="ref-journal">Nat. Chem.</span>
2017, <span class="ref-vol">9</span>, 1025.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5890907/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28937680" ref="reftype=pubmed&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Chem.&amp;volume=9&amp;publication_year=2017&amp;pages=1025&amp;pmid=28937680&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11077643&amp;issue-id=461958&amp;journal-id=2933&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Advanced Science</span> are provided here courtesy of <strong>Wiley</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11077643/pdf/ADVS-11-2307263.pdf" class="int-view">PDF (18M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11077643/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11077643/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11077643" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11077643%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="hWjfpGKw7y1a1EAJhL71lTWqVnMggibZIp0GAw48s6ODPOy3cfeciku6sONzVUcq">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11077643%2F&amp;text=Programming%20a%20Ferroptosis%02010to%02010Apoptosis%20Transition%20Landscape%20Revealed%20Ferroptosis%20Biomarkers%20and%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11077643%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11077643/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11077643/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38441406/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11077643/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38441406/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11077643/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11077643/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC11077643/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
